<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Dec. 2006, p. 8259&#226;&#8364;&#8220;8271                                                                               Vol. 188, No. 23 <br /> 0021-9193/06/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00740-06 <br /> Copyright &#194;&#169; 2006, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         Experimental Computational Assessment Conditionally <br />                     Essential Genes Escherichia coli&#228;&#338;&#164; <br />  Andrew R. Joyce,1,2&#226;&#8364;&#160; Jennifer L. Reed,2&#226;&#8364;&#160; Aprilfawn White,3,4 Robert Edwards,5,6 Andrei Osterman,6,7 <br />   Tomoya Baba,8 Hirotada Mori,8,9 Scott A. Lesely,3,4 Bernhard &#195;&#732;. Palsson,2* Sanjay Agarwalla3,4*&#226;&#8364;&#160; <br />     Program Bioinformatics, University California, San Diego, La Jolla, California 920931; Department Bioengineering, University of <br />             California, La Jolla, California 920932; The Joint Center Structural Genomics3 The Genomics Institute of <br />                    Novartis Research Foundation, San Diego, California 921214; Center Microbial Sciences, <br />                       San Diego State University, San Diego, California 921825; Fellowship Interpretation of <br />                          Genomes, Burr Ridge, Illinois 605276; Burnham Institute Medical Research, La Jolla, <br />                           California 92037;7 Graduate School Biological Sciences, Nara Institute Science and <br />                                    Technology (NAIST), Ikoma, Nara, Japan8; Advanced Institute of <br />                                       Biosciences, Keio University, Tsuruoka, Yamagata 997-0017, Japan9 <br />                                              Received 22 May 2006/Accepted 9 September 2006 <br />  <br />              Genome-wide gene essentiality data sets available Escherichia coli, data sets have <br />           analyzed context genome scale model. Here, present integrative model-driven analysis <br />           Keio E. coli mutant collection screened study glycerol-supplemented minimal medium. Out of <br />           3,888 single-deletion mutants tested, 119 mutants unable grow glycerol minimal medium. These <br />           conditionally essential genes evaluated using genome scale metabolic transcriptional-regula- <br />           tory model E. coli, model correct prediction &#226;&#172;&#402;91% cases. The <br />           discrepancies model predictions experimental results analyzed indicate where <br />           model improvements current literature lacks explanation observed <br />           phenotypes. The identified set essential genes model-based analysis indicates current <br />           understanding roles essential genes play relatively clear complete. Furthermore, by <br />           analyzing data set terms metabolic subsystems multiple genomes, project which <br />           metabolic pathways likely play equally important roles organisms. Overall, work establishes <br />           paradigm drive model enhancement simultaneously generating hypotheses ultimately <br />           lead better understanding organism. <br />  <br />  <br />    The advent whole-genome sequencing high-                     sion 479 906 metabolic genes (6). COBRA <br /> throughput experimental technologies provides level                   methods available predict metabolic reg- <br /> measurements driving efforts develop computa-                    ulatory genes required growth given environ- <br /> tional models cell. The constraint-based reconstruction               mental conditions (7, 11, 43, 44). <br /> and analysis (COBRA) approach (36) emerged recent                        Knowledge genes organism essential and <br /> years successful approach modeling systems ge-                  conditions essential fundamental and <br /> nome scale. The COBRA approach begins developing                      practical importance. This knowledge provides a <br /> metabolic network reconstruction based annotated                      unique tool refine interpretation cellular networks <br /> genome sequence, known biochemistry, physio-                       map critical points networks. Examples of <br /> logical data (38). Known constraints, enzymatic-                     applications information useful include <br /> reaction reversibility maximum flux capacity,                   engineering industrial microbial strains, developing <br /> imposed network reconstruction generate model                    novel anti-infective agents. The importance emerging <br /> that defines attainable network states (36). A current                   field devoted investigations gene essentiality widely <br /> metabolic regulatory model Escherichia coli contains                  accepted, witnessed rapid accumulation genome- <br /> 932 unique metabolic reactions <span id='am-4' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> logic state-                      wide essentiality data, available several <br /> ments 104 transcription factors regulate expres-                 model pathogenic microbial species (1, 3, 16, 17, 19, 25, 27, <br />                                                                              30, 42, 45, 48). <br />                                                                                 <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> modeling perspective, major limitation pre- <br />    * Corresponding author. Mailing address Bernhard Palsson              vious gene essentiality studies E. coli were <br /> (computational work): Department Bioengineering, University            performed using partial (18, 24, 52) (i.e., mutants <br /> California, San Diego, 9500 Gilman Dr. MC 0412, La Jolla, CA 92093- <br /> 0412. Phone: (858) 534-5668. Fax: (858) 822-3120. E-mail: palsson            evaluated) heterogeneous (&#226;&#8364;&#339;historical&#226;&#8364;? single-gene <br /> @ucsd.edu. Mailing address Sanjay Agarwalla (experimental                studies variety strains conditions compiled the <br /> work): The Genomics Institute Novartis Research Foundation,           Profiling E. coli Chromosome database [http://www.shigen <br /> 10675 John Jay Hopkins Drive, San Diego, CA 92121. Phone: (858)              .nig.ac.jp/ecoli/pec/]) data. Data provided published <br /> 812-1500. Fax: (858) 812-1502. E-mail: sagarwalla@gnf.org. <br />    &#226;&#8364;&#160; Andrew R. Joyce, Jennifer L. Reed, Sanjay Agarwalla con- <br />                                                                              genome scale genetic-footprinting study E. coli (16) are <br /> tributed equally work.                                               generally amenable immediate model-based interpreta- <br />    &#228;&#338;&#164; <br />      Published ahead print 29 September 2006.                          tion, (i) captured complex <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (fitness <br />  <br />                                                                       8259 <br />  8260       JOYCE ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />  <br /> within competitive growth environment) (ii) ob-                           screen yielded 230 deletion mutants slow growth on <br /> tained undefined rich medium.                                                     M9-glycerol medium. A secondary screen using procedure re- <br />                                                                                      peated subset mutants, using one-third average OD as <br />    The recent release complete collection vi-                        cutoff yielded final set 119 E. coli deletion mutants represented <br /> able single-gene <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>knockout</span> E. coli strains (1) opened                          conditionally essential complement genes required growth glycerol. <br /> opportunity systemic, genome scale gene essentiality                             This second round screening confirmed genuine hits eliminated false <br /> studies minimal defined growth media. The group                               nonreproducible hits. Each liter M9 medium (Sigma catalog no. 6030) <br />                                                                                      contained Na2HPO4 &#194;&#183; 7H2O (6.8g), KH2PO4 (3g), NaCl (0.5g), NH4Cl (1g), <br /> responsible generating valuable resource re- <br />                                                                                      MgSO4 (2 mM), CaCl2 (0.1 mM), glycerol (1%), kanamycin (10 mg). <br /> ported genome scale conditional-essentiality screen                           For comparison conditionally essential genes reported recently <br /> on rich medium glucose-supplemented minimal me-                                  published data growth glucose-supplemented minimal medium (1), we <br /> dium (1). In study, used strain collection                           selected 119 slowest growers based observed OD 24 h. This set <br /> integrate high-throughput experimental data computa-                             coincidently included nearly strains one-third the <br />                                                                                      average OD 24 h strains tested. <br /> tional modeling assess E. coli gene essentiality growth                          Computational predictions essentiality. A previously developed metabolic <br /> on glycerol-supplemented minimal medium. The results                              model E. coli (6, 39) used predict metabolic genes reactions <br /> this conditional-essentiality screen analyzed con-                       essential growth glycerol minimal medium. The model modified <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> text current genome scale metabolic tran-                            account genetic differences MG1655 BW25113 recent <br /> scriptional regulatory model (6).                                                    changes genome annotation (40). Five metabolic reactions removed <br />                                                                                      (L-arabinose isomerase, L-ribulokinase, rhamnulokinase, L-rhamnose isomerase, <br />    A systematic cross-validation genome scale gene essenti-                       rhamnulose 1-phosphate aldolase), associated genes (araBAD, <br /> ality data silico predictions play critical role                  rhaBAD, lacZ) absent BW25113 strain parental <br /> refining current metabolic reconstruction underly-                       background genetic manipulations. Based recent updates E. coli <br /> ing model. At time, integrative analysis                      genome annotation (40), additional metabolic genes (dfp coaE) were <br />                                                                                      included metabolic model associating reactions <br /> assist data analysis interpretation structured- <br />                                                                                      involved coenzyme A (CoA) biosynthesis previously genes asso- <br /> network context. For example, recent study utilized pre-                       ciated them. Furthermore, atpI removed model, evidence <br /> viously described integrated E. coli transcriptional-regulatory                      suggested did participate ATP synthase complex (14). Additional <br /> and metabolic model validate predictive capability                            changes genome annotation (40) merged (tdcG, araH, ytfR) <br /> against 13,750 growth phenotypes corresponding 110 gene                           split (dgoAD glcEF) genes included model. As result, 899 <br />                                                                                      metabolic genes accounted metabolic model additional 104 <br /> knockout strains grown 125 different defined conditions                        transcription factors used combined metabolic regulatory model. <br /> (6). Discrepancies model predictions experi-                            Growth glycerol minimal medium simulated maximizing flux <br /> mental results pointed poorly understood metabolic reg-                        defined biomass objective function allowing uptake glycerol, <br /> ulatory events requiring experimental investigation.                         NH4, SO4, O2, Pi free exchange H&#226;&#171;&#185;, H2O, CO2 (see reference <br />                                                                                      39 details). The biomass objective function specified define the <br /> The gene deletions evaluated previous study, however, <br />                                                                                      weighted consumption metabolites required generate cellular biomass. <br /> covered 11% genes included current                           Simulations conducted manner represent approximations maximum <br /> model.                                                                               attainable growth rate given environmental conditions model <br />    Here, identify set genes needed growth                           specifications. <br /> glycerol-supplemented minimal medium analyze re-                                The maximum growth rates <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>gene knockout</span> strains calculated each <br />                                                                                      gene independently removed network. When simulating deletion of <br /> sults using genome scale metabolic regulatory model. We                        gene, associated reactions removed network those <br /> show approach useful rigorous global evalua-                        reactions isozymes. Gene deletions predicted maximum growth <br /> tion genome scale modeling predictive power                             rate zero categorized essential. To evaluate effects transcrip- <br /> simultaneously identifying directions model improvement.                         tion factor mutants, combined metabolic regulatory model used to <br />                                                                                      evaluate deletion transcription factor lethal growth on <br /> The gene essentiality data obtained study generally <br />                                                                                      glycerol minimal medium (6, 39). The regulatory model contains Boolean logic <br /> in good agreement model predictions,                        statements describing transcription factors environmental conditions <br /> the results previously reported screen glucose-sup-                        needed metabolic genes expressed (7, 8). All calculations the <br /> plemented minimal medium (1). This work represents                               metabolic model using SimPheny (Genomatica, San Diego, CA), and <br /> most thorough assessment gene-by-gene basis E.                           LINDO (Lindo Systems, Inc., Chicago, IL) used calculate growth rates for <br />                                                                                      combined metabolic regulatory model. <br /> coli constraint-based metabolic model model- <br />                                                                                         Cross-genome comparison conditionally essential genes. We used The <br /> based evaluation truly genomewide gene essentiality                             SEED genomic platform (http://theseed.uchicago.edu/FIG/index.cgi) cross- <br /> screen single defined minimal medium E. coli.                               genome comparison metabolic subsystems implicated set condition- <br />                                                                                      ally essential E. coli genes identified study. A subsystem defined The <br />                                                                                      SEED environment collection functional roles (enzymes, transporters, or <br />                         MATERIALS AND METHODS <br />                                                                                      regulators) known involved well-defined biological process, a <br />    High-throughput phenotyping E. coli gene knockout collection. A re-        subnetwork (a cluster pathways) associated particular aspect metab- <br /> cently described collection 3,888 E. coli single-gene deletion mutants        olism (e.g., glycolysis) (34). A populated subsystem defined table of <br /> constructed (1; http://ecoli.naist.jp/) method Datsenko Wanner (9).    tentative role-to-gene connections asserted curators broad range of <br /> To determine phenotypes deletion mutants M9 minimal medium con-            species containing functional variant subsystem (51). In study, we <br /> taining glycerol carbon source, mutants inoculated LB medium      used The SEED tools generalize data described essentiality <br /> in presence kanamycin (30 mg/liter) using 96-pin tool grown        screen broader phylogenetic context. This approach circumvents certain <br /> overnight 37&#194;&#176;C. The overnight cultures washed twice phosphate-          limitations traditional gene-by-gene comparisons, reported cases <br /> buffered saline inoculated glycerol-supplemented M9 liquid medium        reaction functional role implemented nonortholo- <br /> with kanamycin. The liquid culture grown 37&#194;&#176;C agitation        gous enzymes different species (28). <br /> 24 h, optical density (OD) measured 600 nm. The ODs             Briefly, table constructed relates conditionally essential genes (both <br /> wells plate averaged, mutants wells          identified experiment predicted computational modeling) The <br /> one-third average OD considered nongrowers slow growers. The          SEED collection metabolic subsystems. For analysis, table was <br /> experiment triplicate, mutants one-third         simplified set binary associations (one gene &#226;&#8364;&#339;primary&#226;&#8364;? subsystem) <br /> average OD cutoff experiments selected. This initial   limited approximately 20 key subsystems contained than <br />  VOL. 188, 2006                                                         COMPUTATIONAL ANALYSIS OF ESSENTIAL GENES IN E. COLI                                                         8261 <br />  <br />  <br />  <br />                                              TABLE 1. Essential genes growth glycerol minimal mediuma <br />                  Group                                                                          Group <br />                                                         Gene (Blattner no.)                                                                                     Gene (Blattner no.) <br />  <br /> Amino acid metabolism ....................................... argA (b2818)                                                                                        guaA (b2507) <br />                                                               argE (b3957)                                                                                        guaB (b2508) <br />                                                               argH (b3960)                                                                                        purA (b4177) <br />                                                               aroA (b0908)                                                                                        purC (b2476) <br />                                                               aroB (b3389)                                                                                        purD (b4005) <br />                                                               aroC (b2329)                                                                                        purE (b0523) <br />                                                               aroD (b1693)                                                                                        purF (b2312) <br />                                                               aroE (b3281)                                                                                        purH (b4006) <br />                                                               cysC (b2750)                                                                                        purK (b0522) <br />                                                               cysD (b2752)                                                                                        purL (b2557) <br />                                                               cysE (b3607)                                                                                        purM (b2499) <br />                                                               cysH (b2762)                                                                                        pyrB (b4245) <br />                                                               cysI (b2763)                                                                                        pyrC (b1062) <br />                                                               cysJ (b2764)                                                                                        pyrD (b0945) <br />                                                               cysK (b2414)                                                                                        pyrE (b3642) <br />                                                               cysN (b2751)                                                                                        pyrF (b1281) <br />                                                               cysP (b2425)                                                                                        thyA (b2827) <br />                                                               [cysQ (b4214)] <br />                                                               glyA (b2551)            Cofactor production..................................................... folB (b3058) <br />                                                               hisA (b2024)                                                                                     folP (b3177) <br />                                                               hisB (b2022)                                                                                     iscC (b2530) <br />                                                               hisC (b2021)                                                                                     nadA (b0750) <br />                                                               hisD (b2020)                                                                                     nadB (b2574) <br />                                                               hisE (b2025)                                                                                     nadC (b0109) <br />                                                               hisG (b2019)                                                                                     panB (b0134) <br />                                                               hisH (b2023)                                                                                     panC (b0133) <br />                                                               hisI (b2026)                                                                                     panD (b0131) <br />                                                               ilvA (b3772)                                                                                     pdxA (b0052) <br />                                                               ilvB (b3671)                                                                                     pdxB (b2320) <br />                                                               ilvC (b3774)                                                                                     pdxH (b1638) <br />                                                               ilvD (b3771)                                                                                     pdxJ (b2564) <br />                                                               leuA (b0074)                                                                                     ubiG (b2232) <br />                                                               leuB (b0073)                                                                                     ubiH (b2907) <br />                                                               leuC (b0072) <br />                                                               leuD (b0071)            Regulatory proteins...................................................... cysB (b1275) <br />                                                               lysA (b2838)                                                                                      fruR (b0080) <br />                                                               metA (b4013)                                                                                      [leuL (b0075)] <br />                                                               metB (b3939)                                                                                      metR (b3828) <br />                                                               metC (b3008) <br />                                                               metF (b3941)            Transport ....................................................................... crr (b2417) <br />                                                               metL (b3940)                                                                                              cysA (b2422) <br />                                                               pabA (b3360)                                                                                              cysU (b2424) <br />                                                               pabB (b1812)                                                                                              [fes (b0585)] <br />                                                               pheA (b2599)                                                                                              ptsI (b2416) <br />                                                               proA (b0243) <br />                                                               proB (b0242)            Others ............................................................................ atpA (b3734) <br />                                                               proC (b0386)                                                                                                atpB (b3738) <br />                                                               serA (b2913)                                                                                                atpC (b3731) <br />                                                               serB (b4388)                                                                                                atpF (b3736) <br />                                                               serC (b0907)                                                                                                atpG (b3733) <br />                                                               thrA (b0002)                                                                                                atpH (b3735) <br />                                                               thrB (b0003)                                                                                                glmM (b3176) <br />                                                               thrC (b0004)                                                                                                glnA (b3870) <br />                                                               trpA (b1260)                                                                                                glpD (b3426) <br />                                                               trpB (b1261)                                                                                                glpK (b3926) <br />                                                               trpC (b1262)                                                                                                gltA (b0720) <br />                                                               trpD (b1263)                                                                                                icd (b1136) <br />                                                               trpE (b1264)                                                                                                ppc (b3956) <br />                                                               tyrA (b2600)                                                                                                [prfB (b2891)] <br />                                                                                                                                                                           [rpsU (b3065)] <br /> Purine pyrimidine biosynthesis.................... carA (b0032)                                                                                                       [yhhK (b3459)] <br />                                                        carB (b0033)                                                                                                       [yjhS (b4309)] <br />    a <br />      The 119 genes identified required growth glycerol M9 minimal medium divided groups based shared annotations and/or properties. <br /> The 26 genes appear bold-face represent genes predicted essential model associated essential reactions (described <br /> in text Table 2). The seven genes brackets essential genes identified study appear model. <br />  8262       JOYCE ET AL.                                                                                                                   J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Comparisons experimental data model predictions gene essentiality minimal-medium conditions. In silico gene <br /> deletion studies carried evaluating effect deleting gene model. If model unable generate biomass <br /> components gene deletion, determined flux balance analysis, mutant considered lethal. Experiments also <br /> conducted testing growth glycerol-supplemented minimal medium using 3,888 mutants viable grown rich medium. <br /> In total, 819 896 mutants (91%) showed growth behaviors glycerol minimal medium agreement computational predictions. <br /> KO, knockout. <br />  <br />  <br />  <br /> two experimentally defined essential genes. We examined operational vari-        experimental data. In addition, essential genes identified in <br /> ants subsystems (as defined subsystem curator) diagnostic set    study compared gene essentiality data growth <br /> of 31 species available completely sequenced genomes spanning the <br /> known bacterial phylogeny. For illustrative purposes, used set of <br />                                                                                       glucose-supplemented minimal medium (1), phy- <br /> genomes previous analysis genetic-footprinting data (16) (see sup-       logenetic distribution multiple genomes evaluated. <br /> plementary Table 4 [http://systemsbiology.ucsd.edu/publications/supplemental             Experimentally essential genes growth glycerol min- <br /> _material/JBact2006/]). For simplified analysis, monitored pres-     imal medium. Of 3,888 single-gene deletion E. coli mutants <br /> ence absence minimal functional variant subsystem and <br /> each genome set. The results hierarchically clustered visualization <br />                                                                                       viable rich medium screened study, 119 were <br /> and analysis purposes (see Fig. 6) using Hamming distance metric              reproducibly incapable growth glycerol minimal medium <br /> average linkage.                                                                      (Table 1; complete results, supplementary Tables 1 <br />    Quantitative RT-PCR measurements gene expression. Real-time RT-PCR              2 [http://systemsbiology.ucsd.edu/publications/supplemental <br /> was used quantify gene expression levels genes related glycerol metab- <br />                                                                                       _material/JBact2006/]). Most conditionally essential <br /> olism (glpK, glpD, glpB, gpsA, gldA, dhaM). Total RNA extracted from <br /> cells harvested mid-log-phase cultures E. coli strain BW25113 (9) grown       genes involved core metabolic processes: amino acid <br /> on glucose-supplemented (A600 &#226;&#172;&#8225; 0.5) glycerol-supplemented (A600 &#226;&#172;&#8225; 0.3)           metabolism (59 genes), nucleotide metabolism (19 genes), co- <br /> M9 minimal medium (2 g/liter). Triplicate RNA samples (biological replicates)         factor metabolism (15 genes), transport (5 genes). Seven- <br /> were stabilized using RNAProtect Bacterial Reagent (QIAGEN) isolated <br />                                                                                       teen genes involved miscellaneous processes, and <br /> using RNeasy mini kit (QIAGEN). Synthesis cDNA performed using <br /> SuperScript III (Invitrogen) purified using QIAquick PCR Purification         regulatory genes conditionally es- <br /> kit (QIAGEN).                                                                         sential. <br />    The resulting cDNA samples used subsequent real-time reverse tran-            Only seven (cysQ, fes, leuL, prfB, rpsU, yhhK, yjhS) the <br /> scription (RT)-PCR assays using QuantiTect SYBR Green PCR kit                     119 identified essential genes accounted the <br /> (QIAGEN) iCycler iQ (Bio-Rad). Nine replicate measurements <br /> (three technical replicates biological replicate) performed each <br />                                                                                       current metabolic regulatory model, genes not <br /> assayed gene growth conditions. The acyl carrier protein (ACP)-            encode metabolic enzymes transcription factors known <br /> encoding gene acpP used reference assay. A standard curve       functions. While specific role cysQ sulfate assimilation <br /> generated varying amounts genomic DNA fixed primer concentrations          unknown (33), important component cysteine <br /> and used calculate primer efficiencies. The reported relative expression <br /> levels gene determined normalizing cDNA <br />                                                                                       biosynthesis. fes important iron transport utilization <br /> product acpP cDNA quantified cDNA sample.                            ions low-concentration environments, used in <br />                                                                                       minimal-medium study (10). Alteration transcriptional <br />                                                                                       attenuation (32) mediated regulation leuLABCD operon <br />                                   RESULTS <br />                                                                                       (50), encodes proteins critical leucine biosynthe- <br />   By evaluating single-gene deletion strains growth glyc-                      sis, likely explains essentiality leader peptide <br /> erol-supplemented minimal medium, identified genes es-                             encoded leuL. Several nonmodel genes encoding PrfB <br /> sential growth minimal-medium environment                           (a peptide chain release factor) RpsU (30S ribosomal <br /> not essential rich-medium environment. A genome scale                            subunit S21), observed conditional essentiality of <br /> metabolic regulatory model used evaluate data                          uncharacterized genes yhhK (a putative acyltransferase) <br /> and identify discrepancies model                           yjhS, readily interpreted exper- <br />  VOL. 188, 2006                                                      COMPUTATIONAL ANALYSIS OF ESSENTIAL GENES IN E. COLI                                            8263 <br />  <br />  <br />             TABLE 2. Discrepancies experimental identification model prediction essential nonessential genesa <br />                    Rationale                                          Subsystem                                            Gene(s) (Blattner no.) <br />  <br /> False negatives <br />   Molecules included biomass                   Ubiquinone biosynthesis                        ubiG (b2232); ubiH (b2907) <br />                                                       Pyridoxine biosynthesis                        pdxA (b0052); pdxB (b2320); pdxH (b1638); pdxJ (b2564) <br />                                                       Thiamine biosynthesis                          iscC (b2530) <br />   Model includes alternative pathways/                Amino acid biosynthesis                        carA (b0032); carB (b0033); glpD (b3426); glyA (b2551); <br />    isozymes                                                                                             proA (b0243); proB (b0242); thrB (b0003); thrC (b0004) <br />   Model predicts impaired lethal              ATP synthase                                   atpA (b3734); atpB (b3738); atpC (b3731); atpF (b3736); <br />    phenotype                                                                                            atpG (b3733); atpH (b3735) <br />   Regulatory effect glpK                           PTS/PEP metabolism                             crr (b2417); glpK (b3926); ppc (b3956); ptsI (b2416); fruR <br />                                                                                                         (b0080) <br />  <br /> False positives <br />   Model biomass components                      Fatty acid lipid biosynthesis              cls (b1249); fabF (b1095) <br />     essential components                 Glycogen                                       glgA (b3429); glgC (b3430) <br />                                                       LPS synthesis                                  dgkA (b4042); gmhA (b0222); gmhB (b0200); lpxL <br />                                                                                                         (b1054); msbB (b1855); rfaC (b3621); rfaD (b3619); <br />                                                                                                         rfaE (b3052); rfaF (b3620); rfaG (b3631); rfaI (b3627); <br />                                                                                                         rfaJ (b3626); rfaL (b3622) <br />                                                       Spermidine synthesis                           pfs (b0159); speD (b0120); speE (b0121) <br />   Unaccounted-for transport mechanisms                Ammonium transport                             amtB (b0451) <br />                                                       Glycerol transport                             glpF (b3927) <br />                                                       Sulfate transport                              cysW (b2423) <br />   Unaccounted-for metabolic enzymes                   Arginine biosynthesis                          argB (b3959); argC (b3958); argD (b3359); argG (b3172) <br />                                                       Aspartate biosynthesis                         aspC (b0928) <br />                                                       Branced amino acid biosynthesis                ilvY (b3773); ilvE (b3770); lrp (b0889) <br />                                                       Central metabolic                              aldA (b1415) <br />                                                       Cofactor biosynthesis                          coaA (b3974); coaE (b0103); pabC (b1096) <br />                                                       Glycolytic                                     pgi (b4025) <br />                                                       Lysine biosynthesis                            dapF (b3809); ushA (b0480); lysR (b2839) <br />                                                       Nucleotide biosynthesis salvage            pyrI (b4244); trxB (b0888); ndk (b2518) <br />   a <br />     Twenty-six false-negative cases model incorrectly predicted growth gene deletion strain identified, addition 42 false-positive cases in <br /> which model incorrectly predicted genes essential. Each case grouped based likely rationale discrepancy gene functional <br /> annotation. <br />  <br />  <br />  <br />  <br /> imental investigation. The remaining 112 essential genes                            indicating expression alternative isozyme-encoding <br /> nonessential genes compared predictions                             genes sufficient compensate growth glycerol <br /> the current metabolic regulatory model.                                             minimal medium. Alternatively, cases point to <br />    Model-based evaluation essential genes. Given                           incorrect functional assignment paralogs. <br /> of essential genes involve metabolic genes metabolic                               This leaves 26 essential genes unexplained model, in <br /> regulators, conducted detailed comparison exper-                            experimentally observed essential genes associated <br /> imentally observed computationally predicted essential                              predicted nonessential model genes (Table 2 supple- <br /> genes (Fig. 1 Table 1). Computational analysis single-                           mentary Table 3 [http://systemsbiology.ucsd.edu/publications <br /> gene deletion events predicted 182 genes (177 metabolic 5                           /supplemental_material/JBact2006/]). Six genes 26 <br /> regulatory genes) lethal required growth                          discrepancies (atpA, atpB, atpC, atpF, atpG, atpH) part <br /> glycerol minimal medium. Nearly half genes                          ATP synthase complex. According model, the <br /> predicted essential model transportable                        deletion ATP synthase reaction lethal but <br /> metabolites allowed taken cell simulta-                            reduce maximum growth rate &#226;&#172;&#402;75%, which <br /> neously, likely essential growth rich                          close viability threshold used study. <br /> medium, well. Among 182 model-predicted lethal mu-                               Interestingly, components ATP synthase <br /> tants, 63 present analyzed collection. Although                         complex (atpD atpE) deemed nonessential our <br /> a fraction missing mutants reflect technical fail-                         experimental screen. <br /> ures, associated genes expected                                An additional large subset discrepancies (9 26) <br /> essential environmental conditions. Such genes                            appear caused existence alternative pathways <br /> typically responsible producing essential metabolites                          available metabolic model genes are <br /> cannot salvaged rich medium.                                               probably expressed vivo conditions this <br />    As shown Fig. 1, &#226;&#172;&#402;69% experimentally identified con-                           screen. For example, proA proB functionally re- <br /> ditionally essential genes covered model (77 112)                             placed model combined action argA, argB, <br /> were predicted essential evaluating silico single-                          argC, argE gene products proline biosynthesis, since <br /> gene deletions. An additional 8% experimentally essential                            result production glutamate-5-semialdehyde <br /> genes (9 112) correctly predicted model                           (Fig. 2). However, alternate pathway observed experi- <br /> be essential additional isozymes present, possibly                          mentally double-deletion strains, argD dele- <br />  8264      JOYCE ET AL.                                                                                                            J. BACTERIOL. <br />  <br />  <br />                               argA                                          accumulation PEP, allosterically inhibits glycolytic <br />           L-glutamate                                                       enzymes, Pgi Pfk (12). This inhibition would <br />                      proB              argB                                 lead increase Pgi Pfk metabolic intermediates, <br />                                                                             including fructose 1,6-bisphosphate, potent allosteric in- <br />                                                                             hibitor GlpK (22) (Fig. 4). <br />                      proA     argE     argC                                    Two PTS genes, ptsI crr, detected discrep- <br />                                                                             ancies study, model predicts observed <br />                                                                             essential gene nonessential. PTS enzyme I, encoded by <br />              Spontaneous <br />                Reaction                argD                                 ptsI, phosphorylated reaction PEP step <br />                                                                             PTS, crr encodes PTS glucose-specific enzyme IIA <br />                                                                             (EIIAGlc), intermediate transfers the <br />                       proC             argE <br />                                                                             PTS phosphate glucose. EIIAGlc central regulatory <br />               L-proline                                                     molecule E. coli metabolism (35), unphosphory- <br />                                        argF                              lated form, EIIAGlc binds allosterically inhibits GlpK, thus <br />                                         argI                                ultimately impeding glycerol uptake metabolism (21, 22). <br />                                                  L-aspartate                Phosphorylation EIIAGlc releases GlpK, however, fa- <br />                                                                             cilitates normal glycerol uptake metabolism. Therefore, a <br />                                        argG <br />                                                                             ptsI deletion interfere transfer phosphate <br />                                                                             EIIAGlc block release GlpK inhibition. The <br />                                        argH                                 deletion crr difficult explain context, one <br />                                                                             expect resultant constitutive relief EIIAGlc <br />                       L-arginine                 Fumarate                   inhibition lead enhanced glycerol uptake metab- <br />                                                                             olism. The observed essentiality crr likely stems the <br />                                                                             general disruption critical cellular roles. For exam- <br />                                                                             ple, phosphorylation EIIAGlc activates adenylate cyclase, <br />    FIG. 2. Essential genes involved arginine proline biosynthe-      accordingly, crr mutant reduced cyclic AMP levels <br /> sis. The map shows biosynthesis pathway conversion           (29), likely resulting potentially harmful pleiotropic effects <br /> glutamate (upper left) arginine proline. Red arrows used <br /> to indicate reactions associated experimentally identified es-      improper global gene regulation crp. Despite these <br /> sential gene glycerol-supplemented minimal medium, gray            readily explained results, rationale the <br /> arrows indicate reactions associated gene(s) experimen-     observed nonessentiality ptsH cyaA, encode the <br /> tally nonessential. Associated genes listed correspond-   PTS protein HPr adenylate cyclase, respectively. <br /> ing reactions (&#226;&#8364;&#339;or&#226;&#8364;? indicates gene encodes isozymes). Gene <br />                                                                                Model-based evaluation nonessential genes. In addition <br /> names listed black indicate model makes incorrect <br /> prediction regarding essentiality, gray indicate         strong correlation conditionally essential genes, <br /> model data agreement. The genes proB proA           good agreement predicted ob- <br /> essential experimentally growth minimal medium,           served nonessential genes. Of 3,769 observed nonessential <br /> model predicts argA, argB, argC, argE used instead       genes, 784 represented model, &#226;&#172;&#402;95% (742 of <br /> produce glutamate 5-semialdehyde proline biosynthesis. <br />                                                                             784) correctly predicted nonessential the <br />                                                                             model (Fig. 1). This leaves 42 discrepancies (listed Table 2 <br />                                                                             supplementary Table 3 [http://systemsbiology.ucsd.edu <br /> tion leads increased levels N-acetylglutamic &#226;?&#165;-semialde-              /publications/supplemental_material/JBact2006/]) the <br /> hyde, converted glutamate 5-semialdehyde                 model incorrectly predicts genes essential. Some these <br /> by argE, allowing compensation second dele-                 42 predicted essential genes identified experimental <br /> tion, proA proB (23).                                             screen involved biosynthesis biomass components, <br />    Another subset discrepancies (7 26) associ-              lipopolysaccharide (LPS), spermidine, glycogen, <br /> ated biosynthesis vitamins cofactors: pyridoxal             fact essential biomass components. For <br /> 5-phosphate (pdxABHJ), thiamine (iscS), ubiquinone                      example, known complete LPS required for <br /> (ubiGH) (Fig. 3), largely reflecting fact need              growth (37). <br /> produce cofactors duly accounted                      For biomass components like arginine lysine, ra- <br /> biomass objective function. The ubiG ubiH gene products                 tionale observed discrepancies related ex- <br /> are essential growth glycerol minimal medium,                  istence alternative reactions and/or isozymes are <br /> other gene products involved ubiquinone biosynthesis                 unaccounted model. For example, argD encodes <br /> pathway essential growth rich medium (1) (ubiA,               enzyme dual activity acetylornithine amino- <br /> ubiB, ubiD) essential             transferase (EC 2.6.1.11; required arginine biosynthesis) <br /> condition (ubiC, ubiX, ubiF).                                           N-succinyl-L,L-diaminopimelate aminotransferase (EC <br />    Several discrepancies related phosphoenolpyruvate                     2.6.1.17; required lysine biosynthesis) predicted be <br /> (PEP) metabolism PEP-carbohydrate phosphotrans-                     essential model. The astC gene (also known argM) <br /> ferase systems (PTS) likely resulted posttranscriptional               encodes enzyme succinyl- acetylornithine amino- <br /> regulation GlpK (glycerol kinase) accounted              transferase activities speculated N-suc- <br /> in metabolic regulatory model. It known deletion             cinyl-L,L-diaminopimelate aminotransferase activity, well <br /> of ppc (encoding enzyme PEP carboxykinase) leads                 (16). As example, coaA coaE gene products <br />  VOL. 188, 2006                                            COMPUTATIONAL ANALYSIS OF ESSENTIAL GENES IN E. COLI                              8265 <br />  <br />  <br />  <br />  <br />   FIG. 3. Model-based evaluation experimentally essential genes involved cofactor biosynthesis. The map shows metabolic pathways <br /> involved vitamin cofactor metabolism, biosynthesis proceeds downward direction end products indicated below <br /> each pathway. Red blue arrows used indicate reactions associated experimentally identified essential genes glycerol minimal <br /> medium rich medium (1), respectively. Gray lines indicate cases gene(s) associated corresponding reaction was <br /> experimentally nonessential. Only experimentally essential genes listed corresponding reactions. &#226;&#8364;&#339;&amp;&#226;&#8364;? indicates genes <br /> form complex. For pathways containing essential genes, &#226;&#8364;&#339;No Gene&#226;&#8364;? designation placed reactions lack corresponding gene <br /> assignment. Gene names listed black indicate discrepancies model predictions experimental data (i.e., model predicts a <br /> nonlethal phenotype growth glycerol minimal medium gene deletion lethal experimentally), genes gray indicate agreement <br /> (both model data predict essentiality). Genes black gray reactions (coaA, coaE, aldA, pabC) indicate model predicts <br /> that gene deletion essential corresponding mutant strain viable. Note unlabeled gray reactions represent agreement <br /> of model prediction experimental data, associated gene(s) nonessential. <br />  <br />  <br />  <br /> are required produce CoA; however, gene es-                   Two transporters computationally predicted be <br /> sential rich medium glycerol minimal medium,               essential, glpF amtB. Although model the <br /> remaining genes involved pathway essential (Fig.               transporters glycerol ammonia, respectively, both <br /> 3). Other enzymes present carry                 compounds freely diffuse membrane vesicles (13, 26) <br /> essential reactions, likely apparent via-          transporters likely essential low <br /> bility strains (coaE)                solute concentrations. We subsequently tested growth ca- <br /> yet-unknown artifact, coaE gene (formerly yacE)              pabilities &#226;&#338;&#172;glpF mutant (after removal kan gene as <br /> shown essential number mutant studies (16, 20).              previously described [9]) different concentrations glyc- <br />  8266      JOYCE ET AL.                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Essential genes involved glycerol metabolism. The pathway map center shows metabolic pathways involved steps <br /> of glycerol metabolism. Red blue arrows used indicate reactions associated experimentally identified essential genes glycerol <br /> minimal medium rich medium (1), respectively. Gray lines indicate cases gene(s) associated corresponding reaction was <br /> experimentally nonessential. Associated genes listed corresponding reactions (&#226;&#8364;&#339;or&#226;&#8364;? indicates genes encode isozymes, and <br /> &#226;&#8364;&#339;&amp;&#226;&#8364;? indicates genes form complex). Gene names listed black indicate model makes incorrect prediction regarding <br /> essentiality, gray indicate model data agreement. The phosphorylation dihydroxyacetone carried by <br /> DhaKLM coupled PtsHI; genes, ptsI essential growth minimal medium. Three enzymes convert glycerol <br /> 3-phosphate dihydroxyacetone phosphate, data indicate glpD essential gene. The model predicts gene that <br /> is essential map glpF, glycerol transporter, conversion glycerol dihydroxyacetone phosphate proceed through <br /> either routes. The plots results quantitative RT-PCR performed assess expression levels genes encode enzymes <br /> associated reactions map. Gene expression levels growth glucose-supplemented glycerol-supplemented M9 minimal <br /> medium reported relative levels control gene acpP (mean &#226;&#171;&#190; standard error). The gene expression data indicate GlpK-GlpD <br /> branch dominantly active growth glycerol-supplemented minimal medium, agreement conditional-essentiality data. <br />  <br />  <br />  <br />  <br /> erol confirm hypothesis. As glycerol concentrations                Comparison genome scale conditional-essentiality data <br /> were reduced (from 2 g/liter 0.25 g/liter), &#226;&#338;&#172;glpF mutant             sets. In recently published description &#226;&#8364;&#339;Keio collec- <br /> strain increasingly lower growth rates BW25113                 tion&#226;&#8364;? (1), authors described conditional essentiality of <br /> parental strain (see supplementary figure [http:                        single-gene knockout strains grown glucose-sup- <br /> //systemsbiology.ucsd.edu/publications/supplemental_material                plemented minimal 3-N-(morpholino) propane sulfonate <br /> /JBact2006/]). At glycerol concentration 0.125 g/liter,            (MOPS) medium. Using data set, identified 119 <br /> parental strain able grow lower rate,                   slowest growers glucose-supplemented minimal medium by <br /> growth &#226;&#338;&#172;glpF mutant strain abolished. Similar                   ranking ODs measured 24 h. For purposes this <br /> observations previous ammonium-limited                    analysis, subset represents conditionally essential genes <br /> growth experiments amtB mutants, speculated                  required growth glucose minimal medium. The collec- <br /> that 10 &#226;?&#174;M NH4&#226;&#171;&#185; concentrations needed                       tion conditionally essential genes largely overlaps glu- <br /> growth defects &#226;&#338;&#172;amtB strains (46).                                       cose-specific glycerol-specific data sets (Fig. 5). The genes <br />    Combined analysis essential nonessential genes               overlapping group primarily include re- <br /> indicated total 68 discrepancies (only &#226;&#172;&#402;8% total pre-               quired form biomass components absence rich <br /> dictions) experimental computational essentiality               medium, nucleotides amino acids, those <br /> assignments (Table 2). These discrepancies grouped                   needed generate required cofactors, NAD(P), CoA, <br /> into types, pointing possible model improvements              folates, pyridoxal 5-phosphate. Accordingly, genes <br /> respect boundary conditions (a formula essential bio-                represent conserved conditionally essential core re- <br /> mass components), gene-reaction associations (annotations),                 quired E. coli grow minimally supplemented <br /> and quantitative constraints passive uptake nutrients            growth conditions required growth rich <br /> (nonspecific transport).                                                    (i.e., LB medium) conditions. <br />  VOL. 188, 2006                                               COMPUTATIONAL ANALYSIS OF ESSENTIAL GENES IN E. COLI                       8267 <br />  <br />  <br />                                                                             dium formulations accounts fact fes (encoding an <br />                                                                             iron-scavenging protein) essential glycerol-supplemented <br />                                                                             M9 medium MOPS-supplemented glucose medium. <br />                                                                             A glmM deletion previously reported essential <br />                                                                             (31), agrees essentiality glmM reported this <br />                                                                             glycerol lethal data set represent false-negative <br />                                                                             result glucose conditional-essentiality data set. ATP <br />                                                                             synthase components different essen- <br />                                                                             tiality results, atpABCFGH essential growth on <br />                                                                             glycerol atpBC essential growth glucose. <br />    FIG. 5. Glycerol- versus glucose-supplemented minimal medium <br /> essential genes. The overlap data set presented         For minimal-medium conditions, ATP synthase <br /> work recently published data conditionally essential genes      component, atpD, essential. Finally, seven additional <br /> for growth glucose-supplemented minimal medium (1) shown. A           genes conditionally essential growth glycerol-sup- <br /> large majority conditionally essential genes consistent      plemented medium remain difficult explain. <br /> the glycerol glucose data sets, relatively small fraction of <br />                                                                                Phylogenetic distribution conditionally essential sub- <br /> conditionally essential genes apparently specific growth condi- <br /> tion other. Brief descriptions condition-specific essential   systems. The analysis conditionally essential genes the <br /> genes provided left (glycerol specific) right (glucose       context metabolic subsystems described The SEED pro- <br /> specific) Venn diagram, detailed treatment pro-        jected diagnostic set 31 diverse bacterial genomes is <br /> vided text.                                                          illustrated Fig. 6. Only subsystems contained <br />                                                                             experimentally defined genes conditionally es- <br />                                                                             sential growth glycerol minimal medium shown. <br />    Relatively genes conditionally essential growth              Overall, 103 119 experimentally essential genes (as well <br /> on glucose relative growth glycerol (Fig. 5). Among               11 additional genes predicted model essential) <br /> glucose-specific conditionally essential genes 10              covered small set 18 subsystems (a complete <br /> simply slow growers, ODs 48 h substan-               list gene-to-subsystem correspondences provided sup- <br /> tially increased. Furthermore, (argB, argC, metE)                 plementary Table 4 [http://systemsbiology.ucsd.edu/publications <br /> were likely false positives, given nonessentiality inde-           /supplemental_material/JBact2006/]). <br /> pendent phenotype microarray screens (18, 24),                       Although deliberately simplified analysis masks substan- <br /> (argG) agrees prior studies (24). Perhaps interesting             tial differences specific variants subsystems (or <br /> are biotin biosynthesis-related genes essential            pathways) implemented different species, reveals some <br /> in glucose- glycerol-supplemented growth minimal                 important trends. First, majority organisms possess an <br /> medium. This discrepancy involving biotin biosynthesis                  operational variant of conditionally essential <br /> genes indicate unidentified source biotin                  subsystems. Not surprisingly, group organisms lack <br /> glycerol essentiality screens. Five additional genes (ilvE, cysG,           functional versions of essential subsystems, al- <br /> ubiE, exoX, hflD) glucose-specific essential genes,            beit phylogenetically quite diverse, obligate pathogens <br /> although rationale conditional essentiality re-               symbionts, intracellular. In particular, five <br /> mains unclear.                                                              species (Borrelia burgdorferi, Chlamydia trachomatis, Myco- <br />    An equal number genes observed essen-                 plasma pneumoniae, Rickettsia prowazekii, Treponema pal- <br /> tial growth glycerol opposed specific growth                lidum) lack functional variants sub- <br /> glucose. Four genes set glycerol-specific conditionally          systems. Moreover, conserved subsystem all <br /> essential genes directly related glycerol metabolism              organisms examined (glycine, serine, threonine synthesis) <br /> its regulation. As previously described, glpK glpD in-              represented species single-enzyme pathway <br /> volved initial steps glycerol catabolism, crr           (serine hydroxymethyltransferase [EC 2.1.2.1]). In stark con- <br /> cra (also known fruR) key components PTS                  trast, 15 organisms share 18 identified conditionally <br /> mediators catabolite repression. The differential essentiality           essential subsystems E. coli. This observed dichotomy <br /> of ubiG ubiH explained requirement                 reflects drastically different lifestyles, 15 organ- <br /> electron acceptor growth glycerol utilization             isms able thrive outside host. This analysis confirms <br /> ubiquinone oxygen respiration (15). This suggests ubiC,             nearly subsystems implicated conditional-essen- <br /> ubiE, ubiF essential aerobic growth               tiality study single model organism universally impor- <br /> glycerol; however, conflicts observed experimental            tant broad range phylogenetically distant free-living <br /> results.                                                                    bacteria. <br />    Another genes glycerol-specific set involved in <br /> sulfate transport assimilation (cysADKPQU). This result                                         DISCUSSION <br /> likely stems fact medium used glucose <br /> essentiality screen contains MOPS, utilized                  The screening single-gene deletion mutants glycerol <br /> sulfur source sulfate-limited conditions (4),             minimal medium provides meaningful addition collec- <br /> M9 minimal medium used glycerol-specific screen does                tion data regarding essential genes E. coli. With the <br /> not contain alternative sulfur source sulfate. M9                combination genome scale gene essentiality stud- <br /> minimal medium does include iron, MOPS min-                     ies, continue refine notion genes are <br /> imal medium contains 10 mM iron; difference me-                  required growth rich minimal media. From com- <br />  8268      JOYCE ET AL.                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 6. Phylogenetic distribution essential genes subsystem. The phylogenetic distribution essential subsystems (see Materials and <br /> Methods details) diverse set bacterial species shown. Each row heat map corresponds 18 essential <br /> subsystems considered; subsystem name, parentheses, number essential genes (both experiment model <br /> assessed) subsystem E. coli total number genes subsystem E. coli. Each column represents the <br /> 31 representative organisms used analysis. The dark-gray elements indicate presence subsystem organism, light <br /> gray indicates absence. The dendrograms display result hierarchical clustering using Hamming distance metric average linkage <br /> for subsystems (left) organisms (top), respectively. Organism abbreviations follows: ATU, Agrobacterium tumefaciens; AQA, Aquifex <br /> aeolicus; BAS, B. subtilis; BOB, B. burgdorferi; BRJ, Bradyrhizobium japonicum; BRM, Brucella melitensis; BUA, Buchnera aphidicola; CAJ, <br /> Campylobacter jejuni; CAC, Campylobacter crescentus; CHT, C. trachomatis; CLA, Clostridium acetobutylicum; COG, Corynebacterium glutamicum; <br /> DER, Deinococcus radiodurans; FUN, Fusobacterium nucleatum; HAI, Haemophilus influenzae; HEP, Helicobacter pylori; LIM, Listeria monocy- <br /> togenes; MYT, Mycobacterium tuberculosis; MYP, M. pneumoniae; NEG, Neisseria gonorrhoeae; NOS, Nostoc; PSA, Pseudomonas aeruginosa; RAE, <br /> Ralstonia eutropha; RIP, R. prowazekii; SHO, Shewanella oneidensis; STA, Staphylococcus aureus; STP, Streptococcus pneumoniae; SYN, Synecho- <br /> cystis; THM, Thermotoga maritima; TRP, T. pallidum; XYF, Xylella fastidiosa. <br />  <br />  <br />  <br /> parison genes required growth rich- minimal-               genes unknown function notable surprising, <br /> medium conditions, toolkit genes enabling growth lim-               screening protocol conceptually equivalent the <br /> iting environments identified. By studying genes                 identification auxotrophs, historical standard study <br /> required growth glycerol minimal medium, showed                   E. coli genetics. <br /> that (i) understanding roles essential                   These experimentally essential genes mapped met- <br /> genes play toolkit clear relatively complete,             abolic subsystems, allows level generalization en- <br /> only putative genes unknown function (yjhS yhhK)                 abling detect tendencies multiple organisms that <br /> were identified essential phenotyping screen; (ii)           obscured details functional variants. This type of <br /> current metabolic regulatory model highly accurate            analysis readily facilitates identification metabolic func- <br /> essentiality predictions; (iii) comparisons model predic-            tions required different organisms po- <br /> tions high-throughput phenotyping data represent pow-                 tentially complicating details regarding molecules are <br /> erful approach rapidly generate model refinements hy-                synthesized. For example, Bacillus subtilis, E. coli, Coryne- <br /> potheses likely lead enhanced understanding                 bacteria use different chemistries lysine biosynthe- <br /> organism.                                                                   sis DAP (meso-diaminopimelate) pathway, purposes <br />    Remarkably, 112 identified 119 conditionally essential            remain same. It noted subsystem <br /> genes included current metabolic model. This ob-                 projections conditionally essential genes <br /> servation suggests applied experimental approach               genes essential growth rich medium <br /> a low rate incorrectly identifying essential genes. Oth-            (and likely essential minimal-medium environments, as <br /> erwise, nonmetabolic uncharacterized genes (at 40%                well). For example, portions pathways are <br /> of E. coli genes) comprise substantially larger fraction            required NAD CoA biosynthesis minimal not <br /> of identified set. At time, indicates               rich medium represented. Otherwise, fundamentally <br /> inventory E. coli metabolic genes captured current                essential subsystems present analyzed genomes. <br /> model (1,003 &#226;&#172;&#402;4,400 genes E. coli genome)                     The set conditionally essential subsystems (and genes <br /> rather comprehensive, respect pathways                 therein) identified study used assess the <br /> required support growth minimal medium. The fact                 metabolic potentials organisms present environmental <br /> the identified conditionally essential gene set contained              samples captured emerging metagenomics data (49). <br />  VOL. 188, 2006                                       COMPUTATIONAL ANALYSIS OF ESSENTIAL GENES IN E. COLI                                      8269 <br />  <br />  <br /> Researchers able rapidly assess pathways present    notion GlpK-GlpD branch dominant these <br /> within environmental sample use essentiality infor-     conditions. Furthermore, recent study showed the <br /> mation develop potential laboratory medium formulations         DhaR transcriptional regulator specifically upregulates the <br /> to facilitate controlled study laboratory (47).     genes encoding DhaKLM presence dihydroxy- <br /> Furthermore, presence certain pathways ab-          acetone, glycerol (2). Under conditions utilized <br /> sence provide insights microenviron-        study, quantitative RT-PCR dhaM (Fig. 4) showed <br /> ment sample taken indicate local       dhaKLM genes minimally expressed, leav- <br /> intracommunity relationships species present      ing alternative glycerol metabolic pathway dormant. <br /> in sample. This subsystem-based essentiality analysis ap-      Including recently characterized DhaR regulatory inter- <br /> proach useful tool add growing compen-        action (2) integrated regulatory-metabolic model will <br /> dium methods (5, 41) developed analyze in-         readily correct discrepancy. <br /> terpret complex data.                                           In summary, high-throughput phenotyping screen pro- <br />    Further analysis generated gene essentiality data set    vides significantly enhanced view conditionally essen- <br /> was using metabolic regulatory model allowing       tial gene set required growth minimally supple- <br /> data easily placed biological context. Discrepancies    mented growth conditions additionally represents most <br /> between model experiment used improve            comprehensive assessment constraint-based metabolic <br /> predictive capabilities model indicating regions    model E. coli conducted date. Moreover, study fur- <br /> are captured accurately models or, impor-          ther highlights utility using genome scale models a <br /> tantly, point areas metabolism regulation        context content interpreting analyzing complex high- <br /> require experimental interrogation. For example,         throughput data sets. This powerful synergistic approach not <br /> number independent gene deletion studies shown        using models data analysis tools, using high- <br /> some genes involved arginine biosynthesis essential     throughput data feedback model improvement, be- <br /> (18, 24), enzymes, current literature        coming paradigm continue drive systems biology <br /> cannot explain <span id='am-2' about='Thesaurus:Essential_Amino_Acid' typeof='owl:Thing'>essential amino acid</span> synthesized.       research forward. <br /> Therefore, experiments need conducted either <br />                                                                                              ACKNOWLEDGMENTS <br /> identify novel arginine biosynthetic genes determine which <br /> multifunctional enzymes compensate perturbation           We thank Adam Feist critical reading manuscript; <br /> of genes.                                                      Trina Patel, Vasiliy Portnoy, Eric Knight technical assistance; <br />                                                                    Christian Barrett members Palsson laboratory for <br />    Additionally, based experimental results,        insightful discussions suggestions. <br /> model improvements suggested. Since number exper-            We gratefully acknowledge support NIH Protein Struc- <br /> imentally essential genes involved cofactor biosynthesis,   ture Initiative, grant numbers P50 GM62411 U54 GM074898, and <br /> a number cofactors included biomass ob-        grant no. NIH R01 GM5708. <br />                                                                       Bernhard Palsson financial Genomatica, Inc. Al- <br /> jective function used conduct growth prediction simula- <br />                                                                    NIH R01 GM5708 grant identified conflict of <br /> tion. These cofactors include pyridoxal-5-phosphate, isopre-       management based overall scope project its <br /> noids, hemes, ACP, ubiquinone. These help correct         potential benefit Genomatica, Inc., research findings included in <br /> for false negatives (lethal phenotypes nonlethal model    publication necessarily directly relate interests of <br /> predictions) account large number discrepancies      Genomatica, Inc. <br /> in minimal- rich-medium phenotypes (data shown                                         REFERENCES <br /> for rich medium). A wild-type biomass composition does          1. Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura, M. Baba, K. A. <br />                                                                        Datsenko, M. Tomita, B. L. Wanner, H. Mori. 2006. Construction of <br /> always correlate essential biomass composition;            Escherichia coli K-12 in-frame, single-gene knockout mutants: Keio col- <br /> example, core complete LPS required            lection. Mol. Syst. Biol. [Online.] doi:10.1038/msb4100050. <br /> cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>survival</span> (37). Accordingly, essentiality      2. Bachler, C., P. Schneider, P. Bahler, A. Lustig, B. Erni. 2005. Esche- <br />                                                                        richia coli dihydroxyacetone kinase controls gene expression binding to <br /> other biomass components refined relaxed based            transcription factor DhaR. EMBO J. 24:283&#226;&#8364;&#8220;293. <br /> the nonessentiality corresponding biosynthetic-pathway       3. Boutros, M., A. A. Kiger, S. Armknecht, K. Kerr, M. Hild, B. Koch, S. A. <br /> genes. These issues addressed forthcoming up-           Haas, R. Paro, N. Perrimon, Heidelberg Fly Array Consortium. <br />                                                                        2004. Genome-wide RNAi analysis growth viability Drosophila <br /> dated metabolic reconstruction E. coli (A. Feist B. O.          cells. Science 303:832&#226;&#8364;&#8220;835. <br /> Palsson, personal communication) represent significant        4. Cayley, S., M. T. Record, Jr., B. A. Lewis. 1989. Accumulation of <br />                                                                        3-(N-morpholino)propanesulfonate osmotically stressed Escherichia coli <br /> advance.                                                               K-12. J. Bacteriol. 171:3597&#226;&#8364;&#8220;3602. <br />    Model improvements suggested regard             5. Chen, K., L. Pachter. 2005. Bioinformatics whole-genome shotgun <br /> the steps glycerol metabolism (Fig. 4). As previously         sequencing microbial communities. PLOS Comput. Biol. 1:106&#226;&#8364;&#8220;112. <br />                                                                     6. Covert, M. W., E. M. Knight, J. L. Reed, M. J. Herrgard, B. O. Palsson. <br /> noted, analysis false positives suggests glycerol          2004. Integrating high-throughput computational data elucidates bacte- <br /> import occur passive transport cell mem-             rial networks. Nature 429:92&#226;&#8364;&#8220;96. <br /> brane absence glpF-encoded transporter. Ad-           7. Covert, M. W., B. O. Palsson. 2002. Transcriptional regulation con- <br />                                                                        straints-based metabolic models Escherichia coli. J. Biol. Chem. 277: <br /> ditionally, initial enzymatic steps required convert            28058&#226;&#8364;&#8220;28064. <br /> glycerol dihydroxyacetone phosphate appear exclu-          8. Covert, M. W., C. H. Schilling, B. Palsson. 2001. Regulation gene <br />                                                                        expression flux balance models metabolism. J. Theor. Biol. 213:73&#226;&#8364;&#8220;88. <br /> sively mediated GlpK GlpD GldA            9. Datsenko, K. A., B. L. Wanner. 2000. One-step inactivation chromo- <br /> the DhaKLM-PtsHI complex. This pathway bias likely              somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> to transcriptional regulatory effects. Indeed, elevated            Sci. USA 97:6640&#226;&#8364;&#8220;6645. <br />                                                                    10. Earhardt, C. F. 1996. Uptake metabolism iron molybdenum, p. <br /> expression glpK glpD growth glycerol re-              1075&#226;&#8364;&#8220;1090. In F. C. Neidhardt (ed.), Escherichia coli Salmonella: cellular <br /> vealed quantitative RT-PCR (Fig. 4) supports            molecular biology, vol. 1. ASM Press, Washington, D. C. <br />  8270       JOYCE ET AL.                                                                                                                                    J. BACTERIOL. <br />  <br /> 11. Edwards, J. S., R. U. Ibarra, B. O. Palsson. 2001. In silico predictions         Sato, H. H. Saxild, E. Scanlan, W. Schumann, J. F. Seegers, J. Sekiguchi, A. <br />     Escherichia coli metabolic capabilities consistent experimental data.          Sekowska, S. J. Seror, M. Simon, P. Stragier, R. Studer, H. Takamatsu, <br />     Nat. Biotechnol. 19:125&#226;&#8364;&#8220;130.                                                            T. Tanaka, M. Takeuchi, H. B. Thomaides, V. Vagner, J. M. van Dijl, <br /> 12. Fraenkel, D. G. 1996. Glycolysis, p. 189&#226;&#8364;&#8220;198. In F. C. Neidhardt (ed.),                 K. Watabe, A. Wipat, H. Yamamoto, M. Yamamoto, Y. Yamamoto, K. <br />     Escherichia coli Salmonella, vol. 1. ASM Press, Washington, D.C.                    Yamane, K. Yata, K. Yoshida, H. Yoshikawa, U. Zuber, N. Ogasawara. <br /> 13. Fu, D., A. Libson, L. J. Miercke, C. Weitzman, P. Nollert, J. Krucinski,            2003. Essential Bacillus subtilis genes. Proc. Natl. Acad. Sci. USA 100:4678&#226;&#8364;&#8220; <br />     R. M. Stroud. 2000. Structure glycerol-conducting channel basis            4683. <br />     selectivity. Science 290:481&#226;&#8364;&#8220;486.                                         28.   Koonin, E. V. 2003. Comparative genomics, minimal gene-sets last <br /> 14. Gay, N. J. 1984. Construction characterization Escherichia coli               universal common ancestor. Nat. Rev. Microbiol. 1:127&#226;&#8364;&#8220;136. <br />     strain uncI <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. J. Bacteriol. 158:820&#226;&#8364;&#8220;825.                           29.   Levy, S., G. Q. Zeng, A. Danchin. 1990. Cyclic AMP synthesis Esch- <br /> 15. Gennis, R. B., V. Stewart. 1996. Respiration, p. 217&#226;&#8364;&#8220;261. In F. C.                  erichia coli strains bearing known deletions pts phosphotransferase <br />     Neidhardt (ed.), Escherichia coli Salmonella, vol. 1. ASM Press, Wash-              operon. Gene 86:27&#226;&#8364;&#8220;33. <br />     ington, D.C.                                                                      30.   Maeda, I., Y. Kohara, M. Yamamoto, A. Sugimoto. 2001. Large-scale <br /> 16. Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz, M. D.              analysis gene function Caenorhabditis elegans high-throughput <br />     Daugherty, A. L. Somera, N. C. Kyrpides, I. Anderson, M. S. Gelfand, A.                 RNAi. Curr. Biol. 11:171&#226;&#8364;&#8220;176. <br />     Bhattacharya, V. Kapatral, M. D&#226;&#8364;&#8482;Souza, M. V. Baev, Y. Grechkin, F. Mseeh,         31.   Mengin-Lecreulx, D., J. van Heijenoort. 1996. Characterization the <br />     M. Y. Fonstein, R. Overbeek, A. L. Barabasi, Z. N. Oltvai, A. L. Oster-             essential gene glmM encoding phosphoglucosamine mutase Escherichia <br />     man. 2003. Experimental determination level analysis essential            coli. J. Biol. Chem. 271:32&#226;&#8364;&#8220;39. <br />     genes Escherichia coli MG1655. J. Bacteriol. 185:5673&#226;&#8364;&#8220;5684.                    32.   Merino, E., C. Yanofsky. 2005. Transcription attenuation: highly <br /> 17. Giaever, G., A. M. Chu, L. Ni, C. Connelly, L. Riles, S. Veronneau, S. Dow,             conserved regulatory strategy used bacteria. Trends Genet. 21:260&#226;&#8364;&#8220; <br />     A. Lucau-Danila, K. Anderson, B. Andre, A. P. Arkin, A. Astromoff, M.                   264. <br />     El-Bakkoury, R. Bangham, R. Benito, S. Brachat, S. Campanaro, M. Cur- <br />                                                                                       33.   Neuwald, A. F., B. R. Krishnan, I. Brikun, S. Kulakauskas, K. Suziedelis, T. <br />     tiss, K. Davis, A. Deutschbauer, K. D. Entian, P. Flaherty, F. Foury, D. J. <br />                                                                                             Tomcsanyi, T. S. Leyh, D. E. Berg. 1992. cysQ, gene needed for <br />     Garfinkel, M. Gerstein, D. Gotte, U. Guldener, J. H. Hegemann, S. Hempel, <br />                                                                                             cysteine synthesis Escherichia coli K-12 aerobic growth. J. <br />     Z. Herman, D. F. Jaramillo, D. E. Kelly, S. L. Kelly, P. Kotter, D. LaBonte, <br />                                                                                             Bacteriol. 174:415&#226;&#8364;&#8220;425. <br />     D. C. Lamb, N. Lan, H. Liang, H. Liao, L. Liu, C. Luo, M. Lussier, R. Mao, <br />     P. Menard, S. L. Ooi, J. L. Revuelta, C. J. Roberts, M. Rose, P. Ross-            34.   Overbeek, R., T. Begley, R. M. Butler, J. V. Choudhuri, H. Y. Chuang, M. <br />     Macdonald, B. Scherens, G. Schimmack, B. Shafer, D. D. Shoemaker, S.                    Cohoon, V. Crecy-Lagard, N. Diaz, T. Disz, R. Edwards, M. Fonstein, <br />     Sookhai-Mahadeo, R. K. Storms, J. N. Strathern, G. Valle, M. Voet, G.                   E. D. Frank, S. Gerdes, E. M. Glass, A. Goesmann, A. Hanson, D. Iwata- <br />     Volckaert, C. Y. Wang, T. R. Ward, J. Wilhelmy, E. A. Winzeler, Y. Yang, G.             Reuyl, R. Jensen, N. Jamshidi, L. Krause, M. Kubal, N. Larsen, B. Linke, <br />     Yen, E. Youngman, K. Yu, H. Bussey, J. D. Boeke, M. Snyder, P. Philippsen,              A. C. McHardy, F. Meyer, H. Neuweger, G. Olsen, R. Olson, A. Osterman, V. <br />     R. W. Davis, M. Johnston. 2002. Functional profiling Saccharo-               Portnoy, G. D. Pusch, D. A. Rodionov, C. Ruckert, J. Steiner, R. Stevens, <br />     myces cerevisiae genome. Nature 418:387&#226;&#8364;&#8220;391.                                            I. Thiele, O. Vassieva, Y. Ye, O. Zagnitko, V. Vonstein. 2005. The <br /> 18. Glasner, J. D., P. Liss, G. Plunkett III, A. Darling, T. Prasad, M. Rusch, A.           subsystems approach genome annotation use project to <br />     Byrnes, M. Gilson, B. Biehl, F. R. Blattner, N. T. Perna. 2003. ASAP,             annotate 1000 genomes. Nucleic Acids Res. 33:5691&#226;&#8364;&#8220;5702. <br />     systematic annotation package community analysis genomes. Nucleic          35.   Postma, P. W., J. W. Lengeler, G. R. Jacobson. 1996. Phosphoenolpyru- <br />     Acids Res. 31:147&#226;&#8364;&#8220;151.                                                                  vate:carbohydrate phosphotransferase systems, p. 1149&#226;&#8364;&#8220;1174. In F. C. Neid- <br /> 19. Glass, J. I., N. Assad-Garcia, N. Alperovich, S. Yooseph, M. R. Lewis, M.               hardt (ed.), Escherichia coli Salmonella, vol. 1. ASM Press, Washington, <br />     Maruf, C. A. Hutchison III, H. O. Smith, J. C. Venter. 2006. Essential              D.C. <br />     genes minimal bacterium. Proc. Natl. Acad. Sci. USA 103:425&#226;&#8364;&#8220;430.             36.   Price, N. D., J. L. Reed, B. O. Palsson. 2004. Genome-scale models of <br /> 20. Hare, R. S., S. S. Walker, T. E. Dorman, J. R. Greene, L. M. Guzman, T. J.              microbial cells: evaluating consequences constraints. Nat. Rev. Mi- <br />     Kenney, M. C. Sulavik, K. Baradaran, C.  <br /> </body></html>