<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, July 2007, p. 1108&#226;&#8364;&#8220;1118                                                                                         Vol. 6, No. 7 <br /> 1535-9778/07/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.00107-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> Phosphatidylinositol 4-Kinase III-&#226;?&#164; Is Required Golgi Maintenance <br />                Cytokinesis Trypanosoma brucei&#228;&#338;&#164; <br />                          Melissa J. Rodgers, Joseph P. Albanesi, Margaret A. Phillips* <br />              Department Pharmacology, University Texas Southwestern Medical Center, Dallas, Texas 75390-9041 <br />                                               Received 4 April 2007/Accepted 28 April 2007 <br />  <br />              The parasitic protozoan Trypanosoma brucei contains type III phosphatidylinositol 4-kinases (&#226;?&#163; &#226;?&#164;). <br />           We cloned gene encoding T. brucei type III phosphatidylinositol 4-kinase &#226;?&#164; (TbPI4KIII-&#226;?&#164;), <br />           expressed protein COS-7 cells, confirmed protein catalyzes phosphorylation of <br />           phosphatidylinositol. Depletion TbPI4KIII-&#226;?&#164; procyclic T. brucei <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) resulted in <br />           inhibition cell growth distorted cellular morphology. RNAi cells distorted Golgi apparatus, and <br />           lysosomal flagellar pocket proteins mislocalized. Ultrastructural analysis revealed internal <br />           accumulation heterogeneous population vesicles, abnormal positioning organelles, loss cell <br />           polarity. Scanning electron microcopy revealed twisted <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span>, dividing cells exhibited a <br />           detached daughter flagellum lacked cleavage furrow. Cell cycle analysis confirmed cells depleted of <br />           TbPI4KIII-&#226;?&#164; postmitotic cytokinesis block occurs single round mitosis, suggestive a <br />           specific cell cycle block. In summary, TbPI4KIII-&#226;?&#164; essential protein procyclic T. brucei, required for <br />           maintenance Golgi structure, protein trafficking, normal cellular shape, cytokinesis. <br />  <br />  <br />    Phosphoinositides (PIs), minor constituents eu-               PI4KIII-&#226;?&#163; serve generate pools PI4P required for <br /> karyotic cell membranes, play crucial role cellular regula-             phospholipase C-dependent signaling (5). <br /> tion. PIs phosphorylated derivatives phosphatidylino-                    Pik1, yeast ortholog PI4KIII-&#226;?&#164;, localizes nu- <br /> sitol (PtdIns), differentially phosphorylated            cleus Golgi apparatus, involved the <br /> 3&#226;&#172;&#732;, 4&#226;&#172;&#732;, 5&#226;&#172;&#732; position inositol ring, leading seven                regulation secretion (3, 23, 44). The mammalian enzyme <br /> possible PIs (16). Their synthesis, mediated specific kinases             localizes Golgi apparatus (20, 48) nucleus <br /> and/or phosphatases, partitioned distinct subcellular com-             (12) regulates trafficking Golgi apparatus the <br /> partments. The PIs serve recruit effector proteins                plasma membrane (7, 19). PI4KIII-&#226;?&#164;-dependent synthesis of <br /> interacting lipid binding domains effectors (37).              PI4P Golgi apparatus important ceramide <br />    PtdIns 4-kinases phosphorylate PtdIns 4&#226;&#172;&#732; position               transport endoplasmic reticulum (42), me- <br /> yield PtdIns 4-monophosphate (PI4P). Subsequent phosphor-                    diated PI4P-binding protein CERT (30). More recently, <br /> ylation 3&#226;&#172;&#732; and/or 5&#226;&#172;&#732; position results synthesis PI          PI4KIII-&#226;?&#164; implicated priming neurosecre- <br /> 3,4-P2, PI 4,5-P2, PI 3,4,5-P3, participate wide              tory vesicles regulated exocytosis (11). <br /> variety cell signaling pathways. Recently shown                  Trypanosoma brucei, extracellular protozoan parasite, is <br /> that PI4P essential role Golgi budding, hence,             causative agent African sleeping sickness humans. <br /> role extends serving precursor (45). The D4-                     The parasite exists distinct life forms, mammalian <br /> phosphorylated PIs essential lipid mediators regu-               bloodstream form insect procyclic form. T. brucei has <br /> lation multiple cellular functions, including endocytosis, ve-            highly polarized exocytotic endocytotic system, is <br /> sicular trafficking, actin cytoskeletal organization, Ca2&#226;&#171;&#185;               located nucleus specialized organelle called <br /> signaling (41).                                                              flagellar pocket. This invagination plasma membrane <br />    Eukaryotes express classes PtdIns 4-kinase, termed                 near posterior cell site exocytosis and <br /> type II type III. These classes distinguished                 endocytosis (21). Both forms parasite express stage- <br /> their sequences, biochemical properties, enzymatic parame-                   specific glycosylphosphatidylinositol-anchored surface protein: <br /> ters, inhibitor profiles (4, 18). Most eukaryotes express             variable surface glycoprotein (VSG) bloodstream form <br /> least type II enzyme type III enzymes                       procyclin procyclic form (10, 40). Surface coat pro- <br /> (PI4KIII-&#226;?&#163; PI4KIII-&#226;?&#164;). Although specific functions                teins comprise bulk protein cargo moving the <br /> each isoform poorly understood, appears                secretory (15). Bloodstream form parasites a <br /> type II kinases PI4KIII-&#226;?&#164; important membrane <br />                                                                              high rate VSG endocytosis recycling flagellar <br /> trafficking secretory/endocytotic pathways (4), whereas <br />                                                                              pocket. This process important immune evasion, <br />                                                                              allowing degradation bound host immune complexes. <br />                                                                              Endocytosis flagellar pocket important forms <br />                                                                              uptake nutrients host-parasite interactions. Traf- <br />   * Corresponding author. Mailing address: Department Pharma-             ficking, endocytosis, Golgi maintenance PI-medi- <br /> cology, University Texas Southwestern Medical Center, Dallas, TX <br /> 75390-9041. Phone: (214) 645-6164. Fax: (214) 645-6166. E-mail: <br />                                                                              ated events eukaryotes (4). Many components <br /> margaret.phillips@UTSouthwestern.edu.                                        exocytotic endocytotic systems T. brucei have <br />   &#228;&#338;&#164; <br />     Published ahead print 4 May 2007.                                  identified, including clathrin, adapter proteins, Rab <br />  <br />                                                                       1108 <br />  VOL. 6, 2007                                                                      T. BRUCEI PHOSPHATIDYLINOSITOL 4-KINASE III-&#226;?&#164;                               1109 <br />  <br />  <br /> GTPases (15), known effector proteins                        noglobulin G (IgG) horseradish peroxidase conjugate secondary (Amersham), <br /> D4-phosphorylated PIs eukaryotes. These observa-                         1:5,000. <br />                                                                                      Immune complex kinase assays. PtdIns 4-kinase activity immunoprecipi- <br /> tions suggest synthesis D4-phosphorylated PIs                     tates measured phosphorylation PtdIns&#226;&#8364;&#8220;Triton X-100 micelles using <br /> play key regulatory role secretory pathway T. brucei.                 [&#226;?&#165;-32P]ATP (10 mCi/mmol) phosphate donor. PtdIns&#226;&#8364;&#8220;Triton X-100 mi- <br />    The T. brucei database contains sequences pu-                      celles prepared sonicating PtdIns (Avanti) buffer C (50 mM Tris-HCl <br /> tative PtdIns 4-kinases, TbPI4KIII-&#226;?&#163; TbPI4KIII-&#226;?&#164;. Unlike                      [pH 7.5], 1.0 mM EGTA, 0.4% Triton X-100, 0.5 mg/ml bovine serum <br />                                                                                   albumin [BSA]). Reaction mixtures contained 10 &#226;?&#174;l bead slurry 38 &#226;?&#174;l <br /> other eukaryotes, appears T. brucei lacks type II PtdIns <br />                                                                                   PtdIns&#226;&#8364;&#8220;Triton X-100 micelles initiated addition 2 &#226;?&#174;l ATP <br /> 4-kinase. A class III PtdIns 3-kinase recently identified                  mix (50 mM Tris-HCl [pH 7.5], 375 mM MgCl2, 5 mM ATP, [&#226;?&#165;-32P]ATP [10 <br /> T. brucei, depletion RNA interference (RNAi) im-                       mCi/mmol]). The reaction mixtures contained 200 &#226;?&#174;M PtdIns, 0.3% Triton <br /> plicated Golgi segregation endocytotic trafficking                      X-100, 400 &#226;?&#174;M ATP, 15 mM MgCl, reactions carried room <br />                                                                                   temperature 30 min. Reactions stopped addition 3.75 volumes of <br /> (22). Like yeast, T. brucei does class I II PtdIns <br />                                                                                   chloroform-methanol-HCl (100:200:1), followed extraction 1.25 volumes <br /> 3-kinases, suggesting T. brucei does PtdIns 3-                      chloroform 0.1 N HCl. The lipids separated thin-layer chroma- <br /> kinase-dependent signaling pathways. Four putative PtdIns                         tography (TLC) Whatman Partisil LK5D TLC plates using n-propyl alco- <br /> monophosphate (PIP) kinases identified T.                        hol&#226;&#8364;&#8220;H2O&#226;&#8364;&#8220;NH4OH (65:20:15) solvent system, radioactive spots de- <br /> brucei genome: type I PI4P 5-kinase, type III PI3P 5-kinase                   tected autoradiography, scraped, quantified scintillation counting. <br />                                                                                      RNAi plasmid construction. Primers designed amplify coding <br /> (PIKFYVE), type II enzymes,                        region positions 421 1002 DNA sequence. Primers 5&#226;&#172;&#732;-GGA <br /> similar type II PIP kinase-&#226;?&#163; isoforms. The presence                     ACTGCGACTTAAGCTTTTCAACGATGAGAG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GCTAGCTCCT <br /> these enzymes indicates T. brucei can, principle, syn-                    CTTGCCGCAAAT-3&#226;&#172;&#732; contain restriction sites HindIII NheI, respectively. <br /> thesize mono- bisphosphorylated PIs.                               This fragment subcloned HindIII NheI sites pJM326 (a <br />                                                                                   generous gift Paul Englund, Johns Hopkins University). The complementary <br />    In present study, cloned functionally char-                    strand amplified using primers 5&#226;&#172;&#732;-GGAACTGCGACTTACGCGTTTCAA <br /> acterized T. brucei PI4KIII-&#226;?&#164;. Using RNAi,                       CGATGAGAG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-TCTAGATCCTCTTGCCGCAAAT-3&#226;&#172;&#732;, con- <br /> synthesis PI4P TbPI4KIII-&#226;?&#164; essential normal                          tain restriction sites MluI Xba, respectively. This fragment subcloned <br /> growth morphology procyclic T. brucei. Further analysis                    MluI XbaI sites pLew100 (a generous gift Paul Englund, Johns <br />                                                                                   Hopkins University). The pJM326 construct containing insert digested <br /> revealed TbPI4KIII-&#226;?&#164; required maintenance of <br />                                                                                   HindIII XbaI, releasing larger fragment containing target gene <br /> Golgi structure, protein trafficking, normal cellular shape,                  fragment fused fragment form loop stem-loop tran- <br /> cytokinesis.                                                                      script. This fragment subcloned HindIII XbaI sites the <br />                                                                                   pLew100 construct containing complement target gene. <br />                                                                                      Trypanosome cell culture transfection. Procyclic culture form 29-13 cells <br />                        MATERIALS AND METHODS                                      (a generous gift George Cross, Rockefeller University), stably express <br />                                                                                   T7 polymerase tetracycline repressor, used creating RNAi cell <br />    Identification cloning Tb PI4KIII-&#226;?&#164;. TbPI4KIII-&#226;?&#164; TbPI4KIII-&#226;?&#163;        lines (47). 29-13 parental line cultures grown SDM-79 supplemented with <br /> were identified BLAST searching TIGR T. brucei database     10% FBS presence 25 &#226;?&#174;g/ml G418 25 &#226;?&#174;g/ml hygromycin. 29-13 cells <br /> the human PI4KIII-&#226;?&#164; protein sequence (Q9UBF8) human PI4KIII-&#226;?&#163;              transfected EcoRV-linearized RNAi plasmid, using Bio-Rad Gene <br /> protein sequence (P42356). An open reading frame TbPI4KIII-&#226;?&#164; (accession       Pulser II 1.5 kV 25 &#226;?&#174;F 0.4-cm electrocuvettes (Bio-Rad). After elec- <br /> number XP 844264) bacterial artificial chromosome (BAC) clone        troporation, transfectants selected culturing cells presence of <br /> RPCI93-5E12 chromosome 4, TbPI4KIII-&#226;?&#163; (accession number XP               2.5 &#226;?&#174;g/ml phleomycin addition G418 hygromycin. Synthesis RNA <br /> 843994) BAC clone RPCI93-28C22 chromosome 3. The                induced addition tetracycline 1 &#226;?&#174;g/ml (46). <br /> full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> gene TbPI4KIII-&#226;?&#164; amplified PCR 427 genomic DNA           RNA isolation Northern blot analysis. Total RNA extracted <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> and subcloned mammalian expression vector pCMV-tag2b (Stratagene)        log-phase cultures using RNAqueous kit (Ambion). Total RNA samples <br /> using primers 5&#226;&#172;&#732;-GTTTCCTTTTTCCGGATCCATGTCGAATGCTTTGT                          collected uninduced culture tetracycline-induced cultures <br /> TTTG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CATTCACCACATCCCCTCGAGCTAGAGTATACCATT-3&#226;&#172;&#732;.                           24 h, 6 days. Two micrograms total RNA separated 1% <br /> A fragment TbPI4KIII-&#226;?&#163; gene (nucleotides 5097 6936) likewise        agarose&#226;&#8364;&#8220;formaldehyde gel transferred BrightStar-Plus nylon membranes <br /> amplified PCR 427 genomic DNA subcloned Zero Blunt           (Ambion). Hybridization performed overnight 48&#194;&#176;C ULTRAhyb hy- <br /> TOPO PCR cloning vector (Invitrogen) using primers 5&#226;&#172;&#732;-GCCGGGAACG              bridization buffer (Ambion). 32P-labeled probes TbPI4KIII-&#226;?&#164;, TbPI4KIII-&#226;?&#163;, <br /> GCTAATGAGCCTCATCC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CCCTTCACTCACCGGGTACCCC-3&#226;&#172;&#732;.                            &#226;?&#163;-tubulin synthesized using StripEZ-PCR probe synthesis kit (Am- <br />    Transfections, immunoprecipitations, Western blot analysis. COS-7 cells    bion). Blots stripped reprobed according manufacturer&#226;&#8364;&#8482;s proto- <br /> were grown Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s medium supplemented 10%            cols. The DNA probes designed hybridize following regions of <br /> fetal bovine serum (FBS). Purified plasmid, vector pCMV-          transcripts; TbPI4KIII-&#226;?&#164;, 1 675; TbPI4KIII-&#226;?&#163;, 5097 5770, &#226;?&#163;-tubulin, <br /> TbPI4KIII-&#226;?&#164;, transfected COS-7 cells using Lipofectamine 2000 (In-       1 650. <br /> vitrogen) according manufacturer&#226;&#8364;&#8482;s protocols. Cells incubated         Growth curves. For growth curves, cell counts performed using hemo- <br /> 37&#194;&#176;C 5% CO2 18 h. Transfected COS-7 cells washed phosphate-      cytometer. Cells independent experiments counted, the <br /> buffered saline (PBS), lysed ice buffer A (20 mM Tris-HCl [pH 7.5], 25 mM   averages reported. <br /> sucrose, 100 mM NaCl, 1.0 mM EDTA, 1 mM NaVO3, 1 mM dithiothreitol, 0.2              Inositol labeling. Mid-log-phase trypanosomes uninduced cultures and <br /> mM phenylmethylsulfonyl fluoride, 10 &#226;?&#174;g/ml leupeptin, 20 &#226;?&#174;g/ml antipain, 100      cultures induced 3 5 days centrifuged 400 &#226;&#171;&#187; g washed <br /> &#226;?&#174;g/ml benzamidine, 10 &#226;?&#174;g/ml pepstatin, 10 &#226;?&#174;g/ml chymostatin), soni-       PBS. Trypanosomes resuspended 5 ml fresh SDM-79 supple- <br /> cated. After centrifugation (70,000 rpm 15 min 4&#194;&#176;C), Triton X-100      mented 10% FBS 2 &#226;&#171;&#187; 107 cells/ml, 50 &#226;?&#174;l 10 &#226;?&#174;Ci/&#226;?&#174;l [3H]myoinositol <br /> added supernatant final concentration 0.1%. Cleared lysate (300   (Amersham) added mixture incubated 24 h 25&#194;&#176;C. Cells were <br /> &#226;?&#174;l) incubated overnight 75 &#226;?&#174;l anti-Flag M2 resin (Sigma). Beads     centrifuged 400 &#226;&#171;&#187; g washed PBS. Lipids extracted 375 &#226;?&#174;l <br /> then washed times buffer A plus 0.1% Triton X-100 resus-      methanol-HCl (1:1) 190 &#226;?&#174;l chloroform. The lipids separated by <br /> pended 75 &#226;?&#174;l buffer A plus 0.1% Triton X-100, yielding 50% bead slurry.      TLC Whatman Partisil LK5D TLC plates using n-propyl alcohol&#226;&#8364;&#8220;H2O&#226;&#8364;&#8220; <br />    For Western analysis, 10 &#226;?&#174;l bead slurry separated 10% sodium       NH4OH (65:20:15) solvent system, radioactive spots detected au- <br /> dodecyl sulfate gel transferred Hybond-P polyvinylidene difluoride mem-    toradiography, scraped, quantified scintillation counting. The PIP/PtdIns <br /> branes (Amersham). Membranes blocked 5% milk (wt/vol) TBS-T (20        ratio calculated independent experiments time point, and <br /> mM Tris-HCl, 137 mM NaCl, 0.1% [vol/vol] Tween 20, pH 7.6) 1 h room        results averaged. <br /> temperature. Primary secondary antibodies diluted blocking buffer        Immunofluorescence. Cells immunofluorescence fixed 3.7% <br /> and incubated 1 h room temperature. Antibody dilutions        formaldehyde (EMS) PBS permeabilized 0.1% Triton X-100&#226;&#8364;&#8220;PBS. <br /> follows: anti-Flag M2 antibody (Sigma), 1:1,000, sheep anti-mouse immu-       Cells blocked 1 h 10% normal goat serum&#226;&#8364;&#8220;PBS. Incubations with <br />  1110       RODGERS ET AL.                                                                                                                      EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 1. Domain structure TbPI4KIII-&#226;?&#164; sequence alignments known type III-&#226;?&#164; kinases. (A) TbPI4KIII-&#226;?&#164;, like PI4KIII-&#226;?&#164;s, has <br /> an N-terminal LKU domain C-terminal catalytic domain. (B) Sequence alignment LKU domain TbPI4KIII-&#226;?&#164; Homo <br /> sapiens (Q9UBF8), A. thaliana (NP_201212), S. cerevisiae (NP_014132). Alignments prepared using MUSCLE alignment program <br /> (http://www.bioinformatics.nl/tools/muscle.html). (C) Alignment catalytic domain according MUSCLE program. <br />  <br />  <br />  <br /> primary antibody(ies) secondary antibody(ies) performed 10% nor-      6.4% paraformaldehyde 0.1 M sodium phosphate buffer (pH 7.4) was <br /> mal goat serum&#226;&#8364;&#8220;PBS 1 h room temperature. Primary antibody dilutions        added equal volume cell suspension. After 10 min, small aliquots were <br /> were follows: mouse anti-p67, 1:500 (1), rat-anti YL1/2, 1:500 (Chemicon)      removed scanning electron microscopy. The remaining cell suspension was <br /> (39), rabbit anti-CRAM, 1:300 (28). <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> visualize Golgi apparatus, cells   centrifuged supernatant replaced single-strength fixative. Cells were <br /> were fixed permeabilized ice-cold 100% methanol. Cells blocked      stored overnight fixative. <br /> for 1 h 3% BSA&#226;&#8364;&#8220;PBS, incubations primary antibodies second-         Transmission electron microscopy. Fixed, pelleted cells rinsed sodium <br /> ary antibodies performed 3% BSA&#226;&#8364;&#8220;PBS 1 h room temperature.          phosphate buffer, embedded 2% agarose, postfixed 1% osmium tetroxide, <br /> Primary antibody dilutions follows: rabbit anti-TbGrasp antibody, 1:500   dehydrated graded ethanol series, embedded EMbed-812 resin. Thin <br /> (24), mouse anti-p67, 1:500 (1). Secondary antibody dilutions fol- <br />                                                                                   sections cut Leica EM UC6 microtome, poststained uranyl <br /> lows: Dylight560-labeled goat anti-rabbit IgG (Pierce), 1:200; AlexaFluor488- <br />                                                                                   acetate lead citrate, viewed JEOL JEM-1200EX transmission <br /> labeled goat anti-rat IgG (Molecular Probes), 1:200; AlexaFluor488-labeled <br />                                                                                   electron microscope operating 80 kV. <br /> goat anti-mouse IgG (Molecular Probes), 1:200. All slides mounted in <br />                                                                                      Scanning electron microscopy. Small drops fixed cell suspension in- <br /> Vectashield mounting medium containing DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phenylin- <br />                                                                                   cubated poly-L-lysine-coated coverslips humidity chamber 30 minutes. <br /> dole) (Vector Labs). All fluorescence microscopy performed Zeiss <br /> Axiovert 200 M inverted microscope equipped Zeiss 100&#226;&#171;&#187; 1.30-numerical-     The coverslips rinsed, adhered cells postfixed 1% osmium <br /> aperture oil immersion objective charge-coupled device camera. Digital      tetroxide, dehydrated graded ethanol series, transferred several <br /> images captured analyzed using Slidebook 4.0 software (Intelligent       changes hexamethyldisilazane, allowed air dry overnight. Cells on <br /> Imaging Innovations). For preparation final figures, signal enhance-   coverslips sputter coated gold viewed JEOL JSM-840A <br /> ment background reduction performed images using Adobe            scanning electron microscope 10 kV. <br /> Photoshop (5.5).                                                                     Analysis DNA content. Log-phase cells collected 6-day period, <br />   Electron microscopy. Cell fixation followed procedure Allen et al. (2)   2 &#226;&#171;&#187; 107 cells washed PBS, resuspended 1.0 ml PBS, and <br /> with modifications. Double-strength fixative containing 5% glutaraldehyde    adhered poly-L-lysine-coated slides. Cells fixed 5 min ice-cold <br />  VOL. 6, 2007                                                                            T. BRUCEI PHOSPHATIDYLINOSITOL 4-KINASE III-&#226;?&#164;                       1111 <br />  <br />  <br />  <br />  <br />    FIG. 2. Expression enzymatic characterization TbPI4KIII-&#226;?&#164;. <br /> (A) Immunoblot Flag-tagged TbPI4KIII-&#226;?&#164; expressed COS-7 cells <br /> and immunoprecipitated anti-Flag M2 resin. Anti-Flag antibody <br /> detects band migrating 50 75 kDa immunopre- <br /> cipitates cells expressing TbPI4KIII-&#226;?&#164; (lane 2) those <br /> from control cells expressing vector (lane 1). (B) Kinase assay of <br /> the immune complex Flag-tagged recombinant TbPI4KIII-&#226;?&#164;. Im- <br /> munoprecipitated proteins tested PtdIns kinase activity the <br /> presence absence 100 &#226;?&#174;M wortmannin (WT), inhibits <br /> mammalian PI4KIII-&#226;?&#164; activity. Reaction mixtures contained 200 &#226;?&#174;M <br /> PtdIns, 0.3% Triton X-100, 400 &#226;?&#174;M ATP, 15 mM MgCl2, and <br /> reactions carried room temperature 30 min. Radioac- <br /> tive PI4P detected TLC immunoprecipitates expressing <br /> TbPI4KIII-&#226;?&#164; prevented forming WT. Triplicate mea- <br /> surements shown. <br />  <br />  <br />  <br /> 100% methanol washed PBS. Slides mounted Vectashield <br /> mounting medium containing DAPI (Vector Labs). <br />    Individual cells examined fluorescence microscopy order deter- <br /> mine percentages cells different phases cell cycle. A population <br /> of 100 cells time point (days 0, 4, 5, 6) counted, different <br /> cell cycle phenotypes scored. The phenotypes correspond kineto- <br /> plast nuclear contents. The phenotype, corresponds G1, has <br /> one kinetoplast nucleus. The second phenotype kinetoplasts and <br /> one nucleus. The phenotype, kinetoplasts nuclei, <br /> corresponds cells G2/M. The fourth phenotype corresponded other <br />                                                                                            FIG. 3. Characterization TbPI4KIII-&#226;?&#164; RNAi cell line. <br /> combination kinetoplast nuclear contents. The counts inde- <br />                                                                                         (A) Northern blot analysis TbPI4KIII-&#226;?&#164; RNAi cells. Transcript <br /> pendent experiments averaged, standard deviation calculated. <br />                                                                                         levels samples collected 6 days tetracycline (TET) induc- <br />                                                                                         tion determined. Two micrograms total RNA separated on <br />                                                                                         1% agarose&#226;&#8364;&#8220;formaldehyde gel transferred BrightStar-Plus <br />                                    RESULTS                                              nylon membranes. The blot probed TbPI4KIII-&#226;?&#164; and <br />                                                                                         &#226;?&#163;-tubulin. After 24 h induction, transcripts TbPI4KIII-&#226;?&#164; were <br />    Identification characterization T. brucei PI4KIII-&#226;?&#164;.                          undetectable, remained day 6. (B) Effect of <br /> The gene T. brucei PI4KIII-&#226;?&#164; identified                               TbPI4KIII-&#226;?&#164; depletion growth. Cell growth uninduced (&#226;&#171;&#186;TET) <br />                                                                                         induced (&#226;&#171;&#185;TET) cells monitored 2 weeks. Induced cells <br /> BLAST search TIGR T. brucei database. The search                                 defect growth beginning day 5. Error bars indicate stan- <br /> originally identified open reading frame 1,746 nucleo-                            dard deviations. (C) Effect TbPI4KIII-&#226;?&#164; depletion PIP levels. <br /> tides BAC clone chromosome 4. The deduced amino                               PIP levels monitored inositol labeling. Uninduced cells and <br /> acid sequence corresponded 581-amino-acid protein                             cells induced 3 5 days incubated [3H]inositol 24 h. <br />                                                                                         The lipids extracted, separated TLC, quantitated by <br /> a predicted molecular mass 66 kDa. CLUSTALW analysis                                 scintillation counting. PIP levels normalized PtdIns (PI) <br /> revealed T. brucei enzyme shares 20.7% identity                            levels account differences cell number. Each bar represents <br /> 49.2% similarity human enzyme. The domain struc-                               average independent experiments; error bars represent <br /> ture predicted protein characteristic type III                        standard deviations. After 4 days induction PIP levels reduced <br /> PtdIns 4-kinases, highly conserved C-terminal PtdIns                             25%, 6 days levels reduced 70%. (D) Autoradio- <br />                                                                                         graph TLC-separated lipids. <br /> 4-kinase catalytic domain conserved N-terminal lipid <br /> kinase unique (LKU) domain (Fig. 1A). The catalytic domain, <br /> residues 319 581, shares 41.1% identity 68.8% similarity <br /> with human enzyme, LKU domain, residues 13                              pressed COS-7 cells, using pCMV-tag2b (N-terminally <br /> to 118, shares 14.0% identity 47.6% similarity (Fig. 1B                        Flag-tagged) mammalian expression vector. The overex- <br /> and C).                                                                                 pressed protein immunoprecipitated COS-7 cell ly- <br />    In order confirm identity gene, overex-                         sates using anti-Flag M2 resin. The Flag-tagged product was <br />  1112      RODGERS ET AL.                                                                                                         EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 4. Requirement TbPI4KIII-&#226;?&#164; normal morphology procyclic T. brucei. (A) Bright-field differential interference contrast microcopy <br /> reveals abnormal phenotype cells depleted TbPI4KIII-&#226;?&#164;. The normal phenotype (uninduced cells), fat phenotype, round phenotype <br /> are shown (both fat round phenotype images cultures induced 6 days). Bars, 10 &#226;?&#174;m. (B) Ultrastructural analysis TbPI4KIII- <br /> &#226;?&#164;-depleted cells. Transmission electron microscopy used examine ultrastructure uninduced cells (&#226;&#171;&#186;TET) cells induced 6 days <br /> (&#226;&#171;&#185;TET). Cells depleted TbPI4KIII-&#226;?&#164; abnormally shaped, cytoplasm filled vesicles. There accumulation vesicles <br /> in flagellar pocket (bottom right). Abbreviations: FP, flagellar pocket; Fl, flagellum; G, Golgi apparatus; M, mitochondria; N, nucleus; A, <br /> acidocalcisome. Bars, 500 nm. <br />  VOL. 6, 2007                                                        T. BRUCEI PHOSPHATIDYLINOSITOL 4-KINASE III-&#226;?&#164;                1113 <br />  <br />  <br /> detected lysates cells transfected pCMV-      Instead swimming medium, remain one <br /> TbPI4KIII-&#226;?&#164; vector cells transfected          place, spinning circles. <br /> vector (Fig. 2A). The Flag-tagged product migrated             Ultrastructural analysis TbPI4KIII-&#226;?&#164;-depleted cells by <br /> the 50-kDa 75-kDa markers sodium dodecyl sulfate             transmission electron microscopy revealed cytoplasm <br /> gels, consistent predicted molecular mass 66 kDa.       cells filled numerous vesicles various sizes, <br /> PtdIns kinase assays immunoprecipitated recombinant          ranging approximately 50 nm 750 nm (Fig. 4B). Most <br /> protein confirmed expressed protein able phos-      vesicles translucent, contained elec- <br /> phorylate PtdIns (Fig. 2B). Similarly type III PtdIns 4-ki-      tron-dense material morphologically reminiscent acido- <br /> nases sources, T. brucei enzyme activity wort-    calcisomes (36). The morphology major organelles, <br /> mannin sensitive completely inhibited 100 &#226;?&#174;M              including kinetoplast, flagellum, flagellar pocket, nu- <br /> wortmannin. No PtdIns kinase activity detected           cleus, appeared normal, polarized organization <br /> empty vector control immunoprecipitations.                          cell lost. As light micrographs, shapes <br />    TbPI4KIII-&#226;?&#164; essential growth procyclic T.          cells varied, having lost original slender <br /> brucei. In order investigate possible functional roles        morphologies. In addition, abnormal membrane structures ac- <br /> TbPI4KIII-&#226;?&#164; procyclic T. brucei, stable tetracycline-induc-    cumulated cytoplasm, cells abnor- <br /> ible RNAi procyclic cell line generated. An RNAi vector         mal accumulation vesicles flagellar pocket. <br /> which produces double-stranded RNA stem-loop structure            Scanning electron microscopy employed examine the <br /> in procyclic form T. brucei utilized (46). This construct       surface structure induced cells. The long slender <br /> contains region target gene (nucleotides 421 1002)      shape wild-type T. brucei defined highly polarized <br /> and complement separated &#226;&#172;&#402;550-bp region,              microtubule cytoskeleton (31). The microtubules orga- <br /> forms loop stem-loop transcript. The linearized          nized lengthwise, minus ends anterior and <br /> vector transfected 29-13 procyclic T. brucei elec-      plus ends posterior cell (35). The flagellum <br /> troporation, transfectants selected using phleomycin.      originates flagellar pocket posterior end and <br />    Northern blot analysis employed ascertain ex-        runs length organism (Fig. 5A). In contrast, scanning <br />                                                                     electron microscopy images induced cells revealed aber- <br /> pression TbPI4KIII-&#226;?&#164; double-stranded RNA results deg- <br />                                                                     rant morphology unique characteristic. Depletion of <br /> radation TbPI4KIII-&#226;?&#164; mRNA. After 24 h tetracycline <br />                                                                     TbPI4KIII-&#226;?&#164; resulted twisted phenotype, appears to <br /> induction, transcript levels virtually undetectable, com- <br />                                                                     begin posterior end parasite and, time, <br /> pared uninduced cells, &#226;?&#163;-tubulin levels <br />                                                                     results twisted round cell (Fig. 5). This twisted phenotype <br /> remain relatively unchanged (Fig. 3A). The transcript lev- <br />                                                                     seen round cells dividing nondividing <br /> els remain undetectable 6 days induction. <br />                                                                     cells (Fig. 5A B). Uninduced cells, progressing <br /> TbPI4KIII-&#226;?&#164; TbPI4KIII-&#226;?&#163; share 8% sequence iden- <br />                                                                     cell cycle, detected presence the <br /> tity amino acid level, highly unlikely that <br />                                                                     daughter flagellum/flagellar pocket cleavage furrow (Fig. <br /> knockdown TbPI4KIII-&#226;?&#164; RNAi affect levels <br />                                                                     5B). Induced cells progressing cell cycle a <br /> of TbPI4KIII-&#226;?&#163;. However, control transcript levels for <br />                                                                     second flagellum/flagellar pocket, daughter flagellum is <br /> TbPI4KIII-&#226;?&#163; examined, significantly      detached (Fig. 5B). The presence cleavage furrow <br /> changed 6 days induction (data shown),            detected TbPI4KIII-&#226;?&#164;-depleted cells, indicat- <br /> indicating gene knockdown specific TbPI4KIII-&#226;?&#164;.     ing cytokinesis defective cells. <br />    A cell growth defect observed beginning 4 days             Effect TbPI4KIII-&#226;?&#164; depletion Golgi morphology. <br /> tetracycline induction TbPI4KIII-&#226;?&#164; RNAi (Fig. 3B). The           PI4KIII-&#226;?&#164; plays major role Golgi maintenance secre- <br /> depletion TbPI4KIII-&#226;?&#164; led concomitant decrease PIP       tion organisms. The accumulation vesicles and <br /> levels (Fig. 3C). PIP levels decreased 25% 4 days     abnormal membrane structures seen ultrastructural <br /> of induction 70% 6 days. The decrease PIP           analysis TbPI4KIII-&#226;?&#164;-depleted cells indicated possible de- <br /> levels correlates timing appearance          fect Golgi apparatus. We used immunofluorescence <br /> growth defect, indicating synthesis PI4P             microscopy explore abnormal phenotype seen in <br /> TbPI4KIII-&#226;?&#164; required cell growth. T. brucei          cells depleted TbPI4KIII-&#226;?&#164;. We used TbGRASP a <br /> PtdIns 4-kinases, TbPI4KIII-&#226;?&#163; TbPI4KIII-&#226;?&#164;,        marker Golgi apparatus (24). In uninduced cells (Fig. <br /> presumably contribute total PIP levels. The decrease          6A), TbGRASP stains single Golgi structure located between <br /> PI4P 70% 6 days postinduction indicates                  nucleus kinetoplast. Cells depleted TbPI4KIII-&#226;?&#164; <br /> TbPI4KIII-&#226;?&#164; responsible majority PtdIns kinase        displayed abnormal staining <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. In cells, <br /> activity procyclic form T. brucei.                               TbGRASP stained multiple punctate structures, there <br />    Depletion TbPI4KIII-&#226;?&#164; results severe morphological       increase background staining, indicating cells <br /> defect. Light microscopic analysis induced cultures showed       depleted TbPI4KIII-&#226;?&#164; defect Golgi struc- <br /> the appearance abnormal phenotype, consisting fat          ture. Temperature-sensitive mutants yeast ortholog, <br /> and round cells approximately 4 days induction (Fig. 4A).     Pik1, display aberrant Golgi structure. This indicates <br /> The proportion cells having phenotype increases        TbPI4KIII-&#226;?&#164;, like Pik1, required maintenance of <br /> time. At day 4 6% 100 cells counted displayed        Golgi apparatus. <br /> abnormal morphology, day 5 12% abnormal, day             TbPI4KIII-&#226;?&#164; required proper localization protein <br /> 6 approximately 43% population abnormal. The mor-         markers flagellar pocket lysosome. The effect of <br /> phologically abnormal cells exhibit defects motility.       depleting TbPI4KIII-&#226;?&#164; localization lysosomal and <br />  1114      RODGERS ET AL.                                                                                                            EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 5. Scanning electron microscopic analysis TbPI4KIII-&#226;?&#164;-depleted T. brucei. (A) Scanning electron micrographs cells depleted of <br /> TbPI4KIII-&#226;?&#164; 6-day RNAi induction, showing abnormal twisted shape compared uninduced cells (top left). (B) Dividing TbPI4KIII-&#226;?&#164;- <br /> depleted cells lack cleavage furrow. Normally, dividing T. brucei cells contain evident cleavage furrow (arrow) flagella (*) (left panel). <br /> Dividing cells induced cultures (center right panels) contain flagella (*) contain obvious cleavage furrow. Some induced cells <br /> display detached flagellum. Bars, 1 &#226;?&#174;m. <br />  <br />  <br />  <br /> flagellar pocket markers examined. The p67 protein,                  extreme phenotype, intense staining the <br /> a type I membrane protein, used marker                       cytoplasm (Fig. 6B). Because p67 CRAM integral <br /> lysosome (1). In uninduced cells, p67 stains region                 membrane proteins, targeted respective <br /> the kinetoplast nucleus (Fig. 6A B). After induc-                 locations secretory pathway. The mislocalization of <br /> tion, p67 longer localized region stains             proteins reflect defect sorting and/or <br /> punctate structures cytoplasm. Similarly                trafficking Golgi apparatus defect <br /> staining TbGRASP, increase background p67                   Golgi apparatus itself. <br /> distribution. The CRAM protein, highly expressed                      Effect TbPI4III-&#226;?&#164; depletion cell cycle progression in <br /> type I membrane protein procyclic cells, used                     procyclic form T. brucei. Scanning electron microscopy reveals <br /> marker flagellar pocket. It putative lipoprotein                 dividing cells depleted TbPI4KIII-&#226;?&#164; lacked cleavage <br /> receptor, localizes flagellar pocket (28). Typically             furrow, indicating possible defect cytokinesis. In order to <br /> CRAM stains structures posterior region                  determine effect TbPI4KIII-&#226;?&#164; depletion cell cycle <br /> the cell, near kinetoplast (Fig. 6B). TbPI4KIII-&#226;?&#164; depleted                progression, kinetoplast nuclear contents uninduced <br /> cells display abnormal staining pattern. CRAM mislo-                   induced cells analyzed DAPI-stained cells. The <br /> calized cytoplasm, diffuse staining                   percentages cells, different stages cell cycle, <br /> staining multiple punctate structures, indicating CRAM                calculated population 100 cells sample. <br /> is longer localized flagellar pocket (Fig. 6B). Both                Analysis uninduced cells, log-phase growth, reveals <br /> CRAM p67 distributed cytoplasm                      approximately 65% cells G1 approxi- <br /> the TbPI4KIII-&#226;?&#164; depleted cells, CRAM displays                      mately 15 20% cells G2/M times exam- <br />  VOL. 6, 2007                                                                  T. BRUCEI PHOSPHATIDYLINOSITOL 4-KINASE III-&#226;?&#164;                 1115 <br />  <br />  <br />  <br />  <br />   FIG. 6. Abnormal localization protein markers Golgi apparatus, lysosome, flagellar pocket cells depleted TbPI4KIII-&#226;?&#164;. <br /> Immunofluorescence analysis markers Golgi apparatus (TbGrasp) (red) lysosome (p67) (green) (A) flagellar pocket <br /> (CRAM) (red) lysosome (p67) (green) (B) shown. The kinetoplast nucleus stained DAPI (blue). Cells depleted of <br /> TbPI4KIII-&#226;?&#164; display abnormal staining pattern markers examined, indicating defect Golgi trafficking. &#226;&#171;&#185;TET, cells induced for <br /> 6 days. Bars, 10 &#226;?&#174;m. DIC, differential interference contrast. <br />  <br />  <br /> ined (Fig. 7). After 5 days tetracycline induction, per-               33% percentage cells G2/M increased to <br /> centage cells G1 decreased 68% 43%                      45%. Although cells abnormal DNA content were <br /> percentage cells G2/M increased 16% 31%. By 6                   uninduced induced cultures, depletion of <br /> days induction, percentage cells G1 decreased                TbPI4KIII-&#226;?&#164; did result increase number <br />                                                                               anucleated cells (zoids) cells DNA content of <br />                                                                               greater 2K2N. Cells depleted TbPI4KIII-&#226;?&#164; repli- <br />                                                                               cated kinetoplast nucleus blocked <br />                                                                               cytokinesis. <br />                                                                                  In order investigate effects depletion of <br />                                                                               TbPI4KIII-&#226;?&#164; cell cycle, followed kinetoplast segre- <br />                                                                               gation nuclear division cell cycle (Fig. 8). <br />                                                                               Uninduced induced cells stained DAPI stain <br />                                                                               kinetoplast nucleus YL1/2 stain the <br />                                                                               basal body. The basal body T. brucei physically attached to <br />                                                                               kinetoplast tripartite attachment region, which <br />                                                                               extends cell mitochondrial membranes <br />                                                                               (33). Elongation maturation pro-basal body and <br />                                                                               nucleation new flagellum events cell <br />                                                                               cycle T. brucei (32). Kinetoplast replication division <br />    FIG. 7. Cell cycle analysis TbPI4KIII-&#226;?&#164;-depleted T. brucei. The cell    occur soon after, followed nuclear division (49). Depletion <br /> cycle followed induced RNAi cells time. The number             TbPI4KIII-&#226;?&#164; did affect duplication basal bodies or <br /> kinetoplasts nuclei 100 individual DAPI-stained cells counted     kinetoplasts. Both basal body kinetoplast dupli- <br /> at different time points induction. Cells independent exper- <br /> iments counted averaged day. Error bars represent the <br />                                                                               cated segregated (Fig. 8), direction <br /> standard deviations. In cells depleted TbPI4KIII-&#226;?&#164;, percentage      segregation aberrant. Normally, new basal body/ <br /> cells G2/M begins increase day 5, indicating cytokinesis block.    kinetoplast positioned posterior old basal body/kineto- <br />  1116      RODGERS ET AL.                                                                                                          EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 8. Analysis morphological cell cycle events TbPI4KIII-&#226;?&#164;-depleted T. brucei. The number position basal body (BB), <br /> kinetoplast (K), nucleus (N) examined fluorescence microscopy uninduced cells (&#226;&#171;&#186;TET) cells induced 6 days (&#226;&#171;&#185;TET). Cells <br /> were stained DAPI (blue) YL1/2 (green), identifies basal body. In cells depleted TbPI4KIII-&#226;?&#164;, basal bodies/kinetoplasts <br /> segregate perpendicular normal anterior-to-posterior directional axis. Nuclear division proceeds appropriate time, the <br /> positioning kinetoplasts nuclei abnormal induced cells. These cells characteristic 2K2N configuration, old <br /> basal body/kinetoplast nuclei seen uninduced cells (top right). Bars, 10 &#226;?&#174;m. <br />  <br />  <br />  <br />  <br /> plast. As cell cycle progresses, new basal body/kineto-                                         DISCUSSION <br /> plast moves posterior end cell. During                       Based amino acid sequence analysis enzymatic <br /> mitosis daughter nucleus moves posterior                 characterization expressed protein, suc- <br /> that eventually cell characteristic 2K2N configura-             ceeded cloning PI4KIII-&#226;?&#164; T. brucei. Although T. <br /> tion, old basal body/kinetoplast                   brucei kinase 20.7% identical overall human <br /> nuclei. The induced cells longer maintain strict spatial            PI4KIII-&#226;?&#164;, contains relatively conserved catalytic do- <br /> organization. The basal bodies/kinetoplasts segregate perpen-               main (41.1% identity) somewhat conserved LKU <br /> dicular normal anterior-to-posterior directional axis                domain (14.0% identity). <br /> (Fig. 8). Nuclear division proceeds appropriate time,                   TbPI4KIII-&#226;?&#164; essential protein T. brucei, deple- <br /> although positioning kinetoplasts nuclei                 tion RNAi causes severe growth morphological defects. <br /> abnormal induced cells. These cells                  There 3-day lag RNAi-mediated loss tran- <br /> characteristic 2K2N configuration, old basal body/                 scripts (undetectable 24 h induction) appearance <br /> kinetoplast nuclei.                                         abnormally shaped fat/round cells (beginning 4 days after <br />  VOL. 6, 2007                                                          T. BRUCEI PHOSPHATIDYLINOSITOL 4-KINASE III-&#226;?&#164;                    1117 <br />  <br />  <br /> induction). There corresponding lag RNAi induc-          round cells result TbPI4KIII-&#226;?&#164; depletion actually <br /> tion decreased PI4P levels, diminished &#226;&#172;&#402;25%          twisted, structural abnormality progresses time. <br /> at day 4 &#226;&#172;&#402;70% day 6 induction.                        This twisted phenotype caused defect the <br />    Synthesis PI4P important maintenance struc-       microtubule cytoskeleton, maintains shape the <br /> tural functional organization Golgi apparatus (13).        cell. The twist seen nondividing dividing cells, <br /> Loss PI4KIII-&#226;?&#164; organisms results morphological         indicating cell cycle dependent. Some twisted cells <br /> defects Golgi apparatus defects secretory           fat round appear relatively normal, <br /> pathway (4). Immunofluorescence analysis TbPI4KIII-&#226;?&#164;-              suggesting twist begin cells fat <br /> depleted cells reveals mislocalization Golgi marker          round. Twisting begins posterior cell, the <br /> markers lysosome flagellar pocket. These markers,         flagellum originates flagellar pocket. It possible that <br /> TbGrasp, p67, CRAM, redistribute predominantly            defect structure, attachment, movement flagel- <br /> appear membrane-bounded vesicles cy-             lum causes abnormal surface structure. Because flagellum <br /> toplasm. Ultrastructural analysis confirms cytoplasm         physically attached surface cell helical pattern <br /> of abnormally shaped cells filled heterogeneous pop-        (38), abnormal flagellar attachment subsequent movement <br /> ulation vesicles. These abnormal structures vesicle         induce helical pattern cell surface. <br /> intermediates, originate affected organelles,            Depletion TbPI4KIII-&#226;?&#164; procyclic T. brucei results the <br /> as endoplasmic reticulum, Golgi apparatus, endosomes.          accumulation cells G2/M stage mitosis, suggesting <br /> Ablation clathrin heavy chain procyclic T. brucei results    TbPI4KIII-&#226;?&#164; required cytokinesis. Unlike of <br /> similar morphological abnormalities, including rounded shape,         RNAi phenotypes disrupt cytokinesis, TbPI4KIII-&#226;?&#164;- <br /> accumulation cytoplasmic vesicles, mislocalization          depleted cells continue replicate DNA, indicat- <br /> CRAM (2). Trafficking CRAM flagellar pocket,            ing possible activation cell cycle checkpoint. Knock- <br /> not p67 lysosome EP procyclin surface,      dynamin-like protein procyclic parasites led <br /> clathrin dependent procyclic T. brucei (25). In mammalian          cell cycle block single round mitosis, is <br /> cells, clathrin adapter protein AP-1 recruited          suggestive cytokinesis checkpoint associated mito- <br /> trans-Golgi network pool PI4P synthesized PI4KII-&#226;?&#163;         chondrial fission (8). Although TbPI4KIII-&#226;?&#164; RNAi-induced <br /> (45). T. brucei lacks type II isoforms, possible   cells replicate DNA, unable correctly <br /> this function T. brucei performed TbPI4KIII-&#226;?&#164;. Loss       position kinetoplasts nuclei. Correct positioning of <br /> PI4P synthesis apparently causes global defect se-          cytoskeletal elements organelles important pro- <br /> cretion, affecting trafficking lysosome transport   gression cell cycle cytokinesis (32). After nu- <br /> to flagellar pocket. The importance PIs trafficking      clear division, daughter kinetoplast nucleus normally <br /> T. brucei illustrated finding depletion        segregate cell posterior, resulting kineto- <br /> TbVps34, class III PtdIns 3-kinase, bloodstream form T.          plast situated nuclei (38). In TbPI4KIII- <br /> brucei caused defect export VSG cell surface,       &#226;?&#164;-depleted cells, kinetoplasts nuclei segre- <br /> addition effect Golgi segregation (22).                     gate perpendicular normal lengthwise axis. Moreover, <br />    Impairment PI4P synthesis T. brucei influence       cells exhibit detached daughter flagellum, indic- <br /> lipid homeostasis organisms. For example, synthesis       ative possible abnormality flagellar attachment <br /> sphingomyelin Golgi apparatus COS-7 cells requires          zone. Establishment cleavage furrow requires correct <br /> ceramide transfer protein (CERT)-dependent transport cer-          positioning daughter flagellum and/or flagellar at- <br /> amide endoplasmic reticulum (42). CERT localizes          tachment zone (21). Loss impairment structural <br /> Golgi membranes pleckstrin homology domain,               cues explain block cytokinesis observed in <br /> which binds selectively PI4P. Oxysterol-binding proteins           TbPI4KIII-&#226;?&#164;-depleted cells. <br /> also bind specifically PI4P pleckstrin homol-           PIs implicated reorganization corti- <br /> ogy domains. Oxysterol-binding protein localizes Golgi         cal cytoskeleton cytokinesis (26). In particular, type <br /> apparatus response oxysterols, PI4P- Arf1-de-          III-&#226;?&#164; isoform PtdIns 4-kinase shown essential for <br /> pendent manner (19, 27, 29). Localization Golgi appa-          cytokinesis Saccharomyces cerevisiae (17, 44) cyto- <br /> ratus activates CERT-dependent transport ceramide           kinesis male meiosis Drosophila melanogaster (6). At <br /> Golgi apparatus, integrating sterol homeostasis           present, mechanism cleavage furrow formation T. <br /> sphingomyelin synthesis (34). Procyclic T. brucei able          brucei unclear, appears require actin/ <br /> synthesize ergosterol uses exogenous low-density li-         myosin II contractile ring (14). The lack protein markers for <br /> poprotein source cholesterol lipids (9). Inhibition    T. brucei cleavage furrow prevents localization studies. <br /> sterol biosynthesis Leishmania amazonesis causes similar         However, results indicate PI synthesis likely to <br /> alteration membrane-bound compartments seen de-            important cell division T. brucei, provide <br /> pletion TbPI4KIII-&#226;?&#164; (43). Inhibition 3-hydroxy-3-methyl-        evidence PI signaling pathways involved in <br /> glutaryl&#226;&#8364;&#8220;coenzyme A reductase procyclic T. brucei re-         regulating cytokinesis parasite. <br /> sults cytokinesis block accumulation                  PIs long recognized key regulators mem- <br /> abnormal membrane compartments (9). The similarities             brane trafficking yeast mammalian cells, are <br /> the morphological defects PI4P-depleted cells indicate        studies addressing roles signaling mole- <br /> loss TbPI4KIII-&#226;?&#164; affect lipid homeostasis addition       cules protozoans. Our studies reveal essential roles PI4P <br /> its effects secretory pathway.                                 synthesis Golgi maintenance, secretion, cytokinesis T. <br />    As evident scanning electron micrographs, fat           brucei, providing insight function signaling <br />  1118       RODGERS ET AL.                                                                                                                          EUKARYOT. CELL <br />  <br />  <br /> pathways parasite. Unexpected effects morphology                            24. He, C. Y., H. H. Ho, J. Malsam, C. Chalouni, C. M. West, E. Ullu, D. Toomre, <br />                                                                                            G. Warren. 2004. Golgi duplication Trypanosoma brucei. J. Cell Biol. <br /> were observed, leading observation twisted                                165:313&#226;&#8364;&#8220;321. <br /> phenotype cells lacking TbPI4KIII-&#226;?&#164;.                                                25. Hung, C. H., X. Qiao, P. T. Lee, M. G. Lee. 2004. Clathrin-dependent <br />                                                                                            targeting receptors flagellar pocket procyclic-form Trypanosoma <br />                           ACKNOWLEDGMENTS                                                  brucei. Eukaryot. Cell 3:1004&#226;&#8364;&#8220;1014. <br />                                                                                        26. Janetopoulos, C., P. Devreotes. 2006. Phosphoinositide signaling plays a <br />   This work supported National Institutes Health grant R01                       key role cytokinesis. J. Cell Biol. 174:485&#226;&#8364;&#8220;490. <br /> AI34432 (to M.A.P.) Welch Foundation grant I-1257 (to M.A.P.).                     27. Lagace, T. A., D. M. Byers, H. W. Cook, N. D. Ridgway. 1997. Altered <br />   We thank Jay Bangs, Gwo-Shu Mary Lee, Graham Warren                              regulation cholesterol cholesteryl ester synthesis Chinese-hamster <br /> antibodies used study Laurie Mueller assistance                   ovary cells overexpressing oxysterol-binding protein dependent the <br /> electron microscopy.                                                                       pleckstrin homology domain. Biochem. J. 326:205&#226;&#8364;&#8220;213. <br />                                                                                        28. Lee, M. G., B. E. Bihain, D. G. Russell, R. J. Deckelbaum, L. H. Van der <br />                                  REFERENCES                                                Ploeg. 1990. Characterization cDNA encoding cysteine-rich cell sur- <br />                                                                                            face protein located flagellar pocket protozoan Trypanosoma <br />  1. Alexander, D. L., K. J. Schwartz, A. E. Balber, J. D. Bangs. 2002. <br />                                                                                            brucei. Mol. Cell. Biol. 10:4506&#226;&#8364;&#8220;4517. <br />     Developmentally regulated trafficking lysosomal membrane protein <br />                                                                                        29. Levine, T. P., S. Munro. 1998. The pleckstrin homology domain of <br />     p67 Trypanosoma brucei. J. Cell Sci. 115:3253&#226;&#8364;&#8220;3263. <br />                                                                                            oxysterol-binding protein recognises determinant specific Golgi mem- <br />  2. Allen, C. L., D. Goulding, M. C. Field. 2003. Clathrin-mediated endo- <br />                                                                                            branes. Curr. Biol. 8:729&#226;&#8364;&#8220;739. <br />     cytosis essential Trypanosoma brucei. EMBO J. 22:4991&#226;&#8364;&#8220;5002. <br />                                                                                        30. Levine, T. P., S. Munro. 2002. Targeting Golgi-specific pleckstrin <br />  3. Audhya, A., M. Foti, S. D. Emr. 2000. Distinct roles yeast <br />                                                                                            homology domains involves PtdIns 4-kinase-dependent -indepen- <br />     phosphatidylinositol 4-kinases, Stt4p Pik1p, secretion, cell growth, <br />                                                                                            dent components. Curr. Biol. 12:695&#226;&#8364;&#8220;704. <br />     organelle membrane dynamics. Mol. Biol. Cell 11:2673&#226;&#8364;&#8220;2689. <br />                                                                                        31. Matthews, K. R. 2005. The developmental cell biology Trypanosoma <br />  4. Balla, A., T. Balla. 2006. Phosphatidylinositol 4-kinases: old enzymes <br />                                                                                            brucei. J. Cell Sci. 118:283&#226;&#8364;&#8220;290. <br />     emerging functions. Trends Cell Biol. 16:351&#226;&#8364;&#8220;361. <br />  5. Balla, A., G. Tuymetova, A. Tsiomenko, P. Varnai, T. Balla. 2005. A            32. McKean, P. G., A. Baines, S. Vaughan, K. Gull. 2003. Gamma-tubulin <br />     plasma membrane pool phosphatidylinositol 4-phosphate generated               functions nucleation discrete subset microtubules eu- <br />     phosphatidylinositol 4-kinase type-III alpha: studies PH domains           karyotic flagellum. Curr. Biol. 13:598&#226;&#8364;&#8220;602. <br />     oxysterol binding protein FAPP1. Mol. Biol. Cell 16:1282&#226;&#8364;&#8220;1295.             33. Ogbadoyi, E. O., D. R. Robinson, K. Gull. 2003. A high-order trans- <br />  6. Brill, J. A., G. R. Hime, M. Scharer-Schuksz, M. T. Fuller. 2000. A                membrane structural linkage responsible mitochondrial genome posi- <br />     phospholipid kinase regulates actin organization intercellular bridge              tioning segregation flagellar basal bodies trypanosomes. Mol. Biol. <br />     formation germline cytokinesis. Development 127:3855&#226;&#8364;&#8220;3864.                      Cell 14:1769&#226;&#8364;&#8220;1779. <br />  7. Bruns, J. R., M. A. Ellis, A. Jeromin, O. A. Weisz. 2002. Multiple roles       34. Perry, R. J., N. D. Ridgway. 2006. Oxysterol-binding protein vesicle- <br />     phosphatidylinositol 4-kinase biosynthetic transport polarized               associated membrane protein-associated protein required sterol- <br />     Madin-Darby canine kidney cells. J. Biol. Chem. 277:2012&#226;&#8364;&#8220;2018.                         dependent activation ceramide transport protein. Mol. Biol. Cell <br />  8. Chanez, A. L., A. B. Hehl, M. Engstler, A. Schneider. 2006. Ablation            17:2604&#226;&#8364;&#8220;2616. <br />     single dynamin T. brucei blocks mitochondrial fission endocytosis       35. Robinson, D. R., T. Sherwin, A. Ploubidou, E. H. Byard, K. Gull. 1995. <br />     leads precise cytokinesis arrest. J. Cell Sci. 119:2968&#226;&#8364;&#8220;2974.                 Microtubule polarity dynamics control organelle positioning, <br />  9. Coppens, I., P. J. Courtoy. 1995. Exogenous endogenous sources              segregation, cytokinesis trypanosome cell cycle. J. Cell Biol. <br />     sterols culture-adapted procyclic trypomastigotes Trypanosoma                128:1163&#226;&#8364;&#8220;1172. <br />     brucei. Mol. Biochem. Parasitol. 73:179&#226;&#8364;&#8220;188.                                       36. Rodrigues, C. O., D. A. Scott, R. Docampo. 1999. Characterization a <br /> 10. Cross, G. A. 1975. Identification, purification properties clone-specific       vacuolar pyrophosphatase Trypanosoma brucei localization to <br />     glycoprotein antigens constituting surface coat Trypanosoma brucei.             acidocalcisomes. Mol. Cell. Biol. 19:7712&#226;&#8364;&#8220;7723. <br />     Parasitology 71:393&#226;&#8364;&#8220;417.                                                           37. Roth, M. G. 2004. Phosphoinositides constitutive membrane traffic. <br /> 11. Barry, J., A. Janoshazi, J. L. Dupont, O. Procksch, S. Chasserot-Golaz, A.          Physiol. Rev. 84:699&#226;&#8364;&#8220;730. <br />     Jeromin, N. Vitale. 2006. Functional implication neuronal calcium           38. Sherwin, T., K. Gull. 1989. The cell division cycle Trypanosoma <br />     sensor-1 phosphoinositol 4-kinase-beta interaction regulated exocy-             brucei brucei: timing event markers cytoskeletal modulations. Philos. <br />     tosis PC12 cells. J. Biol. Chem. 281:18098&#226;&#8364;&#8220;18111.                                   Trans. R. Soc. London B 323:573&#226;&#8364;&#8220;588. <br /> 12. Graaf, P., E. E. Klapisz, T. K. Schulz, A. F. Cremers, A. J. Verkleij,      39. Sherwin, T., K. Gull. 1989. Visualization detyrosination single <br />     P. M. van Bergen en Henegouwen. 2002. Nuclear localization phosphati-               microtubules reveals novel mechanisms assembly cytoskeletal du- <br />     dylinositol 4-kinase beta. J. Cell Sci. 115:1769&#226;&#8364;&#8220;1775.                                  <br /> </body></html>