<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, July 1998, p. 3295&#226;&#8364;&#8220;3303                                                                              Vol. 180, No. 13 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />      Transcriptional Analysis Essential Genes Escherichia coli <br />      Fatty Acid Biosynthesis Gene Cluster Functional Replacement <br />          Analogous Salmonella typhimurium Gene Cluster <br />                                          YAN ZHANG1        AND   JOHN E. CRONAN, JR.1,2* <br />                    Departments Microbiology1 Biochemistry,2 University Illinois, Urbana, Illinois 61801 <br />                                             Received 2 February 1998/Accepted 18 April 1998 <br />  <br />             The genes encoding key fatty acid biosynthetic enzymes (called fab cluster) clustered the <br />          order plsX-fabH-fabD-fabG-acpP-fabF min 24 Escherichia coli chromosome. A difficulty analysis of <br />          fab cluster polar allele duplication approach (Y. Zhang J. E. Cronan, Jr., J. Bacteriol. <br />          178:3614&#226;&#8364;&#8220;3620, 1996) genes essential growth E. coli. We overcame this <br />          complication use fab gene cluster Salmonella typhimurium, close relative E. coli, provide <br />          functions necessary growth. The S. typhimurium fab cluster isolated complementation E. coli <br />          fabD mutant encode proteins &gt;94% homology E. coli. However, S. <br />          typhimurium sequences recombine E. coli sequences required direct polar allele duplication <br />          homologous recombination. Using approach, approximately 60% plsX <br />          transcripts initiate promoters located far upstream include upstream rpmF ribosomal protein gene, <br />          promoter located upstream plsX coding sequence (probably upstream gene, rpmF) is <br />          sufficient normal growth. We fabG gene obligatorily cotranscribed upstream <br />          genes. Insertion transcription terminator cassette (V-Cm cassette) fabD fabG genes the <br />          E. coli chromosome abolished fabG transcription blocked cell growth, providing indication <br />          fabG essential gene. Insertion V-Cm cassette fabH fabD caused greatly decreased <br />          transcription fabD fabG genes slower cellular growth, indicating fabD weak <br />          promoter(s). <br />  <br />  <br />    The bacterial fatty acid biosynthetic pathway type II,          acpP) (5), Vibrio harveyi (fabD-fabG-acpP-fabF) (27), and <br /> disassociated-enzyme, system, reactions               Rhodobacter capsulatus (plsX-fabH) (3). The recently com- <br /> the pathway catalyzed discrete cytoplasmic enzyme.                 pleted genomic sequence Helicobacter pylori contains <br /> Fatty acid biosynthesis Escherichia coli paradigm type           fab cluster homologs (31). However, fab cluster homologs <br /> II system, learned pathway                  H. pylori split relative fab cluster genes E. coli. <br /> recent years (12, 25). Recent work shown half            The H. pylori genome contains adjacent plsX fabH genes, <br /> the fatty acid biosynthesis (fab) genes clustered set          ribosomal protein gene (rpmF) located upstream of <br /> contiguous genes min 24 Escherichia coli chromosome              plsX, seen R. capsulatus, remainder genes <br /> in order fabH-fabD-fabG-acpP-fabF (4, 19, 22, 33),             E. coli fab cluster, fabD, fabG, acpP, fabF, are <br /> the rest fab genes scattered chromosome              clustered accA (which encodes acetyl-CoA carboxylase <br /> as separately transcribed genes (12). The proteins encoded              subunit) location 200 kb removed cluster, <br /> the genes cluster PlsX, b-ketoacyl-acyl carrier pro-            ribosomal protein gene (rps21) located upstream <br /> tein (ACP) synthase III, malonyl-coenzyme A (CoA):ACP                      fabD. Among gram-positive bacteria, similar fab gene clus- <br /> transacylase, b-ketoacyl-ACP reductase, ACP, b-ketoacyl-               ters reported Bacillus subtilis (plsX-fabD-fabG- <br /> ACP synthase II, respectively. We consider plsX gene (lo-              acpP) (17) Streptomyces glaucescens (fabD-fabH-acpP- <br /> cated immediately upstream fabH) E. coli              fabB; note acpP called fabC organism that <br /> cluster role (albeit poorly understood) phospho-             gene closely homologous E. coli fabF E. <br /> lipid biosynthesis (10). The plsX <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> defined                coli fabB) (28). <br /> single mutant allele, plsX50, confers sn-glycerol 3-phos-               Although proteins (except PlsX) encoded the <br /> phate auxotrophy strains carrying mutations plsB,                genes E. coli fab gene cluster extensively <br /> gene encodes sn-glycerol 3-phosphate acyltransferase,             studied, transcription regulation genes have <br /> first enzyme phospholipid synthesis. The cluster delimited           recently investigated (20, 21, 36). Podkovyrov and <br /> upstream rpmF gene, encoding L32 ribosomal pro-                 Larson (20) reported promoter probe studies suggesting that <br /> tein (19), downstream gene (pabC) involved p-                  rpmF-plsX genes cotranscribed, promoters <br /> aminobenzoic acid synthesis (7).                                           present, transcripts continue <br />    Similar fab gene clusters recently reported                fabHDG genes (20). However, results ob- <br /> other bacteria: Haemophilus influenzae Rd (fabH-fabD-fabG-                 tained transcriptional fusions carried multicopy plas- <br />                                                                            mids confirmed direct mapping chro- <br />                                                                            mosomal transcription, physiological relevance of <br />   * Corresponding author. Mailing address: B103 Chemical Life          various promoters determined. These workers have <br /> Sciences Laboratory, 601 South Goodwin Ave., Urbana, IL 61801.             reported presence promoter located the <br /> Phone: (217) 333-0425. Fax: (217) 244-6697. E-mail: j-cronan@uiuc          plsX coding sequence reads downstream fab genes <br /> .edu.                                                                      (21). <br />  <br />                                                                     3295 <br />  3296       ZHANG AND CRONAN                                                                                                                             J. BACTERIOL. <br />  <br />  <br />                                                                                       study listed Table 1. S. typhimurium MST2370 contains locked-in Mud- <br />                                                                                       P22 min 21.5 (putA1019::MudQ) S. typhimurium linkage map which <br />                                                                                       packages clockwise direction (2). The Mud-P22 phage DNA isolated <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                       S. typhimurium MST2370 induction mitomycin C (35) digested <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                       completion EcoRV NruI, fragments ligated to <br />                                                                                       pHSG575 cut SmaI. The ligation products transformed E. coli <br />                                                                                       fabD mutant strain, LA2-89 (which deficient malonyl-CoA-ACP transacy- <br />                                                                                       lase activity 42&#194;&#176;C), select complementing clones (13, 34). One of <br />                                                                                       EcoRV- NruI-derived plasmids (called pYZ48 pYZ58, respectively), <br />                                                                                       retained transformed strain LA2-89 confirm complemen- <br />                                                                                       tation. Strain YZ133, harbored plasmid pYZ53, containing S. typhimurium <br />                                                                                       fab cluster DNA, transformed plasmid pYZ47 produce plsX polar <br />                                                                                       duplication strain YZ137. Strain YZ152, harbored plasmid pYZ60, con- <br />                                                                                       taining S. typhimurium fab cluster DNA, transformed plasmid <br />                                                                                       pYZ37 plasmid pYZ69 produce fabG (strain YZ157) fabD <br />                                                                                       (strain YZ167) polar duplication strain, respectively. Plasmids pYZ47, pYZ37, <br />                                                                                       pYZ69 replicate R6Kg replication origin require the <br />                                                                                       plasmid R6K-encoded Pir protein replication. The wild-type E. coli recipient <br />                                                                                       strains lack Pir, transformants plasmid-encoded antibiotic resis- <br />    FIG. 1. Polar allele duplication plsX. Plasmid pYZ47 transformed       tance (to ampicillin chloramphenicol) result integration plasmid <br /> strain YZ133 (strain UB1005 harboring plasmid pYZ53), followed selection           DNA E. coli chromosome homologous recombination 59 <br /> for transformants resistant ampicillin, chloramphenicol, kanamycin. Plas-      portions fabG, plsX, fabD genes. These recombinant strains called <br /> mid pYZ47 (which unable replicate strain) integrated E. coli   YZ157, YZ137, YZ167, respectively. The recA derivatives strains YZ157, <br /> chromosome single-crossover event homologous recombination           YZ137, YZ167, called strains YZ158, YZ141 YZ168, respectively, were <br /> the truncated plsX9 gene pYZ47 intact plsX gene chromosome.         constructed transduction P1 phage lysate grown strain JC10289 with <br /> All elements indicated. The line represents E. coli chromosome,         selection tetracycline resistance, followed screening UV sensitivity. <br /> and line represents plasmid. Half-arrows numbers           Derivatives strain YZ141 kanamycin sensitive (indicating loss the <br /> below represent PCR primers used study (sequences given             S. typhimurium fab cluster plasmid pYZ53) obtained screening colonies <br /> Materials Methods). M13, M13 reverse-sequencing (248) primer purchased            arose cells plated rich broth (RB) agar plates lacking kana- <br /> from New England Biolabs.                                                             mycin. Strains YZ158 YZ168 cured plasmid pYZ60 transforma- <br />                                                                                       tion incompatible plasmids pYZ71 pYZ72 screened <br />                                                                                       colonies resistant spectinomycin sensitive kanamycin to <br />                                                                                       produce strains YZ166 YZ170, respectively. <br />    We began genes 39 end cluster                              Culture media growth conditions. Minimal E medium supplemented with <br /> reported transcriptional analyses fabD, fabG, acpP,                        4,000 mg glucose/liter, 100 mg methionine/liter, 10 mg thiamine/liter <br /> fabF genes (36). We addressed physiological relevance                        RB used growth bacterial strains (16). Antibiotics added the <br />                                                                                       following concentrations (in milligrams liter): kanamycin, 25; ampicillin, 100; <br /> of multiple acpP transcripts powerful genetic ap-                          tetracycline, 30; chloramphenicol, 34. Bacterial growth monitored with <br /> proach, polar allele duplication (Fig. 1). This method allows                         Klett-Summerson colorimeter green filter. <br /> blockage chromosomal transcription sequences up-                                 Plasmid isolation recombinant DNA techniques. Plasmid isolation was <br /> stream given promoter disruption coding                        performed modified alkaline lysis method (11) Qiagen Spin mini- <br />                                                                                       preparations. Southern blot analyses carried according Genius <br /> sequences downstream transcription. By use method,                         System User&#226;&#8364;&#8482;s Guide (Boehringer Mannheim Biochemicals). The probes were <br /> we showed major promoters                               plasmid pYZ37, pYZ47, pYZ69 labeled digoxigenin (DIG)-dUTP via <br /> transcribe acpP required expression physiological lev-                      random-primed labeling Genius 2 DNA labeling kit, purchased from <br /> els protein (36). In present study, polar allele du-                      Boehringer Mannheim. S. typhimurium Mud-P22 phage lysate preparation and <br /> plication used test possibility upstream                          DNA isolation performed according procedure Youderian and <br />                                                                                       coworkers (35). Low-stringency Southern blot analysis performed a <br /> genes fab cluster (plsX-fabH-fabD-fabG) transcribed                        DIG-dUTP-labeled PCR fragment (amplified primers 10 13) that <br /> as operon. We unable isolate polar allele duplica-                         contained E. coli fabD fabG genes, plus 59 fragment acpP. Other <br /> tions fabD-fabG segment methods used acpP,                          DNA manipulations performed standard procedures (26). <br /> suggesting transcription upstream promoter                                  DNA sequencing strands S. typhimurium fab cluster genes on <br />                                                                                       plasmid pYZ48 pYZ58 Genetic Engineering Facility, <br /> might necessary growth. In order conduct positive                         University Illinois Urbana-Champaign, Taq DNA polymerase cycle <br /> test hypothesis, cloned fab gene cluster                          sequencing Applied Biosystems 373 DNA sequencer, primers de- <br /> Salmonella typhimurium, cluster closely                           signed synthesized facility staff. <br /> related bacterium provide functional copies pro-                           RNA analyses. Total RNA isolated exponentially growing cells the <br />                                                                                       rapid isolation method (1). Reverse transcriptase-coupled PCR (RT-PCR) was <br /> teins needed growth E. coli providing                            performed RETROscript kit, purchased Ambion. The primer used <br /> target recombination E. coli gene segments needed                        cDNA strand synthesis random decamer mixture provided in <br /> to direct polar allele duplication (24). We report plsX                     kit. The primers used PCR listed below. Quantitative RT-PCR was <br /> polar allele duplication strain, S. typhimurium fab cluster                       carried Ambion kit according protocol Gilliland and <br /> plasmid, pYZ53, required cell growth, indicating                          coworkers (6). Briefly, plasmids pYZ64 pYZ66 used templates with <br />                                                                                       primers 1 7 primers 6 5, respectively, amplify competitive <br /> that plsX gene promoter(s) that                         DNA fragments. The concentrations competitive DNA fragments were <br /> promoter(s) required expression PlsX protein                        determined absorption 260 nm comparing fluores- <br /> perhaps downstream Fab enzymes physiological levels. In                            cence intensities DNA mass ladder (purchased Gibco BRL) <br /> contrast, viability fabG polar allele                          densitometry ethidium bromide-stained agarose gels. The reverse tran- <br />                                                                                       scriptase (RT) reaction product (1 ml) different concentrations compet- <br /> duplication strain depended expression S. typhi-                            itive DNA (as specified legend Fig. 4) added 25-ml PCR <br /> murium fabG gene distal promoters re-                             mixture. The sets primers used amplify competitive DNA were <br /> quired order obtain physiological levels FabG. Polar                         used RT-PCRs. The PCR products separated agarose gel <br /> allele duplication strains affecting fabD grow                          stained ethidium bromide quantitated densitometry. The ratios of <br />                                                                                       fluorescence intensities PCR products competitive DNA to <br /> very slowly, indicating weak fabD promoter is <br />                                                                                       RT-PCR products plotted function concentration of <br /> present.                                                                              competitive DNA (6). <br />                                                                                          For Northern blot analysis, whole-cell lysates separated electrophore- <br />                         MATERIALS AND METHODS                                         sis 0.8% formaldehyde agarose gels described Kornblum et al. (9). <br />                                                                                       Northern transfer performed standard procedures (26). Hybridization, <br />   Bacterial strains, plasmids, phage. All bacterial strains derivatives    washing, detection carried described Genius System <br /> E. coli K-12 S. typhimurium LT2. The E. coli strains plasmids used     User&#226;&#8364;&#8482;s Guide. Other experimental conditions given legend Fig. 6. <br />  VOL. 180, 1998                                                                TRANSCRIPTION OF THE E. COLI fab GENE CLUSTER                        3297 <br />  <br />  <br />                                                 TABLE 1. Plasmids E. coli strains used study <br />   Strain or <br />                                                            Relevant characteristics                                          Source reference <br />   plasmid <br />  <br /> Strains <br />   UB1005              F2 metB1 relA1 spoT1 gyrA216 lr/l2                                                                Lab collection <br />   LA2-89              fabD(Am) supE1 [fabD(Ts) phenotype]                                                               Lab collection (31) <br />   WM95                F9 128::Tn10-11 lacIq DlacZM15/lDE3 DlacX74 uidA::pir recA1 rpsL                                  W. W. Metcalf (15) <br />   JC10289             thr-1 ara-14 leuB6 D(gpt-proA)62 thi-1 D(recA-srl)306 rpsL31 srlR301::Tn10-84                     CGSCa <br />   YZ125               LA2-89/pYZ48                                                                                      This work <br />   YZ133               UB1005/pYZ53                                                                                      This work <br />   YZ137               YZ133::pYZ47 (plsX polar duplication chromosome)                                               This work <br />   YZ141               YZ137 D(recA-srl)306 srlR301::Tn10-84                                                             This work <br />   YZ142               LA2-89/pYZ58                                                                                      This work <br />   YZ143               Kans derivative strain YZ141 (loss pYZ53)                                                   This work <br />   YZ152               UB1005/pYZ60                                                                                      This work <br />   YZ157               YZ152::pYZ37 (fabG polar duplication chromosome)                                               This work <br />   YZ158               YZ157 D(recA-srl)306 srlR301::Tn10-84                                                             This work <br />   YZ159               UB1005 D(recA-srl)306 srlR301::Tn10-84                                                            This work <br />   YZ166               Transformation pYZ71 strain YZ158 cure pYZ60                                           This work <br />   YZ167               YZ152::pYZ69 (fabD polar duplication chromosome)                                               This work <br />   YZ168               YZ167 D(recA-srl)306 srlR301::Tn10-84                                                             This work <br />   YZ170               Transformation pYZ72 strain YZ168 cure pYZ60                                           This work <br />  <br /> Plasmids <br />   pACYC177            Apr Knr cloning vector; contains p15A replicon                                                    Lab collection <br />   pHSG575             Cmr cloning vector; contains pSC101 replicon                                                      27 <br />   pWM77               Suicide vector derived pJM703.1                                                              W. W. Metcalf (15) <br />   pYZ37               Insertion 700-bp PstI-EcoRV fragment (fabG9) pKM22 pWM77                  This work <br />                         3.5-kb V-Cm fragment pHP45V-Cm immediately upstream BamHI site <br />   pYZ46               500-bp plsX9 PCR product E. coli chromosome (amplified primers 8 3)               This work <br />                         treated T4 DNA Polymerase, cut SalI inserted pWM77 cut with <br />                         SmaI SalI <br />   pYZ47               Insertion 3.5-kb V-Cm BamHI fragment pHP45V-Cm BamHI site                   This work <br />                         pYZ46 <br />   pYZ48               Insertion 3.8-kb EcoRV fab gene fragment Mud-P22 phage DNA S.                      This work <br />                         typhimurium MST2370 SmaI site pHSG575 <br />   pYZ53               Insertion 3.8-kb BamHI-EcoRI fragment pYZ48 pACYC177 cut                      This work <br />                         BamHI DraI <br />   pYZ58               Insertion 2.2-kb NruI fab gene fragment Mud-P22 phage DNA S.                       This work <br />                         typhimurium MST2370 SmaI site pHSG575 <br />   pYZ59               Insertion BamHI-PvuII fragment pYZ48 PvuII-EcoRI fragment                    This work <br />                         pYZ58 pHSG575 cut BamHI EcoRI <br />   pYZ60               Insertion 4.6-kb BamHI-EcoRI (filled-in) fragment pYZ59 pACYA177 cut               This work <br />                         BamHI (filled in) DraI <br />   pYZ63               Insertion 652-bp PCR product E. coli chromosomal DNA (amplified                    This work <br />                         primers 1 3) pUC19 cut BamHI HindIII (filled in) <br />   pYZ64               Plasmid pYZ63 cut HindIII PstI blunt ended T4 DNA                      This work <br />                         polymerase ligated <br />   pYZ65               Insertion 459-bp PCR product E. coli chromosome (amplified primers 2           This work <br />                         5) pUC19 cut BamHI HindIII (filled in) <br />   pYZ66               Plasmid pYZ65 cut PstI ligated                                                  This work <br />   pYZ67               Insertion 600-bp fabD9 PCR product E. coli chromosomal DNA (amplified              This work <br />                         primers 9 11) cut EcoRI HindIII pBluescript II SK cut EcoRI <br />                         HindIII <br />   pYZ68               Insertion 3.5-kb V-Cm BamHI fragment pHP45V-Cm BamHI site                   This work <br />                         pYZ67 <br />   pYZ69               Insertion 4.2-kb SalI-NotI fragment pYZ68 pWM77 cut SalI NotI             This work <br />   pYZ70               Recircularization 4.5-kb XhoI fragment pMPM-K6V (deletion Kanr gene)             This work reference 14 <br />   pYZ71               Insertion 770-bp PCR product pYZ58 (amplified primers 16 17)              This work <br />                         pYZ70 cut NcoI XmnI construct gene fusion S. typhimurium fabG <br />                         control arabinose promoter <br />   pYZ72               Insertion 1,720-bp PCR product pYZ58 (amplified primers 18 17)            This work <br />                         pYZ70 cut NcoI XmnI construct gene fusion S. typhimurium fabD <br />                         fabG control arabinose promoter <br />   a <br />       CGSC, E. coli Genetic Stock Center, Yale University, New Haven, Conn. <br />  <br />  <br />  <br />   Primers used RT-PCR manipulations. In addition M13              GTCGACGTTGTTCAAAGTCAG; primer 4, CTGACTGCGCAGGAATAAT <br /> 16-mer reverse-sequencing primer New England Biolabs, primers used           CTGC; primer 5, CACCAGCATTGTGCTGTCACAACT; primer 6, GTTAG <br /> (sequences shown 59 39) follows: primer 1, GCAATGGTTGAAGA                  ATGTCATGGGAGGGGATTT; primer 7, GACGCGAACGTTGTTCAAAG <br /> TGAAATCATCC; primer 2, GTTAGATCATATGGGAGGG; primer 3, GAC                             TCAG; primer 8, TCGTTGGATCGGGGATAAACCG; primer 9, CTGGCGC <br />  3298       ZHANG AND CRONAN                                                                                                                        J. BACTERIOL. <br />  <br />  <br /> GCACCTGCGATCCAA; primer 10, GGGAATTCTTGACCGTTCTCAACT <br /> GG; primer 11, CGCAACAGATGCAGTCAACAG; primer 12, GCGAATTC <br /> GAAACCAATGGTGATGC; primer 13, GGTCTTCAACCTAAGAAGCATT <br /> GTTGG; primer 14, GAAGTTACCAACAATGCTTC; primer 15, TCCTGAT <br /> CAGACACGTTTGTCCTCCAGGGA; primer 16, GGAAAATCATGAGCTT <br /> TGAAGG; primer 17, CCCTAATAACGCAAATATTTTTC; primer 18, <br /> GGATTTAATCATGACGCAA. <br />    Genetic techniques. Transduction carried according method of <br /> Miller (16). Allele duplication described Metcalf et al. (15). <br /> Plasmids pYZ37, pYZ47, pYZ69 (Table 1), contain 59 portions of <br /> the fabG, plsX, fabD genes, respectively, maintained Pir-con- <br /> taining strain, WM95, transformed wild-type strains <br /> YZ152, YZ133, YZ152, respectively (which lack Pir), followed selection <br /> for transformants resistant ampicillin chloramphenicol. <br />    Nucleotide sequence accession number. The nucleotide sequence S. <br /> typhimurium fab gene cluster submitted GenBank accession <br /> no. AF044668. <br />  <br />                                                                                     FIG. 2. Organization fab cluster genes, promoters, tran- <br />                                 RESULTS                                          scripts. (A) The order genes E. coli S. typhimurium identical. The <br />                                                                                  percentages amino acid residues identical S. typhimurium E. coli fab <br />    Cloning sequencing S. typhimurium fab cluster.                     cluster proteins given arrows representing individual genes. <br />                                                                                  The S. typhimurium nucleotide sequence 85% identical E. coli fab <br /> As described below, failed isolate various                     gene cluster sequences. (B) The known promoters region depicted (the <br /> polar allele duplications E. coli fab cluster. This                   heights arrows intended crude idea relative promoter <br /> could poor luck (successful transformations                  strengths). The promoters upstream g30K (marked T) identified by <br /> 10 20 colonies) disruption essential transcription. In               S1 nuclease mapping Tanaka coworkers (30). The promoter plsX <br />                                                                                  (PL) identified primer extension studies Podkovyrov Larson <br /> order cope possibility, cloned fab                     (21). (C) The transcripts identified study prior studies (36) are <br /> cluster S. typhimurium, close relative E. coli, used               shown. The plsX transcripts detected RT-PCR, lengths <br /> plasmids carrying DNA fragment provide essential                     transcripts unknown. The scarcity longer plsX transcript could <br /> proteins lost polar allele duplication. Our isolation                  termination of transcripts initiated promoters <br />                                                                                  upstream g30K terminator mapped downstream rpmF (30). <br /> the S. typhimurium fab cluster based assumptions: <br /> (i) overall organization S. typhimurium fab <br /> cluster closely resemble E. coli (22), fab <br /> gene cluster widely conserved distantly                       fabD, plus partial gene fragments, 59 end fabG and <br /> related bacteria (5, 17, 27, 28) (ii) that, given similar-               39 end g30k, open reading frame (ORF) unknown <br /> ities genetic maps bacteria, fab gene                      function located upstream rpmF. In order obtain the <br /> cluster S. typhimurium located genome                       remainder fabG gene, phage particle DNA was <br /> min 24. To test assumptions correct, used                     digested NruI, cuts pYZ48 insert <br /> &#226;&#8364;&#339;locked-in&#226;&#8364;? Mud-P22 prophage (putA1019::MudQ) integrated                         DNA (within fabH gene), ligated pHSG575 cut <br /> at min 21.5 S. typhimurium genetic map (2, 35). Upon                      SmaI, plasmids complemented E. coli LA2-89 <br /> induction phage mitomycin C, escape                       selected. One clone, pYZ58, retained, <br /> from bacterial chromosome, packages successive                        sequenced, used construct plasmid carried a <br /> phage headfuls S. typhimurium chromosomal DNA                            cluster intact fabG gene. <br /> clockwise direction (2). The phage particles lysate                     The deduced protein product S. typhimurium gene <br /> isolated, encapsidated DNA purified digested                     number residues E. coli homolog, except <br /> with various restriction enzymes. Low-stringency Southern blot                   plsX ORF product 3 residues longer E. coli <br /> analysis performed DIG-dUTP-labeled fragment                          homolog. Each deduced proteins .90% amino acid <br /> (obtained PCR primers 10 13) contains                      identity analogous E. coli protein (Fig. 2), the <br /> complete E. coli fabD fabG genes plus 59 fragment                   nomenclature S. typhimurium fab genes as <br /> acpP gene. Positive bands detected (data shown),                        E. coli genes. The noteworthy difference be- <br /> suggesting locked-in phage DNA did contain                       tween fab gene clusters organisms 55-bp <br /> the S. typhimurium fab cluster genes.                                            deletion S. typhimurium fabG-acpP intergenic region <br />    We cloned S. typhimurium fab cluster genes                       compared E. coli (the intergenic regions between <br /> phage particle DNA complementation E. coli mutant                       S. typhimurium fab cluster genes similar to <br /> deficient malonyl-CoA-ACP transacylase activity 42&#194;&#176;C.                      E. coli). <br /> Strain LA2-89 carries amber <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> fabD gene                            Construction plsX gene polar allele duplication. We <br /> together supE tRNA suppressor (34). The combination                       used polar allele duplication (Fig. 1) demonstrate the <br /> of characteristics results temperature-                      promoter located immediately upstream acpP gene is <br /> sensitive malonyl-CoA-ACP transacylase temperature-                          sufficient expression ACP physiological levels (36). In <br /> sensitive growth. The phage particle DNA digested                         present study, extended approach upstream <br /> completion variety different restriction en-                   genes E. coli fab cluster. A PCR product beginning 40 bp <br /> zymes, fragments ligated low-copy-                      upstream small rpmF coding sequence ending 150 <br /> number vector pHSG575 (29). The resulting plasmids                          bp plsX coding sequence inserted the <br /> transformed restriction-deficient E. coli strain, plas-               oriR6Kg plasmid, pWM77 (15), immediately downstream of <br /> mid preparations pools resulting transformants                       V-Cm cassette blocks transcription upstream <br /> were used transform strain LA2-89, followed selection                  genes. We transformed resulting plasmid, pYZ47, the <br /> chloramphenicol-resistant clones grew 42&#194;&#176;C. Only                     wild-type E. coli strain UB1005, lacks Pir (and there- <br /> plasmid pool constructed EcoRV fragments gave trans-                        fore unable replicate pYZ47), transformants resistant to <br /> formants. One isolates, pYZ48, sequenced                        chloramphenicol ampicillin selected. Such <br /> was contain homologs E. coli rpmF, plsX, fabH,                   transformants formed single crossover the <br />  VOL. 180, 1998                                                      TRANSCRIPTION OF THE E. COLI fab GENE CLUSTER                                    3299 <br />  <br />  <br /> plasmid chromosome (Fig. 1). Several failed attempts <br /> to construct strain, data Podkovyrov <br /> and Larson (20), suggested cotranscription plsX <br /> gene downstream fab genes upstream ribosomal <br /> protein gene, rpmF, required growth. <br />    To test possibility, plasmid pYZ53, containing com- <br /> plete S. typhimurium rpmF, plsX, fabH, fabD genes, plus <br /> the 59 end S. typhimurium fabG gene (the insert DNA is <br /> the plasmid pYZ48), transformed the <br /> wild-type strain UB1005 produce strain YZ133. This strain <br /> was transformed pYZ47 obtain polar allele <br /> duplication. The rationale S. typhimurium fab <br /> genes provide E. coli chromosomal functions lost as <br /> a result formation polar allele duplication but <br /> would substrate recombination E. coli <br /> sequences plasmid pYZ47. Colonies resistant kanamycin <br /> (indicating presence S. typhimurium fab cluster plas- <br /> mid pYZ53), chloramphenicol, ampicillin (Fig. 1)                 FIG. 3. RT-PCR analysis plsX gene expression plsX polar allele <br /> obtained. PCR analyses using primer 1 M13 reverse-             duplication strain. The E. coli plsX polar allele duplication strain YZ143 the <br />                                                                        wild-type strain UB1005 grown mid-log phase RB, total RNA was <br /> sequencing primer (Fig. 1), plus Southern analysis           prepared described Materials Methods. RT-PCRs performed by <br /> such recombinant strain, YZ137, verified expected integra-         using Ambion RETROscript kit directed. Lanes 2 5, RT-PCRs; <br /> tion plasmid pYZ47 E. coli chromosome (data            lanes 7 10, PCRs approximately 0.5 mg total RNA the <br /> shown). A recA mutation transduced strain         template, lacking RT. One microliter RT reaction product, corre- <br />                                                                        sponded approximately 0.5 mg total RNA, used RT-PCR. The <br /> produce stabilized strain YZ141.                                   primer pairs used 1 4 (lanes 2, 3, 7, 8) 1 3 (lanes 4, 5, 9, and <br />    Strain YZ141 grown RB agar lacking kanamycin (in             10). In lanes 2, 4, 7, 9, RNA strain UB1005 analyzed. In lanes 3, <br /> agreement observations prior workers [23],          5, 8, 10, RNA strain YZ143 analyzed. Lanes 1 11 1-kb <br /> found p15A origin plasmids stably maintained             DNA ladder BRL, lane 6 left vacant. <br /> without selection) test pYZ53 (which contains S. <br /> typhimurium fab genes) essential cell viability. About 300 <br /> colonies screened kanamycin resistance,             transcript containing plsX. The principle this <br /> 20% colonies kanamycin sensitive. One             method utilize known concentration DNA fragment <br /> kanamycin-sensitive derivatives (called YZ143)             (obtained amplification primers used the <br /> tested plasmid isolation Southern analysis confirm           RT-PCRs) compete RT-PCR product. To obtain <br /> both loss S. typhimurium fab gene-containing plasmid            needed competitive DNA fragments, 652- 459-bp PCR <br /> pYZ53 presence expected plsX polar allele               fragments amplified E. coli chromosomal DNA with <br /> duplication E. coli chromosome (data shown). This           primer pair 1 3 primer pair 2 5, respectively, and <br /> strain expected chromosomal map, indicating             amplified fragments cloned vector pUC19. A <br /> promoter(s) located upstream plsX gene provides suffi-          HindIII-PstI fragment PstI fragment, respectively, were <br /> cient transcription support cell viability. This promoter           deleted plasmid inserts produce plasmids pYZ64 <br /> probably lies rpmF gene, based studies            pYZ66, respectively. These plasmids generate amplifica- <br /> Podkovyrov Larson (20), longer plsX tran-              tion products 522 346 bp, respectively, competitive <br /> script probably originates promoters mapped up-             DNA fragments. Two primers, 6 7, paired prim- <br /> stream g30k Tanaka coworkers (30).                           ers 5 1, respectively; primer pair 6 5 primer pair <br />    Transcription plsX gene. RT-PCR used detect           1 7 used amplify competitive DNA fragments from <br /> and quantitate plsX transcription strains UB1005                plasmids pYZ66 pYZ64, respectively, concentra- <br /> YZ143, attempts perform Northern analysis          tions competitive DNA fragment solutions deter- <br /> plsX transcription failed scarcity transcripts.      mined described Materials Methods. RT-PCR was <br /> When primer pair 1 3 (see Fig. 1) used, strain        carried known concentrations competitive DNA <br /> UB1005 strain YZ143 gave RT-PCR products 652 bp                 added reaction mixtures. A total-RNA preparation from <br /> (Fig. 3, lanes 4 5), consistent cotranscription        strain UB1005 used RT template, concen- <br /> plsX gene upstream ribosomal protein gene, sug-            tration cDNA formed taken proportional the <br /> gested Podkovyrov Larson (20). When primers 1 4             mRNA concentration. Primer 1 anneals sequence g30k <br /> (Fig. 1) used prime RT-PCR, product expected          upstream rmpF, primer 6 anneals sequence the <br /> <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> detected strain UB1005 (Fig. 3, lane 2)       59 end plsX gene, primer 5 primer 7 <br /> strain YZ143 (Fig. 3, lane 3), demonstrating polar            anneal sequences center plsX gene (Fig. 1). <br /> allele duplication blocked transcription upstream.         Therefore, RT-PCR product primer pair 1 7 <br /> These total-RNA preparations tested direct PCRs           represents products cotranscription rmpF and <br /> (in absence RT) sets primers (Fig. 3,          plsX, product primer pair 6 5 represents <br /> lanes 7 10) rule possibility DNA contamination        total plsX transcripts. When decreasing concentra- <br /> of RNA preparations. Primers 2 5 (Fig. 1)            tions 522-bp competitive DNA fragment synthesized <br /> used RT-PCR analysis, products expected length           primers 1 7 added reaction mixtures for <br /> were detected strain UB1005 strain YZ143 (Fig.          RT-PCR strain UB1005 primer pair, was <br /> 4 data shown), result consistent viability       shown concentration 652-bp RT-PCR product <br /> strain YZ143.                                                          increased concentration 522-bp PCR product <br />    Quantitative RT-PCR (6) used assess ratio         competitive DNA template decreased (Fig. 4A, lanes <br /> level transcript containing plsX rpmF       1 8). The ratios fluorescence intensities PCR <br />  3300       ZHANG AND CRONAN                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 5. Growth curve plsX polar allele duplication strain YZ143. <br />                                                                                     Strain YZ143 (h) strain YZ159 (a recA derivative strain UB1005) ({) <br />                                                                                     grown supplemented minimal E medium described Materials and <br />                                                                                     Methods. Growth monitored Klett-Summerson colorimeter a <br />                                                                                     green filter. <br />  <br />  <br />  <br />  <br />                                                                                     fabG fabD unsuccessful, avoid possibility <br />                                                                                     disrupting essential transcription, constructed plasmid <br />                                                                                     contained intact copies S. typhimurium rpmF, plsX, <br />                                                                                     fabH, fabD, fabG genes. This construct assembled in <br />     FIG. 4. Analysis plsX gene expression quantitative RT-PCR. RT-PCRs        vector pHSG575 tripartite ligation using inserts of <br /> were performed described Fig. 3, different masses compet-     plasmid pYZ48 pYZ58 subcloned the <br /> itive DNA added given mass RT-PCR mixture. (A) Quantitative <br /> RT-PCR primers 1 7. Different masses (15 3 1025, 12.5 3 1025, 10 3 <br />                                                                                     kanamycin-resistant p15A vector, pACYC177. The resulting <br /> 1025, 7.5 3 1025, 5 3 1025, 2.5 3 1025, 2.25 3 1025 ng) competitive DNA       plasmid, pYZ60, transformed wild-type strain <br /> amplified pYZ64 primers 1 7 added RT-PCR mixtures         UB1005 strain YZ152. Strain YZ152 trans- <br /> in lanes 1 8, respectively. (B) Quantitative RT-PCR primers 6      formed plasmid pYZ37, contains 240-bp segment <br /> 5. Competitive DNA (7.5 3 1025, 6.5 3 1025, 4.5 3 1025, 3.5 3 1025, 2.5 3 <br /> 1025, 2 3 1025, 1 3 1025 ng) added lanes 1 8, respectively. The <br />                                                                                     fabG coding sequence plus 300 bp upstream sequence <br /> fluorescence intensities bands agarose gel lane          (a chromosomal PstI-EcoRV fragment) V-Cm cassette <br /> quantified densitometry following ethidium bromide staining. The ratio      upstream fabG9, plasmid pYZ69, contains a <br /> the intensity PCR product competitive DNA RT-PCR           90-bp segment fabG coding sequence plus 470 bp of <br /> product calculated reaction, ratios plotted a <br /> function competitive DNA concentration. (C D) Plots data from <br />                                                                                     upstream sequence (a chromosomal EcoRI-HindIII fragment) <br /> panels A B, respectively.                                                       V-Cm cassette inserted upstream fabD9, trans- <br />                                                                                     formants resistant chloramphenicol ampicillin <br />                                                                                     selected. The expected integration events confirmed <br /> products RT-PCR products plotted                          PCR M13 reverse-sequencing primer plus either <br /> function concentration competitive DNA (Fig.                          primer 10 (for fabG) primer 6 (for fabD) Southern <br /> 4C). (When ratio 1, molar concentration                           analysis described (data shown). Strains YZ157 <br /> competitive DNA added reaction mixture identical                       (fabG duplication) YZ167 (fabD duplication) then <br /> the molar concentration plsX cDNA, propor-                          stabilized introduction recA mutation produce strains <br /> tional plsX mRNA concentration). The level cDNA                           YZ158 YZ168, respectively. <br /> synthesized rpmF-plsX cotranscripts 6.40 3 1025                           Transcription fabD fabG genes polar allele <br /> ng/mg UB1005 total RNA (Fig. 4C). Likewise,                             duplication strains. Strain YZ158, containing fabG polar <br /> 346-bp competitive DNA fragment obtained primers 6                             allele duplication, strain YZ168, containing fabD polar <br /> and 5 pYZ66 added RT-PCR mixture (Fig. 4B                           allele duplication, tested plasmid pYZ60, <br /> and D), concentration cDNA synthesized using                          carries S. typhimurium fab gene cluster, required <br /> plsX total transcripts original template 7.23 3 1025                     growth. We grew strains kanamycin selec- <br /> ng/mg UB1005 total RNA. Therefore, converted                             tion liquid medium screened kanamycin-sen- <br /> molar quantities, RT-PCR data indicate 60%                         sitive colonies success (300 colonies strain were <br /> of plsX transcription initiated promoters mapped up-                         screened). We cloned S. typhimurium fabG gene into <br /> stream g30K gene Tanaka coworkers (30),                               pYZ70, kanamycin-sensitive derivative vector pMPM- <br /> whereas 40% originated plsX-specific promoter                         K6V (14) intact gene positioned it <br /> (mean experiments).                                                        transcribed exclusively vector arabinose-regu- <br />    The normal growth rate plsX polar allele duplication                      lated araBAD promoter translated using vector <br /> strain YZ143 (Fig. 5) indicated transcription                         ribosome binding site. The resulting spectinomycin-resistant <br /> promoter(s) upstream rpmF required growth.                           plasmid, pYZ71, used transform strain YZ158, with <br />    Construction fabG fabD polar allele duplications.                         selection transformants resistant ampicillin, chloram- <br /> Initial attempts construct polar allele duplications upstream                    phenicol, tetracycline, spectinomycin, followed screen- <br />  VOL. 180, 1998                                                      TRANSCRIPTION OF THE E. COLI fab GENE CLUSTER                                  3301 <br />  <br />  <br /> ing kanamycin sensitivity. All steps carried out <br /> in RB medium supplemented 0.2% arabinose induce <br /> expression S. typhimurium fabG. Since plasmids pYZ60 and <br /> pYZ71 share p15A replication origin, plasmid incompati- <br /> bility presence pYZ71 expected cure this <br /> strain pYZ60. One cured strain (YZ166) resistant to <br /> ampicillin, chloramphenicol, tetracycline, spectinomycin <br /> but sensitive kanamycin retained, presence a <br /> single plasmid, pYZ71, confirmed plasmid isolation and <br /> restriction analysis (data shown). <br />    Strain YZ166 assayed dependence arabi- <br /> nose-induced S. typhimurium fabG gene expression streak- <br /> ing strain media supplemented 0.2% arab- <br /> inose induce arabinose promoter 0.4% glucose plus <br /> 0.002% fucose decrease basal expression arabinose <br /> promoter (18). We strain YZ166 gave detectable <br /> growth plates supplemented glucose-fucose, it <br /> grew wild-type strain, UB1005, plates supple- <br /> mented arabinose. Moreover, strain YZ166 failed grow <br /> in absence arabinose, indicating basal expression <br /> from araBAD promoter unable support growth. <br /> Therefore, blocking transcription fabG up- <br /> stream promoters lethal cell unless source <br /> of FabG supplied. <br />    Northern analysis performed examine fabG tran- <br /> scription strain YZ166. We did use fabG-specific probe           FIG. 6. Northern analysis fabG fabD gene expression fabG and <br />                                                                        fabD polar allele duplication strains YZ166 YZ170. The strains grown <br /> because difficult distinguish transcription      mid-log phase RB unsupplemented (strains UB1005 and <br /> E. coli chromosomal fabG gene plasmid-borne           YZ60), supplemented 0.2% arabinose (strain YZ166), supplemented <br /> S. typhimurium fabG gene. Instead, used probe con-           0.2% arabinose 0.4% glucose plus 0.002% fucose (strain <br /> tained 39 end acpP gene. We previously           YZ170). Cell lysates prepared described Materials Methods. The <br />                                                                        probe used DIG-labeled PCR product 39 end acpP obtained <br /> (36) fabG acpP cotranscribed products         amplification primers 14 15. (A) Northern analysis fabG expres- <br /> of 1.1 2.3 kb (see Fig. 6A, lanes 1 2). Both transcripts       sion. Lanes 1 2, strain UB1005; lane 3, strain YZ60; lane 4, strain YZ166. (B) <br /> disappeared acpP polar duplication strain,         Northern analysis fabD expression. Lane 1, UB1005; lane 2, strain YZ60; lane <br /> other transcripts, 0.3 1.9 kb, initiate acpP       3, strain YZ166; lane 4, strain YZ170 grown RB supplemented glucose <br />                                                                        plus fucose; lane 5, strain YZ170 grown RB supplemented arabinose. <br /> promoter (36), unaffected (Fig. 6A, lane 3). Essentially <br /> the result given insertion V-Cm cassette <br /> between fabD fabG genes (Fig. 6A, lane 4). Therefore, <br /> in contrast findings acpP plsX, fabG gene          cated weak promoter(s) located immediately upstream <br /> lacks promoter immediately upstream coding sequence           fabD coding sequence transcribes fabD fabG. The <br /> that sufficiently strong provide sufficient levels gene       weak nature promoter(s) evident Northern anal- <br /> product support growth. These results indicate        yses. The acpP-specific probe used Northern anal- <br /> 1.1-kb mRNA produced processing longer tran-                ysis fabG transcription (see above) used assay ex- <br /> script(s). This evidence demonstrating fabG       pression fabD gene fabD polar duplication strain, <br /> be essential gene.                                                  YZ170. In contrast results acpP fabG duplica- <br />    Similar constructions gave pYZ72, S. typhi-            tion strains, insertion V-Cm cassette fabH <br /> murium fabD fabG genes placed control            fabD genes resulted greatly decreased levels 1.1- <br /> the araBAD promoter. This plasmid used transform                2.3-kb transcripts (the 2.3-kb transcript virtually un- <br /> strain YZ168, colonies resistant ampicillin, chloram-           detectable) (Fig. 6B, lanes 4 5), result consistent the <br /> phenicol, tetracycline, spectinomycin selected            slow growth strain YZ170. These results indicated the <br /> then screened kanamycin sensitivity presence             promoter located immediately upstream fabD weakly <br /> arabinose described above. The resulting strain, YZ170,         functional 1.1-kb transcript pro- <br /> tested presence plasmids, plasmid pYZ72            cessed product longer transcript(s) initiated promoters <br /> was detected. Strain YZ170 assayed dependence           upstream fabD. <br /> S. typhimurium fabD fabG genes carried plasmid pYZ72 as <br /> described above. To surprise,                                             DISCUSSION <br /> detectable growth strain YZ170 glucose-fucose me- <br /> dium, colonies smaller grew               We developed polar duplication approach overcome a <br /> densely formed medium supplemented                  deficiency standard transcriptional mapping arises <br /> arabinose (data shown). This slow growth              gene multiple transcripts. The usual <br /> due low-level expression fabD (rather fabD plus          assumption protein production direct function of <br /> fabG fabG alone), substitution pYZ71, arab-         transcript abundance, hence, major transcripts consid- <br /> inose-regulated fabG plasmid, fabD fabG plasmid,               ered important minor transcripts. However, is <br /> pYZ72, produced modest increase growth             necessarily valid, minor transcript more <br /> the addition arabinose, indicating FabG levels did         efficiently translated major transcript, major tran- <br /> limit growth conditions.                                   script processed product longer transcript, <br />    The slow growth fabD polar duplication strain indi-          consequently scarce. On hand, a <br />  3302     ZHANG AND CRONAN                                                                                                         J. BACTERIOL. <br />  <br />  <br /> minor transcript reflect incomplete nature          promoter sufficiently strong support normal growth, <br /> most transcription terminators physi-         cotranscription fabD upstream genes is <br /> ological importance. Another complication standard tran-         needed. These transcripts initiate promoters lo- <br /> scriptional mapping processing primary products       cated upstream g30k (29) and/or promoter mapped <br /> of transcription, detected polar duplication    plsX primer extension studies Podkovyrov and <br /> method (see below). Use S. typhimurium homologs           Larson (21). The presence weak fabD promoter consis- <br /> provide possible essential functions construction     tent data Podkovyrov Larson (20). <br /> polar duplication strains increases applicability ap-       The fabG polar allele duplication strain carrying plasmid <br /> proach. We chose S. typhimurium based close relatedness      S. typhimurium fabG gene control the <br /> with E. coli (and interchangeable gene expression       araBAD promoter showed detectable growth unless the <br /> signals) known lack recombination homol-         S. typhimurium fabG induced. These data indicate the <br /> ogous genes organisms, largely mismatches        fabG gene essential growth, conclusion inter- <br /> the positions codons resulting inhibition       est, ORFs addition fabG classified <br /> recombination mismatch repair (24). However, genes          b-ketoacyl-ACP reductases various annotators the <br /> other organisms, H. influenzae, useful.       E. coli genomic sequence. These classifications ex- <br />    Our application polar duplication approach fab     plained b-ketoacyl-ACP reductases exist specific <br /> gene cluster shows that, gene cluster shows oblig-    different acyl chain lengths synthesis satu- <br /> atory cotranscription pairs genes, genes       rated versus unsaturated fatty acids existed. However, fraction- <br /> a promoter located immediately upstream coding se-           ation E. coli cell extracts gave single enzymatic activity <br /> quence provides sufficient transcription normal            functioned acyl chains tested (32), purified <br /> growth. Examples genes acpP plsX. In case       FabG catalyzes b-ketoacyl-ACP reductions required in <br /> of plsX, data bear argument Podkovyrov            novo synthesis long-chain fatty acids E. coli in <br /> Larson (20) cotranscription rpmF plsX play        reconstituted vitro (8). For reasons doubt <br /> an important role coordinating ribosome synthesis cell      ORFs play role membrane lipid synthesis; <br /> membrane synthesis. If rpmF-plsX cotranscription important,      instead, suggest function reductions b-keto- <br /> its lack expected slow block growth. However,        acyl-CoA intermediates pathways (e.g., poly-b-hy- <br /> this conclusion tempered lack information plsX      droxybutyrate synthesis). The fact fabG polar allele <br /> function. This gene discovered ability mutant       duplication strain YZ166 conditionally lethal mutant (the <br /> allele, plsX50, allow effective supplementation plsB mu-      strain grow medium lacking arabinose) allow <br /> tants E. coli sn-glycerol 3-phosphate certain carbon     determination stage fatty acid biosyn- <br /> sources (10). Only plsX allele characterized,      thetic pathway arrested depletion FabG protein in <br /> the interactions studies involved single plsB allele. The         strain YZ166. <br /> plsX50 mutation reported single-<span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> pair             It surprising S. typhimurium fatty acid biosyn- <br /> deletion upstream coding sequence (GenBank accession         thetic gene cluster high sequence identity E. coli <br /> no. M96793), unclear mutation causes       homologs nucleic acid amino acid levels (Fig. <br /> gain function loss function relative wild-type    1). However, 55-bp deletion intergenic region <br /> gene, complementation studies reported.          fabG acpP genes S. typhimurium relative to <br /> On hand, bacterial genomes sequenced date         E. coli unexpected, especially given other <br /> encode PlsX homolog, thus, protein likely       fab cluster intergenic regions similar bac- <br /> play important role cellular physiology. The prevalence    teria. The deletion removes 55 bp located upstream se- <br /> plsX-like genes bacterial genomes indicates         quence identical E. coli acpP promoter <br /> study enigmatic E. coli gene required.                   identified previously (36). Therefore, S. typhimurium acpP <br />    In contrast plsX acpP, fabG lacks proximal pro-         transcription differ somewhat E. coli. <br /> moter. Insertion V-Cm cassette fabD fabG            Why fab genes clustered genes <br /> abolished synthesis 1.1-kb mRNA, cotrans-         retain promoters? Since E. coli fatty acid synthesis is <br /> cript fabG acpP, 2.3-kb mRNA, transcript        tightly coupled pathway traces inter- <br /> fabD, fabG, acpP (Fig. 6). These results indicate      mediates seen (12), unlikely a <br /> abundant 1.1-kb mRNA initiated promoter lo-           need alter ratios proteins encoded genes. <br /> cated immediately upstream fabG gene produced         Internal promoters provide means combat the <br /> by processing longer transcripts. This consistent   natural polarity seen operons increase ex- <br /> the fact 1.1-kb mRNA abundant,          pression noncatalytic protein like ACP, needed in <br /> strong promoter detected immediately upstream          large quantities. However, effects natural polarity can <br /> of fabG coding sequence. Several DNA fragments contain-         canceled increasing relative efficiencies trans- <br /> ing overlapping regions immediately upstream fabG        lation downstream ORFs. It interesting the <br /> coding sequence cloned promoter detection vector        fab clusters bacteria utilize E. coli mix multi- <br /> in inserts drive lacZ expression. None         genic monogenic transcription. <br /> fabG fragments resulted b-galactosidase levels significantly <br /> higher background (data shown). We conclude                                   ACKNOWLEDGMENTS <br /> cotranscription fabG upstream genes required for <br /> growth.                                                               This work supported NIH grant AI15650. <br />                                                                       We thank Charles Miller Stanley Maloy useful suggestions <br />    Transcription fabD provides middle ground the <br />                                                                     regarding work. <br /> extremes acpP-plsX fabG. Our previous Northern anal- <br /> yses fabD transcription inconclusive. In repeated at-                                      REFERENCES <br /> tempts, faint diffuse bands detected fabD      1. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Sudman, J. A. <br /> probe (36). Our present data fabD retains           Smith, K. Struhl (ed.). 1987. Current protocols molecular biology, p. <br /> a proximal promoter 370 bp coding sequence,         4.8.1&#226;&#8364;&#8220;4.8.3. Wiley Interscience, New York, N.Y. <br />  VOL. 180, 1998                                                                         TRANSCRIPTION OF THE E. COLI fab GENE CLUSTER                                  3303 <br />  <br />  <br />  2. Benson, N. R., B. S. Goldman. 1992. Rapid mapping Salmonella            <br /> </body></html>