<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1988 Genetics Society America <br />  <br />  <br />  <br /> Fine-Structure Geneticsof ama-1, Essential Gene Encoding Amanitin- <br />      Binding Subunit RNA Polymerase I1 Caenorhabditis elegans <br />  <br />                                          Anne M. E. Bullerjahn DonaldL. Riddle <br />                              Division Biological Sciences, Universityof Missouri, Columbia, Missouri 65211 <br />                                                   Manuscript received March 18, 1988 <br />                                                   Revised copy accepted June 22, 1988 <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  A fine-structure genetic map constructed ama-1 N,an essential gene Caenorhabditis <br />               elegans encoding amanitin-bindingsubunit RNA polymerase11. Sixteen EMS-induced recessive- <br />               lethal mutations positioned gene determining intragenic recombination <br />               frequencies m118, <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> confers dominant resistance a-amanitin. The 16 mutants, <br />               isolated ama-l(mll8)background, include 13 early larval lethals,and are <br />               mid-larval lethals, 2 5 &quot; . Six mutants exhibit temperature-dependence severity their <br />               <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span>. Intragenic recombination betweenthe lethal siteand theparental resistance mutation was <br />               detected means resistance amanitin. Recombinants detected frequencies low 2 <br />               x         The segregation closely linkedflanking markers, unc-17 unc-5, revealed whether <br />               lethal mutation left right m l l 8 . By adding distances betweenthe extreme <br />               left right mutations, ama-1 gene estimated 0.01 1 map <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> long, m l l 8 positioned <br />               0.004 map <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> left-most lethalmutation. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> order thelethal mutationswith respect each <br />               other, viable heteroallelicstrains constructed using freeduplication, mDpl[unc-l7(e113)dpy- <br />               13(+)ama-1(+)]. The heteroallelic strains weresensitive amanitin, recombinationevents <br />               lethal mutations werespecifically selected means dominant amanitin resistance <br />               encoded recombinant chromosome. The segregation outside markers revealed left-right <br />               order lethal mutations. The position mutations gene nonrandom. Functional <br />               domains ama-1 gene indicated various lethal phenotypesare discussed. <br />  <br />  <br />                                                           1987), Drosophila (MORTINand KAUFMAN1984) and <br /> T      HEenzymeRNA          polymerase I1 synthesizes <br />         mRNA eukaryotic cells. It three <br /> DNA-dependent RNA polymerases, can <br />                                                           Chinese hamsterovary cells (INCLES 1978). A number <br />                                                           nonconditional lethal polymerase mutants Dro- <br /> be distinguished onthe basis function,subunit          sophila <br />                                                                have <br />                                                                   been <br />                                                                     characterized                     biochemically <br /> composition, sensitivity fungal toxin,     a-  (GREENLEAF     et al. 1980). In C. elegans, amanitin- <br /> amanitin (reviewed ROEDER1976). In nema-           binding subunit encoded ama-1 N.Worms car- <br /> tode Caeno-rhabditis elegans,as species, rying certain dominant mutations     gene a <br /> the enzyme hasa complex subunit composition,        100-foldincreaseinresistancetoamanitin            vivo <br /> complicates vitro studies (SANFORD,PRENCER         (ROCALSKI,    BULLERJAHN      RIDDLE1988), andlike- <br /> GOLOMB1985; SENTENAC        1985). Therefore,a genetic    wise, RNA polymerase I1 purified animals <br /> analysis RNA polymerase I1 useful         shows 100-fold increase resistance amanitin in <br /> understanding functional consequencesof various       vitro (SANFORD,    GOLOMBand RIDDLE1983). <br /> alterations subunit structure. Mutational studies      The nematode,C. elegans, particularly suited <br /> other organisms focused amanitin-bindingfor large-scale genetic selections required <br /> subunit, hasbeenshownto             large    detectionof rare mutantsor recombinants. In <br /> (210,000 D) subunit RNA polymerase I1 (GREEN-          addition,thereare       manyduplications,deficiencies <br /> LEAF 1983; BARTOLOMEI       CORDEN1987). There        visible markers available aid genetic mapping <br /> a r e amanitin-resistant mutants cultured cell (EDCLEYand RIDDLE1987). The primary reproduc- <br /> lines (CHAN,WHITMOREand SIMINOVITCH            1972; SO-  tive state self-fertilizing hermaphrodite. One is <br /> MERS,PEARSON INCLES 1975; AMATIet al. 1975;           able generate large populations self- <br /> WULFand BAUTZ1976) amanitin-resistant          ing, property especially useful forfine-structure <br /> mutants Drosophila (GREENLEAF       et al. 1979)   mapping.In C.elegans, fine-structuremaps the <br /> C. elegans (SANFORD,GOLOMBand RIDDLE 1983).               nonessentialgenes, unc-22 (MOERMANand BAILLIE <br /> Also, temperature-sensitive lethal mutations    1979; ROCALSKIa n d BAILLIE1985), unc-54 (WATER- <br /> affecting large subunit    RNA polymerase I1       STON, SMITHand MOERMAN            1982), andunc-15 (ROSE <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> yeast (HIMMELFARB,SIMPSONand FRIESEN                 BAILLIE1980) generated. In case <br /> Genetics 1 2 0 423-434 (October, 1988) <br />  424                                       A. M. E. Bullerjahn D. L. Riddle <br />  <br /> of unc-54, encodes myosin heavy chain,                                              TABLE 1 <br /> physical map mutant sites beeh elucidated                  Mutant phenotypes mapped ama-1 lethal alleles <br /> (DIBBet al. 1985). <br />     The large subunit RNA polymerase I1 presum-                                              <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>Lethal phenotype</span> <br /> ably interacts withseveralsmallersubunits,                Allele&quot;                       20&quot;                              25&quot; <br /> thought function DNA binding (ALLISONet al.                m252                     Embryonic                     L1 <br />  1985). Hence, genetic analysis subunit            m398                         L1                        L1 <br /> reveal numerous insights structure-function       re-       m370                         L1                        L1 <br /> lationships important enzyme. Toward               m365                         L2                        L1 <br />                                                                  m397                         L1                        L1 <br /> end, than30 mutant alleles ama-1 been <br />                                                                  m235      Mid-larval maternal-effect embryonicb Mid-larval <br /> obtained, including amanitin resistant n d lethal al-          m368                    Sterile adult              Early larval <br /> leles (ROGALSKI,    BULLERJAHN    RIDDLE1988),           m396            Maternal-effect embryonicb         Mid-larval <br /> the wild-type ama-1 gene beenclonedand se-                   m236                    Sterile adult              Early larval <br /> quenced (D. BIRDand D. RIDDLE,unpublished data).                 m332                         L1                        L1 <br />                                                                  m364                         L1                        L1 <br /> T h e fine-structure map used guide for <br />                                                                  m22 I                        LI                        L1 <br /> DNA sequence analysis mutant genes.                           m367                         L1                        L1 <br />     Almost lethal ama-1 mutations induced            m369             Mid-larval sterile adult       Mid-larval <br /> ethyl methanesulfonate (EMS) chromosome car-                m328                         L1                        L1 <br /> ryingthedominantamanitin-resistance           allele ama-        m329                         L1                        L1 <br />                                                                  m335      Mid-larval maternal-effect embryonicb Mid-larval <br /> l ( m l l 8 )and theclosely linked, semidominant dumpy <br /> (short body) marker, dpy-13, 0.05 map <span id='am-241' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>               With exception m335, wasinduced ama-I(m322), <br />  <br /> to left ama-1 (ROGALSKI       RIDDLE 1988).         mutations m - l ( m l l 8 ) background, listed in <br />                                                                left-right map order. <br /> Lethal mutations detected semi-Dpy <br />                                                   (dpy-          Adults lay eggs hatch. <br /> 13 ama-I/++) F1 hermaphrodites failed pro- <br /> duce viable, fertile Dpy progeny. Such semi-Dpy             vidually production resistant progenyin microtiter <br /> animals carried lethal mutationsclosely linked dpy-         wells containing 50 pl S medium 1% OP50 40 <br /> 13, semi-Dpy self-progeny tested            pg/ml amanitin (ROGALSKI           RIDDLE1988). <br />                                                                   Genetic markers: The genetic nomenclature used is <br /> loss dominant resistance amanitin    (ROCALSKI <br />                                                                described HORVITZ           et al. (1979), following <br /> and RIDDLE1988; ROGALSKI,BULLERJAHN              RID-      additions. Doubly mutant genes named according to <br /> DLE 1988). Loss amanitin resistance indicated          mutation, e.g., ama-l(ml18m369);and genotypes <br /> the lethal mutation ama-1 gene. Homozy-             chromosomalduplications givenin brackets, e.g., <br /> gous lethalama-1 mutants exhibit phenotypes ranging            mDpl[unc-l7(ell3) ama-I(+)].All mutant strains de- <br />                                                                rived wild-type N 2 strain (BRENNER1974). Muta- <br /> from nonconditional arrest early development             tions usedinthisstudyinclude                  unc-l7(e113) N , unc- <br /> temperature-sensitive sterility.                               17(e245) N , dpy-l3(e184) IV, unc-5(e53) IV, unc-8(e15) IV, <br />     In study, fine-structure genetic map       m - l ( m l l 8 )N , ama-l(m322) IV, lethal ama-1 al- <br /> constructed. Sixteen lethal mutations or-            leles listed inTable l . <br /> dered respect resistance mutation, m l l 8 ,          The isolation characterization ama-1 alleles <br />                                                                mDpl havebeendescribed                   (ROGALSKI   RIDDLE <br /> ninemutationshavebeen           ordered withrespect         1988; ROGALSKI,           BULLERJAHN   RIDDLE   1988).Mutations <br /> each other, recombination distances le-            convey larval lethal sterile adult (genetically lethal) <br /> thalsiteshave      determined usingnovel proce-           phenotypes classed lethals. With exception of <br /> dures based selection rare amanitin-resistant            m221 m335, lethal mutations induced the <br /> recombinants amanitin-sensitive nonrecombi-              m - l ( m l l 8 ) allele chromosomemarked dpy- <br />                                                                13(e184),providing common genetic backgroundfor vir- <br /> nants. T h e positive selection growth amanitin         tually crosses. The 772,221 mutation induced the <br /> permitted screening animals detect            m - l ( m l l 8 ) allele chromosome marked unc-8, <br /> recombinationinintervalsas        small 2 X        map      m335 mutation induced ama-l(m322) <br /> units, corresponding toless 200 bp DNA.                allele. The m l 1 8 m322 mutations confer dominant <br />                                                                resistance amanitin (SANFORD,          GOLOMB   RIDDLE1983; <br />             MATERIALS AND METHODS                              ROGALSKI,          BULLERJAHNand RIDDLE 1988). The dpy- <br />                                                                13(e184) unc-8(e15) alleles semidominant; other <br />    Culture conditions: Nematodes grownon NG agar          mutations recessive. <br /> plateswith OP50 strain Escherichia coli (BRENNER           Strainconstructions: To maplethal ama-1 mutations <br /> 1974). Amanitin-resistant wormswereselectedinliquid            relative m118 mutation, strains genotype + dpy-13 <br /> cultures containing 1 ml 2.5% (w/w)suspensionof           ama-l(ml18mx)+/unc-l7                ++ unc-5 constructed, <br /> OP50 S medium (SULSTON      BRENNER1974) 20         mx lethalmutation m - l ( m l l 8 ) gene. <br /> pg/mlamanitin (a-amanitin, Sigma).Dueto expense            Most lethal mutants maintained heterozygotes <br /> and toxicity amanitin, liquid cultures smallest      balanced nT1, reciprocal translocation between <br /> practicalvolumewereused.Populationdensities          ranged    chromosomes N V (FERGUSON                  HORVITZ 1985). <br /> from approximately 10,000 150,000 larvae/ml. Herma-         Balanced strains crossed withN2 males, semi-Dpy <br /> phrodites grew liquid cultures tested indi-   malecross-progenywerematedwith                    unc-17unc-5 her- <br />                                                         Fine-Structure ama-1 Map                                                     425 <br /> maphrodites. Because double unc animals mated poorly,                                    + <br />                                                                         mDpl[unc-l7]; dpy-13 ama-I(ml18mx) unc-5/unc-l7 <br /> the semi-Dpyhermaphrodite cross-progeny picked                                          +. <br />                                                                         dpy-13 ama-l(mll8my) Hermaphrodites lattergen- <br /> as L4 larvae propagated generation. A her-                  otype segregated semi-Dpy Unc, Wild progeny, and <br /> maphrodite producing unc-17unc-5                  self-progeny(con-     kept. Complementation tests unc-5/+ and <br /> firmed complementation tests unc-l7/+ unc-5/                unc-l7/+ males performed insure recombi- <br /> +   males) used source 20 hermaphrodites            nation outside markers occurred. If unc-5 and <br /> establish populations screened recombinants                unc-17 mutations carried strain, onesemi- <br /> described below.                                                        Dpy hermaphrodite placed 60 large (100 X <br />     To map lethal mutations relative other, free            15 mm) Petri plates begin growing large populations for <br /> chromosomal duplication, m D p l ( N ; f ) ,was used construct       mapping experiments, described below. <br /> viable heteroallelic strains genotype mDpl[unc-l7(el13)                 Mapping lethal mutations relative m118: The follow- <br /> dpy-13(+)ama-l(+)unc-5(+)];unc-lT(e245)dpy-13ama-                       ing procedure used mapping lethal ama-1 mutations <br /> l(ml18my)   +/+        dpy-13 ama-l(ml18mx) unc-5,where mx         (for example, m328) relative resistance mutation, <br /> lethal ama-1 mutation believed to right                                                                 + <br />                                                                         m118. One hermaphrodite (Po)of genotype dpy-13 ama- <br /> lethal my, based position respect m118. The              l(m118m328) +/unc-17      ++       unc-5 placed 20 <br /> duplication marked unc-l7(e113) cross                large (100 X 15 mm) plates lawns OP50 incu- <br /> demonstrated recombination duplication                  bated 20&quot; 1 week (two generations). When bac- <br /> the normal chromosome N (ROGALSKIand RIDDLE1988).                       teria consumed, worms washed into <br /> The unc-17-marked duplication permitted identification               large (150 x 18 mm) sterile culture tubes M9 buffer <br /> self-progeny homoallelic              unc-17dpy-13ama-          (BRENNER1974). Populations plates com- <br /> l(ml18my)    +        chromosome Unc phenotype (see            bined tube,containing 5 ml M9 buffer, the <br /> below). A duplication marked unc-17 unc-5                 tubes shaken 20&quot; 3 days accumulate L1 <br /> might apreferable balancer heteroallelic                  larvae. The larvae (Fs generation) tube then <br /> stocks, duplication available.                       separated gravity adults (most were <br />     Construction heteroallelic lethal strains involved sev-          dead), divided small (1 00 X 13 mm) sterile culture <br /> eral steps. First, hermaphrodites lethal ama-I mutations           tubes 1 mlof 2.5% (w/w) OP50 S medium 20 <br /> balanced nT1 mated N2 males. Semi-Dpy male                 pg/ml amanitin, shaken 20&quot; 4 days. Selections <br /> cross-progeny mated unc-17 and/or unc-5                  performed way L1 L2 stages <br /> hermaphrodites, semi-Dpy hermaphrodite cross-prog-                  sensitive amanitin (SANFORD,         GOLOMBand <br /> eny pickedas L4 larvae. Upon selfing, her-                   RIDDLE1983). The typical selection experiment included <br /> maphrodites segregated homozygousUncs and, low                     independent populations, consisting the <br /> frequency, semi-DpyUnc recombinants. These recombi-                     pooled progeny semi-Dpy Pohermaphrodites. After <br /> nants maintained heterozygous stocks genotype                4 days, adults grown tubes containing aman- <br /> dpy-13 ama-1 unc5/++ unc-5, unc-17 dpyl3 ama-llunc-                  itin retested microtiter wells containing solution <br />  17 ++,     crossed Wild males ofgenotype            1% OP50 S medium 40 pg/ml amanitin. The <br /> mDpl[unc-I7];unc-17 dpy-13/++. If stock carried unc- <br />                                                                         genotypes resistant progeny confirmed through <br /> 5 , semi-Dpy cross-progeny picked asL4 larvae. If <br />                                                                         complementation tests unc-5/+ unc-l7/+ males. <br /> the stock carried unc-17, semi-Dpy Unc cross-progeny <br /> were picked. In case stocks carrying unc-5, semi-            When mapping temperature-sensitive mutations (m235, <br /> Dpy progeny different genotypes: dpy-13                m368, m396, m236 m369), selection amanitin and <br /> ama-l(ml18mx) unc-5/+++, unc-17 dpy-13           +/++       unc-5,   verification resistant worms microtiter wells per- <br /> mDpl[unc-171; unc-17 dpy-13 ++/+ dpy-13 ama-l(mll8mx)                   formed 25&quot;, mutants grow adult <br /> unc-5. The desired hermaphrodites lattergenotype               stage 20&quot; amanitin. <br /> segregated Dpy Dpy Unc progeny, Unc-5                       To determine numberof animals population, <br /> Wild progeny. Semi-Dpy hermaphrodites segregated                   5-pI samples taken small culture tube and <br /> the appropriate progeny classes selfeduntil true-                diluted 45 pl M9 buffer. Two 5-pl samples taken <br /> breeding semi-Dpy stock genotype mDpl[unc-17]; dpy-     +            dilution grown small (60 X 15 mm) Petri <br />                                                                         plates OP50. After 3 days 20&quot; (or 2 days 25&quot;), <br /> 13 m - l ( m l l 8 m x )unc-5 established. <br />     In case stocks carrying unc-17, semi-Dpy Unc             adults plate counted. These counts (up to <br /> progeny different genotypes: unc-17 dpy-               8 1 adults plate) included heterozygotes because <br /> 13 ama-l(mll8my)/unc-17       ++        (selfed), unc-17 dpy-lj/unc-    unc-17 unc-5 segregants grew slowly. <br /> 17  +,    mDpl[unc-l7];unc-17 dpy-13 +/uric-1 7 dpy-13 ama-             The recombinant phenotypes depended relative <br /> I ( m l l 8 m y ) . The genotype desired, her-            positions m l I 8 mx. If mx right m118, <br /> maphrodites genotype recognized                recombinant chromosome restored amanitin <br /> segregated Dpy Unc, homozygous Unc-17 progeny.                  resistance dpy-13 m - l ( m l l 8 ) unc-5, and <br /> Again, semi-Dpy Unc hermaphrodites segregated                  resistant recombinants Dpy semi-Dpy <br /> appropriate progeny classes selfed true-breed-             Unc-5 phenotype (Figure la). If mx left of <br /> ing, semi-Dpy Unc stock genotype mDpl[unc-l7];unc-17                 m l l 8 , recombinant chromosome been <br /> dpy-13 ama-l(mll8my)was established.                                    unc-17 ama-l(ml18), resistant recombinants have <br />     For final step construction heteroallelic lethal          semi-Dpy Unc-17 phenotype (Figure 1b). <br /> strains, hermaphrodites genotype mDpl[unc-17];+ dpy-                    The strain constructed map m221 relative m118 had <br /> 13 m - l ( m l l 8 m x ) unc-5 crossed N2 males,      genotype + ama-l(ml18m221) unc-8 / dpy-13         ++    a <br /> Wild male progeny selected. These males, genotype               semi-Dpy semi-Unc phenotype. If m221 right of <br /> mDpl[unc-l7];+ dpy-13 ama-l(ml18mx) unc-5/++++,were                     m118, recombinant chromosome been <br /> crossed withmDpl[unc-l7];unc-17 dpy-13 ama-I(ml18my)+                   m - l ( m l l 8 ) ++, resistant recombinants wouldhave <br /> semi-Dpy Unc hermaphrodites, semi-Dpy non-Unc L4                    semi-Dpy semi-Unc phenotype. If m221 to <br /> cross-progeny selected. These progeny                  left m l l 8 , recombinant chromosome would <br /> different genotypes: unc-17dpy-13ama-l(ml18my)/+++,                     havebeen dpy-13ama-l(mll8)unc-8,                resistant <br />  426                                                 A. M. E. Bullerjahn D. L. Riddle <br />  <br />                                                                          b <br />            LETHAL <br />                 TOTHE   OF <br />                     RIGHT                                     m178                              OF <br />                                                                                              LEFT <br />                                                                                  LETHAL TO THE                                 ml18 <br />  <br />  <br />  <br />  <br />               +           dpy-13 <br />                                            m -1 <br />  <br />                                          m l l 8 m328          +                    +     m252 <br />                                                                                             13dpy- <br />                                                                                                              -  ma-1 <br />  <br />  <br />  <br />                                                                                                                     &quot; <br />                                                                                                                      ml18 <br />  <br />                                                                                                                      &quot;I <br />                                                                                                                                     + <br />                                                                                &quot;&quot;&quot;&quot;I <br />                                                                                  unc-17           +                +             unc-5 <br />  <br />  <br />  <br />  <br />                                    $r <br />               +           dpy-13         m 1 18                unc-5            unc-17            +                 m118           + <br />                                                          ~~ <br />  <br />  <br />  <br />  <br />               +           dpy-13         m l l 8 m328          +                unc-17            +                    +         unc-5 <br />  <br />                                    DPY                                                               UNC-17 <br />  <br />  <br />  <br />  <br />                +          dpy-13         m 1 18     unc-17 unc-5                                  +                 mll8           + <br />  <br />             unc-17           +             +                   unc-5               +          dpy-13         m252 m l 1 8          + <br />                                    UNC-5                                                             SEMI-DPY <br />      FIGURE 1.-Genotypes strains used position lethal alleles relative mZZ8. (a) If lethal mutation right m118 (e.g., <br /> m 3 2 8 ) , recombinant chromosome resulting recombination event m l Z 8 lethal mutation carries dpy-13, ama- <br /> l ( m l Z 8 ) , unc-5. Animals carrying recombinant chromosome resistant amanitin, Dpy semi-Dpy Unc-5 in <br /> phenotype. . - (b) If lethal mutation left mZ18 (e.a., <br />                                                                    - m252), recombinant chromosome carries m - l ( m l Z 8 ) unc-17, <br /> and recombinant animals semi-Dpy Unc-17 phenotype. <br />  <br /> recombinants semi-Unc Dpy Unc semi-                       maphrodites came parent             genotype <br /> Dpy phenotype.                                                            mDpl[unc-17];unc-17dpy-13ama-l(mll8my)                     +/+ <br />                                                                                                                                        dpy-13 <br />    The recombination frequency ( p ) calculated using                    ama-l(ml18mx) unc-5. When selfed, hermaphrodites <br /> only classes progeny: number resistant Dpy                     segregated classes homozygous progeny addition <br /> recombinants (or semi-Dpy mx left m l l 8 )                 tothe desired heteroallelic class couldgiverise           to <br /> and number sensitive parental-type heterozygotes. A                   recombinants (see Table 3). The homozygous class, of <br /> Punnett square shows expected frequencies                     genotype mDpl[unc-17]; unc-17 dpy-13 ama-l(m1 lamy),         had <br /> classes total zygotes p( 1 - p ) ( 1 - p)&apos;               semi-DpyUnc phenotype waseasily distinguished <br /> respectively, p 2 negligible. Thus, define R                heteroallelic semi-Dpy class. However, second <br />                                              + <br /> the ratio: resistant Dpy/resistant Dpy parentals, R =                   homozygous class, genotype mDpl[unc-l7]; dpy-13 ama- <br />                       + <br /> p(1 - p ) / p(l - p ) (1 - p)&apos;. This solved p = R .                l ( m l 1 8 ) u n c - 5was <br />                                                                                                     , semi-Dpy distinguished <br /> One map <span id='am-242' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> corresponds 1O O p .                                         heteroallelic classby appearance. Consequently, <br />    Mapping lethal mutations relative other: The                      60 plates screened,    days, plates <br /> procedure mapping lethal ama-1 mutations relative                     containing semi-Dpy Unc progeny (usually 35-40% of <br /> each essentially sameas mapping                       total plates) discarded. <br /> mutations relative m118,                         Using orderof lethal alleles predicted two-factor <br /> larger scale. One semi-Dpy Pohermaphrodite placed                     distances mutations m l l 8 , &quot;rightward&quot; lethal <br /> each 60 large plates lawns OP50. The Po her-                      (mx) linked unc-5, &quot;leftward allele (my) was <br />                                                          Fine-Structure ama-1 Map                                                       427 <br />  <br /> a                                                                       b <br />          unc-17        +                     +                   +               unc-17      +                   +                     + <br /> mDp 1                                                                   mDp 1 <br />  <br />                                                                                                      c                       -         + <br />                                                                                  unc-17     dpy-73             m398 m178 <br />                                                                           N <br />                                                                                 &quot;&quot;&quot;1- <br />                                                                                                               L&quot;&quot;&quot;&quot; <br />                                                                           IV <br />            +        dpy-13    L       ml18       m329   1     unc-5                     +   dpy-13 <br />                                                                                                      -m25.2          mil8 <br />                                                                                                                              L <br />                                                                                                                                       unc-5 <br />  <br />  <br />  <br />  <br />                                                          l +                     unc-17     dpy-73   r               mll8     1       unc-5 <br />  <br />  <br />  <br />  <br />             +        dpy-13       L   mll8       m329   1      unc-5                    +   dpy-13   LM52            mll8     A       unc-5 <br />  <br />  <br />                                       DPY                                                                DPY UNC <br />    FIGURE2,&quot;Genotypes strains used position lethal alleles relative other. Viable heteroallelic strains constructed using <br /> the duplication, mDpl[unc-I7(el13)].Using information obtained mapping lethal mutations relative m l 1 8 , chromosome carrying <br /> the rightward lethal mutation mx (e.g., m329) marked unc-5, chromosome carrying leftward lethal mutation, my <br /> (e.g., m328), marked unc-17. (a) If order lethal mutations predicted mapping relative m118, then <br /> recombination lethal mutations resulted chromosome           + d p y - I 3 ama-1(m1l8) +. Recombinants carrying this <br /> chromosome resistant amanitin Dpy. (b) If order lethal mutations reversed, recombination between <br /> the resulted recombinant chromosome unc-17 dpy-13 ama-l(m118) unc-5. Resistant recombinants carrying this <br /> chromosome Dpy Unc. <br />  <br /> linked unc-17, recombinant chromosome                alleles positioned relative other. Both <br /> restored amanitin    resistance        carried dpy-Z3 ama-           strategiesutilized selection recombinants by <br /> l(m118). Resistant recombinants Dpy, or <br /> semi-Dpy carried mDpl, Dpys                  means amanitin resistance. In thefirst strategy, the <br /> the semi-Dpys segregate Dpy Uncs (Figure 2a). If               ama-I/+ parental animals unableto make func- <br /> order lethal alleles reversedfrom predicted              tional amanitin-resistant RNA polymerase 11, to <br /> earlier mapping experiments, recombinant chro-                  presence lethal mutation resistance <br /> mosome carry unc-17 unc-5 dpy-                 gene. The functional large subunit ani- <br /> 13 m - l ( m l l 8 ) shown Figure 2b. Recombinant <br /> animals Dpy Unc, semi-Dpy, semi-Dpy Unc                   mals product wild-type (sensitive) allele <br /> phenotype, semi-Dpys semi-Dpy Uncs                  ama-I. By selecting amanitin-resistant worms, <br /> would segregate Dpy Uncs. For crosses,the calculated               recombinationeventsbetweenthelethalmutations <br /> recombination frequencies based resistant Dpy                andthe resistancemutation, m118, weredetected <br /> (or Dpy Unc order mx - my) sensitive parental             (Figure 1). In second strategy, selected pairs       of <br /> classes. Their respective frequenciesamong total zygotes <br /> were predicted 54 p( 1 - p ) 54 (1 - p)&apos;. We defined           lethal mutations placed trans, strain via- <br />                               + <br /> R ratio, Dpy / Dpy parental. Wherep&apos; negligible,               bility conveyed wild-type, ama-l(+), allele <br /> R = p shown above. The predicted frequencies assume                   carried free duplication, mDpZ[unc-l7(e113)] <br /> equal viabilities ofthe recombinants mDp1               (ROGALSKIand RIDDLE1988). Briefly, mDpl dupli- <br /> the parental types carry mDpl.                                   cates left arm chromosome ZV to, not <br />                                                                          including unc-8, carries       ama-I(+). Although the <br />                            RESULTS                                       heteroallelic strains sensitive amanitin, both <br />                                                                          lethal alleles harbored resistance mutation, m118. <br />    Mapping strategies:Two strategieswere employed                        Recombination lethal sites restored the <br /> to construct fine-structure map   ama-I. First,                   dominant m - I ( m Z l 8 ) allele, recombinants <br /> lethalalleles werepositionedrelativeto      m118, n                    specifically selected background aman- <br /> amanitin-resistance mutation. Second, selected lethal                    itin-sensitive animals (Figure 2). <br />  428                                         A. M. E. Bullerjahn <br />                                                                 D. L. Riddle <br />  <br />    Mapping lethals relative amanitin-resistance:               Dpy recombinants Unc recombinants (Table 2). <br /> The 16 lethal ama-1 alleles positioned relative m l 1 8        This suggests Dpy semi-Dpy recombinants <br /> are listed Table 1 , adapted ROGALSKI,BUL-                grew better liquid cultures did Unc (un- <br /> LERJAHN RIDDLE(1988). Strains genotype              +      coordinated)recombinants,      swim as <br /> dpy-13 ama-l(mll8mx) +/unc-17         ++unc-5, mx           well. Data thatsupported interpretation came <br /> represents lethal mutation, constructed              mapping m221, lethal mutation linked <br /> these experiments. These strains semi-Dpy                unc-8. Semi-Unc worms better do <br /> to thesemidominant nature dpy-l3(e184),               Unc worms, expected survive better <br /> were sensitive amanitin. The animals segregated                liquid culture. There essentially equal num- <br /> the unc-17unc-5 doublemutant, grew                     bers semi-Dpy semi-Unc resistant recombinants <br /> slowly. The phenotype genotype resistant                                                + <br />                                                                   issuing parentsof genotype ama-l(mI18m221) <br /> recombinants indicated lethal mutation                unc-8 / dpy-I3   ++ m221 mapping experiment <br /> was right     left m118 (Figure 1).           (Table 2). Based onthe assumption thatthe Unc <br /> Genotypes recombinants confirmed prog-                 recombinants mappingexperimentswere <br /> eny testing complementation tests. A different             under-represented did grow well, <br /> strain used mapping m221 allele             map distances (except form 3 2 2 ) calculated based <br /> is linked semi-dominant unc-8 marker,              solely numbers semi-Dpy Dpy recombinants, <br /> to dpy-13 (see MATERIALS AND METHODS). Another                    using formula derived MATERIALS AND METH- <br /> exception involved lethal mutation, m335,               ODS. <br /> is linked amanitin resistance mutation m322,                  In cross determined distance between <br /> not m l l 8 (ROGALSKI, BULLERJAHN     RIDDLE1988).            m252 m118, total 40 semi-Dpy <br /> Thus, m335 mapped relative m322                recombinants (Table 2). Thirty-five distributed <br /> manner lethal mutations mapped                  populations, popu- <br /> relative m118, position determined             lations contained zero recombi- <br /> with respect lethal mutations.                       nantseach. In case, assumed thatthe <br />    Several mutants grew sterile adults 20&quot;,             amanitin-resistant animals anomalous pop- <br /> they arrested development early mid-larval              ulations siblings arisingfromrecombination <br /> stages grown 25&quot; (Table 1). Although                  events occurred earlier generation. Thus, <br /> selection foramanitin-resistantrecombinants                   m252 cross, anomalouspopulations <br /> done 25&quot; cases, larvae hatched               scored containingonly recombinant each, <br /> eggs produced 20 . Hence, temper- <br />                        O                                          totalof seven semi-Dpy recombinants distributed <br /> ature differences experiments             independent populations exper- <br /> might affect comparison recombination frequencies              iment. <br /> (ROSE BAILLIE 1979). Nevertheless, acontrol                      In mapping experiment, amanitin-resist- <br /> experiment determineif selection 25&quot;               ant animals recovered unexpected pheno- <br /> alteredthefrequency recombinants                    types flanking markers. This result of <br /> recovered. T o end, m329 mapped relative              eithergene     conversion doublerecombination <br /> m l I8 using selections performed both20 &quot; 25 &quot; .          events. Although data cannotformally distinguish <br /> Essentially results obtained                two, believe excep- <br /> experiments (Table 2).                                            tional animals convertants based number <br />    The results 18 mapping experiments given                animals relative number recombi- <br /> in Table 2. The number m - l ( m l l 8 m x ) / + heter-    nants. For lethal mutations relatively far away from <br /> ozygotes experiment ranged 148,000                m118, m328 m329, more <br /> 843,000, mean 542,000 experiment. A                 recombinantsthan apparent convertants (Table 2). <br /> total 9.6 million heterozygotes subjected              For mutations closer m l l 8 , proportion con- <br /> selection amanitin, 229 recombinants                  vertants relative recombinants increased (e.g., m365 <br /> used todeterminebothrecombinationfrequencies                      m332). Finally, numberof recombinants <br /> and mutation order. Table 2 gives distance               reached zero, case m235, a <br /> each 16 different lethal mutations m l l 8             marked decrease number convertants. Such <br /> 90% confidence intervals (STEVENS 1942),                      relationship recombinants convertants <br /> whether lethal mutation left right            observed fine-structure mapping <br /> m l l 8 . By combining distances leftward-           experiments C . elegans (MOERMAN BAILLIE <br /> most andthe rightward-most lethal mutations                     1979; WATERSTON, SMITHand MOERMAN1982), as <br /> m l l 8 , determined gene definedby          organisms (FOGEL MORTIMER <br /> mutant set 0.01 1 f 0.003 map unit long.                        1969; CHOVNICK, BALLANTYNE        HOLM197 1). <br />    In case moreDpy semi-                 In cases, recombinant genotypes best <br />                                                                           TABLE 2 <br />                                                     Mapping lethal alleles relative ama-l(mll8) <br />  <br />                                                                                                                                 Convertants dou- <br />          Allele         Heterozygotes                   Mau units           Recombinants <br />                                                                                  Left/Rieht                                      ble recombinants <br />         m252               543,000             0.0013 (0.0006-0.002)                  L                7 Semi-Dpy&quot;               4 Wild Type&apos; <br />                                                                                                         4 Unc- 17 <br />         m398               460,000             0.0026 (0.002-0.004)                   L                12 Semi-Dpy               2 DPY <br />                                                                                                         2 Unc-I7                 1 Unc-17 <br />                                                                                                                                 21 Wild Type <br />                                                                                                                                  3 Semi-Dpy <br />         m370               57 1,000            0.003 (0.002-0.005)                    L                19 Semi-Dpy               3 Wild Type <br />                                                                                                                                  3 DPY <br />                                                                                                                                  1 Semi-Dpy Unc <br />                                                                                                                                 11 Semi-Dpy <br />         m365               8 19,000             0.0006 (0.0002-0.001)                 L                 5 Semi-Dpy               3 DPY <br />                                                                                                                                  3 Wild Type <br />                                                                                                                                  8 Semi-Dpy <br />         ,397               438,000              0.002 (0.001-0.004)                   L                 9 Semi-Dpy              20 Semi-Dpy&apos; <br />         m235               365,000              0 (0-0.0002)                                            0                        &apos;   DPY <br />         m368               843,000             0 (0-0.0001)                                             0                        4 DPY <br />         m396               35 1,000             0 (0-0.0002)                                            0                        3 Semi-Dpy <br />                                                                                                                                  2 DPY <br />                                                                                                                                  1 Dpy Unc <br />         m236               488,000              0.0002 (0.0001-0.001)                 R                 1 DPY                    1 DPY <br />                                                                                                                                  2 Semi-Dpy <br />         m332               613,000              0.0003 (0.0001-0.001)                 R                 2 Unc-5                  4 DPY <br />                                                                                                                                  6 Semi-Dpy <br />         m364               806,000              0.0012 (0.0007-0.002)                 R                10 DPY                    3 Semi-Dpyd <br />                                                                                                         1 Unc-5                  1 Wild Type <br />                                                                                                                                  1 Dpy Unc <br />         m221               732,000              0.002 (0.001-0.003)                   R                12 Semi-Unc               2 DPY <br />                                                                                                        11 Semi-Dpy               2 Unc-28 <br />          m367              382,000              0.003 (0.002-0.005)                   R                12 DPY                    1 Semi-Dpy <br />                                                                                                         1 Unc-5                  1 Dpy Unc <br />                                                                                                                                  9 Wild Type <br />          m369              328,000              0.005 (0.003-0.007)                   R                15 DPY                    2 Semi-Dpy <br />         m328               148,000              0.006 (0.003-0.01)                    R                 9 DPY                    1 WildType <br />                                                                                                         2 Unc-5 <br />         m32F               647,000              0.007 (0.005-0.009)                   R                44 DPY                   12 Semi-Dpy <br />                                                                                                        18 Unc-5                 17 Wild Type <br />                                                                                                                                  1 Dpy Unc <br />          m329&apos;             511.000              0.006 (0.005-0.008)                   R                33 DPY                    4 Semi-Dpy <br />                                                                                                                                  2 Semi-Dpy Unc <br />                                                                                                                                  1 Dpy Unc <br />          m3358             383,000                                                                      0                        2 DPY <br />                                                                                                                                     _. <br />     As described text, 40 semi-Dpy recombinants actually recovered, distribution independent populations <br /> indicated tworecombination events occurred earlier generation. Ofthe 40semi-Dpy recombinants, 35 distributed <br /> among populations. These populations counted containing recombinant each, recombinant animals <br /> (distributed independent populations) scored independent events. <br />       Wild-type animals genotype      ++                    +          +++ <br />                                                        m - l ( m l l 8 ) / unc-17               unc-5, recombinant chromosome arisen a <br /> crossover interval unc-17 dpy-13, inteval lethal m118 sites ama-1. This true for <br /> semi-Dpy animals genotype      ++                  + + <br />                                       m - l ( m l l 8 ) / dfy-13 ama-l(mll8mx)     +. <br />     &apos; Of 20 semi-Dpy animals, 14 genotype           ++         m - l ( m l l 8 ) + / + dpy-13 ama-l(m397mll8)+ genotype <br />         +                         +                                 +, <br /> unc-17 m - l ( m l l 8 ) unc-5 / dpy-13 ama-l(m397m118) indicating recombination conversion ama-1 accompanied by <br /> recombination unc-17 dpy-13. <br />       The distribution resistant animals independent populations suggested conversion recombination events occurred an <br /> earlier generation. Although 45 semi-Dpy, Wild, Dpy Unc animals actually recovered, 43 semi-Dpy convertants all <br /> of wild-type Dpy Unc animals population. This anomalous population scored having event type. <br />     e Selection performed 20&quot;. <br />  <br />    &apos;Selection performed 25&quot;. <br />       The m335 mutation linked amanitin-resistance mutation, m322, positioned relative sites within <br /> the gene. <br />  <br />                                                                              429 <br />  430                                                                   A. M. E. Bullerjahn D. L. Riddle <br />                                                                                          TABLE 3 <br />                                                              Mapping lethal ~ m - alleles <br />                                                                                     1       relative other <br />  <br />                                             mDpl [unc-                  mDpl [unc- <br />                                            171; + dpy-13                171; unc-17 <br />                                                 ama-1                   dpy-13 ama- <br />          Allele pair&quot;      Total Fa          ( m l 1 8l (mmxl)l 8 m y )           +&apos;   Heteroallelic                                                 Distance between <br />           (my/m*)           worms              ~nc-5~                       (%)         Recombinants <br />                                                                                         Fa wormsd                   Convertants&apos;       L-R&apos;units)(mapalleles <br />                           1,328,000        15/60 = 0.25          657,000 (49)           503,000         3 Dpy Unc        24        m252-m398            0.0006 <br />                           1,026,000        24/60 = 0.40          284,000 (28)           445,000         1 Dpy Unc        18        rn398-m370           0.0002 <br />                           1,739,000        22/60 = 0.37          534,000 (31)           763,000         0                 1                            &#226;&#8218;&#172;0.0001 <br />                           1,166,000        17/55 = 0.31          159,000 (14)           695,000         9 DPYB           9         m252-m365            0.001 <br />                           1,077,000        21/60 = 0.35          333,000 (31)           484,000        13 DPY            7         m364-m367            0.003 <br />                           1,474,000        24/60 = 0.40          560,000 (38)           548,000         4 DPY             8        m367-m328            0.0007 <br />                           1,190,000        26/60 = 0.43          393,000 (33)           454,000         0                 2                            &#226;&#8218;&#172;0.0002 <br />                           1,385,000        18/52 = 0.35          424,000 (31)           625,000        19 DPY            14*       m328-m329            0.003 <br />  ~               ~~   ~     ~         ~~                             ~~                      ~~~ <br />  <br />  <br />  <br />  <br />      The genotypes heteroallelic animals mDpl[unc-17]; unc-17 dpy-13 <br />                                                                                                                    + <br />                                                                               ama-I(mll8my) / + dpy-13ama-l(mll8mx)unc-5. <br />      The fraction frequency semi-Dpy Pohermaphrodites homoallelic dpy-13ama-l(ml18mx)unc-5chromosome, identifiedby <br /> their failure segregate Unc-17 progeny. <br />    &apos; T h e total number percentage total population animals homoallelic unc-17 dpy-13 <br />                                                                                                           ama-l(mll8my)chromosome, as <br /> determined scoring phenotypes animals diluted samples. These animals semi-Dpy Unc phenotype. <br />      (Total worms-semi-Dpy Unc worms) X (1 - frequency dpy-13ama-l(ml18mx)unc-5homoallelic worms semi-Dpys). <br />      Convertant phenotypes: semi-Dpys segregated Dpy Uncs, Dpys segregated Dpy Uncs, semi-Dpy Uncs, Dpy Uncs. <br />    f Left-right order mutations. <br />  <br />    8 19 Dpy recombinants actually recovered, 1 1 population considered arisen single <br />  <br /> event earlier generation. <br />      &apos; <br />      28 convertants actually recovered, 15 Dpy convertants population considered arisen from <br /> a single event earlier generation, andwere counted one. <br />  <br /> explained recombination outside markers                                                 selection true-breedingstocks (see MA- <br /> occurred addition recombination conversion                                                TERIALS AND METHODS). T h e percentage pop <br /> within ama-I. For example, resistant animals                                              ulation homozygous unc-5 chromosome (usu- <br /> with wild-type movement body shape                                                   ally 35-40%) calculated based per- <br /> selected m252 mapping experiment (Table 2)                                                  centage P,s belonged class. (Such P,s <br /> carried  the <br />            recombinant <br />                      chromosome,                    ama-                     ++                    did produce Unc-17 progeny dis- <br />                   +, <br /> I ( m l J 8 ) generated                                                carded.) T h e percentage population homozy- <br /> events, recombination 3.0-mapunit                                                   gous unc-17 marked chromosome deter- <br /> interval unc-17 dpy-13, recombina-                                                 mined countingthe semi-Dpy Uncs diluted <br /> tion m252 m l l 8 . These events need                                              samples populationprior treatment with <br /> not occurred generation,                                                amanitin. <br />                                                                                                           Heterozygous          animals genotype <br /> selection amanitin resistance applied FB                                            mDpI[unc-I 71;unc-I 7 dpy-13 ama-l(ml18my)+/+ dpy- <br /> progeny constructedheterozygotes,          de-                                           13 ama-l(mll8mx)unc-5 expected segregate <br /> scribed MATERIALS AND METHODS.                                                                  homozygous animals genotypes mDpl [unc-I71;unc- <br />     Mapping lethals relative   other:The second                                                                            + <br />                                                                                                    17 dpy-13 ama-l(ml18my) mDpl[unc-17]; dpy-              + <br /> mapping strategy involved positioning lethal alleles                                               I3 ama-l(mll8mx) unc-5 roughly equal numbers. <br /> relative nearest neighbors. Strains con-                                             Usually, percentage population homozy- <br /> structed lethal mutation (mx)was linked                                               gous unc-I7 marked chromosome slightly <br /> to dpy-I3 unc-5, lethal mutation                                                 smaller percentage population homo- <br /> (my) linked unc-17 dpy-13 (Figure 2). The                                               zygous unc-5 marked chromosome, the <br /> heteroallelic strains genotype mDpI[unc-I 71;                                         +         difference rarelylarge(Table    3). In cases <br /> dpy-13 ama-l(ml18mx)     unc-5/unc-l7 dpy-I3        ama-                                           involving m252, large(two-fold) difference <br /> I(ml18my) + hada semi-Dpy non-Uncphenotype.                                                        number unc-5 homozygotes unc- <br /> Recombinants selected means resist-                                               17 homozygotes. This presumably result of <br /> ance amanitin.                                                                                  slowed development mop1 strains carry two <br />     The left-right order neighboring lethal mutations                                           doses ama-l(ml18m252) allele. Generation time <br /> was determined, distance                                                  (egg egg) slowed 30% +/m118m252/ <br /> estimated. It necessary determine number                                                ml18m252 strains comparison +/+/ml18m252 <br /> of heteroallelic worms experiment order                                              strains (ROGALSKI, BULLERJAHN RIDDLE 1988). <br /> estimate recombination frequencies (Table       3),                                             Infour cases, neighboring lethal mutations were <br /> the hermaphrodites used establish populations                                               separated, order deduced    as <br />                                                Fine-Structure ama-1 Map                                                  43 I <br />  <br />  <br />                               ,m332 <br />  <br />  <br />   0 <br />            m397 <br />                        e <br />                                m236- <br />  <br />                               fl <br />                                 396 <br />                               f1360 <br />                                        -                                                   FIGURE 3.-Fine-structure      genetic <br />                                                                                            map ama-1. Mutations have <br />                                                                                            positioned relative their <br />                               fla35                                fl*g                    nearest neighbors placed the <br />    fl f139836                                                                              line, allele numbers stacked ver- <br />            $83                r&lt;\\I0                        f136&apos; ($25                     tically mutations <br />            I                  1        I <br />                                                             I      1            1          sepa- <br />                                                                                            rated. Alleles separated <br />                                                                                            m l l 8 been <br />  <br />       }.00I t &quot;    -003+- <br />                         .oo12+&quot;                .m.-~.0007+                .m__1            positioned respect near- <br />                                                                                            est neighbors placed the <br />                                                                                            line 90% confidence in- <br />  <br />                                +-a                 +                                       tervals (U           ). <br />                                                                                                               Below map are <br />  <br />  <br />    7\ <br /> .om6       .oO02               &quot;007- <br />                                                                                            given distances various <br />                                                                                            mutations (not drawn scale) aswell <br />                                                                                            distances certain mu- <br />                                                                                            tations m118. <br />  <br />  <br />  <br />  <br /> that predicted mapping lethal sites relative            Mapping lethal alleles region relative each <br /> m l l 8 (Table 3). In cases, m369/m328 m370/            determined    order correct. T h e <br /> m365, mutations separated,              m328 mutation given position m369 <br /> cases, m370/m398 m398/m252, order mu-                alleles separated, and <br /> tations probable predicted dis-             position agreed data mapping <br /> tance m118. T h e 90% confidence intervals             m328 relative m329. T h e m367 mutation posi- <br /> the data obtained mapping mutations               tioned 0.004 map unit right m l l 8 because <br /> relative m118 overlap (Table 2), surpris-       position agreed data mapping <br /> ing differences.                                   m367 relative m364 m328, the <br />    Drawing map: The fine-structure map ama-              90% confidenceintervaldetermined          mapping <br /> 1 shown Figure 3. In general, comparisonof map            m367 relative m118. T o left m118, m370 <br /> distances Tables 2 3 goodadditivity                 mutation placed onthemap          according its <br /> throughout map. The m118 mutation located                distancefrom m118 (Table 2). Since order of <br /> &apos;/s way left end gene. Three          m370 m398 differed predicted map- <br /> the adult-sterile mutations couldnot separated          ping mutations relative m118, m398 posi- <br /> from m118. T h e adult-sterilemutations,              tioned map based recombination frequency <br /> m236 m369, maptotheright            m118. T h e          m370. Since order m398 m252 also <br /> embryonic lethal mutation, m252, leftward-most           differed predicted mapping muta- <br /> mutation,and       early larval lethal mutationsmap             tions relative m118, m252was positioned according <br /> throughout gene.                                            recombination data relative m398. T h e m365 <br />    All data Tables 2 3 reconciled            mutation given position m370 because <br /> obtain themost consistent position lethal sites         alleles separated, posi- <br /> with respect respect m118             tion agreed wellwith data mapping m365 <br />                                                                 relative m252. <br /> (Figure 3). In general, recombination frequencies ob- <br /> tained mapping mutations relative    m l 1 8 were <br />                                                                                     DISCUSSION <br /> considered reliable. Estimation number <br /> of heteroallelic worms crosses designed position             T h e ama-1 fine-structure map linear, addi- <br /> two lethal mutations relative some-           tive, based dataobtainedfrommapping           lethal <br /> what inaccurate number homoallelic               alleles relative m118, data obtained from <br /> progeny segregated generations              mapping lethal alleles relative other. In most <br /> growth prior selection recombinants couldonly            cases, recombination lethal mutations <br /> beroughlyestimated.       T o theright m118,             detected, order lethal mutations re- <br /> mutations m364, m369 m329 placed                spect inferred by <br /> the mapaccording distances m l 1 8 .              mapping lethal alleles relative m118. A total 20 <br />  432                                    A. M. E. Bullerjahn <br />                                                             D. L. Riddle <br />  <br />  million worms screened recombinantsin          bination smaller intervals. WATERSTON,          SMITH <br />  26 crosses used generate fine-structure map.         MOERMAN(1982)observed recombination <br />  An average 11 recombinants convertants           event unc-54 mutations, s291 e1328, <br />  detected cross. The resolution genetic       later shown 26 bp apart (DIBBet al. <br /> map sufficient tell mutations        1985). Intragenic recombination frequencies         not <br /> are small rearrangements deficiencies, prelim-        determined study. <br /> inary Southern blot analyses suggest           The early larval lethal mutations map throughout <br /> mutations mapped duplications deficien-          gene. Arrested development death L1 <br /> cies larger 50 bp (L. NAEGER, D. BIRDand D.             stage ama-1 null phenotype. Evidence this <br />  RIDDLE,unpublished data).                                   comes m D f l 0 deficiency, conveys an <br />    In addition recombinants, resistant animals        L1-lethal phenotype deficiency homozygotes (Ro- <br /> were recovered unexpectedcombinations                GALSKI RIDDLE1988). Five mutations clus- <br /> flanking markers. These exceptionalprogeny               tered veryclose tothe resistance mutation, m118, <br /> likely arisenfromgene         conversion events,     maps approximately &apos;/s way the <br /> based ontheirfrequency.          FOGELand MORTIMER           gene dpy-I3 (left) end. Molecular analysis <br /> (1969) studied gene conversion arg4 locus          shown dpy-I3 end 5&apos; end the <br /> yeast andfoundthat         distance       gene (D. BIRDand D. RIDDLE,unpublished data). This <br /> alleles decreased, number convertants relative        position resistance mutation agrees with <br /> to recombinants increased. CHOVNICK,       BALLANTYNE        position resistance mutation mouse <br /> and HOLM(197 1) observedthe phenomenon               RNA polymerase I1 large subunit gene (BARTOLOMEI <br /> their study rosy locus Drosophila. Further-        CORDEN1987). <br /> more, distance alleles              The m252 embryonic lethal mutation po- <br /> extremely small, recombinants con-           sitioned left end map. The data suggest <br /> vertants recovered. Our results similar.            allele encodes product interferes with <br />    The general additivity map observed        normal enzyme function (ROGALSKI,           BULLERJAHN    and <br /> results crosses utilized mDpI[unc-I 71 com-      RIDDLE1988). T h e m l 1 8 m 2 5 2 allele unique that <br /> pared results crosses did involve       conveys anembryonic lethal phenotype at20&#194;&#176;, <br /> the duplication. This indicates duplication         null phenotype ofama-I L1-lethal. The <br /> itself does significantly affect recombination fre-      m l 1 8 m 2 5 2 product apparently interfereswith ma- <br /> quencies ama-I. It noted, however,          ternally encoded wild-type RNA polymerase I1 that <br /> that m o p 1 recombines normal chromosome           sufficient completion em- <br /> IV frequency 20-fold reduced            bryonic development. Furthermore, mDp I [ama- <br /> observed normal homologs (ROGALSKI               I(+)] strain genotype m 2 5 2 / m 2 5 2 / + slower <br /> and RIDDLE1988). No recombinants detected            developing 20 O m252/+ m 2 5 2 / + / + strains, <br /> study readily attributed torecombination         suggesting predominance m252 product <br /> gene conversion events involving m D p l .                   antagonizes function wild-type product. Such <br />    The ama-1 gene cloned C. elegans,           interference reduced 25&quot;,        indicating the <br /> and 8.7 kb long (D. BIRDand D. RIDDLE,           abnormalfunction thermolabile. If does <br /> unpublished data). Given genetic size        encode altered, deleterious product, mu- <br /> gene 0.01 1 map unit, estimate DNA            tation islikely theproteincoding se- <br /> map unit ratio 790 kb/map unit, assuming            quence gene. Therefore,              lethal <br /> thatthe map spans theentiregene.           T h e average     mutations affect sequences upstream         coding <br /> amount DNA map unit C. elegans 266 kb,          region. <br /> but amountincreases 900 kb/map unitin               The m335 sterile mutation allelic other <br /> the LGIII gene cluster decrease recombi-         ama-1 lethals. It linked m322 resistance allele <br /> nation frequency cluster (GREENWALD         et    ama-1 m l l 8 . The m335 mutation could <br /> al. 1987). The ama-1 gene edgeof LGIV          notbeseparatedfrom           m322, just sterile <br /> genecluster (EDGLEYand RIDDLE1987),                 mutations separated m l I 8 . These <br /> consistent higher average kb/map unit            mapping data support theconclusion the <br /> value.                                                       dominant amanitin-resistance mutations, m 1I8 and <br />    The specific, positive selection recombinants       m322, allelic (ROGALSKI,           BULLERJAHN <br /> means amanitin-resistance allowed detection re-        RIDDLE1988). It possible m l l 8 m322 <br /> combination interval small 0.0002 map            mutations site close. <br /> unit (betweenm370 m398, m l 1 8             T h e amanitin binding site RNA polymerase I1 is <br /> m 2 3 6 ) , corresponds 170 bp. Screening     highly conserved divergent species (ROEDER <br /> larger populations allow detection recom-          1976). In themouse, amanitin-resistance mutation <br />                                                        Fine-Structure ama-1 Map                                                    433 <br />  <br /> is region protein strongly homolo-                                   LITERATURE CITED <br /> gous yeast protein, adjacent region <br />                                                                      AHEARN,  J. M., M. S. BARTOLOMEI,  M. L. WEST,L. S. CISEKand J. <br /> that highly conserved E. coli (BARTOLOMEI                   L. CORDEN,    1987 Cloning sequence analysis mouse <br /> CORDEN1987; AHEARNet al. 1987). Hence, struc-                       genomic locus encoding largest subunit RNA polymerase <br /> ture ofthisregion       thoughttobe     essential                11. J. Biol. Chem. 262: 10695-10705. <br /> polymerase function. It interesting outof               ALLISON,L., M. MOYLE, M. SHALESand C. J. INGLES, <br /> five sterile mutations map close m118. Aman-                    1985 Extensive homology largest subunits eu- <br />                                                                         karyotic prokaryotic RNA polymerases. Cell 42: 599-610. <br /> itin interferes elongation reaction tran-                AMATI,P., F. BLASI,U.DI PORZIO,A. RICCIOand C. TRABONI, <br /> scription (COCHET-MEILHAC       CHAMBON1974),                       1975 Hamster a-amanitine-resistant RNA polymerase I1 able <br /> and RNA polymerase I1 isolated amanitin-resist-                    transcribe polyoma virus genome somatic cell hybrids. <br /> ant Drosophila reduced elongation ratein vitro                   Proc. Natl. Acad. Sci. USA 72: 753-757. <br /> (COULTERand GREENLEAF          1985). Other mutations                BARTOLOMEI,    M. S., J. L. CORDEN,1987 Localization an <br />                                                                         a-amanitin resistance mutation gene encoding largest <br /> that slow elongation result RNA polym-                      subunit mouse RNA polymerase 11. Mol. Cell. Biol. 7: 586- <br /> erase I1 satisfy demand forhigh tran-                    594. <br /> scription levels particular tissues particular              BRENNER,  S., 1974 The genetics Caenorhabditiselegans. Ge- <br /> times development. Such mutations ex-                       netics 77: 71-94. <br /> pected result slow growth, inability              CHAN, V. L., F.G.           WHITMOREand L. SIMINOVITCH, <br />                                                                         1972 Mammalian cells withaltered forms RNA polymerase <br /> synthesize components necessary oo-                      11. Proc. Natl. Acad. Sci. USA 6 9 31 19-3123. <br /> genesis. This describes phenotype sterile                 CHOVNICK,  A., G. BALLANTYNE    D. HOLM,1971 Studies on <br /> mutantsthatmap        close m118,                   gene conversion relationship linked exchange in <br /> possible affect RNA chain elongation. Bio-                    Drosophila melanogaster. Genetics 6 9 179-209. <br /> chemical study mutants aimed obtain-                COCHET-MEILHAC,    M., P. CHAMBON,     1974 AnimalDNA- <br />                                                                         dependent RNA polymerases. 11. Mechanism inhibition <br /> ing information protein structure      activity               RNA polymerase B amatoxins. Biochem. Biophys. Acta <br /> vitro, complement genetic studies                       353: 160-184. <br /> reported here(M. GOLOMB,      personal communication).               COULTER,   D. E., A. L. GREENLEAF,    1985 A mutation the <br />    This study describesthe mappingof lethal alleles                  largest subunit RNA polymerase I1 alters RNA chain elon- <br /> an essential gene, demonstrates feasibility                  gation vitro. J. Biol. Chem. 2 6 0 13190-13198. <br />                                                                      DIBB,N. J., D. M. BROWN,    J. KARN,D. G. MOERMAN,    S. L. BOLTEN <br /> using duplication construct viable heteroallelic                   R. H. WATERSTON,       1985 Sequence analysis mutations <br /> lethal strains organism. T h e ability detect                affect synthesis, assembly, enzymatic activity of <br /> rare amanitin-resistantrecombinants intervals                      unc-54 myosin heavy chain Caenorhabditis elegans.J. Mol. <br /> estimated small 170 nucleotides provides                    Biol. 183: 543-551. <br /> the basis high-resolution map. Mutations                  EDGLEY, M., D.L. RIDDLE,1987 Caenorhabditiselegans. pp. <br />                                                                         351-365. In: Genetic Maps, Vol. 4, Edited S. J. O&#226;&#8364;&#8482;BRIEN. <br /> affect RNA polymerase I1 large subunitare useful <br />                                                                         Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br /> because identify regions polypeptide                FERGUSON,   E. L., H. R. HORVITZ,1985 Identification and <br /> are important forvarious aspects enzyme structure                    characterization 22 genes affect vulval cell lineages <br /> and function. T h e fine-structure map     ama-1                 nematode Caenorhabditis elegans. Genetics 1 1 0 17-72. <br /> gene provides essential information de-                 FOGEL,S., R. K. MORTIMER, 1969 Informational transfer in <br />                                                                         meiotic gene conversion. Proc. Natl. Acad. Sci. USA 62: 96- <br /> ducing function different domains the <br />                                                                          103. <br /> resistant subunit, guiding sequence anal-               GREENLEAF, A. L., 1983 Amanitin-resistant RNA polymerase I1 <br /> yses mutantalleles. Finding point mutations                   mutations enzyme&#226;&#8364;&#8482;s largest subunit. J. Biol. Chem. <br /> an 8.7-kb gene difficult sort                      258: 13403-13406. <br /> of guide,andthefine-structuremap           provides              GREENLEAF,  A. L., L. BORSETT,P. JIAMACHELLO D. COULTER, <br />                                                                         1979 a-Amanitin-resistant D. melanogaster altered <br /> needed link genetic molecular analyses. <br />                                                                         RNA polymerase 11. Cell 18: 613-622. <br /> An advantage mapping      physical methods              GREENLEAF,  A. L., J. R. WEEKS,R. A. VOELKER,   S. OHNISHIand <br /> that identify <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span>-pair changes DNA (MYERS, LARIN                    B. DICKSON,   1980 Genetic biochemical characterization <br /> and MANIATIS1985) thefunctionally important                     mutants RNA polymerase I1 locus D. melanogaster. <br /> alterations distinguishedfrom        possible                 Cell 21: 785-792. <br /> changes.                                                             GREENWALD,   A. L., A. COULSON,     J. SULSTONand J. PRIESS, <br />                                                                         1987 Correlation physical genetic maps lin- <br />                                                                         12 region Caenorhabditiselegans. NucleicAcidsRes.     15: <br />    This work supported Department Health Human             2295-2307. <br /> Services grant GM36674; A.B. predoctoral trainee supported     HIMMELFARB,     H. J., E. M. SIMPSON      J. D. FRIESEN, <br /> by GM07494. Some strains obtained Caenorhabditis          1987 Isolation characterization temperature-sensitive <br /> Genetics Center, supported contract NO1 RR-4-2111           RNA polymerase I1 mutants Saccharomyces cerevisiae. Mol. <br /> with National Institutes Health Division Research Re-         Cell. Biol. 7: 2155-2164. <br /> sources. We thank T . ROGALSKI,  G. SMITH,M. GOLOMB,    K. NEW-      HORVITZ,H. R., S. BRENNER,     J. HODCKINand R. K. HERMAN, <br /> TON E. COE helpful discussions, R. ROSENBLUTH critical       1979 A uniform genetic nomenclaturefor nematode <br /> comments manuscript, D. BIRDfor sharing unpublished              Caenorhabditis elegans. Mol. Gen. Genet. 175: 129-133. <br /> results, M. AUDSLEYfor processing manuscript.                INGLES, C. J., 1978 Temperature-sensitive RNA polymerase I1 <br />  434                                            A. M .D. <br />                                                       E. Bullerjahn <br />                                                                         L. Riddle <br />    mutations Chinese hamster ovary cells. Proc. Natl. Acad.        ROSE,A. M., D. L. BAILLIE,1980 Genetic organization the <br />    Sci. USA 75: 405-409.                                                 region unc-Z5(1), gene affecting paramyosin Cae- <br /> MOERMAN,   D. G., D. L. BAILLIE,1979 Genetic organization            norhabditis elegans. Genetics 96 639-648. <br />    Caenorhabditis elegans: fine-structure analysis unc-22   SANFORD,   T., M. GOLOMBand D. L. RIDDLE,1983 RNA polym- <br />    gene. Genetics 91: 95-103.                                            erase I1 wild typeanda-amanitin-resistantstrains        of <br /> MORTIN,M. A,, T. C. KAUFMAN,      1984 Developmental effects         Caenorhabditis elegans. J. Biol. Chem. 2 5 8 12804-12809. <br />    atemperature-sensitiveRNApolymerase           I1 mutationin     SANFORD,   T.,J. PRENCER   M. GOLOMB,1985 Purification and <br />    Drosophila melanogaster. Dev. Biol. 103: 343-354.                     immunological analysis RNA polymerase11 Caenorhab- <br /> MYERS,R.M., Z. LARINand T. MANIATIS,1985 Detectionof                     ditis elegans. J. Biol. Chem. 260: 8064-8069. <br />    single base substitutionsby ribonuclease cleavage mismatches    SENTENAC,A,, 1985 EukaryoticRNApolymerases.CRCCrit. <br />    RNA:DNA duplexes. Science 2 3 0 1242-1246.                         Rev. Biochem. 18: 3 1-9 1 . <br /> ROEDER,R.G., 1976 EukaryoticnuclearRNApolymerases.pp.                 SOMERS,D. G., M. L.PEARSONand C. J. INCLES, 1975 Isolation <br />    285-329. In: RNA Polymerase, Edited R. LOSICKand M.                characterization a-amanitin-resistant rat myoblast <br />    CHAMBERLIN. Cold Spring Harb <br /> </body></html>