<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A genome-scale analysis identification of <br /> genes required growth <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> of <br /> Haemophilus influenzae <br /> Brian J. Akerley*, Eric J. Rubin&#226;&#8364;&#160;, Veronica L. Novick&#226;&#8364;&#161;, Kensey Amaya*, Nicholas Judson&#226;&#8364;&#161;, John J. Mekalanos&#226;&#8364;&#161;&#194;&#167; <br /> *Department Microbiology Immunology, University Michigan Medical School, Ann Arbor, MI 48109; &#226;&#8364;&#160;Department Immunology Infectious <br /> Diseases, Harvard School Public Health, Boston, MA 02115; &#226;&#8364;&#161;Department Microbiology Molecular Genetics, Harvard Medical School, <br /> Boston, MA 02115 <br />  <br /> Contributed John J. Mekalanos, November 9, 2001 <br />  <br /> A high-density transposon mutagenesis strategy applied                 We described approach (9) detecting essential <br /> the Haemophilus influenzae genome identify genes required           genes applies high-density vitro transposon mutagenesis <br /> growth viability. This analysis detected putative essential roles       genetic footprinting discrete chromosomal segments of <br /> for products 259 ORFs unknown function. Comparisons              genomes naturally transformable organisms. This approach <br /> between complete genomes defined subset proteins             rapidly detected known previously unknown genes essen- <br /> H. influenzae having homologs Mycobacterium tuberculosis           tial growth viability vitro. We report application <br /> are absent Saccharomyces cerevisiae, distribution <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>        method complete genome H. influenzae, a <br /> favors use development antimicrobial therapeutics.             common human pathogen causing range diseases includ- <br /> Three genes set essential viability           ing otitis media pneumonia (10, 11). The data obtained <br /> bacteria. Interfacing set essential gene products H.             study indicate &#226;&#172;&#8225;54% essential genes lack <br /> influenzae distribution homologs microorgan-          defined functional roles. Sequence comparison detects poten- <br /> isms detect components unrecognized cellular pathways               tial orthologs large number putatively essential <br /> essential diverse bacteria. This genome-scale phenotypic anal-          genes bacterial species. This information relevant <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ysis identifies potential roles large set genes unknown        defining common essential pathways life and <br /> function.                                                                  potential targets development antimicrobial therapeu- <br />                                                                            tics. This genome-scale database provides experimen- <br />                                                                            tal evidence putative biological roles large group of <br /> T    availability complete microbial genome sequences <br />      allows large-scale functional analysis genes known only <br /> by DNA sequences. A growing number high- <br />                                                                            genes unknown function H. influenzae. <br />  <br /> throughput approaches developed use in-                  Methods <br /> formation. Expression profiling DNA microarrays (1) mea-                Plasmid Construction. To increase efficiency in <br /> sures gene expression highly parallel fashion                 vitro transposon mutagenesis procedure constructed a <br /> identify possible functional relationships genes                mariner-based minitransposon containing H. influenzae <br /> evaluation coexpression (2). Two-hybrid protein interaction             DNA uptake consensus sequence (US) increases trans- <br /> technologies determine proteins physically interact              formation frequency (12). A KpnI site pENT3 (9) was <br /> providing possible means linking genes known pheno-                removed digestion SpeI SapI followed blunt-end <br /> types. Construction specific gene-knockouts genomic                creation T4 polymerase, religation, cloning a <br /> scale initiated Escherichia coli (3) Bacillus subtilis          derivative containing unique KpnI site transposon, <br /> (4). This strategy provides direct means assess genes           pENT3kpnI. Primers (US1-BA, 5&#226;&#172;&#732;-GGTACCA A A AC- <br /> are required given <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> laborious                 CAGTTTCAGGTCAGCAAG-3&#226;&#172;&#732;; US2-BA, 5&#226;&#172;&#732;-GGTAC- <br /> generally impractical individual laboratories. Several al-             CTCGCCTTTGTTCCTTGTGTGCC-3&#226;&#172;&#732;) used PCR- <br /> ternative approaches recently used. In Saccharo-                 amplify 177-bp segment H. influenzae chromosome <br /> myces cerevisiae, genetic footprinting applied chromosome               containing strong match US consensus distrib- <br /> V detected &#226;&#172;&#8225;53 essential genes (5). Sequencing randomly                 uted genome (13). This US segment cloned <br /> generated mutants Mycoplasma genitalium Mycoplasma                  KpnI site pENTkpnI create pENTUS carrying <br /> pneumoniae shown allow rapid identification                 Tnmagellan4 minitransposon. <br /> putatively nonessential genes (6), technique feasible <br /> for laboratories access high-throughput DNA sequenc- <br /> ing capabilities. This approach potentially identify essen-            Mutagenesis H. influenzae Genomic Segments Genetic Foot- <br /> tial genes confidence level depends character-              printing. H. influenzae Rd genome (ATCC #9008) mu- <br /> izing transposon junctions cover complete                    tagenized vitro transposition, using series overlapping <br /> genome high level redundancy (6). Several random                   &#226;&#172;&#8225;10-kb PCR products targets (see Fig. 1 overview). <br /> genetic screens applicable essential gene identification.           Primers designed create overlapping &#226;&#172;&#8225;10-kb PCR <br /> A transposon-based conditional expression used               products representing complete H. influenzae genome <br /> identify 16 essential genes Vibrio cholerae (7). A screen <br /> based inhibition gene function antisense RNA applied <br /> to Staphylococcus aureus ascribed roles viability &#226;&#172;&#8225;150               Abbreviations: US, uptake consensus sequence; BHI, brain heart infusion. <br />  <br /> genes (8). These approaches offer ability easily identify           &#194;&#167;To <br />                                                                              reprint requests addressed at: Department Microbiology and <br />                                                                            Molecular Genetics, Harvard Medical School, Building D1, Room 411, 200 Longwood <br /> the subset essential genes amenable conditional expres-              Avenue, Boston, MA 02115. E-mail: jmekalanos@hms.harvard.edu. <br /> sion inhibition. However, methods stochastic and, <br />                                                                            The publication costs article defrayed page charge payment. This <br /> therefore, provide information genes encoun-                article marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 U.S.C. <br /> tered screen.                                                   &#194;&#167;1734 solely indicate fact. <br />  <br />  <br />  <br /> 966 &#226;&#8364;&#8220;971 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; January 22, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 2                                                          www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.012602299 <br />  Fig. 1. Strategy whole-genome mutagenesis vitro mariner mutagenesis functional analysis genetic footprinting. (A) In vitro transposon <br /> mutagenesis Tnmagellan4 conducted PCR products corresponding regions H. influenzae chromosome. Each mutagenized DNA segment <br /> was introduced separately H. influenzae natural transformation homologous recombination. The Tnmagellan4 element encodes kanamycin <br /> resistance (KmR) contains DNA US, flanked transposon-inverted repeats (black triangles). (B) PCR products (10 kb) were <br /> generated targets mutagenesis &#226;&#172;&#8225;5 kb overlap adjacent segments. These products mutagenized (as A) generate 366 <br /> individual pools transposon mutants. The primers used obtain genetic footprinting data &#226;&#172;&#8225;2.5 kb genome, primer <br /> was used independent pools mutants. <br />  <br />  <br />  <br /> sequence (14). Automated primer design conducted                    30 cycles. A GIBCO&#229;&#8230;&#190;BRL 1-kb DNA ladder used size <br /> the MACVECTOR program desired melting point                         marker gel electrophoresis. <br /> parameter set minimum 62&#194;&#176;C. The resulting primer set <br />  <br />  <br />  <br />  <br />                                                                                                                                                           MICROBIOLOGY <br /> was filtered using Microsoft EXCEL select primers that <br />                                                                              Results <br /> produced &#226;&#172;&#8225;10-kb products &#226;&#172;&#8225;5-kb overlap. This pro- <br /> cedure produced set primers generate targets vitro            Generation Ordered Collection Transposon Insertion Mutants <br />                                                                              Encompassing H. influenzae Genome. Our previous analysis of <br /> transposition querying 2.5-kb segment the <br /> chromosome genetic footprinting. In vitro transposition                   H. influenzae genome analysis mapping vitro <br />                                                                              transposition (GAMBIT) procedure indicated ap- <br /> reactions conducted essentially described (9) and <br />                                                                              proach applied complete genome define <br /> contained 500 ng 10-kb PCR product, 100 ng of <br />                                                                              functional roles genes based DNA sequence information <br /> pENTUS, purified hyperactive C9 mutant (15) Himar1 <br />                                                                              sole input data (9). Genes essential viability under <br /> transposase, gift D. Lampe (Duquesne Univ., Pittsburgh). <br />                                                                              defined growth condition (in case, ability form <br /> The C9 mutant transposase increases efficiency this <br />                                                                              colonies BHIs agar medium) sustain markedly reduced <br /> mutagenesis procedure 10 &#226;&#8364;&#8220;50-fold (15). Each reaction                 frequency transposon insertions detected genetic <br /> transformed competent H. influenzae (16) outgrown                   footprinting. Results approach correlate with <br /> 1 h brain heart infusion (BHI) supplemented hemin (10                independent methods assigning essentiality (9, 17) and, <br /> &#226;?&#174;g&#229;&#8230;&#190;ml) nicotinamide adenine dinucleotide [10 &#226;?&#174;g&#229;&#8230;&#190;ml                       therefore, genes sustain transposon insertions are <br /> (BHIs)], selected colony formation 36 h growth              designated putatively essential. Adjacent nonessential genes <br /> at 37&#194;&#176;C BHIs agar containing 20 &#226;?&#174;g&#229;&#8230;&#190;ml kanamycin                        intergenic regions DNA segment serve internal <br /> sulfate. Transformants pooled BHIs 15% glycerol                 controls sustain insertions. Extended-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> PCR <br /> and stored ordered set &#226;&#171;&#186;80&#194;&#176;C. Dilutions frozen              reactions used amplify 366 &#226;&#172;&#8225;10-kb DNA segments <br /> stock plated obtain 2,000 colonies. Colonies                representing H. influenzae genome. The segments were <br /> were pooled adjusted approximate OD600 0.1,                 designed overlap &#226;&#172;&#8225;5-kb (Fig. 1). This design achieves two <br /> 1 &#226;?&#174;l added PCR reactions containing                       objectives. (i) Mutations segments genome <br /> chromosomal primer corresponding mutagenized region                   represented independent pools transpo- <br /> and primer specific Himar1 inverted repeat                  son mutants. (ii) Primers set complementary to <br /> ends Tnmagellan4, Marout (5&#226;&#172;&#732;-CCGGGGACTTATCAGC-                            sites H. influenzae chromosome spaced, average, <br /> CAACC-3&#226;&#172;&#732;). Genetic footprinting conducted essentially                 2.5 kb. Analysis gene independent pools <br /> described (9) following reaction cycle used:             provides verification. The distance 2.5 kb primer <br /> 94&#194;&#176;C 30 s followed 68&#194;&#176;C 5 min &#226;&#171;&#185; 15 s cycle                binding sites allows obtain overlapping data some <br />  <br /> Akerley et al.                                                                                      PNAS &#229;&#8230;&#169; January 22, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 2 &#229;&#8230;&#169; 967 <br />                                                                               Table 1. Summary putative essential nonessential genes <br />                                                                              Functional designation                                           No. <br />  <br />                                                                              Essential gene candidates (no insertions)                        478 <br />                                                                                Hypothetical genes (unknown functions)                         259 <br />                                                                                Genes known functions                                     219 <br />                                                                              Nonessential genes (insertions distributed ORF)            538 <br />                                                                              One insertion detected                                           191 <br />                                                                              Two insertions detected                                           65 <br />                                                                              Not scored (too far primers)                                453 <br />  <br />  <br />  <br />  <br />                                                                              ping segments, contain relatively high <br />                                                                              proportion putatively essential genes (22 27 genes <br />                                                                              examined). Therefore, relative scarcity nonessential genes <br />                                                                              regions account recovery fewer <br />                                                                              colonies. The regions yielded colonies were <br />                                                                              adjacent segments corresponding putative phage gene prod- <br />                                                                              ucts homologous phage Mu (14). The significance <br />                                                                              reduced insertional mutagenesis frequency Mu-like <br />                                                                              locus clear. <br />  <br />  <br />                                                                              Analysis Transposon Insertion Mutants Each Subgenomic Chro- <br />                                                                              mosomal Region. Mutants representing chromosomal re- <br />                                                                              gion pooled analyzed genetic footprinting as <br />                                                                              described (9). Distances chromosome-specific <br />                                                                              primer calculated, map positions ORF <br />                                                                              respect primer directly superimposed the <br /> Fig. 2.   Scoring genetic footprinting H. influenzae <br />                                                                              gel images shown Fig. 2. Each region scored and <br /> genome. Maps ORF locations respect binding sites primer   color-coded (Fig. 2) according insertions be <br /> were superimposed gel images. Dashed (putative nonessential regions)      detected. The intensity bands footprinting gels, <br /> or solid (putative essential regions) lines gel image represent       precisely quantitative, provides indication of <br /> predicted segments H. influenzae coding sequences. Primer reference       relative number mutants containing insertion mutations <br /> numbers (red) listed near primer positions gene map (Left)    chromosomal position. As previous observations (5, 9) <br /> on gel image lane. Tick marks gene map <br /> represent approximate positions insertions detected. Pools of <br />                                                                              indicated essential genes able sustain insertion <br /> mutants derived independent transposition reactions named        mutations 3&#226;&#172;&#732; end, exclusively scored 5&#226;&#172;&#732; 75% of <br /> the primer pairs used generate targets vitro transposon mutagen-   length ORF criteria. Genes sustaining no <br /> esis (top, brackets). The gel image shows analysis overlapping        insertions region designated putatively essential <br /> &#226;&#172;&#8225;10-kb regions corresponding mutant pools 327&#226;&#8364;&#8220;328 329 &#226;&#8364;&#8220;330.           growth viability. Those capable sustaining insertions <br /> Colors denote summarized results. Red , insertions; blue,         single apparent position likely essential <br /> insertion site; white, insertion mutations partially reduce growth; and <br /> yellow, evenly distributed insertions&#229;&#8230;&#190;putative nonessential gene. Absence <br />                                                                              essential genes encode nonessential domains or <br /> of insertions 300 bp end target PCR product       divided complementing ORFs (although some <br /> considered significant minimum distance required          bands represent insertions multiple adjacent sites <br /> homologous recombination (e.g., blue line marked asterisk lane       depending resolution gel system). We scored <br /> 327).                                                                        genes containing single permissive insertion sites when <br />                                                                              resolve bands 10% length gene. <br />                                                                              Sixty-five ORFs contained insertions apparently limited two <br /> primers average resolution limit electro- <br />                                                                              locations coding region. This group includes both <br /> phoresis gels &#226;&#172;&#8225;4 kb. <br />                                                                              genes known essential (e.g., secA; ref. 9) those <br />   Each PCR product mutagenized vitro transposition <br /> in reactions containing hyperactive C9 mutant                     demonstrated nonessential (such hktE gene <br /> Himar1 transposase mariner-based minitransposon                        encoding catalase; ref. 18) and, therefore, information <br /> (Tnmagellan4) carrying H. influenzae DNA uptake signal                    considered equivocal. Genes encoding untranslated RNA <br /> and gene encoding kanamycin resistance. Mutagenized DNA                    scored. Small genes (&#226;&#172;&#8225;300 bp less) scored <br /> was transformed H. influenzae selected colony                   unless located sufficiently close analysis primers <br /> formation BHIs agar containing kanamycin (Methods). The                   allow accurate resolution insertion site positions. <br /> majority (351&#229;&#8230;&#190;366) reactions yielded 80                      The overall numbers results distribution known <br /> 5,000 colonies average 564. Six reactions yielded                 vs. unknown functional categories listed Table 1. Zero <br /> fewer 50 colonies 9 target PCR products differed               insertion detected 38% 15%, respectively, of <br /> in size predicted segments based genome                      total 1,272 ORFs scored. Thus, identified 670 <br /> sequence omitted. Variation numbers colonies                      putative essential genes 478 genes contained no <br /> obtained, part, ref lects 10-kb stretches DNA                insertions detected method. The complete list of <br /> contain different proportions essential nonessential                  putative essential genes available Table 3, is <br /> genes. For example, 4 6 regions yielded fewer               published supporting information PNAS web site, <br /> than 50 colonies analyzed means adjacent overlap-                 www.pnas.org. <br />  <br /> 968 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.012602299                                                                                    Akerley et al. <br />  Table 2. H. influenzae putative essential genes conserved M. tuberculosis absent S. cerevisae <br /> Hin          Mtu     Bsu     Mge       Cpn     Ctr     Nme       Cje     Hpy     Eco      Pmu      Pae      Vch     Bbu      Tpa               PROSITE <br />  <br /> HI0004           &#226;&#171;&#185;   &#226;&#171;&#185;        &#226;&#171;&#185;E        &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;N       &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;      UPF0054 <br /> HI0175           &#226;&#171;&#185;   &#226;&#171;&#185;N                 &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;        &#226;&#171;&#185;               &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185; <br /> HI0303           &#226;&#171;&#185;   &#226;&#171;&#185;                                   &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185; <br /> HI0338           &#226;&#171;&#185;   &#226;&#171;&#185;                                   &#226;&#171;&#185;                        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185;               Predicted transmembrane <br /> HI0466           &#226;&#171;&#185;                                       &#226;&#171;&#185;                        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185; <br /> HI0570           &#226;&#171;&#185;                                                                &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185; <br /> HI0767           &#226;&#171;&#185;   &#226;&#171;&#185;N                 &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;N       &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;      N6&#227;&#8250;&#173;MTASE <br /> HI1086           &#226;&#171;&#185;                                       &#226;&#171;&#185;                        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;                       PROKAR&#227;&#8250;&#173;LIPOPROTEIN <br /> HI1282           &#226;&#171;&#185;   &#226;&#171;&#185;N                                  &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;N       &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185; <br /> HI1608           &#226;&#171;&#185;   &#226;&#171;&#185;E                         &#226;&#171;&#185;        &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;E       &#226;&#171;&#185;       &#226;&#171;&#185;         &#226;&#171;&#185;                &#226;&#171;&#185; <br /> HI1654           &#226;&#171;&#185;   &#226;&#171;&#185;N       &#226;&#171;&#185;E                         &#226;&#171;&#185;        &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;N       &#226;&#171;&#185;       &#226;&#171;&#185;N        &#226;&#171;&#185;       &#226;&#171;&#185;        &#226;&#171;&#185;      UPF0011 <br /> HI1731           &#226;&#171;&#185;                                       &#226;&#171;&#185;        &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#171;&#185;         &#226;&#171;&#185; <br />  <br />    Presence (&#226;&#171;&#185;) absence (blank) homologs putatively essential H. influenzae proteins range bacteria. Homologs H. influenzae proteins <br /> included excluded table yielded PAM250 DP values &#226;&#172;&#381; 80 P values &#226;&#172;?10&#226;&#171;&#186;3 based precomputed BLASTP comparisons 37 microbial genomes <br /> (http:&#229;&#8230;&#190;&#229;&#8230;&#190;mbgd.genome.ad.jp&#229;&#8230;&#190;). Growth phenotypes previously determined mutants deficient homologs species (see text) summarized as <br /> essential (E) nonessential (N) data available. H. influenzae (hin), M. tuberculosis (mtu), B. subtilis (bsu), Chlamydia pneumoniae (cpn), Chlamydia <br /> trachomatis (ctr), N. meningitidis (nme), Campylobacter jejuni (cje), H. pylori (hpy), E. coli (eco), Pasteurella multocida (pmu), P. aeruginosa (pae), V. cholerae <br /> (vch), Borrelia burgdorferi (bbu), Treponema pallidum (tpa). <br />  <br />  <br />  <br /> Genome Analysis Mapping Vitro Transposition (GAMBIT)                     Essential genes unknown function this <br /> Analysis Gene Products Homologous Genes Assigned                        category important characterize better understand <br /> Functions Other Organisms. Many putative essential                       microbial cell biology defining optimal bacterial <br /> genes detected H. influenzae essential homologs                      targets development antimicrobial therapeutic <br /> other organisms. These include cell division proteins, murein                      agents. We determined putatively essential gene prod- <br /> synthesis enzymes, enzymes one-carbon metabolism,                           ucts unknown functions conserved a <br /> many (19 &#226;&#8364;&#8220;21) (Table 3). Essentiality gene                       phylogenetically distant pathogen, Mycobacterium tuberculosis. <br /> products initially unexpected based functional                          To exclude proteins potentially present in <br /> annotation provided genome sequence. HI0119                           humans, homologs S. cerevisiae removed. <br /> detected essential gene screen, annotated                     Absence proteins humans evaluated TBLASTN <br /> as fimA, encoding major subunit fimbrial surface                          comparisons list (Table 2) human genome <br /> structure nonessential growth organisms.                       sequence database (September 2001). Table 2 notes roles in <br /> However, HI0119 shown Lu et al. (22) encode                            viability conserved genes (homologs HI0004, <br /> protein critical zinc transport. Deletion ORF H.                    HI0767, HI1282, HI1608, HI1654) exam- <br /> influenzae confers severe growth defect aerobic con-                       ined B. subtilis (ref. 4; http:&#229;&#8230;&#190;&#229;&#8230;&#190;locus.jouy.inra.fr&#229;&#8230;&#190;cgi-bin&#229;&#8230;&#190; <br /> ditions absence supplementation high con-                         genmic&#229;&#8230;&#190;madbase&#227;&#8250;&#173;home.pl), E. coli (27), M. genitalium (6), or <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       MICROBIOLOGY <br /> centration zinc (22). Large-scale functional screens                    Pseudomonas aeruginosa (17). Both putatively essential <br /> GAMBIT aid reconciling genome annotation                               H. influenzae genes list conserved M. <br /> subsequent experimental data literature detecting                        genitalium potentially essential M. genitalium. Homologs <br /> such apparent inconsistencies.                                                     HI1608 gene product essential other <br />    We detected putatively essential genes H. influenzae                    organisms examined, B. subtilis E. coli. <br /> that nonessential bacteria. The smpB gene H.                       This gene characterized independently other <br /> influenzae identified essential screen                        groups course study shown encode a <br /> nonessential E. coli B. subtilis (23, 24). Transposon                       component essential isoprenoid biosynthesis pathway in <br /> insertions smpB M. pneumoniae                             E. coli (27&#226;&#8364;&#8220;29). It possible genes identified our <br /> detected (6). SmpB essential activity tmRNA (23,                     analysis encode previously unrecognized components sim- <br /> 25). The tmRNA functions mRNA tRNA                             ilarly characterized cellular pathways. Conversely, highly <br /> adds short peptide tag prematurely terminated translation                     conserved products gene HI1654 putatively <br /> products, targeting degradation. Although tmRNA                        essential Mycoplasma spp. B. subtilis, E. coli, or <br /> nonessential E. coli essential viability Neisseria                 P. aeruginosa. Products HI1654, therefore, participate <br /> gonorrhoeae (26). H. influenzae N. gonorrhoeae                        potential essential pathway bacteria <br /> obligate human parasites relatively small genomes (&#226;&#172;&#8225;1.8                       produce functionally redundant components. Although com- <br /> and &#226;&#172;&#8225;2.2-Mbp, respectively) potentially lack factors                     parisons kind subject specific experimental <br /> present E. coli suppress growth defect smpB                      conditions methods used, results indicate the <br /> mutant.                                                                            requirement given gene viability varies between <br />                                                                                    organisms. <br /> Identification Putative Essential Genes Unknown Functions. <br /> Comparison list essential genes (Table 3) current                   Discussion <br /> genome annotation revealed 259 putatively essential genes                          Based results, estimate 38% H. influenzae genes <br /> lacking defined functional roles. At 159                       critical growth viability colonies rich medium. <br /> initially conserved organisms (14),                         Putatively essential genes likely been <br /> number increases additional genomes sequenced.                         previously characterized genes nonessential functions. <br /> Genes conserved evolutionarily divergent genera                        In cases, putatively essential genes identified <br /> potentially encode components basic cellular processes.                         nonessential organisms. Studies H. influenzae can <br />  <br /> Akerley et al.                                                                                              PNAS &#229;&#8230;&#169; January 22, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 2 &#229;&#8230;&#169; 969 <br />  potentially detect essential cellular pathways                        targets narrow-spectrum antibiotics (31). However, as <br /> alternate mechanisms organisms larger genomes. Alter-                           genomes sequenced, likely fewer genes will <br /> natively, H. influenzae contain additional factors be-                         considered organism-specific. Although directed ap- <br /> come lethal essential gene depleted. Moreover,                              proaches examining single genes produce high <br /> organisms larger genome sizes compared with                             confidence level given gene needed growth, low <br /> smaller coding capabilities increase gene func-                          output essential gene candidates makes finding ideal can- <br /> tions redundant given growth condition.                               didates likely. To obtain reliable assessment of <br />    An estimate essential gene content 55&#226;&#8364;&#8220;73%                                  critical biological pathways organism requires studies <br /> generated Mycoplasma spp. based large-scale sequenc-                           natural environment growth pathogen <br /> ing project identified genes containing transposon inser-                          infected host. Our studies identified com- <br /> tions (6). Two small-scale surveys examining total 53                              prehensive list factors essential vitro a <br /> selected genes E. coli indicated lower proportions                         standard growth condition. These results provide information <br /> (&#226;&#172;&#8225;22%) genes examined essential (6&#229;&#8230;&#190;27 6&#229;&#8230;&#190;26,                              used identify genes essential during <br /> respectively) (27, 30). In fact, true proportion E. coli                         infection. We completed functional genomic analysis of <br /> could potentially lower authors chose targets                          H. influenzae provide publicly available database of <br /> conserved divergent bacterial species attempt favor                         putatively essential nonessential genes organism <br /> detection essential genes. Analysis chromosome V S.                            (see Table 3). Detailed analysis gene required to <br /> cerevisiae indicated &#226;&#172;&#8225;20% genes examined                                confirm specific functions, list provides starting <br /> essential (5). The causes presence greater propor-                         point targeted analysis subsets genes interesting to <br /> tion essential genes H. influenzae Mycoplasma spp.                            investigators examining critical biological processes. For ex- <br /> are likely complex. It likely obligate                      ample, kinetic approach monitoring loss mutants <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> parasites reliant nutrients host,                        pool directly vitro transposon mutagenesis can <br /> fewer biosynthetic capabilities overlapping functions                              provide rapid means initial verification essentiality (5, <br /> compared bacteria environmental niches                                32). In addition, insertions specific DNA regions be <br /> water soil. It possible compare H.                               precisely mapped PAGE examine genes were <br /> influenzae data list &#226;&#172;&#8225;150 critical genes detected                        evaluated (17, 33). This study integrates genomic sequence <br /> the Gram-positive pathogen, S. aureus, exact gene                              data genome-scale mutagenesis. Other global analyses <br /> identifiers genes mediating antisense RNA-inhibited phe-                            genomic expression profiling combined with <br /> notypes bacterium available (8). However,                          results define genetic functions. Genome <br /> important note experiments diverse bacteria                          analysis mapping vitro transposition (GAMBIT) and <br /> were conducted different culture conditions growth                             genetic footprinting results study essenti- <br /> media, factors predicted major impact require-                             ality bacterial genes cultures grown complex medium <br /> ments viability.                                                                    does follow simple patterns. Essentiality predicted <br />                                                                                         exclusively conservation species, genes needed in <br />    A number &#226;&#8364;&#732;&#226;&#8364;&#732;postgenomic&#226;&#8364;&#8482;&#226;&#8364;&#8482; approaches ad- <br />                                                                                         organism necessarily needed another. However, <br /> vanced refine list potential targets antimicrobial <br />                                                                                         comparing essential gene databases divergent organisms can <br /> drug development small numbers genes be <br />                                                                                         used prioritize genetic bioch <br /> </body></html>