<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Apr. 1997, p. 2267&#226;&#8364;&#8220;2273                                                                                               Vol. 179, No. 7 <br /> 0021-9193/97/$04.0010 <br /> Copyright q 1997, American Society Microbiology <br />  <br />  <br />  <br />    aarC, Essential Gene Involved Density-Dependent Regulation <br />            29-N-Acetyltransferase Providencia stuartii <br />                        PHILIP N. RATHER,1,2* KIMBERLY A. SOLINSKY,2 MICHAEL R. PARADISE,2 <br />                                                                      1 <br />                                               AND MILICA M. PAROJCIC <br />  <br />           Research Service, Veterans Affairs Medical Center,1 Departments Medicine Molecular Biology and <br />                    Microbiology, Case Western Reserve University School Medicine,2 Cleveland, Ohio 44106 <br />                                              Received 30 August 1996/Accepted 3 December 1996 <br />  <br />              The 2*-N-acetyltransferase [AAC(2*)-Ia] Providencia stuartii dual function involved the <br />           acetylation peptidoglycan certain aminoglycosides. A search negative regulators aac(2*)-Ia <br />           gene resulted identification aarC. A missense allele (aarC1) resulted 8.9-fold increase in <br />           b-galactosidase accumulation aac(2*)-lacZ transcriptional fusion. Northern blot analysis demon- <br />           strated increase aac(2*)-Ia mRNA accumulation specific cells high density. In addition, the <br />           aarC1 allele resulted substantial increase expression aarP, transcriptional activator the <br />           aac(2*)-Ia gene. The wild-type aarC gene isolated complementation encodes predicted protein of <br />           365 amino acids molecular mass 39,815 Da. The predicted AarC protein exhibited 88% amino acid <br />           homology previously identified GcpE protein Escherichia coli 86% homology gene product <br />           Haemophilus influenzae. The E. coli gcpE gene able functionally complement aarC1 allele in <br />           P. stuartii. The aarC1 allele identified T G transversion resulted valine glycine substi- <br />           tution position 136 AarC protein. The aarC gene appears essential cell viability construc- <br />           tion disrupted copy (aarC::lacZ) possible cells carried episomal copy aarC gcpE. <br />  <br />  <br />    Providencia stuartii gram-negative enteric bacterium re-                high density; therefore, aarC act pathway which <br /> sponsible variety human infections, especially                P. stuartii responds density-dependent signal. <br /> the urinary tract (11, 16, 25). Among family Enterobacte- <br /> riaceae, P. stuartii members tribe Proteeae                                            MATERIALS AND METHODS <br /> (Proteus Morganella spp.) contain unique modification                    Bacterial strains, plasmids, growth conditions. P. stuartii PR50 wild- <br /> of peptidoglycan corresponding O acetylation C-6 hy-                  type strain described previously (20) parent strain for <br /> droxyl muramic acid (4). This modification proposed                    isolation mutants (Table 1). E. coli XL1-Blue (Stratagene) DH5a <br /> control activity autolysins, serving sub-                 (Gibco/BRL) used hosts propagation plasmids. In addition, <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                 propagate R6K pir mutant derivatives, strains DH5a lpir (14) SM10 lpir <br /> strate blocking activity (4&#226;&#8364;&#8220;6, 19). Recently, chro-             (18) used. Plasmids pACYC184 (2), pBluescript SK(2) (Stratagene), <br /> mosomally encoded aminoglycoside 29-N-acetyltransferase                     pBCKS (2) (Stratagene), pKNG101 (13) used cloning vectors. All <br /> been shown role O acetylation peptidogly-                   bacteria grown Luria-Bertani (LB) medium supplemented 15 g of <br /> can P. stuartii (19). This enzyme, encoded aac(29)-Ia                 agar/liter needed. Antibiotics used following concentrations for <br />                                                                                 growth E. coli: ampicillin, 150 mg/ml; chloramphenicol, 25 mg/ml; streptomy- <br /> gene, universal P. stuartii initially identified           cin, 25 mg/ml; tetracycline, 15 mg/ml; kanamycin, 20 mg/ml. For growth P. <br /> role aminoglycoside resistance (3, 20, 26). Presumably, ami-                 stuartii, antibiotics used following concentrations: ampicillin, 150 <br /> noglycosides true substrate AAC(29)-Ia                   mg/ml; chloramphenicol, 100 mg/ml; streptomycin 75, mg/ml; tetracycline, 30 <br /> acetylated structural similarity peptidoglycan.               mg/ml. Selection sucrose resistance P. stuartii 308C LB agar <br />                                                                                 plates NaCl containing 5% sucrose. <br />    Regulation aac(29)-Ia expression complex. Recessive                       Isolation subcloning aarC. A Sau3AI recombinant library P. stuartii <br /> mutations loci, aarA, aarB, aarD, aarG,               chromosomal DNA pACYC184 described previously (14). Plasmid <br /> increase aac(29)-Ia mRNA accumulation identified                      pACYC184.aarC contains approximately 1.7-kb Sau3A partial fragment in <br /> (15, 20, 23). In addition, transcriptional activator, aarP,               pACYC184. Digestion plasmid SalI released 2.0-kb fragment <br />                                                                                 containing P. stuartii insert DNA 276 bp pACYC184 <br /> been identified (14). An additional level regulation includes                DNA BamHI cloning site SalI site. This 2.0-kb SalI fragment was <br /> repression mediated diffusible extracellular factor, AR-                   cloned SalI site pBluescript II SK(2), resulting pSK-2.0. Digestion <br /> factor, acts unknown mechanism decrease                          pSK-2.0 BamHI released 600-bp fragment, subsequent religation of <br /> aac(29)-Ia mRNA accumulation cells approach high density                     vector-containing fragment resulted pSK.aarC1.4, contained 1.4- <br />                                                                                 kb insert entirely composed P. stuartii DNA. This 1.4-kb insert cloned <br /> (22).                                                                           pBCKS (2) 1.4-kb BamHI-SalI fragment, resulting pBC.aarC1.4. <br />    This study reports identification characterization                Plasmid pBC.aarC1.4 subcloned digestion SspI BamHI, <br /> the aarC gene P. stuartii. The deduced AarC protein dis-                     released approximately 1.0-kb fragment subcloned into <br /> played extensive identity GcpE Escherichia coli, gene                   EcoRV- BamHI-digested pBCKS (2) pBluescript II SK(2). This re- <br />                                                                                 sulted pBC.aarC1.0 pSK.aarC1.0, respectively. To isolate DNA frag- <br /> product function unknown re-                        ment containing full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> version aarC gene, chromosomal DNA <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> ported essential (1, 7). In P. stuartii, role aarC             PR50 digested SalI fragments 4 6 kb ligated pBlue- <br /> gene product regulation aac(29)-Ia specific cells                script II SK(2). Previously, Southern blot analysis demonstrated aarC was <br />                                                                                 approximately 5-kb SalI fragment. Colony hybridization 309-bp <br />                                                                                 EcoRV fragment pBC.aarC1.4 used identify recombinant plasmids <br />                                                                                 containing 5-kb SalI fragment full-length aarC gene. This plasmid <br />                                                                                 designated pSK.aarC5.0. Plasmid pPR25 contains approximately 0.7-kb <br />    * Corresponding author. Mailing address: Infectious Diseases Sec-            NruI fragment pSK.aarC5.0 cloned EcoRV site pBCKS (2). <br /> tion 1110W, Wade Park Medical Center, 10701 E. Boulevard, Cleve-                Plasmid pPR26 contains 3.9-kb NruI-SalI fragment pSK.aarC5.0 cloned <br /> land, OH 44106. Phone: (216) 791-3800, ext. 4674. Fax: (216) 229-8509.          EcoRV SalI sites pBCKS (2). <br /> E-mail: pxr17@po.cwru.edu.                                                         To isolate aarC1 <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span>, chromosomal DNA PR60 digested <br />  <br />                                                                          2267 <br />  2268       RATHER ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />                                                             TABLE 1. Bacterial strains plasmids <br />   Strain plasmid                                              Genotype relevant markers                                                       Source <br />  <br /> Strains <br />   E. coli <br />      XL1-Blue                  recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 Dlac-pro [F9 proAB lacIq lacZDM15 Tn10]                      Stratagene <br />      DH5a                      recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 DlacZYA-argF U169 f80dlacZDM15                               Gibco/BRL <br />      DH5a lpir                 DH5a lysogenized lpir                                                                                Macinga et al. (14) <br />      SM10 lpir                 thi thr leu tonA lacY supE recA RP4-2-Tc::Mu Kmr lpir                                                     Miller Mekalanos (18) <br />   P. stuartii <br />      PR50                      Wild type                                                                                                 Rather et al. (20) <br />      PR500                     PR50 containing pR401 pACYC184.aarG                                                                   This study <br />      PR60                      PR50 aarC1                                                                                                This study <br /> Plasmids <br />   pACYC184                     Medium-copy-number vector; Cmr Tcr                                                                        Chang Cohen (2) <br />   pKNG101                      R6K-derived suicide plasmid containing Strr sacB                                                      Kaniga et al. (13) <br />   pBluescript SK(2)            High-copy-number vector; Apr                                                                              Stratagene <br />   pBCKS (2)                    High-copy-number vector; Cmr                                                                              Stratagene <br />   pR401                        aac(29)-lacZ transcriptional fusion                                                                       Rather Orosz (21) <br />   pACYC184.aarC                pACYC184 containing 1.7-kb P. stuartii chromosomal fragment                                               This study <br />   pSK-2.0                      pBluescript SK(2)::2.0-kb SalI fragment pACYC184.aarC                                                This study <br />   pSK.aarC1.4                  pBluescript SK(2)::1.4-kb SalI-BamHI fragment pSK-2.0                                                This study <br />   pBC.aarC1.4                  pBCKS (2)::1.4-kb SalI-BamHI fragment pSK-2.0                                                        This study <br />   pSK.aarC1.0                  pBluescript SK(2)::1.0-kb SspI-BamHI fragment pSK.aarC1.4                                            This study <br />   pBC.aarC1.0                  pBCKS (2)::1.0-kb SspI-BamHI fragment pSK.aarC1.4                                                    This study <br />   pBC.aarC::lacZ               pBC.aarC1.0::lacZ inserted unique NruI site                                                            This study <br />   pKNG101.aarC::lacZ           pKNG101::4.5-kb SalI-BamHI fragment pBC.aarC::lacZ                                                   This study <br />   pSK.aarC5.0                  pBluescript SK(2)::5.0-kb SalI chromosomal fragment PR50                                             This study <br />   pPR25                        pBluescript SK(2)::0.7-kb NruI fragment pSK.aarC5.0                                                  This study <br />   pPR26                        pBluescript SK(2)::3.9-kb NruI-SalI fragment pSK.aarC5.0                                             This study <br />   pSK.aarC1.4mut               pBluescript SK(2)::1.4-kb SalI-BamHI fragment containing aarC1 allele                                     This study <br />   pSK.aarC1.0mut               pSK.aarC1.0 substituted 320-bp NruI-BclI fragment pSK.aarC1.4mut                                This study <br />  <br />  <br />  <br />  <br /> with BamHI SalI, fragments range 1 2 kb ligated         cloned EcoRV site pBluescript II SK(2) used <br /> pBluescript II SK(2) digested BamHI SalI. Colony hybridization           sequencing template. DNA sequence analysis insert within <br /> described used identify plasmids 1.4-kb fragment            pBC.aarC1.4 indicated presence partial open reading frame 738 bp <br /> containing mutant aarC gene. This plasmid designated pSK.aarCmut.1.4.        capable encoding protein 246 amino acids truncated carboxy <br />    Construction aarC::lacZ disruption. Plasmid pBC.aarC1.0 containing        terminus. To sequence remainder aarC coding region, cloned a <br /> 1.0-kb portion aarC coding region linearized digestion NruI,      0.5-kb fragment pACYC184.aarC containing additional 39 coding region. <br /> which cuts position 454 aarC coding region. A promoterless lacZ        This resulted additional 266 nucleotides downstream previous se- <br /> cassette obtained pQF50 (8) digestion SmaI ScaI             quence; however, stop codon open reading frame been <br /> ligated correct orientation pBC.aarC, creating aarC::lacZ transcrip-    reached. To complete sequence, plasmid pSK.aarC5.0, containing entire <br /> tional fusion. A DNA fragment containing fusion excised digestion        aarC gene 5.0-kb SalI fragment, isolated. Custom primers then <br /> with BamHI SalI cloned suicide vector pKNG101 (13), creating        used finish DNA sequence analysis. Protein homology searches were <br /> pKNG101.aarC::lacZ. This plasmid contains pir-dependent origin replica-         performed using blastp program National Center Biotechnology <br /> tion, counterselectable sacB gene, streptomycin resistance gene. To          Information. <br /> introduce pKNG101.aarC::lacZ P. stuartii chromosome, E. coli SM10              b-Galactosidase assays. Plasmid pR401 containing aac(29)-lacZ transcrip- <br /> lpir (18) containing pKNG101.aarC::lacZ mated P. stuartii strains          tional fusion described previously (21). b-Galactosidase activity is <br /> described previously (21). Mating mixtures plated LB plates con-        reported Miller units (17). Reported values represent averages triplicate <br /> taining streptomycin (75 mg/ml), tetracycline (30 mg/ml), X-Gal (5-bromo-        samples independent experiments. <br /> 4-chloro-3-indolyl-b-D-galactopyranoside). In case P. stuartii strains con-      Nucleotide sequence accession number. The sequence aarC sub- <br /> taining plasmid DNA, ampicillin added concentration 300 mg/ml.      mitted GenBank assigned accession number U67933. <br /> Selection loss integrated pKNG101.aarC::lacZ achieved growth <br /> at 308C LB plates NaCl containing 5% sucrose. Southern blot <br /> analysis performed strains digestion chromosomal DNA SalI                                            RESULTS <br /> followed transfer nylon membrane. The aarC probe contained 1.4-kb <br /> BamHI-SalI fragment. The probe specific lacZ 3.2-kb PstI fragment <br /> from pMC1871 (Pharmacia). <br />                                                                                         Isolation aarC1 allele. The chromosomal 29-N-acetyl- <br />    Northern blot analysis. RNA prepared Northern blot analysis using      transferase [AAC(29)-Ia] role O acetylation of <br /> the TRIzol reagent (Gibco/BRL) fractionated 1.2% agarose gel            peptidoglycan P. stuartii (19) acetylates ami- <br /> containing 2.2 M formaldehyde transferred nylon membrane capillary       noglycosides (3, 20, 26). As result, P. stuartii cells in- <br /> transfer. To ensure equal loading transfer RNA nylon mem- <br /> brane, RNAs directly photographed nylon filter UV illumination. <br />                                                                                      creased aac(29)-Ia expression resistant higher levels <br /> Filters probed digoxigenin-labeled 602-bp TaqI-SspI fragment        aminoglycosides (14, 15, 20, 21). To identify genes involved <br /> containing aac(29)-Ia coding sequence. Filters developed Lumi-         regulating aac(29)-Ia expression, strategy gener- <br /> Phos 530 (Boehringer Mannheim) exposed X-ray film.                            ate mutations simultaneously increased expression the <br />    DNA sequencing. Fluorescein-labeled universal reverse primers used <br /> to sequence double-stranded DNA using AutoRead sequencing kit (Phar- <br />                                                                                      chromosomal copy aac(29)-Ia aac(29)-lacZ fusion <br /> macia). Internal primers used needed gaps contiguous          present plasmid pR401 (21). In addition, wanted to <br /> sequence. <span id='am-4' about='rdfs:label' typeof='owl:Thing'>Label</span> incorporated sequencing reactions using flu-       prevent reisolation aarG mutations alter reg- <br /> orescein-labeled dATP. All sequencing reactions analyzed A.L.F.           ulation aac(29)-Ia (23). Therefore, used strain PR500, <br /> automated sequencer (Pharmacia). The entire nucleotide sequence both <br /> strands aarC gene flanking DNA determined manner. <br />                                                                                      PR50 derivative containing pR401 [aac(29)-lacZ] <br /> Specific templates sequencing generated follows. Digestion            pACYC184.aarG, compatible pACYC184 derivative <br /> pBC.aarC1.4 EcoRV released fragments 335, 420, 267, 309, 80 bp,          containing aarG gene. PR500 forms white colonies on <br />  VOL. 179, 1997                                                        DENSITY-DEPENDENT REGULATOR OF aac(29)-Ia                           2269 <br />  <br />  <br /> plates containing X-Gal aminoglycoside resistance 4 <br /> mg gentamicin/ml. Selection mutants achieved by <br /> plating approximately 108 cells PR500 LB plates contain- <br /> ing 15 mg gentamicin/ml, X-Gal (to select identify reg- <br /> ulatory mutants), ampicillin (150 mg/ml; maintain pR401), <br /> and chloramphenicol (100 mg/ml; maintain pACYC184.aarG). <br /> As observed previously, colonies arose frequency 1026 <br /> to 1027 (20). Colonies grew conditions and <br /> exhibited blue color analyzed further. These colonies <br /> presumably resulted mutations trans-acting regulator <br /> of aac(29)-Ia expression chromosomal copy of <br /> aac(29)-Ia plasmid-encoded aac(29)-lacZ fusion was <br /> increased. One class mutants formed small blue <br /> colonies identified, representative isolate, PR60/ <br /> pR401/pACYC184.aarG, characterized. This strain was <br /> then cured pACYC184.aarG, resulting PR60/pR401, <br /> which displayed <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> identical PR500, indicating <br /> that presence aarG, multicopy did alter effects <br /> of aarC1 mutation. Introduction previously cloned <br /> aarA, aarB, aarD genes (15, 20, 21) PR60/pR401 did <br /> not result complementation (data shown), indicating <br /> that PR60 contained new regulatory mutation, was <br /> designated aarC1. <br />    aarC1 increases aac(2*)-Ia high cell density. The accumu- <br /> lation b-galactosidase wild-type PR50/pR401 PR60             FIG. 1. aarC density-dependent regulator. Total RNA (20 mg) pre- <br /> aarC1/pR401 measured cells mid-log phase,          pared PR50 (wild-type) PR60 (aarC1) cells early log phase <br /> 8.9-fold increase b-galactosidase accumulation observed      (OD600 5 0.2) early stationary phase (OD600 5 1.2). In experiment, <br />                                                                     lane 1 represents RNA PR50 lane 2 represents RNA PR60 aarC1. <br /> in PR60 (32.1 6 2.4 Miller units) relative PR50 (3.6 6 0.2       (A) RNA probed 602-bp TaqI-SspI fragment specific aac(29)-Ia. <br /> Miller units). This experiment repeated times           (B) RNA probed digoxigenin-labeled 0.4-kb PCR-generated fragment <br /> gave increases 8.8- 9.2-fold, respectively, PR60.         specific aarP. <br /> Northern blot analysis used directly examine the <br /> effects aarC1 allele aac(29)-Ia mRNA accumulation. <br /> RNA prepared wild-type PR50 PR60 <br /> (aarC1) cured plasmid pR401 log phase (optical density     introduced electroporation PR60/pR401 [aac(29)- <br /> 600 nm [OD600] 5 0.2) examined Northern blot anal-           lacZ] colonies normal growth phenotype iso- <br /> ysis. Unexpectedly, levels aac(29)-Ia mRNA accumula-         lated. This resulted isolation plasmids different <br /> tion PR60 (aarC1) significantly different wild     inserts, containing common region DNA based on <br /> type instances appeared decrease. Based      restriction mapping. Retransformation representative <br /> result, effects aarC1 allele aac(29)-Ia expression    plasmid, pACYC184.aarC, demonstrated insert P. stu- <br /> were examined various stages growth. RNA prepared         artii DNA responsible correcting slow growth phe- <br /> from cells log phase (OD600 5 0.2) stationary phase          notype. In addition, resulting transformants formed <br /> (OD600 5 1.2), aac(29)-Ia mRNA accumulation ex-             white colonies X-Gal plates. Since slow growth <br /> amined Northern blot analysis (Fig. 1A). The levels           phenotype increased aac(29)-Ia expression corrected <br /> aac(29)-Ia mRNA cells PR50 (wild type) PR60               wild type introduction plasmids, reason- <br /> (aarC1) approximately equal log phase (Fig. 1). How-        able recombinant plasmids contained wild-type <br /> ever, late log/early stationary phase, PR60 demonstrated       aarC gene. <br /> considerable increase accumulation aac(29)-Ia mRNA           The sequence aarC coding region determined as <br /> relative wild-type PR50.                                         described Materials Methods. The deduced AarC pro- <br />    Increased expression transcriptional activator aarP    tein contained 365 amino acids exhibited 88% identity <br /> an aarC1 background. Previously, identified tran-          stretch 362 amino acids GcpE protein E. coli <br /> scriptional activator, AarP, activates aac(29)-Ia gene    (1) (Fig. 2). The function GcpE unknown; however, has <br /> (14). The increased aac(29)-Ia expression observed      proposed essential (1). Introduction E. coli <br /> the aarC1 background resulted increased aarP          gcpE gene plasmid pSE401 (7) PR60 aarC1 restored <br /> expression. The accumulation aarP mRNA examined           wild-type growth levels, indicating gcpE functionally <br /> Northern blot analysis shown Fig. 1B. At low cell density,    substitute aarC. The deduced AarC protein displayed <br /> the aarC1 allele did significantly affect accumulation   86% homology 361 amino acids open reading frame <br /> aarP mRNA PR60 relative wild-type PR50. However,           identified Haemophilus influenzae genome (accession <br /> cells high density, accumulation aarP mRNA            number H64063) (9) 51% identity 333 amino acids to <br /> significantly increased aarC1 background.                    YqfY Bacillus subtilis (accession number D84432). <br />    Cloning analysis aarC. PR60 aarC1 formed colonies            Identification aarC1 allele. Plasmid pSK.5.0 contain- <br /> that considerably smaller wild-type PR50 24 h       ing entire aarC gene flanking DNA 5.0-kb SalI <br /> of growth. Since aarC1 mutation selected spontane-          fragment capable complementing aarC1 mutation. <br /> ously, reasoned single mutation responsible        Subclones fragment generated pBluescript SK <br /> for slow growth phenotype increased aac(29)-       (2) contained approximately 700-bp NruI fragment <br /> Ia expression. Therefore, isolate wild-type aarC gene,     (pPR25) 3.9-kb NruI-SalI fragment (pPR26) (Fig. 3). <br /> recombinant library P. stuartii DNA pACYC184 (2, 14)          Neither plasmid capable complementing aarC1 mu- <br />  2270       RATHER ET AL.                                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />     FIG. 2. Comparison AarC GcpE. The deduced amino acid sequences AarC GcpE aligned. Vertical lines indicate identical amino acids, <br /> colons indicate conserved amino acids, dots indicate amino acids low similarity. The position valine glycine substitution AarC mutant protein <br /> is indicated position 136. <br />  <br />  <br />  <br /> tation introduced PR60; however, PR60 cells con-                            aarC::lacZ, integrated chromosome of <br /> taining pPR26 unstable colonies rapidly                          PR50 conjugal mating described Materials Meth- <br /> papillated fast-growing sectors. In contrast, cells contain-                     ods. This integration resulted streptomycin-resistant (Smr) <br /> ing pPR25 remained small. The fast-growing cells result                         sucrose-sensitive (Sucs) colonies blue the <br /> from correction aarC1 allele high frequency re-                          presence X-Gal. Integration pKNG101.aarC::lacZ could <br /> combination pPR26 chromosome. In case,                           occur crossovers (Fig. 4) result a <br /> aarC1 mutation predicted downstream NruI site.                        strain diploid aarC. Crossover 1 yield functional <br /> Furthermore, plasmid pSK.aarC1.0 containing 1.0-kb SspI-                            copy aarC; however, crossover 2 result non- <br /> BamHI fragment shown partially complement aarC1 <br /> allele. These results indicated aarC1 mutation was <br /> likely reside NruI-to-BamHI interval. <br />    A chromosomal DNA fragment containing aarC1 allele <br /> was cloned PR60 1.4-kb SalI-BamHI fragment in <br /> pBluescript SK(2), resulting plasmid pSK.aarC1.4mut. This <br /> plasmid failed complement aarC1 allele intro- <br /> duced PR60. The presence pSK.aarC1.4mut eith- <br /> er wild-type P. stuartii PR50 E. coli XL1 substantially <br /> reduced growth rate cells, indicating mutant <br /> AarC gene product dominant-negative effect. To <br /> further localize aarC1 mutation 1.4-kb fragment, <br /> restriction fragment exchanges performed between <br /> pSK.aarC1.0 pSK.aarC1.4mut. Substitution wild- <br /> type 320-bp NruI-BclI fragment pSK.aarC1.0 mu- <br /> tant fragment pSK.aarC1.4mut resulted E. coli trans- <br /> formants slow growing, resulting hybrid <br /> plasmid, pSK.aarC1.0mut, failed complement PR60. This <br /> indicated aarC1 mutation 320-bp NruI- <br /> BclI interval. The nucleotide sequence fragment re- <br /> vealed single T G transversion resulting valine to <br /> glycine substitution position 136, position conserved with <br /> the E. coli H. influenzae homologs (Fig. 2). <br />    aarC essential gene. A previous report indicated                           FIG. 3. Mapping aarC1 allele. Recombinant plasmids containing the <br /> gcpE essential gene E. coli (1). To assess role                      aarC region shown. The shaded area represents portion aarC gene <br /> aarC P. stuartii, inserted promoterless lacZ cassette                    plasmid. The ability plasmid complement marker rescue <br />                                                                                       aarC1 mutation shown. 1 (marker rescue), extensive formation papillae <br /> a unique NruI site approximately one-third aarC gene.                        colonies; 11 (complementation), restoration wild-type growth rate the <br /> This aarC::lacZ insertion transferred sacB-con-                          aarC1 mutant; 1 (complementation), partial complementation; 2, comple- <br /> taining suicide plasmid pKNG101 (13), resulting pKNG101.                           mentation. <br />  VOL. 179, 1997                                                        DENSITY-DEPENDENT REGULATOR OF aac(29)-Ia                                   2271 <br />  <br />  <br />  <br />  <br />                                                                        FIG. 4. Southern blot analysis aarC::lacZ disruption. (A) The structure <br />                                                                     plasmid pKNG101.aarC::lacZ depicted top, 5.0-kb SalI <br />                                                                     chromosomal region PR50 containing aarC shown immediately below. <br />                                                                     Integration pKNG101.aarC::lacZ chromosome occur the <br />                                                                     552-bp interval 59 inserted lacZ cassette depicted crossover 1 (part 1) <br />                                                                     579-bp interval 39 inserted lacZ cassette depicted in <br />                                                                     crossover 2 (part 2). The predicted structure chromosomal region a <br />                                                                     strain containing aarC::lacZ disruption shown 3. The probe used for <br />                                                                     Southern blot analysis 1.4-kb SalI-BamHI fragment depicted the <br />                                                                     PR50 chromosomal region. (B C) Southern blot verification the <br />                                                                     aarC::lacZ disruption. All DNAs digested SalI. Panel B represents <br />                                                                     DNA probed aarC. Lane 1, purified pSE401 plasmid DNA; lane 2, PR50/ <br />                                                                     pSE401; lanes 3 7, putative PR50 aarC::lacZ/pSE401 mutants. Panel C rep- <br />                                                                     resents DNA probed lacZ gene. All DNAs SalI digested. Lane 1, <br />                                                                     PR50/pSE401; lanes 2 6, putative PR50 aarC::lacZ/pSE401 mutants. <br />  <br />  <br />  <br />  <br /> functional copies aarC, truncation con-       6). No hybridization detected DNA PR50/ <br /> taining lacZ disruption. Southern blot analysis 28 Smr       pSE401 (Fig. 4C, lane 1). <br /> Sucs blue colonies demonstrated contained pKNG101.            To determine viability strain aarC::lacZ <br /> aarC::lacZ integrated crossover 1 (data shown)      disruption dependent presence pSE401, we <br /> retained functional copy aarC. This unexpected,         grew PR50 (aarC::lacZ)/pSE401 absence ampicillin <br /> there homology crossover 2 (579 bp versus 552 bp        approximately 12 generations plated single <br /> for crossover 1). Excision integrated plasmid selection   colonies LB plates. All resulting colonies (474 474 tested) <br /> on 5% sucrose failed yield blue colonies corresponding        remained ampicillin resistant, indicating retained pSE401. <br /> the aarC::lacZ disruption. These results obtained       In contrast, strain PR50/pKNG101.aarC::lacZ/pSE401, which <br /> independent experiments, suggesting colonies containing        diploid aarC (Fig. 4), cured pSE401 frequencies <br /> the aarC::lacZ disruption viable.                          50% conditions. Thus, presence of <br />    To test possibility, attempted resolve    gcpE pSE401 essential construction an <br /> integrated copy pKNG101.aarC-lacZ PR50 transformed            aarC::lacZ disruption chromosome, resulting <br /> with plasmid pSE401 containing E. coli gcpE gene (7). The       strain requires pSE401 viability. <br /> gcpE gene chosen previously shown to <br /> complement growth defect PR60 aarC1, use                                          DISCUSSION <br /> this gene limit potential problem recombina- <br /> tional rescue aarC::lacZ disruption. Selection su-          The aarC gene identified negative regulator of <br /> crose resistance strain resulted approximately       aac(29)-Ia P. stuartii. A missense allele (aarC1) resulted in <br /> equal numbers blue white colonies. Southern blot anal-       increased accumulation aac(29)-Ia mRNA cells <br /> ysis randomly chosen blue colonies confirmed pres-      high density. The aarC1 allele increased ac- <br /> ence aarC::lacZ disruption, demonstrated Fig. 4B       cumulation mRNA specific aarP, gene encoding a <br /> and C. When probe specific aarC used, SalI fragment      transcriptional activator aac(29)-Ia (14). This increase in <br /> of 5 kb corresponds aarC locus present           aarP expression specific cells high density (Fig. <br /> PR50/pSE401 (Fig. 4B, lane 2). In addition, cross-hybridiza-        1). A possible model consistent observations that <br /> tion seen fragment pSE401 (Fig. 4B, lanes 1       aac(29)-Ia overexpression aarC1 background simply <br /> 7). In putative aarC::lacZ mutants 5.0-kb SalI         overexpression aarP. To test possibility, have <br /> fragment longer present replaced fragment       attempted introduce aarP::Cm disruption aarC1 <br /> of approximately 8.5 kb size (lanes 3 7). This         background; however, attempts construct double mu- <br /> expected size, lacZ cassette inserted aarC approx-   tant failed repeatedly. <br /> imately 3.5 kb. To confirm this, probe specific lacZ          The aarC1 allele recessive wild type, suggesting the <br /> used putative aarC::lacZ mutants demonstrated          regulatory consequences aarC1 result loss of <br /> hybridization 8.5-kb fragment (Fig. 4C, lanes 2         AarC function. Interestingly, presence aarC1 allele <br />  2272     RATHER ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />  <br /> on high-copy-number plasmid detrimental growth          probably essential function early cell growth. The <br /> of P. stuartii E. coli. One interpretation result   essential nature aarC remarkable conservation of <br /> is AarC functions multimer overexpression       gene product different gram-negative one <br /> mutant subunit interferes function wild-type         gram-positive bacteria make attractive target de- <br /> subunits. Alternatively, AarC interact complex         velopment new antimicrobials. <br /> other proteins. Analysis AarC protein did reveal any <br /> obvious functional motifs, therefore, mechanism                                     ACKNOWLEDGMENTS <br /> which regulates mRNA accumulation remains deter-              We grateful J. Parker plasmid pSE401 comments on <br /> mined. In addition, hydropathy analysis AarC did reveal       manuscript. In addition, thank D. Macinga review the <br /> any obvious signal sequence hydrophobic regions, suggest-         manuscript. <br /> ing AarC cytoplasmic. AarC shared striking homology            This work supported grant MCB9405882 National <br /> to GcpE E. coli (1) H. influenzae (9) YqfY         Science Foundation Merit Review Award Depart- <br /> product B. subtilis. The high degree conservation      ment Veterans Affairs. <br /> these proteins suggests involved important <br />                                                                                                         REFERENCES <br /> housekeeping function. Attempts disrupt gcpE E. coli <br />                                                                       1. Baker, J., D. B. Franklin, J. Parker. 1992. Sequence characteriza- <br /> were possible cells contained episomal copy           tion gcpE gene E. coli. FEMS Microbiol. Lett. 92:175&#226;&#8364;&#8220;180. <br /> gcpE, indicating gcpE essential (1). This study dem-      2. Chang, A. C. Y., S. N. Cohen. 1978. Construction characterization of <br /> onstrated aarC required viability P.             amplifiable DNA cloning vehicles derived p15A cryptic miniplas- <br /> stuartii. In addition, AarC GcpE functionally equiva-            mid. J. Bacteriol. 134:1141&#226;&#8364;&#8220;1156. <br />                                                                       3. Chevereau, M., P. Daniels, J. Davies, F. LeGoffic. 1974. Aminoglycoside <br /> lent, introduction pSE401 containing E. coli gcpE              resistance bacteria mediated gentamicin acetyltransferase II, en- <br /> gene PR60 resulted complementation aarC1                  zyme modifying 29-amino group aminoglycoside antibiotics. Biochem- <br /> allele allowed construction chromosomal                istry 13:598&#226;&#8364;&#8220;603. <br /> aarC::lacZ disruption. The presence gene E. coli       4. Clarke, A. J. 1993. Extent peptidoglycan O acetylation tribe Pro- <br />                                                                          teeae. J. Bacteriol. 175:4550&#226;&#8364;&#8220;4553. <br /> H. influenzae, contains aac(29)-Ia, suggests         5. Dupont, C., A. J. Clarke. 1991. Evidence N3O acetyl migration as <br /> that additional functions carried AarC. Consistent            mechanism O acetylation peptidoglycan Proteus mirabilis. J. <br /> with this, aarC1 allele resulted slow growth phe-          Bacteriol. 173:4318&#226;&#8364;&#8220;4324. <br /> notype altered cell morphology formation         6. Dupont, C., A. J. Clarke. 1991. Dependence lysozyme catalysed <br />                                                                          solubilization Proteus mirabilis peptidoglycan extent O-acetyla- <br /> short rods, spherical.                                tion. Eur. J. Biochem. 195:763&#226;&#8364;&#8220;769. <br />    The aarC gene added list genes,         7. Eisenbeis, S. J., J. Parker. 1981. Strains Escherichia coli containing <br /> includes aarA, aarB, aarD, aarG, serve negative              structural gene histidyl-tRNA synthetase high copy plasmid. <br /> regulators aac(29)-Ia (15, 20, 21, 23). Among genes             Mol. Gen. Genet. 183:115&#226;&#8364;&#8220;122. <br />                                                                       8. Farinha, M., A. Kropinski. 1990. Construction broad-host-range <br /> only aarG locus, encodes putative response regu-             plasmid vectors easy visible selection analysis promoters. J. Bac- <br /> lator sensor kinase, exhibits clear similarity transcrip-       teriol. 172:3496&#226;&#8364;&#8220;3499. <br /> tional regulators (23). The aarA gene product appears        9. Fleishman, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness, <br /> integral membrane protein, aarD gene encodes CydD,               A. R. Kerlavage, C. J. Bult, J. F. Tomb, B. A. Dougherty, J. M. Merrick, K. <br />                                                                          McKenney, G. Sutton, W. FithHugh, C. A. Fields, J. D. Gocayne, J. D. Scott, <br /> a component ABC transporter complex involved                R. Shirley, L. I. Liu, A. Glodeck, J. M. Kelley, J. F. Weidman, C. A. Phillips, <br /> formation functional cytochrome bd oxidase (15). The                T. Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T. <br /> product aarB presently unknown. The lack similarity             Nguyen, D. M. Saudeck, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. I. <br /> among aar genes transcriptional regulators suggests             Furmann, N. S. Geoghagen, C. L. Gnehm, L. A. McDonald, K. V. Small, <br />                                                                          C. M. Fraser, H. O. Smith, J. C. Venter. 1995. Whole-genome random <br /> they act indirectly affect common cellular process          sequencing assembly Haemophilus influenzae Rd. Science 269:496&#226;&#8364;&#8220; <br /> which aac(29)-Ia and/or aarP regulation coupled. An obvious           512. <br /> candidate process given identification aarD          10. Fuqua, W. C., S. C. Winans, E. Peter Greenberg. 1994. Quorum sensing <br /> (cydD) electron transport. However, purely specula-           bacteria: LuxR-LuxI family cell density-responsive transcriptional <br />                                                                          regulators. J. Bacteriol. 176:269&#226;&#8364;&#8220;275. <br /> tive time.                                                   11. Hawkey, P. 1984. Providencia stuartii: review multiply antibiotic resis- <br />    We shown accumulation aac(29)-Ia mRNA                tant organism. J. Antimicrob. Chemother. 13:209&#226;&#8364;&#8220;226. <br /> is decreased high cell density accumulation      12. Kaiser, D., R. Losick. 1993. How bacteria talk other. <br /> extracellular signaling molecule termed AR-factor (22). This             Cell 73:873&#226;&#8364;&#8220;885. <br />                                                                      13. Kaniga, K., I. Delor, G. Cornelis. 1991. A wide-host-range suicide vector <br /> type regulation termed quorum sensing mediated              improving reverse genetics gram-negative bacteria: inactivation the <br /> by accumulation diverse chemical signals, including               blaA gene Yersinia enterocolitica. Gene 109:137&#226;&#8364;&#8220;141. <br /> amino acids, peptides, acyl derivatives homoserine lac-       14. Macinga, D. R., M. M. Parojcic, P. N. Rather. 1995. Identification and <br /> tone (10, 12, 24). The requirement high cell density              analysis aarP, transcriptional activator 29-N-acetyltransferase in <br />                                                                          Providencia stuartii. J. Bacteriol. 177:3407&#226;&#8364;&#8220;3413. <br /> observe aarC mutant phenotype raises possibility        15. Macinga, D. R., P. N. Rather. 1996. aarD, Providencia stuartii homolog <br /> AarC involved extracellular signaling process. Several          cydD: role 29-N-acetyltransferase expression, cell morphology and <br /> possibilities exist including requirement aarC AR-              growth presence extracellular factor. Mol. Microbiol. 19:511&#226;&#8364;&#8220; <br /> factor production ability respond AR-factor,         520. <br />                                                                      16. McHale, P., F. Walker, B. Scully, L. English, C. T. Keane. 1981. Provi- <br /> experiments progress examine possibilities. Al-          dencia stuartii infections: review 117 cases year period. J. <br /> ternatively, aarC regulate gene expression response            Hosp. Infect. 2:155&#226;&#8364;&#8220;165. <br /> other density-dependent signals oxygen and/or <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>       17. Miller, J. H. 1972. Experiments molecular genetics. Cold Spring Harbor <br /> depletion growth rate. The physiological significance         Laboratory, Cold Spring Harbor, N.Y. <br />                                                                      18. Miller, V. H., J. J. Mekalanos. 1988. A novel suicide vector use <br /> regulation related role aac(29)-Ia O                construction insertion mutations: osmoregulation outer membrane <br /> acetylation peptidoglycan, modification affects deg-          proteins virulence determinants Vibrio cholerae requires toxR. J. <br /> radation muramidases control autolytic activ-            Bacteriol. 170:2575&#226;&#8364;&#8220;2583. <br /> ity (4&#226;&#8364;&#8220;6, 19). The decrease aac(29)-Ia high density        19. Payie, K. G., P. N. Rathe <br /> </body></html>