<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A SNARE required for retrograde transport <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> the <br /> endoplasmic reticulum <br />  <br />  <br /> SNAREs (soluble N-ethylmaleimide-sensitive factor attachment                   the overall membrane flow. Candidate proteins involved in the <br /> protein receptors) are central components of the machinery me-                 fusion of retrograde-directed vesicles with the ER have been <br /> diating membrane fusion in all eukaryotic cells. Sequence analysis             identified in yeast. Genetic and biochemical evidence implicates <br /> of the yeast genome revealed a previously uncharacterized SNARE,               that Sec22 functions as a retrograde SNARE (in addition to its <br /> SNARE-like tail-anchored protein 1 (Slt1). Slt1 is an essential pro-           role as a cis-Golgi t-SNARE light chain) (12, 13). Indeed, the <br /> tein localized in the endoplasmic reticulum (ER). It forms a SNARE             sec22-3 mutant is characterized by an increased secretion of the <br /> complex with Sec22 and the ER syntaxin Ufe1. Down-regulation of                ER-resident protein Kar2 (BIP), suggesting a defect in retro- <br /> Slt1 levels leads to improper secretion of proteins normally resi-             grade transport (14). The syntaxin Ufe1 is localized to the ER <br /> dent in the ER. We suggest that Slt1 is a component of the                     and genetically and biochemically interacts with Sec22, indicat- <br /> SNAREpin required for retrograde traffic to the ER. Based on the               ing a t-SNARE function (13, 14). Two additional proteins, Sec20 <br /> previously reported association with Ufe1 and Sec22, Sec20 likely              and Tip20, are found in a complex with Ufe1 (14&#226;&#8364;&#8220;16). Although <br /> contributes the fourth SNARE to the SNAREpin.                                  Sec20, based on its sequence, has not been classified as a <br />                                                                                SNARE, it contains a coil-coiled motif adjacent to the trans- <br /> membrane fusion &#229;&#8230;&#169; Slt1 &#229;&#8230;&#169; secretion &#229;&#8230;&#169; tail-anchored protein <br />                                                                                membrane domain. By lowering the stringency for a sequence <br />                                                                                alignment, it can be classified as a SNARE (Fig. 1B). In contrast, <br />  <br /> F   usion of biological membranes is a fundamental process <br />     governing transport of cargo molecules throughout the se- <br /> cretory and endocytic pathway. The central components driving <br />                                                                                Tip20 is a peripheral protein associated with Sec20 and could <br />                                                                                play a regulatory role. All known functional SNAREpins are <br />                                                                                assembled <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> four SNAREs, yet there are only three known <br /> intracellular membrane fusion are compartment-specific pro-                    candidates for the subunits of the retrograde SNAREpin: Ufe1, <br /> teins called v- and t-SNAREs [(SNARE, soluble N-ethylmale-                     Sec22, and Sec20. <br /> imide-sensitive factor attachment protein receptor localized on                   To identify the missing fourth SNARE, which functions in the <br /> vesicles (v) or target (t) membranes)] that assemble into cognate              retrograde pathway from the Golgi to the ER, we undertook a <br /> pairs between two lipid bilayers forming SNAREpins (trans                      comprehensive sequence analysis of the Saccharomyces cerevi- <br /> v-&#229;&#8230;&#190;t-SNARE complexes) (1, 2). Each SNAREpin is composed of                     siae genome. Our analysis revealed a SNARE protein termed <br /> a bundle of four &#226;&#65533;&#163;-helices, three derived from the t-SNARE (a                  SNARE-like tail-anchored protein 1 (Slt1). Slt1 proved to be an <br /> syntaxin heavy chain and two SNARE light chains) and the <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          CELL BIOLOGY <br />                                                                                essential, type II integral membrane protein localized to the ER. <br /> fourth from the cognate v-SNARE (3, 4). SNAREpin assembly <br />                                                                                Slt1 is associated with the retrograde SNAREs Ufe1 and Sec22. <br /> likely starts at the amino-terminal, membrane-distal end, pro- <br />                                                                                It is required for the retention of ER resident proteins. There- <br /> ceeds in a membrane-proximal direction, bringing the two lipid <br />                                                                                fore, Slt1 is a component of the retrograde transport machinery, <br /> bilayers in close apposition, and finally culminates in membrane <br />                                                                                likely to be the fourth subunit of the retrograde SNAREpin. <br /> fusion (2, 5&#226;&#8364;&#8220;7). Fully assembled cis v-&#229;&#8230;&#190;t-SNARE complexes (all <br /> SNAREs residing in the same lipid bilayer) then bind N-                        Materials and Methods <br /> ethylmaleimide-sensitive factor attachment protein &#226;&#65533;&#163; (&#226;&#65533;&#163;-SNAP) <br />                                                                                Sequence Analysis. The S. cerevisiae genome was scanned by using <br /> and NSF. Subsequent ATP-hydrolysis by NSF disassembles the <br />                                                                                TMPRED (www.ch.embnet.org&#229;&#8230;&#190;software&#229;&#8230;&#190;TMPRED&#227;&#8250;&#173;form.html) <br /> complex and recycles individual SNAREs for another round of <br /> transport (8).                                                                 to identify those ORFs that might encode proteins with a single <br />    The initial membrane trafficking event in the secretory path-               transmembrane segment. Constraints were imposed on the <br /> way requires the fusion of endoplasmic reticulum (ER)-derived                  initial data: (i) the hydrophobic segment should be positioned <br /> COPII-coated, anterograde-directed vesicles with compart-                      such that less than one-third of the protein would be located as <br /> ments on the cis side of the Golgi. SNAREs required for this                   a luminal (trans) domain; (ii) to be rid of sequences encoding <br /> anterograde transport have been identified in yeast and include                GPI-anchored proteins and type I membrane proteins, ORFs <br /> Sed5, Bos1, Sec22, and Bet1 (9). Although all four SNAREs are                  were screened for the presence of a signal sequence by using <br /> present both on the vesicle and cis-Golgi membranes, they seem                 SIGNALP (17); (iii) to remove fortuitous, small, and nonfunctional <br /> to be required in an asymmetric fashion, either on the vesicle or              ORFs, transcriptional databases were analyzed by using each <br /> target membrane (10). In vitro fusion assays have demonstrated                 sequence as a query. <br /> that SNAREs indeed function in a topologically restricted <br /> manner. Liposomes containing the t-SNARE heavy chain (syn- <br />                                                                                Abbreviations: ER, endoplasmic reticulum; &#226;&#65533;&#163;-SNAP, N-ethylmaleimide-sensitive factor at- <br /> taxin) Sed5 and the light chains Bos1 and Sec22 specifically fuse              tachment protein &#226;&#65533;&#163;; SNARE, soluble N-ethylmaleimide-sensitive factor attachment protein <br /> with v-SNARE liposomes containing Bet1; other topological                      receptor. <br /> combinations of these SNAREs are not fusogenic (11).                           &#226;&#8364;&#160;L.B.   and O.V. contributed equally to this work. <br />    To prevent a prominent net membrane efflux from the ER, to                  &#194;&#182;Present address: Department of Molecular and Cell Biology and the Howard Hughes <br /> recycle the transport machinery, and to retrieve resident proteins              Medical Institute, University of California, Berkeley, CA 94720-3200. <br /> that escaped the ER, retrograde-directed vesicles budding at the               &#229;&#8218;&#168;To   whom correspondence may be addressed. E-mail: j-rothman@ski.mskcc.org or <br /> cis face of the Golgi and throughout the Golgi stack balance                    t-sollner@ski.mskcc.org. <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1734000100                                                       PNAS &#229;&#8230;&#169; August 19, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 17 &#229;&#8230;&#169; 9873&#226;&#8364;&#8220;9877 <br />  Fig. 1. (A) Amino acid sequence of Slt1. The SNARE motif is boxed in gray and the transmembrane domain is in boldface type. (B) Alignment of Slt1 and Sec20 <br /> with SNARE motif classes in yeast. Numbers denote residues in helix&#226;&#8364;&#8220; helix contacts, the residue in the &#226;&#8364;&#732;&#226;&#8364;&#732;0 layer&#226;&#8364;&#8482;&#226;&#8364;&#8482; is boxed in black, and hydrophobic residues in <br /> the helix&#226;&#8364;&#8220; helix contacts positions are shaded (37&#226;&#8364;&#8220;39). <br />  <br />  <br />  <br /> Plasmids, Yeast Strain, and Media. The plasmid coding for recom-                   sion in either 1% Triton X-100 or 100 mM Na2CO3 and <br /> binant GST-&#226;&#65533;&#163;-SNAP has been described (18). Oligonucleotide                         incubation for 30 min on ice with intermittent vortexing (22). <br /> primers were used to attach a signal sequence and the HDEL                         Soluble and insoluble proteins were separated by centrifugation <br /> retention sequence to GFP-S65T, equivalent to the construct                        at 100,000 &#226;&#171;&#187; g in a Beckman Airfuge. Trypsin treatments were <br /> ER-GFP (19), but for the expression in yeast the plasmid chosen                    performed as described (23). <br /> has the MET25 promoter (20) and the carboxyl-terminal se- <br /> quence of the modified GFP-S65T is YKSGLRSRAHDEL.                                  HA Pull-Down Assays. Detergent extracts of sec18-1 were prepared <br /> DNA fragments corresponding to the Slt1 ORF (YGL098w)                              as described (24). Briefly, 20,000 OD units of sec18-1 cells were <br /> were amplified by PCR using primers that generated one in-                         spheroplasted at 25&#194;&#176;C and subsequently incubated for 1 h at <br /> frame restriction site immediately preceding the start codon, and                  37&#194;&#176;C. After incubation, cells were isolated by centrifugation and <br /> another following the stop codon (oligonucleotide sequences                        disrupted with glass beads in 50&#226;&#8364;&#8220;100 ml of ice-cold lysis buffer <br /> available on request). PCR products were subcloned behind                          (50 mM Hepes, pH 7.4&#229;&#8230;&#190;0.1 M KCl&#229;&#8230;&#190;1% Triton X-100&#229;&#8230;&#190;1% <br /> GFP-S65T under the control of the MET25 promoter (20) or                           glycerol&#229;&#8230;&#190;1 mM EDTA&#229;&#8230;&#190;1 mM DTT&#229;&#8230;&#190;protease inhibitors: 1 &#226;&#65533;&#174;g/ml <br /> behind a sequence encoding a triple hemagglutinin (HA)                             leupeptin&#229;&#8230;&#190;2 &#226;&#65533;&#174;g/ml antipain&#229;&#8230;&#190;20 &#226;&#65533;&#174;g/ml trypsin inhibitor&#229;&#8230;&#190;10 &#226;&#65533;&#174;g/ml <br /> epitope under the control of the GAL1 promoter in the plasmid                      benzamidine&#229;&#8230;&#190;5 &#226;&#65533;&#174; g/ml pefabloc SC&#229;&#8230;&#190;2 &#226;&#65533;&#174; g/ml aprotinin&#229;&#8230;&#190;4 <br /> pYX213. The wild-type strain BY4742 (MAT&#226;&#65533;&#163;, his3&#226;&#338;&#172;, leu2&#226;&#338;&#172;,                           &#226;&#65533;&#174;g/ml chymostatin&#229;&#8230;&#190;2 &#226;&#65533;&#174;g/ml pepstatin). Unbroken cells were <br /> lys2&#226;&#338;&#172;, ura3&#226;&#338;&#172;) and the SLT1&#229;&#8230;&#190;&#226;&#338;&#172;slt1 strain 24465 (American Type                       removed by centrifugation for 1 h at 100,000 &#226;&#171;&#187; g. The protein <br /> Culture Collection) were used as a basis for the Kar2 secretion                    concentration of the supernatant was adjusted to 2 mg&#229;&#8230;&#190;ml with <br /> assay, whereas the temperature-sensitive strain sec18-1 (MAT&#226;&#65533;&#163;,                     lysis buffer. A 50-ml extract was incubated overnight at 4&#194;&#176;C with <br /> sec18-1, his4-619, ura3-52) (21) was used for HA-Slt1 pull-down                    2 ml of glutathione beads containing 500&#226;&#8364;&#8220;1,000 &#226;&#65533;&#174;g of GST-&#226;&#65533;&#163;- <br /> assays. The SLT1&#229;&#8230;&#190;&#226;&#338;&#172;slt1 strain was transformed with these plas-                     SNAP. In control experiments, the extract was incubated with <br /> mids, sporulated and dissected to derive haploid strains express-                  GST-glutathione beads. Beads were washed with lysis buffer, <br /> ing only the epitope-tagged Slt1.                                                  and assembled SNARE complexes were eluted with two 5-ml <br />                                                                                    portions of lysis buffer containing 1 M NaCl. The eluate was <br /> Na2CO3 Extraction and Protease Sensitivity. Microsomal membrane                    adjusted to 0.15 M NaCl with lysis buffer and incubated over- <br /> fractions were prepared by differential centrifugation. One                        night at 4&#194;&#176;C with 1.8 ml of high affinity anti-HA Matrix (Roche <br /> hundred OD600 units of cells were suspended in lysis buffer (250                   Diagnostics). After incubation, beads were washed with lysis <br /> mM sorbitol&#229;&#8230;&#190;150 mM potassium acetate&#229;&#8230;&#190;5 mM magnesium                                buffer and then with lysis buffer containing 0.3 M NaCl. Protein <br /> acetate&#229;&#8230;&#190;0.5 mM PMSF&#229;&#8230;&#190;1.2 &#226;&#65533;&#174;g/ml leupeptin&#229;&#8230;&#190;0.75 &#226;&#65533;&#174;g/ml anti-                           complexes were eluted with 1 ml of 0.2 M glycine (pH 2.8) and <br /> pain&#229;&#8230;&#190;0.25 &#226;&#65533;&#174;g/ml chymostatin&#229;&#8230;&#190;1 &#226;&#65533;&#174;g/ml pepstatin&#229;&#8230;&#190;50 mM Hepes,                         precipitated by using 5% (vol&#229;&#8230;&#190;vol) trichloroacetic acid. Precip- <br /> pH 6.8) and disrupted by two 2-min bursts in a minibeadbeater-8                    itated proteins were subjected to Western blot analysis using the <br /> (Biospec Products, Bartlesville, OK) with silica&#229;&#8230;&#190;zirconia beads.                   ECL Plus Western Blotting Detection System (Amersham Phar- <br /> Cell debris was removed by centrifugation at 500 &#226;&#171;&#187; g for 5 min.                    macia). Typically, 10% of the eluted material was loaded in each <br /> A crude membrane fraction was collected by centrifugation at                       lane. Where indicated, yeast lysate was directly incubated with <br /> 16,000 &#226;&#171;&#187; g for 10 min. Membranes were extracted by resuspen-                       anti-HA affinity beads. Polyclonal antibodies to Sed5, Bos1, <br />  <br /> 9874 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1734000100                                                                                                    Burri et al. <br />  Fig. 2. Slt1 is an essential protein. SLT1&#229;&#8230;&#190;&#226;&#338;&#172;slt1 yeast cells were transformed <br /> with a plasmid encoding HA-tagged Slt1 or a control plasmid. The cells were <br /> transferred to sporulation medium to induce meiosis, and the resulting spores <br /> were dissected onto plates of rich medium. Colonies were viewed after 3 days&#226;&#8364;&#8482; <br /> growth at 30&#194;&#176;C. <br />  <br />  <br />  <br /> Sec22, Bet1 (24), Gos1 (25, 26), and Sft1 (27) have been <br /> described. Polyclonal antibodies to Vti1 and Snc1 were gener- <br /> ated in rabbit by using the corresponding His-tagged proteins. <br /> Monoclonal anti-HA peroxidase antibodies (Roche Molecular <br /> Biochemicals) were used for the detection of HA-Slt1. The <br /> anti-Ufe1 antibody was a generous gift of Hugh Pelham (Medical <br /> Research Council Laborator y of Molecular Biology, <br /> Cambridge, U.K.). <br />  <br /> COPII Budding Assay. In vitro budding reactions with purified <br /> COPII proteins were performed as described (28). Briefly, <br /> microsomes were prepared from cells expressing GFP-Slt1 and <br /> stripped of endogenous coat proteins by washing with 2.5 M urea <br /> in B88 [20 mM Hepes, pH 6.8&#229;&#8230;&#190;250 mM sorbitol&#229;&#8230;&#190;150 mM <br />                                                                                 Fig. 3. Slt1 is an integral membrane protein permanently localized in the ER. <br /> KOAc&#229;&#8230;&#190;5 mM Mg(OAC)2]. Purified COPII proteins Sar-1,                             (A) Slt1 is localized to the ER. Yeast cells expressing GFP-Slt1 or GFP-HDEL (ER <br /> Sec23&#229;&#8230;&#190;24, and Sec13&#229;&#8230;&#190;31 were added in a ratio of 10:10:20 mg&#229;&#8230;&#190;ml                  retention signal) were analyzed by confocal fluorescence microscopy. Both <br /> to 300 mg of washed microsomes in the presence or absence of                    fusions target GFP to the ER (arrows, peripheral membrane; N, nucleus). (B) <br /> 0.1 mM guanylyl imidodiphosphate (GMP-PNP) and incubated                        Slt1 is an integral membrane protein. A crude membrane fraction was isolated <br /> for 30 min at room temperature. Of the total reaction, 10% was                  from yeast cells expressing GFP-Slt1, treated with 0.1M Na2CO3, and centri- <br /> removed and diluted in B88. Vesicles in the remaining sample                    fuged to separate solubilized proteins (S) from insoluble material (P). (C) Slt1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                    CELL BIOLOGY <br /> were purified by differential centrifugation, first at 17,610 &#226;&#171;&#187; g for           is a type II membrane protein. The membrane fraction was incubated in the <br />                                                                                 presence (&#226;&#171;&#185;) or absence (&#226;&#171;&#186;) of 1.5 &#226;&#65533;&#174;g of trypsin. (D) Slt1 is not packaged into <br /> 15 min, using a Tomy (Tokyo) MTX-150 swing-out centrifuge, to                   COPII vesicles. Isolated microsomes were incubated with purified Sar-1, Sec23&#229;&#8230;&#190; <br /> sediment donor ER membranes, then at 125,000 &#226;&#171;&#187; g for 20 min,                    24, Sec13&#229;&#8230;&#190;31, and GMP-PNP as indicated. COP II vesicles were isolated as <br /> using a Beckman bench-top ultracentrifuge with a TL100.3 rotor,                 described in Material and Methods. T shows 10% of the microsomes in the <br /> to pellet vesicles. The diluted total membranes were collected in               budding reaction. Proteins were separated by SDS&#229;&#8230;&#190;PAGE and detected by <br /> parallel. Membrane pellets were solubilized in SDS sample                       Western blot analyses. <br /> buffer and analyzed by Western blot. <br />  <br /> Miscellaneous. Fluorescence images were captured by using a                     both containing an aspartate at the &#226;&#8364;&#732;&#226;&#8364;&#732;0 layer&#226;&#8364;&#8482;&#226;&#8364;&#8482; of the SNARE <br /> Bio-Rad MRC1024 Confocal Scanning Laser microscope                              motif (Fig. 1B). <br /> mounted on a Zeiss Axioscop. Cells were grown to mid log phase                    Slt1 is an essential protein: when one copy of the SLT1 gene <br /> at 30&#194;&#176;C in selective media before analysis.                                     was deleted in diploid yeast cells and the SLT1&#229;&#8230;&#190;&#226;&#338;&#172;slt1 cells <br />                                                                                 induced to undergo meiosis, the haploid &#226;&#338;&#172;slt1 cells failed to grow <br /> Results                                                                         (Fig. 2). Transformation of the cells with a plasmid encoding <br /> Because the vast majority of SNARE proteins are type II                         GFP protein fused to the amino terminus of Slt1 (data not <br /> membrane proteins, we analyzed all ORFs in the yeast genome                     shown) or a HA-tagged Slt1 (Fig. 2) rescues the cells. <br /> that encode a single hydrophobic segment and could function as                    Because the tagged Slt1 proteins are functional, they were <br /> a carboxyl-terminal membrane anchor. In a total of 55 proteins,                 used to determine the intracellular localization of Slt1. Both <br /> a SNARE protein termed Slt1 was identified (YGL098w). Slt1                      GFP-Slt1 and HA-Slt1 <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> the nuclear membrane and mem- <br /> is a 245-aa protein and contains an amino-terminal domain                       brane structures underlying the plasma membrane, which are <br /> followed by a predicted carboxyl-terminal transmembrane do-                     both characteristic for the typical ER localization in yeast (Fig. <br /> main (residues 219&#226;&#8364;&#8220;239) (Fig. 1A). Directly preceding the trans-                3A and data not shown). A GFP fusion protein containing the <br /> membrane domain is a sequence that has a significant propensity                 ER-retrieval sequence HDEL, which is an established ER <br /> to form coiled-coils and shows the characteristics of a SNARE                   marker, shows a similar staining <span id='am-82' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Fig. 3A; ref. 19). To <br /> motif. Sequence alignment classifies Slt1 as a Qc&#229;&#8230;&#190;SNAP Cs-                      establish that Slt1 is indeed a type II membrane protein, crude <br /> SNARE (Fig. 1B; ref. 39). Although the overall sequence                         yeast membrane preparations were treated with either sodium <br /> similarity between Slt1 and the other SNAREs is low, the                        carbonate or low amounts of trypsin. As expected, Slt1, like the <br /> sequence of Slt1 is most homologous to the Golgi SNARE Sft1,                    ER marker Sec61, is not extracted at pH 11.5 and therefore <br />  <br /> Burri et al.                                                                                          PNAS &#229;&#8230;&#169; August 19, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 17 &#229;&#8230;&#169; 9875 <br />                                                                                     Fig. 5. Inactivation of Slt1 results in the secretion of ER residents. Wild-type <br />                                                                                    cells or &#226;&#338;&#172;slt1 cells expressing SLT1 under control of the GAL promoter were <br />                                                                                    grown to log phase in rich medium with galactose as a carbon source (YPGal). <br />                                                                                    To repress expression of SLT1, cells were spotted onto glucose plates (YPAD) <br />                                                                                    and incubated at 30&#194;&#176;C for 0, 2, or 4 h, then overlaid with nitrocellulose filters <br />                                                                                    and allowed to grow at 30&#194;&#176;C for an additional hour. Kar2 that had been <br />                                                                                    secreted and bound to the nitrocellulose was detected by immunoblotting <br />                                                                                    with an anti-Kar2 antibody. <br />  <br />  <br />  <br />                                                                                    Golgi syntaxin Sed5. It also interacts with Sec22 and Ykt6, but <br />                                                                                    not with any other SNAREs tested (Fig. 4A), demonstrating the <br />                                                                                    specificity of the interaction. To further confirm the presence of <br />                                                                                    Slt1 in assembled v-&#229;&#8230;&#190;t-SNARE complexes, we made use of an <br />                                                                                    &#226;&#65533;&#163;-SNAP column, which binds with high affinity assembled <br />                                                                                    v-&#229;&#8230;&#190;t-SNARE complexes, but not at all, or only weakly, mono- <br />                                                                                    meric SNAREs (30). SNARE complexes were eluted from the <br />                                                                                    &#226;&#65533;&#163;-SNAP affinity column, and HA-Slt1-containing complexes <br />                                                                                    were further purified with anti-HA antibodies. Western blot <br />                                                                                    analysis confirmed the presence of Ufe1, Sec22, and Ykt6 in the <br />                                                                                    Slt1 precipitates (Fig. 4B). <br />                                                                                       If Slt1 is required for retrograde transport, its inactivation <br />                                                                                    should result in the secretion of ER residents, which are char- <br />                                                                                    acterized by ER retrieval signals. Such a <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> was observed <br /> Fig. 4. Slt1 forms a complex with SNAREs implicated in retrograde transport <br /> to the ER. (A) Detergent extracts of sec18-1 cells, which had been transformed <br />                                                                                    when Ufe1, Sec22, and Sec20, all implicated in retrograde <br /> with HA-Slt1, were incubated with an anti-HA affinity column, and coprecipi-       transport, had been inactivated (12&#226;&#8364;&#8220;14). Therefore, a yeast strain <br /> tated proteins were detected by Western blot analysis. Asterisks indicate          that allowed the controlled expression of the SLT1 gene was <br /> nonspecific IgG cross-reactivity. (B) Detergent extracts of sec18-1 cells, which   engineered, and the secretion of Kar2 (the prototype protein <br /> had been transformed with HA-Slt1, were incubated with a GST-&#226;&#65533;&#163;-SNAP                containing an ER retrieval signal) was monitored (31). After 4 h <br /> affinity column. Assembled v-&#229;&#8230;&#190;t-SNARE complexes were eluted with 1 M NaCl          of Slt1 repression, a significant amount of Kar2 was secreted into <br /> and immunoprecipitated with an anti-HA antibody, and Slt1-associated pro-          the extracellular medium (Fig. 5). This finding demonstrates that <br /> teins were detected by Western blot analysis. <br />                                                                                    Slt1 is a SNARE required for retrograde transport to the ER. <br />  <br />                                                                                    Discussion <br /> behaves as an integral membrane protein (Fig. 3B). Slt1 is <br /> protease-sensitive, which clearly demonstrates that the majority                   The integrity of the secretory pathway relies on the balanced <br /> of the protein (the amino terminus) is exposed in the cytoplasm                    interplay between anterograde and retrograde transport. <br /> (Fig. 3C). In contrast, the soluble, luminal ER marker Kar2 is                     Whereas the anterograde pathway between the ER and the <br /> protected from the protease digestion but sensitive to sodium                      Golgi is relatively well defined, very little is known about the <br /> carbonate extraction. To analyze whether Slt1, like Sec22, enters                  retrograde transport machinery functioning at the ER. Thus, <br /> COPII-coated vesicles budding at the ER, isolated microsomes                       the characterization of components mediating retrograde trans- <br /> were incubated with purified COPII components and budding of                       port, in particular the identification of the corresponding <br /> vesicles was monitored (28). In contrast to Sec22, but like Ufe1,                  SNAREpin that contributes targeting specificity and confers <br /> Slt1 does not enter in significant amounts COPII vesicles and                      membrane fusion, will be of great importance. <br /> behaves as a permanent ER resident (Fig. 3D; ref. 29).                                Our bioinformatic analysis revealed a previously uncharacter- <br />    To test whether Slt1 forms a specific complex with SNAREs                       ized SNARE protein, Slt1, and also uncovered the recently <br /> required for the retrograde transport to the ER, we prepared                       identified Syn8, indicating that (at least in terms of classical type <br /> detergent extract from a temperature-sensitive sec18-1 (yeast                      II membrane proteins) we are approaching complete set of <br /> NSF) strain expressing HA-Slt1. This yeast strain accumulates                      SNAREs in the yeast genome (this work and ref. 32). Thus, the <br /> SNARE complexes at nonpermissive temperature (24). HA-Slt1                         total number of SNAREs in yeast has reached 24 (counting <br /> was isolated with an anti-HA antibody, and coprecipitated                          SNARE isoforms, such as Snc1&#229;&#8230;&#190;2 and Sso1&#229;&#8230;&#190;2, and considering <br /> proteins were detected by Western blot analysis. HA-Slt1 is                        Sec20 as a putative SNARE). Slt1 shows all characteristics of a <br /> found in a complex with the ER syntaxin Ufe1 but not with the                      SNARE required for retrograde transport to the ER. First, Slt1 <br />  <br /> 9876 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1734000100                                                                                                      Burri et al. <br />  is localized to the ER; second, it forms a specific complex with                        SNAREs functionally participate in membrane fusion within the <br /> Ufe1 and Sec22, which have been implicated in the retrograde                            same SNAREpin (27). Therefore, we favor a model where Sec22 <br /> pathway in yeast; and finally, it is functionally required for the                      and Ykt6 are part of two distinct SNAREpins. The observation <br /> retention of the ER-resident protein Kar2 (refs. 14, 33, and 34                         that overexpression of Ykt6 can replace Sec22 in vivo would be <br /> and this work).                                                                         consistent with the above model and the partial functional <br />    Our immunoprecipitation indicates that Ufe1, Slt1, and Sec22                         redundancy of Sec22, and could explain why Sec22 is not an <br /> form a SNARE complex. Previous results of Pelham and                                    essential protein (35). Further, Ykt6 has also been found to <br /> coworkers (14) already showed the existence of a complex                                associate with vacuolar SNAREs, and in this context, it has been <br /> between Ufe1, Sec22, and Sec20, suggesting that the ER                                  shown that Ykt6 and the v-SNARE Nyv1 compete for binding <br /> SNAREpin consists of Ufe1, Slt1, Sec20, and Sec22. Because of                           to the vacuolar t-SNARE, again suggesting the presence of two <br /> a lack of anti-Sec20 antibodies, we were unable to provide direct                       distinct SNARE complexes (4, 36). However, the exact role of <br /> evidence for the existence of this tetrameric complex. Thus, we                         Ykt6 remains to be determined, and a regulatory function (in <br /> cannot completely rule out the possibility that distinct SNARE                          contrast to a direct fusogenic role) cannot be excluded. <br /> complexes containing either Slt1 or Sec20 might exist. The fact                            In summary, we identified a SNARE that is involved in <br /> that both Ufe1 and Slt1 are not packed into COPII-coated                                retrograde transport to the ER. The putative ER SNAREpin <br /> vesicles suggests that these proteins are part of the t-SNARE                           likely consists of the t-SNARE Ufe1&#229;&#8230;&#190;Slt1, Sec20, and the <br /> with Sec20 as the putative additional t-SNARE light chain and                           v-SNARE Sec22. Ykt6 may be part of the second ER SNARE- <br /> Sec22 as the cognate v-SNARE.                                                           pin or play a regulatory function. To unambiguously reveal the <br />    The presence of Ykt6 in the Slt1 immunoprecipitates was                              exact role of these proteins, a functional reconstitution in an in <br /> unexpected. We also found a small amount of endogenous Ykt6                             vitro fusion assay will be required. <br /> in precipitates with antibodies against Ufe1 (data not shown). In <br /> contrast, a myc-tagged Ykt6 expressed in yeast seems not to                            <br /> associate with Ufe1 (14); the reason for this difference remains                        <br /> to be resolved. Interestingly, overexpression of Ykt6 can rescue                       <br /> Sec22-deficient yeast (35). Given the fact that Sec22 does not                           <br /> interact with Ykt6 in vivo, and that Ykt6 and Sec22 belong to the                       <br /> same SNARE class (R-SNARE), it is unlikely that these two                    <br />  <br /> </body></html>