<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Apr. 1998, p. 2079&#226;&#8364;&#8220;2086                                                                                           Vol. 180, No. 8 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology <br />  <br />  <br />  <br />         Isolation Characterization EPD1, Essential Gene for <br />         Pseudohyphal Growth Dimorphic Yeast, Candida maltosa <br />           TAKANOBU NAKAZAWA, HIROYUKI HORIUCHI, AKINORI OHTA,                                            AND   MASAMICHI TAKAGI* <br />                      Department Biotechnology, The University Tokyo, Yayoi, Bunkyo-ku, Tokyo 113, Japan <br />                                             Received 21 August 1997/Accepted 17 February 1998 <br />  <br />             Additional copies centromeric DNA (CEN) region induce pseudohyphal growth dimorphic yeast, <br />           Candida maltosa (T. Nakazawa, T. Motoyama, H. Horiuchi, A. Ohta, M. Takagi, J. Bacteriol. 179:5030&#226;&#8364;&#8220; <br />           5036, 1997). <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> understand mechanism transition, screened gene library C. maltosa for <br />           sequences suppress morphological change. As result, isolated 5* end new gene, <br />           EPD1 (for essential pseudohyphal development), cloned entire gene. The predicted amino acid <br />           sequence Epd1p highly homologous Ggp1/Gas1/Cwh52p, glycosylphosphatidylinositol- <br />           anchored protein Saccharomyces cerevisiae, Phr1p Phr2p Candida albicans. The expression of <br />           EPD1 moderately regulated environmental pH. A homozygous EPD1 null mutant showed mor- <br />           phological defects reduction growth rate reduced levels alkali-soluble alkali-insoluble <br />           b-glucans. Moreover, mutant undergo transition yeast form pseudohyphal form <br />           induced additional copies CEN sequence pH 4 n-hexadecane pH 4 pH 7, suggesting that <br />           EPD1 essential yeast form growth essential transition pseudohyphal form. Over- <br />           expression amino-terminal Epd1p control GAL promoter suppressed the <br />           pseudohyphal development induced additional copies CEN sequence, overexpression the <br />           full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> EPD1 did not. This result initial isolation 5* end EPD1 suppressor the <br />           pseudohyphal growth induced CEN sequence suggest amino-terminal Epd1p have <br />           dominant-negative effect functions Epd1p pseudohyphal growth induced CEN <br />           sequence. <br />  <br />  <br />    Dimorphic fungi exhibit yeast-like form                      analysis closely related C. albicans (34). For recombi- <br /> (pseudo)hyphal form mainly response environmental                       nant DNA technology, host-vector systems con- <br /> conditions, switch yeast-like form filamen-               structed laboratory (13, 16, 35&#226;&#8364;&#8220;37, 51). Recently, we <br /> tous form correlates pathogenicities. Candida                centromeric DNA (CEN) region, when <br /> albicans typical dimorphic fungus cause life-                  present plasmid, induces pseudohyphal growth this <br /> threatening infections immunocompromised patients. In                 yeast. It suggested trans-acting factors in- <br /> organism, dimorphic transition thought critical              teract sequence essential centromeric activity, GGT <br /> pathogenesis, elongated hyphal form facilitates tissue                  AGCG, involved regulation transition <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> penetration (47&#226;&#8364;&#8220;49). Environmental conditions dimor-                    yeast form pseudohyphal form (31). To help under- <br /> phic transition studied related species                 stand mechanism transition, screened gene <br /> Candida tropicalis (52). Saccharomyces cerevisiae shows                library C. maltosa DNA sequences suppress <br /> dimorphic transition (12, 20, 23, 43, 57), induced                   pseudohyphal growth induced additional copies <br /> mainly nitrogen starvation (12). A number genes                    CEN sequence. <br /> been shown participate dimorphism C. albicans and <br /> S. cerevisiae. In S. cerevisiae, reported mito-                                   MATERIALS AND METHODS <br /> gen-activated protein kinase (MAPK) involved pseudohy-                     Strains media. C. maltosa IAM12247 (the wild-type strain) deriv- <br /> phal development nitrogen starvation activated                   atives, CHA1 (his5 ade1) (16) CHAU1 (his5 ade1 ura3) (37), used. The <br /> Ras proteins stimulate pseudohyphal growth                       media C. maltosa YPD (1% yeast extract, 2% Bacto Peptone, 2% <br />                                                                               glucose), SD (pH 4) [0.17% yeast nitogen <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> amino acid ammonium <br /> MAPK-dependent MAPK-independent pathways (7, 23,                          sulfate (Difco), 0.5% (NH4)2SO4, 2% glucose, appropriate nutrients], SG (pH 4) <br /> 25, 29). In addition, genes reported                  (SD glucose replaced 2% galactose), hexadecane <br /> regulation influences pseudohyphal growth (2, 3, 9, 11, 56).                  medium (pH 4) (the SD SG, n-hexadecane, is <br /> Furthermore, variety genes, PHR1, HYR1, CPH1                     supplied vapor, sole carbon source). The SD (pH 7), SG (pH 7), and <br />                                                                               hexadecane (pH 7) media buffered 150 mM HEPES adjusted to <br /> (ACPR), CST20, HST7, EFG1, RBF1, TUP1,                          pH 7.0. When necessary, agar added concentration 2%. <br /> shown affect dimorphism C. albicans (1, 4, 15, 18, 21, 22,                 Escherichia coli MV1190 [D(srl-recA)306::Tn10(Tetr) D(lac-pro) thi supE (F9 <br /> 24, 26, 30, 45, 50). In spite investigations, mecha-             proAB lacIq lacZDM15 traD36)], HB101 [hsdS20(r2 m2) recA13 ara-14 proA2 <br /> nisms dimorphism clear.                                      lacY1 galK2 rpsL20(Smr) xyl-5 mtl-1 l2 mcrA1 mcrB], DH5 (supE44 hsdR1 <br />                                                                               recA1 endA1 gyrA96 thi-1 relA1) grown Luria-Bertani broth used as <br />    Candida maltosa dimorphic fungus diploid ge-                   hosts propagation plasmids construction genomic sub- <br /> nome (14, 19, 31), shown phylogenetic                      genomic libraries C. maltosa. <br />                                                                                  Plasmid construction transformation. The construction pUAH2A and <br />                                                                               pUAHHH1 described previously (31). Ligation DNA ends not <br />                                                                               cohesive blunt ending T4 DNA <br />   * Corresponding author. Mailing address: Department Biotech-             polymerase. The DraI-EcoT22I fragment URA3 C. maltosa, AatII-ClaI <br /> nology, The University Tokyo, Yayoi, Bunkyo-ku, Tokyo 113, Japan.          fragment containing ARS (13, 51), 208-bp HincII-HindIII fragment of <br /> Phone: 81-3-3812-2111, ext. 5169. Fax: 81-3-3812-9246. E-mail:                CEN sequence ligated NaeI site, NspI site, EcoRI <br /> amtakag@hongo.ecc.u-tokyo.ac.jp.                                              site, respectively, pBluescript II KS(1) yield plasmid pBLU1. The <br />  <br />                                                                        2079 <br />  2080       NAKAZAWA ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />                                                                                         Construction genomic subgenomic libraries C. maltosa. Total DNA <br />                                                                                      prepared C. maltosa IAM12247 described previously (36). After <br />                                                                                      partial digestion Sau3AI, total DNA size fractionated sucrose <br />                                                                                      density gradient centrifugation. The DNA fragments 6 8 kb ligated to <br />                                                                                      BamHI-digested phosphatase-treated pUAA10 transformed E. coli <br />                                                                                      DH5. A subgenomic library pUC119 constructed follows. Total DNA <br />                                                                                      C. maltosa IAM12247 digested EcoRI electrophoresed through <br />                                                                                      0.6% agarose gel. A piece gel containing EcoRI fragments 4.5 <br />                                                                                      kb cut out, DNA contained eluted, purified ligated with <br />                                                                                      EcoRI-digested phosphatase-treated pUC119. E. coli MV1190 trans- <br />                                                                                      formed ligated DNA, approximately 5,000 ampicillin-resistant col- <br />                                                                                      onies selected used screening colony hybridization. <br />                                                                                         Deletion EPD1. pEPD1::HIS5 digested ApaLI used trans- <br />                                                                                      form C. maltosa CHAU1. Stable His1 transformants selected obtain the <br />                                                                                      strain UEP11. pEPD1::ADE1 digested ApaLI used transform <br />                                                                                      strain UEP11. Stable His1 Ade1 transformants selected obtain the <br />                                                                                      strain UEP21 (epd1/epd1). Disruption EPD1 gene strains was <br />                                                                                      confirmed Southern hybridization PCR. <br />                                                                                         Glucan analysis. The alkali-insoluble -soluble 1,3- 1,6-b-D-glucans <br />                                                                                      isolated quantified described Popolo et al. (40). <br />                                                                                         Southern blot analysis. Total DNA C. maltosa isolated 10-ml <br />                                                                                      culture grown 15 h selective medium described previously (36). South- <br />                                                                                      ern blot analysis performed using Amersham ECL direct nucleic acid <br />                                                                                      labelling detection according instructions supplier. A <br />                                                                                      548-bp EcoRV-PvuII fragment EPD1 used probe. Colony hybridiza- <br />                                                                                      tion carried Hybond-N1 membrane (Amersham) follows. An <br />                                                                                      EcoRI-XbaI fragment insert pPHS1SX labelled [a-32P]dCTP <br />                                                                                      (Amersham) use random primer labelling kit (Takara) used a <br />                                                                                      probe. Hybridization membrane washing carried method <br />                                                                                      suggested membrane supplier. <br />                                                                                         Northern blot analysis. Total yeast RNA various culture conditions was <br />                                                                                      extracted method Schmitt et al. (46), separated agarose gel electro- <br />                                                                                      phoresis, blotted Hybond-N membrane (Amersham), analyzed. North- <br />                                                                                      ern hybridization carried described previously (38). <br />                                                                                         DNA sequence analysis EPD1. Appropriate restriction fragments the <br />    FIG. 1. Colony morphology C. maltosa CHA1 containing plasmids                  EPD1 gene subcloned pUC series plasmids, DNA sequencing <br /> pUAHHH1 pUAA10 (vector alone) (A), pUAHHH1 pPHS1 (suppres-                   carried automated DNA sequencer (LI-COR model 4000L). <br /> sor) (B), pUAHHH1 pPHS1SX (subclone insert pPHS1) (C).                Nucleotide sequence accession number. The nucleotide sequence data re- <br /> The cells grown SG agar plates (pH 4) 3 days.                            ported paper submitted DDBJ, EMBL, GenBank <br />                                                                                      nucleotide sequence databases accession no. AB005130. <br />  <br />  <br /> AatII-ClaI fragment ARS DraIII-SspI fragment ADE1 (16) C. <br /> maltosa ligated EcoO109I-SspI site SalI site, respectively,                                     RESULTS <br /> of pUC119 yield plasmid pUAA10. The XbaI-XbaI fragment containing <br /> EPD1 (see Fig. 4) ligated XbaI site pUC119, the <br /> DraIII-SspI fragment ADE1 (16) inserted PvuII site EPD1           Isolation multicopy suppressor pseudohyphal growth <br /> yield plasmid pEPD1::ADE1. The XbaI-XbaI fragment containing                induced additional copies CEN sequence. Previously, <br /> EPD1 ligated XbaI site pUC119, SalI fragment            CEN region plasmid induce <br /> containing HIS5 (13) inserted PvuII site EPD1 yield the <br /> plasmid pEPD1::HIS5. The EcoRI-BstPI fragment containing full-length <br />                                                                                      pseudohyphal growth introduced C. maltosa <br /> EPD1 (see Fig. 4) XbaI-XbaI fragment GAL promoter (39)           (Fig. 1A) (31). To analyze mechanism induction, we <br /> ligated SmaI site XbaI site pUAA10, respectively, yield   carried screening genes suppress pseudohyphal <br /> plasmid pUAAEB5. The EcoRI-BglII fragment amino-terminal              growth conditions. First, confirmed the <br /> Epd1p (see Fig. 4) XbaI-XbaI fragment GAL promoter (39) were <br /> ligated SmaI site pUAA10 yield plasmid pUAAEBG3. The <br />                                                                                      host strain does pseudohyphal growth SG agar <br /> EcoRI-SalI, XbaI-EcoRV, SalI-XbaI fragments pPHS1 (see Fig. 2)           plates pH 4 pH 7 host strain containing <br /> ligated SmaI site pUAA10 construct plasmids pPHS1ES,                  pUAHHH1 carrying truncated CEN sequence does it <br /> pPHS1XE, pPHS1SX, respectively.                                                  SG agar plates pHs, pseudohyphal <br />    Transformation C. maltosa performed electroporation described <br /> previously (27). DNA manipulations E. coli transformation as <br />                                                                                      morphology evident pH 4. Then strain was <br /> described previously (44). DNA enzymes purchased Takara Shuzo Co.          transformed C. maltosa genomic library constructed in <br /> (Ohtu, Japan) used according manufacturer&#226;&#8364;&#8482;s instructions.                 high-copy-number vector pUAA10. After 3 days, about <br />  <br />  <br />  <br />  <br />   FIG. 2. Delimitation inserted C. maltosa DNA pPHS1. Colony morphology observed 3 days SG agar plates (pH 4). 1, suppression the <br /> pseudohyphal growth induced additional copies CEN sequence; 2, suppression pseudohyphal growth. <br />  VOL. 180, 1998                                    EPD1 ESSENTIAL FOR PSEUDOHYPHAL GROWTH OF C. MALTOSA                                        2081 <br />  <br />  <br /> 35,000 transformants obtained SG (pH 4) agar plates <br /> and examined microscopically. Most colonies showed pseudo- <br /> hyphal growth, showed reduced pseudohyphal growth. <br /> Plasmids isolated colonies reintro- <br /> duced C. maltosa CHA1 containing pUAHHH1, the <br /> suppression pseudohyphal growth plasmids was <br /> confirmed. One plasmids designated pPHS1 (Fig. <br /> 1B). Restriction enzyme analyses subcloning insert <br /> of pPHS1 performed shown Fig. 2. The plasmid <br /> pPHS1SX, carried smallest fragment, suppressed <br /> pseudohyphal growth level pPHS1 (Fig. 1C). <br /> Only open reading frame (ORF) possible <br /> promoter region inserted DNA of <br /> pPHS1SX. The deduced amino acid sequence ORF is <br /> homologous N-terminal parts S. cerevisiae protein <br /> Ggp1/Gas1/Cwh52p C. albicans proteins Phr1p and <br /> Phr2p. In isolated plasmids, HIS5 gene was <br /> cloned. The transformants YRp-type plasmids car- <br /> rying ADE1 HIS5 lost pUAHHH1 carrying HIS5 and <br /> the truncated CEN sequence, hence, did show <br /> pseudohyphal growth. We investigated newly <br /> isolated plasmid (pPHS1) suppress pseudohyphal <br /> transition induced n-hexadecane, a <br /> strong inducer transition (see Fig. 6 7). We did not <br /> observe suppression plasmid conditions <br /> (data shown). <br />    Isolation entire EPD1 gene. To clone full-length <br /> ORF, EcoRI-XbaI fragment insert pPHS1SX <br /> was used hybridization probe. Southern blot analysis on <br /> EcoRI digests C. maltosa IAM12247 genomic DNA was <br /> carried conditions high stringency, pos- <br /> itive band 4.5 kb detected (data shown). Then, the <br /> C. maltosa subgenomic DNA library contained DNA <br /> fragment 4.5 kb constructed pUC119 and <br /> screened colony hybridization probe. One <br /> positive clone obtained, isolated plasmid des- <br /> ignated pPCO10. Subcloning sequencing inserted <br /> DNA plasmid revealed single, uninterrupted ORF of <br /> 1,650 nucleotides, encode 549-residue protein. <br /> We designated gene EPD1. The plasmid pPHS1SX con- <br /> tained promoter region region encoding amino <br /> acids Met-1 Asn-126 Epd1p. The deduced amino acid <br /> sequence entire Epd1p shown Fig. 3. <br />    Three proteins amino acid sequences similar of <br /> the full-length Epd1p identified databases       FIG. 3. Comparison predicted amino acid sequences Epd1p, Ggp1/ <br /> FASTA program. They products S. cerevisiae GGP1/           Gas1/Cwh52p, Phr1p, Phr2p. Identical similar residues indicated by <br />                                                                    asterisks dots, respectively. The hydrophobic amino carboxy termini are <br /> GAS1/CWH52 C. albicans PHR1 PHR2. The Epd1                 boxed. The proposed GPI attachment sites Epd1p Ggp1/Gas1/Cwh52p <br /> protein 58.4, 56.0, 74.0% identical proteins        indicated bold letters, serine-rich regions underlined. Amino- <br /> Ggp1/Gas1/Cwh52p, Phr1p, Phr2p, respectively. GGP1/            acid N-126, EPD1 gene plasmid pPHS1SX truncated (see <br />                                                                    Results), boxed inversely. <br /> GAS1/CWH52 codes GPI-anchored plasma membrane <br /> glycoprotein (6, 32, 33, 40&#226;&#8364;&#8220;42, 53&#226;&#8364;&#8220;55), PHR1 encodes a <br /> putative GPI-anchored cell surface glycoprotein, production of <br /> which differentially regulated response pH     respectively (10, 32). The serine-rich region located near the <br /> growth medium (45). PHR2 encodes functional homolog           carboxy terminus conserved. <br /> PHR1, expression regulated pH, way        Disruption EPD1. An epd1 null mutant constructed to <br /> exactly inverse PHR1 (30). Mutants         analyze role EPD1 morphogenesis. Two EPD1 alleles <br /> genes showed defects general morphogenesis (30, 40, 41,         C. maltosa CHAU1 inactivated successive gene <br /> 45). The homologous proteins showed similarity          replacements insertion alleles epd1::ADE1 and <br /> their entire sequences. Several regions function-     epd1::HIS5 (Fig. 4A). Single double EPD1 mutants were <br /> ally significant conserved (Fig. 3).          generated described Materials Methods desig- <br /> Epd1p hydrophobic amino terminus, characteristic          nated UEP11 (EPD1/epd1) UEP21 (epd1/epd1), respec- <br /> secretory signal sequences, hydrophobic carboxy termi-       tively. Replacement EPD1 gene homologous recom- <br /> nus, characteristic GPI-linked proteins (8, 28). The Ser-523,   bination confirmed hybridization genomic <br /> Ala-524, Ala-525 residues Epd1p correspond v,        Southern blot (Fig. 4B) PCR (data shown). The null <br /> v11, v12 sites consensus GPI attachment site,           mutants viable, indicating EPD1 essential for <br />  2082      NAKAZAWA ET AL.                                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Disruption EPD1. (A) Construction disrupted genes epd1::ADE1 epd1::HIS5. The detailed methods disruption described in <br /> Materials Methods. (B) Southern blot analysis EPD1 mutants. Genomic DNAs indicated strains digested XbaI hybridized the <br /> labelled EPD1 DNA region indicated panel A probe. The electrophoretic positions DNA size markers indicated left. <br />  <br />  <br />  <br /> cell viability. The duplication times strains CHAU1                       indicate EPD1 involved maintaining normal yeast <br /> UEP21 exponential growth phase SD (pH 4)                           form growth SD pH 4. The morphology yeast form <br /> were 80 120 min, respectively.                                               cells UEP21 (epd1/epd1) different of <br />    The yeast form cells strains grown SD (pH 4)                      wild-type cells SD pH 7 (data shown). <br /> were examined microscopically. UEP11 (EPD1/epd1) cells                             Next, examined effect EPD1 disruption on <br /> were somewhat larger wild-type cells slightly                       pseudohyphal growth. The wild-type strain CHAU1 (Fig. 6A) <br /> round. At early time incubation, UEP21 (epd1/epd1)                        UEP11 (data shown), contained pBLU1 <br /> cells larger wild-type cells round,                   (containing CEN sequence), showed pseudohy- <br /> frequently buds instead one. This <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> is <br /> similar observed ggp1 mutant S. cerevisiae and <br /> the C. albicans phr1/phr1 phr2/phr2 mutants (30, 40, 45). <br /> After prolonged incubation, morphological characteris- <br /> tics mutants evident (Fig. 5). These results <br />  <br />  <br />  <br />  <br />                                                                                     FIG. 6. Effect disruption EPD1 colony morphology pH 4. (A and <br />                                                                                  B) CHAU1 bearing pBLU1 UEP21 containing pBLU1, respectively, were <br />   FIG. 5. Effect disruption EPD1 yeast form growth C. maltosa        grown SG agar plates (pH 4) 1 day. (C D) CHAU1 UEP21, <br /> CHA1 (EPD1/EPD1) (A), UEP11 (EPD1/epd1) (B), UEP21 (epd1/epd1)               respectively, grown plates containing n-hexadecane sole carbon source <br /> (C). Cells grown SD (pH 4) early stationary phase.             (pH 4) 3 days. <br />  VOL. 180, 1998                                             EPD1 ESSENTIAL FOR PSEUDOHYPHAL GROWTH OF C. MALTOSA                                            2083 <br />  <br />  <br />                                                                                         TABLE 1. b-Glucan contents epd1D mutant cellsa <br />                                                                                                                                                           Alkali <br />                                                                                                                            Alkali insoluble <br />                                                                                                                                                          soluble <br />                                                                                          Strain                  1,3- 1                                  1,3- 1 <br />                                                                                                                                 1,6-b-D- <br />                                                                                                                 1,6-b-D-                      ZUPb      1,6-b-D- <br />                                                                                                                                  glucan <br />                                                                                                                  glucan                                  glucan <br />  <br />                                                                               CHAU1 (EPD1/EPD1)                    94             72           28         100 <br />                                                                               UEP11 (EPD1/epd1)                    96             75           30         107 <br />                                                                               UEP21 (epd1/epd1)                    61             43           62          87 <br />                                                                                a <br />                                                                                    All fractions expressed micrograms milligram (dry weight) cells. <br />                                                                                b <br />                                                                                    ZUP, Zymolyase-undigestible pellet. <br />  <br />  <br />  <br />  <br />                                                                               tionary phase SD (pH 4) (Table 1). In strain UEP21, the <br />                                                                               total alkali-insoluble fraction decreased the <br />                                                                               alkali-insoluble 1,6-b-D-glucan 50% of <br />                                                                               wild-type strain. The carbohydrate content the <br />                                                                               alkali-insoluble material obtained Zymolyase diges- <br />                                                                               tion (Table 1). This fraction increased strain UEP21 in <br />                                                                               comparison wild-type strain. The carbohydrate <br />                                                                               contents fractions strain UEP11 similar those <br />                                                                               wild-type strain. The carbohydrate content alkali- <br />                                                                               soluble fraction strain UEP21 lower the <br />                                                                               wild-type strain; different case S. <br />                                                                               cerevisiae ggp1 mutant, 150% the <br />                                                                               wild-type strain (40). <br />                                                                                  Transcriptional regulation EPD1. We examined the <br />                                                                               transcriptional regulation EPD1 Northern blot analysis. <br />                                                                               Total RNA isolated CHA1 containing plasmid <br />    FIG. 7. Effect disruption EPD1 colony morphology pH 7. CHAU1   pUAH2A, does induce pseudohyphal growth, the <br /> carrying pBLU1 (A) UEP21 bearing pBLU1 (B) grown SG (pH 7) <br /> plates 1 day. CHAU1 (C) UEP21 (D) grown hexadecane            centromere plasmid pUAHHH1, induces pseudohyphal <br /> medium (pH 7) 2 3 days, respectively.                                 growth various growth phases (early log phase mid-log <br />                                                                               phase) SG (pH 4). The transcriptional level influ- <br />                                                                               enced growth phase pseudohyphal morphogen- <br />                                                                               esis, transcript detected stationary-phase <br /> phal growth SG agar plates (pH 4) 1 day incubation,               cells (data shown). We examined effect pH on <br /> whereas homozygous mutant, UEP21, containing pBLU1                        transcriptional regulation EPD1. Total RNA iso- <br /> did (Fig. 6B). In addition, strain CHAU1 growing n-                    lated early log phase (optical densities 600 nm [OD600], <br /> hexadecane sole carbon source (pH 4) showed pro-                         0.5 2) SD (pH 4) SD (pH 7). The transcripts were <br /> nounced pseudohyphal growth 3 days (Fig. 6C), strain                 detected samples, level EPD1 mRNA was <br /> UEP21 undergo transition yeast form                   reduced cells grown pH 7 (Fig. 8). This result similar <br /> the pseudohyphal form (Fig. 6D). Strain UEP11 showed                     PHR2 (30). However, expression of <br /> a slight transition pseudohyphal form hexadecane                    EPD1 strictly regulated environmental pH <br /> plates (pH 4) (data shown). In contrast C. albicans                signal PHR2. <br /> phr2 null mutant, failed grow pH 4 (30), epd1                    The amino-terminal Epd1p dominant-negative <br /> null mutant grow SG (pH 4) hexadecane medium                     effect pseudohyphal growth induced CEN se- <br /> (pH 4) SD (pH 4), duplication time                 quence. The 59 end EPD1 initially isolated sup- <br /> UEP21 (120 min) 1.5 times wild-type strain (80                pressor pseudohyphal growth induced additional <br /> min). Thus, incapability strain UEP21 demonstrate                   copies CEN sequence. To confirm phenotype <br /> pseudohyphal growth attributed growth defect.                truncated EPD1 gene, did following <br /> We examined morphological characteristics                       experiments. Plasmids pUAAEB5 pUAAEBG3, ex- <br /> strains SG agar plates (pH 7). Strain CHAU1 containing                     press full-length truncated EPD1, respectively, the <br /> pBLU1 showed pseudohyphal growth (Fig. 7A),                      control GAL promoter (39), constructed trans- <br /> frequency transition high seen pH                formed wild-type strain CHA1 containing pUAHHH1. <br /> 4. To surprise, strain UEP21 containing pBLU1 showed                      After 3 days incubation SG (pH 4), transformants were <br /> pseudohyphal growth equal parent strain (Fig.                  examined microscope. CHA1 containing both <br /> 7B). On hand, UEP21 undergo transition                  pUAHHH1 pUAAEB5 showed pseudohyphal growth, <br /> from yeast form pseudohyphal form hexadecane plates                     CHA1 containing pUAHHH1 pUAAEBG3 <br /> (pH 7) (Fig. 7D), contrast pseudohyphal growth               did (Fig. 9). This result initial isolation 59 <br /> parent strain (Fig. 7C).                                                      end EPD1 suggest overexpression ami- <br />    In ggp1 mutant S. cerevisiae, content alkali-                no-terminal region Epd1p multicopy plasmid a <br /> insoluble 1,6-b-D-glucan shown 50%                 dominant-negative effect pseudohyphal growth induced by <br /> the wild-type strain (40). Therefore, examined                    extra CEN sequence. On hand, overexpression of <br /> amount glucan epd1 mutants entry sta-                 full-length EPD1 using GAL promoter CHA1 did <br />  2084       NAKAZAWA ET AL.                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                                                    form, comparable pseudohypha- <br />                                                                                    forming wild-type strain hexadecane medium pH 4, in <br />                                                                                    contrast failure growth disruptant PHR2, a <br />                                                                                    homolog EPD1. The incapability strain UEP21 show <br />                                                                                    pseudohyphal growth, therefore, attributed its <br />                                                                                    growth defect. UEP21 cells round larger wild- <br />                                                                                    type cells SD (pH 4), observed ggp1, phr1, and <br />                                                                                    phr2 mutants (30, 40, 45), lack cell <br />                                                                                    wall integrity caused defects b-glucan assembly. All these <br />                                                                                    results suggest function Epd1p identical, <br />                                                                                    quite similar, GGP1/GAS1/CWH52, <br />                                                                                    PHR1, PHR2 products. <br />                                                                                       We conclude increased expression EPD1 not <br />                                                                                    required pseudohyphal growth, over- <br />                                                                                    expression full-length EPD1 did affect transition <br />                                                                                    Northern blot analysis showed ex- <br />    FIG. 8. Northern blot analysis EPD1 transcript. C. maltosa CHA1      pression level EPD1 affected pseudohyphal <br /> incubated SD (pH 4) SD (pH 7) media. RNA isolated, Northern         morphogenesis. EPD1 expression constitutive during <br /> blot hybridization performed described Materials Methods. Lane       early mid-log phases, contrast expression of <br /> 1, OD600 0.5 SD (pH 4); lane 2, OD600 0.5 SD (pH 7); lane 3, OD600 <br /> of 2 SD (pH 4); lane 4, OD600 2 SD (pH 7). The upper panel displays   HYR1, C. albicans gene encoding GPI-anchored surface <br /> results hybridization EPD1, lower panel displays results    protein activated response hyphal development <br /> ethidium bromide (EtBr) staining rRNA. The positions 25S 18S rRNA        (1). The EPD1 gene product general function <br /> are shown right.                                                            transition yeast form pseudohyphal form. We <br />                                                                                    examined effect pH EPD1 expression found <br /> not induce pseudohyphal growth SG agar plates (pH 4) <br /> (data shown). <br />  <br />                               DISCUSSION <br />  <br />    Cells yeasts fungi surrounded walls are <br /> responsible mechanical strength cell (17) are <br /> essential maintenance cell shape (58). Fungal cell <br /> walls composed primarily classes polysaccha- <br /> rides: mannoproteins, b-glucans, chitins (5). The b-glucans <br /> constitute 60% cell wall carbohydrate are <br /> believed important elements determining cell <br /> morphology S. cerevisiae (58) probably C. albicans <br /> and C. maltosa. The deduced protein product EPD1 isolated <br /> from C. maltosa present work contains conserved <br /> structural features GPI-anchored proteins, a <br /> hydrophobic signal sequence N terminus, C-terminal <br /> hydrophobic domain, GPI attachment site (8, 10, 28, 32). <br /> These structural similarities suggest Epd1p is <br /> modified addition GPI homolog the <br /> S. cerevisiae GGP1/GAS1/CWH52 gene product C. <br /> albicans PHR1 PHR2 gene products. Although primary <br /> function family genes currently unknown (30, 40, <br /> 45), essential assembly b-glucan, the <br /> lack EPD1 C. maltosa GGP1 S. cerevisiae (40) <br /> reduces level b-glucans (Table 1). <br />    It reported C. albicans phr1/phr1 mutants display <br /> defects formation germ tubes apical growth of <br /> hyphal forms restrictive pH. Prolonged incubation of <br /> wild-type C. albicans higher pH resulted mixture of <br /> yeast form pseudohyphal-form cells, incubation the <br /> phr1/phr1 null mutant did result pseudohyphal-form <br /> cells (45). The effect EPD1 disruption similar results <br /> obtained Saporito-Irwin et al. (45), function of <br /> EPD1 transition yeast form pseudohyphal form <br /> may similar PHR1 morphological transition <br /> of C. albicans. But, case induction pseudohy- <br /> phae n-hexadecane, disruption EPD1 affected <br />                                                                                      FIG. 9. Effect GAL promoter-induced expression EPD1 colony <br /> pseudohyphal morphology pH 4 7. Although                           morphology. CHA1 containing pUAHHH1 pUAAEB5 full-length <br /> growth rate homozygous null mutant, UEP21, lower                        EPD1 (A) pUAHHH1 pUAAEBG3 amino-terminal of <br /> than wild-type strain SD pH 4 yeast                       EPD1 (B) grown SG agar plates (pH 4) 3 days. <br />  VOL. 180, 1998                                                  EPD1 ESSENTIAL FOR PSEUDOHYPHAL GROWTH OF C. MALTOSA                                               2085 <br />  <br />  <br /> that expression EPD1 regulated pH, similar                        7. Cook, J. G., L. Bardwell, J. Thorner. 1997. Inhibitory activating <br /> not strictly PHR2 (30). In C. maltosa, pseudohyphal                                functions MAPK Kss1 S. cerevisiae filamentous-growth signalling <br />                                                                                          pathway. Nature 390:85&#226;&#8364;&#8220;88. <br /> growth pronounced pH 7 pH 4. This                      8. Ferguson, M. A. J. 1988. Cell-surface anchoring proteins glycosylphos- <br /> be explained lower level expression EPD1 pH 7.                           phatidylinositol structures. Annu. Rev. Biochem. 57:285&#226;&#8364;&#8220;320. <br /> These results suggest EPD1 works mainly pH 4,                             9. Gavrias, V., A. Andriannopoulos, C. J. Gimeno, W. E. Timberlake. 1996. <br /> partially pH 7. In addition, hexadecane plates, UEP11                              Saccharomyces cerevisiae TEC1 required pseudohyphal growth. Mol. <br />                                                                                          Microbiol. 19:1255&#226;&#8364;&#8220;1263. <br /> (EPD1/epd1) showed reduced pseudohyphal growth                                   10. Gerber, L. D., K. Kodukula, S. Udenfriend. 1992. Phosphatidylinositol <br /> UEP21 (epd1/epd1) did pseudohyphal growth                               glycan (PI-G) anchored membrane proteins. J. Biol. Chem. 267:12168&#226;&#8364;&#8220; <br /> pH 7. Pseudohyphal growth hexadecane require                                      12173. <br /> greater amounts Epd1p does SG agar plates.                             11. Gimeno, C. J., G. R. Fink. 1994. Induction pseudohyphal growth by <br />                                                                                          overexpression PHD1, Saccharomyces cerevisiae gene related tran- <br />   The interesting finding work amino-                          scriptional regulators fungal development. Mol. Cell. Biol. 14:2100&#226;&#8364;&#8220;2112. <br /> terminal Epd1p dominant-negative effect                         12. Gimeno, C. J., P. O. Ljungdahl, C. A. Styles, G. R. Fink. 1992. Unipolar <br /> pseudohyphal growth induced CEN sequence. It                                  cell divisions yeast S. cerevisiae lead filamentous growth: regulation <br /> inhibit functions Epd1p pseudohyphal growth                                starvation RAS. Cell 68:1077&#226;&#8364;&#8220;1090. <br />                                                                                      13. Hikiji, T., M. Ohkuma, M. Takagi, K. Yano. 1989. An improved host- <br /> induced CEN sequence cells. As total glucan                            vector Candida maltosa using gene isolated genome that <br /> content wild-type strain CHA1 contains pUAAEBG3                              complements his5 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>mutation</span> Saccharomyces cerevisiae. Curr. Genet. <br /> bearing truncated EPD1 downstream GAL promoter                                16:261&#226;&#8364;&#8220;266. <br /> was identical strain CHA1 containing vector pUAA10                        14. Hofmann, K. H., E. Polnisch. 1990. Characterization mutant the <br />                                                                                          yeast Candida maltosa defective catabolite inactivation gluconeogenic <br /> (data shown), expression amino-terminal                           enzymes. Arch. Microbiol. 154:514&#226;&#8364;&#8220;517. <br /> Epd1p cause abnormal architecture, ab-                           15. Ishii, N., M. Yamamoto, F. Yoshihara, M. Arisawa, Y. Aoki. 1997. <br /> normal composition, cell wall. However, expression                            Biochemical genetic characterization Rbf1p, putative transcription <br /> of amino terminus Epd1p promoter                                factor Candida albicans. Microbiology 143:429&#226;&#8364;&#8220;435. <br />                                                                                      16. Kawai, S., T. Hikiji, S. Murao, M. Takagi, K. Yano. 1991. Isolation and <br /> suppressive effect pseudohyphal growth induced n-                              sequencing gene, C-ADE1, use host-vector in <br /> hexadecane sole carbon source. These differential pheno-                            Candida maltosa genetic markers. Agric. Biol. Chem. 55:59&#226;&#8364;&#8220;65. <br /> types differences cell wall                          17. Klis, F. M. 1994. Cell wall assembly yeast. Yeast 10:851&#226;&#8364;&#8220;869. <br /> architectures pseudohypha-forming cells induced n-                         18. Ko&#194;&#168;hler, R. J., G. R. Fink. 1996. Candida albicans strains heterozygous <br />                                                                                          mutations mitogen-activated protein kinase signaling components have <br /> hexadecane induced CEN sequence.                                        defects hyphal development. Proc. Natl. Acad. Sci. USA 93:13223&#226;&#8364;&#8220;13228. <br />   There examples suggesting dose                           19. Komagata, K., T. Nakase, N. Katsuya. 1964. Assimilation hydrocar- <br /> gene affects filamentous growth yeast. For example,                               bons yeasts. II. Determination hydrocarbon-assimilating yeast. J. Gen. <br /> hyphal formation sensitive dose MAPK cascade                            Appl. Microbiol. 4:323&#226;&#8364;&#8220;331. <br />                                                                                      20. Kron, S. J., C. A. Styles, G. R. Fink. 1994. Symmetric cell division in <br /> components C. albicans (18). Our previous data indicated                              pseudohyphae yeast Saccharomyces cerevisiae. Mol. Biol. Cell 5:1003&#226;&#8364;&#8220; <br /> that copies YCp-type plasmid induce marked                                 1022. <br /> pseudohyphal growth single copy C. maltosa (31),                         21. Lambrechts, M. G., F. F. Bauer, J. Marmur, I. S. Pretorius. 1996. Muc1, <br /> and present results indicate heterozygous disruptant                          mucin-like protein regulated Mss10, critical pseudohyphal <br /> of EPD1 shows slight transition yeast form                            differentiation yeast. Proc. Natl. Acad. Sci. USA 93:8419&#226;&#8364;&#8220;8424. <br />                                                                                      22. Leberer, E., D. Harcus, I. D. Broadbent, K. L. Clark, D. Dignard, K. <br /> pseudohyphal form induced n-hexadecane. These results                                 Ziegelbauer, A. Schmidt, N. A. R. Gow, A. J. P. Brown, D. Y. Thomas. <br /> taken suggest copy numbers cellular                            1996. Signal transduction homologs Ste20p protein kinase can <br /> components important induction pseudohyphal                               trigger hyphal formation pathogenic fungus Candida albicans. Proc. <br /> growth.                                                                                  Natl. Acad. Sci. USA 93:13217&#226;&#8364;&#8220;13222. <br />                                                                                      23. Liu, H., C. A. Styles, G. R. Fink. 1993. Elements yeast pheromone <br />   EPD1 essential growth, essential                          response pathway required filamentous growth diploid. Science 262: <br /> pseudohyphal transition. Defining precise role Epd1p                              1741&#226;&#8364;&#8220;1744. <br /> will helpful understanding mechanism dimor-                        24. Liu, H., J. Ko&#194;&#168;hler, G. R. Fink. 1994. Suppression hyphal formation in <br /> phism fungi.                                                                          Candida albicans mutation STE12 homolog. Science 266:1723&#226;&#8364;&#8220;1725. <br />                                                                                      25. Madhani, H. D., G. R. Fink. 1997. Combinational control required for <br />                                                                                          specificity yeast MAPK signaling. Science 275:1314&#226;&#8364;&#8220;1317. <br />                          ACKNOWLEDGMENTS                                             26. Malathi, K., K. Ganesan, A. Datta. 1994. Identification putative <br />                                                                                          transcription factor Candida albicans complement mating <br />   Part work performed using facilities Biotechnology                  defect Saccharomyces cerevisiae ste12 mutants. J. Biol. Chem. 269:22945&#226;&#8364;&#8220; <br /> Research Center The University Tokyo. T.N. thanks members                      22951. <br /> of Takagi Laboratory helpful discussions support.                  27. Mauersberger, S., M. Ohkuma, W.-H. Schunck, M. Takagi. 1995. Can- <br />                                                                                          dida maltosa, p. 411&#226;&#8364;&#8220;580. In K. Wolf (ed.), Nonconventional yeasts bio- <br />                                                                                          technology. Springer, Heidelberg, Germany. <br />                                 REFERENCES                                           28. McConville, M. J., M. A. J. Ferguson. 1993. The structure, biosynthesis <br />  1. Bailey, D. A., P. J. F. Feldmann, M. Bovey, N. A. R. Gow, A. J. P. Brown.        function glycosylated phosphatidylinositols parasitic protozoa <br />     1996. The Candida albicans HYR1 gene, activated response              higher eukaryotes. Biochem. J. 294:305&#226;&#8364;&#8220;324. <br />     hyphal development, belongs gene family encoding yeast cell wall pro-       29. Mo &#194;&#168;sch, H.-U., R. L. Roberts, G. R. Fink. 1996. Ras2 signals the <br />     teins. J. Bacteriol. 178:5353&#226;&#8364;&#8220;5360.                                                  Cdc42/Ste20/mitogen-activated protein kinase module induce filamentous <br />  2. Blacketer, M. J., C. M. Koehler, S. G. Coats, A. M. Myers, P. Madaule.           growth Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA 93:5352&#226;&#8364;&#8220; <br />     1993. Regulation dimorphism Saccharomyces cerevisiae: involvement           5356. <br />     novel protein kinase homolog Elm1p protein phosphatase 2A. Mol.          30. Mu &#194;&#168;hlschlegel, F. A., W. A. Fonzi. 1997. PHR2 Candida albicans <br />     Cell. Biol. 13:5567&#226;&#8364;&#8220;5581.                                                            encodes functional homolog pH-regulated gene PHR1 an <br />  3. Blacketer, M. J., P. Madaule, A. M. Myers. 1995. Mutational analysis          inverted <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> pH-dependent expression. Mol. Cell. Biol. 17:5960&#226;&#8364;&#8220;5967. <br />     morphologic differentiation Saccharomyces cerevisiae. Genetics 140:1259&#226;&#8364;&#8220;      31. Nakazawa, T., T. Motoyama, H. Horiuchi, A. Ohta, M. Takagi. 1997. <br />     1275.                                                                                Evidence centromeric DNA region induces pseudohyphal <br />  4. Braun, B. R., A. D. Johnson. 1997. Control filament formation              growth dimorphic yeast, Candida maltosa. J. Bacteriol. 179:5030&#226;&#8364;&#8220;5036. <br />     Candida albicans transcriptional repressor TUP1. Science 277:105&#226;&#8364;&#8220;109.         32. Nuoffer, C., A. Horvath, H. Riezman. 1993. Analysis sequence <br />  5. Cabib, E., R. Roberts, A. Sburlati, J. Silverman. 1982. Synthesis yeast       requirements glycosylphosphatidylinositol anchoring Saccharomyces <br />     cell wall regulation. Annu. Rev. Biochem. 51:763&#226;&#8364;&#8220;793.                        cerevisiae Gas1 protein. J. Biol. Chem. 268:10558&#226;&#8364;&#8220;10563. <br />  6. Conzelmann, A., H. Riezman, C. Desponds, C. Bron. 1988. A major              33. Nuoffer, C., P. Jeno&#194;&#168;, A. Conzelmann, H. Riezman. 1991. Determinants <br />     125-kd membrane glycoprotein Saccharomyces cerevisiae attached          glycophospholipid anchoring Saccharomyces cerevisiae GAS1 pro- <br />     lipid bilayer inositol-containing phospholipid. EMBO J. 7:2233&#226;&#8364;&#8220;           tein plasma membrane. Mol. Cell. Biol. 11:27&#226;&#8364;&#8220;37. <br />     2240.                                                                            34. Ohkuma, M., C. W. Hwang, Y. Masuda, H. Nishida, J. Sugiyama, A. Ohta, <br />  2086         NAKAZAWA ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />       M. Takagi. 1993. Evolutionary position n-alkane-assimilating yeast         46. Schmitt, M., T. A. Brown, B. L. Trumpower. 1990. A rapid simple <br />       Candida maltosa shown nucleotide sequence small-subunit ribosomal               method preparation RNA Saccharomyces cerevisiae. Nucleic <br />       RNA gene. Biosci. Biotechnol. Biochem. 57:1793&#226;&#8364;&#8220;1794.                                  Acids Res. 18:3091. <br /> 35.   Ohkuma, M., K. Kobayashi, S. Kawai, C. W. Hwang, A. Ohta, M.                  47. Soll, D. R. 1996. The emerging molecular biology switching Candida <br />       Takagi. 1995. Identification centromeric activity autonomously            albicans. ASM News 8:415&#226;&#8364;&#8220;420. <br />       replicating TRA region allows improvement host-vector           48. Soll, D. R., B. Morrow, T. Srikantha. 1993. High-frequency phenotypic <br />       Candida maltosa. Mol. Gen. Genet. 249:447&#226;&#8364;&#8220;455.                                        switching Candida albicans. Trends Genet. 9:61&#226;&#8364;&#8220;65. <br /> 36.   Ohkuma, M., S. Muraoka, C. W. Hwang, A. Ohta, M. Takagi. 1993.                49. Stewart, S., P. T. Magee. 1990. Genetics Candida albicans. Microbiol. <br />       Cloning C-URA3 gene construction triple auxotroph (his5,              Rev. 54:226&#226;&#8364;&#8220;241. <br />       ade1, ura3) useful host genetic engineering Candida maltosa.      50. Stoldt, V. R., A. Sonneborn, C. E. Leuker, J. F. Ernst. 1997. Efg1p, an <br />       Curr. Genet. 23:205&#226;&#8364;&#8220;210.                                                              essential regulator morphogenesis human pathogen Candida albi- <br /> 37.   Ohkuma, M., S. Muraoka, T. Tanimoto, M. Fujii, A. <br /> </body></html>