<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 2007, p. 3721&#226;&#8364;&#8220;3728                                                                                     Vol. 189, No. 10 <br /> 0021-9193/07/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.01740-06 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />        Functional Complementation Essential Gene fabG1 of <br />          Mycobacterium tuberculosis Mycobacterium smegmatis fabG <br />                        Not Escherichia coli fabG&#228;&#338;&#164; <br />                      Tanya Parish,1* Gretta Roberts,1&#226;&#8364;&#160; Francoise Laval,2 Merrill Schaeffer,3&#226;&#8364;&#161; <br />                                      Mamadou Daffe&#194;&#180;,2 Ken Duncan,4&#194;&#167; <br />  Centre Infectious Disease, Institute Cell Molecular Science, Barts London, London E1 2AT, United Kingdom1; <br />    Institut Pharmacologie et Biologie Structurale (UMR5089), Department Molecular Mechanisms Mycobacterial Infections, <br />      Centre National la Recherche Scientifique Universite&#194;&#180; Paul Sabatier (Toulouse III), 205 Route Narbonne, <br />        F-31077 Toulouse Cedex, France2; GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 194263; and <br />                     GlaxoSmithKline, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom4 <br />                                            Received 13 November 2006/Accepted 27 February 2007 <br />  <br />             Mycolic acids key component mycobacterial cell wall, providing structure forming major <br />           permeability barrier. In Mycobacterium tuberculosis mycolic acids synthesized type I type II fatty acid <br />           synthases. One enzymes type II encoded fabG1. We demonstrate gene <br />           deleted M. tuberculosis chromosome functional copy provided elsewhere, <br />           showing normal culture conditions fabG1 essential. FabG1 activity replaced the <br />           corresponding enzyme closely related species Mycobacterium smegmatis enzyme <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />           Escherichia coli. M. tuberculosis carrying FabG M. smegmatis showed phenotypic changes, the <br />           mycolic acids cell wall permeability unchanged. Thus, M. tuberculosis M. smegmatis enzymes are <br />           interchangeable control lengths types mycolic acids synthesized. <br />  <br />  <br />    The mycobacterial cell wall complex lipid-rich struc-              acid synthase (FAS). FAS I multienzyme, single-polypep- <br /> ture composed outer layer mycolic acids covalently                    tide complex responsible novo synthesis C22 <br /> linked arabinogalactan, turn linked pepti-                   C26 fatty acids. FAS II composed number individ- <br /> doglycan. This structure surrounded outermost layer,                 ual enzymes, including enzymes encoded inhA fabG, <br /> the &#226;&#8364;&#339;capsule,&#226;&#8364;? pathogenic species (8). Much work fo-                    elongate FAS I products produce long-chain <br /> cused understanding role biosynthesis com-                   fatty acids (for review, reference 29). <br /> ponents cell wall, particularly mycolic acids. Current                 Mycolic acids particular interest, biosynthesis <br /> models propose mycolic acids form inner leaflet                   target antibiotics, isoniazid ethionamide. A <br /> of asymmetric bilayer outside peptidoglycan-arabino-                    number compounds, including triclosan thiolacto- <br /> galactan layer, free lipids forming outer leaflet (5, 8).             mycin, affect mycolic acid biosynthesis inhibiting enzymes of <br /> Numerous molecules, including important immune                       FAS II pathway. Global gene expression profiling drug <br /> modulator lipoarabinomannan, outer                      treatment demonstrated isoniazid, thiolactomycin, and <br /> layer. This extremely hydrophobic layer forms important                     triclosan different effects cells are <br /> permeability barrier reasons intrinsic               likely different modes action (2). Triclosan isoni- <br /> resistance mycobacteria antibacterial agents. <br />                                                                                azid inhibit InhA (21), thiolactomycin inhibits KasA <br />    Mycolic acids &#226;?&#163;-alkyl, &#226;?&#164;-hydroxy fatty acids two <br />                                                                                KasB (12, 15, 26, 27). There debate concern- <br /> alkyl chains, &#226;&#8364;&#339;short&#226;&#8364;? chain (C22 C26) long &#226;&#8364;&#339;mero- <br />                                                                                ing primary target isoniazid, prodrug is <br /> mycolate&#226;&#8364;? chain (C&#226;&#172;&#381;50). Mycobacterium tuberculosis synthe- <br />                                                                                activated mycobacterial KatG catalase (13, 33, 34). How- <br /> sizes distinct types mycolic acids, alpha-, methoxy-, <br />                                                                                ever, recent work demonstrated single <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> and keto-mycolic acids C24 C26 C54 C63 side <br />                                                                                inhA reduces binding isoniazid-NAD adduct is <br /> chains. Synthesis mycolic acids involves types fatty <br />                                                                                sufficient confer isoniazid resistance (31). Interestingly, isonia- <br />                                                                                zid inhibits FabG1 vitro (11). <br />   * Corresponding author. Mailing address: Centre Infectious Dis-             We interested M. tuberculosis inhA operon, <br /> ease, Institute Cell Molecular Science, Barts London,           consists genes, fabG1 (mabA), inhA, hemZ, <br /> London E1 2AT, United Kingdom. Phone: 44 20 7882 2306. Fax: 44 20              cotranscribed polycistronic message (1). We have <br /> 7882 2189. E-mail: t.parish@qmul.ac.uk. <br />                                                                                previously demonstrated hemZ required heme bio- <br />   &#226;&#8364;&#160; Present address: Department Biosurgery &amp; Surgical Technol- <br /> ogy, Imperial College London, St. Mary&#226;&#8364;&#8482;s Hospital, Praed Street, Lon-          synthesis hemZ mutants hemin auxotrophs (22). <br /> don W2 1NY, United Kingdom.                                                    Although intuitively expect inhA fabG1 <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />   &#226;&#8364;&#161; Present address: Boehringer Ingelheim Vetmedica, Inc., 2621                essential genes, definitive genetic proof been <br /> North Belt Highway, Saint Joseph, MO 64506. <br />                                                                                obtained. The fact FAS II essential been <br />   &#194;&#167; Present address: Bill Melinda Gates Foundation, P.O. Box <br /> 23350, Seattle, WA 98102.                                                      demonstrated Mycobacterium smegmatis, inactiva- <br />   &#228;&#338;&#164; <br />     Published ahead print 2 March 2007.                                  tion InhA KasA leads cell lysis (4, 31). However, the <br />  <br />                                                                         3721 <br />  3722       PARISH ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />                                                                 TABLE 1. M. tuberculosis strains <br />  Name                                      Description                                                                   Genotype <br />  <br /> H37Rv                     Wild-type laboratory strain                                fabG1 <br /> Mess 4                    Single crossover                                           fabG1 fabG1&#226;&#338;&#172; kan hyg lacZ sacB <br /> Mess 10                   Merodiploid                                                fabG1 fabG1&#226;&#338;&#172; kan hyg lacZ sacB &#229;&#8230;&#179;fabG1 inhA hemZ gm L5 int&#229;&#8230;&#180; <br /> Mess 19                   Delinquent strain                                          fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ gm L5 int&#229;&#8230;&#180; <br /> Mess 25                   Switched strain (pDUCK1)                                   fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ kan L5 int&#229;&#8230;&#180; <br /> Mess 26                   Switched strain (pDUCK2)                                   fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ hyg L5 int&#229;&#8230;&#180; <br /> Mess 29                   Switched strain (pDUCK8)                                   fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 hyg L5 int&#229;&#8230;&#180; <br /> Mess 27                   Switched strain (M. smegmatis fabG1)                       fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabGMsm gm L5 int&#229;&#8230;&#180; <br /> Mess 30                   Cointegrated strain                                        fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ hyg L5 int&#229;&#8230;&#180;&#229;&#8230;&#179;fabGEcE233K,A154T gm L5 int&#229;&#8230;&#180; <br /> Mess 31                   Cointegrated strain                                        fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ hyg L5 int&#229;&#8230;&#180;&#229;&#8230;&#179;fabGEcA154T gm L5 int&#229;&#8230;&#180; <br /> Mess 32                   Cointegrated strain                                        fabG1&#226;&#338;&#172; &#229;&#8230;&#179;fabG1 inhA hemZ hyg L5 int&#229;&#8230;&#180;&#229;&#8230;&#179;fabGEcE233K gm L5 int&#229;&#8230;&#180; <br />  <br />  <br />  <br /> results saturating transposon mutagenesis M. tuberculosis                     single PacI site cloning inserts) generate plasmids pDUCK15, <br /> suggested kasA kasB essential growth,                        pDUCK16, pDUCK18, respectively. <br />                                                                                        The switching experiment replace integrated pMabA-HemZ-Int vector <br /> both inhA fabG1 (25). In addition, mycolic acid-                        various versions alternative integrating vectors carried previ- <br /> defective mutants M. smegmatis Mycobacterium bovis                           ously described (24). Mess 19 (Table 1) electroporated, transformants <br /> BCG isolated (3, 18), suggesting mycolic acids                       isolated appropriate antibiotic selection (for incoming vector). <br /> may essential cell viability. In order determine                      Transformants patch tested antibiotic resistance required. <br />                                                                                        Phenotypic analyses. Colony morphology assessed 7H10-OADC agar <br /> whether FAS II complex essential, investigated <br />                                                                                     plates (19 g liter&#226;&#171;&#186;1 Middlebrook 7H10, 10% [vol/vol] oleic acid-albumin-dex- <br /> whether key genes, fabG1, encodes 3-keto-                        trose-catalase [OADC] supplement [Becton Dickinson]) antibiotics. To <br /> acyl reductase (1), essential. We examined specificity                  determine resistance isoniazid, M. tuberculosis strains plated 7H10- <br /> of homologs fabG M. smegmatis Escherichia coli                         OADC agar plates containing various concentrations isoniazid (0, 0.01, 0.05, <br /> determine complement function                         0.1, 0.5, 1, 5 &#226;?&#174;g ml&#226;&#171;&#186;1), growth scored 7 days. Growth also <br />                                                                                     monitored 5 ml 7H9-OADC-Tw liquid medium (4.7 g liter&#226;&#171;&#186;1 Middlebrook <br /> M. tuberculosis enzyme.                                                             7H9, 10% [vol/vol] OADC supplement, 0.05% [wt/vol] Tween 80) containing <br />                                                                                     various concentrations isoniazid (0, 0.01, 0.05, 0.1, 0.5, 1, 5 &#226;?&#174;g ml&#226;&#171;&#186;1); the <br />                         MATERIALS AND METHODS                                       cultures stirred 250 rpm 8-mm flea, growth monitored <br />                                                                                     21 days. Detergent sensitivity determined 7H9-OADC-Tw liquid <br />    Gene replacement M. tuberculosis. A fabG1 deletion delivery vector was <br />                                                                                     medium 7H10-OADC agar plates. Growth curves obtained using <br /> constructed amplifying flanking regions fabG1 using primers fabG1-1 <br />                                                                                     12-mm-diameter borosilicate tubes containing 4 5 ml medium stirred <br /> (AAG CTT GTA CTC GGC GAC GAC C) fabG1-2 (GTT TCC TCC GGT <br />                                                                                     250 rpm 8-mm flea. <br /> AAC CAG) primers fabG1-3 (TAT ATC TCC GGT GCG GTC A) and <br />                                                                                        Mycolic acid analyses. Cultures grown late log phase 100 ml of <br /> fabG1-4 (AAC AGG TAC CCG CGT CCA ACC GTT CG). Primers fabG1-1 <br />                                                                                     7H9-OADC medium (containing Tween 80) autoclaved. Pellets were <br /> and fabG1-4 engineered carry HindIII KpnI sites (underlined). The <br />                                                                                     recovered extracted organic solvents (14). The bacterial residues ob- <br /> products cloned HindIII-KpnI sites p2NIL (23). The PacI <br />                                                                                     tained lipid extraction saponified mixture 40% KOH and <br /> cassette pGOAL19 (23) carrying hyg, lacZ, sacB inserted obtain <br />                                                                                     methoxyethanol (1:7, vol/vol) 110&#194;&#176;C 3 h screw-cap tube (9). After <br /> the final vector, pFabG1 (data available request). A two-step recombination <br />                                                                                     acidification, fatty acids extracted diethyl ether methylated an <br /> method used isolate single-crossover strain (Mess 4) double <br />                                                                                     ethereal solution diazomethane. The mycolate patterns strains were <br /> crossovers subsequently (23). Double crossovers screened PCR using <br />                                                                                     determined analytical thin-layer chromatography (TLC) Silica Gel 60 <br /> primers fabG1 (5&#226;&#172;&#732; CGG CCG CGG CGA GAC GAT AG 3&#226;&#172;&#732;) fabG1 rev <br />                                                                                     (Macherey-Nagel) using eluent A (petroleum ether-diethyl ether, 9:1 <br /> (5&#226;&#172;&#732; GGT CGC CGG CAG CCA GTC AGA 3&#226;&#172;&#732;). The complementing vector <br />                                                                                     [vol/vol]; runs) eluent B (dichloromethane). Lipid spots revealed by <br /> pMabA-HemZ-Int (22), carrying complete fabG1 operon gentamicin <br />                                                                                     spraying plates molybdophosphoric acid (10% ethanol), followed by <br /> resistance L5 integrating vector, used generate merodiploid strain <br />                                                                                     charring. The crude mycolate fraction obtained precipitating ethereal <br /> (Mess 10) (Table 1). Double crossovers isolated merodiploid <br />                                                                                     solution fatty acid methyl esters methanol 4&#194;&#176;C, followed centrifu- <br /> strain screened PCR described confirmed Southern <br />                                                                                     gation 4,000 &#226;&#171;&#187; g 20 min (10). The different classes mycolates were <br /> hybridization. <br />                                                                                     separated preparative TLC using eluent A. The lengths various types of <br />    Switching experiments &#226;&#8364;&#339;delinquent&#226;&#8364;? strain. Integrating vectors carrying <br />                                                                                     purified mycolates determined matrix-assisted laser desorption ioniza- <br /> the fabG1 operon different resistance markers constructed (data <br />                                                                                     tion&#226;&#8364;&#8221;time flight (MALDI-TOF) mass spectrometry previously described <br /> available request). pDUCK1 constructed cloning KpnI-HindIII <br />                                                                                     (17). The spectra acquired reflectron mode Applied Biosystems <br /> fragment pMabA-HemZ-Int pMV306 (kan) (28), pDUCK2 was <br />                                                                                     4700 Analyser mass spectrometer (Applied Biosystems, Framingham, MA) <br /> constructed replacing gm-int fragment pMabA-HemZ-Int the <br />                                                                                     equipped Nd:YAG laser (wavelength, 355 nm; pulse, &#226;&#172;?500 ps; repetition <br /> hyg-int HindIII cassette pUC-Hyg-Int (19). pDUCK8 carrying fabG1 <br />                                                                                     rate, 200 Hz). A total 2,500 shots accumulated positive ion mode, and <br /> was constructed amplifying M. tuberculosis gene primers Messy8 (5&#226;&#172;&#732; <br />                                                                                     mass spectrometry data acquired using instrument default calibration. <br /> GGT ACC ATG GAA ATC GAC TGG TCA GG 3&#226;&#172;&#732;) Messy9 (5&#226;&#172;&#732; AAG TCC <br />                                                                                     Mycolate samples dissolved chloroform concentration 1 mM and <br /> ATC GAC GAG TCG GTG ATG AT 3&#226;&#172;&#732;), subcloning product pGEM-T <br />                                                                                     directly spotted target plate 0.5-&#226;?&#174;l droplets, followed addi- <br /> Easy, adding hyg-int cassette pUC-Hyg-Int (19) HindIII <br />                                                                                     tion 0.5 &#226;?&#174;l matrix solution. Samples allowed crystallize room <br /> fragment. M. smegmatis fabG native promoter amplified from <br />                                                                                     temperature. The matrix used 2,5-dihydroxybenzoic acid (10 mg/ml) in <br /> genomic DNA using primers MAP3 (5&#226;&#172;&#732; GGT ACC CCG CGC TTC TGA TCA <br />                                                                                     CHCl3-CH3OH (1:1, vol/vol). <br /> ACC 3&#226;&#172;&#732;) MAP4 (5&#226;&#172;&#732; AAG TCC TGA CTC CTT GTG AGC GAG AA 3&#226;&#172;&#732;), <br /> cloned pGEM-T Easy (Promega), excised EcoRI fragment, cloned <br /> into L5 integrating vector pINT3 (22) obtain pDUCK9. E. coli fabG alleles <br /> were amplified plasmids pCL33, pCL79, pCL80 (16) using primers                                 RESULTS AND DISCUSSION <br /> Ec-FabG-for (5&#226;&#172;&#732; CCT TAA TTA AGC GCT CGA GCT TTA AAA GAG 3&#226;&#172;&#732;) <br /> and Ec-FabG-rev (5&#226;&#172;&#732; CCT TAA TTA ACA ACT AAA TCC CGG CAG GTC T <br />                                                                                       Essentiality M. tuberculosis fabG1. There much <br /> 3&#226;&#172;&#732;). Products cloned PacI fragments (sites primers underlined)      work understanding biosynthesis mycolic acids be- <br /> pAPA3 (a derivative pINT3 carrying mycobacterial Ag85a promoter          cause key role cell wall mycobacteria their <br />  VOL. 189, 2007                                                                   FUNCTIONAL COMPLEMENTATION OF fabG1                       3723 <br />  <br />  <br />  <br />  <br />   FIG. 1. Chromosomal arrangement fabG1 isolation mutant strains. (A) Chromosomal arrangement fabG1 M. tuberculosis. <br /> Regions amplified delivery vector complementing vector indicated bars. (B) PCR screening double crossovers isolated from <br /> merodiploid strain Mess 10. Primers fabG1 fabG1 rev used amplify fabG1 alleles; expected sizes 1.5 kb wild type <br /> and 0.9 kb deletion. Lane 1, 100-bp marker; lane 2, lambda HindIII marker; lanes 3 18, double crossovers generated Mess 10; lane <br /> 19, Mess 10 (merodiploid); lane 20, Mess 4 (single crossover). <br />  <br />  <br />  <br /> attractiveness drug targets. The target isoniazid, key            10 27 strains deletion allele (Fig. 2). The fact a <br /> antitubercular agent, mycolic acid biosynthesis (21). The              chromosomal deletion obtained presence <br /> FAS II required elongation mycolic acids             functional copy formal genetic proof fabG1 <br /> and predicted essential cellular component.              essential gene culture conditions. One <br /> The majority individual enzymes make                  &#226;&#8364;&#339;delinquent&#226;&#8364;? strain, Mess 19 (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ gm <br /> FAS II elongation identified (6, 7),          L5 int]) (Table 1), analyzed Southern hybridization and <br /> been confirmed InhA KasA required                   used study. <br /> cellular <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>survival</span> integrity M. smegmatis (4, 31). How-                Functional complementation M. smegmatis fabG. Differ- <br /> ever, contradictory findings raised            ent bacterial species produce structural variants mycolic <br /> the possibility certain enzymes FAS II cycle          acids. For example, M. smegmatis produces ethylenic &#226;?&#163;-, &#226;?&#163;&#226;&#172;&#732;-, <br /> be essential. These findings include prediction based              epoxy-mycolic acids different lengths, M. <br /> transposon mutagenesis inhA fabG1 required               tuberculosis synthesizes cyclopropanated &#226;?&#163;-, methoxy-, and <br /> for growth M. tuberculosis (25) isolation myco-             keto-mycolic acids. In addition, oxygenated mycolates are <br /> bacterial mutants defective mycolic acid synthesis (3, 18). In         longer M. tuberculosis M. smegmatis (32). It not <br /> addition, genes encoding potential homologs              known FAS II enzymes control chain <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, although <br /> FabG (FabG2 FabG5), overlapping func-                   probable structural limitation the <br /> tions, identified M. tuberculosis genome (7).            length fatty acid accommodated active <br />    In order determine fabG1 essential vitro,          site FAS II enzymes (20). In addition, protein-protein <br /> used two-step method try generate deletion strain               interactions FabG members FAS II <br /> (23). A suicide delivery vector containing unmarked in-                complex demonstrated (30). We wanted deter- <br /> frame deletion allele fabG1 constructed used                FabG enzyme plays role determining <br /> electroporate M. tuberculosis (Fig. 1). Since deletion allele         chain length mycolic acid type. We determined whether <br /> was unmarked frame, polar effects expression             M. smegmatis fabG gene complement fabG1 <br /> inhA hemZ expected. Single-crossover strains                 defect M. tuberculosis. <br /> were isolated entire vector incorporated                 We previously demonstrated high-efficiency replace- <br /> homologous recombination. One strain (Mess 4) (Table 1)               ment L5-based integrating vectors occurs M. tuberculosis <br /> verified Southern blotting used isolate double-             (22, 24). We used &#226;&#8364;&#339;switching&#226;&#8364;? technique test fabG ho- <br /> crossover strains. We screened 40 strains PCR                  mologues species order determine whether <br /> determine wild-type deletion allele fabG1.          functional M. tuberculosis. The basic premise the <br /> All wild-type allele, indicating            method efficiently replace resident inte- <br /> not isolate mutant. A merodiploid strain constructed             grated vector, carried functional copy fabG1, <br /> transforming single-crossover strain Mess 4 L5               transformation selection second version the <br /> integrating plasmid pMabA-HemZ-Int carrying M. tuber-                 integrating vector carrying alternative alleles. If incoming <br /> culosis complete operon (fabG1, inhA, hemZ)                  allele functional, obtain viable cells; was <br /> native promoter. In merodiploid background, able             not, switched strains obtained. <br /> to isolate wild-type deletion double-crossover strains;             First, confirmed switching method worked effi- <br />  3724     PARISH ET AL.                                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Genomic analyses recombinant strains. (A) Restriction maps chromosomal wild-type deletion alleles fabG1. The expected <br /> sizes fragments indicated maps positions probes used. (B) Switching M. smegmatis fabG. Genomic DNA <br /> was digested XhoI. Lanes 1 4, switched strains carrying M. smegmatis fabG M. tuberculosis fabG1 deleted; lane 5, wild type. <br /> fabG-msm, M. smegmatis fabG; fabG1-wt, wild-type fabG1. (C) Switching E. coli fabG. Genomic DNA digested BamHI. Lane 1, <br /> lambda HindIII marker; lanes 2 4, cointegrated strains Mess 30, Mess 31, Mess 32 carrying M. tuberculosis wild-type E. coli fabG genes. <br /> (D) PCR amplification E. coli fabG cointegrated strains. Lane 1, Mess 30; lane 2, Mess 31; lane 3, Mess 32; lane 4, lambda HindIII marker. <br /> fabG Ec, E. coli fabG. The expected product size (0.9 kb) product identity confirmed sequencing. <br />  <br />  <br />  <br /> ciently. Three plasmids carrying complete M. tuberculosis              confirming fabG1 required were <br /> fabG1-hemZ operon different resistance markers (gm, kan,               polar effects inhA hemZ. <br /> and hyg) used. Transformation delinquent strain                  Next, examined functional complementation M. smeg- <br /> Mess 19 (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ gm L5 int])               matis FabG. M. smegmatis fabG (fabGMsm) cloned its <br /> pDUCK1 (fabG1 inhA hemZ kan L5 int) pDUCK2 (fabG1 <br /> inhA hemZ hyg L5 int) vector resulted high-efficiency vector <br /> replacement (Table 2). To confirm vector replacement                     TABLE 2. Functional complementation M. tuberculosis with <br /> rather cointegration occurred, 24 samples                                         fabG homologs <br /> transformant patch tested gentamicin resistance,                                                                  Antibiotic   Transformation <br /> all gentamicin sensitive, confirming original plas-           Strain       Plasmid             Allele <br />                                                                                                                                selection      efficiencya <br /> mid lost. One transformant obtained switching <br />                                                                            Mess   19   pUC-Hyg-Int   None                     Hygromycin          10 <br /> plasmid selected study. The genotypes                   Mess   19   pDUCK1        M. tuberculosis operon   Kanamycin         1 &#226;&#171;&#187; 105 <br /> strains Mess 25 (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ kan L5 int])                  Mess   19   pDUCK2        M. tuberculosis operon   Hygromycin        2 &#226;&#171;&#187; 106 <br />                                                                            Mess   25   pDUCK8        M. tuberculosis fabG1    Hygromcyin      1.7 &#226;&#171;&#187; 105 <br /> Mess 26 (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ hyg L5 int]) con-                    Mess   26   pDUCK9        M. smegmatis fabG        Gentamicin        1 &#226;&#171;&#187; 105 <br /> firmed Southern hybridization (not shown). We con-                 Mess   26   pDUCK15       E. coli (Ts)             Gentamicinb          1 <br />                                                                            Mess   26   pDUCK16       E. coli                  Gentamicinb          1 <br /> firmed complementation performed using                   Mess   26   pDUCK18       E. coli (Ts)             Gentamicinb          2 <br /> fabG1 gene alone. Transformation Mess 25 (containing                   a <br />                                                                                 Transformation efficiencies plasmids expressed number of <br /> M. tuberculosis fabG1 [fabG1Mtb] operon) pDUCK8 (con-                 transformants &#226;?&#174;g plasmid DNA. <br /> taining fabG1Mtb) occurred high efficiency (Table 2),               b <br />                                                                                 Transformants isolated 30&#194;&#176;C. <br />  VOL. 189, 2007                                                                   FUNCTIONAL COMPLEMENTATION OF fabG1                        3725 <br />  <br />  <br />  <br />  <br />   FIG. 3. Multiple-sequence alignment fabG alleles. fabG genes M. tuberculosis H37Rv (Mtb), M. smegmatis (Msm), E. coli (Eco) <br /> were aligned using Clustal (www.ebi.ac.uk). <br />  <br />  <br /> own promoter integrating vector obtain plasmid                  acids synthesized using type II FAS system. Hence, ho- <br /> pDUCK9 (fabGMsm gm L5 int) transformed strain                     mologs mycobacterial FAS II enzymes, including FabG, <br /> Mess 26 (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ hyg L5 int]). Gentamicin-                 present E. coli (Fig. 3). We tested ability E. coli <br /> resistant transformants obtained high frequency (Ta-             fabG (fabGEc) complement M. tuberculosis fabG1. Vec- <br /> ble 2) tested determine sensitivities hygromycin           tors carrying fabGEc expressed mycobacterial pro- <br /> (24 24 transformants Gmr Hygs). PCR Southern                   moter constructed. We tested E. coli alleles, one <br /> analysis used confirm expected genotypes trans-             encoding protein normal activity (FabGA154T) and <br /> formants carrying deletion allele M. tuberculosis               encoding temperature-sensitive derivatives (FabGE233K <br /> fabG1 gene functional copy M. smegmatis fabG (Fig.                FabGE233K,A154T), nonpermissive <br /> 2). The results confirmed M. smegmatis gene able              42&#194;&#176;C E. coli (16). We reasoned E. coli allele was <br /> to complement function M. tuberculosis gene.                    functional, able construct temperature-sen- <br />   Lack functional complementation E. coli fabG. Fatty                sitive strain M. tuberculosis. Electroporation Mess 26 <br /> acid biosynthesis critical metabolic process bacteria.         (fabG1&#226;&#338;&#172; [fabG1 inhA hemZ hyg L5 int]) plasmids <br /> Although E. coli does produce mycolic acids, fatty               pDUCK15, pDUCK16, pDUCK18 resulted low <br />  <br />  <br />  <br />  <br />    FIG. 4. Growth recombinant M. tuberculosis strains: growth M. tuberculosis wild type (WT), Mess 25 ([fabG1 inhA hemZ]Mtb), Mess 27 <br /> (fabGMsm), Mess 29 (fabGMtb) liquid culture. The data averages &#226;&#171;&#190; standard deviations cultures. Growth curves constructed <br /> three times, similar results obtained. <br />    FIG. 5. Mycolic acid analyses: TLC mycolates (A) MALDI-TOF mass spectra purified &#226;?&#163;-mycolates (B), methoxy-mycolates (C), and <br /> keto-mycolates different strains M. tuberculosis expressing FabG1 M. tuberculosis (Mtb), M. smegmatis (Msm), E. coli (Eco). <br /> TLC performed eluent A (petroleum ether-diethyl ether, 9:1 [vol/vol]; runs). Lipid spots revealed spraying plates with <br /> molybdophosphoric acid (10% ethanol), followed charring. <br />  <br />  <br />                                                                       3726 <br />  VOL. 189, 2007                                                              FUNCTIONAL COMPLEMENTATION OF fabG1                                  3727 <br />  <br />  <br /> frequencies transformation organisms selected          Since enzymes FAS II pathway interact form <br /> on gentamicin 30&#194;&#176;C (Table 2). This suggested vector         complex (30), possibility. <br /> replacement occurred. PCR amplification using prim-            Conclusions. We confirmed fabG1 essential gene <br /> ers specific M. tuberculosis E. coli gene (Fig.   using robust genetic methods. This conclusion contrasts with <br /> 3D), followed sequencing, revealed case         prediction based transposon mutagenesis results. How- <br /> strain copy gene, indicating incoming        ever, transposon method predicts essentiality does <br /> plasmid cointegrated L5 site (which occurred      result percentage false negatives. For example, a <br /> very low frequency). Southern analysis confirmed presence       transposon insertion occurred near 3&#226;&#172;&#732; end gene, it <br /> of functional fabG1Mtb gene (Fig. 3C). The transformation         result partially fully functional proteins. Our <br /> procedure repeated, results obtained.         data confirm fabG1 essential fabG ho- <br /> These results indicated E. coli FabG complement         mologs able complement function normal <br /> a defect fabG1 gene M. tuberculosis. Since wild-      culture conditions. However, stage say <br /> type gene nonfunctional, able isolate tem-       lack expression genes a <br /> perature-sensitive mutants.                                         lack required enzymatic activity FabG proteins. <br />    Phenotypic consequences FabG replacement. Although            The equivalence M. smegmatis homolog demon- <br /> replacement native M. tuberculosis fabG1 gene M.      strated fact changes viability mycolic acid <br /> smegmatis fabG homolog resulted viable cells,       structure observed functional copy the <br /> possible changes mycolic acid biosynthesis and/or cell      cell fabGMsm. In contrast, E. coli fabG functional; <br /> wall structure occurred. We investigated number pheno-         unlikely related lack expression, since <br /> types indicated cell wall          provided mycobacterial promoter expression. It is <br /> changes mycolic acid profiles recombinant            likely structure E. coli enzyme able <br /> strains.                                                            accommodate long-chain fatty acids protein- <br />    No differences colony morphology sensitivity        protein interactions necessary FAS II to <br /> detergent sodium dodecyl sulfate isoniazid (in liquid media      function disrupted (20, 30). <br /> or solid media) strains carrying fabGMtb and <br /> strains carrying fabGMsm observed (data shown). In                                   ACKNOWLEDGMENTS <br /> addition, differences growth rate liquid medium          G.R. funded GlaxoSmithKline Action TB Initiative. The <br /> were observed (Fig. 4). The specificity FabG relation      work mycolic acids partially funded grant QLK2-CT-2000- <br /> the type, length, quantity mycolic acids synthesized     01761 European Community. <br /> was investigated. Strains carrying fabG1Mtb fabGMsm         We thank J. Cronan providing plasmids pCL33, pCL79, and <br />                                                                     pCL80. <br /> or fabG1Mtb fabGEc grown medium lacking <br /> Tween 80, total mycolic acid profiles generated (Fig.                                       REFERENCES <br /> 5). TLC analysis mycolates different strains         1. Banerjee, A., M. Sugantino, J. C. Sacchettini, W. R. Jacobs. 1998. The <br /> showed mycolates produced (Fig. 5A). The             mabA gene inhA operon Mycobacterium tuberculosis encodes a <br />                                                                         3-ketoacyl reductase fails confer isoniazid resistance. Microbiology <br /> three types mycolates purified various strains,        144:2697&#226;&#8364;&#8220;2704. <br /> and lengths determined MALDI-TOF mass spec-            2. Betts, J. C., A. McLaren, M. G. Lennon, F. M. Kelly, P. T. Lukey, S. J. <br /> trometry (17). Superimposable mass spectra obtained            Blakemore, K. Duncan. 2003. Signature gene expression profiles dis- <br />                                                                         criminate isoniazid-, thiolactomycin-, triclosan-treated Myco- <br /> the different &#226;?&#163;-mycolates (Fig. 5B), methoxy-mycolates (Fig.            bacterium tuberculosis. Antimicrob. Agents Chemother. 47:2903&#226;&#8364;&#8220;2913. <br /> 5C), keto-mycolates (Fig. 5D). The pseudomolecular mass          3. Bhakta, S., G. S. Besra, A. M. Upton, T. Parish, C. Sholto-Douglas-Vernon, <br /> (M&#226;&#171;&#185;Na)&#226;&#171;&#185; peaks observed spectra corresponded               K. J. C. Gibson, S. Knutton, S. Gordon, R. P. daSilva, M. C. Anderton, and <br />                                                                         E. Sim. 2004. Arylamine N-acetyltransferase required synthesis of <br /> peaks determined previously mycolates synthesized            mycolic acids complex lipids Mycobacterium bovis BCG represents <br /> the wild-type strain M. tuberculosis (strain H37Rv) (10). No         novel drug target. J. Exp. Med. 199:1191&#226;&#8364;&#8220;1199. <br />                                                                      4. Bhatt, A., L. Kremer, A. Z. Dai, J. C. Sacchettini, W. R. Jacobs. 2005. <br /> additional lipid spots structural changes transfor-           Conditional depletion KasA, key enzyme mycolic acid biosynthesis, <br /> mants detected TLC MALDI-TOF mass spec-                     leads mycobacterial cell lysis. J. Bacteriol. 187:7596&#226;&#8364;&#8220;7606. <br /> trometry (data shown).                                           5. Brennan, P. J. 2003. Structure, function, biogenesis cell wall of <br />                                                                         Mycobacterium tuberculosis. Tuberculosis 83:91&#226;&#8364;&#8220;97. <br />    These results indicated M. smegmatis FabG protein        6. Camus, J. C., M. J. Pryor, C. Medigue, S. T. Cole. 2002. Re-annotation <br /> is functionally equivalent M. tuberculosis FabG1 protein         genome sequence Mycobacterium tuberculosis H37Rv. Microbiology <br />                                                                         148:2967&#226;&#8364;&#8220;2973. <br /> and utilize endogenous substrates normally. Thus,            7. Cole, S. T., R. Brosch, J. Parkhill, T. Garnier, C. Churcher, D. Harris, S. V. <br /> FabG does define length fatty acid chains. In            Gordon, K. Eiglmeier, S. Gas, C. E. Barry, F. Tekaia, K. Badcock, D. <br /> addition, interference FabG1 function          Basham, D. Brown, T. Chillingworth, R. Connor, R. Davies, K. Devlin, T. <br />                                                                         Feltwell, S. Gentles, N. Hamlin, S. Holroyd, T. Hornby, K. Jagels, A. Krogh, <br /> cells contained E. coli enzyme.                          J. McLean, S. Moule, L. Murphy, K. Oliver, J. Osborne, M. A. Quail, M. A. <br />    Unusually, strain carrying complete M. tuberculosis          Rajandream, J. Rogers, S. Rutter, K. Seeger, J. Skelton, R. Squares, S. <br /> operon integrated vector (Mess 25) grew slower          Squares, J. E. Sulston, K. Taylor, S. Whitehead, B. G. Barrell. 1998. <br />                                                                         Deciphering biology Mycobacterium tuberculosis complete <br /> strain carrying fabG1 (Mess 29) grew (Fig. 4). The rea-           genome sequence. Nature 393:537&#226;&#8364;&#8220;544. <br /> son difference immediately obvious,      8. Daffe, M., P. Draper. 1998. The envelope layers mycobacteria with <br />                                                                         reference pathogenicity. Adv. Microb. Physiol. 39:131&#226;&#8364;&#8220;203. <br /> difference indicate relative amounts fabG1         9. Daffe, M., M.-A. Lane&#194;&#180;elle, C. Asselineau, V. Le&#194;&#180;vy-Fre&#194;&#180;bault, H. L. David. <br /> and inhA important; strain carrying complete                1983. Inte&#194;&#180;re&#203;&#8224;t taxonomique des acides gras des Mycobacte&#194;&#180;ries: proposition <br /> operon functional copies inhA                d&#226;&#8364;&#8482;une me&#194;&#180;thode d&#226;&#8364;&#8482;analyse. Ann. Microbiol. Inst. Pasteur 134:241&#226;&#8364;&#8220;256. <br />                                                                     10. Daffe&#194;&#180;, M., M.-A. Lane&#194;&#180;elle, P. L. Valero-Guillen. 1988. Tetraenoic and <br /> expected produce enzyme strain car-            pentaenoic mycolic acids Mycobacterium thamnopheos. Structure, tax- <br /> rying fabG1 (containing functional copy gene).          onomic biosynthetic implications. Eur. J. Biochem. 177:339&#226;&#8364;&#8220;344. <br />  3728        PARISH ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br /> 11. Ducasse-Cabanot, S., M. Cohen-Gonsaud, H. Marrakchi, M. Nguyen, D.                  22. Parish, T., M. Schaeffer, G. Roberts, K. Duncan. 2005. HemZ essen- <br />     Zerbib, J. Bernadou, M. Daffe, G. Labesse, A. Quemard. 2004. In vitro               tial growth Mycobacterium tuberculosis. Tuberculosis 85:197&#226;&#8364;&#8220;204. <br />     inhibition Mycobacterium tuberculosis beta-ketoacyl-acyl carrier pro-        23. Parish, T., N. G. Stoker. 2000. Use flexible cassette method to <br />     tein reductase MabA isoniazid. Antimicrob. Agents Chemother. 48:242&#226;&#8364;&#8220;                 generate double unmarked Mycobacterium tuberculosis tlyA plcABC mutant <br />     249.                                                                                    gene replacement. Microbiology 146:1969&#226;&#8364;&#8220;1975. <br /> 12. Hayashi, T., O. Yamamoto, H. Sasaki, H. okazaki, A. Kawaguchi. 1984.            24. Pashley, C. A., T. Parish. 2003. Efficient switching mycobacteriophage <br />     Inhibition fatty acid synthesis antibiotic thiolactomycin. J. Antibiot.       L5-based integrating plasmids Mycobacterium tuberculosis. FEMS Micro- <br />     37:1456&#226;&#8364;&#8220;1461.                                                                           biol. Lett. 229:211&#226;&#8364;&#8220;215. <br /> 13. Heym, B., P. M. Alzari, N. Honore, S. T. Cole. 1995. Missense mutations         25. Sassetti, C. M., D. H. Boyd, E. J. Rubin. 2003. Genes required for <br />     catalase-peroxidase gene, katG, associated isoniazid resis-             mycobacterial growth defined high density mutagenesis. Mol. Microbiol. <br />     tance Mycobacterium tuberculosis. Mol. Microbiol. 15:235&#226;&#8364;&#8220;245.                        48:77&#226;&#8364;&#8220;84. <br /> 14. Jackson, M., C. Raynaud, M. A. Laneelle, C. Guilhot, C. LaurentWinter, D.           26. Schaeffer, M. L., G. Agnihotri, C. Volker, H. Kallender, P. J. Brennan, and <br />     Ensergueix, B. Gicquel, M. Daffe. 1999. Inactivation antigen 85C             J. T. Lonsdale. 2001. Purification biochemical characterization the <br />     gene profoundly affects mycolate content alters permeability             Mycobacterium tuberculosis beta-ketoacyl-acyl carrier protein synthases <br />     Mycobacterium tuberculosis cell envelope. Mol. Microbiol. 31:1573&#226;&#8364;&#8220;                  KasA KasB. J. Biol. Chem. 276:47029&#226;&#8364;&#8220;47037. <br />     1587.                                                                               27. Slayden, R. A., R. E. Lee, J. W. Armour, A. M. Cooper, I. M. Orme, P. J. <br /> 15. Kremer, L., J. D. Douglas, A. Baulard, C. Morehouse, M. R. Guy, D. Alland,              Brennan, G. S. Besra. 1996. Antimycobacterial action thiolactomycin: <br />     L. G. Dover, J. H. Lakey, W. R. Jacobs, P. J. Brennan, D. E. Minnikin,              inhibitor fatty acid mycolic acid synthesis. Antimicrob. Agents <br />     G. S. Besra. 2000. Thiolactomycin related analogues novel anti-                  Chemother. 40:2813&#226;&#8364;&#8220;2819. <br />     mycobacterial agents targeting KasA KasB condensing enzymes                  28. Stover, C. K., V. F. Delacruz, T. R. Fuerst, J. E. Burlein, L. A. Benson, L. T. <br />     Mycobacterium tuberculosis. J. Biol. Chem. 275:16857&#226;&#8364;&#8220;16864.                             Bennett, G. P. Bansal, J. F. Young, M. H. Lee, G. F. Hatfull, S. B. Snapper, <br /> 16. Lai, C.-Y., J. E. Cronan. 2004. Isolation characterization &#226;?&#164;-                R. G. Barletta, W. R. Jacobs, B. R. Bloom. 1991. New use BCG for <br />     ketoacyl-acyl carrier protein reduc <br /> </body></html>