<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>REPORT <br />  <br />  <br />  <br />  <br /> RNA editing Drosophila melanogaster: New targets <br /> and functional consequences <br />  <br />  <br /> MARK STAPLETON, JOSEPH W. CARLSON, SUSAN E. CELNIKER <br /> Berkeley Drosophila Genome Project, Department Genome Biology, Life Sciences Division, Lawrence Berkeley National Laboratory, Berkeley, <br /> California 94720, USA <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Adenosine deaminases act RNA [adenosine deaminase, RNA specific (ADAR)] catalyze site-specific conversion of <br /> adenosine inosine primary mRNA transcripts. These re-coding events affect coding potential, splice sites, stability of <br /> mature mRNAs. ADAR essential gene, studies mouse, Caenorhabditis elegans, Drosophila suggest its <br /> primary function modify adult behavior altering signaling components nervous system. By comparing the <br /> sequence isogenic cDNAs genomic DNA, identified experimentally verified 27 new targets Drosophila <br /> ADAR. Our analyses led identify new classes genes transcripts targets ADAR, including components the <br /> actin cytoskeleton genes involved ion homeostasis signal transduction. Our results indicate editing Drosophila <br /> increases diversity proteome, does manner direct functional consequences protein function. <br /> Keywords: adenosine deaminase; Drosophila; RNA editing; ADAR; nervous system; transcriptome <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                   RNA editing drawn studies mammalian <br />                                                                                glutamate receptor gene, GluR-B. Receptor pre-mRNA <br /> RNA editing well-established mechanism which <br />                                                                                forms double-stranded stem structure imperfect <br /> precursor messenger RNA transcripts subject re- <br />                                                                                <span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span>-pairing exonic sequence, edit(s) <br /> coding enzyme adenosine deaminase, RNA specific <br />                                                                                occur, noncoding intronic element called editing <br /> (ADAR) (Bass 2002). ADAR catalyzes deamination of <br />                                                                                site complementary sequence (ECS) (Higuchi et al. 1993). <br /> adenosine inosine. Inosine mimics guanosine base- <br />                                                                                Most targets, however, serendipitously when <br /> pairing properties, translational machinery inter- <br />                                                                                isolating genes cDNA libraries. Computational iden- <br /> prets I G. In way, A-<span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span>-I conversion mRNA <br />                                                                                tification ADAR substrates difficult, little progress <br /> can alter genetic information lead changes in <br />                                                                                designing algorithms predicting <br /> mRNA splicing stability changes protein <br />                                                                                putative target transcripts. ADAR targets human <br /> function. ADAR essential animals examined date, <br />                                                                                genome identified located predomi- <br /> and studies mouse, Caenorhabditis elegans, Drosoph- <br />                                                                                nantly embedded Alu sequences, suggesting role of <br /> ila suggest function pre-mRNA editing to <br />                                                                                ADAR mRNA stability (Kim et al. 2004; Levanon et al. <br /> modify adult behavior altering signaling components in <br />                                                                                2004). Editing introduce new splice sites (Rueter <br /> the nervous (Higuchi et al. 2000; Palladino et al. <br />                                                                                et al. 1999), editing occurs 59- 39-untranslated <br /> 2000; Wang et al. 2000; Tonkin et al. 2002). Considerable <br />                                                                                regions (UTRs) presumably alters stability, localization, <br /> progress understanding mech- <br />                                                                                translatability target mRNA (Morse et al. 2002; <br /> anism action ADAR enzyme (Cho et al. 2003; <br />                                                                                Prasanth et al. 2005). Recently, Yang coworkers de- <br /> Haudenschild et al. 2004; Athanasiadis et al. 2005; Macbeth <br />                                                                                scribed additional role RNA editing biogenesis <br /> et al. 2005). The molecular details target site specificity of <br />                                                                                select miRNAs involved mammalian hematopoiesis <br />                                                                                (Yang et al. 2006). <br />    Reprint requests to: Mark Stapleton, Berkeley Drosophila Genome                The majority verified targets ADAR Drosophila <br /> Project, Department Genome Biology, Life Sciences Division, Lawrence        melanogaster come comparative genomic approach <br /> Berkeley National Laboratory, Berkeley, CA 94720, USA; e-mail: staple@         examining distinct subset genes consisting ion <br /> fruitfly.org; fax: (510) 486-6798. <br />    Article published online ahead print. Article publication date   channels, G-protein-coupled receptors (GPCRs), pro- <br /> at http://www.rnajournal.org/cgi/doi/10.1261/rna.254306.                       teins involved synaptic transmission (Hoopengardner <br />  <br /> 1922                                 RNA (2006), 12:1922&#226;&#8364;&#8220;1932. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2006 RNA Society. <br />                                                                                                          ADAR targets Drosophila <br />  <br />  <br />  <br /> et al. 2003). Using approach, con- <br /> servation putative ECS site within <br /> neighboring introns infor- <br /> mative predictor; however, exons with <br /> the highest evolutionary conservation <br /> within gene best <br /> predictive tool identify ADAR targets <br /> for subset genes. The prevalence <br /> of editing sites coding <br /> portions Drosophila genes, as <br /> the stage-specific &#226;&#8364;&#732;&#226;&#8364;&#732;self-tuning&#226;&#8364;&#8482;&#226;&#8364;&#8482; of <br /> ADAR, principal mechanism <br /> of increasing neuronal protein diversity <br /> (Keegan et al. 2005). <br />    Although directed approaches to <br /> identify ADAR targets FIGURE 1. A-to-G transitions head library cDNAs 2.5&#226;&#8364;&#8220;3-fold higher types <br /> employed successfully, taken base-pair substitutions libraries derived strain. Base-pair substitutions <br /> a different approach based system- identified grouped library expressed percent total number of <br /> atic analysis Drosophila Gene substitutions library. The embryo, larval/pupal (L/P), head cDNA libraries are <br />                                               derived isogenic strain used delineating genome sequence, the <br /> Collection (DGC; http://www.fruit- ovary, testes, cell-line S2 libraries not. Numbers library type correspond <br /> fly.org/DGC). The DGC contains total number base pairs (in parentheses, megabase pairs, Mb) total number of <br /> cDNAs representing 10,398 pre- substitutions library. <br /> dicted 13,449 genes D. melanogaster <br /> (Release 4.1). This collection originates cDNA librar-          RESULTS AND DISCUSSION <br /> ies derived variety tissues developmental <br /> stages (Rubin et al. 2000; Stapleton et al. 2002a,b). Utilizing      Identifying putative targets ADAR <br /> high-quality sequence data full-insert sequences of <br /> adult head cDNA clones, identified 27 new targets            In order identify putative new targets ADAR, took <br /> of ADAR, doubling total 55. The edited sites verified         advantage full-insert sequenced gene collection. As <br /> in analysis coding regions 39              cDNA analysis pipeline, aligned high- <br /> UTRs targets. Previous studies (Dutzler et al. 2002,        quality cDNA sequence genome sequence using <br /> 2003) targets (CG31116), member               Sim4 (Florea et al. 1998) recorded base-pair <br /> ClC family chloride channels, suggest clear functional          discrepancies (see Materials Methods). Figure 1 repre- <br /> consequence editing gating properties             sents results tabulating possible substitutions <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> ion channel. Some target genes, like G-protein-coupled           z15 Mb data full-insert sequenced cDNAs. When <br /> receptor bride-of-sevenless (boss) calcium-activated         substitutions grouped library expressed as <br /> potassium channel small conductance calcium-activated                percentage total, clones isogenic head <br /> potassium channel (SK), clear roles neuronal                 library possess threefold increase level A-to-G <br /> development signaling, target genes, like            transitions compared clones two <br /> spir, actin nucleating factor, represent compo-          isogenic libraries. Moreover, A &gt; G transition frequency <br /> nents cytoskeleton. The largest class ADAR targets         head libraries type substitution within <br /> consists novel genes assigned              isogenic libraries discernible frequency above <br /> a functional attribute Gene Ontology (GO) database            background substitutions. This analysis led generate <br /> (Ashburner et al. 2000).                                             list clones highest likelihood having an <br />    Transcriptional analyses human genome (Kapranov            A-to-G substitution editing reducing possi- <br /> et al. 2002), Arabidopsis genome (Yamada et al. 2003),           bility strain-dependent polymorphisms. This approach <br /> and Drosophila genome (Stolc et al. 2004) suggest           identifies authentic editing sites errors <br /> upward 50% predicted noncoding genome                   introduced reverse transcriptase used generate the <br /> expressed. In Drosophila, noncoding transcription           cDNA libraries. To distinguish events, exper- <br /> well characterized includes microRNAs (Lai et al. 2003;          imental validation sites necessary described <br /> Lai 2005; Leaman et al. 2005), noncoding RNAs (Tupy et al.           below. <br /> 2005), anti-sense RNAs (Misra et al. 2002). The                     The analysis resulted list 198 putative editing <br /> complexity eukaryotic transcriptomes continues ex-             targets. Most known edited genes D. melanogaster <br /> pand, editing pre-mRNA transcripts adds           involved rapid electrical chemical neurotrans- <br /> layer intricacy.                                                  mission (Stapleton et al. 2002a; Hoopengardner et al. 2003; <br />  <br />                                                                                                         www.rnajournal.org       1923 <br />  Stapleton et al. <br />  <br />  <br />  <br /> Xia et al. 2005). This observation is, part,     fly.org). To rule unlikely possibility mix of <br /> neurological <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> associated ADAR-<span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>null mutants</span>       A/G signals pure G signal seen RT-PCR was <br /> coupled fact genes type        genome polymorphisms accumulated the <br /> specifically evaluated presence editing. Although   isogenic strain, PCR-amplified sequenced same <br /> our approach biased manner, identified 11    regions genomic DNA isolated concomitantly the <br /> of known 28 targets ADAR putative targets did    RNA. None 58 sites polymorphisms the <br /> not pursue further. Additionally, 10 remaining     genomic DNA (Fig. 2B; Supplemental Material, contact <br /> 17 known target genes represented <br /> in collections, possess <br /> the A-to-G transition, suggesting that <br /> since editing &lt;100% efficient, we <br /> captured unedited mRNA our <br /> cDNA collection. The remaining seven <br /> known targets representa- <br /> tive EST cDNA gene collec- <br /> tion. The Sim4 alignment program <br /> encounters difficulties near edges of <br /> sequence alignments, putative tar- <br /> gets alignment artifacts were <br /> removed list, collapsing <br /> the set total 108 clones contain- <br /> ing 149 potential editing sites. <br />    We interrogated 149 putative <br /> editing sites experimentally perform- <br /> ing reverse-transcription&#226;&#8364;&#8220;polymerase <br /> chain reaction (RT-PCR) followed by <br /> sequencing amplified products. <br /> RT-PCR performed total RNA <br /> isolated adult heads iso- <br /> genic strain. Amplicons ranged size <br /> from z300 500 bp spanning the <br /> region(s) surrounding potential <br /> editing site(s). Sequence traces from <br /> 27 108 genes yielded 58 positions <br /> with mixed A G signal (n = <br /> 51) pure G signal (n = 7), in- <br /> dicating presence edited tran- <br /> scripts (Fig. 2B; Supplemental Material, <br /> contact staple@fruitfly.org). We success- <br /> fully amplified regions 108 <br /> putative targets obtained high-qual- <br /> ity sequencing data one <br /> (CG7569). For targets chromato- <br /> grams showed &#226;&#8364;&#732;&#226;&#8364;&#732;weak&#226;&#8364;&#8482;&#226;&#8364;&#8482; evidence edit- <br /> ing (predominant A putative <br /> editing site), cloned products <br /> from independent RT-PCR reac- <br /> tions sequenced 48 isolates re- <br /> action. CG31116 example this <br /> weak editing, four <br /> putative sites displayed mixed <br /> signal. Cloning revealed sites are <br /> edited frequencies ranging from <br /> 4.5% 10.4% (Supplemental Fig. S12 <br /> and table therein, contact staple@fruit-                           FIGURE 2. (Legend page) <br />  <br />  <br /> 1924      RNA, Vol. 12, No. 11 <br />                                                                                                                   ADAR targets Drosophila <br />  <br />  <br />  <br /> staple@fruitfly.org). Similar analysis performed                              The ion homeostasis class consists genes involved <br /> a subset 27 new targets using RNA isolated                          sodium calcium ion exchange calcium seques- <br /> ADAR-deficient animals, sites tested (n = 23) showed                      tration. Members group represent new class of <br /> a pure A signal, indicating unedited transcript (data                     molecules serve substrates ADAR include <br /> shown). Although method measuring ratio                            Calphotin (Cpn), Nckx30C, related CG1090, Na <br /> edited unedited transcripts reported accu-                     pump a-subunit (Atpa), CG32699. Atpa encodes <br /> rate low 5% editing (Palladino et al. 2000),                   sodium:potassium-exchanging ATPase undergoes <br /> be used determine relative distribution sites cases                  tyrosine-to-cysteine substitution editing. The <br /> where transcript contains edited site.                            Y390C substitution lies ATP hydrolase domain. <br />                                                                                   The Atpa protein predicted contain 10 transmem- <br />                                                                                   brane-spanning segments detected subset of <br /> New targets dADAR <br />                                                                                   medial lateral ventral cord neurons other <br /> The 27 new targets ADAR categorized seven                             larval adult Drosophila tissues (Lebovitz et al. 1989). <br /> classes genes, listed Table 1. The seven gene                     Animals carrying mutations Atpa behavioral <br /> classes include vesicular trafficking, ion homeostasis, signal                    abnormalities, reduced life span, severe neuronal <br /> transduction, ion channels, cytoskeletal components,                           hyperexcitability occurrence age-depen- <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;other&#226;&#8364;&#8482;&#226;&#8364;&#8482; class, novel unknown class. For                           dent neurodegeneration (Palladino et al. 2003). The two <br /> target, gene symbol, cDNA representative                             related genes Nckx30C CG1090 encode proteins <br /> initially suggested editing, protein, amino acid                          function potassium-dependent sodium&#226;&#8364;&#8220;calcium <br /> residue(s), 39-UTR nucleotide(s) affected,                           exchangers. Nckx30C critical rapid extrusion of <br /> along molecular function, known, listed.                          calcium expressed adult neurons during <br />    The vesicular traffic class targets contains genes,                     ventral nerve cord development embryo (Haug- <br /> Rab26, Rlip, rab3-GEF, endophilin A (endoA), a-Adaptin,                           Collet et al. 1999). Cpn encodes photoreceptor cell- <br /> and sunday driver (syd). Small Ras-like G-proteins                      specific calcium ion binding protein (Ballinger et al. <br /> effectors well-documented proteins function                       1993; Martin et al. 1993), CG32699 gene has <br /> intracellular transport vesicles (Takai et al. 2001). Three                    assigned GO functional attribute having &#226;&#8364;&#732;&#226;&#8364;&#732;calcium ion <br /> of members fall category repre-                         binding acyltransferase activity.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> sented GTPase Rab26, Ral GTPase activator                                 The signal transduction ion channel groups contain <br /> protein Rlip, Rab3 guanyl-nucleotide exchange                             members each. The genes Mob1 boss grouped as <br /> factor Rab3-GEF. The endoA a-Adaptin genes encode                             signaling molecules, genes SK CG31116 ion <br /> proteins involved endocytic pathway: endoA                        channels. The G-protein-coupled receptor boss two <br /> promotes synaptic vesicle budding, a-Adaptin sub-                        ion channels discussed greater next <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> AP-2 complex (Guichet et al. 2002; Jafar-Nejad                        section. Mob1 encodes highly conserved protein shown to <br /> et al. 2002; Seto et al. 2002). The protein encoded syd                    genetically physically interact Tricornered (trc), <br /> gene identified genetic screen Drosophila                         member Drosophila Nuclear Dbf2 related <br /> proteins involved axonal transport process (Bowman                         (Ndr) subfamily serine/threonine protein kinases (He <br /> et al. 2000). Flies carrying mutations syd reveal massive                      et al. 2005). trc required normal morphogenesis <br /> accumulations various axonal membrane-bound car-                               polarized outgrowths epidermal hairs, sensory <br /> goes, axonal transport motors kinesin-I                        bristles, arista laterals, sensory neuron dendrites (Geng <br /> within larval segmental nerves multiple independent                        et al. 2000; He Adler 2002; Emoto et al. 2004). <br /> syd alleles. Syd thought function cargo adaptor                     Furthermore, mutations Mob1 isolated behav- <br /> axonal transport interacting directly kinesin                         ioral screen genes involved long-term memory <br /> light chain.                                                                      (Dubnau et al. 2003). <br />                                                                                                             Another unique class edited target <br />                                                                                                          genes involved actin nucleation <br /> FIGURE 2. Structure editing boss. (A) The transmembrane protein bride sevenless (boss) is <br /> portrayed background predicted pass lipid bilayer seven times large organization, represented spire <br /> extracellular domain. An enlargement region edited boxed foreground. (spir), Ataxin 2 (Atx2), CG32809. <br /> Threonines 529 533 lie junction extracellular transmembrane spir belongs posterior group of <br /> domains, which, edited ADAR, recoded alanines. Representation boss visualized genes required properly <br /> with TMRPres2D (Spyropoulos et al. 2004), tool used modeling transmembrane proteins. (B) <br /> Sequence chromatograms edited region boss. The upper panel sequence trace RT- specifying axis developing <br /> PCR products boss wild-type animals shows mixed signal A G threonine 529 oocyte embryo (Manseau and <br /> when compared trace derived control genomic DNA (lower panel). Threonine 533 Schupbach 1989). spir mutants have <br /> appears completely edited displays pure G signal wild-type trace compared to <br /> the genomic trace. The larval chromatogram middle panel displays intermediate level of <br />                                                                                                          defect microtubule plus-end orien- <br /> editing positions, suggesting editing boss developmentally regulated. The open tation oogenesis (Wellington <br /> box protein translation edited threonine codons indicated.                    et al. 1999). Recently, gene product, <br />  <br />                                                                                                                 www.rnajournal.org         1925 <br />  Stapleton et al. <br />  <br />  <br />  <br />  TABLE 1. New targets ADAR editing <br />  <br />  Gene class           Symbol      cDNA        Protein         Residue changea             39-UTRb                Molecular function <br />  <br />  Vesicular traffic   Rab26     GH21984 CG7605-PA                    K365R                               GTPase (Yoshie et al. 2000) <br />                      Rlip      GH01995 CG11622-PA           I229V, E230G, K233E,      2478, 2480, 2482, Ral GTPase activator (Jullien-Flores <br />                                                                E254G, K265R              2486, 2740     et al. 2000) <br />                      rab3-GEF HL01222      CG5627-PA          Q2022R, S2054G                1306        Rab guanyl-nucleotide exchange factor <br />                                                                                                         (Lloyd et al. 2000) <br />                      endoA     GH12907 CG14296-PA               K129R, K137E                            Promotes synaptic vesicle budding <br />                                                                                                         (Verstreken et al. 2002) <br />                      a-AdaptinRH30202      CG4260-PA                T207A                               Component endocytosis, subunit of <br />                                                                                                         AP-2 (Gonzalez-Gaitan Jackle 1997) <br />                      syd       GH19969 CG8110-PA                    S983G                               Kinesin-dependent axonal transport <br />                                                                                                         (Bowman et al. 2000) <br />  <br />  Ion homeostasis     Cpn       GH08002 CG4795-PB                    S402G                              Ca+2 sequestration (Ballinger et al. <br />                                                                                                        1993; Martin et al. 1993) <br />                      Nckx30C HL01989, CG18860-PC                    K365R                              K+-dependent Na+, Ca2+ antiporter <br />                              GH04818                                                                   (Haug-Collet et al. 1999) <br />                      CG1090 GH23040 CG1090-PA               L357L, S358G, L387L                        K+-dependent Na+, Ca2+ antiporter <br />                                                                                                        (Webel et al. 2002) <br />                      Atpa      GH23483 CG5670-PD                    Y390C                              Na+, K+ exchanging ATPase (Palladino <br />                                                                                                        et al.2003) <br />                      CG32699 HL01250       CG32699-PA               I175M                              Ca+2 binding, acyltransferase activityc <br />  <br />  Signal transductionMob1       RH70633 CG11711-PD                  N91D                                Activator Trc kinase (He et al. 2005) <br />                     boss       GH10049 CG8285-PA                T529A, T533A                           G-protein-coupled receptor (Reinke and <br />                                                                                                        Zipursky 1988; Kramer et al. 1991) <br />  <br />  Ion channel         SK      GH16664 CG10706-PD                    Y377C                      2788     K+ channel (Kohler et al. 1996) <br />                      CG31116 GH23529 CG31116-PA                 K232R, T259A,                          Cl channel (Jentsch et al. 2005a) <br />                                                                 K268R, E269G <br />  <br />  Cytoskeletal        spir      GH13327 CG10076-PC                   K339R                              Actin nucleation factor (Quinlan et al. <br />    components                                                                                          2005) <br />                      Atx2      GH01409 CG5166-PA                K320R, K337R                           Regulator actin filament formation <br />                                                                                                        (Satterfield et al. 2002) <br />                      CG32245 GH04632, CG32245-PB                R296R, N297D             2225, 2248,   Structural constituent cytoskeletonc <br />                               GH25458                                                    2249, 2656 <br />  <br />  Other               CG32809 GH23439 CG32809-PB                     K179R                              ATP bindingc <br />                      retm    GH05975 CG9528-PA                      Q245R                              Phosphatidylinositol transporterc <br />  <br />  Unknown             CG1552 GH14443        CG1552-PA               K121R                               None <br />                      CG31531 GH25780       CG31531-PB              K679E                               None <br />                      CG3556 GH17087        CG3556-PA               I572V                               None <br />                      CG9801 GH23026        CG9801-PA              S345G                                None <br />                      l(1)G0196GH02989      CG14616-PC         Q1148R, S1172G,                          Noned <br />                                                                   Q1176R <br />                      CG12001 HL01040       CG12001-PA              I325V                               None <br />                      CG30079 HL05615       CG30079-PA      I127M, T303A, Q343R,                        None <br />                                                                Q358R, S360G <br />  a <br />   Corresponds amino acid position indicated protein. <br />  b <br />   Corresponds nucleotide position indicated cDNA clone. <br />  c <br />   Inferred function based Gene Ontology annotation. <br />  d <br />   Animals mutant alleles cause lethality second instar larval stages. <br />  <br />  <br />  <br /> Spir, shown nucleate actin filaments unique                served domains, FYVE domain Spir box, which <br /> way novel link actin organization                      shown involved localization on <br /> and intracellular vesicle transport (Kerkhoff et al. 2001;                  intracellular membranes possible association with <br /> Quinlan et al. 2005). The amino acid change Spir                         Rab11, respectively. Whether ADAR-modified residue <br /> introduced editing lies important con-                       Spir alters ability nucleate actin filaments to <br />  <br /> 1926       RNA, Vol. 12, No. 11 <br />                                                                                                 ADAR targets Drosophila <br />  <br />  <br />  <br /> be shown. Another target ADAR class, Atx2,   Functional consequences editing targets <br /> implicated actin nucleation polymerization, rather <br /> as possible regulator specific mRNAs function     Three notable targets ADAR integral membrane- <br /> mediate formation actin filaments. Atx2          spanning proteins encoded genes boss, CG31116, and <br /> Drosophila homolog human polyglutamine disease          SK. The G-protein-coupled receptor boss unique it <br /> gene SCA2, responsible dominantly inherited   acts ligand determine fate R7 photorecep- <br /> neurodegenerative disorder spinocerebellar ataxia type 2       tor cell Drosophila compound eye specific <br /> (Huynh et al. 2000; Satterfield et al. 2002). In Drosophila,   inductive interaction boss-bearing R8 photore- <br /> loss-of-function mutants Atx2 suggest principal    ceptor neuron R7 precursor cell (Fortini et al. 1992; <br /> function regulate formation bundling actin     Hart et al. 1993). Interestingly, edited sites convert <br /> filaments regulating subset RNA transcripts       threonines 529 533 alanines, lie junction <br /> encode mediators actin filament formation. Studies   large extracellular domain transmem- <br /> SCA2 homolog, ATX-2, C. elegans, demonstrate        brane domain (Fig. 2A). At larval stage, T533 codon <br /> forms complex poly(A)-bind- <br /> ing protein PAB-1 functions in <br /> translational regulation develop- <br /> ment germline (Ciosk et al. <br /> 2004). The member class is <br /> CG32245 placed through <br /> its associated GO functional attribute of <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;structural constituent cytoskeleton.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />     There genes comprise <br /> the &#226;&#8364;&#732;&#226;&#8364;&#732;other&#226;&#8364;&#8482;&#226;&#8364;&#8482; class, CG32809 realtime <br /> (retm). CG32809 encodes novel pro- <br /> tein classified GO proton- <br /> transporting ATPase. It contains <br /> a partial Actin interacting protein 3 <br /> domain (AIP3, pfam03915) identified <br /> using CD-Search engine the <br /> National Center Biotechnology In- <br /> formation (NCBI) (Marchler-Bauer <br /> and Bryant 2004). However, the <br /> K179R substitution editing of <br /> this transcript does lie near the <br /> AIP3 domain. The gene retm an <br /> assigned function phosphatidylino- <br /> sitol transporter protein contains <br /> SEC14 (cd00170), MSF1 (pfam04707), <br /> and CRAL_TRIO_N (pfam03765) <br /> domains. retm belongs novel family <br /> of mitochondrial proteins, expressed <br /> in developing midgut central <br /> nervous embryogenesis, <br /> and subcellularly localized mito- <br /> chondria (Dee Moffat 2005). <br />     The largest class genes identified <br /> has assigned function GO <br /> database contains seven members <br /> (Table 1). Although class does not <br /> have associated GO attributes, <br /> mutations exist mem- <br /> bers. Mutations CG14616 result in <br /> lethality second <br /> instar larval stages (Drysdale and <br /> Crosby 2005).                                                      FIGURE 3. (Legend page) <br />  <br />  <br />                                                                                               www.rnajournal.org      1927 <br />  Stapleton et al. <br />  <br />  <br />  <br /> is mix signals, adult, T533 codon                     anionic gate. We glutamate codon be <br /> contains pure G signal, indicating likely                       edited z10% adult head transcripts encode a <br /> predominant form boss adult fly (Fig. 2B).                          Gly residue, interestingly residue does not <br /> Furthermore, suggests editing boss sites                contain carboxyl chain capable closing ion <br /> is developmentally regulated. These substitutions                    pore consistent open pore conforma- <br /> an effect way R8 photoreceptor cell                       tion. Electrophysiologic crystallographic studies testing <br /> presents boss neighboring R7 cells, potentially modifying                  channel function support observation. Converting the <br /> the action boss later stages eye development.                        glutamate residue Torpedo ray ClC-0 channel Ala, <br />    The gene CG31116 encodes chloride channel belonging                      Gln, Val causes channel gate stay open (Dutzler et <br /> to highly conserved ClC chloride channel family. In                       al. 2003). Moreover, CG31116 expressed ventral <br /> humans, mutations ClC family members associated                        nerve cord, lateral cord glia, ventral midline, <br /> with underlying myotonia, Dent&#226;&#8364;&#8482;s disease, Bartter                         consistent neuronal function (Kearney et al. 2004). <br /> syndrome, osteoporosis, neurodegeneration, possibly                          Another striking example putative functional conse- <br /> epilepsy (Jentsch et al. 2005b). Some ion channels,                   quences RNA editing protein function transcript <br /> K+ channels, possess structurally distinct elements                      gene SK. This gene encodes voltage-independent <br /> perform selective conduction ions (filtering)                  ion channel activated submicromolar concentra- <br /> actual opening closing pore (gating) (Jiang et al.                  tions intracellular calcium. These channels high- <br /> 2002). Crystallographic electrophysiological studies                   affinity calcium sensors transduce fluctuations in <br /> the anionic ClC channels Escherichia coli Salmo-                     calcium concentrations changes membrane potential. <br /> nella typhimurium elegantly processes gating                 SK channels gated direct calcium binding, are <br /> and filtering closely tied (Dutzler et al. 2002)             heteromeric complexes calmodulin (CaM), where <br /> side-chain carboxyl group glutamate residue serves                    gating mediated binding calcium calmodulin <br /> the gate pore lying selectivity filter (Fig.                subsequent conformational alterations channel <br /> 3A,B). Obstructing pore chain closes                    protein (Xia et al. 1998). The edited form SK <br /> pore, binding Cl ion opens it. We identified                  transcript results tyrosine position 377 being <br /> four edited sites, affect residues                        converted cysteine. This Y377C substitution directly <br /> selectivity filter (Fig. 3C). One residues highly             highly conserved calmodulin-binding domain <br /> conserved glutamate residue chain acts                      (CaMBD) (Fig. 3D). The crystal structure CaMBD <br />                                                                                                    rat SK2 complexed apoCal- <br />                                                                                                    modulin reveals key region of <br /> FIGURE 3. Structure editing ClC SK ion channels. (A) Schematic ClC chloride <br /> channel conductance filter showing closed open conformations. The chloride ion CaMBD involved apoCaM binding <br /> represented red spheres, glutamate chain shown red. Notice closed consists 10 residues, form an <br /> configuration glutamate 148 chain occupying ion binding site, open a-helix binds C lobe of <br /> configuration, moved site replaced Cl ion. Reprinted (with <br />                                                                                                    CaM (Schumacher et al. 2001). Interest- <br /> permission AAAS &#195;&#8220; 2003) Dutzler et al. (2003). (B) Ribbon structure ClC <br /> conductance filter depicted closed configuration chloride ion shown red ingly, structure suggests van der <br /> sphere chain edited glutamate 148 colored red. The amino acids ends Waals contact chain of <br /> of a-helices, including glutamate 148 end helix F, brought near Y435 residues E H a- <br /> the membrane center form ion-binding site, shown light blue. The vestibules toward <br /> the interior exterior membrane depicted hatched domains. Reprinted <br />                                                                                                    helix CaM (Fig. 3E). Tyrosine 435 in <br /> (with permission Macmillan Publishers Ltd. &#195;&#8220; 2002, http://www.nature.com/nature/ rat SK2 channel cognate <br /> index.html) Dutzler et al. (2002). (C) Sequence alignment edited portion ClC tyrosine position 377 edited <br /> channels. Residues Cl selectivity filter represented gray shaded boxes, Drosophila SK channel. Whether <br /> identical residues indicated asterisk. The residues codons edited in <br /> D. melanogaster indicated black boxes. The a-helices depicted open boxes Y377C substitution functional <br /> the alignment. The amino acid sequences GenBank numbers Mus musculus ClC2, role ability calmodulin bind <br /> NP_034030.1; Oryctolagus cuniculus ClC2, AAB05937.1; Homo sapiens ClC2, AAB34722.2; channel affect the <br /> D. melanogaster GH23529p, AAM50193.1; D. melanogaster CG31116-PA, AAF54701.3; Anoph- transductive capacity calmodulin <br /> eles gambiae, XP_312021.2; S. typhimurium, AE008704. The sequence alignments were <br /> performed CLUSTALX (Thompson et al. 1997). (D) Sequence alignment remains tested. <br /> Calmodulin binding domain SK channels. The tyrosine 377 residue codon edited              Editing D. melanogaster thought <br /> in D. melanogaster indicated black box. Identical residues indicated asterisk, gene sparing strategy. That is, <br /> and Calmodulin binding domain depicted open box alignment. The different form protein is <br /> amino acid sequences GenBank numbers D. melanogaster CG10706-PD, NP_726987.1; <br /> D. melanogaster GH16664p, AAM50183.1; A. gambiae, XP_314474.2; M. musculus SK1, used different times animal&#226;&#8364;&#8482;s <br /> AAK48900.1; H. sapiens SK1, AAH75037.3; H. sapiens SK2, AAP45946.1. The sequence life cycle perform similar functions <br /> alignments performed CLUSTALX (Thompson et al. 1997). (E) Ribbon structure facilitated differential <br /> of rat SK2 calmodulin binding domain (colored light blue) complexed the <br />                                                                                                    activities ADAR devel- <br /> apoCalmodulin C lobe (in purple). Notice tyrosine 435 (colored red) SK2, is <br /> the highly conserved tyrosine 377 D. melanogaster, involved interaction opment (Keegan et al. 2005). Our <br /> apoCalmodulin. Reprinted (with permission Elsevier &#195;&#8220; 2004) Schumacher et al. (2004). observations, including example of <br />  <br />  <br /> 1928       RNA, Vol. 12, No. 11 <br />                                                                                                              ADAR targets Drosophila <br />  <br />  <br />  <br />  TABLE 2. Primer pairs RT-PCR <br />  <br />  Gene                                                                 Sense primer/antisense primers RT-PCR <br />  <br />  CG10076                                  59-CAGAGCAACCTGGACGAGA-39/59-TTCGCATCTTGGTATAGCA-39 <br />  CG10706                                  59-CTCTAATGACAATATGTCCGGG-39/59-TGTGTTTGCATTATCCATTAGT-39 <br />  CG1090                                   59-CTGGTTTCCTGGTACGTGGT-39/59-TATTACGTTGACGTGCGAGC-39 <br />  CG11622                                  59-AGTGCTGGCACTTGGGTATC-39/59-ATTGCACTTGGGCAATTGTT-39 <br />  CG11711                                  59-GAGTTCTCCTCCATTAGCCAA-39/59-AGTCCAAACCAGCTGGAAG-39 <br />  CG12001                                  59-TTTCTTGACCTGTTGGAGGC-39/59-TGTCGTAGCCAAATCCCTTC-39 <br />  CG14296                                  59-GATCAACAAGGCCAACCAGT-39/59-TGGCTCCAAAAAGTTCTGCT-39 <br />  CG14616                                  59-CTAAACGAGGAGCTGGCATC-39/59-TTGCTAAAACCACCGAAACC-39 <br />  CG1552                                   59-GCCTCGTCCTTTTGAGCCT-39/59-TGAGGAGCGAGTTATTGGG-39 <br />  CG18860                                  59-AACTCGACCACCACGAAAAC-39/59-GATCAGCACCAGCAAACTGA-39 <br />  CG30079_1                                59-TGCGAAACCCTCCAGAAAC-39/59-CGGTGAGGGATACGTTTGCT-39 <br />  CG30079_2                                59-CGCGTTACCAGCCCATCC-39/59-TGATCTGGGGTATGGATAGCG-39 <br />  CG31116                                  59-GATCTGACGTCACAGCCTTTTG-39/59-TGGAATGATGTGACAAGTTTAC-39 <br />  CG31531                                  59-GCGGTCAACCAAATTGAA-39/59-CTGGGTAGGAATCGGAGT-39 <br />  CG32245                                  59-TATCTTCCGTTTGGGTCGTC-39/59-CGTCATACGCTCGCTGAGTA-39 <br />  CG32699                                  59-CAAATGCAATACATGACGGC-39/59-GGCGTAATACTCCAGGTCCA-39 <br />  CG32809                                  59-GAAAACTCTGGAACGACCG-39/59-TTCCGTTCTGAAGAGTACA-39 <br />  CG3556                                   59-CCGAAATGGATCCCAGATT-39/59-TTGATTACGCCATTATTAT-39 <br />  CG4260                                   59-TTCCAACGAAATACCCAAGC-39/59-GTAGTTTTGCAGCAATCGCA-39 <br />  CG4795                                   59-GTGCTTCGACTGAGCCCC-39/59-GCCGTAGTTGGTGGAGTA-39 <br />  CG5166                                   59-ACCGCAGCTATCGCATGT-39/59-CGCCCATGTGAGATTGGT-39 <br />  CG5627                                   59-GGCCATTTCACGTAGCGG-39/59-CACAGCCAGGGGATACAGA-39 <br />  CG5670                                   59-GGCTGGACGCTGTCATCTT-39/59-CGGCAATCTTCTTATTACG-39 <br />  CG7605                                   59-GAGTACGCGCAGGAGGAC-39/59-CGGGCTTATTGTTGTCTT-39 <br />  CG8110                                   59-AATCGCTGAACCCCATACTG-39/59-GAGGCAACGTATCGAATGGT-39 <br />  CG8285                                   59-CCCATGATACGGCTGAGAT-39/59-TGCTCACCCACGTATTCAA-39 <br />  CG9528                                   59-GATCAGCAGCATAGCATCTTG-39/59-CGGCGGATTCATTGATCTTCT-39 <br />  CG9801                                   59-TGCGACTTCGCATATGGT-39/59-CTCAGCTCTGGCTTGTTG-39 <br />  CG11622_UTR                              59-CCTCCTTGGCCTCGGGAT-39/59-CGCCGTTTAACAGTAATTATAGTTTGAA-39 <br />  CG5627_UTR                               59-ATCGATGTCATCATCATCCG-39/59-GGACTTACAAATGTTTTAACAATTTTGAA-39 <br />  CG10706_UTR                              59-AACGGCCATCACATTTCTGT-39/59-CCAATGATATTTCCCCATTCC-39 <br />  CG32245_UTR1                             59-CCGCAGCAGCAAGATCAA-39/59-AGATCCCTCGTTTCTTTCTTGA-39 <br />  CG32245_UTR2                             59-TTGAGTTACTATCCTTTAGGAAATGCT-39/59-TGTCAACAGTTCATTGAATATTTTGG-39 <br />  CG32245_UTR3                             59-AACCAGTTTTGAAAGTCTTGAATTG-39/59-AATTAAGGTGTCAATTGGTAAAACAGT-39 <br />  CG32245_UTR4                             59-AATACAACTGTTAATCTGTTTCCACTG-39/59-GCTCTTTTGGATACCAAAAAAGTTGAAGAAAT-39 <br />  <br />  <br />  <br />  <br /> the effect Cl gating, possible developmental                     genome sequence Release 4.1 computed gene models. <br /> alteration activity boss ligand,            Using Sim4 (Florea et al. 1998), recorded A&gt;G substitu- <br /> possible functional implications editing 39 UTRs,             tions, manually inspected clone sequence quality <br /> suggest greater role editing modulating                   discrepancies Sim4 artifacts using Consed Autofinish (Gor- <br />                                                                         don et al. 2001). This produced curated list 108 genes that <br /> specific protein function transcript stability. It very <br />                                                                         chosen candidates experimental testing. <br /> likely targets editing exist Drosophila <br /> because bias sampling method limited                  Validation targets <br /> the abundant mRNAs represented cDNA libraries <br /> coupled observation editing 100%                   We used similar method RNA isolation RT-PCR <br /> efficient. In summary, analysis identified 38 editing          described previously (Stapleton et al. 2002a). RNA genomic <br /> targets, 27 newly discovered, doubles total                   DNA isolated time using Trizol reagent <br />                                                                         (Invitrogen) adult heads isogenic strain y1; cn1 bw1 <br /> number editing targets D. melanogaster 55. <br />                                                                         sp1. For boss analysis, isolated RNA 0&#226;&#8364;&#8220;24-h embryos, <br />                                                                         mixed stage larvae, pupae isogenic strain. RT- <br />                                                                         PCR amplicons boss region embryo pupal RNA <br /> MATERIALS AND METHODS                                                   negative, positive larva RNA. RNA preps were <br />                                                                         digested RNAase-free DNAase I (Roche) tested PCR <br /> Identification putative ADAR editing targets                         ensure absence contaminating genomic DNA. We <br />                                                                         designed gene-specific primers RT-PCR genomic <br /> We compared nucleotide sequence full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> cDNA             PCR amplifications using Primer3 (Rozen Skaletsky 2000) <br /> clones derived libraries adult head tissue Release 4     primer-picking feature Consed Autofinish. The <br />  <br />                                                                                                            www.rnajournal.org      1929 <br />  Stapleton et al. <br />  <br />  <br />  <br />  TABLE 3. Primer pairs genomic PCR <br />  <br />  Gene                                                           Sense primer/antisense primers genomic PCR <br />  <br />  CG10076                                 59-TAAGAGAAGTCCTGCGGAGC-39/59-ATCAACGACTTGGGGAACTG-39 <br />  CG10706                                 59-GCAGTTGACGAAACGGGTAT-39/59-ATTTCCCCATTCCAAAGACC-39 <br />  CG1090                                  59-TCAGGAAAACATGGGCTACC-39/59-TATTACGTTGACGTGCGAGC-39 <br />  CG11622                                 59-CCCACAAGGTCAAGGAGAAA-39/59-ATTGCACTTGGGCAATTGTT-39 <br />  CG11711                                 59-GCCACTCCTTACCGACAGAA-39/59-AGCGTTTAGTTCCCCCAGTT-39 <br />  CG12001                                 59-GCAATGCATTTGGTTTGATG-39/59-AGCTACCATACTTCCGCCCT-39 <br />  CG14296                                 59-GATCAACAAGGCCAACCAGT-39/59-CCACGAATCTCATCGTCCTT-39 <br />  CG14616                                 59-GAGACATCGCTCCTACAGCC-39/59-CCGGAAGAGGAGACCGTTAT-39 <br />  CG1552                                  59-TTGGTTCACAAATCGCAGAA-39/59-CACCCTAACCCAACCTAGCA-39 <br />  CG18860                                 59-CTCCCGATTAAAGCCATCAA-39/59-CTAGCTTCCTGCTAATGCCG-39 <br />  CG30079_1                               59-AGAGCCCGTCAGTGAAGTGT-39/59-AAGGAGTGCGAAGAGAACGA-39 <br />  CG30079_2                               59-TGCTGCTCTTGATGTTCGAC-39/59-ACACTTCACTGACGGGCTCT-39 <br />  CG31116                                 59-CCGCCAACATTTCAGAGTTT-39/59-GCACCTCCAATCTGAGAACC-39 <br />  CG31531                                 59-ATTGCGGGACATGGAATTTA-39/59-TTAAAAATCTCCACACGCCC-39 <br />  CG32245                                 59-CGGCTATAAAGTGCACAGCA-39/59-GCTCGACGAAGTCGCTAGAT-39 <br />  CG32699                                 59-CAGCCAGAGAAGGCGTAATC-39/59-CAAATGCAATACATGACGGC-39 <br />  CG32809                                 59-AAGCCCTACATCCTCACCCT-39/59-ATCATGTCGGAGGTGGAAAC-39 <br />  CG3556                                  59-GGATCCCAGGTAAAACAGCA-39/59-ACCATTAGCTCGTCCACACC-39 <br />  CG4260                                  59-GTAGTTTTGCAGCAATCGCA-39/59-TTCCAACGAAATACCCAAGC-39 <br />  CG4795                                  59-TGTTGTAACTTCGACAGGCG-39/59-ACCTGTTGTTGCAGAAACCC-39 <br />  CG5166                                  59-GGTGACTTCCAGTTGAGGGA-39/59-TTGCGAGTTACCCTGGTAGG-39 <br />  CG5627                                  59-TACACCGCCACAGTATACGC-39/59-CACAGCCAGGGGATACAGAT-39 <br />  CG5670                                  59-GATCCACCATTTCATCCACC-39/59-GTTCCATGCACTTGAGCAGA-39 <br />  CG7605                                  59-TGTGGAGCTGTCCTTCACAG-39/59-ATCGGGAGGATATGGCTCTT-39 <br />  CG8110                                  59-TACGTACGGCTCAATGTCCA-39/59-ACTGGAGACCAAGGATCGTG-39 <br />  CG8285                                  59-AGATTGCGTCGCAGAACTTT-39/59-GGGTATTCAGAGTCTGGGCA-39 <br />  CG9528                                  59-AAGTGTTCCACTACCACGGC-39/59-TGAAGCAGTACACCCAGACG-39 <br />  CG9801                                  59-TTAGTTCCCGTACTCCGTGC-39/59-ATGGTGTCAACTGGGGTGAT-39 <br />  CG11622_UTR                             59-TGGGCATGGAACTAAAGGAG-39/59-AGATTTCTTTGGGGCTTTGC-39 <br />  CG5627_UTR                              59-CTTGTCATCATCCACATCGC-39/59-ACATGGATTAGGAAAGCCCC-39 <br />  CG10706_UTR                             59-GCAGTTGACGAAACGGGTAT-39/59-CCAATGATATTTCCCCATTCC-39 <br />  CG32245_UTR1                            59-CCGCAGCAGCAAGATCAA-39/59-AGATCCCTCGTTTCTTTCTTGA-39 <br />  CG32245_UTR2                            59-TTGAGTTACTATCCTTTAGGAAATGCT-39/59-TGTCAACAGTTCATTGAATATTTTGG-39 <br />  CG32245_UTR3                            59-AACCAGTTTTGAAAGTCTTGAATTG-39/59-AATTAAGGTGTCAATTGGTAAAACAGT-39 <br />  CG32245_UTR4                            59-AATACAACTGTTAATCTGTTTCCACTG-39/59-GCTCTTTTGGATACCAAAAAAGTTGAAGAAAT-39 <br />  <br />  <br />  <br />  <br /> sense/antisense primer pair sequences regions 27          dependent RT-PCR reactions picked 48 clones for <br /> validated genes shown Tables 2 3                          sequence analysis. <br />    Previously, cloned PCR products multiple RT reac- <br /> tions order evaluate potential editing. However, in <br /> this report, directly sequenced PCR products. For sequenc-    ACKNOWLEDGMENTS <br /> ing, PCR products treated 0.3 mL Exonuclease I <br /> (USB), 0.5 mL Shrimp Alkaline Phosphatase (Roche), 0.5        We thank Gerry Rubin providing support encouragement; <br /> mL 1 M Tris (ph 8.0) final reaction volume 75 mL      Ken Wan, Soo Park, Richard Weiszmann excellent <br /> incubated 45 min 37&#194;&#176;C heat-inactivated 10 min    technical support; Roger Hoskins critically reading and <br /> at 65&#194;&#176;C. The sense antisense primers used RT-PCR        improving manuscript. This work supported NIH <br /> and genomic PCR used sequencing primers 10-mL           grant HG002673 (to S.E.C.) US Department Energy <br /> reaction containing 1 mL ABI Big Dye III terminator mix           Contract No. DE-AC02-05CH11231. <br /> (Applied Biosystems). Sequencing reactions precipitated <br /> according manufacturer&#226;&#8364;&#8482;s protocol loaded          Received August 7, 2006; accepted August 24, 2006. <br /> ABI Prism 3730 DNA Analyzer. Sequence data assembled <br /> and analyzed Phred/Phrap suite programs and <br /> visualized Consed. Chromatogram trace peaks positions        REFERENCES <br /> chosen interrogation displayed unambiguous mixture       Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., <br /> of A G signals pure G signal recorded            Cherry, J.M., Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., <br /> a positive target ADAR. For RT-PCR amplicons           et al. The Gene Ontology Consortium. 2000. Gene Ontology: Tool <br /> ambiguous weak, cloned PCR products in-                  unification biology. Nat. Genet. 25: 25&#226;&#8364;&#8220;29. <br />  <br />  <br /> 1930      RNA, Vol. 12, No. 11 <br />                                                                                                                      ADAR targets Drosophila <br />  <br />  <br />  <br /> Athanasiadis, A., Placido, D., Maas, S., Brown II, B.A., Lowenhaupt, K.,    He, B. Adler, P.N. 2002. The genetic control arista lateral <br />     Rich, A. 2005. The crystal structure Zb domain            morphogenesis Drosophila. Dev. Genes Evol. 212: 218&#226;&#8364;&#8220;229. <br />     RNA-editing enzyme ADAR1 reveals distinct conserved surfaces            He, Y., Emoto, K., Fang, X., Ren, N., Tian, X., Jan, Y.N., and <br />     Z-domains. J. Mol. Biol. 351: 496&#226;&#8364;&#8220;507.                                Adler, P.N. 2005. Drosophila Mob family proteins interact with <br /> Ballinger, D.G., Xue, N., Harshman, K.D. 1993. A Drosophila                 related tricornered (Trc) warts (Wts) kinases. Mol. Biol. <br />     photoreceptor cell-specific protein, calphotin, binds calcium           Cell 16: 4139&#226;&#8364;&#8220;4152. <br />     contains leucine zipper. Proc. Natl. Acad. Sci. 90: 1536&#226;&#8364;&#8220;1540.        Higuchi, M., Single, F.N., Kohler, M., Sommer, B., Sprengel, R., and <br /> Bass, B.L. 2002. RNA editing adenosine deaminases act                Seeburg, P.H. 1993. RNA editing AMPA receptor subunit GluR- <br />     RNA. Annu. Rev. Biochem. 71: 817&#226;&#8364;&#8220;846.                                       B: A base-paired intron&#226;&#8364;&#8220;exon structure determines position and <br /> Bowman, A.B., Kamal, A., Ritchings, B.W., Philp, A.V., McGrail, M.,             efficiency. Cell 75: 1361&#226;&#8364;&#8220;1370. <br />     Gindhart, J.G., Goldstein, L.S. 2000. Kinesin-dependent axonal      Higuchi, M., Maas, S., Single, F.N., Hartner, J., Rozov, A., Burnashev, N., <br />     transport mediated Sunday driver (SYD) protein. Cell              Feldmeyer, D., Sprengel, R., Seeburg, P.H. 2000. <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Point mutation</span> <br />     103: 583&#226;&#8364;&#8220;594.                                                               AMPA receptor gene rescues lethality mice deficient the <br /> Cho, D.S., Yang, W., Lee, J.T., Shiekhattar, R., Murray, J.M.,              RNA-editing enzyme ADAR2. Nature 406: 78&#226;&#8364;&#8220;81. <br />     Nishikura, K. 2003. Requirement dimerization RNA editing         Hoopengardner, B., Bhalla, T., Staber, C., Reenan, R. 2003. <br />     activity adenosine deaminases acting RNA. J. Biol. Chem.              Nervous targets RNA editing identified comparative <br />     278: 17093&#226;&#8364;&#8220;17102.                                                           genomics. Science 301: 832&#226;&#8364;&#8220;836. <br /> Ciosk, R., DePalma, M., Priess, J.R. 2004. ATX-2, C. elegans        Huynh, D.P., Figueroa, K., Hoang, N., Pulst, S.M. 2000. Nuclear <br />     ortholog ataxin 2, functions translational regulation          localization inclusion body formation ataxin-2 not <br />     germline. Development 131: 4831&#226;&#8364;&#8220;4841.                                       necessary SCA2 pathogenesis mouse human. Nat. Genet. <br /> Dee, C.T. Moffat, K.G. 2005. A novel family mitochondrial                26: 44&#226;&#8364;&#8220;50. <br />     proteins represented Drosophila genes slmo, preli-like    Jafar-Nejad, H., Norga, K., Bellen, H. 2002. Numb: &#226;&#8364;&#732;&#226;&#8364;&#732;Adapting&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />     real-time. Dev. Genes Evol. 215: 248&#226;&#8364;&#8220;254.                                   notch endocytosis. Dev. Cell 3: 155&#226;&#8364;&#8220;156. <br /> Drysdale, R.A. Crosby, M.A. 2005. FlyBase: Genes gene               Jentsch, T.J., Neagoe, I., Scheel, O. 2005a. CLC chloride channels <br />     models. Nucleic Acids Res. 33: D390&#226;&#8364;&#8220;D395.                                   transporters. Curr. Opin. Neurobiol. 15: 319&#226;&#8364;&#8220;325. <br /> Dubnau, J., Chiang, A.S., Grady, L., Barditch, J., Gossweiler, S.,          Jentsch, T.J., Poet, M., Fuhrmann, J.C., Zdebik, A.A. 2005b. <br />     McNeil, J., Smith, P., Buldoc, F., Scott, R., Certa, U., et al. 2003.       Physiological functions CLC Cl channels gleaned human <br />     The staufen/pumilio pathway involved Drosophila long-term             genetic disease mouse models. Annu. Rev. Physiol. 67: 779&#226;&#8364;&#8220;807. <br />     memory. Curr. Biol. 13: 286&#226;&#8364;&#8220;296.                                        Jiang, Y., Lee, A., Chen, J., Cadene, M., Chait, B.T., and <br />                                                                                 MacKinnon, R. 2002. The open pore conformation potassium <br /> Dutzler, R., Campbell, E.B., Cadene, M., Chait, B.T., and <br />                                                                                 channels. Nature 417: 523&#226;&#8364;&#8220;526. <br />     MacKinnon, R. 2002. X-Ray structure ClC chloride channel <br />     3.0 A&#203;&#353; reveals molecular basis anion selectivity. Nature      Jullien-Flores, V., Mahe, Y., Mirey, G., Leprince, C., Meunier- <br />     415: 287&#226;&#8364;&#8220;294.                                                               Bisceuil, B., Sorkin, A., Camonis, J.H. 2000. RLIP76, an <br />                                                                                 effector GTPase Ral, interacts AP2 complex: <br /> Dutzler, R., Campbell, E.B., MacKinnon, R. 2003. Gating the <br />                                                                                 Involvement Ral pathway receptor endocytosis. J. Cell <br />     selectivity filter ClC chloride channels. Science 300: 108&#226;&#8364;&#8220;112. <br />                                                                                 Sci. 113: 2837&#226;&#8364;&#8220;2844. <br /> Emoto, K., He, Y., Ye, B., Grueber, W.B., Adler, P.N., Jan, L.Y., and <br />                                                                             Kapranov, P., Cawley, S.E., Drenkow, J., Bekiranov, S., Strausberg, R.L., <br />     Jan, Y.N. 2004. Control dendritic branching tiling the <br />                                                                                 Fodor, S.P., Gingeras, T.R. 2002. Large-scale transcriptional <br />     Tricornered-kinase/Furry signaling pathway Drosophila sensory <br />                                                                                 activity chromosomes 21 22. Science 296: 916&#226;&#8364;&#8220;919. <br />     neurons. Cell 119: 245&#226;&#8364;&#8220;256. <br />                                                                             Kearney, J.B., Wheeler, S.R., Estes, P., Parente, B., Crews, S.T. <br /> Florea, L., Hartzell, G., Zhang, Z., Rubin, G.M., Miller, W. 1998.          2004. Gene expression profiling developing Drosophila CNS <br />     A program aligning cDNA sequence geno-                midline cells. Dev. Biol. 275: 473&#226;&#8364;&#8220;492. <br />     mic DNA sequence. Genome Res. 8: 967&#226;&#8364;&#8220;974.                               Keegan, L.P., Brindle, J., Gallo, A., Leroy, A., Reenan, R.A., and <br /> Fortini, M.E., Simon, M.A., Rubin, G.M. 1992. Signalling             O&#226;&#8364;&#8482;Connell, M.A. 2005. Tuning RNA editing ADAR is <br />     sevenless protein tyrosine kinase mimicked Ras1 activation.           required Drosophila. EMBO J. 24: 2183&#226;&#8364;&#8220;2193. <br />     Nature 355: 559&#226;&#8364;&#8220;561.                                                    Kerkhoff, E., Simpson, J.C., Leberfinger, C.B., Otto, I.M., Doerks, T., <br /> Geng, W., He, B., Wang, M., Adler, P.N. 2000. The tricornered               Bork, P., Rapp, U.R., Raabe, T., Pepperkok, R. 2001. The Spir <br />     gene, required integrity epidermal cell                 actin organizers involved vesicle transport processes. Curr. <br />     extensions, encodes Drosophila nuclear DBF2-related kinase.             Biol. 11: 1963&#226;&#8364;&#8220;1968. <br />     Genetics 156: 1817&#226;&#8364;&#8220;1828.                                                Kim, D.D., Kim, T.T., Walsh, T., Kobayashi, Y., Matise, T.C., <br /> Gonzalez-Gaitan, M. Jackle, H. 1997. Role Drosophila a-adaptin           Buyske, S., Gabriel, A. 2004. Widespread RNA editing of <br />     presynaptic vesicle recycling. Cell 88: 767&#226;&#8364;&#8220;776.                         embedded alu elements human transcriptome. Genome Res. <br /> Gordon, D., Desmarais, C., Green, P. 2001. Automated finishing              14: 1719&#226;&#8364;&#8220;1725. <br />     autofinish. Genome Res. 11: 614&#226;&#8364;&#8220;625.                               Kohler, M., Hirschberg, B., Bond, C.T., Kinzie, J.M., Marrion, N.V., <br /> Guichet, A., Wucherpfennig, T., Dudu, V., Etter, S., Wilsch-                    Maylie, J., Adelman, J.P. 1996. Small-conductance, calcium- <br />     Brauniger, M., Hellwig, A., Gonzalez-Gaitan, M., Huttner, W.B.,             activated potassium channels mammalian brain. Science 273: <br />     Schmidt, A.A. 2002. Essential role endophilin A synaptic          1709&#226;&#8364;&#8220;1714. <br />     vesicle budding Drosophila neuromuscular junction. EMBO          Kramer, H., Cagan, R.L., Zipursky, S.L. 1991. Interaction bride <br />     J. 21: 1661&#226;&#8364;&#8220;1672.                                                           sevenless membrane-bound ligand sevenless tyrosine- <br /> Hart, A.C., Kramer, H., Zipursky, S.L. 1993. Extracellular domain           kinase receptor. Nature 352: 207&#226;&#8364;&#8220;212. <br />     boss transmembrane ligand acts antagonist sev       Lai, E.C. 2005. miRNAs. Whys wherefores miRNA-mediated <br />     receptor. Nature 361: 732&#226;&#8364;&#8220;736.                                              regulation. Curr. Biol. 15: R458&#226;&#8364;&#8220;R460. <br /> Haudenschild, B.L., Maydanovych, O., Veliz, E.A., Macbeth, M.R.,            Lai, E.C., Tomancak, P., Williams, R.W., Rubin, G.M. 2003. <br />     Bass, B.L., Beal, P.A. 2004. A transition state analogue         Computational identification Drosophila microRNA genes. <br />     RNA-editing reaction. J. Am. Chem. Soc. 126: 11213&#226;&#8364;&#8220;11219.                   Genome Biol. 4: R42. <br /> Haug-Collet, K., Pearson, B., Webel, R., Szerencsei, R.T.,                  Leaman, D., Chen, P.Y., Fak, J., Yalcin, A., Pearce, M., Unnerstall, U., <br />     Winkfein, R.J., Schnetkamp, P.P., Colley, N.J. 1999. Cloning            Marks, D.S., Sander, C., Tuschl, T., Gaul, U. 2005. Antisense- <br />     characterization potassium-dependent sodium/calcium                mediated depletion reveals essential specific functions of <br />     exchanger Drosophila. J. Cell Biol. 147: 659&#226;&#8364;&#8220;670.                        microRNAs Drosophila development. Cell 121: 1097&#226;&#8364;&#8220;1108. <br />  <br />  <br />                                                                                                                    www.rnajournal.org           1931 <br />  Stapleton et al. <br />  <br />  <br />  <br /> Lebovitz, R.M., Takeyasu, K., Fambrough, D.M. 1989. Molecular            Spyropoulos, I.C., Liakopoulos, T.D., Bagos, P.G., and <br />     characterization expression (Na++K+)-ATPase a-sub-               Hamodrakas, S.J. 2004. TMRPres2D: High quality visual repre- <br />     <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Drosophila melanogaster. EMBO J. 8: 193&#226;&#8364;&#8220;202.                        sentation transmembrane protein models. Bioinformatics 20: <br /> Levanon, E.Y., Eisenberg, E., Yelin, R., Nemzer, S., Hallegger, M.,             3258&#226;&#8364;&#8220;3260. <br />     Shemesh, R., Fligelman, Z.Y., Shoshan, A., Pollock, S.R., Sztybel, D.,   Stapleton, M., Carlson, J., Brokstein, P., Yu, C., Champe, M., <br />     et al. 2004. Systematic identification abundant A-to-I editing           George, R., Guarin, H., Kronmiller, B., Pacleb, J., Park, S., et al. <br />     sites human transcriptome. Nat. Biotechnol. 22: 1001&#226;&#8364;&#8220;1005.           2002a. A Drosophila full-length cDNA resource. Genome Biol. 3: <br /> Lloyd, T.E., Verstreken, P., Ostrin, E.J., Phillippi, A., Lichtarge, O.,        RESEARCH0080. <br />     Bellen, H.J. 2000. A genome-wide search synaptic vesicle         Stapleton, M., Liao, G., Brokstein, P., Hong, L., Carninci, P., <br />     cycle proteins Drosophila. Neuron 26: 45&#226;&#8364;&#8220;50.                             Shiraki, T., Hayashizaki, Y., Champe, M., Pacleb, J., Wan, K., <br /> Macbeth, M.R., Schubert, H.L., Vandemark, A.P., Lingam, A.T.,                   et al. 2002b. The Drosophila gene collection: Identification of <br />     Hill, C.P., Bass, B.L. 2005. Inositol hexakisphosphate bound         putative full-length cDNAs 70% D. melanogaster genes. <br />     ADAR2 core required RNA editing. Science 309:                Genome Res. 12: 1294&#226;&#8364;&#8220;1300. <br />     1534&#226;&#8364;&#8220;1539.                                                               Stolc, V., Gauhar, Z., Mason, C., Halasz, G., van Batenburg, M.F., <br /> Manseau, L.J. Schupbach, T. 1989. cappuccino spire: Two                 Rifkin, S.A., Hua, S., Herreman, T., Tongprasit, W., Barbano, P.E., <br />     unique maternal-effect loci required anteroposterior           et al. 2004. A gene expression map euchromatic genome of <br />     dorsoventral patterns Drosophila embryo. Genes &amp; Dev.            Drosophila melanogaster. Science 306: 655&#226;&#8364;&#8220;660. <br />     3: 1437&#226;&#8364;&#8220;1452.                                                            Takai, Y., Sasaki, T., Matozaki, T. 2001. Small GTP-binding <br /> Marchler-Bauer, A. Bryant, S.H. 2004. CD-Search: Protein                    proteins. Physiol. Rev. 81: 153&#226;&#8364;&#8220;208. <br />     domain annotations fly. Nucleic Acids Res. 32: W327&#226;&#8364;&#8220;W331.         Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F., and <br /> Martin, J.H., Benzer, S., Rudnicka, M., Miller, C.A. 1993.                  Higgins, D.G. 1997. The CLUSTAL_X windows interface: Flexible <br />     Calphotin: A Drosophila photoreceptor cell calcium-binding pro-             strategies multiple sequence alignment aided quality analysis <br />     tein. Proc. Natl. Acad. Sci. 90: 1531&#226;&#8364;&#8220;1535.                                 tools. Nucleic Acids Res. 25: 4876&#226;&#8364;&#8220;4882. <br /> Misra, S., Crosby, M.A., Mungall, C.J., Matthews, B.B., Campbell, K.S.,      Tonkin, L.A., Saccomanno, L., Morse, D.P., Brodigan, T., Krause, M., <br />     Hradecky, P., Huang, Y., Kaminker, J.S., Millburn, G.H.,                    Bass, B.L. 2002. RNA editing ADARs important for <br />     Prochnik, S.E., et al. 2002. Annotation Drosophila mela-             normal behavior Caenorhabditis elegans. EMBO J. 21: 6025&#226;&#8364;&#8220; <br />     nogaster euchromatic genome: A systematic review. Genome Biol. 3:           6035. <br />     RESEARCH0083.                                                            Tupy, J.L., Bailey, A.M., Dailey, G., Evans-Holm, M., Siebel, C.W., <br /> Morse, D.P., Aruscavage, P.J., Bass, B.L. 2002. RNA hairpins             Misra, S., Celniker, S.E., Rubin, G.M. 2005. Identification of <br />     noncoding regions human brain Caenorhabditis elegans                 putative noncoding polyadenylated transcripts Drosophila <br />     mRNA edited adenosine deaminases act RNA. Proc.              melanogaster. Proc. Natl. Acad. Sci. 102: 5495&#226;&#8364;&#8220;5500. <br />     Natl. Acad. Sci. 99: 7906&#226;&#8364;&#8220;7911.                                          Verstreken, P., Kjaerulff, O., Lloyd, T.E., Atkinson, R., Zhou, Y., <br /> Palladino, M.J., Keegan, L.P., O&#226;&#8364;&#8482;Connell, M.A., Reenan, R.A.                Meinertzhagen, I.A., Bellen, H.J. 2002. Endophilin mutations <br />     2000. A-to-I pre-mRNA editing Drosophila primarily involved           block clathrin-mediated endocytosis neurotransmitter <br />     adult nervous function integrity. Cell 102: 437&#226;&#8364;&#8220;449.          release. Cell 109: 101&#226;&#8364;&#8220;112. <br /> Palladino, M.J., Bower, J.E., Kreber, R., Ganetzky, B. 2003. Neural      Wang, Q., Khillan, J., Gadue, P., Nishikura, K. 2000. Requirement <br />     dysfunction neurodegeneration Drosophila Na+/K+ATPase                RNA editing deaminase ADAR1 gene embryonic <br />     subunit mutants. J. Neurosci. 23: 1276&#226;&#8364;&#8220;1286.                              erythropoiesis. Science 290: 1765&#226;&#8364;&#8220;1768. <br /> Prasanth, K.V., Prasanth, S.G., Xuan, Z., Hearn, S., Freier, S.M.,           Webel, R., Haug-Collet, K., Pearson, B., Szerencsei, R.T., <br />     Bennett, C.F., Zhang, M.Q., Spector, D.L. 2005. Regulating              Winkfein, R.J., Schnetkamp, P.P., Colley, N.J. 2002. Potas- <br />     gene expression RNA nuclear retention. Cell 123: 249&#226;&#8364;&#8220;263.           sium-dependent sodium-calcium exchange eye the <br /> Quinlan, M.E., Heuser, J.E., Kerkhoff, E., Mullins, R.D. 2005.              fly. Ann. N.Y. Acad. Sci. 976: 300&#226;&#8364;&#8220;314. <br />     Drosophila Spire actin nucleation factor. Nature 433: 382&#226;&#8364;&#8220;388.     Wellington, A., Emmons, S., James, B., Calley, J., Grover, M., <br /> Reinke, R. Zipursky, S.L. 1988. Cell&#226;&#8364;&#8220;cell interaction                Tolias, P., Manseau, L. 1999. Spire contains actin binding <br />     Drosophila retina: The bride sevenless gene required               domains related ascidian posterior end mark-5. De- <br />     photoreceptor cell R8 R7 cell development. Cell 55: 321&#226;&#8364;&#8220;330.            velopment 126: 5267&#226;&#8364;&#8220;5274. <br /> Rozen, S. Skaletsky, H. 2000. Primer3 WWW general users       Xia, X.M., Fakler, B., Rivard, A., Wayman, G., Johnson-Pais, T., <br />     biologist programmers. Methods Mol. Biol. 132: 365&#226;&#8364;&#8220;386.             Keen, J.E., Ishii, T., Hirschberg, B., Bond, C.T., Lutsenko, S., et al. <br /> Rubin, G.M., Hong, L., Brokstein, P., Evans-Holm, M., Frise, E.,                1998. Mechanism calcium gating small-conductance cal- <br />     Stapleton, M., Harvey, D.A. 2000. A Drosophila complemen-               cium-activated potassium channels. Nature 395: 503&#226;&#8364;&#8220;507. <br />     tary DNA resource. Science 287: 2222&#226;&#8364;&#8220;2224.                               Xia, S., Yang, J., Su, Y., Qian, J., Ma, E., Haddad, G.G. 2005. <br /> Rueter, S.M., Dawson, T.R., Emeson, R.B. 1999. Regulation                Identification new targets Drosophila pre-mRNA adenosine <br />     alternative splicing RNA editing. Nature 399: 75&#226;&#8364;&#8220;80.                     deaminase. Physiol. Genomics 20: 195&#226;&#8364;&#8220;202. <br /> Satterfield, T.F., Jackson, S.M., Pallanck, L.J. 2002. A Drosophila      Yamada, K., Lim, J., Dale, J.M., Chen, H., Shinn, P., Palm, C.J., <br />     homolog polyglutamine disease gene SCA2 dosage-sensitive        Southwick, A.M., Wu, H.C., Kim, C., Nguyen, M., et al. 2003. <br />     regulator actin filament formation. Genetics 162: 1687&#226;&#8364;&#8220;1702.             Empirical analysis transcriptional activity Arabidopsis <br /> Schumacher, M.A., Rivard, A.F., Bachinger, H.P., Adelman, J.P.              genome. Science 302: 842&#226;&#8364;&#8220;846. <br />     2001. Structure gating domain Ca2+-activated K+channel       Yang, W., Chendrimada, T.P., Wang, Q., Higuchi, M., Seeburg, P.H., <br />     complexed Ca2+/calmodulin. Nature 410: 1120&#226;&#8364;&#8220;1124.                      Shiekhattar, R., Nishikura, K. 2006. Modulation microRNA <br /> Schumacher, M.A., Crum, M., Miller, M. 2004. Crystal structure           processing expression RNA editing ADAR <br />     apocalmodulin apocamodulin/SK2 CaMBD complex: Mech-                  deaminases. Nat. Struct. Mol. Biol. 13: 13&#226;&#8364;&#8220;21. <br />     anism Ca2+-activated SK channel grating. Structure 12: 849&#226;&#8364;&#8220;860.       Yoshie, S., Imai, A., Nashida, T., Shimomura, H. 2000. Expres- <br /> Seto, E.S., Bellen, H.J., Lloyd, T.E. 2002. When cell biology meets         sion, characterization, localization Rab26, low molecular <br />     development: Endocyti <br /> </body></html>