<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 1998 Genetics Society America <br />  <br />  <br />  <br /> The PBN1 Gene Saccharomyces cerevisiae: An Essential Gene That Is Required <br />       Post-translational Processing Protease B Precursor <br />  <br />                                              Rajesh R. Naik Elizabeth W. Jones <br />                          Department Biological Sciences, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213 <br />                                                         Manuscript received March 4, 1998 <br />                                                       Accepted publication April 21, 1998 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The vacuolar hydrolase protease B Saccharomyces cerevisiae synthesized inactive precursor <br />               (Prb1p). The precursor undergoes post-translational modifications transiting secretory pathway. <br />               In addition N- O -linked glycosylations, proteolytic cleavages occur maturation of <br />               Prb1p. Removal signal peptide signal peptidase autocatalytic cleavage large amino- <br />               terminal propeptide occur endoplasmic reticulum (ER). Two carboxy-terminal cleavages the <br />               post regions occur vacuole: cleavage catalyzed protease A second results <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />               autocatalysis. We isolated mutant, pbn1-1, exhibits defect ER processing Prb1p. The <br />               autocatalytic cleavage propeptide Prb1p does occur Prb1p rapidly degraded the <br />               cytosol. PBN1 cloned identical YCL052c chromosome III. PBN1 essential gene that <br />               encodes novel protein. Pbn1p predicted contain sub-C-terminal transmembrane domain no <br />               signal sequence. A functional HA epitope-tagged Pbn1p fusion localizes ER. Pbn1p N-glycosylated <br />               amino-terminal domain, indicating lumenal orientation despite lack signal sequence. <br />               Based results, propose functions Pbn1p aid autocatalytic processing <br />               Prb1p. <br />  <br />  <br />  <br /> T    HE yeast vacuole contains ensemble hydrolases, <br />      including aminopeptidase I (ApI), proteinase A <br /> (PrA), proteinase B (PrB) carboxypeptidase Y <br />                                                                                PrB encoded PRB1 structural gene yeast <br />                                                                             initially synthesized large 76-kD inactive pre- <br />                                                                             cursor (Prb1p). Prb1p undergoes extensive post-transla- <br /> (CpY ). These proteases participate protein degrada-                     tional processing modification traverses the <br /> tion turnover. Vacuolar hydrolases synthesized                      secretory pathway (Figure 3A). The 76-kD precursor <br /> as inactive precursors reach vacuole                   enters endoplasmic reticulum (ER) post- <br /> secretory pathway cytoplasm-<span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span>-vacuole pathway                       translational translocation pathway. The signal peptide <br /> ( Jones Murdock 1994; Jones et al. 1997).                               cleaved signal peptidase evidenced its <br />    The vacuolar serine endoproteinase B (PrB) Sac-                       failure occur sec11 mutant (V. L. Nebes and <br /> charomyces cerevisiae member subtilisin family                  E. W. Jones, unpublished results). In addition, single <br /> of proteases (Moehle et al. 1987). PrB plays major                        N-linked carbohydrate chain O -linked carbohy- <br /> role degradation proteins peptides                     drate groups added. Next, large, 260-amino-acid <br /> starvation nitrogen carbon sources, autophagy,                       propeptide cleaved ER intramolecular <br /> meiosis sporulation, constitutive and/or                     autocatalytic process. The autocatalytic cleavage does <br /> environmentally triggered degradation endocytosed                        occur active site residue Asp45 Ser 239 is <br /> plasma membrane proteins (Zubenko Jones 1981;                           mutated, indicating functional PrB catalytic site <br /> Teichert et al. 1989; Takeshige et al. 1992; Jones                      required ER autocatalytic processing step. The <br /> Murdock 1994). Processing precursors                          autocatalytic cleavage intramolecular be <br /> vacuolar hydrolases involves specific cleavages catalyzed                   catalyzed trans (Nebes Jones 1991). The 39-kD <br /> by PrB. These include cleavages generate ma-                       precursor formed ER modified the <br /> ture N termini protease A (PrA), carboxypeptidase                        Golgi, molecular mass increases 40-kD, due <br /> Y (CpY), carboxypeptidase S (CpS), aminopeptidase I                         elaboration Asn-linked chain. Either <br /> (ApI), aminopeptidase Y (ApY ), PrB,                     prevacuolar compartment or, likely, the <br /> mature C termini yeast vacuolar alkaline phosphatase                     vacuole, PrA catalyzes cleavage C-terminal <br /> (ALP) PrB itself (Jones Murdock 1994; van                           post region, P1, removing 30 C-terminal amino <br /> den Hazel et al. 1996; Jones et al. 1997).                                  acids form 37-kD precursor. The 40-kD form accu- <br />                                                                             mulates PrA-deficient strains. The final proteolytic <br />                                                                             cleavage removes second post region, P2, con- <br />                                                                             taining N-linked carbohydrate chain, autocatalytic <br />   Corresponding author: Rajesh R. Naik, Department Biological Sci- <br /> ences, Carnegie Mellon University, 4400 Fifth Avenue, Pittsburgh, PA        cleaved PrB itself trans. The cleaved <br /> 15213. E-mail: rnaik@cmu.edu                                                propeptide targeted vacuole degradation <br />  <br /> Genetics 149: 1277&#226;&#8364;&#8220;1292 ( July 1998) <br />  1278                                             R. R. Naik E. W. Jones <br />  <br /> subsequent autocatalytic cleavage (Mechler et            work listed Table 1. Growth media used YEPD, <br /> al. 1982; Mechler et al. 1988; Moehle et al. 1989; Nebes        synthetic complete, synthetic medium containing galactose or <br />                                                                 raffinose carbon source, omission media previously <br /> and Jones 1991; Hirsch et al. 1992).                            described ( Jones et al. 1982). Escherichia coli strains grown <br />   The amino-terminal propeptide PrB 260 amino             Luria Broth medium; plasmid selection, ampicillin (100 <br /> acids long represents 40% Prb1p. Inspec-           mg/ml) added (Sambrook et al. 1989). All plasmids were <br /> tion amino acid sequence propeptide shows         propagated strain LM1035 (Cepko et al. 1984). All yeast <br /> a relatively high frequency charged amino acids          strains derived laboratory strain X2180-1B <br />                                                                 (MATa gal2 SUC2) crosses strains our <br /> the potential coiled-coil domain (amino acids 193         isogenic series strains congenic strain X2180-1B unless <br /> to 209 counting N terminus) propeptide.         stated. <br /> This N-terminal propeptide essential             DNA manipulations: Standard procedures plasmid DNA <br /> structure; absence processing pre-        isolations, restriction enzyme digestion, ligations, gel purifi- <br /> cursor does occur unless missing peptide             cation transformation bacteria performed as <br />                                                                 described (Sambrook et al. 1989). Yeast transformation was <br /> supplied trans (Nebes Jones 1991).                       performed using lithium acetate/dimethyl sulfoxide pro- <br />   Analogous Prb1p, yeast prohormone processing           cedure (Hill et al. 1991). Yeast genomic DNA prepared <br /> protease kexin precursor (Kex2p) contains 77-residue          method Hoffman Winston (1987) plas- <br /> propeptide cleaved ER intramolecu-         mid DNA isolated yeast shuttled E. coli. Poly- <br /> lar autocatalytic mechanism requires functional           merase chain reaction (PCR) performed using standard <br />                                                                 methods Perkin-Elmer Cetus Thermal Cycler. DNA se- <br /> Kex2p active site autocatalysis (Gluschankof            quence determined using Sequenase Kit (U.S. Bio- <br /> Fuller 1994). The propeptide essential fold-         chemicals, Cleveland, OH). DNA used sequencing was <br /> ing maturation Kex2p. Unlike Prb1p Kex2p,            prepared using Wizard Plus Minipreps DNA purification sys- <br /> the propeptides PrA, ApI, CpY cleaved             tem (Promega, Madison, WI). <br /> the vacuole ( Jones et al. 1997). This striking difference         Genetic analysis: Genetic crosses, sporulation, tetrad analy- <br />                                                                 sis complementation analysis performed using stan- <br /> in processing Prb1p amd Kex2p suggests          dard procedures previously described (Hawthrone and <br /> ER provides unique environment supports              Mortimer 1960). <br /> autoprocessing event.                                              Mutagenesis: Cultures BJ6252 bearing pBJ7934 were <br />   In paper report identification novel         grown stationary phase SC-Leu density 8 3 107 <br /> essential gene PBN1 required post-transla-      cells/ml. Cells harvested washed twice 50 mm <br />                                                                 potassium phosphate buffer pH 7.0 resuspended 5 ml <br /> tional processing Prb1p. In pbn1-1 mutant, Prb1p <br />                                                                 medium. Cells mutagenized using 3% <br /> enters ER lumen, fails undergo auto-             (w/v) ethyl methanesulfonate (EMS) described (Preston <br /> processing. Furthermore, unprocessed Prb1p de-           et al. 1992). The mutagenized cells plated directly onto <br /> graded cytosol. Pbn1p type I integral mem-          SC-leucine plates. About 18% cells survived treat- <br /> brane glycoprotein ER lumenally oriented         ment. After days incubation 308, plates rep- <br />                                                                 lica plated plate SC-leucine containing 5-FOA and <br /> despite lack signal sequence.                          second containing YEP-Glycerol (YEPG). The colonies on <br />                                                                 YEPG tested PrB activity means HPA overlay <br />                                                                 plate assay days growth 308 (see below). Colonies <br />               MATERIALS AND METHODS                             5-FOAR Prb2 chosen character- <br />                                                                 ization. <br />   Materials: Restriction enzymes DNA-modifying       HPA overlay plate assay Protease B activity: HPA overlay <br /> enzymes purchased Boehringer Mannheim Bio-            plate assay performed described Jones (1991). <br /> chemicals (Indianapolis), Promega (Madison, WI), New            Biochemical assays: b-galactosidase activity assayed as <br /> England Biolabs (Beverly, MA). Taq DNA Polymerase           described Kippert (1995). Activities represented in <br /> purchased Fisher Scientific (Pittsburgh, PA). Lyticase     Miller units (Miller 1972). PrB activity stationary phase <br /> L-8012, b-glucuronidase G-7770, 3-aminotriazole, Pon-       cells determined Azocoll (Calbiochem, San Diego, <br /> ceau S obtained Sigma Chemical Company (St.           CA) substrate described Jones (1991); specific ac- <br /> Louis, MO). Protein A Sepharose CL4B purchased         tivity defined (1.91)(DA520nm)/(mg protein/min) previ- <br /> Pharmacia (Piscataway, NJ). 5-fluro-orotic acid (5-FOA)     ously described ( Jones 1991). <br /> obtained Toronto Research Chemicals, Inc. (Toronto,           Preparation yeast cell extracts immunoblot- <br /> Canada). Trans35S purchased ICN Biochemicals           ting: Cell-free protein extracts prepared immunoblot <br /> (Costa Mesa, CA). Goat anti-rabbit IgG-HRPO conjugate       analysis carried described previously (Naik et al. <br /> purchased Bio-Rad (Hercules, CA). Nitrocellulose mem-      1997). Affinity-purified rabbit anti-PrB open reading frame <br /> branes &#226;&#8364;&#339;Optitran&#226;&#8364;? type HA85 obtained Schleicher &amp;     (ORF), anti-CpY anti-PrA antibodies used the <br /> Schuell (Keene, NH). Anti-HA 12CA5 monoclonal antibodies        primary antibodies. Immune complexes visualized ac- <br /> were purchased Boehringer Mannheim Biochemicals.           cording manufacturer&#226;&#8364;&#8482;s instructions use second- <br /> Antibodies Sec61p Kar2p kind gifts J. Brod-    ary goat anti-rabbit horseradish peroxidase conjugate activity <br /> sky M. Rose, respectively. Fluorescein-conjugated goat      (Bio-Rad, Hercules, CA). <br /> anti-mouse IgG antibody rhodamine-conjugated goat anti-        Spheroplast labeling immunoprecipitation: Radiolabel- <br /> rabbit secondary antibody purchased Molecular         ing spheroplasts immunoprecipitations carried out <br /> Probes (Eugene, OR). Tunicamycin, Azocoll, Hide Pow-        described Webb et al. (1997). Labeling temperature- <br /> der Azure (HPA) purchased Calbiochem (La Jolla,       sensitive strains required following changes: (1) steps <br /> CA). Oligonucleotide primers obtained Ransom          spheroplast formation room temperature, (2) <br /> Hill Bioscience (Ramona, CA).                                   spheroplasts preincubated restrictive temperature <br />   Media strains: The strains plasmids used      20&#226;&#8364;&#8220;30 min prior labeling, (3) labeling carried <br />                                                   The PBN1 Gene S. cerevisiae                                               1279 <br />  <br />                                                             TABLE 1 <br />                                          List strains plasmids used study <br />  <br />          Strain                                      Genotype                                               Source <br />                                                     A. Yeast Strains <br />          BJ3497      MATa his4 leu2-3,-112 ura3-52 sec61-1 (RDM7-4B)                                 R. Schekman <br />          BJ5414      MATa his3D200 ura3-52 lys2-801 trp1D1                                           Laboratory strain <br />          BJ5440      MATa his3D200 ura3-52 lys2-801 trp1D1 pep4::HIS3                                Laboratory strain <br />          BJ5902      MATa ura3-52 leu2D1 can1 prb1D1.6R                                              Laboratory strain <br />          BJ5991      MATa suc2D9 ura3-52 can1 trp1 prb1D1.6R                                         Laboratory strain <br />          BJ6252      MATa ura3-52 leu2D1 his3D200                                                    Laboratory strain <br />          BJ6253      MATa ura3-52 leu2D1 lys2-801                                                    Laboratory strain <br />          BJ6879      MATa ura3-52 leu2D1 his4-401 sec62-1                                            Laboratory strain <br />          BJ6988      MATa ura3-52 leu2,3-112 his3D200 sec65-1                                        Laboratory strain <br />          BJ7667      MATa gal4D gal80D his3 trp1-901 ade2-101 ura3-52 leu2-3,-112                    S. Elledge <br />                         URA3::UASGAL1-lacZ LYS2::UASGAL1-HIS3 (Strain Y190) <br />          BJ7668      MATa gal4D gal80D his3 trp1-901 ade2-101 ura3-52 leu2-3,-112                    S. Elledge <br />                         URA3::UASGAL1-lacZ LYS2::UASGAL1-HIS3 met (Strain Y187) <br />          BJ8589      MATa ura3-52 leu2D1 his3D200 pbn1-1                                             This study <br />          BJ8594      MATa leu2D1 ura3-52                                                             This study <br />          BJ8596      MATa leu2D1 ura3-52 his3D200 pbn1-1                                             This study <br />          BJ9048      his3D200 ura3-52 lys2-801 leu2D1 pbn1-1 pep4::HIS3                              This study <br />          BJ9094      his3D200 leu2D1 ade2-101 trp1D1 ura3-D1::TRP1 sen3-1                            M. Hochstrasser <br />          BJ9146      MATa trp1D1 leu2-3, 112 pbn1-1 sen3-1                                           This study <br />          BJ9195      MATa leu2D1 his3D200 lys2-801 sec62-1                                           This study <br />          BJ9196      MATa leu2D1 his3D200 lys2-801 sec62-1 pbn1-1                                    This study <br />          Strain                                       Plasmid                                               Source <br />                                                     B. Plasmids <br />          pBJ2465     BamHI-HindIII fragment containing PRB1 YEp13                                 Moehle et al. <br />                                                                                                        (1987) <br />          pBJ3637     PstI-XhoI fragment PRB1 pUC19                                             C. Moehle <br />          pBJ4680     4 kb BamHI-HindIII fragment containing PRB1                                     Nebes Jones <br />                                                                                                        (1991) <br />          pBJ6895     HIS3 Cassette pUC19                                                          Laboratory plasmid <br />          pBJ7115     URA3 pRS513                                                                  D. Proteau <br />          pBJ7662     pACTII (GAL4 activation domain plasmid)                                         Durfee et al. (1993) <br />          pBJ7663     pAS1-CYH2 (GAL4 DNA binding domain plasmid)                                     Durfee et al. (1993) <br />          pBJ7804     Propeptide fragment PRB1 pAS1-CYH2                                        This study <br />          pBJ7819     pACTIIDNcoI (NcoI site filled-in)                                            This study <br />          pBJ7934     UASPRB1 -URA3 pRS315                                                         This study <br />          pBJ8025     Mature domain PRB1 pACTII                                                 This study <br />          pBJ8588     Library plasmid; pC1c                                                           This study <br />          pBJ8700     pC1cDNX. A NcoI-XbaI deletion pC1c                                           This study <br />          pBJ8701     1.994 kb genomic PCR fragment PBN1 (YCL052c) cloned pRS315              This study <br />          pBJ8702     BamHI-XbaI fragment PBN1 cloned pRS305                                  This study <br />          pBJ8705     PBN1::URA3 pRS315                                                            This study <br />          pBJ8706     EcoRI-XhoI fragment PBN1 pRD54                                            This study <br />          pBJ8707     Same pBJ8706 expect UASGAL1 replaced UASPBN1                This study <br />          pBJ8827     BamHI-XbaI fragment PBN1 cloned pRS316                                  This study <br />          pBJ8828     PBN1D::HIS3 pRS315                                                           This study <br />          pBJ8830     PBN1 pAS1-CYH2                                                               This study <br />          pBJ8831     PBN1 pACTII                                                                  This study <br />          pBJ9194     4XHA::PBN1 pRS316                                                            This study <br />          pBJ9197     3.4 kb ClaI-BamHI fragment pC1c pRS316                                This study <br />          pBJ9198     3.9 kb ClaI-BamHI fragment pC1c pRS316                                This study <br />  <br />  <br />  <br /> out restrictive temperature unless stated. If      ferential centrifugation carried previously described <br /> tunicamycin used, added 1 mg/ml dimethyl          (Becherer et al. 1996). To determine nature associa- <br /> sulfoxide stock final concentration 20 mg/ml 15 min         tion HA::Pbn1p membranes, cell extracts pre- <br /> prior adding <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span>. The immunoprecipitates sepa-        pared processed described Feldheim et al. (1993). <br /> rated SDS-PAGE subjected autoradiography.                    Cloning complementation: The wild-type PBN1 gene was <br />   Subcellular fractionation: Subcellular fractionation dif-      cloned complementation Prb2 <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. A CEN <br />  1280                                               R. R. Naik E. W. Jones <br />  <br /> LEU2 based yeast genomic library (obtained P. Hieter)           Epitope tagging PBN1: Construction HA-tagged <br /> was transformed strain BJ8591 (pbn1-1 leu2D1 ura3-52         version PBN1 carried follows: EcoRI site was <br /> lys2-801) transformants selected SC-leu plates.       introduced PCR immediately ATG codon <br /> The plates incubated 308 3 days. Colonies        PBN1 ORF. An XhoI site introduced PCR approxi- <br /> then replica-plated YEPG incubated 2 days         mately 65 bp downstream stop codon. The PCR amplifi- <br /> 308; colonies tested PrB activity means HPA    cation product obtained using ORF52-HA1 (59-GCTG <br /> overlay plate assay. Plasmids colonies gave Prb1      AGAATTCGGTGACAAGA-39) ORF52-HA2 (59-GAAAA <br /> phenotype isolated. Plasmids shuttled         CTCGAGTGCACAGTAG-39) primers digested using EcoRI <br /> E. coli yeast retested complementing        XhoI, restriction fragment ligated the <br /> activity. The plasmids sequenced define ends       enzyme sites HA-tagging vector pRD54 GAL1::HA <br /> the insert DNA using primers MHCP506 (59-CAGCAACCGC               pRS316 (constructed R. Deshaies), yielding plasmid <br /> ACCTGTG-39) MHCP507 (59-GACTACGCGATCATG-39)                   pBJ8706 single copy HA epitope fused frame <br /> that flank BamHI site YCp50. The sequence obtained         59 end PBN1 ORF; ATG fusion is <br /> was compared yeast genome using BLAST program.         provided HA epitope. This HA-tagged version PBN1 <br /> The DNA sequence obtained Saccharomyces         (pBJ8706) detected immunoblots by <br /> Genome Database (SGD) (http://genome-www.stanford.edu/            immunoprecipitation radiolabeled cell protein <br /> Saccharomyces/) restriction map region           extracts; corresponding protein band observed in <br /> generated using DNA Strider program.                          strains carrying vector alone. The 670-bp GAL1 promoter <br />    Plasmid constructions: pBJ8701 constructed follows:     pBJ8706 completely replaced native upstream <br /> Using sequence obtained SGD, primers         region PBN1 (without adding extraneous sequence) as <br /> regions approximately 513 nucleotides upstream (ORF52-2           follows: 540-bp region upstream PBN1 ORF was <br /> 59-GGATTTGGATCCGATATTTGC-39) 233 nucleotides                  amplified pBJ8701 using PCR primers ORF52HA-422 <br /> downstream (ORF52-1 59-GATGGTGTCTAGAAGAATG-39)                    (59-GCACATCTAGAGTATGCG-39) ORF52HA16 (59- <br /> of YCL052c synthesized. PCR genomic DNA yielded           CACCATGGCTCAGCTAGAAACG-39) restriction sites, <br /> a 1.994-kb amplification product restriction sites   XbaI NcoI; fragment cloned corresponding <br /> end introduced primers. The amplification product          sites pBJ8706 resulting plasmid pBJ8707. For intro- <br /> was digested BamHI XbaI, restriction frag-       duction triple HA tag, 121-bp DNA fragment with <br /> ment gel purified cloned corresponding      flanking Eco RI sites encoding triple HA epitope (provided <br /> sites pRS315. The BamHI-XbaI fragment resulting       A. Srivastava) digested EcoRI ligated <br /> plasmid, pBJ8701, cloned single- high-copy     site resident single HA tag the <br /> plasmids pRS series (Sikorski Hieter 1989).            ATG PBN1 ORF pBJ8707. Orientation the <br />    The UASPRB1 -URA3 reporter plasmid, pBJ7934, con-          triple tag checked PCR followed testing ability <br /> structed cloning 1520-bp upstream sequence PRB1         plasmid complement pbn1-1. The plasmid pBJ9194 <br /> into pBJ7115. Primers PRB1-UAS1 (59-CCACGATCTAGAC                 satisfied criteria. However, pBJ8707 pBJ9194 <br /> CCCGTTGTCC-39) PRB1-UAS2 (59-GTATTTTGGATCC                    versions PBN1 detected immunoblots or <br /> TCATCTTTGC-39) used amplify UAS fragment              immunoprecipitation radiolabeled cell protein <br /> of PRB1. The PCR fragment digested BamHI             extracts. <br /> XbaI cloned sites pBJ7115.                      Immunofluorescence microscopy: Immunofluorescence mi- <br />    A disrupted allele PBN1 (pBJ8828) constructed        croscopy essentially described Pringle et al. <br /> replacing HindIII-NsiI fragment PBN1 HIS3         (1991). Cells grown overnight SC-uracil containing raf- <br />                                                                   finose sole carbon source. The day cells in- <br /> gene. pBJ8705 digested HindIII-Nsi I drop a <br />                                                                   oculated 25 ml SC-uracil containing galactose the <br /> 588-bp internal fragment PBN1 replaced HindIII- <br />                                                                   sole carbon source allowed grow 5&#226;&#8364;&#8220;8 hr (OD600nm 5 <br /> PstI HIS3 fragment pBJ7143. To disrupt PBN1 gene, <br />                                                                   0.5&#226;&#8364;&#8220;1.0). In cases, galactose induction carried out <br /> the pbn1D::HIS3 fragment PCR amplified using primers <br />                                                                   16 hr, growing cells galactose overnight then <br /> ORF52-1 ORF52-2 pBJ8828, introduced into <br />                                                                   transferring cells 25 ml SC-uracil 1 galactose. <br /> the chromosome diploid strain homologous recombi- <br />                                                                   Cells collected processed immunofluorescence <br /> nation. The transformants plated SC-histidine. The      microscopy. The staining HA-tagged Pbn1p observed <br /> integration events confirmed DNA blot analysis. Three     using 12CA5 monoclonal antibody (25 mg/ml) the <br /> different transformants selected sporulated.             primary antibody. Decoration 12CA5 performed by <br />    To clone PBN1 dihybrid vectors (Fields Song       addition 5 mg/ml fluorescein isothiocyanate-conjugated <br /> 1989), EcoRI-XhoI fragment pBJ8706 cloned        goat anti-mouse secondary antibody. Visualization anti- <br /> the EcoRI-XhoI EcoRI-Sal I sites pACTII pAS1-CYH2,     Kar2p antibody achieved addition 10 mg/ml of <br /> respectively. The propeptide region Prb1p cloned      rhodamine-conjugated goat anti-rabbit secondary antibody. <br /> pAS1-CYH2 using PCR fragment NdeI BglII restric-       For double-labeling experiments Kar2p HA::Pbn1p, <br /> tion sites flanking propeptide coding sequence; PCR       permeabilized cells incubated primary antibod- <br /> primers RN1-Pro (59-AGCATCTCTGCTCATATGGTCATCC                     ies. After washing, second incubation 5 mg/ <br /> C-39) VN7 (59-CGAGACGCCTAAGGAA AG-39) used               ml fluorescein isothiocyanate-conjugated goat anti-mouse sec- <br /> to amplify propeptide region pBJ3637. The fragment       ondary 10 mg/ml rhodamine-conjugated goat anti- <br /> obtained digesting amplified product restriction      rabbit secondary antibody. As controls, reactions only <br /> enzymes NdeI BglII ligated NdeI BamHI        primary antibodies assayed confirm that <br /> sites pAS1-CYH2. Primers PRB1-RN2 (59-GTCGAGGATC               leakage emission channels <br /> CAGAATTTGAC-39) PRB1-RN3 (59-CCGACTTGGATC                     observation. The cells observed using Nikon EFD- <br /> CTCGAGAACG-39) used PCR amplify catalytic             3 microscope. <br /> domain pBJ4680. The amplified product digested <br /> with BamHI XhoI ligated sites pACTI- <br /> IDNcoI. The full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> Prb1p lacking 13 amino acids                              RESULTS <br /> was cloned pACTII ligating Avr II-BamHI fragment <br /> from pBJ4780 NcoI-BamHI sites pACTII (pBJ9153). All         Isolation pbn1-1: The primary objective ge- <br /> fusion proteins detected immunoblots.                 netic scheme identify genes involved in <br />                                                   The PBN1 Gene S. cerevisiae                                        1281 <br />  <br />                                                                    sized using 3% EMS plated selective complete <br />                                                                    minus leucine (SC-Leu) plates. After days, these <br />                                                                    plates replica-plated 5-FOA medium (SC-Leu 1 <br />                                                                    5-FOA) select mutants unable express <br />                                                                    URA3 gene YEPG HPA overlay plate <br />                                                                    assay PrB activity. Colonies Prb2 FOAR <br />                                                                    likely candidates mutations interest <br />                                                                    genes positive regulators PRB1 expression. <br />                                                                    These mutants chosen analysis. The <br />                                                                    step exclude plasmid-borne mutations and <br />                                                                    mutations linked PRB1 locus. This achieved <br />                                                                    crossing mutants prb1D ura3 strain (BJ5902). <br />                                                                    Diploids FOAS Prb1 , indicating recessive <br />                                                                    <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span>(s), desired mutants. <br />                                                                       We identified mutant colony FOAR and <br />                                                                    Prb2 diploid obtained mating prb1D ura3 <br />                                                                    strain gave Prb1 5-FOA R phenotype. This suggested <br />                                                                    mutant harbored mutations, which <br />                                                                    genomic, plasmid-borne. This mutant was <br />                                                                    designated pbn1-1 (P rotease B Negative) Prb2 <br />                                                                    phenotype assessed HPA overlay assay (Figure <br />                                                                    2A). The pbn1-1 mutant cured retransformed <br />                                                                    fresh plasmid pBJ7934. This newly transformed <br />                                                                    pbn1-1 mutant unable grow SC-Leu 1 5-FOA, <br />                                                                    indicating 5-FOAR phenotype origi- <br />                                                                    nal mutant plasmid-borne mutation. <br />                                                                    The PRB1 gene, 2m plasmid, failed comple- <br />                                                                    ment Prb2 phenotype pbn1-1 mutant; thus, <br />                                                                    increasing copy number PRB1 gene fails to <br />                                                                    overcome Prb2 phenotype pbn1-1 mutant <br />                                                                    (Figure 2A). <br />                                                                       The pbn1-1 mutant crossed wild-type strain <br />                                                                    opposite mating type (BJ6253); heterozygous dip- <br />                                                                    loid obtained sporulated. All 33 tetrads segre- <br />                                                                    gated 2 Prb1:2 Prb2 , indicating pbn1-1 segregates <br />                                                                    single mutation. <br />                                                                       pbn1-1 affects PrB levels: To determine ef- <br />                                                                    fects pbn1-1 mutation levels PrB under <br />                                                                    steady-state conditions, performed immunoblot anal- <br />                                                                    ysis cell extracts pbn1-1. Under steady- <br />                                                                    state conditions, mature PrB processing interme- <br />                                                                    diates seen pbn1-1 mutant extract (Figure <br />   Figure 1.&#226;&#8364;&#8221;Genetic scheme isolating regulatory mu-            2B). However, levels lumenal hydrolases protease <br /> tants. Cells (BJ6252) bearing UASPRB1-URA3 plasmid,            A (PrA) carboxypeptidase Y (CpY) unaffected <br /> pBJ7934, mutagenized 3% EMS. Survivors         precursors processed forms that <br /> screened 5-FOA Prb phenotypes. By HPA overlay          accumulate PrB-deficient cells. Furthermore little or <br /> plate assay, Prb2 colonies unable solubilize HPA <br />                                                                    PrB activity detected Azocoll assay the <br /> particles, remain covered (d). Prb1 colonies uncov- <br /> ered solubilization particles surrounded     mutant (Figure 2C). PrB activity pbn1-1 mutant <br /> by clear halo (s).                                               ,10% wild-type parent. Increasing PRB1 <br />                                                                    copy number pbn1-1 mutant did lead in- <br />                                                                    creased PrB activity. Thus pbn1-1 mutation affects <br /> the transcriptional regulation PRB1. Figure 1 shows             PrB, PrA CpY levels. <br /> the strategy used selecting mutants. A                Post-translational processing defect pbn1-1 mu- <br /> ura3-52 strain (BJ6252) transformed plas-             tant: Since pbn1-1 mutant showed processing <br /> mid, pBJ7934, bearing URA3 gene fused 1250                  intermediates mature PrB accumulating steady- <br /> bp PRB1 upstream sequence: UASPRB1 -URA3 reporter               state conditions, performed kinetic experiment to <br /> plasmid. The Leu1 transformants bearing URA3                   follow maturation PrB pbn1-1 mutant. We <br /> promotor fusions Ura1. This strain mutagene-               earlier confirmed Northern blot analysis that <br />  1282                                                R. R. Naik E. W. Jones <br />  <br />                                                                    pbn1-1 mutation does affect transcriptional <br />                                                                    levels PRB1 (data shown). As shown Figure <br />                                                                    3A, PrB precursor undergoes extensive post-transla- <br />                                                                    tional processing modification traverses the <br />                                                                    secretory pathway. <br />                                                                       In wild-type cells, Prb1p completely processed <br />                                                                    series intermediates mature 31-kD <br />                                                                    species 30-min chase period (Figure 3B). <br />                                                                    Under conditions, pbn1-1 mutant the <br />                                                                    76-kD precursor detected, processed <br />                                                                    forms Prb1p observed. The 76-kD precursor was <br />                                                                    eventually degraded 30-min chase period. <br />                                                                    No evidence autocatalysis 39-kD and <br />                                                                    40-kD species evident; however, overexposure the <br />                                                                    autoradiographs revealed faint 40-kD band, no <br />                                                                    smaller intermediates observed. This result <br />                                                                    indicates defect intramolecular autoprocessing <br />                                                                    step Prb1p pbn1-1 mutant. In contrast, the <br />                                                                    kinetic maturation PrA CpY precursors in <br />                                                                    pbn1-1 mutant indistinguishable of <br />                                                                    wild-type strain (data shown). Thus Prb2 pheno- <br />                                                                    type pbn1-1 mutant defect post- <br />                                                                    translational processing Prb1p. <br />                                                                       Translocation N-glycosylation Prb1p the <br />                                                                    pbn1-1 mutant: Although able detect 76- <br />                                                                    kD precursor pbn1-1 mutant, observation does <br />                                                                    differentiate cytosolic 76-kD precursor <br />                                                                    ER lumenal 76-kD precursor (see Figure 3A). <br />                                                                    Because estimated molecular mass signal se- <br />                                                                    quence removed Prb1p similar single <br />                                                                    N-glycosyl chain (about 2 kD), cytosolic and <br />                                                                    glycosylated ER-lumenal species Prb1p hard <br />                                                                    distinguish SDS-PAGE. <br />                                                                       To determine Prb1p cytosolic lumenal <br />                                                                    pbn1-1 mutant, treated spheroplasts tuni- <br />                                                                    camycin prior metabolic labeling. Because tuni- <br />                                                                    camycin inhibits synthesis glycosyl donor (Kuo <br />                                                                    Lampen 1974; Lehle Tanner 1976), proteins <br />                                                                    fail N-linked glycosylated presence. When <br />   Figure 2.&#226;&#8364;&#8221;pbn1-1 mutation affects PrB levels. (A) Prb phe-       pbn1-1 mutant treated tunicamycin prior <br /> notypes PBN1 (BJ8594), pbn1-1 (BJ8596), pbn1-1/pPBN1            radiolabeling immunoprecipitation anti- <br /> (BJ8596 transformed pBJ8701), pbn1-1/pPRB1 (BJ8596            Prb1p, observed increase mobility the <br /> transformed pBJ2465), prb1D (BJ5991 transformed with <br />                                                                    Prb1p decrease molecular mass about <br /> vector) prb1D/PRB1 (BJ5991 transformed pBJ2465), <br /> as assessed HPA overlay plate assay. (B) Steady-state levels    2 kD (Figure 4). Prb1p untreated mutant mi- <br /> of lumenal hydrolases. Whole cell extracts prepared      grated slowly. Thus Prb1p species seen the <br /> PBN1 (BJ8594), pbn1-1 (BJ8596), pbn1-1/pPBN1 (BJ8596               pbn1-1 mutant N-glycosylated. <br /> transformed pBJ8701), prb1D (BJ5991), sub-               Translocation Prb1p ER lumen occurs <br /> jected SDS-PAGE (extract protein acrylamide concen- <br /> trations hydrolase were: CpY, 5 mg, 10% gel; PrA, 15 <br />                                                                    post-translationally requires Sec61p Sec62p. <br /> mg, 10% gel; PrB, 10 mg, 12% gel) followed immunoblot           The translocation Prb1p blocked sec61 <br /> analysis. (C) Whole cell extracts PBN1 (BJ8594), pbn1-1       sec62 mutant nonpermissive temperature, lead- <br /> (BJ8596), pbn1-1/pPBN1 (BJ8596 transformed pBJ8701),           ing accumulation cytosolic, unglycosylated <br /> pbn1-1/pPRB1 (BJ8596 transformed pBJ2465), as-           76-kD precursor (Moehle et al. 1989; R. R. Naik and <br /> sayed PrB activity using Azocoll substrate. <br />                                                                    E. W. Jones, unpublished results). The molecular mass <br />                                                                    untranslocated Prb1p accumulates a <br />                                                                    sec62-1 mutant unaffected treatment tunica- <br />                                                                    mycin. In pbn1-1 sec62-1 double mutant molecular <br />                                                                    mass Prb1p identical seen sec62-1 <br />                                                   The PBN1 Gene S. cerevisiae                                           1283 <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;The post-translational processing Prb1p. (A) A schematic representation post-translational modifications <br /> and processing Prb1p yeast secretory pathway. Solid circle open circle represent O- N-linked glycosylations, <br /> respectively. The molecular mass intermediate shown. (B) Kinetic maturation Prb1p PBN1 (BJ8594) pbn1-1 <br /> mutant (BJ8596). Autoradiograph proteins labeled Trans-35S vivo, immunoprecipitated anti-Prb1p separated <br /> by SDS-PAGE. Spheroplasts labeled 2 min, excess cold methionine cysteine added spheroplasts were <br /> harvested times shown start chase. The numbers left indicate apparent molecular mass each <br /> intermediate. Lane 1 loaded labeled molecular weight (MW) markers (91, 67, 43, 30 kD). <br />  <br />  <br /> mutant, indicating sec62 epistatic pbn1 (Figure            Degradation Prb1p pbn1-1 cytosolic: In the <br /> 4). Furthermore, presence tunicamycin,               pbn1-1 mutant, 76-kD precursor degraded during <br /> unglycosylated Prb1p species pbn1-1 mutant z2              30-min chase. In principle degradation Prb1p <br /> kD smaller untranslocated species sec62-1            pbn1-1 mutant occur ways: <br /> mutant. This size difference presumably              (1) secretion degradation Prb1p extra- <br /> processing Prb1p signal sequence, indicating               cellular medium, (2) transport degradation <br /> Prb1p translocated pbn1-1 mutant. Thus trans-           Prb1p vacuole, (3) retro-translocation <br /> location ER N-glycosylation Prb1p              degradation Prb1p cytosol. We tested each <br /> unaffected pbn1-1 mutant.                                   possibilities. <br />                                                                       As shown Figure 5A, unable detect the <br />                                                                    76-kD Prb1p extracellular fraction pbn1-1 <br />                                                                    mutant 30-min chase. Prb1p retained in <br />                                                                    intracellular fraction degraded chase <br />                                                                    period. This indicates disappearance 76-kD <br />                                                                    Prb1p kinetic experiments its <br />                                                                    secretion extracellular medium pbn1-1 mu- <br />                                                                    tant. In order differentiate pathways <br />                                                                    intracellular degradation Prb1p, vacuolar versus <br />    Figure 4.&#226;&#8364;&#8221;Prb1p translocated ER N-gly- <br /> cosylated. Spheroplasts PBN1 (BJ8594), pbn1-1 (BJ8596) <br />                                                                    cytosolic, constructed pep4::HIS3 pbn1-1 sen3-1 <br /> sec62-1 (BJ9195), sec62-1 pbn1-1 (BJ9196) preincu-        pbn1-1 double mutants. Degradation short-lived <br /> bated 378 20 min additional 10 min            proteins attributed vacuolar proteolysis and <br /> the presence tunicamycin (20 mg/ml) prior metabolic          dependent PrA, encoded PEP4 gene ( Jones <br /> labeling Trans-35 S. Spheroplasts labeled 2 min      et al. 1997). Sen3p encodes regulatory subunit the <br /> and chased excess cold methionine cysteine <br /> for additional 5 min. The spheroplasts lysed       26S proteasome required cytosolic ubiquitin- <br /> labeled proteins immunoprecipitated anti-Prb1p. As       dependent protein degradation. The sen3-1 mutant <br /> a control, wild-type cells subjected regimen.     shows severe defects degradation ubiquitinated <br />  1284                                               R. R. Naik E. W. Jones <br />  <br />                                                                   cloned complementation Prb2 phenotype <br />                                                                   pbn1-1. The mutant strain BJ8591 transformed <br />                                                                   CEN LEU2 based yeast genomic library. Trans- <br />                                                                   formants selected SC-Leu plates. The Leu1 <br />                                                                   transformants replica plated YEPG HPA <br />                                                                   overlay plate assay complementation Prb2 <br />                                                                   phenotype. Of 13,000 colonies screened HPA <br />                                                                   assay, colonies gave Prb1 phenotype. The library <br />                                                                   plasmids recovered transformants comple- <br />                                                                   mented pbn1-1 mutant shuttling proved <br />                                                                   identical. DNA sequencing ends the <br />                                                                   inserted yeast sequences clone pC1c revealed 11- <br />                                                                   kb fragment left arm chromosome III, con- <br />                                                                   taining complete ORFs (Figure 6). Subcloning of <br />                                                                   region indicated plasmids containing YC- <br />                                                                   L052c able complement Prb2 phenotype <br />                                                                   pbn1-1. Using sequence obtained SGD, <br />                                                                   PBN1 gene cloned genomic PCR. Primers <br />                                                                   designed amplified insert contained <br />                                                                   sufficient upstream (625-bp) downstream (233-bp) <br />                                                                   regions ensure proper expression gene. Restric- <br />                                                                   tion sites introduced primers facilitate <br />                                                                   cloning amplification product yeast plasmid <br />   Figure 5.&#226;&#8364;&#8221;Degradation Prb1p pbn1-1 mutant is <br /> cytosolic. (A) Spheroplasts pbn1-1 (BJ8596) labeled       vector pRS315. The resulting plasmid pBJ8701 able <br /> with Trans-35 S 2 min followed addition excess unla-    complement Prb2 phenotype pbn1-1 mu- <br /> beled methionine cysteine, samples taken      tant examined HPA plate assay (Figure 2A). <br /> indicated times. Spheroplasts lysed labeled proteins     In addition, restoration PrB activty pbn1-1 mutant <br /> were immunoprecipitated anti-Prb1p cell associated <br />                                                                   transformed pBJ8701 confirmed biochemically <br /> (I) medium (E) fractions. The immunoprecipitated sam- <br /> ples subjected SDS-PAGE autoradiography. (B)          (Figure 2C). <br /> Spheroplast pbn1-1 (BJ8596), pbn1-1 pep4::HIS3 (BJ9048),          To determine PBN1 gene corresponds <br /> and pbn1-1 sen3-1 (BJ9146) labeled Trans-35 S       YCL052c, YCL052c cloned integrating <br /> 2 min subsequently chased indicated times. Sphe-      vector, pRS305, carrying LEU2 marker. The re- <br /> roplasts lysed labeled proteins immunopre- <br />                                                                   sulting plasmid, pBJ8702, linearized Nsi I to <br /> cipitated anti-Prb1p. The immunoprecipitated samples <br /> were subjected SDS-PAGE autoradiography.                   direct integration YCL052c chromosomal locus <br />                                                                   wild-type strain (BJ6253). This integrant then <br />                                                                   crossed pbn1-1 mutant. All 20 tetrads obtained <br /> proteins (Demarini et al. 1995). If degradation vacuo-         sporulating heterozygous diploid segregated <br /> lar, expect pep4::HIS3 pbn1-1 strain de-           2 Leu1 Prb1: 2 Leu2 Prb2, indicating tight linkage of <br /> creased degradation; degradation cytosolic          LEU2 wild-type allele pbn1-1, PBN1. <br /> proteasome, expect sen3-1 pbn1-1 strain            Thus, YCL052c bona fide PBN1 gene. <br /> decreased degradation.                                               The pbn1-1 allele recovered BJ8591 gap <br />   In pep4::HIS3 pbn1-1 double mutant, Prb1p               repair. The gap repaired plasmid sequenced and <br /> degraded rate comparable pbn1-1               compared wild-type allele. The pbn1-1 allele con- <br /> mutant, suggesting degradation Prb1p pbn1-1          tains single amino acid substitution, glycine (GGA) <br /> mutant dependent vacuolar proteases (Figure             arginine (AGA) change position 182 Pbn1p <br /> 5B). In contrast, rate degradation Prb1p            amino acid sequence. <br /> reduced sen3-1 pbn1-1 double mutant (note espe-               Features Pbn1p: The PBN1 (YCL052c) ORF en- <br /> cially 12- 45-min points), Prb1p             codes uncharacterized protein 416 amino acids <br /> immunoprecipitated extracts pbn1-1                predicted Mr 48 kD (Yeast Proteome Data- <br /> sen3-1 mutant chased 45 min. The                <span id='am-320' about='xsp:base' typeof='owl:Thing'>base</span>). Hydropathy analysis method Kyte and <br /> reduction Prb1p degradation sen3-1 pbn1-1               Doolittle (1982) revealed 19-amino-acid (residues <br /> double mutant suggests 26S proteasome re-             386&#226;&#8364;&#8220;404), sub-C-terminal hydrophobic region pre- <br /> sponsible degradation Prb1p pbn1-1 mutant.          dicted transmembrane domain (Figure 7A). The <br /> Therefore, results suggest Prb1p exported           region amino acids 62 78 predicted to <br /> back cytosol degradation 26S protea-          adopt a-helical coiled-coil conformation (Figure 7B). <br /> some pbn1-1 mutant.                                        The probability region Pbn1p form <br />   Cloning PBN1 gene: The wild-type PBN1 gene               coiled-coil calculated using COILS 2.1 program <br />                                                 The PBN1 Gene S. cerevisiae                                            1285 <br />  <br />  <br />                                                                                                     Figure 6.&#226;&#8364;&#8221;Cloning of <br />                                                                                                  PBN1. Physical maps the <br />                                                                                                  constructed plasmids and <br />                                                                                                  ability complement <br />                                                                                                  Prb2 phenotype a <br />                                                                                                  pbn1-1 mutant (BJ8586). <br />                                                                                                  The original complement- <br />                                                                                                  ing clone, pC1c showing the <br />                                                                                                  location YCL052c on <br />                                                                                                  chromosome III. pBJ8701 <br />                                                                                                  carries smallest frag- <br />                                                                                                  ment complements the <br />                                                                                                  pbn1-1 mutation. The re- <br />                                                                                                  gion nucleotide 1 144 <br />                                                                                                  (ATG reference <br />                                                                                                  point) 1 732 ORF <br />                                                                                                  deleted replaced <br />                                                                                                  HIS3 gene. The ab- <br />                                                                                                  breviations follows: B, <br />                                                                                                  BamHI; C, ClaI; H, HindIII; <br />                                                                                                  N, NcoI; S, NsiI; X, XbaI. Pa- <br />                                                                                                  rentheses indicate restric- <br />                                                                                                  tion enzyme sites engi- <br />                                                                                                  neered PCR. <br />  <br />  <br />  <br /> (McLachlan Stewart 1975; McLachlan Karn                  immunofluorescence using monoclonal antibody <br /> 1982; Cohen Parry 1986; Lupas et al. 1991). Pbn1p            (12CA5) HA epitope, distinctive ER staining <br /> also contains potential N-glycosylation sites (Asn24,        <span id='am-321' about='xsp:pattern' typeof='owl:Thing'>pattern</span> seen, typified perinuclear staining <br /> Asn85, Asn120, Asn212, Asn365 Asn386) lie               staining plasma membrane (Figure 8C). <br /> within portion polypeptide N-terminal          This staining pattern resembles Kar2p, an <br /> predicted transmembrane domain (Figure 7C). Pbn1p                ER lumenal chaperone. Double-labeling experiments <br /> does contain clearly defined signal sequence. The          Kar2p, ER protein, HA::Pbn1p indicated <br /> C-terminal tail Pbn1p highly charged contains <br /> several lysine residues. No significant similarity to <br /> any proteins defined functional motifs evi- <br /> dent Pbn1p sequence. <br />    PBN1 essential gene:We constructed pbn1D::HIS3 <br /> allele amino acids 40&#226;&#8364;&#8220;243 PBN1 coding <br /> region replaced HIS3 marker (Figure 6). <br /> This allele introduced BJ6252 3 BJ5414 <br /> diploid strain, heterozygous diploid allele <br /> was constructed one-step gene disruption method. <br /> Sporulation heterozygous diploid gave tetrads <br /> (n 5 24) viable spore clones/tetrad. All <br /> surviving spore clones His2 Prb1. The sporulation <br /> of heterozygous diploid bearing PBN1 plasmid <br /> carrying URA3 selectable marker (pBJ8827) <br /> yielded viable spore clones His1 Prb1 Ura1 ; <br /> however, spore clones unable grow on <br /> selective medium containing 5-FOA. Therefore the <br /> PBN1 gene essential vegetative growth. <br />    ER-localization epitope-tagged Pbn1p: To deter- <br /> mine intracellular localization Pbn1p, epitope- <br /> tagged version Pbn1p constructed introducing             Figure 7.&#226;&#8364;&#8221;Features Pbn1p. (A) Hydropathy plot the <br /> the HA epitope N terminus protein coding           predicted Pbn1p polypeptide. (B) Coiled-coil formation prob- <br /> sequence (pBJ8706). This modified HA::Pbn1p                  ability generated using COILS 2.1 program (MTIK matrix, a <br />                                                                  d positions weighted, 14-residue window). (C) Cartoon <br /> functional, able complement Prb2 phe-           Pbn1p, 416-amino-acid polypeptide N-terminal <br /> notype pbn1-1. When functional HA::Pbn1p              coiled-coil domain ( ), sub-C-terminal transmembrane do- <br /> overproduced GAL1 promoter localized              main (j), predicted N-glycosylation sites (*). <br />  1286                                              R. R. Naik E. W. Jones <br />  <br />  <br />  <br />  <br />                                                                     Figure 9.&#226;&#8364;&#8221;Subcellular fractionation Pbn1p. (A) pbn1-1 <br />   Figure 8.&#226;&#8364;&#8221;Immunofluorescence localization overex-           cells (BJ8596) carrying pBJ8706 vector grown <br /> pressed HA::Pbn1p. pbn1-1 (BJ8596) cells carrying pBJ8706        presence galactose OD600nm 0.5 0.8 <br /> were grown selective medium containing galactose       converted spheroplasts. The spheroplasts lysed <br /> than glucose carbon source. Cells fixed processed    fractionated differential centrifugation. The lysate was <br /> for immunofluorescence using antibodies HA epi-      centrifuged 13,000 3 g, 15 min yielding pellet fraction <br /> tope Kar2p, ER marker protein. (A) DAPI staining       (P13) supernatant fraction (S13). The S13 fraction was <br /> localize nucleus; (B) immunofluorescence Kar2p;       subjected second centrifugation 100,000 3 g 30 <br /> (C) immunofluorescence HA::Pbn1p induced 6 hr; (D)        min resulting pellet fraction (P100) supernatant <br /> HA::Pbn1p induced 16 hr. No immunofluorescence           fraction (S100). Fractions separated SDS-PAGE fol- <br /> observed pbn1-1 cells carrying vector alone.              lowed immunoblot analysis using 12CA5 HA::Pbn1p <br />                                                                  affinity-purified anti-Sec61p. Empty vector served nega- <br />                                                                  tive control. (B) Membrane fractions prepared and <br /> that proteins colocalize. The induction               treated buffer, 0.1 m Na2CO3 (pH 11), 0.5 m NaCl, <br /> pBJ8706 greater 16 hr presence galac-         0.8 m urea, 1% Triton X-100. After incubation ice for <br />                                                                  20 min, samples separated supernatant (S) or <br /> tose led extensive proliferation ER membranes <br />                                                                  pellet (P) fractions centrifugation 100,000 3 g 30 <br /> (Figure 8D), feature observed ER pro-           min, subjected SDS-PAGE, immunoblotted anti- <br /> teins, HMG-CoA reductase cytochromes                 HA (12CA5). <br /> when overexpressed (Wright et al. 1988; Vergeres et <br /> al. 1993). <br />    Attempts visualize HA::Pbn1p fusion ex-           ER-membrane-associated protein (Stirling et <br /> pressed normal levels (on CEN plasmid             al. 1992). <br /> control native promoter) inser-            The hydropathy analysis Pbn1p shown Figure <br /> tion additional epitopes proved unsuccessful.           7 revealed single 19-amino-acid sub-C-terminal hydro- <br /> In addition antibodies raised Pbn1p detect           phobic region sufficient length span <br /> Pbn1p antigen immunoblots protein               lipid bilayer. In addition, PredictProtein program <br /> is overexpressed. One possible explanation ob-          (www.emblheidelberg.de/predictprotein/predictprote <br /> servation low level expression PBN1; b-galac-       in.html) predicted high probability amino <br /> tosidase activity levels expressed UASPBN1 -lacZ re-      acids 386&#226;&#8364;&#8220;404 form transmembrane domain. To <br /> porter fusion low (data shown).                     determine Pbn1p integral membrane pro- <br />    Pbn1p integral membrane protein: In conjunc-            tein, P13 pellet extracted 0.1 m Na2CO3 <br /> tion results obtained indirect immunofluo-           pH 11, 0.5 m NaCl, 0.8 m urea, 1% Triton X-100. As <br /> rescence, confirmed ER localization            1% Trition X-100 treatment extracted Pbn1p <br /> HA::Pbn1p subcellular fractionation. Cell-free ex-            pellet, infer Pbn1p integral mem- <br /> tracts subjected differential centrifugation          brane protein (Figure 9B). <br /> separate various organelles. Lysates fraction-             Pbn1p topology based glycosylation: The pre- <br /> ated 13,000 3 g pellet (P13), 100,000 3 g pellet        dicted molecular mass Pbn1p 48 kD, the <br /> (P100) 100,000 3 g supernatant (S100). The                 protein migrates 60-kD species. As possible <br /> majority 60-kD HA::Pbn1p P13             source increased mass glycosylation, we <br /> membrane fraction (Figure 9A). As control, nearly          examined expression HA::Pbn1p presence or <br /> of ER integral membrane protein Sec61p              absence tunicamycin. The cell extracts immuno- <br /> found P13 fraction, consistent          precipitated anti-HA (12CA5), resolved SDS- <br />                                                   The PBN1 Gene S. cerevisiae                                             1287 <br />  <br />  <br />  <br />  <br />                                                                       Figure 11.&#226;&#8364;&#8221;Translocation Pbn1p blocked sec61- <br />                                                                    2 mutant. Wild-type (BJ8594) pbn1-1 (BJ8569), sec61-2 (BJ- <br />     Figure 10.&#226;&#8364;&#8221;Pbn1p N-glycosylated. pbn1-1 cells (BJ8569)      3947), sec62-1 (BJ6879), sec65-1 (BJ6988) strains carrying <br /> carrying pBJ8706 grown Wickeram&#226;&#8364;&#8482;s medium con-              pBJ8706 grown 238 Wickeram&#226;&#8364;&#8482;s medium containing <br /> taining galactose carbon source 6 hr (OD600 5 0.6).     galactose carbon source 6 hr (OD600 5 0.6). Sphero- <br /> Spheroplasts preincubated presence tuni-            plasts preincubated presence tunicamycin (20 <br /> camycin (20 mg/ml) 10 min labeled Trans-         mg/ml) 10 min 378 followed additional 30 min <br /> 35 <br />    S 10 min, followed 10 min chase excess      incubation 378. Spheroplasts labeled Trans-35S for <br /> unlabeled methionine cysteine. Spheroplasts lysed         10 min, followed 10 min chase excess unlabeled <br /> and labeled proteins immunoprecipitated anti-HA.         methionine cysteine. Spheroplasts lysed labeled <br /> The immunoprecipitated samples subjected SDS-              proteins immunoprecipitated anti-HA. The immuno- <br /> PAGE autoradiography. pbn1-1 carrying vector         precipitated samples subjected SDS-PAGE autoradi- <br /> used negative control.                                      ography. The untranslocated Pbn1p species indicated an <br />                                                                    asterisk. <br />  <br />  <br /> PAGE visualized autoradiography. Immunopre- <br /> cipitates cells expressing HA::Pbn1p absence           SEC62 SEC65 effect translocation of <br /> of tunicamycin contained 60-kD species (Figure 10).            Pbn1p; migrated 60-kD species, had <br /> Treatment cells tunicamycin resulted ap-            received N-glycosyl chains. In contrast, an <br /> pearance faster migrating band molecular               unglycosylated, presumably cytosolic, 50-kD species of <br /> mass 50 kD, quite close calculated 48 kD. Thus           Pbn1p seen sec61 mutant non- <br /> Pbn1p appears carry glycosyl chains.         permissive temperature. Thus, introduction translo- <br /> All potential N-glycosylation sites             cation block sec61 prevents Pbn1p receiving its <br /> amino acids 64&#226;&#8364;&#8220;386, N-terminal transmembrane                N-glycosyl chains. This result indicates Sec61p <br /> domain. Therefore, taken data indicate              plays role insertion Pbn1p ER mem- <br /> that amino terminus lies ER lumen de-               brane; Sec62p Sec65p likely involved. <br /> spite lack signal sequence.                                  The pbn1-1 allele does affect Kex2p processing: <br />    A sec61 mutant accumulates unglycosylated Pbn1p:                Kex2p undergoes autoproteolytic cleavage its <br /> The experiments characterization Pbn1p sug-              propeptide ER, similar Prb1p. We decided to <br /> gested ER integral membrane protein. More-           test pbn1-1 mutation affected auto- <br /> over, amino terminus lumenally oriented, despite            processing Kex2p. In wild-type cells, Kex2p <br /> lacking signal sequence. The unusual topology                 precursor undergoes cotranslational signal sequence <br /> Pbn1p beckoned determine insertion               cleavage addition N-linked O -linked modifi- <br /> into ER required known components translo-              cations ER leading 129-kD I1 form. The <br /> cation machinery. We determined incorpo-               amino-terminal propeptide cleaved, yielding the <br /> ration Pbn1p ER membrane blocked                ER, 120-kD I 2 form. During later stages its <br /> sec mutants making use temperature-sensitive al-             lifetime, Kex2p undergoes elaborate modifications of <br /> leles SEC61, SEC62, SEC65 block translocation            O-linked chains Golgi, causing gradual in- <br /> at restrictive temperature. Sec61p key compo-             crease molecular weight approximately 126- <br /> nent translocation apparatus, mutations              kD J form (Wilcox Fuller 1991). In wild type, <br /> the SEC61 gene affect translocation proteins            processing 129-kD I1 form 126-kD J form, <br /> tested (Stirling et al. 1992). Sec62p required              observed 30-min chase period (Figure 12). <br /> post-translation translocation, Sec65p required           The pbn1-1 mutant showed similar kinetics matu- <br /> for cotranslational translocation proteins (Deshaies            ration Kex2p. These data suggest pbn1-1 <br /> and Schekman 1987; Stirling Hewitt 1992).                      allele does affect autocatalytic processing of <br /> pBJ8706 transformed sec mutants               Kex2p. <br /> effect mutations translocation Pbn1p               Pbn1p interacts propeptide Prb1p: The <br /> into ER determined (Figure 11). A block             feature distinguishes processing Prb1p from <br /> translocation Pbn1p result accumula-              vacuolar protease precursors auto- <br /> tion 50-kD unglycosylated form. Mutations                proteolytic processing propeptide ER. The <br />  1288                                                 R. R. Naik E. W. Jones <br />  <br />                                                                     observations indicate propeptide cata- <br />                                                                     lytic domain strongly interact other. These <br />                                                                     observations support earlier observations and, futh- <br />                                                                     ermore, confirm specificity interaction. <br />                                                                        Interestingly, propeptide capable inter- <br />                                                                     acting Pbn1p. The interaction Pbn1p and <br />                                                                     propeptide resulted 36 units b-galactosidase activ- <br />   Figure 12.&#226;&#8364;&#8221;Kinetic maturation Kex2p PBN1 (BJ8594)           ity growth diploid bearing plasmids encoding <br /> and pbn1-1 (BJ8596) strains. Spheroplasts labeled 2 min, <br /> excess cold methionine cysteine added sphero- <br />                                                                     Pbn1p propeptide fusions synthetic medium <br /> plasts harvested times shown start     containing 25 mm 3-AT (sector C). This interaction <br /> chase. The autoradiograph proteins labeled Trans-35S     propeptide Pbn1p specific, be- <br /> in vivo, immunoprecipitated anti-Kex2p, separated       cause negative control Snf4p (sector D) vector <br /> SDS-PAGE.                                                           did interact Pbn1p (data shown). <br />                                                                     No interaction catalytic domain Pbn1p <br />                                                                     observed; ,0.5 units b-galactosidase activity and <br /> ER localization Pbn1p effect pbn1-1               growth diploid bearing plasmids encoding the <br /> mutation autocatalytic processing Prb1p                Pbn1p catalytic domain fusions synthetic me- <br /> the ER led determine Pbn1p Prb1p                  dium containing 25 mm 3-AT seen (Sector G). Thus <br /> interact other. Attempts coimmunoprecipi-              activation lacZ HIS3 genes to <br /> tate HA::Pbn1p Prb1p unsuccessful. Based                fortuitous activation protein fusion or <br /> the predicted amino acid sequences, Prb1p contains                  nonspecific interactions. Thus results the <br /> four cysteine residues, catalytic domain           two-hybrid experiments indicate Pbn1p specifically <br /> one propeptide, Pbn1p contains                  interacts propeptide domain Prb1p vivo. <br /> cysteine residues. Immunoprecipitations carried in <br /> the absence reducing agents did reveal high <br /> molecular mass species, indicating Pbn1p                                     DISCUSSION <br /> Prb1p covalently linked disulfide bridges                   In study report identification charac- <br /> (data shown). However, transient interaction be-              terization Pbn1p, ER membrane protein is <br /> tween Pbn1p Prb1p likely de-                  involved post-translational processing vacu- <br /> tected using methods. As seen Figure 3B,               olar protease B precursor. Our results using two- <br /> the 76-kD precursor completely processed                 hybrid demonstrate Pbn1p interacts with <br /> 40-kD species 2 min chase, indicating            propeptide Prb1p. <br /> autocatalytic event rapid.                                          The propeptide acts intramolecular chaperone <br />    We used two-hybrid method test interaction            aids folding maturation Prb1p. It <br /> between pro regions Prb1p Pbn1p (Fields                  prevents N-glycosylation catalytic domain and <br /> and Song 1989; Durfee et al. 1993). We used DNA-                allows precursor processed PrA, presum- <br /> binding domain Gal4p fused propeptide                  ably catalyzing conformational change (Nebes and <br /> Prb1p Gal4p activation domain fused Pbn1p                Jones 1991). In pbn1-1 mutant observed little or <br /> and tested activation reporter genes fused            appearance 40-kD form PrB, product <br /> the GAL1UAS:lacZ HIS3. The results experi-             results autocatalytic cleavage event. Thus <br /> ments presented Figure 13. The propeptide                Prb1p fails undergo autoprocessing ER. The <br /> catalytic domain Prb1p individually tested              unprocessed precursor polypeptide exported the <br /> interaction Pbn1p (see Figure 13A). In addition,               cytosol, eventually degraded 26S pro- <br /> we tested interaction propeptide                teasome. Since recent evidence suggests misfolded <br /> and catalytic domain Prb1p control, previ-            secretory proteins exported ER the <br /> ous results indicated propeptide interacts            cytosol degradation (Hiller et al. 1996; Werner et <br /> the catalytic domain trans vivo (Nebes Jones              al. 1996; Sommer Wolf 1997), export Prb1p <br /> 1991). As shown Figure 13B, propep-                cytosol degradation pbn1-1 mutant <br /> tide catalytic domain present diploid,              suggests recognized misfolded protein by <br /> 82 units b-galactosidase activity obtained (sector          ER quality control mechanism. <br /> E). The positive control Snf1p Snf4p gave b-galac-               The failure Prb1p autoproteolytically pro- <br /> tosidase activity 47 units (sector A). Both diploids             cessed ER pbn1-1 mutant suggests that <br /> were able grow synthetic medium containing 25                 precursor polypeptide assumed correct <br /> mm 3-aminotriazole (3-AT). Thus reporter genes                 conformation favors autocatalysis. No autoproteo- <br /> were activated. The propeptide failed interact            lytic processing observed PRB1 translated <br /> with vector containing GAL4 sequences (data                vitro using cell-free extracts vivo transloca- <br /> shown) Snf1p control (sector B). These                  tion ER blocked sec61 mutant and <br />                                                   The PBN1 Gene S. cerevisiae                                               1289 <br />  <br />  <br />  <br />  <br />   Figure 13.&#226;&#8364;&#8221;Two-hybrid interaction propeptide Prb1p Pbn1p. (A) Schematic representation fusion <br /> plasmids used two-hybrid experiments. (B) Plasmids present diploid (Y187 3 Y190) indicated sector (pACT, <br /> GAL4 activation domain; pAS1, GAL4 DNA 5 binding domain). (C) Growth diploids bearing different plasmid combinations <br /> on SC minus Trp, Leu media, SC minus Trp, Leu, His media containing 25 mm 3-AT. The levels b-galactosidase activity <br /> (presented Miller units) produced plasmid combinations indicated outside circle. <br />  <br />  <br /> when PRB1 expressed E. coli (Mechler et al. 1988;             attaining autoproteolytically active conformation <br /> Moehle et al. 1989). The ER provide factor(s)                 help Pbn1p, precursor transiently <br /> that aid Prb1p attaining proper conformation             activated. The fact Prb1p propeptide interacts <br /> autoprocessing. The evidence presented clearly                 Pbn1p two-hybrid assay raises possibility <br /> suggests order autocatalytic cleavage            autoproteolytically active conformation be <br /> occur ER, propeptide interact                  achieved interaction. The interaction be- <br /> Pbn1p achieve conformation allows autocataly-             tween propeptide Pbn1p prevent the <br /> sis prevents export cytosol ER.                 escape free propeptide ER autocataly- <br />    The mechanism autoprocessing bacte-               sis, allowing catalytic domain interact <br /> rial subtilisin propeptide requires C terminus             propeptide trans: possibly interaction <br /> of propeptide close proximity                 leads silencing activated protease ER. <br /> active site triad catalytic domain. Upon attaining           It known coexpression propeptide and <br /> this conformation, precursor polypeptide able                catalytic domain separate molecules restores PrB <br /> to undergo autoproteolytic cleavage (Shinde                 activity prb1 mutant (Nebes Jones 1991), but <br /> Inouye 1995). Therefore, similarity               does pbn1-1 mutant (data shown). <br /> Prb1p bacterial subtilisins autocatalysis            This observation suggests Pbn1p provide an <br /> is known occur ER (Moehle et al. 1989), Prb1p             additional function downstream autocatalytic <br /> presumably undergoes similar conformation change                  event. <br /> in ER. Once inside ER, Prb1p capable                     Our favored model account observations <br />  1290                                           R. R. Naik E. W. Jones <br />  <br />  <br />  <br />                                                                                                 Figure      14.&#226;&#8364;&#8221;Proposed <br />                                                                                              model Pbn1p function. <br />                                                                                              Prb1p enters ER post- <br />                                                                                              translationally (1), pro- <br />                                                                                              peptide interacts Pbn1p <br />                                                                                              enters lumen (2). <br />                                                                                              The autocatalytic cleavage of <br />                                                                                              propeptide occurs after <br />                                                                                              Prb1p completely translo- <br />                                                                                              cated ER (3). After <br />                                                                                              autocatalysis, propeptide <br />                                                                                              transferred Pbn1p to <br />                                                                                              catalytic domain (4). The <br />                                                                                              propeptide-catalytic domain <br />                                                                                              complex transported out <br />                                                                                              ER pro- <br />                                                                                              cessing (5). <br />  <br />  <br />  <br />  <br /> presented paper Pbn1p contributes      vacuole (no genes branch essential <br /> autoprocessing Prb1p interacting propep-       date) accords function Pbn1p the <br /> tide ER drive Prb1p autoproteolytically     central secretory pathway. <br /> competent state (Figure 14). We propose pro-            The analysis Pbn1p function presented has <br /> peptide Prb1p interacts Pbn1p emerges        used single pbn1 mutation. The requirement PBN1 <br /> the lumenal ER. After entirety Prb1p       cell <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>survival</span> suggests Pbn1p plays global <br /> been completely translocated ER, propep-         role likely additional interact- <br /> tide autocatalytically cleaved precursor.         ing partners. Attempts isolate conditional alleles of <br /> Once autocatalysis occurred, free propeptide          PBN1 using conventional methods far un- <br /> bound Pbn1p transferred catalytic domain.        successful. Further analysis Pbn1p interact- <br /> The two-hybrid results presented article suggest      ing proteins analysis additional alleles PBN1 <br /> that catalytic domain strongly interacts pro-    allow definitively determine role <br /> peptide trans. This interaction lead silencing    Pbn1p ER. <br /> of protease prevent premature activation          Degradation cytosol dependent large pro- <br /> transit secretory pathway. Possibly    teolytic complexes, 26S proteasomes, re- <br /> interaction effects conformational change       sponsible majority protein turnover in <br /> allows PrA process C-terminal end precur-       cytosol (Demarini et al. 1995; Hiller et al. 1996; <br /> sor vacuole. We know catalytic domain         Plemper et al. 1997; Sommer Wolf 1997). Our <br /> leaves ER undergoes modifications         results clearly indicate degradation Prb1p the <br /> processing prior arrival vacuole (Nebes     pbn1-1 mutant involves proteasome, degrada- <br /> Jones 1991; Hirsch et al. 1992), propeptide      tion greatly reduced sen3-1 pbn1-1 mutant. A <br /> is subsequently degraded proteases vacuole          prerequisite degradation aberrant <br /> (Hirsch et al. 1992). The proposed interaction        proteins proteasome exported proteins <br /> the propeptide 39-kD intermediate raises          ubiquitinated (Hiller et al. 1996). Two ER mem- <br /> possibility targeting vacuole       brane-bound ubiquitin-conjugating enzymes, Ubc6p <br /> two interacting peptides &#226;&#8364;&#339;piggybacking&#226;&#8364;?         Ubc7p, identified yeast. The deletion <br /> polypeptide other, vacuolar targeting         UBC6 UBC7 leads defect degradation <br /> signal suffices polypeptides.                        aberrant proteins degradation <br />    Although localization studies using overproduced           MATa2 Hmg2p (reviewed Chen et al. 1993; <br /> proteins interpreted caution, facts     Hampton Bhakta 1997; Sommer Wolf 1997). <br /> the pbn1-1 mutation affects ER-dependent pro-             However, Werner et al. (1996), demonstrated ER- <br /> cessing Prb1p Prb1p recognized mis-       associated degradation mutant form human <br /> folded protein resulting export ER deg-       a-1-proteinase inhibitor, A1PiZ, known substrate of <br /> radation proteasome clearly support ER              ER-associated degradation yeast mammalian <br /> localization Pbn1p. In addition, fact PBN1        cells, independent Ubc6p Ubc7p ubiqui- <br /> is essential gene (PRB1 not) clearly indicates     tin-conjugating enzymes. This observation suggests that <br /> Pbn1p branch pathway Golgi          ubiquitination occur activity of <br />                                                The PBN1 Gene S. cerevisiae                                                     1291 <br />  <br /> other ubiquitin-conjugating enzymes grossly mis-             contain hydrophobic signal sequences been <br /> folded proteins require ubiquitination            shown bypass general Sec-dependent transloca- <br /> degradation proteasome. We observed moder-             tion machinery (reviewed Rapoport et al. 1996). That <br /> ate reduction rate Prb1p degradation ubc6D       inactivation components transloca- <br /> ubc7D pbn1-1 triple mutant, dramatic          tion machinery does affect Pbn1p translocation sug- <br /> as observed sen3-1 pbn1-1 mutant (data            gests Pbn1p, too, use nonclassical pathway <br /> shown).                                                         insertion ER membrane. Further studies of <br />    Pbn1p classified type I membrane protein        Pbn1p permit analysis mechanism mem- <br /> a large lumenal domain short cytoplasmic domain           brane insertion class type I membrane pro- <br /> despite absence N-terminal signal sequence.           teins. <br /> Ordinarily, type I integral membrane proteins               We thank M. Hochstrasser, R. Schekman, J. Brodsky for <br /> ER, like Wbp1p Ost1p, contain N-terminal sig-            providing strains, S. Elledge two-hybrid strains plasmids, <br /> nal sequence (Karaoglu et al. 1995). It predicted            A. Linstedt help fluorescence microscopy. We thank <br /> that signal-anchor proteins assume type I membrane            C. Woolford, A. Srivastava, members Jones labora- <br />                                                                 tory helpful discussions. This research supported grant <br /> (Nlumen-Ccyt ) topology segment C-terminal to <br />                                                                 National Institutes Health (DK-18090 E.W.J). <br /> the signal-anchor positively charged respect to <br /> the N-terminal flanking region (Haeuptle et al. 1989; <br /> Hartmann et al. 1989). This case Pbn1p. We                                LITERATURE CITED <br /> have demonstrated Pbn1p integral membrane <br />                                                                 Becherer, K. A., S. E. Rieder, S. D. Emr E. W. Jones, 1996 Novel <br /> protein lumenal N-terminal domain; therefore,                syntaxin homologue, Pep12p, required sorting lumenal <br /> the sub-C-terminal hydrophobic domain Pbn1p                   hydrolases lysosome-like vacuole yeast. Mol. Biol. Cell <br /> presumed serve membrane anchor posi-                7: 579&#226;&#8364;&#8220;594. <br />                                                                 Cepko, C. L., B. E. Roberts R. C. Mulligan, 1984 Construction <br /> tively charged segment C-terminal transmem-                  applications highly transmissible murine retrovirus shut- <br /> brane domain cytoplasmic tail. These results                tle vector. Cell 37: 1953&#226;&#8364;&#8220;1962. <br /> accord predictions.                                   Cohen, C., D. A. D. Parry, 1986 a-helical coiled-coils: wide- <br />                                                                     spread motif proteins. Trends Biochem. Sci. 11: 245&#226;&#8364;&#8220;248. <br />    The insertion membrane proteins believed            Chen, P., P. Johnson, T. Sommer, S. Jentsch M. Hochstrasser, <br /> occur machinery used translocation              1993 Multiple ubiquitin-conjugating enzymes participate the <br /> secretory proteins, mechanism poorly under-              vivo degradation yeast MAT alpha 2 repressor. Cell 74: <br />                                                                     357&#226;&#8364;&#8220;369. <br /> stood. In vitro experiments using proteins single        Demarini, D. J., F. R. Papa, S. Swaminathan, D. Ursic, T. P. Rasmus- <br /> membrane anchor (signal-anchor proteins) shown                 sen et al., 1995 The yeast SEN3 gene encodes regulatory sub- <br /> that hydrophobic segment cross-linked                 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> 26S proteasome complex required ubiquitin- <br />                                                                     dependent protein degradation vivo. Mol. Cell. Biol. 15: 6311&#226;&#8364;&#8220; <br /> Sec61p, suggesting insertion proteins              6321. <br /> ER membrane dependent Sec61p (Martoglio et                Deshaies, R. J., R. Schekman, 1987 Structural functional <br /> al. 1995). Proteins translocated ER            dissection Sec62p, membrane-bound component yeast <br />                                                                     endoplasmic reticulum protein import machinery. Mol. Cell. <br /> a protein channel (the translocon pore). Sec61p              Biol. 10: 6024&#226;&#8364;&#8220;6035. <br /> pore-forming subunit protein translocation chan-         Durfee, T., K. Becherer, P.-L. Chen, S.-H. Yeh, Y. Yang et al., 1993 <br /> nel ER membrane (Hanein et al. 1996). The                    The retinoblastoma protein associates protein phospha- <br />                                                                     tase type 1 catalytic subunit. Genes Dev. 7: 555&#226;&#8364;&#8220;569. <br /> insertion Pbn1p ER membrane dependent            Feldheim, D., K. Yoshimua, A. Admon R. Schekman, 1993 <br /> on Sec61p independent Sec62p Sec65p func-                Structural functional characterization Sec66p, new sub- <br /> tion. Since signal recognition particle (SRP) interacts             <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> polypeptide translocation apparatus yeast endo- <br />                                                                     plasmic reticulum. Mol. Biol. Cell. 4: 931&#226;&#8364;&#8220;939. <br /> with signal sequence nascent polypeptide          Fields, S., O. Song, 1989 A novel genetic detect <br /> is bound ribosome (Stirling Hewitt                 protein:protein interactions. Nature 340: 245&#226;&#8364;&#8220;246. <br /> 1992; Rapoport et al. 1996), absence N-termi-         Gluschankof, P., R. S. Fuller, 1994 A C-terminal domain <br />                                                                     conserved precursor processing proteases required intra- <br /> nal signal sequence Pbn1p account                  molecular N-terminal maturation prokex2 protease. EMBO J. <br /> lack dependence Sec65p, SRP component. It                  13: 2280&#226;&#8364;&#8220;2288. <br /> is unlikely Pbn1p inserted cotranslationally       Haeuptle, M.-T., N. Flint, N. M. Gough B. Dobberstein, 1989 <br />                                                                     A tripartite structure signals determine protein inser- <br /> the ER membrane, does contain signal               tion endoplasmic reticulum membrane. J. Cell Biol. 108: <br /> sequence membrane anchor                 1227&#226;&#8364;&#8220;1236. <br /> close C terminus buried        Hampton, R. Y., H. Bhakta, 1997 Ubiquitin-mediated regula- <br />                                                                     tion 3-hydroxy-3-methylglutaryl-CoA reductase. Proc. Natl. <br /> the ribosome termination translation occurred;              Acad. Sci. USA 94: 12844&#226;&#8364;&#8220;12948. <br /> therefore membrane insertion post-transla-              Hanein, D., K. E. S. Matlack, B. Jungnickel, K. Plath, K.-U. Kalies <br /> tional. The translocation Pbn1p remained unaffected              et al., 1996 Oligomeric rings Sec61p complex induced <br />                                                                     ligands required protein translocation. Cell 87: 721&#226;&#8364;&#8220;732. <br /> in sec62 mutant nonpermissive temperature;             Hartmann, E., T. A. Rapoport H. F. Lodish, 1989 Predicting <br /> Sec62p involved post-translational translocation          orientation eukaryotic membrane-spanning proteins. Proc. <br /> of proteins ER (Deshaies Shekman 1987).                Natl. Acad. Sci. USA 86: 5786&#226;&#8364;&#8220;5790. <br />                                                                 Hawthrone, D., R. Mortimer, 1960 Chromosome mapping <br /> The C-terminal tail-anchored membrane proteins                 Saccharomyces cerevisiae : centromere-linked genes. Genetics 45: <br /> possess N-terminal cytoplasmic domain                1085&#226;&#8364;&#8220;1110. <br />  1292                                                         R. R. Naik E. W. Jones <br /> Hill, J., K. A. Ian, G. Donald D. Griffiths, 1991 DMSO-                   Naik, R. R., V. Nebes E. W. Jones, 1997 Regulation the <br />     enhanced cell yeast transformation. Nucleic Acids Res. 19:              proteinase B structural gene PRB1 Saccharomyces cerevisiae. J. <br />     5791.                                                                         Bacteriol. 179: 1469&#226;&#8364;&#8220;1474. <br /> Hiller, M. M., A. Finger, M. Schweiger D. H. Wolf, 1996                   Nebes, V. L., E. W. Jones, 1991 Activation proteinase B <br />     ER degradation misfolded luminal protein cytosolic                precursor yeast Saccharomyces cerevisiae autocatalysis and <br />     ubiquitin-proteasome pathway. Science 273: 1725&#226;&#8364;&#8220;1728.                         internal sequence. J. Biol. Chem. 266: 22851&#226;&#8364;&#8220;22857. <br /> Hirsch, H. H., H. H. Schiffer D. H. Wolf, 1992 Biogenesis                 Plemper, R. K., S. Bohmler, J. Bordallo, T. Sommer D. H. <br />     yeast vacuole (lysosome). Mutation active site               Wolf, 1997 Mutant analysis links translocon BiP to <br />     vacuolar serine proteinase yscB abolishes proteolytic matura-             retrograde transport ER degradation. Nature 388: 891&#226;&#8364;&#8220;895. <br />     tion 73kDa precursor 41.5kDa pro-enzyme               Preston, R. A., P. S. Reinagel E. W. Jones, 1992 Genes required <br />     newly detected 41kDa peptide. Eur. J. Biochem. 207: 867&#226;&#8364;&#8220;876.                  vacuolar acidity Saccharomyces cerevisiae. Genetics 131: 551&#226;&#8364;&#8220; <br /> Hoffman, C. S., R. Winston, 1987 A ten-minute DNA prepara-                    558. <br />     tion yeast efficiently releases autonomous plasmids trans-       Pringle, J. R., A. E. M. Adams, D. Drubin B. K. Haarer, 1991 <br />     formation Escherichia coli. Gene 57: 267&#226;&#8364;&#8220;272.                              Immunofluorescence methods yeast. Methods Enzymol. 194: <br /> Jones, E. W., 1991 Tackling protease problem Saccharomyces                 565&#226;&#8364;&#8220;601. <br />     cerevisiae, pp. 428&#226;&#8364;&#8220;453 Methods Enzymology. Academic Press,         Rapoport, T. A., B. Jungnickel U. Kutay, 1996 Protein translo- <br />     San Diego.                                                                    cation eukaryotic endoplasmic reticulum bacterial <br /> Jones, E. W., D. G. Murdock, 1994 Proteolysis yeast                    inner membranes. Annu. Rev. Biochem. 65: 271&#226;&#8364;&#8220;303. <br />     vacuole, pp. 115&#226;&#8364;&#8220;134 Cellular Proteolytic Systems, edited A. J.     Sambrook, J., E. F. Fritsch T. Maniatis, 1989 Molecular Clon- <br />                                                                                   ing: A Laboratory Manual. Cold Spring Harbor Laboratory Press, <br />     Ciechanover A. L. Schwartz. Wiley-Liss, New York. <br />                                                                                   Cold Spring Harbor, NY. <br /> Jones, E. W., G. Zubenko R. Parker, 1982 PEP4 gene function <br />                                                                               Shinde, U., M. Inouye, 1995 Folding mediated intramo- <br />     required expression vacuolar hydrolases Sac- <br />                                                                                   lecular chaperone: autoprocessing pathway precursor re- <br />     charomyces cerevisiae. Genetics 102: 665&#226;&#8364;&#8220;677. <br />                                                                                   solved substrate assisted catalysis mechanism. J. Mol. Biol. <br /> Jones, E. W., G. C. Webb M. A. Hiller, 1997 Biogenesis and <br />                                                                                   247: 390&#226;&#8364;&#8220;395. <br />     function yeast vacuole, pp. 363&#226;&#8364;&#8220;470 Molecular Cellular <br />                                                                               Sikorski, R. S., P. Hieter, 1989 A shuttle vectors and <br />     Biology Yeast Saccharomyces: Cell Cycle Cell Biology, edited <br />                                                                                   yeast host strains designed efficient manipulation DNA in <br />     J. R. Pringle, J. R. Broach E. W. Jones. Cold Spring                   Saccharomyces cerevisiae. Genetics 122: 19&#226;&#8364;&#8220;27. <br />     Harbor Laboratory Press, Cold Spring Harbor, New York.                    Sommer, T., D. H. Wolf, 1997 Endoplasmic degradation: re- <br /> Karaoglu, D., S. Silberstein, D. J. Kelleher R. Gilmore, 1995                 verse protein flow return. FASEB J. 11: 1227&#226;&#8364;&#8220;1234. <br />     The Saccharomyces cerevisiae oligosaccharyltransferase: large he-       Stirling, C. J., E. W. Hewitt, 1992 The Saccharomyces cerevisiae <br />     tro-oligomeric complex endoplasmic reticulum. Cold                     SEC65 gene encodes component yeast signal recognition <br />     Spring Harbor  <br /> </body></html>