<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol.18 No.18 pp.5108&#226;&#8364;&#8220;5119, 1999 <br />  <br />  <br />  <br /> NuA4, essential transcription adaptor/histone H4 <br /> acetyltransferase complex containing Esa1p the <br /> ATM-related cofactor Tra1p <br />  <br />  <br /> Ste&#194;&#180;phane Allard, Rhea T.Utley1,                                      sitive sites regulatory elements (reviewed Steger and <br /> Julie Savard, Astrid Clarke2, Patrick Grant1,                         Workman, 1996). Genetic biochemical studies have <br /> Christopher J.Brandl3, Lorraine Pillus2,                              identified multiprotein complexes implicated this <br /> Jerry L.Workman1 Jacques Co&#203;&#8224;te&#194;&#180;4                                  remodeling. Complexes use energy ATP <br />                                                                       hydrolysis perturb chromatin structure include yeast <br /> Laval University Cancer Research Center, Ho&#203;&#8224;tel-Dieu Que&#194;&#180;bec,      mammalian SWI/SNF complexes (Co&#203;&#8224;te&#194;&#180; et al., 1994, <br /> Quebec City, Canada G1R 2J6, 1Howard Hughes Medical Institute,        1998; Kwon et al., 1994; Owen-Hughes et al., 1996; <br /> Department Biochemistry Molecular Biology, The Pennsylvania <br /> State University, University Park, PA 16802, USA, 2Department of <br />                                                                       Wang et al., 1996), Drosophila NURF, CHRAC ACF <br /> Biology, University California, San Diego, La Jolla, CA 92093,     complexes (Tsukiyama Wu, 1995; Ito et al., 1997; <br /> USA 3Department Biochemistry, University Western Ontario,   Varga-Weisz et al., 1997) yeast RSC complex <br /> London, Ontario, Canada N6A 5C1                                       (Cairns et al., 1996). <br /> 4Corresponding author                                                    Another group complexes modify histones chro- <br /> e-mail: jacques.cote@crhdq.ulaval.ca                                  matin order regulate transcription. Histone acetyl- <br />                                                                       transferases (HATs) histone deacetylases (HDs) play <br /> Post-translational acetylation histone H4 N-terminal               critical role positive negative regulation of <br /> tail chromatin associated                    transcription (reviewed Grunstein, 1997; Struhl, 1998). <br /> nuclear processes including transcription. We report                  A number studies correlated acetylation specific <br /> the purification characterization native multi-              lysine residues N-terminal tails histones the <br /> subunit complex (NuA4) yeast acetylates                     transcriptional activity chromatin (reviewed Turner <br /> nucleosomal histone H4. NuA4 apparent molecu-                  O&#226;&#8364;&#8482;Neill, 1995). Hyperacetylated histones accumulate <br /> lar mass 1.3 MDa. All conserved lysines                    actively transcribed chromatin (Hebbes et al., 1994), <br /> histone H4 acetylated NuA4. We                         hypoacetylated histones enriched transcrip- <br /> identified catalytic subunit complex                tionally repressed domains (Braunstein et al., 1993). <br /> product ESA1, essential gene required cell                  Acetylation histone N-terminal tails thought reduce <br /> cycle progression yeast. Antibodies Esa1p                  affinity nucleosomal DNA (Hong et al., 1993), <br /> specifically immunoprecipitate NuA4 activity                  increase binding transcription factors their <br /> the complex purified temperature-sensitive                     binding sites nucleosome (Lee et al., 1993; <br /> esa1 mutant loses acetyltransferase activity               Vettese-Dadey et al., 1996). Becker colleagues showed <br /> restrictive temperature. Additionally, identi-                histone hyperacetylation increases DNase I sensitivity <br /> fied subunit complex product                 conformational flexibility chromatin, as <br /> TRA1, ATM-related essential gene homologous                     facilitates transcription Drosophila hsp26 gene <br /> human TRRAP, essential cofactor c-Myc-                     chromatin (Krajewski Becker, 1998; Nightingale <br /> E2F-mediated oncogenic transformation. Finally,                   et al., 1998). <br /> ability NuA4 stimulate GAL4&#226;&#8364;&#8220;VP16-driven tran-                      Several transcriptional regulatory proteins intrinsic <br /> scription chromatin templates vitro lost              HAT activity. These include coactivators like Gcn5 <br /> in temperature-sensitive esa1 mutant. The function                (Brownell et al., 1996), p300/CBP (Bannister and <br /> of essential Esa1 protein HAT subunit                   Kouzarides, 1996; Ogryzko et al., 1996), p/CAF (Yang <br /> NuA4 presence Tra1p, putative transcrip-                 et al., 1996), ACTR (Chen et al., 1997), Src-1 (Spencer <br /> tion activator-interacting subunit, supports essential             et al., 1997) TAFII250 (Mizzen et al., 1996). All these <br /> link nuclear H4 acetylation, transcriptional                  proteins members multisubunit complexes involved <br /> regulation cell cycle control.                                    transcriptional activation interaction with <br /> Keywords: acetyltransferase/ESA1/nucleosome/                          sequence-specific activators. Similarly, HDs present <br /> transcription                                                         corepressor complexes (see Struhl, 1998 references <br />                                                                       therein). Other proteins having vitro HAT activity <br />                                                                       include Tip60 (Yamamoto Horikoshi, 1997) Esa1 <br />                                                                       (Smith et al., 1998; Clarke et al., 1999), members of <br /> Introduction                                                          MYST family putative acetyltransferases (Borrow <br /> Eukaryotic gene transcription regulated             et al., 1996; Reifsnyder et al., 1996; Hilfiker et al., 1997), <br /> ways, involve chromatin structure. Nucleo-              Hat1, cytosolic deposition-related HAT (Kleff <br /> somes suppress basal transcription initiation, increasing         et al., 1995; Parthun et al., 1996). The identification of <br /> dependence function sequence-specific activator             proteins HATs demonstrated differing <br /> proteins (reviewed Owen-Hughes Workman, 1994).                 substrate specificities. Recombinant Gcn5, p/CAF, <br /> Chromatin structures remodeled prior concurrent             TAFII250 src-1 acetylate mainly histone H3, whereas <br /> with transcription activation, creating DNase I-hypersen-             Tip60, Esa1p Hat1p prefer histone H4. CBP/p300 and <br />  <br /> 5108                                                                                   &#194;&#169; European Molecular Biology Organization <br />                                                                            Nucleosomal H4 acetylation Esa1p complex <br />  <br />  <br /> ACTR modify histone H3 H4 (see respective              yeast extracts (Grant et al., 1997). When fraction <br /> references above). Importantly, ability acetylate        separated monoQ column, distinct <br /> histone tails context chromatin thought      native nucleosomal HATs identified. An example of <br /> a necessary characteristic transcription-related HATs.       fractionation shown Figure 1A. Two HATs, Ada <br />    We previously identified native high molecular     SAGA, preferentially acetylate N-termini H3/H2B <br /> weight complexes yeast extracts contain HAT           (Figure 1A, fractions 22&#226;&#8364;&#8220;24 42) shown <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> activities function polynucleosomal substrates         contain Gcn5p catalytic subunit other <br /> (Grant et al., 1997). Gcn5p catalytic subunit         transcription regulatory proteins (Grant et al., 1997). The <br /> two complexes, termed SAGA Ada. Both               nucleosomal HATs character- <br /> complexes contain ADA2 ADA3 gene products,             ized aside substrate specificities. NuA3, like <br /> confirming role native transcription adaptor com-      Ada SAGA, preferentially acetylates N-termini <br /> plexes, modify primarily histone H3 chromatin,           nucleosomal H3, fails modify H2B (Figure 1A, <br /> and H2B lesser extent. The SAGA complex               fractions 36&#226;&#8364;&#8220;38). NuA4 unique HATs <br /> contains Spt20/Ada5, Spt3 Spt7, group transcrip-       targets mainly N-termini histone H4 in <br /> tional regulators thought involved helping TBP         oligonucleosomes, H2A lesser extent (Figure 1A, <br /> function (Winston, 1992), subset TBP-associated        fractions 24&#226;&#8364;&#8220;28). Further fractionation NuA4 gel <br /> factors (TAFIIs) originally identified TFIID complex     filtration Superose 6 indicated apparent molecular <br /> (Grant et al., 1998a). The presence ADA, SPT             weight ~1.3 MDa (Figure 1B, fraction 21). Therefore, <br /> TAFII gene products SAGA complex indicates               like SAGA Ada complexes, NuA4 high <br /> the importance histone acetylation transcription <br />                                                                 molecular weight multisubunit complex. <br /> activation, reaction mediated interaction tran- <br /> scription activators general transcription factors. It is <br />                                                                 NuA4 produce fully tetra-acetylated histone <br /> important note presence Gcn5p native <br /> complexes stimulates activity chromatin substrates,      H4 oligonucleosomes <br /> suggesting presence subunit(s) involved chro-         To characterize acetyltransferase specificity of <br /> matin binding and/or stimulating Gcn5p activity. Finally,       NuA4, compared activity HeLa oligonucleo- <br /> we identified HAT activities, termed             somes, HeLa free histones yeast free histones. As <br /> complexes 2 3 (Grant et al., 1997), acetylate         shown above, NuA4 efficiently acetylates H4/H2A in <br /> nucleosomal histones H4 H3, respectively. Since         oligonucleosomes, Ada complex targets H3/ <br /> subunit composition complexes unknown,             H2B (Figure 2A, lanes 2 3). When oligonucleosomes <br /> they named NuA4 (Nucleosome acetyltransferase              replaced free histones substrate reaction, <br /> of histone H4) NuA3 according substrate            specificity NuA4 mainly histone H4, while <br /> specificity.                                                    Ada targeted mainly H3 (Figure 2A, lanes 5 6). <br />    In report purification characteriza-    However, important note free histones <br /> tion NuA4 complex. Using genetic biochemical         H3 acetylated NuA4, modified <br /> approaches, Esa1p catalytic subunit         oligonucleosomes (compare lanes 3 6, also <br /> of 1.3 MDa native HAT complex. We demonstrate         Figure 4D, lanes 1 5). NuA4 activity tested <br /> that stimulation GAL4&#226;&#8364;&#8220;VP16-driven transcription           yeast free histones gave similar results the <br /> chromatin templates NuA4 requires functional Esa1p         ones obtained human free histones, expected <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> subunit. Since ESA1 gene product essential cell      strongly conserved lysine residues N-terminal <br /> cycle progression yeast (Clarke et al., 1999),          tails histones (Figure 2A, compare lanes 6 9). <br /> demonstrates purification native essential          Histone H4 isoforms acetylated specific lysines (K5, <br /> HAT complex. Furthermore, native           K8, K12 K16) selectively localized Drosophila <br /> complex targeting mainly histone H4 chromatin,         chromosomes (Turner et al., 1992). For example, <br /> are thoroughly studied N-termini. Esa1p             acetylated lysine 16 H4 male X chromosome is <br /> a direct homolog MOF, putative acetyltransferase           associated dosage compensation Drosophila (Bone <br /> necessary X chromosome hypertranscription H4            et al., 1994), heterochromatin acetylated <br /> acetylation Drosophila males (dosage compensation)           lysine 12 (Braunstein et al., 1996). Nevertheless, <br /> (Hilfiker et al., 1997). We identified ATM/       potentially active euchromatin modified the <br /> PI-3 kinase-related protein Tra1p subunit NuA4.         H4 acetylatable lysines, silent heterochro- <br /> Tra1p essential cell viability (Saleh et al., 1998)      matin hypoacetylated (O&#226;&#8364;&#8482;Neill Turner, 1995). Unlike <br /> and homolog human protein TRRAP, c-Myc-             H4 chromatin, newly synthesized H4 diacetylated at <br /> and E2F-interacting protein essential oncogenic       K5/K12, thought involved chromatin <br /> activities (McMahon et al., 1998). These facts          assembly (Sobel et al., 1995). Thus, important to <br /> finding yeast NuA4 stimulates vitro transcription       determine specific lysine N-terminal tails of <br /> from chromatin templates implicate NuA4 transcrip-         nucleosomal H4 acetylated NuA4. Oligonucleo- <br /> tional adaptor/regulatory complex essential cell cycle <br />                                                                 somes incubated presence absence of <br /> progression. <br />                                                                 NuA4 radioactive acetyl-CoA. Histones then <br />                                                                 separated Triton&#226;&#8364;&#8220;acetic acid&#226;&#8364;&#8220;urea (TAU) gel which <br /> Results                                                         resolve different levels H4 acetylation (0&#226;&#8364;&#8220;4 <br /> Identification native yeast HAT complex               acetylated lysines). In left panel Figure 2B, the <br /> acetylates nucleosomal histone H4                               absence NuA4, H4 migrates mainly non-acetylated <br /> In previous report shown nickel&#226;&#8364;&#8220;agarose          form 10&#226;&#8364;&#8220;15% population monoacetylated. In <br /> resin binds native nucleosomal HAT activities           presence NuA4, mono-, di-, tri- tetra-acetylated <br />  <br />                                                                                                                       5109 <br />  S.Allard et al. <br />  <br />  <br />  <br />  <br /> Fig. 1. Fractionation different native nucleosomal HATs yeast. (A) Gel HAT assay fractions monoQ column using <br /> oligonucleosomes substrate. One microgram oligonucleosomes incubated 1 &#194;&#181;l fraction presence radioactive acetyl-CoA <br /> for 30 min 30&#194;&#176;C; reaction loaded 18% SDS&#226;&#8364;&#8220;polyacrylamide gel, Coomassie Blue stained treated fluorography detect <br /> histone acetylation (upper panel). The different activities identified showing respective histone specificity (corresponding histone <br /> labeled right). The lower panel shows equal histones present reaction revealed Coomassie Blue staining the <br /> gel. (B) Gel filtration calibrated Superose 6 H4-specific monoQ pool. HAT activity fractions oligonucleosomes shown the <br /> fluorogram. NuA4 high molecular weight complex ~1.3 MDa. The H3-specific Ada complex ~0.8 MDa. Two markers used for <br /> calibration indicated. <br />  <br />                                                                              using synthetic peptides corresponding 24 <br />                                                                              amino acids H4. By using lysine acetylated lysine <br />                                                                              chemical synthesis, different peptides were <br />                                                                              produced, having different lysines available <br />                                                                              acetylation. Incubation NuA4 liquid assay <br />                                                                              quantification showed conserved <br />                                                                              lysine residues H4 tail independently <br />                                                                              acetylated (data shown). <br />  <br />                                                                              Esa1p functions catalytic subunit NuA4 <br />                                                                              Besides NuA4, proteins shown to <br />                                                                              acetylate primarily H4 N-termini vitro. These <br />                                                                              include yeast Hat1p, major deposition-related cyto- <br />                                                                              plasmic activity (Parthun et al., 1996), human HIV Tat- <br />                                                                              interacting protein Tip60 (Yamamoto Horikoshi, 1997) <br />                                                                              recently identified yeast essential protein <br /> Fig. 2. Substrate specificity NuA4 HAT complex.                       Esa1 (Smith et al., 1998; Clarke et al., 1999). A hat1- <br /> (A) Comparison NuA4 activity mammalian <br /> oligonucleosomes (lanes 1&#226;&#8364;&#8220;3) free histones (lanes 4&#226;&#8364;&#8220;6), yeast        null mutant strain known wild-type level <br /> free histones (lanes 7&#226;&#8364;&#8220;9). The Ada complex shown                 NuA4 activity (L.Duggan, P.Grant, J.Savard, J.Co&#203;&#8224;te&#194;&#180;, <br /> comparison. One microgram substrate 0.5 &#194;&#181;l Superose 6              J.Workman S.Berger, unpublished data). While these <br /> respective peak fractions used HAT assay. NuA4                   proteins weak activities histones nucleo- <br /> specificity unchanged yeast histones used instead of <br /> human ones. Also H3-specific acetylation NuA4 apparent <br />                                                                              somes, recombinant Tip60 Esa1p very <br /> when chromatin used substrate. (B) TAU gel HAT assay showing           similar substrate specificity free histones, i.e. <br /> that NuA4 able create fully tetra-acetylated histone H4             H4..H3.H2A, identical NuA4 specificity <br /> chromatin. Oligonucleosomes (8 &#194;&#181;g) incubated absence          substrate (Figures 2A 4D). We decided <br /> presence purified NuA4 (over columns, Figure 5A) and <br /> radioactive acetyl-CoA 1 h 30&#194;&#176;C. Reactions loaded on <br />                                                                              investigate possibility Esa1p contributed the <br /> a TAU gel distinguish N-terminal acetylation levels histone        catalytic activity NuA4 HAT complex. We produced <br /> H4. The left panel shows Coomassie Blue-stained gel              antiserum 14 amino acids Esa1p <br /> position non-acetylated H4. Note oligonucleosomes        analyzed fractions Western analysis. A monoQ <br /> contain 10&#226;&#8364;&#8220;15% monoacetylated H4 traces diacetylated           fractionation nickel&#226;&#8364;&#8220;agarose eluate used look <br /> form. The right panel shows NuA4-dependent incorporation of <br /> radioactive acetate H4, producing mono-, di-, tri- tetra-             potential cofractionation (Figure 3A). As previously <br /> acetylated forms, visualized Coomassie Blue staining           shown, Ada SAGA complexes co- <br /> (left panel).                                                                elute tightly Ada2p Swi2p, subunit the <br />                                                                              SWI/SNF complex, did co-elute HAT activity. <br /> forms produced. This clearly shown auto-                      The antiserum directed Esa1p detected strict <br /> radiography (right panel, Figure 2B), demonstrates                     cofractionation 55 kDa band NuA4 activity. <br /> the NuA4-dependent incorporation radioactive acetate                      From predicted molecular weight Esa1p (52.5 kDa) <br /> in levels H4 acetylation. These data                     fact antibody specifically recognized <br /> reveal NuA4 able create fully tetra-acetylated                    recombinant Esa1p (data shown), concluded that <br /> histone H4 nucleosomes. Specific acetylation                       55 kDa signal likely represent Esa1 protein. <br /> lysines position 5, 8, 12 16 confirmed                       To confirm NuA4&#226;&#8364;&#8220;Esa1p relationship, prepared <br /> microsequencing reported recently (Ohba et al., 1999).                    resins containing cross-linked pre-immune immune <br /> Since lysine residues H4 tails                    serum. When NuA4 fraction Superose 6 column <br /> acetylated NuA4, wanted determine                      incubated pre-immune resin, depletion <br /> was cascade lysine acetylation specific                     HAT activity supernatant detected <br /> order histone tail. This partially answered                  (Figure 3B). By contrast, anti-Esa1p immune <br />  <br /> 5110 <br />                                                                                           Nucleosomal H4 acetylation Esa1p complex <br />  <br />  <br />  <br />  <br /> Fig. 3. Esa1p co-elutes NuA4 complex anti-Esa1p serum immunoprecipitates NuA4 activity. (A) Fluorogram HAT assay on <br /> monoQ fractions similar Figure 1 (upper panel) immunoscreening Esa1p, Ada2p Swi2p corresponding fractions indicated (lower <br /> panels). One microliter fractions used HAT assay 5 &#194;&#181;l Western blot analysis. As shown previously Ada SAGA <br /> complexes co-elute Ada2p, direct interaction Gcn5p complexes (Grant et al., 1997). Interestingly, Esa1p signal by <br /> Western blotting co-elutes perfectly NuA4 activity. (B) Liquid HAT assay showing purified NuA4 activity depleted anti-Esa1p <br /> serum pre-immune serum. NuA4-containing fraction 21 Superose 6 column Figure 1B incubated pre-immune <br /> serum cross-linked protein A beads. After incubation 4&#194;&#176;C supernatant incubated anti-Esa1p beads similar conditions. <br /> Equivalent amounts initial, pre-immune immune supernatants assayed HAT activity oligonucleosomes liquid scintillation <br /> counting. The results presented histogram standard deviation independent experiments. (C) Gel HAT assay showing the <br /> depletion NuA4 activity partial recovery anti-Esa1p beads. Experiment similar (B) sample used fraction <br /> containing NuA4 Ada activities. Also beads tested HAT assay results visualized fluorography. As <br /> expected pre-immune beads did deplete HAT activity initial sample (lane 2 versus 1). In contrast, H4-specific acetylation <br /> (NuA4) clearly depleted anti-Esa1p beads (lane 4 versus 1), H3-specific acetylation (Ada) unaffected. Some NuA4 activity <br /> could detected immune beads tested acetylation (lower activity depleted presumably efficient <br /> assay and/or steric hindrance using beads, lane 5). <br />  <br />  <br /> resin used, strong depletion NuA4 activity                     bearing disrupted copy candidate gene (Grant <br /> was observed. We confirmed specificity deple-                    et al., 1997). However, Esa1p essential growth <br /> tion using cruder fraction containing NuA4                     yeast (Smith et al., 1998; Clarke et al., 1999), used <br /> Ada HATs looked activities antibody                   strain ESA1 ORF deleted, carried <br /> beads binding reaction (Figure 3C). Again, NuA4                    wild-type temperature-sensitive allele of <br /> activity histones H4/H2A affected                          ESA1 plasmid. The <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> frameshift amino <br /> incubation pre-immune beads incubation                        acid 414 Esa1p (Clarke et al., 1999). Mutant wild- <br /> with anti-Esa1p beads resulted depletion (Figure 3C,                      type strains grown permissive temperature to <br /> lanes 1, 2 4). Importantly, H3/H2B HAT activity                       early log phase. At time half culture was <br /> Ada unaffected incubation resins.                     shifted non-permissive temperature incubations <br /> Moreover, partially detect specific recovery                    continued 4 h. As expected, temperature- <br /> of NuA4 activity anti-Esa1p beads (Figure 3C,                         sensitive culture placed 37&#194;&#176;C rapidly stopped growing <br /> lane 5, weaker activity likely efficient              wild-type culture unaffected (data not <br /> HAT assay beads suspension, steric hindrance                     shown; Clarke et al., 1999). Whole cell extracts were <br /> to IgG binding Esa1). These data strongly suggest                    prepared fractionated nickel&#226;&#8364;&#8220;agarose resin. <br /> Esa1p NuA4 complex.                                           Equivalent amounts proteins growth condi- <br />    The step determine role Esa1p                       tion fractionated monoQ column and <br /> the nucleosomal HAT activity NuA4. The direct                        assayed HAT activity (Figure 4A). When wild- <br /> way purify HAT activity yeast                         type strain analyzed, typical successive elution of <br />  <br />                                                                                                                                             5111 <br />  S.Allard et al. <br />  <br />  <br />                                                                            Ada, NuA4 NuA3 HAT complexes seen <br />                                                                            (Figure 4A, lower panel). The ts-esa1 mutant strain grown <br />                                                                            room temperature gave similar results, for <br />                                                                            fact NuA4 HAT activity weaker Ada <br />                                                                            NuA3 equivalent activity detected the <br />                                                                            wild-type strain (Figure 4A, upper panel). When the <br />                                                                            extract prepared mutant strain been <br />                                                                            shifted 37&#194;&#176;C, Ada NuA3 activities again <br />                                                                            unchanged compared permissive growth condition <br />                                                                            wild-type strain (middle panel). However, no <br />                                                                            NuA4 activity detected. This demonstrates clearly the <br />                                                                            essential role Esa1p nucleosomal HAT activity <br />                                                                            NuA4. The variation NuA4 activities the <br />                                                                            wild-type mutant strains different <br />                                                                            levels episomal Esa1 protein expression, Western <br />                                                                            analysis anti-Esa1p serum showed equivalent <br />                                                                            amounts protein samples (Figure 4C). Also the <br />                                                                            equivalent mutation bacterially expressed recombinant <br />                                                                            Esa1p shown destroy HAT activity free histones <br />                                                                            (Figure 4D, lanes 2 3; Clarke et al., 1999). In fact, <br />                                                                            purify NuA4 complex the <br />                                                                            ts-esa1 mutant strain grown room temperature since <br />                                                                            weak remaining HAT activity highly unstable and <br />                                                                            lost couple freeze&#226;&#8364;&#8220;thaw treatments (data <br />                                                                            shown). <br />                                                                               The vivo inactivation Esa1p indicates is <br />                                                                            directly indirectly necessary NuA4 activity vivo. <br />                                                                            In fact, Esa1p detected Western blotting <br />                                                                            fractions prepared mutant strain grown non- <br />                                                                            permissive temperature (data shown). To distinguish <br />                                                                            possibilities tested temper- <br />                                                                            ature sensitivity mutant Esa1p reproduced <br /> Fig. 4. Temperature-sensitive mutation Esa1p destroys NuA4 HAT          native protein vitro, using partially purified <br /> activity non-permissive temperature. (A) In vivo temperature        fractions. We used NuA4 monoQ fractions Esa1 <br /> sensitivity NuA4 Esa1-414 ts strain. Gel HAT assays            wild-type mutant strains grown room temperature <br /> monoQ fractions prepared yeast strains producing wild-type         incubated 37&#194;&#176;C 15 min performing <br /> ts-Esa1p. Equivalent amounts proteins corresponding fractions <br /> in strains/growth conditions assayed; incubation room <br />                                                                            HAT assay (Figure 4B). Using wild-type fraction, <br /> temperature (RT). Cultures grown RT exponential growth          obvious decrease NuA4 HAT activity detected <br /> and 37&#194;&#176;C RT 4 h indicated. Ada NuA3         treatment 37&#194;&#176;C (Figure 4B, compare lanes 1 and <br /> activity given internal controls. Note NuA4 ts strain   3). In contrast, ts-Esa1 fraction completely lost its <br /> grown RT (upper panel) 10 times active NuA4        NuA4 HAT activity treatment 37&#194;&#176;C <br /> from wt strain (lower panel). Shifting ts strain 37&#194;&#176;C <br /> provokes loss NuA4 activity complexes       (Figure 4B, compare lanes 2 4). Since mutant <br /> unaffected (middle panel). (B) In vitro temperature sensitivity NuA4    NuA4 complex falling apart heat <br /> in fraction purified esa1-ts strain. MonoQ peak fractions     treatment suggested vivo data, performed <br /> ESA1 wt ts mutant strains grown RT incubated non-       assay using free histones determine other <br /> permissive temperature time indicated performing the <br /> HAT assay RT oligonucleosomes (lanes 1&#226;&#8364;&#8220;4) free <br />                                                                            HAT proteins present complex (Figure 4B, <br /> histones (lanes 5&#226;&#8364;&#8220;8) substrates. Three times ts fraction used     lanes 5&#226;&#8364;&#8220;8). This based previously published data <br /> than wt achieve closer levels NuA4 activity. Note that,        showing monomeric HAT proteins acetylate free <br /> NuA4 activity specifically lost ts-Esa1 mutant fraction       histones easily (e.g. Gcn5p, Hat1p, Esa1p, p300/CBP, <br /> treatment 37&#194;&#176;C, weaker H3-specific HAT activity coming             p/CAF, TAFII250, SRC-1) need part <br /> contaminating Ada complex remains unaffected (lane 8 versus 6). <br /> (C) Anti-Esa1p Western blotting shows amounts wt ts        native complex work efficiently chromatin (e.g. <br /> Esa1p comparable monoQ fractions; NuA4                Esa1p NuA4, Figure 4D; Gcn5p SAGA/ <br /> complex physically destabilized mutation. The total     Ada, Grant et al., 1997). NuA4 activity free <br /> amounts proteins loaded monoQ fraction 26 ESA1 wt          histones mutant fraction lost heat <br /> and ts mutant strains grown RT. Note ts-Esa1p (Esa1-414p) <br /> migrates quickly wt <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> <br />                                                                            treatment wild-type complex unaffected <br /> making 22 amino acids shorter. (D) Comparison NuA4 activity       (compare lanes 6 8 5 7). As convenient <br /> bacterially produced Esa1p Esa1-414p. Clarified bacterial extracts     internal control, weaker H3-specific HAT activity <br /> containing approximately recombinant Esa1               present fraction (from contaminating Ada <br /> proteins incubated 1 &#194;&#181;g HeLa free histones                 complex) remained unaffected heat treatment. <br /> oligonucleosomes HAT assay conditions. Unlike purified NuA4 <br /> (after columns, Figure 5A), recombinant Esa1p able      Taken together, data obtained anti-Esa1p <br /> acetylate H4 chromatin. Also ts-mutant completely inactive    antibodies effect Esa1 ts-mutant on <br /> bacteria free histones, suggesting subunit(s)      NuA4 activity, demonstrate function Esa1p a <br /> NuA4 gives Esa1p chromatin capabilities.                                   true nucleosomal HAT NuA4 complex. More- <br />  <br /> 5112 <br />                                                                            Nucleosomal H4 acetylation Esa1p complex <br />  <br />  <br /> over, essential requirement functional Esa1p         27.3% identity/58.9% similarity), necessary cofactor for <br /> both nucleosomal free HAT activity native          c-Myc E2F-mediated cell transformation (Grant et al., <br /> NuA4 complex strongly argues primary             1998b; McMahon et al., 1998; Saleh et al., 1998). Using <br /> catalytic subunit NuA4.                                      Tra1p antiserum, determined 400 kDa <br />    Since NuA4 binds nickel column           band heparin fraction fact Tra1p. As shown <br /> the presence His-tagged protein complex,         lower panel Figure 5B, Tra1p high <br /> was possible portion cellular Esa1p           molecular weight band perfectly co-eluting with <br /> associated NuA4 bind column           NuA4 activity Esa1p. Since serum raised <br /> (we detect Esa1p Western blotting            N-terminal peptide sequence Tra1p, potential <br /> cell extract, data shown). We answered question        cross-reactivity ATM-related proteins avoided <br /> by using strain expressing His-tagged version Esa1p      (Grant et al., 1998b). The region homology Tra1p <br /> (i.e. Esa1p molecules bind nickel column).     human homolog ATM/PI-3 kinase-related <br /> Since did new Esa1p Western blot signal          proteins limited ~300 amino acids C-terminus <br /> or stronger NuA4 activity appearing subsequent           (McMahon et al., 1998). Interestingly, amino acids <br /> fractions, concluded cellular Esa1p associated   required PI-3 related kinase activity this <br /> with NuA4 complex (data shown). As Esa1p             region present Tra1p TRRAP (McMahon <br /> required cell cycle progression, makes NuA4        et al., 1998). <br /> first essential nucleosomal HAT identified yeast.               Strict co-elution Tra1p Esa1p NuA4 activity <br />    Unlike NuA4, recombinant Esa1p unable acetylate        confirmed earlier columns including DNA&#226;&#8364;&#8220; <br /> H4 nucleosomes conditions (Figure 4D,            cellulose, histone&#226;&#8364;&#8220;agarose Superose 6 (Figure 5C). <br /> compare lanes 1 5 2 6) (Smith et al., 1998;        Physical interaction Tra1p NuA4 confirmed <br /> Clarke et al., 1999). This suggests proteins      immunoprecipitation experiments (Figure 6). Affinity- <br /> the NuA4 complex nucleosomal capabilities Esa1p         purified Esa1p IgG produced cross-linked to <br /> HAT activity presenting H4/H2A N-termini            protein A&#226;&#8364;&#8220;Sepharose. A partially purified NuA4 fraction <br /> within nucleosomes. Obvious candidates kind         (Ni21&#226;&#8364;&#8220;agarose/monoQ/Superose 6 fraction 20) pre- <br /> histone binding activity Hat2 Cac3/Msi1 proteins,      cleared incubated &#206;&#177;Esa1p resin. After <br /> respectively subunits cytoplasmic HAT complex            washes, equivalent amounts initial sample, bead <br /> and chromatin assembly factor-1 complex,          supernatant (FT) beads tested presence <br /> proposed involved histone binding (Parthun et al.,     Esa1p Tra1p Western blotting (Figure 6A). As <br /> 1996; Kaufman et al., 1997). We purified NuA4              expected 80&#226;&#8364;&#8220;90% depletion NuA4 activity <br /> yeast strains bearing wild-type disrupted HAT2           (data shown), Esa1p signal com- <br /> MSI1 genes, failed detect effect NuA4             pletely recovered bound fraction (Figure 6A, com- <br /> nucleosomal HAT activity (data shown). In agreement <br />                                                                 pare lane 3 versus lane 1, lower panel). Strikingly, Tra1p <br /> with this, recently suggested Hat2 Cac3/ <br />                                                                 signal follows exactly <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, the <br /> Msi1 involved binding core histones soluble <br />                                                                 initial signal Esa1p-bound fraction. This <br /> form, nucleosomal DNA inhibited binding <br />                                                                 shows Tra1p present fraction a <br /> (Verreault et al., 1998). <br />                                                                 direct indirect interaction Esa1p. To confirm <br />    To demonstrate association Esa1p with <br /> NuA4 visualize subunit complexity, purified       interaction, performed reciprocal co- <br /> the complex seven columns (Figure 5A). We            immunoprecipitation experiments. We prepared Superose <br /> continued co-fractionation Esa1p HAT activity       6 NuA4 fractions (as Figure 6A) yeast strains <br /> of NuA4 columns. Figure 5B shows             expressing natural Tra1p N-terminal Myc-tagged <br /> protein profile NuA4-containing fractions       version, capable sustaining growth (Saleh <br /> final column, heparin&#226;&#8364;&#8220;Sepharose. No purification        et al., 1998). When fractions incubated anti- <br /> was obtained binding NuA4 glutathione S-transferase       myc beads assayed NuA4 activity, very <br /> (GST)&#226;&#8364;&#8220;H4 tail GST&#226;&#8364;&#8220;VP16 resins (data shown). As           different results obtained (Figure 6B). With the <br /> expected large size complex,         untagged fraction (Figure 6B, compare lanes 1&#226;&#8364;&#8220;3) NuA4 <br /> numerous polypeptides purified fractions.           activity stayed supernatant fraction expected. On <br /> Importantly, NuA4 co-eluting bands appear           hand, using myc-tagged fraction, majority <br /> to present similar amounts, including band       NuA4 activity disappeared supernatant and <br /> predicted size Esa1p (55 kDa) corresponds           recovered anti-myc beads (Figure 6B, compare <br /> the Western blot signal (arrowhead Figure 5B)            lanes 4&#226;&#8364;&#8220;6; note HAT activity physically linked to <br /> follows H4-specific HAT activity.                           beads underestimated compared equivalent <br />                                                                 fully soluble one). The remaining NuA4 activity present <br /> Essential ATM-related cofactor Tra1p                       supernatant likely explained presence <br /> component NuA4                                               endogenous untagged Tra1p expressed cell <br /> Interestingly, high molecular weight protein         (myc&#226;&#8364;&#8220;Tra1p expressed centromeric plasmid in <br /> detected purified fraction (asterisk             wild-type background) (Saleh et al., 1998). These <br /> Figure 5B). Only yeast ORF produce             immunoprecipitation experiments confirm stable asso- <br /> size protein (~400 kDa). The SAGA complex              ciation Tra1p NuA4 complex. The relative <br /> very recently shown contain Tra1p, essential ATM-         Tra1p NuA4 complex versus SAGA <br /> related 433 kDa protein homologous human TRRAP               complex addressed. This fact <br /> (transformation/transcription domain-associated protein;        Tra1p complexes <br />  <br />                                                                                                                      5113 <br />  S.Allard et al. <br />  <br />  <br />  <br />  <br /> Fig. 5. Purification NuA4 complex. (A) The NuA4 complex purification scheme seven chromatographic steps. (B) Silver stained SDS&#226;&#8364;&#8220; <br /> PAGE, fluorogram HAT assays oligonucleosomes anti-Esa1p anti-Tra1p Western blot analysis fractions last <br /> column, heparin&#226;&#8364;&#8220;Sepharose. Twenty-four microliters fractions used gel staining 15 2 &#194;&#181;l used Western blot and <br /> HAT assay, respectively. Based Western signal, position 55 kDa Esa1p band shown closed circle arrow. The asterisk <br /> shows co-eluting 400 kDa band corresponding Tra1p Western signal. Positions molecular weight markers indicated. (C) Gel <br /> HAT assay Western blots sixth column (Superose 6) size NuA4 complex changed purification <br /> and Esa1p Tra1p closely co-eluting. <br />  <br />  <br /> also binding nickel monoQ columns,                            617, 798&#226;&#8364;&#8220;813, 3217&#226;&#8364;&#8220;3236 3529&#226;&#8364;&#8220;3545) three <br /> partially co-eluting NuA4 (data shown).                             peptides derived Esa1p (amino acids 98&#226;&#8364;&#8220;118, 334&#226;&#8364;&#8220; <br />    To characterize subunit composition                        350 351&#226;&#8364;&#8220;366). <br /> NuA4, immunoprecipitations scaled visual- <br /> ize true Esa1p-associated polypeptides. NuA4 fraction                        A fully functional Esa1p required for <br /> from Superose 6 Figure 6A incubated                           NuA4-mediated stimulation transcription <br /> affinity-purified Esa1p antibodies cross-linked protein                   Recently, presented biochemical evidence inter- <br /> A beads. After binding, beads washed extensively                    actions native HAT complexes, SAGA and <br /> in 350 mM NaCl. The bound complex eluted                            NuA4, VP16 activation domain target GAL4&#226;&#8364;&#8220; <br /> 100 mM glycine&#226;&#8364;&#8220;HCl pH 2.5 loaded SDS&#226;&#8364;&#8220;                               VP16-bound nucleosomes acetylation (Utley et al., <br /> polyacrylamide gel (Figure 6C). This allowed visual-                   1998). Moreover, GAL4&#226;&#8364;&#8220;VP16-driven transcription from <br /> ize complete set Esa1p-interacting polypeptides,                        chromatin templates stimulated HAT com- <br /> subunits NuA4 complex. Importantly, protein                       plexes acetyl-CoA-dependent manner. These data <br /> bands appear equivalent amounts, including Esa1p                      showed transcription activation domain targeting NuA4 <br /> Tra1p indicated. This experiment repeated                     participation nucleosomal acetyltransferase in <br /> times using NuA4 fractions different stages conven-                    transcription activation. Importantly, stimulation tran- <br /> tional purification protein bands equivalent                 scription NuA4 complex apparent on <br /> stoichiometry repeatedly obtained. Migration                     nucleosomal templates (Utley et al., 1998) be <br /> gradient gel showed p59 contain fact                    targeted VP16 activation domain (Ikeda et al., <br /> different proteins, explains apparently stronger                   1999). We analyzed role Esa1p the <br /> signal Figure 6C (data shown). A shorter migration                    transcriptional stimulatory activity NuA4 complex. <br /> showed subunits smaller molecular                       Figure 7A shows vitro transcription assay primer <br /> weights (p32 p36, data shown). These 11 bands                        extension using naked DNA HIV promoter internal <br /> add .1 MDa good agreement NuA4 size                            control chromatin reconstituted E4 promoter con- <br /> determined gel filtration. Finally, presence                  taining GAL4 binding sites. Since E4 promoter <br /> Tra1p Esa1p affinity-purified NuA4 complex (as                        containing GAL4 sites spaced array 5S <br /> in Figure 6C) confirmed attempt                    rDNA nucleosome positioning sequences, reconstituted <br /> mass spectrometry analysis. MS/MS peptide sequences                          template nucleosomes formed promoter <br /> were obtained trypsin digestion included                       region, phase repeated 5S nucleosomes (Utley <br /> peptides derived Tra1p (amino acids 150&#226;&#8364;&#8220;164, 603&#226;&#8364;&#8220;                       et al., 1998). GAL4&#226;&#8364;&#8220;VP16 present lanes and, <br />  <br /> 5114 <br />                                                                                       Nucleosomal H4 acetylation Esa1p complex <br />  <br />  <br />  <br />  <br />                                                                          Fig. 7. Chromatin-specific acetyl-CoA-dependent stimulation of <br />                                                                          transcription NuA4 requires fully functional Esa1p subunit. <br />                                                                          (A) In vitro transcription assay chromatin-reconstituted E4 <br />                                                                          promoter bearing GAL4 sites. Neighboring 5S rDNA nucleosome- <br />                                                                          positioning sequences force formation in-phase positioned <br />                                                                          nucleosomes GAL4 sites E4 promoter downstream <br />                                                                          sequence (Utley et al., 1998). Unreconstituted HIV LTR transcription <br />                                                                          shown control. Five nanomolar GAL4&#226;&#8364;&#8220;VP16 present all <br />                                                                          lanes. Acetyl-CoA present lanes 2 4 1 &#194;&#181;l purified <br /> Fig. 6. Tra1p component purified NuA4 complex. (A) Anti- <br />                                                                          NuA4 (fifth column) present lanes 1 2. Note acetyl-CoA <br /> Esa1p antibodies co-immunoprecipitate Tra1p. Affinity purified anti- <br />                                                                          dependence NuA4 stimulation E4 transcript. (B) Comparison <br /> Esa1p antibodies used prepare immunoaffinity resin. Pre- <br />                                                                          NuA4 monoQ fractions obtained ESA1 wt ts <br /> cleared partially purified NuA4 complex (Ni21&#226;&#8364;&#8220;agarose/monoQ/ <br />                                                                          strains. In vitro transcription performed (A). The NuA4 <br /> Superose 6 fraction 20) incubated resin beads were <br />                                                                          monoQ peak fraction ESA1 wild-type strain (1 &#194;&#181;l, lanes 1 <br /> then washed twice 500 mM NaCl twice 150 mM NaCl. <br />                                                                          2) able stimulate transcription chromatin template as <br /> Equivalent amounts initial, bound (beads) unbound (FT) <br />                                                                          efficiently purified NuA4 complex (lanes 5 6). In contrast, <br /> fractions analyzed Western blotting using Esa1p Tra1p <br />                                                                          times proteins NuA4 monoQ peak fraction <br /> antiserum. (B) Antibodies Tra1p specifically immunoprecipitate <br />                                                                          esa1 temperature-sensitive mutant strain able to <br /> NuA4 activity. Superose 6 NuA4 fractions (A) prepared <br />                                                                          stimulate transcription (4 &#194;&#181;l, lanes 3 4). <br /> from yeast strains FY23 KY320 expressing episomal myc- <br /> tagged version Tra1p. Pre-cleared Superose 6 NuA4 fractions were <br /> incubated anti-myc&#226;&#8364;&#8220;protein G&#226;&#8364;&#8220;Sepharose 2.5 h 4&#194;&#176;C,            supporting need subunits NuA4 efficient <br /> washed times equivalent amounts initial, unbound (FT) <br /> and bound (&#206;&#177;myc beads) fractions tested nucleosomal HAT <br />                                                                          targeting action chromatin (data shown; Ohba <br /> activity. NuA4 activity recovered beads myc-         et al., 1999). We used NuA4 monoQ fraction <br /> Tra1p-expressing strain (lane 6), NuA4 activity remained    prepared yeast strain episomal expression of <br /> the supernatant using wild-type Tra1p-expressing strain         wild-type ESA1 gene. The level acetyl-CoA-dependent <br /> (compare lanes 1 2). (C) Pre-cleared Superose 6 NuA4 fraction     stimulation transcription NuA4 fraction was <br /> in (A) incubated affinity-purified anti-Esa1p&#226;&#8364;&#8220;protein A&#226;&#8364;&#8220; <br /> Sepharose protein A. Beads washed twice 300 mM        similar obtained purified natural <br /> twice 150 mM NaCl. The bound NuA4 complex eluted           NuA4 (Figure 7B, compare lanes 1and 2 5 6). <br /> 100 mM glycine&#226;&#8364;&#8220;HCl pH 2.5, neutralized precipitated TCA.        In contrast, NuA4 fraction obtained the <br /> The sample loaded 10% SDS&#226;&#8364;&#8220;polyacrylamide gel               temperature-sensitive esa1 mutant strain used, no <br /> followed silver staining. Positions Esa1p, Tra1p seven other <br /> protein bands reproducibly obtained several <br />                                                                          obvious stimulation transcription detected <br /> independent experiments indicated arrowheads. Positions        (Figure 7B, lanes 3 4). Since amounts Esa1 <br /> molecular weight marker bands indicated (kDa).                  protein present different NuA4 fractions used in <br />                                                                          studies similar (see Figure 4C), can <br />                                                                          conclude fully functional Esa1 acetyltransferase <br /> as expected, able produce low level                  subunit required NuA4 stimulate transcription <br /> of transcription chromatin template (Figure 7A,                     assay. <br /> lanes 3 4). However, GAL4&#226;&#8364;&#8220;VP16-driven transcription <br /> was greatly stimulated NuA4 complex (Figure 7A, <br />                                                                          Discussion <br /> lane 2). Importantly, NuA4-dependent stimulation <br /> required presence acetyl-CoA, indicating impor-               NuA4 acetylates H4 N-termini nucleosome <br /> tance acetyltransferase activity (Figure 7A, compare              cores catalytic subunit Esa1p <br /> lanes 1 2 ). On hand, recombinant Esa1p                 There striking correlation presence of <br /> unable stimulate transcription assay,                   acetylated lysines core histone N-termini transcrip- <br />  <br />                                                                                                                                         5115 <br />  S.Allard et al. <br />  <br />  <br /> tional activity chromatin (Turner O&#226;&#8364;&#8482;Neill, 1995).          transcription regulators colocalization acetylated <br /> The discovery multiple yeast HATs having different             H4 transcriptionally competent chromatin, strongly <br /> core histone substrate specificities contribute        support primary role NuA4 transcription mechan- <br /> distinct functions different histone tails transcription    isms vivo. The importance NuA4 emphas- <br /> activation repression vivo (Fisher-Adams               ized fact NuA4 catalytic subunit, Esa1p, <br /> Grunstein, 1995). Acetylation histone tails thought         essential cell growth (Smith et al., 1998; Clarke <br /> to reduce interactions transcriptional repressors         et al., 1999). Such important role transcription <br /> nucleosomes (Hecht et al., 1995; Edmonson et al., 1996),          implies NuA4 generally involved the <br /> and enhance binding transcriptional activators           activation great number yeast genes, subset <br /> basal transcription factors (Imbalzano et al., 1994; Vettese-     genes expression essential cell cycle <br /> Dadey et al., 1996). The fact H4 N-terminal acetylation      progression. A recent study suggests acetyl- <br /> is histone modification frequently implicated         CoA binding domain Esa1p necessary Esa1p <br /> those proposed mechanisms suggested presence             function vivo (Clarke et al., 1999), the <br /> very important nucleosomal H4-specific HAT vivo. In            case Gcn5p (Kuo et al., 1998; Wang et al., 1998). <br /> vitro binding experiments demonstrated             Temperature-sensitive esa1 mutant strains replicate their <br /> the transcription activator USF binds preferentially           DNA fail proceed normally mitosis a <br /> acetylated H4-containing nucleosomes compared                manner dependent RAD9 checkpoint gene, pheno- <br /> those containing acetylated H3 (Vettese-Dadey et al.,             type similarly obtained mutations H4 N-terminal <br /> 1996). In initial fractionation native yeast nucleoso-     lysines (Megee et al., 1995; Clarke et al., 1999). <br /> mal HATs different activities targeting               The specificity NuA4 activity shared a <br /> mainly H3 tails preferentially acetylating H4        recently identified nucleosomal HAT Tetrahymena, <br /> in chromatin, NuA4 (Figure 1A) (Grant et al., 1997).              work single 80 kDa polypeptide <br />    The discovery Esa1 protein functions HAT         (Ohba et al., 1999). The ability NuA4 fully modify <br /> preferentially modifying H4 tails free histones (Smith         H4 H2A N-termini chromatin have <br /> et al., 1998; Clarke et al., 1999) suggested        structural effects chromatin. NuA4 increases <br /> be involved NuA4 activity. Esa1p essential        cutting efficiencies restriction endonucleases on <br /> HAT clearly functions cell cycle (Clarke et al.,       HIV-1 chromatin vitro, independently transcription <br /> 1999). It member MYST family proteins,             (Steger et al., 1998). The Tup1 repressor protein was <br /> which includes putative acetyltransferases Sas2p              shown bind preferentially underacetylated forms of <br /> Sas3p, affect gene silencing (Reifsnyder et al.,            H4 (Edmonson et al., 1996) acetylation H4 Lys16 <br /> 1996), MOZ, involved chromosomal                thought destabilize interaction Sir3, another <br /> translocation resulting form acute myeloid leukemia       repressor protein (Hecht et al., 1995). USF shown to <br /> (Borrow et al., 1996). Two members family           preferentially bind nucleosomes H4 tails bearing <br /> have stronger homology Esa1 sharing                acetylated Lys5 (Vettese-Dadey et al., 1996). Indeed, <br /> chromo-like domain, identified proposed chro-          specific acetylation lysine decreased <br /> matin-binding transcription regulators (Koonin et al.,            temperature-sensitive esa1 mutant strains non- <br /> 1995). Tip60 cloned two-hybrid screening              permissive temperature (Clarke et al., 1999). The X-ray <br /> HIV Tat transactivator recently shown acetylate        crystal structure nucleosome core particle illustrates <br /> free histones chromatin vitro, specificity      residues 16&#226;&#8364;&#8220;25 H4 tails interact negatively <br /> very similar Esa1p (Yamamoto Horikoshi, 1997).             charged pocket H2A-H2B dimer adjacent <br /> MOF discovered genetic screen genes                  nucleosome (Luger et al., 1997). This raises possibility <br /> necessary dosage compensation Drosophila (Hilfiker         H4 acetylation Lys16 (and probably Lys5, 8, <br /> et al., 1997). It thought dosage             12) negative effect internucleosomal <br /> compensation complex required male X chromo-           interactions. The acetylation histone H2A N-terminal <br /> some hypertranscription associated H4 acetylation         tail relatively poorly studied. Nevertheless, this <br /> at Lys16. Significantly, mof mutant obtained           domain H2A bind DNA defined <br /> screen contained single <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> putative          locations nucleosome cores, centered a <br /> HAT domain protein (Hilfiker et al., 1997).                position 40 bp nucleosomal dyad (Lee Hayes, <br />                                                                   1997). The defined structure adopted tail its <br /> NuA4 essential transcription coactivator                    interaction DNA certainly greatly affected <br /> complex                                                           acetylation lysine residues. <br /> Although recombinant Esa1p potent acetyltransferase <br /> of free histones, little ability access and/or modify   Potential recruitment NuA4 its <br /> core histones associated DNA oligonucleosomes,            essential Tra1p subunit <br /> the expected relevant substrate nuclear HATs                  We identified Tra1p stably associated subunit the <br /> (Figure 4D) (Smith et al., 1998). In report          NuA4 complex. Like Esa1p, Tra1p essential cell <br /> that Esa1p functions nucleosomal HAT context          growth (Saleh et al., 1998). It ATM/PI-3 kinase- <br /> of native multisubunit high molecular weight complex,           related protein homologous entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> the <br /> i.e. NuA4. Biochemical data illustrated targeting         human protein TRRAP, cofactor interacting c-Myc <br /> NuA4 activity transcription activators (Gcn4p, VP16)           E2F transactivation domains essential their <br /> and stimulated transcription chromatin templates       oncogenic activities (McMahon et al., 1998). Unlike the <br /> (Steger et al., 1998; Utley et al., 1998; Ikeda et al., 1999).    members ATM superfamily, Tra1p does not <br /> These data, homology Esa1p                 specific motifs conserved catalytic <br />  <br /> 5116 <br />                                                                                            Nucleosomal H4 acetylation Esa1p complex <br />  <br />  <br /> site PI-3 kinases, does rule activity.              protein described (Grant et al., 1998b). Anti-myc <br />                                                                              monoclonal antibodies obtained Babco. <br /> Tra1p component SAGA SAGA-like <br /> complexes (Grant et al., 1998b; Saleh et al., 1998)                      HAT assays <br /> TRRAP homologous human P/CAF complex                          Liquid HAT assays performed essentially described (Grant et al., <br /> (Vassilev et al., 1998). Tra1p structural role                  1997) using 0.5 &#194;&#181;g free core histones oligonucleosomes, protein <br /> in multisubunit complexes suggested fact                   fractions 0.125 &#194;&#181;Ci [3H]acetyl-CoA (4.7 Ci/mmol) HAT buffer <br />                                                                              [50 mM Tris&#226;&#8364;&#8220;HCl pH 8.0, 50 mM KCl, 0.1 mM EDTA, 5% glycerol, <br /> interacts separately independently Spt7p                        1 mM dithiothreitol (DTT), 1 mM phenylmethylsulfonyl fluoride (PMSF), <br /> Ada2p SAGA complex (Saleh et al., 1998). More                         10 mM sodium butyrate] 30 min 30&#194;&#176;C. Quantification was <br /> importantly, transcription activator-binding abilities                performed spotting reactions Whatman P81 filters and <br /> its mammalian homolog suggest play                           liquid scintillation counting described (Brownell Allis, 1995). <br />                                                                              Visualization histone specificity obtained loading reactions <br /> similar role yeast multisubunit complexes. This                     18% SDS&#226;&#8364;&#8220;polyacrylamide gel, followed Coomassie Blue staining <br /> explain fact SAGA NuA4 complexes                           ensure equivalent loading histones lane, destaining and <br /> interact VP16 GCN4 transcription activation                         fluorography Enhance (DuPont NEN). Assays H4 N-terminal <br /> domains vitro Ada NuA3 complexes                             peptides performed 70 ng quantified scintillation. The <br /> not (Utley et al., 1998). Interactions transcription                    wild-type peptide sequence SGRGKGGKGLGKGCAKRHR- <br />                                                                              RVLRDC. Lysine residues bold replaced acetylated lysines <br /> activators promoting cell cycle progression (as c-Myc                    mutant peptides allow remaining free lysine in <br /> E2F) definitely plausible light cell cycle                 peptide. <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> temperature-sensitive esa1 strains (Clarke                         For TAU gel Figure 2B, HAT assays performed using <br /> et al., 1999). The need Tra1p cell growth                      8 &#194;&#181;g oligonucleosomes 0.075 &#194;&#181;Ci [14C]acetyl-CoA (51 mCi/ <br />                                                                              mmol) 30&#194;&#176;C 1 h. Electrophoresis performed described <br /> reflect primarily presence NuA4 HAT                         (Braunstein et al., 1993), gel Coomassie Blue stained, dried <br /> subunit SAGA, Gcn5p, non-essential, like                       exposed. <br /> other subunits (Ada, Spt). This best understood by <br /> analyzing temperature-sensitive mutants TRA1 gene.                    Purification NuA4 HAT complex <br />                                                                              Preparation whole-cell extract 20 l yeast strain CY396 and <br />    In conclusion, purified characterized                       fractionation Ni21&#226;&#8364;&#8220;NTA agarose (Qiagen) monoQ HR 5/5 <br /> native transcription-related nucleosomal H4 HAT complex,                     column (Pharmacia) described (Co&#203;&#8224;te&#194;&#180; et al., 1994; Grant et al., <br /> NuA4. Identification primary catalytic subunit                     1997). Typically 1 &#194;&#181;l fraction tested HAT activity 0.5 &#194;&#181;g <br /> essential Esa1 protein supports necessary link                     oligonucleosomes 30 min 30&#194;&#176;C. Peak nucleosomal H4/H2A <br /> histone H4 acetylation chromatin, transcriptional activa-                 acetyltransferase activity salt gradient elution monoQ <br />                                                                              pooled (fractions 24&#226;&#8364;&#8220;28), diluted 100 mM NaCl buffer A <br /> tion cell cycle control. Furthermore, presence                    (20 mM HEPES 7.5, 0.5 mM DTT, 0.1% Tween-20, 10% glycerol and <br /> Tra1p NuA4 suggests role cofactor                           protease inhibitors) loaded directly monoS HR 5/5 column <br /> transcriptional regulation recruitment NuA4                   (Pharmacia). Bound proteins eluted 20 ml linear gradient <br /> HAT activity activator-bound promoter important                        100&#226;&#8364;&#8220;500 mM NaCl. Peak fractions NuA4 activity pooled, <br />                                                                              diluted 50 mM NaCl loaded 1 ml histone&#226;&#8364;&#8220;agarose column <br /> for cell cycle progression. Tra1p member                      (Sigma). Bound proteins eluted 10 ml linear gradient of <br /> ATM superfamily similarity PI-3 kinases suggests                     50&#226;&#8364;&#8220;500 mM NaCl. Peak fractions NuA4 activity pooled, diluted <br /> a possible function transduction stimulatory and/or                    50 mM loaded directly 1 ml DNA&#226;&#8364;&#8220;cellulose column (Sigma). <br /> inhibitory signals. Identification subunits                 NuA4 peak fractions 10 ml 50&#226;&#8364;&#8220;500 mM NaCl gradient were <br />                                                                              pooled concentrated 0.6 ml Centriprep-30 concentrator <br /> 1.3 MDa complex provide important insights                      (Amicon). The sample split run successively calibrated <br /> into regulation role gene expression linked                    Superose 6 HR 10/30 column (Pharmacia) buffer A plus 350 mM <br /> cell cycle control.                                                          NaCl 20% glycerol. Pooled NuA4 fractions (19&#226;&#8364;&#8220;23) then <br />                                                                              diluted 150 mM NaCl loaded directly 1 ml heparin&#226;&#8364;&#8220; <br />                                                                              Sepharose column (Pharmacia). Purified NuA4 eluted a <br />                                                                              7.5 ml linear gradient 0.15 1 M NaCl. Fractions (0.25 ml) were <br /> Materials methods                                                        supplemented 50 &#194;&#181;g/ml insulin, frozen liquid nitrogen and <br />                                                                              stored &#226;&#8364;&#8220;80&#194;&#176;C. <br /> Yeast strains reagents                                                      Whole-cell extracts prepared 2&#226;&#8364;&#8220;4 l yeast strains <br /> The yeast strain CY396 described (Peterson et al., 1994). The       grown described text, bound Ni21&#226;&#8364;&#8220;NTA agarose, eluted with <br /> yeast strains LPY3317 LPY3123 episomal wild-type ESA1            300 mM imidazole fractionated monoQ Superose 6 columns <br /> temperature-sensitive mutant esa1-414 esa1 null mutant genomic         described larger cultures (Grant et al., 1997). <br /> background described (Clarke et al., 1999). Recombinant <br /> wild-type Esa1p mutant Esa1-414p produced Escherichia            Western blotting immunoprecipitations <br /> coli described (Clarke et al., 1999). The yeast KY320 strain expressing   Aliquots 10&#226;&#8364;&#8220;15 &#194;&#181;l fractions typically used Western blotting <br /> an N-terminal myc-tagged version TRA1 gene centromeric           electrophoresis 10% SDS&#226;&#8364;&#8220;polyacrylamide gel. Immunoblotting <br /> plasmid described (Saleh et al., 1998).                             anti-Esa1p partially purified serum carried 1:3000 <br />    HeLa core histones H1-depleted oligonucleosomes isolated         dilution 1% non-fat dry milk PBS&#226;&#8364;&#8220;0.1% Tween-20 4 h room <br /> as described (Co&#203;&#8224;te&#194;&#180; et al., 1995). Yeast core histones generously      temperature. For immunoprecipitation studies, partially purified antisera <br /> provided Dr Sharon Roth, University Texas MD Anderson Cancer           cross-linked protein A&#226;&#8364;&#8220;Sepharose resin (Pharmacia) with <br /> Center prepared described (Edmonson et al., 1996). Sera       dimethylpimelimidate (Pierce) following standard protocols (Harlow and <br /> a peptide corresponding N-terminal 14 amino acids        Lane, 1988). Samples diluted buffer B (40 mM HEPES pH 7.5, <br /> yeast Esa1 protein produced rabbits. Partial purification     10% glycerol) plus 150 mM NaCl, 1% Triton buffer precleared <br /> IgG fraction sera performed ammonium sulfate                 protein A&#226;&#8364;&#8220;Sepharose. Precleared supernatant incubated with <br /> precipitation chromatography CM-Affigel-blue resin (Bio-Rad).       preimmune immune cross-linked beads 2 h 4&#194;&#176;C a <br /> For immunopurification NuA4 complex, anti-Esa1p antibodies            30% slurry. After incubation, beads washed twice the <br /> from partially purified serum obtained affinity-purification    following buffers: buffer B containing 150 mM NaCl&#226;&#8364;&#8220;1% Triton then <br /> a resin (Sulfolink, Pierce) harboring immunizing peptide following       buffer B containing 350 mM NaCl finally buffer B 150 mM <br /> standard protocols (Harlow Lane, 1988). Anti-HA antibody             NaCl. HAT assays performed directly equivalent amounts of <br /> Swi2p detection purchased Boehringer Mannheim. Anti-Ada2p           supernatant resuspended beads. For immunoprecipitations using <br /> serum kindly provided Dr Shelley Berger, The Wistar Institute.        anti-myc monoclonal antibodies (Figure 6B), protein G&#226;&#8364;&#8220;Sepharose was <br /> Anti-Tra1p serum produced N-terminal 20 amino acids        used instead protein A&#226;&#8364;&#8220;Sepharose. For immunopurification the <br />  <br />  <br />                                                                                                                                                 5117 <br />  S.Allard et al. <br />  <br />  <br /> NuA4 complex shown Figure 6A C, affinity-purified anti-Esa1p           Co&#203;&#8224;te&#194;&#180;,J., Utley,R.T. Workman,J.L. (1995) Basic analysis of <br /> antibodies used (0.5&#226;&#8364;&#8220;1 mg Ab/ml resin) complex               transcription factor binding nucleosomes. Methods Mol. Genet., 6, <br /> eluted resuspending beads 100 mM glycine&#226;&#8364;&#8220;HCl pH 2.5,                   108&#226;&#8364;&#8220;128. <br /> neutralized, followed TCA precipitation SDS&#226;&#8364;&#8220;PAGE. Sequence             Co&#203;&#8224;te&#194;&#180;,J., Peterson,C.L. Workman,J.L. (1998) Perturbation of <br /> analysis obtained peptides performed Harvard Micro-            nucleosome core structure SWI/SNF complex persists following <br /> chemistry Facility microcapillary reverse-phase HPLC nano-electro-            detachment, enhancing subsequent transcription factor binding. <br /> spray tandem mass spectrometry (uLC/MS/MS) Finnigan LCQ                     Proc. Natl Acad. Sci. USA, 95, 4947&#226;&#8364;&#8220;4952. <br /> quadrupole ion trap mass spectrometer.                                        Edmonson,D.G., Smith,M.M. Roth,S.Y. (1996) Repression domain <br />                                                                                  yeast global repressor tup1 directly interacts histones H3 <br /> In vitro transcription assay                                                     H4. Genes Dev., 10, 1247&#226;&#8364;&#8220;1259. <br /> Protocol reagents vitro transcription assays chromatin          Fisher-Adams,G. Grunstein,M. (1995) Yeast histone H4 H3 <br /> templates described thoroughly (Steger et al., 1998; Utley         N-termini different effects chromatin structure the <br /> et al., 1998). Reactions containing 15&#226;&#8364;&#8220;21 ng nucleosome reconstituted         GAL1 promoter. EMBO J., 14, 1468&#226;&#8364;&#8220;1477. <br /> template DNA incubated 5 nM GAL4&#226;&#8364;&#8220;VP16 absence             Grant,P.A. et al. (1997) Yeast Gcn5 functions multisubunit <br /> or presence NuA4 complex final conditions (60 mM NaCl, 10 mM               complexes acetylate nucleosomal histones: characterization an <br /> HEPES pH 7.8, 5 mM DTT, 0.5 mM PMSF, 5% glycerol, 0.25 mg/ml                     Ada complex SAGA (Spt/Ada) complex. Genes Dev., 11, <br /> bovine serum albumin, 1.25 &#194;&#181;M acetyl CoA, 10 mM sodium butyrate)                 1640&#226;&#8364;&#8220;1650. <br /> at 30&#194;&#176;C 20 min. HeLa nuclear extracts nucleoside triphosphates        Grant,P.A., Schieltz,D., Pray-Grant,M.G., Steger,D.J., Reese,J.C., <br /> were added transcription/primer extension assays carried            Yates,J.R.,III Workman,J.L. (1998a) A subset TAFIIs are <br /> according Steger et al. (1998).                                               integral components SAGA complex required nucleosome <br />                                                                                  acetylation transcriptional stimulation. Cell, 94, 45&#226;&#8364;&#8220;53. <br />                                                                               Grant,P.A., Schieltz,D., Pray-Grant,M.G., Yates,J.R.,III Workman, <br /> Acknowledgements                                                                 J.L. (1998b) The ATM-related cofactor Tra1 component the <br /> We thank Isabelle Ruel technical assistance rEsa1p, Benoit              purified SAGA complex. Mol. Cell, 2, 863&#226;&#8364;&#8220;867. <br /> Leblanc David Clark His-tagged ESA1 strain, Shelley Berger        Grunstein,M. (1997) Histone acetylation chromatin structure and <br /> for Ada2p anti-serum, Sharon Roth yeast core histones, Mark Parthun          transcription. Nature, 389, 349&#226;&#8364;&#8220;352. <br /> and Daniel Gottschling hat2&#226;&#8364;&#8220; isogenic wild-type strains, Paul         Harlow,E. Lane,D. (1988) Antibodies: A Laboratory Manual. Cold <br /> Kaufman msi1&#226;&#8364;&#8220; isogenic wild-type strains, Michael Grunstein              Spring Harbor Laboratory Press, Cold Spring Harbor, NY, pp. 522&#226;&#8364;&#8220;523. <br /> for GST&#226;&#8364;&#8220;H4 expression plasmid, Michael Green Joe Reese            Hebbes,T.R., Clayton,A.L., Thorne,A.W. Crane-Robinson,C. (1994) <br /> GST&#226;&#8364;&#8220;VP16 expression plasmid. We grateful William S.Lane               Core histone hyperacetylation co-maps generalized DNase I <br /> the Harvard Microchemistry Facility preliminary mass spectrometry            sensitivity chicken &#206;&#178;-globin chromosomal domain. EMBO J., <br /> analysis. We thank Edwin Smith, David Allis, Arri Eisen John            13, 1823&#226;&#8364;&#8220;1830. <br /> Lucchesi sharing results publication. This work supported      Hecht,A., Laroche,T., Strahl-Bolsinger,S., Gasser,S.M. Grunstein,M. <br /> by grants Medical Research Council Canada (MRC) J.C.              (1995) Histone H3 H4 N-termini interact SIR3 SIR4 <br /> and C.J.B., NIGMS J.L.W. NIH L.P. P.G. supported               proteins: A molecular model formation heterochromatin in <br /> Postdoctoral Fellowship American Cancer Society. J.L.W.            yeast. Cell, 80, 583&#226;&#8364;&#8220;592. <br /> HHMI investigator. J.C. Canadian MRC Scholar.                            Hilfiker,A., Hilfiker-Kleiner,D., Pannuti,A. Lucchesi,J.C. (1997) mof, <br />                                                                                  putative acetyl transferase gene related Tip60 MOZ <br />                                                                                  human genes SAS genes yeast, required dosage <br /> References                                                                       compensation Drosophila. EMBO J., 16, 2054&#226;&#8364;&#8220;2060. <br />                                                                               Hong,L., Schroth,G.P., Matthews,H.R., Yau,P. Bradbury,E.M. (1993) <br /> Bannister,A.J. Kouzarides,T. (1996) The CBP co-activator a <br />                                                                                  Studies DNA binding properties histone H4 amino <br />    histone acetyltransferase. Nature, 384, 641&#226;&#8364;&#8220;643. <br />                                                                                  terminus. J. Biol. Chem., 268, 305&#226;&#8364;&#8220;314. <br /> Bone,J.R., Lavender,J., Richman,R., Palmer,M.J., Turner,B.M. and <br />                                                                               Ikeda,K., Steger,D.J., Eberharter,A. Workman,J.L. (1999) Activation <br />    Kuroda,M.I. (1994) Acetylated histone H4 male X chromosome <br />                                                                                  domain-specific general transcription stimulation native histone <br />    associated dosage compensation Drosophila. Genes Dev., <br />                                                                                  acetyltransferase complexes. Mol. Cell. Biol., 19, 855&#226;&#8364;&#8220;863. <br />    8, 96&#226;&#8364;&#8220;104. <br />                                                                               Imbalzano,A.N., Kwon,H., Green,M.R. Kingston,R.E. (1994) <br /> Borrow,J. et al. (1996) The translocation t (8;16) (p11;p13) acute <br />                                                                                  Facilitated binding TATA-binding protein nucleosomal DNA. <br />    myeloid leukaemia fuses putative acetyltransferase CREB- <br />                                                                                  Nature, 370, 481&#226;&#8364;&#8220;485. <br />    binding protein. Nature Genet., 14, 33&#226;&#8364;&#8220;41. <br /> Braunstein,M., Rose,A.B., Holmes,S.G., Allis,C.D. Broach,J.R.             Ito,T., Bulger,M., Pazin,M.J., Kobayashi,R. Kadonaga,J.T. (1997) <br />    (1993) Transcriptional silencing yeast associated reduced          ACF, ISWI-containing ATP-utilizing chromatin assembly and <br />    nucleosome acetylation. Genes Dev., 7, 592&#226;&#8364;&#8220;604.                               remodeling factor. Cell, 90, 145&#226;&#8364;&#8220;155. <br /> Braunstein,M., Sobel,R.E., Allis,C.D., Turner,B.M. Broach,J.R.            Kaufman,P.D., Kobayashi,R. Stillman,B. (1997) Ultraviolet radiation <br />    (1996) Efficient transcriptional silencing Saccharomyces cerevisiae        sensitivity reduction telomeric silencing Saccharomyces <br />    requires heterochromatin histone acetylation pattern. Mol. Cell.            cerevisiae cells lacking chromatin assembly factor-1. Genes Dev., 11, <br />    Biol., 16, 4349&#226;&#8364;&#8220;4356.                                                         245&#226;&#8364;&#8220;357. <br /> Brownell,J.E. Allis,C.D. (1995) An activity gel assay detects             Kleff,S., Andrulis,E.D., Anderson,C.W. Sternglanz,R. (1995) <br />    single, catalytically active histone acetyltransferase subunit           Identification gene encoding yeast histone H4 acetyltransferase. <br />    Tetrahymena macronuclei. Proc. Natl Acad. Sci. USA, 92, 6364&#226;&#8364;&#8220;6368.            J. Biol. Chem., 270, 24674&#226;&#8364;&#8220;24677. <br /> Brownell,J.E., Zhou,J., Ranalli,T., Kobayashi,R., Edmondson,D.G.,             Koonin,E.V., Zhou,S. Lucchesi,J.C. (1995) The chromo superfamily: <br />    Roth,S.Y. Allis,C.D. (1996) Tetrahymena histone acetyltransferase         new members, duplication chromo domain possible role in <br />    A: homolog yeast Gcn5p linking histone acetylation gene               delivering transcription regulators chromatin. Nucleic Acids Res., <br />    activation. Cell, 84, 843&#226;&#8364;&#8220;851.                                                23, 4229&#226;&#8364;&#8220;4233. <br /> Cairns,B.R. et al. (1996) RSC, essential, abundant chromatin-              Krajewski,W.A. Becker,P.B. (1998) Reconstitution hyper- <br />    remodeling complex. Cell, 87, 1249&#226;&#8364;&#8220;1260.                                      acetylated, DNase I-sensitive chromatin, characterized high <br /> Chen,H., Lin,R.J., Schiltz,R.L., Chakravarti,D., Nash,A., Nagy,L.,               conformational flexibility nucleosomal DNA. Proc. Natl Acad. Sci. <br />    Privalsky,M.L., Nakatani,Y. Evans,R.M. (1997) Nuclear receptor            USA, 95, 1540&#226;&#8364;&#8220;1545. <br />    coactivator ACTR novel histone acetyltransferase forms          Kuo,M.H., Zhou,J., Jambeck,P., Churchill,M.E. Allis,C.D. (1998) <br />    multimeric activation complex P/CAF CBP/p300. Cell, 90,              Histone acetyltransferase activity yeast Gcn5p required the <br />    569&#226;&#8364;&#8220;580.                                                                      activation target genes vivo. Genes Dev., 12, 627&#226;&#8364;&#8220;639. <br /> Clarke,A.S., Lowell,J.E., Jacobson,S.J. Pillus,L. (1999) Esa1p      Kwon,H., Imbalzano,A.N., Khavarl,P.A., Kingston,R.E. Green,M.R. <br />    essential histone acetyltransferase required cell cycle progression.      (1994) Nucleosome disruption enhancement activator binding <br />    Mol. Cell. Biol., 19, 2515&#226;&#8364;&#8220;2526.                                              human SWI/SNF complex. Nature, 370, 477&#226;&#8364;&#8220;481. <br /> Co&#203;&#8224;te&#194;&#180;,J., Quinn,J., Workman,J.L. Peterson,C.L. (1994) Stimulation        Lee,D.Y., Hayes,J.J., Pruss,D. Wolffe,A.P. (1993) A positive rol <br /> </body></html>