<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Conservation links gene transcription and <br /> chromosomal organization highly reduced genome of <br /> Buchnera aphidicola <br /> Jos&#195;&#169; Vi&#195;&#177;uelas*1, Federica Calevro1, Didier Remond2, Jacques Bernillon3, <br /> Yvan Rahb&#195;&#169;1, G&#195;&#169;rard Febvay1, Jean-Michel Fayard1 Hubert Charles1 <br />  <br /> Address: 1UMR 203 Biologie Fonctionnelle Insectes et Interactions, IFR41, INRA, INSA-Lyon, F-69621 Villeurbanne, France, 2Laboratoire de <br /> Dynamique des Machines et des Structures, INSA-Lyon, F-69621 Villeurbanne, France 3DTAMB, Universit&#195;&#169; Claude Bernard Lyon-1, F-69622 <br /> Villeurbanne, France <br /> Email: Jos&#195;&#169; Vi&#195;&#177;uelas* - jose.vinuelas@insa-lyon.fr; Federica Calevro - federica.calevro@insa-lyon.fr; Didier Remond - didier.remond@insa- <br /> lyon.fr; Jacques Bernillon - jbernill@biomserv.univ-lyon1.fr; Yvan Rahb&#195;&#169; - Yvan.Rahbe@lyon.inra.fr; <br /> G&#195;&#169;rard Febvay - Gerard.Febvay@lyon.inra.fr; Jean-Michel Fayard - jean-michel.fayard@insa-lyon.fr; Hubert Charles - hubert.charles@insa- <br /> lyon.fr <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 4 June 2007                                                         Received: 7 February 2007 <br />                                                                                Accepted: 4 June 2007 <br /> BMC Genomics 2007, 8:143    doi:10.1186/1471-2164-8-143 <br /> This article available <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/8/143 <br /> &#194;&#169; 2007 Vi&#195;&#177;uelas et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Genomic studies bacteria clearly shown existence chromosomal <br />                  organization regards, example, gene localization, order orientation. Moreover, <br />                  transcriptomic analyses demonstrated that, free-living bacteria, gene transcription levels <br />                  chromosomal organization mutually influenced. We explored possible <br />                  conservation relationships mRNA abundances chromosomal organization the <br />                  highly reduced genome Buchnera aphidicola, primary endosymbiont aphids, close <br />                  relative Escherichia coli. <br />                  Results: Using oligonucleotide-based microarray, normalized transcriptomic data by <br />                  genomic DNA signals order access inter-gene comparison data. Our analysis showed <br />                  mRNA abundances, gene organization (operon) gene essentiality correlated in <br />                  Buchnera (i.e., expressed genes essential genes organized operons) link <br />                  mRNA abundances gene strand bias found. The effect Buchnera genome <br />                  evolution gene expression levels analysed order assess constraints <br />                  imposed obligate symbiosis aphids, underlining importance gene sets for <br />                  <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>survival</span> partners. Finally, results existence spatial periodic <br />                  transcriptional patterns genome Buchnera. <br />                  Conclusion: Despite important reduction genome size apparent decay its <br />                  capacity regulating transcription, work reveals significant correlation mRNA <br />                  abundances chromosomal organization aphid-symbiont Buchnera. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 12 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                   http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br />  <br /> Background                                                     Buchnera different aphid species been <br /> Past genomic studies comprehensively described        sequenced far, sizes ranging 0.42 0.65 Mb <br /> organization bacterial chromosome, example       [18-21]. In Buchnera genomes, genes <br /> terms gene localization, order orientation. The         involved biosynthesis essential amino acids <br /> degree organization shown increase         (EAAs) insect synthesize, suffi- <br /> genome size, overall GC composition presence        cient amounts phloem sap, retained, whereas <br /> nucleoid-binding proteins [1]. This organization        genes regulating expression lost [22]. <br /> chromosome described adaptive           In Buchnera pea aphid Acyrthosiphon pisum (BAp), <br /> functional tool, essential survival bacterial   608 chromosomal genes, 56% located on <br /> cell. More precisely, studies identified strand   leading strand DNA. This observation reveals small <br /> asymmetries distribution genes           gene-strand bias, equivalent closely <br /> leading lagging strand DNA (for review        related free-living bacteria:E. coli (55%) [23]. <br /> Rocha [2]). Indeed, bacterial genomes carry, average, <br /> from 78% (for genomes containing polymerase PolC)          Gene essentiality defined experimental tech- <br /> <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> 58% (for genomes) genes lead-     niques cultivable bacteria, E. coli a <br /> ing strand [3]. This bias important      large repertoire <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> available. For small <br /> essentiality genes taken account, essential     number genes, absence mutants, essenti- <br /> genes distribution bias reaches 76% 94%                 ality defined basis function. For example, <br /> Escherichia coli Bacillus subtilis respectively [4]. The   genes encoding ribosomal proteins usually described <br /> asymmetry distribution genes         essential, genes involved flagellar appa- <br /> two strands DNA explained means min-       ratus classified non-essential bacteria [24]. Due <br /> imizing interruptions gene transcription colli-      specific lifestyle symbiotic functions Buchn- <br /> sions DNA RNA polymerases. The            era, <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> essentials genes bacterium is <br /> generally accepted hypothesis co-directional colli-    particularly difficult. For reason, considered the <br /> sions leading strand weaker effect RNA        minimal gene set supporting bacterial life proposed by <br /> polymerase processivity head-on collisions            Gil et al. [25] appropriate study. <br /> occurring lagging strand. This &quot;replicational selec-    Indeed, determination set based compar- <br /> tion&quot; guarantee genes leading strand,       ative analyses bacterial insect endosymbionts with <br /> particularly essential ones, efficiently transcribed   small genomes (among Buchnera genomes), <br /> [5-8].                                                         smallest bacterial genome sequenced (Myco- <br />                                                                plasma genitalium), essential gene lists available <br /> Gene organization bacterial chromosome         B. subtilis E. coli literature. Based this <br /> well known effects transcription standard exam-       minimal set essential genes, BAp harbour 190 essential <br /> ple organization genes operons,           encoding-protein genes 60% located on <br /> allows sophisticated regulation gene expression       leading strand. This indicates strand bias essenti- <br /> [9]. It shown neighbouring genes       ality, albeit lower observed free-living bacte- <br /> bacterial chromosome tend co-expressed,          ria. Recent transcriptomic analyses shown that, <br /> they operon [10], suggesting exist-    despite low response different physiological condi- <br /> ence &quot;supra-operonic&quot; organization [11]. This per-   tions imposed aphid host, Buchnera retained the <br /> fect illustration mutual influence gene         capability express genes different levels basal con- <br /> expression levels chromosomal organization bac-         ditions. This expression correlated genome <br /> teria. Another example significant effect DNA        organization putative transcription units (pTU), <br /> supercoiling transcription. Recent studies sug-        absence of known transcriptional <br /> gested dynamical structure nucleoid acts    regulatory proteins [26]. <br /> a &quot;transcription factor&quot; E. coli [12], probably <br /> responsible presence spatial transcriptional pat-   This work focuses studying relationships between <br /> terns free-living bacteria [13-15].                         gene transcription chromosome organization the <br />                                                                highly reduced genome Buchnera, coupling genomic <br /> While observations free-living bacte-       information transcriptomic data. For that, mRNA <br /> ria, data exist symbiotic bacteria characterized   abundances measured full-genome oligo- <br /> by reduced genomes, shaped adaptation          array dedicated BAp. The basal transcriptome Buchn- <br /> host metabolic requirements high evolution        era analysed regarding (i) location the <br /> rate. Buchnera aphidicola, endosymbiont aphids,     genes leading/lagging strand, gene essentiality <br /> is best studied examples intracellular bac-   operon organization, (ii) gene evolution rate (as <br /> terium reduced genome [16,17]. The genomes           measured GC rate), (iii) spatial location of <br />  <br />  <br />                                                                                                                    Page 2 12 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                  http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> genes chromosome. One crucial point this <br /> study choice data normalization procedure. <br /> Since 2002, authors described possible advan- <br /> tages genomic DNA (gDNA) standard microarray <br /> data normalization investigation microbial <br /> genomes [27-29]. This useful attractive standard readily <br /> available, inexpensive, invariant time from <br /> laboratory laboratory. Moreover, main advantage of <br /> the normalization gDNA takes account <br /> the probe/target affinity order allow inter-gene <br /> calibration expression level comparisons. For these <br /> reasons, developed validated study a <br /> &quot;spot spot&quot; microarray data normalization method <br /> based gDNA.                                                Validation <br />                                                               Figure 1 gDNA normalization procedure <br />                                                               Validation gDNA normalization procedure. <br /> Results                                                       Differences (before gDNA normalization) the <br /> Effectiveness using genomic DNA microarray data        standard deviations fluorescent signals 43 3- <br /> normalization                                                 probe genes Buchnera oligo-array reported here. <br /> As gene repeat occurs Buchnera genome [18],         Genes ranged increasing differences standard devia- <br /> microarray hybridization gDNA produce             tion. The 34 genes yellow area (positive differences) <br /> equivalent fluorescent signals Buchnera gene      reduction probe standard deviation after <br />                                                               gDNA normalization. <br /> data set. Moreover, fluorescence variability gDNA <br /> hybridization low mainly explained the <br /> thermodynamic properties probes driving tar- <br /> get affinities. Indeed, supplemental Figure 1A Addi- <br /> tional file 1 shows shrunken fluorescent distribution for <br /> the slides hybridized Buchnera gDNA, compared <br /> with cDNA slides. gDNA fluorescence variability was <br /> analysed (ANOVA F-test regression models) regard- <br /> ing following parameters: probe GC rate, probe size, <br /> probe specificity, putative hairpin homoduplex <br /> entropies (&#226;&#710;&#8224;G). We 12% gDNA signal <br /> variability significantly explained Buchn- <br /> era probe GC rate: GC-rich probes fluores- <br /> cent AT-rich ones (Additional file 1, supplemental <br /> Figure 1B). On contrary, significant influence on <br /> gDNA signals factors ana- <br /> lysed (data shown), indicating optimal probe <br /> design software ROSO [30]. <br />  <br /> The normalization procedure consists weighting each <br /> spot inverse gDNA fluorescence signal. <br /> This described method section. The normaliza-       Figureof2 log2 nBGT levels Buchnera <br />                                                               Means <br /> tion procedure does affect shape, mean     Means log2 nBGT levels Buchnera. Genes split <br /> median mRNA fluorescent signal distribution            essential (&quot;E&quot;) non-essential (&quot;nE&quot;) genes pre- <br /> (Additional file 2, supplemental Figure 2). We         dicted putative operons (&quot;Op&quot;) (&quot;nOp&quot;). The <br /> mention difference expression levels          error bars indicate 95% confidence interval mean. <br />                                                               After applying Bonferroni correction multiple tests, <br /> between the highly expressed genes in <br />                                                               significant differences obtained ANOVA Likeli- <br /> Buchnera (more 10 fold) confirmed existence       hood ratio tests classes genes marked <br /> a basal transcriptional regulation aphid endosym-      symbol: * significant difference &quot;Op&quot; <br /> biont.                                                        &quot;nOp&quot; &quot;E&quot;; &#226;&#8364;&#160; significant difference &quot;Op&quot; and <br />                                                               &quot;nOp&quot; &quot;nE&quot;; &#226;&#8364;&#161; significant difference &quot;E&quot; &quot;nE&quot; <br /> Finally, took advantage design probes    &quot;Op&quot;; significant difference observed &quot;E&quot; <br /> (2 3) gene microarray validate gDNA      &quot;nE&quot; &quot;nOp&quot;. <br /> normalization procedure. Hence, compared the <br />  <br />  <br />                                                                                                                   Page 3 12 <br />                                                                                             (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                       http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> within-gene fluorescence signal standard deviations      value &lt; 10-4): highly expressed genes GC rich <br /> the 3-probe genes, normalization. The          (the highly conserved Buchnera). More precisely, <br /> results presented Figure 1 show, normalization,     areas 1 2 Figure 3 correspond genes are <br /> overall reduction fluorescent signals variability    slowly evolving weakly highly expressed <br /> about 79% genes. Close results obtained         respectively, areas 3 4 correspond genes <br /> the 2-probe genes (65%, data shown). Although low           rapidly evolving weakly highly <br /> mRNA abundances major source fail-          expressed respectively. In second area found <br /> ure gDNA normalization (data shown) inten-           genes encoding ribosomal proteins (among <br /> sity correction included estimation          them: rplL, rplP, rpmB, rpsJ, rpsK) genes encoding <br /> variability powerful 2 3 dif-       chaperone proteins (dnaK, dnaJ mopA,mopB). An <br /> ferent probes gene.                                         important proportion flagellar genes evolve <br />                                                                 rapidly: 16 26 Buchnera (among 9/12 fli <br /> Analysis links Buchnera normalized mRNA          genes) highly expressed (11 out <br /> abundances chromosome organization                          16). Moreover, 11 16 highly evolving flag- <br /> DNA strand position, putative operons gene essentiality     ellar genes located leading strand (among them <br /> In work, analysed effect gene        8 fli genes). Interestingly, Buchnera orphan <br /> DNA strand position, operon organization gene               genes (yba3 yba4), highly expressed a <br /> essentiality variability basal mRNA abun-         low GC content (area 4). The genes encoding enzymes <br /> dances Buchnera.                                             involved biosynthesis EAAs generally evolve <br />                                                                 slowly high mRNA abundances. Among them, <br /> Analysis pTU factor revealed normalized       observed genes belonging two <br /> basal gene transcription (nBGT) variation            operons involved biosynthesis isoleucine and <br /> smaller pTU them, indicating           valine particularly highly expressed con- <br /> genes belonging pTU tend transcribed       served Buchnera (area 2). Moreover, ileS, valS, six <br /> similar rates (ANOVA F-test, R2 = 0.640, P-value &lt; 10-4).       genes encoding aminoacyl-tRNA synthetase for <br /> Analysis essentiality revealed Buchnera       EAAs weakly expressed, genes encoding ami- <br /> genome contains 32% essential genes versus 6%           noacyl-tRNA synthetase non-EAAs moderately or <br /> for E. coli genome [31].                                    highly expressed (Median test, P-value = 1.4 &#195;&#8212; 10-2). <br />                                                                 Finally, area 3 includes pseudogenes, apbE, cvpA, <br /> Using complete ANOVA model (including essentiality,           yigL. <br /> strand position pTU factors), shown that <br /> pTU essentiality significant factors,        Spatial location genes chromosome <br /> significant effect observed strand position (Table      The question addressed final work <br /> 1). Gene distributions different categories pre-     existence spatial periodic patterns transcrip- <br /> sented supplemental Figure 3 Additional file      tional activity chromosome Buchnera. As has <br /> 3. The analysis split level signifi-   reported genomes free-living bacteria [15], <br /> cant factor (operon/singleton, essential/non-essen- <br /> tial). The data analysed fluorescence comparison        Table 1: Effects interactions global ANOVA model <br />                                                                 fitted Buchnera log2 nBGT levels <br /> (Figure 2) distribution comparison (data not <br /> shown) similar conclusions. When tested fac-         Source                    DF &#226;&#8364;&#160;     Sum       F Ratio ||    P-value <br /> tor &quot;pTU&quot; Buchnera mRNA abundances                                                Squares &#226;&#8364;&#161; <br /> the genes pTU highly expressed genes <br /> in singletons essential non essential genes.        Operon                      1       22.216       43.204      &lt; 0.0001 * <br /> Analysing &quot;essentiality&quot; factor, showed that,         Essentiality                1        9.627       18.722      &lt; 0.0001 * <br />                                                                  Strand                      1        0.009        0.017        0.897 <br /> genes pTU, highest expressed ones essential, <br />                                                                  Essentiality &#195;&#8212; Operon       1        7.246       14.092       0.0002 * <br /> whereas effect true singletons (Figure 2).      Essentiality &#195;&#8212; Strand       1        0.055        0.107        0.744 <br />                                                                  Strand &#195;&#8212; Operon             1        0.280        0.545        0.461 <br /> Gene evolution rate                                              Essentiality &#195;&#8212; Strand &#195;&#8212;     1        0.653        1.271        0.260 <br /> In second work analysed rela-       Operon <br /> tionship nBGT levels gene evolution rates         Residual error             533     274.067 <br /> Buchnera. Gene evolution rates estimated the <br /> GC content genes instead non-synonymous sub-            *: significant P-value 95% confidence limits, &#226;&#8364;&#160;: degrees freedom <br />                                                                  source variation, &#226;&#8364;&#161;: sum squared distances each <br /> stitution rates (Ka), parameters highly cor-      source variation, ||: model mean square divided error mean <br /> related Buchnera. Figure 3 shows mRNA                    square <br /> abundances GC content correlated (R2 = 0.151, P- <br />  <br />  <br />                                                                                                                            Page 4 12 <br />                                                                                                     (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                      http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br />  <br />                                                                  Figure  4 <br />                                                                  Autocorrelation <br />                                                                  Buchnera  log 2 nBGTofalong <br />                                                                                    function         chromosome <br />                                                                                                spatial series corresponding <br />                                                                  Autocorrelation function spatial series corre- <br />                                                                  sponding Buchnera log2 nBGT chro- <br />                                                                  mosome. The blue lines represent &#194;&#177; 2 standard errors for <br />                                                                  approximate 95% confidence limits (i.e., values the <br /> Plot <br /> Figureof log <br />           3 2 nBGT levels versus GC rate Buchnera genes <br />                                                                  threshold significant). <br /> Plot log2 nBGT levels versus GC rate Buchnera <br /> genes. The bivariate normal ellipse (% = 0.95) drawn in <br /> black main areas delimited median of <br /> the gene GC rate parameter fourth quar-     464 551 genes. These minima, specific loca- <br /> tile mRNA abundance level variable. Pearson correla-      tion genes chromosome, superim- <br /> tion coefficient (R2) shown. The names genes           posed red curve corresponds to <br /> outside ellipse zones specified.             experimental Buchnera transcript abundances. The red <br />                                                                  curve shows maxima, absent black one, <br />                                                                  belonging particular functional categories genes <br /> spectral analysis revealed significant periodic components       encoding ribosomal proteins flagella. <br /> in gene transcription levels chromosome of <br /> the aphid endosymbiont (Fisher&apos;s Kappa test, P-value =           Some recent papers [32-34] reported spurious perio- <br /> 4.6 &#195;&#8212; 10-4; Bartlett&apos;s Kolmogorov-Smirnov test, P-value &lt;        dicities transcriptomic data attributed spa- <br /> 10-2). To substantiate results, analysed auto-      tial arrangements probes oligo-array. Even if <br /> correlation function nBGT levels regard gene          construction gDNA normalization procedure is <br /> order chromosome smaller scale (with              appropriate account bias, checked the <br /> inter-gene distances ranging 1 50 genes). Figure 4       possible presence periodic components gDNA sig- <br /> shows closely spaced genes chromosome,           nals. The absence significant periodicities gDNA <br /> more precisely groups 2 8 genes, highly corre-        (black curve Figure 5A) supports idea the <br /> lated transcriptional patterns chromosome           observed spatial patterns mRNA signals proper to <br /> Buchnera. We observed correlation decreases       Buchnera transcriptome. <br /> with size groups. Such small autocorrelated <br /> structures (of 2&#226;&#8364;&#8220;8 genes) interpreted          To ascertain observed periodic patterns were <br /> being operon effect Buchnera, reported      caused pTU Buchnera, performed differ- <br /> for E. coli [15].                                                ent simulated permutations Buchnera gene posi- <br />                                                                  tions (Figure 6) ran autocorrelation spectral <br /> More generally, plot spectral density auto-    analyses resulting data. The permutation pre- <br /> correlation function displays periods contained          served rank spacing pTU <br /> 2 152 gene lengths. These periods grouped            modified location singletons; the <br /> long-range, medium-range short-range periods              second conserved rank pTU we <br /> Figure 5A. To illustrate results, Figure 5B presents   modified number singletons them; the <br /> averaged smoothed signal seven 87 gene seg-           permutation changed rank location <br /> ments constituting period &quot;86.86&quot; complete            pTU; and, finally, fourth genes <br /> Buchnera chromosome. On figure, black curve cor-        randomly distributed. From real series the <br /> responding period &quot;86.86&quot; shows periodic oscilla-         permutation (&quot;Perm. 0&quot; &quot;Perm. 3&quot;), observed <br /> tions minima distance 29, 116, 203, 290, 377,       decrease, 8 2, maximum size auto- <br />  <br />  <br />                                                                                                                       Page 5 12 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                       http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br />  <br />                                                                  Figure to <br />                                                                  Different <br />                                                                  applied     Buchnera <br />                                                                           6 kinds simulated <br />                                                                                           chromosome <br />                                                                                                permutations gene positions <br />                                                                  Different kinds simulated permutations gene <br />                                                                  positions applied Buchnera chromosome. Sin- <br />                                                                  gletons drawn green genes predicted in <br />                                                                  operons drawn red. 0: Original location genes on <br /> Buchnera <br /> Spatial <br /> Figure  periodic <br />          5       transcriptional patterns genome       chromosome, 1: permutation preserving rank and <br /> Spatial periodic transcriptional patterns                 spacing pTU modifying rank <br /> genome Buchnera. (A) Periodogram Buchnera log2             genes pTU, 2: permutation preserving the <br /> nBGT. The main periods composing long, medium short-         rank spacing pTU, 3: permutation of <br /> range spatial patterns respectively specified black,      rank location pTU, 4: gene positions on <br /> green blue. For comparison, periodogram log2          chromosome randomly assigned. <br /> gDNA signals shown black. (B) Averaged smoothed sig- <br /> nal related period &quot;86.86&quot; drawn black, the <br /> chromosome, Buchnera transcript abundances (real <br /> data) appear red. <br />  <br />  <br />  <br /> correlated groups genes (Table 2). Interestingly, the <br /> mean size pTU Buchnera 2 (45 of <br /> the 82 pTU doublets). Moreover, analysing differ- <br /> ent spectral density (periodograms) appeared the <br /> presence spatial periodic patterns transcriptional         Table 2: Autocorrelation values Buchnera nBGT levels the <br /> activity Buchnera less pronounced,        different permutations gene positions <br /> eventually disappeared, genes randomly <br /> assigned Buchnera chromosome (Table 3,              Inter-gene    Perm. 0    Perm. 1     Perm. 2     Perm. 3     Perm. 4 <br />                                                                    distances <br /> supplemental Figure 4 Additional file 4). The peri- <br /> ods preferentially affected permutations             1         0.3992 *    0.2842 *   0.2498 *    0.2240 *     -0.0249 <br /> are: (1) short-range periods permutation            2         0.1977 *    0.1474 *   0.1385 *    0.1386 *      0.0467 <br /> (shuffling singletons preserving operon back-                 3         0.1430 *    0.1346 *    0.0834      0.0582       0.0311 <br /> bone), (2) medium-range periods second                4         0.1336 *    0.0987 *   0.0993 *     0.0067       0.0078 <br /> (slight degradation operon backbone), (3)              5         0.1277 *    0.1165 *   0.1571 *     0.0185       0.0595 <br />                                                                       6         0.1577 *    0.1153 *   0.0968 *    -0.0069      -0.0314 <br /> long-range periods permutation (operons are <br />                                                                       7         0.1079 *    0.0983 *    0.0375      0.0745      -0.0070 <br /> shuffled). This observation suggests that, struc-         8         0.0925 *     0.0093     0.0279      0.0415     -0.0934 * <br /> ture putative operons play key role              9          0.0920      0.0259     0.0345      0.0237      -0.0013 <br /> existence transcriptomic spatial patterns Buchnera,             10        0.1025 *     0.0784     0.0387     -0.0063      -0.0502 <br /> the order spacing singletons pTU              11         0.0866      0.0081     0.0121      0.0266      -0.0291 <br /> are important establishing observed periodici-            12        0.1071 *     0.0598     0.0854     -0.0336       0.0014 <br />                                                                       13         0.0372      0.0428     0.0632     -0.0822       0.0508 <br /> ties. Moreover, genes located neighbourhood of <br />                                                                       14         0.0888      0.0689     0.0554     -0.0703       0.0079 <br /> operons co-expressed operons                  15         0.0786     0.0891 *    0.0079     -0.0469      -0.0382 <br /> kinds &quot;supra-operonic&quot; structures previously <br /> suggested work Moran et al. [35].                       0: Actual data; 1: conserved operons, order spacing; 2: conserved <br />                                                                   operon order; 3: conserved operons; 4: gene scrambling (see text and <br /> Discussion                                                        Figure 6 details). Autocorrelation values reported at <br />                                                                   operon size scale (1 15 inter-gene distances). <br /> To knowledge, presented analy-         *: significant P-value 95% confidence limits <br /> sis showing i) representation whole-genome tran- <br />  <br />                                                                                                                          Page 6 12 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                                 http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> Table 3: Results spectral analysis testing periodicity         abundances gene essential functions reduced <br /> Buchnera log2 nBGT                                                           genome Buchnera. <br />              Fisher&apos;s Kappa        Bartlett&apos;s Kolmogorov-Smirnov <br />                                                                              In work analysed relationship between <br />  Perm. 0        13.148 **                        0.275 **                    mRNA abundances genes GC ratio Buchnera (taken <br />  Perm. 1        10.930 **                        0.194 **                    indicator gene evolution rate). By combining <br />  Perm. 2         9.174 *                         0.177 **                    transcription analysis, evolution rates comparative <br />  Perm. 3          6.941                          0.173 **                    genomics, able define new candidates the <br />  Perm. 4          6.595                           0.041                      essential gene set Buchnera. In bacteria, genes encoding <br />                                                                              ribosomal, cell division chaperone/protease proteins <br />  The periodicity tested original location genes the <br />                                                                              considered essential known be <br />  chromosome according different simulated permutations of <br />  gene positions. 0: Actual data; 1: conserved operons, order             constitutively highly expressed. As expected, data <br />  spacing; 2: conserved operon order; 3: conserved operons; 4: gene           showed genes encoding ribosomal pro- <br />  scrambling (see text Figure 6 details).                             teins, genes encoding chaperonins, relatively well <br />  **: significant P-value 99% confidence limits, *: significant P-value <br />  95% confidence limits <br />                                                                              conserved highly expressed Buchnera. Interest- <br />                                                                              ingly, highly conserved expressed <br /> scriptional data Buchnera ii) links                       genes, genes involved biosynthesis of <br /> mRNA abundances chromosomal organization                            EAAs and, particular, genes isoleucine and <br /> highly reduced bacterial genome, basis experi-                     valine pathway. Also remarkable genes encod- <br /> mental data. The large number prokaryotic genome                          ing aminoacyl-tRNA synthetases 8 10 <br /> sequences available databases possible                     aphid EAAs weakly expressed, genes encoding <br /> study links bacterial chromosomes. Using                       aminoacyl-tRNA synthetases non-EAAs either <br /> codon adaptation index (CAI) values computed                            moderately highly expressed. A possible explanation <br /> ribosomal proteins, Rocha Danchin shown that,                       observation Buchnera, relatively <br /> in B. subtilis E. coli, importance expres-                 slow-growing bacterium, does necessitate high rates of <br /> siveness determining localization genes                     protein production constitutively synthesizes EAAs in <br /> the leading strand negligible, absent,                       order furnish aphid. Reducing abun- <br /> essentiality taken account [36]. They con-                      dance specific aminoacyl-tRNA synthetases might <br /> firmed results sequenced genomes,                       increase concentration free EAAs Buchnera cells <br /> the exception non-free-living bacteria                         facilitating transport amino acids aphid <br /> which Buchnera important [4]. How-                       host cell. This observation reminiscent similar puta- <br /> ever, authors underlined difficulty obtaining                     tive adaptive response Buchnera, selectively <br /> correct CAI values genomes intracellular bacteria,                 underexpresses pheT aromatic EEAs shortage [26]. <br /> which generally sufficient codon usage bias. <br /> Moreover, explanation exceptions                         In study, orphan genes yba3 <br /> found Buchnera attributed authors&apos; assign-                     yba4 evolve rapidly highly expressed, <br /> ment gene essentiality, exclusively based                    yba4 highest expressed genes Buch- <br /> the homology E. coli. Indeed, Rocha et al. [4]                    nera. The conservation genes BAp the <br /> found essential genes strand distribution bias Buch-                   Buchnera harboured aphid Schizaphis graminum <br /> nera, bias 60% using minimal gene set                      (BSg), coupled high expression level, suggest <br /> proposed Gil et al. [25]. More recently, Price et al. [37]                particular relevance symbiosis <br /> have revisited question gene-strand bias bacteria,                 Buchnera aphids S. graminum A. pisum, as <br /> using gene expression microarray data,                         proposed Shimomura et al. [38] A. <br /> shown that, B. subtilis E. coli, genes operons                 pisum. <br /> on leading strand DNA highly expressed <br /> than genes operons lagging strand. This observa-                   Among genes rapidly evolving (this work, Tamas et al. <br /> tion true essential non-essential genes.                    [19] Reymond et al. [26]) highly expressed in <br />                                                                              Buchnera, flagellar genes. These data, <br /> In Buchnera, highly expressed genes gen-                   taken recent experimental evidence the <br /> erally essential genes pTU. Independently                      Buchnera incomplete flagellar apparatus function a <br /> the essentiality factor, genes pTU                       &quot;protein transporter&quot; [39], support idea flagellar <br /> highly expressed singletons. We                     genes taking new important functions sym- <br /> genes pTU essential genes highly                         biotic context. The flagellar gene set BAp composed <br /> expressed non essential ones effect                    26 genes fliEFGHIJKMNPQR flhAB genes <br /> not observed singletons. These results underline                     located leading strand (except fliE), flgNA- <br /> conservation coherent relationship mRNA                         BCDEFGHIJK genes located lagging strand <br />  <br />                                                                                                                                  Page 7 12 <br />                                                                                                            (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                           http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> DNA (except flgN flgA). These flagellar genes             tion. Indeed, presence spatial periodic compo- <br /> conserved BSg. However, Buchnera distant aphid               nents Buchnera gene expression the <br /> lineage, Baizongia pistaciae (BBp), lost flagellar            conservation operon structures, modification the <br /> genes (flgA, flgD, flgE, flgK, flgN) fli            order and/or number singletons located <br /> ones. Finally, Buchnera dramatically                     pTU affect periodicities and <br /> reduced genome, aphid Cinara cedri (BCc),                 autocorrelation values. We shown, however, that <br /> lost genes BBp flg genes               modifications reduced spatial patterns, alleviat- <br /> (flgB,flgC, flgG, flgJ) fli genes (fliE, fliJ, fliK, fliM),   ing importance high-order arrangement the <br /> hence preserving minimal type III virulence secre-              genes chromosome endosymbiont. <br /> tion [21]. The evolutionary selection major- <br /> ity fli genes suggests possible importance          Jeong et al. [14] classified transcriptional periods <br /> the new putative transport function genes                  E. coli categories: short- <br /> Buchnera. It note, conserved fli genes             range (up 16 genes); medium-range (100&#226;&#8364;&#8220;125 genes); <br /> among Buchnera lineages, probably involved                  long-range (600&#226;&#8364;&#8220;800 genes). The existence short <br /> new function, located leading strand,               periods, 17 genes Buchnera, allows corrobo- <br /> most gene losses occurred lagging strand.                rate hypothesis, proposed previous study B. <br /> Previous comparative genomics analyses [40,41]                     subtilis E. coli, short range element could <br /> tempted dissect evolutionary forces driving                 property structural nucleoid common other <br /> genome organization Buchnera lineages (i.e.,                bacteria, corresponding large DNA spirals nucle- <br /> gene strand bias, gene protein composition, gene                   <span id='am-2' about='Thesaurus:OID' typeof='owl:Thing'>oid</span> surface [15]. The medium long-range periods <br /> expressivity, gene evolution rate gene loss). Our                  shorter Buchnera identified free-living <br /> results partly consistent previous analyses             bacteria, probably greatly reduced size <br /> as (1) highly conserved genes highly                      genome. However, kinds periods are <br /> expressed, (2) essential genes pTU highly expressed             understood correspond domains <br /> and probably preserved mutations purifying                     identified far nucleoid [15]. The effect sec- <br /> selection, (3) positive selection shape new &quot;sym-              ond simulated gene permutations (Perm. 2 and <br /> biotic&quot; functions genes highly expressed                  Perm. 3 Figure 6) medium long-range peri- <br /> highly evolving. However, reject idea, based             ods respectively, explained importance of <br /> on CAI analyses, expressiveness factor driving               spatial location operons chromosome <br /> gene strand bias Buchnera.                                          neighbourhood singletons form &quot;supra- <br />                                                                        operonic&quot; structures Buchnera. Moreover, decrease <br /> Finally, important result study discovery           maximum size autocorrelated groups of <br /> of spatial patterns transcriptional activity chro-           genes (Table 2), different permutations gene <br /> mosome Buchnera: i.e., transcription genes               positions, corroborates hypothesis &quot;supra-oper- <br /> along chromosome determined according spatial                onic&quot; structures Buchnera. But speculation needs to <br /> constraints. From autocorrelation spectral analysis,               studied experimentally confirmed. Finally, <br /> four groups spatial patterns defined: (i) autocor-           observation transcriptional periodic patterns, cou- <br /> related short-range (between 2 8 genes), (ii) periodic             pled conservation genome Nucle- <br /> short-range (up 17 genes), (iii) periodic medium-range              <span id='am-3' about='Thesaurus:OID' typeof='owl:Thing'>oid</span> Associated Proteins (NAP) H-NS, IHF Fis, <br /> (between 23 61 genes) (iv) periodic long-range                 suggest Buchnera maintained nucleoid structure <br /> (over 87 genes) structural components.                                 responsible differences gene transcription levels <br />                                                                        basal conditions. These NAPs previously <br /> Autocorrelated short-range patterns, determined                 differentially expressed Buchnera [26] fac- <br /> autocorrelation function gene transcription levels,                 ing nutritional constraints. However, role tran- <br /> showed genes spaced gene-to-gene distance                 scriptional regulation remains presently speculative the <br /> less 8 highly correlated expressions. As            aphid endosymbiont. <br /> suggested E. coli B. subtilis, propose Buchn- <br /> era correlations reflect co-ordinated tran-             Conclusion <br /> scription genes operons [14,15]. This                        In conclusion, analysis mRNA abundances with <br /> observation reinforces result mentioned con-                 regard chromosomal organization Buchnera has <br /> cerning conservation functional transcription units             shown, despite important reduction genome size <br /> in Buchnera. However, permuting gene position                 apparent decay transcriptional regulatory <br /> the chromosome, observed organization                   capacity, conservation relationship these <br /> genes putative operons sufficient fully                 parameters aphid symbiont. Our work shows <br /> explain observed periodic spatial patterns transcrip-           organization genes operons their <br />  <br />  <br />                                                                                                                            Page 8 12 <br />                                                                                                      (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                   http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> essential functions influence transcription Buchnera,       Slide preparation hybridization <br /> whereas significant leading/lagging strand bias         The Buchnera oligo-array composed 6144 spots, <br /> observed. This work underlines difficulties        including probes Buchnera gene were <br /> defining gene essentiality intracellular symbiotic bacte-   designed ROSO software [30]. A additional probe <br /> ria importance additional specific gene set      designed specific genes chosen function <br /> for BAp. Finally, showed existence transcrip-      symbiotic relationships (amino acid biosynthetic <br /> tional periodicity chromosome Buchnera.           genes putative transporters). The 35-mer oligonucle- <br /> From results, comparative analyses performed          otide probes printed, quadruplets, differ- <br /> Buchnera taken species aphids,       ent pins Quantifoil Micro Tools aldehyde slides <br /> other endosymbiotic bacteria, Blochmannia spp.         (Interchim, Montlu&#195;&#167;on, France). Positive negative <br /> from carpenter ant Wigglesworthia spp.         controls spread slide. <br /> tsetse fly establish results presented <br /> here common trait insect endosymbionts.           The slides hybridized using automated Ventana <br />                                                                Discovery station (Ventana Medical Systems, Illkirch, <br /> Methods                                                        France) 45&#194;&#176;C 8 hours. Following hybridization, the <br /> Aphid rearing                                                  slides washed solutions variable stringency, <br /> A long-established parthenogenetic clone (LL01) A.          dried centrifugation scanned fluorescence <br /> pisum maintained 21&#194;&#176;C, 16 hour light pho-        using GeneTac LSIV scanner (Genomic Solutions, <br /> toperiod, Vicia fabae. It shown free of    Huntingdon, UK). The signal intensity value spot <br /> the taxa secondary endosymbionts identified         (pixel medians) recorded qualitatively analysed <br /> date. For experiments, used aphids         GenePix Pro 4.1 software (Axon Instruments, Foster <br /> reared birth day-7 plant (basal conditions      City, CA, USA). <br /> of gene expression Buchnera) Buchnera puri- <br /> fied host described Charles Ishikawa      Four slides hybridized: 2 Buchnera gDNA 2 <br /> [16].                                                          cDNA. Microarray data available Array <br />                                                                Express database, accession E-TABM-193. <br /> Microarray experiments <br /> Nucleic acids purification labelling                       Data analysis <br /> The protocols RNA extraction labelling       Genomic DNA normalization <br /> previously described Calevro et al. [42]. Briefly, total    Fluorescence data filtrated spots low <br /> RNA purified using Trizol method possi-        quality removed. In study did consider <br /> ble gDNA contaminants removed using DNase RQ1             plasmidic genes present copies the <br /> RNase-free (Promega, Madison, WI, USA). Total RNA          Buchnera genome, excluded data set <br /> subsequently purified RNeasy&#194;&#174; column (Qiagen,            genes encoding RNA genes (tRNAs, ribosomal RNAs and <br /> Hilden, Germany). Then, 15 &#194;&#181;g total RNA indi-          enzyme RNA components), specific hybridi- <br /> rectly labelled incorporating aminoallyl-dUTP          zation properties [43]. Moreover, 30 genes high <br /> reverse transcript cDNA (Amersham Biosciences, Piscata-        quality signal excluded. To sure dele- <br /> way, NJ, USA). Finally, Cy3 fluorescent dyes (Amersham)        tions introduce biases study, checked <br /> were coupled targets prior purification        genes uniformly distributed dif- <br /> Autoseq&#226;&#8222;&#162; G-50 column (Amersham).                               ferent categories analysed (pTU, essentiality, strand, <br />                                                                mRNA abundances). Data different repeated <br /> Genomic DNA treated RNase A (Sigma-Aldrich,           slides scaled averaged. Fluorescence means <br /> St Quentin Fallavier, France) classically purified    equal print-tip group slides <br /> phenol/chloroform/isoamyl alcohol (25:24:1, v/v/v) (Q-         (no print-tip effect). For data normalization, cal- <br /> Biogene, Irvine, CA, USA). After testing different concen-     culated mean gDNA fluorescent signals (M) to <br /> trations gDNA (from 1 6 &#194;&#181;g), concentration 5 &#194;&#181;g     determine weighting coefficient Ki S spots (i) of <br /> was chosen normalization mRNA signals (data         oligo-array. Finally, weights applied to <br /> not shown). gDNA labelling performed using             Buchnera mRNAi signals order normalize them <br /> Nick Translation Kit (Amersham) directly incorpo-       (mRNAnormi). The following formulas used nor- <br /> rating dUTP-Cy3 (Amersham). Non-incorporated fluores-          malize mRNA signals: <br /> cent dyes eliminated purification Autoseq&#226;&#8222;&#162; <br /> G-50 column. <br />                                                                     1 S                   gDNAi                         mRNAi <br />                                                                M=     &#226;&#710;&#8216; gDNAi <br />                                                                     S =1 <br />                                                                                    Ki = <br />                                                                                             M <br />                                                                                                    mRNAnormi = <br />                                                                                                                          Ki <br />  <br />  <br />  <br />  <br />                                                                                                                   Page 9 12 <br />                                                                                             (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                       http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> For validation normalization procedure,              Buchnera. The standard error autocorrelation estimates <br /> used specific data sets 43 371 genes 3         computed as: <br /> 2 different probes gene respectively. These probes sets <br /> were characterized high quality signals types <br />                                                                                                  1 k &#226;&#710;&#8217;1 2 <br /> of hybridization (2 slides gDNA 2 slides with <br /> cDNA) <br />                                                                                        SEk =       &#226;&#710;&#8216; ri <br />                                                                                                  N =1 <br />  <br /> Buchnera genomic data                                              Note distance genes used arti- <br /> Sequence data Buchnera aphidicola genome,          cle difference ranks chromosome <br /> corresponding annotations, retrieved GenBank             (approximately equivalent number kb). <br /> [44]. The <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> essential genes based the <br /> minimal gene set supporting bacterial life (see intro-         In order analyse dependence transcriptional <br /> duction part) described Gil et al. [25]. Putative operons       activity function chromosomal position, also <br /> in Buchnera manually identified comparison            used spectral density autocorrelation function, <br /> with E. coli genome. For that, E. coli orthologous        estimated Fourier transform techniques. The periodog- <br /> gene Buchnera searched EcoCyc database               ram obtained method illustrates periodic com- <br /> [45], groups Buchnera genes corresponding tran-          ponents, corresponding intensities, the <br /> scription units E. coli collected. After removing      transcriptomic signals chromosome Buchn- <br /> singletons, determined 82 pTUs Buchnera consisting           era. The periodic signals main periods Buchnera <br /> of 2 13 genes. In study, estimated gene evolu-          drawn averaging smoothing signals all <br /> tion rates GC content genes instead            segments making period. Smoothing the <br /> the non-synonymous substitution rates (Ka)              signals performed &quot;Fit Spline&quot; function the <br /> parameters highly correlated AT-rich genome          JMP software. <br /> Buchnera, GC content allows estima- <br /> tions orphan genes Buchnera.                         Statistical analysis <br />                                                                    Statistical analyses microarray data, mean comparisons <br /> Spectral analysis estimation periodicities              (ANOVA F-test) distribution comparisons (Likeli- <br /> This analysis performed platform &quot;Time                hood ratio test) performed using JMP software. For <br /> series&quot; JMP 5.0.1.2 software (SAS Institute, Inc.).         distribution comparisons, previously split Buchnera <br /> The presence spatial periodic components Buchnera            gene expression levels bins: 100 highly <br /> gene expression levels tested Fisher&apos;s Kappa            expressed genes (labelled &quot;highly expressed&quot;); 100 <br /> Bartlett&apos;s Kolmogorov-Smirnov statistical tests. Fisher&apos;s          expressed genes (labelled &quot;weakly expressed&quot;); <br /> Kappa tests null hypothesis series Gaussian        (labelled &quot;moderately expressed&quot;). <br /> white noise alternative hypothesis the <br /> series contains added deterministic periodic compo-             Abbreviations <br /> nent unspecified frequency. The Bartlett&apos;s Kolmogorov-          BAp: Buchnera aphid Acyrthosiphon pisum <br /> Smirnov test compares normalized cumulative perio- <br /> dogram cumulative distribution function            BBp: Buchnera aphid Baizongia pistaciae <br /> uniform (0, 1) test null hypothesis series is <br /> white noise. These tests applied Buchnera         BCc: Buchnera aphid Cinara cedri <br /> data set following different permutations gene posi- <br /> tions, preserving rank spacing          BSg: Buchnera aphid Schizaphis graminum <br /> the pTU chromosome. To determine spatial pat- <br /> terns, calculated autocorrelation function           EAAs: essential amino acids <br /> Buchnera transcriptional spatial series. The autocorrela- <br /> tion kth lag (inter-gene distance) given by:            gDNA: genomic DNA <br />  <br />                                      N                             nBGT: normalized basal gene transcription <br />           ck                 1 <br />    rk = <br />           c0 <br />                ck =           &#226;&#710;&#8216; <br />                             N &#226;&#710;&#8217; k x =k +1 <br />                                          (y x &#226;&#710;&#8217; y )(y x &#226;&#710;&#8217;k &#226;&#710;&#8217; y ) <br />                                                                    pTU: putative transcription units <br /> with k = 0, 1, 2..., N/4, yx mRNA abundance <br /> at index corresponding given Buchnera gene loca-          Authors&apos; contributions <br /> tion y mean N gene expression values          JV carried sample labelling hybridization, per- <br />                                                                    formed statistical analysis drafted manuscript. <br />                                                                    FC participated design study, microarray <br />  <br />  <br />                                                                                                                       Page 10 12 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                                          http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br /> labelling hybridization, draft manuscript. DR                          3.    Rocha E: Is role replication fork asymmetry the <br />                                                                                         distribution genes bacterial genomes? Trends Microbiol <br /> participated analysis periodic transcriptional                                2002, 10:393-395. <br /> components. JB carried slide preparation                              4.    Rocha EP, Danchin A: Gene essentiality determines chromo- <br /> helped labelling protocols. YR GF participated                              organisation bacteria.             Nucleic Acids Res 2003, <br />                                                                                         31:6570-6577. <br /> statistical analysis interpretation results. JMF par-                      5.    Brewer BJ: When polymerases collide: replication the <br /> ticipated coordination study. HC conceived                                transcriptional organization E. coli chromosome. Cell <br /> the study, participated statistical analysis help                            1988, 53:679-686. <br />                                                                                   6.    French S: Consequences replication fork movement <br /> to draft manuscript.                                                                    transcription units vivo.                  Science 1992, <br />                                                                                         258:1362-1365. <br />                                                                                   7.    Omont N, Kepes F: Transcription/replication collisions cause <br /> Additional material                                                                     bacterial transcription units longer leading <br />                                                                                         strand replication. Bioinformatics 2004, 20:2719-2725. <br />                                                                                   8.    Mirkin EV, Mirkin SM: Mechanisms transcription-replication <br />      Additional file 1                                                                  collisions bacteria. Mol Cell Biol 2005, 25:888-895. <br />      Analysis genomic DNA signals hybridized Buchnera oligo-            9.    Lawrence JG: Gene organization: selection, selfishness, and <br />                                                                                         serendipity. Annu Rev Microbiol 2003, 57:419-440. <br />      array. This figure shows genomic DNA hybridization signals        10.   Korbel JO, Jensen LJ, von Mering C, Bork P: Analysis genomic <br />      Buchnera oligo-array.                                                              context: prediction functional associations con- <br />      Click file                                                                served bidirectionally transcribed gene pairs. Nat Biotechnol <br />      [http://www.biomedcentral.com/content/supplementary/1471-                          2004, 22:911-917. <br />      2164-8-143-S1.pdf]                                                           11.   Audit B, Ouzounis CA: From genes genomes: universal <br />                                                                                         scale-invariant properties microbial chromosome organi- <br />                                                                                         sation. J Mol Biol 2003, 332:617-633. <br />      Additional file 2                                                            12.   Peter BJ, Arsuaga J, Breier AM, Khodursky AB, Brown PO, Cozzarelli <br />      Distribution log2 mRNA abundances (A) (B) gDNA                 NR: Genomic transcriptional response loss chromo- <br />      normalization. This figure compares distributions Buchnera gene             somal supercoiling Escherichia coli. Genome Biol 2004, 5:R87. <br />                                                                                   13.   Kepes F: Periodic transcriptional organization E. coli <br />      transcription levels genomic DNA normalization.                   genome. J Mol Biol 2004, 340:957-964. <br />      Click file                                                          14.   Jeong KS, Ahn J, Khodursky AB: Spatial patterns transcrip- <br />      [http://www.biomedcentral.com/content/supplementary/1471-                          tional activity chromosome Escherichia coli. Genome <br />      2164-8-143-S2.pdf]                                                                 Biol 2004, 5:R86. <br />                                                                                   15.   Carpentier AS, Torresani B, Grossmann A, Henaut A: Decoding the <br />                                                                                         nucleoid organisation Bacillus subtilis Escherichia coli <br />      Additional file 3                                                                  gene expression data. BMC Genomics 2005, 6:84. <br />      Organization genes chromosome Buchnera aphidicola. This         16.   Charles H, Ishikawa H: Physical genetic map genome <br />      figure gives information gene organization Buchnera                   Buchnera, primary endosymbiont pea aphid <br />      aphidicola genome.                                                                 Acyrthosiphon pisum. J Mol Evol 1999, 48:142-150. <br />      Click file                                                          17.   Gil R, Sabater-Munoz B, Latorre A, Silva FJ, Moya A: Extreme <br />                                                                                         genome reduction Buchnera spp.: minimal <br />      [http://www.biomedcentral.com/content/supplementary/1471-                          genome needed symbiotic life. Proc Natl Acad Sci USA 2002, <br />      2164-8-143-S3.pdf]                                                                 99:4454-4458. <br />                                                                                   18.   Shigenobu S, Watanabe H, Hattori M, Sakaki Y, Ishikawa H: Genome <br />      Additional file 4                                                                  sequence endocellular bacterial symbiont aphids <br />                                                                                         Buchnera sp. APS. Nature 2000, 407:81-86. <br />      Periodograms Buchnera log2 normalized mRNA abundances. This fig-          19.   Tamas I, Klasson L, Canback B, Naslund AK, Eriksson AS, <br />      ure illustrates periodograms Buchnera normalized mRNA abundances                Wernegreen JJ, Sandstrom JP, Moran NA, Andersson SG: 50 million <br />      original location genes chromosome according dif-         years genomic stasis endosymbiotic bacteria. Science <br />      ferent simulated permutations gene positions.                                   2002, 296:2376-2379. <br />      Click file                                                          20.   van Ham RC, Kamerbeek J, Palacios C, Rausell C, Abascal F, Bastolla <br />                                                                                         U, Fernandez JM, Jimenez L, Postigo M, Silva FJ, Tamames J, Viguera E, <br />      [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                         Latorre A, Valencia A, Moran F, Moya A: Reductive genome evo- <br />      2164-8-143-S4.pdf]                                                                 lution Buchnera aphidicola. Proc Natl Acad Sci USA 2003, <br />                                                                                         100:581-586. <br />                                                                                   21.   Perez-Brocal V, Gil R, Ramos S, Lamelas A, Postigo M, Michelena JM, <br />                                                                                         Silva FJ, Moya A, Latorre A: A small microbial genome: end <br />                                                                                         long symbiotic relationship? Science 2006, 314:312-313. <br /> Acknowledgements                                                                  22.   Douglas AE: The nutritional physiology aphids. Adv Insect <br /> We thank G. Duport technical assistance, N. Gadot ANI-                 Physiol 2003, 31:73-140. <br />                                                                                   23.   Blattner FR, Plunkett G 3rd, Bloch CA, Perna NT, Burland V, Riley M, <br /> PATH platform (Faculty Medicine, RTH Laennec, Lyon) helping                 Collado-Vides J, Glasner JD, Rode CK, Mayhew GF, Gregor J, Davis <br /> the hybridization protocol, V. James critical revision Eng-          NW, Kirkpatrick HA, Goeden MA, Rose DJ, Mau B, Shao Y: The <br /> lish language. We thank Rh&#195;&#180;ne-Alpes Genopole transcriptomic                    complete genome sequence Escherichia coli K-12. Science <br /> platform DTAMB slide spotting. This work supported &quot;Pro-                 1997, 277:1453-1474. <br />                                                                                   24.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes are <br /> gramme F&#195;&#169;d&#195;&#169;rateur INRA Biologie Int&#195;&#169;grative AgroBI 2006&quot;,                 evolutionarily conserved nonessential genes <br /> &quot;action bioinformatique INSA Lyon 2003&#226;&#8364;&#8220;2006&quot;.                                        bacteria. Genome Res 2002, 12:962-968. <br />                                                                                   25.   Gil R, Silva FJ, Pereto J, Moya A: Determination core a <br /> References                                                                              minimal bacterial gene set. Microbiol Mol Biol Rev 2004, <br />                                                                                         68:518-537. <br /> 1.      Allen TE, Price ND, Joyce AR, Palsson BO: Long-range periodic             26.   Reymond N, Calevro F, Vinuelas J, Morin N, Rahbe Y, Febvay G, <br />         patterns microbial genomes indicate significant multi-                       Laugier C, Douglas A, Fayard JM, Charles H: Different levels of <br />         scale chromosomal organization. PLoS Comput Biol 2006, 2:e2.                    transcriptional regulation trophic constraints the <br /> 2.      Rocha EP: The replication-related organization bacterial <br />         genomes. Microbiology 2004, 150:1609-1627. <br />  <br />  <br />                                                                                                                                             Page 11 12 <br />                                                                                                                        (page number citation purposes) <br />  BMC Genomics 2007, 8:143                                                                      http://www.biomedcentral.com/1471-2164/8/143 <br />  <br />  <br />  <br />       reduced genome Buchnera aphidicola APS. Appl Environ <br />       Microbiol 2006, 72:7760-7766. <br /> 27.   Talaat AM, Howard ST, Hale W, Lyons R, Garner H, Johnston SA: <br />       Genomic DNA standards gene expression profiling in <br />       Mycobacterium tuberculosis. Nucleic Acids Res 2002, 30:e104. <br /> 28.   Weil MR, Macatee T, Garner HR: Toward universal standard: <br />       comparing methods standardizing spotted microar- <br />       ray data. Biotechniques 2002, 32:1310-1314. <br /> 29.   Kim H, Zhao B, Snesrud EC, Haas BJ, Town CD, Quackenbush J: Use <br />       RNA genomic DNA references inferred compari- <br />       sons DNA microarray analyses.                  Biotechniques 2002, <br />       33:924-930. <br /> 30.   Reymond N, Charles H, Duret L, Calevro F, Beslon G, Fayard JM: <br />       ROSO: optimizing oligonucleotide probes microarrays. <br />       Bioinformatics 2004, 20:271-273. <br /> 31.   The Profiling Escherichia coli Chromosome database <br />       [http://www.shigen.nig.ac.jp/ecoli/pec] <br /> 32.   Balazsi G, Kay KA, Barabasi AL, Oltvai ZN: Spurious spatial perio- <br />       dicity co-expression microarray data printing <br />       design. Nucleic Acids Res 2003, 31:4425-4433. <br /> 33.   Kluger Y, Yu H, Qian J, Gerstein M: Relationship gene <br />       co-expression probe localization microarray slides. <br />       BMC Genomics 2003, 4:49. <br /> 34.   Lercher MJ, Hurst LD: Co-expressed yeast genes cluster a <br />       long range regularly spaced. J Mol Biol 2006, <br />       359:825-831. <br /> 35.   Moran NA, Mira A: The process genome shrinkage the <br />       obligate symbiont Buchnera aphidicola. Genome Biol 2001, <br />       2:RESEARCH0054. <br /> 36.   Rocha EP, Danchin A: Essentiality, expressiveness, drives <br />       gene-strand bias bacteria. Nat Genet 2003, 34:377-378. <br /> 37.   Price MN, Alm EJ, Arkin A <br /> </body></html>