<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 96, pp. 1409&#226;&#8364;&#8220;1414, February 1999 <br /> Cell Biology <br />  <br />  <br />  <br /> Identification SSF1, CNS1, HCH1 multicopy <br /> suppressors Saccharomyces cerevisiae Hsp90 <br /> loss-of-function <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> <br />      (Hsp90ychaperoneyyeastymulticopy suppressionysignal transduction) <br />  <br /> DEBRA F. NATHAN*&#226;&#8364;&#160;, MELISSA HARJU VOS*&#226;&#8364;&#161;,                      AND    SUSAN LINDQUIST&#194;&#167; <br /> Department Molecular Genetics Cell Biology Howard Hughes Medical Institute, The University Chicago, 5841 South Maryland Avenue, <br /> Chicago, IL 60637 <br />  <br /> Contributed Susan Lindquist, December 14, 1998 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>          Hsp90 functions multicomponent chap-                         folding-competent conformations acted by <br /> erone promote maturation maintenance                           chaperones (18, 20, 21, 23, 24). <br /> a diverse, specific, set target proteins play key roles                    In vivo, normal temperatures, Hsp90 highly selective <br /> in regulation cell growth development. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> identify                       chaperone, closely related proteins p60c-src <br /> additional components Hsp90 chaperone                         p60v-src varying greatly dependence function <br /> targets, searched multicopy suppressors various                           (12, 25). Most known Hsp90 target proteins components of <br /> temperature-sensitive mutations yeast Hsp90 gene,                            signal transduction pathways, include steroid hormone <br /> HSP82. Three suppressors isolated Hsp90 mutant                         receptors, serineythreonine kinases, tyrosine kinases, and <br /> (glutamate 3 lysine amino acid 381). Each exhibited                            transcription factors (13&#226;&#8364;&#8220;16). The functions signaling pro- <br /> unique, allele-specific <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> suppression                           teins require exist inactive conformations are <br /> Hsp90 mutants unique structural biological                              poised immediate activation. Biochemical genetic data <br /> properties. SSF1 member essential gene family                        laboratories suggest Hsp90 helps target <br /> functions response mating pheromones. CNS1                          proteins achieve maintain inherently unstable confor- <br /> essential gene encodes component Hsp90 chap-                          mations required function (8, 10, 12, 14&#226;&#8364;&#8220;16). <br /> erone machinery. The role HCH1 unknown; sequence                             Perhaps extensively studied chaperone function of <br /> has strong homology protein known function.                            Hsp90 role maturation progesterone (PR) <br /> SSF1 CNS1 weak suppressors, HCH1 re-                               glucocorticoid (GR) receptors, Hsp90 forms <br /> stored wild-type growth rates temperatures tested                         stoichiometric complexes. Association Hsp90 several <br /> cells expressing E381K mutant. Overexpression CNS1                           cochaperones serves receptors inactive <br /> or HCH1, SSF1, enhanced maturation heter-                          conformations bind ligand high affinity (8, 10, <br /> ologous Hsp90 target protein, p60v-src. These results suggest                       14&#226;&#8364;&#8220;16). Studies reticulocyte lysates indicate PR matu- <br /> that like Cns1p, Hch1p general modulator Hsp90                              ration involves proteins occurs cycle <br /> chaperone functions, Ssf1p likely Hsp90                        sequential interactions distinct chaperone complexes, <br /> target protein functions pathway Hsp90                         contain Hsp90 (13, 26). <br /> target protein.                                                                        In yeast S. cerevisiae, nearly identical genes, HSP82 <br />                                                                                     HSC82, encode Hsp90 proteins. Although Hsp90 ex- <br /> The structure function Hsp90 family molecular                         tremely abundant yeast, levels reduced one <br /> chaperones conserved billion years                         order magnitude affecting cell viability 25&#194;&#176;C. <br /> evolution. Hsp90 proteins organisms diverse                          Much higher concentrations required growth higher <br /> budding yeast, Saccharomyces cerevisiae humans share                            temperatures (4). Previously, isolated S. cerevisiae <br /> 60% amino acid identity. Moreover, crystal structures                        Hsp90 temperature-sensitive (ts) mutants (10), which <br /> the N-terminal domains S. cerevisiae bovine Hsp90                        contains single amino acid substitution distinct <br /> proteins superimposable (1&#226;&#8364;&#8220;3). Hsp90 essential                           <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> respect cell morphology growth. As is <br /> viability eukaryotic organisms                  case ts mutations heat-shock proteins, seven <br /> been tested, S. cerevisiae (4), Drosophila melanogaster (5),                    partial loss-of-function mutants: have <br /> Schizosaccharomyces pombe (6). Both Drosophila                              reduced activity temperatures, ts they <br /> human family members substitute endogenous                              fulfill increased demand Hsp90 function high <br /> Hsp90 S. cerevisiae (7&#226;&#8364;&#8220;8), yeast Hsp90 proteins                      temperatures. One, however, behaves classic ts mutant: it <br /> effectively chaperone heterologously expressed target proteins                      nearly wild-type activity 25&#194;&#176;C, rapidly loses function <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> organisms (8&#226;&#8364;&#8220;12). Furthermore, Hsp90 functions                           shift high temperature. This mutant, G170D, contains a <br /> as dynamic chaperone complex, known                             substitution aspartic acid glycine amino acid position <br /> components highly conserved (13&#226;&#8364;&#8220;16).                              170 (10). The instability G170D high temperatures is <br />    In vitro, Hsp90 exhibits intrinsic, general chaperone                         consistent location residue 170 hydrophobic <br /> activity, preventing aggregation inherently unstable                      core N-terminal domain Hsp90 (1&#226;&#8364;&#8220;3). <br /> chemically denatured proteins (17&#226;&#8364;&#8220;23). In assays, Hsp90 <br /> is required large excess refold denatured pro-                        Abbreviation: ts, temperature sensitive. <br /> teins. Rather, maintains proteins nonaggregated,                        *D.F.N. M.H.V. contributed equally work. <br />                                                                                     &#226;&#8364;&#160;Present address: Cadus Pharmaceutical Corporation, 777 Old Saw <br />  <br /> The publication costs article defrayed page charge           Mill River Road, Tarrytown, NY 10591-6705. <br />                                                                                     &#226;&#8364;&#161;Present address: Abbott Laboratories, Advanced Technology Divi- <br /> payment. This article marked &#226;&#8364;&#732;&#226;&#8364;&#732;advertisement&#226;&#8364;&#8482;&#226;&#8364;&#8482; in <br />                                                                                      sion, 100 Abbott Park Road, Abbott Park, IL 60064-3500. <br /> accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                       &#194;&#167;To reprint requests addressed. e-mail: s-lindquist@ <br /> PNAS available online www.pnas.org.                                            uchicago.edu. <br />  <br />                                                                              1409 <br />  1410     Cell Biology: Nathan et al.                                                Proc. Natl. Acad. Sci. USA 96 (1999) <br />  <br />    The unique G170D mutant allowed examine                 plasmids designated piHGpdyP82 (wild-type HSP82), piH- <br /> breadth Hsp90&#226;&#8364;&#8482;s chaperone functions vivo. In tempera-          GpdyT22I (threonine amino acid residue 22 replaced by <br /> ture-shift experiments, Hsp90 does appear protect             isoleucine), piHGpdyA41V, piHGpdyG81S, piHGpdyT101I, <br /> proteins thermal inactivation. Hsp90 does, however,             piHGpdyG170D, piHGpdyG313S, piHGpdyE381K piH- <br /> facilitate refolding heterologous heat-damaged test         GpdyA587T. <br /> substrate, firefly luciferase (11). Pharmacological inhibition       Strains iP82a, iT22Ia (T22I), iA41Va (A41V), iG81Sa <br /> vertebrate Hsp90 produced similar results (18). These data           (G81S), iT101Ia (T101I), iG170Da (G170D), iG313Sa <br /> suggest function Hsp90 stress interact      (G313S), iE381Ka (E381K), iA587Ta (A587T) deriv- <br /> with proteins unfold maintain confor-         atives W303 (MATa ade2&#226;&#8364;&#8220;1 can1&#226;&#8364;&#8220;100 his3&#226;&#8364;&#8220;12, 16 leu2&#226;&#8364;&#8220;3, 112 <br /> mations amenable rapid refolding stress        trp1&#226;&#8364;&#8220;1 ura3&#226;&#8364;&#8220;1; R. Rothstein, Columbia University). In these <br /> has removed. This function related                strains, endogenous Hsp90 genes, HSC82 HSP82 are <br /> above-mentioned general chaperone activity Hsp90             deleted wild-type mutant HSP82 sequences, regulated <br /> unfolded test substrates vitro.                                   strong constitutive promoter GPD, integrated into <br />    Experiments G170D demonstrated Hsp90            genome. To create strains, integrating plasmids <br /> not required novo folding cellular proteins       described linearized NheI transformed <br /> in yeast. Rather, Hsp90 functions maturation and <br />                                                                      DPCLDa (10). Transformants grown 5-fluoroo- <br /> maintenance certain target proteins difficulty <br />                                                                      rotic acid (5-FOA) medium select lost the <br /> achieving stable structures (10, 11). Presumably, yeast cells <br />                                                                      wild-type HSC82 expression plasmid DPCLDa. The <br /> require Hsp90 function Hsp90 target <br /> proteins essential viability. Given diversity         ilep1a strain constructed similar manner that <br /> mammalian Hsp90 target proteins (13&#226;&#8364;&#8220;16) distinct             case wild-type HSP82 sequences control a <br /> phenotypes different S. cerevisiae Hsp90 point mutants (9,        low-expression promoter integrated the <br /> 10, 27), likely yeast Hsp90 target proteins   genome (29). <br /> include wide variety signal transducers function           Yeast bacterial transformations obtained using <br /> many pathways. Indeed, target recently             lithium acetate (35) calcium chloride (36) methods, re- <br /> identified; Ste11p MEK kinase involved MAP           spectively, using electroporation Gene Pulser <br /> kinase signaling pathways yeast (ref. 28; B. Errede, personal     (Bio-Rad). <br /> communication). The diversity Hsp90 target proteins              Yeast Media. Cells cultured SD, SR, Sgal (20 <br /> their central importance regulators growth develop-        gyliter glucose, raffinose, galactose, respectively, 5 <br /> ment makes Hsp90 particularly interesting chaperone             gyliter ammonium sulfate 1.7 gyliter yeast nitrogen <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> <br /> study. However, number range targets          amino acids, supplemented essential amino acids <br /> multicomponent, dynamic nature chaperone complexes            nucleotides) 5-FOA (1.7 gyl yeast nitrogen base without <br /> in functions present formidable problems elucidat-       amino acids ammonium sulfate, 1 gyl proline, 20 gyliter <br /> ing molecular mechanism.                                         glucose, 10 mgyliter uracil, 600 mgyliter 5-FOA, supple- <br />    We employed variety biochemical              mented essential amino acids nucleotides (37). Solid <br /> methods isolate identify proteins interact          media contained 20 gyliter Difco Bacto agar. <br /> Hsp90. Although methods successful,                Screen Multicopy Suppressors Hsp90 ts Mutants. <br /> they applicable proteins fairly abundant        Hsp90 ts mutant strains T101I, G170D, G313S, E381K <br /> and form stable complexes Hsp90 known substrates         transformed 1 mg genomic library (gift Y. <br /> (13, 15, 16, 29). Genetic screens enhancers suppressors       Kimura, Tokyo Metropolitan Institute Medical Science; <br /> of Hsp90 mutations provide tool identifying pro-         library constructed strain 5CG2 (27) pTV3 (38), <br /> teins function pathway. Indeed, ydj1,             high copy number plasmid marked TRP1). Transfor- <br /> member dnaJ family chaperones, identified         mants grew SD minus tryptophan lowest <br /> screen mutations exhibit synthetic lethality           temperature nonpermissive growth (36&#194;&#176;C for <br /> G170D. This finding turn led discovery Ydj1p          G170D E381K, 37&#194;&#176;C T101I G313S) selected <br /> plays important role maturation Hsp90 target            retested. To ensure suppression caused the <br /> proteins (30, 31).                                                   plasmid DNA, plasmids recovered, transformed the <br />    Here continued search Hsp90 target proteins        appropriate strain, tested growth nonpermissive <br /> cochaperones, searching suppressors yeast Hsp90 ts            temperature. <br /> point mutants. Because reduction Hsp90 function                The inserts suppressor plasmids, E20, E26 and <br /> affect activity Hsp90 target proteins, seemed <br />                                                                      E28 sequenced University Chicago Cancer <br /> likely overexpression single target only <br />                                                                      Research Center DNA Sequencing Facility using primers <br /> subtle effects cell growth. Furthermore, overexpression of <br />                                                                      specific vector sequences insert. <br /> the Hsp90 cochaperone STI1 suppresses ts growth phe- <br /> notype Hsp90 mutants, exacerbates growth          Sequences compared yeast genome by <br /> defects (32). Therefore, searched suppressors       using Saccharomyces Genome Database World Wide Web <br /> of phenotypically distinct mutants containing            site (http:yygenome-www.stanford.eduySaccharomycesy). All <br /> single amino acid substitution different region Hsp90        suppressor plasmids contained multiple ORFs. To iden- <br /> and performed screens lowest possible nonpermis-          tify specific genes responsible suppression, indi- <br /> sive temperature. This approach allowed identify         vidual ORFs suppressor subcloned pTV3 <br /> additional members Hsp90 chaperone network,              tested ability suppress growth defect of <br /> with unique properties.                                              E381K cells. The original suppressor plasmids E20, E26, and <br />                                                                      E28 used experiments described paper. <br />                                                                         Assessment p60v-src Activity Accumulation. The <br />              MATERIALS AND METHODS                                   galactose-inducible p60v-src expression plasmid Y316v-src (39) <br />    Plasmid Strain Construction. Integrating plasmids             transformed E381K cells containing plasmid <br /> containing wild-type mutant HSP82 sequences regulated          encoding wild-type Hsp82 (pTGpdyP82; ref. 10), empty <br /> the strong constitutive promoter GPD constructed             vector (pTV3), suppressor plasmids E20, <br /> subcloning ClaI-NotI fragments corresponding            E26, E28. Activity accumulation p60v-src assessed <br /> single-copy expression plasmids (10) pRS303 (34). These         described (10). <br />            Cell Biology: Nathan et al.                                                  Proc. Natl. Acad. Sci. USA 96 (1999)        1411 <br />  <br />                            RESULTS                                      6.3 kb chromosome 14 (105358&#226;&#8364;&#8220;111625). This region en- <br />                                                                         codes complete ORFs, POP3, YNL281w, ERG24. It <br />    Isolation Multicopy Suppressors Hsp90 ts Mutants.              contains 1,335 bp 1,511-bp ORF WSC2 and <br /> Hsp90 interacts target proteins context            710 bp 1,275-bp ORF YNL279w. <br /> dynamic, multicomponent chaperone complex (13&#226;&#8364;&#8220;16). Struc-                  The specific genes responsible suppression were <br /> turally, Hsp90 contains multiple domains presumably                identified subcloning individual ORFs suppres- <br /> interact different proteins different ways. Indeed,             sor (Fig. 1). A ScaI-KpnI fragment containing entire ORF <br /> Bohen Yamamoto (9) isolated mutation S.                    SSF1, 275 bases upstream sequences (including first <br /> cerevisae HSP82 gene, E431K, specifically affects interac-         103 bases YHRO65c) 16 bases downstream sequences <br /> tions single target protein, GR. Therefore,          contained suppressing function E20 plasmid. An <br /> searching suppressors general Hsp90 loss-of-function             EcoRI fragment containing entire CNS1 ORF, 696 bp of <br /> mutations (deletions promoter rearrangements), in-               59-untranslated sequences 175 bp 39-untranslated se- <br /> stead searched suppressors phenotypically distinct          quences possessed suppressing function E26 plas- <br /> ts mutants, containing single amino acid substitution         mid. An NcoI-MscI fragment containing entire YNL281w <br /> a different region Hsp90 (T101I, G170D, G313S,                   (HCH1&#226;&#8364;&#8220;High Copy Hsp90 suppressor) ORF 398 bp of <br /> E381K).                                                                 upstream sequences (including 207 bp YNL282w) <br />    Because mutations recessive (10), large           333 bp downstream sequences (including 166 bp <br /> number plasmids recovered suppressor screen           ERG24) contained suppressing activity plasmid E28. <br /> likely encode yeast Hsp90 proteins, Hsp82p           This analysis revealed multicopy suppressors of <br /> Hsc82p. To prevent this, used genomic library con-                 E381K SSF1, CNS1, HCH1. <br /> structed strain HSC82 disrupted               SSF1, CNS1, HCH1 Suppress ts Growth Phenotype <br /> HSP82 replaced construct containing HSP82               E381K Cells Different Extents. E381K cells severe <br /> coding sequences control galactose-inducible           growth defect. They grow half rate wild-type yeast at <br /> promoter GAL1 (27). Because HSP82 HSC82                      25&#194;&#176;C, slowly intermediate temperatures (34&#226;&#8364;&#8220; <br /> essential gene family yeast, strain viable         35&#194;&#176;C), 35&#194;&#176;C (10). The E381K mutation is <br /> grown galactose media. More importantly, wild-              recessive; introduction plasmid encoding wild-type HSP82 <br /> type Hsp90 genes present genomic library constructed             E381K cells restored normal growth rates temper- <br /> from strain control GAL1 promoter.            atures tested (25&#226;&#8364;&#8220;37&#194;&#176;C) (Fig. 2, data shown). A plasmid <br /> By performing screens glucose media, represses             encoding HCH1 restored wild-type growth rate to <br /> the GAL1 promoter, avoided recovery wild-type                 E381K cells 37&#194;&#176;C, highest temperature tested. <br /> Hsp90 genes present library.                                    Plasmids encoding SSF1 CNS1 increased maximal <br />    To ensure weak suppressors missed,              growth temperature E381K cells 36&#194;&#176;C resulted in <br /> looked transformants grow lowest non-             faster, wild-type, growth rates temperatures <br /> permissive temperature hsp82 point mutants.             25&#194;&#176;C 35&#194;&#176;C. Haploid E381K cells mating <br /> G170D E381K screened 36&#194;&#176;C, T101I                types responded equally suppressors. Diploid E381K <br /> G313S screened 37&#194;&#176;C. Approximately 5,000 transfor-              cells, however, did respond CNS1 HCH1 <br /> mants tested ability suppress        plasmids, growth completely unaffected the <br /> mutants. No suppressors T101I, G170D, G313S                  SSF1 plasmid (Table 1). <br /> isolated. We did, however, identify plasmids (E20, E26,              SSF1, CNS1, HCH1 Allele-Specific Suppressors. <br /> and E28) suppressed E381K.                                         Next, asked overexpression SSF1, CNS1, or <br />    Identification Suppressor Proteins. Sequence analysis             HCH1 suppress growth defects seven ts <br /> revealed E20 contained 3.4 kb chromosome 8 (228060&#226;&#8364;&#8220;             point mutants collection. We examined effect <br /> 231468) (Fig. 1). This region encodes complete ORF,            suppressors growth cells express high <br /> SSF1, contains 1,104 bp 1,631-bp ORF          levels wild-type HSP82 ts (strain iP82a) of <br /> RRP3 497 bp 842-bp ORF YHR067w. E26                cells express low levels wild-type HSP82 are <br /> contained 4.8 kb chromosome 2 (546677&#226;&#8364;&#8220;551480). This                  ts (strain ilep1a). Growth assessed permissive <br /> region encodes complete ORFs, RIB7, RPB5, CNS1.               (25&#194;&#176;C), semipermissive (34&#226;&#8364;&#8220;36&#194;&#176;C, depending mutant), <br /> It contains 528 bp 875-bp ORF SPP381           nonpermissive (36&#226;&#8364;&#8220;37&#194;&#176;C, depending mutant) tem- <br /> the 422 bp 2,096-bp ORF YBR156c. E28 contained             peratures. <br />                                                                            The growth iP82a cells affected the <br />                                                                         suppressors temperature tested (Table 2, data not <br />                                                                         shown). Introduction plasmid encoding SSF1 enhanced <br />                                                                         growth mutant, A41V, semipermissive tem- <br />                                                                         peratures, effect maximal growth tempera- <br />                                                                         ture. SSF1 effect growth of six <br />                                                                         point mutants collection, did affect growth of <br />                                                                         ilep1a cells. Overexpression CNS1 affected one <br />                                                                         mutant, T22I. The growth rate T22I cells in- <br />                                                                         creased 25&#194;&#176;C, remained unchanged higher tempera- <br />                                                                         tures. In marked contrast, overexpression HCH1 pro- <br />                                                                         effects growth of mutants our <br />                                                                         collection. It reduced growth rates mutants, A41V, <br />                                                                         G170D, G313S, A587T, ilep1a cells. It effect on <br />                                                                         mutants, G81S T101I, suppressed growth <br />                                                                         defect mutant, T22I. As E381K, overexpres- <br />                                                                         sion HCH1 completely suppressed growth defect T22I <br />    FIG. 1. Maps individual suppressor plasmids fragments     cells temperatures tested (Table 2, data shown). <br /> tested suppressor activity. Fragments (Top) E20, (Middle) E26,   Thus, suppressor exhibits distinct pattern allele <br /> and (Bottom) E28 provided suppressor activity shown.           specificity. <br />  1412     Cell Biology: Nathan et al.                                                     Proc. Natl. Acad. Sci. USA 96 (1999) <br />  <br />  <br />  <br />  <br />   FIG. 2. Effects SSF1, CNS1, HCH1 growth E381K cells. E381K cells containing vector vector expressing HSP82, <br /> SSF1, CNS1, HCH1 grown 25&#194;&#176;C (Right) 35&#194;&#176;C (Left) 3 days SD medium lacking tryptophan. <br />  <br />    Overexpression CNS1 HCH1 Increases Activity              activation p60v-src (10, 12). Introduction SSF1 plasmid <br /> a Heterologous Target Protein E381K Cells. The primary                E381K cells effect p60v-src activity or <br /> function Hsp90 normal cell growth appears to             accumulation. In marked contrast, overexpression either <br /> promote maintain activity highly selective group            CNS1 HCH1 increased p60v-src activity. In cases, the <br /> of target proteins (11). This function aided         increased activity accompanied increased accumulation <br /> other proteins form Hsp90 chaperone machine (13&#226;&#8364;&#8220;                p60v-src protein. The ability CNS1 HCH1 both <br /> 16). Therefore, likely suppressors                 promote maturation heterologous Hsp90 target pro- <br /> isolated (i) Hsp90 target proteins proteins          tein E381K cells enhance growth E381K cells in <br /> function biochemical pathway Hsp90 target              absence p60v-src indicates increase general Hsp90 <br /> protein (ii) proteins function modulate activity       function. This, turn, strongly suggests suppressor <br /> of dynamic Hsp90 chaperone complex. The biological                   proteins components Hsp90 chaperone <br /> activities affected Hsp90 mutations; <br />                                                                          machine modulators activity. <br /> the affect activity Hsp90 target <br /> proteins. Because functions Cns1p, Ssf1p, Hch1p are <br /> not completely understood, unable directly deter-                                       DISCUSSION <br /> mine activities affected Hsp90 mutations. <br />                                                                          The chaperone activities Hsp90 believed essential <br /> We could, however, determine Cns1p, Ssf1p, or <br />                                                                          maturation specific set critical target proteins. <br /> Hch1p affected capacity Hsp90 function proper <br /> maturation known target protein.                                    The identities target proteins, however, just <br />    To asked overexpression                 beginning elucidated. Moreover, means clear <br /> proteins enhanced maturation oncogenic tyrosine                  identified all, most, the <br /> kinase p60v-src. This target protein chosen                cochaperones modulators Hsp90 function. In at- <br /> reasons. First, maturation highly dependent Hsp90              tempt identify Hsp90 target proteins modulators of <br /> (12). Second, E381K cells severe defect p60v-src               Hsp90 function yeast, searched genes that, when <br /> maturation, 25&#194;&#176;C (10). Third, yeast cells low          overexpressed, suppress ts growth defect several <br /> levels endogenous tyrosine kinase activity, making p60v-src           distinct Hsp90 mutants. We identified multicopy sup- <br /> activity easy detect Western blot analysis                 pressors, SSF1, HCH1, CNS1. Each suppressor produced <br /> anti-phosphotyrosine antibody.                                           distinct pattern allele specificity different <br />    E381K cells containing vector,             character. Our data suggest SSF1 likely an <br /> three suppressors, wild-type HSP82 transformed              Hsp90 target protein function pathway an <br /> a GAL1-p60v-src expression plasmid. Tyrosine phosphorylation             Hsp90 target protein, CNS1 HCH1 likely are <br /> of total cellular proteins assessed 6 hours raffinose-         involved regulating promoting general Hsp90 chaperone <br /> grown cells shifted galactose-containing media. As               function. <br /> previously described (10, 12), cells expressing wild-type <br /> HSP82, p60v-src promiscuously phosphorylated broad array            Table 2. Allele-specific effects SSF1, CNS1, HCH1 the <br /> yeast proteins (Fig. 3). As expected, activity p60v-src       growth Hsp82 mutants <br /> severely reduced E381K cells containing vector.                  Mutant        SSF1        CNS1        HCH1         STI1 <br /> This reduction activity accompanied decrease in <br /> p60v-src accumulation (Fig. 3), consistent previous obser-               E381K          1           1            _            2 <br /> vations Hsp90 plays role stability                     iP82a          2           2            2 <br />                                                                               T22I           2           1            _           1 <br /> Table 1. Suppression haploid diploid E381K cells SSF1,              A41V           1           2            2           2 <br /> CNS1, HCH1                                                                G81S           2           2            2           2 <br />        Plasmid             Mata          Mata          Mataya                 T101I          2           2            2           2 <br />                                                                               G170D          2           2            2           1 <br />     Vector                 2             2               2                    G313S          2           2            2           1 <br />     HSP82 (Hsp90)         1111          1111           1111                   A587T          2           2            2           1 <br />     SSF1                   1             1               2                    ilep1a         2           2            2 <br />     CNS1                   1             1              1y2 <br />                                                                            _, Greatly enhanced growth; 1, somewhat enhanced growth; 2, no <br />     HCH1                  1111          1111            11 <br />                                                                          effect; 2, inhibited growth. <br />            Cell Biology: Nathan et al.                                                    Proc. Natl. Acad. Sci. USA 96 (1999)         1413 <br />  <br />                                                                           (TPR) domains. Several members Hsp90 heterocom- <br />                                                                           plex, including Sti1p immunophilins, contain TPR <br />                                                                           domains, domains required interactions <br />                                                                           Hsp90 (16, 43&#226;&#8364;&#8220;47). <br />                                                                              Overexpression CNS1 partially suppressed defect in <br />                                                                           p60v-src maturation E381K cells, suggesting Cns1p <br />                                                                           influences chaperone functions Hsp90. While this <br />                                                                           manuscript preparation, CNS1 reported a <br />                                                                           multicopy suppressor slow-growth phenotype cells <br />                                                                           lacking Hsp90 cochaperone CPR7 (48, 49). In addition to <br />                                                                           restoring rapid growth cpr7 mutant cells, overexpression of <br />                                                                           CNS1 rescued defect Hsp90 target protein maturation <br />                                                                           (49). Moreover, Cns1p protein complexes with <br />                                                                           Hsp90 Cpr7 (48, 49). Thus, likely that <br />                                                                           CNS1 member Hsp90 chaperone machinery. <br />                                                                              HCH1: A Modulator Hsp90 Function. HCH1 identi- <br />                                                                           fied 153-aa ORF (YNL281w) unknown function. It <br />                                                                           shares 37% sequence identity 55% sequence homology to <br />                                                                           N-terminal half 350-aa protein encoded un- <br />                                                                           characterized yeast ORF, YDR214w. Otherwise, HCH1 no <br />                                                                           striking homology known proteins. Disruption HCH1 <br />   FIG. 3. Effects SSF1, CNS1, HCH1 p60v-src activity         produced obvious growth defect, did exhibit an <br /> E381K cells. E381K cells expressing vector vector           obvious defect mating sporulation (B. Scherens, personal <br /> expressing HSP82, SSF1, CNS1, HCH1 transformed             communication). <br /> galactose-inducible p60v-src expression vector. Cells grown mid- <br />                                                                              HCH1 strongest suppressor identified screen. <br /> logarithmic phase 25&#194;&#176;C raffinose-containing medium shifted <br /> to galactose-containing medium 6 hours induce expression        It completely suppressed growth defect E381K and <br /> p60v-src. Total cellular proteins isolated, separated 7.5%        T22I cells. Interestingly, overexpression HCH1 exacerbated <br /> SDSyPAGE gels transferred Immobilon-P membranes. (A)               growth defect A41V, G170D, G313S, A587T cells, <br /> Equal sample loading confirmed Coomassie blue staining. (B)        effect G81S T101I cells. Such allele <br /> p60v-src activity assessed reacting blot antibody      specificity reminiscent of, identical to, that <br /> specific phosphotyrosine. (C) p60v-src accumulation assessed   observed Hsp90 cochaperone STI1 (32). Overexpres- <br /> reacting blot antibody specific p60v-src.                 sion STI1 enhances growth T22I, G170D, G313S, and <br />                                                                           A587T, reduces growth A41V T101I, no <br />    SSF1: A Potential Hsp90 Target Protein. SSF1 originally            effect G81S E381K. Mutations STI1 inhibit the <br /> isolated screen proteins that, overexpressed,               maturation diverse Hsp90 target proteins yeast (32), and <br /> increase mating efficiency ste4 ts mutant (40). A closely        Sti1p associated Hsp90 complexes yeast and <br /> related gene, SSF2, (the proteins share 94% amino acid                mammalian cells (13&#226;&#8364;&#8220;16, 32). Thus, STI1, type of <br /> identity), identified. Null mutations SSF1          allele-specificity consistent Hsp90 functioning sev- <br /> SSF2 detectable phenotype, mutations                  eral steps target protein maturation point <br /> genes lethal (40). Depletion Ssf gene products                mutations affecting different steps&#226;&#8364;&#8221;before, during, after <br /> growing cultures significantly impaired ability cells          interaction Sti1p. The likely true HCH1. <br /> to mate (40). Recently, determined mating                Indeed, overexpression HCH1 enhanced maturation of <br /> defect consequence failure carry morpho-             heterologous Hsp90 target protein p60v-src E381K cells. <br /> logical response pheromone (41). Thus, Ssf1p essential             Although investigation HCH1 relative <br /> functions mating vegetative growth.                       YDR214w required, data suggest Hch1p a <br />    Overexpression SSF1 did suppress defect                  modulator Hsp90 function itself chaperone can <br /> p60v-src maturation observed E381K cells, suggesting           affect target protein maturation. <br /> that Ssf1p does affect general chaperone functions                Our approach, employing point mutations different do- <br /> Hsp90. It possible, however, Ssf1p cochaperone          mains Hsp90 taking extreme care ensure even <br /> affects subset Hsp90 functions. Alternatively, Ssf1p,           subtle suppressors isolated, yielded promising <br /> or protein functions biochemical                 candidates proteins involved Hsp90 function. Moreover, <br /> pathway Ssf1p, Hsp90 target protein. This                 patterns allele-specific suppression uncovered this <br /> consistent observations (i) Ssf1p functions          study earlier work (32) STI1 suggest various <br /> response mating pheromones, (ii) suppressor effects             Hsp90 point mutations isolated provide valuable <br /> Ssf1p, Cns1p Hch1p, specific haploid cells,            tools dissecting specific steps complex, multicom- <br /> (iii) Hsp90 mutants defective pheromone signaling (28),            ponent Hsp90 chaperone machinery. Notably, screens <br /> and (iv) Ste11p recently identified Hsp90 target           saturating: point mutants used study, <br /> protein (ref. 28; B. Errede, personal communiation). Ste11p            suppressors them, E381K, isolated; <br /> a MEK kinase involved MAP kinase-signaling                     suppressor obtained once; finally, did not <br /> pathways yeast, including pheromone response pathway.              isolate STI1, known multicopy suppressor G170D <br /> Thus, overexpression Ssf1p suppress effects             mutation (32). It likely, therefore, screening <br /> E381K mutation effects MAP                  using Hsp90 point mutants, particularly mu- <br /> kinase pathways.                                                          tants distributed different surfaces domain, will <br />    CNS1: An Hsp90 Hetercomplex Member. CNS1 iden-                     provide powerful mechanism identifying additional <br /> tified essential ORF chromonsome 2 (42). Spores                  Hsp90 target proteins cofactors. Furthermore, recent <br /> containing null mutations CNS1 germinate undergo              work Drosophila demonstrates different genetic back- <br /> 2&#226;&#8364;&#8220;3 divisions. Interestingly, cells spores              grounds contain preexisting genetic variants sensitize <br /> elongated buds (42), morphology remarkably similar              different growth differentiation pathways Hsp90 defects <br /> to certain Hsp90 ts mutants (27). CNS1 codes                (50). Searching suppressors Hsp90 point mutations in <br /> 386-aa protein contains tetratricopeptide repeat               different genetic backgrounds S. cerevisiae provide a <br />  1414        Cell Biology: Nathan et al.                                                       Proc. Natl. Acad. Sci. USA 96 (1999) <br />  <br /> richer source target proteins (and proteins function             22.   Wiech, H., Buchner, J., Zimmermann, R. &amp; Jakob, U. (1992) <br /> target protein pathways) search individual strain.                  Nature (London) 358, 169&#226;&#8364;&#8220;170. <br />                                                                             23.   Yonehara, M., Minami, Y., Kawata, Y., Nagai, J. &amp; Yahara, I. <br />   We extremely grateful Yoko Kimura providing                      (1996) J. Biol. Chem. 271, 2641&#226;&#8364;&#8220;2645. <br /> plasmid expression library Jeanne Hirsch, Beverly Errede, Rick       24.   Freeman, B. C. &amp; Morimoto, R. I. (1996) EMBO J. 15, 2969&#226;&#8364;&#8220; <br /> Gaber, Didier Picard, Bart Scherens sharing unpublished data.             2979. <br /> We thank Julie Feder Mike Singer thoughtful              25.   Xu, Y., Singer, M. &amp; Lindquist, S. (1998) Proc. Natl. Acad. Sci. <br /> comments manuscript. D.F.N. supported National                      USA 96, 109&#226;&#8364;&#8220;114. <br /> Institutes Health Fellowship GM14745. This work supported         26.   Smith, D. F. (1993) Mol. Endocrinol. 7, 4&#226;&#8364;&#8220;11. <br /> the Howard Hughes Medical Institute National Institutes              27.   Kimura, Y., Matsumoto, S. &amp; Yahara, I. (1994) Mol. Gen. Genet. <br /> Health Grant GM25843.                                                             242, 517&#226;&#8364;&#8220;527. <br />                                                                             28.   Louvion, J.-F., Abbas-Terki, T. &amp; Picard, D. (1998) Mol. Biol. <br />  1.    Prodromou, C., Roe, S. M., O&#226;&#8364;&#8482;Brien, R., Ladbury, J. E., Piper,             Cell 9, 3071&#226;&#8364;&#8220;3083. <br />        P. W. &amp; Pearl, L. H. (1997) Cell 90, 65&#226;&#8364;&#8220;75.                          29.   Chang, H.-C. J. &amp; Lindquist, S. (1994) J. Biol. Chem. 269, <br />  2.    Prodromou, C., Roe, S. M., Piper, P. W. &amp; Pearl, L. H. (1997)              24983&#226;&#8364;&#8220;24988. <br />        Nat. Struct. Biol. 4, 477&#226;&#8364;&#8220;482.                                       30.   Caplan, A. J., Langley, E., Wilson, E. M. &amp; Vidal, J. (1995) J. Biol. <br />  3.    Stebbins, C. E., Russo, A. A., Schneider, C., Rosen, N., Hartl,            Chem. 270, 5251&#226;&#8364;&#8220;5257. <br />        F. U. &amp; Pavletich, N. P. (1997) Cell 89, 239&#226;&#8364;&#8220;250.                    31.   Kimura, Y., Yahara, I. &amp; Lindquist, S. (1995) Science 268, <br />  4.    Borkovich, K. A., Farrelly, F. W., Finkelstein, D. B., Taulien, J.         1362&#226;&#8364;&#8220;1365. <br />        &amp; Lindquist, S. (1989) Mol. Cell. Biol. 9, 3919&#226;&#8364;&#8220;3930.                32.   Chang, H.-C. J., Nathan, D. F. &amp; Lindquist, S. (1997) Mol. Cell. <br />  5.    Cutforth, T. &amp; Rubin, G. M (1994) Cell 77, 1027&#226;&#8364;&#8220;1036.                      Biol. 17, 318&#226;&#8364;&#8220;325. <br />  6.    Aligue, R., Akhavan-Niak, H. &amp; Russell, P. (1994) EMBO J. 15,        33.   Duina, A. A., Chang, H.-C. J., Marsh, J. A., Lindquist, S. &amp; Gaber <br />        6099&#226;&#8364;&#8220;6106.                                                                 R. F. (1996) Science 274, 1713&#226;&#8364;&#8220;1715. <br />  7.    Minami, Y., Kimura, Y., Kawasaki, H., Suzuki, K. &amp; Yahara, I.        34.   Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19&#226;&#8364;&#8220;27. <br />        (1994) Mol. Cell. Biol. 14, 1459&#226;&#8364;&#8220;1464.                               35.   Ito, H., Fukuda, Y., Murata, K. &amp; Kimura, A. (1983) J. Bacteriol. <br />  8.    Picard, D., Khursheed, B., Garabedian, M. J., Fortin, M. G.,               153, 163&#226;&#8364;&#8220;168. <br />        Lindquist, S. &amp; Yamamoto, K. R. (1990) Nature (London) 348,          36.   Mandel, M. &amp; Higa, A. (1970) J. Mol. Biol. 53, 159&#226;&#8364;&#8220;162. <br />        166&#226;&#8364;&#8220;168.                                                             37.   McCusker, J. H. &amp; Davis, R. W. (1991) Yeast 7, 607&#226;&#8364;&#8220;608. <br />  9.    Bohen, S. P. &amp; Yamamoto, K. R. (1993) Proc. Natl. Acad. Sci.         38.   Rose, M. D. &amp; Broach, J. R. (1991) Methods Enzymol. 194, <br />        USA 90, 11424&#226;&#8364;&#8220;11428.                                                       195&#226;&#8364;&#8220;230. <br /> 10.    Nathan, D. F. &amp; Lindquist, S. L. (1995) Mol. Cell. Biol. 15,         39.   Murphy, S. M., Bergman, M. &amp; Morgan, D. O. (1993) Mol. Cell. <br />        3917&#226;&#8364;&#8220;3925.                                                                 Biol. 13, 5290&#226;&#8364;&#8220;5300. <br /> 11.    Nathan, D. F., Vos, M. H. &amp; Lindquist, S. (1997) Proc. Natl. Acad.   40.   Yu, Y. &amp; Hirsch, J. P. (1995) DNA Cell Biol. 14, 411&#226;&#8364;&#8220;418. <br />        Sci. USA 94, 12949&#226;&#8364;&#8220;12956.                                            41.   Kim, J. &amp; Hirsch, J. P. (1998) Genetics 149, 795&#226;&#8364;&#8220;805. <br /> 12.    Xu, Y. &amp; Lindquist, S. (1993) Proc. Natl. Acad. Sci. USA 90,         42.   Baur, A., Schaaff-Gerstenschlager, I., Boles, E., Miosga, T., <br />        7074&#226;&#8364;&#8220;7078.                                                                 Rose, M. &amp; Zimmermann, F. K. (1993) Yeast 9, 289&#226;&#8364;&#8220;293. <br /> 13.    Nair, S. C., Toran, <br /> </body></html>