<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />     Mcm10 Is Required for the Maintenance of Transcriptional Silencing in <br />                           Saccharomyces cerevisiae <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Mcm10 is an essential protein that participates in both the initiation and the elongation of DNA <br />              replication. In this study we demonstrate a role for Mcm10 in the maintenance of heterochromatic <br />              silencing at telomeres and HM loci of budding yeast. Two mcm10 mutants drastically reduce silencing of <br />              both URA3 and ADE2 reporter genes integrated into these silent loci. When exposed <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> a-factor, mcm10 <br />              mutant cells display a &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo-cluster&#226;&#8364;&#8482;&#226;&#8364;&#8482; <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> associated with a defect in the maintenance of silencing. <br />              In addition, when combined with a defect in the establishment of silent chromatin, mcm10 mutants dem- <br />              onstrate a synergistic defect in HML silencing. Consistent with a direct silencing function, Mcm10p shows <br />              a two-hybrid interaction with Sir2p and Sir3p that is destroyed by the mcm10-1 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>mutation</span> and dependent on <br />              the C-terminal 108 amino acids. Tethering GBD-MCM10 to a defective HMR-E silencer is not sufficient to <br />              restore silencing. Furthermore, mutations in MCM10 inhibit the ability of GBD-SIR3 to restore silencing <br />              when tethered to a defective HMR-E. Suppressor mutations in MCM2, which suppress the temperature <br />              sensitivity of mcm10-1, fail to overcome the mcm10-1 silencing defect, suggesting that MCM10&#226;&#8364;&#8482;s role in <br />              transcriptional silencing may be separate <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> its essential functions in DNA replication. <br />  <br />  <br />  <br />  <br /> T     RANSCRIPTIONAL silencing in the budding yeast <br />       Saccharomyces cerevisiae renders large regions of the <br /> genome inactive in a fashion similar to that in hetero- <br />                                                                            bound to telomeric DNA (Aparicio et al. 1991; Kyrion <br />                                                                            et al. 1993; Moretti et al. 1994; Rusche et al. 2003). <br />                                                                            Silencing is thought to consist of two phases, establish- <br /> chromatic regions in higher eukaryotes (Grewal and                         ment and maintenance (Miller and Nasmyth 1984; <br /> Elgin 2002). Transcriptional silencing occurs at yeast                     Pillus and Rine 1989). During the establishment phase, <br /> telomeres and at silent mating-type (HM) loci, HML                         silent chromatin is formed on nascent DNA and in the <br /> and HMR (reviewed in Rusche et al. 2003), which house                      maintenance phase it has to be sustained until the next <br /> silent endogenous copies of the mating-type genes                          establishment event. Mutations in genes such as SIR1 <br /> (reviewed in Haber 1998). The HM loci are flanked by                       and RAP1 can lead to defects in the establishment but <br /> cis-acting regulatory elements called silencers (Feldman                   not maintenance of silent chromatin (Pillus and Rine <br /> et al. 1984; Brand et al. 1985, 1987). These elements                      1989; Mahoney et al. 1991; Sussel and Shore 1991), <br /> consist of A, E, and B sites that are bound by the origin                  and mutations in genes such as CAC1 and SIR3 can cause <br /> recognition complex (ORC) (Bell et al. 1993; Micklem                       maintenance but not establishment defects (Enomoto <br /> et al. 1993; Loo et al. 1995), Rap1 (Shore and Nasmyth                     and Berman 1998; Enomoto et al. 2000). <br /> 1987), and Abf1 (Diffley and Stillman 1988), respec-                          A link between transcriptional silencing and DNA <br /> tively. Once bound, these factors recruit the silent infor-                replication has been suggested by the observation that <br /> mation regulator (Sir) proteins, Sir1&#226;&#8364;&#8220;4. Although Orc1                     passage through S phase is required for the establish- <br /> directly interacts with Sir1 and Rap1 with Sir4 to es-                     ment of transcriptional silencing (Miller and Nasmyth <br /> tablish a silencing complex that includes Sir2 and Sir3,                   1984). Another connection between silencing and DNA <br /> it is unclear if each of these proteins can be recruited                   replication is evidenced by the fact that silencers can be <br /> to the silencer independently of each other. A stepwise                    used to initiate replication of plasmids (Brand et al. 1987) <br /> spreading of the Sir2/3/4 complex from the silencer                        and that some origins have silencer activity (Grunweller <br /> leads to the silencing of nearby genes (Hecht et al. 1996;                 and Ehrenhofer-Murray 2002). However, other stud- <br /> Hoppe et al. 2002; Rusche et al. 2002). Telomeric si-                      ies have demonstrated that the S-phase requirement <br /> lencing is similar to silencing at HM loci, with the excep-                does not involve DNA replication (Kirchmaier and <br /> tion that the Sir2/3/4 complex is recruited by Rap1                        Rine 2001; Li et al. 2001). In S. cerevisiae, DNA replica- <br />                                                                            tion is a tightly regulated process initiated in early S <br />   1 <br />                                                                            phase at specific loci called origins of replication or au- <br />    Corresponding author: Department of Molecular Biology and Genetics, <br /> 325 Biotechnology Bldg., Cornell University, Ithaca, NY 14853.             tonomously replicating sequences (ARSs) (reviewed in <br /> E-mail: bt16@cornell.edu                                                   Dutta and Bell 1997). Budding yeast have  400 such <br />  <br /> Genetics 171: 503&#226;&#8364;&#8220;515 (October 2005) <br />  504                                              I. Liachko and B. K. Tye <br />  <br /> replication origins, but most have not been shown to           strate a novel function for MCM10 in the maintenance <br /> play a role in silencing. Although the connection be-          of silent chromatin that may be separate from its essen- <br /> tween DNA replication and transcriptional silencing is         tial role in DNA replication. <br /> not completely understood, several DNA replication <br /> proteins have been implicated in silencing. These in- <br />                                                                               MATERIALS AND METHODS <br /> clude S. cerevisiae ORC (Bell et al. 1993; Micklem et al. <br /> 1993), PCNA, RF-C, Cdc45 (Ehrenhofer-Murray et al.                Strains and plasmids: Strains used in this study are listed in <br /> 1999; Zhang et al. 2000), and Mcm5 (Dziak et al. 2003).        Table 1. All strains were isogenic derivatives of W303-1A, unless <br /> Similarly, DNA replication factors have also been im-          otherwise indicated. All procedures were performed accord- <br />                                                                ing to standard yeast methodology (Sherman 1991). Strains <br /> plicated in the organization of chromatin structure            carrying silencing reporters were made by crossing strain <br /> in higher eukaryotes such as the fruit fly Drosophila          YB541 or YB697 to the appropriate mutant strain and selecting <br /> melanogaster (Pflumm and Botchan 2001; Christensen             desired segregants by their conditional phenotypes and/or <br /> and Tye 2003) and Schizosaccharomyces pombe (Bailis et al.     auxotrophy. Genotypes were confirmed by PCR or by plas- <br /> 2003). One of these proteins, Mcm10, physically interacts      mid complementation where applicable. Plasmids used in this <br />                                                                study are listed in Table 2. Plasmids used for the expression of <br /> with the heterochromatin factor HP1 (Christensen and           tethering fusions were constructed by the Gateway system <br /> Tye 2003; Kellum 2003) in D. melanogaster, indicating a        (Invitrogen, San Diego). To create pGBKgw, the kanr gene of <br /> possible direct role in chromatin structure.                   plasmid pGBKT7 (CLONTECH, Palo Alto, CA) was replaced <br />    MCM10 was originally identified in yeast screens            with an ampr gene and a Gateway (Invitrogen) cassette was <br /> that searched for mutants in DNA replication (Dumas            inserted into the multiple-cloning sequence. Gateway cassettes <br />                                                                were ligated into plasmids pBTM116 and pGAD2F, creating <br /> et al. 1982; Solomon et al. 1992) and minichromosome           pBTMgw and pGADgw, respectively. Gateway cassette B was <br /> maintenance (Maine et al. 1984). Mcm10 is an essential         ligated into the SmaI site of pGBT9.C (Bartel et al. 1996), <br /> chromatin-associated protein (Homesley et al. 2000; Ricke      creating plasmid pGBT9gw. pDONR201 entry clones contain- <br /> and Bielinsky 2004; Yang et al. 2005). Temperature-            ing MCM10 and SIR1 ready for N-terminal fusions were con- <br /> sensitive mcm10 mutants show sluggish passage through          structed according to Invitrogen instructions and sequenced. <br />                                                                LR recombination reactions (Invitrogen) were set up between <br /> S phase (Kawasaki et al. 2000), an inability to complete S     pGBT9gw, pGADgw, pBTMgw, or pGBKgw and each of the <br /> phase (Merchant et al. 1997), and a replication fork pause     aforementioned entry clones. All yeast transformations were <br /> phenotype at inactive origins of replication (Merchant         carried out using standard lithium acetate protocols. <br /> et al. 1997). MCM10 has also been implicated in the               Silencing assays: Strains bearing the URA3 reporter were <br /> initiation of DNA replication (Homesley et al. 2000;           grown overnight in appropriate dropout media. Tenfold serial <br />                                                                dilutions were set up in sterile 96-well plates and  25 ml of <br /> Wohlschlegel et al. 2002; Sawyer et al. 2004). While its       each dilution was spotted onto the appropriate plate using a <br /> exact roles in DNA replication are not yet completely un-      multipipettor. 5-Fluoroorotic acid (5-FOA) was used at a con- <br /> derstood, Mcm10 shows physical and genetic interactions        centration of 1 mg/ml unless otherwise stated. In cases where <br /> with subunits of ORC (Merchant et al. 1997; Homesley           the ADE2 reporter gene was used, cultures were diluted to <br /> et al. 2000; Kawasaki et al. 2000; Douglas 2003), which has     300 cells/dropout plate. Colonies were grown for 3 days at <br />                                                                30&#194;&#176; and incubated for 3 days at 4&#194;&#176; for color development be- <br /> been implicated in heterochromatin-induced silencing.          fore being counted. <br /> Intriguingly, there are estimated to be  40,000 copies of         a-Factor confrontation (shmooing) and mating assays: For <br /> Mcm10p per cell (Kawasaki et al. 2000), far exceeding in       the shmooing assay, 50 ml of a saturated MATa culture was <br /> abundance the  400 DNA replication origins in budding          streaked along the middle of a YPD plate and incubated at 30&#194;&#176; <br /> yeast. Although the reason for such abundance is not clear,    for 4&#226;&#8364;&#8220;6 hr for the a-factor to diffuse into the media. A small <br />                                                                sample of MATa culture being tested was then spotted near the <br /> it seems likely that Mcm10 may have other, previously          edge of the plate and a dissecting microscope needle was used <br /> unidentified functions.                                        to line up individual unbudded MATa cells approximately one- <br />    In this study we demonstrate that Mcm10 plays a role        half of an optical field (at 3160 magnification) away from the <br /> in heterochromatin silencing in S. cerevisiae. Two mutant      MATa streak. Cells were incubated at 30&#194;&#176; for 16&#226;&#8364;&#8220;18 hr and the <br /> alleles of MCM10 manifest significant defects in silenc-       resulting phenotypes of MATa cells were scored. For patch- <br />                                                                mating assays, the strains being tested were streaked in a par- <br /> ing at both the telomeric and HMR loci. Consistent with        allel <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> on YPD plates, grown overnight at 30&#194;&#176;, and then <br /> this observation, when mcm10 mutant cells are exposed          replica plated onto tester strains 6697-1 or 6697-3, of opposite <br /> to a-factor, they show a &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo-cluster&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype, sig-     mating type on a minimal media plate. Plates were then incu- <br /> nifying a defect in the maintenance of silent chromatin        bated for 2 days at 30&#194;&#176;. <br /> at the HML locus (Enomoto and Berman 1998). We                    Protein-protein interactions: The pGADgw SIR and RAP1 <br />                                                                two-hybrid plasmids were constructed using the Invitrogen <br /> also found two-hybrid interactions between Mcm10               Gateway cloning system. pGAD2F and pBTM116 plasmids <br /> and components of the silencing machinery and that             (Fields and Song 1989) were converted into destination <br /> these interactions are disrupted by the Mcm10-1 and            vectors by insertion of the Gateway recombination cassette <br /> Mcm10-43 mutant proteins. Interestingly, the temperature-      into the BamHI site. The full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> pBTM.MCM10 was de- <br /> sensitive phenotype of mcm10 mutant cells can be genet-        scribed in Merchant et al. (1997). pGAD2F and pBTM116 <br />                                                                constructs were transformed into the two-hybrid strain EGY40 <br /> ically separated from the silencing defect by second-site      carrying the pSH18-34 reporter plasmid (Fields and Song <br /> mutations that suppress the temperature sensitivity but        1989). Interactions were assessed by the appearance of blue <br /> not the silencing defect. Together our findings demon-         colonies on plates containing X-gal (Sigma, St. Louis). Ten <br />                                 Mcm10 in Gene Silencing                                        505 <br />  <br />                                         TABLE 1 <br />                                 Strains used in this study <br />  <br /> Strains                                   Genotype                               Source <br />                                     Isogenic to W303 <br /> W303-1A           MATa ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3-1 <br /> W303-1B           MATa ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3-1 <br /> BTY100            W303 MATa mcm10-1                                           This lab <br /> BTY101            W303 MATa mcm10-1                                           This lab <br /> BTY103            W303 MATa mcm10-43                                          This lab <br /> BTY102            W303 MATa mcm10-43                                          This lab <br /> YB541             MATa adh4TURA3 Tel (VII-L)                                  B. Stillman <br /> YB697             MATa hmrTADE2                                               B. Stillman <br /> ILY171            W303 MATa hmrTADE2 adh4TURA3 Tel (VII-L)                    This study <br /> ILY180            ILY171 mcm10-1                                              This study <br /> ILY270            ILY171 mcm10-43                                             This study <br /> GCY23             W303 MATa RDN1TMET15 DAhmrTTRP1                             D. Shore <br />                     adh4TURA3 Tel (VII-L) sir2D met15 ADE2 <br /> ILY163            W303 MATa adh4TADE2 Tel (VII-L)                             This study <br /> ILY165            ILY163 mcm10-1                                              This study <br /> ILY167            ILY163 mcm10-43                                             This study <br /> YJB1838           W303 MATa cac1TLEU2                                         J. Berman <br /> YJB1940           W303 MATa sir1THIS3                                         J. Berman <br /> ILY153            W303 MATa sir1THIS3                                         This study <br /> ILY154            W303 MATa cac1TLEU2                                         This study <br /> ILY155            YJB1940 mcm10-1                                             This study <br /> ILY156            ILY153 mcm10-1                                              This study <br /> ILY157            YJB1940 mcm10-43                                            This study <br /> ILY158            ILY153 mcm10-43                                             This study <br /> ILY159            YJB1838 mcm10-1                                             This study <br /> ILY160            ILY154 mcm10-1                                              This study <br /> ILY161            YJB1838 mcm10-43                                            This study <br /> ILY162            ILY154 mcm10-43                                             This study <br /> ILY155            YJB1940 mcm10-1                                             This study <br /> ILY157            YJB1940 mcm10-43                                            This study <br /> ILY159            YJB1838 mcm10-1                                             This study <br /> ILY161            YJB1838 mcm10-43                                            This study <br /> ILY212            W303 MATa sir1THIS3 cac1TLEU2                               This study <br /> ILY235            YB541 sir1THIS3                                             This study <br /> ILY237            YB541 cac1TLEU2                                             This study <br /> ILY239            ILY235 mcm10-1                                              This study <br /> ILY241            ILY237 mcm10-1                                              This study <br /> ILY220            ILY171 Mcm2-S619Y                                           This study <br /> ILY221            ILY171 Mcm2-S619F                                           This study <br /> ILY222            ILY171 Mcm2-S619Y mcm10-1                                   This study <br /> ILY223            ILY171 Mcm2-S619F mcm10-1                                   This study <br /> YEA80             MATa AebTUASg hmrTURA3                                      R. Sternglanz <br /> YEA82             MATa aeBTUASg hmrTURA3                                      R. Sternglanz <br /> ILY137            YEA80 MATa mcm10-1                                          This study <br /> ILY139            YEA80 MATa mcm10-43                                         This study <br />  <br />                                   Other backgrounds <br /> EGY40[pSH18-34]   MATa ura3-52 trp1-1 leu2-3,112 [pSH18-34]                   E. Golemis <br /> ILY267            EGY40[pSH18-34] sir4THIS3                                   This study <br /> ILY268            EGY40[pSH18-34] sir3THIS3                                   This study <br /> ILY269            EGY40[pSH18-34] sir2THIS3                                   This study <br />                     MATa ura3-52 his3-D200 leu2-3,112 trp1-901 gal4D gal80D <br /> PJ69-4a           LYS2TGAL1-HIS3 GAL2-ADE2 met2TGAL7-LacZ                     P. James <br /> 6697-1            MATa met4                                                   This lab <br /> 6697-3            MATa met4                                                   This lab <br />  506                                                    I. Liachko and B. K. Tye <br />  <br />                                                                 TABLE 2 <br />                                                       Plasmids used in this study <br />  <br />          Plasmid name                                          Description                                Source <br />          pRS315                                  YCP LEU2                                          New England Biolabs <br />          pRS315MCM10                             YCP LEU2 MCM10                                    This lab <br />          pGAD2F                                  2m LEU2 GAD4-AD                                   S. Fields <br />          pBTM116                                 2m TRP1 LEXA-DBD                                  S. Fields <br />          pSH18-34                                URA3 LacZ with LEXA binding sites                 S. Fields <br />          pGADgw                                  pGAD2F with Gateway cassette                      This study <br />          pBTMgw                                  pBTM116 with Gateway cassette                     This study <br />          pGBKgw                                  pGBKT7 with Gateway cassette ampr                 This study <br />          pBTMMCM10                               pBTMgw MCM10                                      This study <br />          pBTMmcm10-1                             pBTMgw mcm10-1                                    This study <br />          pBTMmcm10-43                            pBTMgw mcm10-43                                   This study <br />          pBTMMCM10-N128                          pBTMgw MCM10 (128-571)                            This study <br />          pBTMMCM10-RIGHT                         pBTMgw MCM10 (292-571)                            This study <br />          pBTMMCM10-C108                          pBTMgw MCM10 (1-463)                              This study <br />          pBTMMCM10-LEFT                          pBTMgw MCM10 (1-291)                              This study <br />          pBTMMCM10-MIN                           pBTMgw MCM10 (128-463)                            This study <br />          pBTMSIR2                                pBTMgw SIR2                                       This study <br />          pBTMSIR3                                pBTMgw SIR3                                       This study <br />          pBTMSIR4                                pBTMgw SIR4                                       This study <br />          pGADSIR1                                pGADgw SIR1                                       This study <br />          pGADSIR2                                pGADgw SIR2                                       This study <br />          pGADSIR3                                pGADgw SIR3                                       This study <br />          pGADSIR4                                pGADgw SIR4                                       This study <br />          pGADRAP1                                pGADgw RAP1                                       This study <br />          pGADMCM10                               pGADgw MCM10                                      This study <br />          pGADmcm10-1                             pGADgw mcm10-1                                    This study <br />          pGBKMCM10                               pGBKgw MCM10                                      This study <br />          pGBKmcm10-1                             pGBKgw mcm10-1                                    This study <br />          pGBT9.C                                 YCP GBD cloning vector TRP1                       P. Bartel <br />          pGBT9gw                                 pGBT9.C with Gateway cassette                     This study <br />          pGBT9MCM10                              pGBDgw MCM10                                      This study <br />          pGBT9SIR1                               pGBDgw SIR1                                       This study <br />          pM165                                   GBD-Sir1p HIS3 2mm                                D. Shore <br />          pM2046                                  GBD-Sir2p HIS3 RAP1prom                           D. Shore <br />          pM1588                                  GBD-Sir4p HIS3 RAP1prom                           D. Shore <br />  <br />  <br />  <br /> microliters of a saturated culture started from 10 to 20 colo-         containing recessive, temperature-sensitive mcm10 mu- <br /> nies per transformation was spotted onto X-gal plates for              tations (mcm10-1 or mcm10-43) as well as reporter genes <br /> photographs. <br />                                                                        (URA3 and ADE2) integrated at the telomeric and HMR <br />    Microcolony growth rate assay: Individual, unbudded YB541 <br /> cells from an overnight culture were lined up in a row parallel to     silent loci in the yeast genome. In the URA3 constructs, <br /> a row of ILY100 cells, using a dissecting microscope. Cells were       silencing is assayed by measuring the sensitivity of cells <br /> incubated on media plates as indicated and pictures were taken         to 5-FOA, a toxic precursor of uracil that kills cells ex- <br /> at 3160 magnification.                                                 pressing the URA3 gene product. If URA3 is silenced, <br />    mcm10-1 suppressor screen: One-hundred-microliter sam- <br />                                                                        more cells will survive on 5-FOA and if silencing is <br /> ples of overnight cultures of a mcm10-1 strain were plated on <br /> YPD plates and incubated for 2 days at 37&#194;&#176;. Resulting colonies         defective, more cells become inviable. In strains con- <br /> were screened for recessive cold sensitivity at 14&#194;&#176;. Tetrad anal-      taining a telVIILTURA3 reporter, mcm10 mutant cells <br /> ysis was used to identify single-gene second-site suppressors.         showed a sensitivity to 5-FOA that is approximately 4 <br /> The mutations were cloned by transforming a YCp50 yeast                orders of magnitude greater than that observed in wild- <br /> genomic library (Rose et al. 1987) and screening for comple- <br />                                                                        type cells (Figure 1A). In addition, a larger proportion <br /> mentation of the cold-sensitive defect followed by sequencing. <br />                                                                        of mcm10 cells with the telomeric URA3 reporter were <br />                                                                        able to form colonies on media lacking uracil than that <br />                            RESULTS <br />                                                                        observed in MCM10 cells with the same reporter (data <br />   mcm10 mutations affect silencing at telomeric and                    not shown). Derepression of the URA3 reporter in these <br /> HMR loci: To investigate the effects of mcm10 mutants                  experiments indicates that the mcm10 mutants are <br /> on silencing, we constructed a series of isogenic strains              defective in telomeric silencing. To ensure that the <br />                                                      Mcm10 in Gene Silencing                                                      507 <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;MCM10 is required for proper telomeric and HMR silencing. (A) Telomeric silencing assays with mcm10 mutants. Strains <br /> ILY171, ILY180, ILY270, and GCY23, transformed with either pRS315 (top three rows and bottom row) or pRS315MCM10 (rows <br /> labeled &#226;&#8364;&#732;&#226;&#8364;&#732;1MCM10&#226;&#8364;&#8482;&#226;&#8364;&#8482;) are shown. Tenfold serial dilutions of overnight cultures were plated on CM-LEU media with or without the in- <br /> dicated concentration of 5-FOA. Plates were grown for 2 days at 30&#194;&#176;. (B) Silencing assays of mcm10 mutants using the ADE2 reporter <br /> gene. Cultures were grown overnight in liquid selective media, diluted, and plated on CM-URA plates at  300 cells/plate. Plates were <br /> grown for 3 days at 30&#194;&#176; and then incubated at 4&#194;&#176; for 3 days before colonies were scored for color. Strains ILY171, ILY180, and ILY270 <br /> were used for the left and ILY163, ILY165, and ILY167 for the right. All strains were transformed with pRS315 (&#195;&#191;) or pRS315MCM10 <br /> (1). All data points are the averages of at least two trials. Error bars that appear to be missing are too small to be displayed. <br />  <br />  <br /> silencing defect of the mcm10 mutants is not promoter                (Figure 1, A and B). Together these findings show that <br /> specific, we also assayed telomeric silencing by utilizing           the silencing defects caused by both mcm10-1 and <br /> a telVIILTADE2 reporter (Figure 1B, left). If the ADE2               mcm10-43 mutations are recessive and independent of <br /> gene is silenced, the resulting yeast cells will form pink           reporter genes used and that they have a global effect on <br /> colonies, whereas loss of ADE2 silencing will give rise to           telomeric and HMR loci alike. <br /> white colonies. In wild-type cells, the vast majority of the            Mcm10p plays a role in the maintenance but not the <br /> colonies were either pink or white/pink sectored, but                establishment of silent chromatin at HML: To address <br /> mcm10 colonies were exclusively white. The results from              the role of MCM10 in HML silencing, we asked whether <br /> this experiment showed that mcm10 mutations engen-                   MCM10 is required for proper silencing of the endog- <br /> der a dramatic derepression of ADE2 at the telomere.                 enous MATa1 and MATa2 genes present at the HML <br />    To investigate whether the silencing defect of the                locus. A wild-type MATa cell responds to a pheromone <br /> mcm10 mutants also applies to other silent loci, we inte-            (a-factor) produced by MATa cells by arresting and <br /> grated ADE2 into the HMR locus. Results from this ex-                forming a projection called a &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo.&#226;&#8364;&#8482;&#226;&#8364;&#8482; A MATa cell <br /> periment were similar to those observed at the telomeric             where the HMLa is derepressed expresses information <br /> locus; mcm10 strains with a hmrTADE2 reporter gave                   of both mating types and is no longer responsive to <br /> rise to almost exclusively white colonies, whereas wild-             a-factor (Pillus and Rine 1989). Therefore, it is possible <br /> type strains with the same reporter were predominantly               to measure the silencing of HMLa in individual MATa <br /> pink (Figure 1B, right). These results indicate that the             cells by assaying their response to a-factor. Further- <br /> silencing defect of mcm10 mutants affects both HMR                   more, by comparing how mcm10 mutant cells respond to <br /> and telomeric loci.                                                  a-factor to known silencing mutants it is possible to <br />    To investigate whether the silencing defects caused               determine whether mcm10 mutants are defective in the <br /> by these mcm10 mutations are dominant or recessive,                  establishment or in the maintenance of HML silencing. <br /> single-copy ARS/CEN plasmids with or without a wild-                    A sir1 mutant, which has a defect in establishment, <br /> type copy of MCM10 were transformed into each of the                 but not in maintenance of the silent state (Pillus and <br /> reporter strains. The presence of a MCM10-expressing                 Rine 1989; Rusche et al. 2002), formed shmoos and <br /> plasmid restored silencing in all mcm10 mutant strains               colonies representing the silenced and derepressed <br /> to wild-type levels whereas empty vectors alone did not              populations of cells, respectively (Figure 2A). In our <br />  508                                               I. Liachko and B. K. Tye <br />  <br />  <br />  <br />  <br />                                                                                         Figure 2.&#226;&#8364;&#8221;Mcm10 is involved in the <br />                                                                                      maintenance of HML silencing. (A) <br />                                                                                      Shmoo phenotype analysis of mcm10-1. <br />                                                                                      Individual, unbudded cells from strains <br />                                                                                      W303-1A, YJB1940, YJB1838, BTY100, <br />                                                                                      ILY155, ILY159, and ILY212 were ex- <br />                                                                                      posed to a-factor overnight on YPD <br />                                                                                      plates before their shmooing pheno- <br />                                                                                      types were scored. Approximately 150 <br />                                                                                      cells of each strain were analyzed in three <br />                                                                                      independent trials. (B) Patch-mating <br />                                                                                      assays of various mcm10 and silenc- <br />                                                                                      ing mutants. Cells of the indicated rele- <br />                                                                                      vant genotype were mated to tester <br />                                                                                      strains. MATa strains were mated to tes- <br />                                                                                      ter strain 6697-1 (a-lawn) and MATa <br />                                                                                      strains were mated to 6697-3 (a-lawn). <br />                                                                                      Mated cells were allowed to grow on <br />                                                                                      SD plates at 30&#194;&#176; for 2 days. (C) mcm10 <br />                                                                                      mutants are defective in the mainte- <br />                                                                                      nance/inheritance of telomeric silenc- <br />                                                                                      ing. The rates of colony size increase <br />                                                                                      of mutant (top) and wild-type (bottom) <br />                                                                                      strains were compared by lining up indi- <br />                                                                                      vidual unbudded cells on synthetic com- <br />                                                                                      plete media with or without 5-FOA. <br />                                                                                      BTY100 (top, left), W303-1A (bottom, <br />                                                                                      left), ILY180 (top, right four panels), <br />                                                                                      and YB541 (bottom, right four panels) <br />                                                                                      are shown. The three right panels show <br />                                                                                      pictures of the same two colonies at dif- <br />                                                                                      ferent time intervals. At least 30 such <br />                                                                                      cells were analyzed. Plates were incu- <br />                                                                                      bated at 30&#194;&#176;. <br />  <br />  <br />  <br />  <br /> study,  30% of the sir1D cells formed colonies, consis-         the cells that form shmoos are able to stably maintain <br /> tent with previous results (Pillus and Rine 1989). Cac1         their silent state for a prolonged period of time, whereas <br /> is the largest subunit of the chromatin assembly factor 1       cells that form shmoo clusters are not. Both mcm10 <br /> (CAF-1) complex that deposits histones H3/H4 onto               mutants (only mcm10-1 data shown) displayed a pheno- <br /> newly replicated DNA (Verreault et al. 1996). A cac1D           type similar to that of the cac1D strain but not to that of <br /> strain, which is defective in the maintenance of the si-        the sir1D strain (Figure 2A). Since mcm10 cells did not <br /> lent state, but not in its establishment (Enomoto et al.        form colonies, it is likely that the establishment pathway <br /> 1997; Enomoto and Berman 1998) did not form any                 in these mutants is still intact, since all cells are able to <br /> colonies, but formed an increased number of shmoo               respond to a-factor. These results suggest that Mcm10p <br /> clusters (Figure 2A). This phenotype is indicative of           plays a role in maintaining the HML silent chromatin, <br /> cells that are able to respond to a-factor, but are not able    rather than establishing it. <br /> to maintain the arrested phenotype for an extended                To confirm our previous results, we tested the re- <br /> period of time and divide before arresting and becom-           sponse of double mutants to a-factor. sir1D mcm10 cells <br /> ing shmoos again. Both shmoos and shmoo clusters                showed a striking ablation of silencing, just like a sir1D <br /> represent cells that can respond to a-factor, indicating a      cac1D strain (Figure 2A). In these strains, cells were <br /> stable establishment of silencing. The difference is that       almost completely unresponsive to a-factor, with  85% <br />                                                  Mcm10 in Gene Silencing                                              509 <br />  <br /> of cells becoming colonies, whereas only  5% of cells            To investigate whether mcm10 mutant cells properly <br /> were able to form shmoos and  10% formed shmoo                propagate the telomeric silent state, we placed individ- <br /> clusters. This observation indicates a much greater           ual wild-type and mutant unbudded cells containing the <br /> derepression of the HMLa locus in the double mutants          telVIILTURA3 reporter on media containing 5-FOA <br /> than in either of the single mutants. In strains bearing      and compared growth rates by colony size (Figure 2C). <br /> both cac1D and mcm10 mutations there was a 10&#226;&#8364;&#8220;20%             MCM10 ura3 cells bearing the telomeric URA3 reporter <br /> increase in the number of shmoo clusters formed               proliferated at a rate comparable to that of isogenic cells <br /> (Figure 2A), signifying a slightly greater defect in the      without the reporter whether on 5-FOA or on regular <br /> maintenance of the silent state than in either of the two     complete media. Similarly, mcm10-1 ura3 cells grew into <br /> mutants alone. The absence of colonies formed by the          colonies as fast as MCM10 ura3 cells on 5-FOA. In <br /> mcm10 cac1D double mutant supports the notion that            contrast, although mcm10-1 cells bearing the URA3 re- <br /> both proteins play a role in maintenance, but not             porter proliferated at a rate comparable to that of their <br /> establishment of silencing at HML.                            wild-type counterparts on media without 5-FOA, 5-FOA <br />    Another way to measure HML silencing is to measure         significantly retarded their proliferation. Similar results <br /> the ability of MATa cells bearing appropriate mutations       were observed for mcm10-43 cells (not shown). These <br /> to mate with MATa cells in patch-mating assays (Figure        results suggest that mcm10 mutants are defective in the <br /> 2B). If the HMLa region is derepressed in MATa cells,         maintenance and/or inheritance of silent telomeric <br /> the cells will not mate efficiently. This experiment can      chromatin. <br /> also be done in reverse to measure silencing of the HMRa         Mcm10 interacts with Sir2 and Sir3: If Mcm10 plays a <br /> locus by assaying the ability of MATa cells to mate. In       direct role in silencing, it should interact with compo- <br /> our studies none of the single mutants of MCM10, SIR1,        nents of the silencing machinery. To test this possibility, <br /> or CAC1 showed mating defects for either mating type,         we used yeast two-hybrid assays to investigate whether <br /> whereas the double mutants did. sir1D mcm10 double            Mcm10 interacts with Sir1, Sir2, Sir3, Sir4, or Rap1. The <br /> mutants showed significant MATa-specific mating de-           combinations of BTM-MCM10 and GAD-SIR2 or GAD- <br /> fects (Figure 2B), consistent with an abolishment of          SIR3 showed activation of the LacZ reporter indicated <br /> HMLa silencing observed in Figure 2A. cac1D mcm10             by the blue color of the colonies formed on media <br /> double mutants showed a less dramatic MATa-specific           containing X-gal (Figure 3A), signifying Mcm10 inter- <br /> mating defect. Similar effects have been observed be-         actions with Sir2 and Sir3. These interactions are <br /> fore with other maintenance mutants (Enomoto et al.           disrupted by two single-amino-acid substitution muta- <br /> 2000). Together these findings are consistent with a role     tions, mcm10-1 (P269L) and mcm10-43 (C320Y). Western <br /> for Mcm10 in the maintenance of silencing.                    blot analysis of BTM-mcm10-1 and BTM-mcm10-43 in- <br />    mcm10 mutants have a defect in the maintenance             dicated that these fusion proteins are stably expressed <br /> and/or inheritance of telomeric silencing: Pillus and         (Figure 3B). These results were largely confirmed by <br /> Rine (1989) showed that HM chromatin in sir1 mutant           switching the bait/prey orientation in the two-hybrid <br /> cells can stably exist in one of two epigenetic states&#226;&#8364;&#8221;-      experiment (Figure 3C). The lack of interaction be- <br /> silent and active. These cells are able to propagate this     tween BTM-SIR3 and GAD-MCM10 is probably due to <br /> state to their progeny for many generations. To stably        interference of the BTM tag with Sir3 function as pre- <br /> propagate silencing to the next generation, a mother          viously observed (A. Lustig, personal communication). <br /> cell must also maintain the silent chromatin until it is      None of the other plasmid combinations showed an <br /> inherited by the daughter cell. Inheritance of the silent     increase in LacZ activity (Figure 3A). <br /> state can be assayed by measuring how often a silenced           We also set up two-hybrid experiments in another <br /> mother cell gives rise to a derepressed daughter cell         system, using GBK-tagged proteins as bait and a HIS3 <br /> (Monson et al. 1997). In the case where a telomeric           gene as reporter (Figure 3D). In this system, GBK- <br /> URA3 silencing reporter is used, when a cell divides, if      MCM10 showed interactions with GAD-SIR2 and GAD- <br /> the silencing is not properly inherited by the daughter       SIR3 while GBK-mcm10-1 disrupted these interactions, <br /> cell, then it will be Ura1 and will die in the presence of    further confirming results observed in Figure 3A. <br /> 5-FOA. Thus, a telVIILTURA3 cell with a defect in the            To identify the Sir2- and Sir3-interacting domain of <br /> inheritance or maintenance of silencing will lead to a        Mcm10, we carried out two-hybrid analysis with trunca- <br /> slower growing colony, since a proportion of its progeny      tions of Mcm10 (Figure 3E). As shown above, full-length <br /> will be dying on 5-FOA. By observing the rate of colony       Mcm10 showed strong interaction with Sir2 and signif- <br /> growth on 5-FOA it is possible to assay maintenance/          icant interaction with Sir3. Interactions with Sir2 and <br /> inheritance of telomeric silencing in cells bearing a         Sir3 were uncompromised by the N-terminal deletion of <br /> URA3 reporter. However, this assay is not able to ade-        128 amino acids from Mcm10 (128-571) and somewhat <br /> quately distinguish between defects in the inheritance        compromised by deletion from Mcm10 (292-571). In <br /> and the maintenance of silent chromatin, since the si-        contrast, C-terminal deletion of 108 amino acids from <br /> lencing may be lost due to a maintenance defect before        Mcm10 (1-463) completely destroyed interactions with <br /> the inheritance step takes place.                             both Sir2 and Sir3. Western blot analysis indicated that <br />  510                                                 I. Liachko and B. K. Tye <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;The C terminus of Mcm10 interacts with silencing proteins. Yeast two-hybrid bait/prey constructs were transformed <br /> into reporter strains EGY40[pSH18-34] (A, B, D, and E), or PJ69-4a (C). (A, B, D, and E) Cultures were spotted onto selective X-gal <br /> plates. An interaction is indicated by blue color resulting from activation of the LacZ reporter. (B) Cultures of EGY40[pSH18-34] <br /> expressing BTM-Mcm10 fusions were grown overnight in CM-TRP media. Cells from 1-ml aliquots of these cultures were boiled <br /> and run on Western blots according to standard protocols. The blot was probed with rabbitt anti-LexA antibody (a gift from Jeff <br /> Roberts), stripped, and reprobed with rabbit anti-actin antibody (a gift from Tim Huffaker). (C) Saturated cultures were serially <br /> diluted and spotted on selective plates. An interaction is indicated by growth on CM-HIS media caused by activation of the HIS3 <br /> reporter. (E) Two-hybrid experiments were performed in Dsir2, Dsir3, or Dsir4 deletion strains ILY267, ILY268, and ILY269. <br />  <br /> all of these truncation fusions are stably expressed                mediated by Sir3 or vice versa in their interactions with <br /> (Figure 3B). These results indicate that the interacting            Mcm10, we carried out the two-hybrid analysis in strains <br /> domain of Mcm10 with Sir2 and Sir3 is located at the                deleted in SIR2, SIR3, or SIR4 (Figure 3F). We found <br /> C-terminal region. We speculate that disruption of this             that Sir2 and Sir3 interacted with Mcm10 independent <br /> interaction by mcm10-1(P269L) may be the result of an               of any of the Sir proteins. <br /> overall structural change in the Mcm10 protein that                   Tethering Mcm10p to a defective HMR-E silencer is <br /> compromises the C-terminal interacting domain.                      not sufficient to restore silencing. Directly recruiting <br />    The finding that both Sir2 and Sir3 interact with                one of the Sir proteins or Rap1 to a defective silencer <br /> Mcm10 through the C-terminal domain of Mcm10                        can restore silencing (Chien et al. 1993; Buck and <br /> could be the result of indirect interaction of one Sir              Shore 1995; Lustig et al. 1996; Marcand et al. 1996). <br /> protein mediated by another. To investigate if Sir2 is              To investigate whether tethering Mcm10 is sufficient <br />                                                  Mcm10 in Gene Silencing                                                511 <br />  <br />                                                                                            Figure 4.&#226;&#8364;&#8221;Tethering Mcm10 <br />                                                                                         to a defective silencer is not suf- <br />                                                                                         ficient to restore silencing. (A) <br />                                                                                         GBD-Mcm10 is not able to restore <br />                                                                                         silencing at HMR-E (depicted in <br />                                                                                         schematic). Strains YEA80 (top) <br />                                                                                         or YEA82 (bottom) were trans- <br />                                                                                         formed with pGBT9, pGBT9SIR1, <br />                                                                                         or pGBT9MCM10. Tenfold serial <br />                                                                                         dilutions were plated on the indi- <br />                                                                                         cated media and grown at 30&#194;&#176; for <br />                                                                                         2 days. (B) GBD-MCM10 construct <br />                                                                                         is able to restore both the temper- <br />                                                                                         ature sensitivity and the silencing <br />                                                                                         defect of mcm10 strains. Strains <br />                                                                                         YB541, ILY180, and ILY102 were <br />                                                                                         transformed with either pGBT9 <br />                                                                                         or pGBT9MCM10 (two bottom <br />                                                                                         rows). Tenfold serial dilutions <br />                                                                                         were plated on selective media <br />                                                                                         with or without 5-FOA at 30&#194;&#176; (per- <br />                                                                                         missive temperature) or on com- <br />                                                                                         plete media at 37&#194;&#176; (restrictive <br />                                                                                         temperature). <br />  <br />  <br /> to restore silencing, we used strains in which the si-        complement this defect by adding a plasmid containing <br /> lencer was disrupted and replaced by Gal4 binding sites       a single copy of MCM10 (Figure 5B). Interestingly, <br /> (Andrulis et al. 2002). The mating-type gene cassette at      mutations in MCM10 did not have an effect on targeted <br /> the HMR was replaced with URA3, providing a way to            silencing by GBD-SIR1, GBD-SIR2, or GBD-SIR4, all of <br /> measure silencing at this locus. We made a plasmid            which would require the participation of Sir3 for silenc- <br /> construct expressing a GBD-MCM10 fusion. This hybrid          ing. Together these findings suggest that the silencing <br /> gene is unable to restore silencing to an HMR-E, where        function of MCM10 may lie downstream of the Sir <br /> either the Rap1 and Abf1 binding sites (Aeb, Figure 4A,       proteins, with Mcm10 regulating the silencing activity <br /> top) or the ORC and Rap1 binding sites (aeB, Figure 4A,       of Sir3. <br /> bottom) are deleted, while expression of GBD-SIR1 was            The temperature sensitivity and silencing defects of <br /> able to restore silencing in both cases (Figure 4A). Since    mcm10-1 can be genetically separated: Since Mcm10 <br /> it is possible that the fusion protein is unstable or oth-    plays a role in DNA replication, it is possible that the <br /> erwise defective, we checked its ability to complement        silencing defect caused by mcm10 mutations is a by- <br /> mcm10 mutations. GBD-MCM10 was able to complement             product of a replication defect or of an overall defective <br /> both the temperature sensitivity and the silencing de-        S phase. To address this point we asked if the temper- <br /> fect of mcm10-1 and mcm10-43 (Figure 4B), suggesting          ature sensitivity (which is attributed to a defect in <br /> that the hybrid protein retains its silencing function.       replication) and silencing defects of mcm10-1 could be <br /> Taken together, these data show that recruiting Mcm10         separated. In our studies of Mcm10 function we con- <br /> is not sufficient to restore silencing to a defective         ducted a screen for second-site suppressors of the <br /> silencer.                                                     temperature-sensitive phenotype of mcm10-1. We screened <br />    Mutations in MCM10 inhibit the ability of GBD-SIR3         for spontaneous reversion of the mcm10-1 temperature <br /> to restore silencing: Normally, tethering any of the Sirs     sensitivity that also conferred a second phenotype (cold <br /> to a defective silencer is sufficient to restore silencing    sensitivity) so that the mutation could be easily identi- <br /> (Chien et al. 1993; Lustig et al. 1996; Marcand et al.        fied. We isolated two mutants, MCM2-S619Y and MCM2- <br /> 1996). If Mcm10 functions downstream of the Sirs, then        S619F. These suppressors are capable of overcoming <br /> a mcm10 mutation may inhibit the ability of GBD-SIR           the temperature sensitivity of mcm10-1 (Figure 6D) in a <br /> constructs to restore silencing to a defective HMR-E. We      dominant fashion (data not shown). We constructed <br /> constructed tethering yeast strains containing either         strains that contain the HMR and telomeric silencing <br /> mcm10-1 or mcm10-43 mutations as well as the defective        reporters, mcm10-1, and/or the suppressor mutations to <br /> silencer with GBD binding sites and HMR marked with           assay whether these mutations can also suppress the <br /> URA3. As expected, tethering any of the Sirs in a MCM10       telomeric silencing defect conferred by mcm10-1. The <br /> strain restored silencing to approximately wild-type          mutations Mcm2-S619Y or Mcm2-S619F did not confer a <br /> levels (Figure 5A, top). On the other hand, targeted          silencing defect on their own (Figure 6, A and B). While <br /> silencing by GBD-SIR3 was defective in both mcm10             these mcm2 alleles were able to suppress the temperature- <br /> mutant strains (Figure 5A, bottom). We were able to           sensitive phenotype caused by the mcm10-1 mutation, <br />  512                                                   I. Liachko and B. K. Tye <br />  <br />                                                                     Recently, involvement of silencing proteins such as Sir2 <br />                                                                     and Sir3 in the regulation of replication initiation has <br />                                                                     also been reported (Pappas et al. 2004). The relation- <br />                                                                     ship between these two multistep processes and their <br />                                                                     participating components remains an enigma. Mcm10 <br />                                                                     is a DNA replication factor involved in both the initi- <br />                                                                     ation and elongation steps of DNA replication (Homesley <br />                                                                     et al. 2000; Wohlschlegel et al. 2002; Lee et al. 2003; Fien <br />                                                                     et al. 2004; Sawyer et al. 2004). Although Mcm10 has <br />                                                                     previously been shown to interact with DNA replication <br />                                                                     proteins implicated in silencing (Homesley et al. 2000; <br />                                                                     Kawasaki et al. 2000), a direct silencing function has never <br />                                                                     been described. The results presented in this study suggest <br />                                                                     that Mcm10 interacts with Sir2 and Sir3 in the mainte- <br />                                                                     nance of transcriptional silencing and that this function <br />                                                                     appears to be separable from its functions essential for <br />                                                                     DNA replication. <br />                                                                        We have demonstrated that two recessive alleles of <br />                                                                     MCM10 were able to derepress URA3 and ADE2 reporter <br />                                                                     genes at telomeric and HMR loci (Figure 1). In addition, <br />                                                                     these mutations were able to derepress endogenous <br />                                                                     genes at HML (Figure 2, A and B). Thus, Mcm10 has a <br />                                                                     rather ubiquitous role in silencing, in contrast to ORC, <br />                                                                     which plays a role only in HM silencing. Although si- <br />                                                                     lencing in S. cerevisiae also occurs in ribosomal DNA <br />                                                                     loci, the effects of Mcm10 on rDNA silencing were not <br />                                                                     examined in this study. Using reporter genes at HMR, <br />                                                                     mcm10 mutants clearly showed derepression of HMR <br />                                                                     (Figure 1B). However, mating assays in the mcm10 sir1D <br />                                                                     double mutants showed a MATa-specific mating defect <br />                                                                     supporting the notion that these mutants derepress <br />                                                                     HML more efficiently than HMR (Figure 2B). This phe- <br />                                                                     nomenon has been observed before with other mainte- <br />                                                                     nance mutants (Enomoto et al. 2000) and can most <br />    Figure 5.&#226;&#8364;&#8221;Mutations in MCM10 inhibit the ability of GBD-         likely be attributed to the fact that silencing is stronger <br /> Sir3 to restore silencing when tethered to a defective silencer.    at the HMR than at the HML. <br /> (A) Tethering assays in the presence of mcm10 mutations.               The possibility that Mcm10 may have a separate <br /> Strains YEA80 (WT), ILY137 (mcm10-1), and ILY139 (mcm10-            silencing function is supported by the observation that <br /> 43) were transformed with pM165, pM2046, pM1587, and <br /> pM1588, respectively. Tenfold dilutions of overnight cultures <br />                                                                     dominant extragenic suppressors (MCM2-S619Y and <br /> were spotted on appropriate dropout media with or without           MCM2-S619F ) of the temperature sensitivity of mcm10-1 <br /> 5-FOA. The strain background (Strain) and tethering plasmid         are not able to suppress the silencing defect (Figure 6). <br /> used (Teth.) are indicated. All strains were also transformed       In fact, it is possible that Mcm10 may execute its si- <br /> with empty vector pRS315 (GBD) as controls. The strains that        lencing function at the replication fork and yet its <br /> show a silencing defect are indicated by an asterisk. (B) MCM10 <br /> restores silencing to strains from A by transforming these <br />                                                                     silencing function could be uncoupled from it replica- <br /> strains with pRS315MCM10. Silencing was assayed as in A.            tion activity. However, we cannot rule out the unlikely <br />                                                                     possibility that these two apparently independent phe- <br /> they failed to restore proper telomeric (Figure 6, A and            notypes are a manifestation of the same defect at dif- <br /> B) and HMR (data not shown) silencing to a mcm10-1                  ferent levels of severity. Future experiments with the aid <br /> strain. These results suggest that the temperature-                 of separation-of-function mutations in MCM10 should <br /> sensitive phenotype of mcm10-1 is not linked to the                 clarify this point. <br /> silencing defect.                                                      Tethering silencing proteins to a defective silencer <br />                                                                     can restore silencing. However, tethering GBD-Mcm10 <br />                                                                     to a defective silencer could not restore silencing <br />                         DISCUSSION <br />                                                                     (Figure 5). This finding suggests that Mcm10 functions <br />   Involvement of numerous DNA replication proteins                  downstream of the establishment of silent chromatin in <br /> such as the ORC, PCNA, RF-C, Cdc45, and Mcm5 in tran-               the propagation or maintenance of silent chromatin. <br /> scriptional silencing has been implicated in S. cerevisiae.         This hypothesis is supported by epistasis analysis with <br />                                                   Mcm10 in Gene Silencing                                                        513 <br />  <br />                                                                                           Figure 6.&#226;&#8364;&#8221;The temperature sensitiv- <br />                                                                                        ity of mcm10-1 can be separated from its <br />                                                                                        silencing defect. MCM2-S619Y and MCM2- <br />                                                                                        S619F suppress the temperature sensitivity <br />                                                                                        of mcm10-1, but not the silencing defect. <br />                                                                                        Tenfold serial dilutions of overnight cul- <br />                                                                                        tures of strains ILY171, ILY180, ILY220, <br />                                                                                        ILY221, ILY222, and ILY223 were plated <br />                                                                                        on complete media plates at 37&#194;&#176; (D) or <br />                                                                                        30&#194;&#176; (A&#226;&#8364;&#8220;C). A and B show growth on <br />                                                                                        synthetic complete media containing <br />                                                                                        0.5 mg/ml (low FOA) or 1 mg/ml (high <br />                                                                                        FOA). Plates were grown for  2 days. <br />  <br />  <br />  <br /> sir1 and cac1 in the shmoo assay and the reporter              the interaction of Mcm10 with both Sir2 and Sir3. <br /> silencing assays (Figure 2, A and B). MCM10 is in the          Deposition of Mcm10 on propagating heterochromatin <br /> same epistasis group as CAC1, which encodes a subunit          may be mediated by the elongation machinery (Figure <br /> of the histone chaperone CAF-1 known to be involved in         7A) or drawn directly from soluble Mcm10 (Figure 7, A <br /> the maintenance of heterochromatin. It is in a different       and B). However, if the function of Mcm10 is to simply <br /> epistasis group from SIR1, whose role in silencing is          stabilize established silent chromatin, it should not mat- <br /> restricted to establishment. Previous tethering experi-        ter which GBD-SIR fusion is used to initiate silencing in <br /> ments with Sir proteins showed that recruiting any of          our tethering experiments. Our data suggest that in the <br /> the four Sirs to a defective silencer can restore silencing    absence of wild-type Mcm10, GBD-Sir3 has difficulty <br /> to wild-type levels (Chien et al. 1993; Marcand et al.         nucleating the spread of silent chromatin. The fact that <br /> 1996; Triolo and Sternglanz 1996). When we con-                only GBD-Sir3-mediated silencing is affected in mcm10 <br /> ducted similar experiments in mcm10 strains, we noted          strains may indicate that this defect occurs during the <br /> that GBD-Sir3 constructs lose their ability to properly        steps when GBD-Sir3 first binds DNA and initiates <br /> restore silencing (Figure 5). This result suggests that        silencing before the other Sirs are recruited. Perhaps <br /> Mcm10 may modulate Sir3 activity and is consistent with        mcm10 mutations are inhibiting the first step in this <br /> the two-hybrid interaction between Mcm10 and Sir3              artificial establishment of silencing such that GBD-Sir3 <br /> (Figure 3). However, although Mcm10 also interacts             has trouble recruiting the other Sirs to begin the assem- <br /> with Sir2, silencing by GBD-Sir2 is not inhibited. One         bly and spreading of Sir complexes from the silencer <br /> explanation is that the GBD fusion may be altering the         (Figure 7C). In other words, GBD-Sir3 alone may not <br /> structure of Sir3 in such a way to make it hypersensitive      completely fulfill its proper function in mcm10 strains <br /> to mutations in MCM10. Another explanation is that             and that transition of establishment to maintenance <br /> Sir2 and Mcm10 interact indirectly, through Sir3 or            requires the participation of the other Sirs if Mcm10 <br /> another yet unidentified factor. However, deletion of          is compromised. Furthering this idea (Figure 7C), we <br /> SIR3 had no effect on the two-hybrid interaction of Sir2       believe that Mcm10 cooperates with Sir3 during the <br /> and Mcm10, suggesting that Sir3 cannot be the medi-            normal transition of nucleation to spreading of hetero- <br /> ator. Taken together, these data indicate that the nature      chromatin at the silent mating-type and telomeric loci <br /> of the Mcm10-Sir2 interaction may be different from            (Figures 1 and 2) independent of GBD-Sir3. This ex- <br /> that of the Mcm10-Sir3 interaction. Sir3 has been shown        planation is consistent with the established model of <br /> to have homology to Orc1, a member of the origin               a cooperative mechanism of silencing between the Sir <br /> recognition complex (Zhang et al. 2002). Mcm10 also            proteins. This explanation is also consistent with the <br /> interacts with members of the ORC including Orc1               hypothesized role of Mcm10 at replication origins as a <br /> (Kawasaki et al. 2000), Orc2 (Kawasaki et al. 2000;            stabilizing scaffold in the transition of the pre-replication <br /> Christensen and Tye 2003), Orc3, Orc5, and Orc6                complex to the pre-initiation complex (IC) and subse- <br /> (Douglas 2003). It is possible that functional and/or          quently from the pre-IC to the elongation complex <br /> structural similarities between silencing and replication      (Homesley et al. 2000; Wohlschlegel et al. 2002; Lee <br /> proteins may mediate the Mcm10-Sir3 interaction.               et al. 2003; Fien et al. 2004; Sawyer et al. 2004). By study- <br />    The fact that Mcm10&#226;&#8364;&#8482;s function lies in the mainte-          ing the roles of Mcm10 in DNA replication and in gene <br /> nance of silent chromatin and yet tethering Mcm10              silencing we may begin to elucidate the connection be- <br /> does not restore silencing suggests that Mcm10 may act         tween DNA replication and heterochromatin assembly. <br /> to stabilize silent chromatin structures, such as Sir2-4 <br /> complexes, after they have been recruited. Stabilization         We thank Nancy Douglas and Erik Andrulis for many of the strains, <br />                                                                plasmids, and reagents used in this study and for their discussions <br /> may be accomplished by Mcm10 binding to assembled              and critical reading of this manuscript. We also thank Bruce Stillman, <br /> Sir2-4 complexes on chromatin (Figure 7A) or escorting         Rolf Sternglanz, Erica Golemis, Judith Berman, David Shore, Ravi <br /> soluble Sir2-4 to chromatin (Figure 7B), consistent with       Ram, Jeff Roberts, and Tim Huffaker for providing strains, plasmids, <br />  514                                                       I. Liachko and B. K. Tye <br />  <br />  <br />  <br />  <br />                                                                                                           Figure 7.&#226;&#8364;&#8221;Hypotheses for <br />                                                                                                        Mcm10 function in maintenance <br />                                                                                                        of silencing. (A) Mcm10 associ- <br />                                                                                                        ates with the chromatin-bound <br />                                                                                                        Sir2-4 complex to stabilize silent <br />                                                                                                        chromatin. Mutant Mcm10 pro- <br />                                                                                                        teins are not able to maintain <br />                                                                                                        interaction with the Sirs. Dis- <br />                                                                                                        sociation of Sirs from chromatin <br />                                                                                                        results in derepression of silent <br />                                                                                                        loci. (B) Mcm10p acts to recruit <br />                                                                                                        Sir proteins to silent chromatin <br />                                                                                                        where needed. In the case of <br />                                                                                                        mcm10 mutants, inability of <br />                                                                                                        Mcm10-1p to facilitate the bind- <br />                                                                                                        ing of a sufficient number of Sir <br />                                                                                                        complexes to the silent region <br />                                                                                                        leads to the degradation of silent <br />                                                                                                        chromatin over time. (C) Mcm10 <br />                                                                                                        facilitates the transition from the <br />                                                                                                        nucleation of Sir2-4 complexes <br />                                                                                                        at the silencer to the spreading <br />                                                                                                        of Sir2-4 complexes along the <br />                                                                                                        silenced region. A defective <br />                                                                                                        Mcm10 does not allow the proper <br />                                                                                                        spreading of silent chromatin. <br />  <br />  <br />  <br />  <br /> and antibodies.  <br /> </body></html>