<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOBNAME: RNA 12#7 2006 PAGE: 1 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />     Use targetrons disrupt essential nonessential <br />     genes Staphylococcus aureus reveals temperature <br />     sensitivity Ll.LtrB group II intron splicing <br />  <br />  <br />     JUN YAO,1 JIN ZHONG,1 YUAN FANG,2 EDWARD GEISINGER,2 RICHARD P. NOVICK,2 <br />     ALAN M. LAMBOWITZ1 <br />     1 <br />      Institute Cellular Molecular Biology, Department Chemistry Biochemistry, Section Molecular Genetics <br />     Microbiology, School Biological Sciences, The University Texas Austin, Austin, Texas 78712, USA <br />     2 <br />      Skirball Institute Biomolecular Medicine, Program Molecular Pathogenesis Departments Microbiology Medicine, <br />     New York University Medical Center, New York, New York 10016, USA <br />  <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />     We targetron based Lactococcus lactis Ll.LtrB group II intron used efficient chromosomal gene <br />     disruption human pathogen Staphylococcus aureus. Targetrons expressed derivatives vector pCN37, uses <br />     cadmium-inducible promoter, pCN39, derivative pCN37 temperature-sensitive replicon, gave site-specific <br />     disruptants hsa seb genes 37%&#226;&#8364;&#8220;100% plated colonies selection. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> disrupt hsa, essential gene, we <br />     used group II intron integrates sense orientation relative target gene transcription removed by <br />     RNA splicing, enabling production functional HSa protein. We splicing Ll.LtrB intron the <br />     intron-encoded protein temperature-sensitive, method yields conditional hsa disruptant grows 32&#194;&#176;C not <br />     43&#194;&#176;C. The temperature sensitivity splicing reaction suggests general means obtaining one-step conditional disruptions <br />     organism. In nature, temperature sensitivity group II intron splicing limit temperature range organism <br />     containing group II intron inserted essential gene. <br />     Keywords: functional genomics; gene targeting; retrotransposon; reverse transcriptase; ribozyme <br />  <br />  <br />  <br />  <br />     INTRODUCTION                                                                      tion. Unfortunately, gene targeting S. aureus been <br />                                                                                       hampered low efficiency homologous <br />     Staphylococcus aureus facultative pathogen, responsible <br />                                                                                       recombination. Beyond traditional strategies, as <br />     infections suffered hospital patients and <br />                                                                                       chemical transposon mutagenesis, recently developed <br />     increasingly implicated community infections <br />                                                                                       methods inactivating genes S. aureus use antisense <br />     (Fridkin et al. 2005). Its danger pathogen reflects that <br />                                                                                       RNAs (Kernodle et al. 1997) counterselection rare <br />     S. aureus adapt widely varying conditions, including <br />                                                                                       recombination events, e.g., using dominant strepto- <br />     hostile environment infected animal tissues, can <br />                                                                                       mycin-sensitivity allele (Chen Novick 2006) an <br />     develop resistance antibiotics. At present, <br />                                                                                       antisense RNA blocking expression essential gene, <br />     urgent need understand pathobiology of <br />                                                                                       secA (Bae Schneewind 2006). All methods have <br />     organism develop new anti-bacterial agents to <br />                                                                                       advantages disadvantages. Transposon mutagenesis is <br />     counter it. A major approach objectives to <br />                                                                                       fairly easy, transposons hot spots useless <br />     identify targetable metabolic processes evaluating the <br />                                                                                       essential genes. Counterselection strategies require labo- <br />     function specific genes targeted gene inactiva- <br />                                                                                       rious gene-specific constructions and, again, useless for <br />                                                                                       essential genes. Antisense RNAs target gene have <br />                                                                                       significant failure rate and, repress not <br />        Reprint requests to: Alan M. Lambowitz, Institute Cellular and <br />     Molecular Biology, Department Chemistry Biochemistry, Section           eliminate gene function, reveal essentiality. <br />     Molecular Genetics Microbiology, The University Texas Austin,              A different strategy overcomes these <br />     Austin, TX 78712, USA; e-mail: lambowitz@mail.utexas.edu; fax: (512)              disadvantages gene targeting mobile group II <br />     232-3420. <br />        Article published online ahead print. Article publication date   introns (&#226;&#8364;&#732;&#226;&#8364;&#732;targetrons&#226;&#8364;&#8482;&#226;&#8364;&#8482;) (for review, Lambowitz et al. <br />     http://www.rnajournal.org/cgi/doi/10.1261/rna.68706.                              2005). A targetron based Lactococcus lactis Ll.LtrB <br />  <br />     RNA (2006), 12:1271&#226;&#8364;&#8220;1281. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2006 RNA Society.                                 1271 <br />  <br />  <br />  <br />  <br />   rna687 Yao et al. ARTICLE RA <br />  JOBNAME: RNA 12#7 2006 PAGE: 2 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />    intron used targeted gene disruption site-       algorithm generally identifies multiple rank-ordered target <br />    specific DNA insertion Escherichia coli Gram-        sites gene. The required modifications intro- <br />    negative enterobacteria (Karberg et al. 2001; Perutka et al.      duced donor plasmid PCR (Guo et al. 2000; <br />    2004), Gram-positive bacteria L. lactis     Karberg et al. 2001). In E. coli, targetrons commonly insert <br />    Clostridium perfringens (Frazier et al. 2003; Chen et al.         frequencies 1%&#226;&#8364;&#8220;80%, insertions detected <br />    2005). The utility group II introns gene targeting         colony PCR incorporating genetic marker <br />    derives unique DNA integration mechanism            intron (Zhong et al. 2003; Perutka et al. 2004). <br />    intron RNA uses ribozyme activity insert         Thus, targetrons potentially used disrupt gene <br />    (reverse splice) directly DNA target site      sequence available. <br />    reverse-transcribed intron-encoded protein (IEP)              In addition targeted gene disruption, targetrons have <br />    (for review, Lambowitz Zimmerly 2004). The                used insert (&#226;&#8364;&#732;&#226;&#8364;&#732;knock in&#226;&#8364;&#8482;&#226;&#8364;&#8482;) cargo genes site-specifically <br />    mobility reactions carried ribonucleoprotein         desired chromosomal locations (Frazier et al. 2003) to <br />    (RNP) particle formed RNA splicing             introduce targeted double-strand breaks be <br />    contains IEP excised intron lariat RNA. RNPs              repaired homologous recombination cotrans- <br />    initiate mobility using IEP <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span>-pairing          formed DNA fragment, enabling introduction point <br />    intron RNA recognize DNA target sequences. The          mutations (Karberg et al. 2001). Additionally, targetron <br />    base-pairing interactions span 13- 16-nt region           randomized target-site recognition sequences was <br />    DNA target site, involve intron RNA sequences             used generate library random disruptants, analogous <br />    denoted exon-binding sites 1 2 (EBS1 EBS2) d          global transposon mutagenesis (Zhong et al. 2003). <br />    (a sequence adjacent EBS1) DNA target site                 Individual targetrons re-isolated library <br />    sequences denoted intron-binding sites 1 2 (IBS1          DNA PCR used modification obtain <br />    IBS2) 59-exon, d9 39-exon (see below)           desired single disruptants (Yao et al. 2005). <br />    (Eskes et al. 1997; Guo et al. 1997, 2000; Mohr et al. 2000).        In principle, targetrons used obtain either <br />    Because base-pairing interactions contribute        unconditional conditional disruptions, including condi- <br />    DNA target specificity, feature makes possible     tional disruptions essential genes. This the <br />    retarget group II introns insert desired sites simply     targetron integrate orientation <br />    modifying EBS1, EBS2, d sequences               relative target gene transcription selecting target <br />    intron RNA (Lambowitz et al. 2005).                               sequences sense antisense strands. A targetron <br />       In bacteria, targetrons generally expressed           inserted antisense orientation transcribed the <br />    donor plasmid, RNPs assembled intracellularly           complement intron RNA sequence, spliced, <br />    (Guo et al. 2000; Karberg et al. 2001). The commonly used         yields unconditional disruption. In contrast, tar- <br />    E. coli donor plasmid pACD3 specifies precursor RNA             getron inserted sense orientation transcribed into <br />    containing 0.9-kb Ll.LtrB-DORF intron short flank-          functional intron RNA potentially spliced <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />    ing exon sequences, IEP (denoted LtrA protein)           precursor RNA IEP, restoring mRNA (Karberg et <br />    expressed position just downstream 39-exon          al. 2001; Frazier et al. 2003). Because splicing reaction is <br />    (Guo et al. 2000; Karberg et al. 2001). The IEP binds          dependent IEP, suggested conditional dis- <br />    intron precursor RNA, promotes splicing            ruptants obtained controlling expression the <br />    stabilizing catalytically active RNA structure,        IEP separate construct inducible promoter <br />    remains bound excised intron lariat RNA RNPs            (Karberg et al. 2001; Frazier et al. 2003). <br />    promote mobility. For mobility, Ll.LtrB intron              Here, constructed targetron donor plasmids for <br />    reverse-splices strand target DNA,      S. aureus used disrupt essential hsa <br />    IEP cleaves opposite strand uses 39 DNA end           gene nonessential seb gene high frequencies <br />    cleavage site primer reverse transcription     selection. The hsa gene disrupted using <br />    inserted intron RNA. The resulting intron cDNA             targetron inserts sense orientation relative to <br />    integrated recipient DNA using host DNA               target gene transcription spliced IEP- <br />    replication repair enzymes (Matsuura et al. 1997;             assisted manner precursor RNA, resulting the <br />    Cousineau et al. 1998; Smith et al. 2005).                        production functional HSa protein. Further, show <br />       The Ll.LtrB intron targeted desired sites aid   IEP-assisted splicing reaction inherently tem- <br />    algorithm, scans target sequence          perature-sensitive, consequently, procedure yields <br />    best matches positions recognized IEP          conditional hsa disruptant grow 32&#194;&#176;C not <br />    designs PCR primers optimal base-pairing             43&#194;&#176;C. Our results demonstrate time that <br />    interactions intron RNA&#226;&#8364;&#8482;s EBS1, EBS2, d           targetrons used disrupt essential genes, they <br />    sequences IBS1, IBS2, d9 sequences DNA         suggest novel general strategy obtaining one-step <br />    target site (Perutka et al. 2004). The positions recognized       conditional gene disruptions organism. In nature, <br />    IEP sufficiently flexible             temperature sensitivity group II intron splicing <br />  <br />    1272         RNA, Vol. 12, No. 7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 3 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                          Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />     reactions limit temperature range organisms           night 37&#194;&#176;C, colonies screened targetron in- <br />     containing group II introns inserted essential genes.       sertion PCR, using primers flanking predicted <br />                                                                        insertion site target gene. The initial hsa disruption <br />                                                                        avirulent S. aureus strain RN4220. The seb <br />     RESULTS <br />                                                                        disruption strain RN8908, contains the <br />                                                                        SaPI3 pathogenicity island encoding SEB, condition- <br />     Use Ll.LtrB targetron gene disruption <br />                                                                        ing donor plasmid passage RN4220, which <br />     S. aureus <br />                                                                        does contain SaPI3 (see Materials Methods). <br />     For gene targeting S. aureus, constructed intron-               The Pcad promoter low basal level transcription <br />     donor plasmids patterned intron-donor plasmid            absence cadmium (Charpentier et al. 2004), at <br />     pACD3 used E. coli (Guo et al. 2000; Karberg et al. 2001).      outset, clear cadmium induction <br />     pNL9161 contains 0.9-kb Ll.LtrB-DORF intron flank-           necessary obtain disruptants. Indeed, hsa, <br />     ing exons cloned cadmium-inducible promoter               PCR-screened 10 colonies cells cadmium <br />     (Pcad) S. aureus vector pCN37 (Charpentier et al.           induction 10 desired disruption <br />     2004). The LtrA ORF deleted normal location            (Fig. 3A, left panel). Insertion targetron correct <br />     intron inserted just downstream 39-          site confirmed sequencing PCR products (data <br />     exon. In E. coli, Ll.LtrB-DORF intron susceptible   shown). For seb, 24 colonies cells <br />     nuclease degradation full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> intron, leading    induced cadmium desired <br />     substantially higher integration frequencies (Guo et al. 2000).    disruption, cadmium induction, 24 <br />        Two candidate genes chosen test intron targeting        colonies desired disruption (Fig. 4A; left panel <br />     S. aureus: hsa, encodes member HU                shows PCR data subset colonies). The correct <br />     family bacterial histone-like proteins (Viter et al. 1999),     disruption confirmed sequencing colony <br />     seb, encodes enterotoxin B (Shafer Iandolo           PCR products (data shown). For hsa seb, two <br />     1978). First, analyzed genes algo-            randomly chosen disruptants analyzed Southern hybrid- <br />     rithm identify potential insertion sites Ll.LtrB        ization using intron probe showed just single band of <br />     targetron (see Materials Methods), selected             size expected integration desired target site, <br />     single target sites potential target sites        larger band corresponding donor <br />     seb potential target sites hsa. Because         plasmid, additional bands nonspecific <br />     laboratories independently failed obtain hsa             insertions (Figs. 3A, 4A, right panels figure legends <br />     <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>knockout</span> (A. Cheung, pers. comm.; Y. Fang R.P.                 molecular weight range spanned blot). In other <br />     Novick, unpubl.), suspected gene             experiments, Hsa-24s targetron used disrupt <br />     essential, selected anti-Hsa targetron       hsa gene S. aureus strain RN7206 several <br />     inserts sense orientation potentially          plasmid-carrying derivatives RN4220 RN7206, with <br />     removed RNA splicing, enabling production                disruption small number <br />     functional HSa protein. An additional challenge             colonies analyzed (data shown). <br />     hsa gene small size (270 bp) (Viter et al. 1999), <br />     limits choice target sites. <br />                                                                        Ll.LtrB gene targeting S. aureus using <br />        Donor plasmids containing Ll.LtrB targetrons insert <br />                                                                        temperature-sensitive vector <br />     selected sites hsa seb genes were <br />     constructed PCR using primers designed algo-            Because donor plasmid expresses LtrA protein, which <br />     rithm modify intron&#226;&#8364;&#8482;s EBS1, EBS2, d sequences           potentially remobilize splice inserted Ll.LtrB- <br />     base-pair optimally DNA target site sequences IBS1,          DORF intron, desirable cure disruptants <br />     IBS2, d9. The IBS1 IBS2 sequences 59-exon           prior analysis. To facilitate curing donor plasmid, <br />     donor plasmid modified comple-              recloned Ll.LtrB targetron constructs vector <br />     mentary intron&#226;&#8364;&#8482;s retargeted EBS1 EBS2 sequen-           backbone pCN39, temperature-sensitive <br />     ces efficient RNA splicing donor construct (see       replicon (Fig. 1B). The resulting donor plasmid containing <br />     Fig. 1; Materials Methods). The pCN37-based donor              wild-type Ll.LtrB intron denoted pNL9162, the <br />     plasmids containing targetrons Hsa-24s Seb-123a            donor plasmids containing targetrons Hsa-24s Seb- <br />     denoted pNL9163 pNL9165, respectively (Table 1).               123a denoted pNL9164 pNL9166, respectively <br />     The predicted base-pairing interactions              (Table 1). The donor plasmids gave targeted dis- <br />     targetrons target sites shown Figure 2.           ruptions hsa seb genes high <br />        S. aureus strains containing donor plasmids            efficiency specificity pCN37-based donor plasmids <br />     grown early log phase plated directly        (Figs. 3B, 4B). The Hsa-24s targetron gave 100% <br />     BHI medium containing erythromycin induced                 disruptants (24/24) cadmium induction, the <br />     CdCl2 prior plating. After incubating plates over-          Seb-123a targetron gave 7/19 disruptants cadmium <br />  <br />                                                                                                         www.rnajournal.org       1273 <br />  JOBNAME: RNA 12#7 2006 PAGE: 4 OUTPUT: Wednesday June 7 03:20:17 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />  <br />    FIGURE 1. Intron-donor plasmids S. aureus gene targeting. (A) pNL9161 derivative S. aureus expression vector pCN37, which <br />    employs cadmium-inducible promoter (Pcad) cadmium regulator (cadR) express precursor RNA containing Ll.LtrB-DORF <br />    intron flanking 59- 39-exon sequences (E1 E2, respectively), plus LtrA ORF downstream E2. The plasmid contains <br />    staphylococcal pT181 replicon colE1 replicon, erythromycin-resistance (ermC) ampicillin-resistance (ampR) genes. (B) <br />    pNL9162 derivative S. aureus expression vector pCN39 similar pNL9161 (above), contains temperature- <br />    sensitive replicon (pT181 cop-634ts repC4) place wild-type pT181 replicon. (C) PCR reactions used modify targetron insertion <br />    different genes. For targeting, EBS1, EB2, d sequences Ll.LtrB-DORF intron complementary IBS1, IBS2, d9 in <br />    DNA target site, IBS1 IBS2 E1 donor plasmid complementary intron&#226;&#8364;&#8482;s retargeted EBS1 EBS2 sequences <br />    efficient RNA splicing. The modifications introduced donor plasmid two-step PCR (PCR1 2), using primers p1 + p2 and <br />    p3 + p4, diagrammed figure described in Materials Methods. The PCRs yield 353-bp HindIII/BsrGI fragment, is <br />    swapped corresponding fragment donor plasmid. <br />  <br />  <br />    induction (Figs. 3B, 4B; left panels PCR data                    To determine splicing reaction tempera- <br />    subset colonies). The pCN39-based donor plasmid                   ture-sensitive, splicing Ll.LtrB-DORF intron <br />    readily cured seb disruptants plating cells                 inserted hsa gene assayed RT-PCR, using <br />    43&#194;&#176;C (Fig. 4B, Southern blot, right-hand lanes). However,                  primers flanking intron-insertion site (Fig. 5B). At <br />    donor plasmid cured hsa                          32&#194;&#176;C, wild-type (WT) strain, predominant <br />    disruptants, presumably IEP-assisted splicing                  product 0.1-kb band corresponding hsa mRNA <br />    inserted intron required produce HSa protein.                 (Fig. 5B, lane 1), band, corresponding to <br />    The inability cure donor plasmid confirms                       mRNA spliced exons, predominant <br />    essentiality hsa gene.                                              product hsa disruptants obtained donor <br />                                                                               plasmids pNL9163 pNL9164 (Fig. 5B, lanes 2 3, <br />    Targetron insertion creates conditional <br />    hsa disruptant <br />                                                                                TABLE 1. S. aureus intron-donor plasmids used study <br />    We noticed hsa disruptants containing pCN37- <br />                                                                                Plasmid              Vector backbone                 Targetron <br />    pCN39-based donor plasmids temperature-sensitive <br />    grow plates 32&#194;&#176;&#226;&#8364;&#8220;41&#194;&#176;C 43&#194;&#176;C (Fig.                  pNL9161                 pCN37                    WT Ll.LtrB-DORF <br />    5A; seb disruptants temperature-sensitive; data                pNL9162                 pCN39 (ts)a              WT Ll.LtrB-DORF <br />                                                                                pNL9163                 pCN37                    Hsa-24s <br />    shown). Because temperature sensitivity hsa                  pNL9164                 pCN39 (ts)               Hsa-24s <br />    disruptants observed plates contained                pNL9165                 pCN37                    Seb-123a <br />    erythromycin force retention donor plasmid                   pNL9166                 pCN39 (ts)               Seb-123a <br />    supplies IEP, suspected reflect                    a <br />                                                                                 ts, temperature-sensitive. <br />    temperature sensitivity IEP-assisted splicing reaction. <br />  <br />    1274         RNA, Vol. 12, No. 7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 5 OUTPUT: Wednesday June 7 03:20:21 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                 Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />                                                                               RT-PCR analysis wild-type Ll.LtrB-DORF intron tran- <br />                                                                               scripts expressed donor plasmids pACD2X E. coli <br />                                                                               pNL9161 S. aureus. As shown Figure 6, we <br />                                                                               observed similar inhibition splicing 1 1.5 h <br />                                                                               temperature shift 30&#194;&#176;C (E. coli) 32&#194;&#176;C <br />                                                                               (S. aureus) 37&#194;&#176;C 43&#194;&#176;C, accumulation <br />                                                                               unspliced precursor RNA decrease level of <br />                                                                               spliced-exon RNA longer incubation higher <br />                                                                               temperature. In experiments, real-time RT-PCR <br />                                                                               showed splicing full-length wild-type Ll.LtrB <br />                                                                               intron expressed plasmid pACD-LtrB E. coli <br />     FIGURE 2. DNA target sites base-pairing interactions the <br />     Hsa-24s Seb-123a targetrons. The figure shows DNA target site         HMS174(DE3) temperature-sensitive (unspliced pre- <br />     sequences positions &#195;&#191;30 +15 intron-insertion site        cursor RNA, 2.57 6 0.05 32&#194;&#176;C 11.1 6 0.6 43&#194;&#176;C; <br />     intron RNA/DNA target site base-pairing interactions          spliced exons, 4.2 6 0.2 32&#194;&#176;C 2.10 6 0.05 43&#194;&#176;C, <br />     wild-type (WT) Ll.LtrB intron targetrons Hsa-24s Seb-123a, <br />     designed insert S. aureus hsa seb genes, respectively. <br />                                                                               overnight incubation indicated temperatures). <br />     Targetrons named nucleotide position 59 insertion     Thus, IEP-assisted splicing reaction Ll.LtrB group <br />     site target gene&#226;&#8364;&#8482;s coding sequence, followed &#226;&#8364;&#732;&#226;&#8364;&#732;s&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;a&#226;&#8364;&#8482;&#226;&#8364;&#8482;     II intron appears inherently temperature-sensitive. <br />     indicating sense antisense strands, respectively. The intron-in- <br />     sertion site strand (IS) bottom-strand cleavage site <br />     IEP (CS) indicated arrowheads. Nucleotide residues       DISCUSSION <br />     DNA target site match wild-type Ll.LtrB intron target site <br />     highlighted gray.                                                  We Ll.LtrB targetron used for <br />                                                                               efficient chromosomal gene disruption human <br />                                                                               pathogen S. aureus. Targetron Hsa-24s, used disrupt <br />     respectively). The disruptants lighter 1.0-kb                 hsa gene, exceptionally efficient, giving 100% <br />     band corresponding unspliced precursor RNA,                      disruptant colonies, cadmium induction of <br />     prominent cells containing pNL9163 (the                       targetron transcription, targetron Seb-123a, used to <br />     pCN37-based donor plasmid) (Fig. 5B, lane 2)                      disrupt seb gene encoding enterotoxin B, somewhat <br />     containing pNL9164 (the pCN39-based donor plas-                     efficient, giving z37% disruptant colonies after <br />     mid) (Fig. 5B, lane 3). This difference reflect different             cadmium induction. The difference frequency the <br />     plasmid copy numbers (20&#226;&#8364;&#8220;25 300&#226;&#8364;&#8220;400 copies                        Hsa-24s Seb-123a targetrons likely reflects primarily <br />     cell vectors pCN37 pCN39, respectively)                       retargeted introns recognize new target <br />     (Charpentier et al. 2004), resulting different levels               sites, seen previously different targetrons E. coli <br />     expression IEP needed precursor RNA splicing.                  (Perutka et al. 2004). The Ll.LtrB targetron used <br />        Strikingly, cells grown 32&#194;&#176;C shifted               previously targeted gene disruption other <br />     43&#194;&#176;C 2 h, wild-type strain showed single band             Gram-positive bacteria, L. lactis C. perfringens. In <br />     corresponding hsa mRNA (Fig. 5B, lane 4),                    L. lactis, different Ll.LtrB targetrons expressed from <br />     disruptants showed substantial accumulation larger                 nisin-inducible promoter gave disruptants mleS gene <br />     band corresponding unspliced precursor RNA,                        100% screened colonies, targetrons <br />     concomitant reduction intensity band corre-                 used site-specifically insert cargo gene cloned <br />     sponding mRNA spliced exons (Fig. 5B,                         intron (the abiD gene conferring phage re- <br />     lanes 5,6). Quantitation precursor mRNA levels                  sistance) (Frazier et al. 2003). In C. perfringens, Ll.LtrB <br />     real-time RT-PCR confirmed accumulation pre-                       targetron expressed constitutive promoter gave <br />     cursor RNA reduction hsa mRNA levels                        disruptants a-toxin gene (plc) 38 screened <br />     disruptants shift 43&#194;&#176;C (Table 2). Together,            colonies (Chen et al. 2005). Based examples, it <br />     findings indicate temperature sensitivity                 likely Ll.LtrB targetron generally useful <br />     hsa disruptants temperature sensitivity                  gene targeting Gram-positive bacteria. <br />     IEP-assisted splicing reaction inserted                           Our results time targetron that <br />     Ll.LtrB-DORF intron.                                                      inserts sense orientation removed from <br />                                                                               precursor RNA RNA splicing used disrupt <br />                                                                               essential gene. Further, IEP-assisted splic- <br />     IEP-assisted splicing wild-type Ll.LtrB-DORF <br />                                                                               ing reaction Ll.LtrB intron inherently temperature- <br />     intron temperature-sensitive both <br />                                                                               sensitive, resulting one-step conditional <br />     E. coli S. aureus <br />                                                                               disruption. We note ability obtain conditional <br />     To determine IEP-assisted splicing reaction                disruptant dependent factors, including the <br />     inherently temperature-sensitive, carried similar                  splicing efficiency inserted intron, stability the <br />  <br />                                                                                                                www.rnajournal.org       1275 <br />  JOBNAME: RNA 12#7 2006 PAGE: 6 OUTPUT: Wednesday June 7 03:20:22 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                                                       In principle, temperature sensi- <br />                                                                                                    tivity RNA splicing reaction <br />                                                                                                    reflect temperature sensitivity of <br />                                                                                                    IEP, intron RNA, some <br />                                                                                                    combination two. Previous stud- <br />                                                                                                    ies showed purified LtrA protein is <br />                                                                                                    rapidly irreversibly denaturated <br />                                                                                                    incubated splicing reaction <br />                                                                                                    medium 37&#194;&#176;C absence of <br />                                                                                                    intron RNA lower temper- <br />                                                                                                    atures favor expression active <br />                                                                                                    LtrA protein E. coli (Saldanha et al. <br />                                                                                                    1999). Additionally, immunoblot ex- <br />                                                                                                    periments E. coli HMS174(DE3) <br />                                                                                                    expressing donor plasmid pACD2X <br />                                                                                                    showed decreased levels soluble LtrA <br />                                                                                                    protein 43&#194;&#176;C (J. Yao A.M. <br />                                                                                                    Lambowitz, unpubl.). Thus, tem- <br />                                                                                                    perature sensitivity IEP likely <br />                                                                                                    major factor temperature sen- <br />                                                                                                    sitivity splicing reaction. Further, <br />                                                                                                    based vitro results, expect this <br />                                                                                                    temperature sensitivity exacer- <br />                                                                                                    bated IEP expressed from <br />                                                                                                    ectopic location bind <br />                                                                                                    immediately intron RNA (see <br />                                                                                                    Saldanha et al. 1999; Cui et al. 2004). <br />                                                                                                    The thermal stability IEP <br />    FIGURE 3. Use Ll.LtrB targetron disrupt S. aureus hsa gene. (A,B) Disruption of <br />    hsa gene S. aureus RN4220 using targetron Hsa-24s expressed pNL9163 and intron RNA structure may <br />    pNL9164, respectively. For steps, cells containing pNL9163 (pCN37-derivative) grown differ different hosts, latter <br />    37&#194;&#176;C, containing pNL9164 (pCN39-derivative; temperature-sensitive replicon) influenced location <br />    grown 32&#194;&#176;C. Donor plasmids electroporated RN4220, cells grown intron-insertion site the <br />    BHI medium containing erythromycin 1 h, plated medium. The left <br />    panels colony PCR wild-type (WT) RN4220 disruptants obtained target gene. Consequently, be <br />    donor plasmids. The right panels Southern blots MfeI-digested donor plasmids necessary define optimal temper- <br />    pNL9163 pNL9164 (left lane) genomic DNA randomly chosen hsa disruptants ature range conditional disruption <br />    obtained corresponding donor plasmid (right lanes). The blots hybridized with <br />                                                                                                    case. In future, be <br />    32P-labeled intron probe. The bands shown ones visible blots (molecular <br />    weight range 100 bp&#226;&#8364;&#8220;12 kb). The schematic shows diagram hsa gene, possible change operational tem- <br />    (top) (bottom) integrated targetron, including location primers perature range genetically selecting <br />    pf pr used detect targetron integration colony PCR, MfeI sites used variants IEP having greater or <br />    Southern hybridization analysis. <br />                                                                                                    lesser temperature sensitivity. <br />                                                                                                       Notably, bacteria contain natu- <br />                                                                                                    ral group II introns inserted essen- <br />    target gene mRNA, gene product                           tial genes (e.g., groEL Azotobacter) (Ferat et al. 2003). It <br />    needed sustain growth. These factors differ                     determine other <br />    different genes different organisms,                     group II introns display temperature-sensitive splicing <br />    possible obtain conditional disruptants cases.                similar Ll.LtrB intron. If so, temperature <br />    The splicing efficiency inserted targetron de-                   range bacteria grow limited by <br />    pendent sequences flanking insertion site,                 inherent temperature sensitivity splicing reaction <br />    EBS/IBS d&#226;&#8364;&#8220;d9 interactions                   inserted group II intron. Moreover, sporadic in- <br />    intron flanking exon sequences                         sertion group II introns nature rapidly change <br />    flanking exon sequences potentially form obstructing                   fitness organisms grow different temperatures. <br />    RNA structures. Ultimately, possible assess                   Nevertheless, bacterial group II introns can <br />    parameters systematically obtain general rules                splice elevated temperatures, possibly reflecting adapta- <br />    predicting splicing efficiency targetrons inserted               tion intron host insertion (Chee and <br />    different sites.                                                           Takami 2005). <br />  <br />    1276         RNA, Vol. 12, No. 7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 7 OUTPUT: Wednesday June 7 03:20:26 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                      Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />                                                                                                        sufficient express targetrons levels <br />                                                                                                        highly efficient gene disruption. <br />                                                                                                        A caveat studies used <br />                                                                                                        short Ll.LtrB-DORF intron without <br />                                                                                                        selectable marker, some <br />                                                                                                        indication longer forms in- <br />                                                                                                        tron containing genetic marker not <br />                                                                                                        expressed efficiently pro- <br />                                                                                                        moters (J. Yao A.M. Lambowitz, <br />                                                                                                        unpubl.). The efficiency translation <br />                                                                                                        IEP, required both <br />                                                                                                        RNA splicing intron integration, is <br />                                                                                                        parameter affect tar- <br />                                                                                                        getron expression different hosts. This <br />                                                                                                        parameter influenced Shine- <br />                                                                                                        Dalgarno sequences around <br />                                                                                                        translation initiation site, as <br />                                                                                                        codon usage LtrA protein com- <br />                                                                                                        pared optimal host organ- <br />                                                                                                        ism. However, work with <br />                                                                                                        S. aureus Chen et al. (2005) <br />                                                                                                        C. perfringens Shine-Dal- <br />                                                                                                        garno sequence optimized spe- <br />                                                                                                        cific host. Although far, targetrons <br />                                                                                                        introduced donor plas- <br />                                                                                                        mids replicate desired host, it <br />                                                                                                        possible introduce <br />                                                                                                        conjugation (Belhocine et al. <br />                                                                                                        2004) nonreplicating DNA per- <br />     FIGURE 4. Use Ll.LtrB targetron disrupt S. aureus seb gene. (A,B) Disruption      haps pre-assembled RNPs. We <br />     seb gene S. aureus RN8098 using targetron Seb-123a expressed pNL9165               anticipate procedures be <br />     pNL9166. For steps, cells containing pNL9165 (pCN37-derivative) grown 37&#194;&#176;C, and <br />     containing pNL9166 (pCN39-derivative; temperature-sensitive replicon) grown          broadly applicable use targe- <br />     32&#194;&#176;C. Donor plasmids electroporated S. aureus RN4220, reisolated                trons wide variety prokaryotes. <br />     electroporated RN8098 (see Materials Methods). Cells grown BHI medium <br />     containing erythromycin, induced 10 mM CdCl2 90 min, plated BHI <br />     medium containing erythromycin. The left panels colony PCR wild-type RN8098                MATERIALS AND METHODS <br />     (WT) representative colonies obtained donor plasmids. The right panels <br />     Southern blots AseI-digested donor plasmids pNL9165 pNL9166 (left lane) and <br />     randomly chosen seb disruptants obtained corresponding donor plasmid (right lane).        Bacterial strains growth <br />     The blots hybridized 32P-labeled intron probe. The samples rightmost        conditions <br />     lanes Southern blot panel B cells pNL9166 cured the <br />     disruptants overnight incubation 43&#194;&#176;C BHI plates. The bands shown ones     The S. aureus strains RN4220 and <br />     visible blots (molecular weight range 100 bp&#226;&#8364;&#8220;12 kb). The schematic shows    RN8098. RN4220 restriction-defective, <br />     diagram seb gene, (top) (bottom) integrated targetron, including <br />                                                                                                      nonlysogenic derivative NCTC8325 is <br />     location primers pf pr used detect targetron integration colony PCR the <br />     location AseI sites used Southern hybridization analysis.                             efficient recipient E. coli DNA (Kreis- <br />                                                                                                      wirth et al. 1983). RN8098 contains the <br />                                                                                                      SaPI3 pathogenicity island encodes the <br />        The methods used adapting targetrons use                    staphylococcal superantigen enterotoxin B (SEB) (Novick et al. <br />     S. aureus (this study) C. perfringens (Chen et al. 2005)               2001). S. aureus strains grown brain heart infusion (BHI) <br />     relatively straightforward. Principally, targetron             trypticase soy agar (TSA) media, erythromycin added at <br />                                                                               10 mg/mL. Unless specified otherwise, cells containing pCN37- <br />     recloned donor plasmid promoter suitable for <br />                                                                               based plasmids grown 37&#194;&#176;C, containing pCN39- <br />     expressing intron desired host. In E. coli, targetrons <br />                                                                               based plasmids, temperature-sensitive replicon, were <br />     expressed phage T7 promoter, using T7                    grown 32&#194;&#176;C. <br />     RNA polymerase introduced lDE3 lysogen                         E. coli strain DH5a used cloning, HMS174 (DE3) was <br />     separate plasmid (Guo et al. 2000; Karberg et al. 2001).                used RT-PCR analysis Ll.LtrB intron splicing. E. coli strains <br />     However, results S. aureus C. perfringens                grown LB medium, ampicillin added 100 mg/mL, <br />     native host promoter, inducible constitutive,            chloramphenicol 25 mg/mL, erythromycin 200 mg/mL. <br />  <br />                                                                                                                     www.rnajournal.org       1277 <br />  JOBNAME: RNA 12#7 2006 PAGE: 8 OUTPUT: Wednesday June 7 03:20:30 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                                                     just downstream 39-exon, the <br />                                                                                                     Ll.LtrB-DORF intron additional <br />                                                                                                     phage T7 promoter inserted intron RNA <br />                                                                                                     domain IV use E. coli genetic assays of <br />                                                                                                     intron mobility (see Guo et al. 2000; Karberg <br />                                                                                                     et al. 2001). pACD-LtrB related plasmid <br />                                                                                                     expresses full-length wild-type <br />                                                                                                     Ll.LtrB intron LtrA ORF encoded <br />                                                                                                     intron domain IV (Guo et al. 2000). <br />                                                                                                        The S. aureus targetron donor plasmids <br />                                                                                                     used work summarized Table 1. <br />                                                                                                     pNL9161 contains Ll.LtrB targe- <br />                                                                                                     tron construct pACD2X cloned down- <br />                                                                                                     stream cadmium-inducible promoter <br />                                                                                                     (Pcad) S. aureus vector pCN37 <br />                                                                                                     (Charpentier et al. 2004). To construct this <br />                                                                                                     plasmid, 3-kb XbaI/XhoI fragment of <br />                                                                                                     pACD2X, containing Ll.LtrB-DORF in- <br />                                                                                                     tron flanking exon sequences plus the <br />                                                                                                     downstream LtrA ORF, gel-purified, <br />                                                                                                     blunt-ended T4 DNA polymerase, and <br />                                                                                                     cloned SmaI site pCN37. To <br />                                                                                                     facilitate introduction modifications <br />                                                                                                     needed gene targeting, XbaI site <br />                                                                                                     downstream Pcad replaced an <br />                                                                                                     NheI site cleaving pCN37 multiclon- <br />                                                                                                     ing site PstI EcoRI, ligating in <br />                                                                                                     DNA oligonucleotide containing NheI <br />                                                                                                     site flanked PstI EcoRI sites. In <br />                                                                                                     addition, HindIII site pT181 repC <br />                                                                                                     eliminated altering amino <br />                                                                                                     acid sequence (AAGCTT AAGCTA; Lys- <br />                                                                                                     Leu Lys-Leu), BsrGI site down- <br />                                                                                                     stream repC removed BsrGI <br />                                                                                                     digestion, followed fill-in T4 DNA <br />                                                                                                     polymerase, religation. <br />    FIGURE 5. hsa disruptants temperature sensitive growth splicing Ll.LtrB-         pNL9162 contains Ll.LtrB <br />    DORF intron. (A) Plating assay. Wild-type RN4220 hsa disruptants obtained               targetron construct cloned pCN39 <br />    pNL9163 pNL9164 (ts) streaked TSA agar plates temperatures 32&#194;&#176;        (Charpentier et al. 2004), a <br />    43&#194;&#176;C. The plates incubated 24&#226;&#8364;&#8220;48 h photographed. The temperature-sensitive          temperature-sensitive replicon. It con- <br />    plasmid pNL9164 expressing LtrA (pCN39-derivative) present higher copy number than <br />    pNL9163 (pCN37-derivative) (Charpentier et al. 2004), likely resulting somewhat       structed follows: First, Pcad promoter <br />    residual growth temperature shift 43&#194;&#176;C. (B) RT-PCR assay Ll.LtrB-DORF intron     cut pCN37 digestion SphI <br />    splicing hsa gene. Cells grown BHI medium overnight, diluted 1:20     PstI, cloned SphI and <br />    fresh medium, incubated 4 h 32&#194;&#176;C 2 h 43&#194;&#176;C. RNA extracted            PstI sites pCN39. Then, XbaI site <br />    analyzed RT-PCR, using primers pE1 pE2 flanking Ll.LtrB-DORF intron inserted       downstream Pcad replaced an <br />    hsa gene. The smaller RT-PCR product (0.1 kb) derived hsa mRNA           NheI site HindIII BsrGI sites <br />    inserted intron, larger product (1.0 kb) derived precursor RNA containing <br />                                                                                                     removed, described for <br />    inserted intron. The identities RT-PCR products confirmed sequencing <br />    (data shown). Controls showed products initial reverse transcription step      pNL9161. Finally, pACD2X digested <br />    omitted reactions carried absence RNA substrate (data shown),   XbaI EcoRV, 3-kb frag- <br />    indicating amplified bands experimental samples derived cellular RNA.   ment containing Ll.LtrB-DORF intron <br />    The schematic right gel shows hsa precursor mRNAs, including           flanking exons plus downstream <br />    location primers pE1 pE2 expected RT-PCR products.                                LtrA ORF cloned NheI <br />                                                                                                     SmaI sites pCN39. <br />  <br />    Recombinant plasmids                                                      Retargeting Ll.LtrB intron insert into <br />                                                                              S. aureus genes <br />    pACD2X contains 0.9-kb Ll.LtrB-DORF intron flanking <br />    exons cloned downstream T7lac promoter CamR                     The Ll.LtrB targetron retargeted insert S. aureus genes <br />    pACYC184-based vector (San Filippo Lambowitz 2002).                   using algorithm identifies potential insertion <br />    The ORF encoding LtrA protein expressed position            sites designs PCR primers modifying intron RNA to <br />  <br />  <br />    1278         RNA, Vol. 12, No. 7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 9 OUTPUT: Wednesday June 7 03:20:35 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                  Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />       TABLE 2. Real-time RT-PCR analysis levels hsa transcripts S. aureus strains <br />  <br />                                                           Unspliced precursor RNAb                                   hsa mRNAb <br />  <br />       Straina                                           32&#194;&#176;C                      43&#194;&#176;C                     32&#194;&#176;C                      43&#194;&#176;C <br />  <br />       WT                                                N.A.                       N.A.                0.23 6 0.02               0.25 6 0.02 <br />       RN4220 hsar::LtrB/PNL9163                     2.12 6 0.10                2.73 6 0.07             0.17 6 0.01              0.048 6 0.002 <br />       RN4220 hsar::LtrB/PNL9164 (ts)c               0.29 6 0.03                2.97 6 0.06             0.24 6 0.03              0.013 6 0.002 <br />       a <br />        WT, wild-type S. aureus RN4220; RN4220 hsa::LtrB/pNL9163, hsa disruptant obtained pNL9163; RN4220 hsa::LtrB/pNL9164, hsa <br />       disruptant obtained pNL9164. <br />       b <br />         Amounts hsa transcripts expressed percentage 16S rRNA. <br />       c <br />        ts, temperature-sensitive. <br />       N.A., applicable. <br />  <br />  <br />  <br />     base-pair sites (Perutka et al. 2004). Single target sites     1992). E. coli DNAs difficult transform directly into <br />     hsa seb chosen potential target sites                  S. aureus RN8098 restriction. Consequently, seb <br />     identified algorithm gene. Donor plasmids              targetron plasmid propagated S. aureus RN4220, re- <br />     constructed intron RNA&#226;&#8364;&#8482;s EBS1, EBS2,                 isolated using Qiaquick Miniprep Kit (Qiagen), electro- <br />     d sequences modified optimally base-pair DNA target             porated RN8098. Cells grown overnight BHI <br />     site sequences IBS1, IBS2, d9. The IBS2 IBS1 sequences          containing erythromycin, diluted 1:100 5 mL fresh <br />     59-exon donor plasmid modified com-             medium, grown early log phase (OD595 = 0.5), then <br />     plementary retargeted EBS2 EBS1 sequences effi-             plated directly BHI containing erythromycin induced <br />     cient RNA splicing (Perutka et al. 2004).                                  10 mM CdCl2 90 min prior plating. Targetron <br />        The required modifications introduced donor               integration assayed colony PCR, using primers flanking <br />     plasmid two-step PCR diagrammed Figure 1C (Karberg                target gene referred generically pf pr. For <br />     et al. 2001). In step, segments donor plasmid         hsa, pf pr primers 59-CGGAATCAGGAGGTGAA <br />     amplified PCRs. One PCR used primers p1 (59-exon                   TGTCTAATGA 59-CGGAATTCACTTAGAAGAACATTACAA <br />     positions &#195;&#191;25 +18 modifications positions &#195;&#191;12 &#195;&#191;1             TTTAT; seb, pf pr primers 59-GGATAAAG <br />     IBS1 IBS2; Seb-123a donor plasmids, positions &#195;&#191;25              GAGATAAAAAATGTATAAGAG 59-AGGTACTCTATAAGTGC <br />     &#195;&#191;13 changed ATAATTATCCTTA prevent                       CTGCCTTAATTAC. <br />     self-targeting) plus p2 (59-CGAAATTAGAAACTTGCGTTCAGTA <br />     AAC). The PCR used primers p3 (intron positions +198              Southern hybridization <br />     +246 modifications positions +223 +227 EBS2) plus <br />     p4 (intron positions +259 +326 modifications positions          Cells grown BHI medium overnight, DNA was <br />     +276 +285 EBS1 d). In second step, PCR              isolated using Bacterial Genomic DNA Prep kit (QIAGEN). <br />     products step gel-purified, mixed             Southern hybridization described (Perutka et al. 2004), <br />     amplified outside primers p1 p4 generate 353-bp          using 32P-labeled intron probe generated PCR pACD2X <br />     PCR product corresponding 59-exon 59-end                 primers 59-TCTTGCAAGGGTACGGAGTA 59-GTAGG <br />     intron (nucleotide positions E1&#226;&#8364;&#8220;25 I + 326), modified              GAGGTACCGCCTTGTTC. The probe labeled using High <br />     IBS1/2, EBS2, EBS1/d sequences. The final PCR product              Prime DNA Labeling kit (Roche Diagnostics). <br />     purified 0.8% agarose gel cloned BsrGI and <br />     HindIII sites donor plasmid. In initial experiments             RT-PCR qRT-PCR analysis RNA splicing <br />     disruption hsa seb, modifications introduced <br />     pACD2X order validate targetron, using E. coli           S. aureus RN4220 strains containing pCN37- pCN39-based <br />     plasmid assay targetron integration (Guo et al. 2000; Karberg          plasmids grown overnight BHI medium containing <br />     et al. 2001). The modified pACD2X segment recloned                erythromycin 37&#194;&#176;C 32&#194;&#176;C, respectively, 200 mL the <br />     pCN37 (3-kb XbaI + XhoI fragment pACD2X blunt-ended                overnight culture inoculated 20 mL fresh medium and <br />     T4 DNA polymerase ligated SmaI site                   incubated conditions specified individual experiments <br />     pCN37) generate S. aureus donor plasmids pNL9163                Figures 5 6. After incubations, RNA extracted from <br />     pNL9165, contain targetrons Hsa-24s Seb-123a, re-                1 mL culture using RNeasy mini Kit (Qiagen). RT-PCR <br />     spectively (Table 1). For pCN39 derivatives, modifica-            carried 300 ng RNA using SuperScript One- <br />     tions targeting intron hsa seb genes introduced        Step RT-PCR (Invitrogen), primers flanking the <br />     directly pNL9162 generate donor plasmids pNL9164               inserted intron. For Ll.LtrB-DORF intron inserted hsa <br />     pNL9166, respectively (Table 1).                                           (Fig. 5), primers 59-AGGTGAATGTCTAATGAA <br />                                                                                CAAAACAGA 59-CATCTACTGCTGAACCAGCTTCTT, and <br />     Gene targeting S. aureus                                                RT-PCR 50&#194;&#176;C, 30 min; 94&#194;&#176;C, 5 min; 20 <br />                                                                                cycles 94&#194;&#176;C, 30 sec; 55&#194;&#176;C, 30 sec; 72&#194;&#176;C, 75 sec; followed by <br />     The donor plasmids containing retargeted introns                  72&#194;&#176;C, 7 min. For Ll.LtrB-DORF intron pNL9161 (Fig. 6), <br />     electroporated S. aureus RN4220 (Schenk Laddaga                   primers 59-CTAGCAGCACGCCATAGTG and <br />  <br />  <br />                                                                                                                   www.rnajournal.org        1279 <br />  JOBNAME: RNA 12#7 2006 PAGE: 10 OUTPUT: Wednesday June 7 03:20:35 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                           primers conditions used analyze splicing the <br />                                                                           Ll.LtrB-DORF intron pNL9161. <br />                                                                              Real-time reverse transcription PCR (qRT-PCR) S. aureus <br />                                                                           RNA carried 300 ng cellular RNA samples using <br />                                                                           SYBR green RT-PCR kit (Applied Biosystems), primers: <br />                                                                           59-ATGAACAAAACAGATTTAATCAATGCAG [PF(E1&#226;&#8364;&#8220;E2)] plus <br />                                                                           59- CATCTACTGCTGAACCAGCTTCTT (PR) ligated exons, <br />                                                                           59-AGGCGGTACCTCCCTACTTC [PF(Pre)] plus PR for <br />                                                                           unspliced precursor RNA. The forward primer precursor <br />                                                                           RNA corresponds 39-end intron, ending 2 nt <br />                                                                           upstream 39-splice site, forward primer ligated <br />                                                                           exons spans splice junction, 24 nt 59-exon 4 nt <br />                                                                           39-exon, ensuring amplify ligated <br />                                                                           exons precursor RNA. The common reverse primer is <br />                                                                           complementary sequence 39-exon. The specificity of <br />                                                                           primers intended RNAs checked RT-PCR <br />                                                                           (data shown). <br />                                                                              qRT-PCR using universal thermal cycling conditions, <br />                                                                           specified manufacturer (48&#194;&#176;C, 30 min; 95&#194;&#176;C, 10 min; <br />                                                                           40 cycles 95&#194;&#176;C, 15 sec, 60&#194;&#176;C, 1 min) ABI 7900HT fast <br />                                                                           real-time PCR analyzed using SDS software <br />                                                                           (Applied Biosystems). Serial twofold dilutions (1:5&#226;&#8364;&#8220;1:1280) of <br />                                                                           S. aureus wild-type RN4220 cellular RNA used generate <br />                                                                           standard curve fluorescence versus cycle number 16S <br />                                                                           rRNA using primers 59-CTAATCAGAAAGCCACGGCTAAC and <br />                                                                           59-CGGATAACGCTTGCCACCTA. The amounts hsa precur- <br />                                                                           sor RNA mRNA calculated plotting correspond- <br />                                                                           ing CT (cycle number fluorescence crosses threshold <br />                                                                           value set SDS program) standard curve and <br />                                                                           expressed percent 16S rRNA. The CT values 16S rRNA <br />                                                                           similar RNA samples, control reactions without <br />    FIGURE 6. Temperature sensitivity splicing wild-type         addition RT RNA substrate showed nonspecific <br />    Ll.LtrB-DORF intron E. coli S. aureus. (A) E. coli              amplification. <br />    HMS174(DE3) containing intron-donor plasmid pACD2X grown <br />    log phase, 100 mM IPTG added induce intron                 qRT-PCR E. coli RNAs similarly using primers: <br />    expression, cultures incubated 1 h overnight       59-GATCGTGAACACATCCATAACCATA plus 59-AAATTGTT <br />    (O/N) 30&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C. (B) S. aureus strain RN4220 containing     TGCCAGTATAAAGATTCG ligated exons; 59-CAAGGCGG <br />    pNL9161 grown overnight, diluted 1:20 fresh BHI          TACCTCCCTACTTC plus 59-AAATTGTTTGCCAGTATAAAGA <br />    containing erythromycin. Then 10 mM CdCl2 added induce          TTCG unspliced precursor RNA; 59-CGTGTTGTGAAAT <br />    intron expression, cultures incubated 90 min 6 h   GTTGGGTTAA 59-CCGCTGGCAACAAAGGATA 16S <br />    32&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C. RNA extracted analyzed RT-PCR, <br />                                                                           rRNA. <br />    using primers pE1 pE2, hybridize 59 (E1) 39 (E2) <br />    exon sequences flanking Ll.LtrB-DORF intron pACD2X or <br />    pNL9161. The schematic beneath gel shows precursor and <br />    mRNAs, location primers pE1 pE2 expected <br />    RT-PCR products.                                                       ACKNOWLEDGMENTS <br />                                                                           This work supported NIH grant GM37949 Welch <br />                                                                           Foundation grant F-1607 A.M.L., NIH grant AI30138 to <br />                                                                           R.P.N. We thank David Goerlitz, Hong Cui, Sijiong Mou for <br />    59-TCGTGAACACATCCATAAC, RT-PCR                        contributions initial experiments S. aureus gene <br />    52&#194;&#176;C, 30 min; 94&#194;&#176;C, 5 min; 40 cycles 94&#194;&#176;C, 30 sec, 58&#194;&#176;C,    targeting; Jiri Perutka help targetron design; Greg <br />    30 sec, 72&#194;&#176;C, 90 sec; followed 72&#194;&#176;C, 7 min. The PCR products        Davis (Sigma-Aldrich) helpful suggestions. <br />    analyzed 2% agarose gel, stained ethidium <br />    bromide.                                                               Received February 17, 2006; accepted April 17, 2006. <br />       For E. coli, strain HMS174 (DE3), contains IPTG- <br />    inducible phage T7 RNA polymerase, transformed with <br />    pACD2X pACD-LtrB plated LB medium containing                 REFERENCES <br />    chloramphenicol. A single colony grown LB medium <br />                                                                           Bae, T. Schneewind, O. 2006. Allelic replacement Staphylococ- <br />    containing chloramphenicol OD595 = 0.2&#226;&#8364;&#8220;0.3, 100 mM             cus aureus inducible counter-selection. Plasmid 55: 58&#226;&#8364;&#8220;63. <br />    IPTG added induce T7 RNA polymerase transcription,          Belhocine, K., Plante, I., Cousineau, B. 2004. Conjugation <br />    culture incubated 30&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C 1 h overnight.      mediates transfer Ll.LtrB group II intron different <br />    RNA extracted analyzed RT-PCR above, using             bacterial species. Mol. Microbiol. 51: 1459&#226;&#8364;&#8220;1469. <br />  <br />  <br />    1280         RNA, Vol. 12, No. 7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 11 OUTPUT: Wednesday June 7 03:20:39 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                       Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />     Charpentier, E., Anton, A.I., Barry, P., Alfonso, B., Fang, Y.,           Kreiswirth, B.N., Lo&#194;&#168;fdahl, S., Betley, M.J., O&#226;&#8364;&#8482;Reilly, M., <br />         Novick, R.P. 2004. Novel cassette-based shuttle vector            Schlievert, P.M., Bergdoll, M.S., Novick, R.P. 1983. The toxic <br />         Gram-positive bacteria. Appl. Environ. Microbiol. 70: 6076&#226;&#8364;&#8220;6085.             shock syndrome exotoxin structural gene detectably trans- <br />     Chee, G.-J. Takami, H. 2005. Housekeeping recA gene interrupted              mitted prophage. Nature 305: 709&#226;&#8364;&#8220;712. <br />         group II intron thermophilic Geobacillus kaustophilus.          Lambowitz, A.M. Zimmerly, S. 2004. Mobile group II introns. <br />         Gene 363: 211&#226;&#8364;&#8220;220.                                                           Annu. Rev. Genet. 38: 1&#226;&#8364;&#8220;35. <br />     Chen, J. Novick, R. 2006. SvrA, multidrug exporter (MepA) does          La <br /> </body></html>