<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />       An essential role for Orc6 in DNA replication <br />       through maintenance of pre-replicative complexes <br />  <br />  <br />                                                        G1 phase, ORC promotes the origin-association of the clamp <br />                                                        loading protein Cdc6 in an ATP-dependent manner (Speck <br />                                                 et al, 2005). Another factor, Cdt1 (Devault et al, 2002; Tanaka <br />                                                     and Diffley, 2002) directs the nuclear import of the MCM <br />        <br />                                                                                               (minichromosome maintenance) family of proteins, Mcm2&#226;&#8364;&#8220;7, <br />         <br />                                                                                               which act as replication fork helicases (reviewed in Bell and <br />        <br />                         Dutta, 2002). Once ORC and Cdc6 are present at origins, Cdt1&#226;&#8364;&#8220; <br />                              Mcm2&#226;&#8364;&#8220;7 can also associate with origin DNA. Hydrolysis of <br />                                                                                               ATP by Cdc6 is then thought <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> result in Cdt1 dissociation and <br />       The heterohexameric origin recognition complex (ORC)                                    a stronger Mcm2&#226;&#8364;&#8220;7 binding at origins. Subsequent ATP hydro- <br />       acts as a scaffold for the G1 phase assembly of pre-replica-                            lysis by ORC catalyzes the loading of additional Mcm2&#226;&#8364;&#8220;7 <br />       tive complexes (pre-RC). Only the Orc1-5 subunits appear                                complexes (Kawasaki et al, 2006; Randell et al, 2006). Once <br />       to be required for origin binding in budding yeast, yet                                 tightly bound, the continued association of at least Mcm2 with <br />       Orc6 is an essential protein for cell proliferation. Imaging                            chromatin in G1 phase requires the presence of another ORC- <br />       of Orc6-YFP in live cells revealed a punctate <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                                   associated protein, Mcm10 (Homesley et al, 2000). <br />       consistent with the organization of replication origins                                 Collectively, this assemblage of proteins is known as the <br />       into subnuclear foci. Orc6 was not detected at the site of                              pre-replicative complex (pre-RC). In addition to pre-RC for- <br />       division between mother and daughter cells, in contrast to                              mation, the initiation of DNA replication requires the activa- <br />       observations for metazoans, and is not required for mitosis                             tion of two kinase complexes, Clb5/Cdc28 and Dbf4/Cdc7, <br />       or cytokinesis. An essential role for Orc6 in DNA replica-                              which promote the origin-association of Cdc45 (Nougarede <br />       tion was identified by depleting it at specific cell cycle                              et al, 2000; Zou and Stillman, 2000). Cdc45 in turn recruits <br />       stages. Interestingly, Orc6 was required for entry into S                               DNA polymerases to origins (Mimura and Takisawa, 1998; <br />       phase after pre-RC formation, in contrast to previous                                   Aparicio et al, 1999; Zou and Stillman, 2000). <br />       models suggesting ORC is dispensable at this point in the                                   Curiously, only five of the six ORC subunits are required <br />       cell cycle. When Orc6 was depleted in late G1, Mcm2 and                                 for origin recognition and binding in vitro (Lee and Bell, <br />       Mcm10 were displaced <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> chromatin, cells failed to                                    1997). Even though Orc6 is an essential protein in budding <br />       progress through S phase, and DNA combing analysis                                      yeast (Li and Herskowitz, 1993), it appears to be dispensable <br />       following bromodeoxyuridine incorporation revealed that                                 for these functions and its role in cell cycle progression has <br />       the efficiency of replication origin firing was severely                                yet to be determined. Clearly, Orc6 association with the other <br />       compromised.                                                                            budding yeast ORC subunits suggests a function in DNA <br />       The EMBO Journal (2006) 25, 5150&#226;&#8364;&#8220;5158. doi:10.1038/                                     replication. Li and Herskowitz disrupted one copy of ORC6 <br />       sj.emboj.7601391; Published online 19 October 2006                                      in a diploid yeast strain and, following sporulation, were able <br />       Subject Categories: cell cycle                                                          to observe up to two of rounds of cell division from spores <br />       Keywords: DNA replication; minichromosome maintenance                                   inheriting the ORC6 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span>. Arrested cells had a large <br />       protein; origin recognition complex; pre-replicative complex                            budded <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> often observed for DNA replication mu- <br />                                                                                               tants, but the stage of cell cycle arrest could not be deter- <br />                                                                                               mined by FACS analysis due to an insufficient number of <br />                                                                                               cells. Studies involving the replication of Xenopus sperm <br />       Introduction                                                                            DNA in Drosophila egg extracts indicate that Orc6 can pro- <br />       The origin recognition complex (ORC) plays an essential role                            mote DNA replication in this in vitro system (Chesnokov <br />       in the initiation of DNA replication by binding to origin                               et al, 2001). With human cancer cells, depletion of Orc6 by <br />       sequences throughout the cell cycle and acting as a scaffold                            transfection with siRNA duplexes resulted in a significant <br />       for the association of additional protein factors in G1 phase                           reduction in the number of positive cells in BrdU incorpora- <br />       (reviewed in Bell, 2002). Originally isolated and characterized                         tion assays, consistent with a replicative function (Prasanth <br />       in the budding yeast Saccharomyces cerevisiae (Bell and                                 et al, 2002). As well, research with both human and fruit fly <br />       Stillman, 1992), ORC is composed of six distinct subunits,                              cells point to mitotic and/or cytokinetic functions in addition <br />       and orthologs of each have now been found in a wide range of                            to a role for Orc6 in DNA replication (Prasanth et al, 2002; <br />       eukaryotic species (reviewed in DePamphilis, 2005). In early                            Chesnokov et al, 2003). <br />                                                                                                   Here, we demonstrate that Orc6 is required for the initia- <br />       *Corresponding author. Department of Biology, University of Waterloo,                   tion of DNA replication in budding yeast cells and is dis- <br />       200 University Ave., Waterloo, Ontario, Canada N2L 3G1.                                 pensable for progression through mitosis and cytokinesis. We <br />       Tel.: &#195;&#190; 1 519 888 4567 x 33957; Fax: &#195;&#190; 1 519 746 0614;                                  further show that Orc6 is required for the maintenance of <br />       E-mail: bduncker@sciborg.uwaterloo.ca <br />                                                                                               MCM protein association with chromatin, and that depletion <br />       Received: 5 May 2006; accepted: 19 September 2006; published                            of Orc6 after pre-RC formation inhibits replication origin <br />       online: 19 October 2006                                                                 firing. <br />  <br /> 5150 The EMBO Journal VOL 25 | NO 21 | 2006                                                                           &amp; 2006 European Molecular Biology Organization <br />                                                                                                        Orc6 is required for pre-RC maintenance <br />                                                                                                                                JW Semple et al <br />  <br />  <br />  <br /> Results <br /> Orc6 localizes to subnuclear foci in living yeast cells <br /> As a first step in characterizing the essential role of Orc6, <br /> we examined its cellular localization. This was accomplished <br /> by tagging genomic ORC6 with a sequence encoding the GFP <br /> variant eYFP in a haploid yeast strain (DY-1), resulting in <br /> expression of full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> Orc6 with a C-terminal eYFP tag, <br /> under the control of the natural ORC6 promoter (DY-41). <br /> Studies with human cancer cells indicate that, in addition <br /> to the expected nuclear localization during interphase, cyto- <br /> plasmic pools of Orc6 exist with localization observed at both <br /> the cell periphery during mitosis and at the midbody during <br /> cytokinesis (Prasanth et al, 2002). Similar results have been <br /> reported for Drosophila cells (Chesnokov et al, 2003). When <br /> we followed single cells as they progressed through the cell <br /> cycle, only nuclear Orc6 localization was observed <br /> (Figure 1A). To confirm that S. cerevisiae Orc6 is confined <br /> to the nucleus, we constructed a double-tagged haploid strain <br /> expressing Orc6 with a C-terminal eCFP fusion as well as the <br /> nuclear pore protein Nup49 with a C-terminal eYFP fusion <br /> (DY-65). Irrespective of cell cycle stage, Orc6 was consistently <br /> bounded by the nuclear membrane, as designated by the <br /> signal for Nup49 (Figure 1C). Although we cannot rule out <br />                                                                    Figure 1 Orc6 localizes to subnuclear foci throughout the cell <br /> additional pools of Orc6 below our threshold of detection, we      cycle. Orc6-eYFP, Orc6-eCFP and Nup49-eYFP cell preparation and <br /> conclude that there was no significant localization of budding     live imaging was performed as described in Materials and methods. <br /> yeast Orc6 at either the cell periphery or at the mother-bud       (A) A single cell was imaged over a 3 h time period. Both a bright <br />                                                                    field and a single fluorescent image were taken every 10 min; <br /> neck. <br />                                                                    representative images of different cell cycle stages are shown. <br />    To further investigate the subnuclear localization of Orc6 in   Multiple fluorescent images could not be taken at each time point <br /> live cells, 20 images taken in rapid succession were stacked to    to reduce background as in (B), since increased exposure to the light <br /> increase the signal as described in the Materials and methods.     source results in cell bleaching over the 3 h time course. (B) A <br />                                                                    representative field of cells is shown. Since expression of Orc6-eYFP <br /> The resultant higher resolution images revealed a pattern <br />                                                                    is low under its endogenous promoter, a stronger signal with <br /> of punctate Orc6 foci (Figure 1B), similar to what has been        reduced background was obtained by taking 20 fluorescent images <br /> previously observed in fixed cells for other presumed replica-     under low gain in rapid succession and stacking them using ImageJ <br /> tion factory constituents such as Orc2 (Pasero et al, 1999;        1.30v. (C) Co-localization of Orc6 and a nuclear membrane protein <br /> Lengronne et al, 2001). Chromatin immunoprecipitation              (Nup49) in a series of Orc6-eCFP/Nup49-eYFP cells shows that Orc6 <br />                                                                    is found exclusively within the nucleus at all stages of the cell cycle <br /> (ChIP) analysis further indicated that Orc6 specifically binds     observed, as judged by bud morphology. All scale bars correspond <br /> to replication origin sequence (Supplementary Figure 1).           to 5 mm. <br />  <br />  <br /> Depletion of Orc6 leads to an S-phase arrest <br /> Given that the cellular localization of Orc6 was indicative of     contrast, by 3 h the GAL1-ORC6 cells were already showing <br /> a role in DNA replication, and provided no evidence for            defects in S phase progression, with an accumulation of cells <br /> additional cellular function(s), we wanted to determine the        with a DNA content between 1C and 2C. No significant <br /> point in the cell cycle at which cells arrest when depleted of     accumulation of cells with 2C or greater DNA content was <br /> Orc6. A haploid yeast strain (DY-36) was generated replacing       observed, as would be expected for mitotic or cytokinetic <br /> the natural ORC6 promoter with a glucose-repressible GAL1          defects. Indeed, by 6 h, the size of the 2C peak was markedly <br /> promoter. As part of this strain construction, a sequence          reduced compared to earlier time points, and the wt control. <br /> encoding three copies of the HA epitope was fused to the <br /> start of the ORC6 ORF, to facilitate the monitoring of Orc6        Orc6 is required for entry into S phase <br /> levels. Cultures of DY-36 and isogenic wild-type (wt) strain       In order to determine whether Orc6 is required for entry into <br /> DY-26, were grown on 2% galactose/1% raffinose (YPG/R)             and/or progression through S phase, Orc6 was depleted in <br /> medium overnight, washed in ddH2O, and resuspended in              GAL1-ORC6 cells synchronized at G2/M by initially arresting <br /> 2% glucose (YPD) medium. Following 4 h of growth in YPD,           cells in YPG/R supplemented with nocodazole, washing the <br /> the amount of Orc6 in the GAL1-ORC6 cells had fallen below         culture and resuspending the cells in YPD/nocodazole. Since <br /> normal endogenous levels as judged by immunodetection              origins are rendered competent for DNA replication during G1 <br /> of whole-cell extracts (Figure 2A), and these cells clearly        phase, if Orc6 is essential in this process one would expect to <br /> showed growth defects at subsequent time points, relative to       see the formation of a 1C peak as Orc6-depleted cells fail to <br /> the wt controls (Figure 2B). We further compared GAL1-ORC6         efficiently fire origins following release from the block. Such <br /> and wt strain growth in YPD by removing aliquots at 3-h            a 1C peak was indeed observed with the GAL1-ORC6 strain <br /> intervals for FACS analysis. At all time points following the      following Orc6 depletion (Figure 3A). <br /> shift to YPD, the wt strain exhibited prominent 1C and 2C             A central function of the ORC complex in promoting DNA <br /> peaks characteristic of asynchronous cultures (Figure 2C). In      replication is to facilitate the loading of the MCM proteins at <br />  <br /> &amp; 2006 European Molecular Biology Organization                                              The EMBO Journal       VOL 25 | NO 21 | 2006 5151 <br />        Orc6 is required for pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />  <br />                                                                             Figure 3 Orc6 is required for S-phase progression. (A) <br />                                                                             Asynchronous cultures of GAL1-3HA-ORC6 cells (DY-36) and its <br />                                                                             wt parental strain (DY-26; labeled wt) were grown to 5  106 cells/ <br />                                                                             ml (Exp) and then arrested at G2/M with nocodazole (15 mg/ml) for <br />                                                                             3 h in YPG/R. Once blocked, the cells were washed and transferred <br />                                                                             to YPD again with nocodazole (12 mg/ml) for 4 h to maintain the cell <br />                                                                             cycle arrest. The cells were then washed and released into fresh <br />                                                                             YPD without any arrest agents and samples were removed at the <br />                                                                             indicated intervals for FACS analysis. (B) GAL1-3HA-ORC6 and wt <br />                                                                             cells were arrested in YPG/R with nocodazole and then transferred <br />                                                                             to YPD/nocodazole for 4 h as described in (A). Cultures were <br />                                                                             subsequently released into YPD supplemented with a-factor to <br />                                                                             arrest cells in late G1 phase. Samples of the culture were taken <br />                                                                             after 2 h in a-factor for chromatin binding assays. Thirty microliters <br />       Figure 2 Depletion of Orc6 results in an accumulation of S phase      of the chromatin-associated pellet (P) and 15 ml of the supernatant <br />       cells. (A) An asynchronous GAL1-3HA-ORC6 (DY-36) culture was          (S) were analyzed by immunoblot. HA-tagged Orc6 was detected <br />       grown in YPG/R (2% galactose, 1% raffinose) medium to 106 cells/      using an anti-HA antibody (Roche); anti-Mcm2 (1:500; goat poly- <br />       ml, washed and resuspended in YPD (2% glucose) medium. Whole-         clonal, Santa Cruz) and anti-Orc2 (1:1000; rabbit polyclonal) anti- <br />       cell extracts were prepared from culture aliquots taken prior to      bodies were also used to analyze pellet and supernatant fractions <br />       (GAL) and at the indicated time points following resuspension in      for isogenic strains DY-36 (GAL1-3HA-ORC6) and DY-93 (ORC6- <br />       YPD. A whole-cell extract was also prepared from an asynchronous      3HA). Cdt1 was detected using (anti-Myc; Sigma) for isogenic <br />       exponential culture of an isogenic strain (DY-93) in which Orc6 is    strains DY-83 (CDT1-13Myc) and DY-91 (GAL1-3HA-ORC6, CDT1- <br />       tagged with the same number of HA epitopes, but remains under the     13Myc), while Mcm10 was detected with the same antibodies for <br />       control of its endogenous promoter (wt). Eighty micrograms of each    strains DY-84 (MCM10-13Myc) and DY-92 (GAL1-3HA-ORC6, <br />       extract was used for immunoblot analysis. HA-tagged Orc6 was          MCM10-13Myc). (C) Liquid two-hybrid assays were performed <br />       detected using an anti-HA antibody (Roche) and fluorescent sec-       using pEG-Mcm2, pEG-Cdc6, pEG-Cdt1 and pEG-Mcm10 as baits <br />       ondary antibody (Invitrogen). Ponceau S staining of the region        in combination with each ORC subunit (pJG-ORC1-6) or empty <br />       detected by the blot to verify equal loading of whole-cell extracts   pJG4-6 vector (E) as preys. All two-hybrid results are representative <br />       is also shown. (B) Asynchronous GAL1-3HA-ORC6 (DY-36) and             of two or more assays and error bars represent standard deviation. <br />       isogenic wt (DY-26) cultures were grown as described in (A). Cell <br />       counts of both cultures were taken every 2 h for 8 h following <br />       transfer to YPD. Growth defects were observed in the GAL1-3HA-        therefore appears that the reduction in Mcm2 loading was <br />       ORC6 strain after 4 h, compared to what was seen for wt cells.        specific to the absence of Orc6. <br />       (C) FACS analysis of culture aliquots removed at indicated time <br />       points after transfer to YPD. <br />                                                                             Orc6 interacts with members of the pre-RC involved <br />                                                                             in MCM loading and maintenance <br />       origins (reviewed in Bell and Dutta, 2002). To further explore        To determine whether Orc6 physically interacts with Mcm2 or <br />       the role of Orc6 in rendering origins competent, we moni-             factors that play a role in MCM chromatin association, the <br />       tored Mcm2 loading by means of a chromatin fractionation              full-length MCM2, CDC6, CDT1 and MCM10 coding sequences <br />       assay (Donovan et al, 1997; Liang and Stillman, 1997) using           were cloned into the two-hybrid bait vector pEG-202 <br />       GAL1-ORC6 cells that had been depleted of Orc6 during a G2/           (Ausubel et al, 1995). Each of these constructs was separately <br />       M arrest. Mcm2 association with the chromatin pellet was              transformed into the wt yeast strain DY-1, already harboring <br />       reduced, in Orc6-depleted cells relative to wt cells, following       the lacZ reporter-plasmid pSH18-34, and one of six prey <br />       release into a G1 block with a-factor (Figure 3B). To confirm         plasmids expressing Orc1-6. When Mcm2 was tested as <br />       that the decline of Mcm2 loading following Orc6 depletion             bait, the strongest b-galactosidase signal was obtained with <br />       in G2/M was not the result of ORC destabilization, we also            Orc2, indicating that this ORC subunit and not Orc6, is most <br />       monitored the level of Orc2 to examine the overall state of the       likely to associate directly with Mcm2 (Figure 3C). We next <br />       ORC complex. GAL1-ORC6 cells depleted of Orc6 in G2/M and             evaluated ORC subunit interactions with Cdc6 and Cdt1, <br />       then released into the G1 block had levels of chromatin-              which are required for MCM loading onto chromatin (re- <br />       bound Orc2 comparable to wt cells (Figure 3B), consistent             viewed in Bell and Dutta, 2002). The strongest signal for Cdc6 <br />       with previous in vitro studies indicating Orc6 is not required        was obtained with the Orc1 subunit, consistent with the <br />       for ORC to bind replication origins (Lee and Bell, 1997). It          previously reported Orc1&#226;&#8364;&#8220;Cdc6 interaction in both yeast <br />  <br /> 5152 The EMBO Journal VOL 25 | NO 21 | 2006                                                      &amp; 2006 European Molecular Biology Organization <br />                                                                                                      Orc6 is required for pre-RC maintenance <br />                                                                                                                              JW Semple et al <br />  <br />  <br />  <br /> (Wang et al, 1999) and human cells (Saha et al, 1998). <br /> However, Orc2, Orc5 and Orc6 preys produced positive two- <br /> hybrid signals when co-expressed with the Cdt1 bait, with <br /> Orc6 the highest of the three. Finally, we tested the ORC <br /> subunits against Mcm10, which is required to preserve the <br /> interaction of Mcm2 with chromatin in G1 phase (Homesley <br /> et al, 2000). Here, we noticed a very similar pattern to that <br /> observed for Cdt1, with Orc6 again exhibiting the highest <br /> two-hybrid signal. Since both Cdt1 and Mcm10 interacted <br /> with Orc6, we evaluated the effect of G2/M Orc6 depletion on <br /> their chromatin association in G1. There was no appreciable <br /> effect on the degree chromatin binding of either protein <br /> (Figure 3B). <br />  <br /> Orc6 is required to maintain MCM chromatin <br /> association following pre-RC formation <br /> Studies with Xenopus egg extracts indicate that once the <br /> MCM proteins are assembled onto chromatin, ORC and <br /> Cdc6 can be removed without affecting their putative DNA <br /> helicase function (Hua and Newport, 1998; Rowles et al, <br /> 1999). Therefore, we were interested to determine whether <br /> Orc6 is still required for the efficient initiation of DNA <br /> replication after MCM proteins have been loaded onto chro- <br /> matin. This was carried out by synchronizing cultures of <br /> GAL1-ORC6 and its parental strain growing in YPG/R in late <br /> G1 phase by adding a-factor. Following the initial arrest, cells <br /> were transferred to YPD again with a-factor to deplete Orc6 <br /> in the GAL1-ORC6 strain. After release from the block, an <br /> accumulation of 1C cells was observed for the Orc6-depleted <br /> GAL1-ORC6 culture by 30 min, while the isogenic wt strain <br /> displayed a prominent 2C peak at the same time point, <br /> consistent with a role for Orc6 in S phase progression <br /> (Figure 4A). As an additional control, a GAL1-ORC4 strain <br /> was similarly assessed. Orc4 depletion in this strain occurs <br /> with similar kinetics to what is observed for Orc6 depletion <br /> (Supplementary Figure 2), but in this case cells were able to <br />                                                                    Figure 4 Depletion of Orc6 in late G1 destabilizes MCM association <br /> replicate their DNA, albeit at a slower rate than the wt control   with chromatin and impedes S phase entry (A) wt (DY-26), GAL1- <br /> (Figure 4A). Therefore, it appears that Orc6, as opposed to the    3HA-ORC6 (DY-36) and GAL1-3HA-ORC4 (DY-103) strains were <br /> entire ORC complex, is required in late G1 phase for subse-        grown to 5  106 cells/ml (Exp) and arrested at G1 with a-factor <br />                                                                    (50 mg/ml) for 2&#226;&#8364;&#8220;3 h in YPG/R. The cells were then pelleted, washed <br /> quent DNA replication. Since the most likely explanation for <br />                                                                    with ddH2O and transferred to YPD containing a-factor (50 mg/ml) <br /> a lack of DNA replication following Orc6 depletion in late G1      for 4 h. Cells were subsequently washed and released in YPD <br /> phase was again a destabilization of the MCM complex at            without a-factor Culture aliquots were taken at the indicated <br /> origins, we monitored the chromatin association of Mcm2            times for FACS analysis. (B) Isogenic wt, GAL1-3HA-ORC6 and <br /> prior to and following Orc6 depletion. While equivalent            GAL1-3HA-ORC4 cells were arrested with a-factor in YPG/R, <br />                                                                    washed and incubated in YPD with a-factor, as described in (A); <br /> chromatin pellet to supernatant Mcm2 ratios were observed          DY-109 (ORC4-3HA) was used as the wt control for the experiments <br /> during the initial a-factor arrest between the wt and GAL1-        with GAL1-3HA-ORC4 cells. Chromatin fractionation and immuno- <br /> ORC6 strains (GAL; Figure 4B), there was a clear displace-         blotting was performed as described in Figure 3, with culture <br /> ment of Mcm2 from the pellet to the supernatant fraction           aliquots removed following the initial arrest (GAL), and at the <br />                                                                    end of the YPD/a-factor incubation (GLU). (C) ChIP was carried <br /> following Orc6 depletion in the GAL1-ORC6 strain (GLU;             out with a mixture of Mcm2 and Mcm5 antibodies as described in <br /> Figure 4B). Densitometry indicated that there was close to         Materials and methods, for DY-26 and DY-36 cells cultured as <br /> a 50% reduction in the Mcm2 pellet:supernatant ratio in            described in (A). PCR using sample aliquots taken after sonication <br /> the GAL1-ORC6 strain relative to the isogenic wt strain            (INPUT) and following purification of co-immunoprecipitated DNA <br />                                                                    (IP) was carried out using primers specific for ARS1 origin <br /> (Supplementary Figure 3). Interestingly, we also observed a        sequence, and a region 9 kb upstream of ARS1 (ARS1&#226;&#8364;&#8221;9 kb). <br /> dramatic reduction in the level of chromatin-associated Orc2 <br /> following Orc6 depletion. Although we cannot rule out the <br /> possibility that an overall destabilization of ORC is respon-      chromatin fraction (Randell et al, 2006). In contrast, the high <br /> sible for the displacement of Mcm2, the observation that a         levels of chromatin-bound Mcm10 initially detected were <br /> robust Mcm2 chromatin association is maintained in cells           markedly reduced following Orc6 shut-off (Figure 4B and <br /> depleted of Orc4 (Figure 4B) argues against this. In the case of   Supplementary Figure 3). These results are consistent with a <br /> Cdt1, little difference was observed between cells with nor-       mechanism whereby Orc6 promotes Mcm10 chromatin asso- <br /> mal Orc6 levels or those that had been depleted, although in       ciation, and Mcm10 in turn stabilizes the MCM complex <br /> each case, as expected, very little Cdt1 was detected in the       (Homesley et al, 2000). Finally, we confirmed that MCM <br />  <br /> &amp; 2006 European Molecular Biology Organization                                             The EMBO Journal      VOL 25 | NO 21 | 2006 5153 <br />        Orc6 is required for pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />       proteins are actually displaced from replication origins fol- <br />       lowing late G1 Orc6 depletion, by carrying out ChIP analysis <br />       (Figure 4C), which demonstrated a dramatic drop in the level <br />       of ARS1 DNA co-immunoprecipitating with MCMs when Orc6 <br />       is absent. <br />  <br />       Depletion of Orc6 in late G1 reduces the efficiency of <br />       DNA replication initiation <br />       The inability of cells lacking Orc6 to progress through S phase <br />       may have been due to a reduction of origin firing, inhibition <br />       of elongation, or a combination of both. To investigate the <br />       effect of Orc6 depletion on initiation events, we constructed a <br />       TK &#195;&#190; GAL1-ORC6 strain (DY-79), to allow the incorporation <br />       of BrdU into newly synthesized DNA. Cultures of DY-79 and <br />       its parental TK &#195;&#190; strain (DY-67) were initially synchronized <br />       in late G1 with a-factor in YPG/R, then shifted to YPD with <br />       a-factor, to deplete Orc6 in the DY-79 cells. The cells were <br />       subsequently washed and resuspended in YPD supplemented <br />       with BrdU (0.4 mg/ml) and the ribonucleotide reductase <br />       inhibitor hydroxyurea (HU; 0.2 M), which normally results <br />       in a mid-S phase arrest after about half the replication origins <br />       have fired. Following 90 min in HU, cells were harvested, <br />       genomic DNA was isolated and single molecule DNA comb- <br />       ing was carried out, as previously described (Versini et al, <br />       2003). Comparison of inter-origin distances from GAL1-ORC6 <br />                                                                          Figure 5 Orc6 is required for the initiation of DNA replication <br />       and wt cells revealed that approximately half the number of        following pre-RC formation. (A) Initiation frequency was monitored <br />       origins fired in cells depleted of Orc6 (Figure 5A), consistent    in both wt (DY-67) and GAL1-3HA-ORC6 (DY-79) TK &#195;&#190; cells. Cells <br />       with a role for Orc6 in promoting initiation events. The length    were synchronized in G1 using a-factor and then transferred to YPD <br />       of individual BrdU tracks was more than double those               medium for 4 h to deplete Orc6 while still arrested with the <br />                                                                          pheromone. Cells were then released into YPD containing HU <br />       observed with the wt controls. This can be explained by the        (0.2 M) to block elongation, and BrdU to monitor new strand <br />       fact that nucleotide pools are greatly reduced when HU is          synthesis. DNA plugs were prepared 90 min after the transfer. <br />       administered, thus cells that have fewer origins firing have       FACS profiles of each stage of the experiment are presented in the <br />       more nucleotides available for the migration of each indivi-       bottom left. DNA combing was performed on silanized coverslips as <br />                                                                          described in Materials and methods. Representative DNA fibers <br />       dual replication fork. To examine the possibility that origin      from both the wt and GAL1-3HA-ORC6 strains are shown. BrdU <br />       firing inhibition was caused by Orc6 depletion triggering a G1     incorporation appears green, DNA red. Inter-origin distances as well <br />       checkpoint, we monitored the abundance and phosphoryla-            as BrdU tract lengths following analysis of over 200 origins are <br />       tion status of Rad53 and did not detect upregulation or higher     shown in the histograms, and in each case the average value is <br />                                                                          indicated. Scale bar represents 50 kb. The Mann&#226;&#8364;&#8220;Whitney rank-sum <br />       mobility forms on immunoblots for samples taken either             test was used to show that the differences between wt and GAL1- <br />       immediately following Orc6 depletion or an hour after release      ORC6 cells are significant (Po0.001). (B) wt (DY-93) and GAL1- <br />       from the a-factor block (Supplementary Figure 4).                  3HA-ORC6 (DY-36) strains were grown in YPG/R to 5  106 cells/ml <br />           Whether Orc6 might play a role in the elongation stage         and arrested at S phase with HU (0.2 M). The cells were washed <br />                                                                          with ddH2O and transferred to YPD with HU (0.36 M) for 4 h. Cells <br />       as well as the initiation stage of DNA replication was also        were then washed twice with ddH2O and released into fresh YPD. <br />       investigated. GAL1-ORC6 and isogenic wt cells were arrested        Aliquots of each culture were taken every 20 min for FACS analysis. <br />       in YPG/R supplemented with HU, then washed and resus- <br />       pended in YPD/HU for 4 h. Following release from the HU <br />       block in YPD, the two cultures progressed through S phase at       an essential protein for cell proliferation (Li and Herskowitz, <br />       comparable rates (Figure 5B), suggesting that Orc6 was not         1993), and previous studies suggest that some budding yeast <br />       required for efficient replication fork progression. Although      ORC subunits may have important functions at other cell <br />       we cannot exclude the possibility that residual Orc6 following     cycle stages (Bell et al, 1993; Dillin and Rine, 1998). Recent <br />       our shut-off regime may have played a role in replication fork     attention has focused on additional roles for the Orc6 subunit <br />       progression, we can rule it out as the cause of the S-phase        in mitosis and cytokinesis in metazoans (Prasanth et al, 2002; <br />       arrest we see for this length of depletion (Figure 2).             Chesnokov et al, 2003). It is therefore reasonable to ask <br />                                                                          whether budding yeast Orc6 is required for DNA replication <br />                                                                          and/or other cell cycle events. Our localization of Orc6-YFP/ <br />       Discussion                                                         CFP to subnuclear foci in living cells provides compelling <br />       The essential cell cycle role of Orc6 is specific to S phase       evidence for a replicative function, since it is similar to the <br />       Since its original biochemical purification from budding yeast     punctate pattern previously shown for immunolocalization of <br />       (Bell and Stillman, 1992), ORC has been shown to play a            Orc2 in fixed cells (Pasero et al, 1999), and with the detection <br />       central role in the initiation of DNA replication (reviewed in     of discrete zones of new DNA synthesis in TK &#195;&#190; yeast cells <br />       Bell, 2002). ORC lacking Orc6 is fully competent to bind           following a BrdU pulse (Lengronne et al, 2001). To our <br />       origin sequences, but omission of any other subunit abro-          knowledge, the visualization of Orc6 foci represents the <br />       gates this capability (Lee and Bell, 1997). Orc6 is nevertheless   first detection of a protein constituent of what are thought <br />  <br /> 5154 The EMBO Journal VOL 25 | NO 21 | 2006                                                  &amp; 2006 European Molecular Biology Organization <br />                                                                                                    Orc6 is required for pre-RC maintenance <br />                                                                                                                            JW Semple et al <br />  <br />  <br />  <br /> to be replication factories involving clusters of replication      with the type of large bud morphology often associated with <br /> forks (reviewed in Laskey and Madine, 2003) in living              mutations affecting DNA replication. More recently, the hy- <br /> S. cerevisiae cells. Intriguingly, we have also found that among   perphosphorylation of both Orc6 and Orc2 following START <br /> ORC subunits, both Orc5 and Orc6 are able to homodimerize          was shown to be part of a series of overlapping mechanisms <br /> and thus may conceivably contribute to the formation               that prevent DNA re-replication, which also include Cdc6 <br /> of replication compartments by bridging different origin           degradation and export of Mcm2&#226;&#8364;&#8220;7 from the nucleus (Nguyen <br /> sequences (unpublished data).                                      et al, 2001). All three processes are Clb/Cdc28 (CDK) <br />    In contrast to what has been seen in both human and fruit       mediated, and a direct interaction between Clb5 and Orc6 <br /> fly cells, we observed no accumulation of Orc6 at the mid-         has been reported in budding yeast (Wilmes et al, 2004). <br /> plane of division between mother and daughter cells (Figure        Either <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>mutation</span> of both the Orc2 and Orc6 consensus CDK <br /> 1A and C). Given that the nuclear envelope in budding yeast        phosphorylation sites or abrogation of the Clb5/Orc6 inter- <br /> remains intact throughout the cell cycle, it is possible that      action predisposes cells to re-replication in combination with <br /> Orc6 could play a karyokinetic role that is analogous to the       stabilized Cdc6 and/or constitutively nuclear Mcm2&#226;&#8364;&#8220;7 <br /> cytokinetic function of its metazoan orthologs. Our results        (Nguyen et al, 2001; Wilmes et al, 2004; Green et al, 2006; <br /> argue against this, however, as we do not observe a popula-        Tanny et al, 2006). However, neither change has any effect on <br /> tion of Orc6 at the site of nuclear division in any of the cells   cell cycle progression in the presence of wt Cdc6 and MCM <br /> we observed (Figure 1A and results not shown). Moreover,           proteins. Thus, prevention of re-replication is not the essen- <br /> FACS analysis following depletion of Orc6 from an asynchro-        tial function of Orc6. <br /> nous culture shows no accumulation of cells with X2C DNA              In the present report, we provide strong evidence that Orc6 <br /> content (Figure 2C). The fact that no significant 2C DNA peak      is required for efficient initiation of DNA replication. Cells <br /> was observed even 6 h after Orc6 was depleted suggests that        that were released from a G2/M block during which Orc6 was <br /> chromosome segregation was not compromised as reported             depleted, arrested in the next cell cycle with a 1C DNA <br /> for human cells treated with Orc6 siRNA (Prasanth et al,           content (Figure 3A). How is the lack of Orc6 inhibiting <br /> 2002). Indeed, cells depleted for Orc6 during a G2/M block         DNA replication? One possibility is that the absence of Orc6 <br /> were fully capable of completing mitosis and proceeding into       prevents proper pre-RC assembly, which normally culminates <br /> G1 phase following release (Figure 3A). Although we cannot         in the chromatin loading of the MCM complex. There was a <br /> rule out that residual levels of Orc6 capable of carrying out      modest drop in the level of chromatin-associated Mcm2 after <br /> mitotic and/or cytokinetic functions remain following our          cells depleted of Orc6 in nocodazole were released and <br /> depletion protocol, it is clear from the marked accumulation       subsequently blocked with a-factor (Figure 3B). More strik- <br /> of cells with DNA content between 1C and 2C following Orc6         ing, however, was the strong displacement of previously <br /> depletion in an asynchronous culture, that the cell cycle          loaded Mcm2 by depleting Orc6 during an extended a-factor <br /> arrest we observed under these conditions was primarily            arrest (Figure 4B and C). This pointed to a role for Orc6 in <br /> due to S phase defects.                                            maintaining the origin association of MCM complexes after <br />    Why would metazoan roles for Orc6 in cytokinesis and            they have associated with the pre-RC. Possible mechanisms <br /> mitosis not be present in budding yeast? An important              for how Orc6 facilitates MCM chromatin association were <br /> consideration in answering this question is that S. cerevisiae     suggested by our observations that Orc6 interacts with both <br /> Orc6 shares only low conservation with its Drosophila (5%          Cdt1 and Mcm10, two factors required for stable association <br /> identity, 19% similarity) and human (6% identity, 15%              of Mcm2 with chromatin (Homesley et al, 2000; Devault et al, <br /> similarity) orthologs. Furthermore, sequence comparison of         2002; Tanaka and Diffley, 2002). While there appeared to be <br /> Orc1&#226;&#8364;&#8220;Orc6 from these species suggests that Orc6 evolved            no significant effect on the chromatin levels of either Cdt1 or <br /> more quickly than any of the other ORC subunits (Dhar and          Mcm10 following the G2/M Orc6 depletion, the chromatin <br /> Dutta, 2000). Thus, it appears that metazoan Orc6 has              pellet to supernatant ratio of Mcm10 dropped by approxi- <br /> conserved its original role in DNA replication, while acquir-      mately 50% when Orc6 expression was shut-off in late G1 <br /> ing cytokinetic and, at least in the case of human Orc6,           phase (Figure 4B, Supplementary Figure 3), again suggesting <br /> mitotic functions. It was proposed that Orc6 helps to coordi-      a maintenance role. Interestingly, Mcm10 has also been <br /> nate the processes of DNA replication and chromosome               shown to regulate the chromatin association of DNA poly- <br /> segregation with cytokinesis in metazoans (Prasanth et al,         merase-a (Ricke and Bielinsky, 2004), providing another way <br /> 2002), which might be more critical for multicellular than for     in which its displacement may inhibit DNA replication. In <br /> unicellular organisms. The fact that these additional func-        contrast to what was seen with Orc6 depletion at the earlier <br /> tions are not present in S. cerevisiae makes it an advanta-        time point, Orc2 was displaced from chromatin (Figure 4B). <br /> geous system to further characterize the role of Orc6 in DNA       Thus, a general destabilization of ORC may be responsible for <br /> replication, without the complication of mitotic and cyto-         the MCM displacement, and subsequent replication defects; <br /> kinetic phenotypes.                                                however, the fact that Orc4 depletion neither displaces Mcm2 <br />                                                                    from chromatin (Figure 4B) nor prevents progression through <br /> Orc6 mediates the G1 phase chromatin association                   S phase (Figure 4A) argues against this and points to a role <br /> of Mcm2                                                            for Orc6 that is distinct from other ORC subunits. <br /> To our knowledge, the most substantial evidence of an <br /> essential role for S. cerevisiae Orc6 in DNA replication           Orc6 plays a role in the initiation of DNA replication <br /> comes from the original work of Li and Herskowitz (1993)           after pre-RC formation <br /> in which they found that following sporulation and tetrad          Previous models of pre-RC assembly and function have <br /> dissection of ORC6 &#195;&#190; /  heterozygotes, haploid ORC6                proposed that once MCM proteins have been loaded onto <br /> spores undergo up to two rounds of cell division, arresting        chromatin, ORC is dispensable for DNA replication. <br />  <br /> &amp; 2006 European Molecular Biology Organization                                           The EMBO Journal      VOL 25 | NO 21 | 2006 5155 <br />        Orc6 is required for pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />       Treatment of either budding yeast late G1 chromatin or pre-              Plasmid construction <br />                                                                                Genomic tagging of ORFs was performed by homologous recombi- <br />       RCs assembled on magnetic beads with high salt has been <br />                                                                                nation with linear PCR fragments amplified using plasmid <br />       reported to remove ORC, while MCM proteins remain present                templates. To create an eYFP tagging vector, the DNA sequence <br />       (Donovan et al, 1997; Bowers et al, 2004). In each case,                 encoding the GST tag was removed from pFA6a-GST-KanMX6 <br />       however, there was a reduction in the level of associated                (Longtine et al, 1998) through digestion with PacI and AscI. The <br />                                                                                gene encoding eYFP was amplified from pEYFP (Clontech) using <br />       MCM proteins and the consequences of the ORC removal on <br />                                                                                a High Fidelity PCR Kit (Roche), with forward and reverse primers <br />       DNA replication were not evaluated. Subsequent DNA repli-                incorporating 50 PacI and AscI sites, respectively. Following <br />       cation was determined to be unaffected by salt extraction of             digestion, EYFP was ligated into the pFA6a-KanMX6 vector to <br />       ORC after pre-RC formation in a Xenopus cell-free system,                create pFA6a-eYFP-KanMX6. Similarly, ECFP was amplified from <br />       although the successful removal of only the Orc1 and Orc2                pECFP (Clontech) and cloned as above, creating pFA6a-eCFP-TRP1. <br />                                                                                All fluorescent strains were created by integration of a PCR-based <br />       subunits was monitored (Rowles et al, 1999). In contrast to              cassette amplified from the plasmids created above at the 30 -end of <br />       these results, our data clearly points to a role for Orc6 in             the genes being tagged (Longtine et al, 1998). <br />       promoting initiation events after the pre-RC has formed, since              All bait and prey constructs for two-hybrid analysis were <br />       its depletion in late G1 severely reduced the number of origins          developed by amplifying the entire coding sequence of each gene <br />                                                                                using a High Fidelity PCR Kit (Roche), and cloning the purified <br />       that fired (Figure 5A) and clearly inhibited S phase progres-            products into each vector in frame with the fusion cassette. <br />       sion (Figure 4A). One explanation for varying observations <br />       with budding yeast and Xenopus is that there are simply                  Two-hybrid assays <br />       differences between the two species with respect to the cell             Two-hybrid assays were performed as described previously <br />                                                                                (Ausubel et al, 1995; Duncker et al, 2002). LacZ-reporter (pSH18- <br />       cycle window during which ORC is required to ensure origin               34), bait (derived from pEG202; Ausubel et al, 1995) and prey <br />       firing. Another possibility is that different ORC subunits are           (derived from pJG4-6, a variant of pJG4-5 with a more extensive <br />       required to varying degrees for initiation after pre-RC forma-           polylinker; Gyuris et al, 1993) plasmids were serially transformed <br />       tion. We favor this idea, since we observed that depletion of            into yeast strain DY-1. Exponential cultures were grown in SC (2% <br />                                                                                glucose) medium lacking uracil, histidine and tryptophan to a <br />       Orc4 in late G1 does not result in Mcm2 chromatin displace-              concentration of 5  106 cells/ml, washed with ddH2O and then <br />       ment or prevent cells from traversing S phase, in contrast to            resuspended in SC (2% galactose/1% raffinose) lacking the same <br />       what is seen with Orc6 turnover (Figure 4A and B). A recent              three components for 6 h to induce prey expression. Following <br />       study (Gibson et al, 2006) reported that orc1-161, orc2-1 and            induction, 5  106 cells were harvested and permeabilized. <br />                                                                                The relative strength of interaction was quantified through a <br />       orc5-1 temperature-sensitive mutants each exhibited slower               b-galactosidase assay utilizing the substrate o-nitrophenyl-b-D-galac- <br />       progression through S phase than a wt control, after a shift to          topyranoside (ONPG) (Burke et al, 2000) and the formula: <br />       restrictive temperature in a-factor, and subsequent release at           b-galactosidase activity &#194;&#188; 1000   A420 nm/(t   v   A600 nm) where t is <br />       the same temperature. However, these effects were consider-              the time of reaction (min) and v is the volume of culture used in the <br />                                                                                assay (ml). <br />       ably less severe than we have observed following Orc6 <br />       depletion. A potential late G1 requirement for Orc2 has                  Microscopy <br />       been examined in a number of additional studies, with                    All images were taken with live cells. Prior to imaging cells were <br />       varying outcomes. Strong inhibition of S phase progression               grown to B5  106 cells/ml in SC medium (2% glucose, 0.02% <br />       was observed when an orc2-1 strain was shifted to non-                   adenine), washed in ddH2O and resuspended in fresh medium. <br />                                                                                Cultures were diluted with fresh medium to B1  106 cells/ml, with <br />       permissive temperature in late G1 (Zhang et al, 2002). Late              500 ml added to growth chambers for imaging. Chambers were <br />       G1 Orc2 depletion at permissive temperature using a GAL1-                created using 0.500 ID glass tubing cut to 0.500 and glued to a <br />       orc2-1 strain had little effect on S phase progression in one            standard glass (22   66 mm) coverslip with medical grade silicon <br />       report (Shimada et al, 2002), while another study using the              adhesive. Cells were imaged on a Zeiss Axiovert 100 with a   63, <br />                                                                                1.4 N/A objective lens. To image eCFP and eYFP we used filter sets <br />       same strain found that Orc2 depletion resulted in less efficient         consisting of exciter D436/   20; dichroic 455DCLP; emitter D480/ <br />       DNA replication (Weinberger et al, 2005). A possible expla-              40 m and exciter HQ500/   20; dichroic Q515LP; emitter HQ520LP <br />       nation for the differing outcomes of the two depletion studies           respectively (Chroma Technology, Rockingham, VT). Images were <br />       is that Orc2 removal and/or pre-RC destabilization is more               collected with a Sony SX700 CCD (1024   768) and processed in <br />                                                                                ImageJ 1.30v (NIH, Bethesda). The maximum exposure time with <br />       efficient under one set of conditions used (2 h, 231C) than              this digital camera is only 2 s and as a result imaging of some FP- <br />       the other (1 h, 301C); however, this remains to be tested                tagged proteins under their endogenous promoters resulted in <br />       experimentally.                                                          rather weak signals. To obtain a higher signal, 20 fluorescent images <br />           Clearly, a better understanding of the functions of indivi-          were taken in succession and subsequently summed using ImageJ <br />                                                                                software. Camera gain was adjusted to maximize signal-to-noise <br />       dual ORC components is required to fully characterize the                ratios in individual frames. <br />       multiple roles of this complex in cell cycle progression. For <br />       the first time, we have demonstrated an essential function for           Chromatin binding assay <br />       the Orc6 subunit in promoting DNA replication by mediating               Approximately 2.5  107 cells were harvested at 1000 g and <br />                                                                                spheroplasted as performed previously (Pasero et al, 1999), with <br />       the stability of pre-RCs after their formation. It will now be of <br />                                                                                modifications. Cells were washed once with ddH2O and incubated <br />       considerable interest to determine whether this function is              at 301C for 10 min with gentle mixing in 10 ml/g pre-spheroplasting <br />       conserved among eukaryotes, and to investigate the precise               buffer (100 mM EDTA-KOH (pH 8), 10 mM DTT), followed by <br />       cell cycle roles of other ORC subunits.                                  incubation in 10 ml/g spheroplasting buffer (0.5XYPD, 1.1 M <br />                                                                                sorbitol) containing 0.5 mg/ml Zymolyase 20T (Seikagaku Corp., <br />                                                                                Japan) at 301C for 10&#226;&#8364;&#8220;15 min with gentle mixing. Cells were washed <br />                                                                                once with 20 ml spheroplasting buffer containing 0.5 mM PMSF <br />       Materials and methods                                                    followed by resuspension in 1 ml ice-cold wash buffer (5 mM Tris&#226;&#8364;&#8220; <br />                                                                                HCl (pH 7.4), 20 mM KCl, 2 mM EDTA-KOH (pH 7.4), 1 M sorbitol, <br />       Yeast strains                                                            1% thiodiglycol, 125 mM spermidine, 50 mM spermine) and protease <br />       The genotypes of the strains used in this study are listed in Table I.   inhibitors (0.1 mM benzamidine HCl, 1 mg/ml pepstatin, 2 mg/ml <br />       Unless otherwise indicated, DY-1 was used as our standard wt             antipain, 2 mg/ml leupeptin, 0.5 mM PMSF). Cells were pelleted at <br />       strain.                                                                  400 g for 2 min in a microcentrifuge at 41C, washed twice with 1 ml <br />  <br /> 5156 The EMBO Journal VOL 25 | NO 21 | 2006                                                         &amp; 2006 European Molecular Biology Organization <br />                                                                                                           Orc6 is required for pre-RC maintenance <br />                                                                                                                                   JW Semple et al <br />  <br />  <br />  <br /> Table I Yeast strains used in this study <br /> Strain         Genotype                                                                                                     Source <br />  <br /> DY-1           MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2                        S. Gasser <br />                                                                                                                             Formerly <br />                                                                                                                             (GA1020) <br /> DY-26          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0                                                             ATCC (BY4733) <br /> DY-36          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1                                   This study <br /> DY-39          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-            This study <br />                13Myc/TRP1 <br /> DY-41          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-EYFP/       This study <br />                kanMX6 <br /> DY-65          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-ECFP/       This study <br />                TRP, nup49HNUP49-EYFP/kanMX6 <br /> DY-67          MATa, ura3HURA3[GPD-TK(7x)] ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 GAL psi+                            E Schwob <br />                                                                                                                             Formerly (E1000) <br /> DY-79          MATa ura3HURA3[GPD-TK(7x)] ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 GAL psi+, orc6HPgal1-                This study <br />                3HA-ORC6/TRP1 <br /> DY-81          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc2HORC2-            This study <br />                13Myc/TRP1 <br /> DY-83          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, cdt1HCDT1-13Myc/HIS3                                       This study <br /> DY-84          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, mcm10HMCM10-13Myc/HIS3                                     This study <br /> DY-91          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1, cdt1HCDT1-13Myc/                 This study <br />                HIS3 <br /> DY-92          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1, mcm10HMCM10-                     This study <br />                13Myc/HIS3 <br /> DY-93          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HORC6-3HA/TRP1                                         This study <br /> DY-103         MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc4HPgal1-3HA-ORC4/TRP1                                   This study <br /> DY-109         MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc4HORC4-3HA/TRP1                                         This study <br />  <br />  <br />  <br />  <br /> ice-cold wash buffer, and resuspended in 0.4 ml ice-cold breakage       incubation in YPD with pheromone to deplete Orc6. The cells were <br /> buffer (5 mM Tris&#226;&#8364;&#8220;HCl (pH 7.4), 20 mM KCl, 2 mM EDTA-KOH (pH            released into YPD containing both HU (0.2 M) to stop S phase <br /> 7.4), 0.4 M sorbitol, 1% thiodiglycol, 125 mM spermidine, 50 mM         progression, and BrdU (0.4 mg/ml) to mark nascent DNA synthesis. <br /> spermine) and protease inhibitors as above. Cells were lysed with       After 90 min, genomic DNA plugs (800 ng DNA/plug) were made <br /> 0.5 ml ice-cold breakage buffer containing 2% Triton X-100 and          and stored at 41C until combing. The plugs were initially stained <br /> incubated on ice for 5 min with occasional mixing. The lysed cells      with YOYO-1 (Molecular Probes), and then digested with agarase <br /> were spun at 16 000 g for 5 min in a microcentrifuge at 41C. The        (Roche) and resuspended at 150 ng/ml in 50 mM MES pH 5.7. <br /> chromatin pellet was digested on ice for 10 min in 100 ml ice-cold      Isolated DNA strands were combed on silanized coverslips. <br /> breakage buffer containing 5 mM MgCl2 and 5 ml DNaseI (1 mg/ml).        Appropriate antibodies were used to detect both the DNA molecules <br /> Digestion was stopped by adding EDTA-KOH (pH 7.4) to 10 mM.             (Argene) and BrdU incorporation (Sera Labs). Images were <br />                                                                         captured through a Leica DM6000B microscope with a CoolSNAP <br /> Chromatin immunoprecipitation assay                                     HQ CCD camera (Roper). Analysis of track lengths was performed <br /> Chromatin immunoprecipitation (ChIP) was performed as pre-              using MetaMorph (Universal Imaging Corp.). Adenovirus DNA was <br /> viously described (Tanaka et al, 1997) with modifications. Cross-       used as a standard to convert distances to <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairs. <br /> linking was carried out with 1% formaldehyde for 20 min at 301C <br /> and the reaction quenched with 125 mM glycine. Cells were             <br /> harvested, washed with ice-cold PBS and the pellet was snap             <br /> frozen in liquid nitrogen and stored at  801C. Cell lysis was         <br /> performed with glass beads in 500 ml lysis buffer containing <br /> protease inhibitors. The soluble fraction of the whole-cell extract <br /> was discarded and the insoluble pellet was resuspended in 500 ml       <br /> lysis buffer containing protease inhibitors. This extract was <br /> sonicated five times for 25 s at 5&#226;&#8364;&#8220;6 W to achieve an average DNA     <br /> fragment size of 500 bp using a MicrosonTM XL 2000 (Misonix).           <br /> Samples were spun down twice for 2 min at 4500 g to remove cell         <br /> debris. Immunoprecipitation was carried out with 30 ml anti-goat        <br /> IgG agarose beads (Sigma, A9294), which were preincubated with          <br /> 2 mg each of anti-Mcm2 (Santa Cruz, sc-6680) and anti-Mcm5 (Santa       <br /> Cruz, sc-6686) antibodies. Samples were incubated overnight at         <br /> 41C on a rotator.                                                       <br />                                                                    <br /> DNA combing                                                             <br /> DNA combing was performed as in Versini et al (2003). In brief, wt   <br /> and GAL1-ORC6 cultures were released synchronously from G1               <br /> following an initial arrest with a-factor and subsequent 4 h          <br />  <br />  <br />  <br />  <br /> </body></html>