<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />   A Novel Functional Domain of Cdc15 Kinase Is Required for Its Interaction <br />                 With Tem1 GTPase in Saccharomyces cerevisiae <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Exit <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> mitosis requires the inactivation of cyclin-dependent kinase (CDK) activity. In the budding <br />              yeast Saccharomyces cerevisiae, a number of gene products have been identified as components of the signal <br />              transduction network regulating inactivation of CDK (called the MEN, for the mitotic exit network). <br />              Cdc15, one of such components of the MEN, is an essential protein kinase. By the two-hybrid screening, <br />              we identified Cdc15 as a binding protein of Tem1 GTPase, another essential regulator of the MEN. <br />              Coprecipitation experiments revealed that Tem1 binds <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> Cdc15 in vivo. By deletion analysis, we found <br />              that the Tem1-binding domain resides near the conserved kinase domain of Cdc15. The cdc15-LF <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span>, <br />              which was introduced into the Tem1-binding domain, reduced the interaction with Cdc15 and Tem1 and <br />              caused temperature-sensitive growth.The kinase activity of Cdc15 was not so much affected by the cdc15- <br />              LF mutation. However, Cdc15-LF failed to localize to the SPB at the restrictive temperature. Our data <br />              show that the interaction with Tem1 is important for the function of Cdc15 and that Cdc15 and Tem1 <br />              function in a complex to direct the exit from mitosis. <br />  <br />  <br />  <br /> E    UKARYOTIC cell cycle is governed by the oscilla- <br />        tion of cyclin-dependent kinase (CDK) activity. The <br /> cell cycle progression from anaphase to G1 phase, when <br />                                                                          is an APC regulator for mitotic cyclin destruction <br />                                                                          (Charles et al. 1998; Shirayama et al. 1998), consistent <br />                                                                          with the report that its mammalian homologue Plk di- <br /> two new cells are generated by division, is one of the                   rectly phosphorylates and activates APC (Kotani et al. <br /> most important steps for the faithful inheritance of ge-                 1998). Dbf2 functions in mitotic exit with its binding <br /> netic information. While entry into mitosis is initiated                 partner Mob1 (Komarnitsky et al. 1998). Cdc14 func- <br /> by activation of the mitotic cyclin/CDK complex, exit                    tions as a key regulator for the exit from mitosis. Cdh1, <br /> from mitosis is promoted by inactivation of it. One way                  a specificity factor of APCCdh1, is dephosphorylated by <br /> for CDK to be inactivated at mitotic exit is the destruc-                Cdc14 at late anaphase and activates APCCdh1, there- <br /> tion of mitotic cyclins. It depends on a multisubunit                    by promoting the ubiquitination of mitotic cyclins <br /> ubiquitin ligase called the anaphase promoting complex                   (Schwab et al. 1997; Visintin et al. 1997; Zachariae et <br /> (APC or cyclosome), which catalyzes ubiquitination of                    al. 1998; Jaspersen et al. 1999). The expression of the <br /> cyclins (King et al. 1995; Sudakin et al. 1995; Zachariae                CDK inhibitor Sic1 is stimulated by the transcription <br /> et al. 1996). The resulting multiubiquitinated cyclins are               factor Swi5 (Knapp et al. 1996; Toyn et al. 1997), which <br /> recognized and degraded by the 26S proteasome. An                        is allowed to enter the nucleus by dephosphorylation <br /> alternative way for CDK inactivation is the attachment                   of phosphorylated Swi5 by Cdc14 (Visintin et al. 1998). <br /> of the CDK inhibitor Sic1 (Mendenhall 1993; Schwob                       Cdc14 also dephosphorylates phosphorylated Sic1 to <br /> et al. 1994), whose expression dramatically increases at                 protect it from degradation (Visintin et al. 1998), while <br /> late anaphase and persists until late G1 phase.                          the phosphorylation of Sic1 by CDK targets Sic1 for <br />    In budding yeast, a group of genes essential for the                  ubiquitin-dependent degradation (Feldman et al. 1997; <br /> exit from mitosis were isolated by various genetic                       Verma et al. 1997; Nishizawa et al. 1998). <br /> screens. Such genes including CDC5, CDC14, CDC15,                           Recent studies about the MEN components pointed <br /> DBF2, LTE1, MOB1, and TEM1 are components of the                         out the importance of their subcelullar localization in <br /> mitotic exit network (MEN) regulating mitotic cyclin/                    their function in mitotic exit. During G1 phase through <br /> CDK inactivation (Johnston et al. 1990; Schweitzer                       metaphase/anaphase transition, Cdc14 is localized and <br /> and Philippsen 1991; Kitada et al. 1993; Shirayama <br />                                                                          inhibited at the nucleolus by being anchored by Net1, <br /> et al. 1994a,b; Luca and Winey 1998; Morgan 1999). <br />                                                                          a component of the nucleolar RENT complex (Shou <br /> Cdc5, a member of the conserved family of Polo-kinase, <br />                                                                          et al. 1999; Straight et al. 1999; Visintin et al. 1999). <br />                                                                          At late anaphase, Cdc14 is released from the nucleolus <br />                                                                          and dispersed to the entire nucleus and cytoplasm, <br />   Corresponding author: Akio Toh-e, Department of Biological Sciences, <br /> Graduate School of Science, The University of Tokyo, 7-3-1 Hongo,        where it encounters its targets, including Cdh1, Sic1, <br /> Tokyo 113-0033, Japan. E-mail: toh-e@biol.s.u-tokyo.ac.jp                and Swi5. The release of Cdc14 from the nucleolus does <br /> Genetics 157: 1437&#226;&#8364;&#8220;1450 (April 2001) <br />  1438                                                     K. Asakawa et al. <br />  <br /> not occur in the absence of the function of Cdc5, Cdc15,            crose were replaced with glucose in SC. 5-Fluoroorotic acid <br /> Dbf2, or Tem1 (Shou et al. 1999; Visintin et al. 1999),             (5-FOA, 0.1% w/v) was added to appropriate medium for the <br />                                                                     plasmid shuffling assay. Yeast transformations were performed <br /> suggesting that these factors function in the regulation            by the method of Ito et al. (1983), and other standard yeast <br /> of Cdc14 localization. Thus, the release of Cdc14 from              genetic manipulations were performed as described by Sher- <br /> the nucleolus is regarded as the trigger of mitotic exit.           man et al. (1986). The Escherichia coli strain used is DH5 <br /> Either of Cdc5, Cdc15, Dbf2, or Tem1, at least its frac-            &#226;&#65533;&#163;[supE44 &#226;&#338;&#172;lacU169 (&#207;&#8224;80lacZ &#226;&#338;&#172; m15) hsdR17 recA1 endA1 gyrA96 <br /> tion, localized to the spindle pole body (SPB; Shira-               thi-1 relA1]. <br />                                                                        TEM1 plasmids: Site-directed mutagenesis of TEM1 was car- <br /> yama et al. 1998; Cenamor et al. 1999; Bardin et al.                ried out as follows. The BamHI-SalI fragment containing the <br /> 2000; Frenz et al. 2000; Pereira et al. 2000; Song et al.           open reading frame (ORF) of TEM1 was amplified by PCR <br /> 2000; Xu et al. 2000), indicating that the SPB takes an             using a pair of synthetic oligonucleotides TEM1#3 (5&#226;&#172;&#732;- <br /> important role for mitotic exit. However, the molecular             CCGGATCCATGGCTACACCAAGCAC-3&#226;&#172;&#732;) and TEM1#4 (5&#226;&#172;&#732;- <br /> mechanisms by which Cdc5, Cdc15, Dbf2, and Tem1                     TGCGGTCGACTCATGTATTAACGCCCGG-3&#226;&#172;&#732;) and cloned <br />                                                                     between the BamHI site and the SalI site of pBluescript KS&#226;&#171;&#185; <br /> regulate Cdc14 localization remain elusive. The ana-                (Stratagene, La Jolla, CA) generating pKZ126. To construct <br /> phase inhibitor Pds1 (Cohen-Fix et al. 1996; Yamamoto               the tem1T34A mutant gene, the 34th codon (ACA for Thr) <br /> et al. 1996a,b) and Esp1, which promotes sister chroma-             of TEM1 was replaced with GCA (for Ala) by PCR-aided site- <br /> tid separation (Ciosk et al. 1998; Uhlmann et al. 1999),            directed mutagenesis by using TEM1#3, TEM1#4, and a pair <br /> also regulate the release of Cdc14 and CDK inactivation             of complementary oligonucleotides as mutagenic primers (5&#226;&#172;&#732;- <br />                                                                     GTAGGGAAAGCATCGCTG-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-CAGCGATGCTTTCC <br /> at mitotic exit (Cohen-Fix and Koshland 1999; Shira-                CTAC-3&#226;&#172;&#732;). pKZ126 was used as a template. The tem1T52A <br /> yama et al. 1999; Tinker-Kulberg and Morgan 1999).                  mutant gene (in which the 52nd codon ACG for Thr was <br />    These components of the MEN display various genetic              changed to GCG for Ala) was constructed in the same way as <br /> interactions and consist of a complicated signal trans-             the tem1T34A mutagenesis using a pair of oligonucleotides <br /> duction network (summarized in Jaspersen et al. 1998).              (5&#226;&#172;&#732;-AGGAATACACACAGGCGCTGGGAGTGAACT-3&#226;&#172;&#732; and 5&#226;&#172;&#732;- <br />                                                                     AGTTCACTCCCAGCGCCTGTGTGTATTCCT-3&#226;&#172;&#732;) as muta- <br /> To understand how a cell exits mitosis, it is very impor-           genic primers. To generate the expression plasmids of tem1 <br /> tant to characterize the molecular interaction of these             mutant genes, the BamHI site and the SalI site were introduced <br /> MEN components. CDC15 was isolated as the responsi-                 just in front of the start codon and right after the stop codon <br /> ble gene of the extragenic suppressor of &#226;&#338;&#172;lte1, the wild            of TEM1, respectively, in pKZ134 containing the SacI&#226;&#171;&#186;293- <br /> type of which encodes a protein containing a highly                 PvuII&#226;&#171;&#185;1525 fragment (for the sequence coordinate, see below) <br />                                                                     of TEM1 in YCplac111 (Gietz and Sugino 1988). The ORF <br /> conserved motif of the guanine nucleotide exchange                  of wild-type TEM1 was replaced by either the BamHI-SalI frag- <br /> factor (Shirayama et al. 1994b, 1996). TEM1 encodes                 ment of tem1T34A or that of tem1T52A generating pKZ154 or <br /> a small GTPase originally isolated as a multicopy sup-              pKZ141, respectively. Plasmids for the expression of tagged <br /> pressor of &#226;&#338;&#172;lte1 (Shirayama et al. 1994a). The overpro-             Tem1 or its derivatives were constructed as follows. The frag- <br /> duction of Cdc15 is able to suppress the lethality of the           ment containing the promoter region (300 bp) and the ORF <br />                                                                     of TEM1 or mutant tem1 was introduced into pTS903CL, <br /> gene disruption of TEM1 (Shirayama et al. 1994a), <br />                                                                     pTS902CL, pTS905CL, or pTS910CU carrying the fragment <br /> indicating that a cell is able to grow in the absence of            encoding two hemagglutinin (HA) epitopes and six histidines <br /> Tem1 if the Cdc15 function is elevated. These results               tandemly (pTS903CL), two HA epitopes (pTS902CL), nine <br /> suggest that the Cdc15/Tem1 pathway control the exit                myc epitopes and six histidines tandemly (pTS905CL), or a <br /> from mitosis (Shirayama et al. 1994a). Indeed, Bardin               green fluorescent protein (GFP; pTS910CU) followed by the <br /> et al. (2000) demonstrated that Tem1 and Cdc15 are in               300-bp TDH3 terminator. The expression plasmid of TEM1- <br />                                                                     HA2, TEM1-HA2His6, tem1T34A-HA2His6, or tem1T52A-HA2His6 <br /> a complex during mitosis.                                           was designated as pKZ178, pKZ173, pKZ181, or pKZ180, respec- <br />    In this study, we isolated Cdc15 as a binding protein            tively. The expression plasmid of TEM1-myc9His6 was designated <br /> of Tem1 by two-hybrid screening and showed that Tem1                as pKZ174. The expression plasmid of TEM1-GFP, tem1T34A- <br /> physically binds to Cdc15. We also determined the novel             GFP, tem1T52A-GFP, or tem1-3-GFP was designated as pTGCU1, <br /> functional domain of Cdc15 that is required for the                 pT34G, pT52G, or pT135G, respectively. For two-hybrid assays, <br />                                                                     the BamHI-SalI fragment of the ORF of wild-type TEM1, <br /> interaction with Tem1. Using various mutants of CDC15               tem1T34A, tem1T52A, or tem1-3 was introduced between the <br /> or TEM1, we investgated the interaction between Cdc15               BamHI site and the SalI site of pGBDUC1 ( James et al. 1996) <br /> and Tem1. Our results indicate that Cdc15 and Tem1                  generating pKZ104, pKZ158, pKZ142, or pKZ131, respectively. <br /> function as a complex to direct the exit from mitosis.              The DNA sequence of each mutant tem1 gene (and of each <br />                                                                     mutant cdc15 gene, see below) was determined by the method <br />                                                                     of Sanger et al. (1977) using Thermo Sequenase dye termina- <br />               MATERIALS AND METHODS                                 tor cycle sequencing premix kit (Amersham Life Science, Inc., <br />                                                                     Cleveland) and the ABI 373A DNA sequencer (Applied Biosys- <br />    Microbial manipulation: The principal yeast strains used in      tems, Foster City, CA). Number &#226;&#171;&#185;1 indicates the position of <br /> this study are listed in Table 1. Strains derived from them         the adenine residue of the start codon. <br /> were also used as described in the text. Yeast cells were grown        CDC15 plasmids: Site-directed mutagenesis of CDC15 was <br /> either in rich medium (YPD) consisting of yeast extract (Difco,     carried out as follows. The PvuII&#226;&#171;&#186;410-PvuII&#226;&#171;&#185;3637 fragment con- <br /> Detroit), peptone (Nihon Seiyaku, Tokyo), and glucose or            taining the CDC15 gene was cloned into PvuII-digested YCp <br /> in synthetic glucose medium (SC), which is SD containing            lac111 generating pKZ412. To construct the cdc15-DD gene, <br /> appropriate auxotrophic supplements (Sherman et al. 1986).          the 298th codon (GAU for Asp) and the 300th codon (GAU <br /> For synthetic galactose medium, 5% galactose and 0.2% su-           for Asp) of CDC15 were replaced with GGU (for Gly) and GGC <br />                                          Tem1 GTPase-Binding Domain of Cdc15 Kinase                                              1439 <br />  <br />                                                              TABLE 1 <br />                                                   Yeast strains used in this study <br />  <br /> Strain                                      Relevant genotype                                             Reference or source <br /> E0096            MAT&#226;&#65533;&#163; ura3 leu2 his3 trp1 tem1::HIS3 PGAL7 -TEM1-URA3                               Shirayama et al. (1994a) <br /> YKZ0023          MATa/MAT&#226;&#65533;&#163; ade2-1/ade2-1 ura3/ura3 leu2-3, 12/leu2-3, 12 his3-11,                   This study <br />                     15/his3-11, 15 trp1-1/trp1-1 can 1-100/can1-100 <br /> YKZ0024          MATa ade2-1 ura3 leu2-3, 12 his3-11, 15 trp1-1 can1-100                            This study <br /> YKZ0181          MATa tem1-3                                                                        This study <br /> YKZ0200          MATa cdc15-2                                                                       This study <br /> YKZ0237          MATa/MAT&#226;&#65533;&#163; CDC15/cdc15::HIS3                                                        This study <br /> YKZ0255          MATa cdc15::HIS3 [pKZ406]                                                          This study <br /> YKZ0376          MAT&#226;&#65533;&#163; cdc15-LF                                                                      This study <br /> YKZ0454          MAT&#226;&#65533;&#163; tem1::URA3 [pKZ174]                                                           This study <br /> SLJ511           MATa cdc15::HIS3 leu2::CDC15-HA3                                                   Jaspersen and Morgan (2000) <br /> PJ69-4A          MATa trp1-901 leu2-3, 112 ura3-52 his3-200 &#226;&#338;&#172; gal4 &#226;&#338;&#172; gal80                          James et al. (1996) <br />                   LYS2::GAL1-HIS3 GAL2-ADE2 met2::GAL7-lacZ <br />   All strains listed above, except for E0096 and PJ69-4A, are in the W303 background. E0096 is a strain with a RAY background. <br />  <br />  <br /> (for Gly), respectively, by PCR-aided site-directed mutagenesis      containing the CDC15 gene was cloned into the PvuII-digested <br /> using CDC15-77as (5&#226;&#172;&#732;-GATCTGCCTCACAAAAGG-3&#226;&#172;&#732;), CDC                     pBluescript KS&#226;&#171;&#185; and the part of the CDC15 fragment (from <br /> 15#4 (5&#226;&#172;&#732;-TCCCCCGGGGTTTGAAATGTTGTAATGG-3&#226;&#172;&#732;), and a                     the XhoI&#226;&#171;&#186;57 site to the EcoRI&#226;&#171;&#185;2715 site) was replaced by the HIS3 <br /> pair of complementary oligonucleotides as mutagenic primers          gene, generating pFM418. TS902CL, pTS903CL, pTS905CL, <br /> (5&#226;&#172;&#732;-TATCATTGGGGTGCCGGCTTTCAAGAAG-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-                          pTS910CU, and pTS910EU were gifts from T. Sasaki (Depart- <br /> CTTCTTGAAAGCCGGCACCCCAATGATA-3&#226;&#172;&#732;). pKZ412 was                         ment of Biological Sciences, Graduate School of Science, Uni- <br /> used as a template. For creating the cdc15-S309A gene, in            versity of Tokyo). <br /> which the 309th codon UCA for Ser was changed to GCA for                Disruption of CDC15: The PvuII-PvuII fragment of pFM418 <br /> Ala, a pair of complementary oligonucleotides (5&#226;&#172;&#732;-CTAAATA            was introduced to YKZ0023 to obtain a His&#226;&#171;&#185; cell of which one <br /> TAGCACCCTCTAAA-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-TTTAGAGGGTGCTATATT                          of the CDC15 alleles was replaced by the HIS3 gene (YKZ0237). <br /> TAG-3&#226;&#172;&#732;) was used as mutagenic primers. For creating the cdc15-       The gene disruption of CDC15 was confirmed by PCR. For <br /> LF gene, in which the 357th codon CUU for Leu and the                the plasmid shuffling assay, we obtained His&#226;&#171;&#185; Ura&#226;&#171;&#185; progeny <br /> 358th codon UUC for Phe were changed to CGU for Arg                  (YKZ0255) from tetrads derived from YKZ0237 harboring <br /> and CGC for Arg, respectively, the pair of complementary             pKZ406, which is the centromeric plasmid carrying the CDC15 <br /> oligonucleotides (5&#226;&#172;&#732;-CTTGCATGTGCGTCGCAGTGTTTGC-                      gene and the URA3 gene. <br /> 3&#226;&#172;&#732; and 5&#226;&#172;&#732;-GCAAACACTGCGACGCACATGCAAG-3&#226;&#172;&#732;) was used                        Protein extraction and immunoblotting: Cells grown to mid- <br /> as mutagenic primers. For creating the cdc15-lyt1 gene (the          log phase in 5 ml of appropriate medium were washed with <br /> 410th codon GGA for Gly was changed to GAA for Glu; Jime-            lysis buffer [100 mm Tris-HCl, pH 7.5, 200 mm NaCl, 1 mm <br /> nez et al. 1998), a pair of complementary oligonucleotides           EDTA, 5% glycerol, 0.5 mm dithiothreitol (DTT), 1 mm phe- <br /> (5&#226;&#172;&#732;-TATGGGAGAAATTCCACTG-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-CTGTGGAATTT                        nylmethylsulfonyl fluoride (PMSF), 1 &#226;&#65533;&#174;g/ml pepstatin, 1 &#226;&#65533;&#174;g/ <br /> CTCCCATA-3&#226;&#172;&#732;) was used as mutagenic primers. The SalI&#226;&#171;&#185;838-            ml leupeptin, 1 &#226;&#65533;&#174;g/ml antipain, 1 &#226;&#65533;&#174;g/ml aprotinin], resus- <br /> XhoI&#226;&#171;&#185;1084 fragment of the CDC15 gene on pKZ412 was replaced          pended in 30 &#226;&#65533;&#174;l of lysis buffer, and vortexed for 10 min at 4&#226;&#172;&#353; <br /> with the corresponding SalI-XhoI fragment containing each of         with acid-washed glass beads (&#207;&#8224; &#226;&#171;&#189; 0.5 mm). Lysis buffer (100 <br /> the mutations (cdc15-DD, cdc15-S309A, or cdc15-LF ) generating       &#226;&#65533;&#174;l) was added and the extract was clarified by 10-min spins <br /> pKZ434, pKZ436, or pFM420, respectively. The SalI&#226;&#171;&#185;838-               at 14,000 rpm (Sakuma M150-IV, Tokyo) at 4&#226;&#172;&#353;. The concentra- <br /> NcoI&#226;&#171;&#185;1504 fragment of the CDC15 gene on pKZ412 was replaced          tion of protein in the extract was determined by BCA Protein <br /> with the corresponding SalI-NcoI fragment containing the             Assay Reagent (Pierce, Rockford, IL) using bovine serum albu- <br /> cdc15-lyt1 mutation generating pFM414. The cdc15d82SX was            min (BSA) as a standard. For immunoblots, samples were <br /> constructed by deleting the SalI&#226;&#171;&#185;838-XhoI&#226;&#171;&#185;1084 fragment from         boiled in SDS gel sample buffer (Laemmli 1970) and sepa- <br /> CDC15 and religating the cohesive ends generated by the              rated by SDS-polyacrylamide gel electrophoresis (SDS-PAGE). <br /> SalI and the XhoI digestion. The expression plasmid carrying         Proteins were electrophoretically transferred to nitrocellulose <br /> CDC15-HA5 or CDC15-GFP was constructed as follows. The               membrane followed by Western blotting using mouse mono- <br /> fragment containing the promoter region (400 bp) and the             clonal anti-HA antibody 16B12 (BAbCO, Richmond, CA) or <br /> ORF of CDC15 were introduced into pTS901CU or pTS910EU,              mouse monoclonal anti-myc antibody 9E10 (Calbiochem, <br /> carrying the fragment encoding five HA epitopes or a GFP             Cambridge, MA) as primary antibody. As secondary antibody, <br /> followed by the 300-bp TDH3 terminator, generating pKZ416            anti-mouse IgG goat antibody conjugated to alkaline phospha- <br /> or pCFG1, respectively. The fragment encoding Cdc15-HA5              tase [anti-mouse IgG(H&#226;&#171;&#185;L) AP conjugate, Promega, Madison, <br /> was introduced into multicopy plasmid YEplac195 (Gietz and           WI] or to horseradish peroxidase [anti-mouse IgG(H&#226;&#171;&#185;L) HRP <br /> Sugino 1988) or centromeric plasmid YCplac22 (Gietz and              conjugate, Promega] was used. For detection of alkaline phos- <br /> Sugino 1988) generating pKZ425 or pKZ438, respectively.              phatase, BCIP/NBT color substrate (Promega) was used. For <br /> The expression plasmids of mutant Cdc15 proteins were con-           detection of horseradish peroxidase, Western blot chemilumi- <br /> structed by replacing the BglII&#226;&#171;&#186;95-BamHI&#226;&#171;&#185;2592 fragment of            nescence reagent (NEN Life Science Products, Boston) was <br /> CDC15 on pKZ425, pKZ438, or pCFG1 with the corresponding             used. <br /> BglII-BamHI fragment containing each of the cdc15 mutations.            &#226;&#65533;&#164;-Galactosidase assay: PJ69-4A cells expressing both GBD- <br /> For the disruption of CDC15, the PvuII&#226;&#171;&#186;410-PvuII&#226;&#171;&#185;3637 fragment       fusion protein and GAD-fusion protein were grown in 3 ml <br />  1440                                                      K. Asakawa et al. <br />  <br /> of SC-(URA, LEU) at 25&#226;&#172;&#353; to mid-log phase and harvested. Cells        used as a fluorogenic substrate of alkaline phosphatase. Fluo- <br /> washed once with Z buffer (Miller 1972) were suspended in            rescence was detected by using Fluoro Image Analyzer (FLA- <br /> 30 &#226;&#65533;&#174;l of Z buffer and vortexed with acid-washed glass beads          2000, Fuji, Tokyo). For the kinase assay, the immunoprecipi- <br /> for 10 min at 4&#226;&#172;&#353;. Z buffer (100 &#226;&#65533;&#174;l) was added to the extract         tants were washed three times with 300 &#226;&#65533;&#174;l of LLB and once <br /> followed by 10-min spins at 14,000 rpm at 4&#226;&#172;&#353;. Fifty microliters      with 300 &#226;&#65533;&#174;l of kinase buffer (50 mm HEPES-NaOH, pH 7.4, <br /> of extract was diluted to 1 ml with Z buffer. Then, 200 &#226;&#65533;&#174;l of        5 mm MgCl2, 2.5 mm MnCl2, 5 mm &#226;&#65533;&#164;-gylcerophosphate, and <br /> 4 mg/ml o-nitrophenyl &#226;&#65533;&#164;-d-galactopyranoside was added and            1 mm DTT) and incubated at either 25&#226;&#172;&#353; or 37&#226;&#172;&#353; for 10 min in <br /> samples were incubated at 30&#226;&#172;&#353; until a pale yellow color devel-       20 &#226;&#65533;&#174;l of kinase buffer containing 20 &#226;&#65533;&#174;m ATP, 2 &#226;&#65533;&#174;g myelin basic <br /> oped. The reactions were stopped by adding 500 &#226;&#65533;&#174;l of 1 m             protein (MBP), and 5 &#226;&#65533;&#174;Ci of [&#226;&#65533;&#165;-32P]ATP (3000 Ci/mmol). <br /> Na2CO3. Units were calculated with the following equation:           Reaction was stopped by boiling in SDS gel sample buffer and <br /> U &#226;&#171;&#189; 1.7 &#226;&#171;&#187; OD420/{0.0045 &#226;&#171;&#187; protein (mg) &#226;&#171;&#187; t (min)}, where             products were analyzed on a 12% polyacrylamide gel followed <br /> t &#226;&#171;&#189; time of reaction (Adams et al. 1997). Amounts of protein         by autoradiography using Bio Imaging Analyzer (Fujix BAS- <br /> in the extract were determined by a Bio-Rad protein assay kit        1000, Fuji, Tokyo). <br /> (Bio-Rad, Hercules, CA) using BSA as a standard.                        Microscopic analysis: For detection of GFP fluorescence, <br />    Coprecipitation experiments: For His6-tagged Tem1 precip-         cells expressing GFP-tagged proteins grown at 25&#226;&#172;&#353; were har- <br /> itation, cell extracts were prepared as described above. Protein     vested, washed three times with phosphate-buffered saline, <br /> (1 mg in 500 &#226;&#65533;&#174;l of lysis buffer) was incubated at 4&#226;&#172;&#353; for 1 hr        and immediately subjected to microscopic analysis with an <br /> with 20 &#226;&#65533;&#174;l of 50% Ni-NTA beads (QIAGEN, Valencia, CA)                epifluorescence microscope Olympus IX70 (Olympus, Tokyo) <br /> that had been washed with wash buffer (50 mm Tris-HCl, pH            with a lens UPlanApo100x (Olympus) and a CCD camera <br /> 7.5, 50 mm NaCl, 20 mm imidazole, 1 mm PMSF, 1 &#226;&#65533;&#174;g/ml                 (SENSYS, Photometrics, Tucson, AZ). Cells grown at 37&#226;&#172;&#353; were <br /> pepstatin, 1 &#226;&#65533;&#174;g/ml leupeptin, 1 &#226;&#65533;&#174;g/ml antipain, 1 &#226;&#65533;&#174;g/ml              fixed with 3.7% formaldehyde for 10 min at 37&#226;&#172;&#353;, washed three <br /> aprotinin). Beads were washed with 500 &#226;&#65533;&#174;l of wash buffer five        times with phosphate-buffered saline, and subjected to micro- <br /> times and then with 50 &#226;&#65533;&#174;l of elution buffer (wash buffer             scopic analysis as above. <br /> containing 100 mm imidazole) twice. Twenty microliters of <br /> extract before or after incubation with Ni-NTA beads and 20 <br /> &#226;&#65533;&#174;l of the eluate were boiled in SDS gel sample buffer and <br /> loaded on a 10% polyacrylamide gel. For Tem1myc9His6 pre-                                     RESULTS <br /> cipitation, cell extracts were prepared as described above using <br />                                                                         Isolation of Cdc15 as a Tem1-binding protein: To <br /> lysis buffer containing 0.1% NP-40. Protein (1 mg in 500 &#226;&#65533;&#174;l <br /> of lysis buffer) was incubated with anti-myc antibody (1 &#226;&#65533;&#174;g of       isolate Tem1-binding proteins, we performed two- <br /> 9E10) at 4&#226;&#172;&#353; for 1 hr followed by another 1-hr incubation with        hybrid screening using Tem1 as bait. To generate <br /> 60 &#226;&#65533;&#174;l of 50% (v/v) protein A-Sepharose (Amersham, Uppsala,           pKZ104, wild-type TEM1 was fused to the coding region <br /> Sweden) that had been swollen with lysis buffer. Protein             of the Gal4-DNA binding domain on pGBDUC1 ( James <br /> A-Sepharose-associated immunoprecipitates were washed with <br />                                                                      et al. 1996). The library containing genomic DNA fused <br /> lysis buffer, boiled in SDS gel sample buffer, and loaded on <br /> polyacrylamide gels. For detection of Tem1myc9His6 by West-          to the coding region of the Gal4-transcription activation <br /> ern blotting, rabbit polyclonal anti-myc antibody [c-MYC (A-         domain (GAD; Y2HL, James et al. 1996) was introduced <br /> 14)-G, Santa Cruz Biotechnology, Inc.] was used as primary           into PJ69-4A cells (James et al. 1996) carrying pKZ104. <br /> antibody and mouse monoclonal anti-rabbit IgG antibody con-          We isolated 94 His&#226;&#171;&#185;Ade&#226;&#171;&#185; clones out of &#226;&#181;&#8216;96,000 trans- <br /> jugated to horseradish peroxidase (RG-16, Sigma, St. Louis) <br />                                                                      formants and rescued library plasmid from each of the <br /> was used as secondary antibody. Preparation of cell extracts <br /> and immunoprecipitation of Cdc15-HA3 was performed as                His&#226;&#171;&#185;Ade&#226;&#171;&#185; clones. DNA sequencing at the cloning junc- <br /> described in Bardin et al. (2000) using 16B12 antibody. HA-          tion revealed that the cloned fragments were classified <br /> tagged, myc-tagged, or GFP-tagged proteins were detected             into 34 genes. Uetz et al. (2000) also performed the <br /> by Western blotting using 16B12, 9E10, or mouse anti-GFP             two-hybrid screening using Tem1 as bait and isolated <br /> monoclonal antibody (mixture of clones 7.1 and 13.1, Boeh-           24 genes. Surprisingly, none of the 34 genes that we <br /> ringer Mannheim, Indianapolis), respectively, as primary anti- <br /> body and anti-mouse IgG goat antibody conjugated to horse-           isolated was found in the collection of Uetz et al. (2000). <br /> radish peroxidase or anti-mouse IgG goat antibody conjugated            One of such isolates was found to contain a part of <br /> to alkaline phosphatase as secondary antibody.                       the CDC15 gene, the 250-bp TaqI-TaqI fragment (from <br />    Kinase assay: Cells (YKZ0024) expressing Cdc15-HA5 and            839th thymine to 1088th adenine) introduced into the <br /> its derivatives from multicopy plasmid were grown in 10 ml           ClaI site of pGADC1 (James et al. 1996; Table 2). CDC15 <br /> of SC-URA medium at 25&#226;&#172;&#353; to mid-log phase and harvested. <br /> Cell extracts were prepared as described above except that           encodes a protein kinase that is essential for the exit <br /> LLB (50 mm HEPES-NaOH, pH 7.4, 75 mm KCl, 50 mm NaF,                 from mitosis, consisting of 974 amino acid residues (Mw <br /> 50 mm &#226;&#65533;&#164;-gylcerophosphate, 1 mm EGTA, 0.1% NP-40, 1 mm                110 kD; Schweitzer and Philippsen 1991). The fact <br /> DTT, 1 mm PMSF, 1 &#226;&#65533;&#174;g/ml pepstatin, 1 &#226;&#65533;&#174;g/ml leupeptin,                that Cdc15 and Tem1, two essential regulators for the <br /> 2 &#226;&#65533;&#174;g/ml aprotinin; Jaspersen et al. 1998) was used instead of        exit from mitosis, have two-hybrid interaction suggests <br /> lysis buffer. A total of 300 &#226;&#65533;&#174;l of cell extract containing 750 <br /> &#226;&#65533;&#174;g protein was incubated at 4&#226;&#172;&#353; for 2 hr with 3 &#226;&#65533;&#174;g of 16B12           that they form a complex to function during mitotic <br /> and 60 &#226;&#65533;&#174;l 50% (v/v) protein A-Sepharose that had been swol-          exit. <br /> len with LLB. The precipitants with protein A-Sepharose were            To confirm the physical interaction in vivo between <br /> divided into three equal parts, one for the kinase assay at 25&#226;&#172;&#353;,     Cdc15 and Tem1, we performed a coprecipitation ex- <br /> one for the kinase assay at 37&#226;&#172;&#353;, and one for the Western             periment. Cdc15-HA5 and Tem1-HA2His6 or Cdc15- <br /> blotting. For estimation of the amount of HA-tagged Cdc15 <br /> proteins, anti-mouse IgG goat antibody conjugated to alkaline        HA5 and Tem1-HA2 were simultaneously expressed in <br /> phosphatase was used as secondary antibody. AttoPhos sub-            the wild-type cell. Cell extract was incubated with Ni- <br /> strate (Boehringer Mannheim, Mannheim, Germany) was                  NTA beads. The precipitants with the beads were <br />                                         Tem1 GTPase-Binding Domain of Cdc15 Kinase                                       1441 <br />  <br />                          TABLE 2 <br />           Two-hybrid interaction between TEM1 <br />                 and the CDC15 fragment <br />  <br />                                     &#226;&#65533;&#164;-Galactosidase activity <br />                                             (units) <br />                                      GBD        GBD-TEM1 <br /> GAD                                 4.4/4.8      19.1/8.3        Figure 1.&#226;&#8364;&#8221;Cdc15 binds to Tem1 in vivo. The wild-type <br /> GAD-CDC15 (TaqI&#226;&#171;&#185;839-TaqI &#226;&#171;&#185;1085)     7.9/3.5      31.5/43.7     cells (YKZ0024) expressing Cdc15-HA5 (pKZ416) and Tem1- <br />                                                                HA2His6 (pKZ173) or those expressing Cdc15-HA5 and <br />    Interactions between GBD-TEM1 and GAD-CDC15 TaqI&#226;&#171;&#185;839-       Tem1-HA2 (pKZ178) were grown at 25&#226;&#172;&#353; to mid-log phase and <br /> TaqI &#226;&#171;&#185;1085), isolated by the TemI two-hybrid screening, were   harvested. Extracts were prepared and incubated with Ni-NTA <br /> estimated by measuring &#226;&#65533;&#164;-galactosidase activity as described   beads for 1 hr at 4&#226;&#172;&#353;. Pull-down experiments with Ni-NTA beads <br /> in materials and methods. Results of two independent assays    were carried out as described in materials and methods. <br /> are shown (first trial/second trial).                          Tem1-HA2His6 (&#226;&#171;&#185;) or Tem1-HA2 (&#226;&#171;&#186;) in the extract before <br />                                                                (Ext) or after (Sup) the incubation with Ni-NTA beads and <br />                                                                in the eluate (Elu) with 100 mm imidazole was detected by <br /> washed five times with wash buffer containing 20 mm            Western blotting using 16B12 as primary antibody and anti- <br /> imidazole followed by elution with elution buffer con-         mouse IgG goat antibody conjugated to horseradish peroxi- <br /> taining 100 mm imidazole. Western blots of eluates             dase as secondary antibody. Cdc15-5HA in the extract or eluate <br /> showed that Tem1-HA2His6, in contrast to Tem1-HA2,             was also detected. Asterisk indicates the nonspecific cross- <br />                                                                reacting protein. <br /> efficiently precipitated with Ni-NTA beads (Figure 1). <br /> Cdc15-HA5 was detected in the eluate from Ni-NTA <br /> beads incubated with the extract containing Tem1- <br /> HA2His6 but not from those incubated with the extract          mutant Tem1 proteins with a HA2-His6 tag at the car- <br /> containing Tem1-HA2. These data indicate that Cdc15            boxy terminus (TEM1-HA2His6, tem1T34A-HA2His6, <br /> coprecipitated with Tem1. We concluded that Cdc15              and tem1T52A-HA2His6, respectively). The tem1 dis- <br /> and Tem1 form a complex in vivo.                               ruptant harboring TEM1-HA2His6 plasmid was able to <br />    We noted that there are nucleotide sequence discrep-        grow on the glucose plate as well as the one harboring <br /> ancies between the sequence of the TaqI-TaqI fragment          wild-type TEM1 plasmid, indicating that Tem1-HA2His6 <br /> isolated from the library (Y2HL) in this study and that        protein is fully functional (data not shown). Western <br /> reported by Schweitzer and Philippsen (1991); the              blot revealed that mutant proteins were expressed in <br /> 946th cytosine, the 947th guanine, and the 961st cyto-         an amount comparable to that of wild-type Tem1 (Fig- <br /> sine in the results of Schweitzer and Philippsen are           ure 2C), showing that Tem1 lost its function by the <br /> guanine, cytosine, and guanine in our results, respec-         mutation of T34A or T52A. These data indicate that <br /> tively.                                                        the GTP binding and hydrolysis or the interaction with <br />    Tem1 binds to Cdc15 in a manner dependent on                its effectors are essential for the function of Tem1. The <br /> the properties of small GTPase: To characterize the            overexpression of Tem1T34A protein from multicopy <br /> interaction between Cdc15 and Tem1, we investigated            plasmid under the control of the GAL7 promoter, as <br /> the interaction between Cdc15 and mutant Tem1. We              well as that of wild-type Tem1 protein, did not affect <br /> introduced a <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>novel mutation</span> into the motif that is well        the growth of wild-type cells on the galactose plate (data <br /> conserved among small GTPases by site-directed muta-           not shown) while that of Tem1T52A protein inhibited <br /> genesis (Figure 2A). One mutation substitutes the 34th         cell growth (data not shown), indicating that the <br /> threonine in the motif involved in GTP binding and             Tem1T52A protein is not simply a loss-of-function mu- <br /> hydrolysis with alanine (T34A) and the other substitutes       tant but has some dominantly negative effects for cell <br /> the 52nd threonine in the effector domain with alanine         growth. <br /> (T52A).                                                           Next, we evaluated the interaction between Cdc15 <br />    To examine whether the tem1 mutants thus con-               and mutant Tem1 by the two-hybrid assays. Each of the <br /> structed cause a phenotypic change, these mutant genes         three mutant tem1 (tem1T34A, tem1T52A, and tem1-3) <br /> were expressed under the control of the TEM1 pro-              was fused to the DNA segment encoding the Gal4-DNA <br /> moter in the tem1 disruptant carrying integrated PGAL7 -       binding domain and the entire open reading frame of <br /> TEM1 (E0096; Figure 2B). On a glucose plate, in which          CDC15 was fused to the DNA segment encoding the <br /> the expression of the TEM1 gene from the GAL7 pro-             Gal4-transcription activation domain. The two-hybrid <br /> moter is shut off, cells carrying the wild-type TEM1 plas-     interactions between Tem1 mutants and Cdc15 were <br /> mid grew well while the cells carrying the vector, the         estimated by measuring &#226;&#65533;&#164;-galactosidase activity (Figure <br /> tem1T34A plasmid, or the tem1T52A plasmid failed to            2D). The interaction was abolished either by the <br /> grow. To estimate the expression levels of these mutant        tem1T34A mutation or by the tem1-3 mutation and was <br /> proteins, we constructed plasmids encoding Tem1 or             reduced by half by the tem1T52A mutation. The tem1-3 <br />  1442                                                     K. Asakawa et al. <br />  <br />                                                                                             Figure 2.&#226;&#8364;&#8221;Tem1 binds to Cdc15 in <br />                                                                                          a manner dependent on properties of <br />                                                                                          small GTPase. (A) Schematic diagram of <br />                                                                                          Tem1 and three mutation sites intro- <br />                                                                                          duced by site-directed mutagenesis. The <br />                                                                                          34th threonine residue (in the motif in- <br />                                                                                          volved in GTP binding and hydrolysis <br />                                                                                          shown as solid boxes, I) and the 52nd <br />                                                                                          threonine (in the effector domain, <br />                                                                                          shown as III) were replaced with alanine <br />                                                                                          (T34A and T52A, respectively) by site- <br />                                                                                          directed PCR mutagenesis. The mutated <br />                                                                                          threonine residues are highlighted. The <br />                                                                                          mutation site of tem1-3 is also shown (II; <br />                                                                                          Shirayama et al. 1994a). (B) Growth of <br />                                                                                          tem1 mutants. E0096 cells (tem1::HIS3 <br />                                                                                          PGAL7-TEM1) carrying YCplac111 (vector), <br />                                                                                          pKZ134 (TEM1), pKZ141 (T52A), and <br />                                                                                          pKZ154 (T34A) were grown on a syn- <br />                                                                                          thetic galactose (left) or a glucose <br />                                                                                          (right) plate at 25&#226;&#172;&#353; for 3 days. (C) Estima- <br />                                                                                          tion of intracellular levels of Tem1 and <br />                                                                                          its derivatives. Extracts prepared from <br />                                                                                          E0096 cells expressing tagged Tem1 or <br />                                                                                          its derivatives, which were grown in SG- <br />                                                                                          LEU medium to mid-log phase followed <br />                                                                                          by 5 hr of incubation in SC-LEU me- <br />                                                                                          dium, were analyzed by Western blotting <br />                                                                                          using anti-HA monoclonal antibody <br />                                                                                          (16B12) as primary antibody and anti- <br /> mouse IgG goat antibody conjugated to horseradish peroxidase as secondary antibody. Thirty micrograms protein of each extract <br /> was loaded on a 10% polyacrylamide gel. Arrowheads point out the blots of HA-tagged Tem1. Lane no tag, E0096[YCplac111]; <br /> lane TEM1, E0096[pKZ173]; lane T52A, E0096[pKZ180]; lane T34A, E0096[pKZ181]. (D) Interactions between Cdc15 and Tem1 <br /> mutants in the two-hybrid system. Interactions between GAD-CDC15 and GBD (1), GBD-TEM1 (2), GBD-tem1T34A (3), GBD- <br /> tem1T52A (4), or GBD-tem1-3 (5) were estimated by measuring &#226;&#65533;&#164;-galactosidase activity as described in materials and methods <br /> and shown by solid bars. Open bars show the interactions between each construct and GAD. The results shown are representative <br /> of two independent trials. (E) Immunoprecipitation experiment. Extracts prepared from SLJ511 (CDC15-HA3) cells harboring <br /> indicated plasmid grown at 25&#226;&#172;&#353; in SC-URA medium to mid-log phase were subjected to immunoprecipitation experiment as <br /> described in materials and methods. Anti-HA antibody (16B12) was used to precipitate Cdc15-HA3. Lane TEM1 (right), <br /> pTGCU1 (YCpTEM1-GFP); lane tem1-3, pT135G (YCpctem1-3-GFP); lane T52A, pT52G (YCptem1T52A-GFP); lane T34A, pT34G <br /> (YCptem1T34A-GFP). The sample prepared from cells with pTGCU1 expressing untagged Cdc15 was shown in lane TEM1 (left) <br /> as a control. Amounts of Tem1-GFP and Cdc15-HA3 in extracts were shown as (ext Tem1) and (ext Cdc15), respectively. Asterisk <br /> or double asterisks indicate the nonspecific cross-reacting protein to anti-HA antibody or IgG, respectively. Arrowheads indicate <br /> the immunoprecipitated Tem1 or Cdc15. <br />  <br />  <br />  <br /> mutation is a temperature-sensitive mutation that substi-            detected as a doublet (Figure 2C). This observation <br /> tuted the 135th aspartate residue, in the consensus motif            suggests that some forms of Tem1 are post-transcription- <br /> needed for GTP binding and hydrolysis, with alanine                  ally modified; however, phosphorylation was found not <br /> (D135A; Figure 2A; Shirayama et al. 1994a). We further               to be responsible for the modification (data not shown). <br /> tested whether these tem1 mutations affect the interac-                 Determination of Tem1-binding domain of Cdc15: <br /> tion with Tem1 and Cdc15 in vivo by immunoprecipita-                 Besides a report that the deletion of 32 amino acid <br /> tion assays. Each tem1 mutant protein that had a GFP                 residues of its carboxy terminus abolished the function <br /> tag at the carboxy terminus was expressed in the cell                of Cdc15 (Schweitzer and Philippsen 1991), there is <br /> with the chromosomal CDC15 gene tagged with three                    no report on a functional domain except for the kinase <br /> HA epitopes (CDC15-HA3). We confirmed that Tem1-                     domain. Now that we know Tem1 binds to Cdc15, it is <br /> GFP is functional as Tem1 (data not shown). We hardly                important for understanding the function of Cdc15 to <br /> detected either of the tem1 mutant proteins precipitated             determine the domain required for the interaction with <br /> with Cdc15-HA3 (Figure 2E) whereas wild-type Tem1                    Tem1. To determine the Tem1-binding domain of <br /> coprecipitated with Cdc15-HA3. These results indicate                Cdc15, we analyzed the interaction between Tem1 and <br /> that Tem1 binds to Cdc15 in a manner dependent on                    truncated Cdc15 by two-hybrid assays. Each truncated <br /> the molecular properties of small GTPase. Notably,                   fragment of CDC15 was fused to the coding region of <br /> Tem1T34A-HA2His6 was detected as a single band                       the Gal4-DNA transcription activation domain and in- <br /> whereas Tem1-HA2His6 and Tem1T52A-HA2His6 were                       troduced into PJ69-4A cells expressing GBD-Tem1 (Fig- <br />                                          Tem1 GTPase-Binding Domain of Cdc15 Kinase                                            1443 <br />  <br />                                                                                            Figure 3.&#226;&#8364;&#8221;The Tem1-binding do- <br />                                                                                          main of CDC15. (A) Construction of <br />                                                                                          truncated CDC15. The nucleotide se- <br />                                                                                          quence AGACTA (&#226;&#171;&#186;5 to &#226;&#171;&#186;10) upstream <br />                                                                                          of CDC15 was replaced by the BglII site <br />                                                                                          (AGATCT) by PCR mutagenesis. The <br />                                                                                          SalI site (&#226;&#171;&#185;838) and the XhoI site <br />                                                                                          (&#226;&#171;&#185;1084) are shown. The open bar shows <br />                                                                                          the coding region of the conserved ki- <br />                                                                                          nase domain. Truncated fragments of <br />                                                                                          CDC15 were introduced into the GAD <br />                                                                                          vector ( James et al. 1996). a, pKZ401 <br />                                                                                          (CDC15); b, pKZ415 (cdc15d82SX); c, <br />                                                                                          pKZ420 (the SalI-XhoI fragment); d, <br />                                                                                          pKZ422 (the BglII-XhoI fragment); e, <br />                                                                                          pKZ421 (the BglII-SalI fragment); f, <br />                                                                                          pKZ402 (the fragment from the SalI site <br />                                                                                          to the stop codon). (B) Interactions be- <br />                                                                                          tween Tem1 and truncated Cdc15 in the <br />                                                                                          two-hybrid system. Interactions between <br />                                                                                          GBD-TEM1 and truncated CDC15 frag- <br />                                                                                          ments shown in A were estimated by <br />                                                                                          measuring &#226;&#65533;&#164;-galactosidase activity and <br />                                                                                          are shown by solid bars. Open bars show <br />                                                                                          the interactions between each construct <br />                                                                                          and GBD. The interaction between GAD <br />                                                                                          and GBD-TEM1 or GBD is shown as con- <br />                                                                                          trol. The result shown is representative <br />                                                                                          of two independent experiments. (C) <br />                                                                                          Functional analysis of Cdc15d82SX by <br />                                                                                          the plasmid shuffling method. YKZ0255 <br />                                                                                          (&#226;&#338;&#172;cdc15 [YCpCDC15-URA3]) was trans- <br />                                                                                          formed with YCplac22 (vec), pKZ439 <br />                                                                                          (CDC15), pKZ438 (CDC15-HA5), pKZ448 <br />                                                                                          (d82SX-HA5 [YCp]) or pKZ449 (d82SX- <br />                                                                                          HA5 [YEp]). Trp&#226;&#171;&#185; Ura&#226;&#171;&#185; transformants <br />                                                                                          were streaked on a SC-TRP plate with or <br />                                                                                          without 5-FOA and incubated at 25&#226;&#172;&#353; for 3 <br />                                                                                          days. (D) &#226;&#338;&#172;cdc15 [YEpcdc15d82SX-HA5] <br />                                                                                          cells are temperature sensitive. &#226;&#338;&#172;cdc15 <br />                                                                                          [pKZ449] (left) cells and &#226;&#338;&#172;cdc15 <br />                                                                                          [pKZ438] (right) cells were streaked on <br />                                                                                          a YPD plate and incubated at 37&#226;&#172;&#353; for 3 <br />                                                                                          days. (E) Western blot of Cdc15d82SX- <br />                                                                                          HA5. Extracts from YKZ0255 cells har- <br />                                                                                          boring pKZ438 (1, 2; Cdc15-HA5), <br /> pKZ448 (3, 4; Cdc15d82SX-HA5 expressed from centromeric plasmid), or pKZ449 (5, 6; Cdc15d82SX-HA5 expressed from <br /> multicopy plasmid) grown in SC-(URA, TRP) medium to mid-log phase were prepared and analyzed by Western blotting using <br /> anti-HA monoclonal antibody (16B12) as primary antibody and anti-mouse IgG goat antibody conjugated to alkaline phosphatase <br /> as secondary antibody. The samples of cells grown at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; were loaded in odd number lanes or even number lanes, <br /> respectively. Thirty micrograms protein of each extract was loaded on a 7.5% polyacrylamide gel. Asterisk indicates the nonspecific <br /> cross-reacting protein. <br />  <br />  <br /> ure 3A). By measuring &#226;&#65533;&#164;-galactosidase activity, we                  acid domain plays a major role for the interaction. The <br /> examined the interaction between Tem1 and Cdc15                     construct containing the kinase domain and the 82- <br /> fragments.                                                          amino-acid domain (Figure 3B, d) interacted with Tem1 <br />    Knowing that the segment consisting of 82 amino acid             as strongly as the full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Cdc15, while the kinase <br /> residues encoded by the SalI-XhoI fragment of CDC15                 domain alone hardly interacted with Tem1 (Figure 3B, <br /> (corresponding to the TaqI-TaqI fragment isolated by                e). The Western blot showed that the GAD fusion of <br /> the two-hybrid screening; Table 2) alone was able to                kinase domain (e) was expressed more than that con- <br /> interact with Tem1 (Figure 3B, c), we constructed the               taining both the kinase domain and the 82-amino-acid <br /> mutant gene encoding Cdc15 without the 82-amino-acid                domain (d, data not shown), suggesting that the differ- <br /> residues (cdc15d82SX; see materials and methods). As                ence of the &#226;&#65533;&#164;-galactosidase activity was not due to the <br /> expected, Cdc15d82SX showed a reduced interaction                   difference of the expression levels of the GAD fusions <br /> with Tem1 (Figure 3B, b), indicating that the 82-amino-             but to the intensity of the interaction. The fragment <br />  1444                                               K. Asakawa et al. <br />  <br /> without the kinase domain showed the interaction at a         performed immunoprecipitation assay. Consistent with <br /> level comparable to that of the 82-amino-acid domain          the result obtained from the two-hybrid assay (see Figure <br /> (Figure 3B, c and f) showing that the carboxy terminus        3B), Cdc15d82SX-HA5 was not precipitated with Tem1- <br /> of Cdc15 has less significance for the interaction. These     myc9His6. We hardly detected Cdc15-LF-HA5 coprecip- <br /> results indicate that the region consisting of the kinase     itated with Tem1-myc9His6 (Figure 4E), indicating that <br /> domain and the following 82-amino-acid residues is im-        the cdc15-LF mutation affects the interaction between <br /> portant for the interaction between Tem1 and Cdc15.           Tem1 and Cdc15. Interestingly, some amount of Cdc15- <br /> We tentatively designated the segment consisting of the       DD-HA5 was precipitated with Tem1-myc9His6, indicat- <br /> 82-amino-acid residues the Tem1-binding domain.               ing that the cdc15-DD mutation does not abolish the <br />    To test if the Tem1-binding domain is important for        ability to form the Cdc15/Tem1 complex (see discus- <br /> the function of Cdc15, we expressed Cdc15d82SX                sion). We found that the temperature-sensitive growth <br /> in the cdc15 disruptant. The centromeric plasmid carry-       of the cdc15-LF cell was efficiently suppressed by the <br /> ing cdc15d82SX was introduced into the cdc15 disrupt-         overexpression of Tem1 (Figure 4C). This result sug- <br /> ant, which was kept alive by expressing the wild-type         gests that the primary defect of the cdc15-LF mutation <br /> CDC15 from plasmid harboring the URA3 gene. On                is the reduction in the interaction with Tem1 and, there- <br /> medium containing 5-FOA, the cdc15 disruptant with            fore, the interaction with Tem1 is required for the <br /> Cdc15d82SX no longer survived (Figure 3C). Western            Cdc15 function under physiological conditions. We also <br /> blots showed that Cdc15d82SX was expressed from cen-          found that the overproduction of Cdc15d82SX, Cdc15- <br /> tromeric plasmid at a level lower than that of the wild-      DD, or Cdc15-LF suppressed the lethality of tem1 disrup- <br /> type Cdc15 (Figure 3E). However, when Cdc15d82SX              tion (Figure 4F). These results are consistent with the <br /> was expressed from multicopy plasmid, the cdc15 dis-          notion that the <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> or the <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>deletion mutation</span> <br /> ruptant restored growth at 25&#226;&#172;&#353; but displayed the temper-      of the Tem1-binding domain affects only the interaction <br /> ature-sensitive growth (Figure 3, C and D). The amount        between Tem1 and Cdc15. We failed to detect an effect <br /> of Cdc15d82SX expressed from multicopy plasmid was            of the cdc15-S309A mutation at least under the normal <br /> more than that of the wild-type Cdc15 (Figure 3E),            growth conditions. The phosphorylation at the 309th <br /> suggesting that the deletion of the Tem1-binding do-          serine, if it occurs, alone does not affect the cell growth <br /> main impaired the function of Cdc15.                          ( Jaspersen and Morgan 2000). <br />    Mutagenesis analysis of Tem1-binding domain: In                Effects of mutations in the Tem1-binding domain on <br /> contrast to the high conservation of the sequence of          kinase activity: Analysis of the point mutations (cdc15- <br /> the kinase domain of Cdc15, any obvious sequence simi-        DD and cdc15-LF) in addition to the deletion mutation <br /> lar to the Tem1-binding domain is not found near the          (cdc15d82SX) confirmed the importance of the Tem1- <br /> kinase domain of other protein kinases including the          binding domain for the function of Cdc15. Next, we <br /> Cdc15 homologue in fission yeast, Cdc7 kinase (Fank-          tested the possibility that these mutations in the Tem1- <br /> hauser and Simanis 1994). The Tem1-binding domain             binding domain affect the kinase activity of Cdc15. To <br /> is rich in charged amino acid residues in the amino-          measure the kinase activity, Cdc15-HA5 and its mutant <br /> terminal side and has one putative Cdc28-phosphoryla-         derivatives expressed from multicopy plasmid in the <br /> tion site. Further, we found an amino acid sequence           wild-type cells were immunoprecipitated by anti-HA an- <br /> that is slightly similar to the carboxy-terminal region       tibody and the immunocomplex was subjected to the <br /> of the fission yeast Cdc7 (LxxLFxVCxL) in the Tem1-           kinase assay using MBP as an artificial substrate (Figure <br /> binding domain. To characterize the function of the           5A, top; Jaspersen et al. 1998). The autoradiogram of <br /> Tem1-binding domain more precisely, we introduced             the autophosphorylation of Cdc15 and its mutant deriv- <br /> the point mutations into the indicated sites by PCR-          atives was also shown in Figure 5A (bottom). We found <br /> aided site-directed mutagenesis (Figure 4A).                  that Cdc15-DD-HA5 or Cdc15-LF-HA5 retains, if not full, <br />    To analyze the effects of these mutations, centromeric     kinase activity toward MBP (Figure 5C). These results <br /> TRP1 plasmid carrying each of the mutant genes under          indicate that a defect caused by either of the two muta- <br /> the CDC15 promoter was introduced into the cdc15 dis-         tions is not due to a simple reduction in kinase activity. <br /> ruptant carrying pKZ416 followed by the plasmid shuf-             We estimated kinase activity of temperature-sensitive <br /> fling on the FOA plate. Western blots showed that all the     Cdc15, other than Cdc15-LF (Figure 6A). The cdc15-rlt1 <br /> mutant proteins tested were produced at a comparable          mutation resides in the kinase domain (Shirayama et <br /> level (Figure 4D). The cdc15-DD strain was not able to        al. 1996) and the cdc15-lyt1 mutation in the middle re- <br /> grow in the presence of 5-FOA, indicating that the cdc15-     gion of the Cdc15 (Jimenez et al. 1998). Either the <br /> DD mutation is lethal (Figure 4B). The cdc15-LF strain        cdc15-lyt1 mutation or the cdc15-rlt1 mutation caused <br /> displayed temperature-sensitive growth (Figure 4, B and       a reduction in kinase activity, although the cdc15-rlt1 <br /> C) that is not due to the instability of Cdc15-LF protein     mutation much more severely impaired the activity (Fig- <br /> at a high temperature (Figures 4D and 6E).                    ure 6, B and C). We performed the kinase assay at a <br />    To examine whether Cdc15d82SX, Cdc15-DD, or                high temperature to determine how the temperature <br /> Cdc15-LF is defective in binding to Tem1 in vivo, we          shift affects the kinase activity. At 37&#226;&#172;&#353;, the kinase activity <br />                                         Tem1 GTPase-Binding Domain of Cdc15 Kinase                                           1445 <br />  <br />                                                                                                     Figure 4.&#226;&#8364;&#8221;Mutagenesis an- <br />                                                                                                  alysis of Tem1-binding do- <br />                                                                                                  main. (A) Amino acid se- <br />                                                                                                  quence of the Tem1-binding <br />                                                                                                  domain and the mutation sites <br />                                                                                                  introduced by site-directed mu- <br />                                                                                                  tagenesis. Construction of each <br />                                                                                                  mutant was described in mate- <br />                                                                                                  rials and methods. The re- <br />                                                                                                  gion rich in charged amino <br />                                                                                                  acid residues was underlined. <br />                                                                                                  (B) Functional analysis of <br />                                                                                                  Tem1-binding domain mu- <br />                                                                                                  tants by the plasmid shuffling <br />                                                                                                  method. YKZ0255 (&#226;&#338;&#172;cdc15 <br />                                                                                                  [YCpCDC15-URA3]) was trans- <br />                                                                                                  formed with YCplac111 (vec), <br />                                                                                                  pKZ412 (CDC15), pKZ434 <br />                                                                                                  (DD), pKZ436 (S309A), or <br />                                                                                                  pFM420 (LF ). Transformants <br />                                                                                                  were streaked on each of the <br />                                                                                                  SC-LEU plates with or without <br />                                                                                                  5-FOA and incubated at 25&#226;&#172;&#353; for <br />                                                                                                  3 days. (C) The cdc15-LF cells <br />                                                                                                  displayed the temperature-sen- <br />                                                                                                  sitive growth that is suppressed <br />                                                                                                  by overproduction of Tem1. <br /> YKZ0376 (cdc15-LF ) was transformed with YCplac33 (vec), pKZ406 (CDC15), or L11 (TEM1[YEp], Shirayama et al. 1994a). <br /> Transformants were streaked on a SC-URA plate and incubated at 37&#226;&#172;&#353; for 3 days. (D) Western blot of mutant Cdc15 proteins. <br /> Extracts prepared from YKZ0255 cells harboring indicated plasmid grown at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; in SC-(URA, TRP) medium to mid-log <br /> phase were analyzed by Western blotting using anti-HA monoclonal antibody (16B12) as primary antibody and anti-mouse IgG <br /> goat antibody conjugated to horseradish peroxidase as secondary antibody. Asterisk indicates the nonspecific cross-reacting <br /> protein. Lane 1, pKZ438 (YCpCDC15-HA5) at 25&#226;&#172;&#353;; lane 2, pKZ455 (YCpcdc15-DD-HA5) at 25&#226;&#172;&#353;; lane 3, pKZ456 (YCpcdc15-S309A- <br /> HA5) at 25&#226;&#172;&#353;; lane 4, pKZ442 (YCpcdc15-LF-HA5) at 25&#226;&#172;&#353;; lane 5, pKZ442 (YCpcdc15-LF-HA5) at 37&#226;&#172;&#353;. (E) Coimmunoprecipitation <br /> experiment between Tem1 and Cdc15 mutants. Extracts prepared from YKZ0454 (tem1::URA3 [YCpTEM1-myc9His6]) cells harbor- <br /> ing indicated plasmid grown at 25&#226;&#172;&#353; in SC-(LEU, TRP) medium to mid-log phase were subjected to immunoprecipitation experiment <br /> as described in materials and methods. Lane CDC15 (right), pKZ453 (YEpCDC15-HA5); lane d82SX, pKZ449 (YEpcdc15d82SX- <br /> HA5), lane DD; pKZ465 (YEpcdc15-DD-HA5), lane LF; pKZ464 (YEpcdc15-LF-HA5). The sample prepared from cells with pKZ453 <br /> expressing Tem1 without a myc9His6-tag is shown in lane CDC15 (left) as a control. Amounts of Cdc15-HA5 in extracts are <br /> shown as (ext Cdc15). (F) Overexpression of cdc15 mutants complements the tem1 disruption. E0096 cells (tem1::HIS3 PGAL7 - <br /> TEM1) carrying indicated plasmid were grown on a synthetic galactose (left) or a glucose (right) plate at 25&#226;&#172;&#353; for 3 days. Section <br /> vec, YCplac 22; section TEM1, CP (YCpTEM1); section CDC15, pKZ453 (YEpCDC15); section d82SX, pKZ449 (YEpcdc15d82SX ); <br /> section DD, pKZ455 (YEpcdc15DD); section S309A, pKZ456 (YEpcdc15S309A); section LF, pKZ454 (YEpcdc15LF ). <br />  <br />  <br />  <br /> toward MBP of the wild-type Cdc15-HA5 was higher                   type Tem1-GFP was detected at the SPB as one or two <br /> than that at 25&#226;&#172;&#353; by &#226;&#181;&#8216;1.6-fold (Figure 6D). Cdc15-LF-               dots associated with DNA (Figure 7A, a and b). We <br /> HA5 and Cdc15-lyt1-HA5 each displayed the increase                 hardly detected Tem1T34A-GFP localized to the SPB <br /> in kinase activity at a high temperature as did the wild-          (Figure 7A, c and d). In turn, we observed dispersed <br /> type Cdc15-HA5. However, Cdc15-rlt1-HA5 showed a                   GFP fluorescence throughout the cytosol. Tem1-3-GFP <br /> decrease in the activity at a high temperature. These              weakly localized to the SPB at 25&#226;&#172;&#353; but not at 37&#226;&#172;&#353; (Figure <br /> results indicate that the Cdc15-rlt1 enzyme itself is tem-         7A, g and i). The cytosolic GFP signal of Tem1-3-GFP <br /> perature sensitive; in contrast, Cdc15-LF or Cdc15-lyt1            at 37&#226;&#172;&#353; was much brighter than that at 25&#226;&#172;&#353;. These results <br /> has a reduced enzymatic activity that is not temperature           suggest that the SPB localization of Tem1 is regulated <br /> sensitive.                                                         by nucleotide bound to it because the tem1T34A muta- <br />    Effects of the tem1 mutations or the cdc15 mutations            tion or the tem1-3 mutation resides in the motif involved <br /> on their localization to the spindle pole body: Tem1               in GTP binding and hydrolysis (Figure 2A). We detected <br /> and Cdc15 localize to the spindle pole body (Cenamor               Tem1T52A-GFP localized to the SPB (Figure 7A, e and <br /> et al. 1999; Bardin et al. 2000; Pereira et al. 2000; Xu           f), suggesting that the functional defect of Tem1 by the <br /> et al. 2000). We examined whether the mutation that                tem1T52A mutation is not due to its inability to localize <br /> reduced the function of Tem1 or Cdc15 might cause                  properly to the SPB but due to its inability to interact <br /> any effect on their SPB localization. Carboxy-terminally           with Cdc15. <br /> GFP-tagged Tem1 or its mutant derivative was expressed                Next, we tested the localization of Cdc15 mutants. <br /> from the centromeric plasmid in the wild-type cell. Wild-          Carboxy-terminally GFP-tagged Cdc15 or its mutant de- <br />  1446                                                     K. Asakawa et al. <br />  <br />                                                                     (data not shown) and SPB localization was cell cycle <br />                                                                     independent (data not shown). In some cases, we also <br />                                                                     detected highly fluorescent bodies (indicated by arrow- <br />                                                                     heads in Figure 7B, e and g) in the cell that probably <br />                                                                     were artifacts resulting from overexpression of GFP- <br />                                                                     tagged protein and were easily distinguished from the <br />                                                                     GFP signal associated with the SPB by their separation <br />                                                                     from the 4&#226;&#172;&#732;,6-diamidino-2-phenylindole (DAPI) signal. <br />                                                                     By examining the localization of GFP-tagged Cdc15 mu- <br />                                                                     tant proteins, we detected Cdc15-DD-GFP at the SPB <br />                                                                     either at 25&#226;&#172;&#353; or at 37&#226;&#172;&#353; as a spot brighter than that of <br />                                                                     the wild-type Cdc15-GFP (data not shown and Figure <br />                                                                     7B, i) and we detected Cdc15-LF-GFP at the SPB at 25&#226;&#172;&#353; <br />                                                                     (data not shown) but not at 37&#226;&#172;&#353; (Figure 7B, e). These <br />                                                                     results suggest that the Tem1-binding domain has some <br />                                                                     role in the proper SPB localization of Cdc15. We further <br />                                                                     tested the localization of Cdc15-rlt1-GFP and Cdc15- <br />                                                                     lyt1-GFP. Cdc15-rlt1-GFP was mainly cytosolic and was <br />                                                                     hardly detected at the SPB at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; (data not shown <br />                                                                     and Figure 7B, c), suggesting that the kinase activity of <br />                                                                     Cdc15 is required for its SPB localization. We detected <br />                                                                     Cdc15-lyt1-GFP at the SPB at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; (data not shown <br />                                                                     and Figure 7B, g), suggesting that the cdc15-lyt1 muta- <br />                                                                     tion has little effect on the SPB localization of Cdc15. <br />  <br />  <br />                                                                                          DISCUSSION <br />                                                                        Cdc15/Tem1 complex: The MEN, which eventually <br />                                                                     causes the inactivation of CDK at mitotic exit, includes <br />                                                                     protein kinases (Cdc5, Cdc15, and Dbf2), a protein <br />                                                                     phosphatase (Cdc14), and a small GTPase (Tem1), sug- <br />                                                                     gesting that one aspect of the MEN is a complicated <br />    Figure 5.&#226;&#8364;&#8221;Cdc15 kinase activity of Tem1-binding domain           phosphorylation/dephosphorylation network. Various <br /> mutants. (A) Wild-type cells (YKZ0024) carrying the multi-          genetic interactions among the MEN genes reported so <br /> copy plasmid encoding Cdc15-HA5 (pKZ425), Cdc15d82SX-               far (Jaspersen et al. 1998) reflect this complexity and <br /> HA5 (pKZ419), Cdc15-DD-HA5 (pKZ435), Cdc15-S309A-HA5                the characterization of the molecular interaction be- <br /> (pKZ454), Cdc15-LF-HA5 (pKZ431), or Cdc15-2-HA5 (pKZ429)            tween the MEN components is very important to deepen <br /> were grown at 25&#226;&#172;&#353;. Extract was prepared for Cdc15-HA5 or <br /> its derivatives and was immunoprecipitated with monoclonal          our understanding of this feature of the MEN. <br /> antibody (16B12). The kinase activity associated with the pre-         In this study, we showed that Tem1 bound to Cdc15 <br /> cipitate was assayed by using MBP as substrate (top) or by          (Figure 1). While we were preparing this article, Bardin <br /> measuring the autophosphorylation activity (bottom). (B)            et al. (2000) showed that Tem1 formed a complex with <br /> Cdc15-HA5 or its derivatives were detected by Western blotting      Cdc15 during mitosis. We also provided evidence that <br /> using anti-HA monoclonal antibody (16B12) as primary anti- <br /> body and anti-mouse IgG goat antibody conjugated to horse-          each of the mutations that are believed to reduce the <br /> radish peroxidase as secondary antibody. For the estimation         function of Tem1 (tem1T34A, tem1T52A, or tem1-3) also <br /> of Cdc15-HA5 and mutant derivatives, anti-mouse IgG goat            weakened the interaction with Cdc15 (Figure 2, D and <br /> antibody conjugated to alkaline phosphatase as secondary anti-      E), suggesting that the activation of Tem1 is necessary <br /> body was used. AttoPhos substrate was used as a fluorogenic         for the formation of the Cdc15/Tem1 complex in vivo. <br /> substrate of alkaline phosphatase (data not shown, see materi- <br /> als and methods). (C) Specific activity of MBP phosphoryla-         The result that the mutation in the putative effector <br /> tion. The intensity of the signal of autoradiography was nor-       domain (tem1T52A) reduced the interaction suggests <br /> malized by the amount of immunoprecipitated Cdc15-HA5               that the Cdc15 is likely to be an effector of Tem1. <br /> or its mutant derivatives. The results shown are representative        Domain analysis of Cdc15: Cdc15 is a protein kinase <br /> of two independent trials.                                          that has a kinase domain at its amino-terminal region <br />                                                                     (Figure 6A). In this study, we determined a novel func- <br />                                                                     tional domain of Cdc15, the Tem1-binding domain. The <br /> rivative was expressed from multicopy plasmid in cdc15-2            deletion analysis showed that Tem1 binds to the region <br /> cells. The SPB localization of Cdc15-GFP was detected               consisting of the 82-amino-acid residues that resides <br /> in &#226;&#181;&#8216;20% of the cells at 37&#226;&#172;&#353; (Figure 7B, a) or at 25&#226;&#172;&#353;                next to the kinase domain of Cdc15 (Figures 3B and <br />                                          Tem1 GTPase-Binding Domain of Cdc15 Kinase                                             1447 <br />  <br />                                                                                                        Figure 6.&#226;&#8364;&#8221;Cdc15 kinase ac- <br />                                                                                                     tivity of temperature-sensitive <br />                                                                                                     mutants. (A) Schematic dia- <br />                                                                                                     gram of Cdc15 and the temper- <br />                                                                                                     ature-sensitive mutation sites. <br />                                                                                                     The open box and the solid <br />                                                                                                     box represent the kinase do- <br />                                                                                                     main and the Tem1-binding <br />                                                                                                     domain, respectively. Arrow- <br />                                                                                                     heads indicate the tempera- <br />                                                                                                     ture-sensitive mutation sites. <br />                                                                                                     Wild-type cells (YKZ0024) <br />                                                                                                     carrying the multicopy plas- <br />                                                                                                     mid encoding Cdc15-HA5 <br />                                                                                                     (pKZ425), Cdc15-LF-HA5 (pKZ- <br />                                                                                                     431), Cdc15-lyt1-HA5 (pKZ- <br />                                                                                                     430), or Cdc15-rlt1-HA5 (pKZ- <br />                                                                                                     428) were grown at 25&#226;&#172;&#353;. Extract <br />                                                                                                     of each culture was prepared <br />                                                                                                     and Cdc15-HA5 or its deriva- <br />                                                                                                     tives was immunoprecipitated <br />                                                                                                     with monoclonal antibody <br />                                                                                                     (16B12). The kinase activity as- <br />                                                                                                     sociated with the precipitate <br />                                                                                                     was assayed at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; by us- <br />                                                                                                     ing MBP as substrate (B) or <br />                                                                                                     by measuring the autophos- <br /> phorylaion activity (C). (D) The amount of the Cdc15-HA5 or its derivatives was estimated by Western blotting using anti-HA <br /> monoclonal antibody (16B12) as primary antibody and anti-mouse IgG goat antibody conjugated to alkaline phosphatase as <br /> secondary antibody. AttoPhos substrate was used as a fluorogenic substrate of alkaline phosphatase. (E) Specific activity of MBP <br /> phosphorylation. The intensity of the signal of autoradiography was normalized by the amount of immunoprecipitated Cdc15- <br /> HA5 or its mutant derivatives. Solid boxes or open boxes show the specific activities at 25&#226;&#172;&#353; or 37&#226;&#172;&#353;, respectively. The numbers <br /> above the open boxes represent the ratio of the specific activity at 37&#226;&#172;&#353; to that at 25&#226;&#172;&#353;. The result shown is representative of two <br /> independent trials. <br />  <br />  <br /> 4E). Determination of the Tem1-binding domain pro-                   4E). This suggests that there exists a subdomain in the <br /> vided us a clue to characterize the domain function of               amino-terminal region of the Tem1-binding domain <br /> Cdc15.                                                               that is less important for the interaction between Cdc15 <br />    The properties of Cdc15d82SX, which has been con-                 and Tem1 but is important for an essential step(s) of <br /> structed by deleting the Tem1-binding domain, are of                 the action of Cdc15. It remains to be examined whether <br /> particular interest. Cdc15d82SX retained low kinase ac-              the enhanced localization of Cdc15-DD to the SPB is <br /> tivity toward MBP (Figure 5D) and, when overexpressed,               toxic or not. <br /> restored the growth of the cdc15 disruptant at least at                 From previous reports and this study, the amino-ter- <br /> a permissive temperature (Figure 3C). These results led              minal one-third region of Cdc15 is required for phos- <br /> us to assume that the deletion of the Tem1-binding                   phorylation reaction and the interaction with Tem1. <br /> domain does not abolish the entire function of Cdc15                 However, the function of the middle or the carboxy- <br /> and that the interaction between Cdc15 and Tem1 is                   terminal region of Cdc15 is still largely unknown. The <br /> required for only a limited step of the action of Cdc15.             cdc15-lyt1 mutation resides in the middle region of <br /> Supporting this notion, the results of the mutagenesis               Cdc15 (Figure 6A; Jimenez et al. 1998). The kinase activ- <br /> analysis showed that the Tem1-binding domain is not                  ity of Cdc15-lyt1 was lower than that of the wild-type <br /> essential for the intrinsic kinase activity of Cdc15 be-             Cdc15 (Figure 6D) but was not temperature sensitive, <br /> cause Cdc15-LF or Cdc15-DD still retained the kinase                 suggesting that the temperature sensitivity of the cdc15- <br /> activity (Figures 5C and 6E) even though each of the                 lyt1 mutant should be explained by a mechanism other <br /> mutations caused a defect in cell growth (Figure 4, B                than thermolabile enzyme activity. It has been suggested <br /> and C). The results that the cdc15-DD mutation en-                   that the region containing the cdc15-lyt1 site might be <br /> hanced the SPB localization of Cdc15-DD while the                    involved in the function of Cdc15 in cytokinesis ( Jime- <br /> cdc15-LF mutation diminished it (Figure 7B, e and i)                 nez et al. 1998). Whether or not the region where the <br /> suggest that the Tem1-binding domain is involved in                  cdc15-lyt1 mutation resides is directly involved in the <br /> the regulation of the subcellular localization of Cdc15.             interaction with Tem1 is elusive. There are seven puta- <br /> The cdc15-DD mutation does not completely abolish the                tive phosphorylation sites in Cdc15, one in the Tem1- <br /> interaction with Cdc15-DD and Tem1 as we slightly de-                binding domain and the rest in the carboxy-terminal <br /> tected the Cdc15-DD coprecipitated with Tem1 (Figure                 two-thirds of Cdc15. The mutations in all of them to <br />  1448                                                    K. Asakawa et al. <br />  <br />                                                                    tion (Schmidt et al. 1997). Cdc7 localizes to the SPB <br />                                                                    during mitosis and this SPB localization is dependent <br />                                                                    on Spg1 function (Sohrmann et al. 1998). On the other <br />                                                                    hand, the kinase activity of Cdc7 is not regulated by <br />                                                                    Spg1 (Sohrmann et al. 1998). We estimated the kinase <br />                                                                    activity toward MBP of Cdc15-HA5 immunoprecipitated <br />                                                                    from the extract of the temperature-sensitive tem1-3 cells <br />                                                                    grown under the restrictive temperature or from the <br />                                                                    extract overexpressing the dominant negative Tem1- <br />                                                                    T52A protein (Figure 2A) and found no effect of the <br />                                                                    tem1 mutation on the kinase activity of Cdc15 (data not <br />                                                                    shown). Together with our observations that Cdc15-LF <br />                                                                    defective in binding to Tem1 showed no significant <br />                                                                    defect in the kinase activity, these results may suggest <br />                                                                    that Tem1 is not required for the kinase activity of <br />                                                                    Cdc15, at least, toward a nonphysiological substrate. We <br />                                                                    found that Cdc15-LF failed to localize to the SPB at the <br />                                                                    restrictive temperature (Figure 7B, e), suggesting that <br />                                                                    the proper interaction with Cdc15 and Tem1 may be <br />                                                                    required for the SPB localization of Cdc15. This inter- <br />                                                                    pretation appears inconsistent with the previous report <br />                                                                    that overproduced Cdc15 was able to localize to the SPB <br />                                                                    in the absence of Tem1 function (Cenamor et al. 1999; <br />                                                                    our unpublished result). However, localization of a pro- <br />    Figure 7.&#226;&#8364;&#8221;Localization analysis of GFP-tagged Tem1 pro-         tein under its overproduction should be evaluated with <br /> teins or Cdc15 proteins. (A) Localization of GFP-tagged Tem1 <br />                                                                    caution. Because of a limited amount of Cdc15 in vivo, <br /> and its mutant derivatives. The wild-type cell (YKZ0024) ex- <br /> pressing indicated plasmind was grown to mid-log phase at          it is very hard to detect it by microscopic analysis under <br /> 25&#226;&#172;&#353; (a, c, e, or g) or to mid-log phase at 25&#226;&#172;&#353; followed by         the physiological conditions (Xu et al. 2000; data not <br /> 3-hr incubation at 37&#226;&#172;&#353;. The GFP fluorescence was detected as       shown). Furthermore, the physiological substrates of <br /> described in materials and methods. a, pTGCU1 (YCp-                Cdc15 are not yet identified. Analysis of the physiologi- <br /> TEM1-GFP); c, pT52G (YCptem1T52A-GFP); e, pT34G (YCp- <br />                                                                    cal level of Cdc15 using physiological substrates is neces- <br /> tem1T34A-GFP); g and i, pT135G (YCptem1-3-GFP). DNA was <br /> visualized by DAPI staining and is shown in b, d, f, h, or j.      sary to characterize precisely the regulatory mechanism <br /> (B) Localization of GFP-tagged Cdc15 and its mutant deriva-        of Cdc15 by Tem1. <br /> tives. The cdc15-2 (YKZ0200) cell expressing indicated plasmid      <br /> was grown to mid-log phase at 25&#226;&#172;&#353; followed by a 3-hr incuba-     <br /> tion at 37&#226;&#172;&#353;. The GFP fluorescence was detected as described        <br /> in materials and methods. a, pCFG1 (YEpCDC15-GFP); c,           <br /> pRLTG (YEpcdc15-rlt1-GFP); e, pLFG (YEpcdc15-LF-GFP); g,           <br /> pLYTG (YEpcdc15-lyt-GFP); i, pDDG (YEpcdc15-DD-GFP). Ar- <br /> rowheads indicate the highly fluorescent bodies that probably <br /> were artifacts resulting from overexpression of GFP-tagged <br /> protein. DNA was visualized by DAPI staining and is shown                               <br /> in b, d, f, h, or j. <br /> </body></html>