<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                                                                                      Open Access <br /> LinkHub: a Semantic Web system that facilitates cross-database <br /> queries and information retrieval in proteomics <br />  <br />  <br />                  <span id='am-3' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: A key abstraction in representing proteomics knowledge is the notion of unique <br />                  identifiers for individual entities (e.g. proteins) and the massive graph of relationships among them. <br />                  These relationships are sometimes simple (e.g. synonyms) but are often more complex (e.g. one- <br />                  <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span>-many relationships in protein family membership). <br />                  Results: We have built a software system called LinkHub using Semantic Web RDF that manages <br />                  the graph of identifier relationships and allows exploration with a variety of interfaces. For <br />                  efficiency, we also provide relational-database access and translation between the relational and <br />                  RDF versions. LinkHub is practically useful in creating small, local hubs on common topics and then <br />                  connecting these to major portals in a federated architecture; we have used LinkHub to establish <br />                  such a relationship between UniProt and the North East Structural Genomics Consortium. <br />                  LinkHub also facilitates queries and access to information and documents related to identifiers <br />                  spread across multiple databases, acting as &quot;connecting glue&quot; between different identifier spaces. <br />                  We demonstrate this with example queries discovering &quot;interologs&quot; of yeast protein interactions <br />                  in the worm and exploring the relationship between gene essentiality and pseudogene content. We <br />                  also show how &quot;protein family based&quot; retrieval of documents can be achieved. LinkHub is available <br />                  at hub.gersteinlab.org and hub.nesg.org with supplement, database models and full-source <span id='am-6' about='Thesaurus:code' typeof='owl:Thing'>code</span>. <br />                  Conclusion: LinkHub leverages Semantic Web standards-based integrated data to provide novel <br />                  information retrieval to identifier-related documents through relational graph queries, simplifies <br />                  and manages connections to major hubs such as UniProt, and provides useful interactive and query <br />                  interfaces for exploring the integrated data. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                Page 1 of 12 <br />                                                                                                                                                                                                                      (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                    http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br />  <br /> Background                                                      records also contain cross-references to related informa- <br /> Biological research is producing vast amounts of data (e.g.     tion in other databases, e.g. related Gene Ontology and <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> high throughput experiments such as sequencing             PFAM identifiers (although the relationship types, e.g. <br /> projects, and microarray experiments) at a prodigious           &quot;functional annotation&quot; and &quot;family membership&quot; <br /> rate. Most of this data is made freely available to the pub-    respectively, are not specified). Two identifiers such as <br /> lic, and this has created a large and growing number of         Q60996 and GO:0005634 and the cross-reference <br /> internet and world wide web-accessible biological data          between them can be viewed as a single edge between two <br /> resources which are characterized by being distributed,         nodes in a graph, and conceptually then a large, impor- <br /> heterogeneous, and having a large size variance, i.e. huge,     tant part of biological knowledge can be viewed as a mas- <br /> mega-databases such as UniProt [1] down to medium,              sive graph whose nodes are biological entities such as <br /> small or &quot;boutique&quot; databases (e.g., TRIPLES [2]) gener-        proteins, genes, etc. represented by identifiers and the <br /> ated for medium or small scale experiments or particular        links in the graph are typed and are the specific relation- <br /> purposes. Most computational analyses of biological data        ships among the biological entities. Figure 1a is a concep- <br /> will require using multiple integrated datasets, and inte-      tual illustration of the graph of biological identifier <br /> grated data along with rich, flexible and efficient inter-      relationships; the problem is that this graph only con- <br /> faces to it encourages exploratory data analysis which can      cretely exists piecemeal or not at all. <br /> lead to serendipitous new discoveries: the sum is greater <br /> than its parts. Currently, integration often must be done       A basic problem preventing this graph of relationships <br /> manually in a labor and time intensive way by finding rel-      from being more fully realized is the problem of nomen- <br /> evant datasets, understanding them, writing <span id='am-87' about='Thesaurus:code' typeof='owl:Thing'>code</span> to com-        clature. Often, there are many synonyms for the same <br /> bine them, and finally doing the desired analysis. The          underlying entity caused by independent naming, e.g. <br /> basic requirements for better, more seamless integrated         structural genomics centers assigning their own protein <br /> analysis are uniformity and accessibility; data are ineffec-    identifiers in addition to UniProt&apos;s. There can also be lex- <br /> tual if scattered among incompatible resources.                 ical variants of the same underlying identifier (e.g. <br />                                                                 GO:0008150 vs. GO0008150 vs. GO-8150). Synonyms <br /> Web search engines and hyperlinks are the basic and com-        are a small part of the overall problem, however, and <br /> monly used ways to find things on the web and navigate          more generally there are many kinds of relationships <br /> web content but they do not enable fine-grained cross-site      including one-to-one and one-to-many relationships. For <br /> analysis of data. To improve upon this, one key issue is the    example, a single Gene Ontology or PFAM identifier can <br /> need for standardization and its widespread use, and tools      be related with many UniProt identifiers (i.e. they all <br /> supporting and enabling it. Biological data is too vast for     share the same functional annotation or family member- <br /> brute-force centralization to be the complete solution to       ship). PFAM represents an important structuring principle <br /> data interoperability. We must have standards and sys-          for proteins and the genes they come from, the notion of <br /> tems for people and groups to work independently creat-         families (or domains) based on evolution; proteins shar- <br /> ing and making data available (although ultimately              ing common PFAM domains are evolutionarily related <br /> cooperatively and collaboratively) but still in the end all     (called homologs) and likely have the same or similar func- <br /> or most of the pieces of biological knowledge and data are      tions. Gene Ontology consists of three widely used struc- <br /> connected together in semantically rich ways. The W3C&apos;s         tured, controlled vocabularies (ontologies) that describe <br /> [3]Semantic Web [4-6] is a promising candidate: it allows       gene products such as proteins in terms of their associated <br /> web information to be expressed in fine-grained struc-          biological processes, cellular components and molecular <br /> tured ways so applications can more readily and precisely       functions in a species-independent manner. The concep- <br /> extract and cross-reference key facts and information from      tual graph of identifier relationships is richly connected, <br /> it without having to worry about disambiguating meaning         and a transitive closure even a few levels deep can lead to <br /> from natural language texts. Standard and machine-read-         indirect relationships with a great number of other enti- <br /> able ontologies such as the Gene Ontology [7] are also          ties. Being able to store, manage, and work with this graph <br /> created and their common use encouraged to further              of entities and relationships can lead to many opportuni- <br /> reduce semantic ambiguity, although there is a need to          ties for interesting exploratory analysis and LinkHub is <br /> make these ontologies more machine-friendly [8].                such a system for doing this. <br />  <br /> A key abstraction or &quot;scaffold&quot; for representing biological     LinkHub: a system for loosely coupled, collaborative <br /> data is the notion of unique identifiers for biological enti-   integration of biological identifier relationships <br /> ties and relationships among them. For example, each            The Semantic Web is increasingly gaining traction as the <br /> protein sequence in the UniProt database is given a             key standards-based platform for biological data integra- <br /> unique accession, e.g. Q60996, which can be used as a key       tion [9,10], and since LinkHub is a natural fit to Semantic <br /> to access its UniProt sequence record. UniProt sequence         Web technologies we use them as the basis of LinkHub. <br />  <br />  <br />                                                                                                                    Page 2 of 12 <br />                                                                                              (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                        http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br />  <br />    A) <br />  <br />  <br />  <br />  <br />    B) <br />  <br />  <br />  <br />  <br /> Figureof1 relationships and hub of hubs organization <br /> Graph <br /> Graph of relationships and hub of hubs organization. (a) A conceptualization of the semantic graph of interrelationships <br /> among biological identifiers, with boxes representing instances of biological identifiers (originating database names given inside) <br /> and different edge types representing different kinds of relationships (b) LinkHub as an enabler of an efficient &quot;hub of hubs&quot; <br /> organization of biological data. The different colors represent different labs, organizations, or logical groupings of data <br /> resources. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 3 of 12 <br />                                                                                                    (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                    http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br /> LinkHub is designed based on a semantic graph model,            to identifier nodes in the graph based on the graph&apos;s rela- <br /> which captures the graph of relationships among biologi-        tional structure. Normal search engines relying on key- <br /> cal entities discussed above. To provide a scalable imple-      word searches could not provide such access, and <br /> mentation while also exploring Semantic Web database            LinkHub thus enables novel information retrieval to its <br /> technologies, we implemented LinkHub in both MySQL              known web documents. An important practical use of this <br /> [11] and Resource Description Framework or RDF [12]             &quot;path type&quot; interface is as a secondary, orthogonal inter- <br /> databases. LinkHub provides various interfaces to interact      face to other biological databases in order to provide dif- <br /> with this graph, such as a web frontend for viewing and         ferent views of their underlying data. For example, <br /> traversing the graph as a dynamic expandable/collapsible        MolMovDB [13] provides movie clips of likely 3D <br /> HTML list (see figure 2) and a mechanism for viewing par-       motions of proteins, and one can access it by PDB [14] <br /> ticular path types in the graph, as well as with RDF query      identifiers. However, an alternative useful interface (actu- <br /> languages.                                                      ally provided by LinkHub) is a &quot;family view&quot; where one <br />                                                                 queries with a PDB identifier and can view all available <br /> Centralized data integration to an extent does make sense,      motion pages for proteins in the same family as the query <br /> e.g. a lab or organization might want to create a local data    PDB identifier. LinkHub also provides a similar &quot;family <br /> warehouse of interconnections among its individual data         view&quot; interface to structural genomics data in the SPINE <br /> resources; but it does not want to have to explicitly con-      system [15]. The system is flexible and one can easily <br /> nect its data resources up to everything in existence, which    imagine other similar applications, e.g. a &quot;functional <br /> is impossible. The key idea is that if groups independently     view&quot; where all pages for proteins that have the same <br /> maintaining data resources each connect their resources         Gene Ontology function as a given protein are shown or a <br /> up to some other resource X, then any of them can reach         &quot;pseudogene family view&quot; where all pages for pseudo- <br /> any other through these connections to X, and we can col-       genes of proteins in the same family are shown. While the <br /> lectively achieve incremental global data integration in        &quot;path type&quot; interface is a simple way of providing novel, <br /> this way. LinkHub is a software architecture and system         relational access to LinkHub identifier node-linked docu- <br /> which aims to help realize this goal by enabling one to         ments, RDF query language access to the LinkHub rela- <br /> create such local minor hubs of data interconnections and       tional graph would allow the most flexible novel <br /> connect them to major hubs such as UniProt in a feder-          information retrieval. <br /> ated &quot;hub of hubs&quot; framework and this is illustrated in fig- <br /> ure 1b.                                                         Cross-database RDF queries <br />                                                                 To demonstrate the data interaction and exploration capa- <br /> Paper organization                                              bilities engendered by the RDF version of LinkHub, the <br /> In the results section next, we will demonstrate how            RDF-formatted LinkHub dataset is loaded into our Yeast- <br /> LinkHub enables novel information retrieval to docu-            Hub [16] system which uses Sesame [17] as its native RDF <br /> ments attached to LinkHub graph nodes based on the              repository. Two demonstration queries below written in <br /> relational structure of the LinkHub graph; a particular         SeRQL (Sesame implementation of RQL) [18] demon- <br /> practical use case of this, providing &quot;family views&quot; to data,   strate one can efficiently do the kinds of interesting pre- <br /> will be given. We will then give concrete examples of the       liminary scientific investigation and exploratory analysis <br /> kinds of integrated, cross-database queries that can be         commonly done at the beginning of research initiatives <br /> done with LinkHub, in combination with a previous sys-          (e.g. to see whether they are worth pursuing further). <br /> tem of ours called YeastHub, in support of scientific           These queries make use of information present in both <br /> exploratory analysis; example queries discovering &quot;inter-       YeastHub and LinkHub (and thus could not be done <br /> ologs&quot; of yeast protein interactions in the worm and            without joining the two systems), and LinkHub is used as <br /> exploring the relationship between gene essentiality and        &apos;glue&apos; to provide connections (both direct and indirect) <br /> pseudogene content will be given. We will then discuss          between different identifiers. It is noteworthy that these <br /> related work to LinkHub and future directions before con-       queries can be formulated and run in relatively little time <br /> cluding. In the methods section we describe implementa-         (a few hours at most) and they roughly duplicate some <br /> tion details of LinkHub, including its data models and          results from published papers. In effect, LinkHub does the <br /> how they are populated with data and LinkHub&apos;s web              up-front time-consuming manual work of integrating <br /> interactive and query interfaces.                               multiple datasets, and this integrated data is generally use- <br />                                                                 ful for efficient formulation and execution of queries, <br /> Results                                                         which is in contrast to the papers which likely required <br /> Novel information retrieval based on LinkHub relational         extensive &quot;one-off&quot; effort to combine the necessary data to <br /> graph structure                                                 achieve their results. <br /> The &quot;path type&quot; interface to LinkHub allows one to flexi- <br /> bly retrieve useful subsets of the web documents attached <br />  <br />  <br />                                                                                                                     Page 4 of 12 <br />                                                                                               (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                          http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br />  <br /> Figure <br /> The  basic2 DHTML list interface to LinkHub <br /> The basic DHTML list interface to LinkHub. Here, the data and relationships for UniProt identifier P26364 are pre- <br /> sented. P26364 is presented at the root of the list, and lower levels contain information on additional related identifiers. Each <br /> identifier has two subsections: Links which gives a list of hyperlinks to web documents directly relevant to the identifier; and <br /> Equivalent or Related Ids which contains a list of additional identifiers related to the first identifier (the relationship type if it <br /> exists is given in parentheses; a <span id='am-1' about='Thesaurus:Synonym' typeof='owl:Thing'>synonym</span> relationship is assumed if no relationship is given). The identifiers in the Equivalent and <br /> Related Ids section may themselves be further related to other identifiers which will have their own Links and Equivalent or <br /> Related Ids sections, ad nauseum. The initial display shows the transitive closure of the root identifier one level deep, and <br /> dynamic callbacks to the server retrieve additional data when the user clicks on identifiers whose subsections have not yet <br /> been loaded; in this way, the user can explore the relationship paths he desires without performance penalties (of loading the <br /> whole graph) or &apos;information overload&apos;. The interface is dynamic, and a &apos;+&apos; list icon can be expanded to view the hidden under- <br /> lying content, and a &apos;-&apos; list icon can be clicked to hide the content. <br />  <br />  <br /> Query 1: finding worm &apos;interologs&apos; of yeast protein             For each essential yeast gene we then determine how <br /> interactions                                                    many pseudogenes it has. We can then inspect the list of <br /> Proteins rarely act in isolation and often interact with one    essential genes to see if there is a relationship between <br /> another and other molecules to perform necessary cellular       essentiality and number of pseudogenes. Humans have a <br /> actions. Experimental determinations of protein interac-        large number of known pseudogenes [25] but gene essen- <br /> tions are expensive and computational methods can lev-          tiality is difficult to characterize in humans (with many <br /> erage them for further interaction predictions. With this       tissue types and developmental states complicating the <br /> query we want to consider all the protein interactions in       issue). Since essentiality is well studied in yeast, one thing <br /> yeast (S. cervisiae) and see how many and which of them         we can do is determine the human homologs of yeast <br /> are present as evolutionarily related homologs in worm          essential genes, which would perhaps likely be &quot;more <br /> (C. elegans), also known as interologs [19], i.e. protein       important&quot; in a <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>survival</span> sense, and examine them for pat- <br /> pairs in worm corresponding to evolutionarily related           terns associated with essentiality. For each yeast gene <br /> known interacting pairs in yeast. We thus start with a data-    name in the list of essential genes, we can find the homol- <br /> set containing known and predicted yeast protein interac-       ogous pseudogenes in human by traversing identifier type <br /> tions which is already loaded into YeastHub; here the           paths in the LinkHub graph like the following: <br /> interactions are expressed between yeast gene names. Part <br /> of the SeRQL statement for this query together with a por-      yeast gene name &#226;&#8224;&#8217; UniProt Accession &#226;&#8224;&#8217; Pfam accession <br /> tion of its corresponding output can be seen in figure 3.       &#226;&#8224;&#8217; human UniProt Id &#226;&#8224;&#8217; UniProt Accession &#226;&#8224;&#8217; Pseudog- <br /> However, abstractly, the query is doing the following. For      ene LSID <br /> each yeast gene name in the interaction set we can use <br /> LinkHub&apos;s data as &apos;glue&apos; to determine its homologs (via         Part of the SeRQL for the first query (for yeast pseudo- <br /> Pfam) in worm by traversing identifier type paths in the        genes) and results from both can be seen in figure 3, and <br /> LinkHub relationship graph like the following:                  they show that few yeast essential genes are associated <br />                                                                 with pseudogenes whereas this is not the case with <br /> yeast gene name &#226;&#8224;&#8217; UniProt Accession &#226;&#8224;&#8217; Pfam accession            human. This may reflect the difference in processes of cre- <br /> &#226;&#8224;&#8217; UniProt Accession &#226;&#8224;&#8217; WormBase ID.                              ation of the predominate numbers of yeast and human <br />                                                                 pseudogenes (duplication vs retrotransposition, see <br /> Then, for each pair in the yeast protein interaction dataset,   [21,22]). <br /> we determine if both of its yeast gene names lead to <br /> WormBase IDs [20] in this way and print those WormBase          Discussion <br /> IDs as possible protein interactions if so.                     Related work <br />                                                                 The basic conceptual underpinnings of LinkHub, i.e., the <br /> Query 2: exploring pseudogene content versus gene               importance of biological identifiers and linking them, was <br /> essentiality in yeast and humans                                given by Karp [26]. LinkHub uses a Semantic Web <br /> Pseudogenes are genomic DNA sequences similar to nor-           approach to build a practical system based on and extend- <br /> mal genes (and usually derived from them) but are not           ing Karp&apos;s ideas on database links. The Semantic Web <br /> expressed into functional proteins; they are regarded as        approach can also be used to implement database integra- <br /> defunct relatives of functional genes [21,22]. In the que-      tion solutions based on the general approaches of data <br /> ries here we explore the relationship between gene essen-       warehousing [27,28] and federation [29-31]. Essentially, <br /> tiality (a measure of how important a gene is to survival of    data warehousing focuses on data translation, i.e. translat- <br /> an organism) and the number of pseudogenes in an                ing and combining multiple datasets into a single data- <br /> organism. We might hypothesize that more essential              <span id='am-86' about='xsp:base' typeof='owl:Thing'>base</span>, whereas federation focuses on query translation, i.e. <br /> genes might have larger numbers of pseudogenes, and we          translating and distributing the parts of a query across <br /> explore this idea with queries of the joined YeastHub and       multiple distinct databases and collating their results into <br /> LinkHub data. First, YeastHub has the MIPS [23] Essential       one. A methodological overview and comparison of these <br /> Genes dataset, and we use this as our data on gene essen-       database integration approaches was discussed in the bio- <br /> tiality; LinkHub contains a small dataset of yeast pseudo-      medical context [32]. LinkHub&apos;s architecture is a hybrid of <br /> genes [24].                                                     these two approaches: individual LinkHub instantiations <br />                                                                 are a kind of mini, local data warehouse of commonly <br /> Abstractly, for each yeast gene name in the list of essential   grouped data and these are connected to large major hubs <br /> genes, we determine its pseudogenes by traversing identi-       such as UniProt in a federated fashion; efficiency is gained <br /> fier type paths in the LinkHub graph like the following:        by obviating the need for all source datasets to be individ- <br />                                                                 ually connected to the major hubs. <br /> yeast gene name &#226;&#8224;&#8217; UniProt Accession &#226;&#8224;&#8217; yeast pseudog- <br /> ene <br />  <br />  <br />                                                                                                                     Page 6 of 12 <br />                                                                                               (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                         http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br />  <br /> Figure 3RDF queries <br /> Example <br /> Example RDF queries. (a) shows a part of the SeRQL query that finds pairs of worm (C. elegans) proteins homologous to <br /> pairs of interacting proteins in yeast (C. cervisiae), i.e. &quot;interologs&quot;. b) shows part of the corresponding query results. (c) shows <br /> the SeRQL query that explores the relationship between gene essentiality and the level of pseudogene content in yeast, which <br /> is one feature that might be hypothesized to be associated with essentiality, with queries of the joined YeastHub and LinkHub <br /> data. (d) shows the yeast pseudogenes found, interestingly only one. (e) shows part of the list of pseudogenes found in human <br /> homologs for a similar query; the full list is long, around 20000, consistent with there being many known pseudogenes in <br /> humans. <br />  <br />  <br />  <br />  <br />                                                                                                                           Page 7 of 12 <br />                                                                                                     (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                   http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br /> LinkHub differentiates itself by not integrating all aspects   fier nodes, e.g. proteomics identifiers, in the graph. A <br /> of biological data but rather focusing on an important and     simple search for the identifier itself would likely not give <br /> more manageable high-level structuring principal, namely       optimal results due to conflated senses of the identifier <br /> biological identifiers and the relationships (and relation-    text and identifier synonyms. In general, we need to con- <br /> ship types) among them; hyperlinks to identifier-specific      sider and query for the key related concepts of an identi- <br /> pages present in the &quot;Links&quot; section of the LinkHub web        fier, and these are present in the LinkHub subgraph <br /> interface give access to additional attributes and data. In    surrounding the identifier. We consider the web pages <br /> fact, our YeastHub system addressed integration more           attached to the identifiers in the subgraph as a &quot;gold <br /> generally by transforming many datasets to common RDF          standard&quot; for what additional relevant documents should <br /> format and storing and giving RDF query access to them         be like, and we plan to use them as training sets to con- <br /> in an RDF database. The problem with YeastHub was that         struct classifiers used to score and rank additional docu- <br /> the integration was thin, with rich connections among the      ments for relevance. We feel that this idea could be <br /> integrated datasets being limited. LinkHub is thus useful      generalized and that the Semantic Web, which provides <br /> and complementary to YeastHub in this respect as a &quot;con-       detailed information about terms and their relationships, <br /> necting glue&quot; among the datasets in that it makes and          could be leveraged to provide enhanced automated infor- <br /> stores these cross-references and enables better integrated    mation retrieval or web search for Semantic Web terms. <br /> access to the YeastHub data; the example queries above <br /> demonstrated this.                                             We also hope to explore how other relevant Semantic <br />                                                                Web-related technologies could be effectively used in <br /> LinkHub&apos;s primary web interface can be viewed as a kind        LinkHub, in particular named graphs [37] and Life Sci- <br /> of &quot;Semantic Web browser&quot;. Other work has also                 ence IDentifiers or LSIDs [38]. Named graphs allow RDF <br /> attempted to build browsers for Semantic Web data,             graphs to be named by URI, allowing them to be <br /> including HayStack [33], Sealife [34], and BioGuide [35].      described by RDF statements; named graphs could be <br /> LinkHub is a more lightweight browser than HayStack            used to provide additional information (metadata) about <br /> (with a focus on biological relationship browsing) and         identifier mappings, such as source, version, and quality <br /> differs from Sealife by being data-centric (establishing       information. LSID is a standard object naming and dis- <br /> semantic links between data identifiers while treating web     tributed lookup mechanism being promoted for use on <br /> documents as metadata associated with the identifiers) as      the Semantic Web, with emphasis on life sciences applica- <br /> opposed to document-centric (establishing semantic links       tions. An LSID names and refers to one unchanging data <br /> between terms/phrases appeared in different web docu-          object, and allows versioning to handle updates. The LSID <br /> ments). BioGuide uses RDF similar to LinkHub, but it is        lookup system is in essence like what Domain Name Serv- <br /> limited in that it focuses on abstract conceptual modelling    ice (DNS) does for converting named internet locations to <br /> of resources and their interconnections rather than on         IP numbers. We could possibly use LSID for naming <br /> instance data as LinkHub; also its interface presents the      objects in LinkHub and incorporate LSID lookup func- <br /> data using graph drawing software with Java, whereas           tionality. Finally, like software such as Napster and <br /> LinkHub is more lightweight and relies only on the web         Gnutella did for online file sharing, we plan to explore <br /> browser with JavaScript. Finally, there have been a            enhancing LinkHub to enable multiple distributed <br /> number of graph database systems and query languages           LinkHub instantiations to interact in peer-to-peer net- <br /> developed through the years but they suffer from being         works for dynamic biological data sharing, possibly using <br /> proprietary; none have developed into widely used stand-       web services technologies such as Web Services Descrip- <br /> ard systems. However, it should be pointed out that some       tion Language (or WSDL) [39] and Universal Description, <br /> of these systems support advanced graph data mining and        Discovery and Integration (or UDDI) [40] for dynamic <br /> analysis operations not supported by RDF query lan-            service discovery, and available peer-to-peer toolkits. <br /> guages and these features might be necessary for effective <br /> analysis of biological data represented in RDF [36].           Conclusion <br />                                                                Our paper demonstrates the natural use of Semantic Web <br /> Future directions                                              RDF to inter-connect identifiers of data entries residing in <br /> Currently, LinkHub has limited web document hyperlinks         separate web-accessible biological databases. Based on <br /> attached to its nodes, and if this could be increased the      such a semantic RDF graph of biological identifiers and <br /> utility of the novel information retrieval based on query-     their relationships, useful, non-trivial cross-database que- <br /> ing the LinkHub relational graph, e.g. &quot;path type&quot; inter-      ries, inferences, and semantic data navigation can be per- <br /> face, would be enhanced. We are working to leverage the        formed through web interactive and query access. In <br /> rich information in the LinkHub relational graph for           addition, these semantic relationships enable flexible and <br /> enhanced automated information retrieval to web or sci-        novel information retrieval access based on queries of the <br /> entific literature (MedLine) documents relevant to identi-     LinkHub graph&apos;s relational structure to web documents <br />  <br />  <br />                                                                                                                    Page 8 of 12 <br />                                                                                              (page number not for citation purposes) <br />  BMC Bioinformatics 2007, 8(Suppl 3):S5                                 http://www.biomedcentral.com/1471-2105/8/S3/S5 <br />  <br />  <br />  <br /> attached to identifier nodes. LinkHub also can simplify      related information were extracted from the sources and <br /> and manage connections to major hubs such as UniProt         inserted into the LinkHub MySQL database (which is also <br /> for a lab or organization. LinkHub can be evaluated by       converted to RDF and inserted into the RDF version of <br /> considering its current active and practical use in a        LinkHub; see below). A running instantiation of the <br /> number of settings. We have already established the &quot;hub     LinkHub system is at http://hub.gersteinlab.org and http:/ <br /> of hubs&quot; relationship between UniProt and LinkHub (i.e.      /hub.nesg.org, and it is actively used and populated with <br /> UniProt cross-references to our LinkHub). In addition,       data from the Gerstein Lab [47] and related to the lab&apos;s <br /> LinkHub cross-references the targets of the structural       research interests. Thus while the ideas of LinkHub can be <br /> genomics initiative to UniProt and serves as a &quot;related      applicable more generally to biological data, the concrete <br /> links&quot; and &quot;family viewer&quot; gateway for the Northeast         instantiation of LinkHub focuses heavily on proteomics <br /> Structural Genomics Consortium with which we are affil-      data, as that is a key research initiative of the Gerstein Lab. <br /> iated; LinkHub also serves as the &quot;family viewer&quot; for Mol-   The &quot;hub of hubs&quot; relationship described above has <br /> MovDB. LinkHub is a step towards answering the               already been established between UniProt and LinkHub <br /> question &quot;a life science Semantic Web: are we there yet?&quot;    (i.e. UniProt hyperlinks to the LinkHub instantiation and <br /> [41].                                                        cross-references to it in its DR lines). In addition, LinkHub <br />                                                              cross-references the proteins which are targets of the struc- <br /> Methods                                                      tural genomics initiative (obtained from the TargetDB <br /> Obtaining LinkHub data                                       resource [48]) to UniProt and the LinkHub instantiation <br /> A key problem in populating the LinkHub database             serves as a &quot;related links&quot; and &quot;family viewer&quot; (more <br /> (described below) is how to determine the relationships      below) gateway for the Northeast Structural Genomics <br /> among biological identifiers, a specific case of the so-     Consortium (NESG) [49] with which the Gerstein Lab is <br /> called ontology alignment problem [42,43]. Biology is        affiliated. Additional focuses of the LinkHub instantiation <br /> blessed with a fundamental, commonly accepted princi-        are yeast resources, macromolecular motions [13], and <br /> ple around which data can be organized, namely biologi-      pseudogenes [50]. <br /> cal sequences such as DNA, RNA, and protein, and various <br /> <span id='am-4' about='xsd:string' typeof='owl:Thing'>string</span> matching techniques (such as dynamic program-         LinkHub database models <br /> ming [44] and BLAST [45]) for biological sequences can       LinkHub is conceptually based on the Semantic Web <br /> solve a large part of the ontology alignment problem in      (graph) model and we thus represent it and store it in <br /> biology. LinkHub thus takes advantage of biological          RDF. RDF is a popular data model (or ontological lan- <br /> sequence matching, in particular conservative, exact         guage) for the Semantic Web that represents data as a <br /> sequence matching, to cross-reference or align biological    directed labelled graph. Essentially, in RDF URIs [51] are <br /> identifiers. LinkHub also takes advantage of available       used for globally unique naming of the nodes (which rep- <br /> sources of pre-computed identifier mappings, with the        resent objects) and the edges (which represent relation- <br /> most important one being UniProt which is arguably the       ships between nodes) of the graph, and literal values may <br /> most important major proteomics resource and serves as       also be used in place of pointed to nodes. In addition, <br /> LinkHub&apos;s backbone content (i.e. most relationships          RDF comes with query languages (e.g., RDQL [52]) to <br /> between identifiers in LinkHub are indirect through Uni-     allow the user to pose semantic queries against graph <br /> Prot). The general strategy for mapping identifiers in       data. While there are more advanced ontological lan- <br /> LinkHub is to first take advantage of known and trusted      guages such as the Web Ontology Language or OWL [53] <br /> pre-computed identifier mappings; if such pre-computed       that support data reasoning based on Description Logics <br /> mappings are unavailable, an attempt is made to map          or DL [54], RDF is easy to learn and use and much can be <br /> identifiers based on exact sequence matches of their         effectively modelled with it. For example, the benefits of <br /> underlying sequences to UniProt and other sources of         representing proteomics data in RDF were discussed [9] <br /> sequence data whose identifiers are stored in LinkHub.       and UniProt data has also recently been made available in <br />                                                              RDF format [55]. However, there could be a potential <br /> Efficient, exact sequence matching programs were devel-      problem in performance and scalability when using the <br /> oped and used to do quick inter-database cross-referenc-     new RDF database technology, which can be an important <br /> ing or alignment based on exact sequence matches (e.g. to    impediment to more active and widespread use of the <br /> cross-reference TargetDB to UniProt, see below). A custom    Semantic Web. In this regard, the creation of high-per- <br /> Perl module was developed and used to index UniProt          formance RDF databases should be a research priority of <br /> (and in general sequence databases in FASTA format [46])     the Semantic Web community. Thus, while we would ide- <br /> to support this fast exact sequence matching. Specialized    ally use only RDF, to support LinkHub&apos;s practical daily use <br /> Perl web crawlers and other scripts were written to fetch    for its web interactive interfaces we also model and store <br /> and extract data from different sources in different for-    its data using relational database technology (MySQL) for <br /> mats; identifiers, identifier relationships, and other       efficiency and robustness. A drawback is that relational <br />  <br />  <br />  <br />  <br /> databases do not naturally model graph structures or pro-        supports grouping of web resources, e.g. all web resources <br /> vide efficient graph operations for which special proce-         maintained by the Gerstein Lab or relating to protein <br /> dural codes are necessary (e.g. for the &quot;path type&quot; view         structure. Finally, the resource_attribute table allows free <br /> described below). It is straightforward mapping between          text attributes to be associated with web resource, however <br /> the relational and RDF versions of LinkHub and we have           it is not currently used. Figure 4 also provides details of <br /> written Java code to do this.                                    the LinkHub RDF model and how it is related to the rela- <br />                                                                  tional model; a simple example RDF graph is also given. <br /> The relational structure of LinkHub, shown in figure 4a, <br /> reflects how the graph of biological identifier relation-        LinkHub web interfaces <br /> ships and associated data, such as URLs of identifier-spe-       The primary interactive interface to the LinkHub database <br /> cific web pages, are managed and stored. Biological              is a web-based interface (implemented using the so-called <br /> identifiers are stored in the identifier table and are typed,    AJAX technologies [56], i.e. DHTML, JavaScript, DOM, <br /> where the identifier_types table gives the type. Thus, for       CSS, etc.) which presents subsets of the graph of relation- <br /> example, two different identifiers in separate databases         ships in a dynamic expandable/collapsible list view. This <br /> which happen to have the same identifier text can never-         interface allows viewing and exploring of the transitive <br /> theless be distinguished by differing identifier types           closure of the relationships stemming from a given iden- <br /> (based on the databases they come from). The mappings            tifier interactively one layer at a time: direct edges from the <br /> table is used to store the relationships between identifiers,    given identifier are initially shown and the user may then <br /> with the &quot;type&quot; attribute giving the description or mean-        selectively expand fringe nodes an additional layer at a <br /> ing of the relationship. The identifier table thus gives the     time to explore further relationships (computing the full <br /> nodes and the mappings table the edges of the graph of           transitive closure is prohibitive, and could also cause the <br /> biological identifier relationships. The resource,               user to &quot;drown&quot; in the data, and we thus limit it initially, <br /> resource_accepts, and link_exceptions tables together            and in each subsequent expansion, to anything one edge <br /> manage and store URLs for identifier-specific web pages          away, with the user then guiding further extensions based <br /> (e.g. the web page at UniProt giving specific information        on which relationships he would like to explore). Figure 2 <br /> particular to some UniProt identifier). The basic idea is        is a screenshot of the interface and gives more details of it. <br /> that web resources such as UniProt have template URLs            The second, &quot;path type&quot; interface presents results the same <br /> which can be interpolated with particular identifiers to         as the first interface (i.e. dynamic expandable/collapsible <br /> generate identifier-specific URLs. The resource table con-       list view) but allows users to query and view particular <br /> tains a short name, longer description, and the template         identifier type paths in the LinkHub graph. For example, <br /> URL of web resources such as UniProt. The                        one might want to view all proteins in some database in <br /> resource_accepts table lists the particular types of identifi-   the same Pfam family as a given protein; in LinkHub Pfam <br /> ers that can be interpolated into a resource&apos;s template          relationships are stored for UniProt proteins, so one could <br /> URL, as well as an exception type except_type. The excep-        view the fellow family members of the given protein by <br /> tion type is to handle cases where not all identifiers of an     specifying to view all relational paths in the LinkHub <br /> accepted type are legal, i.e. some of the identifiers cannot     graph whose identifier types match: <br /> be interpolated into the template URL to generate a valid <br /> URL. If except_type is NONE then there are no exceptions         Given protein in database &#226;&#8224;&#8217; equivalent UniProt protein <br /> and all identifiers of the type are accepted. Otherwise          &#226;&#8224;&#8217; Pfam family &#226;&#8224;&#8217; UniProt proteins &#226;&#8224;&#8217; other equivalent <br /> except_type has value NACC or ACC. If except_type is             proteins in database. <br /> NACC, then the exceptions are explicitly given in the <br /> link_exceptions table (i.e. the identifiers in the               Competing interests <br /> link_exceptions table of the given type for the resource are     The authors declare that they have no competing interests. <br /> the ones that cannot be interpolated into the template <br /> URL, and all other identifiers of the type CAN be interpo-       Authors&apos; contributions <br /> lated). If except_type is ACC then the behaviour is the          AKS is the primary (first) author and is responsible for the <br /> opposite: the identifiers NOT listed in the link_exceptions      majority of the work, implementation, and writing. KYY <br /> table are the exceptions and the ones explicitly listed are      did the conversion of the relational (MySQL) version of <br /> the only ones that can be interpolated into the resource&apos;s       LinkHub to RDF, integrated it into YeastHub, and wrote <br /> template URL. NACC and ACC exception types are both              and executed the demo RDF queries over the joined Yeast- <br /> supported to allow the most efficient handling of excep-         Hub/LinkHub. KHC, MS, and MBG are faculty advisors <br /> tions, i.e. whichever is smaller between the set of accepted     and provided high-level direction and guidance to the <br /> identifiers and the set of exception identifiers can be listed   work. <br /> in link_exceptions thus minimizing the amount of space <br /> necessary for storing exceptions. The resource_group table <br />  <br />  <br />  <br />  <br /> Figure 4relational and RDF data models <br /> LinkHub <br /> LinkHub relational and RDF data models. (a) LinkHub relational model (b) An example RDF graph of two statements or <br /> triplets: (Q60996, gene_annotation, GO:0005634) and (Q60996, protein family, PF01603), which describe that there is a pro- <br /> tein (Q60996) whose gene annotation is identified by GO: 0005634 and protein_family by PF01603. This also exemplifies using <br /> an RDF graph to connect multiple resources. Here, it connects UniProt, Gene Ontology, and Pfam. The detailed description <br /> associated with each identifier can be provided by the corresponding resource (the URL or URI can provide access to such <br /> detailed descriptions). (c) Illustrates how the key LinkHub relational tables identifier_types, identifiers, and mappings (repro- <br /> duced above the RDF structure) are mapped to the corresponding RDF structure. The resulting RDF graph captures different <br /> types of object identifiers stored in different databases and the relations (or mappings) between these object identifiers. The <br /> mapping types are explicitly represented as RDF properties. (d) Shows how the rest of the LinkHub relational tables (repro- <br /> duced above the RDF structure) map to the RDF structure. The resulting RDF graph captures the different web resources <br /> (which can be grouped) accessible by LinkHub. In addition, the graph captures information about which web resources accept <br /> which types of object identifiers, as well as exceptions. <br /> </body></html>