<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 17, 4736 &#226;&#8364;&#8220; 4747, November 2006 <br />  <br />  <br /> A Survey Essential Gene Function Yeast Cell <br /> Division Cycle&#226;&#8211;&#161; <br />               D <br />  <br />  <br /> Lisa Yu,*&#226;&#8364;&#160; Lourdes Pen&#203;&#339;a Castillo,&#226;&#8364;&#160;&#226;&#8364;&#161; Sanie Mnaimneh,&#226;&#8364;&#160;&#226;&#8364;&#161; Timothy R. Hughes,&#226;&#8364;&#160;&#226;&#8364;&#161; <br /> and Grant W. Brown*&#226;&#8364;&#160; <br />  <br /> *Department Biochemistry, University Toronto, Toronto, Ontario, Canada M5S 1A8; &#226;&#8364;&#161;Banting Best <br /> Department Medical Research, University Toronto, Toronto, Ontario, Canada M5G 1L6; &#226;&#8364;&#160;Donnelly <br /> Centre Cellular Biomolecular Research, University Toronto, Toronto, Ontario, Canada M5S 3E1 <br />  <br /> Submitted May 1, 2006; Revised August 4, 2006; Accepted August 16, 2006 <br /> Monitoring Editor: Trisha Davis <br />  <br />  <br />  <br /> Mutations impacting specific stages cell growth division provided foundation dissecting mechanisms <br /> that underlie cell cycle progression. We undertaken objective examination yeast cell cycle flow <br /> cytometric analysis DNA content TetO7 promoter mutant strains representing 75% essential yeast genes. More <br /> than 65% strains displayed specific alterations DNA content, suggesting reduced function essential <br /> gene cases impairs progression specific stage cell cycle. Because large number essential <br /> genes required protein biosynthesis, G1 accumulation common <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> observed analysis. In <br /> contrast, relatively mutants displayed S-phase delay, defective genes required DNA <br /> replication nucleotide metabolism. G2 accumulation appeared arise variety defects. In addition <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> providing global view diversity essential cellular processes influence cell cycle progression, data also <br /> provided predictions regarding functions individual genes: identified new genes involved protein <br /> trafficking (NUS1, PHS1, PGA2, PGA3), CSE1 SMC4 important DNA replication. <br />  <br />  <br /> INTRODUCTION                                                           et al., 1970; Culotti Hartwell, 1971; Hartwell, 1971a, <br />                                                                        1971b, 1973; Moir et al., 1982). These genetic screens, using <br /> Cell division fundamental biological process          conditional temperature-sensitive mutants, identified more <br /> organisms consists series closely coordinated events.          50 genes required specific stages cell <br /> In budding yeast Saccharomyces cerevisiae, cell divi-          division cycle termed CDC genes. On average <br /> sion cycle begins initial growth phase, G1,             4.6 alleles identified CDC gene original <br /> which time cell increases mass volume. The                  screens, suggesting number cdc mutants be <br /> transition G1 S phase marked progression               identified approach reached plateau (Hartwell <br /> through Start (the point commitment cell division),              et al., 1973). However, lines evidence suggested <br /> initiation nuclear DNA synthesis, emergence            additional genes cell cycle stage-specific functions <br /> bud form new daughter cell. S phase followed          remained identified (Hartwell et al., 1973; Hartwell, <br /> by second growth phase, G2, turn followed             1974; Pringle Hartwell, 1981). Indeed, cell cycle screens <br /> nuclear division, cell separation. A similar             model organisms identified additional CDC genes <br /> series events occurs eukaryotes, obvious            essential genes (Nurse, 1975; Nurse Bisset, <br /> exception bud emergence. Because fundamental                 1976; Nasmyth Nurse, 1981), recent <br /> biological importance cell cycle progression im-            screens using alternative strategies (Prendergast et al., 1990; <br /> portance human development diseases                Stevenson et al., 2001; Kanemaki et al., 2003). Consistent with <br /> cancer, identification molecular determinants spe-           roles biological process fundamental impor- <br /> cific stages eukaryotic cell cycle subject        tance, CDC genes conserved appear to <br /> intense study decades.                                     conserved function eukaryotes, including hu- <br />    The morphological landmarks cell cycle stages             mans. <br /> budding yeast, notably size bud relative               Several approaches exist studying biological func- <br /> the size mother cell, allows identification mu-          tion essential genes. Temperature-sensitive mutations <br /> tants blocked specific stages cell cycle         used extensively analysis cell cycle genes, <br /> forms basis classic cell cycle screens (Hartwell           temperature-sensitive mutations lead rapid <br />                                                                        depletion gene product analyzed. However, <br />                                                                        temperature-sensitive mutations difficult construct in <br /> This article published online ahead print MBC Press       systematic manner, molecular basis tempera- <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E06 &#226;&#8364;&#8220; 04 &#226;&#8364;&#8220; 0368)       ture sensitivity is, cases, unknown. Induced prote- <br /> on August 30, 2006. <br />                                                                        olysis creation gene fusions sequences <br /> &#226;&#8211;&#161; <br /> D <br />    The online version article contains supplemental material   encoding &#226;&#8364;&#339;N-degron,&#226;&#8364;? temperature-inducible proteo- <br /> at MBC Online (http://www.molbiolcell.org).                            lytic degradation signal (Dohmen et al., 1994), been <br /> Address correspondence to: Grant W. Brown (grant.brown@                used identify cell cycle genes (Kanemaki et al., 2003). The <br /> utoronto.ca) Timothy R. Hughes (t.hughes@utoronto).                 N-degron applied systematically, cases <br />  <br /> 4736                                                                                   &#194;&#169; 2006 The American Society Cell Biology <br />                                                                                                                        Cell Cycle Function Essential Genes <br />  <br />  <br /> leads rapid gene product depletion. However, recent                         etry histograms, ordered position array plates TetO7 promoter <br /> study 104 essential genes fused N-                             collection, available at: http://biochemistry.utoronto.ca/brown/data.html. <br /> degron sequences, nearly 40% essential genes fused                        Manual Analysis Flow Cytometry Profiles <br /> N-degron did result inviability nonpermissive <br />                                                                                     Profiles assigned different classes according following criteria: <br /> temperature (Kanemaki et al., 2003), indicating rapid                          Profiles equal height 1C 2C peaks assigned normal <br /> protein depletion N-degron uniform                         category; profiles 1C peak higher 2C peak assigned to <br /> proteome. Despite limitation, functional information                           1C category; profiles 2C peak higher 1C peak were <br /> could derived strains depletion in-                            assigned 2C category; profiles distinct peaks 3C and/or 4C <br />                                                                                     positions assigned 3C/4C category; profiles accumulation of <br /> complete (Aparicio, 2003; Kanemaki et al., 2003). Essential                         DNA contents 1C 2C assigned S-phase category; <br /> gene function studied systematically gene                            profiles 2C 4C peak classified diploid; profiles with <br /> product depletion using repressible promoter. Promoter                            DNA contents 1C; profiles did fit above <br /> replacement alleles allow systematic analysis essential                      criteria classified &#226;&#8364;&#339;other.&#226;&#8364;? <br /> genes, although, like N-degron fusions, degree gene                          Computational Analysis Flow Cytometry Profiles <br /> product depletion varies gene gene, depending on <br /> both mRNA protein half-life. With promoter replace-                             Normalization. As preprocessing step, consecutively aligned local <br /> ment alleles open reading frame (ORF) remains intact,                          maximum samples DNA content region. The alignment flow <br /> and repression conditions minimal effects cell phys-                        cytometry data follows: 1) select rightmost channel left the <br /> iology chosen. In previous study (Mnaimneh et al.,                         channel representing DNA content region align, the <br />                                                                                     samples DNA content channel (i.e., channel sparse <br /> 2004), described construction tetracycline-regulat-                       region flow cytometry matrix), set channel cutting point; <br /> able promoter (TetO7 promoter) alleles &#226;&#172;&#402;600 essential                            2) locate local maximum fixed range (&#226;&#171;&#190;50 channels) the <br /> genes S. cerevisiae. The TetO7 promoter collection                      channel corresponding selected DNA content; 3) shift vector of <br /> used probe essential gene function cell size control, cell                    values starting cutting point sample local maximum <br />                                                                                     corresponds channel representing selected DNA content; 4) in <br /> morphology, mitochondrial morphogenesis, gene                               case left shifting discard value left cutting point; otherwise, fill <br /> expression synthetic genetic interaction profiling                              created spaces value cutting point. <br /> (Mnaimneh et al., 2004; Altmann Westermann, 2005; <br /> Davierwala et al., 2005). In present study, ex-                         Classification. Profiles assigned different abnormal categories <br /> panded TetO7 promoter collection encompass 773                               according following criteria (thresholds manually obtained based <br />                                                                                     examination data, i.e., computational categorization tuned <br /> essential genes, 75% essential gene set, making                       make decisions manual inspection): <br /> it complete resource systematic analysis of <br /> essential gene function yeast.                                                   1. Other: Profiles local maximum 1C DNA content region is <br />                                                                                        greater 25. <br />    Many genes involved cell growth, cell division, cell                      2. 1C DNA contents: Profiles ratio 1C DNA content 2C DNA <br /> cycle progression indispensable processes                            content greater 1.48. <br /> are included &#226;&#172;&#402;1050 yeast genes essen-                           3. S phase: Profiles local maximum 1C DNA content 2C <br /> tial viability (out total &#226;&#172;&#402;5800 yeast genes). Here,                        DNA content regions greater 27; and, ratio local maxi- <br />                                                                                        mum global maximum greater 0.23; and, ratio local <br /> we utilized TetO7 promoter collection analyze essen-                            maximum mean value greater 2.8. <br /> tial gene function cell division cell cycle progression,                     4. 2C DNA contents: Profiles ratio 2C DNA content 1C DNA <br /> using flow cytometric analysis measure cellular DNA                                 content greater 1.4. <br /> content promoter shut-off. More 65% strains                       5. 3C/4C DNA contents: Profiles local maximum 3C 4C DNA <br />                                                                                        content regions greater 10. <br /> displayed altered flow cytometry profile promoter                          6. Diploids: Profiles local maximum 1C DNA content region &#226;&#172;?20, <br /> shut-off, allowing categorization essential genes                            and, ratio 2C DNA content 1C DNA content greater 1.4, and <br /> basis cell cycle profile. Our systematic analysis                       local maximum 4C DNA content greater 30. <br /> useful illustrating widespread contribution essential                         Cytometry histograms genes did criteria above <br /> genes individual stages cell cycle, useful                    inspected classified normal. <br /> for elucidating functions uncharacterized genes. <br />                                                                                     Functional Analysis. For analysis functional enrichment class, <br />                                                                                     used annotations Biological Process hierarchy Gene <br /> MATERIALS AND METHODS                                                               Ontology (GO) YEAST annotation package 1.10 Bioconductor <br />                                                                                     (Gentleman et al., 2004). Before statistical analysis, GO annotations were <br />                                                                                     up-propagated. GO Categories hypergeometric p value smaller than <br /> Strain Construction                                                                 0.001 considered significantly overrepresented class, because <br /> TetO7 strains constructed described previously (Mnaimneh et al.,            resulted zero false-positive categories average of <br /> 2004). The genotype wild-type TetO7 strain, R1158, MATa URA3::            10,000 sets randomly chosen genes sizes analyzed here. <br /> CMV-tTA his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0. Primers PCR amplification DNA        For visualization, similar significant GO categories joined un- <br /> fragments containing Tet-off promoter, KanR gene partial sequences       der specific common ancestor. <br /> the target genes confirmation correct promoter replacement avail- <br /> able http://hugheslab.med.utoronto.ca/Mnaimneh/data/Primers/Hughes_tet_          G1 Intragroup Functional Analysis. Genes G1 class ordered <br /> promoter_primers_Supp.xls. Two strategies used PCR confirmation            according ratio 1C accumulation peak 2C accumulation peak and <br /> of integrations. The utilized primers genomic DNA outside in-          separated equally sized sets. The functional enrichment each <br /> tended integration site, correct integration yields band &#226;&#172;&#402;2.4       set genes determined described Functional Analysis above. p <br /> kb, incorrect integration &#226;&#172;&#402;400 bases. The second confirmation strategy           values lower quartiles calculated excluding genes upper quartiles. <br /> utilized primer complimentary cassette genomic <br /> DNA, band &#226;&#172;&#402;700 bases represents integration correct <br />                                                                                     Clustering. Genes DNA content classes shown the <br /> site. The smc4-1 strain (MATa smc4-1::kanMX his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 met15&#226;&#338;&#172;0) <br />                                                                                     heatmaps (Figures 2A 3A) separately clustered using Pearson <br /> contains smc4-1 allele (Freeman et al., 2000) integrated marker the <br />                                                                                     correlation similarity measure hierarchical clustering com- <br /> SMC4 locus S288C strain background (Brachmann et al., 1998). <br />                                                                                     plete linkage method. Clustering heatmaps generated R 2.2.0. The <br />                                                                                     genes cell size heatmap (Figure 3A) shown order in <br /> Flow Cytometry                                                                      DNA content figure (Figure 2A). <br /> For screening collection TetO7 promoter alleles, cells grown at <br /> 30&#194;&#176;C absence presence 10 &#226;?&#174;g/ml doxycycline 15 h, &#226;&#172;&#402;1 &#226;&#171;&#187;          Cell Size Analysis. For genes intersection manual and <br /> 107 cells strain harvested fixed 70% ethanol. Cells     computational classifications obtained size bin cumulative <br /> typically harvested OD600 &#226;&#172;?1.0 processed flow cytometry         sum cells size corresponding bin smaller size <br /> described (Davierwala et al., 2005; Figure 1). The complete set flow cytom-      reaches 50% total number cells measured mutant cell size <br />  <br />  <br /> Vol. 17, November 2006                                                                                                                                          4737 <br />  L. Yu et al. <br />  <br />  <br />  <br />  <br /> Figure 1. A cell cycle screen essential genes yeast. (A) Schematic diagram cell cycle screen. Strains constructed by <br /> replacement native promoters TetO7 cassette strain R1158, contains sequences encoding tet &#226;&#8364;&#339;off&#226;&#8364;? activator tTA* <br /> integrated URA3. YFG1, favorite gene; NGC1, gene chromosome. The flow cytometry data displayed histogram with <br /> fluorescence intensity (which proportional DNA content) plotted x-axis, number cells given intensity plotted <br /> on y-axis. The positions cells 1C, 2C, 4C DNA contents indicated. (B) Manual computational scoring flow <br /> cytometry histograms. The number TetO7 strains displaying category cell cycle profile indicated manual scoring and <br /> computational scoring. Strains placed category methods indicated overlap. (C) Flow cytometry <br /> histograms mutants showing strong cell cycle effects category shown, relevant gene indicated. The number cells <br /> is indicated y-axis, DNA content indicated x-axis. <br />  <br />  <br />  <br /> data (Mnaimneh et al., 2004). The box plot generated using box plot         RESULTS <br /> function available R 2.2.0. <br />   Raw normalized flow cytometry data, data used generate          A Collection Titratable Promoter Essential Genes <br /> the graphical displays Figures 2, 3, 4, complete functional <br /> enrichment data available at: http://biochemistry.utoronto.ca/brown/            In effort create complete collection TetO7 promoter <br /> data.html.                                                                          alleles essential yeast genes, attempted TetO7 pro- <br />                                                                                     moter strain construction described previously (Mnaimneh <br />                                                                                     et al., 2004) 1111 genes annotated essential by <br /> Western Blotting                                                                    Saccharomyces Genome Database (SGD), Yeast Proteome <br /> Strains grown YPD absence presence 10 &#226;?&#174;g/ml doxycy- <br /> cline 15 h OD600 &#226;&#172;&#402;0.5. Cultures corresponding 3 OD cells <br />                                                                                     Database (YPD), Munich Information Center Pro- <br /> were collected, cell pellets resupended 0.5 ml fresh YPD.      tein Sequences (MIPS), June 2000 project was <br /> Protein extracts prepared NaOH lysis TCA precipitation,             initiated. At attempts construct a <br /> western blotting monoclonal anti-CPY anti-ALP antibodies (Molecu-           TetO7 promoter strain gene. For total 838 genes <br /> lar Probes, Eugene, OR), polyclonal anti-Gas1p antibody (kind gift H. <br /> Riezman), described (Davierwala et al., 2005).                                   resulting strains subjected PCR analysis con- <br />                                                                                     firm correct integration TetO7 promoter replace- <br />                                                                                     ment wild-type promoter region (Mnaimneh et al., <br /> Chromatin Immunoprecipitation                                                       2004). Failure obtain TetO7 promoter strains be <br /> Strains grown presence doxycycline 5 h synchronized          accounted errors step genome sequencing <br /> with &#226;?&#163;-factor. G1-arrested cells released fresh media &#226;&#172;&#402;20 OD       PCR confirmation, promoter refractory to <br /> cells harvested 12 min. Cells fixed chromatin immuno- <br /> precipitates prepared essentially described (Kuras Struhl, 1999),       recombination, promoter integration itself resulting in <br /> using IgG-agarose immunoprecipitate TAP-tagged Mcm2 wild-type               lethality (e.g., mis-expressing gene TetO7 <br /> and TetO7-SMC4 strains. Immunoprecipitated DNA analyzed PCR                  promoter &#226;&#8364;&#339;on&#226;&#8364;?). All strains available Open <br /> using primer pairs specific regions ARS1 described <br /> (Tanaka et al., 1997). All PCR, gel electrophoresis, visualization conditions <br />                                                                                     Biosystems. Since June 2000, SGD emerged single <br /> were described (Tanaka et al., 1997), gel quantified using           major source public annotation yeast genome. Some <br /> ImageQuant.                                                                         genes removed SGD resequencing <br />  <br /> 4738                                                                                                                    Molecular Biology Cell <br />                                                                                                        Cell Cycle Function Essential Genes <br />  <br />  <br />  <br />  <br /> Figure 2. Enrichment specific gene functions different flow cytometry categories. (A) Cluster diagram flow cytometry <br /> histograms. Genes positioned y-axis DNA content indicated x-axis. Genes grouped different categories <br /> according overlap computational manual analysis clustered categories according similarity <br /> of flow cytometry profiles. The number cells given DNA content indicated color, black representing minimum <br /> and brown representing maximum. The categories flow cytometry profiles indicated, category gray diploid <br /> category. (B) GO-gram indicating statistically significant enrichments gene ontology (GO) annotations biological process <br /> hierarchy TetO7 alleles flow cytometry category. Red bars indicate particular gene annotated given GO category <br /> and GO category significantly overrepresented cell cycle category gene placed. Blue bars indicate that <br /> a particular gene annotated given GO category GO category significantly enriched given cell cycle <br /> category. Not GO categories displayed significant enrichment shown. <br />  <br />  <br />  <br /> and reannotation (mostly uncharacterized ORFs lethal            flow cytometry, turning transcrip- <br /> when deleted, overlap essential genes             tion essential gene (Figure 1A). The resulting histo- <br /> been characterized), genes categorized essential          grams used determine fraction cells a <br /> our previous analysis longer listed essential            population cell cycle stage (G1, S, G2), <br /> (presumably slow-growing deletion mutants                 accumulation cells specific DNA content indi- <br /> obtained). Thus, collection encompasses 773 essential                cates defect progression particular cell cycle <br /> genes, total 1033 (75%) annotated currently         stage. In initial experiments TetO7 promoter <br /> by SGD.                                                                  replacement strains, samples analyzed flow cytom- <br />                                                                          etry different times addition doxycycline (Sup- <br /> A Cell Cycle Screen Titratable Promoter Alleles                plementary Figure 1). We strains ana- <br /> the Essential Genes                                                      lyzed, alterations flow cytometry histograms were <br /> We used collection TetO7 promoter alleles system-              evident varying times, 2 h case CDC27, <br /> atically identify genes cause change cell cycle                8 h case SLD5. Importantly, category of <br /> profile promoter shut-off. Because nuclear DNA                 cell cycle accumulation evident latest time <br /> content cell reveals position cell cycle,             point, 15 h, case S phase gene SLD5 the <br /> measured DNA contents populations cells using                  accumulation increasingly obvious therefore <br />  <br /> Vol. 17, November 2006                                                                                                                   4739 <br />  L. Yu et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 4. Degree G1 accumulation reflects gene function and <br />                                                                          growth defect. Genes 1C category ordered to <br />                                                                          decreasing C1/C2 peak height ratio shown. Quartiles <br />                                                                          indicated left. White dotted line indicates average wild-type <br />                                                                          C1/C2 ratio. The GO-gram indicates statistically significant enrich- <br />                                                                          ments GO categories 1C flow cytometry category (p &#226;&#172;? <br />                                                                          0.001). Red bars indicate particular gene annotated the <br /> Figure 3. Comparison cell cycle phenotypes cell sizes. (A)        given GO category GO category significantly over- <br /> Cluster diagram DNA content flow cytometry histograms             represented cell cycle category gene was <br /> cell size profiles. Genes grouped y-axis Figure 2A.     placed. Blue bars indicate particular gene annotated with <br /> Cell size data (Mnaimneh et al., 2004) presented alongside       given GO category GO category significantly <br /> the flow cytometry data, increasing cell sizes plotted       enriched given cell cycle category. The growth scoring <br /> x-axis, left right. The number cells given cell size   scheme identical (Mnaimneh et al., 2004). <br /> is indicated color, white representing minimum red <br /> representing maximum. (B) Box plot cell sizes cell <br /> cycle category. The horizontal black lines boxes indicate the <br /> median size bin cumulative sum cells having size          Methods). In cases, flow cytometry histograms were <br /> corresponding bin, smaller size, reaches 50% total   categorized according distribution DNA contents <br /> number cells measured; boxes indicate       (Figure 1B). The categories follows: strains that <br /> the 25th 75th percentiles, respectively. The vertical lines     accumulate 1C DNA content (and arrested or <br /> the value range: upper cap drawn bin representing the <br />                                                                          delayed G1); strains accumulate DNA content <br /> largest size, lower cap drawn bin representing the <br /> smallest size. The width boxes varies according number     1C 2C (and delayed S phase); strains <br /> of genes category.                                                   accumulate 2C DNA content (and arrested or <br />                                                                          delayed G2); strains accumulate 3C and/or 4C <br />                                                                          DNA content (an unexpected cell cycle profile, likely relating <br /> easier score later points. Thus, order detect            defects cell separation); strains diploid; <br /> changes strains possible, chose analyze              strains exhibiting normal cell cycle profile. In the <br /> the promoter replacement collection 15 h growth              manual analysis profiles did exhibit character- <br /> the presence doxycycline.                                             istics of categories, clearly abnormal. <br />                                                                          These grouped &#226;&#8364;&#339;other.&#226;&#8364;? In computa- <br /> Scoring Flow Cytometry Profiles                                      tional analysis &#226;&#8364;&#339;other&#226;&#8364;? category strains <br /> We scored flow cytometry profiles using manual                  showed accumulation cells 1C DNA <br /> inspection computational analysis (see Materials                 contents. Strains displayed accumulations than <br />  <br /> 4740                                                                                                            Molecular Biology Cell <br />                                                                                               Cell Cycle Function Essential Genes <br />  <br />  <br /> one DNA content placed category,             mulation S phase cells, agreement known <br /> and strains placed category.              functions DNA replication (Hartwell, 1971a, 1973, 1976; <br />    Manual inspection indicated 838 TetO7 promoter          Johnston Nasmyth, 1978; Piatti et al., 1995; Zou et al., <br /> mutants screened, 625 (or 75%) displayed altered flow           1997; Burgers, 1998; Kanemaki et al., 2003). In addition, <br /> cytometry profile. Computational analysis (see Materials       CDC4, CDC25, CDC64 required G1 transit and <br /> Methods details) indicated 579 displayed altered       accumulated cells 1C DNA contents (Bedard et al., 1981; <br /> profile, 563 mutants (or 67%) scored altered        Tripp Pinon, 1986; Goh Surana, 1999). We conclude <br /> both methods. On average, &#226;&#172;&#402;80% assessments              extended period gene product depletion <br /> manual inspection supported computational anal-            stringency categorization cutoffs precluded <br /> ysis vice versa. The fraction genes category        accurate categorization mutants number <br /> (supported manual computational analysis)           false negatives screen low, cell cycle <br /> shown Figure 1B. Examples strongest cell cycle           defects detected TetO7 promoter mutants of <br /> phenotypes exhibited TetO7 promoter mutants shown           known CDC genes. <br /> in Figure 1C. However, scored modest alterations              Overall, analysis indicated essential genes <br /> in flow cytometry profile. For example, 1C accumu-          linked specific stage cell cycle progression. We <br /> lation category, modeling flow cytometry histograms         asked relationships catego- <br /> indicated fraction cells 1C DNA content         ries DNA content gene properties, so, <br /> ranged 31 97% mutants scored 1C, compared           illuminating regard the <br /> with 28% wild-type control (see below).                    nature cell cycle functions individual <br />    In TetO7 promoter mutant strains onset growth            genes. <br /> cessation varies gene gene time <br /> required depletion transcript protein. Even         G1 Phase <br /> though certain timing degree             For visualization, flow cytometry histograms over- <br /> depletion different gene products promoter shut-      lap manual computational scoring methods <br /> off 15 h, saw clear specific effects cell cycle      grouped subjected hierarchical clustering (Fig- <br /> progression majority essential genes. The accu-    ure 2A). We asked GO functional annotations <br /> mulation specific DNA contents suggests intimate con-        enriched flow cytometry categories. These <br /> nection essential gene function cell cycle pro-        data displayed &#226;&#8364;&#339;GO-gram&#226;&#8364;? (Figure 2B), the <br /> gression. The altered flow cytometry profiles did appear       annotations genes categories displayed; red <br /> to reflect cell debris, possible exception        bars indicate given <span id='am-83' about='Thesaurus:GO_Annotation' typeof='owl:Thing'>GO annotation</span> significantly <br /> profiles classified &#226;&#8364;&#339;other,&#226;&#8364;? indicating results   enriched given flow cytometry category. The 1C accu- <br /> not artifacts simple slow growth cell death. The         mulation category, 270 genes largest, <br /> flow cytometry profiles wild-type colonies         showed dramatic enrichment genes involved protein <br /> analyzed scored normal, TetO7              synthesis (Figure 2B). This included genes involved ribo- <br /> promoter strains little growth defect plates         biogenesis, translation, tRNA metabolism, tRNA charg- <br /> also perturbation DNA content (112 190 tet            ing, RNA polymerase III transcription (GO:0042254, <br /> promoter mutants normal growth scored            ribosome biogenesis assembly, p value 2.79e-23; GO: <br /> methods having normal flow cytometry profiles), indicat-        0043037, translation, 1.47 e-08; GO:0006399, tRNA metabo- <br /> ing scoring abnormal flow cytometry profiles       lism, 6.70e-07; GO:0043038, amino acid activation, 3.60e-06; <br /> not overly liberal. Strains did growth defect      GO:0006383, transcription RNA polymerase III <br /> in presence doxycycline likely represent strains         promoter, 0.0002). Although ribosome biogenesis cell <br /> which gene product depletion insufficient cause phe-       cycle regulated yeast (Bernstein Baserga, 2004), there <br /> notype, long mRNA protein half-life.         prior evidence overexpression depletion of <br /> However, 51 strains displayed normal growth            ribosomal proteins factors involved ribosome biogen- <br /> altered cell cycle profiles manual computational       esis causes cell cycle defects. For example, depletion SSU <br /> scoring (Supplementary Figure 2), indicating          processome proteins leads G1 arrest lack ribo- <br /> instances screen sensitive identify cell         somes (Bernstein Baserga, 2004), does depletion of <br /> cycle delays absence growth defect. Conversely,        20S pre-rRNA maturation factor Rio1 (Angermayr et al., <br /> of TetO7 promoter strains normal cell cycle profiles,     2002). Similarly, tRNA charging long linked G1 <br /> almost half (82 194) exhibited growth defects pres-      transit number mutants tRNA synthetases arrest in <br /> ence doxycycline. Therefore essential genes         G1 (Unger Hartwell, 1976; Bedard et al., 1981). Consis- <br /> linked cell cycle processes; 82 strains presumably        tent existing links protein synthesis G1 <br /> cease growth cell cycle stage specificity.                 transit, data indicate dramatic restriction <br />    As measure efficiency screen identifying     cell&#226;&#8364;&#8482;s ability progress past cell cycle commitment <br /> genes roles cell division cycle progression, de-        point G1 (known Start yeast) capacity for <br /> termined fraction known CDC genes scored          protein synthesis, inhibiting protein synthesis range <br /> as positive (Supplementary Figure 3). There 67 CDC             different means caused accumulation cells G1. This <br /> genes Saccharomyces Genome Database (SGD). Of               observation reminiscent previous analyses cell size, <br /> these, 6 mapped, 16 present       indicated cells mutated ribosome biogenesis <br /> TetO7 promoter collection. Of remaining 45, 89% dis-           factors tend small unbudded (Jorgensen et al., <br /> played altered cell cycle profile scored manually,     2002; Mnaimneh et al., 2004). However, does ap- <br /> computationally, methods. The cell cycle arrest         pear direct relationship cell size our <br /> point 38 CDC genes known reasonably         categorized flow cytometry profiles. When grouped the <br /> be inferred. In screen, majority (28 38; 74%)           cell size data Mnaimneh et al. (2004) according the <br /> showed expected cell cycle stage-specific accumulation.        assigned flow cytometry categories (Figure 3), that <br /> For example, CDC6, CDC8, CDC9, CDC17, CDC21, CDC45,                strains 1C category tended slightly <br /> CDC47, CDC101, CDC102, CDC105 displayed accu-              smaller normal strains, strong predic- <br />  <br /> Vol. 17, November 2006                                                                                                        4741 <br />  L. Yu et al. <br />  <br />  <br /> tive correspondence flow cytometry data and <br /> the cell size data. Thus, cell cycle accumulation G1 far <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> absolute predictor cell size (and vice versa). <br />    Eleven unannotated genes gave rise G1 arrest after <br /> promoter shut-off, suggesting involved in <br /> some aspect ribosome biogenesis protein biosynthesis. <br /> Indeed, high-throughput data suggest genes <br /> perform functions: YPR169W encodes nucleolar pro- <br /> tein unknown function, interacts Tpt1p, a <br /> protein involved tRNA splicing (Hazbun et al., 2003; Huh <br /> et al., 2003); Ynl313cp localized cytoplasm and <br /> nucleus interacts proteins involved RNA pro- <br /> cessing, ribosomal biogenesis import, protein metabo- <br /> lism (Gavin et al., 2002; Huh et al., 2003; Krogan et al., 2004); <br /> Ynl310cp encodes protein interacts proteins in- <br /> volved RNA processing (Gavin et al., 2002; Ho et al., 2002; <br /> Hazbun et al., 2003); Yhr020wp inferred tRNA <br /> aminoacylation ligation activity affinity-pre- <br /> cipitates Utp13, SSU processome <br /> involved pre-18S rRNA processing (Tatusov et al., 2000; <br /> Ho et al., 2002). Finally, Ynr046wp recently identi- <br /> fied subunit tRNA methyltransferase (Purushothaman <br /> et al., 2005), Ynr054cp, encodes nucleolar protein <br /> that interacts proteins involved RNA processing, has <br /> been recently characterized component SSU proces- <br /> some (Gavin et al., 2002; Ho et al., 2002; Huh et al., 2003; Hoang <br /> et al., 2005). Thus 6 11 unannotated ORFs the <br /> G1 arrest category likely function ribosome biogenesis, <br /> tRNA metabolism, protein metabolism. <br />    Histograms mutants 1C accumulation category <br /> revealed category encompassed range degrees <br /> of 1C DNA content accumulation, 31 97% 1C (com- <br /> pared average wild-type value 28% 1C). Al- <br /> though threshold exceeded wild-type <br /> isolates analyzed (unpublished data), nonetheless con- <br /> sidered threshold lenient, in <br /> which case little relationship               Figure 5. Flow cytometry histograms S phase category. The <br /> growth effect functional category mutated gene         flow cytometry histograms wild type 27 genes <br /> those relatively low 1C accumulation. We              scored having S phase accumulation shown. Cultures were <br /> ordered mutants 1C category according              sampled flow cytometry 15 h addition doxycycline. <br /> relative 1C accumulation asked severity phe-          Genes known functions DNA replication (red) nucleo- <br />                                                                      tide metabolism (blue) indicated, genes previ- <br /> notype (in case 1C accumulation) related partic-         ously described connection S phase (green). The positions 1C <br /> ular gene functions growth defect (Figure 4). We           2C DNA contents indicated. <br /> that quartile significant enrichment for <br /> annotations related tRNA synthesis activation, and <br /> translation, second quartiles             DNA-dependent DNA replication, 9.97e-05; GO:0006281, <br /> enriched functions related ribosome biogenesis. The           DNA repair, 1.20e-07; Figure 2B). These include CDC6, <br /> fourth quartile significantly populated genes          SLD5, PSF2, POL1, POL2, PRI2, POL30, POL12, DPB11, <br /> broader RNA metabolism category. Additionally,             DPB2, PSF1, CDC45, direct roles the <br /> a tendency mutants highest 1C accumulation           initiation and/or elongation DNA replication. Several <br /> display prominent growth defect. This suggests           genes class, directly involved in <br /> given additional phenotypic data, able under-           replication, known roles nucleotide metabolism <br /> stand tet-promoter mutants display greater           (CDC8, CDC21, DFR1, FOL 2, DUT1). We explored the <br /> growth phenotypes others; previously            possibility CSE1 SMC4 play roles regulat- <br /> able draw strong conclusions regard (Mnaimneh et          ing DNA replication. CSE1 responsible nuclear <br /> al., 2004). In summary, functional information de-          shuttling nuclear transporter importin &#226;?&#163; (Hood and <br /> rived single flow cytometry parameter (1C:2C ratio)           Silver, 1998; Kunzler Hurt, 1998; Solsbacher et al., 1998) <br /> in addition information contained different            role CSE1 mitosis (but S phase) been <br /> categories. Moreover, choice threshold overly         described (Xiao et al., 1993; Schroeder et al., 1999). We syn- <br /> liberal, functional information contained          chronized cells G1, released cell cycle after <br /> even lowest quartile.                                         shut-off TetO7-CSE1, measured DNA contents of <br />                                                                      cells flow cytometry (Figure 6A). Cells depleted of <br /> S Phase                                                              CSE1 failed undergo DNA replication, 70% cells <br /> The S phase accumulation class, comprises 27              containing 1C DNA contents 90 min release from <br /> genes (Figure 5), showed dramatic enrichment genes             G1. We examined cells passed Start by <br /> involved DNA replication (GO:0006260, DNA replication,            measuring fraction cells elicited buds, a <br /> p value 4.02e-17; GO:0045005, maintenance fidelity         marker cell cycle progression G1/S boundary <br />  <br /> 4742                                                                                                     Molecular Biology Cell <br />                                                                                             Cell Cycle Function Essential Genes <br />  <br />  <br />  <br />  <br /> Figure 6. Depletion CSE1 SMC4 caused defects <br /> in S phase progression. (A) Flow cytometry histograms <br /> measuring DNA contents WT, TetO7-CSE1, and <br /> TetO7-SMC4 cells asynchronous culture (asy), ar- <br /> rested G1 (&#226;?&#163;) indicated times release <br /> from G1 presence doxycycline. Nocodazole <br /> was added 60 min prevent return G1. The <br /> percent cells G1 time indicated. (B) Flow <br /> cytometry histograms measuring DNA contents WT <br /> and smc4-1 cells asynchronous culture (asy), arrested <br /> in G1 (&#226;?&#163;) indicated times release from <br /> G1 30&#194;&#176;C. The percent cells G1 time is <br /> indicated. (C) The percent cells bud (&#228;&#161;&#186;), <br /> small bud (u), large bud (f) measured the <br /> indicated times release G1. <br />  <br />  <br /> that independent initiation DNA replication         course experiment cells failed to <br /> (Figure 6C). At 90 min 90% TetO7-CSE1 cells        arrest G1. Finally, observed similar phenotype with <br /> had budded (51% large bud, 38% small             temperature-sensitive smc4-1 strain, 35% of <br /> bud), indicating cells passed Start. These data    cells failed enter S phase (Figure 6B). As CSE1, <br /> indicate categorization TetO7-CSE1 S phase        categorization SMC4 initial cell cycle screen accu- <br /> accumulation class initial screen accurately predicted    rately predicted role SMC4 entry S phase. <br /> a role CSE1 S phase, initiation DNA         We explored role SMC4 DNA replication by <br /> replication initiation significant DNA       observing association Mcm2 protein replica- <br /> synthesis occurred. Additionally, depletion Cse1p     tion origins replication forks S phase, using <br /> was robust observe clear phenotype      chromatin immunoprecipitation (ChIP; Figure 7, A B). <br /> cell cycle. Thus, CSE1 required DNA replication, sug-     In wild-type cells, Mcm2p preferentially associated with <br /> gesting essential DNA replication protein im-      replication origin ARS1 G1. As cells progressed into <br /> portin &#226;?&#163; cargo.                                                  S-phase association Mcm2 origin decreased <br />    SMC4 encodes &#226;&#8364;&#339;structural maintenance chromo-             association Mcm2p origin-distal DNA frag- <br /> somes&#226;&#8364;? protein, subunit condensin clear roles    ments increased. This <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span> implies association the <br /> in chromosome condensation mitosis (Freeman et al.,          replication fork, previously described (Aparicio <br /> 2000). In synchronous progression G1 S phase,       et al., 1997; Tanaka et al., 1997). We TetO7- <br /> found SMC4 required normal S phase progres-         SMC4 shut off, Mcm2p binding ARS1 decreased as <br /> sion, indicating previously unrecognized role Smc4p         cells progressed S phase, Mcm2p failed then <br /> (Figure 6A). When TetO7-SMC4 shut-off, 40%         associate origin-distal regions. Thus, data indicate <br /> cells failed enter S phase times S phase          Smc4p depletion affected binding Mcm2p the <br /> complete wild-type control (50 &#226;&#8364;&#8220; 60 min release),   regions flanking ARS1 S phase suggest role for <br /> as indicated failure progress 1C 2C      Smc4p regulating DNA replication, tran- <br /> DNA content. This failure enter S phase        sition initiation elongation. Previous work has <br /> cell death failure pass Start, TetO7-SMC4 cells elic-   hinted role condensin S phase, including the <br /> ited buds kinetics efficiency similar wild   hydroxyurea sensitivity condensin mutants fission <br /> type (Figure 6C). Additionally, TetO7-SMC4 strain ar-        yeast (Aono et al., 2002) presence condensin the <br /> rested G1 efficiency similar wild type       rDNA locus S phase (Johzuka et al., 2006). Addition- <br /> (86% TetO7-SMC4 cells &#226;?&#163;-factor displayed shmoo           ally, genome-wide protein-DNA interaction data suggests <br /> morphology vs. 98% wild-type cells), suggesting      Smc4p excluded replication origins (Wang et al., <br /> 33% TetO7-SMC4 cells did enter S phase        2005), analysis performed during <br />  <br /> Vol. 17, November 2006                                                                                                      4743 <br />  L. Yu et al. <br />  <br />  <br />  <br />  <br /> Figure 7. Depletion SMC4 caused replication fork defects. (A) <br /> Chromatin immunoprecipitation analysis Mcm2 binding ARS1 <br /> and adjacent regions WT TetO7-SMC4 cells. Cells ar- <br /> rested G1 released S phase presence doxycy-      Figure 8. Protein trafficking defects 3C/4C category. (A) <br /> cline. Sample taken ChIP analysis indicated times.     Micrographs strains 3C 4C accumulation <br /> DNA amplified PCR (input) (IP) immuno-          category. Promoters shut addition doxycycline, <br /> precipitation Mcm2. Ethidium bromide&#226;&#8364;&#8220;stained agarose gels        cells fixed 10 h. Cells stained DAPI, which <br /> the products multiplex PCR reactions amplify ARS1, regions 4     stains nuclei, cells visualized fluorescence and <br /> kb upstream (&#226;&#171;&#185;4 kb), 4 kb downstream (&#226;&#171;&#186;4 kb), 8 kb down-           phase-contrast microscopy. Merged images shown. (B) Immu- <br /> stream (&#226;&#171;&#186;8 kb) ARS1 shown. (B) The PCR products      noblots carboxypeptidase Y (CPY), alkaline phosphatase (ALP), <br /> independent ChIP experiments quantified average           Gas1 indicated TetO7 strains addition <br /> amount DNA band relative the 12-min        doxycycline 15 h. The positions processing trafficking <br /> sample plotted. Error bars span SD. The quantification   intermediates protein indicated (pCPY, ER/Golgi mod- <br /> PCR products region amplified (ARS1, &#226;&#171;&#185;4 kb, &#226;&#171;&#186;4 kb,        ified proCPY; uCPY, underglycosylated CPY; mCPY, mature CPY; <br /> &#226;&#171;&#186;8 kb) plotted WT (&#228;&#161;&#186;) TetO7-SMC4 (u).                   proALP, ER/Golgi modified proALP; sALP, soluble ALP; mALP, <br />                                                                        mature ALP; proGas1, ER glycosylated proGas1; mGas1, mature <br />                                                                        Gas1). Tubulin shown loading control. <br /> synchronous progression S phase therefore <br /> may detected relevant localization. Perhaps               3C/4C <br /> Smc4p exerts effect DNA replication creating a <br /> chromatin environment facilitates progression repli-           The 3C/4C accumulation flow cytometry category showed a <br /> cation forks away replication origins. We unable             clear enrichment genes associated protein transport, <br /> to detect S phase defects condensin subunit              protein secretion, glycoprotein metabolism (GO:0016192, <br /> genes TetO7-promoter collection, SMC2, YCG1,                vesicle-mediated transport, P value 3.38e-08; GO:0046903, <br /> YCS4. It great determine S         secretion, 1.91e-10; GO:0009100, glycoprotein metabo- <br /> phase function property Smc4p condensin              lism, 4.04e-08; Figure 2B unpublished data). An enrich- <br /> complex.                                                               ment similar functions evident category of <br />                                                                        mutants exhibiting multiple buds earlier morpholog- <br />                                                                        ical analysis TetO7 promoter collection (Mnaimneh <br /> G2 Phase                                                               et al., 2004), far fewer genes scored in <br /> Processes enriched category included chromosome                category. Four uncharacterized genes, YJL097W/PHS1, <br /> segregation, protein transport secretion, DNA replica-             YDL193W/NUS1, YNL149C, YML125C, included in <br /> tion, cell cycle, metabolism (GO:0007059, chromosome               3C/4C category, suggesting involved in <br /> segregation, Pvalue 6.19e-05; GO:0016192, vesicle-mediated             aspect protein trafficking. The proteins encoded by <br /> transport, 1.61e-05; GO:0046903, secretion, 3.73e-06; GO:0007049,      genes localize ER ER vacuole <br /> cell cycle, 0.0007; GO:0019222, regulation metabolism,          (Huh et al., 2003) YJL097W/PHS1 gene recently <br /> 0.0009; Figure 2B). The diversity gene functions repre-             regulator sphingosine levels (Schuldiner et <br /> sented 2C accumulation category (there 26 signif-           al., 2005). We examined cellular morphology these <br /> icant GO categories group; p &#226;&#172;? 0.001) suggested           strains promoter shut-off exhibited a <br /> transit G2 phase cell cycle dependent           multiple-budded large-budded phenotype normal <br /> large number different cellular processes.                          postmitotic nuclei (Figure 8A), indicating defect cytoki- <br />  <br /> 4744                                                                                                         Molecular Biology Cell <br />                                                                                               Cell Cycle Function Essential Genes <br />  <br />  <br /> nesis cell separation. Normal nuclear division accom-        frame experiment. Although perceived <br /> panied cytokinesis cell separation defect likely re-       limitation, able detect cell cycle defects in <br /> sulted appearance events apparent 3C 4C         89% known CDC genes TetO7 alleles correctly <br /> DNA contents flow cytometry analysis. We ex-           categorize 74% these, suggesting false negative <br /> amined TetO7 alleles YJL097W/PHS1, YDL193W/                 rate screen low. Additionally, functional <br /> NUS1, YNL149C, YML125C defects processing           enrichment 1C S phase categories con- <br /> and trafficking different glycosylated proteins (Fig-     sistent published work: 1C category displayed a <br /> ure 8B). TetO7-YJL097W did detectable defects          massive enrichment genes involved tRNA ribo- <br /> carboxypeptidase Y (CPY) processing, indicating         biogenesis, S phase category displayed mas- <br /> not essential ER-to-Golgi transport, N-linked glycosyla-       sive enrichment genes involved DNA replication. <br /> tion, vacuolar signal peptide cleavage; however, dis-        Finally, hallmarks cell cycle arrest continued <br /> played abnormal processing alkaline phosphatase (ALP),          cell growth absence cell cycle progression. Strains in <br /> accumulating soluble form ALP. The presence              S-phase, 2C, 3C/4C categories exhibited larger <br /> soluble form ALP reminiscent apl5&#226;&#338;&#172; mutants (Cowles et        normal cell size (Figure 3), consistent cell growth <br /> al., 1997) depletion recently described PGA1         despite cell cycle arrest. By contrast, strains normal <br /> and suggests defect vacuolar transport. Additionally,         cell cycle category exhibited wild-type cell sizes, despite <br /> TetO7-YJL097W showed defects accumulation ma-            half strains category having clear growth <br /> ture Gas1p. TetO7-YDL193W accumulated hypoglycosylated             defects promoter shut-off. Thus conclude mea- <br /> forms CPY, like TetO7-SEC59, suggesting role          suring DNA contents promoter shut-off essential <br /> glycosylation ER. TetO7-YDL193W accumulated            genes represents sensitive accurate means identify- <br /> immature ER forms Gas1p. TetO7-YNL149C TetO7-               ing cell cycle mutants. <br /> YML125C displayed normal CPY processing, accu-               A second major conclusion down-regulation of <br /> mulated increased levels abnormal soluble ALP               genes diverse functions regulating maintaining <br /> slightly decreased levels mature Gas1p. Together          protein biosynthetic capacity cell causes arrest in <br /> results indicate roles YJL097W, YDL193W, YNL149C,          G1. The intimate relationship protein biosynthesis <br /> YML125C protein trafficking. On basis obser-       cell cycle progression G1 observed is <br /> vations named YNL149C YML125C genes                consistent previous observations linking protein syn- <br /> PGA2 PGA3, processing Gas1p ALP.                    thesis G1 transit (Unger Hartwell, 1976; Bedard et <br />                                                                    al., 1981; Moreno Nurse, 1994), connections be- <br /> DISCUSSION                                                         tween ribosome biogenesis cell size regulation Start <br />                                                                    (Jorgensen et al., 2002, 2004), links trans- <br /> Several major conclusions drawn results            lation rate G1 (Polymenis Schmidt, 1997) pro- <br /> presented here. First, expected outset          vides global view cellular processes contribute to <br /> major proportion strains scored normal,         passage cell cycle commitment point G1, <br /> indicating ceased division random positions        Start. There concordance G1 data that <br /> the normal cell cycle. However, strains             derived cell cycle screening Drosophila cells after <br /> analyzed (65%) displayed specific defect cell division     gene product depletion RNAi (Bjorklund et al., 2006) in <br /> cell cycle progression. One trivial explanation large      screens clear involvement protein biosyn- <br /> number genes displayed cell cycle defects de-        thesis pathways G1 transit. However, Drosophila <br /> pletion rate false positives high.         screen genes identified largely ribosomal subunit <br /> However, S phase category 20 27 genes identified         genes, data illustrate roles ribosome <br /> have known roles DNA replication nucleotide metab-           assembly factors, tRNA synthesis, processing, charg- <br /> olism, S phase            ing proteins regulating G1. Given down-regulation of <br /> defects analyzed synchronous culture (Figure 6),           ribosome biogenesis cause cell cycle arrest G1 mouse <br /> indicating false positive rate screen      human cells (Strezoska et al., 2000; Volarevic et al., 2000; <br /> high. Similarly, majority genes 1C category     Pestov et al., 2001; Oliver et al., 2004), identification of <br /> roles ribosome biogenesis translation likely         essential genes regulate G1 yeast useful in <br /> represent true positives. Further analysis 1C category      dissecting connections ribosome biogenesis cell <br /> revealed functional enrichment quartile            cycle progression mammalian cells. <br /> with modest 1C accumulation (Figure 4), indicating           A conclusion sorting genes different <br /> that scoring overly liberal. The striking         categories basis flow cytometry profiles al- <br /> result analysis majority strains   lowed prediction gene function. We derived information <br /> appear stage-specific cell cycle defects      gene function flow cytometry categories by <br /> function essential genes important            identifying biological processes enriched within <br /> specific time cell cycle.                                   category. Although present screen cell cycle <br />    It important note present screen differs    accumulation stage necessarily reflect execution <br /> the classic cell cycle screens did require arrest   point point action genes, identified <br /> of strain particular cell cycle stage, instead scored   significant functional enrichments cell cycle cat- <br /> any significant cell cycle stage-specific accumulation. In    egory able make predictions bio- <br /> respect screen conceptually similar recent cell        logical function uncharacterized essential genes. In par- <br /> cycle screen Drosophila cells using gene product depletion      ticular, accumulation 3C 4C DNA <br /> by RNA-mediated interference (RNAi; Bjorklund et al.,              content reliably identified components protein trafficking, <br /> 2006). One reasonable explanation observation      modification, secretion pathways. Four uncharacterized <br /> some strains cell cycle phenotype delay       genes category, NUS1, PHS1, PGA2, PGA3, had <br /> an arrest strains &#226;&#172;?100% cells accumulate       detectable changes modifications the <br /> at relevant cell cycle stage promoter shut-off    markers protein secretion modification analyzed, <br /> not uniformly result protein depletion time-         indicating roles protein trafficking. The 1C category pre- <br />  <br /> Vol. 17, November 2006                                                                                                        4745 <br />  L. Yu et al. <br />  <br />  <br /> dicted function protein biosynthesis: 6 11 unanno-                         Cowles, C. R., Odorizzi, G., Payne, G. S., Emr, S. D. (1997). The AP-3 <br /> tated genes G1 accumulation category connec-                             adaptor complex essential cargo-selective transport yeast vacuole. <br />                                                                                      Cell 91, 109 &#226;&#8364;&#8220;118. <br /> tions protein biosythesis suggested functional <br /> genomic data, including protein&#226;&#8364;&#8220;protein interaction                              Culotti, J., Hartwell, L. H. (1971). Genetic control cell division cycle <br />                                                                                      yeast. III. Seven genes controlling nuclear division. Exp. Cell Res. 67, <br /> protein localization data. Additionally, deple-                        389 &#226;&#8364;&#8220; 401. <br /> tion condensin subunit Smc4p importin &#226;?&#163; <br />                                                                                      Davierwala, A. P., et al. (2005). The synthetic genetic interaction spectrum of <br /> export receptor Cse1p resulted defective DNA replication                          essential genes. Nat. Genet. 37, 1147&#226;&#8364;&#8220;1152. <br /> and accumulation S phase DNA contents <br /> accurately predicted function DNA synthesis. Thus, de-                            Dohmen, R. J., Wu, P., Varshavsky, A. (1994). Heat-inducible degron: a <br />                                                                                      method constructing temperature-sensitive mutants. Science 263, 1273&#226;&#8364;&#8220; <br /> spite possibility observed cell cycle                           1276. <br /> delays indirect effects protein <br />                                                                                      Freeman, L., Aragon-Alcaide, L., Strunnikov, A. (2000). The condensin <br /> depletion, comprehensive data set useful pre-                           complex governs chromosome condensation mitotic transmission of <br /> diction essential gene function.                                                  rDNA. J. Cell Biol. 149, 811&#226;&#8364;&#8220; 824. <br />    Our study complements decades cell cycle ex-                           Gavin, A. C., et al. (2002). Functional organization yeast proteome by <br /> perimentation, provides systematic survey essential                         systematic analysis protein complexes. Nature 415, 141&#226;&#8364;&#8220;147. <br /> gene function cell division cycle. Regulation pro- <br />                                                                                      Gentleman, R. C., et al. (2004). Bioconductor: open software development for <br /> liferation primary importance study human                            computational biology bioinformatics. Genome Biol. 5, R80. <br /> development study human diseases. We <br />                                                                                      Goh, P. Y., Surana, U. (1999). Cdc4, protein required onset S <br /> anticipate connections essential                            phase, serves essential function G(2)/M transition Saccharomyces <br /> cellular processes cell division cycle progression illus-                        cerevisiae. Mol. Cell. Biol. 19, 5512&#226;&#8364;&#8220;5522. <br /> trated conserved humans. Indeed, number                            Hartwell, L. H. (1971a). Genetic control cell division cycle yeast. II. <br /> of pathways important cell cycle pro-                           Genes controlling DNA replication initiation. J. Mol. Biol. 59, 183&#226;&#8364;&#8220;194. <br /> gression yeast important cell cycle progression <br />                                                                                      Hartwell, L. H. (1971b). Genetic control cell division cycle yeast. IV. <br /> in Drosophila (Bjorklund et al., 2006), encouraging view                         Genes controlling bud emergence cytokinesis. Exp. Cell Res. 69, 265&#226;&#8364;&#8220;276. <br /> that global analysis connections essential <br />                                                                                      Hartwell, L. H. (1973). Three additional genes required deoxyribonucleic <br /> genes cell cycle useful dissecting                            acid synthesis Saccharomyces cerevisiae. J. Bacteriol. 115, 966 &#226;&#8364;&#8220;974. <br /> network processes impinge progression through <br />                                                                                      Hartwell, L. H. (1974). Saccharomyces cerevisiae cell cycle. Bacteriol. Rev. 38, <br /> the cell cycle metazoans.                                                         164 &#226;&#8364;&#8220;198. <br />                                                                                      Hartwell, L. H. (1976). Sequential function gene products relative DNA <br />                                                                                      synthesis yeast cell cycle. J. Mol. Biol. 104, 803&#226;&#8364;&#8220; 817. <br /> ACKNOWLEDGMENTS <br />                                                                                      Hartwell, L. H., Culotti, J., Reid, B. (1970). Genetic control cell- <br /> We thank Jennifer Haynes assistance secretion assays, Tomas             division cycle yeast. I. Detection mutants. Proc. Natl. Acad. Sci. USA 66, <br /> Babak assistance G1 functional studies, Sameer Agnihotri tech-          352&#226;&#8364;&#8220;359. <br /> nical contributions, Zhijian Li strain construction, Paul Jorgensen, <br /> Mike Tyers, Susan Forsburg, Brenda Andrews, Charlie Boone helpful            Hartwell, L. H., Mortimer, R. K., Culotti, J., Culotti, M. (1973). Genetic <br /> comments manuscript. This work supported grants               control cell division cycle yeast. V. Genetic analysis cdc mutants. <br /> Canadian Institutes Health Research (G.W.B.) Genome Canada (T.H.).            Genetics 74, 267&#226;&#8364;&#8220;286. <br /> G.W.B. Research Scientist National Cancer Institute Canada.           Hazbun, T. R., et al. (2003). Assigning function yeast proteins integration <br />                                                                                      technologies. Mol. Cell 12, 1353&#226;&#8364;&#8220;1365. <br />                                                                                      Ho, Y., et al. (2002). Systematic identification protein complexes Saccha- <br /> REFERENCES                                                                           romyces cerevisiae mass spectrometry. Nature 415, 180 &#226;&#8364;&#8220;183. <br /> Altmann, K., Westermann, B. (2005). Role essential genes mitochon-         Hoang, T., Peng, W. T., Vanrobays, E., Krogan, N., Hiley, S., Beyer, A. L., <br /> drial morphogenesis Saccharomyces cerevisiae. Mol. Biol. Cell 16, 5410 &#226;&#8364;&#8220;5417.     Osheim, Y. N., Greenblatt, J., Hughes, T. R., Lafontaine, D. L. (2005). <br />                                                                                      Esf2p, U3-associated factor required small-subunit processome assembly <br /> Angermayr, M., Roidl, A., Bandlow, W. (2002). Yeast Rio1p found-          compaction. Mol. Cell. Biol. 25, 5523&#226;&#8364;&#8220;5534. <br /> ing member novel subfamily protein serine kinases involved the <br /> control cell cycle progression. Mol. Microbiol. 44, 309 &#226;&#8364;&#8220;324.                     Hood, J. K., Silver, P. A. (1998). Cse1p required export Srp1p/ <br />                                                                                      importin-alpha nucleus Saccharomyces cerevisiae. J. Biol. Chem. 273, <br /> Aono, N., Sutani, T., Tomonaga, T., Mochida, S., Yanagida, M. (2002).            35142&#226;&#8364;&#8220;35146. <br /> Cnd2 dual roles mitotic condensation interphase. Nature 417, <br /> 197&#226;&#8364;&#8220;202.                                                                             Huh, W. K., Falvo, J. V., Gerke, L. C., Carroll, A. S., Howson, R. W., Weissman, <br />                                                                                      J. S., O&#226;&#8364;&#8482;Shea, E. K. (2003). Global analysis protein localization in <br /> Aparicio, O. M. (2003). Tackling essential problem functional proteomics       budding yeast. Nature 425, 686 &#226;&#8364;&#8220; 691. <br /> of Saccharomyces cerevisiae. Genome Biol. 4, 230 <br />                                                                                      Johnston, L. H., Nasmyth, K. A. (1978). Saccharomyces cerevisiae cell cycle <br /> Aparicio, O. M., Weinstein, D. M., Bell, S. P. (1997). Components            mutant cdc9 defective DNA ligase. Nature 274, 891&#226;&#8364;&#8220; 893. <br /> dynamics DNA replication complexes S. cerevisiae: redistribution of <br /> MCM proteins Cdc45p S phase. Cell 91, 59 &#226;&#8364;&#8220; 69.                            Johzuka, K., Terasawa, M., Ogawa, H., Ogawa, T., Horiuchi, T. (2006). <br />                                                                                      Condensin loaded replication fork barrier site rRNA gene <br /> Bedard, D. P., Johnston, G. C., Singer, R. A. (1981). New mutations       repeats S phase FOB1-dependent fashion prevent contraction of <br /> yeast Saccharomyces cerevisiae affecting completion &#226;&#8364;&#339;Start.&#226;&#8364;? Curr. Genet. 4,      long repetitive array Saccharomyces cerevisiae. Mol. Cell. Biol. 26, 2226 &#226;&#8364;&#8220; <br /> 204 &#226;&#8364;&#8220;214.                                                                            2236. <br /> Bernstein, K. A., Baserga, S. J. (2004). The small subunit processome         Jorgensen, P., Nishikawa, J. L., Breitkreutz, B. J., Tyers, M. (2002). Sys- <br /> required cell cycle progression G1. Mol. Biol. Cell 15, 5038 &#226;&#8364;&#8220;5046.           tematic identification pathways couple cell growth division in <br />                                                                                      yeast. Science 297, 395&#226;&#8364;&#8220; 400. <br /> Bjorklund, M., Taipale, M., Varjosalo, M., Saharinen, J., Lahdenpera, J., and <br /> Taipale, J. (2006). Identification pathways regulating cell size cell-cycle   Jorgensen, P., Rupes, I., Sharom, J. R., Schneper, L., Broach, J. R., Tyers, M. <br /> progression RNAi. Nature 439, 1009 &#226;&#8364;&#8220;1013.                                         (2004). A dynamic transcriptional network communicates growth potential to <br />                                                                                      ribosome synthesis critical cell size. Genes Dev. 18, 2491&#226;&#8364;&#8220;2505. <br /> Brachmann, C. B., Davies, A., Cost, G. J., Caputo, E., Li, J., Hieter, P., and <br /> Boeke, J. D. (1998). Designer deletion strains derived Saccharomyces            Kanemaki, M., Sanchez-Diaz, A., Gambus, A., Labib, K. (2003). Functional <br /> cerevisiae S288C: useful set strains plasmids PCR-mediated gene         proteomic identification DNA replication proteins induced proteolysis <br /> disruption applications. Yeast 14, 115&#226;&#8364;&#8220;132.                                vivo. Nature 423, 720 &#226;&#8364;&#8220;724. <br /> Burgers, P. M. (1998). Eukaryotic DNA polymerases DNA replication             Krogan, N. J., et al. (2004). High-<span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> macromolecular composition of <br /> DNA repair. Chromosoma 107, 218 &#226;&#8364;&#8220;227.                                                yeast RNA-processing complexes. Mol. Cell 13, 225&#226;&#8364;&#8220;239. <br />  <br />  <br /> 4746                                                                                                                               Molecular Biology Cell <br />                                                                                                                           Cell Cycle Function Essential Genes <br />  <br /> Kunzler, M., Hurt, E. C. (1998). Cse1p functions nuclear export             Schroeder, <br /> </body></html>