<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biochem. J. (2003) 371, 743&#226;&#8364;&#8220;752 (Printed Great Britain)                                                                                                              743 <br />  <br />  <br /> Thioredoxin peroxidases foster cytoprotection cell death response <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> different stressors: over- under-expression thioredoxin peroxidase <br /> in Drosophila cells <br /> Svetlana N. RADYUK*1 , Rajindar S. SOHAL&#226;&#8364;&#160; William C. ORR*2 <br /> *Department Biological Sciences, Southern Methodist University, Dallas, TX 75275, U.S.A., &#226;&#8364;&#160;Department Molecular Pharmacology Toxicology, University Southern <br /> California, Los Angeles, CA 90033-9121, U.S.A. <br />  <br />  <br />  <br />  <br /> Recently, identified set genes constituting                                 exposure hydrogen peroxide paraquat, unaffected <br /> the peroxiredoxin gene family Drosophila melanogaster                                 challenged copper heat stress. In contrast, TPx <br /> [Radyuk, Klichko, Spinola, Sohal Orr (2001) Free Radical                             overexpressing cells susceptible copper heat <br /> Biol. Med. 31, 1090&#226;&#8364;&#8220;1100]. This set includes abundant                                stress compared control cells exhibited DNA <br /> thioredoxin peroxidase (TPx) species, Drosophila per-                             fragmentation. Furthermore, cells supplemented with <br /> oxiredoxin DPx-4783, cytosolic TPx DPx-5037, mito-                               N-acetyl-L-cysteine copper, clear <br /> chondrial TPx. Overexpression                                   negative effect cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span>, exacerbated TPx <br /> Drosophila S2 cells conferred increased resistance toxicity                           overexpression. Manipulations levels TPxs demonstrated <br /> induced hydrogen peroxide, paraquat cadmium. To                                    that, different stress conditions, enzymes might <br /> understand functional roles enzymes                              beneficial detrimental effects Drosophila cell <br /> vivo, report present study effects decreased                            viability. <br /> expression, using <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span>, response S2 cells <br /> to different stressors. When TPxs blocked, cells <br /> became relatively susceptible oxidative stress caused                         Key words: oxidative stress, peroxiredoxin, RNA interference. <br />  <br />  <br />  <br /> INTRODUCTION                                                                             lines overexpressing TPx resistant oxidative stress. <br />                                                                                          In contrast, treatments anticipated lead increased <br /> The antioxidative function thioredoxin peroxidases                                oxidative stress, elevated temperature exposure <br /> (TPxs), eliminate hydrogen organic peroxides, using                            copper, cells overexpressing TPx demonstrated increased <br /> thioredoxin electron donor, amply demonstrated.                          susceptibility, underexpressors showed increase <br /> Since insects possess detectable glutathione peroxidase                               susceptibility relative controls (and fact more <br /> activity, supposed TPxs play particularly central role                      resistant heat treatment). In words, enzymes can <br /> in enzymic removal reactive oxygen species (ROS)                          beneficial detrimental effects cell viability, <br /> group. Recent evidence TPx possess peroxynitrite                           depending type stressor. <br /> reductase activity underscores capacity these <br /> enzymes play protective role reactive species. <br />    Previously, reported identification different                         MATERIALS AND METHODS <br /> TPxs Drosophila melanogaster, distinct patterns <br />                                                                                          Cell culture <br /> of subcellular localization expression [1]. Two them <br /> (associated cytosol mitochondria) appear related                          Schneider S2 cells propagated complete Drosophila <br /> to TPxs mammals yeast; relatively abundant,                          expression (DES) medium (Invitrogen, Carlsbad, CA, <br /> are presumed play housekeeping role mitigating effects                       U.S.A.), containing 10 % foetal bovine serum 50 &#194;&#181;g/ml <br /> of pro-oxidants.                                                                         gentamicin (Sigma, St. Louis, MO, U.S.A.). For stable cell <br />    The purpose present study elucidate                         lines, medium supplemented 300 &#194;&#181;g/ml hygromycin <br /> functional role TPxs D. melanogaster cells, using under-                           (Sigma). Cell cultures split 1 : 2 fresh medium every <br /> or overexpressing cell lines, exposed treatments                            3&#226;&#8364;&#8220;5 days, retaining one-third conditioned media. <br /> been demonstrated elicit oxidative stress. Underexpressing <br /> cell lines created treatment gene-specific double- <br />                                                                                          dsRNA production conditions RNA interference (RNAi) <br /> stranded RNA (dsRNA), expression individual TPx <br /> genes essentially blocked. Results indicate cell                          Ambion MAXIscript(tm)T7/ T3 kit (Ambion, Austin, TX, U.S.A.) <br /> lines susceptible hydrogen peroxide                               used generate dsRNAs specific Drosophila <br /> superoxide generator paraquat, stably transfected cell                           peroxiredoxin DPx-4783 DPx-5037 genes. For sake of <br />  <br />  <br />  <br />   Abbreviations used: BSO, buthionine sulphoximine; DES, Drosophila expression system; dsRNA, double-stranded RNA; DPx, Drosophila peroxiredoxin; <br /> NAC, N -acetyl-L-cysteine; N2 ph, 2,4-dinitrophenyl; RNAi, RNA interference; ROS, reactive oxygen species; rp, ribosomal protein; RT, reverse transcriptase; <br /> TPx, thioredoxin peroxidase; cTPx, cytosolic TPx; mtTPx, mitochondrial TPx. <br />   1 <br />     Present address: Advanced Biosystems, George Mason University, 10900 University Blvd, MSN 1A8, Manassas, VA 20110, U.S.A. <br />   2 <br />     To correspondence addressed (e-mail BORR@mail.smu.edu). <br />  <br />                                                                                                                                                      <br />                                                                                                                                                    c 2003 Biochemical Society <br />  744               S. N. Radyuk, R. S. Sohal W. C. Orr <br />  <br />  <br />  <br />  <br /> Figure 1     Primers used vitro RNA generation RT&#226;&#8364;&#8220;PCR <br /> (A) Primer names sequences shown T7 promoter sequence shaded. Primer positions differentially spliced mRNAs [first variant (B); second variant (C)] cTPx gene <br /> are shown. Primer positions mRNA mtTPx gene (D) shown. T7 sequence indicated dotted lines. Bars represent coding parts cDNAs, bold lines represent non-coding <br /> 5  3  regions, regular line represents vector sequence containing T7 promoter sequence. <br />  <br />  <br /> clarity, genes henceforth referred cytosolic TPx                               precipitated ethanol, dissolved water heated 65 &#226;&#8212;&#166;C for <br /> (cTPx) mitochondrial TPx (mtTPx) respectively, reflecting                                      30 min followed slow cooling room temperature (25 &#226;&#8212;&#166;C) <br /> their cytosolic mitochondrial localizations. DNA fragments                                     allow annealing RNA. As negative control, 3 kb <br />                                                                                                                                                               R <br /> for vitro transcription prepared PCR using primers                                      dsRNA generated Escherichia coli pGEM vector <br /> specific gene containing 5  -end T7 RNA-polymerase-                                      (Promega) using T7 T3 primers. The concentration the <br /> binding sequence indicated Figure 1. Two fragments, approx.                                  dsRNAs adjusted 3 &#194;&#181;g/&#194;&#181;l used treatment of <br /> 700 bp, generated differentially spliced cTPx gene                                    S2 cells. <br /> by using T7-4-F1/T7-4-R T7-4-F2/T7-4-R primer pairs.                                          The introduction dsRNA Drosophila S2 cells was <br /> These fragments differ 5  non-coding regions. A                                      performed essentially described Clemens et al. [2]. Briefly, <br /> 300-bp DNA fragment mtTPx gene generated                                            1 ml cells reaction placed 35 mm plate at <br /> primers T7-5-F/T7-5-R contains 5  non-coding sequence                                        final concentration 1 &#195;&#8212; 106 cells/ml DES serum- <br /> and gene coding region. A 1000-bp fragment                                   free medium (Invitrogen). dsRNAs (5&#226;&#8364;&#8220;20 &#194;&#181;g) added <br /> mtTPx gene generated primers T7-5-F/T7 encompassing                                         approximate concentration 40 nM plates were <br /> the entire cDNA sequence. PCR products purified                                        swirled vigorously subsequently incubated 1 h room <br /> Nucleotide Removal kit (Qiagen, Valencia, CA, U.S.A.)                                          temperature. After incubation, 2 ml complete DES medium <br />                                             R <br /> transcribed vitro, using RiboProbe In Vitro Transcription                                   added cultures incubated 3 days allow protein <br /> System (Promega, Madison, WI, U.S.A.). The RNA products                                       turnover. <br />  <br />   <br /> c 2003 Biochemical Society <br />                                                                         Thioredoxin peroxidase cell viability stress conditions                         745 <br />  <br />  <br /> Reverse transcriptase (RT)&#226;&#8364;&#8220;PCR Western-blot analysis <br /> Aliquots (100 &#194;&#181;l) cell suspension centrifuged at <br /> 1000 g protein lysates immunoblot analysis extracted <br /> by homogenizing pellet 20 &#194;&#181;l lysis buffer containing <br /> protease inhibitors (Roche Molecular Biochemicals, Indianapolis, <br /> IN, U.S.A.) described previously [3]. Protein extracts were <br /> then fractionated SDS/PAGE, followed transfer PVDF <br /> membrane (Millipore, Bedford, MA, U.S.A.). Protein carbonyl <br /> content determined described Yan et al. [4]. To <br /> visualize cTPx protein, antibodies raised recombinant <br /> cTPx protein used, reported previously [1]. In similar <br /> manner, mtTPx recombinant protein purified E. coli <br /> cells described sent Covance Research Products          Figure 2 RT&#226;&#8364;&#8220;PCR analysis dsRNA-mediated impact TPx mRNA <br /> (Denver, PA, U.S.A.) preparation antisera rabbits. The    transcription Drosophila S2 cells <br />                                                <br /> immunoblots visualized using ECL + Western-blotting <br />                                              R <br />  <br />  <br /> detection (Amersham) according manufacturer&#226;&#8364;&#8482;s             Transcriptional levels variant 1 cTPx gene (cTPx-1) shown (A) mtTPx <br />                                                                         gene (mtTPx) (B). PCR signals cTPx obtained using primers 4783-F-1/4783-R, <br /> instructions. Densitometric scanning gels films           signals mtTPx obtained using primers 5037-F/5037-R. As internal <br /> performed digital imaging analysis AlphaEase         control, PCR fragments generated using primers rp49 gene. RNA samples PCR <br /> Stand Alone Software (Alpha Innotech Corp., San Leandro, CA,            extracted cell cultures treated 700 bp dsRNA generated cTPx-1 variant <br /> U.S.A.).                                                                (lane 1), 700 bp dsRNA cTPx-2 variant (lane 2), 300 bp dsRNA mtTPx (lane 3), <br />                                                                R <br />   RNA RT&#226;&#8364;&#8220;PCR analysis isolated using TRIzol LS                  1000 bp dsRNA mtTPx (lane 4), non-specific dsRNA (lane 5) untreated cells <br /> Reagent (Gibco BRL, Gaithersburg, MD, U.S.A.) 200 ng                (lane 6). <br /> was used assay. Reverse transcription performed <br /> as reported previously [1]. Reverse primers 4783-R                 Statistical analysis <br /> for cTPx gene 5037-R mtTPx gene indicated <br /> in Figure 1. RNA&#226;&#8364;&#8220;DNA hybrids denatured heating              All experiments performed times results <br /> 5 min 99 &#226;&#8212;&#166;C. The cDNA preparations used PCRs            means +&#226;&#710;&#8217; S.D., determined using INSTAT 3.00 GraphPad <br /> primer pairs TPx gene primers ribosomal            Prism 3.0a software. <br /> protein (rp) 49 gene control. Forward primers 4783- <br /> F-1 cTPx-1 variant, 4783-F-2 cTPx-2 variant <br />                                                                         RESULTS <br /> and 5037-F mtTPx gene. Reverse primers used for <br /> PCR used reverse transcription.              Gene silencing Drosophila cells <br /> Primers amplification rp49 used <br />                                                                         The efficiency kinetics gene silencing, mediated RNAi, <br /> previously [5]. <br />                                                                         determined S2 cells 2, 3, 7 45 days treatment, <br />                                                                         using immunoblot analysis RT&#226;&#8364;&#8220;PCR. As reported <br /> Experimentally induced stress cell viability                        previously [1], cTPx gene (DPx-4783) alternatively <br />                                                                         spliced transcripts differ 5  non- <br /> Untransfected Schneider S2 cells stable transfected cell <br />                                                                         coding segments. RT&#226;&#8364;&#8220;PCR used examine efficiency of <br /> lines overexpressing TPx genes [1] transfected <br />                                                                         gene silencing variants dsRNAs, 700 bp, had <br /> with vector cell lines treated dsRNAs were <br />                                                                         approx. 500 bp coding domain common upstream <br /> grown 22&#226;&#8364;&#8220;24 &#226;&#8212;&#166;C DES complete media 1&#226;&#8364;&#8220;2 days until <br />                                                                         sequence specific variants. Relative abundance <br /> a density approx. 106 cells/ml reached. Cell cultures <br />                                                                         assessed primer pairs domains <br /> were exposed 10 mM paraquat, 20 mM hydrogen peroxide, <br />                                                                         used generate dsRNAs. On 7th day treatment, <br /> 0.5 mM CdCl2 , 2 mM CuSO4 , incubated 32 42 &#226;&#8212;&#166;C <br />                                                                         marked perdurance dsRNAs observed (results not <br /> for different time intervals. In addition, cells were <br />                                                                         shown), indicating propagation quite efficient. <br /> incubated 0.5 mg/ml N-acetyl-L-cysteine (NAC; Sigma) <br />                                                                            To differentiate endogenous mRNAs added <br /> or 1 mM buthionine sulphoximine (BSO; Sigma) 2&#226;&#8364;&#8220;24 h <br />                                                                         dsRNAs, primer pairs (4783-F-1/4783-R 4783-F-2/4783-R) <br /> before exposure heat, heavy metal hydrogen peroxide. <br />                                                                         devised target mRNA sequence outside region used in <br /> After treatment, cells stained Trypan Blue according <br />                                                                         generation dsRNA. A severe reduction variant 1 mRNA <br /> to manufacturer&#226;&#8364;&#8482;s instructions (Invitrogen) counted in <br />                                                                         levels cell line, treated cTPx-1 dsRNA (Figure 2A, <br /> a haemocytometer. Viability determined ratio the <br />                                                                         lane 1), approx. 3 days treatment. cTPx-1 <br /> percentage live cells total number cells. <br />                                                                         transcript levels altered control non-treated cells <br />                                                                         (Figure 2A, lane 6), cells treated non-specific dsRNA <br />                                                                         (lane 5) dsRNA targeting mtTPx gene (lanes 3 4). <br /> Apoptosis                                                               Partial suppression mRNA levels observed cells treated <br /> DNA fragmentation characteristic apoptosis evaluated        dsRNA, specifically targeted cTPx-2 variant <br /> by agarose-gel electrophoresis [6]. Briefly, total DNA              (lane 2). <br /> isolated S2 cells using Puregene kit (PCG Scientific           For mtTPx gene, primers 5037-F/5037-R used <br /> Corp., Frederick, MD, U.S.A.) according manufacturer&#226;&#8364;&#8482;s           identify specifically mRNA dsRNA the <br /> instructions, DNA precipitation propan-2-ol          cell line treated 300 bp dsRNA. The use 300 bp <br /> performed incubation ice 30 min. DNA concentration            dsRNA did result discernible gene silencing, measured <br /> was determined spectrophotometrically 260 nm equal               transcript levels. However, 1000 bp dsRNA used, <br /> amounts loaded 1.2 % agarose gel. After                    significant decrease RNA levels relative controls was <br /> electrophoresis, gels examined UV.                           observed approx. 7 days treatment (Figure 2B, lane 4). <br />  <br />                                                                                                                                            <br />                                                                                                                                          c 2003 Biochemical Society <br />  746                S. N. Radyuk, R. S. Sohal W. C. Orr <br />  <br />  <br />  <br />  <br /> Figure 3 Immunoblot analysis proteins isolated S2 cell lines after <br /> exposure dsRNAs <br /> Cells treated dsRNAs crude lysates prepared day 3 subjected <br /> to immunoblot analysis. Cells treated 300 bp dsRNA corresponding mtTPx <br /> (lane 1), 1000 bp dsRNA corresponding mtTPx (lane 2), 700 bp dsRNA cTPx-1 variant <br /> (lane 3), 700 bp dsRNA cTPx-2 variant (lane 4), non-specific dsRNA (lane 5), untreated <br /> (lane 6). Signals obtained cTPx, mtTPx CuZn&#226;&#8364;&#8220;superoxide dismutase antibodies. <br />  <br />  <br />    To assess effects RNAi protein production, <br /> immunoblot analysis performed using antibodies raised <br /> against cTPx mtTPx proteins. After treatment either <br /> cTPx variant, significant decrease level cTPx protein <br /> was observed day 2, day 3 virtually undetectable                                    Figure 4     Analysis protein DNA extracts RNAi cell lines <br /> (Figure 3B, lanes 3 4). At time, significant decrease <br />                                                                                                      (A) Immunoblot analysis protein carbonyl content different cell cultures. Protein extracts <br /> in level mtTPx protein observed cell line <br />                                                                                                      subjected dinitrophenyl hydrazine treatment separated PAGE followed by <br /> treated 1000 bp dsRNA mtTPx gene (Figure 3A,                                             immunoblot analysis using anti-N2 ph antibodies. (B) Gel electrophoresis DNA isolated <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> lane 2) lesser extent cell line treated 300 bp                                  different cell cultures. Total DNA preparations (5 &#194;&#181;g) loaded sample. Samples <br /> dsRNA mtTPx gene (Figure 3A, lane 1). Antibodies raised                                       isolated cells treated dsRNA derived cTPx-1 gene variant denoted as <br /> against CuZn&#226;&#8364;&#8220;superoxide dismutase protein used control                                       cTPx, samples cells treated dsRNA derived mtTPx gene denoted <br /> protein-loading amounts (Figure 3C). The block cTPx protein                                       mtTPx. Control, S2 cells treated non-specific dsRNA; M, Hi-Lo DNA molecular-mass <br />                                                                                                      marker. Arrows indicate DNA fragments correspond 180 bp increments. <br /> accumulation continued 45 days cell line treated the <br /> cTPx-1-specific dsRNA variant reached pretreatment levels <br /> only 60 days (results shown). Cell lines underexpressing                                   vitro protect S2 cells oxidative damage caused <br /> TPx assessed absence environmental                                          hydrogen peroxide exposure [1]. In present study, we <br /> challenge respect steady-state levels oxidative damage                                    observed cell lines decreased TPx expression were <br /> and susceptibility apoptosis. To estimate impact TPx                                       susceptible hydrogen peroxide exposure, measured <br /> underexpression steady-state levels oxidative damage, crude                                    cell viability (see Figure 5A). Similar results obtained <br /> cell lysates analysed protein carbonyl content, using anti-                                 cells challenged paraquat (Figure 5B). Cell lines <br /> 2,4-dinitrophenyl (N2 ph) monoclonal antibodies (Sigma),                                     overexpressing cTPx mtTPx proteins resistant <br /> results presented Figure 4(A). The carbonyl content                                       paraquat relative vector-only cell lines (Figure 5C). Taken <br /> substantially lower control line compared cell                                      together, observations strongly suggest cTPx mtTPx <br /> lines production TPx blocked. In addition,                                          play antioxidative cytoprotective role Drosophila cells. <br /> preliminary analysis indicated cell lines underexpressing <br /> TPx showed greater susceptibility apoptosis, shown the <br /> laddered DNA fragmentation observed Figure 4(B).                                                  Effects temperature <br />                                                                                                      Cells, under- overexpressing TPxs, exposed <br /> Effects over- under-expression TPx genes S2 cell                                        heat stress (32 &#226;&#8212;&#166;C) heat shock (42 &#226;&#8212;&#166;C). In cell lines lacking <br /> viability different stress conditions                                                          cTPx mtTPx, detectable difference cell <br /> To assess impact over- under-expression cTPx                                       viability relative controls heat shock (results shown) <br /> and mtTPx genes cell physiology, cells challenged                                            increase cell viability dsRNA- <br /> with different stressors supposed increase oxidative stress.                                      treated cells 32 &#226;&#8212;&#166;C abrogated 48 h incubation <br /> In experiments, &#226;&#8364;&#732;null&#226;&#8364;&#8482; lines used                                        (Figure 6A). Cells overproducing cTPx mtTPx were <br /> <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> established confirmed; specifically, cell                                      susceptible heat treatment 32 42 &#226;&#8212;&#166;C. At <br /> line cTPx-RNAi, generated treatment 700 bp dsRNA                                             42 &#226;&#8212;&#166;C, difference viability observed early as <br /> derived cTPx-1, cell line mtTPx-RNAi, generated                                      30 min initiation heat shock (Figure 6B), the <br /> treatment 1000 bp dsRNA derived mtTPx. As negative                                         effects heat stress observed 24 h (Figure 6C). <br /> controls, cell lines treated non-specific dsRNA cell lines <br /> transfected pMTV5 vector used. <br />                                                                                                      Effects heavy metals <br />                                                                                                      The concentrations metals (0.5 mM CdCl2 2 mM <br /> Effects hydrogen peroxide paraquat                                                            CuSO4 ) selected based ability cause 40&#226;&#8364;&#8220;60 % <br /> It demonstrated previously cTPx mtTPx                                               cell death control cells 24 h incubation period. Cells <br /> gene products possess thioredoxin-dependent peroxidase activity                                      subjected dsRNA treatment did differ control cells in <br />  <br />   <br /> c 2003 Biochemical Society <br />                                                                                             Thioredoxin peroxidase cell viability stress conditions                             747 <br />  <br />  <br />                                                                                             resistance cadmium copper (results shown). <br />                                                                                             Cell lines overexpressing TPx variants more <br />                                                                                             resistant cadmium exposure compared control cells <br />                                                                                             (Figure 7A). In contrast, treatment 2 mM CuSO4 led to <br />                                                                                             increased mortality cell lines overexpressing cTPx or <br />                                                                                             mtTPx (Figure 7B). <br />  <br />                                                                                             Effects NAC BSO <br />                                                                                             Different agents modify cellular glutathione concentrations <br />                                                                                             [7,8] increase decrease oxidative stress tested <br />                                                                                             ability sensitize protect S2 cells exposed different <br />                                                                                             environmental insults. Cells exposed 0.5 mg/ml <br />                                                                                             NAC, glutathione precursor, 1 mM BSO, glutathione- <br />                                                                                             depleting agent. NAC BSO, taken separately, effect <br />                                                                                             <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> characteristics chosen concentrations a <br />                                                                                             period 24 h. After 36 h incubation BSO, cell viability <br />                                                                                             decreased approx. 70 % control cells, 48 h <br />                                                                                             incubation period required achieve comparable effect in <br />                                                                                             cTPx cell line (results shown). The cell line overexpressing <br />                                                                                             mtTPx remained 90&#226;&#8364;&#8220;95 % viable 48 h incubation BSO, <br />                                                                                             indicating significant sparing effects TPx overexpression at <br />                                                                                             decreased glutathione levels. <br />                                                                                                No statistically significant differences observed between <br />                                                                                             samples subjected heat stress (Figure 8A) heat shock (results <br />                                                                                             shown) combination NAC BSO. <br />                                                                                                Cytotoxicity 24 h exposure cadmium, resulted <br />                                                                                             50 % cell mortality control cells, substan- <br />                                                                                             tially reversed cells exposed NAC, comparable <br />                                                                                             observed cells overexpressing TPx (Figure 8B). <br />                                                                                             When co-treated cadmium BSO, TPx overexpressing <br />                                                                                             cell lines resistant cadmium compared <br />                                                                                             control cells. The effects copper treatment contrasted <br />                                                                                             markedly cadmium (Figure 8C). In control cells, <br />                                                                                             combination copper NAC led decrease cell <br />                                                                                             viability compared cells treated copper alone <br />                                                                                             effect significantly enhanced cells overexpressing <br />                                                                                             cTPx greater extent cells overexpressing mtTPx. On <br />                                                                                             hand, BSO treatment conferred greater resistance to <br />                                                                                             cells overexpressing TPx relative controls. To complement <br />                                                                                             analysis, cells underexpressing cTPx mtTPx were <br />                                                                                             exposed similar treatments. These RNAi cell lines did appear <br />                                                                                             exhibit slightly greater resistance Cu + NAC Cu + <br />                                                                                             BSO combination, differences statistically <br />                                                                                             significant (results shown). <br />  <br />                                                                                             Overexpression TPx levels, carbonyl content DNA <br />                                                                                             fragmentation analysis <br />                                                                                             Protein carbonyls measured assess levels oxidative <br />                                                                                             damage caused different stressors (Figures 9A, 9C 9F). In <br />                                                                                             control cells, treatment paraquat, cadmium hydrogen <br />                                                                                             peroxide elicits carbonyl accumulation relative non- <br />                                                                                             treated cells cells subjected elevated temperature <br />                                                                                             treatment copper (Figure 9A). The effect reduced <br />                                                                                             paraquat hydrogen peroxide treatments cell lines <br />                                                                                             overexpressing TPx (Figures 9C 9F). No correlation was <br />  <br />  <br />                                                                                             mtTPx-RNAi cells, mock-treated cell line used. As control cTPx[+] mtTPx[+] cell <br /> Figure 5     S2 cell viability treatment hydrogen peroxide                   lines, pMTV5 vector-transfected cell line used. Percentage cell survival determined <br /> paraquat                                                                                    various time intervals. Bar charts cell survival experimental cell lines relative <br />                                                                                             controls. Inset: percentage live cells control experimental cell lines. Statistically <br /> S2 cells depleted cTPx (cTPx-RNAi) mtTPx (mtTPx-RNAi) exposed 20 mM           significant differences (*P &lt; 0.05) treated non-treated cells indicated. Note that <br /> hydrogen peroxide (A) 10 mM paraquat (B). Cell lines overexpressing cTPx (cTPx[+])    vector-only control (C) susceptible paraquat compared mock-treated <br /> mtTPx (mtTPx[+]) exposed 10 mM paraquat (C). As control cTPx-RNAi         control (B), presumably presence additional genetic component. <br />  <br />                                                                                                                                                                   <br />                                                                                                                                                                 c 2003 Biochemical Society <br />  748                S. N. Radyuk, R. S. Sohal W. C. Orr <br />  <br />  <br />                                                                                                      observed levels protein carbonyls levels <br />                                                                                                      overexpressed TPxs measured immunoblot analysis <br />                                                                                                      anti-V5 antibodies (Figures 9E 9H). For example, <br />                                                                                                      expression levels similar samples treated hydrogen <br />                                                                                                      peroxide, paraquat heat, levels protein carbony- <br />                                                                                                      lation higher samples treated paraquat hydrogen <br />                                                                                                      peroxide compared heat (Figures 9C, 9E, 9F and <br />                                                                                                      9H). These results confirmed using antibodies specific <br />                                                                                                      cTPx mtTPx (results shown), showing that, all <br />                                                                                                      instances, TPx expression overexpressing cell lines was <br />                                                                                                      elevated relative endogenous levels control cells. <br />                                                                                                      Finally, significant variations TPx expression observed in <br />                                                                                                      response different stressors present study partly <br />                                                                                                      ascribed transcriptional control, additional levels post- <br />                                                                                                      transcriptional post-translational regulation ruled <br />                                                                                                      out. <br />                                                                                                         Analysis DNA fragmentation used evaluate <br />                                                                                                      apoptotic changes cell lines exposed different stress <br />                                                                                                      conditions (Figure 10). No DNA degradation observed in <br />                                                                                                      samples isolated non-treated cells cells subjected <br />                                                                                                      short periods exposure hydrogen peroxide, heat shock <br />                                                                                                      treatment cadmium. Barely detectable DNA laddering <br />                                                                                                      observed samples paraquat treatment. A typical <br />                                                                                                      internucleosomal fragmentation 180&#226;&#8364;&#8220;200 bp multiples was <br />                                                                                                      observed longer exposures hydrogen peroxide, heat stress <br />                                                                                                      treatment copper. In samples treated copper and <br />                                                                                                      heat stress, DNA fragmentation appeared pronounced in <br />                                                                                                      cell lines overexpressing TPx compared control. <br />  <br />                                                                                                      DISCUSSION <br />                                                                                                      The use RNAi Drosophila cell culture proved a <br />                                                                                                      effective rapid means characterize possible functional <br />                                                                                                      roles TPx genes permitting significant specific down- <br />                                                                                                      regulation. A significant reduction mRNA levels observed <br />                                                                                                      approx. 72 h dsRNA administration, similar that <br />                                                                                                      reported previously [9]. A surprising finding RNAi <br />                                                                                                      effect relatively long lasting, 45 days case, which <br />                                                                                                      suggests dsRNA quite stable cause effective <br />                                                                                                      gene silencing low levels. These findings consistent <br />                                                                                                      notion post-transcriptional mechanism a <br />                                                                                                      nuclease incorporates dsRNAs guide target specifically <br />                                                                                                      mRNAs, proposed Hammond et al. [10]. <br />                                                                                                         The efficiency RNAi dependent <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> and <br />                                                                                                      sequence dsRNAs. Consistent reports for <br />                                                                                                      Drosophila [11], observed suppression generally <br />                                                                                                      effective length dsRNA 500&#226;&#8364;&#8220;1000 bp <br />                                                                                                      range (the present study; S. N. Radyuk, R. S. Sohal W. C. <br />                                                                                                      Orr, unpublished work). Overall, ease, duration <br />                                                                                                      selectivity target genes &#226;&#8364;&#732;knocked-out&#226;&#8364;&#8482; in <br />                                                                                                      cell culture render approach effective tool functional <br />                                                                                                      analysis. <br />                                                                                                         In recent study, showed overexpression either <br />                                                                                                      cTPx mtTPx Drosophila cells conferred greater <br />                                                                                                      resistance hydrogen peroxide treatment [1]. In present study, <br />                                                                                                      demonstrate blocking TPx expression opposite <br />                                                                                                      effect, making cells relatively susceptible hydrogen <br />                                                                                                      peroxide treatment. This finding extended exposing <br />                                                                                                      cells over- underexpressing TPx paraquat, source of <br />                                                                                                      superoxide radical generation. Again, cells overexpressing either <br /> Figure 6      S2 cell viability heat stress heat shock conditions                          TPx variants resistant compared with <br /> S2 cells depleted cTPx (cTPx-RNAi) mtTPx (mtTPx-RNAi) exposed 32 &#226;&#8212;&#166;C <br /> heat stress (A). Overexpressing cell lines cTPx[+] mtTPx[+] exposed 42 &#226;&#8212;&#166;C heat <br /> shock (B) 32 &#226;&#8212;&#166;C heat stress (C). Percentage cell survival determined indicated       Inset: percentage live cells control experimental cell lines. Statistically significant <br /> time intervals. Bar charts cell survival experimental cell lines relative controls.   differences (*P &lt; 0.05) treated non-treated cells indicated. <br />  <br />   <br /> c 2003 Biochemical Society <br />                                                                                                           Thioredoxin peroxidase cell viability stress conditions                       749 <br />  <br />  <br />  <br />  <br /> Figure 7      S2 cell viability cadmium (A) copper (B) treatments <br /> Stable overexpressing cell lines cTPx[+] mtTPx[+] exposed 0.5 mM CdCl2 (A) or <br /> 2 mM CuSO4 (B) percentage cell survival determined indicated time intervals. <br /> Bar charts cell survival transfected cell lines relative controls. Inset: percentage of <br />                                                                                                           Figure 8 Viability stably transfected TPx overexpressing cell lines after <br /> live cells control transfected cell lines. Statistically significant differences (*P &lt; 0.05) <br />                                                                                                           treatment stressors combination glutathione-modulating agents <br /> between treated non-treated cells indicated. <br />                                                                                                           (A) Heat stress (Hstr) combination NAC BSO. Overexpressing cell lines cTPx[+] and <br />                                                                                                           mtTPx[+] exposed 0.5 mg/ml NAC 1 mM BSO 2 h room temperature followed <br /> control cells, cells underexpressing TPx                                                24 h incubation 32 &#226;&#8212;&#166;C. (B) Cadmium treatment combination NAC BSO. <br /> sensitive. Furthermore, hydrogen peroxide- paraquat-induced                                           Overexpressing cell lines cTPx[+] mtTPx[+] exposed 0.5 mg/ml NAC 1 mM <br />                                                                                                           BSO 0.5 mM CdCl2 24 h. (C) Copper treatment combination NAC <br /> protein oxidative damage, indicated carbonyl content,                                              BSO. Overexpressing cell lines cTPx[+] mtTPx[+] exposed 0.5 mg/ml NAC or <br /> lower TPx overexpressing cell lines relative control                                            1 mM BSO 2 mM CuSO4 12 h. Bar charts percentage live cells. <br /> (Figure 9). This finding demonstrates role TPxs directly <br /> shielding cells elevated ROS. In absence any <br /> environmental challenge, modulation TPx activity,                                            stress. Overall, results point important role these <br /> or down, impact cell growth rate cell survival. It                                       TPx species, protecting direct oxidative damage <br /> surprising absence TPx appears little negative                                        leading cell necrosis and/or preventing progression towards <br /> impact normal conditions, result ascribed                                        apoptosis. <br /> overlapping functions intracellular antioxidants [12,13].                                            Stressors, elevated temperature copper, been <br /> These results agreement results obtained                                            reported act, part, oxidative stress-related <br /> manipulations TPx expression yeast cells [14]. Although                                             mechanisms. In present study, however, impact on <br /> our TPx &#226;&#8364;&#732;<span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>knock-out</span>&#226;&#8364;&#8482; cell lines appear unaffected normal                                             cells over- underexpressing TPx quite distinct from <br /> conditions terms cell growth rate viability,                                          treatments involving direct application ROS, as <br /> increased levels protein carbonyls signs DNA                                             hydrogen peroxide paraquat. Indeed, cell lines overexpressing <br /> laddering, suggesting cells degree                                           cTPx mtTPx susceptible stressors <br />  <br />                                                                                                                                                                            <br />                                                                                                                                                                          c 2003 Biochemical Society <br />  750               S. N. Radyuk, R. S. Sohal W. C. Orr <br />  <br />  <br />  <br />  <br /> Figure 9     Protein carbonyl content TPx overexpressing cell lines subjected various stressors <br /> The protein extracts derived overexpressing cell line cTPx[+] (C&#226;&#8364;&#8220;E) mtTPx[+] (F&#226;&#8364;&#8220;H) control cells transfected pMTV-5 vector (A, B). Protein carbonyl content, <br /> obtained using anti-N2 ph antibodies, shown (A), (C) (F). Coomassie Blue protein staining shown (B), (D) (G), signals obtained anti-V5 antibodies shown in <br /> (E) (H). Cells cultured density 106 cells/ml exposed following stressors: 42 &#226;&#8212;&#166;C (t &#226;&#8212;&#166;) 1 h, 2 mM CuSO4 (Cu) 6 h, 10 mM paraquat (PQ) 24 h, 0.5 mM CdCl2 <br /> (Cd) 24 h, 20 mM hydrogen peroxide (H2 O2 ) 1 h, untreated (&#226;&#8364;&#8220;). The bar chart shows relative protein carbonyl content. Statistically significant differences (*P &lt; 0.05) control and <br /> overexpressing cell lines indicated. OD, absorbance. <br />  <br />  <br /> relative vector-only control cell lines, cells                                         postulated relevant, peroxiredoxins <br /> decreased TPx levels resistant similar                                          shown possess peroxynitrite reductase activity <br /> resistance controls. In contrast, earlier study Lee                                       [19]. One possible explanation detrimental effects TPx <br /> Park [15] determined mutant yeast strain lacking TPx                                            overexpression cell survival thermal challenge be <br /> was thermosensitive compared isogenic wild-                                         alteration balance antioxidant defences <br /> type control. Moreover, demonstrated protein damage,                                         required minimize oxidative stress sustain optimal <br /> measured carbonyl content, higher &#226;&#8364;&#732;knock-out&#226;&#8364;&#8482;                                            ROS- RNS-mediated redox state signalling mechanisms. <br /> cell line relative wild-type, carbonyl content increased                                        The observed increase DNA degradation laddering in <br /> when cells challenged heat. They concluded                                          heat-treated cell lines overexpressing TPxs consistent with <br /> antioxidative function TPx facilitates cell&#226;&#8364;&#8482;s defence                                   role signalling mechanisms mediation heat- <br /> heat shock. We observed higher protein carbonyl content                                         induced cytotoxicity. Alternatively, possible TPx labile <br /> in TPx &#226;&#8364;&#732;knock-out&#226;&#8364;&#8482; cell lines normal conditions,                                         hyperthermia and, present high levels, form <br /> observed change carbonyl levels exposure                                         aggregates cause cytotoxicity [20]. These possibilities are <br /> heat stress heat shock.                                                                             currently investigated. <br />    Heat stress known enhance cytotoxicity,                                             The exposure cell lines, underexpressing TPx, cadmium <br /> precise mechanisms involved cell death currently                                       copper impact cell viability. On <br /> defined. In instances, oxidative stress considered                                    hand, TPx overexpressing cells resistant to <br /> play role hyperthermia-induced cell death [16,17]. However,                                       cadmium treatment susceptible copper treatment. It <br /> it increased ROS did necessarily correlate                                          known elevation concentrations transition <br /> with increased cytotoxicity [18], observation supported                                      metals results pathological conditions consistent <br /> finding modulation oxidative status NAC, glutathione                                      oxidative damage. Cadmium induces oxidative stress a <br /> precursor, BSO, inhibitor GSH synthesis,                                            secondary response, causing depletion glutathione [21] <br /> impact rate cell death heat-stress conditions. A role                                  protein-bound thiol groups, resulting increased free radical <br /> for NO intermediary messenger cytoprotective effect                                       generation [22] lipid peroxidation [23]. Cadmium also <br />  <br />   <br /> c 2003 Biochemical Society <br />                                                                                                    Thioredoxin peroxidase cell viability stress conditions                          751 <br />  <br />  <br />  <br />  <br /> Figure 10      DNA fragmentation analysis TPx overexpressing cell lines subjected various stressors <br /> DNA preparations derived overexpressing cell lines cTPx[+] mtTPx[+] control cells transfected pMTV-5 vector. Other gel untreated cells, the <br /> first lane HiLo DNA marker (M), lane control, second cTPx overexpressor mtTPx overexpressor. The conditions used indicated. <br />  <br />  <br />  <br /> known inhibit enzymes thioredoxin                                       beneficial detrimental effects viability Drosophila <br /> via interaction vicinal thiol groups [24]. The lack                                     cells. <br /> significant effect cadmium viability underexpressing <br /> cell lines yeast TPx knockouts suggests                                      We grateful Irina Dobrosotskaya (UT Southwestern Medical Center, Dallas, TX, <br /> detrimental effect cell survival caused cadmium treatment                                     U.S.A.) advice conducting RNAi studies. This work supported NIH grant no. <br /> is inactivation TPx. According findings,                                       AG15122. <br /> resistance cadmium increased TPx overexpressing cell <br /> lines extent control cells treated NAC,                                REFERENCES <br /> delivers intracellular free cysteine required glutathione <br /> synthesis. When TPx overexpressing cell lines treated                                      1 Radyuk, S. N., Klichko, V. I., Spinola, B., Sohal, R. S. Orr, W. C. (2001) The <br /> BSO, inhibitor glutathione synthesis,                                         peroxiredoxin gene family Drosophila melanogaster . Free Radical Biol. Med. 31, <br /> be resistant control cells, confirming overlapping                                       1090&#226;&#8364;&#8220;1100 <br />                                                                                                      2 Clemens, J. C., Worby, C. A., Simonson-Leff, N., Muda, M., Maehama, T., Hemmings, <br /> functional roles glutathione TPxs cell. We suggest <br />                                                                                                        B. A. Dixon, J. E. (2000) Use double-stranded RNA interference Drosophila cell <br /> that main protective role TPxs cadmium-induced                                          lines dissect signal transduction pathways. Proc. Natl. Acad. Sci. U.S.A. 97, <br /> cytotoxicity substitution glutathione.                                                6499&#226;&#8364;&#8220;6503 <br />    By contrast, cells overexpressing TPx vulnerable                                     3 Sambrook, J., Fritsch, E. F. Maniatis, T. (1989) Molecular Cloning: A Laboratory <br /> copper compared control cells. Studies shown                                       Manual, 2nd edn, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY <br /> copper undergo redox cycling, resulting production                                     4 Yan, L. J., Levine, R. L. Sohal, R. S. (1997) Oxidative damage aging targets <br /> hydroxyl radicals Fenton chemistry [25]. There evidence                                         mitochondrial aconitase. Proc. Natl. Acad. Sci. U.S.A. 94, 11168&#226;&#8364;&#8220;11172 <br /> that antioxidants, including containing thiol groups,                                     5 Candas, M., Sohal, R. S., Radyuk, S. N., Klichko, V. I. Orr, W. C. (1997) Molecular <br /> such homocysteine, initiate cytotoxic effects                                          organization glutathione reductase gene Drosophila melanogaster . Arch. <br />                                                                                                        Biochem. Biophys. 339, 323&#226;&#8364;&#8220;334 <br /> induction oxidative stress presence Cu2+ [26,27]. In <br />                                                                                                      6 Arends, M. J., Morris, R. G. Wyllie, A. H. (1990) Apoptosis. The role the <br /> the proposed scheme, Cu(II) reduced Cu(I) subsequently                                       endonuclease. Am. J. Pathol. 136, 593&#226;&#8364;&#8220;608 <br /> re-oxidized Fenton-like reaction hydrogen peroxide                                     7 Shen, H., Yang, C., Liu, J. Ong, C. (2000) Dual role glutathione selenite-induced <br /> O2 , leading formation O2 &#194;&#183;&#226;&#710;&#8217; ,&#194;&#183; OH ROS [28]. In                                 oxidative stress apoptosis human hepatoma cells. Free Radical Biol. Med. 28, <br /> experiments, NAC clear detrimental effect cell survival                                       1115&#226;&#8364;&#8220;1124 <br /> when combined copper, effect enhanced                                       8 Vairetti, M., Griffini, P., Pietrocola, G., Richelmi, P. Freitas, I. (2001) Cold-induced <br /> by TPx overexpression (Figure 8). This effect particularly                                         apoptosis isolated rat hepatocytes: protective role glutathione. Free Radical Biol. <br /> pronounced TPx overexpressed mitochondrion,                                             Med. 31, 954&#226;&#8364;&#8220;961 <br /> which unexpected given presumed role                                    9 Caplen, N. J., Fleenor, J., Fire, A. Morgan, R. A. (2000) dsRNA-mediated gene <br /> compartment plays oxidative stress. We propose Cu(II)                                          silencing cultured Drosophila cells: tissue culture model analysis RNA <br />                                                                                                        interference. Gene 252, 95&#226;&#8364;&#8220;105 <br /> may reduced thiol groups cysteine active site of <br />                                                                                                     10 Hammond, S. M., Bernstein, E., Beach, D. Hannon, G. J. (2000) An RNA-directed <br /> TPx, competing reduction hydrogen peroxide.                                           nuclease mediates post-transcriptional gene silencing Drosophila cells. Nature <br /> Our results (Figure 10) suggest formation ROS                                       (London) 404, 293&#226;&#8364;&#8220;296 <br /> this process trigger apoptosis.                                                            11 Yang, D., Lu, H. Erickson, J. W. (2000) Evidence processed small dsRNAs may <br />    Overall, manipulations levels TPx demonstrated                                     mediate sequence-specific mRNA degradation RNAi Drosophila embryos. Curr. <br /> different conditions stress enzymes                                           Biol. 10, 1191&#226;&#8364;&#8220;1200 <br />  <br />  <br />                                                                                                                                                                        <br />                                                                                                                                                                      c 2003 Biochemical Society <br />  752               S. N. Radyuk, R. S. Sohal W. C. Orr <br />  <br />  <br /> 12 Demasi, A. P., Pereira, G. A. Netto, L. E. (2001) Cytosolic thioredoxin peroxidase I   21 Almazan, G., Liu, H. N., Khorchid, A., Sundararajan, S., Martinez-Bermudez, A. K. and <br />    essential antioxidant defense yeast dysfunctional mitochondria. FEBS Lett.       Chemtob, S. (2000) E <br /> </body></html>