<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biochem. J. (2006) 396, 287&#226;&#8364;&#8220;295 (Printed Great Britain)   doi:10.1042/BJ20051825                                                                               287 <br />  <br />  <br /> Phosphatidylinositol synthesis essential bloodstream form <br /> Trypanosoma brucei <br /> Kirstee L. MARTIN Terry K. SMITH1 <br /> Division Biological Chemistry Molecular Microbiology, School Life Sciences, University Dundee, Dundee DD1 5EH, Scotland, U.K. <br />  <br />  <br />  <br />  <br /> PI (phosphatidylinositol) ubiquitous eukaryotic phospholipid                          gene. In vivo labelling conditional double <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>knockout</span> cells <br /> which serves precursor messenger molecules GPI                                confirmed result, showing decrease PI <br /> (glycosylphosphatidylinositol) anchors. PI synthesized                           formed cells grown non-permissive conditions. <br /> de novo head group exchange PIS (PI synthase). The                              Furthermore, quantitative qualitative analysis GLC-MS <br /> synthesis GPI anchors previously validated                                ESI-MS/MS (electrospray ionization MS/MS) respectively <br /> genetically chemically drug target Trypanosoma                                 showed significant decrease (70 %) cellular PI, appears <br /> brucei, causative parasite African sleeping sickness. How-                          affect major PI species equally. A consequence fall <br /> ever, known synthesis PI organism.                         PI level knock-on reduction GPI biosynthesis which <br /> Database mining revealed putative TbPIS gene T. brucei                            essential parasite&#226;&#8364;&#8482;s <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span>. The results presented here <br /> genome recombinant expression characterization                               PI synthesis essential bloodstream form T. brucei, <br /> was shown encode catalytically active PIS, high spe-                           knowledge report dependence on <br /> cificity myo-inositol. Immunofluorescence revealed                             PI synthesis protozoan parasite genetic validation. <br /> T. brucei, PIS endoplasmic reticulum and <br /> Golgi. We created conditional double knockout TbPIS                            Key words: bloodstream form, essentiality, glycosylphosphat- <br /> bloodstream form T. brucei, grown non-                                 idylinositol, myo-inositol, phosphatidylinositol synthase (PIS), <br /> permissive conditions, clearly showed TbPIS essential                           Trypanosoma. <br />  <br />  <br />  <br /> INTRODUCTION                                                                                  African trypanosomiasis caused protozoan parasite <br />                                                                                            Trypanosoma brucei potentially fatal disease a <br /> In eukaryotes, PI (phosphatidylinositol) ubiquitous phospho-                          economic problem sub-Saharan Africa. This unicellular <br /> lipid forms 3 10 % cell membranes, functions                           parasite able avoid host&#226;&#8364;&#8482;s innate immune by <br /> as precursor cell signalling molecules provides basic                        undergoing antigenic variation involves switching GPI- <br /> building block used GPI (glycosylphosphatidylinositol) anchor                           anchored VSGs (variant-surface glycoproteins) [11]. Despite the <br /> biosynthesis. PI synthesized novo action PIS                            variation VSG protein, GPI core structure attached <br /> (PI synthase; EC 2.7.8.11) using myo-inositol CDP-DAG                                  protein remains unchanged comprises NH2 CH2 CH2 PO4 H- <br /> (CDP diacylglycerol) releasing CMP. Alternatively,                              6Man&#206;&#177;1-2Man&#206;&#177;1-6Man&#206;&#177;1-4GlcN&#206;&#177;1-6D-myo-inositol-1-HPO4 - <br /> absence CDP-DAG, head group exchanged                                dimyristylglycerol [12]. The biosynthesis GPI anchors has <br /> pre-existing PI free myo-inositol. Native recombinant                              genetically chemically validated potential <br /> PIS enzymes studied organisms including                                therapeutic drug target bloodstream form T. brucei [13&#226;&#8364;&#8220;15]. <br /> Saccharomyces cerevisiae [1&#226;&#8364;&#8220;4], Arabidopsis thaliana [5&#226;&#8364;&#8220;7]                                PI utilized initial step GPI anchor biosynthesis, <br /> Toxoplasma gondii [8]. PIS enzymes appear predominantly                              GlcNAc transferred UDP-GlcNAc PI form <br /> localized ER (endoplasmic reticulum),                            GlcNAc-PI (see [16] references contained therein). Surpris- <br /> also detected cellular locations Golgi [9],                          ingly, despite essentiality GPI anchors bloodstream form <br /> outer mitochondrial membrane S. cerevisiae [1,4] plasma                             T. brucei, little known PI biosynthesis these <br /> membrane rat pituitary GH3 cells [10]. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> date, PIS enzymes                         parasites. The synthesis PI demonstrated Plas- <br /> require Mg2+ Mn2+ activity neutral pH optima.                              modium knowlesi [17], Plasmodium falciparum [17], Crithidia <br /> Although ability catalyse PI synthesis exchange                        fasciculate [18] Giardi lamblia [19], date PIS <br /> reactions investigated PIS enzymes,                       synthesis shown essential survival of <br /> clearly shown recombinant PISs organisms,                              parasites. The report molecular cloning charac- <br /> particular S. cerevisiae [2] A. thaliana [6]. However, exact                       terization protozoan PIS To. gondii, which <br /> mechanism reaction physiological significance                             developmentally regulated PIS genes identified <br /> remain unknown.                                                                            [8]. <br />  <br />  <br />   Abbreviations used: BiP, endoplasmic reticulum luminal chaperone binding protein; DAG, diacylglycerol; DAPI, 4,6-diamidino-2-phenylindole; ER, <br /> endoplasmic reticulum; ESI-MS, electrospray ionization MS; GPI, glycosylphosphatidylinositol; HA, haemagglutinin; HPTLC, high-performance TLC; HYG, <br /> hygromycin phosphotransferase; IPTG, isopropyl &#206;&#178;-D-thiogalactoside; LB, Luria&#226;&#8364;&#8220;Bertani; Ni-NTA, Ni2+ -nitrilotriacetate; ORF, open reading frame; PAC, <br /> puromycin acetyltransferase; PI, phosphatidylinositol; PLC, phospholipase C; PI-PLC, PI-specific PLC; PIS, PI synthase; TbGRASP, T. brucei Golgi matrix <br /> protein; TbPIS, T. brucei PI synthase; TDB, trypanosome dilution buffer; Ti, tetracycline-inducible; TRITC, tetramethylrhodamine &#206;&#178;-isothiocyanate; UTR, <br /> untranslated region; VSG, variant-surface glycoprotein. <br />   1 <br />      To correspondence addressed (email T.K.Smith@dundee.ac.uk). <br />   The nucleotide sequence reported paper submitted DDBJ, EMBL, GenBank&#194;&#174; GSDB Nucleotide Sequence Databases under <br /> the accession number AJ716153. <br />  <br />                                                                                                                                                <br />                                                                                                                                              c 2006 Biochemical Society <br />  288             K. L. Martin T. K. Smith <br />  <br />  <br />    In present study, report investigations PI synthesis      (0.2 &#195;&#8212; SSC 0.1 % SDS). Bound probe detected using <br /> in bloodstream form T. brucei, involving molecular cloning            CDP-STAR detection module (Amersham) autoradio- <br /> the PIS, recombinant expression preliminary enzyme charac-           graphy. <br /> terization. We show, creation conditional double          For Northern blots, total RNA purified using RNeasy <br /> knockout, PI synthesis essential survival blood-   Mini kit (Qiagen) separated formaldehyde gel trans- <br /> stream form parasite. To knowledge,         ferred Hybond N+ (Amersham). Hybridization conditions <br /> report essentiality PI synthesis protozoan parasite.        stringency washes described Southern blotting, <br />                                                                          using 32 P-labelled PIS ORF probe. Bound probe was <br />                                                                          detected autoradiography. <br /> EXPERIMENTAL <br />                                                                          Cultivation genetic modification T. brucei <br /> Nucleic acid manipulations <br />                                                                          Bloodstream form T. brucei strain 427, pre- <br /> Using S. cerevisiae PIS, putative PIS gene identified <br />                                                                          viously modified express T7 polymerase tetra- <br /> in T. brucei genome database (Sanger Centre, Cambridge, <br />                                                                          cycline repressor protein [20], referred wild-type <br /> U.K.) using tBlastN. The ORF (open reading frame) PCR- <br />                                                                          cells convenience. Cells grown HMI-9 medium sup- <br /> amplified T. brucei genomic DNA Pfu polymerase using <br />                                                                          plemented G418 (2.5 &#194;&#181;g/ml), 37 &#226;&#8212;&#166;C 5 % CO2 as <br /> the forward reverse primers 5  -GAGGAGAAGCTTATG- <br />                                                                          described [13,20&#226;&#8364;&#8220;22]. Transformation conditions and <br /> CCGAAAGCTAAAACT-3  5  -TCGTTAATTAACTGGCG- <br />                                                                          subsequent drug selection described [13,20&#226;&#8364;&#8220; <br /> GCTTCCCGCAGC-3  respectively. The amplicon purified <br />                                                                          22]. For experiments requiring tetracycline-free conditions, Tet- <br /> (QIAquick PCR purification kit; Qiagen), cloned pCR- <br />                                                                          system-approved foetal calf serum (Clontech) used. When <br /> Blunt II TOPO (Invitrogen) sequenced. Using HindIII and <br />                                                                          tetracycline added medium, final concentration of <br /> PacI restriction sites (underlined primer sequences), putative <br />                                                                          1 &#194;&#181;g/ml used. <br /> TbPIS (T. brucei PIS gene) ligated tetracycline- <br /> inducible expression vectors pLew82 pLew100 [20] the <br /> HindIII PacI restriction sites.                                      Immunofluorescence <br />    To construct T. brucei gene replacement cassettes, 5  -       Mid-exponential cells collected centrifugation (800 g, <br /> UTR (5  -untranslated region) 3  -UTR immediately adjacent           10 min) fixed 4 % (w/v) paraformaldehyde. After wash- <br /> to PIS ORF amplified T. brucei genomic DNA using           ing PBS, fixed cells allowed adhere poly- <br /> Pfu polymerase. The primers 5  -ATAAGAATGCGGCCGCAT-                      lysine slides prior staining DAPI (4,6-diamidino- <br /> AATCACTTTAGCGTCGCGTGG-3  5  -GTTTAAACTTA-                            2-phenylindole; 2 &#194;&#181;g/ml) antibodies. Cells rehydrated <br /> CGGACCGTCAAGCTTTGGTGCTGCGTTGCTTGC-3                                 PBS washed PBS&#226;&#8364;&#8220;glycine (0.1 M) prior per- <br /> used 5  -UTR, primers 5  -GACGGTCCGTAAGTTTA-                 meabilization Triton X-100 (0.1 %) blocking 1 % <br /> AACGGATCCGGAGTTGTGTGTTAAAGG-3  5  -ATAA-                             BSA PBS. To detect HA (haemagglutinin) tag, cells were <br /> GAATGCGGCCGCATTCCACACCAATAAAAGGAGAT-3                                incubated rat monoclonal anti-HA antibody (Roche) and <br /> the 3  -UTR. These amplified products used knitting PCR,       FITC-conjugated rabbit anti-rat immunoglobulins (DakoCyto- <br /> in 5  -UTR joined 3  -UTR short BamHI&#226;&#8364;&#8220;        mation). ER Golgi localization determined incubation <br /> HindIII linker region contained described primers             1 h room temperature (&#226;&#710;&#188; 20 &#226;&#8212;&#166;C) rabbit anti- <br /> (italics) NotI site (underlined) end. This PCR product     BiP (ER luminal chaperone binding protein) [23] rabbit <br /> was ligated pGEM-5Zf(+) (Promega) NotI sites            anti-TbGRASP (T. brucei Golgi matrix protein) antibodies [24] <br /> the hygromycin [HYG (hygromycin phosphotransferase)]                  respectively, prior incubation 1 h room temperature with <br /> puromycin [PAC (puromycin acetyltransferase)] resistance genes           TRITC (tetramethylrhodamine &#206;&#178;-isothiocyanate)-conjugated <br /> were ligated BamHI HindIII restriction sites.            anti-rabbit immunoglobulins (Sigma). <br /> Plasmid DNA prepared using QIAprep Miniprep Plasmid <br /> kit (Qiagen); digestion NotI, precipitated        Recombinant expression Escherichia coli <br /> sodium acetate/ethanol dissolved sterile water final <br /> concentration 1 &#194;&#181;g/&#194;&#181;l electroporation.                            TbPIS amplified genomic DNA using primers 5  - <br />                                                                          GGCGATATCGGATCCCATGCCGAAAGCTAAAACT-3  and <br />                                                                          5  -CCGCAAGCTTGGGCTGGCGGCTTCCCGCAGCAGCAT- <br /> Southern Northern blots                                              CCA-3  ligated pET32b (Novagen) sites BamHI <br /> The PIS ORF PCR-amplified using primers de-                 HindIII, underlined primer sequences. For <br /> scribed previous section ligation pLew vectors       recombinant expression, freshly transformed BL21 cells were <br /> gel-purified QIAquick gel extraction kit (Qiagen). This           used inoculate 100 ml M9 glucose minimal medium (M9 <br /> fragment labelled fluorescein (Gene Images-         salts, 0.2 % glucose, 1 mM MgSO4 0.001 % thiamine) sup- <br /> Random prime module; Amersham) Southern blotting                     plemented casamino acids (2 g/l) ampicillin (100 &#194;&#181;g/ <br /> or [&#206;&#177;-32 P]dCTP (RediprimeII random prime labelling system;              ml). Cells grown 37 &#226;&#8212;&#166;C absorbance (A) 600 nm <br /> Amersham) Northern blotting.                                         0.5 0.6, induced IPTG (isopropyl <br />    For Southern blots, genomic T. brucei DNA (2 &#194;&#181;g) digested         &#206;&#178;-D-thiogalactoside; 50 &#194;&#181;M) grown overnight 25 &#226;&#8212;&#166;C. The <br /> with various restriction enzymes, digestion products            cells collected centrifugation resuspended lysis <br /> separated 0.8 % agarose gel transferred Hybond-N        buffer [50 mM sodium phosphate, pH 7, 300 mM NaCl, 5 mM <br /> membrane (Amersham). The membrane hybridized overnight               2-mercaptoethanol, 5 % (v/v) glycerol, 0.1 % Triton X-100, <br /> in ULTRA-HYB (Ambion) 42 &#226;&#8212;&#166;C fluorescein-labelled             40 mM imidazole complete protease inhibitor cocktail with- <br /> PIS ORF probe. Stringency washes carried 42 &#226;&#8212;&#166;C,          EDTA (Roche)], disruption sonication. After the <br /> consisted washes low stringency 15 min (2 &#195;&#8212;           supernatant cleared centrifugation, prewashed Ni-NTA <br /> SSC 0.1 % SDS; 1 &#195;&#8212; SSC 0.15 M NaCl 0.015 M                    (Ni2+ -nitrilotriacetate) beads (Qiagen) added mix- <br /> sodium citrate) washes high stringency 15 min       ture incubated 4 &#226;&#8212;&#166;C 1 h. The beads collected, <br />  <br />   <br /> c 2006 Biochemical Society <br />                                                                                Phosphatidylinositol synthase Trypanosoma brucei            289 <br />  <br />  <br /> washed lysis buffer bound protein eluted lysis     resuspended medium final concentration 1 &#195;&#8212; <br /> buffer containing 250 mM imidazole, detected Western          107 cells/ml. Cells labelled 1 h 37 &#226;&#8212;&#166;C 50 &#194;&#181;Ci/ <br /> blotting using anti-His antibodies (Clontech).                       ml D-[2-3 H]inositol (30 Ci/mmol; Amersham) D-[2- <br />                                                                      3 <br />                                                                        H]mannose (14 Ci/mmol; Amersham). The cells collected <br /> myo -[3 H]Inositol labelling E. coli cultures                     centrifugation (800 g, 10 min) lipids extracted in <br />                                                                      chloroform/methanol/water (10:10:3) 1 h, supernatant <br /> In vivo labelling E. coli cells expressing TbPIS performed    removed pellet re-extracted chloroform/ <br /> as described previously [5]. Briefly, cells grown 10 ml   methanol (2:1) 1 h. The supernatants pooled dried <br /> LB (Luria&#226;&#8364;&#8220;Bertani) medium supplemented 0.1 mM myo-              stream nitrogen prior desalting using butanol/water <br /> inositol A600 0.8, 4 ml culture        partitioning. An aliquot lipid fraction taken and <br /> removed, washed LB medium resuspended 2 ml              total 3 H c.p.m. lipid fraction determined by <br /> LB medium supplemented 20 &#194;&#181;M myo-inositol 1.2 &#194;&#181;M            scintillation spectrometry using Beckman LS6000SE with <br /> [3 H]myo-inositol. Expression induced 0.5 mM IPTG           Formula 989 scintillation fluid (Packard Bioscience). Lipids were <br /> and cells grown 37 &#226;&#8212;&#166;C 3 h. The cells       separated HPTLC using silica 60 HPTLC plates chlo- <br /> were collected centrifugation, resuspended sonication buffer   roform/methanol/water (10:10:3) solvent. Radiolabelled <br /> (50 mM Tris/HCl, pH 8, 2 mM EDTA 0.2 mg/ml lyso-                 lipids detected fluorography &#226;&#710;&#8217; 70 &#226;&#8212;&#166;C, spraying <br /> zyme) disrupted sonication, cells debris        En3 HanceTM using Kodak XAR-5 film intens- <br /> were removed centrifugation (4500 g, 10 min). To extract          ifying screen. [3 H]PI quantified using AR-2000 imaging <br /> lipids, chloroform methanol added supernatant        scanner (Bioscan). <br /> to achieve final concentration 10:10:3 (by vol.) chloroform/       When labelling [35 S]methionine, 1 &#195;&#8212; 107 mid-exponential <br /> methanol/water incubated shaking room temperature        cells collected centrifugation, washed methionine-free <br /> for 1 h, samples nitrogen dried desalted        minimal essential media resuspended medium at <br /> butanol/water partitioning. Lipids separated HPTLC           final concentration 1 &#195;&#8212; 107 cells/ml. The cells labelled <br /> (high-performance TLC) using silica 60 plates chloroform/       30 min 20 &#194;&#181;Ci [35 S]methionine (1175 Ci/mmol; MP <br /> methanol/water (10:10:3) solvent. Radiolabelled lipids        Biomedicals) 37 &#226;&#8212;&#166;C. To quench labelling, cells were <br /> were detected fluorography &#226;&#710;&#8217; 70 &#226;&#8212;&#166;C spraying         diluted 20 ml cold TDB (trypanosome dilution buffer; <br /> En3 HanceTM (NEN) using Kodak XAR-5 film               25 mM KCl, 400 mM NaCl, 5 mM MgSO4 , 100 mM Na2 HPO4 , <br /> intensifying screen.                                                 NaH2 PO4 100 mM glucose) containing 1 mM L-methionine <br />                                                                      centrifuged (800 g, 10 min, 4 &#226;&#8212;&#166;C). After supernatant was <br /> E. coli membrane preparation PIS enzyme assay                    removed, cells resuspended TDB, equal volume of <br /> For E. coli membrane purification, BL21 cells freshly transformed    2&#195;&#8212; SDS/PAGE sample buffer (100 mM Tris, pH 6.8, 2 % SDS, <br /> with TbPIS-pET32b vector induced grown overnight        0.1 % Bromophenol Blue, 10 % glycerol 5 % 2-mercapto- <br /> as described above. After collection centrifugation, cells   ethanol) added heated 100 &#226;&#8212;&#166;C. Proteins separated <br /> resuspended PBS disrupted sonication. Cell debris,         SDS/10 % polyacrylamide gel visualized Coomassie <br /> whole cells ghosts removed slow-speed centrifug-         Blue staining. To detect 35 S-labelled proteins, destained gel was <br /> ation (14 500 g, 10 min); subsequently, membranes collected     soaked En3 HanceTM 30 min, washed water twice, <br /> by high-speed centrifugation (100 000 g, 1 h), resuspended        soaked 10 % glycerol dried. The dried gel exposed <br /> 50 mM Tris (pH 8), 5 mM EDTA 20 % glycerol stored            XAR-5 film overnight &#226;&#710;&#8217; 70 &#226;&#8212;&#166;C. <br /> as aliquots &#226;&#710;&#8217; 20 &#226;&#8212;&#166;C. <br />    Unless stated, reaction mixture PIS activity    Enzymatic digests chemical characterization radiolabelled <br /> consisted 50 mM Tris/HCl (pH 8), 0.15 mM CDP-DAG (Avanti          lipid species <br /> Polar Lipids), 0.3 % n-octyl glucopyranoside, 2 &#194;&#181;Ci [3 H]myo-     Digestion PI-PLC [PI-specific PLC (phospholipase C); <br /> inositol (Amersham; 14 Ci/mmol), 2.5 mM MnCl2 , 20 mM                Glyko], deamination, <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span> hydrolysis HF (hydrogen fluoride) <br /> MgCl2 50 &#194;&#181;g membrane protein, final volume            treatment described previously [25&#226;&#8364;&#8220;27]. <br /> 100 &#194;&#181;l. These reaction conditions used investigate sub- <br /> strate recognition various isomers analogues inositol <br /> at 25 &#194;&#181;M 1 mM. Reaction mixtures incubated 30 &#226;&#8212;&#166;C         Inositol analysis <br /> for 1 h terminated addition 666 &#194;&#181;l chloro-          Mid-exponential cells collected centrifugation (800 g, <br /> form/methanol (1:1, v/v). Lipid extractions separations     10 min), washed TDB stored &#226;&#710;&#8217; 20 &#226;&#8212;&#166;C. Lipids were <br /> performed described [3 H]myo-inositol labelling E. coli    extracted samples addition 500 &#194;&#181;l chloro- <br /> cells. [3 H]PI quantified using AR-2000 imaging scanner       form/methanol mixture (2:1) incubated room temperature <br /> (Bioscan).                                                           1 h. The supernatant removed pellet re- <br />    The reaction mixture used investigate exchange reaction    extracted chloroform/methanol/water mixture (1:2:0.8). The <br /> was based Klezovitch et al. [2] contained 50&#226;&#8364;&#8220;100 &#194;&#181;g   supernatants pooled dried nitrogen prior to <br /> of membrane protein, 0.5 &#194;&#181;M PI (soya bean; Sigma), 2 &#194;&#181;Ci          desalting biphasic partitioning using 2:1 (v/v) butanol/water. <br /> [3 H]myo-inositol, 25 mM MnCl2 , 50 mM MgCl2 100 mM              An internal standard D6 myo-inositol added samples prior <br /> Tris/HCl (pH 8) CMP (4 mM).                   hydrolysis strong acid (6 M HCl, 110 &#226;&#8212;&#166;C), derivitization with <br />                                                                      TMS [trimethylsilyl (Me3 Si)] analysis GLC-MS, the <br /> In vivo T. brucei metabolic labelling                                method Ferguson [28]. myo-Inositol quantified and <br />                                                                      means separate analyses determined. <br /> All results presented vivo metabolic labelling repres- <br /> entative independent experiments errors were <br /> within 5 %. For metabolic labelling, 2 &#195;&#8212; 107 mid-exponential cells   ESI-MS <br /> (1 &#195;&#8212; 106 cells/ml) centrifuged (800 g, 10 min), washed       Mid-exponential cells collected centrifugation (800 g, <br /> inositol-free minimal essential medium [15,25,26],      10 min), washed PBS resuspended PBS. Lipids <br />  <br />                                                                                                                           <br />                                                                                                                         c 2006 Biochemical Society <br />  290              K. L. Martin T. K. Smith <br />  <br />  <br />  <br />  <br /> Figure 1 Clustal W alignment predicted amino acid sequence TbPIS (AJ716153) S. cerevisiae (S. cere, GenBank&#194;&#174; accession <br /> no. J02697), To. gondii (AF069132) A. thaliana (AJ000539) <br /> Identical residues shown reverse type black background. Predicted transmembrane domains TbPIS underlined. <br />  <br />  <br /> were extracted method Bligh Dyer [29]. Qualitative                                  TMPRED form.html). The putative TbPIS contains per- <br /> analysis lipids performed ESI-MS (electrospray                                          fect copy motif [DG(X)2 AR(X)8 G(X)3 D(X)3 D] between <br /> ionization MS) ESI-MS/MS nanospray tips (Micromass                                       residues 55 78, described phos- <br /> type F) using Micromass Quattro Ultima triple quadrupole mass                                   pholipid-synthesizing enzymes identified date. The TbPIS has <br /> spectrometer argon [3.0 &#195;&#8212; 10&#226;&#710;&#8217;3 Torr (1 Torr = 133.3 Pa)]                                  submitted GenBank&#194;&#174; Nucleotide Sequence Database <br /> the collision gas. A capillary voltage 0.9 kV used                                accession number AJ716153. <br /> positive- negative-ion modes. Cone voltages 50 30 V <br /> were used positive- negative-ion modes respecttively. For <br /> analysis inositol-phospholipids, negative-ion parent ion scann-                                Recombinant expression <br /> ing m/z 241 (corresponding collision-induced fragment                                   To confirm enzymatic activity putative TbPIS, was <br /> inositol-1,2-cyclic phosphate) used collision energy                                  expressed E. coli using pET32b vector encodes N- <br /> 45 V.                                                                                             C-terminal His6 tags. The production His-tagged protein <br />                                                                                                   correct molecular mass (38 kDa) confirmed Western <br /> RESULTS AND DISCUSSION                                                                            blotting detection anti-His antibodies (Figure 2A). <br />                                                                                                      As E. coli possess endogenous PIS activity, re- <br /> Cloning TbPIS <br />                                                                                                   combinant TbPIS activity confirmed methods. Firstly, <br /> A putative PIS identified T. brucei genome database                                    vivo [3 H]myo-inositol labelling E. coli cells ex- <br /> using S. cerevisiae PIS query. This putative TbPIS                                     pressing TbPIS (Figure 2B) undertaken, [3 H]myo- <br /> ORF amplified sequence confirmed. The pre-                                        inositol lipid species present cell extracts E. <br /> dicted molecular mass deduced amino acid sequence                                       coli cells expressing TbPIS (Figure 2B, lane 1). This [3 H]myo- <br /> 24 kDa; alignment deduced amino acid sequence                                      inositol lipid species Rf (retention factor) identical with <br /> deduced PIS proteins organisms shown Figure 1.                                   PI; characterization showed sensitive PI- <br /> As PISs (e.g. 1, 5 8), TbPIS contains                                 PLC, base hydrolysis HF treatment (results shown). These <br /> putative transmembrane domains (underlined Figure 1) pre-                                   results consistent [3 H]myo-inositol species PI <br /> dicted TMHMM (http://www.cbs.dtu.dk/services/TMHMM-                                            clearly putative TbPIS catalytically active <br /> 2.0/)       Tmpred      (http://www.ch.embnet.org/software/                                   PIS. <br />  <br />   <br /> c 2006 Biochemical Society <br />                                                                                                            Phosphatidylinositol synthase Trypanosoma brucei            291 <br />  <br />  <br />                                                                                                   CDP-DAG (results shown). With assay conditions <br />                                                                                                   used present study, K m (app) myo-inositol was <br />                                                                                                   2 &#194;&#181;M (Figure 2C). This significantly lower than <br />                                                                                                   reported native PISs Chlamydomonas reinhardtii <br />                                                                                                   (0.2 mM) [30], turkey erythrocyte membranes (0.3 mM) [31], <br />                                                                                                   human placenta (0.28 mM) [32] yeast (0.1 mM) [1]. <br />                                                                                                      Like previously described PISs [2,6], recombinant TbPIS <br />                                                                                                   able catalyse head group exchange inositol pre- <br />                                                                                                   existing PI (soya bean) free [3 H]myo-inositol; activity <br />                                                                                                   detected absence CMP greatly enhanced <br />                                                                                                   (&#226;&#710;&#188; 5-fold) presence 4 &#194;&#181;M CMP (results shown). <br />                                                                                                      The specificity TbPIS investigated respect to <br />                                                                                                   myo-inositol; various inositol isomers analogues added <br />                                                                                                   standard assay mixture ability compete the <br />                                                                                                   [3 H]myo-inositol form [3 H]PI determined (Figure 2D). <br />                                                                                                   When 25 &#194;&#181;M (12.5 times K m ) myo-inositol added to <br />                                                                                                   standard reaction mixture, incorporation [3 H]myo- <br />                                                                                                   inositol [3 H]PI decreased approx. 15 % normal as <br />                                                                                                   expected, showing unlabelled myo-inositol competing <br />                                                                                                   [3 H]myo-inositol. However, isomers <br />                                                                                                   inositol [muco-, neo-, epi-, D(+)-chiro-, L(&#226;&#710;&#8217;)-chiro, cis-, <br />                                                                                                   allo- scyllo-] inositol analogues (see Supplementary <br />                                                                                                   Figure S1 http://www.BiochemJ.org/bj/396/bj3960287add.htm <br />                                                                                                   structures) able affect incorporation [3 H]myo- <br />                                                                                                   inositol [3 H]PI. At 1 mM, neo-, epi-, L(&#226;&#710;&#8217;)-chiro-, <br />                                                                                                   allo-, scyllo-, 1-deoxy-1-fluoro-scyllo-, 2-OMe-myo-, 1-deoxy-1- <br />                                                                                                   fluoro-myo-, 3-deoxy-3-fluoro-myo- D6 -myo-inositol caused <br />                                                                                                   reduction incorporation [3 H]myo-inositol [3 H]PI, <br />                                                                                                   suggesting high concentration able <br />                                                                                                   interact TbPIS. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> findings, propose <br />                                                                                                   orientation 1-, 3- 4-hydroxyls im- <br />                                                                                                   portant represents minimum recognition motif shown <br />                                                                                                   Supplementary Figure S1 http://www.BiochemJ.org/bj/ <br />                                                                                                   396/bj3960287add.htm. <br />  <br />  <br /> Figure 2 Expression TbPIS E. coli biochemical                                      The synthesis PI essential bloodstream form T. brucei <br /> characterization                                                                                  PIS appeared single copy gene haploid genome T. <br /> (A) TbPIS cloned expression vector pET32b expressed BL21 E. coli              brucei genome sequence database (Sanger Centre), which <br /> cells. Recombinant protein purified lysed cells presence detergent using       confirmed Southern blot analysis (see Supplementary Fig- <br /> Ni-NTA beads. Purified protein run SDS/10 % polyacrylamide gel, transferred             ure S2 http://www.BiochemJ.org/bj/396/bj3960287add.htm). <br /> on nylon membrane Western blotting detected using anti-His antibodies. Arrow            A schematic representation strategy employed con- <br /> indicates purified His-tagged TbPIS. (B) E. coli cells expressing TbPIS (lane 1) unrelated     struction conditional double knockout bloodstream form <br /> protein (lane 2) labelled vivo [3 H]myo -inositol, lipids extracted and <br /> separated HPTLC described text. [3 H]myo -Inositol-labelled lipids detected <br />                                                                                                   T. brucei shown Figure 3(A). <br /> by autoradiography; arrow indicates [3 H]myo -inositol-labelled PI. (C) Kinetics recombinant      Attempts create double null mutant TbPIS un- <br /> TbPIS myo -inositol. Enzyme activity measured described Experimental section;   successful, suggesting TbPIS essential gene (see Sup- <br /> CDP-DAG concentration held constant (0.15 mM) inositol concentration varied. Inset        plementary data). Therefore, allow creation conditional <br /> shows Lineweaver&#226;&#8364;&#8220;Burk plot data. (D) Ability TbPIS utilize inositol isomers     double knockout cell line, Myc-tagged ectopic copy TbPIS <br /> chemically modified inositol form PI. Compound 1, muco -; compound 2, neo -; compound 3,       integrated  PIS::PAC cell line pLew100 vec- <br /> epi -; compound 4, D(+)-chiro -; compound 5, L(&#226;&#710;&#8217;)-chiro ; compound 6, cis -; compound 7, <br /> allo -; compound 8, scyllo -; compound 9, myo -; compound 10, 2-OMe-chiro -; compound 11, <br />                                                                                                   tor [30]. Although integration ectopic copy success- <br /> 1-deoxy-1-fluoro-scyllo -; compound 12, 2-OMe-myo -; compound 13, 1-deoxy-1-fluoro-myo -;         ful, attempts delete second TbPIS allele un- <br /> compound 14, 3-deoxy-3-fluoro-myo -; compound 15, 3-deoxy-3-azido-myo -; compound 16,             successful (see Supplementary data, Figures S3 S4 http:// <br /> myo -inositol-2-phosphate; compound 17, DL-1,2-anhydro-myo -inositol; compound 18,                www.BiochemJ.org/bj/396/bj3960287add.htm). One possibility <br /> D6-myo -inositol; compound 19, myo -inositol-1-phosphate; compound 20, (see Supple-          postulated C-terminal Myc tag encoded by <br /> mentary Figure S1 http://www.BiochemJ.org/bj/396/bj3960287add.htm structures).             pLew100 vector interfering TbPIS localization <br /> White area represents percentage [3 H]PI formed compounds used 25 &#194;&#181;M, <br /> and black area represents percentage [3 H]PI formed compounds used 1 mM. <br />                                                                                                   and/or activity. Therefore Myc tag exchanged HA <br /> All determinations performed triplicate.                                                  tag, separate experiments appeared hinder the <br />                                                                                                   transcription/translation TbPIS. <br />                                                                                                      The HA-tagged ectopic copy electroporated the <br />   Secondly, cell-free using membranes isolated E.                                    PIS::PAC cell line using modified pLew100-HA vector and <br /> coli expressing TbPIS used subsequent biochemical                                     clones selected puromycin phleomycin. Several <br /> characterization recombinant protein. Under conditions                                  PIS::PAC PIS-HATi clones (where &#226;&#8364;&#732;Ti&#226;&#8364;&#8482; tetracycline-inducible) <br /> used, PIS activity linear 2 h (results                                 obtained integration ectopic copy initially <br /> shown), optimal pH range 7&#226;&#8364;&#8220;8 (results shown). The                                  confirmed PCR using primers specific pLew100 vector <br /> TbPIS catalytic activity dependent presence Mg2+                                    (results shown) later Southern blotting (Figure 3B, <br />  <br />                                                                                                                                                       <br />                                                                                                                                                     c 2006 Biochemical Society <br />  292                K. L. Martin T. K. Smith <br />  <br />  <br />  <br />  <br /> Figure 3      Construction PIS conditional double knockout cell line <br /> (A) Schematic representation construction strategy used. (i) One allele PIS replaced by <br /> puromycin resistance gene (PAC) homologous recombination, generating  PIS::PAC cell <br /> line, (ii) tetracycline-inducible ectopic copy PIS introduced rDNA, generating <br /> PIS-HA Ti  PIS::PAC cell line, (iii) tetracycline induces expression ectopic copy, <br /> the remaining allele replaced hygromycin resistance gene homologous recom- <br /> bination, resulting conditional double knockout cell line PIS-HA Ti  PIS::PAC/ PIS::HYG . <br /> (B) Confirmation genotype T. brucei PIS conditional double knockout cell line. Southern-blot <br /> analysis HinfI-digested genomic DNA (&#226;&#710;&#188; 2 &#194;&#181;g) wild-type T. brucei cells (lane 1), <br />  PIS::PAC (lane 2), PIS-HA Ti  PIS::PAC (lane 3) PIS-HA Ti  PIS::PAC/ PIS::HYG <br /> (lane 4) cells; PIS ORF probe shows allelic TbPIS 1 kb ectopic copy TbPIS-HA Ti <br /> at approx. 1.2 kb. <br />  <br />                                                                                                       Figure 4 Growth curves Northern blot TbPIS conditional double <br /> lane 3). Transcription ectopic copy induced ad-                                        knockout cell line <br /> dition tetracycline medium prior deletion the <br />                                                                                                       Cells washed tetracycline-free HMI-9, transferred tetracycline-free medium <br /> second allele TbPIS. Deletion endogenous alleles                                       counted daily; experiments performed independent occasions, set of <br /> confirmed Southern blotting, resulting cell line  PIS::                                     data shown representative experiments. Growth curves shown wild-type <br /> PAC/PIS::HYG/PIS-HATi (Figure 3B, lane 4).                                                            cells medium (A), TbPIS conditional double knockout added tetracycline (B), and <br />    The conditional double knockout  PIS::PAC/PIS::HYG/PIS-                                            TbPIS conditional double knockout tetracycline (C). (D) The Figure shows Northern <br /> HATi cell line used investigate dependence blood-                                       blot total RNA procyclic cells (lane 1), wild-type cells (lane 2), TbPIS conditional <br />                                                                                                       double knockout tetracycline day 2 (lane 3), TbPIS conditional double knockout without <br /> stream form cells synthesis PI. After washed <br />                                                                                                       tetracycline day 2 (lane 4) TbPIS conditional double knockout tetracycline on <br /> three times tetracycline-free medium, cells incubated                                         day 14 (lane 5). The blot probed TbPIS ORF &#206;&#178;-tubulin, detailed the <br /> in medium absence presence tetra-                                        Experimental section. <br /> cycline. The initial concentration cells 5 &#195;&#8212; 104 ml&#226;&#710;&#8217;1 and <br /> the cells diluted concentration                                        tetracycline 2 days, detectable transcript the <br /> 1 3 &#195;&#8212; 106 ml&#226;&#710;&#8217;1 , normally second day. The                                         TbPIS ectopic copy. However, cells spontaneously <br /> conditional double knockout cells displayed normal growth                                             resumed growth absence tetracycline, clear <br /> rates presence tetracycline compared wild-                                        TbPIS transcript (Figure 4D, lane 5). This suggests these <br /> type cells (Figures 4A 4B). In absence tetracycline,                                       revertent cells able overcome tetracycline control, <br /> the cells grew normally 2 days; however,                                     enabling cells resume expression TbPIS. This spon- <br /> time, cell numbers decreased distinct morphological change                                      taneous recovery T. brucei conditional double knockout cell <br /> was observed (see Supplementary Figure S5 http://www.                                              lines grown absence tetracycline described for <br /> BiochemJ.org/bj/396/bj3960287add.htm); concurrently increas-                                          essential genes [13,21,22] loss tetracycline <br /> ing levels cell debris observed, suggesting cell death                                        control, case shown to deletion of <br /> (Figure 4C). However, 9&#226;&#8364;&#8220;10 days absence tetra-                                        tetracycline repressor gene [22]. <br /> cycline, live cells observed, suggesting small propor- <br /> tion dying cells able spontaneously resume normal <br />                                                                                                       Biochemical <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> TbPIS conditional knockout cells <br /> growth rates. To determine adaptation <br /> (e.g. uptake sufficient PI extracellular environment)                                     The biochemical phenotype TbPIS conditional knockout <br /> or re-expression TbPIS ectopic copy, transcription                                      cells investigated quantitative qualitative analyses <br /> levels TbPIS investigated. After culturing absence                                  inositol-phospholipids GLC-MS MS respectively and <br />  <br />   <br /> c 2006 Biochemical Society <br />                                                                                                                   Phosphatidylinositol synthase Trypanosoma brucei                               293 <br />  <br />  <br />  <br />  <br />                                                                                                      Figure 6 Biochemical phenotype analyses TbPIS conditional <br />                                                                                                      knockout cells <br />                                                                                                      Prior labelling, cells grown tetracycline-free medium 2 days; tetracycline <br /> Figure 5      PI analyses TbPIS conditional double knockout cells                                 necessary, added daily 1 &#194;&#181;g/ml. (A) Cells labelled [3 H]myo -inositol, <br />                                                                                                      lipids extracted detected detailed Experimental section. Lane 1, wild-type cells; <br /> (A) Lipid myo -inositol levels quantified GLC-MS detailed text                lane 2, TbPIS conditional double knockout cells grown presence tetracycline; lane 3, <br /> wild-type cells (cell line 1), TbPIS conditional double knockout cells grown presence      TbPIS conditional double knockout cells grown absence tetracycline. (B) Cells were <br /> tetracycline (cell line 2) TbPIS conditional double knockout cells grown absence          labelled [35 S]methionine, protein samples prepared, separated detected by <br /> of tetracycline (cell line 3); 100 % represents 55 pmol myo -inositol/5 &#195;&#8212; 104 cell equivalents.   using autoradiography (lanes 1&#226;&#8364;&#8220;4) Coomassie Blue staining (lanes 5&#226;&#8364;&#8220;8). Lanes 1 5, <br /> Inositol-phospholipids qualitatively analysed ESI-MS/MS using parent-ion scanning            wild-type cells pre-incubated 5 min 60 &#194;&#181;g/ml cycloheximide prior labelling; lanes 2 <br /> of m /z 241 (the collision-induced fragment PI, inositol-1,2-cyclic phosphate) negative ion    6, wild-type cells; lanes 3 7, TbPIS conditional double knockout cells grown the <br /> mode (B) wild-type cells (C) TbPIS conditional double knockout cells grown         presence tetracycline; lanes 4 8, TbPIS conditional double knockout cells grown in <br /> the absence tetracycline.                                                                         absence tetracycline. (C) Cells labelled [3 H]mannose, lipids extracted <br />                                                                                                      detected detailed Experimental section. Lane 1, wild-type cells; lane 2, TbPIS <br />                                                                                                      conditional double knockout cells grown absence tetracycline. <br /> in vivo labelling. For analyses, conditional double <br /> knockout cells grown absence presence tetra-                                       change ratios peaks conditional double knockout <br /> cycline 2 days results compared                                     cells grown non-permissive conditions 2 days were <br /> wild-type cells grown conditions.                                                     compared wild-type cells (compare Figures 5B 5C). <br />    Total phospholipids extracted wild-type cells                                       Also, apparent change phospholipids as <br /> conditional double knockouts grown presence                                         determined ESI-MS (results shown). These results coupled <br /> absence tetracycline 2 days myo-inositol                                    GLC-MS analysis overall decrease <br /> was determined GLC-MS. As shown Figure 5(A),                                         inositol-containing phospholipids, ap- <br /> a slight decrease lipid containing myo-inositol                                     peared affect species similar extent. This reduction in <br /> in conditional double knockouts grown presence                                         PI detrimental effect cell signalling and/or GPI <br /> tetracycline compared wild-type cells. This                                     anchor biosynthesis. <br /> due ectopic copy PIS enzymatically                                        In vivo labelling [3 H]myo-inositol suggested total <br /> efficient HA tag regulated                                        radioactivity incorporated lipid fraction of <br /> same level endogenous PIS. When conditional double                                        conditional double knockout cells grown presence or <br /> knockout cells grown absence tetracycline                                         absence tetracycline reduced approx. 90 27 % <br /> 2 days, significant decrease lipid-                                     respectively compared wild-type cells. Analysis the <br /> bound myo-inositol approx. 30 % present wild-type                                      [3 H]lipids HPTLC revealed [3 H]lipid, an <br /> cells (Figure 5A).                                                                                   identical Rf PI (Figure 6A, lanes 1&#226;&#8364;&#8220;3). These results, together <br />    To ascertain global decrease PI species                                  GLC-MS myo-inositol analysis, that <br /> the conditional double knockout cells grown non-permiss-                                       conditional double knockout cells grown absence <br /> ive conditions, specific species decreased, inositol-                                 tetracycline, PI synthesis substantially decreased, thus <br /> phospholipids qualitatively analysed ESI-MS/MS using                                         confirming deleted gene functional PIS bloodstream <br /> parent-ion scanning m/z 241 negative ion mode, indicative                                  form T. brucei. However, residual PIS activity remained <br /> collision-induced fragment PI species (inositol-1,2-cyclic                                    cells account [3 H]PI, albeit reduced <br /> phosphate). Two peaks identified C18:0 /C18:2 C18:0 /C22:4 ;                             level. As detectable TbPIS mRNA detectable the <br /> both previously identified major forms DAG PI                                        removal tetracycline 2 days (Figure 4D, lane 4), take <br /> from bloodstream form T. brucei [33,34]. There significant                                    time total cellular PIS loss protein turnover <br />  <br />                                                                                                                                                                            <br />                                                                                                                                                                          c 2006 Biochemical Society <br />  294               K. L. Martin T. K. Smith <br />  <br />  <br />  <br />  <br /> Figure 7     Subcellular localization PIS&#226;&#8364;&#8220;HATi bloodstream form T. brucei cells <br /> Cells expressing PIS&#226;&#8364;&#8220;HATi co-stained HA epitope cellular markers. Co-staining antibodies Golgi protein TbGRASP shown (A&#226;&#8364;&#8220;E). (A) Phase contrast image; <br /> (B) DAPI staining; (C) HA epitope staining FITC detection; (D) TbGRASP staining TRITC detection; (E) merged image HA detection TbGRASP detection. Co-staining with <br /> antibodies ER protein BiP shown (F&#226;&#8364;&#8220;J). (F) Phase contrast image; (G) DAPI staining; (H) HA epitope staining FITC detection; (I) BiP staining TRITC detection; (J) merged <br /> image HA detection BiP detection. <br />  <br />  <br /> and dilution cell division, explaining cells                                     biosynthesis slowed greater 90 %, parasites die <br /> are alive point.                                                                         [13&#226;&#8364;&#8220;15]. <br />    To confirm cells viable point labell- <br /> ing, ability synthesize protein investigated parallel <br /> with myo-inositol vivo labelling. The wild-type cells                                     TbPIS localized ER Golgi <br /> able incorporate significant [35 S]methionine                                     To investigate subcellular location PIS T. brucei blood- <br /> newly synthesized protein (Figure 6B, lane 2) compared                                        stream form cells, tetracycline-inducible ectopic copy intro- <br /> those pretreated cycloheximide, known protein synthesis                                        duced rRNA locus using expression vector pLew82, <br /> inhibitor (Figure 6B, lane 1). The conditional knockout cells                                         encodes C-terminal HA tag. Integration ectopic <br /> grown presence absence tetracycline 2 days showed                                    copy confirmed PCR, Southern Northern blotting <br /> similar amounts [35 S]methionine incorporation compared                                         (results shown). Primarily cell line (TbPIS-HATi ) was <br /> with wild-type cells (Figure 6B, compare lanes 2&#226;&#8364;&#8220;4), indicating                                       used immunofluorescence results presented here; <br /> that point vivo labelling, cells able syn-                                   however, conditional double knockout cell line  PIS:: <br /> thesize protein. Therefore vivo labelling clearly shows                                       PAC/PIS::HYG/PIS-HATi created, immunofluorescent detec- <br /> when PIS conditional double knockout cells grown                                        tion HA tag repeated identical with <br /> non-permissive conditions 2 days, ability synthesize                                     observed TbPIS-HATi cell line. Transcription the <br /> PI decreased direct result deletion PIS                                    TbPIS-HATi ectopic copy induced addition tetra- <br /> because cells lost viability.                                                                cycline medium. The TbPIS&#226;&#8364;&#8220;HATi protein detected <br />    To investigate knock-on effects decrease PI                                          immunofluorescence using primary antibody HA <br /> synthesis, status GPI biosynthesis assessed. The con-                                      tag secondary antibody FITC-conjugated. Two <br /> ditional double knockout cells grown absence presence                                       concomitant signals observed (Figures 7C 7H); was <br /> of tetracycline 2 days labelled [3 H]myristate                                      distinct signal positioned nucleus kinetoplast, <br /> [3 H]mannose compared labelled wild-type cells. When                                         suggesting Golgi location. The second reticular cyto- <br /> labelled [3 H]myristate, cell lines (wild-type,                                        plasmic signal perinuclear staining indicative ER. <br /> conditional double knockout with/without tetracycline) showed                                         When cells co-stained rat antibodies the <br /> similar amounts 3 H-labelled lipid species (results                                            HA epitope rabbit antibodies TbGRASP, known <br /> shown), suggesting deletion PIS detri-                                         Golgi protein [24], observed complete overlap of <br /> mental effects general lipid biosynthesis time point.                                      signals, suggesting TbPIS Golgi <br /> In vivo [3 H]mannose labelling wild-type cells showed                                      (Figure 7E). We co-stained cells rat antibodies <br /> expected mature GPI glycolipids A C (Figure 6C)                                        HA epitope rabbit antibodies T. brucei <br /> been described previously [35,36]. When conditional double                                        ER-luminal protein BiP [23]. Both proteins displayed reticular <br /> knockout cells grown non-permissive conditions com-                                        staining cytoplasm, perinuclear stain- <br /> pared wild-type cells, significant decrease                                   ing. There significant co-localization TbPIS and <br /> the labelled glycolipids A C (Figure 6C). This                                          BiP, suggesting TbPIS ER (Figure 7J). <br /> shows decrease PI synthesis deletion PIS                                    Localization TbPIS ER totally unexpected; to <br /> has direct knock-on effect GPI biosynthetic pathway.                                     date PISs organisms predominately, some <br /> In vivo labelling [3 H]myo-inositol showed PIS                                     cases solely, ER (e.g. [4,31,32]). A secondary <br /> activity decreased, residual activity remaining                                    location TbPIS Golgi consistent localization of <br /> (Figure 6A). This remaining PIS activity continue                                            yeast PIS, ER Golgi [1,9]. <br /> decrease PI synthesis ceases reaches critical point where <br /> there insufficient PI available GPI synthesis. As result, <br /> newly synthesized GPI-anchored VSG decrease,                                             Conclusions <br /> situation previously suggested lethal                                      The identity putative TbPIS identified T. brucei genome <br /> parasite [13&#226;&#8364;&#8220;15]. This similar chemical genetic                                         confirmed preliminary characterization recombinant <br /> validation T. brucei GPI pathway, soon GPI                                         protein expressed E. coli. The TbPIS ER and <br />  <br />   <br /> c 2006 Biochemical Society <br />                                                                                                                   Phosphatidylinositol synthase Trypanosoma brucei                            295 <br />  <br />  <br /> Golgi bloodstream form parasite. Through cre-                                       14 Nagamune, K., Nozaki, T., Maeda, Y., Ohishi, K., Fukuma, T., Hara, T., Schwarz, R. and <br /> ation conditional double knockout, TbPIS gene shown                                         Schneider, P. (2000) Critical roles glycosylphosphatidylinositol Trypanosoma <br /> to essential gene bloodstream form T. brucei, gen-                                         brucei . Proc. Natl. Acad. Sci. U.S.A. 97, 10336&#226;&#8364;&#8220;10341 <br /> etically validating particular enzyme potential drug target                                 15 Smith, T. K., Crossman, A., Brimacombe, J. S. Ferguson, M. A. J. (2004) Chemical <br />                                                                                                          validation GPI biosynthesis drug target African sleeping sickness. EMBO J. <br /> against African sleeping sickness potentially diseases <br />                                                                                                          23, 4701&#226;&#8364;&#8220;4708 <br /> caused protozoa. Quantitative qualitative analyses                                     16 Ferguson, M. A. J., Brimacombe, J. S., Brown, J. R., Crossman, A., Dix, A., Field, R. A., <br /> as vivo metabolic labelling conditional double knockout                                        G&#194;&#168;uther, M. L. S., Milne, K. G., Sharma, D. K. Smith, T. K. (1999) The GPI biosynthetic <br /> cells confirmed significant decrease                                         pathway therapeutic target African sleeping sickness. Biochim. Biophys. Acta <br /> major PI species cell. This decrease detrimental                                            1455, 327&#226;&#8364;&#8220;340 <br /> effect GPI biosynthesis, ultimately lethal                                       17 Elabbadi, N., Ancelin, M. Vial, H. (1994) Characterisation phosphatidylinositol <br /> the parasite.                                                                                            synthase evidence polyphosphoinositide cycle Plasmodium -infected <br />    Genetic validation PI synthesis anti-trypanosomal                                            erythrocytes. Mol. Biochem. Parasitol. 63, 179&#226;&#8364;&#8220;192 <br /> target demonstrates bloodstream form parasite                                         18 Daniels, C. J. Palmer, F. B. (1980) Biosynthesis phosphatidylinositol Crithidia <br />                                                                                                          fasciculate . Biochim. Biophys. Acta 618, 263&#226;&#8364;&#8220;272 <br /> cannot compensate loss novo synthesis scavenging <br />                                                                                                       19 Subramanian, A., Navarro, S., Carrasco, R. A., Marti, M. Das, S. (2000) Role of <br /> from environment vitro. These results suggest                                           exogenous inositol phosphatidlyinositol glycosylphosphatidylinositol anchor <br /> biosynthetic steps upstream PI synthesis potential                                        synthesis GP49 Giardia lamblia . Biochim. Biophys. Acta 1483, 69&#226;&#8364;&#8220;80 <br /> drug targets, hypothesis currently investigating.                                            20 Wirtz, E., Leal, S., Ochatt, C. Cross, G. A. M. (1999) A tightly regulated inducible <br />                                                                                                          expression conditional gene knock-outs dominant-negative genetics in <br /> This work supported Wellcome Trust Senior Research Fellowship 067441. We                        Trypanosoma brucei . Mol. Biochem. Parasitol. 99, 89&#226;&#8364;&#8220;101 <br /> thank Graham Warren (Department Cell Biology, Ludwig Institute Cancer Research,                21 Milne, K. G., G&#194;&#168;uther, M. L. S. Ferguson, M. A. J. (2001) Acyl-CoA binding protein is <br /> Yale University School Medicine, New Haven, CT, U.S.A.) TbGRASP antibody                      essential bloodstream form. T brucei . Mol. Biochem. Parasitol. 112, 301&#226;&#8364;&#8220;304 <br /> and Jay Bangs (Department Medical Microbiology Immunology, University                       22 Roper, J. R., G&#194;&#168;uther, M. L., Milne, K. G. Ferguson, M. A. J. (2002) Galactose <br /> Wisconsin-Madison Medical School, Madison, WI, U.S.A.) BiP antibody.                             metabolism essential African sleeping sickness parasite Trypanosoma brucei . <br />                                                                                                          Proc. Natl. Acad. Sci. U.S.A. 99, 5884&#226;&#8364;&#8220;5889 <br />                                                                                                       23 Bangs, J. D., Uyetake, L., Brickman, M. J., Balber, A. E. Boothroyd, J. C. (1993) <br /> REFERENCES <br />                                                                                                          Molecular cloning cellular localization BiP homologue Trypanosoma brucei. <br />  1 Fischl, A. S. Carman, G. M. (1983) Phosphatidylinositol biosynthesis                           Divergent ER retention signals lower eukaryote. J. Cell Sci. 105, 1101&#226;&#8364;&#8220;1113 <br />    Saccharomyces cerevisiae : purification properties microsome-associated                     24 He, C. Y., Ho, H. H., Malsam, J., Chalouni, C., West, C. M., Ullu, E., Toomre, D. and <br />    phosphatidylinositol synthase. J. Bacteriol. 154, 304&#226;&#8364;&#8220;311                                             Warren, G. (2004) Golgi dulplication Trypanosoma brucei . J. Cell Biol. 165, 313&#226;&#8364;&#8220;321 <br />  2 Klezovitch, O., Brandenburger, Y., Geindre, M. Deshusses, J. (1993) Characterization           25 G&#194;&#168;uther, M. L. S. Ferguson, M. A. J. (1995) The role inositol acylation inositol <br />    reactions catalysed yeast phosphatidylinositol synthase. FEBS Lett. 320, 256&#226;&#8364;&#8220;260                deacylation GPI biosynthesis Trypanosoma brucei . EMBO J. 14, 3080&#226;&#8364;&#8220;3093 <br />  3 Nikawa, J., Kodaki, T. Yamashita, S. (1988) Expression Saccharomyces                    26 G&#194;&#168;uther, M. L. S., Masterson, W. J. Ferguson, M. A. J. (1994) The effects of <br />    cerevisiae PIS gene synthesis phosphatidylinositol Escherichia coli .                       phenylmethylsulfonyl fluoride inositol-acylation fatty acid remodelling African <br />    J. Bacteriol. 170, 4727&#226;&#8364;&#8220;4731                                                                          trypanosomes. J. Biol. Chem. 269, 18694&#226;&#8364;&#8220;18701 <br />  4 Nikawa, J. Yamashita, S. (1997) Phosphatidylinositol synthase yeast.                      27 Smith, T. K., Cottaz, S., Brimacombe, J. S. Ferguson, M. A. J. (1996) Substrate <br />    Biochim. Biophys. Acta 1348, 173&#226;&#8364;&#8220;178                                                                  specificities dolicol phosphate mannose: glucosaminyl phosphatidylinositol <br />  5 Collin, S., Justin, A., Cantrel, C., Arondel, V. Kader, J. (1999) Identification AtPIS,      &#206;&#177;1-4-mannosyltransferase glycosylphosphatidylinositol biosynthetic pathway of <br />    phosphatidylinositol synthase Arabidopsis . Eur. J. Biochem. 262, 652&#226;&#8364;&#8220;658                        African trypanosomes. J. Biol. Chem. 271, 6476&#226;&#8364;&#8220;6482 <br />  6 Justin, A., Kader, J. Collin, S. (2002) Phosphatidylinositol synthesis exchange         28 Ferguson, M. A. J. (1993) GPI membrane anchors: isolation analysis. In <br />    inositol head catalysed single phosphatidylinositol synthase 1                    Glycobiology: A Practical Approach (Fukuda, M. Kobata, A., eds.), pp. 349&#226;&#8364;&#8220;383, <br />    Arabidopsis . Eur. J. Biochem. 269, 2347&#226;&#8364;&#8220;2352                                                         IRL Press, Oxford, U.K. <br />  7 Justin, A., Kader, J. Collin, S. (2003) Synthetic capacity Arabidopsis                      29 Bligh, E. G. Dyer, W. J. (1959) A rapid method total lipid extraction and <br />    phosphatidylinositol synthase 1 expressed Escherichia coli . Biochim. Biophys. Acta                purification. Can. J. Biochem. Physiol. 37, 911&#226;&#8364;&#8220;917 <br />    1634, 52&#226;&#8364;&#8220;60                                                                                        30 Blouin, A., Lavezzi, T. Moore, T. S. (2003) Membrane lipid biosynthesis in <br />  8 Seron, K., Dzierszinski, F. Tomavo, S. (2000) Molecular cloning, functional                       Chlamydomonas reinhardtii . Partial characterization CDP-diacylglycerol:myo -inositol <br />    complementation Saccharomyces cerevisiae enzymatic properties                               3-phophatidyltransferase. Plant Physiol. Biochem. 41, 11&#226;&#8364;&#8220;16 <br />    phosphatidylinositol synthase protozoan parasite Toxoplasma gondii .                      31 McPhee, F., Lowe, G., Vaziri, C. Downes, C. P. (1991) Phosphatidylinositol synthase <br />    Eur. J. Biochem. 267, 6571&#226;&#8364;&#8220;6579                                                                       phosphatidylinositol/inositol exchange reactions turkey erythrocyte membranes. <br />  9 Leber, A., Hrastnik, C. Daum, G. (1995) Phospholipid-synthesizing enzymes Golgi                Biochem. J. 275, 187&#226;&#8364;&#8220;192 <br />    membranes yeast, Saccharomyces cerevisiae . FEBS Lett. 18, 271&#226;&#8364;&#8220;274                          32 Antonsson, B. (1994) Purification characterization phosphatidylinositol synthase <br /> 10 Imai, A. Gershengorn, M. C. (1987) Independent phosphatidylinositol synthase                   human placenta. Biochem. J. 297, 517&#226;&#8364;&#8220;522 <br />    pituitary plasma membrane endoplasmic reticulum. Nature (London) 325,                          33 Patnaik, P. K., Field, M. C., Menon, A. K., Cross, G. A., Yee, M. C. Butikofer, P. (1993) <br />    726&#226;&#8364;&#8220;728                                                                                               Molecular species analysis phospholipids Trypanosoma brucei bloodstream and <br /> 11 Cross, G. A. M. (1996) Antigenic variation trypanosomes: secrets surface slowly.                   procyclic forms. Mol. Biochem. Parasitol. 58, 97&#226;&#8364;&#8220;105 <br />    BioEssays 18, 283&#226;&#8364;&#8220;287                                                                              34 Doering, T. L., Pessin, M. S., Hart, G. W., Raben, D. M. Englund, P. T. (1994) The fatty <br /> 12 Ferguson, M. A. J. (1999) The structure, biosynthesis functions                                acids unremodelled trypanosome glycosyl-phosphatidylinositols. Biochem. J. 299, <br />    glycosylphosphatidylinositol anchors, contributions trypanosome research.                  741&#226;&#8364;&#8220;746 <br />    J. Cell Sci. 112, 2799&#226;&#8364;&#8220;2809                                                                        35 Menon, A. K., Mayor, S., Ferguson, M. A., Duszenko, M. Cross, G. A. (1998) <br /> 13 Chang, T., Milne, K. G., G&#194;&#168;uther, M. L. S., Smith, T. K. Ferguson, M. A. J. (2002)                Candidate glycophospholipid precursor glycosylphosphatidylinositol membrane <br />    Cloning Trypansoma brucei Leishmania major genes encoding                              anchor Trypanosoma brucei variant surface glycoproteins. J. Biol. Chem. 263, <br />    GlcNAc-phoshatidylinositol de-N-acetylase glycosylphosphatidylinositol biosynthesis                1970&#226;&#8364;&#8220;1977 <br />    essential African sleeping sickness parasite. J. Biol. Chem. 277,                   36 Mayor, S., Menon, A. K. Cross, G. A. M. (1992) Galactose containing <br />    50176&#226;&#8364;&#8220;50182                                                                                           glycosylphosphatidylinositols Trypanosoma brucei . J. Biol. Chem. 267, 754&#226;&#8364;&#8220;761 <br />  <br />  <br /> Received 14 November 2005/3 February 2006; accepted 14 February 2006 <br /> Published BJ Immediate Publication 14 February 2006, doi:10.1042/BJ20051825 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           <br />                                                      <br /> </body></html>