<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1997 Genetics Society America <br />  <br />  <br />  <br />               Beaded Goldschmidt, Antimorphic Allele Serrate, Encodes a <br />                  Protein Lacking Transmembrane Intracellular Domains <br />  <br />                                                Neil A. Hukriede RobertJ. Fleming <br />                                 Department Biology, The University Rochester, Rochester, New York 14627 <br />                                                         Manuscript received August 12, 1996 <br />                                                       Accepted publication October 30, 1996 <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  Serrate (Ser) essential gene inDrosophila melanogaster best known theSer dominant ( S # ) allele <br />               effects wing development. Animals heterozygous homozygous S# viable exhibit <br />               loss wing margin tissue associated bristles hairs. The Beaded Goldschmidt (Bd&#226;&#8364;?) allele Ser, <br />               heterozygous wild type, produce animals exhibiting loss wing margin material. <br />               However, animals homozygous Bdc exhibit larval lethal <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>phenotype</span> comparableto animals homozy- <br />               gous loss-of-function Ser alleles. B&amp; partial duplication Ser locus single 5&#226;&#8364;&#732; Ser- <br />               homologous region distinct 3&#226;&#8364;&#732; regions. Meiotic recombination Bdc wild-type Ser <br />               chromosome demonstrated DNA lesion, caused insertion transposable roo <br />               element coding regions Ser transcript, appears capable generating Bd&#226;&#8364;? phenotypes. <br />               Due roo insertion, protein product predicted prematurely truncated lack an <br />               extracellular cysteine-rich region transmembrane andintracellular domains within <br />               normal SERRATE (SER) protein. The loss protein domains apparently contributes the <br />               antimorphic nature <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. <br />  <br />  <br />  <br /> T     HE specification cellfates multicellular organ- <br />        isms requires sequential subdivision cell <br /> populations restricted subsets known com- <br />                                                                    embryonic patterning genes ( A K A M 1987; INGHAM <br />                                                                    1988; ST.JOHNSTON NUSSLEIN-VOLHARD <br />                                                                    express proneural genes, competent <br />                                                                                                                               1992) and <br />  <br /> partments (GARCIA-BELLIDO            1975). Within com-       assume neuronal fates (SKEATH             CARROLL        1992). <br /> partment, localized activation transcription fac-           Interactions NOTCH receptor mem- <br /> tors &#226;&#8364;&#339;selector genes&#226;&#8364;? assign compartment-specific               brane-bound ligand DELTA (DL) adjacent cells <br /> identities cellswithin thegroup (GARCIA-BELLIDO cluster lead selection single <br /> 1975; reviewed LAWRENCE MORATA1994; LAW-                    neuroblasts equipotential cells the <br /> RENCE STRUHL1996). Subsequently, short range                   clusters    (reviewed ARTAVANIS-TSAKONAS              et al. 1995). <br /> intercellular communication                    The    Ser gene    identified    member       Notch <br /> compartment boundaries results specification             signaling     pathway      genetic   interactions      has <br /> individual cell fates (MEINHARDT1983; DIAZ-BENJUMEA                      encode     product     overall   structural <br /> and COHEN1993;TABATAand KORNBERG1994). For                         similarity DL (FLEMING            et al 1990;THOMAS et al. <br /> many morphogenetic cell-fate decisions, intercellu-            1991).    Both   SER     DL    transmembrane proteins, <br /> lar communication mediated                    extracellular   epidermal      growth factor (EGF)-like <br /> the L.IN-l2/Notch family receptors inverte-             repeats      share    N-terminal    DSL domain (named <br /> brate vertebrate developmental processes (GREEN-                 Delta,   Serrate   LAG2;      TAX   et al. 1994) is <br /> WALD RUBIN1992; COFFMAN                et al. 1993; KOPANet    conserved         ligands     similar    Notch-like recep- <br /> al. 1994; ARTAVANIS-TSAKONAS            et al. 1995). Although     tors    Caenorhabditis   elegans   (MELLO     et al. 1994; TAXet <br /> Notch activity required processes Drosophila,            al.  1994).    Furthermore,         SER       DL the <br /> including mesodermal differentiation, oocyte develop-              ability    cause  aggregation         nonadherent       Dro- <br /> ment processes leading cell fate specification           sophila    S2  cells    expressed      presence <br /> bristle patterning, adult eye wings (CAGAN             S2 cells expressing NOTCH (FEHON               et al. 1990;REBAY <br /> READY 1989; CORBIN         et al. 1991; RUOHOLA      et al. 1991;  et  al. 1991).    These   data   suggest   strongly    So, like <br /> XU et al. 1992; RULIFSON        BLAIR 1995), Notch best     Dl,  encodes     NOTCH       ligand. <br /> characterized role Drosophila neurogenesis.                The convincing evidence Ser encodes li- <br /> In process, clusters cells embryonic            gand    NOTCH SER partially re- <br /> neuroectoderm     obtain compartmental              information    place    DL embryonic neurogenesis process <br />                                                                    (Gu et al. 1995). Itmust stressed, however, SER <br />                                                                    normally expressed Drosophila neurogen- <br />    Corresponding author: Robert J. Fleming, Department Biology, <br /> Hutchison Hall, The University Rochester, River Campus, Roches- esis essentially complete (FLEMING          et al. 1990;THOMAS <br /> ter, NY 14627. E-mail: fleming@landru.biology.rochester.edu        et al. 1991). Thus, similar signals sent <br /> Genetics 145 359-374 (February, 1997) <br />  360                                         N. A. Hukriede R. J. Fleming <br />  <br /> bySER expressed ectopically developing              able element B104 (SCHERER   et ul. 1982). Our analyses <br /> nervous system, native environment, possible       indicate resulting BdC-encoded protein func- <br /> that SER-NOTCH interactions function distinc-               tions antimorph likely encodes truncated <br /> tively DLNOTCH interactions. Indeed,              form SER consisting N-terminal domains in- <br /> defined role Serfunction thedevelopment           cluding 14EGF-like repeats lacking extracel- <br /> of adult wing blade appears SER           lular cysteine-rich domain,the transmembrane seg- <br /> DL send qualitatively distinct signals NOTCH.           ment intracellular domains. <br />    In wing blade development, Notch required forcell <br /> proliferation, vein differentiation specification                         MATERIALSANDMETHODS <br /> the wing margin (SHELLENBARGER           MOHLER 1978;           Strains: All strains usedweregrown          25&quot; standard <br /> DE CELISand GARCIA-BELLIDO1994;RULIFSON                     media. Most strains, balancers mutations pre- <br /> BWR 1995). The effects loss-of-function mutations            viously described LINDSLEXand ZIMM (1992), FLEMING et <br /> at dorsal wing margin restricted overproli-         al. (1990) LEHMANN        et al. (1983). Additional aberrations <br /> feration cells ventral compartment ec-           used thiswork include Ser+T*3kand UAS-Sm line D (Gu <br />                                                                 et al. 1995), Y488 [w&apos;] Df(3R)SQ219 chromosomes <br /> topic expression suggest SER functions dorsal          ( SHELTON     WA.%ERMAN      1993), Df(3R)Tl-P, defi- <br /> <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> ventral signal inthe establishment wing margin        ciency for97A 98A1-2 removes Ser locus 97F <br /> (COUSOet al. 1995; DIAZ-BENJUMEA          COHEN1995).       (HASHIMOTO et al. 1988), Gd417A second chromo- <br /> DL likewisebeen proposed tospecify reciprocal           some, GAL4 activator insert line (gift N. PERRIMON). <br /> signal ventral dorsal compartments based            M8-1 small deficiency Sm (see text) induced by <br />                                                                 treating 3- 4-day-old males genotype w&quot;&quot;/,            Y488 <br /> upon similar experiments Dl loss-of-function muta-         [w+] Bdc/TMGB 4000 R gamma rays -450 R/min <br /> tions ectopic DL expression (DE CELISet al. 1996;           (J. L. SHEPPARLI    Cs source). They mass mated with <br /> DOHERTY      et al. 1996). Wing margin specification ap-        w1118/w1118. sm+r83h/sm+r83h     . <br />                                                                                                nrgin   females F1 progeny were <br /> pears require specific dorsal ventral           selected fArviable Y488 Bdc*/Ser&quot;83h (phenotypically non- <br /> (SER) ventral dorsal signals (DL) converging          white-eyed) animals mated TMGBbearing fliesto estab- <br />                                                                 lish lines. <br /> the common NOTCH receptor torestrict maintain                   Generation Bdc suppressors: Crosses originallyde- <br /> margin-specific expression genes wingless, ves-      signed isolate Gal4 enhancer traps (BRANDand PERRIMON <br /> tigzal, cut scalloped (WILLIAMS    et al. 1993; DOHERTY     1993) capable expressing UAS-Serconstruct rescu- <br /> et al. 1996). This model suggests NOTCH           ing thepupal lethality associated withBdc/Ser+&apos;83hanimals (Gu <br /> can activated SER DL interaction          et al. 1995). w&apos;&quot;~/Y; Gal41 7A/+; e B&amp;/A2-3 Sb 99Bmales were <br />                                                                 crossed w &quot; ~ ~ / w &apos; ~UAS-Ser/UAS-Ser; <br />                                                                                              &apos;~;              red Ser+T83k/redSer+T83h <br /> of ligand yield dorsal ventral compart-           females. F, progeny examined individuals carrying <br /> ment-specific response based interaction.             e Bdc Ser+r83k    chromosomes (phenotypicallynon- <br />    In attempt understand interaction           Stubble). We examined 13,051 F1 animals recovered <br /> SER NOTCH generates signal distinct            viablemales (inferred genotype: wl&apos;&quot;/,               + or <br /> of DLNOTCH interaction, focused dom-             Gal41 7A/UAS-Sm;e B&amp;/red Ser+r83h). <br />                                                                     Recombinationstudies: The Y488 [ w + ] Bdc chromosome <br /> inant mutations Ser. Loss-of-function Seralleles behave      generated meiotic recombination crossing wlZf8/ <br /> as recessive mutations result larval lethality      ~ 1 1 1 8 ; Y488/e Bdc females W I &quot; ~ / Ymales selecting <br /> produce mutant animals exhibit anterior defects         F, progeny co-segregation w+ (Y488) and <br /> spiracle mouth-hook development whenhomo-              Bdc dominant phenotypes. Among 38,341 progeny, single <br /> zygous (SPEICHER     et al. 1994).In contrast, mutations   phenotypically w+ Bdc male recovered crossed to <br />                                                                 w&quot;18/w&apos; I&quot;; TM3,Sb/TM6B females establish balanced <br /> as Beaded (Bd) Serrut~milLant    (St#) alleles gener-   stock. <br /> ate characteristic wing margin defects heterozygous              Our initial recombination study utilized heterozygous      w1&apos;l8/ <br /> adults (DEXTER 1914;GOTTSCHEWSKI 1935; BELT 1971;               w1&apos;I8; Y488 B&amp;/+ females crossed W &apos; &apos; &apos; ~ / Y ; red S e ~ - + &apos; ~ ~ ~ / r e d <br />                                                                 sm+r83k males. All progeny receiving w+ (Y488) chromo- <br /> FLEMING     et al. 1990; THOMASet al. 1991). Since Ser a <br />                                                                 expected die interaction Bdc <br /> haplo-sufficient locus, alleles produce       mutation linked Y488 Ser+rg3hmutation unless a <br /> products capable interfering           recombination event Y488 theinteracting lethal <br /> normal Ser functions. In paper, examine domi-         segment Bdc chromosome occurred. <br /> nant mutation Ser, known Beaded Goldschmidt                A second recombination screen emplo ed w1118/w1118;Y488 <br /> (Bdc).When transheterozygousto deficiencies point           BdC/DS(R)SQ219females crossed w &apos; &quot; Df(3R)SQ219 ~ ~             ca/ <br />                                                                  TM3, Sb males. Since Df(3R)SQ219 homozygous lethal <br /> mutations Ser, Bdc animals display larval lethal <br />                                                                  (SHELTON      W A S S E W 1993), phenotypically non-Stub- <br /> phenotype similar seen homozygous amor-             ble, whiteeyed progeny sought represent <br /> phic Ser alleles. In heterozygous condition, Bdc ani-       progeny recombinant interval Y488 inser- <br /> mals display variably penetrant, dominant wing phe-           tion proximal end Df(3R)Sy219 deficiency. <br /> notype dramatically severe                 Such progeny (inferred genotype: w ~ ~ &apos; ~ / w &apos;or     &apos;     recombi- <br />                                                                 nant*/Df(R)SQ219 ca), crossed TMGBca males or <br /> animals heterozygous amorphic alleles         females w+ background phenotypically non-claret, <br /> of Dl. Molecular characterization Bdc mutation           recombinant progeny recovered maintained bal- <br /> demonstrates associated witha partial duplica-       anced stocks. <br /> tion Ser locus insertion transpos-            Temperature sensitivity: Balanced stocksof e B&amp;/TM6B or <br />                                                Characterization a361 <br />                                                                     Ser Antimorph <br />  <br /> Y488 BdC/TMGB allowed lay eggs 24 hr room            program ( ~ T S C H U Let al. 1990) used search Na- <br /> temperature (23&quot;) placed tempera-         tional Center Biotechnology Information databank. <br /> tures (18&quot;, 23&quot;and 29&quot;) eclosion wherewing phenotypes <br /> were assessed. <br />    Standard DNA techniques: Isolation genomic DNA, re-                                    RESULTS <br /> striction site mapping, subcloning, labelingof&quot;P DNA and <br /> RNA probes, Southern blotting, library screening     hybrid-       BdG functions antimorph Chromosomal dele- <br /> ization techniqueswere performed essentially described        tions loss-of-function point mutationsof Serlocus <br /> SAMBROOK     et al. (1989). Densitometry performed using        phenotypically recessive result larval lethality <br /> National Institutes Health Image 1.54 MacIntosh <br />                                                                     homozygotes(FLEMING et al. 1990; THOMAS            et al. <br /> by W. RASBAND. <br />    Isolation Bdc polymorphic EmRI fragments: Genomic             1991; SPEICHER    et al. 1994). Dominant mutationsof the <br /> DNA extracted heterozygouse BdC/TMZ, red e flies           locus, original Ser allele, Ser&quot;, B&amp; <br /> and digested completionwith enzyme EcoRI. The EcoRI          allele produce dominantwing blade phenotypes het- <br /> fragments cloned EMBL4 lambda phage arms(Pro-             erozygotes (seeFigure1,        B C, respectively). Al- <br /> mega)alsodigested EcoRI, ligated subsequently              thesetwo mutations appearphenotypically simi- <br /> packaged phage particles using Gigapack (Stratagene). <br /> The BdC/TMZ, rede library screened standard tech-            they <br />                                                                     lar, distinct <br />                                                                       exhibit properties.                              Animals <br /> niques using radiolabeledDNA probes spanningthe 0 + 1            homozygous S d viable display wing phe- <br /> region Ser (FLEMING et al. 1990).Bdcfragments identi-       notype severe S d / + animals (Figure <br /> fied sizes asexpected genomic Southern     analysis    1D). Animals heterozygous S d Ser deficiency <br />  (11 14 kb I 5 L 7R, respectively)relative       viable. In contrast, animals  homozygous Bdc <br />  TMZ, red e (Serf) band (20 kb). <br />                                                                     o r heterozygous B&amp; Ser deficiency larval <br />    WAse protection analysis: This technique performed <br /> as CPMB Protocols (AUSUBELet al. 1990) using 10 p g B&amp;/          lethal. It suggested insertion the <br />  + wild-type total RNA followingchanges: 1OX            transposable element Tirunt 3&apos; untranslated re- <br /> elution buffer 3 M NaAc, 1% SDS, 25 pg/ml tRN&amp; ribo-            gion S@ allele stabilize Ser mRNA, leading <br /> nuclease digestionwas room temperature 30 min            higher levels protein expression mutant <br /> the omission ribonuclease A 10 mM Tris-HC1 pH 7.5,             (THOMASet ul. 1995).Thus,the           S@ allele be <br /> 500 mM NaCl, 5 mM EDTA pH 8.0,2 pg/ml ribonuclease T1; <br /> hybridization 50&quot;. The origins RNAase            thought hypermorphicvariant.Interestingly, <br /> protection fragmentsare follows[using nucleotide posi-       S@ Bdc alleles heterozygous, wing <br /> tionsgivenin FLEMING         et al. (1990); fragments named     phenotype similar S@/+ observed (Figure <br /> based exon numbersthey span, Figure21: fragment           1E). This suggests nature theBdc mutation <br /> E6(1004-1302),fragmentE9-10(1658-2119),fragmentEll-                 fundamentally different thatof S d , Bdc <br />  13(2761-3252),fragment           E1314 (3019-3503),fragment        acted similar manner S d , resulting wing <br /> E13-142 (3128-3759) fragment E14 (3498-3837). RNA <br /> was produced fragments subcloned       Bluescript   phenotype appear comparable toS @ / S d (Fig- <br /> vectors (Stratagene)using T7 T3 RNA polymerases.          ure 1D). <br />     Generation 3&apos; Bdc cDNA Reverse transcription using               <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> P-element mutagenesis screen,       isolated <br /> B@/+ polyA+RNA performed Superscript I1 reverse            suppressors B&amp; phenotype suggest Bdc en- <br />  transcriptase (GIBCO-BRL) using synthetic dTTP(,,,-EcoRI-         codes antimorphic variant Ser. The screen was <br /> dTTP,,, oligonucleotide.Two micrograms polyA+RNA and <br />                                                                      designed recover mutations capable rescuing the <br /> 5 pl (20 p~ stock concentration) oligonucleotide in10 p1 <br /> volume heated 70&quot; 10 min quick chilled           late pupal <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>       associated B&amp;/ <br /> ice. To added 4 pl 5X transcription buffer(GIBCO- <br />                                                                      ser+rX3k combination ser alleles (see MATERIALS AND <br /> BRL), 2 pl 100 mM dithiothreitol, 2 pl 10 mM dNTP mix            METHODS;     Ser+rx3K described Gu et al. 1995). From <br /> and 2 p1 reverse transcriptase. The reaction incubated       13,051 F1 progeny, recovered indepen- <br /> for 1 hr 37&quot; followed 30 min 45&quot;. Two microliters (2        dent &quot;mutant&quot; males (inferred genotype: w&apos;&apos;&apos;&apos;/,       UAS- <br /> U/pl) RNAse H added reaction incubated <br /> at 37&quot; 20 min. The reaction pheno1:chloroform ex-            Ser/+;e Bdc/red Ser+rx3k).   One surviving <br /> tracted ethanol precipitated. PCR amplification per-         males tobe sterile. The remainingtwo, desig- <br /> formed reaction using 5 p1 lox Taq Buffer           nated Bdo&quot;&quot; SFa7&quot;, fertile crossed to <br /> with 15mMMgC12 (Promega),5 pl dNTP stock (final concen-              TM?, Sb/TM6B virgin females furtheranalysis. Both <br /> tration: 200 p~ dNTP), 1 p1 (20 pM stock concentration)         BdG&quot;&quot; Bdo&quot;&apos;&quot; similar phenotypicand molec- <br /> of Ser primer artificial EcoRI site added 5&apos; end <br />                                                                     ular properties Bd&quot;&quot; used following de- <br />  (coordinates primer3019-3042), 3 pl dTTP,,,,- <br /> EcoM-~TTP,~,     oligonucleotide (20 p~ stock concentration)        scription. In balanced line e Bdorev7A/TM4B,the <br /> and 2.5 units Taq polymerase(Promega)in 50-pl reaction         Bdc phenotype completely suppressed (Figure 1F). <br /> volume. PCR conditions follows: 1 min 94&quot;, 1 min         Moreover, suppressionoccurredin          animals dis- <br /> at 55&quot;, 2 min 72&quot; 40 cycles followed cycle 72&quot;     playing completely white eye phenotype indicating <br /> for 10 min. The resultant product digested EcoRI           suppressionwas likely presence <br /> and subcloned Bluescript. <br />    Sequencing sequence analysis: The BdG cDNAse-                w+-bearing Pelements expectedof screen. Geno- <br /> quence determined using dideoxy chain-termination           mic Southern analysis e Bflm7&quot; confirmed pres- <br /> technique (SANGER     et al. 1977) double-stranded DNAusing      ence Bdc mutation detected addi- <br /> Sequenase reaction protocols (U.S. Biochemical).A BLAST             tional wild-type copy Ser locus segregating with <br />  362                                              N. A.and <br />                                                        Hukriede               R.J. Fleming <br />  <br />  <br />  <br />  <br />                                                  ~ &quot; &quot; - ~ - ~ ~ - ~- - <br />                                                                         ~   ~   .=   c <br />  <br />  <br />  <br />  <br />    FIGURE    1.-Wing phenotypes wild-type B&amp;-associated mutants. (A) Wild type (Canlon-S). (B) Sd&apos;/+with characteristic <br /> distal wing notches ( C ) B&amp;/+; evident loss anterior wing margin associated bristles hairs. (D) .Yn&quot;/.Yd&apos;; <br /> loss anterior margin bristles increased distal notching evident [compare B (Sd&apos;/+)].   (E) B&amp;/.SdJ; B&amp; anterior <br /> margin phenotype evident, producing wing similar SdJ/+ (see B). (F) Bd&quot;&quot;&quot;&apos;/TMfiB; phenotypically wild-type wings <br /> are formed B&amp; presence wild-type SPY+doses (see text) ( G ) Dpy6/+; characteristically thickened wing <br /> veins evident intersection margin L2 L5 intersection posterior crossvein 15. <br /> (H) e B&amp;/Dl&quot;&apos;6; replacement regions proximal ebony produces weakly viable animals greatly reduced wing blades (see <br /> text). ( I ) Bd&apos;m&apos;iA/Dpy6;phenotypes characteristic Bd&apos; DZ evident presence S d doses. All wing <br /> pictures taken magnification. <br />  <br />  <br /> the e Bd&apos;;&quot;&apos;&apos; chromosome (data shown). Thisfind-                      alleles ofDlwere lethal. Additionally, theyreported that <br /> ing suggested rescuing ability B&amp;;&quot;&quot;&quot;                     Bd&apos;; allele Serresulted lethality heterozy- <br /> (and Bd&apos;;&quot;&quot;* ) stock arose duplication inser-                   gous amorphic Dl alleles. In attempts repro- <br /> tion wild-type copy Ser e B&amp; chromo-                     duce findings, foundthat we <br /> some.                                                                     utilized different Ser deficiencies (Df(3R)Tl-P and <br />    The ability additional copy wild-type Ser                 Df(3R)Ser+R2/24 TD),when heterozygous DFX6de- <br /> gene tosuppress Bd&apos; phenotypes supports possibility                   ficiency, produced fully viable animals (Table l , <br /> that Bd&apos; encodes antimorphic form Ser. We crossed                   lines 1 2 ) . Although interactions deficien- <br /> the e Bd;&quot;&apos;7A chromosome <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>point mutation</span>                      cies Ser Dl found, analyses con- <br /> Ser locus ( S P P &apos; &apos; ~ &apos; ) to <br />                             , Ser deficiency (Df(3R)Ser+82p4)           firmed reported lethality animals heterozygous <br /> and wild-type chromosomes (Cantons). In cases                      B&amp; Dfx6    deficiency (only viable animal <br /> of e B&amp;&quot;&apos;&quot;/SeF6&apos;        e Bd&apos;;&quot;&apos;7*/Df(3R)Ser+82/24,the effec-         recovered 2471 total progeny; Table 1 , line 3). <br /> tive Ser dosages 1 B&amp;: 1 wild type appearpheno-                   However, noted Bn&apos;; heterozygous point <br /> typically identical original e B&amp;/+ flies. In                  mutation Dl locus, Df&apos;, produced viable animals <br /> case ofe Bd&apos;;m&apos;7&quot;/Canton3,the effective Serdoseis 1 B&amp;:2                  (Table 1 , line 4). <br /> wild type flies appear phenotypically wild type.                     The finding  severe, apparently amorphic DP&quot; <br /> Thus, conclude Bd&quot; antimorphic effects                 allele, transheterozygous B&amp;, produced via- <br /> can diminished addition wild-type Ser.                       ble phenotype suggested lethal interaction be- <br />    Interactions Bdc Dk THOMAS           et al. (1991) re-          tween mutations specifically caused <br /> portedthat animals transheterozygous combina-                         Dl B&amp; ratherbetween mutation (s) on <br /> tions Ser deficiencies deficiencies amorphic                    B&amp; Dl chromosomes used studies. To <br />                                               Characterization Ser Antimorph                                          363 <br />  <br />                                                             TABLE 1 <br />                                      Viability transheterozygous Serrate/Delta mutations <br />  <br />                                                                       Percentage <br />                                                                          Total                           expected <br />                                             Delta parent                progeny                   Ser/Dl animals <br />         Df(3L)Tr/ TM3                     DPY6 e ro/ TM6B                  375                         116 <br />         Df(3R)Seri-82&apos;24/ T M 3           DPX6e ro/TMGB                    224                          96 <br />         Bd&quot;/TM3                           DPx6 e ro/ TM6B                 2471                           0 <br />         Bd&quot;/TM3                 85        DLyp/T M 3                       163 <br />         e BE/ TM3                         Of&quot; e ro/ T M 3                 2230                          70 <br />         Y488 Bd&quot;/ T M 3                   DPx6 e ro/ TMGB                  812                         104 <br />         Y488 Bd&quot;/ TM6B                    DEBx6/ TM6B                      248                          88 <br />         Y488 B&amp;/ TM6B           93        DL&quot;/ TM6B                        177 <br />         Y488 B&amp;/ TMGB 94*                 DfXI2/TM6C                       158 <br />         Y488 B&amp;/ TM6B 84*                 DP2/ TMGC                        176 <br />         Y488 Bd&quot;/ TM6B                    Dl&quot;&quot;/TMGC                        162                          67* <br />           Animals bearing deficiencies Ser locus Bd&quot; containing chromosomes crossed animals <br />         heterozygous alleles Dl locus determine complementation behavior loci. Percentage of <br />         expected Ser/DL animals calculated percentage total progeny expected Ser&quot;&quot;~&quot;f&apos;&quot;&quot;/Dl these <br />         animals fully viable. *indicates expected percentage in <br />                                                                            cases likely underestimate <br />         TMGB/TMGC class animals poorly viable does constitute expected 25% progeny. <br />  <br /> test hypothesis, proximal regions B&amp;               Y488 Bd&quot;; sample size perhaps <br /> chromosome including theDL region replaced                genetic backgrounds associated DL alleles. <br /> meiotic recombination ebony chromosome. This              Most interesting Dl alleles, when <br /> &quot;cleaned&quot; e Bd&apos; chromosome tested combina-                 transheterozygous B&amp;, exhibit severe winginterac- <br /> tion DFx6 chromosome. From crosses,                tion phenotype similar seen Figure 1H. Al- <br /> viable e Bd&quot;/DPx6animals recovered (Table 1, line            DL alleles used generated <br /> 5), suggesting production original B&amp;/            laboratory (M. MUSKAVITCH)and have <br /> DPx6 lethal interaction was, part,           similar genetic backgrounds, allele, Dl&quot;, <br /> second site lethal mutation. However, e BdC/DPX6ani-              originated independent laboratory (C. NUSSLEIN- <br /> malsshow reduced viability (only -70% ex-                  VOLHARD)and likelyhas         different genetic back- <br /> pected number animals recovered) display              ground. Thefact alleles generate similar <br /> roughened eyes, duplicated bristles severely de-              wing interaction phenotypes suggests strongly the <br /> formed wings (Figure 1H; compare Figure l, C               wing phenotypes interactions be- <br /> G, respectively).Further replacementof proximal chro-             tween Ser DL loci. <br /> mosomal regions 20 kb 3&apos; end                 Bde/D1 wing interactions: The Ser DL loci be- <br /> BdG mutation accomplished recombining                  lieved encode ligands capable binding induc- <br />  Y488 w+ P-element insertion (SHELTON     WASSERMAN           ing signals NOTCH receptor (ARTAVANIS <br />  1993) Bd&apos; chromosome (see MATERIALS AND                 TSAKONAS    SIMPSON   1991; HEITZLER    SIMPSON <br /> METHODS). Y488 Bd&quot;/DPx6 animals fully viable              1991, 1993; REBAY et al. 1991;ARTAVANIS-TSAKONAS        et <br /> fertile (Table 1, line 6); wing phenotypes con-           al. 1995; GU et al. 1995). Recent studies demon- <br /> sistent previously mentioned crosses             strated SER DL required distinctly in <br /> Bd&apos; DL deficiencies stilljust severe (compara-         adjacent compartments wing proper wing <br /> ble Figure 1H). These data suggest exist            blade development (COUSOet al. 1995; DIAZ-BENJUMEA <br /> at separable mutations Dl Bdc               COHEN1995; DE CELLSet al. 1996; DOHERTYet al. <br /> that influence viability and/or expressivity           1996). Since SER DL proteins exhibit the <br /> original Bd&quot; chromosome transheterozygous                 ability bind 11th 12th EGF-like <br /> the DPx6 deficiency,                                              repeats NOTCH (REBAY et al. 1991), Bdc <br />    To demonstrate thephenotypic wing in-             appears encode dominant-negative form Ser, <br /> teractions associated Bd&apos;/DPx6 animals caused            possible explanation phenotypic interactions <br /> by interactionbetween Serand Dlloci, crossed            Bd&apos;: DL mutations Bd&apos;: <br /> B&amp; additional DL alleles: Dl&quot;&apos;, DPX&quot;, 018&apos;          product inhibit Ser Dl signaling, thus <br /> OrM2.As shown Table 1, Y488 Bdc animals live              generating phenotype severe muta- <br /> transheterozygous DL alleles. Two al-            tion alone. We tested hypothesis using the <br /> leles Delta, OF2 (an EMS-induced allele) DtM2              ~ d &apos; ;7A <br />                                                                          &quot; chromosome. This chromosome carries one <br /> (deficiency 91C7-Dl-92Al) appear produce                  normal Ser+ copy Bd&quot; antimorphic allele, yet <br /> slightly decreased viability transheterozygous            BdGa&apos;7A/+ animals exhibit phenotypically wild-type <br />  364                                                N. A.and <br />                                                          Hukriede     R. J. Fleming <br />  <br />                                                                 TABLE 2 <br />                                   Phenotypic wing interactions Bdc revertants Dl mutations <br />  <br />                                                                                                        Percentage wing margin <br />                BdC&quot;&quot;                                      Percentage wing blisters                        phenotype <br />   Dl                     No. Dl         Total <br /> parent         parent    allele/rev* <br />                                  progeny                Dl/Bd&quot;&quot;&apos;      Dl/   +       Bd &apos;&quot;&quot;/ <br />                                                                                        ~ <br />                                                                                                +   Dl/ Bd &apos;km&apos; <br />                                                                                                       ~~ <br />                                                                                                                  Dl/   +-       Bd &quot;&quot;&apos;/         f <br />                                                                                                                                     ~ <br />  <br />  <br />  <br />  <br />   YP39 <br />                rev 2A        98             313                43         IO               1               14      0                    1 <br />                 rev 7A      136             435                41          1               0                3      0                    0 <br /> DlHXh          rev 2A        86             303                 0          1               0               43      0                    9 <br />                 rev 7A      135             400                 0          0               0               54      0                    9 <br /> DlHXJ2         rev 2A       123             361                20         IO               0               16      0                    0 <br />                rev 7A       133             387                11         2                0               20      0                    1 <br /> Dl&quot;2           rev 2A        63             218                 5         0                0                8      0                    0 <br />                rev 7A       100             325                 1         0                0               30      0                    0 <br /> DlB2           rev 2A        89             317                18         0                0               10      0                    2 <br />                rev 7A       134             397                 6         0                0               19      0                    0 <br />          ~~~                       ~~                     ~~        ~ ~ ~ _ _ _~_~ _ _ _ _ ~ <br />                                                                                 ~ <br />                                                                                                                                             ~ <br />  <br />  <br />  <br />  <br />   Animals heterozygous alleles Dllocuswere crossed animals heterozygousfor BdCIPl&apos;g&quot;ant <br />                                                                                                       (lines 2A 7A). Progeny <br /> were allowed eclose adults scored wing blade phenotypes.Data presented percentage given class <br /> of animals expressing mutant phenotype relative total number animals class. Animals scored positive <br /> for wing blisters margin loss wings exhibited phenotype. <br />  <br />  <br /> wings. These results indicate thelevels Ser&apos; activ-          ported previously Bd&quot; phenotype cold-sensi- <br /> ity normal near normal levels                 tive with-88% heterozygous flies showing wing nicks <br /> animals. Importantly, B&amp; product pro-                raised restrictive temperature 1 7 , but <br /> duced animals capable directly induc-             2% flies displaying nicked wings per- <br /> ing Deltcclike phenotype. Hence, Bd&quot; product                missive temperature 29&quot; (THOMASet al. 1991; COUSO <br /> is capable interfering DL function, effects           et al. 1995).Due findings second site mutations <br /> this interference lesssevere lossof                 Bd&quot; chromosome, tested Bd&quot; recom- <br /> one dose Dl itself (i.e., compared DPx6/+             binant chromosomes, e Bdc&apos; Y488 Bd&quot;, determine <br /> instance; Figure 1G).                                                retained cold-sensitive characteristics the <br />     We tested possibility interference           original Bd&quot; chromosome (see MATERIALS ANDMETH- <br /> Dl function Bd&quot; combinations Bdc&quot;&quot;7*                ODS). The combined percentages e Bd&quot; Y488 <br /> ~ f k r n ~ 2 A <br />              Dl mutations. Since Dl chromosomes             Bd&apos;; flies displayed wing nicks 66% 29&quot;, 77% <br /> carry wild-type Serf allele, resulting B~7pU/ol ani-           23&quot; 98% 18&quot;. Therefore, appears the <br /> mals Ser&apos; doses Bd&quot; dose, pro-                Bd&quot; mutation does produce cold-sensitive effect, but <br /> duces wild-type wingphenotype. However,                pronounced previously reported. <br /> animals onlyone Dt&apos;dose, exhibit                   Molecularcharacterization BdG genomic DNA: <br /> expected DL/+ phenotype interference                  Using sequence comparisons 5.6-kb Ser cDNA <br /> from B&amp;, exhibit worsened phenotype                    genomic DNA,we generated exon map <br /> Bd&quot; interfere Dl function.                                  depicted Figure 2. The 5.6-kb Ser message tran- <br />     As shown Table 2, animals heterozygous BdX&quot;               scribed region spanning -25 kb genomic <br /> and Dl viable cases. Concomitantly cases,          DNA contains 15exons. Southern analyses utilizing <br /> the resulting wing phenotypes increase phe-               DNA probes 25-kb region revealed two <br /> notypic severity compared animals carrying                 regions polymorphism B&amp; genomic DNA <br /> the Dl allele absence B&amp; mutation                 wild-type chromosomes (FLEMING        et al. <br /> (compare Figure 1, I G). With exception              1990). The 5&apos; polymorphism located coordi- <br /> DPX6allele, Dl/Bfk&quot;&apos; animals generalincrease               nates +14              + <br />                                                                                          17 beenreported by <br /> severity wing phenotypes margin loss blis-                THOMASet al. (1991). The polymorphic region <br /> tering. The DPX6/Bd&quot;&quot; animals appear specifically                 located 3&apos; coordinates 0 + l . We <br /> increase margin loss phenotype formation                cloned genomic DNA regions corresponding 3&apos; <br /> of blisters. The cause phenotypic margin loss                polymorphic sequences conduct molecular charac- <br /> specificity DBX6crosses unknown. From                terization B&amp;-associated chromosomal lesions (see <br /> results, conclude presenceof Bd&quot; muta-               MATERIALS AND METHODS). <br /> tion influencethe phenotypic expression Dl;                     Genomic probes coordinates 0to 1 detect     + <br /> however, data allow establish               polymorphic EcoRI bands Southernblots B&amp; DNA <br /> effect direct indirect (see DISCUSSION).                       (FLEMING  et al. 1990). The new EcoRI bands 11 and <br />     Temperature-sensitiveeffects BdG: It re-             14 kb designated 1% X,respectively (Figure <br />                                                 Characterization Sm Antimorph <br />  <br />     -10                -5                0         +15    +5               +10                                 +20              +25 <br />  <br />       I                 I                 I                I                 I                 I                 I <br />  <br />  <br />  <br />                  B                     BB <br />  <br />                                                                                                                       5&apos; <br />  <br />  <br />  <br />                      BdGISL <br />                . <br />                . <br />                .. <br />                 .. <br />                  . <br />                  .. <br />                   .. <br />                    . <br />                    .. <br />                     .. <br />                      .. <br />                       .. <br />                        . <br />                        . <br />                     .. <br />                      .... <br />                         ... <br />                           ... <br />                             ... <br />                               ... <br />                                 ... <br />                                   ... <br />                                     ... <br />                             BdG 7R <br />    FIGURE  2.--Molecular map Serrate region. A representative map genomic region surrounding wild-typc S p r <br /> gene drawn centromere left (proximal) telomere right (distal) depicted. Scale kh top. <br /> Restriction enzymes follows: E, EcoRI; B, BnmHI; H, HindII. Restriction sites polymorphic hetwccn wild-typc <br /> strains placed parentheses. The .%message iLs corresponding 1.5 cxons ( W ) 14 introns rcprrsented l x * l o w <br /> the map. The location 5&apos; B&amp; polymorphism drawn triangle coordinate + 14, roo insertion clrment drawn <br /> as triangle coordinate + I SPr-homologous segment.. 3&apos; Bd&apos; polymorphisms 1 2 , n t l /R arc shown        at <br /> the hottom map. indicates     5&apos; ( + 14 + 17) genomic prohe used Figure S     indicates 3&apos; ( - 9 + 3 ) <br /> prohe used Figure 5 . The locations RNAse protection fragments Ef, E9-10, E l 1-13, E13-14, E13-14? antl E14 arc shown <br /> ahove map (see text). <br />  <br />   2). We isolated EmRI fragments Bn&apos;; gencr             gous IX 7R reveals single site hybrid- <br />   mic library forcharacterization(see       MATERIAIS AND            ization (datanotshown).           Any d~~plicationof S t v <br />   METHODS) surprised theI X               sequences Bd; chromosome re- <br />   7R fragmentscontainthe        I h R I site -9           close proximity normal 97F location. To <br />   encode homologous sequencesup past BnmHI site                  determinewhethertheduplicationencompassesthe <br />   5&apos; position 0. We characterized IX frag-            entire S/rlocus o r 3&apos; regions, employed <br />   ment determined PCR contained wild-                 Southern blot analysis examine 5&apos; regions IM&quot; <br />   type .&apos;%sequences thatjoined non-SoencodingDNA                  DNA. A$ seen Figure SA, genomic lld;/TM6lj DNA <br /> just 3&apos; intron 13 (Figure 2). DNA sequencing               cut LroRI probed +14 t o +17 reveals the <br />  .5&apos; fusion fragment revealed encoded So            9.5-kh HA; band -3-kb 7j146B counterpart.            l   3   ~   r <br />  <br />  <br />  sequences nucleotide position33.58 (of 5561 total;             placing IW heterozygous asecond chromosome that <br />  FLEMING    et nl. 1990) 3&apos; end SPTbut con-                                           +       + <br />                                                                      polymorphic 14 17 EmRI fragment <br />   tainednon-Sosequences        5&apos; 3358. An homology               (called l-Dll), clear 118 chromosome con- <br />  search revealed non-,%DNA IX homolo-                   tains 9.5-kh hand band comigrating at <br />   gous right terminal repeat B104 (roo), dis-            -3 kb. Furthermore, densitometry -3- 9.5- <br />   persed repeated genefamily present -80-100 copies               kb bands demonstrates      hands prcs- <br />   genome D. m.elanognskr (SCHERER        pt nl. 1982).        ent equivaIent amounts. \?e conclude B d ; com- <br />  The 7R EcoRI fragment likewise characterized                prises partial duplication Snlocuswith single <br />   encode 3&apos; Sosequences beginning -9                 5&apos; distinct 3&apos; regions. <br />   EcoRI site continuing towithin ninth intron                Recombination studies define Bdc functional do- <br />   So.Homology searches non-SPr DNA associated                 mains: To determinei f o r polynor- <br />  7R fragment failed reveal anysignificant                phicregionscontribute         observed B d &apos; pheno- <br />   homologies known sequences.                                   types, employed meiotic recombination generate <br />      The finding Bd&apos;chromosome contains                intragenic recombinants retaining portions Ild; <br />   I X 7R fragments suggests exists                mutation. We utilized Y488 [ I d &apos; chromosome tle- <br />   partial duplication Sm locus             scribed retains described Bd&apos; poly- <br />  chromosome. In situ hybridization polytene chromo-               morphisms.Femalesheterozygousforthe              Y488 Bd&apos; <br />  somes Bn&apos;; animals using DNA probe homolo-                   chromosome wild-type chromosome were <br />  366                                                         N. A. Hukriede R.J. Fleming <br />  <br />                                                               2                Y488 (70&apos; )bearing chromosomes established as <br />                                                &apos;9 <br />                                                Y)             E                balanced stocks TMhB TM3,Sh. Also, line <br />       &apos; 9 Q Y &apos;9 <br />                )                               2&apos;9         &apos; 9sz               w- produced held-out Ser+rX3k       wing phe- <br />                                                                                notype recovered established balanced <br />       2G <br />       w        ? <br />                S <br />                            $ <br />                            S <br />                                                $- . g$ +%; z $                 stock TM6B TM3,Sb. Of 13 lines, three <br />       ? , ,                        M           2 3 3 S 4 k l                   homozygousviable       10 homozygous lethal. <br />                                                                                The genetic properties recombinant tested <br />  <br />  <br />            o1).I. 9.6                          -     p~ <br />  <br />                                                      f , <br />                                                            c <br />                                                                    &quot;-   23.6 <br />  <br />                                                                    c~ 9.46 <br />  <br />                                                                    r 6.64 <br />                                                                                crossing Bd&apos;, Serloss-of-functionpoint <br />                                                                                mutation ( S f l &quot; &apos; ) Ser+r&apos;y&apos;y&quot; allele. The results <br />                                                                                presented Table 3. Three recombinants <br />                                                                                (I2A,I7B, I9A) produce homozygousviable animals <br />                                                                                complement recessive phenotypes three <br />  <br />  <br />                                                      1 <br />                                    c)   6.64                                   Ser alleles tested. Seven additional recombinants (IlA, <br />                                                                                I2B, MA, I3B, I6A, I7A, I8A) produce animals are <br /> *     c    l           l       o                                   y    434    homozygous lethal lethal heterozygous to <br />                                    m 434             d$                        Bd&apos;, wildtypewhen         heterozygous SPJ&quot;&apos;&amp;&apos; <br />                                                                                mutation Ser+rx3kallele. The remaining re- <br />                                                                                combinants, I4A, I9B I9C, produce animals that <br />                                                                                exhibit homozygous lethality lethality hetero- <br />                                                                         2.26   zygous B&amp; S f l &quot; &apos; , produce viable animals <br />            C       .                    2.26                            I .%   heterozygous Ser+rx3k.I9C dominant <br />                                         I .%                                   male sterile phenotype unrelated theSer <br />                                                                                locus. <br />                                                                                   Genomic DNA recombinant systemati- <br />                                                                                cally probed presence 5&apos; 3&apos; polymor- <br />                        A                              B                        phic regions Bd&apos;. For recombinants IlA, I2A, I2B, <br />   FIGURE3.-Genomic Southern analysis B&amp; 5&apos; polymor-                         13A, I3B, I6A, I7A, I7B, I8A I9A, B&amp; <br /> phism. Probe used +14 + I 7 D N A wild-type                   polymorphic regions retained. As seen Figure 4, <br /> Smlocus (see Figure 2 ) . ( A ) rDB/ TM6Bdisplays wild-                        places likely recombination site events <br /> kb) YDB polymorphic (2.8 kb) fragments. e B / T M(3.0 6B <br /> shows 9.5kb B f band wild-type (3.0 kb) bands. The <br />                                                                    F           Y488 Ser 3&apos; regions proximal <br />                                                                                Bdc abnormalities. The difference the <br /> e B&amp;/rDB lane shows B&amp; (9.5 kb) YDB (2.8 kb) <br /> polymorphisms band migrating wild type (3.0 kb).                    homozygous viableand homozygous lethal phenotypes <br /> Band indicated * homologous plasmid sequences asso-                   associated recombinants likely the <br /> ciated D N A probe used. ( B ) Y488 B&amp;/TM6B show-                     presence second site mutation distal Ser the <br /> ing expected B&amp; wild-type bands (see A). I9B/TM6B                          Bd&quot; chromosome             removed the <br /> does notretain 5&apos; Bd&apos;; polymorphism. II2B/II2B pheno-                   time experiment (T. T. TRANG      R. J. FLEMING, <br /> typically wild type homozygous Bcf: 5&apos; polymor- <br /> phism. II6A/TM6B retains 5&apos; Bd&apos;; polymorphism ex-                      unpublished data). <br /> pected wild-type band. Df(3R)SQ219/ TM6B control lane                        Southern analysis line I9B revealed 3&apos; <br /> wild-type <span id='am-322' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. M indicates lambda Hind111 markers; corre-                  IX polymorphism Bd&apos; remains (Figure 5). The 7R <br /> sponding sizes right ladder.                                 polymorphism removed 5&apos; poly- <br />                                                                                morphic region. In case however, appears un- <br /> crossed homozygous Ser+&apos;83k/Ser+r83kmales. Since Bd&apos;;                       likely loss polymorphisms a <br /> results pupal lethality heterozygous Ser+r83k                       meiotic recombination event. Examination 5&apos; <br /> (Gu et al. 1995), w+ progeny expected die                           Ser regions demonstrated deletion 5&apos; Ser <br /> (inferred genotype Y488 B&amp;/Ser+r83k)unless recombi-                          encoding regions distal +14 occurred (Figure <br /> nation event occurred distal Y488 transposable                          3B). The origins deletion remain unclear, how- <br /> element proximal portion Bd&apos;; muta-                         I9B line demonstrates 1X polymor- <br /> tion responsible lethal interaction (see Figure                        phism itself does bring thelethal pheno- <br /> 4). In addition, phenotypically whiteeyed flies                         type Bd&apos;; heterozygous Ser+r83k. <br /> emerged exhibited     recessive held-out wing phe-                       One shortcoming recombination studyjust de- <br /> notype Ser+r83k (Gu et al. 1995), flies                     scribed screen selected loss of <br /> isolated suspected recombinants Bd&quot; re-                          Bd&quot; lethal phenotype heterozygousto Ser+rx3k. We <br /> gion (Ser+rx3kheld-out wing phenotype isrecessive;                             designed second screenthat selected recombinant5 in <br /> ser+r83k/ S e r deficieny produces viable animals).                            region including Bd&apos; did rely selection <br />    From -14,000 progeny, 12 independent w+, viable                             &amp;r.expression. We crossed w+ Y488 Bdc <br /> progeny recovered (inferred genotype: Y488                                chromosome Df(3R)SQ219,a deficiency including the <br /> Bdc*/ Ser+rx3k;where * denotes recombinant)                            gene @!le   , <br />                                                                                               proximalbreakpoint lies -50 kb distalto <br />                                                   Characterization   Spr   Antimorph                                   367 <br />  <br />                +-      centromere <br />                        Recombinantsin region <br />                 havewild-typegenomicSouthern     patterns <br />  <br />  <br />                               n <br />                                   I     \    \     \ <br />                   A               1     1    1     1 <br />                                                  &quot; <br />                                                                                                              wt <br />                                                        Serrate+ <br />  <br />  <br />  <br />  <br />                                                                           1        I         I       I <br />                   B                                                       I        I         I <br />                                                                                                  -   I <br />                                                                                                       ,     wt <br />                                                                                        Serrate+ <br />    FIGURE  4.-Recombination events selecting loss Rn&quot; phenotypes. Figures drawn relative orientation as <br /> Figure 2. Upper line indicates likely arrangement Rn&quot; chromosome (see DISCUSSION) lower line <br /> represents wild-type copy homologous chromosome. Possible recombination events designated hatched lines. <br /> Since 3&apos;-homologous segment? Spr, alternative regions crossover RB&apos; wild type possible. <br /> Females type depicted crossed Sprcnqnhomozygous males viable, phenotypically white&apos; progeny were <br /> selected. (A) Recombination Rd&apos; wild-type chromosomes region shown selection the <br /> Y488 [ d l phenotype expected produce animals wild-type Ser genomic Southern pattern. (B) Recombinants the <br /> interval shown expected retain 7R possibly 3&apos; 5&apos; polymorphic portions B&amp; chromosome are <br /> not associated ttie &amp; ~ / ~ S W + &apos; &apos;lethal <br />                                           &apos;~~    pheAotype&apos;(see text). <br />  <br /> .%r (SHELTON    WASERMAN1993; C. SHELTON,         personal              Recombinant IIIZB, homozygous viable line, re- <br /> communication). Y488 B#/DK3R)sQ219 females                          tained 5&apos; Bd&apos;: polymorphism (Figure 3B). This find- <br /> then crossed Df(3R)SQ219 m/TM3,SI, males                     ing suggests 5&apos; polymorphism, likely <br /> phenotypically whiteeyedprogeny          selected. A.               located second intron   Sm, insufficient to <br /> shown Figure 6, screen limit. recombination events               generate phenotypes associated  B&amp; <br /> to -100-kb region encompaqsing Sm locus                       mutation n o phenotypic effect o n Sm expres- <br /> selects forthe     loss ofthe    flanking Y488 [w&apos;]                  sion. The recombinants IIlB IIGA homozygous <br /> Df(3R)SQ219 (lethal) phenotypes.                                         lethal lines fail complement mutations within <br />    From 15,500 flies, recombinants isolated.                  Sm. Southern analysis IIlB IIGA DNA demon- <br /> Five (IIlA, II2B, 1 1 4 , IISA IISB) produce homozy-                 strated Bd&quot; pattern, 7R 3&apos; polymor- <br /> gous viable animals (IIlB, II2A II6A)                      phism eliminated line (IIGA shown in <br /> resultinhomozygouslethality.         All recombi-               Figure 5). Phenotypically, IIlB IIGA act similar to <br /> nants crossed ~ d &apos; ~, e + ~      &apos; ~S &apos;P T + ~ &apos;As <br />                                                            ~~.       original Scr&apos; mutation crossed alleles <br /> depicted Table 3, recombinants        IIlB          Smsuggesting the3&apos; 7R polymorphism does not <br /> and II6A complemented recessive phenotypess                       contribute outwardly expressed  Bd&apos; wing pheno- <br /> the SPT mutations tested. Genomic Southern             analysis          type. <br /> was employed search polymorphisms           associ-            Mutagenesis: To determine the <br /> ated B6:. Five recombinants (IIlA,                     3&apos; polymorphisms essential B 8 function, we <br /> II2A, 1 1 4 , IISA IISB) completely wild-type re-                employed gamma ray mutagenesis screen hope <br /> striction patterns analysis indicating there-               generating deletions Bd&apos; region. The screen <br /> combination events theseisolates                   relied rescuing lethality associated with <br /> occurred distal B6: region (see Figure 6). These                  B8/Sm+&quot;&quot; allelic combination. We crossed muta- <br /> findings suggest lethal phenotype associ-                  genized Y488 (zu&apos; )B&amp;/TMSB males Sm+rx3k/Sm+r83k <br /> ated recombinant II2A toa second site lethal                 females selected surviving 7u+ progeny. <br /> mutation residing outside SPTlocus.                                  Of 2881 flies screened, independent viable lines <br />  368                                            N. A. Hukriede R J. Fleming <br />  <br />                                                           TABLE 3 <br />                                    Complementation Bdc recombinants Ser alleles <br />  <br />  <br />  <br />                   IlA                      + (137/118)                -   (0/147)                 + (136/113) <br />                   I2A                      + (142/115)                +   (96,423)                + (220)t <br />                   I2B                      + (117/112)                -   (OW)                    + (187/143) <br />                   13A                      + (83/89)                  +   (100/115)               + (179)t <br />                   13B                      + (91/89)                  +   (102/121)               + (143)t <br />                   14A                      -   (0/106)                -   (0/130)                 - (127)+ <br />                   16A                      + (126/117)                -   (0/157)                 + (137/103) <br />                   17A                      + (105/85)                 -   (0/174)                 + (164/118) <br />                   17B                      + (150/129)                +   (71/111)                + (250)t <br />                   I8A                      + (96/81)                  -   (0/84)                  + (138/117) <br />                   19A                      + (159/139)                +   (67/83)                 + (213)t <br />                   19B                      - (0/85)                   -   (0/154)                 - (146/94) <br />                   19C                      - (0/78)                   -   (0/106)                -   (77/78) <br />                   I1 1A                    + (132/111)                + (86/67)                   + (156)t <br />                   I1 1B                    - (0/75)                   - (0/97)                       (0/77)* <br />                   I1 2A                    + (77/62)                  + (51/68)                  + (104/88) <br />                   I1 2B                    + (189/180)                + (174/144)                + (194)t <br />                   I1 4A                    + (79/89)                  + (76/77)                  + (150)t <br />                   I1 6 A                   - (0/61)                   - (0/87)                       (0/76)   * <br />                   I1 9A                    + (78/66)                  + (77/64)                  + (107)t <br />                   I1 9B                    + (89/65)                  + (71/94)                  + (129)+ <br />                   ~ 8 M 8 - l <br />                                            - (0/108)                  - (0/126)                  + (142)t <br />            BdC recombinant lines complementSerm6&apos; Bdc denotedas +; failure complement denoted <br />         -. For complementation behavior Serir83k, + denotes viability normal wing appearance; - denotes <br />         viability mutant wing appearance (Ser+&apos;83k recessive phenotype displays held-out wings); *indicates that <br />         combination results lethality wing phenotype assessed. Numerical complementation data <br />         (in parentheses) presented asthenumber       recombinant/Set&quot;&apos;Ut&quot;t animals compared number of <br />         recombinant/Balancer siblings Ser+&apos;83k mutation homozygous viable. Ser+&apos;83k data are <br />         presented single numbers recombinant line used homozygous viableor thenumber recombinant/ <br />         ser+r83k animals compared number Ser+r83k/balancer  sibling animals. <br />  <br /> were recovered. Both wf (M8-1)              lated messages approximately the differ- <br /> erect wings. The second line, MlO-1,was sterile               ence wild-type Ser transcripts suggests strongly <br /> could retained.Genetic analysis ofM8-1 revealed            Bdc-related messagesoriginate pro- <br /> that mutation behaved like Ser null. Southern              duction endogenous Ser promoter. <br /> analysis M8-1 mutant showed 5&apos; Bdc                   To assess structuresof novel Bdc-related tran- <br /> polymorphism present, I X              scripts, employed RNAse protection analysis. We <br /> and 7R 3&apos; polymorphisms (data shown). Further                 generated RNAs specific regions within <br /> analysis revealed presence small deletion just           codingregion Ser (depicted Figure 2; MATE- <br /> proximal 5&apos; polymorphism extending ap-                RIALS AND METHODS details), butfocused atten- <br /> proximately +7.5 +13 Sergenetic map. Since              tion 3&apos; polymorphic regions meiotic <br /> the 5&apos; aswellas I X 7R polymorphisms                  recombination experiments demonstrated the <br /> retained mutant, indicates          5&apos; polymorphism, homozygous, did gen- <br /> described polymorphisms are, isolation, sufficient          erate detectable phenotypes. If the <br /> generate B&amp; phenotype.                                        B&amp;-associated 3&apos; DNA polymorphisms encode actively <br />     RNAse protection analysis: Northern analysis ofBdc/           transcribed R N A s , sequences hybridize <br /> TM6B heterozygous animals reveals presence             Ser encoding anti-sense RNAs tothepoint <br /> newSer-homologous transcripts of3.8 3.7 kb                 diverge wild-type Ser sequences. <br /> addition expected 5.6- 5.5-kb wild-type Ser               We generated antisense RNAs spanned 5&apos;- <br /> messages (data shown; FLEMING       et al. 1990). The         divergence point I X 3&apos; polymorphism <br /> -100-nucleotide (nt) difference wild-type             (E13-14; Figure 2) 7R 3&apos; polymorphism <br /> 5 . 6 5.5-kb Sertranscripts attributedto differ-       (E9-10). Welikewise generated control fragments to <br /> ences 5&apos; untranslated regions Ser message           distal (E6), proximal (E14) be- <br /> (R. J. FLEMING,   unpublished data). The fact            tween polymorphisms (El 1-13). RNAse protec- <br /> difference observed lengths Bdc-re-             tion analysis performed total Poly A&apos; RNA <br />                                                    Characterization Ser Antimorph                                      369 <br />  <br />                                          p9                           phism fused foreign promoter, producing a <br />                                        v, <br />                                                                       3&#226;&#8364;&#732; Ser message species,then likely divergence point <br />                       p9 <br />                       v,                 2&amp;                           approximately position 3200. To confirm <br />                       s q        as.;;                                5&#226;&#8364;&#8482; 3&#226;&#8364;&#732; Ser-specific message corre- <br />                       L6 2&#226;&#8364;? % <br />                             f q g                                     sponds tothis RNA,we repositioned protection <br />                       z ? @ ?a:q                                    fragment region (E13-14,; Figure2). TheRNAse <br />                       E s S Q M                                     protection data&#226;&#8364;&#732; experiment generated <br />                                                                       RNA species. The larger fragment 630 nt again <br />           + <br />  <br />          7R <br />          15L- <br />               - <br />               4 <br />                                                                       corresponded expected wild-type RNA the <br />                                                                       smaller fragment 120 nt; expected size 5&#226;&#8364;&#8482; <br />                                                                       Serdriven message diverging   approximately 3350- <br />                                                                       nt location (data shown). From data, con- <br />                                                                       clude Bdc allele produces 5&#226;&#8364;&#732; Serspecific mes- <br />                                                                       sage diverges wild-type sequence or <br />                                                                       near 5&#226;&#8364;&#732; end IX 3&#226;&#8364;&#8482; polymorphism. <br />                                                                          Generation Bdc cDNA The results our <br />                                                                       RNAse protection analysis demonstrated approxi- <br />                                                                       mate location putative change B&amp; RNA struc- <br />                                                                       ture. The consequences RNA alteration could <br />                                                                       precisely determined structure the <br />                                                                       altered RNA species determined. We employed an <br />                                                                       RT-PCR strategy isolate partial B&amp; specific cDNA <br />                                                                       (see MATERIALS AND METHODS). <br />                                                                          The Bdc cDNA sequenced h e <br />                   1         % <br />                             .&amp;                w   1.98                mologous position 3019 (position 5&#226;&#8364;&#8482; PCR <br />                             6&#226;&#8364;&#732;:: <br />                             &lt;!*~ <br />                                    -                                  primer) nt 3357 Ser cDNA. Thereafter, <br />                                                                       diverges includes &#226;&#8364;&#339;400 nt foreign DNA (Fig- <br />                                                                       ure 8A). A database search revealed foreign DNA <br />                                                                       homologous left terminal repeat the <br />                                                                       B104 (roo) element (SCHERER     et ul. 1982). The finding <br />    FIGURE 5.--Genomic Southern analysis Bdc 3&#226;&#8364;&#8482; polymor-            left terminal repeat B104 Bdc cDNA <br /> phism. Probe used -9 +5 DNA Serlocus (see            correlates presence right terminal <br /> Figure 2). Y488 B&amp;/ TMGB control shows expected wild-type             repeat element 5&#226;&#8364;&#732; divergence point <br /> (+) 3&#226;&#8364;&#8482; I  ?&amp;polymorphic fragments ( 7 R 1%). I9B/             IX polymorphic fragment suggests the <br /> TMGB retains 1% 7R polymorphisms. II2B/ <br /> II2B shows wild-type 3&#226;&#8364;&#732; pattern. II6A/ TMGB retains 1X          Bdc allele results insertion B104 element <br /> but 7R polymorphisms. Df(3R)SQ219/ TMGB control lane              Ser. <br /> with wild-type pattern. M indicates lambda Hind111 markers;              Sequencing entire cDNA subclone revealed that <br /> corresponding sizes right ladder.                   left terminal repeat B104 incorporated into <br />                                                                       Bdc cDNA. The left terminal repeat contains poly <br /> isolatedfrom 4 to12-hr CuntonS (control) Bdc/                      A+ addition signal Bdc mRNA splices the <br /> TM6B embryos. The B&amp;/TM6B embryos, hetere                       B104 left terminal repeat, ending -400 nt B104. <br /> zygous, provided additional, internal RNA control                  An examination expected Seropen reading frame <br /> for experiment. As shown Figure 7, control                revealed presence stop codon amino acids <br /> and experimental RNAs shown lane E 1 3                 foreign sequence (Figure 8A). Thus, pre- <br /> 14 generate single protected RNA fragment                    dicted protein product encoded chimeric Bdc <br /> predicted <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span>, indicating Bdc wild-type               RNA include N-terminal972 amino acids of <br /> alleles produce identical products. Two distinct RNA                  Ser, ending justGterminal 14th EGF-like repeat, <br /> species protected E1314 fragment. The                      terminating foreign amino acids. The <br /> larger (&#226;&#8364;&#339;480 nt) corresponds ex-                    predicted Bdc protein product lacks transmembrane <br /> pected wild-typeRNA fragment. The second, lower                       domain present withinwild-typeSER, a <br /> band -300 nt represents Ser-related RNA unique                   secreted protein (Figure 8B). <br /> to Bdc. This 300-ntprotected fragment places ioca- <br /> tion divergence point IX approximately                                        DISCUSSION <br /> position 3350 Ser message, message <br /> originates fromthe Serpromoter regions. This position                   Penetrance Bdc phenotype: Both Ser DZ <br /> corresponds IX divergence point                 believed encode ligands bind NOTCH <br /> 3358 nt (see above). Alternatively, IX polymor-                theinitial steps of <br />                                                                                                signal transduction pathway <br />  370                                                          N. A. Hukriede R. J. Fleming <br />  <br />       +-       centromere <br />                                                                    Recombinantsinthisregion                      FIGURE 6.-Recombina- <br />                                                                                                               tion B&amp; region. <br />  <br />  <br />  <br />  <br />                                                                                       - <br />                                                              wild-type genomicSouthern        patterns <br />                                                                                                               Map orientation Fig- <br />                                                                                                               ure 4. Females heterozy- <br />                                                                                                               gous Y488 Bd&quot; the <br />       Y488 w+                                                                                                 distally adjacent deficiency <br />  <br />        y                           ,~,15L                               f -, <br />                                                                         I        I       I <br />                                                                                                               Df(3R)SQ219 mated to <br />                                                                                                               Df(3R)SQ219 ca/TM3, Sb <br />                                                                                                               males phenotypically <br />                                                                                                               white-, non-Stubble (non- <br /> A                                                                       I        I       I <br />                                                                                                               TM3, S b ) progeny were <br />                                                                                                               selected(see <br />                                                                                                                         text).         Re- <br />                                    I                                        I        / <br />                                                                                      -       DfSQP 7 9 )-     combination events this <br />                                                                                                               screen limited toan <br />  <br />  <br />  <br />  <br />                                                                                       - <br />                                                Serrate+                                                       -100-kb region. (A) Dis- <br />                                                                                                               tally located recombination <br />                                                                                                               events (hatched lines) are <br />       Y488 vclf. <br />  <br />        y <br />                f <br />                    *    7,R <br />                         f <br />                               ~~ <br />  <br />  <br />  <br />  <br />                               f        1 <br />                                                1:1 <br />                                                  I       I <br />                                                                         9            ,                        expected produce wild- <br />                                                                                                               type Sergenomic restriction <br />                                                                                                               patterns (8).(B) Recombi- <br />                                                                                                               nation events <br />                                                                                                                           occurring <br /> B              I        I     I        I        I        I <br />                                                                                                               proximal the <br />                                                                                                               regions B&amp; poly- <br />   I <br />                        d &quot; <br />                                            I <br />                                                      I <br />                                                                                              DfSQZ 19 )-      morphisms expected to <br />                                                                                                               generate recombinants re- <br />                Serrate+                                                                                       taining someor Bd&apos;poly- <br />           k-                                   - 100Kb          4-                                            morphic regions. <br />  <br />  <br />  <br /> (ARTAVANIS-TSAKONAS         SIMPSON1991; HEITZLER                           animals exhibit synergistic wingphenotype more <br /> and SIMPSON     1991, 1993; REBAY et al. 1991; ARTAVANIS-                       severe associated Bd&quot; Dl muta- <br /> TSAKONAS    et al. 1995; GUet al. 1995).As such, under-                      tions alone. Finally, identified region <br /> standing potential interactions                            distal Bd&quot; that, homozygous, resultsin lethality <br /> loci reveal regulatory redundantroles                        (T. T. TUNG R. J. FLEMING,       unpublished results). <br /> products. The finding Bdc&apos; resulted lethality                           Of regions identified, proximal <br /> when transheterozygous amorphic alleles Dl                                regions appear significantlyaffect viabilityof <br /> (THOMAS    et al. 1991; initial results, study) partic-                B&amp;/DI animals. <br /> ularly intriguing light antimorphic nature                            Bdc represents antimorphic Ser isoform: Our iso- <br /> the Bd&quot; mutation. Upon closer examination however,                              lation suppressors B&amp; result duplica- <br /> we thereis evidence strong interac-                       tions wild-type Ser locus clearly demonstrate that <br /> tions. The Bdc mutation described 60 years                       SdG protein product (BD&quot;) compete wild- <br /> ago (GOTTSCHEWSKI        1935) maintained                       type SER generate observed dominant pheno- <br /> heterozygous condition time. Given this,                             type. By increasing availability wild-type SER,the <br /> there selective pressure prevent ac-                         antagonistic effects BDGare diminished thewild- <br /> cumulation second site mutations (or remove                           type phenotype restored. Based criteria, Bd&quot; <br /> that arisen Bdc itself). The contrary                          fits classical <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> antimorph defined by <br /> actually likely. Onlyrecently cold-sensitive                       MULLER(1932). <br /> nature Bdc&apos; recognized, heterozygous                               We traced cause Bd&apos; mutation an <br /> dominant wing phenotype Bdc&apos; exhibits highly vari-                         insertion transposable element roo (B104; SCHERER <br /> able expressivity (THOMASet al. 1991; COUSOet al.                               et al. 1982) codingregions Ser transcript. <br /> 1995). It likely selection                         This insertion predicted generate truncatedform <br /> penetrant phenotype resulted accumulation                            ofSER lacking transmembrane intracellular do- <br /> of numerous second site mutations. We identified                           mains. The antimorphiccharacter mutation <br /> at regions co-segregatingwith Bd&apos;. The                         result competition wild <br /> first identified replacing regions proximal                           type Bdc level protein products. Since <br /> ebony. This resulted production moderately                            Ser function essential proper wing development, <br /> viable B&amp;/Dpx6 adults exhibited greatly reduced                            particularly wing margin zones (DIAZ-BENJUMEA <br /> wing blade differentiation. The second region lies prox-                        COHEN1995; COUSOet al. 1995; DE CELISet al. <br /> imal Bd&apos; Serand ebony. Replacement                           1996; DOHERTYet al. 1996), diminutionof wild-type SER <br /> region allows production robust Bdc&apos;/Dl&quot;&quot;&apos;                           brought presence BDG readily ex- <br />  !:I Flpp#l <br /> ! <br /> % <br /> 404- <br />  <br />  <br />  <br />  <br /> 201 <br />  <br /> 160 <br /> 147 <br />  <br /> 123 <br /> 110 <br />     = <br />  <br />     - <br />     - <br />     - <br />     - <br />     - <br />         N <br />  <br />          ! <br />            7 <br />                A&quot;13 <br />                - <br />  <br />  <br />  <br />  <br />                 I <br />                        E9-IO <br />                        m        Ell-13 <br />                                 A 1% <br />  <br />  <br />  <br />  <br />                                ..* <br />  <br />  <br />                                  5 <br />                                       Y <br />                                       - <br />                                           Characterization (&gt;f Sm Antimorph <br />  <br />                                          El?-14 <br />                                           A&apos; B <br />  <br />  <br />  <br />  <br />                                                L <br />                                                i <br />  <br />                                                e <br />                                                    u <br />                                                    A l3 <br />  <br />  <br />  <br />  <br />                                                           b <br />                                                                respective contributions overall S&amp; pheno- <br />                                                                                                                     371 <br />  <br />  <br />                                                                tvpes. The interesting conclusion experi- <br />                                                                ments 3&apos; polymorphisms the <br />                                                                5&apos; polymorphism appear contribute Sd&apos; phe- <br />                                                                notypes. As seen recombinant II2A, 5&apos; polymor- <br />                                                                phic region homozygous does not <br />                                                                produce detectable phenotype. Me&apos; able re- <br />                                                                7R 3&apos; polymorphism recombinant II6A <br />                                                                and <br />                                                                stock. <br />                                                                      animals behave like original Bd&apos; <br />                                                                        Finally, recombinant I9B retains IX 3&apos; <br />                                                                polymorphism does produce heterozygous <br />                                                                dominant phenotype. This finding argues that <br />                                                                I X fragment probably driven a <br />                                                                foreign promoter produce Sd&apos; phenotype. <br />                                                                   Our RNAse protection data support con- <br />                                                                clusions recombinants region <br />                      c     *                                 I spanning interruption caused roo insertion <br />                                                                (designated fragment 1%) appears actively tran- <br />    FIGURE 7.-RNAse      protection analysisof B d &apos; total RNA. scribed. This finding, coupled isolation and <br /> Lanes marked A represent RNA probe RNAse <br /> treatment. The sizeof RNA expected beslightly       characterization partial Sd&apos; cDNA, leads to <br /> larger full-length protected RNA fragments clue t o   conclude 5&apos; polymorphism associated with <br /> presence Bluescript vector sequences flanking Smspecific    active Bn&apos;; transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> polymorphism <br /> sequences.Lanesmarked &#226;&#8218;&#172;3 represent probe hybridized         itself does significantly affect transcription. The B&amp; <br /> total B&amp;/+ RNA digestedwithRNkse T I . M, p8R322-M.spI <br /> cut DNA marker lane (the figure composite multiple,    transcript originate normal Ser promoter <br /> comparable gels; hence, depicted marker sizes left   region   continue position 3358 message <br /> the ladder approximation given lane). E6     continues left terminal repeat the <br /> the distal control fragment (for relative locations fragments <br />                                                                roo element (SHEER et nl. 1982). Evidence support <br /> within Ser coding region, Figure 2 ) . E9-10 spans conclusion comes presence new <br /> 7R polymorphism. Ell-13 control fragment between <br /> the 3&apos; polymorphisms. E 1 3 1 4 spansthe I X polymor-      messages S&amp; differing fromone anotherby approxi- <br /> phism. E14 proximal control fragment. Note pres-    mately   number nucleotides wild-type St?r <br /> ence RNAse protection fragmentsin lane E1314B (see      messages presence small deletion M8- <br /> text).                                                         1 residing 5&apos; polymorphism IX <br />                                                                roo insertion site. These data support conclu- <br /> plain generation dominant Bd&apos; wing margin           sion IX S&apos;polymorphism functionally inert <br /> phenotypes.                                                    Bd&apos; genomic DNA fused 5&apos; end <br />    By combination genomic Southern analysis           IX corresponds roo sequencesthat lie down- <br /> the isolation ofBd&quot;specific          3&apos; polymorphisms,      stream truncated Sd&apos; message. <br /> found Bd&apos; mutation molecularly complex.              The BDC product: As result roo insertion, <br /> There exist distinct 3&apos; regions ( I X 7 R ) linked     predicted BD&apos;; protein truncated version the <br /> to single 5&apos; region. Given earlier arguments con-        wild-type SER protein (see Figure 8B). This protein is <br /> cerning accumulation second site mutations,          predicted include N-terminal regionsof SER <br /> is unclear whetheror presenceof 3&apos; duplication         amino acid 972, terminating addition <br /> arose simultaneously insertion roo ele-        foreign amino acids. This predicted protein is <br /> ment occurred separate event. However,           expected retain theSER signal peptide andits poten- <br /> chromosomal rearrangementsare commonly associ-                 tialcleavage site amino acid 80, DSL domain <br /> ated roo elements(MCGINNISand BECKENDOW                   (TAXet (11. 1994) comprises region capa- <br /> 1983; TSUROTA      et nl. 1989; MONTGOMFKSPI (I/. 1991)        ble binding NOTCH (R. J. FLEMING,         X. SUN and <br /> and nearly identical genomic duplication event           S. ARTA\JANIS-TSAKONAS,    unpublished results), the <br /> been associated roo insertion Anfp&quot; muta-        14 EGF-like repeats. The BD&apos;; protein lack SER <br /> tion (TAIBERT GAKRER           1994). These findings sug-  sequences Cterminal point, including cys- <br /> gest partial duplication Ser associated       teine-rich, transmembrane intracellulardomains <br /> Bd&apos; arose coincidentally roo insertion.               (FLEMINGef nl. 1990; THOMASet al. 1991). With this <br />    Our meiotic recombination screensallowed usto s e p         predicted structure, Bd&apos; potentially produce a <br /> arate intragenic segments     Bd&apos;mutation andiden-      secreted protein predicted membrane- <br /> tify segments based presence noted         bound wild-type isoform. The BD&apos;; protein beca- <br /> polymorphic regions. M&apos;e individually assess        pable nonproductive competition normal SER <br />  372 <br />  <br />  <br />  <br />  <br />    FICXRE8.-BD&apos;; coding capacity result roo element insertion. (A) Upper line indicates predicted amino acid <br /> sequence forwild-type SER protein beginningwith amino acid 968 beneath coding cDNA sequence beginningwith <br /> nucleotide 3343 (FI.EMINGrt nl. 1990). The comparable Bd; nucleotide sequence aligned wild type diverges <br /> after nucleotide 3356 rodlornologous sequence. The predicted B 8 amino acid sequence presented diverges <br /> from S P after <br />            ~   amino acid 972 foreign amino acids stop codon. Divergent sequences (roo) boxed. (&#226;&#8218;&#172;5) <br /> Cartooned representationof BD&quot; protein compared wild-type SER illustrates loss cysteine-rich, transmem- <br /> brane (TM)and intracellular domainsin mutantform. Amino terminus, H,N; carboxyl terminus, COOH; SP, signal peptide; <br /> clarkls hatched region, ,\&apos;otOlch binding domain includingDSL domain (see text); chevroned boxes, EGF-like repeats. <br />  <br /> product generate antimorphic characteristics           regulator SER expression (IRVINEand WEISCHAUS <br /> the ]In&apos;: allele.                                                1994; KIM et al. 1995). <br />    Since BD&apos;; predicted retain portion SER             With considerations mind, exist two <br /> capable NOTCH binding, delete extracellular           possible mechanisms explain phenotypic interac- <br /> cysteine-rich domain Gterminal regions,              tions noted S&amp; amorphic Dl alleles. The <br /> may capable binding NOTCH leading               model BD&quot; protein encodes inactive, <br /> to NOTCH activation. The simplest model explain               secreted form SER diffuse cell(s) <br /> the dominant-negative effects l3cp mutation sug-          withinwhich       synthesized interfere with <br /> gests N-terminal NOTCH-binding region                 NOTCH activity distance. At juxtaposition of <br /> BD&apos;; protein effectively compete wild-typeSER           dorsal ventral wing compartments, be <br /> for bindingwith NOTCH receptor. Binding BD&apos;;              possible BD&quot; competewith DL NOTCH bind- <br /> would result nonproductive interaction,             ing. Again, nonproductive association BD&quot; with <br /> reducing SER-activated signaling. Consistent           NOTCH inappropriate compartment        reduce <br /> hypothesis, increasing relative amounts wild-type         DL signaling, increasing severity wing <br /> SER, shown dosage studies, competes away               blade phenotype RdC;/DZ animals. For model to <br /> the BD&quot; effect.                                                  fit available data, hypothesize theBD&apos;; <br />    The expression activatedforms NOTCHin            form SER longerconstrained act exclusively <br /> the dorsal ventral wing compartments results cell          ventral compartment interfere <br /> proliferation theexpression genes rctin~glm       withDL signaling dorsal compartment. This <br /> and cut (DE CEIJSP/ nl. 1996; DOHERTY   e/ al. 1996). Inter-     model suggest regions residing C <br /> estingly, misexpression NOTCH ligand SER              terminal portion SER regulate compartment-spe- <br /> only produce cell proliferation expression            cific response. <br /> ruin,gl~.s.s <br />           cu/ ventral compartment. These find-           A similar distinct model does require <br /> ings suggest ability SER activate NOTCH           direct inhibition DL signaling BD&apos;; plausi- <br /> is constrained compartment-specific manner. The             ble. It proposed SER acts dorsal, <br /> mechanism SER restricted ventral activa-          NOTCH-mediated signal ventral compartment <br /> tion NOTCH presently understood. However,              inducesor reinforces production ventral <br /> a potential candidate regulation product         signal DL,which turn signals NOTCH to <br /> of gene Ji-ing~that expressed           dorsal compartment. As DL reciprocally signals to <br /> exclusively dorsal wing compartment act           dorsal compartment, servesto reinforce SER <br />                                               Characterization Ser Antimorph                                                   373 <br />  <br /> production (DE CELISet al. 1996; DOHERTY        et al. 1996).    possible truncated forms may <br /> If case, BdG/Dl wing pheno-          act mechanism (e.g., loss of <br /> type result BD&quot; competing SER               intracellular sequences), given apparent differences <br /> to reduce effectiveness SER signaling            NOTCH/SER NOTCH/DL interactions the <br /> dorsal compartment. This reduction     dorsal signal      eye wing, remains possible mecha- <br /> would expected produce   effectivereinforce-        nisms BD&apos;; SERTM truncations exert <br /> ment DL signaling ventral compartment.               dominant-negative effects differ. <br /> In situation DL signaling reduced              We gratefulto     C. SHELTONand S. WASSERMAN            forthe <br /> by approximately one-half, loss ventral           DJ(jR)SQZ219and Y488 stocks mapping data, lo I.. AKGERER for <br /> to dorsal signaling DL increase B&amp;/DZ pheno-            assistance RNAse protection Northern protocols, M. KAN <br /> typic severity relativeto associated muta-      KEI. TUNG  T. TRANG technicalassistance R. C. ANGERER, <br /> tion alone.                                                      S. ARTAVANIS-TSAKONAS,     DIEDERICI~ <br />                                                                                                R.            hwranonymous reviewers <br />                                                                  helpful suggestions manuscript. This research sup- <br />    Our current data   inconclusive ofthese       ported award National Science Foundation (MCB- <br /> two models likely correct. Using Bdo&apos;&quot;u/ol ani-         9418678). N.A.H. partially supported NationalInstitute of <br /> mals, expected levelofSER product                Health Genetics Regulation Training Grant GM07102-19. <br /> sufficient wild-typewing blade formation, ob- <br /> served animals severe DZ-like phenotypes wing                                 LITERATURECITED <br /> margin loss. This phenotype appears combination <br /> of Bdc&apos; Dl phenotypes. Since loss ofDL          AKA??, M., 1987 The molecular basis metameric pattern the <br />                                                                     Drosophila embryo. Development 101: 1-22. <br /> the ventral wing margin results wing nicks similar         ALTSCHUL,  S. F., W. GISH,W. MILLER,E. W. MYERSand D. J. LIPMAN, <br /> those seen loss SER dorsal margin (DE               1990 Basic local alignmentsearchtool.       J. Mol. Biol. 215: <br /> CELISet al. 1996; DOHERTYet aZ. 1996), experi-                 403-410. <br />                                                                  ARTAVANISTSAKONAS,    S., P. SIMPSON, <br />                                                                                                          1991 Choosing cell fate: <br /> ments notallow conclude observed wing                view Notch locus. Trends Genet. 7: 403-408. <br /> margin loss phenotype Bd&quot;/Dl animals              ARTAVANIS-TSAKONAS,S., C. DELIDAKIS   R. G. FEHON,1991 The <br /> reductionin       SER signaling, DL signaling orboth.               A&apos;otch locus cell biology neuroblast segregation. Annu. <br />                                                                     Rev. Cell Bid. 7: 427-452. <br /> Therefore, determinationas notBD&quot;              ARTAVANIS-TSAKONAS,   S., K. MATSIN) M. E. FoRnNI, 1995 Notch <br /> is SER-specific signaling inhibitor             signaling. Science 268: 225-232. <br /> general NOTCH signaling inhibitor await             AUSUREI., F. A., R. BRENT,R. E. KINC;STON, D. D. MOORE,J. D. SLID- <br />                                                                     MAN et al., (Editors), 1990 Current Profocols lMolecuZar Biology. <br /> the application sophisticated techniques.                   Greene Publishing Wiley-Interscience, New York. <br />    It worth noting anothertruncated form SER          BELT,A. L., 1971 Research notes. Dros. Inf. Sew. 46: 116. <br /> demonstrates dominant-negative properties ectop-            BRAND, A. H., N. PERRIMON,    1993 Targeted gene expression as <br />                                                                     means altering cell fates generating dominant phenn- <br /> ically expressed Drosophila ommatidial develop-              types. Development 118: 401-415. <br /> ment (SUNand ARTAVANIS-TSAKONAS         1996).A truncated        CAGAN, R. L., D. F. READY,  1989 Notchis required successive <br /> SER isoform lacking wild-type intracellular domain              cell decisions developing Drosophila retina. Genes Dev. 3: <br />                                                                       1099-1112. <br />  (called SERTM),expressed theseuenksspromoter,             COFFMAK,C. R., P. SKOGLUNU,    1%&apos;. A. HARRISand C. R. KINTER, 1993 <br /> is capable antagonizing NOTCH receptor function                   Expression extracellular deletion Xotrh diverts cell fate <br /> and generating additional photoreceptors. A similarly                Xenopus embyryos. Cell 73: 659-671. <br />                                                                  CORBIN,  V., A. M. MICHEI.SON, S. M. AHMAI&apos;K, V. NEEI.,E. AICAMO et <br /> truncated form DL (DLTM) behavesin qualitatively                al., 1991 A role Drosophila neurogenic genes inmeso- <br /> similar fashion. In eye, expression length SER           derm differentiation. Cell 67: 31 1-323. <br /> and DL forms generate similar phenotypic effects           COUSO,J. P., E. KNusr A. &quot;WINEL-ARL.~$,        1995 Serrate and <br />                                                                      wingless cooperate induce vestigial gene expression wing <br /> the expression SERTM DLTM forms (SUN                      formation Drosophila. Curr. Bid. 5: 1437-1448. <br /> and ARTAVANIS-TSAKONAS       1996). Thus,       seu-   DE CELIS, J. F., A. GARCW-BEI.LIDO, 1994 Roles Notch gene <br /> e n h s promoter eye, SER DL act interchange-             Drosophila wing morphogenesis. Mech. Dev. 46: 109-122. <br />                                                                  DE CEI.IS,J. F., A. GARCW-BF,I.I.IDO S. J. BRAY, 1996 Activation <br /> ably. During wing margin formation however, SER                  function Notch dorsal-ventral boundary wing <br /> DLhave distinctiveroles respective compart-                 imaginal disc. Development 122: 359-369. <br /> ments, produce different phenotypic effects,         DEXTER,  J. S., 1914 The analysis case continuous variation <br />                                                                      Drosophila study linkage relations. Am. Nat. 48: <br /> interchangeable   (Dm-BENJUMEA           COHEN 1995;             712-758. <br /> COUSO et aZ. 1995; DE CELISet al. 1996; DOHERTYet aZ.            DIAZ-BENJUMEA,  F. J., S. M. COHFX, 1993 Interaction between <br /> 1996). The BD&quot; SERTM truncations struc-                dorsal ventral cells imaginal disc directs wing develop- <br />                                                                     ment Drosophila. Cell 75: 741-752. <br /> turally different another. BD&quot;islikely            DIU-BENIUMEA,  F. J., S. M. COHEN,1995 Serratesignals through <br /> secreted, lacks transmembrane segment. It              Notch establish Winglescdependent organizer dorsal/ <br /> lacks evolutionarily conserved extracellular cysteine-           ventral boundary Dro.sophilawing. Development 121: 4215- <br />                                                                      4225. <br /> rich region recently shown essential          DOHERTY, D.,G. FEGER,S. YOUNGER-SHEPHERD,           L. Y. JAN Y. N. <br /> for Ser-specific signaling (Y. GU R. J. FLEMING,     un-        JAN, 1996 Deltais ventral dorsal signal complementary to <br /> published results). In contrast, SERTMis membrane-                   Serrate, Notch ligand, Drosophila wing formation. Genes <br />                                                                      Dev. 10: 421-434. <br /> bound retains entire extracellular domain. Both          FEHON,R. G., P. J. KOOH,I. REBAY, C. L. REGAN, T. Xu et al., 1990 <br /> BDGand SERTM lack intracellular domain. Whereas                  Molecular interactions protein products neu- <br />  374                                                     N. A. Hukriede R. J. Fleming <br />      rogenic loci Notch Delta, EGF-homologous genes inDro-                  SCHLITZ,W. K BENZet ul., 1988 A stable genomic source P <br />      sophila. Cell 61: 523-534.                                                     element transposase Drosophila melanogmter, Genetics 118 <br /> FLEMING,   R. J., T. N. SCOTTGALE, R.J. DIEDERICH        S. ARTAVANIS           461-470. <br />      TSAKONAS,      1990 The gene Serrate encodes putative EGF-like          RULIFSON,    E. J., S. S. BLAIR, 1995 Notch regulates wingless expres- <br />      transmembrane proteinessential proper ectodermal develop                   sion required reception paracrine wingless <br />      ment Drosophila mlanogmter. Genes Dev. 4 2188-2201.                         signal wing margin eurogenesis Drotophila. Develop- <br /> GARCIA-BELLIDO,      A., 1975 Genetic control wing disc development              ment 121: 2813-2824. <br />     Drosophila. CIBA Foundation Symp. 29: 161- 183.                         RUOHOLA,H., K. A. BREMER,           D. BAKER,J. R. SWEDLOW, L. Y. JAN et <br /> GOTTSCHEWSKI,       G., 1935 New mutants: report G. Gottschewski.                al., 1991 Role neurogenic genes establishment follicle <br />     Dros. Inf. Serv. 4: 14-16.                                                      cell fate oocyte polarity oogenesis Drosophila. Cell <br /> GREENWALD,      I., G. M. RUBIN,1992 Making adifference:                    66: 433-449. <br />     role cell-cell interactions inestablishing separate identities      ST. JOHNSTON, D., C. NLJSSLEIN-VOLHARU,          1992 The origin of <br />     equivalent cells. Cell 68: 271-281.                                             pattern polarity Drosophila embryo. Cell 68: 201-219. <br /> Gu, Y., N. A. HUKRIEDE      R. J. FLEMING,   1995 Serrate expression       SAMBROOK,    J., E. F. FRITSCH    T. MANIATIS,  1989 Molecular Clon- <br />     functionally replace Delta activity neuroblast segrega-              ing: A Laboratory Manual, Ed. 2. Cold Spring Harbor Laboratory <br />     tion Drosophila embryo. Development 121: 855-865.                        Press, Cold Spring Harbor, NY. <br /> HASHIMOTO,     C., K. L. HUDSON     K. V. ANDERSON,    1988 The Toll       SANGER, F.,S. NICKLENand A. R. Coul.sON, 1977 DNA sequencing <br />     gene Drosophila, required dorsal-ventral embryonic po-                   chain termination inhibitors. Proc. Natl. Acad. Sci. USA 74: <br />     larity, appears encode transmembrane protein. Cell 52:                     5463-5467. <br />     269-279.                                                                   SCHERER,    G., C. TSCHL~DI,    j . PERERA,H. DEI.K~S andV. PIRROTTA, <br /> HEITLIXR,P., P. SIMPSON,1991 The choice cell fate                     1982 B104, new dispersed repeated gene family Drosophila <br />     epidermis Drosophila. Cell 64: 1083-1092.                                    melanogmterand analogies retroviruses. J. Mol. Biol. 157: <br /> HEITTIER, P.,and P. SIMPSON, 1993 Altered epidermal growth fac-                     435-451. <br />     tor-like sequences provide evidencefor role Notch recep-         SHELLENBARGER,       D. L., J. D. MOHLER,1978 Temperaturesensi- <br />     tor cell fate decisions. Development 117: 1113-1123.                         tive periods autonomy pleiotropic effects 1(1)Ntsl, a <br /> INGHAM,   P. W., 1988 The molecular genetics embryonic pattern                   conditional Notch lethal Drosophila. Dev. Biol. 6 2 432-446. <br />     formation Drosophila. Nature 335 25-34.                                 SHELTON,    C . A,, S. A. WASSEKMAN,    1993 pelk encodes protein <br /> IRVINE,K. D., E. WIESGHAUS,        I994 f ~ n g e a, boundary-specific          kinase required establish dorsoventral polarity Drosoph- <br />     signaling molecule, mediates interactions dorsal                    ila embryo. Cell 72: 515-525. <br />     ventral cells Drosophila wing development. Cell 79:                 SWTH, J. B., S. B. CARROLL,1992 Regulation proneural <br />     595-606.                                                                        gene expression cell fate neuroblast segregation in <br /> KIM,J., K. D. IRVINE   S. B. CARROLL,  1995 Cell recognition, signa1            Drosophila embryo. Development 114 939-946. <br />     induction, symmetrical gene activation dorsal-ventral           SPEICHER,   S. A., U. THOMAS,    U. HINZ E. KNUST, 1994 The Serrate <br />     boundary developing Drosophila wing. Cell 82: 795-802.                   locus Drosophila role morphogenesis wing <br /> KOPAN, R.,J. S. NYE H. WEINTRAUB,         1994 The intracellular do-            imaginal discs: control cell proliferation. Development 120: <br />     main mouse Notch: constitutively activated repressor myo-               535-544. <br />     genesis directed basic helix-loophelix region MyoD.              SUN, X., S. ARTAVANIS-TSAKONAS,        1996 The intracellular dele- <br />     Development 120 2385-2396.                                                      tions DELTA SERRATE define dominant negative forms <br /> LAWRENCE,    P.A., G.MORATA, 1994 Homeobox genes:                         Drosophila Notch ligands. Development 122: 2465-2474. <br />     function Drosophila segmentation pattern formation.        Cell     TABATA,   T., T. B. KORNBERG,   <br /> </body></html>