<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2007 Genetics Society America <br /> DOI: 10.1534/genetics.106.066027 <br />  <br />  <br />  <br />   Impact Transcriptional Properties Essentiality Evolutionary Rate <br />  <br />       Jung Kyoon Choi,* Sang Cheol Kim,&#226;&#8364;&#160; Jungmin Seo,* Sangsoo Kim1,&#226;&#8364;&#161; Jong Bhak*,1,2 <br />    *Korean BioInformation Center, KRIBB, Daejeon 305-806, Korea, &#226;&#8364;&#160;Department Applied Statistics, Yonsei University, Seoul 120-749, <br />                   Korea &#226;&#8364;&#161;Department Bioinformatics Life Science, Soongsil University, Seoul 156-743, Korea <br />                                                    Manuscript received September 19, 2006 <br />                                                   Accepted publication October 14, 2006 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 We characterized general transcriptional activity variability eukaryotic genes global <br />              expression profiles human, mouse, rat, fly, plants, yeast. The variability shows higher degree of <br />              divergence distant species, implying closely related phenotypic evolution, than <br />              activity. More specifically, transcriptional variability true indicator of <br />              evolutionary rate. If rule effect translational selection, operate yeast, <br />              apparent slow evolution highly expressed genes attributed low variability. <br />              Meanwhile, rapidly evolving genes acquire high level transcriptional variability contribute <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />              phenotypic variations. Essentiality correlated variability, activity. We show <br />              indispensable highly interactive proteins tend present high abundance maintain low <br />              variability. Our results challenge current theory highly expressed genes essential evolve <br />              slowly. Transcriptional variability, transcriptional activity, common indicator of <br />              essentiality evolutionary rate, contributing correlation variables. <br />  <br />  <br />  <br />  <br /> E    VOLUTION gene expression, long <br />        subject great (King Wilson <br /> 1975), studied genomic scale the <br />                                                                          highly expressed. To rule the <br />                                                                          direct effect essentiality evolutionary rate, they <br />                                                                          argued basis following hypothetical <br /> help rapidly growing microarray genome                            relations: <br /> sequence data (Enard et al. 2002; Oleksiak et al. <br />                                                                          1. Essentiality transcriptional activity: Each protein <br /> 2002; Meiklejohn et al. 2003; Ranz et al. 2003; Rifkin <br />                                                                             molecule phenotypic <br /> et al. 2003; Khaitovich et al. 2004; Denver et al. 2005). <br />                                                                             contribution organism&#226;&#8364;&#8482;s fitness. Under this <br /> Of particular importance, expression level been <br />                                                                             hypothesis, proteins phenotypic <br /> believed best indicator evolutionary <br />                                                                             contribution higher levels active <br /> rate encoded proteins. Highly expressed genes were <br />                                                                             molecules cell. However, validity this <br /> found evolve slowly bacteria mammals <br />                                                                             hypothesis highly questionable genes involved in <br /> (Sharp 1991; Duret Mouchiroud 2000; Pal et al. <br />                                                                             functions transcriptional regulation, ligand <br /> 2001; Herbeck et al. 2003; Urrutia Hurst 2003; <br />                                                                             binding, signal transduction required only <br /> Subramanian Kumar 2004; Drummond et al. <br />                                                                             small quantities vital the <br /> 2005). In addition, recently emerged gov- <br />                                                                             organism. High abundance does necessarily <br /> erning factor apparent relationships be- <br />                                                                             mean high fitness effect. <br /> tween evolutionary rate important genomic <br />                                                                          2. Transcriptional activity selective pressure: Highly <br /> features. Specifically, influences protein&#226;&#8364;&#8220;protein <br />                                                                             expressed genes prefer translationally efficient <br /> interactions dispensability evolutionary rate <br />                                                                             codons, leads slow rate nucleotide <br /> have disputed grounds effects <br />                                                                             sequence changes (Akashi 2001, 2003; Akashi and <br /> may confounded gene expression level (Hirsh <br />                                                                             Gojobori 2002). Recently, using Saccharomyces as <br /> and Fraser 2001; Fraser et al. 2002; Bloom Adami <br />                                                                             model organism, Drummond et al. (2005) argued <br /> 2003; Pal et al. 2003; Bloom Adami 2004; Fraser <br />                                                                             selection act codon preference (for <br /> and Hirsh 2004; Wall et al. 2005). In words, <br />                                                                             translational accuracy) amino acid sequence <br /> when expression level statistically controlled, the <br />                                                                             (for translational robustness) minimize detri- <br /> effects decreased disappeared. Pal et al. (2003) ar- <br />                                                                             mental effects protein misfolding. Since they <br /> gued essential proteins evolve slowly only <br />                                                                             experience translation events, highly expressed <br />   1 <br />                                                                             genes subject stronger selective pres- <br />    These authors contributed equally work. <br />   2 <br />                                                                             sure. However, hardly applicable higher eu- <br />    Corresponding author: Korean BioInformation Center, Korea Research <br /> Institute Bioscience Biotechnology, 52 Eoeun-dong, Yooseong-         karyotes. First, lack translational selection on <br /> goo, Daejeon 305-333, Korea. E-mail: jong@kribb.re.kr                       codon preference larger genomes well-known <br />  <br /> Genetics 175: 199&#226;&#8364;&#8220;206 ( January 2007) <br />  200                                                   J. K. Choi et al. <br />  <br />    phenomenon (Shields et al. 1988; Sharp et al.                 on, defined general transcriptional activity. Coefficient <br />    1995; Akashi 1997, 2001; dos Reis et al. 2004).               variation (CV), ratio standard deviation (SD) <br />    Second, recent study (Wright et al. 2004),            mean, adopted measure general tran- <br />    shown number tissues            scriptional variability. CV used measure of <br />    gene expressed, total trans-           stochastic fluctuation &#226;&#8364;&#732;&#226;&#8364;&#732;noise&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene expression <br />    lation events tissues, important determi-        (Elowitz et al. 2002; Ozbudak et al. 2002; Blake et al. <br />    nant evolutionary rate Arabidopsis. We expect           2003; Raser O&#226;&#8364;&#8482;Shea 2004). According the <br />    case multicellular         models stochastic gene expression, noise in- <br />    organisms.                                                    crease transcript decreases (Hasty <br />                                                                  Collins 2002; Swain et al. 2002). Low expression <br />   Here propose alternative relations:                     level coupled high physiological genetic <br />                                                                  variation stochastic variation. Small changes in <br /> 1. Essentiality transcriptional variability: One         proteins normally expressed a <br />    anticipate essential proteins low            low level greater impact cell or <br />    genetic physiological variation. To perform core          organism large changes proteins <br />    functions cell, constitutively required       normal expression level high. Therefore, may <br />    different individuals physiological conditions.        close correspondence tran- <br />    This leads expectation transcriptional            scriptional activity variability. <br />    variability, expressed genetic, phys- <br />    iological, random variation different levels, may <br />    better indicator essentiality transcrip- <br />                                                                                 MATERIALS AND METHODS <br />    tional activity. To test hypothesis, compared <br />    transcriptional variability genes associated         Measuring transcriptional activity variability: Supple- <br />    essential gene ontology (GO) categories            mental Table 1 http://www.genetics.org/supplemental/ shows <br />    genes. For experimental validations, pro-           compendium multispecies microarray data obtained <br />                                                                  GEO database (http://www.ncbi.nlm.nih.gov/geo/). <br />    tein dispensability interaction data yeast <br />                                                                  The raw Affymetrix data used log transformation. <br />    proteins used measure essentiality.             A flooring lowest 1% ceiling 1% was <br /> 2. Transcriptional variability selective pressure: The       applied. <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> microarray data sets arrays, <br />    presence high variation individuals             computed mean CV expression values each <br />    population indicate action weak purifying          gene. To integrate different scales microarray data sets, we <br />                                                                  scaled mean values zij &#194;&#188; &#195;&#176;mij   mj &#195;&#382;=SD&#195;&#176;f mj &#195;&#382;, mij is <br />    selection gene. One envision those <br />                                                                  mean expression level ith gene jth data set, while <br />    genes evolved possess high transcriptional          mj SD&#195;&#176;f   mj &#195;&#382; indicate average standard deviation of <br />    variability expressed specific conditions. In        mean values jth data set. Finally, global <br />                                                                                                                    Pn transcrip- <br />    contrast, genes strong purifying selection          Pn properties gene obtained j&#194;&#188;1 zij =n and <br />                                                                  tional <br />    likely exhibit constant level expression          j&#194;&#188;1 vij =n, namely, mean z-score mean CV n <br />                                                                  data sets. These final estimates transcriptional activity and <br />    various conditions, maintaining limited level of <br />                                                                  variability downloaded supplemental website, <br />    variability. This explains observation corre-        http://centi.kribb.re.kr, raw microarray <br />    lation evolutionary rate tissue specific-         data. <br />    ity (Wright et al. 2004). Duplicates used               Evolutionary conservation index: To provide measure of <br />    good source quantitative measure            evolutionary rate macroevolutionary timescale, used <br />                                                                  HomoloGene database (http://www.ncbi.nlm.nih.gov/ <br />    correlation expression divergence se- <br />                                                                  HomoloGene). We assigned gene conservation score <br />    quence divergence (Gu 2004; Gu et al. 2004, 2005).            defined number orthologs 18 eukaryotic <br />    Drummond et al. (2005) demonstrated diver-               species. Thus, score ranges 1, meaning gene is <br />    gence transcriptional activity correlated             specific species, 18, meaning gene conserved <br />    sequence divergence duplicates yeast.              18 species. Rapidly evolving genes, including <br />                                                                  recently emerged genes (Long et al. 2003), probably be <br />    However, speculated divergence transcrip- <br />                                                                  conserved organisms low <br />    tional variability correlate better se-            conservation score. On contrary, slowly evolving genes, <br />    quence divergence eukaryotes. We tested                ancient genes, high conservation score. This <br />    second hypothesis genomewide analysis fol-             score expected reflect selective pressure differently than <br />    lowed duplicates studies.                              substitution rates. As such, used conservation score as <br />                                                                  inverse estimate evolutionary rate. We refer the <br />                                                                  evolutionary conservation index (ECI). <br />   To provide experimental support hypothe-                Measuring substitution rates: Orthologous relationships <br /> ses, use microarray database extract             human, mouse, rat proteins obtained from <br /> general transcriptional properties gene. An av-          HomoloGene database. Synonymous (dS) nonsynon- <br />                                                                  ymous (dN) substitution rates calculated align- <br /> erage expression level wide range biological         ments coding sequences using PAML&#226;&#8364;&#8482;s codonml (Yang <br /> conditions, including different individuals, times, tis-         1997). PAML&#226;&#8364;&#8482;s baseml used estimate Jukes&#226;&#8364;&#8220;Cantor <br /> sues, disease states, environmental conditions,           distances orthologous nucleotide sequences. A total <br />                                                 Gene Expression, Essentiality Evolution                                           201 <br />  <br /> of 12,671 human&#226;&#8364;&#8220;mouse 7154 mouse&#226;&#8364;&#8220;rat ortholog pairs                                             TABLE 1 <br /> with observed expression <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> produced appropriate pro- <br /> tein nucleotide sequence alignments PAML input.                   Correlation transcriptional properties orthologs <br /> Paralogs identified HomoloGene database. <br /> We performed all-against-all comparisons paralogs                           Human      Mouse      Rat      Fly    Plant    Yeast <br /> to estimate dN, dS, transcriptional activity difference, and <br /> transcriptional variability difference. Transcriptional-activity         Human                 0.460     0.428   0.213    0.246    0.165 <br /> divergence measured jz1   z2j, z1 z2 mean           Mouse       0.469               0.454   0.240    0.263    0.190 <br /> transcriptional activity paralogous genes. Transcrip-             Rat         0.443     0.507             0.256    0.258    0.237 <br /> tional-variability divergence measured jCV1   CV2j,               Fly         0.352     0.380     0.404            0.193    0.164 <br /> where CV1 CV2 mean transcriptional variability            Plant       0.318     0.395     0.387   0.418             0.248 <br /> paralogous genes.                                                        Yeast       0.385     0.446     0.467   0.511    0.511 <br />    Principal-component analysis-based regression: Although <br /> similar principle, representative principal-compo-               For 31,046 HomoloGene entries expression pattern <br /> nent analysis (PCA)-based regression methods different              observed species, specieswise correla- <br /> purposes. Principal-component regression (PCR) determines                tions transcriptional activity (below diagonal) tran- <br /> the linear combinations predictors explain           scriptional variability (above diagonal) computed. <br /> the variation predictors, ignoring response vari-           To rule effect sample-size bias, randomly se- <br /> able. Partial least-squares regression (PLS), contrast, finds         lected 20 data sets human, mouse, rat. However, <br /> the linear combinations best explain response,                  use selected data did differences that <br /> yielding different results according type response         data, underscoring measures reflect in- <br /> (in case, ECI, dN, dS, fitness effect, protein in-          herent biological features regard sample size. <br /> teraction number). Although practically produces similar              Spearman&#226;&#8364;&#8482;s rank correlation coefficient used. <br /> results PLS, PCR designed determine influence <br /> of predictors response. In fact, PCR failed to <br /> distinguish contribution predictors (transcriptional          properties gene. To test this, compar- <br /> activity variability) cases analysis. The R <br />                                                                          isons z-score CV species the <br /> package &#226;&#8364;&#732;&#226;&#8364;&#732;pls&#226;&#8364;&#8482;&#226;&#8364;&#8482; used perform PLS. We scaled pre- <br /> dictors zero mean <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> variance conducting PLS.            assumption transcriptional properties should <br />    Functional category analysis: For GO category               evolutionarily conserved. Since human, mouse, and <br /> .10 genes, computed normalized transcriptional vari-                rat microarray data sets other <br /> ability (z-score) CVs members category. <br />                                                             p&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;   species (supplemental Table 1 http://www.genetics. <br /> The z-score defined z &#194;&#188; &#195;&#176;vGO   vall &#195;&#382;=&#195;&#176;SD&#195;&#176;vg    GO &#195;&#382;= nGO &#195;&#382;, <br />                                                                          org/supplemental/), randomly selected 20 data sets <br /> where vGO average CV genes GO category, <br /> vall average CV genes species, SD&#195;&#176;vgGO &#195;&#382; the <br />                                                                          species remove sample-size effect. For <br /> standard deviation genes category, nGO         HomoloGene entries transcriptional properties <br /> number genes category. The z-scores listed         observed species, calculated <br /> each species supporting information available http://          correlation coefficient species pair and <br /> centi.kribb.re.kr. <br />              P      For category, obtained specieswise <br />                                                                          striking correlations cases (Table 1). The <br /> z-score nk&#194;&#188;1 zk =n, zk indicates z-score kth <br /> species n $ 3. The specieswise z-scores given sup-           transcriptional activity conserved yeast to <br /> plemental Table 4 http://www.genetics.org/supplemental/.              human. Whereas maintaining high level similarity <br /> The significant categories total 100 genes           human, mouse, rat, variability shows <br /> across species selected presentation supplemen-          degree divergence distant species. Us- <br /> tal Tables 2 3 http://www.genetics.org/supplemental/. <br />                                                                          ing data human, mouse, rat resulted <br />                                                                          patterns. These findings suggest our <br />                                                                          measures transcriptional activity variability re- <br />                                                                          flect inherent biological features highly con- <br />                RESULTS AND DISCUSSION <br />                                                                          served orthologs, remaining free sampling <br />    Evolution general transcriptional properties: We                   problems. Moreover, higher level divergence in <br /> collected Affymetrix microarray data species                 variability measured activity implies <br /> which considerable expression profiles                       transcriptional variability play impor- <br /> have produced (supplemental Table 1 http://                      tant role evolution phenotypic variations. <br /> www.genetics.org/supplemental/). For example, 177                           Transcriptional activity vs. transcriptional variability: <br /> different data sets containing 16,446 HomoloGene                         To address main question, essentially needed to <br /> entries 4136 arrays gathered human ex-                      measure correlations transcriptional prop- <br /> pression profile. The mean CV expression values                   erties various response variables, estimates <br /> were obtained gene. The mean value scaled                   essentiality evolutionary rate. Spearman&#226;&#8364;&#8482;s rank <br /> to z-score. The average z-score CV computed                   correlation coefficient used robust to <br /> across data sets species represent general               outliers able properly handle different scales of <br /> transcriptional activity variability, respectively.                  various estimates. We denote correlation tran- <br />    We needed check expression profiles used                    scriptional activity Ra transcriptional <br /> in analysis contained sufficient data                    variability Rv. A primary concern analysis <br /> to convey information universal transcriptional                   association transcriptional activity and <br />  202                                                        J. K. Choi et al. <br />  <br />                                                               TABLE 2 <br />                 Comparative analysis influences transcriptional activity variability essentiality in <br />                                                         yeast mouse <br />  <br />         Species          Ra          Rv           Rajv         Rvja            Ca              Cv             PCa          PCv <br />                                                             Fitness effect <br />         Yeast          0.152        0.280        0.057        0.244         0.698          15.966***         21.45        78.55 <br />                                                               Lethality <br />         Mouse          0.103        0.153        0.015        0.115            0.621        6.972***         36.88        63.12 <br />                                                     Protein&#226;&#8364;&#8220;protein interaction <br />         Yeast          0.123        0.215        0.049       0.184        0.132            10.786***         18.30        81.70 <br />            Transcriptional activity (a) variability (v) predictor variables. Fitness effect, lethality, protein&#226;&#8364;&#8220; <br />         protein interaction response variables. Ra Rv denote Spearman&#226;&#8364;&#8482;s rank correlation. Rajv means Ra <br />         controlling v Rvja means Rv controlling a. Ca Cv t-values regression coefficients from <br />         multivariate regression analysis. The significance t-values represented Tables 2&#226;&#8364;&#8220;5 ***P , 10 9, <br />         **P , 10 6, *P , 10 3. PCa PCv mean percentage contribution predictors first <br />         principal component estimated PCA-based regression. The statistics underlined signif- <br />         icantly greater predictor other. Italics used statistics positive (or negative) <br />         negative (or positive) values expected. <br />  <br />  <br /> variability. As mentioned Introduction,                   Mouse Genome Informatics (MGI) database <br /> seems correspondence them. We                          (http://www.informatics.jax.org). We selected genes with <br /> observed pattern expected species (data              <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>knockout</span> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> lethality (MP:0005373, <br /> not shown). This led use partial correlation               &#226;&#8364;&#732;&#226;&#8364;&#732;lethality postnatal&#226;&#8364;&#8482;&#226;&#8364;&#8482;; MP:0005374, &#226;&#8364;&#732;&#226;&#8364;&#732;lethality embryonic/ <br /> with measure correlation transcrip-                perinatal&#226;&#8364;&#8482;&#226;&#8364;&#8482;). The phenotype database included 1427 <br /> tional activity (or variability) response variables           essential genes (lethality &#194;&#188; 1) 1956 nonessential <br /> when controlling transcriptional variability (or                   genes (lethality &#194;&#188; 0) according criteria. Ra and <br /> activity). We denote Rajv (or Rvja). We employed            Rv fitness effect lethality estimated <br /> multivariate regression compare influences               (Table 2). We observed significant partial correlations <br /> two predictors response variables, estimated              transcriptional variability (Rvja &#194;&#188;  0.244  0.115) <br /> the regression coefficient slope (Ca Cv).                       jRvj . jRaj. On contrary, transcriptional activity <br />    According Drummond et al. (2006), noisy variables                showed weak correlations (Rajv &#194;&#188; 0.057 0.015). The <br /> may result spurious partial correlations. In                   regression methods confirmed disproportionate <br /> regard, suggest use PCA-based regression                 contribution transcriptional variability fitness effect <br /> approach. However, original correlations,                lethality. Hubs protein networks known be <br /> Ra Rv, higher other,               essential, prompted use number pro- <br /> noise problem change conclusion based                  tein interactions estimate essentiality (He and <br /> the partial correlations. Therefore, simply ask                 Zhang 2006). The trend data <br /> if jRaj . jRvj (or jRvj . jRaj) Rajv (or Rvja)                 (Table 2), good agreement pre- <br /> significant. However, use PCA-based                  viously reported negative relationship genetic <br /> regression method suggested. We present per-                    variation gene expression number protein <br /> centage contributions transcriptional activity               interactions (Lemos et al. 2004). It shown that <br /> variability principal component PCa                relationship confounded gene expression <br /> PCv following sections.                                         level. <br />    First hypothesis&#226;&#8364;&#8221;essentiality transcriptional                       To extend conclusion taxa, functional <br /> variability: To estimate essentiality, adopted data                 category analysis carried out. For GO category, <br /> from yeast mouse deletion experiments. Growth                      average CV genes category obtained, <br /> rates yeast deletion strains measured array-             normalized z-score species, converted <br /> based method (Giaever et al. 2004). Using data set,               specieswise z-score (see materials methods). <br /> Wall et al. (2005) recently reported positive relation-              Significantly negative z-scores indicate genes <br /> ship dispensability evolutionary rate. As               category exhibit low variability relative the <br /> inverse measure essentiality, used fitness effect,           average genes species. Supplemental Table 2 <br /> f(i), 1   g(i)/g(max), g(i) growth rate             http://www.genetics.org/supplemental/ lists GO <br /> the strain gene deleted, g(max) maxi-                terms significantly low z-scores supplemental <br /> mal growth rate (Hirsh Fraser 2001). Mouse genes                   Table 3 http://www.genetics.org/supplemental/ shows <br /> subject deletion experiments obtained                     significantly high z-scores (the list the <br />                                            Gene Expression, Essentiality Evolution                                          203 <br />  <br />                                                              TABLE 3 <br />                    Comparative analysis influences transcriptional activity variability the <br />                                              evolutionary conservation index <br />  <br />         Species        Ra           Rv         Rajv         Rvja            Ca               Cv            PCa         PCv <br />                                                                ECI <br />         Human        0.207        0.242       0.074        0.146        0.051            29.981***        33.13       66.88 <br />         Mouse        0.245        0.312       0.053        0.207        2.634            32.972***        39.58       60.42 <br />         Rat          0.233        0.273       0.080        0.166        3.739*           21.267***        39.85       60.15 <br />         Fly          0.259        0.296       0.099        0.178        0.383            23.521***        37.36       62.64 <br />         Plant        0.260        0.348       0.046        0.244        5.543**          24.293***        40.13       59.87 <br />         Yeast        0.294        0.196       0.248        0.111       12.364***          9.661***        53.78       46.22 <br />           Transcriptional activity (a) variability (v) predictor variables ECI response variable. See <br />         Table 2 legend details. <br />  <br />  <br /> specieswise z-scores given supplemental Table 4,           related species. Therefore, calculated be- <br /> the species-specific z-scores available supple-           tween mammalian orthologs compared to <br /> mental website, http://centi.kribb.re.kr). We              ECI data. Strong inverse correlations found: <br /> the GO categories supplemental Table 2 biased                 Spearman&#226;&#8364;&#8482;s R &#194;&#188;  0.362&#226;&#8364;&#8220;  0.420 (P-values &gt;10 16). <br /> toward essential cellular processes transcrip-               Moreover, correlations ECI protein&#226;&#8364;&#8220;protein <br /> tional translational regulation, protein folding             interactions (Spearman&#226;&#8364;&#8482;s R &#194;&#188; 0.224) dispensability <br /> transport, protein catabolism, protein complexation,                 (Spearman&#226;&#8364;&#8482;s R &#194;&#188;  0.270) compatible the <br /> RNA processing, etc. In terms cellular compo-                 reported correlations substitution rates protein&#226;&#8364;&#8220; <br /> nent, genes located nucleus, cytoplasm,                   protein interactions (Spearman&#226;&#8364;&#8482;s R &#194;&#188;  0.21, Fraser <br /> mitochondrion, Golgi apparatus low tran-                    et al. 2002) dispensability (Spearman&#226;&#8364;&#8482;s R &#194;&#188; 0.230, <br /> scriptional variability. In contrast, supplemental Table 3           Wall et al. 2005). However, using different reference <br /> is enriched GO terms associated extracel-               species computation substitution rates has <br /> lular communication immune response, cell&#226;&#8364;&#8220;                   fueled arguments association protein&#226;&#8364;&#8220; <br /> cell signaling, cell adhesion, surface receptors, growth             protein interactions dispensability evolution- <br /> factor activity, hormone activity, chemotaxis, etc. In           ary rate ( Jordan et al. 2003; Pal et al. 2003; Wall et al. <br /> same context, genes located extracellular                 2005). In contrast, ECI expected offer global <br /> space plasma membrane high variability.                     unified measure evolutionary rate diverse <br /> Notably, findings strikingly consistent               organisms spanning long evolutionary time period. <br /> the results recent study Chuang Li (2004).                  Second hypothesis&#226;&#8364;&#8221;transcriptional variability and <br /> Most GO categories reported low sub-                  selective pressure: Tables 3&#226;&#8364;&#8220;5 statistical anal- <br /> stitution rates list GO categories           ysis results regarding relationship evolu- <br /> with variable genes, vice versa. Likewise,            tionary rate transcriptional properties. First, a <br /> is remarkable overlap GO categories                    genomewide analysis, investigated influence of <br /> with high substitution rates highly                   transcriptional activity variability ECI. As shown <br /> variable genes. These findings suggestive cor-                Table 3, correlation, partial correlation, multi- <br /> respondence transcriptional variability                  variate regression, PCA-based regression analyses <br /> substitution rate.                                                   indicate influence variability much <br />    Evolutionary conservation macroevolutionary                  stronger activity. This tendency holds true <br /> timescale: As study spans long evolutionary time              use traditional substitution rates between <br /> from yeast human, needed estimate evolution-                mammalian orthologs (Table 4). The exception <br /> ary rates differently substitution rates                occurred yeast discussed later. Using ECI the <br /> paired nucleotide sequences. One solution count               response variable, different statistical techniques <br /> the number genomes gene present.                  showed pattern, favor transcriptional <br /> This approach successfully used previous                  activity yeast variability species. <br /> studies relationships evolutionary                 Moreover, levels statistics comparable <br /> rate protein&#226;&#8364;&#8220;protein interactions protein dis-                various species (e.g.,  0.1 , Rvja ,  0.25,  21 , <br /> pensability ( Jordan et al. 2002, 2003; Wuchty et al.                Cv ,  33). <br /> 2003).                                                                  Next, extracted paralog information the <br />    The ECI produced results consistent                 HomoloGene database. Accelerated evolution dupli- <br /> conventional substitution rates. Substitution rates be-              cate gene expression shown yeast fruit <br /> tween orthologs reliably used distantly                fly (Gu et al. 2004). Here, compared transcriptional <br />  204                                                         J. K. Choi et al. <br />  <br />                                                                TABLE 4 <br />                Comparative analysis influences transcriptional activity variability substitution <br />                                             rates mammalian orthologs <br />  <br />         Species pair           Ra           Rv       Rajv         Rvja             Ca            Cv         PCa        PCv <br />                                                                   dN <br />         Human              0.186       0.201        0.079        0.111           6.341***    18.425***     41.52      58.48 <br />         Mouse              0.222       0.252        0.079        0.146           7.389***    21.011***     44.01      55.99 <br />         Mouse              0.210       0.251        0.065        0.155           6.380***    15.110***     45.21      54.79 <br />         Rat                0.162       0.208        0.042        0.139           5.202**     12.695***     43.63      56.37 <br />                                                                   dS <br />         Human              0.093       0.165        0.012        0.137           1.034       16.001***     30.83      69.17 <br />         Mouse              0.075       0.128        0.010        0.104           0.628       11.695***     36.30      63.70 <br />         Mouse              0.075       0.133        0.014        0.111           0.028        8.676***     38.27      61.73 <br />         Rat                0.093       0.132        0.014        0.095           2.492        8.157***     41.93      58.07 <br />                                                       Jukes&#226;&#8364;&#8220;Cantor distance <br />         Human              0.179       0.201        0.070     0.116      4.541*              19.266***     39.35      60.65 <br />         Mouse              0.196       0.234        0.059     0.143      4.403*              20.752***     41.97      58.03 <br />         Mouse              0.180       0.230        0.043     0.152      3.470*              14.960***     42.63      57.37 <br />         Rat                0.148       0.188        0.039     0.124      3.738*              12.169***     42.10      57.90 <br />           Substitution rates dN, dS, Jukes&#226;&#8364;&#8220;Cantor distance calculated described materials and <br />         methods. Transcriptional activity (a) variability (v) predictor variables substitution rates are <br />         response variables. See Table 2 legend details. <br />  <br />  <br /> variability duplicates singletons                 plants yeast compared higher eukaryotes. <br /> the species obtained data. As shown                         We asked sequence divergence, measured <br /> in supplemental Figure 1 http://www.genetics.org/                     dN dS, better explained transcriptional- <br /> supplemental/, distributions shifted high                     activity divergence (jz1   z2j, z1 z2 mean the <br /> CV values gene duplications, indicating increased                   transcriptional activity paralogous genes) by <br /> transcriptional variability duplicates. We                    transcriptional-variability divergence (jCV1   CV2j, <br /> observed overall CV values relatively lower                  CV1 CV2 mean transcriptional variability <br />  <br />  <br />                                                                TABLE 5 <br />          Comparative analysis influences transcriptional-activity divergence transcriptional-variability <br />                               divergence sequence divergence paralogous genes <br />  <br />         Species        Rad           Rvd         Radjvd         Rvdjad            Cad          Cvd        PCad        PCvd <br />                                                                   dN <br />         Human          0.126        0.258         0.139         0.264            2.443       3.322*       20.73       79.27 <br />         Mouse          0.140        0.230         0.119         0.219            4.489       7.131***     23.19       76.81 <br />         Rat            0.174        0.157         0.093         0.053            1.466      19.670***     13.80       86.20 <br />         Fly            0.057        0.152         0.358         0.382            3.333      10.404***      4.14       95.86 <br />         Plant          0.348        0.351         0.017         0.048            1.338       3.939*       34.71       65.29 <br />         Yeast          0.132        0.051         0.124         0.023            0.037       0.907        44.21       55.79 <br />                                                                    dS <br />         Human          0.176        0.298         0.108          0.265           0.749       3.140         1.55       98.45 <br />         Mouse          0.218        0.250         0.005          0.126           1.721       3.274        29.65       70.35 <br />         Rat            0.164        0.179         0.059          0.093           0.764      18.286***     19.57       80.43 <br />         Fly            0.288        0.354         0.228          0.310           0.812       6.031**      25.86       74.14 <br />         Plant          0.270        0.268         0.036          0.013           2.920       2.338        53.52       46.48 <br />         Yeast          0.141        0.097         0.105          0.026           0.721       0.968        49.22       50.78 <br />           Transcriptional-activity divergence (ad) transcriptional-variability divergence (vd) predictor var- <br />         iables. Synonymous substitution rate (dS) nonsynonymous substitution rate (dN) response variables. <br />         Transcriptional-activity divergence measured jz1   z2j, z1 z2 mean transcriptional activity of <br />         paralogous genes. Transcriptional-variability divergence measured jCV1   CV2j, CV1 CV2 <br />         mean transcriptional variability paralogous genes. See Table 2 legend details. <br />                                           Gene Expression, Essentiality Evolution                                              205 <br />  <br /> of paralogous genes). For human, mouse, fly,             transcriptional properties shown remarkably <br /> all results supportive hypothesis (Table 5).      conserved evolutionary processes, vari- <br /> In case rat, correlation partial         ability showed higher degree divergence between <br /> correlation analysis failed detect specific trend,      distant species. Transcriptional variability be <br /> regression analysis results strongly support hypoth-         related phenotypic variations more <br /> esis. The signals relatively weak plants yeast;       subject selective pressure. Indeed, showed that <br /> nonetheless, agreement ECI results,              transcriptional variability true indicator of <br /> partial correlations indicate effect transcrip-      evolutionary rate. If rule effect trans- <br /> tional activity stronger yeast.                            lational selection, operate yeast, <br />    Do highly expressed genes evolve slowly? We              apparent slow evolution highly expressed genes <br /> shown contrary consensus transcription-         attributed low variability. Selective <br /> ally active genes evolve slowly, evolution shapes tran-          forces enable phenotypic variations evolve <br /> scriptional variability transcriptional              mainly shaping transcriptional variability. Further- <br /> activity. Selective pressure act primarily           more, suggest high abundance essential <br /> transcriptional variability. If so,           proteins favored maintain low variability <br /> explain apparent correlation transcrip-              amount. Transcriptional variability, than <br /> tional activity evolutionary rate observed            transcriptional activity, common indicator of <br /> previous studies?                                                essentiality evolutionary rate, contributing the <br />    First, described Introduction, translational        correlation variables. <br /> selection mainly explained correlation (Akashi              We grateful members Integrative Evolutionary <br /> 2001, 2003; Akashi Gojobori 2002; Drummond                   Analysis Team, ArrayPort Team, Core Team Korean BioInfor- <br /> et al. 2005). Tables 3 5 evolutionary rate      mation Center producing data maintaining computer <br /> still correlated transcriptional activity yeast,    systems used analysis. This work supported Korean <br />                                                                  Ministry Science Technology grant no. M10407010001- <br /> after controlling transcriptional variability. There- <br />                                                                  04N0701-00110. <br /> fore, rule effect translational selection <br /> such higher eukaryotes (Shields et al. 1988; Sharp <br /> et al. 1995; Akashi 1997, 2001; dos Reis et al. 2004;                                 LITERATURE CITED <br /> Wright et al. 2004), conclude invariable <br />                                                                  Akashi, H., 1997 Codon bias evolution Drosophila. Population <br /> genes evolve slowly.                                                 genetics <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>-selection drift. Gene 205: 269&#226;&#8364;&#8220;278. <br />    Second, inverse relationship             Akashi, H., 2001 Gene expression molecular evolution. Curr. <br /> mean CV, speculate transcriptional                Opin. Genet. Dev. 11: 660&#226;&#8364;&#8220;666. <br />                                                                  Akashi, H., 2003 Translational selection yeast proteome evolu- <br /> activity increases, relative transcriptional variability        tion. Genetics 164: 1291&#226;&#8364;&#8220;1303. <br /> tend decrease. In biological terms, expression                Akashi, H., T. Gojobori, 2002 Metabolic efficiency amino <br /> changes abundant proteins likely smaller              acid composition proteomes Escherichia coli Bacillus <br />                                                                      subtilis. Proc. Natl. Acad. Sci. USA 99: 3695&#226;&#8364;&#8220;3700. <br /> effects cell scanty proteins. High          Blake, W. J., M. Kaern, C. R. Cantor J. J. Collins, 2003 Noise <br /> expression levels confer tolerance fluctuation              gene expression. Nature 422: 633&#226;&#8364;&#8220;637. <br /> in amount, low expression levels enable            Bloom, J. D., C. Adami, 2003 Apparent dependence pro- <br />                                                                      tein evolutionary rate number interactions linked to <br /> delicate transcriptional regulation. From stochastic               biases protein-protein interactions data sets. BMC Evol. Biol. <br /> perspective, high level transcriptional activity leads          3: 21. <br /> to reduction random noise, means                  Bloom, J. D., C. Adami, 2004 Evolutionary rate depends on <br />                                                                      number protein-protein interactions independently gene <br /> reduced variability. This explanation applied            expression level: response. BMC Evol. Biol. 4: 14. <br /> to correlation expression level essentiality.         Chuang, J. H., H. Li, 2004 Functional bias spatial organi- <br /> Indeed, production essential proteins shown               zation genes mutational hot cold regions human <br />                                                                      genome. PLoS Biol. 2: e29. <br /> to involve lower levels noise (Fraser et al. 2004).           Denver, D. R., K. Morris, J. T. Streelman, S. K. Kim, M. Lynch <br /> These speculations suggest high abundance                  et al., 2005 The transcriptional consequences mutation <br /> slowly evolving essential proteins evolution-              natural selection Caenohabditis elegans. Nat. Genet. 37: <br />                                                                      544&#226;&#8364;&#8220;548. <br /> arily favored maintain low variability expression.       dos Reis, M., R. Savva L. Wernisch, 2004 Solving riddle <br />    Taken together, aspects, action trans-             codon usage preferences: test translational selection. <br /> lational selection yeast correspondence                   Nucleic Acids Res. 32: 5036&#226;&#8364;&#8220;5044. <br />                                                                  Drummond, D. A., J. D. Bloom, C. Adami, C. O. Wilke F. H. <br /> between transcriptional activity variability,                Arnold, 2005 Why highly expressed proteins evolve slowly. <br /> explain apparent correlation transcrip-                  Proc. Natl. Acad. Sci. USA 102: 14338&#226;&#8364;&#8220;14343. <br /> tional activity evolutionary rate previous            Drummond, D. A., A. Raval C. O. Wilke, 2006 A single deter- <br />                                                                      minant dominates rate yeast protein evolution. Mol. Biol. <br /> studies.                                                             Evol. 23: 327&#226;&#8364;&#8220;337. <br />    Conclusion: In study, characterized general           Duret, L., D. Mouchiroud, 2000 Determinants substitution <br /> transcriptional activity variability eukaryotic               rates mammalian genes: expression pattern affects selection <br />                                                                      intensity mutation rate. Mol. Biol. Evol. 17: 68&#226;&#8364;&#8220;74. <br /> genes global expression profiles various species         Elowitz, M. B., A. J. Levine, E. D. Siggia P. S. Swain, 2002 Sto- <br /> spanning long evolutionary time period. While                  chastic gene expression single cell. Science 297: 1183&#226;&#8364;&#8220;1186. <br />  206                                                            J. K. Choi et al. <br />  <br /> Enard, W., P. Khaitovich, J. Klose, S. Zollner, F. Heissig et al.,        Meiklejohn, C. D., J. Parsch, J. M. Ranz D. L. Hartl, <br />     2002 Intra- interspecific variation primate gene expres-            2003 Rapid evolution male-based gene expression Drosoph- <br />     sion patterns. Science 296: 340&#226;&#8364;&#8220;343.                                       ila. Proc. Natl. Acad. Sci. USA 100: 9894&#226;&#8364;&#8220;9899. <br /> Fraser, H. B., A. E. Hirsh, 2004 Evolutionary rate depends         Oleksiak, M. F., G. A. Churchill D. L. Crawford, 2002 Varia- <br />     number protein-protein interactions independently gene               tion gene expression natural populations. <br />     expression level. BMC Evol. Biol. 4: 13.                                   Nat. Genet. 32: 261&#226;&#8364;&#8220;266. <br /> Fraser, H. B., A. E. Hirsh, L. M. Steinmetz, C. Scharfe M. W.         Ozbudak, E. M., M. Thattai, I. Kurtser, A. D. Grossman A. van <br />     Feldman, 2002 Evolutionary rate protein interaction net-            Oudenaarden, 2002 Regulation noise expression a <br />     work. Science 296: 750&#226;&#8364;&#8220;752.                                                single gene. Nat. Genet. 31: 69&#226;&#8364;&#8220;73. <br /> Fraser, H. B., A. E. Hirsh, G. Giaever, J. Kumm M. B. Eisen,          Pal, C., B. Papp L. D. Hurst, 2001 Highly expressed genes in <br />     2004 Noise minimization eukaryotic gene expression. PLoS                yeast evolve slowly. Genetics 158: 927&#226;&#8364;&#8220;931. <br />     Biol. 2: e137.                                                        Pal, C., B. Papp L. D. Hurst, 2003 Rate evolution gene <br /> Giaever, G., P. Flaherty, J. Kumm, M. Proctor, C. Nislow et al.,               dispensability. Nature 421: 496&#226;&#8364;&#8220;497. <br />     2004 Chemogenomic profiling: identifying functional in-           Ranz, J. M., C. I. Castillo-Davis, C. D. Meiklejohn D. L. <br />     teractions small molecules yeast. Proc. Natl. Acad. Sci.             Hartl, 2003 Sex-dependent gene expression evolution <br />     USA 101: 793&#226;&#8364;&#8220;798.                                                          Drosophila transcriptome. Science 300: 1742&#226;&#8364;&#8220;1745. <br /> Gu, X., 2004 Statistical framework phylogenomic analysis           Raser, J. M., E. K. O&#226;&#8364;&#8482;Shea, 2004 Control stochasticity eu- <br />     gene family expression profiles. Genetics 167: 531&#226;&#8364;&#8220;542.                    karyotic gene expression. Science 304: 1811&#226;&#8364;&#8220;1814. <br /> Gu, X., Z. Zhang W. Huang, 2005 Rapid evolution expres-            Rifkin, S. A., J. Kim K. P. White, 2003 Evolution gene expres- <br />     sion regulatory divergences yeast gene duplication.              sion Drosophila melanogaster subgroup. Nat. Genet. 33: 138&#226;&#8364;&#8220; <br />     Proc. Natl. Acad. Sci. USA 102: 707&#226;&#8364;&#8220;712.                                   144. <br /> Gu, Z., S. A. Rifkin, K. P. White W.-H. Li, 2004 Duplicate genes      Sharp, P. M., 1991 Determinants DNA sequence divergence be- <br />     increase gene expression diversity species.             tween Escherichia coli Salmonella typhimurium: codon usage, <br />     Nat. Genet. 36: 577&#226;&#8364;&#8220;579.                                                   map position, concerted evolution. J. Mol. Evol. 33: 23&#226;&#8364;&#8220;33. <br /> Hasty, J., J. J. Collins, 2002 Translating noise. Nat. Genet.     Sharp, P. M., M. Averof, A. T. Lloyd, G. Matassi J. F. Peden, <br />     31: 13&#226;&#8364;&#8220;14.                                                                 1995 DNA sequence evolution: sounds silence. Philos. <br /> He, X., J. Zhang, 2006 Why hubs tend essential pro-            Trans. R. Soc. Lond. Ser. B 349: 241&#226;&#8364;&#8220;247. <br />     tein networks? PLoS Genet. 2: e88.                                    Shields, D. C., P. M. Sharp, D. G. Higgins F. Wright, 1988 &#226;&#8364;&#732;&#226;&#8364;&#732;Si- <br /> Herbeck, J. T., D. P. Wall J. J. Wernegreen, 2003 Gene ex-                 lent&#226;&#8364;&#8482;&#226;&#8364;&#8482; sites Drosophila genes neutral: evidence selec- <br />     pression level influences amino acid usage, codon usage,           tion synonymous codons. Mol. Biol. Evol. 5: 704&#226;&#8364;&#8220;716. <br />     tsetse fly endosymbiont Wigglesworthia. Microbiology 149:      Subramanian, S., S. Kumar, 2004 Gene expression intensity <br />     2585&#226;&#8364;&#8220;2596.                                                                 shape <br /> </body></html>