<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Oct. 2006, p. 7178&#226;&#8364;&#8220;7189                                                                             Vol. 26, No. 19 <br /> 0270-7306/06/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.00130-06 <br /> Copyright &#194;&#169; 2006, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />           The Essential Gene wda Encodes WD40 Repeat Subunit of <br />             Drosophila SAGA Required Histone H3 Acetylation <br />        Sebastia&#194;&#180;n Guelman, Tamaki Suganuma, Laurence Florens, Vikki Weake, Selene K. Swanson, <br />                     Michael P. Washburn, Susan M. Abmayr, Jerry L. Workman* <br />                           Stowers Institute Medical Research, 1000 E. 50th St., Kansas City, Missouri 64110 <br />                            Received 21 January 2006/Returned modification 3 March 2006/Accepted 17 July 2006 <br />  <br />              Histone acetylation provides switch transcriptionally repressive permissive chromatin. By <br />           regulating chromatin structure specific promoters, histone acetyltransferases (HATs) carry impor- <br />           tant functions differentiation development higher eukaryotes. HAT complexes present in <br />           organisms diverse Saccharomyces cerevisiae, humans, flies. For example, well-studied yeast SAGA <br />           related mammalian complexes. We previously identified Drosophila melanogaster orthologues yeast <br />           SAGA components Ada2, Ada3, Spt3, Tra1 demonstrated associate dGcn5 a <br />           high-molecular-weight complex. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> better understand function Drosophila SAGA (dSAGA), sought to <br />           affinity purify characterize complex detail. A proteomic approach led identification of <br />           orthologue yeast protein Ada1 novel protein encoded CG4448, referred WDA (will <br />           decrease acetylation). Embryos lacking alleles wda gene exhibited reduced levels histone H3 <br />           acetylation develop adult flies. Our results point critical function dSAGA histone <br />           acetylation Drosophila development. <br />  <br />  <br />    The packaging DNA chromatin dramatic conse-                        carry paralogues, Gcn5 PCAF, highly sim- <br /> quences enzymatic processes require DNA: replication,                     ilar primary structure vitro functions (6, 56). Al- <br /> recombination, repair, transcription. Eukaryotic cells                    orthologues yeast Tra1 TAFs present in <br /> overcome repressive effects chromatin covalently                        higher eukaryotes, yeast Ada Spt proteins <br /> modifying core histone tails (30). The posttranslational                      SAGA share amino acid identity proteins <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> modifications histone tails include acetylation, methylation                   higher eukaryotes. Three SAGA-related PCAF/Gcn5-contain- <br /> and ubiquitylation lysine residues, phosphorylation serines                 ing complexes isolated mammalian cells: <br /> and threonines, methylation arginine residues (16).                        PCAF, STAGA, TFTC. These complexes activate <br />    The histone <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> hypothesis states combination                     transcription vitro chromatin templates (4, 35, 39). <br /> different marks single histones, single nucleosomes, nu-                      Studies multicellular organisms provided evidence the <br /> cleosomal domains determine recruitment specific                       fundamental roles Gcn5 development. Experiments <br /> factors result unique downstream events (53).                              mice revealed Gcn5 essential viability, but <br />    The steady-state balance histone acetylation nu-                     PCAF not. Gcn5 <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>knockout</span> mice fail develop particular <br /> cleus maintained opposing actions histone acetyl-                    mesodermal lineages die 10 11 days em- <br /> transferases (HATs) deacetylases (7, 38). The Saccharo-                       bryogenesis (59). In addition, mutations plant ada2b and <br /> myces cerevisiae transcriptional coactivator protein Gcn5                    gcn5 affect development gene expression (55). Flies ho- <br /> HAT serves catalytic subunit multiprotein                      mozygous dGcn5 mutant allele display defects meta- <br /> HAT complexes SAGA, SLIK, ADA, A2 (22, 48, 51). The                           morphosis proliferation imaginal discs. Furthermore, <br /> 1.8-MDa SAGA complex comprised groups                            dGcn5 mutants significantly reduced levels acetylated <br /> proteins. The group consists Ada family proteins                  histone H3 polytene chromosomes (8). <br /> (which includes Gcn5, named Ada4). Ada genes                               The relevance Gcn5-containing complexes develop- <br /> isolated mutations allow yeast growth GAL4-                          ment supported evidence links mamma- <br /> VP16 overexpressed (22). The second group contains Spt                         lian STAGA complex neurodegenerative disorders. Spino- <br /> proteins, isolated suppressors transcription initiation de-                 cerebellar ataxia type 7 caused polyglutamine expansions <br /> fects caused promoter insertions Ty transposon (22,                     ataxin-7 protein, recently characterized subunit of <br /> 52). The TATA-binding protein-associated factors (TAFs)                           STAGA. Incorporation mutated version ataxin-7 into <br /> present SAGA carry structural function (23,                      STAGA dramatically reduces stability complex and <br /> 57). In addition, SAGA ATM/phosphatidylinositol                           ability acetylate histone H3, leading retinal degener- <br /> 3-kinase-related protein Tra1, ubiquitin protease Ubp8,                       ation mice (40). Similar mutations yeast homologue <br /> and recently identified Sgf proteins (24, 32, 46).                       Sgf73 negatively affect HAT activity SAGA and <br />    Gcn5 conserved yeast humans. In fact, mammals                       SLIK complexes (37). <br />                                                                                      We previously fractionated Drosophila melanogaster cells <br />   * Corresponding author. Mailing address: Stowers Institute Medical <br />                                                                                   demonstrated association dGcn5, dAda3, dAda2B, <br /> Research, 1000 E. 50th St., Kansas City, MO 64110. Phone: (816) 926-              dSpt3, dTra1 high-molecular-weight HAT complex <br /> 4312. Fax: (816) 926-4692. E-mail: jlw@stowers-institute.org.                     similar yeast SAGA (31). This Drosophila HAT complex will <br />  <br />                                                                            7178 <br />  VOL. 26, 2006                                                                          IDENTIFICATION OF NEW SUBUNITS IN dSAGA                                  7179 <br />  <br />  <br />                                                           TABLE 1. Plasmids generated study <br />        Construct                                                   Vector                                                     Insert (accession no.) <br />  <br /> pRmHa3-wda-HA2FL2                                               pRmHa3-CHA2FL2                                         cDNA CG4448 (NM_142879) <br /> pRmHa3-dAda1-HA2FL2                                             pRmHa3-CHA2FL2                                         cDNA CG31866 (NM_164980) <br /> pRmHa3-wda&#226;&#338;&#172;C-HA2FL2                                             pRmHa3-CHA2FL2                                         cDNA coding amino acids 1 393 <br />                                                                                                                          WDA (NP_651136) <br /> pQE12-wda&#226;&#338;&#172;C                                                     pQE12                                                  cDNA coding amino acids 1 393 <br />                                                                                                                          WDA <br /> pUAST-wda                                                       pUAST                                                  cDNA CG4448 (NM_142879) <br />  <br />  <br />  <br /> be referred dSAGA. To address subunit composition                         purified HAT complexes complexes immobilized protein A-Sepharose <br /> and function dSAGA, affinity purified complex                        beads (Amersham). <br />                                                                                        Affinity purifications mass spectrometry. TAP-dGcn5 purification and <br /> characterized detail. MudPIT analysis affinity                        MudPIT analysis previously described (25). Anti-FLAG purifications were <br /> purified dSAGA revealed new subunits. These uncharac-                           performed described previously (25) using stable lines express dAda1- <br /> terized proteins dSAGA encoded genes                                  HA2FL2 WDA-HA2FL2. MudPIT analysis affinity-purified complexes <br /> CG31865/CG31866 CG4448. By sequence homology,                                carried previously described (32). <br /> determined CG31865/CG31866 gene product                                 Generation deletions wda gene. All fly stocks used study were <br />                                                                                     kindly provided Bloomington Drosophila Stock Center Indiana Uni- <br /> orthologue yeast human Ada1 proteins. We named                           versity. <br /> the CG4448 gene product WDA (will decrease acetylation).                               To generate deletions disrupt wda gene, mobilized P-element <br /> WDA contains WD repeats, necessary                            KG03744, located &#226;&#172;&#402;300 bp upstream wda gene, screened impre- <br /> incorporation WDA dSAGA. Chromatographic frac-                              cise excisions. The P-element mobilized crossing stock y1 w67c23; ry506 <br /> tionation confirmed WDA 1.5-MDa dSAGA                           P{SUPor-P}KG03744 (Bloomington stock number 13345) y1 w*; ry506 Sb1 <br />                                                                                     P{&#226;&#338;&#172;2-3}99B/TM6 flies (Bloomington stock number 3664). Males containing the <br /> complex. We generated deletions wda gene ob-                             P-element KG03744 transposase gene &#226;&#338;&#172;2-3 crossed y1 w1118; D3, <br /> served mutant animals died second larval in-                        gl3/TM3, Sb1, Ser1 virgins. The P-element excisions screened loss of <br /> star. The homozygous mutant embryos showed reduced levels                           eye color (due loss white gene P-element). To determine whether <br /> of histone H3 acetylation, predominantly central ner-                        hop-out events imprecise, isolated DNA 15 flies corre- <br />                                                                                     sponding excision event using DNAzol (Invitrogen) according the <br /> vous system. Ubiquitous expression WDA transgene <br />                                                                                     manufacturer&#226;&#8364;&#8482;s protocol. To screen deletions, PCRs performed using <br /> restored viability histone H3 acetylation.                                      primers designed way amplify &#226;&#172;&#402;3-kb fragment the <br />                                                                                     excision precise DNA molecules smaller size excisions impre- <br />                         MATERIALS AND METHODS                                       cise. The independent deletion lines obtained referred wda4, <br />                                                                                     wda8, wda11. <br />    Plasmids used study. The plasmids generated study de- <br />                                                                                        Lethality test wda mutants. Flies wda11/TM3, Sb1, Ser1 mated the <br /> scribed Table 1. pRmHa3-CHA2FL2 pUAST previously described <br />                                                                                     stock w1118; DrMio/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 <br /> (3, 25). The cDNAs wda, dAda1, wda&#226;&#338;&#172;C isolated reverse <br />                                                                                     (Bloomington stock number 6663). The progeny genotype wda11/TM3, <br /> transcription-PCR. For reverse transcription, total RNA isolated 12- to <br /> 18-h Oregon R embryos using Trizol (Invitrogen). Single-stranded cDNA mol-          P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 propagated fur- <br /> ecules generated using SuperScript first-strand synthesis (In-      ther studies. <br /> vitrogen) subsequently amplified PCR using Pfu turbo (Stratagene).              Embryos wda11/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, <br />    Transfections generation stable lines. S2 cells transfected     Ser1 population collected apple juice plates aged generate a <br /> Effectene (QIAGEN) according manufacturer&#226;&#8364;&#8482;s instructions. Stable cell        collection embryos stages 12 15. The homozygous mutant embryos <br /> lines established cotransfection pRmha3-derived constructs       wda11/wda11, identified lack fluorescence, heterozygous em- <br /> pCoHygro (Invitrogen), followed 2- 3-week selection media containing     bryos wda11/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1, show- <br /> 0.25 mg/ml hygromycin (Invitrogen).                                                 ing intermediate fluorescence, transferred fresh apple plates deter- <br />    Preparation fractionation nuclear extracts. Nuclear extracts pre-    stage died. The heterozygous embryos used as <br /> pared described previously (49). Affinity purifications carried using   controls. <br /> nuclear extracts prepared 8 liters cells, grown density 3 &#226;&#171;&#187; 106        Generation transgenic flies. The cDNA wda cloned pUAST (3). <br /> cells/ml. CuSO4 added medium low levels expression        Injections carried Genetic Services, Inc. Determination chro- <br /> of tagged proteins.                                                             mosomal locations transgene carried using standard genetic <br />    To obtain protein fractions enriched HATs, nuclear extracts incubated    crosses. A transgene (UAS-wda) chromosome 3 recombined the <br /> with Ni-nitrilotriacetic acid-agarose beads, previously described. Anion-ex-     chromosome carrying deletion wda generate stock wda11, UAS-wda/ <br /> change (MonoQ) gel filtration (Superose 6) chromatography fractionations        TM3, Sb1, Ser1. In addition, tubulin-GAL4 driver stock y1 w*; <br /> were performed described previously (25).                                        P{tubP-GAL4}LL7/TM3, Sb1 (Bloomington stock number 5138) recom- <br />    Generation antibodies WDA. A DNA fragment coding N-terminal        bined chromosome carrying wda11 obtain wda11, P{tubP-GAL4}LL7/ <br /> part WDA (amino acids 1 393) amplified PCR using pRmHa3-wda-           TM3, Sb1, Ser1. In flies, yeast GAL4 protein expressed same <br /> HA2FL2 template inserted pQE12 (QIAGEN). Expression               ubiquitous <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> tubulin. To determine UAS-wda transgene <br /> purification recombinant His-tagged protein performed described      rescue lethality wda11/wda11 animals, flies genotype wda11, <br /> previously (25). Rats rabbits immunized recombinant soluble       UAS-wda/TM3, Sb1, Ser1 mated wda11, P{tubP-GAL4}LL7/TM3, Sb1, <br /> WDA fragment (Pocono Rabbit Farm Laboratory, Inc.).                             Ser1 flies, bristle <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span> (Sb1 Sb&#226;&#171;&#185;) progeny determined <br />    Coimmunoprecipitations, Western blots, HAT assays. Coimmunoprecipi-          200 adult flies. <br /> tations using anti-FLAG antibodies polyclonal antibodies performed          Immunohistochemistry. Embryos stock wda11/TM3, P{GAL4- <br /> described previously (25). Antibodies directed dGcn5 (rabbit, 1:3,000),          twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 collected apple juice plates <br /> dAda2B (guinea pig, 1:1,000), dSpt3 (rabbit, 1:1,000), dAda2A (rabbit, 1:1,000),    0- 4-h intervals 25&#194;&#176;C aged 14 h 25&#194;&#176;C obtain population of <br /> Atac1 (rat, 1:1,000), WDA (rat, 1:500) used Western blots (25, 31)      embryos stages 16 17. Embryos derived cross wda11, UAS-wda/ <br /> as antibodies FLAG (1:5,000; Sigma), acetylated H3 K9 (1:500;       TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 wda11, P{tubP- <br /> Upstate), H3 (1:500; Abcam).                                                    GAL4}LL7/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 were <br />    HAT reactions performed described previously (15) using           collected conditions. Embryos dechorionated 3 min in <br />  7180       GUELMAN ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br /> 50% bleach fixed 1:1 mixture 4% formaldehyde heptane. Staining <br /> was performed described previously (29). The primary antibodies used were <br /> anti-acetylated H3 K9 (rabbit, 1:100; Upstate), anti-tetra-acetylated H4 (rabbit, <br /> 1:100; Upstate) anti-green fluorescent protein (anti-GFP, mouse, 1:100; <br /> Roche). The anti-GFP antibodies included staining reaction to <br /> distinguish homozygous mutant embryos (GFP negative) balancer-contain- <br /> ing embryos (GFP positive). The secondary antibodies used Alexa Fluor <br /> 488 goat anti-rabbit (1:250; Molecular Probes) Alexa Fluor 660 goat anti- <br /> mouse (1:250; Molecular Probes). <br />   Determination bulk levels H3 acetylation embryos. Embryos from <br /> the stock wda11/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 <br /> were collected apple juice plates aged generate collection stages <br /> 16 17. Embryos derived cross wda11, UAS-wda/TM3, P{GAL4- <br /> twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 wda11, P{tubP-GAL4}LL7/TM3, <br /> P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 collected the <br /> same conditions. Embryos dechorionated 3 min 50% bleach, and <br /> GFP-negative embryos (wda11/wda11 wda11, UAS-wda/wda11, tubP-GAL4) <br /> were homogenized sodium dodecyl sulfate-containing loading buffer (1 &#226;?&#174;l <br /> buffer embryo). Extracts prepared stage 16 17 Oregon R embryos <br /> were used controls. Five microliters extract analyzed Western <br /> blotting. <br />   GFP-sorting homozygous mutant embryos. To isolate large amounts of <br /> homozygous mutant embryos, stage 16 17 embryos stock wda11/TM3, <br /> P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb1, Ser1 dechorionated                 FIG. 1. Conserved domains WDA dAda1. (A) The amino <br /> sorted lack fluorescence using COPAS PLUS Drosophila embryo         acid sequences conserved regions human TAF5 (hTAF5), <br /> sorting instrument (Union Biometrica) (17). After sorting, embryo lysates      Drosophila TAF5 (dTAF5), cannonball (can), conserved re- <br /> prepared presence 50 mM Tris-Cl, pH 8, 300 mM NaCl, 1% NP-40, 1           gions WDA compared human TAF5-like (hTAF5L). <br /> mM phenylmethylsulfonyl fluoride, 1 &#226;?&#174;g/ml pepstatin A, 1 &#226;?&#174;g/ml leupeptin.       Region I (white) consists WD40-associated domain. Region II <br />                                                                                     (light gray) contains noncanonical WD repeat. Region III (dark gray) <br />                                                                                     includes consensus WD repeats. The percent identity/similarity <br />                                  RESULTS                                            relative hTAF5L shown conserved domain. Different <br />                                                                                     150-amino-acid stretches contained box compared <br />    The products genes CG4448 CG31865/CG31866                             region I hTAF5L, average value identity/similarity is <br /> associate TAP-dGcn5. In previous report, affinity                         shown. (B) The amino acid sequence conserved histone fold in <br /> purified dGcn5-containing complexes identified dGcn5-                           Drosophila Ada1 (dAda1) compared human Ada1 <br />                                                                                     (hAda1) yeast Ada1 (yAda1). The percent identity/similarity rel- <br /> associated proteins multidimensional protein identification <br />                                                                                     ative dAda1 shown. Arrowheads point different peptides for <br /> technology (MudPIT) (25). In addition known dGcn5-inter-                         WDA dAda1 obtained MudPIT. <br /> acting proteins, dAda3, dAda2A, dAda2B, the <br /> MudPIT analysis revealed peptides encoded the <br /> transcripts CG4448 CG31865/CG31866. CG4448 and <br /> CG31865/CG31866 identified 9 4 different pep-                           group human TAF5-like/PAF65&#226;?&#164; (hTAF5L), a <br /> tides, respectively (Fig. 1A B).                                                component STAGA, PCAF, TFTC HAT com- <br />    CG4448 encodes 743-amino-acid polypeptide pre-                          plexes (9). Pairwise alignments different ver- <br /> dicted molecular mass 84 kDa (NP_651136). Analysis                        tebrate proteins identified BLAST indicated they <br /> primary structure revealed protein belongs                         65% identical entire <br /> superfamily WD40-repeat proteins (Pfam accession number                          sequence, WDA 30% identity of <br /> PF00400). We named protein WDA                             vertebrate proteins. Other proteins identified BLAST <br /> phenotype mutant, described below. WD repeats                         include human Drosophila TAF5 (hTAF5 dTAF5) <br /> have <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> 44 60 amino acids contain conserved                       cannonball (can), fly tissue-specific TAF required for <br /> core region flanked dipeptides GH WD (50). The                           male gametogenesis (26). <br /> original member family &#226;?&#164; subunit trimeric G                          We compared domain structure WDA of <br /> proteins couple receptors extracellular signals in-                     hTAF5L, hTAF5, dTAF5, (Fig. 1A). All polypep- <br /> tracellular enzymes (21). The crystal structure protein,                    tides carry noncanonical WD repeat (region II) (Fig. 1A), <br /> which contains seven WD repeats, revealed &#226;?&#164;-propeller                             followed 5 conserved WD repeats (region III) (Fig. 1A). <br /> structure seven blades four-stranded &#226;?&#164;-twisted                         Moreover, regions II III hTAF5L closely <br /> sheets. WD-repeat proteins present eukaryotes,                      related TAF5 WDA. In addition, TAF5L TAF5 <br /> over 100 members humans. These proteins involved                        proteins, WDA, share conserved N-terminal region, <br /> wide variety cellular functions, including cell division, vesicle                known WD40-associated domain (region I, Pfam accession <br /> transport, RNA synthesis processing, chromatin assembly,                        number PF04494). While WD40-repeat-containing pro- <br /> cell cycle regulation, cytoskeletal assembly. In addition,                      tein, lack WD40-associated domain suggests that <br /> mutations WD-repeat proteins implicated                        WDA included TAF5 TAF5-like <br /> in human diseases (34).                                                             families proteins. This supported BLAST searches <br />    A BLAST search performed recover potential or-                            using N terminus WDA, detect pro- <br /> thologues WDA organisms. We identified                          teins species significant homology. <br /> WD-repeat-containing proteins vertebrates share                              CG31865 CG31866 independent transcription <br /> some amino acid sequence WDA. One proteins                              units located region 33B5 chromosome 2 separated <br />  VOL. 26, 2006                                                                IDENTIFICATION OF NEW SUBUNITS IN dSAGA                       7181 <br />  <br />  <br />  <br />  <br />    FIG. 2. WDA dAda1 specifically associate dSAGA subunits. (A) S2 cells transfected pRmHa3-wda-HA2FL2 pRmHa3- <br /> dAda1-HA2FL2 plasmids. After 1-day induction, whole-cell extracts prepared incubated M2-agarose beads. Untransfected S2 cells <br /> (WT) used negative controls. The immunoprecipitated material analyzed Western blotting using antibodies dGcn5 (rabbit), <br /> dSpt3 (rabbit), dAda2B (guinea pig), dAda2A (rabbit). Lanes 1 3 correspond 40 &#226;?&#174;g whole-cell extract (2% input). Lanes 4 6 <br /> correspond immunoprecipitated material (IP). (B) Ten microliters HAT-enriched extract (lane 1) 20 pg recombinant WDA&#226;&#338;&#172;C <br /> (lanes 2) analyzed Western blotting using antibodies WDA (rat). (C) Nuclear extracts immunoprecipitated using antibodies <br /> against dGcn5, dSpt3, WDA, Atac1, preimmune bleed (pre). The immunocomplexes analyzed Western blotting using antibodies directed <br /> to dGcn5 (rabbit), WDA (rat), dAda2B (guinea pig), Atac1 (rat). (D) Nuclear extracts immunoprecipitated using antibodies dGcn5, <br /> dSpt3, WDA, preimmune bleed (pre). The HAT activity beads assayed using core histones (top panels) nucleosomes (bottom <br /> panels) substrates. For substrate, fluorography showing labeled histones located corresponding Coomassie gel. The <br /> migrations histones H3 H4 shown. &#226;?&#163;, anti. <br />  <br />  <br /> by &#226;&#172;&#402;3-kb stretch DNA harbors genes CG18789                 identify human yeast Ada1 orthologues CG31865/ <br /> and CG31864. The amino acid sequences predicted               CG31866. The polypeptides encoded CG31865/CG31866 <br /> proteins (NP_723707 NP_723703) identical               hAda1 identical 26% amino acid sequences. <br /> a single substitution position 97. In addition, predicted 5&#226;&#172;&#732;       Thus, refer CG31865/CG31866 Drosophila Ada1 <br /> (90 bp) 3&#226;&#172;&#732; (70 bp) untranslated regions RNA                  (dAda1) (Fig. 1B). <br /> transcripts identical nucleotide (10, 11). The            WDA dAda1 associate dSAGA subunits. To con- <br /> genomic organization CG31865 CG31866 suggests                 firm interactions WDA, dAda1, dGcn5, gen- <br /> these genes arose gene duplication event.                        erated plasmids pRmHa3-wda-HA2FL2 pRmHa3- <br />    Further analysis sequence CG31865/CG31866                dAda1-HA2FL2, express C-terminally FLAG-tagged <br /> gene product revealed histone fold motif located                 WDA dAda1, respectively. We prepared whole-cell ex- <br /> center polypeptide chain (Fig. 1B). Histone fold motifs,           tracts S2 cells transiently transfected plasmids <br /> present globular domains core histones,           used anti-FLAG antibodies coimmunoprecipitation ex- <br /> in subunits general transcription factor TFIID           periments. Western blots demonstrated WDA dAda1 <br /> transcription factors. In fact, 9 14 TAFs yeast TFIID           associated dGcn5 (Fig. 2A, panel). In addition, tagged <br /> contain motif (18). In addition, TAFs (TAF4,           WDA dAda1 interacted dSAGA-specific subunits <br /> 6, 9, 12) reconstituted vitro octamer-like          dSpt3 dAda2B (Fig. 2A, dAda2B dSpt3 blots) not <br /> structure (47). The yeast SAGA subunits Spt3, Spt7, Ada1              dAda2A, dSAGA ATAC histone <br /> also carry histone folds (2, 19, 20). Blast searches enabled        acetyltransferase complex (Fig. 2A, panel) (25). From <br />  7182     GUELMAN ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. WDA subunit 1.5-MDa dSAGA complex. (A) A HAT-enriched extract applied Mono Q column. The bound proteins <br /> were resolved linear gradient 100 500 mM NaCl, odd fractions analyzed Western blotting using antibodies dAda2B <br /> (guinea pig), dSpt3 (rabbit), WDA (rat). I, 10 &#226;?&#174;l input; FT, 10 &#226;?&#174;l Mono Q flowthrough. (B) Fractions 17 23 pooled, concentrated, <br /> and loaded Superose 6 column. The eluted fractions analyzed presence WDA, dAda2B, Spt3 Western blotting. The <br /> migration molecular mass standards shown. I, 10 &#226;?&#174;l input; &#226;?&#163;, anti. <br />  <br />  <br />  <br /> this experiment, conclude dAda1 WDA spe-                   strong H3 signal weaker H4 characteristic yeast <br /> cific components dSAGA ATAC.                                    Drosophila SAGA complexes (32). <br />    To investigate association WDA dSAGA,                  WDA stable subunit 1.5-MDa dSAGA complex. <br /> we generated polyclonal antibodies N-terminal                  Since observations provided evidence stable associa- <br /> nonconserved portion WDA, lacks WD repeats.                   tion WDA dSAGA, wanted determine WDA <br /> We used antibodies Western blots unable               cofractionated dSAGA components anion-exchange <br /> detect endogenous protein nuclear extracts S2 cells            chromatography. To end, applied Ni-agarose HAT- <br /> (data shown). However, antisera recognized WDA                   enriched fraction Mono-Q column compared the <br /> (predicted molecular mass &#226;&#172;&#402;84 kDa) present Ni-aga-                 elution profiles WDA, dAda2B, dAda2A Western <br /> rose HAT complex-enriched fraction (Fig. 2B, lane 1). In ad-               blotting. This analysis showed WDA dSAGA <br /> dition, serum detected truncated recombinant protein                 subunit dAda2B elute range salt gradient <br /> purified E. coli, molecular mass &#226;&#172;&#402;50 kDa (Fig.              (Fig. 3A), corresponding elution profile dSAGA. The <br /> 2B, lane 2). An identical blot probed preimmune bleed               WDA peak did overlap ATAC complex subunit <br /> from animal did contain signals endogenous                dAda2A, elutes column higher salt (Fig. <br /> or recombinant WDA (data shown), led con-                  3A, panel) (25). <br /> clude antiserum specifically recognized WDA,                 To determine apparent molecular mass WDA- <br /> not sensitive detect nuclear extracts.                     containing dSAGA complex, pooled fractions where <br />    We wished establish associations endoge-                WDA present (fractions 17 23), concentrated them, <br /> nous WDA dSAGA subunits, dGcn5 dSpt3.                      applied Superose 6 gel filtration column, per- <br /> Coimmunoprecipitation experiments showed antibodies                   formed Western analysis eluted fractions. Figure 3B <br /> directed dGcn5 dSpt3 pull WDA nuclear                     points WDA, dSpt3, dAda2B associate com- <br /> extracts (Fig. 2C, lanes 2 3, WDA blot). In addition,             plex apparent molecular mass &#226;&#172;&#402;1.5 MDa. <br /> we used WDA serum reciprocal immunoprecipita-                     Affinity purification dSAGA. Coimmunoprecipitations in- <br /> tion reaction, dSAGA subunits dGcn5 dAda2B                    dicated dAda1 WDA components dSAGA. <br /> brought ATAC complex subunit Atac1 (Fig.                  Nuclear extract fractionation provided additional evidence <br /> 2C, lane 4). dGcn5 antibodies confirmed presence              WDA subunit dSAGA. To gain insight the <br /> both dSAGA ATAC (Fig. 2C, lane 2). Conversely, pre-                  subunit composition dSAGA complex, wished to <br /> immune bleed did precipitate dGcn5, WDA, dAda2B,                    obtain mass spectrometry data complexes affinity purified <br /> Atac1, pulled nonspecific band cross-              tagged dSAGA-specific subunits. To end, estab- <br /> reacts Atac1 serum (Fig. 2C, lane 1). Another way              lished stable cell lines express C-terminally FLAG-tagged <br /> confirm association WDA dSAGA HAT com-                     WDA dAda1 prepared nuclear extracts these <br /> plex testing HAT activity immunoprecipitated              cell lines. The extracts subjected anti-FLAG affinity <br /> complexes. Antibodies WDA immunoprecipitated                     purification, purified material analyzed nucleo- <br /> HAT pattern identical dSpt3 dGcn5               somal HAT activity confirm success dSAGA <br /> core histones (Fig. 2D, panels) nucleosomes (Fig. 2D,               purifications (Fig. 4A). The affinity-purified complexes (lanes 3 <br /> lower panels) used substrates. This pattern, consisting            4) generated acetylation pattern identical H3 <br />  VOL. 26, 2006                                                             IDENTIFICATION OF NEW SUBUNITS IN dSAGA                    7183 <br />  <br />  <br />  <br />  <br />   FIG. 4. Affinity purification MudPIT analysis dSAGA. (A) The nucleosomal HAT activity affinity-purified dSAGA complexes (lanes <br /> 2 3) compared HAT pattern yeast SAGA (ySAGA) dGcn5-containing complexes purified cells expressing TAP-dGcn5 <br /> (TAP-dGcn5). The fluorography shown Coomassie gel. (B) MudPIT analysis dSAGA, purified cells expressing tagged dAda1 <br /> or WDA. The table shows number nonredundant spectra protein (total peptides) amino acid sequence coverage (% <br /> coverage). <br />  <br />  <br /> pattern yeast SAGA (lane 1) nucleosomes. To determine            subunits ySAGA (22, 43). Another potential dSAGA sub- <br /> the protein composition dSAGA identify new sub-              <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> revealed MudPIT orthologue yeast Ubp8 (data <br /> units, purified complexes subjected MudPIT (Fig.           shown). We currently tagging new putative <br /> 4B). MudPIT analyses turned informative             dSAGA subunits confirm interaction dGcn5. <br /> only confirmed previous findings              Fifth, MudPIT confirmed WDA dAda1 specifically <br /> they provided evidence new subunits complex. First,         associate dSAGA dGcn5-containing complex <br /> we noticed previously identified dSAGA compo-                 ATAC, purifications did yield peptides the <br /> nents dGcn5, dAda3, dAda2B, dSpt3 dTra1 copurified                 ATAC-specific proteins dAda2A Atac1 (25). Finally, the <br /> with dAda1 WDA. Second, identified peptides             fact identify peptides TATA-binding <br /> SAGA-specific TAFs, dTAF9 dTAF12 (23). Thus,            protein TDIID-exclusive TAFs TAF1 purifi- <br /> appears yeast fly SAGA subset TAFs               cations supports hypothesis WDA TAF (data <br /> common. One possible explanation obtaining peptides            shown). <br /> for TAF6 dAda1-FLAG purification partial                  The WD repeats WDA required incorporation <br /> disruption interaction TAF6 dSAGA            dSAGA. WD repeats present diverse group of <br /> tags fused C terminus dAda1. Third, confirm         eukaryotic proteins different cellular functions. WD re- <br /> that dAda1 WDA present complex,              peats create surfaces protein-protein interactions (50), <br /> they copurify other. Fourth, results MudPIT        feature exploited WDA. Therefore, wished <br /> experiment point presence uncharacterized              establish function WD repeats WDA. To <br /> proteins dSAGA, number conserved             achieve this, expressed C-terminally FLAG-tagged trun- <br /> evolution. For instance, proteins CG30390 CG6506               cated form WDA, lacking WD repeats, S2 cells. <br /> are orthologues yeast Sgf29 Spt7,             This protein, referred WDA&#226;&#338;&#172;C-HA2FL2, stable and <br />  7184     GUELMAN ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. The WD repeats WDA required incorporation WDA dSAGA. S2 cells transfected pRmHa3-wda- <br /> HA2FL2 pRmHa3-wda&#226;&#338;&#172;C-HA2FL2 plasmids. After 1-day induction, whole-cell extracts prepared incubated M2-agarose <br /> beads. Untransfected S2 cells (WT) used negative controls. The immunoprecipitated material analyzed Western blotting using <br /> antibodies dGcn5 (rabbit), dSpt3 (rabbit), dAda2B (guinea pig), FLAG. Lanes 1 3 correspond 40 &#226;?&#174;g whole-cell extract (2% <br /> input). Lanes 4 6 correspond immunoprecipitated material (FLAG IP). The arrows migration tagged WDA WDA&#226;&#338;&#172;C. <br /> The asterisk points cross-reacting band recognized anti-FLAG antibodies. &#226;?&#163;, anti. <br />  <br />  <br />  <br />  <br /> could detected Western blots using anti-FLAG anti-             interesting related fly protein WDS (will die <br /> body (Fig. 5, lane 3, panel). The anti-FLAG beads                slowly), encoded essential gene closest ortho- <br /> could bring truncated polypeptide (Fig. 5, lane 6,             logue WDR5, bind methylated histones like hu- <br /> bottom panel). However, dSAGA subunits dGcn5,                       man counterpart does (27, 58). <br /> dAda2B, dSpt3 detected immunopre-                  WDA encoded essential gene. To ascertain the <br /> cipitated material (Fig. 5, lane 6). When identical immuno-          phenotype flies devoid WDA, generated wda null <br /> precipitation performed full-length tagged WDA,            allele imprecise excision KG03744 transposon, lo- <br /> FLAG antibody coimmunoprecipitated tagged                  cated &#226;&#172;&#402;300 bp upstream wda gene. We analyzed the <br /> protein dSAGA subunits tested (Fig. 5, lane 5).        hop-out events PCR sequencing (data shown) and <br /> This experiment confirms crucial role WD repeats            confirmed imprecise excisions P-element inde- <br /> the association WDA dSAGA.                                      pendent events. The mutant alleles wda4, wda8, wda11 were <br />    Using pull-down assay followed mass spectrometry                missing predicted TATA box, translation initiation <br /> analysis, Wysocka et al. identified human WD-repeat pro-            codon, exon, second exon wda <br /> tein WDR5 interaction partner methylated histone H3            gene (Fig. 6A). Due absence transcription trans- <br /> on lysine 4 (K4). WDR5, subunit number methyl-                lation regulatory regions, anticipate alleles be <br /> transferase complexes, transactivation properties            protein null. In addition, alleles lethal when <br /> important conversion di- tri-methylated H3              homozygous. wda11 chosen studies of <br /> (58). This observation raises possibility WDA,            removal larger portion wda gene the <br /> similar fashion, bind methylated nucleosomes prop-          alleles. <br /> agate acetylation dSAGA chromatin. To address                   To determine lethal phase wda11 flies, embryos from <br /> this question, performed vitro binding experiments             wda11/TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, <br /> test H3 peptides methylated K4 pull recom-             Sb1, Ser1 population collected apple juice plates and <br /> binant WDA stronger affinity unmodified H3                aged generate collection embryos stages 12 15. The <br /> peptides. We did observe WDA preferentially bound              homozygous mutant wda11/wda11 embryos identified by <br /> to di-methylated H3 K4 (data shown). It                 lack fluorescence separated fluorescent, <br />  VOL. 26, 2006                                                                  IDENTIFICATION OF NEW SUBUNITS IN dSAGA                        7185 <br />  <br />  <br />  <br />  <br />    FIG. 6. Deletions wda gene homozygous lethal. (A) Schematic representation wda locus, showing position P-element <br /> KG03744 (insertion) DNA deleted P-element excision (deletions 4, 8, 11). Exons represented boxes. 5&#226;&#172;&#732; 3&#226;&#172;&#732; untranslated <br /> regions open boxes, translated sequences filled gray. (B) Genetic crosses carried flies genotypes are <br /> shown left. The bristle phenotype (Sb&#226;&#171;&#185; Sb1) scored progeny (200 flies). The percentage flies corresponding genotype <br /> is shown. a, flies phenotype Sb&#226;&#171;&#185;; b, flies phenotype Sb1; c, animals carrying copies balancer chromosome die embryos. <br />  <br />  <br />  <br /> balancer-containing embryos. The majority mutant                   H3 acetylation (8, 41, 44). Therefore, wished determine <br /> embryos (98%) hatch larvae. Mutant first-instar                  novel dSAGA protein WDA linked histone <br /> larvae did present abnormalities mouth parts, spi-             H3 acetylation. To determine overall level acetylated H3 <br /> racles, movement behavior develop second                  embryos, prepared whole-cell extracts wild-type <br /> instar. However, second-instar larvae develop be-                 embryos stage 16 17, homozygous wda11/wda11 embryos <br /> yond developmental stage.                                              homozygous mutant embryos expressing WDA a <br />    When examined genomic sequence alleles                     transgene (wda11, UAS-wda/wda11, P{tubP-GAL4}LL7). We <br /> wda4, wda8, wda11, realized predicted tran-                 analyzed equivalent amounts extract Western blotting, <br /> scription start gene CG13827, adjacent wda,                  probing antibodies H3 acetylated lysine 9 on <br /> missing. To address lethality loss            histone H3 (ac H3 K9) (Fig. 7A). The blots reveal significant <br /> wda CG13827, attempted rescue lethality wda11               decrease histone H3 acetylation K9 mutant em- <br /> flies using GAL4/UAS binary (3). Transgenic flies                bryos (compare lanes 1 2, panel), relative total <br /> carrying cDNA coding WDA control UAS                       levels histone H3. When introduced transgene carrying <br /> were recombined wda11 chromosome. Simulta-                         cDNA wda mutant background, acetylated H3 is <br /> neously, recombined tubulin-GAL4 driver                     restored levels closer wild type (compare <br /> wda11 chromosome. The genotypes resulting stocks                     lanes 2 3, panel). The H3 blots confirm equal loading <br /> were wda11, UAS-wda/TM3, Sb1, Ser1 wda11, P{tubP-                       samples (Fig. 7A, middle panel). <br /> GAL4}LL7/TM3, Sb1, Ser1. These stocks crossed                     Quantitation bands revealed reduction 50% <br /> other, adult progeny scored absence               H3 K9 acetylation mutants, restored over <br /> the dominant marker Sb1 balancer chromosome TM3,                     80% wild-type levels introduction transgene (Fig. 7, <br /> Sb1, Ser1 (Fig. 6B). We observed 39% adult progeny              panel). <br /> did carry balancer chromosome, having genotype                     Western analysis revealed reduction global levels of <br /> wda11, UAS-wda/wda11, P{tubP-GAL4}LL7. The rest                      histone H3 K9 acetylation mutant embryos. We wished to <br /> progeny Sb1, corresponding genotypes wda11, UAS-                investigate reduction observed immu- <br /> wda/TM3, Sb1, Ser1 wda11, P{tubP-GAL4}LL7/TM3, Sb1,                     nostaining embryos stage development. We <br /> Ser1 (both carry balancer chromosome). These                    collected stage 16 17 embryos stained anti- <br /> results indicated wda transgene rescue le-               bodies ac H3 K9 (see Materials Methods). We <br /> thality wda11 allele, expressed control              observed reduction staining intensity homozy- <br /> ubiquitous promoter, suggested lethality wda11              gous mutant embryos, compared control embryos, carrying <br /> flies lack expression WDA CG13827.                 wild-type copy wda gene (compare left middle <br />    WDA required histone H3 acetylation Drosophila                 panels). This reduction ac H3 K9 signal clear the <br /> embryos. Previous studies demonstrated inactivation                 central nervous stage 16 embryos (Fig. 7B). Further- <br /> the dSAGA subunits dGcn5 dAda2B causes decrease                    more, WDA expressed transgene, same <br />  7186     GUELMAN ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Deletions wda gene result reduced levels histone H3 acetylation. (A) Whole-cell extracts homozygous mutant (lane <br /> 1), wild-type (lane 2), mutant embryos expressing WDA transgene (lane 3) analyzed Western blotting using antibodies against <br /> H3 acetylated H3 K9. The intensities bands Western blots independent experiments quantified. The levels of <br /> acetylated H3 normalized total H3 sample determine levels acetylated H3 relative wild type. <br /> (B) Embryos stained antibodies acetylated H3 K9 (top panels) tetra-acetylated H4 (bottom panels). The genotypes the <br /> embryos indicated. &#226;?&#163;, anti. <br />  <br />  <br />  <br /> pattern tubulin, acetylation H3 central nervous           WDA required structural integrity dSAGA. <br /> system restored (compare middle right panels). In             One possible explanation defects H3 acetylation in <br /> contrast, did observe differences staining pattern         wda mutants WDA architectural subunit <br /> when used tetra-acetylated H4 antibodies (Fig. 7B,                removal disrupts integrity complex. To ad- <br /> bottom panels). These results consistent experiments            dress possibility, prepared whole-cell extracts from <br /> performed dAda2b mutant flies. dAda2b mutants ho-               wild-type wda11/wda11 embryos. We carried immuno- <br /> mozygous lethal display reduced levels H3 acetylation          precipitations using antibodies directed dSpt3 analyzed <br /> stage 16 embryos. In addition, overall acetylated H3 signal          immunoprecipitates antibodies dAda2B (Fig. 8A). <br /> in polytene chromosomes severely reduced (41, 44). We                 Western blot analysis indicates dSpt3 antibodies pull <br /> could test acetylation levels wda mutants polytene         dAda2B mutant extracts efficiently wild- <br /> chromosomes, wda mutants die second larval in-              type embryos (lanes 3 4), suggesting removal WDA <br /> star, dAda2b mutants die pupal stage.                       does interfere stability complex. A low- <br />  VOL. 26, 2006                                                               IDENTIFICATION OF NEW SUBUNITS IN dSAGA                       7187 <br />  <br />  <br />  <br />  <br />    FIG. 8. WDA required structural integrity dSAGA. (A) Three milligrams whole-cell extracts wild-type wda11/wda11 <br /> embryos immunoprecipitated antibodies directed dSpt3 analyzed Western blotting dAda2B antibodies. Lanes 1 2 <br /> correspond 50 &#226;?&#174;g whole-cell extract mutant wild-type embryos, respectively. Lanes 3 4 immunoprecipitated material <br /> from mutant wild-type extracts, respectively. (B) Forty milligrams whole-cell extracts wild-type wda11/wda11 embryos loaded <br /> onto Superose 6 gel filtration column. The elution profiles determined probing column fractions anti-WDA -dAda2B <br /> antibodies. Molecular mass standards run conditions. I, 40 &#226;?&#174;g whole-cell extracts; &#226;?&#163;, anti. <br />  <br />  <br />  <br /> resolution structure yeast SAGA complex indicates             Tra1 cofractionated anion-exchange size-exclusion <br /> Spt3 Gcn5 physically separated different modules               chromatography, suggesting flies harbor complex similar <br /> within SAGA (57). Since Ada2 Ada3 potentiate nu-                  SAGA (dSAGA) (31). This study describes affinity pu- <br /> cleosomal HAT activity Gcn5, likely             rification complex time. Proteomic ap- <br /> proteins reside module SAGA (1). Our                   proaches combined biochemical assays allowed con- <br /> coimmunoprecipitation experiment indicates dAda2B                firm dSAGA contains dGcn5, dAda3, dSpt3, dAda2B, <br /> dSpt3, likely distinct modules dSAGA,               dTra1, TAFs. In addition previously characterized <br /> associate absence WDA.                                          subunits, study uncovered new components dAda1 <br />    We tested integrity complex lacking WDA             WDA. <br /> gel filtration chromatography. To end, applied extracts              Our mass spectrometry study affinity-purified dSAGA <br /> from wild-type wda11/wda11 embryos Superose 6 col-                unveiled peptides corresponding CG31865/CG31866. We <br /> umn looked shifts elution profiles wild-type            utilized BLAST searches multiple-sequence alignments as <br /> and WDA-lacking dSAGA, reflect changes                     tools determine CG31865/CG31866 encodes ortho- <br /> their apparent molecular weights. Western analysis col-            logue yeast human Ada1. Coimmunoprecipitation <br /> umn fractions corresponding wild-type dSAGA indicated                  experiments proved dAda1 associated dSAGA. Yeast <br /> cofractionation WDA Ada2B 1.5-MDa complex                     Ada1 required structural integrity SAGA and, <br /> (Fig. 8B, fractions 10 12, middle panels). In addition,        Spt7 Spt20, subset SAGA <br /> these proteins present small complex (fractions 18        subunits severe phenotypes their <br /> 20), represent real dSAGA subcomplex present             genomic copies deleted (52). For example, ada1&#226;&#338;&#172; cells, <br /> embryos result degradation 1.5-MDa complex                 unlike spt3&#226;&#338;&#172; gcn5&#226;&#338;&#172; cells, fail grow media containing <br /> during fractionation procedure. The elution profile mutant         caffeine DNA synthesis inhibitor hydroxyurea. More- <br /> dSAGA, determined dAda2B antibodies, identical             over, ada1&#226;&#338;&#172; synthetically lethal mutations SWI/SNF <br /> that wild-type dSAGA (Fig. 8B, panel),           subunits; however, ada2&#226;&#338;&#172; ada3&#226;&#338;&#172; (52). We suggest <br /> fact WDA antibodies detect signal Western              dAda1 provide structural role dSAGA similar to <br /> blots (data shown). The gel filtration results agree         yeast counterpart. <br /> coimmunoprecipitation experiment, pointing conclude               A novel protein identified MudPIT WDA. We ob- <br /> WDA architectural protein required hold dSAGA                served WD-repeat-containing protein unique <br /> subunits together.                                                        N-terminal domain different amino acid composition <br />                                                                           WD-containing proteins database. The pres- <br />                           DISCUSSION                                      ence subunit containing WD repeats be <br />                                                                           evolutionary feature SAGA-related complexes. Yeast <br />   Previous studies lab shown Drosophila                SAGA human STAGA contain WD-repeat proteins <br /> orthologues yeast proteins Gcn5, Spt3, Ada2, Ada3,             yTAF5 hPAF65&#226;?&#164;/hTAF5L, respectively (23, 35). Temper- <br />  7188     GUELMAN ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> ature-sensitive mutations yTAF5 impair association         tures. First, preference H3 substrate. <br /> yTAF5 SAGA subunits cause transcription defects. In         Second, dSAGA bind Ni-agarose way the <br /> addition, mutations yTAF5 affect overall stability         yeast complex does (22). Third, complexes purified from <br /> yAda1 ySpt7. Interestingly, mutations yTAF5 local-        yeast flies similar size, having apparent molecular <br /> ize WD repeats, suggesting essential role         masses 1.5 2 MDa (22). Fourth, like yeast <br /> repeats ySAGA structure function (14).                        complex, dSAGA STAGA, interact acidic acti- <br />    The WD repeats WDA required interaction         vators (31, 35, 54). In fact, conserved proteins Tra1 and <br /> WDA dSAGA. Different chromatin-related proteins                 TRRAP, subunits SAGA STAGA, respectively, directly <br /> make protein-protein interactions WD repeats.          interact transcription activators (5, 42). Fifth, SAGA <br /> For instance, transcription corepressors Groucho (Gro)           STAGA activate transcription nucleosomal tem- <br /> and Tup1 display WD repeats C termini. The WD               plates acetyl coenzyme A-dependent manner (28, 35). <br /> domain Drosophila Gro required direct interactions            Although different SAGA complexes different or- <br /> with DNA-binding proteins Engrailed Hairy. In addi-          ganisms share number components, possess <br /> tion, point mutations WD domain yeast Tup1 disrupt         nonconserved subunits. For example, yeast SAGA contains <br /> its interaction repressor &#226;?&#163;2 (12). Another example       Spt8 Spt20, human STAGA contains STAF 46, 55, <br /> a WD-repeat protein involved chromatin metabolism Dro-         60 dSAGA WDA nonconserved poten- <br /> sophila p55, subunit remodeling complex NURF,           tial subunits currently investigating. These species- <br /> chromatin assembly factor (CAF1) uncharacterized              specific subunits provide unique features complex <br /> HAT (36). In cases, WD-repeat-containing pro-           carry specialized function. <br /> teins complex, illustrated yeast <br /> COMPASS complex, involved histone methylation,                                         ACKNOWLEDGMENTS <br /> contains Swd1, Swd2, Swd3, WD <br />                                                                        We thank members Workman Abmayr labs helpful <br /> repeats (13).                                                        discussions technical advice. We thank Jeff Haug Joshua <br />    We generated small deficiencies wda gene impre-         Wunderlich EGFP sorting Drosophila embryos Danny <br /> cise excision P-element observed gene         Stark Kiran Kocherlakota help confocal microscopy. We <br /> required viability. The homozygous mutant animals died           thank Joanna Wysocka David Allis providing H3 peptides <br /> during second larval instar. A transgene containing cDNA         pull-down experiments. <br /> for wda rescue lethality expressed pattern                                        REFERENCES <br /> similar tubulin. The recombinant adult flies, homozy- <br />                                                                       1. Balasubramanian, R., M. G. Pray-Grant, W. Selleck, P. A. Grant, S. <br /> gous deletion expressing WDA transgene,             Tan. 2002. Role Ada2 Ada3 transcriptional coactivators histone <br /> had obvious morphological abnormalities fertile.             acetylation. J. Biol. Chem. 277:7989&#226;&#8364;&#8220;7995. <br />                                                                       2. Birck, C., O. Poch, C. Romier, M. Ruff, G. Mengus, A. C. Lavigne, I. <br /> The fertility adults fact UAS-wda          Davidson, D. Moras. 1998. Human TAF(II)28 TAF(II)18 interact <br /> expressed germ line indicate WDA dispensable              histone fold encoded atypical evolutionary conserved motifs <br /> for oogenesis (45). In contrast, dGcn5 dAda2B,               SPT3 family. Cell 94:239&#226;&#8364;&#8220;249. <br />                                                                       3. Brand, A. H., N. Perrimon. 1993. Targeted gene expression means <br /> which dSAGA, required oogenesis. These ob-                altering cell fates generating dominant phenotypes. Development <br /> servations illustrate different dSAGA subunits in-               118:401&#226;&#8364;&#8220;415. <br /> volved distinct functions development. In fact, mu-         4. Brand, M., K. Yamamoto, A. Staub, L. Tora. 1999. Identification of <br />                                                                          TATA-binding protein-free TAFII-containing complex subunits suggests a <br /> tations different yeast SAGA subunits display different          role nucleosome acetylation signal transduction. J. Biol. Chem. 274: <br /> phenotypes specific growth conditions (52). Moreover,              18285&#226;&#8364;&#8220;18289. <br />                                                                       5. Brown, C. E., L. Howe, K. Sousa, S. C. Alley, M. J. Carrozza, S. Tan, and <br /> genome-wide analysis yeast demonstrated set               J. L. Workman. 2001. Recruitment HAT complexes direct activator <br /> genes affected loss Spt3 does completely overlap           interactions ATM-related Tra1 subunit. Science 292:2333&#226;&#8364;&#8220;2337. <br /> with groups genes regulated Gcn5 (33).                      6. Candau, R., P. A. Moore, L. Wang, N. Barlev, C. Y. Ying, C. A. Rosen, and <br />                                                                          S. L. Berger. 1996. Identification human proteins functionally conserved <br />    The immunofluorescent staining embryos antibodies             yeast putative adaptors ADA2 GCN5. Mol. Cell. Biol. 16:593&#226;&#8364;&#8220; <br /> directed acetylated K9 H3 complemented analysis             602. <br /> bulk levels acetylated K9 H3, suggesting K9 H3       7. Candido, E. P., R. Reeves, J. R. Davie. 1978. Sodium butyrate inhibits <br />                                                                          histone deacetylation cultured cells. Cell 14:105&#226;&#8364;&#8220;113. <br /> one vivo targets dSAGA. Our results consis-         8. Carre, C., D. Szymczak, J. Pidoux, C. Antoniewski. 2005. The histone H3 <br /> tent previous report describes requirement            acetylase dGcn5 key player Drosophila melanogaster metamorphosis. <br />                                                                          Mol. Cell. Biol. 25:8228&#226;&#8364;&#8220;8238. <br /> dAda2b viability histone acetylation (44). Both dAda2B        9. Carrozza, M. J., R. T. Utley, J. L. Workman, J. Cote. 2003. The diverse <br /> and WDA contribute vivo H3 HAT activity                     functions histone acetyltransferase complexes. Trends Genet. 19:321&#226;&#8364;&#8220;329. <br /> dSAGA. However, wda null animals die earlier dAda2b             10. Celniker, S. E., G. M. Rubin. 2003. The Drosophila melanogaster <br />                                                                          genome. Annu. Rev. Genomics Hum. Genet. 4:89&#226;&#8364;&#8220;117. <br /> null strains, indicate distinct functions   11. Celniker, S. E., D. A. Wheeler, B. Kronmiller, J. W. Carlson, A. Halpern, S. <br /> proteins dSAGA. An alternative explanation                Patel, M. Adams, M. Champe, S. P. Dugan, E. Frise, A. Hodgson, R. A. <br /> difference lethality phases wda dAda2b mutants           George, R. A. Hoskins, T. Laverty, D. M. Muzny, C. R. Nelson, J. M. Pacleb, <br />                                                                          S. Park, B. D. Pfeiffer, S. Richards, E. J. Sodergren, R. Svirskas, P. E. Tabor, <br /> be difference maternal loading stability WDA             K. Wan, M. Stapleton, G. G. Sutton, C. Venter, G. Weinstock, S. E. Scherer, <br /> dAda2B proteins. A higher higher stability ma-              E. W. Myers, R. A. Gibbs, G. M. Rubin. 2002. Finishing whole-genome <br />                                                                          shotgun: release 3 Drosophila melanogaster euchromatic genome <br /> ternal dAda2B enable development mutants              sequence. Genome Biol. 3:RESEARCH0079. <br /> pupal stage.                                                         12. Chen, G., A. J. Courey. 2000. Groucho/TLE family proteins tran- <br />    This study provides additional evidence SAGA com-            scriptional repression. Gene 249:1&#226;&#8364;&#8220;16. <br />                                                                      13. Cheng, H., X. He, C. Moore. 2004. The essential WD repeat protein <br /> plex conserved evolution. The protein complexes            Swd2 dual functions RNA polymerase II transcription termination and <br /> purified yeast, human, fly cells display similar fea-           lysine 4 methylation histone H3. Mol. Cell. Biol. 24:2932&#226;&#8364;&#8220;2943. <br />  VOL. 26, 2006                                                                             IDENTIFICATION OF NEW SUBUNITS IN dSAGA                                    7189 <br />  <br /> 14. Durso, R. J., A. K. Fisher, T. J. Albright-Frey, J. C. Reese. 2001. Analysis   37. McMahon, S. J., M. G. Pray-Grant, D. Schieltz, J. R. Yates III, P. A. <br />     TAF90 mutants displaying allele-specific broad defects transcrip-            Grant. 2005. Polyglutamine-expanded spinocerebellar ataxia-7 protein dis- <br />     tion. Mol. Cell. Biol. 21:7331&#226;&#8364;&#8220;7344.                                                   rupts normal SAGA SLIK histone acetyltransferase activity. Proc. Natl. <br /> 15. Eberharter, A., S. John, P. A. Grant, R. T. Utley, J. L. Workman. 1998.            Acad. Sci. USA 102:8478&#226;&#8364;&#8220;8482. <br />     Identification analysis yeast nucleosomal histone acetyltransferase         38. Noland, B. J., J. M. Hardin, G. R. Shepherd. 1971. Histone acetyltrans- <br />     complexes. Methods 15:315&#226;&#8364;&#8220;321.                                                         ferase activity synchronized mammalian cells. Biochim. Biophys. Acta <br /> 16. Fischle, W., Y. Wang, C. D. Allis. 2003. Histone chromatin cross-              246:263&#226;&#8364;&#8220;268. <br />     talk. Curr. Opin. Cell Biol. 15:172&#226;&#8364;&#8220;183.                                           39. Ogryzko, V. V., T. Kotani, X. Zhang, R. L. Schlitz, T. Howard, X.-J. Yang, <br /> 17. Furlong, E. E., D. Profitt, M. P. Scott. 2001. Automated sorting live           B. H. Howard, J. Qin, Y. Nakatani. 1998. Histone-like TAFs the <br />     transgenic embryos. Nat. Biotechnol. 19:153&#226;&#8364;&#8220;156.                                       PCAF histone acetylase complex. Cell 94:35&#226;&#8364;&#8220;44. <br /> 18. Gangloff, Y. G., C. Romier, S. Thuault, S. Werten, I. Davidson. 2001.          40. Palhan, V. B., S. Chen, G. H. Peng, A. Tjernberg, A. M. Gamper, Y. Fan, <br />     The histone fold key structural motif transcription factor TFIID.              B. T. Chait, A. R. La Spada, R. G. Roeder. 2005. Polyglutamine-ex- <br />     Trends Biochem. Sci. 26:250&#226;&#8364;&#8220;257.                                                       panded ataxin-7 inhibits STAGA histone acetyltransferase activity pro- <br /> 19. Gangloff, Y. G., S. L. Sanders, C. Romier, D. Kirschner, P. A. Weil, L. Tora,          duce retinal degeneration. Proc. Natl. Acad. Sci. USA 102:8472&#226;&#8364;&#8220;8477. <br />     I. Davidson. 2001. Histone folds mediate selective heterodimerization       41. Pankotai, T., O. Komonyi, L. Bodai, Z. Ujfaludi, S. Muratoglu, A. Ciurciu, <br />     yeast TAF(II)25 TFIID components yTAF(II)47 yTAF(II)65                    L. Tora, J. Szabad, I. Boros. 2005. The homologous Drosophila tran- <br />     SAGA component ySPT7. Mol. Cell. Biol. 21:1841&#226;&#8364;&#8220;1853.                              scriptional adaptors ADA2a ADA2b required normal <br /> 20. Gangloff, Y. G., S. Werten, C. Romier, L. Carre, O. Poch, D. Moras, I.             development different functions. Mol. Cell. Biol. 25:8215&#226;&#8364;&#8220;8227. <br />     Davidson. 2000. The human TFIID components TAF(II)135 TAF(II)20                42. Park, J., S. Kunjibettu, S. B. McMahon, M. D. Cole. 2001. The ATM- <br />     yeast SAGA components ADA1 TAF(II)68 heterodimerize                     related domain TRRAP required histone acetyltransferase recruit- <br />     form histone-like pairs. Mol. Cell. Biol. 20:340&#226;&#8364;&#8220;351.                                  ment Myc-dependent oncogenesis. Genes Dev. 15:1619&#226;&#8364;&#8220;1624. <br /> 21. Garcia-Higuera, I., J. Fenoglio, Y. Li, C. Lewis, M. P. Panchenko, O. Reiner,      43. Powell, D. W., C. M. Weaver, J. L. Jennings, K. J. McAfee, Y. He, P. A. Weil, <br />     T. F. Smith, E. J. Neer. 1996. Folding proteins WD-repeats:                A. J. Link. 2004. Cluster analysis mass spectrometry data reveals a <br />     comparison members WD-repeat superfamily G protein                novel component SAGA. Mol. Cell. Biol. 24:7249&#226;&#8364;&#8220;7259. <br />     beta subunit. Biochemistry 35:13985&#226;&#8364;&#8220;13994.                                         44. Qi, D., J. Larsson, M. Mannervik. 2004. Drosophila Ada2b required <br /> 22. Grant, P. A., L. Duggan, J. Co  &#203;&#8224;te&#194;&#180;, S. M. Roberts, J. E. Brownell, R. Candau,        viability normal histone H3 acetylation. Mol. Cell. Biol. 24:8080&#226;&#8364;&#8220; <br />     R. Ohba, T. Owen-Hughes, C. D. Allis, F. Winston, S. L. Berger, J. L.              8089. <br />     Workman. 1997. Yeast Gcn5 functions multisubunit complexes               45. Rorth, P. 1998. Gal4 Drosophila female germline. Mech. Dev. 78:113&#226;&#8364;&#8220; <br />     acetylate nucleosomal histones: characterization Ada complex             118. <br />     SAGA (Spt/Ada) complex. Genes Dev. 11:1640&#226;&#8364;&#8220;1650.                                   46. Sanders, S. L., J. Jennings, A. Canutescu, A. J. Link, P. A. Weil. 2002. <br /> 23. Grant, P. A., D. Schieltz, M. G. Pray-Grant, D. J. Steger, J. C. Reese, J. R.          Proteomics eukaryotic transcription machinery: identification pro- <br />     Yates III, J. L. Workman. 1998. A subset TAFIIs integral com-               teins associated components yeast TFIID multidimensional mass <br />     ponents SAGA complex required nucleosome acetylation                    spectrometry. Mol. Cell. Biol. 22:4723&#226;&#8364;&#8220;4738. <br />     transcriptional stimulation. Cell 94:45&#226;&#8364;&#8220;53. <br />                                                                                        47. Selleck, W., R. Howley, Q. Fang, V. Podolny, M. G. Fried, S. Buratowski, and <br /> 24. Grant, P. A., D. Schieltz, M. G. Pray-Grant, J. R. Yates III, J. L. <br />                                                                                            S. Tan. 2001. A histone fold TAF octamer yeast TFIID transcrip- <br />     Workman. 1998. The ATM-related cofactor Tra1 component the <br />                                                                                            tional coactivator. Nat. Struct. Biol. 8:695&#226;&#8364;&#8220;700. <br />     purified SAGA complex. Mol. Cell 2:863&#226;&#8364;&#8220;867. <br />                                                                                        48. Sendra, R., C. Tse, J. C. Hansen. 2000. The yeast histone acetyltrans- <br /> 25. Guelman, S., T. Suganuma, L. Florens, S. K. Swanson, C. L. Kiesecker, T. <br />                                                                                            ferase A2 complex, free Gcn5p, binds stably nucleosomal arrays. <br />     Kusch, S. Anderson, J. R. Yates III, M. P. Washburn, S. M. Abmayr, and <br />                                                                                            J. Biol. Chem. 275:24928&#226;&#8364;&#8220;24934. <br />     J. L. Workman. 2006. Host cell factor uncharacterized SANT domain <br />                                                                                        49. Smith, E. R., A. Pannuti, W. Gu, A. Steurnagel, R. G. Cook, C. D. Allis, and <br />     protein stable components ATAC, novel dAda2A/dGcn5-containing <br />                                                                                            J. C. Lucchesi. 2000. The Drosophila MSL complex acetylates histone H4 at <br />     histone acetyltransferase complex Drosophila. Mol. Cell. Biol. 26:871&#226;&#8364;&#8220;882. <br />                                                                                            lysine 16, chromatin modification linked dosage compensation. Mol. <br /> 26. Hiller, M. A., T. Y. Lin, C. Wood, M. T. Fuller. 2001. Developmental <br />                                                                                            Cell. Biol. 20:312&#226;&#8364;&#8220;318. <br />     regulation transcription tissue-specific TAF homolog. Genes Dev. <br />     15:1021&#226;&#8364;&#8220;1030.                                                                      50. Smith, T. F., C. Gaitatzes, K. Saxena, E. J. Neer. 1999. The WD repeat: <br /> 27. Hollmann, M., E. Simmerl, U. Schafer, M. A. Schafer. 2002. The es-                 common architecture diverse functions. Trends Biochem. Sci. 24:181&#226;&#8364;&#8220; <br />     sential Drosophila melanogaster gene wds (will die slowly) codes                 185. <br />     WD-repeat protein seven repeats. Mol. Genet. Genomics 268:425&#226;&#8364;&#8220;433.            51. Sterner, D. E., S. L. Berger. 2000. Acetylation histones transcrip- <br /> 28. Ikeda, K., D. J. Steger, A. Eberharter, J. L. Workman. 1999. Activation            tion-related factors. Microbiol. Mol. Biol. Rev. 64:435&#226;&#8364;&#8220;459. <br />     domain-specific general transcription stimulation native histone            52. Sterner, D. E., P. A. Grant, S. M. Roberts, L. J. Duggan, R. Belotser- <br />     acetyltransferase complexes. Mol. Cell. Biol. 19:855&#226;&#8364;&#8220;863.                              kovskaya, L. A. Pacella, F. Winston, J. L. Workman, S. L. Berger. 1999. <br /> 29. Keller, C. A., M. A. Grill, S. M. Abmayr. 1998. A role nautilus         Functional organization yeast SAGA complex: distinct components <br />     differentiation muscle precursors. Dev. Biol. 202:157&#226;&#8364;&#8220;171.                          involved structural integrity, nucleosome acetylation, TATA-binding <br /> 30. Khorasanizadeh, S. 2004. The nucleosome: genomic organization                  protein interaction. Mol. Cell. Biol. 19:86&#226;&#8364;&#8220;98. <br />     genomic regulation. Cell 116:259&#226;&#8364;&#8220;272.                                              53. Strahl, B. D., C. D. Allis. 2000. The language covalent histone <br /> 31. Kusch, T., S. Guelman, S. M. Abmayr, J. L. Workman. 2003. Two                      modifications. Nature 403:41&#226;&#8364;&#8220;45. <br />     Drosophila Ada2 homologues function different multiprotein complexes.           54. Utley, R. T., K. Ikeda, P. A. Grant, J. Co &#203;&#8224;te&#194;&#180;, D. J. Steger, A. Eberharter, S. <br />     Mol. Cell. Biol. 23:3305&#226;&#8364;&#8220;3319.                                                         John, J. L. Workman. 1998. Transcriptional activators direct histone <br /> 32. Lee, K. K., L. Florens, S. K. Swanson, M. P. Washburn, J. L. Workman.              acetyltrans <br /> </body></html>