<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2002 Genetics Society America <br />  <br />  <br />  <br />                       Protein Phosphatase Type 1 Regulates Ion Homeostasis <br />                                     Saccharomyces cerevisiae <br />  <br />                                     Tara Williams-Hart, Xiaolin Wu Kelly Tatchell1 <br />     Department Biochemistry Molecular Biology, Louisiana State University Health Sciences Center, Shreveport, Louisiana 71130 <br />                                                       Manuscript received September 18, 2001 <br />                                                       Accepted publication January 25, 2002 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                Protein phosphatase type 1 (PP1) encoded essential gene GLC7 Saccharomyces cerevisiae. glc7- <br />              109 (K259A, R260A) dominant, hyperglycogen defect recessive, ion drug sensitivity. <br />              Surprisingly, hyperglycogen <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> partially retained <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>null mutants</span> GAC1, GIP2, PIG1, <br />              encode potential glycogen-targeting subunits Glc7. The R260A substitution GLC7 responsible <br />              dominant recessive traits glc7-109. Another <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span> residue, glc7-R260P, confers <br />              salt sensitivity, indicating glycogen salt traits glc7-109 defects distinct <br />              physiological pathways. The glc7-109 mutant sensitive cations, aminoglycosides, alkaline pH and <br />              exhibits increased rates l-leucine 3,3&#226;&#172;&#732;-dihexyloxacarbocyanine iodide uptake, resistant <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />              molar concentrations sorbitol KCl, indicating normal osmoregulation. KCl suppresses the <br />              ion drug sensitivities glc7-109 mutant. The CsCl sensitivity mutant suppressed by <br />              recessive mutations PMA1, encodes essential plasma membrane H&#226;&#171;&#185;ATPase. Together, these <br />              results indicate Glc7 regulates ion homeostasis controlling ion transport and/or plasma membrane <br />              potential, new role Glc7 budding yeast. <br />  <br />  <br />  <br /> P     ROTEIN phosphatase type 1 (PP1) well-charac- <br />      terized serine/threonine protein phosphatase (Shen- <br /> olikar Nairn 1991; Shenolikar 1994) roles <br />                                                                              1995; Baker et al. 1997; Bloecher Tatchell 1999; <br />                                                                              Sassoon et al. 1999), glucose repression (Tu Carl- <br />                                                                              son 1995; Baker et al. 1997), glycogen metabolism <br /> in wide range physiological processes. Multiple iso-                    (Feng et al. 1991; Cannon et al. 1994; Baker et al. 1997; <br /> forms mammalian PP1 implicated cell                          Ramaswamy et al. 1998), vacuole fusion (Peters et al. <br /> cycle regulation (reviewed Shenolikar 1994), facili-                      1999) cell wall integrity (Andrews Stark <br /> tation CREB gene transcription, activation gly-                    2000). Although mammalian PP1 linked to <br /> cogen synthesis lipid metabolism (reviewed                            ion transport, previous work charac- <br /> Shenolikar 1994; Villafranca et al. 1996), neuro-                            terizing role Glc7 regulator ion homeostasis <br /> transmission (Browning et al. 1985; Mulkey et al. 1994;                      S. cerevisiae. <br /> Allen et al. 1997; Greengard et al. 1998; MacMillan                             The maintenance ion homeostasis essential <br /> et al. 1999; Smith et al. 1999), control Ca2&#226;&#171;&#185; trans-                  highly regulated process involving numerous trans- <br /> port cardiac sarcoplasmic reticulum (reviewed                          port systems. These transport systems involved in <br /> Shenolikar Nairn 1991), regulation                            regulating cell volume, maintaining intracellular pH <br /> translational initiation (He et al. 1998; Novoa et al.                       ionic composition, extracting concentrating <br /> 2001). It widely accepted substrate specificity                   environmental metabolic fuels, extruding toxic sub- <br /> dictated targeting subunits facilitate associa-                  stances, generating essential ionic gradients (re- <br /> tion catalytic subunit PP1 (PP1c)                         viewed Andre 1995; van der Rest et al. 1995). The <br /> specific substrates (Egloff et al. 1997).                                    maintenance transport systems fungi and <br />    In Saccharomyces cerevisiae, single, essential gene                   plants driven plasma membrane H&#226;&#171;&#185;ATPase, <br /> GLC7 encodes PP1c case mammalian                            encoded essential gene, PMA1 S. <br /> PP1, Glc7 implicated wide range pro- <br />                                                                              cerevisiae. Pma1 protein H&#226;&#171;&#185; pump responsi- <br /> cesses based primarily diversity GLC7 mutant <br />                                                                              ble regulation intracellular pH, ion transport, <br /> phenotypes. GLC7 mutants described with <br />                                                                              membrane electrochemical potential (Serrano <br /> defects meiosis sporulation (Tu et al. 1996; Baker <br />                                                                              1983a; Chang Slayman 1991; Serrano et al. 1991, <br /> et al. 1997; Ramaswamy et al. 1998; Bailis Roeder <br />                                                                              1992). Evidence suggests Pma1 transcrip- <br /> 2000), mitosis (Hisamoto et al. 1995; MacKelvie et al. <br />                                                                              tionally post-translationally regulated. Transcription <br />                                                                              Pma1 influenced various transcription factors <br />   1 <br />                                                                              stimulated glucose metabolism, ammonium <br />     Corresponding author: Department Biochemistry Molecular <br /> Biology, Louisiana State University Health Sciences Center, 1501 Kings       starvation, cell growth, cell wall integrity (Capieaux <br /> Highway, Shreveport, LA 71130. E-mail: ktatch@lsuhsc.edu                     et al. 1989; Rao Slayman 1993; Garcia-Arranz et <br />  <br /> Genetics 160: 1423&#226;&#8364;&#8220;1437 (April 2002) <br />  1424                                         T. Williams-Hart, X. Wu K. Tatchell <br />  <br /> al. 1994; Ozer et al. 1998). However, mRNA                 (0.67% yeast nitrogen <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span>, 2% glucose, appropriate <br /> protein levels Pma1 change significantly dur-            0.01&#226;&#8364;&#8220;0.05% amino acids). Plate media contained 2% agar. <br />                                                                       Assays ion tolerance performed liquid growth <br /> ing activation glucose, suggesting case            assays spot dilution tests agar media. Liquid growth <br /> post-translational modulation ATPase activity            assays performed diluting stationary phase cultures <br /> the primary mechanism Pma1 regulation (Eraso et                4 &#226;&#171;&#187; 107 cells/ml media described below. Cell concen- <br /> al. 1987).                                                         trations determined measuring optical density 600 <br />   The activity Pma1 changes response extracellu-          nm (OD600) and/or microscopic cell counting using stan- <br /> lar glucose (Portillo et al. 1991; Garcia-Arranz et al.            dard hemacytometer. Cultures incubated specified <br />                                                                    media 1&#226;&#8364;&#8220;3 hr. Spot dilution tests spotting <br /> 1994; van der Rest et al. 1995) Pma1 highly                 serial 10-fold dilutions 1 &#226;&#171;&#187; 108 cells/ml strain <br /> phosphorylated multiple serine threonine resi-              YPD plates, synthetic media, YPD supplemented with <br /> dues vivo (Kolarov et al. 1988; Bertorello et al.               various concentrations salts drugs. Growth strains <br /> 1991; Chang Slayman 1991; Serrano et al. 1991).                pH 3, 5, 8.5 tested liquid YPD medium was <br /> Two-dimensional phosphopeptide gel analysis Pma1                adjusted 10 n KOH 1 m HCl buffered 100 <br />                                                                    mm potassium phosphate 50 mm succinate, respectively. <br /> in temperature-sensitive secretion (sec) mutants re- <br />                                                                    Medium containing 0.2 mm KCl consists synthetic medium <br /> vealed phosphorylation plasma membrane-spe-                containing 0.5% ammonium sulfate, 2% glucose, 8 mm ammo- <br /> cific sites associated increased ATPase activity           nium phosphate, 17 mm NaCl, 2 mm MgSO4, 0.2 mm CaCl2, <br /> during growth glucose (Holcomb et al. 1988; Chang               0.2 mm KCl, plus amino acids, vitamins, trace elements <br /> and Slayman 1991). Upon glucose starvation, dephos-                described (Guthrie Fink 1991). 3,3&#226;&#172;&#732;-Dihexyloxacarbo- <br /> phorylation occurs concomitantly decrease enzy-          cyanine iodide [DiOC6(3)] fluorescence assayed flow <br />                                                                    cytometery described (Madrid et al. 1998). To assay strains <br /> matic activity rapidly reversed addition       0.4 m KCl, strains incubated overnight YPD and <br /> of glucose (Chang Slayman 1991; Estrada et al.                 diluted YPD containing 0.4 m KCl incubated 4 hr <br /> 1996). These results suggest reversible, site-specific        30&#226;&#172;&#353; prior flow cytometry analysis. <br /> phosphorylation adjusts ATPase activity protein               General methods: Escherichia coli strains DH5&#226;?&#163;F&#226;&#172;&#732;, XL1-Blue, <br /> stability (Hasper et al. 1999; Gong Chang 2001)                HB101 used cloning propagation plas- <br /> in response glucose.                                            mids. Diploid cells induced sporulate YPA (2% <br />                                                                    potassium acetate, 2% peptone, 1% yeast extract) agar media. <br />   This work motivated observation glc7-            Yeast transformations performed lithium acetate <br /> 109, 20 charged alanine-scanning alleles,                method described (Gietz et al. 1992). Qualitative glycogen <br /> uniquely confers NaCl-sensitive phenotype (Baker et              accumulation measurements performed described <br /> al. 1997). Defects ion homeostasis pre-               (Chester 1968). Quantitative glycogen accumulation mea- <br /> viously noted null mutants genes encoding               surements performed described (Wu Tatchell <br />                                                                    2001). Isolation plasmid yeast DNA genomic DNA was <br /> phosphatases calcineurin/PP2B (Nakamura et al. 1993; <br />                                                                    performed variation &#226;&#8364;&#339;smash grab&#226;&#8364;? protocols <br /> Breuder et al. 1994; Mendoza et al. 1994; Farcasanu                Hoffman (Hoffman Winston 1987) described by <br /> et al. 1995) Ppz1 (Posas et al. 1995)              Ausubel et al. (1989). DNA sequence determined using <br /> mutants GLC7. We glc7-109 mutants                T7 Sequenase Version 2.0 kit (Amersham, Life Sciences) <br /> are sensitive cations, alkaline pH, aminoglycoside          according manufacturer&#226;&#8364;&#8482;s instructions. <br /> antibiotics. These defects partially ameliorated               Plasmid construction: Plasmids used study listed <br />                                                                    Table 2. Plasmid p1407-1 genomic library clone the <br /> by exogenous K&#226;&#171;&#185; mutations PMA1. The mutant                   CEN4 URA3 vector, YCp50, contains PMA1 460- <br /> exhibits increased rate uptake fluorescent            bp open reading frame, YGL007. To analyze previously <br /> dye DiOC6(3) amino acid l-leucine. Together,               described alanine-scanning allele, glc7-109 (Baker et al. 1997), <br /> these results suggest Glc7 plays role cellular       genomic integration glc7-109 (K259A, R260A) mutation <br /> response salt stress, possibly regulating ion trans-         performed described (Venturi et al. 2000) using plas- <br /> port membrane potential.                                        mid pTW2. Plasmid pXZ03 (a gift James Haber) contains <br />                                                                    HA epitope-tagged PMA1 gene. To create K259A, <br />                                                                    R260A, R260P mutations GLC7, site-directed mutagene- <br />                                                                    sis carried described (Wu Tatchell 2001) <br />                MATERIALS AND METHODS                               using unique oligonucleotide primers. Plasmid pWu361 was <br />    Yeast strains media: The yeast strains used study   constructed using oligonucleotide primer K259A&#226;&#171;&#185; (5&#226;&#172;&#732;-GGTTA <br /> are listed Table 1. All strains used study       TGAATTCTTTAGTGCCAGACAATTGGTGACA-3&#226;&#172;&#732;) the <br /> KT1849 congenic strain KT1112 (MATa ura3-52 leu2            complementary primer K259A&#226;&#171;&#186; (5&#226;&#172;&#732;-TGTCACCAATTGTCTC <br /> his3; Stuart et al. 1994). The trk1::KanMX gip2::KanMX         GGACTAAAGAATTCATAACC-3&#226;&#172;&#732;). Plasmid pWu363 con- <br /> (Research Genetics, Huntsville, AL) cnb1::LEU2 (Cyert          structed using oligonucleotide primer R260A&#226;&#171;&#185; (5&#226;&#172;&#732;-GAATTCT <br /> and Thorner 1992) mutations introduced               TTAGTAAAGCCCAATTGGTGACACTT-3&#226;&#172;&#732;) comple- <br /> KT1112 background seven serial backcrosses. All growth          mentary primer R260A&#226;&#171;&#186; (5&#226;&#172;&#732;-AAGTGTCACCAATTGGGCTT <br /> phenotypes shown study required media contained       TACTAAAGAATTC-3&#226;&#172;&#732;). Plasmid pWu365 constructed us- <br /> Difco (Detroit) peptone, yeast extract, agar. In study,   ing oligonucleotide primer R260P&#226;&#171;&#185; (5&#226;&#172;&#732;-GAATTCTTTAGT <br /> media constituents manufactured Angus Buffers         AAACCGCAATTGGTGACACTT-3&#226;&#172;&#732;) complementary <br /> and Biochemicals impaired growth glc7-109 sup-          primer R260P&#226;&#171;&#186; (5&#226;&#172;&#732;-AAGTGTCACCAATTGCGGTTTACTAAA <br /> pressor mutants salt antibiotic media. Yeast strains        GAATTC-3&#226;&#172;&#732;). The underlined nucleotide residues represent <br /> were routinely incubated 24&#226;&#172;&#353; 30&#226;&#172;&#353; YPD (1% yeast     differences complementary strands R260A <br /> extract, 2% Bacto-peptone, 2% dextrose) synthetic media         R260P oligonucleotide primers. <br />                                                   Glc7-Dependent Ion Transport                                                  1425 <br />  <br />                                                              TABLE 1 <br />                                                            Yeast strains <br />  <br />          Strain                                       Genotype                                              Source <br />          KT1112       MATa his3 leu2 ura3-52                                                        Stuart et al. (1994) <br />          KT1596       MAT&#226;?&#163; his3 leu2 ura3-52 glc7-109                                               Venturi et al. (2000) <br />          KT1935       MATa his3 leu2 trp1 ura3-52 glc7-109                                          This study <br />          KT1937       MATa his3 leu2 trp1 ura3-52                                                   This study <br />          KT1939       MATa his3 leu2 trp1 ura3-52 gac1::LEU2 pig1::URA3                             This study <br />          KT1943       MATa/&#226;?&#163; his3/his3 leu2/leu2 trp1/trp1 ura3-52/ura3-52                          This study <br />          KT1944       MATa/&#226;?&#163; his3/his3 leu2/leu2 trp1/trp1 ura3-52/ura3-52 GLC7/glc7-109            This study <br />          KT1945       MATa/&#226;?&#163; his3/his3 leu2/leu2 trp1/trp1 ura3-52/ura3-52 glc7-109/glc7-109        This study <br />          KT1849       MATa his3 ura3-52 leu1                                                        This study <br />          KT1850       MAT&#226;?&#163; his3 ura3-52 glc7-109 pma1-s4                                            This study <br />          TW25         MATa his3 leu2 ura3-52 glc7-109                                               This study <br />          TW30         MATa his3 leu2 ura3-52 pma1-s1 glc7-109                                       This study <br />          TW31         MATa his3 leu2 ura3-52 pma1-s2 glc7-109                                       This study <br />          TW33         MATa his3 leu2 ura3-52 pma1-s3 glc7-109                                       This study <br />          TW38         MAT&#226;?&#163; his3 leu2 ura3-52 pma1-s4 glc7-109                                       This study <br />          TW43         MAT&#226;?&#163; his3 leu2 ura3-52 pma1-s5 glc7-109                                       This study <br />          TW45         MAT&#226;?&#163; his3 leu2 ura3-52 pma1-s6 glc7-109                                       This study <br />          TW61         MAT&#226;?&#163; his3 leu2 ura3-52 pma1-s1 glc7-109                                       This study <br />          TW62         MATa his3 leu2 ura3-52 pma1-s1                                                This study <br />          TW64         MAT&#226;?&#163; his3 leu2 ura3-52 pma1-s2 glc7-109                                       This study <br />          TW65         MATa his3 leu2 ura3-52 pma1-s2                                                This study <br />          TW68         MAT&#226;?&#163; his3 leu2 ura 3-52 pma1-s3 glc7-109                                      This study <br />          TW69         MATa his3 leu2 ura3-52 pma1-s3                                                This study <br />          TW71         MATa his3 leu2 ura3-52 pma1-s4 glc7-109                                       This study <br />          TW72         MATa his3 leu2 ura3-52 pma1-s4                                                This study <br />          TW75         MATa his3 leu2 ura3-52 pma1-s5 glc7-109                                       This study <br />          TW76         MATa his3 leu2 ura3-52 pma1-s5                                                This study <br />          TW79         MATa his3 leu2 ura3-52 pma1-s6 glc7-109                                       This study <br />          TW80         MATa his3 leu2 ura3-52 pma1-s6                                                This study <br />          TW82         MATa his3 leu2 ura3-52 cnb1::LEU2                                             This study <br />          TW83         MAT&#226;?&#163; his3 leu2 ura3-52 cnb1::LEU2                                             This study <br />          TW90         MAT&#226;?&#163; his3 leu2 ura3-52 cnb1::LEU2 glc7-109                                    This study <br />          TW272        MAT&#226;?&#163; his3 leu2 ura3 trk1::KanMX                                               This study <br />          TW321        MAT&#226;?&#163; his3 leu2 ura3-52 trp1 gip2::KanMX glc7-109                              This study <br />          WU318        MATa his3 leu2 ura3-52 trp1 glc7::LEU2, pNC160-HA-glc7-R260A                  This study <br />          WU320        MATa his3 leu2 ura3-52 trp1 glc7::LEU2, pNC160-HA-glc7-R260P                  This study <br />          WU322        MATa his3 leu2 ura3-52 trp1 glc7::LEU2, pNC160-HA-glc7-109                    This study <br />          WU341        MATa his3 leu2 ura3-52 trp1 glc7::LEU2, pNC160-HA-glc7-K259A                  This study <br />  <br />  <br />  <br />  <br />    Isolation characterization glc7-109 revertants: Three     forming tetrad analysis glc7-109 diploid strains heteroallelic <br /> independent cultures glc7-109 strains (KT1596 TW25)          different suppressors. On basis complementa- <br /> were grown YPD 24&#226;&#172;&#353; 3 &#226;&#171;&#187; 107 cells/ml, plated 2 &#226;&#171;&#187;         tion genetic linkage analysis, suppressors were <br /> 104 cells/ml YPD plates containing 0.1 m CsCl, incu-         assigned complementation group. <br /> bated 2 days 24&#226;&#172;&#353;. This concentration CsCl completely         Cloning suppressor locus: A pma1-s2 glc7-109 mutant <br /> inhibits growth glc7-109 mutant. Six 105 Cs&#226;&#171;&#185;-         strain (TW31) transformed yeast genomic library <br /> tolerant revertants characterized detail. TW30 (pma1-       CEN URA3 vector YCp50, transformants were <br /> s1 glc7-109), TW31 (pma1-s2 glc7-109), TW33 (pma1-s3            screened growth 37&#226;&#172;&#353; failure grow synthetic <br /> glc7-109) isolated culture, TW38 (pma1-s4       media containing 0.1 m CsCl. One transformant com- <br /> glc7-109) isolated second culture, TW43            plemented traits obtained approximately <br /> (pma1-s5glc7-109) TW45 (pma1-s6 glc7-109) isolated         66,000 transformants. A plasmid (p1407-1) recovered <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> from culture. To assay dominance, revertant      transformant verified responsible com- <br /> was mated glc7-109 strain resulting diploids      plementation retransformation TW31. p1407-1 was <br /> assayed resistance CsCl glycogen accumulation.           able complement cold sensitivity TW60 (pma1-s1 glc7- <br /> To determine reversion events extragenic,          109), TW38 (pma1-s4 glc7-109), TW78 (pma1-s6 glc7-109). <br /> revertant mated GLC7 strain, resultant diploids        The sequence p1407-1 determined junctions <br /> were sporulated, tetrad analysis performed assay         vector genomic insert using T7 Sequenase <br /> for recovery glc7-109 phenotype. Linkage comple-         Version 2.0 (Amersham, Arlington Heights, IL) primers <br /> mentation suppressors determined per-            annealed YCp50 vector flanking insert. The <br />  1426                                         T. Williams-Hart, X. Wu K. Tatchell <br />  <br />                                                                 TABLE 2 <br />                                                                 Plasmids <br />  <br />          Name               Vector                Description                                   Source <br />          p1407-1           YCp50            CEN4 URA3 PMA1                 YCp50-based genomic library D. S. Conklin <br />          pAMS366                            2&#226;?&#174; URA3 4XCDRE                 Stathopoulos Cyert (1997) <br />          pNC160                             CEN3 TRP1                      Rhodes et al. (1990) <br />          pRS306                             URA3                           Sikorski Hieter (1989) <br />          pRS316                             CEN6 URA3                      Sikorski Hieter (1989) <br />          pSB64             pNC160           CEN3 TRP1 glc7-109             Baker et al. (1997) <br />          pTW2              pRS306           URA3 glc7-109                  This study <br />          pWu361            pNC160           CEN3 TRP1 K259A                This study <br />          pWu363            pNC160           CEN3 TRP1 R260A                This study <br />          pWu365            pNC160           CEN3 TRP1 R260P                This study <br />          pXZ03             YCp50            CEN4 URA3 HA-PMA1              James Haber <br />          YCp50                              CEN4 URA3                      Rose et al. (1987) <br />          YEp352                             2&#226;?&#174; URA3                        Hill et al. (1986) <br />  <br />  <br />  <br /> PMA1 gene p1407-1 solely responsible complemen-            dase-conjugated secondary antibody (Amersham). Pma1p lev- <br /> tation revertant phenotype, shown ability         els quantitatively measured using NIH Image phos- <br /> plasmid pXZ03 (a gift James Haber), contains               phoglycerate kinase loading control compare Pma1p <br /> only PMA1 gene, complement conditional pheno-              5, 10, 15 &#226;?&#174;g total protein extracts wild-type and <br /> types restore CsCl sensitivity TW61 (pma1-s1 glc7-109),        glc7-109 cells. Protein concentrations determined de- <br /> TW64 (pma1-s2 glc7-109), TW71 (pma1-s4 glc7-109).                 scribed (Bradford 1976) Bio-Rad protein assay re- <br />    Genetic analysis pma1 calcineurin strains: Each          agent (Bio-Rad Laboratories, Hercules, CA) using bovine se- <br /> the suppressor strains TW61 (pma1-s1 glc7-109), TW64 (pma1-           rum albumin standard. Membrane fractions separated <br /> s2 glc7-109), TW68 (pma1-s3 glc7-109), TW71 (pma1-s4 glc7-            10% SDS-polyacrylamide gels immunoblotted de- <br /> 109), TW75 (pma1-s5 glc7-109), TW79 (pma1-s6 glc7-109)            scribed (Ausubel et al. 1989). Total membrane fractions were <br /> were mated strain TW82 (cnb1::LEU2) TW83 (cnb1::                isolated VO4-sensitive ATPase activity assayed pH 5.7 <br /> LEU2). The resultant diploid strains sporulated          5 mm ATP described (Serrano 1983b). To determine <br /> haploid segregants assayed conditional growth defects        relative membrane potential strains, performed flow <br /> and growth leu&#226;&#171;&#186;, salt, drug agar media described               cytometry analysis using FACS Calibur (Becton Dickinson) <br /> above. The mutants containing suppressor (pma1)              described (Madrid et al. 1998). <br /> calcineurin (cnb1::LEU2) characterized. <br />    l-Leucine uptake assays: l-Leucine uptake assays per- <br /> formed previously described (Kotliar et al. 1994; Norbeck <br /> and Blomberg 1998) modifications. Briefly,                                        RESULTS <br /> l-leucine uptake assays performed wild-type and <br /> glc7-109 haploid strains grown OD600 &#226;&#181;&#8216;0.5 (5 &#226;&#171;&#187; 106              glc7-109 mutants recessive, salt sensitive and <br /> cells/ml) 30&#226;&#172;&#353; synthetic media supplemented 0.01%           dominant, hyperglycogen phenotype: A single GLC7 <br /> leucine, histidine, uracil. A solution l-14C-radiolabeled      mutant (glc7-109) group 20 alanine-scanning <br /> leucine (11.2 GBq/mmol; CEB67, Amersham) 304 mCi/                  mutants uniquely sensitive high concentrations <br /> mmol added cell culture final concentration           NaCl (Baker et al. 1997). It accumulated higher <br /> of 0.05 mm. At 1, 2, 3, 4, 5 min, 750-&#226;?&#174;l sample cells <br /> containing l-14C-leucine withdrawn mixed 750 &#226;?&#174;l          normal levels glycogen, observed previously <br /> ice-cold, nonlabeled 0.1 m l-leucine solution stop uptake.         alleles (Baker et al. 1997; Ramaswamy <br /> The cells immediately filtered washed ice-cold          et al. 1998). In crosses strains containing chro- <br /> 0.1 m l-leucine solution described. The l-14C-radiolabeled         mosomal glc7-109, sensitivity variety monovalent <br /> leucine measurements represent average triplicate ex- <br />                                                                       divalent ions aminoglycoside antibiotics <br /> periments (Norbeck Blomberg 1998). <br />    Biochemical methods: To assay calcineurin-dependent re-            segregated hyperglycogen phenotype, indicat- <br /> sponsive element (CDRE)-dependent gene expression, strains            ing single mutation responsible both <br /> were transformed pAMS366 (Stathopoulos-Geron-                    traits. In crosses noted salt drug <br /> tides et al. 1999) exponentially growing transformants            sensitivities recessive GLC7 glycogen <br /> (1&#226;&#8364;&#8220;5 &#226;&#171;&#187; 107 cells/ml) assayed &#226;?&#164;-galactosidase activity <br /> as described (Miller 1972; Kaiser et al. 1994). Yeast protein <br />                                                                       hyperaccumulation trait clearly dominant. As shown <br /> extracts prepared described (Stuart et al. 1994)         Figure 1A, diploid strain heterozygous glc7-109 <br /> cells density 1 &#226;&#171;&#187; 108 cells/ml precipitated          resistant CsCl paromomycin sulfate the <br /> 10% trichloroacetic acid described (Dufour Goffeau             wild-type GLC7 strain accumulates glycogen levels <br /> 1978). Pma1p levels determined total protein ex-            intermediate glc7-109 homozygous strain <br /> tracts described (Withee et al. 1998) using affinity-purified <br /> rabbit polyclonal antibody generously provided Carolyn             wild type, indicating hyperglycogen <br /> Slayman (Nakamoto et al. 1991). The signal visualized             phenotype glc7-109 semidominant. <br /> with enhanced chemiluminesence horseradish peroxi-                 To test possibility dominant glycogen <br />                                                  Glc7-Dependent Ion Transport                                             1427 <br />  <br />  <br />  <br />  <br />    Figure 1.&#226;&#8364;&#8221;The salt sensitivity glycogen hyperaccumula-      Figure 2.&#226;&#8364;&#8221;An nonhydrogen atom model crystal <br /> tion traits glc7-109 mutant separable. (A) Strains   structure rabbit PP1c (Goldberg et al. 1995) showing the <br /> were incubated YPD medium 24&#226;&#172;&#353; 2 days         locations amino acid residues altered glc7-109 <br /> replica plated agar containing YPD, YPD supplemented       mutant. Amino acid residues make contact GM[63-75] <br /> with 1 mg/ml paromomycin sulfate (Paro), 0.1 m CsCl,         (Egloff et al. 1997) shown red. R261, amino acid <br /> and synthetic complete medium stained iodine (I2)       residue altered Glc7-109, shown yellow. All colored <br /> described materials methods. The glycogen [Gly]          residues PP1c identical Glc7 D242, is <br /> accumulation strains quantitated described     E241 Glc7. Glc7 lacks amino acid residue NH2 <br /> materials methods. The strains used wild-type hap-      terminus present PP1c, related amino acids <br /> loid diploid, KT1937 KT1943, respectively; GLC7/        Glc7 PP1c offset residue (i.e., R261 PP1c <br /> glc7-109, KT1944; glc7-109/glc7-109, KT1945; K259A; glc7-   R260 Glc7). Modeling performed using RasMac v2.6 <br /> 109, KT1935. (B) Strains incubated YPD medium           software (Roger Sayle, Glaxo Wellcome Medicines Research <br /> at 24&#226;&#172;&#353; 2 days replica plated YPD, YPD         Centre). <br /> supplemented 1 mg/ml paromomycin sulfate, 0.1 m <br /> CsCl, 0.9 m NaCl, YPD stained iodine described <br /> in materials methods. The strains used GLC7,            glycogen accumulation (Franc&#194;&#184;ois et al. 1992; Stu- <br /> KT1937; glc7-109 (K259A R260A), WU322; glc7-K259A, <br />                                                                 art et al. 1994) mutations GLC7 GAC1 that <br /> WU341; glc7-R260A, WU318; glc7-R260P, WU320. <br />                                                                 prevent association cause reduction glycogen <br />                                                                 levels (Stuart et al. 1994; Wu et al. 2001). Gac1 most <br /> and recessive salt-sensitive phenotype glc7-109          likely makes multiple contacts Glc7, judged by <br /> independently caused missense mutations           locations amino acid substitutions Glc7 that <br /> glc7-109 (K259A R260A), constructed mutants              exhibit reduced interactions Gac1 (Baker et al. <br /> that contained single missense mutations. As shown          1997; Ramaswamy et al. 1998; Wu et al. 2001). One of <br /> in Figure 1B, glc7-R260A confers phenotype similar         contacts hydrophobic channel thought <br /> that glc7-109 glc7-K259A confers phenotype           interact conserved VXF-motif many <br /> very similar wild type. Another missense         PP1- Glc7-targeting subunits (Egloff et al. 1997). <br /> mutant proline substituted Arg260 retains          Arg260, residue altered Glc7-109 responsi- <br /> salt paromomycin sulfate sensitivities R260A,        ble hyperglycogen phenotype, near F256, a <br /> has lost hyperglycogen phenotype. In fact, glycogen         residue hydrophobic channel makes contact <br /> levels reduced wild type           phenylalanine residue VXF-motif (Egloff <br /> glc7-R260P mutant (Figure 1B). The fact glc7-          et al. 1997). A Glc7 F256A mutant defects binding <br /> R260P strain retains salt-sensitive trait glc7-     multiple targeting subunits (Wu Tatchell <br /> 109 supports hypothesis salt-sensitive         2001), supporting hypothesis residue has <br /> glycogen traits physiologically unrelated.                  role binding VXF-containing proteins. The loca- <br />    glc7-109 partially bypasses requirement          tions R260 hydrophobic channel high- <br /> Glc7-regulatory subunits glycogen accumulation:             lighted crystal structure PP1 (Figure 2). On <br /> The Glc7 holoenzyme dephosphorylates acti-             basis structure, propose R260 might <br /> vates glycogen synthase thought contain Gac1,           partially occlude binding Gac1; substitution of <br /> targeting subunit binds directly Glc7          residue alanine increase affinity for <br /> glycogen synthase (Wu et al. 2001). Gac1 rate limiting       Gac1, resulting increase concentration of <br />  1428                                        T. Williams-Hart, X. Wu K. Tatchell <br />  <br />                                                                   did lower serial dilution, presumably reflecting <br />                                                                   later stage growth phase. These results indicate <br />                                                                   glc7-109 partially bypasses need Gac1, Pig1, <br />                                                                   Gip2 glycogen accumulation, have <br />                                                                   important implications role Glc7 glycogen <br />                                                                   metabolism (see discussion). <br />                                                                      Glc7 calcineurin act different pathways <br />                                                                   affect ion homeostasis: Yeast strains lacking ser/ <br />                                                                   thr protein phosphatase calcineurin/PP2B sensitive <br />                                                                   high concentrations Na&#226;&#171;&#185;, Li&#226;&#171;&#185;, Mn2&#226;&#171;&#185; (Naka- <br />                                                                   mura et al. 1993; Breuder et al. 1994; Mendoza et al. <br />                                                                   1994; Farcasanu et al. 1995), failure to <br />                                                                   induce expression number membrane ATPases. <br />                                                                   The sensitivity calcineurin mutants Mn2&#226;&#171;&#185; least <br />                                                                   partly caused reduced transcription PMR1 (Cun- <br />                                                                   ningham Fink 1996), encoding Golgi P-type <br />                                                                   ATPase (Antebi Fink 1992); sensitivity Na&#226;&#171;&#185; and <br />                                                                   Li&#226;&#171;&#185; caused reduced transcription PMR2/ENA1 <br />                                                                   (Mendoza et al. 1994), gene encoding major <br />                                                                   plasma membrane Na&#226;&#171;&#185;ATPase (Rudolph et al. 1989). <br />                                                                   Calcineurin mutants defective induction of <br />                                                                   Pmc1, Ca2&#226;&#171;&#185;ATPase, causes resistant <br />    Figure 3.&#226;&#8364;&#8221;glc7-109 glycogen-targeting subunits. Serial     media containing 200 mm CaCl2 (Cunningham and <br /> dilutions strains designated genotypes incu- <br /> bated 24 hr 30&#226;&#172;&#353; YPD, YPD containing 0.1 m CsCl, <br />                                                                   Fink 1994). These transcriptional defects the <br /> and YPD stained iodine. Only patches lowest     failure calcineurin mutants activate Crz1/Tcn1 <br /> serial dilution imaged YPD YPD &#226;&#171;&#185; CsCl        transcription factor (Matheos et al. 1997; Statho- <br /> the lowest second lowest dilutions imaged        poulos Cyert 1997). <br /> iodine-treated samples. The strains ascosporal clones       The apparent similarity calcineurin and <br /> a cross KT1939 (MATa gac1::LEU2 pig1::URA3) and <br /> TW321 (MAT&#226;?&#163; gip2::KanMX glc7-109). <br />                                                                   glc7-109 phenotypes prompted directly compare <br />                                                                   mutants. We used cnb1::LEU2 null mutant for <br />                                                                   comparison. CNB1 encodes conserved regulatory <br /> the Gac1-Glc7 holoenzyme increased glycogen lev-              subunit calcineurin (Kuno et al. 1991; Cyert and <br /> els. Consistent hypothesis, detected in-          Thorner 1992) necessary phosphatase activ- <br /> creased interaction Glc7-109 Gac1 using               ity. The cnb1::LEU2 glc7-109 mutants qualita- <br /> the two-hybrid assay (D. L. Frederick K. Tatchell,            tive quantitative differences ion sensitivity. The <br /> unpublished data). If occlusion model respon-          glc7-109 mutant generally sensitive mono- <br /> sible hyperglycogen phenotype glc7-109             divalent ions. As shown Figure 4, glc7-109 <br /> mutant, gac1 null mutation epistatic           mutant fails grow YPD medium containing 25 mm <br /> to glc7-109. Contrary expectations, glc7-109 gac1::LEU2        CsCl, cnb1::LEU2 strain exhibits defect <br /> mutants accumulate high levels glycogen (Figure 3).            higher concentrations tested (50 100 mm). <br /> Thus, Gac1 binding responsible glc7-109            glc7-109 cnb1::LEU2 mutants equally sensitive to <br /> phenotype.                                                        5 mm MnCl2, glc7-109 mutant sensitive <br />    The yeast genome contains additional genes               Li&#226;&#171;&#185; Na&#226;&#171;&#185; (data shown). Furthermore, the <br /> whose products similar Gac1. PIG1, PIG2,               glc7-109 strain does exhibit calcium resistance <br /> GIP2 identified two-hybrid screen Gsy2-             observed calcineurin mutants (Figure 4). Differ- <br /> interaction proteins (Cheng et al. 1997; Wu et al. 2001)          ences noted temperature dependence <br /> while Gip2 identified independently Glc7p-          Li&#226;&#171;&#185; sensitivity glc7-109 cnb1::LEU2 strains. The <br /> interacting protein (Tu et al. 1996). Although null muta-         glc7-109 mutant exhibits greater sensitivity LiCl 24&#226;&#172;&#353; <br /> tions genes reported little effect          30&#226;&#172;&#353;, temperature differential was <br /> on glycogen levels, tested products             noted cnb1::LEU2 strain. An additional differ- <br /> several necessary hyperglyco-           ence sensitivity aminoglycoside antibiotics. <br /> gen phenotype glc7-109 mutant. Glycogen levels             The cnb1::LEU2 mutant does exhibit growth defects <br /> were elevated glc7-109 strains containing deletions            media containing 50 &#226;?&#174;g/ml hygromycin B (Figure <br /> of GAC1, PIG1, GIP2 stationary phase,              4) 1 mg/ml paromomycin sulfate (data shown) <br /> glycogen levels dropped wild type. As           contrast hypersensitivity glc7-109 mutants. <br /> shown Figure 3, higher serial dilution gac1         An indication Glc7 calcineurin act separate <br /> pig1 gip2 glc7-109 strain stained iodine vapor          pathways finding glc7-109 cnb1::LEU2 dou- <br />                                                 Glc7-Dependent Ion Transport                                              1429 <br />  <br />  <br />  <br />  <br />   Figure 4.&#226;&#8364;&#8221;The phenotypes glc7-109 calcineurin-deficient (cnb1::LEU2) mutants. Serial dilutions strains were <br /> incubated indicated agar media 2&#226;&#8364;&#8220;4 days 24&#226;&#172;&#353;. The strains used wild type (KT1112), glc7-109 (TW25), <br /> cnb1::LEU2 (TW82), glc7-109 cnb1::LEU2 (TW90). <br />  <br />  <br />  <br /> ble mutant sensitive ions                   The glc7-109 strain exhibits increased rates l-leu- <br /> single mutants. If Glc7 calcineurin act           cine uptake: The glc7-109 mutant hypersensitive to <br /> pathway affect ion homeostasis, pre-             aminoglycoside antibiotics paromomycin sulfate <br /> dict epistatic relationship mutants.           hygromycin B. Since mechanisms action of <br /> Instead, ion defects conferred           antibiotics different, likely the <br /> two mutations additive synergistic; glc7-109           hypersensitivity glc7-109 strains drugs is <br /> cnb1 mutant sensitive Mn2&#226;&#171;&#185;, Li&#226;&#171;&#185;, Cs&#226;&#171;&#185;               caused defects uptake export. Although we <br /> than single mutant (Figure 4). Calcineurin              able distinguish two <br /> exerts physiological effects regu-        possibilities, note glc7-109 strains grow more <br /> lation transcription factor Tcn1/Crz1 (Matheos             rapidly wild type synthetic media containing <br /> et al. 1997; Stathopoulos Cyert 1997; Statho-                 low concentrations amino acids (data shown). <br /> poulos-Gerontides et al. 1999). As shown Table 3,              To confirm difference change in <br /> the cnb1::LEU2 strain fails induce expression           amino acid uptake measured l-leucine uptake in <br /> CRZ1-dependent Lac-Z gene. The levels &#226;?&#164;-galactosi-             glc7-109 mutants The rate l-leucine uptake the <br /> dase activity glc7-109 strain 35&#226;&#8364;&#8220;70%          glc7-109 mutant approximately twofold higher than <br /> found wild type orders magni-        wild type (Table 3). <br /> tude higher cnb1::LEU2 strain.               Many defects glc7-109 K&#226;&#180;&#8482; remedial: To rule <br /> These data indicate substrates calcineurin            possibility sensitivity glc7-109 mutants <br /> and Glc7 responsible ion sensitivities         high concentrations ions osmotic ef- <br /> are likely different.                                       fects, tested growth properties glc7-109 strains <br />  <br />  <br />  <br />                                                           TABLE 3 <br />                                           Comparison glc7-109 cnb1 mutants <br />  <br />                                           CDRE activationa <br />                                                                                l-Leucine      Pma1          Pma1 <br />         Genotype          Uninduced     Induced (Ca2&#226;&#171;&#185;)       Induced (Mn2&#226;&#171;&#185;)     uptakeb      amountc       activityd <br />         GLC7 CNB1          88 &#226;&#171;&#190; 7.0       1314 &#226;&#171;&#190; 185          2212 &#226;&#171;&#190; 56       2.10 &#226;&#171;&#190; 0.26      100       0.24 &#226;&#171;&#190; 0.01 <br />         glc7-109 CNB1      73 &#226;&#171;&#190; 9.1        946 &#226;&#171;&#190; 245           770 &#226;&#171;&#190; 153      5.62 &#226;&#171;&#190; 1.2     90 &#226;&#171;&#190; 17     0.20 &#226;&#171;&#190; 0.02 <br />         GLC7 cnb1           3 &#226;&#171;&#190; 0.6          3 &#226;&#171;&#190; 0.6             2 &#226;&#171;&#190; 0.6          ND           ND            ND <br />           ND, determined. <br />           a <br />             Wild-type (KT1112), glc7-109 (KT1596), calcineurin-deficient (TW82) strains transformed with <br />         pAMS366, contains calcineurin-dependent responsive elements (4XCDREs) LacZ reporter gene <br />         (Stathopoulos Cyert 1997). CDRE activation quantitated Miller units &#226;?&#164;-galactosidase activity <br />         (Miller 1972). &#226;?&#164;-Galactosidase activity measured described materials methods growth <br />         selective media media supplemented 200 mm CaCl2 10 mm MnCl2 indicated. Values represent <br />         average separate experiments. <br />           b <br />             Micromoles [14C]leucine gram dry weight minute; data represent average separate <br />         experiments. <br />           c <br />             The abundance Pma1 protein cell extracts determined described materials and <br />         methods. Values represented median percentage Pma1 relative 100% wild type (KT1112) <br />         separate determinations. <br />           d <br />             Inorganic phosphate (micromoles milligram minute) liberated; values presented represent the <br />         average separate experiments. <br />  1430                                       T. Williams-Hart, X. Wu K. Tatchell <br />  <br />                                                                  cessive, shown CsCl sensitivity diploid strains <br />                                                                  heterozygous suppressor mutations homozy- <br />                                                                  gous glc7-109. To determine mutations respon- <br />                                                                  sible suppression extragenic GLC7, tetrad <br />                                                                  analysis performed diploid strains created by <br />                                                                  mating revertants wild-type strain. <br />                                                                  The salt-sensitive phenotype glc7-109 observed in <br />                                                                  approximately one-fourth spore clones each <br />                                                                  cross, indicating mutation responsible the <br />                                                                  reversion unlinked GLC7. To determine the <br />                                                                  suppressors complementation group, <br />                                                                  diploid strains constructed homozygous <br />                                                                  glc7-109 heterozygous different suppres- <br />                                                                  sors. All diploids resistant 0.1 m CsCl, indicating <br />                                                                  suppressors complementation <br />                                                                  group. Furthermore, tetrad analysis meiotic progeny <br />                                                                  diploid strains revealed spore clones <br />                                                                  retained Cs&#226;&#171;&#185;-resistant phenotype parents, <br />                                                                  indicating suppressor loci tightly linked. <br />                                                                     Although suppressors genetically linked, <br />                                                                  confer diverse phenotypes. While revertants <br />   Figure 5.&#226;&#8364;&#8221;KCl suppresses salt drug sensitivities    able grow media containing 0.1 m CsCl 0.9 <br /> the glc7-109 mutant. Serial dilutions wild type (KT1112)      m NaCl, pma1-s1, pma1-s3, pma1-s4 suppress <br /> and glc7-109 (TW25) strains incubated 2&#226;&#8364;&#8220;4 days       Li&#226;&#171;&#185; defect glc7-109 (Figure 6). In contrast, pma1- <br /> 24&#226;&#172;&#353; media containing YPD, YPD containing 1 m KCl, 0.05 m      s1 pma1-s4 strains actually grow better wild- <br /> CsCl, 0.05 m CsCl &#226;&#171;&#185; 0.2 m KCl, 0.1 m LiCl, 0.1 m LiCl &#226;&#171;&#185; 0.2 m <br /> KCl, 50 &#226;?&#174;g/ml hygromycin B, 50 &#226;?&#174;g/ml hygromycin B &#226;&#171;&#185; 0.2 m        type strain 0.1 m LiCl (Figure 6). No revertant strains <br /> KCl, 1 mg/ml paromomycin sulfate, 1 mg/ml paromomy-          containing pma1-s2 exhibit glc7-109 <br /> cin sulfate &#226;&#171;&#185; 0.2 M KCl.                                         growth defect 10 mm MnCl2 (Figure 6). Some of <br />                                                                  revertants conditional growth defects in <br />                                                                  glc7-109 genetic background. pma1-s2 pma1-s5 <br />                                                                  strains grow slowly wild-type strain 37&#226;&#172;&#353; <br /> on YPD media containing 1 m KCl 1 m sorbitol.                pma1-s1, pma1-s4, pma1-s6 strains grow more <br /> While growth defects observed media              slowly wild type 15&#226;&#172;&#353; (Figure 6). These six <br /> only KCl sorbitol, KCl partially relieved salt        suppressor mutants characterized wild- <br /> drug sensitivities glc7-109 strains. The presence          type GLC7 background. pma1-s6 strains grow poorly on <br /> 200 mm KCl medium reduces sensitivity              0.1 m CsCl 0.1 m LiCl, pma1-s2 grows poorly <br /> glc7-109 Cs&#226;&#171;&#185;, Li&#226;&#171;&#185;, hygromycin B, paromomycin              LiCl (Figure 7). In contrast, pma1-s1, pma1-s3, <br /> sulfate (Figure 5), Mn2&#226;&#171;&#185; Na&#226;&#171;&#185; (data                pma1-s4, pma1-s5 strains grow faster wild- <br /> shown). The K&#226;&#171;&#185; remedial phenotype caused glc7-109             type strain LiCl (Figure 7). Most conditional <br /> led examine effect growth low concentra-         growth defects observed glc7-109 background were <br /> tions KCl, mutants defective potassium         apparent GLC7&#226;&#171;&#185; background. For example, <br /> transporters Trk1 Trk2 hypersensitive hygro-          pma1-s1 pma1-s4 strains grow slowly the <br /> mycin B toxic cations (Mulet et al. 1999). Growth            wild type 15&#226;&#172;&#353; (Figure 7) pma1-s2 strains grow <br /> of yeast cells media containing low concentrations            slowly wild type 37&#226;&#172;&#353; (data shown). <br /> of KCl (0.2 mm) requires activity high-affinity          The glc7-109 suppressors allelic PMA1, encod- <br /> K&#226;&#171;&#185; transporter encoded TRK1 (Gaber et al. 1988).              ing essential plasma membrane H&#226;&#180;&#8482;ATPase: The sup- <br /> We trk1 strain (TW272) failed grow            pressor locus cloned complementation the <br /> synthetic medium containing 0.2 mm KCl glc7-             temperature sensitivity strain TW64 (pma1-s2 glc7- <br /> 109 wild-type strains grew equally me-          109) described materials methods. The <br /> dium. Therefore, ion sensitivities glc7-109           complementing plasmid clone (p1407-1) comple- <br /> mutant unlikely caused defect high-            mented cold-sensitive defects strains TW30 (pma1- <br /> affinity K&#226;&#171;&#185; transport.                                           s1 glc7-109) TW38 (pma1-s4 glc7-109). These trans- <br />    Suppressors Cs&#226;&#180;&#8482;-sensitive phenotype glc7-           formants grew 37&#226;&#172;&#353; 15&#226;&#172;&#353;, failed grow <br /> 109: We characterized spontaneous glc7-109 re-               0.1 m CsCl media. Sequence analysis revealed that <br /> vertants grew YPD medium containing 0.1 m                genomic insert p1407-1 contained PMA1 an <br /> CsCl, described materials methods. The                 adjacent 378-bp open reading frame, YGL007. A plasmid <br /> mutations responsible reversion events re-          (pXZ03) containing PMA1 lacking YGL007 also <br />                                                 Glc7-Dependent Ion Transport                                               1431 <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;pma1 mutations suppress Cs&#226;&#171;&#185; sensitivity conferred glc7-109. Serial dilutions pma1 mutants glc7-109 <br /> background plated YPD medium, YPD media supplemented designated salts, synthetic complete (SC) <br /> medium. The plates incubated 2 4 days. The strains used wild type (KT1112), glc7-109 (TW25), pma1- <br /> s1 glc7-109 (TW30), pma1-s2 glc7-109 (TW31), pma1-s3 glc7-109, (TW33), pma1-s4 glc7-109 (TW38), pma1-s5 glc7-109 (TW43), and <br /> pma1-s6 glc7-109 (TW45). <br />  <br />  <br />  <br /> able suppress temperature sensitivity Cs&#226;&#171;&#185;             strongly suppress glc7-109 showed greatest <br /> resistance TW64 (pma1-s2 glc7-109), indicating           suppression cnb1::LEU2. <br /> PMA1 responsible complementation. To con-                Many pma1 mutants sensitive low pH (McCusker <br /> firm suppressor mutations alleles PMA1,          et al. 1987; Withee et al. 1998). We compared growth <br /> we mapped distance pma1-s4 leu1,                 glc7-109 mutant cold-sensitive pma1 mutant <br /> which lies &#226;&#181;&#8216;1 cM centromere proximal PMA1. Tetrad             strain, TW71 (pma1-s4 glc7-109), liquid YPD media at <br /> analysis cross KT1850 (MAT&#226;?&#163; ura3-52 his3            pH 3, 5, 8.5. At pH 5, growth wild-type, <br /> glc7-109 pma1-s4) KT1849 (MATa ura3-52 his3 leu1)            glc7-109, pma1-s4 glc7-109 strains compara- <br /> revealed distance leu1 pma1-s4 1.1       ble (data shown). As expected, suppressor strain <br /> cM (P:NP:T &#226;&#171;&#189; 43:0:1), identical previously deter-         TW71 (pma1-s4 glc7-109) grows slowly acidic pH but <br /> mined map distance (P:NP:T &#226;&#171;&#189; 869:0:22; McCusker              unexpectedly, glc7-109 strain grows slowly alkaline <br /> Haber 1988). Together, linkage complementa-              pH (Figure 8). <br /> tion data indicate glc7-109 suppressors            The ability pma1 mutations suppress salt- <br /> alleles PMA1.                                                 sensitive phenotype glc7-109 led assay Pma1 <br />    PMA1 mutants isolated suppress             levels activity glc7-109 pma1-s4 mutants. Pma1 <br /> salt sensitivities calcineurin mutants (Nass et al. 1997;     levels glc7-109 strain &#226;&#181;&#8216;90% wild <br /> Withee et al. 1998). To determine glc7-109 sup-           type, assayed immunoblot analysis. Vanadate-sensi- <br /> pressors suppress defects, crossed strains         tive ATPase activities, measure Pma1 activity, were <br /> containing suppressors cnb1 strain          comparable wild-type glc7-109 strains (Table <br /> assayed meiotic products salt sensitivity. Three         3), pma1-s4 mutant 61% wild <br /> of suppressors fail suppress ion sensitivities        type (data shown). Indirect immunofluorescence <br /> of cnb1, partially suppress Na&#226;&#171;&#185;          revealed obvious differences glc7-109 and <br /> and/or Li&#226;&#171;&#185; defects cnb1 strain (Table 4). The             wild-type strains Pma1 localization (data shown). <br /> two cold-sensitive mutations (pma1-s1 pma1-s4)          Thus, compelling evidence glc7-109 di- <br />  <br />  <br />  <br />  <br />                                                                                 Figure 7.&#226;&#8364;&#8221;The phenotype pma1 mutants <br />                                                                              GLC7 background. Serial dilutions pma1 <br />                                                                              mutants GLC7&#226;&#171;&#185; background plated onto <br />                                                                              YPD medium, YPD media supplemented the <br />                                                                              designated salts, synthetic complete (SC) me- <br />                                                                              dium. The plates incubated 2&#226;&#8364;&#8220;4 days. <br />                                                                              The strains used wild type (KT1112), <br />                                                                              glc7-109 (TW25), pma1-s1 (TW62), pma1-s2 (TW65), <br />                                                                              pma1-s3 (TW69), pma1-s4 (TW72), pma1-s5 (TW76), <br />                                                                              pma1-s6 (TW80). <br />  1432                                      T. Williams-Hart, X. Wu K. Tatchell <br />  <br />                                                           TABLE 4 <br />                                               Phenotype glc7-109 suppressors <br />  <br />                              Phenotype                                                             cnb1::LEU2 <br />                            pma1 mutant                  glc7-109 mutant defects                mutant defects <br />         Suppressor         GLC7 background                     suppressed pma1a                suppressed pma1 <br />         pma1-s1                    cs                Css, Nas, Lis, Mns, paro.s, hygro.s          Nas, Lis <br />         pma1-s2                    ts                Css, Mns, paro.s, hygro.s                    Lis <br />         pma1-s3                    &#226;&#8364;&#8221;                 Css, Nas, Lis, paro.s, hygro.s                          &#226;&#8364;&#8221; <br />         pma1-s4                    cs                Css, Nas, Lis, Mns, OH&#226;&#171;&#186;s, paro.s, hygro.s    Nas, Lis <br />         pma1-s5                    &#226;&#8364;&#8221;                 Css, Nas, Mns, paro.s, hygro.s                          &#226;&#8364;&#8221; <br />                                                      Css, Nas, Mns, paro.s, hygro.s <br />         pma1-s6                   Lis                                    &#226;&#8364;&#8221; <br />           The growth pma1 mutants genetic backgrounds tested various media described in <br />         materials methods. cs, reduced growth rate 15&#226;&#172;&#353;; ts, reduced growth rate 37&#226;&#172;&#353;; Lis, unable grow <br />         0.1 m LiCl; Css, unable grow 0.1 m CsCl; Nas, unable grow 0.9 m NaCl media; Mns, unable to <br />         grow 0.01 m MnCl2 media; paro., unable grow 1 mg/ml paromomycin sulfate media; hygro., unable <br />         grow 50 &#226;?&#174;g/ml hygromycin B media; OH&#226;&#171;&#186;s, reduced growth alkaline pH. <br />           a <br />             The pma1-s4 glc7-109 mutant (TW75) glc7-109 suppressor tested suppression glc7- <br />         109 growth acidic alkaline pH. <br />  <br />  <br />  <br />  <br /> rectly influences activity, abundance, distribution                                  DISCUSSION <br /> of Pma1.                                                              Like glc7 mutations, glc7-109 pleiotropic. <br />    glc7-109 cells hyperpolarized: The potassium-            The glc7-109 mutant hyperaccumulates glycogen and <br /> remedial sensitivity glc7-109 mutants toxic cations,         sensitive wide range cations, high pH, and <br /> high pH, aminoglycoside antibiotics sup-               aminoglycoside antibiotics. The analysis single <br /> pression defects mutations PMA1                 missense mutations make glc7-109 indicated that <br /> consistent hypothesis plasma mem-                R260A substitution responsible traits <br /> brane potential glc7-109 mutant hyperpolarized.          conferred glc7-109. It likely hyperglycogen <br /> A conventional determination membrane potential                 trait ion drug hypersensitivity traits reflect <br /> using electrophysiological methods possible              influence Glc7 separate physiologi- <br /> yeast, relative membrane potentials assessed            cal processes. This glycogen phenotype <br /> using fluorescent dyes. Although methods suffer               glc7-109 dominant, traits are <br /> from complications caused contribution               recessive, glc7-R260P confers ion- <br /> mitochondria uptake process, Madrid et al.                  related defects hyperglycogen phenotype. <br /> (1998) shown cellular fluorescence                Yeast PP1 associated glycogen metabolism <br /> cyanine dye DiOC6(3) determined condi-                years; acronym derived glycogen <br /> tions mitochondria interfere. We              deficiency glc7-1 (Cannon et al. 1994). However, glc7- <br /> followed procedure closely growing strains             109 allele glc7 knowledge has <br /> low density glucose-containing medium assaying              linked ion homeostasis. <br /> DiOC6(3) fluorescence levels flow cytometry. As                    glc7-109 glycogen: In yeast, mammals, PP1 <br /> shown Table 5, DiOC6(3) fluorescence glc7-               thought dephosphorylate activate glycogen <br /> 109 strain (KT1596) &#226;&#181;&#8216;150&#226;&#8364;&#8220;200% wild-                synthase. PP1 activity glycogen synthase is <br /> type strain. In contrast, DiOC6(3) fluorescence               thought regulated targeting subunits tether <br /> pma1-s4 strain, TW72, 30&#226;&#8364;&#8220;40% wild type            PP1c glycogen synthase. Abundant evidence indicates <br /> and pma1-s4 glc7-109 strain (TW71)            Gac1 plays role yeast. gac1 null mutants <br /> 50&#226;&#8364;&#8220;60% wild type. As control, assayed DiOC6(3)             accumulate low levels glycogen glycogen synthase <br /> fluorescence wild-type strain grown synthetic              strains remains phosphorylated, inactive <br /> media containing low concentration K&#226;&#171;&#185;,               form (Franc&#194;&#184;ois et al. 1992). The product glc7-1, the <br /> known hyperpolarize plasma membrane (Madrid                 original glycogen-deficient allele GLC7, fails inter- <br /> et al. 1998). DiOC6(3) fluorescence wild-type cells             act Gac1 (Stuart et al. 1994). Gac1 contains sepa- <br /> under conditions 450&#226;&#8364;&#8220;650% wild-type               rate domains required binding Glc7 glycogen <br /> cells grown YPD medium (data shown). The re-                synthase, required functional activ- <br /> sults assays suggest defects          ity (Wu et al. 2001). Mutations glycogen synthase <br /> glc7-109 strains caused hyperpolarized             kinase, Pho85/Pcl10, suppress glycogen deficiency <br /> plasma membrane activation nonspecific              glc7-1 gac1 strains (Huang et al. 1996, 1998; Tim- <br /> ion conductance.                                                   blin et al. 1996). Furthermore, mutations major <br />                                                Glc7-Dependent Ion Transport                                           1433 <br />  <br />                                                                                         TABLE 5 <br />                                                                        Relative membrane potential measurements <br />  <br />                                                                              Relevant                      Fluorescence <br />                                                               Strain         genotype       400 mm KCl     DiOC6(3)a <br />                                                               KT1112     GLC7 PMA1                &#226;&#171;&#186;            100 <br />                                                                                                   &#226;&#171;&#185;          55 &#226;&#171;&#190; 5 <br />                                                               KT1596     glc7-109 PMA1            &#226;&#171;&#186;         177 &#226;&#171;&#190; 18 <br />                                                                                                   &#226;&#171;&#185;         100 &#226;&#171;&#190; 10 <br />                                                               TW72       GLC7 pma1-s4             &#226;&#171;&#186;          36 &#226;&#171;&#190; 14 <br />                                                               TW71       glc7-109 pma1-s4         &#226;&#171;&#186;          56 &#226;&#171;&#190; 14 <br />                                                                 a <br />                                                                   Fluorescence intensity DiOC6(3) glc7-109 (KT1596), <br />                                                               pma1-s4 (TW72), pma1-s4 glc7-109 (TW71) strains ex- <br />                                                               pressed percentage wild-type levels (see materials <br />                                                               methods). Values represent average separate <br />                                                               experiments performed separate days. <br />  <br />  <br />  <br />                                                               possibility degradative pathway altered <br />                                                               glc7-109 strains. It worth noting Pho85, the <br />                                                               kinase phosphorylates glycogen synthase, regu- <br />                                                               lates activity glycogen phosphorylase through <br />                                                               cyclins Pcl6 Pcl7 (Wang et al. 2001). Furthermore, <br />                                                               large-scale analysis protein complexes found <br />                                                               Gip2 Glc7 associate glycogen phosphory- <br />                                                               lase (Ho et al. 2002), suggesting Glc7, possibly <br />                                                               targeting subunits, regu- <br />                                                               late glycogen phosphorylase glycogen syn- <br />                                                               thase. A possibility Glc7-109 mutant <br />   Figure 8.&#226;&#8364;&#8221;The growth glc7-109 mutant pma1-   protein longer requires glycogen-targeting subunits <br /> s4 glc7-109 mutant YPD medium adjusted pH 3 8.5. <br /> Log-phase cultures strain incubated liquid <br />                                                               increase Vmax glycogen synthase, assuming <br /> YPD media pH 3 8.5 described materials       role glycogen-targeting subunits yeast. <br /> methods. The growth strain monitored optical   There direct biochemical evidence effect with <br /> density 660 nm. The strains used wild type   PP1-targeting subunit smooth muscle (Tanaka et <br /> (KT1112), glc7-109 (TW25), pma1-s4 glc7-109 (TW71).       al. 1998) genetic evidence role in <br />                                                               yeast. Small fragments Gac1 contain the <br /> isoform glycogen synthase, Gsy2, phos-      Glc7-binding domain partially complement glyco- <br /> phorylated suppress glycogen deficiencies         gen deficiency gac1 null mutant expressed at <br /> glc7-1 gac1 strains (Hardy Roach 1993; Ander-         high levels (Wu et al. 2001). Clearly, genetic <br /> son Tatchell 2001). Together, data suggest          biochemical studies necessary sort the <br /> that Gac1 acts molecular scaffold tether Glc7         complex roles Glc7 glycogen metabolism. <br /> and Gsy2. We surprised discover           Glc7 ion homeostasis: The pleiotropic salt- and <br /> the hyperglycogen trait glc7-109 epistatic gac1    drug-sensitive phenotype conferred glc7-109 be <br /> null mutation. Glycogen levels remain high strains         accounted broad mechanisms. One <br /> disrupted GAC1 GAC1-related genes, PIG1           possible explanation glc7-109 mutant a <br /> and GIP2.                                                     hyperpolarized plasma membrane potential. In Neuro- <br />    One explanation results additional       spora crassa, membrane voltage measurements <br /> glycogen-targeting subunits substitute loss       performed (Slayman 1965a,b), plasma mem- <br /> of Gac1, Pig1, Gip2. Pig2 candidate       brane H&#226;&#171;&#185;ATPase largely responsible maintaining <br /> redundant subunit. It similar sequence          membrane voltage (Slayman et al. 1973; Gradmann <br /> Gip2 loss-of-function mutations PIG2          et al. 1978). In S. cerevisiae, mutants gene encoding <br /> have alter glycogen levels, possi-    plasma membrane H&#226;&#171;&#185;ATPase, PMA1, in- <br /> ble Glc7-109 recruit Pig2 absence        ferred reduced membrane potential (Perlin <br /> other targeting subunits. Another possibility     et al. 1988; Seto-Young Perlin 1991), defects <br /> high levels glycogen glc7-109 strains partially     general opposite glc7-109 mutants, <br /> due changes activity enzymes          exhibiting increased resistance drugs toxic ca- <br /> glycogen synthase. We assayed activity levels        tions (McCusker et al. 1987). In contrast, yeast cells <br /> of glycogen synthase strains rule        defects plasma membrane K&#226;&#171;&#185; transporters <br />  1434                                      T. Williams-Hart, X. Wu K. Tatchell <br />  <br /> encoded TRK1 TRK2 inferred             Trk2. Surprisingly, current independent K&#226;&#171;&#185; <br /> an increased membrane potential (Madrid et al. 1998)            independent Trk1. A possible scenario <br /> and phenotype similar, identical,        arising studies K&#226;&#171;&#185; transporters <br /> to glc7-109 mutant (Vidal et al. 1990; Madrid       auxiliary function unrelated K&#226;&#171;&#185; trans- <br /> et al. 1998; Mulet et al. 1999). Direct measurements         port, strongly regulates membrane potential. Two <br /> membrane potential S. cerevisiae possible        related protein kinases, Hal4 Hal5, modulate <br /> technical reasons indirect determinations using fluo-       Trk1 Trk2 activity (Mulet et al. 1999). Yeast cells <br /> rescent dyes subject artifacts (Ballarin-Denti et al.    lacking Hal4 Hal5 phenotype similar that <br /> 1994). Nevertheless, analysis DiOC6(3) fluores-          trk1 trk2 strains, including hypersensitivity toxic <br /> cence consistent hyperpolarization plasma        cations hygromycin B deficiency K&#226;&#171;&#185; uptake. <br /> membrane glc7-109 cells. The isolation mutations          In contrast, increased expression Hal4 Hal5 results <br /> in PMA1 suppressors CsCl sensitivity glc7-         resistance Li&#226;&#171;&#185; Na&#226;&#171;&#185; (Mulet et al. 1999). These <br /> 109 mutants supports hypothesis.                   results suggest Hal4 Hal5 positive regulators <br />    How Glc7 act influence membrane poten-              Trk1 Trk2. However, Glc7 regulates K&#226;&#171;&#185; trans- <br /> tial? The obvious possibilities Pma1           port, probably does simply opposing the <br /> and potassium transporters, Trk1 Trk2. Pma1             kinase activity Hal4 Hal5 glc7-109 confers <br /> is essential maintenance plasma membrane             similar phenotype hal4 hal5 null mutant. <br /> potential activity regulated response           An alternative explanation hyperpolarization for <br /> range external factors. Pma1 phosphorylated            pleiotropic drug- ion-hypersensitive phenotype <br /> multiple sites classes protein kinases      glc7-109 mutant glc7-109 results the <br /> may regulate activity (Kolarov et al. 1988; Berto-          activation nonspecific uptake mechanism con- <br /> rello et al. 1991; Serrano et al. 1991). The protein            ductance. In scenario, reduction Pma1 activity <br /> kinase Ptk2 positive regulator Pma1 appears         application exogenous K&#226;&#171;&#185; suppress the <br /> to act activating phosphorylation site           drug ion toxicity glc7-109 mutation through <br /> Ser-899 (Goossens et al. 2000). In contrast, Yck1           indirect effects. In regard, note Bihler et <br /> and Yck2 kinases act negative regulators Pma1         al. (1998) identified nonspecific cation channel <br /> activity (Estrada et al. 1996). Thus, Pma1 likely          S. cerevisiae patch-clamping yeast spheroplasts. This <br /> candidate regulated Glc7. However,         conductance, termed nonselective cation channel 1 <br /> found direct involvement Glc7 Pma1 regulation.          (NSC1), carries large inward current low ca- <br /> Pma1 levels Pma1-dependent H&#226;&#171;&#185;ATPase activities              tion specificity. NSC1 independent Trk1, Trk2, and <br /> are similar glc7-109 GLC7 strains,         potassium channel Duk1/Tok1, gene or <br /> exclude possibility glc7-109 exerts subtle effect    genes encoding channel identified. <br /> on Pma1 activity Glc7 regulates Pma1 response        It possible NSC1 remains open glc7-109 mu- <br /> to specific environmental stresses.                             tants, causing increased toxicity cations. Of <br />    The potassium transporters Trk1 Trk2            course, exclude possibility Glc7 has <br /> candidates Glc7 regulation. Like glc7-109, trk1         multiple targets influence ion homeostasis. Many <br /> trk1 trk2 double mutants exhibit high levels DiOC6(3)        GLC7 mutants pleiotropic unreasonable <br /> fluorescence (Madrid et al. 1998) share traits           propose multiple substrates affected glc7- <br /> common glc7-109, including sensitivity toxic ca-        109 strains. Further genetic analysis allow to <br /> tions hygromycin B (Mulet et al. 1999). However,            identify targets. <br /> unlike mutants lacking potassium transporters,            We thank Carolyn Slayman Ken Allen providing antibody <br /> glc7-109 mutant exhibits obvious growth defect            Pma1 helpful advice concerning Pma1 activity assays. We <br /> media containing low concentrations potassium. glc7-         acknowledge Martha Cyert, James Haber, Peter Roach kindly <br />                                                                 providing strains plasmids. We thank Andrew Bloecher, Lucy <br /> 109 confers sensitivity alkaline pH opposed <br />                                                                 Robinson, Heather Panek, Clifford Slayman, Guglielmo M. Ven- <br /> to acidic pH sensitivity trk1 mutants. Surprisingly,     turi helpful discussions. We gratefully acknowledge Clifford Slay- <br /> K&#226;&#171;&#185; transport regulation membrane potential           man Lucy Robinson critically reading manuscript. This <br /> be independent properties Trk1 Trk2. Madrid              work funded National Institutes Health research grant GM- <br /> et al. (1998) trk1 trk2 mutants expressing      47789. <br /> a K&#226;&#171;&#185; transporter (HAK1) Schwanniomyces occiden- <br /> talis unrelated structure Trk1 Trk2 exhibit <br />                                                                                       LITERATURE CITED <br /> normal levels K&#226;&#171;&#185; transport retain hyperpolar- <br /> ization trk1 trk2 mutant. In contrast, expression        Allen, P. B., C. C. Ouimet P. Greengard, 1997 Spinophilin, <br />                                                                     novel protein phosphatase 1 binding protein localized den- <br /> of plant K&#226;&#171;&#185; transporter related Trk1 suppresses                dritic spines. Proc. Natl. Acad. Sci. USA 94: 9956&#226;&#8364;&#8220;9961. <br /> both K&#226;&#171;&#185; transport membrane potential defects            Anderson, C., K. Tatchell, 2001 Hyperactive glycogen syn- <br /> of trk1 trk2 mutants. Using patch clamping yeast                 thase mutants Saccharomyces cerevisiae suppress glc7-1 protein <br />                                                                     phosphatase mutant. J. Bacteriol. 183: 821&#226;&#8364;&#8220;829. <br /> spheroplasts, Bihler et al. (1999) observed in-         Andre, B., 1995 An overview membrane transport proteins in <br /> ward, H&#226;&#171;&#185;-dependent current dependent                   Saccharomyces cerevisiae. Yeast 11: 1575&#226;&#8364;&#8220;1611. <br />                                                            Glc7-Dependent Ion Transport                                                          1435 <br /> Andrews, P. D., M. J. Stark, 2000 Type 1 protein phosphatase                 yeast plasma membrane ATPase changes growth <br />     required maintenance cell wall integrity, morphogenesis            conditions gene dosage. FEBS Lett. 224: 193&#226;&#8364;&#8220;197. <br />     cell cycle progression Saccharomyces cerevisiae. J. Cell Sci.     Estrada, E., P. Agostinis, J. R. Vandenheede, J. Goris, W. Merlevede <br />     113: 507&#226;&#8364;&#8220;520.                                                                et al., 1996 Phosphorylation yeast plasma membrane H&#226;&#171;&#185;- <br /> Antebi, A., G. R. Fink, 1992 The yeast Ca(2&#226;&#171;&#185;)-ATPase homo-                   ATPase casein kinase I. J. Biol. Chem. 271: 32064&#226;&#8364;&#8220;32072. <br />     logue, PMR1, required normal Golgi function localizes         Farcasanu, I. C., D. Hirata, E. Tsuchiya, F. Nishiyama T. <br />     novel Golgi-like distribution. Mol. Biol. Cell 3: 633&#226;&#8364;&#8220;654.              Miyakawa, 1995 Protein phosphatase 2B Saccharomyces cere- <br /> Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman             visiae required tolerance manganese, blocking the <br />     et al., 1989 Current Protocols Molecular Biology. John Wiley &amp;            entry ions cells. Eur. J. Biochem. 232: 712&#226;&#8364;&#8220;717. <br />     Sons, New York.                                                          Feng, Z., S. E. Wilson, Z.-Y. Peng, K. K. Schlender, E. M. Reimann <br /> Bailis, J. M., G. S. Roeder, 2000 Pachytene exit controlled               et al., 1991 The yeast GLC7 gene required glycogen accumu- <br />     reversal Mek1-dependent phosphorylation. Cell 101: 211&#226;&#8364;&#8220;221.               lation encodes type 1 protein phosphatase. J. Biol. Chem. 266: <br /> Baker, S. H., D. L. Frederick, A. Bloecher K. Tatchell, 1997                 23796&#226;&#8364;&#8220;23801. <br />     Alanine scanning mutagenesis protein phosphatase type 1            Franc&#194;&#184;ois, J. M., S. Thompson-Jaeger, J. Skroch, U. Zellenka, W. <br />     yeast Saccharomyces cerevisiae. Genetics 145: 615&#226;&#8364;&#8220;626.                   Spevak et al., 1992 GAC1 encode regulatory subunit for <br /> Ballarin-Denti, A., C. L. Slayman H. Kuroda, 1994 Small                      protein phosphatase type 1 Saccharomyces cerevisiae. EMBO J. <br />     lipid-soluble cations membrane voltage probes Neuro-             11: 87&#226;&#8364;&#8220;96. <br />     spora Saccharomyces. Biochim. Biophys. Acta 1190: 43&#226;&#8364;&#8220;56.              Gaber, R. F., C. A. Styles G. R. Fink, 1988 TRK1 encodes a <br /> Bertorello, A. M., A. Aperia, S. I. Walaas, A. C. Nairn P.                   plasma membrane protein required high-affinity potassium <br />     Greengard, 1991 Phosphorylation catalytic subunit                  transport Saccharomyces cerevisiae. Mol. Cell. Biol. 8: 2848&#226;&#8364;&#8220;2859. <br />     Na&#226;&#171;&#185;,K(&#226;&#171;&#185;)-ATPase inhibits activity enzyme. Proc. Natl.         Garcia-Arranz, M., A. M. Maldonado, M. J. Mazon F. Portillo, <br />     Acad. Sci. USA 88: 11359&#226;&#8364;&#8220;11362.                                              1994 Transcriptional control yeast plasma membrane H(&#226;&#171;&#185;)- <br /> Bihler, H., C. L. Slayman A. Bertl, 1998 NSC1: novel high-                 ATPase glucose. Cloning characterization new gene <br />     current inward rectifier cations plasma membrane               involved regulation. J. Biol. Chem. 269: 18076&#226;&#8364;&#8220;18082. <br />     Saccharomyces cerevisiae. FEBS Lett. 432: 59&#226;&#8364;&#8220;64.                         Gietz, D., A. St. Jean, R. A. Woods R. H. Schiestl, 1992 Im- <br /> Bihler, H., R. F. Gaber, C. L. Slayman A. Bertl, 1999 The                    proved method high efficiency transformation intact yeast <br />     presumed potassium carrier Trk2p Saccharomyces cerevisiae de-             cells. Nucleic Acids Res. 20: 1425. <br />     termines H&#226;&#171;&#185;-dependent, K&#226;&#171;&#185;-independent current. FEBS                   Goldberg, J., H. B. Huang, Y. G. Kwon, P. Greengard, A. C. Nairn <br />     Lett. 447: 115&#226;&#8364;&#8220;120.                                                          et al., 1995 Three-dimensional structure catalytic subunit <br /> Bloecher, A., K. Tatchell, 1999 Defects Saccharomyces cere-               protein serine/threonine phosphatase-1. Nature 376: 745&#226;&#8364;&#8220;753. <br />     visiae protein phosphatase type I activate spindle/kinetochore       Gong, X., A. Chang, 2001 A mutant plasma membrane ATPase, <br />     checkpoint. Genes Dev. 13: 517&#226;&#8364;&#8220;522.                                          Pma1&#226;&#8364;&#8220;10, defective stability yeast cell surface. Proc. <br /> Bradford, M. M., 1976 Rapid sensitive method quantitation                Natl. Acad. Sci. USA 98: 9104&#226;&#8364;&#8220;9109. <br />     microgram quantities protein utilizing principle pro-       Goossens, A., N. La Fuente, J. Forment, R. Serrano F. <br />     tein-dye binding. Anal. Biochem. 72: 248&#226;&#8364;&#8220;254.                                Portillo, 2000 Regulation yeast H(&#226;&#171;&#185;)-ATPase protein <br /> Breuder, T., C. S. Hemenway, N. R. Movva, M. E. Cardenas                     kinases belonging family dedicated activation plasma <br />     J. Heitman, 1994 Calcineurin essential cyclosporin A-              membrane transporters. Mol. Cell. Biol. 20: 7654&#226;&#8364;&#8220;7661. <br />     FK506-sensitive yeast strains. Proc. Natl. Acad. Sci. USA 91: 5372&#226;&#8364;&#8220;      Gradmann, D., U. P. Hansen, W. S. Long, C. L. Slayman J. <br />     5376.                                                                        Warncke, 1978 Current-voltage relationships plasma <br /> Browning, M. D., R. Huganir P. Greengard, 1985 Protein                       membrane principal electrogenic pump Neurospora <br />     phosphorylation neuronal function. J. Neurochem. 45:                     crassa: I. Steady-state conditions. J. Membr. Biol. 39: 333&#226;&#8364;&#8220;367. <br />                                                                              Greengard, P., A. C. Nairn, J. A. Girault, C. C. Ouimet, G. L. <br />     11&#226;&#8364;&#8220;23. <br />                                                                                  Snyder et al., 1998 The DARPP-32/protein phosphatase-1 cas- <br /> Cannon, J. F., J. R. Pringle, A. Fiechter M. Khalil, 1994 Char- <br />                                                                                  cade: model signal integration. Brain Res. Brain Res. Rev. <br />     acterization glycogen-deficient glc mutants Saccharomyces <br />                                                                                  26: 274&#226;&#8364;&#8220;284. <br />     cerevisiae. Genetics 136: 485&#226;&#8364;&#8220;503. <br />                                                                              Guthrie, C., G. R. Fink, 1991 Guide Yeast Genetics Molecular <br /> Capieaux, E., M. L. Vignais, A. Sentenac A. Goffeau, 1989 <br />                                                                                  Biology. (Methods Enzymology, Vol. 194). Academic Press, San <br />     The yeast H&#226;&#171;&#185;-ATPase gene controlled promoter binding <br />                                                                                  Diego. <br />     factor TUF. J. Biol. Chem. 264: 7437&#226;&#8364;&#8220;7446. <br />                                                                              Hardy, T. A., P. J. Roach, 1993 Control yeast glycogen <br /> Chang, A., C. W. Slayman, 1991 Maturation yeast plasma <br />                                                                                  synthase-2 COOH-terminal phosphorylation. J. Biol. Chem. <br />     membrane [H&#226;&#171;&#185;]ATPase involves phosphorylation intra- <br />                                                                                  268: 23799&#226;&#8364;&#8220;23805. <br />     cellular transport. J. Cell Biol. 115: 289&#226;&#8364;&#8220;295.                          Hasper, A., P. Soteropoulos D. S. Perlin, 1999 Modification <br /> Cheng, C., D. Huang P. J. Roach, 1997 Yeast PIG genes: PIG1                  N-terminal polyserine cluster alters stability plasma <br />     encodes putative type 1 phosphatase subunit interacts            membrane H(&#226;&#171;&#185;)-ATPase Saccharomyces cerevisiae. Biochim. <br />     yeast glycogen synthase Gsy2p. Yeast 13: 1&#226;&#8364;&#8220;8.                            Biophys. Acta 1420: 214&#226;&#8364;&#8220;222. <br /> Chester, V. E., 1968 Heritable glycogen-storage deficiency yeast          He, B., M. Gross B. Roizman, 1998 The gamma134.5 protein <br />     induction ultra-violet light. J. Gen. Microbiol. 51: 49&#226;&#8364;&#8220;56.       herpes simplex virus 1 structural functional attri- <br /> Cunningham, K. W., G. R. Fink, 1994 Calcineurin-dependent                    butes protein phosphatase 1 regulatory subunit present <br />     growth control Saccharomyces cerevisiae mutants lacking PMC1,             high molecular weight complex enzyme infected <br />     homolog plasma membrane Ca2&#226;&#171;&#185; ATPases. J. Cell Biol. 124:                cells. J. Biol. Chem. 273: 20737&#226;&#8364;&#8220;20743. <br />     351&#226;&#8364;&#8220;363.                                                                 Hill, J. E., A. M. Myers, T. J. Koerner A. Tzagoloff, <br /> Cunningham, K. W., G. R. Fink, 1996 Calcineurin inhibits                     1986 Yeast/E. coli shuttle vectors multiple unique restric- <br />     VCX1-dependent H&#226;&#171;&#185;/Ca2&#226;&#171;&#185; exchange induces Ca2&#226;&#171;&#185;                             tion sites. Yeast 2: 163&#226;&#8364;&#8220;167. <br />     ATPases Saccharomyces cerevisiae. Mol. Cell. Biol. 16: 2226&#226;&#8364;&#8220;2237.     Hisamoto, N., D. L. Frederick, K. Sugimoto, K. Tatchell K. <br /> Cyert, M. S., J. Thorner, 1992 Regulatory subunit (CNB1 gene                 Matsumoto, 1995 The EGP1 gene positive regulator <br />     product) yeast Ca2&#226;&#171;&#185;/calmodulin-dependent phosphoprotein                   protein phosphatase type 1 growth control Saccharo- <br />     phosphatases required adaptation pheromone. Mol. Cell.             myces cerevisiae. Mol. Cell. Biol. 15: 3767&#226;&#8364;&#8220;3776. <br />     Biol. 12: 3460&#226;&#8364;&#8220;3469.                                                     Ho, Y., A. Gruhler, A. Heilburt, G. D. Bader, L. Moore et al., 2002 <br /> Dufour, J. P., A. Goffeau, 1978 Solubilization lysolecithin               Systematic identification protein complexes Saccharomyces <br />     purification plasma membrane ATPase yeast                  cerevisiae mass spectrometry. Nature 415: 180&#226;&#8364;&#8220;183. <br />     Schizosaccharomyces pombe. J. Biol. Chem. 253: 7026&#226;&#8364;&#8220;7032.                Hoffman, C. S., F. Winston, 1987 A ten-minute DNA prepara- <br /> Egloff, M. P., D. F. Johnson, G. Moorhead, P. T. Cohen, P. Cohen                 tion yeast efficiently releases autonomous plasmids trans- <br />     et al., 1997 Structural basis recognition regulatory              formation Escherichia coli. Gene 57: 267&#226;&#8364;&#8220;272. <br />     subunits catalytic subunit protein phosphatase 1. EMBO         Holcomb, C. L., W. J. Hansen, T. Etcheverry R. Schekman, <br />     J. 16: 1876&#226;&#8364;&#8220;1887.                                                            1988 Secretory vesicles externalize major plasma membrane <br /> Eraso, P., A. Cid R. Serrano, 1987 Tight control               ATPase yeast. J. Cell Biol. 106: 641&#226;&#8364;&#8220;648. <br />  1436                                                T. Williams-Hart, X. Wu K. Tatchell <br /> Huang, D., I. Farkas P. J. Roach, 1996 Pho85p, cyclin-depen-              Association transcription factor IIA TATA binding protein <br />     dent protein kinase, Snf1p protein kinase act antagonisti-          required transcriptional activation subset promoters <br />     cally control glycogen accumulation Saccharomyces cerevisiae.         cell cycle progression Saccharomyces cerevisiae. Mol. Cell. <br />     Mol. Cell. Biol. 16: 4357&#226;&#8364;&#8220;4365.                                             Biol. 18: 2559&#226;&#8364;&#8220;2570. <br /> Huang, D., J. Moffat, W. A. Wilson, L. Moore, C. Cheng et al.,              Perlin, D. S., C. L. Brown J. E. Haber, 1988 Membrane poten- <br />     1998 Cyclin partners determine Pho85 protein kinase substrate               tial defect hygromycin B-resistant pma1 mutants Saccharo- <br />     specificity vitro vivo: control glycogen biosynthesis          myces cerevisiae. J. Biol. Chem. 263: 18118&#226;&#8364;&#8220;18122. <br />     Pcl8 Pcl10. Mol. Cell. Biol. 18: 3289&#226;&#8364;&#8220;3299.                      Peters, C., P. D. Andrews, M. J. Stark, S. Cesaro-Tadic, A. Glatz et <br /> Kaiser, C., S. Michaelis A. Mitchell, 1994 Methods Yeast                 al., 1999 Control terminal step intracellular membrane <br />     Genetics: A Cold Spring Harbor Laboratory Course Manual. Cold               fusion protein phosphatase 1. Science 285: 1084&#226;&#8364;&#8220;1087. <br />     Spring Harbor Laboratory Press, Plainview, NY.                          Portillo, F., P. Eraso R. Serrano, 1991 Analysis regula- <br /> Kolarov, J., J. Kulpa, M. Baijot A. Goffeau, 1988 Character-                tory domain yeast plasma membrane H&#226;&#171;&#185;-ATPase directed <br />     ization protein serine kinase yeast plasma membrane.              mutagenesis intragenic suppression. FEBS Lett. 287: 71&#226;&#8364;&#8220;74. <br />     J. Biol. Chem. 263: 10613&#226;&#8364;&#8220;10619.                                        Posas, F., M. Camps J. Arino, 1995 The PPZ protein phospha- <br /> Kotliar, N., C. A. Stella, E. H. Ramos J. R. Mattoon, 1994                  tases important determinants salt tolerance yeast cells. <br />     L-leucine transport systems Saccharomyces cerevisiae participation       J. Biol. Chem. 270: 13036&#226;&#8364;&#8220;13041. <br />     GAP1, S1 S2 transport systems. Cell Mol. Biol. 40: 833&#226;&#8364;&#8220;842.      Ramaswamy, N. T., L. Li, M. Khalil J. F. Cannon, 1998 Regula- <br /> Kuno, T., H. Tanaka, H. Mukai, C. D. Chang, K. Hiraga et al.,                   tion yeast glycogen metabolism sporulation Glc7p pro- <br />     1991 cDNA cloning calcineurin B homolog Saccharomyces               tein phosphatase. Genetics 149: 57&#226;&#8364;&#8220;72. <br />     cerevisiae. Biochem. Biophys. Res. Commun. 180: 1159&#226;&#8364;&#8220;1163.              Rao, R., C. W. Slayman, 1993 Mutagenesis conserved resi- <br /> MacKelvie, S. H., P. D. Andrews M. J. R. Stark, 1995 The                    dues phosphorylation domain yeast plasma mem- <br />     Saccharomyces cerevisiae gene SDS22 encodes potential regulator           brane H(&#226;&#171;&#185;)-ATPase. Effects structure function. J. Biol. <br />     mitotic function yeast type 1 protein phosphatase. Mol.           Chem. 268: 6708&#226;&#8364;&#8220;6713. <br />     Cell. Biol. 15: 3777&#226;&#8364;&#8220;3785.                                              Rhodes, N., M. Company B. Errede, 1990 A yeast-Escherichia <br /> MacMillan, L. B., M. A. Bass, N. Cheng, E. F. Howard, M. Tamura                 coli shuttle vector containing M13 origin replication. Plas- <br />     et al., 1999 Brain actin-associated protein phosphatase 1 holoen-           mid 23: 159&#226;&#8364;&#8220;162. <br />     zymes containing spinophilin, neurabin, selected catalytic          Rose, M. D., P. Novick, J. H. Thomas, D. Botstein G. R. Fink, <br />     subunit isoforms. J. Biol. Chem. 274: 35845&#226;&#8364;&#8220;35854.                          1987 A Saccharomyces cerevisiae genomic plasmid bank based on <br /> Madrid, R., M. J. Gomez, J. Ramos A. Rodriguez-Navarro, 1998                centromere-containing shuttle vector. Gene 60: 237&#226;&#8364;&#8220;243. <br />     Ectopic potassium uptake trk1 trk2 mutants Saccharomyces          Rudolph, H. K., A. Antebi, G. R. Fink, C. M. Buckley, T. E. Dorman <br />     cerevisiae correlates highly hyperpolarized membrane po-             et al., 1989 The yeast secretory pathway perturbed muta- <br />     tential. J. Biol. Chem. 273: 14838&#226;&#8364;&#8220;14844.                                   tions PMR1, member Ca2&#226;&#171;&#185; ATPase family. Cell 58: <br /> Matheos, D. P., T. J. Kingsbury, U. S. Ahsan K. W. Cunning-                 133&#226;&#8364;&#8220;145. <br />     ham, 1997 Tcn1p/Crz1p, calcineurin-dependent transcrip-               Sassoon, I., F. F. Severin, P. D. Andrews, M.-R. Taba, K. B. Kaplan <br />     tion factor differentially regulates gene expression Saccha-        et al., 1999 Regulation Saccharomyces cerevisiae kinetochores <br />     romyces cerevisiae. Genes Dev. 11: 3445&#226;&#8364;&#8220;3458.                               type 1 phosphatase Glc7p. Genes Dev. 13: 545&#226;&#8364;&#8220;555. <br /> McCusker, J. H., J. E. Haber, 1988 Cycloheximide-resistant              Serrano, R., 1983a In vivo glucose activation yeast plasma <br />     temperature-sensitive lethal mutations Saccharomyces cerevisiae.         membrane ATPase. FEBS Lett. 156: 11&#226;&#8364;&#8220;14. <br />     Genetics 119: 303&#226;&#8364;&#8220;315.                                                  Serrano, R., 1983b Purification reconstitution proton- <br /> McCusker, J. H., D. S. Perlin J. E. Haber, 1987 Pleiotropic                 pumping ATPase fungal plant plasma membranes. Arch. <br />     plasma membrane ATPase mutations Saccharomyces cerevisiae.               Biochem. Biophys. 227: 1&#226;&#8364;&#8220;8. <br />     Mol. Cell. Biol. 7: 4082&#226;&#8364;&#8220;4088. <br />                                                                             Serrano, R., C. Montesinos, M. Roldan, G. Garrido, C. Ferguson <br /> Mendoza, I., F. Rubio, A. Rodriguez-Navarro J. M. Pardo, <br />                                                                                 et al., 1991 Domains yeast plasma membrane ATPase- <br />     1994 The protein phosphatase calcineurin essential NaCl <br />                                                                                 associated glycoprotein. Biochim. Biophys. Acta 1062: 157&#226;&#8364;&#8220;164. <br />     tolerance Saccharomyces cerevisiae. J. Biol. Chem. 269: 8792&#226;&#8364;&#8220; <br />                                                                             Serrano, R., J. M. Villalba, M. G. Palmgren, F. Portillo, A. Parets- <br />     8796. <br />                                                                                 Soler et al., 1992 Studies plasma membrane H(&#226;&#171;&#185;)- <br /> Miller, J. H., 1972 Experiments Molecular Genetics. Cold Spring <br />                                                                                 ATPase yeast plants. Biochem. Soc. Trans. 20: 562&#226;&#8364;&#8220;566. <br />     Harbor Laboratory Press, Cold Spring Harbor, NY. <br />                                                                             Seto-Young, D., D. S. Perlin, 1991 Effect membrane voltage <br /> Mulet, J. M., M. P. Leube, S. J. Kron, G. Rios, G. R. Fink et al., 1999 <br />                                                                                 plasma membrane H(&#226;&#171;&#185;)-ATPase Saccharomyces cerevisiae. <br />     A novel mechanism ion homeostasis salt tolerance in <br />                                                                                 J. Biol. Chem. 266: 1383&#226;&#8364;&#8220;1389. <br />     yeast: Hal4 Hal5 protein kinases modulate Trk1-Trk2 <br />     potassium transporter. Mol. Cell. Biol. 19: 3328&#226;&#8364;&#8220;3337.                  Shenolikar, S., 1994 Protein serine/threonine phosphatases&#226;&#8364;&#8221;new <br /> Mulkey, R. M., S. Endo, S. Shenolikar R. C. Malenka, 1994                   avenues cell regulation. Annu. Rev. Cell Biol. 10: 55&#226;&#8364;&#8220;86. <br />     Involvement calcineurin/inhibitor-1 phosphatase cascade         Shenolikar, S., A. C. Nairn, 1991 Protein phosphatases: recent <br />     hippocampal long-term depression. Nature 369: 486&#226;&#8364;&#8220;488.                      progress. Adv. Second Messenger Phosphoprotein Res. 23: 1&#226;&#8364;&#8220;121. <br /> Nakamoto, R. K., R. Rao C. W. Slayman, 1991 Expression               Sikorski, R. S., P. Hieter, 1989 A shuttle vectors and <br />     yeast plasma membrane [H&#226;&#171;&#185;]ATPase secretory vesicles.                 yeast host strains designed efficient manipulation DNA in <br />     A new strategy directed mutagenesis. J. Biol. Chem. 266:                Saccharomyces cerevisiae. Genetics 122: 19&#226;&#8364;&#8220;27. <br />     7940&#226;&#8364;&#8220;7949.                                                              Slayman, C. L., 1965a Electrical properties Neurospora crassa. Ef- <br /> Nakamura, T., Y. Liu, D. Hirata, H. Namba, S. Harada et al., 1993               fects external cations intracellular potential. J. Gen. <br />     Protein phosphatase type 2B (calcineurin)-mediated, FK506-sen-              Physiol. 49: 69&#226;&#8364;&#8220;92. <br />     sitive regulation intracellular ions yeast important        Slayman, C. L., 1965b Electrical properties Neurospora crassa. <br />     determinant adaptation high salt stress conditions. EMBO             Respiration intracellular potential. J. Gen. Physiol. 49: <br />     J. 12: 4063&#226;&#8364;&#8220;4071.                                                           93&#226;&#8364;&#8220;116. <br /> Nass, R., K. W. Cunningham R. Rao, 1997 Intracellular seques-           Slayman, C. L., W. S. Long C. Y. Lu, 1973 The relationship <br />     tration sodium novel Na&#226;&#171;&#185;/H&#226;&#171;&#185; exchanger yeast                   ATP electrogenic pump plasma membrane <br />     enhanced mutations plasma membrane H&#226;&#171;&#185;-ATPase.                     Neurospora crassa. J. Membr. Biol. 14: 305&#226;&#8364;&#8220;338. <br />     Insights mechanisms sodium tolerance. J. Biol. Chem.            Smith, F. D., G. S. Oxford S. L. Milgram, 1999 Association <br />     272: 26145&#226;&#8364;&#8220;26152.                                                           D2 dopamine receptor cytoplasmic loop with <br /> Norbeck, J., A. Blomberg, 1998 Amino acid uptake strongly                spinophilin, protein phosphatase-1-interacting protein. J. Biol. <br />     affected exponential growth Saccharomyces cerevisiae           Chem. 274: 19894&#226;&#8364;&#8220;19900. <br />     0.7 M NaCl medium. FEMS Microbiol. Lett. 158: 121&#226;&#8364;&#8220;126.                  Stathopoulos, A. M., M. S. Cyert, 1997 Calcineurin acts <br /> Novoa, I., H. Zeng, H. P. Harding D. Ron, 2001 Feedback                     CRZ1/TCN1-encoded transcription factor regulate <br />     inhibition unfolded protein response GADD34-medi-                 gene expression yeast. Genes Dev. 11: 3432&#226;&#8364;&#8220;3444. <br />     ated dephosphorylation eIF2&#226;?&#163;. J. Cell Biol. 153: 1011&#226;&#8364;&#8220;1021.          Stathopoulos-Gerontides, A., J. J. Guo M. S. Cyert, 1999 <br /> Ozer, J., L. E. Lezina, J. Ewing, S. Audi P. M. Lieberman, 1998             Yeast calcineurin regulates nuclear localization Crz1p tran- <br />                                                            Glc7-Dependent Ion Transport                                                       1437 <br />     scription factor dephosphorylation. Genes Dev. 13: 798&#226;&#8364;&#8220;          Venturi, G. M., A. Bloecher, T. Williams-Hart K. Tatchell, <br />     803.                                                                         2000 Genetic interactions GLC7, PPZ1 PPZ2 in <br /> Stuart, J. S., D. L. Frederick, C. M. Varner K. Tatchell, 1994               Saccharomyces cerevisiae. Geneti <br /> </body></html>