<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 1998 Genetics Society America <br />  <br />  <br />  <br />  Sum1, Highly Conserved WD-Repeat Protein, Suppresses S-M Checkpoint <br />  Mutants Inhibits Osmotic Stress Cell Cycle Response Fission Yeast <br />  <br />                                               Timothy Humphrey Tamar Enoch <br />                                   Department Genetics, Harvard Medical School, Boston, Massachusetts 02115 <br />                                                       Manuscript received June 23, 1997 <br />                                                   Accepted publication December 9, 1997 <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The S-M checkpoint ensures entry mitosis dependent completion DNA replication. <br />               In fission yeast Schizosaccharomyces pombe, S-M checkpoint mutant cdc2-3w thought defective <br />               receiving checkpoint signal. <span id='am-9' about='protege:TO' typeof='owl:Thing'>To</span> isolate genes function checkpoint pathway, screened <br />               S. pombe cDNA library genes that, overexpressed, suppress checkpoint defect of <br />               cdc2-3w. Using approach, identified novel gene, sum11 (suppressor uncontrolled mitosis). <br />               sum11 encodes highly conserved WD-transducin repeat protein striking sequence similarity the <br />               human transforming growth factor (TGF)-b-receptor interacting protein TRIP-1 translation <br />               initiation factor 3 subunit eIF3-p39, encoded TIF34 gene Saccharomyces cerevisiae. S. pombe sum11 <br />               essential gene, required normal cell growth division. In addition restoring checkpoint <br />               control, overexpression sum11 inhibits normal cell cycle response osmotic stress. Furthermore, <br />               demonstrate inactivation stress-activated MAP kinase pathway, required cell cycle stress <br />               response, restores S-M checkpoint cdc2-3w cells. These results suggest Sum1 interacts the <br />               stress-activated MAP kinase pathway raise possibility environmental conditions influence <br />               checkpoint response fission yeast. <br />  <br />  <br />  <br /> C    HECKPOINTS maintain order cell cycle <br />        events blocking late events early events <br /> have completed (Hartwell Weinert <br />                                                                           spite importance Cdc2 tyrosine phosphorylation <br />                                                                           fission yeast cell cycle control, tyrosine phosphoryla- <br />                                                                           tion CDKs budding yeast essential is <br /> 1989). The S-M checkpoint ensures initiation                      required cell cycle control certain cir- <br /> mitosis (M) dependent completion DNA synthe-                     cumstances (Lew Kornbluth 1996). <br /> sis (S). In fission yeast, inhibitory tyrosine phosphory-                   In addition cell cycle regulators, genes <br /> lation Cdc2, catalytic subunit cyclin-depen-                    required S-M checkpoint identified <br /> dent kinase (CDK), required S-M checkpoint                     fission yeast; rad11, rad3 1, rad9 1, rad17 1, <br /> (Enoch et al. 1991; Enoch Nurse 1990). Mutations                      rad261 hus11 (Humphrey Enoch 1995). An- <br /> that completely eliminate Cdc2 tyrosine phosphoryla-                      gene, cds11 involved (Murakami and <br /> tion, cdc2F15, result loss S-M checkpoint                   Okayama 1995). Related genes eu- <br /> control lethal premature mitosis known &#226;&#8364;&#339;mitotic                    karyotes including Saccharomyces cerevisiae humans <br /> catastrophe&#226;&#8364;? (Enoch et al. 1991). Other cdc2 mutations                    (Lieberman et al. 1996; Stewart Enoch 1996). In <br /> disrupt checkpoint control altering mitotic reg-                    particular, rad3 1 gene functionally structur- <br /> ulation subtle fashion. One mutant,                        ally similar MEC1 gene S. cerevisiae the <br /> cdc2-3w, viable divides reduced size,                 human ATM gene. Mutations ATM responsible <br /> indicating timing mitosis advanced. Unlike                 severe cancer prone syndrome, ataxia-telangiec- <br /> wild-type cells, DNA replication blocked                     tasia (Savitsky et al. 1995a). Thus appears to <br /> hydroxyurea (HU) mutations DNA replication                          evolutionarily conserved checkpoint pathway in <br /> enzymes, cdc2-3w cells proceed mitosis, attempting                   eukaryotes plays important role preventing <br /> to segregate single set chromosomes (Enoch et al.                    cancer human cells (Savitsky et al. 1995a,b). rad3 1 <br /> 1991; Enoch Nurse 1990). Mutations alter                     ATM related gene encoding the <br /> activity Cdc25, major Cdc2 tyrosine phosphatase,                   catalytic subunit DNA dependent protein kinase <br /> and/or Cdc2 tyrosine kinases Mik1 Wee1                   (DDK), activated specifically breaks dou- <br /> disrupt S-M checkpoint (Enoch Nurse 1990;                         ble-stranded DNA (Gottlieb Jackson 1993; Hart- <br /> Lundgren et al. 1991; Sheldrick Carr 1993). De-                       ley et al. 1995). By analogy likely Rad3 is <br />                                                                           activated DNA structures formed S-phase is <br />                                                                           inhibited, activation probably requires products <br />                                                                           checkpoint genes. However, known <br />   Corresponding author: Timothy Humphrey, Radiation Genome <br /> Stability <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Medical Research Council, Harwell, Didcot, Oxford-        Rad3 activated proteins link Rad3 activa- <br /> shire, OX11 ORD, UK. E-mail: t.humphrey@har.mrc.ac.uk                     tion tyrosine phosphorylation Cdc2. <br />  <br /> Genetics 148: 1731&#226;&#8364;&#8220;1742 (April, 1998) <br />  1732                                                T. Humphrey T. Enoch <br />  <br />   To identify novel components checkpoint path-             replica plated EMM1A plates thiamine 24 hr <br /> way, screened genes suppress check-           induce nmt11 promoter (Maundrell 1990). Colonies <br />                                                                    replica plated EMM1A plates Phloxine B (Fisher <br /> point defect cdc2-3w mutants. We reasoned be-              Scientific, Pittsburgh) 5 mm HU grown <br /> cause mutant partially defective mitotic           2&#226;&#8364;&#8220;3 days 298. Colonies grew presence <br /> control, overexpression genes encoding positive regu-           absence HU examined microscopically. Colonies <br /> lators upstream components S-M checkpoint                consisting cells normal size absence of <br /> pathway suppress checkpoint defect cdc2-3w.           HU elongated presence HU analyzed fur- <br />                                                                    ther. Eighteen transformants 40,000 screened <br /> Using approach, identified high copy sup-           formed colonies presence absence HU. Plasmids <br /> pressor cdc2-3w checkpoint defect,            recovered transformants 15 were <br /> named sum11 (suppressor uncontrolled mitosis). sum11            able retransform original TE361 strain. One trans- <br /> is predicted encode highly conserved WD-trans-                formant displayed consistently convincing <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. The <br /> ducin repeat protein high sequence similarity              corresponding gene, sum1 1, subject analysis. A <br />                                                                    blunt ended SalI BamHI DNA fragment encoding sum11 <br /> the human TGF-b-receptor interacting protein TRIP-1                cDNA S. pombe expression vector subcloned into <br /> (Chen et al. 1995), translation initiation fac-         EcoRV BamHI (blunt-ended) site Bluescript <br /> tor 3 subunit eIF3-p39, encoded TIF34 gene               SK 1 plasmid (Stratagene, La Jolla, CA) generate plasmid <br /> Saccharomyces cerevisiae (Naranda et al. 1997). sum11           6.3SK. Complete DNA sequence analysis insert revealed <br /> an essential gene, required             39 end sum11 cDNA fused DNA fragment <br />                                                                    encoding 50S ribosomal subunit. This 39 region deleted <br /> normal cell growth division.                                   original sum11 expression plasmid, subse- <br />   In addition restoring cdc2-3w checkpoint, over-           quently retested, suppress HU sensitivity of <br /> expression sum11 disrupts normal cellular re-               TE361. <br /> sponse osmotic stress. In wild-type fission yeast cells,           Nucleotide sequence accession number: The cDNA sequence <br /> exposure osmotic stress stimulates entry mitosis.          sum11 deposited GenBank accession no. <br />                                                                    Y09529. <br /> This response controlled kinase cascade involving             Construction sum1::ura41 mutant: A sum1::ura4 1 dis- <br /> a MAP kinase kinase (MAPKK) encoded wis11,                  ruptant mutant generated inserting 1.7-kb EcoRI frag- <br /> a MAP kinase (MAPK) encoded sty11/spc11 (Millar                 ment encoding ura4 1 gene EcoRI site sum11 <br /> et al. 1995; Shiozaki Russell 1995). Activation             plasmid 6.3SK. An NdeI AgeI DNA fragment isolated <br /> this pathway negatively regulated phosphotyrosine            plasmid (6.3SKura4) used disrupt copy <br />                                                                    sum11 gene, one-step disruption method, in <br /> phosphatases encoded pyp11 pyp2 1, de-                h1/h 1 ura4-D18/ura4-D18 stable diploid (TE480 gift G. <br /> phosphorylate Sty1/Spc1(Millar et al. 1995; Shiozaki               Cottarel, Mitotix Inc., Boston). Two stable ura1 transformants <br /> and Russell 1995). Like overexpression sum11, muta-             selected, NdeI Age I DNA fragment con- <br /> tions block activation Sty1/Spc1 MAP kinase            firmed homologously integrated sum11 <br /> abolish normal cell cycle stress response. Cells har-          locus transformants Southern blotting. One of <br />                                                                    strains (TE557) crossed h2/h2 ura4-D18/ura4- <br /> boring mutations undergo cell cycle arrest re- <br />                                                                    D18 stable diploid (TE558) generate sporulating diploids <br /> sponse osmotic stress. Interestingly, mutations            heterozygous sum1::ura4 1 disruption (TE554). These <br /> abolish cell cycle stress response suppress           diploids sporulated minimal media plates, 20 <br /> checkpoint defect cdc2-3w. These results suggest           tetrads dissected. A 2 1:2 2 ratio viability observed in <br /> Sum1 interacts stress-response pathway                dissected tetrads, viable colonies failed grow in <br />                                                                    absence uracil, indicating disruption sum11 <br /> raise intriguing possibility that, stress- <br />                                                                    gene lethal. The sum1::ura4 1 disruption recessive <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> response checkpoint response pathways op-                 wild-type phenotype observed heterozygous <br /> posing effects mitotic initiation, S-M checkpoint           diploid strain. Furthermore, overexpression sum1::ura4 1 <br /> may function efficiently stressful conditions.          pREP3X promoter (Basi et al. 1993) suppress <br />                                                                    checkpoint defect TE361 does cause obvi- <br />                                                                    ous phenotypes. <br />                MATERIALS AND METHODS                                  Sum1 depletion study: Strain TE578 (sum1::ura4 1 ura4-D18 <br />                                                                    leu1-32 ade6-M210 h1 pREP41X-sum1 1 LEU2) grown in <br />    Growth S. pombe : Standard media growth conditions       EMM1A media absence thiamine midexponential <br /> were used described Moreno et al. (1991), cells          phase. This culture diluted optical density (OD) <br /> were transformed electroporation described Prentice       595 0.02, split halves, thiamine (2 mm final <br /> (1992). Phase contrast, Nomarski fluorescence micro-           concentration) added half culture repress <br /> graphs obtained using Zeiss Axiophot microscope,        sum11 transcription weak nmt11 REP41X promoter <br /> a Photonic Microscope Image Processor C1966 (Hamamatsu             (Basi et al. 1993). The OD 595 cell number (cells/ml) <br /> Photonic Sys. Corp., Bridgewater, NJ). All strains listed   cultures recorded 24 hr addition of <br /> Table 1.                                                           thiamine, strains photographed 36 hr. <br />    Screening high copy plasmid suppressors cdc2-3w :           Cloning expression sum11 homologues fission yeast: <br /> TE361 (cdc2-3w ade6-M210 leu1-32 h2) transformed          The TRIP-1 gene PCR amplified human cardiac <br /> an S. pombe cDNA library cDNA expression regu-         cDNA library (a generous gift J. Seidman Lab, Har- <br /> lated thiamine repressible nmt1 1 promoter (B. Edgar        vard Medical School, Boston) using primers: 59CCGGGAT <br /> and C. Norbury, unpublished results). Leu1 transformants           GAAGCCGATCCT 39and 59GTTGGTCAGGCTGGTCTTAA 39 <br /> were selected Edinburgh minimal medium (EMM) plates             1088nt PCR product subcloned pCRII <br /> with adenine 2 mm thiamine (EMM1A1T), colonies             vector (Invitrogen, San Diego). An SpeI (blunt-ended) Not I <br />                                                          Fission Yeast sum11 Gene                                                 1733 <br />  <br />                                                                   TABLE 1 <br />                                                     S. pombe strains used study <br />  <br />           Strain no.                                                    Genotype <br />           TE361            cdc2-3w ade6-M210 leu1-32 h2 <br />           TE480            DIPLOID ade6-M216/ade6-M216 leu1-32/leu1-32 ura4-D18/ura4-D18 h1/h1 (GP109) <br />           TE554            DIPLOID sum11/sum1::ura4 1 ade6-M210/ade6-M216 leu1-32/leu1-32 ura4-D18/ura4-D18 h1/h2 <br />           TE557            DIPLOID sum11/sum1::ura4 1 ade6-M216/ade6-M216 leu1-32/leu1-32 ura4-D18/ura4-D18 h1/h1 <br />           TE558            DIPLOID ade6-M210/ade6-M210 leu1-32/leu1-32 ura4-D18/ura4-D18 h2/h2 <br />           TE577            sum1::ura4 ura4-D18 leu1-32 ade6-M210 h1 pREP3X-sum11(LEU2) <br />           TE578            sum1::ura4 ura4-D18 leu1-32 ade6-M210 h1 pREP41X-sum11(LEU2) <br />           TE596            cdc2-3w ade6-M210 leu1-32 h2 pREP3X(LEU2) <br />           TE742            cdc2-3w spc1-M13 ura4-D18 leu1-32 h2 <br />           TE805            cdc2-3w ade6-M210 leu1-32 h2 pREP3X-sum11(LEU2) <br />           TE806            cdc2-3w ade6-M210 leu1-32 h2 pREP1-pyp11(LEU2) <br />           TE807            leu1-32 h2 pREP81X(LEU2) <br />           TE808            leu1-32 h2 pREP3X-sum11(LEU2) <br />           TE809            leu1-32 h2 pREP1-pyp11(LEU2) <br />           TE810            spc1HA 6H(ura 1) leu1-32 ura4-D18 h2 pREP3X(LEU2) <br />           TE811            spc1HA 6H(ura 1) leu1-32 ura4-D18 h2 pREP3X-sum11(LEU2) <br />           TE812            spc1HA 6H(ura 1) leu1-32 ura4-D18 h2 pREP1-pyp11(LEU2) <br />           TE813            spc1-M13, ura4-D18, leu1-32 h2 <br />           TE814            pyp1::ura4 1 leu1-32 ura4-D18 h1 pREP3X-sum11(LEU2) <br />           TE815            pyp2::ura4 1 leu1-32 ura4-D18 h2 pREP3X-sum11(LEU2) <br />             All strains Enoch strain collection TE480 (GP109), generous gift G. Cottarel <br />           (Mitotix Inc., Boston). Strains TE742 TE813 derived KS1147. Strains TE810, TE811, TE812 <br />           derived KS1376. KS1147 KS1376 described Shiozaki Russell (1995, 1996) and <br />           generous gifts P. Russell. <br />  <br />  <br /> DNA fragment (1154nt) subcloned Xho I (blunt-              b-mercaptoethanol, 10 mg/ml aprotinin, 10 mg/ml leupeptin, <br /> ended) Not I sites S. pombe expression plasmids REP3X        10 mg/ml pepstatin A, 100 mm TPCK, 2 mm PMSF, 60 mm b- <br /> and REP41X, form plasmids REP3X-TRIP-1 REP41X-                   glycerophosphate, 15 mm paranitrophenol phosphate, 1 mm <br /> TRIP-1, respectively. The S. cerevisiae TIF34 gene PCR              okadaic acid. The Sty1/Spc1 protein isolated affinity <br /> amplified S. cerevisiae total DNA using primers:               purification using Ni2 1 charged His-Bind beads (Novagen, <br /> 59GCAGAAGAGTTACTGATGAAC39 59CCCCTCGTG                               Madison, Wisconsin), according manufacturer&#226;&#8364;&#8482;s instruc- <br /> GTTAAAGTGAC 39, yield 1356nt PCR product,                    tions. The proteins resolved 12% sodium dodecyl <br /> was subcloned pCRII vector. An Xho I BamHI DNA              sulphate (SDS) PAGE transferred electrophoretically a <br /> fragment (1434nt) subcloned XhoI BamHI                 nitrocellulose membrane. The membrane immunoblotted <br /> sites S. pombe expression plasmids REP3X REP41X              anti-HA (12CA5) anti-pTyr (4G10; Upstate Biotech- <br /> to form plasmids REP3X-TIF34 REP41X-TIF34, respec-                  nology, Inc., Lake Placid, NY ) antibodies, detected <br /> tively. To test ability strains functionally comple-    enhanced chemiluminescence (ECL; Amersham, Arlington <br /> ment sum1::ura4 1 disruption, strain TE554 (sum11/                  Heights, IL) according manufacturer&#226;&#8364;&#8482;s instructions. <br /> sum1::ura4 1 ade6-M210/ade6-M216 leu1-32/leu1-32 ura4-D18/                 RNA analysis: Strains TE554 (sum1::ura41 ura4-D18 leu1-32 <br /> ura4-D18 h1/h2) transformed plasmids, Leu1           ade6-M210 h1 pREP41X-sum11LEU2), TE807 (leu1-32 h2 <br /> transformants selected sporulated using standard con-          pREP81X), TE808 (leu1-32 h2 pREP3X-sum11LEU2) TE809 <br /> ditions. Leu1 spores germinated EMM1A plates               (leu1-32 h2 pREP1-pyp11LEU2) grown 200-ml cultures <br /> uracil (EMM1A1U), colonies replica plated EMM1A                  midlog phase EMM1A. Strain TE554 grown the <br /> plates. Only colonies transformed plasmids com-           presence 2 mm thiamine 18 hr repress sum11 transcrip- <br /> plement sum1::ura4 1 disruption able grow              tion REP41X promoter. KCl added final con- <br /> in absence uracil.                                               centration 0.9 m 90 min half culture before <br />    Phosphotyrosine levels Sty1/Spc1 protein: Strain KS1376           harvesting. The pelleted cells lysed glass beads in <br /> (a generous gift P. Russell, Scripps Research Institute,           1 ml solution containing 0.32 m sucrose, 20 mm Tris-HCL <br /> La Jolla, California) Sty1/Spc1 tagged           (pH 7.5) 10 mm EDTA 0.5 mg/ml heparin, diluted <br /> two copies HA epitope histidine residues             10 ml solution containing 1% SDS. Phenol <br /> transformed pREP3X, pREP3X-sum11 pREP1-                  extraction performed 608 3 min followed further <br /> pyp1 1 (a generous gift P. Russell) generate strains            phenol/chloroform extraction 228, total RNA etha- <br /> TE810, TE811 TE812, respectively (see Table 1). These               nol precipitated. For Northern hybridizations, 10 mg total <br /> strains grown midlog phase EMM media 298                  RNA separated denaturing formaldehyde gel ac- <br /> following derepression nmt11 promoter 24 hr. KCl             cording method Rave et al. (1979). Following transfer <br /> was added final concentration 0.9 m 20 min           nitrocellulose (Genescreen, Life Science Products, Boston), <br /> harvesting. Pelleted cells lysed glass beads (Sigma           bound RNA hybridized gpd11 act11 probes. <br /> Chemical Co., St. Louis) lysis buffer containing 6 m urea,         The gpd11 probe generated 1253 bp HindIII to <br /> 0.5% NP-40, 0.5% deoxycholate, 50 mm Tris-HCl (pH 7.5),                 XbaI DNA fragment plasmid PCR-gpd11. The act11 <br /> 50 mm NaF, 10% glycerol, 2 mm Na-orthovanadate, 10 mm                   probe generated 1126 bp EcoRV HindIII DNA <br />  1734                                               T. Humphrey T. Enoch <br />  <br /> fragment plasmid PCR-act1 1. Primers labeled <br /> as described Feinberg Vogelstein (1984). <br />    Genetic analysis role sum11 stress response: To <br /> determine lethality sum1::ura4 1 disruption <br /> was suppressed mutations sty1 1/spc1 1 gene, strain <br /> TE577, sum1::ura4 1 mutation complemented <br /> by plasmid-borne, thiamine-repressible REP3X-sum11 <br /> gene, crossed strain TE813 (derived KS1147, a <br /> generous gift P. Russell), carrying mutation the <br /> Spc1/Sty1 MAP kinase, spc1-M13. The products cross <br /> were examined free spore analysis using standard genetic <br /> techniques. No ura1 colonies isolated cross on <br /> plates containing thiamine, indicating lethality of <br /> sum1::ura4 1 suppressed spc1-M13 mutation. To <br /> determine overexpression sum11 suppress <br /> the lethality pyp1 pyp2 double mutant, pyp1 (TE814) and <br /> pyp2 (TE815) strains sum11 overexpressed, were <br /> crossed, progeny examined tetrad analysis. No <br /> viable double mutants obtained experiment, <br /> indicating overexpression sum1 1 suppress the <br /> lethality pyp1 pyp2 double mutant. <br />  <br />  <br />                           RESULTS <br />  <br />    Identification genes suppress cdc2-3w check- <br /> point defect: To identify genes positively regulate <br /> the S-M checkpoint fission yeast, screened for <br /> genes restore S-M checkpoint over- <br /> expressed cdc2-3w cells. Such genes encode <br /> upstream components checkpoint pathway or <br /> positive regulators checkpoint response. As the <br /> cdc2-3w mutant defective S-M checkpoint, expo-            Figure 1.&#226;&#8364;&#8221;OPsum11 suppresses HU sensitivity cdc2-3w. <br /> sure cdc2-3w HU causes cells enter mitosis inap-         (A) TE361 transformed pREP3X plasmid (vector) or <br /> propriately, incompletely replicated chromosomes             pREP3X-sum11 (OPsum1) grown 298, EMM1A in <br /> (Enoch et al. 1992; Enoch Nurse 1990). In               absence (2HU) presence (1HU) 5 mm HU. Wild- <br />                                                                   type cells transformed vector (TE807) shown <br /> cells septum bisects single nucleus resulting in <br />                                                                   (wild-type 1 vector) positive control. (B) Overexpression <br /> aneuploid anucleate cells; resembles mor-             sum11 delays cell cycle cdc2-3w HU. The above <br /> phology &#226;&#8364;&#339;cut&#226;&#8364;? mutants (Hirano et al. 1986). As               strains grown EMM1A liquid media absence <br /> result, cdc2-3w cells fail form colonies presence       (2HU) presence (1HU) 5 mm HU 6 hr 298. Cells <br /> of HU (Enoch Nurse 1990). In contrast, wild-type              fixed DAPI stained visualize nuclear material <br />                                                                   described materials methods. White arrows indi- <br /> cells initially undergo cell cycle arrest presence <br />                                                                   cate cells showing &#226;&#8364;&#339;cut&#226;&#8364;? phenotype. Bar, 10 mm. <br /> of HU, resume cell cycle longer <br /> S-phase, forming slow-growing colonies consisting of <br /> highly elongated cells. Overexpression genes encod-            thiamine. Leu1 transformants selected could <br /> ing positive regulators checkpoint function allow          form colonies minimal media plates presence <br /> growth cdc2-3w HU elongated cells. These             5 mm hydroxyurea following derepression nmt11 <br /> be distinguished genes counteract effects           promoter (see materials methods). Colonies <br /> of HU, catalytic subunit ribonucleotide            examined microscopically, transformants were <br /> reductase, cells overexpressing genes divide        selected elongated presence HU were <br /> normal <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> HU (T. Enoch, unpublished results).              significantly elongated absence. Using this <br /> Suppressors distinguished general neg-           approach, 40,000 transformants screened, six <br /> ative regulators cell cycle, genes block      transformants appeared delay cell cycle <br /> cell division presence absence HU.             HU-dependent manner. The strongest these <br />    To identify genes encoding positive regulators check-       suppressors encoded novel gene, <br /> point function, strain TE361 (cdc2-3w, ade6-M210,             sum11, subject analysis. <br /> leu1-32 h2) transformed S. pombe cDNA li-                The phenotype cdc2-3w cells overexpressing sum11 <br /> brary control thiamine-repressible               shown Figure 1. cdc2-3w transformed vector <br /> nmt11 promoter (Maundrell 1990) vector encod-                fails grow plates presence HU. In <br /> ing LEU2 gene (B. Edgar C. Norbury, unpub-                contrast, cdc2-3w transformed nmt11 (pREP3X) <br /> lished results). Expression cDNA conditional            plasmid expressing sum11 (OPsum11) able form colo- <br /> can activated replica plating cells media        nies presence HU. These cells fail grow the <br />                                                      Fission Yeast sum11 Gene                                                  1735 <br />  <br />  <br />  <br />  <br />   Figure 2.&#226;&#8364;&#8221;sum11 encodes highly conserved WD-transducin repeat protein. (A) Alignment amino acid sequence <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> the S. pombe (S.p) Sum1 ORF TRIP-1 H. sapiens (H.s) TIF34 S. cerevisiae (S.c). Boxes indicate identities <br /> among proteins. The WD/transducin repeats underlined. The sequence alignment performed using DNASTAR <br /> &#226;&#8364;&#339;Megalign&#226;&#8364;? program. The nucleotide sequence sum11 cDNA GenBank nucleotide sequence data <span id='am-244' about='xsp:base' typeof='owl:Thing'>base</span> <br /> under accession number Y09529. (B) Alignment WD-transducin repeats Sum1 WD-transducin repeat consensus <br /> sequence, defined Neer et al. (1994). The WD transducin domains (i&#226;&#8364;&#8220;v) correspond underlined (A). <br />  <br />  <br /> presence HU nmt11 promoter transcrip-                resembling wild-type cells HU (Figure 1B, upper right <br /> tionally repressed (data shown). Cytological exami-             panel). Thus, overexpression sum11 restores check- <br /> nation reveals overexpression sum11 cdc2-3w              point function cdc2-3w mutants. Overexpression of <br /> cells absence HU induces modest G2 delay                sum11 restores checkpoint function wee1.50 mik1 <br /> compared cdc2-3w transformed vector                   permissive temperature restores strains over- <br /> (compare Figure 1B, middle lower left panels; FACS              expressing cdc25 1 (data shown). However, checkpoint <br /> data shown). In presence HU, cdc2-3w cells               function restored checkpoint rad mutants, hus1 <br /> transformed vector form &#226;&#8364;&#339;cut&#226;&#8364;? phenotype,               rad3 (data shown). <br /> as described previously (Figure 1B, middle right panel).               sum11 encodes highly conserved WD-repeat protein: <br /> However, cdc2-3w cells sum11 overexpressed              The sum11 cDNA sequenced predicted <br /> became elongated HU (Figure 1B, lower right panel),              encode protein 328 amino acids (Figure 2A). The <br />  1736                                           T. Humphrey T. Enoch <br />  <br /> cDNA sequence sum11 deposited Gen-           promoter (Basi et al. 1993; materials meth- <br /> Bank accession number Y09529. (This sequence            ods). This strain form colonies presence <br /> has deposited S. Yoshioka, K. Kato           thiamine. To examine effects Sum1 depletion <br /> H. Okayama complete sequencing              cell growth division, exponentially growing <br /> Schizosaccharomyces cDNA project GenBank acces-         culture diluted, thiamine added half <br /> sion no. D89187.) A search protein data-          culture transcriptionally repress comple- <br /> bases reveals Sum1 related proteins wide      menting pREP41X-sum11 gene. It took approximately <br /> range eukaryotes. In particular, genes predicted        18 hr deplete sum11 function conditions. <br /> encode proteins striking degree amino acid          Cell mass (OD 595) cell number (cells/ml) these <br /> identity sum11 identified genomes           cultures recorded 24 hr following addition of <br /> humans S. cerevisiae (Figure 2A). The human homo-         thiamine (Figure 3B). An exponential increase both <br /> logue, TRIP-1, shares 49% amino acid identity 65%         cell mass cell number observed 24-hr period <br /> similarity Sum1 identified protein       absence thiamine (promoter on). In contrast, <br /> is phosphorylated transforming growth factor           exponential increase cell mass cell number <br /> (TGF)-b type II receptor (Chen et al. 1995). The S. cere-     longer observed following transcriptional repres- <br /> visiae homologue, encoded TIF34, predicted           sion pREP41X-sum11 thiamine. Instead, cell growth <br /> be 53% identical 72% similar Sum1. TIF34           greatly retarded culture (OD-promoter ), <br /> recently shown encode 39 kD subunit            cell division observed 22 hr follow- <br /> the eukaryotic translation initiation factor 3 (Naranda       ing addition thiamine (cells/ml promoter ). In <br /> et al. 1997). All genes encode members           addition, transcriptional repression pREP41X- <br /> WD-repeat family proteins (Neer et al. 1994)           sum11 gene resulted irreversible loss viability <br /> each possess putative core WD-repeat domains             (data shown). These results indicate sum11 is <br /> (underlined Figure 2A). These WD-transducin do-            required normal cell growth cell division in <br /> mains number regulatory pro-          fission yeast. The cell cycle distribution cells in <br /> teins thought involved protein-protein       sum11 transcriptionally repressed indistin- <br /> interactions. A comparison Sum1 WD-repeat         guishable wild-type cells indicating that <br /> domains WD-repeat consensus sequence              Sum1 depletion does arrest cell cycle (FACS <br /> shown Figure 2B. The region sequence homology           data shown). Furthermore, &#226;&#8364;&#339;cut&#226;&#8364;? phenotype was <br /> is extended proteins re-          observed strain following transcriptional repres- <br /> stricted WD-repeat regions.                            sion sum11 presence HU (data shown). <br />    Sum11 essential gene: To examine sum1 null          Microscopic analysis cells sum11 tran- <br /> phenotype fission yeast, construct generated      scriptionally repressed (Figure 3C, promoter ) re- <br /> which sum11 disrupted copy ura4 1           veals morphologically distinct cells <br /> gene (see Figure 3A; materials methods). A DNA            expressing sum11 (Figure 3C, promoter on). Sum1 de- <br /> fragment encoding disrupted sum11 gene trans-         pleted cells swollen, pear-shaped, slightly larger <br /> formed stable h1/h1ura4-D18/ura4-D18 diploid           wild-type cells (Nomarski used). Moreover, DAPI <br /> (TE480). Ura1 transformants selected, ho-            staining cells, weakly stains cytoplasm <br /> mologous integration confirmed Southern blot-          addition nuclear material, reveals dark non- <br /> ting. Transformants crossed h2/h2 ura4-D18/        staining regions cells appearing time. <br /> ura4-D18 diploid (TE558) generate sporulating dip-         These regions vacuolar represent regions <br /> loids heterozygous sum1::ura4 1 disruption. The       cytoplasm dissociated from <br /> heterozygous diploids wild-type phenotype, indi-        cell wall. <br /> cating sum1::ura4 1 disruption loss func-      Because lethality sum1::ura4 1 strain be <br /> tion mutation. Tetrad analysis independent ura1        rescued plasmid expression sum11 gene, we <br /> diploid transformants revealed 2 1:22 segregation       examined human TRIP-1 yeast TIF34 <br /> viability 20 asci dissected yeast extract       genes functionally complement sum1::ura4 1 <br /> plates. The viable colonies ura2,        strain. These genes subcloned S. pombe expres- <br /> indicating disruption sum11 gene lethal.       sion vectors transformed diploid strain het- <br /> Upon microscopic examination, nonviable segre-            erozygous sum1::ura4 1 disruption mutation (see <br /> gants indistinguishable ungermi-        materials methods). Sporulation diploid <br /> nated spores, suggesting sum11 required ei-       transformants revealed that, contrast cells express- <br /> ther germination, cell growth, both.                       ing sum11 gene, human TRIP-1 yeast TIF34 genes <br />    To determine sum11 necessary normal         unable functionally complement sum1::ura4 1 <br /> cell growth division, strain constructed            strain. In addition, fission yeast sum11 gene and <br /> (TE578) lethality sum1::ura4 1 disrup-    human TRIP-1 gene unable functionally <br /> tion rescued plasmid expression sum11           complement lethality TIF34::HIS3 S. cerevisiae <br /> gene control weakened nmt11 (REP41X)           strain (data shown), consistent findings re- <br />                                                 Fission Yeast sum11 Gene                                                1737 <br />  <br />  <br />  <br />  <br />                                                                Figure 3.&#226;&#8364;&#8221;Construction analysis sum1 disrup- <br />                                                             tion mutant. (A) Diagram DNA construct used make <br />                                                             sum1::ura4 1 disruption mutant. The sum11 ORF indi- <br />                                                             cated grey. The ura4 1 gene inserted EcoRI site <br />                                                             sum11 orientation shown described materials <br />                                                             methods. (B) Effects Sum1 depletion cell growth <br />                                                             division. The optical density (OD 595; h, j) cell <br />                                                             number (cells/ml) (s, d) exponential culture TE578 <br />                                                             (sum1::ura4 1, ura4-D18, leu1-32 ade6-M210 h1 pREP41X-sum11 <br />                                                             LEU2) EMM1A recorded absence (promoter <br />                                                             on; h, s) presence (promoter off; j, d) 2 mm thiamine <br />                                                             added 0 hr. (C) Phenotype Sum1 depleted cells. Strain <br />                                                             TE578 grown EMM1A absence (promoter on) <br />                                                             presence (promoter off) 2 mm thiamine described <br />                                                             above. Cells fixed, DAPI-stained photographed using <br />                                                             Nomarski fluoresence (DAPI) microscopy, indicated. <br />                                                             Bar, 10 mm. <br />  <br />  <br />  <br /> ported (Naranda et al. 1997). Overexpression       Sty1/Spc1 stress-activated MAP kinase (Millar et al. <br /> of TRIP-1 TIF34 genes unable sup-        1995; Shiozaki Russell 1995). Overexpression of <br /> press checkpoint defective phenotype cdc2-3w          pyp11 pyp2 1 inactivates MAP kinase path- <br /> (data shown). Thus, despite high degree se-         way, causing cell cycle delay G2/M normal <br /> quence similarity, functional complementation         growth conditions, cell cycle arrest response to <br /> observed species.                                    stress (Millar et al. 1995; Shiozaki Russell 1995). <br />    Overexpression sum11 inhibits normal cell cycle    As overexpression sum11 causes G2/M delay, <br /> response osmotic stress: During course       resembles pyp1 genes respect. Furthermore, <br /> experiments noticed phenotypes caused         simultaneous inactivation Pyp1 Pyp2 PTPases, <br /> sum1 mutations similar phenotypes caused           like disruption sum11, gives rise inviable spores that <br /> mutations altering expression S. pombe phospho-       germinate (Millar et al. 1992), inhibition of <br /> tyrosine phosphatases (PTPases) Pyp1 Pyp2. The           pyp11 expression pyp2 strain results similar <br /> Pyp1 Pyp2 PTPases, encoded pyp11 pyp21        phenotype Sum1-depleted cells; cells exhibit <br /> genes, function negatively regulate highly con-       greatly retarded growth large swollen (Mil- <br /> served stress-activated MAP kinase encoded sty11/     lar et al. 1995). Thus, loss Sum1 Pyp1 Pyp2 <br /> spc11 gene fission yeast (Millar et al. 1995; Shiozaki    leads severe growth defects, overexpression <br /> and Russell 1995). This kinase activated response      genes results extended G2 phase. <br /> to environmental stress including heat shock, oxidative          These similarities prompted analyze effects <br /> stress, osmotic stress (Degols et al. 1996; Millar       overexpression sum11 response cells to <br /> et al. 1995; Shiozaki Russell 1995). Activation       osmotic stress. Although wild-type cells osmotic stress <br /> the stress-activated MAP kinase pathway stimulates entry     stimulates entry mitosis, cells overexpressing pyp11, <br /> into mitosis (Hannig et al. 1994; Millar et al. 1992).       lacking functional Sty1/Spc1 MAP kinase become <br /> The Pyp1 Pyp2 PTPases negatively regulate           highly elongated presence 1 m KCl 1.5 m <br /> pathway tyrosine dephosphorylation            sorbitol (Shiozaki Russell 1995). As shown in <br />  1738                                           T. Humphrey T. Enoch <br />  <br />  <br />  <br />  <br />                                                                                                 Figure 4.&#226;&#8364;&#8221;Overexpres- <br />                                                                                              sion sum11 disrupts the <br />                                                                                              cell cycle response osmotic <br />                                                                                              stress. Strains TE807 (wild- <br />                                                                                              type 1vector) TE808 <br />                                                                                              (wild-type 1OPsum11) were <br />                                                                                              grown EMM1T plates, re- <br />                                                                                              streaked EMM plates <br />                                                                                              24 hr derepress the <br />                                                                                              nmt1 1 promoter, rep- <br />                                                                                              lica plated EMM plates <br />                                                                                              containing 1 m KCl or <br />                                                                                              1.5 m sorbitol. These plates <br />                                                                                              incubated 24 hr at <br />                                                                                              298 cells photographed. <br />                                                                                              Bar, 10 mm. <br />  <br />  <br />  <br />  <br /> Figure 4, wild-type cells overexpressing sum11        (Figure 5A, 1KCl), phosphotyrosine levels Sty1/ <br /> presence 1 m KCl 1.5 m sorbitol display remark-      Spc1 cells determined Western blot analy- <br /> ably similar phenotype. Wild-type cells transformed     sis (Figure 5A). As described previously (Degols et al. <br /> vector (Figure 4, 1vector) divide smaller size    1996; Millar et al. 1995; Shiozaki Russell 1995), <br /> in response osmotic stress (Figure 4, b c). In        increased HA-tagged Sty1/Spc1 phosphotyrosine levels <br /> contrast, cells overexpressing sum11 (Figure 4, OPsum1)      observed wild-type cells, transformed vec- <br /> become highly elongated (3&#226;&#8364;&#8220;43 length controls)        tor alone, exposed osmotic stress (compare Figure <br /> in presence 1 m KCl 1.5 m sorbitol (Figure 4,      5A, lanes 1 4). In contrast, considerable decrease <br /> compare e f b c, respectively). These results   HA-tagged Sty1/Spc1 phosphotyrosine levels ob- <br /> demonstrate overexpression sum11, like inactiva-     served cells overexpressing pyp11 condi- <br /> tion MAP kinase pathway, interferes          tions (Figure 5A, compare lanes 1 3). In cells overex- <br /> normal cell cycle response osmotic stress.                pressing sum11, KCl, phosphotyrosine level the <br />    Overexpression sum11 does inhibit stress-acti-     HA-tagged Sty1/Spc1 resembled cells trans- <br /> vation Sty1/Spc1 MAP kinase: The similarities be-     formed vector (compare Figure 5A, lanes 1 <br /> tween Sum1 Pyp1 suggest proteins          2). Thus, overexpression sum11 does inhibit <br /> perform similar function. Pyp1 Pyp2 negatively         tyrosine phosphorylation Sty1/Spc1 MAP kinase, <br /> regulate stress-response pathway dephosphoryla-       unlike overexpression pyp11. <br /> tion phosphotyrosine residue required Sty1/            To investigate possible role sum11 in <br /> Spc1 MAP kinase activity (Millar et al. 1995; Shiozaki       regulating MAP kinase pathway, examined the <br /> and Russell 1995). Stimulation stress-activated       transcriptional response cells stress cells overex- <br /> MAP kinase pathway results tyrosine phosphorylation       pressing lacking sum11. In wild-type cells, activation <br /> of Sty1/Spc1 MAP kinase (Degols et al. 1996; Mil-        Sty1/Spc1 MAP kinase leads transcriptional <br /> lar et al. 1995; Shiozaki Russell 1995). When            induction number genes, including gpd11 <br /> pyp11 overexpressed stressed cells, greatly reduced    gene, encoding glycerol-3-phosphate dehydrogenase <br /> levels tyrosine phosphorylated Sty1/Spc1 observed     (Degols et al. 1996; Shiozaki Russell 1996; Wil- <br /> (Millar et al. 1995; Shiozaki Russell 1995). To          kinson et al. 1996). Northern blot analysis thus <br /> determine overexpression sum11 regulates          performed examine gpd11 mRNA levels cells over- <br /> the stress-activated MAP kinase pathway similar         expressing sum11 pyp11, cells lacking sum11 <br /> mechanism, Sty1/Spc1 phosphotyrosine levels         function (see materials methods). Figure 5B <br /> examined cells overexpressing sum11      demonstrates gpd11 mRNA levels greatly stimu- <br /> exposure osmotic stress. A strain expressing HA        lated addition KCl wild-type cells (Figure 5B, <br /> epitope-tagged Sty1/Spc1 MAP kinase transformed          compare lanes 2 6). In contrast, transcriptional <br /> with control plasmid (Figure 5A, vector) plas-     activation gpd11 inhibited cells overexpressing <br /> mids sum11 (Figure 5A, OPsum11)            pyp11 (Figure 5B, compare lanes 4 8). However, in <br /> pyp11 (Figure 5A, OPpyp11) genes highly expressed.      cells overexpressing sum11 (Figure 5B, lanes 3 7), <br /> Cells subjected osmotic stress indicated       cells lacking sum11 (Figure 5B, lanes 1 5), <br />                                                       Fission Yeast sum11 Gene                                              1739 <br />  <br />                                                                    transcriptional activation gpd11 gene ob- <br />                                                                    served addition KCl. Moreover, loss Sum1 <br />                                                                    does lead constitutive activation MAP ki- <br />                                                                    nase pathway gpd11 gene expressed the <br />                                                                    absence sum11 (Figure 5B, lane 1). These results <br />                                                                    suggest stress-responsive MAP kinase activated <br />                                                                    normally sum11 overexpressed. <br />                                                                       We performed genetic experiments determine <br />                                                                    Sum1 negatively regulates stress-responsive <br />                                                                    MAP kinase pathway. Mutations sty11/spc11 gene, <br />                                                                    encoding stress-activated MAP kinase suppress <br />                                                                    lethality pyp1 pyp2 double mutation (Millar et <br />                                                                    al. 1995; Shiozaki Russell 1995). These results <br />                                                                    suggest lethality pyp1 pyp2 double mutation results <br />                                                                    constitutive activation stress-responsive MAP <br />                                                                    kinase. However, lethality sum1::ura4 1 disrup- <br />                                                                    tion suppressed mutations sty11/spc11 <br />                                                                    gene (data shown; materials methods). <br />                                                                    Thus, lethality sum1::ura4 1 disruption be <br />                                                                    solely constitutive activation stress-responsive <br />                                                                    MAP kinase. In addition, overexpression sum11 does <br />                                                                    rescue lethality pyp1 pyp2 double mutants <br />                                                                    (data shown; materials methods), demon- <br />                                                                    strating sum11 block lethal conse- <br />                                                                    quences constitutive activation stress-activated <br />                                                                    MAP kinase pathway. Thus overexpression of <br />                                                                    sum11 blocks cell cycle stress response, genetic and <br />                                                                    biochemical studies suggest does by <br />    Figure 5.&#226;&#8364;&#8221;Overexpression sum11 does inhibit Sty1/        interfering activation Sty1/Spc1 MAP kinase. <br /> Spc1 activation gpd11 transcription response osmotic <br /> stress. (A) Phosphotyrosine levels Sty1/Spc1 protein        Overexpression sum11 inhibit cell cycle <br /> not altered strains overexpressing sum1 1. Strain KS1376        stress response downstream MAP kinase activation. <br /> (Shiozaki Russell 1995), Sty1/Spc1p          Alternatively, sum11 negatively regulating a <br /> tagged copies HA epitope histidine         MAP kinase independent pathway activates <br /> residues, transformed REP3X (vector),              cell cycle stress response. To determine whether <br /> pREP3X-sum11 (OPsum1) pREP1-pyp11 (OPpyp1). These <br /> strains (TE810, TE811, TE812, respectively) grown         overexpression sum11 inhibits cell cycle stress <br /> to midlog phase EMM medium 298 following derepres-           response independently Spc1 MAP kinase, exam- <br /> sion nmt1 1 promoter 24 hr. KCl added        ined additive effect cell <br /> medium final concentration 0.9 m, 20 min        cycle arrest 1 m KCl following overexpression sum11 <br /> harvesting, indicated (1). The Sty1/Spc1 protein iso-       spc1-M13 strain. spc1-M13 (TE813) transformed <br /> lated affinity purification probed Western blot for <br /> the presence phoshotyrosine (anti-YP) HA epitope tag         vector REP3X-sum11, cell length <br /> (anti-HA). (B) Northern blot analysis gpd11 transcription       determined following exposure 1 m KCl. spc1-M13 <br /> levels strains overexpressing sum1 1 pyp1 1. Strain TE807:   cells sum11 overexpressed to <br /> leu1-32 h2 pREP81X (shown wt 1vector), TE808: leu1-32           marginally longer 1 m KCl spc1-M13 trans- <br /> h2 pREP3X-sum11 LEU2 (shown wt 1OPsum1), TE809:             formed vector (data shown). Such result <br /> leu1-32 h2 pREP1-pyp11 LEU2 (shown wt 1OPpyp1) were <br /> grown mid-log phase EMM1A. Strain TE578: <br />                                                                    indicate overexpression sum11 inhibits the <br /> sum1::ura4 1, ura4-D18, leu1-32 ade6-M210 h1 pREP41X-sum1 1        cell cycle stress response independently Spc1 MAP <br /> LEU2 (shown sum1), grown EMM1A presence           kinase (Figure 7, pathway b). However, interpretation <br /> of 2 mm thiamine 18 hr repress sum1 1 transcription         experiment problematic effect so <br /> from REP41X promoter. KCl added final concen-         small. It clear expect ad- <br /> tration 0.9 m 90 min indicated (1). Total RNA <br /> generated strains separated Northern       ditivity clearly experiment single mu- <br /> gel, blotted nitrocellulose transcripts visualized     tations completely block cell cycle progression in <br /> gpd11 act11 probes indicated.                               1 m KCl. <br />                                                                       Inactivating stress-response pathway suppresses <br />                                                                    checkpoint defect cdc2-3w: We shown that <br />                                                                    sum11 overexpression disrupts normal cell cycle re- <br />                                                                    sponse osmotic stress. A corollary observation <br />                                                                    inhibiting cell cycle stress response could <br />  1740                                           T. Humphrey T. Enoch <br />  <br />  <br />  <br />  <br />                                                                                                  Figure 6.&#226;&#8364;&#8221;HU sensitivity <br />                                                                                               cdc2-3w suppressed by <br />                                                                                               mutants inactivate the <br />                                                                                               stress-activated MAP kinase <br />                                                                                               pathway. TE596 (cdc2-3w 1 <br />                                                                                               vector) TE806(cdc2-3w1 <br />                                                                                               pREP1-pyp11) grown at <br />                                                                                               298 3 days EMM1A <br />                                                                                               plates (in absence of <br />                                                                                               thiamine) restreaked <br />                                                                                               EMM1A plates the <br />                                                                                               absence (2HU) pres- <br />                                                                                               ence (1HU) 5 mm HU. <br />                                                                                               Strains TE361 (cdc2-3w) and <br />                                                                                               TE742 (cdc2-3w spc1-M13) <br />                                                                                               grown EMM1U1 <br />                                                                                               A1L plates 298 4 <br />                                                                                               days absence (2HU) <br />                                                                                               presence (1HU) 5 <br />                                                                                               mm HU. <br />  <br />  <br />  <br />  <br /> restore checkpoint control cdc2-3w sum11           Thus, know Sum1 directly involved <br /> was initially isolated screen genes phe-   checkpoint control normal circumstances. <br /> notype. To investigate possibility, examined         Overexpression sum11 inhibits cell cycle response <br /> effects inactivating stress-response pathway        stress: In fission yeast, entry mitosis advanced <br /> checkpoint control cdc2-3w mutants. As shown Fig-       cells subjected stress, changes in <br /> ure 6 (upper panels), overexpression pyp11 enables         <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> conditions osmotic pressure (Shiozaki and <br /> cdc2-3w cells form colonies presence HU,         Russell 1995). Such response advance cells <br /> indicating overexpression pyp11 restores          G1 undergo conjugation sporu- <br /> checkpoint response cdc2-3w cells. A loss function      lation adverse conditions. Stress-induced <br /> mutation Sty1/Spc1 MAP kinase, spc1-M13,          mitotic initiation requires activation MAP kinase <br /> restores checkpoint response, double mutant       cascade culminating activation MAP kinase <br /> strain (cdc2-3w spc1-M13) form colonies pres-      Spc1/Sty1 (Millar et al. 1995; Shiozaki Russell <br /> ence HU (Figure 6, lower panels).                          1995). Activation pathway negatively regulated <br />                                                               phosphotyrosine phosphatases, Pyp1 Pyp2, <br />                                                               dephosphorylate tyrosine residue Spc1/ <br />                       DISCUSSION                              Sty1 required activity (Millar et al. 1995; <br />                                                               Shiozaki Russell 1995). We discovered strik- <br />    Overexpression sum11 restores S-M checkpoint        ing similarities cells overexpressing Sum1 and <br /> in fission yeast: To identify positive regulators      mutants activate MAP kinase pathway <br /> S-M checkpoint, screened high-copy plasmid        inhibitory tyrosine phosphatases are <br /> suppressors checkpoint mutant cdc2-3w. cdc2-3w         overexpressed sty11/spc11 mutated. In each <br /> mutants overexpressing sum11 modest increase           strains, exposure osmotic stress results in <br /> in length G2 able arrest normally        mitotic arrest (Figure 4). In addition, muta- <br /> the presence HU. Thus, presence excess Sum1         tions restore checkpoint control cdc2-3w mutants (Fig- <br /> rescues checkpoint response mutants. The         ure 6). Thus, consequences overexpressing sum11 <br /> restoration checkpoint response likely       similar consequences inhibiting stress- <br /> be trivial consequence extending G2, muta-      activated MAP kinase pathway. Furthermore, pheno- <br /> tions extend G2 cdc2-3w restore check-         type mutants lacking Sum1 resembles mutants in <br /> point control (Enoch et al. 1992). Although overexpres-       MAP kinase pathway constitutively active <br /> sion sum11 restores checkpoint control, able    (Figure 3C). Such strains similar defects spore <br /> to determine disrupting Sum1 activity reduces         germination, proliferation, morphology. However, <br /> the checkpoint response loss Sum1 lethal.       know similarities circum- <br />                                                      Fission Yeast sum11 Gene                                            1741 <br />  <br />                                                                   et al. 1996). Our studies sum11 overexpression <br />                                                                   causes converse phenotype; transcriptional re- <br />                                                                   sponse stress apparently intact (Figure 5B), the <br />                                                                   cell cycle response blocked (Figure 4). These results <br />                                                                   suggest overproduction Sum1 interferes the <br />                                                                   stress response point downstream MAP kinase <br />                                                                   activation, branch pathway leads to <br />                                                                   activation mitosis. <br />                                                                      How does Sum1 inhibit cell cycle stress response? <br />                                                                   We considering explanations inhibition <br />                                                                   stress response Sum1 overexpression. First, <br />                                                                   sum11 overexpression directly interfere phos- <br />                                                                   phorylation MAP kinase target. For example, analy- <br />                                                                   sis homologous S. cerevisiae protein TIF34 has <br />                                                                   established associated essential multi- <br />                                                                   protein complex, eIF3, required translational initia- <br />                                                                   tion (Naranda et al. 1997). This complex a <br />                                                                   MAP kinase target, overexpression subunit <br />                                                                   interfere ability MAP kinase to <br />                                                                   interact target (Figure 7, pathway a). If phos- <br />                                                                   phorylation MAP kinase normally inhibits the <br />                                                                   function eIF-3, explain constitutive <br />                                                                   activation MAP kinase lethal are <br />                                                                   similarities phenotypes cells and <br />                                                                   sum12 cells. In regard, interesting hu- <br />                                                                   man gene related sum11, TRIP-1 associates and <br />                                                                   phosphorylated TGF-b type II receptor (Chen <br />                                                                   et al. 1995). Perhaps humans yeasts, Sum1- <br />                                                                   related proteins function transduce information from <br />                                                                   environment translation machinery. <br />                                                                      Another possibility overexpression sum11 <br />                                                                   inactivation MAP kinase pathway inhibit par- <br />                                                                   allel pathways interact common target (Fig- <br />    Figure 7.&#226;&#8364;&#8221;Model OPsum11 disrupts cell cycle stress <br /> response fission yeast. Activation Sty1/Spc1 MAP <br />                                                                   ure 7, pathway b). For example, possible MAP <br /> kinase environmental stress results transcriptional re-   kinase inactivation indirectly causes cell cycle arrest by <br /> sponse activation Atf1, advanced timing         inhibiting translation. Overexpression Sum1 also <br /> mitosis unkwnown mechanism. Overproduction Sum1          inhibit translation cause similar cell cycle <br /> may disrupt cell cycle stress-response inhibiting down-    response. Distinguishing models will <br /> stream cell cycle branch MAP kinase pathway directly <br /> (pathway a) indirectly (pathway b). See text details.      require analysis Sum1 function stress re- <br />                                                                   sponse. It possible effects Sum1 on <br />                                                                   stress response pathway indirect. For example, <br /> stantial represent similar molecular de-          overexpression proteins interfere with <br /> fects cell cycle targets constitutively acti-       translation, result interference the <br /> vated MAP kinase identified.                        cell cycle stress response. However, think un- <br />    Overexpression sum11 inhibits cell cycle stress         likely, screened 100,000 transformants <br /> response transcriptional stress response: We          gene phenotype <br /> have investigated mechanism overexpres-              sum11 (K. Forbes T. Enoch, unpublished re- <br /> sion sum11 interferes cell cycle response stress.      sults). <br /> As shown Figure 7, stress response pathway bifur-             The stress response checkpoint control: Abolish- <br /> cates activation Sty1/Spc1 MAP kinase. One           ing cell cycle stress response, mutating <br /> branch, requiring transcription factor Atf1, leads         sty11/spc11 overexpressing pyp11, restores checkpoint <br /> transcriptional activation genes required stress       control cdc2-3w. Because abolishing cell cycle <br /> response (Shiozaki Russell 1996; Wilkinson et                 stress response increases effectiveness S-M <br /> al. 1996). A second branch leads stimulation cell           checkpoint, activation stress-response pathway <br /> division. In atf1 mutants, transcriptional response           opposite effect; S-M checkpoint may <br /> to stress abolished, cell cycle stress response        effective conditions stress compared <br /> is affected (Shiozaki Russell 1996; Wilkinson             normal growth conditions. We think that <br />  1742                                                    T. Humphrey T. Enoch <br />  <br /> suppression checkpoint defects mutation                     Hannig, G., S. Ottilie R. L. Erikson, 1994 Negative regulation <br />                                                                              mitosis catalytically inactive pyp1 pyp2 mutants. Proc. <br /> stress response pathway trivial consequence de-                      Natl. Acad. Sci. USA 91: 10084&#226;&#8364;&#8220;10088. <br /> laying mitosis demonstrated previously                   Hartley, K. O., D. Gell, G. C. M. Smith, H. Zhang, N. Divecha <br /> increasing length G2 does correlate                        et al., 1995 DNA-dependent protein kinase catalytic subunit: a <br />                                                                              relative phosphatidylinositol 3-kinase Ataxia Telangiec- <br /> with suppression checkpoint defects (Enoch                            tasia gene product. Cell 82: 849&#226;&#8364;&#8220;856. <br /> Nurse 1990). It possible stress-                   Hartwell, L. H., T. A. Weinert, 1989 Checkpoints: controls <br /> response pathway stimulates mitosis inhibiting com-                       ensure order cell cycle events. Science 246: 629&#226;&#8364;&#8220;634. <br />                                                                          Hirano, T., S. Funahashi, T. Uemura M. Yanagida, 1986 Isola- <br /> ponents cell cycle machinery normally re-                        tion characterization Schizosaccharomyces pombe cut mutants <br /> strain mitosis response checkpoint signals. Com-                       block nuclear division cytokinesis. EMBO J. 5: 2973&#226;&#8364;&#8220; <br /> promising checkpoint control confer selective                          2979. <br />                                                                          Humphrey, T., T. Enoch, 1995 Keeping mitosis check. Cur. <br /> advantage allows cells complete cell                       Biol. 5: 376&#226;&#8364;&#8220;379. <br /> cycle rapidly, enter G1 prepare conjugation                      Lew, D. J., S. Kornbluth, 1996 Regulatory roles cyclin depen- <br /> and sporulation. Normally reducing cell cycle fidelity                    dent kinase phosphorylation cell cycle control. Curr. Opin. <br />                                                                              Cell. Biol. 8: 795&#226;&#8364;&#8220;804. <br /> deleterious increases frequency mutations                   Lieberman, H. B., K. M. Hopkins, M. Nass, D. Demetrick S. <br /> (Hartwell Weinert 1989). However, un-                            Davey, 1996 A human homolog Schizosaccharomyces pombe <br /> der stressful conditions, mechanism stimulates                        rad91 checkpoint control gene. Proc. Natl. Acad. Sci. USA 93: <br />                                                                              13890&#226;&#8364;&#8220;13895. <br /> mitosis generates new mutations advantageous,                     Lundgren, K., N. Walworth, R. Booher, M. Dembski, M. Kirsch- <br /> as mutations promote <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>survival</span> harsh                          ner et al., 1991 mik1 wee1 cooperate inhibitory tyro- <br /> environment. Our studies raise intriguing                      sine phosphorylation cdc2. Cell 64: 1111&#226;&#8364;&#8220;1122. <br />                                                                          Maundrell, K., 1990 nmt1 Fission Yeast. J. Biol. Chem. 265: <br /> possibility stringency checkpoint response                     10857&#226;&#8364;&#8220;10864. <br /> is variable modulated environmental condi-                  Millar, J. B. A., V. Buck M. G. Wilkinson, 1995 Pyp1 Pyp2 <br /> tions fixed property cell cycle.                        PTPases dephosphorylate osmosensing MAP kinase controlling <br />                                                                              cell size division fission yeast. Genes Dev. 9: 2117&#226;&#8364;&#8220;2130. <br />    We thank Paul Russell helpful discussions generous gifts      Millar, J. B. A., P. Russell, J. E. Dixon K. L. Guan, 1992 Nega- <br /> of invaluable strains used analysis stress response. We       tive regulation mitosis functionally overlapping PTPases <br /> also thank Hamid Ghazizadeh, Angela Vilche, Gladys Reimundo,                 fission yeast. EMBO J. 11: 4943&#226;&#8364;&#8220;4952. <br /> and Maria Sanchez technical assistance; Fred Winston, Li-Huei        Moreno, S., A. Klar P. Nurse, 1991 Molecular genetic analysis <br />                                                                              fission yeast Schizosaccharomyces pombe, pp. 795&#226;&#8364;&#8220;823 Methods <br /> Tsai, Peter Adams, Carolyn Chapman, Kristi Forbes, Cory Kos- <br />                                                                              Enzymology: Guide Yeast Genetics Molecular Biology, edited <br /> trub, Elizabeth Moynihan Elspeth Stewart valuable ad- <br />                                                                              C. Guthrie G. R. Fink. Academic Press, New York. <br /> vice comments manuscript. T.H. supported            Murakami, H., H. Okayama, 1995 A kinase fission yeast <br /> European Molecular Biology Organization long-term fellowship,              responsible blocking mitosis S phase. Nature 374: 817&#226;&#8364;&#8220;819. <br /> Human Frontier Science Program Organization fellowship,         Naranda, T., M. Kainuma, S. E. Macmillan J. W. B. Hershey, <br /> Charles A. King Trust Fellow (Fleet Investment Management, Trustee,          1997 The 39-Kilodalton subunit eukaryotic translation initia- <br /> Boston). Work T.E.&#226;&#8364;&#8482;s laboratory supported grant          tion factor 3 essential complex&#226;&#8364;&#8482;s integrity cell <br /> National Institutes Health (GM50015).                                     viability Saccharomyces cerevisiae. Mol. Cell. Biol. 17: 145&#226;&#8364;&#8220;153. <br />                                                                          Neer, E. J., C. J. Schmidt, R. Nambudripad T. F. Smith, <br />                                                                              1994 The ancient regulatory-protein family WD-repeat pro- <br />                                                                              teins. Nature 371: 297&#226;&#8364;&#8220;300. <br />                      LITERATURE CITED                                    Prentice, H. L., 1992 High efficiency transformation Schizosac- <br />                                                                              charomyces pombe electroporation. Nucleic Acids Res. 20: 621. <br /> Basi, G., E. Schmid K. Maundrell, 1993 TATA box mutations            Rave, N., R. Crkvenjakov H. Boedtker, 1979 Identification of <br />     Schizosaccharomyces pombe nmt1 promoter affect transcrip-         procollagen mRNAs transfe <br /> </body></html>