<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Aug. 2003, p. 4490&#226;&#8364;&#8220;4498                                                                               Vol. 185, No. 15 <br /> 0021-9193/03/$08.00&#226;&#171;&#185;0 DOI: 10.1128/JB.185.15.4490&#226;&#8364;&#8220;4498.2003 <br /> Copyright &#194;&#169; 2003, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          Essential Nature mreC Determinant Bacillus subtilis&#226;&#8364;&#160; <br />                                            Joong-Chul Lee George C. Stewart* <br />                            Department Diagnostic Medicine/Pathobiology, College Veterinary Medicine, <br />                                          Kansas State University, Manhattan, Kansas 66506 <br />                                                Received 10 March 2003/Accepted 29 April 2003 <br />  <br />              The mre genes Escherichia coli Bacillus subtilis cell shape determination genes. Mutants affected <br />           mre function spheres instead normal rods. Although mre determinants required for <br />           viability E. coli, mreB determinant essential gene B. subtilis. Conflicting results re- <br />           ported membrane-associated proteins MreC MreD essential proteins. Further- <br />           more, MreB protein studied detail, roles MreC MreD proteins <br />           cell shape determination unknown. We constructed strain B. subtilis expression mreC <br />           determinant dependent addition isopropyl-&#226;?&#164;-D-thiogalactopyranoside culture medium. <br />           Utilizing conditional strain, shown mreC essential gene B. subtilis. Furthermore, was <br />           shown cells lacking sufficient quantities MreC undergo morphological changes, namely, swelling and <br />           twisting cells, followed cell lysis. Electron microscopy utilized demonstrate a <br />           polymeric material accumulated division septum cells presence material <br />           correlated bending cell. The best explanation results MreC protein involved <br />           control septal versus long-axis peptidoglycan synthesis, cells lacking MreC perform aberrant <br />           septal peptidoglycan synthesis, lysis results deficiency long-axis peptidoglycan synthesis. <br />  <br />  <br />    Bacteria characteristic shapes, important                    prevalence site septum formation leads for- <br /> taxonomic features. Most bacteria possess fixed shape, al-                   mation coccobacilli cocci. Bacteria char- <br /> though members genus Arthrobacter transition rods                  acteristic coccal shape fall categories. Some possess <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> cocci depending growth conditions (12). The bacterial                    site septum formation exist cocci, <br /> cell wall thought major determinant cell shape,                carry peptidoglycan biosynthetic sites and <br /> as purified murein sacculus retains characteristic mor-                transition rods septum-specific site blocked <br /> phology cell, specific changes chemical com-                 <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> antibiotic treatment (21). <br /> position peptidoglycan noted shape-altered                    Mutations genome Escherichia coli result in <br /> mutants (25). However, shape determination complicated                    altered cell morphology identified, precise <br /> process, involves topology peptidoglycan                 biochemical functions affected genes been <br /> biosynthesis contributions components                 determined. The mre operon E. coli identified locus <br /> cell envelope involvement cell division ma-                  associated cell shape determination sensitivity to <br /> chinery. A mutation tagF (rodC) determinant Bacillus                 amdinocillin (11, 27&#226;&#8364;&#8220;30). The operon consists genes, <br /> subtilis gives rise shift rod-shaped spherical cells,             mreBCD, orfE, cafA. A mutation mreB deletion of <br /> indicating contribution teichoic acid determination                mreBCD produces morphological shift cell population, <br /> of cell morphology (4, 16, 23).                                                resulting spherical cells. The MreB protein shows sequence <br />    A model bacterial shape regulation, two-competing-                  similarity members Hsp70 superfamily, to <br /> sites model, relates cell shape balance cell                FtsA cell division protein (6, 14). Inactivation mreB is <br /> wall synthesis long axis rod-shaped cell                    associated increased activity FtsI (penicillin <br /> septum-specific peptidoglycan synthesis, proposed <br />                                                                                binding protein 3) protein, responsible septal pep- <br /> (21, 24). This model proposes shape determination in <br />                                                                                tidoglycan synthesis (30). It proposed increase <br /> bacterial rods depends activity peptidoglycan <br />                                                                                leads hyperseptation activity spherical cell formation. <br /> synthesis reactions (sites) mutually exclusive. Con- <br />                                                                                Introduction mre genes multicopy plasmid led a <br /> sequently, lateral wall extended septum <br />                                                                                reduction FtsI activity associated filamentation phe- <br /> formation step cell division, septal peptidoglycan syn- <br />                                                                                notype, presumably increased production MreB. <br /> thesis does occur lateral wall elongation. The shape <br />                                                                                On basis observations, postulated MreB <br /> of cell determined balance competing <br />                                                                                negative regulator FtsI (30). The roles MreC, <br /> reactions. The normal balance produces rods, abnormal <br />                                                                                MreD, OrfE proteins shape determination remain un- <br /> prevalence lateral wall elongation yields filaments, a <br />                                                                                defined. <br />                                                                                   The MreB homolog B. subtilis contained the <br />    * Corresponding author. Mailing address: Department Diagnos-             divIVB minicell operon (20, 26). This operon contains the <br /> tic Medicine/Pathobiology, College Veterinary Medicine, Kansas              mreBCD determinants minCD minicell-associ- <br /> State University, 1800 Denison Ave., Manhattan, KS 66506-5603. Phone: <br /> (785) 532-4419. Fax: (785) 532-4039. E-mail: stewart@vet.ksu.edu. <br />                                                                                ated genes. A mutant allele mreD, rodB1, induces for- <br />    &#226;&#8364;&#160; Contribution 03-298-J Kansas Agricultural Experiment             mation spherical cells. Thus, B. subtilis mre genes are <br /> Station.                                                                       associated cell shape determination. However, was <br />  <br />                                                                         4490 <br />  VOL. 185, 2003                                                                                                                B. SUBTILIS mreC             4491 <br />  <br />  <br />                                               TABLE 1. Bacterial strains plasmids used study <br />   Strain plasmid                                               Genotype description                                                   Source reference <br />  <br /> B. subtilis 168 strains <br />   KSS1001                       trpC2                                                                                                       K. Bott <br />   KSS1570                       amyE::pdivIVB mreD-minCD                                                                                    This study <br />   KSS1571                       &#226;?&#8364; pLEE100, amyE::pdivIVB mreD-minCD                                                                         This study <br />   KSS1572                       trpC2, &#226;?&#8364; pLEE100                                                                                            This study <br />   KSS1573                       KSS1571 (pLEE504)                                                                                           This study <br />   MO1099                        amyE::erm                                                                                                   9 <br />  <br /> E. coli strains <br />   DH5&#226;?&#163;                          endA1 gyrA96 hsdR17 &#226;&#338;&#172;lacU169 (&#226;?&#190;80lacZ&#226;&#338;&#172;M15) recA1 supE44 thi-1                                               BRLc <br />   M15                           Strr lacZ&#226;&#338;&#172;M15 (pRep4 Kmr)                                                                                   Qiagen <br />  <br /> Plasmids <br />   pCHA2489                      mreC ORFa PCR product cloned pT7Blue                                                                   This study <br />   pCHA2506                      NotI (in mreC, blunted)-SalI (vector site) mreC DNA fragment pCHA2489 cloned                      This study <br />                                   EcoICRI-SalI site pQE30 <br />   pCR2.1                        PCR fragment cloning vector, Apr Kmr                                                                        Invitrogen <br />   pDG364                        amyE locus integration vector                                                                               BGSCd (9) <br />   pDG3053                       Shuttle expression plasmid utilizing S. aureus lactose operon promoter; Apr Kanr                        This study <br />   pLEE50                        pUS19 lacI spac EcoRI-BamHI insert pSI-1                                                      This study <br />   pLEE100                       MreC-inducible expression integration vector                                                                This study <br />   pLEE200                       pDG364 containing pdivIVB mreC-minCD                                                                        This study <br />   pLEE504                       pDG3053 containing truncated mreCb                                                                          This study <br />   pQE30                         Six-histidine tag vector, Apr                                                                               Qiagen <br />   pSI-1                         IPTG-inducible expression shuttle plasmid                                                                   D. Henner (2, 33) <br />   pT7Blue                       PCR product cloning vector, Apr                                                                             Invitrogen <br />   pUS19                         pUC19 Spr expressed B. subtilis                                                                     3 <br />   a <br />     ORF, open reading frame. <br />   b <br />     Truncated mreC (contains bp 1 540). <br />   c <br />     BRL, Bethesda Research Laboratories. <br />   d <br />     BGSC, Bacillus Genetic Stock Center, Ohio State University. <br />  <br />  <br />  <br /> reported B. subtilis mre genes appear essential                 cell. Their studies confirmed earlier suggestions the <br /> for cell viability, unlike E. coli counterparts (20, 26). Two            mreB determinant essential cell viability the <br /> additional determinants products substantial se-                    mreC mreD determinants inactivated loss <br /> quence identity MreB identified B. subtilis                viability &#226;&#8364;&#339;mild phenotypic consequences.&#226;&#8364;? The <br /> chromosome. The mbl determinant appears play role                      Streptomyces coelicolor mreB determinant shown be <br /> cell morphology, inactivation nonessential determi-                essential viability, mreC gene inactivated <br /> nant does result spherical cell formation (1). The mreBH                gene disruption resulted mutants viable but <br /> determinant, maps near kinC B. subtilis chromo-                   showed significant growth retardation comparison the <br /> some, characterized (<span id='am-1' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span>                     wild type (7). To better understand biological role the <br /> D37799). The B. subtilis Mre proteins studied               MreC protein B. subtilis reconcile conflicting <br /> than E. coli counterparts. Their sequence similarity              conclusions regarding essential nature determinant, <br /> E. coli Mre proteins, <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span> rodB1 allele,                   created strain B. subtilis conditionally expresses <br /> suggests function involves interaction                 MreC protein examined effect depletion this <br /> the septum-specific peptidoglycan biosynthetic enzyme, name-                   protein growth morphology cell. <br /> ly, Pbp2B (20, 26, 32). <br />    Introduction mreC mreD genes Bacillus <br /> stearothermophilus B. subtilis resulted overexpression                                      MATERIALS AND METHODS <br />  <br /> of protease (18). This effect protease expression required                     Bacterial strains, plasmids, growth conditions. The bacterial strains and <br /> the presence mre determinants; effect seen                 plasmids used study listed Table 1. Media growth conditions for <br />                                                                                E. coli B. subtilis described previously (8, 26). Phleomycin (zeocin) <br /> either determinant introduced host. It                 utilized 10 7 &#226;?&#174;g/ml selection E. coli B. subtilis, respec- <br /> been determined effect protease expression                  tively; kanamycin utilized 20 5 &#226;?&#174;g/ml selection E. coli and <br /> a direct indirect effect production B. stearo-                B. subtilis, respectively; spectinomycin utilized concentration 100 <br /> thermophilus MreCD proteins B. subtilis. The MreC protein                   &#226;?&#174;g/ml, chloramphenicol utilized 10 &#226;?&#174;g/ml. <br />                                                                                   To grow B. subtilis strain KSS1571, tryptic soy broth (TSB) (Difco) tryptic <br /> has shown localize division septum dividing <br />                                                                                soy agar (TSA) used. The culture media supplemented specti- <br /> cells B. subtilis (19).                                                     nomycin 2 mM isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG). After cells <br />    Jones coworkers indicated MreB Mbl pro-                    grown broth overnight, harvested centrifugation, washed <br /> teins B. subtilis filamentous helical structures              equal volume phosphate-buffered saline, diluted 100-fold fresh <br /> have cytoskeleton-like role bacterial shape determination                 broth, grown 6 h 100 &#226;?&#174;g spectinomycin ml various <br />                                                                                amounts IPTG. All incubations performed 37&#194;&#176;C shaking 200 <br /> (17). They proposed model MreB protein                       rpm, cell density monitored spectrophotometrically 500 nm. At the <br /> concerned cell width control, Mbl im-                     end 6-h cultivation period, absorbance values determined. The cells <br /> portant maintaining linearity longitudinal axis               harvested centrifugation, cells resuspended iden- <br />  4492       LEE AND STEWART                                                                                                                             J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. (A) Integration pLEE100 B. subtilis chromosome. As result plasmid integration, mreB determinant divIVA <br /> operon remains control natural promoter (pdivIVA), mreB-distal genes operon control the <br /> IPTG-inducible spac promoter (pspac). (B) The mreD-minC-minD determinants, expressed divIVA operon promoter, positioned <br /> within amyE locus (9). Abbreviations: Apr, ampicillin resistance determinant; Spr, spectinomycin resistance determinant; pspac, spac promoter; <br /> pdivIVB, divIVB operon promoter. <br />  <br />  <br /> tical cell densities. One-milliliter samples collected electron microscopy      To eliminate possible polar effects mreC-distal genes divIVB <br /> observation methylene blue staining conventional microscopy. The remainder     operon, merodiploid strain B. subtilis constructed extra copy <br /> of culture utilized Western blot analyses muramic acid assays.        mreD minCD, controlled divIVB operon promoter, present. The <br />    To induce switch teichoic acid teichuronic acid synthesis, low-        divIVB promoter amplified PCR (forward primer, 5&#226;&#172;&#732; CGGAATTCATC <br /> phosphate medium used described previously (31).                               TATTGTCCACCCGCG 3&#226;&#172;&#732;; reverse primer, 5&#226;&#172;&#732; GCTCTAGAGCAAAAATAC <br />    General methods. Competent E. coli B. subtilis cells prepared         CCTAAAGGG 3&#226;&#172;&#732;) cloned using EcoRI XbaI sites incorporated into <br /> transformed methods Dagert Ehrlich (10) Erickson                5&#226;&#172;&#732; ends primers. The promoter fragment positioned the <br /> Copeland (13), respectively. Plasmid DNA E. coli prepared             mreD-minC-minD determinants (amplified using 5&#226;&#172;&#732; GCTCTAGAACAGAG <br /> alkaline lysate procedure Birnboim Doly (5). Isolation chromosomal          GAGGAAGGATCG 3&#226;&#172;&#732; 5&#226;&#172;&#732; CGGGATCCATTAAGATCTTACTCCG 3&#226;&#172;&#732; as <br /> and plasmid DNAs B. subtilis performed described previously (8, 26).      PCR primers, introducing XbaI BamHI sites, respectively) was <br />    Construction MreC-inducible strain. The HincII-HindIII sequence           inserted plasmid pDG364 (9) EcoRI-BamHI fragment. All cloned <br /> the multiple cloning site pUS19 (3) removed, DNA fragment              PCR products sequenced ensure constructs constructed <br /> containing spac promoter lacI coding region excised              correctly. The resulting plasmid, pLEE200, transformed B. subtilis <br /> plasmid pSI-1 (2, 33) EcoRI BamHI ligated modified         strain MO1099 create strain KSS1570, merodiploid mreD, <br /> pUS19 plasmid cleaved enzymes, generating plasmid pLEE50. A             minC, minD genes. The second copy determinants positioned <br /> 500-bp fragment encoding 3&#226;&#172;&#732; end mreB generated PCR                  amyE locus (9) (Fig. 1B). <br /> positioned HindIII-SphI fragment downstream IPTG-inducible spac              To express truncated form MreC, sequence corresponding the <br /> promoter plasmid pLEE50 create pLEE100 (Fig. 1). This plasmid               ribosome binding site 504 bp mreC open reading frame was <br /> cannot replicate B. subtilis cells, contains spectinomycin resistance     amplified PCR (forward primer, 5&#226;&#172;&#732; GAGCTCGAGTTCAATAGAAGAG 3&#226;&#172;&#732;; <br /> determinant expressed gram-positive host. Integration      reverse primer, 5&#226;&#172;&#732; GCATGCCAGACGTAAAATTGTTAA 3&#226;&#172;&#732;). The PCR prod- <br /> plasmid B. subtilis chromosome mreB locus results mreB             uct cloned SacI-SphI fragment immediately downstream Staph- <br /> expression divIVB operon promoter, expression mreCD di-            ylococcus aureus lactose operon promoter plasmid pDG3053 create plasmid <br /> rected spac. The distal determinants minC minD driven         pLEE504. This plasmid shuttle plasmid gram-positive replication func- <br /> the spac promoter weak internal promoter divIVB operon (26). The         tions provided staphylococcal plasmid pUB110 ampicillin resistance <br /> 3&#226;&#172;&#732; terminus mreB positioned downstream spac, instead 5&#226;&#172;&#732;              replication functions E. coli provided pUC19. Expression this <br /> terminus mreC. The reason avoid duplication       promoter constitutive plasmid lacks functional lacR repressor- <br /> mreC determinant integration plasmid. It preferable 5&#226;&#172;&#732;      encoding determinant. <br /> untranslated leader mreC-containing transcript expressed          Scanning electron microscopy. B. subtilis cells cultured mid- <br /> truncated mreC, include transmembrane domain (19).                    logarithmic growth phase, 1 ml culture fixed overnight 4&#194;&#176;C in <br />  VOL. 185, 2003                                                                                                               B. SUBTILIS mreC        4493 <br />  <br /> 20 volumes modified Karnowsky&#226;&#8364;&#8482;s fixative (2% paraformaldehyde, 2.5% glu- <br /> taraldehyde, 1.7 mM CaCl2 0.1 M cacodylate buffer [pH 7.4]). After cells <br /> were fixed, washed twice 0.1 M cacodylate buffer (pH 7.4) and <br /> postfixed 1% osmium tetroxide 0.1 M cacodylate buffer overnight 4&#194;&#176;C. <br /> After cells washed twice double-distilled water, resus- <br /> pended double-distilled water, mounted coverslips coated poly-L- <br /> lysine, dehydrated series alcohol solutions (30, 50, 70, 95, 100%). <br /> Dehydrated samples treated 5 min hexamethyldisilazane (Electron <br /> Microscopy Supplies, Ft. Washington, Pa.), dried, mounted, sputter coated <br /> with gold (S150A sputter coater). Samples viewed Hitachi H-300 <br /> electron microscope 3010 scanning image accessory. <br />     Transmission electron microscopy. B. subtilis cells cultured mid- <br /> logarithmic growth phase. The cells pelleted fixed modified Kar- <br /> nowsky&#226;&#8364;&#8482;s fixative, washed, postfixed 1% osmium tetroxide described <br /> above. After 5-min washes cold double-distilled water, cells were <br /> immobilized 2% TSA dehydrated series ethanol solutions (50, 70, <br /> 70, 95, 100%). Dehydrated specimens stained 60 min en bloc with <br /> 5% uranyl acetate 70% ethanol, dehydration completed two <br /> 20-min rinses 100% acetone, used transitional solvent. The <br /> cell blocks infiltrated increasing concentrations resin in <br /> acetone, ending 100% resin periods 2 12 h 8 12 h. All samples <br /> were embedded Epon LX112 embedding medium 18 20 h 42&#194;&#176;C <br /> and 24 h 60&#194;&#176;C. Embedded samples trimmed sectioned            FIG. 2. Growth MreC expression MreC-inducible strain. <br /> Ultracut E-Reichert-Jung ultramicrotome (C. Reichert Optische Werke AG,               B. subtilis KSS1571 cultured TSB containing 2 mM IPTG over- <br /> Vienna, Austria). Thin sections (approximately 80 &#226;?&#174;m thick) sputter coated       night 37&#194;&#176;C, diluted 1:100 fresh medium supplemented dif- <br /> silver gold retrieved copper grids stained uranyl acetate      ferent concentrations IPTG, incubated 6 h. MO1099, <br /> and lead citrate. Grids examined viewed Hitachi H-300 electron        wild-type MreC control strain, cultured TSB IPTG. <br /> microscope.                                                                           Growth determined absorbance 500 nm (B). Then cells <br />     Muramic acid assay. The peptidoglycan content bacterial cells          harvested lysed, cell lysates electrophoresed <br /> determined method Hadzija (15). B. subtilis cells cultured TSB      Western blotted. The blots probed rabbit anti-MreC <br /> to late-logarithmic growth phase, harvested centrifugation, washed      antibody (A). In panel B, mean values determinations <br /> TE buffer (10 mM Tris-HCl, 1 mM EDTA [pH 8.0]). The cell amounts            shown, range values indicated. <br /> adjusted culture densities (A500 &#226;&#171;&#189; 1.8). The cells lysed <br /> using mini beadbeater-8 (Biospec Products, Bartlesville, Okla.). Following cell <br /> disruption, samples incubated 15 min 75&#194;&#176;C. Samples kept       defined. To examine effect loss MreC produc- <br /> &#226;&#171;&#186;80&#194;&#176;C assayed. The cell wall samples resuspended 20 ml of <br />                                                                                       tion B. subtilis, created strain conditional expres- <br /> chloroform-saturated TE buffer. DNase (0.5 ml 1-mg/ml solution) and <br /> RNase (1.0 ml 5-mg/ml solution) added incubated 37&#194;&#176;C 30          sion MreC protein. This construct pLEE100 is <br /> min. Trypsin (100 &#226;?&#174;g) added incubated 37&#194;&#176;C 6 h. The cell wall          depicted Fig. 1. To eliminate possible polar effects the <br /> fraction recovered centrifugation (12,000 &#226;&#171;&#187; g, 45 min) washed          mreC-distal genes divIVB operon, merodiploid strain <br /> time TE buffer.                                                                  B. subtilis, KSS1571, constructed. In strain, extra <br />     Aliquots lysed cells brought 0.5-ml volume 1.0 M NaOH <br /> and incubated 38&#194;&#176;C 30 min. Then 0.5 ml 0.5 M H2SO4 5 ml of <br />                                                                                       copy mreD minCD, controlled divIVB operon pro- <br /> concentrated H2SO4 added, samples placed boiling water         moter, inserted amyE locus chromosome of <br /> bath 5 min. The samples cooled, 0.05 ml 4% (wt/vol) CuSO4        B. subtilis. Western blot analysis utilizing rabbit polyclonal <br /> &#194;&#183; 5H2O 0.1 ml 1.5% (wt/vol) p-hydroxydiphenyl (in 95% ethanol)            anti-MinC antiserum indicated MinC produced at <br /> added. The samples incubated 30&#194;&#176;C 30 min, absorbance <br />                                                                                       normal levels IPTG-independent fashion, indicating that <br /> was determined 560 nm. A standard curve consisting 0 20 &#226;?&#174;g muramic <br /> acid (Sigma Chemical Co., St. Louis, Mo.) utilized assay.                  determinants amyE locus expressed (data not <br />     Preparation polyclonal antiserum MreC. Histidine-tagged MreC           shown). <br /> protein purified, anti-MreC polyclonal antiserum raised rabbits        B. subtilis strain KSS1571 fails grown TSA plates lack- <br /> previously described (19).                                                            ing inducer MreC expression. Only residual growth in <br />     Western blot analysis. B. subtilis cells cultured TSB without <br /> IPTG mid-logarithmic growth phase. The cultures densities deter- <br />                                                                                       initial area streak plate obtained, presum- <br /> mined spectrophotometrically 500 nm, culture volumes adjusted         ably intracellular carryover IPTG over- <br /> to equivalent cell densities. Equivalent amounts cells cultures      night broth culture used inoculate plates. Therefore, <br /> were harvested centrifugation resuspended 1 ml TE buffer. The cells      strain conditional mutant dependent the <br /> were lysed using mini beadbeater-8, lysate loaded sodium <br />                                                                                       presence IPTG culture medium growth. This <br /> dodecyl sulfate&#226;&#8364;&#8220;4 20% polyacrylamide gradient gels (Ready Gels; Bio-Rad). <br /> The electrophoretically resolved proteins transferred nitrocellulose        result suggests MreC essential B. subtilis viability. <br /> filter probed anti-MreC antiserum. Goat anti-rabbit immunoglob-             Effect MreC concentration cell growth. Because mreC <br /> ulin G conjugated alkaline phosphatase (Sigma Chemical Company) used           appeared required growth, B. subtilis KSS1571 was <br /> as secondary antibody, immunoreactive proteins detected using        cultured TSB containing different concentrations IPTG <br /> nitroblue tetrazolium 5-bromo-4-chloro-3-indolylphosphate substrates. <br />                                                                                       (0, 10, 50, 250 &#226;?&#174;M) investigate growth yields this <br />                                                                                       B. subtilis strain function IPTG concentration. In addi- <br />                                                                                       tion, Western blots cellular proteins prepared, utilizing <br />                                   RESULTS <br />                                                                                       rabbit polyclonal anti-MreC antibody. The growth <br />   Construction MreC-inducible strain. Although                              achieved concentration IPTG correlated the <br /> MreB protein proposed helical cytoskeletal                           MreC protein produced, indicated Western <br /> element involved cell <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> determination B. subtilis (17),                    blot analysis (Fig. 2). Low levels MreC observed the <br /> the functions mreC mreD determinants                              absence IPTG induction result leaky unin- <br />  4494     LEE AND STEWART                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />  FIG. 3. Morphology B. subtilis cells. Smears MO1099 (wild-type parent) (A), KSS1571 cultured presence 250 &#226;?&#174;M IPTG (B), and <br /> KSS1571 cultured absence IPTG (C) stained crystal violet examined microscopy. All images same <br /> magnification. <br />  <br />  <br />  <br /> duced expression spac promoter carryover IPTG            cell morphology, phenotypic changes B. subtilis <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> inoculum culture. The growth level,             KSS1571 examined light microscopy differ- <br /> of MreC present cells, higher wild-type strain          ent IPTG conditions, 250 0 &#226;?&#174;M, compared the <br /> B. subtilis achieved strain KSS1571 pres-         &#226;&#8364;&#339;wild-type&#226;&#8364;? MO1099 (Fig. 3). MO1099 normal rod <br /> ence 250 &#226;?&#174;M IPTG. Increasing IPTG concentration                  shape (Fig. 3A), altered cell morphologies ob- <br /> above 250 &#226;?&#174;M did noticeably improve level growth             served KSS1571 cultures. At 250 &#226;?&#174;M IPTG, cells <br /> or MreC production, did eliminate changes             shorter, evidence curvature cells, <br /> cell morphology described (data shown). The de-               especially ends (Fig. 3B). In absence IPTG in- <br /> pendence cell growth MreC production supports                 duction, swollen cells observed, cells more <br /> conclusion MreC protein essential protein             spherical, presence debris cell ghosts indicated <br /> B. subtilis. Integration pLEE100 B. subtilis lacking        lysis occurred (Fig. 3C). <br /> extra copies mreD minCD determinants (KSS1572) gives                Effect MreC cell morphology B. subtilis. To <br /> the growth profile KSS1571, indicating             better define morphological changes obtained different <br /> increased expression divIVB-distal determinants             IPTG concentrations, cells prepared scanning elec- <br /> not responsible observed growth deficit.                        tron microscopy (Fig. 4). Reduction cellular MreC levels <br />    To determine effects reduced mreC expression levels           resulted shift normal rod shape bent <br />  <br />  <br />  <br />  <br />   FIG. 4. Scanning electron microscopy MreC-depleted B. subtilis cells. Images strain MO1099 (A) KSS1571 (B F). The <br /> MreC-inducible strain cultured presence 250 &#226;?&#174;M (B), 50 &#226;?&#174;M (C), 10 &#226;?&#174;M (D) IPTG absence IPTG (E F). <br /> (F) The MreC-inducible strain cultured presence 0.3 M sucrose. Bars micrographs indicated sizes micrometers. <br />  VOL. 185, 2003                                                                                                       B. SUBTILIS mreC           4495 <br />  <br />  <br />  <br />  <br />   FIG. 5. Transmission electron microscopy MreC-depleted B. subtilis cells. Images strain MO1099 (A) KSS1571 (B F). The <br /> MreC-inducible strain cultured presence 250 &#226;?&#174;M (B), 50 &#226;?&#174;M (C), 10 &#226;?&#174;M (D) IPTG absence IPTG (E F). <br /> (F) The MreC-inducible strain cultured presence 0.3 M sucrose. The arrows panels C D indicate areas polymer <br /> accumulation. The arrowhead panel E indicates region cell wall thinner ruptured. Bars, 1 &#226;?&#174;m. <br />  <br />  <br />  <br /> rod, bend usually occurring near end cell,        ent concave bent cell. On basis its <br /> giving &#226;&#8364;&#339;shepherd&#226;&#8364;&#8482;s crook&#226;&#8364;? appearance (Fig. 4B C). At            location cell envelope appearance the <br /> 250 &#226;?&#174;M IPTG, hook observed end cell,            electron microscope, likely accumulated material <br /> with rest cell having normal appearance (Fig. 4B). As        peptidoglycan. <br /> the concentration IPTG reduced, bent rods begin            At lower IPTG concentrations, accumulation poly- <br /> swell, indicated increased cell diameter (Fig. 4C),      meric material increased. The cells began evidence of <br /> twist irregular shapes (Fig. 4C D). Further reduction        swelling rounding, twisting rod-shaped cells oc- <br /> in IPTG concentrations produced cells irregular spher-        curred (Fig. 5C D). Removal IPTG inducer from <br /> ical shape reduction cell length increased           B. subtilis KSS1571 resulted cells irregular-appearing <br /> diameter relative control strain MO1099. Culture        septa, giving appearance different-sized daughter cells <br /> absence IPTG results little growth. Under condi-          (Fig. 5E). However, result sectioning <br /> tions, B. subtilis cells predominantly spherical, resid-     cells twisted. The frequency of <br /> ual rods tended remain associated chains, cell         septation affected, producing different- <br /> lysis evident production ghost cells (Fig. 4E).       sized progeny cells. The micrographs suggest mislocaliza- <br /> MreC-depleted B. subtilis appear vulnerable osmotic          tion misdirection septum formation occur the <br /> pressure-induced lysis weakened cell wall structure. To        MreC-depleted cells. Cell lysis typical observation the <br /> minimize cell lysis osmotic imbalance, 0.3 M sucrose           absence IPTG. The cell walls resulting ghosts did <br /> incorporated growth medium containing 0 &#226;?&#174;M IPTG.             uniform thickness (Fig. 5E). The apparent rupture <br /> Under condition, majority cells assumed             point cell wall thinner rest wall (Fig. <br /> spherical shape (Fig. 4F). The observed morphological shifts          5E). Sucrose supplementation growth medium reduced <br /> and stabilization cells iso-osmotic environment      apparent cell lysis (Fig. 5F). Growth these <br /> suggest reduction MreC levels associated with <br /> a deficiency cell wall structure bacteria. <br />    Transmission electron microscopy used examine the <br />                                                                               TABLE 2. Muramic acid content MreC-deficient cells <br /> ultrastructure MreC-deficient strain (Fig. 5). At an <br /> IPTG concentration 250 &#226;?&#174;M, rod-shaped cells showed               Strain <br />                                                                                                  IPTG concn                   Muramic acid content a <br />                                                                                                     (&#226;?&#174;M)                     (&#226;?&#174;g/mg cells [dry wt]) <br /> evidence bending. The bending appeared cen- <br /> tered site septum formation (Fig. 5B). Examination         KSS1001                          0                            1.9 &#226;&#171;&#190; 0.1 <br /> the septum site indicated accumulation polymeric         KSS1571                         10                            1.8 &#226;&#171;&#190; 0.1 <br /> material appeared cell site septa-       KSS1571                         50                            2.0 &#226;&#171;&#190; 0.2 <br /> tion. Furthermore, accumulated material pres-           a <br />                                                                             Values means &#226;&#171;&#190; standard deviations determinations. <br />  4496     LEE AND STEWART                                                                                                            J. BACTERIOL. <br />  <br />  <br />                                                                          conditions allowed progression cellular defect pro- <br />                                                                          ceed longer period time. With longer time, the <br />                                                                          accumulation polymeric material progressed the <br />                                                                          septum site spread surface spherical cells <br />                                                                          (Fig. 5F). <br />                                                                             Muramic acid content MreC-depleted B. subtilis. MreC <br />                                                                          depletion resulted morphological changes lysis B. <br />                                                                          subtilis, suggesting defect cell wall biosynthesis be <br />                                                                          involved. Peptidoglycan major structural component of <br />                                                                          gram-positive cell wall comprises 40 90% cell <br />                                                                          wall. To determine deviation pepti- <br />                                                                          doglycan content cells conditionally express MreC <br />                                                                          grown medium containing various amounts <br />                                                                          IPTG, muramic acid assay used. Muramic acid assays <br />                                                                          study suffer certain limitations. At lower con- <br />                                                                          centrations IPTG, cell death lysis occur cultures, <br />                                                                          affect muramic acid recovered and <br />                                                                          dry weight harvested cells. Despite limitations, <br />                                                                          muramic acid content remained constant different <br />                                                                          IPTG concentrations did differ wild-type <br />                                                                          level (Table 2). <br />                                                                             The conclusion analysis polymeric material <br />                                                                          detected electron microscopy represents substance other <br />                                                                          peptidoglycan material peptidoglycan the <br />                                                                          distribution, total cell wall material, <br />                                                                          altered MreC-deficient strain. <br />                                                                             To evaluate possibility teichoic acid than <br />                                                                          peptidoglycan accumulated polymer, B. subtilis <br />                                                                          KSS1571 cultured reduced phosphate medium. Under <br />                                                                          conditions, teichoic acid replaced teichuronic <br />                                                                          acid. The morphological changes associated defective <br />                                                                          teichoic acid resulting tagF (rodC) mutation elim- <br />                                                                          inated replacement teichoic acid teichuronic acid <br />                                                                          growth conditions (31). When KSS1571 cul- <br />                                                                          tured phosphate-limiting phosphate-replete growth <br />                                                                          conditions, differences seen MreC-dependent <br />                                                                          growth, morphology, polymeric material accumulation in <br />                                                                          cells (data shown). Therefore, switch synthesis <br />                                                                          teichoic acid teichuronic acid did override the <br />                                                                          MreC-associated defect, appears accumulated ma- <br />                                                                          terial teichoic acid. <br />                                                                             Lack complementation MreC activity truncated <br />                                                                          mreC mutant. The mreC mutant described Jones et al. was <br />                                                                          created integrating plasmid containing internal frag- <br />                                                                          ment mreC (a 242-bp fragment initiating unique <br />  <br />  <br />  <br />  <br />                                                                          KSS1573 cultured presence 1 mM IPTG; E, KSS1573 cul- <br />                                                                          tured absence IPTG. (C) Western blot B. subtilis cell lysates <br />                                                                          probed rabbit anti-MreC antiserum. Lanes: 1, MO1099 (wild-type <br />                                                                          control); 2, molecular size markers; 3, KSS1571 grown presence <br />                                                                          250 &#226;?&#174;M IPTG; 4, KSS1571 grown absence IPTG; 5, <br />                                                                          KSS1573 grown presence 250 &#226;?&#174;M IPTG; 6, KSS1573 grown <br />    FIG. 6. (A) Map pLEE504, shuttle plasmid expresses         absence IPTG. The positions full-length MreC (32 kDa) <br /> the 168-residue truncated form MreC S. aureus lac pro-       truncated form MreC (19 kDa) indicated. The amount <br /> moter. The ampicillin resistance determinant (amp), replication pro-     cell lysate loaded times used blot <br /> tein gene (rep), kanamycin resistance determinant (kan), phleomycin      shown Fig. 2A. (D) Photomicrographs methylene blue-stained <br /> (zeocin) resistance determinant (ble), NdeI (Nd) PvuII (Pv)      cells. B. subtilis KSS1571 (a c) KSS1573 (b d) were <br /> restriction sites shown. (B) Growth B. subtilis strains TSB.   cultured presence 1 mM IPTG (a b) absence of <br /> Symbols: &#226;&#8364;&#353;, KSS1001 (wild type); &#226;&#8211;&#160;, KSS1571 cultured presence     IPTG (c d). The arrows panel b indicate cells appear <br /> of 1 mM IPTG; &#228;&#352;?, KSS1571 cultured absence IPTG; F,             swollen bent. <br />  VOL. 185, 2003                                                                                                      B. SUBTILIS mreC             4497 <br />  <br />  <br /> HindIII site) B. subtilis chromosome (17). The inte-        water cell. The swelling cell exaggerates the <br /> gration event generate truncated expressed copy           twisting cell. Eventually, cell wall ruptures, the <br /> mreC positions 1 504 open reading frame, re-          cell lyses. The loss wall integrity loss a <br /> sulting loss 122 amino acid residues protein.     regulated switch septum synthesis lateral wall exten- <br /> We unable obtain viable transformants B.             sion abnormal septal wall synthesis <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. Preven- <br /> subtilis utilizing procedure.                                   tion cell lysis culturing cells iso-osmotic envi- <br />    To evaluate biological activity truncated MreC         ronment leads complete progression thickened wall <br /> protein, truncated allele mreC (contains bp 1 504)     perimeter cell. The cells more <br /> positioned downstream constitutive S. aureus lactose op-        spherical shape, consistent loss lateral wall extension <br /> eron promoter (22) plasmid pDG3053 create pLEE504              capability. <br /> (Fig. 6A). This plasmid introduced mreC expres-            The septation process B. subtilis involves formation of <br /> sion strain KSS1571 produce KSS1573. This isolate exhibits        FtsZ ring, structure recruits division- <br /> only limited growth ultimately lyses inclusion       associated proteins (2). This division ring forms leading <br /> of IPTG culture medium, indicating truncated         edge constriction progresses division <br /> MreC protein lacks sufficient biological activity support         process. Up now, evidence sidedness <br /> growth (Fig. 6B). Western blot analysis indicated trun-     cytokinesis ring. The electron microscopic observations <br /> cated MreC protein expressed cell, lack          reported indicate septal ring associated with <br /> expression explanation failure comple-        cytokinesis B. subtilis does sidedness it. The <br /> ment MreC deficiency (Fig. 6C). In fact, presence         overexpression wall polymeric material occurs one <br /> the plasmid resulted higher level expression wild-type    cell. The differences cell envelope, the <br /> MreC cell. This results presence pUC          ring formed FtsZ division proteins, accounts <br /> plasmid-derived lactose operon operator site plasmid,         septal ring asymmetry unknown. <br /> which able bind LacI repressor protein titrate       The mreC determinant essential determinant B. <br /> the repressor spac promoter operator. Despite          subtilis appears function septation process. MreC <br /> increased production wild-type MreC, culture grew       shown localize division septum B. subtilis <br /> poorly (Fig. 6B) displayed characteristic cell swell-   (19). Truncation protein destroys proper functioning <br /> ing twisting MreC-depleted cells (Fig. 6D). The strain        cell, 168-amino-acid polypeptide does ap- <br /> expressing truncated MreC protein grew poorly          pear able compete full-length version the <br /> KSS1571 absence presence IPTG. Thus,               protein. The truncated protein able compete with <br /> truncated MreC protein able complement pheno-          wild-type protein interaction compo- <br /> type associated loss MreC protein cells,      nent(s) septation apparatus, MreC multimeric <br /> it appears able compete wild-type protein.         protein, able included MreC complex, <br /> This gives cell appearance severe MreC-            inclusion result loss biological activity. <br /> deficient phenotype warranted level full-length <br /> MreC cell.                                                                             ACKNOWLEDGMENTS <br />                                                                         We thank David George technical assistance operation of <br />                                                                      DNA sequencer, Lloyd Willard electron microscopy, and <br />                          DISCUSSION                                  Patricia W. Stewart DVM assistance production the <br />                                                                      anti-MreC antiserum. <br />    The two-competing-sites model peptidoglycan assembly             This work supported grant GM57049 Na- <br /> predicts peptidoglycan assembly long axis      tional Institutes Health. <br /> cell septal peptidoglycan synthesis division mu- <br />                                                                                                      REFERENCES <br /> tually exclusive processes control activi- <br />                                                                      1. Abhayawardhane, Y. K., G. C. Stewart. 1995. Bacillus subtilis possesses <br /> ties influence cell morphology (21, 24). Hyperexpression         second determinant extensive sequence similarity Escherichia <br /> septal peptidoglycan biosynthetic apparatus leads spheri-          coli mreB morphogene. J. Bacteriol. 177:765&#226;&#8364;&#8220;773. <br /> cal shape, hyposeptation activity results filamenta-      2. Beall, B., J. Lutkenhaus. 1991. FtsZ Bacillus subtilis required for <br />                                                                         vegetative septation asymmetric septation sporulation. Genes <br /> tion. Wachi Matsuhashi concluded MreB                  Dev. 5:447&#226;&#8364;&#8220;455. <br /> protein play regulatory role septal synthesis regu-      3. Benson, A. K., W. G. Haldenwang. 1993. Regulation &#226;?&#180;B levels and <br />                                                                         activity Bacillus subtilis. J. Bacteriol. 175:2347&#226;&#8364;&#8220;2356. <br /> lating expression FtsI (30). Our results extend findings   4. Bhavsar, A. P., T. J. Beveridge, E. D. Brown. 2001. Precise deletion of <br /> Wachi Matsuhashi suggesting MreC transmem-              tagD controlled depletion product, glycerol 3-phosphate cytidylyl- <br /> brane protein function control network               transferase, leads irregular morphology lysis Bacillus subtilis grown <br />                                                                         physiological temperature. J. Bacteriol. 183:6688&#226;&#8364;&#8220;6693. <br /> which determines peptidoglycan biosynthesis process         5. Birnboim, H. C., J. Doly. 1979. A rapid alkaline extraction procedure for <br /> active. When MreC production turned off, cells begin          screening recombinant plasmid DNA. Nucleic Acids Res. 7:1513&#226;&#8364;&#8220;1523. <br /> synthesize cell wall material division site aberrant    6. Bork, P., C. Sander, A. Valencia. 1992. An ATPase domain common to <br />                                                                         prokaryotic cell cycle proteins, sugar kinases, actin, Hsp70 shock pro- <br /> manner. Overproduction wall polymer occurs                teins. Proc. Natl. Acad. Sci. USA 89:7290&#226;&#8364;&#8220;7294. <br /> side cell, consequence bending       7. Burger, A., K. Sichler, G. Kelemen, M. Buttner, W. Wohlleben. 2000. <br />                                                                         Identification characterization mre gene region Streptomyces <br /> cell, concave overproduction         coelicolor A3(2). Mol. Gen. Genet. 263:1053&#226;&#8364;&#8220;1060. <br /> this material. Continued synthesis cell wall leads       8. Cha, J.-H., G. C. Stewart. 1997. The divIVA minicell locus Bacillus <br /> progressive spread material septum         subtilis. J. Bacteriol. 179:1671&#226;&#8364;&#8220;1683. <br />                                                                      9. Cutting, S. M., P. B. Vander Horn. 1990. Genetic analysis, p. 27&#226;&#8364;&#8220;74. In <br /> until eventually covers cell. However, process,      C. R. Harwood S. M. Cutting (ed.), Molecular biological methods for <br /> the overall cell wall appears weaken. This leads influx        Bacillus. John Wiley &amp; Sons, Chichester, United Kingdom. <br />  4498       LEE AND STEWART                                                                                                                              J. BACTERIOL. <br />  <br /> 10. Dagert, M., S. D. Ehrlich. 1979. Prolonged incubation calcium chlo-         22. Oskouian, B., G. C. Stewart. 1990. Repression catabolite repression <br />     ride improves competence Escherichia coli cells. Gene 6:23&#226;&#8364;&#8220;28.                  lactose operon Staphylococcus aureus. J. Bacteriol. 172:3804&#226;&#8364;&#8220;3812. <br /> 11. Doi, M., M. Wachi, F. Ishino, S. Tomioka, M. Ito, Y. Sakagami, A. Suzuki,          23. Pooley, H. M., F.-X. Abellan, D. Karamata. 1992. CDP-glycerol:poly- <br />     M. Matsuhashi. 1988. Determinations DNA sequence                     (glycerophosphate) glycerophosphotransferase, involved syn- <br />     mreB gene gene products mre region function                  thesis major wall teichoic acid Bacillus subtilis 168, encoded by <br />     formation rod shape Escherichia coli cells. J. Bacteriol. 170:4619&#226;&#8364;&#8220;          tagF (rodC). J. Bacteriol. 174:646&#226;&#8364;&#8220;649. <br />     4624.                                                                              24. Satta, <br /> </body></html>