<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2004 Genetics Society America <br /> DOI: 10.1534/genetics.104.026252 <br />  <br />  <br />  <br />               Nucleotide Variation Egfr Locus Drosophila melanogaster <br />  <br />         Arnar Palsson,1,2 Ann Rouse,1,3 Rebecca Riley-Berger, Ian Dworkin Greg Gibson4 <br />                          Department Genetics, North Carolina State University, Raleigh, North Carolina 27513-7614 <br />                                                        Manuscript received January 6, 2004 <br />                                                       Accepted publication April 7, 2004 <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The Epidermal growth factor receptor essential gene diverse pleiotropic roles development <br />               animal kingdom. Analysis sequence diversity 10.9 kb covering complete coding <br />               region 6.4 kb potential regulatory regions sample 250 alleles populations of <br />               Drosophila melanogaster suggests intensity different population genetic forces varies the <br />               locus. A total 238 independent common SNPs 20 indel polymorphisms detected, just <br />               common replacements affecting &#226;&#172;&#381;1475 amino acids, short alternate exon. <br />               Sequence diversity lowest 2-kb portion intron 2, highly conserved comparison <br />               D. simulans D. pseudoobscura. Linkage disequilibrium decays background levels 500 bp <br />               sites, haplotypes generally restricted 5 polymorphisms. The North American <br />               samples North Carolina California diverged allele frequency handful individual <br />               SNPs, Kenyan sample divergent polymorphic. The effect sample size on <br />               inference roles population structure, uneven recombination, weak selection patterning <br />               nucleotide variation locus discussed. <br />  <br />  <br />  <br /> T    HE Epidermal growth factor receptor (Egfr) Drosoph- <br />      ila involved quite essential wide range <br /> of activities, little indication ever <br />                                                                            primarily quantitative genetic questions, requiring <br />                                                                            confidence estimates level and <br />                                                                            variance linkage disequilibrium&#226;&#8364;&#8221;and haplo- <br /> had central role promoting evolutionary change.                       type structure&#226;&#8364;&#8221;throughout locus, assessment <br /> Unlike Hox family, example, widely                        presence absence population structure. As <br /> recognized key mediator morphological diversi-                     shown accompanying article (Palsson Gib- <br /> fication (Gellon McGinnis 1998), numerous                           son 2004, accompanying article issue), turns <br /> enzymes central metabolism facilitate ecological                   complete sequence coverage selec- <br /> adaptation (Eanes 1999), Egfr obviously re-                        tive genotyping critical detection genotype- <br /> garded generic signaling component likely                        <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span> associations. We argue some <br /> helps promote developmental stability. In ab-                       purposes sample sizes alleles are <br /> sence compelling implication involvement                     likely yield information molecular evolu- <br /> unusual phenomenon, sort gene escapes                    tionists gleaned smaller samples. In <br /> the notice evolutionary geneticists. Yet extent                  particular, significance testing population structure, <br /> that quantitative variation product subtle poly-                 inference recombination rates vary a <br /> morphisms hundreds loci, arguably                     locus, monitoring frequency rare and <br /> sort gene receive attention.                                complex alleles depend larger sample sizes. <br />    Point estimation basic parameters nucleotide                     We chose Egfr thousands typical loci for <br /> variation generally recognized unaffected                      main reasons. First, highly conserved roles in <br /> sample size, tests neutrality based                    cell growth differentiation animal <br /> these estimates low power detect weak selection,                   kingdom (Freeman 1998; Stein Staros 2000; <br /> most population genetic surveys deal samples                    Shilo 2003), intrinsically interesting terms of <br /> more 30 alleles. However, study motivated                     understanding genetics developmental pathways. <br />                                                                            Second, excellent candidate gene quantitative <br />                                                                            association range traits Drosophila (Dwor- <br />   1 <br />    These authors contributed equally work. <br />                                                                            kin et al. 2003; Palsson Gibson 2004, accompa- <br />   2 <br />    Present address: Department Ecology Evolution, University        nying article issue) pleiotropic <br /> of Chicago, Chicago, IL 60637.                                             functions (e.g., Clifford Schu&#194;&#168;pbach 1994; Wang <br />   3 <br />    Present address: Department Biology, Duke University, Durham,        et al. 2000; Duchek Rorth 2001; Galindo et al. <br /> NC 27708. <br />   4 <br />                                                                            2002; Johnson-Hamlet Perkins 2002; Zecca and <br />    Corresponding author: Department Genetics, Gardner Hall, North <br /> Carolina State University, Raleigh, NC 27695-7614.                         Struhl 2002; Yang Baker 2003) easy <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> E-mail: ggibson@unity.ncsu.edu                                             imagine little variation tolerated in <br /> Genetics 167: 1199&#226;&#8364;&#8220;1212 ( July 2004) <br />  1200                                                       A. Palsson et al. <br />  <br /> the protein structure, case. These                 limited success, consequently depth coverage varies <br /> different functions set conflicting trade-              locus. No attempt sequence frag- <br />                                                                       ments strands, trade-off time expense <br /> offs different stages development, linkage dis-             gain accuracy. However, estimate <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span>-calling <br /> equilibrium regulatory sites affecting different              error rate &#226;&#172;?0.1% polymorphic site basis repeat <br /> domains expression impact polymorphism,                      sequencing alleles 100-bp overlaps between <br /> perhaps resulting strong balancing selection. Similar              ends sequence fragments resulted &#226;&#181;&#8216;1.5&#226;&#171;&#187; coverage <br /> to downstream signaling components (Riley et al.                      18% nucleotides represented traces. <br />                                                                       There slightly &#226;&#172;&#381;2 Mb completed sequence, just four <br /> 2003), evidence latter, discuss            unresolved polymorphisms remain manual inspection <br /> some unexpected haplotype structure population                    comparison pairs traces. <br /> differentiation context weak selection, uneven                 Nevertheless, genotype matrix entire collection <br /> recombination, predominance purifying selec-                 257 partial Egfr alleles 74% complete, average <br /> tion.                                                                 read <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 8067 (&#226;&#171;&#190;131) bp. The main reasons for <br />                                                                       incomplete coverage (i) reduced sampling problematic <br />                                                                       regions, notably exon 2 CA lines; (ii) <br />                                                                       lingering heterozygosity portions alleles, primarily <br />                MATERIALS AND METHODS                                  NC lines; (iii) reduced finishing effort the <br />    Drosophila stocks: Thirty-six Kenyan lines established        Kenyan lines, relation <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> stocks obtained Bowling Green Stock Center              genotype-phenotype mapping studies. Consequently <br /> in 1997, based flies collected R. Woodruff 1970s.        molecular evolutionary conclusions interpreted <br /> Second chromosomes lines isogenized pas-             caveats possible sequence error sampling <br /> sage CyO balancer Samarkand background.               bias mind. Locations variants specified relative to <br /> Several cases probable gene conversion CyO allele           GenBank entry 17571116 (also available supplementary <br /> were observed sequencing, excluded                information http://statgen.ncsu.edu/ggibson/SupplInfo/ <br /> from analysis. Eighty-three Californian isofemale lines       SupplInfo6.htm), spans 48 kb encompassing com- <br /> were collected S. Nuzhdin Wolfskill orchard near          plete Egfr locus 5 kb end. Coverage of <br /> Winters, California (CA), 1998, subjected &#226;&#172;&#381;40 gener-        predicted coding region sequence 100%; the <br /> ations pairwise sib mating generate near-isogenic           extent Egfr regulatory sequences unknown, least <br /> lines surveyed (Yang Nuzhdin 2003). Similarly, 150           one-third 24-kb intron 1 occupied open <br /> North Carolinian near-isogenic lines established            reading frames (ORFs), flanking genes closely <br /> isofemales trapped bins peach orchard         adjacent, coverage entire locus lies 25 <br /> near West End, North Carolina (NC), July 2000. After 10            75%. <br /> generations sib mating 70% Egfr loci             All polymorphisms aligned manually inspected by <br /> lines homozygous, sib mating continued multi-          observers, unambiguous SNPs reported. <br /> ple sublines 5 generations, homozygous lines        Trace files imported ContigExpress module <br /> were chosen basis sequencing. A total 130           VectorNTI Version 5 (Informax) primary editing and <br /> NC near-isogenic lines survived, 15 remained        assembly contiguous alleles region overlap- <br /> partially heterozygous (cf. 2 CA lines), case         ping sequence reads. Sequence alignment completed us- <br /> that portion locus ignored. It probable        ing ClustalW (Thompson et al. 1994), matrix alleles <br /> process inbreeding selection homozygous chromo-             transferred Genedoc (Nicholas et al. 1997), which <br /> somes biases sampling, indication departure             facilitates manual adjustment indel polymorphism. Final <br /> from Hardy-Weinberg proportions observed data              sequences available GenBank: D. melanogaster alleles <br /> and bias data set, excess     divided exons populations follows: exon 1 [CA, <br /> rare alleles. The Drosophila simulans allele obtained        AY460697&#226;&#8364;&#8220;AY460774; Kenya (K), AY460775&#226;&#8364;&#8220;AY460801; NC, <br /> the population West End flies. Flies maintained          AY460802&#226;&#8364;&#8220;AY460927], exon 2 (CA, AY460928&#226;&#8364;&#8220;AY460979; K, <br /> in vials 10 ml standard cornmeal medium supplemented               AY460980&#226;&#8364;&#8220;AY461000; NC, AY461001&#226;&#8364;&#8220;AY461101), exons <br /> with yeast.                                                           3&#226;&#8364;&#8220;6 (CA, AY461102&#226;&#8364;&#8220;AY461184; K, AY461185&#226;&#8364;&#8220;AY461220; NC, <br />    Sequencing: All single-nucleotide polymorphism (SNP)           AY461221&#226;&#8364;&#8220;AY461357). Outgroup sequences D. sim- <br /> insertion/deletion (indel) genotypes obtained direct          ulans, exons 1 (AY460690&#226;&#8364;&#8220;AY460692), 2 (AY460693), and <br /> sequencing PCR products obtained squish-preps            3&#226;&#8364;&#8220;6 (AY460694), addition D. sechellia allele exon <br /> single male fly line. The 10.9 kb surveyed 6             1 (AY460689). <br /> amplification reactions, ranging 1.2 2.1 kb length,           Analysis nucleotide diversity: Since Kenyan sample <br /> and 17 sequencing reactions generated external PCR           large number unconfirmed singletons less <br /> primers internal primer (listed supplementary Table         one-fifth size North American sample, all <br /> 1 http://statgen.ncsu.edu/ggibson/supplinfo/supplinfo6.            analyses aside FST comparison based com- <br /> htm) fragments, diagrammed Figure 1.            bined NC CA sample. Nucleotide diversity initially <br /> Amplification reactions performed Promega (Madi-            analyzed pairwise distance alleles (&#226;?&#178;) on <br /> son, WI) Taq polymerase, products purified agarose          basis number segregating sites (&#226;?&#170;) DnaSP <br /> gels QIAquick columns (QIAGEN, Valencia, CA),                Version 3.53 (Rozas Rozas 1999) Tassel Version 2.0 <br /> sequencing performed Big Dye II mix (Perkin-Elmer,           (E. Buckler, http://www.maizegenetics.org). Inconsistencies <br /> Norwalk, CT) ABI 3700 sequencers North Carolina             estimates exons 1 2 arose result <br /> State University Genome Research Laboratory. Most high-               way programs handle missing data, report <br /> quality reads range 650&#226;&#8364;&#8220;700 bases,            direct manual estimates. Nucleotide diversity, &#226;?&#178;, was <br /> regions proved particularly refractory amplification          estimated per-nucleotide expected heterozygosity a <br /> and sequencing, mainly high SNP indel polymorph-           sliding window 100 bp Figure 2A functional <br /> ism homopolymeric runs. New primer combinations                region gene Table 1. The estimates &#226;?&#170; nucleotide <br /> were tested, particularly vicinity exons 1 2,      Table 1 obtained S/nai, S observed <br />                                                   Drosophila EGFR Sequence Diversity                                           1201 <br />  <br />                                                                                 Figure 1.&#226;&#8364;&#8221;Structure Egfr locus se- <br />                                                                               quencing strategy. The exons indicated as <br />                                                                               shaded boxes, open boxes preceding exons <br />                                                                               1 2 representing putative signal peptides. Al- <br />                                                                               ternative splicing results transcripts, RA <br />                                                                               RB, alternate exons joined <br />                                                                               common 3&#226;&#172;&#732; exons. The introns <br />                                                                               24.4 6.9 kb length, respectively, and <br />                                                                               extent unsequenced DNA (including three <br />                                                                               ORFs opposite strand, shown) indi- <br />                                                                               cated kilobases. Three noteworthy sequence <br /> features mentioned text CAA microsatellite (open diamond) unusual CN repeat (solid diamond) upstream <br /> of exon 2, conserved 30-bp motif intron 2 (open triangle). A short pogo element insertion NC <br /> alleles, 253 bp upstream start codon (solid triangle). The sequencing strategy shown gene resulted three <br /> contigs assembled PCR fragments 17 sequencing reactions, half-arrows indicating direction sequence <br /> extension. <br />  <br />  <br /> number segregating sites, ai sum &#226;&#171;&#189; 1         goodness-of-fit test (Schaeffer 2002). The effect sample <br /> N &#226;&#171;&#186; 1 1/i N, mean number alleles segment        size test variance indel size also <br /> of length n nucleotides (Watterson 1975). These esti-            estimated 1000 jackknife iterations, sample size of <br /> mates track closely tendency &#226;?&#170; &#226;&#172;&#381; &#226;?&#178;,         122. The shape microsatellite length distribution was <br /> this significant region entire locus.    studied fitting normal distribution assessing devia- <br /> Tests neutrality (Hudson et al. 1987; Tajima 1989; Fu         tions uniformity Kolmogorov-Smirnov test. Skewness <br /> Li 1993; Fay Wu 2000) implemented DnaSP                  kurtosis observed distributions compared <br /> Version 3.53 reduced data sets containing complete           estimates derived permutations programmed R, <br /> sequences sliding windows and/or specific segments            individual repeats scrambled respect another. <br /> the locus, did provide strong evidence either <br /> directional balancing selection: Palsson (2003) provides <br /> a detailed description. The effects sample size tests                                RESULTS <br /> neutrality determined 100 jackknife iterations with <br /> N &#226;&#171;&#189; 12, 48, 96, regions &#226;&#181;&#8216;1000 bp              Parameters molecular variation Egfr: Almost <br /> had missing data. Protein divergence (McDonald                10.9 kb Egfr sequenced segments 1250, <br /> Kreitman 1991) assessed Fisher&#226;&#8364;&#8482;s exact test. Align-         2090, 7523 nucleotides, corresponding, respec- <br /> ment D. pseudoobscura homolog (raw contig 1071 <br /> downloaded http://www.hgsc.bcm.tmc.edu) ana-                tively, DNA including second alternate <br /> lyzed AVID visualized VISTA (Mayor et al.              5&#226;&#172;&#732; exons cluster 3&#226;&#172;&#732; exons encode the <br /> 2000).                                                               bulk protein. A schematic gene structure is <br />     Linkage disequilibrium (LD) assessed Tassel, using        shown Figure 1, parameters summarizing the <br /> Fisher&#226;&#8364;&#8482;s exact test (Lewontin 1988; Weir 1996) assess         polymorphism provided Table 1. A total 523 <br /> significance squared correlation allele frequencies, <br /> r 2. All analyses performed NC CA separately            biallelic SNPs, 24 triallelic SNPs, 70 indels were <br /> and combined data set, results reported         detected, including 22 amino acid replacement poly- <br /> combined North American data set qualitative differ-        morphisms. A Hudson-Kreitman-Aguade&#194;&#180; test in- <br /> ences patterns LD populations detected.             tronic vs. exonic sequences significant (P &#226;&#171;&#189; <br /> Only sites present 50 alleles included    0.89). Sequence diversity estimated basis the <br /> in analyses, variants observed 5 <br /> or alleles. The second criterion critical limita-     number segregating sites (&#226;?&#170;) average nucleo- <br /> tions contingency tests small marginal counts (Upton         tide heterozygosity (&#226;?&#178;) hovers 0.01 substitutions <br /> 1982), demonstrated effects estimates link-      nucleotide, average 1 SNP/20 <br /> age disequilibrium (Lewontin 1995). The effects sample            bp. This normal range D. melanogas- <br /> size, minimum number alleles, rare variants ex-          ter genes (Moriyama Powell 1996). The estimates <br /> plored jackknife procedures Tassel (with missing data) <br /> and sample size DnaSP 3.53 (without missing         &#226;?&#170; &#226;?&#178; vary locus, differ signifi- <br /> data).                                                               cantly region, suggesting that <br />     Population differentiation populations       broad approximation Egfr evolving according <br /> estimated using AMOVA procedure Arlequin Version              expectations neutral theory subject varying <br /> 2.0 (Schneider et al. 2000) pairs populations. The            levels constraint regulatory, exonic, intronic <br /> significance FST parameters assessed 10,000 permuta- <br /> tions Bonferroni adjustment 258 comparisons             sequences. Sample size &#226;&#172;&#381;20 sequences little effect <br /> involving common SNPs indels populations.           point estimation &#226;?&#178;, &#226;?&#170;, haplotype diversity, but <br /> Initial analyses performed sliding windows haplo-       various tests neutrality heterogeneous re- <br /> types 10 5 sites, subsequently site separately.   sults smaller samples, indicated supplemen- <br /> Demonstration power gained larger samples            tary Table 3 http://statgen.ncsu.edu/ggibson/Suppl <br /> demonstrated 1000 jackknife samples most <br /> population-stratified sites.                                         Info/SupplInfo6.htm. <br />     Homogeneity distribution indel events           There low level amino acid replacement poly- <br /> locus, regions defined Table 1, analyzed            morphism Drosophila EGF receptor (DER) pro- <br />  1202                                                     A. Palsson et al. <br />  <br />                                                             TABLE 1 <br />                                   Parameters nucleotide diversity Drosophila EGFR <br />  <br />                                                   Segregating polymorphisms                                      cf. D. simulans <br /> Region        Length (bp)      Start   Het b      &#226;?&#170;c         Stotd    Scomm    Repl (rare)e   Indel f   Silent      Repl     Indel <br /> 5&#226;&#172;&#732; exon 1           605        5402      0.015    0.013        46         23         &#226;&#8364;&#8221;             7        13         &#226;&#8364;&#8221;            7 <br /> Exon 1              154        6016      0.009    0.008         7          5        6 (2)          1         0          6           1 <br /> Intron 1            492        6170      0.007    0.008        21          9         &#226;&#8364;&#8221;             7       ND          &#226;&#8364;&#8221;           ND <br /> 5&#226;&#172;&#732; exon 2           416       30120      0.006    0.010        23          6         &#226;&#8364;&#8221;             9        13         &#226;&#8364;&#8221;            1 <br /> Exon 2              295       30518      0.008    0.013        20          6        3 (3)          0         5          0           0 <br /> 3&#226;&#172;&#732;exon 2          1,384       30819      0.012    0.011        82         42         &#226;&#8364;&#8221;            16        27         &#226;&#8364;&#8221;            8 <br /> Intron 2          2,425       35340      0.004    0.008       112         27         &#226;&#8364;&#8221;            13        25         &#226;&#8364;&#8221;            2 <br /> Exon 3              223       37757      0.015    0.010        13         10        0 (0)          0         6          0           0 <br /> Intron 3            170       37980      0.052    0.037        30         20         &#226;&#8364;&#8221;             5        10         &#226;&#8364;&#8221;            1 <br /> Exon 4            1,175       38116      0.010    0.008        54         32        1(1)           0        22          0           0 <br /> Intron 4             66       39291      0.012    0.015         6          2         &#226;&#8364;&#8221;             3         6         &#226;&#8364;&#8221;            0 <br /> Exon 5              133       39358      0.017    0.016        12          5        2 (1)          0         0          2           0 <br /> Intron 5             74       39491      0.009    0.012         5          1         &#226;&#8364;&#8221;             1         7         &#226;&#8364;&#8221;            1 <br /> Exon 6            2,446       39561      0.008    0.007        95         56        8 (8)          1        44          2           0 <br /> 3&#226;&#172;&#732; UTR              347       42010      0.010    0.010        20         11         &#226;&#8364;&#8221;             2         6         &#226;&#8364;&#8221;            1 <br /> Intergenic          455       42355      0.003    0.005        13          4         &#226;&#8364;&#8221;             0         9         &#226;&#8364;&#8221;            0 <br /> Total            10,863                  0.009    0.009       546        259       20 (15)        65       193         10          22 <br />   a <br />     The number base region GenBank Drosophila accession NG_000184 (17571116). Segment lengths do <br /> not necessarily match perfectly sequence lengths indel variation sequences. <br />   b <br />     Het (heterozygosity) estimated average expected nucleotide heterozygosity Hardy-Weinberg proportions. <br />   c <br />     &#226;?&#170; nucleotide, estimated Stot divided sum &#226;&#171;&#189; 1 n 1/n, n, average number alleles the <br /> region, number nucleotides sequence segment. <br />   d <br />     Stot total number segregating SNPs, Scomm number common SNPs (less common allele &#226;&#172;&#381;5%). <br />   e <br />     Repl (rare) number replacement (or rare replacement) polymorphisms. <br />   f <br />     Indel information North American sample only. <br />  <br />  <br />  <br /> tein sequence. Only &#226;&#8364;&#339;common&#226;&#8364;? replacement poly-                     trasted divergence short (Figure 2B, D. sim- <br /> morphisms (where alleles frequencies &#226;&#172;&#381;0.05)                  ulans, &#226;&#181;&#8216;2.5 million years) long (Figure 2C, D. pseudo- <br /> are observed, located                 obscura, 45 million years) timescales (Powell 1996). In <br /> putative signal peptide region alternate exon 1. All                paragraphs, discuss parameters of <br /> of remaining 15 nonsynonymous SNPs present                     variation alternate 5&#226;&#172;&#732; exons, intron 2, and <br /> in just 1, 2, 4 210 sampled North American                   main coding region. <br /> sequences listed supplementary Table 2 (http://                     Exon 1 remarkable considerably more <br /> statgen.ncsu.edu/ggibson/SupplInfo/SupplInfo6.htm),                    polymorphic alternate exon 2 protein <br /> and half affect amino acids conserved                level. It encodes far polymorphic 51-amino- <br /> in D. pseudoobscura DER sequence. An exonic 9-bp                   acid segment DER protein, six <br /> deletion polymorphism replaces conserved Pro-                   replacements relative D. simulans allele compared <br /> Asn3 His exon 6 Californian allele likely              just remaining 1424 residues. Noncod- <br /> to deleterious. Two North Carolinian alleles                 ing variation slightly higher level ob- <br /> appear instead initiating methionine                served locus, 15 indel <br /> codons alternate exon 1, polarization                  variants just 1.1 kb flanking exon. Oddly, there <br /> sibling species indicates derived allele               synonymous substitutions relative D. simulans, <br /> insertion 3 bp approaching fixation D.                   sufficient provide evidence positive <br /> melanogaster. The ratio replacement synonymous                   selection McDonald-Kreitman test (McDonald <br /> substitutions entire coding region compar-              Kreitman 1991; two-tailed P &#226;&#171;&#189; 0.068, Fisher&#226;&#8364;&#8482;s exact <br /> ison D. simulans allele 0.05, normal                test). Comparison D. pseudoobscura indicates <br /> for Drosophila, generally taken sign strong             relaxed constraint: unlike remainder locus, <br /> functional constraint protein (Aquadro et al.                   short stretches &#226;&#172;&#381;50% nucleotide identity <br /> 2001).                                                                 seen, apart putative start site splice <br />    Each region Egfr tells slightly different story con-           donor sites, alignment impossible. <br /> cerning possible evolutionary forces acting                    Exon 2 characterized unusually low level of <br /> gene, intraspecific variation (Figure 2A) con-                 haplotype structure slight nonsignificant <br />                                              Drosophila EGFR Sequence Diversity                                          1203 <br />  <br />  <br />  <br />                                                                                     Figure 2.&#226;&#8364;&#8221;Polymorphism diver- <br />                                                                                  gence. (A) Sequence diversity D. mel- <br />                                                                                  anogaster North American popula- <br />                                                                                  tions plotted heterozygosity per <br />                                                                                  nucleotide 100-bp sliding window from <br />                                                                                  5&#226;&#172;&#732; 3&#226;&#172;&#732; breaks contigs. <br />                                                                                  (B) Sequence divergence relative D. <br />                                                                                  simulans allele sampled NC collec- <br />                                                                                  tion site: note peaks divergence in <br />                                                                                  introns 3, 4, 5, intron 3 is <br />                                                                                  highly polymorphic D. melanogaster. <br />                                                                                  (C) VISTA plot sequence conservation <br />                                                                                  (percentage nucleotide identity) relative <br />                                                                                  D. pseudoobscura, 100-bp stretches <br />                                                                                  &#226;&#172;&#381;75% identical shaded. Solid bars the <br />                                                                                  location exons. Note peaks <br />                                                                                  conservation correspond broadly with <br />                                                                                  conserved regions D. melanogaster <br />                                                                                  group, exon 1 poorly <br />                                                                                  conserved. <br />  <br />  <br />  <br />  <br /> excess rare segregating sites nucleotide (see Table     ulans allele. Although cis-acting regulatory regions <br /> 1). This <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> extends upstream coding region,        mapped Egfr locus, phylo- <br /> but 1.4 kb downstream exon conforms            genetic intraspecific shadowing data Figure 2 <br /> to usual pattern similar numbers rare            strongly suggest intron 2 candidate regulatory re- <br /> common polymorphisms. Another intriguing feature as-           gion. <br /> sociated exon 2 79-bp element 350 bp upstream           Exons 3&#226;&#8364;&#8220;6 encode bulk DER protein, are <br /> of translation start site. The element consists     highly conserved species, low levels <br /> sets CN dinucleotide repeats arranged (CN)15X21          polymorphism D. melanogaster. Only 4 1325 <br /> (CN)13 conserved D. pseudoobscura           amino acids region differ D. simulans, and <br /> with reduced spacing sets repeats. Ten          segregating common replacement <br /> of sites different D. simulans, 9 change   polymorphisms. The introns 170, 66, 74 <br /> the N nucleotide 28 repeats. Of            bp 4.3 kb sequence covering exons <br /> segregating sites D. melanogaster, common,      dramatically elevated divergence relative the <br /> and affects C associated variation        exons species, seen Figure 2, <br /> wing shape (Palsson Gibson 2004, accompanying              B C, increase intraspecific polymor- <br /> article). These data strongly suggest element      phism observed smaller introns (Figure <br /> part motif affects Egfr transcription Drosoph-    2A). Despite unusual features, formal <br /> ila. In addition, complex CAA microsatellite described       support departure neutrality, because <br /> in discussion located 250 bp upstream            intron sequences short. Intron 3 particularly <br /> translation start site exon 2.                             polymorphic, 1 6 nucleotides harboring a <br />    The 2.4-kb portion intron 2 immediately preceding        polymorphism. The ends intron, 10 bases <br /> exon 3 overall polymorphic region          respective splice sites, different recom- <br /> entire Egfr locus includes stretches      bination histories. A remarkable 3-base stretch before <br /> 100 bp monomorphic D. melanogaster          start exon 4 polymorphisms rare <br /> and &#226;&#172;&#381;50% nucleotide identity D. pseudoobscura.         allele frequencies 0.20, 0.34, 0.50 are <br /> Most 102 segregating sites region rare      linkage equilibrium another, whereas <br /> (only 26 common sites observed compared 57 ex-              substitutions frequency 0.04 7-base <br /> pected), similar exon 2, little haplotype      stretch following exon 3 perfect linkage disequi- <br /> structure. By contrast sequences flanking exons       librium. Sequence conservation extends 1 kb <br /> 1 2, contain 22 common deletions            3&#226;&#172;&#732; termination codon, 3&#226;&#172;&#732;-untranslated <br /> half &#226;&#172;&#381;4 bp, 2 long deletions present just          region (UTR) intergenic region. <br /> three alleles upstream exon 3. Similarly,            Linkage disequilibrium: Regional variation ex- <br /> only short deletions seen relative D. sim-      tent haplotype block structure locus is <br />  1204                                                 A. Palsson et al. <br />  <br />  <br />  <br />  <br />                                                                                                   Figure 3.&#226;&#8364;&#8221;Linkage dis- <br />                                                                                                equilibrium plot. The ex- <br />                                                                                                tent (D&#226;&#172;&#732;) significance <br />                                                                                                (P-value) LD plotted <br />                                                                                                pairwise compari- <br />                                                                                                son 114 common SNPs <br />                                                                                                (more rare allele frequency <br />                                                                                                &#226;&#172;&#381;0.25, sampled least <br />                                                                                                50 alleles) 5&#226;&#172;&#732; 3&#226;&#172;&#732; (top <br />                                                                                                left right). <br />                                                                                                See supplementary informa- <br />                                                                                                tion http://statgen.ncsu. <br />                                                                                                edu/ggibson/SupplInfo/ <br />                                                                                                SupplInfo6.htm color <br />                                                                                                version larger set of <br />                                                                                                238 SNPs rare allele <br />                                                                                                &#226;&#172;&#381;0.05. Approximate loca- <br />                                                                                                tion exons variation <br />                                                                                                SNP density indicated <br />                                                                                                plot. Three broad <br />                                                                                                regions LD evident, <br />                                                                                                particularly signifi- <br />                                                                                                cance plot, corresponding <br />                                                                                                sequence con- <br />                                                                                                tigs, consider- <br />                                                                                                ably reduced LD around <br />                                                                                                exon 2. The large block of <br />                                                                                                LD center plot <br />                                                                                                corresponds highly poly- <br />                                                                                                morphic 400-bp region in <br />                                                                                                exon 3 intron 3. <br />  <br />  <br />  <br />  <br /> shown Figure 3, plot degree (above              populations, proved quite similar Pear- <br /> diagonal, D&#226;&#172;&#732;) significance (below diagonal)          son correlation populations 0.86 <br /> LD sites gene. The plot            r 2 0.54 D&#226;&#172;&#732;. Table 2 shows power to <br /> combined NC CA data consisting partial se-               detect pairwise associations significant experi- <br /> quence 210 lines clarity includes SNPs        ment wide rises sample size, depends criti- <br /> where common allele frequency        cally absolute count rare alleles, argued by <br /> 0.25. Two large blocks strong LD correspond 1.25          Lewontin (1995). <br /> kb exon 1 highly polymorphic                 Linkage disequilibrium generally expected decay <br /> 400-bp stretch encompassing exon 3 intron 3. Both           distance, result recombination breaking up <br /> the exon 2 intron 2 sequence stretches essen-           haplotypes generated new alleles arise are <br /> tially devoid significant LD, 3&#226;&#172;&#732; half       introduced admixture. The plot LD distance <br /> gene including long exons shows <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span> imper-          Figure 4 confirms tends case, <br /> fect LD blocks 1 kb length suggestion            sites 50 bp linkage <br /> of occasional long-range associations. Significance        disequilibrium. LD rarely rises background <br /> function allele frequencies correlation be-          level sites separated &#226;&#172;&#381;1 kb. Note that <br /> tween (r 2) total sample size,      gaps plot points arise there <br /> D&#226;&#172;&#732; shows extent LD relative maximum               unsequenced regions 23.5 3 kb in- <br /> possible association. Even blocks highly          trons 1 2. Some long-range associations ex- <br /> significant LD, adjacent sites rarely complete LD,       pected arise chance 20,100 pairs alleles, <br /> and sites bases             r 2 measures 5 kb probably <br /> can linkage disequilibrium. The depth sampling         sampling artifacts. Over short range, LD main- <br /> enabled comparison LD North Ameri-             tained factors low recombination, gene con- <br />                                                Drosophila EGFR Sequence Diversity                                             1205 <br />  <br />                                                               TABLE 2 <br />                                     Effect sample size LD metrics significance <br />  <br />                                         Jackknifed subsamplesa                                            Observed <br />                 N &#226;&#171;&#189; 64, X &#226;&#171;&#190; (SD)         N &#226;&#171;&#189; 128, X &#226;&#171;&#190; (SD)        N &#226;&#171;&#189; 255, X &#226;&#171;&#190; (SD)              A           B              C <br /> Tests            5797.4 (1949.4)           5649.1 (622.7)          5662.5 (29.4)           5671.0        5460.0         13530 <br /> Average r 2         0.068 (0.006)             0.062 (0.004)           0.058 (0)               0.058         0.056           0.040 <br /> Average D &#226;&#172;&#732;         0.388 (0.096)             0.394 (0.039)           0.400 (0.001)           0.400         0.252           0.464 <br />  <br /> &#226;&#172;&#381;0.05            76.4%   (0.026%)          62.2%   (0.024%)        47.3%   (0.002%)         47.3%          61.4%          63.2% <br /> &#226;&#172;?0.05            23.6%   (0.026%)          37.8%   (0.024%)        52.7%   (0.002%)         52.7%          38.6%          36.8% <br /> &#226;&#172;?0.01            13.3%   (0.017%)          24.5%   (0.019%)        39.2%   (0.002%)         39.4%          27.7%          24.8% <br /> &#226;&#172;?0.001            7.4%   (0.01%)           15.3%   (0.014%)        27.8%   (0.002%)         27.9%          21.0%          16.3% <br /> &#226;&#172;?0.0001           4.9%   (0.009%)          10.7%   (0.009%)        20.5%   (0.001%)         20.6%          16.8%          11.8% <br /> &#226;&#172;?0.00001          3.5%   (0.008%)           8.2%   (0.007%)        16.1%   (0.001%)         16.2%          14.1%           9.0% <br /> &#226;&#172;?0.000001*        2.4%   (0.006%)           6.6%   (0.005%)        13.0%   (0.001%)         13.1%          12.4%           7.0% <br /> &#226;&#172;?0.0000001        1.7%   (0.005%)           5.4%   (0.005%)        10.5%   (0.001%)         10.5%          10.8%           5.5% <br />   *The Bonferroni significance threshold 0.000001 5000 tests. <br />   a <br />     Based 25 random subsamples allele matrix. Sites extracted basis minimum number of <br /> polymorphic alleles site absolute count common allele. For N &#226;&#171;&#189; 64, sites represented 50 or <br /> more alleles included, rare variant present 5 alleles. These numbers 100 10 for <br /> N &#226;&#171;&#189; 128, 200 20 N &#226;&#171;&#189; 255. The observed sets were: A, 200, 20; B, 150, 20; C, 200, 5. <br />  <br />  <br /> version, epistatic selection, comprehensive              North American populations abscissa, <br /> analysis tendencies study.                 compared smaller Kenyan <br />    Another way represent haplotype structure              sample it. Bars plot highlight regions <br /> plot actual alleles haplotype networks,         significance contrasts bars <br /> for representative sequence segments Figure 5.             representing P &#226;&#172;? 0.0001 bar 0.01 &#226;&#172;? P &#226;&#172;? 0.05. <br /> On left, row represents SNP, alleles              For comparison North Carolina California, <br /> are columns &#226;&#181;&#8216;0.5 mm wide, light shading indicates              window shows large FST significant after <br /> the rare genotype. In networks right, circle            Bonferroni correction multiple comparisons (P &#226;&#172;? <br /> diameters proportional number alleles             0.0002), regarded marginally <br /> a common haplotype, joined lines                  significant best. Analysis single sites summarized <br /> with lengths proportional number substitutions           Table 3 indicates cases the <br /> distinguishing haplotypes. The short segments             population structure focused single SNP around <br /> exons 1 5 examples true haplotype blocks,               linkage disequilibrium decays rapidly, excep- <br /> with just handful alleles case having arisen          tion trio sites 40428, 40458, 40464 in <br /> by recombination. Exon 5 provides example                  exon 6. <br /> dimorphic haplotype, common type is <br /> four substitutions distinct two <br /> common types. The 5&#226;&#172;&#732; end intron 2 contrast shows <br /> several haplotypes recombination stretch of <br /> 250 bp. Haplotype dimorphism observed short <br /> stretches exon 3 intron 3, deep branches <br /> of substitutions separating major <br /> types case, recombination and/or gene con- <br /> version generated networks haplotypes that <br /> cannot generated solely stepwise <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span>. Both <br /> of blocks present instances linkage equilib- <br /> rium adjacent sites interspersed with <br /> other sites LD. <br />    Population structure: Population differentiation was <br /> investigated using Wright&#226;&#8364;&#8482;s estimator proportion              Figure 4.&#226;&#8364;&#8221;Rate decay LD distance the <br /> of variance allele frequencies attributable           locus. Each spot shows squared correlation coefficient (r 2) <br />                                                                    pair sites separated indicated distance. <br /> more subpopulations relative total population,              Gaps arise result large gaps 23.5 3.0 kb <br /> FST (Weir Hill 2002). Figure 6 shows FST sliding          contigs. The 20,100 comparisons derived from <br /> window 10 polymorphic sites locus             201 alleles rare allele frequency &#226;&#172;&#381;0.1. <br />  1206                                               A. Palsson et al. <br />  <br />  <br />  <br />  <br />                                                                                Figure 5.&#226;&#8364;&#8221;Five representative haplotype <br />                                                                             blocks. Blocks left alignment of <br />                                                                             indicated number SNPs indicated <br />                                                                             stretch DNA: solid, common allele; <br />                                                                             shaded, common allele. Numbers of <br />                                                                             alleles differ incomplete sequence cov- <br />                                                                             erage. Networks right drawn by <br />                                                                             hand represent number alleles of <br />                                                                             haplotype class sample, the <br />                                                                             number SNPs separate (small bars <br />                                                                             crossing lines joining haplotypes). Haplotypes <br />                                                                             arisen recombination re- <br />                                                                             current mutation connected the <br />                                                                             remainder network. <br />  <br />  <br />  <br />  <br />    Divergence pronounced             larger North Carolina sample shares 4 <br /> North American Kenyan samples, maximum             sites Kenyans. <br /> FST 0.4. There tendency elevated divergence <br /> associated exon 2, intron 2, exon 6, is <br />                                                                                     DISCUSSION <br /> perhaps noteworthy regions are <br /> among polymorphic sample. Population            Our survey 20 times larger molecular <br /> structure seen comparison private       evolutionary studies, focus 2 3 kb 30 or <br /> allele frequencies, alleles observed       alleles. Increasing sample size &#226;&#172;&#381;200 alleles has <br /> only populations (Slatkin 1985). The Ken-          little effect point estimation parameters of <br /> yan sample 92 private alleles, times      nucleotide variation, molecular evolutionary <br /> many Californian sample,         studies necessary. For reason, Pluzhnikov <br /> half times larger. The North American samples share           Donelly (1996) concluded sequencing effort <br /> 50 sites observed Kenya, 14      invested length, number, alleles. <br /> are common polymorphisms sense           However, consistent Simonsen et al.&#226;&#8364;&#8482;s (1995) simu- <br /> common allele frequency &#226;&#172;&#381;0.05. Moreover,            lation study power Tajima&#226;&#8364;&#8482;s D detect selective <br /> Californian sample shares 14 sites Kenyans           sweeps, large samples decrease variance <br />  <br />  <br />  <br />  <br />                                                                                            Figure 6.&#226;&#8364;&#8221;Population differenti- <br />                                                                                         ation Egfr. (Top) Plots FST <br />                                                                                         sliding window 10 SNPs rela- <br />                                                                                         tive position locus for <br />                                                                                         NC vs. Kenya (dotted line) CA <br />                                                                                         vs. Kenya (solid line abscissa) <br />                                                                                         NC vs. CA (below abscissa). <br />                                                                                         (Bottom) Significance FST con- <br />                                                                                         trasts shown columns respec- <br />                                                                                         tive contrasts (one bar, P &#226;&#172;? 0.05; two <br />                                                                                         bars, P &#226;&#172;? 0.01; bars, P &#226;&#172;? 0.001; <br />                                                                                         bars, P &#226;&#172;? 0.0001). <br />                                               Drosophila EGFR Sequence Diversity                                                   1207 <br />  <br />                                                           TABLE 3 <br />                Frequencies FST parameters comparison North Carolina California populations <br />  <br />                                                                                           Haplotypesc         Haplotypesc <br />                                        N                  NC CA contrast b               (NC)                (CA) <br />         Site        Variant   NC   CAa    Kenyaa   Window        F ST     P-value      N     Diversity     N     Diversity <br />         35345          A       73     72      NA         1       0.10093    0.0013        2     0.2949        2     0.0526 <br />                        C       16      2      NA        10       0.28514    0.0026        4     0.6570        4     0.5833 <br />         35697          T       64     10      25         1       0.17558    0.0004        2     0.4807        2     0.3945 <br />                        C       43     27       3        10       0.07428    0.0183        6     0.5754        3     0.4917 <br />         36214          G       93     43      30         1       0.13641    0.0003        2     0.2817        2     0.4868 <br />                        A       19     31       3        10       0.0765     0.0001       10     0.4976        5     0.5889 <br />         39010          C       79     67      30         1       0.08844    0.0009        2     0.4652        2     0.2722 <br />                        T       46     13       5        10       0.03822    0.0077        8     0.7818       12     0.8129 <br />         40428 d        G       73     31      20         1       0.07552    0.0032        2     0.4806        2     0.4747 <br />                        A       49     49      16        10       0.06561    0.0026        6     0.5875        5     0.5309 <br />         40464 d        T       54     51      20         1       0.0688     0.0041        2     0.4917        2     0.4622 <br />                        C       70     29      16        10       0.06561    0.0026        6     0.5875        5     0.5309 <br />         42023          A       90     36      23         1       0.0958     0.0016        2     0.4175        2     0.4986 <br />                        G       38     40      12        10       0.02461    0.0181       10     0.8110       12     0.7625 <br />           a <br />             Absolute counts SNP states seven sites populations (NA, site surveyed the <br />         Kenyan sample). <br />           b <br />             FST NC CA, individual sites haplotypes spanning 10 segregating sites corresponding <br />         P-value estimate. <br />           c <br />             Diversity number (N ) haplotypes (alleles) population. <br />           d <br />             Sites 40428 40464 nearly complete LD (D&#226;&#172;&#732; &#226;&#171;&#189; 1, r 2 &#226;&#171;&#189; 0.9, P &#226;&#172;? 0.0001 Fisher&#226;&#8364;&#8482;s exact test). <br />         They separated 38 bp contribute 10-site haplotype. <br />  <br />  <br /> of estimates degree sufficient markedly           suggestion departure neutrality. Of 341 <br /> increase power tests neutrality. We argue          tests, 21 significant P &#226;&#172;? 0.05, also <br /> that detailed analysis large samples select regions         noted exceed conservative <br /> of Drosophila genome provide extra insight               Bonferroni threshold multiple comparisons. Conse- <br /> into evolution population structure, haplotype             quently, marginal evidence purifying selection in <br /> structure, complex insertion-deletion polymor-                3&#226;&#172;&#732; half gene rendered insignificant inclu- <br /> phisms.                                                           sion 5&#226;&#172;&#732; regions analysis. The fact there <br />    The tests neutrality produce nominally            large excess negative test statistics also <br /> significant deviations null hypothesis Fu            imply excess rare alleles, Bustamante et al.&#226;&#8364;&#8482;s <br /> and Li&#226;&#8364;&#8482;s statistics, D* plotted Figure 7       (2002) comparison multiple loci species <br /> 100-nucleotide windows Egfr locus. These sites          implies baseline comparison species spe- <br /> exceed significant thresholds complete sam-          cific negative test statistics common in <br /> ple: jackknife samples &#226;&#172;?100 alleles summarized              Drosophila. What studied carefully the <br /> supplementary Table 3 (http://statgen.ncsu.edu/ggib               variance tests like Tajima&#226;&#8364;&#8482;s D Fu Li&#226;&#8364;&#8482;s D* <br /> son/SupplInfo/SupplInfo6.htm) provide                  loci effect weak selection that <br />  <br />  <br />  <br />                                                                                    Figure 7.&#226;&#8364;&#8221;Weak evidence purifying selec- <br />                                                                                 tion Egfr locus. Plot Fu Li&#226;&#8364;&#8482;s D* statistic <br />                                                                                 (Fu Li 1993) 100-bp sliding windows 25- <br />                                                                                 bp steps locus showing generally nega- <br />                                                                                 tive values, consistent deficit intermedi- <br />                                                                                 ate frequency variants purifying selection. <br />                                                                                 Of 341 windows, 21 significant 0.05 &#226;&#172;&#381; <br />                                                                                 P &#226;&#172;&#381; 0.01, 9 0.01 &#226;&#172;&#381; P &#226;&#172;&#381; 0.001, in <br />                                                                                 cases associated negative values <br />                                                                                 (indicated solid diamonds). The sample size <br />                                                                                 ranges 78 3&#226;&#172;&#732; UTR 162 exon <br />                                                                                 1 promoter. <br />  1208                                                  A. Palsson et al. <br />  <br /> variance. Since Egfr regions nomi-          bimodality random sequence stretches (Teeter et al. <br /> nally significant samples &#226;&#172;?40 alleles, genome-scale        2000), appears putative haplotype <br /> sequencing sample size resolu-         blocks Egfr bimodal. Coalescent theory suggests <br /> tion address molecular evolutionary processes.            deep branches unexpected sample <br />    Haplotype structure: Estimation haplotype struc-           genes, depth prevalence be <br /> ture essential understanding evolutionary        affected variety demographic factors (Slatkin <br /> history locus determining density             Hudson 1991). While present new analyt- <br /> of SNP sampling support genotype-phenotype             ical statistics, hope suggestions variation <br /> association studies. The analytical approaches used          haplotype structure locus inspire more <br /> to infer haplotypes assessment level linkage       theoretical work variance branch lengths un- <br /> disequilibrium (Figures 3 4) phylogenetic anal-          der uneven recombination weak selection. <br /> ysis regions low recombination (Figure 5). Both               An subtle pattern sequence variation <br /> analyses imply haplotypes Egfr generally re-         existence strong linkage disequilibrium be- <br /> stricted fewer common polymorphisms            tween sites 1 kb neverthe- <br /> all lie 500-bp stretch sequence. Beyond            separated common SNPs linkage equi- <br /> 1 kb, linkage disequilibrium drops 10% maxi-            librium. A level heterogeneity LD distance is <br /> mum, sites effectively linkage equi-       anticipated stochastic evolutionary forces like <br /> librium. The examples stronger association            mutation, genetic drift, gene conversion could <br /> due relatively rare alleles chance, given       consequence population history direc- <br /> very large number tests (Pritchard Przeworski             tional epistatic selection. Our analysis hypermor- <br /> 2001). Assessment significance LD area          phic Egfr phenotype eye (Dworkin et al. 2003) <br /> where larger sample sizes dramatic impact,             identified example significant epistatic interac- <br /> the number significant associations increases &#226;&#172;&#381;10-            tion sites 40119 40620 exon 6. Those <br /> fold measurement 128 68 alleles. Com-          sites LD separated common and <br /> parison LD multiple populations emerging           uncoupled SNPs. In case, linkage disequilibrium <br /> crucial aspect human HapMap project,              reduces frequency hyperactive two-site haplo- <br /> results confirm importance large samples. The             type, consistent possibility purifying selec- <br /> documented levels LD consistent surveys        tion contributes maintenance association be- <br /> of variation D. melanogaster (Schaeffer Miller            tween sites. In general, mechanism to <br /> 1993; Richter et al. 1997; Langley et al. 2000; Zurov-           operate, selection pressure greater than <br /> cova Ayala 2002) imply haplotypes                recombination rate (Lewontin 1964), of <br /> the fruitfly orders magnitude shorter          order 10&#226;&#171;&#186;5 events/generation sites sepa- <br /> than observed humans (Reich et al. 2001;                rated 100 bp Drosophila euchromatin. Conse- <br /> Ardlie et al. 2002). Both low level linkage disequi-      quently, weak epistatic selection expected <br /> librium high polymorphism attributed           maintain linkage disequilibrium pairs sites <br /> fact flies outcrossing enormous         base pairs. Several <br /> effective population size millions years (Aquadro         instances departure monotonic decrease within <br /> et al. 2001).                                                    linkage disequilibrium blocks evident Figure <br />    More detailed analysis haplotype blocks, however,      5, greater sampling depth required to <br /> suggests fine scale evolutionary forces acting          ascertain general feature fly <br /> on locus. Several authors remarked           genome determine causes events. <br /> is fact excess linkage disequilibrium non-              Population structure: D. melanogaster human com- <br /> African flies relative theoretical expectations based         mensal species thought spread Af- <br /> on empirical measurement recombination rate               rica past 15,000 years (Powell 1996) con- <br /> and neutral mutation parameter (Schaeffer Miller             ventionally regarded, largely basis allozyme <br /> 1993; Kirby Stephan 1996; Andolfatto                     data, panmictic range. Numerous re- <br /> Przeworski 2000; Wall 2001). Possible biological ex-             cent studies challenged assumption: sev- <br /> planations include pervasive balancing selection              eral populations southern Africa incipient <br /> small selective sweeps, history admixture            reproductive isolation remainder species <br /> species, unequal gene conversion and/or recom-               (Wu et al. 1995; Ting et al. 2001), microsatellite frequen- <br /> bination rates chromosome (Nachman 2002;                 cies European samples geneti- <br /> Wall et al. 2002; Andolfatto Wall 2003). Figure              cally restricted North American ones (Caracristi <br /> 5 provides hint latter, nonoverlapping 250-       Schlo&#194;&#168;tterer 2003), examples clines ge- <br /> bp segments Egfr similar SNP densities clearly           netic variation documented (Berry Kreit- <br /> produce haplotype networks distinct topologies              man 1993; Verelli Eanes 2001; Freydenberg et al. <br /> ranging deep branches network recom-            2003). The common measures population <br /> bined alleles. Similar previous observation            structure differences nucleotide diversity FST <br />                                                Drosophila EGFR Sequence Diversity                                          1209 <br />  <br />                                                           TABLE 4 <br />                    Effects sampling depth significance goodness-of-fit test indel distributions <br />  <br />                                  All indels                       Deletions                        Insertions <br />                        N &#226;&#171;&#189; 20     N &#226;&#171;&#189; 50      N &#226;&#171;&#189; 100   N &#226;&#171;&#189; 20    N &#226;&#171;&#189; 50      N &#226;&#171;&#189; 100    N &#226;&#171;&#189; 20    N &#226;&#171;&#189; 50       N &#226;&#171;&#189; 100 <br />         X               42.35     49.00       54.81     16.40      19.95      23.08      16.25      18.87       21.27 <br />         (SD)            (2.38)    (2.48)      (2.43)    (1.67)     (1.62)     (1.46)     (1.32)     (1.36)      (1.36) <br />         Tests <br />         &#226;&#172;&#381;0.05            2.3%      0.8%         0.0%     3.1%       0.5%        0.0%      2.3%       0.8%        0.0% <br />         &#226;&#172;?0.05           97.7%     99.2%       100.0%    96.9%      99.5%      100.0%     64.0%      60.8%       78.4% <br />         &#226;&#172;?0.01           82.6%     90.0%        94.9%    84.5%      98.8%      100.0%     22.1%      43.6%       70.4% <br />         &#226;&#172;?0.001          35.7%     50.4%        72.7%    78.0%      96.5%       99.8%     15.4%      10.2%        6.7% <br />         &#226;&#172;?0.0001         13.2%     28.8%        35.9%    68.4%      88.5%       93.1%      1.3%       0.0%        0.0% <br />         &#226;&#172;?0.00001         5.1%      3.8%         1.0%    47.3%      66.7%       84.3%      0.0%       0.0%        0.0% <br />         &#226;&#172;?0.000001*       0.2%      0.0%         0.0%    29.0%      52.1%       71.4%      0.0%       0.0%        0.0% <br />         &#226;&#172;?0.0000001       0.0%      0.0%         0.0%    16.9%      41.1%       59.9%      0.0%       0.0%        0.0% <br />            The number indels segregating fraction significant goodness-of-fit tests 1000 jackknifing <br />         samples sizes 20, 50, 100 shown. The observed number indels, deletions, insertions in <br />         noncoding regions 63, 27, 25, respectively (11 classified deletions insertions). <br />         Italic numbers correspond observed significance level North American data set (220 <br />         alleles) respective mutation classes. <br />  <br />  <br />  <br /> statistics. Averaged entire 10.9 kb sequence,       manner sample, quite possible demes <br /> the Kenyan sample diverse Cali-           congregate collection points. We tested random set <br /> fornian North Carolinian samples,              15 SNPs locus them <br /> rare common SNPs. Among alleles shared                  evenly distributed NC CA using the <br /> all populations, Figure 6 provides strong evidence         Structure algorithm Pritchard et al. (2000). How- <br /> for allele frequency differences regions        ever, Figure 6 provides suggestive evidence sev- <br /> gene African American samples. There               eral SNPs having significantly different allele frequen- <br /> no reason suppose differences               cies continent. Further analysis in <br /> simply genetic drift. A surprising finding            Table 3 shows case differentiation <br /> is 50 SNPs seen Africa shared CA           restricted single site haplotype. <br /> and NC samples, implies extensive period              None allele frequency differences nomi- <br /> isolation New World flies source popu-          nally significant comparisons &#226;&#172;?30 alleles from <br /> lation prior spread North America. Our            population (supplementary Table 5 http://stat <br /> data confirm southern African sample               gen.ncsu.edu/ggibson/SupplInfo/SupplInfo6.htm). It <br /> more diverse derived North American ones                     unclear cause change frequency of <br /> (Schlo&#194;&#168;tterer Harr 2002) add layer                40% single site, affecting closely <br /> of evidence population differentiation New             linked sites, selection contributes, very <br /> World. Glinka et al. (2003) observed similar differenti-        weak relative recombination rate, possible <br /> ation European African flies broad               high FST values sampling artifact. <br /> survey loci X chromosome discussed                 Even so, locus fly genome <br /> possible role selection New World producing          harbors just couple SNPs allele frequency <br /> the observed differentiation.                                     variation, total 20,000 SNPs <br />    Population stratification cause false-positive asso-       differentiating geographic races, combined <br /> ciations nucleotide variants phenotypes            measure provide unambiguous geographic <br /> population-based linkage disequilibrium mapping,              identifier. In sense, flies different from <br /> for reason critical establish      humans: 95% diversity unstructured <br /> global evidence structure data set. The depth         geographic regions, just couple per- <br /> of sampling North American data sets allows            cent sites provide differentiation the <br /> more accurate assessment population differentiation            DNA level (Aquadro et al. 2001; Rosenberg et al. 2002). <br /> along locus. One important question                   Insertion deletion polymorphism: As insertion <br /> there structure population. Our NC popula-            deletion polymorphism constitute &#226;&#181;&#8216;10% of <br /> tion collected single afternoon series            segregating polymorphisms D. melanogaster, tests of <br /> of peach bins. Since alleles identical           deviation neutral models inevitably power- <br /> unlikely single male contributed biased           ful class mutations. Schaeffer (2002) intro- <br />  1210                                                      A. Palsson et al. <br />  <br />                                                                      SupplInfo6.htm). The distribution summed repeat <br />                                                                      lengths deviates significantly normal (Kolmogorov- <br />                                                                      Smirnov goodness-of-fit test, P &#226;&#172;? 0.01), extremely <br />                                                                      platykurtic (kurtosis &#226;&#171;&#186;0.41, P &#226;&#172;? 0.0001) right <br />                                                                      skewed (0.91, P &#226;&#172;? 0.0001) assessed 1000 jackknife <br />                                                                      iterations. The fact classes separated <br />                                                                      12 bp, corresponds turn helix, <br />                                                                      suggests hypothesis length distribution may <br />                                                                      molded selective constraints orientation <br />                                                                      conserved flanking regions. These results should <br />                                                                      encourage wider survey length distributions com- <br />                                                                      posite microsatellites relation functional <br />                                                                      dissection evolutionarily conserved noncoding re- <br />                                                                      gions. <br />                                                                         Evolution Egfr: The DER protein the <br />                                                                      variable fly proteins. Disregarding alternate <br />                                                                      exon 1, just handful rare, com- <br />                                                                      mon, replacement polymorphisms &#226;&#172;&#381;1400 amino <br />                                                                      acids sequence. Three proteins initiate <br />                                                                      signaling cascade downstream DER, DRK, <br />    Figure 8.&#226;&#8364;&#8221;Nonneutral distribution microsatellite               RAS, DSOR, actually polymorphic and <br /> lengths. (A) Distribution polymorphic trinucleo- <br /> tide repeat (N &#226;&#171;&#189; 123 alleles). (B) The summed length            identical sibling species (Gasperini Gibson <br /> all repeats. Note distillation bimodal pattern      1999; Riley et al. 2003), DER clearly subject to <br /> when considering length microsatellite,         strong purifying selection. Aside structural con- <br /> that peaks 18 30 bp separated 12 bp.           straints, possible explanations functional con- <br /> Length classes 9, 24, 27 represented single        straint high levels pleiotropy (Waxman Peck <br /> allele. The invariable CA middle element not <br /> included summed length.                                       1998) occupancy critical control point a <br />                                                                      biochemical pathway (Nijhout 2002; Olsen et al. 2002; <br />                                                                      Riley et al. 2003). Egfr exhibits features, <br /> duced goodness-of-fit test equal distribution               involved dozens developmental processes <br /> indels segments Adh Adh-related loci               integrating intercellular signaling events. Given <br /> in D. pseudoobscura. The North American sample 65                Fraser et al.&#226;&#8364;&#8482;s (2002) demonstration relationship <br /> indel variants, half &#226;&#172;?0.05                 sequence constraint complexity pro- <br /> frequency, evident constraint indel size:               tein-protein interactions yeast, informa- <br /> 78% &#226;&#172;?10 bp, compared 56% anticipated                 tive conduct comparative phylogenetic analyses of <br /> mutation spectra Drosophila (Petrov Hartl                    seven receptor tyrosine kinase loci fly <br /> 1998). The deletion events scattered randomly                genome. <br /> among noncoding regions determined good-                    The high level amino acid polymorphism exon <br /> ness-of-fit test (P &#226;&#172;? 0.0001), insertions              1, including derived second initiator codon, quite <br /> marginally significant (P &#226;&#171;&#189; 0.03). In particular,              surprising given alternate exon 2 just con- <br /> is deficit large indels 1900 bp upstream exon             strained common portion protein. Most <br /> 3 3&#226;&#172;&#732; UTR larger indels reach high                 variation presumptive signal peptide, so <br /> frequencies vicinity exons 1 2                  does alter mature protein. The exons are <br /> the short introns (see supplementary Figure 3               used tissues, exon 1 used a <br /> http://statgen.ncsu.edu/ggibson/SupplInfo/SupplInfo6.                subset adult neurons (Lev et al. 1985; Schejter et al. <br /> htm). Resampling demonstrates test does depend             1986). Lesokhin et al. (1999) tested effect two <br /> quite markedly sample size, shown Table 4.                  exon 1 polymorphisms happened pres- <br />    An compelling indication purifying se-               ent Egfr Ellipse gain-of-function allele (W15R and <br /> lection observed complex microsatellite                 L21W) qualitatively neutral with <br /> promoter 2. The consensus sequence (CAA)2-9CA(GCA)1-3                respect phosphorylation DER targets transgenic <br /> (ACC)0-1 flanked 35-bp stretches nearly           flies. Since sequence conservation reduced <br /> invariant D. melanogaster differ SNPs             D. pseudoobscura undetectable mosquito <br /> when compared D. pseudoobscura. The variable                 Anopheles gambiae, presumably precise function exon <br /> repeat tendency bimodal distribution                 1 important organism exon 2. <br /> length classes greatly enhanced addition               As remarked, current sample size decreases the <br /> the shorter trinucleotides (Figure 8 supplementary               variance estimators linkage disequilibrium, molec- <br /> Table 4 http://statgen.ncsu.edu/ggibson/SupplInfo/                ular evolution, population subdivision also <br />                                                      Drosophila EGFR Sequence Diversity                                                        1211 <br />  <br /> influence analysis length distribution indels                   Freeman, M., 1998 Complexity EGF receptor signaling revealed <br />                                                                                Drosophila. Curr. Opin. Genet. Dev. 8: 407&#226;&#8364;&#8220;411. <br /> and composite microsatellites. We regard strik-                  Freydenberg, J., A. A. Hoffmann V. Loeschcke, 2003 DNA <br /> ing aspect analysis, however, subtle                          sequence variation latitudinal associations hsp23, hsp26 <br /> shifts sequence diversity gene. Statistical                       hsp27 natural populations Drosophila melanogaster. <br />                                                                                Mol. Ecol. 12: 2025&#226;&#8364;&#8220;2032. <br /> measures associated individual blocks 1 2                      Fu, Y. X., W. H. Li, 1993 Statistical tests neutrality muta- <br /> kb differ marginally expectations                       tions. Genetics 133: 693&#226;&#8364;&#8220;709. <br /> neutral theory, diversity patterns                Galindo, M. I., S. Bishop, S. Greig J. P. Couso, 2002 Leg <br />                                                                                patterning driven proximal-distal interactions EGFR sig- <br /> such small region genome calls explana-                      naling. Science 297: 256&#226;&#8364;&#8220;259. <br /> tion. As vast majority genome,                    Gasperini, R., G. Gibson, 1999 Absence protein polymor- <br /> nothing particularly noteworthy diversity                         phism Ras genes Drosophila melanogaster. J. Mol. Evol. <br />                                                                                49: 583&#226;&#8364;&#8220;590. <br /> Egfr. Yet subtle signs various modes                    Gellon, G., W. McGinnis, 1998 Shaping animal body plans in <br /> selection, drift populations, uneven recombina-                         development evolution modulation Hox expression <br /> tion rates, migration helped shape                            patterns. BioEssays 20: 116&#226;&#8364;&#8220;125. <br />                                                                           Glinka, S., L. Ometto, S. Mousset, W. Stephan D. De Lorenzo, <br /> variation typical Drosophila gene.                                     2003 Demography natural selection shaped genetic <br />    A number people instrumental helping sequence           variation Drosophila melanogaster: multi-locus approach. Genet- <br />                                                                                ics 165: 1269&#226;&#8364;&#8220;1278. <br /> analysis, including Bryon Sosinski Regina Brierley North <br />                                                                           Hudson, R. R., M. Kreitman M. Aguade&#194;&#180;, 1987 A test neutral <br /> Carolina State University Genome Research Laboratory April                 molecular evolution based nucleotide data. Genetics 116: <br /> Duty Naruo Nikoh lab. We thank Roland Carrillo             153&#226;&#8364;&#8220;159. <br /> help inbreeding North Carolina lines, Sergey Nuzhdin         Johnson-Hamlet, M., L. A. Perkins, 2002 Analysis corkscrew <br /> the Californian lines, Naruo Nikoh help analysis, Ed              signaling Drosophila epidermal growth factor receptor <br /> Buckler use Tassel software. A.P. funded             pathway myogenesis. Genetics 159: 1073&#226;&#8364;&#8220;1087. <br /> awards American Scandinavian Foundation North Atlan-         Kirby, D. A., W. Stephan, 1996 Multi-locus selection the <br /> tic Treaty Organization, project funded National            structure variation white gene Drosophila melanogaster. <br />                                                                                Genetics 144: 635&#226;&#8364;&#8220;645. <br /> Institutes Health (R01 GM61600). <br />                                                                           Langley, C. H., B. P. Lazzaro, W. Phillips, E. Heikkinen J. M. <br />                                                                                Braverman, 2000 Linkage disequilibria site frequency <br />                                                                                spectra su(s ) su(w(a )) regions Drosophila melano- <br />                                                                                gaster X chromosome. Genetics 156: 1837&#226;&#8364;&#8220;1852. <br />                      LITERATURE CITED                                     Lesokhin, A. M., S. Y. Yu, J. Katz N. E. Baker, 1999 Several <br />                                                                                levels EGF receptor signaling photoreceptor specifica- <br /> Andolfatto, P., M. Przeworski, 2000 A genome-wide depar-                   tion wild-type, Ellipse null mutant Drosophila. Dev. Biol. <br />     ture standard neutral model natural populations             205: 129&#226;&#8364;&#8220;144. <br />     Drosophila. Genetics 156: 257&#226;&#8364;&#8220;268.                                    Lev, Z., B. Z. Shilo Z. Kimchie, 1985 Developmental changes <br /> Andolfatto, P., J. Wall, 2003 Linkage disequilibrium patterns              expression Drosophila melanogaster epidermal growth <br />     recombination gradient African Drosophila melanogaster.        factor receptor gene. Dev. Biol. 110: 499&#226;&#8364;&#8220;502. <br />     Genetics 165: 1289&#226;&#8364;&#8220;1305.                                              Lewontin, R. C., 1964 The interaction selection linkage. I. <br /> Aquadro, C. F., D. Bauer F. Reed, 2001 Genome-wide variation               General considerations; heterotic models. Genetics 49: 49&#226;&#8364;&#8220;67. <br />     human fruitfly: comparison. Curr. Opin. Genet.           Lewontin, R. C., 1988 On measures gametic disequilibrium. Ge- <br />     Dev. 11: 627&#226;&#8364;&#8220;634.                                                          netics 120: 849&#226;&#8364;&#8220;852. <br /> Ardlie, K. G., L. Kruglyak M. Seielstad, 2002 Patterns             Lewontin, R. C., 1995 The detection linkage disequilibrium in <br />     linkage disequilibrium human genome. Nat. Rev. Genet.               molecular sequence data. Genetics 140: 377&#226;&#8364;&#8220;388. <br />     3: 299&#226;&#8364;&#8220;309.                                                           Mayor, C., M. Brudno, J. Schwartz, A. Poliakov, E. M. Rubin et <br /> Berry, A., M. Kreitman, 1993 Molecular analysis allozyme             al., 2000 Vista: visualizing global DNA sequence alignments of <br />     cline: alcohol dehydrogenase Drosophila melanogaster east        arbitrary length. Bioinformatics 16: 1046&#226;&#8364;&#8220;1047. <br />     coast North America. Genetics 134: 869&#226;&#8364;&#8220;893.                        McDonald, J. H., M. Kreitman, 1991 Adaptive protein evolu- <br /> Bustamante, C. D., R. Nielsen, S. Sawyer, K. Olsen, M. D. Purugga-             tion Adh locus Drosophila. Nature 351: 652&#226;&#8364;&#8220;654. <br />     nan et al., 2002 The cost inbreeding Arabidopsis. Nature        Moriyama, E. N., J. R. Powell, 1996 Intraspecific nuclear DNA <br />     416: 531&#226;&#8364;&#8220;534.                                                              variation Drosophila. Mol. Biol. Evol. 13: 261&#226;&#8364;&#8220;277. <br /> Caracristi, G., C. Schlo&#194;&#168;tterer, 2003 Genetic differentiation         Nachman, M. W., 2002 Variation recombination rate the <br />     American European Drosophila melanogaster popula-              genome: evidence implications. Curr. Opin. Genet. Dev. 12: <br />     tions attributed admixture African alleles. Mol.            657&#226;&#8364;&#8220;663. <br />     Biol. Evol. 20: 792&#226;&#8364;&#8220;799.                                              Nicholas, K. B., H. Nicholas D. Deerfield, 1997 GeneDoc: <br /> Clifford, R., T. Schu&#194;&#168;pbach, 1994 Molecular analysis                analysis visualization genetic variation. EMBO News 4: 14. <br />     Drosophila EGF receptor homolog reveals genetically      Nijhout, H. F., 2002 The nature robustness development. <br />     defined classes alleles cluster subdomains receptor           BioEssays 24: 553&#226;&#8364;&#8220;563. <br />     protein. Genetics 137: 531&#226;&#8364;&#8220;550.                                       Olsen, K. M., A. Womack, A. Garrett, J. Suddith M. D. Purug- <br /> Duchek, P., P. Rorth, 2001 Guidance cell migration EGF               ganan, 2002 Contrasting evolutionary forces Arabidopsis <br />     receptor signaling Drosophila oogenesis. Science 291:               thaliana floral developmental pathway. Genetics 160: 1641&#226;&#8364;&#8220;1650. <br />     131&#226;&#8364;&#8220;133.                                                              Palsson, A., 2003 Molecular quantitative genetics wing shape <br /> Dworkin, I. M., A. Palsson, K. Birdsall G. Gibson, 2003 Evi-               Drosophila melanogaster. Ph.D. Thesis, North Carolina State <br />     dence Egfr contributes cryptic genetic variation photo-        University, Raleigh, NC. <br />     receptor determination natural populations Drosophila mela-     Palsson, A., G. Gibson, 2004 Association nucleotide <br />     nogaster. Curr. Biol. 13: 1888&#226;&#8364;&#8220;1893.                                       variation Egfr wing shape Drosophila melanogaster. Genet- <br /> Eanes, W. F., 1999 Analysis selection enzyme polymorphisms.              ics 167: 1187&#226;&#8364;&#8220;1198. <br />     Annu. Rev. Ecol. Syst. 30: 301&#226;&#8364;&#8220;326.                                   Petrov, D. A., D. L. Hartl, 1998 High rate DNA loss the <br /> Fay, J. C., C.-I Wu, 2000 Hitchhiking positive Darwinian             D. melanogaster D. virilis species groups. Mol. Biol. Evol. 15: <br />     selection. Genetics 155: 1405&#226;&#8364;&#8220;1413.                                        293&#226;&#8364;&#8220;302. <br /> Fraser, H. B., A. Hirsh, L. Steinmetz, C. Scharfe M. W. Feld-         Pluzhnikov, A., P. Donnelly, 1996 Optimal sequencing strat- <br />     man, 2002 Evolutionary rate protein interaction network.            egy surveying molecular genetic diversity. Genetics 144: 1247&#226;&#8364;&#8220; <br />     Science 296: 750&#226;&#8364;&#8220;752.                                                      1262. <br />  1212                                                               A. Palsson et al. <br /> Powell, J. R., 1996 Progress Prospects Evolutionary Biology: The       Tajima, F., 1989 Statistical method testing neutral mutation <br />     Drosophila Model. Oxford University Press, New York.                          hypothesis DNA polymorphism. Genetics 123: 585&#226;&#8364;&#8220;595. <br /> Pritchard, J. K., M. Przeworski, 2001 Linkage disequilibrium              Teeter, K., M. Naeemuddin, R. Gasperini, E. Zimmerman, K. P. <br />     humans: models data. Am. J. Hum. Genet. 69: 1&#226;&#8364;&#8220;14.                      White et al., 2000 Haplotype dimorphism SNP collection <br /> Pritchard, J. K., M. Stephens P. Donnelly, 2000 Inference                     Drosophila melanogaster. J. Exp. Zool. 288: 63&#226;&#8364;&#8220;75. <br />     population structure using multilocus genotype data. Genetics          Thompson, J. D., D. G. Higgins T. J. Gibson, 1994 CLUSTAL <br />     155: 945&#226;&#8364;&#8220;959.                                                                 W: improving sensitivity progressive multiple sequence <br /> Reich, D. E., M. Cargill, S. Bolk, J. Ireland, P. Sabeti et al., 2001             alignment sequence weighting, positions-specific gap <br />     Linkage disequilibrium human genome. Nature 411: 199&#226;&#8364;&#8220;                  penalties weight matrix choice. Nucleic Acids Res. 22: 4673&#226;&#8364;&#8220; <br />     204.                                                                          4680. <br /> Richter, B., M. Long, R. C. Lewontin E. Nitasaka, 1997 Nu-                Ting, C. T., A. Takahashi C.-I Wu, 2001 Incipient speciation <br />     cleotide variation conservation dpp locus, gene con-             sexual isolation Drosophila: concurrent evolution multi- <br />     trolling early development Drosophila. Genetics 145: 311&#226;&#8364;&#8220;323.              ple loci. Proc. Natl. Acad. Sci. USA 98: 6709&#226;&#8364;&#8220;6713. <br /> Riley, R., W. Jin G. Gibson, 2003 Contrasting selection pres-             Upton, G. J. G., 1982 A comparison alternative tests 2 &#226;&#171;&#187; <br />     sures components Ras-mediated signal transduction                   2 comparative trial. J. R. Stat. Soc. A 145: 86&#226;&#8364;&#8220;105. <br />     pathway Drosophila. Mol. Ecol. 12: 1315&#226;&#8364;&#8220;1323.                          Verelli, B. C., W. F. Eanes, 2001 Clinal variation amino <br /> Rosenberg, N. A., J. K. Pritchard, J. Weber, H. M. Cann, K. K.                    acid polymorphisms Pgm locus Drosophila melanogaster. <br />     Kidd et al., 2002 Genetic structure human populations. Sci-                Genetics 157: 1649&#226;&#8364;&#8220;1663. <br />     ence 298: 2381&#226;&#8364;&#8220;2385.                                                      Wall, J. D., 2001 Insights linked single nucleotide polymor- <br /> Rozas, J., R. Rozas, 1999 DnaSP version 3: integrated pro-                 phisms: learn linkage disequilibrium. Curr. <br />     gram molecular population genetics molecular evolution                Opin. Genet. Dev. 11: 647&#226;&#8364;&#8220;651. <br />     analysis. Bioinformatics 15: 174&#226;&#8364;&#8220;175.                                     Wall, J. D., P. Andolfatto M. Przeworski, 2002 Testing mod- <br /> Schaeffer, S. W., 2002 Molecular population genetics sequence                  els selection demography Drosophila simulans. Genetics <br />     length diversity Adh region Drosophila pseudoobscura.               162: 203&#226;&#8364;&#8220;216. <br />     Genet. Res. 80: 163&#226;&#8364;&#8220;175.                                                  Wang, S. H., A. Simcox G. Campbell, 2000 Dual role Dro- <br /> Schaeffer, S. W., E. L. Miller, 1993 Estimates linkage dis-                sophila epidermal growth factor receptor signaling early wing <br />     equilibrium recombination parameter determined                   disc development. Genes Dev. 14: 2271&#226;&#8364;&#8220;2276. <br />     segregating nucleotide sites alcohol dehydrogenase region          Watterson, G. A., 1975 On number segregating sites in <br />     Drosophila pseudoobscura. Genetics 135: 541&#226;&#8364;&#8220;552.                           genetical models recombination. Theor. Popul. Biol. 7: <br /> Schejter, E. D., D. Segal, L. Glazer B. Z. Shilo, 1986 Alterna-               256&#226;&#8364;&#8220;276. <br />     tive 5&#226;&#172;&#732; exons tissue-specific expression Drosophila EGF        Waxman, D., J. R. Peck, 1998 Pleiotropy preservation <br />     receptor homolog transcrip <br /> </body></html>