<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The candidate tumor suppressor ING4 represses <br /> activation hypoxia inducible factor (HIF) <br /> Abdullah Ozer, Leeju C. Wu, Richard K. Bruick* <br /> Department Biochemistry, University Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, TX 75390-9038 <br />  <br /> Communicated Steven L. McKnight, University Texas Southwestern Medical Center, Dallas, TX, April 1, 2005 (received review January 31, 2005) <br />  <br /> The hypoxia inducible factor (HIF) plays important role              HIF regulated posttranslational hydroxylation (11) by <br /> progression number pathophysiological processes including              O2-dependent hydroxylase, factor inhibiting HIF 1 (FIH-1) <br /> tumorigenesis. In addition characterized oxygen-               (12&#226;&#8364;&#8220;14). <br /> dependent modes regulation, function HIF transcrip-                 The hypoxic response pathway recognized an <br /> tion factor influenced action                 important contributor number cancers, increased <br /> regulatory pathways. Misregulation factors resulting               levels HIF associated increased tumor aggres- <br /> inappropriate HIF expression activity contribute                 siveness, therapeutic resistance, mortality (15). HIF be <br /> progression human cancers induction genes                    induced result intratumoral hypoxia stemming rapid <br /> promoting angiogenesis, glycolysis, cell <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>survival</span>, metastasis,             growth and&#229;&#8230;&#190;or inefficient O2 delivery tortuous vasculature. <br /> among processes. The candidate tumor suppressor protein                  Alternatively, HIF function altered O2-independent <br /> inhibitor growth family member 4 (ING4) recently                   manner genetic alterations activate signaling <br /> implicated repressor angiogenesis tumor growth                     pathways inactivate tumor suppressors (15). Understanding of <br /> through association NF-&#226;?&#172;B. Here demonstrate sup-                  mechanisms oncogenes tumor suppressor genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              BIOCHEMISTRY <br /> pression ING4 induces HIF transcriptional activity well.         affect HIF function sheds light misregu- <br /> ING4 directly associates HIF prolyl hydroxylase, Fe(II)-           lation promotes cancer reveals physiologically relevant <br /> dependent oxygenase previously shown mediate HIF stability               mechanisms cells fine-tune HIF activity the <br /> a function oxygen availability. However, affecting              borderline hypoxic conditions encountered &#226;&#8364;&#732;&#226;&#8364;&#732;normoxic&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> HIF&#226;&#8364;&#8482;s stability, ING4 mediates HIF&#226;&#8364;&#8482;s activity. These data support            tissues (16&#226;&#8364;&#8220;18). <br /> model which, addition regulating HIF stability, HIF prolyl               Recently, inhibitor growth family member 4 (ING4) has <br /> hydroxylases modulate HIF function recruitment                 emerged strong candidate tumor suppressor protein, re- <br /> of ING4, likely component chromatin-remodeling complex.                 pressing tumor growth angiogenesis (19) loss of <br />                                                                                contact inhibition (20). ING4 member family proteins <br /> prolyl hydroxylase &#229;&#8230;&#169; inhibitor growth <br />                                                                                characterized highly conserved C-terminal plant homeodo- <br />                                                                                main (PHD)-like zinc-finger domain implicated variety <br />  <br /> T    o maintain adequate levels aerobic respiration, mammalian <br />      cells able sense status O2 availability and <br /> respond changes O2 levels fall critical levels. <br />                                                                                processes including oncogenesis, apoptosis, DNA repair, and <br />                                                                                cell cycle control (21). The best-studied ING family members <br />                                                                                reside nucleus positively negatively regulate <br /> A key component hypoxic response pathway hypoxia                 gene expression interactions N terminus <br /> inducible factor (HIF). Decreased O2 cell results HIF              chromatin-remodeling complexes containing histone acetyl- <br /> induction subsequent transcription number target               transferases, histone deacetyltransferases, factor acetyltrans- <br /> genes promote adaptation environmental stress. The list           ferases (21). <br /> of HIF targets includes genes affecting metabolism, O2 delivery,                  Although ING4&#226;&#8364;&#8482;s precise mode action elucidated, <br /> angiogenesis, cellular survival (1). Consequently, HIF plays            ING4 shown interact RelA subunit NF-&#226;?&#172;B <br /> important role physiological pathophysiological states               suppress expression angiogenesis-related genes including <br /> which O2 supply limiting (1, 2).                                            IL-6, IL-8, Cox-2 (19). Here ING4 serves <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />     HIF obligate heterodimer composed HIF-&#226;?&#164; subunit                 suppress expression HIF target genes hypoxic conditions. <br /> that essentially insensitive O2 availability HIF-&#226;?&#163; subunit         Furthermore, HPH-2 directly interacts ING4, providing a <br /> whose accumulation activity acutely responsive O2 levels             possible mechanism ING4 recruitment HIF. Interestingly, <br /> (reviewed refs. 1&#226;&#8364;&#8220;3). Briefly, normoxic conditions HIF-&#226;?&#163;              ING4 association HPH-2 does affect hydroxylase <br /> is targeted rapid proteosomal degradation association           activity HIF stability. Instead, data consistent a <br /> a protein ubiquitin ligase complex containing product von           model ING4, recruited HPH-2 HIF hypoxic <br /> Hippel&#226;&#8364;&#8220;Lindau tumor suppressor gene (pVHL). pVHL fails bind                 conditions, acts adapter protein recruit transcriptional <br /> the &#226;?&#163;-subunit hypoxic conditions, allowing HIF accumu-                repressors mediate HIF activity. <br /> late. Differential recognition HIF-&#226;?&#163; VHL mediated by <br /> hydroxylation conserved proline residues HIF-&#226;?&#163;                   Materials Methods <br /> O2-dependent degradation domain (4). Under normoxic condi-                     Recombinant Protein Expression Purification. Protein coding <br /> tions, HIF prolyl hydroxylases (HPHs; called PHDs EGLNs)               sequences HPH-2 (GenBank accession no. AF229245) and <br /> efficiently modify residues (5&#226;&#8364;&#8220;7), prerequisite VHL                ING4 (GenBank accession no. NM&#227;&#8250;&#173;016162) amplified PCR <br /> binding HIF-&#226;?&#163; (8, 9). The HPH active site contains Fe(II)                   confirmed sequencing. Recombinant FIH-1 prepared as <br /> coordinated His-Xaa-Asp . . . His triad. These enzymes turn <br />                                                                                described (13). The sequence encoding HPH-2 residues 181&#226;&#8364;&#8220;426 <br /> bind 2-oxoglutarate, HIF-&#226;?&#163;, O2 effect hydroxylation both <br /> the HIF-&#226;?&#163; subunit 2-oxoglutarate. Hydroxylated 2-oxogluta- <br /> rate undergoes decarboxylation produce succinate CO2.                   Abbreviations: HIF, hypoxia inducible factor; VHL, von Hippel&#226;&#8364;&#8220;Lindau tumor suppressor <br /> Because enzymes require O2 substrate hydroxy-               gene; HPH, HIF prolyl hydroxylase; FIH-1, factor inhibiting HIF-1; ING, inhibitor growth <br /> lation reaction, implicated direct O2 sensors         family member; PHD, plant homeodomain; siRNA, short interfering RNA. <br /> hypoxic response pathway (10). Independent protein stability,               *To correspondence addressed. E-mail: bruick@biochem.swmed.edu. <br /> coactivator recruitment C-terminal transactivation domain               &#194;&#169; 2005 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                        PNAS &#229;&#8230;&#169; May 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 21 &#229;&#8230;&#169; 7481&#226;&#8364;&#8220;7486 <br />  (HPH-2C) subcloned pHIS parallel vector (22)         2,400 &#226;&#171;&#187; g 4&#194;&#176;C 5 min. Pellets washed 25 mM <br /> purified following procedure used FIH-1 (13).         Hepes-KOH (pH 7.5), 3 mM MgCl2, 1 mM DTT, 0.1% Nonidet <br />   ING4 expressed Escherichia coli G&#226;?&#164;1 fusion protein.     P-40, 1:200 Protease Inhibitor Mixture resuspended the <br /> Cells lysed buffer containing 20 mM Tris&#228;&#161;&#160;HCl (pH 8.0), 50     buffer. Cells lysed passage 25-gauge needle <br /> mM NaCl, 5 mM 2-mercaptoethanol, 100 &#226;?&#174;M ZnCl2, 1:200              centrifuged 30 s. The cytosolic supernatant removed, <br /> Protease Inhibitor Mixture (Sigma) purified Source Q         nuclear pellets washed times resuspen- <br /> resin (Amersham Pharmacia). The G&#226;?&#164;1 tag removed TEV            sion 20 mM Hepes-KOH (pH 7.5), 400 mM KCl, 1 mM DTT, <br /> protease digestion followed ion exchange chromatography using      10% glycerol incubated 4&#194;&#176;C 30 min. The samples <br /> Mono S resin (Amersham Pharmacia). ING4 stored buffer          centrifuged 200,000 &#226;&#171;&#187; g 45 min 4&#194;&#176;C separate the <br /> containing 50 mM NaPO4 (pH 5.9), 300 mM NaCl, 1.0 mM                  extracted nuclear proteins insoluble material. <br /> 2-mercaptoethanol, 10 &#226;?&#174;M ZnCl2 4&#194;&#176;C. <br />                                                                       [14C]-2-Oxoglutarate Decarboxylation Assay. HPH activity mea- <br /> Yeast Two-Hybrid Assay. The yeast strain L40 transformed     sured modified [14C]-2-oxoglutarate decarboxylation assay <br /> vectors encoding HPH-2C fused LexA DNA-binding domain          derived ref. 24. Briefly, 1-ml reaction containing 3.0 &#226;?&#174;M <br /> (DBD) (HPH-2C&#229;&#8230;&#190;pVJL10) ING4 fragments fused                 peptide substrate (HIF-1&#226;?&#163; residues 556&#226;&#8364;&#8220;574) incubated with <br /> GAL4 activation domain (ING4&#229;&#8230;&#190;pGAD-NotII). Positive protein&#226;&#8364;&#8220;           recombinant HPH-2C enzyme presence 50 mM Tris&#228;&#161;&#160;HCl <br /> protein interactions denoted LacZ reporter expression.        (pH 8.0), 2 mg&#229;&#8230;&#190;ml BSA, 0.2 mg&#229;&#8230;&#190;ml catalase, 5 mM KCl, 1.5 mM <br />                                                                       MgCl2, 1.0 mM DTT, 64 &#226;?&#174;M [14C]-2-oxoglutarate [14.6 nCi&#229;&#8230;&#190;nmol <br /> Western Blot Analysis. Cells resuspended SDS sample buffer,   specific activity (1 Ci &#226;&#171;&#189; 37 GBq)], 50 &#226;?&#174;M ascorbate, 20 &#226;?&#174;M <br /> and proteins resolved SDS&#229;&#8230;&#190;PAGE Western blot            FeSO4 sealed 15-ml tube. After 1-h incubation room <br /> analysis. Polyclonal antiserum obtained rabbits immu-        temperature gentle shaking, pH solution was <br /> nized HPH-2C, ING4, 140 amino acids          lowered &#226;&#172;&#8225;2 diluted HClO4. Released [14C]CO2 cap- <br /> HIF-&#226;?&#164; subunit. The HPH-2 antiserum failed detect endogenous        tured 3M Whatman paper saturated 10 M NaOH and <br /> HPH-1 HPH-3. The ING4 antiserum displayed minimal cross-           measured scintillation counting. <br /> reactivity ING family members, weak <br /> detection ING1 ING5 seen overexpression. Mouse       <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>RNA Interference</span>. Cells plated 24-well plates (3 &#226;&#171;&#187; 104 cells <br /> monoclonal antibodies p-ATF-2, annexin I (Santa Cruz Bio-          well) 16 h transfection, 200 nM small interfering <br /> technology), HIF-1&#226;?&#163; (BD Transduction Laboratories, Lexing-        RNA (siRNA) duplexes (Dharmacon, Lafayette, CO) trans- <br /> ton, KY) purchased. Immune complexes detected            fected using Oligofectamine reagent (Invitrogen). Media were <br /> enhanced chemiluminescence using peroxidase-conjugated sec-           replaced 16&#226;&#8364;&#8220;20 h transfection. Cells maintained either <br /> ondary antibodies (Jackson ImmunoResearch).                           normoxic conditions (20% O2) 72 h transfection or <br />                                                                       normoxic conditions 57 h followed 12&#226;&#8364;&#8220;15 h hypoxia <br /> Cell Culture. HeLa cell lines maintained Dulbecco&#226;&#8364;&#8482;s           (1% O2). Total RNA preparation Northern blot analysis were <br /> modified Eagle&#226;&#8364;&#8482;s medium (HyQ DME, HyClone) containing                 performed according methods ref. 23. For transient <br /> high glucose supplemented 10% FBS (Gemini Biolog-            transfections, HeLa cells transfected 3XHRE-tk-Luc <br /> ical Products, Calabasas, CA) presence 5.0% CO2          reporter construct 51 h siRNA transfection using Lipo- <br /> 37&#194;&#176;C. Cells maintained hypoxic conditions 37&#194;&#176;C          fectamine Plus (Invitrogen). siRNA duplexes composed the <br /> within humidified hypoxic chamber (Coy Laboratory Products,         following oligonucleotides: ING4#1, UGAGGGACCUAGAC- <br /> Ann Arbor, MI) filled 1% O2&#229;&#8230;&#190;5.0% CO2 balanced           CAAAGTT CUUUGGUCUAGGUCCCUCATT; ING4#2, <br /> N2. Luciferase assays performed according method          GAACGGAAGAAGAAAUAGATT UCUAUUUCU- <br /> of ref. 23.                                                           UCUUCCGUUCTT; GFP, GGCUACGUCCAGGAGCG- <br />                                                                       CACC UGCGCUCCUGGACGUAGCCUU. <br /> Coimmunoprecipitation Assay. HeLa cells transfected with <br /> ING4 p3XFLAG-CMV10 vector (Sigma) using Lipo-               Chromatin Immunoprecipitation Assay. Briefly, HeLa cell line <br /> fectamine plus reagent (Invitrogen). After 24 h, cells lysed     stably transfected 3XHRE-tk-Luc reporter construct was <br /> with 20 mM Tris&#228;&#161;&#160;HCl (pH 7.5), 100 mM NaCl, 5 mM 2-mercap-             maintained hypoxic conditions (1% O2) 15 h. Crosslink- <br /> toethanol, 1% Nonidet P-40, Protease Inhibitor Mixture            ing, lysate preparation, immunoprecipitations, DNA purifica- <br /> (Sigma). Lysates precleared Protein A-agarose beads         tion performed according experimental procedures <br /> (Roche Diagnostics) incubated presence preimmune        described ref. 25. A 227-bp region 3XHRE promoter was <br /> serum HPH-2C immune serum overnight 4&#194;&#176;C. Antibody&#226;&#8364;&#8220;              amplified using following primers: 5&#226;&#172;&#732;-AGTGCAGGTGC- <br /> protein complexes precipitated Protein A-agarose beads,     CAGAACATT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CGGTAGGTCGAGAGGTCAGA-3&#226;&#172;&#732;. <br /> washed extensively lysis buffer, eluted SDS loading     PCR products separated electrophoresis 1.5% agarose <br /> buffer. Coimmunoprecipitated 3XFLAG-tagged ING4 de-               gel stained ethidium bromide. Relative band intensities <br /> tected Western blot analysis using anti-FLAG M2 antibody           quantitated QUANTITY ONE (Bio-Rad) software. <br /> (Sigma). <br />                                                                       Results <br />                   35S-labeled <br /> GST Pull-Downs.               proteins prepared TNT       ING4 Represses Hypoxic Induction HIF Target Genes. siRNAs were <br /> Coupled Reticulocyte Lysate System (Promega). Lysates contain-        used efficiently suppress expression ING4 HeLa cells as <br /> ing 35S-labeled proteins incubated GST fusion proteins      determined Western blot analysis (Fig. 1A). The remaining <br /> immobilized glutathione Sepharose 4B resin (Amersham Phar-         protein signal low levels unsuppressed ING4 or <br /> macia) 1 h 4&#194;&#176;C. After washes buffer containing    potential cross-reactivity endogenous ING5. After siRNA <br /> 20 mM Tris&#228;&#161;&#160;HCl (pH 7.5), 200 mM NaCl, 1 mM DTT, 0.5%              treatment, cells incubated normoxic conditions or <br /> Nonidet P-40, bound proteins eluted boiling SDS sample     subjected hypoxia promote induction HIF-responsive genes. <br /> buffer resolved SDS&#229;&#8230;&#190;PAGE.                                      When cells treated control siRNA targeting GFP, two <br />                                                                       characterized HIF target genes, Nip3 (23) adenylate kinase <br /> Subcellular Fractionation. HeLa cells washed ice-cold PBS   3 (AK3) (26), induced response hypoxia assessed <br /> and harvested scraping. Cell pellets resuspended 10 mM     Northern blot analysis (Fig. 1B). When ING4 expression was <br /> Hepes-KOH (pH 7.5), incubated ice 10 min, centrifuged      suppressed ING4-targeted siRNAs, levels of <br />  <br /> 7482 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                                                Ozer et al. <br />                                                                                                                                                                   BIOCHEMISTRY <br /> Fig. 1. siRNA suppression ING4 enhances HIF target gene expression <br /> under hypoxic conditions. (A) Western blot analysis indicating relative protein <br /> levels HPH-2, ING4, nuclear HIF-1&#226;?&#163; normoxic (20% O2) hypoxic <br /> (1% O2) conditions 12 h treatment HeLa cells siRNA duplexes <br /> specific ING4 (ING4#1 ING4#2) GFP control. HIF-2&#226;?&#163; undetectable <br /> in Western blots HeLa cell extracts. (B) Northern blot analysis the <br /> endogenous HIF target genes Nip3 AK3 siRNA-mediated suppression <br /> of ING4. Actin mRNA levels shown confirm equivalent RNA loading. All <br /> results representative multiple experiments. <br />  <br />  <br />  <br /> both endogenous HIF target genes hypoxic condi- <br /> tions typically increased additional 2- 3-fold relative the <br /> GFP siRNA control-treated cells (Fig. 1B). These increases in <br />                                                                                   Fig. 2. ING4 interacts HPH-2 C-terminal PHD (residues <br /> HIF-responsive gene expression increase in <br />                                                                                   191&#226;&#8364;&#8220;249). (A Upper) Yeasts transformed vectors encoding HPH-2C <br /> nuclear HIF-1&#226;?&#163; levels (Fig. 1 A), suggesting HIF activity,            fused LexA DNA-binding domain (DBD) ING4 fused GAL4 <br /> than HIF stability, suppressed ING4. As expected,                 activation domain. Positive protein&#226;&#8364;&#8220;protein interactions denoted LacZ <br /> effect ING4 suppression observed HIF target genes                reporter expression. Transformations performed duplicate. (Lower) <br /> in conditions (normoxia) HIF-1&#226;?&#163; does accumulate                   Yeast two-hybrid analysis HPH-2C interactions N- C-terminal <br /> (Fig. 1 A). Consistent ING4-dependent <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> ob-                      truncations ING4 shown. (B) 35S-labeled full-<span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> ING4 (residues <br /> served NF-&#226;?&#172;B-responsive genes (19), inverse rela-                 1&#226;&#8364;&#8220;249) ING4 lacking PHD (residues 1&#226;&#8364;&#8220;198) incubated immo- <br /> tionship ING4 levels expression HIF target genes                   bilized GST GST&#229;&#8230;&#190;HPH-2C fusion protein. Bound 35S-labeled ING4 was <br />                                                                                   visualized SDS&#229;&#8230;&#190;PAGE. (C) Lysates HeLa cells transfected N- <br /> in cells HIF induced hypoxia. <br />                                                                                   terminal 3XFLAG-tagged ING4 immunoprecipitated anti-HPH-2 <br />                                                                                   serum preimmune serum control. ING4 coimmunoprecipitated <br /> HPH-2 Interacts ING4. Although hypothesized ING4                     HPH-2 detected Western blot analysis using anti-FLAG M2 <br /> might interact directly HIF, unable detect                antibody. IP, immunoprecipitate. <br /> interaction variety assays (data shown). However, did <br /> detect interaction ING4 factor known regulate <br /> HIF-&#226;?&#163; stability, HPH-2. An interaction proteins                 ING4 weakly ING4 lacking PHD in vitro GST <br /> was initially observed yeast two-hybrid screen                  pull-down assays (Fig. 2B). An interaction HPH-2 and <br /> LexA DNA-binding domain (DBD) fused catalytic                          ING4 detected coimmunoprecipitation assay <br /> hydroxylase domain HPH-2 (HPH-2C). By itself, LexA                         (Fig. 2C). <br /> DBD&#229;&#8230;&#190;HPH-2C fusion protein capable activating re-                          To demonstrate direct association ING4 and <br /> porter genes driven LexA promoter (Fig. 2A). When yeast                HPH-2 vitro, proteins expressed bacteria and <br />                                                                                   purified near homogeneity. As shown Fig. 3, HPH-2C mi- <br /> cells cotransformed construct expressing ING4 fused to <br />                                                                                   grated predicted 27-kDa monomer size exclusion <br /> the GAL4 activation domain, activation LexA-responsive                     column (Fig. 3B), behavior ING4 (predicted mo- <br /> LacZ reporter gene observed, indicative positive interac-                lecular mass 29 kDa) consistent multimeric <br /> tion HPH-2C ING4 (Fig. 2 A).                                          species (Fig. 3A). When proteins incubated together, <br />    To identify region ING4 associates HPH-2C,                  observed migrate larger complexes (Fig. <br /> series N- C-terminal deletion constructs assayed. As                  3C). In contrast, incubation ING4 regulatory HIF <br /> shown Fig. 2 A, residues 191&#226;&#8364;&#8220;249 ING4, encompassing                     hydroxylase, FIH-1, did result appearance larger <br /> PHD, sufficient interact HPH-2C. Almost interaction                protein complex (Fig. 3E). Together, independent lines of <br /> was observed 198 residues ING4. Consistent                 evidence demonstrate direct association ING4 and <br /> these results, HPH-2C able interact strongly full-length              HPH-2. <br />  <br /> Ozer et al.                                                                                              PNAS &#229;&#8230;&#169; May 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 21 &#229;&#8230;&#169; 7483 <br />  Fig. 3. Purified ING4 HPH-2C directly associate vitro. One         Fig. 5. ING4 HPH-2 substrate vitro. Substrate utilization by <br /> micrograms purified recombinant ING4, HPH-2C, and&#229;&#8230;&#190;or FIH-1 proteins        HPH-2C assessed [14C]CO2 generation presence control <br /> incubated resolution Superdex 200 10&#229;&#8230;&#190;30             HIF-1&#226;?&#163; peptide substrate recombinant ING4 protein. Assays per- <br /> column (Amersham Pharmacia Biotech). Eluted fractions resolved            formed triplicate. <br /> SDS&#229;&#8230;&#190;PAGE visualized Coomassie blue staining. The elution profile of <br /> protein standards given top. <br />                                                                                   lation substrate HPH-2, employed hydroxylase assay using <br />                                                                                   [14C]-2-oxoglutarate cosubstrate (24). [14C]-2-oxoglutarate is <br /> Both HPH-2 ING4 Reside Nucleus. The results indicate             efficiently hydroxylated HPH-2C presence a <br /> that HPH-2 directly interacts ING4. In cells, however, ING                   substrate HIF-1&#226;?&#163; peptide. Hydroxylated [14C]-2- <br /> family members predicted reside nucleus (27).                   oxoglutarate undergoes decarboxylation release [14C]CO2, <br /> However, localization studies performed overexpressed GFP&#226;&#8364;&#8220;                   captured quantified. The levels emitted <br /> HPH protein fusions indicate HPH-2 resides primarily                  [14C]CO2 reflect relative utilization putative <br /> cytoplasm (28, 29). To determine endogenous ING4                      polypeptide substrate hydroxylation reaction (Fig. 5). Unlike <br /> HPH-2 reside subcellular compartment,                     results obtained HIF-1&#226;?&#163; substrate, incubation of <br /> examined distribution HeLa cell extracts (Fig. 4). After                 HPH-2C ING4 did result increased levels [14C]CO2 <br /> incubation, cells lysed, soluble cytosolic proteins                 production, indicating ING4 HPH-2 substrate. <br /> separated insoluble cellular material, including intact nuclei, <br /> which subjected high-salt extraction nuclear proteins.             ING4 Does Not Affect HPH-2 Activity HIF Stability. Because <br /> As predicted, ING4 exclusively fraction containing               ING4 did appear HPH-2 substrate affect HPH-2 <br /> extracted nuclear proteins. Contrary GFP&#226;&#8364;&#8220;HPH-2 fusion                      HIF accumulation (Fig. 1 A), reasoned ING4 association <br /> results, endogenous HPH-2 equally distributed frac-                   HPH-2 directly affect hydroxylase activity. To test this <br /> tions. These data provide evidence substantial portion of <br />                                                                                   hypothesis, hydroxylation HIF peptide substrate HPH-2C <br /> HPH-2 does reside nucleus, promote O2- <br />                                                                                   followed presence increasing amounts purified <br /> dependent degradation nuclear HIF associate with <br />                                                                                   ING4. The ability HPH-2C hydroxylate HIF-&#226;?&#163; assay was <br /> nuclear factors ING4. <br />                                                                                   assessed subsequent interaction peptide substrate with <br />                                                                                   35S-labeled VHL pull-down assay (6). As shown Fig. 6, <br /> ING4 Is Not HPH-2 Substrate. In addition ING4, two-hybrid <br /> interactions observed HPH-2C known                         recombinant ING4 stimulated inhibited hydroxylation <br /> HPH substrates, HIF-1&#226;?&#163; HIF-2&#226;?&#163; (data shown). Therefore,                    peptide substrate HPH-2C. The lack effect ING4 <br /> we speculated positive interaction reflect enzyme&#226;&#8364;&#8220; <br /> substrate relationship HPH-2 ING4. Known targets <br /> for HPHs contain conserved LXXLAP motif (30). Although this <br /> sequence present ING family members, <br /> mutations tolerated adjacent proline residue in <br /> peptide substrates (31). To test ING4 hydroxy- <br />  <br />  <br />  <br />  <br />                                                                                   Fig. 6. ING4 does affect HPH-2C activity vitro. Hydroxylase activity of <br />                                                                                   recombinant HPH-2C measured 35S-labeled VHL pull-down assay in <br /> Fig. 4.    Both ING4 HPH-2 salt-extracted nuclear             presence HIF-1&#226;?&#163; peptide substrate increasing amounts recom- <br /> proteins. HeLa cells incubated normoxic (20% O2) hypoxic (1%        binant ING4 (the ING4 protein relative HPH-2C indicated). <br /> O2) conditions 12 h. Soluble cytoplasmic (C) salt-extracted nuclear (N)   Reaction conditions chosen provide approximately half-maximal <br /> proteins separated analyzed Western blot analysis antibod-       substrate hydroxylation observe stimulation inhibition hydrox- <br /> ies raised ING4 HPH-2. Antibodies p-ATF-2 annexin I used    ylase activity. 35S-labeled VHL binding fully hydroxylated peptide (&#226;&#171;&#185; <br /> to assess integrity nuclear cytoplasmic samples, respectively.     control) shown reference. Assays performed triplicate. <br />  <br />  <br /> 7484 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                                                                   Ozer et al. <br />                                                                                     Fig. 8. ING4, HPH-2, HIF associate HRE promoter. HeLa cells <br />                                                                                    stably transfected 3XHRE-tk-Luc reporter construct incubated <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  BIOCHEMISTRY <br />                                                                                    hypoxic (1% O2) conditions 15 h followed crosslinking, sonica- <br /> Fig. 7. ING4 affects HIF activity chromatin-dependent manner. (A)             tion, immunoprecipitation using antibodies specific ING4, HIF-&#226;?&#164;, or <br /> Suppression ING4 increases expression stably transfected HIF-driven        HPH-2. (Upper) Primers flanking 3XHRE promoter element used to <br /> luciferase reporter gene. A HeLa cell line stably expressing HIF-responsive    amplify associated chromatin DNA PCR. (Lower) Relative band intensities <br /> 3XHRE-tk-Luc reporter gene transfected siRNA duplexes specific        indicated. No Ab, antibody control; Pre Immune, preimmune serum <br /> either ING4 (duplex ING4#1) GFP followed incubation normoxic           control. <br /> (20% O2) hypoxic (1% O2) conditions 15 h. Similar results obtained <br /> with independently isolated stably transfected HeLa cell lines (data not <br /> shown). (B) siRNA suppression ING4 does affect hypoxic induction a <br />                                                                                    factors expected affect transcription <br /> transiently transfected HIF reporter gene. Wild-type HeLa cells trans- <br /> fected siRNA duplexes specific ING4 GFP control followed <br />                                                                                    transiently transfected (nonchromosomal) DNA. <br /> by 3XHRE-tk-Luc HIF reporter construct incubation normoxic or <br /> hypoxic conditions. All assays performed triplicate, results   Discussion <br /> representative multiple experiments.                                            The HIF-dependent hypoxic response pathway present virtu- <br />                                                                                    ally cell body participates physiological processes <br />                                                                                    disease states ischemia cancer (1, 2, 15). The <br /> on hydroxylase activity vitro consistent lack change             discovery hydroxylases mediate HIF-&#226;?&#163; stability and <br /> in nuclear HIF-1&#226;?&#163; protein levels ING4 levels suppressed                   coactivator recruitment provided mechanism HIF <br /> by siRNA (Fig. 1 A).                                                               function directly regulated O2. Although availability <br />                                                                                    O2 serve hydroxylase substrate plays major role in <br /> ING4 Suppresses HIF Activity Chromatin-Dependent Manner.                      determining status HIF induction given cell (10), both <br /> Because ING family members interact chromatin-                          HIF expression hydroxylase function likely subject to <br /> remodeling factors (21), speculated ING4 likewise                    additional layers regulation involving signaling pathways, post- <br /> suppress HIF activity recruiting factors restrict accessibility <br />                                                                                    translational modification, feedback loops, protein&#226;&#8364;&#8220;protein inter- <br /> of transcriptional machinery DNA. To begin test this <br />                                                                                    actions, protein localization (1). The extensive dynamic <br /> model, compared effect siRNA-mediated ING4 suppres- <br />                                                                                    regulation HIF reflects critical biological role also <br /> sion hypoxic induction HIF-responsive luciferase <br />                                                                                    need fine-tune HIF induction physiologically relevant <br /> reporter driven multiple HIF responsive elements, 3XHRE-tk- <br /> Luc (32), stably (Fig. 7A) transiently (Fig. 7B)                O2 concentrations encountered &#226;&#8364;&#732;&#226;&#8364;&#732;normoxic&#226;&#8364;&#8482;&#226;&#8364;&#8482; conditions in <br /> transfected. After siRNA treatment, stably transfected cells                  HIF transcription factors partially induced. <br /> incubated hypoxic conditions promote induction                 Although relationship O2 availability hydroxylase <br /> luciferase reporter gene. When cells treated control                   activity serve general determinant HIF induction, <br /> GFP siRNA, relative luciferase activity induced 10-fold                 multiple layers regulation likely required subtly adapt HIF <br /> in response hypoxia (Fig. 7A). When ING4 expression                         function variety dynamic microenvironments. <br /> suppressed, luciferase reporter activity hypoxic conditions                     The physiological importance potential therapeutic utility of <br /> typically increased additional 2-fold relative GFP-targeted              hypoxic response pathway driven search additional <br /> siRNA controls. This result entirely consistent data                   regulatory components. To end, investigated candidate <br /> presented Fig. 1B endogenous HIF target genes. Consistent                   tumor suppressor ING4. An inverse relationship ING4 <br /> with hypothesis, ING4, HPH-2, HIF subunit                      expression tumor growth vascular volume human <br /> found associate 3XHRE element hypoxic con-                       glioblastomas mouse xenograft models observed. <br /> ditions chromatin immunoprecipitation assay (Fig. 8).                         Whereas two-hybrid coimmunoprecipitation experiments have <br />    In contrast results observed stably transfected                 revealed interaction ING4 NF-&#226;?&#172;B (19), precise <br /> reporter, hypoxic induction transiently transfected reporter                mechanism ING4 suppresses NF-&#226;?&#172;B function been <br /> was unaffected ING4-targeted siRNA (Fig. 7B). These                         determined. Although HIF transcription factors fre- <br /> results consistent model ING4 suppresses HIF                   quently induced cancers, they, too, promote tumor- <br /> activity recruitment chromatin-remodeling factors,                  igenesis, aforementioned cell culture studies change in <br />  <br /> Ozer et al.                                                                                               PNAS &#229;&#8230;&#169; May 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 21 &#229;&#8230;&#169; 7485 <br />  expression levels proangiogenic HIF target genes (i.e., VEGF)                                   ING4. In case HIF-responsive genes HeLa cells, these <br /> was observed manipulation ING4 (19). However,                                  putative ING4-associated factors modestly suppress <br /> noted studies performed hypoxic con-                                     transcription. This local changes histone <br /> ditions expect HIF induction. Suspecting                                    acetylation state changes acetylation state of <br /> that ING4 suppress HIF-mediated effects tumor                                        nonhistone proteins including HIF itself. In support chromatin- <br /> growth angiogenesis, examined effects ING4                                    remodeling model, suppression ING4 did affect expres- <br /> levels HIF target gene induction normoxic                                        sion transiently transfected HIF reporter subject to <br /> hypoxic conditions.                                                                                regulation function chromatin structure. <br />    Consistent hypothesis, suppression ING4 HeLa                                        We far focused attention ING4 association with <br /> cells led significant reproducible increase                                        HPH-2. Given high conservation C-terminal regions <br /> induction endogenous HIF target genes hypoxic con-                                        HPH proteins, possible ING4 also <br /> ditions. A similar result obtained using stably trans-                                    interact members HPH family. Likewise, <br /> fected reporter gene driven minimal HIF-responsive pro-                                       PHD highly conserved members ING <br /> moter, supporting contention effects ING4                                     family, HPHs interact ING proteins could <br /> mediated HIF transcriptional regu-                                       turn recruit different sets proteins HIF HIF-dependent <br /> lators. Although transcriptional targets NF-&#226;?&#172;B                                        promoters. Therefore, precise consequence(s) ING associ- <br /> reported promote HIF induction increasing HIF-1&#226;?&#163; sta-                                        ation HPHs requires additional study likely be <br /> bility (33), believe ING4 indirectly regulates HIF                                  complicated differences function, protein recruitment, and <br /> through NF-&#226;?&#172;B mechanism. In studies, change                                       expression patterns ING HPH family members. <br /> in HIF-1&#226;?&#163; stability observed ING4 suppression,                                          Together, data demonstrate candidate tumor <br /> the effects ING4 suppression pronounced                                         suppressor protein ING4 represses function factors <br /> hypoxic conditions, unlike effects NF-&#226;?&#172;B HIF,                                      mediating tumor growth angiogenesis addition NF-&#226;?&#172;B. <br /> have observed normoxia (33).                                                       Furthermore, direct interaction ING4 and <br />    Although ING4 directly recruited NF-&#226;?&#172;B,                                  HIF regulatory factor, suggesting mechanism ING4 recruit- <br /> unable detect direct association ING4 HIF. <br />                                                                                                    ment HIF additional role HIF prolyl <br /> Instead, ING4 associated directly known HIF regulator, <br />                                                                                                    hydroxylases mediating HIF function hypoxic conditions. <br /> HPH-2, multiple assays. ING4 substrate HPH-2, nor <br />                                                                                                    Lastly, provide evidence repressive effects ING4 <br /> did affect HPH activity like recently reported HPH- <br />                                                                                                    depend context transcribed target genes, implying that <br /> interacting factor OS-9 (34). Instead, study provides evidence <br />                                                                                                    ING4&#226;&#8364;&#8482;s mode action depend ability recruit <br /> that substantial fraction HPH-2 resides nucleus it <br />                                                                                                    chromatin-remodeling factors. Although candidate chromatin- <br /> can mediate HIF degradation reoxygenation as <br /> associate ING4 affect HIF activity.                                                        remodeling factors proposed interact ING4 (35), <br />    ING family members shown promote chromatin                                         additional studies required identify physiological <br /> remodeling recruitment factors promote                                      complexes ING4 resides. <br /> suppress transcriptional activation (21). When O2 levels high, <br />                                                                                                    We thank B. Horazdovsky, C. Dann, P. Erbel, J. Rutter helpful <br /> HPH-2 hydroxylates proline residues O2-dependent <br />                                                                                                    advice. This work supported Burroughs Wellcome Fund Career <br /> degradation domain subsequently displaced product                                    Award, University Texas Southwestern President&#226;&#8364;&#8482;s Research <br /> of VHL (pVHL), leading rapid HIF-&#226;?&#163; degradation                                      Council, Robert A. Welch Foundation, American Cancer <br /> transcriptional repression pVHL-mediated recruitment                                    Society Young Investigator Award (to R.K.B.) conducted a <br /> histone deacetylases (12). When O2 levels low, HIF-&#226;?&#163; recog-                                    facility constructed support Research Facilities Improvement <br /> nition HPH-2 instead promote recruitment chromatin-                                      Program Grant C06 RR-15437. R.K.B. Michael L. Rosenberg <br /> remodeling factors promoter regions HIF target genes                                 Scholar Medical Research. <br />  <br />  <br />  1.   Wenger, R. H. (2002) FASEB J. 16, 1151&#226;&#8364;&#8220;1162.                                                 19. Garkavtsev, I., Kozin, S. V., Chernova, O., Xu, L., Winkler, F., Brown, E., Barnett, <br />  2.   Maxwell, P. H. &amp; Ratcliffe, P. J. (2002) Semin. Cell Dev. Biol. 13, 29&#226;&#8364;&#8220;37.                       G. H. &amp; Jain, R. K. (2004) Nature 428, 328&#226;&#8364;&#8220;332. <br />  3.   Bruick, R. K. (2003) Genes Dev. 17, 2614&#226;&#8364;&#8220;2623.                                               20. Kim, S., Chin, K., Gray, J. W. &amp; Bishop, J. M. (2004) Proc. Natl. Acad. Sci. USA 101, <br />  4.   Huang, L. E., Gu, J., Schau, M. &amp; Bunn, H. F. (1998) Proc. Natl. Acad. Sci. USA 95,              16251&#226;&#8364;&#8220;16256. <br />       7987&#226;&#8364;&#8220;7992.                                                                                   21. Campos, E. I., Chin, M. Y., Kuo, W. H. &amp; Li, G. (2004) Cell. Mol. Life Sci. 61, <br />  5.   Epstein, A. C. R., Gleadle, J. M., McNeil, L. A., Hewitson, K. S., O&#226;&#8364;&#8482;Rourke, J., Mole,           2597&#226;&#8364;&#8220;2613. <br />       D. R., Mukherji, M., Metzen, E., Wilson, M. I., Dhanda, A., et al. (2001) Cell 107, 43&#226;&#8364;&#8220;54.   22. Sheffield, P., Garrard, S. &amp; Derewenda, Z. (1999) Protein Expression Purif. 15, 34&#226;&#8364;&#8220;39. <br />  6.   Bruick, R. K. &amp; McKnight, S. L. (2001) Science 294, 1337&#226;&#8364;&#8220;1340. <br />                                                                                                    23. Bruick, R. K. (2000) Proc. Natl. Acad. Sci. USA 97, 9082&#226;&#8364;&#8220;9087. <br />  7.   Ivan, M., Haberberger, T., Gervasi, D. C., Michelson, K. S., Gu   &#194;&#168;nzler, V., Kondo, K., <br />                                                                                                    24. Kivirikko, K. I. &amp; Myllyla, R. (1982) Methods Enzymol. 82, 245&#226;&#8364;&#8220;304. <br />       Yang, H., Sorokina, I., Conaway, R. C., Conaway, J. W. &amp; Kaelin, W. G. (2002) Proc. <br />       Natl. Acad. Sci. USA 99, 13459&#226;&#8364;&#8220;13464.                                                        25. Wang, F., Zhang, R., Beischlag, T. V., Muchardt, C., Yaniv, M. &amp; Hankinson, O. <br />  8.   Ivan, M., Kondo, K., Yang, H., Kim, W., Valiando, J., Ohh, M., Salic, A., M., A. J.,             (2004) J. Biol. Chem. 279, 46733&#226;&#8364;&#8220;46741. <br />       Lane, W. S. &amp; Kaelin, W. G. (2001) Science 292, 464&#226;&#8364;&#8220;468.                                     26. O&#226;&#8364;&#8482;Rourke, J. F., Pugh, C. W., Bartlett, S. M. &amp; Ratcliffe, P. J. (1996) Eur. J. Biochem. <br />  9.   Jaakkola, P., Mole, D. R., Tian, Y.-M., Wilson, M. I., Gielbert, J., Gaskell, S. J., von         241, 403&#226;&#8364;&#8220;410. <br />       Kriegsheim, A., Hebestreit, H. F., Mukherji, M., Schofield, C. J., et al. (2001) Science     27. Feng, X., Hara, Y. &amp; Riabowol, K. (2002) Trends Cell Biol. 12, 532&#226;&#8364;&#8220;538. <br />       292, 468&#226;&#8364;&#8220;472.                                                                         <br /> </body></html>