<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2005 Genetics Society America <br /> DOI: 10.1534/genetics.105.046557 <br />  <br />  <br />  <br /> TFIID Spt-Ada-Gcn5-Acetyltransferase Functions Probed Genome-wide <br /> Synthetic Genetic Array Analysis Using Saccharomyces cerevisiae taf9-ts Allele <br />  <br />                 Elena Milgrom,1 Robert W. West, Jr.,1 Chen Gao1 W.-C. Winston Shen2 <br />   Department Biochemistry Molecular Biology, State University New York Upstate Medical University, Syracuse, New York 13210 <br />                                                         Manuscript received June 7, 2005 <br />                                                       Accepted publication July 30, 2005 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                TAF9 TATA-binding protein associated factor (TAF) conserved yeast humans shared by <br />              transcription coactivator complexes, TFIID SAGA. The essentiality TAFs difficult <br />              ascertain roles TFIID SAGA function. Here performed genomic synthetic genetic <br />              array analysis using temperature-sensitive allele TAF9 query. Results experiment showed <br />              TAF9 interacts genetically with: (1) genes multiple transcription factor complexes predominantly <br />              involving Mediator, chromatin modification/remodeling complexes, regulators transcription elon- <br />              gation; (2) virtually nonessential genes encoding subunits SWR-C chromatin-remodeling com- <br />              plex TAF9 SWR-C required expressing essential housekeeping gene RPS5; (3) <br />              key genes cell cycle control G1/S transition, genes involved cell polarity, cell in- <br />              tegrity, protein synthesis, suggesting link TAF9 function cell growth control. We also <br />              showed disruption SAGA deletion SPT20 alters histone-DNA contacts phosphorylated <br />              forms RNA polymerase II coding sequences. Our results raise possibility unappreciated role <br />              TAF9 transcription elongation, context SAGA, provide support for <br />              TAF9 involvement cell cycle progression growth control. <br />  <br />  <br />  <br />  <br /> T    RANSCRIPTION initiation RNA polymerase II <br />        involves assembly general transcription <br /> factors core promoter form preinitiation <br />                                                                             tered, Spt8 absent complex (Pray-Grant et al. 2002; <br />                                                                             Sterner et al. 2002), involved transcrip- <br />                                                                             tion subset RNA polymerase II-dependent genes. <br /> complex (PIC). Transcriptional activators bind spe-                      Genetic mutations (Durso et al. 2001; Michel et al. <br /> cific cis-acting promoter elements upstream acti-                    1998) experimental depletion (Moqtaderi et al. <br /> vating sequences (UASs)/enhancers stimulate PIC                         1998) shared TAFs shown disrupt in- <br /> assembly mechanism thought involve direct                      tegrity TFIID SAGA complexes. <br /> interactions components tran-                          Previous systematic analysis function yeast <br /> scription machinery (Taatjes et al. 2004). The step                   TAFs achieved isolating temperature-sensitive <br /> in PIC assembly binding TATA-box-binding                          mutants TAF followed genome-wide expres- <br /> protein (TBP) TFIID TATA box. TFIID multi-                   sion profiling. These experiments revealed selective <br /> subunit complex consisting TBP set TBP-                         requirement TAFs genome-wide transcription <br /> associated factors (TAFs) (Albright Tjian 2000).                        (Apone et al. 1998; Holstege et al. 1998; Lee et al. 2000; <br /> TAFs highly conserved yeast humans. In                          Shen et al. 2003). That is, 13 essential TAFs is <br /> yeast, 14 TAFs identified, 13 re-                    globally required transcription, TAF <br /> quired viability (Green 2000).                                          appears regulate transcription subset genes <br />    Several TAFs associated transcription factor                    yeast genome (Shen et al. 2003). The selective <br /> complexes TFIID. In Saccharomyces cerevisiae,                    requirement TAFs genomic transcription raised <br /> for example, TAF5, TAF6, TAF9, TAF10, TAF12                             possibility TAF specific role tran- <br /> (formerly known Taf90, Taf60, Taf17, Taf 25,                         scriptional regulation. Conceivably, selective require- <br /> Taf61/68, respectively) (Tora 2002) integral                       ment TAFs transcription arise specific <br /> components Spt-Ada-Gcn5-acetyltransferase (SAGA)                     biochemical and/or genetic interactions involving each <br /> complex (Grant et al. 1998) SAGA-like/SAGA al-                          TAF. Supporting possibility, shown by <br />                                                                             immunopurification multi-dimensional mass spec- <br />                                                                             trometry TAF associates distinct set <br />   1 <br />    These authors contributed equally work.                          transcription factors (Sanders et al. 2002). Despite <br />   2 <br />    Corresponding author: Department Biochemistry Molecular           extensive biochemical characterization TAFs <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Biology, State University New York, Upstate Medical University, 750 <br /> E. Adams St., WHA Room 4281, Syracuse, NY 13210.                            different organisms, systematic genetic analysis not <br /> E-mail: shenw@upstate.edu                                                   performed essential nature. <br />  <br /> Genetics 171: 959&#226;&#8364;&#8220;973 (November 2005) <br />  960                                                         E. Milgrom et al. <br />  <br />    Large-scale genetic interaction analyses                  ligated filled-in 932-bp StuI-EcoRI fragment containing <br /> employed various model organisms dissect com-                    TRP1 isolated pJJ246/280 (Jones Prakash 1990). <br />                                                                        The resulting plasmid, pWCS206, contains TRP1-marked <br /> plex biochemical pathways identifying mutations in <br />                                                                        taf9-ts2 allele EcoRI fragment. The 1.83-kb EcoRI frag- <br /> genes enhance (synthetic lethality sick-                ment pWCS206 used transform YPH500 (MATa, <br /> ness growth defects) suppress (extragenic suppres-                  generating WCS123). WCS123 harboring TRP1-marked <br /> sors) <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>mutation</span> known gene                   taf9-ts2 allele mated PSY137 (MATa ade2 ade3 strain; <br /> that pathway. Recent development yeast synthetic                    Koepp et al. 1996), resulting diploid strain spor- <br /> genetic array (SGA) analysis facilitated screen-               ulated dissected isolate haploid strain WCS129. <br />                                                                        WCS129 (MATa ade2 ade3 leu2 ura3 his3 lys2 taf9-ts2-TRP1) <br /> ing process synthetic lethality/sickness utilizing            transformed plasmid pRS416-TAF9-ADE3 (a URA3- <br /> comprehensive collection yeast strains,               marked, centromere-containing plasmid) used the <br /> is devoid nonessential gene ( 4700  6200                     starting strain mutagenesis screening synthetic <br /> total genes; Winzeler et al. 1999). The automation                  lethality monitoring red/white sectoring colonies. <br /> pinning cells plates streamlined identifi-                   The cloning approach creating genetically <br />                                                                        tractable taf9-ts2 allele described used generate <br /> cation synthetic genetic interactions cells bearing             query strain suitable genome-wide SGA screen, except <br /> double mutations (Tong et al. 2001). Synthetic inter-                  NAT1 (a nourseothricin-resistant gene) instead TRP1 <br /> actions usually imply functional connection                  used mark taf9-ts2 allele. The taf9-ts2-NAT1 allele was <br /> the genes involved. For instance, secretory pathway                introduced strain Y3656 (Tong et al. 2001) replace the <br /> of yeast divided 10 different biochemical                endogenous TAF9 gene. The genotype resulting query <br />                                                                        strain, WCS444, MATa can1DTMFA1pr-HIS3-MFa1pr-LEU2 <br /> steps translocation Golgi, maturation ER,                ura3 leu3D his3D lys2D met15D taf9-ts2-NAT1. BY4741 (MATa <br /> etc., synthetic lethal interactions                his3D1 leu2D0 met15D0 ura3D0) parental strain SGA <br /> for genes pathway (Finger Novick                       deletion strains. The strain bearing C-terminal Myc-tagged <br /> 2000). On basis study, 75%                 endogenous TAF9 (WCS311) used chromatin immuno- <br /> interactions subsequently shown involve genes                  precipitation analyses generated previously described <br />                                                                        (Longtine et al. 1998; Gao et al. 2004). <br /> acting identical separate steps                   A genome-wide screen TAF9 genetic interactions by <br /> same pathway (Hartman et al. 2001).                                    SGA: The SGA screen performed essentially previ- <br />    <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> understand TAFs function, sought                  ously described (Tong et al. 2001) temperature- <br /> to identify genes genetically interact TAF9.                 sensitive taf9-ts2 mutation used query strain instead <br /> TAF9 essential component TFIID                       <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>null allele</span> nonessential gene. We performed the <br />                                                                        screen times, candidates scored reproducibly <br /> SAGA (Grant et al. 1998) shown micro-                    positive confirmed tetrad analysis. The confirmatory <br /> array analysis TAF9 required expression                spot test comparing growth taf9-ts2 xxxD double-mutant <br /> of  60% yeast genome. This percentage                    strains vs. taf9-ts2 xxxD strains performed four <br /> highest TAF genes analyzed (Shen et al.                      different temperatures, 25&#194;&#176;, 30&#194;&#176;, 34&#194;&#176;, 37&#194;&#176;, SC and/or <br /> 2003). We report results synthetic lethality/              YPD media. In course studies, determined that <br />                                                                        majority false negatives arise SGA interact <br /> sickness screens using taf9-ts2 temperature-sensitive                weakly taf9-ts2 cause growth defects themselves, <br /> allele query. Both conventional ade2/ade3 sector-               hindering identification. Genetic interactions <br /> ing phenotype (Bender Pringle 1991)-based                          scored strong growth defects evident at <br /> screen SGA-based genome-wide screen per-                    30&#194;&#176; 34&#194;&#176; (Figure 1, red), scored conditional <br /> formed study. To knowledge,              growth defects evident 34&#194;&#176; 30&#194;&#176; <br />                                                                        (Figure 1, blue). <br /> systematic genome-wide genetic analysis essential <br />                                                                           A synthetic lethality screen taf9-ts2 allele ade2/ade3- <br /> transcription factor. The genetic interactions TAF9                 mediated red/white colony sectoring: Ethyl methanesulfonate <br /> revealed significant functional relationship reg-               (EMS) utilized treat strain WCS129. EMS concen- <br /> ulators transcription elongation initiation.             trations 20 30 ml/ml cell culture employed, which <br /> Genetic interactions TAF9 genes                  rendered cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>survival</span> 25 15%, respectively. We screened <br />                                                                         2.2 3 104 viable cells, 443 mutant strains ( 2%) passed <br /> specific cellular processes, including cell cycle progres- <br />                                                                        criterion screen exhibited constitutive red- <br /> sion growth control, uncovered.                          color colony (nonsectoring), indicating failure lose plasmid <br />                                                                        pRS416-TAF9-ADE3. Among 443 candidate strains, 127 strains <br />                                                                        passed second criterion 5-FOA sensitivity, alternative <br />                MATERIALS AND METHODS                                   assay failure lose plasmid pRS416-TAF9-ADE3 (a URA3- <br />                                                                        marked plasmid). As criterion, pRS413-TAF9 (a <br />    Plasmids, yeast strains, genetic methods: Growth             HIS3-marked low-copy plasmid) present, 82 strains 127 <br /> yeast cells rich (YPD) synthetic media (SC, synthetic            candidates grow 5-FOA plates, selected Ura&#195;&#191; <br /> complete) performed according standard procedures               cells. Those 82 strains mated WCS128 (isogenic to <br /> (Sherman 1991). The construction starting strain            WCS129 opposite mating type) analyses of <br /> conventional synthetic lethality screen taf9-ts2 (Apone et al.     dominant/recessive nature EMS-introduced mutations. <br /> 1998) follows. A 0.9-kb EcoRI fragment containing taf9-         If mutations recessive, resulting diploid strains <br /> ts2 isolated plasmid Lp35 (Apone et al. 1998) cloned          expected resistant 5-FOA able form red/ <br /> into pBSKS(1) generate pWCS204. Plasmid pWCS204                 white sectoring phenotype. We recovered 33 diploid strains, <br /> cut BglII site, located 176 bp downstream       candidate strains 12 confirmed as <br /> TAF9 stop codon, blunt ended T4 DNA polymerase,                 recessive 1 dominant. Standard tetrad analysis the <br />                                                 Synthetic Genetic Interactions TAF9                                        961 <br />  <br /> 12 candidates performed determine candidate(s)                                  TABLE 1 <br /> harbored single recessive gene rendered lethality in <br /> combination taf9-ts2 mutation. All spores derived        Genes exhibit synthetic growth phenotype <br /> tetrad analyses temperature sensitive presence                               TAF9 <br /> of taf9-ts2 allele. Sets haploid strains derived from <br /> each tetrad exhibited 2:2 (5-FOAr sectoring:5-FOAs       Category                              Gene <br /> and nonsectoring) segregation <span id='am-396' about='xsp:pattern' typeof='owl:Thing'>pattern</span> 5-FOA sensi- <br /> tivity colony-sectoring assays indication        Chromatin             HDA1, CHD1, IES3, SAP30, PHO23 <br /> presence single recessive mutation.                              structure <br />    To clone corresponding wild-type genes             Mediator              NUT1 <br /> synthetic lethal strains, sl54/WCS151, sl72/WCS148, sl284/          Transcription         SPT4 <br /> WCS154, sl336/WCS157, YEp13-based genomic DNA                   elongation <br /> library transformed strains resulting trans-   Repression            TUP1, SSN6, SKO1 <br /> formants selected restoration red/white colony          Gene-specific         GLN3 <br /> sectoring 5-FOA-resistant phenotypes. Since YEp13             activator <br /> vector high-copy-number plasmid, restoration sec-       Other transcription   MBF1, INO2, RIM9, RIM20, RPI1 <br /> toring 5-FOA-resistant phenotypes reflect dosage            Other gene            CAF40, CDC40, LOC1 <br /> dependency. To exclude possibility, subcloned             expression <br /> genomic insert low-copy plasmid pRS415 re-             Cell cycle            PHO80 <br /> peated phenotypic analysis. Sequencing respec-           Cell polarity         SLA1, SMY1 <br /> tive genomic DNA fragments revealed identities           Protein synthesis     GCN20, RPS7A, RPS12, RPL13B, RPL39 <br /> genes CDC7, RPT5, SAC3, VPS72, corresponding              Protein sorting       VPS4, VPS20, VPS24, VPS28 <br /> to mutations synthetic lethal strains sl54, sl336, sl284,    Protein               DOA1, UBR2, UMP1 <br /> and sl72, respectively.                                               degradation <br />    Northern blot primer extension analyses: Preparation         Mitochondria          FMC1, LIP2, POR1, POS5, YHM1 <br /> of total RNA Northern blot analysis performed           Carbohydrate          GPM2, TPS2 <br /> described (Shen Green 1997). Probes GLN1, RPS5,               metabolism <br /> and TRX1 transcripts obtained PCR amplification             Other cytoplasmic     ICE2, LTV1, NEW1, YOR302W <br /> from genomic DNA described (Shen et al. 2003). Primer            Uncharacterized       FYV5, YMR247C, YPR044C, YDR442W <br /> extension performed described (Shen et al. 2003); <br /> the sequence PDR5 primer 59-CGAAAGTTCCT                    Spot tests used compare growth defects taf9-ts2, <br /> AGTTGCCAAT-39.                                                      xxxD, corresponding double-mutant strains 30&#194;&#176; 34&#194;&#176; <br />    Chromatin immunoprecipitation: The conditions form-          SC media. <br /> aldehyde-based vivo crosslinking chromatin immuno- <br /> precipitation (ChIP) performed essentially described <br /> (Gao et al. 2004) radioactively labeled nucleotide        subsequently confirmed tetrad analysis using stan- <br /> was used PCR, quantification precipitated DNA after <br />                                                                     dard synthetic-complete media drugs were <br /> PCR measured ethidium bromide staining. Strains <br /> bearing Myc-tagged histone H4 source histone         identified. Approximately 50% TAF9-interacting <br /> H4, immunoprecipitations using anti-Myc antibody, sets          genes encode transcription factors. In addition the <br /> of PDR5 primers used PCR described (Gao et al.        SGA screen, conventional ade2/ade3 synthetic lethal <br /> 2004). Primers used ACT1 following:                    screen performed hopes identifying essential <br /> promoter: ACT1-1, 59-CTTATCGGATCCTCAAAACCC-39                   genes interact taf9-ts2 (see materials and <br />   ACT1-2, 59-GGAGAGAGAGAGGCGAGTTTG-39;                          methods). From screen obtained muta- <br /> coding sequences: ACT1-3, 59-GAAGTGTGATGTCGATGTC                    tions genes, essential (CDC7 RPT5) and <br />   CG-39 ACT1-4, 59-CACTTGTGGTGAACGATAGATG-39.                   nonessential (SAC3 VPS72). Notably, neither <br /> Washing conditions immunoprecipitations using anti-Ser5         sac3D vps72D detected taf9-ts2 SGA <br /> C-terminal domain (CTD) antibody (H14 clone) anti-Ser2          screens, conditional synthetic interac- <br /> CTD antibody (H5 clone) based previous descriptions         tions taf9-ts2 34&#194;&#176; subsequently confirmed by <br /> with minor modifications (Komarnitsky et al. 2000). <br />    Genetic nomenclature: Standard nomenclature S. cerevi- <br />                                                                     tetrad analysis. This observation suggested SGA <br /> siae used text TAF proteins           screen missed conditional synthetic <br /> (capitalized according unified nomenclature TBP-         interactions, consistent previous reports indicating <br /> associated factors (Tora 2002).                                     presence false negatives SGA screens (Sarin <br />                                                                     et al. 2004; Tong et al. 2004; materials and <br />                                                                     methods). Genes definitively interact with <br />               RESULTS AND DISCUSSION <br />                                                                     TAF9, studied course experi- <br />    Synthetic lethal screens using taf9-ts2 query: We           ments, provided Table 1. <br /> first performed genome-wide SGA screen col-                   The combined results synthetic lethal/sick <br /> lection  4700 viable deletion strains (Tong et al.               screens summarized Figure 1. Most TAF9-interacting <br /> 2001) using taf9-ts2 allele (Apone et al. 1998) query.         genes nuclear, majority include previously <br /> The screen performed times, time                  characterized transcription factors fall three <br /> duplicate, reproducible candidates                 predominant categories: (1) Mediator; (2) complexes <br /> analyzed. A total 106 genetic interactions             having roles histone modification chromatin <br />  962                                                       E. Milgrom et al. <br />  <br />  <br />  <br />  <br />    Figure 1.&#226;&#8364;&#8221;Genome-wide synthetic interactions involving TAF9. Genes transcription factors classified major <br /> groups: Mediator, chromatin-modifying complexes, regulators transcription elongation. Yellow ovals denote sets genes <br /> that encode transcription factors demonstrated biochemically associate TAFs (see text). Genes marked in <br /> red indicate strong genetic interactions; marked blue indicate conditional synthetic interactions. Underlined genes are <br /> present multiple categories. Asterisks denote essential genes identified conventional synthetic lethality screen. Solid <br /> lines denote functional connections involving specific transcription factors associated particular cellular processes (see text). <br />  <br />  <br /> remodeling; (3) factors involved transcription                 ing robust relationship Mediator TAF9 <br /> elongation. While interactions catego-             function. This consistent reported biochemical <br /> ries transcription factors expected, remarkably           interactions Mediator TFIID (Sanders <br /> few genes roles gene expression downstream                 et al. 2002), SAGA-dependent recruitment of <br /> transcription elongation identified screens.              Mediator GAL1 UAS (Larschan Winston <br /> Cellular processes gene expression                2005). While Srb8-11 module Mediator thought <br /> overrepresented screens, however, include in-              play predominantly negative role transcription <br /> tegrated processes cell cycle, cell polarity, cell             (Bjorklund Gustafsson 2005), evidence suggests <br /> integrity protein synthesis protein sorting.           play positive role (see Larschan and <br /> Below, class TAF9-interacting gene described            Winston 2005 references therein). Isolation our <br /> more detail.                                                          screen subunits Srb8-11 module as <br />    Mediator: Mediator general transcription factor               TAF9 interactors consistent with <br /> (Kornberg 2005b) composed core Mediator (21 sub-                   possibility. Interestingly, genes for <br /> units) plus negative regulatory module designated                   subunits tail module, of are <br /> Srb8-11 (four subunits) (Bjorklund Gustafsson                     nonessential, obtained SGA screen. <br /> 2005; Kornberg 2005a). Core mediator itself com-                      Chromatin modification (SAGA, HAT complexes, <br /> posed distinct modules called &#226;&#8364;&#732;&#226;&#8364;&#732;head,&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;mid-               chromatin-remodeling complexes, Rpd3-HDAC complex, <br /> dle,&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;tail&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Chadick Asturias 2005). The head             Asf1): SAGA  15-subunit histone acetyltransfer- <br /> module contacts RNA polymerase II PIC,                 ase (HAT) complex (Grant et al. 1998) harbors the <br /> surprisingly, of respective subunits             HAT enzyme Gcn5, TAF9 itself. Not surpris- <br /> essential. One gene head core module                 ingly, gcn5 mutations multiple com- <br /> from middle identified TAF9 interactors,               ponents SAGA, isolated SGA screen. <br /> were genes Srb8-11 module, suggest-                 Other HAT complexes S. cerevisiae include ADA, <br />                                            Synthetic Genetic Interactions TAF9                                        963 <br />  <br /> NuA3, NuA4, Spt10, SAS, TFIID, Hat1, Elongator             enzyme Elongator, nucleosome-remodeling com- <br /> (Brown et al. 2000), SGA screen identified          plex SWR-C (including Swr1, Bdf1 kinase, H2A.Z), <br /> least gene complexes, Hat1,        E2-ubiquitinating enzyme Rad6, ubiquiti- <br /> suggesting tight correspondence func-            nates histone H2B, nonessential PolII subunits <br /> tion TAF9 HAT enzymes general.                       Rpb9 Rpb4, 19S proteosomal cap complex. <br />    Chromatin-remodeling complexes S. cerevisiae in-         Notwithstanding, SPT4, encoding subunits <br /> clude Swi/Snf, RSC, Ino80, SWR-C, Chd1, ISWI               (including SPT5, essential) elongation <br /> (ISW1a, ISW1b, ISW2) (Narlikar et al. 2002). Rep-          factor DSIF (Winston 2001), non- <br /> resentative members of complexes,         essential genes class does syn- <br /> ISWI Chd1, obtained SGA screen.                thetic relationship TAF9 (Table 1). These results raise <br /> This suggests that, Mediator HAT complexes,         possibility addition well-established role <br /> a robust functional association exists TAF9            transcription initiation, TAF9 play important <br /> and chromatin-remodeling complexes. The absence             previously unappreciated role, directly indirectly, <br /> genetic interactions Chd1 ISWI reflect            transcription elongation. <br /> unique functions specific chromatin-remodeling           The SWR-C complex, containing Snf2 ATPase <br /> complexes.                                                     family member Swr1, plays role exchanging the <br />    Rpd3-HDAC histone deacetylase             histone variant H2A.Z (Htz1) conventional H2A <br /> (HDAC) complexes S. cerevisiae,        (Mizuguchi et al. 2004). While modification has <br /> are ordinarily associated gene silencing (Kurdistani      shown promote gene expression near silent <br /> and Grunstein 2003). While HDAC complexes                 heterochromatin (Meneghini et al. 2003), SWR-C also <br /> were identified genetic interactors TAF9, genes    interacts genetically multiple transcription elonga- <br /> for subunits Rpd3-HDAC complex, Rpd3,             tion factors, suggesting specific role transcription <br /> Sin3, Sds3, obtained. Two different possibilities     elongation (Krogan et al. 2003). Interestingly, Htz1 also <br /> might account result. First, Rpd3-HDAC re-        proposed play role transcriptional <br /> cently shown involved specific instances gene      elongation (Santisteban et al. 2000). In regard, <br /> activation addition role silencing (De Nadal      relationship TAF9 SWR-C especially <br /> et al. 2004; Kurdistani et al. 2004). Second, genetic        noteworthy, seven nonessential subunits of <br /> relationship Rpd3-HDAC TAF9 could,              purified SWR-C complex (Figure 1; Krogan et al. <br /> principle, involve activation repression     2003) identified SGA screen, including <br /> mechanisms regard different genes. Identification        Htz1. <br /> of direct biochemical interactions Rpd3-HDAC              The Paf1 complex physically functionally asso- <br /> and TFIID (Sanders et al. 2002) consistent         ciates specific elongation factors, including DSIF <br /> genetic interactions reported here.                            FACT (Spt16 Pob3; Squazzo et al. 2002). <br />    The histone chaperone Asf1 shown ac-            Moreover, Paf1 essential histone H2B monoubiq- <br /> tivate PHO5 PHO8 virtue nucleosome               uitination Rad6-Bre1-Lge1 complex, sub- <br /> disassembly function (Adkins et al. 2004). Genome-wide         sequently signals histone methylation Lys4 the <br /> transcriptional analysis yeast lacking Asf1 shows        Set1/COMPASS complex early stage elongation <br /> is involved activation repression          (Wood et al. 2003). Results Madhani col- <br /> genes (Zabaronick Tyler 2005). Identification           leagues (Hwang et al. 2003) suggest H2B ubiquiti- <br /> ASF1 synthetic interactor TAF9 consistent         nation H2A.Z substitution redundant functions <br /> with reported biochemical genetic interactions             formation active chromatin. Strong condi- <br /> between Asf1 TFIID (Chimura et al. 2002) sug-          tional synthetic interactions identified involving <br /> gests close relationship factors con-       members Paf1 complex, Rad6, Set1/ <br /> trol gene expression.                                       COMPASS complex, addition SWR-C H2A.Z, <br />    Transcription elongation (TFIIS, SWR-C, H2A.Z, Bdf1         raise possibility role TAF9 transcription <br /> kinase, Paf1 complex, Rad6, Set1/COMPASS, Elongator,           elongation related redundant functions H2B <br /> CTD kinase, Rpb9 Rpb4 PolII subunits, 19S cap): A          ubiquitination H2A.Z substitution. Supporting this <br /> wide variety factors control transcription elon-       possibility fact Paf1 complex biochemi- <br /> gation RNA polymerase II recently                 cally interacts TFIID (Sanders et al. 2002). <br /> characterized (Sims et al. 2004 references therein).          Elongator, transcription-elongation-specific HAT <br /> It intriguing TAF9 showed strong (at 30&#194;&#176;)       enzyme (Gilbert et al. 2004), interacts genetically with <br /> or conditional (at 34&#194;&#176;) synthetic interactions     TAF9 Elp1 subunit (Figure 1). This consistent <br /> all factors. They include TFIIS (Dst1), PolII        biochemical interactions TAF9 six <br /> C-terminal domain (CTD) kinase CTDK-I (composed                subunits Elongator (Sanders et al. 2002). <br /> of Ctk1, Ctk2, Ctk3), Paf1 complex (including             CTDK-I kinase phosphorylates RNA po- <br /> Paf1 Cdc73), Set1-containing histone methyla-          lymerase II CTD, subsequently affecting transcrip- <br /> tion complex COMPASS, elongation-specific HAT              tion elongation pre-mRNA processing (Ahn et al. <br />  964                                                   E. Milgrom et al. <br />  <br /> 2004). CTK1 encodes catalytic subunit kinase,        TAF9. Interaction TAF9 far global <br /> which hyperphosphorylates serine 2 PolII CTD              transcription factors gene-specific factors consis- <br /> (Lee Greenleaf 1991), affecting RNA                  tent notion TAFs function as <br /> polymerase II elongation efficiency. Deletion CTK1            general coactivators (Shen Green 1997). <br /> prevents Set2 recruitment, required Lys36 methyla-              Other stages gene expression: poly(A) tail formation, <br /> tion, necessary addition Lys4 methylation         mRNA export, nuclear transport, mRNA turnover: In con- <br /> by Set1/COMPASS transcription elongation (Krogan             trast results, hits involving factors with <br /> et al. 2002). All subunits kinase complex          roles gene expression downstream transcription <br /> (Ctk1, Ctk2, Ctk3) identified SGA screen         obtained. The fact interactions <br /> as TAF9 interactors. Additionally, nonessential sub-        identified, however, suggests correspondence <br /> units PolII, Rbp9 Rpb4, functionally            function TAF9 downstream stages of <br /> related TFIIS (Wery et al. 2004), interacted        gene expression. For example, identification REF2 <br /> TAF9.                                                            SGA, encodes protein involved poly(A) <br />    The 19S regulatory cap 26S proteasome              tail formation (Russnak et al. 1995), implicates link <br /> been shown play important roles transcription              transcription initiation termination. In- <br /> elongation (Ferdous et al. 2001) initiation        deed, widespread evidence supports coupling of <br /> (Gonzalez et al. 2002). Biochemical interactions be-             processes (Proudfoot 2004), including the <br /> tween components 19S cap general tran-                fact Ref2 interacts biochemically TFIID <br /> scription factors TFIID reported          (Sanders et al. 2002). Both Npl3 (Lei et al. 2001) and <br /> (Sun et al. 2002). RPT5 encodes ATPase                Sac3 (Lei et al. 2003) obtained interactors <br /> components located &#226;&#8364;&#732;&#226;&#8364;&#732;<span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span>&#226;&#8364;&#8482;&#226;&#8364;&#8482; module 19S             TAF9, suggesting transcription coupled to <br /> cap. All subunits essential; identification         mRNA processing mRNA export, consistent with <br /> of RPT5 interactor TAF9 using              current notions nuclear organization function <br /> conventional ade2/ade3 screen. While identification              (Maniatis Reed 2002). On basis regu- <br /> of RPT5 interactor TAF9 consistent            latory integration predicted (Maniatis Reed 2002), <br /> a proposed role 19S cap transcription, in-             large number proteins known involved <br /> volvement 26S proteasome wide range               mRNA processing mRNA export (Burckin <br /> cellular processes, however, raises possibility         et al. 2005), imagined hits between <br /> this interaction likewise result transcription-         TAF9 genes categories. Nevertheless, four <br /> independent mechanisms.                                          hits obtained genes encoding proteins with <br />    Gene-specific activators (Swi6, Zap1, Gcr2): On basis     functions involved nuclear transport, including <br /> of original coactivator hypothesis function           Mog1, Soy1, Msn5, Nup188. Recent studies suggest <br /> of TAFs (Dynlacht et al. 1991) fact .200            transcriptional activation S. cerevisiae occur <br /> gene-specific activators identified S.              nuclear periphery involve contributions <br /> cerevisiae, nonessential (Harbison             nuclear pore proteins (see Casolari et al. 2005; <br /> et al. 2004), synthetic interactions multiple gene-         Menon et al. 2005; references therein). In re- <br /> specific activators anticipated. How-            gard, intriguing, light results, TAF9 <br /> ever, gene-specific activators identified        interacts physically Nup57 (Uetz et al. 2000; Menon <br /> in investigation. Identification swi6          et al. 2005). Further support possible involve- <br /> them consistent results previous synthetic          ment TAFs nuclear transport derives the <br /> lethality screen swi6D used query              finding mutation Schizosaccharomyces pombe TAF7 <br /> (Macpherson et al. 2000). It likewise consistent         causes poly(A)1 mRNAs accumulate nucleus <br /> biochemical interaction TAF9 Swi6                    (Shibuya et al. 1999). <br /> (Sanders et al. 2002). Swi6 binds Swi4                  In addition identification nuclear genes in- <br /> Mbp1 form heterodimeric sequence-specific                 volved gene expression, synthetic interactions were <br /> transcription factors SBF MBF, respectively,            TAF9 genes NOT4 LSM1, <br /> key regulators transcription genes involved         encode components cytoplasmic mRNA <br /> the progression G1 S phase (Macpherson et al.            turnover apparatus. mRNA turnover yeast requires <br /> 2000). In regard, connections cell cycle         highly conserved Ccr4-Not complex, predomi- <br /> control identified study (see below) support           nant cytoplasmic mRNA deadenylase (Parker and <br /> prevalence mechanisms linking TAF9 cell cycle              Song 2004), Not4 component. Synthetic <br /> control.                                                         genetic interaction NOT4 TAF9 in- <br />    A genetic relationship TAF9 ZAP1              dicate altered stoichiometry cellular <br /> be consequence large number zinc-regulated           mRNA population, resulting mRNA stabilization <br /> genes S. cerevisiae (Eide 2000). Gcr2 involved          not4 combination reduced mRNA abun- <br /> expression glycolytic genes, essentiality glycol-      dance taf9-ts2, detrimental cell growth. <br /> ysis underlie conditional synthetic relationship         Alternatively, given Ccr4/Not complex has <br />                                              Synthetic Genetic Interactions TAF9                                       965 <br />  <br /> been reported play specific roles transcrip-          cell wall stability (Harrison et al. 2004). SWI6 appears to <br /> tion initiation (Badarinarayana et al. 2000) elon-           critical function pathway (Madden <br /> gation (Denis et al. 2001), shown                et al. 1997), genes, BCK1 SLT2, <br /> interact biochemically functionally TAF1           isolated conditional synthetic interactors <br /> (Deluen et al. 2002), possible synthetic in-          TAF9 (Figure 1). BCK1 SLT2 encode key <br /> teraction NOT4 TAF9 results inter-              protein kinases, MAPKKK (MEK kinase) MAP <br /> actions transcription.                                        kinase (Harrison et al. 2004), respectively, func- <br />    Cell cycle: Coordination transcription            tion linear pathway downstream Pkc1. The <br /> cell cycle control S. cerevisiae established (see     MAP kinase Slt2, particular, required recruiting <br /> Breeden 2003 references therein). In fact, pre-              heterodimeric sequence-specific transcription fac- <br /> vious studies supported involvement specific              tor SBF (Swi6 Swi4) promoters G1 cyclin <br /> TAFs cell cycle control (Apone et al. 1996; Walker            genes conditions cell stress, process may <br /> et al. 1997; Reese Green 2001), including fact           involve direct interactions Slt2 DNA- <br /> that TAF9 interacts SWI6 genetically biochemi-          binding component SBF, Swi4 (Baetz et al. 2001). <br /> cally (Figure 1; Macpherson et al. 2000; Sanders et al.          Collectively, synthetic interactions TAF9 key <br /> 2002). Identification SGA screen MSN5, en-             genes involved cell cycle, cell polarity, cell integ- <br /> coding karyopherin required proper nucleocyto-             rity pathways support major role TAF9 coordi- <br /> plasmic transport Swi6 (Queralt Igual 2003),              nating transcription cell growth. <br /> provides additional support notion,                Protein synthesis: A significant number TAF9- <br /> linking TAF9 control G1/S transition Swi6          interacting genes involved protein synthesis <br /> localization. Synthetic genetic interaction TAF9            identified SGA, majority encoding ribosomal <br /> may involve requirement TAF9, Swi6,              proteins, seven of 40S subunit specific <br /> for activating G1 cyclin transcription (Macpherson et al.        (RPS24A, RPS6A, RPS10A, RPS10B, RPS18A, RPS19B, <br /> 2000).                                                           RPS27B, RPL27A; Figure 1). While ribosomal pro- <br />    Other genes identified study potentially link-        tein genes commonly obtained hits SGA screens, <br /> ing TAF9 cell cycle control include CDC7, SIT4,           higher-than-average number obtained TAF9 <br /> BUB1. CDC7, isolated conventional ade2/ade3               raises question TAF9 specifically <br /> screen, encodes essential Dbf4-dependent kinase               involved process couples transcription trans- <br /> required G1/S transition role activating      lation. In regard, noteworthy genes 40S <br /> prereplication complexes (Lei Tye 2001). Lethality           ribosomal proteins S. cerevisiae <br /> of taf9-ts2 cdc7 double mutants unlikely tran-         genes identified TAF dependent, specifically requir- <br /> scriptional downregulation CDC7, transcription          ing TAF1 (Shen Green 1997) and, shown below, <br /> of CDC7 independent TAF9 basis micro-            TAF9. Moreover, TFIID required transcription of <br /> array analysis (Holstege et al. 1998). SIT4 encodes              ribosomal protein gene regulon (Mencia et al. 2002), <br /> a type 2A-related serine-threonine phosphatase              involving 137 ribosomal protein genes, accounts <br /> functions G1/S transition cell cycle               50% total mRNA vivo (Warner 1999). Since <br /> (Sutton et al. 1991). In summary, synthetic interactions         ribosomal proteins produced duplicate <br /> with SWI6, MSN5, CDC7, SIT4 strongly support               genes S. cerevisiae (Warner 1999), strong condi- <br /> specific role TAF9 regulating G1/S transition.        tional synthetic interactions conceivably derive <br />    Cell polarity/cell integrity: Related results    simply insufficient abundance proteins <br /> was identification large number TAF9-                absence TAF9. Alternatively, been <br /> interacting genes having roles cell polarity, including       shown translation coupled cell cycle control <br /> BEM1, BEM4, CLA4, MDM20, SHE4, STE20, VRP1.                  ribosomal proteins Rpl11 Rpl23 (Dez <br /> This consistent tight coupling cell polarity          Tollervey 2004), analogous mecha- <br /> with cell cycle control (Lew Reed 1993)           nism couples translation transcription involving <br /> explained, part, requirement G1            ribosomal proteins. In regard, it <br /> cyclins normal cell morphogenesis (Moffat                noteworthy that, addition RPS24A iso- <br /> Andrews 2004). Remarkably, genes, BEM1,            lated TAF9 interactor (Figure 1), Rps24A shown <br /> BEM4, CLA4, STE20, encode proteins interact             co-immunopurification mass spectrometry anal- <br /> during G1 key small GTPase, Cdc42, trig-            ysis interact biochemically TFIID (Sanders et al. <br /> gers polarized assembly actin cytoskeleton                2002). <br /> (Chang Peter 2003).                                             Protein sorting: We identified different genes <br />    Coinciding processes cell polarity            interacting TAF9 products play roles related <br /> cell cycle control cell integrity pathway, Pkc1-        protein sorting, including DOA4, MNN10, VAC14, <br /> mediated MAP kinase signal transduction cascade in-              VAN1, VMA10, VMA11, VPS3, VPS18 (Figure 1). <br /> duced periods morphogenetic growth              Connections transcription protein sort- <br /> response environmental conditions jeopardize             ing described previously. For example, five <br />  966                                                  E. Milgrom et al. <br />  <br /> (Aor1, Vps71, Yaf9, Arp6, Vps72) seven non-              role SAGA elongation (Van Mullem et al. <br /> essential subunits SWR-C Bdf1          2002; Rodriguez-Navarro et al. 2004; Wery et al. 2004), <br /> in large-scale screen identify genes required          including fact SAGA physically functionally <br /> vacuolar protein sorting (vps genes; Bonangelino et al.         interacts transcription elongation factor <br /> 2002), suggesting SWR-C specifically required       TFIIS (Wery et al. 2004) PolII subunit Rpb9, <br /> the expression vps genes (Krogan et al. 2003). The           linked elongation control (Van <br /> fact TAF9 interacts genetically protein-        Mullem et al. 2002). <br /> sorting genes seven nonessential            Hence, examine possible involvement of <br /> SWR-C genes raises possibility TAF9 plays            SAGA transcriptional elongation, used ChIP to <br /> special role vps gene expression alongside SWR-C.            analyze integrity nucleosomes coding <br />    Other nuclear/other cytoplasmic: Four genes pro-       sequence gene known controlled SAGA, <br /> ducts localize nucleus functions           PDR5 (Gao et al. 2004). Previous studies shown that <br /> unclear include FYV6, IWR1, NCE4, YPR045C. Genes            nucleosomes partially disassemble loss histone <br /> whose products localize cytoplasm func-        H3-H4 tetramers genes undergoing transcription. <br /> tions unclear include FSH2, ILM1, RIM13, RIM21,        This disruption detected loss signal when <br /> YOR322C, YDR532C discussed.                 epitope-tagged versions respective histones, pre- <br />    Mitochondrial genes interacting synthetically           sent place normal histones, probed with <br /> TAF9 include MRP6, MRP10, MRP20, MSU1, FYV4.                corresponding antibodies ChIP (Kristjuhan <br /> MRP6, MRP10, MRP20 encode subunits                   Svejstrup 2004; Lee et al. 2004; Schwabish and <br /> mitochondrial ribosome, strong conditional syn-          Struhl 2004). For assay used antibodies against <br /> thetic interactions genes raise            Myc-tagged histone H4, source cellular his- <br /> question connection exists nuclear         tone H4 (Gao et al. 2004; materials methods), <br /> transcription mitochondrial protein synthesis. While        probe integrity nucleosomes PDR5 <br /> FYV4 encodes deduced protein unknown function,             promoter coding sequences wild-type strain <br /> originally isolated virtue hypersensitivity     vs. strain harboring deletion SPT20, re- <br /> fyv4 strain K1 killer toxin (Page et al. 2003),       quired integrity SAGA complex (Sterner <br /> recently shown dst1 (TFIIS) rpb9 (PolII           et al. 1999). Activation PDR5 requires binding the <br /> subunit) cause synthetic phenotype combination             activator protein encoded PDR1 upstream <br /> with fyv4 (Malagon et al. 2004). The fact TAF9 ge-         sites called PDREs, shown Figure 2A. Activation of <br /> netically interacts FYV4, DST1, RPB4 (Figure 1),       PDR5 increased ways: addi- <br /> and DST1 RPB9 linked transcription         tion drug (such cycloheximide) second by <br /> elongation, supports role FYV4 control          mutation PDR1 activator gene constitutively <br /> nuclear transcriptional elongation RNA PolII.                active form, designated pdr1-3. As shown Figure 2B in <br />    Spt20 (a SAGA subunit) required transcrip-            pdr1-3 strain, characterized resistance to <br /> tional elongation PDR5: The large number             drugs cycloheximide, PDR5 expressed a <br /> TAF9 interactors involving complexes responsible            significantly higher degree corresponding <br /> transcription elongation (Figure 1) raised question         PDR1 strain (with drug treatment). This in- <br /> of TFIID SAGA play role transcrip-          creased expression involves enhanced recruitment of <br /> tion elongation addition initiation. Although ge-         specific transcription coactivators SAGA (Gao <br /> netic interaction TAF9 particular elongation            et al. 2004) associated loss of <br /> factors SGA screens result combined             contacts histone H4 DNA PDR5 <br /> block initiation elongation es-           promoter region coding sequences regardless of <br /> sential genes, remained possible rep-         presence drug, shown Figure 2C (left) (see <br /> resent interactions direct nature. Indeed,            Gao et al. 2004). However, pdr1-3 spt20D strain, <br /> Berger colleagues (Ingvarsdottir et al. 2005)               PDR5-coding sequences appear associated with <br /> recently showed SGA analysis genes encoding             wild-type levels Myc-histone H4, promoter <br /> two SAGA-specific components, Spt3 Spt8, interact           region remains depleted (Figure 2C, right, sum- <br /> synthetically genes transcrip-        mary histogram right). This suggests that <br /> tion elongation proteins identified interacting         Spt20, likely SAGA, bound PDR5 promoter <br /> with TAF9, Mediator            (Gao et al. 2004), required loss histone- <br /> chromatin modification/remodeling complexes (Figure             DNA contacts PDR5-coding sequences, would <br /> 1). This raises possibility TAF9-interacting           likely impact transcription elongation (Gao et al. 2004; <br /> genes regulators transcription elongation         Kristjuhan Svejstrup 2004; Schwabish and <br /> represent SAGA-specific interactions,           Struhl 2004). Supporting possibility, found <br /> Mediator chromatin modification/remodeling             ChIP analysis PDR5-coding sequences, RNA <br /> complexes largely represent TFIID-specific interactions.        polymerase II containing phosphorylated serine 2 (Ser2) <br /> Supporting notion, studies consistent          CTD large subunit significantly less <br />                                          Synthetic Genetic Interactions TAF9                                        967 <br />  <br />  <br />  <br />  <br />                                                                                       Figure 2.&#226;&#8364;&#8221;Spt20 required for <br />                                                                                    transcriptional elongation through <br />                                                                                    PDR5. (A) Diagram PDR5, show- <br />                                                                                    ing Pdr1 recognition elements <br />                                                                                    (PDREs), TATA box, coding <br />                                                                                    sequences (CDS) relative tran- <br />                                                                                    scription start site (11). The pro- <br />                                                                                    moter CDS drawn different <br />                                                                                    scales. (B) Primer extension analysis <br />                                                                                    PDR5 mRNA levels wild type <br />                                                                                    (PDR1) drug-resistant (pdr1-3) <br />                                                                                    strains (1) (&#195;&#191;) cyclo- <br />                                                                                    heximide (CYH) induction. (C) ChIP <br />                                                                                    Myc-tagged histone H4 bound at <br />                                                                                    PDR5 sequences PDR1, pdr1-3, <br />                                                                                    pdr1-3 spt20D strains. Inset (be- <br />                                                                                    low) shows PDR5 RNA levels pdr1-3 <br />                                                                                    vs. pdr1-3 spt20D strains; SCR1 served <br />                                                                                    loading control. Relative immu- <br />                                                                                    noprecipitate/input ratio ChIP ex- <br />                                                                                    periments quantified, the <br />                                                                                    results summarized histogram <br />                                                                                    (bottom right). (D) ChIP phos- <br />                                                                                    phorylated serine 5 (at TATA sequen- <br />                                                                                    ces) vs. serine 2 (at coding sequences) <br />                                                                                    CTD RNA polymerase II <br />                                                                                    large subunit wild-type PDR1 <br />                                                                                    strain CYH induction. PDR5 <br />                                                                                    (a SAGA-dependent gene) ACT1 <br />                                                                                    (a TFIID-dependent gene) ana- <br />                                                                                    lyzed. (E) ChIP C-terminal Myc- <br />                                                                                    tagged TAF9 promoters and <br />                                                                                    coding sequences PDR5 ACT1 <br />                                                                                    wild-type PDR1 strain without <br />                                                                                    CYH induction. Accompanying histo- <br />                                                                                    grams D E quantitation. <br />                                                                                    Standard errors ChIP experiments <br />                                                                                    ,30%. <br />  <br />  <br />  <br />  <br /> abundant spt20D strain corresponding       defective transcription initiation, proportional <br /> wild-type (SPT20) strain. Ser2 phosphorylated pre-        loss Ser5-phosphorylated CTD PDR5 TATA sequen- <br /> dominantly elongation phase transcrip-         ces substantially Ser2-phosphorylated <br /> tion, Ser5 phosphorylated predominantly             CTD PDR5-coding sequences corresponding <br /> during initiation phase (Komarnitsky et al. 2000).       strains (Figure 2D). By contrast, ACT1, belongs to <br /> This effect spt20D simply consequence      class TFIID-dependent genes (Huisinga Pugh <br />  968                                                 E. Milgrom et al. <br />  <br /> 2004), demonstrates comparable levels spt20D            Spt5 (DSIF mammals). Conceivably, TAF9 Chd1, <br /> SPT20 strains Ser5-phosphorylated CTD               occupying neighboring domains SAGA and <br /> ACT1 TATA sequences Ser2-phosphorylated CTD             bound nuclear transcription factory, provide <br /> ACT1-coding sequences (Figure 2D). We conclude            discrete functions temporally spatially <br /> the function Spt20, presumably SAGA,          distinct formation mRNA transcript. In <br /> restricted promoter regions SAGA-dependent               manner, TAF9 physically associated with <br /> genes extends coding sequences well.                 Chd1 functionally associated various transcrip- <br />    To determine direct role TAF9 play           tion elongation factors, affiliated itself pro- <br /> elongation, compared recruitment pro-              moter sequences. The observation chd1D nor <br /> moters vs. coding sequences PDR5 ACT1 (Figure           spt4D exhibits synthetic genetic interaction taf9-ts2 <br /> 2E). As expected, consistent previous analyses        (Table 1), coupled fact TAF9 recruited <br /> of SAGA subunits (Gao et al. 2004), Myc-tagged           primarily promoters (Figure 2E) Chd1 coding <br /> TAF9 recruited significantly PDR5 UASs              sequences (Simic et al. 2003), suggests Chd1 and <br /> (PDREs) PDR5-coding sequences (Figure 2E)              TAF9 play disparate roles SAGA function and <br /> and ACT1 TATA sequence ACT1-coding se-              transcription. Third, SAGA act conventional <br /> quences (Figure 2E). Notwithstanding, level re-         fashion coactivator promoter sequences during <br /> cruitment Myc-tagged TAF9 PDR5-coding sequences          preinitiation/initiation travel tran- <br /> is measurably greater no-Myc tag control,     scription machinery template elonga- <br /> raising possibility TAF9 context SAGA       tion, Chd1 participates directly TAF9 <br /> contacts coding sequences PDR5 pro-          indirectly elongation-related processes. Fourth, <br /> moter. While Myc-tagged TAF9 showed higher-than-          TAF9 acts independently SAGA TFIID tran- <br /> background recruitment ACT1-coding sequences,           scription elongation, component an- <br /> the difference Myc-tagged TAF9 no-Myc          protein complex. The low molecular weight of <br /> tag control experimental error ( 30%; Figure        TAF9 (17 kD), example, caused elude <br /> 2 legend). This suggests TAF9, presumably          detection protein complex previously affiliated <br /> context SAGA, contact coding sequences          transcription elongation. Given fact the <br /> as promoters, albeit lower efficiency.                    taf9-ts2 mutant used study insensitive the <br />    A role SAGA transcription elongation             transcription elongation inhibitor 6-azauracil (data not <br /> not unwarranted. For example, SAGA contains             shown), however, fourth possibility unlikely. <br /> distinct chromatin-modifying activities, histone acetyla-      Regardless molecular mechanism, culmina- <br /> tion involving Gcn5 histone deubiqutination in-            tion results, particularly light others <br /> volving Ubp8, required transcription          (Ingvarsdottir et al. 2005; Kong et al. 2005), suggests <br /> elongation certain genes. In regard, Rad6-         TAF9 SAGA contribute control tran- <br /> Bre1-Lge1 complex required ubiquitinating histone       scription elongation addition transcription initiation. <br /> H2B, cycle histone ubiquitination fol-                   TAF9 SWR-C required express es- <br /> lowed deubiquitination required yeast GAL            sential housekeeping gene RPS5: While work was <br /> gene transcription, SAGA plays role (Henry          progress SWR-C complex characterized by <br /> et al. 2003; Xiao et al. 2005). A putative role SAGA       different studies (Krogan et al. 2003; Kobor et al. <br /> and TAF9 transcription elongation involve             2004; Mizuguchi et al. 2004). Due large number <br /> direct indirect interactions coding sequences.         hits obtained SGA screen involving members <br /> Consider following alternative possibilities: First,       complex (virtually nonessential subunits of <br /> SAGA, bound UASs core promoters, recruit             13-subunit complex; Krogan et al. 2003), wanted <br /> elongation factors preinitiation complex. By       study relationship TAF9. We asked <br /> an indirect mechanism SAGA thought recruit,          SWR-C complex required express- <br /> its Sus1 subunit, Sac3-Thp1 complex involved            ing genes known require TAF9 expression. <br /> mRNA export (Rodriguez-Navarro et al. 2004). Sec-              For purpose, analyzed expression RPS5 <br /> ond, transcription derives DNA templates thread-       (Shen Green 1997) TRX1 (Shen et al. 2003). <br /> ing PolII enzymes immobilized transcription         RPS5 encodes small ribosomal protein subunit is <br /> factories (Cook 1999), opportunities direct cooper-        essential cell growth, TRX1 encodes thiore- <br /> ation SAGA traditional elongation factors          doxin, nonessential. GLN1 (Shen et al. 2003), <br /> may arise following transcription initiation. In re-      transcription insensitive inactivation TAFs, <br /> gard, recent identification chromodomain pro-           used control experiment. Northern blot <br /> tein Chd1 subunit SAGA (Pray-Grant et al. 2005)        analysis performed total RNA collected a <br /> is particular interest, Chd1 shown        taf9-ts2 strain strains harbor null muta- <br /> localize coding sequences transcribed genes (Simic       tions genes different members SWR-C <br /> et al. 2003) and, moreover, interacts biochemically       complex, SWR1, VPS72, AOR1. Expression the <br /> and genetically elongation factor complex Spt4-       corresponding mutant strains examined before <br />                                             Synthetic Genetic Interactions TAF9                                         969 <br />  <br />  <br />  <br />  <br />                                                                                           Figure 3.&#226;&#8364;&#8221;Conditional synthetic <br />                                                                                        genetic interactions TAF9 <br />                                                                                        SWR-C. (A) Northern blot analy- <br />                                                                                        sis showing RPS5 TRX1 (TAF de- <br />                                                                                        pendent) GLN1 (TAF <br />                                                                                        independent) RNA levels single <br />                                                                                        vs. double mutants. (Bottom) Histo- <br />                                                                                        gram shows respective mRNA ratios <br />                                                                                        (34&#194;&#176;/25&#194;&#176;). Cells grown per- <br />                                                                                        missive temperature (25&#194;&#176;) or <br />                                                                                        shifting semipermissive <br />                                                                                        temperature (34&#194;&#176;) 45 min. (B) <br />                                                                                        Western blot analysis rabbit poly- <br />                                                                                        clonal antibodies showing TAF9 pro- <br />                                                                                        tein levels strains indicated A. <br />                                                                                        Glucose-6-phosphate dehydrogenase <br />                                                                                        (G6PDH) served loading con- <br />                                                                                        trol. (C) Genetic interactions be- <br />                                                                                        tween taf9-ts2 null alleles of <br />                                                                                        SWR1, VPS72, AOR1 assayed a <br />                                                                                        10-fold dilution series spot test on <br />                                                                                        SC media. Images recorded af- <br />                                                                                        ter 2 days incubation indicated <br />                                                                                        temperatures. <br />  <br />  <br />  <br />  <br /> and shift temperature 25&#194;&#176; 34&#194;&#176; (semi-         swr1D taf9 vps72D taf9 strains, corresponding a <br /> permissive taf9-ts2). Figure 3A shows RPS5         greater growth defect compared the <br /> TRX1 mRNA levels modestly reduced swr1D,           (Figure 3C). Less detectable effect the <br /> vps72D, aor1D strains corresponding wild-type           double mutations TRX1 RPS5 to <br /> strain shift semipermissive temperature          fact taf9-ts2 greater effect TRX1 relative <br /> (Figure 3A, compare lanes 6, 10, 14 lane 2),         RPS5 (Shen et al. 2003). No difference transcrip- <br /> mRNA levels reduced taf9-ts2 strain            tion GLN1, temperature shift, was <br /> (Figure 3A, lane 4). However, synergistic decrease         detectable strains. To test possibil- <br /> mRNA levels observed double-mutant strains               ity expression TAF9 dependent on <br /> containing taf9-ts2 combination swr1D, vps72D,          SWR-C, analyzed TAF9 protein levels single and <br /> or aor1D (Figure 3A, lanes 8, 12, 16). Interestingly,       double mutants Western blotting. Figure 3B shows <br /> this synergistic effect greater aor1D taf9     TAF9 protein levels affected deletion <br />  970                                                    E. Milgrom et al. <br />  <br /> of components SWR-C. The combined results indi-                   The authors especially thank Dave Amberg introducing use <br /> cate SWR-C critical transcription         robotic SGA screens SUNY Upstate Medical University. Construc- <br />                                                                   tive input members yeast data club SUNY Upstate <br /> essential housekeeping gene RPS5 function             Medical University Syracuse University highly appreciated. <br /> TAF9 compromised. These results support hypoth-              We thank anonymous reviewers critical comments <br /> esis mechanism underlying synthetic                improved manuscript. A Faculty Development Fund and <br /> genetic relationship TAF9 entail combined            Hendrick&#226;&#8364;&#8482;s Fund provided SUNY Upstate Medical University to <br /> effect double block expression single            W.-C.W.S supported work. <br /> essential gene. In light results presented pre- <br /> ceding section, conjunction possibility <br /> that SWR-C involved aspect transcrip- <br /> tion elongation (Santisteban et al. 2000; Krogan et al.                                LITERATURE CITED <br /> 2003), double block arise defects ini-          Adkins, M. W., S. R. Howar J. K. Tyler, 2004 Chromatin dis- <br /> tiation plus elongation defects elongation only.                assembly mediated histone chaperone Asf1 essential for <br />                                                                       transcriptional activation yeast PHO5 PHO8 genes. <br />    Concluding remarks: Over genetic                       Mol. Cell 14: 657&#226;&#8364;&#8220;666. <br /> interactions TAF9 identified, approximately             Ahn, S. H., M. Kim S. Buratowski, 2004 Phosphorylation of <br /> half involving previously characterized transcription                 serine 2 RNA polymerase II C-terminal domain cou- <br />                                                                       ples transcription 39 end processing. Mol. Cell 13: 67&#226;&#8364;&#8220;76. <br /> factors. The higher-than-average number genetic in-            Albright, S. R., R. Tjian, 2000 TAFs revisited: data reveal <br /> teractions, compared  30 average nonessen-                  new twists confirm old ideas. Gene 242: 1&#226;&#8364;&#8220;13. <br /> tial gene (Tong et al. 2004), reflect fact           Apone, L. M., C. M. Virbasius, J. C. Reese M. R. Green, 1996 Yeast <br />                                                                       TAF(II)90 required cell-cycle progression G2/M <br /> TAF9 essential gene required transcription               general transcription activation. Genes Dev. 10: <br />  60% S. cerevisiae genome (Shen et al. 2003). The              2368&#226;&#8364;&#8220;2380. <br /> central role TAF9 presumably derives             Apone, L. M., C. A. Virbasius, F. C. Holstege, J. Wang, R. A. Young <br />                                                                       et al., 1998 Broad, universal, transcriptional require- <br /> the fact present TFIID SAGA,               ment yTAFII17, histone H3-like TAFII present TFIID <br /> also fact histone-fold             SAGA. Mol. Cell 2: 653&#226;&#8364;&#8220;661. <br /> TAFs critical maintaining structural integrity         Badarinarayana, V., Y. C. Chiang C. L. Denis, 2000 Func- <br />                                                                       tional interaction CCR4-NOT proteins TATAA-binding <br /> these complexes (Moqtaderi et al. 1998; Yatherajam                    protein (TBP) associated factors yeast. Genetics 155: <br /> et al. 2003 ; Wu et al. 2004; Timmers Tora 2005).                 1045&#226;&#8364;&#8220;1054. <br /> Considering estimated 384 nonessential genes in-               Baetz, K., J. Moffat, J. Haynes, M. Chang B. Andrews, <br />                                                                       2001 Transcriptional coregulation cell integrity mitogen- <br /> volved transcriptional regulation S. cerevisiae (Krogan         activated protein kinase Slt2 cell cycle regulator Swi4. <br /> et al. 2003), identification  50 TAF9               Mol. Cell. Biol. 21: 6515&#226;&#8364;&#8220;6528. <br /> interactors suggests genetic interactions identified          Bender, A., J. R. Pringle, 1991 Use screen synthetic <br />                                                                       lethal multicopy suppressee mutants identify new <br /> here quite selective (see Table 1).             genes involved morphogenesis Saccharomyces cerevisiae. <br />    Data presented study expand regulatory                 Mol. Cell. Biol. 11: 1295&#226;&#8364;&#8220;1305. <br /> roles TAF9 include transcriptional elongation           Bjorklund, S., C. M. Gustafsson, 2005 The yeast Mediator <br />                                                                       complex regulation. Trends Biochem. Sci. 30: 240&#226;&#8364;&#8220;244. <br /> general growth control providing genetic support              Bonangelino, C. J., E. M. Chavez J. S. Bonifacino, 2002 Genomic <br /> for specific biochemical interactions reported occur               screen vacuolar protein sorting genes Saccharomyces <br /> between various transcription factors TAF9 TFIID               cerevisiae. Mol. Biol. Cell 13: 2486&#226;&#8364;&#8220;2501. <br />                                                                   Breeden, L. L., 2003 Periodic transcription: cycle cycle. <br /> (Figure 1, factors shown yellow ovals; Sanders et al.              Curr. Biol. 13: R31&#226;&#8364;&#8220;R38. <br /> 2002). Significantly, genetic data combination             Brown, C. E., T. Lechner, L. Howe J. L. Workman, 2000 The <br /> with Berger colleagues (Ingvarsdottir                    HATs transcription coactivators. Trends Biochem. Sci. <br />                                                                       25: 15&#226;&#8364;&#8220;19. <br /> et al. 2005) (Kong et al. 2005) suggest            Burckin, T., R. Nagel, Y. Mandel-Gutfreund, L. Shiue, T. A. Clark <br /> possibility TAF9 involvement transcription elon-              et al., 2005 Exploring functional relationships compo- <br /> gation occurs specifically context SAGA. This               nents gene expression machinery. Nat. Struct. Mol. Biol. <br />                                                                       12: 175&#226;&#8364;&#8220;182. <br /> prediction supported observation               Casolari, J. M., C. R. Brown, D. A. Drubin, O. J. Rando P. A. <br /> SAGA subunit Spt20 required proper histone-DNA                 Silver, 2005 Developmentally induced changes transcrip- <br /> contacts elongating forms RNA polymerase II                 tional program alter spatial organization chromosomes. <br />                                                                       Genes Dev. 19: 1188&#226;&#8364;&#8220;1198. <br /> PDR5-coding region sequences. Extensive genetic inter-            Chadick, J. Z., F. J. Asturias, 2005 Structure eukaryotic <br /> actions TAF9 components SWR-C,                       Mediator complexes. Trends Biochem. Sci. 30: 264&#226;&#8364;&#8220;271. <br /> function affiliated transcriptional elongation            Chang, F., M. Peter, 2003 Yeasts make mark. Nat. Cell <br />                                                                       Biol. 5: 294&#226;&#8364;&#8220;299. <br /> (Santisteban et al. 2000; Krogan et al. 2003), supports           Chimura, T., T. Kuzuhara M. Horikoshi, 2002 Identification <br /> involvement TAF9 elongation control, physi-               characterization CIA/ASF1 interactor bromodo- <br /> cal interactions TAF9 subunits                 mains associated TFIID. Proc. Natl. Acad. Sci. USA 99: 9334&#226;&#8364;&#8220; <br />                                                                       9339. <br /> Elongator (Sanders et al. 2002), overlapping           Cook, P. R., 1999 The organization replication transcription. <br /> roles HAT activities SAGA Elongator                    Science 284: 1790&#226;&#8364;&#8220;1795. <br /> (Wittschieben et al. 2000). During preparation            Deluen, C., N. James, L. Maillet, M. Molinete, G. Theiler <br />                                                                       et al., 2002 The Ccr4-not complex yTAF1 (yTaf(II)130p/ <br /> article, Govind et al. (2005) published data supporting               yTaf(II)145p) physical functional interactions. Mol. <br /> a role SAGA regulating transcription elongation.               Cell. Biol. 22: 6735&#226;&#8364;&#8220;6749. <br />                                                       Synthetic Genetic Interactions TAF9                                                    971 <br />  <br /> De Nadal, E., M. Zapater, P. M. Alepuz, L. Sumoy, G. Mas et al.,             Kobor, M. S., S. Venkatasubrahmanyam, M. D. Meneghini, J. W. <br />     2004 The MAPK Hog1 recruits Rpd3 histone deacetylase ac-                   Gin, J. L. Jennings et al., 2004 A protein complex containing <br />     tivate osmoresponsive genes. Nature 427: 370&#226;&#8364;&#8220;374.                             conserved Swi2/Snf2-related ATPase Swr1p deposits histone <br /> Denis, C. L., Y. C. Chiang, Y. Cui J. Chen, 2001 Genetic evi-                 variant H2A.Z euchromatin. PLoS Biol. 2: E131. <br />     dence supports role yeast CCR4-NOT complex tran-            Koepp, D. M., D. H. Wong, A. H. Corbett P. A. Silver, <br />     scriptional elongation. Genetics 158: 627&#226;&#8364;&#8220;634.                                1996 Dynamic localization nuclear import receptor <br /> Dez, C., D. Tollervey, 2004 Ribosome synthesis meets cell                 interactions transport factors. J. Cell Biol. 133: <br />     cycle. Curr. Opin. Microbiol. 7: 631&#226;&#8364;&#8220;637.                                     1163&#226;&#8364;&#8220;1176. <br /> Durso, R. J., A. K. Fisher, T. J. Albright-Frey J. C. Reese,             Komarnitsky, P., E. J. Cho S. Buratowski, 2000 Different <br />     2001 Analysis TAF90 mutants displaying allele-specific                     phosphorylated forms RNA polymerase II associated <br />     broad defects transcription. Mol. Cell. Biol. 21: 7331&#226;&#8364;&#8220;                mRNA processing factors transcription. Genes Dev. 14: <br />     7344.                                                                         2452&#226;&#8364;&#8220;2460. <br /> Dynlacht, B. D., T. Hoey R. Tjian, 1991 Isolation coactiva-           Kong, S. E., M. S. Kobor, N. J. Krogan, B. P. Somesh, T. M. Sogaard <br />     tors associated TATA-binding protein mediate tran-              et al., 2005 Interaction Fcp1 phosphatase elongating <br />     scriptional activation. Cell 66: 563&#226;&#8364;&#8220;576.                                     RNA polymerase II holoenzyme, enzymatic mechanism action, <br /> Eide, D. J., 2000 Metal ion transport eukaryotic microorganisms:               genetic interaction elongator. J. Biol. Chem. 280: 4299&#226;&#8364;&#8220; <br />     insights Saccharomyces cerevisiae. Adv. Microb. Physiol. 43:             4306. <br />     1&#226;&#8364;&#8220;38.                                                                    Kornberg, R. D., 2005a Mediator mechanism transcrip- <br /> Ferdous, A., F. Gonzalez, L. Sun, T. Kodadek S. A. Johnston,                  tional activation. Trends Biochem. Sci. 30: 235&#226;&#8364;&#8220;239. <br />     2001 The 19S regulatory particle proteasome required           Kornberg, R. D., 2005b Mediator comes age. Trends Biochem. <br />     efficient transcription elongation RNA polymerase II. Mol.             Sci. 30: 221. <br />     Cell 7: 981&#226;&#8364;&#8220;991.                                                         Kristjuhan, A., J. Q. Svejstrup, 2004 Evidence distinct <br /> Finger, F. P., P. Novick, 2000 Synthetic interactions post-            mechanisms facilitating transcript elongation chroma- <br />     Golgi sec mutations Saccharomyces cerevisiae. Genetics 156: 943&#226;&#8364;&#8220;           tin vivo. EMBO J. 23: 4243&#226;&#8364;&#8220;4252. <br />     951.                                                                     Krogan, N. J., J. Dover, S. Khorrami, J. F. Greenblatt, <br /> Gao, C., L. Wang, E. Milgrom W. C. Shen, 2004 On mech-                    J. Schneider et al., 2002 COMPASS, histone H3 (lysine 4) <br />     anism constitutive Pdr1 activator-mediated PDR5 transcription              methyltransferase required telomeric silencing gene ex- <br />     Saccharomyces cerevisiae: evidence enhanced recruitment                pression. J. Biol. Chem. 277: 10753&#226;&#8364;&#8220;10755. <br />     coactivators altered nucleosome structures. J. Biol. Chem.        Krogan, N. J., M. C. Keogh, N. Datta, C. Sawa, O. W. Ryan et al., <br />     279: 42677&#226;&#8364;&#8220;42686.                                                             2003 A Snf2 family ATPase complex required recruitment <br /> Gilbert, C., A. Kristjuhan, G. S. Winkler J. Q. Svejstrup,                    histone H2A variant Htz1. Mol. Cell 12: 1565&#226;&#8364;&#8220;1576. <br />     2004 Elongator interactions nascent mRNA revealed                Kurdistani, S. K., M. Grunstein, 2003 Histone acetylation and <br />     RNA immunoprecipitation. Mol. Cell 14: 457&#226;&#8364;&#8220;464.                               deacetylation yeast. Nat. Rev. Mol. Cell Biol 4: 276&#226;&#8364;&#8220;284. <br /> Gonzalez, F., A. Delahodde, T. Kodadek S. A. Johnston,                   Kurdistani, S. K., S. Tavazoie M. Grunstein, 2004 Mapping <br />     2002 Recruitment 19S proteasome subcomplex acti-                   global histone acetylation patterns gene expression. Cell 117: <br />     vated promoter. Science 296: 548&#226;&#8364;&#8220;550.                                         721&#226;&#8364;&#8220;733. <br /> Govind, C. K., S. Yoon, H. Qiu, S. Govind A. G. Hinnebusch,              Larschan, E., F. Winston, 2005 The Saccharomyces cerevisiae <br />     2005 Simultaneous recruitment coactivators Gcn4p stim-                  Srb8-Srb11 complex functions SAGA complex during <br />     ulates multiple steps transcription vivo. Mol. Cell. Biol. 25:          Gal4-activated transcription. Mol. Cell. Biol. 25: 114&#226;&#8364;&#8220;123. <br />     5626&#226;&#8364;&#8220;5638.                                                               Lee, C. K., Y. Shibata, B. Rao, B. D. Strahl J. D. Lieb, <br /> Grant, P. A., D. Schieltz, M. G. Pray-Grant, D. J. Steger, J. C.                  2004 Evidence nucleosome depletion active regulatory <br />     Reese et al., 1998 A subset TAF(II)s integral components               regions genome-wide. Nat. Genet. 36: 900&#226;&#8364;&#8220;905. <br />     SAGA complex required nucleosome acetylation              Lee, J. M., A. L. Greenleaf, 1991 CTD kinase large subunit is <br />     transcriptional stimulation. Cell 94: 45&#226;&#8364;&#8220;53.                                  encoded CTK1, gene required normal growth Saccha- <br /> Green, M. R., 2000 TBP-associated factors (TAFIIs): multiple, selec-              romyces cerevisiae. Gene Expr. 1: 149&#226;&#8364;&#8220;167. <br />     tive transcriptional mediators common complexes. Trends               Lee, T. I., H. C. Causton, F. C. Holstege, W. C. Shen, N. Hannett <br />     Biochem. Sci. 25: 59&#226;&#8364;&#8220;63.                                                      et al., 2000 Redundant roles TFIID SAGA com- <br /> Harbison, C. T., D. B. Gordon, T. I. Lee, N. J. Rinaldi, K. D.                    plexes global transcription. Nature 405: 701&#226;&#8364;&#8220;704. <br />     Macisaac et al., 2004 Transcriptional regulatory <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> eu-           Lei, E. P., H. Krebber P. A. Silver, 2001 Messenger RNAs are <br />     karyotic genome. Nature 431: 99&#226;&#8364;&#8220;104.                                          recruited nuclear export transcription. Genes Dev. 15: <br /> Harrison, J. C., T. R. Zyla, E. S. Bardes D. J. Lew, 2004 Stress-             1771&#226;&#8364;&#8220;1782. <br />     specific activation mechanisms &#226;&#8364;&#732;&#226;&#8364;&#732;cell integrity&#226;&#8364;&#8482;&#226;&#8364;&#8482; MAPK           Lei, E. P., C. A. Stern, B. Fahrenkrog, H. Krebber, T. I. Moy et al., <br />     pathway. J. Biol. Chem. 279: 2616&#226;&#8364;&#8220;2622.                                       2003 Sac3 mRNA export factor localizes cyto- <br /> Hartman, J. L., IV, B. Garvik L. Hartwell, 2001 Principles                plasmic fibrils nuclear pore complex. Mol. Biol. Cell 14: <br />     buffering genetic variation. Science 291: 1001&#226;&#8364;&#8220;1004.                   836&#226;&#8364;&#8220;847. <br /> Henry, K. W., A. Wyce, W. S. Lo, L. J. Duggan, N. C. Emre et al.,            Lei, M., B. K. Tye, 2001 Initiating DNA synthesis: recruit- <br />     2003 Transcriptional activation sequential histone H2B ubiq-              ing activating MCM complex. J. Cell Sci 114: 1447&#226;&#8364;&#8220;1454. <br />     uitylation deubiquitylation, mediated SAGA-associated             Lew, D. J., S. I. Reed, 1993 Morphogenesis yeast cell <br />     Ubp8. Genes Dev. 17: 2648&#226;&#8364;&#8220;2663.                                               cycle: regulation Cdc28 cyclins. J. Cell Biol. 120: 1305&#226;&#8364;&#8220; <br /> Holstege, F. C., E. G. Jennings, J. J. Wyrick, T. I. Lee, C. J. Hengartner        1320. <br />     et al., 1998 Dissecting regulatory circuitry eukaryotic         Longtine, M. S., A. McKenzie, III, D. J. Demarini, N. G. Shah, <br />     genome. Cell 95: 717&#226;&#8364;&#8220;728.                                                     A. Wach et al., 1998 Additional modules versatile eco- <br /> Huisinga, K. L., B. F. Pugh, 2004 A genome-wide housekeeping                  nomical PCR-based gene deletion modification Saccharo- <br />     role TFIID highly regulated stress-related role SAGA            myces cerevisiae. Yeast 14: 953&#226;&#8364;&#8220;961. <br />     Saccharomyces cerevisiae. Mol. Cell 13: 573&#226;&#8364;&#8220;585.                      Macpherson, N., V. Measday, L. Moore B. Andrews, 2000 A <br /> Hwang, W. W., S. Venkatasubrahmanyam, A. G. Ianculescu,                           yeast taf17 mutant requires Swi6 transcriptional activator for <br />     A. Tong, C. Boone et al., 2003 A conserved RING finger protein                viability shows defects cell cycle-regulated transcription. <br />     required histone H2B monoubiquitination cell size con-                Genetics 154: 1561&#226;&#8364;&#8220;1576. <br />     trol. Mol. Cell 11: 261&#226;&#8364;&#8220;266.                                             Madden, K., Y. J. Sheu, K. Baetz, B. Andrews M. Snyder, <br /> Ingvarsdottir, K., N. J. Krogan, N. C. T. Emre, A. Wyce, N. J.                    1997 SBF cell cycle regulator target yeast PKC- <br />     Thompson et al., 2005 H2B ubiquitin protease Ubp8 Sgf11                   MAP kinase pathway. Science 275: 1781&#226;&#8364;&#8220;1784. <br />     constitute discrete functional module Saccharomy-           Malagon, F., A. H. Tong, B. K. Shafer J. N. Strathern, <br />     ces cerevisiae SAGA complex. Mol. Cell. Biol. 25: 1162&#226;&#8364;&#8220;1172.                  2004 Genetic interactions DST1 Saccharomyces cerevisiae <br /> Jones, J. S., L. Prakash, 1990 Yeast Saccharomyces cerevisiae                 suggest role TFIIS initiation-elongation transition. <br />     selectable markers pUC18 polylinkers. Yeast 6: 363&#226;&#8364;&#8220;366.                    Genetics 166: 1215&#226;&#8364;&#8220;1227. <br />  972                                                               E. Milgrom et al. <br />  <br /> Maniatis, T., R. Reed, 2002 An extensive network coupling             Shen, W. C., S. R. Bhaumik, H. C. Causton, I. Simon, X. Zhu et al., <br />     gene expression machines. Nature 416: 499&#226;&#8364;&#8220;506.                         2003 Systematic analysis essential yeast TAFs genome-wide <br /> Mencia, M., Z. Moqtaderi, J. V. Geisberg, L. Kuras K. Struhl,                transcription preinitiation complex assembly. EMBO J. 22: <br />     2002 Activator-specific recruitment TFIID regulation               3395&#226;&#8364;&#8220;3402. <br />     ribosomal protein genes yeast. Mol. Cell 9: 823&#226;&#8364;&#8220;833.                  Sherman, F., 1991 Getting started yeast. Methods Enzymol. <br /> Meneghini, M. D., M. Wu H. D. Madhani, 2003 Conserved his-                   194: 3&#226;&#8364;&#8220;21. <br />     tone variant H2A.Z protects euchromatin ectopic                 Shibuya, T., S. Tsuneyoshi, A. K. Azad, S. Urushiyama, Y. Ohshima <br />     spread silent heterochromatin. Cell 112: 725&#226;&#8364;&#8220;736.                         et al., 1999 Characterization ptr6(1) gene fission <br /> Menon, B. B., N. J. Sarma, S. Pasula, S. J. Deminoff, K. A. Willis               yeast: possible involvement transcriptional coactivator <br />     et al., 2005 Reverse recruitment: Nup84 nuclear pore sub-                TAF nucleocytoplasmic transport mRNA. Genetics 152: <br />     complex mediates Rap1/Gcr1/Gcr2 transcriptional activation.                  869&#226;&#8364;&#8220;880. <br />     Proc. Natl. Acad. Sci. USA 102: 5749&#226;&#8364;&#8220;5754.                               Simic, R., D. L. Lindstrom, H. G. Tran, K. L. Roinick, P. J. Costa <br /> Michel, B., P. Komarnitsky S. Buratowski, 1998 Histone-                      et al., 2003 Chromatin remodeling protein Chd1 interacts with <br />     like TAFs essential transcription vivo. Mol. Cell 2:              transcription elongation factors localizes transcribed <br />     663&#226;&#8364;&#8220;673.                                                                     genes. EMBO J. 22: 1846&#226;&#8364;&#8220;1856. <br /> Mizuguchi, G., X. Shen, J. Landry, W. H. Wu, S. Sen et al.,                  Sims, R. J., III, R. Belotserkovskaya D. Reinberg, <br />     2004 ATP-driven exchange histone H2AZ variant cata-                       2004 Elongation RNA polymerase II: short long <br />     lyzed SWR1 chromatin remodeling complex. Science 303:                     it. Genes Dev. 18: 2437&#226;&#8364;&#8220;2468. <br />     343&#226;&#8364;&#8220;348.                                                                 Squazzo, S. L., P. J. Costa, D. L. Lindstrom, K. E. Kumer, R. Simic <br /> Moffat, J., B. Andrews, 2004 Late-G1 cyclin-CDK activity es-              et al., 2002 The Paf1 complex physically functionally asso- <br />     sential control cell morphogenesis budding yeast. Nat.             ciates transcription elongation factors vivo. EMBO J. 21: <br />     Cell Biol. 6: 59&#226;&#8364;&#8220;66.                                                         1764&#226;&#8364;&#8220;1774. <br /> Moqtaderi, Z., M. Keaveney K. Struhl, 1998 The histone                   Sterner, D. E., P. A. Grant, S. M. Roberts, L. J. Duggan, <br />     H3-like TAF broadly required transcription yeast. Mol.             R. Belotserkovskaya et al., 1999 Functional organization of <br />     Cell 2: 675&#226;&#8364;&#8220;682.                                                             yeast SAGA complex: distinct components involved struc- <br /> Narlikar, G. J., H. Y. Fan R. E. Kingston, 2002 Cooperation                  tural integrity, nucleosome acetylation, TATA-binding pro- <br />     complexes regulate chromatin structure tran-                tein interaction. Mol. Cell. Biol. 19: 86&#226;&#8364;&#8220;98. <br />     scription. Cell 108: 475&#226;&#8364;&#8220;487.                                            Sterner, D. E., R. Belotserkovskaya S. L. Berger, 2002 SALSA, <br /> Page, N., M. Gerard-Vincent, P. Menard, M. Beaulieu, M. Azuma                    variant yeast SAGA, contains truncated Spt7, correlates <br />     et al., 2003 A Saccharomyces cerevisiae genome-wide mutant                   activated transcription. Proc. Natl. Acad. Sci. USA 99: 11622&#226;&#8364;&#8220; <br />     screen altered sensitivity K1 killer toxin. Genetics 163:             11627. <br />     875&#226;&#8364;&#8220;894.                                                                 Sun, L., S. A. Johnston T. Kodadek, 2002 Physical association <br /> Parker, R., H. Song, 2004 The enzymes control eukary-                 APIS complex general transcription factors. Biochem. <br />     otic mRNA turnover. Nat. Struct. Mol. Biol. 11: 121&#226;&#8364;&#8220;127.                     Biophys. Res. Commun <br /> </body></html>