<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>4642&#226;&#8364;&#8220;4648 Nucleic Acids Research, 2000, Vol. 28, No. 23                                                              &#194;&#169; 2000 Oxford University Press <br />  <br />  <br />  <br /> The NAD-dependent ligase encoded yerG an <br /> essential gene Bacillus subtilis <br /> Marie-Agn&#195;&#168;s Petit* S. Dusko Ehrlich <br /> Laboratoire G&#195;&#169;n&#195;&#169;tique Microbienne, INRA, 78352 Jouy en Josas, France <br />  <br /> Received August 29, 2000; Revised Accepted October 12, 2000 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                       formed. The crystal structure 41 kDa ATP-dependent <br />                                                                                ligase bacteriophage T7 solved (1), the <br /> DNA ligases grouped families, ATP- <br />                                                                                adenylation domain NAD-dependent DNA ligase of <br /> dependent NAD-dependent, according the <br />                                                                                Bacillus stearothermophilus (2). This enabled Wigley and <br /> cofactor required activity. A surprising capa- <br />                                                                                colleagues (2) make structural alignment adenylation <br /> bility kinds ligases complement                          domain proteins, prove similar fold <br /> another vivo observed. Bacillus subtilis                           share critical residues. More recently three- <br /> harbours NAD-dependent ligase, YerG,                               dimensional structure complete 76 kDa NAD-dependent <br /> ATP-dependent ligases, YkoU YoqV,                                ligase Thermus filiformis reported (3). <br /> one encoded 134 kb lysogenic bacterio-                               NAD-dependent ligases uniform size (~70 kDa) and <br /> phage SP&#206;&#178; consisting single adenylation                               reveal extensive amino acid sequence conservation throughout <br /> domain typical ATP-dependent ligases. Because                               entire protein. ATP-dependent ligases diverse in <br /> the genetics ligases B.subtilis                             size, share common ligase domain, &#226;&#710;&#188;270 residues in <br /> studied previously, genes encoding ligase                          <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>, possibly flanked different domains. Until recently, <br /> of kind, yerG yoqV, investigated. We                             common view NAD-dependent ligases present <br /> found yerG gene essential B.subtilis.                          eubacteria only, ATP-dependent ligases are <br /> This suggests ATP-dependent                                   common living organisms, including bacterio- <br /> ligases able complement yerG defect. In                             phages, archeabacteria, viruses eukaryotes. However, a <br /> addition, ATP-dependent ligase encoded                                  different picture emerged rising number of <br /> yoqV, cloned plasmid appropriate                               complete bacterial genomes sequenced. Among <br /> expression signals, unable rescue yerG                                17 complete genomes eubacteria presently available, all <br /> mutant strain. The B.subtilis ligase genes yerG                        contain gene homologous NAD-dependent <br /> yoqV introduced Escherichia coli                               ligase family, contain copies genes <br /> strain encoding thermosensitive ligase (ligts),                          homologous ATP-dependent ligase family well: <br /> whereas yoqV did complement ligts defects,                             Bacillus subtilis, Mycobacterium tuberculosis Haemo- <br />                                                                                philus influenzae (Table 1). <br /> yerG fully complemented growth UV sensi- <br /> tivity defects lig mutant. We propose rename <br /> the yerG yoqV genes B.subtilis ligA ligB <br />                                                                                Table 1. Distribution DNA ligases B.subtilis, H.influenzae and <br /> respectively.                                                                  M.tuberculosis <br />  <br />  <br /> INTRODUCTION                                                                   Bacterium species        NAD-dependent ligase      ATP-dependent ligase <br />                                                                                                         (length amino acids)   (length amino acids) <br /> DNA ligases catalyse joining breaks duplex DNA                       B.subtilis               YerG (668)                YoqV (270) <br /> during DNA replication, repair recombination. They are <br />                                                                                                                                   YkoUa (611) <br /> grouped families, ATP-dependent DNA ligases and <br /> NAD-dependent ligases, according cofactor required                  M.tuberculosis           LigA (691)                LigB (507) <br /> their activity. Apart common KXDG motif,                                                                         LigC (358) <br /> families share amino acid similarities. However, reac-                                                                    O28549 (759) <br /> tions catalysed classes DNA ligase identical, <br />                                                                                H.influenzae             DNLJ (679)                DNL1 (268) <br /> except fact AMP group derived two <br /> different cofactors. This implies covalent <br />                                                                                aThisprotein mentionned putative ligase data banks, the <br /> attachment AMP lysine residue KXDG motif, <br />                                                                                Pfam algorithm protein alignment includes ATP-dependent ligase <br /> followed transfer 5&#226;&#8364;&#178;-phosphoryl group nicked                family. Indeed conserved motives ATP-dependent ligase are <br /> site duplex DNA. The AMP group finally released                     present protein (<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> alignment: http//www.sanger.ac.uk/ <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> adenylated DNA phosphodiester bond                          Software/Pfam). <br />  <br /> *To correspondence addressed. Tel: +33 1 34 65 25 12; Fax: +33 1 34 65 25 21; Email: petit@biotec.jouy.inra.fr <br />                                                                                            Nucleic Acids Research, 2000, Vol. 28, No. 23 4643 <br />  <br />  <br />    In eubacterial species, Escherichia coli Salmonella                 Saccharomyces cerevisiae strain encoding thermosensitive <br /> typhimurium, NAD-dependent ligase gene, encoded                     ligase (cdc9ts) (8). <br /> lig gene, known essential (4&#226;&#8364;&#8220;6). However                    Because high degree complementation between <br /> species ligase gene present,                 classes enzymes reported literature, not <br /> known genes essential. This question              possible tell priori ligase genes the <br /> is addressed regard B.subtilis. The B.subtilis genome             B.subtilis genome overlapping functions, or <br /> harbours NAD-dependent ligase-like gene, yerG,                     essential viability. We therefore <br /> ATP-dependent ligase-like genes, yoqV ykoU. The yoqV                       decided investigate NAD-dependent ligase <br /> gene encoded 134 kb lysogenic bacteriophage SP&#206;&#178;,                     encoding gene, yerG, essential dispensable B.subtilis. <br /> but ykoU apparent bacteriophage origin. YoqV                       We gene essential B.subtilis. In <br /> small protein composed 270 amino acids, consist                  addition, ATP-dependent ligase encoded by <br /> single domain, adenylation domain typical ATP-dependent                 yoqV, cloned plasmid appropriate expression <br /> ligases. The YkoU protein 611 amino acids length,                signals, unable rescue yerG mutant strain. This indicates <br /> adenylation domain encompasses 292 residues. Of                ligases substitute in <br /> two, smaller one, YoqV, studied detail. Both              B.subtilis. The B.subtilis ligase genes yerG yoqV were <br /> ATP-dependent ligase genes dispensable, strain                   introduced E.coli ligts strain, that <br /> cured SP&#206;&#178; prophage viable, ykoU mutant,                 yoqV does complement ligts defects, yerG <br /> which constructed frame systematic                        fully complements growth UV sensitivity defects of <br /> analysis unknown genes B.subtilis (Micado database:                      lig mutant. <br /> http://locus.jouy.inra.fr/cgi-bin/genmic/madbase_home.pl; strain <br /> constructed laboratory Dr K. Devine).                                MATERIALS AND METHODS <br />    Various studies shown heterologous complementation <br /> is possible ATP-dependent NAD-dependent                            Strains plasmids <br /> enzymes. Among examples ATP-dependent ligases able                       Strains plasmids listed Table 2. Bacillus subtilis and <br /> complement bacterial ligase mutants, shown                    E.coli cells grown LB medium unless otherwise <br /> the bacteriophage T4 ligase able rescue lig mutant                  stated. When needed, antibiotics added final concen- <br /> S.typhimurium (4), catalytic domain human ligase I                  tration 5 &#194;&#181;g/ml kanamycin 0.5 &#194;&#181;g/ml erythro- <br /> restores thermoresistance E.coli strain encoding                   mycin B.subtilis, 100 &#194;&#181;g/ml ampicillin E.coli. The <br /> thermosensitive ligase (ligts) (7). Conversely, NAD-                       chemically defined medium described (9). <br /> dependent ligase E.coli restores thermoresistance                  Constructions performed described below. The integrity <br />  <br />  <br />                Table 2. Strains plasmids <br />  <br />                Name                                Relevant genotype, description                           Reference, source <br />                Strains           JJC75a            lig7ts                                                   B.Michel <br />                                  168               trpC2                                                    C.Anagnostopoulos <br />                                  HVS610            168 insb pMAP149 yerH                                 This work <br />                                  HVS609            168 ins pMAP52 yerG                                   This work <br />                                  HVS611            168 ins pMAP59 yerG, plasmid pMAP141             This work <br />  <br />  <br />                Plasmids          pDG148            pBR322-pUB110 KanR lacI Pspac                            (15) <br />                                  pMUTIN2           pBR322 lacZ lacI EmR Pspac                               (16) <br />                                  pMAP52            pMUTIN2 Pspac:(nt &#226;&#8364;&#8220;20c 275 yerG)                   This work <br />                                  pMAP59            pMUTIN2MCS Pspac:(nt 330&#226;&#8364;&#8220;666 yerG)                    This work <br />                                  pMAP65            pUB110 KanR lacI                                         (9) <br />                                  pMAP138           pDG148 Pspac:yoqV                                        This work <br />                                  pMAP140           pDG148 Pspac:yerG                                        This work <br />                                  pMAP149           pMUTIN2MCS Pspac:(nt 540&#226;&#8364;&#8220;854 yerH)                    This work <br />                                  pIL253            EmR                                                      (17) <br />                                  pMAP139           pMAP138&#226;&#710;&#8224;(pBR322 lacI) AatII PstI, KanR           This work <br />                                  pMAP141           pMAP140&#226;&#710;&#8224;(pBR322 lacI) AatII PstI, KanR           This work <br />  <br />                aAB1157  background, hsdR thr+ pro+. <br />                binsindicates insertion given plasmid B.subtilis chromosome. <br />                cGene numbering starts start codon. <br />  4644 Nucleic Acids Research, 2000, Vol. 28, No. 23 <br />  <br />  <br />  <br />  <br /> Figure 1. Chromosomal structure wild-type strain 168 mutant strains. Strain 168 shown figure. In strain HVS609, plasmid pMAP52 is <br /> integrated yerG, fuse yerG Pspac promoter. In strain HVS610, plasmid pMAP149 disrupting yerH gene. <br />  <br />  <br />  <br /> of sequence DNA fragments generated PCR                        immediately upstream XbaI site followed SalI site. <br /> verified automated dideoxysequencing.                                         A DNA fragment containing yoqV ORF flanked artificially <br />                                                                                  introduced XbaI SalI sites generated PCR and <br /> yerG transcriptional fusion strain HVS609. To fuse yerG                    cloned pDG148 E.coli. The resulting plasmid, <br /> inducible promoter, plasmid pMUTIN2 used. This                          pMAP138, used complementation tests E.coli. For <br /> pBR322 derivative containing Pspac IPTG inducible promoter                     B.subtilis smaller derivative, pMAP139, deleted pBR322 <br /> upstream multiple cloning site flanked BamHI EcoRI.                  sequences lacI gene, AatII&#226;&#8364;&#8220;PstI deletion <br /> The promoter preceded transcription terminators.                       pMAP138. Using strategy, yerG ORF was <br /> Further downstream plasmid lacI gene                     cloned pDG148 pMAP140, smaller derivative <br /> a selectable erythromycin resistance marker (EmR)                            devoid pBR322 lacI sequences, pMAP141, was <br /> B.subtilis. A DNA fragment starting 20 bp upstream                        obtained AatII&#226;&#8364;&#8220;PstI deletion pMAP140. <br /> yerG start codon, include ribosome binding site <br /> 296 bp length, flanked artificially introduced EcoRI                   Adenylation test <br /> and BamHI sites, generated PCR cloned                            Bacillus subtilis E.coli crude extracts prepared the <br /> pMUTIN2. The resulting plasmid, pMAP52, contained tran-                        following way. A 100 ml overnight culture centrifuged, <br /> scriptional fusion Pspac                       pellet stored &#226;&#8364;&#8220;80&#194;&#176;C. To lyse cells, frozen pellet <br /> nucleotides yerG. To generate fusion Pspac                    resuspended 1 ml 50 mM Tris pH 7.5, 5 mM DTT, <br /> the complete ORF yerG, plasmid pMAP52 introduced                          1 mM EDTA, 2 mg/ml lysozyme, complemented cocktail <br /> by single crossing chromosome B.subtilis wild-                  protease inhibitors (Roche). The suspension incubated for <br /> type 168 strain. The resulting strain, HVS609 (Fig. 1), harbors                  20 min ice. Sonication applied 5 s using 1 s pulse/9 s <br /> a single copy yerG gene control Pspac                    pause cycles, small probe (Vibracell apparatus). The <br /> promoter.                                                                        soluble fraction (2 ml) collected 30 min ultracentri- <br />                                                                                  fugation 100 000 g, concentration ~4 mg/ml total <br /> yerH mutant strain HVS610. An internal fragment yerH                      protein. <br /> gene, 350 bp length, generated PCR cloned                            To test adenylation activity crude extracts, 4 &#194;&#181;l of <br /> between EcoRI BamHI sites pMUTIN2                             extract incubated buffer A (50 mM Tris pH 7.5, 10 mM <br /> plasmid pMAP149. To disrupt yerH, plasmid pMAP149                            MgCl2, 2 mM DTT, 0.1 mM ATP), final volume 20 &#194;&#181;l, <br /> introduced single crossing chromosome                      presence 5 &#194;&#181;Ci [&#206;&#177;-32P]ATP, 15 min 20&#194;&#176;C (the <br /> B.subtilis strain HVS610 (Fig. 1).                                       ratio cold- radiolabelled ATP critical for <br />                                                                                  optimal detection adenylation activity crude extracts). <br /> Plasmid pMAP59. To disrupt yerG gene, 314 bp internal                      Proteins denatured SDS buffer separated by <br /> segment yerG generated PCR cloned                          polyacrylamide gel electrophoresis (PAGE). The gel was <br /> the EcoRI BamHI sites pMUTIN2 plasmid                             fixed, colored dried prior exposition. Radioactivity was <br /> pMAP59.                                                                          detected using Storm apparatus (Molecular Dynamics). <br />  <br /> Plasmids pMAP138 141. To clone yoqV strong                              UV sensitivity test <br /> expression signals, plasmid pDG148 used vector. It                   Exponentially growing cells diluted serially plated on <br /> a pBR322-pUB110 shuttle vector, replicates                         LB. Plates irradiated fluence 2 J/m2/s different <br /> E.coli B.subtilis. It contains strong promoter active                 times, incubated growth 30&#194;&#176;C (for strain JJC75) <br /> in B.subtilis E.coli canonical ribosome binding site                   42&#194;&#176;C strains. <br />                                                                                                  Nucleic Acids Research, 2000, Vol. 28, No. 23 4645 <br />  <br />  <br />        Table 3. A yerG, yoqV overproducing strain allows selection B.subtilis transformants harbouring yerG disruption <br />  <br />                                                          Proportion disrupted yerG clonesa transformantsb B.subtilis 168 strain containing: <br />        Transforming DNA                                  plasmid                      pMAP141 (yerG)                  pMAP139 (yoqV) <br />        Plasmid DNA pMAP59 (disrupts yerG)                N.R.                            5/15                            0/2 <br />        Total DNA strain <br />        HVS611 (yerG disrupted)                           0/48                            18/18                           0/18 <br />  <br />        aThe proportion transformants effectively containing yerG disruption resulting congression indicated (see text details). <br />        bCompetent   cells indicated strains transformed EmR indicated DNA. The strains tested similar level of <br />        competence, estimated transformation plasmid pIL253 (between 1.3 &#195;&#8212; 105 3.7 &#195;&#8212; 105 transformants &#194;&#181;g DNA obtained). <br />        N.R., relevant (no transformants obtained). <br />  <br />  <br /> RESULTS <br /> yerG essential gene <br /> The NAD-dependent ligase gene located 55&#194;&#176; the <br /> B.subtilis chromosome, gene putative operon <br /> encoding pcrB, pcrA, yerG yerH (Fig. 1). Among these <br /> four genes, pcrA, encoding DNA helicase SF1 <br /> family, characterised essential <br /> gene (9). The pcrB yerH genes unknown not <br /> match known protein organisms. <br />   A yerG mutant strain constructed examine                          Figure 2. Growth curves strain HVS609 harbouring plasmid pMAP65. <br /> the encoded NAD-dependent ligase enzyme essential                               Colony forming units (cfu, left) optical density (OD650, right) strain <br /> in vivo. To disrupt yerG gene, B.subtilis transformed                      grown presence (squares) absence (circles) IPTG are <br />                                                                                    reported function time. <br /> with pMAP59, non-replicative plasmid containing an <br /> internal fragment gene. No transformants obtained <br /> on rich media, chemically defined media. <br />   The impossibility disrupting yerG result                         Overexpression yoqV encoded ATP-dependent ligase <br /> polar effect insertion downstream gene                        does rescue yerG <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> operon (yerH), requirement yerG viability. <br /> To test alternative, yerH mutant constructed.                  Two potential ATP-dependent ligases encoded the <br /> As yerG gene, non-replicative plasmid containing                      B.subtilis genome, ykoU yoqV, essential. <br /> internal fragment yerH used transform B.subtilis.                        Since yerG essential wild-type strain B.subtilis, this <br />                                                                                    suggests ATP-dependent ligase complement <br /> Transformants obtained, contained appropriate <br />                                                                                    yerG activity. Because yoqV gene belongs SP&#206;&#178; <br /> gene disruption, resulting strain (strain HVS610, Fig. 1) <br />                                                                                    prophage, reasoned transcription likely be <br /> did growth defect. We concluded that <br />                                                                                    repressed chromosome. The possibility existed therefore <br /> yerH essential, turn, impossibility to <br />                                                                                    ATP-dependent ligase fused appropriate <br /> disrupt yerG likely to requirement                           expression signals able complement yerG <br /> viability.                                                                         mutation. To test possibility, plasmid constructed, <br />   To confirm conclusion, strain HVS609 constructed,                       pMAP139, yoqV ORF cloned pUB110 <br /> in yerG fused IPTG inducible promoter. We                          derivative, strong expression signals, including the <br /> investigated strain HVS609 able grow                         Pspac promoter canonical ribosome binding site. As a <br /> absence IPTG. Almost sign lethality observed                          control, yerG gene cloned similarly pUB110 <br /> after approximately 30 generations growth absence                     derivative, plasmid pMAP141. <br /> inducer. This low concentration LacI                      In order investigate new strain, the <br /> cell (only copy lacI gene inserted                    yoqV ATP-dependent ligase constitutively expressed, <br /> chromosome, Pspac&#226;&#8364;&#8220;yerG fusion), allowing                         survive absence yerG encoded NAD- <br /> leaky yerG expression. When plasmid pMAP65, pUB110                               dependent ligase, attempt disrupting yerG non- <br /> derivative encoding LacI, introduced strain HVS609,                       replicative plasmid pMAP59 (see paragraph) repeated. <br /> arrest growth observed absence inducer,                        Table 3, lane 1, shows proportion transformants which <br /> shown Figure 2. After 2 h, number viable cells ceased                    yerG gene effectively interrupted various B.subtilis <br /> to increase optical density continued increase                      strains. <br /> additional hour, result cell filamentation,                            In B.subtilis strain harbouring yerG encoding plasmid <br /> leveling off. Therefore, strain HVS609 containing pMAP65                        pMAP141, transformants obtained (50/&#194;&#181;g of <br /> IPTG-dependent growth. This confirms yerG essential.                   plasmid DNA). This expected resulted the <br />  4646 Nucleic Acids Research, 2000, Vol. 28, No. 23 <br />  <br />  <br /> reduced homology recipient transforming DNA: <br /> plasmid pMAP59 shares 337 bp homology yerG <br /> gene, 293 bp homology plasmids <br /> pMAP139 pMAP141 segment encompassing the <br /> Pspac promoter. To distinguish transformants in <br /> which pMAP59 recombined chromosomal copy <br /> of yerG, pMAP59 recombined the <br /> resident plasmid pMAP141, total DNA extracted from <br /> 15 transformants. Five 15 clones contained disrupted <br /> copy yerG (Table 3, lane 1) determined Southern blot <br /> hybridisation (not shown). The 10 clones contained a <br /> recombinant plasmid pMAP141 pMAP59. From <br /> this analysis concluded possible construct a <br /> yerG disruption B.subtilis strain containing plasmidic       Figure 3. Presence adenylate activity specific E.coli B.subtilis <br /> copy gene. The resulting strain named HVS611.            crude extracts strains containing plasmid overproducing YoqV. Crude <br />                                                                     extracts E.coli strains JM105 (lane 1), JM105 + pMAP138 grown without <br />    In B.subtilis strain harbouring yoqV encoding            IPTG (lane 2), JM105 + pMAP138 grown IPTG (lane 3), B.subtilis <br /> plasmid pMAP139, transformation plasmid pMAP59                 strains 168 (lane 4) 168 + pMAP139 (lane 5), incubated radio- <br /> yielded transformants only, trans-        labelled ATP, separated PAGE, stained Coomassie blue (right, the <br /> formants contained disrupted copy yerG. Rather,        marker lane shown), exposed (left). Lane M, molecular weight markers, <br /> them mixture plasmids, resident pMAP139,           relevant sizes indicated kDa. <br /> and 14 kb plasmid recombinant pMAP139 and <br /> pMAP59. The low transformation efficiency, absence <br /> of transformants harbouring yerG disruption, suggested that <br /> a yerG mutation tolerated strain expressing yoqV.         In conclusion, plasmid pMAP141 present the <br />                                                                     recipient wild-type strain B.subtilis, chromosomal copy <br /> However, given low number transformants, possi- <br />                                                                     yerG disrupted. However, yoqV encoding <br /> bility disruption tolerated be <br />                                                                     plasmid pMAP139 present, transformants containing <br /> completely excluded. <br />                                                                     disrupted copy yerG obtained. <br />    To increase chances yerG disruption strain 168 <br />                                                                        To verify ATP-dependent ligase cloned into <br /> harbouring pMAP139, attempt disrupting yerG          pMAP139 active, adenylation test performed on <br /> made. Instead pMAP59 source DNA, total DNA            B.subtilis crude extracts. The step ligation process <br /> strain HVS611, pMAP59 integrated           consists covalent attachment AMP lysine <br /> yerG, used (Table 3, lane 2). In case, homology      residue ligase, visualised easily a <br /> between resident transforming DNA greatly increased,         protein gel AMP radiolabelled. Extracts treated with <br /> and higher transformation frequencies expected. However,        RNase incubated presence [&#206;&#177;-32P]ATP 15 min <br /> because total DNA strain HVS611 contains                room temperature analysed denaturing PAGE. As <br /> pMAP141 plasmid DNA, possibility exists obtain &#226;&#8364;&#732;false        shown Figure 3, lanes 4 5, radioactive signal was <br /> positive transformants&#226;&#8364;&#8482;, resulting congression, i.e.     detected samples extracted YoqV-containing <br /> simultaneous acquisition independent markers,                strain, absent B.subtilis wild-type extract. This <br /> yerG::EmR plasmid pMAP141. In strain 168 harbouring             signal corresponded protein ~30 kDa, close to <br /> the yerG encoding plasmid, high yield transformants            expected size 31 kDa YoqV. Therefore YoqV <br /> (3.5 &#195;&#8212; 104/&#194;&#181;g DNA) obtained expected,             ligase present active strain containing plasmid <br /> among 18 clones analysed, contained interrupted copy      pMAP139. The absence signal crude extracts of <br /> yerG (Table 3, lane 2) determined PCR analysis. With        B.subtilis wild-type extracts suggests YoqV <br /> strain 168, 10 times transformants (2 &#195;&#8212; 103/&#194;&#181;g DNA)         YkoU ATP-dependent ligases expressed detect- <br /> were obtained, 48 clones analysed,               able level strain. <br /> resistant kanamycin, marker plasmid pMAP141. This <br /> showed transformants &#226;&#8364;&#732;false positives&#226;&#8364;&#8482;. In          The yoqV ligase does complement ligts mutant of <br /> strain 168 harbouring pMAP139, 6 &#195;&#8212; 103 transformants &#194;&#181;g         E.coli, yerG ligase does <br /> of DNA obtained, result similar strain 168. This          In E.coli, C-terminal human ATP-dependent <br /> suggested transformants resulted congression   ligase I able restore thermoresistance ligts strain (7), <br /> of yerG::EmR pMAP141 DNA. Unlike recipient strain               S.typhimurium bacteriophage T4 ATP-dependent <br /> 168, strain containing plasmid pMAP139              ligase, contains minimal adenylation domain, like <br /> resistant kanamycin congression         yoqV gene product, allows growth NAD-dependent <br /> tested phenotypically. To check presence plasmid         ligase disrupted strain (4). We investigated whether <br /> pMAP141 transformants, total DNA 18 clones            YoqV able complement ligts strain E.coli. For this, <br /> extracted, plasmid chromosomal DNA                  plasmid pMAP138, hybrid composed pBR322, lacI and <br /> analysed Southern blotting. In clones, plasmid pMAP141       pMAP139, constructed. The plasmid transformed into <br /> was effectively, combination          strain JJC75, containing lig7 ts allele, resulting <br /> plasmid pMAP139. Therefore proportion 0 18 clones        strain grown various temperatures. Plates incubated at <br /> contained yerG disruption (Table 3, lane 2).                      42 30&#194;&#176;C shown Figure 4. As ligts, strain <br />                                                                                                Nucleic Acids Research, 2000, Vol. 28, No. 23 4647 <br />  <br />  <br />  <br />  <br />                                                                                    Figure 5. UV sensitivity wild-type E.coli strain JJC40 grown 42&#194;&#176;C <br />                                                                                    (squares), ligts derivative JJC75 grown 30&#194;&#176;C (circles) JJC75 harbouring <br />                                                                                    yerG encoding plasmid pMAP140, grown 42&#194;&#176;C (triangles). The fraction <br /> Figure 4. yerG, yoqV, restores thermoresistance E.coli ligts        surviving clones reported function UV dose. <br /> strain. Petri plates showing streaks LB E.coli ligts strain harbouring <br /> no plasmid (no plasmid), plasmid pMAP138 [pMAP138 (yoqV)] pMAP140 <br /> [pMAP140 (yerG)], grown 42 (top) 30&#194;&#176;C (bottom). <br />                                                                                    (squares). In conclusion, yerG gene product allowed the <br />                                                                                    restoration growth UV resistance defects the <br />                                                                                    ligts strain. <br /> harbouring plasmid pMAP138 did grow 42 37&#194;&#176;C (not <br /> shown), presence absence IPTG. To further <br /> confirm defect, cells grown exponentially 30&#194;&#176;C                    DISCUSSION <br /> the plating efficiency 42&#194;&#176;C relative 30&#194;&#176;C determined.                    In B.subtilis, shown yerG-encoded NAD- <br /> Only 10&#226;&#8364;&#8220;5 survivors 42&#194;&#176;C ligts strain,                      dependent ligase, propose rename ligA, essential <br /> with pMAP138. This shows B.subtilis ATP-                       growth. This somewhat unexpected, given pres- <br /> dependent ligase does complement growth defect                      ence potential ligase genes chromosome. <br /> ligts strain.                                                                      These ligases, ykoU yoqV, belong family <br />    To verify ATP-dependent ligase cloned                             ATP-dependent ligases. Observations shown some <br /> pMAP138 active, adenylation test performed                           ATP-dependent ligases, originating organisms <br /> E.coli ligts crude extracts (Fig. 3, lanes 1&#226;&#8364;&#8220;3). A radioactive                     completely unrelated bacteria, T4 bacteriophage <br /> signal detected samples extracted YoqV                             Homo sapiens, able restore viability a <br /> containing strain, absent extract did                      S.typhimurium E.coli lig mutant (4,7), therefore <br /> contain YoqV. It 10-fold intense sample                         suggesting surprisingly broad range activity compati- <br /> which expression yoqV derepressed addition                        bility enzymes. <br /> of IPTG. This showed ATP-dependent ligase encoded                            Because yoqV gene SP&#206;&#178; prophage, the <br /> by yoqV active E.coli crude extracts.                                       possibility existed absence complementation was <br />    Similar heterologous complementation experiments                           simply lack sufficient level expression. To <br /> carried parallel yerG encoded NAD ligase.                          increase expression yoqV encoded ATP-dependent <br /> Plasmid pMAP140, hybrid pBR322, lacI pMAP141,                             ligase, propose rename ligB, gene cloned <br /> which provides IPTG-dependent expression B.subtilis                         plasmid appropriate expression signals. Even this <br /> NAD-dependent ligase E.coli, constructed                                case, yerG gene disrupted. To investigate <br /> introduced ligts background. In contrast yoqV                        activity yoqV, gene cloned E.coli its <br /> experiment, strain did form individual colonies 42&#194;&#176;C                       ability complement growth defect lig7ts strain was <br /> (Fig. 4), plated efficiency close 100%                     tested. Again, 42&#194;&#176;C yoqV failed restore growth ligts <br /> temperature, absence IPTG. This shows                      mutant. To exclude possibility ligase encoded by <br /> B.subtilis NAD-dependent ligase complements growth                             yoqV fact inactive enzyme, adenylation activity in <br /> defect ligts strain.                                                        B.subtilis E.coli crude extracts tested be <br />    Because YerG ligase able restore growth                       present. It appears that, B.subtilis in <br /> ligts strain 42&#194;&#176;C, ability complement UV sensitivity              E.coli, ATP-dependent ligases provide sufficient <br /> the ts strain investigated. Whereas ligts strain               activity maintaining alive bacterial cells devoid their <br /> UV sensitive 30&#194;&#176;C (Fig. 5, circles), defect disappeared                 native NAD-dependent ligase. <br /> the presence yerG, 42&#194;&#176;C (triangles), strain                        In cases complementation reported, it <br /> became UV resistant original wild-type strain                            noted phenotypes restored. Neither <br />  4648 Nucleic Acids Research, 2000, Vol. 28, No. 23 <br />  <br />  <br /> the Chlorella virus ATP-dependent ligase E.coli NAD-         H.influenzae, harbour ligase gene,  <br /> </body></html>