<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The mitotic spindle required <br /> for loading DASH complex <br /> onto kinetochore <br /> Yumei Li,1 Jeff Bachant,1 Annette A. Alcasabas,1 Yanchang Wang,1 Jun Qin,1,2 <br /> and Stephen J. Elledge1,3,4,5 <br /> 1 <br />  Verna Marrs McLean Department Biochemistry Molecular Biology, 2Department Cell Biology, 3Department <br /> of Molecular Human Genetics, 4Howard Hughes Medical Institute, Baylor College Medicine, <br /> Houston, Texas 77030, USA <br />  <br />  <br />  <br />  <br /> A role mitotic spindle maturation kinetochore defined previously. Here we <br /> describe isolation novel conserved essential gene, ASK1, Saccharomyces cerevisiae involved <br /> in process. ask1 mutants display G2/M arrest segregation DNA masses separation <br /> of sister chromatids, resulting massive nondisjunction broken spindles. Ask1 localizes mitotic <br /> spindles kinetochores, cross-links centromeric DNA. Microtubules required Ask1 binding <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> kinetochores, partially required maintain association. We Ask1 a <br /> multisubunit complex, DASH, contains &#226;&#710;&#188;10 components, including proteins essential mitosis <br /> including Dam1, Duo1, Spc34, Spc19, Hsk1. The Ipl1 kinase controls phosphorylation Dam1 the <br /> DASH complex regulate function. We propose DASH microtubule-binding complex that <br /> is transferred kinetochore prior mitosis, defining new step kinetochore maturation. <br /> [Key Words: Ask1; spindle; kinetochore; chromosome segregation] <br /> Received November 2, 2001; revised version accepted November 27, 2001. <br />  <br />  <br />  <br />  <br /> Mitosis period cell cycle chro-                lyzed protease, Esp1, activated de- <br /> mosomes segregated daughter cells                      struction inhibitor, Pds1 (Cohen-Fix et al. 1996; <br /> process anaphase. The ability accurately distribute                 Ciosk et al. 1998; Uhlmann et al. 1999). <br /> replicated chromosomes critical proper transfer                   Anaphase does occur chromosomes form <br /> of genetic material generation next. Er-                  bipolar attachment spindle. A centromere- <br /> rors process lead developmental defects,                   based mitotic checkpoint referred the <br /> cancer, death. Cells ensure fidelity process               spindle checkpoint (Amon 1999) monitors spindle&#226;&#8364;&#8220;chro- <br /> through multiple layers control operate                    mosome interactions blocks anaphase sister <br /> mitosis.                                                                  chromatids successfully attached spindle micro- <br />    The physical movement chromosomes ana-                       tubules bipolar fashion (Wang Burke 1995; Pan- <br /> phase carried mitotic spindle. The mitotic                  gilinan Spencer 1996). In budding yeast, six <br /> spindle bipolar microtubule-based structure                     genes, MAD1&#226;&#8364;&#8220;MAD3, BUB1, BUB3, MPS1 (Hoyt et <br /> emanates organizing centers, called centro-                      al. 1991; Li Murray 1991; Hardwick Murray <br /> somes spindle-pole bodies (SPB), connects chro-                 1995; Hardwick et al. 1996; Farr Hoyt 1998), are <br /> mosomes (Hoyt Geiser 1996; Winey O&#226;&#8364;&#8482;Toole                          required spindle checkpoint prevent anaphase. <br /> 2001). The mitotic spindle binds newly replicated                      When active, spindle checkpoint prevents anaphase <br /> chromosomes (called sister chromatids), chromo-                       inhibiting ubiquitin-mediated degradation ana- <br /> some segregation does place cohesion                       phase inhibitor Pds1/Cut2 (Cohen-Fix et al. 1996; Fun- <br /> along sister chromatids eliminated. The elimina-                   abiki et al. 1996; Yamamoto et al. 1996; Hwang et al <br /> tion cohesion highly regulated step cata-                 1998; Kim et al. 1998). <br />                                                                              How spindle forms, recognizes, binds chro- <br />                                                                           mosomes currently topic extensive research. Sev- <br /> 5 <br />  Corresponding author.                                                    eral proteins known associated spindles <br /> E-MAIL seledge@bcm.tmc.edu; FAX (713) 798-8717. <br /> Article publication http://www.genesdev.org/cgi/doi/10.1101/   play important functions spindle assembly and <br /> gad.959402.                                                               dynamics. These proteins include motor proteins, such <br />  <br />  <br />       GENES &amp; DEVELOPMENT 16:183&#226;&#8364;&#8220;197 &#194;&#169; 2002 Cold Spring Harbor Laboratory Press ISSN 0890-9369/02 $5.00; www.genesdev.org    183 <br />  Li et al. <br />  <br />  <br />  <br /> as Kar3, Kip1, Cin8 (Hoyt et al. 1992; Roof et al.        Results <br /> 1992; Saunders Hoyt 1992), microtubule <br /> associated proteins, Ase1 (Pellman et al. 1995)       The ask1-1 mutant defective maintaining spindle <br /> and Esp1 (Jensen et al. 2001).                                integrity HU <br />    The spindle microtubules attach chromosomes             To identify genes involved restraint anaphase <br /> through kinetochore, specialized structure consist-     entry S-phase arrest, genetic screen con- <br /> ing centromeric DNA associated proteins        ducted search mutants elongated <br /> (Lechner Ortiz 1996; Pidoux Allshire 2000). In        spindle <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> presence hydroxyurea (HU), <br /> budding yeast, centromere (CEN) short sequence       inhibitor DNA replication. Whereas wild-type cells <br /> consisting well-conserved elements important         arrested HU short spindles, mutants isolated <br /> for centromere function, CDEI, CDEII, CDEIII.             screen display elongated spindles pres- <br /> CDEIII CDEII indispensable       ence HU (Alcasabas et al. 2001). Among mu- <br /> centromere function (Hegemann Fleig 1993). Several        tants isolated screen ask1-1. ask1-1 mutant <br /> kinetochore proteins identified bind      cells failed grow plates containing 150 mM HU <br /> specific centromere region. Cbf1 binds CDEI         (Fig. 1A). Cells lost viability time incubated <br /> involved fidelity chromosome transmission           HU, indicating undergoing cata- <br /> (Cai Davis 1990). Ndc10 (Doheny et al. 1993; Goh          strophic event (Fig. 1B). To investigate ASK1 is <br /> and Kilmartin 1993; Jiang et al. 1993), Ctf13 (Doheny et      involved replication checkpoint pathway, ex- <br /> al. 1993), Cep3 (Lechner 1994; Strunnikov et al. 1995),       amined well-characterized checkpoint responses in <br /> and Skp1 (Connelly Hieter 1996) bind CDEIII       ask1-1 mutant cells, Rad53 hyperphosphorylation and <br /> multisubunit protein complex (CBF3). In addition, Cse4        RNR3 transcriptional induction (Elledge Davis 1990; <br /> (Meluh et al. 1998), homolog CENP-A, hypoth-          Sanchez et al. 1996). To examine Rad53 phosphoryla- <br /> esized involved formation specialized     tion, extracts wild-type ask1-1 mutants pre- <br /> centromere nucleosome. Mif2 (Meluh Koshland               pared cells treated 200 mM HU 0.5% MMS, <br /> 1995, 1997), homolog CENP-C, interacts CEN          Western blotting carried using anti-Rad53 <br /> DNA CDEIII-dependent manner. Other proteins,             antibodies. In wild-type ask1-1 mutant cells, <br /> such Ctf19, Mcm21, Okp1 important         modification Rad53 observed HU MMS <br /> the formation kinetochore (Hyland et al. 1999;         treatment (Fig. 1C). The transcriptional induction of <br /> Ortiz et al. 1999).                                           RNR3 examined similar conditions and <br />    Once kinetochore attached microtubules,       fully intact (data shown). Thus, were <br /> is likely mechanisms exist alter properties.      unable establish connection Ask1 and <br /> One piece evidence supporting hypothesis       known components replication checkpoint path- <br /> presence particular phosphorylation event recog-         way, indicating Ask1 downstream effec- <br /> nized 3F3/2 antibody localized specifically     tor pathway Ask1 independent <br /> kinetochores unattached chromosomes (Gorbsky           mechanism required maintenance spindle in- <br /> Ricketts 1993). Further evidence comes pres-         tegrity prolonged S-phase arrest. <br /> ence Mad2 protein chromosomes, <br /> presumably signaling prevent anaphase entry (Chen et <br />                                                               ASK1 essential gene encoding novel protein <br /> al. 1996). It unknown kinetochore altered <br /> by microtubule attachment. Presumably, sig-        ASK1 (YKL052C) cloned complementing HU <br /> naling event, possibly structural change, alters   sensitivity ask1-1 mutants. It encodes protein of <br /> kinetochore microtubule bound. Other changes        293 amino acids obvious conserved motifs. Da- <br /> may accompany binding strengthening           tabase analysis identified sequence homologs present in <br /> the attachment itself pursuant segregation possi-      Schizosaccharomyces pombe (SPBC27.02c) Dro- <br /> bly signaling kinetochore spindle regu-    sophila melanogaster (Fig. 1D). Deletion ASK1 was <br /> late spindle dynamics cis. In study,      performed diploid cells gene replacement. Tetrad <br /> the identification novel protein, Ask1 (associated       dissection revealed ask1 null mutant inviable <br /> with spindles kinetochores), required maintain-       genetically linked original ask1-1 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> ing proper kinetochore-microtubule attachments            (data shown). <br /> chromosome segregation. Our data suggests Ask1              To investigate function ASK1, conditional al- <br /> initially binds microtubules subsequently binds        leles ASK1 generated PCR mutagenesis. Six <br /> kinetochores microtubule attachment S             temperature-sensitive alleles isolated used to <br /> phase, new kinetochores assembled. We dis-           replace endogenous ASK1 gene. The phenotypes of <br /> covered Ask1 complex DASH           mutants investigated use synchronous <br /> that contains multiple essential proteins. The Ipl1 ki-       cultures. Cells arrested G1 &#226;?&#163;-factor 25&#194;&#176;C, <br /> nase controls phosphorylation component       shifted 37&#194;&#176;C 30 min release <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> DASH complex, Dam1, regulate function.            block. On basis DNA spindle morpholo- <br /> Thus, DASH complex alter property          gies, temperature-sensitive alleles divided <br /> kinetochore allow achieve mature spindle-        classes. One class contains single allele, ask1- <br /> bound form.                                                   2, arrested large budded cells 2N DNA <br />  <br />  <br /> 184         GENES &amp; DEVELOPMENT <br />                                                                                      Spindle-dependent loading DASH kinetochore <br />  <br />  <br />  <br />  <br /> Figure 1. Characterization ask1-1 mutant cells. (A) ask1-1 mutants HU sensitive. Wild-type (Y300) ask1-1 (Y928) mutants <br /> were streaked YPD plates containing 150 mM HU 30&#194;&#176;C incubated 3 d. (B) ask1-1 loses viability HU. &#226;?&#163;-factor arrested <br /> wild-type (Y300) ask1-1 (Y928) mutant cells released YPD containing 200 mM HU indicated times plated <br /> on YPD 30&#194;&#176;C determine <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span>. (C) Proper regulation Rad53 ask1-1 mutants. Log-phase cells wild-type (Y300), ask1-1 <br /> (Y928), mec1-21 (Y604) treated 200 mM HU (lane 2) 0.5% MMS (lane 3) 1.5 h 30&#194;&#176;C. Protein extracts were <br /> prepared, separated SDS/PAGE, immunoblotted antibodies Rad53. (Lane 1) Untreated control. (D) Alignment of <br /> S. cerevisiae Ask1 protein homologs S. pombe D. melanogaster. Identical residues highlighted black and <br /> conserved residues gray. <br />  <br />  <br />  <br /> content, undivided nuclei, short spindles (Fig. 2A).            reappeared (Fig. 2B). Cells analyzed 90-min time <br /> These mutant cells maintained viability in-              point showed unusually stretched DNA masses an <br /> cubated 37&#194;&#176;C (data shown). The remaining tem-                abnormal spindle morphology &#226;&#8364;&#8221; microtubule stain- <br /> perature-sensitive alleles fall second class,              ing SPBs greatly reduced the <br /> whose representative mutant ask1-3. When ask1-3                  SPBs apart relative normal preanaphase <br /> mutant cells released G1 arrest restric-           SPBs (Fig. 2B). Meanwhile, viability ask1-3 mutant <br /> tive temperature (35&#194;&#176;C), arrested temporarily           cells began decline 60 min release (data not <br /> 2N DNA content &#226;&#710;&#188;120 min G1 cells                    shown). <br />  <br />  <br />                                                                                                   GENES &amp; DEVELOPMENT             185 <br />  Li et al. <br />  <br />  <br />  <br />                                                                    ask1-2 mad2 double mutants failed arrest at- <br />                                                                    tempted segregate chromosomes elongated <br />                                                                    spindles (Fig. 2C). Thus, ask1-2 mutants activate the <br />                                                                    spindle checkpoint cause preanaphase arrest. <br />  <br />  <br />                                                                    ask1-3 mutants undergo massive nondisjuction <br />                                                                    sister chromatid separation <br />                                                                    The ask1-3 class mutants appear severe <br />                                                                    ask1-2 mutants expected to <br />                                                                    activate spindle checkpoint. Yet, ask1-3 mu- <br />                                                                    tants appear segregate chromosomes eventually <br />                                                                    exit mitosis (Fig. 2B, 220-min timepoint). To explore this <br />                                                                    hypothesis, examined aspects spindle <br />                                                                    checkpoint function ask1-3 mutants including the <br />                                                                    phenotypic consequences mad2 mutants, degra- <br />                                                                    dation Pds1, sister chromatid segregation. <br />                                                                       To examine effects mad2 phenotype of <br />                                                                    ask1-3 mutants, ask1-3 mad2 double mutant was <br />                                                                    constructed cell cycle progression examined <br />                                                                    synchronous cultures 35&#194;&#176;C. ask1-3 mutant cells <br />                                                                    displayed delayed mitotic progression. Cells 1N <br />                                                                    DNA content did reappear, 150 min G1 <br />                                                                    release. However, ask1-3 mad2 double mutant cells pro- <br />                                                                    gressed mitosis delay (FACS, Fig. 3A) <br />                                                                    G1 cells reappeared 90 min release, <br />                                                                    similar mad2 cells. In addition, abnormal spindle <br />                                                                    defects evident ask1-3 mutants longer ob- <br />                                                                    served double mutants (Fig. 3A), suggesting that <br />                                                                    spindle checkpoint active ask1-3 mutants <br /> Figure 2. Characterization temperature-sensitive alleles     profound effect spindle phenotype of <br /> ASK1. (A) ask1-2 mutant cells arrest anaphase entry. <br />                                                                    cells. However, unequal distribution chromatin <br /> ask1-2 mutant cells (Y1104) synchronized &#226;?&#163;-factor and <br />                                                                    evident ask1-3 mad2 cells, indicating a <br /> released YPD 37&#194;&#176;C. (Left) The DNA spindle mor- <br /> phology ask1-2 cells 90 min following release. (Right)   defect anaphase. <br /> The DNA content determined FACS analysis. (B) ask1-3 mu-           Pds1 inhibitor Esp1, protease cleaves <br /> tants display unusual spindle phenotypes restrictive tem-   Mcd1/Scc1 cohesin subunit. Pds1 ubiquityla- <br /> perature. Wild-type (Y300) ask1-3 (Y1103) mutant cells         tion substrate anaphase promoting complex (APC) <br /> were synchronized &#226;?&#163;-factor released 35&#194;&#176;C. At in-     degradation required cells enter ana- <br /> dicated times release, aliquots withdrawn ex-    phase. Both wild-type ask1-3 mutant cells carrying <br /> amine viability, DNA content, nuclear spindle morphology.      Pds1&#226;&#8364;&#8220;18xMyc analyzed Pds1 protein levels. <br /> Photomicrographs wild-type ask1-3 mutant cells         Cells arrested G1, shifted 35&#194;&#176;C 30 min, and <br /> 90 min release. (C) ask1-2 (Y1104) ask1-2 mad2 <br />                                                                    released. Fifty-five minutes release, &#226;?&#163;-fac- <br /> (Y1106) mutant cells synchronized released A. <br />                                                                    tor added prevent cells entering the <br /> Aliquots collected FACS analysis, DNA, spindle <br /> staining indicated times. Photomicrographs taken    cell cycle. In wild-type cells, Pds1 levels increased <br /> cells 90 min release.                                 30 min release, declined 75 min. In ask1-3 <br />                                                                    mutant cells, Pds1 levels maintained high levels <br />                                                                    experiment, indicating Pds1 sta- <br /> The spindle checkpoint activated ask1-2 mutants <br />                                                                    bilized ask1-3 mutant cells, presumably the <br /> Anaphase entry tightly regulated checkpoints. Ac-            activation spindle checkpoint (Fig. 3B). <br /> tivation DNA damage checkpoint                   If Pds1 levels remain high activation the <br /> spindle checkpoint leads preanaphase arrest. To deter-          spindle checkpoint, cohesion main- <br /> mine cause arrest ask1-2 mutants, double mu-             tained unless ASK1 plays role cohesion process <br /> tants ask1-2 rad9 mad2 con-                 itself. To address this, followed sister-chromatid <br /> structed. RAD9 key component DNA damage                    separation using GFP-marked chromosomes (Michaelis <br /> checkpoint, MAD2 required spindle               et al. 1997). &#226;?&#163;-factor arrested wild-type, ask1-3, and <br /> checkpoint pathway. In synchronous cultures, ask1-2                ask1-3 mad2 mutant cells followed time after <br /> rad9 mutant cells arrested large budded cells            release 35&#194;&#176;C. In wild-type ask1-3 mad2 cells, <br /> 2N DNA content restrictive temperature (data            sister-chromatid separation began 60 90 <br /> shown), indicating ask1-2 mutants require              min release. However, ask1-3 mutants, despite <br /> the DNA damage checkpoint arrest. In contrast,                 segregated DNA masses, sister chromatids failed to <br />  <br />  <br /> 186         GENES &amp; DEVELOPMENT <br />                                                                              Spindle-dependent loading DASH kinetochore <br />  <br />  <br />  <br />  <br />                                                                               Figure 3. ask1-3 mutants undergo massive <br />                                                                               nondisjunction segregate chromosomes <br />                                                                               separation sister chromatids. <br />                                                                               (A) The spindle checkpoint active ask1-3 <br />                                                                               mutants. ask1-3 (Y1103), ask1-3 mad2 <br />                                                                               (Y1107), mad2 (Y1101) mutants ar- <br />                                                                               rested G1 &#226;?&#163;-factor released at <br />                                                                               35&#194;&#176;C. Ninety minutes release, <br />                                                                               samples collected stained with <br />                                                                               DAPI (DNA) anti-tubulin antibodies <br />                                                                               (spindles). (B) Pds1 stabilized ask1-3 mu- <br />                                                                               tants. Wild-type (Y998) ask1-3 mutant <br />                                                                               (Y1108) cells carrying PDS1 gene fused to <br />                                                                               18 copies myc epitope synchro- <br />                                                                               nized &#226;?&#163;-factor 24&#194;&#176;C released <br />                                                                               35&#194;&#176;C. Cells harvested 15 min <br />                                                                               processed protein preparation. Pds1 <br />                                                                               levels determined Western blotting <br />                                                                               using anti-Myc antibodies. (C) Analysis sis- <br />                                                                               ter-chromatid separation arrested ask1-3 <br />                                                                               mutant cells. Wild-type (Y974), ask1-3 mu- <br />                                                                               tant (Y1109), ask1-3 mad2 (Y1110) cells <br />                                                                               containing Tet&#226;&#8364;&#8220;GFP fusion tandem re- <br />                                                                               peats Tet operator integrated the <br />                                                                               URA3 locus 35 kb away centromere on <br />                                                                               left arm ChrV synchronized by <br />                                                                               &#226;?&#163;-factor 24&#194;&#176;C released 35&#194;&#176;C. <br />                                                                               Ninety minutes release, samples were <br />                                                                               collected examine sister chromatid separa- <br />                                                                               tion immunofluorescence microscopy. <br />  <br />  <br />  <br /> separate 150 min release (Fig. 3C). Thus,       cells set elongated spindles, dots were <br /> ask1-3 mutant cells undergo premature segregation          observed vicinity spindle pole bodies. Ask1 <br /> chromosome separation sister chroma-           staining mitotic spindle pre- <br /> tids, leading massive nondisjunction. In addition,      anaphase anaphase B (Fig. 4A). The re- <br /> ask1-3 mad2 cells, sister chromatids separated,      sult obtained using strains Ask1 was <br /> 60% time sisters remained cell body        tagged copies Myc-epitope car- <br /> and failed segregate. This form nondisjunction       boxyl terminus (data shown). <br /> consistent defect bipolar spindle attachment.         During mitosis, centromeres sister chromatids tran- <br />                                                               siently separate opposite ends spindles and <br />                                                               localize close SPBs (Goshima Yanagida <br /> Ask1 associates centromeres vivo <br />                                                               2000; He et al. 2000). To examine Ask1 lo- <br /> Cellular localization indicative protein func-    calized SPB kinetochore, localization of <br /> tion. Ask1 tagged GFP carboxyl terminus       Ask1 examined ndc10-1 mutant background. <br /> and integrated ASK1 locus wild-type cells. In-      Ndc10 essential kinetochore component. ndc10-1 <br /> direct immunofluorescence microscopy performed            mutants elongate spindles segregating chro- <br /> using DAPI, antibodies GFP, tubulin vi-        mosomes. Cells grown mid-log phase 24&#194;&#176;C <br /> sualize DNA, Ask1p, spindles, respectively. A dot-        synchronized G1 using &#226;?&#163;-factor. After release at <br /> like GFP signal clearly seen cells cell cycle   35&#194;&#176;C, cells collected immunofluorescence <br /> stages. A single dot seen G1 cells. Later,        staining. In ndc10-1 mutants, staining Ask1 along <br />  <br />  <br />                                                                                           GENES &amp; DEVELOPMENT             187 <br />  Li et al. <br />  <br />  <br />  <br />  <br /> Figure 4. Ask1 centromere binding pro- <br /> tein. (A) Ask1 localizes mitotic spindle <br /> and spindle-pole bodies. Cells carrying in- <br /> tegrated ASK1&#226;&#8364;&#8220;GFP gene (Y1111) syn- <br /> chronized G1 &#226;?&#163;-factor released at <br /> 30&#194;&#176;C. Samples different stages cell <br /> cycle fixed 5% formaldehyde 30 <br /> min immunofluorescence staining. (B) <br /> Localization Ask1 wild-type and <br /> ndc10-1 mutants. G1-arrested wild-type <br /> (Y1111) ndc10-1 (Y1112) cells containing <br /> Ask1&#226;&#8364;&#8220;GFP released 35&#194;&#176;C immu- <br /> nofluorescence microscopy. Representative <br /> telophase cells shown. (C) Chromatin im- <br /> munoprecipitation analysis (ChIP) Ask1&#226;&#8364;&#8220; <br /> Myc wild-type (Y1113) ndc10-1 <br /> (Y1115) cells. Untagged wild-type (Y300) cells <br /> (U) included control. Log-phase cul- <br /> tures shifted 35&#194;&#176;C 3 h col- <br /> lecting samples. (D) Synthetic growth defects <br /> in ask1-3 ndc10-1 double mutants (Y1120). <br /> Tenfold dilutions cells spotted on <br /> plates incubated 24&#194;&#176;C 30&#194;&#176;C. (E) ask1-3 <br /> mutants exhibit elevated rates chromo- <br /> some loss. Wild-type (Y1117) ask1-3 mu- <br /> tant (Y1118) strains harboring CFIII were <br /> grown SC-uracil medium 24&#194;&#176;C then <br /> plated YPD plates 32&#194;&#176;C incu- <br /> bated 4 d. Red sectors represent chromo- <br /> some loss events. <br />  <br /> the spindle present, however, dot-like SPB   creased ndc10-1 mutants (Fig. 4C). Consistent with <br /> localization longer observed telophase cells;    centromere binding Ask1, ask1-3 displayed syn- <br /> instead, diffuse nuclear staining (Fig. 4B).   thetic, enhanced temperature sensitivity phenotype <br /> Thus, dot-like localization Ask1 depended        ndc10-1 30&#194;&#176;C (Fig. 4D). <br /> Ndc10 raised possibility Ask1 associates        Given dramatic nondisjunction phenotypes of <br /> with kinetochore.                                      ask1-3 mutants nonpermissive temperature, we <br />   To test hypothesis, used chromatin immu-     anticipated ask1 mutants display signifi- <br /> noprecipitation (ChIP) assay. Formadehyde cross-linked     chromosome loss phenotype. To test this, ex- <br /> chromatin prepared ASK1&#226;&#8364;&#8220;MYC untagged con-         amined chromosome segregation ask1-3 mutants <br /> trol strains sonicated immunoprecipitated         colony color sectoring assay (Spencer et al. 1990). <br /> with anti-Myc antibodies. The DNA bound Ask1        The strains analyzed harbored mini-chromosome CFIII <br /> analyzed PCR using specific primers. Two CEN DNA        carrying SUP11 suppresses ade2-101 mutation <br /> sequences tested, CEN3 CEN16,       present cells. In absence SUP11, ade2- <br /> found enriched Ask1&#226;&#8364;&#8220;Myc immunoprecipitates,         101 mutation causes accumulation red pigment. <br /> but untagged control strain. The noncentro-     Loss CFIII results red sector. In wild- <br /> meric region PGK1 analyzed control      type cells, mini-chromosome stable colonies <br /> found Ask1&#226;&#8364;&#8220;Myc immunoprecipitates (Fig. 4C).        appear white. However, ask1-3 mutant cells, highly <br /> This suggested Ask1 specifically associates      red-sectored colonies appear semipermissive tempera- <br /> centromeres. Because dot-like localization Ask1     tures 30&#194;&#176;C 32&#194;&#176;C (Fig. 4E). <br /> depends Ndc10, examined centromere <br /> binding Ask1 ndc10-1 mutant background. Un- <br />                                                            The centromere binding Ask1 microtubule <br /> tagged control, wild-type ASK1&#226;&#8364;&#8220;MYC ndc10-1 <br />                                                            dependent <br /> ASK1&#226;&#8364;&#8220;MYC strains shifted 35&#194;&#176;C 3 h before <br /> the collection samples. ChIP analysis revealed     Because Ask1 associated spindles the <br /> the centromere binding Ask1 significantly de-        kinetochore, raised possibility kineto- <br />  <br />  <br /> 188         GENES &amp; DEVELOPMENT <br />                                                                                        Spindle-dependent loading DASH kinetochore <br />  <br />  <br />  <br /> chore association dependent spindle.                (10-fold reduction) cells entering G2 phase cell <br /> To examine this, carried ChIP analysis                  cycle presence nocodazole (Fig. 5A,F,G). The <br /> presence nocodazole, microtubule depolymerization                result observed wild-type cells releas- <br /> reagent. To eliminate cell cycle stage differences be-                ing G1 arrest nocodazole (data shown). Ti- <br /> tween experimental control samples, cdc13-1                     tration template DNA indicated PCR am- <br /> ASK1&#226;&#8364;&#8220;MYC strain used experiments be-                     plification linear range (Fig. 5D). <br /> cause cdc13-1 arrested preanaphase. Cells ar-                   In addition depolymerizing microtubules, noco- <br /> rested G1 &#226;?&#163;-factor released 32&#194;&#176;C                dazole treatment activates spindle checkpoint <br /> presence nocodazole. Seventy minutes release,                pathway. To distinguish decrease centro- <br /> cells reached G2 collected. We cen-                mere binding loss microtubules the <br /> tromere binding Ask1 significantly diminished                  activation spindle checkpoint, MAD2 deleted <br />  <br />  <br />                                                                                           Figure 5. The association Ask1 the <br />                                                                                           centromere requires intact microtubules. <br />                                                                                           (A) Ask1 centromeric binding inhibited <br />                                                                                           nocodazole. &#226;?&#163;-factor-arrested cdc13-1 <br />                                                                                           ASK1&#226;&#8364;&#8220;MYC (Y1114) cells released <br />                                                                                           prewarmed YPD (G1&#226;&#8364;&#8220;G2 + Noc) <br />                                                                                           (G1&#226;&#8364;&#8220;G2) nocodazole 32&#194;&#176;C. <br />                                                                                           Seventy minutes later, samples col- <br />                                                                                           lected ChIP analysis. Untagged wild- <br />                                                                                           type cells included control (U). (B) <br />                                                                                           Depolymerization microtubules, not <br />                                                                                           activation spindle checkpoint, <br />                                                                                           blocks loading Ask1 centromeric <br />                                                                                           DNA. G1-arrested cdc13-1 mad2 (Y1116) <br />                                                                                           cells containing Ask1&#226;&#8364;&#8220;Myc released <br />                                                                                           medium containing nocodazole 70 <br />                                                                                           min processed ChIP analysis as <br />                                                                                           above. Untagged wild-type cells in- <br />                                                                                           cluded control (U). (C) The noco- <br />                                                                                           dazole-dependent centromere binding of <br />                                                                                           Ask1 reversible. &#226;?&#163;-factor-arrested <br />                                                                                           cdc13-1 ASK1&#226;&#8364;&#8220;MYC (Y1114) cells (G1) <br />                                                                                           released prewarmed YPD with <br />                                                                                           nocodazole 32&#194;&#176;C (G1&#226;&#8364;&#8220;G2 + Noc). Sev- <br />                                                                                           enty minutes later, cells spun down <br />                                                                                           resuspended fresh YPD 32&#194;&#176;C to <br />                                                                                           remove nocodazole. Samples col- <br />                                                                                           lected 30-min intervals ChIP analy- <br />                                                                                           sis (&#226;&#710;&#8217;Noc 30&#226;&#8364;&#8482;, &#226;&#710;&#8217;Noc 60&#226;&#8364;&#8482;). Untagged wild- <br />                                                                                           type cells included control (U). (D) <br />                                                                                           Control showing PCR reactions <br />                                                                                           linear range. Twofold serial di- <br />                                                                                           lutions precipitated input <br />                                                                                           samples used lanes 1 7 F were <br />                                                                                           subjected PCR. (E) The centromere <br />                                                                                           binding Ask1 cell cycle. ASK1&#226;&#8364;&#8220; <br />                                                                                           MYC (Y1113) cells arrested G1 <br />                                                                                           &#226;?&#163;-factor released 30&#194;&#176;C. <br />                                                                                           Samples collected 20 min for <br />                                                                                           ChIP analysis. DNAs total (T) and <br />                                                                                           immunoprecipitaed (IP) samples ana- <br />                                                                                           lyzed PCR products quanti- <br />                                                                                           tated using NIH image software. Percent <br />                                                                                           budding samples shown below. <br />                                                                                           (F) The centromere binding Ask1 G1 <br />                                                                                           G2-arrested cells sensitive to <br /> nocodazole treatment. cdc13-1 ASK1&#226;&#8364;&#8220;MYC (Y1114) cells arrested G1 &#226;?&#163;-factor G2 incubating 32&#194;&#176;C before <br /> nocodazole added. Seventy minutes later, samples collected ChIP analysis. Cells released G1 G2 were <br /> included control (lanes 4,5). (Bottom) The amounts Ask1&#226;&#8364;&#8220;Myc protein immunoprecipitated determined Western analysis. <br /> (Right) The control showing nocodazole treatment destabilized microtubules. Shown anti-tubulin immunoflourescence <br /> images cells used prepare DNA used lanes 2&#226;&#8364;&#8220;7. (G) Quantification PCR reactions F. PCR products F run <br /> on 3% agarose gel, visualized ethidium bromide staining, scanned quantitation using NIH Image software. <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT             189 <br />  Li et al. <br />  <br />  <br />  <br /> from strain ChIP analysis carried out.       complex DASH (for Dam1/Duo1, Ask1, Spc34/Spc19, <br /> When cdc13-1 mad2 cells released G1 32&#194;&#176;C          Hsk1). <br /> in presence nocodazole, centromere binding          Dam1 Duo1 identified previously proteins <br /> Ask1 decreased (Fig. 5B), indicating failure     localized spindles kinetochores like <br /> in Ask1 binding centromeric DNA presence          Ask1, function maintain spindle integrity the <br /> nocodazole activation spindle         fidelity chromosome segregation (Jones et al. 1999; <br /> checkpoint, loss microtubules. We con-       Cheeseman et al. 2001; He et al. 2001). The interaction <br /> clude mitotic spindle required Ask1 bind-      Ask1 proteins confirmed by <br /> ing centromere.                                         coimmunoprecicpitation using Ask1&#226;&#8364;&#8220;Myc (Fig. 6B), indi- <br />   Nocodazole does permanently inactivate Ask1          cating association dependent the <br /> protein nocodazole-dependent interference          epitope tag used. The reciprocal immunoprecipitation of <br /> with Ask1&#226;&#8364;&#8482;s centromeric association reversible. When        Duo1 anti-Duo1 antibodies led coprecipita- <br /> nocodazole removed cells spindles           tion Ask1&#226;&#8364;&#8220;Myc (Fig. 6B), supporting exis- <br /> are allowed form captured kinetochore,        tence complex. The interaction evident by <br /> Ask1 resumed centromeric association (Fig. 5C). This       genetic analysis. We ask1 mutants syn- <br /> provides support notion microtubules      thetically lethal mutations DAM1 or <br /> are required binding Ask1 centromere.        DUO1. Double mutants ask1-3 dam1-1 ask1-3 <br />    To gather insight nature Ask1&#226;&#8364;&#8482;s centro-         duo1-2 survive presence wild-type <br /> meric association, examined association        ASK1 (Fig. 6C). Finally, observed Ask1&#226;&#8364;&#8220;GFP <br /> the cell cycle. Using synchronous cultures,           failed localize spindle kinetochore dam1 <br /> that Ask1 remained centromere associated through-              duo1 mutants immunofluorescence microscopy <br /> out cycle showed twofold reduction cells          (data shown). <br /> entered S phase (Fig. 5E). We examined asso-          From mass spectrometry, identified Spc34, <br /> ciation cells arrested different cell cycle stages   protein copurified biochemically spindle- <br /> the effects nocodazole arrests. We       pole bodies (Wigge et al. 1998). Like Ask1, Dam1, and <br /> that G1 G2-arrested cells, Ask1 bound centro-           Duo1, binds kinetochores spindles, <br /> meric DNA. However, treatment cells              kinetochore binding Ndc10 dependent (Wigge <br /> nocodazloe resulted significantly interference,        et al. 1998; He et al. 2001). That Spc34 component of <br /> 59% 41% residual binding, respectively (Fig. 5F,G)         DASH confirmed immunoprecipitation experi- <br /> than observed cells traversing S phase        ments carried strains expressing Spc34-HA (Fig. <br /> presence nodcodazole (Fig. 5F,G). These differences         6D). <br /> are different amounts Ask1 protein immu-            Spc34 copurifies colocalizes Spc19 (He et al. <br /> noprecipitated different conditions (Fig. 5F). These     2001) interacts Spc19 two-hybrid assay <br /> data suggest nocodazole effective G1    (Yeast Protein Database). To test Spc19 is <br /> or G2-arrested cells, traversing S phase        present DASH, precipitated Ask1&#226;&#8364;&#8220;TAP from <br /> presence nocodazole causes greater sensitivity. Be-       strains expressing Spc19-HA Spc19 pre- <br /> cause nocodazole appears equally effective dis-       sent immunoprecipitates (Fig. 6E). Precipitating <br /> rupting microtubule staining condi-        Spc19-HA led precipitation Duo1, con- <br /> tions (Fig. 5F), explanation unlikely. A         firming Spc19 exists DASH Ask1 Duo1 <br /> plausible explanation kinetochore structures       (Fig. 6E). <br /> disassembled replication forks replicate centromeric           Ask1 interacts novel ORF (YKR083C) two- <br /> DNA. After replication, kinetochores reas-             hybrid assay (Yeast Protein Database). We named the <br /> sembled reattach spindle load Ask1. Thus,        protein Hsk1 (Helper Ask1) tagged the <br /> S phase constitute highly nocodazole-sensitive         HA-epitope test DASH. Immu- <br /> period consistent slight reduction          noprecipitation experiments indicated Hsk1 a <br /> binding synchronous cultures passing S phase        component complex (Fig. 6F). <br /> (Fig. 5E).                                                        The integrity large complexes dependent <br />                                                                components. Therefore, exam- <br />                                                                ined complex formation mutants ASK1, <br /> The identification DASH complex <br />                                                                DAM1, DUO1. Immunoprecipitation experiments <br /> To understand pathways Ask1 functions,            carried ask1-2&#226;&#8364;&#8220;MYC, dam1-1 ASK1&#226;&#8364;&#8220;MYC, <br /> we carried large-scale affinity purification using       duo1-2 ASK1&#226;&#8364;&#8220;MYC strains. Although total <br /> TAP-tagged Ask1 (Rigaut et al. 1999). Approximately 10         proteins affected sig- <br /> bands specifically present sequential af-       nificantly mutants, Dam1 and <br /> finity purifications ASK1&#226;&#8364;&#8220;TAP strain         Duo1 immunoprecipitates greatly reduced <br /> absent strains lacking tag. By use mass spec-      mutant, Ask1 immuno- <br /> trometry, bands identified Ask1,      precipitated remained constant (Fig. 6G). This indi- <br /> Dam1, Duo1, Spc34, ribosomal proteins L17A             cates Ask1, Dam1, Duo1 required the <br /> S17A (Fig. 6A), additional proteins, Spc19         integrity DASH complex function mi- <br /> Hsk1 identified described below. We named          tosis. <br />  <br />  <br /> 190         GENES &amp; DEVELOPMENT <br />                                                                                    Spindle-dependent loading DASH kinetochore <br />  <br />  <br />                                                                                      Figure 6. The identification DASH <br />                                                                                      complex. (A) TAP-purified protein extracts <br />                                                                                      2L yeast culture either <br />                                                                                      ASK1&#226;&#8364;&#8220;TAP (Y1173) untagged strains <br />                                                                                      (Y300) resolved 4%&#226;&#8364;&#8220;20% SDS&#226;&#8364;&#8220; <br />                                                                                      polyacrylamide gels stained Coo- <br />                                                                                      massie blue. Protein bands cut out <br />                                                                                      identified mass spectrometry de- <br />                                                                                      scribed previously (Wang et al. 2000). <br />                                                                                      Question marks indicate proteins whose <br />                                                                                      identities suspected deter- <br />                                                                                      mined. (B) Coimmunoprecipitation of <br />                                                                                      Dam1 Duo1 Ask1&#226;&#8364;&#8220;Myc. (Top) <br />                                                                                      Ask1&#226;&#8364;&#8220;Myc immunoprecipitated with <br />                                                                                      anti-Myc antibodies. Proteins total <br />                                                                                      protein extracts (lysate) immu- <br />                                                                                      noprecipitated fractions (IP) analyzed <br />                                                                                      immunoblotting, using antibodies <br />                                                                                      Dam1 Duo1. (Bottom) The <br />                                                                                      extracts immunopre- <br />                                                                                      cipitated anti-Duo1 antibodies and <br />                                                                                      Western blots probed anti-Myc <br />                                                                                      antibodies. (C) ask1-3 synthetically le- <br />                                                                                      thal dam1-1 duo1-2. Both ask1-3 <br />                                                                                      dam1-1 (Y1171) ask1-3 duo1-2 <br />                                                                                      (Y1172) containing pJBN81 (ASK1&#226;&#8364;&#8220;URA3) <br />                                                                                      struck SC-Ura plates SC <br />                                                                                      plates containing 5-FOA. (D) Coimmuno- <br />                                                                                      precipitation Duo1 Spc34. Immu- <br />                                                                                      noprecipitation carried anti- <br />                                                                                      HA antibodies Western blots were <br />                                                                                      probed anti-Duo1 antibodies. (E) <br />                                                                                      Spc19 component DASH com- <br />                                                                                      plex. (Left) Extracts SPC19-HA and <br />                                                                                      SPC19-HA ASK&#226;&#8364;&#8220;-TAP strains immu- <br />                                                                                      noprecipitated anti-TAP antibodies <br />                                                                                      (PAP: peroxidase-anti-peroxidase) and <br />                                                                                      Western blots probed anti-TAP <br />                                                                                      anti-HA antibodies. (Right) Extracts <br />                                                                                      SPC19-HA nontagged strains <br />                                                                                      immunoprecipitated anti-HA <br />                                                                                      antibodies Western blots probed <br />                                                                                      anti-HA anti-Duo1 antibodies as <br /> indicated. (F) Hsk1 component DASH complex. (Left) Extracts HSK1-HA HSK1-HA ASK1-TAP strains were <br /> immunoprecipitated anti-TAP antibodies (PAP) Western blots probed anti-TAP anti-HA antibodies. (Right) <br /> Extracts HSK1-HA nontagged strains immunoprecipitated anti-HA antibodies Western blots probed with <br /> anti-HA anti-Duo1 antibodies. (G) ASK1, DAM1, DUO1 required formation DASH complex. Immunopre- <br /> cipitations carried extracts ask1-2&#226;&#8364;&#8220;MYC (Y1174), dam1-1 ASK1&#226;&#8364;&#8220;MYC (Y1176), duo1-2 ASK1&#226;&#8364;&#8220;MYC (Y1175) strains <br /> and probed Dam1, Duo1, Ask1&#226;&#8364;&#8220;Myc. <br />  <br />  <br />  <br /> Ipl1 controls Dam1 phosphorylation                                 ated SDS-PAGE silver stained visualize pro- <br />                                                                    teins (Fig. 7A). We observed difference mobility of <br /> Purification DASH complex revealed number              band ipl1 mutants. That band corresponded the <br /> protein bands, appeared het-            Dam1 protein identified mass spectrometry, suggest- <br /> erogeneous migration <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> characteristic phospho-            ing Dam1 potential target Ipl1. <br /> proteins. This prompted examine mobility                 To determine Dam1 phosphoprotein, <br /> these complex members strains mutant protein                Dam1 tagged Myc epitope prepared <br /> kinases localize kinetochores. Ipl1 protein           synchronized cells. The protein treated with <br /> kinase important chromosome segregation (Chan et               phosphatase (AP) phosphatase inhibitor <br /> al. 1993; Biggins et al. 1999) localizes kineto-            (PI). Dam1 migrates doublet phosphatase treat- <br /> chores (He et al. 2001). We examined effect          ment significantly reduced mobility faster mi- <br /> of Ipl1 DASH. TAP-purified DASH complexes                  grating species, indicating Dam1 phosphopro- <br /> either wild-type ipl1-321 mutant cells fraction-           tein (Fig. 7B). To provide evidence Ipl1 is <br />  <br />  <br />                                                                                                 GENES &amp; DEVELOPMENT             191 <br />  Li et al. <br />  <br />  <br />  <br />  <br /> Figure 7. Ipl1 controls Dam1 phos- <br /> phorylation. (A) The mobility DASH <br /> complex proteins altered ipl1 mu- <br /> tants. TAP-purified protein extracts from <br /> 2 L yeast culture ASK1&#226;&#8364;&#8220; <br /> TAP (Y1173) ASK1&#226;&#8364;&#8220;TAP ipl1-321 mu- <br /> tants (Y1182) resolved 4%&#226;&#8364;&#8220;20% <br /> SDS&#226;&#8364;&#8220;polyacrylamide gel silver <br /> stained. An expanded section shown to <br /> illustrate absence certain bands in <br /> an ilp1 mutant strain. (B) Dam1 phos- <br /> phoprotein. Dam1&#226;&#8364;&#8220;Myc prepared <br /> from extracts Y1183 treated with <br /> alkaline phosphatase presence or <br /> absence phosphatase inhibitors. Pro- <br /> teins immunoblotted and <br /> probed anti-Myc antibodies. (C) Ipl1 <br /> controls phosphorylation Dam1 in <br /> vivo. Protein extracts prepared from <br /> Y1183 (DAM1&#226;&#8364;&#8220;MYC) Y1184 (DAM1&#226;&#8364;&#8220; <br /> MYC ipl1-321) cells synchronized G1 <br /> with &#226;?&#163;-factor released cell <br /> cycle indicated time, were <br /> then immunoblotted probed with <br /> anti-MYC antibodies. (D) Ipl1 directly <br /> phosphorylates Dam1. Kinase assays <br /> were performed using GST&#226;&#8364;&#8220;Ipl1 kinase <br /> (lanes 1,2) GST control (lane <br /> 3). The substrate tested DASH <br /> complex purified ipl1-321 ASK1&#226;&#8364;&#8220; <br /> TAP (Y1182). The autophosphorylation of <br /> GST&#226;&#8364;&#8220;Ipl1 phosphorylation of <br /> Dam1 indicated left. (E) Hypotheti- <br /> cal model depicting microtubule-de- <br /> pendent loading DASH complex to <br /> the kinetochore. <br />  <br />  <br />  <br />  <br /> controlling Dam1 phosphorylation vivo, mobility        tional members DASH complex essential <br /> of Dam1 wild-type ipl1 mutants examined         genes required proper execution anaphase and <br /> synchronized cells. The presence ipl1 mutation         maintenance spindle-kinetochore integrity. The simi- <br /> abolished slower migrating form Dam1 (Fig. 7C).        larity phenotype components DASH sug- <br /> To test Ipl1 phosphorylated Dam1 directly,            gests entire complex required common <br /> GST GST&#226;&#8364;&#8220;Ipl1 purified bacteria (Biggins         function. <br /> et al. 1999) tested kinase activity TAP-          We Ask1 centromere-binding protein by <br /> purified DASH complexes. GST&#226;&#8364;&#8220;Ipl1 specifically phos-          lines evidence. First, ask1-3 mutants exhibit <br /> phorylated band corresponding position Dam1       severe chromosome loss, phenotype associated with <br /> on gel (Fig. 7D), indicating Dam1 substrate     kinetochore defects. Second, genetic interaction was <br /> of Ipl1 Ipl1 control Dam1 phosphorylation      ASK1 NDC10, gene encoding the <br /> vivo likely direct.                                  essential component CBF3 kinetochore protein com- <br />                                                               plex. Third, immunofluorescence staining Ask1 <br />                                                               shows that, addition associating mitotic <br />                                                               spindle, localizes vicinity SPBs. This <br /> Discussion <br />                                                               consistent immunolocalization many <br /> In study, identification Ask1     centromere-binding proteins Ndc10 and <br /> the DASH complex resides. Genetic analysis        Ctf19 (Hyland et al. 1999). This localization abolished <br /> of ask1 mutants revealed plays essential role   ndc10-1 mutants. Finally, formaldehyde cross-linking <br /> chromosome segregation. ASK1 addi-         followed chromatin immunoprecipitation reveals <br />  <br />  <br /> 192         GENES &amp; DEVELOPMENT <br />                                                                                Spindle-dependent loading DASH kinetochore <br />  <br />  <br />  <br /> that Ask1 localizes CEN DNA vivo Ndc10-              transferring information kinetochore order to <br /> dependent manner.                                               prepare future activities. How kinetochore <br />   Ask1, presumably entire DASH complex,                 benefit DASH loading? One possibility that <br /> plays critical role chromosome segregation             DASH stabilizes microtubule&#226;&#8364;&#8220;kinetochore attach- <br /> maintenance spindle-kinetochore integrity. ask1-3            ment maturation kinetochore, possibil- <br /> mutants lose chromosomes high rate semi-           ity consistent role preventing nondisjunction <br /> permissive temperatures restrictive tem-           maintaining short spindle cell cycle arrest. <br /> perature, undergo massive nondisjunction. Furthermore,          Alternatively, DASH act turn spindle <br /> in ask1-3 mutant cells, observed abnormal            checkpoint, causing dissociation Mad2 asso- <br /> spindle checkpoint-arrested cells spindle       ciated proteins. This consistent ob- <br /> staining SPBs disappeared segregation          servation spindle checkpoint maintained in <br /> the DNA masses. This defect similar observed         active state ask1 mutants. Another possibility is <br /> in dam1-1 mutants (Jones et al. 1999; Cheeseman et al.          DASH help orient sister kinetochores a <br /> 2001), electron micrographs severe de-          back-to-back orientation facilitate bipolar attachment <br /> fect spindle integrity.                                      kinetochore captured microtubule. <br />    The phenotypes ask1 mutants best explained            The DASH complex consists copurifying <br /> through defect kinetochore&#226;&#8364;&#8220;spindle interaction.            proteins. We identified Ask1, Dam1, Duo1, Spc34, <br /> ask1-3 mutants attempted segregate chromosomes            Spc19, Hsk1, RPL17A, RPS17A these. We <br /> daughter cells despite activation spindle check-         confirmed these, exception of <br /> point, suggesting cohesed sisters pulled randomly        ribosomal proteins, exist complex least <br /> one spindle poles spindles functional.       components DASH, suggesting large <br /> Even absence spindle checkpoint          complex. From sizing columns, detected Ask1 <br /> normal elongated spindle observed, nondisjunction oc-        complex &gt;1 Mda, large accom- <br /> curs high rate. A possible explanation that,   modate proteins (data shown). Beside the <br /> on average, spindle&#226;&#8364;&#8220;kinetochore interac-         phenotypic similarity mutants ASK1, <br /> tions pair sister chromatids defective. Whether      DUO1, DAM1, proteins localize kineto- <br /> this random, stochastic event particular       chore Ndc10-dependent manner (He et al. 2001). In <br /> kinetochore defective misoriented unknown. An          addition, Spc34 binds spindles kineto- <br /> alternative explanation chromosomes remain              chores, Ndc10-dependent manner (Wigge <br /> monoattached given spindle-pole body fail set       et al. 1998; He et al. 2001). Another, unidentified, <br /> up bipolar attachment fail release. This      component likely known protein, YDR016C <br /> phenotype similar ipl1 mutant (T.       (Dad1), interacts Duo1 two-hybrid assay <br /> Tanaka et al., prep.), shows high rates        (Yeast Protein Database) shown recently to <br /> chromosome non-disjunction (Chan Botstein 1993).            exist complex Dam1 Duo1 (Enquist-New- <br />    Ask1 unique kinetochore-binding proteins            man et al. 2001). <br /> because centromeric association dependent              The connection Ipl1 DASH complex <br /> an intact spindle. Ask1&#226;&#8364;&#8482;s kinetochore localization           ties pathways important proper control <br /> most sensitive spindle perturbation cells           chromosome segregation. Ipl1 implicated re- <br /> traversing S phase, period new kinetochores        cently controlling sensing tension chromo- <br /> are assembled re-establish spindle attach-       somes. In absence DNA replication in <br /> ment. Because Ask1 associated spindle,          cdc6 mutants, Pds1 degradation delayed spindle <br /> simplest interpretation Ask1 transferred di-         checkpoint-dependent manner (Stern Murray 2001). <br /> rectly spindle kinetochore kineto-         Unlike aspects spindle checkpoint function <br /> chores replicated microtubules captured (Fig.       cell cycle arrest response nocodazole treat- <br /> 7E). Once Ask1 loaded kinetochore,          ment, aspect spindle-assembly checkpoint is <br /> partially requires presence microtubules main-        specifically dependent Ipl1 (Biggins Murray 2001). <br /> tain kinetochore localization, indicating        The fact Ipl1 controls Dam1 phosphorylation in <br /> simply microtubule-binding protein localized          vivo, likely direct phosphorylation, suggests <br /> near kinetochore microtubule capture,          Ipl1 controlling DASH function capacity. <br /> actually loaded kinetochore microtubule-          Whether Ipl1&#226;&#8364;&#8482;s role controlling the <br /> dependent manner. Thus, DASH requires microtubules              spindle checkpoint distinct role control chro- <br /> for loading maintenance kinetochore bind-        mosome segregation remains determined. How- <br /> ing. It noted, however, results       ever, ipl1 ask1 mutants severe nondis- <br /> distinguish loading Ask1 protein directly        junction phenotypes, likely working <br /> from spindle kinetochore versus induced bind-        ensure proper bipolar kinetochore&#226;&#8364;&#8220;spindle in- <br /> ing Ask1 kinetochore spindle interaction.        teractions. <br />    The dynamic microtubule-dependent association                All identified members DASH essential <br /> Ask1 kinetochore indicates spindle bind-          proteins, indicating likely perform functions <br /> ing additional consequences kinetochore             essential mitosis. The discovery DASH estab- <br /> other establishing connection. DASH               lished existence new pathway controlling inter- <br />  <br />  <br />                                                                                             GENES &amp; DEVELOPMENT             193 <br />  Li et al. <br />  <br />  <br />  <br /> actions spindle kinetochore. Fur-               Table 1.   Strains used study <br /> thermore, analysis Ask1 DASH complex <br />                                                                     Y300        MATa trp-l-l ura3-1 his3-11, 3 leu2-3, 112 ade2-l <br /> has established new step kinetochore maturation                              can1-100 <br /> and revealed spindle transfers information             Y604        Y300 mecl-21 <br /> to newly captured kinetochore alter function          Y928        Y300 LEU2, askl-l <br /> an important manner. The finding Ask1                Y974        Y300 ura3::tetO-URA3, leu2-3, <br /> DASH component homologs exist organisms                                  112::GFP::TetR-LEU2 <br /> indicates protein complex conserved               Y998        Y300 PDSI-18MYC-LEU2 <br /> throughout eukaryotic evolution likely play            Y1101       Y300 &#226;&#338;&#172;mad2::URA3 <br /> important role regulation kinetochores higher              Y1102       Y300 &#226;&#338;&#172;ask1::HIS3 + <br /> eukaryotes.                                                                       JBN81(pRS426-ASK1) <br />                                                                     Y1103       Y300 ask1-3 <br />                                                                     Y1104       Y300 ask1-2 <br />                                                                     Y1106       Y300 ask1-2 &#226;&#338;&#172;mad2::URA3 <br /> Materials methods                                               Y1107       Y300 ask1-3 &#226;&#338;&#172;mad2::URA3 <br />                                                                     Y1108       Y300 PDS1-18MYC-LEU2, ask1-3 <br /> Yeast strains plasmid construction <br />                                                                     Y1109       Y300 ura3::tetO-URA3, <br /> All strains used experiments isogenic                       leu2-3,112::GFP::TetR-LEU2, ask1-3 <br /> W303-derived Y300 strain listed Table 1. Gene dis-       Y1110       Y300 ura3::tetO-URA3, <br /> ruptions introduced Y300 background use                       leu2-3,112::GFP::TetR-LEU2, ask1-3, <br /> standard methods. Additional strains subsequently con-                       &#226;&#338;&#172;mad2::URA3 <br /> structed use standard genetic crosses. The ASK1 deletion      Y1111       Y300 ASK1-GFP-TRP <br /> strain generated use 1-kb &#226;&#338;&#172;ask1&#226;&#172;&#8249;HIS3 PCR frag-          Y1112       Y300 ASK1-GFP-TRP, ndc10-1 <br /> ment. This fragment replaces amino acids 0&#226;&#8364;&#8220;177 293           Y1113       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3 <br /> amino acid ASK1 ORF HIS3. The deletion strain cov-         Y1114       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3, <br /> ered pASK1-CEN/ARS/URA3 (pJBN81) maintain viabil-                         cdc13-1 <br /> ity.                                                                Y1115       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3, <br />   The GFP-tagged Ask1 generated follows: A 490-bp frag-                    ndc10-1 <br /> ment Ask1 amplified PCR using following prim-         Y1116       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3, <br /> ers: Ask1-23, 5&#226;&#172;&#732;-CCCGCGGTGCCTTCAAGGGAGCAAACA                                      cdc13-1, &#226;&#338;&#172;mad2::URA3 <br /> GACC-3&#226;&#172;&#732; Ask1-27, 5&#226;&#172;&#732;-CTAATCGATTTCTATTCGTAGA                      Y1117       Y300 CFIII, URA3, SUP11 <br /> AAAATGAATG-3&#226;&#172;&#732;. As result, ClaI site introduced           Y1118       Y300 CFIII, URA3, SUP11, ask1-3 <br /> the termination codon ASK1 gene. The resulting SacII&#226;&#8364;&#8220;        Y1119       Y300 ndc10-1 <br /> ClaI fragment cloned pRS404 ClaI&#226;&#8364;&#8220;          Y1120       Y300 ask1-3 ndc10-1 <br /> ClaI cassette containing GFP provided Dr. John Kilmartin         Y1171       Y300 ask1-3 dam1-1 pJBN81 <br /> (MRC, Cambridge, UK). The resulting plasmid introduced          Y1172       Y300 ask1-3 duo1-2 pJBN81 <br /> into haploid wild-type strain Y300 endogenous ASK1         Y1173       Y300 ASK1-TAP-TRP <br /> locus linearization SacI.                                Y1174       Y300 ask1-2-9Myc <br />                                                                     Y1175       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3, <br />                                                                                   duol-2 <br /> Isolation temperature-sensitive mutants ASK1                  Y1176       Y300 &#226;&#338;&#172;ask1::LEU2, ASK1-9MYC-HIS3, <br /> Temperature-sensitive alleles ASK1 generated using                    dam1-1 <br /> PCR random mutagenesis method described previously (Wang            Y1177       Y300 SPC34-HA <br /> and Elledge 1999). pJBN79 (pASK1-CEN/ARS/TRP1)                  Y1178       Y300 SPC19-HA <br /> used template. Primers used PCR follows:        Y1179       Y300 SPC19-HA, ASK1-TAP-TRP <br /> Ask1-5 containing PstI site, 5&#226;&#172;&#732;-GGTTGATCAACGTATGGA                  Y1180       Y300 HSK1-HA <br /> AAGAGCAACATAAGGGATTGTTGACACAAAAAATGG-                               Y1181       Y300 HSK1-HA, ASK1-TAP-TRP <br /> 3&#226;&#172;&#732;, Ask1-6 containing KpnI site, 5&#226;&#172;&#732;-GAGCCGGTACCAT               Y1182       Y300 ipl1-321, ASK1-TAP-TRP <br /> GCTGGAACTAGACCCGTACTGCTGCTGCAAAGTAGCG                               Y1183       Y300 DAM1-MYC-TRP1 <br /> AATC-3&#226;&#172;&#732;. Mutagenic PCR carried using 20 ng plas-         Y1184       Y300 ipl1-321, DAM1-MYC-TRP1 <br /> mid DNA, 1 &#194;&#181;M primer, 5 mM MgCl2, 1 mM <br /> MnCl2. The PCR products purified, digested KpnI <br /> and PstI, cloned pYL61 (pJBN79 KpnI site on <br />                                                                     Cytological techniques <br /> vector filled in). Approximately 10,000 Escherichia coli trans- <br /> formants obtained. This library used transform          FACS analysis performed described previously (Desany <br /> &#226;&#338;&#172;ask1&#226;&#172;&#8249;HIS3 covered pJBN81 (Y1102) Trp+ transfor-             et al. 1998). Immunofluorescence microscopy using anti- <br /> mants selected room temperature. They rep-        tubulin anti-GFP antibodies performed formalde- <br /> lica plated plates containing 0.1% 5-fluoroorotic acid (5-FOA)   hyde fixation described (Sanchez et al. 1996). Cells fixed <br /> at 24&#194;&#176;C 37&#194;&#176;C identify temperature-sensitive mutants.         5% formaldehyde 0.5 h stained anti-GFP <br /> From 10 temperature-sensitive clones, plasmid DNA re-          polyclonal (1:2000 dilution, provided Dr. Silver, Harvard <br /> covered transformed confirm phenotype. Six          Medical School Dana Farber Cancer Institute, Boston, MA) <br /> temperature-sensitive alleles integrated ASK1 lo-     anti-tubulin Yol1/34 (1:50 dilution, Desany et al. 1998) an- <br /> cus using two-step gene replacement, temperature-sen-       tibodies methanol/acetone treatment. Other immunoflu- <br /> sitive phenotypes allele complemented       orescence microscopy using anti-tubulin antibodies as <br /> wild-type ASK1 gene CEN plasmid.                               described previously (Desany et al. 1998). DAPI used for <br />  <br />  <br /> 194         GENES &amp; DEVELOPMENT <br />                                                                                       Spindle-dependent loading DASH kinetochore <br />  <br />  <br /> staining DNA fixation. Observation TetR&#226;&#8364;&#8220;GFP          Health Grant. S. J. E. investigator Howard Hughes <br /> performed directly Zeiss Axioskop cells fixed    Medical Institute Welch professor biochemistry. <br /> 5 min.                                                                 The publication costs article defrayed by <br />                                                                      payment page charges. This article hereby <br />                                                                      marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 USC section <br /> Protein purification <br />                                                                      1734 solely indicate fact. <br /> Protein extracts 2-L yeast culture OD 1.2 pre- <br /> pared using RIPA buffer (150 mM NaCl, 50 mM Tris-Cl, 5 mM <br /> EDTA, 0.5% Triton X-100), subsequently, TAP purification         References <br /> was carried described (Rigaut et al. 1999). After centrifu- <br /> gation obtain supernatant, Tris-Cl (pH 8.0) added      Alcasabas, A.A., Osborn, A.J., Bachant, J., Hu, F., Werler, P.J.H., <br /> a final concentration 10 mM NP40 added final            Bousset, K., Kanji Furuya, K., Diffley, J.F.X., Carr, A. and <br /> concentration 0.1%. The extracts subjected two-         Elledge, S.J. 2001. Mrc1 transduces DNA replication stress <br /> step affinity purification. After 2-h incubation IgG Seph-       signals activate Rad53. Nat. Cell. Biol.. 3: 958&#226;&#8364;&#8220;965. <br /> arose (400 &#194;&#181;L) 4&#194;&#176;C, beads washed times        Amon, A. 1999. The spindle checkpoint. Curr. Opin. Genet. <br /> IPP150 (10 mM Tris-Cl pH 8.0, 150 mM NaCl, 0.1%                  Dev. 9: 69&#226;&#8364;&#8220;75. <br /> NP-40), TEV buffer (IPP150 containing 0.5 mM               Aparicio, O.M., Weinstein, D.M., Bell, S.P. 1997. Compo- <br /> EDTA 1 mM DTT), incubated 1-mL TEV buffer                    nents dynamics DNA replication complexes S. cer- <br /> containing 100 U TEV protease (GIBCO BRL) room tem-               evisiae: Redistribution MCM proteins Cdc45p during <br /> perature 2 h column. The eluate diluted 4.8 mL          S phase. Cell 91: 59&#226;&#8364;&#8220;69. <br /> with calmodulin-binding buffer (IPP150 containing 10 mM              Biggins, S. Murray, A.W. 2001. The budding yeast protein <br /> &#226;?&#164;-mercaptoethanol, 1 mM Mg-acetate, 1 mM imidazole, 2 mM                kinase Ipl1/Aurora allows absence tension activate <br /> CaCl2), 3.6 &#194;&#181;L 1M CaCl2 200 &#194;&#181;L calmodulin beads             spindle checkpoint. Genes &amp; Dev. 15: 3118&#226;&#8364;&#8220;3129. <br /> added incubated 4&#194;&#176;C 1 hr. After washing, bound pro-       Biggins, S., Severin, F.F., Bhalla, N., Sassoon, I., Hyman, A.A., <br /> teins eluted (five times, 200 &#194;&#181;L each) calmodulin             Murray, A.W. 1999. The conserved protein kinase Ipl1 <br /> elution buffer (IPP150 containing 10 mM &#226;?&#164;-mercaptoethanol, 1            regulates microtubule binding kinetochores budding <br /> mM Mg-acetate, 1 mM imidazole, 2 mM EGTA). Purified                 yeast. Genes &amp; Dev. 13: 532&#226;&#8364;&#8220;544. <br /> proteins precipitated TCA analyzed 4%&#226;&#8364;&#8220;20%           Cai, M. Davis, R.W. 1990. Yeast centromere binding protein <br /> SDS-PAGE, followed Coomassie blue staining silver                CBF1, helix-loop-helix protein family, required for <br /> staining.                                                               chromosome stability methionine prototrophy. Cell <br />                                                                         61: 437&#226;&#8364;&#8220;446. <br />                                                                      Chan, C.S. Botstein, D. 1993. Isolation characterization <br /> Kinase assay                                                            chromosome-gain increase-in- ploidy mutants in <br /> TAP-purified DASH complex ipl1-321 ASK1&#226;&#8364;&#8220;TAP                     yeast. Genetics 135: 677&#226;&#8364;&#8220;691. <br /> strain eluted 200 &#194;&#181;L kinase buffer (50 mM Tris-Cl pH       Cheeseman, I.M., Enquist-Newman, M., Muller-Reichert, T., <br /> 8.0, 100 mM NaCl, 2 mM EGTA, 1 mM imidazole, 25 mM                      Drubin, D.G., Barnes, G. 2001. Mitotic spindle integrity <br /> &#226;?&#164;-glycerophosphate, 1 mM DTT, 10 mM ATP, 5 mM MgCl2, 2.5                kinetochore gunction linked Duo1p/Dam1p com- <br /> &#194;&#181;Ci [&#226;?&#165;32-P]ATP) 1 &#194;&#181;g GST GST&#226;&#8364;&#8220;Ipl1 beads added            plex. J. Cell Biol. 152: 197&#226;&#8364;&#8220;212. <br /> to reaction 20 min 30&#194;&#176;C described (Biggins et al.      Chen, R.H., Waters, J.C., Salmon, E.D., Murray, A.W. 1996. <br /> 1999).                                                                  Association spindle assembly checkpoint component <br />                                                                         XMAD2 unattached kinetochores. Science 274: 242&#226;&#8364;&#8220; <br />                                                                         246. <br /> Chromatin immunoprecipitations                                       Ciosk, R., Zachariae, W., Michaelis, C., Shevchenko, A., Mann, <br /> Chromatin immunoprecipitation assays conducted essen-              M., Nasmyth, K. 1998. An ESP1/PDS1 complex regu- <br /> tially described previously (Aparicio et al. 1997; Zou           lates loss sister chromatid cohesion metaphase to <br /> Stillman 2000). Ask1&#226;&#8364;&#8220;Myc immunoprecipitated                  anaphase transition yeast. Cell 93: 1067&#226;&#8364;&#8220;1076. <br /> monoclonal antibody 9E10. Primers CEN3, CEN16,           Cohen-Fix, O., Peters, J.M., Kirschner, M.W., Koshland, D. <br /> AT-rich region near PGK1 described (Meluh           1996. Anaphase initiation Saccharomyces cerevisiae is <br /> Koshland 1997).                                                         controlled APC-dependent degradation ana- <br />    The nocodazole treatment cells carried            phase inhibitor Pds1p. Genes &amp; Dev. 10: 3081&#226;&#8364;&#8220;3093. <br /> follows: cells arrested &#226;?&#163;-factor 3 h, 32&#194;&#176;C incu-    Connelly, C. Hieter, P. 1996. Budding yeast SKP1 encodes <br /> bation 2 h spun resuspended YPD            evolutionarily conserved kinetochore protein required for <br /> containing 15 &#194;&#181;g/mL nocodazole 1% DMSO. YPD contain-                cell cycle progression. Cell 86: 275&#226;&#8364;&#8220;285. <br /> ing 1% DMSO used control. Seventy minutes later,          Desany, B.A., Alcasabas, A.A., Bachant, J.B., Elledge, S.J. <br /> cells spun ChIP analysis.                                 1998. Recovery DNA replicational stress essen- <br />                                                                         tial function S-phase checkpoint pathway. Genes &amp; <br />                                                                         Dev. 12: 2956&#226;&#8364;&#8220;2970. <br />                                                                      Doheny, K.F., Sorger, P.K., Hyman, A.A., Tugendreich, S., Spen- <br /> Acknowledgments <br />                                                                         cer, F., Hieter, P. 1993. Identification essential <br /> We thank K. Nasmyth, P. Hieter, J. Kilmartin, S. Biggins        components S. cerevisiae kinetochore. Cell 73: 761&#226;&#8364;&#8220; <br /> strains; P. Silver, D.G. Drubin, G. Barnes antibodies;      774. <br /> K. Nasmyth T. Tanaka sharing unpublished results. We         Elledge, S.J. Davis, R.W. 1990. Two genes differentially <br /> also thank X. He, M. Winey, A. Hyman, members                regulated cell cycle DNA- damaging agents <br /> Elledge laboratory comments helpful discussions. We             encode alternative regulatory subunits ribonucleotide re- <br /> thank L. Zou D. Zhang sharing technical expertise.        ductase. Genes &amp; Dev. 4: 740&#226;&#8364;&#8220;751. <br /> This work supported National Institute           Enquist-Newman, M., Cheeseman, I.M., Van Goor, D., Drubin, <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT             195 <br />  Li et al. <br />  <br />  <br />     D.G., Meluh, P.B., Barnes, G. 2001. Dad1p, com-            dependent spindle checkpoint. Science 279: 1045&#226;&#8364;&#8220;1047. <br />     ponent duo1p/dam1p complex involved kineto-            Lechner, J. 1994. A zinc finger protein, essential chromo- <br />     chore function mitotic spindle integrity. Mol. Biol. Cell        segregation, constitutes putative DNA binding sub- <br />     12: 2601&#226;&#8364;&#8220;2613.                                                       <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Saccharomyces cerevisiae kinetochore complex, <br /> Farr, K.A. Hoyt, M.A. 1998. Bub1p kinase activates               Cbf3. EMBO J. 13: 5203&#226;&#8364;&#8220;5211. <br />     Saccharomyces cerevisiae spindle assembly checkpoint.            Lechner, J. Ortiz, J. 1996. The Saccharomyces cerevisiae <br />     Mol. Cell. Biol. 18: 2738&#226;&#8364;&#8220;2747.                                      kinetochore. FEBS Lett. 389: 70&#226;&#8364;&#8220;74. <br /> Funabiki, H., Yamano, H., Kumada, K., Nagao, K., Hunt, T.,       Li, R. Murray, A.W. 1991. Feedback control mitosis in <br />     Yanagida, M. 1996 Cut2 proteolysis required sister-chro-         budding yeast. Cell 66: 519&#226;&#8364;&#8220;531. <br />     matid seperation fission yeast. Nature 381: 438&#226;&#8364;&#8220;441.          Meluh, P.B. Koshland, D. 1995. Evidence MIF2 <br /> Goh, P.Y. Kilmartin, J.V. 1993. NDC10: A gene involved            gene Saccharomyces cerevisiae encodes centromere pro- <br />     chromosome segregation Saccharomyces cerevisiae. J.               tein homology mammalian centromere protein <br />     Cell Biol. 121: 503&#226;&#8364;&#8220;512.                                             CENP-C. Mol. Biol. Cell 6: 793&#226;&#8364;&#8220;807. <br /> Gorbsky, G.J. Ricketts, W.A. 1993. Differential expression       &#226;&#8364;&#8221;&#226;&#8364;&#8221;&#226;&#8364;&#8221;. 1997. Budding yeast centromere composition as- <br />     phosphoepitope kinetochores moving chromo-            sembly revealed vivo cross-linking. Genes &amp; Dev. <br />     somes. J. Cell Biol. 122: 1311&#226;&#8364;&#8220;1321.                                 11: 3401&#226;&#8364;&#8220;3412. <br /> Goshima, G. Yanagida, M. 2000. Establishing biorientation        Meluh, P.B., Yang, P., Glowczewski, L., Koshland, D., and <br />     occurs precocious separation sister kinetochores,        Smith, M.M. 1998. Cse4p component core cen- <br />     arms, early spindle budding yeast. Cell        tromere Saccharomyces cerevisiae. Cell 94: 607&#226;&#8364;&#8220;613. <br />     100: 619&#226;&#8364;&#8220;633.                                                    Michaelis, C., Ciosk, R., Nasmyth, K. 1997. Cohesins: <br /> Hardwick, K.G. Murray, A.W. 1995. Mad1p, phosphopro-               Chromosomal proteins prevent premature separation of <br />     tein component spindle assembly checkpoint bud-            sister chromatids. Cell 91: 35&#226;&#8364;&#8220;45. <br />     ding yeast. J. Cell Biol. 131: 709&#226;&#8364;&#8220;720.                          Ortiz, J., Stemmann, O., Rank, S., Lechner, J. 1999. A pu- <br /> Hardwick, K.G., Weiss, E., Luca, F.C., Winey, M., Murray,            tative protein complex consisting Ctf19, Mcm21, and <br />     A.W. 1996. Activation budding yeast spindle assembly          Okp1 represents missing link budding yeast kineto- <br />     checkpoint mitotic spindle disruption. Science               chore. Genes &amp; Dev. 13: 1140&#226;&#8364; <br /> </body></html>