<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Analysis of 2&#226;&#180;&#8226;-phosphotransferase (Tpt1p) <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Saccharomyces cerevisiae: Evidence for a conserved <br /> two-step reaction mechanism <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Tpt1p is an essential protein responsible for the 2&#226;&#180;&#8226;-phosphotransferase step of tRNA splicing in Saccharomyces cerevisiae, in <br /> which the splice junction 2&#226;&#180;&#8226;-phosphate of ligated tRNA is transferred <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> NAD to form mature tRNA and ADP-ribose 1&#226;&#180;&#8211;-2&#226;&#180;&#8211; cyclic <br /> phosphate. We showed previously that Tpt1p is a member of a family of functional 2&#226;&#180;&#8226;-phosphotransferases found in eukaryotes, <br /> eubacteria, and archaea, that the Escherichia coli protein (KptA) is highly specific for 2&#226;&#180;&#8226;-phosphorylated RNAs despite the lack <br /> of obvious natural substrates, and that KptA acts on a trinucleotide substrate through an intermediate in which RNA is <br /> ADP-ribosylated at the 2&#226;&#180;&#8226;-phosphate. This mechanism is similar to a proposed mechanism of NAD-dependent histone deacety- <br /> lases. We present evidence here that this mechanism is conserved in S. cerevisiae, and we identify residues important for the <br /> second step of the reaction, during which the intermediate is resolved into products. We examined 21 Tpt1 protein variants <br /> mutated in conserved residues or blocks of residues and show that one of them, Tpt1 K69A/R71S protein, accumulates large <br /> amounts of intermediate with trinucleotide substrate due to a very slow second step. This intermediate can be trapped on beads <br /> when formed with biotin-NAD. We also show that Tpt1 K69A/R71S protein forms an intermediate with the natural ligated tRNA <br /> substrate and demonstrate that, as expected, this <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> is lethal in yeast. The high degree of conservation of these residues <br /> suggests that the entire Tpt1p family is involved in a similar two-step chemical reaction. <br /> Keywords: yeast; tRNA; splicing; processing; ADP-ribosylation <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                ecule, a cyclic phosphodiesterase activity that opens the 2&#226;&#172;&#732;- <br />                                                                             3&#226;&#172;&#732; cyclic phosphate of the 5&#226;&#172;&#732;-half-molecule to a 2&#226;&#172;&#732;-phos- <br /> 2&#226;&#172;&#732;-phosphotransferase (Tpt1) is an essential protein that <br />                                                                             phate, and a ligase activity that fuses the two half-molecules <br /> catalyzes the final step of tRNA splicing in the yeast Sac- <br />                                                                             through an adenylylated intermediate to form ligated tRNA <br /> charomyces cerevisiae (Spinelli et al. 1997). tRNA splicing in <br />                                                                             bearing a splice junction 2&#226;&#172;&#732;-phosphate (Greer et al. 1983; <br /> S. cerevisiae initiates with excision of the intron by the tRNA <br />                                                                             Phizicky et al. 1986, 1992; Belford et al. 1993; Westaway et <br /> splicing endonuclease, by cleavage of each intron-exon bor- <br />                                                                             al. 1993; Sawaya et al. 2003). In the final splicing step this <br /> der by the two corresponding catalytic subunits of the four <br />                                                                             2&#226;&#172;&#732;-phosphate is transferred from ligated tRNA to NAD by <br /> subunit endonuclease (Ho et al. 1990; Trotta et al. 1997; for <br />                                                                             Tpt1p, producing mature tRNA and ADP-ribose 1&#226;&#172;&#8482;-2&#226;&#172;&#8482; cy- <br /> review see Abelson et al. 1998), yielding 5&#226;&#172;&#732; half-molecules <br />                                                                             clic phosphate (Appr&gt;p) (Culver et al. 1993; Spinelli et al. <br /> ending with 2&#226;&#172;&#732;-3&#226;&#172;&#732; cyclic phosphate and 3&#226;&#172;&#732; half-molecules <br />                                                                             1997). Yeast Tpt1p is known to participate in tRNA splicing <br /> beginning with a 5&#226;&#172;&#732;-OH group (Peebles et al. 1983). The <br />                                                                             because conditional tpt1 mutants accumulate ligated tRNAs <br /> half-molecules are joined by the single subunit tRNA ligase <br />                                                                             with a 2&#226;&#172;&#732;-phosphate under nonpermissive conditions <br /> containing three separable domains: a kinase activity that <br />                                                                             (Spinelli et al. 1997). Amazingly, although both tRNA ligase <br /> uses GTP to phosphorylate the 5&#226;&#172;&#732;-OH of the 3&#226;&#172;&#732;-half mol- <br />                                                                             and Tpt1 protein are essential (Phizicky et al. 1992; Culver <br />                                                                             et al. 1997; Spinelli et al. 1997), both proteins are dispens- <br />    Reprint requests to: Eric Phizicky, Department of Biochemistry and       able in yeast if the cells express T4 polynucleotide kinase/ <br /> Biophysics, University of Rochester School of Medicine and Dentistry, 601   phosphatase and T4 RNA ligase, which together can remove <br /> Elmwood Avenue, Box 712, Rochester, New York 14642, USA; e-mail:            the 2&#226;&#172;&#732;-3&#226;&#172;&#732; cyclic phosphate and then ligate tRNA half-mol- <br /> eric_phizicky@urmc.rochester.edu; fax: (585) 271-2683. <br />    Article and publication are at http://www.rnajournal.org/cgi/doi/        ecules (Schwer et al. 2004). <br /> 10.1261/rna.7194605.                                                           Yeast Tpt1p may also participate in the unique splicing <br />  <br /> RNA (2005), 11:99&#226;&#8364;&#8220;106. Published by Cold Spring Harbor Laboratory Press. Copyright &#194;&#169; 2005 RNA Society.                                 99 <br />  Steiger et al. <br />  <br />  <br />  <br /> reaction of HAC1 mRNA that occurs when the unfolded <br /> protein response pathway is activated in yeast (Sidrauski et <br /> al. 1996; Sidrauski and Walter 1997; Gonzalez et al. 1999). <br /> This reaction includes endonucleolytic excision of the in- <br /> tron by Ire1p, and ligation of exons by tRNA ligase, leading <br /> to the formation of a splice junction 2&#226;&#172;&#732;-phosphate (Gonza- <br /> lez et al. 1999) that may require dephosphorylation by <br /> Tpt1p.                                                           FIGURE 1. A schematic of the second step of the 2&#226;&#172;&#732;-phosphotrans- <br />                                                                  ferase reaction. In the first step the 2&#226;&#172;&#732;-phosphate of the RNA substrates <br />    Although the Tpt1 protein family is widely conserved,         attacks NAD, generating ADP-ribosylated phosphorylated RNA, and <br /> with members in eukaryotes, bacteria, and archaea (Spinelli      releasing nicotinamide (Spinelli et al. 1999). In the second step, a <br /> et al. 1998), its role is poorly understood in organisms other   presumed transesterification as shown generates ADP-ribose 1&#226;&#172;&#8482;-2&#226;&#172;&#8482;- <br />                                                                  cyclic phosphate and dephosphorylated RNA product. <br /> than S. cerevisiae. Since Tpt1p orthologs from diverse or- <br /> ganisms including mouse, Escherichia coli, Arabidopsis <br /> thaliana, Schizosaccharomyces pombe, and humans can              intermediate, and when isolated, can be converted to both <br /> complement a tpt1 mutant strain (Spinelli et al. 1998), a        products (Appr&gt;p and pApApA) and substrates (NAD and <br /> role for 2&#226;&#172;&#732;-dephosphorylation of RNA is indicated. It is         pApAppA) in the presence of KptA protein, depending on <br /> possible that Tpt1p participates in tRNA splicing or the         reaction conditions. Tpt1p is presumed to act by the same <br /> unfolded response in vertebrates and plants, where there are     mechanism, because Tpt1 protein can act in both directions <br /> companion ligase activities that generate RNA products           on the reaction intermediate generated by KptA protein; <br /> with a 2&#226;&#172;&#732;-phosphate (Konarska et al. 1981; Gegenheimer et        however, to date there has been no direct observation of a <br /> al. 1983; Pick et al. 1986; Zillmann et al. 1991). However,      similar ADP-ribosylated intermediate produced by Tpt1p. <br /> there is strong evidence in vertebrates that tRNA splicing       There is a striking similarity between this mechanism for <br /> uses a ligase that does not generate a 2&#226;&#172;&#732;-phosphorylated         KptA protein (Spinelli et al. 1999) and one model recently <br /> tRNA (Nishikura and De Robertis 1981; Filipowicz and             suggested for formation of 2&#226;&#172;&#732;-O-acetyl-ADP-ribose during <br /> Shatkin 1983; Laski et al. 1983), and tRNA splicing in at        the NAD-dependent histone deacetylase reaction catalyzed <br /> least one archaeal species may use a similar ligase (Gomes       by Sir2 protein (Jackson and Denu 2002). <br /> and Gupta 1997; Zofallova et al. 2000). The function of             To further examine the mechanism of Tpt1p, we exam- <br /> bacterial Tpt1p family members is particularly puzzling be-      ined a series of 21 Tpt1 variants to define regions respon- <br /> cause phylogenetic analysis indicates that bacteria have re-     sible for the second step of the reaction. Such Tpt1p vari- <br /> tained this protein for 3 billion years, but there is no known   ants would accumulate the intermediate, if it follows a <br /> bacterial RNA ligase that generates a 2&#226;&#172;&#732;-phosphate, and the      mechanism similar to that observed with E. coli KptA. We <br /> tRNA splicing that does occur in bacteria is self-catalyzed      found that Tpt1p K69A/R71S accumulates significant <br /> (Xu et al. 1990; Reinhold-Hurek and Shub 1992; Ferat and         amounts of the reaction intermediate under all conditions <br /> Michel 1993). Moreover, the E. coli ortholog, KptA protein,      tested, including in the presence of tRNA substrate. The <br /> is very similar to Tpt1p in its kinetic parameters and in its    intermediate formed with trinucleotide substrate is stable <br /> relative substrate specificity, as measured with a variety of    enough to allow capture on beads, and can be converted to <br /> substrate RNAs with different phosphates and with 2&#226;&#172;&#732;-            products by the addition of wild-type Tpt1p. Consistent <br /> phosphates in different chemical environments (Steiger et        with a crucial role of K69/R71 residues of Tpt1p in vitro, <br /> al. 2001).                                                       overexpression of Tpt1 K69A/R71S protein cannot comple- <br />    The phosphotransferase reaction catalyzed by E. coli          ment a yeast tpt1-&#226;&#338;&#172; strain. These findings show directly that <br /> KptA consists of two distinct chemical steps involving for-      Tpt1p follows the same two-step mechanism as E. coli KptA <br /> mation of a covalent intermediate from the reactants and its     protein and implicate one or both of these two conserved <br /> subsequent resolution to products (Spinelli et al. 1999).        residues directly in the second step of the reaction. By <br /> First, the 2&#226;&#172;&#732;-phosphate on the RNA substrate is ADP-ribo-        extension, it seems likely that the conserved function of <br /> sylated, causing the release of nicotinamide and the forma-      the protein in other organisms involves a similar two-step <br /> tion of the reaction intermediate. This step is formally simi-   reaction. <br /> lar to the broad class of ADP-ribosylation reactions cata- <br /> lyzed by toxins such as Diphtheria toxin (Bell and Eisenberg <br />                                                                  RESULTS <br /> 1996). The second step (Fig. 1) is a presumed transesteri- <br /> fication reaction that resolves the intermediate into dephos- <br />                                                                  Tpt1 K69A/R71S protein produces significant <br /> phorylated RNA and Appr&gt;p. This mechanism is based on <br />                                                                  amounts of the reaction intermediate, <br /> the chemical and biochemical characterization of a reaction <br />                                                                  ADP-ribosylated RNA <br /> intermediate observed with KptA protein, when acting on <br /> the synthetic substrate pApAppA in the presence of NAD.          We made 21 different mutations of Tpt1 protein to find <br /> The intermediate forms with a time course expected of an         regions that might be responsible for the second step of the <br />  <br /> 100      RNA, Vol. 11, No. 1 <br />                                                                                             2&#226;&#180;&#8226;-phosphotransferase reaction of yeast Tpt1p <br />  <br />  <br />  <br /> 2&#226;&#172;&#732;-phosphotransferase reaction, transesterification of the <br /> proposed ADP-ribosylated RNA intermediate previously <br /> observed only with E. coli KptA protein. These mutations <br /> could be divided into four groups (see Fig. 2). First, we <br /> mutated residues or blocks of residues within completely or <br /> nearly completely conserved blocks of amino acids among <br /> those orthologous proteins known to complement activity <br /> of a tpt1-&#226;&#338;&#172; mutant; these included S15A/K16A, R23A/ <br /> H24A, K69A/R71S, Q88A/H90A/S91A, and H140A/V141A/ <br /> H142A, H140A/H142A, as well as the single mutants H90A, <br /> S91A, H117A, R138A, and H142A. Second, we mutated <br /> several residues that were different between S. cerevisiae and <br /> E. coli, since E. coli KptA protein has a slow second step and            FIGURE 3. Titration of Tpt1 K69A/R71S and Tpt1 protein with tri- <br />                                                                           nucleotide substrate. Tpt1 K69A/R71S protein and Tpt1 protein were <br /> a visible intermediate under most conditions with a tri-                  assayed for 2&#226;&#172;&#732;-phosphotransferase activity as described in Materials <br /> nucleotide substrate (pApAppA), whereas yeast Tpt1p does                  and Methods, in 10 &#194;&#181;L reaction mixtures containing 1 fmole <br /> not. These included C85R, M136G, and S183A variants of                    p*ApAppA, the indicated concentration of NAD, and enzyme, for 30 <br /> Tpt1p. Third, we made mutations at several positions that                 min at 30&#194;&#176;C, and products were separated by thin-layer chromatog- <br />                                                                           raphy as described in Materials and Methods. a, no protein; b&#226;&#8364;&#8220;g, h&#226;&#8364;&#8220;m, <br /> were different in E. coli and another organism, relative to S.            fivefold serial titrations of Tpt1 K69A/R71S protein, starting with 4.3 <br /> cerevisiae, including H140A, S155A, and I167L/D168K.                      &#194;&#181;M protein, at 0.25 mM NAD (b&#226;&#8364;&#8220;g) and at 10 mM NAD (h&#226;&#8364;&#8220;m); n&#226;&#8364;&#8220;r, <br /> Fourth, we made changes at other positions that are con-                  s&#226;&#8364;&#8220;w, fivefold serial titrations of Tpt1 protein, starting with 280 nM <br /> served between yeast and E. coli, but variable in other or-               protein at 0.25 mM NAD (n&#226;&#8364;&#8220;r) and at 10 mM NAD (s&#226;&#8364;&#8220;w). <br /> ganisms, including Q88Y, R158H, N185A, and D34A (see <br /> Fig. 2). Each of the Tpt1p variants was purified after ex- <br /> pression in E. coli, and then examined for the accumulation               protein produces an extra radioactive spot that appears at <br /> of intermediate.                                                          concentrations of protein near the titration point (see Fig. 3, <br />    Of the 21 Tpt1p variants examined, only Tpt1 K69A/                     lanes c,i). This spot appears to correspond to the interme- <br /> R71S protein produces a substantial amount of the reaction                diate formed by KptA protein, in which trinucleotide is <br /> intermediate. The titration of Tpt1 K69A/R71S with labeled                ADP-ribosylated at the 2&#226;&#172;&#732;-phosphate (Spinelli et al. 1999). <br /> trinucleotide (p*ApAppA) substrate, like that of wild-type                The mobility of the extra spot is similar to that of interme- <br /> Tpt1p, reveals two results, as shown in Figure 3: First, for-             diate formed by KptA protein with this substrate (data not <br /> mation of product RNA requires &#226;&#710;&#188;0.4 &#194;&#181;M Tpt1 K69A/R71S                     shown), and the spot appears to be converted to product at <br /> protein, whereas only 0.3 nM Tpt1p is required for similar                higher concentrations of Tpt1 K69A/R71S protein (Fig. 3, <br /> or greater product formation. Second, Tpt1 K69A/R71S                      lanes b,h). Also like the intermediate produced by KptA <br />                                                                                               protein, more of this material is present <br />                                                                                               in Tpt1 K69A/R71S titrations at high <br />                                                                                               concentrations of NAD (Spinelli et al. <br />                                                                                               1999) than at low concentrations of <br />                                                                                               NAD (Fig. 3). We provide further evi- <br />                                                                                               dence below that this material is the in- <br />                                                                                               termediate. <br />                                                                                                  Tpt1 K69A/R71S protein has a slow <br />                                                                                                  second step with <br />                                                                                                  trinucleotide substrate <br />                                                                                                  A time course of the 2&#226;&#172;&#732;-phosphotrans- <br />                                                                                                  ferase reaction with Tpt1 K69A/R71S <br />                                                                                                  protein and labeled trinucleotide reveals <br />                                                                                                  that the protein is specifically deficient <br />                                                                                                  in the second step of the reaction, reso- <br /> FIGURE 2. Amino acid alignment of 2&#226;&#172;&#732;-phosphotransferases. The amino acid alignment               lution of the intermediate to product <br /> shown contains 2&#226;&#172;&#732;-phosphotransferase orthologs from M. musculus, Schizosaccharomyces             (Fig. 4). As shown in Figure 4B and <br /> pombe, Arabidoposis thaliana, Escherichia coli, A. fulgidis, and Saccharomyces cerevisiae. The   quantified in Figure 4D, intermediate <br /> sequences resulted from a BLAST search of the Saccharomyces cerevisiae sequence; the align- <br /> ment was done using clustalW (http://searchlauncher.bcm.tmc.edu/multi-align.html) and box- <br />                                                                                                  formation coincides with the decrease in <br /> shade (http://www.ch.embnet.org/software/boxform.html). The positions of K69 and R71 are         substrate concentration at early times <br /> shown with arrows.                                                                               and distinctly precedes product forma- <br />  <br />                                                                                                                www.rnajournal.org           101 <br />  Steiger et al. <br />  <br />  <br />  <br />                                                                                termediate is formed when NAD is labeled rather than the <br />                                                                                trinucleotide substrate. Figure 5A shows a 2&#226;&#172;&#732;-phosphotrans- <br />                                                                                ferase reaction with Tpt1 K69A/R71S protein, using [&#226;&#65533;&#163;-32P <br />                                                                                adenylate]NAD (abbreviated Ap*pN) and unlabeled tri- <br />                                                                                nucleotide pApAppA. Tpt1 protein readily converts most of <br />                                                                                the labeled NAD to product Appr&gt;p (Fig. 5A, lanes b,c), <br />                                                                                demonstrating that most of the NAD is competent for re- <br />                                                                                action in the presence of excess trimer. As shown in Figure <br />                                                                                5A, lanes f&#226;&#8364;&#8220;h, Tpt1 K69A/R71S protein converts the labeled <br />                                                                                NAD both to product Appr&gt;p and to intermediate, as pre- <br />                                                                                viously observed with KptA protein (Spinelli et al. 1999). <br />                                                                                Much less product is formed with Tpt1 K69A/R71S protein <br />                                                                                than with Tpt1 protein under these conditions, presumably <br />                                                                                because Tpt1 K69A/R71S protein is much less active than <br />                                                                                Tpt1 protein; nonetheless, the intermediate accounts for a <br />                                                                                significant percentage of the products formed by Tpt1 <br />                                                                                K69A/R71S protein. <br />                                                                                   Second, as expected of an intermediate containing NAD, <br />                                                                                the mobility of the intermediate is altered when formed <br /> FIGURE 4. Time course of reaction with Tpt1 protein or Tpt1p                   with an NAD derivative. We compared the 2&#226;&#172;&#732;-phos- <br /> K69A/R71S and trinucleotide substrate. Tpt1 K69A/R71S protein (2.2             photransferase activity of Tpt1 K69A/R71S protein, using <br /> &#194;&#181;M) and Tpt1 protein (0.14 nM) were assayed for 2&#226;&#172;&#732;-phosphotrans- <br /> ferase activity as described in Materials and Methods in 50 &#194;&#181;L reaction        NAD and biotin-NAD (Fig. 5B). Both Tpt1 protein and <br /> mixtures containing 2 &#194;&#181;M p*ApAppA substrate and 10 mM NAD, and                 Tpt1 K69A/R71S protein will convert labeled trinucleotide <br /> 3 &#194;&#181;L aliquots were sampled at different times, spotted to PEI cellulose        substrate to product with either NAD or biotinylated NAD <br /> plates, and resolved in buffer containing 2M sodium formate (pH 3.5).          (Fig. 5B, lanes d&#226;&#8364;&#8220;f,j,k). Tpt1 K69A/R71S protein forms in- <br /> (A) Time course of Tpt1 protein activity. (Lane a) No protein, 0 min; <br /> (lanes b&#226;&#8364;&#8220;m) products formed with Tpt1 protein, at 0, 1, 2, 5, 10, 15,          termediate with both NAD derivatives (Fig. 5B, lanes d&#226;&#8364;&#8220;g), <br /> 20, 30, 45, 60, 75, 90 min. (B) Time course of Tpt1 K69A/R71S protein          but the intermediate has different mobility with biotin- <br /> activity. (Lanes a&#226;&#8364;&#8220;l) Products formed with Tpt1 K69A/R71S protein, at          NAD than with NAD. Since the intermediate has an altered <br /> 0, 1, 2, 5, 10, 15, 20, 30, 45, 60, 75, 90 min; (lane m) no protein, 90 min. <br /> (C,D) Quantification of time course of reactions of Tpt1 protein and <br /> Tpt1 K69A/R71S protein with trinucleotide substrate. Products, inter- <br /> mediate, and substrate were quantified by PhosphorImager and plot- <br /> ted using SigmaPlot. <br />  <br />  <br />  <br /> tion, since only &#226;&#710;&#188;30% of the starting substrate becomes <br /> product over the extended 90-min time course, whereas <br /> 40% of the substrate is converted to intermediate in the first <br /> 5 min. The appearance of intermediate earlier than product <br /> is required of a true intermediate along the reaction path- <br /> way, and is similar to that observed with KptA protein and <br /> this substrate (Spinelli et al. 1999). In contrast, wild-type <br /> Tpt1 protein does not show any accumulation of interme- <br /> diate (Fig. 4A,C). The accumulation of intermediate by <br /> Tpt1 K69A/R71S protein is biphasic (Fig. 4), with a distinct <br /> burst phase of intermediate formation followed by a slow <br /> steady state of product formation. During this early burst                     FIGURE 5. The intermediate formed with Tpt1K69A/R71S protein <br /> phase, 75 pmol of intermediate is formed by 110 pmol of                        contains NAD. (A) Intermediate is formed with labeled NAD. Reac- <br /> Tpt1 K69A/R71S enzyme, indicating that at least 70% of the                     tion mixtures containing 5 nM pApAppA substrate and 0.5 nM <br /> Tpt1 K69A/R71S protein is catalytically active.                                Ap*pN, as well as the indicated amount of unlabeled NAD, were <br />                                                                                incubated with no protein (lanes a,e), 0.7 &#194;&#181;M Tpt1 protein (lanes b&#226;&#8364;&#8220;d), <br />                                                                                or 2.2 &#194;&#181;M Tpt1 K69A/R71S protein (lanes f&#226;&#8364;&#8220;h). (B) Intermediate is <br />                                                                                formed with biotin-NAD. Reaction mixtures containing 1 nM <br /> The intermediate contains NAD                                                  p*ApAppA and 75 &#194;&#181;M NAD or biotin-NAD, as indicated, were incu- <br />                                                                                bated with protein, and products were resolved on thin-layer plates as <br /> Two lines of evidence indicate that the observed interme-                      described. (Lanes a&#226;&#8364;&#8220;c) Buffer; (lanes d&#226;&#8364;&#8220;h) Tpt1 K69A/R71S protein at <br /> diate, as expected of ADP-ribosylated RNA, is derived from                     4.3 &#194;&#181;M (lanes d,f,h) or 2.2 &#194;&#181;M (lanes e,g); (lanes i&#226;&#8364;&#8220;k) 0.7 &#194;&#181;M Tpt1 <br /> NAD as well as from trinucleotide substrate. First, the in-                    protein. <br />  <br />  <br /> 102       RNA, Vol. 11, No. 1 <br />                                                                                            2&#226;&#180;&#8226;-phosphotransferase reaction of yeast Tpt1p <br />  <br />  <br />  <br /> mobility, it must contain the biotin moiety of NAD, located                 beads treated with wild-type Tpt1 protein release product <br /> on the adenine nucleotide. We conclude that the interme-                    p*ApApA (Fig. 6, lane k) that is indistinguishable from <br /> diate formed by Tpt1 K69A/R71S protein with trinucleotide                   product generated from the reaction mixture (Fig. 6, lane l). <br /> substrate and NAD, like that formed by KptA protein, is                     This experiment provides direct evidence that the interme- <br /> comprised of RNA that is ADP-ribosylated at its 2&#226;&#172;&#732;-phos-                    diate formed with Tpt1 K69A/R71S protein and biotin- <br /> phate. If so, the intermediate ought to be competent for step               NAD can be converted to the product pApApA. Further- <br /> 2 of the reaction. This is shown explicitly below.                          more, this experimental approach can be used in principle <br />                                                                             to provide a means for identifying Tpt1p substrates. <br /> The intermediate formed by Tpt1 K69A/R71S protein <br /> can be trapped on streptavidin beads and converted                          Reaction intermediate is formed with ligated tRNA <br /> to product                                                                  substrate, using Tpt1 K69A/R71S protein <br />  <br /> Generation of the ADP-ribosylated RNA intermediate with                     Although the above analysis documents the formation of <br /> biotin-NAD allows for purification of intermediate on                       intermediate with Tpt1 K69A/R71S protein and the syn- <br /> streptavidin beads, as demonstrated with trinucleotide sub-                 thetic trinucleotide substrate p*ApAppA, we also wanted to <br /> strate p*ApAppA (Fig. 6). Whereas the intermediate that is                  assess whether and to what extent intermediate was formed <br /> formed with Tpt1 K69A/R71S protein and NAD (Fig. 6,                         with the in vivo substrate, ligated tRNA (Spinelli et al. <br /> lane b) washes through the streptavidin beads very effi-                    1997). Figure 7 shows the NAD dependence of the phos- <br /> ciently (Fig. 6, lanes c,d), intermediate formed with Tpt1                  photransferase reaction using tRNA labeled during tran- <br /> K69A/R71S protein and biotin-NAD (Fig. 6, lane e) is spe-                   scription with [&#226;&#65533;&#163;-32P]ATP and then spliced in vitro. Under <br /> cifically absent from the washes after incubation with strep-               these labeling conditions all adenosine residues and the 2&#226;&#172;&#732;- <br /> tavidin beads (Fig. 6, lanes f,g). Two experiments indicate                 phosphate are labeled; thus both Appr&gt;p formation (Fig. <br /> that intermediate remains bound to the streptavidin beads                   7A) and loss of the splice junction (Fig. 7B) can be moni- <br /> after the washes: First, washed beads that are treated with                 tored under appropriate conditions. <br /> phenol release the intermediate from the streptavidin col-                     As shown in Figure 7A, Tpt1 K69A/R71S protein is ex- <br /> umn (Fig. 6, lane h); this material has the same mobility as                tremely ineffective in Appr&gt;p formation using ligated <br /> intermediate in the original reaction mixture (Fig. 6, lane e)              tRNA. It requires 2 &#194;&#181;M Tpt1 K69A/R71S protein to produce <br /> and as intermediate from the original reaction mixture after                comparable amounts of Appr&gt;p as are produced with 50 <br /> treatment with phenol (Fig. 6, lanes i,j). Second, washed                   pM Tpt1 protein; furthermore, Tpt1 K69A/R71S protein <br />                                                                             appears to require significantly more NAD than does Tpt1 <br />                                                                             protein to produce Appr&gt;p product. <br />                                                                                Tpt1 K69A/R71S protein also appears to form interme- <br />                                                                             diate with ligated tRNA, as determined by examination of <br />                                                                             the splice junction after precipitation of the tRNA and <br />                                                                             nuclease P1 treatment (Fig. 7B). This treatment generates <br />                                                                             labeled pA from the tRNA backbone and dinucleotide <br />                                                                             pGppA from the splice junction (Zillmann et al. 1991). As <br />                                                                             expected, with either Tpt1p or Tpt1 K69A/R71S protein the <br />                                                                             amount of splice junction dinucleotide is significantly re- <br />                                                                             duced as reaction proceeds at higher concentrations of <br />                                                                             NAD. However, reactions containing Tpt1 K69A/R71S pro- <br />                                                                             tein also accumulate an additional spot at higher concen- <br />                                                                             trations of NAD (Fig. 7B, lanes q&#226;&#8364;&#8220;t). This spot is likely the <br />                                                                             P1 nuclease product of the intermediate, splice junction <br />                                                                             dinucleotide that is ADP-ribosylated at its 2&#226;&#172;&#732;-phosphate. <br /> FIGURE 6. Intermediate formed with biotin-NAD can be resolved on            Like the intermediate formed with trinucleotide substrate, <br /> streptavidin beads. Reaction mixtures containing 1 nM p*ApAppA              the spot has slower mobility than the dinucleotide from <br /> substrate, 12 &#194;&#181;M Tpt1 K69A/R71S protein, and 75 &#194;&#181;M NAD (lanes <br /> b&#226;&#8364;&#8220;d) or biotin NAD (lanes e&#226;&#8364;&#8220;l) were incubated for 40 min and then <br />                                                                             which it is derived, it only forms with Tpt1 K69A/R71S <br /> incubated with streptavidin beads as described in Materials and Meth-       protein, and it is formed maximally at high concentrations <br /> ods. (Lane a) No protein; (lanes b,d) reaction mixture; (lanes c,f) first   of NAD. <br /> wash after streptavidin beads; (lanes d,g) second wash after streptavi- <br /> din beads; (lane h) washed beads extracted with phenol; (lane j) re- <br /> action mixture (from lane e) extracted with phenol; (lane i) mixture of     Expression of Tpt1 K69A/R71S protein cannot <br /> material from lanes h and j, cospotted on thin-layer plate; (lane k)        complement a tpt1-&#226;&#338;&#172; mutant strain <br /> washed beads treated with Tpt1p; (lane l) reaction mixture (from lane <br /> e) treated with Tpt1p. R, reaction mixture; w1, w2 eluant from washes       Because Tpt1 K69A/R71S protein is so ineffective with li- <br /> of beads; B, beads after washes; cs, cospotting.                            gated tRNA substrate and produces substantial amounts of <br />  <br />                                                                                                            www.rnajournal.org       103 <br />  Steiger et al. <br />  <br />  <br />  <br />                                                                            acterized for its E. coli ortholog KptA protein (Spinelli et al. <br />                                                                            1999). Evidence for the intermediate with trinucleotide sub- <br />                                                                            strate includes: the appearance of an extra spot at Tpt1 <br />                                                                            K69A/R71S protein concentrations near the titration point, <br />                                                                            but not in samples containing more protein; and the ap- <br />                                                                            pearance of this spot prior to the appearance of product and <br />                                                                            concurrent with the loss of substrate in time courses. Evi- <br />                                                                            dence that the intermediate contains NAD includes: the <br />                                                                            demonstration of intermediate formation with labeled <br />                                                                            NAD, formation of intermediates with differing mobilities <br />                                                                            using NAD or biotin-NAD, and physical trapping of the <br />                                                                            intermediate formed with biotin-NAD on streptavidin <br />                                                                            beads. Thus Tpt1 K69A/R71S protein, like KptA protein, <br />                                                                            forms an intermediate containing 2&#226;&#172;&#732;-phosphorylated RNA <br />                                                                            that is ADP-ribosylated at its 2&#226;&#172;&#732;-phosphate. Since Tpt1 pro- <br />                                                                            tein recognizes the intermediate formed with Tpt1 K69A/ <br />                                                                            R71S protein and biotin-NAD and converts it to product, <br />                                                                            we argue that Tpt1 protein also acts through the interme- <br />                                                                            diate. That Tpt1 K69A/R71S mutant protein also accumu- <br /> FIGURE 7. Detection of the reaction intermediate with Tpt1 K69A/           lates an intermediate with tRNA underscores the conclusion <br /> R71S protein and ligated tRNA substrate. (A) NAD dependence of             that the mechanism by which Tpt1 removes the 2&#226;&#172;&#732; phos- <br /> Appr&gt;p formation from ligated tRNA. 50 pM Tpt1p (lanes a&#226;&#8364;&#8220;i) or 2.2 <br />                                                                            phate from its natural tRNA substrate occurs through this <br /> &#194;&#181;M Tpt1 K69A/R71S protein (lanes j&#226;&#8364;&#8220;r) was incubated with 0.15 nM <br /> ligated tRNA and increasing concentrations of NAD, and products            intermediate. Thus the Tpt1 protein family has a conserved <br /> were resolved on thin-layer plates as described above. (Lanes a,j) No      two-step mechanism. Moreover, the finding that alteration <br /> NAD; (lanes b&#226;&#8364;&#8220;i,k&#226;&#8364;&#8220;r) 2 nM, 20 nM, 200 nM, 2 &#194;&#181;M, 20 &#194;&#181;M, 200 &#194;&#181;M,             of conserved residues (as opposed to those specifically dif- <br /> 2 mM, and 20 mM NAD. (B) NAD dependence of intermediate for- <br /> mation from ligated tRNA. Samples from A above were treated with <br />                                                                            ferent between S. cerevisiae and E. coli) affects the accumu- <br /> P1 nuclease, and products were resolved on thin-layer plates. SJ, splice   lation of the intermediate further underscores the impor- <br /> junction; Appr-SJ, ADP-ribosylated splice junction.                        tance of the intermediate in the general catalytic mechanism <br />                                                                            of this class of enzymes. <br />                                                                               It is not known how the K69A/R71S mutation slows the <br /> the intermediate, we inferred that Tpt1 K69A/R71S protein                  second step of the Tpt1 reaction after intermediate forma- <br /> would be unable to complement a tpt1-&#226;&#338;&#172; strain, even if                     tion. The intermediate has a phosphodiester bond connect- <br /> overproduced. To test this, we used a previously described                 ing the 1&#226;&#172;&#8482;-oxygen of ADP-ribose and the oxygen of the <br /> plasmid shuffle technique to test for Tpt1 protein function                2&#226;&#172;&#732;-phosphate of the RNA (Fig. 1). The second step of the <br /> in vivo (Spinelli et al. 1998). We transformed a tpt1-&#226;&#338;&#172; [p                 reaction is presumed to be a transesterification reaction in <br /> CEN CYH2 TRP1 TPT1] strain with a plasmid expressing                       which the 2&#226;&#172;&#8482;-oxygen of ADP-ribose displaces the oxygen of the <br /> Tpt1 K69A/R71S protein under control of the PGAL pro-                      2&#226;&#172;&#732;-phosphate of the RNA, releasing the dephosphorylated <br /> moter [PGAL-tpt1-K69A/R71S], and then selecting against                    RNA, and forming ADP-ribose 1&#226;&#172;&#8482;-2&#226;&#172;&#8482; cyclic phosphate. One <br /> the CYH2 TRP1 plasmid on plates containing cyclohex-                       possibility is that the K69 and/or R71 residues are critical <br /> amide and galactose. No growth was observed under these                    for efficient catalysis of this transesterification reaction. Ear- <br /> conditions in strains transformed with the PGAL- vector                    lier studies of RNase A support the idea that a lysine residue <br /> expressing Tpt1 K69A/R71S protein or no protein, whereas                   is critical for the transesterification steps that result in cleav- <br /> normal growth was observed in strains expressing wild-type                 age (Barnard 1969; Raines 1998); in this case two lysine <br /> Tpt1p (data not shown). Since Tpt1p is overproduced                        residues stabilize the transition state of the reaction. A sec- <br /> eightfold to 20-fold over wild-type levels when expressed                  ond possibility is that the K69/R71 residues are important <br /> under PGAL control (Spinelli et al. 1997), we conclude that                for enzyme-intermediate binding. Thus, the reaction inter- <br /> Tpt1 K69A/R71S protein is unable to act as the only source                 mediate might form, but not remain bound long enough to <br /> of Tpt1p in yeast.                                                         allow the normal second step to occur. This would explain <br />                                                                            the observation that more intermediate is formed with <br />                                                                            higher concentrations of NAD, using trinucleotide substrate <br /> DISCUSSION <br />                                                                            (Fig. 3) or tRNA substrate (Fig. 7; Spinelli et al. 1999). <br /> We have provided a number of lines of evidence that Tpt1                   However, the very low KM observed with tRNA substrate <br /> K69A/R71S protein forms an intermediate during the 2&#226;&#172;&#732;-                     relative to trinucleotide substrate argues against weak bind- <br /> phosphotransferase reaction, and that this intermediate is                 ing of an intermediate formed with tRNA (Steiger et al. <br /> the same ADP-ribosylated RNA intermediate as that char-                    2001). Because K69 is not an essential residue, but R71 is <br />  <br /> 104       RNA, Vol. 11, No. 1 <br />                                                                                         2&#226;&#180;&#8226;-phosphotransferase reaction of yeast Tpt1p <br />  <br />  <br />  <br /> essential (Sawaya et al. 2005), we infer that the essential           in vitro transcription of yeast tRNAPhe with T7 RNA polymerase <br /> residue for the second step of the reaction is R71. Further           and [&#226;&#65533;&#163;-32P] ATP. Where indicated, nuclease P1 (Sigma) treatment <br /> analysis of Tpt1 K69A/R71S will aid in defining the precise           of reaction mixtures containing ligated tRNAPhe was done by pre- <br /> role of the residues involved, and the nature of the enzy-            cipitation of RNA, followed by treatment with 1 &#194;&#181;g nuclease P1 in <br />                                                                       buffer containing 20 mM Na acetate (pH 5.0) and 10 &#194;&#181;g carrier <br /> matic defect that causes accumulation of the ADP-ribo- <br />                                                                       tRNA. [&#226;&#65533;&#163;-32P adenylate]NAD (abbreviated Ap*pN) was made as <br /> sylated phosphorylated RNA intermediate.                              described (Spinelli et al. 1999). Biotin-NAD (6-biotin-17-NAD) <br />    The results presented here cause us to speculate that the          was obtained from Trevigen. Isolation of biotinylated reaction <br /> conserved function of the Tpt1 protein family involves de-            intermediate was done by incubation of 3 &#194;&#181;L of reaction mixtures <br /> phosphorylation of phosphorylated RNA, or a chemically                with 45 &#194;&#181;L of streptavidin agarose beads (Sigma) in binding buffer <br /> similar two-step chemical reaction, although the role of              (10 mM Tris-HCl pH 7.5, 1 mM EDTA, 0.1 mM NaCl, 100 &#194;&#181;g/mL <br /> Tpt1 protein orthologs in many organisms, particularly                BSA) for 20 min at room temperature, followed by three washes <br /> from eubacteria and archaea, is currently perplexing                  with 100 &#194;&#181;L binding buffer containing 1 M NaCl. RNA was re- <br /> (Spinelli et al. 1998). As shown in Figure 2, R71 is com-             leased either by treatment of the beads with Tpt1 protein at 30&#194;&#176;C <br /> pletely conserved among Tpt1 protein family members that              for 30 min, or by phenol extraction of beads at 50&#194;&#176;C. All reaction <br /> complement tpt1-&#226;&#338;&#172; strains, and more extensive alignment               products were separated on PEI-cellulose thin-layer plates devel- <br />                                                                       oped in buffer containing 2 M sodium formate pH 3.5. <br /> shows that R71 is one of the very few residues that is nearly <br /> 100% conserved in Tpt1 members. If R71 is critical for <br /> efficient catalysis of the second step, and the residue is so          <br /> highly conserved, it seems highly likely that it is important        <br /> for the second step of a dephosphorylation or similar reac-            <br /> tion in other organisms. Our method of isolating the reac-             <br /> tion intermediate using biotin-NAD and Tpt1 K69A/R71S <br /> protein (Fig. 6) may aid in identification of other substrates         <br /> of the yeast enzyme, such as HAC1 mRNA (Gonzalez et al. <br /> 1999), or of substrates of other Tpt1 family members.                  <br />                                                                     <br /> MATERIALS AND METHODS                                 <br /> Construction and verification of site-directed mutants                 <br /> in TPT1                                                              <br /> Site-directed mutagenesis was performed using the Kunkel muta-   <br /> genesis procedure (Kunkel et al. 1987), using oligomers with the       <br /> appropriate nucleotide changes and pET 24b containing wild-type <br />                                                                      <br /> TPT1, followed by transformation into E. coli XL1 Blue strains.           <br /> Mutations were confirmed initially by restriction analysis since       <br /> mutagenic oligomers also alter the restriction enzyme <span id='am-82' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, and   <br /> then by DNA sequencing.         <br /> Purification of Tpt1 protein variants        <br /> pET 24b plasmids expressing Tpt1p-His6 variants were trans-              <br /> formed into E. coli strain BL21 (DE3), followed by growth at 37&#194;&#176;C   <br /> and induction of expression at A600 0.4 by addition of 1 mM IPTG         <br /> for 2 h. Tpt1-His6 proteins were purified by immobilized metal        <br /> ion affinity chromatography using a Ni2+ agarose (Clonetech) col-    <br /> umn as previously described (Steiger et al. 2001), yielding protein    <br /> at 0.5&#226;&#8364;&#8220;2.4 mg/mL (Bradford 1976) that was &gt; 80% pure, as judged       <br /> by SDS-PAGE and silver staining.          <br /> Assay of activity of Tpt1 protein variants        <br /> Enzymatic analysis of Tpt1 protein variants was done as previously        <br /> described (Steiger et al. 2001) using 10 mM NAD and either 0.1        <br /> nM p*ApAppA (Kierzek et al. 2000), 5&#226;&#172;&#732;-end labeled with poly-              <br /> nucleotide kinase and [&#226;&#65533;&#165;-32P] ATP, or ligated tRNA prepared by           <br /> </body></html>