<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Apr. 2002, p. 229&#226;&#8364;&#8220;240                                                                                            Vol. 1, No. 2 <br /> 1535-9778/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/EC.1.2.229&#226;&#8364;&#8220;240.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> Nnf1p, Dsn1p, Mtw1p, Nsl1p: New Group Proteins Important <br />       Chromosome Segregation Saccharomyces cerevisiae <br />                                                         Ghia M. Euskirchen* <br />                          Department Biological Sciences, Columbia University, New York, New York 10027 <br />                                             Received 22 August 2001/Accepted 10 January 2002 <br />  <br />              Previously, antibodies raised nuclear envelope-enriched fraction yeast, essential <br />           gene NNF1 cloned reverse genetics. Here shown conditional nnf1-17 mutant decreased <br />           stability minichromosome addition mitotic spindle defects. I identified novel essential genes <br />           DSN1, DSN3, NSL1 genetic interactions nnf1-17. Dsn3p equivalent the <br />           kinetochore protein Mtw1p. By indirect immunofluorescence, proteins, Nnf1p, Mtw1p, Dsn1p, and <br />           Nsl1p, colocalize region spindle poles. Based colocalization four <br />           proteins, minichromosome instability spindle defects seen nnf1 mutants, I propose Nnf1p <br />           new group proteins necessary chromosome segregation. <br />  <br />  <br />    The process mitotic cell division ensures chromo-                    Mutations components mitotic spindle, SPB, <br /> somes faithfully duplicated equally segregated               kinetochores, MT-associated motor proteins, various reg- <br /> mother daughter cells, absence genes                  ulatory enzymes result aneuploidy. In set mu- <br /> presence irregular numbers typically lethal. The micro-                tants, chromosome segregation asymmetric, chromo- <br /> tubule (MT) cytoskeleton associated structures                   somal DNA segregating pole bipolar spindle. <br /> responsible orchestrating chromosome segregation                     Mutants group include kinetochore proteins the <br /> maintaining genetic continuity. In eukaryotic cells, chro-              Ndc80p complex (76), Ndc10p (16), protein kinase <br /> mosomal movements mitosis conserved                       Ipl1p, localizes mitotic spindle kinetochores <br /> mediated organelles: bipolar mitotic spindle,               (2, 9, 23, 37). Alternatively, duplication SPB termi- <br /> kinetochores (centromere DNA associated proteins),                   nate particular stage, resulting monopolar spindle and <br /> the centrosomes (MT organizing centers). These common fea-                   diploidization, seen mps2-1 ndc1-1 mutants (70, 78, <br /> tures chromosome segregation observed microscop-                   79). In chromosome missegregation mutants, chromo- <br /> ically, aided fluorescence situ hybridization studies            loss seemingly random. In certain tub4 spc110 <br /> individual chromosomes fixed cells (20) green fluores-                mutants, defective &#226;?&#165;-tubulin SPB compo- <br /> cent protein (GFP)-tagged chromosomes living cells (23, 46,               nent Spc110p, respectively, SPBs assembled the <br /> 49, 66, 67). Separation sister chromatids initiated             spindle compromised mitosis (36, 61, 64, 68, 69). <br /> progresses centromeric regions kineto-                    Failure chromosome segregation spindle defects are <br /> chores mediate attachment sister chromatids                    seen duo1 dam1 mutants, expected <br /> spindle MTs. In Saccharomyces cerevisiae, MTs nucle-                 Duo1p Dam1p mitotic spindle <br /> ated spindle pole body (SPB), yeast equivalent             kinetochores wild-type cells (10, 23, 24, 34). These <br /> centrosome. The SPB large laminar structure em-                 mutants demonstrate deficient SPBs, kinetochores, or <br /> bedded nuclear membrane cell cycle                 spindles lead disomy individual chromosomes or <br /> nucleates cytoplasmic nuclear MTs (5, 6). One array                 diploidization entire genome single nucleus. <br /> of nuclear MTs interdigitate span pole pole,                  In S. cerevisiae, nuclear division occurs bud axis, <br /> a second array captured kinetochores. Chromosome                   positioning mitotic spindle nar- <br /> segregation requires shortening kinetochore MTs (an-                  row bud neck critical ensure equal distribution DNA <br /> aphase A) elongation pole-<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-pole MTs (anaphase                   mother daughter cells. Dynamic contacts be- <br /> B). In budding yeast, dramatic movement chroma-                  tween cytoplasmic MTs cell cortex chiefly <br /> tin concomitant spindle elongation, anaphase                responsible nuclear migration orientation mi- <br /> A observed (21, 49, 67). Sister centromeres               totic spindle (8, 57). Mutants complete spindle exten- <br /> are proximally marked GFP shown align                      sion defective nuclear positioning characterized <br /> along spindle equator, separate, oscillate                 populations anucleate daughter cells binucleate <br /> spindle poles anaphase B spindle elongation (49). Fol-                mother cells. These mutants typically defective cytoplas- <br /> lowing end mitosis cell cycle,             mic MT proteins perturbations SPB. For exam- <br /> centromeres clustered near SPBs (21, 33).                            ple, detachment cytoplasmic MTs SPBs <br />                                                                              loss of outer plaque proteins Cnm67p <br />                                                                              Spc72p leads multinucleated cells (4, 60). Therefore, <br />    * Present address: Yale University, Department Molecular, Cel- <br /> lular, Developmental Biology, KBT Room 918, 266 Whitney Ave., <br />                                                                              observation binucleate cells indicate SPB defect. <br /> New Haven, CT 06511. Phone: (203) 432-3510. Fax: (203) 432-6161.                In earlier study, raised antibodies nuclear <br /> E-mail: ghia.euskirchen@yale.edu.                                            envelope-enriched fraction yeast subsequently cloned <br />  <br />                                                                        229 <br />  230       EUSKIRCHEN                                                                                                                              EUKARYOT. CELL <br />  <br />  <br /> the essential gene NNF1 (necessary nuclear function 1) <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> yeast expression library (56). Nnf1p small coiled- <br /> coil protein 201 amino acids homology known                                            TABLE 1. Yeast strains used study <br /> proteins. Cells carrying mutations NNF1 exhibit primarily                                Strain(s)a                              Genotype <br /> short mitotic spindles and, lesser extent, aberrant cyto-                        W961-5A....................MATa leu2-3,112 trp1-1 ura3-1 ade2-1 HIS3&#226;&#171;&#185; <br /> plasmic MTs defects nuclear migration. Nnf1p                                                               can1-100 <br /> protein low abundance detected Western                            GEY145.....................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 <br /> blotting unless overexpressed. When cells overexpressing                                                         trp1-1/TRP1&#226;&#171;&#185; ura3-1/ura3-1 ade2-ade2-1 <br />                                                                                                                        HIS3&#226;&#171;&#185;/his3-11,15 can1-100/can1-100 <br /> NNF1 fractionated, Nnf1p predominantly                                                             NNF1::LEU2/NNF1::LEU2 <br /> nuclear fraction extracted presence 8                       GEY146.....................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 <br /> M urea guanidine hydrochloride. These studies veri-                                                            trp1-1/TRP1&#226;&#171;&#185; ura3-1/ura3-1 ade2-/ade2-1 <br /> fied Nnf1p nuclear protein elucidated                                                       HIS3&#226;&#171;&#185;/his3-11,15 can1-100/can1-100 <br /> nature function. In work, I present results                                                     nnf1-17::LEU2/nnf1-17::LEU2 <br />                                                                                       CH1305......................MATa lys2 leu2 ura3 ade2 ade3 can1 <br /> extensive genetic screens initiated condi-                         ABY112 .....................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-289/ <br /> tional nnf1-17 allele. I Nnf1p important chro-                                                        trp1-289 ura3-52/ura3-52 <br /> mosome segregation identify functionally related                                                             his3&#226;&#338;&#172;1/his3&#226;&#338;&#172;1CNM67::GFP <br /> proteins.                                                                                                              (S65T)-kanMX6/CNM67::GFP(S65T)-kanMX6 <br />                                                                                       CY6 ............................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br />                                                                                                                        ura3-1/ura3-1 ade2-/ade2-1 his3-11,15 his3- <br />                         MATERIALS AND METHODS <br />                                                                                                                        11,15 can1-100/can1-100 <br />                                                                                       GEY160.....................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br />    Strains, media, culture conditions. Yeast cell culture genetic manip-                                       ura3-1/ura3-1 ade2-/ade2-1 ade3&#226;&#338;&#172;/ade3&#226;&#338;&#172; his3- <br /> ulations, including mating, diploid isolation, sporulation, tetrad analysis,                                       11,15/his3-11,15 can1-100/can1-100 <br /> were performed described previously (52, 58). Yeast strains (Table 1)          GEY122.....................MAT&#226;?&#163; leu2-3,112 trp1-1 ura3-1 ade2-1 his3- <br /> derivatives W303, CH1305 (42) ABY112 (4). Media                                                 11,15 can1-100 NNF1::LEU2 <br /> prepared according standard recipes (3, 58, 74). Sectoring assays per-        GEY138.....................MAT&#226;?&#163; leu2-3,112 trp1-1 ura3-1 ade2-1 his3- <br /> formed low-adenine (5 mg/liter) synthetic medium.                                                                   11,15 can1-100 nnf1-17::LEU2 <br />    Plasmid constructions. All DNA bacterial manipulations stan-           GEY110.....................MATa leu2-3,112 trp1-1 ura3-1 ade2-1 HIS3&#226;&#171;&#185; <br /> dard protocols (55). Plasmids listed Table 2. The CEN vectors pRS313                                            can1-100 NNF1-myc6::LEU2 <br /> (HIS3 marker), pRS314 (TRP1 marker), pRS316 (URA3 marker) (59)               GEY111.....................MATa leu2-3,112 trp1-1 ura3-1 ade2-1 HIS3&#226;&#171;&#185; <br /> primarily used cloning. The CEN plasmid pCT3 contains URA3 marker                                              can1-100 NNF1-myc6::LEU2 <br /> and gift C. Thompson. The 2&#226;?&#174;m vector YEp352 contains URA3                                               CNM67::GFP(S65T)-kanMX6 <br /> marker gift A. Tzagoloff. Three plasmids (pGE98 pGE100)             GEY165.....................MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br /> were constructed synthetic lethal screen. A 1.4-kb XmnI/PvuII fragment                                         ura3-1/ura3-1 ade2-1/ade2-1 ade3&#226;&#338;&#172;/ade3&#226;&#338;&#172; <br /> containing NNF1 promoter ligated SmaI site pCH1122                                          his3-11,15/his3-11,15 can1-100/can1-100 <br /> create pGE98 SmaI site pRS314 create pGE99. Plasmid                                                 nnf1-17::LEU2/nnf1-17::LEU2 <br /> pGE100 constructed pRS314 vector steps. First, ADE3               YSLP1........................MATa lys2 leu2 trp1 ura3 ade2 ade3 can1 <br /> marker band isolated 3.6-kb EagI/NheI fragment pCH1023                                               nnf1-17::LEU2 <br /> cloned EagI SpeI sites pRS314. Second, 1.4-kb XmnI/PvuII          YSL6 ..........................MATa lys2 leu2 trp1 ura3 ade2 ade3 can1 nsl1-6 <br /> fragment containing NNF1 promoter inserted SmaI site                                         nnf1-17::LEU2 <br /> multiple cloning site complete pGE100.                                             YSL8 ..........................MATa lys2 leu2 trp1 ura3 ade2 ade3 can1 nsl1-8 <br />    Generation NNF1 mutants. Temperature-sensitive mutations NNF1                                            nnf1-17::LEU2 <br /> generated methods described previously(43, 47). Briefly, NNF1 amplified        YSL68 ........................MATa lys2 leu2 trp1 ura3 ade2 ade3 can1 nsl-68 <br /> by mutagenic PCR, cotransformed gapped vector nnf1::URA3                                               nnf1-17::LEU2 <br /> strain carrying PGAL1-NNF1 HIS3 plasmid. Transformants selected          YSL29 ........................MATa lys2 leu2 trp1 ura3 ade2 ade3 can1 nsl2- <br /> 25&#194;&#176;C SC-His-Leu medium (containing 2% galactose sole carbon                                                  29(dsn3-29) nnf1-17::LEU2 <br /> source) maintain expression NNF1. The transformation plates           GEY170 172.........MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br /> replica plated sets SC-Leu plates incubated 25 37&#194;&#176;C.                                             ura3-1/ura3-1 ade2-/ade2-1 his3-11,15 his3- <br /> Plasmid DNA potential temperature-sensitive nnf1 mutants rescued                                              11,15 can1-100/can1-100 dsn1::HIS3/DSN1 <br /> into Escherichia coli, retested nnf1::URA3 yeast strain, sequenced. The    GEY190 191.........MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br /> nnf1-17 allele analyzed amino acid changes: C53S, V78D,                                             ura3-1/ura3-1 ade2-1/ade2-1 ade3&#226;&#338;&#172;/ade3&#226;&#338;&#172; <br /> C121G. For integration NNF1 chromosomal locus one-step gene re-                                              his3-11,15/his3-11,15 can1-100/can1-100 <br /> placement, LEU2 selectable marker NNF1 3&#226;&#172;&#732; noncoding region                                                nsl1::LEU2/NSL1 <br /> added C-terminal ends NNF1 nnf1-17. The coding sequence           GEY210 212.........MATa/MAT&#226;?&#163; leu2-3,112/leu2-3,112 trp1-1/trp1-1 <br /> the 3&#226;&#172;&#732; ends NNF1 nnf1-17 changed ENY* ENYWILV*                                                      ura3-1/ura3-1 ade2-/ade2-1 ade3&#226;&#338;&#172;/ade3&#226;&#338;&#172; his3- <br /> (asterisks designate stop codons). Band-isolated 3.2-kb HindIII fragments                                         11,15 his3-11,15 can1-100/can1-100 <br /> transformed W303 strain W961-5A Leu&#226;&#171;&#185; selection. Correct integra-                                          nsl2(dsn3)::LEU2/NSL2(DSN3) <br /> tion NNF1 locus verified Southern analysis.                             GEY176.....................MAT&#226;?&#163; leu2-3,112 trp1-1 ura3-1 ade2-1 his3- <br />    Selection dosage suppressors. High- low-copy suppressors                                             11,15 can1-100 dsn1::HIS3 [pGE36; DSN1- <br /> temperature-sensitive growth defect nnf1-17 cells selected 35&#194;&#176;C fol-                                        GFP] <br /> lowing transformation strain GEY138 S. cerevisiae genomic libraries        GEY206.....................MATa leu2-3,112 trp1-1 ura3-1 ade2-1 ade3&#226;&#338;&#172; <br /> either 2&#226;?&#174;m (7) CEN (71) plasmids. Cells plated SC-Ura medium                                            his3-11,15 can1-100 nsl1::LEU2 [pGE55; <br /> plasmid selection allowed initial recovery 12 h 25&#194;&#176;C                                          NSL1-GFP] <br /> plates shifted restrictive temperature (35&#194;&#176;C) 4 days. A total 24   GEY216.....................MATa leu2-3,112 trp1-1 ura3-1 ade2-1 ade3&#226;&#338;&#172; <br /> colonies isolated 35&#194;&#176;C, corresponding 150,000 transformants                                         his3-11,15 can1-100 nsl2(dsn3)::LEU2 <br /> the permissive temperature 25&#194;&#176;C. Twenty 24 plasmids recovered                                           [pGE74; DSN3(NSL2)-GFP] <br /> these colonies allowed growth 35&#194;&#176;C retransformed strain GEY138. <br />                                                                                         a <br /> Restriction digests showed 20 plasmids contained different in-              Strains GEY prefix W303 background. <br /> serts. Subclones 2&#226;?&#174;m vector YEp352 identify genes <br /> suppressing nnf1-17 growth defect. Plasmids pGE164, pGE190, pGE151 <br />  VOL. 1, 2002                                                               NEW CHROMOSOME SEGREGATION PROTEINS IN YEAST                                         231 <br />  <br />  <br />                  TABLE 2. Plasmids used study                             genomic library (71) used clone gene(s) causing synthetic lethality <br />                                                                                   nnf1-17. Ura&#226;&#171;&#185; transformants selected SC-Ura low-Ade plates at <br />  Plasmida                                Relevant markers <br />                                                                                   30&#194;&#176;C. Two sectoring colonies identified transformation library <br /> pCH1122b ..............CEN, URA3, ADE3                                            DNA: colony isolate YSL6 (12,000 transformants) colony from <br /> pCH1023b ..............CEN, LEU2, ADE3                                            isolate YSL8 (14,000 transformants). Plasmid DNA sectored colonies <br /> pDK243c ................CEN, LEU2; ade3-2p                                        recovered checked ability confer sectoring the <br /> pW35d ....................2&#226;?&#174;m, URA3; SSD1/SRK1                                    synthetic lethal strains. Both plasmids complemented nonsectoring pheno- <br /> pGE36....................CEN, TRP1; DSN1-GFP                                      type strains YSL6 YSL8 nonsectoring <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> third <br /> pGE55....................CEN, TRP1; NSL1-GFP                                      strain, YSL68. The plasmids contained common yeast genomic insert from <br /> pGE74....................CEN, TRP1; DSN3-GFP                                      chromosome XVI. Subcloning insert pRS316 create pGE180 <br /> pGE81....................CEN, URA3; NSL1; pGE180 gap-repaired SnaBI-              reduced complementing region 2.0-kb PvuII/NruI fragment con- <br />                             EcoNI yeast strain YSLP1                           tained NSL1 (YPL233w) open reading frame (ORF). <br /> pGE82....................CEN, URA3; nsl1-6; pGE180 gap-repaired SnaBI-               Chromosomal NSL1 alleles nsl1-6, nsl1-8, nsl1-68 recovered <br />                             EcoNI yeast strain YSL6                            gap repair pGE180 parent strain YSLP1 (NSL1) mutant <br /> pGE83....................CEN, URA3; nsl1-8; pGE180 gap-repaired SnaBI-            strains YSL6 (nsl1-6), YSL8 (nsl1-8), YSL68 (nsl1-68). For gap repair, <br />                             EcoNI yeast strain YSL8                            pGE180 digested SnaBI EcoNI band isolated remove a <br /> pGE84....................CEN, URA3; nsl1-68; pGE180 gap-repaired                  920-bp fragment containing 640 bp wild-type NSL1 280 bp 5&#226;&#172;&#732; noncod- <br />                             SnaBI-EcoNI yeast strain YSL68                     ing region. Plasmid DNA isolated Ura&#226;&#171;&#185; transformants, gap repair <br /> pGE89....................CEN, URA3; DSN3(NSL2); pGE170 gap-repaired               verified EcoRV digest produce 1.2-kb fragment. The gap- <br />                             MluI-SpeI yeast strain YSLP1                       repaired vectors designated pGE81, pGE82, pGE83, pGE84 NSL1, <br /> pGE90....................CEN, URA3; dsn3-29/nsl2-29; pGE170 gap-                  nsl1-6, nsl1-8, nsl1-68, respectively (Table 2). The gap-repaired plasmids <br />                             repaired MluI-SpeI yeast strain YSL29              pGE81 pGE84 transformed nnf1-17 nsl1 strains YSL6, YSL8, <br /> pGE99....................CEN, TRP1; NNF1 1.4-kb XmnI-PvuII insert            YSL68 carrying pGE100 (CEN/NNF1/ADE3/TRP1). Only pGE81 wild- <br /> pGE100..................CEN, TRP1; NNF1 1.4-kb XmnI-PvuII insert             type NSL1 plasmid shuffled pGE100 YSL6, YSL8, YSL68 <br />                             ADE3 3.6-kb EagI-NheI insert                 restore sectoring. <br /> pGE101..................CEN, URA3; NNF1 1.4-kb XmnI-PvuII insert                Cloning allelic rescue NSL2. All synthetic lethal strains <br /> pGE102..................CEN, URA3; ade3-2p                                        carrying pGE100 (CEN/NNF1/ADE3/TRP1) transformed dos- <br /> pGE151..................2&#226;?&#174;m, URA3; 1.4-kb HindIII subclone                   age suppressors SSD1-v, SLG1, DSN1, DSN3 subcloned CEN/URA3 <br />                             YEp24-based library; contains DSN3(NSL2)              plasmid pRS316 ascertain genes multicopy <br /> pGE164..................2&#226;?&#174;m, URA3; 2.7-kb BsaAI subclone YEp24-              suppressor nnf1-17 allele <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>mutation</span> synthetically <br />                             based library; contains DSN1                          lethal combination nnf1-17. Plasmid pGE171 containing DSN3 able <br /> pGE165..................CEN, URA3; 2.7-kb BsaAI subclone YEp24-              complement nonsectoring phenotype mutant YSL29. To distinguish <br />                             based library; contains DSN1                          possibilities DSN3(NSL2) able act high- and <br /> pGE170..................CEN, URA3; 3.1-kb KpnI subclone YEp24-               low-copy suppressor nnf1-17 dsn3(nsl2) mutation synthetically <br />                             based library; contains DSN3(NSL2)                    lethal nnf1-17, plasmid shuffle pGE100 repeated synthetic <br /> pGE171..................CEN, URA3; 1.4-kb HindIII subclone                   lethal strain YSL29 DSN3(NSL2) dsn3-29(nsl2-29) strains YSLP1 <br />                             YEp24-based library; contains DSN3(NSL2)              YSL29 cloned pRS316. To isolate DSN3(NSL2) dsn3-29(nsl2- <br /> pGE180..................CEN, URA3; 2.0-kb PvuII-NruI subclone                29) allele, genomic DNA prepared parent strain YSLP1 the <br />                             pCT3-based library; contains NSL1                     mutant strain YSL29, 1.7-kb region ORFs PCR amplified <br /> pGE190..................2&#226;?&#174;m, URA3; 3.3-kb HindIII subclone                   primers YAL560 (5&#226;&#172;&#732;-CTATCTGGGGTCGTCATGTTGA-3&#226;&#172;&#732;) YAL2220 <br />                             YEp24-based library; contains SLG1                    (5&#226;&#172;&#732;-CTGCTGAACTAGTTCTCGCTCA-3&#226;&#172;&#732;). The 1.7-kb PCR product then <br />                                                                                   restricted 1.3 kb MluI [which cuts 105 bp ATG DSN3 <br />   a <br />     Plasmids study unless indicated.                      (NSL2)] SpeI [which cuts 350 bp DSN3(NSL2) stop codon]. This <br />   b <br />     From C. Holm. <br />   c                                                                               1.3-kb MluI/SpeI fragment used replace 1.3-kb MluI/SpeI se- <br />     From D. Koshland. <br />   d <br />     From K. Tatchell.                                                             quence pGE170, resulting pGE89 [DSN3(NSL2) parent strain) <br />                                                                                   pGE90 (dsn3-29(nsl2-29) UV-mutagenized strain]. Plasmid pGE89 <br />                                                                                   able complement nonsectoring phenotype YSL29, plasmid <br />                                                                                   pGE90 able complement nonsectoring phenotype strain. <br /> contained smallest functional subclones DSN1 (YIR010w 2.7-kb         Therefore, NSL2 identical DSN3. <br /> BsaAI fragment), DSN2 (YOR008c 3.3-kb HindIII fragment), DSN3               Gene disruptions. DSN1, DSN3(NSL2), NSL1 disrupted singly a <br /> (YAL034w-a 1.4-kb HindIII fragment), respectively. The nnf1-17 phenotype     W303 background one-step gene replacement (53). Correct integrations the <br /> was complemented 35&#194;&#176;C distinct plasmids low-copy CEN           DSN1, DSN3(NSL2), NSL1 chromosomal loci verified PCR or <br /> library. Of 58,000 transformants, colony grew 35&#194;&#176;C, colony    Southern blotting. For gene disruption, independent transformants <br /> grew poorly temperature. Sequencing rescued plasmids indicated    single integration intended locus chosen tetrad <br /> their inserts included NNF1 SSD1-v, respectively. Plasmids pW31 pW35,     dissection analysis. For dsn1::HIS3 disruption, resulting heterozygous <br /> containing SSD1/SRK1 (a gift K. Tatchell) (77) transformed       diploid strains GEY170, GEY171, GEY172. For dsn3::LEU2 dis- <br /> nnf1-17 mutant confirm partial suppression effect 35&#194;&#176;C.                 ruption, resulting heterozygous diploid strains GEY210, GEY211, and <br />    Synthetic lethal screen. The nnf1-17 ade2 ade3 strain YSLP1 segregant    GEY212. For nsl1::LEU2 disruption, resulting heterozygous diploid <br /> from cross GEY138 &#226;&#171;&#187; CH1305. For synthetic lethal screen, strain YSLP1     strains GEY190, GEY191, GEY192. <br /> carrying plasmid pGE98 (CEN/NNF1/ADE3/URA3) grown SC-Ura                   Epitope tagging. Six repeats Myc epitope 270-bp DraI/XbaI frag- <br /> 25&#194;&#176;C, sonicated, plated SC low-Ade plates, UV irradiated &#226;&#172;&#402;10%           ment inserted 3&#226;&#172;&#732; end NNF1. The sequence C terminus of <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>survival</span>. The mutagenized cells incubated 30&#194;&#176;C 7 days,       NNF1 translates YWIKA(MEQKLISEEDLNE)6SCSP*. For integration <br /> colony color fully developed. From 102,000 colonies survived UV      selection NNF1 chromosomal locus, LEU2 selectable marker on <br /> mutagenesis, 17 nonsectoring derivatives identified. As integration   2.0-kb XbaI fragment ligated NNF1 stop codon 164 <br /> plasmid mutations chromosomal ade3 locus result         bp 3&#226;&#172;&#732; noncoding region. Yeast strain W961-5A transformed with <br /> nonsectoring phenotype, stably red strains transformed pGE99        NNF1-myc6::LEU2 3.5-kb HindIII fragment, genomic DNA Leu&#226;&#171;&#185; <br /> (CEN/NNF1/TRP1) pRS314 (CEN/TRP1) vector. Six mutants              transformants analyzed Southern blotting. Strain GEY110 has <br /> sectored transformation pGE99, vector,          NNF1-myc6::LEU2 successfully integrated NNF1 locus. Strain GEY111 <br /> selected cloning complementation nonsectoring phenotype 30&#194;&#176;C.    derived GEY110 expresses GFP(S65T)-tagged allele CNM67 <br />    Cloning allelic rescue NSL1. The screening plasmid pGE98 (CEN/          CNM67 chromosomal locus. The CNM67-GFP(S65T)-kanMX6 gene <br /> NNF1/ADE3/URA3) candidate strains synthetic lethal screen     fusion PCR amplified ABY112 genomic DNA primers CNM1940 <br /> was replaced pGE100 (CEN/NNF1/ADE3/TRP1) plasmid shuffling            (5&#226;&#172;&#732;-GAGCTGAGCCGCATTTCTCAG-3&#226;&#172;&#732;) CNM3200 (5&#226;&#172;&#732;-CTTCATAAGC <br /> counterselection 5-fluoroorotic acid (5-FOA) URA3/CEN yeast          GGCTCGAATCACAG-3&#226;&#172;&#732;). The PCR product band isolated trans- <br />  232       EUSKIRCHEN                                                                                                                              EUKARYOT. CELL <br />  <br /> formed strain GEY110. Transformants selected YPD-G418 plates.            GEY111 cells incubated anti-GFP polyclonal antibody only <br /> Correct integration CNM67 locus verified PCR analysis.                  primary antibody FITC- Cy3-conjugated secondary antibod- <br />    GFP(S65T) fused C terminus DSN1 steps. DSN1           ies. The control experiment anti-Myc 9E10 monoclonal antibody <br /> amplified pGE164 using primers YIRSacI (5&#226;&#172;&#732;-GCCTTTGAAAGTTG                     primary antibody FITC- Cy3-conjugated secondary <br /> AGCTCAGCAAGAG-3&#226;&#172;&#732;) YIREcoRV (5&#226;&#172;&#732;-CTTTTATTTCAGATATCCA                                antibodies carried Nnf1p-Myc6 cells strain GEY110 <br /> GTTTTTTACTGA-3&#226;&#172;&#732;). Primer YIRSacI anneals 500 bp ATG                     GFP fluorescence seen amplification the <br /> DSN1. Primer YIREcoRV mutates EcoRV site C terminus DSN1,                anti-GFP antibody. In summary, crossover fluorescence Cy3 and <br /> which EcoRV digestion removes final codon (leucine) stop            FITC detected absence anti-Myc anti-GFP <br /> codon. The resulting 2.3-kb PCR product digested SacI EcoRV,             primary antibodies. <br /> and fragment ligated SacI EcoRV sites pRS313.                   For MT depolymerizations, cultures haploid yeast strains (GEY110, <br /> GFP(S65T) NotI fragment cut pSF-GP1 (a gift J. Hirsch), filled       GEY176, GEY206, GEY216) expressing Nnf1p-Myc6, Dsn1p-GFP, <br /> in Klenow polymerase, ligated EcoRV site create pGE35.           Dsn3p-GFP, Nsl1p-GFP endogenous promoters split and <br />    For GFP(S65T)-tagging NSL1, NSL1 promoter amplified             treated 17 &#226;?&#174;g nocodazole (Sigma Chemical Co.) ml a <br /> PCR pGE180 using primers T7 YPL1640 (5&#226;&#172;&#732;-GTTATTTATTCA                     1.5-mg/ml stock dimethyl sulfoxide (DMSO) mock treated DMSO for <br /> ATCCTCCTGCAGGAAG-3&#226;&#172;&#732;). The NSL1 PCR product digested                          2.5 h 30&#194;&#176;C. Cells fixed prepared indirect immunofluorescence, as <br /> SnaBI PstI cloned EcoRV PstI sites pBS SK(&#226;&#171;&#186;). The             described above, using antitubulin antibodies anti-Myc anti-GFP <br /> resulting plasmid cut SmaI (in multiple cloning site region      antibodies. For nocodazole experiment, cells imaged Leica <br /> PstI), GFP(S65T) 750-bp NotI fragment (filled Klenow polymer-           fluorescence microscope CCD camera (Princeton Instruments, Inc.). <br /> ase) ligated SmaI site. This cloning changes amino acids of <br /> NSL1 EED QP. The entire NSL1-GFP fusion removed pBS <br /> SK(&#226;&#171;&#186;) backbone HindIII-XbaI digest cloned HindIII                                              RESULTS <br /> XbaI sites pRS316, make pGE54. <br />    GFP(S65T) fused C terminus DSN3 PCR amplifying             The nnf1-17 mutant defective nuclear division. NNF1 is <br /> DSN3 promoter ORF pGE170 primers YAL120EcoRI (5&#226;&#172;&#732;-TCC                    essential viability. To determine Nnf1p a <br /> GCGGGAATTCCCAGAGTATGCCGTGATGTTATG-3&#226;&#172;&#732;) YAL1860NotI <br />                                                                                       specific role cell division cycle analyze onset <br /> (5&#226;&#172;&#732;-GTAACGATTGCGGCCGCCTAACACATCATCAAGTAAATCCAATT <br /> GAGG-3&#226;&#172;&#732;). Primer YAL120EcoRI introduces EcoRI site 900 bp               nnf1-17 defect, phenotypes logarithmically growing <br /> DSN3 start codon, primer YAL1860NotI anneals stop DSN3,             nnf1-17 cells examined shift nonpermissive <br /> to remove termination codon add NotI restriction site. The 1,740-bp         temperature 37&#194;&#176;C. In wild-type strains, large buds indic- <br /> PCR product digested EcoRI NotI cloned EcoRI            ative cells G2/M border (reviewed reference 51). <br /> NotI sites pRS316. GFP(S65T) 750-bp NotI fragment ligated <br /> into NotI site, resulting plasmid pGE73. <br />                                                                                       After 3 h 37&#194;&#176;C, large-budded cells comprised than <br />    Plasmid stability assay. The plasmid stability assay based greater inci-     70% nnf1-17 population, compared &#226;&#172;&#402;30% the <br /> dence plasmid loss nonselective conditions affected mutants (40, 41).     NNF1 population (Fig. 1A). The majority nnf1-17 cells <br /> Diploid strains GEY160 GEY165 transformed pCT3/ade3-2p                  uninucleate nucleus positioned bud neck. <br /> (pGE102), single transformants grown overnight 25&#194;&#176;C SC-Ura <br />                                                                                       In small percentage nnf1-17 cells, nuclear division was <br /> medium. The day, starter cultures diluted SC complete medium <br /> at 30&#194;&#176;C &#226;&#172;&#402;4 doublings plating SC low-Ade plates prewarmed             completed mother cell, resulting binucleate cells. To <br /> 30&#194;&#176;C. Colonies scored 5 days 30&#194;&#176;C follows: (i) half-sectored,        estimate DNA content look aneuploidy, flow cytometry <br /> colonies half pink (with red white sectors) half white; (ii) pink,   carried asynchronous cultures NNF1 nnf1-17 <br /> pink colonies pink white sectors, including half-sectored           cells. At time zero (Fig. 1B) homozygous diploid NNF1 and <br /> colonies; (iii) white, colonies solid white sectors; (iv) red, <br /> colonies dark red. To confirm plasmid loss nnf1-17 mutant <br />                                                                                       nnf1-17 cells generated peaks, corresponding 2N 4N <br /> was nnf1-17 mutation alone, nnf1-17 diploid strain GEY165          DNA contents. After 3 h 37&#194;&#176;C, peaks maintained <br /> cotransformed pCT3/ade3-2p (pGE102) pRS314/NNF1 (pGE98)                   wild-type population, nnf1-17 population, <br /> pRS314/empty vector, NNF1 diploid strain GEY160 cotransformed             peaks merge broaden, indicating cells a <br /> with pCT3/ade3-2p pRS314/empty vector. Single transformants grown <br />                                                                                       wide range DNA content. Thus, appears nnf1-17 <br /> overnight 25&#194;&#176;C SC-Ura-Trp medium, diluted SC-Trp medium 30&#194;&#176;C for <br /> &#226;&#172;&#402;4 doublings, plated SC-Trp low-Ade plates prewarmed 30&#194;&#176;C.             mutant fails arrest G2/M border. The nnf1-17 muta- <br /> After 5 days 30&#194;&#176;C, colonies scored described above.                        tion leads mitotic defect, DNA does segregate <br />    Flow cytometry. Cells prepared flow cytometry described previ-         evenly mutant. <br /> ously (27). For sample minimum 25,000 cells analyzed using              MT morphologies large-budded cells examined after <br /> FACS Vantage flow cytometer (Becton Dickinson, San Jose, Calif.). <br />                                                                                       3 h 37&#194;&#176;C (Fig. 1C). The mitotic spindle short most <br />    Fluorescence microscopy. Indirect immunofluorescence tubulin was <br /> carried essentially described previously (50). For localization SPB         (&#226;&#172;&#402;85%) large-budded nnf1-17 cells (columns d f h <br /> antigens, short fixation times used, suggested previously (54). Cells         Fig. 1C). These short spindles misoriented <br /> carrying GFP and/or Myc6 fusions fixed 5 10 min 3.7% formalde-         respect mother-daughter axis (column b Fig. 1C). <br /> hyde 25&#194;&#176;C. Spheroplasts mounted polyethyleneimine-coated slides            In smaller percentage nnf1-17 population, cyto- <br /> and rinsed &#226;&#171;&#186;20&#194;&#176;C methanol 6 min followed &#226;&#171;&#186;20&#194;&#176;C acetone for <br /> 30 s. Primary antibody sources dilutions rat anti-&#226;?&#163;-tubulin monoclonal <br />                                                                                       plasmic MTs affected, elon- <br /> antibody YOL1/34 1:200 (Harlan Sera-lab, Indianapolis, Ind.), mouse anti-          gated (columns d h) missing altogether (columns c and <br /> c-Myc epitope monoclonal 9E10 1:50 (BAbCO, Richmond, Calif.), rabbit           g). The nnf1-17 mutation does cause global defect MT <br /> anti-GFP 1:30 (Clontech, Palo Alto, Calif.). Secondary antibody sources        function, mutant hypersensitive resistant <br /> dilutions fluorescein isothiocyanate (FITC)-conjugated goat anti-rat immu- <br />                                                                                       benomyl displays karyogamy defect (data not <br /> noglobulin G (IgG) 1:200, Cy3-conjugated goat anti-mouse IgG 1:500, and <br /> FITC-conjugated goat anti-rabbit IgG 1:200. All secondary antibodies          shown). Therefore, Nnf1p required mitotic spindle elon- <br /> purchased Jackson Immunoresearch, West Grove, Pa. Cells photo-              gation orientation. <br /> graphed Kodak T-MAX p3200 film ASA 1600.                                         nnf1-17 cells display plasmid instability. If mitotic spindle <br />    Three controls lacking primary antibody used make sure      function primary nnf1-17 defect, a <br /> not crossover fluorescence FITC Cy3 fluorophores. First, <br /> only faint GFP fluorescence seen cells strain GEY111 were <br />                                                                                       partial spindle defect nnf1-17 cells semipermissive <br /> incubated FITC- Cy3-conjugated secondary antibodies              growth conditions. A centromere-plasmid stability assay was <br /> absence primary antibodies. Second, FITC signal detected            used detect mitotic segregation defects (40, 41). NNF1 ade2 <br />  VOL. 1, 2002                                                        NEW CHROMOSOME SEGREGATION PROTEINS IN YEAST                             233 <br />  <br />  <br />                                                                            growth conditions. These observations support hypothesis <br />                                                                            Nnf1p required mitotic spindle function. <br />                                                                               Isolation characterization suppressors nnf1-17. <br />                                                                            Dosage suppressors nnf1-17 isolated selection for <br />                                                                            growth minimum restrictive temperature (35&#194;&#176;C). In ad- <br />                                                                            dition NNF1 itself, genes identified: <br />                                                                            SSD1(SRK1), SLG1(WSC1), newly identified genes <br />                                                                            DSN1 DSN3 (dosage suppressor NNF1). Only NNF1 in <br />                                                                            CEN 2&#226;?&#174;m vector able restore wild-type <br />                                                                            growth nnf1-17 strain 37&#194;&#176;C. High-copy DSN1, SLG1, <br />                                                                            DSN3 allowed growth 35&#194;&#176;C various degrees (Fig. 3). <br />  <br />  <br />  <br />  <br />    FIG. 1. nnf1-17 cells mitotic defect. (A) Nuclear distribution <br /> (percent occurrence) nnf1-17 cells. Wild-type (GEY145) nnf1-17 <br /> (GEY146) cells shifted 25 37&#194;&#176;C time zero. After 3 h at <br /> 37&#194;&#176;C, cells (n &#226;&#172;&#381; 200 time point) fixed stained with <br /> DAPI. Anucleate cells observed, cell <br /> lysis difficulty detection. (B) The DNA content logarithmically <br /> growing NNF1 (GEY145) nnf1-17 (GEY146) cells analyzed by <br /> flow cytometry. (C) MT morphologies large-budded nnf1-17 cells <br /> (percent occurrence) 3 h 37&#194;&#176;C. Wild-type (GEY122) nnf1-17 <br /> (GEY138) cells shifted 25 37&#194;&#176;C time zero. MTs were <br /> visualized indirect immunofluorescence antitubulin antibody. <br /> Cells scored follows: a, short spindle inserted bud <br /> neck; b, short, misoriented spindle; c, short spindle missing cyto- <br /> plasmic MTs; d, short spindle elongated cytoplasmic MTs; e, <br /> extended spindle divided nucleus; f, anaphase occurring the <br /> mother cell; g. anaphase occurring mother cell missing <br /> cytoplasmic MTs; h. anaphase occurring mother cell elon- <br /> gated cytoplasmic MTs; i, binucleated mother cell. <br />                                                                               FIG. 2. nnf1-17 cells unstable transmission centromeric <br />                                                                            plasmid. Assays performed described Materials Meth- <br />                                                                            ods. (A) The nnf1-17 mutant (GEY165) rapidly loses centromeric <br /> ade3 nnf1-17 ade2 ade3 strains carrying CEN/ade3-2p                ade3-2p plasmid (pGE102) grown nonselective conditions. <br /> plasmid pGE102 grown nonselective conditions                 The arrow points half-sectored colony. A wild-type strain <br /> 30&#194;&#176;C (Fig. 2A B). The rate plasmid loss deter-               (GEY160) carrying pGE102 shown comparison. (B) Percentages <br /> mined number colonies half pink (one                  colony type scored &#226;&#172;&#381;800 colonies following <br /> copy pGE102) half white (zero copies pGE102)                 strains carrying pGE102: NNF1, GEY160; nnf1-17, GEY165; NNF1 &#226;&#171;&#185; <br />                                                                            vector, GEY160 &#226;&#171;&#185; pRS314; nnf1-17 &#226;&#171;&#185; CEN/NNF1, GEY165 &#226;&#171;&#185; <br /> to plasmid loss mitotic division plating.            pGE99; nnf1-17 &#226;&#171;&#185; vector, GEY165 &#226;&#171;&#185; pRS314. Small colonies <br /> For NNF1 cells, CEN plasmid loss rate 1.8% &#226;&#171;&#190; 0.3%                  &#226;&#172;&#402;1 mm diameter irregular margins be <br /> per generation; nnf1-17 cells, rate 17.0% &#226;&#171;&#190; 1.8%               scored half-sectoring. (C) The frequency half-sectored colonies <br /> (see Materials Methods) (Fig. 2C). The presence                  calculated transformant number half-sectored <br />                                                                            colonies divided total number colonies copy of <br /> NNF1 plasmid effect wild-type plasmid stability              pCEN/ade3-2p (pGE102) plating. Errors standard deviations <br /> complemented nnf1-17 defect. Therefore, nnf1-17 mu-                means transformants. Approximately 200 <br /> tation causes CEN plasmid instability semipermissive                 colonies scored transformant. <br />  234     EUSKIRCHEN                                                                                                     EUKARYOT. CELL <br />  <br />  <br />                                                                       wild-type strain S288C, single or <br />                                                                       multiple copies partially suppress growth defects associated <br />                                                                       diverse mutations (13, 14, 38, 63). In study nnf1-17 <br />                                                                       present W303 background, wild-type strain <br />                                                                       W303 recessive ssd1-d allele. <br />                                                                          Identification mutations lethal nnf1-17 <br />                                                                       strain. A synthetic lethal screen used identify gene <br />                                                                       products act conjunction Nnf1p. These clones <br />                                                                       called NSL genes (NNF1 synthetic lethal). An ade2 ade3 <br />                                                                       colony sectoring assay (1, 42) carried 30&#194;&#176;C. A total <br />                                                                       102,000 mutagenized colonies strain YSLP1 (ade2 ade3 <br />                                                                       nnf1-17) carrying ADE3/NNF1 plasmid screened to <br />                                                                       isolates stable red, nonsectoring phenotype. <br />                                                                       Four isolates (YSL6, YSL8, YSL29, YSL68) chosen <br />                                                                       cloning complementation nonsectoring pheno- <br />                                                                       type. <br />                                                                          Cloning NSL1. Plasmids carrying YPL233w small <br />                                                                       flanking regions complemented nsl1 mutants YSL6, YSL8, <br />                                                                       YSL68. To verify mutants YSL6, YSL8, YSL68 carry <br />                                                                       nsl1 mutations, alleles nsl1-6, nsl1-8, nsl1-68, NSL1 <br />                                                                       isolated gap repair YPL233w strains YSL6, <br />                                                                       YSL8, YSL68, YSLP1, respectively. Colony sectoring as- <br />                                                                       says indicated retrieved NSL1 allele complemented <br />                                                                       mutants YSL6, -8, -68, retrieved nsl1-6, -8, <br />                                                                       -68 did (Fig. 4). Therefore, YPL233w NSL1, it <br />    FIG. 3. Growth nnf1-17 mutant various suppressors at <br /> 30, 35, 37&#194;&#176;C. The nnf1-17 strain GEY138 transformed Ura&#226;&#171;&#185; <br />                                                                       encodes novel protein 216 amino acids predicted <br /> at 25&#194;&#176;C pGE101 (CEN/NNF1), pRS316 (empty vector), pGE164         molecular mass 25 kDa predicted pI 4.7. No sig- <br /> (2&#226;?&#174;m/DSN1), pGE190 (2&#226;?&#174;m/SLG1), pGE151 (2&#226;?&#174;m/DSN3). The              nificant sequence similarity Nsl1p pro- <br /> transformed strains streaked minimal SC-Ura plates   teins current databases. <br /> and incubated 2 days indicated temperatures. <br />  <br />  <br />  <br />    The DSN1 protein 576 amino acids predicted <br /> molecular mass 66 kDa predicted pI 5.2. A match <br /> to consensus bipartite nuclear localization sequence found <br /> at residues 414 430; potential EF-hand calcium binding <br /> site residues 398 410. Expression DSN1 peaks <br /> during G2 (62). Dsn1p significant overall similarity to <br /> any known protein available databases. <br />    The DSN3 protein 289 amino acids predicted <br /> molecular mass 33 kDa predicted pI 5.1. DSN3 was <br /> found equivalent MTW1, gene identified S. cer- <br /> evisiae homology Schizosaccharomyces <br /> pombe gene mis12 (17, 18). In budding yeast, Mtw1p-GFP is <br /> closely situated near SPBs seen fluorescence micros- <br /> copy, Mtw1p coimmunoprecipitates centromere DNA, <br /> and temperature sensitive mtw1-1 mutant exhibits unequal <br /> chromosome segregation (18). Recently Mtw1p reported <br /> to present highly enriched preparation spindle poles, <br /> as detected matrix-assisted laser desorption-ionization mass <br /> spectrometric analysis (76). <br />    SLG1 SSD1 nonessential genes. Slg1p                 FIG. 4. Complementation strain YSL68 NSL1 strain <br /> plasma membrane protein Pkc1p pathway             YSL29 DSN3(MTW1). The synthetic lethal mutant YSL68 carrying <br />                                                                       plasmid pGE100 (CEN/TRP1/ADE3/NNF1) transformed ei- <br /> (19, 31, 44, 73). Overexpression SLG1 maintain viability       ther pGE84 (CEN/URA3/nsl1-68) pGE81 (CEN/URA3/NSL1). The <br /> in nnf1-17 background heat shock response             synthetic lethal mutant YSL29 carrying pGE100 transformed with <br /> and cell integrity signaling effects cell       pGE90 (CEN/URA3/dsn3-29) pGE89 (CEN/URA3/DSN3). <br /> cycle SPB duplication (29, 35, 65). Ssd1p cytoplasmic        Transformants selected SC-Ura low-Ade plates 30&#194;&#176;C 5 <br />                                                                       days. Wild-type NSL1 DSN3(MTW1) restore sectoring pheno- <br /> protein control RNA metabolism affecting RNA             type strains YSL68 YSL29, respectively. Comparable results <br /> stability (72). Various wild-type yeast laboratory strains       obtained plasmid pGE82 strain YSL6 plasmid pGE83 <br /> polymorphisms SSD1. The dominant SSD1-v allele, present            strain YSL8. <br />  VOL. 1, 2002                                                               NEW CHROMOSOME SEGREGATION PROTEINS IN YEAST                                         235 <br />  <br />  <br />                                                        TABLE 3. Tetrad data DSN1 disruption <br />                                                                                                                                                dsn1::HIS3 &#226;&#171;&#185; pCEN/ <br />                                                                                % Spore              DSN1              DSN1/dsn1::HIS3 <br />                          Strain(s) (genotype)                                                                                                   URA3/DSN1 (His&#226;&#171;&#185; <br />                                                                                viability        (His&#226;&#171;&#186; spores)          (His&#226;&#171;&#185; spores)a <br />                                                                                                                                                   Ura&#226;&#171;&#185; spores)b <br />  <br /> CY6 (DSN1/DSN1)                                                                   95                 19                     NAc                         NA <br />  <br /> GEY170, GEY171, GEY172 (dsn1::HIS3/DSN1)                                          28                 19                       3                         NA <br />  <br /> GEY170, GEY171, GEY172 &#226;&#171;&#185; pGE165 (dsn1::HIS3/DSN1 &#226;&#171;&#185;                                43                 17                       6                          6 <br />  pCEN/URA3/DSN1) <br />   a <br />     All His&#226;&#171;&#185; spores nonmating wild-type copy DSN1 detected Southern analysis, described Materials Methods. <br />   b <br />     All His&#226;&#171;&#185; Ura&#226;&#171;&#185; spores able lose pCEN/URA3/DSN1 5-FOA plates. <br />   c <br />     NA, applicable. <br />  <br />  <br />  <br />    The UV mutagenesis produced nsl1 mutations                               single integration DSN1, DSN3, NSL1 locus were <br /> conferring distinct growth defect. However, growth de-                            chosen tetrad analysis. Dissection dsn1::HIS3, <br /> fects observed nsl1-6, nsl1-8, nsl1-68 alleles                       dsn3::LEU2, nsl1::LEU2 heterozygotes resulted poor <br /> were placed nsl1::LEU2 mutant strain. Hence, nsl1-6,                              spore viability (28 40%) number His&#226;&#171;&#185; Leu&#226;&#171;&#185; <br /> nsl1-8, nsl1-68 probably represent partial loss                          progeny (Tables 3 5). Southern analysis showed all <br /> Nsl1p function.                                                                        His&#226;&#171;&#185; spores dsn1::HIS3 heterozygote main- <br />    Mutations DSN3(MTW1) synthetically lethal com-                            tained wild-type copy DSN1, spores not <br /> bination nnf1-17. I reasoned genes                         able mate MATa MAT&#226;?&#163; tester strains (Table 3). <br /> isolated nnf1-17 synthetic lethality dosage sup-                           Therefore, His&#226;&#171;&#185; spores diploids disomes. <br /> pression. Therefore low-copy plasmids carrying dosage                             Similar results obtained DSN3 NSL1 disrup- <br /> suppressor tested complementation YSL mu-                              tions (Tables 4 5). Low spore viability disome or <br /> tants. DSN3(MTW1) complemented nonsectoring pheno-                                 diploid progeny appear mutant phenotypes, as <br /> type mutant YSL29. <br />                                                                                        sporulation dissection CY6 GEY160 parent <br />    To establish mutation DSN3 synthetically lethal <br />                                                                                        strains resulted 95 93% viability, respectively. None of <br /> in combination nnf1-17 DSN3 just a <br />                                                                                        spores nonmater. By inference, His&#226;&#171;&#185; Leu&#226;&#171;&#185; <br /> low-copy suppressor mutations YSL29, chromosomal <br />                                                                                        spores recovered dsn1::HIS3, dsn3::LEU2, nsl1:: <br /> dsn3-29 DSN3 recovered mutant strain <br />                                                                                        LEU2 heterozygotes result nuclear chromosome <br /> YSL29 parent strain YSLP1, respectively, plas- <br /> mid shuffle experiment similar performed                           missegregation. <br /> nsl1 isolates carried (see Materials Methods).                           The failure recover haploid dsn1::HIS3, dsn3::LEU2, or <br /> DSN3 (in plasmid pGE89) fully complemented nonsector-                              nsl1::LEU2 spores suggested DSN1, DSN3, NSL1 are <br /> ing phenotype YSL29 mutant, dsn3-29 (in plasmid                             essential spore germination viability. To verify <br /> pGE90) did (Fig. 4). Therefore, YAL034w-a displays                            conclusion, dsn1::HIS3, dsn3::LEU2, nsl1::LEU2 <br /> synthetic lethality dosage suppression nnf1-17. The                           heterozygous diploids carrying CEN/URA3 plasmids with <br /> dsn3-29 allele does confer growth defect, tem-                          DSN1, DSN3, NSL1, respectively, tested abil- <br /> perature sensitivity observed dsn3::LEU2 strain car-                          ity produce meiotic segregants lacking plasmid (Tables <br /> rying dsn3-29 CEN plasmid source Dsn3p.                            3 5). All His&#226;&#171;&#185; Leu&#226;&#171;&#185; spores unable to <br />    DSN1, DSN3(MTW1), NSL1 essential genes. Wild-                               grow 5-FOA medium (which selects Ura&#226;&#171;&#185; thus <br /> type diploid strains (CY6 GEY160) transformed                             plasmid loss) maintained wild-type copy the <br /> dsn1::HIS3, dsn3::LEU2, nsl1::LEU2 constructs,                              gene targeted disruption, detected PCR analysis. To <br /> each disruption, independent transformants showed                           conclude, DSN1, DSN3, NSL1 essential genes. <br />  <br />  <br />                                                        TABLE 4. Tetrad data DSN3 disruption <br />                                                                                                                                               dsn3::LEU2 &#226;&#171;&#185; pCEN/ <br />                                                                                % Spore             DSN3              DSN3/dsn3::LEU2 <br />                          Strain(s) (genotype)                                                                                                  URA3/DSN3 (Leu&#226;&#171;&#185; <br />                                                                                viability       (Leu&#226;&#171;&#186; spores)          (Leu&#226;&#171;&#185; spores)a <br />                                                                                                                                                  Ura&#226;&#171;&#185; spores)b <br />  <br /> GEY160 (DSN3/DSN3)                                                                93                 41                    NAc                         NA <br />  <br /> GEY210, GEY211, GEY212 (dsn3::LEU2/DSN3)                                          40                 35                      4                         NA <br />  <br /> GEY210, GEY211, GEY212 &#226;&#171;&#185; pGE171 (dsn3::LEU2/DSN3 &#226;&#171;&#185;                                60                 43                      5                          19 <br />  pCEN/URA3/DSN3) <br />  a <br />    Six Leu&#226;&#171;&#185; spores nonmating. All Leu&#226;&#171;&#185; spores wild-type copy DSN3 detected PCR, described in <br /> Materials Methods. <br />  b <br />    All 19 Leu&#226;&#171;&#185; Ura&#226;&#171;&#185; spores able lose pCEN/URA3/DSN3 5-FOA plates. <br />  c <br />    NA, applicable. <br />  236      EUSKIRCHEN                                                                                                                        EUKARYOT. CELL <br />  <br />  <br />                                                      TABLE 5. Tetrad data NSL1 disruption <br />                                                                                                                                          nsl1::LEU2 &#226;&#171;&#185; pCEN/ <br />                                                                            % Spore              NSL1             NSL1/nsl1::LEU2 <br />                         Strain(s) (genotype)                                                                                              URA3/NSL1 (Leu&#226;&#171;&#185; <br />                                                                            viability        (Leu&#226;&#171;&#186; spores)         (Leu&#226;&#171;&#185; spores)a <br />                                                                                                                                              Ura&#226;&#171;&#185; spores)b <br />  <br /> GEY160 (NSL1/NSL1)                                                            93                 41                    NAc                       NA <br />  <br /> GEY190, GEY191, GEY192 (nsl1::LEU2/NSL1)                                      30                 18                     5                        NA <br />  <br /> GEY190, GEY191, GEY192 &#226;&#171;&#185; pGE180 (nsl1::LEU2/NSL1 &#226;&#171;&#185;                            32                 27                     0                          8 <br />  pCEN/URA3/NSL1) <br />  a <br />    Three Leu&#226;&#171;&#185; spores nonmating. All Leu&#226;&#171;&#185; spores wild-type copy NSL1 detected PCR, described in <br /> Materials Methods. <br />  b <br />    All Leu&#226;&#171;&#185; Ura&#226;&#171;&#185; spores able lose pCEN/URA3/NSL1 5-FOA plates. <br />  c <br />    NA, applicable. <br />  <br />  <br />  <br />    Nnf1p, Mtw1p, Dsn1p. Nsl1p colocalize                         kinetochore protein Mtw1p. However, resolution, it <br /> near spindle poles. Epitope-tagged Nnf1p-Myc6 expressed                        determined Nnf1p, Dsn1p, Nsl1p are <br /> from NNF1 locus (strain GEY110) used subcellular                       present SPBs kinetochores, Mtw1p been <br /> localization studies. Indirect immunofluorescence showed                      shown accumulate near SPBs (18, 76). Labeling cy- <br /> Nnf1p-Myc6 present nuclear periphery single dot                     toplasmic MTs mitotic spindle observed for <br /> in unbudded cells side-by-side dots small-budded                         Nnf1p, Dsn1p, Mtw1p, Nsl1p. <br /> cells. In large-budded cells, Nnf1p appears opposite ends of <br /> the dividing nucleus. This localization <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> typical of <br /> spindle pole body proteins (see, e.g., reference 75) certain <br /> kinetochore proteins transiently cluster spindle <br /> poles various stages mitosis (10, 16, 32, 33, 45, 75, <br /> 76). <br />    Double-<span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> immunofluorescence Nnf1p-Myc6 an <br /> SPB marker, GFP-tagged Cnm67p (4, 75), undertaken to <br /> verify spindle pole proximity Nnf1p (strain GEY111) (Fig. <br /> 5). Several controls indicated absence crossover fluores- <br /> cence GFP Myc detection systems (see Ma- <br /> terials Methods). In cells Nnf1p-Myc6 and <br /> Cnm67p-GFP visible, fluorescent dots closely <br /> positioned (in 54 54 cells scored), implying Nnf1p <br /> resides near spindle poles. This Nnf1p-Myc6 staining pat- <br /> tern visible cell cycle, MT staining <br /> was observed. <br />    Fusions GFP(S65T) C-terminal ends of <br /> DSN1, MTW1, NSL1 constructed allow visualiza- <br /> tion Dsn1p, Mtw1p, Nsl1p living fixed cells. <br /> All fusions expressed control endogenous <br /> promoters CEN plasmids able rescue the <br /> respective disruptions plasmid shuffle experiments. The <br /> same localization patterns characteristic SPBs or <br /> some kinetochore proteins seen Dsn1p-GFP, Mtw1p- <br /> GFP, Nsl1p-GFP. Single cells showed fluorescent dot <br /> which seen nuclear cells grown the <br />                                                                                       FIG. 5. Localization Nnf1p, Mtw1p, Dsn1p, Nsl1p the <br /> presence DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phenylindole). Two flu-                        region spindle poles. First row, localization Nnf1p the <br /> orescent dots seen budding cells. The dots adja-                      SPB component Cnm67p strain GEY111. For colocalization of <br /> cent small-budded cells separated large-budded                      Nnf1p Mtw1p, Dsn1p, Nsl1p, strain GEY110 trans- <br /> cells. For colocalization Nnf1p, Dsn1p-GFP, Mtw1p-GFP,                        formed pGE74, pGE36, pGE55, respectively. NNF1-myc6 and <br /> and Nsl1p-GFP transformed separately strain                          CNM67-GFP integrated place wild-type NNF1 CNM67 in <br />                                                                                    haploid strain expressed endogenous promoters. <br /> GEY110, stably expresses Nnf1p-Myc6 NNF1                            The MTW1-GFP, DSN1-GFP, NSL1-GFP fusions expressed <br /> locus. Double-label immunofluorescence performed                          promoters CEN plasmids. For indirect double- <br /> monoclonal anti-c-Myc polyclonal anti-GFP antibodies.                          label immunofluorescence, cells short fixed formaldehyde <br /> Each Dsn1p-GFP, Mtw1p-GFP, Nsl1p-GFP signals                            methanol acetone. The Myc6 epitope detected <br />                                                                                    monoclonal antibody 9E10 Cy3-conjugated goat anti- <br /> was colocalize signal obtained Nnf1p-                        mouse secondary antibody. GFP fluorescence enhanced a <br /> Myc6 (Fig. 5). By single-plane immunofluorescence, Nnf1p                           rabbit anti-GFP antibody FITC-labeled goat anti-rabbit sec- <br /> localizes SPB protein Cnm67p known                           ondary antibody. Bar, 5 &#226;?&#174;m. <br />  VOL. 1, 2002                                                       NEW CHROMOSOME SEGREGATION PROTEINS IN YEAST                         237 <br />  <br />  <br />                                                                                                    DISCUSSION <br />  <br />                                                                              We previously identified Nnf1p protein associated with <br />                                                                           nuclear envelope S. cerevisiae. In study, I that <br />                                                                           Nnf1p needed genetic stability. I performed genetic <br />                                                                           screens identified novel proteins genetic inter- <br />                                                                           actions NNF1. MTW1, gene encoding kinetochore <br />                                                                           protein, isolated screens. Mutations MTW1 <br />                                                                           enhance temperature-sensitive nnf1-17 allele, overex- <br />                                                                           pression MTW1 suppresses nnf1-17 phenotype. Use of <br />                                                                           temperature-sensitive nnf1-17 mutant enabled to <br />                                                                           propose Nnf1p necessary chromosome segregation. <br />                                                                           Loss Nnf1p function leads abnormal MTs, accumula- <br />                                                                           tion large-budded cells, altered DNA content, de- <br />                                                                           creased plasmid stability. Moreover, genetically related <br />                                                                           proteins Mtw1p, Dsn1p, Nsl1p share subcellular <br />                                                                           localization Nnf1p. Although Nnf1p, Mtw1p, Nsl1p are <br />                                                                           similar size, significant sequence similarities found <br />                                                                           proteins. These proteins must <br />                                                                           unique functions, essential cell viabil- <br />                                                                           ity. <br />                                                                              Previously (56), described additional phenotypes ob- <br />                                                                           served nnf1-17 mutant. First, small percentage of <br />    FIG. 6. The localization Nnf1p, Dsn1p, Mtw1p, Nsl1p does        nnf1-17 cells (&#226;&#172;&#402;15%) slight nuclear accumulation of <br /> not depend MTs. Cultures haploid yeast expressing NNF1-      poly(A)&#226;&#171;&#185; RNA 3 h 35&#194;&#176;C. This nuclear accumulation of <br /> myc6 (strain GEY110), DSN1-GFP (strain GEY176), MTW1-GFP                  poly(A)&#226;&#171;&#185; RNA uncertain significance simply be <br /> (strain GEY216), NSL1-GFP (strain GEY206) split with <br /> treated 17 &#226;?&#174;g nocodazole ml mock treated DMSO <br />                                                                           terminal phenotype late onset low <br /> for 2.5 h. Cells fixed prepared indirect immunofluores-      penetrance. Second, nnf1-17 cells changes <br /> cence using antitubulin antibodies (not shown) anti-Myc      nucleolus detected immunofluorescence a <br /> anti-GFP antibodies. Since MT arrays lost nocodazole-treated      nucleolar antigen. The changes nucleolar morphology may <br /> cells, Nnf1p, Dsn1p, Mtw1p, Nsl1p localizations independent       related phenomenon nucleolus wild-type <br /> of MTs (n &#226;&#172;&#381; 90 strain). Each fusion expressed its <br /> endogenous promoter, localizations dependent un-          cells opposite SPBs (80). Conceivably, the <br /> tagged versions proteins tagged copies present   loss crescent-shaped nucleolus chromosome <br /> each strain. Bar, 5 &#226;?&#174;m.                                                   loss, nnf1-17 cells wide range DNA content and <br />                                                                           nucleolar structure determined ribosomal <br />                                                                           DNA. The genetic screens reported did uncover any <br />                                                                           links nucleolus mRNA export. <br />    To determine Nnf1p-Myc6, Dsn1p-GFP,                           MT defects prominent cytological defect seen <br /> Mtw1p-GFP, Nsl1p-GFP localizations main-                     nnf1-17 cells, pleiotropic MT defects observed in <br /> tained absence MTs, cultures haploid yeast strains           large-budded nnf1-17 cells nonpermissive temperature. <br /> expressing fusion proteins treated nocodazole             The range MT defects observed nnf1-17 mutant in <br /> to depolymerize MTs mock treated DMSO. After                 line morphologies seen mutants lacking spindle <br /> depolymerization, antitubulin immunofluorescence revealed                 function. In nnf1-17 mutant, cytoplasmic spindle <br /> that MT arrays lost 93% cells (data              MTs aberrant, cell. How- <br /> not shown), indicative effective nocodazole treatment (30).            ever, particular MT pattern diagnostic the <br /> As shown (Fig. 6), characteristic dot staining patterns          role protein specific cellular substructure, due <br /> still present Nnf1p-Myc6, Dsn1p-GFP, Mtw1p-GFP,                   delicate stoichiometry components cy- <br /> and Nsl1p-GFP proteins. Nocodazole treatment did vary                 toskeleton. For example, mutations SPB outer plaque <br /> the number cells showing Nnf1p-Myc6, Dsn1p-GFP, Mtw1p-                 protein Spc72p result mainly loss cytoplasmic MTs and <br /> GFP, Nsl1p-GFP staining (&#226;&#172;&#381;85% nocodazole                 binucleate cells, spc72 mutants exhibit spindle <br /> DMSO treatments), staining intensities comparable                defects (11, 39, 60). After shifting synchronized nnf1-17 pop- <br /> in treated untreated cultures. In large-budded cells           ulation restrictive temperature (56), short, mitotic <br /> single nucleus, percentage cells visible dot         spindles nuclei bud necks observed early time <br /> was higher nocodazole-treated Nnf1p-Myc6, Dsn1p-GFP,                   points (1 h shift). Binucleate cells cells with <br /> Mtw1p-GFP, Nsl1p-GFP cultures (&#226;&#172;&#402;40 50%; n &#226;&#172;&#381; 90)                    elongated cytoplasmic MTs observed later time <br /> than untreated cultures (&#226;&#172;&#402;3%; n &#226;&#172;&#381; 100). This increase              points (3 h shift). Therefore, possibility that <br /> in single-dot staining pattern reflect duplicated          spindle elongation primary MT defect nnf1-17 cells, <br /> but unseparated SPBs sister chromatids (15, 30, 66). In                nuclear migration defect occurring subsequently to <br /> short, Nnf1p, Dsn1p, Mtw1p, Nsl1p staining did di-                perturbations nuclear envelope tubulin pools. <br /> minish nocodazole treatment spindle collapse.                       MTs essential mitotic chromosome segregation, and <br />  238    EUSKIRCHEN                                                                                                     EUKARYOT. CELL <br />  <br />  <br /> so mutants MT defects higher frequencies <br /> of chromosome plasmid loss. To test this, nnf1-17 cells were <br /> assayed plasmid stability lose marked <br /> plasmid 10-fold-higher frequency wild-type strain. <br /> Simple loss MT defects or, possibly, <br /> errors DNA replication repair (26). If plasmid loss <br /> phenotype accompanied increase mitotic recombi- <br /> nation, imply mutant DNA lesions and <br /> that wild-type protein role DNA metabolism. <br /> Plasmid loss accompanied nondisjunction sup- <br /> port role MT cytoskeleton, DNA replication <br /> defects prevent sister chromatid separation (25, 48). <br /> Neither recombination nondisjunction tested in <br /> nnf1-17 cells attempt distinguish possi- <br /> bilities. However, flow cytometry analysis DNA content in <br /> nnf1-17 cells restrictive temperature colocalization <br /> of Nnf1p Mtw1p strongly suggest defect the <br /> nnf1-17 mutant MT spindle function. <br />    Mtw1p recently characterized separate study, <br />                                                                      FIG. 7. Summary interactions NNF1, DSN1, MTW1, and <br /> and cytological behavior mtw1-1 mutant consistent     NSL1. DSN1 MTW1(DSN3) isolated high-copy suppres- <br /> with defects observed nnf1-17 cells. The mtw1-1 mutant      sors nnf1-17 growth defect 35&#194;&#176;C. Mutations NSL1 or <br /> exhibits unstable transmission CEN plasmid semi-         MTW1(NSL2) synthetically lethal nnf1-17. Finally, Dsn1p and <br /> restrictive temperature, shown colony color assay        Nsl1p interact two-hybrid (G. Euskirchen, unpublished <br />                                                                    data). <br /> (18). Synchronous cultures mtw1-1 cells accumulate as <br /> large-budded cells short spindles early time points, <br /> while later time points, binucleate cells seen (18), <br /> as similarly observed synchronous population           Dsn1p Nsl1p interact two-hybrid <br /> nnf1-17 cells (56). Goshima Yanagida (18)      (G. Euskirchen, unpublished results). Although the <br /> in unbudded cells, Mtw1p-GFP seen single dot       mechanism Dsn1p Nsl1p interaction remains be <br /> nuclear periphery, Mtw1p-GFP dots seen            determined, two-hybrid result establishes link be- <br /> near SPBs budding cells. Mtw1p-GFP fluorescence         tween genes cloned suppressor synthetic <br /> lost ndc10-1 mutant, Ndc10p Mtw1p colo-         lethal analysis starting nnf1-17 mutant (Fig. 7). The <br /> calization studies, proteins share kineto-      disomy diploidization occurs DSN1, <br /> chore localization Ndc10p additionally      MTW1, NSL1 disrupted implies proteins are <br /> mitotic spindle (16, 18). The combined use Mtw1p-GFP,           required precise stoichiometry chromosome segrega- <br /> Tub4p-GFP (&#226;?&#165;-tubulin), GFP-tagged chromosomes              tion suggests loci haploinsufficient. <br /> facilitated observations yeast sister chromatids sepa-    Interestingly, haploinsufficient phenotypes ob- <br /> rate early cell cycle situated near SPBs    served. In yeast, loss diploid dosage SPB duplication <br /> that sister arms remain connected spindle elongation         gene, NDC1, results aneuploidy (12). Chromosome disor- <br /> (18). Further support phenomenon come He          ders heterozygous dsn1, mtw1, nsl1 diploids consis- <br /> et al. (22), used Mtw1p-GFP kinetochore              tent plasmid loss phenotype observed nnf1-17 <br /> marker sister chromatids undergo transient sep-       mutant strong spindle defects observed nnf1-17 <br /> arations metaphase centromeric chromatin       cells. The direct explanation proteins <br /> an elastic quality.                                                needed maintain chromosome number act together <br />    A variety different relationships Nnf1p, Dsn1p,        spindle function. <br /> Mtw1p, Nsl1p envisioned. In straightforward              An extension model Nnf1p complex is <br /> model, Nnf1p, Dsn1p, Mtw1p, Nsl1p exist subcom-           peripherally associated SPB. This model derives from <br /> plex. In model, DSN1 MTW1 interaction suppres-        biochemical identification Nnf1p nuclear enve- <br /> sors nnf1-17. An excess Dsn1p Mtw1p non-          lope preparation (56); localization patterns Nnf1p, <br /> permissive temperature prevent degradation nnf1-17          Cnm67p, Dsn1p, Mtw1p, Nsl1p; copurification of <br /> protein sequestering unstable monomer. Similarly, in-       Mtw1p highly enriched spindle pole preparation (76); <br /> creases concentration Dsn1p Mtw1p stabilize       genetic interactions NNF1, MTW1, DSN1, and <br /> a structure formed nnf1-17 protein maintaining     NSL1 (Fig. 7). In S. cerevisiae, comparable patterns proximal <br /> an assembled complex mass action. At lower temper-         SPB localization seen kinetochore proteins <br /> atures, Nnf1p subcomplex functional          Ndc10p (16), Cse4p (45), Ctf19p (28), Ndc80p (75, 76), and <br /> nnf1-17 protein, partial loss function Mtw1p   Slk19p (81). Cytological observations centromeres tran- <br /> Nsl1p destabilize Nnf1p assembly cells           siently cluster near SPBs support hypothesis <br /> would inviable.                                                 (21, 22, 33, 67). A kinetochore association for <br />    Thus, future question addressed Nnf1p,       Spc24p Spc25p (32, 75), proteins originally iden- <br /> Dsn1p, Mtw1p, Nsl1p present subcomplex. I        tified SPB components, biochemically immu- <br />  VOL. 1, 2002                                                                  NEW CHROMOSOME SEGREGATION PROTEINS IN YEAST                                       239 <br />  <br />  <br /> noelectron microscopy (76). The essential proteins identified                        18. Goshima, G., M. Yanagida. 2000. Establishing biorientation occurs with <br />                                                                                          precocious separation sister kinetochores, arms, the <br /> in study positioned link data various                              early spindle budding yeast. Cell 100:619&#226;&#8364;&#8220;633. <br /> chromosome segregation studies.                                                      19. Gray, J. V., J. P. Ogas, Y. Kamada, M. Stone, D. E. Levin, I. Herskowitz. <br />                                                                                          1997. A role Pkc1 MAP kinase pathway Saccharomyces cerevisiae <br />                           ACKNOWLEDGMENTS                                                bud emergence identification putative upstream regulator. <br />                                                                                          EMBO J. 16:4924&#226;&#8364;&#8220;4937. <br />   Foremost, I grateful Teri M&#195;&#169;l&#195;&#168;se Aaron Mitchell                 20. Guacci, V., E. Hogan, D. Koshland. 1994. Chromosome condensation <br /> their generous support critical review course                      sister chromatid pairing budding yeast. J. Cell Biol. 125:517&#226;&#8364;&#8220;530. <br /> this work. The CNM67-GFP fusion gift Philippsen                       21. Guacci, V., E. Hogan, D. Koshland. 1997. Centromere position bud- <br /> laboratory. Other strains plasmids freely given Marion                       ding yeast: evidence anaphase A. Mol. Biol. Cell 8:957&#226;&#8364;&#8220;972. <br /> Carlson, Craig Thompson, Rodney Rothstein, Kelly Tatchell, Connie                    22. He, X., S. Asthana, P. K. Sorger. 2000. Transient sister chromatid <br /> Holm, Elaine Yeh, Jeanne Hirsch, Alex Tzagoloff, Charles di-                         separation elastic deformation chromosomes mitosis bud- <br />                                                                                          ding yeast. Cell 101:763&#226;&#8364;&#8220;775. <br /> Como. I thank Rocco Carbone help flow cytometric analysis.                  23. He, X., D. R. Rines, C. W. Espelin, P. K. Sorger. 2001. Molecular <br /> Thanks Mike Rout, Elaine Yeh, Mike Snyder, Florian Sch&#195;&#164;rer,                        analysis kinetochore-microtubule attachment budding yeast. Cell 106: <br /> Catherine Yan, Xiaoyin Shan, Zhixiong Xue, Anuj Kumar, Mike                              195&#226;&#8364;&#8220;206. <br /> Smith, Jackie Vogel, members Mitchell laboratory                      24. Hofmann, C., I. M. Cheeseman, B. L. Goode, K. L. McDonald, G. Barnes, <br /> providing suggestions discussions.                                                   D. G. Drubin. 1998. Saccharomyces cerevisiae Duo1p Dam1p, novel <br />   This work supported grant National Institutes                        proteins involved mitotic spindle function. J. Cell Biol. 143:1029&#226;&#8364;&#8220;1040. <br /> of Health (GM44901) Teri M&#195;&#169;l&#195;&#168;se. Flow cytometry carried                       25. Hoyt, M. A., T. Stearns, D. Botstein. 1990. Chromosome instability <br /> out support Yale Cancer Center Flow Cytometry                              mutants Saccharomyces cerevisiae defective microtubule-medi- <br /> Shared Resource, Public Health Service grant CA-16359                           ated processes. Mol. Cell. Biol. 10:223&#226;&#8364;&#8220;234. <br />                                                                                      26. Huffaker, T. C., M. A. Hoyt, D. Botstein. 1987. Genetic analysis the <br /> National Cancer Institute. I supported GAANN fellowships <br />                                                                                          yeast cytoskeleton. Annu. Rev. Genet. 21:259&#226;&#8364;&#8220;284. <br /> (P200A50013-97 P200A80111-99) Department Edu-                        27. Hutter, K. J., H. E. Eipel. 1979. Microbial determinations flow <br /> cation.                                                                                  cytometry. J. Gen. Microbiol. 113:369&#226;&#8364;&#8220;375. <br />                                                                                      28. Hyland, K. M., J. Kingsbury, D. Koshland, P. Hieter. 1999. Ctf19p: a <br />                                  REFERENCES <br />                                                                                          novel kinetochore protein Saccharomyces cerevisiae potential link <br />  1. Bender, A., J. R. Pringle. 1991. Use screen synthetic lethal        kinetochore mitotic spindle. J. Cell Biol. 145:15&#226;&#8364;&#8220;28. <br />     multicopy suppressee mutants identify new genes involved mor-          29. Ivanovska, I., M. D. Rose. 2000. SLG1 plays role G1 the <br />     phogenesis Saccharomyces cerevisiae. Mol. Cell. Biol. 11:1295&#226;&#8364;&#8220;1305.               decision enter exit cell cycle. Mol. Gen. Genet. 262:1147&#226;&#8364;&#8220;1156. <br />  2. Biggins, S., F. F. Severin, N. Bhalla, I. Sassoon, A. A. Hyman, A. W.        30. Jacobs, C. W., A. E. M. Adams, P. J. Szanislo, J. R. Pringle. 1988. <br />     Murray. 1999. The conserved protein kinase Ipl1 regulates microtubule                Functions microtubules Saccharomyces cerevisiae cell cycle. J. Cell <br />     binding kinetochores budding yeast. Genes Dev. 13:532&#226;&#8364;&#8220;544.                     Biol. 107:1409&#226;&#8364;&#8220;1426. <br />  3. Boeke, J. D., J. Trueheart, G. Natsoulis, G. R. Fink. 1987. 5-Fluoroorotic   31. Jacoby, J. J., S. M. Nilius, J. J. Heinisch. 1998. A screen upstream <br />     acid selective agent yeast molecular genetics. Methods Enzymol.              components yeast protein kinase C signal transduction pathway iden- <br />     154:164&#226;&#8364;&#8220;175.                                                                         tifies product SLG1 gene. Mol. Gen. Genet. 258:148&#226;&#8364;&#8220;155. <br />  4. Brachat, A., J. V. Kilmartin, A. Wach, P. Philippsen. 1998. Saccharo-        32. Janke, C., J. Ortiz, J. Lechner, A. Shevchenko, A. Shevchenko, M. M. <br />     myces cerevisiae cells defective spindle pole body outer plaques accom-         Magiera, C. Schramm, E. Schiebel. 2001. The budding yeast proteins <br />     plish nuclear migration half-bridge-organized microtubules. Mol. Biol.           Spc24p Spc25p interact Ndc80p Nuf2p kinetochore and <br />     Cell 9:977&#226;&#8364;&#8220;991.                                                                      important kinetochore clustering checkpoint control. EMBO J. <br />  5. Byers, B., L. Goetsch. 1974. Duplication spindle plaques integra-         20:777&#226;&#8364;&#8220;791. <br />     tion yeast cell cycle. Cold Spring Harbor Symp. Quant. Biol. 38:123&#226;&#8364;&#8220;      33. Jin, Q., J. Fuchs, J. Loidl. 2000. Centromere clustering major <br />     131.                                                                                 determinant yeast interphase nuclear organization. J. Cell Sci. 113:1903&#226;&#8364;&#8220; <br />  6. Byers, B., L. Goetsch. 1975. Behavior spindles spindle plaques         1912. <br />     cell cycle conjugation Saccharomyces cerevisiae. J. Bacteriol. 124: <br />                                                                                      34. Jones, M. H., J. B. Bachant, A. R. Castillo, T. H. Giddings, M. Winey. <br />     511&#226;&#8364;&#8220;523. <br />                                                                                          1999. Yeast Dam1p required maintain spindle integrity mitosis <br />  7. Carlson, M., D. Botstein. 1982. Two differentially regulated mRNAs <br />                                                                                          interacts Mps1p kinase. Mol. Biol. Cell 10:2377&#226;&#8364;&#8220;2391. <br />     different 5&#226;&#172;&#732; ends encode secreted intracellular forms yeast <br />                                                                                      35. Khalfan, W., I. Ivanovska, M. D. Rose. 2000. Functional interaction <br />     invertase. Cell 28:145&#226;&#8364;&#8220;154. <br />                                                                                          PKC1 pathway CDC31 network SPB duplication genes. <br />  8. Carminati, J. L., T. Stearns. 1997. Microtubules orient mitotic <br />                                                                                          Genetics 155:1543&#226;&#8364;&#8220;1559. <br />     spindle yeast dynein-dependent interactions cell cortex. <br />                                                                                      36. Kilmartin, J. V., P. Y. Goh. 1996. Spc110p: assembly properties role <br />     J. Cell Biol. 138:629&#226;&#8364;&#8220;641. <br />                                                                                          connection nuclear microtubules yeast spindle pole body. <br />  9. Chan, C. S. M., D. Botstein. 1993. Isolation characterization of <br />                                                                                          EMBO J. 15:4592&#226;&#8364;&#8220;4602. <br />     chromosome-gain increase-in-ploidy mutants yeast. Genetics 135: <br />     677&#226;&#8364;&#8220;691.                                                                         37. Kim, J., J. Kang, C. S. M. Chan. 1999. Sli15 associates Ipl1 <br /> 10. Cheeseman. I. M., M. Enquist-Newman, T. M&#195;&#188;ller-Reichert, D. G. Drubin,               protein kinase promote proper chromosome segregation Saccharomy- <br />     G. Barnes. 2001. Mitotic spindle integrity kinetochore function              ces cerevisiae. J. Cell Biol. 145:1381&#226;&#8364;&#8220;1394. <br />     linked Duo1p/Dam1p complex. J. Cell Biol. 15:197&#226;&#8364;&#8220;212.                     38. Kim, Y. J., L. Francisco, G. C. Chen, E. Marcotte, C. S. Chan. 1994. <br /> 11. Chen, X. P., H. Yi, T. C. Huffaker. 1998. The yeast spindle pole body            Control cellular morphogenesis Ip12/Bem2 GTPase-activating pro- <br />     component Spc72p interacts Stu2p required proper microtu-            tein: possible role protein phosphorylation. J. Cell Biol. 127:1381&#226;&#8364;&#8220;1394. <br />     bule assembly. J. Cell Biol. 141:1169&#226;&#8364;&#8220;1179.                                      39. Knop, M., E. Schiebel. 1998. Receptors determine cellular localiza- <br /> 12. Chial, H. J., T. H. Giddings, E. A. Siewert, M. A. Hoyt, M. Winey. 1999.         tion &#226;?&#165;-tubulin complex site microtubule formation. <br />     Altered dosage Saccharomyces cerevisiae spindle pole body duplication         EMBO J. 17:3952&#226;&#8364;&#8220;3967. <br />     gene, NDC1, leads aneuploidy polyploidy. Proc. Natl. Acad. Sci. USA       40. Koshland, D., P. Hieter. 1987. Visual assay chromosome ploidy. <br />     96:10200&#226;&#8364;&#8220;10205.                                                                      Methods Enzymol. 155:351&#226;&#8364;&#8220;372. <br /> 13. Chiannilkulchai, N., A. Moenne, A. Sentenac, C. Mann. 1992. Biochem-         41. Koshland, D., J. C. Kent, L. H. Hartwell. 1985. Genetic analysis the <br />     ical genetic dissection Saccharomyces cerevisiae RNA polymerase           mitotic transmission minichromosomes. Cell 40:393&#226;&#8364;&#8220;403. <br />     C53 subunit analysis mitochondrially mis-sorted mutant          42. Kranz, J. E., C. Holm. 1990. Cloning function: alternative ap- <br />     construct. J. Biol. Chem. 267:23099&#226;&#8364;&#8220;23107.                                           proach identifying yeast homologs genes organisms. Proc. <br /> 14. Evans, D. R., M. J. Stark. 1997. Mutations Saccharomyces cerevi-          Natl. Acad. Sci. USA 87:6629&#226;&#8364;&#8220;6633. <br />     siae type 2A protein phosphatase catalytic subunit reveal roles cell wall     43. Leung, D. W., E. Chen, D. V. Goeddel. 1989. A method random <br />     integrity, actin cytoskeleton organization mitosis. Genetics 145:227&#226;&#8364;&#8220;241.        mutagenesis defined DNA segment using modified polymerase chain <br /> 15. Fraschini, R., E. Formenti, G. Lucchini, S. Piatti. 1999. Budding yeast          reaction. Technique 1:11&#226;&#8364;&#8220;15. <br />     Bub2 localized spindle pole bodies activates mitotic checkpoint    44. Lodder, A. L., T. K. Lee, R. Ballester. 1999. Characterization the <br />     different pathway Mad2. J. Cell Biol. 145:979&#226;&#8364;&#8220;991.                        Wsc1 protein, putative receptor stress response Saccharomyces <br /> 16. Goh., P. Y., J. V. Kilmartin. 1993. NDC10: gene involved chromo-            cerevisiae. Genetics 152:1487&#226;&#8364;&#8220;1499. <br />     segregation Saccharomyces cerevisiae. J. Cell Biol. 121:503&#226;&#8364;&#8220;512.         45. Meluh, P. B., P. Yang, L. Glowczewski, D. Koshland, M. M. Smith. 1998. <br /> 17. Goshima, G., S. Saitoh, M. Yanagida. 1999. Proper metaphase spindle              Cse4p component core centromere Saccharomyces cerevisiae. <br />     <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> determined centromere proteins Mis12 Mis6 required              Cell 94:607&#226;&#8364;&#8220;613. <br />     faithful chromosome segregation. Genes Dev. 13:1664&#226;&#8364;&#8220;1677.                        46. Michaelis, C., R. Ciosk, K. Nasmyth. 1997. Cohesins: chromosomal <br />  240         EUSKIRCHEN                                                                                                                               EUKARYOT. CELL <br />  <br />       proteins prevent premature separation sister chromatids. Cell 91:35&#226;&#8364;&#8220;      65. Stirling, D. A., M. J. R. Stark. 2000. Mutations SPC110, encoding the <br />       45.                                                                                   yeast spindle pole body calmodulin binding protein, cause defects cell <br /> 47.   Muhlrad, D., R. Hunter, R. Parker. 1992. A rapid method localized             integrity spindle formation. Biochim. Biophys. Acta 1499:85&#226;&#8364;&#8220;100. <br />       mutagenesis yeast genes. Yeast 8:79&#226;&#8364;&#8220;82.                                        66. Straight, A. F., A. S. Belmont, C. C. Robinett, A. W. Murray. 1996. GFP <br /> 48.   Palmer, R. E., E. Hogan, D. Koshland. 1990. Mitotic transmission               tagging budding yeast chromosomes reveals protein-protein interac- <br />       artificial chromosomes cdc mutants yeast Saccharomyces cerevisiae.          tions mediate sister chromatid cohesion. Curr. Biol. 6:1599&#226;&#8364;&#8220;1608. <br />       Genetics 125:763&#226;&#8364;&#8220;774.                                                             67. Straight, A. F., W. F. Marshall, J. W. Sedat, A. W. Murray. 1997. Mitosis <br /> 49.   Pearson, C. G., P. S. Maddox, E. D. Salmon, K. Bloom. 2001. Budding               living budding yeast: anaphase A metaphase plate. Science 277: <br />       yeast chromosome structure dynamics mitosis. J. Cell Biol. 152:            574&#226;&#8364;&#8220;578. <br />       1255&#226;&#8364;&#8220;1266.                                                                        68. Sundberg, H. A., L. Goetsch, B. Byers, T. N. Davis. 1996. Role of <br /> 50.   Pringle, J. R., A. E. Adams, D. G. Drubin, B. K. Haarer. 1991. Immu-              calmodulin Spc110p interaction proper assembly spindle pole <br />       nofluorescence methods yeast. Methods Enzymol. 194:565&#226;&#8364;&#8220;602.                       body components. J. Cell Biol. 133:111&#226;&#8364;&#8220;124. <br /> 51.   Pringle, J. R., L. H. Hartwell. 1981. The Saccharomyces cerevisiae cell       69. Sundberg, H. A., T. N. Davis. 1997. A mutational analysis identifies three <br />       cycle, p. 97&#226;&#8364;&#8220;142. In N. Strathern, E. W. Jones, J. R. Broach (ed.), The           functional regions spindle pole component Spc110p Saccharomyces <br />       molecular biology yeast Saccharomyces, vol. 1. Cold Spring Harbor              cerevisiae. Mol. Biol. Cell 8:2575&#226;&#8364;&#8220;2590. <br />       Laboratory Press, Cold Spring Harbor, N.Y.                                        70. Thomas, J. H., D. Botstein. 1986. A gene required separation of <br /> 52.   Rose, M. D., F. Winston, P. Hieter. 1990. Methods yeast genetics.              chromosomes spindle apparatus yeast. Cell 44:65&#226;&#8364;&#8220;76. <br />       Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.                     71. Thompson, C. M., A. J. Koleske, D. M. Chao, R. A. Young. 1993. A <br /> 53.   Rothstein, R. J. 1983. One-step gene disruption yeast. Methods Enzymol.            multisubunit complex associated RNA polymerase II CTD and <br />       101:203&#226;&#8364;&#8220;211.                                                                          TATA-binding protein yeast. Cell 73:1361&#226;&#8364;&#8220;1375. <br /> 54.   Rout, M. P., J. V. Kilmartin. 1990. Components yeast spindle and <br />                                                                                         72. Uesono, Y., A. Toh-e, Y. Kikuchi. 1997. Ssd1p Saccharomyces cerevi- <br />       spindle pole body. J. Cell Biol. 111:1913&#226;&#8364;&#8220;1927. <br />                                                                                             siae associates RNA. J. Biol. Chem. 272:16103&#226;&#8364;&#8220;16109. <br /> 55.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />                                                                                         73. Verna, J., A. Lodder, K. Lee, A. Vagts, R. Ballester. 1997. A family of <br />       laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Har- <br />                                                                                             genes required maintenance cell wall integrity stress <br />       bor, N.Y. <br />                                                                                             response Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA 94:13804&#226;&#8364;&#8220; <br /> 56.   Shan, X., Z. Xue, G. Euskirchen, T. M&#195;&#169;l&#195;&#168;se. 1997. NNF1 essential <br />                                                                                             13809. <br />       yeast gene required proper spindle orientation, nucleolar nuclear <br />       envelope structure mRNA export. J. Cell Sci. 110:1615&#226;&#8364;&#8220;1624.                   74. Wach, A., A. Brachat, C. Rebischung, S. Steiner, K. Pokorni, S. te Heesen, <br /> 57.   Shaw, S. L., E. Yeah, P. Maddox, E. D. Salmon, K. Bloom. 1997. Astral             P. Philippsen. 1998. PCR-based gene targeting Saccharomyces cer- <br />       microtubule dynamics yeast: microtubule-based searching mechanism                evisiae. Methods Microbiol. 26:67&#226;&#8364;&#8220;81. <br />       spindle orientation nuclear migration bud. J. Cell Biol.         75. Wigge, P. A., O. N. Jensen, S. Holme, S. Soues, M. Mann, J. V. Kilmar- <br />       139:985&#226;&#8364;&#8220;994.                                                                          tin. 1998. Analysis Saccharomyces spindle pole matrix-assisted laser <br /> 58.   Sherman, F., J. B. Hicks, G. R. Fink. 1986. Methods yeast genetics.            desorption/ionization (MALDI) mass spectrometry. J. Cell Biol. 141:967&#226;&#8364;&#8220; <br />       Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.                         977. <br /> 59.   Sikorski, R., P. Hieter. 1989. A shuttle vectors yeast host     76. Wigge, P. A., J. V. Kilmartin. 2001. The Ndc80p complex Saccha- <br />       strains designed efficient manipulation DNA Saccharomyces cerevi-           romyces cerevisiae contains conserved centromere components a <br />       siae. Genetics 122:19&#226;&#8364;&#8220;27.                                                             function chromosome segregation. J. Cell Biol. 152:349&#226;&#8364;&#8220;360. <br /> 60.   Soues, S., I. R. Adams. 1998. SPC72: spindle pole component required        77. Wilson, R. B., A. A. Brenner, T. B. White, M. J. Engler, J. P. Gaughran, and <br />       spindle orientation yeast Saccharomyces cerevisiae. J. Cell Sci.           K. Tatchell. 1991. The Saccharomyces cerevisiae SRK1 gene, suppressor of <br />       111:2809&#226;&#8364;&#8220;2818.                                                                        bcy1 ins1, involved protein phosphatase function. Mol. Cell. <br /> 61.   Spang, A., K. Grein, E. Schiebel. 1996. The spacer protein Spc110p                Biol. 11:3369&#226;&#8364;&#8220;3373. <br />       targets calmodulin central plaque yeast spindle pole body.          78. Winey, M., L. Goetsch, P. Baum, B. Byers. 1991. MPS1 MPS2: novel <br />       J. Cell Sci. 109:2229&#226;&#8364;&#8220;2237.                                                           yeast genes defining distinct steps yeast spindle pole body duplication. <br /> 62.   Spellman, P. T., G. Sherlock, M. Q. Zhang, V. R. Iyer, K. Anders, M. B.               J. Cell Biol. 114:745&#226;&#8364;&#8220;754. <br />       Eisen, P. O. Brown, D. Botstein, B. Futcher. 1998. Comprehensive              79. Winey, M., M. A. Hoyt, C. Chan, L. Goetsch, D. Botstein, B. Byers. 1993. <br />       identification cell cycle-regulated genes yeast Saccharomyces cer-          NDC1: nuclear periphery component required yeast spindle pole body <br />       evisiae microarray hybridization. Mol. Biol. Cell 9:3273&#226;&#8364;&#8220;3297.                     duplication. J. Cell Biol. 122:743&#226;&#8364;&#8220;751. <br /> 63.   Stettler, S., N. Chiannilkulchai, S. Hermann-Le Denmat, D. Lalo, F. La-           80. Yang, C. H., E. J. Lambie, J. Hardin, J. Craft, M. Snyder. 1989. Higher <br />   <br /> </body></html>