<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, June 1998, p. 3260&#226;&#8364;&#8220;3264                                                                                Vol. 180, No. 12 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology <br />  <br />  <br />  <br />                          Characterization glnK-amtB Operon of <br />                                      Azotobacter vinelandii <br />                   DIETMAR MELETZUS,&#226;&#8364;&#160; PAUL RUDNICK, NATALIE DOETSCH, ANDREW GREEN, <br />                                        AND CHRISTINA KENNEDY* <br />  <br />                 Department Plant Pathology, College Agriculture, University Arizona, Tucson, Arizona 85721 <br />                                               Received 20 January 1998/Accepted 7 April 1998 <br />  <br />              <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> determine Azotobacter vinelandii PII protein influences regulation nif gene expression <br />           response fluxes ammonium supply, gene encoding PII isolated characterized. Its <br />           deduced translation product highly similar PII proteins organisms, greatest degree <br />           relatedness exhibited Escherichia coli glnK gene product. A gene designated amtB found <br />           downstream cotranscribed glnK E. coli. The AmtB protein similar functionally <br />           characterized ammonium transport proteins eukaryotes prokaryote. glnK and <br />           amtB comprise operon. Attempts isolate stable glnK mutant strain unsuccessful, suggesting that <br />           glnK, like glnA, essential gene A. vinelandii. amtB mutants isolated, growth on <br />           limiting amounts ammonium similar mutant wild-type strains, mutants unable to <br />           transport [14C]methylammonium. <br />  <br />  <br />    Nitrogen fixation genes (nif) highly conserved               homotrimer consisting identical 12.4-kDa subunits (9), is <br /> nitrogen-fixing bacteria, diazotrophic species              uridylylated Tyr residue uridylytransferase activity <br /> class Proteobacteria examined, transcriptional activator                  glnD gene product (32). The uridylylation state PII <br /> NifA required expression nif genes (33). In              determines transcriptional activator NtrC phos- <br /> diazotrophs b subgroups class Proteobac-                  phorylated dephosphorylated (i.e., active inactive, respec- <br /> teria, Azospirillum brasilense, Rhodobacter capsulatus,               tively) (2, 29) extent glutamine synthetase <br /> and Herbaspirillum seropedicae, NifA proteins inactive                (GS) adenylylated deadenylylated (i.e., inactive active, <br /> in cells grown high levels ammonium (1, 4, 31). The                   respectively) (20). The recent finding second PII-encoding <br /> mechanism(s) unknown involve PII pro-                   genes, i.e., glnK (present addition glnB Escherichia coli <br /> tein, known studies enteric bacteria major                    [38]) glnZ (present addition glnB Azospirillum <br /> component nitrogen-sensing signal transduction cas-                  brasilense [10]), complicates picture, especially respect <br /> cade. In diazotrophs proteobacterial g subgroup, includ-               occurs low-fixed-N conditions. <br /> ing Azotobacter vinelandii Klebsiella pneumoniae, nifL                     In nonenteric bacterium A. vinelandii, glnD gene, <br /> gene lies upstream nifA (3, 5, 18, 26). NifL interacts            originally named nfrX, identified Tn5 mutagenesis (34) <br /> prevents activity NifA cells exposed oxygen                   subsequent DNA sequence analysis (8). While glnD::Tn5 <br /> excess fixed N. Inactivation NifA prevents stimulat-               mutants Nif2, glnD::Tn5 nifL::KIXX double mutants are <br /> ing transcription promoters adjacent                Nif1. Two models developed explain result fol- <br /> nif genes operons, leading failure produce nitroge-               lows: (i) GlnD required conversion active NifL (which <br /> nase enzyme. K. pneumoniae NifA deleted N-terminal                     inhibits NifA) inactive form, (ii) GlnD necessary <br /> domain sensitive inactivation NifL, suggesting                  conversion NifA conformation inac- <br /> that function domain modulate response                 tivated NifL. The question GlnD influences NifL <br /> NifA NifL (12). NifL A. vinelandii recently shown                (or NifA) PII protein basis study. <br /> be redox-sensitive flavoprotein flavin adenine dinucle-                   Isolation sequencing A. vinelandii glnK amtB <br /> otide prosthetic group, reduced,                     genes. The glnB gene E. coli pAH5 (19) hybridized a <br /> effect vitro NifA-dependent open-complex formation                   3.2-kb EcoRI fragment A. vinelandii genomic DNA on <br /> the nifH promoter (17). When oxidized, NifL prevents open-                    Southern analysis (data shown). This fragment cloned <br /> complex formation. How NifL responds fixed-N status                     pACYC184 (6), giving pND183, subsequently into <br /> becoming inhibitory NifA known. It possible              pSVB30 (25), giving pDM508. The glnB-hybridizing region was <br /> that susceptibility NifA NifL inactivation increases             delineated hybridization. The nucleotide sequence <br /> the fixed-N supply increases.                                                 2.3-kb PstI-EcoRI fragment pPR101 (Fig. 1) was <br />    Transduction environmental signal fixed-nitrogen                 determined strands using exonuclease III- and <br /> status best described enteric organisms (for                    nuclease S1-generated deletion derivatives. Analysis the <br /> review, reference 27). Under low-ammonium-concentra-                      2,278-bp sequence revealed potential open reading frames <br /> tion conditions, product glnB gene, PIIB protein,              (ORFs), nucleotide positions 477 (ATG) and <br />                                                                               813(TGA) nucleotides 849 (ATG) 2,157 <br />                                                                               (TGA). These ORFs potentially encode proteins molec- <br />   * Corresponding author. Mailing address: Department Plant Pa- <br /> thology, College Agriculture, P.O. Box 210036, University Ari- <br />                                                                               ular weights 12,240 46,390 Da, respectively. The product <br /> zona, Tucson, AZ 85721. Phone: (520) 621-9835. Fax: (520) 621-9290.           ORF showed high degree overall amino acid <br /> E-mail: ckennedy@u.arizona.edu.                                               sequence similarity PII proteins includes the <br />   &#226;&#8364;&#160; Present address: Gentechnologie/Mikrobiologie, Fakulta&#194;&#168;t fu <br />                                                               &#194;&#168;r Bi-          conserved site uridylylation Tyr51. The similar the <br /> ologie, Universita&#194;&#168;t Bielefeld, 33501 Bielefeld, Germany.                     proteins second PII protein E. coli, designated <br />  <br />                                                                        3260 <br />  VOL. 180, 1998                                                                                                                                 NOTES        3261 <br />  <br />  <br />  <br />  <br />                                                                                         FIG. 2. Northern blot analysis glnK-amtB operon. mRNA am- <br />                                                                                      monium-grown (lanes a, c, e) N2-grown (lanes b, d, f) cultures was <br />                                                                                      prepared using Qiagen (Chatsworth, Calif.) RNeasey Kit (catalog no. <br />                                                                                      74904). Samples containing approximately equal amounts total RNA were <br />                                                                                      separated 0.8% formaldehyde&#226;&#8364;&#8220;agarose gels electrophoresis; fol- <br />                                                                                      lowed capillary blotting RNA nylon membranes. Blots were <br />    FIG. 1. The 2.3-kb glnK-amtB region A. vinelandii. The locations KIXX       hybridized 32P-labeled probes glnK-amtB region: 360-bp <br /> and lacZ interposon constructs directions insertion shown    HincII fragment (glnK only), 1.2-kb StuI fragment (glnK plus amtB), a <br /> hatched rectangles arrows line, respectively. The resulting plas-      0.9-kb StuI fragment (amtB only). L, DNA fragment standards. <br /> mids named (where appropriate) stable mutant strain A. <br /> vinelandii glnK::KIXX glnK::lacZ region replaced wild-type <br /> glnK gene. The fragments used probes identify RNA fragments the <br /> Northern blots (see Fig. 2) shown lines. Abbreviations restriction <br /> sites: B, BamHI; E, EcoRI; H, HincII; Sn, StuI; SnaBI; P, PstI.                      ammonium. Mutations amt gene C. glutamicum trans- <br />                                                                                      ported 20-fold [14C]methylammonium did amt1 <br />                                                                                      parent strain. Thus, possible nrgA-like gene <br /> GlnK; proteins 75% identical 84% similar                        A. vinelandii located downstream glnK encode a <br /> respect amino acid sequence (data shown). E. coli GlnB                        membrane-spanning protein involved ammonium transport <br /> is similar, 66% identical amino acids. High degrees                     (see below). This gene named amtB distinguish it <br /> similarity glnB gene products members                        amtA, unrelated gene initially identified in- <br /> the class Proteobacteria product A. vinelandii                  volved ammonium transport E. coli (14) thought <br /> ORF described evident, identities ranging                         encode CysQ (28). <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> approximately 70 74%. Because high degree                             glnK amtB gene expression. Potential ribosome binding <br /> similarity ORF product E. coli glnK gene product                      sites glnK amtB located 10 bp upstream of <br /> and proximity PII protein-encoding gene puta-                       ATG initiation codons. The small intergenic region be- <br /> tive ammonium transporter organisms (see below),                         tween glnK amtB lack obvious tran- <br /> A. vinelandii gene designated glnK.                                         scriptional terminator- promoter-like structures suggested <br />    The second product second A. vinelandii ORF exhibits                       probable cotranscription glnK amtB. Sequences <br /> the highest degree overall similarity NrgA protein                      similar 210- 235-like regions prokaryotic <br /> Bacillus subtilis (39), 41% identical 56% similar                           genes recognized s70 present 36 66 bp upstream <br /> amino acids. While specific function NrgA                            ATG glnK. Whether sequences significant <br /> determined organism, nrgA gene lies adjacent                      transcription putative glnK-amtB operon not <br /> upstream glnB-like gene named nrgB, contrast                          investigated. About 75 bp downstream amtB begins a <br /> arrangement identified A. vinelandii, glnK                      27-bp region features characteristic factor-indepen- <br /> upstream nrgA-like gene. During course work,                      dent transcription terminator, including 6-<span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span> inverted-re- <br /> five genes encoding NrgA-like proteins described                          peat motif separated 3 bp followed tract T&#226;&#8364;&#8482;s. <br /> for eukaryotes prokaryotes, prod-                           Northern blotting experiments carried deter- <br /> ucts similar A. vinelandii ORF described                     fixed N regulates expression glnK and <br /> here (with 25 38% identity). These proteins signifi-                       amtB genes genes cotranscribed. <br /> cant hydrophobic core, 10 12 transmembrane-spanning                          RNA prepared ammonium- N2-grown A. vine- <br /> domains flanked hydrophilic regions, include products                         landii cultures hybridized extremely weakly reproducibly to <br /> of mep1, mep2, mep3 genes Saccharomyces cer-                            different [32P]dCTP-labeled probes: 360-bp HincII <br /> evisiae (22, 23), amt1 gene Arabidopsis thaliana (30),                    fragment (glnK only), 1.2-kb StuI fragment (glnK amtB), <br /> amt gene Corynebacterium glutamicum, amtB gene                            0.9-kb StuI fragment (amtB only) (Fig. 1). In cases, <br /> of E. coli (35, 38). Of organisms, showing                      hybridizing RNA band corresponded transcript of <br /> definite growth <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span> mep1 mep2 double mutant                           approximately 1.7 kb, size expected glnK amtB <br /> S. cerevisiae unable grow media low levels                       genes cotranscribed (Fig. 2). The intensity labeling of <br /> of exogenous ammonium (13). While mutants                            band slightly N2-grown cultures for <br /> the amt1 gene Arabidopsis thaliana available comparison,                      NH41-grown cultures (Fig. 2). To determine the <br /> the cloned gene organism able complement                            glnK-amtB mRNA present unusually small <br /> the S. cerevisiae mutant growth media low levels                      compared transcripts, Northern blots <br />  3262     NOTES                                                                                                          J. BACTERIOL. <br />  <br />  <br /> were hybridized glnA gene probe. The intensity         construct glnA insertion mutants A. vinelandii (36). <br /> hybridizing band observed 1.5 kb, size glnA tran-      From results previous work concluded that <br /> script previously observed (36) autoradiography,           glnA null mutations lethal GS ammo- <br /> about 100-fold greater glnK-amtB band (data         nium assimilation-associated enzyme present glutamine <br /> not shown). glnK-amtB expression measured            transported A. vinelandii. It appears null <br /> amount b-galactosidase produced amtB-lacZ fusion            mutations glnK tolerated lethal. <br /> strain, MV566, constructed transformation UW136           Similar results obtained attempts construct glnD:: <br /> pPR203 (Fig. 1) grown different conditions. The levels         KIXX <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>null mutants</span> (7). (While original glnD [orignally <br /> of expression consistently low, 75 Miller units      named nfrX] mutants viable Nif2, sites Tn5 <br /> of activity, significantly different ammonium-,      insertion far 39 end gene, leaving doubt to <br /> urea-, N2-grown cultures.                                         represented null mutants [see references 8 and <br />    These experiments glnK amtB genes           36]). Since PII-UMP required enteric bacteria rapid <br /> cotranscribed operon levels expression       deadenylylation GS, possible reason severe growth <br /> are low significantly influenced fixed-N         impairment A. vinelandii glnK (and glnD) mutants is <br /> supply. In contrast, glnK-amtB operon E. coli tran-        GS insufficiently active (i.e., remains adenylylated even <br /> scribed NtrC-activated promoter         low-fixed-N conditions). To test possibility, MV75, a <br /> expressed cells grown high levels ammonium (38).          strain GS adenylylated carrying glnA gene <br /> The glnB genes organisms, including Rhizo-          Tyr407 mutated Phe, (15), trans- <br /> bium etli, Rhodobacter capsulatus, Rhodospirillum rubrum,        formed glnK::KIXX plasmids pDM513 pDM514 <br /> are dependent NtrC expression (14a, 21a, 31a). So        individually. While altered GS state did allow glnD <br /> far, A. vinelandii glnK, E. coli glnB, Azospirillum brasilense   mutations stable complete gene replacement <br /> glnZ known non-fixed-N-regulated PII-encoding           occurred (7), glnK::KIXX MV75 transformants behaved <br /> genes members class Proteobacteria (10, 37a).       like wild-type transformants wild-type chromosomes <br /> The absence s54 NtrC recognition motifs A.        present subcultures antibiotic-con- <br /> vinelandii glnK promoter region consistent results       taining medium resistance quickly lost colonies <br /> obtained expression experiments.                            plated antibiotic-free medium. Therefore, lethality <br />    Whether glnK gene A. vinelandii reported rep-         impairment growth A. vinelandii caused intro- <br /> resents PII-encoding gene organism uncer-        duction glnK mutations possibly effect only <br /> tain. There evidence second func-        GS activity vital cellular function. As <br /> tional PII protein encoded gene; single            indicated above, unlikely second PII-encoding <br /> hybridizing band observed hybridization      gene A. vinelandii, E. coli, Azospirillum <br /> E. coli glnB gene A. vinelandii glnK gene, characterized      brasilense, certain bacteria. It relevance here <br /> here, genomic digests generated different restric-     glnB glnZ double mutants Azospirillum brasilense are <br /> tion enzymes. In experiment involving cloning PII-         severely growth impaired (11), E. coli glnB glnK mutants <br /> encoding genes ligation PCR products generated            (16a), glnB mutations appear lethal in <br /> oligonucleotide primers based conserved amino acid se-            Synecocchus spp. Rhodospirillum rubrum standard <br /> quences GlnK GlnB PII proteins, 10           genetic techniques failed result replacement wild- <br /> products cloned DNA sequence identical          type gene mutated copy (16, 21a). <br /> glnK gene described (24). Also, discussed below, glnB           amtB mutants unable transport [14C]methylammo- <br /> mutants isolated variety growth con-         nium. To construct amtB interposon mutations, pDM508 was <br /> ditions, including glnB-like gene          partially digested StuI ligated KIXX cassette <br /> might expected expressed.                                   blunt-ended SmaI, giving pPR201 and <br />    Attempts construct glnK mutants A. vinelandii. The          pPR202 (Fig. 1). Wild-type strain UW136 MV101 were <br /> kanamycin resistance-encoding KIXX cassette inserted          transformed plasmids, followed se- <br /> or endonuclease restriction sites pPR101 (Fig. 1).        lection kanamycin-containing medium. Complete replace- <br /> A. vinelandii UW136 transformed resulting         ment amtB gene confirmed Southern analysis for <br /> glnK::KIXX plasmids, pDM513, pDM514, pPR401. The                 KIXX insertion transformants strain (data not <br /> kanamycin-resistant, ampicillin-sensitive transformants,       shown); stably kanamycin resistant several <br /> presumably carried desired gene replacements          subcultures nonselective medium. Thus, unlike glnK, the <br /> the occurrence double-crossover event, serially sub-       amtB gene function vital importance A. <br /> cultured times selective medium containing kanamy-          vinelandii. <br /> cin. To verify replacement chromosomal wild-type            It hypothesized amtB gene encodes am- <br /> copies glnK occurred, genomic DNA isolated            monium transport protein, amtB::KIXX mutant <br /> analyzed Southern hybridization experiments using pDM508       strains able grow limiting ammonium con- <br /> a probe. However, transformants examined, wild-          centrations parental wild-type strain. The amtB mu- <br /> type mutated copies glnK::KIXX genes de-             tant strains MV560 MV561 able fix nitrogen and <br /> tected prolonged growth 10 subcultures           grew slightly UW136 N-free BS agar <br /> under selective conditions (i.e., kanamycin) (data          medium. Therefore, Nif2 nifH::KSS (lacZ) amtB::KIXX <br /> shown). Also, colonies isolated 10 cycles selection         mutant strains MV562 MV563 (Fig. 1) used these <br /> followed 1 cycle growth medium kanamycin            experiments growth dependent supply of <br /> had sensitive antibiotic. The <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>      fixed nitrogen. Mutants parent strain MV101 grown <br /> behavior observed transformed cells plated          BS medium plus urea (5 mM) diluted plated <br /> medium ammonium poor excellent            pipetting 20-ml suspension aliquots, containing approx- <br /> carbon sources incubated aerobic microaero-         imately 200 cells, BS medium containing ammonium ac- <br /> bic growth conditions. Therefore, mutants showed aber-         etate concentrations ranging 0.05 10 mM. There was <br /> rant behavior similar observed attempts            difference results obtained strains; in <br />  VOL. 180, 1998                                                                                                                                        NOTES          3263 <br />  <br />                                                                                       lished information, E. Barnes helpful technical suggestions <br />                                                                                       concerning methylammonium uptake experiments. <br />                                                                                                                          REFERENCES <br />                                                                                        1. Arsene, F., P. A. Kaminski, C. Elmerich. 1996. Modulation NifA <br />                                                                                           activity PII Azospirillum brasilense: evidence regulatory role the <br />                                                                                           NifA N-terminal domain. J. Bacteriol. 178:4830&#226;&#8364;&#8220;4838. <br />                                                                                        2. Atkinson, M. R., E. S. Kamberov, R. Weiss, A. J. Ninfa. 1994. Reversible <br />                                                                                           uridylylation Escherichia coli PII signal transduction protein regulates <br />                                                                                           ability stimulate dephosphorylation transcription factor <br />                                                                                           nitrogen regulator I (NRI NtrC). J. Biol. Chem. 269:28288&#226;&#8364;&#8220;28293. <br />                                                                                        3. Bali, A., G. Blanco, S. Hill, C. Kennedy. 1992. Excretion ammonium <br />                                                                                           nifL mutant Azotobacter vinelandii fixing nitrogen. Appl. Environ. <br />                                                                                           Microbiol. 58:1711&#226;&#8364;&#8220;1718. <br />                                                                                        4. Benelli, E. M., E. M. Souza, S. Funayama, L. U. Rigo, F. O. Pedrosa. <br />                                                                                           1997. Evidence possible glnB-type genes Herbaspirillum seropedi- <br />                                                                                           cae. J. Bacteriol. 179:4623&#226;&#8364;&#8220;4626. <br />   FIG. 3. [14C]Methylammonium uptake wild-type (&#226;&#8211;&#160;) amtB mutant <br />                                                                                        5. Blanco, G., M. Drummond, C. Kennedy, P. Woodley. 1993. Sequence <br /> (&#197;&#8217;) strains. Overnight cultures Burke&#226;&#8364;&#8482;s N-free sucrose medium diluted <br />                                                                                           molecular analysis nifL gene Azotobacter vinelandii. Mol. <br /> 50-fold grown optical density 600 nm 0.5 0.7. The details the <br />                                                                                           Microbiol. 9:869&#226;&#8364;&#8220;879. <br /> uptake experiments described Jayakumar Barnes (21). <br />                                                                                        6. Chang, A. C. Y., S. N. Cohen. 1978. Construction characterization of <br />                                                                                           amplifiable multicopy DNA cloning vehicles derived P15A cryptic <br />                                                                                           miniplasmid. J. Bacteriol. 134:1141&#226;&#8364;&#8220;1156. <br />                                                                                        7. Colnaghi, R., L. He, A. Green, D. Yan, P. Rudnick, C. Kennedy. Un- <br /> both cases, rate colony formation size                              published data. <br /> colonies concentrations colony                            8. Contreras, C., M. Drummond, A. Bali, G. Blanco, E. Garcia, G. Bush, C. <br /> size decreased decreasing ammonium concentrations, be-                               Kennedy, M. Merrick. 1991. The product nitrogen fixation reg- <br />                                                                                           ulatory gene nfrX Azotobacter vinelandii functionally structurally <br /> coming barely discernible ammonium concentration                                 homologous uridylyltransferase encoded glnD enteric bacteria. J. <br /> about 100 mM.                                                                             Bacteriol. 173:7741&#226;&#8364;&#8220;7749. <br />    Another test ammonium transport uptake                            9. Mel, V. S. J., E. S. Kamberov, P. D. Martin, J. Zhang, A. J. Ninfa, and <br /> radiolabeled ammonium analog [14C]methylammonium. In                                      B. F. P. Edwards. 1994. Preliminary X-ray diffraction analysis crystals of <br />                                                                                           PII protein Escherichia coli. J. Mol. Biol. 243:796&#226;&#8364;&#8220;798. <br /> these experiments, required aeration small sample                            10. Zamaroczy, M., A. Paquelin, G. Peltre, K. Forchhammer, C. Elmer- <br /> volumes detect transport wild-type strain, UW136                                ich. 1996. Coexistence structurally similar functionally different <br /> (amtB1) accumulated 10.6 nmol methylammonium 35                                  PII proteins Azospirillum brasilense. J. Bacteriol. 178:4143&#226;&#8364;&#8220;4149. <br /> min (Fig. 3). In contrast, amtB::KIXX mutant strain,                              11. Zamaroczy, M. 1997. Personal communication. <br />                                                                                       12. Drummond, M. H., A. Contreras, L. A. Mitchenall. 1990. The function <br /> MV560, failed transport methylammonium (taking                                 isolated domains chimaeric proteins constructed transcrip- <br /> than 5% transported wild-type strain).                               tional activators NifA NtrC Klebsiella pneumoniae. Mol. Microbiol. <br /> The amtB1 strain UW136 failed transport [14C]methylam-                                 4:29&#226;&#8364;&#8220;37. <br /> monium 15 mM ammonium added beginning                                13. Dubois, E., M. Grenson. 1997. Methylamine/ammonia uptake systems in <br />                                                                                           Saccharomyces cerevisiae: multiplicity regulation. Mol. Gen. Genet. 175: <br /> the experiment, expected ammonium shown                                67&#226;&#8364;&#8220;76. <br /> a competitive inhibitor methylammonium transport (21). In                          14. Fabiny, J. M., A. Jayakumar, A. C. Chinault, E. M. Barnes. 1991. <br /> another control experiment, samples                                 Ammonium transport Escherichia coli: localization nucleotide se- <br /> were washed filters, washing removed                               quence amtA gene. J. Gen. Microbiol. 137:983&#226;&#8364;&#8220;989. <br />                                                                                       14a.Foster-Hartnett, D., R. G. Kranz. 1994. The Rhodobacter capsulatus glnB <br /> 20% bound radioactivity, indicating retained                              gene regulated NtrC tandem rpoN-dependent promoters. J. <br /> 14 <br />    C true uptake adventitious binding                            Bacteriol. 176:5171&#226;&#8364;&#8220;5176. <br /> [14C]methylammonium. A clear phenotype amtB::KIXX                              15. Green, A., R. Colnaghi, C. Kennedy. Unpublished data. <br /> mutants evident: transport methylam-                         16. Hanson, T., J. C. Meeks. 1997. Personal communication. <br />                                                                                       16a.He, L. 1998. Personal communication. <br /> monium. While similar phenotype reported amt mu-                            17. Hill, S., S. Austin, T. Eydmann, T. Jones, R. Dixon. 1996. Azotobacter <br /> tants C. glutamicum (35) Azospirillum brasilense (37),                             vinelandii NIFL flavoprotein modulates transcriptional activation of <br /> the function prokaryotic AmtB proteins respect                              nitrogen-fixation genes redox-sensitive switch. Proc. Natl. Acad. Sci. <br /> to ammonium methylammonium uptake re-                                     USA 93:2143&#226;&#8364;&#8220;2148. <br />                                                                                       18. Hill, S., C. Kennedy, E. Kavanagh, R. Goldberg, R. Hanau. 1981. <br /> ported, (with exception S. cerevisiae strain mu-                            Nitrogen fixation gene (nifL) involved oxygen regulation nitrogenase <br /> tated mep genes) amtB mutant shown                              synthesis Klebsiella pneumoniae. Nature 290:424&#226;&#8364;&#8220;426. <br /> deficient ammonium transport se. While widely                            19. Holtel, H., M. Merrick. 1988. Identification Klebsiella pneumoniae <br /> assumed methylammonium transported analog                                glnB gene: nucleotide sequence wild-type mutant alleles. Mol. Gen. <br />                                                                                           Genet. 215:134&#226;&#8364;&#8220;138. <br /> ammonium, case, determi-                           20. Jaggi, R., W. van Heeswijk, H. V. Westerhoff, D. L. Ollis, S. Vasudevan. <br /> nation true physiological function amtB pro-                                 1997. The opposing activities adenylyl transferase reside distinct <br /> karyotes await identification specific physio-                         homologous domains, intramolecular signal transduction. EMBO J. <br /> logical phenotype associated amtB mutants. It                                16:5562&#226;&#8364;&#8220;5571. <br />                                                                                       21. Jayakumar, A., E. M. Barnes. 1984. The role glutamine regulation <br /> likely prokaryotes contain ammo-                              ammonium transport Azotobacter vinelandii. Arch. Biochem. Biophys. <br /> nium transporters, certain affinity substrate,                         231:95&#226;&#8364;&#8220;101. <br /> in S. cerevisiae.                                                                     21a.Johansson, M., S. Nordlund. 1996. Transcription glnB glnA <br />    Nucleotide sequence accession number. The nucleotide se-                               genes photosynthetic bacterium Rhodospirillum rubrum. Microbiology <br />                                                                                           142:1265&#226;&#8364;&#8220;1272. <br /> quence 2.3-kb region pPR101 carrying glnK                          22. Marini, A.-M., S. Soussi-Boudekou, S. Vissers, B. Andre. 1997. A family <br /> amtB genes A. vinelandii deposited GEN-                                ammonium transporters Saccharomyces cerevisiae. Mol. Cell. Biol. 17: <br /> EMBL database accession no. U91902.                                                 4282&#226;&#8364;&#8220;4293. <br />                                                                                       23. Marini, A.-M., S. Vissers, A. Urrestarazu, B. Andre. 1994. Cloning and <br />                                                                                           expression MEP1 gene encoding ammonium transporter Sac- <br />   This work supported USDA NRICG grant no. 95-37305-                               charomyces cerevisiae. EMBO J. 13:3456&#226;&#8364;&#8220;3463. <br /> 2067.                                                                                 24. Meletzus, D. 1996. Unpublished data. <br />   We thank Rita Colnaghi useful discussions critical reading               25. Meletzus, D., R. Eichenlaub. 1991. Transformation phytopatho- <br /> the manuscript, Miklos Zamaroczy permission cite unpub-                         genic bacterium Clavibacter michiganense subsp. michiganense electropo- <br />  3264       NOTES                                                                                                                                     J. BACTERIOL. <br />  <br />  <br />     ration development cloning vector. J. Bacteriol. 173:184&#226;&#8364;&#8220;190.           33. Rudnick, P., D. Meletzus, A. Green, L. He, C. Kennedy. 1997. Regula- <br /> 26. Merrick, M., S. Hill, H. Hennecke, M. Hahn, R. Dixon, C. Kennedy.                tion nitrogen fixation ammonium diazotrophic species Proteobac- <br />     1982. Repressor properties nifL gene product Klebsiella pneu-              teria. Soil Biol. Biochem. 29:831&#226;&#8364;&#8220;842. <br />     moniae. Mol. Gen. Genet. 185:75&#226;&#8364;&#8220;81.                                              34. Santero, E., A. Toukdarian, R. Humphrey, C. Kennedy. 1988. Identifi- <br /> 27. Merrick, M. J., R. A. Edwards. 1995. Nitrogen control bacteria.               cation characterisation nitrogen fixation regulatory regions, nifA <br />     Microbiol. Rev. 59:604&#226;&#8364;&#8220;622.                                                          nfrX, Azotobacter vinelandii Azotobacter chroococcum. Mol. Mi- <br /> 28. Neuwald, A. F., B. R. Krishnan, I. Brikun, S. Kulakauskas, K. Suz&#203;&#732;iede&#203;&#8482;lis, T.       crobiol. 2:303&#226;&#8364;&#8220;314. <br />     Tomcsanyi, T. S. Leyh, D. E. Berg. 1992. cysQ, gene needed             35. Siewe, R. M., B. Weil, A. Burkovski, B. J. Eikmanns, M. Eikmanns, R. <br />     cysteine synthesis Escherichia coli K-12 aerobic growth. J.           Kramer. 1996. Functional genetic characterization (methyl)am- <br />     Bacteriol. 174:415&#226;&#8364;&#8220;425.                                                 <br /> </body></html>