<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                             Proc. R. Soc. B (2005) 272, 1721&#226;&#8364;&#8220;1725 <br />                                                                                                         doi:10.1098/rspb.2005.3128 <br />                                                                                                         Published online 14 July 2005 <br />  <br />  <br />  <br />        Gene essentiality topology protein <br />                  interaction networks <br />                         Ste&#194;&#180;phane Coulomb1, Michel Bauer1, Denis Bernard1 <br />                                Marie-Claude Marsolier-Kergoat2,* <br />              1 <br />                  Service Physique The&#194;&#180;orique, CEA/Saclay-Orme des Merisiers, 91191 Gif-sur-Yvette Cedex, France <br />              2 <br />                  Service Biochimie et Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, CEA/Saclay, 91191 Gif-sur-Yvette Cedex, France <br />         The mechanistic bases gene essentiality cell mutational resistance long disputed. The <br />         recent availability large protein interaction databases fuelled analysis protein interaction <br />         networks authors proposed gene dispensability strongly related some <br />         topological parameters networks. However, results based protein interaction data <br />         biases taken account. In article, essentiality gene yeast is <br />         poorly related number interactants (or degree) corresponding protein the <br />         physiological consequences gene deletions unrelated properties proteins the <br />         interaction networks, average degrees nearest neighbours, clustering coefficients <br />         relative distances. We yeast protein interaction networks lack degree correlation, i.e. <br />         propensity vertices associate according degrees. Gene essentiality generally <br />         cell resistance mutations largely unrelated parameters protein network <br />         topology. <br />            Keywords: network; protein interaction; topology; mutational resistance; gene dispensability; yeast <br />  <br />  <br />  <br /> 1. INTRODUCTION                                                        strongly related gene essentiality cell <br /> Living organisms resistant mutations. Only                 robustness mutations ( Jeong et al. 2001; Maslov &amp; <br /> 18.7% Saccharomyces cerevisiae genes prove essential            Sneppen 2002; Barabasi &amp; Oltvai 2004). However these <br /> growth rich glucose medium 15% viable               conclusions based protein interaction data whose <br /> homozygous deletion strains exhibit slow growth                      biases overlooked. We reassessed <br /> <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> optimal conditions (Giaever et al.                     systematically relationships gene essentiality <br /> 2002). When systematic analysis synthetic lethal               topological characteristics proteins the <br /> sick (SLS) interactions (in combination                   yeast interaction networks. We dispensa- <br /> mutations genes causes cell death reduced                    bility gene weakly related number links <br /> fitness) performed yeast, average number                 corresponding protein completely <br /> SLS partners studied gene 28                 unrelated topological parameters we <br /> 4700 interactions tested (Tong et al. 2004).                           tested, suggesting protein network topology little <br />     Gene dispensability overall cell resistance                 influence essentiality specific genes more <br /> mutations explained redundancy                 globally cell mutational robustness. <br /> duplicate genes kinds functional compen- <br /> sations involving sequence-unrelated genes. Gu et al. <br /> (2003) significantly higher probability                2. METHODS <br /> functional compensation duplicate gene                Our study deals exclusively data related yeast <br /> single-copy gene high correlation                    S. cerevisiae. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> measure degrees essential non- <br /> frequency compensation sequence similarity               essential genes, data corresponding study Ito <br /> two duplicates. However, clear evidence               et al. (2001) downloaded web page http:// <br /> that duplicate genes play significant role mutational             genome.c.kanazawa-u.ac.jp/Y2H/. A list essential genes <br /> robustness, account main              obtained SGD (Saccharomyces Genome <br /> of it.                                                                 Database) webpage (http://db.yeastgenome.org/cgi-bin/ <br />     The accumulation small-scale studies                    SGD/phenotype/phenotype.pl?phenotypeZinviable). To <br /> development high-throughput technologies                       evaluate average numbers articles dealing essential <br /> recently led construction large protein interaction          non-essential genes, considered open reading <br /> networks vertices (the proteins) connected                   frames (ORFs) classified genuine study of <br />                                                                        Ghaemmaghami et al. (2003). The number SGD curated <br /> when evidence physical interaction has <br />                                                                        references retrieved automatically SGD web site at <br /> been found. Several authors proposed the <br />                                                                        http://www.yeastgenome.org/. <br /> topological properties proteins interaction networks <br />                                                                           To measure degrees SLS partners, average <br />                                                                        degrees nearest neighbours, clustering coefficients and <br /> * Author correspondence (marie-claude.marsolier-kergoat@cea.fr).   distances SLS partners, used two <br />  <br /> Received 20 December 2004                                          1721                                      q 2005 The Royal Society <br /> Accepted 7 April 2005 <br />  1722 S. Coulomb Gene essentiality protein network topology <br />  <br /> databases, considered comprehensive           database contains biases regarding number <br /> reliable: database interacting proteins (DIP) core           interactants essential non-essential genes <br /> (Xenarios et al. 2000) &#226;&#8364;&#732;affinity purification&#226;&#8364;&#8482; databases.   records data derived small-scale exper- <br /> The DIP core database records data derived small-         iments. We essential genes objects <br /> scale large-scale experiments validated        articles non-essential ones on <br /> different criteria reproducibility occurrence    average 1.75 times SGD curated references than <br /> of interaction paralogous proteins different         non-essential genes (the SGD web site performs systema- <br /> species (Deane et al. 2002). The DIP core database              tic searches PubMed literature papers <br /> downloaded May 2004 web page http://dip.doe-            mentioning yeast genes). The fact essential genes are <br /> mbi.ucla.edu/dip/Download.cgi. We constructed affinity          intensely studied non-essential ones likely to <br /> purification database GRID database (http://               increase number known interactants. <br /> biodata.mshri.on.ca:80/yeast_grid/servlet/SearchPage)                In order avoid biases databases recording the <br /> May 2004 selecting protein interactions        results small-scale studies, determined use only <br /> been affinity chromatography affinity            interaction data derived large-scale analyses. Besides <br /> precipitation: large majority data (about 90%)           study Uetz et al. (2000), large-scale analyses <br /> corresponds results Gavin et al. (2002) Ho et al.     conducted: Ito et al. (2001) tested all <br /> (2002) data collected small-        possible interactions 6000 yeast proteins <br /> scale studies&#226;&#8364;&#8482; results recorded MIPS database (http://       defined data core containing interactions <br /> mips.gsf.de/genre/proj/yeast/index.jsp). Whereas two-           observed thrice, Gavin <br /> hybrid assumed reveal binary interactions,      et al. (2002) Ho et al. (2002) analysed the <br /> the affinity methods yield complex composition data. In        proteins associated respectively 493 589 <br /> the case proteins B C retrieved affinity        tagged proteins. The study Ito et al. that <br /> purification protein A, databases used record         considered unbiased regarding proteins that <br /> A/B A/C interactions A, B C        analysed yeast proteins tested. In <br /> the complex, B interact C A/C               contrast, Ho et al. Gavin et al. analysed small fraction <br /> interaction mediated B. Thus data              yeast proteins proteins chosen <br /> produced affinity purification techniques subject      randomly. We focused data core Ito <br /> to biases two-hybrid system, certainly          et al. analysis fractions essential <br /> have limits. The purpose double choice (DIP       genes present 10% less-connected 10% <br /> core affinity purification databases) use large          most-connected genes 0.24 0.27, respectively. The <br /> databases priori different biases.                           average degrees essential non-essential genes are <br />     The data analyses Tong et al. (2004) collected      2.2 1.8, respectively, Ito et al. study, which <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> GRID database (http://biodata.mshri.on.ca:80/              means essential genes average 1.2 times <br /> yeast_grid/servlet/SearchPage).                                     links non-essential ones. If highest degrees <br />                                                                     genes present Ito et al. data artificially <br />                                                                     attributed essential genes database, ratio <br /> 3. RESULTS                                                          amounts 3.8. <br /> (a) The correlation essentiality                         As test correlation the <br /> a gene degree interaction networks                       topological positions genes physiological <br /> Protein interaction networks exhibit large degree                 consequences deletions, calculated average <br /> distribution, approximately fitted                degree SLS partners. We used study Tong et al. <br /> power law ( Jeong et al. 2001): probability P(k)         (2004) describes systematic screenings SLS <br /> protein interacts k proteins roughly decreases           partners 132 genes, termed query genes. The degrees of <br /> like kKg. These networks consist majority low-              SLS genes interaction networks estimated <br /> connected vertices minority highly connected               using different databases (the DIP core (Xenarios et al. <br /> vertices hubs. For convenience, consider             2000) affinity purification databases) was <br /> corresponding interaction networks linking genes              difficult evaluate respective biases <br /> protein products interact physically refer       measure (see &#194;&#167;2). If topological positions of <br /> topological parameters gene interaction networks.          genes networks related phenotypes of <br /> Several authors fraction essential           deletions, SLS partners expected exhibit <br /> genes 3 5 times higher highly connected genes           higher degrees bulk non-essential genes. <br /> than low-connected genes proposed               However average degree SLS partners <br /> the phenotypic consequences gene deletion yeast             similar average degree set non- <br /> are affected large extent gene degree            essential genes, 4.4 5.3 links SLS genes versus <br /> interaction networks ( Jeong et al. 2001; Wuchty 2004; Yu           3.6 4.8 links non-essential genes DIP core <br /> et al. 2004). However analyses used data derived              affinity purification databases, respectively. <br /> from Uetz et al. (2000) and/or DIP database                    We observed that, considering interaction results <br /> (Xenarios et al. 2000), present biases. The              priori unbiased essential genes, essential <br /> study Uetz et al. suffers specific bias           genes slightly interactants non- <br /> it mixes results individual analyses 192 genes              essential ones. Similarly, little difference <br /> (which remain unidentified) yielded 281                   average numbers links SLS genes and <br /> interacting protein pairs high-throughput                  non-essential genes. Our results suggest the <br /> screening procedures involving 6000 proteins            physiological consequences gene deletions only <br /> resulted 692 interacting protein pairs. The DIP                  weakly related gene degrees interaction networks. <br />  <br /> Proc. R. Soc. B (2005) <br />                                                 Gene essentiality protein network topology                  S. Coulomb           1723 <br />  <br /> (b) The essentiality gene average                                             25 <br /> degree neighbours <br />  <br />  <br />  <br />  <br />                                                                        average degree K1 <br /> Another structural feature suggested                                      20 <br /> influence gene essentiality average degree K1 their <br />                                                                                            15 <br /> neighbours. Considering genes given degree k, it <br /> has proposed deletion genes low K1                                10 <br /> less deleterious deletion genes high K1 <br /> because low K1 restrict influence                                         5 <br /> perturbations brought gene deletion (Maslov &amp; <br /> Sneppen 2002). We measured average                                            0   10        20    30    40      50         60    70 <br /> degree K1 nearest neighbours function k for <br />                                                                                                                 degree k <br /> essential non-essential genes, using affinity <br /> purification database DIP core database. We           Figure 1. The average degree K1 nearest neighbours a <br /> no significant difference K1 essential          function degree k essential (black circles) non- <br /> non-essential genes degrees k (figure 1         essential genes (grey diamonds). The list interactions used <br />                                                                    defined affinity purification database similar <br /> data shown). Hence essentiality gene does not <br />                                                                    results obtained DIP core database. <br /> seem related average degree neighbours <br /> in interaction networks. <br />     Interestingly, observed K1 independent <br /> of k essential non-essential genes. We                   (a) 25 <br /> confirmed observation measuring K1 function <br />  <br />  <br />  <br />                                                                        average degree K1 <br />                                                                                            20 <br /> of k genes affinity purification DIP <br /> core databases (figure 2a). This absence correlation                                    15 <br /> between k K1 contradicts reports stating that <br /> yeast protein interaction networks exhibit negative                                      10 <br /> degree correlation, propensity high-degree <br /> vertices attach low-degree vertices (Maslov &amp;                                         5 <br /> Sneppen 2002; Newman 2002; Newman 2003). How- <br /> ever, studies used sets data derived                                 0   10        20    30     40     50         60    70 <br /> systematic two-hybrid screens (Uetz et al. 2000; Ito et al.                                                     degree k <br /> 2001), suffer flaws, most <br /> problematic case number false                (b) <br />                                                                                            40 <br /> positives arising spurious activation the <br />                                                                        average degree K1 <br />  <br />  <br />  <br />  <br /> reporter genes proteins behaving weak transcrip-                                     30 <br /> tional activators (Aloy &amp; Russell 2002). To illustrate the <br /> facility spurious hubs bias                                  20 <br /> measures, introduced experimental database <br />                                                                                            10 <br /> affinity purification artificial vertices randomly <br /> connected 100, 150 200 vertices, mimicking the <br /> kind artifacts observed two-hybrid                                        0        40        80       120        160        200 <br /> experiments. As shown figure 2b, introduction                                                         degree k <br /> these hubs enriches network low-degree <br />                                                                    Figure 2. Absence correlation degrees k of <br /> vertices linked high-degree ones brings             genes average degrees K1 nearest neighbours. <br /> negative degree correlation comparable previously          (a) The average degrees K1 nearest neighbours genes <br /> reported (Maslov &amp; Sneppen 2002).                                  plotted degrees k using affinity <br />                                                                    purification database (black squares) DIP core <br /> (c) The essentiality gene clustering                  database (grey triangles)&#226;&#8364;&#8221;the point (kZ111; K1Z5.2) for <br /> coefficient                                                        DIP database range figure. (b) K1 is <br /> Essential non-essential genes proposed          plotted function k affinity purification database <br /> to differ clustering coefficients ( Yu et al. 2004).      complemented artificial hubs randomly connected <br />                                                                    100, 150 200 vertices. The introduction three <br /> The clustering coefficient vertex i, Ci, defined the <br />                                                                    spurious hubs subsequent measure K1 were <br /> ratio ni, number links exist the <br />                                                                    repeated 100 times averaged results represented <br /> ki neighbours vertex i, ki (kiK1)/2, number         here. The point (kZ1) removed it <br /> of possible links ki neighbours. The             corresponds exclusively vertices directly <br /> average clustering coefficients essential non-              affected introduction spurious hubs. <br /> essential genes identical DIP core <br /> database (0.31 0.29, respectively) data core of <br /> the Ito et al. study (0.09 0.11, respectively), quite      relationship clustering coefficient the <br /> different affinity purification database (0.29        essentiality genes specifically protein <br /> 0.18, respectively) database derived              products stable complexes. However this <br /> Gavin et al. study (0.40 0.30, respectively). Since        potential connection valid considering more <br /> affinity purification database results Gavin        diverse sets interactions like recorded Ito <br /> et al. study probably enriched interactions          et al. study DIP core database, cannot <br /> stable protein complexes, observations hint           reliably associate gene essentiality parameter. <br />  <br /> Proc. R. Soc. B (2005) <br />  1724 S. Coulomb Gene essentiality protein network topology <br />  <br />                     35                                                   essentiality gene topological <br />                     30                                                   characteristics interaction networks. One main <br />                                                                          problems studying interaction networks lies the <br />     frequency (%) <br />  <br />                     25 <br />                                                                          choice interaction databases. All interaction <br />                     20                                                   databases probably biased appears crucial to <br />                     15                                                   try understand biases test bases <br />                     10                                                   recording different kinds data. <br />                      5                                                      Several authors proposed topological charac- <br />                                                                          teristics protein interaction networks like degree <br />                      0 <br />                          1   2   3     4    5   6     7     8   9   10   distribution degree correlation account cell <br />                                   distance (number links)             robustness mutations ( Jeong et al. 2001; Maslov &amp; <br />                                                                          Sneppen 2002; Barabasi &amp; Oltvai 2004). However, these <br /> Figure 3. The distributions distances separating query <br /> genes SLS partners (black circles)               hypotheses inconsistent observation the <br /> whole set non-essential genes (grey diamonds). The                    essentiality genes poorly related completely <br /> distances represent numbers intermediate links             unrelated topological parameters interaction <br /> gene interaction network defined DIP core database.               networks. The fact structure interaction <br /> They measured using breadth-first search algorithm.               networks little effect gene essentiality and <br /> Only query genes complete deletion studied                mutational robustness compatible hypotheses <br /> were included analysis. Similar results obtained            topology interaction networks be <br /> with affinity purification database.                                 submitted evolutionary constraints their <br />                                                                          structural features simply consequences of <br /> (d) Synthetic lethality/sickness relative <br />                                                                          construction processes. Indeed, shown that <br /> distances genes <br />                                                                          protein interaction networks large degree distri- <br /> Finally tested distance genes in <br />                                                                          bution generated evolution processes <br /> the interaction networks related pheno- <br />                                                                          involving gene duplication addition deletion of <br /> types double mutations. Using study Tong <br />                                                                          links, invoking natural selection degree <br /> et al. (2004) affinity purification database or <br />                                                                          distribution itself (Wagner 2003). More recently, <br /> the DIP core database, average distance <br />                                                                          Amoutzias et al. (2004) van Noort et al. (2004) have <br /> separating query genes SLS partners the <br />                                                                          shown structural characteristics of <br /> interaction networks (4.7 5.3 links affinity <br />                                                                          experimentally observed coexpression interaction <br /> purification DIP core databases, respectively) is <br />                                                                          networks generated neutralist models without <br /> similar average distance separating query genes <br />                                                                          need selection. Likewise, fact yeast protein <br /> from set non-essential genes (4.6 5.6 <br />                                                                          interaction networks lack degree correlation compatible <br /> links affinity purification DIP core <br />                                                                          random processes network construction, it <br /> databases, respectively). A closer analysis showed that <br />                                                                          excludes models network evolution based on <br /> the distributions distances query genes and <br />                                                                          asymmetric link attachment (Berg et al. 2004). <br /> their SLS partners global set non-essential genes <br />                                                                             In conclusion, results gene dispensability <br /> are identical (figure 3 data shown). The <br />                                                                          little relationship structure protein <br /> synthetic lethality sickness genes yeast thus <br />                                                                          interaction networks. Further investigations be <br /> does related relative topological positions <br />                                                                          required unravel molecular mechanisms gene <br /> in interaction networks. <br />                                                                          essentiality cell mutational resistance. <br />                                                                          We like thank Jean-Louis Sikorav Arach Goldar <br /> 4. DISCUSSION                                                            critical comments manuscript, Raphae&#194;&#168;l <br /> We looked systematically correlations               Gue&#194;&#180;rois automatic retrieval numbers SGD <br /> essentiality genes topological                   curated references. <br /> characteristics interaction networks. We shown <br /> that essentiality genes synthetic lethality or <br /> </body></html>