<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  Individual Subunits of the Ssn6-Tup11/12 Corepressor Are Selectively <br />          Required for Repression of Different Target Genes <br />  <br />  <br />              The Saccharomyces cerevisiae Ssn6 and Tup1 proteins form a corepressor complex that is recruited <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> target <br />           genes by DNA-bound repressor proteins. Repression occurs via several mechanisms, including interaction with <br />           hypoacetylated N termini of histones, recruitment of histone deacetylases (HDACs), and interactions with the <br />           RNA polymerase II holoenzyme. The distantly related fission yeast, Schizosaccharomyces pombe, has two <br />           partially redundant Tup1-like proteins that are dispensable during normal growth. In contrast, we show that <br />           Ssn6 is an essential protein in S. pombe, suggesting a function that is independent of Tup11 and Tup12. <br />           Consistently, the group of genes that requires Ssn6 for their regulation overlaps but is distinct <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> the group <br />           of genes that depend on Tup11 or Tup12. Global chip-on-chip analysis shows that Ssn6 is almost invariably <br />           found in the same genomic locations as Tup11 and/or Tup12. All three corepressor subunits are generally <br />           bound to genes that are selectively regulated by Ssn6 or Tup11/12, and thus, the subunit specificity is probably <br />           manifested in the context of a corepressor complex containing all three subunits. The corepressor binds to both <br />           the intergenic and coding regions of genes, but differential localization of the corepressor within genes does not <br />           appear to account for the selective dependence of target genes on the Ssn6 or Tup11/12 subunits. Ssn6, Tup11, <br />           and Tup12 are preferentially found at genomic locations at which histones are deacetylated, primarily by the <br />           Clr6 class I HDAC. Clr6 is also important for the repression of corepressor target genes. Interestingly, a subset <br />           of corepressor target genes, including direct target genes affected by Ssn6 overexpression, is associated with the <br />           function of class II (Clr3) and III (Hst4 and Sir2) HDACs. <br />  <br />  <br />    Gene regulation by corepressors is an important mechanism                       The Ssn6 protein is characterized by a central 10-copy tet- <br /> for controlling the transcriptional activity of the genome. The                 ratricopeptide repeat domain, which provides a protein inter- <br /> global yeast Ssn6/Tup corepressor, mainly characterized in                      action surface that mediates the interaction with Tup1 (41, 43) <br /> Saccharomyces cerevisiae, is conserved in all fungi and has                     as well as recruitment of the complex to target genes by dif- <br /> functional homologues in higher eukaryotes, such as Groucho                     ferent DNA-bound repressor proteins (26, 47). The different <br /> and TLE (26, 43). The multifunctional Ssn6/Tup corepressor is                   tetratricopeptide repeat motifs play an important role in the <br /> recruited to target genes by interactions with DNA-binding                      specificity of Ssn6 interaction with different repressor proteins <br /> repressor proteins and regulates a wide variety of processes,                   (44, 46). Ssn6 can form functional complexes with mammalian <br /> such as glucose utilization (33), mating type (25), DNA dam-                    Tup1 homologues, such as TLE, suggesting involvement of <br /> age repair (20), and stress response (37). The Tup1 protein                     mammalian Ssn6-like proteins in repressor complexes in mam- <br /> consists of a highly conserved WD 40 repeat domain which is                     mals that are analogous to those found in fungi (17). The <br /> believed to be important for interaction with other proteins via                closest human Ssn6 homologues are encoded by the ubiqui- <br /> a propeller-like ring structure (36, 56). The Tup1 tetramer is                  tously transcribed Y and X chromosome genes, UTY and <br /> thought to interact with the N terminus of the Ssn6 protein to                  UTX, which are involved in dosage compensation and X/Y <br /> form a corepressor complex (14, 23, 48). In the fission yeast,                  chromosome inactivation (30). <br /> Schizosaccharomyces pombe, there are two paralogous TUP1-                          It has been suggested that the Tup1 tetramer is the major <br /> like genes encoding the Tup11 and Tup12 proteins (18, 24, 32).                  contributor to the repression activity of the complex (26, 47, <br /> Tup11 and Tup12 have some redundant functions (18, 24), but                     56). The similarity between the expression profiles of TUP1&#226;&#338;&#172; <br /> there is also a clear difference in the roles of Tup11 and Tup12                and SSN6&#226;&#338;&#172; strains (21) is consistent with a role of Ssn6 as a <br /> in stress responses (13). Tup11 and Tup12 can interact with                     bridging protein between DNA-bound repressor proteins, <br /> each other and with the S. pombe Ssn6 protein (13), but it is                   which define target genes, and Tup1, which represses their <br /> not clear whether they coexist in individual corepressor com-                   activity. The repression activity of the Ssn6/Tup1 corepressor <br /> plexes or whether they participate in distinct corepressor pools                appears to be dependent on several different mechanisms that <br /> that are recruited to overlapping but distinct sets of genes.                   contribute independently (57). First, Ssn6/Tup1 repression is <br />                                                                                 linked to chromatin modifications, and Tup1 has been shown <br />                                                                                 to bind hypoacetylated histone H3 and H4 N-terminal tails <br />    * Corresponding author. Mailing address: School of Life Sciences,            directly (11, 12). Second, Ssn6/Tup1 interacts physically with <br /> So&#194;&#168;derto <br />        &#194;&#168;rns Ho &#194;&#168;gskola, SE-141 89 Huddinge, Sweden. Phone: 46 8                 class I and II histone deacetylases in S. cerevisiae, and the <br /> 6084714. Fax: 46 8 6084510. E-mail: fredrik.fagerstrom@sh.se. <br />    &#226;&#8364;&#160; Supplemental material for this article may be found at http://mcb <br />                                                                                 localization of Tup1 has been correlated to decreased acetyla- <br /> .asm.org/.                                                                      tion levels in the subtelomeric regions of chromosomes (9, 50, <br />    &#228;&#338;&#164; <br />      Published ahead of print on 13 November 2006.                              52). As a result of these or as yet undiscovered mechanisms, <br />  <br />                                                                          1069 <br />  1070                &#194;&#168; M-BILLAI ET AL. <br />             FAGERSTRO                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />                 TABLE 1. S. pombe strains used in this study                   HU118 strain via electroporation, and correct insertion was confirmed by <br />                                                                                Southern blotting and PCR. Strains were cultivated at 30&#194;&#176;C in rich yeast <br />                                                                  Reference     extract medium YES, containing 0.5% yeast extract and 3% glucose and <br />        Strain                       Genotype <br />                                                                  or source     supplemented with 75 mg required amino acids per liter media or in minimal <br /> JY741                   ura4-D18 leu1-32 ade6M216               32             media MM described previously (1). Kanamycin (200 mg/liter) was added to <br /> tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;           &#226;&#338;&#172;tup11::ura4&#226;&#171;&#185; &#226;&#338;&#172;tup12::leu2&#226;&#171;&#185; h&#226;&#171;&#186;          32             rich media 75, and 5-flouroorotic acid (5-FOA, 0.1%) was added to MM- <br />                           ura4-D18 leu1-32 ade6M216                            uracil media after autoclaving. All extractions for immunoprecipitation and <br /> FFB010                  ssn6::kanMX//ssn6&#226;&#171;&#185; h&#226;&#171;&#186; ura4-D18          This study     microarray analysis were made from cells grown in rich media except where <br />                           leu1-32 ade6M210                                     selection for plasmids was appropriate (13). <br /> Hu497                   ssn6GFP kanMX h&#226;&#171;&#185; ura4-D18               13                Immunofluorescence microscopy. An endogenously tagged Ssn6GFP strain <br />                           leu1-32 ade6-210                                     was grown for 18 h to mid log phase (1 &#226;&#171;&#187; 107 cells/ml), harvested, fixed with <br /> Hu853                   tup11GFP kanMX h&#226;&#171;&#185; ura4-D18              13             formaldehyde, and incubated with primary anti-green fluorescent protein (anti- <br />                           leu1-32 ade6M210                                     GFP, polyclonal rabbit 11121; Molecular Probes) and secondary antibodies (fluo- <br /> Hu851                   tup12GFP kanMX h&#226;&#171;&#185; ura4-D18              13             rescein isothiocyanate conjugated; Jackson Immunoresearch Laboratories) as <br />                           leu1-32 ade6-M210                                    described previously (4). Nuclear staining was performed with 4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino- <br /> Hu494                   ssn6HA-ts kanMX h&#226;&#171;&#185; ura4-D18             13             2-phenylindole (DAPI). The cells were subjected to fluorescence microscopy <br />                           leu1-32 ade6-210                                     using a Zeiss Axioscope II microscope. Images were captured and analyzed using <br /> LPY3277                 &#226;&#338;&#172;hst4::his3&#226;&#171;&#185; h&#226;&#171;&#186; ura4-D18 leu1-32        15             the Openlab software by Improvision. <br />                           ade6M216 his3-D1 arg3-D4                                Microarray analysis. RNA was extracted from cells as described in reference <br />                                                                                53, except that the ssn6HA-ts strain was shifted to 36&#194;&#176;C for 1 h before extraction <br />                                                                                of RNA and the Ssn6-overexpressing cells were grown in MM-uracil medium at <br />                                                                                30&#194;&#176;C. Two RNA samples from independent cultures were prepared for each <br />                                                                                condition, and 25 &#226;&#65533;&#174;g of RNA was subjected to reverse transcription (11904-018; <br /> Ssn6/Tup1 has also been shown to influence the organization                    Invitrogen) and labeled with CY3 (CY3 dCTP 53021; Amersham) or CY5 (CY5 <br /> of nucleosomes at target promoters (8). A third mechanism of                   dCTP 55021; Amersham) prior to hybridization on S. pombe gene microarrays <br /> repression by Ssn6/Tup1 is direct interference with the poly-                  from Eurogentec SA, Belgium (53). Altogether, four data points were generated <br /> merase II transcriptional machinery (54). Several subunits of                  for each condition and gene using two microarrays spotted in duplicate with two <br /> RNA polymerase II and the mediator subcomplex have been                        independent biological samples labeled in dye swap. The microarray signals were <br />                                                                                measured using a Scanexpress laser scanner and quantified using the Spotfinder <br /> identified in genetic screens for mutants that are unresponsive                quantification software (TIGR). The data were normalized using the Lowess per <br /> to Tup1 (19, 38). Some of these components, such as the                        spot per chip method, analyzed, and filtered using Genespring software (Silicon <br /> Hrs1/Med3 and Srb10 subunits of the mediator, interact di-                     Genetics). For microarray datasets and generated gene lists, see NCBI GEO <br /> rectly with the corepressor (29, 34). Interestingly, Ssn6/Tup1                 submissions (http://www.ncbi.nlm.nih.gov/geo/), accession number GSE4566, <br />                                                                                and the supplemental material. <br /> appears to play a role in the derepression of genes together <br />                                                                                   RT-PCR. RNA was extracted as described above and subjected to DNase <br /> with the SAGA and SWI/SNF complexes (7, 35, 37, 45). Fur-                      treatment and reverse transcription (RT, 11904-018; Invitrogen) for synthesis of <br /> thermore, Ssn6 has recently been identified as a coactivator                   cDNA. Samples were subjected to PCR analysis in real time with SYBR green <br /> involved in the Gcn4-dependent activation of multiple gene                     quantification. Enrichment relative to the wild type (WT) and actin&#226;&#171;&#185; expression <br /> targets (27).                                                                  levels was calculated with results from triplicate samples. <br />                                                                                   Chromatin immunoprecipitations. Chromatin immunoprecipitations were es- <br />    In this study we have investigated the role of Ssn6 in S.                   sentially performed as described previously (39, 51). Cells were grown to mid log <br /> pombe and its relationship to the function of the Tup11 and                    phase (1 &#226;&#171;&#187; 107cells/ml) in rich YES media. All cells were harvested at room <br /> Tup12 proteins. The results show that (i) Ssn6 and the                         temperature and immediately subjected to cross-linking with 10 mM dimethyl <br /> Tup11/12 proteins are required for regulation of distinct but                  adipimidate and 0.25% dimethyl sulfoxide for 45 min, followed by 1% formal- <br />                                                                                dehyde treatment for 2 h. Cross-linking was stopped for 15 min with 2.5 M <br /> overlapping groups of genes, consistent with the different phe- <br />                                                                                glycine, and cells were washed in phosphate-buffered saline and lysed in a fast <br /> notypes associated with defects in Ssn6 and Tup11/12; (ii)                     prep bead beater with lysis buffer (150 mM NaCl) and protease inhibitors. The <br /> localization of all three corepressor subunits is highly corre-                lysate was subjected to sonication three times for 60 s for generation of chro- <br /> lated at both common and selective targets of Ssn6 and Tup11/                  matin fragments with an average <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> of 1 kb. The lysate was removed by <br /> 12, consistent with the existence of a single predominant form                 centrifugation for 10 min at 13,000 rpm, and cell debris was sonicated for a <br />                                                                                further 60 s. The clear cell lysate from the two sonications was pooled, aliquoted, <br /> of the corepressor complex containing all three subunits; (iii)                and subjected to immunoprecipitation with a polyclonal anti-GFP antibody 1:100 <br /> the corepressor is preferentially associated with intergenic re-               (8372-2; Applied Biosciences) and protein A Sepharose beads (17-5280-01; Amer- <br /> gions as expected but is also found in the coding regions of                   sham). Chromatin-bound beads were washed with 2&#226;&#171;&#187; lysis buffer (500 mM <br /> many genes; and (iv) the class I histone deacetylase (HDAC),                   NaCl) and with 2&#226;&#171;&#187; deoxycholate buffer (0.1% sodium dodecyl sulfate) and <br />                                                                                eluted from beads with N-tris(hydroxymethyl)methyl-2-aminoethanesulfonic <br /> Clr6, is the major HDAC responsible for deacetylation and <br />                                                                                acid (TES, 1.5% sodium dodecyl sulfate) buffer. Cross-linking was reversed <br /> repression of corepressor target genes, but Clr3 (class II) and                overnight at 65&#194;&#176;C, and samples were treated with protease K and glycogen for 2 h <br /> Hst4 (class III) also play an important role at some corepressor               at 57&#194;&#176;C. DNA was purified by phenol-chloroform extraction and ethanol-sodium <br /> target genes.                                                                  acetate precipitation. Input (1:200) and precipitates were subjected to SYBR <br />                                                                                green quantification with real-time PCR using primers specific for intergenic <br />                                                                                regions (IGR). Binding ratios were calculated as the difference in the precipi- <br />                          MATERIALS AND METHODS <br />                                                                                tated fraction from input DNA as Input-IP(AB)/Input-IP(NOAB) relative to the <br />   Strains. The S. pombe strains in this study are listed in Table 1. LPY3277   binding to a control (tRNA glutamyl synthetase) IGR. The average values were <br /> encodes a defective Hst4 protein lacking residues 75 to 162. The ssn6&#226;&#171;&#185; open    derived from triplicate samples from independent ChiP experiments and calcu- <br /> reading frame (ORF) was deleted by a PCR strategy described previously (3).    lated along with the standard error of the mean. <br /> The following primer pairs were used for generation of a KanMX cassette           Genome-wide chromatin binding maps. Precipitated chromatin and input <br /> from the pFA6 plasmid for deletion of ssn6&#226;&#171;&#185;: 5&#226;&#172;&#732;-TTCCCGCGTATCAGCT               DNA fragments from three independent experiments were subjected to a two- <br /> ACACCAGTATCATCAATTTTTAAAATATGTATGACTATTGTAAGCA                                 step linear amplification as described previously (51). Round A was performed <br /> AATTTCAAATGTGAGCGAATTCGAGCTCGTTTAAAC-3&#226;&#172;&#732; and 5&#226;&#172;&#732;GTG                              with Sequenase (USB 70775Y) and TPCRA priming. Round B was performed <br /> GTTTACGTGGATTTCGTTCTCTGAACTTTTCCTTTTCCATAAGCGG                                 with Amplitaq (Roche) with TPCRB priming. Amplified material (500 to 1,000 <br /> ATGTTGCATTCTTGAAGAATTTCTATTCTCGGATCCCCGGGTTAAT                                 ng) was subjected to Klenow labeling (18094-011; Invitrogen) with Cy3 and Cy5 <br /> TAA-3&#226;&#172;&#732;. Amplified fragments were transformed into the wild-type diploid        (53021 and 55021; Amersham). Input fragments were labeled with Cy3, and <br />  VOL. 27, 2007                                                          REQUIREMENT OF SUBUNITS OF Ssn6-Tup11/12 COREPRESSOR                        1071 <br />  <br /> precipitated material was labeled with Cy5. Labeled fragments were hybridized         germination, since microcolonies can form on YES media <br /> to IGR plus ORF microarrays from Eurogentech SA, Seraing, Belgium (51).               plates containing 1.2 M sorbitol. Images from cells obtained <br /> Microarray signals were measured using a ScanExpress laser scanner and quan- <br /> tified with the Spotfinder quantification software (TIGR). Data were analyzed <br />                                                                                       from microcolonies show that ssn6&#226;&#338;&#172; cells exhibit an abnormal <br /> and median normalized to the 50th percentile using GeneSpring software (Sili-         morphology compared to WT cells (Fig. 1D). We conclude <br /> con Genetics) to generate six data points for each binding map. Flagged values        that Ssn6 has cellular roles that are not shared by the Tup11 <br /> were removed from the data set, and ratios were subjected to median percentile        and Tup12 genes, since strains lacking both Tup11 and Tup12 <br /> ranking analysis described previously (5). Median rank values were calculated <br />                                                                                       are viable under equivalent growth conditions. Our results thus <br /> with both combined IGR and ORF values and with separate individual ORF and <br /> IGR values. Cutoff values for the generation of binding lists were set to the 85th    suggest the existence of a class of genes that are regulated by <br /> percentile. For chromatin immunoprecipitation-chip datasets and gene lists, see       Ssn6 but not Tup11 or Tup12. <br /> NCBI GEO submissions (http://www.ncbi.nlm.nih.gov/geo/), accession number                Ssn6 and Tup11/12 affect the regulation of distinct but over- <br /> GSE4566, and the supplemental material. For higher resolution tiling arrays,          lapping sets of genes. To screen for Ssn6 regulated gene tar- <br /> precipitated chromatin was subjected to round B amplification as described <br />                                                                                       gets in S. pombe, we used DNA microarrays (53) to compare <br /> above; 5.0 &#226;&#65533;&#174;g of total DNA was fragmented into approximately 100-bp fragments <br /> by DNase I treatment and labeled with biotin (31). Labeled material was hy-           RNA expression profiles in a strain containing a conditional <br /> bridized to S. pombe Affymetrics high-density oligonucleotide gene chips cover-       ssn6 allele (ssn6HA-ts) (13) with a wild-type strain at the non- <br /> ing chromosome II and half of chromosome III (42). Data were normalized to            permissive temperature. The scatter plot in Fig. 2A shows that <br /> the 50th percentile, and the binding ratio versus untagged wild type was deter-       the expression of a number of genes is affected in the ssn6HA-ts <br /> mined as described previously (42). Normalization and data analysis were per- <br /> formed with Genespring 7.2 (Silicon Genetics), Access, and Excel (Microsoft) <br />                                                                                       strain. Sixty-five genes were reduced in expression, and 29 <br /> software. Complete tiling array datasets are at NCBI GEO submissions (http:           genes were increased in expression by &#226;&#177;&#8211;1.8-fold. To determine <br /> //www.ncbi.nlm.nih.gov/geo/), accession number GSE4566.                               whether these genes are also targets of Tup11 or Tup12, we <br />    Cluster, hypergeometric distribution, and gene ontology analysis. Binding          performed a similar comparison of a tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; double <br /> clusters were built with WT (H4K5Ac, H4K12Ac, H3K14Ac, H4K16Ac, and <br />                                                                                       deletion strain with the wild type (Fig. 2B). Ninety-one genes <br /> H4K9Ac) acetylation ratios corrected for H3 density together with Ssn6, Tup11, <br /> and Tup12 binding ratios (n &#226;&#171;&#189; 4,055). Hierarchical expression clusters were built     were reduced in expression, and 158 genes were increased in <br /> in two steps with Ssn6/Tup11/Tup12 individual common IGR- and ORF-bound               expression by &#226;&#177;&#8211;1.8-fold. The number of genes affected in the <br /> targets of &#226;&#177;&#8211;85% (n &#226;&#171;&#189; 334). First, all the expression levels were hierarchically       ssn6HA-ts array was lower than in the tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; expres- <br /> clustered, and second, the relationship between experiments was built in a con-       sion profile, which could be explained by a lower penetrance of <br /> dition tree. All cluster analysis was performed by using a Pearson correlation <br /> algorithm with the Genespring 7.2 (Silicon Genetics) software. Significant <br />                                                                                       the partly functional ssn6HA-ts allele. We therefore used a <br /> Tup11/12- and Ssn6-specific target association with different gene ontology cat-      third approach to detect potential corepressor targets in which <br /> egories was determined using the GO miner web resource (http://discover.nci           Ssn6 was overexpressed from a plasmid. The scatter plot in Fig. <br /> .nih.gov/gominer/) (55). Significance similarity analysis with previously published   2C shows the expression profile of genes affected by Ssn6 <br /> microarray data (51) was performed with the hypergeometric distribution test          overexpression. Two hundred seventy-nine genes were reduced <br /> using the Genespring 7.2 software (Silicon Genetics). <br />                                                                                       in expression, and 163 genes were increased in expression by <br />                                                                                       &#226;&#177;&#8211;2-fold. The individual genes identified in each of these ex- <br />                                   RESULTS                                             periments are listed in Table S1 in the supplemental material. <br />                                                                                       Several genes showing altered expression in each of these <br />    Ssn6 is an essential protein in S. pombe. The Ssn6 protein in                      DNA microarray studies have been studied independently by <br /> fission yeast is encoded by the ORF SPBC23E6.09 on chromo-                            RT-PCR under the same conditions (data not shown) (13). In <br /> some II which shows a clear match with other Ssn6 orthologues                         general, the results were in good agreement with the DNA <br /> in the Ascomycetes. Fluorescence microscopy of a strain in                            microarray results. <br /> which ssn6&#226;&#171;&#185; is fused at the C terminus to a GFP tag (13) shows                           To identify the group of genes that require both Ssn6 and <br /> a distinct nuclear expression <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Fig. 1A), similar to that                      Tup11/12 for repression, we determined the overlap between <br /> observed for Tup11 and Tup12 previously (13). To analyze the                          genes found in the different studies for both up- and down- <br /> function of Ssn6, we created a diploid strain containing a het-                       regulated genes (Fig. 2D). Since the ssn6HA-ts allele has gen- <br /> erozygous <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>null allele</span> of the ssn6 gene by insertion of a kana-                        erally lower effects on target gene expression, we used a change <br /> mycin cassette. The resulting strain was validated by PCR and                         threshold of &#226;&#177;&#8211;1.5-fold for selection of these genes (145 genes <br /> Southern blot analysis (data not shown). Interestingly, tetrad                        expressed at a lower level and 113 at a higher level). Each of <br /> dissection of the ssn6&#226;&#171;&#185;//ssn6&#226;&#338;&#172; strain revealed that only two                          the three sets of selected genes overlap significantly with each <br /> kanamycin-sensitive spores were viable in each tetrad (Fig.                           other, identifying common groups of genes whose expression is <br /> 1B), and thus, Ssn6 is an essential protein under normal growth                       affected in at least two of the three conditions, but the overlaps <br /> conditions in fission yeast. Viable kanamycin-resistant (ssn6&#226;&#338;&#172;)                       are less pronounced for down-regulated targets, as expected <br /> haploid progeny can be obtained if the ssn6&#226;&#171;&#185;//ssn6&#226;&#338;&#172; strain is                         for defects in a corepressor. Importantly 69 (53 &#226;&#171;&#185; 16) genes <br /> first transformed with a plasmid expressing Ssn6 (pDual-Ssn6),                        require the function of both Ssn6 and Tup11/12. However, <br /> and thus, the observed growth defect is associated with the                           most of the identified genes are not found in the intersecting <br /> ssn6&#226;&#338;&#172; allele and its failure to produce the Ssn6 protein (Fig.                        region affected by ssn6HA-ts and tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;. It could be <br /> 1C). These haploid progeny are uracil protrophs due to the                            argued that many genes that lie outside the overlap are targets <br /> presence of the pDual-Ssn6 plasmid which contains the ura4&#226;&#171;&#185;                           for both Ssn6 and Tup11/12 but that in one case the change in <br /> selectable marker gene. As expected, growth of kanamycin-                             their expression lies just below the relative change threshold <br /> resistant progeny is dependent on the presence of the pDual-                          that we applied. This does not seem to be the case, since the <br /> Ssn6 plasmid, since curing of the plasmid from the strains on                         relative change distribution associated with tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; in <br /> 5-FOA-containing plates leads to loss of viability (Fig. 1C).                         the Ssn6-dependent group of genes (n &#226;&#171;&#189; 189) is not signifi- <br /> The lethality does not appear to be due to defective spore                            cantly changed from that observed in a random set of genes <br />  1072              &#194;&#168; M-BILLAI ET AL. <br />           FAGERSTRO                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. The Ssn6 protein is a nuclear protein that is essential for viability in S. pombe. (A) Immunofluorescence microscopy analysis showing <br /> the nuclear staining pattern of the Ssn6GFP fusion protein expressed from the genomic Ssn6 locus and nuclear DNA staining of the same cell <br /> performed with DAPI. (B) Surviving spores after tetrad dissection of a heterozygous ssn6&#226;&#171;&#185;//ssn6&#226;&#338;&#172; strain. Tetrad dissection results in a 50% <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> <br /> ratio on rich media (YES). All surviving spores are kanamycin sensitive, indicating an essential function of the Ssn6 protein. (C) A ura4&#226;&#171;&#185; plasmid <br /> expressing the full-length Ssn6 protein (pDUAL-Ssn6) can complement the ssn6&#226;&#338;&#172; lethality <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Kanamycin-resistant colonies were <br /> recovered from haploid spores containing pDUAL-Ssn6. Counterselection against pDUAL-Ssn6 with 5-FOA prevents formation of kanamycin- <br /> resistant colonies. (D) Image showing the abnormal morphology of ssn6&#226;&#338;&#172; cells in microcolonies growing on YES with 1.2 M sorbitol together with <br /> WT control cells from normal sized colonies. <br />  <br />  <br />  <br /> (P &#226;&#171;&#189; 0.8). The same is true for the observed effects of                     (data not shown), a condition where tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; and <br /> ssn6HA-ts in the Tup11/12-dependent targets (n &#226;&#171;&#189; 180). Thus,                ssn6HA-ts cause clear phenotypes (13). Initially, we investi- <br /> the majority of these genes selectively require either Ssn6 or              gated the rds1&#226;&#171;&#185; gene to optimize precipitation conditions. Fig- <br /> Tup11/12 for their regulation. From these studies, we conclude              ure 3A shows the binding ratios of the Ssn6, Tup11, and Tup12 <br /> that Ssn6 and Tup11/12 appear to regulate overlapping but                   proteins at the rds1&#226;&#171;&#185; promoter relative to their association <br /> distinct groups of target genes in fission yeast.                           with the tRNA glutamyl synthetase promoter, which is not <br />    Identification of genes directly targeted by Ssn6 and Tup11/             targeted by the corepressor. The results show strong selective <br /> 12. The idea that Ssn6 and Tup11/12 may regulate different                  precipitation of the rds1&#226;&#171;&#185; fragment with all three fusion pro- <br /> groups of genes assumes that the genes identified by expression             teins, while no selective binding was observed in a nontagged <br /> profiling are primarily direct target genes that physically asso-           control strain. <br /> ciate with the corepressor. To determine which corepressor                     Next, we determined the genome-wide localization of the <br /> subunits are localized with the gene sets identified in Fig. 2D,            Ssn6, Tup11, and Tup12 proteins. We used DNA microarrays <br /> we used chromatin immunoprecipitation of extracts prepared                  containing approximately 10,000 genomic fragments spotted in <br /> from strains in which Ssn6, Tup11, and Tup12 were fused to                  duplicate (51). To identify genomic regions that are signifi- <br /> GFP. Strains expressing the GFP-tagged proteins do not cause                cantly enriched in Ssn6, Tup11, and Tup12, we used a median <br /> any detectable growth phenotypes on KCl-containing media                    percentile ranking procedure that has been described previ- <br />  VOL. 27, 2007                                               REQUIREMENT OF SUBUNITS OF Ssn6-Tup11/12 COREPRESSOR                             1073 <br />  <br />  <br />  <br />  <br />    FIG. 2. Ssn6-regulated gene targets are partly distinct from Tup11/12 targets. (A) Sample control plot showing genes affected in the ssn6HA-ts <br /> mutant strain at 36&#194;&#176;C. The values plotted are the averages of signals measured after hybridization of two independent sets of samples from the <br /> ssn6HA-ts strain and WT control after Lowess normalization. The diagonal lines show the position of signals that are the same in both samples <br /> (middle line) as well as genes whose expression is increased (lower line) or decreased (upper line) by twofold in the mutant strain. (B) Sample <br /> control plot showing genes affected in the tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; mutant strain in relation to the wild-type control (JY741). The values plotted are the <br /> Lowess-normalized average of signals measured after hybridization of two independent sets of samples. Annotation of the plot is the same as for <br /> panel A. (C) Sample control plot showing genes affected by Ssn6 overexpression. The values plotted are the averages of signals measured after <br /> hybridization of two independent sets of samples of JY741 containing pDUAL-Ssn6 or the pDUAL-Empty control after Lowess normalization. <br /> Annotation of the plot is the same as for panel A. (D) Venn diagrams showing the overlap between up-regulated and down-regulated genes <br /> affected in the ssn6HA-ts (&#226;&#177;&#8211;1.5-fold) and in the tup11&#226;&#338;&#172; tup12&#226;&#338;&#172; (&#226;&#177;&#8211;1.8-fold) mutant strains together with all genes affected by Ssn6 overexpression <br /> (&#226;&#177;&#8211;2.0-fold). The overlaps between the up-regulated gene sets are all significantly larger than would be expected by chance (P &#226;&#171;&#189; 1.0 &#226;&#171;&#187; 10&#226;&#171;&#186;39, 9.6 &#226;&#171;&#187; <br /> 10&#226;&#171;&#186;9, and 1.8 &#226;&#171;&#187; 10&#226;&#171;&#186;4, respectively) and notably larger than the overlaps between the down-regulated gene sets (P &#226;&#171;&#189; 5.5 &#226;&#171;&#187; 10&#226;&#171;&#186;8, 7.2 &#226;&#171;&#187; 10&#226;&#171;&#186;4, and <br /> 5.6 &#226;&#171;&#187; 10&#226;&#171;&#186;2, respectively). <br />  <br />  <br />  <br /> ously (5). Figure 3B shows the distribution of the median                  data. We selected genes within the 85th percentile and above <br /> percentile-ranked binding ratios for Ssn6, Tup11, and Tup12.               for further analysis. Lists of these genes are shown in Table S2 <br /> Fragments significantly enriched in Ssn6, Tup11, or Tup12 lie              in the supplemental material. To identify direct corepressor <br /> outside the normal distribution described by the bulk of the               targets, we studied the overlap between the group of genes that <br />  1074              &#194;&#168; M-BILLAI ET AL. <br />           FAGERSTRO                                                                                                                 MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Genome-wide localization of Ssn6, Tup11, and Tup12 correlates with the localization of genes whose expression is affected by <br /> ssn6HA-ts, tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;, and Ssn6 overexpression. (A) Ssn6, Tup11, and Tup12 are specifically associated with the rds1&#226;&#171;&#185; promoter in chromatin <br /> immunoprecipitation experiments. Binding to the rds1&#226;&#171;&#185; promoter (mean of triplicate samples &#226;&#171;&#190; standard error) was calculated in relation to <br /> binding observed for the tRNA glutamyl synthetase promoter that was used as a control. The bars show the relative level of precipitated rds1&#226;&#171;&#185; <br /> chromatin fragments after quantification by SYBR green real-time PCR. (B) Identification of genomic regions that are significantly associated with <br /> Ssn6GFP, Tup11GFP, and Tup12GFP. The histograms represent the frequencies of median ranked binding ratios of precipitated chromatin <br /> fragments in relation to input levels. The gray zones above the 85th percentile indicate targets outside the normal distribution (black line) that were <br /> classified as bound targets. A complete list of enriched fragments is found in Table S2 in the supplemental material. (C) Many genes whose <br /> expression is dependent on Ssn6 or Tup11/12 or affected by Ssn6 overexpression are direct corepressor targets. The Venn diagrams show the <br /> correspondence between overlapping groups of up- and down-regulated genes affected by corepressor defects or Ssn6 overexpression (Fig. 2D) <br /> with genes bound by Ssn6, Tup11, or Tup12 (&#226;&#177;&#8211;85th percentile, n &#226;&#171;&#189; 953). (D) The genomic distribution of corepressor subunits correlates with the <br /> localization of genes regulated by Ssn6 and Tup11/12. Moving average plot showing average binding level of Ssn6, Tup11, and Tup12 to intergenic <br /> regions of all genes (median rank binding) after ranking of the genes according to the effects on gene expression caused by ssn6HA-ts, tup11&#226;&#338;&#172; <br /> tup12&#226;&#338;&#172;, and Ssn6 overexpression (ranked expression ratio). The window size for the moving average plot was 100. <br />  VOL. 27, 2007                                           REQUIREMENT OF SUBUNITS OF Ssn6-Tup11/12 COREPRESSOR                     1075 <br />  <br />  <br /> physically interact with one or more corepressor subunits            proteins. However, there is a significant number of fragments <br /> (&#226;&#177;&#8211;85th percentile binding) (Fig. 3B) and the groups of genes         that are identified in only one of the data sets by the selection <br /> whose expression levels depend on Ssn6 or Tup11/12 or whose          criteria used (&#226;&#177;&#8211;85th percentile binding). Thus, either differ- <br /> expression is affected by Ssn6 overexpression (Fig. 2D). Figure      ential localization within target gene or differential subunit <br /> 3C shows that many up- and down-regulated targets identified         composition provides a possible explanation that could ac- <br /> by expression profiling are also physically associated with the      count for target gene-selective roles of Ssn6 and Tup11/12. <br /> corepressor. We observe a bias for binding to up-regulated           These possibilities are investigated in detail below. <br /> targets, but altogether, about one-third of expression targets          To study the relative localization of Ssn6, Tup11, and Tup12 <br /> are bound by corepressor components. This is likely to be an         quantitatively at IGRs and ORFs, we independently calculated <br /> underestimate, since the &#226;&#177;&#8211;85th percentile selection provides a       percentile ranked binding to fragments of each type (see Table <br /> very conservative estimate of binding targets. Furthermore, it       S2 in the supplemental material). The distribution of IGR <br /> is likely that corepressor components bind to regions that are       binding levels for Ssn6, Tup11, and Tup12 in each of the <br /> not covered by the probes on the DNA microarray.                     individually selected gene sets shows that each of the binding <br />    To further investigate the global relationship between core-      distributions is clustered high in the median rank and contrasts <br /> pressor function and its genome-wide localization, we plotted        strongly with a randomly selected set of genes (Fig. 4C). The <br /> moving averages of the combined binding data after ranking           vast majority of Ssn6-bound genes are also associated with <br /> the IGR of the genes according to their expression ratio in          Tup11 and Tup12, since &#226;&#177;&#8211;75% of Ssn6-bound genes are found <br /> each of the three expression profiling studies. Figure 3D shows      within the top 10% of Tup11- and Tup12-bound genes. A <br /> that genes that are derepressed in the ssn6HA-ts and tup11&#226;&#338;&#172;          similar relationship is seen between Tup11 and Tup12, sug- <br /> tup12&#226;&#338;&#172; mutant strains (high expression ratio) are correlated         gesting a very strong correlation in their localization. This <br /> with IGRs with elevated levels of corepressor binding. Genes         correlation extends to Ssn6, but here the 75th percentile lies <br /> that are repressed by Ssn6 overexpression (low expression ra-        just below 0.8 in the median rank. This pattern suggests that <br /> tio) are also highly correlated with IGRs that show high levels      Tup11 and Tup12 may be bound to some targets in the absence <br /> of corepressor binding. Thus, groups of genes from each of the       of Ssn6. To look more closely at corepressor-bound locations <br /> three expression profiling experiments that are putative targets     where one or two subunits might be absent, we performed <br /> for corepressor-dependent repression are enriched in genes           cluster analysis of genes from the three sets where the binding <br /> that bind to the corepressor. Interestingly, the group of genes      level of one or both the other subunits was below the 55th <br /> whose expression is increased by Ssn6 overexpression (high           percentile (Fig. 4D). Importantly, there are almost no Ssn6- <br /> expression ratio) is also associated with direct corepressor         bound genes that pass these criteria. For Tup11 and Tup12, <br /> binding. The possible significance of this correlation is dis-       there is a slightly larger number of genes where the proteins <br /> cussed in Discussion. The high level of correlation between the      might exist alone or in association with only one other core- <br /> functional significance of corepressor subunits and their local-     pressor subunit. Taken together, these results suggest that the <br /> ization allows the selection of sets of genes in the ensuing         vast majority of corepressor-bound locations are associated <br /> sections whose target gene status is supported by both func-         with all three corepressor subunits and that Ssn6 selective <br /> tional and binding data.                                             repression is most likely mediated via corepressors containing <br />    Corepressor components colocalize in both the intergenic          Ssn6, Tup11, and Tup12 subunits. <br /> and coding regions of genes. We have shown that genes whose             To allow direct comparison of Ssn6, Tup11, and Tup12 bind- <br /> expression is differentially dependent on Ssn6 and Tup11/12          ing to IGRs and ORFs, ORF fragments with binding levels in <br /> are bound by corepressor subunits and are thus likely to be          the top 15% were selected (see Table S2 in the supplemental <br /> direct targets. The differential dependence of different genes       material). The main conclusion of these data (Fig. 4E) is that <br /> on Ssn6 and Tup11/12 could in principle depend on factors            all three corepressor components show a strong tendency to <br /> such as differential localization of the corepressor within target   colocalize on ORF binding sites, just as they do on IGRs. The <br /> genes. Alternatively, different versions of the corepressor, con-    ORF data are more consistent with the possibility that a mi- <br /> taining different combinations of subunits, could be associated      nority of corepressor binding sites may not be associated with <br /> with different genes. To address these possibilities, we first       all three corepressor components than the IGR binding results <br /> investigated the nature of the locations to which Ssn6, Tup11,       (Fig. 4F). For example, there are 21 Ssn6-bound ORF sites <br /> and Tup12 are bound. About half the probes on the DNA                that may lack one or both the Tup11 and Tup12 proteins <br /> microarray correspond to ORFs. Even though the corepressor           compared to 3 IGR sites detected using equivalent criteria. <br /> shows a preference for localization within IGRs, it is also          However, this difference may be a consequence of the fact that <br /> frequently found in the ORF regions of genes (Fig. 4A). In the       the level of binding to ORFs tends to be somewhat lower than <br /> majority of cases, the corepressor was found associated with         to IGRs (Fig. 4A). We conclude that the vast majority of <br /> either the IGR or the ORF of individual genes. This, together        corepressor binding sites are bound by all three subunits, con- <br /> with the lack of any detectable tendency for the identified          sistent with a single predominant form of the Ssn6-Tup11/12 <br /> ORFs to be adjacent to downstream corepressor-bound IGRs,            corepressor. Furthermore, the differential requirement for <br /> shows that the apparent binding to ORF fragments is not an           Ssn6 and Tup11/12 at different target genes cannot be due to <br /> artifact resulting from its association with nearby corepressor-     binding of partial corepressor complexes, since there is good <br /> bound IGRs. We conclude that Ssn6, Tup11, and Tup12 are              support for binding of all three subunits to most of these genes. <br /> associated with both IGRs and ORFs. We next studied the                 High-resolution binding maps of Tup11, Tup12, and Ssn6. <br /> relative localization of Ssn6, Tup11, and Tup12 in the genome.       As suggested above, a second mechanism by which Ssn6 and <br /> Figure 4B shows a very high overlap in the localization of the       Tup11/12 could differentially affect target genes is the distri- <br />  1076             &#194;&#168; M-BILLAI ET AL. <br />          FAGERSTRO                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Ssn6, Tup11, and Tup12 colocalize within intergenic regions and open reading frames. (A) Corepressor subunits are bound to both <br /> IGRs and ORFs. The number of fragments (&#226;&#177;&#8211;85th percentile binding) that are bound to IGRs and ORFs, respectively, are shown. The numbers <br /> in parentheses indicate the number of genes for which binding is only observed in either the IGR or the ORF regions of genes. (B) The genomic <br /> localization of Ssn6, Tup11, and Tup12 is highly correlated. The diagram shows the overlap in the localization of the individually selected Ssn6-, <br /> Tup11-, and Tup12-bound locations. (C) Quantitative evaluation of binding by each of the three corepressor subunits to the sets of IGRs that <br /> individually show binding of Tup11, Tup12, or Ssn6 (&#226;&#177;&#8211;85% percentile median ranked binding) confirms a very high degree of subunit colocal- <br /> ization in IGRs. The individual values are plotted, and the boxes show the 75th percentile, the median, and the 25th percentile for each data set. <br /> A control group containing an equivalent number of randomly selected IGRs is analyzed in each panel. (D) Very few corepressor-bound IGRs <br /> provide evidence for binding of two or fewer subunits. IGRs where fewer than three subunits may be associated were analyzed by cluster analysis. <br /> The number of genes bound by each of the subunits but where binding of one or both of the other subunits has a binding level within the lower <br /> 55% of the median binding rank is shown. (E) Quantitative evaluation of binding by each of the three corepressor subunits to the sets of ORFs <br /> that individually show binding of Tup11, Tup12, or Ssn6 (&#226;&#177;&#8211;85% percentile median ranked binding) confirms a very high degree of subunit <br /> colocalization in ORFs. Annotation of the diagram is as for panel C, except that ORF locations are analyzed. (F) Very few corepressor-bound <br /> ORFs provide evidence for binding of two or fewer subunits. Annotation of the diagram is as for panel D, except that ORF locations are analyzed. <br />  <br />  <br />  <br /> bution of the corepressor complex within target genes. How-                enrichment of Ssn6, Tup11, or Tup12 anywhere within the <br /> ever, the array platform used so far might not reveal such                 upstream or coding region of the genes in relation to an un- <br /> differences, since it contains only one IGR and ORF probe per              tagged control strain. <br /> gene. To investigate the localization of Ssn6, Tup11, and                    Figure 5 shows tiling array binding profiles for a group of <br /> Tup12 with higher resolution, we used high-density tiling ar-              genes that are dependent on both Ssn6 and Tup11/12 as well as <br /> rays covering S. pombe chromosome 2 and half of chromosome                 groups that are dependent on either Ssn6 or Tup11/12 and a <br /> 3 with a 250-bp resolution, as described previously (42). Core-            group that is affected by Ssn6 overexpression. Expression levels <br /> pressor-bound genes were defined as those with a &#226;&#177;&#8211;2-fold                   for the representative genes shown under the different condi- <br />  VOL. 27, 2007                                                REQUIREMENT OF SUBUNITS OF Ssn6-Tup11/12 COREPRESSOR                              1077 <br />  <br />  <br />  <br />  <br />    FIG. 5. Distribution of bound Ssn6, Tup11, and Tup12 in the upstream and coding regions of different classes of corepressor-regulated genes. <br /> Corepressor subunits colocalize in the IGR and/or ORF regions of genes that are dependent on either Ssn6 and/or Tup11/12 as well as genes that <br /> are affected by Ssn6 overexpression. The four major panels show the localization of Tup11, Tup12, and Ssn6 using tiling arrays with a resolution <br /> of 250 bp. The average binding distribution of Ssn6, Tup11, and Tup12 for each gene class is shown in the upper panel of each of the major panels <br /> (the number of genes used to calculate the mean profiles is shown in parentheses). As a control, the level of histone H3 binding is shown. The genes <br /> in each group were selected as genes affected &#226;&#177;&#8211;2-fold in expression arrays and with a tiling binding ratio &#226;&#177;&#8211;2 at one or more points in the analyzed <br /> promoter and coding region. The lower panels in each major panel show a selection of individual genes for each group. The binding ratio was <br /> calculated as the binding signal for Ssn6GFP, Tup11GFP, and Tup12GFP relative to an untagged WT control. Expression ratios showing the <br /> dependency of these genes on Ssn6 and Tup11/12 as well as the effect of Ssn6 overexpression are listed in Table S2 in the supplemental material. <br />  1078             &#194;&#168; M-BILLAI ET AL. <br />          FAGERSTRO                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Different classes of direct gene targets of the Ssn6-Tup11/12 corepressor are differentially targeted by HDACs and are enriched in <br /> different functional classes of genes. (A) Ssn6, Tup11, and Tup12 are associated with hypoacetylated regions of chromatin. Gene cluster showing <br /> Ssn6, Tup11, and Tup12 binding ratios clustered together with acetylation levels for histone H4K5Ac, H4K12Ac, H3K14Ac, H4K16Ac, and <br /> H3K9Ac (corrected for H3 occupancy) at IGRs (n &#226;&#171;&#189; 4,055). (B) Direct corepressor target genes tend to be regulated by a specific subset of <br />  VOL. 27, 2007                                                 REQUIREMENT OF SUBUNITS OF Ssn6-Tup11/12 COREPRESSOR                               1079 <br />  <br />  <br /> tions are presented in Table S3 in the supplemental material.                hypoacetylated regions that are not bound by the corepressor, <br /> The upper panel in each group shows the mean distribution of                 and therefore, corepressor binding to hypoacetylated chroma- <br /> the three proteins on genes in the group, while the lower                    tin is selective, as expected for a coregulator recruited by reg- <br /> panels show the binding patterns associated with selected in-                ulatory transcription factors. <br /> dividual genes within each group. As a control, each set also                   We next asked whether Ssn6- and Tup11/12-regulated genes <br /> shows the relative levels of histone H3, which has been pub-                 require different classes of HDAC enzymes for repression. <br /> lished previously (51). The most striking conclusion from these              Figure 6B shows a hierarchical gene tree cluster analysis of the <br /> studies is the similarity of the binding distributions observed              gene expression changes at corepressor-bound genes that are <br /> for Ssn6, Tup11, and Tup12, providing further strong evidence                affected by ssn6HA-ts, tup11&#226;&#338;&#172;, tup12&#226;&#338;&#172;, and Ssn6 overexpres- <br /> for the binding of a corepressor complex containing all three                sion together with changes associated with defects in a range of <br /> subunits. The average distribution patterns generally show                   HDAC enzymes that have been published previously (51) and <br /> higher binding in the IGR than in ORFs, but the data from this               Hst4, which was determined in this study. The data sets from <br /> platform confirm significant binding of the corepressor to both              Ssn6- and Tup11/12-dependent genes cluster together in a <br /> IGR and ORF regions of regulated genes. The observed dis-                    main branch together with the expression profiles associated <br /> tributions are specific for the corepressor subunits because the             with the class I HDAC Clr6. Interestingly, a second branch <br /> histone H3 control pattern is generally higher in ORF regions                contains the Ssn6 overexpression profile together with the pro- <br /> than in IGRs. Maximal corepressor binding differs only slightly              files associated with the class III HDAC Hst4 and the class II <br /> between the different groups, with a binding peak approxi-                   HDAC Clr3. <br /> mately 500 to 1,000 bp upstream of the ORF. The significance                    A prediction of these results is that Clr6 should be the major <br /> of such differences is unclear, since the binding patterns vary              HDAC responsible for deacetylation of Ssn6- and Tup11/12- <br /> greatly for individual genes within each group. We therefore                 regulated genes. To test this, we determined the relationship of <br /> conclude that there are no simple differences in corepressor                 the overlap between binding targets that are affected by <br /> distribution that can account for the differential requirement               ssn6HA-ts, tup11&#226;&#338;&#172;, tup12&#226;&#338;&#172;, or Ssn6 overexpression (see Table <br /> for Ssn6 and Tup11/12 or the selective effects of Ssn6 overex-               S4 in the supplemental material) with lists of IGR that require <br /> pression at different gene targets.                                          the function of different HDACs for their deacetylation (51). <br />    Ssn6- and Tup11/12-dependent genes are primarily hy-                      Note that data for Hst4 are not available. Figure 6C shows that <br /> poacetylated and repressed by Clr6. The Ssn6-Tup1 corepres-                  Ssn6- and Tup11/12-dependent targets genes show a highly <br /> sor has been shown to recruit HDACs to sites in chromatin and                significant overlap with IGRs that require Clr6 for their <br /> to bind to hypoacetylated N termini of histone proteins (10,                 deacetylation at several acetylation sites. Genes affected by <br /> 12). Differences associated with differential modification of                Ssn6 overexpression significantly overlap with genes that are <br /> chromatin could therefore account for the differential roles of              deacetylated by Clr3 and Sir2. These data independently sup- <br /> Ssn6 and Tup11/12 in targeting genes for repression. To inves-               port the conclusion from Fig. 6B that Ssn6 and Tup11/12 <br /> tigate the genomic localization of the corepressor in relation to            selective target genes tend to be targets for regulation by Clr6, <br /> genome-wide patterns of histone acetylation, we clustered the                while genes affected by Ssn6 overexpression tend to be regu- <br /> IGR binding ratios for Ssn6, Tup11, and Tup12 with previously                lated by class II and class III HDACs. <br /> published acetylation profiles for a range of histone acetylation               The association of Clr6 with corepressor-regulated genes <br /> sites (H4K5Ac, H4K12Ac, H3K14Ac, H4K16Ac, and H4K9Ac)                        shows that Clr6 is frequently involved in repression of these <br /> corrected for H3 occupancy (51). Interestingly, a large cluster              genes, but it does not exclude the involvement of other <br /> of Ssn6-, Tup11-, and Tup12-bound IGRs shows a striking                      HDACs. We therefore investigated the effect of different <br /> correlation with a large proportion of the IGRs that are hy-                 HDACs on the expression of individual genes that were spe- <br /> poacetylated at one or more of the modification sites tested                 cifically deacetylated &#226;&#177;&#8211;2-fold by Clr6 and require Ssn6 and <br /> (Fig. 6A). Importantly, Fig. 6A also shows data for clusters of              Tup11/12 for their repression (see Table S5 in the supplemen- <br />  <br />  <br />  <br />  <br /> HDACs. Hierarchical two-dimensional expression cluster analysis of gene expression changes caused by ssn6HA-ts, tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;, and Ssn6 <br /> overexpression as well as mutations defective in different HDACs. The clustered genes also exhibit high binding in the IGR or ORF (&#226;&#177;&#8211;85th <br /> percentile) for each of Ssn6, Tup11, and Tup12 (n &#226;&#171;&#189; 334). The reciprocals of the Ssn6 overexpression data are shown to allow easier comparison <br /> with the other conditions. (C) Direct corepressor target genes depend on a subset of HDACs for deacetylation of histones in their upstream IGRs. <br /> The tables show the significance (P values) of overlaps between direct targets of Ssn6 (&#226;&#177;&#8211;1.5-fold expression change in Ssn6HA-ts, &#226;&#177;&#8211;75% Ssn6 IGR <br /> binding), Tup11/12 (&#226;&#177;&#8211;1.8-fold expression change in tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;, &#226;&#177;&#8211;75% Tup12 IGR binding), and by Ssn6 overexpression (&#226;&#177;&#8211;2.0-fold expression <br /> change, &#226;&#177;&#8211;75% Ssn6 IGR binding) with groups of genes that depend on different HDACs for deacetylation (&#226;&#177;&#8211;2-fold) of different histone residues <br /> within their IGRs (47). (D) The Clr6 HDAC plays the major role in the repression of individual genes that are directly targeted by Ssn6HA and <br /> Tup11/12 and are deacetylated by Clr6, but Clr3 and Hst4 are also involved in repression of some of the genes. The bar graph shows the expression <br /> changes resulting from defects caused by different mutant HDACs published previously (51) and for hst4&#226;&#338;&#172; (this study). A list of all Ssn6HA and <br /> Tup11/12 targets that are dependent on Clr6 for deacetylation (&#226;&#177;&#8211;2-fold, n &#226;&#171;&#189; 34) are listed in Table S5 in the supplemental material. (E) Genes <br /> dependent on Ssn6 (&#226;&#177;&#8211;1.5-fold expression change in Ssn6HA-ts, &#226;&#177;&#8211;75% IGR or ORF binding) or Tup11/12 (&#226;&#177;&#8211;1.8-fold expression change in tup11&#226;&#338;&#172; <br /> tup12&#226;&#338;&#172;, &#226;&#177;&#8211;75% IGR or ORF binding) and genes affected by Ssn6 overexpression (&#226;&#177;&#8211;2.0-fold expression, &#226;&#177;&#8211;75% IGR or ORF binding) are enriched <br /> in genes from different functional categories. Selected gene ontology process categories that are significantly enriched in gene sets targeted by Ssn6 <br /> and/or Tup11/12 are shown. The bars show the extent of category enrichment in the Ssn6 and Tup11/12 target groups relative to their genomic <br /> frequency. Significantly enriched categories (P &#226;&#172;&#65533; 0.05) are indicated (&#226;&#180;&#177;). The gene sets analyzed and the complete GO category analysis are <br /> presented in Table S6 in the supplemental material. <br />  1080             &#194;&#168; M-BILLAI ET AL. <br />          FAGERSTRO                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br /> tal material). Clr6 is the only HDAC that contributes signifi-      their correct regulation (21). However, 40% of Ssn6 targets <br /> cantly to the repression of most of these genes. Interestingly,     and 15% of Tup1 targets in this study do not overlap if a <br /> Hst4 and Clr3 played a significant role in repression of a          twofold threshold is used to select putative corepressor target <br /> minority of the genes. These are the same class II and class III    genes using these S. cerevisiae datasets. Thus, there may also be <br /> HDACs that were associated with corepressor targets affected        a difference in the sets of genes targeted by Ssn6 and Tup1 in <br /> by Ssn6 overexpression.                                             S. cerevisiae even though the difference is less pronounced than <br />    One explanation for the existence of selective gene targets      in S. pombe and C. albicans. <br /> for Ssn6 and Tup11/12 is that the selective aspects of corepres-       The identification of target genes that are selectively regu- <br /> sor function have coevolved together with the evolution of          lated by Ssn6 and Tup11/12 suggested the possibility that the <br /> genes involved in distinct cellular functions. To test this pos-    corepressor components could be recruited to some genes <br /> sibility, we sought to determine whether sets of Ssn6HA-ts-         independently of each other. The high degree of overlap be- <br /> and Tup11/12-dependent gene targets, as well as gene sets           tween the localization data for Ssn6, Tup11, and Tup12 suggest <br /> affected by Ssn6 overexpression were significantly enriched in      that this is not the case. The overlap is seen most clearly in data <br /> any of the gene ontology (GO) process terms. We found GO            for the higher resolution tiling arrays where changes in binding <br /> categories that are selectively and specifically overrepresented    of Ssn6, Tup11, and Tup12 between adjacent probe locations <br /> in the different groups as well as some categories that were        are highly correlated. For the vast majority of genes that are <br /> significantly enriched in all groups (Fig. 6E). Both the Ssn6-      selectively regulated by Ssn6 or Tup11/12, there is strong evi- <br /> and Tup11/12-dependent genes were selectively enriched in           dence for binding of all three components of the corepressor. <br /> functional categories involved in ion transport, conjugation,       Therefore, selective repression of gene targets by Ssn6 or <br /> and carbohydrate metabolism among other categories. Inter-          Tup11/12 must be accounted for in the context of recruited <br /> estingly, Tup11/12-dependent targets were selectively enriched      repressor complexes containing all three subunits. However, <br /> in the meiosis category but played a subsidiary role in the         the nature of genome-wide localization data makes it difficult <br /> cellular catabolism, alcohol metabolism, and amino carbohy-         to categorically exclude the existence of corepressor forms <br /> drate categories. A complete list of the three groups of genes      containing fewer than three of the subunits. Our data provide <br /> and the associated GO categories can be found in Table S6 in        little if any clear evidence for recruitment of Ssn6 in the ab- <br /> the supplemental material. We conclude that selective gene          sence of Tup11 or Tup12. There is a larger set of genes that <br /> targets dependent on Ssn6 and Tup11/12 and those affected by        may be associated with Tup11 and/or Tup12 in the absence of <br /> Ssn6 overexpression are enriched significantly for different        Ssn6. Interestingly, Tup1 has previously been shown to interact <br /> classes of genes that contain genes involved in distinct cellular   with the Mat&#226;&#65533;&#163;2 repressor protein independently of Ssn6 in S. <br /> processes.                                                          cerevisiae (28). Even though all three corepressor subunits are <br />                                                                     generally colocalized in the wild type, our gene-profiling results <br />                                                                     predict that Ssn6 should be recruited to Ssn6-selective genes in <br />                         DISCUSSION <br />                                                                     strains lacking Tup11 and Tup12. Tup11/12-independent re- <br />    In S. cerevisiae, Ssn6 has been shown to be a bridging com-      cruitment of Ssn6 is consistent with the known ability of Ssn6 <br /> ponent of the Ssn6-Tup1 complex that connects the repression        to interact directly with DNA-bound repressor proteins (see <br /> activity found in the Tup1 protein to DNA-bound repressor           the introduction), but the Tup11/12 dependency of its recruit- <br /> proteins in the regulatory regions of target genes (46). Our        ment will be tested directly in future studies. <br /> studies of Ssn6 in S. pombe provide important new insights into        Ssn6, Tup11, and Tup12 are localized in regions of the <br /> the function of Ssn6, which are likely to be significant for        genome that are relatively underacetylated on histone resi- <br /> understanding corepressors in other organisms, including S.         dues, most significantly H3-K14, H4-K5, H4-K12, and H4-K16. <br /> cerevisiae and higher eukaryotes. First, the essential nature of    The overlap between Ssn6- and Tup11/12-dependent genes <br /> the S. pombe Ssn6 protein suggests an active repression role of     and genes that are deacetylated and regulated by Clr6 suggests <br /> Ssn6 in addition to the repressive activity associated with         that Clr6 is the major HDAC regulating Ssn6- and Tup11/12- <br /> Tup11/12 because a strain lacking both Tup11 and Tup12 is           repressed genes. Indeed, Clr6 could be recruited to these genes <br /> viable. Since the Tup11/12-independent function of Ssn6 is          by the Ssn6-Tup11/12 corepressor, as has been shown for the <br /> likely to involve gene regulation, we used DNA microarray           analogous system in S. cerevisiae (50). Interestingly, histone <br /> analysis of a temperature-sensitive ssn6HA-ts strain to identify    H4-K5, which is underacetylated in Ssn6 and Tup11/12 target <br /> Ssn6 targets. Comparison of Ssn6-dependent target genes with        genes, does not seem to require Clr6 for deacetylation in these <br /> genes that require Tup11 and/or Tup12 for correct expression        genes. In S. cerevisiae, hypoacetylation of Ssn6-Tup1 target <br /> showed that many targets require both Ssn6 and Tup11/12             genes has been associated with the class II HDAC Hda1 (40). <br /> subunits for repression, while many others are selectively de-      In our data, the Hda1 orthologue in S. pombe, Clr3, and class <br /> pendent on either Ssn6 or Tup11/12. Thus, the Ssn6 and              III HDACs are required for deacetylation of acetylated his- <br /> Tup11/12 components of the corepressor appear to have dis-          tone residues (including H4-K5Ac) within corepressor target <br /> tinct but overlapping functions that are manifested by differ-      genes that are affected by Ssn6 overexpression as well as their <br /> ences in the phenotype of mutant strains as well as in their        repression. Although Clr6 is the only HDAC that is required <br /> target genes. Interestingly, similar findings have been reported    for repression of many Ssn6-, Tup11/12-, and Clr6-dependent <br /> for C. albicans where Ssn6 and Tup1 play different roles in         genes, a group of these genes are also derepressed by defects <br /> dimorphic growth (16, 22). DNA microarray studies of tup1&#226;&#338;&#172;          in Clr3 and the class III HDAC Hst4. To our knowledge, this <br /> and ssn6&#226;&#338;&#172; mutants in S. cerevisiae have shown a highly signif-      is the first report associating class III HDACs in the repression <br /> icant overlap in the target genes that require each protein for     of Ssn6-Tup1 targeted genes. <br />  <br />    We used Ssn6 overexpression as a complementary approach                        <br /> to identify corepressor target genes because the number of                    <br /> genes affected by ssn6HA-ts and the extent of the changes in                 <br /> their expression was much less than for tup11&#226;&#338;&#172; tup12&#226;&#338;&#172;. Al-            <br /> though the mechanism by which Ssn6 overexpression influ-                            <br /> ences corepressor activity is unclear, the genes identified show                 <br /> significant overlaps with Ssn6 and Tup11/12 genes as well as                      <br /> localization data for corepressor subunits. As for Ssn6- and                      <br /> Tup11/12-dependent genes, the analysis of corepressor target                      <br /> genes in this work is exclusively based on Ssn6 overexpression                 <br /> target genes for which there is strong independent evidence of                 <br /> corepressor binding. Interestingly, many corepressor binding                      <br /> target genes are up-regulated in response to Ssn6 overexpres-                <br /> sion. Although a range of mechanisms could account for this                 <br /> result, it is possible that the corepressor complex adopts a                 <br /> coactivator function under some conditions in S. pombe. Pre-               <br /> vious work with S. cerevisiae has suggested such a coactivator                <br /> role for the Tup1-Ssn6 complex (7, 27).                               <br />    Many previous studies have shown that the Ssn6-Tup1 core-              <br /> pressor complex is recruited to regulatory sites in the inter-              <br /> genic regions of the genome by DNA-bound repressor pro-                         <br /> teins. Consistently, we find the fission yeast corepressor              <br /> preferentially located in the upstream region of genes. In the                 <br /> tiling microarray data, we often see sharp peaks in the level of          <br /> corepressor, as would be expected for recruitment to specific                   <br /> upstream sites by repressor proteins. Surprisingly, however, we               <br /> also see binding of Ssn6, Tup11, and Tup12 to sites in the                    <br /> transcribed regions of genes. Some HDACs, such as Hos2 and               <br /> Rpd3, are also associated with the transcribed regions of genes                 <br /> (6, 49, 51). The RNA polymerase II mediator complex, which                           <br /> is also a target for Ssn6-Tup1-mediated repression in S. cerevi-                  <br /> siae, has also recently been identified in the transcribed region                 <br /> of genes in S. cerevisiae and S. pombe (2, 58). It is thus possible              <br /> that Ssn6-Tup11/12 recruits HDACs and/or the mediator com-                     <br /> plex to transcribed regions or that it is recruited by one of these               <br /> components to this location. The functional significance of                        <br /> Ssn6-Tup11/12 corepressors in transcribed regions will be the                     <br /> subject of future studies.                                                    <br /> </body></html>