<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br />  <br />         Histone Deacetylase Homologs Regulate Epigenetic Inheritance of <br />       Transcriptional Silencing and Chromosome Segregation in Fission Yeast <br />  <br />  <br />  <br />                                                                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Position-effect control at the silent mat2-mat3 interval and at centromeres and telomeres in fission yeast <br />              is suggested <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> be mediated through the assembly of heterochromatin-like structures. Therefore, trans- <br />              acting genes that affect silencing may encode either chromatin proteins, factors that modify them, or <br />              factors that affect chromatin assembly. Here, we report the identification of an essential gene, clr6 (c ryptic <br />              loci r egulator), which encodes a putative histone deacetylase that when mutated affects epigenetically <br />              maintained repression at the mat2-mat3 region and at centromeres and reduces the fidelity of chromosome <br />              segregation. Furthermore, we show that the Clr3 protein, when mutated, alleviates recombination block <br />              at mat region as well as silencing at donor loci and at centromeres and telomeres, also shares strong <br />              homology to known histone deacetylases. Genetic analyses indicate that silencing might be regulated by <br />              at least two overlapping histone deacetylase activities. We also found that transient inhibition of histone <br />              deacetylase activity by trichostatin A results in the increased missegregation of chromosomes in subsequent <br />              generations and, remarkably, alters the imprint at the mat locus, causing the heritable conversion of the <br />              repressed epigenetic state to the expressed state. This work supports the model that the level of histone <br />              deacetylation has a role in the assembly of repressive heterochromatin and provides insight into the <br />              mechanism of epigenetic inheritance. <br />  <br />  <br />  <br /> P    ROPAGATION of stable states of gene expression <br />     is thought to be critical for the development and <br /> maintenance of differentiated cell types. Examples in <br />                                                                            shire 1996), share many parallels with heterochromatic <br />                                                                            regions in higher eukaryotes. In fission yeast, the mat2 <br />                                                                            and mat3 loci, which serve as donors for mat1 alleles <br /> higher eukaryotes include X-chromosome inactivation                        during mating-type interconversion, are silenced even <br /> in female mammals (reviewed by Riggs and Porter                            though they contain genetic information identical to <br /> 1996), parental imprinting (reviewed by Barlow 1995;                       that of the expressed mat1-P or mat1-M alleles, respec- <br /> Ainscough and Surani 1996), and stable expression                          tively (reviewed by Klar et al. 1998). Furthermore, it <br /> of homeotic genes in Drosophila (Paro 1993). The                           was recently observed that an z11-kb recombinationally <br /> underlying mechanisms by which differentiated cells                        &#226;&#8364;&#339;cold&#226;&#8364;&#65533; region between the donor loci, referred to as the <br /> establish and maintain stable patterns of gene expres-                     K region, is also subject to position effects; i.e., expres- <br /> sion are not clear. Studies of position-effect variegation                 sion of the ura41 marker gene inserted in this region <br /> (PEV), a phenomenon in which heterochromatin vari-                         is repressed severely (Thon et al. 1994; Grewal and <br /> ably but stably silences nearby genes, have provided a                     Klar 1997). Mutations in six trans-acting genes [clr1, <br /> model for understanding the propagation of committed                       clr2, clr3, clr4 (c ryptic loci r egulator), swi6, and rik1] <br /> states of gene expression. It was recently suggested that                  partially derepress the donor loci and alleviate the re- <br /> polycomb-mediated stable inactivation of homeotic                          pression of ura41 at different locations within the mat2- <br /> genes and PEV in Drosophila are related phenomena                          mat3 interval (Egel et al. 1989; Lorentz et al. 1992; <br /> (Orlando and Paro 1995), supporting the idea that                          Thon and Klar 1992; Ekwall and Ruusala 1994; <br /> position effects are mediated through long-range con-                      Thon et al. 1994; Grewal and Klar 1996, 1997), sug- <br /> trol exerted by repressive chromatin structures.                           gesting that these gene functions affect this region in <br />   Silenced genomic regions in the distantly related                        a global fashion. These genes are also shown to prohibit <br /> yeasts Saccharomyces cerevisiae and Schizosaccharomyces                    meiotic and mitotic recombination in the entire mat2- <br /> pombe, such as silent donor mating-type loci (reviewed                     mat3 region (Egel et al. 1989; Klar and Bonaduce <br /> by Klar 1989), telomeres (Gottschling et al. 1990;                         1991; Lorentz et al. 1992; Thon and Klar 1992; Thon <br /> Nimmo et al. 1994), and centromeres (reviewed by All-                      et al. 1994). <br />                                                                               All six trans-acting genes, originally identified for their <br />                                                                            role in silencing at the mating-type region, were subse- <br />   Corresponding author: Amar J. S. Klar, Gene Regulation and Chromo-       quently shown to affect position-effect control of expres- <br /> some Biology Laboratory, NCI&#226;&#8364;&#8220;Frederick Cancer Research and Devel- <br /> opment Center, ABL&#226;&#8364;&#8220;Basic Research Program, P.O. Box B, Frederick,          sion of a marker gene integrated at centromere 1 (cen1; <br /> MD 21702-1201. E-mail: klarmail@ncifcrf.gov                                Allshire et al. 1995), indicating their involvement in <br />  <br /> Genetics 150: 563&#226;&#8364;&#8220;576 (October 1998) <br />  564                                     S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br /> the functional organization of centromeres. A mecha-               scribed by Allshire et al. (1995). The construction of HM248 <br /> nistic link between centromeric and mating-type region             and PRZ119 was described by Niwa et al. (1986) and Reynolds <br />                                                                    et al. (1990), respectively. The KD::ura41 allele was constructed <br /> silencing is further suggested by our recent findings              as part of an earlier study (Grewal and Klar 1996). We used <br /> that approximately one-third of the K region is homolo-            standard genetic crosses to construct all other strains. <br /> gous to the centromeric repeat sequences (Grewal and                  Standard conditions were used for growth, sporulation, <br /> Klar 1997). Furthermore, analogous to the existence                random-spore analysis, tetrad analysis, gap repair, transfor- <br /> of an epigenetic component in centromere function                  mations, and construction of diploids <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> haploid cells <br />                                                                    (Moreno et al. 1991). YEA and PMA (supplemented with <br /> (Steiner and Clarke 1994; Ekwall et al. 1997), silenc-             appropriate amino acids) were used as rich and sporulation <br /> ing, mating-type interconversion, and recombinational              media, respectively. Synthetic medium lacking an appropriate <br /> suppression at the mating-type region are also regulated           <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> such as uracil (AA-URA) was used in most of the <br /> by an epigenetic mechanism (Grewal and Klar 1996).                 experiments. FOA medium contains 5-fluoroorotic acid (0.8 <br /> Many studies have suggested that centromeres and the               g/liter) in synthetic complete medium. Ura1 and Ura2 deriva- <br />                                                                    tives of KD::ura41 strains were obtained by selecting for their <br /> mating-type locus are assembled into a higher-order                growth on AA-URA or FOA medium, respectively. Quantita- <br /> chromatin structure refractory to transcription and to             tion of ura41 expression was carried out by growing cells on <br /> recombination machinery, yet recombination required                synthetic Edinburgh minimal medium containing glutamate, <br /> for mat1 switching is allowed. Similarly, the clr1-clr4, swi6,     adenine, leucine, and uracil, each at 75 mg/liter (N/S), URA2 <br /> and rik1 gene products are believed to facilitate the              (N/S lacking uracil), and FOA (N/S supplemented with <br />                                                                    FOA). The temperature and ultraviolet radiation (UV) sensi- <br /> assembly of a heterochromatin-like structure, which,               tivities were determined as described in Figure 1. <br /> once assembled, is capable of self-templating (Grewal                 Iodine-staining assay: We used the iodine-staining assay to <br /> and Klar 1996). Consistent with the chromatin model,               estimate the efficiency of mating-type switching and the level <br /> Swi6 (Lorentz et al. 1994) and Clr4 (Ivanova et al.                of derepression of donor loci. Individual colonies grown on <br /> 1998) proteins share homology with the chromodomain                sporulation medium for 3 days were exposed to iodine vapors. <br />                                                                    Synthesis of a starch-like compound by sporulating cells results <br /> motif found in heterochromatin-associated proteins in              in black staining of colonies after exposure to iodine vapors <br /> higher eukaryotes (reviewed by Singh 1994), and Swi6               (Bresch et al. 1968). In switching-competent cells, the inten- <br /> colocalizes with centromeres, telomeres, and the mat-              sity of iodine staining indicates the efficiency of mating-type <br /> ing-type region (Ekwall et al. 1995). The exact mecha-             switching; in a nonswitching background, it indicates the level <br /> nism of the assembly of heterochromatin-like structures            of &#226;&#8364;&#339;haploid meiosis,&#226;&#8364;&#65533; a <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> resulting from concurrent <br />                                                                    expression of both P and M information in a haploid cell <br /> is not known. However, genetic and biochemical evi-                (Thon and Klar 1992). <br /> dence from other systems suggests that the acetylation                Isolation of the clr6-1 <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> strain: SP1167 containing <br /> status of conserved lysine residues in the amino-terminal          the nonswitchable mat1-Msmto (Engelke et al. 1987) allele and <br /> tails of histones H3 and H4 plays a role in chromatin              the mutant clr1-5 allele displays only partial derepression of <br /> assembly and transcription regulation (see Braunstein              the mat2-P locus (Thon et al. 1994). Therefore, when grown <br />                                                                    on sporulation (PMA1) medium, SP1167 colonies showed <br /> et al. 1993; Brownell and Allis 1996; Wolffe 1996; <br />                                                                    weak haploid meiosis and, hence, stained very lightly with <br /> Hartzog and Winston 1997; Pazin and Kadonaga                       iodine vapors. To identify new mutants, SP1167 was mutagen- <br /> 1997).                                                             ized with ethylmethanesulfonate to 50% viability by following <br />   Independent of any perceived molecular model of                  the procedure of Moreno et al. (1991). The mutagenized cells <br /> repression, we have used a genetic approach to identify            were plated onto PMA1 medium and allowed to grow at 258 <br />                                                                    for 6 days. Several mutants showing dark iodine staining and <br /> trans-acting genes essential for silencing. A new locus, <br />                                                                    high levels of haploid meiosis were isolated. One mutant, <br /> clr6, was identified that encodes a putative histone               clr6-1, conferred a growth defect at 378 and was investigated <br /> deacetylase. We have investigated the effects of clr6-1            further. <br /> mutation on silencing at the mating-type region and                   Cloning and sequencing: To clone the clr6 gene and the <br /> at centromeres and telomeres. We also examined the                 previously identified clr3 (Thon et al. 1994) gene, we trans- <br /> effects of general inhibition of histone deacetylation on          formed SP1359 and SP1195, respectively, with the S. pombe <br />                                                                    partial Sau3A genomic library cloned in the pWH5 vector <br /> epigenetic inheritance and demonstrated that transient             (Wright et al. 1986). The vector contains the S. cerevisiae LEU2 <br /> inhibition by the drug trichostatin A (TSA) alters the             gene, which upon transformation, can complement the leul- <br /> imprint at the mat2-mat3 region. We also describe the              32 defect in S. pombe. The screening of Leu 1 transformants <br /> molecular characterization of the previously identified            identified plasmids containing 8.0-kb (pAK86) and 5.0-kb <br /> clr3 gene and discuss the implications of our findings for         (pClr6) inserts that were able to complement the recessive <br />                                                                    clr3-735 and clr6-1 alleles, respectively. Upon retransformation, <br /> current models of structural organization at the mating- <br />                                                                    both plasmids complemented mutant phenotypes of the re- <br /> type region and at centromeres in S. pombe, as well as             spective strains. To confirm that cloned DNA contained wild- <br /> for the mechanism of epigenetic inheritance in general.            type copies of clr3 and clr6 rather than extragenic suppressors, <br />                                                                    the plasmids were integrated by homologous recombination <br />                                                                    and analyzed for linkage. The plasmid-borne LEU2 marker <br />               MATERIALS AND METHODS                                showed tight linkage to the respective loci because only the <br />                                                                    parental ditype segregation <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> was found in all the tetrads <br />   Strains and culture conditions: Most of the S. pombe strains     we analyzed (69 tetrads for clr3 and 44 tetrads for clr6 locus). <br /> used in this study were constructed in our laboratory (Table          The nucleotide sequence of the subcloned genomic frag- <br /> 1). Strains FY 336, FY498, FY 520, and FY 648 have been de-        ments was determined by using the Prism Ready Reaction Dye <br />                                                  Histone Deacetylation                                                    565 <br />  <br />                                                         TABLE 1 <br />                                                     S. pombe strains <br />  <br /> Strain       ura41 insertion            mat1                 Auxotrophic markers/Ch16                       clr loci <br /> SP412                             mat1-M mat2,3D     leu1-32 <br /> SP837                             h90                leu1-32 ura4-D18 ade6-216 <br /> SP1001                            mat1-Msmt-o        his2 ade6-216 <br /> SP1005                            mat1-PD17::LEU2    leu1-32 ura4 ade6-216 <br /> SP1167   mat2-Pint::ura41         mat1-Msmt-o        ura4-D18 ade6-216                            clr1-5 <br /> SP1173                            mat1-Msmt-o        leu1-32 ura4-D18 his2 ade6-216 <br /> SP1195   mat2-Pint::ura41         mat1-Msmt-o        ura4-D18 ade6-216                            clr1-5 clr6-1 <br /> SP1240   mat2-Pint::ura41         mat1-Msmt-o        leu1-32 ura4-D18 ade6-216                    clr6-1 <br /> SP1242   mat2-Pint::ura4          h1                 leu1-32 ura4-D18 ade6-210                    clr6-1 <br /> SP1249   mat2-Pint::ura41         mat1-PD17::LEU2    leu1-32 ura4-D18 ade6-216                    clr1-5 clr6-1 <br /> SP1251   mat2-Pint::ura41         mat1-Msmt-o        leu1-32 ura4-D18 ade6-210                    clr1-5 clr6-1 <br /> SP1286   mat2-Pint::ura41         mat1-Msmt-o        ura4-D18 ade6-210                            clr3-735 clr6-1 <br /> SP1302   mat3-Mint::ura41         h90                leu1-32 ura4-D18 ade6-210 <br /> SP1306   mat2-Pint::ura41         mat1-Msmt-o        leu1-32 ura4 ade6-210                        clr1-5 clr6-1::LEU2::clr61 <br /> SP1359   mat3-Mint::ura41         h90                leu1-32 ura4-D18 ade6-210                    clr3-735 <br /> SP1464   KD::ura41                h90                ura4-D18 ade6-210                            clr6-1 <br /> SPG27    KD::ura41                h90                leu1-32 ura4 ade6-210 his2 <br /> SPG32    KD::ura41                mat1-Msmt-o        leu1-32 ura4 ade6-210 <br /> SPG51    KD::ura41                h90                leu1-32 ura4 ade6-216 <br /> SPG60    KD::ura41                h90                leu1-32 ura4 his2 ade6-216                   clr1-5 <br /> SPG62    KD::ura41                h90                leu1-32 ura4 his2 ade6-210                   clr3-735 <br /> SPG133   mat2-Pint::ura41         mat1-PD17::LEU2    ura4 ade6-216                                clr6-1 <br /> SPG134                            mat1-PD17::LEU2    ura4 ade6-210 (Ch16 ade6-M216)               clr1-5 <br /> SPG135   mat2-Pint::ura41         mat1-PD17::LEU2    leu1-32 ura4 ade6-210 (Ch16 ade6-216)        clr6-1 <br /> SPG136   mat2-Pint::ura41         mat1-Msmt-o        ura4 ade6-210 (Ch16 ade6-216)                clr1-5 clr6-1 <br /> SPG137   KD::ura41                h90                ura4 his2 ade6-210                           clr1-5 clr6-1 <br /> SPG138   cnt1/ TM1(NcoI)::ura41   h1                 leu1-32 ura4 ade6-216                        clr6-1 <br /> SPG139   cnt1/ TM1(NcoI)::ura41   h90                leu1-32 ura4 his2 ade6-216                   clrl-5 <br /> SPG140   cnt1/ TM1(NcoI)::ura41   h1                 leu1-32 ura4 ade6-216                        clr1-5 clr6-1 <br /> SPG141   imr1R(NcoI)::ura41       h1                 leu1-32 ura4 ade6-210                        clr6-1 <br /> SPG142   imr1R(NcoI)::ura41       h1                 leu1-32 ura4 ade6-216                        clr1-5 <br /> SPG143   imr1R(NcoI)::ura41       h1                 leu1-32 ura4 ade6-210                        clr1-5 clr6-1 <br /> SPG144   otr1R(SphI)::ura41       h1                 leu1-32 ura4 ade6-216                        clr6-1 <br /> SPG145   otr1R(SphI)::ura41       h1                 leu1-32 ura4 ade6-210                        clr1-5 <br /> SPG146   otr1R(SphI)::ura41       h90                leu1-32 ura4 his2 ade6-210                   clr1-5 clr6-1 <br /> SPG147   Ch16m23::ura41-TEL       h1                 leu1-32 ura4 ade6-210 (Ch16 ade6-216)        clr6-1 <br /> SPG148   Ch16m23::ura41-TEL       h1                 leu1-32 ura4 ade6-210 (Ch16 ade6-216)        clr1-5 <br /> SPG149   Ch16m23::ura41-TEL       h90                leu1-32 ura4 his2 ade6-210 (Ch16 ade6-216)   clr1-5 clr6-1 <br /> SPG155   mat2-Pint::ura41         mat1-PD17::LEU2    leu1-32 ura4 ade6-210 (Ch16 ade6-216)        clr3-735 <br /> SPG156                            mat1-M mat2,3D     leu1-32 ura4 ade6-210 (Ch16 ade6-216)        clr3-735 clr6-1 <br /> SPG157   KD::ura41                h90                ura4 his2 ade6-210                           clr3-735 clr6-1 <br /> SPG158   imr1R(NcoI)::ura41       h1                 leu1-32 ura4 ade6-210                        clr3-735 <br /> SPG159   imr1R(NcoI)::ura41       h1                 leu1-32 ura4 ade6-216                        clr3-735 clr6-1 <br /> SPG160   otr1R(SphI)::ura41       h1                 ura4 ade6-216                                clr3-735 <br /> SPG161   otr1R(SphI)::ura41       h1                 leu1-32 ura4 ade6-210                        clr3-735 clr6-1 <br /> SPG162   cnt1/TM1(NcoI)::ura41    h1                 leu1-32 ura4 ade6-216                        clr3-735 <br /> SPG163   cnt/TM1(NcoI)::ura41     mat1-M mat2,3D     leu1-32 ura4 ade6-210                        clr3-735 clr6-1 <br /> PG383                             mat1-PD17::LEU2    leu1-32 ura4-D18 ade6-210                    clr1-5 <br /> FY 336   cnt1/ TM1(NcoI)::ura41   h1                 leu1-32 ura4-DS/E ade6-210 <br /> FY498    imr1R(NcoI)::ura41       h1                 leu1-32 ura4-DS/E ade6-210 <br /> FY 520   Ch16m23::ura41-TEL       h1                 leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) <br /> FY 648   otr1R(SphI)::ura41       h1                 leu1-32 ura4-DS/E ade6-210 <br /> HM248                             h2                 his2 ade6-210 (Ch16 ade6-216) <br /> PRZ119                            h1                 leu1-32 ura4-D18 lys1-131 ade6-210           rhp6-1D <br />  566                                       S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br /> Deoxy Terminator Cycle Sequencing Kit (Applied Biosystems,           mutations in all the known trans-acting (clrl-clr4 and <br /> Foster City, CA). Samples were applied to the Applied Biosys-        swi6) genes (Thon et al. 1994). Therefore, it is likely <br /> tems model 373A sequencing system. Data were analyzed by <br /> using GCG software (University of Wisconsin). To determine           that there exist additional trans-acting factors that are <br /> the location of introns, cDNA was amplified from the S. pombe        involved in silencing. To identify the hypothesized addi- <br /> cDNA library and sequenced. The nucleotide sequences of              tional factors, we mutagenized a nonswitchable mat1- <br /> the clr6 and clr3 genes have been deposited in the GenBank           Msmto strain carrying the mutant clr1-5 allele (SP1167) <br /> database (accession numbers AFO64206 and AFO64207, re-               and sought mutants exhibiting strong derepression of <br /> spectively). <br />    Construction of the clr6 disruption allele: A 0.9-kb fragment     the mat2-P locus. We obtained several trans-acting muta- <br /> containing part of the clr6 gene was subcloned into pBS/KS           tions that increased haploid meiosis, a phenotype that <br /> (Stratagene, La Jolla, CA). The ura41 gene in a 1.8-kb HindIII       occurs in haploid cells expressing both P and M infor- <br /> fragment was then inserted into the HindIII site to construct        mation (Thon and Klar 1992) and causes increased <br /> pBS/clr6::ura41. A 2.7-kb XhoI-PstI insert containing the dis-       iodine staining of colonies formed by nonswitching <br /> rupted clr6 open reading frame (ORF) was used to transform <br /> the clr61/clr61 ura4/ura4 diploid strain to uracil prototrophy.      strains. Here we present the analysis of one of these <br /> Transformants with the correct integration were identified by        mutations, clr6-1, which, according to our linkage analy- <br /> Southern analysis.                                                   sis, was distinct from previously identified trans-acting <br />    DNA and RNA analyses: DNA from S. pombe cultures grown            mutations and caused increased haploid meiosis (Figure <br /> overnight was prepared as described by Moreno et al. (1991),         1A). Analysis of the clr6-1 mutant cells revealed that the <br /> and RNA was prepared as described by Schmitt et al. (1990). <br /> Southern and Northern hybridizations were carried out as             mutation caused a temperature-sensitive (Ts2) growth <br /> described by Sambrook et al. (1989). The probes used in this         defect at 378 (Figure 1B). Interestingly, the clr6-1 mutant <br /> study were as follows: a 1.8-kb HindIII fragment containing          cells did not show the Ts2 growth defect in the first <br /> ura41, a 0.9-kb fragment containing part of the clr6 gene, a         five to six cell divisions after shifting to the restrictive <br /> 1.0-kb Bcl I-TaqI DNA fragment carrying part of the M cassette,      temperature of 378. In addition to the Ts 2 phenotype, <br /> and a NdeI-PstI fragment containing cdc2. <br />    Fluctuation test: The fluctuation test was carried out as         we also found that the mutant cells displayed increased <br /> described by Luria and Delbruck (1943). The Ura2 and                 UV sensitivity (Figure 1C). <br /> Ura1 derivatives of SPG27 and SP1464 were diluted to a con-             The observed increase in haploid meiosis could be <br /> centration of about six cells per milliliter in YEA nonselective     caused by a defect in the pat1 or mei2 genes, which <br /> liquid medium. Aliquots (100-ml) of cultures were dispensed          control meiosis and whose mutants confer this pheno- <br /> into microtiter plates and allowed to grow at 308 for 24 hr <br /> before plating onto YEA plates. After 3 days at 308, the colonies    type, even in the absence of donor loci (reviewed by <br /> were replicated onto PMA1 sporulation and AA-URA media.              Egel 1994). Alternatively, the increase in haploid meio- <br /> Iodine staining and KD::ura41-expression phenotypes were             sis could result from derepression of donor loci. We <br /> scored after 3 days, and the rate of appearance of the variant       have differentiated between these possibilities. First, ge- <br /> (m) was calculated by using the formula m 5 2[ln(B/A)]/N,            netic analysis showed that the haploid meiosis pheno- <br /> where N is the average number of colony-forming units per <br /> culture, B is the number of cultures without variation, and A <br />                                                                      type was dependent on the mat2-P and mat3-M donor <br /> is the total number of cultures in the experiment.                   loci (data not shown). Second, Northern analysis (Fig- <br />    Chromosome segregation assay: The chromosome loss ex-             ure 2) revealed an increase in the mat3 transcript level <br /> periment was carried out as described by Allshire et al.             in the clr1-5 clr6-1 double mutant as compared with <br /> (1995). Cells from Ade1 colonies were plated onto adenine-           clr1-5 alone, indicating that the clr6 gene regulates si- <br /> limited medium (Y E) and incubated at 308 for 4 days. If chro- <br />                                                                      lencing of mat3. Furthermore, the clr6-1 mutation, origi- <br /> mosome loss occurs in the first division of a cell plated on <br /> YE, half of the resultant colony carrying Ch16 will be white,        nally isolated for its effect on mat2-P, also derepresses <br /> whereas the other half without Ch16 will be red. The number          mat3-M in a clr1 background. To test whether the <br /> of half-sectored red/white colonies was determined, and the          clr6-1 mutation had a cumulative effect with clr1-5 or <br /> rate of chromosome loss per cell division was calculated by          was epistatic to clr1 function, we constructed a clr11 <br /> dividing the number of half-sectored colonies by the total           clr6-1 strain. As expected for a mutation that affects <br /> number of white colonies plus half-sectored colonies. For TSA <br /> treatment, 2 ml of TSA [10 mg/ml dissolved in dimethylsulfox-        only part of the redundant or overlapping silencing <br /> ide (DMSO)] was spotted onto filter paper discs (1-cm diame-         mechanisms, the clr6-1 allele by itself had no visible <br /> ter) laid over a lawn of cells (HM248) spread on a plate. The        effect on the silencing of mat2 and mat3 (Figures 1A <br /> plates were incubated at 338 for different time intervals, and       and 2). We also analyzed the cumulative effect of the <br /> cells growing underneath the filters were plated onto YE me-         clr6-1 allele with mutations in clr2, clr3, clr4, and swi6, <br /> dium. <br />                                                                      which are suggested to work in a single pathway in <br />                                                                      conjunction with clr1, as pairwise combinations do not <br />                                                                      show cumulative effect (Thon et al. 1994). Surprisingly, <br />                            RESULTS <br />                                                                      the clr6-1 allele caused increased haploid meiosis only in <br />   Isolation and characterization of the clr6-1 mutation:             the clr3 mutant background (SP1286; data not shown). <br /> Our past studies have indicated that silencing of the                Furthermore, the clr6-1 mutation did not show a cumula- <br /> donor mating-type loci is regulated by redundant mech-               tive effect when combined with mat2-P-proximal dele- <br /> anisms and that deletion of a 1.5-kb cis-acting mat2-P-              tion. We explain these findings by suggesting that Clr6 <br /> proximal region potentiates the derepressing effect of               acts in an overlapping manner with Clr1/Clr3 and that <br />                                                         Histone Deacetylation                                                     567 <br />  <br />  <br />  <br />  <br />                                                                        Figure 2.&#226;&#8364;&#8221;Northern analysis showing derepression of <br />                                                                     mat3-M in a mutant background. (A) Diagrammatic represen- <br />                                                                     tation of the M-cassette transcripts. The direction of transcrip- <br />                                                                     tion and size of Mc (constitutive) and Mi (inducible) tran- <br />                                                                     scripts are indicated. M-probe indicates the DNA fragment <br />                                                                     used as a probe in the Northern blot experiment shown in <br />                                                                     B. (B) Northern blot analysis of RNA from nitrogen-starved, <br />                                                                     nonswitching cells containing mat2-P and mat3-M. To directly <br />                                                                     monitor derepression of mat3-M, nonswitchable mat1-PD17 <br />                                                                     (Arcongioli and Klar 1991) allele-containing strains were <br />                                                                     used. The WT (wild-type) mat1-Msmto (SP1001), a positive <br />                                                                     control for identifying M transcripts originating from the ex- <br />                                                                     pressed mat1 locus, and PD17 (SP1005) strains were used as <br />                                                                     controls of the clr 1 genotype. The other strains used were as <br />                                                                     follows: SPG133, clr6-1; PG383, clr1-5; and SP1249, clr1-5 <br />                                                                     clr6-1. The cdc2 mRNA was included as an internal loading <br />                                                                     control. <br />    Figure 1.&#226;&#8364;&#8221;The clr6-1 mutant phenotypes. (A) Cumulative <br /> effect of clr6-1 and clr1-5 mutations on mat2-P derepression. <br /> The colonies of nonswitchable M cells (mat1-Msmto) were spor- <br /> ulated and exposed to iodine vapors before photography. <br />                                                                     both these functions feed into a silencing pathway in- <br /> Because mat1-Msmto strains are prohibited from switching,           volving clr2, clr4, and swi6 products. <br /> their intensity of staining indicates the level of haploid meio-       Cloning and sequencing of clr6 reveals a similarity <br /> sis within the colonies, qualitatively reflecting the level of      to histone deacetylases: Because the clr6-1 mutant is <br /> mat2-P derepression. In cells of this genotype, derepression        temperature sensitive, it is likely that the lethality is <br /> of mat3-M cannot be determined by the staining assay. The <br /> strains used and their relevant genotypes were as follows: <br />                                                                     caused by pleiotropic effects on diverse cellular func- <br /> SP1173, WT; SP1167, clr1-5; SP1240, clr6-1; and SP1195, clr1-5      tions. An important clue to clr6 function was provided <br /> clr6-1. (B) Temperature sensitivity of clr6-1. The strains pre-     by its nucleotide sequence. For this purpose, we isolated <br /> viously grown under permissive growth conditions (308) were         a clone from a partial Sau3A genomic library of S. pombe <br /> replicated onto YEA plates and incubated overnight at 308 or        by complementing Ts2 growth and the haploid meiosis <br /> 378. clr6-1 displayed a gradual Ts2 growth defect. The cells <br /> grown overnight were further replicated onto Y EA and incu- <br />                                                                     defects in the mat1-Msmto clr1-5 clr6-1 (SP1195) mutant <br /> bated at the respective temperatures for 2 days before photog-      strain. A 5.0-kb insert complemented both defects. Se- <br /> raphy. The strains used were SPG27, WT and SP1464, clr6-            quencing of the genomic clr6 and its cDNA clones iden- <br /> 1. (C) UV sensitivity. Patches of the indicated strains were        tified an ORF that is interrupted by 13 introns. (Figure <br /> replicated onto Y EA plates in duplicate, and one of the plates     3A). The ORF is capable of encoding a protein of 405 <br /> was UV irradiated (200 mJ/cm2 for 2 min). Plates were incu- <br /> bated in the dark at 308 for 4 days before photography. As a <br />                                                                     amino acids (Figure 3B). Comparison of the predicted <br /> control, cultures grown without UV treatment are shown. The         Clr6 amino acid sequence with the GenBank database <br /> strains used were as follows: SP1302, WT; SP1251, clr1-5 clr6-      revealed that Clr6 shares strong homology to members <br /> 1; SP1306, clr1-5 clr6-1::LEU2::clr61 (a strain that is isogenic    of a recently described family of histone deacetylases, <br /> with SP1251 but carries clr61 integrated next to the clr6-1         such as human HDAC1 (formerly HD1; Taunton et al. <br /> allele); SP1242, clr6-1; and PRZ119, rhp6-1D (a known UV- <br /> sensitive control strain). <br />                                                                     1996; 61% identity and 78% similarity) and S. cerevisiae <br />                                                                     Rpd3 (Vidal and Gaber 1991; 63% identity and 77% <br />  568                                   S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br />  <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;The clr6 gene. (A) The clr6 ORF. Thirteen introns interrupting the ORF and the filled boxes indicating exons are <br /> shown. (B) Comparison of the predicted Clr6, S. serevisiae Rpd3, and human HDAC1 amino acid sequences. The sequences were <br /> aligned using the PILEUP program. The consensus sequence indicating identical residues in all three sequences is shown at the <br /> bottom. The asterisk indicates the residue that is changed to aspartic acid [D] in the clr6-1 mutant allele. <br />  <br />  <br /> similarity; Figure 3B). Clr6 also shares similarity to the       the clr6-1 Ts2 strain when it was shifted to the restrictive <br /> predicted amino acid sequences of two recently se-               temperature (see above). Apparently, the clr6-1 allele <br /> quenced ORFs from S. pombe (Sanger Genome Sequenc-               must cause a partial loss of function because the deletion <br /> ing Center), which we have designated homologs of clr6           is lethal. <br /> (hoc1 and hoc2 mapping to cosmids c3G9 and SP41410,                 Mutation in clr6 affects propagation of the epigenetic <br /> respectively). The sequence alignments revealed that             states regulating silencing, recombinational suppres- <br /> Clr6 shares greater homology to Hoc1 (50% identity               sion, and directionality of switching: It has been shown <br /> and 73% similarity) than to Hoc2 (26% identity and               that in a strain with part of the K region replaced by <br /> 51% similarity).                                                 the ura41 marker gene (KD::ura41 allele), ura41 expres- <br />    To map the mutation in the clr6-1 mutant, we used             sion, recombination between flanking markers, and the <br /> PCR to amplify the genomic clr6 DNA containing clr6              efficiency of mating-type interconversion are covarie- <br /> from wild-type and mutant cells. The sequencing of the           gated and regulated by an epigenetic mechanism <br /> PCR products identified a single G-to-A transition in            (Grewal and Klar 1996, 1997). More importantly, <br /> codon 269, converting GGC (Gly) to GAC (Asp). Inter-             Ura2 efficient switching and Ura1 inefficient switching <br /> estingly, the mutation maps to a region of Clr6 that is          variegated states (referred to as Ura2 and Ura1 states, <br /> conserved in Rpd3 from S. cerevisiae and human HDAC1.            respectively) are inherited in cis and proposed to be <br /> The genomic location of clr6 was determined as de-               the result of specific chromatin organization, with Ura1 <br /> scribed by Hoheisel et al. (1993). The results revealed          cells being defective in assembly of the structure <br /> that clr6 maps near the telomere of the short arm of             (Grewal and Klar 1996). These results have been sub- <br /> chromosome II between markers mei3 and rad3. A dis-              sequently confirmed (Thon and Friis 1997) and sup- <br /> ruption allele of clr6 was made by DNA-mediated trans-           port our suggestion that deletion of part of the K region <br /> formation of a diploid strain (see materials and meth-           compromises the establishment of a heterochromatin- <br /> ods). Upon dissection of the heterozygous diploid,               like repressive structure (Grewal and Klar 1996). <br /> haploid segregants carrying the clr6 <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>null allele</span> were               Here, we determined whether a mutation in clr6, a <br /> found to be essentially inviable (data not shown). How-          histone deacetylase homolog, affects propagation of the <br /> ever, microscopic examination revealed that these hap-           epigenetic states. By genetic crosses, the clr6-1 mutation <br /> loid cells were able to divide for three to four genera-         was combined with derepressed (Ura1 ) and repressed <br /> tions before growth arrest, a phenotype similar to that of       (Ura2) &#226;&#8364;&#339;epialleles&#226;&#8364;&#65533; of KD::ura41 . First, donor loci re- <br />                                                         Histone Deacetylation                                                      569 <br />  <br />                                                                                                          Figure 4.&#226;&#8364;&#8221;The clr6-1 <br />                                                                                                       mutation affects the propa- <br />                                                                                                       gation of the transcriptional <br />                                                                                                       states in KD::ura41 cells. <br />                                                                                                       The line drawing shows the <br />                                                                                                       arrangements of the mating- <br />                                                                                                       type region in KD::ura41 <br />                                                                                                       strains. H indicates the Hin- <br />                                                                                                       dIII site. The Ura2 state was <br />                                                                                                       combined with different mu- <br />                                                                                                       tant alleles by genetic crosses, <br />                                                                                                       and ura41 expression was as- <br />                                                                                                       sayed by dilution analysis. <br />                                                                                                       The cells were suspended in <br />                                                                                                       water, and fivefold serial di- <br />                                                                                                       lutions were spotted onto <br />                                                                                                       the indicated media. The <br />                                                                                                       plates were incubated at 308 <br />                                                                                                       for 3&#226;&#8364;&#8220;4 days before photog- <br />                                                                                                       raphy. Strains carrying ei- <br />                                                                                                       ther a deletion of the ura4 <br />                                                                                                       gene (SP837) or the func- <br />                                                                                                       tional ura4 1 gene at an in- <br />                                                                                                       digenous chromosomal lo- <br />                                                                                                       cation (SP412) were used as <br />                                                                                                       controls. The strains used <br /> were as follows: SPG27, WT; SP1464, clr6-1; SPG60, clr1-5; SPG137, clr1-5 clr6-1; SPG62, clr3-735; and SPG157, clr3-735 clr6-1. All <br /> strains containing KD::ura41 carried the mutation of the indigenous ura41 gene. WT indicates the clr1 genotype. <br />  <br />  <br /> mained silent in cells carrying either the Ura1 or Ura2               gation of epigenetic states. As a control, KD::ura41 <br /> epistate. Second, comparison of the rate of Ura2 to                   strains carrying a mutation in clr1 or clr3, which were <br /> Ura1 transition in the clr6-1 with the rate in the clr61              previously shown to completely suppress variegation <br /> cells by dilution assay revealed that the mutation by                 (Grewal and Klar 1996), were also included. Dilution <br /> itself did not significantly affect the mitotic stability of          analysis (Figure 4) demonstrated that like the clr1 or <br /> the Ura2 state (Figure 4). We also used the fluctuation               clr3 mutant strains, double mutants did not grow on <br /> test to quantitatively measure the effect of the clr6 muta-           FOA plates, indicating alleviation of ura41 repression. <br /> tion on the stability of epigenetic states (see materials             More significantly, Ura1 cells with the double mutation <br /> and methods). Confirming the results of serial dilution               were unable to revert to the Ura2 state. Overall, these <br /> (Figure 4), the clr6 mutation had only a subtle effect                data suggest that mutation in clr6 leads to increased <br /> on the Ura2 to Ura1 transition, as compared with its                  conversion of the transcribed state into the repressed <br /> wild-type counterpart (Table 2). However, we observed                 state, presumably through assembly of a higher-order <br /> that the clr6-1 allele caused a significant (z20-fold) in-            chromatin structure, and that increased repression in <br /> crease in Ura1 to Ura2 conversion. This result shows                  the clr6 mutant is dependent on the clr1 and clr3 gene <br /> that the mutation paradoxically increases the efficiency              products. <br /> of the establishment of a transcriptionally repressed                    TSA treatment adversely affects propagation of the <br /> state of KD::ura41.                                                   epigenetic imprint at the mat2-mat3 region: Parallel to <br />    The silencing function of clr6 seems to overlap with               the clr6-1 analysis, we also investigated the effect of inhi- <br /> that of clr1 and clr3; therefore, we tested the effect of             bition of histone deacetylation on propagation of the <br /> the clr1 clr6 and clr3 clr6 double mutations on the propa-            Ura2 state by treating cells with TSA, a specific inhibitor <br />                                                                       of histone deacetylases (Yoshida et al. 1995). A brief <br />                                                                       treatment by TSA for z10 generations converted the <br />                            TABLE 2 <br />                                                                       majority of cells that originally displayed the Ura2 state <br />             Effect of the clr6 mutation on stability                  to the Ura1 state. Remarkably, subsequent plating re- <br />                  of the Ura2 and Ura1 states                          vealed that the newly acquired Ura1 state was stable <br />                                                                       for many (.30) generations after the drug treatment <br />                          Transition rate (per cell division)          (Figure 5). Measurement of the mitotic stability of the <br /> Genotype              Ura2 to Ura1               Ura1 to Ura2         TSA-induced state indicated that these Ura1 cells re- <br />                                                                       verted to the Ura2 state at a frequency comparable to <br /> Wild type               5.6 3 1024                8.4 3 1024 <br /> clr6-1                  9.4 3 1024                1.6 3 1022 <br />                                                                       that of standard Ura1 cells (data not shown). Further- <br />                                                                       more, we observed that z30% of the cells exposed to <br />  570                                    S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br />  <br />  <br />  <br />    Figure 5.&#226;&#8364;&#8221;TSA treatment converts the Ura2 state to Ura1. <br /> Ura2 cells of the KD::ura41 strain (SPG27) were patched onto <br /> YEA medium and exposed to filter paper discs carrying 1 ml <br /> of 10 mg/ml TSA dissolved in DMSO. As a control, cells were <br />                                                                      Figure 6.&#226;&#8364;&#8221;The TSA-induced Ura1 sate is tightly linked <br /> likewise exposed to the DMSO solvent. The plates were incu- <br />                                                                   to the mat locus. Cells showing the Ura1 state, obtained by <br /> bated at 338 overnight before a loopful of cells taken from <br />                                                                   TSA treatment of the KD::ura4 1 his2 Ura2 (SPG27) strain, <br /> the edge of the filters was streaked onto fresh Y EA plates. <br />                                                                   were mated to KD::ura41 his21 Ura2 (SPG51) to construct <br /> After growth for 3 days, KD::ura41 expression was scored by <br />                                                                   diploids. The diploids were allowed to grow for at least 30 <br /> replicating colonies onto the indicated medium. Plates were <br />                                                                   generations, sporulated, and subjected to tetrad analysis. The <br /> photographed after overnight growth at 338 <br />                                                                   meiotic segregants were analyzed for linkage of the Ura1 state <br />                                                                   to his2, a marker tightly linked (,1 cM) to the mat locus. The <br />                                                                   2 Ura2His1 : 2 Ura1 His2 segregation pattern in each tetrad <br /> TSA for 1 day exhibited the haploid meiosis phenotype             suggested that the TSA-induced Ura1 state is inherited in cis <br /> when transferred to sporulation medium in the absence             as a marker linked to the mating-type region. Closed and open <br /> of the drug, suggesting that the drug treatment causes            boxes indicate the Ura2 and Ura1 states of KD::ura41 or the <br /> heritable derepression of donor loci as well. Similarly,          mutated and wild-type alleles of his2, respectively. <br /> cells displaying the Ura1 state were also exposed to TSA <br /> and, as expected, the treatment did not change them <br /> to the Ura2 phenotype. Briefly, these data show that              gene activity, at the mat2-mat3 locus. Similarly, a herita- <br /> transient exposure to TSA causes conversion of the tran-          ble change from Ura2 to Ura1 linked to the mating- <br /> scriptionally repressed state to the expressed state, and         type region was also observed in nonswitching mat1- <br /> that the expressed state is stably propagated in the prog-        Msmto KD::ura41 (SPG32) strain (data not shown). <br /> eny for many generations in the absence of TSA.                      The clr6 mutation affects ura4 1 repression at three <br />    In principle, the TSA effect (transition from Ura2 to          sites within cen1: Assembly of the repressive chromatin <br /> Ura1) could be caused either by stable changes in the             structure is also believed to be essential for the proper <br /> expression of trans-acting loci that are essential for si-        functioning of fission yeast centromeres (Clarke et al. <br /> lencing or by alteration in the chromosomally inherited           1993). As mentioned above, trans-acting elements origi- <br /> imprint at the mat locus (Grewal and Klar 1996). We               nally identified as essential for silencing in the mating- <br /> differentiated between these possibilities by the follow-         type region also affect the silencing of markers artifi- <br /> ing cross (see Figure 6). A strain carrying the Ura1              cially inserted into the centromeric sequences (All- <br /> epillele (SPG27, KD::ura41 his2), obtained by TSA treat-          shire et al. 1995). Therefore, it was conceivable that <br /> ment of Ura2 cells, was crossed to a Ura2 (SPG51,                 clr6 might also affect centromeric silencing. To test this, <br /> KD::ura41 his21) strain. The resultant diploid was sporu-         we combined the clr6-1 mutation with ura41 integrated <br /> lated and subjected to tetrad analysis. If the effect of          at three sites within cen1: one in the central domain, <br /> TSA is localized to the mating-type region, two Ura2              cnt1/TM1, and one in each of the outer flanking re- <br /> and two Ura1 segregants should be found in each tetrad,           peats, imr1R and otr1R (Allshire et al. 1995). Interest- <br /> and these states should cosegregate with respective al-           ingly, the mutation showed contrasting effects on ura41 <br /> leles of his2, a marker that is closely linked to the mating-     integrated at different centromeric domains. Dilution <br /> type region. This segregation pattern was indeed ob-              analysis of strains carrying imr1R(NcoI)::ura41 and otr R <br /> served in all 30 tetrads analyzed. Thus, inhibition of            (SphI)::ura41 insertions showed that the clr6-1 mutation <br /> histone deacetylase activity adversely affects propagation        significantly increased their ability to grow on URA2 <br /> of the stable chromosomal imprint, which determines               medium, but there was no growth difference between <br />                                                        Histone Deacetylation                                                      571 <br />  <br />  <br />  <br />  <br />    Figure 7.&#226;&#8364;&#8221;The effect of clr6 mutation on silencing of ura4 1 inserted at three different locations within cen1 and adjacent to <br /> a telomere. Cultures with imr1R(NcoI)::ura41 (A), otr1R(SphI)::ura41 (B), cnt1/TM1::ura41 (C), or Ch16m23-ura41 -TEL (D) in <br /> N/S medium were suspended in water. Fivefold serial dilutions of cell suspensions were spotted onto N/S, URA2, and FOA <br /> media, and the plates were incubated at 308 for 3&#226;&#8364;&#8220;4 days before photography. A strain containing the functional ura41 gene at <br /> the wild-type chromosomal location (SP412) and another with the ura4 deletion (SP837) were used as controls. The strains used <br /> were as follows: (A) FY498, WT; SPG141, clr6-1; SPG142, clr1-5; SPG143, clr1-5 clr6-1; SPG158, clr3-735; and SPG159, clr3-735 <br /> clr6-1; (B) FY648, WT; SPG144, clr6-1; SPG145, clr1-5; SPG146, clr1-5 clr6-1; SPG160, clr3-735; and SPG161, clr3-735 clr6-1; (C) <br /> FY 336, WT; SPG138, clr6-1; SPG139, clr1-5; SPG140, clr1-5 clr6-1; SPG162, clr3-735; and SPG163, clr3-735 clr6-1; and (D) FY 520, <br /> WT; SPG147, clr6-1; SPG148, clr1-5; and SPG149, clr1-5 clr6-1. All strains containing ura4 1 at cen1 or a telomere carried a mutation <br /> of the indigenous ura4. WT indicates the clr1 genotype. <br />  <br />  <br />  <br /> clr6-1 and wild-type cells on FOA medium (Figure 7, A                tinct domains within cen1 and that clr6 differentially <br /> and B). This result suggests that the level of expression            affects these domains, presumably by participating in <br /> of ura41 inserted at both locations was increased, but               their assembly into heterochromatin-like structures. <br /> that the mutation did not completely abolish repression,             Furthermore, the observed additive derepression in <br /> with ura41 expression presumably remaining below the                 double mutants further suggests that Clr6 might per- <br /> level required to cause FOA sensitivity. In contrast and             form an overlapping function with Clr1 plus Clr3 ac- <br /> paradoxically, the mutation resulted in increased re-                tivity. <br /> pression of cnt1/TM1(NcoI)::ura41 insertion at the cen-                 The clr6 mutation reduces the fidelity of chromosome <br /> tral domain of cen1, as indicated by the impaired ability            segregation: It was previously shown that mutations <br /> of cells to form colonies on URA2 medium and the                     affecting centromeric silencing also disrupt chromo- <br /> corresponding increased growth on FOA plates (Fig-                   some segregation (Allshire et al. 1995). Therefore, it <br /> ure 7C).                                                             was formally possible that the clr6-1 allele could also <br />    We also measured the expression of cen1 ura41 con-                impair centromere function. We tested the effect of the <br /> structs in the clr1-5 clr6-1 and clr3-735 clr6-1 double-             clr6-1 mutation on the mitotic stability of the minichro- <br /> mutant backgrounds. As expected, this double-mutant                  mosome Ch16, a 530-kb derivative of chromosome III <br /> combination resulted in additive derepression of ura4 1              (Niwa et al. 1986), as described in materials and meth- <br /> in the imr1R(NcoI)::ura41 and otr1R(SphI)::ura41 strains             ods. As shown in Table 3, strains bearing a lesion at <br /> (Figure 7, A and B). Additionally, clr12 promoted dere-              clr6 missegregated the minichromosome at a 26-fold <br /> pression was reduced by the clr6 mutation at the cnt1/               higher rate compared with their wild-type counterpart. <br /> TM1(NcoI)::ura41 insertion (Figure 7C). We also tested               This result further indicated that clr6 gene function <br /> the clr6 mutation for its effect on telomere-mediated                might also be involved in the formation of fully func- <br /> repression of ura4 1 (Ch16m23-ura41-TEL), but found                  tional centromeres. <br /> that it had no visible effect alone or in combination                   It was recently demonstrated that mutations in clr4, <br /> with clr1-5 (Figure 7D).                                             rik1, and swi6 also cause elevated rates of chromosome <br />    In summary, these results indicate that there are dis-            loss, whereas clr1, clr2, and clr3 mutations have a negligi- <br />  572                                         S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br />                                                                 TABLE 3 <br />                                       Effect of clr mutations on loss of 530-kb minichromosome <br />  <br />         Strain                       Genotype                   Loss rate (per cell division)           Relative loss ratea <br />         HM248                       Wild type                            1.2   3   1023                      Control <br />         SPG134                      clr1-5                               3.4   3   1023                      3-fold <br />         SPG155                      clr3-735                             2.2   3   1023                      2-fold <br />         SPG135                      clr6-1                               3.1   3   1022                      26-fold <br />         SPG136                      clr1-5 clr6-1                        7.9   3   1022                      66-fold <br />         SPG156                      clr3-735 clr6-1                      5.4   3   1022                      45-fold <br />           a <br />               Increase in rate compared with that of the wild-type control. <br />  <br />  <br /> ble effect on chromosome segregation (Allshire et al.                     The encoded Clr3 protein sequence was compared <br /> 1995). Because Clr6 acts in an overlapping manner with                 with the sequences in the GenBank database. The best <br /> Clr1 and Clr3 (this study), we investigated chromosome                 scores were obtained for a group of proteins that include <br /> loss in the clr1 clr6 and clr3 clr6 double mutants. The                known histone deacetylases, such as Hda1 (Rundlet <br /> data presented in Table 3 suggest that the clr6 mutation               et al. 1996) and Rpd3 (Vidal and Gaber 1991) from <br /> further stimulates clr1- and clr3 -promoted chromosome                 S. cerevisiae and HDAC1 (TAunton et al. 1996) from <br /> loss. Likewise, we also studied the effect of TSA treat-               humans. Furthermore, we noticed that similarity be- <br /> ment on the fidelity of chromosome segregation. Re-                    tween Clr3 and Hda1 (41% identity, 65% similarity; <br /> markably, transient exposure of minichromosome-                        Figure 8) was greater than that between Clr3 and <br /> containing cells to TSA resulted in elevated levels of                 S. cerevisiae Rpd3 (27% identity, 49% similarity) or <br /> missegregation in subsequent generations (Table 4).                    human HDAC1 (26% identity, 49% similarity). In addi- <br /> The effect of transient exposure to TSA clearly persists               tion to eukaryotic sequences, Clr3 also shares a high <br /> over many generations.                                                 degree of similarity (41&#226;&#8364;&#8220;52%) to acetylpolyamine hy- <br />    Clr3 is related to histone deacetylases: The partially              drolase proteins from Synechocystis, Mycoplasma ramosa, <br /> redundant function of the putative histone deacetylase                 and Methanococcus janaschii. Further comparison of Clr3 <br /> Clr6 and proteins encoded by the clr1 and clr3 genes                   to other S. pombe sequences revealed that it shares only <br /> suggests that the latter might also encode components                  26% (51% similarity) and 28% (52% similarity) identity <br /> of the histone deacetylase machinery. In this regard,                  to Clr6 and Hoc1 (see above) sequences, respectively, <br /> Clr1 was found to contain the DNA-binding zinc finger                  but was found to be homologous to the subsequently <br /> motifs (G. Thon and A. Klar, unpublished results), but                 reported Hoc2 protein (see above) identified by Sanger <br /> Clr3 remains to be characterized. To define the mech-                  Genome Sequencing Center (cosmid SP41410). Physi- <br /> anism of action of Clr3, we cloned the gene by trans-                  cal mapping of the clr3 gene confirmed that it is the <br /> forming a h90 mat3-M::ura41 clr3-735 strain (SP1359)                   same gene as hoc2, which maps to the short arm of <br /> with a partial Sau3A library. The transformants were                   chromosome II, which is located centromere distal to <br /> screened for repression of mat3-M::ura41 by select-                    rad11. Cloning and sequencing analysis of the clr3-735 <br /> ing for their growth on FOA-containing medium. An                      mutant allele identified a single G-to-A transition in <br /> 8.0-kb insert resulted in complementation of the ura4 1                codon 232, resulting in the D232N change (Figure 8). <br /> expression phenotype and of the sporulation defect                        These data suggest that the putative Clr3 protein be- <br /> caused by derepression of donor loci in the clr3-735                   longs to the family of histone deacetylases. Unlike Clr6, <br /> background. Subcloning and sequencing analysis iden-                   which is more similar to S. cerevisiae Rpd3 and its human <br /> tified an ORF that is capable of encoding a protein of                 homolog, HDAC1, Clr3 shares greater similarity to <br /> 687 amino acids.                                                       S. cerevisiae Hda1. Therefore, it is possible that Clr6 and <br />                                                                        Clr3 proteins are fission yeast homologs of Rpd3 and <br />                             TABLE 4                                    Hda1, respectively. <br />        Effect of <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> of TSA treatment on fidelity <br />              of minichromosome segregation <br />                                                                                                 DISCUSSION <br />                             Loss rate (per cell division)                In fission yeast, there are three known domains that <br /> Days                    DMSO                          DMSO1TSA         exhibit position-effect control, namely the silent mating- <br />                                                                        type region, centromeres, and telomeres. It has been <br /> 1                     1.31 3 1023                     1.48 3 1022 <br />                                                                        suggested that repression at these loci is mediated <br /> 3                     1.41 3 1023                     4.90 3 1022 <br /> 5                     1.46 3 1023                     5.80 3 1022 <br />                                                                        through assembly of a heterochromatin-like structure. <br />                                                                        Consistent with this model, we report the identification <br />                                                     Histone Deacetylation                                              573 <br />  <br />  <br />  <br />  <br />    Figure 8.&#226;&#8364;&#8221;Alignment of the predicted Clr3 and S. cerevisiae Hda1 amino acid sequences. The sequences were aligned using <br /> the GAP program. Identical amino acids are marked by vertical lines, and similar amino acids are indicated by colons. The <br /> asterisk indicates the residue that is changed to asparagine [N] in the clr3-735 mutant allele. <br />  <br />  <br /> of an essential gene, clr6, encoding a putative histone         and Winston 1997; Pazin and Kadonaga 1997), it has <br /> deacetylase, which, when mutated, affects centromeric           been suggested that histone deacetylation leads to as- <br /> as well as donor loci silencing and reduces the fidelity        sembly of &#226;&#8364;&#339;compact&#226;&#8364;&#65533; chromatin, which prevents access <br /> of chromosome segregation. Furthermore, the clr6 mu-            to transcription factors, whereas acetylation makes it <br /> tation affects propagation of the epigenetic states regu-       &#226;&#8364;&#339;open,&#226;&#8364;&#65533; allowing access to the transcription machinery. <br /> lating expression of a marker gene integrated in the            In contradiction to this model, mutations in RPD3 and <br /> silent mating-type region, recombination suppression,           related genes were recently shown to enhance silencing <br /> and directionality of mating-type switching.                    in the heterochromatic regions of both budding yeast <br />    Genetic evidence suggests that Clr6 function is par-         and Drosophila (Derubertis et al. 1996; Rundlet et al. <br /> tially redundant to the function of Clr1/Clr3. In this          1996; Vannier et al. 1996). Similarly, mutation in clr6 <br /> respect, we find that the putative Clr3 protein also shares     causes a significant (20-fold) increase in Ura1 to Ura2 <br /> similarity to histone deacetylases. Our earlier study indi-     transition of KD::ura41 epialleles and silencing at the <br /> cated that clr1 and clr3 work together, either in the same      central domain (TM1) of cen1. However, the effects of <br /> pathway or as a complex (Thon et al. 1994). Taken               clr6-1 are not uniform across all sites tested, and the <br /> together, these data suggest that silencing at above-men-       marker gene integrated at either of two outer flanking <br /> tioned loci might be regulated by two distinct yet func-        repeats (imr1 and otr1) of cen1 was weakly derepressed <br /> tionally overlapping histone deacetylation mechanisms:          (this study). In contrast to increased repression and <br /> Clr1 and Clr3 might be components of one such system,           consistent with the proposed role of histone deacety- <br /> whereas Clr6 is a component of a second system. Like-           lases, a mutation in clr3 adversely affects silencing both <br /> wise, the budding yeast proteins Hda1 and Rpd3, which           at the silent mating-type region (Thon et al. 1994; <br /> share high similarity to Clr3 and Clr6, respectively, have      Grewal and Klar 1997) and at centromeres (Allshire <br /> been found to be associated with the distinct histone           et al. 1995). Also, functional clr31 is required for prop- <br /> deacetylase complexes HDA and HDB, respectively                 agation of the repressed (Ura2) epigenetic state <br /> (Rundlett et al. 1996).                                         (Grewal and Klar 1996). To accommodate these <br />    On the basis of recent studies (Braunstein et al. 1993;      seemingly contradictory results, we propose a hypotheti- <br /> Brownell and Allis 1996; Wolffe 1996; Hartzog                   cal &#226;&#8364;&#339;competition model.&#226;&#8364;&#65533; We suggest that, as in budding <br />  574                                    S. I. S. Grewal, M. J. Bonaduce and A. J. S. Klar <br />  <br /> yeast, where the subtelomeric region and silent mating-           interconversion. In our chromatin-replication model, <br /> type loci compete for silencing proteins (Buck and                we previously suggested that silencing, efficiency of mat- <br /> Shore 1995; Maillet et al. 1996), different heterocho-            ing-type switching, and recombination suppression in <br /> matin domains in fission yeast might compete for a                the mat2-mat3 interval are regulated by a heterochroma- <br /> defined pool of heterochromatic proteins (HPs) such               tin structure that, once assembled, is capable of self- <br /> as Swi6. Furthermore, the affinity of a particular chro-          replicating for many generations through cooperative <br /> mosomal domain for HPs may depend on its state of                 binding of HPs (Grewal and Klar 1996, 1997). It is <br /> histone acetylation. If, for example, HPs preferentially          possible that inhibition of histone deacetylation com- <br /> bind to deacetylated histone H3 and H4, regional differ-          promises the assembly of the proposed chromatin struc- <br /> ences in acetylation state could ensure their targeting           ture. It remains to be determined whether the change <br /> to specific chromosomal domains. In turn, differences             in imprint is caused by direct alteration of the acetyla- <br /> in acetylation pattern could depend on region-specific            tion state at the silent mating-type region or by an indi- <br /> preferential recruitment of different combinations of             rect transitory change in the expression of trans -acting <br /> Clr3, Clr6, Hoc1, or other deacetylating proteins. In this        factors required for silencing, which, in turn, cause a <br /> way, mutation in a particular deacetylase might result in         change in the imprint. There is also a possibility that <br /> relocalization of HPs from one chromosomal domain to              TSA treatment affects the timing of replication and/or <br /> another. For example, mutation in clr6 might delocalize           nuclear localization of the mat region, causing changes <br /> HPs from the imr and otr regions of centromeres. The              in the structural and functional imprint. Regardless of <br /> free HPs could then be recruited to the mating-type               whether the TSA effect is direct or indirect, the final <br /> and central centromeric regions, leading to contrasting           effect is shown to cause a heritable change in the imprint <br /> effects on corresponding loci. Consistent with this com-          at the mat2-mat3 region. Interestingly, Ekwall et al. <br /> petition model, we see that the increase in repression            (1997) recently showed that transient TSA treatment <br /> at the mat2-mat3 region observed in clr6 mutants is de-           induces a heritable hyperacetylated state in centromeric <br /> pendent on the presence of functional Clr1 and Clr3               chromatin, which correlates with functionally defective <br /> (this study). Also, the clr6-1-mediated increased silenc-         centromeres, suggesting a link between histone acetyla- <br /> ing is sensitive to TSA treatment (our unpublished                tion and the assembly of functional centromeres. <br /> data), indicating the probable involvement of other                  Unlike histone deacetylase from S. cerevisiae (Vidal <br /> deacetylases.                                                     and Gaber 1991), S. pombe clr6 is an essential gene. We <br />    As an alternative to the competition model, we imag-           believe that lethality is at least partly related to a reduced <br /> ine that the clr6 mutation might affect silencing indi-           fidelity of chromosome segregation in mutant strains. In <br /> rectly by altering the regulation of trans-acting genes,          addition, the possible disruption of chromatin structure <br /> including clr1-clr4, swi6, and rik1. It has been suggested        throughout the genome in the clr6 mutant may result <br /> that enhanced repression in mutant cells might be                 in deleterious deregulation of gene expression. We note <br /> caused by a specific pattern of histone acetylation (such         that clr6-1 mutant is UV sensitive, a phenotype common <br /> as increased acetylation of H4 lysine 12; Rundlett et             in mutants of genes controlling the assembly of chroma- <br /> al. 1996) or by the assembly of a specialized chromatin           tin structure (Kaufman et al. 1997) and the DNA repair <br /> structure (Derubertis et al. 1996). Supporting the latter         process (Reynolds et al. 1990). In conclusion, this study <br /> argument, it was previously observed that the central             implicates putative histone deacetylases in the assembly <br /> centromeric domain in fission yeast is packaged into              of heterochromatin-like structures at the silent mating- <br /> an unusual chromatin structure (Polizzi and Clarke                type region and at centromeres. Supporting our chro- <br /> 1991; Takahashi et al. 1992). These three models are              matin replication model, we demonstrate that inhibi- <br /> not mutually exclusive, and further work is needed to             tion of histone deacetylase activity can compromise the <br /> differentiate among them.                                         propagation of the stable chromosomal imprint at the <br />    During development and differentiation, propaga-               mat locus. <br /> tion of the stable states of gene expression through              <br /> numerous cell divisions is essential (Felsenfeld 1992).           <br /> It is possible that the histone acetylation pattern of chro-       <br /> mosomal domains acts as a &#226;&#8364;&#339;cell memory&#226;&#8364;&#65533; and helps                <br /> to maintain the respective states of gene expression. <br />                                                                 <br /> Consistent with this idea, we observed that transient        <br /> inhibition of histone deacetylation by TSA causes a re-           <br /> markable heritable change in the expression of a marker           <br /> gene integrated in the silent mating-type region, con- <br /> verting the epigenetically repressed state to the ex- <br /> pressed state in the majority of cells. The expressed state                            <br /> is inherited in cis as a marker linked to the mating-type <br />                                                           <br /> locus and, as observed previously (Grewal and Klar          <br /> 1996), correlates with reduced efficiency of mating-type      <br /> </body></html>