<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 18, 3472&#226;&#8364;&#8220;3485, September 2007 <br />  <br />  <br /> Pga1 Is Essential Component of <br /> Glycosylphosphatidylinositol-Mannosyltransferase II <br /> of Saccharomyces cerevisiae <br /> Keisuke Sato, Yoichi Noda, Koji Yoda <br />  <br /> Department Biotechnology, University Tokyo, Tokyo 113-8657, Japan <br />  <br /> Submitted March 19, 2007; Revised June 21, 2007; Accepted June 22, 2007 <br /> Monitoring Editor: Akihiko Nakano <br />  <br />  <br />  <br /> The Saccharomyces cerevisiae essential gene YNL158w/PGA1 encodes endoplasmic reticulum (ER)-localized membrane <br /> protein. We constructed temperature-sensitive alleles PGA1 error-prone polymerase chain reaction mutagenesis <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> explore biological role. Pulse-chase experiments revealed pga1ts mutants accumulated ER-form precursor <br /> of Gas1 protein restrictive temperature. Transport invertase carboxypeptidase Y affected. Triton <br /> X-114 phase separation [3H]inositol labeling indicated glycosylphosphatidylinositol (GPI)-anchoring was <br /> defective pga1ts mutants, suggesting Pga1 involved GPI synthesis transfer target proteins. We <br /> found GPI18, recently reported encode GPI-mannosyltransferase II (GPI-MT II), high-copy suppressor <br /> of temperature sensitivity pga1ts. Both Gpi18 Pga1 detected ER immunofluorescence, they <br /> were coprecipitated Triton X-100 &#226;&#8364;&#8220;solubilized membrane. The gpi18ts pga1ts mutants accumulated same <br /> GPI synthetic intermediate restrictive temperature. <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> results, concluded Pga1 additional <br /> essential component yeast GPI-MT II. <br />  <br />  <br /> INTRODUCTION                                                         remaining mannose residues, contributing to <br />                                                                      cell wall integrity (Lu et al., 1995). Construction and <br /> Many proteins synthesized surface endoplasmic          attachment target protein GPI carried the <br /> reticulum (ER) transported secretory path-           ER membrane. The GPI synthesized stepwise addi- <br /> way proper locations cell function              tion sugars, acyl chain, EtN-P phosphatidylino- <br /> (Palade, 1975; Mellman Warren, 2000). The ER               sitol (PI). The GPI transamidase complex removes C- <br /> organelle major secretory pathway starts,          terminal GPI attachment signal peptide target protein <br /> folding status proteins monitored pass            links newly generated C-terminal end EtN-P <br /> pathway (Kleizen Braakman, 2004). In ER,             complete GPI precursor lipid (Eisenhaber et al., 2003). <br /> modifications performed proteins, N-              All genes known involved GPI <br /> glycosylation, O-glycosylation, glycosylphosphatidyli-           synthesis GPI-anchoring S. cerevisiae homo- <br /> nositol (GPI)-anchoring (Sipos Fuller, 2002). The GPI         logues mammals. Moreover, GPI essential via- <br /> a complex glycolipid core structure, phosphoethano-           bility protozoan species, Leishmania mexicana <br /> lamine (EtN-P)-6-mannose (Man)-&#226;?&#163;1,2-Man-&#226;?&#163;1,6-Man-&#226;?&#163;1,4-               bloodstream form Trypanosoma brucei. It been <br /> glucosamine (GlcN)-&#226;?&#163;1,6-inositol-phospholipid, con-          suggested pathway GPI synthesis con- <br /> served eukaryotic cells (Kinoshita Inoue, 2000)       served species; thus, S. cerevisiae contribute to <br /> functions membrane anchor cell surface pro-            GPI research model organism. <br /> teins (Englund, 1993). In budding yeast Saccharomyces cerevi-           The GPI S. cerevisiae contains Man residues. The <br /> siae, GPI synthesis GPI-anchoring carried &#226;&#172;&#381;20         fourth Man essential roles SMP3, encodes <br /> gene products, genes essential cell          enzyme add fourth Man, essential viability <br /> viability (Pittet Conzelmann, 2006). Many GPI-anchored           (Grimme et al., 2001), addition fourth Man is <br /> proteins lose lipid moiety certain stage maturation,         essential GPIs contain Man residues <br /> and attached cell wall &#226;?&#164;1,6-glucan                organisms (Taron et al., 2004). The enzymes add <br />                                                                      Man GPI precursors called GPI-mannosyltransferase <br />                                                                      (GPI-MT) I, -II, -III, -IV, according order man- <br /> This article published online ahead print MBC Press     nose addition. Each GPI-MT consists protein <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E07&#226;&#8364;&#8220; 03&#226;&#8364;&#8220; 0258)       transmembrane domains homology to <br /> on July 5, 2007.                                                     enzymes responsible glycosylation (Takahashi et <br /> Address correspondence to: Koji Yoda (asdfg@mail.ecc.u-tokyo.        al., 1996; Grimme et al., 2001; Maeda et al., 2001; Ashida et al., <br /> ac.jp).                                                              2005; Fabre et al., 2005; Kang et al., 2005). So far, GPI-MT I has <br />                                                                      enzyme consist multisub- <br /> Abbreviations used: ALP, alkaline phosphatase; CPY, carboxypep- <br /> tidase Y; endoH, endoglycosidase H; ER, endoplasmic reticulum; <br />                                                                      units. Gpi18 recently reported required the <br /> EtN-P, phosphoethanolamine; GFP, green fluorescent protein;          addition second Man GPI precursor, its <br /> GlcN, glucosamine; GPI, glycosylphosphatidylinositol; GPI-MT,        mammalian homologue PIG-V acts GPI-MT II Chinese <br /> GPI-mannosyltransferase; Man, mannose; PI, phosphatidylinositol;     hamster ovary (CHO) cells. PIG-V partially rescue the <br /> SGD, Saccharomyces Genome Database.                                  lethality &#226;&#338;&#172;gpi18 cells (Fabre et al., 2005; Kang et al., 2005). <br />  <br /> 3472                                                                                  &#194;&#169; 2007 The American Society Cell Biology <br />                                                                                                   Novel GPI-Mannosyltransferase II Subunit <br />  <br />  <br />  <br /> Table 1. S. cerevisiae strains used study <br />  <br /> Strain                                              Genotype plasmid                                                           Origin <br />  <br /> KA31a              MATa, his3&#226;&#338;&#172; leu2&#226;&#338;&#172; trp1&#226;&#338;&#172; ura3&#226;&#338;&#172;                                                                           Laboratory stock <br /> BY4741             MATa, his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0                                                                      Euroscarf <br /> BY4743             MATa/MATa, his3&#226;&#338;&#172;1/his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0/leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0/MET15 LYS2/lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0/ura3&#226;&#338;&#172;0 <br /> KSY99              BY4741, MET15                                                                                        This study <br /> KSY33              BY4741, pga1&#226;&#338;&#172;::kanMX4, pKS5 (CEN, HIS3 PGA1-GFP)                                                     This study <br /> KSY63              BY4741, pga1&#226;&#338;&#172;::kanMX4, pKS22 (CEN, HIS3 PGA1-6myc)                                                   This study <br /> KSY216             BY4741, pga1&#226;&#338;&#172;::kanMX4, pKS38 (CEN, HIS3 PGA1), pKS128 (CEN, GFP-GPI18)                               This study <br /> KSY217             BY4741, pga1&#226;&#338;&#172;::kanMX4, pKS110 (CEN, HIS3 PGA1-6myc), pKS128 (CEN, GFP-GPI18)                         This study <br /> KSY182             KSY99, pga1&#226;&#338;&#172;::pga1-3 LEU2                                                                            This study <br /> AKY19              KSY99, pga1&#226;&#338;&#172;::pga1-1 LEU2                                                                            This study <br /> KSY274             KSY99, pga1&#226;&#338;&#172;::gpi18-3 LEU2                                                                           This study <br /> KSY277             KSY99, pga1&#226;&#338;&#172;::gpi18-6 LEU2                                                                           This study <br /> KSY279             KSY99, gpi18&#226;&#338;&#172;::PGAL1-GPI18 LEU2                                                                      This study <br /> KSY307             KSY99, pKS128 (CEN, GFP-GPI18)                                                                       This study <br /> KSY308             KSY99, gpi18&#226;&#338;&#172;::PGAL1-PGA1 LEU2, pKS128 (CEN, GFP-GPI18)                                              This study <br /> KSY309             KSY99, gpi18&#226;&#338;&#172;::PGAL1-PGA1 LEU2, pKS128 (CEN, GFP-GPI18)                                              This study <br /> KSY312             AKY19, pKS128 (CEN, GFP-GPI18)                                                                       This study <br /> KSY313             KSY277, pKS128 (CEN, GFP-GPI18)                                                                      This study <br /> YKI59              KA31a, erd1&#226;&#338;&#172;                                                                                         Imai et al. (2005) <br /> YNY309             KA31a, sec12ts                                                                                       Inadome et al. (2005) <br /> YNY401             KA31a, emp24&#226;&#338;&#172;                                                                                        This study <br /> KSY269             BY4741, gpi7&#226;&#338;&#172;::kanMX4                                                                                Euroscarf <br /> CAY341             BY4743, pga1&#226;&#338;&#172;::kanMX4/PGA1                                                                           Euroscarf <br /> KSY223             BY4743, gpi18&#226;&#338;&#172;::kanMX4/GPI18                                                                         Euroscarf <br />  <br />  <br />  <br />    Recently, large-scale analyses essential genes S.       addition GPI precursors partner Gpi18. This the <br /> cerevisiae reported laboratories (Hazbun et        report gene mammalian homo- <br /> al., 2003; Mnaimneh et al., 2004; Davierwala et al., 2005;           logue involved GPI synthesis S. cerevisiae. <br /> Schuldiner et al., 2005). In analyses, target genes were <br /> placed control regulatable promoters, the <br /> physiological roles encoded proteins predicted            MATERIALS AND METHODS <br /> the phenotypes showed expression shut-off, <br /> and, cases, synthetic sickness lethality      Strains Media <br /> other genes encode proteins known functions.               S. cerevisiae strains used study listed Table 1. They grown <br /> Although approaches proved useful               YPD (1% Bacto yeast extract [BD Biosciences, Franklin Lakes, NJ], 2% Bacto <br />                                                                      peptone [BD Biosciences], 2% glucose) SD (0.17% yeast nitrogen <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span> <br /> dealing genes, suggested possible in-             amino acids [BD Biosciences], 0.5% ammonium sulfate, 2% glucose, <br /> volvement given gene product certain biological          appropriate supplements) medium 30&#194;&#176;C unless temperatures <br /> pathway, functional mechanistic information ob-              indicated. An SG (2% galactose instead glucose SD) medium was <br /> tained gene product lim-             used induce genes GAL1 promoter. Ammonium sulfate replaced <br />                                                                      ammonium chloride low-sulfate SD medium. Solid media were <br /> ited. So, took different approach proteins         1.5% agar; 20 &#226;?&#174;g/ml histidine, tryptophan, uracil and <br /> execute undefined essential roles ER. We created          0.1% 5-fluoroorotic acid (Sigma-Aldrich, St. Louis, MO) added SD to <br /> the temperature-sensitive (ts) mutants essential genes       make 5-fluoroorotic acid (5-FOA) plate Ura3&#226;&#171;&#185; counterselection, 20 <br /> encode ER-localized proteins error-prone polymerase           &#226;?&#174;g/ml phloxine B (Sigma-Aldrich) added YPD make phloxine <br />                                                                      plate (Tsukada Ohsumi, 1993) selection temperature-sensitive <br /> chain reaction (PCR). The advantage using ts mutants           mutants. Escherichia coli DH5&#226;?&#163; (F&#226;&#171;&#186;, supE44 &#226;&#338;&#172;lacU169&#226;?&#190;80lacZ&#226;&#338;&#172;M15 hsdR17 <br /> that display phenotypes immediately         recA1 endA1 gyrA96 thi-1 relA1) used plasmid propagation. E. coli was <br /> temperature shift; display allele-specific            grown LB (1% Bacto tryptone [BD Biosciences], 0.5% Bacto yeast extract <br /> phenotypes useful regional analyses; suit-          [BD Biosciences]) 0.5% NaCl) medium. <br /> able screening genetic interactions, especially genes that <br /> function multicopy suppressors, provide               Plasmid Construction <br /> with valuable clues characterize protein function.       Primers used study listed Table 2. The nucleotide sequence of <br /> The ER-localized proteins selected membrane                amplified fragment confirmed. <br /> proteins encoded essential genes observing local-             Low-copy PGA1 expression plasmid (pCA148 pKS38) by <br />                                                                      ligation PCR amplification fragment genomic DNA using <br /> ization green fluorescent protein (GFP)-fusion pro-           primers CA177 CA178 plasmid pRS316 pRS313, respec- <br /> teins. One them, YNL158w/PGA1, encodes protein             tively. Low-copy GPI18 expression plasmid (pKS71) ligation of <br /> has similarity mammalian proteins,                PCR amplification fragment genomic DNA using primers <br /> reported required proper processing Gas1 (a         KS52 KS53 plasmid pRS316. High-copy GPI18 expression plas- <br />                                                                      mid (pKS72) replacing plasmid vector pRS426. High- <br /> GPI-anchored protein plasma membrane) Pho8                copy GPI14, PBN1, GPI10, SMP3 expression plasmid (pKS97, pKS98, <br /> (an alkaline phosphatase vacuole); therefore, Pga1        pKS96, pKS99, respectively) ligation PCR amplification <br /> been suggested important normal ER function                fragment genomic DNA using primers (KS59 KS60, KS61 <br /> (Davierwala et al., 2005). Here, Pga1             KS62, KS57 KS58, KS63 KS64, respectively) plasmid <br />                                                                      pRS426. The C-terminal tagging GFP Pga1 (pKS5) ligation <br /> essential component GPI-MT II. We identified GPI18           PCR amplification fragment genomic DNA using the <br /> high-copy suppressor pga1ts, characterization         primers CA159 YN209 plasmid pRS316-GFP-TDH1 terminator <br /> indicated Pga1 responsible second mannose            (pCA42). <br />  <br />  <br /> Vol. 18, September 2007                                                                                                                       3473 <br />  K. Sato et al. <br />  <br />  <br />  <br /> Table 2. Primers used study <br />  <br /> Primer                 Gene                  Restriction site                   Direction                                       Sequence <br />  <br /> CA159                  PGA1                       BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCTTATCGTCTCTTTCTGTCC-3&#226;&#172;&#732; <br /> CA160                  PGA1                       XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGTTGGGCGAGAATTTGCTG-3&#226;&#172;&#732; <br /> CA186                  PGA1                       BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCATGGTCCGGCCTCAGAAT-3&#226;&#172;&#732; <br /> YN209                  PGA1                       XhoI                          Reverse                  5&#226;&#172;&#732;-CTCGAGAAAATTTCAGCAAAAGATC-3&#226;&#172;&#732; <br /> KS5                    PGA1                       EcoRI                         Reverse                  5&#226;&#172;&#732;-GGGAATTCGAACCTATGTTGGCTTAG-3&#226;&#172;&#732; <br /> KS6                    PGA1                       EcoRI                         Forward                  5&#226;&#172;&#732;-GGGAATTCTTTGGCTGGATAGCTTGG-3&#226;&#172;&#732; <br /> KS52                   GPI18                      BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCACTGCCATGACGACGATG-3&#226;&#172;&#732; <br /> KS53                   GPI18                      XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGTCAGAGAGGGGTGATGTT-3&#226;&#172;&#732; <br /> KS54                   PGA1                       SacI                          Forward                  5&#226;&#172;&#732;-GCGAGCTCCTCATATTCCTCCTAAGG-3&#226;&#172;&#732; <br /> KS57                   GPI10                      BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCCGAGTCCTATTCCCTAAA-3&#226;&#172;&#732; <br /> KS58                   GPI10                      XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGCGCTCTGTATTTATCCTC-3&#226;&#172;&#732; <br /> KS59                   GPI14                      BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCTCCCAAGGCCTAGGTCTT-3&#226;&#172;&#732; <br /> KS60                   GPI14                      XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGCACATCACATCAAGTATC-3&#226;&#172;&#732; <br /> KS61                   PBN1                       BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCCGGATTAACAGTATGCTC-3&#226;&#172;&#732; <br /> KS62                   PBN1                       XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGAGAATGAACAAAGAGCCC-3&#226;&#172;&#732; <br /> KS63                   SMP3                       BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCGGTACGTGCAATATAGAG-3&#226;&#172;&#732; <br /> KS64                   SMP3                       XhoI                          Reverse                  5&#226;&#172;&#732;-CCCTCGAGTGAGCTGAGTACCGAGTT-3&#226;&#172;&#732; <br /> KS65                   GPI18                      BamHI                         Forward                  5&#226;&#172;&#732;-CGGGATCCATGATTGTGGGGTTGACA-3&#226;&#172;&#732; <br /> KS83                   GPI18                      XbaI                          Forward                  5&#226;&#172;&#732;-GCTCTAGAGCTTGGTGAACTACAGCA-3&#226;&#172;&#732; <br /> KS85                   GPI18                      EcoRI                         Reverse                  5&#226;&#172;&#732;-GGGAATTCTCTCTCGAAGCCTCTCTT-3&#226;&#172;&#732; <br /> KS86                   GPI18                      EcoRI                         Forward                  5&#226;&#172;&#732;-GGGAATTCGCCTGATCCTTAATGCTT-3&#226;&#172;&#732; <br />  <br />  <br />  <br />    The C-terminal fusion 6myc Pga1 (pKS22 pKS110)             Construction Temperature-sensitive Mutant Alleles of <br /> replacing vector plasmid pKS5 pNT125-2 (pRS316-6myc-TDH1           PGA1 GPI18 <br /> terminator) pKS55 (pRS313-6myc-TDH1 terminator), respectively. The N- <br /> terminal fusion GFP Gpi18 (pKS128) ligation PCR           The DNA fragment containing PGA1 gene authentic <br /> amplification fragment genomic DNA using primers KS65              promoter terminator amplified using primers KS159 KS160 <br />                                                                                    error-prone PCR condition (Muhlrad et al., 1992). KSY33 trans- <br /> and KS53 plasmid pRS316-YPT1 promoter-GFP (pCA42). The plas- <br />                                                                                    formed mixture amplified fragment EcoRI-cleaved <br /> mid pKS213 gap repair used construct pga1ts mutants as <br />                                                                                    pKS213 integrate fragments plasmid gap repair. The His&#226;&#171;&#185; <br /> follows: PCR amplification fragment genomic DNA using prim- <br />                                                                                    transformants replicated 5-FOA plates remove pKS5, they <br /> ers CA159 KS5 ligated pRS313, PCR amplifi- <br />                                                                                    replicated phloxine B plates incubated 37&#194;&#176;C 2 d. The cells <br /> cation fragment genomic DNA using primers KS6 CA160               formed pink colonies tested single colony formation 25 and <br /> ligated newly generated plasmid. The plasmid pKS170 gap repair        37&#194;&#176;C. After confirming temperature sensitivity plas- <br /> used construct gpi18ts mutants follows: PCR amplification       mid, nucleotide sequences mutant pga1 determined. Construc- <br /> fragment genomic DNA using primers KS52 KS85                  tion ts mutant alleles GPI18 performed way as <br /> ligated pRS313 PCR amplification fragment           described above, primers KS52 KS53 used, gap <br /> genomic DNA using primers KS86 KS53 ligated newly             repair plasmid SmaI-cleaved pKS170, transformed strain was <br /> generated plasmid. Plasmid pAK35 pKS81 used integration pga1ts       KSY239. <br /> alleles yeast chromosome insertion DNA fragment <br /> containing encoding ORF downstream sequence pKSTS16 or <br /> pKSTS15 pTY23 (pBluescript SK&#226;&#171;&#185;-LEU2) following insertion the <br />                                                                                    Subcellular Fractionation Solubility Test of <br /> PCR amplification fragment genomic DNA using primers KS54             Membrane Proteins <br /> and CA160. Plasmid pKS188 pKS195 integration gpi18ts allele      Cells converted spheroplasts burst B88 (20 mM HEPES, pH 6.8, <br /> to yeast chromosome insertion PCR amplification             150 mM potassium acetate, 5 mM magnesium acetate, 200 mM sorbitol) <br /> fragment genomic DNA using primers KS83 KS85                 containing protease inhibitors (1 &#226;?&#174;g/ml chymostatin, aprotinin, leu- <br /> pTY23 following insertion DNA fragment containing encoding              peptin, pepstatin A, antipain, 1 mM phenylmethylsulfonyl fluoride). <br /> open reading frame (ORF) downstream sequence pKSTS113              Unbroken cells removed centrifugation 1000 &#226;&#171;&#187; g 5 min. For <br /> pKSTS114. The plasmid pKS173 integration GPI18 expressing         subcellular fractionation, cleared lysate sequentially centrifuged to <br /> control GAL1 promoter follows. A DNA fragment containing            generate 10,000 &#226;&#171;&#187; g pellet (P10), 100,000 &#226;&#171;&#187; g pellet (P100), 100,000 &#226;&#171;&#187; g <br /> GAL1 promoter pYES2 inserted pTY23, PCR amplifi-             supernatant (S100). Each fraction adjusted original volume the <br /> cation fragment genomic DNA using primers KS65 KS53               lysate applied SDS-polyacrylamide gel electro- <br /> was ligated newly generated plasmid LEU2 gene. The             phoresis (PAGE) Western blotting. As solubility test, portion lysate <br /> plasmid pKS189 integration PGA1 expressing control         mixed B88 containing 2% Triton X-100, 2 M <br /> GAL1 promoter chromosome follows: DNA fragment                NaCl, 0.2 M Na2CO3, 4 M urea, mixture kept ice 15 min. <br /> containing GAL1 promoter pYES2 inserted pTY23, PCR           After removing sample total, mixture centrifuged at <br /> amplification fragment genomic DNA using primers CA186            100,000 &#226;&#171;&#187; g 60 min. The precipitate suspended volume of <br />                                                                                    buffer supernatant. <br /> CA160 ligated newly generated plasmid LEU2 gene. <br />  <br />                                                                                    BiP Secretion Blots <br /> Antibodies                                                                         The secretion lumenal ER proteins analyzed described previously <br />                                                                                    (Lewis et al., 1997). <br /> Antisera Scs2 Kar2 kindly provided Dr. S. Kagiwada <br /> (Nara Woman&#226;&#8364;&#8482;s University, Japan) Dr. Masao Tokunaga (Kagoshima <br /> University, Japan), respectively. Anti-3-phosphoglycerate kinase mouse (anti-      Triton X-114 Phase Separation <br /> PGK, 22C5; Invitrogen, Carlsbad, CA), anti-myc mouse (9E10; Berkeley An-           The Triton X-114 phase separation performed essentially described by <br /> tibody, Richmond, CA), anti-alkaline phosphatase (ALP) mouse (1D3; Invitro-        Doering Schekman (1996) modifications. In brief, cells <br /> gen) monoclonal antibodies (mAbs), anti-carboxypeptidase Y rabbit          incubated 34&#194;&#176;C 12 h, collected, disrupted glass beads. <br /> (anti-CPY; Rockland, Gilbertsville, PA) polyclonal antibody purchased.        After centrifugation 1000 &#226;&#171;&#187; g 5 min, supernatant incubated in <br /> The anti-GFP antiserum prepared immunizing rabbits glu-            B88 containing 1% Triton X-114 ice 30 min. After extract was <br /> tathione S-transferase-GFP fusion protein antigen. For Western blot-        centrifuged 10,000 &#226;&#171;&#187; g 4 min 4&#194;&#176;C, supernatant kept 32&#194;&#176;C <br /> ting, antisera diluted 1:1000.                                       5 min centrifuged 13,000 &#226;&#171;&#187; g 30 s room temperature to <br />  <br />  <br /> 3474                                                                                                                           Molecular Biology Cell <br />                                                                                                                 Novel GPI-Mannosyltransferase II Subunit <br />  <br /> separate detergent aqueous phases. Each phase re-extracted        PAGE sample buffer 5 min resolved SDS-PAGE 7.5% gel. <br />                                                                                    35 <br /> either 1% Triton X-114 (final concentration) buffer appropriate. Each         S-labeled CPY Gas1 detected autoradiography analyzed by <br /> re-extracted phase subjected SDS-PAGE, Gas1, Scs2, Pgk1          BAS-3000 (Fuji Film, Tokyo, Japan). <br /> detected Western blotting. <br />                                                                                    Radiolabeling Lipids myo-2-[3H]Inositol <br /> Indirect Immunofluorescence                                                        For myo-2-[3H]inositol labeling temperature-sensitive strains, cultures of <br /> Fixation permeabilization yeast cells indirect immunofluores-       strain grown 25&#194;&#176;C SD medium OD600 nm &#226;&#171;&#189; 1.0, 10 <br /> cence performed described Wooding Pelham (1998),                 OD units cells washed twice SD-inositol resuspended in <br /> subsequent steps performed described Vashist et al. (2002)         1 ml SD-inositol. After incubation 25&#194;&#176;C 30 min, cultures were <br /> some modifications; description procedure follows. Log-phase        shifted 37&#194;&#176;C 20 min maintained 25&#194;&#176;C 15 &#226;?&#174;Ci [3H]inositol <br /> cells grown 30&#194;&#176;C fixed adding 2.5 ml fresh 10% paraformalde-         (American Radiolabeled Chemicals, St. Louis, MO) added them, and <br /> hyde 7.5 ml yeast culture pelleted centrifugation. They          radiolabeling continued 90 min temperature. For [3H]i- <br /> resuspended 3.2 ml PP (0.1 M potassium phosphate, pH 7.5), 1.8 ml         nositol labeling gpi18::LEU2-PGAL1-GPI18 strains, cells maintained on <br /> paraformaldehyde solution added, fixation continued             SG medium shifted fresh SG SD 20 h OD600 &#226;&#172;&#402;1. Ten <br /> additional 15 min. Cells washed times PP resuspended         OD cells washed twice SG-inositol SD-inositol, resuspended 1 ml <br /> in 1 ml SPP (PP 1.2 M sorbitol) containing 100 mM dithiothreitol. Ten      SG-inositol SD-inositol, incubated 50 min 30&#194;&#176;C. After the <br /> microliters lyticase added, incubation continued 30&#194;&#176;C. The       incubation, 15 &#226;?&#174;Ci [3H]inositol added them, radiolabeling was <br /> spheroplasts harvested centrifugation resuspended 50 mM             continued 90 min 30&#194;&#176;C. To stop [3H]inositol labeling, NaN3 added <br /> NH4Cl SPP, SPP transferred polylysine-coated        10 mM final concentration each, cells placed ice. Cells were <br /> slides. The slides immersed methanol 6 min acetone 30 s,       collected centrifugation, washed twice 10 mM NaN3, resuspended <br /> both &#226;&#171;&#186;20&#194;&#176;C, air-dried. The anti-myc mouse mAb anti-            500 &#226;?&#174;l CM extraction solvent (CHCl3/CH3OH, 1:1, vol/vol). Glass beads <br /> Kar2 rabbit polyclonal antibody diluted 1/40, 1/100 1% skim        (0.5 g) added suspension, cells lysed vortexing four <br /> milk, 0.1% bovine serum albumin, 0.05% Tween 20 Tris-buffered saline        times 1 min each. The lysate spun 15,000 &#226;&#171;&#187; g 5 min, the <br /> (TBS; 50 mM Tris, pH 7.4, 150 mM NaCl), incubations carried           supernatant collected transferred new tube. Three hundred <br /> out overnight 4&#194;&#176;C. Slides washed incubated secondary            microliters CMW (CHCl3/CH3OH/H2O, 10:10:3, vol/vol) added to <br /> antibodies (Alexa 488-conjugated goat antibody mouse immunoglobulin G           remaining glass beads cell debris, mixture vortexed for <br /> and Texas Red-conjugated goat antibody rabbit immunoglobulin G) 2 h         1 min. After centrifugation, supernatant added lipid extract. <br /> at room temperature, washed phosphate-buffered saline, mounted            Pooled lipids dried speed-vac. The resulting pellet resuspended <br /> as described Kilmartin Adams (1984). Images obtained using          150 &#226;?&#174;l H2O-saturated 1-butanol extracted 75 &#226;?&#174;l 1-butanol- <br /> FV500 confocal laser-scanning microscope (Olympus, Tokyo, Japan).                  saturated water. The organic phase collected, aqueous phase was <br />                                                                                    backextracted additional 75 &#226;?&#174;l H2O-saturated 1-butanol. The <br /> Immunoprecipitation                                                                pooled organic phases dried SpeedVac resuspended 30 &#226;?&#174;l of <br />                                                                                    CMW. All amounts samples loaded layer chromatography <br /> Cells grown 30&#194;&#176;C SD medium OD600 nm &#226;&#171;&#189; 1.0, 50 OD              (TLC) plates (Kieselgel 60; Merck, Darmstadt, Germany), developed in <br /> units cells converted spheroplasts burst 1 ml B88             CHCl3/CH3OH/H2O (5:5:1, vol/vol). TLC-separated lipids detected by <br /> containing protease inhibitors. Unbroken cells removed centrifuga-         autoradiography. <br /> tion 1000 &#226;&#171;&#187; g 5 min. Then, 990 &#226;?&#174;l cleared lysate mixed with <br /> 110 &#226;?&#174;l 10% Triton X-100 kept ice 15 min. The mixture was <br /> centrifuged 100,000 &#226;&#171;&#187; g 60 min. The supernatant recovered        Radiolabeling Proteins myo-2-[3H]Inositol <br /> fraction &#226;&#8364;&#339;input.&#226;&#8364;? The rest supernatant mixed anti-myc mono-        Labeling cells performed described labeling with <br /> clonal (9E10) antibody kept gently rotating 4&#194;&#176;C 1 h. Protein            120 &#226;?&#174;Ci [3H]inositol 2 h. Washed cells resuspended 200 &#226;?&#174;l of <br /> A-Sepharose beads (GE Healthcare, Little Chalfont, Buckinghamshire, United         lysis buffer (50 mM Tris-Cl, pH 6.8, 5 mM EDTA, pH 8.0, 1% SDS, 1% <br /> Kingdom) washed B88 containing 1% Triton X-100 added                 2-melchaptoethanol, protease inhibitors), glass beads (0.2 g) were <br /> incubation continued 4&#194;&#176;C 1 h further. The mixture centrifuged       added suspension. Cells lysed vortexing times 1 min <br /> at 1000 &#226;&#171;&#187; g 5 min, supernatant recovered fraction           each, placing cells ice vortexing cycle, boiled for <br /> &#226;&#8364;&#339;unbound.&#226;&#8364;? The beads washed 4 times B88 containing 1% Triton             5 min. Four milliliters Con A buffer (50 mM Tris-Cl, pH 7.5, 1% Triton <br /> X-100 washed 1 ml 5 mM ammonium acetate. The bound                X-100, 500 mM NaCl, 1 mM CaCl2, 1 mM MgCl2, 1 mM MnCl2) was <br /> proteins eluted 50 &#226;?&#174;l 0.5 M acetic acid/ammonium acetate, pH          added boiled suspension, mixture centrifuged 15,000 &#226;&#171;&#187; g. <br /> 3.4, recovered fraction &#226;&#8364;&#339;bound.&#226;&#8364;? The volume fraction       Con A-Sepharose (GE Healthcare) washed twice Con A buffer added <br /> was subjected SDS-PAGE, indicated proteins detected         supernatant rotated overnight. Proteins recovered boiling <br /> Western blotting.                                                                  Con A-Sepharose SDS-PAGE sample buffer 5 min, separated SDS- <br />                                                                                    PAGE 4 &#226;&#8364;&#8220;20% gradient gel (Daiichi, Tokyo, Japan), de- <br /> Invertase Detection                                                                tected autoradiography. <br /> Cells incubated YPD OD600 nm &#226;&#171;&#189; 1.0 25&#194;&#176;C, 10 OD <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cells <br /> were shifted 25 37&#194;&#176;C, incubation continued 30 min. Cells         RESULTS <br /> were washed YPS (2% sucrose instead glucose YPD) incubated in <br /> YPS 3 h temperature incubated before,           Previously Uncharacterized Pga1 Protein Is an <br /> which cells collected resuspended solution A (25 mM Tris, pH <br /> 8.0, 0.6 M sorbitol, 2.5 mM sodium azide, 0.1% &#226;?&#164;-mercaptoethanol, 0.25         ER-resident Membrane Glycoprotein <br /> mg/ml Zymolyase [Zymo Research, Orange, CA]). After 37&#194;&#176;C incubation            We searched Saccharomyces Genome Database (SGD) for <br /> 30 min, mixture centrifuged 13,400 &#226;&#171;&#187; g 10 min 4&#194;&#176;C,       essential proteins predicted transmembrane domain(s) <br /> supernatant subjected native PAGE. Secretory invertase gel was <br /> detected activity staining described Ballou (1990).                       proteins execute essential functions ER. <br />                                                                                    We introduced plasmids constructed produce GFP- <br /> Pulse-Chase Experiments CPY Gas1                                            fusion protein diploid strain the <br /> Yeast strains grown 25&#194;&#176;C low-sulfate SD OD600 nm &#226;&#171;&#189; 1.0. Two          alleles replaced kanMX4. A haploid deletion strain <br /> and half OD units cells harvested centrifugation resus-           producing GFP-fusion protein obtained tetrad anal- <br /> pended 100 &#226;?&#174;l SD lacking sulfate. Pulse labeling initiated 30 min        ysis, localization GFP observed fluores- <br /> after shift restrictive temperature 37&#194;&#176;C addition 50 &#226;?&#174;Ci <br /> of Tran[35S]-<span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span> (MP Biomedicals, Irvine, CA). After 5 min pulse labeling, <br />                                                                                    cent microscopy. We Ynl158w-GFP showed the <br /> the chase period initiated addition prewarmed 500 &#226;?&#174;l chase        localization nucleus cell periphery, <br /> solution (SD 0.5% casamino acids [BD Biosciences]) terminated          <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> common ER proteins (Figure 2A). <br /> transferring cells tube ice containing 100 &#226;?&#174;l 60 mM sodium          YNL158w/PGA1 essential gene gene disruptants <br /> azide. After minimum 10 min ice, cells collected broken <br /> by vortexing glass beads TBS containing 1% SDS protease <br />                                                                                    reported inviable laboratory <br /> inhibitors, boiled 5 min. The boiled mixture added      culture conditions (Duenas et al., 1999). According SGD, it <br /> with 4 volumes TBS containing 2% Triton X-100, centrifuged               encodes 198 amino-acid polypeptide molecular <br /> 15,000 &#226;&#171;&#187; g 5 min. The supernatant added anti-CPY anti-Gas1         weight &#226;&#171;&#189; 22,383 pI &#226;&#171;&#189; 6.63, biochemical function of <br /> antibody, rotated overnight. Protein A-Sepharose washed <br /> with B88 containing 1% Triton X-100 added mixture rotated for <br />                                                                                    remains unknown. Homologous sequences were <br /> 1 h. Recovered protein A-Sepharose washed TBS containing 1% Triton          fungal genome databases FASTAP <br /> X-100 5 times. Immunoprecipitated proteins eluted boiling SDS-          search (Figure 1A), homologues the <br />  <br /> Vol. 18, September 2007                                                                                                                                      3475 <br />  K. Sato et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Sequence alignment Pga1 protein homologues hydropathy profile Pga1 protein. (A) Amino acid sequence <br /> alignment Pga1 homologues. Two hydrophobic segments indicated bars sequences, predicted signal peptide <br /> cleavage site shown arrowhead. Nine terminal amino acids A. gossypii sequence omitted adjust amino-acid <br /> alignment predicted signal peptides. The identical amino acids highlighted reverse letters. (B) The hydrophobicity index was <br /> calculated according Kyte Doolittle (1982), window size 10 amino acids. Bold lines indicate hydrophobic domains. <br />  <br />  <br />  <br /> database higher eukaryotes. The hydropathy plot indi-                 protein. Next, tested Pga1 actually integral to <br /> cated hydrophobic segments N               ER membrane. As shown Figure 2C, Pga1-6myc was <br /> C termini (Figure 1B). SignalP predicted N-terminal                  recovered precipitate presence 1 M <br /> segment functions signal peptide. The sequence 23Ala-             sodium chloride, 0.1 M sodium carbonate, pH 11, 2 M <br /> Asn probable cleavage site, cleavage                urea, soluble presence 1% Triton <br /> finally generates polypeptide 175 amino-acid residues.              X-100. Therefore, concluded Pga1 integral the <br />    To confirm localization ER, fused 6myc           ER membrane. <br /> C terminus Pga1 expressed &#226;&#338;&#172;pga1 cells,                There potential N-glycosylation sites, 72Asn-Thr- <br /> compared localization ER-marker Kar2 indi-               Thr 80Asn-Thr-Thr, hydrophilic region Pga1 <br /> rect fluorescence confocal microscopy. The Pga1-6myc                polypeptide. To obtain topological information, exam- <br /> showed pattern Pga1-GFP, colocalized                 ined effect endoglycosidase H (endoH) treatment on <br /> Kar2 (Figure 2A). Next, performed biochemical analysis                Pga1-6myc. As shown Figure 2D, 41-kDa band disap- <br /> of Pga1-6myc. The Pga1-6myc detected immu-                    peared 36-kDa band appeared endoH digestion. <br /> noblotting, using anti-myc antibody main band                Therefore, Pga1 accepted N-glycosylation, indicates <br /> of 41 kDa. By differential centrifugation cell lysate,            hydrophilic domain localizes lumen the <br /> Pga1-6myc recovered mainly P10 fraction                   ER. Cells expressing Pga1 tagged 6myc N <br /> some P100 fraction, detected                terminus (6myc-Pga1) accumulated 6myc-Pga1 whose <br /> soluble S100 fraction (Figure 2B). The ER membrane protein               mobility SDS-PAGE affected endoH treat- <br /> Scs2 similarly recovered Pga1, early                  ment (data shown), indicate N-termi- <br /> Golgi marker Van1 recovered evenly P10                 nal tagging Pga1 disturbed translocation ER <br /> P100 fraction, late Golgi marker Kex2 mainly                 lumen. This consistent SignalP prediction that <br /> recovered P100 fraction. Together results                 N-terminal hydrophobic segment Pga1 functions as <br /> microscopy observation, Pga1 ER-localized                  signal peptide. <br />  <br /> 3476                                                                                                          Molecular Biology Cell <br />                                                                                              Novel GPI-Mannosyltransferase II Subunit <br />  <br />  <br />                                                                       ER. Therefore, examined translocation, <br />                                                                       modification, and/or transport secretory proteins was <br />                                                                       affected pga1 mutants. The process protein trans- <br />                                                                       port analyzed pulse-chase experiments Gas1 <br />                                                                       (Nuoffer al., 1993; Popolo Vai, 1999) CPY (Van <br />                                                                       Den Hazel et al., 1996). A GPI-anchored plasma membrane <br />                                                                       protein Gas1 enters ER accompanied cleav- <br />                                                                       age N-terminal secretion signal peptide, core <br />                                                                       glycosylation N- O-carbohydrate modification <br />                                                                       sites, 105-kDa ER form proGas1 <br />                                                                       (Kodukula et al., 1993). Then, proGas1 C-terminal hydro- <br />                                                                       phobic signal peptide replaced GPI-anchor. This <br />                                                                       GPI-anchored proGas1 transported plasma mem- <br />                                                                       brane Golgi receives addi- <br />                                                                       tion carbohydrates glycosyl chains, becomes <br />                                                                       125-kDa mature form. ProCPY receives core N-glycosyl- <br />                                                                       ation ER (form p1), additional glycosylation the <br />                                                                       Golgi (form p2), finally mature enzyme <br />                                                                       (form m) removal propeptide vacuole. As <br />                                                                       shown Figure 3A, 105-kDa precursor Gas1 was <br />                                                                       largely converted mature 125-kDa form a <br />                                                                       30-min chase wild type 25 37&#194;&#176;C. In the <br />                                                                       pga1-1 mutant, mature form appeared 25&#194;&#176;C at <br />                                                                       37&#194;&#176;C. A large ER-form Gas1 precursor accu- <br />                                                                       mulated 37&#194;&#176;C faint signal mature form <br />                                                                       appeared 30-min chase. In sec12-ts mutant de- <br />                                                                       fective formation COPII transport vesicles also <br />                                                                       failed produce mature-form Gas1 37&#194;&#176;C accu- <br />                                                                       mulated precursor somewhat reduced electrophoretic <br /> Figure 2. Pga1 localizes ER glycosylated integral mem-    mobility. In contrast, transport CPY normally oc- <br /> brane protein. Pga1-GFP Pga1-6myc produced CEN            curred pga1-1 mutant 25 37&#194;&#176;C, the <br /> plasmid &#226;&#338;&#172;pga1 cells, KSY33 KSY63. (A) Confocal microscopic      sec12-ts mutant accumulated ER-form precursor CPY <br /> observation Pga1-GFP Pga1-6myc. The immunofluorescence         (Novick et al., 1980). These results suggest Pga1 an <br /> images ER-marker Kar2 protein merged images       essential role processing transport GPI-an- <br /> Pga1-6myc shown. (B) Subcellular fractionation Pga1-      chored membrane protein Gas1 soluble <br /> 6myc, ER-marker Scs2, early Golgi marker Van1, late   vacuolar protein CPY. Secretion N-glycosylation the <br /> Golgi marker Kex2. (C) Pga1-6myc ER membrane protein Scs2 <br /> were subjected solubilization test using 1 M NaCl, 0.1 M <br />                                                                       periplasmic invertase normal pga1-1 mutant <br /> Na2CO3, 2 M urea, 1% Triton X-100. (D) The molecular mass      (Figure 3B; Novick et al., 1980). The O-glycosylation the <br /> Pga1p-6myc (&#226;&#171;&#186;) (&#226;&#171;&#185;) endoglycosidase H treatment   extracellular Kre9 (Lussier et al., 1995) normal in <br /> determined SDS-PAGE Western blotting.                          pga1-1 mutant (data shown). The retention of <br />                                                                       ER lumenal proteins (Lewis et al., 1997) normal, <br />                                                                       leakage Kar2 medium detected <br /> Isolation Mutants Temperature-sensitive Alleles               (Figure 3C). ProALP transported Golgi to <br /> of PGA1                                                               vacuole passing endosome, the <br /> Next, sought create temperature-sensitive alleles            propeptide removed vacuole (Klinosky Emr, <br /> PGA1 error-prone PCR mutagenesis investigate                1989). An accumulation soluble-form ALP in <br /> biological role Pga1. We obtained mutant PGA1 plas-          pga1-1 mutant (Figure 3D), consistent the <br /> mids [pKSTS16 (pga1-1), pKSTS13 (pga1-2), pKSTS15 (pga1-          observation previous report (Davierwala et al., 2005). <br /> 3)], supported growth &#226;&#338;&#172;pga1 haploid strain            Because sec12ts cells showed similar accumulation of <br /> 25&#194;&#176;C, 37&#194;&#176;C. Nucleotide sequences indicated          soluble-form ALP permissive temperature, might <br /> number amino acid substitutions occurred            consequence aberrant condition ER. <br /> mutant alleles. The amino acid substitutions pga1-1 <br /> <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> F20I, M64T, T74P, D75G, I107F, N111S,                   The proGas1 Protein Accumulated pga1ts Mutants Is <br /> T126A, Y144H, P152S, Y169H; pga1-2                  Not GPI-Anchored <br /> F36Y, I57T, Y95C, C100Y, N110S; pga1-3               Accumulation proGas1 ER result either <br /> were T74A, D92G, W101R, I107V, Q113R, T121A, T136A,                   deficiencies. First, GPI-anchor attachment is <br /> N192S, L196P, K197E. These substitutions collectively             required exit GPI target proteins ER <br /> contributed mutant <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>, reversion            (Nuoffer et al., 1993; Doering Schekman, 1996), failure <br /> of residues resulted loss temperature sensitivity,     produce attach GPI result proGas1 accumu- <br /> but residues did temperature sensitivity.        lation. Second, mutants specifically defective trans- <br /> The chromosomal wild-type PGA1 allele replaced           port GPI-anchored proteins accumulate 105-kDa <br /> mutant allele pga1-1 pga1-3 homologous recombination            Gas1. Because GPI-anchored 105-kDa Gas1 protein is <br /> for analysis.                                                 transported Golgi COPII vesicle assis- <br />                                                                       tance p24-family proteins Emp24 Erv25 (Muniz et al., <br /> Characterization pga1ts Mutants                                    2000; Belden Barlowe, 2001), &#226;&#338;&#172;emp24 mutant cell, <br /> Localization ER suggested Pga1 essential           exit Gas1 ER delayed 105-kDa <br /> role involved phenomenon specifically occurs           Gas1 precursor accumulates. To determine deficiency <br />  <br /> Vol. 18, September 2007                                                                                                        3477 <br />  K. Sato et al. <br />  <br />  <br />  <br />  <br />                                                                               Figure 3. The secretion-related characteristics of <br />                                                                               temperature-sensitive pga1-1 mutant. (A) The <br />                                                                               transport CPY Gas1 ER pga1-1 <br />                                                                               cells analyzed pulse-chase experiment at <br />                                                                               25 37&#194;&#176;C. Proteins pulse labeled with <br />                                                                               Tran[35S]-label 5 min chased indicated <br />                                                                               times. The immunoprecipitated CPY Gas1 were <br />                                                                               detected SDS-PAGE autoradiography. ER, <br />                                                                               ER-form proGas1(105 kDa); Golgi, Golgi- <br />                                                                               form Gas1(125 kDa); p1, precursor CPY ER; <br />                                                                               p2, precursor CPY Golgi; m, mature CPY in <br />                                                                               vacuole. (B) The N-glycosylation secretory <br />                                                                               invertase pga1-1 cells 25 37&#194;&#176;C was <br />                                                                               analyzed native PAGE activity staining. (C) <br />                                                                               The leakage Kar2 protein (BiP) medium in <br />                                                                               wild-type (WT), &#226;&#338;&#172;erd1, &#226;&#338;&#172;pga1/PGA1 pga1-1 <br />                                                                               cells analyzed 25&#194;&#176;C membrane filter trap- <br />                                                                               ping immunological detection BiP. (D) The <br />                                                                               processing ALP precursor WT, sec12ts, and <br />                                                                               pga1-1 cells 25 37&#194;&#176;C analyzed by <br />                                                                               SDS-PAGE Western blotting. p, precursor (ER) <br />                                                                               form; m, mature (vacuolar) form; s, soluble form of <br />                                                                               ALP. <br />  <br />  <br /> caused defect Gas1 processing observed pga1-1      GPI anchoring. Because major GPI-anchored proteins are <br /> mutant, proteins subjected Triton X-114 phase        glycoproteins, inositol-incorporated proteins can <br /> separation distinguishes hydrophobicity (Doering        recovered Con A-Sepharose (Guillas et al., 2000). As <br /> Schekman, 1996). Cells grown semipermissive          shown Figure 4C, [3H]inositol incorporated a <br /> temperature pga1-1 mutant (34&#194;&#176;C) 12 h. The           number proteins wild type, small <br /> 105-kDa band, represents precursor form Gas1        [3H]inositol detected pga1-1 mutant <br /> in ER faintly detected total sample   cells 37&#194;&#176;C. The equivalence applied proteins con- <br /> wild-type cell (Figure 4A, lane 1), signifi-      firmed Coomassie Brilliant Blue staining (Figure 4B). <br /> cantly increased &#226;&#338;&#172;emp24 pga1-1 mutant cells (Fig-     These results indicated pga1-1 defective GPI <br /> ure 4A, lanes 4 7). In pga1-1 mutant cells, signifi-   synthesis anchoring target protein. <br /> cant 125-kDa band represents mature <br /> form Gas1 detected, probably mutant <br /> Pga1-1 completely inactive condition; how-       pga1ts Mutants Display Phenotypes Related to <br /> ever, 105-kDa band abundant 125-           GPI-anchoring Defect <br /> kDa band. GPI attachment increases hydrophobicity         The mutants defective GPI-anchoring generally cell <br /> target proteins known cause increase         wall defects partially suppressed increasing the <br /> amount target proteins recovered detergent phase       osmolarity adding 1 M sorbitol growth medium. <br /> when separated Triton X-114. Consistent pre-       Similarly, pga1-1 mutant did form colonies the <br /> diction, 125-kDa Gas1 cell 105-kDa Gas1      YPD plate 36&#194;&#176;C 2 d, temperature sensitivity <br /> accumulated &#226;&#338;&#172;emp24 mutant mainly detected          partially suppressed adding 1 M sorbitol YPD <br /> detergent phase, indicating Gas1 GPI-anchored         (Figure 5A). At sublethal temperature (34&#194;&#176;C), significant <br /> (Figure 4A, lanes 3, 6, 9). In contrast, ER-form      increase Kar2p protein ER lumen was <br /> Gas1 accumulated pga1-1 cells mainly detected      pga1-1 mutant (Figure 5B). This indicates the <br /> aqueous phase (Figure 4A, lane 8). So, 105-kDa Gas1       pga1 mutant induced unfolded protein response (UPR), <br /> the pga1-1 mutant cell suggested represent Gas1       commonly observed GPI mutants (Ng et al., 2000, <br /> that GPI-anchored.                                       Davydenko et al., 2005). In contrast, &#226;&#338;&#172;emp24 mutant that <br />   Because detectable protein-linked inositol yeast     accumulates ER-form Gas1 precursor defect in <br /> present GPI-anchors, distinguish       transport GPI-anchored proteins ER does not <br /> GPIs attached target proteins checking        UPR induction, suggesting UPR induced by <br /> inositol incorporation proteins. Therefore, metaboli-      accumulation GPI precursor target protein <br /> cally labeled cell [3H]inositol, analyzed    GPI-anchoring, accumulation GPI- <br /> total protein modification GPI-anchor examine      anchored proteins. Mutants defective GPI-anchoring are <br /> whether pga1-1 mutant cell actually defective      typically hypersensitive chitin-binding dye Calcofluor <br />  <br /> 3478                                                                                                 Molecular Biology Cell <br />                                                           Novel GPI-Mannosyltransferase II Subunit <br />  <br />  <br />  <br />  <br /> Figure 4. The pga1-1 mutant showed temperature- <br /> sensitive defects GPI-anchoring Gas1 and <br /> other GPI-anchored proteins. (A) Cleared lysate (to- <br /> tal, T) WT, &#226;&#338;&#172;emp24, pga1-1 cells was <br /> treated 1% Triton X-114 separated the <br /> aqueous (A) detergent (D) phases. Gas1, inte- <br /> gral ER membrane protein Scs2 cytosolic <br /> 3-phosphoglycerate kinase (Pgk1) detected by <br /> SDS-PAGE Western blotting. (B) Cells were <br /> metabolically labeled [3H]inositol glycop- <br /> roteins collected using Con A-Sepharose. The <br /> wild-type (lanes 1 2) pga1-1 (lanes 3 4) <br /> cells grown 25&#194;&#176;C (lanes 1 3) or <br /> 37&#194;&#176;C (lanes 2 4). The concentrated glycoproteins <br /> were resolved SDS-PAGE 4 &#226;&#8364;&#8220;20% gradient gel <br /> and detected Coomassie Brilliant Blue stain to <br /> confirm proteins was <br /> loaded lane. Lane M, molecular weight <br /> markers. (C) [3H]inositol incorporated gly- <br /> coproteins detected autoradiography the <br /> same gel. Lanes B. <br />  <br />  <br />  <br />  <br /> Figure 5. The pga1-1 mutant displayed pheno- <br /> types related defect GPI-anchoring. (A) The <br /> temperature sensitivity pga1-1 partially sup- <br /> pressed high osmolarity. The WT pga1-1 <br /> mutant streaked YPD 1 M <br /> sorbitol grown 2 d 36&#194;&#176;C. (B) Unfolded protein <br /> response induced pga1-1 mutant cells. The <br /> amount Kar2 protein (BiP) WT, pga1-1 <br /> mutant &#226;&#338;&#172;emp24 mutant grown 34&#194;&#176;C an- <br /> alyzed SDS-PAGE Western blotting, using <br /> the ER membrane protein Scs2 loading con- <br /> trol. (C) The pga1-1 mutant showed increased sensi- <br /> tivity Calcofluor white. Ten microliters 10-fold <br /> serially diluted suspension WT, &#226;&#338;&#172;gas1, and <br /> pga1-1 mutant cells spotted SD agar or <br /> without 40 &#226;?&#174;g/ml Calcofluor White, the <br /> cells incubated 2 d 30&#194;&#176;C. <br />  <br /> Vol. 18, September 2007                                                                     3479 <br />  K. Sato et al. <br />  <br />  <br /> White (CFW). At permissive temperature (30&#194;&#176;C), the <br /> pga1-1 mutant showed slightly increased sensitivity to <br /> CFW (Figure 5C). The &#226;&#338;&#172;gas1 mutant showed higher sensi- <br /> tivity. These results strongly support likeliness Pga1 <br /> is actually defective GPI anchoring. <br />  <br /> Interaction Pga1 Gpi18, Essential <br /> Glycosylphosphatidylinositol-Mannosyltransferase II <br /> Component <br /> The defect GPI anchoring pga1-1 mutant strongly <br /> suggested involvement PGA1 GPI synthesis or <br /> anchoring process, steps GPI synthesis <br /> and anchoring target proteins. To determine where <br /> Pga1 functions pathway, screened multi- <br /> copy suppressor genes temperature sensitivity the <br /> pga1-1 mutant, effective clue to <br /> elucidate functional role uncharacterized gene prod- <br /> ucts. We introduced 2&#226;?&#174; plasmid library pga1-1 <br /> mutant cells sought colonies viability 37&#194;&#176;C <br /> were dependent introduced plasmids. As result, we <br /> found GPI18 gene recover growth defect <br /> of pga1-1 mutant 37&#194;&#176;C introduction 2&#226;?&#174; <br /> plasmid (Figure 6A). GPI18 recently reported encode <br /> GPI-MT-II (Fabre et al., 2005; Kang et al., 2005). GPI-MT-II <br /> transfers second Man dolichol-P-Man Man- <br /> GlcN-PI precursor GPI anchor, supposed that <br /> Pga1 function related mannosylation GPI <br /> synthesis. To test possibility Pga1 genetic inter- <br /> action GPI-MTs, introduced GPI14, PBN1, <br /> GPI10, SMP3 2&#226;?&#174; plasmid. GPI14 PBN1 encode <br /> subunits GPI-MT-I (Maeda et al., 2001; Ashida et al., 2005), <br /> GPI10 encodes &#226;?&#163;1,2-mannosyltransferase, and <br /> SMP3 encodes &#226;?&#163;1,2-mannosyltransferase (Grimme et <br /> al., 2001). However, multicopy expression gene <br /> could restore growth pga1-1 mutant 37&#194;&#176;C <br /> (Figure 6B). The multicopy GPI18 suppress the <br /> lethality &#226;&#338;&#172;pga1 null mutant (Figure 6C), indicating <br /> that Pga1 indispensable role related GPI18.             Figure 6. Temperature sensitivity mutants PGA1 and <br />    Because GPI18 rescue growth defects          GPI18 suppressed introducing mutual multicopy gene. <br /> pga1-1 mutant cells, genes encoding GPI-MTs             (A) The colony-forming activity 37&#194;&#176;C pga1-1 mutant re- <br />                                                                   covered GPI18 gene 2&#226;?&#174; plasmid gpi18 &#226;&#8364;&#8220; 6 <br /> could not, specific association PGA1 GPI18 <br />                                                                   mutant recovered PGA1 gene 2&#226;?&#174; plasmid. (B) The <br /> was indicated. Therefore, investigated interaction      high-copy introduction GPI14, PBN1, GPI10, SMP3 genes <br /> the proteins yeast cells. For purpose, PGA1-6myc      responsible mannosylation yeast GPI synthesis did not <br /> and GFP-GPI18 low-copy plasmids expressed              suppress temperature sensitivity pga1-1 mutant 37&#194;&#176;C. (C) <br /> &#226;&#338;&#172;pga1 cells. The fluorescent imaging using GFP-tagged             The heterozygous PGA1/&#226;&#338;&#172;pga1::kanMX4 diploid cells trans- <br /> Gpi18 Pga1-6myc indicated proteins mainly           formed CEN-PGA1 (top) 2&#226;?&#174;-GPI18 (bottom) plasmid, <br /> localized ER, merged images indicated         sporulated. The spores dissected assess <br /> the proteins compartments ER          complementation tetrad analysis. <br /> (Figure 7A). GFP-Gpi18 mainly recovered P10 <br /> fraction subcellular fractionation (Figure 7B), was <br /> only solubilized Triton X-100 solubilization test       potential GPI-anchored proteins 37&#194;&#176;C, tem- <br /> (Figure 7C), indicates Gpi18 integral ER         perature-sensitive growth defect suppressed in- <br /> membrane protein. To examine physical          troduction multicopy GPI18 gene, Pga1 Gpi18 <br /> protein&#226;&#8364;&#8220;protein interaction Gpi18 Pga1,            ER-membrane protein complex the <br /> conducted immunoprecipitation experiments. The mem-               Triton X-100-solubilized lysate, suspected Pga1 and <br /> branes cell lysate solubilized addition     Gpi18 cooperate functioning enzyme to <br /> 1% Triton X-100, Pga1-6myc precipitated using         produce GPI anchor. It reported GPI18- <br /> anti-myc mAb cleared lysate. The GFP-tagged              depleted cell accumulated GPI precursor &#226;&#8364;&#339;lipid 004-1,&#226;&#8364;? <br /> Gpi18 clearly detected precipitate fraction         identified Man[EtN-P]-GlcN-PI (Fabre et al., <br /> immunoblotting (Figure 7D). Therefore, Gpi18 Pga1         2005); thus, tested pga1 mutant accumulates <br /> only colocalized ER physical interaction    GPI precursor. For purpose, constructed <br /> in vivo.                                                          PGAL1-GPI18 strain gpi18ts mutants. Screening for <br />                                                                   gpi18ts mutants performed method by <br /> Pga1 Novel Subunit Glycosylphosphatidylinositol-          pga1ts mutants constructed. As result, we <br /> Mannosyltransferase II                                            obtained 7 gpi18ts alleles replaced GPI18 gene the <br /> Because pga1-1 mutant cells showed defects mod-        chromosome BY4741 gpi18-6 analysis. <br /> ification GPI anchor Gas1 precursor ER       To analyze accumulation GPI anchor precursors, the <br />  <br /> 3480                                                                                                  Molecular Biology Cell <br />                                                                                               Novel GPI-Mannosyltransferase II Subunit <br />  <br />  <br />                                                                       small lipid 004-1 pga1-1/PGA1 be <br />                                                                       caused codominance pga1-1 mutation (Figure 8C, <br />                                                                       lane 4). On contrary, introduction high-copy- <br />                                                                       suppressor genes 2&#226;?&#174; plasmid did diminish accu- <br />                                                                       mulation (Figure 8D). <br />  <br />  <br />                                                                       Gpi18 Is Stable Irrespective Pga1 <br />                                                                       GPI-MT I consists proteins mammalian cells, <br />                                                                       PIG-M PIG-X (Ashida et al., 2005). PIG-M mul- <br />                                                                       timembrane-spanning protein like Gpi18, PIG-X a <br />                                                                       protein single transmembrane domain like Pga1. <br />                                                                       Because PIG-X reported stabilize PIG-M, specu- <br />                                                                       lated loss Pga1 37&#194;&#176;C incubation ts allele of <br />                                                                       Pga1 cause destabilization Gpi18. To test this <br />                                                                       possibility, GFP-Gpi18 expressed PGAL1-PGA1 mu- <br />                                                                       tant pga1-1 mutant cells. Even PGAL1-PGA1 was <br />                                                                       incubated SD medium 20 h lipid 004-1 <br />                                                                       significantly accumulated (data shown), reduction in <br />                                                                       GFP-Gpi18 observed (Figure 9A). Simi- <br />                                                                       larly, incubation 37&#194;&#176;C 2 h did affect stability of <br />                                                                       GFP-Gpi18 pga1ts mutants (Figure 9B), 105- <br />                                                                       kDa proGas1 significantly accumulated (data not <br />                                                                       shown). Therefore, stability Gpi18 independent of <br />                                                                       Pga1. <br />  <br />  <br />                                                                       Human PIG-V Rescues Viability &#226;&#338;&#172;pga1 Cells <br />                                                                       It curious PGA1 homologues fungi not <br />                                                                       higher eukaryotes including mammals, S. <br />                                                                       cerevisiae genes identified far involved GPI syn- <br />                                                                       thesis anchoring mammalian homologues. So, <br />                                                                       natural assume existence counterpart Pga1 <br />                                                                       mammalian cells. If Pga1 counterpart exists, two <br />                                                                       probable reasons apparent PGA1 homologue <br /> Figure 7. Pga1 Gpi18 proteins colocalized ER      concerned. The sequence similarity <br /> found complex Triton X-100 &#226;&#8364;&#8220;solubilized lysate. (A) The   proteins high detected <br /> fluorescent image GFP-Gpi18 indirect immunofluores-        homology search, functionally homol- <br /> cent staining image Pga1-6myc overlapping (merge). (B)        ogous. The function mammalian PIG-V <br /> Subcellular fractionation GFP-Gpi18 performed described     contains Gpi18 Pga1. To <br /> in Figure 2B. (C) Solubilization test GFP-Gpi18 performed as <br />                                                                       discern possibility likely, introduced FLAG- <br /> in Figure 2C. (D) The cleared lysate cells having either <br /> Pga1-6myc untagged Pga1 addition GFP-Gpi18 sub-          human PIG-V pRS316 (pFLAG-hPIG-V, generous gift <br /> jected immunoprecipitation using anti-myc mAb, GFP-Gpi18       Dr. Taroh Kinoshita, Research Institute Microbial <br /> and Pga1-6myc detected SDS-PAGE Western blotting.         Disease; Osaka; Kang et al., 2005) heterozygous <br />                                                                       PGA1/&#226;&#338;&#172;pga1::kanMX4 diploid cells. The spores trans- <br />                                                                       formants dissected assess complementation by <br /> cells metabolically labeled [3H]inositol,             tetrad analysis. As shown Figure 10A, tetrads from <br /> which lipids extracted cells separated           PGA1/&#226;&#338;&#172;pga1::kanNMX4 diploid cells harboring pFLAG- <br /> TLC. [3H]Inositol-containing lipids TLC de-               hPIG-V generated viable haploid progenies of <br /> tected autoradiography. As shown Figure 8, pga1ts           normal growth slow growth. The slow growth <br /> mutants gpi18ts mutants accumulated spot             cells &#226;&#338;&#172;pga1 alleles viability dependent on <br /> same RF 37&#194;&#176;C (Figure 8A, lanes 6 8, B, lanes 6, 8,         pFLAG-hPIG-V resistant Geneticin <br /> 10, 12). This spot detected sample         (G418; Invitrogen) lethal 5-FOA plate, nor- <br /> transcription arrested cells PGAL1-GPI18 promoter              mal growth cells wild-type cells were <br /> shut-off mutant SD (Figure 8A, lane 4); so, corre-            sensitive G418 grew normally 5-FOA plate. <br /> sponds &#226;&#8364;&#339;lipid 004-1&#226;&#8364;?, namely, Man[EtN-P]-GlcN-PI.              The result obtained pFLAG-hPIG-V was <br /> However, spot detected wild type               introduced GPI18/&#226;&#338;&#172;gpi18::kanMX4 diploid; nor- <br /> &#226;&#338;&#172;gpi7 25 37&#194;&#176;C (Figure 8A, lanes 9 10, B, lanes          mal growth haploids sensitive G418 slow growth <br /> 1&#226;&#8364;&#8220; 4), moderate accumulation detected pga1ts           haploids resistant G418 observed tetrads <br /> or gpi18ts mutant cells 25&#194;&#176;C (Figure 8A, lanes 5 7,        (Figure 10B). These results indicate human PIG-V res- <br /> and B, lanes 5, 7, 10, 11). Therefore, functional Pga1        cues viability &#226;&#338;&#172;gpi18 &#226;&#338;&#172;pga1. Further- <br /> was necessary enzyme activity GPI-MT II. We                more, introduction GPI18 &#226;&#338;&#172;pga1 2&#226;?&#174; plas- <br /> concluded Pga1 essential subunit GPI-MT II.             mid rescue viability, indicating the <br /> The accumulation lipid 004-1 pga1-1 gpi18-6             function Gpi18 completely distinct Pga1. <br /> mutants complemented introduction wild-             Thus, human PIG-V strongly suggested possess the <br /> type gene CEN plasmid (Figure 8C). The presence               functions Gpi18 Pga1 S. cerevisiae. <br />  <br /> Vol. 18, September 2007                                                                                                           3481 <br />  K. Sato et al. <br />  <br />  <br /> Figure 8. The temperature-sensitive pga1 <br /> mutants accumulated GPI anchor <br /> precursor accumulated Gpi18- <br /> deprived cells. (A) The myo-2-[3H]inositol-la- <br /> beled lipids extracted, developed a <br /> TLC plate CHCl3/CH3OH/H2O (5:5:1, <br /> vol/vol), detected autoradiography. <br /> The wild-type (lanes 1 2) PGAL1-GPI18 <br /> (lanes 3 4) cells grown 30&#194;&#176;C SG <br /> (lanes 1 3) SD (lanes 2 4). An <br /> intermediate lipid (lipid 004-1, indicated by <br /> the arrowhead) accumulated sample of <br /> lane 4 expression GPI18 was <br /> repressed 20 h glucose-containing <br /> medium. The wild-type (lanes 9 10), <br /> pga1-1 (lanes 5 6), gpi18-6 (lanes 7 and <br /> 8) ts mutant cells labeled 25&#194;&#176;C <br /> (lanes 5, 7, 9) 37&#194;&#176;C (lanes 6, 8, 10). <br /> An accumulation indicated GPI biosyn- <br /> thetic precursor lanes 5, 6, 7, and <br /> 8. (B) The wild-type (WT, lanes 1 2), &#226;&#338;&#172;gpi7 <br /> (lanes 3 4), pga1-1 (lanes 5 6), pga1-3 <br /> (lanes 7 8), gpi18-3 (lanes 9 10), or <br /> gpi18-6 (lanes 11 12) cells grown at <br /> either 25&#194;&#176;C (lanes 1, 3, 5, 7, 9, 11) 37&#194;&#176;C <br /> (lanes 2, 4, 6, 8, 10, 12), labeled with <br /> myo-2-[3H]inositol, accumulation of <br /> the GPI biosynthetic precursor analyzed <br /> as described A. The intermediate lipid <br /> 004-1 indicated arrowhead. (C) The <br /> complementation lipid 004-1 (arrow- <br /> head) accumulation pga1-1 (lanes 1&#226;&#8364;&#8220; 4) <br /> or gpi18-6 (lanes 5&#226;&#8364;&#8220; 8) cells introducing <br /> PGA1 (lanes 3 4) GPI18 (lanes 7 8) <br /> on CEN plasmid examined. The mutants <br /> transformed pRS316 used the <br /> controls (lanes 1, 2, 5, 6). (D) The suppres- <br /> sion lipid 004-1 (arrowhead) accumula- <br /> tion pga1-1 (lanes 1&#226;&#8364;&#8220; 4) gpi18-6 (lanes 5&#226;&#8364;&#8220; 8) cells introducing GPI18 (lanes 3 4) PGA1 (lanes 7 8) 2&#226;?&#174; plasmid was <br /> examined. The mutants transformed pRS426 used controls (lanes 1, 2, 5, 6). <br />  <br />  <br />  <br /> DISCUSSION                                                               screen mutants defective inositol incorpo- <br />                                                                          ration, homologues mamma- <br /> Enormous efforts identify genes                    lian proteins proven involved GPI biosyn- <br /> required yeast GPI biosynthesis. Two major strate- <br />                                                                          thesis. Despite intensive studies, flippase the <br /> gies generally taken S. cerevisiae researchers; <br />                                                                          GPI intermediate membrane GPI remod- <br />                                                                          elase introduce ceramide GPI attached to <br />                                                                          target proteins identified far. In con- <br />                                                                          trast, vitro reaction GPI synthesis using <br />                                                                          purified enzymes substrates demonstrated <br />                                                                          yet, possible exclude undiscovered participant <br />                                                                          step GPI biosynthesis. <br />                                                                             We focused characterization ER-localized <br />                                                                          essential proteins functions remain unknown even <br />                                                                          enormous comprehensive studies S. cerevisiae. <br />                                                                          Our strategy uses construction temperature-sensitive <br />                                                                          mutants suitable phenotypic genetic studies. <br />                                                                          The temperature-sensitive pga1 mutants showed defects in <br />                                                                          processing transport GPI-anchored Gas1 pro- <br />                                                                          tein (Figure 3A) biochemical analysis indicated GPI <br /> Figure 9. The stability Gpi18 affected pga1 mutants.       anchor synthesis defective (Figure 4). Genetic analysis <br /> (A) GFP-Gpi18 expressed WT (lanes 1 4) PGAL1-          identified GPI18 gene multicopy suppressor the <br /> PGA1 (KSY308, lanes 2 5; KSY309, lanes 3 6) cells. After 20-h    growth defect pga1-1 (Figure 6A). Gpi18 reported to <br /> incubation SG (lanes 1&#226;&#8364;&#8220;3) SD medium (lanes 4 &#226;&#8364;&#8220; 6), GFP-Gpi18       GPI-MT II, adds Man Man-GlcN-PI, the <br /> was detected SDS-PAGE Western blotting. KSY308 and <br />                                                                          mutants pga1 gpi18 accumulated same <br /> KSY309 independent isolates construction. (B) GFP- <br /> Gpi18 expressed WT (lanes 1 2) pga1ts           intermediate, Man[EtN-P]-GlcN-PI (Figure 8). Gpi18 and <br /> mutants (pga1-1, lanes 3 4; pga1-2, lanes 5 6). After 2-h        Pga1 colocalized ER membrane were <br /> incubation 25&#194;&#176;C (lanes 1, 3, 5) 37&#194;&#176;C (lanes 2, 4, 6),   complex Triton X-100 solubilization the <br /> GFP-Gpi18 detected described A.                                cell lysate (Figure 7). From results, concluded <br />  <br /> 3482                                                                                                        Molecular Biology Cell <br />                                                                                         Novel GPI-Mannosyltransferase II Subunit <br />  <br />  <br />  <br />  <br /> Figure 10. Human PIG-V rescues viability both <br /> &#226;&#338;&#172;gpi18 &#226;&#338;&#172;pga1. (A) The heterozygous PGA1/&#226;&#338;&#172;pga1:: <br /> kanMX4 diploid cells transformed pFLAG- <br /> hPIG-V sporulated. The spores dissected <br /> to assess complementation tetrad analysis. The <br /> genotypes numbered progenies (1&#226;&#8364;&#8220; 8) were <br /> checked streaking indicated cells G418 <br /> plate 5-FOA plate. (B) The heterozygous <br /> GPI18/&#226;&#338;&#172;gpi18::kanMX4 diploid cells transformed <br /> with pFLAG-hPIG-V sporulated. The geno- <br /> types numbered progenies (9 &#226;&#8364;&#8220;16) checked <br /> as described A. <br />  <br />  <br /> that Pga1 novel subunit GPI-MT II collaborates <br /> with subunit Gpi18 (Figure 10). <br />    As shown Figure 11, steps GPI biosynthesis <br /> are reported carried participation of <br /> multiple proteins (Kinoshita Inoue, 2000; Pittet and <br /> Conzelmann, 2006). Both mammalian budding yeast <br /> GPI-MT I complex multimembrane-spanning pro- <br /> tein (PIG-M/Gpi14) single membrane-spanning pro- <br /> tein (PIG-X/Pbn1). Most GPI-MT polypeptides DXD <br /> motif, generally required activity glyco- <br /> syltransferase. Because PIG-M Gpi14 DXD se- <br /> quence, multimembrane-spanning subunit probably rep- <br /> resents catalytic subunit. The molecular function of <br /> PIG-X Pbn1 unknown, PIG-X reported be <br /> required stability PIG-M (Ashida et al., 2005). So, the <br /> single membrane-spanning subunit play structural <br /> role partner. However, Pga1, repression <br /> of PGA1 GAL1 promoter shut-off incubation of <br /> the pga1-1 mutant 37&#194;&#176;C affected Gpi18 <br /> (Figure 9), suggesting Pga1 required the <br /> stability partner. The overexpression GPI18 could <br /> not rescue lethality &#226;&#338;&#172;pga1. It rescued temper- <br /> ature sensitivity pga1ts. Conversely, PGA1 overexpression     Figure 11. The mannosylation process GPI anchor synthesis. <br /> suppressed temperature sensitivity gpi18ts (Figure        Pga1 partner Gpi18. GPI-MT I GPI-MT II composed <br /> 6A). We immunoprecipitated Pga1-6myc digitonin-         protein complex. The GPI-MT subunits containing DXD motif <br /> solubilized membrane fraction anti-myc antibody,        shown bold. <br />  <br /> Vol. 18, September 2007                                                                                                    3483 <br />  K. Sato et al. <br />  <br />  <br /> but detect interacting protein                 Society Promotion Science (to K.S.), grants-in-aid Scientific <br /> Gpi18 coprecipitated proteins (data shown). These                 Research Japan Society Promotion Science (to Y.N.), and <br />                                                                              grants &#226;&#8364;&#339;Bioarchitect Research&#226;&#8364;? Institute Physical Chemical <br /> data indicate Pga1 Gpi18 required                  Research (RIKEN) Japan (to K.Y.) grant Noda Institute of <br /> generate overall structure necessary substrate                   Scientific Research (to K.Y.). <br /> recognition enzymatic activity yeast GPI-MT II. <br />    The sequences polypeptides GPI-MT II con- <br />                                                                              REFERENCES <br /> tain DXD motif, commonly present glycosyl- <br /> transferases. Instead, Gpi18 PIG-V structural fea-                 Ashida, H., Hong, Y., Murakami, Y., Shishioh, N., Sugimoto, N., Kim, Y. U., <br /> ture common Pmt2-family proteins (Kang et al., 2005).                   Maeda, Y., Kinoshita, T. (2005). Mammalian PIG-X yeast Pbn1p are <br />                                                                              essential components glycosylphosphatidylinositol-mannosyltrans- <br /> The Pmt2-family proteins O-mannosyltransferases,                   ferase I. Mol. Biol. Cell 16, 1439 &#226;&#8364;&#8220;1448. <br /> have multiple transmembrane domains DXD motif. <br /> They conserved hydrophilic loops face ER                   Ballou, C. E. (1990). Isolation, characterization, properties Saccharomy- <br />                                                                              ces cerevisiae mnn mutants nonconditional protein glycosylation defects. <br /> lumen. One loop second transmem-                    Methods Enzymol. 185, 440 &#226;&#8364;&#8220; 470. <br /> brane domains the fifth sixth. <br />                                                                              Belden, W. J., Barlowe, C. (2001). Deletion yeast p24 genes activates the <br /> The loop Pmt2-family protein Trp-Asp                     unfolded protein response. Mol. Biol. Cell 12, 957&#226;&#8364;&#8220;969. <br /> motif Gpi18 PIG-V. In CHO PIG-V mutants, <br />                                                                              Davierwala, A. P. et al. (2005). The synthetic genetic interaction spectrum of <br /> single amino-acid substitutions W66L D67A                       essential genes. Nat. Genet. 37, 1147&#226;&#8364;&#8220;1152. <br /> conserved Trp-Asp motif responsible mutant <br />                                                                              Davydenko, S. G., Feng, D., Ja&#194;&#168;ntti, J., Kera&#194;&#168;nen, S. (2005). Characterization <br /> phenotype (Kang et al., 2005). Considering result                   GPI14/YJR013w mutation induces cell wall integrity signaling path- <br /> structural features, Gpi18 likely catalytic subunit            way results increased protein production Saccharomyces cerevisiae. <br /> the yeast GPI-MT II, requires partner Pga1                Yeast 22, 993&#226;&#8364;&#8220;1009. <br /> activity.                                                                    Doering, T. L., Schekman, R. (1996). GPI anchor attachment required <br />    No obvious Pga1 homologue mammalian                          Gas1p transport endoplasmic reticulum COP II vesicles. <br /> cells. There possible explanations Pga1                   EMBO J. 15, 182&#226;&#8364;&#8220;191. <br /> required fungi. First, Pga1 function necessary                Duenas, E., Revuelta, J. L., del Rey, F., Vazquez Aldana, C. R. (1999). <br /> only yeast cells completely specific yeast. Second,                Disruption basic phenotypic analysis novel genes left arm <br />                                                                              chromosome XIV Saccharomyces cerevisiae. Yeast 15, 63&#226;&#8364;&#8220;72. <br /> there functional homologue low similarity. <br /> Third, PIG-V possesses functions Gpi18                       Eisenhaber, B., Maurer-Stroh, S., Novatchkova, M., Schneider, G., and <br />                                                                              Eisenhaber, F. (2003). Enzymes auxiliary factors GPI lipid anchor <br /> Pga1. We supposed possibility unlikely                    biosynthesis post-translational transfer proteins. Bioessays 25, 367&#226;&#8364;&#8220;385. <br /> because GPI biosynthetic pathway conserved between <br /> the yeast mammalian cell intermediates                  Englund, P. T. (1993). The structure biosynthesis glycosyl phosphati- <br />                                                                              dylinositol protein anchors. Annu. Rev. Biochem. 62, 121&#226;&#8364;&#8220;138. <br /> and enzymes pathway common. The sec- <br /> ond possibility likely; however, possibility                Fabre, A. L., Orlean, P., Taron, C. H. (2005). Saccharomyces cerevisiae <br />                                                                              Ybr004c human homologue required addition second <br /> was likely expression human PIG-V                    mannose glycosylphosphatidylinositol precursor assembly. FEBS J. <br /> gene &#226;&#338;&#172;gpi18 &#226;&#338;&#172;pga1 cells suppress                     272, 1160 &#226;&#8364;&#8220;1168. <br /> lethality (Figure 10). So, natural suppose             Grimme, S. J., Westfall, B. A., Wiedman, J. M., Taron, C. H., Orlean, P. <br /> yeast proteins compose GPI-MT II complex,                       (2001). The essential Smp3 protein required addition side- <br /> functionally homologous single human PIG-V protein.                   branching fourth mannose assembly yeast glycosylphosphatidyli- <br /> Compared human PIG-V, hydrophilic segments                       nositols. J. Biol. Chem. 276, 27731&#226;&#8364;&#8220;27739. <br /> the sides seventh transmembrane domain                       Guillas, I., Pfefferli, M., Conzelmann, A. (2000). Analysis ceramides <br /> missing yeast Gpi18 (figure 4 Kang et al., 2005). It                present glycosylphosphatidylinositol anchored proteins Saccharomyces <br />                                                                              cerevisiae. Methods Enzymol. 312, 506 &#226;&#8364;&#8220;515. <br /> possible Pga1 compensates function these <br /> missing segments forming complex Gpi18                      Hazbun, T. R. et al. (2003). Assigning function yeast proteins integration <br />                                                                              technologies. Mol. Cell 12, 1353&#226;&#8364;&#8220;1365. <br /> membrane. Because PIG-V homologue fruit fly has <br /> similar missing stretches, require homologue                     Imai, K., Noda, Y., Adachi, H., Yoda, K. (2005). A novel endoplasmic <br />                                                                              reticulum membrane protein Rcr1 regulates chitin deposition cell wall <br /> Pga1 activity.                                                       Saccharomyces cerevisiae. J. Biol. Chem. 280, 8275&#226;&#8364;&#8220; 8284. <br />    Considering GPI-MT I GPI-MT II com- <br />                                                                              Inadome, H., Noda, Y., Adachi, H., Yoda, K. (2005). Immunoisolation of <br /> posed (at least) subunits S. cerevisiae, natural          yeast Golgi subcompartments characterization novel membrane <br /> ask GPI-MT III IV forms similar complex,                        protein, Svp26, discovered Sed5-containing compartments. Mol. Cell. <br /> because using substrates structures similar                 Biol. 25, 7696 &#226;&#8364;&#8220;7710. <br /> those used GPI-MT I GPI-MT II. The proteins                      Kang, J. Y., Hong, Y., Ashida, H., Shishioh, N., Murakami, Y., Morita, Y. S., <br /> have proved responsible mannosylation                  Maeda, Y., Kinoshita, T. (2005). PIG-V involved transferring second <br /> GPI-MT III IV Gpi10 Smp3, respec-                      mannose glycosylphosphatidylinositol. J. Biol. Chem. 280, 9489 &#226;&#8364;&#8220;9497. <br /> tively, multimembrane-spanning proteins                    Kilmartin, J. V., Adams, A, E. (1984). Structural rearrangements tubulin <br /> like Gpi14 Gpi18. It interesting per-                   actin cell cycle yeast Saccharomyces. J. Cell Biol. 98, <br /> form multicopy suppressor screening gpi10 smp3                     922&#226;&#8364;&#8220;933. <br /> temperature-sensitive mutants                     Kinoshita, T., Inoue, N. (2000). Dissecting manipulating pathway <br /> be partner proteins them.                                             glycosylphosphatidylinositol-anchor biogenesis. Curr. Opin. Chem. Biol. <br />                                                                              4, 632&#226;&#8364;&#8220; 638. <br />                                                                              Klinosky, D. J., Emr, S. D. (1989). Membrane protein sorting: biosynthesis, <br /> ACKNOWLEDGMENTS                                                              transport processing yeast vacuolar alkaline phosphatase. EMBO J. 8, <br />                                                                              2241&#226;&#8364;&#8220;2250. <br /> We thank Drs. S. Kagiwada M. Tokunaga kindly providing antisera; <br />                                                                              Kleizen, B., Braakman, I. (2004). Protein folding quality control the <br /> Dr. Taroh Kinoshita kindly providing human PIG-V expression plas- <br />                                                                              endoplasmic reticulum. Curr. Opin. Cell Biol. 16, 343&#226;&#8364;&#8220;349. <br /> mids; Chihiro Akimoto Atsushi Kamiya help preliminary charac- <br /> terization PGA1; Mariko Umemura Dr. Takehiko Yoko-o valuable      Kodukula, K., Gerber, L. D., Amthauer, R., Brink, L., Udenfriend, S. <br /> advices analysis GPI precursors; Dr. Hironori Inadome, Dr. Shah    (1993). Biosynthesis glycosylphosphatidylinositol (GPI)-anchored mem- <br /> Alam Bhuiyan Dr. Hiroyuki Adachi helpful discussions. This work      brane proteins intact cells: specific amino acid requirements adjacent the <br /> was supported Research Fellowship Young Scientists Japan   site cleavage GPI attachment. J. Cell Biol. 120, 657&#226;&#8364;&#8220; 664. <br />  <br />  <br /> 3484                                                                                                                      Molecular Biology Cell <br />                                                                                                                       Novel GPI-Mannosyltransferase II Subunit <br />  <br /> Kyte, J., Doolittle, R. F. (1982). A simple method displaying               Nuoffer, C., Horvath, A., Riezman, H. (1993). Analysis sequence <br /> hydropathic character protein. J. Mol. Biol. 157, 105&#226;&#8364;&#8220;132.                           requirements glycosylphosphatidylinositol anchoring Saccharomyces cer- <br />                                                                                         evisiae Gas1 protein. J. Biol. Chem. 268, 10558 &#226;&#8364;&#8220;10563. <br /> Lewis, M. J., Rayner, J. C., Pelham, H. R. (1997). A novel SNARE complex <br /> implicated vesicle fusion endoplasmic reticulum. EMBO J. 16,                Palade, G. (1975). Intracellular aspects process protein synthesis. <br /> 3017&#226;&#8364;&#8220;3024.                                                                              Science 189, 347&#226;&#8364;&#8220;358. <br />  <br /> Lu, C. F., Montijn, R. C., Brown, J. L., Klis, F., Kurjan, J., Bussey, H., Lipke,   Pittet, M., Conzelmann, A. (2006). Biosynthesis function GPI <br /> P. N. (1995). Glycosyl phosphatidylinositol-dependent cross-linking &#226;?&#163;-ag-            proteins yeast Saccharomyces cerevisiae. Biochim. Biophys. Acta 1771, <br /> glutinin &#226;?&#164; 1,6-glucan Saccharomyces cer <br /> </body></html>