<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Nov. 2006, p. 3665&#226;&#8364;&#8220;3673                                                                                      Vol. 50, No. 11 <br /> 0066-4804/06/$08.00&#226;&#171;&#185;0 doi:10.1128/AAC.00555-06 <br /> Copyright &#194;&#169; 2006, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  Peptide Deformylase Inhibitors Potent Antimycobacterial Agents&#228;&#338;&#164;&#226;&#8364;&#160; <br />    Jeanette W. P. Teo,1&#226;&#8364;&#161; Pamela Thayalan,1 David Beer,1 Amelia S. L. Yap,1 Mahesh Nanjundappa,1 <br />         Xinyi Ngew,1 Jeyaraj Duraiswamy,1 Sarah Liung,1 Veronique Dartois,1 Mark Schreiber,1 <br />            Samiul Hasan,1 Michael Cynamon,2 Neil S. Ryder,3 Xia Yang,3 Beat Weidmann,3 <br />                      Kathryn Bracken,3 Thomas Dick,1 Kakoli Mukherjee1* <br />        Novartis Institute Tropical Diseases, 10 Biopolis Road, 05-01 Chromos, Singapore 138670, Republic Singapore1; <br />         Central New York Research Corporation, New York, New York2; Novartis Institutes Biomedical Research, <br />                        Inc., Infectious Disease Area, 100 Technology Square, Cambridge, Massachusetts 021393 <br />                              Received 4 May 2006/Returned modification 6 June 2006/Accepted 17 August 2006 <br />  <br />              Peptide deformylase (PDF) catalyzes hydrolytic removal N-terminal formyl group nascent <br />           proteins. This essential step bacterial protein synthesis, making PDF attractive target anti- <br />           bacterial drug development. Essentiality def gene, encoding PDF Mycobacterium tuberculosis, was <br />           demonstrated genetic <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>knockout</span> experiments Mycobacterium bovis BCG. PDF M. tuberculosis <br />           strain H37Rv cloned, expressed, purified N-terminal histidine-tagged recombinant protein in <br />           Escherichia coli. A novel class PDF inhibitors (PDF-I), N-alkyl urea hydroxamic acids, synthesized <br />           evaluated activities M. tuberculosis PDF enzyme antimycobacterial <br />           effects. Several compounds new class 50% inhibitory concentration (IC50) values &lt;100 nM. <br />           Some PDF-I displayed antibacterial activity M. tuberculosis, including MDR strains MIC90 <br />           values &lt;1 &#226;?&#174;M. Pharmacokinetic studies potential leads showed compounds orally bio- <br />           available. Spontaneous resistance inhibitors arose frequency &lt;5 &#226;&#180;&#8250; 10&#226;&#180;&#353;7 M. bovis BCG. <br />           DNA sequence analysis spontaneous PDF-I-resistant mutants revealed half mutants had <br />           acquired point mutations formyl methyltransferase gene (fmt), formylated Met-tRNA. The <br />           results study validate M. tuberculosis PDF drug target suggest class compounds <br />           potential developed novel antimycobacterial agents. <br />  <br />  <br />    Mycobacterium tuberculosis, pathogen causing 2 million                       active bacterial PDF challenge (51). So far, Ni2&#226;&#171;&#185;, <br /> deaths 8 million new cases tuberculosis (TB) year,                    Co2&#226;&#171;&#185;, Mn2&#226;&#171;&#185; described cations stably <br /> is leading health threat. This fueled human                   replace Fe&#226;&#171;&#185;2 retaining high enzymatic activity (25, 49). <br /> immunodeficiency virus epidemic appearance mul-                          Naturally occurring antibiotics, like actinonin, inhibit the <br /> tidrug-resistant TB (17, 18, 46). Infections involving multidrug-                 activity PDF enzyme (2). Based structural (7, 39, 41) <br /> resistant TB harder treat limited efficacy                 mechanistic design, high-throughput screening, <br /> TB control programs (29). This necessitated search                    classes PDF inhibitors (PDF-I) studied <br /> new antituberculosis drugs novel mode action.                           previously (64). Thus far, compounds having hydroxamic <br />    In prokaryotes certain organelles, like plastids mi-                acid-chelating N-formyl hydroxylamine-chelating groups <br /> tochondria eukaryotes, nascent proteins carry N-formylated                  potent enzyme inhibition, good antibacterial activity, <br /> methionine (40). After translational initiation, peptide defor-                   vivo efficacy, including oral activity (9, 12, 27). <br /> mylase (PDF) (EC 3.5.1.27) removes N-formyl group                          In work, validate def (gene encoding PDF) drug <br /> the nascent proteins (42). Deformylated proteins undergo fur-                     target TB proving essentiality Mycobacterium bovis <br /> ther N-terminal processing mature proteins (61). PDF                      BCG. PDF M. tuberculosis strain H37Rv cloned and <br /> is attractive candidate discovery antibacterial                     expressed heterologously E. coli. Recombinant His-tagged <br /> agents, essentiality gene (def) encoding PDF                    M. tuberculosis PDF enzyme purified, biochemical <br /> been shown Escherichia coli Streptococcus pneumoniae                      characterization carried out. Activities novel class of <br /> (8, 36). The enzyme characterized highly unsta-                     PDF-I, N-alkyl urea reverse hydroxamates (27), de- <br /> ble metallopeptidase uses Fe2&#226;&#171;&#185; catalytic metal (51).                  termined M. tuberculosis PDF enzyme cell-based <br /> Oxidation Fe2&#226;&#171;&#185; inactivates enzyme, purification                     assays antimycobacterial activity. A potent lead, PDF-611 <br />                                                                                   (LBK-611) (15), identified novel antimycobacterial <br />                                                                                   agent. <br />    * Corresponding author. Mailing address: Novartis Institute Trop- <br /> ical Diseases, 10 Biopolis Road, 05-01 Chromos, Singapore 138670, Re- <br />                                                                                                            MATERIALS AND METHODS <br /> public Singapore. Phone: 65-67222923. Fax: 65-67222917. E-mail: <br /> kakoli.mukherjee@novartis.com.                                                      Enzymes, chemicals, antibiotics. Chemicals used highest <br />    &#226;&#8364;&#161; Present address: National University Hospital, Department                 grade commercially available. Chemicals antibiotics purchased <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Laboratory Medicine, 5 Lower Kent Ridge Road, Main Building,                      Sigma (St. Louis, Mo.), unless indicated. Restriction enzymes and <br /> Level 3, Singapore 119 074, Republic Singapore.                                modifying enzymes purchased New England Biolabs, Inc. (Beverly, <br />    &#226;&#8364;&#160; Supplemental material article http://aac            Mass.), used according manufacturer&#226;&#8364;&#8482;s recommendations. <br /> .asm.org/.                                                                          Bacterial strains, plasmids, media. Bacterial strains plasmids used in <br />    &#228;&#338;&#164; <br />      Published ahead print 11 September 2006.                               study listed Table 1. E. coli strains cultured Luria-Bertani (LB) <br />  <br />                                                                            3665 <br />  3666       TEO ET AL.                                                                                                            ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />                                                         TABLE 1. Strains plasmids used work <br />   Strain plasmid                                                       Relevant characteristic(s)                                                  Source reference <br />  <br /> Strains <br />   E. coli <br />      XL1-Blue                     recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac &#229;&#8230;&#179;F&#226;&#172;&#732; proAB lacIqZ&#226;&#338;&#172;M15 Tn10 (Tetr)&#229;&#8230;&#180;;                                     Stratagene <br />                                      cloning strain <br />        M15(pREP4)                 Nals Strs Rif s Thi&#226;&#171;&#186; Lac&#226;&#171;&#186; Ara&#226;&#171;&#185; Gal&#226;&#171;&#185; Mtl&#226;&#171;&#186; F&#226;&#171;&#186; RecA&#226;&#171;&#185; Uvr&#226;&#171;&#185; Lon&#226;&#171;&#185;; expression strain                                      QIAGEN <br />                                      PT5-regulated gene expression <br />  <br />   M. bovis BCG                    Pasteur strain ATCC 35734                                                                                               ATCC <br />    XO22                           Single-crossover strain; Sucs Hygr                                                                                      This work <br />    XOdef                          Single-crossover strain XO22 carrying complementation plasmid pMV262-def                                                This work <br />  <br />  <br /> Plasmids <br />   pCR-XL-TOPO                     Ampr; TA-cloning vector longer PCR products                                                                         Invitrogen <br />   pCR2.1-TOPO                     Ampr; TA-cloning vector                                                                                                 Invitrogen <br />   pQE30                           Ampr; low-copy-number plasmid inducible T5 promoter production N-                                           QIAGEN <br />                                     terminal His6 tag translational fusion <br />   pCR2.1-PDF                      M. tuberculosis def gene cloned pCR2.1-TOPO vector                                                                 This   work <br />   pQE30-PDF                       M. tuberculosis def gene cloned BamHI site pQE30                                                                This   work <br />   pYUB657                         Ampr, Hygr; sacB counterselectable marker, E. coli-Mycobacterium shuttle vector                                         47 <br />   pYUB657-dimer                   DNA fragment containing in-frame def deletion cloned pYUB657 suicide vector                                     This   work <br />   pMV262                          E. coli-Mycobacterium shuttle plasmid, hsp60 promoter; Kanr ColE1 OriM                                                  62 <br />   pMV262-def                      pMV262 containing def gene, expressed translational fusion                                  This   work <br />                                     amino acids hsp60 <br />  <br />  <br />  <br />  <br /> broth (Becton Dickinson, Franklin Lakes, NJ) LB agar plates supplemented             (QIAGEN, Valencia, CA). For Southern blot analysis, DNA samples di- <br /> with 100 &#226;?&#174;g/ml ampicillin 50 &#226;?&#174;g/ml kanamycin cloning             gested appropriate restriction enzyme fragments separated 1% <br /> maintenance. Terrific broth (12 g tryptone, 24 g yeast extract, 4 ml glycerol, 2.3 g     agarose gel transferred Hybond-N&#226;&#171;&#185; nylon membrane (Amersham). <br /> potassium phosphate [monobasic], 9.4 g potassium phosphate [dibasic], pH 7.5,            Southern hybridization carried using enhanced chemiluminescence <br /> water make 1 liter) used growth medium expression PDF           direct nucleic acid labeling detection systems (Amersham). <br /> from E. coli.                                                                               Expression purification M. tuberculosis PDF. The M. tuberculosis <br />     M. bovis BCG cultured Middlebrook 7H9 (catalog no. 271310; Difco)             H37Rv genomic DNA sequence (<span id='am-3' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> NC_000962) was <br /> broth supplemented 10% (vol/vol) albumin-dextrose saline (0.81% NaCl,               used design primers 5&#226;&#172;&#732;-ATGGATCCATGGCAGTCGTACCCATCCG-3&#226;&#172;&#732; <br /> 5% bovine serum albumin fraction V [Roche, Mannheim, Germany], 2%                    5&#226;&#172;&#732;-ATGGATCCGTGACCGAACGGGTCGGG-3&#226;&#172;&#732;, BamHI sites <br /> glucose), 0.2% glycerol, 0.05% Tween 80 Middlebrook 7H10 (catalog              (underlined) incorporated them. The native TAA stop codon M. <br /> no. 262710; Difco) agar plates supplemented 10% (vol/vol) oleic acid-               tuberculosis def gene included utilize vector-encoded stop codon, <br /> albumin-dextrose-catalase enrichment (catalog no. 212240; BBL), 0.5% glycerol,           resulting fusion protein MRGSHHHHHHGS-M. tuberculosis PDF-GSAC <br /> and 0.05% Tween 80. When required, hygromycin B (Roche) used                  ELGTPGRPAAKLN, residues italics originating vector. PCR <br /> concentration 50 &#226;?&#174;g/ml. Sucrose used 7H10 medium 4%                   carried using high-fidelity Platinum PCR Supermix (Invitrogen, Carls- <br /> concentration.                                                                           bad, CA) according manufacturer&#226;&#8364;&#8482;s protocol. PCR products initially <br />     Buffers. Lysis buffer composed 50 mM HEPES (pH 7.5), 5 mM NiSO4,              cloned pCR2.1-TOPO cloning vector (Invitrogen), generating plasmid <br /> 5 mM MgSO4, 10 &#226;?&#174;g/ml catalase, 0.25 mM Tris(2-carboxyethyl)phosphine hy- <br />                                                                                          pCR2.1-PDF; this, def gene subcloned BamHI-linearized E. <br /> drochloride, 10 &#226;?&#174;g/ml DNase I, 1 ml protease cocktail inhibitor, 1 mg/ml ly- <br />                                                                                          coli expression vector pQE30 (QIAGEN), generating plasmid pQE30-PDF (Ta- <br /> sozyme, 1% Triton X-100. Buffer I composed 50 mM NaH2PO4, 300 <br />                                                                                          ble 1). Both pCR2.1-PDF pQE30-PDF sequenced. DNA sequences <br /> mM NaCl, 10 mM imidazole, pH 8.0. Buffer II composed 50 mM <br />                                                                                          analyzed using software Vector NTI, version 7.0 (InforMax, Frederick, <br /> NaH2PO4, 300 mM NaCl, 20 mM imidazole, pH 8.0. Buffer III com- <br />                                                                                          MD). Amino acid sequence alignments prepared using CLUSTALW, ver- <br /> posed 50 mM NaH2PO4, 300 mM NaCl, 250 mM imidazole, pH 8.0. Gel <br />                                                                                          sion 1.83 (11). All DNA manipulations performed standard condi- <br /> filtration buffer composed 50 mM NaH2PO4, 1 M NaCl, 5% glycerol. <br />                                                                                          tions described previously (55). <br />     Construction BCG def suicide plasmid def complementation <br />                                                                                             E. coli M15(pREP4) cells carrying pQE30-PDF grown terrific broth <br /> plasmid. Gene regions flanking def gene PCR amplified ligated <br />                                                                                          containing 50 &#226;?&#174;g/ml kanamycin 100 &#226;?&#174;g/ml ampicillin 37&#194;&#176;C the <br /> together obtain DNA fragment def gene (636 bp) excluded <br /> in frame (Fig. 1A). Primers BglF (5&#226;&#172;&#732;-TGGTAAATAGAGATCTGTGTGCGA                            optical density 600 nm reached &#226;&#172;&#402;0.5. The culture induced 0.1 mM <br /> CGTCATAGCCGAGTT-3&#226;&#172;&#732;) NdeR (5&#226;&#172;&#732;-TGTCTTTTCATATGTAATGGT                                   isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG) incubated 37&#194;&#176;C addi- <br /> CTCGTGGCCGGGGC-3&#226;&#172;&#732;) amplify 1,150-bp fragment upstream def (left                    tional 4 h. Cells (from 1 liter) harvested, cell pellet washed <br /> fragment). Primers NdeFF (5&#226;&#172;&#732;-GAAGTGTACATATGGGGCTGAGGAGGC                                 twice sterile phosphate-buffered saline suspended 20 ml lysis buffer. <br /> GGGCAAT-3&#226;&#172;&#732;) BglRR (5&#226;&#172;&#732;-TTTAATTTTAGATCTGGCGGGCTTTGGC                                   The cells lysed incubation ice 30 min, followed sonication. Cell <br /> ATAGCG-3&#226;&#172;&#732;) amplify 1,173-bp fragment downstream def (right fragment).              debris removed centrifugation 35,000 &#226;&#171;&#187; g 4&#194;&#176;C 15 min. The <br /> Primers BglF/BglRR NdefR/NdeFF carry BglII NdeI restriction sites,               proteins supernatant precipitated adjustment 60% saturation <br /> respectively, enabled left right fragments ligated cloned        ammonium sulfate. The pellet dissolved 10 ml buffer I added <br /> into pCR-XL-TOPO (Table 1). The dimer fragment cloned                  Ni-nitrilotriacetic acid (NTA) slurry (QIAGEN) batch purification by <br /> suicide vector pYUB657 (47), generating def suicide plasmid pYUB657-                 metal affinity chromatography. The resulting mixture incubated 1 h with <br /> dimer (Table 1). The def complementation vector constructed cloning           gentle shaking 4&#194;&#176;C. The slurry washed twice buffer II, bound <br /> def gene BamHI site pMV262 (Table 1), generating pMV262-def.                 PDF eluted buffer III. Active fractions obtained Ni-NTA batch <br /> This vector introduced def single-crossover strain allow chromo-           purification purified gel filtration using HiLoad Superdex 26/60 <br /> somal def deleted.                                                                 prep grade column (Amersham) equilibrated gel filtration <br />     Genomic DNA isolation Southern blot analysis. Genomic DNA M.                buffer. Elution buffer flow rate 1 ml/min. The <br /> bovis BCG isolated using protocol described DNeasy tissue kit               active fractions pooled concentrated using Vivaspin 20 centrifugal <br />  VOL. 50, 2006                                                                              PDF INHIBITORS AS ANTIMYCOBACTERIALS                                  3667 <br />  <br />  <br />  <br />  <br />   FIG. 1. (A) Arrangement def neighboring genes M. tuberculosis genome. PCR fragments generated construction def&#226;&#338;&#172; <br /> suicide vector indicated solid arrows. See Materials Methods details. The genetic map drawn scale. (B) Southern analysis <br /> of def&#226;&#338;&#172; mutants. def&#226;&#338;&#172; mutants (lanes 2 4) obtained single-crossover strain complemented, trans, def carried <br /> on plasmid pMV262. Lane 1, wild-type BCG; lane 5, single-crossover strain; lane 6, single-crossover strain XOdef, harboring def gene on <br /> plasmid pMV262-def; lane 7, plasmid pMV262-def; lane M, 1-kb DNA marker. Genomic DNA digested HindIII probed def. <br /> Wild-type BCG yields 10.3-kb fragment, single-crossover strain yields 8.3-kb fragment. Neither fragments seen in <br /> complemented def&#226;&#338;&#172; mutants, indicating chromosomal copy def gene deleted. <br />  <br />  <br /> concentrator (Sartorius, Hanover, Germany). Glycerol added final           well. The plates reincubated overnight 37&#194;&#176;C. STR used reference <br /> concentration 33% (vol/vol), enzyme stored frozen &#226;&#171;&#186;80&#194;&#176;C.          drug plate. The MICs obtained quantifying fluorescence of <br />    PDF enzyme assay. The PDF enzyme assay format based published           Alamar Blue, using excitation emission wavelengths 530 nm 590 nm, <br /> method (12), assay carried final volume 50 &#226;?&#174;l black,     respectively. Values 150,000 relative fluorescence units considered <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> 96-well, half-area plates (Corning Costar, Etobicoke, Ontario, Canada). The         reflect growth. <br /> reaction mixture contained 50 mM HEPES (pH 7.5) &#226;&#172;&#402;30 ng E. coli-                 Frequency spontaneous resistant mutants. Log-phase BCG cultures 109 <br /> expressed M. tuberculosis PDF well. The reaction initiated adding        CFU/ml plated 7H10 plates containing drug 10&#226;&#171;&#187; MIC onto <br /> N-formyl-methionine-alanine-serine (fMAS) substrate final concentration     drug-free media. Plates incubated 37&#194;&#176;C approximately 6 weeks or <br /> 2 mM (Bachem, Germany). The reaction room temperature 2.5           colonies appeared. The frequency spontaneous resistance calculated <br /> min terminated adding 25 &#226;?&#174;l fluorescamine (0.02 mg/ml dioxane).        ratio number colonies grew drug-supplemented plates to <br /> The microplate incubated 5 min read fluorescence       drug-free plates. <br /> Tecan Saffire plate reader using excitation wavelength 380 nm             Mutational analyses def fmt genes. The def fmt genes PDF-I- <br /> emission wavelength 470 nm.                                                      resistant BCG mutants analyzed sequence variation. The def gene <br />    Biochemical characterization M. tuberculosis PDF. Biochemical character-      region amplified using primers Updef (5&#226;&#172;&#732;-GGTCCCGGTCTTGGTCTGC <br /> ization M. tuberculosis PDF performed using pure batch-purified          A-3&#226;&#172;&#732;) Dndef (5&#226;&#172;&#732;-CGGCGATGATGCCCGCCG-3&#226;&#172;&#732;), generating 838-bp <br /> recombinant enzyme. The effect substrate concentrations M. tuberculosis       amplicon. The primers Upfmt (5&#226;&#172;&#732;-TAGCGTGCCTTGCGTACCCA-3&#226;&#172;&#732;) and <br /> PDF activity studied using N-formylated peptide substrates, N-formyl-       Dnfmt (5&#226;&#172;&#732;-CAGCGCGGGCAACACCAG-3&#226;&#172;&#732;) amplify 1,178-bp ampli- <br /> methionine-alanine (fMA) fMAS (Bachem), concentrations ranging             con. High-fidelity PCR carried using Proof Start DNA polymerase <br /> from 0 10 mM. The effects divalent metal ions Ca2&#226;&#171;&#185;, Mg2&#226;&#171;&#185;, Mn2&#226;&#171;&#185;,           (QIAGEN). The PCR products purified agarose gels using a <br /> Co2&#226;&#171;&#185;, Cu2&#226;&#171;&#185;, Ni2&#226;&#171;&#185;, Zn2&#226;&#171;&#185; PDF activity assessed addition         QIAquick gel extraction kit (QIAGEN). The purified PCR products se- <br /> different ions final concentration 1 mM assay buffer.                quenced Research Biolabs Technologies (Singapore). Jalview software was <br />    For inhibition studies M. tuberculosis PDF, 30-min preincubation     used visualizing alignment (11), software used for <br /> inhibitor enzyme included prior initiating reaction. The        bioinformatics analysis secondary-structure prediction mutants (13). <br /> concentrations inhibitors varied 0 10 &#226;?&#174;M. Fifty percent inhibitory      In vivo pharmacokinetic studies. CD-1 female outbred mice (BioArc, India) <br /> concentration (IC50) values obtained fitting data sigmoid dose-    used pharmacokinetic studies selected PDF inhibitors. For intrave- <br /> response equation using GraphPad Prism software, version 3.0 (GraphPad Soft-        nous (i.v.) oral (p.o.) administration, compounds formulated 4% <br /> ware, Inc., San Diego, CA).                                                         10% polyethylene glycol 300, respectively. After 1 week acclimation, mice <br />    Antimicrobial agents drug susceptibility testing. Streptomycin (STR),        received dose 1 mg/kg body weight i.v. tail vein injection dose of <br /> isoniazid (INH), rifampin (RIF), linezolid (LZD), erythromycin (ERY)            5 mg/kg p.o. gavage. Groups mice euthanized CO2 inhalation <br /> were obtained Sigma (St. Louis, Mo.). Stock solutions STR, LZD,         different intervals 5 min 4 h following dosing. Blood samples were <br /> ERY prepared 10 mM concentration deionized water, INH,             collected cardiac puncture heparinized tubes. Blood centrifuged, and <br /> RIF, PDF-I dissolved 90% dimethyl sulfoxide. Drug susceptibility        plasma recovered stored frozen &#226;&#171;&#186;80&#194;&#176;C. Plasma samples ex- <br /> testing based previously published method (20). Logarithmic-phase M.       tracted 1 ml ethyl acetate, analyzed, quantified drug content by <br /> bovis BCG cultures diluted complete 7H9 broth obtain optical          liquid chromatography coupled triple quadrupole mass spectrometry (X- <br /> density 600 nm &#226;&#172;&#402;0.04, corresponded approximately 106 CFU/ml,         Terra, C18, 4.6 50 mm, 5 &#226;?&#174;m), carbamazepine internal standard. <br /> before added test plate. The plates incubated 37&#194;&#176;C 4       For quantitative calibration, standard curves established using spiked com- <br /> days. On fourth day, 50 &#226;?&#174;l freshly prepared 1:1 mixture Alamar Blue     pounds plasma. The limit detection 10 ng/ml. A noncompartmental <br /> (Serotec Ltd., Oxford, United Kingdom) 10% Tween 80 added           model (WINNONLIN, version 4.1) used calculate pharmacokinetic pa- <br />  3668       TEO ET AL.                                                                                                 ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 2. Expression M. tuberculosis PDF E. coli. (A) SDS-PAGE analysis PDF expressed E. coli M15(pREP4) carrying pQE30-PDF. <br /> Lane 1, total cell protein uninduced; lane 2, total cell protein induced; lane 3, soluble fraction; lane 4, insoluble fraction; lane 5, Ni-NTA <br /> batch-purified M. tuberculosis PDF; lane 6, M. tuberculosis PDF purified gel filtration; lane M, low-range molecular mass marker (Amersham). <br /> Arrows indicate position recombinant His-tagged PDF. (B) Effects various substrate (fMA fMAS) concentrations recombinant <br /> M. tuberculosis PDF activity. RFU, relative fluorescence units. <br />  <br />  <br />  <br /> rameters. The p.o. bioavailability calculated ratio area   confirmed contained def deletion chromo- <br /> curve (AUC) p.o. administration AUC i.v. administration corrected       (Fig. 1B). When mutants probed def, the <br /> for dose (F &#226;&#171;&#189; AUCp.o. &#226;&#171;&#187; dosei.v./AUCi.v. &#226;&#171;&#187; dosep.o.). <br />                                                                                        chromosomal copy def detected (Fig. 1B). <br />                                                                                            Expression purification M. tuberculosis PDF. As <br />                                   RESULTS                                              shown Fig. 2A, recombinant M. tuberculosis PDF ex- <br />    Essentiality def, gene encoding peptide deformylase                       pressed high levels E. coli. However, expressed pro- <br /> M. bovis BCG. In order determine essentiality peptide                        tein largely insoluble form (Fig. 2A, lane 4). Insol- <br /> deformylase M. bovis BCG, two-step allelic-exchange                               ubility recombinant M. tuberculosis PDF reported <br /> methodology (47) employed generate def deletion mu-                             earlier (57). Despite low solubility M. tuberculosis PDF, <br /> tants. A suicide plasmid, pYUB657-dimer (Table 1), elec-                           purification E. coli extracts performed native <br /> troporated BCG, single-crossover strain, XO22,                              conditions. The enzyme purified homogeneity com- <br /> confirmed Southern hybridization (Table 1; Fig. 1B),                            bination ammonium sulfate fractionation, Ni-NTA, gel <br /> obtained. This strain sucrose sensitivity hygromycin                         filtration chromatography. Typical yields 1.2 mg/liter. The <br /> resistance (Sucs Hygr) <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> conferred inte-                           resulting purified protein showed single band sodium <br /> gration suicide vector chromosome. After                           dodecyl sulfate-polyacrylamide gel electrophoresis (SDS- <br /> second recombination event, selection markers elim-                           PAGE) gels (Fig. 2A, lane 6) showed enzymatic activity on <br /> inated 75 revertant (Sucr Hygs) colonies screened                          N-formylmethionyl peptides. <br /> PCR. All positive chromosomal def. The 28 colonies                                SDS-PAGE analysis (Fig. 2A, lane 6) showed the <br /> that hyg negative PCR analyzed Southern                                purified protein molecular mass &#226;&#172;&#402;30 kDa. This was <br /> blotting probing def. Hybridization results showed                        slightly larger predicted molecular mass 23.96 <br /> all recombinants retained wild-type def gene (3.8-kb                        kDa (inclusive His6 tag additional 17 amino <br /> band), single-crossover strain second-                        acids C terminus introduced the <br /> ary recombinants derived generated larger, 4.6-kb frag-                         vector). Such anomalous migration recombinant M. tu- <br /> ment (data shown), suggesting def gene essential                       berculosis PDF observed earlier (57). Analytical gel <br /> and deleted.                                                                 filtration HiLoad Superdex 26/60 prep grade column <br />    A copy def cloned pMV262 (pMV262-def, Kanr [Ta-                             gave apparent molecular mass 24.5 kDa, close that <br /> ble 1]) introduced single-crossover strain XO22,                          deduced amino acid sequence (23.96 kDa). This <br /> resulting transformant XOdef (Table 1). Double crossovers                           suggested peptide deformylase enzyme behaved as <br /> were isolated XOdef culturing strain ab-                            monomeric species native conditions used for <br /> sence hygromycin, allowing integrated vector se-                           purification. Identity 30-kDa protein con- <br /> quence lost. All 28 Sucr Hygs Kmr colonies                            firmed Western blotting using anti-PDF antibody and <br /> selected lost integrated suicide vector. Of these, 25                          anti-His antibody (data shown). <br /> contained wild-type def gene chromosome (data                               There significant loss protein enzyme activity <br /> shown). Southern hybridization remaining isolates                         fractions obtained gel filtration suit- <br />  VOL. 50, 2006                                                                   PDF INHIBITORS AS ANTIMYCOBACTERIALS                3669 <br />  <br />  <br />                          TABLE 2. Effects PDF-I M. tuberculosis PDF H. sapiens PDF enzyme activities <br />                                      (IC50s) antibacterial activities (MIC90s) M. bovis BCG <br />                                                                             IC50 (nM) <br />                                                                                                       MIC90 (&#226;?&#174;M) in <br />        Compound                      Structure                                                                              Reference(s) <br />                                                               M. tuberculosis           H. sapiens    M. bovis BCGa <br />                                                                   PDFa                    PDF <br />  <br />  <br />  <br /> Actinonin                                                      50.5 &#226;&#171;&#190; 7.7                  43            &#226;&#172;&#381;25                   12, 32 <br />  <br />  <br />  <br />  <br /> PDF-709                                                        90.0 &#226;&#171;&#190; 4.2                 147a             6.25                27 <br />  <br />  <br />  <br />  <br />                                                                                               a <br /> PDF-611 (LBK-611)                                              69.5 &#226;&#171;&#190; 0.7                 115              0.78                15 <br />  <br />  <br />  <br />  <br /> BB-3497                                                        23.9 &#226;&#171;&#190; 7.5                    8             3.125               12, 45 <br />  <br />  <br />  <br />  a <br />      This work. <br />  <br />  <br />  <br /> able large-scale screens. Instead, reported earlier E.      lier reports indicated Co2&#226;&#171;&#185;, Ni2&#226;&#171;&#185;, Zn2&#226;&#171;&#185; no <br /> coli PDF (10), batch-purified M. tuberculosis PDF (Fig. 2A,             effect M. tuberculosis PDF enzyme activity (57). <br /> lane 5) used screening large library. After         Enzyme inhibition studies. Several inhibitors avail- <br /> background (induced E. coli M15 cell extracts transformed               able PDF-I library (27) tested M. tuberculosis PDF <br /> with pQE30 vector, treated similar way) sub-               enzyme assay (Table 2). The active compound, PDF-611, <br /> tracted, enzyme activity batch specific         LBK-611 (15), IC50 69.5 nM, similar <br /> could completely inhibited PDF-I. The hits recon-            compound, PDF-709, IC50 90 nM. Two known <br /> firmed using purified enzyme.                                       antibacterial PDF-I, actinonin (10) BB-3497 (12, 14), <br />    Biochemical characterization E. coli-expressed M. tuber-          showed potent inhibition M. tuberculosis PDF enzyme, <br /> culosis PDF. Biochemical characterization            IC50s 50.5 23.9 nM, respectively. <br /> pure batch-purified enzyme, results               In order assess issue selectivity, compounds <br /> similar. During optimization substrate concentrations             tested purified Homo sapiens PDF and <br /> fMA fMAS, optimal                 human matrilysin (MMP-7). Matrilysin belongs ma- <br /> substrate M. tuberculosis PDF (Fig. 2B). A prominent fea-           trix metalloproteinase (MMP) family zinc-containing en- <br /> ture kinetics observed substrates tested       zymes related PDF (44). PDF-611 (LBK-611) PDF- <br /> the enzyme showed saturation kinetics &#226;&#172;&#402;10 mM                   709 inhibited H. sapiens PDF IC50s 115 nM 147 <br /> substrate, concentrations 12.5 mM fMAS in-               nM, respectively (Table 2). BB-3497 IC50 8 nM <br /> hibitory M. tuberculosis PDF. This finding similar           H. sapiens PDF (45) actinonin IC50 43 nM <br /> that earlier report (57) (Fig. 2B). Subsequently, 2 mM          (32). However, compounds displayed significant inhibi- <br /> fMAS used evaluating parameters enzyme             tion MMP-7 concentrations 10 &#226;?&#174;M inhibited <br /> assays. Velocity versus substrate concentration plots typically         growth K562 cell line concentrations 10 <br /> produced straight line, indicating Km values       &#226;?&#174;M, actinonin, IC50 8.5 &#226;?&#174;M (data <br /> substrates &#226;&#172;&#381;5 mM (data shown).                                  shown). <br />    The enzyme displayed pH optimum 7.5, retaining                Antimycobacterial activities PDF-611 (LBK-611) and <br /> than 20% optimal activity extreme pH, example,         PDF-709. PDF-611 (LBK-611) potent terms of <br /> pH 5.5 pH 10 (data shown). Ionic strength sig-             antimycobacterial activity, MIC90 value 0.78 &#226;?&#174;M (0.25 <br /> nificant detrimental effect M. tuberculosis PDF activity          &#226;?&#174;g/ml) (Table 2). Both PDF-709 BB-3497 displayed good <br /> concentrations greater 0.1 M KCl (data shown). This            antimycobacterial activities growing M. bovis BCG cul- <br /> result agreed earlier results M. tuberculosis PDF (57).       tures (MIC90s 6.25 &#226;?&#174;M 3.125 &#226;?&#174;M, respectively); how- <br /> Co2&#226;&#171;&#185;, Ni2&#226;&#171;&#185;, Zn2&#226;&#171;&#185; partially inhibitory, Cu2&#226;&#171;&#185;              ever, actinonin showed poor antimycobacterial effect, a <br /> ions completely abolished PDF activity (data shown). Ear-           MIC90 &#226;&#172;&#381;25 &#226;?&#174;M (&#226;&#172;&#381;16 &#226;?&#174;g/ml). Based results obtained <br />  3670        TEO ET AL.                                                                                                     ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />        TABLE 3. MICs PDF-611 (LBK-611) various                                          TABLE 4. Bioinformatics analysis fmt mutants <br />                     M. tuberculosis isolates <br />                                                                                                                                             Predicted within <br />                                                                                                       <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>Mutation</span>    Predicted structural <br />                                                    MIC (&#226;?&#174;g/ml)b                          Mutant                                              8 &#195;&#8230; fMet-tRNA <br />                                                                                                         FMT               locationa <br />           Strain   a <br />                                                                                                                                                 binding sitea <br />                                    PDF-611           BB-3497          Gatifloxacin <br />                                                                                         fmt-100         P100L              Loop                     No <br /> H37Rv (ATCC 27294)              0.125&#226;&#8364;&#8220;0.5          0.125&#226;&#8364;&#8220;0.5           0.03&#226;&#8364;&#8220;0.06        fmt-222a        E222K              Helix                    No <br /> Beijing MDR                        0.125           0.125&#226;&#8364;&#8220;0.25          0.03&#226;&#8364;&#8220;0.06        fmt-53          A53V               Helix                    No <br /> W-Beijing MDR                      0.125              0.125             0.25            fmt-32a         T32N               Loop                     Yes <br /> Beijing F29 MDR                  0.03&#226;&#8364;&#8220;0.25          0.03&#226;&#8364;&#8220;0.5            0.015           fmt-32b         T32N               Loop                     Yes <br /> Inhr SDR                         0.06&#226;&#8364;&#8220;0.125           0.125             0.015           fmt-230         P230S              Loop                     Yes <br /> Strr SDR                            0.5               1.0               0.03            fmt-105         V105F              Sheet                    Yes <br /> Rifr SDR                            0.25              1.2               0.015           fmt-32c         T32N               Loop                     Yes <br /> Pzar SDR                            0.5               1.0               0.03            fmt-192         R192P              Loop                     No <br /> Clinical MDR                     0.06&#226;&#8364;&#8220;1.0           0.06&#226;&#8364;&#8220;2           &#226;&#172;?0.008&#226;&#8364;&#8220;0.06        fmt-276         T276A              Loop                     No <br />                                                                                       fmt-104a        W104L              Loop                     No <br />      MDR, multidrug resistance; SDR, single-drug resistance; Pza, pyrazinamide. <br /> The clinical MDR isolates 10 clinical isolates described MDR;         fmt-104b        W104L              Loop                     No <br /> however, detailed susceptibility <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> TB drugs available.               fmt-117a        G117C              Sheet                    Yes <br />    b <br />      All MIC determinations twice. Single MIC values represent iden-          fmt-124         A124G              Helix                    Yes <br /> tical values experiments, multiple values indicate different results   fmt-222b        E222K              Helix                    No <br /> two experiments. For clinical MDR, multiple values indicate range            fmt-117b        G117C              Sheet                    Yes <br /> MIC values 10 different clinical isolates. <br />                                                                                           a <br />                                                                                             Deduced model M. tuberculosis FMT, based E. coli FMT <br />                                                                                         crystal structure (59). <br />  <br /> from M. tuberculosis PDF inhibition antimycobacterial <br /> studies, compounds PDF-709 PDF-611 (LBK-611) <br /> were selected analyses.                                                     formyl methyltransferase (FMT) mutants mutations <br />    The bactericidal effects PDF-611 (LBK-611) PDF-                               mapped regions active site, deduced a <br /> 709 M. bovis BCG 4 days determined                                 homology model M. tuberculosis FMT based E. coli FMT <br /> time-kill studies (data shown). At 1&#226;&#171;&#187; 10&#226;&#171;&#187; MIC                              crystal structure (Table 4; Fig. S3 supplemental <br /> for compound, &#226;&#172;?1 log <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> decrease viable                             material). <br /> counts, demonstrating bacteriostatic effect time pe-                             For studies mutant strains, fmt-192, fmt-276, fmt- <br /> riod tested. However, increasing incubation time                            104a, fmt-104b characterized growth antimi- <br /> inhibitor, higher bacterial killing observed                            crobial susceptibility, generated against <br /> concentrations (data shown), indicating PDF-I                          PDF-709. Two isolates, fmt-192 fmt-104a, slower dou- <br /> slow acting. PDF-611 (LBK-611) shown effective                           bling times (32 h 35 h, respectively) wild-type M. bovis <br /> against clinical isolates M. tuberculosis. The com-                          BCG (23 h), fmt-276 fmt-104b doubling times <br /> pound demonstrated low MICs (&#226;&#172;?1 &#226;?&#174;g/ml) multi-                               (22 h 24 h, respectively) similar wild-type M. <br /> drug-resistant clinical TB isolates (Table 3).                                          bovis BCG (23 h). In MIC testing, FMT mutants were <br />    Spontaneous mutation rates analysis PDF-I-resis-                              16-fold increase MIC (&#226;&#172;&#381;100 &#226;?&#174;M) <br /> tant M. bovis BCG isolates. The frequency spontaneous                                PDF-611 (LBK-611) PDF-709. However, the <br /> resistance development evaluated using concentration                              mutants retained susceptibility TB drugs, like RIF and <br /> of 10&#226;&#171;&#187; MIC. PDF-611- PDF-709-resistant mutants                                      STR, unrelated drugs, like LZD ERY. Target speci- <br /> arose frequencies &#226;&#177;&#8226;5 &#226;&#171;&#187; 10&#226;&#171;&#186;7 M. bovis BCG. These                                ficity confirmed expression extra chromo- <br /> mutation frequencies lower development                                 somal copies M. tuberculosis def gene BCG, resulting <br /> resistance INH, occurred frequency 7 &#226;&#171;&#187;                                 16-fold increase MIC PDF-611 (LBK-611) (data <br /> 10&#226;&#171;&#186;5, higher RIF, &#226;&#172;&#402;1 &#226;&#171;&#187; 10&#226;&#171;&#186;8                                 shown). For comparative purposes, mutant a <br /> (data shown).                                                                       mutation def fmt gene included study. <br />    The development PDF-I resistance bacteria                            This mutant slightly slower doubling time (26 h) than <br /> progress main mechanisms, mutations                             wild-type M. bovis BCG, susceptibility profile similar to <br /> the def gene mutations formyl methyltransferase gene                          fmt mutants. All fmt mutants regained susceptibility <br /> (fmt), catalyzes formylation methionylated initia-                         PDF-I wild-type fmt gene supplied trans, <br /> tor tRNA (12, 21, 34, 35). Twenty-nine spontaneous PDF-I-                               confirming resistance arose mutation fmt <br /> resistant M. bovis BCG mutants characterized. The com-                             (data shown). <br /> plete def fmt genes isolates sequenced.                                Pharmacokinetic studies mice. Clearance PDF-709 <br /> None isolates mutations def gene. Of 29                           PDF-611 (LBK-611) mouse serum measured after <br /> isolates, 16 missense mutations fmt                           compounds administered i.v. p.o. The concentra- <br /> gene (Table 4). In remaining 13 isolates, changes                           tions serum different times used calculate the <br /> detected def fmt genes. It noted indepen-                           corresponding pharmacokinetic parameters presented Table <br /> dently isolated mutants amino acid mutation                                5. PDF-611 (LBK-611) lower half-life (0.5 h) PDF- <br /> (fmt-32a, fmt-32b, fmt-32c T32N mutation; fmt-                              709 (1.6 h) i.v. administration. However, PDF-611 (LBK- <br /> 222a fmt-222b E222K mutation; fmt-117a                              611) higher half-life (3.7 h) PDF-709 (2.7 h) after <br /> fmt-117b G117C mutation). Nonconserved amino acid                               p.o. administration. Both compounds similar bioavailabili- <br /> changes observed half mutants. About half                            ties (&#226;&#172;&#402;40%). <br />  VOL. 50, 2006                                                                           PDF INHIBITORS AS ANTIMYCOBACTERIALS                            3671 <br />  <br />  <br />                               TABLE 5. Pharmacokinetic parametersa PDF-611 (LBK-611) PDF-709 mice <br />                              Type            Dose        Cmax                        AUCinf                       CL              Vss       Bioavailability <br />       Compound                                                          Tmax (h)                     t1/2 (h) <br />                            administration      (mg/kg)     (ng/ml)                    (ng &#194;&#183; h/ml)                (ml/min/kg)     (liter/kg)        (%) <br />  <br /> PDF-611(LBK-611)                i.v.           1.000        603.6        0.083          253.5        0.500           65.8           3.0 <br />                                 p.o.           5.000        194.3        0.250          566.0        3.70                                           45.0 <br />  <br /> PDF-709                         i.v.           1.000        902.0        0.083          145.6        1.600         114.5            3.9 <br />                                 p.o.           5.000        186.0        0.500          293.0        2.700                                          40.0 <br />   a <br />     Cmax, maximum concentration drug serum; Tmax, time maximum concentration drug serum; AUCinf, area curve extrapolated &#226;&#172;?; t1/2, <br /> half-life; CL, clearance; Vss, volume distribution steady state. <br />  <br />  <br />  <br />  <br />                              DISCUSSION                                            mutations residues important function (by use of <br />                                                                                    homology model E. coli FMT crystal structure) (Table <br />    M. tuberculosis PDF earlier proposed potential drug                    4; Fig. S3 supplemental material) (23, 26, 59). <br /> target (14) BB-3497, PDF inhibitor, potent vitro                    Three independently isolated mutants (fmt-32a, fmt-32b, and <br /> activity M. tuberculosis. The gene encoding M. tubercu-                    fmt-32c [with T32N mutation]) Ser-to-Ala mutation <br /> losis PDF shown genes required <br />                                                                                    region involved recognition initiator tRNA <br /> for optimal growth M. tuberculosis transposon site <br />                                                                                    (53, 54). Mutant fmt-53 (with A53V mutation) lies just <br /> hybridization (56). In work, failure obtain def dele- <br />                                                                                    outside region. Several mutants mutations the <br /> tion mutants M. bovis BCG strongly suggested the <br />                                                                                    C-terminal region involved recognition tRNA by <br /> deletion def lethal. After second recombination <br />                                                                                    FMT obtained (fmt-222a, fmt-222b, fmt-230, fmt- <br /> event, chromosomal wild-type def gene deleted <br />                                                                                    276) (23). Three mutants, fmt-104a fmt-104b (with the <br /> only presence extra chromosomal complementing <br />                                                                                    W104L mutation) fmt-100 (with P100L mutation), <br /> copy def. Taken together, results def <br />                                                                                    mapping close conserved residues (Asn-106, His-108, and <br /> gene essential M. bovis BCG. <br />                                                                                    Asp-144) predicted participate catalytic steps (1, <br />    It observed previously E. coli PDF the <br />                                                                                    28) (see Fig. S3 supplemental material). Without for- <br /> metal ions reaction conditions (substrate type as- <br />                                                                                    mylation assays Met-tRNAfMet, unsure the <br /> say) dramatically change activity properties the <br />                                                                                    extent different fmt mutations affect transformylase <br /> enzyme (10, 38, 49, 50). During biochemical characterization <br />                                                                                    activity. Further studies needed understand func- <br /> of M. tuberculosis PDF, observed differences an <br /> earlier published work (57, 58). These differences possi-                    tional effect mutations. <br /> bly attributed difference assay protocol ex-                         Mutants did mutations def fmt <br /> pression/purification conditions followed study com-                       gene efflux pump mutants, observed E. coli and <br /> pared earlier published work. In earlier work,                      Haemophilus influenzae (12, 15). However, actual mecha- <br /> investigators purified enzyme presence                       nism resistance mutants remains elucidated. <br /> excess Ni&#226;&#171;&#185;2 enzyme activity determined using fMA                        Thus, case mycobacteria, mechanism <br /> the substrate trinitrobenzenesulfonic acid assay (57).                        operative confer resistance PDF-I. <br /> Moreover, assay temperatures different (room                            Recently, eukaryotic PDF enzymes iden- <br /> temperature study versus 30&#194;&#176;C previous study).                      tified characterized, e.g., Arabidopsis thaliana (22), <br />    Structure-activity relationship studies actinonin                  Plasmodium falciparum (5), mitochondria H. sapiens <br /> N-alkyl urea hydroxamates (9, 27) indicate efficacious                        (22, 32, 33, 45, 60). The mRNA H. sapiens PDF been <br /> PDF-I share common structural features,                          shown expressed similar levels types human <br /> metal-chelating group, usually hydroxamate/reverse hydrox-                       tissues (22), enzyme active deformylase both <br /> amate, second n-alkyl butyl residue P1&#226;&#172;&#732;                    vitro vivo (32, 33, 60). Actinonin to <br /> position mimics methionine chain (4). The PDF-I                      inhibit cell growth various human tumor cell lines in <br /> tested work reverse hydroxamates, carrying                         tumor models (32, 33, 63). In addition, well-characterized <br /> n-butyl residue P1&#226;&#172;&#732; position (Table 2). This class                       bacterial PDF inhibitors, BB-3497 actinonin (also used in <br /> compounds inhibit M. tuberculosis PDF IC50s &#226;&#172;?100 nM                        study), inhibit H. sapiens PDF nanomolar concentra- <br /> and bacteriostatic M. bovis BCG, observed                       tions similar bacterial counterpart in- <br /> previously PDF-I Staphylococcus aureus                      hibited (32, 45). <br /> and E. coli (3, 12, 35). The frequency resistance class                    However, despite inhibition cell-free H. sapiens <br /> of inhibitors M. bovis BCG similar previously                    PDF antiproliferative effects tumor cell lines, num- <br /> reported S. aureus (12, 35) E. coli (3, 12).                               ber normal cell lines resistant acti- <br />    Spontaneous PDF-I-resistant isolates M. bovis BCG dem-                       nonin (33) BB-3497 (45). Actinonin tol- <br /> onstrated elevated MICs specifically PDF-I                         erated mice doses 400 mg/kg (24), and <br /> not drug classes. In work, resistance                        successors BB-3497 (BB-83698) tested hu- <br /> arose missense mutations fmt gene 16                      mans levels 475 mg clinically signif- <br /> 29 mutants, rest mutation def                    icant adverse effects (52). Furthermore, recent biochemical <br /> the fmt gene. Several fmt mutants study                      studies H. sapiens PDF showed enzyme had <br />  3672      TEO ET AL.                                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> significantly lower activity (kcat/Km) bacterial en-                          Locher, M. G. Page, W. Pirson, G. Rosse, J. L. Specklin. 2000. Hydrox- <br />                                                                                        amic acid derivatives potent peptide deformylase inhibitors antibac- <br /> zyme (E. coli PDF) (32, 33, 45, 52, 60).                                               terial agents. J. Med. Chem. 43:2324&#226;&#8364;&#8220;2331. <br />    In addition, unambiguous localization H. sapiens PDF                    3.   Apfel, C. M., H. Locher, S. Evers, B. Takacs, C. Hubschwerlen, W. Pirson, <br /> in mitochondria important, evidence suggests                        M. G. Page, W. Keck. 2001. Peptide deformylase antibacterial drug <br />                                                                                        target: target validation resistance development. Antimicrob. Agents <br /> mitochondria tumor cells sensitive mito-                             Chemother. 45:1058&#226;&#8364;&#8220;1064. <br /> chondrial insult (6, 48). There examples drugs selectively                 4.   Boularot, A., C. Giglione, I. Artaud, T. Meinnel. 2004. Structure-activity <br /> toxic cancer cells working inhibition mito-                          relationship analysis therapeutic potential peptide deformylase inhib- <br />                                                                                        itors. Curr. Opin. Investig. Drugs 5:809&#226;&#8364;&#8220;822. <br /> chondrial respiration (19, 43). Mitochondrial membranes                       5.   Bracchi-Ricard, V., K. T. Nguyen, Y. Zhou, P. T. Rajagopalan, D. <br /> less permeable small molecules membranes                            Chakrabarti, D. Pei. 2001. Characterization eukaryotic peptide <br />                                                                                        deformylase Plasmodium falciparum. Arch. Biochem. Biophys. 396: <br /> and act efficient supplementary filter drugs. For                       162&#226;&#8364;&#8220;170. <br /> this reason, number antibiotics (tetracycline, macrolides,                   6.   Cavalli, L. R., B. C. Liang. 1998. Mutagenesis, tumorigenicity, and <br /> etc.) inhibit mitochondrial targets (16, 30, 31, 37)                          apoptosis: mitochondria involved? Mutat. Res. 398:19&#226;&#8364;&#8220;26. <br />                                                                                   7.   Chan, M. K., W. Gong, P. T. Rajagopalan, B. Hao, C. M. Tsai, D. Pei. <br /> nevertheless currently used human medicine. Thus, acti-                             1997. Crystal structure Escherichia coli peptide deformylase. Bio- <br /> nonin&#226;&#8364;&#8482;s inhibition H. sapiens PDF, resulting mitochon-                           chemistry 36:13904&#226;&#8364;&#8220;13909. <br /> drial disruption, similar tumor specificity (33). A                  8.   Chan, P. F., K. M. O&#226;&#8364;&#8482;Dwyer, L. M. Palmer, J. D. Ambrad, K. A. Ingraham, <br />                                                                                        C. So, M. A. Lonetto, S. Biswas, M. Rosenberg, D. J. Holmes, M. <br /> major, striking difference H. sapiens PDF (also mouse                          Zalacain. 2003. Characterization novel fucose-regulated promoter <br /> PDF) bacterial PDF (E. coli PDF) highly con-                             (PfcsK) suitable gene essentiality antibacterial mode-of-action studies <br />                                                                                        Streptococcus pneumoniae. J. Bacteriol. 185:2051&#226;&#8364;&#8220;2058. <br /> served EGCLS motif (motif III), leucine mutated into <br />                                                                                   9.   Chen, D., C. Hackbarth, Z. J. Ni, C. Wu, W. Wang, R. Jain, Y. He, K. <br /> glutamic acid mammalian PDF (Glu-173 H. sapiens PDF),                            Bracken, B. Weidmann, D. V. Patel, J. Trias, R. J. White, Z. Yuan. 2004. <br /> which results low activity H. sapiens PDF (60). From                         Peptide deformylase inhibitors antibacterial agents: identification of <br />                                                                                        VRC3375, proline-3-alkylsuccinyl hydroxamate derivative, using in- <br /> the sequence similarity, M. tuberculosis PDF 11% identical                          tegrated combinatorial medicinal chemistry approach. Antimicrob. <br /> overall 6% identity active site H.                            Agents Chemother. 48:250&#226;&#8364;&#8220;261. <br /> sapiens PDF. These differences exploited make com-                     10.   Chen, D. Z., D. V. Patel, C. J. Hackbarth, W. Wang, G. Dreyer, D. C. Young, <br />                                                                                        P. S. Margolis, C. Wu, Z. J. Ni, J. Trias, R. J. White, Z. Yuan. 2000. <br /> pounds decreased affinity H. sapiens PDF                              Actinonin, naturally occurring antibacterial agent, potent deformylase <br /> affecting potency respect M. tuberculosis PDF. In                        inhibitor. Biochemistry 39:1256&#226;&#8364;&#8220;1262. <br /> this study, PDF-I selectivity M. tuber-                  11.   Clamp, M., J. Cuff, S. M. Searle, G. J. Barton. 2004. The Jalview Java <br />                                                                                        alignment editor. Bioinformatics 20:426&#226;&#8364;&#8220;427. <br /> culosis PDF, considered studies                    12.   Clements, J. M., R. P. Beckett, A. Brown, G. Catlin, M. Lobell, S. Palan, W. <br /> to tolerability mice, addition lack effect                    Thomas, M. Whittaker, S. Wood, S. Salama, P. J. Baker, H. F. Rodgers, V. <br />                                                                                        Barynin, D. W. Rice, M. G. Hunter. 2001. Antibiotic activity char- <br /> MMP-7 K562 cell line 10 &#226;?&#174;M (&#226;&#172;&#402;100-fold-higher                             acterization BB-3497, novel peptide deformylase inhibitor. Antimicrob. <br /> concentration H. sapiens PDF IC50).                                               Agents Chemother. 45:563&#226;&#8364;&#8220;570. <br />    In order ensure treatment compliance, antituberculosis                     13.   Cuff, J. A., G. J. Barton. 1999. Evaluation improvement multiple <br />                                                                                        sequence methods protein secondary structure prediction. Proteins 34: <br /> therapy requires oral dosing regimen day less.                         508&#226;&#8364;&#8220;519. <br /> The oral bioavailabilities PDF-611 (LBK-611) PDF-709                      14.   Cynamon, M. H., E. Alvirez-Freites, A. E. Yeo. 2004. BB-3497, peptide <br /> were range 40% mouse, indicating potential                            deformylase inhibitor, active Mycobacterium tuberculosis. J. An- <br />                                                                                        timicrob. Chemother. 53:403&#226;&#8364;&#8220;405. <br /> oral administration humans. This reinforced                   15.   Dean, C. R., S. Narayan, D. M. Daigle, J. L. Dzink-Fox, X. Puyang, K. R. <br /> relatively long half-lives 3.7 h 2.7 h, respectively, fol-                      Bracken, K. E. Dean, B. Weidmann, Z. Yuan, R. Jain, N. S. Ryder. 2005. <br />                                                                                        Role AcrAB-TolC efflux pump determining susceptibility Hae- <br /> lowing oral administration. Assuming interspecies scaling                              mophilus influenzae novel peptide deformylase inhibitor LBM415. <br /> within normal range, half-life mouse sup-                          Antimicrob. Agents Chemother. 49:3129&#226;&#8364;&#8220;3135. <br /> port once-daily oral dosing humans. These figures constitute                  16.   Vries, H., A. J. Arendzen, A. M. Kroon. 1973. The interference the <br />                                                                                        macrolide antibiotics mitochondrial protein synthesis. Biochim. Bio- <br /> an improvement previous half-life reports PDF inhib-                          phys. Acta 331:264&#226;&#8364;&#8220;275. <br /> itors (27). Oral bioavailabilities PDF-611 (LBK-611)                      17.   Dye, C. 2006. Global epidemiology tuberculosis. Lancet 367:938&#226;&#8364;&#8220;940. <br /> PDF-709 clear improvement reported                     18.   Dye, C., C. J. Watt, D. M. Bleed, S. M. Hosseini, M. C. Raviglione. 2005. <br />                                                                                        Evolution tuberculosis control prospects reducing tuberculosis <br /> other PDF inhibitors class N-alkyl urea hydroxamic acids                        incidence, prevalence, deaths globally. JAMA 293:2767&#226;&#8364;&#8220;2775. <br /> (0.1 3.2%) (27) BB-83698 (i.v. application only)                 19.   Fantin, V. R., M. J. Berardi, L. Scorrano, S. J. Korsmeyer, P. Leder. <br />                                                                                        2002. A novel mitochondriotoxic small molecule selectively inhibits <br /> (52). The total clearance levels moderate com-                           tumor cell growth. Cancer Cell 2:29&#226;&#8364;&#8220;42. <br /> pounds, i.e., 66 114 ml/min/kg PDF-611 (LBK-611)                     20.   Franzblau, S. G., R. S. Witzig, J. C. McLaughlin, P. Torres, G. Madico, A. <br /> PDF-709, respectively, lie range mouse liver                           Hernandez, M. T. Degnan, M. B. Cook, V. K. Quenzer, R. M. Ferguson, and <br />                                                                                        R. H. Gilman. 1998. Rapid, low-technology MIC determination clinical <br /> blood flow (90 ml/min/kg). Thus, group PDF inhibitors                          Mycobacterium tuberculosis isolates using microplate Alamar Blue <br /> has potential developed new class                            assay. J. Clin. Microbiol. 36:362&#226;&#8364;&#8220;366. <br /> antituberculosis agent.                                                          21.   Giglione, C., T. Meinnel. 2001. Resistance anti-peptide deformylase <br />                                                                                        drugs. Expert Opin. Ther. Targets 5:415&#226;&#8364;&#8220;418. <br />                                                                                  22.   Giglione, C., A. Serero, M. Pierre, B. Boisson, T. Meinnel. 2000. Iden- <br />                         ACKNOWLEDGMENTS                                                tification eukaryotic peptide deformylases reveals universality N-termi- <br />                                                                                        nal protein processing mechanisms. EMBO J. 19:5916&#226;&#8364;&#8220;5929. <br />   We thank Sabine Daugelat providing advice knockout pro-             23.   Gite, S., Y. Li, V. Ramesh, U. L. RajBhandary. 2000. Escherichia coli <br /> cedure Marty Pavelka pYUB657. We thank Brigitte Gicquel                    methionyl-tRNA formyltransferase: role amino acids conserved the <br /> providing pJEM15, used overexpression studies.                           linker region C-terminal domain specific recognition the <br />                                                                                        initiator tRNA. Biochemistry 39:2218&#226;&#8364;&#8220;2226. <br />                                                                                  24.   Gordon, J. J., B. K. Kelly, G. A. Miller. 1962. Actinonin: antibiotic <br />                               REFERENCES <br />                                                                                        substance produced actinomycete. Nature 195:701&#226;&#8364;&#8220;702. <br /> 1. Almassy, R. J., C. A. Janson, C. C. Kan, Z. Hostomska. 1992. Structures   25.   Groche, D., A. Becker, I. Schlichting, W. Kabsch, S. Schultz, A. F. <br />    apo complexed Escherichia coli glycinamide ribonucleotide trans-             Wagner. 1998. Isolation crystallization functionally competent Esch- <br />    formylase. Proc. Natl. Acad. Sci. USA 89:6114&#226;&#8364;&#8220;6118.                                 erichia coli peptide deformylase forms containing iron nickel the <br /> 2. Apfel, C., D. W. Banner, D. Bur, M. Dietz, T. Hirata, C. Hubschwerlen, H.           active site. Biochem. Biophys. Res. Commun. 246:342&#226;&#8364;&#8220;346. <br />  VOL. 50, 2006                                                                                 PDF INHIBITORS AS ANTIMYCOBACTERIALS                                    3673 <br />  <br /> 26. Guillon, J.-M., Y. Mechulam, J.-M. Schmitter, S. Blanquet, G. Fayat.               Hydroxamic acids pharmacological agents. Curr. Med. Chem. 9:1631&#226;&#8364;&#8220; <br />     1992. Disruption gene Met-tRNAfMetformyltransferase severely                1653. <br />     impairs growth Escherichia coli. J. Bacteriol. 174:4294&#226;&#8364;&#8220;4301.                 45.   Nguyen, K. T., X. Hu, C. Colton, R. Chakrabarti, M. X. Zhu, D. Pei. <br /> 27. Hackbarth, C. J., D. Z. Chen, J. G. Lewis, K. Clark, J. B. Mangold, J. A.              2003. Characterization human peptide deformylase: implications for <br />     Cramer, P. S. Margolis, W. Wang, J. Koehn, C. Wu, S. Lopez, G. Withers III,            antibacterial drug design. Biochemistry 42:9952&#226;&#8364;&#8220;9958. <br />     H. Gu, E. Dunn, R. Kulathila, S.-H. Pan, W. L. Porter, J. Jacobs, J. Trias,      46.   Nunn, P., B. Williams, K. Floyd, C. Dye, G. Elzinga, M. Raviglione. <br />     D. V. Patel, B. Weidmann, R. J. White, Z. Yuan. 2002. N-Alkyl urea                 2005. Tuberculosis control era HIV. Nat. Rev. Immunol. 5:819&#226;&#8364;&#8220;826. <br />     hydroxamic acids new class peptide deformylase inhibitors           47.   Pavelka, M. S., Jr., W. R. Jacobs, Jr. 1999. Comparison construc- <br />     antibacterial activity. Antimicrob. Agents Chemother. 46:2752&#226;&#8364;&#8220;2764.                    tion unmarked deletion mutations Mycobacterium smegmatis, Mycobac- <br /> 28. Klein, C., P. Chen, J. H. Arevalo, E. A. Stura, A. Marolewski, M. S. Warren,           terium bovis bacillus Calmette-Gue&#194;&#180;rin, Mycobacterium tuberculosis <br />     S. J. Benkovic, I. A. Wilson. 1995. Towards structure-based drug design:           H37Rv allelic exchange. J. Bacteriol. 181:4780&#226;&#8364;&#8220;4789. <br />     crystal structure multisubstrate adduct complex glycinamide ribonu-      48.   Penta, J. S., F. M. Johnson, J. T. Wachsman, W. C. Copeland. 2001. <br />     cleotide transformylase 1.96 A resolution. J. Mol. Biol. 249:153&#226;&#8364;&#8220;175.               Mitochondrial DNA human malignancy. Mutat. Res. 488:119&#226;&#8364;&#8220;133. <br /> 29. Kremer, L. S., G. S. Besra. 2002. Current status future development      49.   Ragusa, S., S. Blanquet, T. Meinnel. 1998. Control peptide deformylase <br />     antitubercular chemotherapy. Expert Opin. Investig. Drugs 11:1033&#226;&#8364;&#8220;1049.             activity metal cations. J. Mol. Biol. 280:515&#226;&#8364;&#8220;523. <br /> 30. Kroon, A. M., B. H. Dontje, M. Holtrop, C. Van den Bogert. 1984. The         50.   Rajagopalan, P. T., A. Datta, D. Pei. 1997. Purification, characterization, <br />     mitochondrial genetic target chemotherapy: tetracyclines            inhibition peptide deformylase Escherichia coli. Biochemistry <br />     cytostatics. Cancer Lett. 25:33&#226;&#8364;&#8220;40.                                                    36:13910&#226;&#8364;&#8220;13918. <br /> 31. Kroon, A. M., C. Van den Bogert. 1983. Antibacterial drugs         51.   Rajagopalan, P. T., D. Pei. 1998. Oxygen-mediated inactivation pep- <br />     interference biogenesis mitochondria animal human cells.            tide deformylase. J. Biol. Chem. 273:22305&#226;&#8364;&#8220;22310. <br />     Pharm. Weekbl. Sci. 5:81&#226;&#8364;&#8220;87.                                                     52.   Ramanathan-Girish, S., J. McColm, J. M. Clements, P. Taupin, S. Barrowcliffe, <br /> 32. Lee, M. D., C. Antczak, Y. Li, F. M. Sirotnak, W. G. Bornmann, D. A.               J. Hevizi, S. Safrin, C. Moore, G. Patou, H. Moser, A. Gadd, U. Hoch, V. Jiang, <br />     Scheinberg. 2003. A new human peptide deformylase inhibitable acti-                 D. Lofland, K. W. Johnson. 2004. Pharmacokinetics animals humans <br />     nonin. Biochem. Biophys. Res. Commun. 312:309&#226;&#8364;&#8220;315.                                     first-in-class peptide deformylase inhibitor. Antimicrob. Agents Chemother. <br /> 33. Lee, M. D., Y. She, M. J. Soskis, C. P. Borella, J. R. Gardner, P. A. Hayes,           48:4835&#226;&#8364;&#8220;4842. <br />     B. M. Dy, M. L. Heaney, M. R. Philips, W. G. Bornmann, F. M. Sirotnak,       53.   Ramesh, V., S. Gite, Y. Li, U. L. RajBhandary. 1997. Suppressor muta- <br />     D. A. Scheinberg. 2004. Human mitochondrial peptide deformylase, new                 tions Escherichia coli methionyl-tRNA formyltransferase: role 16- <br />     anticancer target actinonin-based antibiotics. J. Clin. Investig. 114:1107&#226;&#8364;&#8220;         amino acid insertion module initiator tRNA recognition. Proc. Natl. Acad. <br />     1116.                                                                                  Sci. USA 94:13524&#226;&#8364;&#8220;13529. <br /> 34. Margolis, P., C. Hackbarth, S. Lopez, M. Maniar, W. Wang, Z. Yuan, R.            54.   Ramesh, V., S. Gite, U. L. RajBhandary. 1998. Functional interaction of <br />     White, J. Trias. 2001. Resistance Streptococcus pneumoniae                   arginine conserved sixteen amino acid insertion module Esch- <br />     deformylase inhibitors mutations defB. Antimicrob. Agents                 erichia coli methionyl-tRNA formyltransferase determinants formyl- <br />     Chemother. 45:2432&#226;&#8364;&#8220;2435.                                                               ation initi <br /> </body></html>