<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Feb. 1997, p. 689&#226;&#8364;&#8220;696                                                                                             Vol. 179, No. 3 <br /> 0021-9193/97/$04.0010 <br /> Copyright q 1997, American Society Microbiology <br />  <br />  <br />  <br />            Characterization In-Frame Proteins Encoded cvaA, an <br />             Essential Gene Colicin V Secretion System: CvaA* <br />                       Stabilizes CvaA <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> Enhance Secretion <br />                           JAIWEON HWANG, MARINA MANUVAKHOVA,&#226;&#8364;&#160;                                  AND   PHANG C. TAI* <br />                                Department Biology, Georgia State University, Atlanta, Georgia 30303 <br />                                           Received 28 August 1996/Accepted 14 November 1996 <br />  <br />             Colicin V (ColV), antibacterial peptide toxin, uses dedicated signal sequence-independent export system <br />           extracellular secretion Escherichia coli. The products genes (a chromosomal tolC gene <br />           plasmid-born cvaA cvaB genes) involved process. To characterize gene products, the <br />           cvaA gene subcloned expressed control T7 RNA polymerase promoter. Two in-frame <br />           proteins, CvaA CvaA*, expressed identified. DNA sequences predicted proteins have <br />           potential translational initiation sites. N-terminal peptide sequencing showed translation CvaA <br />           starts TTG, 11 amino acids upstream previously proposed ATG initiation site. CvaA* is <br />           translated upstream ATG. Expression CvaA CvaA* induced iron chelator <br />           2,2*-dipyridyl, indicating cvaA negatively regulated partially Fur. CvaA*-depleted cells were <br />           secrete ColV, based reduced activity supernatant, did wild type, was <br />           recovered addition plasmid producing CvaA*. Interestingly, CvaA*-depleted wild-type cells had <br />           similar levels intracellular ColV activity. Translational fusions showed syntheses ColV CvaA <br />           affected CvaA* depletion. However, CvaA CvaA*-depleted cells stable in <br />           wild-type cells, indicating CvaA* directly indirectly affect stability CvaA. We conclude that <br />           CvaA* essential ColV secretion enhances ColV secretion stabilizing CvaA protein. <br />  <br />  <br />    Colicin V (ColV), small peptide toxin 9 kDa,               produce ColV (8). The ColV toxin produced cvaC <br /> encoded, specific immunity protein, va-              gene, located directly downstream cvi, encodes <br /> riety large low-copy-number plasmids. The toxin active                103-amino-acid protein does possess typical signal <br /> against Escherichia coli strains members                  peptide (16). However, secreted translation product of <br /> family Enterobacteriaceae (18). Its activity depends pres-           cvaC processed remove N-terminal 15 amino acids, <br /> ence outer membrane ColV receptor Cir Cir-like                    yielding mature toxin forms channels planar lipid <br /> proteins (4), target believed inner mem-               bilayers (10). <br /> brane sensitive cells, membrane potential dis-                 The transport proteins encoded cvaAB operon. <br /> rupted (45). ColV distinguished colicins (A,             The primary transporter gene, cvaB, predicts 78-kDa product <br /> DF13, E1, E2, E3, N) small size (13), noninduc-              potential transmembrane domains (9) C-termi- <br /> ibility mitomycin (21), absence lysis gene                  nal ATP-binding cassette homologous energy-trans- <br /> product release (30). Like E. coli hemolysin (11, 20)           ducing proteins (23). The chromosomally encoded outer mem- <br /> Erwinia protease (25), ColV secreted signal sequence-               brane protein, TolC, involved secretion ColV <br /> independent pathway. The secretion requires exporter                E. coli outer membrane, similar secretion of <br /> proteins (16), (CvaB) contains putative ATP-                 alpha-hemolysin (16, 41). <br /> binding domain (16) high degree sequence similarity                  Previous minicell analysis shown cvaA gene <br /> to MDR-like family proteins involved various trans-               produces in-frame products, CvaA CvaA*, estimated <br /> port processes (14, 22).                                                    43 27 kDa, respectively (15). CvaA believed be <br />    The clue dedicated set proteins required             integral membrane protein, CvaA* lacks hy- <br /> for ColV secretion came observation 900-bp                  drophobic residues N-terminal region CvaA <br /> region pColV-B188 plasmid, contains ColV                   proposed reside cytoplasm (8, 16, 36). In <br /> structural gene (cvaC) immunity gene (cvi), possessed                study, characterized cvaA gene products. <br /> only intracellular bactericidal activity (13). Later, Gilson et al.         Wild-type constructed plasmids used char- <br /> (15) identified 4.4-kb region related plasmid pColV-             acterize CvaA CvaA*. Since ColV increased iron- <br /> K30 required ColV synthesis, export, immunity.              depleted media (2, 9), effect iron cvaA gene expres- <br /> The complete nucleotide sequence genes deter-                   sion investigated. Site-directed mutagenesis was <br /> mined, genes encoded con-                   used prevent cvaA* gene expression determine the <br /> verging operons, cvi cvaC contain-               effect CvaA* extracellular ColV activity. The results <br /> ing cvaA cvaB (16). The cvi gene presumably encodes                   indicate CvaA* essential ColV secre- <br /> small membrane protein confers immunity cells                 tion, stabilizes CvaA, enhancing ColV secretion. <br />  <br />                                                                                                     MATERIALS AND METHODS <br />   * Corresponding author. Phone: (404) 651-3109. Fax: (404) 651- <br /> 2509. E-mail: biopct@panther.gsu.edu.                                         Media growth conditions. TB (10 g tryptone 8 g NaCl liter) <br />   &#226;&#8364;&#160; Present address: Department Microbiology, University Ala-         used liquid solid growth media activity tests protein <br /> bama Birmingham, Birmingham, AL 35294.                                   overproduction. For vivo labeling CvaA CvaA* proteins, M9 minimal <br />  <br />                                                                       689 <br />  690       HWANG ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                  TABLE 1. Bacterial strains plasmids used study <br /> Strain plasmid                                                          Genotype description                                                         Reference <br />  <br /> Strains <br />   BL21(lDE3)                                  F2 ompT hsdSB l prophage carrying T7 RNA pol gene                                                39 <br />   MC4100                                      F2 araD139 DlacU169 rpsL relA thiA                                                                          15 <br />   ZK4                                         MC4100 recA56                                                                                               15 <br />   71-18                                       D(lac-proAB) thi supE F9 lacIq lacZM15                                                                      46 <br />  <br /> Plasmids <br />   pHK11                                       pBR322 cvaABC cvi genes                                                                            15 <br />   pHK11-4                                     pHK11 cvaB::Tn5                                                                                        15 <br />   pTJ112                                      pHK11 mutated cvaA (M156A, M160A); CvaA* expression                                                 This study <br />   pTJ117                                      pBR322 cvaA*                                                                                           This study <br />   pTJ118                                      pHK11 cvaC cvi                                                                                     This study <br />   pHK22                                       pACYC184 cvaABC cvi genes                                                                          15 <br />   pTJ222                                      pHK22 mutated cvaA (M156A, M160A); CvaA* expression                                                 This study <br />   pHK22-5                                     pHK22 cvaB::Tn5                                                                                        15 <br />   pTJ227                                      pHK22 cvaA*                                                                                       This study <br />   pT7-6                                       Vector T7 RNA pol promoter                                                                             40 <br />   pTM11                                       pT7-6 PCR-made cvaA*                                                                                   This study <br />   pTM6                                        pT7-6 PCR-made cvaA position 228 Fig. 1                                                        This study <br />   pTM14                                       pT7-6 PCR-made cvaA position 9 Fig. 1                                                          This study <br />   pTM15                                       pT7-6 mutated cvaA; CvaA* expression                                                                This study <br />   pQF52                                       Translational lacZ fusion vector                                                                            24 <br />   pQF52cvaA15                                 Translational cvaA-lacZ fusion residue 15                                                                This study <br />   pQF52cvaC13                                 Translational cvaC-lacZ fusion residue 13 mature CvaC                                                 This study <br />  <br />  <br />  <br /> medium used. Antibiotics used following final concentrations:         otide sequences ColV operon defined Gilson et al. [16], artificial <br /> ampicillin, 150 mg/ml; chloramphenicol, 30 mg/ml.                                     restriction enzyme sites underlined): primer 1, 224 266 59 end of <br />    Bacterial strains plasmids. The strains plasmids used study    pTM6 HindIII site, 59-GCG CCA TGA AGC TTT AGG AGA CAG <br /> listed Table 1. E. coli strains (ZK4 71-18) plasmids (pHK11, pHK11-4,      AAA AAT GAA GTG GCA GGG A-39, artificial Shine-Dalgarno (SD) <br /> pHK22, pHK22-5) kindly provided R. Kolter (Harvard Medical                sequences (boldface) introduced; primer 2, 1527 1485 39 end <br /> School, Boston, Mass.) (15). BL21(lDE3), containing T7 RNA polymerase             pTM6 EcoRI site, 59-CGC GAG CAG ATT TAG AAT TCG TCT <br /> gene integrated chromosome, used protein overexpression,           GAA ATT CCT GTT AGT CAT T-39; primer 3, 5 42 59 end of <br /> strain ZK4 used ColV activity test cvaA expression. The    pTM14 HindIII site, 59-GCG CAT TTA ATA AAG CTT GAC ATT <br /> ColV-sensitive strain 71-18 used lawn cells activity tests (7).            CTT TCA TCA TAA AT-39; primer 4, 1744 1711 39 end of <br />    pHK11 derived pBR322 contains cvaABC cvi genes (15).           pTM14 natural SmaI site, 59-ACT CCA GTG AAA GAG CCC GGG <br /> Site-directed mutagenesis used construct plasmids lacking CvaA* expres-        TGG CCA TCC CCA G-39; primer 5, 685 712 59 end of <br /> sion; pTJ112 derivative pHK11, pTJ222 derivative pHK22,           pTM11 HindIII site, 59-GCG CTA TAA AGC TTG CAG AGG AAG <br /> pTM15 derivative pT7-6. pTJ227 constructed follows using            GGA T-39. Primer 2 used 39 end pTM11. HindIII-EcoRI-digested <br /> pHK22-5, derivative pHK22 cvaB::Tn5 (15). An in-frame              PCR products pTM6 pTM11 HindIII-SmaI-restricted fragments for <br /> deletion cvaA introduced KpnI site pHK22-5 using S1 nuclease      pTM14 subcloned pT7-6 vector digestion corresponding <br /> to delete cvaA gene retaining intact cvaA*. Then, SalI fragment     enzymes purification agarose gels. For pTJ118, HindIII pieces were <br /> removed in-frame-mutated plasmid delete cvaB cvi cvaC, yield-        removed pHK11-4 (15) self-ligated. <br /> ing pTJ227. HindIII fragments pTJ227, contains cvaA*,                 Site-directed mutagenesis. pTJ112 constructed PCR mutated <br /> inserted pBR322 produce pTJ117. Constructions pT7-6 cvaA          oligonucleotides putative ATG starting codons CvaA*&#226;&#8364;&#8221;at <br /> gene starting ATG nucleotide number 252 (pTM6) TTG                  positions 720 732 Fig. 1&#226;&#8364;&#8221;were changed GCG alanine. Two internal <br /> number 219 (pTM14) carried PCR (Fig. 1). pTM11 con-              primers used mutagenesis: primer 6, nucleotide positions 705 to <br /> structed pT7-6 PCR, cvaA* gene. PCRs carried        748, 59-GAA GGG ATA AAA ATA GCG AAA AAT AAT GCG GAG AAT <br /> pHK11 template following primers (numbers denote nucle-            TAC AGA T-39, primer 7 748 705, complementary sequence, <br />                                                                                       59-A TCT GTA ATT CTC CGC ATT ATT TTT CGC TAT TTT TAT CCC <br />                                                                                       TTC-39. Primers 3 4, described above, used 59- 39-end <br />                                                                                       primers, respectively. PCR left fragment performed primers 3 <br />                                                                                       7, PCR right fragment primers 4 6, using <br />                                                                                       pHK11 template. PCR products separated extracted 1% <br />                                                                                       agarose gels. The purified PCR products containing mutated sequences were <br />                                                                                       used template final PCR primers 3 4. The generated <br />                                                                                       PCR fragment used replace HindIII-SmaI fragment pHK11 after <br />                                                                                       digestion enzymes. However, resulted deletion unse- <br />                                                                                       quenced upstream region cvaA. This cvaA upstream-deleted plasmid then <br />                                                                                       digested HindIII KpnI, HindIII-KpnI fragment original <br />                                                                                       pHK11 reinserted regenerate unsequenced upstream portion cvaA, <br />                                                                                       yielding pTJ112. This plasmid cvaBC, cvi, mutated cvaA gene (M156A <br />                                                                                       M160A) lacks CvaA* expression. HindIII-SalI fragments pTJ112 <br />                                                                                       subcloned pACYC produce pTJ222. An identical mutated cvaA <br />                                                                                       gene (M156A M160A) cloned pT7-6 similar manner gen- <br />                                                                                       erate pTM15, primer 2 used instead primer 4 ampli- <br />                                                                                       fied fragment subcloned pT7-6 digestion HindIII EcoRI. <br />    FIG. 1. Translational start CvaA/A*. The previously proposed transla-           The constructed pTM15 mutated cvaA cvaB cvi-cvaC, as <br /> tional start site CvaA (top) possible sites CvaA* (bottom)      verified sequence analysis. <br /> ColV operon depicted (16). The numbers sides refer nucle-             Translational lacZ fusion b-galactosidase activity assay. A cvaA-lacZ <br /> otide position. A new possible initiation site (TTG) previously          fusion constructed promoter region N-terminal 15 amino acids <br /> reported (ATG) CvaA putative ATG sites CvaA*                  CvaA PCR SmaI sites end inserted the <br /> boldfaced. SD sequences protein underlined.                              translational fusion vector, pQF52 (24). Transformants selected 5-bro- <br />  VOL. 179, 1997                                                                                    ROLE OF IN-FRAME cvaA GENE PRODUCT                              691 <br />  <br />  <br /> mo-4-chloro-3-indolyl-b-D-galactopyranoside (X-Gal) plates (28). A cvaC-lacZ           tive 71-18-overlaid TB plates. ColV activity quantified critical <br /> fusion prepared PCR entire cvi gene promoter region           dilution method (27). <br /> and N-terminal 13 amino acids cvaC. The cvaA-lacZ cvaC-lacZ fusion             Reagents. All chemicals reagents reagent grade purchased <br /> plasmids called pQF52cvaA15 pQF52cvaC13, respectively.                        Sigma Chemical Co. unless noted. Restriction enzymes, T4 DNA <br />    b-Galactosidase activities determined follows: MC4100 cells contain-        ligase, S1 nuclease Boehringer Mannheim. Q-Sepharose and <br /> ing pQF52cvaA15 pQF52cvaC13 grown overnight TB ampicillin              Sephacryl S-200 resins Pharmacia Biotech. [35S]methionine (1,100 <br /> and 0.5 mM FeCl3 repress cva operons. Cells washed twice               Ci/mmol) used vivo labeling purchased DuPont NEN. <br /> diluted optical density (OD) 0.1 fresh TB containing ampicillin 0.1 <br /> mM FeCl3. When cell OD reached 0.5, 2,29-dipyridyl (final concentration, 0.5 <br /> mM) added half culture volume deplete iron. Then, 1 ml cells                                       RESULTS <br /> cultured used assay times indicated (see Fig. 4 6). To test <br /> the effect CvaA*, ZK4 cells containing fusion plasmids                The cvaA gene encodes proteins translated differ- <br /> CvaA* induced, b-galactosidase activity measured following             ent in-frame start sites. Previously, proteins, CvaA (43 <br /> Miller&#226;&#8364;&#8482;s methods (28). <br />    DNA manipulations. Plasmid DNA isolation, digestion, transformations,           kDa) CvaA* (27 kDa), detected cvaA mini- <br /> other routine DNA manipulations described Sambrook et al. (32).             cell analysis (15). To verify characterize pro- <br />    In vivo labeling cvaA/A* gene products. BL21(lDE3) cells grown              teins, PCR-synthesized cvaA cvaA* subcloned into <br /> overnight, diluted 1:100 TB, allowed grow 378C ampicillin      vector plasmid pT7-6 bacteriophage T7 RNA poly- <br /> the OD 600 nm (OD600) ;0.4. Cells washed twice resus- <br /> pended M9 medium containing ampicillin 19-amino-acid mixture with- <br />                                                                                        merase promoter. The plasmids transformed E. <br /> out methionine. After 1 h growth 378C, T7 RNA polymerase induced             coli BL21(lDE3), contains gene T7 RNA poly- <br /> with addition isopropyl-b-D-thiogalactopyranoside (IPTG) (final concen-         merase chromosome control inducible <br /> tration, 1 mM) incubation 10 min. To inhibit expression host            lacUV5 promoter (39). Based DNA sequence the <br /> proteins, 200 mg rifampin ml added incubated 10 min 428C. <br /> Cells shifted 308C, 20 min later 50 mCi [35S]methionine per <br />                                                                                        ColV operon, plasmids cvaA cvaA* <br /> ml added incubated additional 7 min allow selective labeling      constructed. In addition previously proposed ATG <br /> plasmid-encoded proteins. Cells washed 10 mM Tris-HCl, pH 7.6,           nucleotide position 252 potential translational start <br /> boiled sodium dodecyl sulfate (SDS) sample buffer. Labeled cell samples        site CvaA (16), possible start, TTG, at <br /> autoradiographed SDS&#226;&#8364;&#8220;12% polyacrylamide gel electrophoresis (PAGE).              position 219, better SD sequence (Fig. 1). Thus, <br />    Overproduction CvaA* antibody production. BL21(lDE3)/pTM11 <br /> grown overnight diluted 1:100 6 liters ampicillin-containing TB medium       synthetic cvaA genes prepared: pTM6 designed <br /> and allowed grow 378C OD600 ;0.5. IPTG (final con-            translate proposed ATG using artificial SD <br /> centration, 1 mM) added incubated 3 h, cells harvested         sequence, pTM14 constructed include entire <br /> centrifugation. Cells resuspended 50 mM Tris-HCl (pH 8.0)&#226;&#8364;&#8220;10 mM                cvaA gene, upstream regions. To determine whether <br /> EDTA&#226;&#8364;&#8220;1 mM phenylmethylsulfonyl fluoride lysed French press. <br /> Lysed cells centrifuged 27,000 3 g 10 min. The pellet, containing the <br />                                                                                        CvaA* independently translated, cvaA* gene was <br /> overproduced CvaA* inclusion bodies, washed treated 6 M urea           subcloned separately (pTM11). Figure 2A shows results of <br /> containing 50 mM Tris-HCl, pH 8.0. CvaA* purified 2.5- 25-cm             [35S]methionine labeling vivo cells containing of <br /> Q-Sepharose (Pharmacia Biotech) column 0.3 M NaCl concentrated                different plasmids. Two proteins pTM6 pTM14 <br /> with Amicon concentrator (Amicon Inc.). Concentrated sample then <br />                                                                                        (CvaA CvaA*), pTM11 (CvaA*), <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> fractionated 1.5- 100-cm Sephacryl S-200 gel filtration column (Pharmacia <br /> Biotech) urea buffer. This CvaA* .95% pure remained              pTM15 (CvaA) synthesized. However, size CvaA <br /> soluble dialysis remove urea.                                                 pTM14 cloned TTG appears larger CvaA <br />    For antibody production, CvaA* purified separation prepar-        pTM6 cloned downstream ATG, CvaA <br /> ative SDS&#226;&#8364;&#8220;10% PAGE. CvaA* gel identified, electroeluted             pTM15 start sites present appears larger as <br /> electroelutor (Isco Inc.), concentrated microconcentrator (Amicon <br /> Inc.). Purified CvaA* (1 mg) used primary injection Freund&#226;&#8364;&#8482;s          well, suggesting upstream TTG position 219 the <br /> adjuvant, following boosters serum preparation performed               actual translation site (see below). <br /> general standard procedures (19). Immunoglobulin G (IgG) isolated                  Since CvaA number hydrophobic residues N <br /> DEAE chromatography, CvaA*-specific IgG purified                terminus, proposed protein reside the <br /> CvaA*-conjugated affinity column using Reacti-gel 63 (Pierce Biochemicals <br /> Co.). CvaA*-specific antibodies eluted 0.1 M glycine, pH 2.5 (19). <br />                                                                                        inner membrane. On hand, CvaA* encoded in <br /> Anti-CvaA* antibodies pooled, dialyzed, stored 2208C 50%              frame CvaA identical CvaA that <br /> glycerol.                                                                              lacks hydrophobic N-terminal residues. CvaA* there- <br />    Electroeluted CvaA* used N-terminal sequencing CvaA*. CvaA               fore proposed cytosolic protein (8, 16, 36). Using vivo <br /> from total membrane fraction BL21(lDE3)/pTM14 used N-terminal             labeling experiments followed fractionation cells produc- <br /> sequencing. The sample electrophoresed SDS&#226;&#8364;&#8220;12% PAGE then <br /> blotted polyvinylidene difluoride Problot membrane (Applied Biosys-             ing CvaA CvaA*, confirmed CvaA resides in <br /> tems). CvaA bands, identified Coomassie blue staining, excised used        inner membrane CvaA* located cytosol <br /> for N-terminal sequencing Beckman LF 3000 protein sequencer (Beck-              (data shown). For reasons, smaller protein <br /> man Co.). This performed DNA/Protein Core Facility Georgia               CvaA* used produce anti-CvaA/A* antibodies. <br /> State University. <br />    Protein expression Western blots. BL21(lDE3) cells used                BL21(lDE3)/pTM11 used purify CvaA* pro- <br /> CvaA/A* expression. Cells containing appropriate plasmids grown overnight              duced large quantities strain. After 3 h induction, <br /> were diluted OD600 ;0.1. At OD600 ;0.5, cells containing            expression CvaA* reached maximum CvaA* was <br /> pTM6 pTM14 induced 1 mM IPTG, containing                   detected inclusion bodies. Therefore, CvaA* was <br /> pHK11 pTJ112 induced 0.5 mM 2,29-dipyridyl. Induced cells were <br /> harvested 2 h later, protein concentrations lysates measured by <br />                                                                                        purified presence 6 M urea (Fig. 2B), antibodies <br /> the micro-bicinchoninic acid method (Pierce Biochemicals Co.). To determine            prepared standard methods (19). CvaA/A*- <br /> the effect CvaA* CvaA stability, ZK4 cells used, 200 mg              specific IgGs purified CvaA*-conjugated <br /> kanamycin ml added stop new protein synthesis 30 min               affinity column used immunoblotting experi- <br /> induction. Electrophoresed proteins transferred polyvinylidene difluo-         ments study. <br /> ride membranes, membranes treated purified anti-CvaA/A* an- <br /> tibodies established procedures Western blots (19). Alkaline phosphatase-          Identification likely translational initiation codon <br /> conjugated goat anti-rabbit IgG (Promega) used secondary antibody.          TTG CvaA ATGs CvaA*. Both <br />    ColV activity test. Extracellular intracellular ColV activity detected   CvaA CvaA* possess possible translation initiation <br /> a halo assay described (35) modifications. ZK4 cells containing      sites (Fig. 1). For CvaA*, potential ATG start <br /> various plasmids grown overnight TB diluted OD600 ;0.1. When <br /> the OD600 reached 0.5, cells harvested washed remove existing <br />                                                                                        sites 3 amino acids apart. For CvaA, possible TTG with <br /> extracellular ColV resuspended fresh TB containing 0.5 mM 2,29-            better SD sequence located 11 amino acids upstream the <br /> dipyridyl. Chloroform-treated supernatant samples spotted ColV-sensi-          previously proposed ATG. To identify true initiation site of <br />  692       HWANG ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                           FIG. 3. Translation cvaA genes expressed T7 control, wild-type, and <br />                                                                                        CvaA*-deleted plasmids. (A) BL21(lDE3) cells pTM6, pTM14, pHK11, <br />                                                                                        pTJ112 cultured; induced IPTG 2,29-dipyridyl; harvested. <br />                                                                                        Immunoblot performed purified anti-CvaA* antibodies (see Materials <br />                                                                                        Methods). Lane 1, purified CvaA* (100 ng); lane 2, pTM6 (1 mg), lane <br />                                                                                        3, pTM14 (1 mg); lane 4, pHK11 (10 mg); lane 5, pTJ112 (10 mg). <br />                                                                                        To detect CvaA*, 100 mg total-cell proteins pHK11 (lane 6) pTJ112 (lane <br />                                                                                        7) used. (B) Comparison CvaA size. Samples cells containing pTM6 <br />                                                                                        (lane 1), pTM14 (lane 2), pHK11 (lane 3) electrophoresed CvaA* <br />                                                                                        moved the gel. Only CvaA shown Western <br />    FIG. 2. Overproduction cvaA gene products. (A) 35S-labeled cvaA/A*               immunoblotting. A, CvaA; A*, CvaA*. <br /> gene products. BL21(lDE3) cells pTM6, pTM14, pTM11, pTM15 were <br /> radiolabeled induction autoradiographed SDS-PAGE (see Ma- <br /> terials Methods). A, CvaA; A*, CvaA*. (B) CvaA* expression from <br /> BL21(lDE3)/pTM11. Cells TB induced 1 mM IPTG OD of <br /> ;0.5, 3 ml cells harvested. The cell pellet suspended boiled        (employing complementation) demonstrated crucial role <br /> with 400 ml 2% SDS solution. Twenty micrograms proteins                CvaA ColV secretion (15). However, function of <br /> sample used SDS-PAGE followed Coomassie staining. T0, cells with-           CvaA* elucidated. In addition, CvaA* <br /> out induction; T3, cells 3 h induction; A*, purified CvaA* used for <br /> N-terminal peptide sequencing. (C) Overexpression CvaA. BL21(lDE3)/ <br />                                                                                        shown appears synthesized independently from <br /> pTM14 cells grown described Materials Methods. Twenty micro-            CvaA, possibility arises CvaA degradation <br /> grams protein samples used SDS-PAGE. T0, total cells in-            processing ruled out. We measured the <br /> duction; T3, cells 3 h induction; P, pellet containing total cell membrane;   stability CvaA CvaA* proteins synthesized from <br /> A, CvaA used N-terminal sequencing. <br />                                                                                        pTM14, pTM15, pTM11, conditions pro- <br />                                                                                        teins labeled shortly induction. In pTM15 (Table 1), <br />                                                                                        methionines putative translational initiation sites of <br /> CvaA*, purified CvaA* (Fig. 2B) used N-terminal                                CvaA* changed alanines, vivo labeling showed <br /> peptide sequencing. The sequence Met-Lys-Asn-Asn-Met-                              CvaA* absent (Fig. 2A). Labeling pTM11 showed <br /> Glu-Asn-Tyr-Arg-Tyr-Tyr, indicating Met                             CvaA* produced independently CvaA (Fig. 2A). <br /> CvaA* derived the ATGs (Fig. 1).                              Additionally, CvaA CvaA* synthesized pTM14 <br /> Immunoblotting used compare CvaA pTM6                                  relatively stable, similar patterns observed <br /> pTM14 wild-type plasmid, pHK11 (Fig. 3).                              pTM15 pTM11, produce CvaA and <br /> CvaA pTM14 appeared slightly larger                                CvaA*, respectively (data shown). These data, together <br /> pTM6. When separated SDS-PAGE, difference                               absence CvaA* cells containing mutated CvaA <br /> in size CvaA pTM14 pTM6 pro-                                  (pTM15 Fig. 2A) presence start site for <br /> nounced showed CvaA pTM14 mobility                                 CvaA* (Fig. 1), support previous minicell analysis Gil- <br /> similar CvaA wild-type plasmid (Fig. 3B).                          son et al. (16) CvaA* degradation or <br /> N-terminal sequencing CvaA membranes                                        processing CvaA. <br /> BL21(lDE3)/pTM14 (Fig. 2C) gave sequence Met-Phe-                                     Synthesis cvaA gene products negatively regulated <br /> Arg-His-Asp-Ala-Leu-Glu, confirms upstream                              iron. Chehade Braun (2) previously reported ColV <br /> TTG (position 219 Fig. 1), ATG proposed                             activity increased nutritionally poor medium TB <br /> previously, true translational start CvaA.                                  medium treated 2,29-dipyridyl (an iron chelator). In <br />    Under artificial conditions T7 promoter, con-                        addition, ColV activity constitutively elevated Fur <br /> centration CvaA* equal (with pTM14)                            mutant (2). These findings indicate ColV synthesis reg- <br /> than (with pTM6) CvaA (Fig. 3). However, ratio                          ulated iron Fur. Previously, consensus Fur binding sites <br /> CvaA*/CvaA cells wild-type plasmid (Fig. 3A, lane                          upstream cvaA cvi-cvaC genes (17). <br /> 6) T7 constructs (Fig. 3A, lane                         To investigate further, specific effect iron the <br /> 3). This indicates regulation translation                         cvaA gene products tested BL21(lDE3)/pHK11 cells. <br /> initiation TTG known, used                           Immunoblottings cells incubated 2,29-dipyri- <br /> efficiently standard ATG.                                                           dyl showed CvaA CvaA* produced the <br />    CvaA* degradation product CvaA. Previous stud-                          presence iron chelator (Fig. 4A). With 2,29-dipyridyl, <br /> ies Tn5 insertions different regions cvaA gene                          synthesis CvaA CvaA* maximum 30 <br />  VOL. 179, 1997                                                                                 ROLE OF IN-FRAME cvaA GENE PRODUCT                                693 <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 5. Extracellular activity wild-type CvaA*-depleted cells. Extra- <br />                                                                                    cellular ColV ZK4 cells containing wild-type pHK11 (W) mutated pTJ112 <br />                                                                                    (M) plasmids measured. Cells induced mid-log phase (OD;0.5) and <br />                                                                                    incubated 378C 2 h. To effect CvaA*, pTJ227, contains <br />                                                                                    cvaA*, added cells containing mutated plasmid (M1A*). ColV activity <br />                                                                                    quantified critical dilution method. Five microliters samples diluted <br />                                                                                    chloroform-treated supernatant spotted ColV-sensitive 71-18- <br />                                                                                    overlaid TB plates. The activity obtained based highest twofold serial <br />                                                                                    dilution gave clear ColV halo. <br />  <br />  <br />    FIG. 4. Iron effect cvaA gene expression. (A) Cells containing wild-type <br /> pHK11 plasmid grown 2,29-dipyridyl (Dip) time <br /> indicated, cell extracts used immunoblotting. (Top) Twenty        These data indicate CvaA* contributes increased ColV <br /> micrograms sample used; (bottom) 150 mg sample used <br /> to detect CvaA*. (B) Effect iron chelator cvaA cvaC expression. <br />                                                                                    secretion. <br /> MC4100 cells containing pQF52cvaA15 pQF52cvaC13 induced (1Dip)               CvaA* stabilizes CvaA. Although CvaA* appears contrib- <br /> or induced (2Dip) 2,29-dipyridyl OD reached 0.5, 1 ml cells   ute ColV secretion, observation direct or <br /> was taken time point, b-galactosidase activity measured         indirect: reduced ColV secretion observed cells with <br /> Miller&#226;&#8364;&#8482;s method (see Materials Methods). <br />                                                                                    mutated CvaA decreased synthesis sta- <br />                                                                                    bility CvaC CvaA. Synthesis CvaC CvaA measured <br />                                                                                    translational lacZ fusions showed CvaC nor <br />                                                                                    CvaA affected presence absence CvaA* (Fig. <br /> min, fourfold increase proteins de-                       6A). The effect CvaA* CvaC analyzed an <br /> tected. Translational lacZ fusions showed cvaA                      internal ColV activity test. Internal ColV activities cells <br /> expression cvaC expression induced addition                      CvaA* (pTJ118) CvaA* (pTJ118/pTJ227) <br /> 2,29-dipyridyl (Fig. 4B). These results confirm Fur                       similar, new protein synthesis pre- <br /> repressor, inhibits transcription binding Fur-regu-                    vented kanamycin, ColV activities disappeared similar <br /> lated promoters presence iron, negative                         rates (data shown), indicating CvaA* effect on <br /> regulator expression cvaA cvaC genes.                           CvaC stability CvaC itself rapidly turned the <br />    Depletion CvaA* decreases ColV secretion. Complemen-                         cell secreted (8). However, CvaA* does appear affect <br /> tation (15) frameshift mutagenesis Gilson et al. (16)                       stability CvaA. The degradation CvaA cells lacking <br /> demonstrated ColV secreted CvaA                             CvaA* faster wild-type cells, stability was <br /> CvaB (an observation confirm). To determine                        recovered addition CvaA* (Fig. 6B). <br /> CvaA* required ColV secretion, identical                             The recovery ColV activity (Fig. 5) CvaA stability <br /> site-directed <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> pTM15 (Fig. 2A) introduced                      (Fig. 6B) complementation CvaA* CvaA mutant <br /> into pHK11 knock CvaA* expression retain altered                        complete. This discrepancy re- <br /> CvaA. Results showed mutated plasmid, pTJ112, did                        duced activity mutated CvaA reduced levels CvaA* <br /> not produce CvaA* (Fig. 3A, lanes 5 7). By halo assay,                       introduced lower-copy-number plasmid. The <br /> extracellular ColV activity compared wild-type                         ColV secretion defect CvaA*-depleted cells pro- <br /> and CvaA*-depleted cells. It showed CvaA*-depleted cells                      nounced 208C, reduced activity re- <br /> secreted ColV based reduced activity supernatant                    stored complementing CvaA* (data shown) seen <br /> than did wild-type cells. To quantify difference activity,                  378C, indicating mutated CvaA defective 208C. The <br /> ColV activity measured critical dilution method                         difference copy number addressed follow. Whereas <br /> (27); activity highest twofold serial dilution gave                initially pBR-derived mutated plasmid (pTJ112) comple- <br /> a clear ColV halo. The result showed cells lacking                        mented pACYC-derived cvaA*-containing plasmid <br /> CvaA* secreted times active ColV did                          (pTJ227), reversed scheme complemented pA- <br /> wild-type cells (Fig. 5). To observe complementation                       CYC-derived mutated plasmid pBR-derived cvaA* plas- <br /> CvaA* restore ColV activity, CvaA* introduced                            mid. ColV activity measured cells containing pACYC- <br /> back mutated cells compatible plasmid,                            derived wild-type plasmid (pHK22) CvaA-mutated <br /> pTJ227 (a derivative pACYC), containing cvaA*. Cells                       plasmid (pTJ222) complemented CvaA* pBR322- <br /> complemented CvaA* pTJ227 showed higher ColV                             derived pTJ117. The recovered activity (Fig. 7A) CvaA <br /> activity mutated CvaA (Fig. 5).                          stability high wild-type cells (Fig. 7B). <br />  694      HWANG ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 7. Effect double mutations CvaA function. Extracellular ColV <br />                                                                                    activity (A) stability CvaA (B) examined using cells containing <br />                                                                                    wild-type pHK22 (W) mutated pTJ222 (M) plasmids cells containing <br />                                                                                    pTJ222 pTJ117 (M1A*). The ColV activity test CvaA stability mea- <br />    FIG. 6. Function CvaA* protein. (A) Effect CvaA* CvaC CvaA         surement Fig. 5 6B, respectively. <br /> expression. ZK4/pQF52cvaC13 ZK4/pQF52cvaA15 cells (1A*) or <br /> without (2A*) pTJ227 used determine effect CvaA* the <br /> expression CvaC CvaA, respectively. (B) Role CvaA*. ZK4 cells <br /> containing pHK11(W), pTJ112(M), pTJ112/pTJ227 (M1A*) induced <br /> with 2,29-dipyridyl 30 min. To stop new protein synthesis, kanamycin   binding sites 4 10 nucleotides upstream initia- <br /> added, cells taken time point. Twenty micrograms cell     tion codon, UUG (1, 34). This GGAG sequence found <br /> sample subjected SDS-PAGE followed immunoblotting. Each <br /> CvaA band quantified using Image Analysis System (PDI, Inc.). <br />                                                                                    upstream UUG CvaA (Fig. 1). <br />                                                                                       Minicell analysis cvaA <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> showed that <br />                                                                                    CvaA* result independent translational product, <br />                                                                                    CvaA* produced CvaA synthesis was <br /> These results CvaA mutation dele-                        interrupted frameshift mutation (16). We verified <br /> terious 208C, does affect function CvaA 378C                   CvaA* translated in-frame start site indepen- <br /> and CvaA* restores activity.                                             dently CvaA. Furthermore, CvaA* appears important <br />    Taken together, results indicate CvaA* es-                      enhancing secretion ColV, strains without <br /> sential ColV secretion enhances secretion sta-                      CvaA* fourfold lower activity wild-type cells. This <br /> bilizing CvaA protein.                                                         decreased activity recovered addition CvaA* <br />                                                                                    compatible plasmid. These results clearly demon- <br />                               DISCUSSION                                           strate CvaA* function ColV-producing <br />                                                                                    cells. One possibility CvaA* functions chaperone to <br />    The accessory gene cvaA encodes in-frame proteins,                          guide stabilize CvaC itself. However, CvaA* does affect <br /> CvaA CvaA*, protein possible transla-                         ColV synthesis stability. Instead, study shows that <br /> tion initiation sites. Two AUG methionine codons separated                         CvaA* stabilizes CvaA leads increased ColV secre- <br /> by 3 amino acids exist CvaA*, CvaA UUG 11                            tion. The C terminus CvaA predicted reside the <br /> amino acids upstream previously proposed AUG. N-                            periplasm (36). Therefore, proposed stabilizing interaction of <br /> terminal peptide sequencing determined AUG                      CvaA* CvaA probably occur C terminus. <br /> CvaA* upstream UUG CvaA represent trans-                           Perhaps CvaA* interacts transporter protein(s) CvaA <br /> lational start sites genes. Early experiments                     and/or CvaB cytoplasm forms stable complex. As <br /> genetic <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> showed UUG, AUG GUG,                              overlapping gene, cheA locus Salmonella typhimurium <br /> can stimulate binding N-formylmethionyl-tRNA ri-                         E. coli encodes proteins (CheAL CheAS) <br /> bosomes (3). In vivo experiments confirmed UUG                           translated different in-frame start sites (33, 38), <br /> acts initiator codon proteins (1, 12, 29, 37, 43).                similar CvaA CvaA*. When CheAS deleted E. <br /> In addition, proteins GGAG-type ribosome                            coli, chemotactic behavior efficient expres- <br />  VOL. 179, 1997                                                                       ROLE OF IN-FRAME cvaA GENE PRODUCT                                 695 <br />  <br />  <br /> sion CheAL reduced (33). Recently, reported                                             REFERENCES <br /> that CheAS form complex CheZ protein en-                   1. Adhin, M. R., J. van Duin. 1989. Translational regulation lysis <br /> hances dephosphorylating activity CheY-phosphate (42).                gene RNA bacteriophage fr requires UUG initiation codon. Mol. Gen. <br />                                                                              Genet. 218:137&#226;&#8364;&#8220;142. <br /> Although complexes CvaA* proteins                  1a.Boyer, A., P. C. Tai. Unpublished data. <br /> been identified, decreased levels CvaB detected           2. Chehade, H., V. Braun. 1988. Iron-regulated synthesis uptake of <br /> CvaA*-depleted cells (unpublished data). Furthermore, de-                    colicin V. FEMS Microbiol. Lett. 52:177&#226;&#8364;&#8220;182. <br /> creased amounts CvaA CvaB observed cells                   3. Clark, B. F. C., K. A. Marcker. 1966. The role N-formylmethionyl- <br />                                                                              tRNA protein biosynthesis. J. Mol. Biol. 17:394&#226;&#8364;&#8220;406. <br /> which lack expression CvaB CvaA, respectively,             4. Davies, J. K., P. Reeves. 1975. Genetics resistance colicins in <br /> the amounts proteins drastically reduced TolC-               Escherichia coli K-12: cross-resistance colicins group B. J. Bacte- <br /> minus cells (unpublished data). This result implies                riol. 123:96&#226;&#8364;&#8220;101. <br /> these proteins form complex, stabilize other,             5. deLorenzo, V., S. Wee, M. Herrero, J. B. Heilands. 1987. Operator <br />                                                                              sequences aerobactin operon plasmid ColV-K30 binding ferric <br /> and CvaA* increase stability complex                uptake regulation (fur) repressor. J. Bacteriol. 169:2624&#226;&#8364;&#8220;2630. <br /> cytoplasmic side. Another possibility CvaB            6. Dinh, T., I. T. Paulsen, M. H. Saier, Jr. 1994. A family extracyto- <br /> cytoplasmic ATP-binding domain C terminus, CvaA*                      plasmic proteins allow transport large molecules outer <br /> functions affecting nucleotide binding maintain-            membranes gram-negative bacteria. J. Bacteriol. 176:3825&#226;&#8364;&#8220;3831. <br />                                                                           7. Fath, M. J., R. C. Skvirsky, R. Kolter. 1991. Functional complementa- <br /> ing complex structure stability enhance ColV secretion.                tion bacterial MDR-like export systems: colicin V, alpha-hemolysin, <br /> Thus, overlapping gene products, CheAs CvaAs,                    Erwinia protease. J. Bacteriol. 173:7549&#226;&#8364;&#8220;7556. <br /> smaller protein essential contributes function           8. Fath, M. J., R. C. Skvirsky, L. Gilson, H. K. Mahanty, R. Kolter. 1992. <br /> chemotaxis ColV secretion, respectively.                                 The secretion colicin V, p. 331&#226;&#8364;&#8220;348. In R. James, F. Lazdunski, F. <br />                                                                              Pattus (ed.), Bacteriocins, microcins, lantibiotics. Springer-Verlag, Hei- <br />    It known ColV receptor, Cir,                delberg, Germany. <br /> genes located pColV-K30 plasmids regulated iron (5,             9. Fath, M. J. 1993. Studies secretion colicin V. Ph.D. dissertation. <br /> 44). In presence iron, Fur protein known bind               Harvard University, Cambridge, Mass. <br /> to promoter regions repress expression genes           10. Fath, M. J., L. H. Zhang, J. Rush, R. Kolter. 1994. Purification and <br />                                                                              characterization colicin V Escherichia coli culture supernatant. Bio- <br /> Fur-controlled regulon. The genes functioning synthesis               chemistry 33:6911&#226;&#8364;&#8220;6917. <br /> and uptake catechol siderophore enterobactin              11. Felmlee, T., S. Pellet, R. A. Welch. 1985. Nucleotide sequence an <br /> known involved regulon (5). Previously, Chehade                Escherichia coli chromosomal hemolysin. J. Bacteriol. 163:94&#226;&#8364;&#8220;105. <br /> and Braun (2) showed ColV activity increased cells            12. Files, J. G., K. Weber, C. Coulondre, J. H. Miller. 1975. Identification of <br />                                                                              UUG codon translational initiation codon vivo. J. Mol. Biol. <br /> grown iron chelator ColV production con-                 95:327&#226;&#8364;&#8220;330. <br /> stitutive independent iron Fur mutant. In addition,          13. Frick, K. K., R. L. Quakenbush, J. Konisky. 1981. Cloning immunity <br /> cells produce colicin V grown iron-                 structural genes colicin V. J. Bacteriol. 148:498&#226;&#8364;&#8220;507. <br /> poor medium TB rich medium like Luria                14. Gerlach, J. H., J. A. Endicott, P. F. Juranka, G. Henderson, F. Sarangi, K. L. <br />                                                                              Deuchar, V. Ling. 1986. Homology P-glycoprotein bac- <br /> broth. Additionally, potential Fur binding sites up-               terial haemolysin transport protein suggests model multidrug resis- <br /> stream cvi cvaC cvaA cvaB operons, supporting                     tance. Nature 324:485&#226;&#8364;&#8220;489. <br /> Fur-dependent iron regulation colicin V operons (17).             15. Gilson, L., H. K. Mahanty, R. Kolter. 1987. Four plasmid genes are <br /> Even cvaA, cvi, cvaC induced iron depletion,               required colicin V synthesis, export, immunity. J. Bacteriol. 169: <br />                                                                              2466&#226;&#8364;&#8220;2470. <br /> we Fe31 completely shut synthesis           16. Gilson, L., H. K. Mahanty, R. Kolter. 1990. Genetic analysis an <br /> ColV (Fig. 4) (1a). Since CvaB cotranscribed                     MDR-like export system: secretion colicin V. EMBO J. 9:3875&#226;&#8364;&#8220;3884. <br /> CvaA (9), regulated iron. Indeed, CvaB            17. Gilson, L. 1990. Genetic analysis signal sequence-independent protein <br /> was detected presence iron chelator (data shown).              transport system: secretion colicin V. Ph.D. dissertation. Harvard Univer- <br />                                                                              sity, Cambridge, Mass. <br /> Thus, CvaA CvaB kinds regulation                18. Hardy, K. G. 1975. Colicinogeny related phenomena. Bacteriol. Rev. <br /> similar CvaC, regulated iron                 39:464&#226;&#8364;&#8220;515. <br /> independent iron.                                                     19. Harlow <br /> </body></html>