<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2000 Genetics Society America <br />  <br />  <br />  <br />    Four Chromo-domain Proteins Schizosaccharomyces pombe Differentially <br />           Repress Transcription Various Chromosomal Locations <br />  <br />                                          Genevie`ve Thon Janne Verhein-Hansen <br />              Department Genetics, Institute Molecular Biology, University Copenhagen, DK-1353 Copenhagen K, Denmark <br />                                                           Manuscript received July 9, 1999 <br />                                                       Accepted publication February 7, 2000 <br />  <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Transcription repressed regions fission yeast genome close centromeres, telomeres, or <br />              silent mating-type cassettes mat2-P mat3-M. The repression involves chromo-domain proteins <br />              Swi6 Clr4. We report chromo-domain proteins, Chp1 Chp2, important <br />              position effects. Chp1 showed specificity centromeric regions. Its essentiality the <br />              transcriptional repression centromeric markers correlates importance chromosome stability. <br />              Chp2 appeared pleiotropic. Its effects centromeric silencing pronounced those <br />              Chp1, participated telomeric position effects transcriptional silencing mating-type <br />              region. We PolII-transcribed genes repressed placed Schizosaccharo- <br />              myces pombe rDNA clusters, situation analogous budding yeast Saccharomyces cerevisiae. Chp2, <br />              Swi6, Clr4, and, lesser extent, Chp1 participated repression. <br />  <br />  <br />  <br /> E   UKARYOTIC chromosomes organized do- <br />      mains distinctive properties, are <br /> the potential transcription recombination. <br />                                                                              cause partially symmetrical arrangement relative <br />                                                                              central core, centromeric sequences been <br />                                                                              proposed fold hairpin. The rDNA forms two <br /> These parameters vary chromosomes ac-                          large clusters &#226;&#181;&#8216;500&#226;&#8364;&#8220;1000 kb ends chromo- <br /> cording DNA sequence, proximity special-                       3 (Umesono et al. 1983; Toda et al. 1984; Allshire <br /> ized structures telomeres centromeres,                       et al. 1987; Hoheisel et al. 1993). Telomeres consist of <br /> the specific state individual cells. Repetitive sequences                 degenerate repeats sequence TTACAGG. These <br /> and proteins interact define                        repeats &#226;&#181;&#8216;300 bp long. They adjacent the <br /> regions transcription recombination re-                        rDNA genes ends chromosome 3 <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> duced. For example, heterochromatic areas,                         telomere-associated sequences ends chromo- <br /> predominantly composed repeated sequences,                            somes 1 2 (Sugawara 1989). <br /> poorly transcribed. In Drosophila silence eu-                          Position effects fission yeast transcription have <br /> chromatic genes translocated nearby (reviewed                             observed near centromeres, telomeres, the <br /> Weiler Wakimoto 1995). In yeast Saccharo-                            vicinity mat2-P mat3-M silent mating-type cas- <br /> myces cerevisiae, recombination PolII transcription                      settes right arm chromosome 2. The mat2-P <br /> occur differently rDNA, chromosomal region                          mat3-M cassettes linked 4.3 kb DNA <br /> where repeats occur naturally eukaryotes, com-                       separates displays striking sequence homol- <br /> pared genomic regions (see references                          ogy centromeric repeat (Grewal Klar <br /> Smith et al. 1999).                                                          1997). Marker genes introduced near centromeres, telo- <br />   We interested position effects fission yeast                 meres, mating-type region repressed, either <br /> Schizosaccharomyces pombe determining                         stringently variegated fashion, mechanisms <br /> repeated sequences influence effects. S. pombe                     utilize trans-acting factors Clr1, Clr2, Clr3, Clr4, <br /> three chromosomes respectively 5.7, 4.6, 3.5 Mb                       Swi6, Rik1 (Lorentz et al. 1992; Thon Klar <br /> (Fan et al. 1988; Hoheisel et al. 1993; Mizukami et                          1992; Allshire et al. 1994, 1995; Ekwall Ruusala <br /> al. 1993). Each chromosome contains 40- 100-kb                          1994; Nimmo et al. 1994; Thon et al. 1994). In addition to <br /> centromeric region comprising 4- 7-kb central core                      repressing transcription, factors inhibit meiotic <br /> of unique sequence flanked large inverted repeats                         recombination mating-type region (Egel et al. <br /> and repeats placed tandem (Takahashi et al. 1992;                         1989; Klar Bonaduce 1991; Lorentz et al. 1992; <br /> Steiner Clarke 1994 references therein). Be-                         Thon et al. 1994), indicating position effects on <br />                                                                              transcription recombination involve related <br />                                                                              structures mechanisms. Whether silencing fac- <br />    Corresponding author: Genevie`ve Thon, Department Genetics, In-        tors responsible reduced recombination <br /> stitute Molecular Biology, University Copenhagen, &#195;&#732;ster Fari- <br /> magsgade 2A, DK-1353 Copenhagen K, Denmark.                                  observed centromeric areas (Nakaseko et al. 1986; <br /> E-mail: gen@biobase.dk                                                       Chikashige et al. 1989), repress tran- <br /> Genetics 155: 551&#226;&#8364;&#8220;568 ( June 2000) <br />  552                                          G. Thon J. Verhein-Hansen <br />  <br /> scription recombination genomic locations,           Bacterial strains used cloning: Cloning plasmid DNA <br /> is known. In addition position effects          performed Escherichia coli strains DH5 (Hanahan <br />                                                               1983) S1754 (Thon et al. 1999). <br /> transcription, position effects recombination            Cloning chp1: chp1 contained 4.3-kb BamHI- <br /> been documented S. pombe transplacement study         Ecl 136I fragment genomic DNA. That fragment puri- <br /> of ade6-M26 hot spot recombination              fied cosmid ICRFc60G0618 (Reference Library, ICRF; <br /> showed ability M26 <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> promote        Lehrach 1990) cloned Bluescribe (Stratagene, La <br /> recombination ade6 gene depends chro-           Jolla, CA) digested BamHI HincII, create pGT121. <br />                                                               The presence chp1 pGT121 ascertained restriction <br /> mosomal context (Virgin et al. 1995).                         mapping partial sequencing. <br />    Among proteins associate chromatin          Partial replacement chp1 LEU2 ura4&#226;&#180;&#8482;: EcoRV <br /> determine properties proteins containing chro-      cleaves pGT121 75 bp downstream predicted initiating <br /> matin organization modifier (chromo) domain (for re-          ATG chp1 open reading frame (ORF) Bgl II cleaves <br /> view Cavalli Paro 1998). This motif         pGT121 1355 bp upstream predicted stop codon. A <br />                                                               total 1450 bp pGT121 DNA EcoRV and <br /> identified Drosophila HP1 Pc proteins (Paro        Bgl II sites replaced 2.2-kb EheI-BamHI fragment <br /> and Hogness 1991) subsequently &#226;&#172;&#381;40 proteins            pJJ250 (Jones Prakash 1990), contains S. <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> various organisms (Aasland Stewart 1995;             cerevisiae LEU2 gene, create pGT122, 1.8-kb <br /> Koonin et al. 1995; current sequence databases). It        HincII-BamHI fragment pON94 (a gift Olaf Nielsen), <br /> found, combination do-          contains S. pombe ura4&#226;&#171;&#185; gene, create pGT162. <br />                                                               pON94 constructed filling ends 1.8-kb <br /> mains, repressors activators transcription <br />                                                               HindIII fragment, contains ura4&#226;&#171;&#185; (Grimm et al. 1988), <br /> and proteins localize pericentric het-     adding SphI linkers, ligating product SphI <br /> erochromatin. The chromo-domain related               site modified version pUC19 (Yannish-Perron et al. <br /> chromo-shadow domain mediate formation pro-            1985) HindIII site converted BamHI site. <br /> tein complexes association chromatin           The insert pGT122 released digesting BamHI <br /> (Messmer et al. 1992; Powers Eissenberg 1993;             ScaI used transform diploid strain produced by <br />                                                               mating PG9 PG1141. chp1&#226;&#171;&#185;/chp1&#226;&#338;&#172;::LEU2 diploids <br /> Saunders et al. 1993; Platero et al. 1995; Strutt         identified Leu&#226;&#171;&#185; transformants Southern <br /> Paro 1997). Many potential partners pro-            blot genomic DNA restricted SphI SphI Nsi and <br /> posed interact chromo-domain proteins, includ-        hybridized chp1 probe described (Southern <br /> ing components nuclear membrane (Ye                Northern blot analyses). Two diploids subjected <br /> Worman 1996; Ye et al. 1997) components            tetrad dissection. The insert pGT162 released with <br />                                                               BamHI FspI used transform SP837. Ura&#226;&#171;&#185; trans- <br /> origin recognition complex (Pak et al. 1997).                 formants chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele identified South- <br />    Two chromo-domain proteins, Swi6 Clr4, influ-          ern blot genomic DNA digested NsiI SphI SacI <br /> ence position effects S. pombe (Lorentz et al. 1992;       probed chp1 probe. <br /> Ekwall Ruusala 1994; Thon et al. 1994; Allshire              Deletion chp1 substituted marker: pGT121 was <br /> et al. 1995). Swi6 contains chromo- chromo-             digested BsaAI AccI, filled ends the <br />                                                               Klenow fragment E. coli DNA polymerase I, religated, <br /> shadow domain (Lorentz et al. 1994; Aasland and <br />                                                               create pGT187. The entire chp1 ORF deleted pGT187, <br /> Stewart 1995). Clr4 contains chromo- SET do-            33 nucleotides 5&#226;&#172;&#732; 77 nucleotides 3&#226;&#172;&#732; chp1 <br /> main [Su(var)3-9, Enhancer zeste, trithorax; Tschiersch    ORF. The insert pGT187 released BamHI SphI <br /> et al. 1994; Ivanova et al. 1998]. When swi6        used transform PG1544. The DNA FOAR trans- <br /> clr4 mutated deleted, genes normally subject         formants digested NsiI BamHI Ecl 136III and <br /> transcriptional silencing near centromeres, telomeres,        hybridized chp1 probe, allowed identify <br />                                                               transformants chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele PG1544 had <br /> or mating-type region derepressed. Consistent      replaced chp1&#226;&#338;&#172; allele. <br /> with phenotypes, Swi6 localizes centromeres,            Replacement chp2 ura4&#226;&#180;&#8482;: The 1.8-kb HindIII frag- <br /> telomeres, mating-type region (Ekwall et           ment containing S. pombe ura4&#226;&#171;&#185; gene (Grimm et al. 1988) <br /> al. 1995). Clr4 important localization Swi6     cloned Bluescript KSII(&#226;&#171;&#185;) (Statagene) EcoRV <br /> (Ekwall et al. 1996) and, expressed plas-         site ura4&#226;&#171;&#185; gene close EcoRV site polylinker, <br />                                                               create pGT188. The oligonucleotides GTO-150 (5&#226;&#172;&#732; TCCCC <br /> mid, itself seen rDNA (Sawin Nurse 1996).       CCGGGAGCTCAGATCGTTATACACTTTACGTATCTAGG 3&#226;&#172;&#732;) <br /> We tested chromo-domain proteins            GTO-151 (5&#226;&#172;&#732; CGGGATCCGTCGACTATTAAGACTTTC <br /> of S. pombe, Chp1 (Doe et al. 1998) Chp2,            CAGATATACCAAC 3&#226;&#172;&#732;) used amplify 777 bp geno- <br /> involved processes Swi6 Clr4. In           mic DNA chp2 ORF. GTO-152 (5&#226;&#172;&#732; AACTGC <br /> addition assaying effects Chp1 Chp2          AGTCGACTTGATCTTTGGACTTAATAATTAGAATTAC <br />                                                               G 3&#226;&#172;&#732;) GTO-153 (5&#226;&#172;&#732; TCCCCCGGGAAGCTTCTCGAGATG <br /> the regions silencing previously observed, we <br />                                                               GCGCTTGAAGGGCTTAGTGCGC 3&#226;&#172;&#732;) used amplify <br /> investigated Swi6, Clr4, Chp1, Chp2 modi-          776 bp genomic DNA the chp2 ORF. <br /> fied efficiency PolII transcription rDNA.       The oligonucleotides purchased Operon Technolo- <br />                                                               gies (Alameda, CA). The PCR performed Native <br />                                                               Pfu DNA polymerase (Stratagene) 80 &#226;?&#174;l containing the <br />              MATERIALS AND METHODS                            buffer provided manufacturer, 1 &#226;?&#174;g S. pombe genomic <br />                                                               DNA, 1 &#226;?&#174;m primer, 0.5 mm dNTPs. The amplifica- <br />   Sequence chp1 chp2: Chp1 (GenBank accession no.      tion cycles follows: 3 min 94&#226;&#172;&#353;; 15 times [1 min 94&#226;&#172;&#353;, 1 <br /> Q10103) encoded cosmid SPAC18G6 (Z68198)        min 60&#226;&#172;&#353;, 1 min 72&#226;&#172;&#353;]; 7 min 72&#226;&#172;&#353;. The PCR products gel <br /> Chp2 (CAA16917) SPBC16C6 (AL021767).                       purified. The product amplified GTO-150 GTO-151 <br />                                                Fission Yeast Chromo-domain Proteins                                           553 <br />  <br /> was digested Sac I BamHI. The product amplified           Amersham (Piscataway, NJ), following templates: the <br /> with GTO-152 GTO-153 digested Sal I SmaI.         BamHI-SphI fragment pGT121 chp1; purified PCR <br /> These fragments ligated four-way ligation       product amplified genomic DNA GTO-150 and <br /> Bluescript SKII(&#226;&#171;&#186;) (Stratagene) digested SacI SmaI        GTO-153 chp2; 1.8-kb HindIII fragment ura4 <br /> and 1.8-kb BamHI-Sal I fragment pGT188,          (Grimm et al. 1988); 2-kb BamHI-Sal I pJJ250 for <br /> contains ura4&#226;&#171;&#185;. A clone containing ura4&#226;&#171;&#185; gene flanked          LEU2 ( Jones Prakash 1990). Antisense ura4 riboprobes <br /> by sequences normally flanking chp2 ORF obtained           Northern hybridization Riboprobe II <br /> from ligation named pGT190. The insert pGT190          core (Promega), 3000 Ci/mmol [&#226;?&#163;-32P]UTP (Amer- <br /> was released SacI XhoI used transform SP837.       sham), T7 RNA polymerase (Promega), pON140 (a gift <br /> The DNA Ura&#226;&#171;&#185; transformants digested HindIII         Olaf Nielsen) linearized BamHI template. pON- <br /> BamHI PstI hybridized chp2 probe described          140 clone ura4&#226;&#171;&#185; gene pGEM4 (Promega). <br /> below (Southern Northern analyses). This identified             Stability Ura&#226;&#171;&#186; <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span>: FOAR colonies purified <br /> strain chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele, PG1744.                       rounds streaking FOA-containing plates and <br />    Replacement chp2 LEU2: The 3.7-kb BamHI-PstI            patched YES plates. Following growth nonselective <br /> fragment 0.8-kb Pst I-SalI fragment pGT190         conditions YES &#226;&#181;&#8216;15 generations, cells suspended <br /> ligated three-way ligation 2-kb BamHI-SalI        water used spot tests selective media DNA <br /> pJJ250, contains S. cerevisiae LEU2 gene ( Jones     preparation. <br /> Prakash 1990). A clone similar pGT190, containing              Chromosome stability: Mitotic loss minichromosome <br /> LEU2 instead ura4&#226;&#171;&#185;, obtained ligation,            Ch16m23::ura4&#226;&#171;&#185;-Tel[72] assayed minichro- <br /> pGT193. The insert pGT193 released digesting        mosome Ch16 (Allshire et al. 1995): cells containing the <br /> Sac I XhoI used transform diploid strain produced     ade6-M210 allele chromosome 3, complementing allele <br /> by mating PG1636 PG1637. chp2&#226;&#171;&#185;/chp2&#226;&#338;&#172;::LEU2              ade6-M216 Ch16m23::ura4&#226;&#171;&#185;-Tel[72], mutations in- <br /> diploids identified Leu&#226;&#171;&#185; transformants         terest propagated medium lacking adenine (AA-ade) <br /> Southern blot analysis similar described     33&#226;&#172;&#353;, plated medium containing small ade- <br /> chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele diploids subjected        (MSA supplemented 100 mg uracil, 200 mg l-leu- <br /> tetrad dissection.                                                 cine, 15 mg adenine liter), incubated 33&#226;&#172;&#353;. The <br />    Deletion chp2 substituted marker: pGT190         loss rate Ch16m23::ura4&#226;&#171;&#185;-Tel[72] determined the <br /> digested Sal I. The 3.7-kb Sal I fragment, contains     number colonies red sector equal greater than <br /> Bluescript 777 bp DNA flanking chp2 centromere-      half colony divided sum white sectored <br /> proximal side, ligated 0.8-kb Sal I fragment,   colonies. <br /> contains 776 bp DNA flanking chp2 centromere-distal <br /> side. A clone precise deletion chp2 ORF was <br /> obtained ligation, pGT191. The insert pGT191                                   RESULTS <br /> was released digesting SacI XhoI used to <br /> replace chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele PG1753. The DNA FOAR <br />                                                                       Deletion chromo-domain protein genes, chp1 <br /> transformants digested ClaI BamHI PstI         chp2, S. pombe: A gene capable encoding a <br /> hybridized chp2 probe. This identified strain    109-kD protein N-terminal chromo-domain is <br /> chp2&#226;&#338;&#172; allele, PG1776.                                              present left arm S. pombe chromosome 1. That <br />    Introduction ura4&#226;&#180;&#8482; rDNA repeats: The 1.8-kb           gene subject previous study named <br /> HindIII fragment containing ura4&#226;&#171;&#185; (Grimm et al. 1988) was <br />                                                                    chp1 (chromo-domain protein S. pombe; Doe et al. <br /> cloned pUC8 (Viera Messing 1982). The resulting <br /> construct linearized Bsr FI used transform         1998). Another gene, refer chp2, is <br /> PG1634. Ura&#226;&#171;&#185; transformants analyzed Southern blots         located right arm chromosome 2 encodes <br /> of genomic DNA restricted EcoRV, HindIII, BamHI            43-kD protein displaying extensive sequence similarity <br /> and hybridized LEU2 ura4 probe. A transformant      S. pombe Swi6 protein (Figure 1). Like Swi6 and <br /> pUC8 ura4 integrated pBR322 sequence present         family proteins related Drosophila HP1, Chp2 <br /> the rDNA repeat PG1634 saved PG1661. <br />    Yeast media culture conditions: YES (Thon Friis         contains chromo- chromo-shadow domain. <br /> 1997) used rich medium; MSA (Egel et al. 1994) supple-      The chp1 chp2 ORFs discovered the <br /> mented 100 mg adenine, 100 mg uracil, 200 mg              S. pombe genome sequencing project (Sanger Center). <br /> l-leucine liter used sporulation medium; drop-out          We created deletion alleles chp1 chp2: <br /> media (Rose et al. 1990), FOA medium (Thon et al. 1999),           chp1 entire chp2 ORF replaced <br /> and YE (Moreno et al. 1991) used test auxotrophies. <br /> Plates incubated 33&#226;&#172;&#353;.                                      S. cerevisiae LEU2 gene, functional homolog <br />    Yeast transformation strain construction: A lithium ace-    S. pombe leu1&#226;&#171;&#185; gene, S. pombe ura4&#226;&#171;&#185; <br /> tate protocol adapted Heyer et al. (1986) Moreno          gene. Deletion entire ORF substituted <br /> et al. (1991) used transforming S. pombe. Strains trans-   markers obtained. The deletions originally <br /> formed constructs containing LEU2 ura4&#226;&#171;&#185; se-           introduced chromosome diploid cells. The <br /> lected drop-out media lacking leucine uracil. FOA me- <br />                                                                    chp1 chp2 deletions proved viable hap- <br /> dium used select loss ura4&#226;&#171;&#185;. The strains produced <br /> by transformation progeny subsequent crosses        loid progeny transformed diploids. This been <br /> are listed Table 1.                                             observed independently chp1 (Doe et al. 1998). chp1- <br />    Southern Northern blot analyses: S. pombe DNA           deleted cells formed colonies significantly smaller than <br /> prepared according Moreno et al. (1991) RNA ac-             wild type, chp2-deleted cells formed colo- <br /> cording Schmitt et al. (1990) liquid YES cultures          nies large wild type. <br /> incubated 30&#226;&#172;&#353;. Blotting hybridization conditions were <br /> as described previously (Thon et al. 1999). The probes used           Deleting chp1 chp2 alleviates centromeric position <br /> for Southern hybridization random priming         effects: S. pombe centromeres organized large <br /> kit (Promega, Madison, WI), 3000 Ci/mmol [&#226;?&#163;-32P]dCTP          inverted repeats (Takahashi et al. 1992; Steiner and <br />                                                                                                                                       554 <br />  <br />  <br />                                                                TABLE 1 <br />                                                        Strains genotypes <br />  <br /> Strain                                                  Genotype                                                   Sourcea <br /> FY336           h&#226;&#171;&#185; cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210                                         Allshire et al. (1994) <br /> FY340           h&#226;&#171;&#185; TM1(NcoI)-ura4&#226;&#171;&#185; Random int. leu1-32 ura4-DS/E ade6-210                                   Allshire et al. (1994) <br /> FY489           h&#226;&#171;&#186; leu1-32 ura4-DS/E ade6-210                                                               Allshire et al. (1995) <br /> FY501           h&#226;&#171;&#185; imr L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                      Allshire et al. (1995) <br /> FY520           h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (ch16 ade6-216)                       Nimmo et al. (1995) <br /> FY521           h&#226;&#171;&#186; Ch16 m23::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216)                               Allshire et al. (1995) <br /> FY611           h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115              Allshire et al. (1995) <br /> FY648           h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210                                            Allshire et al. (1995) <br /> FY939           h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                 Allshire et al. (1995) <br /> FY967           h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                Allshire et al. (1995) <br /> FY986           h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                              Allshire et al. (1995) <br /> PG9             mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216                                             Thon Klar (1992) <br /> PG445           mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-M216                                                  Thon et al. (1998) <br /> PG596           mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 swi6-115                                    Thon et al. (1994) <br /> PG912           mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M210 clr4-681                                    Thon et al. (1994) <br /> PG1032          mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 clr4-681                         This study <br /> PG1141          h90 leu1-32 ura4-D18, ade6-210                                                              Thon et al. (1999) <br /> PG1402          mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-M216                                     This study <br /> PG1419          mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-M216 clr4-681                            This study <br /> PG1454          mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 chp1&#226;&#338;&#172;::LEU2                                           This study <br /> PG1544          h90 leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  This study <br /> PG1572          mat1-Msmt-0 leu1-32 ura4-D18 Ylp2.4 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                            This study <br /> PG1594          mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-M216 clr4-681                                         This study <br /> PG1600          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E -D18 ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2   This study <br />                                                                                                                                      G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br /> PG1605          mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp1&#226;&#338;&#172;::LEU2                      This study <br /> PG1634          h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 swi6-115                                                           This study <br /> PG1636          h90 leu1-32 ura4-DS/E ade6-210                                                              This study <br /> PG1637          h90 leu1-32 ura4-DS/E ade6-216                                                              This study <br /> PG1639          mat1-Msmt-0 &#226;&#338;&#172;(BglII-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp1&#226;&#338;&#172;::LEU2       This study <br /> PG1640          mat1-Msmt-0 &#226;&#338;&#172;(BglII-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 swi6-115          This study <br /> PG1661          h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 swi6-115                                                This study <br /> PG1691          h&#226;&#171;&#185; imr L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                          This study <br /> PG1699          h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                     This study <br /> PG1703          h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                    This study <br /> PG1709          h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                  This study <br /> PG1717          h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                                This study <br /> PG1721          h&#226;&#171;&#185; cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                             This study <br /> PG1731          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681              This study <br /> PG1732          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2           This study <br /> PG1734          h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7                                                        This study <br /> PG1736          h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 clr4-681                                               This study <br />   (Continued) <br />                                                                                  TABLE 1 <br />                                                                               (Continued) <br />  <br /> Strain                                                                  Genotype                                                                          Sourcea <br /> PG1744                       h90 ura4-D18 leu1-32 ade6-216 chp2&#226;&#338;&#172;::ura4                                                                            This study <br /> PG1753                       h90 ura4-DS/E leu1-32 ade6-210 chp2&#226;&#338;&#172;::ura4                                                                           This study <br /> PG1763                       h90 cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                     This study <br /> PG1764                       h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                         This study <br /> PG1765                       h&#226;&#171;&#185; imr1 L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                  This study <br /> PG1767                       h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                              This study <br /> PG1769                       h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                             This study <br /> PG1771                       h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                           This study <br /> PG1773                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp2&#226;&#338;&#172;::LEU2                                   This study <br /> PG1774                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp2&#226;&#338;&#172;::LEU2                                    This study <br /> PG1776                       h90 ura4-DS/E leu1-32 ade6-210 chp2&#226;&#338;&#172;                                                                                 This study <br /> PG1778                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  This study <br /> PG1779                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  This study <br /> PG1781                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  This study <br /> PG1783                       mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                          This study <br /> PG1787                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp2&#226;&#338;&#172;::LEU2                               This study <br /> PG1790                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp2&#226;&#338;&#172;::LEU2                                               This study <br /> PG1792                       h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 chp1&#226;&#338;&#172;                                                                            This study <br /> PG1794                       h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 chp2&#226;&#338;&#172;                                                                           This study <br /> PG1796                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               This study <br /> PG1797                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-210 swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               This study <br /> PG1799                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-216 chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               This study <br /> PG1970                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 chp1&#226;&#338;&#172;::LEU2                          This study <br /> PG1972                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681 chp1&#226;&#338;&#172;::LEU2                          This study <br />                                                                                                                                                                              Fission Yeast Chromo-domain Proteins <br />  <br />  <br />  <br />  <br /> PG1974                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 chp2&#226;&#338;&#172;::LEU2                           This study <br /> PG1976                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681 chp2&#226;&#338;&#172;::LEU2                          This study <br /> PG1978                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-D18 ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2 chp2&#226;&#338;&#172;::LEU2                         This study <br /> PG1984                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 clr4-681                              This study <br /> SP837                        h90 leu1-32 ura4-D18 ade6-216                                                                                        Thon et al. (1994) <br /> SP1124                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M216                                                                   Thon et al. (1994) <br /> SP1126                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M210 swi6-115                                                          Thon et al. (1994) <br /> SP1151                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M216                                                   Thon et al. (1994) <br /> SP1165                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 clr4-681                                  Thon et al. (1994) <br /> YIp2.4-1                     h&#226;&#171;&#186; leu1/YIp2.4                                                                                                       Toda et al. (1984) <br />   a <br />     The complete genealogy strains constructed study available request. Mutated engineered alleles used strain constructions described <br /> in following studies: mat1-P&#226;&#338;&#172;17::LEU2: Arcangioli Klar (1991); mat1-Msmt-0 : Engelke et al. (1987) Styrkarsdottir et al. (1993); mat3-M(EcoRV)::ura4&#226;&#171;&#185;: <br /> Thon Klar (1992); mat3-P(EcoRV)::ura4&#226;&#171;&#185;: Thon Klar (1993); mat2-P(XbaI)::ura4&#226;&#171;&#185; &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; : Thon et al. (1994); &#226;&#338;&#172;(482)mat3-M: Thon <br /> et al. (1999); ura4-D18: Grimm et al. (1988); ade6-M210 ade6-M216: Gutz (1963); swi6-115: Gutz Schmidt (1985); swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;: Lorentz et al. (1994); clr4-681: <br /> Thon et al. (1994). <br />                                                                                                                                                                              555 <br />  556                                            G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br />                                                                                                   Figure      1.&#226;&#8364;&#8221;Sequence <br />                                                                                                 comparison Chp2 and <br />                                                                                                 Swi6. The predicted protein <br />                                                                                                 sequences Chp2 Swi6 <br />                                                                                                 aligned using the <br />                                                                                                 CLUSTAL W program <br />                                                                                                 (Thompson et al. 1994). <br />  <br />  <br />  <br />  <br /> Clarke 1994 references therein). The ura4&#226;&#171;&#185; gene             difference FOA-containing media used. <br /> was placed locations centromere           The ura4&#226;&#171;&#185; transcript relative control <br /> of chromosome 1 Allshire et al. (1994, 1995). Inser-         transcript ura4-DS/E similar studies. <br /> tion locations results level expression         As noted (Allshire et al. 1995), of <br /> inferior ura4&#226;&#171;&#185; gene wild-type location    ura4&#226;&#171;&#185; transcript inferior wild type <br /> and variegated phenotype clonally derived            support effective growth medium lacking <br /> fractions cell population express ura4&#226;&#171;&#185;,         uracil. <br /> the rest cells repressed. Swi6 Clr4         Deletion chp1 completely derepressed centro- <br /> required centromeric repression (Allshire et           meric loci caused small derepression cnt1, the <br /> al. 1995).                                                      central centromeric insertion site (Figure 2, A and <br />    We tested effect Chp1 Chp2 expres-         B). Deletion chp2 partially derepressed ura4&#226;&#171;&#185; the <br /> sion ura4&#226;&#171;&#185; centromeric integration sites (Figure      centromeric sites tested. It increased growth me- <br /> 2). In experiment, compared ability            dium lacking uracil reduced growth medium <br /> of wild-type, chp1-, chp2-deleted cells containing         containing FOA (Figure 2A). It increased the <br /> centromeric ura4&#226;&#171;&#185; gene form colonies medium               ura4&#226;&#171;&#185; transcript detected Northern blot <br /> lacking uracil medium containing FOA, toxigenic            relative arising sites in <br /> substrate Ura4 protein (Grimm et al. 1988; Figure        wild-type cells (Figure 2B). However, derepression <br /> 2A). In second experiment, examined                  strikingly small compared caused dele- <br /> amount ura4&#226;&#171;&#185; transcript present cells         tion chp1. This true sites examined except <br /> strains Northern blot analysis (Figure 2B). Tran-            possibly cnt1. <br /> scripts originating ura4DS/E allele (Allshire             The FOA resistance strains containing ura4&#226;&#171;&#185; cen- <br /> et al. 1994), truncated ura4 gene occupying natural       tromeric marker generally reduced deletion of <br /> chromosomal location, used internal control.         chp1 chp2, noted approximately cell <br />    In wild-type cells, tightest repression ura4&#226;&#171;&#185;     thousand able generate full-sized colony <br /> observed outer centromeric repeats (otr1L)               FOA-containing medium; frequency cells <br /> within dg1 region. Insertion central area        varied culture culture. chp&#226;&#171;&#185; strains gave rise to <br /> (cnt) inner repeats (imr1L) allowed               similar kind papillation (Figure 2A). We investi- <br /> higher expression ura4&#226;&#171;&#185;. These phenotypes agree              gated nature FOA resistance propagating <br /> with reported Allshire et al. (1994, 1995).            cells well-growing FOAR colonies nonselec- <br /> Some strains (FY939, FY967, FY501, FY336) grew             tive conditions measuring ability form <br /> poorly FOA medium experiments origi-             colonies absence uracil. We performed <br /> nally observed (compare Figure 2A Figure 1C            Southern blots test possible alterations ura4 <br /> Figure 2B Allshire et al. 1995). We believe        gene. Two independent FOAR colonies strain <br />                                             Fission Yeast Chromo-domain Proteins                                        557 <br />  <br />  <br />  <br />  <br />                                                                                               Figure 2.&#226;&#8364;&#8221;Effect Chp1 <br />                                                                                            Chp2 gene expression <br />                                                                                            near centromere 1. (A) Expres- <br />                                                                                            sion ura4&#226;&#171;&#185; gene assayed <br />                                                                                            plating efficiency selec- <br />                                                                                            tive media. Serial 10-fold dilu- <br />                                                                                            tions cell suspensions were <br />                                                                                            spotted indicated me- <br />                                                                                            dia. The cells contained cen- <br />                                                                                            tromeric insertions ura4&#226;&#171;&#185; <br />                                                                                            depicted B (1&#226;&#8364;&#8220;6), <br />                                                                                            random integration ura4&#226;&#171;&#185; <br />                                                                                            (R.I.), full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> ura4 <br />                                                                                            gene (&#226;&#171;&#186;). The strains were: 1: <br />                                                                                            &#226;&#171;&#185;, FY939; chp1&#226;&#338;&#172;, PG1699; chp- <br />                                                                                            2&#226;&#338;&#172;, PG1767; 2: &#226;&#171;&#185;, FY967; chp1&#226;&#338;&#172;, <br />                                                                                            PG1703; chp2&#226;&#338;&#172;, PG1769; 3: &#226;&#171;&#185;, <br />                                                                                            FY986; chp1&#226;&#338;&#172;, PG1709; chp2&#226;&#338;&#172;, <br />                                                                                            PG1771; 4: &#226;&#171;&#185;, FY648; chp1&#226;&#338;&#172;, <br />                                                                                            PG1717; chp2&#226;&#338;&#172;, PG1764; 5: &#226;&#171;&#185;, <br />                                                                                            FY501; chp1&#226;&#338;&#172;, PG1691; chp2&#226;&#338;&#172;, <br />                                                                                            PG1765; 6: &#226;&#171;&#185;, FY336; chp1&#226;&#338;&#172;, <br />                                                                                            PG1721; chp2&#226;&#338;&#172;, PG1763; R.I., <br />                                                                                            FY340; &#226;&#171;&#186;, FY489. (B) Northern <br />                                                                                            blot analysis ura4&#226;&#171;&#185; tran- <br />                                                                                            script. RNA prepared from <br />                                                                                            strains displayed A and <br />                                                                                            ura4&#226;&#171;&#185; transcript <br />                                                                                            estimated hybridization <br />                                                                                            ura4 riboprobe. <br />  <br />  <br />  <br />  <br /> displayed Figure 2, 1&#226;&#8364;&#8220;6, tested manner.        kb telomere, constructed steps <br /> We FOA resistance loss          (Niwa et al. 1989; Nimmo et al. 1994). ura4&#226;&#171;&#185; expressed <br /> the ura4&#226;&#171;&#185; gene strains examined FY986        variegated manner Ch16m23::ura4&#226;&#171;&#185;-Tel[72] <br /> and FY648. In FY986 FY648, FOA-resistant state         (Nimmo et al. 1994). Clr4 and, lesser extent, Swi6 <br /> returned efficiently Ura&#226;&#171;&#185; ura4&#226;&#171;&#185; gene ap-            contribute repression ura4&#226;&#171;&#185; location <br /> peared unchanged Southern blots. This epigenet-             (Allshire et al. 1995). We crossed Ch16m23::ura4&#226;&#171;&#185;- <br /> ically controlled resistance likely masked dele-    Tel[72] minichromosome into, respectively, chp1- and <br /> tion events occurring strains       chp2-deleted strains. Crosses swi6-115 clr4-681 <br /> tested.                                                        performed time comparison. We <br />    Deletion chp2, chp1, alleviates telomeric     assayed expression ura4&#226;&#171;&#185; mutant and <br /> position effects: Ch16 530-kb S. pombe minichromo-        wild-type strains issued cross spot <br /> some derived chromosome 3 radiation-induced            tests selective media. Strains identical genotypes <br /> breakage (Matsumoto et al. 1987). It contains ade6-        proved similar plating efficiencies the <br /> M216 allele, allows monitor presence       various selective plates (data shown). Spot tests were <br /> cells bearing ade6-M210 allele intragenic       repeated strain genotype the <br /> complementation ade6-M216 ade6-                ura4&#226;&#171;&#185; transcript estimated Northern <br /> M210 alleles (Gutz 1963). A truncated version Ch16,         blot. As seen Figure 3, deleting chp1 no <br /> Ch16m23::ura4&#226;&#171;&#185;-Tel[72], ura4&#226;&#171;&#185; gene &#226;&#181;&#8216;1             influence telomeric position effect, deleting chp2 <br />  558                                         G. Thon J. Verhein-Hansen <br />  <br />                                                              mosome loss (materials methods; Allshire et <br />                                                              al. 1995). In mutant strains tested, noticed the <br />                                                              presence colonies red narrow <br />                                                              white sector (Table 2). This phenotype indicative of <br />                                                              multiple consecutive missegregation events ex- <br />                                                              pected occur infrequently according calcu- <br />                                                              lated rates chromosome loss. The existence a <br />                                                              phenotype suggests mutations tested destabilize <br />                                                              chromosomes heritable manner fraction the <br />                                                              cell population. This property account for <br />                                                              relatively large variations loss rates observed in <br />                                                              independent cultures strain (Table 2). Alter- <br />                                                              natively, red colonies narrow white sector could <br />                                                              produced Ade&#226;&#171;&#185; cells temporarily stopped dividing <br />                                                              missegregation event. <br />                                                                 Deletion chp2, chp1, derepresses tran- <br />                                                              scription mating-type region: The mating-type <br />                                                              region S. pombe comprises linked cassettes, mat1, <br />                                                              transcribed, mat2-P mat3-M, are <br />                                                              silenced (Egel Gutz 1981; Beach 1983). Silencing <br />                                                              extends regions flanking mat2-P mat3-M and <br />                                                              10.9-kb region separates cassettes <br />                                                              (Thon Klar 1992; Thon et al. 1994; Grewal and <br />                                                              Klar 1997). It affects ura4&#226;&#171;&#185; gene placed at <br />                                                              XbaI site located &#226;&#181;&#8216;400 bp centromere-distal the <br />                                                              mat2-P cassette [mat2-P(XbaI)::ura4&#226;&#171;&#185; allele; Thon et al. <br />                                                              1994; Figure 4A] EcoRV site located &#226;&#181;&#8216;150 bp <br />   Figure 2.&#226;&#8364;&#8221;(Continued). <br />                                                              centromere-distal mat3-M cassette [mat3-M(Eco- <br />                                                              RV)::ura4&#226;&#171;&#185; allele; Thon Klar 1992; Figure 4B]. <br />                                                              The products clr1, clr2, clr3, clr4, clr6, swi6, rik1, esp1, <br /> caused increased expression ura4&#226;&#171;&#185;. The increased       esp2, esp3 essential repression. These <br /> expression caused chp2 deletion similar        factors act partially redundant pathways, such <br /> that caused mutation clr4.                           double mutations required completely abol- <br />    Role S. pombe chromo-domain proteins chromo-        ish silencing mating-type region. Double mu- <br /> some segregation: We measured rate loss        tations strong synergistic effects reside <br /> minichromosome Ch16m23::ura4&#226;&#171;&#185;-Tel[72] wild-               trans-acting factors belonging same <br /> type, chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, swi6-115, clr4-681       class consist combination cis- trans-acting <br /> cells. The results presented Table 2. swi6-115    mutations (Thon et al. 1994, 1999; Thon Friis <br /> chp1&#226;&#338;&#172;::LEU2 increased loss Ch16m23::ura4&#226;&#171;&#185;-Tel[72],        1997; Grewal et al. 1998). We examined Chp1 <br /> in accordance previous reports (Allshire et al.         Chp2 contributed repression ura4&#226;&#171;&#185; placed <br /> 1995; Doe et al. 1998). In contrast, deletion chp2     near mat2-P mat3-M, repression mat- <br /> the clr4-681 mutation modest possibly          ing-type genes mat2-P mat3-M. The tests per- <br /> significant effect rate loss compared     formed strains deletion cis-acting <br /> wild type. We constructed strains containing pairwise        silencing elements. <br /> combinations chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, swi6-             First, assayed ura4&#226;&#171;&#185; expression mating- <br /> 115, clr4-681 alleles. We double mu-      type region cells lacking chp1 chp2 spot tests on <br /> tants viable rate loss Ch16m23::     selective media (Figure 4). Wild-type, swi6-115, clr4- <br /> ura4&#226;&#171;&#185;-Tel[72] greater chp1&#226;&#338;&#172;::LEU2 swi6-115        681 cells spotted plates compari- <br /> and chp1&#226;&#338;&#172;::LEU2 clr4-681 double mutants          son. Cells lacking Chp1 grew FOA as <br /> in single mutants (Table 2). For        poorly medium lacking uracil cells containing a <br /> combinations tested, rate loss similar    functional protein, indicating Chp1 effect <br /> caused stronger mutation pair. The chp2        transcription ura4&#226;&#171;&#185; gene placed mating- <br /> deletion did increase chromosome loss      type region. In contrast, lack Chp2 allowed increased <br /> backgrounds tested. The minichromosome loss assay         growth absence uracil, indicating Chp2 <br /> used relies measuring changes white Ade&#226;&#171;&#185;           required repression ura4&#226;&#171;&#185; near mat2-P <br /> phenotype displayed cells containing minichro-        mat3-M. In addition forming colonies the <br /> mosome red Ade&#226;&#171;&#186; phenotype displayed chro-         absence uracil, cells deleted chp2 containing <br />                                              Fission Yeast Chromo-domain Proteins                                               559 <br />  <br />  <br />  <br />  <br />                                                                              Figure 3.&#226;&#8364;&#8221;Effect Chp1, Chp2, Swi6, Clr4 on <br />                                                                            gene expression near telomere. (A) Spot test analysis. <br />                                                                            Suspensions cells minichromosome <br />                                                                            Ch16m23::ura4&#226;&#171;&#185;-Tel[72] indicated mutations <br />                                                                            spotted Figure 2 estimate level of <br />                                                                            expression ura4&#226;&#171;&#185;. m23-TEL[72] strains: &#226;&#171;&#185;, FY520; <br />                                                                            chp1&#226;&#338;&#172;, PG1732; chp2&#226;&#338;&#172;, PG1773; swi6-115, FY611; clr4- <br />                                                                            681, PG1731; m23, FY489; &#226;&#171;&#186;, FY521. (B) Northern blot <br />                                                                            analysis. RNA prepared strains plated in <br />                                                                            A hybridized ura4 riboprobe. <br />  <br />  <br />  <br />  <br /> ura4&#226;&#171;&#185; near mat2-P form colonies FOA-                    deletion chp2 appeared equivalent swi6 and <br /> containing medium, indicating deletion chp2                   clr4 mutants (Figure 4B). <br /> did derepress ura4&#226;&#171;&#185; near mat2-P efficiently                    Wild-type S. pombe cells efficiently switch content <br /> mutations swi6 clr4 (Figure 4A, top). A stronger                mat1 cassette P M copying the <br /> derepression ura4&#226;&#171;&#185; obtained simultaneously                  silent information present mat2-P mat3-M. Follow- <br /> deleting chp2 mat2-P cis-acting element (Figure                 ing switches, cells mate form zygotic asci <br /> 4A, bottom). The derepression seen mat3-M following                isolated colonies. Strains unable switch their <br />  <br />  <br />                                                            TABLE 2 <br />                           Effect Chp1, Chp2, Swi6, Clr4 stability minichromosomea <br />  <br />                                         Half-sectored   Mostly red               Colonies           Loss rate cell   Mutant/ <br /> Strain             Background             colonies       colonies               examined              division (%)       WT ratio <br /> FY520        WT                          1,0,0,3        0,0,0,0          1098,1081,1022,2642               0.07               1 <br /> PG1600       chp1&#226;&#338;&#172;::LEU2                 6,4,7,9,12     2,4,9,6,9        1105,867,1085,421,1791            1.29              18 <br /> PG1774       chp2&#226;&#338;&#172;::LEU2                 1,1,1,1,6,3    0,0,2,0,0,1      896,984,1097,977,3519             0.21               3 <br /> FY611        swi6-115                    4,3,12,4,40    3,2,0,0,13       956,1016,568,663,1804             1.52              22 <br /> PG1731       clr4-681                    3,0,0,2,0      0,0,0,1,0        1230,973,998,854,651              0.13               2 <br /> PG1978       chp1&#226;&#338;&#172;::LEU2 chp2&#226;&#338;&#172;::LEU2     4,6,2          1,4,0            536,723,536                       0.95              14 <br /> PG1970       chp1&#226;&#338;&#172;::LEU2 swi6-115        25,5,24        11,4,31          896,269,618                       5.60              80 <br /> PG1972       chp1&#226;&#338;&#172;::LEU2 clr4-681        21,14,8        7,21,23          749,707,542                       4.75              68 <br /> PG1974       chp2&#226;&#338;&#172;::LEU2 swi6-115        15,4,2,11      1,3,0,0          774,456,231,1150                  1.38              20 <br /> PG1976       chp2&#226;&#338;&#172;::LEU2 clr4-681        2,0,2,3        0,0,0,0          1116,486,884,905                  0.21               3 <br /> PG1984       swi6-115 clr4-681           6,10,9         7,3,4            690,969,785                       1.60              23 <br />   a <br />       Three independent cultures assayed described materials methods. WT, wild type. <br />  560                                            G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br />   Figure 4.&#226;&#8364;&#8221;Effects Chp1, Chp2, Swi6, Clr4 expression ura4&#226;&#171;&#185; mating-type region. (A) Effects near <br /> mat2-P. Cells ura4&#226;&#171;&#185; XbaI site centromere-distal mat2-P (Xb) spotted indicated media Figure 2. <br /> (Top) Strains intact mat2-P silencing elements. (Bottom) Strains mat2-P centromere-proximal deletion &#226;&#181;&#8216;1.5 kb <br /> between Bgl II (Bg) BssHII (Bs) site. mat2-P(XbaI)::ura4 strains: &#226;&#171;&#185;, SP1124; chp1&#226;&#338;&#172;, PG1605; chp2&#226;&#338;&#172;, PG1790; swi6-115, <br /> SP1126; clr4-681, PG1032; &#226;&#338;&#172;(B-B)mat2-P(XbaI)::ura4 strains: &#226;&#171;&#185;, SP1151; chp1&#226;&#338;&#172;, PG1639; chp2&#226;&#338;&#172;, PG1787; swi6-115, SP1640; clr4- <br /> 681, SP1165. (B) Effects near mat3-M. Cells ura4&#226;&#171;&#185; gene EcoRV site centromere-distal mat3-M spotted as <br /> in Figure 2. &#226;&#171;&#185;, PG9; chp1&#226;&#338;&#172;, PG1454; chp2&#226;&#338;&#172;, PG1783; swi6-115, PG596; clr4-681, PG912. <br />                                              Fission Yeast Chromo-domain Proteins                                             561 <br />  <br />  <br />  <br />  <br />   Figure 5.&#226;&#8364;&#8221;Effects Chp1, Chp2, Swi6, Clr4 expression mat2-P mat3-M. (A) Effects mat2-P. Sporulated <br /> colonies stable mat1-M strains grown MSA plates supplemented leucine, uracil, adenine exposed iodine <br /> vapors photographed. Strains row intact mat2-P cis-acting elements. Strains row 1.5-kb <br /> deletion depicted Figure 4A. mat2-P strains: &#226;&#171;&#185;, SP1124; chp1&#226;&#338;&#172;, PG1605; chp2&#226;&#338;&#172;, PG1790; swi6-115, SP1126; clr4-681, PG1032; <br /> &#226;&#338;&#172;(B-B)mat2-P strains: &#226;&#171;&#185;, SP1151; chp1&#226;&#338;&#172;, PG1639; chp2&#226;&#338;&#172;, PG1787; swi6-115; PG1640; clr4-681, SP1165. (B) Effects mat3-M. <br /> Sporulated colonies stable mat1-P strains photographed A. Strains row intact mat3-M cis-acting <br /> elements. Strains row 482-bp deletion depicted Figure 4B. mat3-M strains: &#226;&#171;&#185;, PG445; chp1&#226;&#338;&#172;, PG1796; <br /> chp2&#226;&#338;&#172;, PG1799; swi6&#226;&#338;&#172;, PG1797; clr4-681, PG1594; &#226;&#338;&#172;(482)mat3-M strains: &#226;&#171;&#185;, PG1402; chp1&#226;&#338;&#172;, PG1779; chp2&#226;&#338;&#172;, PG1778; swi6&#226;&#338;&#172;, <br /> PG1781; clr4-681, PG1419. <br />  <br />  <br /> mat1 allele usually form colonies contain              proximal silencing element. A qualitatively similar effect <br /> spores. Under particular circumstances,               observed swi6 clr4 mutant strains (Thon et al. <br /> both mating types coexpressed single cells, hap-            1994, 1999; Figure 5), following deletion chp1 <br /> loid cells sporulate mating process               (Figure 5). Hence, Chp2, Chp1, appears play <br /> referred haploid meiosis. Therefore, sporulation             role similar chromo-domain proteins Swi6 <br /> in unswitchable mat1-M cells used monitor                Clr4 silencing transcription mating-type <br /> mat2-P expression sporulation unswitchable                  region. <br /> mat1-P cells used monitor mat3-M expression.                We constructed mat1-Msmt-0 strains containing all <br /> We examined various strains presence haploid            pairwise combinations chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, <br /> sporulation microscopic examination (data                   swi6-115, clr4-681 alleles. These strains used <br /> shown) exposing colonies iodine vapors,                  monitor levels expression mat2-P, revealed <br /> which selectively stains spores black (Bresch et al. 1968;         haploid meiosis growth limiting nitrogen. <br /> Figure 5). According assay, deletion            The strains examined iodine staining micro- <br /> chp1 chp2 significantly derepressed silent cas-            scopic examination. None combinations led to <br /> settes. Deletion chp2, however, synergistic effect        easily observable derepression mat2-P (data not <br /> with deletion mat2-P mat3-M centromere-           shown). This consistent previous conclusion <br />  562                                             G. Thon J. Verhein-Hansen <br />  <br /> that Chp1 does affect transcriptional silencing             LEU2 gene plasmid DNA integrated the <br /> the mating-type region indicates Chp2, Swi6,              rDNA Toda et al. (1984) purpose mapping <br /> and Clr4 act single pathway.                                  rRNA genes. We observed LEU2 marker <br />    Effects mating-type switching: Switching              expressed poorly strain an <br /> opposite mating type regulated extremely effi-            insertion, YIp2.4-1. When replicated complete me- <br /> cient process homothallic strains mat1 mat2-P            dium medium lacking leucine, colonies the <br /> mat3-M mating-type region, designated h90. Swi6           YIp2.4 insertion did grow evenly (Figure 6A). <br /> Clr4 required process (Egel et al. 1984;              Growth limited sectors colonies, giving <br /> Gutz Schmidt 1985; Thon Klar 1993; G.                      rise wheel-like patterns. This phenotype sup- <br /> Thon A. J. S. Klar, unpublished results; Ivanova               pressed deleting chp1, chp2, swi6, mutation <br /> et al. 1998). More specifically, Swi6 Clr4             clr4 (Figure 6A; data shown). <br /> be important choice silent cassette               We introduced S. pombe ura4&#226;&#171;&#185; gene near LEU2 <br /> converts mat1 mutations swi6 clr4             gene YIp2.4 (Figure 6B). The integration ob- <br /> reduce switching h90 cells, increase switching           tained swi6-115 background tested Southern <br /> cells engineered mat1 mat2-M mat3-P mating-                blots (data shown). As expected, integrated <br /> type region (h09 cells; Thon Klar 1993). Iodine                ura4&#226;&#171;&#185; gene displayed tight linkage LEU2 tetrad <br /> staining microscopic examination type                dissections (0 recombinants/45 viable spores exam- <br /> asci produced colony allow estimating             ined). We compared expression expression <br /> efficiency mating-type switching. High switching rates          ura4&#226;&#171;&#185; integrated random site (Allshire et al. <br /> lead homogenous mixtures P M cells                1994; Figure 6, C D) expression of <br /> colonies high efficiency zygotic ascus           truncated ura4-DS/E allele. <br /> formation dark iodine staining phenotype. On                 In background known silencing defect, ex- <br /> the hand, low switching rates lead reduced              pression ura4&#226;&#171;&#185; rDNA supported limited <br /> frequency zygotic asci speckled iodine staining          growth medium lacking uracil (Figure 6C). Many <br /> phenotypes. Because resemblance Chp2                cells form colonies FOA-containing plates (Fig- <br /> and Swi6, investigated deleting chp2                ure 6C). We tested FOA resistance was <br /> gene affected mating-type switching h90 h09 cells,           genetic alteration ura4&#226;&#171;&#185; gene an <br /> in different backgrounds known affect switching,            epigenetic phenomenon isolating FOA-resistant colo- <br /> swi6-mod&#226;&#171;&#185; swi6-mod&#226;&#171;&#186; (Thon Klar 1993). We                   nies 24 independent cultures determining <br /> found iodine staining sporulation pheno-              rise Ura&#226;&#171;&#185; colonies after <br /> types wild-type chp2-deleted strains similar           growth nonselective medium. Out 24 independent <br /> (data shown), indication mating-type switch-           FOA-resistant cultures, 17 displayed phenotype undis- <br /> ing occur efficiently absence Chp2. Mating-          tinguishable parental Ura&#226;&#171;&#185; strain re- <br /> type switching occurs normally absence              plating selective media (Figure 6C, row 1; data not <br /> Chp1 judged iodine staining phenotype                 shown). <br /> chp1-deleted h90 colonies (Doe et al. 1998; data                  We chromo-domain proteins <br /> shown).                                                            Chp1, Chp2, Swi6, Clr4 participated repres- <br />    Effect swi6, clr4, chp1, chp2 transcription       sion LEU2 ura4&#226;&#171;&#185; integrated rDNA (Figure <br /> of PolII genes rDNA repeats: Genes 25S,             6). Altering genes encoding proteins dele- <br /> 18S, 5.8S ribosomal RNAs present 70&#226;&#8364;&#8220;100                 tion mutation improved growth media lacking <br /> copies genome S. pombe (Hoheisel et al. 1993).           leucine uracil (Figure 6C) led accumulation <br /> Most genes organized large clusters            ura4&#226;&#171;&#185; transcripts (Figure 6D). Chp2, Swi6, Clr4 <br /> located ends chromosome 3 consisting                obvious effects Chp1 smaller but <br /> in tandem repeats 10.4-kb <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (Schaak et al. 1982;           consistently reproducible effect. The ura4&#226;&#171;&#185; transcript <br /> Toda et al. 1984; Hoheisel et al. 1993). The S. cerevisiae         originating rDNA size that <br />  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;Position effects rDNA repeats. (A) Expression S. cerevisiae LEU2 gene chp1&#226;&#171;&#185; chp1&#226;&#171;&#186; cells assayed <br /> on replica. Colonies chp1&#226;&#171;&#185; (top; YIp2.4-1) chp1-deleted (bottom; PG1572) cells integration LEU2 rDNA <br /> were obtained complete AA medium replicated medium lacking (AA-leu) containing (AA) leucine indicated. <br /> (B) Integration ura4&#226;&#171;&#185; rDNA. The S. pombe ura4&#226;&#171;&#185; gene contained pUC8 vector allowed recombine by <br /> homology YIp2-4 allele (Toda et al. 1984). The locations BamHI (Bam), Bsr FI (Bs), HindIII (Hin) restriction <br /> sites indicated. (C) Expression LEU2 ura4&#226;&#171;&#185; assayed plating efficiency. Cells containing LEU2 ura4&#226;&#171;&#185; gene in <br /> the rDNA indicated mutations spotted selective plates Figure 2. 1, PG1734; 2, PG1792; 3, PG1794; 4, <br /> PG1661; 5, PG1736. (D) Northern blot analysis. RNA prepared strains displayed C probed antisense <br /> ura4 probe: 1&#226;&#8364;&#8220;5, C; 6, FY489; 7, FY340. (E) Southern blot analysis. DNA prepared cultures used D, digested <br /> with HindIII, blotted, probed D. <br />  Fission Yeast Chromo-domain Proteins   563 <br />  564                                           G. Thon J. Verhein-Hansen <br />  <br /> from wild-type ura4&#226;&#171;&#185; gene, indicating initiation      In contrast effects transcription, swi6 and <br /> and termination transcription occurred normally. We         chp2 affected mating-type switching differently. Swi6 is <br /> determined Southern blot analysis increased        important choice silent cassette used to <br /> expression ura4&#226;&#171;&#185; caused amplification         convert specific mat1 alleles, lack Swi6 causes <br /> the ura4&#226;&#171;&#185; gene (Figure 6E). We tested stability         characteristic switching defects switching re- <br /> the Ura&#226;&#171;&#186; phenotype chp1&#226;&#338;&#172;, chp2&#226;&#338;&#172;, swi6-115, clr4-         duced strains wild-type mating-type region but <br /> 681 colonies formed FOA. The 24 independent FOA-            increased cells swapped silent cassettes (Egel et <br /> resistant derivatives tested chp2&#226;&#338;&#172;, swi6-115,     al. 1984; Gutz Schmidt 1985; Thon Klar <br /> clr4-681 frequency reversion Ura&#226;&#171;&#185; lower       1993). chp2-deleted cells able form colonies with <br /> 1 106 cells 9 tested Southern blot proved         sporulation phenotype similar wild type, indicat- <br /> have lost ura4&#226;&#171;&#185; gene. Among 24 independent             ing that, unlike Swi6, Chp2 absolutely required <br /> FOA-resistant isolates chp1-deleted cells, 13         efficient mating-type switching. One explanation for <br /> frequency reversion Ura4&#226;&#171;&#185; lower 1 106        different effects Swi6 Chp2 reside <br /> 11 return Ura&#226;&#171;&#185; efficiently. Hence,           degree protein contributes the <br /> FOA resistance chp1-deleted cells result       structure mating-type region. Lack Chp2 might <br /> from mutations loss ura4&#226;&#171;&#185; gene (in           deleterious effect lack Swi6. Indeed, <br /> case, loss tested Southern blot),           derepression transcription caused deleting <br /> reversible silencing ura4&#226;&#171;&#185; gene. This phenotype         chp2 marked caused deleting swi6 <br /> is similar phenotype chp&#226;&#171;&#185; cells different        (Figures 4 5). Hence, organization remaining in <br /> from phenotype chp2&#226;&#338;&#172;, swi6-115, clr4-681 cells,      chp2-deleted cells sufficient support efficient <br /> where reversible silencing ura4&#226;&#171;&#185; ob-           mating-type switching, albeit sufficient complete <br /> served. We conclude experiment epi-         transcriptional silencing. Another possibility Swi6 <br /> genetic repression ura4&#226;&#171;&#185; rDNA dependent           interacts protein required switching whereas <br /> to small extent Chp1 significantly            Chp2 does not. Other differences swi6- and <br /> Chp2, Swi6, Clr4.                                          chp2-deleted cells seen mitotic mei- <br />                                                                otic viability mutant strains. Cultures swi6 mutant <br />                                                                cells contain higher proportions dead cells do <br />                       DISCUSSION <br />                                                                wild-type cultures produce increased number of <br />    We examined contribution chromo-           three-spored asci. chp2-deleted strains healthy <br /> domain proteins, Chp1, Chp2, Swi6, Clr4, posi-          wild type respects (data shown). <br /> tion effects transcription genome fission         Consistent phenotypes, mitotic loss mini- <br /> yeast. We determined PolII-transcribed genes         chromosome increased lack Chp2 whereas <br /> can repressed placed rDNA cluster lo-           absence Swi6 (Table 2). <br /> cated right arm chromosome 3,              Effects chromosome segregation&#226;&#8364;&#8221;Epistasis analy- <br /> repression transcription dependent pres-          sis: Chromo-domain proteins association <br /> ence chromo-domain proteins. We             pericentric heterochromatin eukary- <br /> also shown Chp1 crucial centromeric silencing,      otes. In addition repressing transcription, possi- <br /> but transcriptional silencing mating-type        bly participate formation structures important <br /> region near telomeres. Conversely, major effects        chromosome function, particular kineto- <br /> of Chp2, addition observed rDNA,            chore. Consistent function, S. pombe <br /> were mating-type region near telomeres. We          chromo-domain proteins Swi6, Clr4, Chp1 re- <br /> will compare phenotypes reported          quired efficient chromosome segregation (Allshire <br /> mutations swi6 clr4 discuss light       et al. 1995; Ekwall et al. 1996; Bernard et al. 1998; Doe <br /> of reported subcellular localization pro-         et al. 1998). Swi6 present centromeres (Ekwall et <br /> teins.                                                         al. 1995) proteins known required <br />    Chp2 vs. Swi6: chp2 swi6 encode closely        localization, Rik1 Clr4 (Ekwall et al. 1996). <br /> related known S. pombe chromo-domain proteins.          The mutant allele clr4 used study, clr4-681, did <br /> Both ORFs size contain          increase rate chromosome loss same <br /> a chromo- chromo-shadow domain, structure simi-          extent previously tested allele clr4-S5 (Table 2; <br /> lar HP1 protein flies mammals. We          Allshire et al. 1995; Ekwall et al. 1996). The clr4-681 <br /> that chp2 swi6 overlapping distinguishable         allele contains <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>point mutation</span> causing amino-acid <br /> functions. Altering gene alleviated centromeric         change Clr4 SET domain (G486D mutation; Iva- <br /> and telomeric silencing (Allshire et al. 1995;            nova et al. 1998). It interesting determine <br /> study). It allowed transcription mating-type       allele clr4 mutations affecting the <br /> region derepression observed near mat2-P            SET domain protein influence localization <br /> mat3-M cases increased simultaneous         Swi6 extent clr4-S5 allele. The clr4- <br /> deletion cis-acting elements located near cassettes.    681 swi6-115 allele synergistic effect a <br />                                             Fission Yeast Chromo-domain Proteins                                      565 <br />  <br /> deletion chp1 (Table 2). Hence, Swi6 Clr4          somal regions close centromeres (Nakaseko et al. <br /> to act pathway different Chp1          1986; Chikashige et al. 1989). This inhibition possibly <br /> participates. The observed synergy reveals       reflects existence intramolecular secondary <br /> clr4-681 allele does reduce efficiency chromosome       structure compete pairing ho- <br /> segregation effect readily          mologs, steric hindrance the <br /> seen wild-type background. Consistent            kinetochore. We discovered significant fraction <br /> epistasis analysis, Chp1 localizes discrete    cells containing centromeric ura4&#226;&#171;&#185; insertion could <br /> spots nucleus, coincide centro-        lose ura4&#226;&#171;&#185; gene mitotic growth. One likely <br /> meres, manner does depend Rik1             mechanism leading loss ura4&#226;&#171;&#185; placed in <br /> Clr4 (Doe et al. 1998), centromeric localiza-      centromeric repeat intramolecular gene conver- <br /> tion Swi6 depends Rik1 Clr4. As expected,            sion involving untouched repeat placed symmetri- <br /> swi6-115 epistatic clr4-681 (Table 2). Deleting         cally containing ura4&#226;&#171;&#185;. Such interactions have <br /> chp2 increased chromosome loss threefold com-             proposed responsible high de- <br /> pared wild type, relatively modest effect (Ta-      gree sequence conservation centromeric re- <br /> ble 2). Further experiments required assess         peats presence identical nucleotide <br /> its role chromosome segregation.                            changes central repeats occupying symmetrical posi- <br />    Chp1 target recognition: In contrast pleiotropic     tions relative centromere 1 strains (Taka- <br /> effects Chp2, Chp1 displayed specificity centro-      hashi et al. 1992). ura4&#226;&#171;&#185; lost an <br /> meric regions. A DNA segment homologous centro-           insertion site located central core chromo- <br /> meric repeat mating-type region        1. The central core unique sequence and <br /> mat2-P mat3-M (Grewal Klar 1997). In              expected target mitotic gene <br /> area, sequence similarity centromere 2 reaches        conversion. We determined cen- <br /> 96% 4.3 kb DNA. Deletions mating-type           tral core altered cells having lost ura4&#226;&#171;&#185; the <br /> region include 4.3-kb repeat lead partial        cnt1 integration site. <br /> derepression transcription, &#226;&#8364;&#339;on&#226;&#8364;? &#226;&#8364;&#339;off&#226;&#8364;?               Transcriptional silencing rDNA: Transcrip- <br /> states expression clonally inherited, defects    tional silencing cases clear function its <br /> in mating-type switching, covariegate           own. For example, silencing mat2-P mat3-M <br /> defects transcription (Grewal Klar 1996; Thon           mating-type cassettes crucial yeast sexual cycle <br /> and Friis 1997). The ura4&#226;&#171;&#185; gene silenced intro-        yeast <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span> absence nitrogen source. <br /> duced repeat mating-type region              In cases, transcriptional silencing con- <br /> (Grewal Klar 1997) homologous                sequence phenomena, structural local- <br /> repeat near centromere 1 (Allshire et al. 1995).               ization requirements. The role transcriptional silenc- <br /> Whereas Swi6 Clr4 proteins required            ing rDNA subject speculation. <br /> the repression locations (Allshire et al. 1995;           Ty1 elements prototrophic markers introduced <br /> Grewal Klar 1997), showed Chp1                rDNA S. cerevisiae poorly transcribed (Bryk <br /> was required silencing centromeric insertions           et al. 1997; Fritze et al. 1997; Smith Boeke 1997). <br /> ura4&#226;&#171;&#185;, silencing insertions ura4&#226;&#171;&#185;        Their transcription increased disruption the <br /> mating-type region, mating-type genes. The lack        silencing gene SIR2, ubiquitin-conjugating en- <br /> of effect chp1 deletion gene expression         zyme gene UBC2, reduced levels histone H2A and <br /> mating-type region simply explained re-         H2B, mutations large number genes <br /> dundancy silencing mechanism operating          including genes involved DNA replication regula- <br /> region, occasionally masks silencing defects,      tion chromatin structure (Bryk et al. 1997; Smith <br /> no cumulative effect observed following combina-           Boeke 1997; Smith et al. 1999). Repression PolII <br /> tion chp1 deletion silencing deficien-       transcription rDNA repeats by-product <br /> cies (Figure 5; data shown). The simplest interpreta-      PolI regulation (Smith Boeke 1997), the <br /> tion results Chp1 does act           rDNA control mitotic meiotic recombination <br /> mating-type region, rule           (Gottlieb Esposito 1989 references therein), <br /> it localized action locations       and/or life span regulation (Kennedy et al. 1997). <br /> not tested. The restriction repressive effects       It reflect functions nucleolus distinct from <br /> centromeres suggests tethered pro-       production rRNA linked constituting <br /> tein/DNA complex associated           discrete nuclear subcompartment. Such function was <br /> 4.3-kb sequence. Chp1 exert effect            proposed recently RENT complex, nucleolar <br /> the 4.3-kb repeat distance, possibly pro-       protein complex associated Sir2, sequesters <br /> teins Swi6 Clr4, spread          Cdc14 nucleolus cell cycle <br /> the DNA oligomerization.                                    releases telophase (Shou et al. 1999; Straight <br />    Loss ura4&#226;&#180;&#8482; gene centromeric regions:            et al. 1999). <br /> Meiotic recombination inefficient S. pombe chromo-          The transcriptional repression observed the <br />  566                                           G. Thon J. Verhein-Hansen <br />  <br /> rDNA S. pombe reminiscent S. cerevisiae.      Nurse 1996) clr4&#226;&#171;&#186; cells, Swi6 does associ- <br /> At S. pombe ORF encodes protein related         ate centromeres, nucleolus <br /> S. cerevisiae Sir2 (CAB38511 cosmid SPBC16D10)          (Ekwall et al. 1996). One conceive that <br /> it interesting investigate participation     nucleolar sites normally occupied Clr4 recog- <br /> rDNA silencing. The chromo-domain proteins                nized Swi6, absence Clr4 titrated <br /> whose roles assayed homologs              away centromeres nucleolus. In wild- <br /> S. cerevisiae. S. cerevisiae chromo-domain        type cells, Clr4 repress transcripion PolII genes <br /> proteins: member CHD family (Woodage et al.           rDNA. In clr4 mutant cells, Swi6 mislocalize <br /> 1997; AC P32657) S. pombe closest relatives          sites normally occupied Clr4, fail to <br /> Hrp1 (Jin et al. 1998) uncharacterized ORF (AC          exert repression Clr4. In swi6 mutant cells, <br /> O14139) Esa1 (Hilfiker et al. 1996; Clarke et al.          Clr4 titrated sites normally occupied by <br /> 1999). We S. pombe proteins         Swi6 fail repress transcription rDNA. <br /> the chromo-domain proteins described impor-           Other indirect effects alterations clr4 also <br /> tant transcriptional repression rDNA, in-       possible clr4 controls transcription several <br /> cluding histone deacetylase Clr3 (data shown).         genes, positively negatively (Ivanova et al. <br /> As S. cerevisiae, speculate repression    1998). In type model, imagine that <br /> of transcription growth rate regulation,       changes centromere telomere structure sterically <br /> S. pombe PolI template preferences different          affect nucleolus, possibly facilitating entrance of <br /> those PolII, nucleolus constitutes com-       PolII. Additional localization studies overexpres- <br /> partment special properties, suggested          sion experiments ought help distinguish between <br /> ultrastructure (Leger-Sylvestre et al. 1997). A regula-        models. <br /> tory function dynamic sequestering proteins             We thank Olaf Nielsen gift plasmids, Robin Allshire for <br /> involved mitosis envisioned S.          strains centromeric telomeric insertions ura4&#226;&#171;&#185; gene, <br /> pombe nucleolus. Clr4, protein important chromo-         Kohta Takahashi Mitsuhiro Yanagida strain containing <br /> some segregation rDNA silencing, localized           YIp2.4 allele. We thank Inga Sig Nielsen Stanley Brown <br />                                                                comments manuscript. The reported experiments <br /> the nucleolus interphase expressed plas-        supported grants Novo Nordisk Foundation and <br /> mid green fluorescent protein (GFP)-fusion pro-           Danish Natural Science Research Council. <br /> tein. During mitosis, fraction Clr4 pool relocal- <br /> izes mitotic spindle (S26 Sawin Nurse <br /> 1996).                                                                              LITERATURE CITED <br />    Direct indirect effects&#226;&#8364;&#8221;Localization: rDNA in- <br />                                                                Aasland, R., A. F. Stewart, 1995 The chromo shadow domain, <br /> sertions LEU2 ura4&#226;&#171;&#185; gene transcribed               second chromo domain heterochromatin-binding protein <br /> more efficiently swi6-115 wild-type cells. The          1, HP1. Nucleic Acids Res. 23: 3163&#226;&#8364;&#8220;3173. <br /> swi6-115 mutation substitutes arginine tryp-        Allshire, R., 1996 Transcriptional silencing fission yeast: a <br />                                                                    manifestation higher order chromosome structure func- <br /> tophan position 269, residue conserved                tions, pp. 443&#226;&#8364;&#8220;466 Epigenetic Mechanisms Gene Regulation, <br /> chromo-shadow domains (Allshire 1996). The swi6-                   edited V. E. A. Russo, R. A. Martienssen A. D. Riggs. <br /> 115 mutation results undetectable levels            Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. <br />                                                                Allshire, R., G. Cranston, J. R. Gosden, J. C. Maule, N. D. Hastie <br /> Swi6 protein (Ekwall et al. 1996) proba-             et al., 1987 A fission yeast chromosome replicate autono- <br /> bly constitutes loss function. Hence, observation         mously mouse cells. Cell 50: 391&#226;&#8364;&#8220;403. <br /> indicates wild-type Swi6 protein represses PolII      Allshire, R., J. P. Javerzat, N. J. Redhead G. Cranston, 1994 <br />                                                                    Position effect variegation fission yeast centromeres. Cell 76: <br /> transcription rDNA. The interpretation              157&#226;&#8364;&#8220;169. <br /> repression straightforward Swi6          Allshire, R., E. R. Nimmo, K. Ekwall, J. P. Javerzat G. Cran- <br /> localized nucleolus wild-type cells (Ekwall          ston, 1995 Mutations derepressing silent centromeric domains <br />                                                                    fission yeast disrupt chromosome segregation. Genes Dev. 9: <br /> et al. 1995, 1996). One possible explanation Swi6          218&#226;&#8364;&#8220;233. <br /> recognizes silences PolII-transcribed genes           Arcangioli, B., A. J. S. Klar, 1991 A novel switch-activating <br /> placed context rDNA. In model, local-           site (SAS1) cognate binding factor (SAP1) required for <br />                                                                    efficient mat1 switching S. pombe. EMBO J. 10: 3025&#226;&#8364;&#8220;3032. <br /> ization Swi6 nucleolus precede intro-      Beach, D. H., 1983 Cell type switching DNA transposition in <br /> duction marker gene                 fission yeast. Nature 305: 682&#226;&#8364;&#8220;688. <br /> have observed past experiments. Alternatively,         Bernard, P., K. Hardwick J. P. Javerzat, 1998 Fission yeast <br />                                                                    Bub1 mitotic centromere protein essential spindle <br /> the influence swi6-115 mutation transcription            checkpoint preservation correct ploidy mito- <br /> in rDNA indirect. In absence Swi6            sis. J. Cell Biol. 143: 1775&#226;&#8364;&#8220;1787. <br /> factor normally present rDNA able repress        Bresch, C., G. Muller R. Egel, 1968 Genes involved meiosis <br />                                                                    sporulation yeast. Mol. Gen. Genet. 102: 301&#226;&#8364;&#8220;306. <br /> PolII transcription synthesized,         Bryk, M., M. Banerjee, M. Murphy, K. E. Knudsen, D. J. Garfinkel <br /> be titrated away sites normally occupied Swi6,          et al., 1997 Transcriptional silencing Ty1 elements the <br /> as centromeres. Titration effects account            RDN1 locus yeast. Genes Dev. 11: 255&#226;&#8364;&#8220;269. <br />                                                                Cavalli, G., R. Paro, 1998 Chromo-domain proteins: linking <br /> other observed phenotypes. As mentioned above, GFP-              chromatin structure epigenetic regulation. Curr. Opin. Cell <br /> Clr4 fusion protein localizes nucleolus (Sawin              Biol. 10: 354&#226;&#8364;&#8220;360. <br />                                                      Fission Yeast Chromo-domain Proteins                                                         567 <br /> Chikashige, Y., N. Kinoshita, Y. Nakaseko, T. Matsumoto, S. Mur-                MOZ human genes SAS gene yeast, required <br />     akami et al., 1989 Composite motifs repeat symmetry S.               dosage compensation Drosophila. EMBO J. 16: 2054&#226;&#8364;&#8220;2060. <br />     pombe centromeres: direct analysis integration NotI restric-     Hoheisel, J. D., E. Maier, R. Mott, L. McCarthy, A. V. Grigoriev <br />     tion sites. Cell 57: 739&#226;&#8364;&#8220;751.                                               et al., 1993 High resolution cosmid P1 maps spanning the <br /> Clarke, A. S., J. A. Lowell, S. J. Jacobson L. Pillus, 1999 Esa1p           14 Mb genome fission yeast S. pombe. Cell 73: 109&#226;&#8364;&#8220;120. <br />     essential histone acetyltransferase required cell cycle      Ivanova, A. I., M. J. Bonaduce, S. V. Ivanov A. J. S. Klar, 1998 <br />     progression. Mol. Cell. Biol. 19: 2515&#226;&#8364;&#8220;2526.                                The chromo SET domains Clr4 protein essential <br /> Doe, C. L., G. Wang, C. Chow, M. D. Fricker, P. B. Singh et al.,                silencing fission yeast. Nat. Genet. 19: 192&#226;&#8364;&#8220;194. <br />     1998 The fission yeast chromo domain encoding gene chp1&#226;&#171;&#185;               Jin, Y. H., E. J. Yoo, Y. K. Jang, S. H. Kim, M. J. Kim et al., 1998 Isola- <br />     required chromosome segregation shows genetic                  tion characterization hrp1&#226;&#171;&#185;, new member SNF2/ <br />     interaction alpha-tubulin. Nucleic Acids Res. 26: 4222&#226;&#8364;&#8220;4229.           SWI2 gene family fission yeast Schizosaccharomyces <br /> Egel, R., H. Gutz, 1981 Gene activation copy transposition               pombe. Mol. Gen. Genet. 257: 319&#226;&#8364;&#8220;329. <br />     mating-type switching homothallic fission yeast. Curr.         Jones, J. S., L. Prakash, 1990 Yeast Saccharomyces cerevisiae se- <br />     Genet. 3: 5&#226;&#8364;&#8220;12.                                                             lectable markers pUC18 polylinkers. Yeast 6: 363&#226;&#8364;&#8220;366. <br /> Egel, R., D. H. Beach A. J. S. Klar, 1984 Genes required           Kennedy, B. K., M. Gotta, D. A. Sinclair, K. Mills, D. S. McNabb <br />     initiation resolution steps mating-type switching fission         et al., 1997 Redistribution silencing proteins telomeres <br />     yeast. Proc. Natl. Acad. Sci. USA 81: 3481&#226;&#8364;&#8220;3485.                            nucleolus associated extension life span S. <br /> Egel, R., M. Willer O. Nielsen, 1989 Unblocking meiotic                  cerevisiae. Cell 89: 381&#226;&#8364;&#8220;391. <br />     crossing-over silent mating-type cassettes fission      Klar, A. J. S., M. J. Bonaduce, 1991 swi6, gene required for <br />     yeast, conditioned recessive, pleiotropic mutant rik1. Curr.         mating-type switching, prohibits meiotic recombination the <br />     Genet. 15: 407&#226;&#8364;&#8220;410.                                                         mat2-mat3 &#226;&#8364;&#339;cold spot&#226;&#8364;? fission yeast. Genetics 129: 1033&#226;&#8364;&#8220;1042. <br /> Egel, R., M. Willer, S. Kj&#195;&#166;ligrulff, J. Davey O. Nielsen, 1994         Koonin, E. V., S. Zhou J. C. Lucchesi, 1995 The chromo <br />     Assessment pheromone production response fission                  superfamily: new members, duplication chromo domain <br />     yeast halo test induced sporulation. Yeast 10: 1347&#226;&#8364;&#8220;1354.           possible role delivering transcription regulators chroma- <br /> Ekwall, K., T. Ruusala, 1994 Mutations rik1, clr2, clr3              tin. Nucleic Acids Res. 23: 4229&#226;&#8364;&#8220;4233. <br />     clr4 asymmetrically derepress silent mating-type loci fission   Leger-Sylvestre, I., J. Noaillac-Depeyre, M. Faubladier N. <br />     yeast. Genetics 136: 53&#226;&#8364;&#8220;64.                                                 Gas, 1997 Structural functional analysis nucleolus <br /> Ekwall K., J. P. Javerzat, A. Lorentz, H. Schmidt, G. Cranston                  fission yeast Schizosaccharomyces pombe. Eur. J. Cell Biol. 72: <br />     et al., 1995 The chromodomain protein Swi6: key component                 13&#226;&#8364;&#8220;23. <br />     fission yeast centromeres. Science 269: 1429&#226;&#8364;&#8220;1431.                  Lehrach, H., 1990 Hybridization fingerprinting genome map- <br /> Ekwall, K., E. R. Nimmo, J. P. Javerzat, B. Borgstrom, R. Egel et               ping sequencing, pp. 39&#226;&#8364;&#8220;81 Genome Analysis Volume 1: <br />     al., 1996 Mutations fission yeast silencing factors clr4&#226;&#171;&#185;            Genetic Physical Mapping, edited K. E. Davies S. M. <br />     rik1&#226;&#171;&#185; disrupt localisation chromo domain protein             Tilghman. Cold Spring Harbor Laboratory Press, Cold Spring <br />     Swi6p impair centromere function. J. Cell Sci. 109: 2637&#226;&#8364;&#8220;               Harbor, NY. <br />     2648.                                                                  Lorentz, A., L. Heim H. Schmidt, 1992 The switching gene <br /> Engelke, U., L. Grabowski, H. Gutz, L. Heim H. Schmidt, 1987                swi6 affects recombination gene expression mating- <br />     Molecular characterization h&#226;&#171;&#186; mutants Schizosaccharomyces             type region Schizosaccharomyces pombe. Mol. Gen. Genet. 233: <br />     pombe. Curr. Genet. 12: 535&#226;&#8364;&#8220;542.                                            436&#226;&#8364;&#8220;442. <br /> Fan, J.-B., Y. Chikashige, C. L. Smith, C. L. Niwa, M. Yanagida et         Lorentz, A., K. Osterman, O. Fleck H. Schmidt, 1994 Switch- <br />     al., 1988 Construction NotI restriction map fission             ing gene swi6, involved repression silent mating-type loci <br />     yeast Schizosaccharomyces pombe genome. Nucleic Acids Res. 17:              fission yeast, encodes homologue chromatin-associated <br />                                                                                 proteins Drosophila mammals. Gene 143: 139&#226;&#8364;&#8220;143. <br />     2801&#226;&#8364;&#8220;2818. <br />                                                                            Matsumoto, T., K. Fukui, O. Niwa, N. Sugawara, J. W. Szostak et <br /> Fritze, C. E., K. Verschueren, R. Strich R. E. Esposito, 1997 <br />                                                                                 al., 1987 Identification healed terminal DNA fragments in <br />     Direct evidence SIR2 modulation chromatin structure in <br />                                                                                 linear minichromosomes Schizosaccharomyces pombe. Mol. Cell. <br />     yeast rDNA. EMBO J. 16: 6495&#226;&#8364;&#8220;6509. <br />                                                                                 Biol. 7: 4424&#226;&#8364;&#8220;4430. <br /> Gottlieb, S., R. E. Esposito, 1989 A new role yeast <br />                                                                            Messmer, S., A. Franke R. Paro, 1992 Analysis functional <br />     transcriptional silencer gene, SIR2, regulation recombina- <br />                                                                                 role Polycomb chromo-domain Drosophila melanogaster. <br />     tion ribosomal DNA. Cell 56: 771&#226;&#8364;&#8220;776. <br />                                                                                 Genes Dev. 6: 1241&#226;&#8364;&#8220;1254. <br /> Grewal, S. I. S., A. J. S. Klar, 1996 Chromosomal inheritance <br />                                                                            Mizukami, T., W. I. Chang, I. Garkavtsev, N. Kaplan, D. Lombardi <br />     epigenetic states fission yeast mitosis meiosis. <br />                                                                                 et al., 1993 A 13 kb resolution cosmid map 14 Mb fission <br />     Cell 86: 95&#226;&#8364;&#8220;101.                                                            yeast genome nonrandom sequence-tagged site mapping. Cell <br /> Grewal, S. I. S., A. J. S. Klar, 1997 A recombinationally re-               73: 121&#226;&#8364;&#8220;132. <br />     pressed region mat2 mat3 loci shares homology           Moreno, S., A. J. S. Klar P. Nurse, 1991 Molecular genetic <br />     centromeric repeats regulates directionality mating-type             analysis fission yeast Schizosaccharomyces pombe. Methods Enzy- <br />     switching fission yeast. Genetics 146: 1221&#226;&#8364;&#8220;1238.                        mol. 194: 795&#226;&#8364;&#8220;823. <br /> Grewal, S. I. S., M. J. Bonaduce A. J. S. Klar, 1998 Histone           Nakaseko, Y., Y. Adachi, S. Funahashi, O. Niwa M. Yanagida, <br />     deacetylase homologs regulate epigenetic inheritance tran-               1986 Chromosome walking shows highly homologous repeti- <br />     scriptional silencing chromosome segregation fission                 tive sequence present centromere regions fission <br />     yeast. Genetics 150: 563&#226;&#8364;&#8220;576.                                               yeast. EMBO J. 5: 1011&#226;&#8364;&#8220;1021. <br /> Grimm, C., J. Kohli, J. Murray K. Maundrell, 1988 Genetic              Nimmo, E. R., G. Cranston R. Allshire, 1994 Telomere-associ- <br />     engineering Schizosaccharomyces pombe : gene disrup-        ated chromosome breakage fission yeast results variegated <br />     tion replacement using ura4 gene selectable marker.            expression adjacent genes. EMBO J. 13: 3801&#226;&#8364;&#8220;3811. <br />     Mol. Gen. Genet. 215: 81&#226;&#8364;&#8220;86.                                           Niwa, O., T. Matsumoto, Y. Chikashige M. Yanagida, 1989 <br /> Gutz, H., 1963 Untersuchungen zur Feinstruktur der Gene ad7 und                 Characterization Schizosaccharomyces pombe minichromosome <br />     ad6 von Schizosaccharomyces pombe LIND. Habilitationsschrift, Tech-         deletion derivatives functional allocation centro- <br />     nische Universita&#194;&#168;t Berlin, Berlin.                                         mere. EMBO J. 8: 3045&#226;&#8364;&#8220;3052. <br /> Gutz, H., H. Schmidt, 1985 Switching genes Schizosaccharo-          Pak, D. T., M. Pflumm, I. Chesnokov, D. W. Huang, R. Kellum et <br />     myces pombe. Curr. Genet. 9: 325&#226;&#8364;&#8220;331.                                       al., 1997 Association origin recognition complex with <br /> Hanahan, D., 1983 Studies transformation Escherichia coli            heterochromatin HP1 higher eukaryotes. Cell 91: 311&#226;&#8364;&#8220;323. <br />     plasmids. J. Mol. Biol. 166: 557&#226;&#8364;&#8220;580.                                  Paro, R., D. S. Hogness, 1991 The Polycomb protein shares a <br /> Heyer, W. D., M. Sipiczki J. Kohli, 1986 Replicating plasmids               domain heterochromatin-associated protein Drosophila. <br />     Schizosaccharomyces pombe : improvement symmetric segrega-            Proc. Natl. Acad. Sci. USA 88: 263&#226;&#8364;&#8220;267. <br />     tion new genetic element. Mol. Cell. Biol. 6: 80&#226;&#8364;&#8220;89.              Platero, J. S., T. Hartnett J. C. Eissenberg, 1995 Functional <br /> Hilfiker, A., D. Hilfiker-Kleiner, A. Pannuti J. C. Lucchesi,               analysis chromo domain HP1. EMBO J. 14: 3977&#226;&#8364;&#8220;3986. <br />     1996 mof1, putative acetyltransferase gene related Tip60      Powers, J., J. C. Eissenberg, 1993 Overlapping domains of <br />  568                                                      G. Thon J. Verhein-Hansen <br />     heterochromatin associated protein HP1 mediates nuclear             Thon, G., T. Friis, 1997 Epigenetic inheritance transcrip- <br />     localization heterochromatin binding. J. Cell Biol. 120: 291&#226;&#8364;&#8220;            tional silencing switching competence fission yeast. Genet- <br />     299.                                                                         ics 145: 685&#226;&#8364;&#8220;696. <br /> Rose, M., F. Winston P. Hieter, 1990 Methods Yeast Genetics:         Thon, G., A. J. S. Klar, 1992 The clr1 locus regulates expres- <br />     A Laboratory Course Manual. Cold Spring Harbor Laboratory Press,             sion cryptic mating-type loci fission yeast. Genetics 131: <br />     Cold Spring Harbor, NY.                                                      287&#226;&#8364;&#8220;296. <br /> Saunders, W. S., C. Chue, M. Goebl, C. Craig, R. F. Clark et al.,           Thon, G., A. J. S. Klar, 1993 Directionality fission yeast <br />     1993 Molecular cloning human homolog Drosophila het-                 mating-type interconversion controlled location the <br />     erochromatin protein HP1 using anticentromere autoantobodies                 donor loci. Genetics 134: 1045&#226;&#8364;&#8220;1054. <br />     anti-chromo specificity. J. Cell Sci. 104: 573&#226;&#8364;&#8220;582.                Thon, G., A. Cohen A. J. S. Klar, 1994 Three additional linkage <br /> Sawin, K. E., P. Nurse, 1996 Identification fission yeast nu-             groups repress transcription meiotic recombination in <br />     clear markers using random polypeptide fusions green fluo-              mating-type region Schizosaccharomyces p <br /> </body></html>