<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The missing piece type II fatty acid synthase <br /> system Mycobacterium tuberculosis <br /> Emmanuelle Sacco*, Adrian Suarez Covarrubias&#226;&#8364;&#160;, Helen M. O&#226;&#8364;&#8482;Hare&#226;&#8364;&#161;, Paul Carroll&#194;&#167;, Nathalie Eynard*, T. Alwyn Jones&#226;&#8364;&#160;, <br /> Tanya Parish&#194;&#167;, Mamadou Daffe&#194;&#180;*, Kristina Ba&#194;&#168;ckbro&#226;&#8364;&#160;, Anna&#196;&#177;&#194;&#168;k Que&#194;&#180;mard*&#194;&#182; <br /> *De&#194;&#180;partement des Me&#194;&#180;canismes Mole&#194;&#180;culaires des Infections Mycobacte&#194;&#180;riennes, Institut Pharmacologie et Biologie Structurale, Centre National la <br /> Recherche Scientifique, 31077 Toulouse, France; &#226;&#8364;&#160;Department Cell Molecular Biology, Uppsala University, Biomedical Center, SE-751 24 Uppsala, <br /> Sweden; &#226;&#8364;&#161;Institute Chemical Sciences Engineering, Ecole Polytechnique Fe&#194;&#180;de&#194;&#180;rale Lausanne, CH-1015 Lausanne, Switzerland; &#194;&#167;Centre for <br /> Infectious Disease, Institute Cell Molecular Science Barts The London, London E1 2AT, United Kingdom <br />  <br /> Edited Christian R. H. Raetz, Duke University Medical Center, Durham, NC, approved July 20, 2007 (received review May 4, 2007) <br />  <br /> The Mycobacterium tuberculosis fatty acid synthase type II (FAS-II)             metabolic pathway represents valuable source potential <br /> system unique property producing unusually long-chain                new pharmacological targets (2). <br /> fatty acids involved biosynthesis mycolic acids, key molecules          Isoniazid inhibits elongation process leading for- <br /> of tubercle bacillus. The enzyme(s) responsible dehydration          mation main (meromycolic) chain MAs. The steps <br /> (3R)-hydroxyacyl-ACP elongation cycles mycobac-               elongation cycles monitored acyl carrier protein <br /> terial FAS-II remained unknown. This step classically catalyzed           (ACP)-dependent FA synthase type II (FAS-II) (5). <br /> FabZ- FabA-type enzymes bacteria, proteins               FAS-II systems plants, bacteria, parasites, and <br /> present mycobacteria. Bioinformatic analyses essentiality             mitochondria, usually perform novo biosynthesis <br /> study allowed identification candidate protein cluster,                (6). However, Corynebacterineae unique <br /> Rv0635-Rv0636-Rv0637. Its expression recombinant Escherichia coli            elongates long-chain FAs (C12&#226;&#8364;&#8220;C18) very-long- <br /> strains leads formation heterodimers, Rv0635-Rv0636               chain FAs (5). In mycobacteria, enzymes catalyzing the <br /> (HadAB) Rv0636-Rv0637 (HadBC), occurs Mycobac-                elongation steps characterized [supporting in- <br /> terium smegmatis, shown split-Trp assays. Both heterodimers               formation (SI) Fig. 7]: &#226;?&#164;-ketoacyl-ACP synthetases KasA <br /> exhibit enzymatic properties expected mycobacterial FAS-II              KasB (7, 8), &#226;?&#164;-ketoacyl-ACP reductase MabA (9), and <br /> dehydratases: marked specificity long-chain (&gt;C12)               trans-2-enoyl-ACP reductase InhA (10). However, en- <br /> ACP-linked substrates. Furthermore, function 3-hydroxyacyl              zyme(s) involved step cycle, corresponding <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> dehydratases coupled MabA InhA enzymes                   (3R)-hydroxyacyl-ACP dehydratase(s) (HAD), un- <br /> M. tuberculosis FAS-II system. HadAB HadBC long-sought              known. The peptidic domains proteins carrying function <br /> (3R)-hydroxyacyl-ACP dehydratases. The correlation                  multifunctional synthases FAS-II systems have <br /> substrate specificities enzymes, organization or-           generally difficult identify (11, 12). <br /> thologous gene cluster different Corynebacterineae, struc-             In FAS-II systems, HAD activity classically catalyzed by <br /> ture mycolic acids suggests distinct roles het-               FabZ (dehydratase) FabA (dehydratase&#226;&#8364;&#8220;isomerase) enzymes <br /> erodimers elongation process. This work describes                    (6). Yet previously shown FabZ/FabA-type <br /> bacterial monofunctional (3R)-hydroxyacyl-ACP dehydratases be-                  protein present tubercle bacillus (13). This indicated <br /> longing hydratase 2 family. Their original structure fact        HADs mycobacteria present different or <br /> that essential M. tuberculosis <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>survival</span> make en-             distantly related catalytic sequence motif. After investigation of <br /> zymes good candidates development antimycobacte-                Mtb genome using bioinformatic tools, pool 11 putative <br /> rial drugs.                                                                     (R)-specific enoyl hydratases/3-hydroxyacyl dehydratases has <br />                                                                                 identified (13, 14). Structural determination modeling of <br /> (3R)-hydroxyacyl-ACP dehydratase &#229;&#8230;&#169; hydratase 2 &#229;&#8230;&#169; mycolic acid biosynthesis &#229;&#8230;&#169;    (13&#226;&#8364;&#8220;15) suggested belong are <br /> fatty acid elongation &#229;&#8230;&#169; hot dog fold                                            related hydratase 2 protein family, underlying <br />                                                                                 3D structure FabA/Z-type enzymes maintained the <br />                                                                                 so-called hot dog fold catalytic site distinct (16). One <br /> T   uberculosis kills nearly million people year (i.e., <br />     person 20 seconds). Emergence drug-resistant <br /> Mycobacterium tuberculosis (Mtb) main concerns in <br />                                                                                 family members, Rv0636, proposed a <br />                                                                                 candidate FAS-II HAD basis essentiality <br /> the treatment disease; prevalence cases resistant             prediction (14). The purpose present work use <br /> four main drugs (isoniazid, rifampicin, streptomycin, etham-                complementary experimental strategies determine which, if <br /> butol) high regions world (up 30% for <br /> previously treated cases). Thus, development new drugs <br />                                                                                 Author contributions: E.S. A.S.C. contributed equally work; H.M.O., T.P., K.B., <br /> effective resistant strains priority (1).              A.Q. designed research; E.S., A.S.C., H.M.O., P.C., N.E., T.P., K.B., A.Q. performed <br />    Mycobacteria possess lipid-rich envelope,                 research; H.M.O. contributed new reagents/analytic tools; E.S., A.S.C., H.M.O., T.A.J., T.P., <br /> typically contains long-chain (C60&#226;&#8364;&#8220;C90) &#226;?&#163;-alkylated &#226;?&#164;-                     M.D., K.B., A.Q. analyzed data; T.A.J., T.P., M.D., A.Q. wrote paper. <br />  <br /> hydroxylated fatty acids (FAs) called mycolic acids (MAs) (2).                  The authors declare conflict interest. <br /> These original lipids universally suborder                  This article PNAS Direct Submission. <br /> Corynebacterineae (including Mycobacterium, Corynebacterium,                    Abbreviations: ACP, acyl carrier protein; FA, fatty acid; FAS, FA synthase; HAD, (3R)- <br /> Rhodoccocus, Nocardia, etc.) genus holds molecules                   hydroxyacyl-ACP dehydratase; H-HadA, His-tagged HadA; H-HadB, His-tagged HadB; MA, <br />                                                                                 mycolic acid; Mtb, M. tuberculosis. <br /> of specific chain lengths. MAs crucial architecture             &#194;&#182;To  correspondence addressed at: Institut Pharmacologie et de <br /> the permeability mycobacterial envelope. They play                   Biologie Structurale, Centre National la Recherche Scientifique, 205 Route Nar- <br /> a role virulence persistence tubercle bacillus             bonne, 31077 Toulouse Cedex, France. E-mail: annaik.quemard@ipbs.fr. <br /> within infected organisms (3, 4). Their biosynthetic pathway,                   This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> which essential survival mycobacteria, target              0704132104/DC1. <br /> of frontline antituberculous drug, isoniazid. Therefore,                 &#194;&#169; 2007 The National Academy Sciences USA <br />  <br />  <br />  <br /> 14628 &#226;&#8364;&#8220;14633 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 37                                                        www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104 <br />  Table 1. In silico analysis putative (R)-specific hydratases/dehydratases Mtb <br />                 Size, Hydratase 2                         Ubiquity                                                   Nocardia       Known predicted <br /> Protein         kDa     motif*           Proximity&#226;&#8364;&#160;        mycobacteria&#226;&#8364;&#161; Corynebacterium&#194;&#167; Rhodococcus sp.&#194;&#167;                 farcinica&#194;&#167;         structure&#194;&#182; <br />  <br /> Rv0130          16.0        &#226;&#171;&#185;               fbpC                 &#226;&#171;&#186;            NCgl0284,&#229;&#8218;&#168; 44%        Ro03023, 60%        Nfa34100, 58%              SHD <br /> Rv0216          35.8        &#226;&#171;&#186;                                    &#226;&#171;&#185;                                  Ro02873, 68%                                   DHD <br /> Rv0241c         30.2       &#226;&#171;&#185;&#226;&#171;&#185;            fabG4                   &#226;&#171;&#185;             JK0821,** 42%        Ro05198 , 51%       Nfa54620, 55%              DHD <br /> Rv0504c         18.4        &#226;&#171;&#185;            cmaA2                   &#226;&#171;&#185;                                                                                 SHD <br /> Rv0635 HadA     17.4        &#226;&#171;&#186;         mmaA1-mmaA4                &#226;&#171;&#185;                                   Ro01984, 53%       Nfa51180, 53%              SHD <br /> Rv0636 HadB     14.9       &#226;&#171;&#185;&#226;&#171;&#185;         mmaA1-mmaA4                &#226;&#171;&#185;                                   Ro01983, 63%       Nfa51180, 63%              SHD <br /> Rv0637 HadC     18.9        &#226;&#171;&#186;         mmaA1-mmaA4                &#226;&#171;&#185;                                                                                 SHD <br /> Rv2499c         20.2        &#226;&#171;&#186;                                    &#226;&#171;&#186;             JK1548,** 75%         Ro06098, 76%       Nfa50400, 78%              SHD <br /> Rv3389c         30.3       &#226;&#171;&#185;&#226;&#171;&#185;              cmaA1                 &#226;&#171;&#186;                                                                                 DHD <br /> Rv3538          30.2       &#226;&#171;&#185;&#226;&#171;&#185;                                    &#226;&#171;&#186;                                   Ro04531, 62%       Nfa4620, 59%               DHD <br /> Rv3542c         33.9       &#226;&#171;&#185;                                     &#226;&#171;&#186;                                  &#229;&#8218;&#168;Ro04486, 63%       Nfa4510, 65%               DHD <br />  <br /> *&#226;&#171;&#185;&#226;&#171;&#185;, strictly conserved motif &#229;&#8230;&#179;YF&#229;&#8230;&#180;-x(1,2)-&#229;&#8230;&#179;LIVG&#229;&#8230;&#180;-&#229;&#8230;&#179;STGC&#229;&#8230;&#180;-G-D-x-N-P-&#229;&#8230;&#179;LIV&#229;&#8230;&#180;-H-x(5)-&#229;&#8230;&#179;AS&#229;&#8230;&#180;; &#226;&#171;&#185;, motif including conserved basic catalytic motif &#229;&#8230;&#179;D&#229;&#8230;&#180;-x(4)-H; &#226;&#171;&#186;, <br />  absent. <br /> &#226;&#8364;&#160;Presence ORF(s) demonstrated putatively involved FA/MA biosynthesis transfer. <br /> &#226;&#8364;&#161;&#226;&#171;&#186;, absent present pseudogene; &#226;&#171;&#185;, conserved considered sequenced mycobacterial genomes (see SI Materials Methods). <br /> &#194;&#167;Name orthologous putative protein percentage identity given BlastP alignment &#229;&#8230;&#179;on (quasi-)totality sequence&#229;&#8230;&#180;. The following <br />  <br />  genomes analyzed: Rhodococcus sp. RHA1, Nocardia farcinica IFM10152, Corynebacterium diphtheriae NCTC13129, Corynebacterium glutamicum ATCC <br />  13032, Corynebacterium jeikeium K411, Corynebacterium efficiens YS-314. <br /> &#194;&#182;SHD, single hot dog fold [known (underlined) SHD, ref. 15]; DHD, double hot dog fold [known (underlined) OHO, ref. 14]. <br /> &#229;&#8218;&#168;Detected C. glutamicum. <br />  <br /> **Detected C. jeikeium. <br />  <br />  <br />  <br /> any, putative proteins missing piece                 tity, poorly related Rv0636 (13&#226;&#8364;&#8220;15% <br /> FAS-II puzzle.                                                                    identity) (SI Fig. 8), suggesting Rv0635 Rv0637 might <br />                                                                                   similar role, distinct Rv0636. <br /> Results <br /> Selection HAD Candidates Through Silico Analyses. To select              Essentiality Rv0635-Rv0637 Cluster. We used two-step <br /> silico candidates dehydration step FAS-II cycle                    homologous recombination strategy demonstrate the <br /> among putative (R)-specific hydratases/dehydratases previ-                    Rv0635-Rv0637 gene cluster essential vitro. A deletion <br /> ously identified Mtb (13, 14), defined different criteria              delivery vector entire operon deleted was <br /> (Table 1): (i) ubiquity protein mycobacteria                     constructed. In addition, merodiploid strain constructed in <br /> and related genera (Nocardia Rhodococcus), (ii) absence                   entire operon integrated using L5-based <br /> in Corynebacterium devoid FAS-II (see Search                    vector. Attempts generate unmarked deletion three <br /> for HadA, HadB, HadC Orthologs Other Corynebacteri-                        genes merodiploid WT backgrounds <br /> neae), (iii) occurrence catalytic motifs, (iv)                     (SI Fig. 9). In unable isolate deletion <br /> proximity chromosome genes potentially involved                      mutants. Forty double crossover (DCO) strains analyzed by <br /> FA/MA biosynthesis transfer.                                                   PCR, WT gene. In contrast, merodiploid <br />    Among proteins genes present myco-                    background able delete normal chromosomal <br /> bacterial genomes sequenced far, Rv0216 obvious                         copy; DCOs WT (integrated) the <br /> catalytic motif (14), Rv0504c ortholog Rhodococ-                    deletion alleles (SI Fig. 9), confirmed Southern blot <br /> cus Nocardia (Table 1). In contrast, Rv0241c Rv0636                    analysis (data shown). This provided direct evidence one <br /> both represented genera. They bear characteristic                  genes Rv0635-Rv0637 cluster essential vitro. <br /> well conserved catalytic sequence, called &#226;&#8364;&#732;&#226;&#8364;&#732;hydratase 2 motif&#226;&#8364;&#8482;&#226;&#8364;&#8482;               In contrast, Rv0241c deletion easily obtained WT <br /> (Table 1 SI Fig. 8), includes catalytic residues                     background using methodology (T.P., unpublished <br /> D-x(4)-H (13, 14) observed hydratase 2 family (16).                     results). Because MAs components essential mycobac- <br /> Moreover, Rv0241c adjacent ORF encoding putative                       terial growth, gene(s) encoding HAD(s) FAS-II <br /> &#226;?&#164;-ketoacyl-ACP reductase (FabG4) distinct FAS-II,                    likely essential, like kasA, mabA, inhA <br /> MabA (FabG1), Rv0636 located near mmaA1-4 gene                         (18&#226;&#8364;&#8220;20). Thus, data consistent Rv0635- <br /> cluster involved biosynthesis oxygenated MAs (3).                       Rv0636-Rv0637 cluster best candidate long- <br /> However, Rv0241c ortholog Corynebacte-                        sought dehydratase. For greater convenience, proteins will <br />                                                                                                                                                                     BIOCHEMISTRY <br />  <br />  <br />  <br /> rium species (Table 1). Thus, Rv0636 satisfies selection                called HadA, HadB, HadC, respectively. <br /> criteria represents best candidate. Moreover, Mtb <br /> chromosomes, Rv0635 overlaps Rv0636 Rv0637 4 bp away                       Expression Purification HabA-C Proteins: Analysis Their <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Rv0636, implying genes form operon.                      Quaternary Structure Vitro. HadB gene hadA-hadB-hadC and <br /> The structural prediction data Rv0636 (14), Rv0635,                       hadB-hadC gene clusters cloned TOPO expression <br /> Rv0637 (Table 1) suggested putative proteins                   vector downstream His tag coding sequence. Proteins were <br /> of 15&#226;&#8364;&#8220;19 kDa composed single hot dog fold.                          produced Escherichia coli purified using two-step <br /> Proteins topology form dimers (at                     chromatography procedure Ni Sepharose Superdex 75 <br /> basic structural motif) (16, 17). Because Rv0635 Rv0637 lack                  columns. It noteworthy that, hadB expressed alone, <br /> the hydratase 2 catalytic motif (Table 1 SI Fig. 8),                       His-tagged product [His-tagged HadB (H-HadB)] <br /> hypothesized Rv0636 protein needs associate het-                       insoluble, purified protein relatively unstable in <br /> erodimers Rv0635 Rv0637 functional. It                    solution. In contrast, hadA-hadB-hadC clone, significant <br /> noteworthy Rv0635 Rv0637 share 45% sequence iden-                        protein isolated. Strikingly, His-tagged <br />  <br /> Sacco et al.                                                                                       PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 37 &#229;&#8230;&#169; 14629 <br />  Fig. 1. Protein purification. SDS/PAGE gel filtration chromatograms. The <br /> cloned genes mentioned. Lane a, total soluble proteins; lane b, Ni Sepha- <br /> rose fraction; lanes c d, Superdex 75 fractions. Monomeric sizes pro- <br /> teins: H-HadA, 18.3 kDa; HadB, 14.8 kDa; H-HadB, 15.9 kDa; HadC, 18.9 kDa. <br />  <br />  <br />  <br /> HadA (H-HadA) copurified untagged HadB, the <br /> presence HadC undetectable (Fig. 1). Moreover, when <br /> H-HadB HadC coproduced, H-HadB coeluted with <br /> untagged HadC (Fig. 1). These data strongly suggested there <br /> were interactions HadA HadB HadB <br /> and HadC. <br />   During gel filtration chromatography, performed 4&#194;&#176;C, the <br /> pool Ni-column fractions containing H-HadB HadC <br /> displayed profile major peak estimated 65 kDa and <br /> a minor peak &#226;&#172;&#8225;33 kDa (Fig. 1). Analyses SDS/PAGE (Fig. <br /> 1) peptide mapping demonstrated elution peak <br /> corresponded equimolar amounts H-HadB HadC, and <br /> the second corresponded H-HadB alone. A similar <br /> chromatogram obtained fractions containing H- <br />                                                                                  Fig. 3. Chain <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> specificity profiles HadAB HadBC heterodimers. <br /> HadA HadB, major peak (at 65 kDa) holding both <br />                                                                                  Assays performed fixed concentrations substrate enzyme. (A <br /> proteins minor peak holding H-HadA (Fig. 1).                        B) HadAB 2.5 &#226;?&#174;M enoyl-CoA (A) 25 &#226;?&#174;M enoyl-CoA (B). (C) HadBC <br />   Data suggested heterocomplexes Rv0635-Rv0636                          25 &#226;?&#174;M enoyl-CoA. Data means &#226;&#171;&#190; SD. The activities measured both <br /> (HadAB) Rv0636-Rv0637 (HadBC) formed tetramers. The                          heterodimers presence short chain substrates (C4 C8) not <br />                                                                                  significantly control values (without enzyme). The large SDs ob- <br />                                                                                  tained long-chain substrates solubility problems. <br />  <br />  <br />  <br />                                                                                  quaternary structure HadAB examined dy- <br />                                                                                  namic light scattering, confirmed tetramer at <br />                                                                                  low temperature (8&#194;&#176;C) (SI Table 2). However, higher <br />                                                                                  temperature (20&#194;&#176;C) behaved dimer. <br />  <br />                                                                                  Study Protein&#226;&#8364;&#8220;Protein Interactions Vivo Using Split-Trp <br />                                                                                  Method. To investigate occurrence interactions between <br />                                                                                  proteins encoded hadA-C cluster mycobacteria vivo, <br />                                                                                  split-Trp method used Mycobacterium smegmatis. <br />                                                                                  Briefly, tryptophan auxotrophic &#226;&#338;&#172;hisA strain M. smegmatis <br />                                                                                  cotransformed pair plasmids, encoding a <br />                                                                                  target protein fusion N-terminal fragment <br />                                                                                  (Ntrp) C-terminal fragment (Ctrp) Saccharomyces <br />                                                                                  cerevisiae protein Trp1p involved Trp biosynthesis. If vivo <br /> Fig. 2. Split-Trp growth assay M. smegmatis. Recombinant M. smegmatis         interaction occurs target proteins encoded the <br /> &#226;&#338;&#172;hisA-expressing pairs proteins fused Ntrp Ctrp diluted 1- or <br />                                                                                  pair plasmids, Trp1p function restored the <br /> 10-fold water spotted parallel minimal medium medium <br /> plus Trp. The pair Ntrp-Esat6 Cfp10-Ctrp used positive control <br />                                                                                  recombinant strain grow exogenous Trp supply. In <br /> (the early secreted T cell antigens Esat6 Cfp10 Mtb form tight, 1:1   experiments, heterotypic interactions (at 25&#194;&#176;C) ob- <br /> complex). Negative controls extra positive controls performed (see      served M. smegmatis HadA HadB between <br /> SI Materials Methods). Images taken 3-week incubation.          HadB HadC, HadA HadC (Fig. 2). <br />  <br /> 14630 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104                                                                                             Sacco et al. <br />  Fig. 4. Comparison specific activities presence octenoyl-CoA <br /> and octenoyl-ACP. Assays performed 2 &#226;?&#174;M substrate fixed con- <br /> centrations enzymes HadAB (A) HadBC (B). Data means &#226;&#171;&#190; SD. The <br /> activities measured heterodimers presence C8:1-CoA were <br /> not significantly control values (without enzyme). <br />  <br />  <br />  <br /> Weaker homotypic interactions detected HadA and <br /> HadB, consistently latter. <br />   These results agreement vitro data. <br /> They altogether led conclusion HadB interacts vitro <br /> and vivo HadA HadC form heterodimers (at <br /> a temperature &#226;&#177;&#8211;20&#194;&#176;C). <br />  <br /> Enzymatic Activity HadAB HadBC Heterodimers. Enzymes <br /> belonging (R)-specific enoyl hydratase/hydroxyacyl dehy- <br /> dratase family preferentially catalyze hydration reaction <br /> when isolated enzymatic complex (6). Thus, <br /> their activities studied presence enoyl <br /> derivatives vitro. The activities HadAB HadBC het- <br /> erodimers measured presence short-chain <br /> CoA derivatives, 3-hydroxybutyryl-CoA (C4), crotonoyl-CoA <br /> (C4:1), trans-2-octenoyl-CoA (C8:1). HadBC did display <br /> any significant activity substrate experimental <br /> conditions used, varying enzyme concentration over <br /> a large range (see SI Materials Methods). A slight activity of <br /> HadAB detected presence C8                          Fig. 5. Coupled assay HadAB presence MabA InhA. Shown <br /> substrate: 0.506 &#226;&#171;&#190; 0.001 &#226;?&#174;mol/min milligram protein               results MALDI-TOF MS analyses reaction media containing <br /> highest enzyme concentration tested (280 nM).                                   3-ketododecanoyl-CoA, NADPH, NADH, MabA plus InhA (A), plus 280 <br />                                                                                 nM HadAB (B), plus HadAB InhA (C). In A B, peaks m/z <br />    Kinetic experiments realized presence of <br />                                                                                 966, 988, 1,010, 1,032, 1,054 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185;, [M&#226;&#171;&#185;Na]&#226;&#171;&#185;, [M-H&#226;&#171;&#185;2Na]&#226;&#171;&#185;, <br /> longer-chain derivatives, C12&#226;&#8364;&#8220;C20 trans-2-enoyl-CoAs. To mini-                  [M-2H&#226;&#171;&#185;3Na]&#226;&#171;&#185;, [M-3H&#226;&#171;&#185;4Na]&#226;&#171;&#185; ions 3-hydroxydodecanoyl-CoA (product <br /> mize solubility problems encountered amphipathic                  MabA), respectively. In B, minor peaks m/z 948, 970, 992, 1,014, and <br /> molecules, experiments performed fixed low                  1,036 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185; ion monosodium tetrasodium adducts of <br /> substrate concentration (2.5 &#226;?&#174;M). HadAB heterodimer proved                      unsaturated species, dodecenoyl-CoA. In C, peaks m/z 950, 972, 994, <br /> to active presence medium- long-chain                     1,016, 1,038 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185; ion monosodium tetrasodium <br /> molecules (Fig. 3A). In contrast, signal detected                   adducts saturated species, dodecanoyl-CoA. Insets A C display the <br /> HadBC conditions; exhibited activity higher               kinetics respective reactions followed 340 nm (oxidation InhA <br />                                                                                 coenzyme, NADH). The spectra correspond time points 3 min <br /> substrate concentration (25 &#226;?&#174;M). The activity HadBC <br />                                                                                 reaction. <br /> increased increasing chain length (Fig. 3C), whereas <br /> HadAB behaved differently specificity centered the <br /> hexadecenoyl-CoA (C16) (Fig. 3 A B). <br />                                                                                 SI Materials Methods). The MS analyses demonstrate that <br />    The behaviors enzymes presence ACP CoA <br />                                                                                 dodecanoyl-CoA, saturated product InhA, very <br /> derivatives compared using trans-2-octenoyl- <br />                                                                                                                                                                   BIOCHEMISTRY <br />  <br />  <br />  <br /> ACP trans-2-octenoyl-CoA substrates. Both heterodimers                    quickly formed (within 3 min) presence HadAB (Fig. <br /> displayed strict specificity enoyl-ACP (Fig. 4).                   5C). This biosynthesis did occur absence the <br /> Interestingly, HadB homodimer able metabolize                        heterodimer (Fig. 5A). In absence InhA, small propor- <br /> C8:1-ACP conditions identical used het-                     tion intermediate dodecanoyl-CoA, product the <br /> erodimers (data shown).                                                     dehydration reaction catalyzed HadAB, appeared (Fig. 5B). <br />    Finally, heterodimers tested ability                  This type profile reminiscent described for <br /> function coupled reactions presence MabA InhA,                 HADs (6): absence enoyl reductase, reaction <br /> the FAS-II reductases catalyze reactions upstream                  equilibrium favor hydration reaction. By comparison, <br /> downstream dehydration step cycle (SI Fig. 7).                    complete reaction presence HadBC heterodimer <br /> The MabA reaction performed presence                        went slowly: product InhA began appear 30 <br /> 3-ketododecanoyl-CoA (C12), and, completion re-                    min only, reaction arrived completion 3-h <br /> action, heterodimers plus InhA added. Aliquots                  24-h time points (SI Fig. 10). This weak activity correlates <br /> of reaction mixtures taken different time points (see               relatively poor specific activity HadBC the <br />  <br /> Sacco et al.                                                                                     PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 37 &#229;&#8230;&#169; 14631 <br />  Fig. 6. Model roles HadAB HadBC heterodimers MA biosynthesis pathway. HadAB part, like KasA, early FA elongation <br /> cycles catalyzed FAS-II system, leading formation intermediate-size (C22&#226;&#8364;&#8220;C42) meromycolic chains consequently medium-length MAs <br /> found Rhodococcus Nocardia. In mycobacteria, HadBC, like KasB, elongate intermediate-size meromycolic chains full-size molecules <br /> (C52&#226;&#8364;&#8220;C64) late elongation cycles performed FAS-II, resulting synthesis eumycolic acids (C74&#226;&#8364;&#8220;C90). For clarity, proteins FAS-II <br /> system mentioned. <br />  <br />  <br />  <br /> presence medium-chain CoA derivatives (see above, earlier                      Discussion <br /> this paragraph).                                                                     No typical FabZ FabA enzymes universally far the <br />   In conclusion, properties HadAB HadBC het-                              bacterial FAS-II systems present Mtb sequenced <br /> erodimers reminiscent described mycobac-                        mycobacterial genomes. Our previous investigations seek <br /> terial FAS-II individual enzymes                    potential (R)-specific hydratases/dehydratases tubercle <br /> of complex, i.e., marked specificity medium-                           bacillus led (13, 14) protein family that <br /> long-chain substrates ACP derivatives (7&#226;&#8364;&#8220;10). Further-                       include bacterial eukaryotic enoyl-CoA hydratases 2 (16, <br /> more, exhibit (3R)-hydroacyl dehydratase activity                      17). If shares global topology probably same <br /> presence reductases FAS-II complex.                                   reaction mechanism FabA/FabZ family, exhibits very <br />                                                                                      poor sequence similarity catalytic motif is <br /> Search HadA, HadB, HadC Orthologs Other Corynebacteri- <br />                                                                                      distinct (17). Nonetheless, HAD domain multifunc- <br /> neae. To gather putative information respective <br />                                                                                      tional FAS-I polypeptides Corynebacterineae, Fungi, and <br /> roles HadAB HadBC FAS-II cycle, and <br />                                                                                      Archaeoglobus hydratase 2 catalytic motif (13, 14, 21). <br /> consequently formation meromycolic chain, the <br />                                                                                      Similarly, imagine (3R)-hydroxyacyl-ACP <br /> presence organization hadA-C cluster examined <br />                                                                                      dehydratation step elongation cycles monitored the <br /> in Corynebacterineae genera, genomes were <br />                                                                                      mycobacterial FAS-II catalyzed enzyme <br /> available (see Table 1 legend). The chromosomal region <br /> surrounding hadA-C gene cluster conserved                           hydratase 2 family. <br /> within mycobacteria related genera, Coryne-                        Among putative hydratases 2 related proteins iden- <br /> bacterium, Rhodococcus, Nocardia (SI Fig. 11). However,                       tified Mtb, protein cluster HadA-HadB-HadC, probably <br /> Corynebacterium ORFs corresponding hadA, hadB,                            encoded operon, appeared good candidate the <br /> hadC missing, orthologs detected                          HAD FAS-II. The complete set experimental data reported <br /> region BLAST searches (Table 1), known FAS-II                          present study points clearly HadAB HadBC <br /> enzymes. Indeed, Corynebacterium produces short MAs (C22&#226;&#8364;&#8220;                            heterodimers missing pieces elongation system. <br /> C36), result simple condensation classical FAs                     Unlike Rv3389c, previously studied Mtb hydratase 2 protein <br /> (C8&#226;&#8364;&#8220;C18), FAS-II-linked FA elongation steps.                             (13), HadB-containing enzymes display ACP dependence <br /> Rhodococcus Nocardia possess &#226;&#8364;&#732;&#226;&#8364;&#732;intermediate-size&#226;&#8364;&#8482;&#226;&#8364;&#8482; MAs                           specificity long-chain substrates character- <br /> (C34&#226;&#8364;&#8220;C48 C44&#226;&#8364;&#8220;C60, respectively) medium-length                          istic properties FAS-II components (5). <br /> meromycolic chain (C22&#226;&#8364;&#8220;C30 C32&#226;&#8364;&#8220;C42, respectively), shorter                          When coproduced E. coli M. smegmatis, HadB protein <br /> than mycobacteria (C52&#226;&#8364;&#8220;C64) (2). Consistently,                         associates heterodimers HadA HadC. These <br /> genes orthologous hadA hadB, inhA, mabA,                       partnerships substantially change behavior, HadAB <br /> kasA, present Rhodococcus Nocardia (Table 1 SI                        HadBC heterodimers exhibit distinct chain length specificity <br /> Fig. 11). Interestingly, orthologs hadA hadB fused                    profiles. Strikingly, observed genome analyses that <br /> in long unique gene Nocardia. In contrast, hadC ORF                          hadA hadB genes conserved other <br /> missing Nocardia Rhodococcus (SI Fig. 11),                        mycolate-producing genera, Rhodococcus Nocardia. Con- <br /> ortholog genomes (Table 1).                        sistently, genes overlap fuse long unique gene, <br /> Likewise, BLAST searches revealed genera                           suggesting cotranscription association their <br /> hold protein orthologous KasB (data shown).                               products crucial physiology Corynebacterineae. <br />  <br /> 14632 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104                                                                                                       Sacco et al. <br />  Furthermore, demonstrated gene                        topology equivalent predicted structure Rv3389c (13, <br /> hadA-C cluster essential Mtb survival, case                 14), intervening bridge domains. <br /> most genes encoding FAS-II enzymes (18&#226;&#8364;&#8220;20). This consistent                      The present work description bacterial mono- <br /> with transposon mutagenesis-based predictions essentiality                     functional HADs belonging hydratase 2 enzyme family. <br /> for hadA hadB genes (22). Thus, like KasA (4, 23), HadAB                      Thanks elegant experimental strategy, eukaryotic <br /> is likely involved early FA elongation cycles catalyzed               HAD discovered recently, yeast mitochon- <br /> by FAS-II mycobacteria, Nocardia Rhodococ-                     drial FAS-II (12). Strikingly, corresponds hydratase <br /> cus, leading formation intermediate-size meromycolic                    2-related double hot dog enzyme. The high level sequence <br /> chains (Fig. 6). The preference HadBC longer substrates,                   divergence mycobacterial eukaryotic enzymes, <br /> as compared HadAB, absence HadC ortholog                       importance essential step metabolic <br /> genera bearing medium-chain MAs, KasB ortholog                        pathway specific Mycobacterium related genera, makes <br /> is absent, strongly suggest HadBC implicated,                  HadAB HadBC exciting new targets drug discovery. <br /> as described KasB (4, 23), late steps meromycolic               Materials Methods <br /> chain biosynthesis occurring mycobacteria (Fig. 6). Addition- <br />                                                                                   Sequence analyses described (13). The Mtb Rv0635- <br /> ally, hadC gene does essential Mtb growth <br />                                                                                   Rv0637-KO strain constructed based previously re- <br /> in vitro (22). Consistently, kasB shown nonessential in <br />                                                                                   ported strategy (20). DCOs isolated single <br /> Mtb Mycobacterium marinum (4, 23). Furthermore, HadC                          crossover (SCO) merodiploid strains previously described <br /> seems ubiquitous mycobacteria. From data,                          (24). Protein production purification HadB, HadAB, and <br /> propose HadBC necessary formation                          HadBC performed described (14), expres- <br /> complete meromycolic chains MA mycobacteria (Fig. 6).                       sions induced 37&#194;&#176;C, Triton added lysis buffers <br />    From sequence analysis, HadB bears hydratase 2                      (see SI Table 3), Ni Sepharose FF (GE Healthcare) <br /> motif (13, 14) catalytic subunit                    column used. The split-Trp experiments adapted from <br /> HadAB HadBC heterodimers. Furthermore, structural                         procedure described yeast (25). Enzyme assays MS <br /> predictions strongly suggest subunits                  analyses reaction media performed described (13). <br /> a single hot dog fold. Thus, organize asymmetrical                  Short-chain CoA derivatives used 25 &#226;?&#174;M (and 75 <br /> heterodimers reminiscent asymmetrical double                      &#226;?&#174;M hydroxybutyryl-CoA). The octenoyl-ACP prepared <br /> hot dog fold R-hydratase domain MFE-2 enzymes                       using commercially available E. coli holo-ACP (Sigma, St. <br /> probably Rv3389c R-hydratase/dehydratase protein (13, 17).                     Louis, MO), trans-2-octenoic acid, E. coli acyl-ACP synthase <br /> In double hot dog folds, domain contains catalytic                   reported (26). The mycobacterial FAS-II use the <br /> apparatus necessary stabilize                   E. coli ACP instead natural ACP, AcpM (SI Fig. 7) (5). See <br /> long acyl chain substrates. A role analogous                 additional details SI Materials Methods. <br /> can proposed HadA HadC subunits. Consistently, <br /> structural predictions suggest HadAB HadBC                         We thank F. Laval, H. Montrozier, F. Viala (Institut Pharmacologie <br /> hold open active site tunnels (required long-chain                        et Biologie Structurale), L. Henry (Ecole Polytechnique Federale <br />                                                                                   Lausanne) precious help MS analyses, substrate <br /> meromycolate precursors), detailed description must <br />                                                                                   synthesis, figure preparation, split-Trp experiments, respectively; A. <br /> await crystallographic data. In contrast, closed substrate-                     Hartridge (Institute Cell Molecular Science) technical assis- <br /> binding pocket predicted HadB homodimer (15).                        tance Expression Proteomics Facility (Uppsala University) for <br /> Furthermore, absence activity HadB heterodimer                   MS analyses; J. Shanklin (Brookhaven National Laboratory, Upton, NY) <br /> presence octenoyl-ACP suggests heterotypic partner-                   J. Walker (Medical Research Council, Cambridge, U.K.) their <br /> ships HadB HadA HadC play role                       kind gifts plasmid pAasH E. coli C41(DE3), respectively; S. <br /> the interaction ACP moiety substrate. To                      Cole (Institut Pasteur, Paris, France) total Mtb DNA cosmid <br />                                                                                   MTCY20H10.This work supported doctoral fellowship <br /> knowledge, asymmetrical hot dog folded heterodimer                         (to E.S.) Fondation Pour la Recherche Me&#194;&#180;dicale, Marie <br /> been reported far. From organization single fused                 Curie fellowship, grants European Community (QLK2- <br /> gene, deduced orthologs HadA HadB                       CT-2001-02018 LSHP-CT-2005-018923), Foundation Stra- <br /> in Nocardia covalently bound. They adopt                    tegic Research, Swedish Research Council. <br />  <br />  <br />  1. World Health Organization (2004) Anti-Tuberculosis Drug Resistance     13. Sacco E, Legendre V, Laval F, Zerbib D, Montrozier H, Eynard N, Guilhot C, <br />     World, Report No. 3 (WHO, Geneva).                                                Daffe&#194;&#180; M, Que&#194;&#180;mard A (2007) Biochim Biophys Acta 1774:303&#226;&#8364;&#8220;311. <br />  2. Barry CE, III, Lee RE, Mdluli K, Sampson AE, Schroeder BG, Slayden RA,        14. Castell A, Johansson P, Unge T, Jones TA, Ba&#194;&#168;ckbro K (2005) Protein Sci <br />     Yuan Y (1998) Prog Lipid Res 37:143&#226;&#8364;&#8220;179.                                          14:1850&#226;&#8364;&#8220;1862. <br />  3. Dubnau E, Chan J, Raynaud C, Mohan VP, Lane   &#194;&#180;elle MA, Yu K, Que&#194;&#180;mard A,     15. Johansson P, Castell A, Jones TA, Ba&#194;&#168;ckbro K (2006) Protein Sci 15:2300&#226;&#8364;&#8220;2309. <br />     Smith I, Daffe&#194;&#180;, M (2000) Mol Microbiol 36:630&#226;&#8364;&#8220;637.                           16. Hisano T, Tsuge T, Fukui T, Iwata T, Miki K, Doi Y (2003) J Biol Chem <br />  4. Bhatt A, Fujiwara N, Bhatt K, Gurcha SS, Kremer L, Chen B, Chan J, Porcelli       278:617&#226;&#8364;&#8220;624. <br />     SA, Kobayashi K, Besra GS, Jacobs WR, Jr (2007) Proc Natl Acad Sci USA        17. Koski MK, Haapalainen AM, Hiltunen JK, Glumoff T (2004) J Biol Chem <br />     104:5157&#226;&#8364;&#8220;5162.                                                                    279:24666&#226;&#8364;&#8220;24672. <br />                                                                                                                                                                       BIOCHEMISTRY <br />  <br />  <br />  <br />  5. Bloch K (1977) Adv Enzymol Relat Areas Mol Biol 45:1&#226;&#8364;&#8220;84.                      18. Vilche`ze C, Morbidoni HR, Weisbrod TR, Iwamoto H, Kuo M, Sacchettini JC, <br />  6. Rock CO, Cronan JE (1996) Biochim Biophys Acta 1302:1&#226;&#8364;&#8220;16.                         Jacobs WR, Jr (2000) J Bacteriol 182:4059&#226;&#8364;&#8220;4067. <br />  7. Schaeffer ML, Agnihotri G, Volker C, Kallender H, Brennan PJ, Lonsdale JT     19. Bhatt A, Kremer L, Dai AZ, Sacchettini JC, Jacobs WR, Jr (2005) J Bacteriol <br />     (2001) J Biol Chem 276:47029&#226;&#8364;&#8220;47037.                                               187:7596&#226;&#8364;&#8220;7606. <br />  8. Kremer L, Dover LG, Carrere S, Nampoothiri KM, Lesjean S, Brown AK,           20. Parish T, Roberts G, Laval F, Schaeffer M, Daffe&#194;&#180; M, Duncan K (2007) J <br />     Brennan PJ, Minnikin DE, Locht C, Besra GS (2002) Biochem J 364:423&#226;&#8364;&#8220;              Bacteriol 189:3721&#226;&#8364;&#8220;3728. <br />     430.                                                                          21. Qin YM, Haapalainen AM, Kilpela&#194;&#168;inen SH, Marttila MS, Koski MK, Glumoff <br />  9. Marrakchi H, Ducasse S, Labesse G, Montrozier H, Margeat E, Emorine L,            T, Novikov DK, Hiltunen JK (2000) J Biol Chem 275:4965&#226;&#8364;&#8220;4972. <br />     Charpentier X, Daffe&#194;&#180; M, Que&#194;&#180;mard A (2002) Microbiology 148:951&#226;&#8364;&#8220;960.          22. Sassetti CM, Boyd DH, Rubin EJ (2003) Mol Microbiol 48:77&#226;&#8364;&#8220;84. <br /> 10. Que&#194;&#180;mard A, Sacchettini JC, Dessen A, Vilche <br />                                                `ze C, Bittman R, Jacobs WR, Jr,   23. Gao LY, Laval F, Lawson EH, Groger RK, Woodruff A, Morisaki JH, Cox JS, <br />     Blanchard JS (1995) Biochemistry 34:8235&#226;&#8364;&#8220;8241.                                    Daffe&#194;&#180; M, Brown EJ (2003) Mol Microbiol 49:1547&#226;&#8364;&#8220;1563. <br /> 11. Joshi AK, Smith S (1993) J Biol Chem 268:22508&#226;&#8364;&#8220;22513.                         24. Parish T, Stoker NG (2000) Microbiology 146:1969&#226;&#8364;&#8220;1975. <br /> 12. Kastaniotis AJ, Autio KJ, Sormunen RT, Hiltunen JK (2004) Mol Microbiol       25. Tafelmeyer P, Johnsson N, Johnsson K (2004) Chem Biol 11:681&#226;&#8364;&#8220;689. <br />     53:1407&#226;&#8364;&#8220;1421.                                                                 26. Rock CO, Garwin JL (1979) J Biol Chem 254:7123&#226;&#8364;&#8220;7128. <br />  <br />  <br />  <br />  <br /> Sacco et al.                                                                                       PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 37 &#229;&#8230;&#169; 14 <br /> </body></html>