<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mitochondrial pathology muscle and <br /> dopaminergic neuron degeneration caused by <br /> inactivation Drosophila Pink1 rescued Parkin <br /> Yufeng Yang*&#226;&#8364;&#160;, Stephan Gehrke*&#226;&#8364;&#160;, Yuzuru Imai*&#226;&#8364;&#160;, Zhinong Huang*&#226;&#8364;&#160;, Yingshi Ouyang*&#226;&#8364;&#160;, Ji-Wu Wang*&#226;&#8364;&#160;, Lichuan Yang&#226;&#8364;&#161;, <br /> M. Flint Beal&#226;&#8364;&#161;, Hannes Vogel*, Bingwei Lu*&#226;&#8364;&#160;&#194;&#167; <br /> *Department Pathology &#226;&#8364;&#160;Geriatric Research, Education Clinical Center&#229;&#8230;&#190;Veterans Affairs Palo Alto Health Care System, Stanford University School <br /> of Medicine, Palo Alto, CA 94304; &#226;&#8364;&#161;Department Neurology, Cornell University Medical College, 525 East 68th Street, New York, NY 10021 <br />  <br /> Edited Susan L. Lindquist, Whitehead Institute Biomedical Research, Cambridge, MA, approved June 1, 2006 (received review March 28, 2006) <br />  <br /> Mutations Pink1, gene encoding Ser&#229;&#8230;&#190;Thr kinase                    mitochondrial localization Pink1 protein shown using <br /> mitochondrial-targeting signal, associated Parkinson&#226;&#8364;&#8482;s                transfected cells (8, 25, 26). In vitro biochemical studies have <br /> disease (PD), common movement disorder characterized                  demonstrated Pink1 possesses autophosphorylation activity <br /> by selective loss dopaminergic neurons. The mechanism                    pathogenic mutations Pink1 differential effects on <br /> which loss Pink1 leads neurodegeneration understood.              activity (25, 26). The vivo substrate(s) molecular function <br /> Here inhibition Drosophila Pink1 (dPink1) function             Pink1 unknown. Cell culture studies shown that <br /> results energy depletion, shortened lifespan, degeneration              overexpression wild-type Pink1 lead reduction of <br /> of select indirect flight muscles dopaminergic neurons. The                cytochrome c release mitochondria prevent subse- <br /> muscle pathology preceded mitochondrial enlargement                     quent activation caspases basal apoptotic stress <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             NEUROSCIENCE <br /> and disintegration. These phenotypes rescued wild              conditions (27). This result suggests Pink1 play critical <br /> type pathogenic C-terminal deleted form human <br />                                                                                role regulating mitochondrial physiology and&#229;&#8230;&#190;or mitochon- <br /> Pink1 (hPink1). The muscle dopaminergic phenotypes associ- <br />                                                                                drial pathway cell death, vivo relevance these <br /> ated dPink1 inactivation similarity seen parkin <br />                                                                                findings remains determined. <br /> mutant flies suppressed overexpression of <br />                                                                                   <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> understand physiological function Pink1 its <br /> Parkin DJ-1. Consistent genetic rescue results, we <br /> find that, dPink1 <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) animals, level of <br />                                                                                dysfunction cause PD, used Drosophila model <br /> Parkin protein significantly reduced. Together, results <br />                                                                                system. Here, phenotypes caused inactivation of <br /> implicate Pink1 Parkin common pathway regulates                  Drosophila Pink1 (dPink1). Our results indicate dPink1 is <br /> mitochondrial physiology cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> Drosophila.                      required maintaining proper mitochondria morphology the <br />                                                                                integrity subsets muscle cells dopaminergic neurons. <br /> mitochondria &#229;&#8230;&#169; Parkinson&#226;&#8364;&#8482;s disease &#229;&#8230;&#169; Pten-induced kinase 1 &#229;&#8230;&#169;                   Significantly, Pink1 Parkin clear genetic and <br /> indirect flight muscle                                                         biochemical interactions. Our results suggest Parkin Pink1 <br />                                                                                function common molecular pathway regulates mitochon- <br />                                                                                dria function survival selective cell types. <br /> P    arkinson&#226;&#8364;&#8482;s disease (PD) common movement disor- <br />      der characterized pathologically deficiency brain <br /> dopamine content selective degeneration dopaminergic <br />                                                                                Results <br /> neurons substantia nigra. The common forms PD               Knocking Down dPink1 Expression Transgenic RNAi. In the <br /> sporadic known cause. Nevertheless, postmortem studies                 sequenced fly genome, homologue human Pink1 <br /> have identified common features associated sporadic PD,              (hPink1) dPink1. Although dPink1 hPink1 share &#226;&#172;&#8225;26% <br /> as mitochondrial complex I dysfunction, oxidative stress, ag-              identity amino acid level, homology quite low the <br /> gregation abnormal proteins (1, 2).                                         DNA level, sequence homology prediction algo- <br />    Although initial studies etiology PD focused              rithms failed detect significant homology two. Based <br /> environmental factors, recent genetic studies firmly estab-               fact loss-of-function hPink1 associated familial <br /> lished contribution inheritable factors PD pathogenesis (2,          PD, used transgenic RNAi approach knockdown the <br /> 3). At distinct loci associated rare familial         expression dPink1 effort model Pink1-associated PD. To <br /> forms PD (FPD). It anticipated understanding                    confirm expression dPink1 dsRNA resulted down- <br /> molecular lesions associated FPD genes shed light           regulation endogenous dPink1 transcripts, used RT-PCR to <br /> the pathogenesis common forms disease.                      measure dPink1 mRNA levels ubiquitous induction RNAi. <br /> Dominant mutations &#226;?&#163;-Synuclein (&#226;?&#163;-Syn) LRRK2&#229;&#8230;&#190;dardarin                   An &#226;&#172;&#8225;80% reduction dPink1 mRNA observed (Fig. 1A). We <br /> and recessive mutations parkin, DJ-1, Pink1                   assayed effect RNAi endogenous dPink1 protein <br /> associated FPD (4&#226;&#8364;&#8220;10). Of genes, &#226;?&#163;-Syn, parkin,            expression using polyclonal antibody raised dPink1. As <br /> DJ-1 intensively studied. Studies using vivo                 shown Fig. 1B, ubiquitous dPink1 RNAi resulted similar <br /> animal models vitro cell culture linked mutations               degree reduction endogenous dPink1 protein level. Taken to- <br /> these genes impairments mitochondrial structure function             gether, results demonstrated RNAi approach was <br /> and oxidative stress response, reinforcing general involvement <br /> of mitochondrial dysfunction oxidative stress PD pathogen- <br /> esis (11&#226;&#8364;&#8220;21). Consistent notion, proteins            Conflict statement: No conflicts declared. <br /> shown present mitochondria interact mitochondrial             This paper submitted directly (Track II) PNAS office. <br /> proteins (8, 22&#226;&#8364;&#8220;24), suggesting directly regulate                Abbreviations: DA, dopamine; dPink1, Drosophila Pink1; hPink1, human Pink1; dParkin, <br /> mitochondria function.                                                         Drosophila Parkin; hParkin, human Parkin; IFM, indirect flight muscle; PD, Parkinson&#226;&#8364;&#8482;s <br />    A link mitochondria PD supported                 disease; RNAi, RNA interference; TH, tyrosine hydroxylase; TTM, tergotrochanteral muscles. <br /> the fact Pink1 encodes predicted Ser&#229;&#8230;&#190;Thr kinase                  &#194;&#167;To   correspondence addressed. E-mail: bingwei@stanford.edu. <br /> Ca2&#226;&#171;&#185;&#229;&#8230;&#190;calmodulin family localized mitochondria (8). The                  &#194;&#169; 2006 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602493103                                                      PNAS &#229;&#8230;&#169; July 11, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 28 &#229;&#8230;&#169; 10793&#226;&#8364;&#8220;10798 <br />  Fig. 1. Inhibition dPink1 RNAi causes defects abnormal wing posture, shortened lifespan, ATP deficit. (A B) RT-PCR Western blot analyses <br /> of dPink1 mRNA protein levels RNAi. Genotypes: 1, Da-Gal4&#229;&#8230;&#190;&#226;&#171;&#185;; 2, Da-Gal4&#226;&#172;&#381;dPink1 RNAi. RP49 tubulin serve controls. (C&#226;&#8364;&#8220;E) Wing posture <br /> phenotypes control dPink1 RNAi flies. Left, females; right, males. Flies 7 days old kept 29&#194;&#176;C. (C) Straight wing posture control flies. (D) dPink1 <br /> RNAi flies displaying held-up wings. (E) dPink1 RNAi flies displaying drooped wings. (F) Abolished flight activities dPink1 RNAi flies. Two independent lines <br /> were used. (G) dPink1 RNAi flies exhibit sharper age-dependent decline ATP content. (H) Female dPink1 flies exhibit shortened lifespan. (A&#226;&#8364;&#8220;H) RNAi was <br /> achieved using UAS-dPink1 RNAi construct driven Da-Gal4. *, P &#226;&#172;? 0.01 Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br />  <br /> effective causing significant reduction dPink1 mRNA                         displayed &#226;&#172;&#8225;70% reduction overall ATP level compared with <br /> protein expression.                                                                   control flies (Fig. 6, published supporting information <br />                                                                                       PNAS web site). To test ATP deficit was <br /> Ubiquitous Inhibition dPink1 Results Abnormal Wing Posture,                     age-dependent, took advantage fact level of <br /> Energy Depletion, Shortened Lifespan. We analyzed                        transgene expression temperature-dependent UAS-Gal4 <br /> physiological consequence inhibiting dPink1 function. First,                    system. We raised flies 18&#194;&#176;C eclosion minimize RNAi <br /> used daughterless (Da)-Gal4 drive expression dPink1                         effect shifted 29&#194;&#176;C immediately eclosion to <br /> dsRNA ubiquitously. The emergence adult flies ubiquitous                     induce stronger RNAi effect. Our HPLC analysis clearly demon- <br /> inhibition dPink1 suggested loss dPink1 lead                 strated that, ATP level newly eclosed dPink1 <br /> to lethality Drosophila level reduction dPink1                   RNAi flies comparable control flies, dropped <br /> expression transgenic RNAi approach complete                          sharply &#226;&#172;&#8225;40% control level week, level <br /> enough cause lethality.                                                            remained low 2 weeks experimental condition (Fig. <br />    The discernible <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> dPink1 RNAi flies                     1G). We asked energy deficiency affected lifespan <br /> abnormal wing posture: Both females males exhibited                      dPink1 RNAi flies. We global inhibition dPink1 <br /> held-up (Fig. 1D) drooped (Fig. 1E) wing posture,                        reduced lifespan significantly (Fig. 1H Fig. 7, published <br /> control flies held wings parallel body axis (Fig.                 supporting information PNAS web site). Taken together, <br /> 1C). The penetrance phenotype increased age: When                        data suggest Pink1 plays important role regulating <br /> raised 29&#194;&#176;C, &#226;&#172;&#8225;20% newly eclosed flies exhibited abnormal                        energy metabolism function impact lifespan. <br /> wing posture, 7 days age nearly 100% them <br /> displayed phenotype. These flies problem walking,                    Knocking Down dPink1 Induced Selective Muscle Degeneration. <br /> but climbing ability greatly reduced ability fly               One possible anatomical basis abnormal wing posture could <br /> was completely abolished 10 days age (Fig. 1F). This pheno-                     defective flight muscles. Histological analysis indirect flight <br /> type observed independent dPink1 RNAi lines analyzed.                      muscles (IFMs), major flight muscles, Da-Gal4&#226;&#172;&#381;dPink1 <br />    Considering flight energy-consuming physiolog-                    RNAi flies revealed severe disruption muscle integrity, consistent <br /> ical process relying mitochondrial ATP synthesis hPink1                   finding abolished flight capacity flies (Fig. 2B). <br /> has reported associated mitochondria,                         Disrupted muscle integrity observed wing elevator <br /> measured energy supply dPink1 RNAi flies, using ATP                     muscles [dorsal ventral muscles (DVMs)] depressor muscles <br /> level index. Our HPLC data showed dPink1 RNAi flies                        [dorsal longitudinal muscles (DLMs)]. To determine whether <br />  <br /> 10794 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602493103                                                                                                         Yang et al. <br />  Fig. 2. Inhibition dPink1 results disrupted IFMs. (A-H) Light microscopy was <br /> used examine IFM architecture (stars, dorsal longitudinal muscles; arrows, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                    NEUROSCIENCE <br /> dorsal ventral muscles). Sections resin-embedded thoraces 1-week-old <br /> adult flies stained toluidine blue visualize tissue morphology; <br /> anterior left. (I&#226;&#8364;&#8220;N) Wing posture phenotypes control dPink1 RNAi <br /> flies directed Mhc-Gal4. Fly genotypes Da-Gal4&#229;&#8230;&#190;&#226;&#171;&#185; (A), Da-Gal4&#226;&#172;&#381;UAS- <br /> dPink1 RNAi (B), Mhc-Gal4&#229;&#8230;&#190;&#226;&#171;&#185; (C I), Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi (D J), <br /> Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi; UAS-dPink1 (E K), Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 <br /> RNAi; UAS-GFP (F L), Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi; UAS-hPink1 (G M),             Fig. 3.    Muscle-specific dPink1 RNAi results myopathology age- <br /> Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi; UAS-hPink1&#226;&#338;&#172;C (H N).                                    dependent apoptosis IFMs. (A&#226;&#8364;&#8220;D) EM analysis IFM ultrastructure of <br />                                                                                      flies following genotypes: Mhc-Gal4&#229;&#8230;&#190;&#226;&#171;&#185; (A), Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 <br />                                                                                      RNAi (B), Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi; UAS-hPink1 (C), Mhc-Gal4&#226;&#172;&#381;UAS- <br /> dPink1 plays direct role flight muscles, directed dPink1                 dPink1 RNAi; UAS- hPink1&#226;&#338;&#172;C (D). Arrows, swollen mitochondria; arrowheads, <br /> RNAi specifically muscle myosin heavy chain                          rod body-like deposits. (E F) EM analysis TTM ultrastructure Mhc- <br /> (Mhc)-Gal4 driver. Despite lower penetrance (&#226;&#172;&#8225;40% 7-day-old                     Gal4&#229;&#8230;&#190;&#226;&#171;&#185; (E) Mhc-Gal4&#226;&#172;&#381;UAS-dPink1 RNAi (F) flies. Scale bars (5 &#226;?&#174;m) are <br /> flies raised 29&#194;&#176;C), observed similar abnormal wing posture                     shown lower left corner image. (G H) TUNEL staining of <br />                                                                                      thoracic musculatures newly eclosed (G) 1-week-old (H) Mhc- <br /> and muscle disruption flies muscle-specific dPink1 <br />                                                                                      gal4&#226;&#172;&#381;dPink1 RNAi flies. Left, DAPI staining; Center, TUNEL staining; Right, <br /> knockdown (Fig. 2 D J). The lower penetrance                            merged images. Arrows point TUNEL-positive nuclei. <br /> because efficient RNAi Mhc-Gal4 driver contribution <br /> by nonmuscle cells phenotypes observed ubiquitous RNAi <br /> animals. Several lines evidence suggested muscle                        full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> hPink1 hPink1&#226;&#338;&#172;C constructs, making <br /> phenotype resulted specific inhibition dPink1 RNAi.                       hPink1&#226;&#338;&#172;C competent full-length hPink1 competitive satu- <br /> First, expression white, DJ-1A, DJ-1B dsRNAs driven                     ration RNAi machinery, did happen. Taken together, <br /> same Gal4 driver effect flight muscles (data                       concluded abnormal wing&#229;&#8230;&#190;muscle phenotype specif- <br /> shown). Second, rescue tissue-specific RNAi pheno-                     ically caused inactivation dPink1. <br /> type increased expression dPink1. We reasoned by <br /> raising level dPink1 transcripts, RNAi effect                    Muscle-Specific dPink1 RNAi Flies Exhibited Mitochondria Dysfunc- <br /> dampened. Indeed, coexpression UAS-dPink1 transgene                       tion, DNA Fragmentation, Nemaline-Like Myopathology. We then <br /> suppress abnormal wing disrupted muscle phenotypes                           used transmission electron microscopy investigate the <br /> induced dPink1 RNAi (Fig. 2 E K). This rescuing effect                     nature muscle defects. Wild-type adult IFMs highly <br /> unlikely titration Gal4 protein addition UAS                     regular compact myofibril arrangement, tightly <br /> transgenes, UAS-GFP transgene effect                         packed electron-dense mitochondria interspersed rows of <br /> phenotypes (Fig. 2 F L). Finally, rescue dPink1                     sarcomeres (Fig. 3A). In ubiquitous muscle-specific <br /> RNAi phenotypes coexpression full-length hPink1                         dPink1 RNAi flies, IFMs showed irregular dispersed <br /> muscle (Fig. 2 G M). However, C-terminal truncated form                     myofibril arrangement (Fig. 3B). The number mitochondria <br /> hPink1 (hPink1&#226;&#338;&#172;C) able rescue (Fig. 2 H N),                           myofibril reduced, remaining <br /> consistent findings C-terminal truncations hPink1                   mitochondria grossly swollen, lacking electron-dense material, <br /> are linked familial PD (8, 28). The fact used transgenic                  showing disintegration cristae (Fig. 3B). Interestingly, there <br /> flies expressing comparable levels full-length hPink1                         electron-dense deposits area IFMs suc- <br /> hPink1&#226;&#338;&#172;C ruled possibility differential rescuing                    cumbed severe disruption (Fig. 3B), reminiscent nemaline <br /> effect differential expression hPink1 (Fig. 8,               (rod body) myopathy humans (29). Notably, abnormally swollen <br /> is published supporting information PNAS web site).                        mitochondria present morphologically normal myofi- <br /> Instead, result indicates hPink1can functionally substitute                brils (Fig. 3B). Consistent histological observation, myo- <br /> for dPink1. The divergence DNA sequence level                         fibril mitochondrial integrity restored overex- <br /> dPink1 hPink1 makes unlikely hPink1 RNA interfere                   pression full-length hPink1 (Fig. 3C) hPink1&#226;&#338;&#172;C (Fig. 3D). <br /> with RNAi efficiency dPink1. Further, sequence corre-                        In contrast IFMs, tergotrochanteral muscles (TTMs), or <br /> sponding dPink1 dsRNA target sequence wholly present                        &#226;&#8364;&#732;&#226;&#8364;&#732;jumping&#226;&#8364;&#8482;&#226;&#8364;&#8482; muscles, remained fairly normal muscle- <br />  <br /> Yang et al.                                                                                                PNAS &#229;&#8230;&#169; July 11, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 28 &#229;&#8230;&#169; 10795 <br />  Fig. 4. Dopaminergic defects dPink1 RNAi flies. (A&#226;&#8364;&#8220;E) Whole-mount brain TH immunostaining dorsolateral protocerebral posterior (PPL1) cluster neurons <br /> in 25-day-old flies following genotypes: TH-Gal4&#229;&#8230;&#190;&#226;&#171;&#185; (A), TH-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS-GFP (B), TH-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS- hPink1&#226;&#338;&#172;C (C), <br /> TH-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS-hPink1 (D), TH-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS-hParkin (E). (F) Quantification TH&#226;&#171;&#185; neurons individual dopaminergic <br /> clusters adult brains flies indicated genotypes. *, P &#226;&#172;? 0.01 Student&#226;&#8364;&#8482;s t test. (G) Quantification head dopamine levels TH-Gal4&#229;&#8230;&#190;&#226;&#171;&#185;, <br /> TH-Gal4&#229;&#8230;&#190;UAS-White RNAi, TH-Gal4&#229;&#8230;&#190;UAS-dPink1RNAi flies, white RNAi flies serving control. *, P &#226;&#172;? 0.01 Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br />  <br /> specific dPink1 RNAi flies (Fig. 3F, compared Fig. 3E). These                number TH&#226;&#171;&#185; neurons different dopaminergic clusters in <br /> differential effects unlikely expression <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span>               control dPink1 RNAi flies counted subjected to <br /> Mhc-GAL4, flies ubiquitous dPink1 RNAi                      statistical analysis. As shown Fig. 4B, 25-day-old dPink1 RNAi <br /> normal TTMs. The TTMs apparently fewer mitochondria                          flies raised 29&#194;&#176;C showed significant reduction TH&#226;&#171;&#185; neurons <br /> than IFMs. We speculate phenotypic difference be-                    lateral protocerebral posterior (PPL1) cluster. The dorso- <br /> tween IFMs TTMs related different energy                      medial protocerebral posterior (PPM) cluster [also known as <br /> demands.                                                                          dorsomedial cluster (DMC)] showed modest reduction of <br />    To investigate muscle degeneration dPink1 RNAi flies                neuronal number, clusters relatively unaf- <br /> developed cell death mechanism, IFMs sub-                      fected (Fig. 4F). Importantly, observed muscle, coexpres- <br /> jected terminal deoxynucleotidyltransferase-mediated dUTP                    sion full-length hPink1, hPink1&#226;&#338;&#172;C, able suppress <br /> nick end labeling (TUNEL) assay. Flies ubiquitous                        dopaminergic phenotype (Fig. 4 C, D, F). Overexpression <br /> continuous induction dPink1 RNAi exhibited extensive TUNEL-                    hPink1 hPink1&#226;&#338;&#172;C effect TH&#226;&#171;&#185; neuron number <br /> positive signals IFMs analyzed 4 days age,              (data shown). <br /> age-matched control flies lacked staining (Fig. 9,                     To confirm loss dPink1 leads dopaminergic <br /> published supporting information PNAS web site). To                     dysfunction, measured brain dopamine levels using head <br /> exclude developmental effect test TUNEL-                       extracts prepared control dPink1 RNAi flies. In newly <br /> positive signals induced progressively, muscle-specific                  eclosed flies, dopamine content comparable control <br /> dPink1 RNAi flies subjected temperature-shift protocol                dPink1 RNAi flies (Fig. 4G). As flies age, control and <br /> mentioned earlier. No positive TUNEL signal detected IFMs                  dPink1 RNAi flies showed age-dependent decline dopamine <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> flies newly emerged 18&#194;&#176;C (Fig. 3G). However, shifting               levels. However, dPink1 RNAi flies consistently exhibited more <br /> to 29&#194;&#176;C continuously kept temperature 7 days,                     dramatic reduction control (Fig. 4G). Overexpression of <br /> dPink1 RNAi flies readily showed TUNEL-positive nuclei                    full-length hPink1, hPink1&#226;&#338;&#172;C, fully restored dopamine level <br /> IFMs (Fig. 3H), treatment effect                      dPink1 RNAi animals (Fig. 10, published supporting <br /> control flies. These data suggest muscle-specific dPink1 RNAi                information PNAS web site). Taken together, data <br /> could lead age-dependent muscle degeneration characterized                  suggested dPink1 plays critical role promoting dopami- <br /> extensive DNA fragmentation probably indicative cell death.                    nergic neuronal function survival. <br />  <br /> Inactivation dPink1 Dopaminergic Neurons Leads Loss                   Overexpression Parkin Rescued Muscle Dopaminergic <br /> Tyrosine Hydroxylase (TH)&#226;&#180;&#8482; Neuron Reduction Brain Dopamine                 Phenotypes Caused dPink1 Inactivation. We noticed similarity <br /> Content. We analyzed effects inhibiting dPink1 function               dPink1 RNAi flies parkin mutant flies the <br /> in dopaminergic neurons inducing dPink1 RNAi                          muscle dopaminergic pathology (12, 30, 31). This observation <br /> dopaminergic neuron-specific TH-Gal4 driver. The presence                      prompted investigate Parkin Pink1 be <br /> dopaminergic neurons CNS assayed TH immuno-                         involved common physiological processes. Strikingly, abnor- <br /> staining whole-mount preparations adult fly brain. The                      mal wing postures caused dPink1 RNAi rescued the <br />  <br /> 10796 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602493103                                                                                                  Yang et al. <br />  Fig. 5. Genetic biochemical interaction Pink1 Parkin. Wing posture (A B), thorax musculature histology (C D), IFM EM images <br /> (E F) Mhc-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS-hParkin (A, C, E), Mhc-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi; UAS-hDJ-1 (B, D, F) flies shown. Scale bars (2 &#226;?&#174;m) <br /> in E F shown left corner. (G) Whole-body ATP measurements indicated genotypes. *, P &#226;&#172;? 0.01 Student&#226;&#8364;&#8482;s t test. (H) Western blot <br /> analysis showing reduction dParkin levels dPink1 RNAi animals. Protein extracts prepared Da-Gal4&#229;&#8230;&#190;&#226;&#171;&#185; Da-Gal4&#229;&#8230;&#190;UAS-dPink1 RNAi animals were <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 NEUROSCIENCE <br /> probed anti-dParkin antibody. Actin serves protein loading control. <br />  <br />  <br />  <br /> overexpression human Parkin (hParkin) (Fig. 5A),                     metabolism. When Pink1 function compromised, tissues that <br /> hDJ-1 (Fig. 5B) dDJ-1A overexpression rescuing effect.                greatest demand energy, presumably IFM and <br /> Histological EM analyses indicated suppression              dopaminergic neurons, particularly vulnerable. Our <br /> wing phenotype accompanied restoration myofibril                      observation inhibition Pink1 leads ATP depletion is <br /> integrity mitochondrial morphology (Fig. 5 C E, compared                consistent possibility. The second possibility Pink1 <br /> with Fig. 5 D F). Further, overexpression hParkin able               normally required guard mitochondrial <br /> to restore ATP levels dPink1 RNAi flies (Fig. 5G). Importantly,              pathway apoptotic cell death, suggested earlier (27). In this <br /> overexpression hParkin dPink1 RNAi background restored                regard, interesting note Parkin shown to <br /> number TH&#226;&#171;&#185; neurons PPL1 dorsomedial protocere-                    prevent mitochondrial swelling cytochrome c release mito- <br /> bral posterior (PPM1) clusters (Fig. 4 E F). However,              chondria-dependent cell death (23). Thus, Parkin Pink1 may <br /> hParkin overexpression showed tendency elevate brain dopa-                 participate common pathway protects cells mito- <br /> mine levels dPink1 RNAi animals, effect statistically            chondria-dependent cell death induced toxic insults. The abnor- <br /> significant (Fig. 10). Thus, overexpression hParkin capable            mal mitochondrial morphology associated Parkin Pink1 <br /> rescuing most, all, defects caused dPink1 inacti-             inactivation suggests possibility play <br /> vation.                                                                         fundamental roles regulating mitochondrial biogenesis mito- <br />    To gain insight molecular mechanism                chondrial dynamics, mitochondrial fusion fission events. <br /> Parkin overexpression rescues dPink1 RNAi phenotypes, ex-                    A connection aberrant mitochondrial fission&#229;&#8230;&#190;fusion and <br /> amined Parkin protein level dPink1 RNAi animals. As shown                 neurodegeneration appreciated (32). Further <br /> Fig. 5H, Parkin protein level significantly reduced                      studies needed distinguish possibilities. <br /> dPink1RNAi animals compared controls. Thus, the <br />                                                                                    Our vivo rescue studies clearly showed C terminus of <br /> rescue dPink1 RNAi phenotypes Parkin overexpression is <br />                                                                                 hPink1 required hPink1 rescue dPink1 RNAi phenotypes. <br /> likely restoration Parkin protein expression and <br />                                                                                 Thus, C-terminal deleted form Pink1 have <br /> activity. <br />                                                                                 higher vitro kinase activity terms autophosphorylation (26), <br /> Discussion                                                                      incapable providing spectrum Pink1&#226;&#8364;&#8482;s biological <br /> In study, inhibition dPink1 Drosophila leads            activity. It possible C-terminal deletion affect the <br /> to mitochondrial abnormality degeneration subsets mus-                binding Pink1 substrates cofactors. Alternatively, <br /> cle fibers dopaminergic neurons. The similar phenotypes                     deletion Pink1 C terminus contribute disease pathogen- <br /> caused dPink1 Drosophila Parkin (dParkin) inactivation                   esis causing deregulation Pink1 kinase activity. <br /> prompted investigation genetic biochemical                      Our vivo biochemical study showed dPink1 RNAi <br /> relationships. We overexpression hParkin,                 animals level dParkin significantly reduced. This result <br /> DJ-1, rescue muscle dopaminergic pathology                   provides explanation Pink1 Parkin mutants give <br /> induced dPink1 inactivation. Consistent genetic inter-              similar mutant phenotypes Parkin overexpression can <br /> action results, level dParkin significantly            rescue dPink1 RNAi phenotypes. It supports notion that <br /> reduced dPink1 RNAi animals. Together, results strongly                Parkin acts downstream Pink1 common pathway. The <br /> suggest Parkin Pink1 act common cellular pathway             biochemical mechanism Pink1 regulates Parkin protein <br /> normally promotes function survival select cell types                level requires investigation. In summary, mitochondrial <br /> including dopaminergic neurons Parkin act down-                    pathology IFM dopaminergic neuron degeneration phe- <br /> stream Pink1 pathway.                                                notypes observed dPink1 RNAi animals clear genetic <br />    What molecular mechanism underlying degen-                  interaction Pink1 Parkin suggest genetic <br /> eration phenotypes induced Pink1 dysfunction? At lease                 analysis cellular pathway involving Pink1 Parkin will <br /> possibilities envisioned. First, Pink1 regulate energy             reveal fundamental mechanisms governing mitochondrial cel- <br />  <br /> Yang et al.                                                                                            PNAS &#229;&#8230;&#169; July 11, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 28 &#229;&#8230;&#169; 10797 <br />  lular maintenance. Such mechanisms likely applicable                                           average flies genotype time point were <br /> mammalian systems.                                                                                        examined, experiment repeated once. Muscle <br />                                                                                                           histology transmission electron microscopy analysis per- <br /> Materials Methods                                                                                     formed described (31), Epon resin used for <br /> Drosophila Genetics. Fly culture crosses performed ac-                                           embedding. For TUNEL analysis, adult flies fixed ice-cold <br /> cording standard procedures raised indicated tempera-                                           4% formaldehyde&#229;&#8230;&#190;PBS 3 h. For permeation, 1% Triton and <br /> tures. All general fly stocks GAL4 lines obtained                                       prechilled acetone used. Head abdomen then <br /> Bloomington Drosophila stock center. The TH-GAL4 driver a <br />                                                                                                           dissected away, thoraces subject TUNEL analysis <br /> gift Serge Birman (Developmental Biology, Institute of <br /> Marseille, Marseille, France) (33). The fly stocks                                             according manufacturer&#226;&#8364;&#8482;s instructions (Roche Promega). <br /> described earlier follows: UAS-hParkin (34), UAS-hDJ-1 (21), <br /> and UAS-White RNAi (35). To generate UAS-dPink1 RNAi trans-                                               Flight Longevity Analyses. For flight analysis, &#226;&#172;&#8225;15 flies of <br /> genics, genomic DNA&#229;&#8230;&#190;cDNA hybrid constructs generated                                              control experimental flies placed individual vials. Flight <br /> described (35), cDNA sequence covered following                                           events counted consecutive minutes late <br /> PCR primers (5&#226;&#172;&#732;-TTCTGCCACCACCGCCCCCACACTTC                                                            morning subjected averaging. For lifespan analysis, flies <br /> 3&#226;&#172;&#732;-CCGCAGCACATTGGCAGCGGTGG).                                                                              maintained standard media 29&#194;&#176;C, 15 flies vial, and <br />                                                                                                           transferred new food medium daily. Mortality scored daily. <br /> Molecular Biology. To make UAS-hPink1, UAS-hPink1&#226;&#338;&#172;C, and <br /> UAS-dPink1 transgenics, corresponding cDNAs cloned                                              ATP Dopamine Measurements. For whole-fly ATP measure- <br /> the pUAST vector. The plasmid containing hPink1 cDNA gift                                           ment, live adult flies dropped liquid nitrogen snap <br /> from M. Unoki (36). The hPink1&#226;&#338;&#172;C construct contains 509                                         freezing. Sixty microliters dry ice-embedded acetonitrile was <br /> aa, mimicking reported disease-linked, C-terminal truncated                                           added flies, homogenized adding ice-cold <br /> form hPink1 (28). Details cloning steps available on <br />                                                                                                           deionized H2O. The homogenate centrifuged 13,200 &#226;&#171;&#187; g at <br /> request. For RT-PCR analysis, 4-day-old adult flies cross <br /> between UAS-dPink1 RNAi Da-GAL4 raised 29&#194;&#176;C used                                              4&#194;&#176;C 15 min, 50 &#226;?&#174;l supernatant mixed 50 &#226;?&#174;l <br /> to prepare total RNA using RNeasy Kit (Qiagen). Details                                          deionized H2O, centrifuged again, readied HPLC assay. <br /> the quantitative RT-PCR procedure essentially described                                           HPLC analysis ATP content performed essentially as <br /> (34). Antibody dPINK1 elicited rabbits recom-                                         described (37, 38). HPLC analysis catecholamine levels was <br /> binant proteins purified bacteria culture expressing pGEX-                                           performed described (39, 40). For sample preparation, adult <br /> 6P-1-Pink1C, contains C-terminal 97 aa dPink1 (amino                                         male fly heads dissected homogenized 0.1 M <br /> acids 624 721). Western blot analysis using antibody                                       perchloric acid (generally 50 &#226;?&#174;l heads) using a <br /> performed described (34), primary antibody used                                            motorized hand-held tissue grinder. The homogenate frozen <br /> 1:1,000 dilution. hParkin cDNA, gift N. Hattori (Juntendo                                          immediately dry ice stored &#226;&#171;&#186;80&#194;&#176;C HPLC analysis. <br /> University, Tokyo, Japan), inserted pcDNA3 vector a <br /> Myc tag C terminus. Rabbit anti-Drosophila TH antibody                                             We thank Dr. Serge Birman Bloomington Drosophila Stock <br /> raised recombinant GST-Drosophila TH (longer isoform,                                             Center fly stocks, Dr. M. Unoki hPink1 cDNA, Dr. N. Hattori <br /> 1&#226;&#8364;&#8220;328 aa) produced bacteria. The crude serum immunoab-                                             hParkin cDNA, Dr. Leo Palanck dParkin antibody, Dr. Su Guo <br /> sorbed NHS-activated Sepharose (General Electric Bio-                                                reading manuscripts, Dr. Zhuohua Zhang communicating <br /> sciences) coupled soluble bacteria proteins. Supernatant                                             unpublished results. We thank Jennifer Quach Yali Zhang for <br /> (1:500) used immunostaining.                                                                  excellent technical support members Lu laboratory for <br />                                                                                                           discussions. This work supported McKnight, Beckman, and <br /> Histology Immunohistochemistry, Muscle Histology, Trans-                                          Sloan Foundations (B.L.), Naito Foundation Japan Society the <br /> mission Electron Microscopy Analysis. Whole-mount immunohisto-                                            Promotion Science Postdoctoral Fellowships Research Abroad (to <br /> chemistry TH staining performed described (31). An                                             Y.I.), Stanford Bio X Graduate Fellowship (to Y.Y.). <br />  <br />  <br />  1.   Dunnett, S. B. &amp; Bjorklund, A. (1999) Nature 399, A32&#226;&#8364;&#8220;A39.                                          21. Yang, Y., Gehrke, S., Haque, M. E., Imai, Y., Kosek, J., Yang, L., Beal, M. F., Nishimura, <br />  2.   Dawson, T. M. &amp; Dawson, V. L. (2003) Science 302, 819&#226;&#8364;&#8220;822.                                              I., Wakamatsu, K., Ito, S., et al. (2005) Proc. Natl. Acad. Sci. USA 102, 13670&#226;&#8364;&#8220;13675. <br />  3.   Bertoli-Avella, A. M., Oostra, B. A. &amp; Heutink, P. (2004) Hum. Genet. 114, 413&#226;&#8364;&#8220;438.                 22. Zhang, L., Shimoji, M., Thomas, B., Moore, D. J., Yu, S. W., Marupudi, N. I., Torp, R., <br />  4.   Polymeropoulos, M. H., Lavedan, C., Leroy, E., Ide, S. E., Dehejia, A., Dutra, A., Pike, B.,            Torgner, I. A., Ottersen, O. P., Dawson, T. M. &amp; Dawson, V. L. (2005) Hum. Mol. Genet. <br />       Root, H., Rubenstein, J., Boyer, R., et al. (1997) Science 276, 2045&#226;&#8364;&#8220;2047.                              14, 2063&#226;&#8364;&#8220;2073. <br />  5.   Kitada, T., Asakawa, S., Hattori, N., Matsumine, H., Yamamura, Y., Minoshima, S.,                   23. Darios, F., Corti, O., Lucking, C. B., Hampe, C., Muriel, M. P., Abbas, N., Gu, W. J., Hirsch, <br />       Yokochi, M., Mizuno, Y. &amp; Shimizu, N. (1998) Nature 392, 605&#226;&#8364;&#8220;608.                                       E. C., Rooney, T., Ruberg, M. &amp; Brice, A. (2003) Hum. Mol. Genet. 12, 517&#226;&#8364;&#8220;526. <br />  6.   Leroy, E., Boyer, R., Auburger, G., Leube, B., Ulm, G., Mezey, E., Harta, G., Brownstein,           24. Elkon, H., Don, J., Melamed, E., Ziv, I., Shirvan, A. &amp; Offen, D. (2002) J. Mol. Neurosci. 18, 229&#226;&#8364;&#8220;238. <br />       M. J., Jonnalagada, S., Chernova, T., et al. (1998) Nature 395, 451&#226;&#8364;&#8220;452.                            25. Beilina, A., Van Der Brug, M., Ahmad, R., Kesavapany, S., Miller, D. W., Petsko, G. A. &amp; <br />  7.   Bonifati, V., Rizzu, P., Squitieri, F., Krieger, E., Vanacore, N., van Swieten, J. C., Brice, A.,       Cookson, M. R. (2005) Proc. Natl. Acad. Sci. USA 102, 5703&#226;&#8364;&#8220;5708. <br />       van Duijn, C. M., Oostra, B., Meco, G. &amp; Heutink, P. (2003) Neurol. Sci. 24, 159&#226;&#8364;&#8220;160.               26. Silvestri, L., Caputo, V., Bellacchio, E., Atorino, L., Dallapiccola, B., Valente, E. M. &amp; <br />  8.   Valente, E. M., Abou-Sleiman, P. M., Caputo, V., Muqit, M. M., Harvey, K., Gispert, S., Ali,            Casari, G. (2005) Hum. Mol. Genet. 14, 3477&#226;&#8364;&#8220;3492. <br />       Z., Del Turco, D., Bentivoglio, A. R., Healy, D. G., et al. (2004) Science 304, 1158&#226;&#8364;&#8220;1160.          27. Petit, A., Kawarai, T., Paitel, E., Sanjo, N., Maj, M., Scheid, M., Chen, F., Gu, Y., Hasegawa, <br />  9.   Zimprich, A., Biskup, S., Leitner, P., Lichtner, P., Farrer, M., Lincoln, S., Kachergus, J.,            H., Salehi-Rad, S., et al. (2005) J. Biol. Chem. 280, 34025&#226;&#8364;&#8220;34032. <br />       Hulihan, M., Uitti, R. J., Calne, D. B., et al. (2004) Neuron 44, 601&#226;&#8364;&#8220;607. <br />                                                                                                           28. Rohe, C. F., Montagna, P., Breedveld, G., Cortelli, P., Oostra, B. A. &amp; Bonifati, V. (2004) <br /> 10.   Paisan-Ruiz, C., Jain, S., Evans, E. W., Gilks, W. P., Simon, J., van der Brug, M., Munain, <br />                                                                                                               Ann. Neurol. 56, 427&#226;&#8364;&#8220;431. <br />       A. L., Aparicio, S., Gil, A. M., Khan, N., et al. (2004) Neuron 44, 595&#226;&#8364;&#8220;600. <br />                                                                                                           29. Wallgren-Pettersson, C. (2002) Semin. Pediatr. Neurol. 9, 132&#226;&#8364;&#8220;144. <br /> 11.   Palacino, J. J., Sagi, D., Goldberg, M. S., Krauss, S., Motz, C., Klose, J. &amp; Shen, J. (2004) <br />                                                                                                           30. Whitworth, A. J., Theodore, D. A., Greene, J. C., Benes, H., Wes, P. D. &amp; Pallanck, L. J. <br />       J. Biol. Chem. 279, 18614&#226;&#8364;&#8220;18622. <br />                                                                                                               (2005) Proc. Natl. Acad. Sci. USA 102, 8024&#226;&#8364;&#8220;8029. <br /> 12.   Greene, J. C., Whitworth, A. J., Kuo, I., Andrews, L. A., Feany, M. B. &amp; Pallanck, L. J. (2003) <br />       Proc. Natl. Acad. Sci. USA 100, 4078&#226;&#8364;&#8220;4083.                                                          31. Pesah, Y., Pham, T., Burgess, H., Middlebrooks, B., Verstreken, P., Zhou, Y., Harding, M., <br /> 13.   Hsu, L. J., Sagara, Y., Arroyo, A., Rockenstein, E., Sisk, A., Mallory, M., Wong, J.,                   Bellen, H. &amp; Mardon, G. (2004) Development (Cambridge, U.K.) 131, 2183&#226;&#8364;&#8220;2194. <br />       Takenouchi, T., Hashimoto, M. &amp; Masliah, E. (2000) Am. J. Pathol. 157, 401&#226;&#8364;&#8220;410.                     32. Bossy-Wetzel, E., Barsoum, M. J., Godzik, A., Schwarzenbacher, R. &amp; Lipton, S. A. (2003) <br /> 14.   Hyun, D. H., Lee, M., Hattori, N., Kubo, S., Mizuno, Y., Halliwell, B. &amp; Jenner, P. (2002)              Curr. Opin. Cell Biol. 15, 706&#226;&#8364;&#8220;716. <br />       J. Biol. Chem. 277, 28572&#226;&#8364;&#8220;28577.                                                                    33. Friggi-Grelin, F., Coulom, H., Meller, M., Gomez, D., Hirsh, J. &amp; Birman, S. (2003) <br /> 15.   Orth, M., Tabrizi, S. J., Schapira, A. H. &amp; Cooper, J. M. (2003) Neurosci. Lett. 351, 29&#226;&#8364;&#8220;32.            J. Neurobiol. 54, 618&#226;&#8364;&#8220;627. <br /> 16.   Sherer, T. B., Betarbet, R., Stout, A. K., Lund, S., Baptista, M., Panov, A. V., Cookson, M. R.     34. Yang, Y., Nishimura, I., Imai, Y., Takahashi, R. &amp; Lu, B. (2003) Neuron 37, 911&#226;&#8364;&#8220;924. <br />       &amp; Greenamyre, J. T. (2002) J. Neurosci. 22, 7006&#226;&#8364;&#8220;7015.              <br /> </body></html>