<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Schizosacchromyces pombe Dpb2 Binds <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> Origin DNA <br /> Early in S Phase and Is Required for Chromosomal <br /> DNA Replication <br />  <br />         Genetic evidence suggests that DNA polymerase epsilon (Pol &#226;&#8216;&#8364;) has a noncatalytic essential role <br />         during the early stages of DNA replication initiation. Herein, we report the cloning and charac- <br />         terization of the second largest subunit of Pol &#226;&#8216;&#8364; in fission yeast, called Dpb2. We demonstrate that <br />         Dpb2 is essential for cell viability and that a temperature-sensitive mutant of dpb2 arrests with a <br />         1C DNA content, suggesting that Dpb2 is required for initiation of DNA replication. Using a <br />         chromatin immunoprecipitation assay, we show that Dpb2, binds preferentially to origin DNA at <br />         the beginning of S phase. We also show that the C terminus of Pol &#226;&#8216;&#8364; associates with origin DNA <br />         at the same time as Dpb2. We conclude that Dpb2 is an essential protein required for an early step <br />         in DNA replication. We propose that the primary function of Dpb2 is to facilitate assembly of the <br />         replicative complex at the start of S phase. These conclusions are based on the novel cell cycle <br />         arrest <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of the dpb2 mutant, on the previously uncharacterized binding of Dpb2 to <br />         replication origins, and on the observation that the essential function of Pol &#226;&#8216;&#8364; is not dependent on <br />         its DNA synthesis activity. <br />  <br />  <br /> INTRODUCTION                                                             A second processive DNA polymerase, Pol &#226;&#8216;&#8364;, is re- <br />                                                                       quired for cell viability and chromosomal DNA replica- <br /> Chromosomal DNA replication in eukaryotic cells requires              tion in both fission (D&#226;&#8364;&#8482;Urso and Nurse, 1997) and budding <br /> the activity of at least three DNA polymerases: &#226;&#65533;&#163;, &#226;&#65533;&#166;, and &#226;&#8216;&#8364;           yeast (Morrison et al., 1990; Araki et al., 1992; Budd and <br /> (Waga and Stillman, 1998; Hubscher et al., 2000; Bell and             Campbell, 1993). Pol &#226;&#8216;&#8364; has also been implicated in both <br /> Dutta, 2002). Pol &#226;&#65533;&#163; is tightly associated with a primase              DNA repair (Jessberger et al., 1993; Wang et al., 1993; <br /> activity capable of de novo DNA synthesis, suggesting that            Shivji et al., 1995; Holmes, 1999) and cell cycle checkpoint <br /> this enzyme is responsible for initiation of both leading and         control in eukaryotic cells (Navas et al., 1995). Recently, <br /> lagging strands (Waga and Stillman, 1998). Pol &#226;&#65533;&#163;/primase              we have shown that the DNA polymerase and exonucle- <br /> can only synthesize short primers that must then be ex-               ase domains of Pol &#226;&#8216;&#8364; are dispensable for cell viability in <br /> tended by the activity of a processive DNA polymerase(s).             fission yeast (Feng and D&#226;&#8364;&#8482;Urso, 2001). Similar observa- <br /> Biochemical analysis of simian virus 40 (SV40) DNA repli-             tions have been made for the evolutionarily distant yeast, <br /> cation, which has been extensively used as a model system <br />                                                                       Saccharomyces cerevisiae (Kesti et al., 1999; Dua et al., 2000). <br /> for eukaryotic DNA replication, has shown that primers <br />                                                                       These findings have raised important questions regarding <br /> synthesized by Pol &#226;&#65533;&#163; can be extended by Pol &#226;&#65533;&#166; and that these <br />                                                                       the essential role of this replicative enzyme in DNA syn- <br /> two polymerases are sufficient for SV40 replication in vitro <br />                                                                       thesis. Although the N-terminal catalytic domains are <br /> (Weinberg et al., 1990; Waga et al., 1994). <br />                                                                       dispensable, the C-terminal half of the enzyme is essential <br />                                                                       for cell viability and chromosomal replication in fission <br />  Article published online ahead of print. Mol. Biol. Cell 10.1091/    yeast (D&#226;&#8364;&#8482;Urso and Nurse, 1997). In particular, we have <br /> mbc.E03&#226;&#8364;&#8220; 02&#226;&#8364;&#8220; 0088. Article and publication date are available at      shown that temperature-sensitive mutants of Pol &#226;&#8216;&#8364; in <br /> www.molbiolcell.org/cgi/doi/10.1091/mbc.E03&#226;&#8364;&#8220; 02&#226;&#8364;&#8220; 0088.                fission yeast arrest with a 1C DNA content upon shift to <br />  * Present address: Department of Genome Sciences, University of <br />    Washington, Box 357730, Seattle, WA 98195. <br />                                                                       the restrictive temperature, suggesting that Pol &#226;&#8216;&#8364; (presum- <br />  &#226;&#8364;&#160; <br />    Corresponding author. E-mail address: gdurso@miami.edu.            ably the noncatalytic C-terminal half of the protein) is <br />    Abbreviations used: ChIP, chromatin immunoprecipitation; Pol,      required during the early stages of DNA replication ini- <br />    DNA polymerase; PCR, polymerase chain reaction.                    tiation (D&#226;&#8364;&#8482;Urso and Nurse, 1997). Nevertheless, its precise <br />  <br />  <br /> &#194;&#169; 2003 by The American Society for Cell Biology                                                                                  3427 <br />  W. Feng et al. <br />  <br />  <br />  <br /> Table 1. S. pombe strains used in this study <br />  <br /> Strain                                                  Genotype                                                            Source <br />  <br /> 972                 h&#226;&#171;&#186;                                                                                                Moreno et al., 1991 <br /> 567                 h&#226;&#171;&#186; ade6-704 leu1-32 ura4-D18                                                                      Moreno et al., 1991 <br /> GD28                h&#226;&#171;&#186;/h&#226;&#171;&#185; ade6-M210/ade6-M216 leu1-32/leu1-32 ura4-D18/ura4-D18                                          This study <br /> GD131               h&#226;&#171;&#186;/h&#226;&#171;&#185; dpb2&#226;&#171;&#185;/dpb2&#226;&#172;&#8249;ura4&#226;&#171;&#185; ade6-M210/ade6-M216 leu1-32/leu1-32 ura4-D18/ura4-D18                         This study <br /> GD149               h&#226;&#171;&#185; dpb2&#226;&#172;&#8249;ura4&#226;&#171;&#185; leu1-32 ura4-D18 int pJK148(nmt81)-dpb2&#226;&#171;&#185;                                               This study <br /> GD242               h&#226;&#171;&#186; dpb2&#226;&#172;&#8249;4xflag-dpb2&#226;&#171;&#185;(sup3-5) ade6-704 leu1-32 ura4-D18                                               This study <br /> GD254               h&#226;&#171;&#186; cdc25-22 dpb2&#226;&#172;&#8249;4xflag-dpb2&#226;&#171;&#185;(sup3-5) ade6-704 leu1-32 ura4-D18                                      This study <br /> GD280               h&#226;&#171;&#186; dpb2&#226;&#172;&#8249;ura4&#226;&#171;&#185; leu1-32 ura4-D18 int pJK148(nmt81)-dpb2 413-417                                        This study <br /> GD261               h&#226;&#171;&#186; cdc10-129 dpb2&#226;&#172;&#8249;4xflag-dpb2&#226;&#171;&#185;(sup3-5) ade6-704 leu1-32 ura4-D18                                     This study <br /> GD237               h&#226;&#171;&#186; cdc25-22 cdc20&#226;&#172;&#8249;ura4&#226;&#171;&#185; ade6-M216 leu1-32 ura4-D18 int pJK148 (nmt1) 4xflag-cdc20C1                  This study <br /> GD205               h&#226;&#171;&#186; cdc25-22 ade6-704 leu1-32 ura4-D18                                                                This study <br />  <br />  <br />  <br />  <br /> molecular function in eukaryotic chromosomal DNA rep-                EXPERIMENTAL PROCEDURES <br /> lication remains obscure. <br />    Pol &#226;&#8216;&#8364; has been purified <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> both budding yeast and               Schizosacchromyces pombe Strains and Materials <br /> human cell extracts as a complex of at least four distinct           All S. pombe strains were derived from 972(h&#226;&#171;&#186;), 975(h&#226;&#171;&#185;), and 968(h90) <br /> polypeptides that are conserved from yeast to human                  by using standard genetic procedures (Moreno et al., 1991; Table 1). <br /> (Hamatake et al., 1990; Chui and Linn, 1995). Only the largest       S. pombe cosmid p8B7, containing the complete genomic sequence of <br />                                                                      dpb2&#226;&#171;&#185;, was obtained from The Sanger Center (Hinxton, United <br /> 256-kDa subunit contains the domains required for DNA                Kingdom). S. pombe strain OGC9 and anti-Mcm6 polyclonal anti- <br /> synthesis activity, whereas the function of the smaller sub-         bodies were kindly provided by H. Masukata (Osaka University, <br /> units, called Dpb2&#226;&#8364;&#8220; 4, is currently unknown. Deletion of each        Osaka, Japan). <br /> of the three smaller subunits from the S. cerevisiae genome <br /> results in different phenotypes. Only Dpb2 is essential for <br /> viability and normal S phase progression (Araki et al.,              S. pombe Methods <br /> 1991a). Although both Dpb3 and Dpb4 are not required for             All media and growth conditions, unless otherwise noted, were as <br /> cell viability, deletion of the genes encoding these proteins        described previously (Moreno et al., 1991). DNA content was deter- <br /> does result in partial defects in DNA replication (Araki et al.,     mined by fluorescence-activated cell sorting (FACS) as described <br />                                                                      previously (Sazer and Sherwood, 1990). To block cells at the G2/M <br /> 1991b; Ohya et al., 2000). Interestingly, both Dpb3 and Dpb4         transition, cells containing the cdc25-22 allele were incubated in <br /> contain histone-fold motifs, suggesting these proteins might         minimal media at the restrictive temperature of 36&#194;&#176;C for 4 h. Reen- <br /> be involved in chromatin remodeling (Araki et al., 1991b; Li         try into the cell cycle was induced by rapidly chilling cells to the <br /> et al., 2000; Ohya et al., 2000). Based on the existence of these    permissive temperature of 25&#194;&#176;C. <br /> motifs, it has been proposed these subunits might facilitate <br /> replication through heterochromatic regions of DNA (Fuss <br />                                                                      Molecular Cloning and the Generation of the <br /> and Linn, 2002). <br />    To further explore the function of the Pol &#226;&#8216;&#8364; in fission <br />                                                                      Heterozygous Diploid Strain GD131 <br /> yeast, we have set out to clone and characterize the three           (dpb2&#226;&#180;&#8482;/dpb2::ura4&#226;&#180;&#8482;) and the Haploid Strain GD149 <br /> additional putative subunits of the Pol &#226;&#8216;&#8364; complex. In this           [dpb2::ura4&#226;&#180;&#8482; int pJK148-(nmt81)dpb2&#226;&#180;&#8482;] <br /> manuscript, we report the cloning and characterization of            One to two kilobases of flanking genomic sequence, upstream and <br /> the second largest subunit encoded by dpb2&#226;&#171;&#185;. We show                 downstream of the dpb2&#226;&#171;&#185; open reading frame, respectively, were <br />                                                                      amplified by polymerase chain reaction (PCR). The amplified prod- <br /> that deletion of dpb2 from the yeast genome is lethal and <br />                                                                      ucts were cloned into pKS(&#226;&#171;&#185;)-bluescript on either side of a 1.8-kb <br /> that cells depleted for Dpb2 protein display an S phase              HindIII fragment containing the ura4&#226;&#171;&#185; gene, generating plasmid <br /> delay consistent with an initiation defect. Moreover, we             pKS(&#226;&#171;&#185;)&#226;&#338;&#172;dpb2::ura4&#226;&#171;&#185;. Digestion of this plasmid with EcoRI/XhoI <br /> have constructed a temperature-sensitive dpb2 mutant                 yielded a 4.3-kb fragment containing ura4&#226;&#171;&#185; and dpb2&#226;&#171;&#185; flanking <br /> and found that these cells arrest in late G1/early S phase           sequences that was then transformed into the diploid strain GD28. <br /> upon shift to the restrictive temperature. This phenotype            Stable ura4&#226;&#171;&#185; integrants were selected and deletion of dpb2&#226;&#171;&#185; was <br />                                                                      confirmed by Southern blot analysis. <br /> has never been observed for temperature-sensitive mu- <br />                                                                         To generate GD149, dpb2&#226;&#171;&#185; was amplified from the cosmid p8B7 <br /> tants of either Pol &#226;&#65533;&#163; or Pol &#226;&#65533;&#166;, suggesting that Pol &#226;&#8216;&#8364; and            by using forward primer dpb2-g(F) (5&#226;&#172;&#732;-cggcgcatatgaacaattccattacgg- <br /> Dpb2 provide a novel function(s) during DNA replication              3&#226;&#172;&#732;) and the reverse primer dpb2-b(R) (5&#226;&#172;&#732;-gtccggcccgggttctattat- <br /> initiation (Hughes et al., 1992; D&#226;&#8364;&#8482;Urso et al., 1995; MacNeill       tcggggctcag-3&#226;&#172;&#732;), tagged with NdeI and XmaI sites, respectively. The <br /> et al., 1996; Iino and Yamamoto, 1997; Reynolds et al., 1998;        product was then cloned into pRep81 at the NdeI and XmaI sites. The <br /> Reynolds and MacNeill, 1999). Consistent with this no-               resulting plasmid was then digested with PstI and SacI, yielding a <br />                                                                      4.0-kb fragment containing dpb2&#226;&#171;&#185; flanked by the nmt81 promoter <br /> tion, using a chromatin immunoprecipitation (ChIP) as- <br />                                                                      and terminator sequences. This PstI/SacI fragment was then cloned <br /> say, we find that both Dpb2 and Pol &#226;&#8216;&#8364; bind replication               into the integrative vector pJK148, linearized within the leu1&#226;&#171;&#185; gene <br /> origins early in S phase, very near to the time of DNA               and transformed into the diploid strain GD131. Stable leu&#226;&#171;&#185; trans- <br /> replication initiation.                                              formants were selected, induced to sporulate, and haploid progeny <br />  <br />  <br /> 3428                                                                                                       Molecular Biology of the Cell <br />                                                                                                         Dpb2 Binds to Origin DNA in S Phase <br />  <br />  <br /> prototrophic for both uracil and leucine were selected (strain            distilled H2O followed by resuspension in 100 &#226;&#65533;&#174;l of lysis buffer (50 <br /> GD149).                                                                   mM HEPES, pH 7.9; 140 mM NaCl; 1 mM EDTA; 1% Triton X-100; <br />                                                                           0.1% sodium deoxycholate; 1 mM phenylmethylsulfonyl fluoride; <br />                                                                           10 &#226;&#65533;&#174;g/ml aprotinin; 1 &#226;&#65533;&#174;g/ml leupeptin; 1 &#226;&#65533;&#174;g/ml pepstatin). Glass <br /> Epitope-tagging of Chromosomal dpb2&#226;&#180;&#8482;                                      beads (0.5 g) were then added. Cells were then broken with three <br /> Four tandem copies of the FLAG epitope were first introduced in           20-s pulses in a Savant FP120 bead beater with intermittent chilling <br /> frame at the amino terminus of dpb2&#226;&#171;&#185; in pRep81-dpb2&#226;&#171;&#185; as follows.          on ice. After addition of 900 &#226;&#65533;&#174;l of lysis buffer, the cell lysate was <br /> First, two annealed oligonucleotides of the following sequences,          transferred to a new tube and sonicated until the cellular DNA was <br /> 5&#226;&#172;&#732;-tatggactacaaggacgacgatgacaaggattacaaagatgacgacgataagct-3&#226;&#172;&#732; and          sheared to an average <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> of 0.5&#226;&#8364;&#8220;1 kb. The lysate was then <br /> 5&#226;&#172;&#732;-taagcttatcgtcgtcatctttgtaatccttgtcatcgtcgtccttgtagtcca-3&#226;&#172;&#732;,   were      clarified by centrifugation (12,000g, 15 min) at 4&#194;&#176;C, and 90% of the <br /> cloned into the NdeI site of the plasmid pRep1-dpb2&#226;&#171;&#185;. The transfor-       supernatant was incubated with anti-FLAG or anti-Mcm6p antibod- <br /> mants were screened for the correct orientation of the insertion,         ies bound to magnetic beads (Dynal Biotech, Oslo, Norway). Puri- <br /> resulting in the fusion of two copies of FLAG in frame with dpb2&#226;&#171;&#185;.        fication of DNA from the immunoprecipitates and the remaining <br /> Consequently, the insertion also led to the elimination of the previ-     10% of supernatant (Input control) was performed as described <br /> ous NdeI site at the beginning of the dpb2&#226;&#171;&#185; open reading frame and        previously (Strahl-Bolsinger et al., 1997). Purified DNA was dis- <br /> the simultaneous introduction of a new NdeI site at the beginning of      solved in 30 &#226;&#65533;&#174;l of double distilled H2O and 1 &#226;&#65533;&#174;l of each DNA sample <br /> the two copies of FLAG. The same procedure was repeated to                (IP or Input DNA) was used for each PCR reaction. The nucleotide <br /> introduce another two copies of FLAG in frame with the 2xflag-            sequences of the PCR primers used were as follows: ars2004-F, <br /> dpb2&#226;&#171;&#185;, resulting in the plasmid pRep1-4xflag-dpb2&#226;&#171;&#185;. Then, &#226;&#172;&#402;1 kb of        5&#226;&#172;&#732;-atggtagatggagaaacggg-3&#226;&#172;&#732;; ars2004-R, 5&#226;&#172;&#732;-cacggcatctttcttcacga-3&#226;&#172;&#732;; <br /> genomic DNA immediately upstream of the ATG (including the                ars3002-F, 5&#226;&#172;&#732;-ttggcgctaaacaatctctg-3&#226;&#172;&#732;; ars3002-R, 5&#226;&#172;&#732;-tccttgtcgaactcaat- <br /> dpb2&#226;&#171;&#185; promoter sequences) was amplified from cosmid p8B7 by               tgc-3&#226;&#172;&#732;; nonARS-F, 5&#226;&#172;&#732;-tcgaagatcctaccgctttc-3&#226;&#172;&#732;; nonARS-R, 5&#226;&#172;&#732;-gattca- <br /> using primers tagged with XhoI and NdeI restriction sites. The            cataacccgctagc-3&#226;&#172;&#732;. The ars2004, ars3002, and nonARS primers were <br /> amplified product was then combined with the NdeI/XmaI frag-              used at 0.3, 0.3, and 0.5 &#226;&#65533;&#174;M, respectively. Reaction conditions for the <br /> ment of pRep1-4xflag-dpb2&#226;&#171;&#185;, and in a triple-way ligation, cloned into     PCR were as described by Ogawa et al., (1999): 95&#194;&#176;C, 9 min; 30 cycles <br /> pKS-bluescript at the XhoI and XmaI sites. The resulting plasmid,         of (95&#194;&#176;C, 1 min; 53&#194;&#176;C, 1 min; 72&#194;&#176;C, 2 min); 72&#194;&#176;C, 7 min; 4&#194;&#176;C, &#226;&#172;&#65533;. <br /> called pKS(&#226;&#171;&#185;)-4xflag-dpb2&#226;&#171;&#185; was then digested BamHI/SmaI to de-            Amplified products were separated on a 2.5% agarose gel and <br /> lete the C-terminal half of dpb2&#226;&#171;&#185;, which was then replaced with a         stained with 0.5 &#226;&#65533;&#174;g/ml ethidium bromide. Gel images and quanti- <br /> BamHI/EcoRV fragment containing the sup3-5 gene. This plasmid,            fication of signals were analyzed using an AlphaImager 2000 (Alpha <br /> called pKS(&#226;&#171;&#185;)-4xflag-dpb2N(sup3-5) was linearized at the BglII site       Innotech, San Leandro, CA). <br /> within the dpb2&#226;&#171;&#185; gene and transformed into strain 567. Replacement <br /> of the wild-type dpb2&#226;&#171;&#185; gene with 4xflag-dpb2&#226;&#171;&#185; was confirmed by <br /> both PCR and Western blot analysis of 4x-FLAG Dpb2 by using               RESULTS <br /> anti-FLAG monoclonal antibodies (our unpublished data). <br />                                                                           S. pombe dpb2&#226;&#180;&#8482; Is Essential for Cell Viability <br /> Epitope Tagging of cdc20C1                                                We have identified a gene in the S. pombe genome database <br /> To tag cdc20C1 gene with 4xflag, a fragment containing the nmt1           that is predicted to encode a protein with 31% identity to S. <br /> promoter and 4xflag epitope was amplified from pRep1-4xflag-dpb2&#226;&#171;&#185;         cerevisiae DPB2 and human DPE2 (Figure 1). To test whether <br /> by using the primers nmtP-1F(PstI) (5&#226;&#172;&#732; agcttgcatgccctgcaggtcg 3&#226;&#172;&#732;)         this gene, which we have named dpb2&#226;&#171;&#185;, is essential for cell <br /> and 4xFLAG-R(XhoI) (5&#226;&#172;&#732; ggtaactcgagcaatggaattgttcataag 3&#226;&#172;&#732;). This           viability, we replaced the complete open reading frame with <br /> fragment was then cloned at the PstI and XhoI sites in pRep3X. The        a single copy of the ura4&#226;&#171;&#185; gene, generating the diploid strain <br /> resulting plasmid was named pRep3X-4xflag. A 2.9-kb XhoI/XmaI             GD131. Replacement of dpb2&#226;&#171;&#185; with ura4&#226;&#171;&#185; was confirmed by <br /> fragment encoding the C-terminal half of Pol &#226;&#8216;&#8364; from amino acid            Southern blot analysis (Figure 2, A and B). The diploid strain <br /> position 1319 &#226;&#8364;&#8220;2199 (Cdc20C1) was cloned into the plasmid pRep3X-         GD131 was induced to sporulate, and the germinating <br /> 4xflag. Finally, a 5.1-kb PstI/SacI fragment from plasmid pRep3X- <br />                                                                           spores were subjected to tetrad analysis. Spores deleted for <br /> 4xflag-cdc20C1 was cloned into pJK148, generating pJK148-nmt1&#226;&#8364;&#8220; <br /> 4xflag-cdc20C1, which was then integrated at the leu1 locus as            dpb2&#226;&#171;&#185; fail to form colonies, demonstrating that dpb2&#226;&#171;&#185; is <br /> described previously (Keeney and Boeke, 1994).                            essential for cell viability (Figure 2C). Microscopic examina- <br />                                                                           tion of germinating spores reveals that cells deleted for <br />                                                                           dpb2&#226;&#171;&#185; undergo one to two rounds of cell division before <br /> Site-directed Mutagenesis of dpb2&#226;&#180;&#8482; <br />                                                                           arresting with an elongated &#226;&#8364;&#339;cdc&#226;&#8364;&#65533; phenotype (our unpub- <br /> To delete amino acid residues 413&#226;&#8364;&#8220; 417 in Dpb2, the N- and C-             lished data), indicating that dpb2&#226;&#171;&#185; is essential for normal cell <br /> terminal halves of dpb2&#226;&#171;&#185; were PCR amplified from plasmid pRep81-          cycle progression. <br /> dpb2&#226;&#171;&#185;, by using primers dpb2-g(F) with dpb2&#226;&#338;&#172;413-417-R (5&#226;&#172;&#732;-cccattt- <br /> gtcgtccaagggacaaatatgaattttgtcttttcgca-3&#226;&#172;&#732;) and dpb2-b(R) with <br /> dpb2&#226;&#338;&#172;413-417-F (5&#226;&#172;&#732;-tgcgaaaagacaaaattcatatttgtcccttggacgacaaatggg-         Dpb2 Is Required for Normal S Phase Progression <br /> 3&#226;&#172;&#732;), respectively. The two amplified products were combined and <br /> used as a template for PCR amplification of dpb2&#226;&#338;&#172;413-417 using            Based on the observations that budding yeast DPB2 is re- <br /> primers dpb2-g(F) and dpb2-b(R). The gene was then cloned into            quired for chromosomal DNA replication (Araki et al., <br /> pRep81 at the NdeI and XmaI sites, generating pRep81-dpb2&#226;&#338;&#172;413-            1991a), we anticipated that S. pombe cells lacking dpb2&#226;&#171;&#185; <br /> 417. The PstI/SacI fragment from pRep81-dpb2&#226;&#338;&#172;413-417 was then             would also be defective for DNA synthesis. To test this <br /> cloned into pJK148 and integrated into strain GD131 as described          possibility, we first analyzed DNA content in &#226;&#338;&#172;dpb2 germi- <br /> for the generation of strain GD149 (see above).                           nating spores by FACS. Although spores lacking dpb2&#226;&#171;&#185; pro- <br />                                                                           ceed through S phase more slowly (our unpublished data), <br /> ChIP Analysis                                                             DNA synthesis seemed to be completed before cell cycle <br /> The ChIP assay was performed as described previously (Ogawa et            arrest, suggesting that sufficient amounts of Dpb2 protein <br /> al., 1999) with minor modifications. Fifty milliliters of fission yeast   were present (presumably contributed by the mother cell <br /> cells (107/ml) were cross-linked in 1% formaldehyde at room tem-          during sporulation) to support limited amounts of DNA <br /> perature for 30 min. Cells were washed once with 40 ml of double          synthesis. Therefore, we sought an alternative method to <br />  <br />  <br /> Vol. 14, August 2003                                                                                                                        3429 <br />  W. Feng et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Protein sequence alignment of S. pombe dpb2&#226;&#171;&#185;, S. cerevisiae DPB2, and human DPE2. Fission yeast dpb2&#226;&#171;&#185; shares 31% identity with <br /> both the human and S. cerevisiae genes. Identical residues are indicated in black, whereas homologous residues are indicated in gray. <br /> Sequence alignment was performed using ClustalW 1.8. <br />  <br />  <br />  <br />  <br /> analyze DNA replication in the absence of Dpb2 protein. To            culture was treated with thiamine to inhibit dpb2&#226;&#171;&#185; expres- <br /> this end, we constructed a &#226;&#338;&#172;dpb2 strain containing an inte-           sion. After arrest in G0/G1, cells were stimulated to reenter <br /> grated copy of dpb2&#226;&#171;&#185; under the control of the thiamine                the cell cycle by addition of fresh media containing nitrogen <br /> repressible nmt81 promoter (strain GD149). Cells were                 with or without thiamine (&#226;&#171;&#185; or &#226;&#171;&#186; thiamine). Cells were <br /> grown in the absence of thiamine for several generations              collected every 30 min and DNA content analyzed by FACS. <br /> and then split into two cultures, one of which was treated            As shown in Figure 3C, compared with the control (&#226;&#171;&#186;thia- <br /> with thiamine to repress dpb2&#226;&#171;&#185; expression. FACS analysis              mine), cells treated with thiamine (&#226;&#171;&#185;thiamine) proceed <br /> demonstrates that cells grown in the absence of thiamine              through S phase more slowly. However, these cells still seem <br /> display a 2C DNA content peak typical of exponentially                to complete a substantial amount of DNA replication. The <br /> growing S. pombe cells (Figure 3A, left). In contrast, the            persistence of the 1C DNA content peak in cells depleted for <br /> culture treated with thiamine gradually accumulated cells             Dpb2 (compare &#226;&#171;&#185;thiamine and &#226;&#171;&#186;thiamine at 3.5 h) suggest <br /> with less than 2C DNA content, suggesting DNA replication             that initiation of DNA replication is defective in these cells. <br /> was impaired (Figure 3A, right). Eventually these cells               These data are consistent with Dpb2 being essential for <br /> ceased to divide and elongated with a cdc phenotype (Figure           initiation and possibly elongation of DNA synthesis. <br /> 3B). Moreover, 4,6-diamidino-2-phenylindole staining of nu- <br /> clei revealed that chromosomes became fragmented after <br /> incubation in the presence of thiamine, suggesting that Dpb2 <br />                                                                       A Temperature-sensitive dpb2 Mutant Arrests in <br /> might be important for maintaining chromosomal integrity              Early S Phase after Shift to the Restrictive <br /> after DNA synthesis arrest (Figure 3B, 40 h).                         Temperature <br />    To further examine the role of Dpb2 during chromosomal             The experiments described above were designed to test <br /> DNA replication, Dpb2 protein was depleted in cell cycle-             whether Dpb2 is required for DNA replication. Although <br /> synchronized cultures. Exponentially growing GD149 cells              the data support this hypothesis, the inability to deplete all <br /> grown in the absence of thiamine were synchronized in                 the cellular Dpb2 made it difficult to address its precise role <br /> G0/G1 by nitrogen starvation. During starvation, half of the          in S phase. Therefore, we decided to generate a dpb2 tem- <br />  <br />  <br /> 3430                                                                                                       Molecular Biology of the Cell <br />                                                                                               Dpb2 Binds to Origin DNA in S Phase <br />  <br />  <br />  <br /> Figure 2. dpb2&#226;&#171;&#185; is essential for cell viability. <br /> (A) Strategy for deleting dpb2&#226;&#171;&#185; by gene re- <br /> placement. The 1.76-kb open reading frame of <br /> dpb2&#226;&#171;&#185; was replaced with the 1.8-kb ura4&#226;&#171;&#185; <br /> gene. A HindIII fragment containing the ura4&#226;&#171;&#185; <br /> gene was cloned into pBluescript. A 1.2- and <br /> 2-kb fragment corresponding to sequences <br /> upstream and downstream of the dpb2&#226;&#171;&#185; open <br /> reading frame were cloned into the EcoRI/ <br /> HindIII and HindIII/XhoI sites of pBluescript- <br /> ura4&#226;&#171;&#185;, respectively. The resulting plasmid <br /> was digested with EcoRI and XhoI and the <br /> excised fragment was transformed into the <br /> diploid strain GD28. Positive transformants <br /> were selected on minimal medium minus ura- <br /> cil. (B) Southern blot analysis of heterozygous <br /> diploid &#226;&#171;&#185;/&#226;&#338;&#172; (lanes 1 and 2) and homozygous <br /> diploid &#226;&#171;&#185;/&#226;&#171;&#185;. DNA was digested by NdeI and <br /> Bsu36I and probed with the EcoRI and HindIII <br /> fragment. (C) Tetrad analysis of the heterozy- <br /> gous diploid strain (&#226;&#171;&#185;/&#226;&#338;&#172;) revealed a two via- <br /> ble:two nonviable segregation as expected for <br /> an essential gene disruption. All viable spores <br /> were confirmed to be ura&#226;&#171;&#186;. <br />  <br />  <br />  <br /> perature-sensitive mutant that would allow more rapid and           Dpb2 Protein Binds to DNA Replication Origins <br /> complete inactivation of the protein. To select a target area       Early in S Phase <br /> for mutagenesis, we compared all known Dpb2 proteins to <br /> identify conserved amino acid residues. As previously re-           If Dpb2 is required for initiation of DNA replication, then <br /> ported, the regulatory B-type subunits of eukaryotic DNA            this protein should interact with replication origins at the <br /> polymerases, which includes Dpb2, belong to a protein su-           beginning of S phase. To test this possibility, we examined <br /> perfamily (Makiniemi et al., 1999) that contains a cal-             whether S. pombe Dpb2 associates with origins of replication <br /> cineurin-like phosphatase domain similar to those found in          during the cell cycle by using a ChIP assay coupled with <br /> archaeal X-type DNA polymerases (Aravind and Koonin,                PCR. For these experiments, we N-terminally tagged the <br /> 1999). Although some of the key residues required for phos-         endogenous copy of dpb2&#226;&#171;&#185; with four tandem copies of the <br /> phatase activity are no longer present in the eukaryotic            FLAG epitope. This strain also includes the temperature- <br /> proteins (Aravind and Koonin, 1999), some of the remaining          sensitive cdc25 allele to allow synchronization in G2 <br /> conserved residues might be important to stabilize the pro-         (GD254). Consistent with data from other laboratories <br /> tein. In fact, several temperature-sensitive mutations found        (Ogawa et al., 1999; Wuarin et al., 2002), DNA replication <br /> in eukaryotic DNA polymerase B-type subunits have been              begins &#226;&#172;&#402;60 &#226;&#8364;&#8220; 80 min after release from the cdc25 block (Fig- <br /> localized to this region of the protein (Makiniemi et al., 1999).   ure 5A). It should be pointed out that DNA replication <br /> For example, in the HYS2/POL31 gene of S. cerevisiae, a             begins before cell separation after release from the cdc25 <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> converting an Asp to Asn at amino acid position            block point, and therefore initiation of DNA replication <br /> 304 renders the protein temperature sensitive (Hashimoto et         correlates with a shift from 2C to 4C DNA content. As cells <br /> al., 1998).                                                         separate, most cells have completed DNA replication, result- <br />    Therefore, we set out to mutate key residues in the corre-       ing in the accumulation of cells with a 2C DNA content peak <br /> sponding domain (motif III) of S. pombe dpb2&#226;&#171;&#185; (Figure 4A).          at later times in the experiment. We used primers previously <br /> Based on the analysis of hys2 D304N mutation, we first              designed by Ogawa et al., 1999 to amplify sequences specific <br /> mutated the corresponding Asp417 in dpb2&#226;&#171;&#185; to either Asn or <br />                                                                     to origin DNA (ars2004 or ars3002) and to nonARS DNA, <br /> Ala by site-directed mutagenesis. Neither of these mutations <br />                                                                     located &#226;&#172;&#402;25 kb centromere-proximal to ars2004 on chromo- <br /> had any effect on yeast cell cycle progression or cell viability <br /> (our unpublished data). Likewise, mutation of the highly            some II (Ogawa et al., 1999). Similar primers have been <br /> conserved Pro413 to Ala had no effect on Dpb2 function (our         successfully used for the analysis of Orc1 and Mcm binding <br /> unpublished data). Finally, deletion of all five residues from      to origin DNA (Ogawa et al., 1999; Takahashi and Masukata, <br /> amino acid position 413&#226;&#8364;&#8220; 417 rendered the protein tempera-          2001; Takahashi et al., 2003). These primer sets were used in <br /> ture sensitive. As shown in Figure 4B, mutant dpb2-d1               quantitative PCR on DNA prepared either from immuno- <br /> (&#226;&#338;&#172;413&#226;&#8364;&#8220; 417) failed to form colonies at the restrictive temper-      precipitated chromatin fractions or from total DNA isolated <br /> ature of 36&#194;&#176;C. More importantly, after shift to the restrictive     from whole cell soluble extracts. To control for loading <br /> temperature, dpb2-d1 divides once before arresting with a           differences or variance in amplification from the different <br /> near 1C DNA content (Figure 4, C and D), suggesting that            primer sets, we normalized the relative enrichment at ARS <br /> Dpb2 is required during initiation, or at least during an early     (autonomous replicating sequence) sequences to the nonARS <br /> step in replication elongation (see DISCUSSION).                    PCR product (Figure 5B). <br />  <br />  <br /> Vol. 14, August 2003                                                                                                        3431 <br />  W. Feng et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 3. Cells depleted for Dpb2 are de- <br />                                                                                         fective in S phase progression. (A) A culture <br />                                                                                         of haploid GD149 cells was grown to log <br />                                                                                         phase, the culture was then diluted and one- <br />                                                                                         half was treated with 10 &#226;&#65533;&#174;g/&#226;&#65533;&#174;l thiamine to <br />                                                                                         repress dpb2&#226;&#171;&#185; gene expression. Samples <br />                                                                                         were collected at the indicated times and <br />                                                                                         DNA content analyzed by flow cytometry. <br />                                                                                         (B) Cells stained with the DNA binding dye <br />                                                                                         4,6-diamidino-2-phenylindole after incuba- <br />                                                                                         tion with thiamine at 0, 23, and 40 h. Arrows <br />                                                                                         indicate abnormal nuclear morphology, in- <br />                                                                                         cluding missegregated chromosomes and <br />                                                                                         anucleate cells. (C) DNA replication initia- <br />                                                                                         tion is delayed in cell cycle synchronized <br />                                                                                         cultures depleted for Dpb2. Wild-type (972) <br />                                                                                         or GD149 cells were grown in the absence of <br />                                                                                         nitrogen to arrest cells in G1. During nitro- <br />                                                                                         gen starvation, thiamine was added (10 &#226;&#65533;&#174;g/ <br />                                                                                         &#226;&#65533;&#174;l) to the media to repress transcription <br />                                                                                         from the nmt81 promoter. After 20 h or <br />                                                                                         when &#226;&#172;&#381;90% of the cells showed a G1 arrest, <br />                                                                                         cells were induced to reenter the cell cycle <br />                                                                                         by addition of fresh media containing nitro- <br />                                                                                         gen, as well as thiamine to maintain tran- <br />                                                                                         scriptional repression of dpb2&#226;&#171;&#185;. Cells were <br />                                                                                         collected every 30 min for 6 h and the DNA <br />                                                                                         content analyzed by flow cytometry. The <br />                                                                                         positions of 1C and 2C DNA content are <br />                                                                                         indicated. <br />  <br />  <br />  <br />    Our data demonstrate that Dpb2 binds preferentially to          phase. Therefore, considering that the amount of Dpb2 as- <br /> origins of replication &#226;&#172;&#402;60 min after release from the cdc25        sociation to chromatin detected in cdc10-arrested cells is <br /> block (Figure 5C), very near to the time of DNA replication        similar to the amount detected in either G2-arrested (cdc25) <br /> initiation (Figure 5A). Although the level of enrichment           or exponentially growing 972 cells, we conclude that very <br /> (three- to fivefold) of Dpb2 binding to ARS elements is            little, if any, Dpb2 protein is bound to chromatin in the <br /> similar to that observed for Mcm6 (Figure 5D), a component         absence of Cdc10. <br /> of the prereplicative complex (preRC), the peak of Dpb2               In a previous study, we demonstrated that fission yeast <br /> association with origin DNA seems to occur slightly later          cells are viable in the absence of the N-terminal, but not <br /> than that of Mcm6. Mcm6 is thought to bind to origin DNA           C-terminal half of the enzyme (Feng and D&#226;&#8364;&#8482;Urso, 2001). This <br /> during the later stages of mitosis or in early G1 (Ogawa et al.,   result was surprising because the N terminus of Pol &#226;&#8216;&#8364; con- <br /> 1999). Therefore, our data are consistent with Dpb2 binding        tains all the domains necessary for polymerase activity. <br /> to replication origins in late G1 or early S phase after assem- <br />                                                                    Thus, we concluded that the noncatalytic C-terminal half of <br /> bly of the preRC. To test whether Dpb2 can interact with <br />                                                                    Pol &#226;&#8216;&#8364; is likely to be required during assembly of the IC. To <br /> origin DNA in the absence of preRCs, we examined Dpb2 <br /> binding to origin DNA in cdc10-129 cells at 36&#194;&#176;C. Cdc10 is         provide additional evidence in support of this hypothesis, <br /> required for expression of the MCM loading factors Cdc18           we set out to investigate whether the C termini of Pol &#226;&#8216;&#8364; can <br /> and Cdt1, and in cdc10-V50 cells at 36&#194;&#176;C, Mcm4 fails to            bind specifically to origin DNA in cells lacking the N-termi- <br /> associate with origin-associated DNA sequences (Wuarin et          nal sequences. For these experiments, we used the GD273 <br /> al., 2002). No specific association of Dpb2 to ARS-associated      strain that contains four tandem copies of the FLAG epitope <br /> chromatin was observed in cdc10-129 cells at 36&#194;&#176;C, suggest-        N-terminally tagged to Pol &#226;&#8216;&#8364;-Cterm under the control of the <br /> ing that the binding of Dpb2 to origin DNA is dependent on         thiamine repressible nmt1 promoter (Feng et al., 1989). After <br /> prior assembly of the preRC (Figure 6A). It should be noted        release from the cdc25 block, we observed that the C-termi- <br /> that ChIP analysis is not sensitive enough to detect enrich-       nal half of Pol &#226;&#8216;&#8364; binds preferentially to ARS associated <br /> ment of Dpb2 or Mcm6 binding to origin DNA in exponen-             sequences at 60 min postrelease, similar to the results ob- <br /> tially growing populations where very few cells are in S           tained for Dpb2 (Figure 6B). <br />  <br />  <br /> 3432                                                                                                  Molecular Biology of the Cell <br />                                                                                                             Dpb2 Binds to Origin DNA in S Phase <br />  <br />  <br /> Figure 4. Temperature-sensitive mutant <br /> dpb2-d1 arrests in late G1-early S phase after <br /> shift to the restrictive temperature. (A) Amino <br /> acid sequence alignment of the most highly <br /> conserved region (motif III) of the putative <br /> phosphatase domain with a protein super <br /> family consisting of the B subunits of eukary- <br /> otic DNA polymerases and selected members <br /> of the regulatory subunits of the archael X- <br /> family DNA polymerases. The location of the <br /> conserved domains (motif I-IV) is shown for <br /> S. pombe Dpb2. Asterisks indicate the amino <br /> acids 413&#226;&#8364;&#8220; 417, deleted in the mutant dpb2-d1. <br /> Identical residues are shown in black. Conser- <br /> vative changes are indicated in gray. Species <br /> abbreviations are as follows: Af, Archaeoglobus <br /> fulgidus; Hs, Homo sapiens; Mj, Methanococcus <br /> jannaschii; Mm, Mus musculus; Mta, Methano- <br /> thermobacter thermoautotrophicum; Pt, Pyrococ- <br /> cus furiosus; Sc, S. cerevisiae; and Sp, S. pombe. <br /> (B) Cells carrying wild-type dpb2&#226;&#171;&#185; or dpb2-d1 <br /> (&#226;&#338;&#172;413&#226;&#8364;&#8220; 417) were streaked on minimal me- <br /> dium and incubated at 25&#194;&#176;C or 36&#194;&#176;C. Plates <br /> were photographed after 4 d. (C) Cell number <br /> analysis in dpb2-d1 cells after shift to the re- <br /> strictive temperature. Wild-type 972 cells or <br /> dpb2-d1 (&#226;&#338;&#172;413&#226;&#8364;&#8220; 417) were grown at 25&#194;&#176;C in <br /> minimal medium to mid-log phase and <br /> shifted to 36&#194;&#176;C for 8 h. Please note that after <br /> shift to the restrictive temperature, dpb2-d1 is <br /> limited to one population doubling (typical of <br /> an early S phase arrest), whereas the wild- <br /> type culture continues to grow exponentially. <br /> (D) dpb2-d1 cells arrest with a 1C DNA content after shift to restrictive temperature. dpb2&#226;&#171;&#185; (top) and dpb2-d1 (bottom) were grown at 25&#194;&#176;C overnight. <br /> The following day, cells were shifted to the restrictive temperature of 36&#194;&#176;C, collected at the indicated times, and measured for DNA content by flow <br /> cytometry. The position of 1C and 2C DNA content are indicated. <br />  <br />  <br />  <br /> DISCUSSION                                                                   has never been reported for mutants defective in other DNA <br />                                                                              polymerases or their associated subunits, all of which arrest <br /> Previously, we have demonstrated that the large catalytic                    with a near 2C DNA content (Hughes et al., 1992; Waseem et <br /> subunit of DNA polymerase epsilon, encoded by cdc20, is                      al., 1992; MacNeill et al., 1996; MacNeill and Fantes, 1997). <br /> essential for chromosomal DNA replication in S. pombe.                          Regardless of whether Pol &#226;&#8216;&#8364; is directly involved in DNA repli- <br /> Based on the observation that temperature-sensitive mu-                      cation initiation, it is clear from our studies that Cdc20 and Dpb2 <br /> tants of cdc20 arrest in late G1 or early S phase, we suggested <br />                                                                              provide an essential function that does not rely on their ability to <br /> that Pol &#226;&#8216;&#8364; is required for an early step in DNA replication <br />                                                                              synthesize DNA. One possibility is that Pol &#226;&#8216;&#8364; binds replication <br /> (D&#226;&#8364;&#8482;Urso and Nurse, 1997). Interestingly, the essential fea- <br />                                                                              origins early in S phase and is important for either recruiting or <br /> tures of the Pol &#226;&#8216;&#8364; catalytic subunit do not reside in the <br />                                                                              stabilizing the interaction of other replication proteins to DNA. <br /> N-terminal half of the enzyme that contains the highly con- <br /> served DNA polymerase domains, but rather in the C-ter-                      Along these lines, it is interesting that in budding yeast, Pol &#226;&#8216;&#8364; can <br /> minal half of the enzyme where there are no known catalytic                  interact with replication origins in the absence of Pol &#226;&#65533;&#163; (Masumoto <br /> functions (Fuss and Linn, 2002). The observation that mu-                    et al., 2000). Moreover, association of Pol &#226;&#65533;&#163; with ARS-associated <br /> tants in Pol &#226;&#8216;&#8364; arrest in late G1 or early S phase combined                   chromatin requires Dpb11, a protein that both genetically and <br /> with the fact that the enzyme&#226;&#8364;&#8482;s polymerase activity is dis-                  physically interacts with Pol &#226;&#8216;&#8364; (Masumoto et al., 2000). Similarly, in <br /> pensable, led us to propose that Pol &#226;&#8216;&#8364; might be important for                Xenopus cell-free extracts, association of Pol &#226;&#8216;&#8364; with chromatin <br /> assembly of the initiation complex (Feng and D&#226;&#8364;&#8482;Urso, 2001).                  during DNA replication can occur in the absence of Pol &#226;&#65533;&#163; or <br />    Consistent with this hypothesis, we demonstrate that both                 replication protein A (RPA) (Mimura et al., 2000). Based on these <br /> the C-terminal half of Pol &#226;&#8216;&#8364;, and its smaller putative subunit,              findings, Mimura et al. 2000 proposed that Pol &#226;&#8216;&#8364; might associate <br /> Dpb2, bind preferentially to replication origins early in S                  with chromatin before synthesis of RNA primers by Pol &#226;&#65533;&#163;. We are <br /> phase. Furthermore, we demonstrate that like cdc20 mutants                   currently testing whether the binding of Pol &#226;&#65533;&#163; or RPA to replica- <br /> (Feng and D&#226;&#8364;&#8482;Urso, 2001), a temperature-sensitive mutant of                   tion origins occurs before, or is dependent on, the prior loading of <br /> dpb2 arrests with a 1C DNA content after shift to the restric-               Dpb2 or Pol &#226;&#8216;&#8364;. <br /> tive temperature, suggesting that Dpb2 is required for the                      Recent studies using a Xenopus cell-free replication system <br /> onset of S phase. Although it is currently impossible to rule                have suggested that neither Pol &#226;&#8216;&#8364; or Pol &#226;&#65533;&#166; is required for <br /> out that DNA replication initiation occurs in cdc20 or dpb2                  initiation of DNA replication, but both are necessary for effi- <br /> mutants, it is interesting that early S phase arrest phenotype               cient chain elongation (Waga et al., 2001). However, it is unclear <br />  <br />  <br /> Vol. 14, August 2003                                                                                                                            3433 <br />  W. Feng et al. <br />  <br />  <br />  <br />  <br />                                                                                           Figure 5. 4xFLAG-Dpb2 binds ARS ele- <br />                                                                                           ments early in S phase. (A) Analysis of DNA <br />                                                                                           content by flow cytometry indicates that <br />                                                                                           DNA synthesis begins at 60 &#226;&#8364;&#8220; 80 min after re- <br />                                                                                           lease from the G2 block. (B) Relative enrich- <br />                                                                                           ment ratio of immunoprecipitated ARS DNA <br />                                                                                           to nonARS DNA was calculated and plotted <br />                                                                                           from the raw data in C. (C and D) ChIP anal- <br />                                                                                           yses with antibody to the FLAG-epitope for <br />                                                                                           Dpb2 (C) or antibody to Mcm6 (D) were used <br />                                                                                           to measure the levels of Dpb2 and Mcm6 at <br />                                                                                           ARS loci. DNA isolated from the immunopre- <br />                                                                                           cipitated chromatin (IP) or from whole cell <br />                                                                                           soluble extracts (Total DNA) was subjected to <br />                                                                                           PCR to amplify DNA fragments from either <br />                                                                                           ars2004, ars3002 or from nonARS DNA. Please <br />                                                                                           note that the peak of Dpb2 binding to ARS <br />                                                                                           elements occurs 60 min after release from the <br />                                                                                           G2 block (C) in contrast to Mcm6 binding <br />                                                                                           which peaks at 40 min postrelease (D). No <br />                                                                                           significant chromatin binding was observed <br />                                                                                           when cross-linking agent was omitted, if cell <br />                                                                                           lysates were prepared in the absence of <br />                                                                                           FLAG-tagged protein, or if antibodies were <br />                                                                                           excluded from the immunoprecipitation reac- <br />                                                                                           tion (our unpublished data). <br />  <br />  <br /> whether the limited amounts of DNA synthesis detected in the         these foci might represent sites where replication complexes <br /> absence of Pol &#226;&#65533;&#166; or Pol &#226;&#8216;&#8364; in this system represent true initiation   are being assembled. The adjacent PCNA foci might represent <br /> events, or simply reflect inappropriate binding and synthesis        active replication forks that have migrated away from the <br /> by Pol &#226;&#65533;&#163; in the absence of other polymerases.                        assembly sites. Interestingly, although Pol &#226;&#8216;&#8364; foci did not colo- <br />   Recently, it was reported that in human cells, Pol &#226;&#8216;&#8364; fails to      calize with PCNA during early S phase, they did seem to <br /> colocalize with PCNA or active replication foci early in S phase.    colocalize with active replication foci during late S phase. Con- <br /> Although Pol &#226;&#8216;&#8364; foci are observed as early as late G1, in early S     sidering that most late-replicating DNA regions are believed to <br /> phase these foci do not colocalize with sites of 5-bromo-de-         be associated with heterochromatin, it was suggested that Pol <br /> oxyuridine incorporation or proliferating cell nuclear antigen       &#226;&#8216;&#8364; might have a unique role in replicating these regions through <br /> (PCNA). Consistent with our yeast studies, these observations        facilitating chromatin remodeling (Fuss and Linn, 2002). This <br /> suggest that Pol &#226;&#8216;&#8364; may have a unique function in late G1 or          hypothesis is supported by two observations. First, the mouse <br /> early S phase that is independent of DNA synthesis. The au-          homolog of Dpb2, called Dpe2, interacts with SAP18, a <br /> thors speculated that the presence of Pol &#226;&#8216;&#8364; foci in late G1 or       polypeptide that associates with the transcriptional corepressor <br /> early S phase might represent DNA repair foci or inactive            Sin3. These two proteins have been shown to induce repression <br /> replication foci containing Pol &#226;&#8216;&#8364; (Fuss and Linn, 2002). How-        of transcription in reporter assays that might involve recruit- <br /> ever, we would like to offer an alternative explanation that         ment of remodeling factors such as histone deacetylase to <br />  <br />  <br /> 3434                                                                                                     Molecular Biology of the Cell <br />                                                                                                              Dpb2 Binds to Origin DNA in S Phase <br />  <br />  <br />                                                                              tion. We speculate that this function might be related to T <br />                                                                              antigen&#226;&#8364;&#8482;s ability to recruit Pol &#226;&#65533;&#163;/primase and possibly other <br />                                                                              factors to DNA. Therefore, the association of Pol &#226;&#8216;&#8364; and Dpb2 to <br />                                                                              ARS-associated chromatin might represent a key step in the <br />                                                                              assembly of the replication initiation complex. <br />  <br />   <br /> Figure 6. Dpb2 fails to associate with origin DNA in cdc10-129 cells <br /> at 36&#194;&#176;C. (A) Exponentially growing 972 cells (lane 1), G2 arrested          <br /> GD254 cells (lane 2), or G1 arrested GD261 cells (lanes 3 and 4) were         <br /> subjected to ChIP analysis for the presence of chromatin bound Dpb2.         <br /> Either magnetic beads conjugated to anti-FLAG antibodies (lanes 1, 2 <br />                                                                              <br /> and 4) or magnetic beads alone (lane 3) were used for immunoprecipi- <br />                                                                               <br /> tation. The relative ratio of immunoprecipitated DNA to total DNA <br /> was plotted. No enrichment of Dpb2 binding to ars2004 is observed in         <br /> cdc10-129 cells arrested in G1 (lane 4). The levels of binding to either      <br /> non-ARS or ARS-associated chromatin in cdc10 cells is equivalent to       <br /> the level detected in either G2 arrested (lane 2) or exponentially grow-   <br /> ing cells (lane 1). Very little chromatin is precipitated from cdc10 cells   <br /> in the absence of antibodies (lane 3). (B) ChIP analysis using anti-FLAG <br /> antibodies for detection of Pol &#226;&#8216;&#8364;-Cterm at ars2004.                           <br /> chromatin (Wada et al., 2002). Second, human Pol &#226;&#8216;&#8364; p17, a                     <br /> homolog of the Pol &#226;&#8216;&#8364; subunit Dpb4, has been found as a                       <br /> component of human chromatin accessibility complex (CH-                      <br /> RAC), a chromatin-remodeling factor (Poot et al., 2000). <br />                                                                             <br />    Finally, our analysis of Pol &#226;&#8216;&#8364; function in fission yeast may              <br /> provide an explanation for why Pol &#226;&#8216;&#8364; is dispensable for SV40                <br /> viral DNA replication. In the case of SV40, the virus encodes its            <br /> own initiator protein, called T antigen. T antigen has multiple <br />                                                                             <br /> roles in promoting DNA replication, including site directed <br />                                                                              <br /> binding to the origin, ATP-dependent unwinding of DNA <br /> strands, and recruitment of DNA polymerase &#226;&#65533;&#163;/primase and                   <br /> possibly topoisomerase I to the site of initiation (Fanning, 1992;            <br /> Fanning and Knippers, 1992; Simmons et al., 1996). Perhaps                   <br /> similar to the function of the origin recognition complex (ORC)             <br /> and mini-chromosome maintenance complexes (MCMs), the                       <br /> essential nonreplicative function of Pol &#226;&#8216;&#8364; is provided by T                <br /> antigen, and therefore is not required for viral DNA replica-               <br /> </body></html>