<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 13, 3162&#226;&#8364;&#8220;3177, September 2002 <br />  <br />  <br /> An Essential Subfamily Drs2p-related P-Type <br /> ATPases Is Required Protein Trafficking between <br /> Golgi Complex Endosomal/Vacuolar System <br /> Zhaolin Hua, Parvin Fatheddin, Todd R. Graham* <br />  <br /> Department Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235-1634 <br />  <br /> Submitted March 28, 2002; Revised May 23, 2002; Accepted June 5, 2002 <br /> Monitoring Editor: Randy Schekman <br />  <br />  <br />  <br />         The Saccharomyces cerevisiae genome contains genes encoding P-type ATPases poten- <br />         tial aminophospholipid translocases (APTs): DRS2, NEO1, uncharacterized open read- <br />         ing frames named DNF1, DNF2, DNF3 DRS2/NEO1 family. NEO1 only <br />         essential gene APT family functionally distinct DRS2/DNF genes. The <br />         drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172; quadruple mutant inviable, member group <br />         maintain viability, indicating substantial functional overlap the <br />         encoded proteins. We previously implicated Drs2p clathrin function trans-Golgi <br />         network. In study, constructed strains carrying possible viable combinations null <br />         alleles group analyzed defects protein transport. The drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; mutant <br />         grows slowly, massively accumulates intracellular membranes, exhibits substantial defect in <br />         transport alkaline phosphatase vacuole. Transport carboxypeptidase Y the <br />         vacuole perturbed, lesser extent. In addition, dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172; mutant exhibits <br />         defect recycling GFP-Snc1p early endocytic-late secretory pathways. Drs2p and <br />         Dnf3p colocalize trans-Golgi network marker Kex2p, Dnf1p Dnf2p <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />         localize plasma membrane late exocytic early endocytic membranes. We propose that <br />         eukaryotes express multiple APT subfamily members facilitate protein transport multiple <br />         pathways. <br />  <br />  <br /> INTRODUCTION                                                          translocate phosphatidylserine (PS) and/or phosphati- <br />                                                                       dylethanolamine (PE) leaflet membrane <br /> P-type ATPases large family multitransmembrane               bilayer other, responsible concen- <br /> domain, ATP-dependent transporters. The P-type designa-               trating PS PE cytosolic leaflet biological mem- <br /> tion derives covalent aspartyl-phosphate catalytic             branes. <br /> intermediate formed ATP hydrolysis, cou-                 Genome sequencing projects identified large num- <br /> pled conformational changes drive ion transport           ber potential APT subfamily genes eukaryotic organ- <br /> across membranes electrochemical gradient.                 isms, prokaryotic member identified <br /> Based substrate transported, P-type ATPases clas-          yet. There members subfamily Sac- <br /> sified heavy metal ion ATPases, copper trans-          charomyces cerevisiae Schizosaccharomyces pombe <br /> porters mutated patients Menkes Wilson            (Costanzo et al., 2001). The Caenorhabditis elegans genome <br /> disease; nonheavy metal ion ATPases, Ca2&#226;&#171;&#185;-,               contains members, Arabidopsis 11 (Gomes et al., 2000), <br /> H&#226;&#171;&#185;-, Na&#226;&#171;&#185;,K&#226;&#171;&#185;-ATPases (Moller et al., 1996). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> se-               21 members humans. Little <br /> quence analyses, members subfamily P-type               known cellular function APT subfamily of <br /> ATPases distinguished above-men-                   proteins eukaryotes maintain these <br /> tioned types (Catty et al., 1997; Halleck et al., 1998)       genes. However, biological disorders linked <br /> are proposed aminophospholipid translocases (APTs)              attributed genes subfamily. FIC1 mutations <br /> or flippases (Zachowski et al., 1989; Auland et al., 1994; Tang       cause familial intrahepatic cholestasis humans, a <br /> et al., 1996; Gomes et al., 2000; Ujhazy et al., 2001). APTs          defect bile secretion (Thompson Jansen, 2000; Ujhazy <br />                                                                       et al., 2001). The ATP10C gene linked Angelman <br />   Article published online ahead print. Mol. Biol. Cell 10.1091/   syndrome autism patients (Herzing et al., 2001; <br /> mbc.E02&#226;&#8364;&#8220; 03&#226;&#8364;&#8220; 0172. Article publication date www.molbi-     Meguro et al., 2001), mouse ATP10C homolog pfatp <br /> olcell.org/cgi/doi/10.1091/mbc.E02&#226;&#8364;&#8220; 03&#226;&#8364;&#8220; 0172.                         candidate gene increased body fat (Dhar et al., 2000). <br />   * Corresponding author. E-mail address: tr.graham@vanderbilt.edu.   In addition, PDE1 gene pathogenic factor rice <br />  <br />  <br /> 3162                                                                                  &#194;&#169; 2002 The American Society Cell Biology <br />                                                                                                   Drs2-related ATPases Protein Traffic <br />  <br />  <br /> blast fungus, Magnaporthe grisea (Balhadere Talbot, <br /> 2001), ALA1 gene Arabidopsis required             Table 1. Similarity identity DRS2/NEO1 genes <br /> growth low temperature (Gomes et al., 2000). <br />                                                                                 DRS2         DNF1         DNF2          DNF3         NEO1 <br />    The member APT subfamily identified <br /> genetically cloned DRS2 S. cerevisiae (yeast)          DRS2                       48           48            45           48 <br /> (Ripmaster et al., 1993). Drs2p 47% identical amino acid      DNF1          31                        83            45           43 <br /> sequence mammalian chromaffin granule ATPase II              DNF2          31           69                         45           43 <br /> (Tang et al., 1996), proposed PS translocase     DNF3          29           28           27                         39 <br /> characterized intact chromaffin granules (Zachowski et         NEO1          30           26           26            23 <br /> al., 1989). DRS2 initially identified genetic screen for <br /> mutants exhibit cold-sensitive defect ribosome syn-       Upper right lists similarity DRS2/NEO1 genes (%), lower <br /> thesis shown nonessential gene,            left lists identity (%). (Adapted Halleck et al., 1998). <br /> disruption DRS2 causes strong cold-sensitive growth <br /> defect (Ripmaster et al., 1993). Subsequently, identified <br /> DRS2 genetic screen mutations syn- <br /> thetically lethal arf1 (Chen et al., 1999).                    Drs2p higher temperatures, best candidates <br />    ADP-ribosylation factor (ARF) small GTP-binding             Drs2p homologs encoded yeast ge- <br /> protein functions initiating COPI clathrin-            nome. A phylogenetic sequence comparison 16 <br /> coated vesicle (CCV) budding studied                 P-type ATPases encoded yeast genome suggested <br /> (Donaldson Jackson, 2000). The drs2&#226;&#338;&#172; <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>null allele</span>        proteins belong APT subfamily <br /> synthetically lethal clathrin heavy chain temperature-         (Catty et al., 1997). Two genes (DRS2 NEO1) <br /> sensitive (ts) alleles ts alleles COPI subunits     identified genetically, remaining <br /> (Chen et al., 1999). These genetic interactions DRS2,       genes (YER166W, YDR093W, YMR162C) have <br /> ARF, clathrin suggested protein transport function        characterized. We designated three <br /> Drs2p trans-Golgi network (TGN). In fact, Drs2p lo-          uncharacterized genes DNF1 (YER166W), DNF2 <br /> calizes TGN drs2&#226;&#338;&#172; mutant exhibits TGN                (YDR093W), DNF3 (YMR162C) DRS2/NEO1 fam- <br /> defects comparable exhibited clathrin mu-             ily. The similarity proteins range 39% for <br /> tants. These include 1) mislocalization TGN proteins,        DNF3 NEO1 83% DNF1 DNF2 (Table 1). We <br /> resulting defect proteolytic processing pro-&#226;?&#163;-       constructed strains carrying possible combina- <br /> factor; 2) accumulation aberrant Golgi cisternae;        tions drs2&#226;&#338;&#172; dnf&#226;&#338;&#172; null alleles that <br /> 3) deficiency CCVs subcellular fractions (Chen et al.,      DRS2 DNF genes constitute essential gene <br /> 1999). These phenotypes severe non-            family exert overlapping functions protein trans- <br /> permissive growth temperature. Recently, however,                port. Drs2p Dnf1p redundant functions pro- <br />                                                                     tein transport TGN vacuole, Dnf1p, <br /> have drs2&#226;&#338;&#172; mutant exhibits defect per- <br />                                                                     Dnf2p, Dnf3p redundant functions recycling <br /> missive growth temperatures generating specific class of <br />                                                                     v-SNARE Snc1p endosomes TGN. The <br /> CCVs carrying invertase acid phosphatase plasma <br />                                                                     potential role proteins generating membrane <br /> membrane (Gall et al., 2002). <br />                                                                     asymmetry focus report. <br />    The ribosome synthesis defect observed originally in <br /> drs2&#226;&#338;&#172; likely secondary effect protein transport <br /> defect. Many secretory mutants globally down-regulate               MATERIALS AND METHODS <br /> ribosomal protein synthesis protein kinase C <br /> signal transduction pathway (Nierras Warner, 1999;              Media, Strains, Plasmids <br /> Li et al., 2000). In addition, severe protein transport         Yeast grown standard rich medium (YPD) SD minimal <br /> defects observed immediately shifting cells        media containing required supplements (Sherman, 1991). The cal- <br /> nonpermissive temperature, hours                cofluor white (CW) sensitivity test YPD 2% agar plates <br /> required protein synthesis defect observed              containing 10, 30, 50 &#226;?&#174;g/ml calcofluor white (F6259; Sigma- <br /> (Chen et al., 1999). The drs2&#226;&#338;&#172; mutant re-             Aldrich, St. Louis, MO). <br /> ported exhibit defect APT activity plasma               Yeast strains used study listed Table 2. The original <br /> membrane (Tang et al., 1996; Gomes et al., 2000),          DRS2/NEO1 family gene deletion strains Saccharomy- <br />                                                                     ces genome deletion project (Winzeler et al., 1999) pur- <br /> others report defect activity (Siegmund et al.,          chased American Type Culture Collection (Manassas, VA). <br /> 1998; Marx et al., 1999). Moreover, localization             The drs2/dnf combination deletion strains generated stan- <br /> Drs2p suggests primary function protein           dard genetic crosses gene disruptions. The genotype each <br /> TGN plasma membrane (Chen et al., 1999).            spore determined polymerase chain reaction (PCR) <br /> Although PS and/or PE translocation likely              method described Saccharomyces genome deletion project <br /> biochemical activity Drs2p, direct evidence            (http://sequence-www.stanford.edu/group/yeast_deletion_project/ <br /> remains elusive. In fact, erythrocyte plasma mem-           deletions3.html). ZHY3271G generated mating dnf1,2,3&#226;&#338;&#172; <br /> brane APT activity reconstituted vitro             strains generated tetrad dissection. The dnf deletion diploid <br />                                                                     strains generated transforming cells pHO (Russell et al., <br /> purified enzyme (Auland et al., 1994).                              1986), encoding HO endonuclease. The strains harbor Myc- <br />    The drs2&#226;&#338;&#172; strain grows wild type 24&#194;&#176;C           tagged proteins generated PCR-based gene targeting the <br /> above, fails grow 23&#194;&#176;C below. One possible             BY4742 strain selected G418 plates. The pFA6a-13Myc-kanMX6 <br /> explanation cold sensitivity drs2&#226;&#338;&#172; null strain        used PCR template (Bahler et al., 1998), Myc integrated <br /> that proteins replace essential function         strains confirmed PCR. The epitope-tagged Drs2 Neo1 <br />  <br />  <br /> Vol. 13, September 2002                                                                                                                3163 <br />  Z. Hua et al. <br />  <br />  <br />  <br /> Table 2. Yeast strains used study <br />  <br /> Strain                                                              Genotype                                                     Source <br />  <br /> BY4741                     MATa his3 leu2 ura3 met15                                                                       ATCC <br /> BY4742                     MAT&#226;?&#163; his3 leu2 ura3 lys2                                                                        ATCC <br /> BY4743                     MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 met15/MET15 lys2/LYS2                                      ATCC <br /> BY4739 YAL026C             MAT&#226;?&#163; leu2 ura3 lys2 drs2&#226;&#338;&#172;                                                                       ATCC <br /> BY4741 YER166W             MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172;                                                                 ATCC <br /> BY4742 YER166W             MAT&#226;?&#163; his3 leu2 ura3 lys2 dnf1&#226;&#338;&#172;                                                                  ATCC <br /> BY4741 YDR093W             MATa his3 leu2 ura3 met15 dnf2&#226;&#338;&#172;                                                                 ATCC <br /> BY4742 YDR093W             MAT&#226;?&#163; his3 leu2 ura3 lys2 dnf2&#226;&#338;&#172;                                                                  ATCC <br /> BY4741 YMR162C             MATa his3 leu2 ura3 met15 dnf3&#226;&#338;&#172;                                                                 ATCC <br /> BY4742 YMR162C             MAT&#226;?&#163; his3 leu2 ura3 lys2 dnf3&#226;&#338;&#172;                                                                  ATCC <br /> BY4743 YIL048W             MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 neo1&#226;&#338;&#172;/NEO1                                                 ATCC <br /> ZHY9072B                   MATa his3 leu2 ura3 met15 neo1&#226;&#338;&#172; pBM743-NEO1                                                     This study <br /> ZHY615D1C                  MATa his3 leu2 ura3 lys2 drs2&#226;&#338;&#172;                                                                  This study <br /> ZHY615M2D                  MAT&#226;?&#163; his3 leu2 ura3 lys2 drs2&#226;&#338;&#172;                                                                  This study <br /> ZHY2149D                   MAT&#226;?&#163; his3 leu2 ura3 lys2 drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;                                                            This study <br /> ZHY615D2A                  MAT&#226;?&#163; leu2 ura3 drs2&#226;&#338;&#172; dnf2&#226;&#338;&#172;                                                                      This study <br /> ZHY615M2A                  MAT&#226;?&#163; leu2 ura3 met15 drs2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                                This study <br /> PFY3275F                   MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;                                                           This study <br /> PFY3272G                   MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                           This study <br /> ZHY5057B                   MAT&#226;?&#163; his3 leu2 ura3 met15 dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                           This study <br /> PFY3273A                   MAT&#226;?&#163; his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                     This study <br /> PFY3272C                   MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                     This study <br /> ZHY7282C                   MATa leu2 ura3 drs2&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                                                                This study <br /> ZHY708                     MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172; drs2&#226;&#172;&#8249;LEU2                                                 This study <br /> ZHY709                     MATa his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; drs2&#226;&#172;&#8249;LEU2                                                 This study <br /> ZHY704                     MAT&#226;?&#163; his3 leu2 ura3 met15 dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172; drs2&#226;&#172;&#8249;LEU2 pRS416-DRS2                               This study <br /> ZHY3271G                   MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 met15/met15 dnf1&#226;&#338;&#172;/dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;/dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;/dnf3&#226;&#338;&#172;            This study <br /> ZHY3272G1                  MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 met15/met15 dnf1&#226;&#338;&#172;/dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;/dnf3&#226;&#338;&#172;                        This study <br /> ZHY3275F1                  MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 met15/met15 dnf1&#226;&#338;&#172;/dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;/dnf2&#226;&#338;&#172;                        This study <br /> ZHYDPdnf23                 MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 met15/met15 dnf2&#226;&#338;&#172;/dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;/dnf3&#226;&#338;&#172;                        This study <br /> ZHYDPdnf1                  MATa/&#226;?&#163; his3/his3 leu2/leu2 ura3/ura3 lys2/lys2 dnf1&#226;&#338;&#172;/dnf1&#226;&#338;&#172;                                      This study <br /> ZHYNEO1-MYC                MAT&#226;?&#163; his3 leu2 ura3 lys2 NEO1&#226;&#172;&#8249;13XMYC                                                            This study <br /> ZHYDRS2-MYC                MAT&#226;?&#163; his3 leu2 ura3 lys2 DRS2&#226;&#172;&#8249;13XMYC                                                            This study <br /> ZHYDNF1-MYC                MAT&#226;?&#163; his3 leu2 ura3 lys2 DNF1&#226;&#172;&#8249;13XMYC                                                            This study <br /> ZHYDNF2-MYC                MAT&#226;?&#163; his3 leu2 ura3 lys2 DNF2&#226;&#172;&#8249;13XMYC                                                            This study <br /> ZHYDNF3-MYC                MAT&#226;?&#163; his3 leu2 ura3 lys2 DNF3&#226;&#172;&#8249;13XMYC                                                            This study <br /> 6210 vps35&#226;&#338;&#172;                MAT&#226;?&#163; his3 leu2 ura3 lys2 ade2 trp1 vps35&#226;&#172;&#8249;HIS3                                                   Chen, et al., 1999 <br /> 6210 ap15&#226;&#338;&#172;                 MAT&#226;?&#163; his3 leu2 ura3 lys2 ade2 trp1 apl5&#226;&#172;&#8249;HIS3                                                    Cowles, et al., 1997 <br />  <br /> ATCC, American Type Culture Collection. <br />  <br />  <br />  <br />  <br /> proteins functional support cell growth temper-   Sporulation Test <br /> atures tested. We assume epitope-tagged Dnf proteins are <br />                                                                           To test sporulation efficiency, cells grown overnight YPD <br /> functional tested directly. <br />                                                                           medium, diluted PSP2 (1% KOAc, 0.1% yeast extract, 0.67% <br />    pRS416-DRS2 produced subcloning SpeI-SalI frag- <br />                                                                           yeast nitrogen <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span>) presporulation medium, cultured 18 &#226;&#8364;&#8220;24 <br /> ment pRS315-DRS2 (Chen et al., 1999) SpeI-SalI digested <br />                                                                           h late log phase. Cells washed distilled H2O <br /> pRS416. pZH523 (DRS2 disruption plasmid) constructed by <br />                                                                           resuspended 1% KOAc sporulation medium 1&#226;&#8364;&#8220;2 &#226;&#171;&#187; 107 <br /> replacing BamHI-SnaBI fragment pPR10 &#226;&#172;&#402;2.1-kb <br />                                                                           cells/ml. After shaking 3&#226;&#8364;&#8220;7 d 30&#194;&#176;C, sporulating cells were <br /> BamHI-PvuII fragment containing LEU2 pJJ283 (Jones and <br />                                                                           examined microscope visible tetrads. In addition, cells <br /> Prakash, 1990). The plasmid pZH523 linearized SacI and <br />                                                                           sporulation medium 5&#226;&#8364;&#8220;7 d treated with <br /> HpaI transformed different strains delete DRS2. The <br />                                                                           glusulase ether kill nonsporulated cells spread on <br /> full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> NEO1 gene PCR amplified using primers (forward <br />                                                                           YPD plates germinate. Strains scored &#226;&#8364;&#339;&#226;&#171;&#185;&#226;&#8364;? Table 4 showed <br /> 5&#226;&#172;&#732;-AAGGATATGTCTTTTCGTGTACGTG reverse 5&#226;&#172;&#732;-GCCGCT- <br />                                                                           visible viable spores tests. <br /> TGATATGTAAGTTCGTCA) cloned pYGW1 using the <br /> GeneWeaver I kit Research Genetics (Huntsville, AL). The <br /> EcoRI-SalI fragment NEO1 gene pYGW1-NEO1 was <br /> subcloned EcoRI-SalI site pBM743 NEO1        Immunological Methods <br /> control GAL1 promoter suppressed glucose media           Cell labeling, immunoprecipitation (Gaynor Emr, 1997), and <br /> (Johnston Davis, 1984). GFP-ALP (Cowles et al., 1997), pRS416         immunoblotting (Chen et al., 1999) performed described <br /> SNC1-GFP (Lewis et al., 2000), pRS426 STE2-GFP (Odorizzi et al.,          previously. Anti-&#226;?&#163;-factor (Graham Emr, 1991), anti-car- <br /> 1998), p426 KH (2 &#226;?&#174; Kex2-HA pRS426) (Chen et al., 1999)            boxypeptidase Y (CPY) (Klionsky et al., 1988), anti-ALP (Seeger <br /> were transformed strains indicated.                               Payne, 1992) serum used (1 &#226;?&#174;l/OD cells) immuno- <br />  <br />  <br /> 3164                                                                                                            Molecular Biology Cell <br />                                                                                                       Drs2-related ATPases Protein Traffic <br />  <br />  <br /> precipitations. The 9E10 mouse monoclonal c-Myc antibody (Onco- <br /> gene Research Products, Darmstadt, Germany) used 1:2000             Table 3. Growth profile DRS2/NEO1 family deletion mutants <br /> for Western blot 1:100 immunofluorescence. Polyclonal <br /> affinity-purified rabbit anti-hemagglutinin (HA) antibody (Zymed           Strains                   15&#194;&#176;C    20&#194;&#176;C    24&#194;&#176;C     30&#194;&#176;C    37&#194;&#176;C <br /> Laboratories, South San Francisco, CA) used 1:100 dilution to <br /> detect Kex2-HA immunofluorescence. Alexa-488 goat anti-rabbit           Wild type                 &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> IgG Alexa-594 goat anti-mouse IgG (Molecular Probes, Eugene,           neo1&#226;&#338;&#172;                      &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#186;        &#226;&#171;&#186; <br /> OR) used 1:200 secondary antibodies immunofluo-             drs2&#226;&#338;&#172;                      &#226;&#171;&#186;       &#226;&#171;&#186;      &#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> rescence. Monoclonal CPY ALP antibody (Molecular Probes)               dnf1&#226;&#338;&#172;                     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> were used 1:2000 1:125 Western blot experiments. NIH             dnf2&#226;&#338;&#172;                     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> Image 1.62 used quantify band intensities scanned          dnf3&#226;&#338;&#172;                     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> images Western blots. Immunofluorescence green fluores-             drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;                &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#185;       &#226;&#171;&#185;       &#226;&#171;&#185;&#226;&#171;&#185; <br /> cent protein (GFP) fluorescence observed using Axioplan            drs2&#226;&#338;&#172; dnf2&#226;&#338;&#172;                &#226;&#171;&#186;       &#226;&#171;&#186;      &#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> microscope (Carl Zeiss, Thornwood, NY), fluorescent images             drs2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                &#226;&#171;&#186;       &#226;&#171;&#186;      &#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> were quantified processed using MetaMorph 4.5 software (Uni-           dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;               &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> versal Imaging, Downingtown, PA). Samples electron micros-             dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;               &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> copy prepared described previously (Rieder et al., 1996).          dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;               &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> Sections (50 &#226;&#8364;&#8220; 60 nm) viewed CM12 electron microscope            drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;          &#226;&#171;&#186;       &#226;&#171;&#186;      &#226;&#171;&#185;/&#226;&#171;&#186;      &#226;&#171;&#185;        &#226;&#171;&#185; <br /> (Philips, Eindhoven, Netherlands).                                         drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;          &#226;&#171;&#186;       &#226;&#171;&#186;      &#226;&#171;&#185;/&#226;&#171;&#186;     &#226;&#171;&#185;&#226;&#171;&#185;       &#226;&#171;&#185;&#226;&#171;&#185; <br />                                                                            drs2&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;          &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br />                                                                            dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;         &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;     &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;      &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> Invertase Secretion Assay                                                  drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;    &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#186;       &#226;&#171;&#186;        &#226;&#171;&#186; <br /> Cells grown overnight YP medium 5% glucose 30&#194;&#176;C. <br /> Cells diluted 0.1&#226;&#8364;&#8220; 0.2 OD600/ml YP medium 5%               Growth examined YPD plates incubated temperatures <br /> glucose grown 30&#194;&#176;C log phase (0.6 &#226;&#8364;&#8220;1.0 OD600/ml).           indicated. Symbols indicate relative growth rate wild type <br /> Cells pelleted resuspended 0.1% glucose incubated          (&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;) growth (&#226;&#171;&#186;). <br /> at 30&#194;&#176;C. Aliquots cells collected 0, 15, 30, 45 min after <br /> induction processed invertase activity assay (Novick et al., <br /> 1981). External invertase assayed 30&#194;&#176;C described Gold- <br /> stein Lampen (1975). Total invertase activity determined by <br /> assaying cell lysate prepared freeze-thaw method. Inver-          recovered progeny standard crosses, <br /> tase secretion ratio calculated dividing total invertase activity    technique failed yield viable quadruple <br /> by external activity. The small invertase activity time       mutant. To test quadruple mutant is <br /> 0 subtracted time points.                                     viable, dnf1,2,3&#226;&#338;&#172; strain transformed second <br />                                                                            copy DRS2 gene carried URA3 marked plasmid <br /> RESULTS                                                                    (pRS416-DRS2), chromosomal copy DRS2 was <br />                                                                            disrupted. Serial dilutions strains (dnf1,2,3&#226;&#338;&#172; pRS416- <br /> NEO1 Is Essential Gene All Temperatures                              DRS2 drs2&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172; pRS416-DRS2) replica- <br /> NEO1 identified screen genes confer resis-               plated minimal medium containing 5-fluoroorotic acid <br /> tance aminoglycoside neomycin overexpression                   (5-FOA) select cells retained pRS416- <br /> and reported essential gene 30&#194;&#176;C (Prezant et               DRS2 plasmid. As shown Figure 1B, dnf1,2,3&#226;&#338;&#172; strain <br /> al., 1996). Because disruption DRS2 causes temperature-               lose DRS2 plasmid, drs2&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172; <br /> conditional growth <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span>, tested neo1&#226;&#338;&#172; strain                   strain not. In fact, drs2&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172; pRS416-DRS2 <br /> for viability different temperatures. NEO1                  strain failed yield colonies 5-FOA plates tem- <br /> placed transcriptional regulation GAL1 pro-                   perature tested (15, 20, 24, 30, 37&#194;&#176;C), indicating the <br /> moter, tightly repressed cells grown                  drs2&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172; strain viable. The fact one <br /> glucose expressed strongly galactose (Johnston                  member genes maintain viability the <br /> Davis, 1984). The neo1&#226;&#338;&#172; GAL:NEO1 strain grew                       cells (Table 3 Figure 1) quadruple mutant is <br /> galactose failed grow plated media contain-                 viable indicates DRS2 DNF genes constitute <br /> ing glucose temperature tested (Table 3, neo1&#226;&#338;&#172;). Thus,              essential gene family substantial functional overlap. <br /> NEO1 provides essential function per- <br /> formed DRS2, DNF1, 2, 3. We overex-                  Growth Profiles Double Triple Mutants <br /> pression NEO1 suppress cold-sensitive <br /> growth defect drs2&#226;&#338;&#172; strains, overexpression                Most double mutants grow parental <br /> DRS2 suppress lethality neo1&#226;&#338;&#172;, suggesting             single mutants drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;, exhibits a <br /> no significant genetic overlap DRS2 NEO1.                      strong growth defect 30&#194;&#176;C (Figure 1A). Interestingly, this <br />                                                                            double mutant grows better 37&#194;&#176;C (Figure 1A) and <br />                                                                            propagated temperature. The remaining dou- <br /> DRS2 DNF Genes Constitute Essential                                 ble mutants carrying drs2&#226;&#338;&#172; exhibit cold-sensitive <br /> Family                                                                     growth defect drs2&#226;&#338;&#172; single mutant, dnf <br /> Except NEO1, members DRS2/                    double mutants grow wild-type strain (Table <br /> NEO1 family essential (Winzeler et al., 1999). To test                 3). These data indicate DRS2 DNF1 great- <br /> whether functional redundancy DRS2                    est overlap function, DNF2 DNF3 contribute <br /> and DNF genes, generated strains carrying pos-                  little viability absence DRS2, surpris- <br /> sible combinations drs2 dnf null alleles (Tables 2              ing considering sequence similarity DNF1 and <br /> and 3 Figure 1A). All possible double triple mutants               DNF2. <br />  <br />  <br /> Vol. 13, September 2002                                                                                                                3165 <br />  Z. Hua et al. <br />  <br />  <br />  <br />  <br />                                                                                                        Figure 1. DRS2 DNFs consti- <br />                                                                                                        tute essential gene family. (A) <br />                                                                                                        BY4742      (WT),     ZHY615D1C <br />                                                                                                        (drs2&#226;&#338;&#172;), BY4742 YER166W (dnf1&#226;&#338;&#172;), <br />                                                                                                        ZHY2143A (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;), ZHY709 <br />                                                                                                        (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;), ZHY708 <br />                                                                                                        (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;), ZHY7282C <br />                                                                                                        (drs2&#226;&#338;&#172;     dnf2&#226;&#338;&#172;    dnf3&#226;&#338;&#172;),   and <br />                                                                                                        PFY3273A (dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;) <br />                                                                                                        strains grown YPD plates <br />                                                                                                        30, 37, 20&#194;&#176;C indicated in <br />                                                                                                        lower right panel. (B) The <br />                                                                                                        dnf1,2,3&#226;&#338;&#172; drs2&#226;&#338;&#172; quadruple mutant <br />                                                                                                        inviable. Serial dilutions of <br />                                                                                                        strains PFY3273A pRS416-DRS2 <br />                                                                                                        (dnf1,2,3&#226;&#338;&#172; pRS416-DRS2) and <br />                                                                                                        ZHY704 (dnf1,2,3&#226;&#338;&#172; drs2&#226;&#338;&#172; pRS416- <br />                                                                                                        DRS2) spotted YPD or <br />                                                                                                        minimal 5-FOA plates, incu- <br />                                                                                                        bated 30&#194;&#176;C (shown), 37, 24, <br />                                                                                                        20, 15&#194;&#176;C (not shown). <br />  <br />  <br />  <br />   The drs2&#226;&#338;&#172; dnf1,2&#226;&#338;&#172; drs2&#226;&#338;&#172; dnf1,3&#226;&#338;&#172; triple deletion strains        caused weakened cell wall bud tip. <br /> exhibit severe growth defect predicted, drs2&#226;&#338;&#172;           Although clear sporulation defect and <br /> dnf1&#226;&#338;&#172; exhibits strong growth defect. Interestingly,            CW hypersensitivity strains caused com- <br /> drs2&#226;&#338;&#172; dnf1,2&#226;&#338;&#172; drs2&#226;&#338;&#172; dnf1,3&#226;&#338;&#172; strains grew slightly better         mon defect, data indicate Dnf1p, Dnf2p, and <br /> than drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; 30&#194;&#176;C fared worse 37&#194;&#176;C (Figure             Dnf3p overlap exert important cellular function that <br /> 1A). The drs2&#226;&#338;&#172; dnf2,3&#226;&#338;&#172; strain showed slight temperature-           replaced Drs2p Neo1p. <br /> sensitive growth phenotype compared drs2&#226;&#338;&#172; single <br /> mutant dnf1,2,3&#226;&#338;&#172; strain grew nearly           dnf1&#226;&#338;&#172; Exacerbates Membrane Accumulation Defect <br /> wild-type strain temperatures (Figure 1A), indicating         drs2&#226;&#338;&#172; <br /> that DRS2 itself fully maintain cell growth. <br />                                                                      Mutants perturb protein transport secretory path- <br />                                                                      way typically accumulate abnormal membrane structures. <br /> Sporulation Defect Calcofluor White <br /> Hypersensitivity drs2/dnf Mutants <br /> During course generating strains shown Table <br />                                                                      Table 4. Sporulation defect CW sensitivity drs2/dnf deletion <br /> 2, noticed diploid deletion mutants not <br />                                                                      mutants <br /> sporulate. To define defect, examined several <br /> diploid strains homozygous null alleles indicated                                     Sporulation           Calcofluor white <br /> Table 4 ability sporulate. Unexpectedly,            Strains                       defect                 sensitivity <br /> homozygous dnf1,2,3&#226;&#338;&#172; diploid exhibited severe sporula- <br /> tion defect does obvious growth           Wild type                       &#226;&#171;&#186;                        &#226;&#171;&#186; <br /> defect. The drs2&#226;&#338;&#172;, dnf1,2&#226;&#338;&#172;, dnf1,3&#226;&#338;&#172; homozygous strains           drs2&#226;&#338;&#172;                           &#226;&#171;&#185;                       &#226;&#171;&#185;&#226;&#171;&#185; <br /> also exhibited sporulation defect. However, dnf2,3&#226;&#338;&#172; sporu-         dnf1&#226;&#338;&#172;                           &#226;&#171;&#186;                        &#226;&#171;&#186; <br /> lated normally, did single deletion dnf strains           dnf2&#226;&#338;&#172;                           &#226;&#171;&#186;                        &#226;&#171;&#186; <br /> (Table 4).                                                           dnf3&#226;&#338;&#172;                           &#226;&#171;&#186;                        &#226;&#171;&#186; <br />                                                                      dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172;                     &#226;&#171;&#185;                       &#226;&#171;&#185;&#226;&#171;&#185; <br />    The sporulation defect homozygous diploids correlated <br />                                                                      dnf1&#226;&#338;&#172; dnf3&#226;&#338;&#172;                    &#226;&#171;&#185;/&#226;&#171;&#186;                       &#226;&#171;&#186; <br /> to increased sensitivity haploid mutants CW             dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;                     &#226;&#171;&#186;                        &#226;&#171;&#186; <br /> (Table 4). Strains deficient cell wall chitin resistant   dnf1&#226;&#338;&#172; dnf2&#226;&#338;&#172; dnf3&#226;&#338;&#172;               &#226;&#171;&#185;                       &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> CW, mutants increase chitin deposition <br /> or perturb cell wall structure hypersensitive            Sporulation assessed homozygous diploids, the <br /> agent. The dnf1,2,3&#226;&#338;&#172; strain exhibited greatest sensitivity       Calcofluor white sensitivity examined haploid strains as <br /> to CW, single mutants similar                described MATERIALS AND METHODS. Sporulation defect: &#226;&#171;&#186;, <br /> wild-type strain. The drs2&#226;&#338;&#172; dnf1,2&#226;&#338;&#172; strains showed            significant sporulation defect; &#226;&#171;&#185;, complete sporulation defect; <br /> intermediate sensitivity (Table 4). We noticed         &#226;&#171;&#185;/&#226;&#171;&#186;, sporulation efficiency &#226;&#172;&#402;0.5% wild type. Calcofluor white <br /> dnf1,2,3&#226;&#338;&#172; cells elongated grown lower temper-            sensitivity: growth 50 &#226;?&#174;g/ml CW (&#226;&#171;&#186;); growth 30 &#226;?&#174;g/ml <br />                                                                      CW plates (&#226;&#171;&#185;&#226;&#171;&#185;); growth 10 &#226;?&#174;g/ml CW plates (&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;). <br /> ature minimal media (our unpublished observation), <br />  <br />  <br /> 3166                                                                                                      Molecular Biology Cell <br />                                                                                                    Drs2-related ATPases Protein Traffic <br />  <br />  <br />  <br />  <br /> Figure 2. drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; mutant massively accumulates abnormal membrane bound structures resembling Berkeley bodies. BY4742 (WT), <br /> ZHY2149D (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;), PFY3273A (dnf1,2,3&#226;&#338;&#172;) cells prepared electron microscopy described previously (Chen et al., 1999). <br /> Numerous double-membrane ring structures resembling Berkeley bodies (indicated arrows) accumulate drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells, whereas <br /> the dnf1,2,3&#226;&#338;&#172; cells similar wild-type cells. <br />  <br />  <br />  <br /> The drs2&#226;&#338;&#172; mutant accumulates abnormal membrane-bound                  puncta observed. This defect GFP-ALP localization <br /> structures containing Golgi enzymes similar               strongly exacerbated drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells, approach- <br /> Berkeley bodies morphology (Chen et al., 1999). Exami-             ing defect observed apl5&#226;&#338;&#172; (AP-3) mutant (Figure 3, <br /> nation drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells electron microscopy (EM)               A B). Close 50% drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells contained <br /> revealed massive accumulation membrane-bound struc-              puncta, cells contain- <br /> tures resembling Berkeley bodies (Figure 2), severe            ing normal-appearing vacuole(s). This suggests the <br /> phenotype observed drs2&#226;&#338;&#172; single deletion cells.           primary defect cells GFP-ALP mislocalization <br /> This consistent severe growth defect drs2&#226;&#338;&#172;             vacuole fragmentation. This defect specific to <br /> dnf1&#226;&#338;&#172; cells suggested drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; exhibit          drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; mutant cells, dnf1&#226;&#338;&#172; the <br /> a stronger defect protein transport Golgi com-            dnf1,2,3&#226;&#338;&#172; mutant exhibited defect ALP localization (Fig- <br /> plex single mutant. In contrast, dnf1,2,3&#226;&#338;&#172; cells      ure 3A). <br /> did accumulate abnormal membrane structures                  Western blots indicated drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells accu- <br /> similar appearance wild-type cells (Figure 2). Strains          mulated ALP precursor (49 &#226;&#171;&#190; 29% total ALP, n &#226;&#171;&#189; 7) <br /> constructed BY4741/BY4742 background exhibit              wild-type cells (4 &#226;&#171;&#190; 0%). The drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; strains <br /> internal membranes wild-type strains (e.g.,                tended pick faster growing suppressors, prob- <br /> SEY6210), appearance dnf1,2,3&#226;&#338;&#172; cells shown           ably contributed wide range error mutant. <br /> not significantly different wild-type.                           There intermediate level proALP accumula- <br />                                                                       tion drs2&#226;&#338;&#172; cells (18 &#226;&#171;&#190; 8%), dnf1&#226;&#338;&#172; indis- <br /> drs2 dnf1 Exhibits Defect Protein Transport                   tinguishable wild type (Figure 3C). To follow trans- <br /> Vacuole                                                               port kinetics ALP mutant strains, cells were <br /> The functional redundancy DRS2 DNF                    pulse-labeled chased times indicated Figure <br /> genes suggested Dnf proteins involved            3D. Consistent previous results, ALP processing is <br /> protein transport secretory and/or endocytic path-             significantly delayed (&#226;&#172;&#402;5-fold) drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; mutant <br /> ways. To test this, examined transport ALP              compared wild-type cells (Figure 3E). The drs2&#226;&#338;&#172; <br /> CPY vacuole. ALP transported directly              mutant showed intermediate phenotype, whereas <br /> TGN vacuole adaptor protein (AP)-3 coated vesicles          dnf1,2,3&#226;&#338;&#172; cells exhibited normal kinetics ALP process- <br /> (Cowles et al., 1997) processed high molecular          ing (Figure 3D). We conclude data Drs2p <br /> weight precursor form mature form vacuole               Dnf1p required ALP transport pathway, <br /> (Klionsky Emr, 1989). To determine Dnf                dnf1,2,3&#226;&#338;&#172; does affect pathway. <br /> proteins involved ALP pathway, exam-                 We examined CPY transport pathway these <br /> ined localization GFP-ALP fusion protein              mutants. CPY synthesized endoplasmic reticulum <br /> different mutant strains. In wild-type cells, GFP-ALP             (ER) p1 precursor form modified N-linked <br /> primarily localized vacuole membrane, &#226;&#172;?4%               oligosaccharides Golgi mannosyltransferases form <br /> the cells contained punctate structures outside          p2 precursor. p2 CPY sorted secreted proteins <br /> of vacuole (Figure 3, A B). In drs2&#226;&#338;&#172; cells,       TGN transported late endosome <br /> GFP-ALP localized vacuole membrane, in-                 vacuole processed mature form <br /> crease percentage cells exhibiting extravacuolar            (mCPY) (Stevens et al., 1982). The CPY pathway inde- <br />  <br />  <br /> Vol. 13, September 2002                                                                                                             3167 <br />  Z. Hua et al. <br />  <br />  <br />  <br />  <br /> Figure 3. Drs2p Dnf1p required ALP transport vacuole. (A) Localization GFP-ALP. BY4742 (WT), ZHY615D1C (drs2&#226;&#338;&#172;), <br /> ZHY2149D (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;), BY4742 YER166W (dnf1&#226;&#338;&#172;), PFY3273A (dnf1,2,3&#226;&#338;&#172;), 6210 apl5&#226;&#338;&#172; harboring pGO41 (GFP-ALP) grown 30&#194;&#176;C <br /> to log phase examined fluorescence microscope. (B) Percentage cells exhibiting abnormal GFP-ALP localization. Cells containing <br /> 0, 1&#226;&#8364;&#8220;3, 4 &#226;&#8364;&#8220; 6, 7&#226;&#8364;&#8220;9, &#226;&#177;&#8211;10 punctate structures bearing GFP-ALP counted presented percentage total number cells <br /> counted. Data shown average counting &#226;&#172;&#381;100 cells independent transformants strain. (C) Steady-state <br /> distribution ALP forms. Cells grown YPD media 30&#194;&#176;C log phase. Whole cell lysates prepared immunoblotted to <br /> detect precursor (pro) mature (m) forms ALP. (D) Kinetics ALP transport vacuole. Cells grown log phase 30&#194;&#176;C, <br /> labeled [35S]methione/cysteine 5 min chased 0, 5, 10, 20 min 30&#194;&#176;C. ALP recovered sample by <br /> immunoprecipitation subjected SDS-PAGE. (E) CPY ALP processing rate assessed pulse-chase experiments plotted <br /> relative WT cells. Data shown average experiments. <br />  <br />  <br />  <br /> pendent AP-3 perturbed vps mutants,                fold kinetic delay CPY transport, drs2&#226;&#338;&#172; <br /> secrete p2 form CPY (Conibear Stevens, 1998).              dnf1&#226;&#338;&#172; mutant exhibited 3.2-fold kinetic delay (Figures <br /> For example, vps35 secretes p2 CPY produces dark                3E 4C). We previously noted partial, Golgi- <br /> signal colony blot probed CPY (Figure 4A). The               specific glycosylation defect drs2&#226;&#338;&#172; prevents clear <br /> drs2&#226;&#338;&#172; strain exhibits modest CPY secretion pheno-              separation p1 p2 CPY SDS-PAGE (Chen et al., <br /> type, does exacerbated drs2&#226;&#338;&#172;              1999). This evident drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells (Figure <br /> dnf1&#226;&#338;&#172; double mutant. Neither dnf1&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172;                  4C), making difficult assess kinetics ER-to- <br /> triple mutant secreted CPY (Figure 4A). None drs2/             Golgi transport CPY. However, analysis &#226;?&#163;-factor <br /> dnf deletion strains tested accumulated significant                 transport (described below) indicated protein trans- <br /> amount p2 CPY detected Western Blot (Figure                  port ER TGN unaffected drs2&#226;&#338;&#172; <br /> 4B), modest increase observed drs2&#226;&#338;&#172;             dnf1&#226;&#338;&#172; mutant. Thus, exacerbation CPY trans- <br /> dnf1&#226;&#338;&#172; mutant (Figure 4B). To explore kinetics             port defect drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; double mutant, although <br /> of CPY processing, strains subjected pulse-           substantial observed ALP pathway. In <br /> chase analysis 30&#194;&#176;C. The drs2&#226;&#338;&#172; mutant exhibited 1.7              addition, Western blot was <br />  <br />  <br /> 3168                                                                                                       Molecular Biology Cell <br />                                                                                                  Drs2-related ATPases Protein Traffic <br />  <br />  <br />                                                                       dnf1&#226;&#338;&#172; Does Not Exacerbate Pro-&#226;?&#163;-Factor Processing <br />                                                                       Invertase Secretion Defects drs2&#226;&#338;&#172; <br />                                                                       The yeast &#226;?&#163;-factor mating pheromone synthesized the <br />                                                                       ER high molecular weight precursor mod- <br />                                                                       ified Golgi complex produce heterogeneously <br />                                                                       glycosylated pro-&#226;?&#163;-factor form, migrates high <br />                                                                       molecular mass smear SDS-PAGE. This precursor fur- <br />                                                                       ther processed TGN series proteolytic <br />                                                                       events, initiated Kex2p, produce mature &#226;?&#163;-factor <br />                                                                       peptide, secreted (Fuller et al., 1988). Kex2p nor- <br />                                                                       mally cycles TGN endosomal system <br />                                                                       (Conibear Stevens, 1998). Mutants mislocalize <br />                                                                       Kex2p TGN proteases required pro-&#226;?&#163;-factor <br />                                                                       processing secrete high molecular weight Golgi form of <br />                                                                       pro-&#226;?&#163;-factor (Payne Schekman, 1989). The drs2&#226;&#338;&#172; mutant <br />                                                                       exhibited defect pro-&#226;?&#163;-factor processing 30&#194;&#176;C (Figure <br />                                                                       5A), phenotype severe lower tem- <br />                                                                       peratures (Chen et al., 1999). The drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells also <br />                                                                       exhibited incomplete processing pro-&#226;?&#163;-factor, this <br />                                                                       defect actually severe observed the <br />                                                                       drs2&#226;&#338;&#172; mutant (Figure 5A). The rapid disappearance the <br />                                                                       ER form pro-&#226;?&#163;-factor mutant cells indicates that <br />                                                                       ER-to-Golgi transport perturbed. As shown before, <br />                                                                       clathrin light chain mutant (clc1&#226;&#338;&#172;) exhibited substantial <br />                                                                       defect little mature &#226;?&#163;-factor produced. Fur- <br />                                                                       thermore, dnf1&#226;&#338;&#172;, dnf1,2,3&#226;&#338;&#172;, dnf2,3&#226;&#338;&#172; strains processed <br />                                                                       pro-&#226;?&#163;-factor normally (our unpublished observation). Thus, <br />                                                                       dnf1&#226;&#338;&#172; does exacerbate Kex2p mislocalization pheno- <br />                                                                       type drs2&#226;&#338;&#172;. <br />                                                                          There evidence classes late secretory ves- <br />                                                                       icles yeast: dense class carrying invertase a <br />                                                                       lighter class carrying Pma1p (Harsay Bretscher, 1995). <br />                                                                       In addition, clathrin required sort invertase the <br />                                                                       dense vesicles, absence clathrin, invertase is <br />                                                                       shunted Pma1p vesicle pathway (Gurunathan et al., <br />                                                                       2002; Harsay Schekman, 2002). Thus, secretion of <br />                                                                       invertase blocked, kinetic defect ob- <br />                                                                       served. To test Dnf proteins involved in <br /> Figure 4. CPY pathway partially affected drs2/dnf deletion      exocytosis, measured kinetics invertase secre- <br /> strains 30&#194;&#176;C. (A) Colony-blot analysis CPY secretion. Freshly   tion different dnf deletion strains. Cells grown <br /> streaked BY4742 (WT), BY4742 YER166W (dnf1&#226;&#338;&#172;), PFY3273A                5% YP glucose media log phase shifted to <br /> (dnf1,2,3&#226;&#338;&#172;), ZHY615D1C (drs2&#226;&#338;&#172;), ZHY2149D (drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;) 6210       0.1% YP glucose media induce invertase synthesis. The <br /> vps35&#226;&#338;&#172; cells grown YPD plates 30&#194;&#176;C (shown), 37      rate induction equivalent strains tested (our <br /> 20&#194;&#176;C (data shown) 24 h. Nitrocellulose membranes were <br />                                                                       unpublished observation), initial rate invertase <br /> overlaid colonies incubated 24 h. Mem- <br /> branes washed probed monoclonal CPY antibody.         appearance outside cell reflected rate secre- <br /> (B) Steady-state distribution CPY precursor forms. Cells      tion. In wild-type cells, 40% newly synthesized inver- <br /> grown YPD medium 30&#194;&#176;C log phase. Whole cell lysates     tase secreted 15 min induction and <br /> prepared subjected SDS-PAGE Western blotting de-        started plateau (70%) &#226;&#172;&#402;30 min (Figure 5B). The <br /> tect p1, p2 mature forms CPY. (C) Kinetics CPY transport.   dnf1,2,3&#226;&#338;&#172; cells exhibited wild-type kinetics invertase <br /> Cells grown log phase 30&#194;&#176;C, labeled [35S]methi- <br />                                                                       secretion drs2&#226;&#338;&#172; clathrin mutants exhibited a <br /> onine/cysteine 5 min, chased 0, 5, 10 min 30&#194;&#176;C. <br /> CPY recovered sample immunoprecipitation         significant delay process. Although drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; <br /> subjected SDS-PAGE.                                                mutant exhibits severe growth membrane accumula- <br />                                                                       tion defects, invertase secretion defect more <br />                                                                       severe drs2&#226;&#338;&#172; cells (Figure 5B). These <br /> less CPY precursor dnf1,2,3&#226;&#338;&#172; mutant. However,              results suggest Drs2p family member that <br /> pulse-chase result indicated kinetics CPY trans-          contributes significantly invertase secretion path- <br /> port normal mutant (Figure 4C). Similarly,            way, severe growth membrane accumu- <br /> other mutants displayed wild-type kinetics CPY trans-              lation defects drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; cells, compared with <br /> port (Figure 4C).                                                     drs2&#226;&#338;&#172; cells, severe defect secretion. <br />  <br />  <br /> Vol. 13, September 2002                                                                                                           3169 <br />  Z. Hua et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 6. dnf1,2,3&#226;&#338;&#172; mutant exhibits cold-sensitive defect in <br />                                                                          FM4-64 transport vacuole. Cells stained FM4-64 on <br />                                                                          ice shifted 15&#194;&#176;C 0, 1, 4 h viewed fluores- <br />                                                                          cence microscopy. <br />  <br />  <br />                                                                          rapidly wild-type cells, exhibited delay in <br />                                                                          transport endosomes vacuole (Figure 6). Surpris- <br />                                                                          ingly, dnf1,2&#226;&#338;&#172; dnf1,3&#226;&#338;&#172; exhibited delay FM4-64 <br />                                                                          internalization plasma membrane 15&#194;&#176;C, without <br />                                                                          accumulation internal punctate structures. The reason for <br />                                                                          differences dnf1,2,3&#226;&#338;&#172; triple mutant the <br />                                                                          double mutants unclear. However, defects are <br /> Figure 5. dnf1&#226;&#338;&#172; does exacerbate pro-&#226;?&#163;-factor processing          cold sensitive reversible, cells were <br /> and invertase secretion defects drs2&#226;&#338;&#172;. (A) Kinetics pro-&#226;?&#163;-factor   shifted 22&#194;&#176;C, FM4-64 moved quickly vacuole <br /> processing. Cells grown log phase 30&#194;&#176;C, labeled          (our unpublished observations). <br /> [35]methione/cysteine 5 min, chased 0, 5, 10 min          During FM4-64 labeling experiments, that <br /> 30&#194;&#176;C. Pro&#226;?&#163;-factor recovered sample immunopre-           drs2&#226;&#338;&#172; dnf1,2,3&#226;&#338;&#172; cells internalize <br /> cipitation subjected SDS-PAGE. (B) Invertase secretion assay.     FM4-64 faster wild-type cells 15 30&#194;&#176;C. <br /> Cells grown 5% YP glucose media 30&#194;&#176;C log phase         Others proposed internalization FM4-64 to <br /> shifted 0.1% YP glucose media start induction invertase. <br />                                                                          early endosome, portion dye follows recycling <br /> Aliquots cells collected 0, 15, 30, 45 min after <br /> invertase induced. Secreted total invertase activities       pathway TGN plasma membrane <br /> measured expressed percentage invertase secreted.          (Lewis et al., 2000). In case, defect recycling <br />                                                                          pathway cause apparent increase rate of <br />                                                                          FM4-64 internalization. To directly test recy- <br />                                                                          cling pathway perturbed, examined localization of <br /> dnf1,2,3&#226;&#338;&#172; Exhibits Defect Recycling                             GFP-Snc1p fusion protein marker follow path- <br /> Polarized Localization GFP-Snc1p                                      way. Snc1p exocytic v-SNARE protein (Gerst, 1999) <br /> Other involvement Dnf1p vacuolar protein              shown recycle endocytosis the <br /> transport pathways Drs2p, re-                   plasma membrane early endosome, is <br /> quirement Dnf proteins secretory path-              sorted TGN. From TGN, Snc1p packaged <br /> way. We examined endocytic pathway mutants                  exocytic vesicles target plasma membrane <br /> by labeling chasing cells FM4-64, fluorescent             (Lewis et al., 2000). <br /> endocytic marker (Vida Emr, 1995). All dnf mutants,                 In wild-type cells, GFP-Snc1p primarily localized to <br /> including double triple deletion mutants, able              plasma membrane, concentrating bud re- <br /> to endocytose FM4-64 deliver dye vacuole              gions polarized growth. Only small fraction the <br /> at 30&#194;&#176;C wild-type strain (our unpublished                 GFP-Snc1p observed punctate structures the <br /> observations). Interestingly, 15&#194;&#176;C dnf1,2,3&#226;&#338;&#172; cells                cell (Figure 7A). In contrast, substantial decrease <br /> seemed internalize FM4-64 plasma membrane                    GFP-Snc1p localized plasma membrane dnf1,2,3&#226;&#338;&#172; <br />  <br />  <br /> 3170                                                                                                          Molecular Biology Cell <br />                                                                                                Drs2-related ATPases Protein Traffic <br />  <br />  <br />  <br />  <br /> Figure 7. dnf1,2,3&#226;&#338;&#172; mutant exhibits defect Snc1p-GFP recycling. (A) pRS416 SNC1-GFP introduced BY4742 (WT), PFY3273A <br /> (dnf1,2,3&#226;&#338;&#172;) ZHY615D1C (drs2&#226;&#338;&#172;) strains. Cells grown 30&#194;&#176;C log phase examined fluorescence microscopy. (B) Cells <br /> harboring pRS426 STE2-GFP grown 30&#194;&#176;C log phase examined fluorescence microscopy. MetaMorph 4.5 used process <br /> the image measure fluorescence intensity cell surface vacuoles. <br />  <br />  <br />  <br /> cells, concomitant increase internal punctate struc-      unclear drs2&#226;&#338;&#172; endosome TGN <br /> tures (Figure 7A), early endosomes TGN            transport defect. <br /> compartments. The GFP-Snc1p remains plasma                 To rule possibility Snc1p actually internal- <br /> membrane evenly distributed mother              ized faster dnf1,2,3&#226;&#338;&#172; cells slowed recy- <br /> daughter cells, suggesting defect polarized delivery       cling plasma membrane, examined local- <br /> of Snc1p bud. The mislocalization Snc1p            ization GFP-Ste2p cells (Figure 7B). Ste2p, the <br /> dnf1,2,3&#226;&#338;&#172; cells supports hypothesis recycling         &#226;?&#163;-factor receptor, constitutively internalized the <br /> the early endosome plasma membrane dis-              plasma membrane delivered endosomal <br /> turbed mutant. Because kinetics secretion            vacuole degradation (Schandel Jen- <br /> seems normal dnf1,2,3&#226;&#338;&#172; cells (Figure 5B), propose <br />                                                                     ness, 1994). If endocytosis enhanced dnf1,2,3&#226;&#338;&#172; cells, <br /> that dnf1,2,3&#226;&#338;&#172; mutant defective endosome to <br />                                                                     expect decrease plasma membrane <br /> TGN branch pathway. The dnf single dnf2,3&#226;&#338;&#172; <br /> mutants traffic GFP-Snc1p normally; however, dnf1,2&#226;&#338;&#172;            fluorescence relative internal signal shown Snc1p. <br /> and dnf1,3&#226;&#338;&#172; mutants exhibit noticeable defect,        However, wild-type dnf1,2,3&#226;&#338;&#172; cells, ratio <br /> less severe dnf1,2,3&#226;&#338;&#172; triple mutant (our unpub-         Ste2p-GFP fluorescence intensity plasma mem- <br /> lished observation). The drs2&#226;&#338;&#172; cells exhibit similar GFP-         brane vacuole 0.2 (plasma membrane pixel <br /> Snc1p mislocalization defect shown dnf1,2,3&#226;&#338;&#172; cells            value &#226;&#171;&#188; vacuolar pixel value). This result suggests the <br /> (Figure 7A), consistent efficient       mislocalization Snc1p dnf1,2,3&#226;&#338;&#172; cells caused <br /> internalization FM4-64 observed cells. However,         enhanced endocytosis, inefficient transport <br /> this explained exocytic defect drs2&#226;&#338;&#172;,        endosome 3 TGN 3 plasma membrane loop. <br />  <br />  <br /> Vol. 13, September 2002                                                                                                         3171 <br />  Z. Hua et al. <br />  <br />  <br />  <br />  <br />                                                                                      Figure 8. Expression levels Drs2/Neo1 fam- <br />                                                                                      ily proteins. (A) Strains BY4742 (control), ZHY- <br />                                                                                      NEO1-MYC, ZHYDRS2-MYC, ZHYDNF1-MYC, <br />                                                                                      ZHYDNF2-MYC, ZHYDNF3-MYC were <br />                                                                                      grown YPD media 30&#194;&#176;C. Equal amounts of <br />                                                                                      cell lysates subjected SDS-PAGE <br />                                                                                      immunoblotting monoclonal c-Myc <br />                                                                                      antibody detect fusion proteins. The mi- <br />                                                                                      gration standard protein markers labeled at <br />                                                                                      left. Predicted molecular mass full- <br />                                                                                      length protein fused 13XMyc tag Neo1p, <br />                                                                                      146 kDa; Drs2p, 170 kDa; Dnf1p, 194 kDa; Dnf2p, <br />                                                                                      198 kDa; Dnf3p, 204 kDa. Each lane con- <br />                                                                                      tained equivalent CPY (not <br />                                                                                      shown). (B) Full length near full-length <br />                                                                                      bands A analyzed NIH Image 1.62. <br />                                                                                      The signal intensity fusion protein was <br />                                                                                      plotted percentage Drs2p signal <br />                                                                                      strength. Data shown figure represen- <br />                                                                                      tative experiments. <br />  <br />  <br />  <br /> Dnf Protein Expression Localization                          Dnf2p. Dnf3p staining reveals punctate structures evenly <br /> To study expression <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Dnf proteins,          spread cell, resembles staining <br /> epitope-tagged DRS2/NEO1 family members            pattern Drs2p (Figure 9, c, d, g, h). <br /> c-Myc carboxyl-terminal end proteins. Immu-         Even Dnf1p Dnf2p exhibit unique localization <br /> noblotting cell extracts indicated       patterns clustered near plasma membrane, both <br /> previously uncharacterized genes (DNF1, 2, 3)           distributed internal structures well. <br /> expressed normal growth conditions (Figure 8A).            Drs2p colocalizes substantially TGN marker Kex2p. <br /> Neo1p Drs2p expressed highest level,      We examined colocalization Dnf proteins with <br /> Dnf3p expressed low level compared         Kex2p colocalized Kex2p to <br /> other proteins (Figure 8B). This consistent obser-   extent (Figure 9B). Dnf3p showed greatest overlap <br /> vation Dnf3p contribute cell     Kex2p (Figure 9B, g) comparable staining <br /> viability CW sensitivity. The protein level fam-    pattern observed Drs2p (Figure 9B, h). Dnf1p Dnf2p <br /> ily member greatly decreased cells             showed overlap Kex2p, experi- <br /> grown 37&#194;&#176;C moderately higher 20&#194;&#176;C re-          ments required determine small per- <br /> vealed Western blot immunofluorescence anal-         centage Dnf1p Dnf2p localized TGN. The <br /> yses (our unpublished observations). Consistent         localization Dnf1 Dnf2 proteins plasma mem- <br /> growth profiles, suggests cells greater re-     brane transient exocytic and/or endo- <br /> quirement Drs2p Dnf proteins lower tempera-           cytic membranes suggests proteins cycle in <br /> tures. The marked susceptibility proteolysis, noted previ-    TGN-plasma membrane&#226;&#8364;&#8220; endosome&#226;&#8364;&#8220;TGN pathway. This <br /> ously Drs2p, case Neo1p Dnf          consistent defect GFP-Snc1 recycling the <br /> proteins. We distinguish time         dnf1,2,3&#226;&#338;&#172; mutant. <br /> proteolysis occurs vivo preparation cell <br /> lysates.                                                         DISCUSSION <br />    We examined localization epitope-tagged       We previously implicated Drs2p clathrin function the <br /> proteins immunofluorescence microscopy         TGN (Chen et al., 1999), report focuses require- <br /> all proteins exhibited punctate staining pattern. How-     ments Drs2p Dnfs protein transport. We 1) the <br /> ever, Dnf proteins different localization pat-    DRS2 DNF genes constitute essential gene family with <br /> terns (Figure 9A). Dnf1p internal membranes      overlapping functions. 2) Specific protein transport steps the <br /> the plasma membrane, concentrated         late secretory, endocytic, vacuolar pathways specific <br /> emerging bud site, small buds, mother-daughter           requirements Drs2/Dnf proteins (Figure 10). 3) Drs2p, Dnf1p, <br /> neck dividing cells (Figure 9, e),        Dnf2p exhibit unique localization patterns, suggesting that <br /> similar localization cortical actin patches. This      primary function different transport steps. <br /> suggests Dnf1p concentrated exocytic vesicles        Of DRS2/NEO1 family genes, NEO1 only <br /> targeted polarized sites growth. Dnf2p         essential gene does overlap genetically <br /> unique, smaller punctate structures lie directly under-     family members. Yeast survive with <br /> neath plasma membrane (Figure 9, b f), does       multiple combinations null alleles DRS2, DNF1, 2, and <br /> not exhibit polarized distribution Dnf1p does. The          3 quadruple mutant inviable. The fact any <br /> punctate structures endocytic vesicles plasma        single member Drs2/Dnf group support cell <br /> membrane invaginations. In cells, small regions          growth demonstrates functional interchangeability for <br /> continuous plasma membrane staining seen            proteins, suggests protein shares com- <br />  <br />  <br /> 3172                                                                                                  Molecular Biology Cell <br />                                                                                             Drs2-related ATPases Protein Traffic <br />  <br />  <br />  <br />  <br /> Figure 9. Localization Dnf pro- <br /> teins. Cells grown YPD <br /> media 30&#194;&#176;C prepared im- <br /> munofluorescence. (A) Cells were <br /> visualized mouse monoclonal <br /> c-Myc antibody (a&#226;&#8364;&#8220;d). (B) Cells <br /> transformed 2 &#226;?&#174; KEX2HA <br /> plasmid stained mouse <br /> monoclonal c-Myc antibody a <br /> rabbit polyclonal HA antibody vi- <br /> sualize Myc-tagged Drs2/Dnf pro- <br /> teins (red) HA-tagged Kex2p <br /> (green) (e&#226;&#8364;&#8220;h). The yellow punctate <br /> structures indicate colocalization <br /> of Drs2/Dnf proteins Kex2p. <br />  <br />  <br />  <br /> mon biochemical activity. A substantial body work sug-        strongly exacerbates membrane accumulation pheno- <br /> gests proteins APTs, activity   type drs2&#226;&#338;&#172;, suggesting overlap function protein <br /> not reconstituted purified enzyme. Although        membrane exit Golgi complex. Multiple exit <br /> it possible Drs2 Dnf proteins transport dif-     pathways examined ALP path- <br /> ferent substrates, likely existence    way significantly perturbed drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; <br /> multiple members family reflects requirement       double mutant. A severe synthetic defect CPY <br /> these proteins different subcellular organelles. This view    pathway observed double mutant, although <br /> is supported unique localization patterns observed        defects invertase secretion kinetics pro-&#226;?&#163;-factor <br /> for Drs2p, Dnf1p, Dnf2p.                                     processing observed drs2&#226;&#338;&#172; mutant exacer- <br />                                                                  bated double mutant. <br /> Role Drs2p Dnf1p Protein Transport                        The redundant roles Drs2p Dnf1p ALP <br /> from TGN                                                         pathway somewhat surprising fraction of <br /> Drs2p Dnf1p contribute essential         Dnf1p localize TGN, Drs2p local- <br /> function Drs2/Dnf subgroup drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172;        ized. Rather, Dnf1p membranes concentrated <br /> double mutant grows slowly. In addition, dnf1&#226;&#338;&#172;              near sites polarized growth wild-type cells. These <br />  <br />  <br /> Vol. 13, September 2002                                                                                                      3173 <br />  Z. Hua et al. <br />  <br />  <br />                                                                    separate exocytic pathways, prefer model the <br />                                                                    invertase vesicles bud directly TGN targeting to <br />                                                                    polarized sites growth [Figure 10, Invertase (A)]. <br />                                                                       Surprisingly, sorting invertase Pma1p two <br />                                                                    separate exocytic pathways TGN perturbed by <br />                                                                    mutations disrupt protein trafficking through <br />                                                                    late endosome. These observations led to <br />                                                                    propose invertase sorted late endosome and <br />                                                                    packaged clathrin-coated vesicles exocytosis <br />                                                                    (Gurunathan et al., 2002; Harsay Schekman, 2002) [Fig- <br />                                                                    ure 10, Invertase (B)]. However, interpretations are <br />                                                                    possible. For example, late Golgi proteins continuously cycle <br />                                                                    TGN endosomes (Conibear Stevens, <br />                                                                    1995) strong block endosome trafficking could <br />                                                                    rapidly deplete TGN resident proteins required sort- <br />                                                                    ing invertase CCVs. In addition, direct <br />                                                                    evidence invertase passes late endosome <br />                                                                    (or prevacuolar compartment) wild-type cells is <br />                                                                    secreted. Further work required clearly define the <br />                                                                    donor compartment buds invertase vesicles. <br />                                                                       A function Drs2p vesicle budding sug- <br />                                                                    gested discovery synthetic lethal interaction <br />                                                                    drs2&#226;&#338;&#172; arf1 mutations (Chen et al., 1999). ARF <br />                                                                    implicated recruiting tetrameric adaptins (AP-1, <br />                                                                    AP-3, AP-4) clathrin TGN endosomes <br />                                                                    (Boehm et al., 2001; Ooi et al., 1998; Zhu et al., 1999). The CPY <br />                                                                    pathway use clathrin GGA adaptins, ARF- <br /> Figure 10. Model roles Drs2p Dnf proteins late <br /> secretory, vacuolar, endocytic pathways. PM, plasma mem-       dependent membrane association (Boman et al., 2000; <br /> brane; EE, early endosome; LE, late endosome.                      Hirst et al., 2000; Dell&#226;&#8364;&#8482;Angelica et al., 2000; Deloche et al., <br />                                                                    2001; Costaguta et al., 2001; Mullins Bonifacino, 2001), <br />                                                                    exit TGN. The requirements Drs2p Dnf <br />                                                                    proteins different ARF-dependent pathways are <br /> include incipient bud site, tips small buds,        interesting. Drs2p family member <br /> mother/bud neck cytokinesis, distribution            capable supporting clathrin-dependent invertase <br /> similar actin cortical patches yeast. This dynamic   pathway drs2&#226;&#338;&#172; single mutant exhibits defect in <br /> localization pattern suggests Dnf1p cycling        pathway (Figure 5B; Gall et al., 2002). However, both <br /> the exocytic endocytic pathways         Drs2p Dnf1p capable contributing the <br /> a transient occupant TGN. Because drs2&#226;&#338;&#172; partially           AP-3/ALP pathway. The CPY transport defect observed in <br /> perturbs exocytosis invertase, possible Dnf1p    drs2&#226;&#338;&#172; dnf1&#226;&#338;&#172; mutant suggests Drs2/Dnf proteins <br /> exocytosis inefficient, resulting increased resi-    contribute GGA-dependent pathway well <br /> dence TGN drs2&#226;&#338;&#172; cells. This allow Dnf1p         (Figure 10). Unfortunately, successfully <br /> partially compensate loss Drs2p ALP path-        isolated conditional alleles DRS2/DNF genes that <br /> way. We observed modest defects ALP transport           allow score immediate consequences a <br /> the drs2&#226;&#338;&#172; single mutant dnf1&#226;&#338;&#172; mutant. There-        complete loss function proteins. <br /> fore, suggest Drs2p play primary role, Dnf1p <br /> a compensatory role, budding AP-3&#226;&#8364;&#8220; coated vesicles car- <br /> rying ALP vacuole (Figure 10).                              Requirement Dnf1, 2, 3p Snc1p Recycling <br />    Clathrin recently implicated invertase          Pathway <br /> secretion pathway observation clathrin mu-           The dnf1,2,3&#226;&#338;&#172; mutant grows does exhibit a <br /> tants missort invertase dense vesicle fractions      defect ALP, CPY, invertase pathways. However, <br /> containing Pma1p (Gurunathan et al., 2002; Harsay              mutant does exhibit striking defect GFP-Snc1p <br /> Schekman, 2002). Mutations perturb actin cytoskel-        recycling. Snc1p v-SNARE thought function in <br /> eton affect invertase secretion (Novick Botstein,         exocytic endocytic vesicles, steady-state of <br /> 1985; Mulholland et al., 1997) cause accumulation       GFP-Snc1p fluorescence concentrated plasma <br /> the invertase class exocytic vesicles (Harsay               membrane wild-type cells (Gerst, 1999; Lewis et al., 2000). <br /> Bretscher, 1995). This presumably targeted          This localization pattern suggests internalization of <br /> delivery vesicles polarized sites growth re-        GFP-Snc1p plasma membrane rate limiting the <br /> quires actin. We Drs2p clathrin            recycling pathway shown Figure 10. This localization <br /> required form vesicles accumulate actin disrup-       pattern altered dnf1,2,3&#226;&#338;&#172; mutant the <br /> tion vesicles contain acid phosphatase, an-         GFP-Snc1p concentrated internal structures. Because exo- <br /> other cargo dense exocytic vesicles (Gall et al., sub-      cytosis does perturbed, suggest that <br /> mitted). Because Drs2p localizes TGN         endosome TGN transport partially defective this <br /> to site invertase Pma1p segregated           mutant. The recycling defect explain CW <br />  <br />  <br /> 3174                                                                                                   Molecular Biology Cell <br />                                                                                                 Drs2-related ATPases Protein Traffic <br />  <br />  <br /> hypersensitivity sporulation defects dnf1,2,3&#226;&#338;&#172;          best candidates plasma membrane APT yeast, <br /> mutant. Chitin synthase III (Chs3p) known dynam-          currently testing possibility. <br /> ically recycled endocytic intermediates targeting         The roles Drs2/Dnf family protein traffic yeast <br /> to polarized sites growth (Chuang Schekman, 1996;           shed light function homologs <br /> Santos Snyder, 1997). Recent work strongly suggests            implicated human disease. Both FIC1 BSEP mutations <br /> that ARF, AP-1, clathrin required transport         cause familial intrahepatic cholestasis humans. FIC1 is <br /> Chs3p early endosome TGN. Mutations              homologous DNF1 DNF2 genes yeast and <br /> these clathrin coat components results unregulated           BSEP bile salts export pump (in ABC ATPase family) <br /> transport Chs3p cell surface, increased sensi-       localized bile canalicular plasma membrane hepa- <br /> tivity cells CW (Valdivia et al., 2002). Clathrin AP-1   tocytes (Thompson Jansen, 2000; Ujhazy et al., 2001). <br /> have implicated endosome 3 TGN             How FIC1 involved bile production clear, but <br /> TGN 3 plasma membrane routes mammalian cells (Fol-              based phenotype drs2/dnf yeast mutants, is <br /> sch et al., 1999; Meyer et al., 2000; Boehm Bonifacino,        possible FIC1 required polarized de- <br /> 2001). The formation spore membranes cell walls         livery BSEP canalicular membrane. <br /> also require dynamic recycling materials. In fact, Chs3p         ATP10C, member APT family humans, <br /> is delivered prospore membranes synthesis        linked neurological disorders Angelman <br /> spore cell wall (Pammer et al., 1992), homozygous clath-       syndrome autism (Herzing et al., 2001; Meguro et al., <br /> rin mutants fail sporulate (Lemmon et al., 1990).          2001). Recycling membrane synapses plays critical <br />    The localization Dnf2p internal structures        role synaptic vesicle production, process ARF <br /> well plasma membrane suggests protein         AP-3 implicated (Faundez et al., 1998; Kan- <br /> also cycling endocytic exocytic path-           theti et al., 1998). In addition, recycling polarized <br /> ways. Thus, Dnf1p Dnf2p positioned di-             delivery AMPA receptors specific sites plasma <br /> rectly contribute clathrin/AP-1&#226;&#8364;&#8220; dependent endosome         membrane essential proper function neurons and <br /> to TGN route. Remarkably, studies position Drs2-             contribute long-term potentiation (Luscher and <br /> related P-type ATPase pathway ARF/AP/             Frerking, 2001). Interestingly, mutations mouse <br /> clathrin-dependent vesicle budding event impli-           ATP10C homolog pfatp linked increased body fat <br /> cated. In fact, observed dramatic defect          (Dhar et al., 2000). This phenotype neurological <br /> ability drs2&#226;&#338;&#172; mutants produce CCVs 15&#194;&#176;C,          basis reflect perturbations trafficking glu- <br /> normal-appearing CCV isolated 30&#194;&#176;C. We postu-            cose transporters insulin receptors adipocytes. <br /> late Dnf1, 2, 3 proteins facilitate CCV budding           The precise mechanism Drs2p Dnf <br /> the drs2&#226;&#338;&#172; mutant 30&#194;&#176;C fail lower temper-        proteins contribute protein trafficking remains elu- <br /> atures. This explain clathrin deficiency phe-        cidated. In case Drs2p, <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> aspartic acid <br /> notypes drs2&#226;&#338;&#172; severe 20&#194;&#176;C below.            critical ATPase activity strongly perturbs formation <br />                                                                    clathrin-coated vesicles targeted plasma membrane <br />                                                                    (Gall al., 2002). This suggests lipid translocation is <br /> APT Genes Other Organisms                                       required generate membrane environment per- <br />                                                                    missive vesicle budding help deform the <br /> The DRS2/NEO1 family potential APT genes exists ubiq-           membrane vesicle budding (see Chen et al., 1999, for <br /> uitously eukaryotes, members            discussion potential roles lipid translocases <br /> large family identified genetically. The PDE1 gene       vesicle budding). Further biochemical analysis these <br /> from rice blast fungus identified screen patho-       proteins required better assess role protein <br /> genicity factors similar DNF3. Mutation          trafficking. <br /> PDE1 perturbs formation penetration hyphae, narrow <br /> outgrowth cell surface used puncture plant <br /> cuticle. The Talbot group suggested Pde1p           ACKNOWLEDGMENTS <br /> play role massive recycling membrane cell <br /> wall biosynthetic enzymes required formation            We thank Scott Emr, Hugh Pelham, Jeff Flick, Steve Nothwehr <br />                                                                    antibodies, strains, plasmids. We thank Howard Wynder for <br /> penetration hyphae (Balhadere Talbot, 2001). The defect        assistance electron microscopy. We thank Walter Gall, <br /> in GFP-Snc1p recycling polarized delivery              Sophie Chen, Diane Hopkins, members Graham <br /> observed dnf1,2,3&#226;&#338;&#172; mutant consistent           laboratory help course experiments. This <br /> interpretation.                                                    work supported National Institutes Health Grant GM- <br />    The ALA1 gene Arabidopsis similar DRS2          62367) (to T.R.G.). <br /> and remarkably, decreased expression ALA1 causes cold- <br /> sensitive growth Arabidopsis. In addition, ALA1 gene <br /> can complement cold-sensitive growth defect             REFERENCES <br /> drs2&#226;&#338;&#172; mutant underscoring conservation Drs2p/Ala1           Auland, M.E., Roufogalis, B.D., Devaux, P.F., Zachowski, A. <br /> function phylogenetic boundaries. The ALA1 gene             (1994). Reconstitution ATP-dependent aminophospholipid trans- <br /> also complements cold-sensitive defect APT activity        location proteoliposomes. Proc. Natl. Acad. Sci. USA 91, 10938 &#226;&#8364;&#8220; <br /> the plasma membrane drs2&#226;&#338;&#172; cells (Gomes et al., 2000).           10942. <br /> Because Drs2p localized plasma membrane          Bahler, J., J.Q. Wu, M.S. Longtine, N.G. Shah, A. McKenzie, 3rd, A.B. <br /> is required protein transport late secretory path-      Steever, A. Wach, P. Philippsen, J.R. Pringle. (1998). Heterolo- <br /> way, defect caused mislocalization         gous modules efficient versatile PCR-based gene targeting in <br /> APTs. Our localization data suggest Dnf1 Dnf2         Schizosaccharomyces pombe. Yeast 14, 943&#226;&#8364;&#8220;951. <br />  <br />  <br /> Vol. 13, September 2002                                                                                                            3175 <br />  Z. Hua et al. <br />  <br />  <br /> Balhadere, P.V., Talbot, N.J. (2001). PDE1 Encodes P-type          Gaynor, E.C., Emr, S.D. (1997). COPI-independent anterograde <br /> ATPase involved appressorium-mediated plant infection          transport: cargo-selective ER Golgi protein transport yeast <br /> rice blast fungus Magnaporthe grisea. Plant Cell 13, 1987&#226;&#8364;&#8220;2004.          COPI mutants. J. Cell Biol. 136, 789 &#226;&#8364;&#8220; 802. <br /> Boehm, M., Aguilar, R.C., Bonifacino, J.S. (2001). Functional    Gerst, J.E. (1999). SNAREs SNARE regulators membrane <br /> physical interactions adaptor protein complex AP-4           fusion exocytosis. Cell. Mol. Life Sci. 55, 707&#226;&#8364;&#8220;734. <br /> ADP-ribosylation factors (ARFs). EMBO J. 20, 6265&#226;&#8364;&#8220; 6276. <br />                                                                          Goldstein, A., Lampen, J.O. (1975). &#226;?&#164;-d-Fructofuranoside fruc- <br /> Boehm, M., Bonifacino, J.S. (2001). Adaptins: final recount.     tohydrolase yeast. Methods Enzymol. 42, 504 &#226;&#8364;&#8220;511. <br /> Mol. Biol. Cell. 12, 2907&#226;&#8364;&#8220;2920.                                          Gomes, E., Jakobsen, M.K., Axelsen, K.B., Geisler, M., and <br /> Boman, A.L., Zhang, C., Zhu, X., Kahn, R.A. (2000). A family      Palmgren, M.G. (2000). Chilling tolerance Arabidopsis involves <br /> ADP-ribosylation factor effectors alter membrane transport      ALA1, member new family putative aminophospholipid <br /> through trans-Golgi. Mol. Biol. Cell. 11, 1241&#226;&#8364;&#8220;1255.                 translocases. Plant Cell. 12, 2441&#226;&#8364;&#8220;2454. <br />  <br /> Catty, P., A. Kerchove d&#226;&#8364;&#8482;Exaerde, A. Goffeau. (1997). The         Graham, T.R., Emr, S.D. (1991). Compartmental organization of <br /> complete inventory yeast Saccharomyces cerevisiae P-type          Golgi-specific protein modification vacuolar protein sorting <br /> transport ATPases. FEBS Lett. 409, 325&#226;&#8364;&#8220;332.                              events defined yeast sec18 (NSF) mutant. J. Cell Biol. 114, <br />                                                                          207&#226;&#8364;&#8220;218. <br /> Chen, C.Y., Ingram, M.F., Rosal, P.H., Graham, T.R. (1999). Role <br /> for Drs2p, P-type ATPase potential aminophospholipid trans-        Gurunathan, S., David, D., Gerst, J.E. (2002). Dynamin and <br /> locase, yeast late Golgi function. J. Cell Biol. 147, 1223&#226;&#8364;&#8220;1236.      clathrin required biogenesis distinct class secretory <br />                                                                          vesicles yeast. EMBO J. 21, 602&#226;&#8364;&#8220; 614. <br /> Chuang, J.S., Schekman, R.W. (1996). Differential trafficking and <br />                                                                          Halleck, M.S., Pradhan, D., Blackman, C., Berkes, C., Williamson, P., <br /> timed localization chitin synthase proteins, Chs2p Chs3p <br />                                                                          Schlegel, R.A. (1998). Multiple members subfamily of <br /> [published erratum appears J. Cell Biol. (1996) 135, 1925]. J. Cell <br />                                                                          P-type ATPases identified genomic sequences ESTs. Genome <br /> Biol. 135, 597&#226;&#8364;&#8220; 610. <br />                                                                          Res. 8, 354 &#226;&#8364;&#8220;361. <br /> Conibear, E., Stevens, T.H. (1995). Vacuolar biogenesis yeast: <br />                                                                          Harsay, E., Bretscher, A. (1995). Parallel secretory pathways to <br /> sorting sorting proteins. Cell. 83, 513&#226;&#8364;&#8220;516. <br />                                                                          cell surface yeast. J. Cell Biol. 131, 297&#226;&#8364;&#8220;310. <br /> Conibear, E., Stevens, T.H. (1998). Multiple sorting pathways        Harsay, E., Schekman, R. (2002). A subset yeast vacuolar <br /> between late Golgi vacuole yeast. Biochim. Biophys.       protein sorting mutants blocked branch exocytic <br /> Acta 1404, 211&#226;&#8364;&#8220;230.                                                      pathway. J. Cell Biol. 156, 271&#226;&#8364;&#8220;286. <br /> Costaguta, G., Stefan, C.J., Bensen, E.S., Emr, S.D., Payne, G.S.    Herzing, L.B., Kim, S.J., Cook, Jr., E.H., Ledbetter, D.H. (2001). <br /> (2001). Yeast GGA coat proteins function clathrin Golgi       The human aminophospholipid-transporting ATPase gene ATP10C <br /> endosome transport. Mol. Biol. Cell 12, 1885&#226;&#8364;&#8220;1896.                       maps adjacent UBE3A exhibits similar imprinted expression. <br /> Costanzo, M.C., et al. (2001). YPD, PombePD, WormPD. Model           Am. J. Hum. Genet. 68, 1501&#226;&#8364;&#8220;1505. <br /> organism volumes BioKnowledge library, integrated re-          Hirst, J., Lui, W.W., Bright, N.A., Totty, N., Seaman, M.N., and <br /> source protein information. Nucleic Acids Res. 29, 75&#226;&#8364;&#8220;79.            Robinson, M.S. (2000). A family proteins &#226;?&#165;-adaptin VHS <br /> Cowles, C.R., Odorizzi, G., Payne, G.S., Emr, S.D. (1997). The       domains facilitate trafficking trans-Golgi network <br /> AP-3 adaptor complex essential cargo-selective transport       vacuole/lysosome. J. Cell Biol. 149, 67&#226;&#8364;&#8220; 80. <br /> the yeast vacuole. Cell 91, 109 &#226;&#8364;&#8220;118.                                    Johnston, M., Davis, R.W. (1984). Sequences regulate the <br />                                                                          divergent GAL1-GAL10 promoter Saccharomyces cerevisiae. Mol. <br /> Dell&#226;&#8364;&#8482;Angelica, E.C., R. Puertollano, C. Mullins, R.C. Aguilar, J.D. <br />                                                                          Cell. Biol. 4, 1440 &#226;&#8364;&#8220;1448. <br /> Vargas, L.M. Hartnell, J.S. Bonifacino. (2000). GGAs: family of <br /> ADP ribosylation factor-binding proteins related adaptors         Jones, J.S., Prakash, L. (1990). Yeast Saccharomyces cerevisiae <br /> associated Golgi complex. J. Cell Biol. 149, 81&#226;&#8364;&#8220;94.             selectable markers pUC18 polylinkers. Yeast 6, 363&#226;&#8364;&#8220;366. <br /> Deloche, O., Yeung, B.G., Payne, G.S., Schekman, R. (2001).          Kantheti, P., X. Qiao, M.E. Diaz, A.A. Peden, G.E. Meyer, S.L. <br /> Vps10p transport trans-Golgi network endosome         Carskadon, D. Kapfhamer, D. Sufalko, M.S. Robinson, J.L. Noebels, <br /> mediated clathrin-coated vesicles. Mol. Biol. Cell. 12, 475&#226;&#8364;&#8220; 485.     M. Burmeister. (1998). Mutation AP-3 delta mocha <br />                                                                          mouse links endosomal transport storage deficiency platelets, <br /> Dhar, M., Webb, L.S., Smith, L., Hauser, L., Johnson, D., West,      melanosomes, synaptic vesicles. Neuron 21, 111&#226;&#8364;&#8220;122. <br /> D.B. (2000). A novel ATPase mouse chromosome 7 candidate <br /> gene increased body fat. Physiol. Genomics 4, 93&#226;&#8364;&#8220;100.                Klionsky, D.J., Banta, L.M., Emr, S.D. (1988). Intracellular sort- <br />                                                                          ing processing yeast vacuolar hydrolase: proteinase A <br /> Donaldson, J.G., Jackson, C.L. (2000). Regulators effectors      propeptide contains vacuolar targeting information. Mol. Cell. Biol. <br /> of ARF GTPases. Curr. Opin. Cell Biol. 12, 475&#226;&#8364;&#8220; 482.                 8, 2105&#226;&#8364;&#8220;2116. <br /> Faundez, V., Horng, J.T., Kelly, R.B. (1998). A function     Klionsky, D.J., Emr, S.D. (1989). Membrane protein sorting: <br /> AP3 coat complex synaptic vesicle formation endosomes.           biosynthesis, transport processing yeast vacuolar alkaline <br /> Cell. 93, 423&#226;&#8364;&#8220; 432.                                                      phosphatase. EMBO J. 8, 2241&#226;&#8364;&#8220;2250. <br /> Folsch, H., Ohno, H., Bonifacino, J.S., Mellman, I. (1999). A        Lemmon, S.K., Freund, C., Conley, K., Jones, E.W. (1990). Ge- <br /> novel clathrin adaptor complex mediates basolateral targeting         netic instability clathrin-deficient strains Saccharomyces cerevi- <br /> polarized epithelial cells. Cell. 99, 189 &#226;&#8364;&#8220;198.                          siae. Genetics 124, 27&#226;&#8364;&#8220;38. <br /> Fuller, R.S., Sterne, R.E., Thorner, J. (1988). Enzymes required     Lewis, M.J., Nichols, B.J., Prescianotto-Baschong, C., Riezman, H., <br /> for yeast prohormone processing. Annu. Rev. Physiol. 50, 345&#226;&#8364;&#8220;362.        Pelham, H.R. (2000). Specific retrieval exocytic SNARE <br />                                                                          Snc1p early yeast endosomes. Mol. Biol. Cell. 11, 23&#226;&#8364;&#8220;38. <br /> Gall, W.E., Geething, N.C., Hua, Z., Ingram, M.F., Liu, K., Chen, S., <br /> and Graham, T.R. (2002). Drs2p-dependent budding clathrin-            Li, Y., Moir, R.D., Sethy-Coraci, I.K., Warner, J.R., Willis, I.M. <br /> coated vesicles vivo. (in press).                                     (2000). Repression ribosome tRNA synthesis secretion- <br />  <br />  <br /> 3176                                                                                                             Molecular Biology Cell <br />                                                                                                          Drs2-related ATPases Protein Traffic <br />  <br />  <br /> defective cells signaled novel branch cell integrity       yeast vps28 vacuolar protein sorting mutant. Mol. Biol. Cell. 7, <br /> pathway. Mol. Cell. Biol. 20, 3843&#226;&#8364;&#8220;3851.                                  985&#226;&#8364;&#8220;999. <br /> Luscher, C., Frerking, M. (2001). Restless AMPA receptors.            Ripmaster, T.L., Vaughn, G.P., Woolford, Jr., J.L. (1993). DRS1 <br /> implications synaptic transmission plasticity. Trends Neu-        DRS7, novel genes required ribosome assembly function <br /> rosci. 24, 665&#226;&#8364;&#8220; 670.                                                      Saccharomyces cerevisiae. Mol. Cell. Biol. 13, 7901&#226;&#8364;&#8220;7912. <br /> Marx, U., Polakowski, T., Pomorski, T., Lang, C., Nelson, H., Nel-        Russell, D.W., Jensen, R., Zoller, M.J., Burke, J., Errede, B., Smith, M., <br /> son, N., Herrmann, A. (1999). Rapid transbilayer movement          Herskowitz, I. (1986). Structure Saccharomyces cerevisiae <br /> fluorescent phospholipid analogues plasma membrane              HO gene analysis upstream regulatory region. Mol. Cell. <br /> endocytosis-deficient yeast cells does require Drs2 protein.      Biol. 6, 4281&#226;&#8364;&#8220; 4294. <br /> Eur. J. Biochem. 263, 254 &#226;&#8364;&#8220;263. <br />                                                                           Santos, B., Snyder, M. (1997). Targeting chitin synthase 3 to <br /> Meguro, M., Kashiwagi, A., Mitsuya, K., Nakao, M., Kondo, I.,             polarized growth sites yeast requires Chs5p Myo2p. J. Cell <br /> Saitoh, S., Oshimura, M. (2001). A novel maternally expressed         Biol. 136, 95&#226;&#8364;&#8220;110. <br /> gene, ATP10C, encodes putative aminophospholipid translocase <br /> associated Angelman syndrome. Nat. Genet. 28, 19 &#226;&#8364;&#8220;20.                Schandel, K.A., Jenness, D.D. (1994). Direct evidence ligand- <br />                                                                           induced internalization yeast &#226;?&#163;-factor pheromone receptor. <br /> Meyer, C., Zizioli, D., Lausmann, S., Eskelinen, E.L., Hamann, J.,        Mol. Cell. Biol. 14, 7245&#226;&#8364;&#8220;7255. <br /> Saftig, P., von Figura, K., Schu, P. (2000). &#226;?&#174;1A-adaptin-deficient <br /> mice, lethality, loss AP-1 binding rerouting mannose            Seeger, M., Payne, G.S. (1992). Selective immediate effects <br /> 6-phosphate receptors. EMBO J. 19, 2193&#226;&#8364;&#8220;2203.                             clathrin heavy chain mutations Golgi membrane protein re- <br />                                                                           tention Saccharomyces cerevisiae. J. Cell Biol. 118, 531&#226;&#8364;&#8220;540. <br /> Moller, J.V., Juul, B., le Maire, M. (1996). Structural organiza- <br /> tion, ion transport, energy transduction P-type ATPases.           Sherman, F. (1991). Getting started yeast. Methods Enzymol. <br /> Biochim. Biophys. Acta 1286, 1&#226;&#8364;&#8220;51.                                        194, 3&#226;&#8364;&#8220;21. <br /> Mulholland, J., Wesp, A., Riezman, H., Botstein, D. (1997). Yeast     Siegmund, A., Grant, A., Angeletti, C., Malone, L., Nichols, J.W., <br /> actin cytoskeleton mutants accumulate new class Golgi-derived        Rudolph, H.K. (1998). Loss Drs2p does abolish transfer of <br /> secretary vesicle. Mol. Biol. Cell. 8, 1481&#226;&#8364;&#8220;199.                          fluorescence-labeled phospholipids plasma membrane of <br />                                                                           Saccharomyces cerevisiae. J. Biol. Chem. 273, 34399 &#226;&#8364;&#8220;34405. <br /> Mullins, C., Bonifacino, J.S. (2001). Structural requirements for <br /> function yeast GGAs vacuolar protein sorting, alpha-factor          Stevens, T., Esmon, B., Schekman, R. (1982). Early stages the <br /> maturation, interactions clathrin. Mol. Cell. Biol. 21, 7981&#226;&#8364;&#8220;    yeast secretory pathway required transport carboxypep- <br /> 7994.                                                                     tidase Y vacuole. Cell. 30, 439 &#226;&#8364;&#8220; 448. <br /> Nierras, C.R., Warner, J.R. (1999). Protein kinase C enables      Tang, X., Halleck, M.S., Schlegel, R.A., Williamson, P. <br /> </body></html>