<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Further understanding human disease genes comparing with <br /> housekeeping genes genes <br /> Zhidong Tu&#226;&#8364;&#160;, Li Wang&#226;&#8364;&#160;, Min Xu, Xianghong Zhou, Ting Chen and <br /> Fengzhu Sun* <br />  <br /> Address: Molecular Computational Biology Program, University Southern California, Los Angeles, California 90089, USA <br /> Email: Zhidong Tu - ztu@usc.edu; Li Wang - wang7@usc.edu; Min Xu - mxu@usc.edu; Xianghong Zhou - xjzhou@usc.edu; <br /> Ting Chen - tingchen@usc.edu; Fengzhu Sun* - fsun@usc.edu <br /> * Corresponding author &#226;&#8364;&#160;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 21 February 2006                                                   Received: 04 October 2005 <br />                                                                               Accepted: 21 February 2006 <br /> BMC Genomics 2006, 7:31    doi:10.1186/1471-2164-7-31 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/7/31 <br /> &#194;&#169; 2006 Tu et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Several studies compared various features heritable disease genes with <br />                  called non-disease genes, yielded conflicting results. A potential <br />                  problem studies non-disease genes contained large number essential genes <br />                  &#226;&#8364;&#8220; genes indispensable humans survive reproduce. Since functional disruption <br />                  essential gene fatal consequences, it&apos;s reasonable regard essential genes as <br />                  extremely severe &quot;disease&quot; genes. Here perform comparative study features human <br />                  essential, disease, genes. <br />                  Results: In absence set defined human essential genes, consider set 1,789 <br />                  ubiquitously expressed human genes (UEHGs), known housekeeping genes, an <br />                  approximation. We demonstrate UEHGs likely contain large proportion of <br />                  essential genes. We UEHGs, disease genes genes different their <br />                  evolutionary conservation rates, DNA coding lengths, gene functions, etc. Our findings <br />                  systematically confirm disease genes intermediate essentiality than <br />                  housekeeping genes greater human genes. <br />                  Conclusion: The human genome contain thousands essential genes having features which <br />                  differ significantly disease genes. We propose classify unique group for <br />                  comparisons disease genes non-disease genes. This new way classification and <br />                  comparison enables clearer understanding disease genes. <br />  <br />  <br />  <br />  <br /> Background                                                                    Smith et al. [3] disease genes evolved with <br /> Identification novel genes associated human dis-                      higher nonsynonymous/synonymous substitution rate <br /> eases critical tasks medical research.                   ratios (Ka/Ks) non-disease genes, Huang et al. [4] <br /> Towards goal, various features compared                        significant differences. One common prob- <br /> between heritable disease genes non-disease genes [1-                     lem studies human essential genes were <br /> 4]. Although findings consistent                          ignored simply grouped non-dis- <br /> other, conflicting results showed up. For example,                      ease genes. Essential genes genes functions are <br />  <br />  <br />                                                                                                                                        Page 1 13 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                      http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> necessary organism survive reproduce. Since     genes. We validate set UEHGs con- <br /> the disruption essential genes&apos; function cause fatal   tains large fraction essential genes. Then compar- <br /> consequences, regarded severe       ing groups genes, disease <br /> &quot;disease&quot; genes. Therefore, comparing disease genes       genes distinguished essential genes. <br /> mixture essential non-disease genes reduce     If UEHGs really contain greater fraction essential <br /> clarity signals disease-related features     genes non-UEHGs (i.e. disease genes), we <br /> may lead erroneous findings. Thus, beneficial    expect observe followings. First, essential genes <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> separate human essential genes non-disease       functionally extremely important, selective pres- <br /> genes comparisons made.                             sure higher non-essential <br />                                                                genes, UEHGs slower evolutionary rate <br /> Thousands genes identified essential           disease genes [12,13]. Second, since <br /> genes multiple model organisms, Saccharomyces       Mendelian diseases caused deleterious amino <br /> cerevisiae, Caenorhabditis elegans, Mus musculus [5-7].    acid substitutions, study conservation amino <br /> Although certain human genome       acid level, expect different patterns UEHGs, <br /> contains hundreds thousands essential genes, it&apos;s        disease genes. Third, UEHGs mapped <br /> impractical experimentally determine S. cer-     species, homologous genes more <br /> evisiae C. elegans. The absence set well-defined    likely essential species species evolu- <br /> human essential genes poses challenge studying            tionarily close humans. Fourth, essential proteins <br /> them urges alternative solutions.                      usually tend hub proteins (highly connected) the <br />                                                                protein-protein interaction network [14], UEHGs should <br /> The human genome extremely complex tissue               higher average physical interaction degree than <br /> expression profile. Some genes expressed cer-      non-UEHGs. Fifth, functions UEHGs be <br /> tain tissues specific times, consti-   fundamentally important. To verify hypotheses, we <br /> tutively ubiquitously expressed [8,9]. For      compile lists UEHGs, disease genes other <br /> genes, presumed necessary fun-     human genes. We collect various features com- <br /> damental cellular physiological processes referred     pare selected features gene classifi- <br /> as housekeeping genes [9]. Housekeeping genes             cations. <br /> been studied researchers interesting <br /> observations reported. For example, Zhang        Comparison evolutionary features <br /> Li housekeeping genes evolved slowly           We compare Ka, Ks ratio (Ka/Ks) based on <br /> than tissue-specific genes [10]. Eisenberg Levanon         three-way classification human genome. The <br /> found housekeeping genes compact            Ka, Ks Ka/Ks derived human-rat and <br /> coding lengths, result higher selec-     human-mouse orthologous pairs. The results obtained <br /> tive pressure[11]. Based unique properties       human-mouse orthologous pairs indicate that <br /> ubiquitously expressed human genes (UEHGs),                 UEHGs smallest Ka, Ks Ka/Ks ratio the <br /> believe suitable candidates essential        groups (P-values UEHGs vs. disease 3.4E-39, <br /> genes. Although hypothesis intuitive sounds        6.3E-15, 1.7E-38; P-values UEHGs vs. are <br /> reasonable, efforts required collect support-   5.3E-64, 9.0E-27, 1.3E-57, respectively Ka, Ks and <br /> ive evidence systematic level.                            Ka/Ks), disease genes lower evolutionary rates <br />                                                                genes (P-values 9.1E-5, 5.5E-4 2.6E-4 <br /> In study, consider set 1,789 ubiquitously         Ka, Ks Ka/Ks, respectively) (Fig 1). By various sta- <br /> expressed human genes (UEHGs) approximation          tistical measurements, UEHGs consistently stand as <br /> essential genes. We demonstrate UEHGs            slowest evolved gene group difference <br /> likely contain large proportion essential genes    UEHGs groups greater than <br /> thus approximate human essential genes. By perform-        difference disease genes genes <br /> ing three-way feature comparison UEHGs (presumed          (Table 1). The results similar human-rat orthol- <br /> essential genes), disease genes, rest human         ogous pairs used calculate Ka Ks, P-val- <br /> genes (referred genes), dif-    ues slightly significant. Again, disease genes evolve <br /> ferent aspects evolutionary conserva-      slower rates genes significant differ- <br /> tion rates, DNA coding lengths, gene functions, etc.           ences Ka, Ks Ka/Ks (P-values 0.008, 0.052 and <br />                                                                0.026, respectively). UEHGs evolve slowest rates <br /> Results                                                        differences Ka, Ks Ka/Ks strongly signif- <br /> Instead dividing human genome disease vs.          icant (P-values UEHGs vs. disease 7.0E-32, 3.2E- <br /> non-disease genes, choose three-way classification,       13, 2.7E-28; P-values UEHGs vs. 1.1E- <br /> namely, UEHGs (presumed &quot;essential&quot;), disease,             49, 4.8E-23, 1.2E-44, respectively, Ka, Ks Ka/ <br />  <br />  <br />                                                                                                                    Page 2 13 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                         http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                          (a) <br />  <br />  <br />  <br />  <br />                          (b) <br />  <br />  <br />  <br />  <br /> Figure 1 Ka, Ks Ka/Ks <br /> Distribution <br /> Distribution Ka, Ks Ka/Ks (a) The cumulative density Ka, Ks Ka/Ks derived human-mouse orthologous <br /> pairs. Ka, number non-synonymous substitutions non-synonymous sites. Ks, number synonymous substitu- <br /> tions synonymous site, Ka/Ks ratio. Three groups human genes represented different colors the <br /> number genes group listed right line symbols. (b) The box plots drawn based data. For each <br /> category, central box depicts middle 50% data 25th 75th percentile, enclosed red hor- <br /> izontal line represents median value distribution. Extreme values indicated solid blue dots occur outside <br /> the main bodies data. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 3 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                   http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 1: Comparison evolutionary rate groups genes <br />  <br />                                 UEHGs                                   Disease genes                               Other genes <br />  <br />                 Mean(SEM)       Median         Range       Mean(SEM)       Median         Range       Mean(SEM)        Median          Range <br />  <br />  Ka                0.053         0.033        0&#226;&#8364;&#8220;0.48         0.080          0.061         0&#226;&#8364;&#8220;0.70         0.092          0.066          0&#226;&#8364;&#8220;1.37 <br />                  (0.0015)                                   (0.0017)                                   (8.0E-4) <br />  Ks                0.58          0.57        0.076&#226;&#8364;&#8220;2.0        0.63          0.61         0.14&#226;&#8364;&#8220;2.4        0.66           0.62         0.041&#226;&#8364;&#8220;4.2 <br />                  (0.0047)                                   (0.0043)                                   (0.0022) <br />  Ka/Ks             0.088         0.059        0&#226;&#8364;&#8220;0.59          0.12          0.10          0&#226;&#8364;&#8220;0.70         0.13           0.11            0&#226;&#8364;&#8220;1.2 <br />                  (0.0023)                                   (0.0023)                                    (0.001) <br />  <br />  Ka, number non-synonymous substitutions non-synonymous sites. Ks, number synonymous substitutions synonymous site. Ka <br />  Ks values calculated based human-mouse orthologous pairs described main text. The column group mean <br />  followed standard error mean (SEM). <br />  <br />  <br /> Ks) (Fig S2). Here, results different find-              conservation score specific amino acids based a <br /> ings Smith [3] Huang [4], described before,                 large-scale multiple-sequence alignment 8 species per- <br /> their results consistent either. We               formed recently UCSC research group [22]. The <br /> think partly effect mixing essential                conservation scores derived two-state phylo- <br /> genes non-disease genes previous studies.                     Hidden Markov Model interpreted proba- <br /> Another reason different groups used different set                bilities <span id='am-165' about='xsp:base' typeof='owl:Thing'>base</span> conserved hidden state. <br /> of disease non-disease genes (Smith et al. studied 387                We collect list 6,000 disease <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> sites <br /> disease 2,024 non disease genes, Huang et al. studied                 1,900 polymorphism (neutral) mutation sites <br /> 1,112 disease genes 10,000 non-disease                      SwissProt. We compare conservation these <br /> genes dataset covered &gt;1,700 disease genes                    types sites with back- <br /> &gt;12,000 non-disease genes, supplementary materials                    ground (The background obtained considering the <br /> for detailed comparison). Our results indicate                     conservation score amino acids coding <br /> UEHGs strongest selection pressure. Disease                 regions). As shown Fig 2 (a, b), polymorphism muta- <br /> genes evolve intermediate rate slower                 tion sites significantly biased conserved <br /> other genes, faster UEHGs. Our experiments con-                  sites disease mutation sites significantly biased <br /> sider larger gene sets significant number                   conserved sites (p-values &lt; 10-5 both <br /> essential genes separated non-disease genes.                 cases). This consistent findings Miller and <br /> Therefore, results better reflect relationship             Kumar focused seven disease genes <br /> disease genes genes. The slightly different                    [23]. Then compare UEHGs disease genes and <br /> results calculated rat mouse likely                 results shown Fig 2(c). UEHGs conserved <br /> to different evolutionary rates mouse rat                 disease genes conservation score disease <br /> after divergence 16 million years ago [15].                  mutation sites greater UEHGs. Since we <br /> We confirm positive correlation Ka                   don&apos;t information &quot;essential sites&quot;, unable <br /> Ks gene groups, observed previous                  directly compare &quot;essential sites&quot; disease sites. <br /> studies [16-20]. For example, Ka Ks disease                    Instead, think possible mechanism distin- <br /> genes correlation coefficient 0.45 (P-value                  guishing essential genes disease genes essen- <br /> 1.9E-86 t-test). Although synonymous mutation                    tial genes contain larger fraction highly conserved <br /> no apparent effect protein sequence, affect                 sites (with underlying assumption highly con- <br /> DNA structure, mRNA structure, biological processes                   served sites correspond functionally important loci). <br /> such transcription RNA splicing [21]. The small                    Thus, chance random mutation cause a <br /> average Ks UEHGs indicates synonymous sites                   severe <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> higher essential genes <br /> of UEHGs stronger selection                 disease genes. We select conservation score 0.9 as <br /> two groups, suggests functional impor-                  cut-off value define sites conservation scores <br /> tance UEHGs.                                                           highly conserved sites. The cut-off chosen <br />                                                                           based distribution conservation scores of <br /> As Ka Ks summary statistics nucleotide sub-                disease mutation sites. Different cut-off values tested <br /> stitution rate gene, Mendelian diseases                 results similar. We calculate fraction highly <br /> caused amino acid substitutions coding regions,                  conserved sites coding region distri- <br /> will informative study <span id='am-166' about='xsp:pattern' typeof='owl:Thing'>pattern</span> conserva-                bution fraction UEHGs, disease genes other <br /> tion individual nucleotides amino acids. We obtain                 genes Figure 2(d). It&apos;s clear UEHGs contains a <br />  <br />  <br />                                                                                                                                   Page 4 13 <br />                                                                                                             (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                         80          Disease gene sites                                             80           Background (all sites) <br />                                     Disease mutation sites                                                          Polymorphic mutation sites <br />  <br />  <br />  <br />                         60                                                                             60 <br />         Percentage <br />  <br />  <br />  <br />  <br />                                                                                           Percentage <br />                         40                                                                             40 <br />  <br />  <br />  <br />  <br />                         20                                                                             20 <br />  <br />  <br />  <br />  <br />                         0                                                                               0 <br />                              0.0   0.2              0.4           0.6   0.8   1.0                            0.0      0.2             0.4           0.6                0.8            1.0 <br />                                                   Conservation Score                                                                Conservation Score <br />   (a)                                                                               (b) <br />  <br />  <br />  <br />  <br />                                                                                                                       UEHGs <br />                                          UEHGs sites                                               2.0            disease genes <br />                         80                                                                                            genes <br />                                          Disease mutation sites <br />                                          Disease gene sites <br />  <br />  <br />                                                                                                        1.5 <br />                         60 <br />            Percentage <br />  <br />  <br />  <br />  <br />                                                                                           Density <br />  <br />  <br />  <br />  <br />                         40                                                                             1.0 <br />  <br />  <br />  <br />  <br />                         20                                                                             0.5 <br />  <br />  <br />  <br />  <br />                         0                                                                              0.0 <br />                              0.0   0.2              0.4           0.6   0.8   1.0                            0.0     0.2            0.4          0.6             0.8            1.0 <br />                                                    Conservation score                                                       Fraction higly conserved region <br />   (c)                                                                               (d) <br />  <br />  <br /> Figure conservation <br /> Codon   2               gene groups <br /> Codon conservation gene groups. The conservation score amino acids groups genes are <br /> compared. (a) The distribution disease causing mutation sites&apos; conservation score plotted solid line. The dotted line <br /> is drawn based conservation scores sites coding region (i.e., distribution conservation score <br /> when sites randomly chosen). (b) The distribution polymorphism mutation sites&apos; conservation score vs. random dis- <br /> tribution (a). (c) The distribution conservation score UEHGs (black line), disease gene (broken blue line) dis- <br /> ease causing mutation sites (red broken line). (d) The distribution fraction highly conserved regions (Cons. <br /> Score&gt;0.9). Each human gene group represented different color. <br />  <br />  <br /> much higher fraction highly conserved sites                             achieve. Unfortunately, knowledge gene essential- <br /> other groups, significant difference                          ity high animals limited. For example, <br /> between disease genes genes (p-value = 0.32).                             number mouse genes known mutation pheno- <br />                                                                                     types 10% genome (~2,800 Mouse <br /> Cross-species comparison gene deletion phenotypes                                Genome Informatics database) heavily <br /> Many human diseases studied experimenting                                 biased homologs human disease genes <br /> model organisms mouse. The underneath ration-                               (results shown). By now, S. cerevisiae C. ele- <br /> ale homologous genes similar functions                          gans explored gene essentialities the <br /> the species evolutionarily close. Similarly,                            genome scale [5,6]. Here compare human <br /> essentiality human genes tested evolution-                          genome them, favoured the <br /> arily close species. This approach work                             far evolutionarily distances and <br /> genes, differences species, however,                             humans. Human genes mapped yeast worm <br /> closer species are, higher accuracy                              based homologous relationship. For UEHGs, disease <br />  <br />  <br />                                                                                                                                                                              Page 5 13 <br />                                                                                                                                        (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure <br /> Comparison <br />          3 gene essentiality human C. elegans <br /> Comparison gene essentiality human C. elegans Human genome divided groups as <br /> described main text 20,488 C. elegans genes mapped group based homology. The essentiality C. ele- <br /> gans gene obtained RNAi-interference experiment described main text. Different phenotypes represented <br /> by different colors number homologs group listed. The fraction human genes C. elegans <br /> homologs shown group name. <br />  <br />  <br />  <br /> genes genes, examine fraction genes          UEHGs highest fraction, followed disease <br /> which homologous essential mapped spe-            genes, genes. However, shown the <br /> cies (Table 2 Fig 3). The column Table 2 shows      column, conditional homologous genes, the <br /> that UEHGs contain largest fraction genes           fraction essential genes group does strictly <br /> have homologous counterparts yeast. Disease genes             follow order (i.e., UEHGs&gt;disease genes&gt;other <br /> have smaller fraction UEHGs greater          genes). This prominent yeast C. elegans. <br /> genes. The order observed C. elegans           Therefore phenomenon likely caused the <br /> shown Fig 3. Since yeast C. elegans evolutionary      larger evolutionary distance yeast human. Or <br /> distant human, results support UEHGs con-          say given gene highly conserved (such as <br /> tain greater fraction functionally important genes          human-yeast <span id='am-162' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span>), essentiality the <br /> than disease genes. The second column Table         gene longer strongly linked group the <br /> 2 shows fraction homologous genes           gene belongs to. In addition, results suggest the <br /> groups essential yeast. Again, that <br />  <br />  <br />                                                                                                                      Page 6 13 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                    http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 2: Cross-species comparison gene essentiality human S. cerevisiae. <br />  <br />                                                Total                                 Essential                     Non-essential       Unknown <br />  <br />  Homologs UEHGs                  384 (20.2% UEHGs)*                138 (35.9%) (7.3% UEHGs)*                242(63.0%)         4(1.1%) <br />  Homologs disease genes        196 (9.8% disease genes)*         51 (26.0%) (2.5% disease genes)*           142(72.5%)         3(1.5%) <br />  Homologs genes          1005 (3.5% genes)*          379 (37.7%) (1.3% genes)*           618(61.5%)         8(0.8%) <br />  No homologs                                 4641                                  505 (10.9%)                      3427(73.8%)       709(15.3%) <br />  Total yeast genes                           6179                                 1058 (17.1%)                      4397(71.2%)       724(11.7%) <br />  <br />  Yeast genes mapped human gene groups homologous mapping. The rows human gene group fourth <br />  row yeast genes human homologous genes. The row summarizes yeast genes. The column lists number of <br />  homologous genes human gene group fractions respect number human genes corresponding group <br />  (show brackets marked *). The following columns list number yeast essential, non-essential, unknown genes human gene <br />  group. The fractions respect number homologs group shown brackets. In second column, fraction of <br />  homologous essential genes human genes group given bracket *. <br />  <br />  <br />  <br /> highly conserved genes contain large fraction essential                 elders. People usually think genes associated early <br /> genes too.                                                                 onset diseases higher selection pressure than <br />                                                                            associated late onset diseases. If correct, <br /> Comparison features                                               essential genes critical, evolutionary rates <br /> It noticed years that, protein                 similar early onset disease genes rather <br /> interaction network, essential proteins tend interact                   late onset ones. To verify this, divide disease <br /> with proteins model organisms [14]. Peri et al.                    genes based onset age Jimenez et al. [25] and <br /> manually collected 27,000 interactions involv-                   compare Ka/Ks ratio. Figure 6 shows evolutionary <br /> ing 18,000 human proteins [24]. Although data                   rates tend increase onset age larger. <br /> set quite sparse, accuracy assumed high.             The correlation coefficient onset age the <br /> The distribution protein physical interaction degree                 Ka/Ks ratio positive P-value 0.02 based on <br /> the UEHGs, disease genes shown Fig-                   weighted squares regression [26]. (Weighted least <br /> ure 4. The average degree &#194;&#177; standard error UEHGs, dis-                 square used different age groups contain une- <br /> ease genes 11.0 &#194;&#177; 0.6,8.4 &#194;&#177; 0.4 6.2 &#194;&#177;                qual number genes non-constant variances, by <br /> 0.1, respectively. We consider proteins                 introducing weight regression, effects be <br /> one interaction, 0 degree mean inter-                reduced.) Base visual inspection, Fig. 6 suggests <br /> action absence data unable make                    UEHGs similar Ka/Ks ratios genes <br /> distinction. We acknowledge data collection                       responsible diseases uterus genes have <br /> could biased human disease genes. However,                      similar Ka/Ks ratios genes associated late onset <br /> as set UEHGs defined purely based gene                       diseases. However, regression performed the <br /> expression independent source, it&apos;s unlikely                    group number actual disease onset age <br /> have heavy bias UEHGs. Thus, high interac-                   (the original linear relationship different disease <br /> tion degrees UEHGs regarded supporting                      onset ages distorted extent), P- <br /> evidence essentiality.                                           value just passes stringent cut-off (i.e., 0.05), more <br />                                                                            data analysis needed draw con- <br /> We investigate function annotation                   fident conclusion results. <br /> groups genes. As shown Figure 5, UEHGs, disease <br /> genes genes distinct function distribu-                 Discussion <br /> tions. UEHGs enriched protein biosynthesis                      All results support UEHGs themselves <br /> several fundamentally important physiological                        form distinct group disease genes. The results <br /> processes, disease genes relevant sens-                  endorse UEHGs contain large proportion <br /> ing responding internal/external signals,                 functionally essential genes. Although try show <br /> advanced mechanisms fine tuning certain bio-                    UEHGs good candidates human essential <br /> logical processes.                                                         genes, intention claim only <br />                                                                            best gene set representing human essential <br /> Finally, look relationship gene&apos;s conser-                genes. Because gene needs ubiquitously expressed <br /> vation onset age disease. Different diseases                    considered UEHG, low expressed somehow <br /> exhibit symptoms different ages. Some diseases                    tissue specific expressed essential genes excluded. <br /> develop early utero present                    Also, tissue samples collected mainly from <br />  <br />  <br />                                                                                                                                     Page 7 13 <br />                                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                               http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                    70% <br />  <br />  <br />  <br />                    60% <br />  <br />  <br />  <br />                    50% <br />                                                                                                                UEHGs(762) <br />                                                                                                                Disease(1168) <br />                    40%                                                                                         Others(4063) <br />         Fraction <br />  <br />  <br />  <br />  <br />                    30% <br />  <br />  <br />  <br />                    20% <br />  <br />  <br />  <br />                    10% <br />  <br />  <br />  <br />                    0% <br />                          1-5   6-10   11-15   16-20   21-25   26-30   31-35   36-40   41-45   46-50    51-55   56-60   61-65   66-70   71-75   &gt;76 <br />  <br />                                                               Protein-protein interaction degree <br />  <br />  <br />  <br />  <br /> Figure 4 protein physical interaction degrees <br /> Distribution <br /> Distribution protein physical interaction degrees. UEHGs, disease genes, genes shown differ- <br /> ent colors histogram. It seen interaction degree increases, fraction UEHGs increases. For <br /> the summary statistics, main text. The number genes interaction HPRD listed gene group. <br />  <br />  <br />  <br />  <br /> adult individuals, genes essential early stage                      Different previous studies human housekeeping <br /> development missed too. As revealed cross-                          gene, define UEHGs genes expressed &quot;almost <br /> species comparison, UEHGs failed cover                                all&quot; (not &quot;exactly all&quot;) tissues examined. Due <br /> many essential genes genes classified                      fluctuation gene expression error the <br /> other genes. We study different set genes consider-                       gene expression measurement, tissues being <br /> ing genes conserved yeast, C. elegans                         examined, fewer genes observed expressed all <br /> human. The results indicate contain large                         tissues. We relax criteria allow missing expres- <br /> fraction essential genes (results shown). How-                         sion small fraction tissues size UEHGs <br /> ever, pointed Chervitz et al. [27], set                        sensitive number tissues examined. <br /> miss human essential genes don&apos;t                                  Also different cutoff value expression level was <br /> homologs yeast C. elegans. In contrast, UEHGs                         adopted. In order verify results sensitive <br /> more unbiased sample essential genes. A combi-                           specific criteria used define UEHGs, prepare <br /> nation UEHGs conserved genes generate                              UEHGs set defined genes expressed more <br /> more complete set candidates human essential                               300 standard units 79 tissues. This leads to <br /> genes. We realize set disease genes                       2,038 genes grouped UEHGs, 1,509 genes <br /> study mainly genes associated Mendelian dis-                             contained original set 1,789 genes. The evo- <br /> eases, complex disease genes under-represented.                         lutionary rates compared new set of <br />  <br />  <br />                                                                                                                                                Page 8 13 <br />                                                                                                                          (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                          http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure 5annotation genes groups <br /> Function <br /> Function annotation genes groups. GO categories described row labels columns the <br /> three classes genes. A color scheme (scale shown right) used display significance level over-representation <br /> (numbered negative logarithm P-value, upper half scale) under-representation (numbered logarithm the <br /> P-value, lower half scale) certain gene group function category. Hyper-geometric distribution used cal- <br /> culation P-value. <br />  <br />  <br /> UEHGs, disease genes genes. The results              regions shorter genes. But unclear dis- <br /> almost identical slight changes        ease genes generally longer genes. One pos- <br /> P-values (see details supplementary materials). This            sible explanation functions disease <br /> indicates findings sensitive criteria      genes performed proteins certain <br /> for defining UEHGs.                                            lengths. For example, ion channel proteins (e.g. <br />                                                                    cystic fibrosis transmembrane conductance regulator, <br /> Previous studies shown house-keeping genes               CFTR) need span membrane multiple <br /> have shorter coding <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> [11] disease genes usu-           times form pore structure, task be <br /> ally longer coding length [1]. We confirm find-         fulfilled short protein. Further studies needed to <br /> ings three-way comparison (Table 3). Since UEHGs              explore general cases are. <br /> are required expressed tissues constitu- <br /> tively, it&apos;s beneficial intron untranslated <br />  <br />  <br />                                                                                                                         Page 9 13 <br />                                                                                                   (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 3: Comparison length various parts UEHGs, disease genes, genes. <br />  <br />                                   UEHGs (n = 1400)               Disease (n = 1773)            Others (n = 10304)                   P-value <br />  <br />  Coding sequence length                 1501 &#194;&#177; 38                     2205 &#194;&#177; 73                     1849 &#194;&#177; 15                        3.2E-09 <br />                                           1109                          1537                           1459 <br />  Total exon length                      2545 &#194;&#177; 48                     3250 &#194;&#177; 78                     2752 &#194;&#177; 18                        1.6E-06 <br />                                           2136                          2557                           2343 <br />  Number exons                        10.7 &#194;&#177; 0.2                    13.5 &#194;&#177; 0.3                     9.9 &#194;&#177; 0.1                       9.6E-07 <br />                                             8                             10                             7 <br />  Total intron length                  35698 &#194;&#177; 1558                  60376 &#194;&#177; 2836                   54881 &#194;&#177; 1139                       0.012 <br />                                           15588                         23528                          18540 <br />  5&apos; UTR length                          546 &#194;&#177; 22                      582 &#194;&#177; 21                        560 &#194;&#177; 8                         0.55 <br />                                            238                           251                            245 <br />  3&apos; UTR length                          559 &#194;&#177; 21                      569 &#194;&#177; 21                        575 &#194;&#177; 8                         0.86 <br />                                            247                           243                            254 <br />  <br />  For row line gives average value &#194;&#177; s.e.m, second line gives median. UTR stands untranslated region. Pair-wise rank <br />  sum tests performed largest P-values listed column. <br />  <br /> We want point that, shown Fig 6,                  essential genes. We believe updated global picture <br /> no sharp dividing lines essential genes, disease                        disease genes enable better identify in <br /> genes genes. Some diseases simply lethal                        future [32]. <br /> and associated genes essential genes defini- <br /> tion. Some diseases severe effects it&apos;s                    Methods <br /> hard distinguish true non-disease genes.                         Compiling lists disease genes UEHGs list <br /> Thus, gene essentiality better described                    The list disease genes obtained OMIM [29]. <br /> continuous spectrum artificially divided                       3,962 records listed morbidmap (Jun 6, 2005) <br /> groups. Even complicated situations arise dif-                      entries known sequence (OMIM ID marked with <br /> ferent mutation forms considered. Since different                         *), known sequence phenotype (OMIM ID <br /> mutations usually lead phenotypes different severi-                     marked #), phenotype description, molec- <br /> ties [28,29], disease gene non-essential                  ular basis known (OMIM ID marked +) were <br /> gene essential gene non-lethal mutation form.                     retained study. A total 2,012 genes unique <br /> Thus, simple grouping human genome lack                            OMIM Ids finally collected human disease genes. <br /> the power accurately illustration complex sce- <br /> nario associated human disease genes.                                    Ubiquitously expressed genes obtained the <br />                                                                               result recent large scale microarray experiment on <br /> Conclusion                                                                    human gene expression patterns Su et al. [33]. A total <br /> Our studies suggest human essential genes                          33,698 genes sampled 79 tissues interro- <br /> unique group genes simply ignored                        gated experiments. The overall gene expression <br /> and classified non-disease genes studies                      level 776.5 standard Affymetrix average difference <br /> disease genes. We disease genes sev-                      units, genes expression level greater 550 <br /> eral properties residing essential genes.                   standard units 73/79 tissues selected as <br /> We notice gene essentiality better described                    UEHGs (a conservative estimation percentage of <br /> in continuous spectrum instead assigned class                    essential genes human genome 10%, thus <br /> <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span>. Nevertheless, simplicity three-way classi-                  standards set roughly 2,000 genes would <br /> fication good purpose research com-                  classified UEHGs). A total 1,789 genes were <br /> parisons performed easily.                                             collected. The set UEHGs small overlap dis- <br />                                                                               ease genes 176 genes belong classes. The list <br /> Extensive knowledge human essential genes                           UEHGs Additional file 2. <br /> critical understanding human diseases. It has <br /> been shown essential genes direct associa-                      Collection gene features <br /> tion diseases cancer [30,31]. Studying human                     The mouse rat homologs corresponding synony- <br /> essential genes provide key clues questions                    mous substitution rate (Ks), nonsynomymous substitu- <br /> such human beings evolved. However, limited                            tion rate (Ka) totally 15,726 human genes were <br /> attentions paid little system-                     downloaded NCBI HomoloGene [34]. To prevent <br /> atic studies done. We showed picture                        possible contamination paralogous genes, only <br /> of disease genes gets clearer explicitly consider                     considered one-to-one mapped orthologous pairs. To test <br />  <br />  <br />                                                                                                                                        Page 10 13 <br />                                                                                                                   (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                         http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure 6 disease onset age Ka/Ks <br /> Correlation <br /> Correlation disease onset age Ka/Ks. The correlation disease onset age Ka/Ks. Disease genes divided <br /> into 5 groups based disease onset age. The weighted linear regression applied disease genes (group 2 5) is <br /> shown dotted line. The coefficient onset age +0.0086 P-value 0.02, derived regression. UEHGs and <br /> other genes plotted sides diseases genes visual comparison. The standard deviation indicated the <br /> short horizontal bar mean denoted solid circle. The large variation group hints confounding fac- <br /> tors affect Ka/Ks. <br />  <br />  <br />  <br /> the statistical significance difference Ka, Ks      genes [25]. Weighted square regression used <br /> Ka/Ks distributions groups, Kolmogorov-           correlations disease onset ages and <br /> Smirnov test used calculate p-value [4]       Ka/Ks ratios [26]. <br /> that direct comparisons possible. Nucleotide <br /> conservation scores downloaded UCSC                     Yeast genes collected NCBI Entrez Gene Data- <br /> Genome Browser website [35]. Human sequence varia-                base [37] divided groups: UEHG <br /> tion information obtained Swiss-Prot protein             homologs, disease gene homologs, human gene <br /> knowledgebase [36]. The original amino acid positions             homologs, genes human homologs. The <br /> were mapped nucleotide positions correspond-            homologies obtained NCBI HomoloGene as <br /> ing chromosome obtain conservation score. To               described above. The yeast gene deletion phenotype data <br /> study correlation onset age disease      downloaded Saccharomyces Genome Database <br /> conservation, obtained onset ages 900              [38]. Similarly, genes C. elegans collected from <br />  <br />  <br />                                                                                                                      Page 11 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                              http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> NCBI Entrez Gene Database divided             Additional material <br /> groups. RNAi phenotypes C. elegans genes were <br /> retrieved WormBase [39]. The RNAi phenotypes <br /> were divided categories: lethal (including both <br />                                                                       Additional File 1 <br />                                                                       Supplementary materials Details methods described file. <br /> embryonic larval lethal), wild type, sick (phenotypes             Click file <br /> other two), unknown. For genes anno-               [http://www.biomedcentral.com/content/supplementary/1471- <br /> tated phenotype, severe               2164-7-31-S1.doc] <br /> (assuming lethal&gt;sick&gt;wild) chosen pheno- <br /> types.                                                                Additional File 2 <br />                                                                       List UEHGsThe list UEHGs The list contained file. <br /> The degrees genes protein physical interaction              Click file <br />                                                                       [http://www.biomedcentral.com/content/supplementary/1471- <br /> network retrieved Human Protein Refer-                  2164-7-31-S2.txt] <br /> ence Database (HPRD) [24]. To compare function dis- <br /> tribution genes different categories, used <br /> Gene Ontology (GO) Biological Process protein func- <br /> tion annotation. Gene Ontology annotations 12,715             Acknowledgements <br /> human genes downloaded NCBI [40]               We thank reviewers great suggestions revision. We thank <br /> classifications based biological processes used.         Andrew Su John Hogenesch answering question regarding to <br /> Similar Zhou et al. [41], GO node referred inform-    expression cutoff value UEHGs. We thank Kim Fechtel pro- <br /> ative covers 500 genes,          viding classification genes compare gene sets. <br /> descendant nodes cover genes. 25 GO inform-            We thank Matthew Lebo proofreading manuscript. This research is <br /> ative nodes defined according criterion. To test     supported NIH/NSF joint mathematical biology initiative DMS-0241102 <br />                                                                  NIH P50 HG 002790. <br /> whether UEHGs, disease genes genes over/ <br /> under represented 25 function categories, <br />                                                                  References <br /> we used hyper-geometric distribution calculate p-         1.      Lopez-Bigas N, Ouzounis CA: Genome-wide identification of <br /> value.                                                                   genes likely involved human genetic disease. Nucleic <br />                                                                          Acids Res 2004, 32:3108-3114. <br />                                                                  2.      Bortoluzzi S, Romualdi C, Bisognin A, Danieli GA: Disease genes <br /> Gene length information retrieved UCSC                          intracellular protein networks. Physiol Genomics 2003, <br /> genome table browser [42]. All genes                      15:223-227. <br /> mapped refSeq IDs length information                3.      Simth NGC, Eyre-Walker A: Human disease genes: patterns and <br />                                                                          predictions. Gene 2003, 318:169-175. <br /> retrieval. To assess significance difference   4.      Huang H, Winter EE, Wang H, Weinstock KG, Xing H, Goodstadt L, <br /> length genes different categories, Wilcoxon rank sum               Stenson PD, Cooper DN, Smith D, Alba MM, Pointing CP, Fechtel K: <br />                                                                          Evolutionary conservation selection human disease <br /> test used calculate p-value.                                  gene orthologs rat mouse genomes. Genome Biol <br />                                                                          2004, 5:R47. <br /> In process collecting various features, genes        5.      Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />                                                                          Lucau-Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, Bakk- <br /> were annotated certain databases. We limited                  oury ME, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss M, <br /> comparisons genes information. The number                     Davis K, Deutschbauer A, Entian K, Flaherty P, Foury F, Garfinkel DJ, <br />                                                                          Gerstein M, Gotte D, Guldener U, Hegemann JH, Hempel S, Herman <br /> genes included comparison                   Z, Jaramillow DF, Kelly DE, Kelly SL, Kotter P, LaBonte D, Lamb DC, <br /> corresponding tables figures. For information                 Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard P, Ooi <br /> the method materials, Additional file 1.                         SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Scherens B, <br />                                                                          Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, Storms <br />                                                                          RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang C, Ward TR, <br /> Authors&apos; contributions                                                   Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu K, Bussey <br /> ZT conceived study, performed three-way classi-               H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnson M: Func- <br />                                                                          tional profiling Saccharomyces cerevisiae genome. <br /> fication, carried comparison evolutionary                 Nature 2002, 418:387-391. <br /> rate amino acid level, protein physical interaction           6.      Sonnichsen B, Koski LB, Walsh A, Marschall P, Neumann B, Brehm M, <br />                                                                          Alleaume A-M, Artelt J, Bettencourt P, Cassin E, Hewitson M, Holz C, <br /> degrees, drafted manuscript. LW carried com-                 Khan M, Lazik S, Martin C, Nitzsche B, Ruer M, Stamford J, Winzi M, <br /> parison features. MX studied relation-              Heinkel R, Roder M, Finell J, Hantsch H, Jones SJ, Jones M, Piano F, <br /> ship gene expression correlation protein                         Gunsalus KC, Oegema K, Gonczy P, Coulson A, Hyman AA, <br />                                                                          Echeverri CJ: Full-genome RNAi profiling early embryogen- <br /> essentiality. XZ, TC FS jointly guided research. All            esis Caenorhabditis elegans. Nature 2005, 434:462-469. <br /> authors read approved final manuscript.                  7.      Blake JA, Richardson JE, Bult CJ, Kadin JA, Eppig JT: The Mouse <br />                                                                          Genome Database Group: MGD: The Mouse Genome Data- <br />                                                                          base. Nucleic Acids Res 2003, 31:193-195. <br />                                                                  8.      Warrington JA, Nair A, Mahadevappa M, Tsyganskaya M: Compari- <br />                                                                          son human adult fetal expression identification of <br />                                                                          535 housekeeping/maintenance genes. Physiol Genomics 2000, <br />                                                                          2:143-147. <br />                                                                  9.      Butte AJ, Dzau V, Glueck SB: Further defining housekeeping, or <br />                                                                          &quot;maintenance,&quot; genes Foucus &quot;A compendium gene <br />  <br />  <br />                                                                                                                               Page 12 13 <br />                                                                                                          (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />       expression normal human tissues&quot;. Physiol Genomics 2001,                 sor genes: higher intensities purifying selection other <br />       7:95-96.                                                                    genes. Mol Biol Evol 2003, 20:964-968. <br /> 10.   Zhang L, Li W-H: Mammalian housekeeping genes evolve                  32.   Adie EA, Adams RR, Evans KL, Porteous D, Pickard BS: Speeding <br />       slowly tissue-specific genes. Mol Biol Evol 2004,                 disease gene discovery sequence based candidate prioriti- <br />       21:236-239.                                                                 zation. BMC Bioinformatics 2005, 6:55. <br /> 11.   Eisenberg E, Levanon EY: Human housekeeping genes com-            33.   Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, Block D, Zhang J, <br />       pact. Trends Genet 2003, 19:362-365.                                        Soden R, Hayakawa M, Kreiman G, Cooke MP, Walker JR, Hogenesch <br /> 12.   Hirsh AE, Fraser HB: Protein dispensability rate evolu-              JB.: A gene atlas mouse human protein-encoding <br />       tion. Nature 2001, 411:1046-1049.                                           transcriptomes. Proc Natl Acad Sci USA 2004, 101:6062-6067. <br /> 13.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes        34.   Wheeler DL, Church DM, Edgar R, Federhen S, Helmberg W, Mad- <br />       evolutionarily conserved nonessential genes                   den TL, Pontius JU, Schuler GD, Schriml LM, Sequeira E, Suzek TO, <br />       bacteria. Genome Res 2002, 12:962-968.                                   Tatusova TA, Wagner L: Database resources national <br /> 14.   Jeong H, Mason SP, Barbasi A-L, Oltvai ZN: Lethality central-           center biotechnology information: update. Nuclei Acids Res <br />       ity protein networks. Nature 2001, 411:41-42.                            2004, 32:D35-D40. <br /> 15.   Bourque G, Pevzner PA, Tesler G: Reconstructing genomic           35.   Conservation score              [http://hgdownload.cse.ucsc.edu/down <br />       archeitechture ancestral mammals: lessons human,                    loads.html#human] <br />       mouse, rat genomes. Genome Res 2005, 14:507-516.                  36.   Human sequence variation                  [http://us.expasy.org/sprot/sp- <br /> 16.   Wolfe KH, Sharp PM: Mammalian gene evolution: nucleotide                    docu.html] <br />       sequence divergence mouse rat. J Mol Evol 1993,           37.   NCBI Entrez gene                  [http://www.ncbi.nlm.nih.gov/entrez/ <br />       37:441-456.                                                                 query.fcgi?db=gene] <br /> 17.   Ohta T, Ina Y: Variation synonymous substitution rates             38.   Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, Hester ET, Jia Y, <br />       mammalian genes correlation syn-                      Juvik G, Roe T, Schroeder M, Weng S, Botstein D: SGD: Saccharo- <br />       onymous nonsynonymous divergences. J Mol Evol 1995,                     myces Genome Database. Nucleic Acids Res 1998, 26:73-79. <br />       41:717-720.                                                           39.   Chen N, Harris TW, Antoshechkin I, Bastiani C, Bieri T, Blasiar D, <br /> 18.   Makalowski W, Boguski MS: Evolutionary parameters                    Bradnam K, Canaran P, Chan J, Chen CK, Chen WJ, Cunningham F, <br />       transcribed mammalian genome: analysis 2,820 orthol-                  Davis P, Kenny E, Kishore R, Lawson D, Lee R, Muller HM, Nakamura <br />       ogous rodent human sequences. Proc Natl Acad Sci USA                    C, Pai S, Ozersky P, Petcherski A, Rogers A, Sabo A, Schwarz EM, Van <br />       1998, 95:9407-9412.                                                         Auken K, Wang Q, Durbin R, Spieth J, Sternberg PW, Stein LD: <br /> 19.   Smith NG, Hurst LD: The effect tandem substitutions               WormBase: comprehensive data resource Caenorhab- <br />       correlation synonymous nonsynonymous rates                      ditis biology genomics.                 Nucleic Acids Res 2005, <br />       rodents. Genetics 1999, 153:1395-1402.                                   33:D383-D389. <br /> 20.   Castresana J: Estimation genetic distances human          40.   Gene ontology annotation                 [ftp://ftp.ncbi.nlm.nih.gov/gene/ <br />       mouse introns. Genome Biol 2002, 3:R28.                                     DATA/] <br /> 21.   Pagani F, Raponi M, Baralle FE: Synonymous mutations CFTR          41.   Zhou X, Kao MJ, Wong WH: Transitive functional annotation <br />       exon 12 affect splicing neutral evolution. Proc              shortest-path analysis gene expression data. Proc Natl <br />       Natl Acad Sci USA 2005, 102:6368-6372.                                      Acad Sci USA 2002, 99:12783-12788. <br /> 22.   Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom     42.   Karolchik D, Hinrichs AS, Furey TS, Roskin KM, Sugnet CW, Haussler <br />       K, Clawson H, Spieth J, Hillier LW, Richards S, Weinstock GM, Wil-          D, Kent WJ: The UCSC Table Browser data retrieval tool. <br />       son RK, Gibbs RA, Kent WJ, Miller W, Haussler D: Evolutionarily             Nucl Acids Res 2004, 32(Suppl 1):D493-D496. <br />       conserved elements vertebrate, insect, worm, yeast <br />       genomes. Genome Res 2005, 15:1034-1050. <br /> 23.   Miller MP, Kuman S: Understanding human disease mutations <br />       use interspecific genetic variation. Hum Mol <br />       Genet 2001, 10:2319-2328. <br /> 24.   Peri S, Navarro JD, Amanchy R, Kristiansen TZ, Jonnalagadda CK, <br />       Surendranath V, Niranjan V, Muthusamy B, Gandhi TKB, Gronborg M, <br />       Ibarrola N, Deshpande N, Shanker K, Shivashankar HN, Rashmi BP, <br />       Ramya MA, Zhao Z, Chandrika KN, Padma N, Harsha H <br /> </body></html>