<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>Mutation</span> of yeast Eug1p CXXS active sites <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> CXXC results in a dramatic <br /> increase in protein disulphide isomerase activity <br />  <br /> Protein disulphide isomerase (PDI) is an essential protein which                  To determine the catalytic capacity of the wild-type and mutant <br /> is localized to the endoplasmic reticulum of eukaryotic cells. It                 forms we assayed activity in oxidative refolding of reduced and <br /> catalyses the formation and isomerization of disulphide bonds                     denatured procarboxypeptidase Y as well as refolding of bovine <br /> during the folding of secretory proteins. PDI is composed of                      pancreatic trypsin inhibitor. The wild-type protein showed very <br /> domains with structural homology to thioredoxin and with                          little activity, not only in oxidative refolding but also in assays <br /> CXXC catalytic motifs. EUG1 encodes a yeast protein, Eug1p,                       where only isomerase activity was required. This was surprising, <br /> that is highly homologous to PDI. However, Eug1p contains                         in particular since mutant forms of Eug1p containing a CXXC <br /> CXXS motifs instead of CXXC. In the current model for PDI                         motif displayed activity close to that of genuine PDI. These <br /> function both cysteines in this motif are required for PDI-                       results lead us to propose that general disulphide isomerization <br /> catalysed oxidase activity. To gain more insight into the bio-                    is not the main function of Eug1p in &#194;&#352;i&#194;&#352;o. <br /> chemical properties of this unusual variant of PDI we have <br /> purified and characterized the protein. We have furthermore <br /> generated a number of mutant forms of Eug1p in which either or                    Key words : carboxypeptidase Y, protein folding, Saccharomyces <br /> both of the active sites have been mutated to a CXXC sequence.                    cere&#194;&#352;isiae, thiol oxidation. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                      the purified enzyme. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> these studies it appears that Eug1p is <br />                                                                                   an extremely inefficient PDI. This is surprising as mutation of <br /> The endoplasmic reticulum (ER) constitutes the folding en- <br />                                                                                   either of the active sites in Eug1p to a CXXC motif improves the <br /> vironment for secretory proteins in eukaryotic cells. A secretory <br />                                                                                   PDI-like function in &#194;&#352;itro. We have characterized various mutant <br /> protein can undergo various post-translational modifications, <br />                                                                                   forms of Eug1p, using an in &#194;&#352;itro assay based on oxidative <br /> including formation of disulphide bonds, that are often vital for <br />                                                                                   refolding of denatured and reduced procarboxypeptidase Y <br /> the folding and stability of the protein. The formation and <br />                                                                                   (proCPY). These forms containing one or two CXXC active-site <br /> isomerization of disulphide bonds is catalysed by protein di- <br />                                                                                   sequences all exhibit significantly increased foldase activity as <br /> sulphide isomerase (PDI) [1], which is an abundant protein in the <br />                                                                                   compared with the wild-type protein. This is consistent with <br /> ER. <br />                                                                                   previous results showing that mutation of the Eug1p active sites <br />    PDI is composed of four domains in an a-b-bh-ah arrangement <br />                                                                                   to a CXXC sequence motif improved its ability to complement a <br /> [2]. The thioredoxin-like a and ah domains contain the two active <br />                                                                                   PDI1 deletion [5]. <br /> sites, each with the sequence motif CGHC. The ability of PDI to <br /> form a disulphide bond between the proximal cysteines in this <br /> motif enables the protein to carry oxidizing equivalents and <br /> transfer them to a protein substrate. The EUG1 gene product is                    MATERIALS AND METHODS <br /> highly homologous to PDI from yeast and mammals and appears <br /> to have the same domain organization [3]. It is, however, unique <br />                                                                                   Media and materials <br /> among PDI-like proteins as it contains CXXS active-site                           E. coli was grown in Luria&#226;&#8364;&#8220;Bertani (LB) medium supplemented <br /> sequences. Mutant PDI with the active-site sequences CXXS has                     with the appropriate antibiotic [6]. Restriction enzymes, T4 <br /> almost no oxidase activity, while isomerase activity is much less                 DNA polymerase, T4 polynucleotide kinase and T4 DNA ligase <br /> affected [4]. Whereas Eug1p is therefore not expected to be able                  were from Promega (Madison, WI, U.S.A.). Oligonucleotides <br /> to directly introduce disulphide bonds, it should, in principle, be               were synthesized by DNA Technology (Aarhus, Denmark). <br /> able to catalyse rearrangement of non-native disulphide bonds.                    Sequencing was performed using a Taq Dye Deoxy4 terminator <br />    With its unique active-site sequences, Eug1p is interesting in                 cycle sequencing kit on an ABI 373A DNA sequencer, both from <br /> terms of enzymology, evolution and cell biology. Based on the                     Applied Biosystems (Foster City, CA, U.S.A.). 1,4-Dithiothreitol <br /> logic summarized above it has been suggested that Eug1p acts as                   and o-phthalaldehyde were from Merck (Darmstadt, Germany). <br /> a dedicated isomerase in the yeast ER. To investigate this                        N-(3-[2-furyl]acryloyl)-Phe-Phe,   tris(carboxyethyl)phosphine <br /> possibility further, we have chosen to prepare Eug1p in                           and PMSF were from Sigma (St. Louis, MO, U.S.A.). 5h,5h- <br /> Escherichia coli with a C-terminal His tag and to characterize                    Dithiobis(2-nitrobenzoic acid) was from ICN Biomedicals Inc <br />                                         &apos; <br />  <br />  <br />  <br />   Abbreviations used : proCPY, procarboxypeptidase Y ; ER, endoplasmic reticulum ; GS-proCPY, glutathionylated proCPY ; LB, Luria&#226;&#8364;&#8220;Bertani ; PDI, <br /> protein disulphide isomerase. <br />   1 <br />     To whom correspondence should be addressed (e-mail jrw!crc.dk). <br />  <br />                                                                                                                                # 2001 Biochemical Society <br />  270             P. N&#195;&#184;rgaard and J. R. Winther <br />  <br /> Table 1    Plasmids used for expression in E. coli                  filtration column equilibrated with 50 mM acetate buffer, pH 5.0. <br />                                                                     An eluate of 600 &#194;&#181;l was collected. To 300 &#194;&#181;l of the eluate, 200 &#194;&#181;l <br />                              Plasmid   Active sites                 of 0.1 M potassium phosphate buffer, pH 7.0, and 500 &#194;&#181;l of 8 M <br />                              pCT65     CXXS-CXXS                    guanidinium chloride in 0.1 M phosphate buffer, pH 7.0, were <br />                              pPN335    CXXS-CXXC                    added. The proCPY concentration was measured using <br />                              pPN342    CXXC-CXXC                    fluorescence at excitation and emission wavelengths of 295 and <br />                              pPN365    CXXC-CXXS                    335 nm, respectively, and quantificated using a standard curve. <br />                              pPN398    CXXC-SXXS                    To 250 &#194;&#181;l of the NAP-5 eluate 10 &#194;&#181;l of 10 mM 5h,5h-dithiobis(2- <br />                              pPN406    SXXS-CXXC                    nitrobenzoic acid) were added, incubated for a minimum of 1 h, <br />                                                                     and the peak corresponding to thio-nitro-benzoate was detected <br />                                                                     using reversed-phase HPLC. The concentration of free thiols was <br />                                                                     determined by relating the area of the peak to a standard curve. <br /> (Aurora, OH, U.S.A.). Isopropyl &#206;&#178;-&#239;&#65533;&#164;-thiogalactoside was from        The number of free cysteines residues per proCPY molecule was <br /> US Biological (Swampscott, MA, U.S.A.).                             subsequently calculated. <br />  <br /> Strains and plasmids                                                Glutathionylation of proCPY <br /> pBH1865 is described by N&#195;&#184;rgaard et al. [7]. pCT58 is described     GSSG was added at a concentration of 30 mM to a solution <br /> by Holst et al. [5]. EUG1 was mutated using conven-                 containing denatured and reduced $ 12 &#194;&#181;M proCPY in 6 M <br /> tional techniques [8,9] and subcloned into pBH1865, giving          guanidinium chloride\0.1 M Tris\HCl, pH 7.4. This represents <br /> pCT56 (containing EUG1CXXS-CXXC) and pCT61 (containing              an $ 250-fold molar excess of GSSG over proCPY thiols. The <br /> EUG1CXXC-CXXS). For construction of the expression plasmid          mixture was incubated for 5 h. The subsequent 60-fold dilution <br /> pCT65, the coding sequence without signal sequence was ampli-       of this mixture with 2 mM GSH, which initiates the refolding, <br /> fied by PCR using the primers oCT10 (5h-CCC GGG CAT ATG             resulted in a GSSG concentration of 0.5 mM and a GSH <br /> GCT AGC ACG CCG GGA TCA GAT TTA CTC-3h) and                         concentration of 2 mM. <br /> oCT11 (5h-CCC GGG GCG GCC GCC TCG AGT ACG GTA                          To determine the degree of glutathionylation, the gluta- <br /> TCT TCA GTA GAC-3h). The PCR product was digested with              thionylated proCPY (GS-proCPY) was separated from non- <br /> NheI and XhoI and cloned into the corresponding sites of pET-       bound glutathione by applying the sample to a HiTrap <br /> 24a(j) (Novagen). pPN335, pPN342 and pPN365 were made               (Amersham Pharmacia Biotech) desalting column equilibrated <br /> by subcloning the NcoI\BstBI fragments from pCT56, pCT58            with 50 mM acetate buffer, pH 5.0. Of the 1&#226;&#8364;&#8220;1.5 ml eluate pool, <br /> and pCT61, respectively, into the corresponding sites of            300 &#194;&#181;l were used to determine the proCPY concentration as <br /> pCT65. pPN398 and pPN406 were made by mutagenesis                   described above. The eluate fraction (100 &#194;&#181;l) was mixed with 2 &#194;&#181;l <br /> of pPN335 and pPN365. The sequence of the EUG1 open                 of 1 mM tris(carboxyethyl)phosphine, which reduces all the <br /> reading frame was in all cases verified.                            disulphide bonds. After incubation for 30 min, the amount of <br />                                                                     liberated GSH was determined fluorometrically after deriva- <br /> Purification of Eug1p                                               tization with o-phthalaldehyde as described by Senft et al. [11]. <br />                                                                        By relating the GSH concentration to the proCPY con- <br /> E. coli BL21(DE3) carrying the relevant expression plasmid (see <br />                                                                     centration, the GS-proCPY preparations were found to contain <br /> Table 1) was grown in LB medium containing kanamycin <br />                                                                     9.4p1 mol of GSH\mol of proCPY, corresponding to $ 85 % <br /> (30 &#194;&#181;g\ml). The cells were grown at 30 mC until a D           of <br />                                                           &apos;!!       glutathionylation. <br /> 0.2&#226;&#8364;&#8220;0.3, when the production of Eug1p was induced by addition <br /> of isopropyl &#206;&#178;-&#239;&#65533;&#164;-thiogalactoside at a final concentration of <br /> 1 mM. The culture was harvested after 4&#226;&#8364;&#8220;5 h at 30 mC, and cells     Assay of insulin reduction <br /> were resuspended in $ 5 ml of binding buffer (20 mM Tris\HCl,       The assay mixture contained 0.1 M potassium phosphate, pH 7.0, <br /> pH 8.0\0.5 M NaCl\5 mM imidazole) per litre of culture. PMSF        2 mM EDTA, $ 0.7 mg\ml human insulin and 0.2 M NaCl in a <br /> was added to a concentration of 1 mM, and the cells were            final volume of 200 &#194;&#181;l. The assays were performed in the presence <br /> sonicated for 6i30 s. Cell debris was removed by centrifugation,    or absence of 0.5 &#194;&#181;M yeast PDI or Eug1p. The reaction was <br /> and the lysate was added to a pre-equilibrated Ni#+-chelating       started by addition of 10 &#194;&#181;l of 0.2 M 2-mercaptoethanol. The <br /> column (Novagen), which was eluted as described by the manu-        reactions were carried out in microtitre plates, and the turbidity <br /> facturer. The eluate was dialysed against 25 mM Tris\HCl,           (630 nm) of the reaction mixtures was read, at 2 min intervals, <br /> pH 8.0, and the dialysate was applied to a MonoQ ion-exchange       using a plate reader. <br /> column (Amersham Pharmacia Biotech) equilibrated with the <br /> same buffer. Eug1p was eluted at $ 0.3 M NaCl. Relevant <br /> fractions were pooled and dialysed against 25 mM Tris\HCl,          RESULTS <br /> pH 8.0, and frozen for storage. The purity of Eug1p was analysed    Oxidative refolding of reduced and denatured proCPY is only <br /> by SDS\PAGE and estimated to   90 %.                                marginally enhanced by the presence of wild-type Eug1p <br />                                                                     A prerequisite for the ability of PDI to function as an isomerase <br /> Determination of disulphide-bond formation in proCPY during         is that its active site CXXC motif is in the reduced state. In the <br /> oxidative refolding                                                 oxidation and isomerization of protein thiols in the ER, this <br /> Refolding was carried out in a glutathione redox buffer as          prerequisite puts the cell in a dilemma. On the one hand a high <br /> described by Westphal et al. [10]. Samples of 500 &#194;&#181;l were removed   capacity for oxidation is essential for efficient production of <br /> from the refolding mixture at various times. The refolding          disulphide-bonded secretory proteins. Much evidence indeed <br /> reaction was quenched by addition of 10 &#194;&#181;l of 1 M HCl to give       suggests that protein oxidation is mediated by oxidized PDI [12]. <br /> a final pH of $ 2. Glutathione was removed by applying the          On the other hand, the more efficient the oxidative machinery is, <br /> sample to an NAP4-5 (Amersham Pharmacia Biotech) gel-               the more likely it is that incorrect disulphide bonds are formed. <br />  <br /> # 2001 Biochemical Society <br />                                                                                                              Yeast Eug1p disulphide isomerase activity            271 <br />  <br />                                                                                              active sites to the PDI-like CXXC motifs [5,7]. To obtain a more <br />                                                                                              quantitative measure for the activity of the wild-type and mutant <br />                                                                                              forms we assayed the activity of the proteins produced in E. coli <br />                                                                                              using an assay based on oxidative refolding of unfolded and <br />                                                                                              reduced proCPY [10]. <br />                                                                                                 proCPY is an in &#194;&#352;i&#194;&#352;o PDI substrate in yeast. In its native form, <br />                                                                                              proCPY contains five disulphide bonds. The folding and matu- <br />                                                                                              ration of this protein can be followed in &#194;&#352;itro as well as in &#194;&#352;i&#194;&#352;o. <br />                                                                                              In &#194;&#352;itro, the oxidative refolding of proCPY is studied by dilution <br />                                                                                              of the denatured and reduced protein in a redox buffer, which <br />                                                                                              includes reduced and oxidized glutathione. During refolding, <br />                                                                                              samples are withdrawn and treated with proteinase K, which <br />                                                                                              activates proCPY by removal of the propeptide [13]. Proteinase <br />                                                                                              K also degrades unfolded or partially folded proCPY. The <br />                                                                                              activity of carboxypeptidase Y is subsequently measured and <br />                                                                                              compared with the activity of a non-denatured sample. <br />                                                                                                 The spontaneous oxidative refolding of proCPY leads to <br />                                                                                              $ 40 % recovery of activity after 5 h (Figure 1A). The progress <br />                                                                                              curve appears sigmoidal, with a lag phase of $ 1 h. Addition of <br />                                                                                              wild-type Eug1p has a surprisingly small effect. When catalysed <br />                                                                                              by wild-type Eug1p, the lag phase is the same as the spontaneous <br />                                                                                              refolding, but the recovery of folded proCPY is higher ($ 55 % <br />                                                                                              after 5 h, Figure 1A). Conversion of the two active-site sequences <br />                                                                                              to CXXC, on the other hand, dramatically increases the efficiency <br />                                                                                              of the enzyme. The lag phase is significantly shorter, and the <br />                                                                                              recovery reaches $ 90 % after 5 h (Figure 1A). These data are <br />                                                                                              consistent with the effect observed in &#194;&#352;i&#194;&#352;o, where the CXXS-to- <br />                                                                                              CXXC mutations cause a markedly improved growth rate in an <br />                                                                                              EUG1-complemented strain [5]. It should be noted, however, <br />                                                                                              that in &#194;&#352;itro the oxidative refolding still proceeds significantly <br />                                                                                              more efficiently with yeast PDI than with Eug1pCXXC-CXXC under <br />                                                                                              the same conditions and concentrations of enzyme (Figure 1A). <br />                                                                                                 To compare the relative importance of the two active sites, the <br />                                                                                              mutant forms Eug1pSXXS-CXXC and Eug1pCXXC-SXXS were com- <br />                                                                                              pared. SXXS active sites are catalytically inert and PDISXXS-SXXS <br />                                                                                              does not catalyse proCPY refolding [10]. The ah domain by far <br />                                                                                              plays the most important role, as seen from a decreased lag <br />                                                                                              phase, as well as a higher recovery in the Eug1pSXXS-CXXC- <br />                                                                                              catalysed refolding reaction (Figure 2). This is particularly <br />                                                                                              evident from the observation that the progress curves for the <br />                                                                                              CXXC-CXXC and the SXXS-CXXC mutant forms are very <br />                                                                                              similar (Figures 1A and 1C). This is in spite of the fact that the <br />                                                                                              CXXC-CXXC mutant has double the concentration of functional <br /> Figure 1     Oxidative refolding of reduced and denatured proCPY                             sites as compared with the SXXS-CXXC mutant form. <br />                                                                                                 Instead of an inert SXXS motif, if the mutant forms of Eug1p <br /> Reduced and denatured proCPY (0.15 &#194;&#181;M) was refolded in the absence or presence of 1 &#194;&#181;M       investigated above had the wild-type motif CXXS, would that <br /> Eug1p. The refolding was carried out in 0.1 M Tris/HCl, pH 7.4/0.2 M KCl/0.5 mM GSSG/2 mM <br /> GSH at 25 mC. (A) Refolding in the absence of Eug1p ($) and in the presence of <br />                                                                                              single active-site cysteine contribute to the oxidative refolding of <br /> Eug1pCXXS-CXXS ( <br /> ) and Eug1pCXXC-CXXC (4). The dotted line shows refolding in the presence   proCPY, or would the observed effect solely be due to the CXXC <br /> of 1 &#194;&#181;M yeast PDI [10]. (B) Refolding in the absence of Eug1p ($) and in the presence of     active site ? To answer this question, we compared refolding in <br /> Eug1pCXXC-CXXS ( <br /> ) and Eug1pCXXC-SXXS (4). (C) Refolding in the absence of Eug1p ($) and     the presence of Eug1p mutants with one CXXC motif plus either <br /> in the presence of Eug1pCXXS-CXXC ( <br /> ) and Eug1pSXXS-CXXC (4).                                a CXXS or a SXXS motif. Assay of Eug1pCXXC-CXXS and <br />                                                                                              Eug1pCXXC-SXXS mutant forms showed $ 75 % recovery after 5 h. <br />                                                                                              The two progress curves arising from the presence of either of the <br />                                                                                              two Eug1p mutant forms are superimposable, indicating that <br /> A highly oxidizing environment inherently disfavours the pres-                               the presence of a single cysteine in the ah site does not contribute <br /> ence of those free thiols of PDI that are necessary for maintenance                          significantly to the activity (Figure 1B). On the other hand, <br /> of its isomerizing activity. Yeast is probably the eukaryotic                                comparison of the two mutant forms with Eug1pCXXS-CXXC and <br /> organism in which the disulphide-bond-forming apparatus has                                  Eug1pSXXS-CXXC shows that the presence of a single cysteine in the <br /> been most extensively characterized. Here an obvious role for                                a site has a small effect, leading to a slightly reduced lag phase <br /> Eug1p could be to maintain an isomerase activity under hyper-                                (Figure 1C). However, the main contribution to the foldase <br /> oxidizing conditions. We tested this hypothesis by assaying the                              activity again appears to stem from the CXXC active-site <br /> properties of purified Eug1p.                                                                sequence. <br />   In &#194;&#352;i&#194;&#352;o, EUG1 is a high-copy suppressor of PDI1 deletion [3,7].                               PDI has previously been associated with a disulphide-in- <br /> Whereas this suppression is rather poor, the ability of EUG1 to                              dependent chaperone activity [14]. To gauge this activity we <br /> complement a PDI1 deletion is highly improved by changing its                                asked whether Eug1p would facilitate refolding of denatured <br />  <br />                                                                                                                                             # 2001 Biochemical Society <br />  272               P. N&#195;&#184;rgaard and J. R. Winther <br />  <br />  <br />  <br />  <br /> Figure 2     Recovery of activity after 1 and 5 h of oxidative refolding of proCPY <br /> Reduced and denatured proCPY was refolded in the absence or presence of various mutant forms of Eug1p. Carboxypeptidase Y activity was measured after 1 (closed bars) and 5 (open bars) h. <br /> Refolding for 1 h in the presence of any of the mutant forms with CXXS-to-CXXC mutations shows a dramatically higher activity compared with refolding catalysed by wild-type Eug1p. Also, <br /> the recovery after 5 h is higher. S.E.M. based on 2&#226;&#8364;&#8220;7 independent measurements are shown. <br />  <br />  <br />  <br />  <br />                                                                                                  proCPY forms with no or incorrect disulphide bonds are <br />                                                                                                  substrates for Eug1p. <br />  <br />  <br />  <br />                                                                                                  Unfolded oxidized proCPY as a substrate for Eug1p <br />                                                                                                  The experiments described above investigated the potential of <br />                                                                                                  Eug1p for oxidative folding. However, wild-type Eug1p might <br />                                                                                                  reasonably be expected to be more suited to carrying out <br />                                                                                                  disulphide shuffling reactions, as only the N-proximal cysteine of <br />                                                                                                  the CXXC motif of a PDI should be necessary here. Denatured <br />                                                                                                  and glutathionylated proteins have previously been exploited as <br />                                                                                                  model substrates for PDI [15,16]. Such substrates are in an <br />                                                                                                  oxidized form, but reshuffling is necessary for the protein to <br />                                                                                                  obtain the native disulphide configuration. We wanted to explore <br />                                                                                                  GS-proCPY as a uniformly oxidized substrate. Under the <br />                                                                                                  experimental conditions used, spontaneous oxidation of proCPY <br /> Figure 3     Refolding of non-reduced proCPY                                                     must occur through formation of a mixed disulphide bond <br /> Refolding of denatured proCPY, in the absence or presence of Eug1p. Oxidative refolding in the <br />                                                                                                  between a protein cysteine and glutathione. The lag phase <br /> absence of Eug1p (for comparison ; $) ; non-oxidative refolding in the absence of Eug1p (4)      observed in the progress curve for spontaneous oxidative <br /> and non-oxidative refolding in the presence of Eug1pCXXC-CXXC ( <br /> ).                               refolding could represent this initial oxidation of cysteine <br />                                                                                                  residues. Firstly, to estimate the time scale of this reaction, we <br />                                                                                                  followed the disappearance of free thiol groups in proCPY <br />                                                                                                  during the spontaneous oxidative refolding by taking samples at <br /> proCPY possessing all native disulphide bonds. As seen in                                        various time points. proCPY was then separated from free <br /> Figure 3, the refolding of denatured proCPY containing all                                       glutathione, and the number of free thiol groups per proCPY <br /> native disulphide bonds is much faster than oxidative refolding.                                 molecule was determined. The experiment showed that the level <br /> However, the presence of Eug1p has no effect on the refolding                                    of free protein thiol decreases rapidly within the first $ 20 min <br /> rate or yield under these conditions, demonstrating that only                                    (Figure 4). Under these conditions, essentially no folded proCPY <br />  <br /> # 2001 Biochemical Society <br />                                                                                                                        Yeast Eug1p disulphide isomerase activity                      273 <br />  <br />  <br />  <br />  <br />                                                                                                   Figure 6     Insulin-reduction assay with yeast PDI and wild-type Eug1p <br />                                                                                                   The precipitation of the insulin B-chain, caused by reduction of insulin, was followed <br />                                                                                                   spectrophotometrically at 630 nm. The assay was performed in the absence of PDI or Eug1p <br />                                                                                                   ( <br /> ) and in the presence of Eug1pCXXS-CXXS ($) and yeast PDI (X). <br /> Figure 4 Disappearance of free thiol groups during spontaneous oxidative <br /> refolding of proCPY <br /> The number of free protein thiol groups at various time points during spontaneous oxidative <br /> refolding of proCPY was measured and related to the protein concentration (the curve was fitted   suggesting that Eug1p is a poor isomerase (Figure 5). As this <br /> to an exponential equation). Fully reduced proCPY has 11 free thiol groups. The time course       could be a consequence of the specific substrate used, we wanted <br /> during the oxidative refolding shows establishment of a plateau of 1.5&#226;&#8364;&#8220;2 thiol groups per         to test the isomerase activity of wild-type Eug1p using a different <br /> proCPY molecule after 20&#226;&#8364;&#8220;30 min.                                                                  substrate. Here, we chose the folding intermediate of bovine <br />                                                                                                   pancreatic trypsin inhibitor, called N*, as yeast PDI previously <br />                                                                                                   has been shown to catalyse rearrangement of this particular <br />                                                                                                   substrate [10]. However, consistent with the results obtained <br />                                                                                                   using GS-proCPY as substrate, wild-type Eug1p had no de- <br />                                                                                                   tectable activity in this assay (results not shown). This lack of <br />                                                                                                   isomerase activity could be a consequence of non-reactive active- <br />                                                                                                   site cysteines. To test this, we used wild-type Eug1p in an insulin- <br />                                                                                                   reduction assay [17], where precipitation of the B-chain caused <br />                                                                                                   by reduction of the disulphide bonds between the two chains can <br />                                                                                                   be followed. Wild-type Eug1p had a small but significant activity <br />                                                                                                   in this assay, suggesting that the cysteines are indeed reactive <br />                                                                                                   (Figure 6). The activity is, however, extremely small compared <br />                                                                                                   with the activity of a similar concentration of yeast PDI. <br />                                                                                                      The presence of Eug1pCXXC-CXXC during refolding of GS- <br />                                                                                                   proCPY gives rise to a progress curve almost identical with the <br />                                                                                                   one obtained from refolding of fully reduced proCPY (Figure 5). <br />                                                                                                   This suggests that the shuffling reaction is the rate-limiting step <br />                                                                                                   in the catalysed refolding of proCPY. <br />  <br />                                                                                                   DISCUSSION <br /> Figure 5      Refolding of GS-proCPY <br />                                                                                                   Contrary to what is generally believed, Eug1p does not appear to <br /> The free thiol groups on fully reduced proCPY were oxidized by formation of mixed disulphide      be a good PDI. It may be argued that substrates other than the <br /> bonds with GSH prior to initiating refolding. Refolding of proCPY in the absence of Eug1p ($)     ones chosen for the present study are more suitable. However, <br /> and in the presence of Eug1pCXXC-CXXC ( <br /> ). Refolding of GS-proCPY in the absence of Eug1p         proCPY is in principle a native yeast PDI substrate, and the fact <br /> (&gt;), and in the presence of Eug1pCXXC-CXXC (4) and Eug1pCXXS-CXXS (i).                            that Eug1p can be converted into an efficient PDI by simply <br />                                                                                                   changing the active-site motifs from CXXS to CXXC argues <br />                                                                                                   against proCPY being an inherently poor substrate for Eug1p. <br /> appears within this time period (Figure 1A). Therefore, the lag                                     In &#194;&#352;itro, we found the impact of wild-type Eug1p on the <br /> phase most likely represents the initial formation of disulphide                                  oxidative refolding of proCPY to be very small (Figure 1A). <br /> bonds, both with glutathione and intra-molecularly, whereas the                                   However, CXXS-to-CXXC mutations result in a dramatically <br /> subsequent phase showing regain of activity mainly consists of                                    increased effect on oxidative refolding of proCPY. Spontaneous <br /> disulphide rearrangement and\or final folding to the native                                       oxidative refolding of proCPY progresses via a long lag phase <br /> proCPY structure. Consistent with this interpretation, the lag                                    (Figure 1). Under these conditions, oxidation of proCPY must <br /> phase of the spontaneous refolding reaction of GS-proCPY is                                       occur through the reaction of protein thiol groups with GSSG, <br /> significantly shorter than that of the fully reduced proCPY                                       which in this assay functions as a donor of oxidizing equivalents. <br /> (Figure 5). Surprisingly, the progress curves for refolding of GS-                                This lag phase represents mainly oxidation, as seen by dis- <br /> proCPY with and without wild-type Eug1p are very similar,                                         appearance of free proCPY thiol groups (Figure 4). The lag <br />  <br /> phase is markedly shortened when the refolding is catalysed by                        <br /> an Eug1p mutant with one or two CXXC active sites. This <br />                                                                                <br /> suggests a higher oxidase activity of these mutants compared                            <br /> with wild-type Eug1p. This is also consistent with the in &#194;&#352;i&#194;&#352;o                         <br /> observation that EUG1CXXC-CXXC, unlike the wild-type gene, is                       <br /> able to complement a yeast strain with no auxiliary PDI-like                              <br /> proteins present [7].                                                     <br />    Lyles and Gilbert [18] found that, in the case of PDI, the N- <br />                                                                                     <br /> and C-terminal domains are functionally different. At saturating                         <br /> RNase concentrations, a PDI mutant with an inactivated C-                          <br /> terminal active site retains near wild-type activity. However, a                        <br /> mutant with an inactivated N-terminal site shows only $ 30 %                           <br /> activity as compared with the wild-type. A non-equivalence has                       <br />                                                                                         <br /> also been shown for yeast PDI, but here the ah site is the more <br />                                                                                          <br /> active one [10]. We here show that oxidative refolding of proCPY                    <br /> in the presence of Eug1p with a CXXC active-site sequence in the                        <br /> ah site, similar to yeast PDI, is more efficient than the presence of                 <br /> Eug1p with a CXXC active a site.                                                         <br />    The rate-limiting step in the Eug1pCXXC-CXXC-catalysed folding                   <br />                                                                                       <br /> appears to be reshuffling of incorrectly formed disulphide bonds. <br />                                                                                         <br /> This is supported by the notion that the progress curve from                        <br /> refolding of GS-proCPY is identical with the curve arising                             <br /> from refolding of fully reduced proCPY (Figure 5). GS-proCPY                         <br /> has here been used as a homogeneously oxidized substrate useful                     <br /> for assay of isomerase activity. Surprisingly, wild-type Eug1p did                 <br /> not show any activity during refolding of GS-proCPY (Figure 5). <br />                                                                                         <br /> This suggests that the CXXC enzymes are not only better                         <br /> oxidases, but also better isomerases.                                                <br />    It is remarkable, both from an evolutionary and a biochemical                          <br /> point of view, that the activity of the Eug1pCXXC-CXXC mutant                        <br /> form is around 30&#226;&#8364;&#8220;50 % of what is found for native yeast PDI.                       <br /> It seems reasonable to assume that Eug1p has evolved to perform <br />                                                                                        <br /> a highly specialized function in the yeast ER and that its high-                 <br /> copy suppression of PDI1 deletion does not directly bear on                           <br /> what it normally does in the ER. In this light, the possibility that                <br /> such a specialized function might relate to retrograde transport                    <br /> of proteins out of the ER is tantalizing (M. Zhou and R.                       <br />                                                                                  <br /> Schekman, personal communication). <br />  <br /> </body></html>