<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 2002, p. 2552&#226;&#8364;&#8220;2556                                                                                Vol. 184, No. 9 <br /> 0021-9193/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.184.9.2552&#226;&#8364;&#8220;2556.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />            ZipA Is Required Recruitment FtsK, FtsQ, FtsL, and <br />                    FtsN Septal Ring Escherichia coli <br />                                           Cynthia A. Hale* Piet A. J. Boer <br />                                Department Molecular Biology Microbiology, School Medicine, <br />                                    Case Western Reserve University, Cleveland, Ohio 44106-4960 <br />                                            Received 11 December 2001/Accepted 1 February 2002 <br />  <br />              The septal ring Escherichia coli consists essential gene products order assembly <br />           resembles linear dependency pathway: FtsA ZipA directly bind FtsZ polymers prospective <br />           division site, followed sequential addition FtsK, FtsQ, FtsL, FtsW, FtsI, FtsN. Recruitment of <br />           FtsK downstream components requires prior localization FtsA. Here recruitment <br />           FtsK, FtsQ, FtsL, FtsN equally requires ZipA. The results imply association FtsA ZipA <br />           FtsZ polymers needed maturation nascent organelle. <br />  <br />  <br />    Cell division Escherichia coli mediated septal                use GFP fusions. Relevant features these <br /> ring, membrane-associated structure assembles                   constructs, strains used study, pre- <br /> site division prior onset septal invagination. At                sented Table 1. A gfp-tagged derivative ftsK obtained <br /> least gene products, FtsA, FtsI, FtsK, FtsL, FtsN, FtsQ,                 amplifying 5&#226;&#172;&#732; portion ftsK previously shown be <br /> FtsW, FtsZ, ZipA, shown essential com-                    sufficient support cell division (10, 25), using primers 5&#226;&#172;&#732;- <br /> ponents organelle (18, 23). Recent studies determining                GAGCGACATATGAGCCAGGAATACATTGAAGAC <br /> the ability proteins localize putative division sites          AAAGAAGTC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CGGACTCGAGCGCAGCGTCT <br /> in cells limiting nonfunctional                GTTTGCCGCCCCATCG-3&#226;&#172;&#732;. The product digested with <br /> have suggested likely order assembly cytoki-               NdeI XhoI (underlined) cloned pET21A (Nova- <br /> netic ring. The picture emerged work re-                   gen), yielding pCH203. A gfp tag added the <br /> veals linear dependency pathway (7). The visible               construct control lac promoter in <br /> step involves movement FtsZ cytoplasm                  pMLB1113 (9). The resulting plasmid, pCH205, encodes a <br /> inner surface inner membrane, polymerizes                  57.4-kDa FtsK-Gfp fusion protein, Gfpmut2 fused <br /> form ring-like structure prospective division site (4).              C terminus FtsK amino acid 266. <br /> Both FtsA ZipA bind FtsZ directly (12, 14, 16, 17, 20, 26)                   Gfp-tagged FtsQ, FtsL, FtsN expressed lyso- <br /> and join nascent ring mutually independent fashion,                  genic &#226;?&#173; phages, obtained constructing plasmid deriva- <br /> which followed sequential addition FtsK, FtsQ,                   tives pMLB1113 crossing phage &#226;?&#173;NT5 <br /> FtsL, FtsW, FtsI, finally FtsN (1&#226;&#8364;&#8220;3, 5, 7, 8, 11, 13, 15, 16,             described previously (9). ftsQ amplified using primers <br /> 19, 21, 24, 25, 27, 28). The linear dependency pathway              5&#226;&#172;&#732;-ACGCGAATTCCATATGTCGCAGGCTGCTC-3&#226;&#172;&#732; and <br /> derived observation recruitment                     5&#226;&#172;&#732;-CGCCAAGCTTATTGTTGTTCTGCCTGTGC-3&#226;&#172;&#732;, and <br /> component requires prior localization upstream com-                product digested EcoRI HindIII (underlined) <br /> ponents. However, mutual independence FtsA                         ligated pET21A, yielding pGP1. The NheI-HindIII frag- <br /> ZipA localization notable exception linear sequence              ment pGP1 cloned pDR107C (22), creating <br /> and represents possible branch point assembly path-                  pCH194, encodes 59.9-kDa Gfp-T-FtsQ fusion protein <br /> way. It established that, addition                (T represents T7 tag [Novagen]), Gfpmut2 <br /> FtsZ, FtsA required targeting FtsK downstream                  peptide fused N terminus FtsQ linker pep- <br /> components septal ring (1&#226;&#8364;&#8220;3, 8, 11, 13, 16, 19, 24, 25, 27,            tide ASMTGGQQMGRGSEFH. In pCH196, expression of <br /> 28), relationship established ZipA.                  fusion protein placed control lac <br /> Here report green fluorescent protein (GFP) fusions                promoter ligating BglII-HindIII fragment pCH194 <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> FtsK, FtsQ, FtsL, FtsN localize putative division               BamHI- HindIII-digested pMLB1113. <br /> sites ZipA-depleted filaments. These results imply re-                   ftsL amplified primers 5&#226;&#172;&#732;-CCGAATTCCATA <br /> cruitment FtsK, -Q, -L, -W, -I, -N nascent or-                  TGATCAGCAGAGTGACAG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CGTGTCGACTTA <br /> ganelle requires prior formation intermediate struc-                TTTTTGCACTACGAT-3&#226;&#172;&#732;. The product digested with <br /> ture consisting FtsZ, FtsA, ZipA.                                      BclI SalI (underlined) ligated BamHI- SalI- <br />    Localization FtsK, FtsQ, FtsL, FtsN. Assembly                    digested pET21C (Novagen), yielding pAB13. The NheI-PstI <br /> FtsK, FtsQ, FtsL, FtsN septal ring monitored                 fragment pAB13 cloned pDR107C, generating <br />                                                                               pCH193, encodes 41.5-kDa Gfp-T-FtsL fusion protein <br />                                                                               N-terminal methionine FtsL replaced with <br />   * Corresponding author. Mailing address: Department Molecular <br /> Biology Microbiology, School Medicine, Case Western Reserve <br />                                                                               Gfpmut2 peptide linker peptide ASMTGG <br /> University, 10900 Euclid Ave., Cleveland, OH 44106-4960. Phone:               QQMGR. Expression fusion protein placed under <br /> (216) 368-1699. Fax: (216) 368-3055. E-mail: cah19@po.cwru.edu.               control lac promoter pCH195 outlined above. <br />  <br />                                                                        2552 <br />  VOL. 184, 2002                                                                                                             NOTES        2553 <br />  <br /> TABLE 1. Hosts, plasmids, phages used localization studies <br /> Host, plasmid,                                           Source or <br />                           Relevant genotype <br />   phage                                               reference <br />  <br />  Hosts <br />   CH3             PB103 recA::Tn10                      12 <br />   CH5             PB103 zipA::aph recA::Tn10            12 <br />  <br />  Plasmids <br />    pDB361         aadA&#226;&#171;&#185; repA&#226;&#171;&#185; cI857(ts) P&#226;?&#173;R::ftsZ       13 <br />    pCH205         bla&#226;&#171;&#185; lacIq&#226;&#171;&#185; Plac::ftsK(1&#226;&#8364;&#8220;266)-gfp     This work <br />  <br />  Phages <br />    &#226;?&#173;CH195         imm21 bla&#226;&#171;&#185; lacIq&#226;&#171;&#185; Plac::gfp-ftsL      This work <br />    &#226;?&#173;CH196         imm21 bla&#226;&#171;&#185; lacIq&#226;&#171;&#185; Plac::gfp-ftsQ      This work <br />    &#226;?&#173;CH201         imm21 bla&#226;&#171;&#185; lacIq&#226;&#171;&#185; Plac::gfp-ftsN      This work <br />  <br />  <br />  <br />    ftsN amplified primers 5&#226;&#172;&#732;-CAGCGAATTCCA <br /> TATGGCACAACGAGATTATG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-TGAGAAGCTT <br /> AACCCCCGGCGGCGAG-3&#226;&#172;&#732;, product digested <br /> with EcoRI HindIII (underlined) cloned into <br /> pDR107A (22). The resulting plasmid, pCH198, encodes a <br /> 64.3-kDa Gfp-T-FtsN fusion protein Gfpmut2 <br /> peptide fused N terminus FtsN linker peptide <br /> ASMTGGQQMGRGSEFH. As above, expression this <br /> fusion protein placed control lac pro- <br /> moter, yielding pCH201. <br />    To verify localization FtsK(1-266)-Gfp, Gfp-FtsQ, <br /> Gfp-FtsL, Gfp-FtsN, fusions visualized by <br /> fluorescence microscopy exponentially growing cells of <br /> strain CH3/pDB361 (Fig. 1 4, A panels). Cells grown <br /> overnight 30&#194;&#176;C LB-Amp (Luria-Bertani medium contain- <br /> ing 50 &#226;?&#174;g ampicillin/ml) supplemented 50 &#226;?&#174;g spec- <br /> tinomycin/ml 0.1% glucose. Cultures diluted in <br /> fresh LB-Amp plus 5 &#226;?&#174;M (pCH205), 20 &#226;?&#174;M (&#226;?&#173;CH195), 25 &#226;?&#174;M <br /> (&#226;?&#173;CH196), 35 &#226;?&#174;M (&#226;?&#173;CH201) isopropyl-&#226;?&#164;-D-thiogalactopy- <br /> ranoside (IPTG) grown 30&#194;&#176;C optical density 600 <br /> nm 0.35. Cells chemically fixed, viewed, imaged as <br /> previously described (13). At low levels expression used <br /> in study, fusion proteins interfered the <br /> division process, cells displayed wild-type <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> in <br /> each case (Fig. 1 4, A&#226;&#172;&#732; panels). Consistent earlier <br /> observations (2, 6, 8, 11, 25, 28), majority cells <br /> expressing FtsK(1-266)-Gfp (66%), Gfp-FtsQ (78%), Gfp- <br /> FtsL (58%), Gfp-FtsN (56%) exhibited single band of <br /> fluorescence division site, demonstrating four <br /> fusions preferentially accumulated septal ring (Fig. 1 to <br /> 4, A panels). Similar results obtained fusions <br /> were expressed cells isogenic ZipA-depleted strain <br /> CH5/pDB361 (see below), provided cells grown the <br /> permissive temperature (42&#194;&#176;C; data shown.) <br />    Localization FtsK, FtsQ, FtsL, FtsN ZipA-depleted <br /> filaments. We showed previously growth strain CH5/ <br /> pDB361 (zipA0/&#226;?&#173;PR::zipA cI857) 30&#194;&#176;C leads produc- <br /> tion long nonseptate filaments repression zipA        FIG. 1. Localization FtsK(1&#226;&#8364;&#8220;266)-Gfp. Fluorescence differ- <br /> transcription pDB361 consequent depletion            ential interference contrast micrographs showing location of <br /> ZipA. Moreover, multiple FtsA-decorated FtsZ rings         FtsK(1&#226;&#8364;&#8220;266)-Gfp normally dividing cells strain CH3/pDB361/ <br /> present filaments (13).                                     pCH205 (A) ZipA-depleted filaments strain CH5/pDB361/ <br />                                                                      pCH205 (B). (B&#226;&#172;&#8482;) Location native FtsZ filaments shown <br />    To assess recruitment FtsK FtsQ             panel B. Cells imaged Nomarski (A&#226;&#172;&#732; B&#226;&#172;&#732;) fluo- <br /> septal ring dependent ZipA, strain CH5/pDB361              rescence optics using GFP-specific (A B) Cy3-specific (B&#226;&#172;&#8482;) filter <br /> either transformed plasmid pCH205 [Plac::ftsK (1-266)-          sets. Bar, 2.5 &#226;?&#174;m. <br />  2554     NOTES                                                                                                                J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                            FIG. 3. Localization Gfp-FtsL. Fluorescence (A B) dif- <br />                                                                          ferential interference contrast (A&#226;&#172;&#732; B&#226;&#172;&#732;) micrographs showing the <br />                                                                          location Gfp-FtsL CH3(&#226;?&#173;CH195)/pDB361 cells (A A&#226;&#172;&#732;) in <br />                                                                          CH5(&#226;?&#173;CH195)/pDB361 filaments (B B&#226;&#172;&#732;). Bar, 2.5 &#226;?&#174;m. <br />  <br />  <br />  <br />                                                                          gfp] lysogenized phage &#226;?&#173;CH196 (Plac::gfp-ftsQ). Cells <br />                                                                          grown overnight 42&#194;&#176;C LB-Amp supplemented with <br />                                                                          50 &#226;?&#174;g spectinomycin/ml 0.1% glucose. Cultures were <br />                                                                          diluted fresh LB-Amp plus 5 &#226;?&#174;M (pCH205) 25 &#226;?&#174;M <br />                                                                          (&#226;?&#173;CH196) IPTG grown 42&#194;&#176;C 1 h. Cultures then <br />                                                                          shifted 30&#194;&#176;C growth continued 5 6 h <br />                                                                          cultures reached optical density 0.2 and <br />                                                                          0.3. Cells chemically fixed, immunostained treatment <br />                                                                          affinity-purified anti-FtsZ antibodies Cy-3-conjugated <br />                                                                          secondary antibodies, imaged described previously (13). <br />                                                                             As expected, ZipA-depleted filaments contained multi- <br />                                                                          ple FtsZ rings regularly spaced intervals <span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> <br />                                                                          (Fig. 1 2, B&#226;&#172;&#8482; panels). In contrast, FtsK(1-266)-Gfp <br />                                                                          Gfp-FtsQ completely failed localize rings vast <br />                                                                          majority filaments (Fig. 1 2, B panels). Among <br />                                                                          hundreds filaments examined, observed few <br />                                                                          (&#226;&#172;?5%) single faint accumulation fluorescence resem- <br />                                                                          bling ring. Instead, significant portion FtsK(1-266)-Gfp <br />                                                                          appeared located membrane (Fig. 1B), while <br />                                                                          Gfp-FtsQ showed diffuse <span id='am-81' about='xsp:pattern' typeof='owl:Thing'>pattern</span> fluorescence (Fig. <br />                                                                          2B). Thus, multiple Z rings present, FtsK(1- <br />                                                                          266)-Gfp Gfp-FtsQ failed associate rings due <br />   FIG. 2. Localization Gfp-FtsQ. Fluorescence differential in- <br /> terference contrast micrographs showing location Gfp-FtsQ in <br />                                                                          depletion ZipA. We conclude minimal amount <br /> CH3(&#226;?&#173;CH196)/pDB361 cells (A) CH5(&#226;?&#173;CH196)/pDB361 fila-             ZipA required recruitment FtsK FtsQ the <br /> ments (B). (B&#226;&#172;&#8482;) Location native FtsZ filaments shown   septal ring. <br /> panel B. Cells imaged Nomarski (A&#226;&#172;&#732; B&#226;&#172;&#732;) fluores-           Because FtsK FtsQ required septal recruit- <br /> cence optics using GFP-specific (A B) Cy3-specific (B&#226;&#172;&#8482;) filter    ment FtsL, FtsW, FtsI, FtsN (2, 7, 11, 19, 27), results <br /> sets. Bar, 2.5 &#226;?&#174;m. <br />                                                                          predicted ZipA essential recruitment of <br />  VOL. 184, 2002                                                                                                                        NOTES         2555 <br />  <br />  <br />  <br />  <br />                                                                             FIG. 5. Model showing dependency pathway recruitment of <br />                                                                          division proteins developing septal ring organelle. Efficient re- <br />                                                                          cruitment FtsK FtsN requires presence sufficient <br />                                                                          amounts functional FtsZ FtsA, ZipA. The three <br />                                                                          likely form tripartite complex consisting FtsZ polymers are <br />                                                                          bound FtsA ZipA. Adapted references 7, 11, 27. <br />  <br />  <br />                                                                          Gfp-FtsN induced 20 35 &#226;?&#174;M IPTG, respectively. <br />                                                                          As anticipated, Gfp-FtsL Gfp-FtsN accumulated in <br />                                                                          rings depletion ZipA (Fig. 3 4, B panels). Rather, <br />                                                                          Gfp-FtsL exhibited diffuse pattern fluorescence (Fig. 3B), <br />                                                                          Gfp-FtsN appeared distributed periphery of <br />                                                                          filaments (Fig. 4B). <br />                                                                             We note higher levels expression, GFP <br />                                                                          fusions showed clear preference periphery cells <br />                                                                          (data shown), expected membrane pro- <br />                                                                          teins. Therefore, suspect failure and <br />                                                                          (2, 7, 8, 11, 25) studies detect overt membrane <br />                                                                          localization FtsQ, FtsL, (to lesser extent) FtsK in <br />                                                                          filaments lack upstream septal ring component due <br />                                                                          level Fts-specific signal low clearly <br />                                                                          distinguishable. <br />                                                                             Our observations extend body data addresses the <br />                                                                          order assembly septal ring components E. coli. The <br />                                                                          results, earlier studies, summarized in <br />                                                                          model shown Fig. 5. This model similar linear <br />                                                                          pathway forth Beckwith coworkers (7, 11, 27), but <br />                                                                          incorporates requirement ZipA septal recruit- <br />                                                                          ment FtsK, -Q, -L, -W, -I, -N. Though did not <br />                                                                          directly recruitment FtsW FtsI require <br />                                                                          ZipA, anticipated case, the <br />                                                                          septal localization FtsW shown dependent on <br />                                                                          prior localization FtsQ FtsL (19), FtsI is <br />                                                                          dependent FtsK, -Q, -L (7, 27). <br />                                                                             To date, good evidence direct physical interactions be- <br />                                                                          tween E. coli division proteins obtained for <br />                                                                          FtsZ FtsA (14, 17, 26) FtsZ ZipA (12, 14, 16, <br />                                                                          20). It clear, however, numerous interactions <br />                                                                          septal ring components place the <br />                                                                          organelle. A web interactions indicated the <br />                                                                          recruitment dependency pathway (Fig. 5) fact <br />                                                                          ring&#226;&#8364;&#8482;s known components remain associated with <br />                                                                          invaginating cell wall septal closure, suggesting that <br />                                                                          act concert constric- <br />                                                                          tion process. An immediate challenge precisely define <br />                                                                          interactions, information help meeting <br />                                                                          far larger challenge understanding division <br />                                                                          apparatus, fully assembled, actually stimulates cell con- <br />    FIG. 4. Localization Gfp-FtsN. Fluorescence (A B)          striction. <br /> differential interference contrast (A&#226;&#172;&#732; B&#226;&#172;&#732;) micrographs showing the <br /> location Gfp-FtsN CH3(&#226;?&#173;CH201)/pDB361 cells (A A&#226;&#172;&#732;)            We thank Gregg Pietz Anita Boyapati help plasmid <br /> CH5(&#226;?&#173;CH201)/pDB361 filaments (B B&#226;&#172;&#732;). Bar, 2.5 &#226;?&#174;m.                    construction. <br />                                                                            This work supported NIH grant GM-57059 generous <br />                                                                          donation Wyeth-Ayerst Research. <br /> these downstream components well. To test prediction,                                             REFERENCES <br /> strain CH5/pDB361 lysogenized phages &#226;?&#173;CH195                  1. Addinall, S. G., E. Bi, J. Lutkenhaus. 1996. FtsZ ring formation fts <br />                                                                              mutants. J. Bacteriol. 178:3877&#226;&#8364;&#8220;3884. <br /> (Plac::gfp-ftsL) &#226;?&#173;CH201 (Plac::gfp-ftsN) cells grown          2. Addinall, S. G., C. Cao, J. Lutkenhaus. 1997. FtsN, late recruit the <br /> and imaged above, expression Gfp-FtsL                  septum Escherichia coli. Mol. Microbiol. 25:303&#226;&#8364;&#8220;309. <br />  2556       NOTES                                                                                                                                           J. BACTERIOL. <br />  <br />  3. Addinall, S. G., J. Lutkenhaus. 1996. FtsA localized septum            division gene ftsW possible role FtsW FtsZ function. J. Bacteriol. <br />     FtsZ-dependent manner. J. Bacteriol. 178:7167&#226;&#8364;&#8220;7172.                                  179:784&#226;&#8364;&#8220;793. <br />  4. Bi, E., J. Lutkenhaus. 1991. FtsZ ring structure associated division     16.   Liu, Z., A. Mukherjee, J. Lutkenhaus. 1999. Recruitment ZipA the <br />     Escherichia coli. Nature 354:161&#226;&#8364;&#8220;164.                                                division site interaction FtsZ. Mol. Microbiol. 31:1853&#226;&#8364;&#8220;1861. <br />  5. Boyle, D. S., M. M. Khattar, S. G. Addinall, J. Lutkenhaus, W. D.             17.   Ma, X., W. Margolin. 1999. Genetic functional analyses the <br />     Donachie. 1997. ftsW essential cell-division gene Escherichia coli.            conserved C-terminal core domain Escherichia coli FtsZ. J. Bacteriol. <br />     Mol. Microbiol. 24:1263&#226;&#8364;&#8220;1273.                                                           181:7531&#226;&#8364;&#8220;7544. <br />  6. Buddelmeijer, N., M. E. Aarsman, A. H. Kolk, M. Vicente, N. Nanninga.         18.   Margolin, W. 2000. Themes variations prokaryotic cell division. <br />     1998. Localization cell division protein FtsQ immunofluorescence                  FEMS Microbiol. Rev. 24:531&#226;&#8364;&#8220;548. <br />     microscopy dividing nondividing cells Escherichia coli. J. Bacteriol.   19.   Mercer, K. L. N., D. S. Weiss. 2002. The Escherichia coli cell division <br />     180:6107&#226;&#8364;&#8220;6116.                                                                          protein FtsW required recruit cognate transpeptidase, FtsI (PBP3), <br />  7. Chen, J. C., J. Beckwith. 2001. FtsQ, FtsL FtsI require FtsK,           division site. J. Bacteriol. 184:904&#226;&#8364;&#8220;912. <br />     FtsN, co-localization FtsZ Escherichia coli cell division. Mol.   20.   Mosyak, L., Y. Zhang, E. Glasfeld, S. Haney, M. Stahl, J. Seehra, W. S. <br />     Microbiol. 42:395&#226;&#8364;&#8220;413.                                                                  Somers. 2000. The bacterial cell-division protein ZipA interaction <br />  8. Chen, J. C., D. S. Weiss, J. M. Ghigo, J. Beckwith. 1999. Septal local-             FtsZ fragment revealed X-ray crystallography. EMBO J. 19:3179&#226;&#8364;&#8220; <br />     ization FtsQ, essential cell division protein Escherichia coli J. Bac-         3191. <br />     teriol. 181:521&#226;&#8364;&#8220;530.                                                              21.   Pogliano, J., K. Pogliano, D. S. Weiss, R. Losick, J. Beckwith. 1997. <br />                                                                                             Inactivation FtsI inhibits constriction FtsZ cytokinetic ring and <br />  9. Boer, P. A. J., R. E. Crossley, L. I. Rothfield. 1989. A division <br />                                                                                             delays assembly FtsZ rings potential division sites. Proc. Natl. Acad. <br />     inhibitor topological specificity factor coded minicell locus <br />                                                                                             Sci. USA 94:559&#226;&#8364;&#8220;564. <br />     determine proper placement division septum E. coli. Cell 56:641&#226;&#8364;&#8220; <br />                                                                                       22.   Raskin, D. M., P. A. J. Boer. 1999. Rapid pole-to-pole oscillation of <br />     649. <br />                                                                                             protein required directing division middle Escherichia coli. <br /> 10. Draper, G. C., N. McLennan, K. Begg, M. Masters, W. D. Donachie.                    Proc. Natl. Acad. Sci. USA 96:4971&#226;&#8364;&#8220;4976. <br />     1998. Only N-terminal domain FtsK functions cell division. J.           23.   Rothfield, L., S. Justice, J. Garc&#196;&#177;&#194;&#180;a-Lara. 1999. Bacterial cell division. <br />     Bacteriol. 180:4621&#226;&#8364;&#8220;4627.                                             <br /> </body></html>