<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Aug. 2002, p. 4582&#226;&#8364;&#8220;4593                                                                           Vol. 184, No. 16 <br /> 0021-9193/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.184.16.4582&#226;&#8364;&#8220;4593.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />           Genome-Scale Metabolic Model Helicobacter pylori 26695 <br />               Christophe H. Schilling,1* Markus W. Covert,2 Iman Famili,2 George M. Church,3 <br />                               Jeremy S. Edwards,4 Bernhard O. Palsson2 <br />               Genomatica, Inc., San Diego, California 921211; Department Bioengineering, University California, <br />                    San Diego, La Jolla, California 920932; Department Genetics, Harvard Medical School, <br />                             Boston, Massachusetts 021153; Department Chemical Engineering, <br />                                        University Delaware, Newark, Delaware 197164 <br />                                             Received 17 January 2002/Accepted 14 May 2002 <br />  <br />            A genome-scale metabolic model Helicobacter pylori 26695 constructed genome sequence anno- <br />          tation, biochemical, physiological data. This represents silico model largely derived genomic <br />          information organism substantially biochemical information available relative <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br />          previously modeled organisms Escherichia coli. The reconstructed metabolic network contains 388 <br />          enzymatic transport reactions accounts 291 open reading frames. Within paradigm of <br />          constraint-based modeling, extreme-pathway analysis flux balance analysis used explore the <br />          metabolic capabilities silico model. General network properties analyzed compared to <br />          similar results previously generated Haemophilus influenzae. A minimal medium required model to <br />          generate required biomass constituents calculated, indicating requirement amino acids, of <br />          correspond essential human amino acids. In addition list potential substrates capable fulfilling <br />          bulk carbon requirements H. pylori identified. A deletion study performed reactions <br />          associated genes central metabolism deleted effects simulated variety of <br />          substrate availability conditions, yielding number reactions deemed essential. Deletion results <br />          compared recently published vitro essentiality determinations 17 genes. The silico model <br />          accurately predicted 10 17 deletion cases, partial support additional cases. Collectively, results <br />          presented suggest effective strategy combining silico modeling experimental technologies <br />          enhance biological discovery characterized organisms genomes. <br />  <br />  <br />   Genomics reshaped approaches studying                    number disease states, including gastritis, peptic ulcer- <br /> biology organism. It possible studies com-               ation, gastric cancer (36, 52). This organism serves a <br /> parative genomics (1, 2, 17, 34), genome-wide transcriptional              good example learned primarily ge- <br /> profiling (16), host integrated analyses aimed un-             nome sequence specialized organisms only <br /> derstanding systems biology organisms (14, 25, 68).                 limited supporting literature, particularly relates me- <br />   For model organisms Escherichia coli Saccharo-               tabolism. <br /> myces cerevisiae genome sequencing generated wealth                  Due large availability complete genome <br /> insight physiology microbes. However,                sequence microbe (23, 66), possible recon- <br /> availability extensive biological information literature            struct metabolic network assess organism&#226;&#8364;&#8482;s meta- <br /> regarding organisms undoubtedly easier                bolic capabilities fitness. Previously reconstructed <br /> reconcile data genome interpret charac- <br />                                                                            metabolic network Haemophilus influenzae Rd, based <br /> teristic properties genome genetic contents. <br />                                                                            primarily genome sequence, performed extensive <br />   To date common model organisms had <br />                                                                            constraint-based metabolic modeling pathway analysis of <br /> their genomes sequenced current sequencing efforts are <br />                                                                            organism (18, 55a, 60). In work report compa- <br /> naturally moving less-well characterized organisms. This <br /> begs question: learned characterized            rable analysis H. pylori 26695 present specific <br /> organisms genomic sequence relative absence              results comparisons metabolic poten- <br /> of extensive previous laboratory research? Additionally,               tial fitness H. pylori H. influenzae. The environment <br /> can genome utilized assist guiding future research            lungs stomach shaped evolution <br /> and discovery particular organism interest?                    of bacteria different ways. Both provide <br />   Herein explore aspect questions relates            excellent examples Darwinian evolution <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>survival</span> that <br /> to metabolism. In particular focus metabolic physi-              exploited resources available creative <br /> ology capabilities less-well characterized organism,              ways ensure survival fitness. In silico analysis <br /> namely, Helicobacter pylori. H. pylori human pathogen            supports notion represents new form comparative <br /> colonizes gastric mucosa, resulting acute inflamma-              genomics. <br /> tory response damage epithelial cells progressing               Herein, comprehensive look metabolic <br />                                                                            capabilities H. pylori present examination com- <br />                                                                            puted essential genes, comparison experimentally <br />   * Corresponding author. Mailing address: Genomatica, Inc., 5405 <br /> Morehouse Dr., Suite 210, San Diego, CA 92121. Phone: (858) 362-           validated essential genes. Taken together, results aim to <br /> 8550. Fax: (858) 824-1772. E-mail: cschilling@genomatica.com.              demonstrate learned genome less- <br />  <br />                                                                     4582 <br />  VOL. 184, 2002                                                                                              METABOLIC MODEL OF H. PYLORI                              4583 <br />  <br />  <br />  <br />  <br />   FIG. 1. Overview silico modeling methods conceptual basis. (A) Hypothetical network describing set metabolic <br /> reactions. Based genomic, biochemical, physiological data, reaction network reconstructed represent set chemical <br /> reactions predicted occur organism. (B) Extreme pathways calculated reconstructed reaction network. Reactions do <br /> not occur extreme pathways constitute unused reactions. Certain reactions participate active extreme <br /> pathway. These groups concomitantly occurring reactions constitute reaction subsets, illustrated. (C) FBA used determine input <br /> substrates balanced reaction fluxes required meet demand producing metabolites I J simultaneously fixed ratio. In this <br /> example substrate B available. (D) The reaction metabolite B E eliminated, requiring use substrate A meet the <br /> production demands I J. This situation leads production metabolite H by-product. Note reaction essential if <br /> substrate A present. <br />  <br />  <br />  <br /> characterized organisms analyzed conjunction                          terization precise biochemical functions cell general <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake <br /> in silico model.                                                                      characteristics (5, 7&#226;&#8364;&#8220;9, 11, 13, 15, 17, 22, 24, 26&#226;&#8364;&#8220;32, 34, 35, 37, 39&#226;&#8364;&#8220;51, 53, 54, 55, <br />                                                                                       56, 62&#226;&#8364;&#8220;65). <br />                                                                                         Pathway analysis. The reaction index H. pylori defined mathemati- <br />                         MATERIALS AND METHODS                                         cally form stoichiometric matrix, S, dimensions (m n) <br />    The overall procedures discussed model development simula-           m number metabolites reaction network n the <br /> tion summarized Fig. 1. The detailed computational methods theory          number reactions. Therefore, column represents reaction row <br /> behind work presented published collection       represents stoichiometric participation specific metabolite the <br /> recent primary technical manuscripts reviews (14, 21, 57&#226;&#8364;&#8220;59, 61).                 reactions. A particular flux distribution network, v, indicates flux levels <br />    Metabolic reconstruction. Beginning genomic sequence data, relevant           reactions. Based principles conservation mass and <br /> genes primary metabolism identified assist reconstructing meta-      assumption steady state, flux distribution reaction network <br /> bolic network H. pylori. This information complemented biochemical         characterized following equation: <br /> information cell physiology data used construct organism- <br /> specific reaction index (the set reactions included reconstruction).                                         S &#228;&#161;&#160; v &#226;&#171;&#189; 0                                    (1) <br /> The result process metabolic map illustrated Fig. 1A. The reaction <br /> index underlying map represents best determination biochem-         <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> equation possible determine chemically balanced <br /> ical reactions organism believed capable carrying based   metabolic routes network span entire range metabolic <br /> all available data. The detailed process metabolic reconstruction model        capabilities network. These pathways referred extreme path- <br /> development recently reviewed (14).                                          ways, method calculation based principles convex analysis <br />    The metabolic genotype H. pylori determined annotated ge-          (59). Previously algorithm extreme-pathway calculation pub- <br /> nome sequence (66) provided TIGR Microbial Database              lished pathway classification scheme (59). Figure 1B illustrates the <br /> from GenBank (4). To begin establishing metabolic reaction index H.           extreme pathways hypothetical reaction network. Note extreme <br /> pylori, annotation open reading frames (ORFs) carefully            pathways multiple input substrates multiple output products. <br /> examined, gene assignments based high level sequence similarity         For large networks H. pylori H. influenzae, manageable <br /> and pertaining metabolic enzymes membrane transporters selected        subdivide network smaller subsystems extreme-pathway calcula- <br /> inclusion metabolic genotype. From general biochemistry complete set       tion. The strategy extreme-pathway enumeration subdivided networks is <br /> of potential metabolic reactions carried associated gene products      discussed literature (60) used study determine the <br /> determined. This list reactions augmented using additional experimental        pathway structure H. pylori metabolic network previously used to <br /> information literature pertaining genetic enzymatic charac-       study H. influenzae. <br />  4584        SCHILLING ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />    Briefly, metabolic network H. pylori subdivided discrete          TABLE 1. Forty-seven metabolites considered required for <br /> subsystems, reactions assigned subsystems.               biomass generation maintenance H. pylori <br /> These subsystems include amino acid biosynthesis degradation, nucleotide <br /> biosynthesis degradation, vitamin cofactor biosynthesis, lipid cell               Classification       Abbreviation              Complete name <br /> envelope biosynthesis, central metabolism, transport energy-redox me- <br />                                                                                         Amino acids                 ALA           Alanine <br /> tabolism, including immediate assimilation dissimilation various car- <br />                                                                                                                     ARG           Arginine <br /> bon sources. The extreme pathways subsystems calcu-                                       ASN           Asparagine <br /> lated. From extreme pathways list reactions appear                                    ASP           Aspartate <br /> of pathways determined. In addition reaction subsets identified                                   CYS           Cysteine <br /> as sets reactions occur pathways fixed stoichiometric                                 GLU           Glutamate <br /> ratios (see Fig. 1B examples).                                                                                  GLN           Glutamine <br />    FBA. Using extreme pathways and/or reaction network                                      GLY           Glycine <br /> connectivity allows calculation network properties                                         HIS           Histidine <br /> through use flux balance analysis (FBA). FBA method assess                                       ILE           Isoleucine <br /> production capabilities systemic properties metabolic network. The                                         LEU           Leucine <br /> fundamentals FBA reviewed (6, 21, 58, 67). Extreme-                                          LYS           Lysine <br /> pathway analysis FBA share common theoretical basis                                            MET           Methionine <br /> described previously (57). Specifically, FBA utilizes principles                                   PHE           Phenylalanine <br /> linear programming (LP), subset convex analysis.                                                      PRO           Proline <br />    Constraints placed individual reactions state upper lower                                    SER           Serine <br /> bounds range flux values reactions have. This                                        THR           Threonine <br /> constraint described following form:                                                                      TRP           Tryptophan <br />                                                                                                                     TYR           Tyrosine <br />                                                                                                                     VAL           Valine <br />                                    &#226;?&#163;i &#226;&#177;&#8226; vi &#226;&#177;&#8226; &#226;?&#164;i                                  (2) <br />                                                                                         Polyamines                  PTRC          Putrescine <br /> where &#226;?&#163;i lower bound flux vi &#226;?&#164;i upper bound. If informa-                                    SPMD          Spermidine <br /> tion flux levels available, value &#226;?&#163;i set zero irreversible <br /> fluxes, cases &#226;?&#163;i &#226;?&#164;i left unconstrained, allowing flux   Ribonucleotides             ATP           Adenosine 5&#226;&#172;&#732;-triphosphate <br /> take value, positive negative. The capabilities metabolic                                   GTP           Guanine 5&#226;&#172;&#732;-triphosphate <br /> are explored using LP (12, 67). A reaction selected objective                                   CTP           Cytidine 5&#226;&#172;&#732;-triphosphate <br /> function maximized minimized. A solution calculated                                   UTP           Uridine 5&#226;&#172;&#732;-triphosphate <br /> satisfies constraints equations 1 2. The result optimal flux <br /> distribution allow highest flux chosen objective reaction.    Deoxyribonucleotides        DATP          Deoxyadenosine 5&#226;&#172;&#732;-triphosphate <br /> LINDO used solve LP problems (LINDO Systems, Inc., Chicago, Ill.).                                       DGTP          Deoxyguanine 5&#226;&#172;&#732;-triphosphate <br />    While FBA used generate highly quantitative results                                     DCTP          Deoxycytidine 5&#226;&#172;&#732;-triphosphate <br /> validated experimental data (19, 20), approach used primarily                                   DTTP          Deoxythymidine 5&#226;&#172;&#732;-triphosphate <br /> to assess qualitative metabolic capabilities H. pylori reaction network. <br /> Figure 1C D provide examples FBA used herein. <br />                                                                                         Phospholipids               PS            Phosphatidyl serine <br />                                                                                                                     PE            Phosphatidyl ethanolamine <br />    Objective function. To calculate optimal flux distribution, objective must <br />                                                                                                                     PG            Phosphatidyl glycerol <br /> be defined. For physiologically meaningful results, defined objective <br /> as ability produce required components cellular biomass (e.g.,            Cell envelope               PEPTIDO       Peptidoglycan <br /> amino acids, nucleotides, phospholipids, etc.) enable organism                                      LPS           Lipopolysaccharide <br /> grow survive. This growth objective mathematically defined output <br /> flux utilizing biomass precursor metabolite substrate. Forty-seven (45)       Isoprenoids                 OPP           Octaprenyl pyrophosphate <br /> metabolites selected required biomass constituents (Table 1). An output                                     UDPP          Undecaprenyl pyrophosphate <br /> biomass reaction (exchange flux) created utilized constituents in <br /> equal stoichiometric ratios used means assess ability          Vitamins cofactors      NAD           Nicotinamide adenine dinucleotide <br /> network produce required demands based particular substrate                                        NADP          Nicotinamide adenine dinucleotide <br /> availability conditions. This reaction selected objective                                                  phosphate <br /> calculations. The complete ability network produce biomass                                     FAD           Flavin adenine dinucleotide <br /> constituents leads positive flux value objective reaction.                                            CoA           Coenzyme A <br />    Minimal medium determination. Beginning extracellular me-                                        ACP           Acyl carrier protein <br /> tabolites available metabolic network, metabolites re-                                       PTH           Pantothenate <br /> moved individually determine required producing                                                    Thiamine <br /> biomass constituents. This determination accomplished constraining                                       MTHF          Methyl-tetrahydrofolate <br /> exchange flux uptake reaction metabolite zero optimizing                                      MK            Menaquinone <br /> the biomass objective reaction. After thorough examination set metabolites <br />                                                                                                                     DMK           Dimethylmenaquinone <br /> was arrived removal metabolite render the <br /> network unable produce biomass demands. This set metabolites con- <br /> stitutes defined minimal medium required silico model support           substrates available silico model screened determine their <br /> growth H. pylori.                                                                    ability serve major source carbon organism. These carbon <br />    Alternative carbon source determination. In minimal medium alanine           sources referred alternative carbon sources. <br /> arginine fulfill bulk carbon requirements network (as presented         Deletion studies. All reactions central metabolism examined to <br /> Results section). To determine major carbon sources used           determine essentiality reaction network variety of <br /> the network reactions used assimilate alanine arginine central          medium conditions. To assess consequences deleting reaction the <br /> metabolism simultaneously removed. This accomplished removing               network, constraints particular reaction set zero. A simulation <br /> the D-amino acid dehydrogenase, alanine dehydrogenase, 1-pyrroline-5-               run network support growth optimizing the <br /> carboxylate dehydrogenase. With reactions removed, alanine arginine           biomass objective reaction. If network support growth, the <br /> are utilized network fulfill certain biomass requirements; however,     deleted reaction deemed essential particular environment me- <br /> their broad utilization metabolic network longer possible         dium conditions used simulation. <br /> removal above-mentioned reactions. Under conditions, individual               Four different medium conditions presented deletion study. These <br /> metabolites included minimal medium composition determine             include (i) silico minimal medium (described Results), (ii) minimal <br /> the network support use biomass reaction. All possible          medium plus glucose, (iii) minimal medium plus alternative carbon <br />  VOL. 184, 2002                                                                                        METABOLIC MODEL OF H. PYLORI                    4585 <br />  <br />  <br />   TABLE 2. Comparison genomic characteristics silico                      abolic reconstruction (28). The TCA cycle metabolic <br />  metabolic model characteristics H. pylori H. influenzae (63)                reconstruction complete, traditional sense in <br />                                                          Bacterial strain           E. coli. There reactions included &#226;?&#163;-ketoglutarate <br />                   Properties                                                        oxidase (AKO) (55), succinyl coenzyme A (CoA):acetoacetate <br />                                                  H. pylori 26695 H. influenzae Rd <br />                                                                                     CoA transferase (SCOT) (13), malate:quinone oxi- <br /> Genome characteristics                                                              doreductase (MQO) (31), allow complete connectivity <br />  Genome <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> (bp)                             1,667,867       1,830,135 <br />  G&#226;&#171;&#185;C content (%)                                       39              38 <br />                                                                                     metabolites TCA cycle, these <br />  No. ORFs                                        1,590           1,748           reactions typically considered TCA <br />  No. identified database match                 1,091           1,011           cycle. Notably, fumarate reductase included recon- <br />  No. database match                           499             732           struction reversible, evidenced experimental litera- <br /> In silico metabolic networks <br />                                                                                     ture, indicating oxidation succinate (11, 31) sup- <br />   No. genes included (% known                   291 (&#226;&#172;&#8225;27)        400 (&#226;&#172;&#8225;40)    ported number modeling calculations agreement <br />      ORF)                                                                           physiological findings. <br />   No. associated reactions                        272              412              Extreme-pathway calculations. Based metabolic re- <br />   No. reactions                             116               49 <br />   No. metabolites (internal/external)            339/64           367/84         construction extreme pathways calculated six <br />   No. internal fluxes                             534              632           subsystems. The total number pathways ratios of <br />   No. exchange fluxes                             115              135           pathways reactions provided Table 3. The ratio of <br />   Dimensions S (metabolites reactions)      403 649       451 767 <br />                                                                                     pathways reactions subsystems similar <br />                                                                                     results H. influenzae exception nu- <br />                                                                                     cleotide subsystem, contained nearly 10 times fewer <br /> sources (described Results), (iv) minimal medium plus alter- <br /> native carbon sources amino acids.                                   pathways H. influenzae. This difference <br />                                                                                     indicates significantly flexibility H. pylori <br />                                                                                     metabolic network respect processing nucleo- <br />                                 RESULTS <br />                                                                                     tides lack catabolic pathways available provide <br />    H. pylori metabolic reconstruction. In present metabolic                     different nucleotide substrates cell environ- <br /> reconstruction H. pylori 291 genetic loci ORFs                     mental resources. <br /> included metabolic genotype, corresponding 272 as-                           Unused reactions. As H. influenzae number of <br /> sociated reactions. Based experimental information                          reactions metabolic network appear of <br /> the literature, 116 additional reactions included, bringing                    extreme pathways. These reactions used any <br /> the final number reactions H. pylori reaction index                    flux distribution representing metabolic activity network. <br /> 388. The 116 additional reactions fall categories, (i)                     The reason reactions present ex- <br /> reactions associated genetics unknown                         treme pathway supporting reactions generate sub- <br /> (ii) reactions associated genetics known                     strates consume products present network, <br /> organisms evidence existence                        flux active. The list unused <br /> in H. pylori genome. The complete list reactions                         reactions H. pylori provided Table 4 the <br /> associated genes provided online supplementary material                       comparison numbers H. influenzae. <br /> along list metabolite abbreviations (http:                           The unused reactions listed Table 4 used direct <br /> //gcrg.ucsd.edu/supplementary_data/JBACT/Hpylori.xls). Ta-                          questions related network reconstruction, as <br /> ble 2 provides comparison basic characteristic                     genome annotation, biochemical function particular <br /> properties H. pylori genome metabolic model versus                       enzymes. As example consider METB1 METB2 <br /> the model previously developed H. influenzae.                                   reactions amino acid subsystem coded the <br />    Figure 2 provides graphical illustration central me-                    metB gene (locus HP0106). These reactions unused their <br /> tabolism subsystem reconstructed network. H. pylori                          typical role methionine biosynthesis (a required amino <br /> contains glycolytic reactions; however, key                     acid), indicates experimentation be <br /> enzymes, including phosphofructokinase pyruvate kinase,                         necessary determine role metB gene product. <br /> have identified. The absence enzymes                          ILVC1 ILVC2 unused; however, ILVC3 used, which <br /> affects organism&#226;&#8364;&#8482;s ability extract energy hexoses.                      suggest role ilvC gene (locus HP0330) is <br /> The pentose phosphate reactions present H. pylori                        solely related pantothenate biosynthesis having <br /> with exception 6-phosphogluconate dehydrogenase,                             dual function branched-chain amino acid biosynthesis is <br /> which makes oxidative branch pathway incomplete.                         case H. influenzae E. coli silico models. <br /> The reaction oxidative branch, glucose-6-phosphate                     Other interesting results include GALE reaction (galE, <br /> (G6P) dehydrogenase (G6PD), utilizes redox                        locus HP0360), used conversion of <br /> carriers (NAD NADP). H. pylori reactions                          UDP-galactose UDP-glucose, no <br /> Entner-Doudoroff pathway allowing carbon shunted                          reactions genes appear related UDP-galac- <br /> away pentose phosphate pathway glyceral-                          tose production utilization. <br /> dehyde 3-phosphate pyruvate. Furthermore,                            As H. influenzae metabolic network reactions for <br /> carboxylases identified convert phosphoenolpyruvate (PEP)                        biotin biosynthesis number thiamine biosynthetic re- <br /> into oxaloacetate, did identify anaplerotic reac-                     actions used. The reason biotin reactions being <br /> tion convert PEP tricarboxylic acid (TCA) cycle in-                       used fact initial steps unknown; therefore, no <br /> termediate. However, included malic enzyme met-                         substrate supplied reactions. For thiamine reac- <br />  4586     SCHILLING ET AL.                                                                                                          J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. Reconstructed central metabolic subsystem H. pylori. The different metabolite colors indicate ability metabolite leave <br /> the subsystem participation reactions subsystems. (Supplementary data available http://gcrg.ucsd.edu/supplementary_data/ <br /> JBACT/Hpylori.xls.) <br />  <br />  <br /> tions used missing steps biosynthetic            C4 C5. However, majority genes subsets <br /> pathway, make thiamine auxotrophic requirement                located adjacent another, are <br /> H. pylori similar H. influenzae.                                        scattered far genome. Examples include follow- <br />    Reaction subsets. The reactions occur              ing subsets: A1, A4, A5, A7, C5, V1, V3, V4, N1, N2, N4, and <br /> in ratio extreme pathways            L6. The implications results explored in <br /> are active calculated subsystems. These             discussion section. <br /> reaction subsets expected contain reactions                 Growth requirements. For H. pylori calculated set of <br /> associated genes coregulated possibly exist            required substrates includes amino acids: alanine, argi- <br /> the regulon. The list calculated reaction subsets               nine, histidine, isoleucine, leucine, methionine, phenylalanine, <br /> H. pylori provided Table 5.                                          valine, thiamine, oxygen, phosphate source <br />    If reactions coregulated, good           (inorganic phosphate), sulfate cysteine sulfur <br /> place begin examining regulation                  source. No purine source required network capable <br /> genetic level, particular location genome              performing novo purine biosynthesis (38). Figure 3 illus- <br /> relative another. There subsets                 trates connectivity substrates bio- <br /> reactions associated genes located adjacent              mass constituents biosynthesis associated. <br /> other genome appear                         The requirement oxygen production NAD, <br /> operon. These include subset T2, A2, C2, C6, portions               NADP, pyrimidine nucleotides (CTP UTP) de- <br />  VOL. 184, 2002                                                                                                    METABOLIC MODEL OF H. PYLORI                     4587 <br />  <br />  <br />                  TABLE 3. Subsystem characteristics pathway calculation results H. pylori indicating total number of <br />                                                     extreme pathways subsystema <br />                                                                                      No. extreme                                                        Total no. of <br />                   No.              No. fluxes              No.                                               Total no.       Pathway/flux <br /> Subsystem                                                                           pathways type:                                                      H. influenzaee <br />                  reactions                                     metabolites                                             pathways              ratio <br />                                  Internal       External                           I             II         III                                              pathways <br />  <br />     A               75              97             65              114             36            1          21             58               0.23                 65 <br />     C               34              55             38               48             50            1          17             68               0.55                 83 <br />     L               50              62             33               76             11            1          12             24               0.13                 36 <br />     N               68             105             49               86             51            8          36             95               0.38                764 <br />     T               86             129            122              154            102            3          20            125               0.42                210 <br />     V               75              80             45              113             16            1           5             22               0.14                 21 <br />   a <br />     Abbreviations subsystems: A, amino acid biosynthesis degradation; C, central core metabolism; L, lipid cell envelope biosynthesis; N, nucleotide <br /> biosynthesis degradation; T, transport energy-redox metabolism, (V) vitamin cofactor biosynthesis. Pathways classified according scheme <br /> presented reference,63 primary metabolic pathways functional significance type I. <br />  <br />  <br />  <br />  <br /> oxynucleotides (dCTP dTTP) based need                          methods section). The results deletion study are <br /> electron acceptor electrons flavin-containing                      presented Tables 6 7. <br /> electron carrier, flavin adenine dinucleotide (FAD, oxidized;                          In Table 6 enzymatic deletions render the <br /> FADH, reduced). The reaction network capable                         network incapable generating required energy bio- <br /> oxidizing FADH FAD transfer elec-                         mass constituents support growth conditions ex- <br /> trons menaquinone pool subsequent acceptance                          amined. The corresponding genes reactions aceB, <br /> the electrons oxygen. Without oxygen pathway combi-                          ppa, prsA, tpi. The loss ppa does allow network <br /> nations unable balance generation FADH, vio-                          convert pyrophosphate (PPi) inorganic phosphate (Pi); <br /> lating mass balance equations governing system,                      however, reaction likely occur spontaneously a <br /> a number biomass constituents seen depen-                          reduced rate. Without prsA gene network capable <br /> dency oxygen.                                                                    generating phosphoribosyl pyrophosphate subsequently <br />    Taking closer look connectivity Fig. 3, seen                   make nucleotides deoxynucleotides re- <br /> that alanine arginine major sources carbon                       quired growth. The inability tpi silico mutant to <br /> the network responsible synthesis large number                      grow result H. pylori lacking phosphofructokinase <br /> biomass constituents considering minimal substrates.                       enzyme; therefore, TPI reaction H. pylori unable <br /> To utilize alanine organism use alanine deami-                         make dihydroxyacetone-phosphate. We the <br /> nase reaction converting alanine pyruvate, liberating am-                      tktA gene essential conditions TKTA1 <br /> monia process. Once carbon fed central                           TKTA2, reactions simultaneously deleted. Without <br /> metabolism pyruvate, formation necessary pre-                         reactions network generate sedoheptu- <br /> cursor metabolites widespread biosynthesis accom-                        lose-7-phosphate erythrose-4-phosphate (E4P), required <br /> plished. In case arginine, converted ornithine,               generation number biomass constituents. <br /> then L-glutamate g-semialdehyde, glutamate,                             There biochemical evidence suggesting malate syn- <br /> ultimately &#226;?&#163;-ketoglutarate, routed through-                       thase (ACEB) activity occurs H. pylori (27, 55). However, <br /> out majority central metabolism. <br />    Other substrates capable fulfilling bulk car- <br /> bon requirements organism include glucose, pyruvate,                             TABLE 4. Reactions active extreme pathways each <br /> lactate, malate, fumarate, succinate, &#226;?&#163;-ketoglutarate                                 subsystems H. influenzae H. pyloria <br /> amino acids asparagine, aspartate, glutamine, glutamate,                                                Total no. of <br /> serine, nucleosides adenosine guanosine.                              Sub-             unused reactions                   Unused reactions <br />                                                                                        systemb                                                 H. pylori <br /> Collectively, 14 substrates referred alternative                                      H. influenzae   H. pylori <br /> carbon sources. Glucose enters central metabolic sub- <br /> system Fig. 2 G6P. Lactate, alanine, serine                            A                 28             6          METB1, METB2, ILVC1, ILVC2, <br />                                                                                                                                       ILVE1, ILVE3 <br /> assimilated subsystem pyruvate. Glutamine                                C                  2             0 <br /> glutamate assimilated &#226;?&#163;-ketoglutarate, aspartate                            L                  1             0 <br /> and asparagine assimilated fumarate. The nucleosides                              N                  4            10          ADK2, GMK2, DEOD1, DEOD3, <br />                                                                                                                                        DEOD4, DEOD7, GPT1, DEOB1, <br /> are assimilated ribose-5-phosphate.                                                                                            ADNUC, GNNUC <br />    Deletion study. The effect loss enzymatic function                          T                 11            11          DPEPTP, OPEPTP, GALE, <br /> corresponding gene deletion examined determine                                                                          GLACTP, PMI, DSERTP, <br /> the metabolic network support production                                                                              NUPCTP2, NUPCTP3, <br />                                                                                                                                        NUPCTP6, NARK, NMNTP <br /> required biomass constituents absence selected                             V                  6            10          BIOF, BIOA, BIOD, BIOB, UBIA, <br /> reaction. Each reaction central metabolic network                                                                            THIM, UNKRXNI, THID, THIB, <br /> H. pylori (depicted Fig. 2) individually eliminated                                                                        THIF <br /> the network effects studied different                            a <br />                                                                                              Data unused reactions H. influenzae network reference 60. <br /> environmental/media conditions (compositions described                                b <br />                                                                                              See Table 3, footnote a, subsystem abbreviations. <br />  4588       SCHILLING ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />               TABLE 5. Reaction subsets calculated extreme pathways subsystem H. pylori metabolic networka <br />                                                                 Metabolism group reaction subsets <br />  <br /> Amino acid <br />  DHS1, AROB, AROQ, AROE, AROK, AROA, AROC <br />  TRPD, TRPC1, TRPC2, TRPAB <br />  TYRA1, TYRA2, ASPB2 <br />  METL2, THRB, THRC <br />  DAPA, DAPB, DAPD, DAPC, DAPE, DAPF <br />  ADCSASE_r, METH <br />  SERA, SERC, SERB <br />  SPEA, SPEB <br />  SPED, SPEE, MTHAKN, MTHRKN, MTHIPIS, NE1PH, NE3UNK, TNSUNK <br />  CYSDN, CYSC, CYSH, CYSU, CYSE, CYSK <br />  <br /> Central <br />   FBP, FBA_r <br />   GAP, PGK <br />   PGM, ENO <br />   PGL, EDD, EDA <br />   GLTA, ACNB, ICD <br />   SCOT, ATOB <br />  <br /> Lipid cell envelope <br />   ACCABCD, FABD <br />   FABH1, FABF <br />   C120SN, DGKA, LPXA, ENVA, LPXD, USHA12, LPXB, LPXK, KDTAI, KDOLIPH, ASPISO, KDSA, KDOPH, KDSB, PAPHTSE, GMHA, LPSSYN <br />   C141SY, C160SN, C161SY, C181SY, PLS <br />   PGSA2, PGPP <br />   GLMS, GLMM, GLMU <br />   MURZ, MURB, MURC, MURD, MURE, MURF, GLR, DDLA, MRAY, MURG <br />  <br /> Nucleotide <br />  PYRA, PYRB, PYRC, PYRD <br />  PYRE, PYRF <br />  PURF, PURD, PURL, PURM, PURK, PURE, PURC, PURB1, PURH1, PURH2 <br />  PURA, PURB2 <br />  GUAB, GUAA <br />  NDK4, NRDAB4 <br />  NDK6, NRDAB1 <br />  NDK7, NRDAB3 <br />  NDK8, TMK1 <br />  DEOD2, DEOD8_r <br />  <br /> Transport <br />   ADHE2, ETHTP_r <br />   PTA, ACKA <br />   GALU, ALGC1 <br />   GLCTP, GLK1 <br />   PROTPI, NATP_r <br />   LACTP, DLD <br />   BCRBTP_r, ICFA <br />   GLCD, GLLDHR, KATA <br />  <br /> Vitamin cofactor <br />   FOLE, DNTPH, DHPPH, FOLB, FOLK, PABB, PABC, FOLP, FOLC <br />   FOLD1, FOLD2 <br />   GLTX, HEMA, HEML, HEMB, HEMC, HEMD, HEME, HEMF, HEMG, HEMH <br />   RIBA, RIBD1, RIBD2, PMDPHT, RIBB, RIBE, RIBC, RIBF1, RIBF2 <br />   PANB, ILVC3, PAND, PANC, COAA, PCLIG, PCDCL, PATRAN, DPHCOAK <br />   IPPPISO, ISPA1, ISPA2 <br />   NADB, NADA, NADC, NADD, NADE <br />   MENF, MEND1, MEND2, MENC, MENE, MENB, MENA <br />   a <br />     Each set reactions occurs flux ratios type I type II pathways particular fluxes active. Reverse enzymatic <br /> activity indicated suffix &#226;&#8364;&#339;_r&#226;&#8364;? end flux name. <br />  <br />  <br />  <br /> there little evidence support isocitrate lyase activity,                     silico finding similar results GLCD reaction <br /> leading conclusion incomplete glyoxylate                      supported experimentally determined essentiality the <br /> bypass H. pylori. The silico model predicts ACEB                        glcD gene (HP0509) encoding glycolate oxidase. <br /> essential. This result necessity reaction                      Table 7 contains series 17 deletion results comparing the <br /> the processing glyoxylate, produced net result                      silico-determined essentiality genes vitro essen- <br /> the production glycolaldehyde. There consistency                        tiality recently determined using vector-free allelic re- <br />  VOL. 184, 2002                                                                               METABOLIC MODEL OF H. PYLORI                   4589 <br />  <br />  <br />  <br />  <br />   FIG. 3. Minimal substrate requirements H. pylori generate 47 required biomass constituents synthesized. Lines <br /> connecting substrates biomass constituents indicate network unable generate product substrate. The dashed lines <br /> indicate alanine arginine required production connected biomass requirement. See Table 1 abbreviations. <br />  <br />  <br /> placement mutagenesis technique (10). In 10 17                  essential; however, experimental evidence <br /> cases silico model complete agreement             convincingly support requirement glucose growth <br /> vitro determinations. In 2 remaining cases (fbp mqo)            clearly enhances growth rate. This area <br /> there partial agreement vitro data depending             investigation help refine model and <br /> the conditions considered.                                                 improve predictive capabilities. The gpt gene encoding gua- <br />    The discrepancies silico deletion study              phosphoribosyl transferase essential vitro, which <br /> the experimental study lead interesting testable hypotheses             imply novo purine biosynthesis does occur. <br /> that performed probe organisms&#226;&#8364;&#8482; capa-                If genes purine biosynthesis removed from <br /> bilities. As example, consider eda edd genes             model, gpt calculated essential. <br /> Entner-Doudoroff pathway. These genes essential de- <br /> termined vitro, thought to critical                                      DISCUSSION <br /> role catabolism glucose. However, re- <br /> quired model glucose catabolism re-                     Here initiated silico model development <br /> quired. If glucose used alternative carbon source              process H. pylori, less-characterized organism particularly <br /> alanine arginine, model predicts edd eda                 metabolic standpoint. With complete genome se- <br />  4590       SCHILLING ET AL.                                                                                                             J. BACTERIOL. <br />  <br />  <br />      TABLE 6. Model results reactions deleted central                            As example, exploration reactions central <br />   metabolism different substrate availability conditionsa                 metabolism allow conversion &#226;?&#163;-ketoglu- <br />                                In silico essentiality medium                      tarate succinate complete TCA cycle inves- <br /> Deleted                                                                              tigated. This critical issue central metabolism, it <br />                                                                      Minimal &#226;&#171;&#185; <br /> reaction                  Minimal &#226;&#171;&#185;          Minimal &#226;&#171;&#185;                               appears fumarate reductase function reversible <br />              Minimal                                              carbon sources &#226;&#171;&#185; <br />                            glucose         carbon sources <br />                                                                      amino acids     manner, allowing assimilation wide range <br /> ACEB            &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#186;                       &#226;&#171;&#186;           additional compounds. Growth studies designed that <br /> ACNB            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           test ability organism utilize succinate, &#226;?&#163;-ke- <br /> AKO             &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           toglutarate, compounds assimilated into <br /> ATOB            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br /> ENO             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br />                                                                                      &#226;?&#163;-ketoglutarate major carbon sources. The function the <br /> FBA             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           SCOT reaction associated loci HP0691 HP0692 can <br /> FRD             &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           investigated relation proposed AKO reaction <br /> FUMC            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           experiments introduce acetoacetate avail- <br /> G6PD1           &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br /> G6PD2           &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           able substrate organism. In way silico <br /> GAP             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           model used generate testable hypotheses can <br /> GLTA            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           rapidly validated invalidated integration with <br /> ICD             &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br /> MAEB            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br />                                                                                      experimentation, leading improved insight TCA <br /> PGI             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           cycle critical role plays H. pylori. This is <br /> PGK             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           simple example system-modeling strategies can <br /> PGL             &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           used drive experimental research organism. <br /> PGM             &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br /> PPA             &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#186;                       &#226;&#171;&#186;              In H. pylori examination reaction subsets indicates <br /> PPSA            &#226;&#171;&#186;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           genetic order genes reaction subset not <br /> PRSA            &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#186;                       &#226;&#171;&#186;           conserved terms operon structure. This general result <br /> RPE             &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br /> RPI             &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#185;                       &#226;&#171;&#185; <br />                                                                                      suggest well-coordinated regulation many <br /> SCOT            &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           genes ensure simultaneous expression. How- <br /> TKTA1           &#226;&#171;&#185;              &#226;&#171;&#185;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           ever, original examination genome were <br /> TKTA2           &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#185;                       &#226;&#171;&#185;           helix-turn-helix motifs entire <br /> TPI             &#226;&#171;&#186;              &#226;&#171;&#186;                 &#226;&#171;&#186;                       &#226;&#171;&#186; <br />                                                                                      genome (66), relatively two-component systems <br />   a <br />     The composition of media described Results            (3), indicating limited regulation occurs <br /> section. Symbols: (&#226;&#171;&#185;), ability make 47 biomass constituents absence <br /> of indicated reaction; (&#226;&#171;&#186;), inability make biomass constituents.          genetic level H. pylori. If organism a <br />                                                                                      limited level transcriptional regulation, possible hy- <br />                                                                                      pothesis genes organism (particularly <br /> quence available additional biochemical phys-                         calculated reaction subsets) constitutively ex- <br /> iological data, metabolic network organism                          pressed organism. This investigated the <br /> reconstructed. The constraint-based modeling methodology                             use whole-genome expression analysis examine ex- <br /> was applied assess performance capabilities                       pression genes comprise particular subset. An <br /> organism. This assessment included use extreme-path-                          example subset investigation A5 set <br /> way analysis FBA. The overall modeling process assist                        comprising reactions lysine biosynthesis, asso- <br /> in accelerating pace biological discovery generating                       ciated genes widely dispersed genome (dapA- <br /> experimentally testable hypotheses. These hypotheses                             HP1013, dapB-HP0510, dapD-HP0626, dapE-HP0212, and <br /> represent novel insights organism, help support                          dapF-HP0566). Overall results reaction subset gene <br /> previous insights, resolve existing discrepancies                         locations tend support notion bacterium <br /> different sets experimental results inconclu-                        contains little regulation metabolic network inflexi- <br /> sive. The silico model improved itera-                       ble, adapted limited range environmental <br /> tion modeling experimentation, provide                         conditions. <br /> the concise representation organism&#226;&#8364;&#8482;s known func-                           While clear cell physiology H. pylori requires <br /> tional capabilities.                                                                 oxygen, precise dependence oxygen generate the <br />    There examples topics model                             biomass constituents likely reason true <br /> assist exploring further. Two examples include                        physiological requirement oxygen (33). Without oxygen the <br /> novo biosynthesis alanine organism                          cell effectively accept electrons various carri- <br /> ability perform novo purine biosynthesis. In case                         ers subsequently unable generate energy through <br /> there evidence literature interpreted                       oxidative phosphorylation. In bacteria situation is <br /> support sides argument ability or-                        overcome utilizing electron acceptors shift to <br /> ganism produce biomass constituents. Each                            fermentative growth use substrate level phosphor- <br /> possible cases examined model used                            ylation glycolysis. From reconstruction pathway <br /> design experiments defined media aimed determining                             analysis network clear H. pylori limited <br /> these capabilities. In combination experimental determi-                        ability perform substrate level phosphorylation the <br /> nation whole-genome expression patterns bioinformatic                         absence pyruvate kinase reaction complete glycolysis. <br /> sequence analysis, possible determine loca-                         A particularly interesting conclusion stemming these <br /> tion genes responsible metabolic activities.                            studies relates amino acid requirements organism. <br />  VOL. 184, 2002                                                                                               METABOLIC MODEL OF H. PYLORI                           4591 <br />  <br />  <br />                      TABLE 7. In silico results compared experimental results essentiality 17 genes modela <br />                                                                                                                In silico essentiality medium <br />  Deleted                                                   Locus                   In vitro                                                      Minimal &#226;&#171;&#185;        Agree- <br />                         Gene(s)                                                                              Minimal &#226;&#171;&#185;        Minimal &#226;&#171;&#185; <br />  reaction                                             identification(s)          essentiality   Minimal                                        carbon sources     ment <br />                                                                                                               glucose       carbon sources <br />                                                                                                                                                &#226;&#171;&#185; amino acids <br />  <br /> ACPS           acpS                           HP0808                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> AROQ           aroQ                           HP1038                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> DCD            dcd                            HP0372                                 NE             &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;             Yes <br /> EDA            eda                            HP1099                                 E              &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> EDD            edd                            HP1100                                 E              &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> FBP            fbp                            HP1385                                 E              &#226;&#171;&#186;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> GLCD           glcD                           HP0509                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> GPT1/2/3       gpt                            HP0735                                 E              &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> MQO            mqo                            HP0086                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> MURB           murB                           HP1418                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> MURF           murF                           HP0740                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> OOR            oorA, oorB, oorC, oorD         HP0589, HP0590, HP0591,                E              &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185; <br />                                                HP0588 <br /> POR            porA, porB, porC, porD         HP1110, HP1111, HP1108,                E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br />                                                HP1109 <br /> PYRE           pyrE                           HP1257                                 E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br /> SPEE           speE                           HP0832                                 NE             &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186; <br /> TAL            tal                            HP1495                                 NE             &#226;&#171;&#185;            &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;             Yes <br /> TRXB           trxA, trxB                     HP0824, HP0825                         E              &#226;&#171;&#186;            &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;             Yes <br />    a <br />      All essential silico predictions shaded, &#226;&#171;&#186; sign indicating inability produce biomass demands. The vitro essentiality data taken <br /> from reference 10 (E, essential; NE, essential). <br />  <br />  <br />  <br />  <br /> The human host H. pylori colonizes nutritional re-                              necessitate reprogramming genetic expression. This <br /> quirements amino acids. These referred                            lack regulation extremely <br /> nine human essential amino acids histidine, isoleucine,                          limited competition organism faces the <br /> leucine, lysine, methionine, phenylalanine, threonine, trypto-                           lining stomach protected host&#226;&#8364;&#8482;s <br /> phan, valine. Upon careful examination require-                             immune response. <br /> ments, seen amino acids required                             Due role prominent human pathogen is <br /> bacterium, amino acids human body                             identifying essential genes H. pylori. From the <br /> acquire nutritional sources. Alanine arginine                           deletion analysis clear essential nature gene <br /> two amino acids H. pylori requires required                           dependent environmental conditions, are <br /> by human body (65).                                                                  genes essential conditions others. <br />    This general strategy metabolic design ideal                           Seven genes identified predicted essential <br /> for bacterium. It appears evolved                               conditions examined, repre- <br /> eliminate genes enzymes energetically                             sent interesting candidates experimental validation studies. <br /> unfavorable biosynthetic pathways amino acid production,                             Currently limited studies focusing identifica- <br /> as guaranteed presence host gastric envi-                          tion essential genes H. pylori, exception one <br /> ronment proteolysis continuously occurring. For H.                              recent study Chalker et al. (10). The silico model con- <br /> influenzae number required amino acids                              tained 24 genes corresponding 17 reactions ex- <br /> (glutamate, arginine, cysteine), human                             perimentally examined essentiality provide validation <br /> essential amino acids. These amino acid requirements provide                             model. Ten 17 case predictions (&#226;&#172;&#402;60%) in <br /> an example possible adaptations of organ-                           accordance experimental data, 2 agreement <br /> isms particular environmental niche. In fact,                           certain medium conditions. The experimental medium <br /> use model suggests use amino acids                               conditions complex, making direct comparison diffi- <br /> major carbon source H. pylori leads significant increase                        cult. In general results encouraging. Through inte- <br /> in generation ammonia organism use                                grated modeling experimentation expected <br /> neutralize surrounding extracellular environment.                                    accuracy model increase result of <br />    From comparative modeling distinct prop-                               enhanced biological discovery organism&#226;&#8364;&#8482;s metabolic <br /> erties metabolism of organisms fur-                            capabilities. The accuracy prediction H. pylori than <br /> ther support notion environmental adaptation. These                              previously observed similar predictions E. <br /> include range preferred carbon substrates,                             coli S. cerevisiae, predictions gene deletions <br /> additional growth requirements. In H. pylori                              80 90% accurate (20; I. Famili, J. Foerster, J. Nielsen, <br /> less evidence support coordinated regulation reac-                             B. O. Palsson, submitted publication). For in <br /> tions genes comprising reaction subsets                            silico models, improved accuracy anticipated, are <br /> H. influenzae. This finding indicate H. pylori does                         model organisms thoroughly character- <br /> witness significant changes environment                              ized. <br />  4592       SCHILLING ET AL.                                                                                                                              J. BACTERIOL. <br />  <br />  <br />    At outset posed question learned                         11. Chen, M., L. P. Andersen, L. Zhai, A. Kharazmi. 1999. Characterization <br />                                                                                           respiratory chain Helicobacter pylori. FEMS Immunol. Med. Mi- <br /> from genomes less-characterized organisms. Through                                 crobiol. 24:169&#226;&#8364;&#8220;174. <br /> the use constraint-based modeling demonstrated                             12. Chvatal, V. 1983. Linear programming. W. H. Freeman Company, New <br /> that derived genome                               York, N.Y. <br />                                                                                       13. Corthesy-Theulaz, I. E., G. E. Bergonzelli, H. Henry, D. Bachmann, D. F. <br /> it examined parallel existing experimental data. In-                           Schorderet, A. L. Blum, L. N. Ornston. 1997. Cloning character- <br /> tegrating genomic, biochemical, physiological data to-                                ization Helicobacter pylori succinyl CoA:acetoacetate CoA-transferase, a <br /> gether context silico model serves practical                         novel prokaryotic member CoA-transferase family. J. Biol. Chem. <br />                                                                                           272:25659&#226;&#8364;&#8220;25667. <br /> tool quest understand physiology organism.                      14. Covert, M. W., C. H. Schilling, I. Famili, J. S. Edwards, I. I. Goryanin, E. <br /> In addition task data-driven model development,                                 Selkov, B. O. Palsson. 2001. Metabolic modeling microbial strains in <br />                                                                                           silico. Trends Biochem. Sci. 26:179&#226;&#8364;&#8220;186. <br /> there appears to gained prospective im-                            15. De Reuse, H., A. Labigne, D. Mengin-Lecreulx. 1997. The Helicobacter <br /> plementation silico model biological discovery. The                          pylori ureC gene codes phosphoglucosamine mutase. J. Bacteriol. 179: <br /> success model-driven discovery dependent                                3488&#226;&#8364;&#8220;3493. <br />                                                                                       16. Devaux, F., P. Marc, C. Jacq. 2001. Transcriptomes, transcription acti- <br /> ability design experimentally testable hypotheses carry                            vators microarrays. FEBS Lett. 498:140&#226;&#8364;&#8220;144. <br /> out studies support iterative model development                        17. Doig, P., B. L. Jonge, R. A. Alm, E. D. Brown, M. Uria-Nickelsen, B. <br /> paradigm. For less-characterized organisms notion in-                              Noonan, S. D. Mills, P. Tummino, G. Carmel, B. C. Guild, D. T. Moir, G. F. <br />                                                                                           Vovis, T. J. Trust. 1999. Helicobacter pylori physiology predicted from <br /> tegrating model experimentation lead more-                               genomic comparison strains. Microbiol. Mol. Biol. Rev. 63:675&#226;&#8364;&#8220;707. <br /> efficient discovery, higher rate                      18. Edwards, J., B. Palsson. 1999. Systems properties Haemophilus <br /> common model organisms. This paradigm efficient discov-                               influenzae Rd metabolic genotype. J. Biol. Chem. 274:17410&#226;&#8364;&#8220;17416. <br />                                                                                       19. Edwards, J. S., R. U. Ibarra, B. O. Palsson. 2001. In silico predictions of <br /> ery facilitated enabled information contained                            Escherichia coli metabolic capabilities consistent experimental data. <br /> the genome, gained genome                              Nat. Biotechnol. 19:125&#226;&#8364;&#8220;130. <br />                                                                                       20. Edwards, J. S., B. O. Palsson. 2000. The Escherichia coli MG1655 in <br /> less-characterized organisms H. pylori, provided                             silico metabolic genotype: <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, characteristics, capabilities. <br /> there methods developed capitalize wealth                               Proc. Natl. Acad. Sci. USA 97:5528&#226;&#8364;&#8220;5533. <br /> information. Such genome-enabled science provides additional                          21. Edwards, J. S., R. Ramakrishna, C. H. Schilling, B. O. Palsson. 1999. <br />                                                                                           Metabolic flux balance analysis, p. 13&#226;&#8364;&#8220;57. In S. Y. Lee E. T. Papoutsakis <br /> justification genome sequencing efforts.                                              (ed.), Metabolic engineering. Marcel Dekker, New York, N.Y. <br />                                                                                       22. Finel, M. 1998. Does NADH play central role energy metabolism in <br />                           ACKNOWLEDGMENTS                                                 Helicobacter pylori? Trends Biochem. Sci. 23:412&#226;&#8364;&#8220;413. <br />                                                                                       23. Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness, <br />    We acknowledge insightful highly constructive reviews                     A. R. Kerlavage, C. J. Bult, J. F. Tomb, B. A. Dougherty, J. M. Merrick, K. <br /> were received manuscript. These reviews led number                        McKenney, G. Sutton, W. Fitzhugh, C. Fields, J. D. Gocayne, J. Scott, R. <br />                                                                                           Shirley, L. I. Liu, A. Glodek, J. M. Kelley, J. F. Weidman, C. A. Phillips, T. <br /> enhancements model terms biological content that <br />                                                                                           Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T. <br /> improved predictive capabilities quality results pre-                      Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L. <br /> sented. This exemplary true nature collaborative                         Fuhrmann, N. S. M. Geoghagen, C. L. Gnehm, L. A. McDonald, K. V. Small, <br /> iterative model development.                                                              C. M. Fraser, H. O. Smith, J. C. Venter. 1995. Whole-genome random <br />    This work funded NIH (GM57089), NSF                             sequencing assembly Haemophilus influenzae Rd. Science 269:496&#226;&#8364;&#8220; <br /> (MCB 73384 BES 14092) Whitaker Foundation                             498, 507&#226;&#8364;&#8220;512. <br /> graduate student fellowships biomedical engineering (C.S. I.F.).               24. Ge, Z., D. E. Taylor. 1997. The Helicobacter pylori gene encoding phos- <br /> We acknowledge funding NSF, DOE, DARPA                               phatidylserine synthase: sequence, expression, insertional mutagenesis. <br />                                                                                           J. Bacteriol. 179:4970&#226;&#8364;&#8220;4976. <br /> the Church Laboratory. <br />                                                                                       25. Gombert, A. K., J. Nielsen. 2000. Mathematical modelling metabo- <br />                                                                                           lism. Curr. Opin. Biotechnol. 11:180&#226;&#8364;&#8220;186. <br />                                  REFERENCES                                           26. Hazell, S. L., G. L. Mendz. 1997. How Helicobacter pylori works: an <br />  1. Alm, R. A., J. Bina, B. M. Andrews, P. Doig, R. E. Hancock, T. J. Trust.          overview metabolism Helicobacter pylori. Helicobacter 2:1&#226;&#8364;&#8220;12. <br />     2000. Comparative genomics Helicobacter pylori: analysis outer          27. Hoffman, P. S., A. Goodwin, J. Johnsen, K. Magee, S. J. Veldhuyzen van <br />     membrane protein families. Infect. Immun. 68:4155&#226;&#8364;&#8220;4168.                               Zanten. 1996. Metabolic activities metronidazole-sensitive -resistant <br />  2. Arigoni, F., F. Talabot, M. Peitsch, M. D. Edgerton, E. Meldrum, E. Allet, R.         strains Helicobacter pylori: repression pyruvate oxidoreductase and <br />     Fish, T. Jamotte, M. L. Curchod, H. Loferer. 1998. A genome-based                 expression isocitrate lyase activity correlate resistance. J. Bacteriol. <br />     approach identification essential bacterial genes. Nat. Biotechnol.        178:4822&#226;&#8364;&#8220;4829. <br />     16:851&#226;&#8364;&#8220;856.                                                                       28. Hughes, N. J., P. A. Chalk, C. L. Clayton, D. J. Kelly. 1995. Identification <br />  3. Beier, D., R. Frank. 2000. Molecular characterization two-component            carboxylation enzymes characterization novel four-subunit pyru- <br />     systems Helicobacter pylori. J. Bacteriol. 182:2068&#226;&#8364;&#8220;2076.                          vate:flavodoxin oxidoreductase Helicobacter pylori. J. Bacteriol. 177: <br />  4. Benson, D. A., M. S. Boguski, D. J. Lipman, J. Ostell, B. F. Ouellette.           3953&#226;&#8364;&#8220;3959. <br />     1998. GenBank. Nucleic Acids Res. 26:1&#226;&#8364;&#8220;7.                                         29. Hughes, N. J., C. L. Clayton, P. A. Chalk, D. J. Kelly. 1998. Helicobacter <br />  5. Birkholz, S., U. Knipp, E. Lemma, A. Kroger, W. Opferkuch. 1994.                  pylori porCDAB oorDABC genes encode distinct pyruvate:flavodoxin and <br />     Fumarate reductase Helicobacter pylori&#226;&#8364;&#8221;an immunogenic protein.                     2-oxoglutarate:acceptor oxidoreductases mediate electron transport to <br />     J. Med. Microbiol. 41:56&#226;&#8364;&#8220;62.                                                          NADP. J. Bacteriol. 180:1119&#226;&#8364;&#8220;1128. <br />  6. Bonarius, H. P. J., G. Schmid, J. Tramper. 1997. Flux analysis             30. Jorgensen, M. A., J. Manos, G. L. Mendz, S. L. Hazell. 1998. The mode <br />     underdetermined metabolic networks: quest missing constraints.            action metronidazole Helicobacter pylori: futile cycling reduction? <br />     Trends Biotechnol. 15:308&#226;&#8364;&#8220;314.                                                        J. Antimicrob. Chemother. 41:67&#226;&#8364;&#8220;75. <br />  7. Burns, B. P., S. L. Hazell, G. L. Mendz. 1995. Acetyl-CoA carboxylase         31. Kather, B., K. Stingl, M. E. van der Rest, K. Altendorf, D. Molenaar. <br />     activity Helicobacter pylori requirement increased CO2              2000. Another unusual type citric acid cycle enzyme Helicobacter pylori: <br />     growth. Microbiology 141:3113&#226;&#8364;&#8220;3118.                                                   malate:quinone oxidoreductase. J. Bacteriol. 182:3204&#226;&#8364;&#8220;3209. <br />  8. Burns, B. P., G. L. Mendz, S. L. Hazell. 1998. A novel mechanism          32. Kelly, D. J. 1998. The physiology metabolism human gastric <br />     resistance antimetabolite N-phosphonoacetyl-L-aspartate Helico-             pathogen Helicobacter pylori. Adv. Microb. Physiol. 40:137&#226;&#8364;&#8220;189. <br />     bacter pylori. J. Bacteriol. 180:5574&#226;&#8364;&#8220;5579.                                       33. Kelly, D. J., N. J. Hughes, R. K. Poole. 2001. Microaerobic physiology: <br />  9. Chalk, P. A., A. D. Roberts, W. M. Blows. 1994. Metabolism pyruvate            aerobic respiration, anaerobic respiration, carbon dioxide metabolism, <br />     glucose intact cells Helicobacter pylori studied 13C NMR spec-           p. 113&#226;&#8364;&#8220;124. In H. L. T. Mobley, G. L. Mendz, S. L. Hazell (ed.), <br />     troscopy. Microbiology (Reading) 140:2085&#226;&#8364;&#8220;2092.                                       Helicobacter pylori: physiology genetics. ASM Press, Washington, D.C. <br /> 10. Chalker, A. F., H. W. Minehart, N. J. Hughes, K. K. Koretke, M. A. Lonetto,       34. Marais, A., G. L. Mendz, S. L. Hazell, F. Me&#194;&#180;graud. 1999. Metabolism <br />     K. K. Brinkman, P. V. Warren, A. Lupas, M. J. Stanhope, J. R. Brown,              genetics Helicobacter pylori: genome era. Microbiol. Mol. Biol. <br />     P. S. Hoffman. 2001. Syst. identification selective essential genes Heli-       Rev. 63:642&#226;&#8364;&#8220;674. <br />     cobacter pylori genome prioritization allelic replacement mutagenesis.     35. Marcelli, S. W., H. T. Chang, T. Chapman, P. A. Chalk, R. J. Miles, and <br />     J. Bacteriol. 183:1259&#226;&#8364;&#8220;1268.                                                          R. K. Poole. 1996. The respiratory chain Helicobacter pylori: identification <br />  VOL. 184, 2002                                                                                               METABOLIC MODEL OF H. PYLORI                           4593 <br />  <br />       cytochromes effects oxygen cytochrome menaquinone            55. Pitson, S. M., G. L. Mendz, S. Srinivasan, S. L. Hazell. 1999. The <br />       levels. FEMS Microbiol. Lett. 138:59&#226;&#8364;&#8220;64.                                              tricarboxylic acid cycle Helicobacter pylori. Eur. J. Biochem. 260:258&#226;&#8364;&#8220;267. <br /> 36.   Marshall, B. J. 2001. One years discovery rediscovery           55a.Price, N. D., J. A. Papin, B. O. Palsson. 2002. Determination redun- <br />       Helicobacter pylori association peptic ulcer disease, p. 19&#226;&#8364;&#8220;24. In       dancy systems properties metabolic network Helicobacter <br />       H. L. Mobley, G. L. Mendz, S. L. Hazell (ed.), Helicobacter pylori:               pylori using genome-scale extreme pathway ana <br /> </body></html>