<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2006 Genetics Society America <br /> DOI: 10.1534/genetics.106.061390 <br />  <br />  <br />  <br />     Glyceraldehyde-3-Phosphate Dehydrogenase Mediates Anoxia Response <br />                     <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>Survival</span> Caenorhabditis elegans <br />  <br />                          Alexander R. Mendenhall, Bobby LaRue Pamela A. Padilla1 <br />                                Department Biological Sciences, University North Texas, Denton, Texas 76203 <br />                                                          Manuscript received June 17, 2006 <br />                                                       Accepted publication August 22, 2006 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Oxygen deprivation role pathology human diseases. Thus un- <br />              derstanding genetic cellular responses hypoxia anoxia oxygen-deprivation-tolerant <br />              organisms Caenorhabditis elegans. In C. elegans DAF-2/DAF-16 pathway, IGF-1/insulin-like sig- <br />              naling pathway, involved dauer formation, longevity, stress resistance. In report compared <br />              response wild-type daf-2(e1370) animals anoxia. Unlike wild-type animals, daf-2(e1370) <br />              animals enhanced anoxia-survival <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> survive long-term anoxia high- <br />              temperature anoxia, accumulate significant tissue damage conditions, are <br />              motile 24 hr anoxia. <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>RNA interference</span> used screen DAF-16-regulated genes suppress the <br />              daf-2(e1370)-enhanced anoxia-survival phenotype. We identified gpd-2 gpd-3, nearly identical genes <br />              operon encode glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase. We that <br />              daf-2(e1370)-enhanced anoxia phenotype dependent gpd-2 gpd-3, the <br />              motility animals exposed brief periods anoxia prematurely arrested gpd-2/3(RNAi) daf- <br />              2(e1370);gpd-2/3(RNAi) animals. These data suggest gpd-2 gpd-3 serve protective role tissue <br />              exposed oxygen deprivation. <br />  <br />  <br />  <br />  <br /> O      XYGEN deprivation critical factor several <br />         medical problems, including myocardial infarc- <br /> tion, pulmonary disease dysfunction, anemia, blood <br />                                                                             rhabditis elegans) adapted survive oxygen depri- <br />                                                                             vation (Foe Alberts 1985; Hochachka et al. 1993; <br />                                                                             Hand 1998; Padilla et al. 2002; Fuson et al. 2003). <br /> loss, suffocation, sleep apnea, stroke, resistance                   Thus, likely various oxygen-deprivation protec- <br /> solid tumor cells radiation chemotherapy                             tive mechanisms exist metazoans. <br /> (Semenza 2000). The detrimental effects oceanic                             Identifying molecular genetic responses of <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;dead zone&#226;&#8364;&#8482;&#226;&#8364;&#8482; regions, oxygen levels reduced                   oxygen-deprivation-tolerant organisms hypoxia an- <br /> or depleted natural and/or human activities,                     oxia greatly increased understanding mech- <br /> another concern (Diaz 2001). Thus, health en-                           anisms involved oxygen-deprivation sensing and <br /> vironmental issues associated oxygen deprivation                       survival (Haddad et al. 1997; Haddad 1998; Wingrove <br /> are economically relevant challenges. Animal model sys-                     O&#226;&#8364;&#8482;Farrell 1999; Jiang et al. 2001; Ghabrial et al. <br /> tems used understand molecular physio-                       2003; Nystul et al. 2003; Gray et al. 2004; Morton <br /> logical response organisms, tissues, cells                 2004). For example, analysis hypoxia-inducing <br /> oxygen deprivation. Yet, date, oxygen-deprivation pro-                   factor 1 (HIF-1) HIF-prolyl hydroxylase signal <br /> tective tissue repair mechanisms completely                      transduction pathway demonstrated aspects of <br /> understood (Hochachka et al. 1997; Hermes-Lima                          oxygen sensing conserved vertebrates and <br /> Zenteno-Savin 2002; Lahiri et al. 2005).                                    invertebrates (Epstein et al. 2001; Maxwell et al. 2001; <br />    For vertebrates, including humans, prolonged                        Semenza 2001). Studies C. elegans hif-1(ia04) mu- <br /> exposure severe oxygen deprivation results cellular                   tant demonstrated hif-1 required embryos and <br /> death. However, reduction absence oxygen                       larvae survive hypoxia (Jiang et al. 2001; Treinin et al. <br /> not fatal organisms; survive days                    2003). However, hif-1 required adult <br /> years hypoxia anoxia. For example, harbor seal                    animal survive hypoxia (1% O2) embryo <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (Phoca vitulina), turtle (Chrysemys picta belli), brine                     survive anoxia ( Jiang et al. 2001; Padilla et al. 2002; <br /> shrimp (Artemia franciscana), zebrafish (Danio rerio),                      Shen Powell-Coffman 2003). Together, stud- <br /> fruit fly (Drosophila melanogaster), nematode (Caeno-                   ies suggest genetic pathways involved <br />                                                                             oxygen-deprivation response survival that <br />   1 <br />                                                                             genetic pathways dependent devel- <br />    Corresponding author: University North Texas, Department of <br /> Biological Sciences, P.O. Box 305220, Chestnut Avenue C, Denton,        opmental state organism level available <br /> TX 76203. E-mail: ppadilla@unt.edu                                          oxygen. <br />  <br /> Genetics 174: 1173&#226;&#8364;&#8220;1187 (November 2006) <br />  1174                                       A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br />    Wild-type C. elegans exposed anoxia (,0.001 kPa O2)          daf-2(e1370) high-temperature anoxia survival pheno- <br /> in laboratory culture temperature (20&#194;&#176;) conditions             type, used RNA interference (RNAi) decrease the <br /> enters reversible state suspended animation. In           expression specific genes upregulated daf- <br /> anoxic environment, embryonic postembryonic de-                2(e1370) strain. We determined gpd-2 gpd-3, two <br /> velopment arrests. Furthermore, postembryonic                  nearly identical genes operon encode the <br /> animal stops feeding, immobile, does               glycolytic enzyme glyceraldehyde-3-phosphate dehydro- <br /> lay eggs. After reexposure normoxic environment,              genase (GAPDH), required response anoxia. <br /> the nematodes proceed development. C. elegans          Here, suggest gpd-2 gpd-3 involved with <br /> stages development survive 1 day anoxia        cell tissue protection anoxia exposure. The <br /> a viability  90% (Van Voorhies Ward 2000;                   reduction expression glycolytic genes by <br /> Padilla et al. 2002; Hajeri et al. 2005). The dauer larvae,        RNAi does result phenotypes similar daf- <br /> a developmentally arrested larval stage, survive           2(e1370);gpd-2/3(RNAi), suggesting be <br /> days anoxia, making stage resistant            mere flux glycolysis required for <br /> anoxia (Anderson 1978; Padilla et al. 2002). The long-             anoxia survival. <br /> term anoxia survival phenotype dauer larvae con- <br /> sistent ability survive stress. Yet, studies <br /> are needed understand molecular mechanism                             MATERIALS AND METHODS <br /> the dauer larvae use survive anoxia. <br />    Given dauer larvae enhanced anoxia                Strains growth conditions: The wild-type Bristol strain <br />                                                                    (N2) mutant strains cultured using NGM plates <br /> survival phenotype, investigate rela-     seeded Escherichia coli (OP50) raised 20&#194;&#176; as <br /> tionship anoxia survival genetic pathways               described (Sulston Hodgkin 1988). Synchronized pop- <br /> involved dauer formation. One dauer reg-               ulations animals obtained allowing embryos, <br /> ulatory pathways includes daf-2 daf-16 genes,              obtained hypochlorite-treated adults, grow specific <br /> which encode insulin/IGF-1 receptor-like protein               developmental stages. Anatomical markers gonad <br />                                                                    morphology hours L4 molt used determine <br /> and fork-head transcription factor, respectively                 developmental stage animal. Wild-type daf- <br /> (Larsen et al. 1995; Kimura et al. 1997; Riddle                2(e1370) adult animals collected  21&#226;&#8364;&#8220;25 17&#226;&#8364;&#8220;21 hr <br /> Albert 1997; Gems et al. 1998). In daf-2(e1370) ani-           L4-to-adulthood molt, respectively. The following <br /> mal, <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> insulin receptor kinase           genetic strains obtained Caenorhabditis <br /> domain, DAF-16 protein translocated nu-              elegans Genetics Center colleagues: daf-2(e1370)III, daf- <br />                                                                    16(m26)I, daf-2(e1370)III, daf-16(mu86)I, daf-16(m26)I, and <br /> cleus. Phenotypes observed daf-2(e1370) animal              PD4792(mIs11 IV). <br /> include constitutive dauer formation increased tem-                Oxygen-deprivation experiments: For experiments, the <br /> perature, alteration gene expression, increased long-           nematodes exposed anoxic environment using the <br /> evity, resistance variety stresses, changes         BioBag type A environmental chamber previously described <br /> metabolism (Kenyon et al. 1993; Vanfleteren De                 (Becton Dickinson, Cockeysville, MD; Padilla et al. <br />                                                                    2002). The anoxia exposure time (1&#226;&#8364;&#8220;5 days) temper- <br /> Vreese 1996; Kimura et al. 1997; Ogg et al. 1997; Gems             ature conditions (20&#194;&#176; 28&#194;&#176;) noted experiment. <br /> et al. 1998; Lee et al. 2003; McElwee et al. 2003, 2004;           For purpose studies, refer long-term anoxia <br /> Munoz 2003; Murphy et al. 2003; Van Voorhies 2003;                 $3 days ,0.001 kPa O2, 20&#194;&#176; high-temperature <br /> Burnell et al. 2005). DNA microarray studies deter-                anoxia 1 day ,0.001 kPa O2 28&#194;&#176;. <br /> mined genes involved stress response                    Anoxia survival assays: Nematodes L2 larval <br />                                                                    stage young adult hermaphrodite stage assayed <br /> metabolism upregulated daf-2 mutant. How-               anoxia survival. The L2 larvae collected  24 hr after <br /> ever, known gene expression metabolic            newly hatched L1 placed seeded plates. Due dif- <br /> changes involved mechanism               ferent developmental progression rates wild-type <br /> daf-2 mutant animal survives stress.                               daf-2(e1370) animals, actual time reach adulthood <br />    A study researchers screened genetic               varied strains. Therefore, collected adult <br />                                                                    wild-type daf-2(e1370) animals  16.5&#226;&#8364;&#8220;25 hr L4-to- <br /> mutants survive hypoxia high temperature                 adulthood molt. The adult hermaphrodites assayed a <br /> (20 hr 0.3% O2, 28&#194;&#176;) identified mutation daf-2         developmental stage offspring consisted em- <br /> gene, providing direct genetic evidence             bryos newly hatched L1 larvae. Nematodes exposed <br /> that daf-2/daf-16 insulin-like signaling pathway            anoxia described 24-hr postanoxic animals <br /> involved oxygen deprivation survival C. elegans            examined survival visual inspection movement <br />                                                                    reaction platinum wire. To determine anoxia-exposed <br /> (Scott et al. 2002). In report compared                animals uncoordinated phenotype, animals were <br /> phenotypes daf-2(e1370) wild-type animals               inspected abnormal movement inability along <br /> exposed long-term anoxia (defined $3 days               agar surface. The data collected nematode survival and <br /> of ,0.001 kPa O2, 20&#194;&#176;) high-temperature              phenotype analysis included independent ex- <br /> anoxia (defined 1 day ,0.001 kPa O2 28&#194;&#176;).            periments consisting  50&#226;&#8364;&#8220;150 worms/experiment. <br />                                                                       RNA interference assays: We used RNAi inhibit ex- <br /> We determined that, unlike wild-type animals, daf-             pression genes, including genes upregulated by <br /> 2(e1370) animals survive long-term anoxia high-                DAF-16 genes predicted encode glycolytic enzyme <br /> temperature anoxia. To identify genes involved            (Lee et al. 2003; Murphy et al. 2003). Briefly, daf-2(e1370), <br />                                                    Anoxia Responses C. elegans                                              1175 <br />  <br /> or N2 synchronized L1 larvae, collected grown             Identification putative glycolytic genes C. elegans: We <br /> adulthood NGM (200 mg/ml ampicillin, 12.5 mg/ml tetra-          identified C. elegans genes homologous mammalian glyco- <br /> cycline, 0.5 mg/ml IPTG) seeded E. coli strain    lytic genes using BLASTP analysis Wormbase. Briefly, the <br /> RNAi specified gene. The E. coli strains developed    primary sequences human glycolytic genes obtained <br /> the J. Ahringer laboratory obtained MRC               using NCBI protein database C. elegans ortholog(s) <br /> Geneservice (Cambridge, UK) (Timmons et al. 2001; Kamath           identified using NCBI Wormbase BLASTP <br /> et al. 2003). The bacterial clones used RNAi gpd-2      program. <br /> gpd-3 K10B3.8 K10B3.7, respectively. Throughout          Real time&#226;&#8364;&#8220;quantitative PCR assays: Real time&#226;&#8364;&#8220;quantitative <br /> of experiments used K10B3.8 clone refer        PCR assay (RT&#226;&#8364;&#8220;qPCR) used assay RNAi efficiency well <br /> the RNAi experiment gpd-2/3(RNAi). The long-term anoxia         compare gpd-2/3 transcript level daf-2 wild-type <br /> survival phenotype observed daf-2(e1370);gpd-2/3(RNAi)         animals. Briefly, animals placed pureZOL (Bio-Rad, <br /> was pronounced bacteria grown fol-           Hercules, CA), frozen liquid nitrogen, thawed at <br /> lows: Bacteria plated LB plates supplemented 200      37&#194;&#176;. This freeze/thaw process repeated times before <br /> mg/ml ampicillin (amp) 12.5 mg/ml tetracycline (tet).          RNA extracted using Aurum Fatty Fibrous Tissue <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> plate single colony grown 1 ml LB amp/       RNA extraction kit (Bio-Rad). RNA quality quantity were <br /> tet (200 mg/ml ampicillin, 12.5 mg/ml tetracycline) 6&#226;&#8364;&#8220;8 hr     determined using Experion (Bio-Rad). RT&#226;&#8364;&#8220;qPCR <br /> and 15 ml culture used inoculate 5 ml LB amp/    conducted using iScript One-step RT&#226;&#8364;&#8220;PCR kit (Bio- <br /> tet liquid, grown 4 hr seeding NGM&#226;&#8364;&#8220;      Rad) 10 ng total RNA loaded reac- <br /> IPTG agar plates. These NGM&#226;&#8364;&#8220;IPTG plates placed 37&#194;&#176;         tion. Reactions normalized nonvariable control <br /> for 24 hr transferred 15&#194;&#176; 20&#194;&#176; 24&#226;&#8364;&#8220;48 hr   gene, F23B2.13 (Link et al. 2003). The DCT reference <br /> L1 larvae placed plates. L1 larvae grown     gene method used calculate fold decrease increase in <br /> the RNAi food adulthood. The adult nematodes ex-           transcript (Real-time PCR Applications Guide, Bio-Rad). All <br /> posed anoxia assayed viability described above.      experiments independent experiments <br /> The F01F1.12(RNAi) F25H5.3(RNAi) results embryo             experiment performed triplicate. <br /> lethality larvae arrest, respectively. Therefore, these <br /> RNAi experiments, daf-2(e1370) animals grown the <br /> L4 stage control food 72 hr transferred ap- <br /> propriate RNAi food 48 hr, adults                                    RESULTS <br /> assayed high-temperature anoxia survival. As control, we <br /> grew E. coli strain, insert plasmid,       The daf-2(e1370) animal survives long-term anoxia <br /> identical NGM&#226;&#8364;&#8220;IPTG plates account differences           high-temperature anoxia: Wild-type adult hermaph- <br /> attributable vector.                                        rodites survive 1 day anoxia 20&#194;&#176;, survival rate <br />    Motility assays nematodes exposed anoxia: The wild-       dramatically decreases animals exposed to <br /> type, gpd-2/3(RNAi), daf-2(e1370), daf-2(e1370);gpd-2/3(RNAi), <br /> daf-16(m26)I, daf-16(mu86)I adult hermaphrodites were <br />                                                                    long-term anoxia ($3 days, 20&#194;&#176;) (Van Voorhies and <br /> used compare analyze nematode motion anoxia.             Ward 2000; Padilla et al. 2002). When considering the <br /> Nematode strains placed separate NGM plates         developmental stage C. elegans, dauer larvae have <br /> same anoxia BioBag chamber ensure samples         highest long-term anoxia (.3 days) survival rate <br /> treated equally. To assay animal motility anoxia ex-        (Anderson 1978; Padilla et al. 2002). Unlike wild-type <br /> posure, nematodes observed (through anoxia Bio- <br /> Bag) 4.5-min period 30-sec intervals hour for <br />                                                                    animals, daf-2(e1370) animals raised adulthood at <br /> the time indicated experiment. Motility assayed        20&#194;&#176; transferred high-temperature hypoxia <br /> and images obtained using Zeiss M2Bio stereoscope           (0.3% O2, 28&#194;&#176;) high survival rate (Scott et al. <br /> and Openlab software 3.17 imaging software (Improvision,       2002). To determine daf-2(e1370) wild-type ani- <br /> Lexington, MA). Nematodes maintained 20&#194;&#176;          mals different anoxia survival rates, strains <br /> at room temperature image collection, took <br />                                                                    exposed anoxia 20&#194;&#176;. Wild-type daf- <br /> ,20 min sampling. For wild type, gpd-2/3(RNAi), <br /> daf-2(e1370), daf-2(e1370);gpd-2/3(RNAi) comparisons,        2(e1370) animals collected L2 larvae gravid <br /> worm scored &#226;&#8364;&#732;&#226;&#8364;&#732;positive&#226;&#8364;&#8482;&#226;&#8364;&#8482; movement nematode          adults exposed 1, 3, 4, 5 days anoxia 20&#194;&#176;. <br /> displayed forward reverse motion agar surface.        Figure 1 shows wild-type L2 larvae adult ani- <br /> Data include independent experiments. For fur-      mals high survival rate exposed 1 day of <br /> ther analysis worm motility anoxia exposure, nema- <br />                                                                    anoxia; however, survival rate decreased ex- <br /> todes observed described images <br /> were collected 5-sec intervals 4.5-min period. The       posed 3 days anoxia animals did survive <br /> collected images imported QuickTime 7 Pro (Apple         4 5 days anoxia. In comparison, daf-2(e1370) <br /> Computer) visualization.                                       larvae adult animals survived long-term anoxia ex- <br />    Nomarski microscopy analysis: Nematodes exposed         posures significantly higher rate comparison to <br /> normoxic anoxic conditions allowed recover air        wild-type animals. The daf-2(e1370) adults survived <br /> for time indicated experiment. The animals were <br /> placed 2.5% agar pad containing 10 mm sodium azide        5 days anoxia better L2 larvae (P , 0.001). <br /> covered coverslip. Animals analyzed       Given DAF-16 transcription factor translo- <br /> the use sodium azide verify chemical did       cated nucleus daf-2(e1370) animal, we <br /> contribute factors analysis phenotype. For      wanted determine daf-2(e1370) long-term <br /> experiment independent evaluations done.       anoxia phenotype mediated DAF-16. The daf- <br /> Microscopy conducted using motorized Zeiss Axioscope <br /> with 3100 objective lens imaging Openlab software 3.17.      16(m26);daf-2(e1370) adults larvae animals exposed <br /> Images prepared using Adobe Photoshop CS (Adobe               5 days anoxia 20&#194;&#176; average viability of <br /> Systems).                                                          1.65 6 2.71% (n &#194;&#188; 353) 3.32 6 5.04 (n &#194;&#188; 458), <br />  1176                                       A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;The survival rate daf-2(e1370) wild-type <br /> animals exposed long-term anoxia 20&#194;&#176;. Wild-type L2 lar-        Figure 2.&#226;&#8364;&#8221;The survival rate daf-2(e1370), wild-type, and <br /> vae, wild-type adults, daf-2(e1370) L2 larvae, daf-2(e1370)    daf-16(m26); daf-2(e1370); animals exposed high-tempera- <br /> adults exposed 1, 3, 4, 5 days anoxia 20&#194;&#176;.       ture (28&#194;&#176;) anoxia. Wild-type L2 larvae, daf-2(e1370) L2 larvae, <br /> The data shown represent independent experi-        daf-16(m26); daf-2(e1370) L2 larvae, wild-type adults, daf- <br /> ments total 150 animals. Error bars represent   2(e1370) adults, daf-16(m26); daf-2(e1370) adults ex- <br /> the standard deviations.                                           posed 1 day anoxia 28&#194;&#176;. The data shown represent at <br />                                                                    independent experiments total .150 <br />                                                                    nematodes. Error bars represent standard deviations. <br /> respectively, suggesting DAF-16 mediates long- <br /> term anoxia survival phenotype. <br />    To determine increase temperature affected             wild-type daf-2(e1370) animals exposed to <br /> the daf-2(e1370) wild-type anoxia survival rate, L2            1 day anoxia 20&#194;&#176; significant tissue damage <br /> larvae adult animals exposed 1 day anoxia           comparison normoxic controls. However, few <br /> at 28&#194;&#176;. Figure 2 shows wild-type larvae adults        wild-type animals survived 3 days anoxia 20&#194;&#176; <br /> had reduced survival rate exposed anoxia              showed notable tissue damage (100%, n &#194;&#188; 6). Specifi- <br /> 28&#194;&#176;, indicating combined stress high tem-              cally, increase cavities tissue sur- <br /> perature anoxia detrimental wild-type animals.           rounding pharynx, loss pharynx structure, and <br /> In comparison, daf-2(e1370) larvae adults            overall bent morphology head region (Figure <br /> high survival rate exposed anoxia 28&#194;&#176;. These            3). In comparison, (77.8%, n &#194;&#188; 18) daf- <br /> data support finding wild-type animals            2(e1370) animals exposed 5 days anoxia 20&#194;&#176; did <br /> but daf-2(e1370) animals sensitive oxygen           accumulate tissue damage (Figure 3). To fur- <br /> deprivation (hypoxia anoxia) increased temper-              ther characterize morphological defects wild-type <br /> atures (Scott et al. 2002). To determine DAF-16                 animals survive 3 days anoxia, used strain <br /> mediates high-temperature anoxia survival, daf-                PD4792(mIs11 IV) GFP expressed cyto- <br /> 16(m26);daf-2(e1370) animals exposed 1 day              plasmic pharyngeal muscle nuclear gut (multi- <br /> anoxia 28&#194;&#176;. Figure 2 shows daf-16(m26);daf-            construct array containing myo-2TGFP, pes-10TGFP, <br /> 2(e1370) animals significantly decreased viability           gutTGFP). Due large size, evaluated gut <br /> when exposed high-temperature anoxia, indicating                cells PD4792 animals survived 3 days of <br /> that DAF-16 mediates high-temperature anoxia                   anoxia determined cells abnormal- <br /> survival phenotype.                                                ities included cavities cells cellular loss <br />    Wild-type animals daf-2(e1370) animals ac-          (data shown). Both wild-type daf-2(e1370) adult <br /> cumulate tissue damage long-term anoxia high-               animals survived normal pharynx structure <br /> temperature anoxia exposure: To assay anoxia-                  exposed 1 day normoxia 28&#194;&#176;. However, the <br /> induced tissue damage, wild-type daf-2(e1370)              majority (83%, n &#194;&#188; 6) wild-type animals that <br /> adult animals exposed prolonged anoxia                  survived anoxia 28&#194;&#176; showed extensive tissue damage, <br /> high-temperature anoxia examined using DIC                     including multiple cavities, abnormal pharynx struc- <br /> Nomarski microscopy. We focused analysis                ture, bent head, majority (83.3%, n &#194;&#188; <br /> tissue anterior head region nematode                 12) daf-2(e1370) animals did display such <br /> however entire animal evaluated. Figure 3 shows            tissue abnormalities (Figure 3). These data suggest that <br />                                                     Anoxia Responses C. elegans                                                1177 <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;Tissue morphology wild-type daf-2(e1370) animals exposed long-term anoxia (20&#194;&#176;) high-temperature <br /> anoxia (28&#194;&#176;). The anterior head region wild-type daf-2(e1370) adult hermaphrodites examined using DIC microscopy. <br /> (A) The animals exposed normoxia; 1 day, 3 days (wild type) 1 day, 5 days [daf-2(e1370)] anoxia 20&#194;&#176;; 1 day of <br /> normoxia anoxia 28&#194;&#176;. Animals exposed anoxia allowed recover air 1&#226;&#8364;&#8220;4 hr images obtained. <br /> Animals representative genotype condition shown. Bar, 50 mm. (B) Enlarged image wild-type adult hermaph- <br /> rodite exposed 3 days anoxia. (C) Enlarged image wild-type adult hermaphrodite exposed 1 day anoxia 28&#194;&#176;. In B <br /> and C, arrow points cavity open line drawn lumen. Bar, 50 mm. <br />  <br />  <br />  <br /> daf-2(e1370) animals altered physiologic state                (Van Voorhies Ward 2000; Padilla et al. 2002). <br /> that prevents minimizes tissue damage                   We hypothesized daf-2(e1370) animals an <br /> associated long-term anoxia high-temperature                  altered physiologic state mediates anoxia survival, <br /> anoxia exposure.                                                      animals additional phenotypes, as <br />    During examination morphological differ-                    altered motility exposed anoxia. We used time- <br /> ences wild-type daf-2(e1370) animals                   lapse microscopy compare motility wild-type <br /> was noted daf-2(e1370) animals appeared thin-                daf-2(e1370) animals exposed anoxia specific <br /> ner comparison wild type. Results indicate             time periods. Briefly, wild-type daf-2(e1370) adult <br /> distance terminal bulb pharynx              animals placed anoxia 20&#194;&#176; visualized <br /> wild-type daf-2(e1370) adult animals exposed nor-              using stereomicroscope 4.5 min hour for <br /> moxia, averaged 56.8 6 2.04 mm (n &#194;&#188; 10) 51.3 6                    10 hr 16 24 hr. An animal&#226;&#8364;&#8482;s motility <br /> 1.33 mm (n &#194;&#188; 10), respectively (P , 0.001). These data                scored positive animal displayed forward or <br /> suggest daf-2(e1370) animals slightly thinner            reverse motion agar surface. Results indicate <br /> in comparison wild-type adults. It known            wild-type daf-2(e1370) animals are <br /> anatomical distances daf-2(e1370) animals                 motile transition time (time 0) a <br /> wild-type animals role anoxia response                  normoxic anoxic environment (Figure 4; supple- <br /> survival.                                                             mental Figures 1 2 http://www.genetics.org/ <br />    The daf-2(e1370) animals motile exposed               supplemental/). However, motility wild-type <br /> anoxia 20&#194;&#176;: It previously shown wild-type                 adults decreased 2 hr anoxia exposure and <br /> animals immobile exposed 24 hr anoxia                 continued decrease time. Approximately 5% of <br /> at 20&#194;&#176;, hours reexposure nor-                 wild-type animals motile 8 hr anoxia <br /> moxia, animals proceeded movement                       exposure completely arrest 16 hr anoxia <br /> manner indistinguishable untreated nematodes                     exposure. In comparison, daf-2(e1370) adults <br />  1178                                     A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br />                                                                  standing role daf-2/daf-16 pathway plays <br />                                                                  anoxia survival, used RNAi screen through <br />                                                                  DAF-16 upregulated genes identify suppressors of <br />                                                                  daf-2(e1370) high-temperature anoxia survival phe- <br />                                                                  notype. This screen led identification the <br />                                                                  K10B3.8 clone gpd-2. <br />                                                                     In C. elegans, gpd-2 encodes predicted <br />                                                                  GAPDHs and, basis protein homology, pre- <br />                                                                  dicted reversibly catalyze following reaction in <br />                                                                  glycolysis: glyceraldehyde-3-phosphate 1 NAD1 1 Pi 4 <br />                                                                  1,3-bisphosphoglycerate 1 NADH 1 H1 GPD-2 ex- <br />                                                                  pressed highest levels postembryonic <br />                                                                  development, primarily actin-containing A I <br />                                                                  zones body-wall muscle (Lee et al. 1992). The gpd-2 <br />                                                                  gene operon including gpd-3 mai-1, <br />    Figure 4.&#226;&#8364;&#8221;The motility wild-type daf-2(e1370) ani-     which, respectively, encode GAPDH isoform <br /> mals exposed anoxia 20&#194;&#176;. Wild-type daf-2(e1370)        protein homology human mitochondrial <br /> adult hermaphrodites exposed 1 day anoxia          ATPase inhibitor (Huang et al. 1989; Blumenthal and <br /> 20&#194;&#176; time-lapse microscopy used analyze motility       Gleason 2003). Predicted protein analysis GPD-2 <br /> while exposed anoxia. Time 0 oxygen sensor <br /> (reazurin) anoxia chamber indicated oxygen <br />                                                                  GPD-3, using ClustalW (1.82) multiple alignment <br /> was detected environmental chamber. The data shown        software, indicates proteins 99% identical. <br /> represent independent experiments. A minimum 50 an-       Analysis predicted open reading frame sequence <br /> imals used experiment, range 10&#226;&#8364;&#8220;44    indicates 97% identity gpd-2 and <br /> animals microscope&#226;&#8364;&#8482;s field view mo-     gpd-3; therefore, using K10B3.8 clone specific <br /> tility assay. Error bars represent standard deviations. <br /> There significant difference (P , 0.001) wild- <br />                                                                  gpd-2 K10B3.7 clone specific gpd-3 will <br /> type daf-2(e1370) data points time 16 24.        likely cause reduction gpd-2 gpd-3 <br />                                                                  gene product. Throughout experiments <br />                                                                  used K10B3.8 clone; however, used the <br /> exposed anoxia display decrease motility,       K10B3.7 clone verify results. We refer RNAi <br /> the percentage worms remain motile signifi-           experiments gpd-2/3(RNAi). <br /> cantly higher comparison wild-type animals. A                 Figure 5A shows similar wild type unlike <br /> substantial number daf-2(e1370) adults         daf-2(e1370), daf-2(e1370);gpd-2/3(RNAi) adult ani- <br /> moving 8 hr anoxia exposure. After 24 hr             mals did survive 1 day high-temperature anoxia. <br /> anoxia exposure wild-type animals arrest              Use bacterial clone K10B3.7 specific gpd-3 resulted <br /> motility, daf-2(e1370) continue            similar phenotypes daf-2(e1370);gpd-2/ <br /> remain motile. Quick Time movies demonstrate             3(RNAi) animals did survive high-temperature <br /> the lack motility wild-type animals           anoxia (n &#194;&#188; 50). The daf-2(e1370);gpd-2/3(RNAi) ani- <br /> presence motility daf-2(e1370) animals ex-            mals detectable normoxic phenotype 28&#194;&#176;. The <br /> posed 24 hr anoxia (supplemental Figures 3 4           daf-2(e1370);mai-1(RNAi) animal did result a <br /> at http://www.genetics.org/supplemental/). The daf-              reduced high-temperature anoxia survival phenotype, <br /> 2(e1370) animals motile 24 hr anoxia ex-           indicating gene, gpd-2/3 operon, <br /> posure (supplemental Figure 4 http://www.genetics.            involved anoxia survival (data shown). <br /> org/supplemental/), appeared moving               Another gene regulated DAF-16, lys-7, encodes a <br /> slower comparison animals exposed            putative enzyme homologous antimicrobial lyso- <br /> to anoxia time (supplemental Figure 2 http://        zyme protozoan parasite Entamoeba histolytica. <br /> www.genetics.org/supplemental/).                                 The daf-2(e1370);lys-7(RNAi) animals survived high- <br />    The gpd-2/3 genes predicted encode glycolytic          temperature anoxia (91.3%, n &#194;&#188; 240). Together, these <br /> enzyme glyceraldehyde-3-phosphate dehydrogenase                  data suggest gpd-2 gpd-3 required daf- <br /> are required daf-2(e1370) animals&#226;&#8364;&#8482; anoxia                2(e1370) high-temperature anoxia survival phenotype <br /> survival phenotypes: The long-term anoxia high-              genes upregulated DAF-16 re- <br /> temperature anoxia survival phenotypes daf-2(e1370)           quired high-temperature anoxia survival. <br /> animals suppressed daf-16(m26);daf-2(e1370) ani-            Previously, wild-type animals exposed <br /> mals, providing evidence DAF-16 regulates phy-            1 day anoxia 20&#194;&#176; maintained high survival rate <br /> siologic state advantageous anoxia survival. The             resumed normal processes movement and <br /> genes regulated DAF-16 early adulthood                eating reexposure normoxia (Padilla et al. <br /> been identified using microarrays (Lee et al.          2002). Figure 5B shows gpd-2/3(RNAi) animals <br /> 2003; Murphy et al. 2003). To gain greater under-              exposed 24 hr anoxia 20&#194;&#176;, like wild-type animals, <br />                                                      Anoxia Responses C. elegans                                           1179 <br />  <br />                                                                      Furthermore, daf-2(e1370);gpd-2/3(RNAi) animals <br />                                                                      survived long-term anoxia,  16% uncoor- <br />                                                                      dinated phenotype did normally <br />                                                                      unable agar surface the <br />                                                                      anoxia treatment. Together data suggest gpd-2 <br />                                                                      gpd--3 functional role anoxia response and <br />                                                                      survival. <br />                                                                        The gpd-2/3(RNAi) daf-2(e1370);gpd-2/3(RNAi) <br />                                                                      animals accumulate tissue damage anoxia ex- <br />                                                                      posure: Given gpd-2/3(RNAi) animals display <br />                                                                      anoxia-induced uncoordinated phenotypes, wanted <br />                                                                      determine tissue damage occurred gpd-2/ <br />                                                                      3(RNAi) daf-2(e1370);gpd-2/3(RNAi) animals exposed <br />                                                                      anoxia. We used DIC Nomarski microscopy to <br />                                                                      evaluate animals anoxia treatment focused <br />                                                                      tissue evaluation anterior head region; how- <br />                                                                      ever, entire animal analyzed. Figure 6A <br />                                                                      shows gpd-2/3(RNAi) animals exposed nor- <br />                                                                      moxic conditions normal tissue morphology. How- <br />                                                                      ever, (77.8%, n &#194;&#188; 9) gpd-2/3(RNAi) animals <br />                                                                      abnormal pharynx structure 1 day anoxia <br />                                                                      exposure (Figure 6, A C). In comparison, wild- <br />                                                                      type animals, exposed anoxia 1 day 20&#194;&#176;, did not <br />                                                                      abnormal pharynx morphology (see Figure 3). <br />                                                                        Figure 6B shows daf-2(e1370);gpd-2/3(RNAi) animals <br />                                                                      exposed various anoxia treatments. The daf-2(e1370); <br />                                                                      gpd-2/3(RNAi) animals exposed 1 day anoxia 20&#194;&#176; <br />                                                                      displayed normal tissue structure. The daf-2(e1370);gpd- <br />                                                                      2/3(RNAi) animals survived 5 days anoxia and <br />                                                                      displayed uncoordinated phenotype tissue <br />                                                                      abnormalities (100%, n &#194;&#188; 10) (Figure 6, B E). Of <br />                                                                      daf-2(e1370);gpd-2/3(RNAi) animals survived <br />    Figure 5.&#226;&#8364;&#8221;The survival rate gpd-2/3(RNAi) daf-             anoxia 28&#194;&#176;, majority (85.7%, n &#194;&#188; 14) ex- <br /> 2(e1370);gpd-2/3(RNAi) adult animals exposed anoxia. (A)          tensive damage tissue (Figure 6, B D). The <br /> Wild-type, daf-2(e1370), daf-2(e1370);gpd-2/3(RNAi) adult        enlarged image anterior view representative <br /> hermaphrodites exposed 1 day normoxia anoxia           daf-2(e1370);gpd-2/3(RNAi) animal survived 1 day of <br /> at 28&#194;&#176;. (B) Wild-type, daf-2(e1370), gpd-2/3(RNAi), daf- <br /> 2(e1370);gpd-2/3(RNAi) adult hermaphrodites exposed             anoxia 28&#194;&#176; demonstrates accumulates large <br /> to 1 5 days anoxia 20&#194;&#176;. There significant difference   cavities pharynx tissue surrounding the <br /> between daf-2(e1370) daf-2(e1370);gpd-2/3(RNAi) adult her-       pharynx (Figure 6D). These findings support <br /> maphrodites exposed 5 days anoxia (P , 0.001). The data        idea gpd-2 gpd-3 role anoxia <br /> shown represent independent experiments         survival response. <br /> total  50 animals independent experiment. Error <br /> bars represent standard deviations.                                gpd-2 gpd-3 required motility brief <br />                                                                      periods anoxia: In study, wild- <br />                                                                      type daf-2(e1370) animals remained motile during <br /> also survived 1 day anoxia 20&#194;&#176; (94.6 6 4.1%; n &#194;&#188;               transition normoxia anoxia; however, by <br /> 146) (Figure 5B). However, unlike wild-type animals,                 24 hr anoxia exposure wild-type animals ar- <br /> 52 6 3.8% gpd-2/3(RNAi) animals analyzed 24 hr                rested motility significant percentage daf- <br /> after anoxia treatment uncoordinated pheno-                   2(e1370) animals displayed motility (see Figure 4 <br /> type did agar plate                  supplemental Figures 1&#226;&#8364;&#8220;4 http://www.genetics. <br /> normally displayed head movements. The daf-                  org/supplemental/). We wanted determine RNAi <br /> 2(e1370);gpd-2/3(RNAi) animals survived 1 day                gpd-2 gpd-3 influenced motility daf- <br /> anoxia 20&#194;&#176;, yet, unlike gpd-2/3(RNAi) animals, daf-           2(e1370) wild-type animals exposed anoxia. The ani- <br /> 2(e1370);gpd-2/3(RNAi) animals did un-                   mals placed anoxia 20&#194;&#176; visualized <br /> coordinated phenotype anoxia exposure. Figure                  4.5-min period hour 3 hr 10 <br /> 5B shows that, unlike daf-2(e1370), survival rate                24 hr. An animal&#226;&#8364;&#8482;s motility scored &#226;&#8364;&#732;&#226;&#8364;&#732;positive&#226;&#8364;&#8482;&#226;&#8364;&#8482; it <br /> of daf-2(e1370);gpd-2/3(RNAi) animals exposed long-               displayed forward reverse motion agar <br /> term anoxia reduced comparison daf-2(e1370).               surface. Figure 7 supplemental Figures 5 6 at <br />  1180                                     A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;The tissue morphology gpd-2/3(RNAi) daf-2(e1370);gpd-2/3(RNAi) adult animals exposed long-term <br /> anoxia high-temperature anoxia. The anterior head region gpd-2/3(RNAi) daf-2(e1370);gpd-2/3(RNAi) adult animals <br /> was examined using DIC microscopy. (A) gpd-2/3(RNAi) animals exposed normoxia 1 day anoxia 20&#194;&#176; allowed <br /> to recover air 4 24 hr (as indicated). (B) daf-2(e1370);gpd-2/3(RNAi) animals exposed normoxia (20&#194;&#176;), 1 day of <br /> anoxia (20&#194;&#176;), 5 days anoxia (20&#194;&#176;), 1 day normoxia (28&#194;&#176;), 1 day anoxia (28&#194;&#176;). Animals exposed anoxia allowed to <br /> recover air 4 24 hr anoxia (as indicated) analysis. (C) An enlarged image gpd-2/3(RNAi) animals exposed to <br /> 1 day anoxia 20&#194;&#176; shown. The open line drawn lumen arrow points region isthmus is <br /> abnormal. (D) An enlarged image daf-2(e1370);gpd-2/3(RNAi) animals exposed 1 day anoxia 28&#194;&#176;. (E) An enlarged image <br /> of daf-2(e1370);gpd-2/3(RNAi) animals exposed 5 days anoxia 20&#194;&#176; allowed recover normoxia 36 hr. For C, <br /> the solid arrow points bend pharynx. For D, solid arrow points cavities surrounding pharynx open <br /> arrow points cavities pharynx tissue. For C E, open line drawn lumen. Bar, 50 mm. <br />  <br />  <br />  <br /> http://www.genetics.org/supplemental/ show, respec-                 exposure anoxia (Figure 7B). The daf-2(e1370);gpd- <br /> tively, majority gpd-2/3(RNAi) daf-             2/3(RNAi) gpd-2/3(RNAi) animals exposed nor- <br /> 2(e1370);gpd-2/3(RNAi) adult animals immediately                    moxic conditions motile did display an <br /> arrested motility transition time (time 0)           uncoordinated phenotype (supplemental Figures 7 8 <br /> a normoxic environment anoxic environment.                    http://www.genetics.org/supplemental/). These find- <br /> The detectable movement observed ani-                ings suggest gpd-2/3 function required adult <br /> mals head movement. The gpd-2/3(RNAi) daf-                  animals remain motile initially exposed anoxia. <br /> 2(e1370);gpd-2/3(RNAi) L1 larvae animals remained mo-                  Given DAF-16 regulator gpd-2/3 expression <br /> tile (supplemental Figures 5 6 http://www.genetics.          (Lee et al. 2003; Murphy et al. 2003), wanted deter- <br /> org/supplemental/). The wild-type daf-2(e1370) an-              motility animals affected muta- <br /> imals fed control bacteria contain vector with-            tions daf-16. Figure 7A shows like wild type, and <br /> out gpd-2/3 insert remained motile hours            unlike gpd-2/3(RNAi), daf-16(m26) daf-16(mu86) <br /> after exposure anoxia, consistent              animals remained motile hours an- <br /> results shown Figure 4 supplemental Figures 1                oxia exposure. Figure 7B shows that, like daf-2(e1370), <br /> and 2 http://www.genetics.org/supplemental/             unlike daf-2(e1370);gpd-2/3(RNAi) animals, daf- <br /> the animals fed OP50 E. coli strain. Unlike daf-           16(m26);daf-2(e1370) remained motile few <br /> 2(e1370) animals, majority daf-2(e1370);gpd-2/               hours anoxia exposure. These data suggest a <br /> 3(RNAi) adult animals remained immobile                      factor addition DAF-16 regulates gpd-2/3 expression. <br />                                                    Anoxia Responses C. elegans                                          1181 <br />  <br />                                                                    ble reasons RNAi gpd-2 gpd-3 results an <br />                                                                    anoxia phenotype. For example, possible dis- <br />                                                                    ruption glycolysis leads reduction cellular ATP <br />                                                                    lack GPD-2/3 function leads re- <br />                                                                    duction 1,3-bisphosphoglycerate NADH a <br />                                                                    buildup glyceraldehyde-3-phosphate NAD1. To <br />                                                                    investigate possibilities, used RNAi to <br />                                                                    reduce expression gene products pre- <br />                                                                    dicted involved glycolysis. Glycolysis uses 10 <br />                                                                    enzymatic reactions convert glucose pyruvate, <br />                                                                    C. elegans genes, basis of <br />                                                                    homology, predicted involved the <br />                                                                    glycolytic pathway. We used RNAi reduce expres- <br />                                                                    sion specific predicted glycolytic genes daf- <br />                                                                    2(e1370) animal determine resulted reduced <br />                                                                    survival high-temperature anoxia similar seen <br />                                                                    daf-2(e1370);gpd-2/3(RNAi) animal. Results in- <br />                                                                    dicate that, unlike RNAi gpd-2/3, RNAi predicted <br />                                                                    glycolytic genes did result significantly reduced <br />                                                                    capacity daf-2(e1370) animals survive high- <br />                                                                    temperature anoxia (Table 1). <br />                                                                       The majority predicted glycolytic enzymes are <br />                                                                    encoded genes, possible that <br />                                                                    redundancy contribute lack detectable <br />                                                                    anoxia phenotype RNAi. To analyze role <br />                                                                    glycolysis play anoxia response survival, we <br />                                                                    focused phenotype analysis RNAi specific <br />                                                                    glycolytic genes contain paralog the <br />                                                                    genome. BLASTP analysis indicates E-value for <br />                                                                    proteins similar glucose phosphate isomerase, <br />                                                                    phosphoglycerate kinase, enolase 2.9, 2.6, 4.0, <br />    Figure 7.&#226;&#8364;&#8221;The motility gpd-2/3(RNAi) daf- <br /> 2(e1370);gpd-2/3(RNAi) animals exposed anoxia 20&#194;&#176;. <br />                                                                    respectively, indicating genes encoding these <br /> (A)Wild-type, gpd-2/3(RNAi), daf-16(mu86), daf-16(m26)         glycolytic proteins high homology another <br /> adult animals exposed 1 day anoxia 20&#194;&#176;. There        gene genome. Therefore, genes encode <br /> is significant difference motility gpd-2/3(RNAi)   glucose phosphate isomerase (gpi-1, Y87G2A.8), phos- <br /> wild-type animals exposed #3 hr anoxia (P , 0.001). (B)      phoglycerate kinase (T03F1.3, henceforth referred as <br /> daf-2(e1370), daf-2(e1370);gpd-2/3(RNAi), daf-16(m26);daf- <br /> 2(e1370) adult animals exposed 1 day anoxia 20&#194;&#176;. <br />                                                                    pgk-1), enolase (T21B10.2, henceforth referred as <br /> There significant difference motility daf-        enol-1) analyzed. We did include tpi-1 <br /> 2(e1370);gpd-2/3(RNAi) daf-2(e1370) animals exposed         (Y17G7B.7), does contain paralog, our <br /> #3 hr anoxia (P , 0.001) daf-2(e1370) daf-       analysis product enzyme, glyceralde- <br /> 16(m26);daf-2(e1370) adult animals exposed 24 hr anoxia      hyde-3-phosphate, produced TPI-1 aldolase. <br /> (P , 0.001). For A B, time-lapse microscopy used to <br /> assay animal motility anoxia. Time 0 anoxia sen- <br />                                                                       In glycolysis, GPI-1 functions upstream GPD-2 and <br /> sor anoxia bag indicates oxygen detected.        GPD-3, PGK-1 ENOL-1 function down- <br /> The data shown represent independent experi-        stream GPD-2 GPD-3. Further phenotype analysis <br /> ments minimum 50 animals experiment,         gpi-1(RNAi), pgk-1(RNAi), enol-1(RNAi) animals <br /> which range 6&#226;&#8364;&#8220;43 animals microscope&#226;&#8364;&#8482;s field     indicates animals anoxia- <br /> view motility assay time point. Error bars represent <br /> the standard deviations. <br />                                                                    induced motility arrest phenotype gpd-2/3(RNAi) <br />                                                                    animals (data shown). To verify efficiency <br />                                                                    RNAi experiments, used RT&#226;&#8364;&#8220;qPCR. Table 2 <br /> Furthermore, unlike daf-2(e1370), daf-16(m26);          shows relative reduction mRNA gpd-2, gpd-3, <br /> daf-2(e1370) animals arrested motility 24 hr          gpi-1, pgk-1, enol-1. Results indicate RNAi was <br /> anoxia exposure, indicating daf-2(e1370)             effective significantly reducing mRNA these <br /> animals ability remain motile 24 hr anoxia          specific genes, strain showed anoxia <br /> regulated daf-16.                                          phenotype gpd-2/3(RNAi) (Tables 1 2). These <br />    RNAi predicted glycolytic genes did result          data indicate wild-type levels gpi-1, pgk-1, and <br /> reduced capacity daf-2(e1370) animals sur-              enol-1 mRNA required anoxia response and <br /> vive high-temperature anoxia: There possi-             survival. <br />  1182                                      A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br />                                                                    TABLE 1 <br />                          High-temperature anoxia phenotype daf-2(e1370);glycolytic gene(RNAi) <br />  <br /> Metabolic enzyme                               % survival anoxia <br /> (sequence name, gene)                          28&#194;&#176; (% 6SD)                          Predicted enzymatic activity <br /> Hexokinase                                                            Glucose 1 ATP / glucose-6-phosphate 1 ADP <br />   F14B4.2                                        100.0 6 0 <br />   H25P06.1                                        99.0 6 1.4 <br />   Y77E11A.1                                       84.0 6 12.4 <br /> Glucose phosphate isomerase                                           Glucose-6-phosphate / fructose-6-phosphate <br />   Y87G2A.8, gpi-1                                100.0 6 0 <br /> Phosphofructokinase                                                   Fructose-6-phosphate / fructose-1,6-bisphosphate <br />   Y71H10A.1                                       89.9 6 11.3 <br />   C50F4.2                                         98.0 6 2.0 <br /> Fructose-1,6-bisphosphate aldolase                                    Fructose-1,6-bisphosphate / glyceraldehyde-3-phosphate, <br />   F01F1.12                                        99.1 6 1.5            dihydroxyacetone phosphate <br />   T05D4.1                                         93.6 6 5.5 <br /> Triose phosphate isomerase                                            Dihydroxyacetone phosphate / glyceraldehyde-3-phosphate <br />   Y17G7B.7, tpi-1                                 99.0 6 1.2 <br /> Glyceraldehyde-3-phosphate dehydrogenase                              Glyceraldehyde-3-phosphate 1 NAD 1 Pi / <br />   T09F3.3, gpd-1                                  95.3   6   5.1        1,3-bisphosphoglycerate 1 NADH <br />   K10B3.8, gpd-2                                  36.7   6   16.8* <br />   K10B3.7, gpd-3                                   6.0   6   6.0* <br />   F33H1.2, gpd-4                                  98.6   6   1.5 <br /> Phosphoglycerate kinase                                               1,3-Bisphosphoglycerate 1 ADP / 3-phosphoglycerate 1 ATP <br />   T03F1.3, pgk-1                                  100 6 0 <br /> Phosphoglycerate mutase                                               3-Phosphoglycerate / 2-phosphoglycerate <br />   F53B6.7                                         98.7   6   2.3 <br />   R07G3.5                                         95.5   6   5.3 <br />   F55A11.11                                       96.8   6   5.6 <br />   T07F12.1                                         100   6   0 <br />   F09C12.8                                         100   6   0 <br /> Enolase                                                               2-Phosphoglycerate / phosphoenolpyruvate 1 H2O <br />   T21B10.2, enol-1                                94.3 6 6.4 <br /> Pyruvate kinase                                                       Phosphoenolpyruvate 1 ADP / pyruvate 1 ATP <br />   F25H5.3                                         99.3 6 1.5 <br />   ZK593.1                                           98 6 1.6 <br /> Control vector                                    98.4 6 .9 <br />   The daf-2(e1370) animal fed specific bacterial clone inhibit gene expression interest. For data sets, in- <br /> dependent experiments 50 worms tested. * P , 0.001, determined ANOVA Student&#226;&#8364;&#8482;s t-test. <br />  <br />  <br />  <br />    Thus far, data suggest gpd-2 gpd-3                  Whitney U-test. The gpd-2/3 mRNA levels daf-2(m579) <br /> specifically required anoxia response survival.                 animals averaged 1.91-fold increase compared wild <br /> To test hypothesis increased rate anoxia                 type, range, independent experi- <br /> survival correlates increased level gpd-2/3                  ments, 0.98&#226;&#8364;&#8220;4.23, suggesting gpd-2/3 mRNA <br /> transcript, analyzed anoxia survival rate                level varies daf-2(m579) strain. In conclusion, our <br /> gpd-2/3 mRNA levels different daf-2 alleles. We               data indicate daf-2 allele highest rate of <br /> that daf-2(m579) animal survives high-temperature                   anoxia survival, daf-2(e1370), highest level gpd- <br /> anoxia long-term anoxia, albeit reduced rate                2/3 mRNA increase relative wild type. <br /> comparison daf-2(e1370), daf-2(e1371) daf- <br /> 2(m596) animals survive anoxia significant <br /> level (Table 3). We used RT&#226;&#8364;&#8220;qPCR compare levels <br />                                                                                               DISCUSSION <br /> of gpd-2/3 mRNA wild type daf-2(e1370), <br /> daf-2(m579), daf-2(e1371), daf-2(m596) animals. We                      The use animal model systems understand the <br /> determined daf-2(e1370) highest level                   response organisms oxygen deprivation is <br /> gpd-2/3 transcript increase relative wild type (Table                important understanding human health <br /> 3). In comparison, gpd-2/3 mRNA levels wild-type,                issues. The objective study use C. elegans <br /> daf-2(e1371), daf-2(m579), daf-2(m596)                     gain greater understanding oxygen-deprivation <br /> significantly different, determined Mann&#226;&#8364;&#8220;                     survival mechanisms. In study, that, unlike <br />                                                        Anoxia Responses C. elegans                                         1183 <br />  <br />                              TABLE 2                                    dependent DAF-16, indicating genetic <br />  Efficiency RNAi experiments determined RT&#226;&#8364;&#8220;qPCR                   pathway best known dauer formation longevity <br />                                                                         involved enhanced anoxia response and <br />                                     mRNA              Transcript        survival. <br /> Strain                              assayed         fold decreasea         Use RNAi screen genes upregulated DAF- <br />                                                                         16 daf-2(e1370) animal identified gpd-2 gpd-3 <br /> daf-2(e1370);gpd-2/3(RNAi)          gpd-2/3               9.67 <br /> daf-2(e1370);gpi-1(RNAi)            gpi-1                16.40          suppressors daf-2(e1370) high-temperature <br /> daf-2(e1370);enol-1(RNAi)           enol-1               12.91          anoxia survival phenotype. The gpd-2/3 genes encode <br /> daf-2(e1370);pgk-1(RNAi)            pgk-1                 7.91          highly conserved enzyme GAPDH, catalyzes <br />                                                                         oxidation phosphorylation glyceraldehyde- <br />    For RNAi experiments, level transcript signif- <br /> icantly reduced comparison control strain (P , 0.05,          3-phosphate 1,3-bisphosphoglycerate glycolysis <br /> as determined Mann&#226;&#8364;&#8220;Whitney U-test).                              (Huang et al. 1989; Lee et al. 1992). We determined by <br />    a <br />      Transcript level determined relative daf-2(e1370)       RNAi 17 glycolytic genes, including that <br /> control strain.                                                         encode phosphoglycerate kinase pyruvate kinase, <br />                                                                         catalyze reactions produce ATP, did not <br />                                                                         suppress daf-2(e1370) high-temperature anoxia sur- <br /> wild type, daf-2(e1370) animal survives does                vival phenotype. RNAi glycolytic genes not <br /> display tissue damage exposed long-term an-                     paralogs genome (gpi-1, enol-1, pgk-1) did not <br /> oxia high-temperature anoxia. The daf-2(e1370)                      result anoxia phenotypes. This suggests gpd- <br /> high-temperature anoxia survival phenotype sup-                      2/3 products specifically required anoxia <br /> pressed daf-16(m26) gpd-2/3(RNAi),                    survival phenotype observed daf-2(e1370) animals. <br /> RNAi glycolytic genes. Furthermore, gpd-2/                        We hypothesize increased expression gpd-2 <br /> 3(RNAi) animals sensitive anoxia exposure,               gpd-3 daf-2(e1370) animal role the <br /> seen arrest movement exposed brief per-                   ability survive anoxia. Theoretically, increase in <br /> iods anoxia, supporting idea glyceraldehyde-                gpd-2/3 expression result increased ratios of <br /> 3-phosphate dehydrogenase vital role anoxia                    1,3-bisphosphoglycerate glyceraldehyde-3-phosphate; <br /> response survival adult C. elegans.                              ATP ADP, and/or NADH NAD1. Perhaps an <br />    Genotypes anoxia survival phenotypes: Pheno-                     increase ATP/ADP ratios provides necessary en- <br /> type analysis daf-2(e1370) animal suggests                    ergy survival long-term anoxia high-temper- <br /> altered physiological metabolic state contributes               ature anoxia. Alternatively, consider is <br /> to long-term anoxia high-temperature anoxia                         increasing evidence accumulation glycer- <br /> survival. The factor(s) influencing anoxia survival                 aldehyde-3-phosphate glycate proteins, will <br /> or prevalent daf-2(e1370) adult                  lead deleterious effects cells (Hipkiss 2006). <br /> than daf-2(e1370) L2 larvae. Thus,                   Thus, interpretation daf-2(e1370) ani- <br /> the daf-2(e1370) adult higher survival rate 5 days             mals survive long-term anoxia high-temperature an- <br /> of anoxia comparison daf-2(e1370) larvae,                oxia that, comparison wild type, lower <br /> be fact larvae need                     concentration glyceraldehyde-3-phosphate available <br /> respond damage results anoxia,                glycate proteins lower level altered <br /> must reinitiate development return normoxia,                    proteins beneficial effect organism exposed <br /> whereas adults deal energy-                 anoxia. <br /> requiring process developmental progression. In                         Other daf-2 alleles analyzed indicate that, addition <br /> both daf-2(e1370) L2 larva adult, ability               daf-2(e1370), daf-2(m579) animal survives long- <br /> to survive high-temperature anoxia long-term anoxia                  term anoxia high-temperature anoxia, albeit at <br />  <br />  <br />  <br />                                                                  TABLE 3 <br />                                    Transcript level gpd-2/3 daf-2 alleles relative control <br />  <br />                                         High-temperature                        Long-term                     Transcript <br />          Strain                        anoxia survival rate                 anoxia survival rate            fold increasea <br />          daf-2(e1370)                95.2   6   3.3 (n &#194;&#188; 150)              91.6   6   9.4 (n &#194;&#188; 150)             3.36b <br />          daf-2(e1371)                 4.5   6   5.3 (n &#194;&#188; 200)               0.0   6   0.0 (n &#194;&#188; 200)             1.09 <br />          daf-2(m596)                  0.0   6   0.0 (n &#194;&#188; 200)               0.0   6   0.0 (n &#194;&#188; 200)             1.05 <br />          daf-2(m579)                 77.6   6   17.7 (n &#194;&#188; 300)             75.3   6   32.1 (n &#194;&#188; 300)            1.91 <br />             a <br />                 Transcript level determined relative wild-type control strain. <br />             b <br />                 P , 0.005, determined Mann&#226;&#8364;&#8220;Whitney U-test. <br />  1184                                   A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br /> a slightly reduced rate. However, daf-2(e1371)         detrimental tissue anoxia comparison daf- <br /> daf-2(m596) animals survive anoxic con-           2(e1370) animals. Third, possible reperfusion of <br /> ditions, demonstrating daf-2        oxygen animals did die anoxia <br /> mutants enhanced anoxia survival phenotype.           causes damage daf-2(e1370) animals are <br /> We RT&#226;&#8364;&#8220;qPCR daf-2(e1370) animals              better able wild-type animals respond type <br /> have increased level gpd-2/3 mRNA, comparison         damage. <br /> to wild type, indicating increased level gpd-2/3       The mechanism anoxia-induced tissue damage is <br /> mRNA correlates anoxia survival phenotypes.               understood time hypothesize gpd- <br /> The levels gpd-2/3 mRNA daf-2(m579) animals ap-          2/3 function involved. This idea supported the <br /> peared variable range similar wild type      fact gpd-2/3(RNAi) animals become <br /> to increased level comparison wild type,        uncoordinated accumulate tissue damage <br /> may explain large standard deviation observed          24 hr anoxia daf-2(e1370);gpd-2/ <br /> the survival rate animals exposed high-            3(RNAi) animal accumulates tissue damage expo- <br /> temperature anoxia.                                            sure long-term high-temperature anoxia. The dis- <br />    Others analyzed survival rate high-             ruption gpd-2/3 function result several <br /> temperature hypoxia various daf-2 alleles de-          metabolic changes. The question is, change(s) <br /> termined daf-2(e1370) daf-2(m579) animals             gpd-2/3(RNAi), daf-2(e1370);gpd-2/3, daf- <br /> survive high-temperature hypoxia, half         2(e1370) animals leads anoxia response observed <br /> of daf-2(m596) animals survive, majority      studies? <br /> of daf-2(e1371) animals survive (Scott et al.          Anoxia-induced motility arrest: We the <br /> 2002). The fact daf-2(m579) daf-2(m596) ani-          wild-type animal ability first <br /> mals higher survival rate exposed high-         hours anoxia exposure, longer anoxia ex- <br /> temperature hypoxia comparison high-temperature          posures ($10 hr) animal arrest motility. This <br /> anoxia long-term anoxia supports idea        response anoxia important adaptive mech- <br /> could different requirements surviving anoxia        anism allows animal move, when <br /> comparison hypoxia.                                         initially exposed stressful environment (anoxia), <br />    Anoxia-induced tissue damage: The wild-type animal          stressful environment (normoxia hypoxia); <br /> has overall normal tissue structure, moves normally,        animal unable stressful <br /> and able produce offspring exposure 1 day       environment, energy-requiring processes motil- <br /> of anoxia 20&#194;&#176;. Some gpd-2/3(RNAi) animals            ity arrested. We significant number of <br /> exposed 1 day anoxia tissue damage           daf-2(e1370) animals population did com- <br /> uncoordinated phenotype, suggesting gpd-2             pletely arrest motility exposed 24 hr <br /> gpd-3 role maintaining tissue structure          anoxia. This result supports idea daf- <br /> function exposed 1 day anoxia. Unlike           2(e1370) animal physiological metabolic state <br /> gpd-2/3(RNAi) animals, daf-2(e1370);gpd-2/3(RNAi)          maintain energy-requiring processes anoxia. <br /> animals exposed 1 day anoxia 20&#194;&#176; displayed nor-          The daf-2(e1370) prolonged motility anoxia phe- <br /> mal tissue structure. One interpretation results      notype suppressed daf-16(m26), these <br /> is GPD-2/3 gene product               animals responded anoxia similarly wild type. The <br /> involved tissue structure maintenance          daf-2(e1370);gpd-2/3(RNAi) animals somewhat <br /> daf-2(e1370) animal.                                           different phenotype completely arrested <br />    Of wild-type animals survive long-term         motility exposure anoxia. The gpd-2/3(RNAi) <br /> anoxia high-temperature anoxia, majority accu-          animals, daf-16(m26) daf-16(mu86) ani- <br /> mulates extensive tissue damage, including pharynx             mals, arrested motility initially exposed to <br /> abnormalities cavities tissue. In comparison,       anoxia. Together, data suggest prolonged <br /> the daf-2(e1370) animal enhanced ability main-       motility anoxia phenotype regulated daf-16 <br /> tain tissue structure function exposed long-       manner, motility brief exposures anoxia <br /> term high-temperature anoxia. One possible rea-            dependent gpd-2 gpd-3. Thus, likely that <br /> son phenotypic difference daf-             additional factor(s) mechanisms, than <br /> 2(e1370) wild-type animal daf-2(e1370)     daf-16, regulates gpd-2 gpd-3 motility in <br /> animal necessary physiological metabolic state      anoxia. <br /> to maintain cell tissue homeostasis (Hochachka                The loss motility gpd-2/3(RNAi) animals exposed <br /> 2000). For example, energy-requiring pro-          brief periods anoxia phenotype not <br /> cesses involved tissue/cell viability function    observed previously. This phenotype a <br /> maintained daf-2(e1370) animal longer            direct result decreased GAPDH activity. In wild-type <br /> periods anoxia exposure time exposed         adult C. elegans, correlation the <br /> stresses (high temperature anoxia). Second,     anoxia-induced arrest motility constant levels <br /> plausible wild-type animals accumulate products       protein-bound NADH increase free NADH <br />                                                Anoxia Responses C. elegans                                                 1185 <br />  <br /> (Paul et al. 2000). Thus,              To date, C. elegans genetic mutation <br /> determine compare levels free NADH              causes dramatic reduction anoxia viability in <br /> protein-bound NADH daf-2(e1370), wild-type, gpd-        adult hermaphrodites. However, RNAi experi- <br /> 2(RNAi), daf-2(e1370);gpd-2(RNAi) animals exposed          ments genetic mutations reduce viability of <br /> to anoxia. Such experiments allow de-              embryos exposed anoxia hypoxia [mdf- <br /> termine gpd-2/3(RNAi) animals reduced              2(RNAi) san-1(RNAi) (hif-1(ia04), respectively] <br /> levels NADH reduction GPD-2/3 activity.         (Jiang et al. 2001; Nystul et al. 2003). Thus, re- <br /> It interesting note that      sponse oxygen deprivation likely dependent <br /> sperm-specific glyceraldehyde-3-phosphate dehydroge-           genetic pathways, developmental stage of <br /> nase required sperm motility (Miki et al. 2004),        animal, oxygen tension. In report, found <br /> suggesting isoform role motility       reduction gpd-2/3 function RNAi not <br /> in different cellular types different environmental      reduce survival rate wild-type adult hermaphro- <br /> conditions.                                                    dites exposed 1 day anoxia; however, motility <br />    Glyceraldehyde-3-phosphate dehydrogenase an-            tissue structure affected animals, suggest- <br /> oxia: In mammals, glycolytic enzymes, including        ing gpd-2/3 function important the <br /> GAPDH, induced hypoxic conditions,             maintenance recovery tissues exposed anoxia. <br /> could allow maintenance ATP levels increasing            Studies conducted indicate GAPDH a <br /> anaerobic glycolysis (Duncan Storey 1992;                  role oxygen-deprivation responses vertebrates and <br /> Graven et al. 1994, 2003). Analyses gpd-1(RNAi)         studies presented suggest the <br /> and gpd-4(RNAi) animals indicate gpd-1 gpd-4          case C. elegans, providing evidence that <br /> are required adult respond to            conserved oxygen-deprivation response mech- <br /> survive anoxia shown genes          anisms vertebrates invertebrates. In con- <br /> are required embryo viability, suggesting alter-      clusion, suggest GAPDH expression serves a <br /> nate functional roles GPD-2/3 GPD-             protective role animals exposed oxygen depriva- <br /> 1/4 isoforms (Kamath et al. 2003). In study, did       tion. The ability study oxygen deprivation genetic <br /> not identify requirement putative glycolytic      model C. elegans lead iden- <br /> genes daf-2(e1370) high-temperature anoxia             tification greater understanding oxygen- <br /> phenotype. Furthermore, unlike gpd-2/3(RNAi) ani-              deprivation protective mechanisms. <br /> mals, gpi-1(RNAi), pgk-1(RNAi), enol-1(RNAi) ani-         We appreciate valuable input members Padilla <br /> mals did exhibit immediate arrest motility        Lab. We especially thank Kent Chapman, Pudur Jagadeeswaran, Vinita <br /> response anoxia. These data suggest       Hajeri, Desh Mohan valuable comments regarding the <br /> are compensatory systems place gly-         manuscript. We thank Caenorhabditis elegans Genetics Stock <br />                                                                Center, Donald Riddle, Leon Avery strains. This work was <br /> colytic enzymes, exception gpd-2 gpd-3, <br />                                                                supported grant National Institutes Health <br /> or mere flux glycolysis        National Institute General Medical Sciences (R01GM069419) and <br /> is required enhanced anoxia survival daf-           University North Texas internal research support. <br /> 2(e1370). It possible GPD-2 GPD-3 could <br /> have anoxia-induced cellular function related to <br /> glycolysis understood. For example, con-                               LITERATURE CITED <br /> sider mice brain-cell lines exposed hypoxia,        Anderson, G. L., 1978 Responses dauer larvae Caenorhabditis <br /> there increased expression localization             elegans (Nematoda: Rhabditidae) thermal stress oxygen <br />                                                                    deprivation. Can. J. Zool. 56: 1786&#226;&#8364;&#8220;1791. <br /> specific GAPDH nuclear fractions suggesting            Blumenthal, T., K. S. Gleason, 2003 Caenorhabditis elegans <br /> some GAPDH isoforms alternate                   operons: form function. Nat. Rev. Genet. 4: 112&#226;&#8364;&#8220;120. <br /> functional role addition glycolysis (Sirover 1999;       Burnell, A. M., K. Houthoofd, K. O&#226;&#8364;&#8482;Hanlon J. R. Vanfleteren, <br />                                                                    2005 Alternate metabolism dauer stage nema- <br /> Yamaji et al. 2003; Sirover 2005).                                 tode Caenorhabditis elegans. Exp. Gerontol. 40: 850&#226;&#8364;&#8220;856. <br />    Our RNAi analysis gpd-2 gpd-3 wild-type      Diaz, R. J., 2001 Overview hypoxia world. J. Environ. <br /> and daf-2(e1370) animals indicates gpd-2 gpd-3            Qual. 30: 275&#226;&#8364;&#8220;281. <br />                                                                Duncan, J. A., K. B. Storey, 1992 Subcellular enzyme bind- <br /> are required anoxia response survival               ing regulation glycolysis anoxic turtle brain. Am. <br /> adult nematode. The analysis daf-2 alleles indicates            J. Physiol. 262: R517&#226;&#8364;&#8220;R523. <br /> that gpd-2/3 mRNA level highest daf-2(e1370)        Epstein, A. C., J. M. Gleadle, L. A. McNeill, K. S. Hewitson, <br />                                                                    J. O&#226;&#8364;&#8482;Rourke et al., 2001 C. elegans EGL-9 mammalian <br /> animals comparison wildtype. These data suggest              homologs define family dioxygenases regulate HIF by <br /> that decreased levels gpd-2 gpd-3 correlate            prolyl hydroxylation. Cell 107: 43&#226;&#8364;&#8220;54. <br /> inadequate anoxia response survival           Foe, V. E., B. M. Alberts, 1985 Reversible chromosome con- <br />                                                                    densation induced Drosophila embryos anoxia: visualiza- <br /> increased level gpd-2/3 mRNA correlates anoxia             tion interphase nuclear organization. J. Cell Biol. 100: <br /> survival. Together, data provide additional evi-             1623&#226;&#8364;&#8220;1636. <br /> dence gpd-2 gpd-3 role daf-2(e1370)     Fuson, A. L., D. F. Cowan, S. B. Kanatous, L. K. Polasek R. W. <br />                                                                    Davis, 2003 Adaptations diving hypoxia heart, kidneys <br /> animal&#226;&#8364;&#8482;s ability survive long-term high-temperature          splanchnic organs harbor seals (Phoca vitulina). J. Exp. <br /> anoxia.                                                            Biol. 206: 4139&#226;&#8364;&#8220;4154. <br />  1186                                             A. R. Mendenhall, B. LaRue P. A. Padilla <br />  <br /> Gems, D., A. J. Sutton, M. L. Sundermeyer, P. S. Albert, K. V. King             glyceraldehyde-3-phosphate dehydrogenase-encoding genes <br />      et al., 1998 Two pleiotropic classes daf-2 mutation affect larval       Caenorhabditis briggsae. Gene 121: 227&#226;&#8364;&#8220;235. <br />      arrest, adult behavior, reproduction longevity Caenorhabdi-     Link, C. D., A. Taft, V. Kapulkin, K. Duke, S. Kim et al., 2003 Gene <br />      tis elegans. Genetics 150: 129&#226;&#8364;&#8220;155.                                        expression analysis transgenic Caenorhabditis elegans <br /> Ghabrial, A., S. Luschnig, M. M. Metzstein M. A. Krasnow,                   Alzheimer&#226;&#8364;&#8482;s disease model. Neurobiol. Aging 24: 397&#226;&#8364;&#8220;413. <br />      2003 Branching morphogenesis Drosophila tracheal sys-           Maxwell, P. H., C. W. Pugh P. J. Ratcliffe, 2001 The pVHL- <br />      tem. Annu. Rev. Cell Dev. Biol. 19: 623&#226;&#8364;&#8220;647.                               hIF-1 system. key mediator oxygen homeostasis. Adv. Exp. <br /> Graven, K. K., R. F. Troxler, H. Kornfeld, M. V. Panchenko                  Med. Biol. 502: 365&#226;&#8364;&#8220;376. <br />      H. W. Farber, 1994 Regulation endothelial cell glyceralde-          McElwee, J., K. Bubb J. H. Thomas, 2003 Transcriptional out- <br />      hyde-3-phosphate dehydrogenase expression hypoxia. J. Biol.             puts Caenorhabditis elegans forkhead protein DAF-16. <br />      Chem. 269: 24446&#226;&#8364;&#8220;24453.                                                    Aging Cell 2: 111&#226;&#8364;&#8220;121. <br /> Graven, K. K., D. Bellur, B. D. Klahn, S. L. Lowrey E. Amberger,        McElwee, J. J., E. Schuster, E. Blanc, J. H. Thomas D. Gems, <br />      2003 HIF-2alpha regulates glyceraldehyde-3-phosphate dehy-                 2004 Shared transcriptional signature Caenorhabditis <br />      drogenase expression endothelial cells. Biochim. Biophys. Acta          elegans Dauer larvae long-lived daf-2 mutants implicates <br />      1626: 10&#226;&#8364;&#8220;18.                                                               detoxification longevity assurance. J. Biol. Chem. <br /> Gray, J. M., D. S. Karow, H. Lu, A. J. Chang, J. S. Chang                       279: 44533&#226;&#8364;&#8220;44543. <br />      et al., 2004 Oxygen sensation social feeding mediated           Miki, K., W. Qu, E. H. Goulding, W. D. Willis, D. O. Bunch <br />      C. elegans guanylate cyclase homologue. Nature 430: 317&#226;&#8364;&#8220;322.             et al., 2004 Glyceraldehyde 3-phosphate dehydrogenase-S, a <br /> Haddad, G. G., 1998 Mechanisms anoxia tolerance: novel ap-                 sperm-specific glycolytic enzyme, required sperm motil- <br />      proach using Drosophila model system. Adv. Exp. Med. Biol.               ity male fertility. Proc. Natl. Acad. Sci. USA 101: 16501&#226;&#8364;&#8220; <br />      454: 273&#226;&#8364;&#8220;280.                                                              16506. <br /> Haddad, G. G., Y. Sun, R. J. Wyman T. Xu, 1997 Genetic basis         Morton, D. B., 2004 Atypical soluble guanylyl cyclases Drosoph- <br />      tolerance O2 deprivation Drosophila melanogaster. Proc.              ila function molecular oxygen sensors. J. Biol. Chem. 279: <br />      Natl. Acad. Sci. USA 94: 10809&#226;&#8364;&#8220;10812.                                      50651&#226;&#8364;&#8220;50653. <br /> Hajeri, V. A., J. Trejo P. A. Padilla, 2005 Characterization         Munoz, M. J., 2003 Longevity heat stress regulation Caeno- <br />      sub-nuclear changes Caenorhabditis elegans embryos exposed              rhabditis elegans. Mech. Ageing Dev. 124: 43&#226;&#8364;&#8220;48. <br />      brief, intermediate long-term anoxia analyze anoxia-         Murphy, C. T., S. A. McCarroll, C. I. Bargmann, A. Fraser, R. S. <br />      induced cell cycle arrest. BMC Cell Biol. 6: 47.                           Kamath et al., 2003 Genes act downstream DAF-16 to <br /> Hand, S. C., 1998 Quiescence Artemia franciscana embryos: revers-            influence lifespan Caenorhabditis elegans. Nature 424: <br />      ible arrest metabolism gene expression low oxygen lev-           277&#226;&#8364;&#8220;283. <br />      els. J. Exp. Biol. 201: 1233&#226;&#8364;&#8220;1242.                                     Nystul, T. G., J. P. Goldmark, P. A. Padilla M. B. Roth, <br /> Hermes-Lima, M., T. Zenteno-Savin, 2002 Animal response                  2003 Suspended animation C. elegans requires spindle <br />      drastic changes oxygen availability physiological oxidative         checkpoint. Science 302: 1038&#226;&#8364;&#8220;1041. <br />      stress. Comp. Biochem. Physiol. C Toxicol. Pharmacol. 133: 537&#226;&#8364;&#8220;        Ogg, S., S. Paradis, S. Gottlieb, G. I. Patterson, L. Lee et al., <br />      556.                                                                       1997 The Fork head transcription factor DAF-16 transduces in- <br /> Hipkiss, A. R., 2006 On mechanisms ageing suppression                 sulin-like metabolic longevity signals C. elegans. Nature <br />      dietary restriction: Is persistent glycolysis problem? Mech.           389: 994&#226;&#8364;&#8220;999. <br />      Ageing Dev. 127: 8&#226;&#8364;&#8220;15.                                                 Padilla, P. A., T. G. Nystul, R. A. Zager, A. C. Johnson M. B. <br /> Hochachka, P. W., 2000 Oxygen, homeostasis, metabolic reg-                  Roth, 2002 Dephosphorylation cell cycle-regulated proteins <br />      ulation. Adv. Exp. Med. Biol. 475: 311&#226;&#8364;&#8220;335.                                correlates anoxia-induced suspended animation Caeno- <br /> Hochachka, P. W., P. L. Lutz, T. Sick, M. Rosenthal G. E.                   rhabditis elegans. Mol. Biol. Cell 13: 1473&#226;&#8364;&#8220;1483. <br />      van den Thillart, 1993 Surviving Hypoxia Mechanisms Con-            Paul, R. J., J. Gohla, R. Foll H. Schneckenburger, <br />      trol Adaptation. CRC Press, Boca Raton, FL.                            2000 Metabolic adaptations environmental changes Cae- <br /> Hochachka, P. W., S. C. Land L. T. Buck, 1997 Oxygen sensing                norhabditis elegans. Comp. Biochem. Physiol. B Biochem. Mol. <br />      signal transduction metabolic defense hypoxia:              Biol. 127: 469&#226;&#8364;&#8220;479. <br />      lessons vertebrate facultative anaerobes. Comp. Biochem.          Riddle, D. L., P. S. Albert, 1997 Genetic environmental <br />      Physiol. A Physiol. 118: 23&#226;&#8364;&#8220;29.                                            regulation dauer larva development, pp. 739&#226;&#8364;&#8220;768 C. elegans <br /> Huang, X. Y., L. A. Barrios, P. Vonkhorporn, S. Honda, D. G.                    II, edited D. L. Riddle, T. Blumenthal, B. J. Meyer J. R. <br />      Albertson et al., 1989 Genomic organization glyceralde-             Priess. Cold Spring Harbor Laboratory Press, Plainview, NY. <br />      hyde-3-phosphate dehydrogenase gene family Caenorhabditis           Scott, B. A., M. S. Avidan C. M. Crowder, 2002 Regulation of <br />      elegans. J. Mol. Biol. 206: 411&#226;&#8364;&#8220;424.                                       hypoxic death C. elegans insulin/IGF receptor homo- <br /> Jiang, H., R. Guo J. A. Powell-Coffman, 2001 The Caenor-                    log DAF-2. Science 296: 2388&#226;&#8364;&#8220;2391. <br />      habditis elegans hif-1 gene encodes bHLH-PAS protein            Semenza, G. L., 2000 HIF-1 human disease: highly involved <br />      required adaptation hypoxia. Proc. Natl. Acad. Sci.              factor. Genes Dev. 14: 1983&#226;&#8364;&#8220;1991. <br />      USA 98: 7916&#226;&#8364;&#8220;7921.                                                     Semenza, G. L., 2001 Hif-1, o(2), 3 phds: animal cells <br /> Kamath, R. S., A. G. Fraser, Y. Dong, G. Poulin, R. Durbin et al.,              signal hypoxia nucleus. Cell 107: 1&#226;&#8364;&#8220;3. <br />      2003 Systematic functional analysis Caenorhabditis ele-         Shen, C., J. A. Powell-Coffman, 2003 Genetic analysis hyp- <br />      gans genome using RNAi. Nature 421: 231&#226;&#8364;&#8220;237.                               oxia signaling response C. elegans. Ann. NY Acad. Sci. <br /> Kenyon, C., J. Chang, E. Gensch, A. Rudner R. Tabtiang,                     995: 191&#226;&#8364;&#8220;199. <br />      1993 A C. elegans mutant lives twice long wild type.        Sirover, M. A., 1999 New insights old protein: func- <br />      Nature 366: 461&#226;&#8364;&#8220;464.                                                       tional diversity mammalian glyceraldehyde-3-phosphate dehy- <br /> Kimura, K. D., H. A. Tissenbaum, Y. Liu G. Ruvkun, 1997 daf-2,              drogenase. Biochim. Biophys. Acta 1432: 159&#226;&#8364;&#8220;184. <br />      insulin receptor-like gene regulates longevity dia-        Sirover, M. A., 2005 New nuclear functions glycolytic pro- <br />      pause Caenorhabditis elegans. Science 277: 942&#226;&#8364;&#8220;946.                     tein, glyceraldehyde-3-phosphate dehydrogenase, mammalian <br /> Lahiri, S., A. Roy, S. M. Baby, T. Hoshi, G. L. Semenza                         cells. J. Cell. Biochem. 95: 45&#226;&#8364;&#8220;52. <br />      et al., 2005 Oxygen sensing body. Prog. Biophys. Mol.           Sulston, J., J. Hodgkin, 1988 Methods, pp. 587&#226;&#8364;&#8220;606 The <br />      Biol. 91: 249&#226;&#8364;&#8220;286.                                                         Nematode Caenorhabditis elegans, edited W. Wood. Cold Spring <br /> Larsen, P. L., P. S. Albert D. L. Riddle, 1995 Genes reg-              Harbor Laboratory Press, Plainview, NY. <br />      ulate development longevity Caenorhabditis elegans.        Timmons, L., D. L. Court A. Fire, 2001 Ingestion bacterially <br />      Genetics 139: 1567&#226;&#8364;&#8220;1583.                                                   expressed dsRNAs produce specific potent genetic inter- <br /> Lee, S. S., S. Kennedy, A. C. Tolonen G. Ruvkun, 2003 DAF-16                ference Caenorhabditis elegans. Gene 263: 103&#226;&#8364;&#8220;112. <br />      target genes control C. elegans life-span metabolism.         Treinin, M., J. Shliar, H. Jiang, J. A. Powell-Coffman, Z. <br />      Science 300: 644&#226;&#8364;&#8220;647.                                                      Bromberg et al., 2003 HIF-1 required heat acclimation <br /> Lee, Y. H., X. Y. Huang, D. Hirsh, G. E. Fox R. M. Hecht,                   nematode Caenorhabditis elegans. Physiol. Genomics 14: <br />      1992 Conservation gene organization trans-splicing               17&#226;&#8364;&#8220;24. <br />                                                       Anoxia Responses C. elegans                                                 1187 <br />  <br /> Vanfleteren, J. R., A. De Vreese, 1996 Rate aerobic metab-     Wingrove, J. A., P. H. O&#226;&#8364;&#8482;Farrell, 1999 Nitric oxide contributes <br />     olism superoxide production rate potential nematode       behavioral, cellular, developmental responses low oxy- <br />     Caenorhabditis elegans. J. Exp. Zool. 274: 93&#226;&#8364;&#8220;100.                   gen Drosophila. Cell 98: 105&#226;&#8364;&#8220;114. <br /> Van Voorhies, W., 2003 The metabolic rate Caenorhabditis ele-      Yamaji, R., K. Fujita, S. Takahashi, H. Yoneda, K. Nagao et al., <br />     gans dauer larvae: comments recent paper Houthoofd           2003 Hypoxia up-regulates glyceraldehyde-3-phosphate dehydro- <br />     et al. Exp. Gerontol. 38: 343&#226;&#8364;&#8220;344.                                   genase mouse brain capillary endothelial cells: involvement of <br /> Van Voorhies, W. A., S. Ward, 2000 Broad oxygen tolerance         Na1/Ca21 exchanger. Biochim. Biophys. Acta 1593: 269&#226;&#8364;&#8220;276. <br />     nematode Caenorhabditis elegans. J. Exp. Biol. 203(Pt. 16): <br />     2467 <br /> </body></html>