<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization of the ptr61 Gene in Fission Yeast: A Possible Involvement of <br />  a Transcriptional Coactivator TAF in Nucleocytoplasmic Transport of mRNA <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                   Transport of mRNA <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the nucleus <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the cytoplasm is one of the important steps in gene expression <br />               in eukaryotic cells. To elucidate a mechanism of mRNA export, we identified a novel ptr [p oly(A)1 RNA <br />               tr ansport] <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>mutation</span>, ptr6, which causes accumulation of mRNA in the nucleus and inhibition of growth <br />               at the nonpermissive temperature. The ptr61 gene was found to encode an essential protein of 393 <br />               amino acids, which shares significant homology in amino acid sequence with yTAFII67 of budding yeast <br />               Saccharomyces cerevisiae and human hTAFII55, a subunit of the general transcription factor complex TFIID. <br />               A Ptr6p-GFP fusion protein is localized in the nucleus, suggesting that Ptr6p functions there. Northern <br />               blot analysis using probes for 10 distinct mRNAs showed that the amount of tbp1 mRNA encoding the <br />               TATA-binding protein is increased five- to sixfold, whereas amounts of others are rapidly decreased at <br />               the nonpermissive temperature in ptr6-1. ptr6 has no defects in nuclear import of an NLS-GFP fusion <br />               protein. These results suggest that Ptr6p required for mRNA transport is a Schizosaccharomyces pombe <br />               homologue of yTAFII67 and hTAFII55. This is the first report suggesting that a TAF is involved in the <br />               nucleocytoplasmic transport of mRNA in addition to the transcription of the protein-coding genes. <br />  <br />  <br />  <br /> I  N a eukaryotic cell, the nucleus and the cytoplasm <br />      are spatially separated by nuclear membrane. Thus, <br /> translocation of proteins and RNAs across the nuclear <br />                                                                             immunodeficiency virus type 1 (HIV-1) Rev protein <br />                                                                             (Fischer et al. 1994; Bogerd et al. 1995; Stutz et al. <br />                                                                             1995) and the hnRNP A1 protein (Michael et al. 1995). <br /> membrane is an essential process for cellular functions.                       HIV Rev protein recognizes a specific HIV RNA se- <br /> Although the process of mRNA export from the tran-                          quence, the Rev response element (RRE), and mediates <br /> scription site (nucleus) to the translation site (cyto-                     transport of the unspliced and partially spliced viral <br /> plasm) is an important step for gene expression, related                    mRNA from the nucleus to the cytoplasm. Rev protein <br /> molecular mechanisms are not well understood com-                           contains the short leucine-rich NES. By using the two- <br /> pared to the case of nuclear protein import.                                hybrid method, Rabp from humans (Bogerd et al. 1995) <br />    It was reported that transport of proteins and RNAs                      and Rip1p from yeast (Stutz et al. 1995) have been <br /> between the nucleus and the cytoplasm is a signal- and                      identified as factors interacting with the Rev NES. Re- <br /> receptor-mediated process (Go&#194;&#168;rlich and Mattaj                              cently, chromosomal region maintenance (Crm)1p has <br /> 1996). The nuclear localization signal (NLS) is found                       been suggested to function as an essential direct recep- <br /> in karyophilic proteins such as the SV40 large T antigen                    tor for the leucine-rich NES in eukaryotic cells (For- <br /> and nucleoplasmin. Import of these proteins into the                        nerod et al. 1997; Fukuda et al. 1997; Stade et al. 1997). <br /> nucleus is mediated by soluble and mobile receptors                         The crm11 gene was originally identified by complemen- <br /> through the recognition of the NLS. On the other hand,                      tation of a cold-sensitive S. pombe strain showing abnor- <br /> the nuclear export signals (NES) are present in several                     mal chromosomal structures at the nonpermissive tem- <br /> proteins that shuttle between the nucleus and the cyto-                     perature (Adachi and Yanagida 1989). Crm1p shares <br /> plasm (Gerace 1995). Some of these NES-containing                           significant homology with importin-b in amino acid se- <br /> proteins are RNA-binding proteins and could be in-                          quence. It shuttles between the nucleus and the cyto- <br /> volved in the nuclear export of their cognate cargo                         plasm and mediates nuclear export of proteins con- <br /> RNA. This was most convincingly shown for the human                         taining NES by binding directly to the NES signal <br />                                                                             peptide. This process is dependent on the RanGTPase <br />                                                                             system. <br />   Corresponding author: Tokio Tani, Department of Biology, Faculty             On the other hand, hnRNP A1 shuttles rapidly be- <br /> of Science, Kyushu University, Hakozaki, Higashi-ku, Fukuoka 812-           tween the nucleus and the cytoplasm and is associated <br /> 8581, Japan. E-mail: ttaniscb@mbox.nc.kyushu-u.ac.jp <br />   1 <br />                                                                             with poly(A)1 RNA in both compartments (Pin           &#203;&#339; ol- <br />    Present address: Department of Biochemistry and Molecular Biology, <br /> Pennsylvania State University College of Medicine, Hershey, PA 17033.       Roma and Dreyfuss 1991, 1992; Michael et al. 1995). <br />   2 <br />    Present address: CREST, JST, Division of Biology, Graduate School        The NES of hnRNP A1, called M9, is larger than those <br /> of Science, Nagoya University, Chikusa-ku, Nagoya 464-8602, Japan.          of HIV Rev, protein kinase inhibitor (PKI), and tran- <br />  <br /> Genetics 152: 869&#226;&#8364;&#8220;880 ( July 1999) <br />  870                                                     T. Shibuya et al. <br />  <br /> scription factor IIIA (TFIIIA) and bears no sequence               (Murphy and Wente 1996; Murphy et al. 1996). Gle1p <br /> homology with the previously identified NES (for a re-             contains a sequence similar to the Rev NES and interacts <br /> view, see Go&#194;&#168;rlich and Mattaj 1996). Interestingly, the            with Rip1p and the nucleoporin Nup100p. It is localized <br /> M9 sequence also exhibits NLS activity. The nuclear                predominantly at the nuclear pore complexes. On the <br /> import of M9-containing proteins does not involve use              other hand, Gle2p encodes a 40.5-kD polypeptide with <br /> of the importin-mediated pathway. Recently, a 90-kD                a striking similarity to S. pombe Rae1p. In addition, <br /> protein, called transportin, was identified as the nuclear         Mex67p and Mtr10p were identified by a synthetic lethal <br /> import receptor for M9-bearing proteins (Pollard et                screen with a <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>null allele</span> of NUP85 (Segref et al. 1997; <br /> al. 1996). From a kinetic competition experiment using             Senger et al. 1998). Mex67p is a nuclear pore-associated <br /> injection of hnRNP A1 into Xenopus laevis oocytes,                 protein that contains an NES, binds to poly(A)1 RNA, <br /> hnRNP A1, or other proteins containing an M9 se-                   and is likely to participate directly in export of mRNA. <br /> quence, was considered to play a role in mRNA export               Mtr10p functions as a nuclear import receptor for the <br /> from the nucleus (Izaurralde et al. 1997).                         mRNA-binding protein Npl3p/Mtr13p (Senger et al. <br />    Genetic approaches have been used to identify factors           1998). <br /> involved in nucleocytoplasmic transport of mRNA in                    In attempts to elucidate the mechanism of mRNA <br /> the yeasts Saccharomyces cerevisiae and Schizosaccharomyces        transport from the nucleus to the cytoplasm, we isolated <br /> pombe. In genetic screens using in situ hybridization with         a novel temperature-sensitive mutant (ptr6) defective in <br /> an oligo(dT) probe, temperature-sensitive mutants that             mRNA transport in S. pombe, using in situ hybridization. <br /> accumulated poly(A)1 RNA in the nucleus at the non-                We cloned and characterized the gene complementing <br /> permissive temperature were isolated and analyzed. In              ptr6. The ptr61 gene was found to encode a putative <br /> S. cerevisiae, 16 mRNA transport (mtr) and 10 ribonucleic          homologue of TBP-associated factor (TAF), a subunit <br /> acid trafficking (rat) mutants defective in mRNA export            of the transcription factor complex TFIID known as a <br /> were identified (Amberg et al. 1992; Kadowaki et al.               transcriptional coactivator. <br /> 1994). Among the genes that were mutated in these <br /> mutants, MTR1/PRP20 was shown to be a homologue <br />                                                                                  MATERIALS AND METHODS <br /> of RCC1 in mammalian cells and pim11 in S. pombe <br /> (Kadowaki et al. 1992, 1993). RCC1 protein functions                 Yeast strains, media, and genetic methods: The yeast strains <br /> as a guanine nucleotide exchange factor for the Ras-               used in this study are listed in Table 1. The complete media <br /> like guanosine triphosphatase (GTPase) Ran/TC4 in                  YPD or YE (Gutz et al. 1974; Sherman et al. 1986) and mini- <br />                                                                    mum medium (MM; Moreno et al. 1991) were used for stan- <br /> mammalian cells (Bischoff and Ponstingl 1991) and <br />                                                                    dard cultures of S. pombe. The general genetic method used <br /> is required for nuclear RNA export (Cheng et al. 1995).            for S. pombe was as described (Gutz et al. 1974). <br /> MTR13/NPL3 encodes an RNA-binding protein that                       Isolation of a ptr mutant: The wild-type strain 972 was muta- <br /> shuttles in and out of the nucleus as hnRNP A1 and is              genized with N-methyl-N 9-nitro-N-nitrosoguanidine to gener- <br /> suggested to mediate mRNA export (Lee et al. 1996).                ate a bank of the ts2 mutants that can grow at 268 but not at <br />                                                                    378, as described (Urushiyama et al. 1996). We screened a <br /> RAT1 encodes an essential 116-kD protein with regions <br />                                                                    collection of 100 temperature-sensitive mutants by fluorescent <br /> of homology to the protein encoded by SEP1, which is               in situ hybridization (FISH) with a biotin-labeled oligo(dT)50 <br /> a nucleic acid-binding protein, a 59&#226;&#8224;&#8217;39 exonuclease,               probe (Tani et al. 1995), and identified one that causes accu- <br /> and catalyzes DNA strand transfer reactions in vitro               mulation of mRNA in the nuclei at the nonpermissive temper- <br /> (Amberg et al. 1992). In addition, the RAT7/NUP159                 ature. <br />                                                                      Cloning and sequencing of the ptr61 gene: The ptr61 gene <br /> gene encodes an essential nucleoporin containing 22 <br />                                                                    was cloned by complementation of a temperature-sensitive <br /> XXFG and 3 XFXFG degenerate repeats (Gorsch et al.                 growth defect of ptr6-1 with an S. pombe genomic library con- <br /> 1995). In S. pombe, four poly(A)1 RNA transport (ptr) <br /> and one ribonucleic acid export (rae) mutants involved <br /> in mRNA export were identified. Wild-type genes for                                          TABLE 1 <br /> three, ptr21, ptr31, and rae11, have been cloned to date.                                   Yeast strains <br /> Of those, the ptr21 gene was found to be identical to <br /> the pim11 gene, encoding a homologue of RCC1 (Azad                 Strains                Genotype                    Source <br /> et al. 1997). The ptr31 gene encodes an S. pombe homo- <br />                                                                    975            h1                              M. Yanagida <br /> logue of the ubiquitin-activating enzyme E1 in S. cerevis- <br />                                                                    972            h2                              M. Yanagida <br /> iae (Azad et al. 1997). The Ptr3p essential for cell viability     SU59-1D        h2 leu1-32                      S. Urushiyama <br /> is localized in both the nucleus and the cytoplasm. The            SB14           h2 leu1-32 ptr6-1               This study <br /> rae11 gene encodes a protein of 352 amino acids with               ST1            h2 ura4-D18 ptr6-1              This study <br /> four b-transducin/WD40 repeats (Brown et al. 1995).                S22            h2 leu1-32 ptr2-1               A. Azad <br />    In genetic screens based on synthetic lethality, several        UR443          h1 ura4-D18                     S. Urushiyama <br /> factors involved in mRNA export were identified in S.              UDP6           h1/h2 leu1-32/leu1-32           S. Urushiyama <br />                                                                                     ura4-D18/ura4-D18 <br /> cerevisiae. Two gle mutations that were lethal in combina- <br />                                                                                     ade6-M216/ade6-M210 <br /> tion with a null allele of the gene encoding the nucleo-           CDC10          h2 leu1-32 cdc10-129            P. Nurse <br /> porin Nup100p were isolated in a colony-sectoring assay <br />                                                 Characterization of the ptr61 Gene                                              871 <br />  <br /> structed in pSS10 (Nakaseko et al. 1986). Transformation of       pombe. To construct a gene expressing a reporter protein, the <br /> ptr6-1 was done using the lithium acetate method (Okazaki         double-stranded oligonucleotide encoding the nucleoplasmin <br /> et al. 1990) and 16 ts1 transformants were obtained after incu-   bipartite basic NLS, KRPAAIKKAGQAKKKK (Dingwall and <br /> bation at 378. Cosmid DNAs isolated from ts1 transformants        Laskey 1991), was inserted in frame into the 59-end of the <br /> were then subjected to restriction mapping analysis and were      GFP gene in pEGFP-N1 (Clontech). The fragment encoding <br /> found to contain the same insert DNA. The recovered cosmid        NLS-GFP was then isolated and inserted downstream of the <br /> was retested for complementing ability for ts <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>phenotype</span> of        nmt1 promoter in pREP3 (Maundrell 1993). This plasmid <br /> ptr6-1. After several steps of subcloning into the pSP1 vector    was introduced into the wild-type strain SU59-1D, the ptr6-1 <br /> (Cottarel et al. 1993), the smallest fragment with comple-        strain, or ptr2-1 strain that has a defect in nuclear protein <br /> menting activity was identified. DNA sequencing was done          import (Azad et al. 1997) as a control. For protein import <br /> using an automatic DNA sequencer (ABI PRISM 377 DNA               assay, transformants were incubated at 268 and grown to early <br /> sequencer; Perkin-Elmer, Foster City, CA).                        log phase in MM without thiamine, and then each culture <br />    To isolate ptr61 cDNA, total RNA prepared from the wild-       was preincubated at 378 for 1 hr. After centrifugation at room <br /> type strain as described (Urushiyama et al. 1996) was sub-        temperature in a microcentrifuge, the cell pellets were washed <br /> jected to reverse transcription with a primer 59-CGC AGC GTA      with dH2O and suspended in 10 mm sodium-azide and 10 <br /> CAA ATG ACC AT-39. The cDNA was then amplified by PCR             mm 2-deoxy-d-glucose in glucose-free MM. The mixture was <br /> using the above primer and primer 59-CGT GCT GTT CAG              incubated for 1 hr at 378 to allow for equilibration of the NLS- <br /> CCT CCT CC-39. The PCR products were cloned into the              GFP fusion protein between the nucleus and the cytoplasm. <br /> pGEM-T vector (Promega, Madison, WI) and sequenced.               The cells were pelleted again and suspended in MM pre- <br />    To identify the mutation site in the ptr6-1 gene, we carried   warmed at 378. The suspension was then incubated for 30 min <br /> out a gap repair analysis (Guthrie and Fink 1991). Several        at 378. The import was initiated at time 0 by suspending cells <br /> linear pSP1 plasmids containing the gap within the ptr61 gene     in MM. Localization of the NLS-GFP fusion proteins was exam- <br /> were prepared and introduced into the ptr6-1 mutant. The          ined using a Zeiss AxioplanII fluorescence microscope. <br /> gapped region was repaired using a chromosomal sequence              RNA preparation and Northern blot analysis: Total RNA <br /> as a template. Replicative plasmids that complement growth        from S. pombe was prepared by the glass bead method as de- <br /> at 268 but not at 378 were isolated from the transformants.       scribed (Urushiyama et al. 1996). RNA was quantified by <br /> The sequence corresponding to the gapped region in the            absorbance at 260 nm and visualization on an agarose gel <br /> isolated plasmid was then determined.                             stained with ethidium bromide. For Northern blot analysis, <br />    Integration mapping: To determine if the cloned gene is a      20 mg of each total RNA was subjected to electrophoresis in <br /> multicopy suppressor for ptr6, target integration mapping was     a formaldehyde/agarose gel, followed by blotting onto a nylon <br /> done as described (Guthrie and Fink 1991). The 4.2-kb NcoI-       membrane (Gene Screen; New England Nuclear, Boston). <br /> BglII fragment containing the ptr61 gene was cloned into          The RNA was fixed to the membrane by UV crosslinking. The <br /> multicloning sites of pSKur4, which contains the ura41 marker     membranes were prehybridized in 63 SSC, 50 mm sodium <br /> in the HindIII site of the Bluescript SK1 vector. The resulting   phosphate, pH 7.5, 2 mm EDTA, 0.1% SDS, 53 Denhardt&#226;&#8364;&#8482;s <br /> plasmid was linearized at the NdeI site within the ptr61 gene     solution, and 0.2 mg/ml boiled salmon sperm DNA at 428 for <br /> and introduced into wild-type strain UR443. Stable Ura41          several hours. Hybridization was performed overnight in the <br /> transformants were isolated and the correct integration of the    same solution with 32P-labeled oligonucleotide probes at 428. <br /> plasmid into the ptr61 locus was verified by PCR, using primers   The membranes were washed three times in 63 SSC at room <br /> corresponding to the ura41 gene and the flanking sequence         temperature, then soaked in a solution containing 63 SSC <br /> of ptr61 outside of the used fragment. The obtained strain        and 0.1% SDS at 508 for 5 min. The filters were exposed to <br /> was mated with ST1 (h2, ura4, ptr6-1), and tetrads analysis was   a Fuji imaging plate, and a Bioimaging Analyzer BAS1000 <br /> performed.                                                        (Fuji Photo Film Co., Kanagawa, Japan) was used for analysis. <br />    Disruption of the ptr61 gene: To disrupt the ptr61 gene, the      Sequences of oligonucleotides used as probes are listed in <br /> NruI-NruI fragment in the ptr61 open reading frame (ORF)          Table 2. To determine if the ptr6 mutant was defective in pre- <br /> was replaced with the S. pombe ura41 gene. The HindIII-BglII      mRNA splicing, a probe complementary to the third exon <br /> fragment containing the ptr6::ura4 construct was isolated and     of TATA-binding protein (TBP) pre-mRNA was used. A 32P- <br /> introduced into the wild-type diploid strain UDP6. Stable         labeled oligo(dT)30 probe was used to quantify the amount of <br /> Ura41 transformants were then isolated. The correct replace-      the total poly(A)1 RNA. <br /> ment of the disrupted gene with the ptr61 gene by homologous         Fluorescence-activated cell sorting (FACS) analysis: FACS <br /> recombination was verified by PCR using primers correspond-       analysis was performed as described (Alfa et al. 1993). A total <br /> ing to the ura41 gene and to the flanking sequence of the         of 2 3 106 exponentially growing cells cultured at 268 or shifted <br /> ptr61 gene outside of the fragment used for transformation.       to 378 for 2 or 4 hr were fixed in 70% ethanol at 48 overnight. <br /> The heterozygous diploids were sporulated at 268, and tetrads     After washing with 1 ml of 50 mm sodium citrate, the cells <br />                                                                   were resuspended in 1 ml of a solution containing 50 mm <br /> were dissected. Colonies grown at 268 were then replica plated <br />                                                                   sodium citrate, 0.1 mg/ml RNase A, and 2 mg/ml propidium <br /> to MM plates to examine for auxotrophy. <br />                                                                   iodide and were subjected to an analysis with a Becton-Dickin- <br />    Analysis of localization of Ptr6p-GFP: To construct a gene <br />                                                                   son (San Jose, CA) FACScan. The cdc10-129 strain that arrests <br /> expressing a Ptr6p-GFP fusion protein, the GFP-encoding frag- <br />                                                                   in the G1 phase at 378 (Nurse et al. 1976) was used as a <br /> ment isolated from pEGFP-N1 (Clontech, Palo Alto, CA) was <br />                                                                   control. <br /> inserted in frame into a BstEII site present in a 39-end region <br />                                                                      Nucleotide sequence accession number: The nucleotide se- <br /> of the ptr61 gene. The resulting plasmid was then introduced <br />                                                                   quence reported here has been submitted to the DDBJ data <br /> into the ptr6-1 strain. The Ptr6p-GFP fusion protein rescued <br />                                                                   <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span>. The assigned accession no. is AB016928. <br /> the growth defect of the ptr6-1 strain at the nonpermissive <br /> temperature (378). After staining with 4,6-diamidino-2-phenyl- <br /> indole (DAPI), in vivo localization of the GFP fusion protein <br /> was examined using a Zeiss (Thornwood, NY) Axioplan fluo-                                    RESULTS <br /> rescence microscope. <br />    Analysis of nuclear protein import: To analyze nuclear pro-      Isolation of a novel mutant defective in mRNA trans- <br /> tein import, we used the method of Shulga et al. (1996)           port: From a bank of ts2 mutants generated by ethyl- <br /> developed in S. cerevisiae with modifications for usage in S.     methane sulfonate, we initially isolated four mRNA <br />  872                                                   T. Shibuya et al. <br />  <br />                                                          TABLE 2 <br />                                  Oligonucleotides used as probes in Northern blot analysis <br />  <br />         TBP        59-   GAG      CTT     GGA      GTC        ATC     CTC      GG       -39 <br />         PTR6       59-   GTC      GTC     CCG      ATC        GAT     GCT      GGT      ACC       ACG      AAG       -39 <br />         ACT1       59-   CAG      AGT     CCA      AGA        CGA     TAC      CAG      TGG       TAC      GAC       -39 <br />         RNA1       59-   TCT      GAC     AAC      CAT        CTA     GCA      GCT      TCG       GTT      CCA       -39 <br />         RAE1       59-   TGA      CCC     AGT      TGC        CAG     AAT      AGG      TGA       AGT      ACC       -39 <br />         PIM1       59-   GAA      GTC     GCA      TCT        GCG     TCG      GTT      CAG       CTG      TAC       -39 <br />         TAF72      59-   GAT      CAC     AGC      TCC        AAA     GTT      GTG      CAG       TCT      GGT       -39 <br />         PRP2       59-   CGA      GCC     TGT      CTG        CTA     GCA      CCT      GGT       TGT      AAG       -39 <br />         CDC2       59-   CAC      GAA     TAG      CTG        TGC     TAG      GAA      CTC       CCT      CAG       -39 <br />         RPB1       59-   GCG      TTG     TCC      ACT        TTC     ATC      CAT      GGT       CTC      AGG       -39 <br />  <br />  <br />  <br /> transport mutants (ptr1&#226;&#8364;&#8220;4; Azad et al. 1997) and later <br /> another additional mutant named ptr5 (T. Akimatsu, <br /> T. Tani and Y. Ohshima, unpublished results) by screen- <br /> ing by in situ hybridization. To identify novel mRNA <br /> transport mutants, we screened a collection of ts2 mu- <br /> tants obtained by mutagenesis with N-methyl-N9-nitro- <br /> N-nitrosoguanidine (Urushiyama et al. 1996). Of the <br /> 100 mutants screened, we identified 1 ts2 mutant <br /> (ts376) that accumulates poly(A)1 RNA in the nuclei at <br /> the nonpermissive temperature (Figure 1). This mutant <br /> showed nuclear signals in .90% of the cells 4 hr after <br /> shifting to the nonpermissive temperature. The intensi- <br /> ties of the nuclear signals were weaker than those of <br /> the previously identified ptr mutants. <br />    This mutant was then backcrossed three times with a <br /> wild-type strain to remove extra mutations. Tetrad analy- <br /> sis showed 2:2 segregation of the temperature-sensitive <br /> and wild-type phenotypes in all cases, which means that <br /> the ts2 phenotype is due to a single mutation. Cosegre- <br /> gation of the mRNA export defect with the ts2 pheno- <br /> type was also observed, suggesting that the ts2 growth <br /> phenotype is linked with a mutation responsible for <br /> blocking mRNA transport. This mutation is recessive, <br /> as the heterozygous diploid with the wild-type alleles <br /> grew well at the nonpermissive temperature. <br />    To determine if the isolated mutant is a novel mRNA <br /> transport mutant, complementation analyses were done <br /> with the previously identified mRNA transport mutants, <br /> ptr1&#226;&#8364;&#8220;5. Heterozygous diploids in all combinations grew <br /> at 378, thereby demonstrating that the isolated mutant <br /> belongs to a new complementation group that we <br /> named ptr6. <br />    Growth characteristics of ptr6: To examine the <br /> growth characteristics of ptr6, the growth rates and via- <br /> bilities of the wild-type strain and the ptr6-1 mutant were        Figure 1.&#226;&#8364;&#8221;Accumulation of poly(A)1 RNA in the nucleus <br /> compared over a 12-hr period (Figure 2). The growth              in ptr6 after a shift to the nonpermissive temperature. Wild- <br /> rate and viability of the ptr6-1 mutant were almost the          type and ptr6-1 cells were grown to the midlog phase at 268 <br /> same as those of the wild-type strain at the permissive          and shifted to 378 for 4 hr. The cells were fixed and analyzed <br /> temperature (268). In contrast, ptr6 ceased growing and          by in situ hybridization with a biotin-labeled oligo(dT)50 probe. <br />                                                                  Hybridized signals were detected by FITC-conjugated avidin. <br /> rapidly lost viability after shifting to the nonpermissive       A, C, E, and G show the poly(A)1 RNA distribution, and B, <br /> temperature (378).                                               D, F, and H show the cells stained by DAPI in the correspond- <br />    Cloning of the ptr61 gene: To clone the ptr61 gene,           ing fields. Bar, 10 mm. <br />                                              Characterization of the ptr61 Gene                                            873 <br />  <br />  <br />                                                                                          Figure 2.&#226;&#8364;&#8221;Growth characteristics <br />                                                                                       of ptr6-1 mutant at the permissive and <br />                                                                                       nonpermissive temperatures. Wild- <br />                                                                                       type strain and ptr6-1 were cultured <br />                                                                                       in YPD medium at 268 up to a midlog <br />                                                                                       phase, and then either maintained at <br />                                                                                       268 or transferred to 378. (A) Growth <br />                                                                                       curves. The total cell number was <br />                                                                                       measured using a hemacytometer at <br />                                                                                       the indicated times after shifting. (B) <br />                                                                                       Relative viabilities. The results were <br />                                                                                       obtained from the number of colo- <br />                                                                                       nies grown at 268 after culturing at <br />                                                                                       268 or shifting to 378 for the indicated <br />                                                                                       times. The symbols indicate the <br />                                                                                       strains as follows: d, wild type main- <br />                                                                                       tained at 268; s, wild type shifted to <br />                                                                                       378; j, ptr6-1 maintained at 268; h, <br />                                                                                       ptr6-1 shifted to 378. <br />  <br />  <br />  <br /> we transformed the ptr6-1 mutant with an S. pombe wild-        fragment: one partial ORF encodes a carboxy-terminal <br /> type genomic library and isolated 16 cosmid clones that        region of a putative homologue of Bacillus stearothermoph- <br /> complemented the temperature-sensitive growth of ptr6.         ilus GldAp (glycerol dehydrogenase; Mallinder et al. <br /> Restriction mapping of those clones revealed that they         1992) and the other encodes an amino-terminal region <br /> contained the same insert DNA of z30 kb in <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>.             of a putative homologue of S. cerevisiae Msh1p involved <br /> After several steps of subcloning, the complementing           in mismatch repair of mitochondrial DNA (Reenan and <br /> activity was localized in a 4.2-kb NcoI-BglII fragment. The    Kolodner 1992a,b). As digestion of the fragment with <br /> restriction map is shown in Figure 3A. We sequenced            HindIII or BclI, which disrupts the complete ORF, abol- <br /> this fragment and found a single complete ORF. We              ished the complementing activity, we conclude that this <br /> also found parts of two other ORFs at both ends of the         ORF is responsible for the rescue activity for ptr6-1. <br />  <br />  <br />  <br />  <br />                                                                                             Figure 3.&#226;&#8364;&#8221;(A) Restriction map <br />                                                                                          of the 4.2-kb NcoI-BglII fragment <br />                                                                                          that rescues the ptr6 mutation. <br />                                                                                          The arrows show the ORFs for the <br />                                                                                          ptr61, gldA1, and msh11 genes. <br />                                                                                          Also shown is the direction of tran- <br />                                                                                          scription. The kinked line repre- <br />                                                                                          sents an intron region. Restriction <br />                                                                                          sites are abbreviated as follows: Bc, <br />                                                                                          BclI; Bg, BglII; Bs, BstEII; H, Hin- <br />                                                                                          dIII; N, NcoI; Nd, NdeI; Nr, NruI. <br />                                                                                          The NruI-NruI fragment within <br />                                                                                          the ptr61 gene was replaced by the <br />                                                                                          ura41 gene for gene disruption. <br />                                                                                          (B) Schematic representation of <br />                                                                                          the structures of Ptr6p, yTAFII67, <br />                                                                                          and hTAFII55. The numbers in- <br />                                                                                          dicate amino acid positions. <br />                                                                                          Hatched or stippled boxes denote <br />                                                                                          the regions highly conserved <br />                                                                                          among organisms. Solid boxes <br />                                                                                          represent regions containing <br />                                                                                          stretches of acidic amino acids. <br />                                                                                          The amino acid identities between <br />                                                                                          domains are shown in percent- <br />                                                                                          ages. The asterisk indicates the site <br />                                                                                          of the mutation in ptr6-1. <br />  874                                                  T. Shibuya et al. <br />  <br />    To obtain evidence that the identified gene is not an        mids carrying the leu1 marker and introduced them <br /> extragenic multicopy suppressor for the ptr6-1, we did          into ptr6-1. Plasmids containing the gap between HindIII <br /> integration mapping as described in materials and               and BclI sites in the ptr61 gene could not complement <br /> methods. Integration of the cloned gene linked with             the ts2 growth phenotype of ptr6-1 at 378, indicating that <br /> an ura4 marker into the authentic locus was confirmed           the mutation site is present within the gapped region. The <br /> by PCR (data not shown). One of the stable integrants           repaired plasmid was isolated from the transformant <br /> was then mated with the ptr6-1 strain (ST1), and 20             and sequenced. ptr6-1 has a single mutation site: G at <br /> tetrads were dissected. All of the ts2 spores were found        nucleotide position 586 was changed to A, resulting in <br /> to be uracil auxotrophs and all ts1 spores were uracil          replacement of glycine at amino acid position 182 to <br /> prototrophs, demonstrating that the cloned gene com-            glutamic acid. The mutation is located at a conserved <br /> plementing the ptr6 mutation is genetically linked to           residue within the conserved region (Figure 3B). <br /> the ptr61 locus. Thus, we concluded that the isolated              ptr61 is an essential gene: To determine whether the <br /> gene is not an extragenic suppressor but the authentic          ptr61 gene is required for growth in S. pombe, we carried <br /> ptr61 gene.                                                     out one-step gene disruption. The NruI-NruI fragment <br />    We found a possible intron within the amino-terminal         in the middle of the ptr61 gene was replaced with the <br /> region of the ptr61 gene. To determine the intron re-           ura41 gene to make a null mutation of ptr6 (Figure 3A). <br /> gion precisely, partial cDNA containing the predicted           Approximately 70% of the ptr61 ORF (from amino acid <br /> intron region was isolated using RT-PCR, and sequence           53 to 331) was deleted in the disrupted gene. A HindIII- <br /> analysis revealed that the ptr61 gene has an intron of          BglII fragment carrying the disrupted ptr61 gene was <br /> 41 nucleotides. The ORF of the ptr61 gene is 1220 nucle-        then used to transform wild-type diploid strain UDP6 <br /> otides in length and encodes a predicted protein of 393         lacking the ura41 alleles. We verified disruption of one <br /> amino acids.                                                    of the ptr61 alleles in the diploid cells by PCR (data not <br />    The ptr61 gene encodes a homologue of yTAFII67 of            shown). Tetrad analysis of 38 asci showed two viable <br /> S. cerevisiae and human hTAFII55: A comparison of the           and two inviable spores in all the cases (data not shown). <br /> amino acid sequence of Ptr6p with those in the Swiss            All the viable spores were ura2. These results suggest <br /> Prot database by using the BLAST program (Altschul              that the ptr61 gene is essential for cell viability in S. pombe. <br /> et al. 1990) revealed that the predicted amino acid se-            Ptr6p is localized in the nucleus: To determine the <br /> quence of Ptr6p shares a significant homology with              subcellular localization of Ptr6p, we constructed a gene <br /> those of yTAFII67 of budding yeast S. cerevisiae (Moq-          encoding GFP-tagged Ptr6p and introduced it into the <br /> taderi et al. 1996b) and human hTAFII55 (Chiang and             ptr6-1 mutant. The chimeric gene was capable of com- <br /> Roeder 1995; Lavigne et al. 1996). TAFs are subunits of         plementing temperature sensitivity of ptr6-1, demonstra- <br /> the general transcription factor complex named TFIID,           ting that the fusion protein is functional. The Ptr6p- <br /> a central component of the eukaryotic RNA polymerase            GFP was localized in the nucleus at both 268 and 378, <br /> II transcription apparatus consisting of TBP and a set          suggesting that Ptr6p is a nuclear protein (Figure 4). <br /> of TAFs (for reviews, see Burley 1996; Tansey and               The transformants harboring a plasmid with the un- <br /> Herr 1997; Lee and Young 1998).                                 tagged ptr61 gene gave no fluorescence (data not shown). <br />    The overall amino acid identity and similarity between          ptr6 has no defects in nuclear protein import: The <br /> Ptr6p and yTAFII67 are 32 and 54%, and those between            ptr2/pim1 and ptr3 mutants cause defects in nuclear <br /> Ptr6p and hTAFII55 are 22 and 36%, respectively. The            protein import as well as in mRNA export (Azad et al. <br /> amino-terminal region (from amino acids 64 to 167)              1997). To determine if ptr6 has defects in nuclear pro- <br /> and the central region (from amino acids 177 to 289) <br /> of the Ptr6p are highly conserved among the three or- <br /> ganisms (Figure 3B). In the case of hTAFII55, it was <br /> reported that these two regions are functional domains <br /> for the protein-protein interactions (Chiang and Roe- <br /> der 1995; also see discussion). The same conserved <br /> regions in Ptr6p and yTAFII67 may also be functioning <br /> as domains for protein interaction in yeast. Ptr6p and <br /> yTAFII67 contain a stretch of acidic amino acids (aspar- <br /> tic acid and glutamic acid) in the carboxy-terminal re- <br /> gion, which is lacking in hTAFII55. The acidic stretches <br /> of yTAFII67 are longer than those of Ptr6p. In addition, <br /> yTAFII67 contains a stretch of basic amino acids (lysine <br /> and arginine) in the amino-terminal region, although <br />                                                                   Figure 4.&#226;&#8364;&#8221;Intracellular localization of Ptr6p. Cells trans- <br /> there is no such stretch in Ptr6p and hTAFII55.                 formed with the gene encoding the Ptr6p tagged with GFP <br />    To identify a mutation site in ptr6-1, a gap repair ex-      were analyzed using a fluorescence microscope. DNA shows <br /> periment was done. We constructed several gapped plas-          cells stained with DAPI in the corresponding field. Bar, 10 mm. <br />                                                Characterization of the ptr61 Gene                                        875 <br />  <br /> tein import, we examined localization of a NLS-GFP               or 4 hr, and carried out Northern blot analysis of TBP <br /> fusion protein in the wild type, ptr2-1, and ptr6-1 (Figure      mRNA. The TBP gene of S. pombe contains three introns <br /> 5). At the permissive temperature (268), nuclear accu-           (Hoffmann et al. 1990). We used the oligonucleotide <br /> mulation of the NLS-GFP fusion protein was observed              probe complementary to the third exon sequence of <br /> in wild-type, ptr2-1, and ptr6-1 strains, indicating that the    TBP pre-mRNA (Table 2). As shown in Figure 6A, ptr6 <br /> NLS used in this study is functional in S. pombe (Figure         showed no accumulation of TBP pre-mRNA at both the <br /> 5, A, D, and G). Cells expressing the NLS-GFP fusion             permissive and nonpermissive temperatures and neither <br /> protein were preincubated at the nonpermissive tem-              did the wild-type cells. Therefore, ptr6 has no defects <br /> perature of 378 for 1 hr and then treated with azide and         in pre-mRNA splicing. Interestingly, the amount of TBP <br /> deoxyglucose at 378 for 1 hr, resulting in the poisoning         mRNA in ptr6 at the nonpermissive temperature was <br /> of the energy metabolism and the leakage of the NLS-             increased z5 to 6 times compared to those of wild type <br /> GFP fusion protein into the cytoplasm (Figure 5, B, E,           and ptr6 at the permissive temperature. The increase of <br /> and H). After washing and resuspending the cells in              TBP mRNA at the nonpermissive temperature in ptr6 <br /> glucose-containing MM, the cells were incubated for 30           was rescued completely with pSP1-N/Bg containing the <br /> min at 378 to analyze the reimport of the NLS-GFP                ptr61 gene (Figure 6A). <br /> fusion protein into the nucleus (Figure 5, C, F, and                Homology search showed that Ptr6p is a putative ho- <br /> I). Protein import of the NLS-GFP fusion protein was             mologue of a TFIID subunit, hTAFII55. Also, Northern <br /> inhibited in ptr2-1 cells at 378 (Figure 5I), whereas no         blot analysis using the TBP probe revealed abnormal <br /> defect in nuclear protein import was observed in the             increases in TBP mRNA at the nonpermissive tempera- <br /> ptr6-1 and wild-type cells (Figure 5, C and F). The defect       ture. Therefore, we asked if ptr6 is abnormal in the <br /> in protein import was not observed, even when the                transcription of genes in general. Total RNA of ptr6-1 <br /> ptr6-1 cells were preincubated at 378 for 6 hr (data not         cells cultured at 268 or shifted to 378 for 2 hr or 4 hr <br /> shown).                                                          were subjected to Northern blot analysis using probes <br />    ptr6 has defects in transcription: To determine if ptr6       for tbp1, ptr61, act11, rna11, rae11, pim11, taf721, prp21, <br /> has defects in pre-mRNA splicing, we isolated total RNA          cdc21, and rpb11 mRNAs. Act1p (actin1) is a constitu- <br /> from ptr6-1 cells cultured at 268 or shifted to 378 for 2        tively expressed protein. Rna1p, Rae1p, and Pim1p/ <br />                                                                  Ptr2p are essential factors involved in mRNA transport <br />                                                                  from the nucleus to the cytoplasm in S. pombe (Mel- <br />                                                                  chior et al. 1993; Brown et al. 1995; Azad et al. 1997). <br />                                                                  Taf72p is an S. pombe homologue of yTAFII90 in S. <br />                                                                  cerevisiae (Yamamoto et al. 1997). Prp2p is a homologue <br />                                                                  of the mammalian splicing factor U2AF65 (Potashkin <br />                                                                  et al. 1993). Cdc2p is a factor involved in cell-cycle pro- <br />                                                                  gression (Rhind et al. 1997) and Rpb1p is the largest <br />                                                                  subunit of RNA polymerase II (Azuma et al. 1991). In- <br />                                                                  triguingly, in contrast to the drastic increase of tbp1 <br />                                                                  mRNA in ptr6 at the nonpermissive temperature, the <br />                                                                  amounts of ptr61, act11, rna11, rae11, pim11, taf721, <br />                                                                  prp21, cdc21, and rpb11 mRNAs were decreased rapidly <br />                                                                  at the same temperature (Figure 6A). The decrease of <br />                                                                  ptr61 mRNA at the nonpermissive temperature was also <br />                                                                  rescued with pSP1-N/Bg. The increase of ptr61 mRNA <br />                                                                  in pSP1-N/Bg transformants at both the permissive and <br />                                                                  nonpermissive temperatures is due to overexpression <br />                                                                  of ptr61 mRNA from the gene in the multicopy plasmid. <br />                                                                  These results suggest that ptr6 is defective in the regula- <br />                                                                  tion of transcription of many genes at the nonpermissive <br />                                                                  temperature. <br />                                                                     To determine if ptr6 has defects in transcription of <br />    Figure 5.&#226;&#8364;&#8221;Analysis of nuclear protein import. Wild type <br /> (A&#226;&#8364;&#8220;C), ptr6-1 (D&#226;&#8364;&#8220;F), and ptr2-1 (G&#226;&#8364;&#8220;I) expressing the NLS-GFP     genes driven by RNA polymerase II in general, we evalu- <br /> fusion protein were cultured at 268 (A, D, and G). Cells were    ated the amount of the total poly(A)1 RNA by hybridiza- <br /> preincubated for 1 hr at 378 and then metabolically poisoned     tion with a 32P-labeled oligo(dT)30 probe. The results <br /> to allow nucleocytoplasmic equilibration of the NLS-GFP fu-      are shown in Figure 6B. The quantity of total poly(A)1 <br /> sion protein by using azide and deoxyglucose (B, E, and H).      RNA in the wild-type cells was maintained constant at <br /> After recovery from the drug treatment in glucose-containing <br /> MM medium for 30 min at 378, the reimport of the NLS-GFP         the nonpermissive temperature; however, the amount <br /> fusion protein into the nucleus was analyzed (C, F, and I).      of the total poly(A)1 RNA in ptr6 was remarkably de- <br /> Bar, 10 mm.                                                      creased after shifting to the nonpermissive temperature. <br />  876                                                  T. Shibuya et al. <br />  <br />  <br />  <br />  <br />                                                                                                Figure 6.&#226;&#8364;&#8221;(A) Northern blot <br />                                                                                             analysis of specific mRNAs. Total <br />                                                                                             RNA (20 mg) prepared from the <br />                                                                                             wild type, ptr6-1, or ptr6-1 trans- <br />                                                                                             formed with pSP1-N/Bg that had <br />                                                                                             been incubated at the permissive <br />                                                                                             temperature (268) or shifted to <br />                                                                                             the nonpermissive temperature <br />                                                                                             (378) for 2 or 4 hr was subjected <br />                                                                                             to Northern blot hybridization. <br />                                                                                             Arrowheads on the right indicate <br />                                                                                             the unspliced precursor (P) and <br />                                                                                             the mature TBP mRNA (M). (B) <br />                                                                                             Analysis of total poly(A)1 RNA. <br />                                                                                             Wild type, ptr6-1, and ptr6-1 con- <br />                                                                                             taining pSP1-N/Bg were main- <br />                                                                                             tained for the indicated times at <br />                                                                                             378. Total RNA (20 mg) was then <br />                                                                                             analyzed by Northern blot hybrid- <br />                                                                                             ization with the 32P-oligo(dT)30. In- <br />                                                                                             tensity of the smeared band in <br />                                                                                             each lane was quantitated using a <br />                                                                                             Bioimaging Analyzer BAS1000 <br />                                                                                             (Fuji Photo Film Co.) and is <br />                                                                                             shown at the bottom using photo <br />                                                                                             stimulated luminescence (PSL) <br />                                                                                             units of BAS1000. <br />  <br />  <br />  <br />  <br /> Taken together, we conclude that the transcription of           probe and classified it as ptr6. The ptr6 mutant shows <br /> most genes by RNA polymerase II in ptr6 is repressed            nuclear accumulation of mRNA, severe growth defect, <br /> abnormally at the nonpermissive temperature, support-           and defects in gene transcription at the nonpermissive <br /> ing the hypothesis that the ptr61 gene encodes the S.           temperature. <br /> pombe homologue of hTAFII55.                                      Ptr6p is a homologue of TAFs: The ptr61 gene en- <br />    FACS analysis: It has been reported that mutations <br /> in yTAFII145 and yTAFII90 result in specific arrests in <br /> G1 and G2/M, respectively, in S. cerevisiae (Apone et al. <br /> 1996; Walker et al. 1997). To examine if ptr6 also has <br /> a cell-cycle defect at the nonpermissive temperature, we <br /> performed FACS analysis. In contrast to the cell division <br /> cycle mutant cdc10-129 arresting in the G1 phase, the <br /> ptr6-1 mutant grown in a rich medium appears to have <br /> no specific defects in the cell cycle at the nonpermissive <br /> temperature, although slight delay in DNA synthesis <br /> was observed 4 hr after shifting to the nonpermissive <br /> temperature (Figure 7). In addition, the ptr6-1 mutant <br /> showed no major morphological abnormality as was ob- <br /> served in the cdc mutant, even after shifting to the non- <br /> permissive temperature for 12 hr (data not shown). <br />  <br />  <br />                       DISCUSSION                                  Figure 7.&#226;&#8364;&#8221;FACS analysis of wild-type (A&#226;&#8364;&#8220;C), ptr6-1 (D&#226;&#8364;&#8220;F), <br />                                                                 and cdc10-129 (G&#226;&#8364;&#8220;I) cells following a shift to the nonpermissive <br />   We identified one new S. pombe ptr mutant defective           temperature. The DNA content and relative cell number were <br /> in the transport of poly(A)1 RNA from the nucleus to            plotted along with x and y axes, respectively. The positions of <br /> the cytoplasm by in situ hybridization with the oligo(dT)       1C and 2C DNA content are indicated. <br />                                               Characterization of the ptr61 Gene                                        877 <br />  <br /> codes a putative homologue of S. cerevisiae yTAFII67            shown to be required for transcription of G1/S cyclin <br /> and human hTAFII55. yTAFII67 was identified as a ho-            genes and thereby involved in the transcriptional regula- <br /> mologue of human hTAFII55 by a computer search of               tion of the cell-cycle progression (Walker et al. 1997). <br /> the complete S. cerevisiae genome and cloned by PCR             The yTAFII90 ts mutant also shows the G2/M phase <br /> amplification. yTAFII67 is reported to be essential for         cell-cycle arrest (Apone et al. 1996), suggesting a linkage <br /> cell viability (Moqtaderi et al. 1996b). On the other           between TAFII functions and cell-cycle progression. On <br /> hand, hTAFII55 interacts with multiple transcription            the other hand, studies have also shown that despite <br /> activators through the amino-terminal domain (be-               inactivation or depletion of several yTAFIIs, a variety of <br /> tween amino acids 38 and 113) and also interacts with           yeast genes driven by diverse activators were transcribed <br /> the largest subunit (hTAFII250) of human TFIID, which           normally (Apone et al. 1996; Moqtaderi et al. 1996a; <br /> has been implicated in the regulation of cell-cycle pro-        Walker et al. 1996). A dot blot assay showed no signifi- <br /> gression (Hisatake et al. 1993; Ruppert et al. 1993;            cant decrease in the synthesis of total poly(A)1 RNA <br /> Wang and Tjian 1994) through the central region (be-            following temperature-sensitive inactivation of yTAFII145, <br /> tween amino acids 139 and 249; Chiang and Roeder                TSM1, or yTAFII90 (Walker et al. 1997). These studies <br /> 1995; Lavigne et al. 1996; Figure 3B).                          suggested that multiple yTAFIIs are dispensable for tran- <br />    These previously identified TAFs were shown to asso-         scription of the vast majority of yeast mRNA-encoding <br /> ciate with TBP by immunoprecipitation assay. In the             genes. However, gene disruption experiments in bud- <br /> case of Ptr6p, the possible association with TBP was            ding yeast revealed that most TAFs, except for yTAFII30, <br /> not determined. However, there are lines of evidence            are essential for cell viability (Reese et al. 1994; Poon <br /> suggesting that Ptr6p is a TAF. First, Ptr6p has significant    et al. 1995; Moqtaderi et al. 1996b; Lee and Young <br /> homologies with yTAFII67 and hTAFII55 in overall                1998). Taken together, it was suggested that yTAFs play <br /> amino acid sequence and contains several conserved              a role in the transcription of specific genes involved in <br /> domains present in yTAFII67 and hTAFII55 (Figure                cell-cycle progression. <br /> 3B). Second, Ptr6p localized in the nucleus (Figure 4).            In contrast, the ptr6 mutant does not display the spe- <br /> Third, the ptr6 mutant has defects in the transcription         cific defects in cell-cycle progression in the FACS and <br /> of the tested genes at the nonpermissive temperature            morphological analyses, suggesting that Ptr6p does not <br /> (Figure 6). Finally, the antibody against Ptr6p was found       function in cell-cycle progression primarily. We exam- <br /> to coimmunoprecipitate a homologue of the S. cerevisiae         ined the possibility that Ptr6p functions in the transcrip- <br /> yTAFII145 (T. Miyake and T. Kokubo, personal com-               tion of the genes involved in mRNA transport. Northern <br /> munication). From these results, we believe that the            blot hybridization showed that transcription of the <br /> Ptr6p is the S. pombe homologue of yTAFII67 and                 genes involved in mRNA transport (rna11, rae11, and <br /> hTAFII55.                                                       pim11) was inhibited at the nonpermissive temperature <br />    What is the function of Ptr6p in S. pombe? The tran-         in ptr6 (Figure 6). However, transcription of the act11, <br /> scription factor complex TFIID is composed of TBP and           taf721, prp21, cdc21, and rpb11 genes unrelated to the <br /> a set of TAFs. Although essential roles of TBP in all           mRNA transport was also dramatically decreased, sug- <br /> eukaryotic transcription have been extensively analyzed         gesting that Ptr6p is not a TAF specifically required for <br /> in vivo and in vitro (Hernandez 1993; Struhl 1995),             the transcription of the genes involved in mRNA export. <br /> the functions of the TAFs are less clear.                       In addition, the amount of the total poly(A)1 RNA was <br />    Information on the functions of TAFs has been mostly         decreased at the nonpermissive temperature in ptr6. <br /> obtained from in vitro studies on partially purified mam-       These results suggest that, in the ptr6 mutant, most or <br /> malian and Drosophila factors (Goodrich and Tjian               all protein-coding genes are not transcribed normally <br /> 1994). These in vitro studies suggested that TAFs are           at the nonpermissive temperature. Therefore, in con- <br /> dispensable for basal transcription but are required as         trast to previously reported yTAFII145 and yTAFII90, <br /> transcriptional coactivators for the response to regula-        the Ptr6p may be a TAF required generally for transcrip- <br /> tory proteins or activators (Tjian and Maniatis 1994)           tion of the genes. The slight delay in DNA synthesis in <br /> and that specific TAFs may act as molecular bridges             ptr6 at the nonpermissive temperature is likely to be a <br /> between particular activators and the general transcrip-        secondary effect of the drastic decrease of gene tran- <br /> tion machinery (Chen et al. 1994; Sauer et al. 1995).           scription, considering that many labile proteins are in- <br />    To analyze the in vivo function of a specific TAF,           volved in that process. Rapid repression of the gene <br /> several temperature-sensitive TAF mutants of S. cerevisiae      transcription at the nonpermissive temperature may <br /> have been constructed using reverse genetics. Of those,         also account for the weak nuclear signals in ptr6 by <br /> the yTAFII145 ts mutant shows a cell-cycle arrest in the        in situ hybridization compared to those in previously <br /> G1 phase at the nonpermissive temperature (Walker               identified ptr mutants. <br /> et al. 1996, 1997; Shen and Green 1997). The yTAFII145             Interestingly, Northern blot analysis revealed that the <br /> is a homologue of the mammalian hTAFII250 impli-                amount of TBP mRNA was increased five- to sixfold in <br /> cated in cell-cycle regulation (Hisatake et al. 1993; Rup-      contrast to the other analyzed mRNAs at the nonpermis- <br /> pert et al. 1993; Wang and Tjian 1994). yTAFII145 was           sive temperature. One explanation for this phenome- <br />  878                                                          T. Shibuya et al. <br />  <br /> non is that production of TBP might be increased to                     for valuable discussions on TAFs, Dr. H. Nishitani for assistance in <br /> compensate for reduction of the Ptr6p/hTAFII55 activ-                   the FACS analysis, and M. O&#226;&#8364;&#8482;Hara for comments. This research was <br />                                                                         supported by grants from the Ministry of Education, Science, Sports, <br /> ity and to maintain basal transcription activity.                       and Culture of Japan to T.T. and Y.O., and by a grant from the Japan <br />    Relationship between mRNA transport and transcrip-                   Science and Technology Corporation to T.T. <br /> tion: Severe defects in gene transcription were detected <br /> 2 hr after shifting to the nonpermissive temperature in <br /> ptr6. In contrast, a significant accumulation of mRNA                           <br /> in the nucleus requires a shifting time exceeding 2 hr.      <br /> Thus, block of mRNA export might possibly be a second-                <br /> ary effect of the reduced transcription of genes required               <br /> for mRNA transport, such as the rna11, rae11, and pim11               <br /> genes. However, though the pim11 gene is also essential           <br /> for protein import, defects in protein import were not             <br /> seen in ptr6. In addition, despite the decreased tran-                <br /> scription levels of the prp21 gene, the ptr6 mutant did                    <br /> not show defects in pre-mRNA splicing. Moreover, the                 <br /> inhibition of protein synthesis by cycloheximide in S.                     <br /> pombe did not block mRNA export, suggesting that labile                     <br /> proteins are not involved in the pathway of mRNA trans-            <br /> port (Tani et al. 1995). Therefore, it is likely that defects               <br /> in poly(A)1 RNA transport in the ptr6 mutant are not                       <br /> simply caused by defects in gene transcription leading            <br /> to reduction of proteins required for mRNA transport,               <br /> but rather are a direct consequence of the inactivation                     <br /> of Ptr6p. There is a precedent that a transcription factor           <br /> is directly involved in mRNA export. It has been shown                   <br /> that the TFIIIA, an RNA polymerase III transcription                   <br /> factor, binds to 5S rRNA and mediates its nuclear export               <br /> (Fridell et al. 1996). TFIIIA contains a leucine-rich                   <br /> sequence functioning as an effective NES in both hu-                  <br /> man cells and frog oocytes. Similarly, Ptr6p might also          <br /> have dual roles in mRNA export and gene transcription,                 <br /> although no typical leucine-rich NES was found. To our                  <br /> knowledge, this is the first report suggesting that a TAF                <br /> is involved in poly(A)1 RNA transport besides transcrip-               <br /> tion.                                                                    <br />    Interestingly, hTAFII55 was recently shown to bind           <br /> strongly with the 160-kD subunit of human cleavage-                        <br /> polyadenylation specificity factor (CPSF) by the immu-                    <br /> noprecipitation assay (Dantonel et al. 1997). CPSF is            <br /> a complex that plays a role in 39 processing of precursor                  <br /> mRNA by binding to the AAUAAA signal sequence and                      <br /> coordinating interactions with other polyadenylation               <br /> factors (Keller 1995). Several lines of evidence sug-                    <br /> gested a link between the polyadenylation (39 end for-                    <br /> mation) and the export of RNA from the nucleus (Eck-              <br /> ner et al. 1991; Huang and Carmichael 1996; Kessler                          <br /> et al. 1997). Therefore, if Ptr6p, an S. pombe homologue               <br /> of hTAFII55, also interacts with CPSF in S. pombe, Ptr6p               <br /> may play an important role in coordinated interplay           <br /> between mRNA export and the 39 end processing. Fur-                         <br /> ther investigations of the functions of Ptr6p will shed          <br /> light on the connection among mRNA transport, 39 end                       <br /> processing, and gene transcription.                                       <br /> </body></html>