<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Htm1p, a mannosidase-like protein, is involved in <br /> glycoprotein degradation in yeast <br />  <br /> Misfolded proteins are recognized in the endoplasmic reticulum                   N-linked oligosaccharides are processed by glucosidase I, <br /> (ER), transported back <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> the cytoplasm and degraded by the                      glucosidase II and mannosidase I (Herscovics, 1999a). It has <br /> proteasome. Processing intermediates of N-linked oligo-                          been shown that oligosaccharide processing and protein folding <br /> saccharides on incompletely folded glycoproteins have an                         are linked processes coordinating protein folding, ER retention/ <br /> important role in their folding/refolding, and also in their                     retrieval and protein degradation (Ellgaard et al., 1999; <br /> targeting to proteolytic degradation. In Saccharomyces                           Herscovics, 1999b; Jakob and Burda, 1999). <br /> cerevisiae, we have identified a gene coding for a non-essential                    Mechanisms that discern misfolded secretory and membrane <br /> protein that is homologous to mannosidase I (HTM1) and that                      proteins have been discovered in the past few years (Bonifacino <br /> is required for degradation of glycoproteins. Deletion of                        and Weissman, 1998). Misfolded proteins in the ER lumen are <br /> the HTM1 gene does not affect oligosaccharide trimming.                          specifically recognized by as yet unknown mechanisms and <br /> However, deletion of HTM1 does reduce the rate of degradation                    subjected to a cytosolically located proteolytic process termed <br /> of the mutant glycoproteins such as carboxypeptidase Y, ABC-                     ER-associated degradation (ERAD). The targeted proteins are <br /> transporter Pdr5-26p and oligosaccharyltransferase subunit                       transported back into the cytoplasm by a mechanism that is <br /> Stt3-7p, but not of mutant Sec61-2p, a non-glycoprotein. Our                     formed, at least partially, by constituents of the ER import <br /> results indicate that although Htm1p is not involved in                          machinery (R&#195;&#182;misch, 1999). Many of the retrotranslocated proteins <br /> processing of N-linked oligosaccharides, it is required for their                are polyubiquitin modified by the action of the cytosolically <br /> proteolytic degradation. We propose that this mannosidase                        located enzymes Ubc6p, Ubc7p and Hrd1/Der3p, followed by <br /> homolog is a lectin that recognizes Man8GlcNAc2 oligosaccharides                 degradation by the proteasome (Brodsky and McCracken, 1997; <br /> that serve as signals in the degradation pathway.                                Kopito, 1997; Sommer and Wolf, 1997; Bays et al., 2001; Deak <br />                                                                                  and Wolf, 2001). <br />                                                                                     The degradation of misfolded mutant carboxypeptidase Y [CPY* <br />                      INTRODUCTION                                                (prc1-1); Finger et al., 1993] has been shown to be N-glycan <br /> Most secretory and membrane proteins are co-translationally                      dependent (Knop et al., 1996) and, moreover, the structure of <br /> translocated into the lumen of the endoplasmic reticulum (ER)                    this oligosaccharide plays an important role in rapid protein <br /> (Rapoport et al., 1996) where they acquire N-linked oligo-                       degradation (Jakob et al., 1998). Specifically, glycoproteins <br /> saccharides, catalyzed by oligosaccharyltransferase. Also in the                 carrying Man8GlcNAc2 sugars are more rapidly degraded than <br /> ER, secretory and membrane proteins attain their correct three-                  others. This finding supported the &#226;&#8364;&#732;mannose timer hypothesis&#226;&#8364;&#8482; <br /> dimensional structure by the concerted action of chaperones,                     (Su et al., 1993; Helenius et al., 1997) and has led to the <br /> isomerases and folding mediators (Ellgaard et al., 1999). The                    proposal that the ER lumen contains a Man8GlcNAc2-binding <br />  <br />  <br />  +Corresponding   author. Tel: +41 1 632 3327; Fax: +41 1 632 1148; E-mail: jakob@micro.biol.ethz.ch <br />  &#226;&#8364;&#161;Present  address: Clinic for Neurosurgery, University Hospital Zurich, CH-8091 Zurich, Switzerland <br />  &#194;&#167;Present address: Cytos Biotechnology AG, CH-8952 Schlieren-Zurich, Switzerland <br />  &#194;&#182; Present address: Biochemistry Department, McGill University, Montreal, PQ, H3G 1Y6, Canada <br />  <br />  C.A. Jakob and D. Bodmer contributed equally to this work <br />  <br />  <br />  <br /> &#194;&#169; 2001 European Molecular Biology Organization                                                         EMBO reports vol. 2 | no. 5 | pp 423&#226;&#8364;&#8220;430 | 2001 423 <br />                                                         scientific reports <br />      C.A. Jakob et al. <br />  <br />      protein that initiates glycoprotein ERAD (Liu et al., 1997; Jakob <br />      et al., 1998; Yang et al., 1998). <br />         In the yeast genome we have identified a good candidate for <br />      this Man8GlcNAc2-specific lectin (Htm1p) that is homologous to <br />      mannosidase I (Camirand et al., 1991). We show that strains <br />      deleted for HTM1 are defective in the degradation of a mutant <br />      soluble protein (CPY*), and also two mutant membrane proteins <br />      (Pdr5-26p, Stt3-7p). Our data strongly suggest that the Htm1 <br />      protein is the putative Man8GlcNAc2-binding lectin. <br />  <br />               RESULTS AND DISCUSSION <br />      HTM1 codes for a gene homologous to mannosidase <br />           and is involved in protein degradation <br />      We searched the Saccharomyces cerevisiae genome for open <br />      reading frames with homology to ER mannosidase since we <br />      hypothesized that a &#226;&#8364;&#732;degradation lectin&#226;&#8364;&#8482; might contain similar <br />      domains. The reading frame YHR204w (termed gene homo- <br />      logous to mannosidase; HTM1) was identified, showing a 40% <br />      amino acid sequence similarity to yeast mannosidase. <br />         To test whether HTM1 was involved in protein degradation, <br />      we deleted the gene in a yeast strain containing CPY* (Wolf and <br />      Fink, 1975) and Pdr5* [mutant Pdr5 protein (pdr5-26); Egner et <br />      al., 1998], two substrates that have previously been shown to <br />      undergo ubiquitin-dependent proteasomal degradation. As <br />      controls, we used mutant strains that were deficient in the <br />      biosynthesis of complete oligosaccharide chains (&#226;&#710;&#8224;alg12) or in <br />      trimming of the oligosaccharides (&#226;&#710;&#8224;mns1). These mutations <br />      prevent the biosynthesis of Man8GlcNAc2 oligosaccharide or its <br />      formation by processing, respectively, and stabilize misfolded <br />      CPY* (Jakob et al., 1998). Equal numbers of cells were inocu- <br />      lated and de novo protein synthesis was stopped by the addition <br />      of cycloheximide; the levels of CPY* and Pdr5* were determined <br />      by immunoblotting and quantified at different time points. The <br />      CPY* degradation rate in &#226;&#710;&#8224;alg12 and &#226;&#710;&#8224;mns1 cells was reduced <br />      in comparison to &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; cells, but also in the HTM1- <br />      deficient cells (Figure 1). In addition, we also found a higher <br />      steady-state level of CPY* in &#226;&#710;&#8224;alg12, &#226;&#710;&#8224;mns1 and &#226;&#710;&#8224;htm1 cells as <br />      compared with wild-type cells (time point 0; Figure 1A, compare <br />      lanes 1, 4, 7 and 10). The mutant membrane protein Pdr5* <br />      showed a similar change in its degradation: steady-state levels of <br />      Pdr5* were increased in &#226;&#710;&#8224;alg12 and &#226;&#710;&#8224;mns1, and also &#226;&#710;&#8224;htm1 <br />      cells (Figure 1), and degradation rates were significantly reduced <br />      (Figure 1). <br />         To test whether HTM1 was involved in general protein degra- <br />      dation, we studied the mutant Sec61-2 protein, a membrane <br />      protein with no N-glycosylation sites. The sec61-2 mutant cells      Fig. 1. Oligosaccharide-dependent degradation of the mutant proteins CPY* <br />      display a temperature-sensitive growth <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Biederer et        and Pdr5*. (A) Isogenic cells were grown to mid-log phase. Equal cell <br />                                                                           numbers were then incubated with 100 &#194;&#181;g/ml cycloheximide and aliquots <br />      al. (1996) showed that Sec61-2 protein was degraded at restric-      removed at given time points. Crude protein extract was separated by SDS&#226;&#8364;&#8220;PAGE <br />      tive temperature. Impairing protein degradation by deleting the      and analyzed by immunoblotting using anti-CPY (CPY*) and anti-HA <br />      UBC6 and UBC7 genes allowed the cells to survive restrictive         (Pdr5*) antisera. (B) Protein amounts <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> at least two independent <br />      growth conditions (Figure 2A). We deleted the MNS1 and HTM1          experiments were quantified and plotted against time, setting the quantity of <br />                                                                           time point 0 (steady-state level) to 100%. <br />      genes in the sec61-2 mutants and tested whether these two <br />      genes influenced degradation of the Sec61-2 protein by cell <br />      growth at selective temperature and by western blot analysis. <br />      Neither of the two mutant cells (sec61-2 &#226;&#710;&#8224;mns1 and sec61-2           (Figure 2A) nor reduced the degradation rate of Sec61-2 protein <br />      &#226;&#710;&#8224;htm1) improved growth on plates at restrictive temperature          (Figure 2B). <br />  <br />  <br /> 424 EMBO reports vol. 2 | no. 5 | 2001 <br />                                                            scientific reports <br />                                                                                                  Role of Htm1p in glycoprotein degradation in yeast <br />  <br />  <br />  <br />  <br /> Fig. 2. The HTM1 gene does not influence the degradation of mutant Sec61-2 protein. (A) &#226;&#710;&#8224;mns1 and &#226;&#710;&#8224;htm1 do not suppress temperature-sensitive growth of <br /> sec61-2 mutant cells. Isogenic sec61-2 single- and double-mutant cells were plated in serial dilutions on rich medium and grown at 30 and 37&#194;&#176;C for 3 days. <br /> (B) Degradation of Sec61-2 protein is not affected in mns1- and htm1-deficient cells. Cells were grown at 30&#194;&#176;C to mid-log phase and shifted to 37&#194;&#176;C for 2 h. Equal <br /> cell numbers were incubated with 100 &#194;&#181;g/ml cycloheximide, aliquots were removed and cells broken at given time points. Extracted proteins were separated by <br /> SDS&#226;&#8364;&#8220;PAGE, and analyzed and quantified by immunoblotting using anti-Sec61 antiserum. The protein quantity of time point 0 (steady-state level) was set to 100%. <br />  <br />  <br />  <br />  <br />    These results support the hypothesis that the HTM1 gene is                       sequences similar to Htm1 protein are found in fungal, plant, <br /> required for efficient secretory and membrane glycoprotein                          insect, worm and mammal genomes. Phylogenetic analysis of <br /> degradation. Degradation of proteins lacking N-glycans was not                      these sequences revealed two distinct sequence groups, of <br /> affected. Glycoprotein degradation might require a specific                         which one contained all class I &#206;&#177;1,2-mannosidases and the <br /> lectin that recognizes N-linked Man8GlcNAc2 oligosaccharides                        other proteins homologous to Htm1p (data not shown). Many <br /> (Jakob et al., 1998). For this we reasoned that such a lectin might                 amino acids reported to be involved in oligosaccharide binding, <br /> be similar to ER mannosidase I. The HTM1 gene codes for a                           catalysis and Ca2+ coordination in Mns1p (Vallee et al., 2000) <br /> polypeptide of 796 amino acids with significant sequence                            are conserved in Htm1 protein and its homologs. One of the <br /> similarity to yeast mannosidase I (Figure 3A), and amino acid                       characteristic features of the Htm1 protein family group was the <br />  <br />  <br />                                                                                                                           EMBO reports vol. 2 | no. 5 | 2001 425 <br />                                                                 scientific reports <br />      C.A. Jakob et al. <br />  <br />  <br />  <br />  <br />      Fig. 3. Similarity of Mns1p and Htm1 proteins. (A) Areas of homology of <br />      Mns1p and Htm1p. Areas of high amino acid similarity are drawn as boxes.        Fig. 4. Normal N-linked oligosaccharide processing in htm1-deficient yeast <br />      (B) Htm1 family proteins lack a highly conserved cysteine residue. Amino        cells. N-linked oligosaccharides of isogenic strains were metabolically labeled <br />      acid sequences of &#206;&#177;-mannosidases and Htm1 family proteins of plant,             with [3H]mannose, extracted, released with PNGase F and separated by HPLC. <br />      mammalian, insect, worm and yeast cells were aligned with Clustal_W. The <br />      sequences were grouped by sequence similarity. The arrowhead indicates the <br />      position of an alanine residue in Htm1 proteins at the position of Cys340 (in <br />      S. cerevisiae) conserved in and most likely essential for maintaining the <br />      correct structure of &#206;&#177;1,2-mannosidases. The first two letters indicate the      obtained from cells deleted for HTM1 showed Man8GlcNAc2 <br />      species: at, Arabidopsis thaliana; ce, Caenorhabditis elegans; dm, Drosophila   but no Man9GlcNAc2 oligosaccharides. These results confirm <br />      melanogaster; hs, Homo sapiens; sc, S. cerevisiae; sp, Schizosaccharomyces      that normal processing of N-glycans occurs in &#226;&#710;&#8224;htm1 cells and, <br />      pombe. The following numbers refer to the enzyme name or DDBJ/EMBL/ <br />                                                                                      therefore, the Htm1 protein is not a processing mannosidase. <br />      GenBank accession No. <br />                                                                                      Although Htm1p displays sequence similarity to class I manno- <br />                                                                                      sidases, it is not involved in the specific trimming of N-glycans <br />                                                                                      in the ER, which is the only process that requires mannosidase <br />      absence of the conserved cysteine residues 340 and 385 (position                activity in yeast (Dean, 1999; Gemmill and Trimble, 1999; <br />      given for Mns1p of S. cerevisiae), which form a disulfide bond.                 Herscovics, 1999a). This is consistent with our conclusion that <br />      These two cysteine residues are conserved in &#206;&#177;1,2-mannosidase                   Htm1p is not a mannosidase, and with our inference that it may <br />      family members and are most likely to be important for main-                    be a lectin. <br />      taining the correct structure of processing mannosidases (Lipari                   There have been several reports that indicate a distinct <br />      and Herscovics, 1996) (Figure 3B).                                              function of N-glycan trimming by mannosidase I in glycoprotein <br />                                                                                      degradation (Liu et al., 1997, 1999; Yang et al., 1998; Cabral et <br />                HTM1 is not involved in processing of                                 al., 2000; Mancini et al., 2000; Wang and Androlewicz, 2000; <br />                    N-linked oligosaccharides                                         Wang and White, 2000). These reports are in good <br />                                                                                      agreement with results found in S. cerevisiae, where a specific <br />      Processing of N-glycans to create the Man8GlcNAc2 glycan is <br />                                                                                      Man8GlcNAc2 glycan is required for efficient protein degradation in <br />      required for efficient degradation in yeast (Jakob et al., 1998 and <br />                                                                                      the ER (Knop et al., 1996; Jakob et al., 1998). This specific <br />      see above). We determined whether altered processing of <br />                                                                                      oligosaccharide on degradation substrates might be recognized <br />      oligosaccharides was the reason for reduced glycoprotein <br />      degradation in &#226;&#710;&#8224;htm1 cells. For this, oligosaccharides were                     by the Htm1 protein and by this facilitate glycoprotein degradation. <br />      metabolically labeled in vivo. N-linked sugars were extracted <br />      and released from the protein by peptide N-glycosidase F                                   Degradation of the Stt3-7 protein is <br />      (PNGase F) treatment and analyzed by HPLC. Attention was                                  oligosaccharide specific and mediated <br />      focused on sugars that were not processed by Golgi mannosyl-                              by the ubiquitin&#226;&#8364;&#8220;proteasomal pathway <br />      transferases (Figure 4) and, as expected, wild-type cells <br />      contained Man8GlcNAc2 oligosaccharides, and &#226;&#710;&#8224;mns1 cells                         In yeast, oligosaccharyltransferase (OST), the enzyme complex <br />      contained Man9GlcNAc2 (Figure 4). However, oligosaccharides                     that transfers oligosaccharides from lipid-linked precursors to <br />  <br />  <br /> 426 EMBO reports vol. 2 | no. 5 | 2001 <br />                                                            scientific reports <br />                                                                                             Role of Htm1p in glycoprotein degradation in yeast <br />  <br />  <br />  <br />  <br />                                                                                 Fig. 6. Oligosaccharide-dependent suppression of conditional growth of stt3-7 <br />                                                                                 by deletion of MNS1 and HTM1 genes. Isogenic stt3-7 single- and double-mutant <br />                                                                                 cells were plated in serial dilutions on rich medium and grown at 23 and 30&#194;&#176;C <br />                                                                                 for 3 days. <br />  <br />  <br />  <br />  <br />                                                                                 OST activity, leading to incomplete N-glycosylation of nascent <br />                                                                                 glycoproteins at restrictive temperature (Spirig et al., 1997). We <br />                                                                                 observed that overexpression or deletion of the UBC6 gene <br />                                                                                 could suppress the temperature-sensitive phenotype of mutant <br />                                                                                 stt3-7 strains (Figure 5A). <br />                                                                                    UBC6 encodes the E2 ubiquitin-conjugating enzyme situated <br />                                                                                 on the cytosolic face of the ER membrane and plays a pivotal <br />                                                                                 role in the degradation of misfolded proteins in the ER (Sommer <br />                                                                                 and Wolf, 1997). Overexpression or deletion of the UBC6 gene <br />                                                                                 increased the steady-state level of the mutant Stt3-7 protein as <br />                                                                                 well as levels of the OST subunits Ost3p and Ost6p, as shown by <br />                                                                                 western blot analysis (Figure 5B). N-linked glycosylation was <br />                                                                                 strongly affected in stt3-7 mutant cells (visualized by the <br />                                                                                 presence of hypoglycosylated CPY) and was only slightly <br />                                                                                 improved by reducing protein degradation using UBC6 mutants <br />                                                                                 (Figure 5B). Our finding shows that mutant Stt3-7 protein is <br /> Fig. 5. UBC6-dependent degradation of stt3-7 protein. (A) Overexpression or     degraded by the ubiquitin&#226;&#8364;&#8220;proteasome pathway at restrictive <br /> deletion of the UBC6 gene suppressed the temperature-sensitive growth           temperature. We speculated that reducing protein degradation <br /> phenotype of stt3-7 mutant cells. Isogenic stt3-7 single- and double-mutant     by deleting the UBC6 gene increased the steady-state level of <br /> cells were plated in serial dilutions on rich medium and grown at 23 and 30&#194;&#176;C   Ost3p and Ost6p as well as mutant Stt3-7 protein, permitting the <br /> for 3 days. (B) Overexpression or deletion of the UBC6 gene stabilized the <br /> mutated Stt3-7 protein. Cells were shifted to 37&#194;&#176;C and protein was extracted.   integration of Ost3p, Ost6p and Stt3-7p into a partially functional <br /> Steady-state protein levels and CPY glycosylation were determined by            oligosaccharyltransferase complex. This partially restored enzymatic <br /> western analysis using the indicated antisera. Hexokinase was used as the       activity then increases N-linked glycosylation to allow mutant <br /> control for equal protein loading.                                              cells to survive at restrictive temperature. <br />  <br />                                                                                       Deletion of a gene (HTM1) homologous to <br />                                                                                      mannosidases suppresses the stt3-7 <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> nascent polypeptides at N-X-S/T, is composed of nine <br /> membrane proteins (Knauer and Lehle, 1999), of which Stt3p is <br />                                                                                       and stabilizes misfolded protein in the ER <br /> the largest subunit and is also essential for the N-linked glyco-               UBC6-dependent growth of stt3-7 strains is a convenient <br /> sylation process in vivo (Zufferey et al., 1995). Assembly of the               method to monitor the effect of mutant genes in this process. If <br /> OST complex is a multistep process, and integration of either the               reduced protein degradation can rescue these mutant cells, we <br /> Ost3p or Ost6p subunit is dependent upon the presence of Stt3p                  reasoned that we might find similar results upon creating the <br /> (U. Spirig, D. Bodmer, M. Wacker and M. Aebi, in preparation).                  stt3-7 &#226;&#710;&#8224;mns1 or stt3-7 &#226;&#710;&#8224;htm1 double mutants since Stt3 protein <br /> The stt3-7 conditional mutant has been shown to severely impair                 is N-glycosylated. In validation of this assay, &#226;&#710;&#8224;mns1 suppressed <br />  <br />  <br />                                                                                                                      EMBO reports vol. 2 | no. 5 | 2001 427 <br />                                                                  scientific reports <br />      C.A. Jakob et al. <br />  <br />      Table I. Yeast strains used <br />  <br />      Strain                     Genotype                                                                                  Reference <br />      SS328                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801                                                   Vijayraghavan et al. (1989) <br />      YG543                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 leu2 stt3-7                                       Spirig et al. (1997) <br />      YG885                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;ubc6::HIS3                                       this study <br />      YG950                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 &#226;&#710;&#8224;mns1::KanMX                               this study <br />      YG894                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 leu2 stt3-7 Dubc6::HIS3                           this study <br />      YG618                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 prc1-1                                            Jakob et al. (1998) <br />      YG777                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 tyr1 &#226;&#710;&#8224;mns1::KanMX prc1-1                                   Jakob et al. (1998) <br />      YG807                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;alg12::KanMX prc1-1                              Jakob et al. (1998) <br />      YG1211                     MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;mns1::KanMX prc1-1                               this study <br />      YCJ1                       MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;htm1::KanMX prc1-1                               this study <br />      YCJ82                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 Dhtm1::KanMX                               this study <br />      YCJ88                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 Dhtm1::KanMX                               this study <br />      RSY533                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52                                          R. Schekman <br />      YG1350                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;ubc6::KanMX                             this study <br />      YG1351                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;ubc7::KanMX                             this study <br />      YG1352                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;mns1::KanMX                             this study <br />      YG1353                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;htm1::KanMX                             this study <br />  <br />  <br />  <br />  <br />      Table II. Primers for gene deletions <br />  <br />      Deletion                          Name                                Sequence 5&#226;&#8364;&#178; &#226;&#8224;&#8217; 3&#226;&#8364;&#178;a <br />      HTM1                              HR204forKan                         tcgatcatgtagcatgcgaagacgatgcgtactcattcacttctatcgatgaattcgagctc <br />                                        HR204revKan                         gccattggaagtgagcacaggactatgtttcttgatttgtacaccgtacgctgcaggtcgac <br />                                        HR204&#226;&#8364;&#8220;90u                           gcaataaagaaggacgcg <br />                                        HR204+350L                          ccggtcttccagtatagc <br />      UBC6                              UBC6forKan                          gatggtggaaaaccctccaccatatattcttgctcgccccaacgaagattcgatgaattcgagctc <br />                                        UBC6revKan                          tgaagaactatcattaggttctttgccatctttcttggaattattttctgcgtacgctgcaggtcgac <br />                                        UBC6&#226;&#8364;&#8220;180u                           actaccatcgcatatcgc <br />                                        UBC6+130L                           gttatcattctgatagccg <br />      UBC7                              UBC7forKan                          ctcctcaaggagcttcaacagttaattaaagattctccacctggtatatcgatgaattcgagctc <br />                                        UBC7revKan                          ctcactcaacatgctcataacacttaatagaattttttctacactttgcgtacgctgcaggtcgac <br />                                        UBC7&#226;&#8364;&#8220;70u                            ggcgtttagcgtacgaag <br />                                        UBC7+130L                           agcgtatggcgtatctgg <br />      All                               KanMXu                              gtattgatgttggacgag <br />  <br />      aBold    letters represent sequences specific to plasmid pFA6aKanMX4 for the amplification of the KanMX resistance gene (Wach et al., 1994). <br />  <br />  <br />  <br />      the temperature sensitivity of stt3-7 cells at 30&#194;&#176;C, indicating a                   this process. Detailed biochemical studies will be required to <br />      direct role of N-glycan trimming in the degradation of Stt3-7                       confirm this hypothesis. <br />      protein (Figure 6). Moreover, the stt3-7 &#226;&#710;&#8224;htm1 double mutant <br />      could grow at elevated temperature, similar to the stt3-7 &#226;&#710;&#8224;mns1 <br />      strain (Figure 6). This result further supported the interpretation <br />                                                                                                                      METHODS <br />      that Htm1 protein plays an important role in efficient degradation                  Materials. Yeast strains used are detailed in Table I. The strains <br />      of the mutant glycoprotein Stt3-7p.                                                 harboring the prc1-1 allele were described previously (Jakob <br />         Our phenotypic analysis of &#226;&#710;&#8224;htm1 mutant strains is compatible                    et al., 1998). The pRS313 HA-pdr5-26 plasmid was kindly <br />      with the idea that Htm1 protein is in fact the lectin involved in                   provided by Dr D.H. Wolf (University of Stuttgart, Germany). <br />  <br />  <br />                                                                                            <br />  <br /> For western blot analysis, antisera raised against CPY, Sec61p,                <br /> Ost3p, Ost6p (Dr R. Knauer, University of Regensburg,                             <br /> Germany), hexokinase (Dr S. Schr&#195;&#182;der, G&#195;&#182;ttingen, Germany)                           <br /> and anti-hemagglutinin (anti-HA) (Santa Cruz, CA) were used.                     <br /> Yeast manipulations. Standard protocols were followed for yeast <br />                                                                             <br /> manipulation (Guthrie and Fink, 1991). Cells were grown at                          <br /> permissive temperature in either YPD (2% bactopeptone, 1%                          <br /> yeast extract, 2% glucose) or, for maintaining plasmids, in                    <br /> minimal medium (0.67% yeast nitrogen <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span>, 2% glucose, with               <br /> the appropriate supplements to allow growth). Whole-cell                             <br /> protein was extracted as previously described (Jakob et al., 1998).              <br /> Extracts containing the HA-tagged Pdr5* gene were incubated at                <br /> 50&#194;&#176;C for 15 min instead of boiled. The cycloheximide chase was                    <br /> performed as described by Plemper et al. (1998). Serial dilutions                <br /> of cells for growth assays, starting at 5 &#195;&#8212; 105 cells, were spotted <br />                                                                                     <br /> on YPD plates and incubated at the given temperatures for <br />                                                                             <br /> Construction of yeast strains. The loci YHR204w/HTM1, UBC6                     <br /> and UBC7 were deleted by replacing the major part of the gene                  <br /> with the KanMX cassette (Wach et al., 1994) using the primers                        <br /> given in Table II. The correct integration was verified by PCR.                   <br /> Deletion of the MNS1 gene was described previously (Jakob et                       <br /> al., 1998).                                                                     <br /> Labeling of N-linked oligosaccharides. The procedures for the                         <br /> metabolic labeling, recovery of glycoproteins and release of                   <br /> N-glycans by treatment with PNGase F (Roche Diagnostics, <br />                                                                                      <br /> Rotkreuz, Switzerland) were as described previously (Jakob et <br />                                                                         <br /> al., 1998).                                                                            <br />   <br /> </body></html>