<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />                            <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>Mutation</span> of Drosophila Lsd1 disrupts H3-K4 methylation resulting <br />                            in tissue specific defects during development <br />  <br />   <br />                            Summary <br />                                 Histone tail modifications play a fundamental role in the processes that establish chromatin structure <br />                                 and determine gene expression [1&#226;&#8364;&#8220;4]. One such modification, histone methylation was considered <br />                                 irreversible until the recent discovery of histone demethylases. Lsd1 was the first demethylase <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> be <br />                                 identified [5]. Lsd1 is highly conserved <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to humans but its function has primarily been <br />                                 studied using biochemical approaches. The mammalian ortholog has been shown to demethylate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 mono and dimethyl K4 and K9 residues of histone H3 [5,6]. Here we describe the effects of Lsd1 <br />                                 mutation in Drosophila. The inactivation of dLsd1 strongly affects the global level of mono and <br />                                 dimethyl-H3-K4 methylation and results in elevated expression of a subset of genes. dLsd1 is not an <br />                                 essential gene, but animal viability is strongly reduced in mutant animals in a gender specific manner. <br />                                 Interestingly, dLsd1 mutants are sterile and possess defects in ovary development, indicating that <br />                                 dLsd1 has tissue specific functions. Mutant alleles of dLsd1 suppress positional effect variegation, <br />                                 suggesting a disruption of the balance between euchromatin and heterochromatin. Taken together, <br />                                 these results show that dLsd1-mediated H3-K4 demethylation has a significant and specific role <br />                                 during Drosophila development. <br />  <br />  <br />                            Results and Discussion <br />                                               Originally Lsd1 was found as a component of co-repressor complexes [7&#226;&#8364;&#8220;11]. Lsd1 <br />                                               demethylase activity was only discovered recently [5] and was found to be modulated by its <br />                                               associated proteins, such as CoREST [12,13]. Lsd1 depletion in mammalian cells correlates <br />                                               with increased gene expression and elevated levels of H3-K4 methylation at target promoters <br />                                               [5]. However Lsd1 can also act as a co-activator and demethylates H3-K9, a repressive mark <br />                                               [6]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Lsd1 is evolutionary conserved [5] but little is known about its biological function. To address <br />                                               this question we have generated flies carrying a mutation in the sole Drosophila gene, <br />                                               CG17149/dLsd1, that exhibits high homology to Lsd1 (Fig. S1A). dLsd1 contains both a <br />                                               putative amine oxidase domain and a SWIRM domain (Fig. S1B). In the Exelixis collection <br />                                               of mutants [14] we found two piggyBac insertions in the vicinity of CG17149/dLsd1: <br />                                               f03544 (designated as dLsd11) and f00678 (dLsd12). Using FRT sites in the piggyBac <br />                                               transposon [15] to promote trans recombination between dLsd11 and dLsd12 we generated a <br />                                               deletion allele of dLsd1, dLsd1&#206;&#8221;N (Fig. S1B,C). <br />  <br />  <br />  <br />  <br />                            *Corresponding author: Nicholas Dyson, Email: dyson@helix.mgh.harvard.edu, Fax: +1-617-726-7808, Tel: +1-617-726-7800 <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers <br />                            we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting <br />                            proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could <br />                            affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Di Stefano et al.                                                                                               Page 2 <br />  <br />  <br />                                                Southern blot analysis confirmed the authenticity of the dLsd1 alleles (Fig. S2A, S2B). <br />                                                dLsd1&#206;&#8221;N lacks the presumptive promoter region and the N-terminal portion of the gene, <br />                                                including the SWIRM domain (Fig. S1B). Quantitative PCR analysis using primers specific <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                for the 5&#226;&#8364;&#178; end of dLsd1 confirmed the absence of these sequences in dLsd1&#206;&#8221;N homozygous flies <br />                                                (Fig. S2C). Low levels (&lt;20%) of 3&#226;&#8364;&#178; transcripts persist in the mutant animals (data not shown) <br />                                                but any potential products would lack the putative nuclear localization signal and the SWIRM <br />                                                domain and are unlikely to be functional. The SWIRM domain is thought to function in protein- <br />                                                protein interactions, DNA protein interaction and enzyme catalysis [16&#226;&#8364;&#8220;19]. Inactivation of <br />                                                this domain greatly reduces the stability and demethylase activity of Lsd1 [16,17]. Western <br />                                                blot analysis showed that dLsd1 is expressed at high levels in wild-type (wt) flies but no dLsd1 <br />                                                protein was detected in dLsd1&#206;&#8221;N homozygous flies (Fig. S2D). Hence dLsd1&#206;&#8221;N is, most likely, <br />                                                a <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>null allele</span>. <br />  <br />                                                This collection of mutant alleles provided us with the opportunity to study the biological <br />                                                function of dLsd1 in an animal model system. <br />  <br />                                                First, we assessed the effects of dLsd1 mutation on viability. Crosses of dLsd1&#206;&#8221;N heterozygous <br />                                                animals gave only one-third of the expected number of homozygous dLsd1&#206;&#8221;N progeny (Table <br />                                                S1). Interestingly, this reduction in viability is more dramatic in the male progeny <br />                                                (approximately 90% of the viable dLsd1&#206;&#8221;N homozygotes were females) (Table S2). <br />  <br />                                                dLsd1&#206;&#8221;N mutants are sterile. In these animals, ovary development is severely impaired (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                1C,D). The Drosophila ovary consists of approximately 16 ovarioles, chains of developing <br />                                                egg-chambers with a germarium at the anterior tip. The germarium contains germline stem <br />                                                cells (GSC) and somatic stem cells (SSC) which give rise respectively to the germline cysts <br />                                                and to follicle cells (Fig. 1C,E,G,I,M) [20]. Interestingly, DNA staining shows that dLsd&#206;&#8221;N <br />                                                mutant ovaries lack proper ovariole structures (Fig. 1D,F) and the formation of egg chambers <br />                                                is abnormal at very early stages. Both the germline and follicle cells appear abnormal (Fig. 1H, <br />                                                L, N) and, strikingly, the 16 cells cysts fail to be properly encapsulated by follicle cells (Fig. <br />                                                1H). In males, the testes are morphologically intact, but DNA staining suggests defects during <br />                                                spermatogenesis (data not shown). Interestingly, dLsd1&#206;&#8221;N homozygotes also have a held-out <br />                                                wing <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. 1B) that renders them unable to fly. <br />  <br />                                                To confirm that these defects are due specifically to dLsd1 loss and are not the result of <br />                                                secondary mutations, we performed complementation tests with a deficiency (Df(3L) <br />                                                ED4858) that uncovers the dLsd1 gene (Table 3). Trans-heterozygotes carrying dLsd1&#206;&#8221;N and <br />                                                Df(3L)ED4858 recapitulated the phenotypes observed in dLsd1&#206;&#8221;N homozygous flies (Table <br />                                                S3). <br />  <br />                                                We conclude that dLsd1 mutation reduces viability in a gender-dependent manner, causes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                abnormal ovary development, and results in animal sterility. Collectively, these results point <br />                                                to important roles for dLsd1 in the late stages of development. dLsd1 levels are highest in the <br />                                                embryonic stages (Fig. S2E,F), suggesting that dLsd1 might also have functions during early <br />                                                stages of development that may be masked in the homozygous dLsd1 mutants by maternally <br />                                                supplied products. <br />  <br />                                                Biochemical studies with human Lsd1 have led to reports that Lsd1 can act both as a co- <br />                                                repressor of transcription by demethylating H3-K4 [5], and as a co-activator by demethylating <br />                                                H3-K9 [6]. To ask which of these potential activities is predominant in Drosophila, we <br />                                                examined the levels of histone modification in dLsd1 mutant flies. The global level of mono <br />                                                and dimethyl-H3-K4 was considerably higher in dLsd1 mutants than in wild-type flies (Fig. <br />                                                2A,B); this effect was particularly striking in adult males (Fig. 2B). In contrast, we found no <br />                                                increase in the global levels of methyl-H3-K9, indeed the level of dimethyl and trimethyl-H3- <br />                                                K9 decreased slightly in dLsd1 mutants (Fig. 2A,B). The levels of monomethyl-H4-K20, <br />  <br />                                                       Curr Biol. Author manuscript; available in PMC 2008 May 1. <br />                             Di Stefano et al.                                                                                             Page 3 <br />  <br />  <br />                                                dimethyl-H3-K27, dimethyl-H4-K20, trimethyl-H3-K36 and acetyl-H3-K9 were unaltered in <br />                                                dLsd1&#206;&#8221;N mutant flies (Fig. 2A). Interestingly, ovaries contain a higher level of dLsd1 and a <br />                                                lower level of monomethyl-H3-K4 than the rest of the adult female (Fig. 2C), potentially <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                explaining why dLsd1 mutation selectively perturbs ovary development. <br />  <br />                                                As a further test, we generated UAST-dLsd1 transgenic flies and found that increased levels of <br />                                                dLsd1 reduced di and monomethyl-H3-K4 (Fig. 2D), confirming that dLsd1 is a critical <br />                                                determinant of the global level of H3-K4 methylation. Surprisingly, these animals lacked any <br />                                                clear developmental defects. <br />  <br />                                                H3-K4 methyl residues are highly enriched in euchromatin. To test whether the elevated levels <br />                                                of mono and dimethyl-H3-K4 in dLsd1 mutants alter the balance between euchromatin and <br />                                                heterochromatin, we used three variegating systems (T(2;3)Stubblevariegated (Sbv), In(1) <br />                                                y3Pyellow and In(1)whitem4h) and asked whether dLsd1 alleles modify positional effect <br />                                                variegation (PEV). PEV is the mosaic <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of gene silencing observed when genes that are <br />                                                normally located in euchromatin regions are placed into a heterochromatic environment [21]. <br />                                                Suppressors of PEV include mutants of heterochromatin-associated proteins, such as HP1 and <br />                                                the histone H3-K9 methyltransferase Su(var)3&#226;&#8364;&#8220;9 [22]. <br />  <br />                                                T(2;3)Sbv translocation juxtaposes the Sb mutation and the centric heterochromatin of the <br />                                                second chromosome, resulting in mosaic flies with both Sb and normal bristles [23,24]. <br />                                                Activation of dominant Sb results in stubble bristles. When T(2;3)Sbv was crossed to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                dLsd1&#206;&#8221;N, we observed a significant increase in the frequency of Sb bristles (Fig. 3A). Similar <br />                                                results were found using the yellow locus [25,26]. Analysis of bristles in the wing of In(1) <br />                                                y3P/+; +; dLsd1&#206;&#8221;N/+ flies (Fig. 5B) showed that a single allele of dLsd1&#206;&#8221;N or dLsd12 <br />                                                suppressed yellow variegation, giving a 25% reduction of yellow bristles (Fig. 3C). Suppression <br />                                                of variegation was also observed using wm4h (Fig. S3). These results indicate that dLsd1 alleles <br />                                                are potent suppressors of PEV and suggest that the absence of dLsd1 alters chromatin structure. <br />  <br />                                                Mono and dimethyl forms of H3-K4 are linked to transcriptional activation; the increased level <br />                                                of these modifications in dLsd1&#206;&#8221;N mutants suggests that dLsd1 normally represses <br />                                                transcription. Previous studies with human cells have shown that Lsd1 regulates the expression <br />                                                of neuron-specific genes [12]. To ask whether this function is conserved, we depleted dLsd1 <br />                                                from SL2 cells using RNAi and examined the expression of neuron-specific genes, such as <br />                                                Nicotinic Acetylcholine Receptor beta (nAcr&#206;&#178;) and Na channel (Nach). dLsd1 depletion <br />                                                increased the level of monomethyl-H3-K4 and increased the expression of both nAcr&#206;&#178; and <br />                                                Nach (Fig. 4A), indicating that Lsd1&#226;&#8364;&#8482;s role in the repression of neuron-specific genes is <br />                                                conserved between Drosophila and humans. <br />  <br />                                                The homeobox (Hox) gene locus is subject to extensive H3-K4 methylation by Trithorax group <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                proteins [27]. We therefore asked whether the expression level of the Hox genes Ultrabithorax <br />                                                (Ubx) and abdominal-A (abdA), is affected by dLsd1 depletion. Ubx and abdA mRNA levels <br />                                                increased two-fold in SL2 cells treated with dLsd1 double stranded RNA (dsRNA) (Fig. 4A). <br />                                                These changes were specific and were not seen with other control genes (dDP and Hid). To <br />                                                verify the relevance of these observations in vivo we compared the expression of these genes <br />                                                in wild-type and dLsd1&#206;&#8221;N mutant flies. A significant upregulation of each of these targets was <br />                                                found in dLsd1&#206;&#8221;N mutant flies (Fig. 4B and data not shown), confirming the importance of <br />                                                dLsd1-mediated repression in vivo. Intriguingly we observe that this upregulation is age <br />                                                dependent: the difference in gene expression is minimal in larval stages, consistent with this <br />                                                the Hox gene expression pattern in imaginal discs from dLsd1&#206;&#8221;N mutant larvae and in embryos <br />                                                is largely unaltered (Fig. S4). However the level of nAcr&#206;&#178;, Ubx and Abd-B gradually and <br />                                                significantly increases with age after eclosion (Fig. 4B), suggesting that dLsd1 function is <br />                                                especially important for the regulation of gene expression in adult tissues. <br />  <br />  <br />                                                       Curr Biol. Author manuscript; available in PMC 2008 May 1. <br />                             Di Stefano et al.                                                                                                            Page 4 <br />  <br />  <br />                                                Using Drosophila melanogaster, we describe the consequences of eliminating Lsd1 function <br />                                                in an animal model. The results help to place the previously described biochemical activities <br />                                                of Lsd1 into a biological context and show that dLsd1 is a key determinant of the global levels <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                of mono and dimethyl-H3-K4 in vivo. dLsd1 mutation impacts heterochromatin homeostasis <br />                                                and leads to ectopic expression of a subset of genes in flies. Curiously we observe that the <br />                                                consequences of the global upregulation of H3-K4 methylation on animal development are <br />                                                restricted to a few specific organs. One of the clearest defects is in the ovary where dLsd1&#206;&#8221;N <br />                                                mutant egg chambers are abnormal early in development and follicle cells fail to encapsulate <br />                                                the cyst. This defect is consistent with the observation that dLsd1 is highly expressed in the <br />                                                ovary, and indicates that dLsd1 mediated demethylation of H3-K4 plays a crucial role in this <br />                                                organ. We suggest that, in dLsd1&#206;&#8221;N mutants, the elevated expression level of dLsd1 target <br />                                                genes causes tissue specific defects. We note that both Hox genes and neuron-specific genes <br />                                                are among the target upregulated in dLsd1&#206;&#8221;N mutants. The upregulation occurs late in <br />                                                development and this may explain the lack of homeotic phenotypes or other early <br />                                                developmental defects in dLsd1&#206;&#8221;N mutants. The stronger changes in histone modification seen <br />                                                in dLsd1&#206;&#8221;N mutant males compared to females and the sex dependent lethality caused by <br />                                                dLsd1 mutation raise the possibility that dLsd1 may have a sex-specific distribution on <br />                                                chromatin alternatively its mutation may alter the chromatin distribution of the male specific <br />                                                lethal complex. <br />  <br />                                                Whereas mammalian Lsd1 has been shown to be able to demethylate H3-K9 [6], we saw no <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                increase in the global level of dimethylated H3-K9. This indicates that this activity is either <br />                                                not conserved in Drosophila, or that is redundant with other demethylases, or that is restricted <br />                                                to specific loci and/or specific tissues. <br />  <br />                                                This study opens the road to further studies aimed at delineating the specific functions of dLsd1 <br />                                                in the control of gene expression. Genome-wide studies will be necessary to identify all the <br />                                                dLsd1 regulated genes and to identify the target genes responsible for each of the developmental <br />                                                defects. <br />  <br />                                                One important implication of the tissue specific defects seen in the dLsd1&#206;&#8221;N mutant animals <br />                                                is the possibility that dLsd1 may be manipulated in vivo to modulate specific biological <br />                                                processes. The results support the idea that global changes in the levels of histone methylation <br />                                                can impact specific developmental processes. These results also highlight the need for <br />                                                additional studies to understand how histone methylation is dynamically regulated in vivo and <br />                                                how these changes contribute to normal development and disease. <br />  <br />                               Supplementary Material <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Refer to Web version on PubMed Central for supplementary material. <br />  <br />                               Acknowledgements <br />                                                We would like to thank Spyros Artanavis-Tsakonas and Douglas Dimlich for providing fly stocks and members of <br />                                                the Dyson laboratory for technical assistance and helpful discussion. We also thank Doug Renny for technical <br />                                                assistance in the generation of dLsd1 transgenic lines. We thank Laurel Raftery and Xiaodong Wu for insightful <br />                                                discussion and Anders N&#195;&#164;&#195;&#164;r, Bradley Bernstein and Martha Betson for critical comments on the manuscript. This <br />                                                study was supported by NIH grants PO1CA095281, GM53203 and CA64402 (to N.D). N.S.M was supported by <br />                                                Fellowships from the Canadian Institute of Health Research and the MGH Fund for Medical Discovery. J.Y.J. was <br />                                                supported by a Tosteson Postdoctoral Fellowship from the Massachusetts Biomedical Research Corporation. <br />  <br />  <br />   <br />  <br />  <br />  <br />                                                Figure 1. dLsd1&#206;&#8221;N mutant animals have developmental defects <br />                                                A, B: dLsd1&#206;&#8221;N mutant flies have a held-out wing phenotype. Images of wild-type (w1118) and <br />                                                dLsd1&#206;&#8221;N flies. C, D: dLsd1&#206;&#8221;N mutant ovaries are smaller than wild-type ovaries. Images of the <br />                                                ovaries of wild-type and dLsd1&#206;&#8221;N flies. E, F: dLsd1&#206;&#8221;N mutant ovaries lack proper ovariole <br />                                                structures. DNA staining (YOYO-1) of a wild-type ovary versus the ovary of dLsd1&#206;&#8221;N flies. <br />                                                G, H: In dLsd1&#206;&#8221;N mutant ovaries follicle cells fail to encapsulate the cyst. Wild-type and <br />                                                dLsd1 mutant ovarioles were stained with YOYO-1 (green) and anti-fasiclin III (red) to outline <br />                                                somatic cells. I, L, M, N: Egg chambers of dLsd1&#206;&#8221;N mutant ovaries are abnormal at very early <br />                                                stages. Wild-type and dLsd1&#206;&#8221;N mutant ovarioles were stained with anti-&#206;&#177;spectrin (white), to <br />                                                show the fusomes (I,L) and anti-vasa (white) to <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> the germ cells (M,N). The germarium is <br />                                                indicated by arrows. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />  <br />  <br />  <br />                                                Figure 2. dLsd1 specifically affects global levels of H3-K4 mono and dimethylation <br />                                                A: Western blot analysis of protein extracts from w1118(wt), dLsd1&#206;&#8221;N/dLsd1&#206;&#8221;N (hom) and <br />                                                dLsd1&#206;&#8221;N/TM3 (het) females using antibodies specific for different modified histone H3 and <br />                                                H4 residues as indicated on the right. Equal loading was assessed by blotting with either a <br />                                                histone H3 specific antibody or a tubulin specific antibody. B: Western blot analysis of protein <br />                                                extracts from w1118, dLsd1&#206;&#8221;N/dLsd1&#206;&#8221;N and dLsd1&#206;&#8221;N/TM3 males. C: dLsd1 is highly expressed <br />                                                in the ovary. Western blot analysis of dLsd1 expression and of the level of H3-K4 <br />                                                monomethylation in the ovary. dLsd1 protein level are higher in the ovary than in the rest of <br />                                                the body and this correlates with lower levels of H3-K4 monomethylation. D: Ectopic <br />                                                expression of dLsd1 affects H3-K4 methylation levels and gene expression. Ectopic expression <br />                                                of the pUAST-dLsd1 transgene was driven by the Actin5C-GAL4 driver. dLsd1 expression was <br />                                                verified by Western blot analysis. The level of mono and dimethyl H3-K4 in w1118 versus <br />  <br />                                                       Curr Biol. Author manuscript; available in PMC 2008 May 1. <br />                             Di Stefano et al.                                                                                         Page 9 <br />  <br />  <br />                                                dLsd1 overexpressing flies are shown. Equal loading was assessed using a tubulin specific <br />                                                antibody. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       Curr Biol. Author manuscript; available in PMC 2008 May 1. <br />                             Di Stefano et al.                                                                                          Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Figure 3. dLsd1 is a suppressor of PEV <br />                                                Heterozygotes for the dLsd1&#206;&#8221;N allele were crossed to either T(2;3)Sbv or In(1)y3P. The effects <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                of dLsd1&#206;&#8221;N on Stubble variegation of T(2;3)Sbv are shown in A. The effects of dLsd1&#206;&#8221;N on <br />                                                Yellow variegation are shown in B. Red asterisks indicate the yellow bristles in In(1)y3P/+;+; <br />                                                Me1/dLsd1&#206;&#8221;N flies. C: Quantification of the percentage of yellow bristles in the indicated <br />                                                genotypes. <br />  <br />  <br />  <br />  <br />                                                       Curr Biol. Author manuscript; available in PMC 2008 May 1. <br />  <br />                                                Figure 4. Neuron specific genes and Hox genes are misexpressed in SL2 cells depleted for dLsd1 <br />                                                and in dLsd1&#206;&#8221;N mutant animals <br />                                                A: Western blot analysis of SL2 cells treated for 8 days with white and dLsd1 dsRNA. Three <br />                                                different dsRNAs directed against the N-terminal (Nt), Central (Ce) or C-terminal (Ct) region <br />                                                target dLsd1 protein with the same efficacy while dsRNA against the white gene (wh), used <br />                                                as negative control, does not affect dLsd1 protein expression. qPCR analysis of the expression <br />                                                of the indicated genes 8 days after dsRNA treatment of SL2 cells. The expression level is <br />                                                normalized against the white dsRNA treated control. Experiments were performed in triplicate <br />                                                and standard deviation of the mean is indicated. B: Western blot analysis of wild-type (wt), <br />                                                dLsd1&#206;&#8221;N/dLsd1&#206;&#8221;N (hom) and dLsd1&#206;&#8221;N/TM3 (het) flies with an antibody specific for dLsd1 and <br />  <br />  <br />  <br />  <br />                                                with anti-tubulin (loading control). qPCR analysis of the expression of the indicated genes in <br />                                                wild-type (+) and dLsd1 mutant (&#226;&#710;&#8217;) third instar larvae (L3) and adult flies respectively 1, 2 and <br />                                                3 days after eclosion (A1, A2, A3). Rp49 was used as a control (data not shown). Experiments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                were performed in triplicate and standard deviation of the mean is indicated. <br />  <br /> </body></html>