<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>      Genetic Analysis of the Drosophila DNAprim Gene: The Function of the <br />         60-kD Primase Subunit of DNA Polymerase Opposes the fat facets <br />                       Signaling Pathway in the Developing Eye <br />  <br />  <br />  <br />                                                             <span id='am-3' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                The Drosophila DNAprim gene encodes the large subunit (60 kD) of DNA primase, the part of DNA <br />              polymerase &#226;&#65533;&#163; that synthesizes RNA primers during DNA replication. The precise function of the 60-kD <br />              subunit is unknown. In a mutagenesis screen for suppressors of the fat facets (faf ) mutant eye <span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span>, <br />              we identified mutations in DNAprim. The faf gene encodes a deubiquitinating enzyme required specifically <br />              for patterning the compound eye. The DNA sequences of four DNAprim alleles were determined and <br />              these define essential protein domains. We show that while flies lacking DNAprim activity are lethal, flies <br />              with reduced DNAprim activity display morphological defects in their eyes, and unlike faf mutants, cell <br />              cycle abnormalities in larval eye discs. Mechanisms by which DNA primase levels might influence the faf- <br />              dependent cell communication pathway are discussed. <br />  <br />  <br />      NA polymerase &#226;&#65533;&#163; is a highly conserved protein com- <br /> D     posed of four subunits whose approximate molec- <br /> ular weights are 180, 75, 60, and 50 kD in eukaryotes <br />                                                                                The Drosophila fat facets (faf) gene encodes a deubi- <br />                                                                             quitinating enzyme essential in only two tissues of the <br />                                                                             fly, the eye and ovary (Fischer-Vize et al. 1992; Huang <br /> (Lehman and Kaguni 1987). The 180-kD subunit con-                           et al. 1995). In the compound eye, faf is required early <br /> tains the DNA polymerase activity (Cotterill et al.                         during development for a cell communication event <br /> 1987a; Pizzagalli et al. 1988; Wong et al. 1988) and                        that limits <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> eight the number of photoreceptors in <br /> the two smallest subunits constitute DNA primase, an                        each <span id='am-4' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> eye (also called a facet or ommatidium; <br /> enzyme that synthesizes the RNA primers required dur-                       Fischer-Vize et al. 1992; Huang and Fischer-Vize <br /> ing DNA replication (Conaway and Lehman 1982;                               1996). Ubiquitin (Ub) is a highly conserved 76-amino- <br /> Cotterill et al. 1987b; Bakkenist and Cotterill                             acid peptide that can be covalently linked to other pro- <br /> 1994). The 50-kD subunit alone has been shown to be                         teins, usually via an isopeptide bond between the termi- <br /> sufficient for primase activity (Nasheuer and Grosse                        nal Glycine residue of Ub and an internal Lysine residue <br /> 1988; Santocanale et al. 1993; Bakkenist and Cot-                           of the substrate (Hershko 1998). A chain of isopeptide- <br /> terill 1994). Although its precise function is unknown,                     linked Ub moieties targets the protein to which it is <br /> the 60-kD subunit is considered to be part of DNA pri-                      attached for degradation by the proteasome (Hershko <br /> mase because it can bind ribonucleotides (Foiani et al.                     1998). <br /> 1989) and it can alter the activity of the 50-kD subunit                       Deubiquitinating enzymes cleave Ub-protein bonds <br /> (Copeland 1997).                                                            (Wilkinson and Hochstrasser 1998). Faf protein is <br />   DNA primase is of interest <span id='am-8' about='protege:FROM' typeof='owl:Thing'>from</span> a structural viewpoint                    one of a large family of deubiquitinating enzymes called <br /> and also from a regulatory perspective: how does pri-                       ubiquitin-specific processing proteases (Ubps), which <br /> mase interact with DNA polymerase and DNA and how                           are recognizable by two small conserved amino acid <br /> is the activity of DNA polymerase &#226;&#65533;&#163; coordinated with                        domains that constitute the catalytic core (Wilkinson <br /> the cell cycle? Prerequisite to answering these questions,                  and Hochstrasser 1998); the Drosophila Genome Proj- <br /> the Drosophila genes for all four DNA polymerase &#226;&#65533;&#163;                          ect has identified at least 18 Ubps in the fly genome <br /> subunits have been cloned and analyzed (Hirose et al.                       (Rubin et al. 2000; Chen and Fischer 2000). <br /> 1991; Cotterill et al. 1992; Melov et al. 1992; Bakke-                         The faf gene is of particular interest because it is re- <br /> nist and Cotterill 1994; Huikeshoven and Cot-                               quired for a specific step in eye patterning and also <br /> terill 1999).                                                               because, unique among Ubps, Faf appears to regulate <br />                                                                             the Ub pathway by deubiquitinating a specific substrate, <br />                                                                             thus preventing its degradation. The evidence for this <br />                                                                             model is genetic; mutations in a gene encoding a protea- <br />   Corresponding author: Janice A. Fischer, University of Texas, Moffett <br /> Molecular Biology Bldg., 2500 Speedway, Austin, TX 78712.                   some subunit [l(3)73Ai; Saville and Belote 1993] or <br /> E-mail: jaf@mail.utexas.edu                                                 in a gene encoding a Ub-conjugating enzyme required <br /> Genetics 156: 1787&#226;&#8364;&#8220;1795 (December 2000) <br />  1788                                               X. Chen, Q. Li and J. A. Fischer <br />  <br /> for attaching Ub to target proteins (UbcD1; Treier et                 shown in Figure 1, &#226;&#181;&#8216;15,000 mutagenized second and third <br /> al. 1992; Cenci et al. 1997; Neufeld et al. 1998) act                 chromosomes (&#226;&#181;&#8216;80% of the genome) were screened for S(faf ) <br />                                                                       mutations. All of the suppressors were tested for complementa- <br /> as strong dominant suppressors of the faf mutant eye                  tion with UbcD1 and l(3)73Ai mutations, which were shown <br /> phenotype (Wu et al. 1999). These results indicate that               previously to act as strong suppressors of faf (Huang et al. <br /> faf function antagonizes the function of ubiquitination               1995; Wu et al. 1999). <br /> and the proteasome. In addition, slight overexpression                   Mapping S(faf) mutants: Second chromsome-linked S(faf ) <br /> of a gene called liquid facets (lqf) obviates the require-            mutants were mapped roughly with respect to four P{w&#226;&#171;&#185;} <br />                                                                       insertions in 25D, 28E, 30C, and 57B as follows. Females of <br /> ment for faf in the eye; this observation is consistent with          the genotype w; S(faf )/P{w&#226;&#171;&#185;}; st faf FO8/TM6B were crossed <br /> a model where Faf protects a substrate from degradation               with w; st faf FO8/TM6B males. Approximately 100 male progeny <br /> and where that substrate is the Lqf protein (Cadavid                  with pigmented eyes [thus of the genotype w; P{w&#226;&#171;&#185;}, S(faf)?/&#226;&#171;&#185;; <br /> et al. 2000).                                                         st faf FO8/TM6B] were crossed individually with w; st faf FO8/ <br />    The lqf gene, identified in mutagenesis screens for                TM6B females. The faf FO8 progeny were scored for the pres- <br />                                                                       ence or absence of the S(faf ) <span id='am-167' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. Third chromosome <br /> dominant enhancers of faf mutations, encodes the Dro-                 mutations were localized with respect to four recessive markers <br /> sophila homolog of vertebrate epsin, a component of                   (th, cu, sr, and e) as follows. Female progeny of the genotype <br /> the endocytosis complex (Fischer et al. 1997; Chen et                 w, P{w&#226;&#171;&#185;, faf&#226;&#171;&#185;}/w; S(faf ), st faf FO8/th st cu sr e faf FO8 were crossed <br /> al. 1998; Cadavid et al. 2000). Although it is not certain            with w; th st cu sr e faf FO8/TM6B males. [P{w&#226;&#171;&#185;, faf&#226;&#171;&#185;} is a <br /> whether Lqf is the substrate of Faf in the eye, Lqf must              P-element insertion containing a wild-type faf gene that com- <br />                                                                       plements the faf mutant phenotype completely; it was used <br /> be a critical player, downstream of faf, in the faf-depen-            here to complement the sterility of faf &#226;&#171;&#186; females.] All non- <br /> dent cell communication pathway.                                      TM3, white-eyed flies [without P{w&#226;&#171;&#185;, faf&#226;&#171;&#185;}] with smooth eyes <br />    To identify additional components of the faf pathway,              [with S(faf )] were scored for the presence of the four recessive <br /> we performed a screen for dominant suppressors of the                 markers. After meiotic mapping, physical mapping of some <br /> faf mutant eye phenotype. We found that mutations in                  of the S(faf ) mutations was performed. Individual S(faf ) lines <br />                                                                       were crossed with appropriate third chromosome deficiencies <br /> DNAprim (Spradling et al. 1999), which encodes the                    and S(faf )/Df progeny were examined for lethality or a mutant <br /> 60-kD subunit of DNA primase, are strong dominant                     eye phenotype. <br /> suppressors of faf mutations. We have determined the                     Mobilization of P{lacW } j10B2: One hundred individual <br /> DNA sequences of several mutant DNAprim genes and                     crosses were performed in which two males of the genotype <br /> these reveal important functional domains of the pro-                 w/Y; P{lacW }j10B2/&#226;&#338;&#172;2-3 Sb were crossed with w; st e faf FO8/ <br />                                                                       TM6B females. Progeny with chromosomes in which the <br /> tein. In addition, we found that while strong DNAprim                 P element had hopped out were recognized as white-eyed <br /> mutations are lethal, flies with reduced DNAprim activity             flies; one white-eyed progeny male of the genotype w; <br /> have defects in eye development. Moreover, we show                    P{lacW} j10B2/TM6B from each of the 100 original crosses was <br /> that in larval eye discs with reduced DNAprim activity,               crossed to w; S(faf )240/TM3 females. The w; P{lacW } j10B2&#226;&#338;&#172;/ <br /> the cell cycle is perturbed. We discuss possible roles for            TM3 progeny were used to generate stocks of each hop-out <br />                                                                       chromosome and the other progeny were examined to deter- <br /> DNA primase in the faf-dependent cell communication                   mine if P{lacW } j10B2&#226;&#338;&#172;/S(faf )240 progeny were viable and if <br /> pathway.                                                              so, if they display a mutant phenotype. <br />                                                                          Plasmid rescue: Molecular biology procedures were carried <br />                                                                       out using standard techniques (Sambrook et al. 1989). Total <br />                MATERIALS AND METHODS                                  genomic DNA prepared from P{lacW } j10B2/&#226;&#171;&#185; flies was re- <br />                                                                       stricted to completion with EcoRI, ligated, and electroporated <br />    Drosophila strains: The Bloomington Stock Center pro-              into Escherichia coli. The plasmid recovered was digested with <br /> vided l(3)j10B2 and other lethal P{w&#226;&#171;&#185;} insertions in the region       EcoRI and XbaI to release the Drosophila genomic DNA frag- <br /> (Spradling et al. 1999), In(3L)78Cb1 (Russell et al. 1996), the       ment adjacent to the P element. The end sequences of the <br /> third chromosome deficiency kit, which includes Df(3L)rdgC-           genomic DNA fragment were determined, and as one of them <br /> co2 (Flybase 1999) and Df(3L)ri-79c (Jurgens et al. 1984), the        contains some plasmid sequence, the precise insertion point of <br /> &#226;&#338;&#172;2-3 Sb chromosome (Robertson et al. 1988) used for P-element         the P element within the genomic DNA could be determined. <br /> mobilization, the th st cu sr e chromosome, the P{w&#226;&#171;&#185;} insertion          Molecular analysis of DNAprim alleles: The DNAprim gene, <br /> in 28E used for meiotic mapping, and the Act5C-Gal4 transgene         whose open reading frame is contained in a single exon, was <br /> described in Flybase (1999). The UbcD1 (Cenci et al. 1997;            amplified by polymerase chain reaction (PCR) from genomic <br /> Neufeld et al. 1998) and l(3)73Ai (Saville and Belote 1993)           DNA prepared from the following heterozygotes: S(faf ) <br /> alleles used for complementation tests are described in Huang         240/&#226;&#171;&#185;, l(3)77ABe J1/&#226;&#171;&#185;, l(3)77ABe J9/&#226;&#171;&#185;, l(3)77ABe L5/&#226;&#171;&#185;, and l(3) <br /> et al. (1995) and Wu et al. (1999). The three alleles of l(3)77ABe    77ABeT2/&#226;&#171;&#185;. The primers used for PCR were (1) 5&#226;&#172;&#732;-ATTAC <br /> (Lukinova et al. 1999) were obtained from N. Lukinova and             TATAGCAATTCCCGCGAC-3&#226;&#172;&#732; and (2) 5&#226;&#172;&#732;CGTGTACTTGTG <br /> M. Fortini. The P{w&#226;&#171;&#185;} insertions in 25D, 30C, and 57B used            CCGGAGGTG-3&#226;&#172;&#732;. Genomic DNA was prepared by adding one <br /> for meiotic mapping were provided by K. Moses. The faf alleles        fly to a microfuge tube containing 50 &#226;&#65533;&#174;l of buffer (10 mm <br /> and P{w&#226;&#171;&#185;, faf&#226;&#171;&#185;} transformants are maintained in our labora-           Tris pH 8.2, 1 mm EDTA, 25 mm NaCl) and 1 &#226;&#65533;&#174;l of proteinase <br /> tory and are described in Fischer-Vize et al. (1992). Marker          K (20 mg/ml). The fly was homogenized with a pipette tip, <br /> mutations and balancers are described in Lindsley and Zimm            incubated at 37&#226;&#172;&#353; for 1 hr, and then at 100&#226;&#172;&#353; for 2 min to <br /> (1992). All flies were grown on standard media at 25&#226;&#172;&#353; and             inactivate the proteinase K. A 4-ml aliquot of this homogenate <br /> strains were constructed using standard cross schemes.                was used in a single PCR reaction. The PCR products, mixtures <br />    EMS mutagenesis screen: Males were mutagenized with                of two different DNA populations, were subcloned into <br /> EMS according to the method of Lewis and Bacher (1968),               pBLUESCRIPT (Stratagene, La Jolla, CA) and the DNA se- <br /> crossed to females, and progeny were collected for 4 days. As         quences of several subclones from each PCR reaction were <br />                                                     DNAprim Mutations Suppress faf                                              1789 <br />  <br /> determined. The sequences were compared with the DNA                 Triton X-100, 2N HCl, and then washed twice for 10 min in <br /> sequence in Huikeshoven and Cotterill (1999) and also                PBS &#226;&#171;&#185; 0.3% Triton X-100. The following manipulations were <br /> with those determined by the Drosophila Genome Project (ge-          carried out in 18 &#226;&#65533;&#174;l of solution in 96-well microtiter plates. <br /> nomic clone AC017679 and cDNA clone GM13640). For                    The disc complexes were incubated in primary antibody (anti- <br /> clones where DNA sequence differences from wild type were            BrdU; Becton Dickinson, San Jose, CA) diluted 1:100 in PBS &#226;&#171;&#185; <br /> detected, that sequence was confirmed by sequencing sub-             0.3% Triton X-100 and 10% goat serum overnight at 4&#226;&#172;&#353;. After <br /> clones from a second PCR reaction. The inversion breakpoint          two 10-min washes in PBS &#226;&#171;&#185; 0.3% Triton X-100, the tissues <br /> of In(3L)78Cb1 was localized to a &#226;&#181;&#8216;2.8-kb EcoRI fragment of          were incubated for 2 hr at room temperature in secondary <br /> genomic DNA that includes the first &#226;&#181;&#8216;160 bp of the DNAprim           antibody (HRP-conjugated goat anti-mouse; Vector, Burl- <br /> open reading frame and &#226;&#181;&#8216;2.6 kb of sequence upstream by               ingame, CA) diluted 1:50 in PBS &#226;&#171;&#185; 0.3% Triton X-100 and <br /> the following genomic DNA blotting experiment. A blot of             10% goat serum. After two 10-min washes in PBS &#226;&#171;&#185; 0.3% <br /> genomic DNA prepared from wild-type and In(3L)78Cb1/&#226;&#171;&#185;                Triton X-100, the signal was developed in HRP-staining solu- <br /> flies restricted to completion with EcoRI was hybridized with        tion (1&#226;&#171;&#187; PBS, 0.1% Triton X-100, 0.5 mg/ml diaminobenzi- <br /> a probe corresponding to the open reading frame of the               dine, 0.003% H2O2) for &#226;&#181;&#8216;1 min. The tissues were washed twice <br /> DNAprim gene. The level of the &#226;&#181;&#8216;2.8-kb EcoRI fragment in             for 5 min in PBS &#226;&#171;&#185; 0.3% Triton X-100, transferred to a petri <br /> the In(3L)78Cb1/&#226;&#171;&#185; DNA was half of that in wild type.                 dish containing 50% ethanol for 10 min, and then to 100% <br />    Complementation of DNAprim mutations by a UAS-DNAprim             ethanol twice for 10 min. Finally, the eye discs were dissected <br /> transgene: A 1.8-kb DNA fragment containing the DNAprim              away from the complex and mounted in DPX (Fluka, Buchs, <br /> open reading frame was generated by PCR using the Advan-             Switzerland). <br /> tage-HF2 kit (CLONTECH, Palo Alto, CA) with wild-type geno- <br /> mic DNA prepared as described above as the template and <br /> the following two primers: (1) 5&#226;&#172;&#732;-GGCGCGCCATTACTATAG                                           RESULTS <br /> CAATTCCCGCGAC-3&#226;&#172;&#732; and (2) 5&#226;&#172;&#732;-GGCGCGCCCGTGCTAC <br /> TTGTGCCGGAGGTG-3&#226;&#172;&#732;. The DNA fragment was ligated into                    Mutations in a gene at 77B6/7 are suppressors of <br /> the vector PCR2.1 (Invitrogen, San Diego) to generate PCR2.          faf mutations: To identify genes in the faf pathway, we <br /> 1-DNAprim and its DNA sequence was determined. The DNA-              performed an EMS mutagenesis screen for dominant <br /> prim gene was then isolated as a 1.8-kb AscI fragment from <br /> PCR2.1-DNAprim and ligated into the AscI site of the P-ele-          suppressors of the faf mutant phenotype in the eye (Fig- <br /> ment transformation vector pUAST-XA (Huang and Fischer-              ure 1). We screened &#226;&#181;&#8216;15,000 flies and isolated 13 sup- <br /> Vize 1996) to generate pUAS-DNAprim. One second chro-                pressor mutations [S(faf)], five of which were strong <br /> mosome-linked P-element transformant was generated with              and were analyzed further (Table 1 and Figure 2). One <br /> pUAS-DNAprim using standard procedures (Spradling                    of these is an allele of UbcD1, which was shown previously <br /> 1986) with w1118 flies as the host. Flies homozygous for l(3)j10B2 <br /> that also carry one copy each of the Act5C-Gal4 and UAS- <br />                                                                      to be a strong suppressor of faf mutations (Wu et al. 1999). <br /> DNAprim transgenes and thus express the cloned DNAprim               Of the remaining 4 S(faf) mutations, only S(faf)240 has <br /> gene ubiquitously were viable and apparently wild type except        an essential function; S(faf)240 results in a mutant pheno- <br /> that they were nearly sterile. Expression of the UAS-DNAprim         type when homozygous or in trans to a deficiency chro- <br /> transgene using the Act5C-Gal4 driver resulted in a few viable       mosome in otherwise wild-type flies (see Table 1 foot- <br /> l(3)j10B2/S(faf )240 escapers that died after 1 day. <br />                                                                      note). <br />    Phenotypic analysis of eyes: Scanning electron micrographs <br /> (Huang et al. 1995) and 1-&#226;&#65533;&#174;m plastic sections of adult eyes             The S(faf)240 allele was mapped meiotically between <br /> (Tomlinson and Ready 1987a) were prepared as described               th and cu (materials and methods). Subsequent physi- <br /> previously. Cobalt sulfide staining of pupal eyes was performed      cal mapping positioned S(faf)240 within 77A1&#226;&#8364;&#8220;77C; flies <br /> using a modified version of the procedure described in Wolff         that are S(faf)240/Df(3L)rdgC-co2 (77A1&#226;&#8364;&#8220;77D1) die early <br /> and Ready (1991) as follows. Retina/brain complexes of 40&#226;&#8364;&#8220;           during development, whereas S(faf)240/Df(3L)ri-79c <br /> 60-hr pupae grown at 22&#226;&#172;&#353; were dissected in 0.1 m NaPO4, washed <br /> in H2O, and then fixed in 2% glutaraldehyde in 0.1 m NaPO4           (77C&#226;&#8364;&#8220;78A) flies are wild type. Lethal P{w&#226;&#171;&#185;} insertions <br /> for 10 min. After washing in 0.1 m NaPO4, the retinas were           within 77A1&#226;&#8364;&#8220;77C were tested for complementation of <br /> incubated in 4% Co(NO3)2 for 10&#226;&#8364;&#8220;15 min, and then rinsed              S(faf)240, and the insertion P{lacW}j10B2 (l(3)j10B2) in <br /> thoroughly in H2O. With a fine tungsten needle, the outsides         77B6/7 (Spradling et al. 1999) failed to complement <br /> of the retinas were scraped clean of precipitates and peripodial     the lethality of S(faf)240. In addition, l(3)j10B2 and <br /> membrane. After washing again in H2O, mouth hooks and <br /> brains were removed and the retinas were mounted in 80%              Df(3L)rdgC-co2 are also dominant suppressors of faf (Fig- <br /> glycerol. All eye preparations were photographed with a Zeiss        ure 2G). Three EMS-induced lethal mutations in the <br /> Axioplan microscope.                                                 l(3)77ABe gene and In(3L)78Cb1 (78B5&#226;&#8364;&#8220;9/78C8&#226;&#8364;&#8220;9) were <br />    BrdU incorporation experiments: Eye disc/brain com-               reported to be allelic to l(3)j10B2 (Lukinova et al. 1999) <br /> plexes were dissected in Drosophila Schneider&#226;&#8364;&#8482;s medium at            and all of them are also allelic to S(faf)240 (Table 2). <br /> room temperature and then incubated for 2 hr in Schneider&#226;&#8364;&#8482;s <br />                                                                         As l(3)j10B2 does not complement the lethality of <br /> medium with 75 &#226;&#65533;&#174;g/ml bromodeoxyuridine (BrdU; Boeh- <br /> ringer Mannheim, Indianapolis) in a small petri dish. All of         S(faf)240 and both of these chromosomes as well as a <br /> the following manipulations were also carried out in small           deficiency that uncovers each lethal mutation suppress <br /> petri dishes. Disc complexes were washed 10 min in fresh             faf, the mutant gene causing the lethality is most likely <br /> Schneider&#226;&#8364;&#8482;s, twice for 10 min in PBS, and then fixed in Car-         also causing the suppression of faf mutations. To test <br /> noy&#226;&#8364;&#8482;s fixative (60% ethanol, 10% glacial acetic acid, 30% chlo-      whether the lethality of l(3)j10B2 is due to the P{lacW} <br /> roform) for 30 min. The fixed tissue was rehydrated with <br /> an ethanol series (5 min each in 70, 50, and 30%), then              insertion, the P element was mobilized and 100 chromo- <br /> permeabilized in PBS &#226;&#171;&#185; 0.6% Triton X-100 for 30 min. The             somes from which it had hopped out were isolated (ma- <br /> DNA was denatured by a 30-min incubation in PBS &#226;&#171;&#185; 0.3%               terials and methods). Of these 100 chromosomes, 57 <br />  1790                                              X. Chen, Q. Li and J. A. Fischer <br />  <br />  <br />  <br />  <br />                                                                                    Figure 1.&#226;&#8364;&#8221;Cross scheme for suppressor of faf <br />                                                                                 mutagenesis screen. All of the markers are de- <br />                                                                                 scribed in Lindsley and Zimm (1992). The aster- <br />                                                                                 isk (*) indicates a mutagenized chromosome. The <br />                                                                                 faf FO8 allele is a strong mutant; its eye phenotype <br />                                                                                 when homozygous or in trans to a deficiency is <br />                                                                                 nearly indistinguishable from that of the null al- <br />                                                                                 lele faf BX4 (Fischer-Vize et al. 1992). The DNA <br />                                                                                 lesion of faf FO8 results in a single amino acid <br />                                                                                 change; the second His of Faf &#226;&#8364;&#8482;s catalytic domain <br />                                                                                 is changed to Tyr (Chen and Fischer 2000). This <br />                                                                                 His residue is required for wild-type Faf function, <br />                                                                                 but may not be absolutely essential (Huang et al. <br />                                                                                 1995). <br />  <br />  <br />  <br />  <br /> of them, presumably precise excisions of the P element,              P{lacW}j10B2, and the viable chromosomes are smaller <br /> resulted in viable flies with no obvious mutant pheno-               deletions that disrupt the gene partially (see below). <br /> types when homozygous or in trans to S(faf)240. Thus,                Two of the lethal imprecise excision chromosomes and <br /> the P-element insertion was the cause of the lethal muta-            three of the viable ones were kept for further analysis <br /> tion allelic to S(faf)240 on the chromosome containing               (Table 2). <br /> P{lacW}j10B2. The other 43 hop-out chromosomes, pre-                    The S(faf) gene at 77B6/7 is DNAprim: To isolate <br /> sumably imprecise excisions, fell into two groups: when              the suppressor of faf locus at 77B6/7, genomic DNA <br /> homozygous or in trans to S(faf)240, 40 chromosomes                  adjacent to P{lacW}j10B2 was isolated by plasmid rescue <br /> were lethal and 3 were viable, resulting in flies with               (materials and methods). The DNA sequence of the <br /> rough eyes. Presumably, the lethal chromosomes are                   cloned genomic DNA immediately adjacent to the P <br /> deletions of most or all of the gene disrupted by                    element was determined and compared to the Drosoph- <br />                                                                      ila genomic and expressed sequenced tag DNA se- <br />                                                                      quence databases (materials and methods). This <br />                            TABLE 1                                   analysis revealed that the P element is inserted in the <br />                       Suppressors of faf                             5&#226;&#172;&#732;-untranslated region of the DNAprim gene. To test if <br />                                                                      the S(faf)240 and the l(3)77ABe mutations are also in <br /> Suppressors        Location                 Comments                 the DNAprim gene, the DNA sequences of their DNAprim <br /> S(faf )86       25D&#226;&#8364;&#8220;30C             Homozygotes wild type <br />                                                                      genes were determined (materials and methods). All <br /> S(faf )184      88D2/3              Allelic to UbcD1                 of the DNAprim alleles in these mutants have suffered <br /> S(faf )196      near th             Wild type in trans to Df         DNA lesions likely to result in loss of function (Figure <br /> S(faf )240      77B6/7              Allelic to DNAprim               3). In addition, it was determined in genomic DNA <br /> S(faf )244      Chromosome 3        No recombination with th         blot experiments that one breakpoint of In(3L)78Cb1 is <br />                                       cu sr e chromosome             within 2.6 kb of the start of transcription of DNAprim <br />    The S(faf )86 and S(faf )196 mutations were localized impre-      (data not shown; see materials and methods). Finally, <br /> cisely because it was discovered during the mapping process          a DNAprim transgene expressed ubiquitiously using the <br /> that these mutations are likely to be in genes without essential     Gal4/UAS system (Brand and Perrimon 1993) rescues <br /> functions (the mutants are homozygous viable), which ren-            the lethality of l(3)j10B2 homozygotes (see materials <br /> dered them difficult to map physically. The S(faf )244 mutation      and methods). Thus, we conclude that mutations in <br /> could not be mapped meiotically as no recombination could <br /> be observed between this third chromosome and the mapping            DNAprim are suppressors of faf. We refer to S(faf)240, <br /> chromosome. The S(faf ) mutations are not dominant suppres-          l(3)j10B2, and the three l(3)77ABe alleles as DNAprim <br /> sors of the sterility of faf FO8 females.                            alleles from here on (Table 2). <br />                                                 DNAprim Mutations Suppress faf                                               1791 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 2.&#226;&#8364;&#8221;Suppression of the faf mu- <br />                                                                                   tant eye phenotype. Scanning electron mi- <br />                                                                                   crographs (A&#226;&#8364;&#8220;C) and tangential sections <br />                                                                                   (D&#226;&#8364;&#8220;F) of Drosophila adult eyes are shown. <br />                                                                                   Anterior is to the left in A&#226;&#8364;&#8220;C. The geno- <br />                                                                                   types are as follows: (A and D) wild type; <br />                                                                                   (B and E) faf FO8; and (C and F) S(faf )240 <br />                                                                                   faf FO8/&#226;&#171;&#185; faf FO8. The numbers in D indicate <br />                                                                                   photoreceptor cells R1&#226;&#8364;&#8220;R7. The central R8 <br />                                                                                   cell is not visible in this apical plane of <br />                                                                                   section. (E) Each facet contains more than <br />                                                                                   the normal number of photoreceptors. <br />                                                                                   (G) A histogram showing the dominant <br />                                                                                   suppressor of faf activity of three different <br />                                                                                   DNAprim mutant alleles. For comparison, <br />                                                                                   the extent to which UbcD1 and l(3)73Ai <br />                                                                                   suppress faf FO8 is also shown (Huang et al. <br />                                                                                   1995; Wu et al. 1999). The average fraction <br />                                                                                   of wild-type facets was calculated by scoring <br />                                                                                   the number of wild-type and mutant facets <br />                                                                                   in at least 100 facets in at least three differ- <br />                                                                                   ent flies of each genotype. The error bars <br />                                                                                   indicate standard deviations, which repre- <br />                                                                                   sent the variation between individuals of a <br />                                                                                   given genotype. <br />  <br />  <br />  <br />  <br />    Weak DNAprim mutants have eye defects: Eight of              due to subtle defects in cone cells, pigment cells, and/ <br /> the 11 DNAprim alleles, including In(3L)78Cb1, are early        or bristles in the final stages of eye development; absent <br /> lethal in trans to Dfr(3L)rdgC-co2 or S(faf)240 (Table 2).      bristles and bristle misplacements are apparent in the <br /> As In(3L)78Cb1 breaks within the coding region of DNA-          adult eyes (Figure 4B). <br /> prim, the other 7 alleles are also likely to be <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>null mutants</span>.      The cell cycle is abnormal in DNAprim mutants: To <br /> A weaker allele, DNAprimT2, is pupal lethal in trans to         understand the basis for the genetic interactions ob- <br /> Dfr(3L)rdgC-co2 or S(faf)240. There is cell death in the        served between faf and DNAprim, we asked whether the <br /> eye anterior of late-stage pupae of these genotypes (Fig-       cell cycle is altered in these mutants. DNAprim mutants <br /> ure 4A). The weakest alleles are the homozygous viable          would be expected to have difficulty during S phase <br /> group of imprecise excisions, exemplified by DNA-               when DNA is replicated. If S phase is slower in these <br /> prim j10B2&#226;&#338;&#172;13-15 (Table 2). When homozygous or in trans         mutants, then more cells would be expected to be in S <br /> to Df(3L)rdgC-co2, the DNAprim j10B2&#226;&#338;&#172;13-15 alleles result in    phase at any given time. During larval eye disc develop- <br /> slightly roughened external eyes (Figure 4B). Pat-              ment, undifferentiated cells anterior to the morphoge- <br /> terning defects, either in the accessory cells (cone and        netic furrow divide asynchronously and then arrest in <br /> pigment cells) of pupal eyes (Figure 4C) or in adult            G1 within the morphogenetic furrow prior to differenti- <br /> photoreceptors (Figure 4D), could not be detected in            ation. No cell division occurs again until later in the <br /> these flies. Thus, the external rough eyes are most likely      eye disc, when cells that will give rise to three of the <br />  1792                                            X. Chen, Q. Li and J. A. Fischer <br />  <br />                                                            TABLE 2 <br />                                                     Mutant DNAprim alleles <br />  <br />                                                                Phenotype in trans to <br />          Allele name (<span id='am-1' about='Thesaurus:Synonym' typeof='owl:Thing'>synonym</span>)         How induced         Df(3L)rdgC-co2 or DNAprimSfaf240            References <br />                   Sfaf240 <br />          DNAprim       (S(faf )240)   EM                          Early lethal                   This article <br />          DNAprim j10B2 (l(3)j10B2)    P{lacW } j10B2              Early lethal                   Spradling et al. (1999) <br />          DNAprim J1 (l(3)77ABe J1)    EMS                         Early lethal                   Lukinova et al. (1999) <br />          DNAprim L5 (l(3)77ABe L5)    EMS                         Early lethal                   Lukinova et al. (1999) <br />          DNAprimT2 (l(3)77ABeT2)      EMS                         Pupal lethal                   Lukinova et al. (1999) <br />          DNAprim j10B2&#226;&#338;&#172;11             P{lacW } hop-out            Early lethal                   This article <br />          DNAprim j10B2&#226;&#338;&#172;12             P{lacW } hop-out            Early lethal                   This article <br />          DNAprim j10B2&#226;&#338;&#172;13             P{lacW } hop-out            Viable; rough eyes             This article <br />          DNAprim j10B2&#226;&#338;&#172;14             P{lacW } hop-out            Viable; rough eyes             This article <br />          DNAprim j10B2&#226;&#338;&#172;15             P{lacW } hop-out            Viable; rough eyes             This article <br />          In(3L)78Cb1                  X rays                      Early lethal                   Russell et al. (1996) <br />            All known DNAprim mutant alleles are listed. The allele DNAprim J9 is not listed here because by DNA sequence <br />          analysis we found that it is the same as DNAprim J1 (Lukinova et al. 1999). <br />  <br />  <br />  <br /> eight photoreceptor types and all of the other cell types          indirect interactions between the functions of the two <br /> undergo a single mitosis (Wolff and Ready 1993). By                proteins. <br /> assaying BrdU incorporation with anti-BrdU antibodies <br /> (materials and methods), we visualized cells in S <br /> phase in larval eye discs of flies with a pupal lethal                                        DISCUSSION <br /> combination of DNAprim mutations, as well as wild-type                In a screen for dominant suppressors of the faf mutant <br /> and faf &#226;&#171;&#186; eye discs. We found that BrdU incorporation              eye phenotype, we identified mutations in the DNAprim <br /> in the faf &#226;&#171;&#186; eye discs resembled wild type (Figure 5, A            gene, which encodes the large subunit (60 kD) of DNA <br /> and B). By contrast, in the DNAprim mutants, more cells            primase, a component of DNA polymerase &#226;&#65533;&#163;. The pre- <br /> were in S phase at a given time (Figure 5C). As faf                cise function of the 60-kD primase subunit is unknown. <br /> mutants do not have defects in S phase, the genetic                The locations of the DNA lesions in several DNAprim <br /> interactions between DNAprim and faf most likely reflect           mutants have been determined; this information could <br />  <br />  <br />  <br />                                                                                   Figure 3.&#226;&#8364;&#8221;Mutations in EMS-induced DNAprim <br />                                                                                alleles. Amino acid sequence changes, derived <br />                                                                                from DNA sequence determination, in the puta- <br />                                                                                tive DNA primase proteins produced by four dif- <br />                                                                                ferent EMS-induced mutant alleles are shown. <br />                                                                                Two of the mutations are single nucleotide <br />                                                                                changes; T2 is GAG &#226;&#8224;&#8217; AAG and L5 is CAG &#226;&#8224;&#8217; <br />                                                                                TAG. Two of the mutations are deletions; J1 is an <br />                                                                                in-frame deletion of 27 bp and Sfaf240 is a dele- <br />                                                                                tion of 101 bp that results in a frameshift. The <br />                                                                                mutations in J1, L5, and Sfaf240 (strong alleles) <br />                                                                                are all in regions highly conserved in the yeast, <br />                                                                                human, and mouse genes (Huikeshoven and <br />                                                                                Cotterill 1999). T2 is a weak allele, and E138 is <br />                                                                                somewhat conserved (D in human and mouse, F <br />                                                                                in yeast; Huikeshoven and Cotterill 1999). In <br />                                                                                addition to the amino acid sequence we derived <br />                                                                                shown here, three translated amino acid se- <br />                                                                                quences for Drosophila DNA primase are avail- <br />                                                                                able: genomic clone AC017679 and cDNA clone <br />                                                                                GM13640 from the Drosophila Genome Project, <br /> and also the sequence derived by Huikeshoven and Cotterill (1999). Our translated sequence matches exactly with that <br /> derived from AC017679, except that AC017679 is missing 1 bp, which is present in the GM13640 sequence, that would interrupt <br /> the reading frame. We assume that this is an error in the AC017679 sequence. Our translated sequence differs from the translation <br /> of GM13640 in two amino acids: Y42 in our sequence is F in GM13640 and S49 in our sequence is G in GM13640. There are <br /> numerous differences between our sequence and that in Huikeshoven and Cotterill (1999). The <span id='am-6' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> <br /> for the DNA sequence we report here is AF291873. <br />                                                      DNAprim Mutations Suppress faf                                            1793 <br />  <br />                                                                       prim activity, S phase of the cell cycle is slower relative <br />                                                                       to wild type. <br />                                                                          Why do DNAprim mutations suppress the faf mutant <br />                                                                       eye phenotype? To answer this question, we must con- <br />                                                                       sider the origins of the ectopic photoreceptors in faf <br />                                                                       mutant eyes and also the time and place in the larval <br />                                                                       eye disc in which faf normally functions. <br />                                                                          The ommatidia of faf mutant eyes contain one or <br />                                                                       more photoreceptors in addition to the normal eight. <br />                                                                       The extra photoreceptors originate from the mystery <br />                                                                       cells, which are present in early facet preclusters <br />                                                                       (Fischer-Vize et al. 1992). The precursors to the first <br />                                                                       five photoreceptors (R8, R2/5, and R3/4) and one or <br />                                                                       more mystery cells emerge posterior to the morphoge- <br />                                                                       netic furrow as preclusters of six or more cells, each <br />                                                                       surrounded by undifferentiated cells (Tomlinson and <br />                                                                       Ready 1987b; Wolff and Ready 1991). Normally, the <br />                                                                       mystery cells detach from the preclusters and go on to <br />                                                                       divide in the wave of mitosis posterior to the furrow, <br />                                                                       giving rise to the remainder of the photoreceptors (R1, <br />                                                                       R6, and R7) and to the accessory cells (cone, pigment, <br />                                                                       and bristle cells). In faf mutants, the mystery cells remain <br />                                                                       within the preclusters and become extra R3/4-like cells <br />                                                                       (Fischer-Vize et al. 1992) Although faf is required for <br />    Figure 4.&#226;&#8364;&#8221;Eye phenotypes of weak DNAprim mutants. (A)              inhibition of mystery cell neurogenesis, mosaic analysis <br /> The eye of a late-stage pupa of the genotype DNAprimT2/DNA- <br /> primSfaf240 is shown. The arrow points to a region of cell death      as well as faf transgene complementation experiments <br /> (black dots) at the eye anterior. (B) A scanning electron micro-      indicate that faf is not required within the mystery cells <br /> graph showing the slightly rough eye of a DNAprim j10B2&#226;&#338;&#172;14/           nor within any photoreceptor precursors in the preclus- <br /> DNAprimSfaf240 adult. The inset is a magnification of a portion       ter (Fischer-Vize et al. 1992; Huang and Fischer-Vize <br /> of the micrograph, showing that bristles, which should be             1996). <br /> present at alternate corners of the hexagons, are often missing <br /> or misplaced. (C) A pupal eye disc of the genotype DNA- <br />                                                                          Normally, faf is expressed at high levels anterior to <br /> prim j10B2&#226;&#338;&#172;13/DNAprim Sfaf240 stained with cobalt sulfide is shown.   the morphogenetic furrow, where cells are undergoing <br /> The primary pigment cells (p), the cone cells (c), and the            mitosis, and then its expression shuts off within the <br /> general shape of the hexagonal lattice surrounding them ap-           furrow, just prior to precluster formation (Fischer-Vize <br /> pear normally arranged. (D) A tangential section through an           et al. 1992). Although faf normally functions anterior <br /> adult eye of the genotype DNAprim j10B2&#226;&#338;&#172;13/Df(3L)rdgC-co2 is <br />                                                                       to the furrow, later expression of faf in a subgroup of <br /> shown. The numbers indicate the photoreceptors R1&#226;&#8364;&#8220;R7, <br /> which appear normally arranged.                                       cells within the furrow is sufficient to rescue the faf mu- <br />                                                                       tant eye phenotype (Huang and Fischer-Vize 1996). The <br />                                                                       subgroup of cells that require faf expression are those <br /> help to define functional domains of DNAprim protein.                 cells that express the rough gene; these cells surround <br /> In addition, we find that Drosophila eye development                  those that express the proneural gene atonal, which are <br /> is particularly sensitive to DNAprim activity and that in             destined to become the precluster cells (Dokucu et al. <br /> the developing eyes of flies with reduced levels of DNA-              1996). Thus, faf activity in the dividing cells anterior to <br />  <br />  <br />                                                                                                Figure 5.&#226;&#8364;&#8221;BrdU incorporation <br />                                                                                             in larval eye discs. Cells in S phase <br />                                                                                             during a 2-hr period in third in- <br />                                                                                             star larval eye discs were visualized <br />                                                                                             by BrdU incorporation (materials <br />                                                                                             and methods). The genotypes of <br />                                                                                             the discs are as follows: (A) wild type <br />                                                                                             (w1118); (B) faf FO8/faf BX4; (C) DNA- <br />                                                                                             primT2/DNAprimS240. The black dots <br />                                                                                             are the cells that have incorporated <br />                                                                                             BrdU. (D) A diagram of a third in- <br />                                                                                             star larval eye-antennal disc is <br /> shown; ant, anterior; post, posterior; mf, morphogenetic furrow; a, antennal portion of disc; e, eye portion of disc. In C, more <br /> cells have incorporated BrdU both anterior to the furrow and in the band posterior to the furrow. The discs shown are <br /> representative of all of the discs stained (&#226;&#181;&#8216;20 of each genotype). <br />  <br /> the furrow facilitates a cell communication pathway that      <br /> may occur anterior to or within the furrow. Perhaps            <br /> when DNA primase levels are halved in heterozygous                    <br /> mutants, the cells anterior to the furrow remain in S              <br /> phase longer, which changes subtly the output of the                    <br /> signaling pathway that faf modulates in such a way that             <br /> the requirement for faf is largely overcome.                       <br />    Mutations in cell cycle proteins have been isolated                   <br /> previously in genetic screens for mutants affecting the              <br /> Sevenless receptor tyrosine kinase signaling pathway in         <br /> eye development. For example, mutations in the Dro-                  <br /> sophila homolog of Saccharomyces cerevisiae cell cycle reg-    <br /> ulator cdc37 were identified as dominant enhancers of             <br /> weak sevenless mutants (Cutforth and Rubin 1994).                <br /> Also, mutations in the peanut gene, which encodes a                     <br /> protein similar to yeast bud neck filaments and func-         <br /> tions in cytokinesis, were identified as dominant en-                       <br /> hancers of the phenotype of weak seven in absentia mu-                   <br /> tants (Neufeld and Rubin 1994); seven in absentia is a        <br /> nuclear protein that functions downstream of Sevenless                   <br /> to target a transcriptional repressor for proteolysis (Car-        <br /> thew and Rubin 1990; Li et al. 1997; Zhang et al. 1998).         <br /> In these cases, the genetic interactions may also be an             <br /> indirect consequence of cells that are receiving an in-             <br /> ductive signal remaining longer in a particular phase          <br /> of the cycle than they would normally. The results pre-                    <br /> sented here underscore the connection between the                         <br /> cell cycle and signaling pathways that govern cell deter-       <br /> mination. A less likely possibility is that the Cdc37, Pea-             <br /> nut, and/or the 60-kD subunit of DNA primase are                         <br /> bifunctional proteins, with distinct functions in the cell         <br /> cycle and in signal transduction.                                        <br /> </body></html>