<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Resource <br />  <br /> Inferring genome-wide functional linkages E. coli <br /> by combining improved genome context methods: <br /> Comparison high-throughput experimental data <br /> Sailu Yellaboina,1 Kshama Goyal,1 Shekhar C. Mande2 <br /> Centre DNA Fingerprinting Diagnostics, Hyderabad 500076, India <br />  <br />         Cellular functions determined interactions proteins cells. Recognition interactions forms <br />         important step understanding biology systems level. Here, report interaction network Escherichia <br />         coli, obtained training Support Vector Machine high quality interactions EcoCyc database, and <br />         assumption periplasmic cytoplasmic proteins interact other. The data <br />         features included correlation coefficient bit score phylogenetic profiles, frequency co-occurrence in <br />         predicted operons, new measure&#226;&#8364;&#8221;the distance translational start sites genes. The combined <br />         genome context methods high accuracy prediction test data predict total 78,122 binary <br />         interactions. The majority interactions identified high-throughput experimental methods correspond <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />         indirect interaction (interactions neighbors) predicted network. Correlation predicted network <br />         gene essentiality data shows essential genes E. coli exhibit high linking number, the <br />         nonessential genes exhibit low linking number. Furthermore, predicted protein&#226;&#8364;&#8220;protein interaction network <br />         shows proteins involved replication, DNA repair, transcription, translation, cell wall synthesis are <br />         highly connected. We believe predicted network serve useful resource understanding <br />         prokaryotic biology. <br />         [Supplemental material available online www.genome.org.] <br />  <br /> Living systems molecular entities, interactions                      tation, tandem affinity purification coupled mass spectrom- <br /> among rise complex properties life                    etry. Such techniques amenable high-throughput ex- <br /> not apparent individual molecules. The complex proper-                       perimentation. <br /> ties life addressed systems level                        Experimental detection genome-wide protein&#226;&#8364;&#8220;protein in- <br /> the availability complete genome sequences organ-                     teractions costly, time-consuming, difficult implement <br /> isms. Proteins dominant molecules life, focus                     modestly equipped laboratory. Moreover, little over- <br /> has understanding protein&#226;&#8364;&#8220;protein interactions                       lap results obtained different experimental <br /> recent past (Barabasi Oltvai 2004). A wide range experi-                     methods (von Mering et al. 2002; Arifuzzaman et al. 2006; Kro- <br /> ments carried order map genome-wide pro-                         gan et al. 2006). There exists substantial scope compu- <br /> tein&#226;&#8364;&#8220;protein interactions different organisms (Rain et al. 2001;                 tational analysis areas evaluation quality ex- <br /> Li et al. 2004; Suthram et al. 2005; Krogan et al. 2006; Stelzl et al.              perimental data, importantly predicting genome- <br /> 2006). The high-throughput experiments typically limit ob-                      wide protein&#226;&#8364;&#8220;protein interactions. <br /> servations fraction possible interactions                     Algorithms identify genome-wide interactions between <br /> experiments carried unique conditions. This                       proteins mainly focus coregulation interacting proteins. <br /> led development algorithms predict genome-                        These methods assume genes coregulated often <br /> wide protein&#226;&#8364;&#8220;protein interactions (Bork et al. 2004). By under-                     occur close genomes conserved <br /> standing network complex interactions, hoped                     gene order. Thus genes, operon, be <br /> that understanding living systems enhanced                       functionally linked (Dandekar et al. 1998; Overbeek et al. 1999). <br /> considerably.                                                                       These methods, however, fail identify functional links be- <br />       The interactions proteins classified                  tween proteins distantly located genome. A <br /> major categories: physical functional. The refer                      modified method used infer functional links between <br /> physical association proteins, re-                   genes distantly located genome or- <br /> fer proteins biochemical signaling pathway. In-                    thologs operon genome (Janga et <br /> ferences functional interactions obtained meth-                      al. 2005). <br /> ods coexpression data microarray analysis. Physical                         The coevolution method attempts calculate evolutionary <br /> interactions, hand, detected direct experi-                  rates substitutions different proteins assumes the <br /> mental techniques pull-down assays, coimmunoprecipi-                        proteins evolve similar rates parts interacting <br />                                                                                     pair (Fraser et al. 2002). Alternatively, phylogenetics profile <br /> 1 <br />   These authors contributed equally work.                                   method detects protein interactions identifying correlations <br /> 2 <br />   Corresponding author.                                                             presence absence genes ge- <br /> E-mail shekhar@cdfd.org.in; fax 91-40-27155610.                                     nomes (Pellegrini et al. 1999). <br /> Article published online print. Article publication date online at <br /> http://www.genome.org/cgi/doi/10.1101/gr.5900607. Freely available on-                   In study, propose computational method that <br /> line Genome Research Open Access option.                                identifies protein&#226;&#8364;&#8220;protein interactions basis distance <br />  <br />  <br /> 17:527&#226;&#8364;&#8220;535 &#194;&#169;2007 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/07; www.genome.org                              Genome Research           527 <br />                                                                                                                            www.genome.org <br />  Yellaboina et al. <br />  <br />  <br /> between translational start sites genes frequency         Frequency co-occurrence predicted operons <br /> of co-occurrence genes predicted operons. These two <br />                                                                          Several different prediction methods exist literature the <br /> features combined phylogenetics profiles using Sup- <br />                                                                          identification operons. These based features as <br /> port Vector Machine infer genome-wide functional link- <br />                                                                          intergenic distance, functional correlation, conserved gene <br /> ages Escherichia coli. The Support Vector Machine when <br />                                                                          neighborhood (Salgado et al. 2000; Ermolaeva et al. 2001; Zheng <br /> trained high-quality experimental data performs blind <br />                                                                          et al. 2002). We used Support Vector Machine identify op- <br /> tests, accuracies prediction exceeding 85%. The <br />                                                                          erons prokaryotic genomes using intergenic distance <br /> network interactions constructed using predictions exhib- <br />                                                                          transcription start sites genes data fea- <br /> its characteristic scale-free topology excellent correlation <br />                                                                          ture. <br /> with experimental gene essentiality data. We be- <br />                                                                                The average accuracy fivefold cross-validation operon <br /> lieve network interactions predicted method <br />                                                                          prediction 82%. The model highest accuracy was <br /> will useful tool understanding biology E. coli. <br />                                                                          used predict operons 124 prokaryotic genomes. This led <br />                                                                          prediction total 786 polycistronic transcription <br />                                                                          units E. coli K12. <br /> Results Discussion <br /> Data prediction protein&#226;&#8364;&#8220;protein interactions                      Phylogenetic profile method <br /> Interactions pairs proteins derived using Sup-         The phylogenetic profile gene represented two <br /> port Vector Machine (SVM) trained interactions reported           different ways: binary profile (Pellegrini et al. 1999) nor- <br /> in EcoCyc database (Keseler et al. 2005). These linkages rep-        malized bit score profile (Enault et al. 2003). In former, the <br /> resent hand-curated data obtained essentially low-               presence absence homolog represented 1 0, respec- <br /> throughput experimental approaches. Moreover, interac-             tively, latter, represented nor- <br /> tions deduced multiple experiments            malized bit score obtained BLAST. Two genes displaying a <br /> likely free false positives bias associ-   similar phylogenetic profile, assessed Pearson correlation <br /> ated high-throughput experiments. Supplemental Tables I             coefficient, assumed functionally linked. <br /> and II list data sets used supervised learning. The learning     We constructed phylogenetic profiles genes E. coli <br /> set contained 1082 proteins pairs, divided            based methods. <br /> into operonic (654) (Supplemental Table I) non-operonic <br /> data sets (428) (Supplemental Table II) defined EcoCyc <br />                                                                          Effect gene conservation prediction accuracy <br /> database. <br />       Defining reliable negative data set predicting func-         Assessment functional linkage genes not <br /> tional linkages using machine-learning techniques ac-           possible inspection phylogenetic profiles, <br /> knowledged difficult problem (Jansen et al. 2003; Ben-Hur        especially genes present closely <br /> and Noble 2006). The negative data predicting functional             related organisms. Moreover, phylogenetic profiles the <br /> linkages eukaryotes assumed proteins        genes specific lineage likely high <br /> not colocalized subcellular compartment (Jansen et           correlation coefficient them, absent a <br /> al. 2003). Similarly, prokaryotes reasonable       large number genomes, present closely re- <br /> the proteins required physiological functions extracel-       lated species. This result reporting false-positive inter- <br /> lular periplasmic milieu physically interact       actions genes. Thus, choice training SVM <br /> the proteins cytoplasmic space. We formulated         using data critically affect results prediction. This <br /> negative data set, protein pair comprises se-          especially true negative training data. We therefore <br /> creted protein localized cytoplasm. Thus,           assessed effect training data accuracy predictions <br /> proteins known secretory signal identified,        taking consideration conservation genes the <br /> were considered separately proteins signal          phylogenetic profiles. <br /> peptide. The top-scoring 40 proteins predicted signal                 The conservation score gene defined total <br /> sequence N terminus considered periplasmic,            number genomes possess homologs gene under <br /> while 346 proteins possess known signal           consideration (Sun et al. 2005). The conservation score pair <br /> sequences entire <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> polypeptide con-           genes was, turn, defined minimum conservation <br /> sidered cytoplasmic (Supplemental Table III). Combining            score of genes pair. To understand effect <br /> one periplasmic cytoplasmic protein generated          conservation score gene pair prediction accuracy the <br /> 13,840 pairs. The pairs negative data set did belong       phylogenetic profile method using bit score, used two-thirds <br /> any known complex, metabolic pathway,              randomly picked data positive negative data <br /> assumed lack functional linkages.                                     sets training remaining one-third testing pre- <br />       Having defined positive negative data sets         diction accuracy. The process repeated 50 times, time <br /> supervised machine learning, predictions carried            incrementing cutoff conservation score 1 nega- <br /> assessing functional linkages possible protein           tive data set retaining positive data. Figure 1A <br /> pairs E. coli K12 genome. Three different data features,          shows prediction accuracy protein&#226;&#8364;&#8220;protein interac- <br /> namely, frequency co-occurrence predicted operons, phy-            tions gradually increases conservation score 20 be- <br /> logenetic profile correlation score, minimum distance be-            cause increase sensitivity, decreases of <br /> tween genes 266 genomes, used             decrease specificity. The accuracy remains constant beyond <br /> these predictions described Methods.                               conservation score 12, considered phy- <br />  <br />  <br /> 528     Genome Research <br />           www.genome.org <br />                                                                                          Prediction genome-wide functional linkages <br />  <br />  <br />                                                                                suggests functionally related genes tend occur close to <br />                                                                                genome. <br />                                                                                      Intergenic distance analyzed hypothesized <br />                                                                                noninteracting gene pairs negative data set (Fig. 2B). In- <br />                                                                                terestingly, normalized distance translational <br />                                                                                start sites pairs proteins evenly distributed between <br />                                                                                0 (minimum possible normalized distance) 50 (maximum <br />                                                                                possible normalized distance) E. coli. The distribution is <br />                                                                                skewed lower values minimum normalized dis- <br />                                                                                tance pairs genomes calcu- <br />                                                                                lated. Comparison normalized intergenic distances across <br />                                                                                genomes showed 80% interacting proteins are <br />                                                                                spaced closely (normalized gene distance &lt;1) ge- <br />                                                                                nome, 10% hypothesized noninteracting <br />                                                                                pairs possess normalized gene distance &lt;1. <br />  <br />                                                                                Cross-validation accuracy prediction <br />                                                                                protein&#226;&#8364;&#8220;protein interactions <br />                                                                                To compare accuracy prediction different methods for <br />                                                                                protein&#226;&#8364;&#8220;protein interactions, calculated average sensitiv- <br />                                                                                ity specificity prediction fivefold cross-validation. <br />                                                                                Table 1 shows comparison fivefold cross-validation the <br />                                                                                different data features combinations. The combi- <br />                                                                                nation data features appears best choice <br />  <br />  <br />  <br />  <br /> Figure 1. Effect gene conservation score accuracy protein&#226;&#8364;&#8220; <br /> protein interaction predictions using phylogenetic profiles. (A) Profiles of <br /> the genes pair compared using normalized bit scores. (B) <br /> Profiles genes pair compared using presence (1) or <br /> absence (0) genes. Accuracy defined average sensitivity <br /> and specificity described Methods. It clearly seen predic- <br /> tion accuracy poorer low high conservation scores. See text for <br /> details. <br />  <br />  <br />  <br /> logenetic profiles genes conserved 12 <br /> genomes. <br />      Similar analysis performed binary phyloge- <br /> netic profile, outcome analysis similar. <br /> These results, shown Figure 1B, lead major conclu- <br /> sions: (1) prediction accuracy methods increases <br /> by increasing cutoff value conservation score the <br /> genes negative data set, (2) sensitivity pre- <br /> dictions low low conservation scores, specificity is <br /> low high conservation scores. Specificity rapidly decreases for <br /> both methods high conservation scores. However, the <br /> binary phylogenetic profiles, specificity drops zero high <br /> conservation scores. <br />  <br /> Gene distance method                                                           Figure 2. Normalized gene distance distributions data sets <br /> Genes closer usually stronger func-                used prediction protein&#226;&#8364;&#8220;protein interactions. The normalized gene <br />                                                                                pair distance defined number <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> pairs separate the <br /> tional linkages possibility coregulation operon               translation start sites genes pair, divided total genome <br /> or transcriptional coupling (Korbel et al. 2004). Analysis              length scaled 100. (A) (Gray) Normalized distances between <br /> 1082 gene pairs <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> components protein complexes                  known interacting protein pairs E. coli; (black) minimum nor- <br /> showed large number (428) oper-                 malized distance proteins ge- <br />                                                                                nomes. (B) Normalized distances protein pairs, are <br /> ons. These gene pairs distantly located E. coli K12 <br />                                                                                hypothesized noninteracting. (Gray line) The normalized distance <br /> genome, interestingly orthologs closer                  distribution E. coli; (black line) minimum same <br /> other genome shown Figure 2A. This                 pair proteins genome. <br />  <br />  <br />                                                                                                                         Genome Research             529 <br />                                                                                                                            www.genome.org <br />  Yellaboina et al. <br />  <br />  <br /> Table 1. Prediction accuracy protein&#226;&#8364;&#8220;protein interactions using different data                proteins. Taking consid- <br /> features: minimum distance pair genes prokaryotic genomes,                eration, overlap be- <br /> phylogenetic profile using bit-score values, frequency co-occurrence operons <br />                                                                                                        tween predicted set ex- <br />                                                           Sensitivity   Specificity     Accuracy (%)   perimental data sets quite high <br />                                                                                                        (&#226;&#710;&#188;80%). <br /> Minimum distance                                            0.74            0.99            86.5             A large number new interactions <br /> Frequency co-occurrence operons                       0.68            1.0             84.0 <br />                                                                                                        predicted method. These in- <br /> Phylogenetic profile                                        0.73            0.95            84.0 <br /> Combination methods                                                                       teractions physical, might <br />   Minimum distance frequency co-occurrence                                                      represent functional linkages neces- <br />      operons                                             0.68            1.0             84.0       sarily physical character. For example, <br />   Minimum Distance &amp; Phylogenetic Profile                   0.79            1.0             89.5 <br />                                                                                                        known LacI regulates expres- <br />   Frequency co-occurrence operons and <br />      phylogenetic profile                                   0.76            1.0             88.0       sion lac operon. However, LacI <br /> Combined minimum distance, phylogenetic profile,                                                       does interact physically of <br />   frequency co-occurrence operons                 0.79            1.0             89.5       structural genes lac op- <br />                                                                                                        eron. Such interactions LacI <br /> The accuracy prediction defined average sensitivity specificity. The values    genes lac operon be <br /> reported averages respective values obtained fivefold cross-validation. <br />                                                                                                        identified using pull-down methods. <br />                                                                                                        Similarly, predictions identified in- <br /> for prediction protein&#226;&#8364;&#8220;protein interactions. The average accu-             teractions RpoA Sec A, B, D. These interactions <br /> racy predictions reaches high 89.5% best predic-              represent functional linkages proteins. <br /> tion. The network predicted using features, namely, mini-                 As known, transcription translation prokaryotes are <br /> mum distance phylogenetic profile, showed similar sensitiv-               coupled processes, Sec-dependent pathway also <br /> ity specificity values predicted using data            coupled translation process. Thus, proteins tran- <br /> features. Nonetheless, number false positives predicted                scription machinery functionally linked Sec A, B, <br /> during cross-validation using features higher               D proteins. These examples suggest present method <br /> that using features. Therefore, used data fea-             predicts physical interactions provides insights <br /> tures predict genome-wide functional linkages.                             regulatory functional linkages. <br />  <br /> Genome-wide prediction protein interactions E. coli <br />                                                                            Analysis predicted protein&#226;&#8364;&#8220;protein interaction network <br /> The prediction genome-wide interactions E. coli resulted in <br /> a total 78,122 binary interactions (Table 1). The high specific-        Most real world networks known possess scale-free <br /> ity values clearly suggest number false-positive inter-        topology (Barabasi Albert 1999). The important char- <br /> actions predicted network minimal. On            acteristic scale-free networks degree distribution <br /> hand, sensitivity determined total number iden-            (distribution number links) follows power law, is, <br /> tified interactions theoretically possible interac-          p(k) &#226;&#710;&#188; ak&#226;&#179;&#174;&#226;?&#165;, &#226;?&#165; &lt; 3. The scale-free networks possess small <br /> tions, lower sensitivity values suggest total number           number nodes large number links, large <br /> interactions reported represent lower estimate.              number nodes possess links (Barabasi Bonabeau <br /> Thus, believe number pairwise protein&#226;&#8364;&#8220;protein           2003). We analyzed scale-free behavior predicted inter- <br /> interactions E. coli excess 78,122.                       action network, described following discussion. <br />                                                                                  The degree distribution predicted network 3798 <br />                                                                            nodes 78,122 edges follows power law average <br /> Comparison predicted network <br />                                                                            number links 41 degree exponent 1.26 (Fig. <br /> with high-throughput experimental data <br />                                                                            3A). This indicates network possesses high diversity of <br /> The predicted protein&#226;&#8364;&#8220;protein interactions compared with <br /> the results high-throughput experimental data sets (But- <br /> land et al. 2005; Arifuzzaman et al. 2006). It observed that <br />                                                                            Table 2. Comparison predicted interactions with <br /> among 5418 interactions identified pull-down assays             high-throughput experimental data <br /> by TAP-tagged bait protein method, method predicted <br /> 635 direct interactions 4513 indirect (binary, ternary, qua-                                    TAP-tagged      His-tagged <br />                                                                            Method            EcoCyc       bait             bait        Prediction <br /> ternary) interactions (Butland et al. 2005). Interestingly, 254 pro- <br /> tein pairs 635 direct interactions reportedly             EcoCyc              1082          54              40         875 (954) <br /> high confidence (Butland et al. 2005). On hand,              TAP-tagged bait                 5418             174         635 (4513) <br /> method identified 446 direct 8024 indirect interactions            His-tagged bait                               10,986         446 (8024) <br /> of total 10,986 unique interactions determined pull-down            Prediction                                                78,122 <br /> assays using His-tagged bait method (Arifuzzaman et al. 2006; <br />                                                                            Two high-throughput experimental data sets available the <br /> Table 2).                                                                  literature, namely, derived His-tagged bait analysis (Arifuzza- <br />      The overlap predictions experimental              man et al. 2006) TAP-tagged bait analysis (Butland et al. <br /> studies higher consider indirect interactions. This             2005). The interactions reported data sets the <br /> might identify proteins interact              EcoCyc database (Keseler et al. 2005) compared predic- <br />                                                                            tions. The values quoted table interactions com- <br /> bait directly, indirectly using pull-down assays. It           mon methods. The values parentheses correspond the <br /> therefore likely experimental techniques iden-           indirect interactions, is, mediated neighbors, pre- <br /> tify direct physical interactions, complexes involving            dicted network. <br />  <br />  <br /> 530     Genome Research <br />           www.genome.org <br />                                                                                            Prediction genome-wide functional linkages <br />  <br />  <br />  <br />  <br /> Figure 3. Topological features predicted protein&#226;&#8364;&#8220;protein interaction network E. coli K12. (A) The degree distribution clearly shows the <br /> network follows scale-free properties. (B) The distribution clustering coefficient shows network hierarchical. Both axes represent <br /> the respective logarithmic values. (C) Distribution shortest path pairs proteins predicted network. (D) The null network possesses <br /> a Gaussian degree distribution observed log&#226;&#8364;&#8220;log plot. <br />  <br />  <br /> node degree. Moreover, node degree similar ob-                          Degree distribution essential genes showed cer- <br /> tained experimental methods: TAP-tagged bait-                         tain essential genes possess degrees low 2&#226;&#8364;&#8220;3. As suggested by <br /> associated proteins (1.24), His-tagged bait-associated pro-                  Pr&#197;&#190;ulj et al. (2004), lethal nodes highly con- <br /> teins (1.41). A degree &lt;2 implies total number links                 nected nodes network. Certain nodes lethal because <br /> grows faster total number nodes (Seyed-Allaei et al.                 deletion disconnects network subgraphs and <br /> 2005). Further analysis clustering coefficient, C(k), re-                 disrupts network structure. Such nodes been <br /> vealed distribution clustering coefficient follows              referred &#226;&#8364;&#339;articulation points.&#226;&#8364;? Analysis low- <br /> the power law [C(k) &#226;&#710;&#188; k &#226;&#179;&#174; &#226;?&#164; ], modularity exponent                          degree essential genes, dicA, supported argument. DicA a <br /> &#226;?&#164; = &#226;&#179;&#174;1.94, suggesting low-degree nodes cohe-                   prophage-based transcriptional regulator regulates ex- <br /> sive high-degree nodes (Fig. 3B). This, turn, illustrates            pression cell division inhibitor DicB. DicA predicted <br /> the disassortive nature E. coli interaction network, indicat-             interact prophage integrase IntD DNA-binding <br /> ing high-degree nodes general linked                  transcriptional activator, SdiA (Fig. 4). In predicted network, <br /> other high-degree nodes.                                                         SdiA interacts 35 proteins, IntD interacts Aas <br />       The number links (path length) required connect                 (2-acyl-glycerophospho-ethanolamine acyltransferase). Aas, in <br /> node node predicted network deter-                     turn, interacts 13 proteins. The topology sub- <br /> mined, referred &#226;&#8364;&#339;small world property&#226;&#8364;?                   network clearly shows DicA possesses low degree, <br /> (Fig. 3C). The diameter predicted network, is,                   elimination disrupt connection SdiA and <br /> longest graph distance nodes,                    Aas subgraphs. Intriguingly, topology subnetwork sug- <br /> 11. This implies nodes densely linked pos-                 gests IntD behave similarly DicA, IntD is <br /> sess small world properties. In words, small metabolic per-                known essential. The nonessentiality IntD be <br /> turbations affect overall functioning                       attributed occurrence prophage integrase genes <br /> organism.                                                                        E. coli genome, function analogously. <br />       Hubs protein interaction networks postulated                        Another important correlation predicted network <br /> to represent essential genes genome (Jeong et al. 2001; Albert              available experimental data relates list nones- <br /> 2005). Indeed, observed proteins like DNA polymer-                   sential genes. Analysis degree distribution 741 genes <br /> ase subunits (HolA HolC) RNA polymerase subunits                     identified nonessential Posfai et al. (2006) interestingly <br /> (RpoA, RpoB, RpoC, RpoZ) essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>survival</span>                     showed average number interactions (&#226;&#338;&#169;k&#226;&#338;&#170;) set <br /> possess degree &gt;100. Further analysis revealed &gt;80%                genes 18 (Supplemental Fig. 1b). The &#226;&#338;&#169;k&#226;&#338;&#170; nonessen- <br /> known essential genes possess degree &gt;41 (Baba et al. 2006).                   tial genes low compared the complete <br /> This accordance earlier observation essential                network (&#226;&#338;&#169;k&#226;&#338;&#170; = 41) essential genes (&#226;&#338;&#169;k&#226;&#338;&#170; = 115). This ob- <br /> genes densely connected (Yu et al. 2004). The average number                 servation supports good agreement pre- <br /> of interactions essential genes predicted network                 dicted interaction network independently obtained experi- <br /> 115 (Supplemental Fig. 1a).                                                      mental data. <br />  <br />  <br />                                                                                                                           Genome Research              531 <br />                                                                                                                              www.genome.org <br />  Yellaboina et al. <br />  <br />  <br />                                                                                                          degrees. This appears suggest per- <br />                                                                                                          turbations transcription translation <br />                                                                                                          likely affect physiology <br />                                                                                                          cells perturbations the <br />                                                                                                          metabolic pathways. Interestingly, <br />                                                                                                          Pr&#197;&#190;ulj et al. (2004) earlier observed <br />                                                                                                          yeast protein&#226;&#8364;&#8220;protein inter- <br />                                                                                                          action network, stress defense and <br />                                                                                                          transport pathways connected <br />                                                                                                          transcription translation. It <br />                                                                                                          observed proteins involved <br />                                                                                                          cellular organization possess low de- <br />                                                                                                          gree consist highest articula- <br />                                                                                                          tion points. Encouragingly, high aver- <br />                                                                                                          age degree observed cell wall <br />                                                                                                          synthesis pathway predicted net- <br />                                                                                                          work. Thus, overall physiology a <br />                                                                                                          prokaryotic cell determined by <br />                                                                                                          processes involved mainte- <br />                                                                                                          nance expression genetic infor- <br />                                                                                                          mation cell wall biosynthe- <br />                                                                                                          sis, metabolic <br />                                                                                                          processes. <br />                                                                                                                An interesting analysis pre- <br />                                                                                                          dicted protein&#226;&#8364;&#8220;protein interaction net- <br /> Figure 4. DicA subnetwork. dicA known essential gene E. coli. Intriguingly, observed   work pertains well-known thiore- <br /> that DicA degree 2. However, clear subgraph, deletion DicA leads      doxin system. The thioredoxin system <br /> disconnected islands subnetworks, offering explanation essentiality.            plays important role maintaining <br />                                                                                                          cellular environment reduced <br />       To enhance confidence predicted network,                conditions enabling proper functioning several <br /> null network constructed randomizing edges                   enzymes. Thioredoxin, overexpressed oxidative <br /> network generate 78,122 interactions using number                  stress, essential proper functioning large number of <br /> of nodes. The degree distribution analyzed network.               proteins involved light-activated Calvin cycle tran- <br /> The random network possessed Gaussian degree distribution                    scription regulation (Arner Holmgren 2000). E. coli K12 pos- <br /> unlike scale-free predicted network (Fig. 3D). Moreover,                sesses thioredoxins, TrxA TrxC, reduced by <br /> the random network, average number interactions (&#226;&#338;&#169;k&#226;&#338;&#170;)               thioredoxin reductase, TrxB. In predicted network, TrxA was <br /> the essential genes 40.8, nonessential genes              observed interact 33 proteins, TrxB 47 proteins, and <br /> was 30.8. This suggests predicted network biologically             TrxC 12 proteins (Supplemental Fig. 2). Interestingly, of <br /> relevant provides useful information.                                      total 85 proteins, 80 possess cysteine residue. <br />       We observed &#226;&#710;&#188;2120 proteins interact directly             Since principal mechanism thioredoxin-mediated re- <br /> indirectly essential genes. The shortest path-length dis-             dox reaction dithiol exchange, appears 80 pro- <br /> tribution essential gene network showed large num-               teins natural substrates thioredoxins. Oc- <br /> ber nodes connected essential genes path length             currence well-known physiological partners thioredox- <br /> of 3 (Supplemental Fig. 1c). Similarly, path-length distribu-              ins ribonucleotide reductase (Rnr), alkyl hydroperoxide <br /> tion nonessential gene network showed proteins                reductase (AhpC), chaperone protein (DnaK) suggest that <br /> interact nonessential genes nodes               predicted network thioredoxins high confi- <br /> (Supplemental Fig. 1d). The diameter essential gene net-               dence (Kumar et al. 2004). Apart known partners, we <br /> work 8, nonessential gene net-                 proteins predicted thioredoxin-domain- <br /> work 14. Thus, degree distribution                containing protein YbbN, predicted transferase NAD(P)- <br /> the small world properties able distinguish                 binding domain YbhK, predicted oxidoreductase YjjX to <br /> experimentally characterized essential nonessential genes.                 interact thioredoxins. Thus, predicted thioredoxin <br />                                                                                subnetwork substantiates known interactions but <br />                                                                                enables identification possible additional pathways regu- <br /> Functional analysis predicted network                                       lated thioredoxin system. <br /> It great analyze protein&#226;&#8364;&#8220;protein interactions                  Yet observation predictions pertains the <br /> in context functional categories. KEGG classifies             toxin&#226;&#8364;&#8220;antitoxin helps bacteria maintain segrega- <br /> the E. coli proteins 19 different functional categories. We ob-             tional stability fight stress conditions. Several toxin&#226;&#8364;&#8220; <br /> served important cellular pathways&#226;&#8364;&#8221;transcription                  antitoxin pairs identified E. coli (Hayes 1998). In our <br /> translation&#226;&#8364;&#8221;possess highest average degree interaction               predicted network, toxin&#226;&#8364;&#8220;antitoxin pairs (TA) been <br /> all networks (Fig. 5). Our predicted network showed                 predicted interacting partners. RelBE, well- <br /> high average degree replication repair pathways. Inter-                characterized TA pairs, known alter gene expression levels <br /> estingly, metabolic pathways possessed smaller average                 amino acid carbon source starvation. We observe that <br />  <br />  <br /> 532     Genome Research <br />           www.genome.org <br />                                                                                       Prediction genome-wide functional linkages <br />  <br />  <br />                                                                                                       predicted E. coli interactome can <br />                                                                                                       help deciphering unknown functions <br />                                                                                                       pathways. <br />  <br />  <br />                                                                                                       Methods <br />                                                                                                         Completely sequenced genomes 295 <br />                                                                                                         bacteria downloaded the <br />                                                                                                         NCBI ftp site (ftp://ftp.ncbi.nih.gov/ <br />                                                                                                         genomes/Bacteria/). Bacteria linear <br />                                                                                                         genomes considered the <br />                                                                                                         analysis. A total 266 bacteria genomes <br />                                                                                                         remained excluding bacteria <br />                                                                                                         linear genomes (Supplemental <br />                                                                                                         Table VII). The homologous sequences <br />                                                                                                         known open reading frames <br />                                                                                                         (ORFs) E. coli K-12 searched us- <br /> Figure 5. Comparative analysis average degree interactions proteins belonging          ing BLASTp 266 genomes <br /> different functional pathways experimental predicted interactomes. It clear  e&#226;&#179;&#174;04 cutoff value. Orthologs <br /> distributions transcription translation processes exhibit largest average degree. <br />                                                                                                         E. coli genes identified bi- <br />                                                                                                         directional best BLAST hit (Hirsh and <br /> the identification glutamine transporter (GlnQ) ri-                  Fraser 2001). Wherever genome sequences different bacterial <br /> bose-5-phosphate isomerase (RpiB) RelE-interacting partners               strains species available, selected one <br /> not surprising (Supplemental Fig. 3a). Interestingly Nickel                 shares maximum number orthologs E. coli K12 to <br /> transporter subunit NikE transcription regulator Man-                reduce bias phylogenetic profiles. Therefore, 124 genome <br /> ganese transport protein MntH, MntR, predicted                     sequences belonging different bacterial species used for <br /> interact RelE. This suggests addition <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                  generating phylogenetic profiles determining fre- <br /> stress, RelBE play role overcoming heavy metal stress.               quency co-occurrence gene pair (Supplemental Table VIII). <br /> Furthermore, hypothetical proteins YnjB YnjC, <br /> which predicted act membrane transporters,             Phylogenetic profiles <br /> occur RelBE subnetwork. In words, RelBE             Bit scores open reading frames E. coli obtained <br /> might enable cells recover variety different stresses.         BLAST 124 genomes, used to <br /> This accordance fact RelBE locus tends           generate profile 124 genomes, doubly nor- <br /> protect cells detrimental effects stress,           malized: (1) bit score profile divided maxi- <br /> being suicidal (Pedersen et al. 2002).                                      mum value bit score genes; (2) minimum <br />       E. coli possesses paralogs RelBE system, RelBEK12          bit score E. coli ORFs homologs other <br /> and RelBESOS. RelBESOS, known YafQ/DinJ, possesses                genomes considered. In order assess pair genes were <br /> LexA-binding site promoter region gets acti-           coevolving, Pearson correlation coefficient respective <br />                                                                             phylogenetic profiles calculated. The E. coli genome encodes <br /> vated SOS response elicited DNA damage (Lewis et <br />                                                                             4237 proteins. A 4237 &#226;&#179;&#175; 4237 symmetric matrix con- <br /> al. 1994). The physiological role YafQ/DinJ charac- <br />                                                                             structed, matrix elements represented by <br /> terized. However, basis predicted interacting partners, <br />                                                                             correlation coefficient calculated. <br /> we able propose YafQ/DinJ inhibit HsdR, an <br /> endonuclease, allowing perpetuation modified <br /> DNA (Supplemental Fig. 3b). In addition allowing DNA modi-               Frequency co-occurrence predicted operons <br /> fication combat SOS response, YafQ predicted              A total 1113 operon sequences downloaded the <br /> interact enzymes tryptophan glutamine biosyn-               EcoCyc database (Keseler et al. 2005). Partially identified operons <br /> thesis. Although functional implications observation            alternative transcription termination were <br /> are clear, propose YafQ involved repression            excluded. Among remaining 699 operons, 119 <br /> amino acid biosynthesis. This implies YafQ/DinJ protect            operons gene present in <br /> cell DNA damage allowing DNA modification in-                   gene coordinate file (NC_000913.ptt) E. coli K12. After <br /> hibiting protein synthesis.                                                 mapping remaining 580 operons E. coli K12 gene co- <br />                                                                             ordinate file, 497 gene pairs genes pair <br />       The excellent correlation predicted protein in- <br />                                                                             belong operon adjacent each <br /> teraction network E. coli K12 wide variety experimen- <br />                                                                             other. Furthermore, 616 gene pairs genes <br /> tal data suggests interaction network useful <br />                                                                             pair codirectionally transcribed, is, adjacent <br /> resource understand biology E. coli. The network be <br />                                                                             belong different transcription units. Inter- <br /> used, example, identify functions genes             genic distances genes gene pairs were <br /> been annotated hypothetical, function unknown. We                  taken positive data set (Supplemental Table IV), whereas <br /> suggest cliques network, include genes           taken negative data set (Supple- <br /> characterized functions genes unknown function,              mental Table V). These data sets used train Support <br /> will able assign pathway genes reside.              Vector Machine prediction operons 124 genomes. <br /> Another interesting feature addressed using in-                  The frequency co-occurrence pairs proteins <br /> teraction network identify large multiprotein complexes            (4237 &#226;&#179;&#175; 4237) predicted operons genomes <br /> the identification cliques. Analysis modular nature            assigned score protein pairs. Thus, each <br />  <br />  <br />                                                                                                                   Genome Research             533 <br />                                                                                                                      www.genome.org <br />  Yellaboina et al. <br />  <br />  <br /> element symmetric matrix represents frequency            cross-validation, sets used training, remain- <br /> co-occurrence genes j, predicted operons           ing set used testing. The RBF function used train- <br /> across genomes.                                                ing, best cost &#226;?&#165; selected using grid search <br />                                                                        algorithm grid.py LibSvm software package. In step of <br /> Gene distance method                                                   testing, sensitivity specificity values calculated. The ac- <br /> With 4237 genes E. coli genome, distance        curacy prediction taken average sensitivity <br /> transcriptional start sites gene rest       specificity, defined follows: <br /> genes (4236) calculated dividing absolute value        Sensitivity = True Positives/(True Positives + False Negatives) <br /> distance (in nucleotide bases) total genome length. All the <br /> genomes consideration circular, calculated           Specificity = True Negatives/(True Negatives + False Positives) <br /> absolute value distance clockwise counterclock- <br /> wise directions, considered minimum            Among models generated, model the <br /> values. These values scaled 100.                          highest accuracy retained best model used for <br />      Similarly, calculated distances orthologs       predictions genome-wide scale. <br /> the genes 266 genomes. The interaction score between <br /> any genes taken minimum distances be- <br />                                                                        Analysis network attributes <br /> tween genes orthologs 266 genomes.        A variety graph-theoretic statistics, degree distri- <br /> The final matrix 4237 &#226;&#179;&#175; 4237 symmetric,         bution, clustering coefficient, characteristic path length and <br /> element matrix minimum distances          diameter analyzed. Locally written Perl scripts used to <br /> the genes j 266 genomes.                           analyze degree distribution clustering coefficient. Path <br />                                                                        length analyzed using Cytoscape2.2 (Shannon et al. 2003). <br /> Prediction protein&#226;&#8364;&#8220;protein interactions using Support           The subnetwork graphs illustrations generated using <br /> Vector Machine                                                         VISANT (Hu et al. 2005). <br /> The 4237 &#226;&#179;&#175; 4237 symmetric matrix generated the <br /> above methods considered represent possible pair-       Acknowledgments <br /> wise interactions proteins. Each element ma- <br /> trix represented score interaction. These interaction       We thank Bioinformatics center CDFD SUN Cen- <br /> scores used different data features training    tre Excellence Medical Bioinformatics access their <br /> Support Vector Machine positive data negative data        computational facilities. We thank M. Vidyasagar discus- <br /> as described below.                                                    sions SVMs graph theory, Biju Issac stimu- <br />                                                                        lating discussions. Financial support Department Bio- <br /> Preparation positive negative data set                      technology gratefully acknowledged. K.G. CSIR Senior Re- <br />                                                                        search Fellow, S.C.M. Wellcome Trust International <br /> The data E. coli protein complexes pathways down-          Senior Research Fellow supported grant WT070006. <br /> loaded EcoCyc database (Keseler et al. 2005). Any two <br /> proteins complex considered be <br /> functionally interacting. The self-interactions ho-            References <br /> modimers complexes excluded. <br />      The negative reference data set comprised pairs pro-     Albert, R. 2005. Scale-free networks cell biology. J. Cell Sci. <br /> teins colocalized compartment E. coli.         118: 4947&#226;&#8364;&#8220;4957. <br />                                                                        Arifuzzaman, M., Maeda, M., Itoh, A., Nishikata, K., Takita, C., Saito, R., <br /> Thus, periplasmic location proteins predicted           Ara, T., Nakahigashi, K., Huang, H.C., Hirai, A., et al. 2006. <br /> using SIGCLEAVE EMBOSS (Sarachu Colet 2005). The pro-               Large-scale identification protein&#226;&#8364;&#8220;protein interactions of <br /> tein considered periplasmic contained            Escherichia coli K-12. Genome Res. 16: 686&#226;&#8364;&#8220;691. <br /> predicted signal sequence 50 residues N ter-       Arner, E.S. Holmgren, A. 2000. Physiological functions of <br />                                                                            thioredoxin thioredoxin reductase. Eur. J. Biochem. <br /> minus. The top-scoring 40 proteins considered                   267: 6102&#226;&#8364;&#8220;6109. <br /> periplasmic. In all, 346 proteins did possess signal se-       Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., <br /> quence entire polypeptide stretch,            Datsenko, K.A., Tomita, M., Wanner, B.L., Mori, H. 2006. <br /> were considered cytoplasmic. The accuracy identifying             Construction Escherichia coli K-12 in-frame, single-gene <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>knockout</span> <br />                                                                            mutants: The Keio collection. Mol. Syst. Biol. 2: E1&#226;&#8364;&#8220;E11. <br /> the signal sequence SIGCLEAVE reported 75%&#226;&#8364;&#8220;          Barabasi, A.L. Albert, R. 1999. Emergence scaling random <br /> 80%, sets proteins chosen periplasmic           networks. Science 286: 509&#226;&#8364;&#8220;512. <br /> and cytoplasmic likely correct subcellular localization   Barabasi, A.L. Bonabeau, E. 2003. Scale-free networks. Sci. Am. <br /> (http://bioweb.pasteur.fr/docs/EMBOSS/sigcleave.html). Each                288: 60&#226;&#8364;&#8220;69. <br />                                                                        Barabasi, A.L. Oltvai, Z.N. 2004. Network biology: Understanding <br /> pair unique data sets.                                      cell&#226;&#8364;&#8482;s functional organization. Nat. Rev. Genet. 5: 101&#226;&#8364;&#8220;113. <br />                                                                        Ben-Hur, A. Noble, W.S. 2006. Choosing negative examples the <br /> Cross-validation model selection prediction accuracy               prediction protein&#226;&#8364;&#8220;protein interactions. BMC Bioinformatics <br />                                                                            7: 1&#226;&#8364;&#8220;6. <br /> We used LibSvm predict protein&#226;&#8364;&#8220;protein interactions             Bork, P., Jensen, L.J., von Mering, C., Ramani, A.K., Lee, I., and <br /> (Chih-Chung Chih-Jen 2001). The software enables users              Marcotte, E.M. 2004. Protein interaction networks yeast to <br /> define parameters allows choice inbuilt kernel            human. Curr. Opin. Struct. Biol. 14: 292&#226;&#8364;&#8220;299. <br />                                                                        Butland, G., Peregrin-Alvarez, J.M., Li, J., Yang, W., Yang, X., Canadien, <br /> function including linear, Radial Basis Function (RBF), polyno- <br />                                                                            V., Starostine, A., Richards, D., Beattie, B., Krogan, N., et al. 2005. <br /> mial, sigmoid.                                                         Interaction network containing conserved essential protein <br />       In fivefold cross-validation, data set consisting posi-       complexes Escherichia coli. Nature 433: 531&#226;&#8364;&#8220;537. <br /> tives negatives randomly divided equal size          Chih-Chung, C. Chih-Jen, L. 2001. LIBSVM: A library support <br />                                                                            vector machines. http://www.csie.ntu.edu.tw/&#226;&#710;&#188;cjlin/libsvm. <br /> sets. Training testing carried times, using the <br />                                                                        Dandekar, T., Snel, B., Huynen, M., Bork, P. 1998. Conservation of <br /> &#226;&#8364;&#339;svm-train&#226;&#8364;? &#226;&#8364;&#339;svm-predict&#226;&#8364;? utility LibSvm software               gene order: A fingerprint proteins physically interact. Trends <br /> package (Chih-Chung Chih-Jen 2001). In round                   Biochem. Sci. 23: 324&#226;&#8364;&#8220;328. <br />  <br />  <br />  <br /> 534     Genome Research <br />           www.genome.org <br />                                                                                              Prediction genome-wide functional linkages <br />  <br />  <br /> Enault, F., Suhre, K., Abergel, C., Poirot, O., Claverie, J.M. 2003.              reversal bacteriostatic condition controlled expression of <br />     Annotation bacterial genomes using improved phylogenomic                       toxins antitoxins. Mol. Microbiol. 45: 501&#226;&#8364;&#8220;510. <br />     profiles. Bioinformatics 19: i105&#226;&#8364;&#8220;i107.                                       Pellegrini, M., Marcotte, E.M., Thompson, M.J., Eisenberg, D., and <br /> Ermolaeva, M.D., White, O., Salzberg, S.L. 2001. Prediction                    Yeates, T.O. 1999. Assigning protein functions comparative <br />     operons microbial genomes. Nucleic Acids Res. 29: 1216&#226;&#8364;&#8220;1221.                   genome analysis: Protein phylogenetic profiles. Proc. Natl. Acad. Sci. <br /> Fraser, H.B., Hirsh, A.E., Steinmetz, L.M., Scharfe, C., Feldman,                 96: 4285&#226;&#8364;&#8220;4288. <br />     M.W. 2002. Evolutionary rate protein interaction network.              Posfai, G., Plunkett III, G., Feher, T., Frisch, D., Keil, G.M., Umenhoffer, <br />     Science 296: 750&#226;&#8364;&#8220;752.                                                             K., Kolisnychenko, V., Stahl, B., Sharma, S.S., Arruda, M., et al. <br /> Hayes, F. 1998. A family stability determinants pathogenic                      2006. Emergent properties reduced genome Escherichia coli. Science <br />     bacteria. J. Bacteriol. 180: 6415&#226;&#8364;&#8220;6418.                                           312: 1044&#226;&#8364;&#8220;1046. <br /> Hirsh, A.E. Fraser, H.B. 2001. Protein dispensability rate             Pr&#197;&#190;ulj, N., Wigle, D.A., Jurisica, I. 2004. Functional topology a <br />     evolution. Nature 411: 1046&#226;&#8364;&#8220;1049.                                                 network protein interactions. Bioinformatics 20: 340&#226;&#8364;&#8220;348. <br /> Hu, Z., Mellor, J., Wu, J., DeLisi, C. 2005. VisANT: Data-integrating         Rain, J.C., Selig, L., De Reuse, H., Battaglia, V., Reverdy, C., Simon, S., <br />     visual framework biological networks modules. Nucleic Acids               Lenzen, G., Petel, F., Wojcik, J., Schachter, V., et al. 2001. The <br />     Res. 33: W352&#226;&#8364;&#8220;W357.                                                               protein&#226;&#8364;&#8220;protein interaction map Helicoba <br /> </body></html>