<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> Nip7p Interacts with Nop8p, an Essential Nucleolar Protein Required <br />    for 60S Ribosome Biogenesis, and the Exosome Subunit Rrp43p <br />  <br />              NIP7 encodes a conserved Saccharomyces cerevisiae nucleolar protein that is required for 60S subunit bio- <br />           genesis (N. I. T. Zanchin, P. Roberts, A. DeSilva, F. Sherman, and D. S. Goldfarb, Mol. Cell. Biol. 17:5001&#226;&#8364;&#8220;5015, <br />           1997). Rrp43p and a second essential protein, Nop8p, were identified in a two-hybrid screen as Nip7p-inter- <br />           acting proteins. Biochemical evidence for an interaction was provided by the copurification on immunoglobulin <br />           G-Sepharose of Nip7p with protein A-tagged Rrp43p and Nop8p. Cells depleted of Nop8p contained reduced <br />           levels of free 60S ribosomes and polysomes and accumulated half-mer polysomes. Nop8p-depleted cells also <br />           accumulated 35S pre-rRNA and an aberrant 23S pre-rRNA. Nop8p-depleted cells failed <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> accumulate either <br />           25S or 27S rRNA, although they did synthesize significant levels of 18S rRNA. These results indicate that 27S <br />           or 25S rRNA is degraded in Nop8p-depleted cells after the section containing 18S rRNA is removed. Nip7p- <br />           depleted cells exhibited the same defects as Nop8p-depleted cells, except that they accumulated 27S precursors. <br />           Rrp43p is a component of the exosome, a complex of 3*-to-5* exonucleases whose subunits have been implicated <br />           in 5.8S rRNA processing and mRNA turnover. Whereas both green fluorescent protein (GFP)-Nop8p and GFP- <br />           Nip7p localized to nucleoli, GFP-Rrp43p localized throughout the nucleus and to a lesser extent in the cyto- <br />           plasm. Distinct pools of Rrp43p may interact both with the exosome and with Nip7p, possibly both in the nu- <br />           cleus and in the cytoplasm, to catalyze analogous reactions in the multistep process of 60S ribosome biogenesis <br />           and mRNA turnover. <br />  <br />  <br />    Ribosome biogenesis in eukaryotes occurs mostly in the nu-                 is responsible for endonucleolytic processing at A3 in ITS1 (31). <br /> cleolus, where rRNA is transcribed, processed, and covalently                 Two 59-to-39 exonucleases, Rat1p and Xrn1p, are required for <br /> modified, newly synthesized ribosomal proteins (rproteins) are                maturation of the 5.8SS rRNA 59 end (17) and for the degra- <br /> delivered, and 40S and 60S subunits are assembled. While the                  dation of several excised spacer sequences, including frag- <br /> mechanism of ribosome synthesis is complex and poorly un-                     ments A0 to A1, A2 to A3, and D to A2 (27, 49). The five sub- <br /> derstood, significant progress has been made toward identify-                 units of the exosome complex (35) and Rrp6p (5) are required <br /> ing and characterizing rproteins, and the many nonribosomal                   for maturation of 5.8S rRNA. The putative RNA helicases <br /> factors that are required for ribosome biogenesis, in Saccha-                 Fal1p, Dbp4p, Rok1p, and Rrp3p (25, 30, 39, 56) are required <br /> romyces cerevisiae. The completion of the yeast genome project                for 40S subunit synthesis, and the putative helicases Dbp3p, <br /> secured the identification of 137 genes (59 of which were du-                 Dbp6p, Dbp7p, Dob1p, Drs1p, and Sbp4p are required for 60S <br /> plicated) that encode 32 40S and 46 60S rproteins (32). The                   synthesis (7, 8, 26, 41, 43, 60). <br /> 18S rRNA of 40S subunits and the 5.8S and 25S rRNAs of 60S                       In addition to the snoRNAs, nucleases, and RNA helicases, <br /> subunits are transcribed as a single 35S pre-rRNA. The orga-                  a number of other nucleolar proteins are required for ribo- <br /> nization of 35S pre-rRNA, including the locations of the ma-                  some biogenesis. The best characterized of these proteins is <br /> ture rRNAs; two internal transcribed spacers, ITS1 and ITS2;                  Nop1p, an abundant protein that is required for many steps of <br /> and two external transcribed spacers, 59 ETS and 39 ETS, is                   both 40S and 60S synthesis, including rRNA methylation and <br /> shown in Fig. 1. rRNAs are formed by ordered endo- and exo-                   processing and subunit assembly (52, 53). Nop1p is common to <br /> nucleolytic digestion of the spacer sequences and by covalent                 a large number of snoRNAs (45) and associates with four <br /> modifications that include ribose and <span id='am-6' about='xsp:base' typeof='owl:Thing'>base</span> methylation and                    nucleolar proteins, Nop56p, Nop58p, Nop77p/Nop4p, and <br /> conversion of uridine residues to pseudouridine (27, 40, 61).                 Sof1p (4, 13, 21, 50). Several other proteins, including Gar1p, <br />    Many, but certainly not all, trans-acting factors that function            Mpp10p, Sof1p, and Ssb1p, were shown to interact with one or <br /> in pre-rRNA processing and ribosome assembly have been                        more snoRNAs (6, 11, 15, 21). Genetic depletion of Gar1p, <br /> identified. Four small nucleolar RNAs (snoRNAs), U3, U14,                     Mpp10p, and Sof1p impairs 18S synthesis and 40S subunit <br /> snR10, and snR30 (20, 28, 36, 51), have been implicated in                    formation (11, 15, 21). On the other hand, Nip7p, Nop2p, <br /> pre-rRNA cleavages leading to the synthesis of 18S rRNA, but                  Nop56p, Nop58p, and Nop77p/Nop4p are involved in 60S sub- <br /> most snoRNAs function as guides for enzymes that catalyze                     <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> synthesis (4, 13, 19, 50, 62). Rrp5p has been implicated in <br /> either rRNA methylation or pseudouridinylation (48, 54). Sev-                 18S and 5.8S rRNA synthesis, since its deficiency blocks cleav- <br /> eral nucleases are responsible for endo- and exonucleolytic                   ages in the 59 ETS and ITS1, leading to accumulation of an <br /> processing and for degradation of the spacer sequences. Cleav-                unusual 24S intermediate (55). Although these nucleolar pro- <br /> ages at site A0 and at a site in the 39 ETS are dependent on                  teins are implicated in the process of ribosome biogenesis, the <br /> Rnt1p, a homologue of prokaryote RNase III (12). RNase MRP                    biochemical activities for most are unknown. <br />                                                                                  We previously characterized Nip7p in S. cerevisiae (62). <br />                                                                               Nip7p homologues have been identified in, for example, hu- <br />   * Corresponding author. Mailing address: Department of Biology,             mans, Caenorhabditis elegans, and Arabidopsis thaliana. The <br /> University of Rochester, Rochester, NY 14627. Phone: (716) 275-3890.          depletion of Nip7p preferentially impairs the synthesis of 60S <br /> Fax: (716) 275-2070. E-mail: dasg@uhura.cc.rochester.edu.                     subunits, most probably due to a kinetic delay in 27S pre- <br />  <br />                                                                        1518 <br />  VOL. 19, 1999                                                            Nip7p-ASSOCIATED PROTEINS AND RIBOSOME BIOGENESIS                                       1519 <br />  <br />  <br />                                                                                                        TABLE 1. Plasmids used in this study <br />                                                                                                                                                             Source or <br />                                                                                            Plasmid                  Relevant characteristics <br />                                                                                                                                                             reference <br />                                                                                     pBTM116              lexA DNA-binding domain, TRP1 2mm                2 <br />                                                                                     pGAD-424             GAL4 activation domain, LEU2 2mm                 2 <br />                                                                                     pBTM-NIP7            lexA DNA-binding domain::NIP7 TRP1 2mm           This   study <br />                                                                                     pACT-RRP43           GAL4 activation domain::RRP43 LEU2 2mm           This   study <br />                                                                                     pACT-NOP8            GAL4 activation domain::NOP8 LEU2 2mm            This   study <br />                                                                                     pGFP-N-FUS           MET25::GFP URA3 CEN6                             38 <br />                                                                                     pGFP-RRP43           GFP::RRP43 URA3 CEN6                             This   study <br />                                                                                     pGFP-NOP8            GFP::NOP8 URA3 CEN6                              This   study <br />                                                                                     pDN291               GFP URA3 CEN4                                    37 <br />                                                                                     YCpGAL-RRP43         GAL::PrtA::RRP43 URA3 CEN4                       This   study <br />                                                                                     YCpGAL-NOP8          GAL::PrtA::NOP8 URA3 CEN4                        This   study <br />  <br />  <br />  <br />  <br />                                                                                     striction sites (XbaI and SalI for RRP43 and HindIII and SalI for NOP8) and <br />                                                                                     inserted at the C terminus of GFP into the vector pGFP-N-FUS (38), generating <br />                                                                                     the vectors pGFP-RRP43 and pGFP-NOP8. pDN291 was used for the expres- <br />                                                                                     sion of GFP alone (37). Plasmids YCpGAL-RRP43 and YCpGAL-NOP8 con- <br />                                                                                     tain a protein A (PrtA) tag fused to the N termini of Rrp43p and Nop8p, <br />                                                                                     respectively, under the control of the GAL1 promoter. The vector YCpGAL- <br />                                                                                     RRP43 was constructed by ligating the following four DNA fragments: the DNA <br />                                                                                     of the vector YCplac33 (14) digested with EcoRI and SalI, the GAL1 promoter <br />                                                                                     isolated <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> YCpGAL-NIP7 (62) digested with EcoRI and NdeI, the RRP43 <br />                                                                                     ORF isolated from pGFP-RRP43 digested with XbaI and SalI, and a double <br />    FIG. 1. Structure and relevant processing steps of the 35S pre-rRNA in S.        Staphylococcus aureus PrtA immunoglobulin G (IgG)-binding domain PCR am- <br /> cerevisiae. In the 35S pre-rRNA, sequences of the mature 18S, 5.8S, and 25S         plified from pBD20 (10) as an NdeI-XbaI fragment. YCpGAL-NOP8 was con- <br /> rRNAs are separated by two internal transcribed spacers (ITS1 and ITS2) and         structed by a similar strategy, except that the NOP8 ORF was isolated from <br /> flanked by 59 and 39 external transcribed spacers. The 59 ETS is removed by         pGFP-NOP8 as an EcoRI-SalI DNA fragment and the PrtA tag was an NdeI- <br /> sequential cleavages at A0 and A1 to generate 32S pre-rRNA. Processing of ITS1      EcoRI fragment. Plasmids pACT-RRP43 and pACT-NOP8, which bear genes <br /> and ITS2 is more complex. Cleavage of 32S pre-rRNA at A2 separates 20S              encoding hybrid proteins of the Gal4p activation domain and Rrp43p or Nop8p, <br /> pre-rRNA from 27SA2 pre-rRNA. Cytoplasmic processing of 20S pre-rRNA to             respectively, were isolated from L40-derivative strains that showed positive two- <br /> mature 18S rRNA occurs by cleavage at D. 59 processing of 27SA2 pre-rRNA            hybrid interaction with Nip7p. pGAD424 was used for expression of the Gal4p <br /> follows alternative pathways. Approximately 90% of 27SA2 pre-rRNA undergoes         activation domain (2). <br /> cleavage at A3, producing 27SA3, which is subsequently processed at B1S to             Yeast strains, media, and genetic techniques. Yeast strains used in this work <br /> produce 27SBS. The remaining 10% is processed at B1L, which produces 27SBL          are listed in Table 2. Yeast strains were grown and analyzed as described by <br /> pre-rRNA. At the stage of 27S pre-rRNA, the mature 39 end of 25S rRNA is            Sherman and coworkers (46, 47). Various carbon sources were added to yeast <br /> formed by cleaving 27S pre-rRNAs at B2. Subsequent processing of 27SBS and          extract-peptone medium (YP) and synthetic complete medium (SC). YP-dex- <br /> 27SBL precursors is identical. Cleavage of these precursors at C2 and C1 releases   trose (YPD) or SCGlu and YPGal or SCGal contained either 2% glucose or 1% <br /> 25S rRNA and two forms of 7S pre-rRNA, 7SS and 7SL, respectively. The mature        galactose plus 1% raffinose, respectively, as a carbon source. <br /> end of 5.8SS and 5.8SL rRNAs are produced by 39359 digestion to site E of 7S           NOP8 gene disruption was achieved by transforming strain W303-1a with a <br /> pre-rRNAs. The longest form, 5.8SL rRNA, contains a 59 6- to 8-nucleotide           kanr marker (16), which was PCR amplified with primers containing sequences <br /> extension. Letters below the 35S pre-rRNA diagram (probes A to F) indicate the      overlapping the NOP8 ORF at the 59 and 39 ends. The sequences of the primers <br /> positions of the oligonucleotide probes used in Northern blot analysis.             used to amplify the kanr marker are 59CTGAAGTGAGAACTAGGTAATA <br />                                                                                     TACGACGATGGATAGTGTAATTCCAGCTGAAGCTTCGTACGC39 (59 <br />                                                                                     primer) and 59TATACTATATATGTATATATACTCACTATAGAAGAAGC <br />                                                                                     CCGCTCTGCATAGGCGACTAGTGGATCTG39 (39 primer). kanr colonies <br /> rRNA processing. Nip7p localized to the nucleolus, but sig-                         were allowed to sporulate, and tetrads were dissected. Only two spores per tetrad <br /> nificant amounts cosedimented with free 60S subunits. The                           were able to germinate. Both spores were geneticin sensitive (kan2). In order to <br /> biochemical function of Nip7p, especially its role when it is <br /> associated with free 60S subunits, is unknown. In order to learn <br /> more about the function of Nip7p, we identified several puta-                                                 TABLE 2. Yeast strains <br /> tive Nip7p-interacting proteins, using a two-hybrid screen. We <br />                                                                                                                                                          Source or <br /> report here on the interaction of Nip7p with the exosome sub-                        Strain                    Relevant characteristics <br />                                                                                                                                                          reference <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Rrp43p (34) and with a previously uncharacterized nucle- <br /> olar protein, Nop8p. Rrp43p shows sequence similarity to bac-                       W303-1a          MATa ade2-1 leu2-3,112 his3-11,15 trp1-1          M. Nomura <br /> terial RNase PH, a 39-to-59 exoribonuclease, and in vivo depletion                                     ura3-1 can1-100 <br />                                                                                     W303a            MATa ade2-1 leu2-3,112 his3-11,15 trp1-1          S. Wente <br /> of Rrp43p impairs processing of the 5.8S 39 end (34). The deple- <br />                                                                                                        ura3-1 can1-100 <br /> tion of Nop8p, which is unlike other proteins, revealed that it                     W303a            MATa/MATa ade2-1/ade2-1 leu2-3,112/               This study <br /> too is required for 60S subunit biogenesis. These results sug-                                         leu2-3,112 his3-11,15/his3-11 trp1-1/ <br /> gest that Nip7p, Rrp43p, and Nop8p are subunits of a dynamic                                           trp1-1 ura3-1/ura3-1 can1-100/can1-100 <br /> complex with essential roles in 60S subunit biogenesis.                             DG456            MATa ade2-1 leu2-3,112 his3-11,15 trp1-1          This study <br />                                                                                                        ura3-1 can1-100 Nop8::KAN p(URA3 <br />                                                                                                        GAL::PrtA::NOP8) <br />                         MATERIALS AND METHODS                                       DG457            MATa/MATa NOP8/nop8::KAN ade2-1/                  This study <br />    DNA analysis methods and plasmid construction. DNA cloning and electro-                             ade2-1 leu2-3,112/leu2-3,112 his3-11,15/ <br /> phoretic analysis were performed as described by Sambrook et al. (44). DNA                             his3-11 trp1-1/trp1-1 ura3-1/ura3-1 can1- <br /> sequencing was performed by the Big Dye method (Perkin-Elmer). A cDNA                                  100/can1-100 <br /> library fused to the GAL4 activation domain was obtained from the American          L40              MATa his3D200 trp1-901 leu2-3,311 ade2            18 <br /> Type Culture Collection (ATCC 87002). Plasmids used in this study are sum-                             lys2-801am URA3::(lexAop)8-lacZ LYS2:: <br /> marized in Table 1, and cloning strategies are briefly described below. The                            (lexAop)4-HIS3 <br /> lexA::NIP7 fusion used in the two-hybrid screen was constructed by inserting a      L40-0            L40 pBTM-NIP7, pGAD-424                           This study <br /> PCR-amplified NIP7 open reading frame (ORF) into the BamHI and SalI sites           L40-41           L40 pBTM-NIP7, pACT-NOP8                          This study <br /> of plasmid pBTM116 (2), generating plasmid pBTM-NIP7. For construction of           L40-61           L40 pBTM-NIP7, pACT-RRP43                         This study <br /> green fluorescent protein (GFP) fusion proteins, RRP43 and NOP8 ORFs were <br />                                                                                       a <br /> PCR amplified from yeast genomic DNA with primers containing suitable re-                 Generated by mating W303-1a and W303a. <br />  1520       ZANCHIN AND GOLDFARB                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br /> construct a conditional strain for NOP8, strain DG457 (NOP8/nop8::KAN) was            Subsequently, cells were pulse-chase labeled with 100 mCi of [methyl-3H]methi- <br /> transformed with plasmid YCpGAL-NOP8, which contains NOP8 under the                   onine (DuPont-NEN) per ml for 2 min and chased with 100 mg of unlabeled <br /> control of the GAL1 promoter, and sporulated. kanr spores carrying this plasmid       methionine per ml. At various times, samples were taken and quickly frozen in <br /> were able to grow on medium containing galactose but not on medium contain-           a dry ice-ethanol bath. Total RNA was isolated from yeast cells by the hot-phenol <br /> ing glucose.                                                                          method (24). For analysis of pre-rRNA steady-state levels, RNAs were isolated <br />    Yeast two-hybrid screen for proteins that interact with Nip7p. The host strain     from strains W303-1a and DG456 incubated in YPGal or shifted to YPD. <br /> for the two-hybrid screen, L40 (Table 2) (18), contains both yeast HIS3 and           Samples were collected for RNA extraction at time zero and 28 h after the shift <br /> Escherichia coli lacZ as reporters for two-hybrid interaction integrated into the     to YPD for W303-1a and at time zero and 6, 12, 20, and 28 h after the shift to <br /> genome. Strain L40 was transformed with plasmid pBTM-NIP7, which bears the            YPD for DG456. RNAs were separated by electrophoresis on 1.2% agarose&#226;&#8364;&#8220;6% <br /> gene that encodes a hybrid protein containing the lexA DNA-binding domain and         formaldehyde gels and transferred by Northern blotting to Hybond nylon mem- <br /> the full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> NIP7 ORF. Expression of the fusion protein was confirmed by           branes (Amersham) as described previously (44). Membranes were probed with <br />                                                                                       32 <br /> immunoblot analysis with an antibody raised against Nip7p (data not shown).              P-labeled oligonucleotides complementary to specific regions of the 35S pre- <br /> Subsequently, a large-scale transformation of L40 carrying pBTM-NIP7 was              rRNA under the hybridization conditions described previously (51) and submit- <br /> performed with a yeast cDNA library fused to the GAL4 activation domain               ted to autoradiography. The oligonucleotide probes used (see also Fig. 1) were <br /> (ATCC 87002). Approximately 6 3 104 clones were tested for auxotrophy for             59GGTCTCTCTGCTGCCGGAAATG39 (probe A); 59GCTCTCATGCTCTT <br /> histidine by plating transformed cells on selective synthetic medium. Fast-grow-      GCCAAAAC39 (probe B); 59TGTTACCTCTGGGCCCCG39 (probe C); 59GT <br /> ing colonies were transferred to nitrocellulose filters and tested for b-galactosi-   TCGCCTAGACGCTCTCTTC39 (probe D); 59CGTATCGCATTTCGCTGCG <br /> dase (b-Gal) activity, as described by Vojtek and Hollenberg (58). Plasmid DNA        TTC39 (probe E); and 59GGCCAGCAATTTCAAGTTAAC39 (probe F). <br /> was isolated from colonies showing both fast growth on selective plates and high <br /> b-Gal activity and submitted to sequencing analysis. <br />    IgG affinity column and immunoblot analysis. Two hundred optical density at                                         RESULTS <br /> 600 nm (OD600) units of exponentially growing cells was resuspended in 2 ml of <br /> ice-cold buffer A (10 mM Tris-Cl [pH 7.6], 100 mM NaCl, 5 mM MgCl2, 1 mM <br /> dithiotreitol [DTT], 1 mM phenylmethylsulfonyl fluoride [PMSF], 1 mg of apro- <br />                                                                                          Rrp43p and Nop8p interact with Nip7p. We previously de- <br /> tinin per ml, 1 mg of pepstatin A per ml, 1 mg of leupeptin per ml). Cells were       scribed the function of Nip7p, an evolutionarily conserved pro- <br /> disrupted by vortexing with 1 volume of glass beads, and extracts were cleared by     tein that is required for pre-rRNA processing and the accu- <br /> centrifugation at 30,000 3 g for 15 min. Seventy-five OD280 units of extract was      mulation of 60S subunits in S. cerevisiae (62). A two-hybrid <br /> adjusted to 1.9 ml with buffer A and incubated with 100 ml of IgG-Sepharose <br /> beads (Pharmacia) for 2 h. Subsequently, the suspension was transferred to a <br />                                                                                       screen to identify Nip7p-interacting proteins was performed by <br /> small column, which was washed extensively with buffer A, and eluted with buffer      using the complete NIP7 ORF fused to the E. coli lexA DNA- <br /> A containing 0.5 M KCl. A fraction of both PrtA-tagged Nop8p (PrtA-Nop8p)             binding domain as the bait. This bait was screened against a <br /> and PrtA-Rrp43p eluted from the IgG columns with buffer A containing 0.5 M            yeast cDNA library fused to the Gal4p activation domain in <br /> KCl, indicating that the affinity of the PrtA-tagged proteins for IgG may be less <br /> than that of native PrtA, which normally remains bound under these conditions. <br />                                                                                       L40 cells (18). His1 clones with relatively high b-Gal activities <br /> All of the steps described above were performed at 4&#194;&#176;C. Proteins were separated       were further characterized by DNA sequence analysis. Three <br /> by sodium dodecyl sulfate-polyacrylamide gel electrophoresis and electroblotted       of the positive clones contained cDNA for the exosome com- <br /> to an Immobilon-P membrane (Millipore) as described previously (62). Mem-             plex subunit Rrp43p (34), and another three clones contained <br /> branes were blocked with 2% nonfat dry milk in TST buffer (20 mM Tris-Cl [pH <br />                                                                                       an uncharacterized ORF (YOL144W in the Saccharomyces <br /> 8.0] 150 mM NaCl, 0.05% [vol/vol] Tween 20] and then probed with rabbit <br /> polyclonal antiserum raised against Nip7p. Subsequently, the blots were incu-         Genome Database). YOL144W contains a gene that encodes <br /> bated with alkaline phosphatase-conjugated anti-rabbit IgG and visualized with        a 484-amino-acid (57-kDa) polypeptide that is dissimilar to <br /> 5-bromo-4-chloro-3-indolylphosphate and nitroblue tetrazolium as previously           known proteins and lacks any telling sequence motifs. Based <br /> described (44).                                                                       on the subcellular localization of this gene&#226;&#8364;&#8482;s product (shown <br />    Subcellular localization of Rrp43p and Nop8p. The subcellular localization of <br /> Rrp43p and Nop8p was analyzed by monitoring the fluorescent signal produced           below), we named the gene NOP8 for nucleolar protein. <br /> by green fluorescent protein (GFP) fusions to the amino termini of Rrp43p and            Comparative two-hybrid analysis was performed with the <br /> Nop8p. GFP-Rrp43p and GFP-Nop8p reporter proteins were expressed from                 control strain (L40-0) and with strains carrying the Nip7p bait <br /> the vectors pGFP-RRP43 and pGFP-NOP8 (Table 1), respectively, transformed             together with prey plasmids carrying cDNAs for Rrp43p (L40- <br /> into strain W303-1a. Plasmid pDN291 (37), which bears the gene for the GFP <br /> protein, was used as a control. Images were obtained with a Leica TCS NT              61) and Nop8p (L40-41) (Table 1). The b-Gal activities of <br /> confocal microscope, and digital images were processed with MetaMorph soft-           extracts isolated from strains L40-61 and L40-41 were, respec- <br /> ware (Universal Imaging Corporation).                                                 tively, 21- and 65-fold higher than those of extracts from L40 <br />    Growth curves and analysis of Nop8p depletion. Growth rates of strains             cells (data not shown). The stringency of the selection for his- <br /> W303-1a (NOP8) and DG456 (GAL::NOP8) in YPGal and YPD cultures were <br /> analyzed as follows: exponentially growing YPGal cultures were divided into two       tidine auxotrophy was enhanced by adding 3-aminotriazole (3- <br /> fractions, and cells were harvested by centrifugation and resuspended in either       AT), an inhibitor of imidazoleglycerol phosphase-dehydratase <br /> YPD or YPGal. Cultures were incubated at 30&#194;&#176;C, and the OD600 values were              (23). Whereas the three strains grew at similar rates on media <br /> determined at various time points. In order to keep cultures in exponential           selective for the vector auxotrophy markers, only L40-61 and <br /> growth, they were diluted in fresh medium whenever the OD600 reached 0.8. <br /> Samples were collected at various times from YPD cultures for the analysis of         L40-41 cells grew on 3-AT-supplemented plates (Fig. 2A). <br /> Nop58p depletion. For isolation of cell extracts, cells were harvested by centrif-    L40-41 (Nop8p) cells grew faster than L40-61 (Rrp43p) cells <br /> ugation, resuspended in 200 ml of breaking buffer (20 mM HEPES-KOH [pH                on 3-AT plates, which is consistent with their higher b-Gal <br /> 7.4], 2 mM Mg diacetate, 100 mM KCl, 1 mM DTT, 0.5 mM EDTA, 1 mM                      content. <br /> PMSF), and disrupted by vortexing in the presence of 1 volume of glass beads. <br /> Cell extracts was cleared by centrifugation and submitted to immunoblot analysis         The two-hybrid results suggest that Nip7p associates in vivo <br /> as described above. Rabbit antibody against translation initiation factor eIF-2a      with Rrp43p and Nop8p. Evidence for their association in a <br /> (kindly provided by John McCarthy) was used as an internal control.                   complex was obtained by the copurification of Nip7p with <br />    Polysome profile analysis. For polysome profile analysis cell extracts were        PrtA-tagged Rrp43p (PrtA-Rrp43p) and Nop8p (PrtA-Nop8p). <br /> isolated from 300-ml cultures grown to mid-exponential phase in YPGal or <br /> shifted to YPD for 12 h. Following addition of 3 ml of cycloheximide (10 mg/ml) <br />                                                                                       Nip7p was present in IgG-Sepharose column eluates from <br /> to the cultures, cells were harvested by centrifugation and resuspended in 0.5 ml     whole-cell extracts containing either PrtA-Rrp43p or PrtA- <br /> of breaking buffer (20 mM HEPES-KOH [pH 7.4], 2 mM Mg diacetate, 100 mM               Nop8p but not from control extract (Fig. 2B and C). With the <br /> KCl, 1 mM DTT, 1 mM PMSF, 100 mg of cycloheximide per ml). Cell extracts              same starting amounts of extract, more Nip7p reproducibly <br /> and sucrose gradients were prepared as described previously (62). Polysomes <br /> were separated by centrifugation at 40,000 rpm for 4 h at 4&#194;&#176;C with a Beckman <br />                                                                                       copurified with PrtA-Nop8p than with PrtA-Rrp43p. This re- <br /> SW41 rotor. Gradients were fractionated with a Buchler Auto-densiflow IIC             sult is consistent with results of the two-hybrid analysis, where <br /> fractionator and monitored at 254 nm with a UA-5 absorbance-fluorescence              L40-41 cells (Nop8p) showed both higher b-Gal activity and <br /> monitor (ISCO).                                                                       faster growth on 3-AT medium than L40-61 cells (Rrp43p). <br />    Pulse-chase labeling, rRNA electrophoresis, and Northern blot analysis. Met- <br /> abolic labeling of rRNA was performed as described previously (59, 62). Expo- <br />                                                                                          Subcellular localization of Rrp43p and Nop8p. Proteins in- <br /> nentially growing cultures of W303-1a (NOP8) and DG456 (GAL::NOP8) were               volved in pre-rRNA processing and ribosome biogenesis are <br /> shifted from SCGal to SCGlu lacking methionine and incubated at 30&#194;&#176;C for 18 h.        usually located in the nucleolus, although there is precedent <br />  VOL. 19, 1999                                                          Nip7p-ASSOCIATED PROTEINS AND RIBOSOME BIOGENESIS                       1521 <br />  <br />  <br />                                                                                   was produced by transforming diploid W303 with a PCR-am- <br />                                                                                   plified KAN gene targeted for insertion into NOP8 by the <br />                                                                                   method described by Gu    &#194;&#168;ldener et al. (Materials and Methods) <br />                                                                                   (16). Sporulation of a kanr transformant, followed by tetrad <br />                                                                                   dissection analysis revealed that only two spores per tetrad <br />                                                                                   germinated and grew to form visible colonies (data not shown). <br />                                                                                   Because geneticin sensitivity (Kan2) segregated with spore <br />                                                                                   viability and PCR analysis indicated that the KAN gene had <br />                                                                                   inserted into the NOP8 gene (not shown), we conclude that <br />                                                                                   NOP8 is an essential, single-copy gene. <br />                                                                                      The diploid nop8::KAN disruption strain (DG457) was used <br />                                                                                   to construct a NOP8 conditional strain in order to study the <br />                                                                                   physiological effects of Nop8p depletion. Plasmid YCpGAL- <br />                                                                                   NOP8, in which NOP8 is under the GAL1-inducible promot- <br />                                                                                   er, was transformed into DG457 cells (NOP8/nop8p::KAN). <br />                                                                                   Transformants were sporulated, and haploid cells containing <br />                                                                                   the nop8::KAN disruption and the YCpGAL-NOP8 plasmid <br />                                                                                   were identified based on geneticin resistance (kanr) and con- <br />                                                                                   ditional growth on galactose. One of these spores (DG456) was <br />                                                                                   further characterized. DG456 and control NOP8 W303-1a cells <br />                                                                                   grew similarly on galactose (Fig. 4A and B); however, DG456 <br />                                                                                   did not grow on glucose plates (Fig. 4B). The growth of DG456 <br />                                                                                   cells after a shift from galactose to glucose liquid medium <br />                                                                                   slowed beginning at about 6 h and was severely impeded by 10 <br />                                                                                   to 12 h (Fig. 4A). The reduction in the growth rate of DG456 <br />                                                                                   cells on glucose correlated with the decrease in cellular levels <br />    FIG. 2. Nip7p interacts with Rrp43p and Nop8p. (A) Two-hybrid-assay in-        of PrtA-Nop8p, which were barely detectable after 6 h (Fig. <br /> teraction of Nip7p with Rrp43p and Nop8p. Strains L40-41 (ACT::NOP8) and          4C). In contrast to the effect of glucose on PrtA-Nop8p levels, <br /> L40-61 (ACT::RRP43) grow on medium supplemented with 3-AT that is suffi-          the levels of an internal control protein, eIF-2a, should not and <br /> cient to prevent growth of control strain L40-0 (ACT). (B and C) Copurification   did not decrease during the same time course (Fig. 4C). <br /> of Nip7p with PrtA-Rrp43p and PrtA-Nop8p. Whole-cell extracts were isolated <br /> from cells expressing PrtA-Rrp43 and PrtA-Nop8p. PrtA-tagged fusions were            Depletion of Nop8p reduces cellular levels of 60S ribosomes. <br /> purified by IgG-Sepharose chromatography. Control extracts were prepared          Polysome analysis was a key to gaining insight into the role of <br /> from W303-1a cells, designated RRP43 in panel B and NOP8 in panel C. Samples      Nip7p in ribosome biogenesis (62). Therefore, the polysome <br /> from the last wash (lanes W) and from the 0.5 M KCl eluate (lanes E) were         profile of Nop8p-depleted cells was investigated by sucrose <br /> submitted to immunoblot analysis with anti-Nip7p serum. See Materials and <br /> Methods for details. LW, minus leucine and tryptophan; HLW, minus histidine,      density gradient analysis. Extracts were prepared from DG456 <br /> leucine, and tryptophan.                                                          (GAL::NOP8) and control W303-1a (NOP8) cells maintained <br />                                                                                   in galactose or shifted to glucose for 12 h. As shown in Fig. 5A <br /> for certain nucleases and the RNA helicase Dob1p/Mtr4p to                         and B, the polysome profiles from galactose-grown DG456 and <br /> also be localized throughout the nucleoplasm and in the cyto-                     W303-1a cells were virtually indistinguishable. In contrast, the <br /> plasm (8, 22, 29, 34). GFP-Nip7p was previously shown to be                       polysome profile of glucose-grown DG456 cells was strikingly <br /> predominately nucleolar; however, native Nip7p cosedimented                       different from that of W303-1a cells (Fig. 5C and D). Specifi- <br /> with free 60S ribosomes and thus Nip7p may function both in                       cally, the polysome profile of extracts from Nop8p-depleted <br /> the nucleus and in the cytoplasm (62). Since Rrp43p and                           cells displayed a reduction in the level of free 60S subunits, the <br /> Nop8p interact with Nip7p in the two-hybrid system and Nip7p                      appearance of half-mer polysomes, and a decrease in the total <br /> copurifies with PrtA-Rrp43p and PrtA-Nop8p, it was impor-                         amount of polysomes (Fig. 5D). Because Nip7p-depleted cells <br /> tant to determine if these two proteins colocalized with Nip7p.                   exhibited these same defects (62), these results support the <br /> GFP-Rrp43p and GFP-Nop8p were localized by direct fluores-                        hypothesis that Nip7p and Nop8p function together during the <br /> cence in exponentially growing cultures (Fig. 3). Mitochondrial                   same stage(s) of ribosome biogenesis. <br /> and nuclear DNAs were stained with Hoechst 3323 (Materials                           Pre-rRNA processing is defective in cells depleted of Nop8p. <br /> and Methods). When expressed alone, GFP was distributed                           The polysome analyses shown above indicated that Nop8p is <br /> about equally across the nuclear envelope and was excluded                        required for maintenance of normal 60S ribosome levels. For <br /> from vacuoles (Fig. 3, bottom images). GFP-Nop8p fluores-                         this reason, and because Nip7p-depleted cells were defective in <br /> cence, when GFP-Nop8p was superimposed over the Hoechst                           rRNA processing, we performed [methyl-3H]methionine pulse- <br /> 3323-stained nucleus, appeared as a cap-like extension of the                     chase labeling and Northern blot analysis of rRNA synthesis in <br /> nucleus, which is characteristic of the nucleolus (Fig. 3, middle                 normal and Nop8p-depleted cells. The kinetics of rRNA pro- <br /> images). GFP-Rrp43p appeared to localize to the nucleo-                           cessing were analyzed by pulse-chase labeling with [methyl-3H] <br /> plasm, nucleolus, and, to a lesser extent, cytoplasm (Fig. 3, top                 methionine and cells shifted from galactose to glucose medium <br /> images). Therefore, most of GFP-Nip7p and GFP-Nop8p and                           for 18 h (Fig. 6). In control cells, methyl-3H-labeled precursors <br /> a portion of GFP-Rrp43p localized to the nucleolus. The co-                       were quickly chased to mature rRNAs, whereas cells depleted <br /> localization of these three proteins is consistent with the in vivo               of Nop8p showed a drastic reduction in 25S rRNA labeling <br /> association of Nip7p with Nop8p and Rrp43p, although not                          and a relatively moderate decrease in 18S rRNA labeling. In <br /> necessarily in a single complex. By immunoblot analysis, PrtA-                    comparison to normal cells, Nop8p-depleted cells also tran- <br /> Nop8p did not appear to cosediment with 60S ribosomal sub-                        siently labeled 35S pre-rRNA and an aberrant 23S pre-rRNA <br /> units (not shown).                                                                (Fig. 6). Because there was no accumulation of unprocessed <br />    Gene disruption analysis and construction of a conditional                     27S pre-rRNA that could account for the defect of 25S syn- <br /> lethal NOP8 mutant. A complete gene replacement of NOP8                           thesis, these data suggest that Nop8p depletion may trigger the <br />  1522      ZANCHIN AND GOLDFARB                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Subcellular localization of GFP, GFP-Rrp43p, and GFP-Nop8p in logarithmically growing cells. Positions of nuclei were determined by Hoechst staining. <br /> Overlay images show the superimposition of the GFP (green), Hoechst (blue), and DIC (black and white). See Materials and Methods for details. <br />  <br />  <br />  <br />  <br /> premature degradation of 27S pre-rRNA or 25S rRNA. Con-                           Long periods of Nop8p depletion, such as 28 h (Fig. 7), led to <br /> sistent with this, ethidium bromide staining showed that Nop8p                    a reduction in the total amount of all pre-rRNAs. Finally, <br /> depletion resulted in the simultaneous decline of 5.8S and 25S                    Northern hybridization with probe E (Fig. 1) did not detect any <br /> rRNAs (data not shown).                                                           aberrant precursors associated with 5.8S rRNA synthesis (data <br />    Steady-state pre-rRNA levels were analyzed from control                        not shown). <br /> and Nop8p-depleted cells after the shift to glucose medium for <br /> up to 28 h. Total RNAs were isolated and submitted to North-                                                   DISCUSSION <br /> ern hybridization with oligonucleotide probes specific for the <br /> 59 ETS, ITS1, 5.8S rRNA, and ITS2 (Fig. 1). Consistent with                          Nip7p is an evolutionarily conserved protein that is required <br /> the pulse-chase labeling results, Nop8p-depleted cells accumu-                    for pre-rRNA processing and 60S ribosome biogenesis in <br /> lated both 35S and 23S pre-rRNAs (Fig. 7). The 23S pre-rRNA                       S. cerevisiae (62). We have characterized the interactions of <br /> incorporates the 59 ETS, 18S rRNA, and the ITS1 region                            Nip7p with Rrp43p and Nop8p, all three of which are also <br /> upstream of site A3, since it can be detected by probes A, B,                     required for 60S ribosome biogenesis. Two-hybrid and bio- <br /> and C (Fig. 7A to C). The decrease in the level of 20S pre-                       chemical analyses provided genetic and physical evidence for <br /> rRNA observed in Nop8p-depleted cells appears to be a con-                        the existence of Nip7p-Rrp43p and Nip7p-Nop8p complexes. <br /> sequence of the accumulation of upstream precursors, such as                      Although both the two-hybrid and the copurification experi- <br /> 35S and 23S pre-rRNAs. Hybridization with probe D, which                          ments suggest the existence of Nip7p-Rrp43p and Nip7p-Nop8p <br /> detects all forms of 27S pre-rRNA (Fig. 7D) and probes E and                      interactions, the mechanism of interaction remains to be de- <br /> F (Fig. 1 and data not shown) indicated that the 27S pre-rRNA                     termined since in both cases it may occur via a linker or <br /> levels decreased in accordance with the decline of the Nop8p                      adapter factor(s) in a multisubunit complex. We have not, for <br /> levels. It is unclear from these data whether the decrease in                     example, ruled out the possibility that these proteins associate <br /> 27S pre-rRNA levels is due to destabilization caused by Nop8p                     with a snoRNA. We also do not know if Nip7p interacts <br /> depletion, because at least some reduction of 27S pre-rRNA                        simultaneously and/or exclusively with Nop8p and Rrp43p. <br /> levels can be attributed to accumulation of 35S pre-rRNA.                         These issues will have to be resolved by a structural analysis of <br />  VOL. 19, 1999                                                           Nip7p-ASSOCIATED PROTEINS AND RIBOSOME BIOGENESIS                                  1523 <br />  <br />  <br />  <br />  <br />    FIG. 4. In vivo depletion of Nop8p causes growth arrest. (A) Growth curves        FIG. 5. Nop8p-depleted cells show defects associated with decreasing 60S <br /> of W303-1a (NOP8) and DG456 (GAL::NOP8) cultured at 30&#194;&#176;C in YPGal and              subunit levels. Polysome profiles were analyzed from strains W303-1a (NOP8) <br /> shifted to YPD at time zero. OD600 values are plotted as log ODt/t0 units, where   and DG456 (GAL::NOP8) by sedimentation through 15 to 50% sucrose gradi- <br /> t0 is the initial OD600 and t is time in hours after transfer to YPD. (B) Growth   ents. Cultures were either grown in YPGal (A and B) or shifted to YPD for 12 h <br /> at 30&#194;&#176;C of W303-1a and DG456 on YPGal and YPD plates. (C) Immunoblot               (C and D). <br /> analysis showing levels of PrtA-Nop8p and endogenous eIF-2a in DG456 at <br /> various times after a shift from YPGal to YPD. The same amount of total protein <br /> was loaded in each lane. See Materials and Methods for details. <br />                                                                                    the 7S pre-rRNA 39 end occurs subsequently to the cleavage at <br />                                                                                    C2, Nip7p and Rrp43p may interact for the removal of ITS2 <br />                                                                                    during the process of 5.8S and 25S rRNA maturation. <br /> Nip7p-containing complexes. The functional interaction be-                            Nop8p function was investigated in Nop8p-depleted cells. <br /> tween Nip7p and Nop8p is strongly supported by their colo-                         Nop8p depletion led to a decrease in free 60S subunit levels <br /> calization in nucleoli and by their similar defects in pre-rRNA                    and the appearance of half-mer polysomes. Similar defects <br /> processing and 60S ribosome biogenesis caused by their deple-                      have been described for mutants defective in specific rproteins <br /> tion.                                                                              of the 60S subunit (9, 35, 42, 57) and for deficiencies in other <br />    Evidence for a functional interaction between Nip7p and                         factors required for pre-rRNA processing and 60S subunit <br /> Rrp43p is also compelling, but this situation is more complex                      assembly (4, 7, 8, 13, 19, 26, 41, 43, 50, 60), including Nip7p <br /> owing to the less restricted localization of Rrp43p to the nu-                     (62). We conclude that Nop8p is not an rprotein because it did <br /> cleoplasm, cytoplasm, and nucleolus and its association with                       not cosediment with cytoplasmic ribosomes on sucrose gradi- <br /> the exosome. The exosome contains a minimum of five puta-                          ents (not shown) and it localized exclusively to nucleoli. The <br /> tive 39-to-59 exonucleases that are each required for matura-                      effects of Nop8p-depletion on pre-rRNA processing are con- <br /> tion of 5.8S rRNA (34). Whereas exosome subunits Rrp4p,                            sistent with a role for Nop8p in 60S subunit biogenesis. Syn- <br /> Rrp41p or Ski6p, and Rrp44p exhibit in vitro 39-to-59 exoribo-                     thesis of 25S rRNA was drastically reduced. In addition, <br /> nuclease activities, such an activity for Rrp43p is implied only                   Northern blot analysis revealed a reduction in the steady-state <br /> from its sequence similarity to bacterial RNase PH (34). The <br /> function of certain exosomal subunits is not restricted to 5.8S <br /> rRNA processing. Benard et al. (3) have reported that the <br /> ski6-2 allele of Rrp41p/Ski6p accumulated a 38S particle de- <br /> rived from the 60S subunit containing a truncated part of the <br /> 25S rRNA and lacking the 5.8S rRNA. In addition, Anderson <br /> and Parker (1) have shown that Rrp4p and Rrp41p/Ski6p also <br /> participate in mRNA degradation, and there is evidence that a <br /> fraction of Rrp4p is present in the cytoplasm (34). Our results <br /> with GFP-Rrp43p suggest that it too may be partially localized <br /> in the cytoplasm. <br />    In addition to Nip7p, other polypeptides associate with exo- <br /> some subunits. Dob1p interacts genetically with Rrp4p (8), and <br /> the in vivo depletion of Dob1p and Rrp4p produce the same <br /> pre-rRNA processing defects (8, 33). With Nip7p and Rrp43p, <br /> however, in vivo depletion resulted in the accumulation of <br /> distinct precursors along the pathway leading to the synthesis <br /> of the 60S subunit rRNAs (34, 62). In Nip7p-depleted cells, the <br /> reduced synthesis of 60S subunits is associated with an accu- <br /> mulation of 27S pre-rRNA (62). The exosome complex, which                             FIG. 6. Pulse-chase labeling of rRNA synthesis of Nop8p-depleted cells. <br />                                                                                    Pulse-chase labeling with [methyl-3H]methionine was performed with W303-1a <br /> includes Rrp43p, catalyzes the exonucleolytic digestion of the                     (NOP8) and DG456 (GAL::NOP8) shifted from SCGal to SCGlu lacking me- <br /> part of ITS2 that is left on the 7S pre-rRNA after cleavage of                     thionine for 18 h. RNA samples were collected every 2 min. Total RNA (OD 5 <br /> the 27S pre-rRNA at C2 (34). Although 39-to-59 processing of                       0.9) was loaded in each lane. See Materials and Methods for details. <br />  1524      ZANCHIN AND GOLDFARB                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Analysis of pre-rRNA steady-state levels in Nop8p-depleted cells. RNA was analyzed from strain W303-1a (NOP8) at time zero and 28 h after transfer to <br /> YPD and from strain DG456 (GAL::NOP8) at time zero and 6, 12, 20, and 28 h after transfer to YPD. Equivalent amounts of total RNA were loaded in each lane. <br /> (A) Probe A, complementary to sequences upstream of site A0 in the 59 ETS; (B) probe B, complementary to a region downstream of the 18S rRNA 39 end and <br /> upstream of site A2 in ITS1; (C) probe C, complementary to sequences between sites A2 and A3 in ITS1; (D) probe D, complementary to sequences between sites C1 <br /> and C2 in ITS2. See Materials and Methods and Fig. 1 for details. <br />  <br />  <br />  <br />  <br /> levels of 27S and 20S pre-rRNAs. These data are, however,                         activities. While the identification of factors such as Nip7p and <br /> insufficient to determine whether the decrease in 25S rRNA                        Nop8p is a prerequisite to understanding ribosome biogenesis, <br /> formation is due to destabilization of 27S precursors or is a                     the elucidation of their mechanistic role in ribosome biogen- <br /> consequence of premature degradation of the 60S subunit.                          esis may have to wait for the development of in vitro rRNA <br /> Both pulse-chase labeling and Northern analysis of Nop8p-                         processing and ribosome assembly assays. <br /> depleted cells detected the accumulation of the 35S pre-rRNA <br /> and the appearance of an unusual 23S precursor, indicating                                                   <br /> that multiple steps of pre-rRNA processing are directly or <br />                                                                                      <br /> indirectly blocked or delayed. The accumulation of 35S and                <br /> 23S pre-rRNAs has been described for deficiencies in other               <br /> factors that are required for 60S subunit biogenesis, including                <br /> Nip7p, Dbp3p, Dbp7p, Nop2p, Nop77p/Nop4p, and Nop56p                               <br /> (4, 7, 13, 19, 50, 60, 62).                                                    <br />    Nip7p, Rrp43p, and Nop8p are all required for processing of <br /> 60S subunit rRNAs. Pre-rRNA processing analyses suggest,                                                         <br /> however, that they are required for distinct steps during 60S                       <br /> rRNA maturation. In vivo depletion of Rrp43p affects process-                         <br /> ing of the 5.8S rRNA 39 end leading to accumulation of 7S                              <br /> pre-rRNA (34), which is generated by cleavage of the 27S                            <br /> pre-rRNA at C2. A deficiency of Nip7p, however, causes the                            <br /> accumulation of aberrant precursors containing 5.8S sequence                          <br /> (62). In Nop8p-depleted cells, no accumulation of the 27S                          <br /> precursor was observed and the defect in 60S subunit forma-                          <br /> tion in these cells may be due to 27S pre-rRNA or 25S rRNA                           <br />                                                                                       <br /> degradation.                                                                     <br />    Non-rprotein mutants with deficiencies in 60S subunit syn-                       <br /> thesis fall into two main groups. Deficiencies in Nip7p, Dob1p,                      <br /> Dbp3p, Drs1p, Nop2p, and Nop56p (8, 13, 19, 41, 60, 62) arrest              <br />                                                                                    <br /> 25S and 5.8S rRNA synthesis at the level of 27S pre-rRNA and                         <br /> accumulate 27S precursors. Deficiencies in Nop8p, Dbp6p,                      <br /> Dbp7p, and Nop77p/Nop4p (4, 7, 26, 50) also arrest 25S and                        <br /> 5.8S synthesis but do not accumulate 27S pre-rRNA, which                            <br />                                                                                  <br /> presumably is targeted for degradation. In both cases, the final                   <br /> result is a deficit of 60S subunits. Many of these nucleolar                  <br /> proteins, including Nip7p and Nop8p, lack known enzymatic                           <br /> </body></html>