<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Scm3, an essential Saccharomyces cerevisiae <br /> centromere protein required for G2/M <br /> progression and Cse4 localization <br /> A universal mark of centromeric chromatin is its packaging by a                 are larger, less well defined at the sequence level, and which often <br /> variant of histone H3 known as centromeric H3 (CenH3). The                      contain repetitive elements. For example, Schizosaccharomyces <br /> mechanism by which CenH3s are incorporated specifically into                    pombe centromeres are 40&#226;&#8364;&#8220;100 kbp in <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> and consist of 4- <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> centromere DNA or the specialized function they serve there is not              7-kbp nonhomologous central cores flanked by repetitive sequences <br /> known. In a genetic approach to identify factors involved in CenH3              (12). Mammalian and higher plant centromeres consist of mega- <br /> deposition, we screened for dosage suppressors of a temperature-                bases of highly repetitive satellite DNAs (4, 13). The point centro- <br /> sensitive cse4 allele in Saccharomyces cerevisiae (Cse4 is the S.               meres of S. cerevisiae and close relatives seem to be a recent <br /> cerevisiae CenH3). Independent screens yielded ORF YDL139C,                     evolutionary invention, because the common ancestor to point and <br /> which we named SCM3. Dosage suppression by SCM3 was specific                    regional centromere yeasts seems to be a fungus with regional <br /> for alleles affecting the histone fold domain of Cse4. Copurification           centromeres (1). <br /> and two-hybrid studies showed that Scm3 and Cse4 interact in                       Here, we describe the discovery and characterization of Scm3, an <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                  GENETICS <br /> vivo, and chromatin immunoprecipitation revealed that Scm3, like                essential S. cerevisiae centromere protein. Genetic and biochemical <br /> Cse4, is found associated with centromere DNA. Scm3 contains two                analysis shows that Scm3 is required for mitosis, possibly by serving <br /> essential protein domains, a Leu-rich nuclear export signal and a               as an assembly and/or targeting factor for Cse4. Scm3 contains an <br /> heptad repeat domain that is widely conserved in fungi. A condi-                essential protein domain that is widely conserved in fungi, implying <br /> tional scm3 allele was generated to allow us to deplete Scm3. Upon              a function not limited to point centromeres. We suggest that S. <br /> Scm3 depletion, cells undergo a Mad2-dependent G2/M arrest, and                 cerevisiae Scm3 defines a previously undescribed family of proteins <br /> centromere localization of Cse4 is perturbed. We suggest that S.                involved in CenH3 localization. <br /> cerevisiae Scm3 defines a previously undescribed family of fungal <br /> kinetochore proteins important for CenH3 localization.                          Results <br />                                                                                 YDL139C Is an Allele-Specific Suppressor of Temperature-Sensitive <br /> centromeric H3 &#229;&#8230;&#169; chromosome segregation &#229;&#8230;&#169; kinetochore &#229;&#8230;&#169; yeast &#229;&#8230;&#169; CENP-A          (Ts) cse4 Alleles. Two independent screens for dosage suppressors of <br />                                                                                 the S. cerevisiae Ts cse4-1 allele yielded, in addition to CSE4, only <br />                                                                                 the gene YDL139C. The cse4-1 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation</span> causes an Ala to Thr <br /> C    entromeres are the chromosomal sites at which microtubules <br />      attach during mitotic and meiotic prometaphase. Mediating <br /> the attachment is the kinetochore, a complex structure com- <br />                                                                                 substitution in helix 3 of the Cse4 histone fold domain (HFD). <br />                                                                                 Suppressor screens of two other Ts cse4 HFD alleles, cse4-107 <br /> posed of dozens of proteins, many of which are evolutionarily                   (Q219D) and cse4-162 (W178T), also yielded CSE4 and YDL139C, <br /> conserved across the plant, animal, and fungal kingdoms (1). A                  whereas YDL139C was not identified in any of several dosage <br /> universal feature of centromere DNA is its packaging by nu-                     suppressor screens performed using Ts cse4 alleles affecting the <br /> cleosomes containing a variant of histone H3 known as centro-                   essential END domain located in the N terminus (14). Direct tests <br /> meric H3 (CenH3) (2). CenH3s [CENP-A in humans, Cid in                          confirmed the allele-specificity [supporting information (SI) Fig. 6]. <br /> Drosophila, HTR12 in Arabidopsis, and Cse4 in yeast (3)] are                    High-copy YDL139C plasmids also suppressed the cse4-1 chromo- <br /> essential in all organisms tested, although the mechanism by                    some missegregation <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> (data not shown), so the suppres- <br /> which they are specifically deposited on the centromere DNA or                  sor gene was named SCM3 (suppressor of chromosome missegre- <br /> the function they serve there are not known (4). In contrast to                 gation). SCM3 gene disruption was lethal, indicating that SCM3 is <br /> the conservation observed for many centromere proteins, cen-                    an essential gene. Introducing high-copy SCM3 plasmids into WT <br /> tromere DNAs are diverse. They vary widely in size, and, with                   CSE4 strain backgrounds had no detectable phenotypic effect. <br /> the exception of S. cerevisiae and its close relatives, no unique <br />                                                                                 Essential Domains Within Scm3. SCM3 encodes a 223-amino acid <br /> centromere identifier sequence has been defined (5). It is <br /> proposed that a unique chromatin structure conferred by CenH3                   protein containing several sequence motifs of potential functional <br /> provides the epigenetic mark to signal kinetochore assembly (6). <br />    The first centromeres cloned were the &#226;&#8364;&#732;&#226;&#8364;&#732;point&#226;&#8364;&#8482;&#226;&#8364;&#8482; centromeres of               Author contributions: S.S., M.F.-H., and R.E.B. designed research; S.S., K.R., S.W., L.M., and <br /> Saccharomyces cerevisiae. They are &#226;&#172;&#8225;125 bp in length and are                    R.E.B. performed research; S.S., M.F.-H., and R.E.B. analyzed data; and R.E.B. wrote the <br /> characterized by conserved DNA elements (CDEs) unique to                        paper. <br /> centromere (CEN) DNA in this organism (7). CDEI is the degen-                   The authors declare no conflict of interest. <br /> erate octanucleotide RTCACRTG, CDEII is 79&#226;&#8364;&#8220;88 bp of highly                      This article is a PNAS Direct Submission. <br /> AT-rich DNA, and CDEIII is a 24-bp sequence that binds CBF3,                    Freely available online through the PNAS open access option. <br /> a four subunit, sequence-specific DNA binding protein that is                   Abbreviations: CenH3, centromeric H3; CEN, centromere; CDE, conserved DNA element; <br /> essential for centromere function (8). Binding of CBF3 to CDEIII                HFD, histone fold domain; NES, nuclear export signal; NLS, nuclear localization sequence; <br /> is thought to be the requisite first step in the ordered assembly of            Ts, temperature-sensitive. <br /> the S. cerevisiae kinetochore (9). The core CDEI-CDEII-CDEIII                   &#226;&#8364;&#161;To   whom correspondence should be addressed. E-mail: richard.baker@umassmed.edu. <br /> sequence is apparently assembled onto a single Cse4-containing                  This article contains supporting information online at www.pnas.org/cgi/content/full/ <br /> nucleosome (10), although disagreement exists (11). Higher organ-               0703178104/DC1. <br /> isms and other yeasts have so-called &#226;&#8364;&#732;&#226;&#8364;&#732;regional&#226;&#8364;&#8482;&#226;&#8364;&#8482; centromeres which             &#194;&#169; 2007 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703178104                                                       PNAS &#229;&#8230;&#169; June 19, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 25 &#229;&#8230;&#169; 10571&#226;&#8364;&#8220;10576 <br />                                                                                   similar to WT Scm3 (Fig. 1D). We concluded that the NES <br />                                                                                  homology domain and putative coiled-coil domain are essential for <br />                                                                                  Scm3 function. The D/E-rich domain is also essential, although <br />                                                                                  possibly only for protein stability. <br />  <br />                                                                                  Scm3 Heptad Repeat Domain Conserved in Fungi. BLAST searches of <br />                                                                                  public databases identified 45 potential Scm3 homologs, widely <br />                                                                                  distributed in the Fungi. No homologs were found in metazoans, <br />                                                                                  plants, or archae. Except for closely related sensu stricto and sensu <br />                                                                                  latu yeasts, homology to S. cerevisiae Scm3 was limited to the central <br />                                                                                  heptad repeat domain. An alignment of this region for 38 of the <br />                                                                                  homologs is shown in Fig. 2A. The heptad repeat of hydrophobic <br />                                                                                  residues located between positions 103 and 138 in the S. cerevisiae <br />                                                                                  protein is conserved (positions 6&#226;&#8364;&#8220;43 of the Logo diagram in Fig. <br />                                                                                  2 A); however, alignment required insertion of one to three gaps, <br />                                                                                  leading us to conclude that the structure of this domain is probably <br />                                                                                  not an extended coiled coil. Nonetheless, the amino acid sequence <br />                                                                                  is critical, because changing two of the Ile residues to polar amino <br />                                                                                  acids (scm3-31) resulted in loss of Scm3 function (Fig. 1C). A <br />                                                                                  phylogeny of the conserved protein domain is shown in Fig. 2B. The <br />                                                                                  phylogeny is congruent with species phylogeny (18), suggesting <br />                                                                                  orthology. All yeasts on the branch marked with the asterisk contain <br />                                                                                  CDEI-CDEII-CDEIII point centromeres; thus, the ancestral <br />                                                                                  Scm3 protein seems to have existed before the invention of point <br />                                                                                  centromeres. <br />  <br />                                                                                  The Scm3 NES Motif Is a Functional NES. The putative NES at Scm3 <br />                                                                                  residues 13&#226;&#8364;&#8220;24 is homologous to other known NESs and conforms <br /> Fig. 1. Mutational analysis of Scm3. (A) Amino acid sequence of Scm3             to the NES consensus (Fig. 3A) (15, 19, 20). The ability of this <br /> showing potential functional motifs and site-directed mutations. The nuclear     sequence to direct nuclear export in vivo was tested by using a <br /> export signal (NES) and heptad repeat (HR) domain are boxed with the             genetic assay described by Shulga et al. (21). The assay strain <br /> heptad-spaced hydrophobic residues shown in boldface italic. Potential nu- <br />                                                                                  contains an integrated HIS3 gene under control of a promoter <br /> clear localization sequences are underlined (BR1, BR2), and acidic residues in <br /> the C-terminal acidic (D/E-rich) region are shown in boldface. Deletion end- <br />                                                                                  containing LexA binding sites. The strain also expresses a chimeric <br /> points of the scm3-26 and scm3-27 alleles are indicated. (B) Schematic dia-      transcriptional activator that reports nuclear location. Cells are <br /> grams of HA-tagged WT SCM3 and site-directed scm3 mutants. (C) Plasmid           His&#226;&#171;&#185; when the activator, comprising the LexA DNA binding <br /> shuffle complementation tests of mutants shown in B. Growth on 5-fluoroo-        domain fused to the pseudorabies virus transcription activation <br /> rotic acid (FOA) medium indicates the respective mutant allele provides Scm3     domain, is present in the nucleus; however, when a functional NES <br /> function. (D) Immunoblot of protein extracts <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> the indicated transfor-        is fused to the activator, cells fail to grow on medium lacking His, <br /> mants detected with anti-HA antibody.                                            because the fusion protein is efficiently exported from the nucleus. <br />                                                                                  The putative Scm3 NES motif behaved like the authentic NESs of <br />                                                                                  HIV Rev and the RNA export factor Gle1 in this assay (Fig. 3B). <br /> importance (Fig. 1A). Residues 13&#226;&#8364;&#8220;24 resemble a Leu-rich nuclear                 When three conserved Leu residues in the NES were mutated to <br /> export signal (NES), whereas short patches of basic residues, similar            Ala (Scm3mut), export failed. When introduced into full-length <br /> to nuclear localization sequences (NLSs), are found at positions                 Scm3, the triple Ala substitution (scm3-3) had little effect on <br /> 54&#226;&#8364;&#8220;59 (BR1) and 148&#226;&#8364;&#8220;153 (BR2) (15, 16). The C-terminal 58 aa are                 protein expression (Fig. 1D), but the mutant protein was unable to <br /> acid-rich (40% D &#226;&#171;&#185; E). Finally, the central part of the protein                  confer Scm3 function in the complementation assay (Fig. 1C). We <br /> resembles a coiled-coil domain in having repeating heptad units                  conclude that the Scm3p NES motif directs nuclear export and that <br /> with hydrophobic residues occupying the fourth position and polar                the ability to be exported from the nucleus is critical for Scm3 <br /> residues in the first position (17). Five such units repeat between              function. <br /> residues 103&#226;&#8364;&#8220;138. <br />    Site-directed mutations affecting each of these motifs were                   Scm3 Interacts with Cse4 and CEN DNA. Subcellular fractionation <br /> constructed (Fig. 1B) and the mutant proteins tested for function                suggested that Scm3, like Cse4, is an integral chromatin protein. <br /> by using a plasmid shuffle assay (Fig. 1C). To monitor protein                   Originally present in the low-speed pellet, Scm3 and Cse4 were both <br /> expression, all of the mutant alleles encoded a C-terminal tri-                  released by limited micrococcal nuclease digestion (fraction 4) and <br /> ple-HA epitope tag. SCM3-HA fully complemented scm3::LEU2                        pelleted in a 100,000 &#226;&#171;&#187; g centrifugation (fraction 7; Fig. 4A). As a <br /> and produced an Scm3-HA protein that migrated with an apparent                   control, Cse1, a nuclear export protein known to be associated with <br /> molecular mass of 36 kDa, larger than the predicted 30 kDa (Fig.                 the nuclear envelope, was monitored and found exclusively in the <br /> 1D). Removal of the C-terminal 48 aa (scm3-26), comprising most                  soluble fraction, i.e., fraction 2 (data not shown). The copurification <br /> of the D/E-rich region, did not result in loss of complementing                  of Scm3 and Cse4 prompted us to test for Scm3-Cse4 interaction. <br /> ability even though the truncated protein, undetectable by Western               Yeast two-hybrid assays showed that Scm3 interacts with itself and <br /> blot assay, was poorly expressed or unstable (Fig. 1 C and D).                   with Cse4 (Table 1). Both interactions were abolished when baits <br /> Deletion of 90 C-terminal residues (scm3-27), which removed all of               carried the scm3-31 double <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>missense mutation</span>. Immunoblot anal- <br /> the D/E-rich region and impinged on the heptad repeat region,                    ysis confirmed that the mutant DB-scm3-31 baits were stably <br /> yielded an inactive protein that, like Scm3-26, was undetectable.                expressed in the reporter cells (data not shown). Scm3-Cse4 <br /> Deletion of potential NLSs BR1 and BR2 in mutant scm3-32 did                     interaction was mediated through the HFD of Cse4 and not the N <br /> not result in loss of complementing ability, but deletion of the                 terminus. The lethal cse4-286 H3 swap mutant, in which 9 Cse4 <br /> putative NES (scm3-13) or the heptad repeat region (scm3-30) were                amino acids located in the C-terminal third of the HFD are <br /> lethal. The latter two mutant proteins were expressed at levels                  exchanged with their histone H3 counterparts (22), failed to interact <br />  <br /> 10572 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703178104                                                                                          Stoler et al. <br />                                                                                                                                                                       GENETICS <br /> Fig. 2. Scm3 heptad repeat domain is evolutionarily conserved. (A) Alignment of 38 fungal protein sequences homologous to the heptad repeat domain of <br /> S. cerevisiae Scm3, shown in Logo format (42). The alignment requires insertion of one to three gaps at positions 20 &#226;&#8364;&#8220;22. The complete alignment in Clustal format <br /> is provided in SI Fig. 7. (B) Maximum likelihood phylogeny of the Scm3 homology domain of 29 representative ascomycetes derived by Bayesian inference (details <br /> in SI Materials and Methods). Numbers on the branches are the percentage of times that branch was present in the posterior distribution of trees. All members <br /> of the clade defined by the branch marked with the asterisk (*) have point centromeres. <br />  <br />  <br />  <br /> with Scm3, suggesting that the interaction occurs through the helix                matin fractionated by using anti-HA conjugated Sepharose beads. <br /> 2-helix 3 region of the HFD.                                                       DNA in the Scm3HA and control fractions was analyzed by <br />    The two-hybrid results were confirmed by affinity purification. A               multiplex PCR by using primer sets designed to amplify CEN and <br /> strain was constructed that expressed both HA-tagged Scm3 and a                    non-CEN sequences on chromosome III. Three DNA bands are <br /> functional Cse4-Protein A fusion (Cse4SpA) under control of the                    resolved by agarose gel electrophoresis, the middle band (275-bp) <br /> GAL1-10 promoter (14). Cse4SpA and associated proteins were                        diagnostic for CEN3 (Fig. 4C). Scm3-HA as well as Cse4-HA <br /> purified from cell extracts by using IgG-conjugated agarose beads                  immunoprecipitates were enriched for CEN3 sequences, indicative <br /> and the protein components analyzed by immunoblot analysis.                        of CEN DNA specificity. CEN3 DNA was not enriched in control <br /> Scm3-HA and Cse4SpA copurified (Fig. 4B). Copurification was                       immunoprecipitates prepared from cells expressing Scm3 with a <br /> abolished by deletion of the Scm3 heptad repeat domain (scm3-                      different epitope tag (Scm3-Myc). Thus, Scm3p, like Cse4p, selec- <br /> 30HA) but not by mutations affecting the NES (scm3-3HA, scm3-                      tively associates with CEN DNA. <br /> 13HA), although relatively less of the NES mutant proteins copu- <br /> rified. Whereas this difference could mean that the NES                            Scm3 Depletion Results in G2/M Arrest and Aberrant Cse4 Localization. <br /> contributes to but is not essential for the interaction, it is more likely         A conditional scm3 allele was generated by using the S. cerevisiae <br /> that the yield of NES mutant complexes is reduced by the presence                  degron system of Moqtaderi et al. (23). In this genetic background, <br /> of WT Scm3 in the strains. Because the NES mutations are lethal,                   addition of CuSO4 to the culture medium results in rapid degra- <br /> the mutant alleles must be tested in a WT SCM3 background.                         dation of preexisting Scm3 and repression of scm3 transcription. <br />    Chromatin immunoprecipitation (ChIP) was used to determine                      Strains carrying the scm3deg degron allele rapidly arrest their cell <br /> whether Scm3 is associated with centromeres. Yeast cells expressing                cycle upon CuSO4 addition, accumulating as distinctive large- <br /> Scm3-HA were treated with formaldehyde and the sheared chro-                       budded cells indicative of G2/M arrest. The terminally arrested cells, <br />  <br /> Stoler et al.                                                                                             PNAS &#229;&#8230;&#169; June 19, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 25 &#229;&#8230;&#169; 10573 <br />  Fig. 3. Scm3p contains a functional NES. (A) The NES consensus sequence (15) <br /> and other known NESs are shown in comparison with the Scm3 NES and the <br /> mutated sequence of scm3-3. At the bottom is shown an alignment, in Logo <br /> format (42), of the NES region found in Scm3 orthologs of 11 close relatives of <br /> S. cerevisiae. (B) Results of the NES assay. Positive controls are the NESs of Gle1 <br /> and Rev. Negative controls are empty vector as well as a fusion containing a <br /> nonfunctional Rev sequence (Rev NESmut). Scm3mut is the fusion containing <br /> the mutated NES region of scm3-3. <br />  <br />  <br />  <br /> operationally dubbed &#226;&#8364;&#732;&#226;&#8364;&#732;dumbbells&#226;&#8364;&#8482;&#226;&#8364;&#8482;, are distinguishable from large- <br /> budded G2/M cells in untreated cultures by their larger size and <br /> bi-lobed appearance. The arrest depends on MAD2, suggesting that                      Fig. 4. Scm3 copurifies with Cse4 and cross-links to CEN3. (A) Subcellular <br /> the defect in Scm3-depleted cells is monitored by the spindle                         fractionation and immunoblot analysis of HA-tagged Scm3 and Cse4. (B) <br />                                                                                       Immunoblot of proteins copurifying with Cse4 from extracts of cells coex- <br /> assembly checkpoint (SI Fig. 8).                                                      pressing Protein A-tagged Cse4 (Cse4SpA) and HA-tagged Scm3 or Scm3 <br />    To assess the effect of Scm3 depletion on Cse4 localization, a                     mutant proteins. Purification was performed with IgG-agarose beads and <br /> GFP-tagged Cse4 allele was introduced into the scm3deg back-                          eluted proteins were analyzed by immunoblotting with anti-HA antibody. <br /> ground. Cells were synchronized with &#226;&#65533;&#163;-factor and released from                       Cse4SpA protein was detected by its binding to the enzyme-conjugated <br /> the G1 arrest after depleting Scm3. Ninety minutes after release,                     secondary antibody. T, total cell extract; P, affinity eluate. (C) Formaldehyde <br /> 90% of Scm3-depleted cells had failed to complete mitosis, with                       cross-linked, solubilized chromatin was immunoprecipitated with anti-HA <br />                                                                                       monoclonal antibody. Total (T) and coprecipitated (P) DNA was used for <br /> about half showing the terminal dumbbell phenotype; in contrast,                      multiplex PCR with the primer sets shown above the gel image. <br /> control cells had doubled, few dumbbell figures were observed, and <br /> the bud morphology profile resembled that of an asynchronous <br /> population (Fig. 5C). Cse4 localization in the control cells exhibited                depleted from cells in asynchronous culture or cells previously <br /> the distinctive &#226;&#8364;&#732;&#226;&#8364;&#732;dot&#226;&#8364;&#8482;&#226;&#8364;&#8482; <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> typical of kinetochore proteins (Fig. 5A              arrested in G2 (SI Table 2). <br /> Lower), i.e., the 16 centromeres cluster as a single dot in G1 cells and <br /> as two dots after CEN DNA replication and spindle pole body                           Discussion <br /> separation in early S phase (24). Scoring four microscopic fields, 19                 Taken together, our results suggest that S. cerevisiae Scm3 plays <br /> of 20 control cells were found to have one or two dots associated                     some role in the targeting of Cse4 to CEN chromatin or its <br /> with the DAPI-stained mass. In Scm3-depleted cultures, about half <br /> of the cells (12 of 25 cells scored in four fields) lacked distinct Cse4              Table 1. Results of yeast two-hybrid tests <br /> dots (P &#226;&#172;&#65533; 10&#226;&#171;&#186;4 by &#226;&#65533;&#185;2), and where Cse4 dots were observed, they <br />                                                                                                 BD fusion             AD fusion          Growth on -His <br /> seemed to be smaller and less intense (Fig. 5A Upper). This <br /> judgment was confirmed by quantitating the fluorescence intensity                               Scm3                   Empty                   No <br /> of the kinetochore dots. As seen in Fig. 5B, the median intensity of                            Empty                   Scm3                   No <br /> kinetochore dots in Scm3-depleted cells was significantly less than                             Scm3                    Scm3                   Yes <br /> that of the controls, with the majority of distinguishable dots (solid                          Scm3&#226;&#8364;&#8220;31                 Scm3                   No <br /> circles) having intensities less than twice background. By compar-                              Scm3                    Cse4                   Yes <br /> ison, only two of 19 control kinetochore dots failed to exceed the                              Scm3&#226;&#8364;&#8220;31                 Cse4                   No <br /> twice-background threshold (P &#226;&#172;&#65533; 10&#226;&#171;&#186;4 by &#226;&#65533;&#185;2). Background fluo-                                   Scm3                  Cse4&#226;&#8364;&#8220;286                 No <br />                                                                                                 Scm3                  Cse4 HFD                 Yes <br /> rescence did not differ between control and Scm3-depleted cells. <br />                                                                                                 Empty                 Cse4 HFD                 No <br /> The cell cycle arrest and Cse4 mislocalization phenotypes observed <br />                                                                                                 Cse4 N term             Scm3                   No <br /> upon Scm3 depletion are not restricted to cells previously arrested <br /> in G1. Qualitatively similar results were obtained when Scm3 was                        BD, binding domain; AD, activation domain. <br />  <br />  <br /> 10574 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703178104                                                                                                     Stoler et al. <br />                                                                                      of Cse4 from only a few centromeres would suffice to effect the <br />                                                                                     near complete cell cycle arrest we observe. <br />                                                                                        Assembly of Cse4 into CEN DNA nucleosomes requires several <br />                                                                                     steps that might require a specialized assembly factor. By analogy <br />                                                                                     to canonical nucleosome assembly, the first step would be associ- <br />                                                                                     ation of Cse4 with H4 to form (Cse4-H4)2 tetramers. To accomplish <br />                                                                                     this step, Cse4 must compete with an excess of H3 to bind H4, then <br />                                                                                     Cse4-H4 dimers must selectively self-associate to avoid formation <br />                                                                                     of mixed tetramers (i.e., Cse4-H4/H3-H4), which are not observed <br />                                                                                     (27, 28). Next, (Cse4-H4)2 tetramers must target one of sixteen <br />                                                                                     125-bp CENs, which together comprise only 0.01% of the genomic <br />                                                                                     DNA. Scm3 could confer Cse4-Cse4 specificity by binding to and <br />                                                                                     stabilizing (Cse4-H4)2 tetramers, or it could provide Cse4-CEN <br />                                                                                     DNA specificity by binding Cse4 (either as a monomer or Cse4&#226;&#8364;&#8220;H4 <br />                                                                                     complex) and delivering it to the site of CEN chromatin assembly, <br />                                                                                     perhaps through protein-protein interaction with another centro- <br />                                                                                     mere protein. An attractive partner would be CBF3, the inner <br />                                                                                     kinetochore protein that binds CDEIII and on which the incorpo- <br />                                                                                     ration of all other kinetochore components depends (9). The <br />                                                                                     observed protein-protein interaction between Scm3 and Cse4 is <br />                                                                                     consistent with either scenario. <br />                                                                                        Could Scm3 be a Cse4-specific histone chaperone? Histone <br />                                                                                     chaperones are acidic proteins that associate with core histones and <br />                                                                                     facilitate nucleosome assembly&#226;&#8364;&#8221;NAP-1, CAF-1, and HIRA are <br />  <br />  <br />  <br />  <br />                                                                                                                                                              GENETICS <br />                                                                                     well-known examples (29). In Drosophila, Cid (the Drosophila Cse4 <br />                                                                                     ortholog) is found in a soluble complex with histone H4 and the <br />                                                                                     chaperone RbAp48, and it is proposed that the Cid/H4/RbAp48 <br />                                                                                     complex is responsible for deposition of the CenH3 in vivo (30). But <br />                                                                                     RbAp48 is not centromere-specific. It is also a component of <br />                                                                                     H3.1/CAF-1 and H3.3/HIRA complexes responsible for replica- <br />                                                                                     tion-dependent and replication-independent nucleosome assembly <br />                                                                                     pathways, respectively (31). Likewise the fission yeast RbAp48 <br />                                                                                     homolog, Mis16, is required for deposition of Cnp1 (the S. pombe <br />                                                                                     CenH3), but Mis16 localization is not restricted to centromere <br /> Fig. 5. Scm3 depletion leads to G2/M arrest and aberrant Cse4 localization. (A)     DNA (32). It is proposed that Mis16 centromere specificity is <br /> Strain R421 (scm3deg CSE4GFP) was arrested with &#226;&#65533;&#163;-factor for 2 h after which half   conferred by Mis18 (32). The S. cerevisiae RbAp48 homolog is <br /> of the culture was transferred to a separate flask and CuSO4 added to a final       Msi1/Cac3&#226;&#8364;&#8221;a subunit of yeast CAF-1 (33)&#226;&#8364;&#8221;not Scm3; however, <br /> concentration of 0.5 mM to induce the degron and deplete Scm3. After an             Scm3 could serve a Mis18-like function, lending CEN specificity to <br /> additional 60 min, cells from the control and Scm3-depleted cultures were rapidly <br />                                                                                     a Cse4-chaperone complex. In this model, interaction between <br /> filtered, washed, and resuspended in medium lacking &#226;&#65533;&#163;-factor with (Scm3- <br /> depleted) or without (control) CuSO4. Ninety minutes after release from the <br />                                                                                     Scm3 and Cse4 is indirect; both are components of the same Cse4 <br /> &#226;&#65533;&#163;-factor arrest, cells were processed for fluorescence microscopy. Images of the    chaperone complex. Alternatively, Scm3 could be the Cse4-binding <br /> Scm3-depleted cells (Upper) are from a contiguous microscopic field moved           subunit in a CEN-specific, CAF-1-like assembly complex. The <br /> digitally to minimize empty space. (Scale bars: 2 &#226;&#65533;&#174;m.) Circles, mask used for       presence in Scm3 of the essential D/E-rich domain is consistent with <br /> quantitating fluorescence. (B) Fluorescence intensity of Cse4-GFP dots. Filled      a histone chaperone function. <br /> circles, measurements of distinguishable kinetochore dots; open circles, measure-      The Leu-rich NES homology domain in Scm3 directs nuclear <br /> ments from cells lacking a distinguishable kinetochore dot. The latter were         export in vivo. The NES is conserved in the Scm3 proteins of other <br /> excluded from the statistical analysis. Lines show median (&#226;&#8364;&#732;&#226;&#8364;&#732;Dots&#226;&#8364;&#8482;&#226;&#8364;&#8482;) or mean        point centromere yeasts but is not found in the orthologs of more <br /> (Background) values; significance of differences was assessed by Mann&#226;&#8364;&#8220;Whitney <br />                                                                                     diverged hemiascomycetes or other fungi. Interestingly, NAP-1 <br /> or unpaired t tests, respectively. (C) Bud morphologies of samples from A. UB, <br /> unbudded; SB, small-budded; LB, large-budded; DB, dumbbell. Black shading in <br />                                                                                     contains a Leu-rich NES and is known to shuttle between nucleus <br /> the LB &#226;&#171;&#185; DB bars indicates the proportion of DB.                                    and cytoplasm (34). Mutations that abolish NAP-1 shuttling disrupt <br />                                                                                     transcription, presumably because of defects in histone deposition <br />                                                                                     (34). Possibly, Scm3 acts as a carrier to shuttle Cse4 into and out of <br /> maintenance there. SCM3 is a dosage suppressor of Ts cse4                           the nucleus. The Scm3 NLS was not identified. Whereas Scm3 <br /> mutations, and affinity purification and two-hybrid assays dem-                     contains two short stretches of basic amino acids that are potential <br /> onstrate that Scm3 and Cse4 interact in vivo. The Cse4-Scm3                         NLSs, they are essential neither alone nor in combination; thus, <br /> interaction occurs via the Cse4 HFD, and SCM3 dosage sup-                           some other region in the Scm3 protein can supply NLS function. <br />                                                                                     Scm3 must gain access to the nucleus, because fractionation studies <br /> pression is specific for cse4 HFD alleles; it is the Cse4 HFD <br />                                                                                     indicate that Scm3 is predominantly chromatin-bound, and ChIP <br /> where CEN targeting information resides (25). ChIP assays show <br />                                                                                     shows it to be associated with CEN DNA. <br /> that Scm3 and Cse4 are colocalized at CEN DNA. Finally,                                Scm3 contains a conserved protein domain found in proteins of <br /> Scm3-depleted cells are unable to complete mitosis, and Cse4                        44 other fungi including budding and fission yeasts, filamentous <br /> localization is aberrant. Many cells in the arrested population                     fungi, and basidiomycetes. In the Pfam database (35), the Scm3 <br /> lack the distinctive Cse4 dots indicative of normal kinetochore                     homology domain defines protein family B&#227;&#8250;&#173;19394, although only <br /> localization, and where dots are observed at the DNA margin,                        nine members are annotated. In the S. cerevisiae protein, the <br /> their Cse4 content is significantly less than that of WT cells. We                  domain consists of heptad repeats of hydrophobic amino acids <br /> interpret this finding to mean that Cse4 is missing from many but                   predicted by COILS (36) to have some coiled-coil character. Coiled <br /> not all of the 32 G2 chromatids. Because a single nonfunctional                     coils often provide the interface for specific protein-protein inter- <br /> kinetochore is able to activate the mitotic checkpoint (26), loss                   action (17), and mutation of hydrophobic residues in the heptad <br />  <br /> Stoler et al.                                                                                           PNAS &#229;&#8230;&#169; June 19, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 25 &#229;&#8230;&#169; 10575 <br />  repeat (scm3-31) result in loss of Scm3-Scm3 and Scm3-Cse4                                    and cloned between the EcoRI and BamHI sites of the reporter <br /> interactions detected by two-hybrid analysis. The scm3-31 mutation                            plasmid (pSW715). An analogous construct was made for the <br /> is lethal, suggesting that one or both interactions are essential for                         scm3-3 triple Ala substitution mutant. These plasmids, along with <br /> Scm3 function. In other homologs, amino acid sequence is con-                                 pSW715 (no insert), pSW716 (Gle1 NES insert), pSW722 (Rev <br /> served but spacing is not, leading us to doubt the original propo-                            NES), and pSW723 (Rev nonfunctional NES) were transformed <br /> sition of a coiled-coiled structure. The phylogeny of the Scm3                                into test strain L40. Transformants were streaked on Trp dropout <br /> homology domain is consistent with species phylogeny, suggesting                              and Trp/His double dropout media and scored for growth after 3 <br /> orthology; however, outside this 50 aa domain, little homology is                             days. <br /> observed except between orthologs of close relatives. The proteins                               Yeast transformations were carried out by the lithium acetate <br /> range in size from &#226;&#172;&#8225;200 aa in the Saccharomyces yeasts to &#226;&#172;&#381;1,000                              procedure (38). Media were as described (39). For &#226;&#65533;&#163;-factor arrest, <br /> aa in N. crassa and C. cinerea.                                                               cells were incubated at a cell density of 1 &#226;&#171;&#187; 106 cells per ml in <br />    Orthologs of several S. cerevisiae kinetochore proteins are found                          medium containing 3 &#226;&#171;&#187; 10&#226;&#171;&#186;6 M &#226;&#65533;&#163;-factor. Coprecipitation of protein <br /> in insects, plants, and metazoans including humans, whereas others                            A-tagged Cse4, immunoblotting, and ChIP were carried out as <br /> are restricted to yeasts having point centromeres (1). Conversely,                            described (14). Subcellular fractionation was by the method of <br /> some kinetochore proteins are conserved between fungi and meta-                               Liang and Stillman (40). Bud morphology was scored as follows: <br /> zoans but are limited to organisms having regional centromeres and                            unbudded, no bud; small-budded, bud &#226;&#172;&#65533; 50% size of mother; <br /> are not found in S. cerevisiae and other point centromere yeasts.                             large-budded, bud &#226;&#172;&#381; 50% size of mother; dumbbell, mother and <br /> Finally, other proteins are found associated with both point and                              bud approximately equal in size, both rounded rather than oval in <br /> regional CENs but only in fungi. SCM3 joins the fungi-restricted                              shape. <br /> group. Our biochemical and genetic characterization of Scm3                                      Fluorescence microscopy was performed on a Nikon Eclipse <br /> function in S. cerevisiae suggests that Scm3 plays a role in the                              E800 microscope with &#226;&#171;&#187;100 oil immersion objective (1.40 N.A.). <br /> targeting of Cse4 to CEN chromatin or its maintenance there.                                  Samples were prepared as described (41). Images were captured on <br /> Future work will be needed to determine whether the fungal                                    a Santa Barbara Instrument Group ST-8 CCD camera by using <br /> orthologs are kinetochore proteins in their respective organisms                              identical exposure times for control and Scm3-depleted samples. <br /> and whether they execute an analogous function in localizing                                  Fluorescence intensity of Cse4-GFP kinetochore dots was quanti- <br /> CenH3.                                                                                        tated by using NIH Image 1.63 (http://rsb.info.nih.gov/nih-image) <br />                                                                                               after converting the raw image files to 8-bit TIFFs, keeping bright- <br /> Materials and Methods                                                                         ness and contrast settings equal for control and experimental <br /> Strain CDL151 (MATa ade2-101 his3-11,15 leu2-3 lys2-801 trp1-                                 images, and then inverting the images. Average pixel intensities in <br /> 901 ura3-52 scm3::LEU2) carrying the episomal SCM3-URA3                                       a 5.9-pixel (0.5 &#226;&#65533;&#174;m) diameter circular mask were measured and <br /> plasmid pMU3 was used for scm3 complementation tests.                                         expressed [as optical density (O.D.)]. Budded cells were counted as <br /> CDL151 was transformed with TRP1 plasmids carrying scm3                                       a single cell with the measurement taken in the bud containing the <br /> alleles to be tested. Transformants were then plated onto                                     DNA. All cells were scored. In cases where a kinetochore dot was <br /> 5-flouroorotic acid (FOA) medium to select for loss of pMU3.                                  not observed, a reading was taken at the nuclear periphery where <br /> Noncomplementing mutant alleles produce no FOAr progeny,                                      the fluorescence appeared most intense. Where two dots were <br /> because loss of pMU3 is lethal. Strain PJ69-4A was used for                                   observed (G2/M cells), both were measured, the values were <br /> two-hybrid assays as described (37). The pGAD and pGBD                                        summed, and 1&#226;&#171;&#187; background was subtracted. Background readings <br /> plasmids were cotransformed, and interaction was assessed on                                  were taken over cytoplasmic regions where the fluorescence ap- <br /> His dropout medium containing 1 mM 3-aminotriazole. The                                       peared most dim. Statistical analysis was performed by using Prism <br /> Scm3 degron allele in strain R421 was obtained as described by                                4 software (GraphPad, San Diego, CA). <br /> Moqtaderi et al. (23); the CSE4GFP allele was integrated at the <br />                                                                                       <br /> CSE4 locus. Details of plasmid construction and yeast genetic                             <br /> procedures can be found in SI Materials and Methods.                                          <br />    The genetic assay for NES function was described by Shulga et al.                    <br /> (21). A DNA fragment encoding the Scm3 NES and flanking amino                                 <br /> acids (nucleotides 4&#226;&#8364;&#8220;156 of the SCM3 ORF) was obtained by PCR                           <br /> </body></html>