<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Mar. 2005, p. 604&#226;&#8364;&#8220;614                                                                                            Vol. 4, No. 3 <br /> 1535-9778/05/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.4.3.604&#226;&#8364;&#8220;614.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />                           Genomic Analysis PIS1 Gene Expression <br />                                 Mary E. Gardocki, Margaret Bakewell, Deepa Kamath, <br />                                 Kelly Robinson, Kathy Borovicka, John M. Lopes* <br />                             Department Biological Sciences, Wayne State University, Detroit, Michigan <br />                                          Received 18 September 2004/Accepted 20 December 2004 <br />  <br />              The Saccharomyces cerevisiae PIS1 gene essential required final step novo synthesis <br />           phosphatidylinositol. Transcription PIS1 gene uncoupled factors regulate yeast <br />           phospholipid biosynthetic genes. Most phospholipid biosynthetic genes regulated response <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />           inositol choline regulatory circuit includes Ino2p:Ino4p activator complex Opi1p <br />           repressor. PIS1 regulated response carbon source anaerobic growth conditions. Both these <br />           regulatory responses modest, entirely surprising PIS1 essential. However, modest <br />           regulation PIS1 expression shown affect phosphatidylinositol metabolism affect cell cycle <br />           progression. This prompted present study, employed genomic screen, database mining, more <br />           traditional promoter analysis identify genes affect PIS1 expression. A screen viable yeast deletion <br />           set identified 120 genes affect expression PIS1-lacZ reporter. The gene set included peroxi- <br />           somal genes, silencing genes, transcription factors. Factors suggested database mining, Pho2 <br />           Yfl044c, affect PIS1-lacZ expression. A PIS1 promoter deletion study identified an <br />           upstream regulatory sequence element required carbon source regulation located downstream of <br />           previously defined upstream activation sequence elements. Collectively, studies demonstrate a <br />           collection genomic traditional strategies implemented identify set genes affect the <br />           regulation essential gene. <br />  <br />  <br />    Yeast excellent model study phos-                 inostamycin reduces PI levels inhibits induction S phase <br /> pholipid biosynthesis (Fig. 1) (11, 12, 28, 33). Phosphatidylino-           (18, 36). <br /> sitol (PI) essential phospholipid eukaryotic cells (3,            PI synthase membrane-associated enzyme catalyzes <br /> 11, 12, 28, 33, 38, 54). In yeast, PI synthesized novo         condensation CDP-diacylglycerol inositol PI (23, <br /> product PIS1 gene, PI synthase (18, 23, 24, 36, 53&#226;&#8364;&#8220;56),              54) (Fig. 1). Disruption PIS1 gene results lethality <br /> and represents 12 27% total phospholipid composi-                 (54). Because PIS1 essential, entirely surprising that <br /> tion (11, 12, 28, 33). In addition structural role, PI            yeast cells extensively regulate PIS1 expression PI <br /> precursor phosphoinositides, sphingolipids, inositol                 synthase levels (2, 24, 25). PIS1 gene expression coregu- <br /> polyphosphates (11, 12, 28, 33). PI metabolites               lated phospholipid biosynthetic genes re- <br /> required diverse set processes include glycolipid             sponse inositol choline (2, 11, 12, 28, 33) instead <br /> anchoring proteins (69), signal transduction (21, 58), mRNA              regulated carbon source oxygen. PIS1 expression is <br /> export (57, 64&#226;&#8364;&#8220;66), vesicle trafficking (17). In spite           repressed response glycerol aerobic conditions (2, <br /> importance PI metabolites, relatively little known            25). Promoter deletion analysis identified upstream ac- <br /> about factors regulate PIS1 expression.                                tivation sequence (UAS) elements (UAS1 UAS3) required <br />    Our understanding role PIS1 expression regu-                PIS1 gene expression (25). However, element required <br /> lating PI synthesis conflicted. One study reported over-            glycerol repression identified. The region that <br /> expression human PIS1 gene COS-7 cells yielded                  includes UAS3 element contains upstream regula- <br /> significant increase PI synthase activity (25-fold) mod-           tory sequence (URS) binds Rox1p exert anaerobic <br /> est increase PI levels (8.2%) (50). However, report              regulation (25). The significance anaerobic regulation is <br /> indicated overexpression rat PIS1 gene NIH3T3                evidenced altered membrane composition. PI levels are <br /> cells yielded elevated levels PI, PI-4,5-P2, PI-3,4,5-P3             elevated cells grown anaerobically, phosphatidylcholine <br /> (19). PIS1 overexpression decreased doubling time               (PC) CDP-diacylglycerol levels affected oxygen <br /> transformed cells accelerated G1 progression (19). Con-                 (25). <br /> sistent effect G1 progression, cyclin D1 cyclin                PIS1 gene expression insensitive inositol choline; <br /> E levels elevated (19). Furthermore, Rous sarcoma virus-               however, inositol does affect PI synthase activity. High levels of <br /> infected NIH3T3 cells activated erbB2-transformed                       inositol increase rate PI synthesis Km PI <br /> NIH3T3 cells overexpress PIS1 elevated PI levels              synthase inositol (0.21 mM) ninefold greater the <br /> (37). Finally, specific inhibition PI synthase activity using            intracellular concentration inositol (24 &#226;?&#174;M) (41). When <br />                                                                             cells grown inositol, PI levels double expense of <br />                                                                             PC synthesis inositol noncompetitive inhibitor of <br />   * Corresponding author. Mailing address: Department Biological <br /> Sciences, Wayne State University, 5047 Gullen Mall, Detroit, MI <br />                                                                             phosphatidylserine synthase (CHO1 gene product), first <br /> 48202. Phone: (313) 993-7816. Fax: (313) 577-6891. E-mail: jlopes           enzyme PC branch phospholipid biosynthesis (Fig. 1) <br /> @sun.science.wayne.edu.                                                     (41). <br />  <br />                                                                       604 <br />  VOL. 4, 2005                                                                                                                     PIS1 GENOMICS             605 <br />  <br />                                                                                             TABLE 1. Oligonucleotides used study <br />                                                                                 Oligonucleotide <br />                                                                                                                                   Sequence <br />                                                                                    (position) <br />  <br />                                                                                 PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) ........................5&#226;&#172;&#732;-GGATCCCTTGTACTATCACAC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;918) .........................5&#226;&#172;&#732;-AGATCTGAATTCCCTGCACGC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;621) .........................5&#226;&#172;&#732;-AGATCTAAGCTTTTAGCCATG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;521) .........................5&#226;&#172;&#732;-AGATCTTAAATCCAAATCTTC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;325) .........................5&#226;&#172;&#732;-AGATCTTGTGCTTGAGGCTCA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;224) .........................5&#226;&#172;&#732;-AGATCTTTTTTTCATTTTATA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;205) .........................5&#226;&#172;&#732;-AGATCTTTCACGTCGCTCCGC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;184) .........................5&#226;&#172;&#732;-AGATCTATCCCTATTACGGAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;149F) ......................5&#226;&#172;&#732;-AGATCCGCCCCTCCTATTGTT-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;138) .........................5&#226;&#172;&#732;-AGATCTGTTTTTTCCGTCTCG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;127) .........................5&#226;&#172;&#732;-AGATCTCTCGAGGATTTTTCA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;185) .........................5&#226;&#172;&#732;-GGATCCTTACGTGCGGAGCGA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;149B) ......................5&#226;&#172;&#732;-GGATCCTTTCCCTATTGAGAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;126B) ......................5&#226;&#172;&#732;-GGATCCACGGAAAAAACAATA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;101B) ......................5&#226;&#172;&#732;-GGATCCATACTAAAAGTGAAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;76B) ........................5&#226;&#172;&#732;-GGATCCCTCATGTTTTTTATG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;51B) ........................5&#226;&#172;&#732;-GGATCCTGACGGCAAATAAAC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;101F) ......................5&#226;&#172;&#732;-AGATCTATAAGTAAAACATAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;76F) ........................5&#226;&#172;&#732;-AGATCTAGGTGGTATGGTTTA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;51F) ........................5&#226;&#172;&#732;-AGATCTCTGTGGAGCCTTCAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;26F) ........................5&#226;&#172;&#732;-AGATCTTAAGAGGGAAAGTGT-3&#226;&#172;&#732; <br />                                                                                 PIS1 URS (&#226;&#171;&#186;76) .................5&#226;&#172;&#732;-CTCGAGAGGTGGTATGGTTTA-3&#226;&#172;&#732; <br />                                                                                 PIS1 URS (&#226;&#171;&#186;51) .................5&#226;&#172;&#732;-CTCGAGTGACGGCAAATAAAC-3&#226;&#172;&#732; <br />                                                                                 TCM1 (&#226;&#171;&#186;572)......................5&#226;&#172;&#732;-AAGCTTCTTACGTTATCATTC-3&#226;&#172;&#732; <br />                                                                                 TCM1 (&#226;&#171;&#186;1)..........................5&#226;&#172;&#732;-GAATTCGATTGATTGTTGTAG-3&#226;&#172;&#732; <br />   FIG. 1. Schematic depiction S. cerevisiae phospholipid bio-            pBM2015-SEQ.....................5&#226;&#172;&#732;-TCTCGTGCGGAGATGACTGC-3&#226;&#172;&#732; <br /> synthetic pathway. The CDP-choline pathway, known the <br /> &#226;&#8364;&#339;Kennedy&#226;&#8364;? &#226;&#8364;&#339;salvage pathway,&#226;&#8364;? noted broken arrow. Genes <br /> are designated boldface italic type. Abbreviations: DHA, dihy- <br /> droxyacetone; DHAP, dihydroxyacetone phosphate; PA, phosphatidic                cated, 75 &#226;?&#174;M inositol 1 mM choline added (I&#226;&#171;&#185;C&#226;&#171;&#185; medium) and/or <br /> acid; DGPP, diacylglycerol pyrophosphate; DAG, diacylglcerol.                   NaCl added (0.7 M final concentration). <br />                                                                                    Genomic studies. Plasmid pMA107 (2), containing 629 bp PIS1 pro- <br />                                                                                 moter fused lacZ gene, transformed BY4742-based VYDS <br />                                                                                 (Res Gen Invitrogen Corp., Carlsbad, Calif.) standard lithium acetate yeast <br />                                                                                 transformation procedure (14) 96-well microtiter plates. Transformants were <br />    PIS1 transcript levels established unusual combi- <br />                                                                                 selected complete synthetic medium lacking uracil (Ura&#226;&#171;&#186;), replicated onto <br /> nation low transcription initiation rates stable                    Ura&#226;&#171;&#186; 2% glucose X-Gal (5-bromo-4-chloro-3-indolyl-&#226;?&#164;-D-galactopyranoside) <br /> mRNA. The PIS1 gene expressed weak promoter                      Ura&#226;&#171;&#186; 3% glycerol X-Gal plates allowed grow 30&#194;&#176;C 48 h. VYDS <br /> is comparable activity GAL4, INO2, INO4                   transformants screened altered PIS1-lacZ expression, indicated a <br /> transcription factor genes (1). However, weakness                    light-blue <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> 2% glucose X-Gal plates dark-blue phenotype on <br />                                                                                 3% glycerol X-Gal plates relative isogenic wild-type (BY4742) strain. To <br /> PIS1 promoter compensated stable transcript (half-life,                 eliminate false positives, pMA107 transformants exhibited phenotype were <br /> 58 min) (1). In fact, PIS1 transcript                     transformed plasmids pJH330 (543 bp INO1 gene 132 codons of <br /> stable yeast mRNAs (34). The combination low transcrip-                      INO1 gene fused frame lacZ reporter gene YEp357R) (22) and <br /> tion initiation rates high transcript stability ensure                  YEp357R-TCM1 (TCM1-lacZ) (described below). Mutant alleles VYDS <br />                                                                                 confirmed sequencing bar codes marked mutant. <br /> that PIS1 transcripts change slowly response environmen- <br />                                                                                    Plasmid construction. The construction PIS1-cat reporters has <br /> tal changes.                                                                    previously reported (25). Briefly, nested set PIS1 promoter deletions <br />    In study, utilized genomic strategy identify genes              created PCR using appropriate oligonucleotides (Table 1). The 5&#226;&#172;&#732;-ter- <br /> that affect regulation PIS1 expression. Analysis viable               minal deletion mutants created using reverse primer PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) along <br /> yeast deletion set (VYDS) identified genes affect                  forward primer set PIS1 (&#226;&#171;&#186;918) PIS1 (&#226;&#171;&#186;127) (Table 1). The indi- <br />                                                                                 vidual PCR products cloned pGEM-T (Promega, Madison, Wis.). PIS1 <br /> PIS1 expression regulation. We examined role                    promoter fragments excised pGEM-T digestion BamHI and <br /> transcription factors defined &#226;&#8364;&#339;ChIP-on-chip&#226;&#8364;? (chromatin                    BglII inserted pBM2015 create fusion cat reporter gene <br /> immunoprecipitation-on-microarray chip) approach (62)                        (29). To create PIS1 promoter internal deletions, regions position &#226;&#171;&#186;325 <br /> PIS1 expression. Last, identified URS needed PIS1                      various downstream points amplified using forward primer PIS1 (&#226;&#171;&#186;325) <br />                                                                                 reverse primer set PIS1 (&#226;&#171;&#186;185) PIS1 (&#226;&#171;&#186;51B) (Table 1). These <br /> repression response glycerol. <br />                                                                                 fragments sequentially cloned pGEM-T pBM2015 described <br />                                                                                 above. The regions downstream internal deletion amplified using <br />                                                                                 reverse primer PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) forward primer set PIS1 (&#226;&#171;&#186;101F) <br />                        MATERIALS AND METHODS <br />                                                                                 PIS1 (&#226;&#171;&#186;26F) addition PIS1 (&#226;&#171;&#186;149F) PIS1 (&#226;&#171;&#186;127) (Table 1). These <br />   Strains, media, growth conditions. The Saccharomyces cerevisiae strains   fragments sequentially cloned pGEM-T pBM2015 containing the <br /> used study BRS1001 (MATa ade2-1 his3-11,15 leu2-3,112 can1-100     relevant regions deletion endpoints described above. For each <br /> ura3-1 trp1-1), BY4742 (MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0), BY4741           plasmid, indicates deletion endpoints. The pBM2015 derivatives <br /> (MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0) complete VYDS BY4742         sequenced Wayne State University Core Sequencing Facility using <br /> and BY4741 backgrounds. Yeast cultures grown 30&#194;&#176;C complete syn-      primer pBM2015-SEQ. Yeast strains containing promoter-cat plasmids <br /> thetic medium (42) (containing 2% glucose) lacking uracil. Where appropriate,   integrated GAL4 locus created transformation characterized <br /> glucose concentration varied substituted 3% glycerol, 3% acetate,   Southern blot hybridization previously described (4). <br /> 3% ethanol, 3% lactate, 0.1% oleic acid 0.2% Tween 80. Where indi-         Plasmid YEp357R-TCM1 contains 572 bp sequences upstream the <br />  606       GARDOCKI ET AL.                                                                                                                  EUKARYOT. CELL <br />  <br />  <br />                                                       TABLE 2. Genes affect PIS1-lacZ expression <br />                      Reduced expression glucose                                                      Increased expression glycerol <br />  <br />   PIS1                   PIS1 INO1                   PIS1 TCM1                   PIS1                 PIS1 INO1                  PIS1 TCM1 <br />  <br /> APT1                       APG17                           ACE2                       AAD4                     ACB1                           NYV1 <br /> CPD1                       COQ4                            CIK1                       DCP3                     ADH2                           PAC1 <br /> CPT1                       DYN1                            CPS1                       ENT4                     CTF19                          RIF1 <br /> CRH1                       ENT3                            DFG10                      HST3                     CVT19                          YDR445C <br /> CWH41                      HIR3                            EAP1                       HST4                     EGD1                           YDR521W <br /> FAB1                       PDX3                            GSH1                       OGG1                     HSP104                         YGL250W <br /> GCS1                       REG2                            LSB3                       PEX3                     NTG2                           YLL032C <br /> GPD1                       RMD1                            MUD1                       PEX4                     PEX22                          YLL054C <br /> MUM2                       RPB9                            NHX1                       PEX17                    RIM8                           YLL057C <br /> NRG2                       RPS25A                          PEX7                       PMT3                     THI12                          YPL144W <br /> OSM1                       VTA1                            PHO2                       PTC3                     TIR4 <br /> PET130                     YBR053C                         PTC1                       ROT2                     VPS33 <br /> PIM1                       YDL119C                         RPB4                       RPL8B                    VTS1 <br /> POG1                       YDL129W                         SIN3                       SMY1                     YTP1 <br /> RDH54                      YDL173W                         SIT4                       STM1                     YCR076C <br /> SPO75                      YGL165C                         TTR1                       YDR124W                  YDR417C <br /> SRT1                       YJL178C                         YAL068C                    YGR022C                  YOL153C <br /> WH14                       YNR025C                         YDL025C                    YKR043C                  YOR315W <br /> YBR250W                    YOL107W                         YDL057W                    YOR029W                  YPL162C <br /> YDR332W                    YPR123C                         YDL063C                    YOR322C                  YPL166W <br /> YGL050W                                                    YKL147W                                             YPR151C <br /> YGR242W                                                    YLR254C <br /> YIL028W                                                    YLR366W <br /> YMR073C                                                    YNL134C <br /> YPL077C <br />  <br />  <br />  <br /> TCM1 gene fused frame lacZ reporter gene YEp357R (52). This            lacZ expression strains grown different carbon sources. <br /> plasmid constructed amplifying 572 bp TCM1 promoter using S.          Even PIS1 regulated threefold response to <br /> cerevisiae genomic DNA (Res Gen Invitrogen Corp.) using primers TCM1 <br /> (&#226;&#171;&#186;572) TCM1 (&#226;&#171;&#186;1) (Table 1). The 572-bp PCR product cloned into <br />                                                                                     carbon source, regulation clearly discernible X-Gal <br /> pGEM-T (Promega) excised digestion HindIII EcoRI           medium containing glucose (dark blue) glycerol (light <br /> inserted YEp357R. Insert orientation sequence confirmed            blue). The PIS1-lacZ reporter (pMA107) (2) transformed <br /> DNA sequencing.                                                                     BY4742-based VYDS (MAT&#226;?&#163; set), transformants <br />   Plasmid pLG&#226;&#338;&#172;312 &#226;&#171;&#185; PIS1 URSGLY contains 25-bp fragment PIS1 <br /> promoter (&#226;&#171;&#186;76 &#226;&#171;&#186;51) inserted upstream lacZ reporter gene pLG&#226;&#338;&#172;312 <br />                                                                                     screened light-blue colonies X-Gal glucose me- <br /> (32). This plasmid constructed amplifying 25 bp PIS1 gene pro-        dium dark-blue colonies X-Gal glycerol medium. Three <br /> moter (&#226;&#171;&#186;76 &#226;&#171;&#186;51) plasmid pPIS1(&#226;&#171;&#186;325) using primers PIS1 URS (&#226;&#171;&#186;76)            rounds screening identified 178 mutants yielded re- <br /> and PIS1 URS (&#226;&#171;&#186;51) (Table 1). The 37-bp PCR product cloned                 duced expression glucose 59 mutants increased <br /> pGEM-T (Promega) excised digestion XhoI inserted into <br />                                                                                     expression glycerol. To eliminate mutants nonspecifi- <br /> plasmid pLG&#226;&#338;&#172;312. Insert orientation sequence confirmed DNA <br /> sequencing.                                                                         cally affected lacZ expression, rescreened mutants using <br />   Enzyme assays. &#226;?&#164;-Galactosidase chloramphenicol acetyltransferase              INO1-lacZ (phospholipid biosynthetic gene) (22) TCM1- <br /> (CAT) assays performed described previously (4, 48). However,       lacZ (ribosomal protein gene) reporters. Strains exhibiting al- <br /> the &#226;?&#164;-galactosidase experiments performed using microtiter plate plat-       tered expression reporters (109 strains glucose <br /> form. Units &#226;?&#164;-galactosidase activity defined follows: (A420/minute/ <br /> milligram total protein) &#226;&#171;&#187; 1,000. Units CAT activity defined counts   8 strains glycerol) removed data set. The <br /> per minute organic phase expressed percentage total counts   remaining mutants affected PIS1 specifically, PIS1 INO1, <br /> per minute (percent conversion) divided protein assayed (in        PIS1 TCM1 (Table 2). <br /> micrograms) time incubation (in hours). Protein concentration       In strains grown 2% glucose, 25 mutant strains displayed <br /> extract determined using Bio-Rad (Rockville Center, N.Y.) protein <br /> assay kit. A minimum independent measurements each <br />                                                                                     decreased PIS1-lacZ expression, 20 mutant strains affected <br /> data point.                                                                         PIS1-lacZ INO1-lacZ expression, 24 mutant strains <br />   Growth phase assays. Expression PIS1 gene monitored               yielded altered PIS1-lacZ TCM1-lacZ expression (Table <br /> BRS1001 transformants harboring pPIS1-918 (cat) reporter construct. Cells       2). In strains grown 3% glycerol, PIS1-lacZ expression was <br /> were grown I&#226;&#171;&#186;C&#226;&#171;&#186; I&#226;&#171;&#185;C&#226;&#171;&#185; media absence presence 0.7 M <br />                                                                                     specifically altered 20 mutant strains, 21 mutant strains had <br /> NaCl. Five-milliliter aliquots removed intervals span of <br /> 100 h monitored density (using Klett-Summerson colorimeter). CAT          altered PIS1-lacZ INO1-lacZ expression, 10 strains <br /> activity (described above) determined samples simultaneously.          yielded altered PIS1-lacZ TCM1-lacZ expression (Table <br />                                                                                     2). <br />                                                                                        The mutant set contains overrepresentation mutants <br />                                  RESULTS <br />                                                                                     affect specific biological processes. The genomic <br />    A genome-wide screen genes affect PIS1-lacZ expres-                     screen identified genes involved peroxisome biogenesis <br /> sion glucose glycerol. To obtain comprehensive un-                         (pex3, pex4, pex17, pex22) function (acb1 gpd1). <br /> derstanding regulation PIS1 expression, screened                       The GPD1 gene involved early step phospho- <br /> the VYDS (&#226;&#172;&#402;4,800 viable mutant strains) changes PIS1-                        lipid biosynthesis (Fig. 1). The screen yielded genes <br />  VOL. 4, 2005                                                                                                         PIS1 GENOMICS          607 <br />  <br />  <br />  <br />  <br />    FIG. 2. Peroxisomal biogenesis mutants affect PIS1-lacZ gene expression. PIS1-lacZ gene expression assayed using plasmid pMA107 <br /> containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). Plasmid pMA107 transformed wild-type (WT) strain (BY4742; MAT&#226;?&#163;) and <br /> isogenic strains containing pex&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity measured. Cultures grown Ura&#226;&#171;&#186; medium containing 2% <br /> glucose (solid bars), 3% glycerol (empty bars), 0.1% oleic acid 0.2% Tween 80 (hatched bars). The mutants divided classes <br /> described text. Note y-axis range differs bar graphs. <br />  <br />  <br /> required chromatin silencing (rif1, hst3, hst4, sin3).            present VYDS identified genomic <br /> There overrepresentation mutants DNA repair                   plate screen. We performed liquid assays quantify effect <br /> (ptc3, ogg1, ntg2, rpb4, rpb9) recently as-            known pex&#226;&#338;&#172; mutants grown medium containing 2% <br /> sociated PI metabolism (84). It interesting              glucose 3% glycerol PIS1-lacZ expression. In addition, <br /> subunits RNA polymerase involved transcription-coupled              quantified PIS1-lacZ expression medium containing <br /> repair identified (rpb4 rpb9). The genomic                  0.1% oleic acid, carbon source causes peroxisomes to <br /> screen yielded mutants involved carbon source                 proliferate (70). <br /> regulation play role glycerol repression (nrg2             As expected wild-type strain, PIS1-lacZ expression <br /> reg2). Last, genes required PI metabolism iden-             repressed 3% glycerol unaffected 0.1% oleic acid <br /> tified (FAB1 ACB1 required synthesis phosphoi-             (Fig. 2). The results pex&#226;&#338;&#172; mutants af- <br /> nositides sphingolipids, respectively). We assume            fected expression PIS1-lacZ reporter one <br /> genomic screen described did identify gene                 growth condition. This included mutants were <br /> affecting PIS1-lacZ expression (see below); however, did               identified initial plate screen. That is, PIS1-lacZ gene <br /> provide wealth information                  expression cultures grown 3% glycerol elevated in <br /> possible traditional genetic screens used.                   pex3&#226;&#338;&#172;, pex4&#226;&#338;&#172;, pex22&#226;&#338;&#172; mutant strains. It important that <br />    Peroxisomal biogenesis affects glycerol regulation PIS1-            observed elevated expression pex12&#226;&#338;&#172; pex13&#226;&#338;&#172; <br /> lacZ expression. Peroxisome biogenesis conserved process             mutants grown 3% glycerol (Fig. 2). These mutants were <br /> among eukaryotes, involving 32 known peroxins (20,               present original plate screen eliminated subse- <br /> 60, 70, 72&#226;&#8364;&#8220;74, 77, 78, 80, 81). Peroxisomes membrane-                 quent screens, suggesting screening criteria have <br /> bound organelles function metabolic pathways involved             stringent. However, observed pex17&#226;&#338;&#172; <br /> in inactivation toxic substances (H2O2-based respira-              mutant identified plate screen did reveal difference <br /> tion), regulation cellular oxygen levels, metab-           relative wild type quantified liquid assay. <br /> olism lipids, nitrogen bases, carbohydrates including              This discrepancy explained fact plate <br /> fatty acid &#226;?&#164;-oxidation (60, 72, 74, 77, 78). Genomic analysis          assay reports accumulation &#226;?&#164;-galactosidase several <br /> the VYDS strain collection identified PIS1-lacZ expres-              phases growth, liquid assay quantifies &#226;?&#164;-galac- <br /> sion cells grown 3% glycerol X-Gal plates signifi-              tosidase activity single stage growth. <br /> cantly increased pex3&#226;&#338;&#172;, pex4&#226;&#338;&#172;, pex17&#226;&#338;&#172;, pex22&#226;&#338;&#172; mutant                  The effect pex&#226;&#338;&#172; mutants PIS1-lacZ expression fell <br /> strains. However, assumed screen               classes. Class I contained mutants generally <br /> VYDS relied relatively modest phenotype, probably did             displayed <span id='am-81' about='xsp:pattern' typeof='owl:Thing'>pattern</span> PIS1-lacZ expression the <br /> not identify genes regulate PIS1-lacZ expression. Given          carbon sources wild-type strain, elevated <br /> this possibility, tested known pex&#226;&#338;&#172; mutants             expression glucose oleic acid decreased expression <br />  608     GARDOCKI ET AL.                                                                                                 EUKARYOT. CELL <br />  <br />  <br /> in glycerol (pex2&#226;&#338;&#172;, pex3&#226;&#338;&#172;, pex10&#226;&#338;&#172;, pex11&#226;&#338;&#172;, pex17&#226;&#338;&#172;, pex29&#226;&#338;&#172;) <br /> (Fig. 2). However, pex3&#226;&#338;&#172;, pex10&#226;&#338;&#172;, pex11&#226;&#338;&#172; mutants <br /> clearly yielded altered expression levels relative the <br /> wild-type control. Class II included mutants that <br /> yielded levels expression oleic acid wild- <br /> type expression levels oleic acid (pex1&#226;&#338;&#172;, pex6&#226;&#338;&#172;, pex15&#226;&#338;&#172;, <br /> pex19&#226;&#338;&#172;, pex21&#226;&#338;&#172;, pex31&#226;&#338;&#172;, pex32&#226;&#338;&#172;) (Fig. 2). Class III in- <br /> cluded mutants yielded levels expression glycerol that <br /> were wild-type expression levels glycerol de- <br /> creased levels expression oleic acid (pex4&#226;&#338;&#172;, pex8&#226;&#338;&#172;, <br /> pex12&#226;&#338;&#172;, pex13&#226;&#338;&#172;, pex14&#226;&#338;&#172;, pex18&#226;&#338;&#172;, pex22&#226;&#338;&#172;, pex25&#226;&#338;&#172;, pex27&#226;&#338;&#172;, <br /> pex28&#226;&#338;&#172;, pex30&#226;&#338;&#172;) (Fig. 2). Class IV included mutants that <br /> generally yielded low levels expression carbon <br /> sources (pex5&#226;&#338;&#172; pex7&#226;&#338;&#172;) (Fig. 2). <br />    The results clearly screening VYDS             FIG. 3. Chromatin silencing gene mutants affect PIS1-cat expres- <br /> yield valuable information regarding regulation gene expres-     sion. A PIS1 promoter fragment (pPIS-325) fused cat <br />                                                                     reporter gene integrated single copy GAL4 locus in <br /> sion, modestly regulated gene. These pex mutants         strains BY4742 (MAT&#226;?&#163;) BY4741 (MATa) isogenic rif1&#226;&#338;&#172;, <br /> could possibly identified traditional         hst3&#226;&#338;&#172;, hst4&#226;&#338;&#172; strains. PIS1-cat expression assayed transfor- <br /> means. Most importantly, results provide evi-       mants grown medium containing 2% glucose (MAT&#226;?&#163;, solid <br /> dence coordination synthesis peroxisomes          bars; MATa, hatched bars) 3% glycerol (MAT&#226;?&#163;, bars; MATa, <br />                                                                     horizontal bars). WT, wild type. <br /> and phospholipids. <br />    Chromatin silencing genes affect PIS1-lacZ expression. <br /> Genomic analysis VYDS identified rif1&#226;&#338;&#172;, hst3&#226;&#338;&#172;,      strain (37%) cells grown 2% glucose (Fig. 4). As <br /> hst4&#226;&#338;&#172; mutant strains elevated PIS1-lacZ expression 3%       expected VYDS screen, gpd1&#226;&#338;&#172; strain yielded a <br /> glycerol X-Gal plates. All mutant strains involved     20% decrease &#226;?&#164;-galactosidase activity (Fig. 4). However, the <br /> chromatin silencing telomeres. The HST3 HST4 genes           gpd2&#226;&#338;&#172; mutant strain indistinguishable wild-type <br /> are homologs SIR2 involved regulating short-     control (Fig. 4). <br /> chain fatty acid metabolism (8, 71). Hst3p required        PIS1 promoter binding proteins identified ChIPs are <br /> silencing origin replication endogenous 2&#226;?&#174;m      required PIS1 expression. In effort identify additional <br /> plasmid (31). To determine PIS1-lacZ plate phenotype         factors regulate PIS1 expression, mined ChIP-on- <br /> was genuine effects transcription effect    chip database containing information yeast DNA-binding <br /> plasmid copy number, assayed PIS1-cat reporter stably          proteins (47). This analysis identified Pho2, Yfl044c, Ste12 <br /> integrated single copy GAL4 locus. We deter-         weak candidate PIS1 promoter binding proteins (47). Pho2 <br /> mined mating type affected expression comparing CAT           involved response phosphate starvation (5, 39, 40, <br /> activity MATa (BY4741 background) MAT&#226;?&#163; (BY4742               43, 51, 67, 68), Ste12 involved pheromone and <br /> background) transformants. PIS1-cat expression         pseudohyphal regulation (16, 35, 62, 83). YFL044c encodes a <br /> mutant strains elevated glucose glycerol re-        hypothetical open reading frame (ORF) unknown func- <br /> gardless mating type, hst3&#226;&#338;&#172; hst4&#226;&#338;&#172; mutants       tion. Interestingly, pho2 mutant identified plate <br /> had obvious effect (Fig. 3). These results       screen yielding reduced expression 2% glucose X-Gal <br /> that plate phenotype indirect effect       medium (Table 2). Previously, UAS elements (UAS1 <br /> plasmid copy number. This evidence chromatin <br /> silencing affecting PIS1 expression. <br />    Mutants glycerol utilization affect regulation PIS1-lacZ <br /> expression. Genomic analysis VYDS identified the <br /> gpd1&#226;&#338;&#172; mutant strain decreased PIS1-lacZ expression on <br /> 2% glucose X-Gal plates. GPD1 encodes NADH-depen- <br /> dent cytosolic glycerol-3-phosphate dehydrogenase, key en- <br /> zyme initial stages glycerol metabolism (Fig. 1) (15, <br /> 61). This interesting result given PIS1 gene expres- <br /> sion repressed glycerol. This led determine addi- <br /> tional enzymes involved early stages glycerol utiliza- <br /> tion affect PIS1-lacZ expression. Thus, quantified <br /> &#226;?&#164;-galactosidase activity gpd1&#226;&#338;&#172; mutant strain as <br /> gpd2&#226;&#338;&#172;, gut1&#226;&#338;&#172;, gut2&#226;&#338;&#172; mutant strains grown 2% glucose. <br /> GPD2 encodes isoform glycerol-3-phosphate dehydroge-             FIG. 4. Mutants glycerol utilization affect regulation PIS1- <br /> nase (Fig. 1) (59), GUT1 encodes glycerol kinase,         lacZ expression. PIS1-lacZ expression assayed using plasmid <br /> GUT2 encodes mitochondrial glycerol-3-phosphate dehydro-            pMA107 containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). <br />                                                                     Plasmid pMA107 transformed wild-type (WT) strain <br /> genase (Fig. 1) (26, 27).                                           (BY4742; MAT&#226;?&#163;) isogenic strains containing gpd1&#226;&#338;&#172;, gpd2&#226;&#338;&#172;, gut1&#226;&#338;&#172;, <br />    PIS1-lacZ expression increased substantially gut2&#226;&#338;&#172;        gut2&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity measured. <br /> strain (176%) lesser extent gut1&#226;&#338;&#172; mutant       Cultures grown Ura&#226;&#171;&#186; medium containing 2% glucose. <br />  VOL. 4, 2005                                                                                                     PIS1 GENOMICS        609 <br />  <br />  <br />                                                                          YFL044c ORF (along 5&#226;&#172;&#732;- 3&#226;&#172;&#732;-flanking sequences) into <br />                                                                          yfl044c mutant strain assayed PIS1-lacZ expres- <br />                                                                          sion. The presence YFL044c pRS200 plasmid re- <br />                                                                          stored PIS1-lacZ expression level &#226;&#172;&#402;3-fold higher that <br />                                                                          wild-type strain (data shown). This result suggests <br />                                                                          Yfl044c limiting PIS1 expression wild-type <br />                                                                          strain conditions tested here. <br />                                                                             PIS1 promoter deletion analysis identifies regulatory ele- <br />                                                                          ment required glycerol repression. A nested set dele- <br />                                                                          tions PIS1 promoter fused cat reporter gene in <br />                                                                          plasmid pBM2015 transformed wild-type strain <br />                                                                          (BRS1001) targeting integration GAL4 locus single <br />                                                                          copy single orientation. We previously reported <br />                                                                          analysis deletion subset grown glucose identi- <br />    FIG. 5. Transcription factor mutants affect PIS1-lacZ gene expres-    fied UAS elements positions &#226;&#171;&#186;224 &#226;&#171;&#186;205 (UAS1), <br /> sion. PIS1-lacZ expression assayed using plasmid pMA107 <br /> containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). Plasmid            &#226;&#171;&#186;184 &#226;&#171;&#186;149 (UAS2), &#226;&#171;&#186;149 &#226;&#171;&#186;138 (UAS3) (Fig. 6A) <br /> pMA107 transformed wild-type (WT) strain (BY4742;             (25). Deletion UAS1 decreased PIS1-cat expression half <br /> MAT&#226;?&#163;) isogenic strains containing yfl044&#226;&#338;&#172;, ste12&#226;&#338;&#172;, pho2&#226;&#338;&#172;, pho4&#226;&#338;&#172;,     (Fig. 6A, compare pPIS-224 pPIS-205), deletion of <br /> bas1&#226;&#338;&#172;, swi5&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity mea-   UAS2 exacerbated decrease PIS1-cat expression an- <br /> sured. Cultures grown Ura&#226;&#171;&#186; medium containing 2% glucose. <br />                                                                          90% (Fig. 6A, compare pPIS-205 pPIS-149). PIS1- <br />                                                                          cat expression detectable levels UAS3 was <br />                                                                          deleted (Fig. 6A, compare pPIS-225 pPIS-127). How- <br /> [&#226;&#171;&#186;224 &#226;&#171;&#186;205], UAS2 [&#226;&#171;&#186;184 &#226;&#171;&#186;149], UAS3 [&#226;&#171;&#186;149                   ever, data showed UAS elements were <br /> &#226;&#171;&#186;127]) required PIS1 gene expression identified (25).           required response glycerol. Curiously, deleting any <br /> Consistent results, potential Ste12p Pho2p bind-            UAS elements increased degree re- <br /> ing sites identified PIS1 UAS1, PIS1 UAS2                  pression caused growth glycerol (<span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> &#226;&#172;&#402;3-fold &#226;&#172;&#402;8-fold <br /> contains additional potential Pho2p binding site.                     [Fig. 6A]). One explanation result the <br />    These results prompted quantify PIS1-lacZ expression            UAS elements deleted, glycerol repression factor is <br /> in pho2&#226;&#338;&#172;, yfl044c&#226;&#338;&#172;, ste12&#226;&#338;&#172; mutant strains. PIS1-lacZ ex-             effective repressing transcription promoter <br /> pression generally decreased mutant strains,            activity lower. <br /> the effect obvious pho2&#226;&#338;&#172; yfl044c&#226;&#338;&#172; mutants               Because UAS elements required expres- <br /> (Fig. 5). The homeodomain transcription factor Pho2 known             sion, necessary generate internal deletions order to <br /> to interact Swi5, Pho4, Bas1 vivo activate               analyze promoter sequences downstream UAS <br /> transcription HO gene (7, 9), phosphate utilization genes         elements. Analysis PIS1 promoter downstream the <br /> (7, 82), genes involved purine histidine biosynthesis         UAS elements identified repressor site (URSGLY), <br /> (7), respectively. Because Pho2 associates              located promoter proximal, required glycerol repression. <br /> proteins, quantified PIS1-lacZ expression swi5&#226;&#338;&#172;,              The data identified 25-bp region (&#226;&#171;&#186;76 &#226;&#171;&#186;51) that, when <br /> pho4&#226;&#338;&#172;, bas1&#226;&#338;&#172; mutant strains. PIS1-lacZ expression in-                deleted, abolishes PIS1 glycerol repression (Fig. 6A). This or- <br /> creased 34% bas1&#226;&#338;&#172; strain, pho4&#226;&#338;&#172; swi5&#226;&#338;&#172; mu-               ganization regulatory elements reminiscent GAL4 gene <br /> tant strains exhibited lower levels PIS1-lacZ expression              expression repressed fourfold growth glucose <br /> (53% 34%, respectively) relative wild-type            requires URS element located downstream UAS <br /> strain (Fig. 5). Because Pho4 Pho2 required induc-           element (29, 30). A heterologous employed to <br /> tion target genes response low phosphate concentra-             examine PIS1 URSGLY function. The 25-bp region <br /> tions (5, 39, 40, 43, 51, 67, 68), quantified effect      (&#226;&#171;&#186;76 &#226;&#171;&#186;51) defined deletion analysis inserted into <br /> phosphate PIS1-lacZ expression. However,                plasmid pLG&#226;&#338;&#172;312, previously utilized test URS function <br /> PIS1-lacZ expression significantly affected phos-             (32, 49). Plasmid pLG&#226;&#338;&#172;312 contains CYC1 promoter fused <br /> phate concentration (data shown).                                    lacZ reporter gene. The 25-bp PIS1 URSGLY caused <br />    The ste12&#226;&#338;&#172; mutant did dramatically affect PIS1-lacZ               CYC1-lacZ expression decrease dramatically (Fig. 6B). This <br /> expression condition tested response               result indicates PIS1 URSGLY sequence functions a <br /> pheromone induction (data shown). However, Ste12                 URS element. The URSGLY effect CYC1-lacZ expression in <br /> affect PIS1 expression conditions tested           cells grown glycerol assessed CYC1 <br /> since Ste12 known regulate diverse set genes               promoter regulated carbon source. <br /> response different signals (16, 35, 62, 83).                Glycerol repression PIS1 expression reversed glu- <br />    The effect yfl044c&#226;&#338;&#172; mutant PIS1-lacZ expression          cose. Glycerol-mediated repression PIS1 transcription is <br /> the evidence regulation gene expression              unusual does appear involve previ- <br /> putative transcription factor. While role Yfl044c             ously identified regulators carbon source regulation (1, 2). <br /> defined ChIP-on-chip database, evidence            To determine glycerol-mediated repression reversible, <br /> to suggest regulates PIS1 expression &#226;?&#164;-galacto-          PIS1-cat activity wild-type strain (BRS1001) containing <br /> sidase liquid assay. To corroborate role Yfl044c PIS1          pPIS-325 assayed culture grown medium contain- <br /> expression, transformed pRS200 derivative containing            ing 2% glucose 3% glycerol various concentrations of <br />  610     GARDOCKI ET AL.                                                                                                     EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 6. A. Identification PIS1 promoter element required glycerol-mediated repression. BRS1001 (wild-type) transformants containing <br /> various promoter deletions grown synthetic medium lacking uracil containing 2% glucose 3% glycerol. Yeast extracts from <br /> the transformants prepared assayed CAT activity. MRD1 divergent ORF. The locations MCEs (Mcm1p binding sites) <br /> (black box), Rox1p binding site (gray box), potential TATA boxes (bars), UAS elements, URS element shown. B.D., below <br /> detection. B. PIS1 URSGLY element functions URS element. Plasmid pLG&#226;&#338;&#172;312 &#226;&#171;&#185; URSGLY, containing 25 bp PIS1 gene promoter (&#226;&#171;&#186;76 <br /> to &#226;&#171;&#186;51), transformed wild-type strain (BRS1001), &#226;?&#164;-galactosidase activity measured. Cultures grown Ura&#226;&#171;&#186; 2% glucose <br /> medium. <br />  <br />  <br />  <br /> glucose (range, 0.1 2%). The data glucose con-              I&#226;&#171;&#186;C&#226;&#171;&#186; medium (46, 63). We discovered growing cells <br /> centrations low 0.1% able partially reverse glyc-          hyperosmotic medium (0.7 M NaCl) eliminates growth <br /> erol repression concentrations 1 2% com-                phase regulation affecting inositol-choline-medi- <br /> pletely reversed repression PIS1-cat expression (Fig. 7).             ated regulation (63). These observations prompted exami- <br />    PIS1 expression elevated fermentable carbon sources             nation effect growth phase hyperosmotic condi- <br /> (glucose galactose) relative nonfermentable carbon              tions PIS1 expression. This examination by <br /> source (glycerol) (2). We quantified effect nonfer-         quantifying CAT activity pPIS1-918 transformants of <br /> mentable carbon sources PIS1-cat expression using              BRS1001. These experiments revealed growth <br /> pPIS-325 construct. In addition glycerol, PIS1-cat expression         phase hyperosmotic medium affected expression the <br /> is reduced &#226;&#172;&#402;50% cells grown acetate, ethanol,              PIS1-cat gene (Fig. 9). <br /> or lactate carbon source (Fig. 8). <br />    PIS1 expression affected growth phase hyperos- <br />                                                                                                    DISCUSSION <br /> motic stress. It reported phospholipid <br /> biosynthetic genes regulated response growth phase               Until recently, PIS1 gene expression believed be <br /> (46). For example, CHO1-lacZ expression levels I&#226;&#171;&#186;C&#226;&#171;&#186; me-               regulated (2, 25, 45, 75, 76). However, previously <br /> dium low lag phase increase log                shown PIS1 expression regulated carbon source (2) <br /> phase, peaking beginning stationary phase (46, 63).            response oxygen availability (25, 45, 75, 76). Multiple <br /> Once stationary phase, expression levels decrease precipi-            approaches applied identify genes affect PIS1 <br /> tously reach initial expression levels seen lag        expression. The genomic screen VYDS identified 69 <br /> phase. This expression pattern similar I&#226;&#171;&#185;C&#226;&#171;&#185; medium;                mutants reduced PIS1 expression glucose 51 mu- <br /> however, expression repressed medium,             tants increased PIS1 expression glycerol. PIS1 pro- <br /> overall expression levels lower observed           moter deletion analysis coupled mining databases of <br />  VOL. 4, 2005                                                                                                         PIS1 GENOMICS           611 <br />  <br />  <br />  <br />  <br />    FIG. 7. Glycerol repression PIS1-cat expression reversed by <br /> glucose. BRS1001 transformants containing pPIS-325 grown in <br /> Ura&#226;&#171;&#186; synthetic medium containing 3% glycerol 0, 0.1, 0.25, 0.5, <br /> 1.0, 1.5, 2% glucose, CAT activity assayed. <br />  <br />  <br />  <br /> DNA-binding proteins identified UAS elements one <br /> URS element putative cognate transcription factors. For <br /> example, promoter deletion analysis identified UAS <br /> element potential Ste12 Pho2 binding sites (UAS1 <br /> [&#226;&#171;&#186;224 &#226;&#171;&#186;205]), additional UAS element potential <br /> Pho2 binding site known Mcm1 binding sites (UAS2 <br /> [&#226;&#171;&#186;184 &#226;&#171;&#186;149]), potential Gcr1 binding site                   FIG. 9. A. PIS1-cat expression regulated growth phase or <br /> another UAS element (UAS3 [&#226;&#171;&#186;149 &#226;&#171;&#186;138]),                 hyperosmotic stress. BRS1001 transformants containing pPIS1-918 <br /> Rox1 binding site essential anaerobic regulation PIS1          grown Ura&#226;&#171;&#186; synthetic medium presence inositol and <br /> (25). Comparison promoters related Saccharomyces                  choline (I&#226;&#171;&#185;C&#226;&#171;&#185;) (solid squares), absence inositol choline <br />                                                                           (I&#226;&#171;&#186;C&#226;&#171;&#186;) (open squares), presence 0.7 M NaCl (solid circles), <br /> yeast species revealed strong sequence similarity               presence inositol, choline, 0.7 M NaCl (open circles). <br /> three UAS regions, including potential Ste12 binding sites,           Yeast extracts transformants prepared assayed for <br /> and known Mcm1 Rox1 binding sites S. cerevisiae               CAT activity. B. Growth patterns cultures used described above <br /> and S. castellii, S. bayanus, S. kluyveri, S. kudriavzevii, S.        (A). <br /> mikatae (25). Mining yeast ChIP-on-chip database iden- <br />  <br />  <br />                                                                           tified Yfl044c Mcm1 candidate PIS1 promoter binding <br />                                                                           proteins (47). <br />                                                                              This study demonstrated genomic approaches provide <br />                                                                           excellent means identify regulatory mechanisms con- <br />                                                                           trol modestly regulated essential genes PIS1. How- <br />                                                                           ever, single strategies provide excellent starting point, <br />                                                                           combination multiple genomic approaches is <br />                                                                           essential. For example, Rox1 identified regulator <br />                                                                           PIS1 promoter ChIP-on-chip study (P value of <br />                                                                           0.19) (47) VYDS screen; however, suggested by <br />                                                                           searches databases DNA-binding proteins coupled with <br />                                                                           promoter deletion (25) microarray studies (45, 75, 76). <br />                                                                           Our previous studies clearly shown Rox1 binds and <br />                                                                           bends PIS1 promoter exert anaerobic regulation the <br />                                                                           PIS1 gene (25). Conversely, VYDS screen employed <br />                                                                           did identify Yfl044c, examination PIS1 <br />                                                                           promoter sequence does reveal obvious binding site for <br />    FIG. 8. Analysis PIS1-cat expression response different       Yfl044c (TTCTTKTYYTTTT) (47). However, ChIP-on- <br /> carbon sources. A PIS1 promoter fragment (pPIS-325) created PCR        chip study placed Yfl004c cusp acceptable bind- <br /> was fused cat reporter gene integrated single copy   ing threshold (P value 0.006) (47), shown here <br /> GAL4 locus strain BRS1001 (wild type). PIS1-cat expression (pPIS- <br /> 325) assayed BRS1001 transformants grown Ura&#226;&#171;&#186; medium <br />                                                                           Yfl044c does regulate PIS1-lacZ expression. The most <br /> containing 2% glucose, 3% glycerol, 3% acetate, 3% ethanol, 3%         obvious PIS1 promoter binder defined ChIP-on-chip <br /> lactate.                                                                  study Mcm1 (P value 0.000003), protein has <br />  612     GARDOCKI ET AL.                                                                                                            EUKARYOT. CELL <br />  <br />  <br /> previously shown bind PIS1 promoter (44). How-          affect PIS1 expression (REG1, GLC7, MIG1, etc. [M. E. <br /> ever, VYDS screen identified MCM1          Gardocki J. M. Lopes, unpublished results]). We did, <br /> it essential gene. The use multiple genomic approaches     however, REG2 NRG2 affected PIS1-lacZ expres- <br /> should expected generate overlapping information.        sion genes role carbon source regu- <br /> This case Pho2 (also known Bas2 Grf10),         lation (6). Nrg2 binds DNA directly, levels de- <br /> which suggested regulator PIS1 promoter       creased glycerol- ethanol-grown cells. However, Nrg2 a <br /> deletion analysis potential binding sites        repressor expected increase PIS1 <br /> UAS1 UAS2 (25). The VYDS screen identified Pho2            expression glycerol (6). Moreover, PIS1 promoter does <br /> as regulator PIS1-lacZ expression. However, ChIP-on-       contain binding site Nrg2. Thus, possible that <br /> chip database did identify Pho2 potential binder (P        novel transcription factor involved glycerol-medi- <br /> value 0.5) (47). These results underscore benefits        ated PIS1 expression. This factor present collec- <br /> employing multiple genomic screens coupled database            tion genes defined VYDS screen. <br /> mining traditional approaches.                                Our knowledge PIS1 expression, regulation, the <br />     This study demonstrated genomic approaches         various biological processes intimately <br /> might identify biological functions affect expression   intertwined begin discerned direct <br /> a gene, illuminating interplay biological pro-      result results presented here. Genomic studies coupled <br /> cesses. The VYDS screen identified overrepresentation         database mining classical strategies can <br /> mutants affect biological processes: peroxisome bio-     obviously successfully combined study modestly regu- <br /> genesis (pex3, pex4, pex17, pex22) function (acb1       lated genes. Of course, challenge interpreting the <br /> gpd1), chromatin silencing (rif1, hst3, hst4, sin3), DNA    wealth information genomic approaches provide. <br /> repair (ptc3, ogg1, ntg2, rpb4, rpb9). Alterations PIS1      A critical process distinguish between <br /> expression 26 assayed pex mutants extremely inter-       direct indirect effects. Another important issue will <br /> esting given cellular role peroxisomes play, specifically    need addressed determine genes and <br /> cellular oxygen regulation, metabolism lipids, nitro-    processes defined affect PI metabolism. PI metabolism <br /> gen bases, carbohydrates, fatty acid &#226;?&#164;-oxidation (60, 72, 74,   shown affected anaerobic conditions the <br /> 77, 78). This finding suggests changes peroxi-      Rox1 protein (25) carbon source (D. Kamath J. M. <br /> some, potentially alterations fatty acid metabolism       Lopes, unpublished results), suggesting regulation PIS1 <br /> (60, 72, 74, 77&#226;&#8364;&#8220;79), responsible alterations PIS1   expression does affect PI metabolism. Furthermore, long <br /> expression, reminiscent seen CIT2 cells     half-life PIS1 transcript (57 min) (1) AUG codon <br /> altered mitochondrial activity (10, 13). This finding       located upstream PI synthase AUG codon (25) also <br /> supports existence pathway retrograde regulation       contribute regulation PIS1 gene expression. <br /> between peroxisomes nucleus suggestive the <br /> existence novel mechanism cell adjust changes                              ACKNOWLEDGMENTS <br /> in peroxisomal activity. <br />                                                                        We thank Meng Chen, Linan Chen, Leandria Hancock, Ying He, <br />     Note identification chromatin silencing DNA          Niketa Jani, Sangeeta Oza, Jimmy Lo efforts editing <br /> repair mutants VYDS screen. The dual roles HST3          manuscript. We thank Edward Golenberg George Brush <br /> HST4 play chromatin silencing regulating short-chain         helpful discussions. <br /> fatty acid metabolism suggest relationship             This work initially supported NSF grant (MCB-0110408) <br /> PIS1 expression fatty acid metabolism. Our understanding        completed support NSF grant (MCB-0415511) to <br />                                                                     J.M.L., William A. Turner, Jr., Memorial Foundation scholarship <br /> of role Hst3 Hst4 silencing devel-     M.E.G., undergraduate research grant Wayne State <br /> oped silencing proteins. Moreover, studies     University Undergraduate Research Council K.B. <br /> cannot distinguish direct indirect effects PIS1 <br /> expression. Additional studies focus pro-                                      REFERENCES <br /> teins bind PIS1 promoter directly. The DNA repair genes         1. Anderson, M. S. 1996. Analysis PIS1 gene expression Saccharomyces <br /> defined VYDS screen included RPB4 RPB9. These               cerevisiae. Loyola University Chicago, Maywood, Ill. <br />                                                                     2. Anderson, M. S., J. M. Lopes. 1996. Carbon source regulation PIS1 <br /> genes affect PIS1 expression play            gene expression Saccharomyces cerevisiae involves MCM1 gene the <br /> regulatory role transcription PIS1 role      two-component regulatory gene, SLN1. J. Biol. Chem. 271:26596&#226;&#8364;&#8220;26601. <br />                                                                     3. Antonsson, B. 1997. Phosphatidylinositol synthase mammalian tissues. <br /> in DNA repair. Regardless mechanism               Biochim. Biophys. Acta 1348:179&#226;&#8364;&#8220;186. <br /> two genes affect PIS1 expression, clear DNA        4. Ashburner, B. P., J. M. Lopes. 1995. Autoregulated expression the <br /> repair genes affected PIS1 expression. The identification         yeast INO2 INO4 helix-loop-helix genes effects cooperative regulation <br />                                                                        target genes. Mol. Cell. Biol. 15:1709&#226;&#8364;&#8220;1715. <br /> of link damage checkpoint pathway PI me-            5. Barbaric, S., M. Mu  &#194;&#168;nsterko&#194;&#168;tter, C. Goding, W. Ho&#194;&#168;rz. 1998. Cooperative <br /> tabolism suggested genome-wide studies               Pho2-Pho4 interactions PHO5 promoter critical binding of <br /> (84). This entirely unexpected given PIS1 PI           Pho4 UASp1 efficient transactivation Pho4 UASp2. Mol. <br />                                                                        Cell. Biol. 18:2629&#226;&#8364;&#8220;2639. <br /> have role cell cycle progression (18, 19, 36, 37).             6. Berkey, C. D., V. K. Vyas, M. Carlson. 2004. Nrg1 Nrg2 transcrip- <br />     The promoter deletion study identified URS element,              tional repressors differently regulated response carbon source. <br />                                                                        Eukaryot. Cell 3:311&#226;&#8364;&#8220;317. <br /> downstream UAS elements, required         7. Bhoite, L. T., J. M. Allen, E. Garcia, L. R. Thomas, I. D. Gregory, W. P. Voth, <br /> glycerol repression. Examination sequence defined            K. Whelihan, R. J. Rolfes, D. J. Stillman. 2002. Mutations Pho2 <br /> the URSGLY element did reveal obvious protein-bind-            (Bas2) transcription factor differentially affect activation partner <br />                                                                        proteins Bas1, Pho4, Swi5. J. Biol. Chem. 277:37612&#226;&#8364;&#8220;37618. <br /> ing site. This observation consistent findings     8. Brachmann, C., J. M. Sherman, S. E. Devine, E. E. Cameron, L. Pillus, and <br /> many genes involved carbon source regulation yeast            J. D. Boeke. 1995. The SIR2 gene family, conserved bacteria humans, <br />  VOL. 4, 2005                                                                                                                                 PIS1 GENOMICS               613 <br />  <br />       functions silencing, cell cycle progression, chromosome stability.               phosphatidylinositol synthesis regulation S phase induction. Exp. <br />       Genes Dev. 9:2888&#226;&#8364;&#8220;2902.                                                                 Cell Res. 215:228&#226;&#8364;&#8220;233. <br />  9.   Brazas, R. M., D. J. Stillman. 1993. The Swi5 zinc-finger Grf10           37.   Imoto, M., Y. Taniguchi, H. Fujiwara, K. Umezawa. 1994. Enhancement <br />       homeodomain proteins bind DNA cooperatively yeast HO promoter.                   CDP-DA:inositol transferase activity src- erbB2-transformed cells. <br />       Proc. Natl. Acad. Sci. USA 90:11237&#226;&#8364;&#8220;11241.                                              Exp. Cell Res. 212:151&#226;&#8364;&#8220;154. <br /> 10.   Butow, R. A., N. G. Avadhani. 2004. Mitochondrial signaling: retro-       38.   Jackson, M., D. C. Crick, P. J. Brennan. 2000. Phosphatidylinositol an <br />       grade response. Mol. Cell 14:1&#226;&#8364;&#8220;15.                                                      essential phospholipid mycobacteria. J. Biol. Chem. 275:30092&#226;&#8364;&#8220;30099. <br /> 11.   Carman, G. M., S. A. Henry. 1999. Phospholipid biosynthesis yeast      39.   Kaffman, A., I. Herskowitz, R. Tjian, E. K. O&#226;&#8364;&#8482;Shea. 1994. Phosphoryla- <br />       Saccharomyces cerevisiae interrelationship metabolic pro-                tion transcription factor PHO4 cyclin-CDK complex, PHO80- <br />       cesses. Prog. Lipid Res. 38:361&#226;&#8364;&#8220;399.                                                    PHO85. Science 263:1153&#226;&#8364;&#8220;1156. <br /> 12.   Carman, G. M., G. M. Zeimetz. 1996. Regulation phospholipid                40.   Kaffman, A., N. M. Rank, E. K. O&#226;&#8364;&#8482;Shea. 1998. Phosphorylation regulates <br />       synthesis yeast Saccharomyces cerevisiae. J. Biol. Chem. 271:13293&#226;&#8364;&#8220;              association transcription factor Pho4 import receptor Pse1/ <br />       13296.                                                                                  Kap121. Genes Dev. 12:2673&#226;&#8364;&#8220;2683. <br /> 13.   Chelstowska, A., R. A. Butow. 1995. RTG genes yeast function       41.   Kelley, M. J., A. M. Bailis, S. A. Henry, G. M. Carman. 1988. Regulation <br />       communication mitochondria nucleus required                phospholipid biosynthesis Saccharomyces cerevisiae inositol. Inositol <br />       expression genes encoding peroxisomal proteins. J. Biol. Chem. 270:                  inhibitor phosphatidylserine synthase activity. J. Biol. Chem. 263: <br />       18141&#226;&#8364;&#8220;18146.                                                                            18078&#226;&#8364;&#8220;18085. <br /> 14.   Chen, D.-C., B.-C. Yang, T.-T. Kuo. 1992. One-step transformation          42.   Kelly, B. L., M. L. Greenberg. 1990. Characterization regulation of <br />       yeast stationary phase. Curr. Genet. 21:83&#226;&#8364;&#8220;84.                                       phosphatidylglycerolphosphate phosphatase Saccharomyces cerevisiae. <br /> 15.   Costenoble, R., L. Adler, C. Niklasson, G. Liden. 2003. Engineering              Biochim. Biophys. Acta 1046:144&#226;&#8364;&#8220;150. <br />       metabolism Saccharomyces cerevisiae anaerobic production            43.   Komeili, A., E. K. O&#226;&#8364;&#8482;Shea. 1999. Roles phosphorylation sites in <br />       mannitol. FEMS Yeast Res. 3:17&#226;&#8364;&#8220;25.                                                      regulating activity transcription factor Pho4. Science 284:977&#226;&#8364;&#8220;980. <br /> 16.   Crosby, J. A., J. B. Konopka, S. Fields. 2000. Constitutive activation     44.   Kuo, M.-H., E. Grayhack. 1994. A library yeast genomic MCM1 <br />       Saccharomyces cerevisiae transcriptional regulator Ste12p mutations           binding sites contains genes involved cell cycle control, cell wall and <br />       amino-terminus. Yeast 16:1365&#226;&#8364;&#8220;1375.                                                 membrane structure, metabolism. Mol. Cell. Biol. 14:348&#226;&#8364;&#8220;359. <br /> 17.   Czech, M. P. 2000. PIP2 PIP3: complex roles cell surface. Cell         45.   Kwast, K. E., L.-C. Lai, N. Menda, D. T. James III, S. Aref, P. V. Burke. <br />       100:603&#226;&#8364;&#8220;606.                                                                            2002. Genomic analyses anaerobically induced genes Saccharomyces <br /> 18.   Deguchi, A., M. Imoto, K. Umezawa. 1996. Inhibition G1 cyclin                    cerevisiae: functional roles Rox1 factors mediating anoxic <br />       expression normal rat kidney cells inostamycin, phosphatidylinositol            response. J. Bacteriol. 184:250&#226;&#8364;&#8220;265. <br />       synthesis inhibitor. J. Biochem. (Tokyo) 120:1118&#226;&#8364;&#8220;1122.                           46.   Lamping, E., J. Lu   &#194;&#168;ckl, F. Paltauf, S. A. Henry, S. Kohlwein. 1995. <br /> 19.   Deguchi, A., K. Segawa, K. Hosaka, I. B. Weinstein, K. Umezawa. 2002.               Isolation characterization mutant Saccharomyces cerevisiae with <br />       Overexpression phosphatidylinositol synthase enhances growth G1                  pleiotropic deficiencies transcriptional activation repression. Genetics <br />       progression NIH3T3 cells. Jpn. J. Cancer Res. 93:157&#226;&#8364;&#8220;166.                            137:55&#226;&#8364;&#8220;65. <br /> 20.   Distel, B., R. Erdmann, S. J. Gould, G. Blobel, D. I. Crane, J. M. Cregg, G.      47.   Lee, T. I., N. J. Rinaldi, F. Robert, D. T. Odom, Z. Bar-Joseph, G. K. Gerber, <br />       Dodt, Y. Fujiki, J. M. Goodman, W. W. Just, et al. 1996. A unified nomen-               N. M. Hannett, C. T. Harbison, C. M. Thompson, I. Simon, et al. 2002. <br />       clature peroxisome biogenesis factors. J. Cell Biol. 135:1&#226;&#8364;&#8220;3.                       Transcriptional regulatory networks Saccharomyces cerevisiae. Science <br /> 21.   Divecha, N., R. F. Irvine. 1995. Phospholipid signaling. Cell 80:269&#226;&#8364;&#8220;278.           298:799&#226;&#8364;&#8220;804. <br /> 22.   Elkhaimi, M., M. R. Kaadige, D. Kamath, J. C. Jackson, J. H. C. Biliran,      48.   Lopes, J. M., S. A. Henry. 1991. Interaction trans cis regulatory <br />       J. M. Lopes. 2000. Combinatorial regulation phospholipid biosynthetic                elements INO1 promoter Saccharomyces cerevisiae. Nucleic Acids <br />       gene expression UME6, SIN3, RPD3 genes. Nucleic Acids Res.                   Res. 19:3987&#226;&#8364;&#8220;3994. <br />       28:3160&#226;&#8364;&#8220;3167.                                                                     49.   Luche, R. M., R. Sumrada, T. G. Cooper. 1990. A cis-acting element <br /> 23.   Fischl, A. S., G. M. Carman. 1983. Phosphatidylinositol biosynthesis             present multiple genes serves repressor protein binding site the <br />       Saccharomyces cerevisiae: purification properties microsome-associ-              yeast CAR1 gene. Mol. Cell. Biol. 10:3884&#226;&#8364;&#8220;3895. <br />       ated phosphatidylinositol synthase. J. Bacteriol. 154:304&#226;&#8364;&#8220;311.                    50.   Lykidis, A., P. D. Jackson, C. O. Rock, S. Jackowski. 1997. The role of <br /> 24.   Fischl, A. S., M. J. Homann, M. A. Poole, G. M. Carman. 1986. Phos-                 CDP-diacylglycerol synthetase phosphatidylinositol synthase activity lev- <br />       phatidylinositol synthase Saccharomyces cerevisiae. J. Biol. Chem. 261:            els regulation cellular phosphatidylinositol content. J. Biol. Chem. <br />       3178&#226;&#8364;&#8220;3183.                                                                              272:33402&#226;&#8364;&#8220;33409. <br /> 25.   Gardocki, M. E., J. M. Lopes. 2003. Expression yeast PIS1 gene         51.   McAndrew, P. C., J. Svaren, S. R. Martin, W. Ho&#194;&#168;rz, C. R. Goding. 1998. <br />       requires multiple regulatory elements including Rox1p binding site. J. Biol.          Requirements chromatin modulation transcription activation the <br />       Chem. 278:38646&#226;&#8364;&#8220;38652.                                                                  Pho4 acidic activation domain. Mol. Cell. Biol. 18:5818&#226;&#8364;&#8220;5827. <br /> 26.   Grauslund, M., J. M. Lopes, B. R&#195;&#184;nnow. 1999. Expression GUT1,              52.   Myers, A. M., A. Tzagoloff, D. M. Kinney, C. J. Lusty. 1986. Yeast shuttle <br />       encodes glycerol kinase Saccharomyces cerevisiae, controlled             integrative vectors multiple cloning sites suitable construction of <br />       positive regulators Adr1p, Ino2p, Ino4p, negative regulator             lacZ fusions. Gene 45:299&#226;&#8364;&#8220;310. <br />       Opi1p carbon source-dependent fashion. Nucleic Acids Res. 27:4391&#226;&#8364;&#8220;           53.   Nikawa, J.-I., T. Kodaki, S. Yamashita. 1988. Expression Saccha- <br />       4398.                                                                                   romyces cerevisiae PIS gene synthesis phosphatidylinositol Esche- <br /> 27.   Grauslund, M., B. R&#195;&#184;nnow. 2000. Carbon source-dependent transcrip-                  richia coli. J. Bacteriol. 170:4727&#226;&#8364;&#8220;4731. <br />       tional regulation mitochondrial glycerol-3-phosphate dehydrogenase         54.   Nikawa, J.-I., T. Kodaki, S. Yamashita. 1987. Primary structure and <br />       gene, GUT2, Saccharomyces cerevisiae. Can. J. Microbiol. 46:1096&#226;&#8364;&#8220;                  disruption phosphatidylinositol synthase gene Saccharomyces cer- <br />       1100.                                                                                   evisiae. J. Biol. Chem. 262:4876&#226;&#8364;&#8220;4881. <br /> 28.   Greenberg, M. L., J. M. Lopes. 1996. Genetic regulation phospholipid       55.   Nikawa, J.-I., S. Yamashita. 1984. Molecular cloning gene en- <br />       biosynthesis Saccharomyces cerevisiae. Microbiol. Rev. 60:1&#226;&#8364;&#8220;20.                      coding CDPdiacylglycerol-inositol 3-phosphatidyltransferase Saccharomy- <br /> 29.   Griggs, D. W., M. Johnston. 1993. Promoter elements determining weak                ces cerevisiae. Eur. J. Biochem. 143:251&#226;&#8364;&#8220;256. <br />       expression GAL4 regulatory gene Saccharomyces cerevisiae. Mol.          56.   Nikawa, J.-I., S. Yamashita. 1997. Phosphatidylinositol synthase from <br />       Cell. Biol. 13:4999&#226;&#8364;&#8220;5009.                                                               yeast. Biochim. Biophys. Acta 1348:173&#226;&#8364;&#8220;178. <br /> 30.   Griggs, D. W., M. Johnston. 1991. Regulated expression GAL4            57.   Odom, A. R., A. Stahlberg, S. R. Wente, J. D. York. 2000. A role for <br />       activator gene yeast provides sensitive genetic switch glucose repres-         nuclear inositol 1,4,5-triphosphate kinase transcriptional control. Science <br />       sion. Proc. Natl. Acad. Sci. USA 88:8597&#226;&#8364;&#8220;8601.                                          287:2026&#226;&#8364;&#8220;2029. <br /> 31.   Gru&#194;&#168;nweller, A., A. E. Ehrenhofer-Murray. 2002. A novel yeast silencer:       58.   Ohanian, J., V. Ohanian. 2001. Sphingolipids mammalian cell signal- <br />       2&#226;?&#174; origin Saccharomyces cerevisiae HST3-, MIG1- SIR-depen-               ling. Cell. Mol. Life Sci. 58:2053&#226;&#8364;&#8220;2068. <br />       dent silencing activity. Genetics 162:59&#226;&#8364;&#8220;71.                                      59.   Overkamp, K. M., B. M. Bakker, P. Kotter, A. van Tuijl, S. Vries, J. P. van <br /> 32.   Guarente, L., T. Mason. 1983. Heme regulates transcription                   Dijken, J. T. Pronk. 2000. In vivo analysis mechanisms for <br />       CYC1 gene S. cerevisiae upstream activation site. Cell 32:1279&#226;&#8364;&#8220;1286.          oxidation cytosolic NADH Saccharomyces cerevisiae mitochondria. J. <br /> 33.   Henry, S. A., J. L. Patton-Vogt. 1998. Genetic regulation phospholipid           Bacteriol. 180:2823&#226;&#8364;&#8220;2830. <br />       metabolism: yeast model eukaryote. Prog. Nucleic Acid Res. Mol. Biol.        60.   Purdue, P. E., P. B. Lazarow. 2001. Peroxisome biogenesis. Annu. Rev. <br />       61:133&#226;&#8364;&#8220;179.                                                                             Cell Dev. Biol. 17:701&#226;&#8364;&#8220;752. <br /> 34.   Herrick, D., R. Parker, A. Jacobson. 1990. Identification comparison      61.   Remize, F., B. Cambon, L. Barnavon, S. Dequin. 2003. Glycerol forma- <br />       stable unstable mRNAs Saccharomyces cerevisiae. Mol. Cell. Biol.              tion wine fermentation mainly linked Gpd1p partly <br />       10:2269&#226;&#8364;&#8220;2284.                                                                           controlled HOG pathway. Yeast 20:1243&#226;&#8364;&#8220;1253. <br /> 35.   Holstege, F. C. P., E. G. Jennings, J. J. Wyrick, T. I. Lee, C. J. Hengartner,    62.   Ren, B., F. Robert, J. J. Wyrick, O. Aparicio, E. G. Jennings, I. Simon, J. <br />       M. R. Green, T. R. Golub, E. S. Lander, R. A. Young. 1998. Dissecting               Zeitlinger, J. Schreiber, N. Hannett, E. Kanin, et al. 2000. Genome-wide <br />       regulatory circuitry eukaryotic genome. Cell 95:717&#226;&#8364;&#8220;728.                       location function DNA binding proteins. Science 290:2306&#226;&#8364;&#8220;2309. <br /> 36.   Imoto, M., T. Morii, A. Deguchi, K. Umezawa. 1994. Involvement             63.   Robinson, K. A., J. I. Koepke, M. Kharodawala, J. M. Lopes. 2000. A <br />  614         GARDOCKI ET AL.                                                                                                                          EUKARYOT. CELL <br />  <br />       network yeast basic helix-loop-helix interactions. Nucleic Acids Res. 28:     74. Terlecky, S. R., M. Fransen. 2000. How peroxisomes arise. Traffic <br />       4460&#226;&#8364;&#8220;4466.                                                                           1:465&#226;&#8364;&#8220;473. <br /> 64.   Saiardi, A., J. J. Caffrey, S. H. Snyder, S. B. Shears. 2000. The inositol   75. Ter Linde, J. J. M., H. Y. De Steensma. 2002. A microarray-assisted <br />       hexakisphosphate kinase family. J. Biol. Chem. 275:24686&#226;&#8364;&#8220;24692.                      screen potential Hap1 Rox1 target genes Saccharomyces cerevisiae. <br /> 65.   Saiardi, A., J. J. Caffrey, S. H. Snyder, S. B. Shears. 2000. Inositol           Yeast 19:825&#226;&#8364;&#8220;840. <br />       polyphosphate multikinase (ArgRIII) determines nuclear mRNA export            76. Ter Linde, J. J. M., H. Liang, R. W. Davis, H. Y. De Steensma, J. P. Van <br />       Saccharomyces cerevisiae. FEBS Lett. 468:28&#226;&#8364;&#8220;32.                                      Dijken, J. T. Pronk. 1999. Genome-wide transcriptional analysis of <br /> 66.   Saiardi, A., H. Erdjument-Bromage, A. M. Snowman, P. Tempst, S. H.               aerobic anaerobic chemostat cultures Saccharomyces cerevisiae. J. <br />       Snyder. 1999. Synthesis diphosphoinositol pentakisphosphate newly            Bacteriol. 181:7409&#226;&#8364;&#8220;7413. <br />       identified family higher inositol polyphosphate kinases. Curr. Biol.          77. Titorenko, V. I., R. A. Rachubinski. 2001. The life cycle peroxi- <br />       9:1323&#226;&#8364;&#8220;1326.                                                                         some. Nat. Rev. Mol. Cell Biol. 2:357&#226;&#8364;&#8220;368. <br /> 67.   Shao, D., C. L. Creasy, L. W. Bergman. 1998. A cysteine residue           78. van den Bosch, H., R. B. H. Schutgens, R. J. A. Wanders, J. M. Tager. <br />       helixII bHLH domain essential homodimerization yeast            1992. Biochemistry peroxisomes. Annu. Rev. Biochem. 61:157&#226;&#8364;&#8220;197. <br />       transcription factor Pho4p. Nucleic Acids Res. 26:710&#226;&#8364;&#8220;714.                       79. van Roermund, C. W. T., H. R. Waterham, L. Ijlst, R. J. A. Wanders. <br />                                                                                            2003. Fatty acid metabolism Saccharomyces cerevisiae. Cell. Mol. Life Sci. <br /> 68.   Shao, D., C. L. Creasy, L. W. Bergman. 1996. Interaction Saccharo- <br />                                                                                            60:1838&#226;&#8364;&#8220;1851. <br />       myces cerevisiae Pho2 Pho4 increases accessibility activation <br />                                                                                        80. Vizeacoumar, F. J., J. C. Torres-Guzman, D. Bouard, J. D. Aitchison, and <br />       domain Pho4. Mol. Gen. Genet. 251:358&#226;&#8364;&#8220;364. <br />                                                                                            R. A. Rachubinski. 2004. Pex30p, Pex31p, Pex32p form family of <br /> 69.   Shields, D., P. Arvan. 1999. Disease models provide insights post-          peroxisomal integral membrane proteins regulating peroxisome size and <br />       Golgi protein trafficking, localization processing. Curr. Opin. Cell Biol.       number Saccharomyces cerevisiae. Mol. Biol. Cell 15:665&#226;&#8364;&#8220;677. <br />       11:489&#226;&#8364;&#8220;494.                                                                      81. Vizeacoumar, F. J., J. C. Torres-Guzman, Y. Y. C. Tam, J. D. Aitchison, and <br /> 70.   Smith, J. J., M. Marelli, R. H. Christmas, F. J. Vizeacoumar, D. J. Dilworth,        R. A. Rachubinski. 2003. YHR150w YDR479c encode peroxisomal <br />       T. Ideker, T. Galitski, K. Dimitrov, R. A. Rachubinski, J. D. Aitchison.         integral membrane proteins involved regulation peroxisome num- <br />       2002. Transcriptome profiling identify genes involved peroxisome as-           ber, size, distribution Saccharomyces cerevisiae. J. Cell Biol. 161:321&#226;&#8364;&#8220; <br />       sembly function. J. Cell Biol. 158:259&#226;&#8364;&#8220;271.                                      332. <br /> 71.   Starai, V. J., H. Takahashi, J. D. Boeke, J. C. Escalante-Semerena. 2003.    82. Vogel, K., W. Ho&#194;&#168;rz, A. Hinnen. 1989. The positively acting regulatory <br />       Short-chain fatty acid activation acyl-coenzyme A synthetases requires            proteins PHO2 PHO4 physically interact PHO5 upstream activa- <br />       SIR2 protein function Salmonella enterica Saccharomyces cerevisiae.           tion regions. Mol. <br /> </body></html>