<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2004 Genetics Society America <br />  <br />  <br />  <br />            Caenorhabditis elegans OSR-1 Regulates Behavioral Physiological <br />                         Responses Hyperosmotic Environments <br />  <br />             Aharon Solomon, Sricharan Bandhakavi, Sean Jabbar, Rena Shah, Greg J. Beitel <br />                                      Richard I. Morimoto1 <br />                     Department Biochemistry, Molecular Biology Cell Biology, Rice Institute Biomedical Research, <br />                                              Northwestern University, Evanston, Illinois 60208 <br />                                                        Manuscript received October 26, 2003 <br />                                                       Accepted publication January 29, 2004 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 The molecular mechanisms enable multicellular organisms sense modulate responses <br />              hyperosmotic environments poorly understood. Here, employ Caenorhabditis elegans characterize <br />              response multicellular organism osmotic stress establish genetic screen isolate mutants <br />              osmotic stress resistant (OSR). In study, cloning novel gene, osr-1, and <br />              demonstrate regulates osmosensation, adaptation, <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> hyperosmotic environments. <br />              Whereas wild-type animals exposed hyperosmotic conditions rapidly lose body volume, motility, and <br />              viability, osr-1(rm1) mutant animals maintain normal body volume, motility, viability chronic <br />              exposures high osmolarity environments. In addition, osr-1(rm1) animals specifically resistant <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />              osmotic stress distinct previously characterized osmotic avoidance defective (OSM) and <br />              general stress resistance age-1(hx546) mutants. OSR-1 expressed hypodermis intestine, and <br />              expression OSR-1 hypodermal cells rescues osr-1(rm1) phenotypes. Genetic epistasis analysis <br />              indicates OSR-1 regulates survival osmotic stress CaMKII conserved p38 MAP kinase <br />              signaling cascade regulates osmotic avoidance resistance acute dehydration likely distinct <br />              mechanisms. We suggest OSR-1 plays central role integrating stress detection adaptation <br />              responses invoking multiple signaling pathways promote survival hyperosmotic environments. <br />  <br />  <br />  <br /> W      ATER loss, conditions high salinity and <br />        drought, affects growth development mul- <br /> tiple organisms natural habitats. Cellular struc- <br />                                                                              myces cerevisiae. In yeast, perturbations osmotic <br />                                                                              environment detected two-component system <br />                                                                              (SLN1-YPD1-SSK1; Maeda et al. 1994; Posas et al. 1996) <br /> ture protein function highly dependent                          sensors, SHO1 (Maeda et al. 1995) and <br /> aqueous environments, imbalance water ho-                          MSB2 (O&#226;&#8364;&#8482;Rourke Herskowitz 2002; O&#226;&#8364;&#8482;Rourke et <br /> meostasis cause irreversible damage detrimen-                       al. 2002), regulate activation osmoadaptive <br /> tal effects organisms. To protect                      HOG/p38 pathway (Brewster et al. 1993) glycerol <br /> dehydration, diverse organisms (e.g., bacteria, fungi, tar-                  biosynthesis (Hohmann 2002). <br /> digrades, rotifers, nematodes, desert resurrection                          In contrast unicellular organisms yeast, <br /> plants) developed survival strategies detect                     known sensors, signal transduction <br /> withstand cellular dehydration hyperosmotic environ-                      pathways, mechanisms constitute response <br /> ments (Potts 1994; Clegg 2001; Crowe 2002). The                              high osmolarity multicellular organisms. In mam- <br /> synthesis accumulation compatible osmolytes                           mals, osmotic stress results activation adaptive <br /> (glycerol, trehalose, sucrose, myo-inositol) uni-                   response involves different mitogen-activated <br /> versal osmoadaptive response diverse organisms                         protein (MAP) kinases (SAPKs), p38, JNK, ERK5 <br /> cellular dehydration (Yancey et al. 1982) sug-                        (de Nadal et al. 2002). A novel osmosensing scaffold <br /> gested stabilize membranes native proteins                        protein recently discovered shown acti- <br /> regulate cell volume (Crowe et al. 1984; Welch                           vate mammalian p38 MAPK pathway osmotic <br /> Brown 1996).                                                                 stress conditions, manner strikingly similar that <br />    The eukaryotic response osmotic stress                        seen yeast (Uhlik et al. 2003). Yet osmo- <br /> extensively investigated budding yeast, Saccharo-                     sensor VR-OAC receptor (TRPV4), been <br />                                                                              suggested role sensing systemic osmotic <br />                                                                              hydrostatic pressure (Liedtke et al. 2000). <br />   Sequence data article deposited                  The nematode Caenorhabditis elegans emerged as <br /> EMBL/GenBank Data Libraries accession no. AY360470.                    powerful genetic model studying osmosensation <br />   1 <br />    Corresponding author: Department Biochemistry, Molecular Biol-         multicellular organisms (Culotti Russell 1978; <br /> ogy, Cell Biology, Northwestern University, 2205 Tech Dr., Hogan <br /> 2-100, Evanston, IL 60208-3500.                                              Bargmann et al. 1990). In addition adaptive re- <br /> E-mail: r-morimoto@northwestern.edu                                          sponses (as described mammals yeast), <br /> Genetics 167: 161&#226;&#8364;&#8220;170 (May 2004) <br />  162                                                          A. Solomon et al. <br />  <br /> C. elegans worms developed mechanisms                         Molecular biology: The functional osr-1::gfp operon con- <br /> which avoid hyperosmotic environments. Mutants                     struct tissue-specific expression constructs gener- <br />                                                                         ated using Gateway (Invitrogen, Carlsbad, CA; Wal- <br /> defective sensing high osmolarity (OSM)                    hout et al. 2000). The rescuing fragment (pASRM1) the <br /> described (Culotti Russell 1978; Colbert et al.                     heterologous promoter::osr-1 cDNA inserted pEntry <br /> 1997; Hart et al. 1999). Unlike wild-type animals, OSM                  vector containing SL2-gfp operon cassette. The promoters <br /> mutants fail avoid environments high osmolarity,                  used dpy-7 (hypodermal), vha-6 (intestine; Wang et al. <br /> due partly defective function polymodal              2002), F25B3.3 (pan-neuronal; Altun-Gultekin et al. <br />                                                                         2001). The transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> osr-1 determined <br /> osmosensory ASH neurons (Colbert et al. 1997; Hart                      using partial expressed sequence tag (EST; yk563c9) ob- <br /> et al. 1999). However, entire complement molecu-                 tained Yuji Kohara SMART RACE cDNA kit <br /> lar mechanisms multicellular organisms inte-                   (CLONTECH, Palo Alto, CA). Further details plasmid con- <br /> grate responses lead detection, adaptation,                 struction available request. <br /> survival osmotic stress remain poorly understood.                    RNAi experiments: RNAi experiments performed us- <br />                                                                         ing previously described feeding method (Timmons et al. <br />    To address issues, employed C. elegans                 2001). For osr-1 RNAi, pAS1 constructed subcloning <br /> as model characterized response                     osr-1 cDNA (1932 bp) L4440 vector. For pmk-1 <br /> hyperosmotic environments. Here report                        RNAi, used pDK177 plasmid (Kim et al. 2002). Eggs <br /> cloning characterization novel gene, osr-1,                hatched E. coli (HT115) carrying L4440 expression <br /> show C. elegans, OSR-1 regulates multiple aspects               vector (control groups), HT115 carrying pDK177 plasmid <br />                                                                         (pmk-1), plasmid pAS1 (osr-1) allowed grow for <br /> of osmotic stress physiology distinct genetic path-             3 days (adult stage). For knockdown pmk-1 osr-1, <br /> ways promote survival hyperosmotic environ-                    eggs hatched plates containing equal amounts of <br /> ments.                                                                  bacteria expressing dsRNA pmk-1 osr-1. Animals <br />                                                                         grown RNAi plates exposed 500 mm <br />                                                                         NaCl 24 hr scored viability mentioned above. <br />                                                                         Efficiency feeding RNAi confirmed using one- <br />                MATERIALS AND METHODS <br />                                                                         step RT-PCR kit (Invitrogen). <br />    Strains culture conditions: C. elegans worms cultured <br /> at 22&#226;&#172;&#353; standard growth conditions (Brenner 1974). Strains <br /> used study Bristol strain N2 (wild-type), osr-1                                      RESULTS <br /> (rm1), unc-73(e936);dpy-5(e61), age-1(hx546), daf-16(mgdf50), osm-9 <br /> (ky10), ocr-2(ak47), osm-10(n1052), unc-43(n1186), sek-1(km4), nsy-1       To establish metazoan model osmotic stress <br /> (ky397), pmk-3(ok169), jnk-1(jk7), jkk-1(km2), mek-1(ks54).         signaling, characterized organismal response of <br />    Genetic screens osmotic stress resistant mutants: Stan-          C. elegans hyperosmotic stress. When wild-type (N2 <br /> dard EMS mutagenesis (Brenner 1974) performed N2 <br /> strain hermaphrodites. The F2 generation (&#226;&#181;&#8216;20,000 haploid               strain) animals exposed acute osmotic stress (10 <br /> genomes) screened mutants maintained normal                min, 500 mm NaCl), observed complete loss of <br /> swimming behavior NGM agar plates containing 500 mm                  motility reduction body volume worms <br /> NaCl (high-salt plates) &#226;&#172;&#381;15 min (acute motility assay).             (Figures 1, A&#226;&#8364;&#8220;D, 2A). Remarkably, transfer <br /> Putative mutants showed normal motility assay <br />                                                                         normal growth medium (50 mm NaCl), animals <br /> were isolated analysis. <br />    Mapping cloning osr-1: Standard mapping proce-                recovered original body size regained normal <br /> dures (Brenner 1974) used map osr-1 chromosome               motility minutes (Figure 1, E&#226;&#8364;&#8220;G). However, pro- <br /> I. We used three-factor mapping identify osr-1 &#226;&#171;&#186;0.2 map           longed exposure N2 worms high salt concentra- <br /> units (MU) dpy-5 unc-73. Individual cosmids            tions severely affects motility viability. After 5 hr <br /> the identified genetic interval injected osr-1(rm1) <br /> animals 40 ng/&#226;?&#174;l 100 ng/&#226;?&#174;l myo-2::gfp plasmid a <br />                                                                         exposure 500 mm NaCl, N2 animals were <br /> co-injection marker. Rescue experiments performed             immotile motile (43%; Figure 2B), <br /> at independent transgenic lines.                            exhibited sluggish swimming behavior. This pro- <br />    Behavioral stress resistance assays: Worms placed           longed exposure high osmotic stress eventually <br /> on high-salt plates containing 500 mm NaCl scored               lethal 24 hr 87% wild-type animals died <br /> their motility period 10 min (acute motility assay) <br /> and 1, 5, 12 hr (chronic adaptation assay). Osmotic           (Figure 2C). <br /> stress survival assays performed high-salt plates              To identify genes mediating osmotic stress re- <br /> seeded Escherichia coli (OP50). To score viability,            sponses C. elegans, designed genetic screen to <br /> worms collected salt plates using recovery              isolate mutants osmotic stress resistant (OSR) <br /> buffer (300 mm NaCl M9), transferred regular NGM                  500 mm NaCl. Mutants maintained normal motil- <br /> plates (50 mm NaCl), allowed recover overnight before <br /> scoring viability. Osmotic avoidance behavior (OSM)             ity high-salt plates 15 min isolated for <br /> quantified percentage worms crossed 2-cm               analysis. Among 104 isolated mutant strains, <br /> ring 4 m NaCl 8 m fructose NGM agar plate,           identified different complementation <br /> 5 min. Nose touch response, 1-octanol repellent assays (Hart            groups: osr-1, osr-2, osr-3, osr-4. In study report <br /> et al. 1999), heat (Lithgow et al. 1995) oxidative stress       cloning characterization osr-1. <br /> (Lee et al. 2003) experiments performed described <br /> previously. Statistical significance mutants be-            osr-1(rm1) mutants specifically resistant osmotic <br /> havioral stress assays determined using two-tailed          stress: We osr-1(rm1) animals en- <br /> t-test one-way ANOVA test.                                       hanced ability resist hyperosmotic stress (Figures 1 <br />                                           OSR-1 Osmotic Stress C. elegans                                        163 <br />  <br />  <br />                                                                                            Figure 1.&#226;&#8364;&#8221;Characterization of <br />                                                                                         C. elegans osmotic stress re- <br />                                                                                         sponse. (A) Wild-type (N2 strain) <br />                                                                                         worm isotonic medium (50 mm <br />                                                                                         NaCl). (B&#226;&#8364;&#8220;D) Dehydration an <br />                                                                                         N2 worm 10 min os- <br />                                                                                         motic stress. (E&#226;&#8364;&#8220;G) Recovery a <br />                                                                                         shrunken N2 worm 10 min <br />                                                                                         isotonic medium. (H&#226;&#8364;&#8220;K) osr-1 <br />                                                                                         (rm1) worm acute osmotic <br />                                                                                         stress. (L) Time course percent- <br />                                                                                         age motile N2 (&#228;&#8240;&#171;) osr- <br />                                                                                         1(rm1) animals (&#228;&#352;?) osmotic <br />                                                                                         stress (N &#226;&#172;&#381; 500 scored each <br />                                                                                         strain). Arrow indicates time <br />                                                                                         point inactive N2 <br />                                                                                         worms shifted isotonic me- <br />                                                                                         dium osr-1(rm1) animals re- <br />                                                                                         mained salt stress. The <br />                                                                                         N2 osr-1(rm1) animals shown <br />                                                                                         figure re- <br />                                                                                         spective animal different time <br />                                                                                         points. Bar, 0.2 mm. <br />  <br />  <br />  <br />  <br /> and 2). In contrast N2 animals, osr-1(rm1) animals         preadapted acute osmotic stress like osr-1(rm1) <br /> exhibit normal swimming behavior (Figure 1, H&#226;&#8364;&#8220;K)              animals (Figure 2A). In addition, contrast osr-1 <br /> when exposed transient osmotic stress (10 min, 500       (rm1) animals, age-1(hx546) animals wild-type phe- <br /> mm NaCl) remain viable (88%) 24 hr             notype osmotic avoidance (Figure 2F). Interestingly, <br /> (Figure 2C). osr-1(rm1) animals resistant os-     age-1(hx546) animals exhibited motility viability sim- <br /> motic stress caused sorbitol, glucose, sucrose,        ilar osr-1(rm1) animals prolonged <br /> KCl, demonstrating osr-1(rm1) animals gen-        exposures high osmotic environments (Figure 2, B <br /> eral resistance hyperosmotic stress (data shown).      C). These results indicate AGE-1 does <br />    To determine osr-1(rm1) <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>mutation</span> spe-       regulate osmotic avoidance resistance acute <br /> cific osmotic stress confers resistance multiple     osmotic stress, OSR-1 AGE-1 regulate survival <br /> stresses, observed long-lived mutants        conditions prolonged hyperosmotic stress. <br /> insulin-like signaling (ILS) pathway (Kenyon et al. 1993;         osr-1(rm1) animals display osmotic avoidance ab- <br /> Larsen 1993; Dorman et al. 1995; Lithgow et al. 1995;         normality <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span>: The observed osmotolerance of <br /> Murakami Johnson 1996; Kimura et al. 1997; Tis-           osr-1(rm1) animals led investigate OSR-1 <br /> senbaum Ruvkun 1998), tested ability           involved detection high osmolarity environments <br /> of osr-1(rm1) animals survive heat-shock (35&#226;&#172;&#353;) oxi-     described previously osmotic avoidance defec- <br /> dative stress (300 mm paraquat; Figure 2, D E). In        tive (OSM) mutants (Culotti Russell 1978; Barg- <br /> contrast age-1(hx546) animals, osr-1(rm1) animals     mann et al. 1990). osr-1(rm1) animals exhibited os- <br /> sensitive stresses (P &#226;&#172;? 0.05). Moreover, osr-         motic avoidance abnormality phenotype fail to <br /> 1(rm1) animals did exhibit phenotypes associ-       avoid regions 4 m NaCl 8 m fructose (Figure 2F <br /> ated general stress resistance seen ILS      data shown). The osmotic avoidance defect of <br /> pathway mutants, extended life span constitu-      osr-1(rm1) animals quantitatively similar seen <br /> tive dauer formation (data shown). Thus, osr-1(rm1)       previously characterized osm mutants, osm- <br /> animals new class mutants resistant         9(ky10) (Colbert et al. 1997), osm-10(n1052) (Hart et <br /> specifically osmotic stress.                               al. 1999) ocr-2(ak47) (Tobin et al. 2002). <br />    OSR-1 AGE-1 critical roles resistance           Osmosensation C. elegans dependent in- <br /> chronic osmotic stress: We explored possibility      tegrity osmosensory organs (amphids phasmids) <br /> the general stress resistant mutant, age-1(hx546),      function ASH neurons; defective osmotic <br /> be resistant osmotic stress. However, shown        avoidance phenotype osm-9(ky10), osm-10(n1052), and <br /> in Figure 2A, exposure age-1(hx546) animals 500         ocr-2(ak47) ascribed impaired function of <br /> mm NaCl caused complete loss motility reduc-         ASH neurons (Colbert et al. 1997; Hart et al. <br /> tion body volume 11 min, similar wild-type       1999; Tobin et al. 2002). On basis lipophilic dye <br /> animals. This demonstrates age-1(hx546) animals          staining, detect morphological abnor- <br />  164                                                  A. Solomon et al. <br />  <br />  <br />  <br />  <br />                                                                                           Figure 2.&#226;&#8364;&#8221;osr-1(rm1) worms are <br />                                                                                        specifically resistant osmotic stress. <br />                                                                                        (A B) Motility assays wild-type <br />                                                                                        (N2 strain; blue diamond blue <br />                                                                                        bar), osr-1(rm1) (red square red <br />                                                                                        bar), age-1(hx546) (black triangle <br />                                                                                        black bar) animals following (A) <br />                                                                                        acute (B) chronic exposure to <br />                                                                                        500 mm NaCl. (C) Survival N2 <br />                                                                                        (blue diamond), osr-1(rm1) (red <br />                                                                                        square), age-1(hx546) (black tri- <br />                                                                                        angle) animals 3 days expo- <br />                                                                                        sure 500 mm NaCl. (D E) Sur- <br />                                                                                        vival N2 (blue diamond), osr-1(rm1) <br />                                                                                        (red square), age-1(hx546) (black <br />                                                                                        triangle) animals exposed either <br />                                                                                        (D) heat shock (35&#226;&#172;&#353;) (E) oxidative <br />                                                                                        stress (300 mm paraquat). (F) Os- <br />                                                                                        motic avoidance assay N2 (blue di- <br />                                                                                        amond), osr-1(rm1) (red square), age-1 <br />                                                                                        (hx546) (black triangle), osm-9(ky10) <br />                                                                                        (pink open circle), osm-10 <br />                                                                                        (n1052) (black solid circle) animals. <br />                                                                                        Error bars represent standard error <br />                                                                                        independent repli- <br />                                                                                        cates time point. Each repli- <br />                                                                                        cate contained 50&#226;&#8364;&#8220;100 adult her- <br />                                                                                        maphrodite animals tested. <br />  <br />  <br />  <br />  <br /> malities osmosensory organs (amphids phas-           osr-1(rm1) animals independent pathways <br /> mids) osr-1(rm1) animals (A. Solomon R. Mori-            regulated OSM genes tested us. <br /> moto, unpublished results). In addition, osr-1(rm1)                Positional cloning, sequence analysis, expression <br /> animals normal responses nose touch             patterns osr-1: osr-1 cloned using standard two- <br /> repellent, 1-octanol (data shown),           three-factor mapping transformation rescue <br /> mediated polymodal ASH neurons distinct              approaches (Figure 3A). Two cosmids, C32E12 and <br /> signaling pathways (Kaplan Horvitz 1993; Hart               F56A3, overlapping region 11 kb gave full <br /> et al. 1999). Although rule ASH          rescue osr-1(rm1) phenotypes. In overlapping <br /> neurons impaired osmosensation osr-              region, identified 4-kb minimal rescuing frag- <br /> 1(rm1) animals, results demonstrate       ment (pASRM1) containing single predicted gene, <br /> mechanosensory chemosensory responses medi-                 C32E12.3, completely rescued osr-1(rm1) phe- <br /> ated neurons normal animals.           notypes. We osr-1(rm1) worms G-to-A muta- <br />    We that, unlike osr-1(rm1) animals,          tion predicted splice acceptor site transcrip- <br /> other osmotic avoidance defective mutants, osm-9(ky10),         tion <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> C32E12.3 (Figure 3B). Introduction this <br /> ocr-2(ak47), osm-10(n1052), resistant     mutation pASRM1 completely eliminated ability to <br /> acute chronic osmotic stress. Also,            rescue osr-1(rm1) animals&#226;&#8364;&#8482; phenotypes. In addition, <br /> mutations affect osmotic stress tolerance OSR-1 defi-        RNAi osr-1 wild-type animals results osr-1(rm1)- <br /> cient animals (data shown). This demonstrates          like phenotypes. We conclude osr-1 gene corre- <br /> the osmosensory pathway defined OSM-9, OCR-2,                sponds C32E12.3. <br /> and OSM-10 does mediate acute chronic resis-                The osr-1 gene encodes predicted protein 643 <br /> tance osmotic stress, hyperresistance           amino acids (Figure 3C) lacks domains that <br />                                          OSR-1 Osmotic Stress C. elegans                                              165 <br />  <br />  <br />  <br />  <br />                                                                              Figure 3.&#226;&#8364;&#8221;Positional cloning osr-1. (A) Ge- <br />                                                                           netic physical maps chromosome I. osr-1 <br />                                                                           mapped &#226;&#181;&#8216; &#226;&#171;&#186;0.2 MU cloned <br />                                                                           genes, unc-73 dpy-5. The numbers the <br />                                                                           bar represent recombination events oc- <br />                                                                           curred dpy-5 osr-1 (2/25) unc- <br />                                                                           73 osr-1 (23/25). Numbers parentheses <br />                                                                           fraction transgenic lines rescued <br />                                                                           osr-1(rm1) mutant animals. (B) The osr-1 transcrip- <br />                                                                           tion <span id='am-317' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. The transcription <span id='am-318' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> osr-1 was <br />                                                                           determined using partial EST (yk563c9) ob- <br />                                                                           tained Yuji Kohara using 5&#226;&#172;&#732; random <br />                                                                           amplification cDNA ends experiments. Since <br />                                                                           osr-1 mRNA did SL consensus splice <br />                                                                           sequences, introduced stop codon 5&#226;&#172;&#732; <br />                                                                           untranslated region immediately frame with <br />                                                                           putative ATG site rescuing con- <br />                                                                           struct pASRM1. Injection construct into <br />                                                                           osr-1(rm1) mutant animals gave rescue. (C) <br />                                                                           osr-1 encodes novel protein 643 amino acids <br />                                                                           predicted N-terminal signal peptide <br />                                                                           (underlined). <br />  <br />  <br />  <br />  <br /> would suggest specific molecular function. Proteins        ing hypodermal (Pdpy-7::osr-1::gfp), intestinal (Pvha-6::osr-1::gfp; <br /> with significant similarity OSR-1      Wang et al. 2002) pan-neuronal (PF25B3.3::osr-1::gfp) pro- <br /> the genomes C. briggsae (86% overall identity CBE      moters (Altun-Gultekin et al. 2001). We that <br /> 12837) parasitic nematode Strongyloides ratti        hypodermal expression osr-1 cDNA rescues the <br /> (48% identity partial EST kt82b03.y1).                    osr-1(rm1) phenotypes (Figure 4C). <br />    To identify tissues OSR-1 expressed          OSR-1 genetically interacts CaMKII con- <br /> and functions, placed rescuing fragment,              served p38 MAPK cascade: To understand mecha- <br /> pASRM1, artificial operon green fluorescent       nisms OSR-1 regulates osmotic stress responses <br /> protein (GFP). This construct fully rescued osr-         C. elegans, tested potential role(s) C. <br /> 1(rm1) phenotypes GFP fluorescence seen           elegans homologs HOG/p38 MAPK signaling pathway <br /> the hypodermis intestine developmental        (Brewster et al. 1993) mediating phenotypes seen in <br /> stages (Figure 4, A B). To test tissues       osr-1(rm1) animals. In C. elegans, multiple MAPK <br /> functional importance, performed rescue experi-           encoding genes implicated development resistance <br /> ments using heterologous promoter constructs includ-         abiotic biotic stresses (Kurz Ewbank 2003). <br />  166                                                   A. Solomon et al. <br />  <br />  <br />  <br />  <br />                                                                                   Figure 4.&#226;&#8364;&#8221;OSR-1 expressed hypo- <br />                                                                                dermis intestine. (A) L3 stage N2 ani- <br />                                                                                mals showing expression OSR-1 intes- <br />                                                                                tine hypodermis. (B) Section the <br />                                                                                midbody L3 stage animal. Arrows indi- <br />                                                                                cate hypodermis (H) intestine (I). (C) <br />                                                                                Tissue-specific expression osr-1 cDNA the <br />                                                                                hypodermis (Pdpy-7), intestine (Pvha-6), ner- <br />                                                                                vous (PF25B3.3). Expression osr-1 cDNA <br />                                                                                hypodermal promoter able to <br />                                                                                rescue osr-1(rm1) phenotypes. Error bars <br />                                                                                represent standard error indepen- <br />                                                                                dent replicates time point. Each repli- <br />                                                                                cate contained &#226;&#172;&#381;50 adult hermaphrodite ani- <br />                                                                                mals tested. <br />  <br />  <br />  <br />  <br /> The Esp pathway (NSY-1/MAPKKK &#226;&#8224;&#8217; SEK-1/MAPKK &#226;&#8224;&#8217;                    longevity (Murakami Johnson 1996; Ogg et <br /> PMK-1/p38/MAPK), shown function                   al. 1997; Henderson Johnson 2001), tested <br /> pathogen resistance (Kim et al. 2002), neuronal          effect osr-1 RNAi null mutant daf-16 <br /> symmetry (Nsy) pathway (UNC-43/CaMKII &#226;&#8224;&#8217; NSY-1 &#226;&#8224;&#8217;                  (Ogg et al. 1997). As shown Figure 5A, ability of <br /> SEK-1 &#226;&#8224;&#8217; unknown MAPK) mediates asymmetric neu-                   OSR-1 regulate survival hyperosmotic environ- <br /> ronal cell fate AWC sensory neurons (Sagasti et al.           ments dependent DAF-16. Taken together, <br /> 2001; Tanaka-Hino et al. 2002).                                  results indicate OSR-1 functions upstream of <br />    Downregulation pmk-1/p38 osr-1(rm1) worms            parallel UNC-43/CaMKII conserved p38 <br /> feeding RNAi significantly reduced ability sur-         pathway (NSY-1 &#226;&#8224;&#8217; SEK-1 &#226;&#8224;&#8217; PMK-1) promoting sur- <br /> vive chronic osmotic stress (Figure 5A; P &#226;&#172;? 0.05). Simi-         vival chronic osmotic stress (Figure 5B). <br /> larly, null mutations, NSY-1 SEK-1, completely               We investigated role PMK-1/p38 path- <br /> suppressed ability osr-1 deficient animals survive     way regulating resistance acute dehydration and <br /> osmotic stress (Figure 5A). These results led investi-     osmotic avoidance defects seen osr-1(rm1) animals. <br /> gate potential relationship OSR-1            Interestingly, osr-1(rm1), pmk-1(RNAi) (N &#226;&#171;&#189; <br /> most upstream component Nsy pathway, UNC-                 175); nsy-1(ky379), osr-1(RNAi) (N &#226;&#171;&#189; 1259); sek-1 <br /> 43. As shown Figure 5A, UNC-43, identified           (km4), osr-1(RNAi) (N &#226;&#171;&#189; 376) animals remained resis- <br /> CaMKII C. elegans (Reiner et al. 1999), re-          tant acute effects osmotic stress maintaining <br /> quired viability chronic osmotic stress.               normal swimming behavior (100% motile animals after <br />    The interactions OSR-1 PMK-1/p38 ap-              10 min). In addition, animals retain OSM-like <br /> pear specific OSR-1 does interact         phenotype, &#226;&#181;&#8216;57% animals cross 4 m <br /> two C. elegans p38 homologs (Berman et al. 2001),          NaCl ring (data shown), similar phenotypes <br /> PMK-2 PMK-3 (A. Solomon R. Morimoto,                     seen osm-9(ky10) osr-1(rm1) animals (P &#226;&#172;&#381; 0.05; <br /> unpublished results). Mutations genes mediate            Figure 2F). <br /> heavy metal resistance, including JNK-1, JKK-1 (Vil- <br /> lanueva et al. 2001), MEK-1 (Koga et al. 2000), <br />                                                                                        DISCUSSION <br /> also suppress OSR-1 phenotypes (Figure 5A <br /> and data shown). Because forkhead transcrip-               In study, used osmotic stress response <br /> tion factor, DAF-16, promotes general stress resistance          C. elegans model stress signaling multicel- <br />                                            OSR-1 Osmotic Stress C. elegans                                            167 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 5.&#226;&#8364;&#8221;OSR-1 genetically interacts with <br />                                                                               UNC-43/CaMKII conserved PMK-1/ <br />                                                                               p38 MAPK signaling pathway promote sur- <br />                                                                               vival osmotic stress. (A) Survival N2, osr- <br />                                                                               1(rm1), sek-1(km4), nsy-1(ky397), unc-43(n1186), <br />                                                                               daf-16(mgdf50), jnk-1(gk7) mutant animals <br />                                                                               500 mm NaCl 24 hr. Survival also <br />                                                                               tested designated mutants with- <br />                                                                               RNAi osr-1 pmk-1. Progeny each <br />                                                                               strain grown E. coli (HT115) carrying <br />                                                                               L4440 expression vector (solid blue <br />                                                                               bar; control group), HT115 carrying plas- <br />                                                                               mid pDK177 (for pmk-1), plasmid pAS1 <br />                                                                               (for osr-1), producing double-strand RNA <br />                                                                               (striped blue bar). On x-axis, black text <br />                                                                               indicates genetic background red text <br />                                                                               indicates RNAi target gene. Error bars rep- <br />                                                                               resent standard error repli- <br />                                                                               cates. Each replicate contained &#226;&#172;&#381;250 adult <br />                                                                               hermaphrodite animals. (B) Our model pre- <br />                                                                               dicts normal conditions, OSR-1 <br />                                                                               functions enable worms avoid hyperos- <br />                                                                               motic environments inhibits unidentified <br />                                                                               pathway(s), CaMKII, conserved p38 <br />                                                                               MAPK pathway. In high osmolarity, OSR-1 <br />                                                                               functions probably inhibited, turn <br />                                                                               activates distinct signaling pathways pro- <br />                                                                               mote adaptation survival. <br />  <br />  <br />  <br />  <br /> lular organism. We established novel genetic screen       osmolytes, glycerol (Hohmann 2002). Our <br /> identify genes critical regulators osmotic     genetic epistasis analysis demonstrates OSR-1 regu- <br /> stress response animal report cloning       lates conserved p38 MAPK pathway. This pathway has <br /> and characterization novel gene, osr-1. On basis      shown induce production glycerol yeast <br /> of phenotypes osr-1(rm1) mutant animals, sug-        cells osmotic upshifts (Hohmann 2002). We also <br /> gest OSR-1 regulates ability C. elegans         osr-1(rm1) animals accumulate high levels of <br /> (i) detect avoid high osmolarity, (ii) resist acute        glycerol (1455 &#226;&#171;&#190; 26 nmol glycerol milligram of <br /> exposures osmotic stress, (iii) maintain viability      protein; A. Solomon R. Morimoto, unpublished <br /> upon prolonged exposure osmotic stress. osr-1(rm1)          results) normal growth conditions, while <br /> animals resistant heat oxidative stress         wild-type animals undetectable levels. Since accu- <br /> and display phenotypes associated mutants          mulation glycerol previously implicated in <br /> of ILS pathway, extended life span             osmotic stress tolerance yeast different species <br /> constitutive dauer formation. Thus, OSR-1 specifically         nematodes including C. elegans (Hohmann 2002; Qiu <br /> regulates osmotic stress responses C. elegans.              Bedding 2002; Lamitina et al. 2003), hypothe- <br />    How OSR-1 regulate different osmotic              size activation p38 MAPK cascade glyc- <br /> stress-induced responses C. elegans? Osmotic upshifts       erol accumulation sufficient explain the <br /> in multiple organisms cause perturbations mem-              osmotic stress tolerance osr-1(rm1) animals. <br /> brane-dependent transport processes activation             We demonstrate pmk-1 downregulation sup- <br /> protein kinase cascades lead accumulation           presses partially (&#226;&#172;&#381;50%) chronic osmotic stress re- <br />  168                                                   A. Solomon et al. <br />  <br /> sistance phenotype osr-1(rm-1) animals, does           partial rescue osmotic avoidance phenotype of <br /> not affect resistance acute dehydration osmotic           osr-1(rm-1) animals. <br /> avoidance defects animals. These findings raise            As seen yeast mammals (Han et al. 1994; de <br /> the intriguing possibility OSR-1 regulates re-        Nadal et al. 2002), p38 MAPK signaling pathway <br /> sponses other, identified, mechanisms (Fig-        important role mediating osmotic stress toler- <br /> ure 5B). This supported observations         ance C. elegans. However, case other <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>null mutants</span> upstream component p38                signaling pathways RAS pathway, up- <br /> pathway, SEK-1 NSY-1, suppress               stream regulators conserved MAPK cascade may <br /> osmotic avoidance acute dehydration phenotypes             differ metazoans unicellular organisms like <br /> osr-1 deficient animals.                                         yeast. For C. elegans, suggest CaMKII regulates <br />    For observed acute resistance osmotic stress           p38 pathway OSR-1-dependent manner (Figure <br /> seen osr-1(rm1) animals, suggest          5B); CaMKII implicated osmotic re- <br /> related alterations cuticle structure,     sponses mammalian kidney cells (Tokuda et al. <br /> mutations cuticle collagen genes, DPY-10 DPY-2            2002). In contrast, HOG/p38 cascade S. cerevisiae <br /> (Levy et al. 1993), result osr-1(rm1)-like resistance      regulated two-component (SLN1-YPD1- <br /> phenotype acute chronic osmotic stress condi-             SSK1; Maeda et al. 1994; Posas et al. 1996) no <br /> tions (A. Solomon R. Morimoto, unpublished re-               orthologs two-component iden- <br /> sults). We currently investigating potential genetic         tified animals. <br /> interactions cuticle component genes                    Our genetic data indicate OSR-1 negatively regu- <br /> osr-1 test idea further.                                 lates multiple osmotic stress responses C. elegans prob- <br />    We additionally age-1(hx546) mutants           ably distinct mechanisms. We favor interpreta- <br /> are resistant effects chronic exposure         tion normal conditions, OSR-1 functions <br /> osmotic stress but, unlike osr-1(rm1) animals,          enable worms avoid hyperosmotic environments. <br /> not osmotic avoidance defective resistant acute            However, hyperosmotic environments, OSR-1 func- <br /> osmotic stress. We currently investigating possi-        tions likely inhibited, turn activates distinct <br /> bility OSR-1 AGE-1 regulate resistance               signaling pathways promote adaptation survival. <br /> chronic osmotic stress overlapping mecha-                Such mechanism bears striking resemblance the <br /> nisms. In studies, isolated additional 104 OSR-        reported inactivation response regulator, Ssk1, <br /> like mutants far characterized different        two-component (SLN1-YPD1-SSK1) S. <br /> complementation groups. Understanding interrela-             cerevisiae conditions high osmolarity (Posas <br /> tionships genes, age-1, osr-1-dependent          et al. 1996). We anticipate characterization <br /> pathways reveal greater OSR-1 func-           OSR-1 identification interacting part- <br /> tions regulate osmotic stress responses C. elegans.        ners osr-1-like genes greatly facilitate the <br />    Osmotic avoidance defective phenotypes              understanding metazoans integrate multiple re- <br /> previously described OSM class mutants                sponses survive hyperosmotic environments. <br /> (Culotti Russell 1978). Mutations OSM-9,                     We thank members Morimoto lab help and <br /> OCR-2, OSM-10 influence function osmo-            discussion Jim Morley, Michelle Lee, Jason Merchant, Marta Cami- <br /> sensory ASH neurons (Colbert et al. 1997; Hart et al.            lot, Peter G. Czarnecki, Luca Jahreiss advice assis- <br /> 1999; Tobin et al. 2002). However,              tance project. Special thanks Heather Brignull Anat <br />                                                                  Ben-Zvi extensive comments discussion manuscript <br /> evidence impaired ASH neuron function osr-                Sue Fox technical support. We appreciate efforts of <br /> 1(rm1) animals. Also, osm-9(ky10), ocr-2(ak47), osm-         Alan Coulson cosmids; Yuji Kohara cDNA clones; Theresa <br /> 10(n1052) mutant animals resistant             Stiernagle Caenorhabditis Genetics Center strains; and <br /> acute chronic osmotic stress; moreover,         Mark Edgley, Kunihiro Matsumoto, Ding Xue, Mario deBono, Cathy <br /> mutations suppress osmotic stress resistance osr-1        Savage-Dunn, Kim Dennis, Frederick M. Ausubel strains and <br />                                                                  vectors. We thank Danielle Thierry-Mieg transcriptom <br /> deficient animals (data shown). These results dem-           project sharing information 1F768. A.S. recipient a <br /> onstrate ASH neurons pathways medi-             postdoctoral fellowship Rothschild Foundation. G.J.B. a <br /> ated OSM genes (OSM-9, OSM-10, OCR-2)                 recipient Burroughs Wellcome Fund Career Award Biomed- <br /> play limited role, any, ability OSR-1          ical Sciences. R.I.M. supported grants National Institutes <br /> regulate resistance osmotic stress.                           Health (National Institute General Medical Sciences) the <br />                                                                  Huntington Disease Society Coalition Cure. <br />    Consistent previous suggestions important <br /> role hypodermis osmoregulation C. elegans <br /> (Petalcorin et al. 1999), hypoder- <br /> mal expression osr-1 cDNA rescues osr-1(rm1) phe-                               LITERATURE CITED <br /> notypes. However, rule possibility             Altun-Gultekin, Z., Y. Andachi, E. L. Tsalik, D. Pilgrim, Y. Kohara <br /> that hypodermal promoter (dpy-7) completely              et al., 2001 A regulatory cascade homeobox genes, ceh- <br />                                                                     10, ttx-3 ceh-23, controls cell fate specification defined <br /> off neurons tissues. Weak expression             interneuron class C. elegans. Development 128: 1951&#226;&#8364;&#8220;1969. <br /> nervous system, example, explain            Bargmann, C. I., J. H. Thomas H. R. Horvitz, 1990 Chemosen- <br />                                                    OSR-1 Osmotic Stress C. elegans                                                   169 <br />     sory cell function behavior development Caenorhab-     Lithgow, G. J., T. M. White, S. Melov T. E. Johnson, 1995 <br />     ditis elegans. Cold Spring Harbor Symp. Quant. Biol. 55: 529&#226;&#8364;&#8220;538.        Thermotolerance extended life-span conferred single- <br /> Berman, K., J. McKay, L. Avery M. Cobb, 2001 Isolation               gene mutations induced thermal stress. Proc. Natl. Acad. <br />     characterization pmk-(1&#226;&#8364;&#8220;3): p38 homologs Caenorhab-          Sci. USA 92: 7540&#226;&#8364;&#8220;7544. <br />     ditis elegans. Mol. Cell. Biol. Res. Commun. 4: 337&#226;&#8364;&#8220;344.             Maeda, T., S. M. Wurgler-Murphy H. Saito, 1994 A two- <br /> Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics           component regulates osmosensing MAP kinase <br />     77: 71&#226;&#8364;&#8220;94.                                                               cascade yeast. Nature 369: 242&#226;&#8364;&#8220;245. <br /> Brewster, J. L., T. Valoir, N. D. Dwyer, E. Winter M. C.          Maeda, T., M. Takekawa H. Saito, 1995 Activation yeast <br />     Gustin, 1993 An osmosensing signal transduction pathway               PBS2 MAPKK MAPKKKs binding SH3-containing <br />     yeast. Science 259: 1760&#226;&#8364;&#8220;1763.                                           osmosensor. Science 269: 554&#226;&#8364;&#8220;558. <br /> Clegg, J. S., 2001 Cryptobiosis&#226;&#8364;&#8221;a peculiar state biological organi-   Murakami, S., T. E. Johnson, 1996 A genetic pathway confer- <br />     zation. Comp. Biochem. Physiol. B Biochem. Mol. Biol. 128:               ring life extension resistance UV stress Caenorhabditis <br />     613&#226;&#8364;&#8220;624.                                                                 elegans. Genetics 143: 1207&#226;&#8364;&#8220;1218. <br /> Colbert, H. A., T. L. Smith C. I. Bargmann, 1997 OSM-9,            Ogg, S., S. Paradis, S. Gottlieb, G. I. Patterson, L. Lee et al., 1997 <br />     novel protein structural similarity channels, required        The Fork head transcription factor DAF-16 transduces insulin- <br />     olfaction, mechanosensation, olfactory adaptation             like metabolic longevity signals C. elegans. Nature 389: <br />     Caenorhabditis elegans. J. Neurosci. 17: 8259&#226;&#8364;&#8220;8269.                      994&#226;&#8364;&#8220;999. <br /> Crowe, J. H., L. M. Crowe D. Chapman, 1984 Preservation           O&#226;&#8364;&#8482;Rourke, S. M., I. Herskowitz, 2002 A osmosensing <br />     membranes anhydrobiotic organisms: role trehalose.             branch Saccharomyces cerevisiae requires Msb2 protein and <br />     Science 223: 701&#226;&#8364;&#8220;703.                                                    functions parallel Sho1 branch. Mol. Cell. Biol. 22: <br /> Crowe, L. M., 2002 Lessons nature: role sugars                4739&#226;&#8364;&#8220;4749. <br />     anhydrobiosis. Comp. Biochem. Physiol. A Mol. Integr. Physiol.       O&#226;&#8364;&#8482;Rourke, S. M., I. Herskowitz E. K. O&#226;&#8364;&#8482;Shea, 2002 Yeast go <br />     131: 505&#226;&#8364;&#8220;513.                                                            HOG hyperosmotic response. Trends Genet. <br /> Culotti, J. G., R. L. Russell, 1978 Osmotic avoidance defective          18: 405&#226;&#8364;&#8220;412. <br />     mutants nematode Caenorhabditis elegans. Genetics 90: 243&#226;&#8364;&#8220;    Petalcorin, M. I., T. Oka, M. Koga, K. Ogura, Y. Wada et al., 1999 <br />     256.                                                                     Disruption clh-1, chloride channel gene, results wider <br /> de Nadal, E., P. M. Alepuz F. Posas, 2002 Dealing osmo-             body Caenorhabditis elegans. J. Mol. Biol. 294: 347&#226;&#8364;&#8220;355. <br />     stress MAP kinase activation. EMBO Rep. 3: 735&#226;&#8364;&#8220;740.          Posas, F., S. M. Wurgler-Murphy, T. Maeda, E. A. Witten, T. C. <br /> Dorman, J. B., B. Albinder, T. Shroyer C. Kenyon, 1995 The               Thai et al., 1996 Yeast HOG1 MAP kinase cascade regulated <br />     age-1 daf-2 genes function common pathway control            multistep phosphorelay mechanism SLN1-YPD1-SSK1 <br />     lifespan Caenorhabditis elegans. Genetics 141: 1399&#226;&#8364;&#8220;1406.         &#226;&#8364;&#339;two-component&#226;&#8364;? osmosensor. Cell 86: 865&#226;&#8364;&#8220;875. <br /> Han, J., J. D. Lee, L. Bibbs R. J. Ulevitch, 1994 A MAP kinase       Potts, M., 1994 Desiccation tolerance prokaryotes. Microbiol. <br />     targeted endotoxin hyperosmolarity mammalian cells.            Rev. 58: 755&#226;&#8364;&#8220;805. <br />     Science 265: 808&#226;&#8364;&#8220;811.                                                Qiu, L., R. A. Bedding, 2002 Characteristics protectant syn- <br /> Hart, A. C., J. Kass, J. E. Shapiro J. M. Kaplan, 1999 Distinct          thesis infective juveniles Steinernema carpocapsae impor- <br />     signaling pathways mediate touch osmosensory responses            tance glycerol protectant survival nematodes <br />     polymodal sensory neuron. J. Neurosci. 19: 1952&#226;&#8364;&#8220;1958.                  osmotic dehydration. Comp. Biochem. Physiol. B Bio- <br /> H <br /> </body></html>