<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Arabidopsis UVH6, Homolog Human XPD Yeast <br /> RAD3 DNA Repair Genes, Functions DNA Repair and <br /> Is Essential Plant Growth1 <br />  <br /> Zongrang Liu2, Suk-Whan Hong3, Mindy Escobar, Elizabeth Vierling, David L. Mitchell, <br /> David W. Mount, Jennifer D. Hall* <br /> Departments Molecular Cellular Biology (Z.L., D.W.M., J.D.H.) Biochemistry Molecular <br /> Biophysics (S.-W.H., M.E., E.V., D.W.M., J.D.H.), University Arizona, Tucson, Arizona 85721; and <br /> Department Carcinogenesis, The University Texas M.D. Anderson Cancer Center, 1515 Holcombe <br /> Boulevard, Houston, Texas 77030 (D.L.M.) <br />  <br />  <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> evaluate genetic control stress responses Arabidopsis, analyzed mutant (uvh6-1) exhibits <br /> increased sensitivity UV light, yellow-green leaf coloration, mild growth defects. We mapped uvh6-1 locus <br /> to chromosome I identified candidate gene, AtXPD, corresponding region. This gene shows sequence <br /> similarity human (Homo sapiens) XPD yeast (Saccharomyces cerevisiae) RAD3 genes required nucleotide excision <br /> repair. We propose UVH6 equivalent AtXPD uvh6-1 mutants carry <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> conserved residue of <br /> AtXPD transformation uvh6-1 mutants wild-type AtXPD DNA suppresses UV sensitivity other <br /> defective phenotypes. Furthermore, UVH6/AtXPD protein appears play role repair UV photoproducts because <br /> the uvh6-1 mutant exhibits moderate defect excision UV photoproducts. This defect suppressed by <br /> transformation UVH6/AtXPD DNA. We identified T-DNA insertion UVH6/AtXPD gene (uvh6-2). <br /> Plants carrying homozygous insertions detected analyses progeny plants heterozygous insertion. <br /> Thus, homozygous insertions appear lethal. We conclude UVH6/AtXPD gene required UV resistance and <br /> is essential gene Arabidopsis. <br />  <br />  <br />    DNA damage challenge organisms ex-                     damaged strand sides lesion, re- <br /> posed UV irradiation. UV photoproducts consist                      moval damaged fragment, repair gap <br /> primarily cyclobutane pyrimidine dimers py-                     filling ligation (Batty Wood, 2000; Boer <br /> rimidine (6-4) pyrimidinone dimers (Mitchell                       Hoeijmakers, 2000; Prakash Prakash, 2000). <br /> Nairn, 1989; Pfeifer, 1997). These lesions inhibit DNA                 Several potential plant homologs human yeast <br /> replication transcription promote mu-                     NER genes identified. Genetic analyses of <br /> tagenesis (McGregor, 1999). The effects UV irradi-                  plant genes, including studies pheno- <br /> ation especially detrimental plants, sun-                 types plants carrying mutations these <br /> light source damage requirement                     genes, provide support idea NER <br /> for photosynthesis.                                                    pathway conserved plants. <br />    Increasing evidence suggests plants repair UV-                    Lesion recognition NER involves ho- <br /> damaged chromosomes using mechanisms similar                        mologous heterodimers XPC:HR23B (human) and <br /> those humans (Homo sapiens) yeast (Sac-                   RAD4:RAD23 (yeast; Balajee Bohr, 2000; Batty <br /> charomyces cerevisiae). These mechanisms include                   Wood, 2000; Boer Hoeijmakers, 2000; <br /> nucleotide excision repair (NER) pathway, process                    Prakash Prakash, 2000). The Arabidopsis genome <br /> which involves recognition UV lesions, incision                  contains potential homologs XPC/RAD4 and <br />                                                                        HR23B/RAD23 (Arabidopsis Genome Initiative, 2000). <br />   1 <br />      This work supported National Science Foundation        HR23B expression occurs Arabidopsis, rice (Oryza <br /> (grant no. MCB&#226;&#8364;&#8220;9728125 D.W.M.) U.S. Department        sativa), carrot (Daucus carota), carrot gene <br /> Agriculture National Research Initiative Competitive Grants Pro-       complements repair defect yeast rad23 mutant <br /> gram (grant no. 99 &#226;&#8364;&#8220;3515100 E.V.).                                  (Schultz Quatrano, 1997; Sturm Lienhard, <br />    2 <br />      Present address: Appalachian Fruit Research Station, 45 Wilt-     1998). <br /> shire Road, Kearneysville, WV 25430.                                      Damage-induced incision NER involves two <br />    3 <br />      Present address: Department Applied Plant Sciences, Chon-      nucleases. The nuclease makes cuts 5&#226;&#172;&#732; the <br /> nam National University, 300 Yang-Bong Dong, Buk-Gu, Gwang-ju <br /> 500 &#226;&#8364;&#8220;757, South Korea. <br />                                                                        lesion consists homologous heterodimers XPF: <br />    * Corresponding author; e-mail jdhall@u.arizona.edu; fax 520 &#226;&#8364;&#8220;      ERCC1 (human) RAD1:RAD10 (yeast). A homol- <br /> 621&#226;&#8364;&#8220;3709.                                                              ogous complex implicated plant DNA <br />    Article, publication date, citation information    repair based UV sensitivity Arabidopsis <br /> at www.plantphysiol.org/cgi/doi/10.1104/pp.103.021808.                 derivatives carrying mutation plant <br /> Plant Physiology, July 2003, Vol. 132, pp. 1405&#226;&#8364;&#8220;1414, www.plantphysiol.org &#194;&#169; 2003 American Society Plant Biologists     1405 <br />  Liu et al. <br />  <br />  <br /> XPF homolog (Fidantsef et al., 2000; Liu et al., 2000)     UV sensitive, indicative repair defect, we <br /> or antisense construct gene (Gallego et al.,    hypothesized uvh6-1 mutation maps within <br /> 2000). Arabidopsis XPF mutants defective       AtXPD gene. To test possibility, amplified <br /> repair UV photoproducts (Liu et al., 2000; Li et al.,   AtXPD cDNA wild-type (C10) uvh6-1 mu- <br /> 2002). Homologs ERCC1 component             tant plants (as described &#226;&#8364;&#339;Materials Methods&#226;&#8364;?) <br /> first nuclease occur Arabidopsis (Arabidopsis Ge-       determined corresponding sequences. <br /> nome Initiative, 2000) Lilium longiflorum (Xu et       Compared wild-type sequence, a <br /> al., 1998), L. longiflorum gene complements      single-nucleotide change codon 521 uvh6-1 <br /> repair defect ERCC1-deficient Chinese hamster           mutant cDNA sequence. This mutation con- <br /> (Cricetulus griseus) ovary cells. The nuclease in-    firmed sequencing mutant genomic DNA (data <br /> cises 3&#226;&#172;&#732; DNA lesions called XPG humans        shown). As depicted Figure 1B, mutation <br /> RAD2 yeast. A mutation Arabidopsis XPG           result Gly Glu missense substitution. <br /> homolog results UV sensitivity (Liu et al., 2001b).     Hence, likely UVH6 gene equiv- <br /> In addition, second mutation (uvr1), fails      alent AtXPD sequence change de- <br /> complement AtXPG mutation (Z. Liu D.               tected AtXPD gene uvh6-1 mutation. <br /> Mount, unpublished data) results defective repair          The UVH6/AtXPD sequence obtained <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> of 6-4 photoproducts (Britt et al., 1993).                 wild-type plants nearly identical unpub- <br />    Finally, NER requires DNA unwinding              lished AtXPD mRNA sequences GenBank (acces- <br /> DNA helicases, called XPB XPD humans            sion nos. AF188623 AY062471) contains a <br /> RAD25 RAD3 yeast. Two potential Arabidop-           few, possibly polymorphic, differences the <br /> sis homologs XPB/RAD25 identified             coding region. In comparison AF188623, we <br /> (Ribeiro et al., 1998; Arabidopsis Genome Initiative,      observed differences, codon 541 (AGC/ <br /> 2000). AtXPB1 complements repair defect yeast         AGT), did alter encoded Ser amino <br /> rad25 mutants, plants carrying T-DNA insertions        acid, codon 554 (GAA/AAG), which <br /> in gene sensitive alkylating agents (Costa     changed encoded Glu Lys sequence. In <br /> et al., 2001b). Arabidopsis encodes presumed        comparison AY062471, silent change ob- <br /> homolog XPD/RAD3 genes (Arabidopsis Ge-             served codon 282 (CGA/CGG). The residues that <br /> nome Initiative, 2000).                                    observed positions identical to <br />    The AtXPD gene subject present study.     Arabidopsis consensus genome <br /> We examined Arabidopsis mutation (called           sequence (Arabidopsis Genome Initiative, 2000), <br /> uvh6-1) dramatically increases sensitivity UV      which, like sequence, derived plants of <br /> light. We present evidence mutation results      Columbia ecotype. Our cDNA sequence been <br /> in missense change Arabidopsis XPD/           submitted GenBank (accession no. AY090788) and <br /> RAD3 gene. Our findings support possi-         depicted Figure 1B relation correspond- <br /> bility AtXPD/UVH6 gene product functions          ing genomic sequence. <br /> within plant NER pathway. In addition, ob- <br /> serve T-DNA insertion AtXPD (uvh6-2)          Complementation UV Sensitivity uvh6-1 <br /> lethal, suggesting gene serves essential      Mutant AtXPD DNA <br /> function plant development. <br />                                                              To confirm identification UVH6 gene, we <br />                                                            characterized phenotypes uvh6-1 mutant <br /> RESULTS                                                    plants transformed wild-type <br /> Identification UVH6 Gene                    (C10) AtXPD genomic DNA. Primary T1 transfor- <br /> uvh6-1 Mutation                                            mants examined UV sensitivity, 22 26 <br />                                                            transformants displayed UV-resistant <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br />   We previously identified UV-sensitive mutant        This phenotype depicted Figure 2 T2 gener- <br /> Arabidopsis (originally called uvh6 renamed            ation plants homozygous introduced AtXPD <br /> uvh6-1 study) used map UVH6          transgene. As shown, uvh6-1 mutant plants ab- <br /> locus 6 centimorgans chromosome I            normally sensitive UV-C irradiation, compared <br /> (Jenkins et al., 1995). As shown Figure 1A,     wild-type parent, exhibit severe brown- <br /> further refined location gene region       ing death rosette leaves 3 d treat- <br /> between physical markers nga59 AtEAT1.                 ment (compare Fig. 2A, middle left). This find- <br /> Within region, identified candi-       ing confirms previous report uvh6-1 plants <br /> date gene locus At1g03190 (Bac locus F15K9). We         hypersensitive UV-C UV-B irradia- <br /> have designated gene AtXPD ex-          tion (Harlow et al., 1994). In contrast, transformed <br /> hibits strong similarity ERCC2 (XPD) Chi-          plants exhibit resistance irradiation <br /> nese hamster RAD3 (yeast; The Arabidopsis In-          shown wild-type parents (right). Thus, the <br /> formation Resource, http://www.arabidopsis.org).           mutant phenotype complemented wild-type <br />   Because mammalian XPD yeast RAD3 genes               AtXPD DNA, indicating UVH6/AtXPD gene <br /> are required NER uvh6-1 mutant         required resistance UV irradiation. <br /> 1406                                                                                     Plant Physiol. Vol. 132, 2003 <br />                                                                                     Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br />  <br />  <br />          Figure 1. Mapping UVH6 locus organization corresponding UVH6(AtXPD) gene. A, Top 3.3 megabases <br />          chromosome I shown corresponding physical markers nga59, AtEAT1, nga63. The UVH6 locus localized <br />          markers nga59 AtEAT1, described &#226;&#8364;&#339;Materials Methods,&#226;&#8364;? based analysis DNA 380 <br />          UV-sensitive F2 progeny (760 chromosomes) obtained cross uvh6-1 (Columbia ecotype) wild-type <br />          (Landsberg ecotype) plants. Three UV-sensitive recombinants carrying Landsberg nga59 polymorphism identified, <br />          lacked Landsberg nga63 AtEAT1 markers. Ten recombinants carrying Landsberg nga63 also <br />          obtained. Five carried Landsberg AtEAT1 marker, carried Landsberg nga59 marker. The <br />          position Bac clone F15K9, carries candidate repair gene AtXPD (gene F15K9.20, locus At1g03190), is <br />          indicated. B, The UVH6/AtXPD gene diagrammed, based comparison known genomic cDNA <br />          sequences. The diagram depicts exons (thick bars), introns (thin lines), initiation termination codons, uvh6-1 <br />          mutation codon 521. C, Alignment central portion human XPD (accession no. CAA36463) UVH6/AtXPD <br />          (accession no. AY090788) protein sequences shown, encompassing conserved helicase motifs III IV. The uvh6-1 <br />          mutation amino acid 521 indicated, sites known human mutations (designated Hs), produce the <br />          diseases xeroderma pigmentosum trichothiodystrophy, indicated. <br />  <br />  <br />  <br /> Growth Defects uvh6-1 Mutant                                      grown normal (non-stress) conditions and <br />   In addition radiation sensitivity, uvh6-1 mutant                    result low chlorophyll content these <br /> plants exhibit growth defects sup-                         plants described previously (Jenkins et al., 1997). In <br /> pressed plants transformed wild-type UVH6/                       addition, mutant uvh6-1 plants substantially <br /> AtXPD genomic DNA. As shown Figure 2B (mid-                           smaller compared wild-type plants the <br /> dle), leaves uvh6-1 mutant plants yellow-                     age, seen Figure 2C (left). In contrast, <br /> green, previously reported (Jenkins et al., 1997).                    uvh6-1 plants complemented with <br /> This phenotype observed mutant plants                        AtXPD genomic DNA dark-green exhibit <br /> Plant Physiol. Vol. 132, 2003                                                                                                         1407 <br />  Liu et al. <br />  <br />  <br />                                                                       sequences (accession nos. CAA36463 AAB64698, <br />                                                                       respectively). The Arabidopsis sequence exhibits <br />                                                                       strong similarity homologs entire <br />                                                                       <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, residue mutated uvh6-1 mutant <br />                                                                       conserved comparisons. Figure 1C depicts <br />                                                                       alignment human sequence, includ- <br />                                                                       ing uvh6-1 mutation site. Additional alignments <br />                                                                       (not shown) XPD homologs Schizosaccha- <br />                                                                       romyces pombe, Caenorhabditis elegans, Drosophila <br />                                                                       melanogaster (accession nos. CAA93221, AAK95892, <br />                                                                       AAD33587, respectively) reveal strong se- <br />                                                                       quence similarity conservation G residue <br />                                                                       position uvh6-1 mutation. This strong <br />                                                                       conservation uvh6-1 residue ho- <br />                                                                       mologs suggests mutation disrupts <br />                                                                       function UVH6/AtXPD protein. <br />                                                                          Human XPD yeast RAD3 genes encode DNA <br />                                                                       helicases essential repair (Winkler et al., 2000). <br />                                                                       Seven highly conserved motifs sequences are <br />                                                                       required helicase activity (Gorbalenya et al., <br />                                                                       1989). Our comparison AtXPD sequence with <br />                                                                       human yeast homologs revealed especially <br />                                                                       strong similarity regions seven motifs <br />                                                                       (data shown). Sequence conservation involving <br />                                                                       motifs III IV illustrated Figure 1C. Thus, it <br />                                                                       appears likely AtXPD plant gene acts a <br />                                                                       helicase performs role DNA repair <br />                                                                       previously demonstrated humans yeast. <br />  <br />  <br /> Figure 2. Recovery UV resistance normal growth properties      Repair 6-4 Photoproducts uvh6-1 Mutant <br /> after complementation uvh6-1/uvh6-1 homozygous mutant with <br /> UVH6/AtXPD DNA. Mutant uvh6-1 plants transformed              The UV sensitivity uvh6-1 mutant suggests <br /> UVH6/AtXPD genomic DNA (from C10 wild-type plants) de-             mutant defective repair UV <br /> scribed &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? C10 wild-type (UVH6/UVH6),        photoproducts. To test possibility, examined <br /> uvh6-1 mutant (uvh6/uvh6), T2 generation-complemented mu-         repair 6-4 photoproducts DNA irradi- <br /> tant plants grown 2 3 weeks subsequently exposed to <br /> UV-C irradiation (300 J m2) described &#226;&#8364;&#339;Materials Methods&#226;&#8364;? <br />                                                                       ated plants, comparing wild-type, uvh6-1 mutant, <br /> (A) maintained normal growth conditions 3 weeks (B),     uvh3 mutant plants. The uvh3 mutant UV <br /> 4.5 weeks (c, left), 6.5 weeks (c, right).                         sensitive carries <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> the <br />                                                                       UVH3/AtXPG gene, encoding homolog the <br />                                                                       human XPG 3&#226;&#172;&#732;-incision endonuclease (Jenkins et al., <br /> normal growth rates (Fig. 2B, right). Thus, ad-                 1995; Liu et al., 2001b). As shown Figure 3A, <br /> ditional mutant phenotypes appear result the <br />                                                                       wild-type plants exhibited significant removal (62%) <br /> same mutation causes UV sensitivity. <br />                                                                       6-4 products DNA 24 h irradi- <br />                                                                       ation. In comparison, uvh6-1 mutant (Fig. 3C) <br /> Relationship AtXPD Presumed Human                           exhibited reduced levels repair 24 h (47% <br /> Yeast Homologs                                                        photoproduct removal), suggesting mod- <br />   We originally identified AtXPD gene based                    erate repair defect. No photoproduct removal was <br /> the similarity encoded protein sequence hu-                 detected uvh3 mutant (Fig. 3B). This latter <br /> man XPD yeast RAD3 proteins, well-                     result confirms previous report (Britt et al., 1993) of <br /> established roles NER (Batty Wood, 2000;                    defective repair mutant (uvr1) fails com- <br /> Boer Hoeijmakers, 2000; Prakash Prakash,                      plement uvh3 mutant genetic tests. Finally, for <br /> 2000). We performed protein sequence                     uvh6-1 uvh3 mutants, complementation <br /> comparisons assess similarity gain                     appropriate wild-type genes resulted resto- <br /> insight function Arabidopsis protein.                 ration wild-type level repair. The comple- <br /> Pair-wise alignments (data shown) indicate                   mented uvh3 mutant showed 66% photoproduct re- <br /> the Arabidopsis sequence (accession no. AAF14582)                     moval (Fig. 3B) complemented uvh6-1 <br /> is 56% 50% identical human yeast                       mutant showed 65% removal (Fig. 3C). <br /> 1408                                                                                                Plant Physiol. Vol. 132, 2003 <br />                                                                                     Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br />                                                                          vealed plants heterozygous five <br />                                                                          homozygous wild-type allele. No ho- <br />                                                                          mozygous T-DNA insertions detected, suggest- <br />                                                                          ing homozygous insertions lethal. <br />                                                                             To investigate possibility, analyzed of <br />                                                                          heterozygous plants (called Ga-2 Ga-3) <br />                                                                          determine frequency progeny <br />                                                                          inherit T-DNA insert. The heterozygote parents <br />                                                                          allowed produce seeds self-fertilization, <br />                                                                          DNA individual F1 seedlings subjected <br />                                                                          PCR analysis described above. In total, 97 <br />                                                                          progeny plants examined. As seen Table I, <br />                                                                          56.7% plants heterozygous 43.3% <br />                                                                          carried wild-type alleles, plants ho- <br />                                                                          mozygous insert detected. This result <br /> Figure 3. Repair 6-4 photoproducts DNA UV-irradiated          clearly deviates theoretical expectation that <br /> plants. DNA extracted irradiated plants assayed photo- <br />                                                                          25% progeny carry homozygous <br /> products described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? The plants studied <br /> were C10 wild-type (F), mutant uvh3 (&#228;&#161;&#186;), complemented mutant <br />                                                                          T-DNA insertions conditions inser- <br /> uvh3 (f), mutant uvh6-1 (&#226;&#8364;&#353;), complemented mutant uvh6-1 (&#197;&#8217;).         tion lethal (P &#226;&#172;? 0.01). This result rules out <br /> Each point represents mean separate assays, SDs   possibility lethality caused second <br /> means shown. The difference 24-h data points         mutation unlinked UVH6/AtXPD locus, al- <br /> uvh6-1 complemented uvh6-1 plants statistically signifi-      presence lethal mutation closely <br /> cant (P &#226;&#172;? 0.025).                                                        linked UVH6 possible. Most likely, the <br />                                                                          results indicate homozygous uvh6-2 insertions <br /> Characterization T-DNA Insertion                                 UVH6/AtXPD lethal UVH6/ <br /> UVH6/AtXPD Its Effect Plant Viability                             AtXPD gene serves essential function plant <br />                                                                          growth. <br />   We identified line Arabidopsis plants <br /> (called 825 BO5) carrying T-DNA insertion (which <br />                                                                          Expression UVH6/AtXPD Gene <br /> we designate uvh6-2) UVH6/AtXPD <br /> gene. The site insertion confirmed DNA                           The mRNA expression <span id='am-397' about='xsp:pattern' typeof='owl:Thing'>pattern</span> UVH6/AtXPD <br /> sequencing, depicted Figure 4. Insertion oc-                       gene analyzed shown Figure 5. Total RNA <br /> curred sixth 11 introns UVH6/                       extracted plant tissues, specific <br /> AtXPD gene presumably results truncated                         mRNA sequences amplified RT-PCR, and <br /> UVH6/AtXPD protein altered splicing                        PCR products visualized electrophoresis. <br /> UVH6/AtXPD mRNA. Based site insertion,                         We previously use 50 ng input <br /> 26% coding sequence highly                        plant RNA gave PCR product propor- <br /> conserved helicase motifs deleted.                             tional input RNA concentration, using 25 30 <br />   In preliminary analysis, asked the <br /> 825 BO5 line homozygous heterozygous for <br /> the T-DNA insert. For analysis, 10 plants were <br /> grown initial seed stock, genomic DNA <br /> from plant subjected PCR analysis that <br /> distinguishes wild-type T-DNA- <br /> inserted alleles UVH6/AtXPD. The locations of <br /> primers used analysis shown Figure 4. <br /> Gene-specific primers anneal sites flanking <br /> the insertion site used identify wild-type <br /> UVH6/AtXPD alleles. This test detects wild-type <br /> alleles presence insertion results                 Figure 4. Site T-DNA insertion (uvh6-2) UVH6/AtXPD <br /> an inter-primer distance (greater 10 kb)                    gene. The position insert confirmed DNA sequencing, <br /> too long PCR amplification. To detect alleles car-                   described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? This site shown with <br /> rying insertion, second PCR reaction per-                      respect UVH6/AtXPD genomic DNA, UVH6/AtXPD <br /> formed using UVH6-specific primers                      DNA sequence adjacent left border (LB) shown. The UVH6/ <br />                                                                          AtXPD gene depicted exons (thick lines) introns (thin <br /> primer anneals T-DNA left border. <br />                                                                          lines). Positions primers (UVH6-F, UVH6-R, LB3, LB3D) <br /> Thus, DNA plants heterozygous insert                        used PCR reactions DNA sequencing indicated. For <br /> should amplified sets primers,                           results shown Table I, primers UVH6-F UVH6-R (gene spe- <br /> whereas DNA plants homozygous                        cific) used detect wild-type alleles; primers UVH6-R LB3 <br /> wild-type inserted allele amplified                      (Ga-3 experiment) LB3D (Ga-2 experiment) used detect <br /> only set primers. Results analysis re-                    T-DNA-inserted alleles. <br />  <br /> Plant Physiol. Vol. 132, 2003                                                                                                           1409 <br />  Liu et al. <br />  <br />  <br />  <br /> Table I. PCR analysis DNA progeny Ga-2 Ga-3 plants carrying heterozygous T-DNA insertion (uvh6-2) UVH6/AtXPD <br /> genea <br />                                                                         No. Progeny Plants Specified Genotype <br />          Genotype Progeny Plantsb <br />                                                   Progeny Ga-3 parent                Progeny Ga-2 parent              Total progenyc <br />        Homozygous wild type                                 18                                       24                          42(43.3%) <br />        Heterozygous T-DNA insertion                         29                                       26                          55(56.7%) <br />        Homozygous T-DNA insertion                            0                                        0                           0(0.0%) <br />   a <br />     Genomic DNA extracted leaves F1 seedlings plants (Ga-3 Ga-2) carrying heterozygous T-DNA insertions in <br />                                                                                                                     b <br /> UVH6/AtXPD, DNA subjected PCR analysis, described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;?                       Genotypes scored as <br /> homozygous wild type (amplification observed gene-specific primers), T-DNA insertion heterozygous (amplification observed with <br /> both sets primers), T-DNA insertion homozygous (amplification observed T-DNA/gene-specific primer pair). Figure 4 shows <br /> the primer annealing sites.    c <br />                                  A &#226;?&#185;2 analysis results generated value P &#226;&#172;? 0.01, indicating results consistent with <br /> the expected Mendelian genetic distribution 1:2:1. <br />  <br />  <br /> PCR cycles (Liu et al., 2001b). As seen Figure 5,                         Glu position 521 UVH6/AtXPD protein. <br /> these conditions gave clearly detectable levels                           This mutation appears cause extreme UV sen- <br /> product UBQ10 control, gene ex-                              sitivity exhibited uvh6-1 plants mutant <br /> presses relatively high level. This control                        plants transformed wild-type UVH6/AtXPD <br /> serves verify equivalent quantities RNA                           DNA regain wild-type radiation resistance. Mutant <br /> were present PCR reaction. However, de-                           sensitivity UV light likely result defi- <br /> tect PCR product UVH6/AtXPD, ex-                                 ciency NER, based finding mutant <br /> presses relatively low level, required addi-                         plants exhibit moderate defect removal 6-4 <br /> tional input RNA (200 ng) 40 PCR cycles. Under                           photoproducts DNA. <br /> these conditions, observed UVH6/AtXPD                                  In addition, examined mutant carrying a <br /> mRNA expressed various degrees leaf, root,                          T-DNA insertion (uvh6-2) UVH6/AtXPD <br /> stem, flower bud, meristem tissues. Hence,                               gene. This insertion predicted produce trun- <br /> UVH6/AtXPD appears expressed low levels                             cated UVH6 gene product. Our findings indicate that <br /> in tissues, expected gene required                          homozygous insertions UVH6/AtXPD le- <br /> general transcription.                                                       thal, suggesting gene encodes essential <br />                                                                              function required plant development. <br /> DISCUSSION <br />  <br />   We identified mutation (uvh6-1) Ara-                         Role AtXPD DNA Repair <br /> bidopsis AtXPD gene characterized the <br /> phenotypes uvh6-1 mutant plants. AtXPD pre-                               Genes involved NER highly con- <br /> dicted homolog human XPD yeast                            served yeast humans (Prakash Prakash, <br /> RAD3 genes, required NER DNA.                               2000), suggesting common mechanism. This mech- <br /> The uvh6-1 mutation missense change Gly                            anism appears conserved plants, based <br />                                                                              characterization presumed ho- <br />                                                                              mologs NER genes Arabidopsis (Britt et al., <br />                                                                              1993; Fidantsef et al., 2000; Gallego et al., 2000; Liu et <br />                                                                              al., 2000, 2001b; Li et al., 2002). Our findings the <br />                                                                              uvh6-1 mutant carries mutation presumed <br />                                                                              Arabidopsis homolog XPD (human) RAD3 <br />                                                                              (yeast) mutant sensitive UV <br />                                                                              light moderate repair defect provide addi- <br />                                                                              tional evidence NER conserved plants. <br />                                                                                 Human XPD yeast RAD3 proteins function as <br />                                                                              helicases DNA repair, presumably unwind- <br /> Figure 5. Expression UBQ10 UVH6 mRNA Arabidopsis                   ing DNA surrounding target lesions facilitate in- <br /> tissues, measured reverse transcriptase (RT)-PCR. Total RNA            cision (Sung et al., 1988; Winkler et al., 2000). We <br /> was isolated C10 wild-type plant tissues indicated              think highly likely Arabidopsis homolog <br /> assayed, described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? RT-PCR products            UVH6/AtXPD acts repair helicase. The <br /> were detected agarose gel electrophoresis, product <br />                                                                              UVH6/AtXPD protein sequence contains seven <br /> amplified fully spliced message shown case. <br /> Controls RT run experiment (not shown) and <br />                                                                              highly conserved motifs required helicase <br /> failed produce product, ruling contribution             function (Gorbalenya et al., 1989). The uvh6-1 muta- <br /> genomic DNA amplification. RNA prepared unbolted                 tion lies motifs III IV, suggesting it <br /> flower buds, roots, stems, leaves, using 4-week-old plants        impair helicase activity. In addition, uvh6-1 <br /> 2-week-old meristem tissue.                                                  mutation occurs G residue highly con- <br /> 1410                                                                                                                 Plant Physiol. Vol. 132, 2003 <br />                                                                       Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br /> served humans, yeast, S. pombe, C. elegans, D.        heat resistance. However, rule the <br /> melanogaster, suggesting residue        possibility uvh6-1 mutants impaired in <br /> important normal enzymatic function               transcription se but, repair defect, <br /> UVH6/AtXPD.                                                  accumulate DNA damage blocks transcription. <br />  <br />  <br />                                                              Interaction AtXPD Predicted <br /> Potential Role AtXPD Regulating Gene <br />                                                              TFIIH Complex <br /> Expression Plant Development <br />                                                                 In humans yeast, TFIIH transcription ini- <br />    Mutant uvh6-1 plants exhibit phenotypic defects <br />                                                              tiation complex contains core components and <br /> when grown normal lighting conditions, sug- <br />                                                              additional proteins associated kinase <br /> gesting UVH6/AtXPD protein role <br />                                                              CAK complex. Of components, three <br /> during plant development addition DNA repair. <br />                                                              core proteins CAK protein appear be <br /> Under normal, non-stress growth conditions, mutant <br />                                                              conserved Arabidopsis. Strong matches Arabi- <br /> plants small compared wild-type plants of <br />                                                              dopsis sequences observed human core com- <br /> the age. They yellow-green appearance, <br />                                                              ponents XPD, XPB (two matched loci), p44. <br /> contain subnormal levels chlorophyll. (Jenkins et <br />                                                              These matches occur loci At1g03190, At5g41370, <br /> al., 1997), exhibit poorly organized grana stacks <br />                                                              At5g41360, At1g05050, respectively. The exis- <br /> within thylakoid membranes (M. Jenkins, unpub- <br />                                                              tence expressed AtXPB genes (Costa et al., <br /> lished data). Furthermore, mutants abnor- <br />                                                              2001b) suggests multiple forms TFIIH may <br /> mally sensitive prolonged exposure high tem- <br />                                                              exist plants. In addition, CAK catalytic subunit <br /> perature (37&#194;&#176;C; Jenkins et al., 1997). In addition, we <br />                                                              appears conserved, based clustering an <br /> have observed T-DNA insertions the <br />                                                              Arabidopsis kinase (locus At1g73690) human <br /> UVH6 gene lethal, based failure isolate <br />                                                              (cdk7) yeast (KIN28) CAK sequences phy- <br /> plants homozygous T-DNA-insertion <br />                                                              logenetic tree cyclin-dependent protein kinase se- <br /> (uvh6-2) gene. Thus, UVH6/AtXPD appears to <br />                                                              quences (http://kinase.ucsf.edu/ksd/). In contrast, <br /> serve essential function plant growth. <br />                                                              sequences similar core and <br />    To understand role UVH6/AtXPD during <br />                                                              CAK components Arabidopsis. <br /> plant growth, informative consider known <br />                                                              Although recent report (Costa et al., 2001a) claimed <br /> functions human yeast homologs. Human <br />                                                              components (p52 and <br /> XPD homologous yeast RAD3 proteins have <br />                                                              cyclinH) Arabidopsis sugarcane (Saccharum <br /> dual roles repair transcription initiation (de <br />                                                              officinarum Saccharum spontaneum), sequence <br /> Boer Hoeijmakers, 2000; Prakash Prakash, <br />                                                              analyses suggest report represent <br /> 2000; Lehmann, 2001). In role, facili- <br />                                                              matches isolated protein domains. Thus, only <br /> tate initiation transcription RNA polymerase II <br />                                                              potential TFIIH core CAK compo- <br /> as initiation factor TFIIH. Considerable evi- <br />                                                              nents identified Arabidopsis, suggesting <br /> dence suggests human XPD mutations cause <br />                                                              complexes exist com- <br /> developmental abnormalities defects tran- <br />                                                              posed novel highly diverged subunits plants. <br /> scription. These mutations produce different <br /> genetic diseases (xeroderma pigmentosum, Cock- <br /> ayne syndrome, trichothiodystrophy; Boer              Analysis T-DNA Insertions Suggests That <br /> and Hoeijmakers, 2000; Lehmann, 2001) result        UVH6/AtXPD Is Essential Gene <br /> in sensitivity UV light repair deficiencies but <br /> are distinguished additional, distinct phenotypes.          In contrast subtle growth defects uvh6-1 <br /> For example, trichothiodystrophy associated          mutant plants, failed detect plants carry- <br /> brittle hair Cockayne syndrome skeletal             ing homozygous T-DNA insertions UVH6/ <br /> deformations. Substantial evidence supports idea         AtXPD gene. The specific insertion charac- <br /> that additional phenotypes result tran-           terized (uvh6-2) predicted cause truncation of <br /> scription defects differentiated tissues,     UVH6/AtXPD protein, deleting 26% length <br /> just loss repair capacity (Bootsma Hoeijmak-          seven conserved helicase domains. <br /> ers, 1993; Boer et al., 1998; Lehmann, 2001; Liu et       Thus, overall structure presumed helicase <br /> al., 2001a; Viprakasit et al., 2001; Keriel et al., 2002).   activity protein disrupted. Our fail- <br />    Reminiscent patients human diseases,        ure isolate plants carrying homozygous insertions <br /> uvh6-1 mutants exhibit developmental stress-             UVH6/AtXPD suggests gene essential <br /> response defects. These phenotypes consistent            plant growth. Our observation UVH6/AtXPD <br /> with possibility UVH6/AtXPD gene             expressed plant tissues supports possi- <br /> required transcription plants. Hence, hy-          bility gene important general <br /> pothesize transcription abnormal uvh6-1       transcription. <br /> mutant, causing defects expression genes re-             We observed apparent reduction in <br /> quired chlorophyll synthesis, normal growth rate,        transmission T-DNA insertion prog- <br /> Plant Physiol. Vol. 132, 2003                                                                                   1411 <br />  Liu et al. <br />  <br />  <br /> eny plants carrying heterozygous insertion.                                Genetic Physical Mapping UVH6 Gene <br /> These progeny heterozygous in-                                 Recombinant plants generated crossing uvh6-1/uvh6-1 homozy- <br /> sertion lost insertion wild type.                           gous mutant plant (Columbia ecotype) wild-type Landsberg erecta <br /> However, observed ratio heterozygotes:wild                               plant. F2 progeny screened identify exhibited UV <br /> type close 1:1, expected 2:1.                            sensitivity mutant (Columbia) parent, using protecting foam shield <br />                                                                                 meristem treatment, described (Harlow et al., 1994). UV- <br /> This finding suggests transmission in-                              sensitive progeny examined presence Landsberg- <br /> serted allele impaired female                         associated simple sequence length polymorphism physical markers nga59, <br /> male gametes (or both). We currently testing                                AtEAT1, nga63. Polymorphisms detected agarose gel analysis <br /> these possibilities backcrossing heterozygous                                PCR assays (Bell Ecker, 1994). <br /> plants wild type. <br />   A model explain differences observed be-                               Mutant Complementation <br /> tween uvh6-1 point mutant uvh6-2 <br /> T-DNA insertion mutant suggested stud-                                  A 5.6-kb fragment AtXPD genomic DNA, carries UVH6/ <br />                                                                                 AtXPD coding region, 3&#226;&#172;&#732;-untranslated sequences, 5&#226;&#172;&#732;-flanking region (1.2 <br /> ies yeast RAD3 mutations. These mutations exist in <br />                                                                                 kb) containing AtXPD promoter, amplified PCR wild-type <br /> two classes. Mutations class lethal                        (C10) DNA, using primer pair H6U186 (5&#226;&#172;&#732;-CAACATTCCGATTTTCCGT- <br /> result severe defects RNA polymerase II-                                  CACCT) XDL2532 (5&#226;&#172;&#732;-CCTACAGTGAAAATTTGAGCTCCAACAATT). <br /> dependent transcription restrictive conditions                            The amplified fragment cloned pCRII vector, using TA <br />                                                                                 Cloning Kit (Invitrogen, Carlsbad CA), transferred plasmid <br /> (Guzder et al., 1994; Prakash Prakash, 2000), <br />                                                                                 pBIN19. The resultant recombinant (pAtXPD) introduced Agrobac- <br /> suggesting transcription defect causes                             terium strain GV3101 electroporation, described Mozo Hooykaas <br /> lethality. In contrast, second class mutations                           (1991). Transformed bacteria selected Luria-Bertani medium (Sam- <br /> does impair viability, mutants                          brook et al., 1989) containing 30 mg L&#226;&#171;&#186;1 gentamycin 60 mg L&#226;&#171;&#186;1 kana- <br />                                                                                 mycin. DNA selected bacteria transferred uvh6-1 mutant <br /> sensitive UV irradiation exhibit defects in <br />                                                                                 plants using floral dip method (Clough Bent, 1998), transformed <br /> NER (Prakash Prakash, 2000). Thus, second                              kanamycin-resistant plants selected generation seed on <br /> class appears affect DNA repair, leaving                               media containing one-half-strength Murashige Skoog basal salt mix <br /> transcription largely intact. Presumably, XPD muta-                             (Gibco/Life Technologies, Grand Island, NY), 1&#226;&#171;&#187; B5 vitamins (Bamborg et al., <br /> tions produce diseases humans                               1968), 1% (w/v) Suc, 0.8% (w/v) Bacto-agar, 60 mg L&#226;&#171;&#186;1 kanamycin. <br />                                                                                 Primary transformants tested, described Liu et al. (2001b) to <br /> second type cause repair defects subtle                           identify UV-resistant, complemented transgenic lines. The UV-resistant phe- <br /> impacts transcription. In accord model,                            notypes confirmed T2 generation plants. <br /> the uvh6-1 <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> resemble this <br /> second class, causing repair defect minor ef- <br /> fects transcription, uvh6-2 T-DNA                                DNA Sequencing <br /> insertions resemble class, producing                               To sequence AtXPD/UVH6 cDNA, partial cDNA fragments were <br /> <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> severe transcription defect.                          amplified C10 wild-type uvh6-1 mutant plants RT-PCR, as <br />                                                                                 described below, fragment sequenced. A full-length cDNA <br />                                                                                 sequence assembled, uvh6-1 mutation identified com- <br />                                                                                 parison wild-type mutant sequences. The primers used amplifi- <br /> MATERIALS AND METHODS                                                           cation were: XDU169, 5&#226;&#172;&#732;-GCCAATTTCGAGATCTAGGTAGGAGGAA; <br />                                                                                 XDU740, 5&#226;&#172;&#732;-TAAGGGCGTTTGGTAAGAATC; H6U1375, 5&#226;&#172;&#732;-TACGCCATC- <br /> Strains Growth Conditions                                                   TCGGCACCCAGGCT; H6L2699, 5&#226;&#172;&#732;-AAGTACTGAAGTAATCGGCGT; <br />                                                                                 XDL1740, 5&#226;&#172;&#732;-ACTTCTCATGTCAAATTTGGTGCT; XDL2382, 5&#226;&#172;&#732;-CCCATTG- <br />    The wild-type Columbia (C10) used derived single plant            TACCCGCCTTATCAT; H6U1534, 5&#226;&#172;&#732;-AACCGGTGTTTGATCGTTTCCAGT; <br /> isolate Arabidopsis served parent uvh6-1 mutant,     H6U4835, 5&#226;&#172;&#732;-CTGGTTGGATACTTTCGCATCTGC; H6L5357, 5&#226;&#172;&#732;- <br /> was isolated described (Harlow et al., 1994; Jenkins et al., 1995). Plants   AGGAGCCGACGAAGTATTTCTTGA. To sequence T-DNA insertion <br /> were normally grown room temperature (22&#194;&#176;C&#226;&#8364;&#8220;24&#194;&#176;C) continuous            UVH6, DNA isolated plant carrying insertion, sub- <br /> lighting consisting combination 40WT12 Excella F40 agro bulbs       jected PCR amplification using primers UVH6-R LB3 amplify the <br /> (Philips Lighting Company, Somerset, NJ) positioned approximately 35 cm         region surrounding insertion site (see Fig. 4) sequenced using <br /> from plants (30 &#226;?&#174;mol photons m&#226;&#171;&#186;2 s&#226;&#171;&#186;1).                                      LB3 primer (5&#226;&#172;&#732;-TAGCATCTGAATTTCATAACCAATCTCGATACAC). <br />    Seeds carrying T-DNA insertion (uvh6-2) UVH6 gene (garlic line <br /> 825 BO5) obtained Torrey Mesa Research Institute (San <br /> Diego). Seeds surface sterilized germinated agar plates      Repair Assays <br /> (Haughn Somerville, 1986) germinated soil grown a <br /> growth chamber 22&#194;&#176;C 16 h light (240 &#226;?&#174;mol photons m&#226;&#171;&#186;2 s&#226;&#171;&#186;1)            Assays performed described Liu et al. (2000). In brief, this <br /> 22&#194;&#176;C 8 h dark 21&#194;&#176;C 75% 90% humidity.                           procedure involved irradiating seedlings sprouted agar <br />                                                                                 plates, isolating seedling DNA 0 24 h irradiation, quanti- <br />                                                                                 fying 6-4 photoproducts present DNA lesion-specific radio- <br />                                                                                 immunoassay (Mitchell, 1996). <br /> UV Light Treatment <br />    Two- 3-week-old plants irradiated UV-C light (254 nm)        PCR Analysis UVH6 Alleles <br /> described Liu et al. (2001b). They incubated 3 d F40 <br /> GO gold fluorescent lights (Philips Lighting Company, Somerset, NJ)             Genomic DNA isolated leaves seedlings grown agar or <br /> avoid reversal UV photoproducts photoreactivation, mechanism            leaves older plants grown soil. To prepare seedlings, seeds were <br /> activated visible light, transferred standard growth condi-      surface sterilized, plated agar medium (containing 2.5 mm potassium <br /> tions 7 d. Sensitivity assessed extent leaf yellowing     phosphate [pH 5.5], 5 mm KNO3, 2 mm MgSO4, 2 mm Ca[NO3]2, 49 &#226;?&#174;m <br /> tissue death.                                                                   ethylenediamine-tetraacetic acid micronutrients, 5 g L&#226;&#171;&#186;1 Suc), incu- <br />  <br /> 1412                                                                                                                     Plant Physiol. Vol. 132, 2003 <br />                                                                                              Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br /> bated 4&#194;&#176;C, germinated growth chamber. DNA isolated          Britt AB, Chen JJ, Wykoff D, Mitchell D (1993) A UV-sensitive mutant of <br /> from seedlings described Klimyuk et al. (1993) older plants          Arabidopsis defective repair pyrimidine-pyrimidinone(6-4) <br /> as described Bell Ecker (1994). PCR reactions conducted using           dimers. Science 261: 1571&#226;&#8364;&#8220;1574 <br /> 42 cycles (30 s melting 94&#194;&#176;C, 1 min annealing 65&#194;&#176;C, 2-min        Clough SJ, Bent AF (1998) Floral dip: simplified method for <br /> extension 72&#194;&#176;C) experiments Ga-3 progeny 35 cycles (1 min           Agrobacterium-mediated transformation Arabidopsis thaliana. Plant J 16: <br /> of melting 95&#194;&#176;C, 1 min annealing 57&#194;&#176;C, 2.5-min extension            735&#226;&#8364;&#8220;743 <br /> 72&#194;&#176;C) experiments Ga-2 progeny. Primers used were: UVH6-F,              Costa RMA, Lima WC, Vogel CIG, Berra CM, Luche DD, Medina-Silva R, <br /> 5&#226;&#172;&#732;-ATCGTCACTGAATTCTCAGGC; UVH-6-R, 5&#226;&#172;&#732;-CATGACGGCTGTATC-                              Galhardo RS, Menck CFM, Oliveira VR (2001a) DNA repair-related <br /> TGCAAG; LB3, above; LB3-D, 5&#226;&#172;&#732;-ATTTCATAACCAATCTCGA-                          genes sugarcane expressed sequence tags (ESTs). Genet Mol Biol 24: <br /> TACAC. Reaction products detected 1.5% (w/v) agarose gels.                  131&#226;&#8364;&#8220;140 <br />                                                                                  Costa RMA, Morgante PG, Berra CM, Nakabashi M, Bruneau D, Bouchez <br />                                                                                     D, Sweder KS, Van Sluys MA, Menck CFM (2001b) The participation of <br /> RT-PCR Analysis Gene Expression                                                  AtXPB1, XPB/RAD25 homologue gene Arabidopsis thaliana, in <br />                                                                                     DNA repair plant development. Plant J 28: 385&#226;&#8364;&#8220;395 <br />    Total RNA isolated C10 wild-type plant tissues described       Boer J, Wit J, van Steeg H, Berg RJ, Morreau H, Visser P, Lehmann <br /> Liu et al. (2001b). RT-PCR conducted using OneStep RT-PCR Kit               AR, Duran M, Hoeijmakers JH, Weeda G (1998) A mouse model the <br /> (Qiagen Inc., Valencia, CA) 50 (UBQ10) 200 (UVH6) ng              basal transcription/DNA repair syndrome trichothiodystrophy. Mol Cell <br /> RNA. Either 26 (UBQ10) 40 (UVH6) PCR cycles used amplifi-               1: 981&#226;&#8364;&#8220;990 <br /> cation. Eight microliters 25-&#226;?&#174;L reaction subjected electro-       Boer J, Hoeijmakers JH (2000) Nucleotide excision repair human <br /> phoresis 1% 3% (w/v) agarose, identity RT-PCR                 syndromes. Carcinogenics 21: 453&#226;&#8364;&#8220;460 <br /> product confirmed sequencing. Primers UBQ10U60 (5&#226;&#172;&#732;-GA-                    Fidantsef AL, Mitchell DL, Britt AB (2000) The Arabidopsis UVH1 gene is <br /> CTCTCACCGGAAAGAAAT) UBQ10L573 (5&#226;&#172;&#732;-TTGTCTTGGATCTT-                               homolog yeast repair endonuclease RAD1. Plant Physiol 124: <br /> GGCTTTCA) used amplify UBQ10 gene. Primers XDU740                       579&#226;&#8364;&#8220;586 <br /> (5&#226;&#172;&#732;-TAAGGGCGTTTGGTAAGAATC) XDL2382 (5&#226;&#172;&#732;-CCCATTGTACC-                          Gallego F, Fleck O, Li A, Wyrzykowska J, Tinland B (2000) AtRAD1, a <br /> CGCCTTATCAT) used amplify UVH6 gene.                                    plant homologue human yeast nucleotide excision repair endo- <br />                                                                                     nucleases, involved dark repair UV damages recombination. <br />                                                                                     Plant J 21: 507&#226;&#8364;&#8220;518 <br /> Bioinformatics                                                                   Gorbalenya AE, Koonin EV, Donchenko AP, Blinov VM (1989) Two <br />                                                                                     related superfamilies putative helicases involved replication, recom- <br />    Multiple sequence alignments obtained using ClustalW pro-               bination, repair expression DNA RNA genomes. Nucleic <br /> gram default parameters (Higgins et al., 1996). The BLAST server               Acids Res 26: 4713&#226;&#8364;&#8220;4730 <br /> (http://www.ncbi.nlm.nih.gov/) used database similarity searches         Guzder SN, Qiu H, Sommers CH, Sung P, Prakash L, Prakash S (1994) <br /> (Altschul et al., 1997). Pair-wise alignments alignment AtXPD            DNA repair gene RAD3 S. cerevisiae essential transcription by <br /> cDNA genomic sequences obtained using LALIGN (Pearson                  RNA polymerase II. Nature 367: 91&#226;&#8364;&#8220;94 <br /> Miller, 1992), minor adjustments make exon/intron junctions          Harlow GR, Jenkins ME, Pittalwala TS, Mount DW (1994) Isolation of <br /> consistent consensus sequences Arabidopsis splice junctions.           uvh1, Arabidopsis mutant hypersensitive ultraviolet light and <br />                                                                                     ionizing radiation. Plant Cell 6: 227&#226;&#8364;&#8220;235 <br />                                                                                  Haughn GW, Somerville C (1986) Sulfonylurea-resistant mutants Arabi- <br />                                                                                     dopsis thaliana. Mol Gen Genet 204: 430&#226;&#8364;&#8220;434 <br /> Distribution Materials <br />                                                                                  Higgins DG, Thompson JD, Gibson TJ (1996) Using CLUSTAL multiple <br />    Upon request, novel materials described publication          sequence alignments. Methods Enzymol 266: 383&#226;&#8364;&#8220;402 <br /> made available timely manner noncommercial research purposes,           Jenkins ME, Harlow GR, Liu Z, Shotwell MA, Ma J, Mount DW (1995) <br /> subject requisite permission party owners           Radiation-sensitive mutants Arabidopsis thaliana. Genetics 140: 725&#226;&#8364;&#8220;732 <br /> parts material. Obtaining permissions responsibility   Jenkins ME, Suzuki TC, Mount DW (1997) Evidence heat ultra- <br /> the requestor.                                                                      violet radiation activate common stress-response program plants that <br />                                                                                     altered uvh6 mutant Arabidopsis thaliana. Plant Physiol 115: <br />                                                                                     1351&#226;&#8364;&#8220;1358 <br />                                                                                  Keriel A, Stary A, Sarasin A, Rochette-Egly C, Egly JM (2002) XPD muta- <br /> ACKNOWLEDGMENT                                                                      tions prevent TFIIH-dependent transactivation nuclear receptors and <br />    The authors gratefully acknowledge Torrey Mesa Research Institute            phosphorylation RARalpha. Cell 109: 125&#226;&#8364;&#8220;135 <br /> (San Diego) 825 B05 seeds carrying UVH6 T-DNA insertion.             Klimyuk VI, Carroll BJ, Thomas CM, Jones JD (1993) Alkali treatment for <br />                                                                                     rapid preparation plant material reliable PCR analysis. Plant J 3: <br /> Received February 8, 2003; returned revision March 30, 2003; accepted           493&#226;&#8364;&#8220;494 <br /> April 17, 2003.                                                                  Lehmann AR (2001) The xeroderma pigmentosum group D (XPD) gene: one <br />                                                                                     gene, functions, diseases. Genes Dev 15: 15&#226;&#8364;&#8220;23 <br />                                                                                  Li A, Schuermann D, Gallego F, Kovalchuk I, Tinland B (2002) Repair of <br /> LITERATURE CITED                                                                    damaged DNA Arabidopsis cell extract. Plant Cell 14: 263&#226;&#8364;&#8220;273 <br />                                                                                  Liu J, Akoulitchev S, Weber A, Ge H, Chuikov S, Libutti D, Wang XW, <br /> Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W,                    Conaway JW, Harris CC, Conaway RC et al. (2001a) Defective interplay <br />   Lipman DJ (1997) Gapped BLAST PSI-BLAST: new generation                  activators repressors TFIIH xeroderma pigmentosum. Cell <br />   protein database search programs. Nucleic Acids Res 25: 3389&#226;&#8364;&#8220;3402                 104: 353&#226;&#8364;&#8220;363 <br /> Arabidopsis Genome Initiative (2000) Analysis genome sequence          Liu Z, Hall JD, Mount DW (2001b) Arabidopsis UVH3 gene homolog of <br />   flowering plant Arabidopsis thaliana. Nature 408: 796&#226;&#8364;&#8220;815                     Saccharomyces cerevisiae RAD2 human XPG DNA repair genes. <br /> Balajee AS, Bohr VA (2000) Genomic heterogeneity nucleotide excision             Plant J 26: 329&#226;&#8364;&#8220;338 <br />   repair. Gene 250: 15&#226;&#8364;&#8220;30                                                        Liu Z, Hossain GS, Islas-Osuna MA, Mitchell DL, Mount DW (2000) <br /> Bamborg OL, Miller RA, Ojima K (1968) <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> requirements suspen-              Repair UV damage plants nucleotide excision repair: Arabidopsis <br />   sion cultures soybean root cells. Exp Cell Res 50: 151&#226;&#8364;&#8220;158                     UVH1 DNA repair gene homolog Saccharomyces cerevisiae Rad1. <br /> Batty DP, Wood RD (2000) Damage recognition nucleotide excision                  Plant J 21: 519&#226;&#8364;&#8220;528 <br />   repair DNA. Gene 241: 193&#226;&#8364;&#8220;204                                               McGregor WG (1999) DNA repair, DNA replication, UV mutagenesis. <br /> Bell CJ, Ecker JR (1994) Assignment 30 microsatellite loci linkage        J Invest Dermatol Symp Proc 4: 1&#226;&#8364;&#8220;5 <br />   map Arabidopsis. Genomics 19: 137&#226;&#8364;&#8220;144                                       Mitchell DL (1996) Radioimmunoassay DNA damaged ultraviolet <br /> Bootsma D, Hoeijmakers JH (1993) DNA repair: engagement tran-                  light. In GP Pfeifer, ed, Technologies Detection DNA Damage <br />   scription. Nature 363: 114&#226;&#8364;&#8220;115                                                    Mutations. Plenum, New York, pp 73&#226;&#8364;&#8220;83 <br />  <br /> Plant Physiol. Vol. 132, 2003                                                                                                                            1413 <br />   <br /> </body></html>