<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Gopyright 0 1992 Genetics Society America <br />  <br />  <br />  <br />        Zatheo, New Gene Involved Associative Learning Memory in <br />          Drosophila melanogaster,Identified P Element Mutagenesis <br />                                                       Susan Boynton TimTully&#226;&#8364;&#8482; <br />                                 Department Biology, Brandeis University, Waltham Massachusetts 02254 <br />                                                  Manuscript received January3, 1992 <br />                                                Accepted publication March 28, 1992 <br />  <br />                                                             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  Genetic dissection oflearning memoryin Drosophila limited bythe existence ethyl <br />               methanesulfonate (EMS)-induced mutationsin small number X-linked genes. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> remedy <br />               shortcoming, begun P element mutagenesis screen autosomal mutations that <br />               disrupt associative learning and/or memory. The generation &#226;&#8364;&#339;P-tagged mutant alleleswill expedite <br />               molecularcloningofthesenewgenes.           Here, behavior-geneticcharacterization of <br />               latheoP&#226;&#8364;&#8482;, recessive, hypomorphic <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>mutation</span> essential gene. latheoP&#226;&#8364;&#8482; flies perform poorly in <br />               olfactory avoidance conditioning experiments. This performance deficit attributed to <br />               abnormal olfactory acuity   shock reactivity-two task-relevant&#226;&#8364;&#339;peripheral&#226;&#8364;?behaviors used <br />               classical conditioning. Thus, latheo&#226;&#8364;&#339; mutation appears affect learning/memory specifi- <br />               cally. Consistent chromosomal situ localization P element insertion, deficienciesof the <br />               49F region second chromosome failed complement behavioral effect          latheoP&#226;&#8364;&#8482; <br />               mutation. Further complementation analyses betweenlatheoP&#226;&#8364;&#8482; lethal alleles, produced    excision <br />               latheo&#226;&#8364;&#339; insert byEMS ?-rays, 49F region mappedthe latheo mutation vital <br />               complementation group. Flies heterozygous latheoP&#226;&#8364;&#8482; oftwoEMS lethal alleles one <br />               lethal excision allele showthe behavioral deficits, demonstrating thatthe behavioral and <br />               lethal phenotypes co-map locus. <br />  <br />  <br />                                                                       DUDAIet al. 1985; TULLY <br /> M      UTANT analysis Drosophila allowed ge- <br />         netic dissection behaviors ranging leg <br /> shaking courtship(review: HALL 1985).Associative <br />                                                                                                  GERGEN   1986). Under- <br />                                                                       lying biochemical lesions, however, known only <br />                                                                       dunce rutabaga. Biochemical assays aneu- <br /> learning suchbehavior agenetic                       ploid flies originally suggested dunce gene <br /> approach amenable (review: TULLY       1987).                encoded structural gene CAMP-specific phos- <br /> Wild-type flies capable learning avoid                   phodiesterase rutabaga gene encoded a <br /> odorant previously paired electric                     structural gene Ca*+-stimulated adenylyl cyclase <br /> shock (QUINN, HARRIS     BENZER   1974; TULLY                 (BYERS,DAVISand KIGER 1981; KIGER et al. 1981; <br /> QUINN1985). Five ethylmethanesulfonate (EMS)-                         KAUVAR 1982; SHOTWELL    1983; LIVINGSTONE,     SZI- <br /> induced X-linked mutant strains consistently                     BERand QUINN 1984; DUDAIet a!. 1985; LIVINGSTONE <br /> failed learn remember instrumental      shock                1985). Subsequent molecular cloning genes <br /> avoidance task isolated (DUDAIet al. 1976;                  confirmed biochemical data and, inter- <br /> ACEVES-PINA    QUINN 1979; QUINN,SZIBERand                        estingly, shown gene products arepref- <br /> BOOKER1979). Thesemutant strainssubsequently                          erentially expressedin adult mushroombodies (CHEN, <br /> were shown perform poorly behavioral                      DENOME    DAVIS1986; LEVINet al. 1992; NIGH- <br /> assays thought involve aspects learning                    ORN, HEALY    DAVIS 199  1 ; P. L. HAN,L. R. LEVIN, <br /> (SIEGEL   HALL 1979;BOOKER      QUINN, 1981;                  R. R. REED R. L. DAVIS,unpublisheddata- <br /> DUERRand QUINN1982; FOLKERS1982;GAILEY,                               structures thought  involved olfactory learning <br /> JACKSON   SIEGEL 1984;   KYRIACOUand HALL 1984;                   (HEISENBERG    et al. 1985; MENZELet al. 1991). These <br /> WITTEKIND     SPATZ1988; CORFASand DUDAI                          results generally consistentwith modelin Aplysia <br /> 1989), suggesting gene products act cen-                   associative learning, cAMP cell-signal- <br /> trally general learning process.                                 ing pathway plays crucial role (KANDELet al. 1987). <br />    Behavioral deficits mutant strains dunce, ru-               Recentresultsfromtransgenic          Drosophila are <br /> tabaga amnesiac known derive single-                  consistent model: inhibition the <br /> gene effects (BYERS, DAVIS    KIGER 1981; LIVING- <br />                                                                       targets cAMP signaling-CAMP-dependent protein <br /> STONE, SZIBER    QUINN 1984; LIVINGSTONE     1985;                kinase-appears interfere associative learning <br />    &#226;&#8364;&#8482; <br />    Current address: Beckman Neuroscience Center, Cold Spring Harbor   (DRAIN,FOLKERS      QUINN199 1). <br /> Laboratory, Cold Spring Harbor, New York 11724.                          Historically, molecular geneticidentification      of <br /> Genetics 131: 655-672 uuly, 1992) <br />  656                                              S. Boynton T. Tully <br />  <br />  learning/memory genes hampered                  carrying white&#226;&#8364;?&#226;&#8364;? deletion allele (HAZELRIGG,    LEVISand <br /> technical problems associated genetic mapping              RUBIN1984). It &#226;&#8364;&#339;Cantonized&#226;&#8364;? crossing wll&#226;&#8364;? males to <br />                                                                 virgin Canton-S females. Heterozygous virginfemale F, <br /> and subsequent molecular cloning EMS-induced                 progeny mated Canton-S males, white male <br /> pointmutationsthat       affect behavioral pheno-           F2 progeny collected. This two-generation mating <br /> types. The dunce rutabaga genes cloned,             scheme twice, resulting generations of <br />  fact, pleiotropic effects muta-          outcrossing autosomes enerations out- <br /> tions identified. The effect dunce2 on <br />                                                                                                          Y <br />                                                                 crossing recombination (w&#226;&#8364;&#8482;l &#226;&#8364;&#8482;) X chromosome. <br />                                                                 The mutator P element stock 9.3, containing single <br /> female sterility facilitated recombinantmapping              w+ P element X chromosome (COEN1990). The +/ <br /> the gene, turn led identification          +; SbA2-3/TM6, Ubx stock contained stable, constitutive <br /> gene encoding phosphodiesterase     (BYERS, DAVISand          transposase source inserted chromomere 99B (ROBERT- <br /> KICER 198 1). Restriction fragment <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> polymor-              SON et d . 1988). Flies homozygous given 9.3 transpo- <br /> phisms used biochemical                sition autosome ultimately bred heterozy- <br />                                                                 gous flies using wI1l8; +/CyO; SbITM6B balancer stock. <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> clone gene (DAVISand DAVIDSON                     Excisions 9.3 P element inserts generated <br />  1984). Biochemical results dunce led search          using SPlCyO; SbA2-3/TM6, Ubx transposase source. <br /> for lesions CAMP cell-signaling pathway           Stocks homozygousfor particular excisions bred using <br /> other learning/memory mutants. In manner,              w&#226;&#8364;?&#226;&#8364;?; CyO; +/Xu strain. The Xu chromosome 2;3 <br /> rutabaga mutation discovered affect adenylyl             translocation, forces Cy0 second       +     third <br />                                                                 chromosomes cosegregate meiosis. <br /> cyclase. The biochemical phenotype used                The lutheoP1mutation mapped genetically crosses <br /> deletion-map mutation region 12E-F X              various y-ray- EMS-induced chromosomal deficien- <br /> chromosome. Finally, rutabaga gene cloned               cies point mutations. The DfTZR)OgblSM5 stock, carrying <br /> by using heterologous DNA probes bovine                 deficiency vestigul region (49E-F) second <br /> rat genes encoding adenylyl cyclase (KRUPINSKIet al.            chromosome, obtained National Drosophila <br />                                                                 Stock Center (Bloomington, Indiana). Deficiencystocks <br />  1989; FEINSTEIN  et al. 199l), followed chromosmal          DfTZR)uP/SM5 DfT2R)vgIo7/SM5were provided M. <br /> in situ hybridization identify Drosophila DNA              L. PARDUE,    stocks carrying lethal <br /> clonefrom 12E-F region(LEVIN et al. 1992).                  mutations detectable cytological ab- <br /> Pleiotropic effects extant learning/mem-          normalities (vr3.2, vr3.4,vr3.16,vr3.55,vr3.56,vr4.57, <br /> ory genes known (except possibly turnip;            vr5.48, vr6.6, vr6.7, vr6.35, vr6R6, vr9.1I , vr9.22, vr9R2, <br />                                                                 vr11.14, vr13.24, vr13.47, vr14.33and vr19.41; LASKO <br /> see CHOI et al. 1991), and, consequently, molecular             PARDUE 1988).     The lethal mutations vr9.23 vr9.43 <br /> cloning geneshas accomplished.            provided T. Wu. Each mutant stocks also <br />    To extend expedite genetic dissection,           carriedthe SM5 balancer chromosome. Stocks carrying <br /> we begun screen fornew, autosomal mutations              lethal mutations crossed inter se verify lethal <br /> affecting learning memory using P element muta-              complementation groups. Results experiments <br />                                                                 differed LASKO   PARDUE    (1988) (see Table <br /> genesis (KIDWELL1986; O&#226;&#8364;&#8482;KANEand GEHRINC1987;                    2). Stocks genotype w/w; Df/CyO generated by <br /> COOLEY,KELLY SPRADLINC1988; BIER et al.                     crossing males Df/SM5 stock w/w; Cy0 +/Xu <br />  1989). T o date, generated 1000              females. F, w/Y; Df/CyO progeny mated w/+; Of/ <br /> mutant strains homozygous single P element in-              Cy0 siblings produceDf/CyO F2 progeny homo- <br /> sertions autosomes. Thesemutantstrains                  zygous hemizygous w; flies interbred. <br />                                                                    All strains maintained 25&#226;&#8364;? 1618 hr light/ <br /> screened memory deficits hours olfac-           dark cycle lights 7:OO a.m. Flies raised a <br /> tory avoidance conditioning. In manner, muta-              food medium consisting 8.4   g/liter agar, 31.9 g/liter yeast <br /> tions affecting later memory phases,           (Nutrex #540), 94.2 g/liter dextrose, 8.7 g/liter potassium <br /> affecting acquisition early memory phases,            tartrate, 7 g/liter CaCI2, 76.1 g/liter cornmeal 2 glliter <br /> be isolated (see TULLY    1991). Below          Tegosept M mold inhibitor. This food medium other <br />                                                                 rearing conditions identical used TULLY <br /> behavioral geneticcharacterization latheof&#226;&#8364;&#8482;,             andQUINN (1985). We tried food media, in <br /> one mutations isolated mutagenesis.              particular medium containing molasses, obtained <br />                                                                 lower learning scores Canton-S flies (data shown). <br />             MATERIALS AND METHODS                                  Generation P element insertion excision strains: <br />                                                                 Mutant strains generated using P element transposition <br />    Stocks: Ample evidence exists polygenic variation     depicted Figure 1A. Females homozygousfor single <br /> olfactory responses associativelearning Drosophila    9.3 P element X chromosome mated males <br /> (ALCORTA    RUBIO1989; MONTE et ul. 1989; FUYAMA            carrying A2-3 P element transposase source. The 9.3 P <br /> 1976; GAILEY,     VILLELLA  TULLY  1991). As result,   element contains w+ gene sequence, conferring <br /> replaced chromosomes stocks          red eye color flies homozygous wII&#226;&#8364;&#339;. Transposase <br /> Canton-S (labeled &#226;&#8364;&#339;+&#226;&#8364;? figures text)         activity somatic tissue apparent inSbF1 males; their <br /> possible. Morphological phenotypes dominant markers          eyes mosaics red white tissue. Two of <br /> (Sb S p ) , balancer chromosomes (CyO, TM6B SM5)        dysgenic (mosaic) males mated 5-10 w(C-S4) <br /> and &#226;&#8364;&#339;cosegregator&#226;&#8364;? stock (Xu) described LINDSLEY       females vialincross 11.All male progeny this <br /> and ZIMM (1992).                                                second cross carried w&#226;&#8364;?&#226;&#8364;? X chromosome; red <br />    The following stocks used generation P        eyes carried 9.3 P element transposed <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> element insertion strains: w(C-S4) derived stock     X chromosome autosomes. In cross 111, a <br />                                                      New Gene Learning Drosophila                                           657 <br />                                                                           10 virgin latheo&quot; females cross 111. Crosses 1V V <br />                                                                           necessary establish white-eyed excisionstrains, <br />                                                                           homozygous independent excision <br />                                                                           events. Flies carrying excisions ofthe latheo&quot; P element first <br />                                                                           backcrossed mutant latheo&quot; flies minimize any <br />                                                                           genetic background differences excision strains <br />                                                                           parental mutant strain. For similar reasons, the <br />                                                                           cosegregator Xa used (in cross 111 males) force an <br />                                                                           unmarked chromosome latheo&quot; flies segregate <br />                                                                           latheo&quot; excision (second) chromosome. After <br />                                                                           homozygous excisionstrains maintained than <br />                                                                           generations, outcrossed latheo&quot; <br />                                                                           flies, outcrossed w(C-S4) for <br />                                                                           generations. Heterozygous excisionllatheo&quot;virgin <br />                                                                           female progeny mated latheo&quot; males. Finally, <br />                                                                           progeny crosswere mated en masse generate <br />                                                                           (white-eyed)flies homozygousfor excision allele. <br />                                                                              Lethal excisions identified white-eyed <br />                                                                           progeny resulted cross IV. The red-eyed maleprogeny <br />                                                                           mated w; CyO+/Xa females, w; lE/CyO; +/ <br />                                                                           +  progeny cross interbred establish each <br />                                                                           lethal excision line. Within lethal excision line, Cy+ <br />                                                                           flies observed. These lethal excision lines <br />                                                                           outcrossed. <br />                                                                              Associative learning: Flies trained tested with <br />                                                                           classical conditioning procedure TULLY       QUINN <br />                                                                           (1985). T o begin training, mixture roughly 50 mutant <br />                                                                           50 control flies,1-4days old, trapped inside a <br />                                                                           chamber contained electrifiable grid 90%of its <br />                                                                           inner surface. Flies exposed sequentially odors, <br />                                                                           3-octanol (OCT; ICN-K&amp;K Labs)and 4-methylcyclohexanol <br />                          9.3 <br />                                                                           (MCH; ICN-K&amp;K Labs),carried thetraining cham- <br />                      1 <br />                      2&apos;            x         v +                          ber current  air (750 ml/min). Flies exposed <br />                ?      w * +              7     X8                         60 sec conditioned stimulus (CS+;either OCT or <br />  <br />                            n                                              MCH), time received unconditioned <br />                                                                           stimulus (US), 12 1.25-sec pulses of60 V DC electric shock <br />                                                                           5-sec interpulse intervals. After presentation C S , <br />           IV   ?                                                          chamber flushed fresh air 30 sec. Then, <br />                                                                           flies exposed 60 sec second, control stimulus <br />                                                                           (CS-; MCH OCT), paired with <br />                                                                           electric shock. After presentation CS-, chamber <br />               , + x 7w 6v + <br />           &quot; 9 &quot;-                             gi3 <br />                                                     + <br />                                                         d                 flushed fresh air 30 sec. After training, <br />                                                                           flies tapped gently training chamber an <br />    FIGURE 1.-Generation mutant excision strainsvia P               elevator-like compartment thattransported tothe <br /> element transposition. A, P element transpositions generated <br />                                                                           choice point T-maze, relative (conditioned) <br /> using 9.3, P element containing wild-type copy white,          odor avoidance responses assayed exposing flies <br /> using stable transposase source A2-3. A 15-min retention          converging currents air (1500 ml/min thechoice <br /> assayed flies homozygous single P element insertions. B,           point)-one carrying OCT, MCH, opposite <br /> Excision strains generated mobilizing P element of <br />                                                                           arms T-maze. Flies allowed choose between <br /> the second chromosome A2-3. Flies carrying &quot;excision&quot;             C S CS- 120 sec, time theywere <br /> chromosome mated latheo&quot; mutants cross 111 minimize         trapped inside respective arms T-maze (by <br /> differences genetic backgrounds excision strains       sliding elevator register), anesthetized and <br /> latheoP&apos; mutant. Solid triangles refer latheoP&apos; P element          counted. Flies chose avoid CS+ ran T- <br /> insertion. Open triangles refer excisions latheoP&apos; P element.   maze arm containing CS-, flies chose avoid <br /> See text details.                                                     CS- ran intothearm         containing CS+. A small <br />                                                                           percentage flies (&lt; 5%) usually remained choice <br /> single &quot;transposition&quot; male, did carry trans-               point. <br /> posase source and,therefore, contained stable new P                        Relative concentrations OCT MCH adjusted <br /> element insertion, vial mated 5-10 w &quot; &apos; ~ ;             naive flies distributed 50:50 T- <br /> +/CyO; Sb/TM6B females vial. Strains homozygous                   maze. The absolute concentrations odors were <br /> a given, independent P element insert autosome                 relatively high; 80% naive flies avoided OCT <br /> were generated crosses IV V.                                       MCH given choice the <br />    Excisions latheo&quot; P element insert generated               odors fresh air (see RESULTS). All training testing <br /> as Figure 1B. The latheoP&apos; insertion mobilized               25&quot;and 70% relative humidity dim red light. <br /> using A2-3 transposase cross I, 2-3 dysgenic F,                For details conditioning procedure the <br /> male progeny mated 5-10 w; CyO; +/Xu females                   conditioning apparatus, TULLY     QUINN (1985). <br /> cross 11. One white-eyed, Cy&apos; male progeny (in                     Two groups flies conditioned complete <br /> 9.3 P element excised) vial mated 5-                 experiment. The CS+ OCT CS- MCH for <br />  658                                               S. Boynton T. Tully <br /> one group; CS+ MCH CS-was OCT                 exposed 1.25-secshockpulse          thecenter <br /> the second group. For reciprocal group, &#226;&#8364;&#339;probabil-       compartment opened, allowing flies escape <br /> ity correct&#226;&#8364;? (COR) calculated number flies          opposite, unshocked arm T-maze. There- <br /> avoiding CS+ divided total number flies      after, shock pulses continued delivered 5 sec, <br /> T-maze arms:                                                      flies allowed escape opposite T-maze <br />                                                                   arm. After 60 sec, thecenter compartment closed, <br />               COR = [CS&#226;&#8364;&#8482;]/([CS+]    + [CS-1).                     trapping flies respective arms. Flies center <br /> These COR values averaged, aver-           compartment unshocked T-maze arm consid- <br /> age normalized produce oneperformance index (PI):          ered tohave escapedelectric shock. Percent avoidance (PA) <br />                                                                   calculated number flies unshocked T- <br />                                                                   maze arm center compartment divided the <br />                                                                   total number flies (times 100). During test, ameno- <br />                                                                   tactic cues provided drawing air arms <br />                          + - 13 X 100. <br />           = [(COROCT CORMCH)                                      T-maze center compartment (1500 ml/ <br /> This index creates scores ranging 0 (no learning)         min). Again eliminated biases mean scores, <br /> 100 (perfect learning). Calculation PI manner     equal numbers groups flies shocked ineither the <br /> algebraically equivalent learning index (X) TULLY       left right armof T-maze. <br /> and QUINN(1985), number flies left             Unpaired control: Changes olfactory avoidance re- <br /> the choice point included calculation latter. Like   sponses occur whenflies exposed electric shock <br /> the learning index, PI measure associative learning   (TULLY   QUINN   1985). To assess type nonassocia- <br /> unaltered nonassociative changes odor avoidance         tive change, mixtures mutant control flies were <br /> that occur classical conditioning. [Performance        conditioned usual way (see above). During 2-min <br /> indices calculated data nonassociative         test trial, however, absolute-rather relative-odor <br /> control experiments TULLY      QUINN   (1985). As   avoidance responses assayed presenting flies with <br /> case learning indices, mean PIS nonassociative       CS- T-maze arm fresh air other. PIS <br /> control groups did differ significantly zero (T.         calculated odor individually asin olfactory <br /> TULLY,    unpublished).]                                          acuity experiments. <br />    Memory retention: Groups flieswere trained above,           Locomotoractivity: Locomotor activity measured <br /> then removed training chamber stored          using countercurrent phototaxis apparatus BENZER <br /> dark 15, 30, 60, 180 360 min test tubes containing      (1967)-in absence light. Under conditions, <br /> food medium. Seventy seconds usual 2-min test          tapping flies &#226;&#8364;&#339;start tube&#226;&#8364;?between trials produces an <br /> trial, flies transferred choice point T-       escape response, generally referred locomotor reactiv- <br /> maze. For retention intervals O min, flies weretransferred     ity, decays time spontaneous locomo- <br /> to T-maze 1.5 min training (3.5 min CS+       tor activity expressed (MEEHAN WILSON 1987). <br /> presentation). An equal number performance indices             Roughly 100 flies, halfmutant half control, loaded <br /> were collected retention interval given day      start tube whichwas aligned second, distal <br /> minimize spurious differences groups daily           tube. The flies tapped the start tube, <br /> variation functioning teaching machines         laid horizontally table. The flies were <br /> other uncontrolled variables.                                     allowed distribute start anddistal <br />    Olfactory acuity: Absolute odor avoidance responses            tube(s) 30 sec. The tubes shifted of <br /> were quantified exposing mixture 50 naive mutant          alignment, separating flies ran distal tube <br /> and 50naive control flies odor (either OCT             remained start tube. The flies the <br /> MCH) us. air T-maze. After 2 min, number            distal tube tapped second start tube, then <br /> flies arm T-maze counted, perform-       groups allowed run intodistal tubes for <br />                                                                   30 sec. This procedure separate flies distal start <br /> ance index calculated odor individually: <br />                                                                   tubes times, ultimatelyfractionating flies into <br />                          COROCT- 0.5 x                            groups based number times fly ran a <br />                PIOCT =                                            distal tube. The active flies ran distal tubes five <br />                             0.5 <br />                                                                   times received score five;the active flies stayed <br />                      = [2CORoo     - 11 X 100                     original start tube experiment and <br />                                                                   received score zero. Locomotor activity scoresof mutant <br />                          CORMCH - 0.5                             flies normalized mean score w(C-S4) control <br />               PIMCH <br />                   =                   x 100 <br />                             0.5                                   flies day minimize dailyvariations overall activity. <br />                                                                      Habituation:Based anolfactoryjump assay developed <br />                                    - <br />                      = [ ~ C O R M C H11 x 100 <br />                                                                   CARLSON    coworkers (MCKENNA     et al. 1989), have <br /> Olfactory acuity assayed measuring absolute odor           shown waning jump reflex repeated trials (S. <br /> avoidance responses range odor concentrations,          Koss T. TULLY,in preparation), whichshowsmany <br /> frompure, undiluted odoranttoa         three-log dilution      properties habituation defined THOMPSON            and <br /> heavy mineral oil (Fisher). The odor airwere presented        SPENCER    (1966). Singlemales housed chambers <br /> in left right arms T-maze, respectively,       consisting Lucite <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> inverted plastic test tube <br /> half experiments; reciprocal arrangement           (Falcon #20 17). A vacuum source connected base <br /> used half. In manner, environmental        chamber,drawing air (1000 ml/min) distilled <br /> cues produced small &#226;&#8364;&#339;side biases&#226;&#8364;?            water, 3-mm hole test tube, through <br /> eliminated algebraically mean PI group.          chamber nylon mesh-covered holesin base. <br />    Shock reactivity: The ability sense electric shock      At regular intervals, controlled, 3-way solenoid <br /> to escape quantified inserting electrifiable       valve switched stream &#226;&#8364;&#339;fresh&#226;&#8364;? air that <br /> grids arms T-maze. A mixture 100        bubbled distilled water through <br /> mutant control flies aspirated arm           10% benzaldehyde (Fluka) inheavy mineral oil (Fisher). <br />                                                New Gene Learning Drosophila                                          659 <br /> Each fly habituated exposing repeatedly 10%             KAL ROHLF 1981). Computations VAX <br /> benzaldehyde 4 sec 10-min intertrial intervals. Each          mainframe withSASstatisticalsoftware(SAS <br /> fly judged habituated failed jump             Institute, Inc., Cary, North Carolina). Since individual <br />                                                                                                                            jump <br /> during 4-sec odor presentation consecutive trials        responseswerescoredaszero        habituation <br /> (4 no-jumps). A fly&apos;s habituation score number            experiments, response decrement trials each <br /> trials needed obtain blockof 4 no-jumps (trials              genotype subjected log likelihood analysis covar- <br /> criterion). After reaching criterion, fly vortexed          iance (ANCOVA)on Macintosh I1 JMP 2.0 statistical <br /> in test tube 90 sec dishabituate jump reflex.         software (SAS Institute, Inc.). Mean habituation scores <br /> Dishabituation monitored 2 min fly reached               (which distributed approximately binomially)for mu- <br /> criterion exposing itone times 4-sec presenta-          tant control flies compared Student&apos;s t-test <br /> tion 10% benzaldehyde. The percentage flies               (SOKAL  ROHLF1981). <br /> failed jump test trial reflected average <br /> amount dishabituation eachgenotype. A level                                       RESULTS <br /> dishabituation lower wild-type resultfrom sensory <br /> or motor fatigue (see RESULTS).                                         One thousand sixteen homozygous        P element inser- <br />     Males collectedunder COe anesthesia day           tion strainsinitially screened memory      deficits <br /> eclosion stored food vials 25&quot; 2 <br /> days. Locomotor activityin 2-3-day-old males assessed            3 hr afterclassical conditioning generating onePI <br /> the day habituation experiments (see above). Only             eachstrain.Onehundredand               seventystrains <br /> males activity score 0 discarded. Usually 8          yielded PIS 50% 3-hr memory score <br /> or 16 flies habituated experiment. In             w(C-S4) control flies. Three PIS were <br /> case, equalnumbers mutant control flies weretested.           generated 170 strains obtain reliable <br /> These experiments run room temperature am- <br /> bient relativehumidity (although theaircurrents                 estimates 3-hr memory. Thirty-nine mutant strains <br /> bubbled water) 8 a.m. midnight.                  produced mean 3-hr memory scores 70% <br />     Geneticmapping: The latheo&quot; P element insert                 mean scorefor w(C-S4) control flies. <br /> localized situ polytene salivary chromosomesaccording             Olfactoryacuity(see     below) OCT MCH <br /> to ENCELSet al. (1 986), using DNA sequence fromthe white+ <br />                                                                      varied 39 putative mutant strains and, <br /> gene probe. For complementation testswith deficiency <br /> chromosomes, w/w; lat/latfemales crossedto w/Y; Of/             onaverage, lower thanthat w(C-S4) flies. <br /> Cy0 males, providing w/w (or Y ) ; lat/Df progeny.           Significantpolygenicvariabilityexists        olfactory <br /> These lat/Df flies mixed approximately equal num-            responsesinDrosophila(see          ALCORTAand RUBIO <br /> bers w/w (or Y); +/Offlies, generated crossing w/w;           1989; MONTE et al. 1989;FUYAMA1976). We also <br /> +/+ females w/V; Df/CyO males. Such mixed groups               obtained evidence recently variability ge- <br /> flies trained tested 15-min retention, <br /> olfactory acuity, shock reactivity locomotor reactivity.         netic background thatis specific associative learn- <br /> Females homozygous white viable latheor&apos; exci-           ing (GAILEY,VILLELLAand TULLY            1991). Thus, dif- <br /> sion allele, lE166 (see RESULTS), crossed white-        ferencesinolfactoryacuityamongour               P element <br /> eyed vr6.6/CyO males produce w/w (or y ) ; lE16b/vr6.6            strains mayhaveariseninfourdifferent             ways. (1) <br /> progeny. Immediate learning (retention time 0) assayed <br /> in flies. For experiments, +/vr6.6 control flies         Differences genetic backgrounds existed <br /> were trained tested separately, sincethey               parental stocks, 9.3, A2-3 w(C-S4), <br /> white-eyed.                                                          used generate mutant strains. (2) <br />    The latheo gene localized complementa-            Genetic background differences mutant strains <br /> tion crosses    10 lethal lathed&apos; excision alleles   arisen &quot;founder effect.&quot; Such sam- <br /> and y-ray- EMS-induced lethal mutations the <br /> 49F region (see above). For complementation cross,              pling variation existed homozygous mu- <br /> IEICyO flies crossed vr/CyO flies. The absence            tant strain originatedfrom single transposition male <br /> Cy&apos; progeny indicated 1E mutation failed com-            (see cross I11 Figure 1A). (3) A P element have <br /> plement vr mutation. Based results, latheor&apos;/           inserted gene involved olfaction. (4) Selection <br /> latheoP&apos; femaleswerecrosed 1E49/CyO, vr6.6/Cy0                (less fit) mutantphenotypes have <br /> vr6.35ICyO males verify mutant <br /> alleles l t h failed complement behavioral phe-          occurred (6TULLY        QUINN1985; TULLY           and <br /> notypeassociatedwith         latheo&quot; allele. Here again,         GERGEN 1986), producing      allelic differences &quot;mod- <br /> latheoP&apos;/vr6.6 latheoP&apos;/1E49flies trained tested         ifying&quot; genes mutant strains      stochastic <br /> with +/vr6.6 +/1E49 flies internal control           manner. To extent olfactory acuity affects <br /> behavioral experiments.                                              fitness, phenomenonmay contributed the <br />    Anatomy: Neural anatomy ofadult brains examined <br /> in 6-pm paraffin sections. Heads fixed         overnight     observed variation mutant strains. <br /> FAAG (forma1in:alcohol:acetic acid; 10:85:5 1% glu-                To alleviate differences  genetic background, <br /> teraldehyde), cleared, embedded paraplast (CAMPUS,            39 low scoring mutant strains     outcrossed for <br /> GROSSMAN     WHITE 1985). Sections stained             generations w(C-S4) flies (see MATERIAL AND <br /> hematoxylin examination (HUMASON           1972).             METHODS).Followingoutcrossing, strainswere <br />    Statistics: Since performance index itself average <br /> of percentages, sample performance indices distrib- <br />                                                                      tested 3-hr memory olfactory      acuity. <br /> uted normally. Thus, statisticalanalyses reported               Mean 3-hr memory scores 12 outcrossed mutant <br /> herein based analysis variance (ANOVA)               strains did differ inwild-type controls. <br /> unplanned pairwise comparisionsamong group means (so-                Mean3-hrmemoryscores           significantlylower <br />  660                                                         S. Boynton T. Tully <br />  <br />                                                                                                1 <br />                &apos;&quot;1                                                                        1&quot; <br />  <br />  <br />  <br />  <br />                      0   60   120   180   240   300   380                                          1     0.1     0.01   0.001 <br />  <br />                                TIME (min)                                                                 DILUTION <br />  <br />    FIGURE2.-Memory         retention mutant control flies. <br /> I lomozygous fatheo&quot; (open squares) wild-type w(C-S4) (closed <br /> circles) flies trained tested together. Afterward, two <br /> genotypes distinguished based difference eye color, <br /> and separate PIS calculated. Different groups flies <br /> were used retention interval. Mutant fatheo&apos;&quot; flies a <br /> consistent reduction learning memory retention inter- <br /> vals. n = 8 PIS genotype retention interval. <br />  <br /> than controls 18 remaining 27 outcrossed <br /> mutant strains, showed lower-than-nor- <br /> mal olfactory acuity. (Olfactory acuity strain, <br /> E,] 52, showed noimprovementafteroutcrossing;                                                      1     0.1     0.01   0.001 <br />  <br /> thus, olfactory mutant. We continue                                                       DILUTION <br /> outcross 18 strains.) Nine outcrossed mu-                       FIGURE3.-Olfactory acuity mutant control flies. Homo- <br /> tants maintained poor memory retention       whiledis-                    zygous latheo&quot; (solid bar) wild-type w(C-S4) (striped bar) flies <br /> playing normal olfactory acuity. These strains                    mixed tested olfactory avoidance over <br /> outcrossed w(C-S4) flies generation mini-                      range concentrations exposing T-maze to <br />                                                                           converging currents air-one delivered odorant a <br /> mize build-up modifiers genetic back-                       given concentration &quot;fresh&quot; air. The percentage of <br /> ground.                                                                   flies avoiding T-maze arm containing odor running into <br />    Here, report behavioral genetic char-                       opposite arm used calculate PI. Top, Avoidance <br /> acterization mutant strains. It                      responses three-log dilution methylcyclohexanol (MCH). <br /> carries single P element insertion solely re-                   Bottom, Avoidance responses three-log dilution OCT. <br />                                                                           Olfactory acuity mutants flies did differ significantly from <br /> sponsible mutant phenotype. Thus,                         wild-type controls. n = 4 PIS genotype each <br /> named newly defined gene Zatheo, Greek                       dilution. <br /> &quot;to cause person notto know&quot; (Benet&apos;sReader&apos;s <br /> Encyclopedia 1987)                                                        scores 59% w(C-S4) controls. <br />                                                                              Olfactory acuity: Low performance Zatheo&quot; flies <br /> Behavioral characterization Zutheo&apos;&quot; flies <br />                                                                           conditioning experiments sec- <br />     Memory retention: Different mixed populations                      ondary result reduced ability smell odor <br /> latheoP&apos; mutant w(C-S4) control flies tested                     cues. T o distinguish effect,mixed populations <br /> a t various retention intervals classical condition-                naive Zatheo&quot; w(C-S4) flieswere given choice <br /> ing. After test trial, genotypes sorted                  various concentrations OCT MCH us. <br /> according eye color, PIS calculated                       air T-maze. Figure 3 shows odoravoidance <br /> each genotype separately. Figure2 shows memory                            Zatheo&quot; control flies overa 3-log range of <br /> retention 6 hr training forZatheo&quot;                   dilutions. Seperate 2-way ANOVAs OCT for <br /> mutants andw(C-S4) control flies. Mean PIS lower                      MCH detected significant effects CONCENTRA- <br /> in mutant Zatheo&quot; flies retention intervals. A 2-                  TION ( F p ,241 = 27.05, P = 0.0001 Fls,241= 52.03, <br /> way ANOVA yielded significant effect GENO-                           P = 0.0001, respectively), significant effect of <br /> TYPE (FII, 841 = 118.8, P = O.OOOl), significant effect                 GENOTYPE (FIl,241 = 0.05, P = 0.8206 andFI1,       241 = <br /> of T I M E ( F I I , 8 4 ] = 44.9, P = 0.0001) aninsignif-            2.58, P = 0.1214, respectively) significant <br /> icant GENOTYPE X TIME interaction effect (F15,841                         GENOTYPE X CONCENTRATION interaction <br /> = 1.0, P = 0.3927). These results indicate                       (F13.241 = 3.43, P = 0.0330 FLs, 241 = 0.25, P = <br /> initial difference Zatheo&quot; w(C-S4)flies                       0.8630, respectively). The significant interaction ef- <br /> remained constant 6 hr memory                         fect O C T resultedfroma       higher mean PI for <br /> decay ( e . , memory decays curves parallel).                   latheo&quot; flies 0.001 dilution. Taken together, <br /> When averaged retention intervals, Zatheo&quot;                       results suggest olfactory acuity two <br />                                                        New Gene Learning Drosophila                                                 66 1 <br />  <br />  <br />                 &apos;&quot;1                                                                       &apos;&quot;1                              - <br />  <br />  <br />  <br />                   - .          .      .      .    .     . <br />                       0   10   20    30   40     50    60                                   n v eu n p r e d <br />                                    VOLTAGE <br />    FIGURE4.&quot;Shock reactivity mutant control flies. Homo- <br /> zygous lafheoP&apos; (open squares) wild-type w(C-S4) (solid circles) <br /> flies mixed tested shock reactivity over <br /> ;I range voltages placing arm T-maze, <br /> delivering 1-sec pulses ofelectric shock arm 60 sec while <br /> allowing flies escape opposite (unshocked) T-maze <br /> arm. Shock reactivity quantified PA offlies avoiding <br /> (escaping from) shocked T-maze arm. Here again, shock reac- <br /> tivity normal mutant flies. n = 8 PAS genotype at <br /> each voltage. <br />  <br /> odors used classical conditioning experiments                                            n v eu n p r e d <br /> not defective Zatheo&quot; flies.                                             FIGURE5.-Avoidance I unpaired odor (CS-) normal and <br />    Shock reactivity: Poor performance Zatheo&quot; flies                    mutant flies. Homozygous lathed&quot;&apos; (solid bars) wild-type w(C- <br />                                                                           S4) (striped bars) flies mixed trained the <br /> in conditioning result inability to <br />                                                                           usual classicalconditioning procedure. Immediately afterward, flies <br /> sense escape electric shock. T o assess                     exposed CS- (the odor NOT paired shock) us. air <br /> &quot;peripheral&quot; behavior, mixtures latheo&quot; w(C-                       T-maze. Top, Avoidance MCH. Bottom, Avoidance of <br /> S4) flies exposed T-maze electricshock                     OCT. The nonassociative reduction odor avoidance seen in <br /> pulses range voltages. Figure 4 shows                       unpaired groups; effect normal mutant flies. n = 4 PIS <br />                                                                           genotype treatment group. <br /> resulting PAS normal mutant flies different <br /> voltages. A 2-way ANOVA detected significant ef-                        GENOTYPE forMCH (FII,        nfil = 4.53, P = 0.0403), <br /> fect VOLTAGE ( F p .841 = 156.82, P = 0.000 l),                     andno significant GENOTYPE X TREATMENT <br /> significant effect GENOTYPE (F11,84]         = 2.64, P =               interaction OCT MCH (FII,      361 = 0.66, P <br /> 0. 108),    significant VOLTAGE X GENOTYPE                         = 0.421 5and FI,. 361 = 0.87, P = 0.3571, respectively). <br /> interaction ( F l s . R 4 1 = 0.43, P = 0.800). Thus, shock               These results corroborated observation TULLY <br /> reactivity appeared normal latheo&quot; flies.                        QUINN    (1985) explicitly unpaired proce- <br />    Unpaired control: T h e data presented sug-                      dure producesadecrement         olfactory avoidance <br /> gested memory deficit mutant Zatheo&quot; flies                    responses. More important, however, nonassocia- <br /> cannot attributed abnormal olfactory acuity                      tive change odor avoidance normal mutant <br /> shock reactivity. T h e possibility remained, however,                    latheo&quot; flies. <br /> that exposure electric shock odor during <br /> classical conditioning experiments altered                     Genetic characterization latheo&quot; mutation <br /> olfactory responses differentially Zatheo&quot; vs. control                    Excision lines: T h e Zatheo&quot; P element excised <br /> flies. T o control this, olfactory acuity meas-                   strains homozygous forindependent                     excision <br /> ured unpaired conditioning procedure           (see              events established breeding scheme out- <br /> MATERIALS AND METHODS). Mixtures latheo&quot;                           lined Figure 1B. Fifteen-minute memory retention <br /> w(C-S4) flies trained usual, afterward                    assessedin      13 randomly chosen homozygous- <br /> 120-sec test trials consisted choice                     viable excision strains; results shown Figure <br /> CS- arm T-maze air other.                        6 (stippled bars). Unplanned pairwise comparisons <br /> Figure5 shows olfactory avoidancebehavior                           mean scores revealed lines E 8 , E l 7, E 9 , <br /> such &quot;unpaired&quot; training, compared naive                       E 1 4 E166 significantly higher latheo&quot;, <br /> flies, normal andmutant genotypes. Separate 2-                         lines E 1 2 , E 1 3 , 8 4 , E 6 , E l l , E 1 6 , E 5 , E l , E 8 , <br /> way ANOVAs detected significant TREATMENT                               E 1 7 E 9 significantly lower w(C-S4). <br /> (naive vs. unpaired) effect O C T (Fr1.3sl     = 72.54,               Although statistical analysis permits lump <br /> P = 0.0001)andfor               MCH (Frl,ael = 65.41, P =                 excision lines intothese t w o groups (with <br /> O.OOOl), significant effect GENOTYPEfor                             someoverlap), distribution memory scores- <br /> O C T (FjlI,J61 = 3.86, P = 0.0572), significant effect                 ranging latheoP&apos;-like w(C-S4)-like-and addi- <br />                                                             S. Boynton T . T u l l y <br />  <br />  <br />  <br />  <br />    FIGURE fj.-Fifteen-minute     memory retention homozygous <br /> latheo&quot;&apos; P element excision lines. Flies set excision lines <br /> (stippled bars) maintained homozygotes               FIGURE 7.--&lt;:omplementation  analysis      15-nlin retention <br /> Inore generations and, result, accumulated genetic           various deficiencies EMS-, y-rav- P element excision- <br /> modifiers; flies set (striped bars) outcrossed       induced mutations uestigal region. Flies heterozygous for <br /> f <br /> xi  generations &quot;cantonized&quot; latheo&apos;&quot; flies. PIS group     latheo&quot;&apos; ugh, ug5h. urh.6 latheo&quot;&quot;&apos; showed significantly lower <br /> were normalized mean PI control flies tested         15-min retention compared control flies heterozygous <br /> clay; w s remove daily variation mean scores (and          + <br />                                                                            [w(C-S4)]and genetic variants. Homozygous latheo&apos;&quot; <br /> SEM). Outcrossing produced higher lowermean               PIS.    flies yielded 15-min retention scores lower +/+ <br /> indicating existence (a) differences genetic background          [w(C-S4)] wild-type controls. In contrast, 1 :&apos;-mil1 retention n flies <br /> among lines thesampling variation introduced select-         heterozygous lat&quot;&apos; ug107     +  similar corre- <br /> ing single excision males establish excision line and/or (b)       sponding controls. These data indicate memory deficit <br /> the accumuIation generations genetic modifiers latheo&quot;&apos;         maps 49Fd complementation group. n = 18 PIS lathto&quot;&apos; <br /> Iwmozygotes. n = 4 PIS lines.                                      homozygotes w(C-S4) flies; n = IO PIS latheo&apos;&quot;/ugb, +/ugh <br />                                                                            latheo&quot;/+ flies; n = 4 PIS genotypes. <br /> t.ional behavioral work E266 (see below)-suggest <br /> that latheo gene mutated variousviable,                      Cantonized w(C-S4) line. That this <br /> hypomorphic states. We recovered 10 lethal ex-                        effect genetic, environmental, <br /> cisions latheo&quot; P element insertion, sug-                     factors shown stability mean scores the <br /> gested latheo essential gene. These lethal                      w(C-S4) strain.Samplingvariation         genetic back- <br /> excisions usedin complementation analyses                        ground (for genes affecting   associative learning for <br /> other lethal mutations region verify                       affecting peripheral behaviors,    olfaction) <br /> essential nature latheo (see below).                                    mutagenesis produce &quot;dise- <br />    To examine effects genetic background,                        quilibrium.&quot; Furthermore, genetic back- <br /> assayed15-min retention twice 13                          ground phenotypically mutant excision line (such <br /> excisionlines. T h e firsttime generations                       latheo&quot;&quot;) equilibrated initially, selection against <br /> afterthe lines weremadehomozygous(Figure                    6,             mutant phenotype produce           accumula- <br /> striped bars).T h e excisions lines outcrossed                   tion modifiers (suppressors) generations (6 <br /> t o latheof&apos;&apos; flies generationsto equilibrate                TULLY    QUINN 1985), leading ob- <br /> genetic backgrounds latheoP&apos; strain                       served results. <br /> (see MATERIALS AND METHODS). Afteroutcrossing,                                Deletion mapping andlethal complementation: In <br /> homozygous latheo excision flies tested                     situ hybridization white+ DNA polytene chro- <br />  15-min retention (Figure 6, hatched bars). Each day                       mosomesfrom latheo&quot; flieslocalized latheo P <br /> of behavioral experiments, equal numbers                          elementinsertiontochromomeres4937-8onthe <br /> PIS generated 15 lines [ 13 exci-                     second chromosome (data shown). We deter- <br /> sionlines, w(C-S4) latheo&quot;&apos;]. Then, minimize                        minedthatthemean15-minretentionscore                     of <br /> variability memory scores daily fluctuations                     latheo&quot;/+ heterozygous flies significantly dif- <br /> in teaching machine function, PI       normalized                 ferent of+/+ [w(C-S4)] flies (see Figure 7). <br /> by dividing average PI genotypes                      These results indicated thelatheo&quot; mutation acts <br /> tested day. In general, outcrossing seemedto                       recessive wild-typeallele.Accordingly, <br /> affectmemoryscoresofseveralexcisionstrains.                                chromosomal deficienciesof region, Dj72R) <br /> Planned pairwise comparisonsofthenormalized                                vg&quot;, D f 2 R ) v ~and Df2R)vg&apos;&quot;7 (which will <br /> scoresindicatedthatoutcrossingsignificantly               in-              referred vgb, vg56and vgZO7), used <br /> creased mean memory score latheo&quot;&apos;       flies (F,,, 961            verify behavioral deficit oflatheoP&apos; flies co- <br /> = 4.21, P = 0.043), significantly decreased                       maps region theP element insert. Based on <br /> the mean score 1athe0&quot;~ flies (F11.961= 6.84, P =                       breakpointanalyses thesedeficiencies LASKO <br /> 0.0 10). In   cases, genetic background                  PARDUE (1988),vgI07 expected comple- <br /> of excision line apparently &quot;equilibrated&quot;                     ment latheoP&apos;, vgb vg56 not. Equal <br />                    Drosophilain Learning for <br />                                           New Gene                                                                  663 <br />  <br /> numbers flies heterozygous latheoP&#226;&#8364;&#8482; chro-              Based results, assayed 15-min retention <br /> mosome oneof deficiency chromosomes               flies heterozygous oneof lethal alleles (vr6.6, <br /> mixed control flies, heterozygous          vr6.35 1E49) latheo&#226;&#8364;&#339;. Control flies het- <br /> a wild-type w(C-S4)chromosome correspond-              erozygous lethal alleles lat&#226;&#8364;&#8482; [w(C- <br /> ing deficiency chromosome. These mixed groups                  S4)]. Figure 7 shows results flies carrying vr6.6 <br /> were trained tested 15-min retention; after-           carrying lE49. In cases, mean 15- <br /> ward, genotypes distinguished eye              min retention scores significantly lower het- <br /> color, separate performance indices calcu-            erozygous flies carrying latheoP1allele ( F [ I851 <br />                                                                                                                     , = <br /> lated.                                                         7.54, P = 0.0001 F [ l , 8 5 ] = 4.77, P = 0.0001, <br />    As shown Figure 7, mean 15-min retention scores          respectively). Similar results obtained flies <br /> were significantly lower laf&#226;&#8364;&#8482;lvgb laf&#226;&#8364;&#8482;lvg56             carrying vr6.35 allele: The mean 15 min PI (f <br /> flies +/Ofcontrols ( F [ 1 , 8 5 ] = 43.3, P =   SEM) vr6.35/laf1flies 13 + 4, for <br /> 0.0001 F [ l , 8 5 ] = 40.5, P = 0.0001, respectively).    vr6.35/+ controls 46 f 7 ( t p ] = 4.0, P = 0.008, n <br /> In contrast, mean 15-min retention scores lat/             = 4 genotype). Taken together, data <br /> vg107 flies did differ their+lug107 controls.         indicate clearly latheoP&#226;&#8364;&#8482;-induced behav- <br /> (The lower memory score +lug107 heterozygotes,              ioral  deficit <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> co-map the <br /> as compared  +/+       controls, suggests genetic  vr6  (49Fd)   complementation group. <br /> background effect ug107 deficiency stock           Behavior-genetic characterizationof severe <br /> dominant effect vg107 deficiency.) Thus,            latheo genotypes <br /> former deletions &#226;&#8364;&#339;uncovered&#226;&#8364;? latheoP&#226;&#8364;&#8482; muta- <br />                                                                   Retention excision/lethal heterozygotes: Figure <br /> tion, deletion did not. Interestingly, <br /> mean memory scores latheoP&#226;&#8364;&#732;/vgband latheoP&#226;&#8364;&#8482;/vg56 <br />                                                                7  shows mean 15-min retention scores offlies <br />                                                                heterozygous latheoP&#226;&#8364;&#8482; vgb, ug56, vr6.6 1E49 <br /> heterozygotes significantly lower latheo&#226;&#8364;? <br />                                                                significantly lower latheoP&#226;&#8364;&#8482;homo- <br /> homozygotes (Tukey-Kramerunplannedcompari- <br />                                                                zygotes (Tukey-Kramer unplanned comparisons; = <br /> sons, = 0.05), suggesting latheoP&#226;&#8364;&#8482; hypo- <br />                                                                0.05). This observation suggested strin- <br /> morphic allele essential gene-an observation <br />                                                                gent test identify EatheoP&#226;&#8364;&#8482; excision lines were <br /> consistent results behavioral experiments <br />                                                                true revertants. The mean 15-min retention score of <br /> on excision lines (see above). <br />                                                                homozygous excision line, E16b, did differ <br />    LASKOandPARDUE(1988)saturatedthe                   vestigal <br />                                                                significantly w(C-S4) controls (planned <br /> region EMS- y-ray-induced lethal mutations. <br />                                                                comparison; F [ I 481 <br />                                                                                   , = 1.10, P = 0.30), suggesting that <br /> Their lethal complementationgroups                49Fa-49Fh    latheo allele strain revertedto wild type <br /> map distal breakpoints vg107 vg56.          (see Figure 6). We reasoned, however, latheoE16&#226;&#8364;&#8482; <br /> Recovery lethal Eatheo excisions, then, allowed           weak hypomorph true revertant, <br /> finer localization latheo gene lethal comple-       E16blvr6.6 flies score significantly lower <br /> mentation analysis.We crossed             +/ur6.6 controls. On hand,         if <br /> extant lethal mutations inter se verify lethal          latheoE16&#226;&#8364;&#732; areversion,then         E16b/vr6.6 and <br /> complementation groups (see Table 1). In general,we            +/vr6.6 flies yield similar scores. In fact, we <br /> corroborated theresults LASKO PARDUE (1 988). themean learning score E16blvr6.6 flies <br /> Two discrepancies existed, however. T h e vr9.22           (44 f 2, n = 4) significantly lower +/ <br /> vr9r2 mutations appeared deletions,          vr6.6 flies (76 f 5, n = 4; t [ 6 ] = 5.96, P &lt; 0.001), <br /> point mutations, 117-6.7complemented             indicating latheo excision allele E16b weak <br /> vr6 mutations. Instead, vr6.7 failed          hypomorph. Most likely, then, noneof 13 excision <br /> complement vr8 vr4, failed comple-           lines testedfor 15-min retentionrepre- <br /> ment other, placing mutations          sented complete revertants. <br /> single complementation group.                                     One immediate interpretation results Fig- <br />    We crossed 10 lethal latheo excisions          ure 7 latheo&#226;&#8364;? hypomorphic mutation and <br /> to mutations extant               lower 15-min retention scores forthe laf&#226;&#8364;&#8482;/Df <br /> lethal complementation groups. Table 2 shows              genotypes result gene     dosage effect (see STEW- <br /> lethal latheo excision alleles lE101, 1E107 lE388          A R T MERRIAM1980). Another possibility, how- <br /> failed complement mutations                ever, severe genotypes have <br /> complementationgroups. Incontrast, lethal lathe0               producedpleiotropic effects task-relevant <br /> excision alleles lE34,1E49,1E311,lE344,lE352,lE357             &#226;&#8364;&#339;peripheral&#226;&#8364;? behaviors, af- <br /> and 16372 failed complement mutation(s)             fected preformance learning/memory experiments <br /> the vr6 (49Fd) complementationgroup. Figure 8 sum-             secondarily. Accordingly, addressed notion by <br /> marizes additions corrections thegenetic          assaying olfactory acuity shock reactivity laf&#226;&#8364;&#8482;/ <br /> organization vestigal region.                           Of flies. <br />  664                                                               S. Boynton T. Tully <br />                                                                                TABLE 1 <br />                             Complementation analysis EMS- 7-ray-induced lethal mutations vestigal region <br />  <br />                     ur9r2 ur6ur9.22   ur6.7 <br />                                       ur9         ur8.9                                                        U T I 1.14       UT13             ur5.48 <br />  <br />       vr14.33 <br />       vr4.57               +                                         +                 - <br />                          (6.6)                                    (9.1 1 ) <br />                                                                   (9.43) <br />       vr3                               +                            +                 -              - <br />                                       (3.55)                   (3.55X9.11)         (3.2)(3.16)   (3.2)(3.16) <br />                                                                (3.55X9.43)         (3.4)(3.56)   (3.4)(3.56) <br />       vr6                  -                                         +                 -              - <br />                      (6.6X6.35)                                (6.6X9.11)             (6.6)         (6.6) <br />                      (6.6X6r6)                                 (6.6X9.43)            (6.35)        (6.35) <br />                      (6r6X6.35)                                (6.35X9.11)            ( 6 4         ( 6 3 <br />                                                                (6r6X9.11) <br />       vr6.7                                                          +                 -              -           +                 +               + <br />                                                                   (9.43)                                                       (1 3.24) <br />                                                                   (9.1 1) <br />       vr8.9                                                          +                 - <br />                                                                   (9.1 1 ) <br />                                                                   (9.43) <br />       vr9                                                            -                 -              -           +                 +               + <br />                                                                (9.1 1 X9.23)         (9.1 1)       (9.1 1 )    (9.1 1)        1X13.24) <br />                                                                                                                             (9.1     (9.1               1) <br />                                                                (9.1 1 X9.43)                       (9.22)      (9.43)       (9.43X13.24) <br />                                                                                                                                  (9.43) <br />                                                                (9.23X9.43) <br />       vr9.22                                                                                                                        -               + <br />                                                                                                                               ( 1 3.24) <br />                                                                                                                               (13.47) <br />       vr9r2                                                                                                                         -               + <br />                                                                                                                               (13.24) <br />                                                                                                                               (13.47) <br />    &quot;+&quot; denotes complementation lethality: &quot;-&quot; denotes failure complement; blank spaces represent crosses done. Crosses with <br /> specific mutations complementation group indicated parentheses. Contrary expectations, mutation ur6.7 complements <br /> other mutations vr6 locus fails complement mutations vr4.57 vr8.9. Thus, assigned mutations to <br /> complementation group vr4. Mutations ur9.22 vr9r2 appear deletions spanning complementation groups (also Figure <br /> 8). <br />  <br />                                                                               TABLE 2 <br />  Complementationanalysis lethal lathed&apos; excision alleles EMS- 7-ray-inducedlethal mutations vestigal region <br />  <br />                                ur14         UT4           UT3                ur6        ur9        urll        url3           ur5           url 9 <br />  <br />             lE34                            +             +                  -           +          +            +             + <br />             E49                             +             +                  -           +          +            +             + <br />             lElOl                -          -             -                  -          -           -            -             +             + <br />             E107                 +          +             -                  -          +           +            +             +             + <br />             lE311                           +             +                  -          +           +            +             + <br />             E344                            +             +                  -          +           +            +             + <br />             1E352                           +             +                  -          +           +            +             + <br />             lE357                           +             +                  -          +           +            +             + <br />             lE372                           +             +-                 -          +           +            +             + <br />             LE388                +          +                                -          +           +            +             +             + <br />   &quot;+&quot;  denotes complementation lethality; &quot;-&quot; denotes failure complement; blank spaces represent crosses done. Based the <br /> cytological breakpoints DJr2)g107 DA;?)vg56, LASKOand PARDUE(1988) vr14 ur19 loci proximal distal, <br /> respectively. They able assign linear order remaining loci. Our results indicate, however, ur3 lies vr6 and <br /> that vr5 lies proximal ur19. The linear order remaining loci arbitrary arranged simply numerical order (also see <br /> Figure 8). <br />  <br />   Olfactory acuity laf&apos;/Df heterozygotes: Figure                                  eye color differences, PIS werecalculated <br /> 9, A B, shows olfactory acuity (undiluted MCH                                    separately. As case 15-min retention <br /> and OCT, respectively) ofthese genotypes.                                 scores, mean olfactory acuity scoresin flies heterozy- <br /> Once l f &apos; / D f +/D$ laf&apos;/vr6.6 +/                                  gous l f &apos; vg56, vgb, ~ ~ 6 or. 6E 4 9 were <br /> vr6.6 l t &apos; / + +/+ flies tested together.                             significantlylower controls, whilethose <br /> Afterward, genotypes distinguished based                                                                                 + <br />                                                                                      heterozygous l f &apos; vg107 [w(C-S4)] were <br />                                                             Learning <br />                                                           New Gene              Drosophila                                                 665 <br />  <br />      Db <br />  49 Da             Dc Ea Fb <br />                          Fa       Fd Ff Fg Fh Fc Fj Fk FI FmFo <br />                                                            Fn <br />      vr22 vrl vg 1(2)Cvr14 n 4 n3 n6 vr9 v r l l n13 vr5 M19W10n15vrl6 cos1 n16 <br />  <br />            vg107 <br />  <br />  <br />  <br />  <br />                               - <br />                               v 9 22 WYi? <br />  <br />  <br />                                I&#226;&#8218;&#172; 70&apos; <br />  <br />                            IE107ano ,E388 <br />                                  m <br />                                  /E34 <br />                                  /E49 <br />                                  /E311 <br />                                  IE3M <br />                                  /E352 <br />                                  /E357 <br />                                  IE372 <br />  <br />     FIGURE8.-Organization lethal complementation groups and <br /> chromoson1;d deficiencies n vestigal region. Lethal complemen- <br /> t;ltion groups (Da, Db, etc.), corresponding names mutations <br /> ( v r 2 2 , v r l , etc.), deleted regions (black bars; arrows indicate <br /> t h t deletions continue tested complementatiorl groups) of <br /> l)n2)ogl07, DflZ)vg56 Df2)vgB arranged generally asin <br /> ~ . A S K Oand PARDUE         (1988). Our complementation analyses with <br /> vrY.22 ; ~ n dvrYr2 shownthat they, too. deletions several <br /> vital genes. Complementation analyses latheo lethal exci- <br /> sions (IEIOI, 18107. etc.) indicate excisions l E I U I , lE1U7 and <br /> if&lt;3## delete vital genes n region. the <br /> I~emainingexcisionsonly affect complementations Fd <br /> group. These results, behavioral experiments <br /> (see RFSUI.TS) place latheo mutations complementation group <br /> Fd. I n LASKO          PARDUE    (1%&quot;   linear order vital genes <br /> Fa-Fh. distal breakpoints DJ22)VgIO7 DJZbg56, <br />                                                                                     FIGURE   9.-Complementation       analysis olfactory acuity <br /> W;IS determined. New results complementation anal- <br />                                                                                   various deficiencies EMS-. y-ray- P element excision- <br /> yses lethal latheo excisions indicate Fa adjacent <br />                                                                                   induced mutations vestigal region. Genotypes thesame as <br /> t o Fd anti Fc adjacent Fj (see Tables 1 2). <br />                                                                                   Figure 7. Top, Avoidance undiluted MCH. Bottom, <br />                                                                                   Avoidance undiluted OCT. Olfactory acuity aberrant n more <br /> not (planned comparisons, = 0.05). Interestingly,                               severe latheo genotypes. n = 4 PIS genotype. <br /> olfactory acuity laf1/1B49 flies, genetic <br /> background similar l f &apos; homozygotes, <br /> was significantly higher flies heterozygous                                        W <br />                                                                                                     &apos;&quot;1 <br />                                                                                                0 <br /> lat&quot;&apos; vg56, vg6 vr6.6 (unplanned comparisons,                                           z <br />                                                                                                a <br /> (Y = 0.05; DISCUSSION). <br />  <br />    Shock reactivity laf&apos;lDf heterozygotes: Results <br />                                                                                                5&gt; <br />                                                                                                a <br /> from shock reactivity experiments laf&apos;lvgb                                            !- <br />                                                                                                z <br />                                                                                                w <br /> +/vgb flies shown Figure 10. Here again, flies                                          0 <br />                                                                                                a <br />                                                                                                w <br /> of genotypes tested together; genotypes                                          n <br /> were distinguished differences eye <br /> color, separate PAS calculated. A two-way                                                        0   10   20   30   40   50   60 <br />  <br /> ANOVA indicated significant effect VOLTAGE                                                                     VOLTAGE <br /> (Fp,841 = 62.99, P = 0.000 I), nosignificant effects                            FIGURE    lO.-Shock reactivity heterozygous latheo&quot;/vgb flies <br />                                                                                   (open squares) +/vgb flies (solid circles). Behavior meas- <br /> of GENOTYPE (FI,,841 = 2.5, P = 0.16) inter- <br />                                                                                   ured n Figure 4. Shock reactivity 60 V aberrant more <br /> action VOLTAGEandGENOTYPE                  (F15,841=                           severe latheo genotypes. n = 8 genotype voltage. <br /> 0.66, P = 0.66). Planned pairwise comparisons (a = <br /> 0.05) mean PAS failed reveal significant                               +/vg6 flies 60 V (T = 0, N = 8, P &lt; <br /> differences genotypes partic-                              0.05)-which voltage used conditioning <br /> ular voltage. Visual inspection Figure 10, however,                            experiments. <br /> suggests trend-that shock reactivity laf&apos;lvgb flies <br /> might lower higher voltages +/vg6                              Locomotor activity lat andZat+ flies <br /> flies. In fact, Wilcoxon matched-pairs signed-ranks                               Poorperformance olfactory acuity shock <br /> nonparametric test, used mu-                               reactivity tests latheoP&apos;/Df latheoP&apos;/vr6.6 flies <br /> tant control flies tested together, indicated                            suggested genotypes suffer a <br /> that scores laf&apos;/vgb flies significantly lower                           general effect behavior. The obvious <br />  666                                                         S. Boynton T. Tully <br />  <br />                                                                            notion thatthesemutant      flies represent subviable <br />                                                T                           genotypic variants essential gene. <br />                                                                            Correlation retention olfactory acuity <br />                                                                            latheo excision lines <br />                                                                               T h e appearance strong effect olfactory <br />                                                                            acuity Lat&apos;&quot;/<span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>nuLl mutants</span> prompted a <br />                                                                            closer look olfactory acuity lat&quot;&apos;/Lat&quot;&apos; homozy- <br />                                                                            gotes. Even olfactory acuity scores did not <br />                                                                            differ significantly laf&apos;/Laf&apos; mutants +/ <br />                                                                            +  controls Figure 3, noticed mean values <br />                                                                            mutant flies lower wild-type flies <br />    I:IGURE I I .--Coml&apos;leme,lt;ltion analysis loconlotoractivity <br />                                                                            7 8 groups. Since l f &apos; / L f &apos;  +/+ <br />                                                                                                                             control <br /> ;Inlong various deficiencies EMS-. y-ray- P element excision-       flies mixed olfactory acuity tests, we <br /> intlucrd nlutations uestigul region. Activity scores for.50-100     able apply Wilcoxon matched-pairs signed- <br /> individualflieswereobtainedusing         Benzercountercurrentap-         ranksnonparametric test (SIEGEL1956) reassess <br />           dark. Scores ranged 0 5 . representing the <br /> I);I~;IIIIS                                                                differences genotypes. Even with <br /> rlunlber trials fly ran start tube adistal <br />                                                                            rigorous statistical test, olfactory acuity in <br /> ~ u b ewithin 30 sec. Activity scores normalized      mean <br /> score ur((X4) flies run day. 1.ocomotor activity         mutant flies significantly lower con- <br /> more severe lutheo mutantsalso aberrant. n = 4 experiments              trols 2 8 groups Figure 3. Thus, these <br /> ((lays) genotype.                                                 data yield evidence olfactory acuity lower <br />                                                                            normal latheo&quot; homozygotes. <br />                                                                               This conclusion reinforced data onolfactory <br /> component common behavioral assays                        acuity 13 (outcrossed) excision lines (Figure 6). <br /> locomotion, assayed locomotor activity                        PIS OCT MCH generated expos- <br /> dark Benzer countercurrentapparatus (see                            ing flies undiluted concentrations odor us. <br /> MATERIALS A N D METHODS).          Because flies repeat-               air (see MATERIALS AND METHODS). Although olfactory <br /> edly tapped start tube 30 sec                        acuity scores ranged PI = 83 97 OCT and <br /> this procedure, components locomotor                          PI = 71 92 MCH excision <br /> activity-spontaneous activity, reactivity stimu-                       lines-a rangelargerthantheobserveddifference <br /> lated activity (MEEHANand WILSON 1987)&quot;probably                            lai&quot; homozygotes +/+ flies-no signifi- <br /> are reflected measure. As usual Laf&apos;/Df, l f &apos; l                  correlation olfactory acuity 15-min <br /> ~ ~ 6or. laf&apos;/+ <br />               6       flies tested                retention detected (rr = 0.122, P = 0.65 forOCT; <br /> appropriate     controls <br />                                distinguished afterward                 r, = 0.449, P = 0.08 MCH). <br /> based eye color differences; separate          mean <br /> activity scores (which ranged 0 5) cal-                       Habituation jump reflex toolfactory cues in <br /> culated genotype. In addition, activity scores                    lat lat+ flies <br /> for mutantgenotype normalized tothe                                 A rigorous demonstration            potential ol- <br /> lllean Score ofw(C-S4) control flies day mini-                     factory acuity, shock reactivity locomotor deficits <br /> mize daily variations overall activity.                                 l f &apos; / l f &apos; flies responsible poor per- <br />     Results locomotor activity assays sum-                    formance learning tests accomplished study- <br /> marized Figure 1 1. N o significant differences                     ing habituation jump reflex (see MATERIALS <br /> detected        mean (normalized) scores l f &apos; l             AND METHODS). This form nonassociative learning <br /> l f &apos; , lat&quot;&apos;/+  +/+   [w(C-S4)] flies (unplanned com-                 produced exposing single male flies 4-sec <br /> parisons, alpha = 0.05), suggesting latheo&quot; homo-                      pulses 10% benzaldehdye constant current of <br /> zygotes normal locomotor activity. More impor-                         air. Initially, 100% flies jump response to <br /> tant, locomotoractivity fliesthat heterozygous                     noxious stimulus, percentage flies that <br /> for l f &apos; vg56, vgb, vr6.6 LE49 significantly                 jump decreases significantly repeated presenta- <br /> lower respective controls (FI,,391 =                    tions odor. Mutations affect learning show <br /> 100.2, P = 0.0001; F(1.391 = 74.5, P = 0.0005; Fll, 391                    slower-than-normal response decrement trials <br /> = 116.5,P=0.0001;andF11,391=             101.O,P=O.O001).                  normal levels dishabituation, muta- <br /> Mean scores thesemutant flies werequite low,                            tions affect sensory input motor outputsystems <br /> indicating sluggish locomotor activity. These datasug-                     faster-than-normal response decrement over <br /> gest poor performance genotypes                         trials abnormally low levels dishabituation (S. <br /> in olfactory acuity shock reactivity assays                      KOSS T. TULLY,           preparation). Thus, potential <br /> caused, part, poor locomotor activity.                      pleiotropic effects mutation sensory input or <br /> Such sluggish behavior consistent                         motor output systems learning produce op- <br />                                                          New Gene Drosophila <br />                                                                      Learning                                                  667 <br />                                                                               laf&apos;lvgb flies did (data shown). Moreover, only <br />                                                                               12% flies jumped response 100% <br />                                                                               benzaldehyde stimulus. Since flies genotype <br />                                                                               severely reducedlocomotor activity (see <br />                                                                               Figure 1 l), surmised locomotor ability an <br />                                                                               important component jumpreflex. <br />                                                                               Gross anatomyof adult brain normal latheo <br />                                                                               mutants <br />                     0                                     1 <br />                         o   5    IO    15     20   25     30                     Brain structure latheo&quot; latheo&apos;llvgb adult <br />                                       TRIAL                                   flies examined bothfrontalandhorizontal <br />     FIGURE12.-Habituation jumpreflex olfactory cues                araffin sections verify missing poorly <br />                                           (solid circles) flies. Individual <br /> l t z ~ / l [ l(open squares) laf&apos;/+                                  Formed brain structures root the <br /> flies genotype exposed repeatedly 4 - s pulses    ~        learning deficit latheo&quot; flies. Six-micrometer slices <br /> 10% benzaldehyde delivered constant current air. Habitua- <br /> tion shown decrement percentage flies jumping <br />                                                                               showed gross alteration mushroom bodies <br /> during 4-sec odor stimulus 30 trials. Mutant flies habit-            centralcomplex, structures believed tobe <br /> uated normally, suggesting sensory input motor output                involved learning (6 HEISENBERG         et al. 1985; <br /> systems normal (see text details). n 24 flies           NIGHORN, HEALY        DAVIS1991, other <br /> genotype. Within experiment, flies genotype            region brain (data shown). <br /> tested; experiment repeated times. SEM each <br /> trial genotype generated calculating percent jump <br /> tor experiment averaging percentages. <br />                                                                                                    DISCUSSION <br />  <br /> posite effects habituation assay-unlike                            We isolated, behavioral screen 10 16 <br /> classical conditioning experiments.                                           homozygous P element insertion mutants, new mu- <br />     lndividual l f &apos; / l f &apos; l f &apos; / + flies exposed            tation, latheo&quot;, disrupts associative learning. <br /> to 4-sec pulses 10% benzaldehyde 10 min                          Mutant flies, tested immediately discriminative <br /> 30 trials. The percentages flies jumpedwithin                         classical conditioning using odors conditioned stim- <br /> the 4-min odor presentation genotype                             uli electric shock unconditioned stimulus <br /> plotted function trials Figure 12. Log like-                       (negative reinforcement), average 69% <br /> lihood analysis ofjump responses (0 1) TRIAL                          wild-type levels conditioned avoidance. This <br /> as (third-order polynomial) covariate revealed                           &quot;learning&quot; deficit remains constant 6 <br /> significant effect GENOTYPE (X2,1]= 0.70, 0.1 &lt;                            hr memory retention (see Figure 2), pro- <br /> P &lt; 0.5) significant GENOTYPE X TRIAL                                  ducing mutant memory curvethat parallel that <br /> interaction (X2,1]    = 1.52, 0.1&lt; P &lt; 0.5). In separate                    wild-type flies. Such memory decay curve for <br /> experiment, determined habituation score                            latheo&quot; mutants qualitatively different those <br /> for individual flies number trials required                      dunce, rutabagaand amnesiac mutants, which <br /> reach ablock consecutive no-jump trials (trials                       thought disrupt aspects memory formation <br />  criterion). Once again, mean habituation scores (&amp;                        (see TULLY    et al. 1990). Instead, memory decay <br />  SEM) did differ l f &apos; / l f &apos; (15.4 &amp; 2.2, n                 curve latheoP&apos; mutants similar reported <br />  = 24) l f &apos; / + (13.1 f 1.8, n = 24) flies (t[4,=,]=                   flies heterozygous deletion Ddc (dopa <br />  0.81, P = 0.42). After reaching criterion, flies                       decarboxylase deficient)gene        [DJ&apos;2L)TW130] and <br />  vortexed 90 sec. Thirty seconds later,                         Ddc+ (TEMPEL,     LIVINGSTONE   QUINN  1984). Com- <br />  presented 4-sec stimulation                       bined fact Ddc gene involved with <br />  benzaldehyde assess levels dishabituation,                       synthesis (putative) monoamine neurotrans- <br />  measure sensory motor fatigue (S.                           mitters, dopamine serotonin (LIVINGSTONE        and <br />  KOSS T . TULLY,           preparation). Eighty-one per-               TEMPEL     1983), behavioral results led to <br />  cent 79% l f &apos; / l f &apos; l f &apos; / + flies, respec-             speculation learning (acquisition), than <br />  tively, jumped dishabituation test trial. [Usu-                       memory, disrupted Ddc mutants. Thus, latheo <br /> ally, flies donot spontaneous recovery                               involved acquisition processes. <br />  (jump), left undisturbed for2 min reaching                           The performance deficit homozygous latheof&apos; <br />  criterion.] Thus, data indicate sensory                      mutants does result   aberrant olfactory acuity <br /> or motor systems normal latheo&quot; homozygotes.                           (Figure 3), shock reactivity (Figure 4) locomotor <br />     Results quite different l f &apos; / D f heterozy-                  activity (Figure 10). T h e &quot;peripheral&quot; <br />  gotes. Whereas 100% l f &apos; / l f &apos; laf&quot;/+ flies                    behaviors measured task-relevant manner. <br /> jumped response 4-sec 10% benzaldehyde                                Behavioral responses OCT MCH (including <br />  stimulus trial (see Figure 12), 18%                     concentrations used conditioning) escape <br />  668                                        S. Boynton        T. Tully <br /> from electric shock (including 60 V) measured              dality. Different sensory modalities, however, use <br /> in T-maze classical conditioning apparatus.          different molecular mechanisms produce learning <br /> In addition, possibility poor performance           (HEISENBERG     WOLF 1984; HEISENBERG          et al. <br /> mutant flies (nonassociative) interaction          1985; c j STAUBLI,   FARADAY     LYNCH 1985).So, a <br /> between olfaction escape excluded results            mutation disrupts learningin sensory modal- <br /> from &#226;&#8364;&#339;unpaired&#226;&#8364;? control experiments (Figure 5).             ity necessarily disrupt learning another. <br /> Behavioral assays olfactory acuity shock reac-              Our solution study habituation jump <br /> tivity reliedonlocomotor             activity,   reflex latheoP&#226;&#8364;&#8482; mutants. With assay, waning of <br /> which obtained independent measure well.               jump reflex abnormally fast flies muta- <br /> Furthermore, did detect correlation            tions affect sensory motor (peripheral)systems, <br /> olfactory acuity memory scores flies homozy-             abnormally slow flies mutations that <br /> gous viable excision alleles latheo (Figure 6).          affect learning (S. KOSS T . TULLY,      prepara- <br /> Consequently, results lead believe             tion). Significantly, habituation dishabituation of <br /> latheo gene involved associative learning.            jump reflex latheoP&#226;&#8364;&#8482; mutants normal, again <br />    Although latheoP&#226;&#8364;&#8482; homozygotes performednor-                  indicating sensory motor systems normal <br /> mally battery behavioral tasks, able         mutants (Figure 12). Other forms habit- <br /> to detect aberrant peripheral effects severe            uation shown aberrant extant <br /> latheo mutants (Figures 9, 10 and11). The behavioral            learning/memory mutants (DUERRand QUINN            1982; <br /> data suggest general &#226;&#8364;&#339;pleiotropy&#226;&#8364;? slug-            WITTEKINDand SPATZ 1988; CORFASand DUDAI <br /> gishness subviable genotypes little func-            1989). Our results, thosefromtheun- <br /> tional latheo product. A similar observation re-            paired control experiment, suggest latheo in- <br /> ported thecacophony (cac) mutation, affects           volvedwith associative, ratherthan nonassociative, <br /> components Drosophila courtship song. Homo-              learning. <br /> zygous cac mutants slightly sluggish; cac/Dfflies,            Convergent genetic evidenceindicates P ele- <br /> however, showed severe effects loco-               ment insertion 49F7-8 solely responsible the <br /> motor acitivity (KULKARNIandHALL1987). More                     performance deficit lathed&#226;&#8364;&#8482;&#226;&#8364;&#8482; mutants. (1) A single P <br /> general pleiotropic effects reported         element insertion lathed&#226;&#8364;&#8482;&#226;&#8364;&#8482; flies mapped situ <br /> a majority mutationsaffecting behavior (GRIG-                cytological region 49F7-8of polytene salivary chro- <br /> LIATTI et al. 1973; HOMYK 1977; HOMYK              SHEP-    mosomes. Flies heterozygous latheoP&#226;&#8364;&#8482; chro- <br /> PARD 1977;HOMYK, SZIDONYA SUZUKI 1980;                      mosomal deficiencies region (vg56 vgb) <br /> KYRIACOUand HALL 1980; HOMYK andGRIGLIATTI                      15-min retention deficits; flies heterozygous for <br /> 1983; ROYDEN,        PIRROTTA JAN 1987; KULKARNI,           latheo&#226;&#8364;&#339;&#226;&#8364;&#8482; chromosomal deficiency to, but <br /> STEINLAUF     andHALL1988; MCKENNAet al. 1989;                  including, region (vg107) normal 15- <br /> WHEELER     et al. 1989; WOODWARD        et al. 1989; LILLY     min retention (Figure 7). (2) Althoughexcision the <br /> and CARLSON1990; KYRIACOUet al. 1990). Some                     P element produced revertant, range <br /> genetic variants optomotor blind (omb) gene,         hypomorphic excision alleles obtained (Figure <br /> instance, defective tergite coloration wing            6). (3) Lethal excision alleles recovered; <br /> development (BRUNNER        et al. 1992). Mutants        complementation mapping alleles <br /> pentagon ( p t g ) gene olfactory deficits, hyperpig-      EMS- y-ray-induced lethal alleles from <br /> mentation dorsal thorax female sterility             49F region localized latheo lethal comple- <br /> (HELFAND        CARLSON  1989). Even dunce memory           mentation group (Tables 1 2). Flies heterozygous <br /> mutants female sterile severe morpho-              latheoP&#226;&#8364;&#732; vr6.6 vr6.35-two EMS-induced <br /> logical defects (BYERS,DAVISand KIGER 1981; BEL-                lethal mutations complementation group 49Fd- <br /> LEN KIGER 1987; BELLEN        et al. 1987).                 decrement 15-min memory retention, do <br />    Further consideration potential pleiotropic           flies heterozygous <br />                                                                                       latheoP&#226;&#8364;&#8482;         (Figure 7 <br /> effects mutation olfactory acuity asso-          RESULTS). (4) All mutations latheo tested far <br /> ciative learning olfactory cues raises important          recessive (Figure 7). Taken together, results <br /> conceptual issue. Mutations affect olfactory acuity        strongly suggest ZatheoP&#226;&#8364;&#8482; hypomorphic muta- <br /> or shock reactivity produce low scores classical        tion essential gene. A possibility remains that <br /> conditioning tests, mutations affect asso-         latheo nonessential gene to,or near, vr6. <br /> ciative learning. Thus, use olfactory            For true, however, vr6.6 vr6.35 <br /> classical conditioning procedure     determine <br />                                                              EMS lethal mutations              lethal excision <br /> whether associative learning disrupted mu-           affect essential the <br /> tants aberrant peripheral behaviors. One poten-            nonessential genes, latheo&#226;&#8364;&#339; mutation would <br /> tial solution problem assay learning         affect nonessential one. We consider this <br /> task requires input different sensory mo-           possibility slight. <br />                                          New Gene Drosophila <br />                                                      Learning                                                            669 <br />  <br />     As case pleiotropic effects, mutations     lines generations study also <br />  vital genes identified behavioral       revealed ample evidence amelioration the <br />  experiments. Both cac omb genes, mentioned           mutant phenotype time. This phenomenon, too, <br />  above, vital. The cloned             noted studies learning/ <br />  shows (deduced)proteinsequence            homologies      memory mutants (TULLY         QUINN1985; TULLY <br />  DNA binding proteins (PFLUGFELDER          et al. 1992). A   GERCEN 1986)and apparently ac- <br /> flightless mutation cfli J ) turned viable        cumulation modifiers (heterogeneous) ge- <br />  allele ebryonic lethal abnormal visual         netic background homozygous mutant flies <br />  (elav) gene (HOMYK,ISONOand PAK 1980; CAMPOS,                fit (also BRUNNER    et al. 1992). As result, <br />  GROSSMAN     WHITE1985; HOMYK,        ISONOand PAK       necessary maintain mutant latheo alleles <br />   1985). Molecular cloning gene suggests      outcrossing heterozygotes &#226;&#8364;&#339;fresh&#226;&#8364;?wild-type flies <br />  RNA-binding protein ubiquitously           generation.In manner, obtainedthe <br />  specifically expressing neurons central        strongest mutant phenotype latheo homo- <br />  peripheral nervous (CAMPOSet al. 1987; RO-            zygotes bred behavioral experiments (see <br />  BINOW WHITE,1991). The technical <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>knockout</span> (tko)          Figure 6). <br />  gene identified screen &#226;&#8364;&#339;bang sensitive&#226;&#8364;?            Based onthe facts latheo second <br />  mutants encodes protein homologous ri-            chromosome learning/memory <br />  bosomal S12protein,thereby          suggesting     genes X-linked, latheo represents new gene in- <br />  gene affect mitochondrialfunction(ROYDEN,                volved associative learning Drosophila. Since <br />  PIRROTTA      JAN 1987). T h e optic ganglion reduced    memory decay curve qualitatively different from <br />  (ogre) gene originally identified screen        dunce rutabaga, encode <br />  mutationsaffecting visual behavior (LIPSHITZand              enzymatic components cAMP cell signaling <br />  KANKEL 1985), mutationin pentagon gene             pathway, speculate latheo affect differ- <br />  screenfor olfactory mutants (HELFAND          ent aspect underlying biological machinery. <br />  CARLSON      1989). No molecular insight avail-   This new gene P element&#226;&#8364;&#339;tagged,&#226;&#8364;? will <br />  able genes (but WATANABE            expedite cloning molecular identification. Fi- <br />  KANKEL1990).                                             nally, given vital nature, studies lethal genotypes <br />     A complicating factor mutant screen            shed light natureof latheo&#226;&#8364;&#8482;s role essential <br />  effects genetic background primarily           developmental processes. <br />  olfactory acuity associative learning. Polygenic <br />                                                                  We thank A. SOUTHGATE,    M. L. PARDUE   T. WU provid- <br />  variability olfactory acuity and/or associative          ing stocks carrying lethal mutations chromosomal deficiencies <br />  learning wild-type strains         second chromosome, L. RESTIFOand K. WHITEfor their <br />  reported (FUYAMA 1976;      ALCORTA     RUBIO1989;       expertise advice anatomical studies, CH. BRANDES,  J. M <br />  MONTEet al. 1989). In instances, magnitude          DURA,J. HALL,T. PREAT andR. MIHALEKfor critical readings of <br />                                                               manuscript M. DELVECCHIO    A. VILLELLA  technical <br />  background effects large            assistance. This work supported grant PO1 GM33205 from <br />  effects single-gene mutations (see TULLY              National Institutes Health,a Scholars Award the <br />  QUINN 1985;GAILEY,VILLELLA           TULLY      1991).   McKnight Foundation John    Merck Scholarship Biology <br />  Thus, sampling variation inherentto selecting            Developmental Disabilities Children (to T.T.) grant <br />                                                               F31 MHO9946 National Institute Mental Health (to <br />  single &#226;&#8364;&#339;transposition&#226;&#8364;? &#226;&#8364;&#339;excision&#226;&#8364;? males likely       S.B.). <br />  producedinterstrain variability forthese polygenic <br />  backgrounds homozygous P element inser- <br />  tion excision lines bred (see Figure 1). Out- <br />                                                                                  LITERATURE CITED <br />  crossing lines wild-type &#226;&#8364;&#339;parental&#226;&#8364;? strain <br />  [w(C-S4)] generations served equilibrate      ACEVES-PINA,  E. O., W. G. QUINN, 1979 Learning normal <br />  differences genetic background, leading             mutant Drosophila larvae. Science 206: 93-95. <br />  identification false-positive mutant lines     ALCORTA,  E., J. RUBIO, 1989 Intrapopulational variation of <br />                                                                  olfactory responses Drosophila melanogaster. Behav. Genet. <br />  behavioral screen (see RESULTS) andto few <br />                                                                  19: 285-300. <br />  dramatic changes 15-min retention scores           BELLEN,H. J., J. A. KIGER, 1987 Sexual hyperactivity and <br />  excision lines (Figure 6). [In contrast,tatheo&#226;&#8364;? mutants         reduced longevity dunce females Drosophila melanogaster. <br />  displayed mutant phenotypenow            Genetics 115: 153-160. <br />  20 generations outcrossing w(C-S4) flies          BELLEN,H. J., B. K. GREGORY,      C. L. OLSONand J. A. KIGER, <br />  (see MATERIALS AND METHODS).]This genetic phenom-               1987 Two Drosophila learning mutants, dunce rutabaga, <br /> enon prompted tobreed isogenic parental                   provide evidence maternal role cAMP embryogen- <br />                                                                  esis. Dev. Biol. 121: 432-444. <br /> stocks beforecontinuingtoscreenforany               new P     BENZER,S., 1967 Behavioral mutants Drosophila isolated by <br /> element insertion mutants.                                       countercurrent distribution. Proc. Natl. Acad. Sci. USA 58: <br />     Work homozygous latheo&#226;&#8364;&#732;&#226;&#8364;&#8482; latheo excision             1112-1119. <br />  670                                                   S. Boynton            T. Tully <br /> BIER,E., H. VAESSIN,   S. SHEPHERD,      K. LEE,K, MCCALL,        S. BARBEL,      adenylyl cyclase rat brain. Proc. Natl. Acad. Sci. USA 88: <br />     L. ACKERMAN,     R. CARRETTO,      T . UEMURA,       E. GRELL,L. Y. JAN        10173-10177. <br />    Y . N. JAN, I989 Searching <span id='am-396' about='xsp:pattern' typeof='owl:Thing'>pattern</span> mutation                  FOLKERS,   E., 1982 Visual learning memory Drosophila <br />    Drosophila genome P-lac2 vector. Gene Dev. 3:                       mt$anogaster wild-type C-S mutants dunce&apos;, amnesiac, <br />     1273-1287.                                                                    turnip, rutabaga. J. Insect Physiol. 28: 535-539. <br /> BOOKER,  R., W. G. QUINN,1981 Conditioning leg position                 FUYAMA,   Y . , 1976 Behavior genetics olfactory responses in <br />    normal mutant Drosophila. Proc. Natl. Acad, Sci. USA                    Drosophila I. Olfactometry strain differences Drosophila <br />    78: 3940-3944.                                                                 melanogaster. Behav. Genet. 6: 407-420. <br /> URUNNER,A., R. WOLF,G. 0. PFLUGFELDER,                    B. POECKand M.       GAILEY,D., F. R. JACKSON R.W. SIEGEL,1984 Conditioning <br />    HEISENBERG,      1992 Mutations proximal region                  mutations Drosophila melanogaster affect experience-de- <br />    optomotor-blind locus Drosophila melanogaster reveal gra-                 pendent behavioral modification courting males. Genetics <br />    dient neuroanatomical behavioral phenotypes. J. Neu-                    106: 6 13-623. <br />    rogenet. 8: 43-53.                                                          GAILEY,D. A,, A. VILLELLA     T. TULLY,1991 Reassessment <br /> EYERS,D., R. L. DAVISand J. A. KIGER,1981 Defect incyclic                         effect biological rhythm mutations learning in <br />    AMP phosphodiesterase thedunce mutation learning                     Drosophila melanogaster. J. Comp. Physiol. A 1 6 9 685-697. <br />    Drosophila melanogaster. Nature 289: 79-8 1.                             GRIGLIATTI,   T. A., L. HALL,R. ROSENBLUTH         D. T. SUZUKI, <br /> CAMPOS,  A. R., D. GROSSMANand K. WHITE,1985 Mutant alleles                        1973 Temperature sensitive mutations Drosophilamelano- <br />    locus elav Drosophilamelanogaster lead nervous                    gaster. XIV. A selection immobile adults. Mol. Gen. Genet. <br />    defects. A developmental-genetic analysis. J. Neuroge-                  1 2 0 107-1 14. <br />    net. 2: 197-218. <br />                                                                                HALL, J. C., 1985 Genetic analysis behavior insects, pp. 287- <br /> CAMPOS,A. R., D.R.ROSEN,             S. N. ROBINOWand K. WHITE,                   383 Comprehensive Insect Physiology, Biochemistry Phar- <br />     1987 Molecular analysis locus elav Drosophila mela- <br />                                                                                   macology, Vol. 9. edited G. A. KERKUTand L. I. GILBERT. <br />    nogaster: gene embryonic expression neural specific. <br />                                                                                   Pergamon Press, Oxford. <br />    EMBO J. 6: 425-431. <br />                                                                                HAZELRIGG, R.     T.,LEVISand G. M. RUBIN,1984 Transformation <br /> CHEN,C. N., S. DENoMEand R. L. DAVIS,1986 Molecular analysis <br />                                                                                   white locus DNA Drosophila: dosage compensation, reste <br />    cDNA clones corresponding genomic coding se- <br />                                                                                   interaction position effects. Cell 3 6 469-481. <br />    quences Drosophila dunce gene, structural gene for <br />                                                                                HEISENBERG,     M., R. WOLF, 1984 Visionin Drosophila, in <br />    CAMP-dependent phosphodiesterase. Proc. Natl. Acad. Sci. <br />                                                                                   Studies ofBrain Function X I I , edited V. BRAITENBERG.   Sprin- <br />    USA 83: 93 13-93 17. <br />                                                                                   ger Verlag, Berlin. <br /> CHOI,K.-W., R. SMITH,R. M. BURATOWSKI                    W.G. QUINN, <br />                                                                                HEISENBERG,M., A. BORST, S. WAGNERand D. BYERS, <br />    199 1 Deficient protein kinase C activity turnip, Drosophila <br />                                                                                   1985 Drosophila mushroom body mutants deficient in <br />    learning mutant. J. Biol. Chem. 266: 15999-16006. <br /> COEN,D., 1990 P element regulatory products enhance zeste&apos;                        olfactory learning. J. Neurogenet. 2: 1-30. <br />    repression P [ ~ h t e ~ ~ p &apos;transgene <br />                                          &apos; ~ &apos; ~ ~ ~ ] Drosophila melano-   HELFAND,  S. L., J. CARLSON,    1989 Isolation characteriza- <br />    gaster. Genetics 1 2 6 949-960.                                                tion olfactory mutant Drosophila chemically <br /> COOLEY,  L., R. KELLEYand A. SPRADLING, 1988Insertional mu-                       specific defect. Proc. Natl. Acad. Sci. USA 86: 2908-2912. <br />    tagenesis Drosophila genome single P elements.                  HOMYK,   T., 1977 Behavioral mutants Drosophila melanogaster. <br />    Science 2 3 9 1121-1 128.                                                      11. behavioral analysis focus mapping. Genetics 87: 105- <br /> CORFAS, G., andY . DUDAI,1989 Habituation dishabituation                      128. <br />    cleaning reflex normal mutant Drosophila. J. Neu-               HOMYK,   T., T . A. GRIGLIATTI,     1983 Behavioral mutants of <br />    rosci. 9: 56-62.                                                               Drosophila melanogaster. IV. analysis developmentally tem- <br /> DAVIS,R. L., N. DAVIDSON,             1984 Isolation Drosophila            perature-sensitive mutations affecting flight. Dev. Genet. 4 <br />    melanogaster dunce chromosomal region recombinational                      77-97. <br />    mapping dunce sequences restriction site polymorphisms              HOMYK,   T., K. ISONOand W. L. PAK,1985 Developmental and <br />    genetic markers. Mol. Cell. Biol. 4: 358-367.                               physiological analysis conditional mutation affecting pho- <br /> DRAIN,P., E. F O L K E R W.   S ~G.~ QUINN, <br />                                       ~           1991 CAMP-dependent             toreceptor optic lobe development Drosophila melano- <br />    protein kinase thedisruption learning transgenic flies.              gaster. J. Neurogenet. 2: 309-324. <br />    Neuron 6: 71-82.                                                            HOMYK,   T., D.E. SHEPPARD,1977 Behavioral mutants of <br /> DUDAI,Y., Y. N. JAN, D. BYERS,W. G. QUINN                   S. BENZER,        Drosophila melanogaster. I. Isolation mapping mutations <br />     1976 dunce, amutant Drosophila deficient learning.                      decrease flight ability. Genetics 87: 95-104. <br />    Proc. Natl. Acad. Sci. USA 73: 1684-1688.                                   HOMYK,T., j,SZIDONYA        D. T. SUZUKI,1980 Behavioral <br /> DUDAI,Y . , B. SHER,D. SEGALand Y . YOVELL, 1985 Defective                        mutants Drosophila melanogaster. 111. Isolation mapping <br />    responsiveness ofadenylate cyclase forskolin Drosoph-                mutations direct visual observations behavioral phe- <br />    ila memory mutant rutabaga. J. Neurogenet. 2: 356-380.                         notypes. Mol. Gen. Genet. 177: 553-565. <br /> DUERR, J. S., W. G. QUINN,1982 Three Drosophila mutations                  HUMASON,    G. L., 1972 AnimalTissuesandTechniques,        Ed. 3. W. <br />    block associative learning affect habituation sen-               H. Freeman, San Fransisco. <br />    sitization. Proc. Natl. Acad. Sci. USA 7 9 3646-3650.                       KANDEL,E. R.,M. KLEIN, &#226;&#8218;&#172;3. HOCHNER,        M. SHUSTER,  S. A. SIEGEL- <br /> ENGELS,W. R., C. R. PRESTON,        P. THOMPSON          W.B.EGGLES-          BAUM, R.D. HAWKINS,       D. L. GLANZMAN,      V. F. CASTELLUCCI <br />    TON, 1986 Insitu hybridization Drosophila salivary chro-                    T. W. ABRAMS,1987 Synaptic modulation learning: <br />    mosomes biotinylated DNA probes alkaline phospha-                     new insights synaptic transmission study be- <br />    tase. Focus 8: 6-8.                                                            havior, pp. 471-518 SynapticFunction, edited G. M. <br /> FEINSTEIN, P. F., K. A. SCHRADER,        H. A. BAKALYAR, W.J.         TANG,       EDELMAN,     W. E. GALL   W. M. COWAN.     John Wiley &amp; Sons, <br />    J. KRUPINSKI,   A. G. GILMAN     R. R. REED,1991 Molecular                 New York. <br />    cloning characterization Ca&apos;+/calmodulin insensitive               KAUVER,L. M., 1982 Defectivecyclic adenosine 3&apos;5&apos; monophos- <br />                                                      New Gene Learning Drosophila                                                   67 1 <br />    phate phosphodiesterase Drosophila memory mutant                      edited L. J. GOODMAN     R. C. FISHER.CAB International, <br />    dunce. J. Neurosci. 3: 1347-1 358.                                           London. <br /> KIDWELL.M. G., 1986 P-M mutagenesis, pp. 59-82 Drosophila:               MONTE,P., C. WOODWARD,          R. AYER,M. LILLY,H. SUNand J. <br />    APracticalApproach,     edited byD.B.    ROBERTS.IRL Press,                  CARLSON,1989 Characterization larval olfactory re- <br />    Oxford.                                                                      sponse Drosophila genetic basis. Behav. Genet. 1 9 <br /> KIGER,J. A.,R. L. DAVIS,H. K. SALTZ,T . FLETCHER         M.                 267-283. <br />    BOWLING,    1981 Genetic analysis cyclic nucleotide phos-             NIGHORN,A,, M. J. HEALYand R. L. DAVIS,1991 The cyclic <br />    phodiesterase Drosophila melanogaster. Adv. Cyclic Nucleo-                AMP phosphodiesterase encoded Drosophila dunce gene <br />    tide Res. 14: 273-288.                                                       concentrated mushroom body neuropil. Neuron 6: <br /> KRUPINSKI, J., F. COUSSEN,  H. A. BAKALYAR,   W. J. TANG,P. G.                  455-467. <br />    FEINSTEIN,   K. ORTH,C. SLAUGHTER, R.     R. REEDand A. G.               O&#226;&#8364;&#8482;KANE,C. J., W. J. GEHRING,1987 Detection situ of <br />    GILMAN,   1989 Adenylyl cyclase amino acid sequence: possible                genomic regulatory elements Drosophila. Proc. Natl. Acad. <br />    channel- transporter-like structure. Science 244: 1558-                   Sci. USA 8 4 9123-9127. <br />     1564.                                                                   PFLUGFELDER,     G. O., H. ROTH, B. POECK,S. KERSCHER,            H. <br /> KULKARNI,   S. J., J. C. HALL,1987 Behavioral cytogenetic               SCHWARZ,     B. JONSCHKER M. HEISENBERG,         1992 The <br />     analysis cacophony courtship song mutant interacting             l(1)optomotor-blind gene Drosophila melanogaster major <br />     genetic variants Drosophila melanogaster. Genetics 115: 461-             organizer optic lobe development. Isolation character- <br />     475.                                                                        ization gene. Proc. Natl. Acad. Sci. USA 8 9 1199- <br /> KULKARNI,   S. J., A. F. STEINLAUF     J. C. HALL,1988 The                   1203. <br />     <br /> </body></html>