<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>        YsxC, a Putative GTP-Binding Protein Essential for Growth of <br />                             Bacillus subtilis 168 <br />  <br />              YsxC is a member of a family of GTP-binding proteins carried by a diverse range of organisms <span id='am-7' about='protege:FROM' typeof='owl:Thing'>from</span> bacteria <br />           <span id='am-10' about='protege:TO' typeof='owl:Thing'>to</span> yeasts, plants, and humans. To resolve the issue of whether ysxC of Bacillus subtilis is essential for growth, <br />           we attempted to construct mutants in which ysxC was either inactivated or placed under the control of an <br />           inducible promoter. Viable mutants were obtained only in the latter case, and these were inducer dependent, <br />           demonstrating unambiguously that ysxC is an essential gene. <br />  <br />  <br />    GTP-binding proteins are frequently involved in regulatory                 ferase, and ydiE encodes a putative metalloprotease (probably <br /> pathways as ubiquitous molecular switches (6), operating at                   o-sialoglycoprotein endopeptidase). These conserved essen- <br /> various stages of growth and life cycles. The majority of these               tial bacterial genes with nonessential orthologs in yeast repre- <br /> proteins are guanine nucleotide-binding proteins, while others                sent potential targets for novel broad-spectrum antimicrobial <br /> use GTP as a substrate for phosphorylation and/or guanylation                 agents (6). <br /> (10). In Bacillus subtilis, GTP-binding proteins are involved in                 Construction of integrational mutations in ysxC. To deter- <br /> translation initiation and elongation; cell division; protein se-             mine whether ysxC is essential for growth of B. subtilis 168, <br /> cretion via the signal recognition particle pathway; biosynthesis             integrational mutants were constructed in which ysxC was ei- <br /> of flagella; the synthesis of adenylosuccinate, pyrimidine, folic             ther inactivated (<span id='am-168' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mutant) or placed under the control <br /> acid, and riboflavin; and the oxidation of thiophene (7). The                 of a tightly regulated Pspac promoter (23) (fusion mutant). In <br /> function of several small putative GTP-binding proteins (e.g.,                the case of the knockout mutant, a 303-bp internal fragment of <br /> Obg, Bex, EngA, YloQ, YsxC, and YyaF) is, as yet, unknown.                    ysxC (bp 2879356 to 2879054 [7]) was cloned into the integra- <br /> YloQ is essential for the growth of B. subtilis (2), while Bex has            tional plasmid pMUTIN4 (20), resulting in plasmid pYSXCK <br /> been shown to complement the Escherichia coli essential gene                  (Table 1). For the construction of the fusion mutant, a 164-bp <br /> era (EMBL accession no. U18532, available at http://www.embl                  fragment from the 5&#226;&#172;&#732; end of ysxC, incorporating the ribosome <br /> -heidelberg.de/srs5/). Obg is essential for both growth and                   binding site (RBS) and start codon of ysxC, was also cloned <br /> sporulation of B. subtilis (18, 21) and is required for the acti-             into pMUTIN4, resulting in plasmid pYSXCF (Table 1). <br /> vation of &#226;&#65533;&#180;B by stress (17).                                                     If ysxC is essential for the growth of B. subtilis 168, it <br />    The putative GTP-binding protein YsxC (also called OrfX                    should not be possible to isolate an integrational mutant with <br /> [13]) is encoded by a bicistronic operon that includes lonA (Fig.             pYSXCK, while integration of pYSXCF should lead to isopro- <br /> 1A), which encodes a cytosolic ATP-dependent serine endo-                     pyl-&#226;&#65533;&#164;-D-thiogalactopyranoside (IPTG)-dependent growth. Re- <br /> peptidase (22). ysxC is likely to be transcribed together with                peated but unsuccessful attempts were made to generate a <br /> lonA, since its start codon overlaps the lonA coding sequence                 pYSXCK-based knockout mutant, irrespective of the presence <br /> and no ysxC-specific promoter or transcriptional initiation site              of IPTG. In the case of the pYSXCF-based fusion mutant, <br /> has been detected. Furthermore, lonA and ysxC showed similar                  about 100 erythromycin-resistant (Emr) and lincomycin-resis- <br /> transcription patterns, including induction by heat and other                 tant (Lmr) transformants per &#226;&#65533;&#174;g of pYSXCF DNA were iso- <br /> stresses (13).                                                                lated in the presence of IPTG, while none were isolated in <br />    The insertional inactivation of ysxC was previously reported               its absence. These data indicate that ysxC is essential for <br /> to have no effect on growth, nor did it result in any demon-                  growth. The authenticity of the integration event in this <br /> strable <span id='am-12' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> (16). However, in a more recent genome-                     mutant (BFA2414) (Fig. 1A) was confirmed by PCR (Fig. 1B). To <br /> based approach designed to identify essential genes in E. coli,               ensure tight regulation of the Pspac promoter, BFA2414 was <br /> B. subtilis, and Saccharomyces cerevisiae, ysxC of B. subtilis was            transformed with plasmid p65, which provides multiple copies <br /> reported to be essential (2). In that study, which lacked exper-              of the E. coli lacI gene. <br /> imental data, of six genes of unknown function that were es-                     Expression of ysxC is essential for growth. The IPTG de- <br /> sential for the growth of E. coli, five orthologs were essential              pendence of BFA2414(p65) was confirmed by growing the <br /> for B. subtilis and one was essential for S. cerevisiae. Three of             organism to exponential phase (optical density at 600 nm <br /> the five essential B. subtilis genes, namely, obg, yloQ, and ysxC,            [OD600] &#226;&#171;&#189; &#226;&#172;&#402;0.3) in Luria-Bertani (LB) medium (15) contain- <br /> encode putative GTP-binding proteins, while yrrA (now called                  ing 1 mM IPTG and 0.3 &#226;&#65533;&#174;g of erythromycin and 10 &#226;&#65533;&#174;g of <br /> trmU, SWISS-PROT accession no. O35020) encodes a putative                     kanamycin per ml. The cells were washed twice with pre- <br /> tRNA, (5-methylaminomethyl-2-thiouridylate)-methyltrans-                      warmed LB medium and diluted 10&#226;&#171;&#186;5-fold into prewarmed LB <br />                                                                               media containing erythromycin, kanamycin, and a range of <br />   * Corresponding author. Mailing address: Department of Microbi- <br />                                                                               concentrations of IPTG from 0 to 1 mM. As shown in Fig. 2, <br /> ology and Immunology, The Medical School, University of Newcastle             growth was eventually observed in each of the cultures. How- <br /> upon Tyne, Framlington Pl., Newcastle upon Tyne, NE2 4HH, United              ever, the time at which growth was first observed was increas- <br /> Kingdom. Phone: 44 191 222 7708. Fax: 44 191 222 7736. E-mail: Colin          ingly delayed with decreasing IPTG concentrations. In the case <br /> .Harwood@ncl.ac.uk.                                                           of the culture with 0.1 mM IPTG, growth was delayed by <br />  <br />                                                                        6819 <br />  6820         NOTES                                                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Construction of a fusion B. subtilis mutant of ysxC. (A) Schematic representation of the ysxC region of BFA2414 after integration of pYSXCF, which is a <br /> pMUTIN4-based integration plasmid. Filled thick arrows indicate structural genes, and putative &#226;&#65533;&#179;-independent terminators are shown as stem-loop structures. Two <br /> promoters upstream of lonA (PlonA) are marked with fine broken arrows. Striped boxes show the tandem duplication of the RBS and 5&#226;&#172;&#732; end of ysxC. ysxC&#226;&#172;&#732; is the 5&#226;&#172;&#732; <br /> end of ysxC. Plasmid pMUTIN4 is shown as a thick line. The lacZ reporter gene, lacI, and ampicillin resistance (Apr) and Emr genes are marked with fine arrows, and <br /> promoter Pspac is marked with a fine broken arrow above that of pMUTIN4. The region in pMUTIN4 used for replication in E. coli is labeled &#226;&#8364;&#339;ori,&#226;&#8364;&#65533; and three <br /> terminators (t1t2t0) upstream of Pspac are indicated as a stem-loop structure. The arrows below the genes indicate the location and orientation of the primers, while <br /> the dashed lines indicate their expected PCR products. The positions of the primers specific for B. subtilis 168 in respect to the entire genome (7, 11) are as follows: <br /> DS-REV, 2879355 to 2879372; FS-REV, 2879433 to 2879450; FS-FOR, 2879596 to 2879580; and US-FOR, 2879970 to 2879954. The 5&#226;&#172;&#732; ends of the forward (FOR) <br /> primers included a 10-bp linker with a HindIII restriction site, while the reverse (REV) primers included a 9-bp linker with a BamHI site. Positions of the primers <br /> specific for pMUTIN4 were as follows: MUT-FOR, 147 to 165, and MUT-REV, 361 to 379. The numbers above the dashed lines correspond to the lanes of the agarose <br /> electrophoresis gel shown in panel B. (B) Diagnostic PCR confirming the correct integration of pYSXCF into ysxC in mutant BFA2414. Lane 1, 100-bp ladder <br /> (Amersham Pharmacia Biotech Inc., Little Chalfont, United Kingdom). PCR was performed with BFA2414 (lanes 2 through 7) or B. subtilis 168 (lane 8) chromosomal <br /> DNA. The primers (and expected product lengths) were as follows: lane 2, FS-FOR and DS-REV (261 bp); lane 3, MUT-FOR and DS-REV (399 bp); lane 4, US-FOR <br /> and FS-REV (557 bp); lane 5, US-FOR and MUT-REV (611 bp); lane 6, MUT-FOR and MUT-REV (no PCR product expected since these primers are oriented away <br /> from each other); lane 7, US-FOR and DS-REV (no PCR product expected since the polymerization reaction time of 50 s is too short for the synthesis of the 9,409-bp <br /> fragment); and lane 8, US-FOR and DS-REV (635 bp). <br />  <br />  <br /> approximately 2 h with respect to the culture containing 1 mM                          sidase production increased during exponential phase, reach- <br /> IPTG, while growth of the culture with no added IPTG was                               ing a peak of ca. 22 nmol of ONP/min/OD600 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> at or about <br /> delayed by more than 20 h. During exponential phase, the                               the transition between exponential and stationary growth <br /> mean generation times of cultures with smaller amounts (&#226;&#177;&#8226;0.1                           phase; thereafter, the values declined. <br /> mM) or no IPTG was increased to &#226;&#172;&#402;50 min, compared to &#226;&#172;&#402;30                                  To determine whether the delayed growth and decreased <br /> min for the culture with 1 mM IPTG.                                                    growth rate of BFA2414(p65) at the lower (i.e., &#226;&#177;&#8226;0.1 mM) <br />   The transcription of ysxC was monitored (9) by fusing the                            IPTG concentrations (Fig. 2) were due to overgrowth by sup- <br /> spoVG-lacZ reporter gene of pMUTIN4 to its native promoter                             pressor mutants, stationary-phase samples were plated to de- <br /> (Fig. 1A). Irrespective of the IPTG concentration, &#226;&#65533;&#164;-galacto-                          termine the ratio of the IPTG-independent colonies (putative <br />  <br />  <br />                                                            TABLE 1. Bacterial strains and plasmids <br />       Strain or plasmid                                                 Relevant characteristic(s)a                                               Source or reference <br />  <br /> B. subtilis <br />   168                                 trpC2                                                                                                       1 <br />   BFA2414                             trpC2 ysxC::pYSXCF Emr                                                                                      This study (Fig. 1) <br />   BFA2414(p65)                        trpC2 ysxC::pYSXCF p65 Emr Kmr                                                                              This study <br />   BFA2414SUP(p65)                     trpC2 ysxC::pYSXCF p65 Emr Kmr, constitutive Pspac promoter                                                 This study <br />  <br /> Plasmids <br />   pMUTIN4                             Apr Emr spoVG-lacZ Pspac (8.61 kb)                                                                          20 <br />   pYSXCK                              pMUTIN4 containing a 303-bp internal fragment of ysxC (8.913 kb)                                            This study <br />   pYSXCF                              pMUTIN4 containing a 164-bp fragment with RBS and 5&#226;&#172;&#732; end of ysxC (8.774 kb)                                 This study <br />   p65                                 pUB110-based, Kmr PpenP::lacI (4.962 kb)                                                                    S. D. Ehrlich <br />   a <br />       Kmr, kanamycin resistance. <br />  VOL. 182, 2000                                                                                                                 NOTES       6821 <br />  <br />  <br />                                                                              to be taken when selective pressure is applied to this control- <br />                                                                              lable promoter system. <br />                                                                                 YsxC protein family. YsxC is a member of a family of small <br />                                                                              GTP-binding proteins that are carried by a diverse range of <br />                                                                              organisms from bacteria to yeast, plants, and humans. An anal- <br />                                                                              ysis of 29 members of the YsxC protein family (Fig. 3) shows <br />                                                                              that they can be classified into four distinct phylogenetic groups. <br />                                                                              Group I includes YsxC orthologs from gram-negative bacteria <br />                                                                              and Mycoplasma species, group II includes orthologs from <br />                                                                              gram-positive bacteria (e.g., Bacillus, Clostridium, and Staphy- <br />                                                                              lococcus), group III includes orthologs from the Archaea, and <br />                                                                              group IV includes orthologs from S. cerevisiae, Homo sapiens, <br />                                                                              Arabidopsis thaliana, and, interestingly, two hyperthermophilic <br />                                                                              bacteria, Aquifex aeolicus and Thermotoga maritima. <br />                                                                                 In group II organisms, ysxC is located downstream of lonA or <br />                                                                              clpX, both of which <span id='am-8' about='Thesaurus:code' typeof='owl:Thing'>code</span> for class III ATP-dependent heat <br />    FIG. 2. OD600 of BFA2414(p65) in LB media with the following concentra- <br /> tions of IPTG: 1 mM (}), 0.1 mM (&#226;&#8211;&#160;), 0.01 mM (&#197;&#8217;), 0.001 mM (F), 0.0001 mM    shock proteases. In the case of Bacillus and Brevibacillus, ysxC <br /> (&#226;&#8364;&#353;), and 0 mM (E).                                                           is located downstream of lonA, and its start codon overlaps the <br />                                                                              3&#226;&#172;&#732; end of the lonA coding sequence by one nucleotide. In <br />                                                                              Clostridium difficile and Clostridium acetobutylicum, the coding <br /> suppressor mutants) to total CFU. There was a marked differ-                 sequence of ysxC overlaps the 3&#226;&#172;&#732; end of lonA by eight nucleo- <br /> ence in the plating efficiencies of the various cultures. In the             tides. In Streptococcus spp., Enterococcus faecalis, Staphylococ- <br /> case of the culture with 1 mM IPTG, the plating efficiency was               cus aureus, and Lactococcus lactis, ysxC is in a putative operon <br /> (4.9 &#226;&#171;&#190; 1.2) &#226;&#171;&#187; 10&#226;&#171;&#186;7. The number of the IPTG-independent                       with clpX. In L. lactis the start codon of ysxC overlaps the 3&#226;&#172;&#732; <br /> colonies increased as the IPTG concentration decreased, and                  end of the clpX coding sequence by one nucleotide. In the <br /> cultures with 0.001 mM IPTG or less exhibited a plating effi-                other three phylogenetic groups (I, III, and IV), lonA or clpX <br /> ciency close to 1. These data suggest that the growth in the                 is located at a distal site on the chromosome with respect to <br /> latter cultures was due to the accumulation of derivatives with              ysxC. Linkage between an ATP-dependent protease and ysxC <br /> suppressor mutations.                                                        was also observed in Pyrococcus horikoshii from group III, in <br />    To determine the location of putative suppressor muta-                    which the ysxC ortholog, PH0200, is located 59 bp downstream <br /> tion(s), chromosomal DNA from IPTG-independent colonies                      of a gene encoding a putative regulatory subunit of the ATP- <br /> [BFA2414SUP(p65)] was used to transform B. subtilis 168. In                  dependent 26S protease. In addition to YsxC orthologs in eu- <br /> each case, similar numbers (ca. 103/&#226;&#65533;&#174;g of DNA) of Emr Lmr                    karyotes, two bacterial homologs from A. aeolicus and T. ma- <br /> transformants were observed, irrespective of the presence of                 ritima belong to group IV. A. aeolicus is one of the earliest <br /> IPTG. In contrast, when chromosomal DNA from the IPTG-                       diverging and most thermophilic bacteria known, and as a <br /> dependent BFA2414 was used, transformants (ca. 9 &#226;&#171;&#187; 102/&#226;&#65533;&#174;g                    chemolithoautotroph, it can grow on hydrogen, oxygen, carbon <br /> of DNA) were obtained only in the presence of IPTG. These                    dioxide, and mineral salts (5). T. maritima is one of the deepest <br /> results suggested that the observed suppression was linked to                and most slowly evolving lineages in the Eubacteria. Although <br /> the integrated pYSXCF.                                                       the core of T. maritima may be eubacterial, almost one quarter <br />    One possibility was that the suppressor <span id='am-90' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span>(s) occurred              of the genome is archaeal in nature (12). A. thaliana has at <br /> in the &#226;&#8364;&#339;<span id='am-3' about='Thesaurus:OID' typeof='owl:Thing'>oid</span>&#226;&#8364;&#65533; lac operator (14) of the Pspac promoter, leading to             least two homologs of YsxC, one 219 and the other 318 amino <br /> its constitutive expression. The <span id='am-4' about='Thesaurus:OID' typeof='owl:Thing'>oid</span>, or ideal, lac operator has             acid residues in <span id='am-9' about='xsp:length' typeof='owl:Thing'>length</span>, encoded on chromosomes II and V, <br /> perfect symmetry and a 10-fold-higher affinity for the Lac re-               respectively (8). The length of the smaller homolog is in the <br /> pressor than the native lac operator. Consequently, the native               range of prokaryotic YsxC proteins (190 to 219 amino acids). <br /> lac operator associated with the Pspac promoter of integra-                  The length of the longer YsxC homolog is similar to that of the <br /> tional vector pMUTIN2 was replaced by the <span id='am-324' about='Thesaurus:OID' typeof='owl:Thing'>oid</span> operator in                    YsxC orthologs from S. cerevisiae and H. sapiens. With respect <br /> pMUTIN4 to reduce the noninduced level of expression of this                 to YsxC of B. subtilis, these proteins have a 100- to 110-amino- <br /> promoter (20). The Pspac promoter regions from several IPTG-                 acid extension at their amino terminus. Their N termini, which <br /> independent BFA2414SUP(p65) mutants were PCR amplified                       are highly conserved in S. cerevisiae and H. sapiens but not in <br /> using primers MUT-FOR and DS-REV (Fig. 1A). Sequencing                       A. thaliana, showed no homology to any other bacterial and <br /> of the PCR products revealed that all of the BFA2414SUP                      archaeal proteins. In the case of A. thaliana, the N terminus <br /> (p65) mutants contained a single C3T transition at nucleotide                contained a putative transmembrane helix (19) between amino <br /> 10 of the <span id='am-325' about='Thesaurus:OID' typeof='owl:Thing'>oid</span> lac operator. An identical <span id='am-326' about='xsp:base' typeof='owl:Thing'>base</span> pair substitution              acid residues 11 and 29 and the protein is currently the only <br /> at the same nucleotide of the native lac operator has been                   putative membrane-bound member of the YsxC family. <br /> shown to decrease its affinity for the Lac repressor by 96% and                 No homologs of ysxC were observed on the complete ge- <br /> to generate a constitutive phenotype (3). We therefore con-                  nome sequences of Borrelia burgdorferi, Chlamydia trachomatis, <br /> cluded that the observed IPTG-independent growth is due to                   Chlamydia pneumoniae, Deinococcus radiodurans, Mycobacte- <br /> the selection of clones with a mutation in the integrated                    rium tubercolosis, and Treponema pallidum using databases at <br /> pYSXCF that severely reduces the capacity of the lac opera-                  The Institute for Genomic Research (TIGR; http://www.tigr <br /> tor upstream of the functional ysxC gene to bind the lactose                 .org/tdb) and the Pasteur Institute (http://genolist.pasteur.fr). <br /> repressor. Since a single spontaneous mutation in the lac op-                   An alignment of YsxC and 28 homologs (data not shown) <br /> erator of pMUTIN4 can result in the loss of the IPTG depen-                  revealed four regions of conservation: (i) (G/R)X(S/T)N(V/A) <br /> dence of target gene expression, this may lead to an underes-                GKS(S/T), a putative GTP-binding motif located toward the <br /> timation of the number of essential genes when a gene fusion                 amino terminus; (ii) PGXTXXX(N/I), located 15 to 23 resi- <br /> rather than a <span id='am-246' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-247' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>gene knockout</span> strategy is initially used to isolate            dues downstream of the first region; (iii) DXPG(Y/F)G(Y/F), <br /> such mutants. Our results indicated that appropriate care needs              a second putative GTP-binding motif located 10 to 20 residues <br />  6822       NOTES                                                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Radial phylogenetic tree of the YsxC protein family. Multiple-sequence alignment and phylogenetic analysis were performed using the web site of the <br /> Institut National de la Recherche Agronomique (http://www.toulouse.inra.fr/multalin.html; see reference 4). The symbol comparison table was Blosum62, the gap <br /> weight was 12, and the gap length weight was 2. The root of the tree is marked with a triangle. Proteins with SWISS-PROT accession numbers (in parentheses) are <br /> as follows: B. subtilis YsxC (P38424), E. coli YihA (P24253), Haemophilus influenzae HI1118 (P46453), Helicobacter pylori HP1567 (O26087), Mycoplasma genitalium <br /> MG335 (P47577), M. pneumoniae MP359 (P75303), Methanococcus jannaschii MJ0320 (Q57768), Rickettsia prowazekii RP102 (Q9ZE46), A. aeolicus AQ_1815 <br /> (O67679), and Caulobacter crescentus CgpA (Q9ZG89). Proteins with <span id='am-5' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> are as follows: Archaeoglobus fulgidus AF1326 (AAB89919), <br /> P. horikoshii PH0200 (BAA29269), T. maritima TM1466 (AAD36534), L. lactis (AAF63739), Neisseria meningitidis NMB1806 (AAF42143), Zymomonas mobilis CgpA <br /> (AAD56911), S. cerevisiae Ydr336wp (AAB64772), H. sapiens HSPC135 (AAF29099), A. thaliana 219 is At2g22870 (AAC32434) with a length of 219 amino acid <br /> residues, and A. thaliana 318 is F15N18_70 (CAB87708) with a length of 318 amino acid residues. YsxC orthologs of B. anthracis, B. stearothermophilus, C. aceto- <br /> butylicum, C. difficile, E. faecalis, S. aureus, S. pneumoniae, and S. pyogenes were obtained from unfinished genome sequencing projects at TIGR, the Sanger Centre, <br /> Genome Therapeutics Corporation, and the University of Oklahoma. In B. stearothermophilus, the 130 amino acids at the N terminus of the YsxC homolog were <br /> deduced from sequence contig 552. In B. brevis, the 162 amino acids at the N terminus were deduced from the GenBank sequence (D00863) after inserting one <br /> nucleotide between positions 2745 and 2746 (16). PAM, percent accepted mutation. <br />  <br />  <br />  <br />  <br /> downstream of the second region; and (iv) KXDK, located 56                                                          <br /> to 74 residues downstream of the third motif. The second motif                         <br /> is shorter in S. cerevisiae (GXTXXXN), and only the threonine                           <br /> residue is conserved in the case of H. sapiens.                                       <br />                                                                                          <br />    We were not able to generate a viable mutant of ysxC with                             <br /> pYSXCK, which, after integration into the B. subtilis chromo-                            <br /> some, generates a strain carrying a YsxC protein that is trun-                        <br /> cated at its C terminus by just 23 amino acids. Since this                                <br />                                                                                          <br /> protein includes the four conserved motifs described above,                           <br /> this indicates that the highly charged (5 K residues, 2 E resi-                          <br /> dues, 1 D residue, 1 R residue, and a serine dyad) C terminus                        <br /> is essential for function.                                                               <br />   We thank S. D. Ehrlich for the gift of plasmids pMUTIN4 and p65.                     <br /> We acknowledge the release of preliminary sequence data for the                           <br /> YsxC orthologs in Bacillus anthracis, Bacillus stearothermophilus, C.                 <br /> acetobutylicum, C. difficile, E. faecalis, S. aureus, Streptococcus pneu-               <br /> moniae, and Streptococcus pyogenes, which were obtained using the                        <br /> TIGR website (http://www.tigr.org) from unfinished genome sequenc- <br />                                                                                          <br /> ing projects at TIGR, the Sanger Centre, Genome Therapeutics Cor-                        <br /> poration, and the University of Oklahoma.                                               <br />   The work was funded by the European Commission (BIO4-CT95-                              <br /> 0278). <br /> </body></html>