<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Plant Cell, Vol. 18, 1213&#226;&#8364;&#8220;1225, May 2006, www.plantcell.org &#194;&#170; 2006 American Society Plant Biologists <br />  <br />  <br />  <br />  <br /> Arabidopsis Separase AESP Is Essential Embryo <br /> Development Release Cohesin Meiosis <br />  <br /> Zhe Liu Christopher A. Makaroff1 <br /> Department Chemistry Biochemistry, Miami University, Oxford, Ohio 45056 <br />  <br />  <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> investigate sister chromatid cohesion released chromosomes plants, isolated Arabidopsis <br /> thaliana homolog separase (AESP) investigated role somatic meiotic cells. AESP similar separase proteins <br /> identified organisms contains additional structural motifs. The characterization Arabidopsis T-DNA <br /> insertion alleles AESP demonstrated essential gene. Seeds homozygous T-DNA insertions AESP exhibited <br /> embryo arrest globular stage. The endosperm exhibited weak titan-like <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>phenotype</span>. Transgenic plants expressing <br /> AESP <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) meiosis-specific DMC1 promoter exhibited alterations chromosome segregation <br /> during meiosis I II resulted polyads containing microspores. Consistent predicted role in <br /> the release sister chromatid cohesion, immunolocalization studies showed removal SYN1 chromosome <br /> arms centromeres inhibited RNAi mutants. However, release SYN1 diplotene occurred normally, <br /> indicating process independent AESP. Therefore, results demonstrate AESP plays essential role <br /> in embryo development provide direct evidence AESP required removal cohesin meiotic chromo- <br /> somes. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                 2001). Separase belongs CD clan Cys proteases (Barrett <br />                                                                              Rawlings, 2001; Goyal, 2001). Before metaphase-to- <br /> The faithful transmission chromosomes mitosis                  anaphase transition, separase protease activity inhibited by <br /> meiosis essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span> reproduction eukar-             securin. At onset anaphase, securin degraded the <br /> yotic organisms. A critical aspect chromosome segregation              anaphase-promoting complex/cyclosome (APC/C), freeing sep- <br /> sister chromatid cohesion, required proper                  arase, cleaves SCC1, facilitate release cohesion <br /> attachment chromosomes spindle faithful                    chromosome separation (Cohen-Fix et al., 1996; Ciosk et al., <br /> segregation sister chromatids opposite poles cell               1998; Zou et al., 1999). <br /> during anaphase (reviewed Uhlmann, 2004; Nasmyth                         In budding fission yeast, chromosomal cohesin <br /> Haering, 2005). Sister chromatid cohesion mediated             complexes remain chromosomes cleaved by <br /> a complex highly conserved proteins referred cohesin            separase initiate mitotic anaphase (Uhlmann et al., 1999, 2000; <br /> complex. Four proteins, STRUCTURAL MAINTENANCE OF                            Tomonaga et al., 2000). In vertebrates, cohesin complexes <br /> CHROMOSOME1 (SMC1), SMC3, SISTER CHROMATID COHE-                             dissociate chromatids prophase separase- <br /> SION1 (SCC1), SCC3, form core mitotic cohesin                 independent process (Waizenegger et al., 2000; Losada et al., <br /> complex, used wide range organisms. In yeast,               2002). The small fraction cohesin remains primarily in <br /> the cohesin complex chromosomes S-phase                     centromeric regions released separase metaphase- <br /> to anaphase, preferential binding centromeric regions                to-anaphase transition (Sumara et al., 2000, 2002; Ross and <br /> (Michaelis et al., 1997; Uhlmann Nasmyth, 1998; Megee                Cohen-Fix, 2002). <br /> Koshland, 1999; Blat et al., 2002). The release chromosome                   Sister chromatid cohesion released stages <br /> cohesion metaphase-to-anaphase transition sub-                meiosis. Recombination homologous chromo- <br /> sequent separation sister chromatids triggered                 somes meiotic prophase I forms chiasmata, hold <br /> organisms separase (ESP1), specifically cleaves                 maternal paternal chromosomes together. The resolution of <br /> cohesin kleisin (SCC1) subunit (Ciosk et al., 1998; Uhlmann et al.,          chiasmata separation homologous chromosomes <br /> 1999; Buonomo et al., 2000; Uhlmann et al., 2000; Hauf et al.,               depend separase cleavage meiotic a-kleisin subunit, <br />                                                                              REC8, cohesin complexes arms anaphase I <br /> 1 To <br />                                                                              (Buonomo et al., 2000; Kitajima et al., 2003). Centromeric cohe- <br />      correspondence addressed. E-mail makaroca@ <br /> muohio.edu; fax 513-529-5715.                                                sion protected conserved SGO family proteins until <br /> The authors responsible distribution materials integral        anaphase II, separase cleavage REC8 protein facil- <br /> findings presented article accordance policy described   itates separation sister chromatids (Rabitsch et al., 2003; <br /> in Instructions Authors (www.plantcell.org) are: Zhe Liu (liuz2@     Katis et al., 2004; McGuinness et al., 2005). <br /> muohio.edu) Christopher A. Makaroff (makaroca@muohio. <br /> edu). <br />                                                                                 Although separase required meiotic chromosome seg- <br /> Article, publication date, citation information          regation, mechanisms cohesin removal be <br /> www.plantcell.org/cgi/doi/10.1105/tpc.105.036913.                            operating. Studies distribution cohesin proteins during <br />  1214     The Plant Cell <br />  <br />  <br />  <br /> meiosis number organisms, including Caenorhabditis               separase (AESP) investigated role somatic meiotic <br /> elegans, mammals, Arabidopsis thaliana, shown              cells. Analysis independent T-DNA insertion lines dem- <br /> similar situation mitosis animal cells, significant   onstrated AESP essential gene. Plants heterozygous <br /> amount cohesin removed redistributed             mutations exhibited embryo arrest globular stage of <br /> prophase chromosomes anaphase I higher eukaryotes              development 25% developing seeds, indicating AESP <br /> (Eijpe et al., 2000, 2003; Pasierbek et al., 2001; Cai et al., 2003;     essential embryo development. Reduction AESP tran- <br /> Lee et al., 2003). Furthermore, recently shown budding         scripts expression AESP-RNA interference (RNAi) <br /> yeast portion cohesin released              using meiotic ATDMC1 promoter resulted reduced fertility <br /> metaphase I condensin/Cdc5-dependent process (Yu                caused defects meiosis. Alterations observed the <br /> Koshland, 2005). Finally, experiments Xenopus suggested          segregation homologous chromosomes anaphase I and <br /> chromosome segregation meiosis I place                sister chromatids anaphase II, indicating AESP required <br /> absence APC activity presence high levels            meiosis I meiosis II Arabidopsis. Consistent with <br /> separase&#226;&#8364;&#8482;s inhibitor securin (Peter et al., 2001; Taieb et al., 2001).   observations, SYN1 persist chromosomes <br />    Along essential role chromosome separation,               anaphase I. These results demonstrate cleavage of <br /> separase required number additional cellular pro-            SYN1 AESP responsible release sister chromatid <br /> cesses different organisms. These include anaphase spindle            cohesion meiosis. <br /> stabilization coupling anaphase mitotic exit during <br /> mitosis yeast (Funabiki et al., 1996; Ciosk et al., 1998; Cohen- <br /> Fix Koshland, 1999; Uhlmann et al., 2000; Sullivan               RESULTS <br /> Uhlmann, 2003; Papi et al., 2005) coordination of <br /> chromosome segregation spindle formation meiosis              Identification, Structure, Expression AESP <br /> (Buonomo et al., 2003). Vertebrate separase role cell <br /> cycle progression (Papi et al., 2005; Queralt Uhlmann, 2005).        The Arabidopsis genome contains putative separase gene <br /> In C. elegans, separase required proper positioning        (At4g22970), called AESP. The AESP cDNA iso- <br /> the centrosome asymmetric mitotic division          lated compared published genomic sequence of <br /> for eggshell development embryogenesis (Siomos et al.,            At4g22970. AESP consists 27 exons 26 introns (Figure <br /> 2001; Rappleye et al., 2002). Mutations Drosophila separase           1A), different predicted structure (accession <br /> result defects epithelial cell organization (Pandey et al.,        number NM118426). The predicted cDNA sequence (NM118426), <br /> 2005). Therefore, addition central role controlling     5588 bp, significantly shorter actual mRNA (6884 <br /> release sister chromatid cohesion mitosis meiosis,         bp) result omission exons 3, 9, 10, 16, 18, 19. In <br /> separase implicated number different events          addition, differences predicted actual exon <br /> various organisms.                                                       splice sites observed. <br />    Much known cohesin machinery                    The AESP cDNA (AY823256) capable encoding 2180&#226;&#8364;&#8220; <br /> chromosome cohesion controlled plants. Arabidopsis con-            amino acid protein. AESP exhibits level similarity <br /> tains cohesin-associated a-kleisin homologs. SYN1/DIF1,             (10 14%) vertebrate, worm, fly, yeast separase <br /> which REC8 homolog, essential sister chromatid               proteins other. AESP similar size the <br /> cohesion meiosis (Peirson et al., 1997; Bai et al., 1999;         mammalian enzymes, significantly larger those <br /> Bhatt et al., 1999; Cai et al., 2003). SYN1 expressed primarily       yeast (NP011612), worm (AAK77200), fly (AAQ72557) <br /> in buds, anti-SYN1 antibody labels chromosome axes              (Figure 1B). Although sequence conservation observed <br /> early leptotene metaphase I (Cai et al., 2003). The             proteins, greatest similarity C <br /> remaining kleisin proteins, SYN2, SYN3, SYN4, expressed          terminus, contains C-50 peptidase domain. The AESP <br /> throughout plant (Dong et al., 2001). Antibodies proteins     peptidase domain similar (;20% sequence identity) to <br /> <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> chromosomes somatic cells, suggesting             mammalian enzymes (Figure 2). However, is <br /> participate mitotic cohesion (our unpublished data). The Arabi-       considerably larger (700 amino acids) most <br /> dopsis SMC1, SMC3, SCC3 genes isolated                 organisms (;400 470 amino acids). It contains predicted <br /> characterized; <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>knockout</span> mutants genes indicate        2Fe2S-ferredoxin domain present proteins of <br /> are essential embryo endosperm development (Liu              organisms (Figure 1B). At time, clear whether <br /> Meinke, 1998; Liu et al., 2002; Chelysheva et al., 2005).                AESP actually binds iron, so, role iron center. <br />    Although basic mechanisms control sister chromatid           AESP contains putative EF-hand calcium binding domain. <br /> cohesion conserved wide range organisms, clear         Budding yeast Esp1 contains calcium binding domain, is <br /> that considerable under-          required association spindle (Jensen et al., 2001). <br /> stand regulation process differences          It possible EF-hand motif AESP plays similar role in <br /> exist organisms. Furthermore, little known          plants. <br /> proteins control chromosome structure segregation                AESP transcript levels analyzed different tissues using <br /> plants, known plant separase proteins. To           RT-PCR. The ACTIN8 (ACT8) gene used internal <br /> better understand cohesion released                 standard control cDNA used its <br /> meiotic chromosomes plants, determine role(s)           relatively uniform distribution plant (An et al., <br /> separase plants, isolated Arabidopsis homolog          1996). Low detectable levels AESP transcript found <br />                                                                                                                  Arabidopsis Separase       1215 <br />  <br />  <br />  <br />  <br /> Figure 1. AESP Gene Protein Structures. <br />  <br /> (A) AESP gene structure. The positions exons shown black boxes. The positions directions primers used study shown as <br /> horizontal arrows. The positions T-DNA insertions shown inverted triangles. The scale bar shown bottom. <br /> (B) Motifs ESP1 proteins Arabidopsis (a), human (b), mouse (c), C. elegans (d), Saccharomyces cerevisiae (e), Drosophila (f). <br />  <br />  <br /> in tissues examined (Figure 3A). Transcripts AESP              heterozygous Aesp-1 Aesp-2 plants conducted. Thirty- <br /> present approximately equal levels roots, stems, leaves,         percent (41 116) progeny Aesp-1 plants were <br /> buds. The presence AESP transcripts wide range                  homozygous wild type, 65% (75 116) plants <br /> tissues consistent predicted role cell division         heterozygous Aesp-1 insertion. Likewise, 58 (35.8%) <br /> suggests play role mitosis meiosis.              homozygous wild-type plants 104 (64.2%) plants heterozy- <br />                                                                            gous T-DNA insert identified 162 progeny <br /> AESP Is Required Seed Development                                      Aesp-2 heterozygous plants analyzed. <br />                                                                                A plant homozygous Aesp-1 Aesp-2 T-DNA <br /> Two T-DNA insertion alleles, Aesp-1 Aesp-2, identified            insertion obtained. Our inability identify plants <br /> in SALK T-DNA express database characterized               homozygous Aesp-1 Aesp-2 T-DNA inserts coupled <br /> determine AESP required mitosis. PCR amplifica-             1:2:0 segregation ratios obtained two <br /> tion insertion sites followed DNA sequence analysis              independent lines suggested mutations result in <br /> confirmed Aesp-1 contains T-DNA insert exon 1,                   embryo lethality. To investigate possibility, examined the <br /> whereas Aesp-2 contains T-DNA insert intron 17 (Figure                siliques heterozygous plants carrying mutations. Wild- <br /> 1A). No homozygous mutant plants identified line.          type plants segregating populations Aesp-1 Aesp-2 <br /> Therefore, seeds heterozygous plants collected               plants produced average 56 64 (n &#194;&#188; 50) full-size seeds per <br /> resown, PCR analysis progeny self-pollinated                 silique, respectively. By contrast, Aesp-1 Aesp-2 heterozygous <br />  1216     The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 2. Amino Acid Sequence Alignment Peptidase C-50 Domain ESP1 Proteins. <br />  <br /> The peptidase domains human (Q14674), mouse (P60330), Arabidopsis (AY823256), S. cerevisiae (NP011612), C. elegans (AAK77200), and <br /> Drosophila (AAQ72557) separase proteins shown aligned. Identical amino acids shaded black, similar amino acids shaded gray. <br /> Missing residues shown dashes. ClustalW (http://www.ebi.ac.uk/clustalw) BOXSHADE 3.33 (http://www.ch.embnet.org/software/ <br /> BOX_form.html) used produce alignment. <br />  <br />  <br /> plants produced average 44 48 (n &#194;&#188; 50) normal seeds             ment appeared normal seeds. No dramatic differences were <br /> silique, respectively. Approximately 25% (335 1370)              observed seeds zygote, single-terminal, two- <br /> seeds examined siliques AESP heterozygous plants             terminal cell stages. Some variability development ob- <br /> lines dry shrunken. Reciprocal crosses heterozy-             served beginning approximately quadrant stage. For <br /> gous plants containing Aesp-1 Aesp-2 mutations dem-                example, siliques seeds dermatogen <br /> onstrated allelic linked seed defect.             (16-cell) stage, developed point which <br /> Backcrossing wild-type plants indicated mutations              embryo proper cells (data shown), in <br /> are transmitted normally gametes. This finding              older siliques embryos developed late <br /> consistent observation pollen appeared normal            globular stage, early globular (Figures 4A and <br /> in Aesp-1 Aesp-2 plants (data shown). These results                4D). However, embryo development given silique com- <br /> provided strong evidence AESP essential gene           pletely synchronous (Bowman, 1994). Therefore, we <br /> Aesp null mutations disrupt seed development.                              exclude possibility <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>mutation</span> results delay <br />    We compared development wild-type mutant                development, believe early <br /> seeds siliques Aesp-2 heterozygous plants                differences observed attributable natural variability in <br /> determine nature seed defect. Early embryo develop-             seed development. <br />                                                                                                                Arabidopsis Separase      1217 <br />  <br />  <br />  <br />                                                                            enlarged nucleoli chalazal (Figure 4H) and <br />                                                                            peripheral (Figure 4G) positions. Similar defects observed <br />                                                                            endosperm Aesp-1 seeds (data shown). Therefore, <br />                                                                            Aesp null mutations block cellularization endo- <br />                                                                            sperm result weak ttn-like phenotype endo- <br />                                                                            sperm nuclei. <br />  <br />  <br />                                                                            ATDMC1-AESP-RNAi Plants Exhibit Reduced Fertility <br />  <br />                                                                            The embryo lethality associated T-DNA insertion lines <br /> Figure 3. AESP Transcript Levels Wild-Type RNAi Plants. <br />                                                                            limited analysis role AESP chromosome segre- <br /> (A) AESP transcript levels analyzed root (R), bud (B), stem (S),   gation. Therefore, generated transgenic plants expressing <br /> and leaf (L) using RT-PCR. The ACT8 gene used internal           AESP-RNAi constructs driven 35S meiotic <br /> control.                                                                   ATDMC1 promoter. The ATDMC1 promoter exhibits strong <br /> (B) RT-PCR analysis AESP mRNA levels wild-type transgenic        meiosis-specific expression (Klimyuk Jones, 1997). It has <br /> plants containing AESP-RNAi construct driven ATDMC1 <br />                                                                            used effectively drive meiosis-specific expression of <br /> promoter. Plants 1, 2, 7 exhibited reduced fertility AESP RNA <br />                                                                            RNAi constructs examine role DMC1, BRCA2, and <br /> levels. Plant 5 showed normal levels AESP RNA fertile. The <br /> ACT8 gene used internal control. <br />                                                                            CDC45 meiotic prophase (Siaud et al., 2004; Stevens et al., <br />                                                                            2004). Consistent observations T-DNA insertion <br />                                                                            lines, indicated AESP essential gene, were <br />    The clear difference development observed           unable identify transgenic plants expressing 35S-AESP- <br /> heart stage, ;25% seeds (151 663) arrested             RNAi construct &gt;30,000 plants screened. By contrast, <br /> the embryo globular stage (hereafter referred Aesp-2          numerous transgenic plants obtained transformation <br /> seeds) (Figures 4B 4E). Aesp-2 seeds (129 504) remained             constructs expressing AESP-RNAi ATDMC1 pro- <br /> at globular stage (Figure 4F) siliques containing wild-type         moter. <br /> seeds torpedo cotyledon stages (Figure 4C). No                     Fifteen 20 independent ATDMC1-AESP-RNAi lines <br /> significant alterations appearance morphology             showed reduced fertility. The level fertility varied between <br /> embryo proper suspensor observed vast majority              plants branches plant. Of <br /> (&gt;95%) Aesp-2 seeds examined (n &gt; 300). Embryo develop-                 ATDMC1-AESP-RNAi plants exhibiting reduced fertility, <br /> ment seeds Aesp-1 plants resembled Aesp-2 plants,            ;80% flowers produced reduced numbers seeds. <br /> with ;25% (59 250) embryos arresting globular             Flowers ATDMC1-AESP-RNAi plants typically short sta- <br /> stage (data shown). Therefore, Atesp1 null mutations cause             mens (Figure 5B) produced dramatically reduced levels of <br /> embryo arrest globular stage.                                       pollen. The pollen produced typically shrunken <br />    Alterations endosperm development Aesp-2 seeds               (Figure 5D) relative wild-type plants (Figure 5C). <br /> also observed consistent arrest globular              Siliques RNAi plants shorter (Figure 5E) and <br /> stage. No differences early endosperm development                  produced average 6 full-size seeds silique (n &#194;&#188; 50) (Figure <br /> observed. Two daughter nuclei formed                   5F) compared wild-type siliques, generated aver- <br /> division primary endosperm nucleus migrate             age 55 seeds silique (n &#194;&#188; 50) (Figure 5G). When anthers the <br /> to opposite poles embryo sac. Later, numerous nuclei                ATDMC1-AESP-RNAi plants squashed stained with <br /> were observed, periphery embryo sac                0.1% toluidine blue, polyads containing eight <br /> and dispersed embryo sac. The major              microspores observed; wild-type anthers contained only <br /> difference observed early heart stage, wild-            tetrads microspores. Interestingly, half (66%; n &#194;&#188; <br /> type embryos cellularization endosperm nuclei ob-               200) polyads microspores (Figure 5H). <br /> served (Figure 4B). By contrast, cellularization endosperm              This finding somewhat unexpected. We predicted <br /> nuclei observed Aesp-2 seeds (Figure 4E). Even               inhibition separase activity block nuclear division <br /> siliques containing wild-type curled cotyledon&#226;&#8364;&#8220;stage embryos,              result single tetraploid cell produce polyads <br /> the endosperm Aesp-2 seeds showed signs cellulariza-              fewer microspores chromosome segre- <br /> tion (Figure 4F).                                                          gation occurred. Defects vegetative growth ATDMC1- <br />    A second endosperm-associated defect observed               AESP-RNAi plants observed (data shown). <br /> Aesp-2 seeds enlargement nuclei nucleoli                   AESP transcript levels examined buds ATDMC1- <br /> (Figures 4G 4H). This similar to, dramatic as,       AESP-RNAi plants determine reduced fertility was <br /> titan (ttn)-like phenotype described (Liu et al.,            correlated reduction AESP-RNA (Figure 3B). A dramatic <br /> 2002). Endosperm wild-type seeds late globular stage             reduction AESP RNA levels seen plants <br /> contained nuclei average diameter 7.5 mm nucleoli           examined containing ATDMC1-AESP-RNAi construct. The <br /> with average diameter 3.4 mm (Figure 4I). By contrast,               plants reduced transcript levels exhibited obvious <br /> endosperm Aesp-2 seeds exhibiting ttn-like phenotype                reduced fertility. The ATDMC1-AESP-RNAi plant did not <br /> (n &#194;&#188; 75) contained nuclei nucleoli average diameters           reduced transcript levels did sterility and <br /> 17.2 7.0 mm, respectively. Enlarged endosperm nuclei              phenotypically resembled wild-type plants. The low levels of <br />  1218      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 4. AESP Is Required Seed Development. <br /> Bright-field images stained sections taken embryos ([A] [F]) endosperm nuclei ([G] [I]) wild-type ([A] [C] [I]) abnormal ([D] <br /> to [H]) seeds siliques heterozygous Aesp-2 plants. Mutant embryos arrested globular stage ([D] [F]) relative wild-type <br /> embryos globular (A), heart (B), curled cotyledon (C) stages. Some periphery (G) chalazal (H) endosperm nuclei nucleoli were <br /> enlarged relative wild-type embryos (I). Bars &#194;&#188; 20 mm. <br />  <br />  <br /> AESP transcripts buds reduced-fertility plants                   fragmented chromosomes seen (Figures 6L 6N). <br /> likely derived somatic cells. Therefore, RT-PCR analysis                 Cells resembling shown Figures 6L 6N never <br /> AESP RNA levels indicated reduced AESP transcript levels                    observed wild-type plants. In ;35% (79 228) cells ob- <br /> are correlated reduced fertility.                                           served anaphase I/telophase I, intact bivalents were <br />                                                                                  observed (Figure 6Q), suggesting kinetochores did <br /> ATDMC1-AESP-RNAi Plants Are Defective Meiosis I II                        attach properly spindle persistent cohesion <br />                                                                                  completely blocked chromosome segregation. In ;15% the <br /> Chromosome spreads examined reduced-fertility                        cells (31 225), meiosis I appeared normal (data shown). <br /> plants investigate effect AESP-RNAi expression                         By late anaphase II/telophase II, types cells were <br /> meiotic chromosome segregation. No differences ob-                          observed (Figures 6O 6R). In cells, mixture of <br /> served wild-type RNAi plants prophase I.                      bivalents, chromosome fragments, possibly chromosomes <br /> Chromosome condensation, alignment, synapsis appeared                        observed (Figure 6O). In cells, intact bivalents, <br /> normal leptotene, zygotene, pachytene (Figures 6A                     typically dispersed different quadrants cell, <br /> and 6I). Continued chromosome condensation resolu-                       observed (Figure 6R). Nuclear envelopes formed around <br /> tion bivalents occurred normally diplotene                    bivalent group chromosomes, resulting variable <br /> diakinesis (Figures 6B 6J). No alterations identified                numbers polyads (Figures 6P, 6S, 6T) instead the <br /> metaphase chromosomes, appeared align normally                          normal tetrad microspores observed wild-type cells (Figure <br /> (Figures 6C 6K). The major defect observed                  6H). Our observation cells contain mixture biva- <br /> metaphase I-to-anaphase I transition, alterations ho-                    lents, chromosomes, chromosome fragments, arise <br /> mologous chromosome segregation observed. In ;50%                           chromosome bridges, consistent with, explains, our <br /> (116 228) male meiocytes examined late                         observation polyads average microspores <br /> metaphase I telophase I, chromosome bridges tangled                      instead single tetraploid microspore initially <br />                                                                                                                Arabidopsis Separase      1219 <br />  <br />  <br />  <br />                                                                           tion AESP levels directly affects release cohesion both <br />                                                                           meiosis I meiosis II; however, exclude the <br />                                                                           possibility alterations observed meiosis II are <br />                                                                           result abnormalities associated meiosis I. The pres- <br />                                                                           ence significant numbers cells intact bivalents at <br />                                                                           telophase II raises possibility reduced AESP levels <br />                                                                           affect attachment kinetochores meiotic spindle. <br />  <br />  <br />                                                                           SYN1 Removal Is Defective ATDMC1-AESP-RNAi Plants <br />  <br />                                                                           The meiotic alterations suggested ATDMC1-AESP-RNAi <br />                                                                           blocked normal release sister chromatid cohesion. There- <br />                                                                           fore, distribution cohesin protein SYN1 meiotic <br />                                                                           chromosomes ATDMC1-AESP-RNAi plants investigated <br />                                                                           determine meiotic alterations observed in <br />                                                                           fact attributable defects cohesin release. SYN1 the <br />                                                                           Arabidopsis homolog REC8 essential chromosome <br />                                                                           cohesion meiosis (Bai et al., 1999; Bhatt et al., 1999). In <br />                                                                           wild-type plants, SYN1 localizes developing chromosome <br />                                                                           axes beginning early leptotene lines chromosome <br />                                                                           axes synapsed chromosomes pachytene (Figure 7A). <br />                                                                           During diplotene diakinesis, large portion SYN1 is <br />                                                                           released chromosomes (Figure 7B), prometaphase I <br />                                                                           metaphase I, SYN1 primarily centromeres <br />                                                                           (Figures 7C 7D). Beginning early anaphase I, SYN1 is <br />                                                                           longer detected wild-type meiotic chromosomes (Figure <br />                                                                           7E). However, transient observation SYN1 nucleo- <br />                                                                           plasm meiotic interphase II cells suggested low but <br />                                                                           undetectable levels SYN1 remain chromosomes after <br />                                                                           anaphase I (Cai et al., 2003). These localization patterns closely <br />                                                                           resemble observed Arabidopsis SCC3 cohesin <br />                                                                           protein (Chelysheva et al., 2005) suggest chromatin con- <br />                                                                           formation Arabidopsis centromeres blocks access cohesin <br />                                                                           antibodies. <br />                                                                              The distribution SYN1 meiotic chromosomes of <br />                                                                           ATDMC1-AESP-RNAi plants resembled wild-type cells <br />                                                                           early stages meiosis, including pachytene (Figure 7F), <br />                                                                           diakinesis (Figure 7G), prometaphase I (Figure 7H), meta- <br /> Figure 5. ATDMC1-AESP-RNAi Plants Exhibit Reduced Fertility. <br />                                                                           phase I (Figure 7I). In particular, SYN1 appeared released <br />                                                                           chromosomes normally diplotene diakinesis. <br /> (A) (G) Anthers pollen development wild-type ATDMC1-        The reduction SYN1 signal diplotene diakinesis in <br /> AESP-RNAi plants. Wild-type flowers elongated stamens (A) and <br />                                                                           ATDMC1-AESP-RNAi plants suggests cohesion released <br /> mature pollen grains ([C], arrow), ATDMC1-AESP-RNAi flowers <br />                                                                           two-step mechanism meiosis I plants, similar to <br /> had short stamens (B) shrunken pollen grains (D). The siliques in <br /> RNAi plants dramatically shorter (E) generated fewer seeds (F) <br />                                                                           situation mitotic prophase animal cells (Sumara <br /> in contrast wild-type siliques (G).                                  et al., 2000, 2002; Waizenegger et al., 2000; Losada et al., 2002) <br /> (H) Relative number microspores ATDMC1-AESP-RNAi mutant             meiotic prophase yeast (Yu Koshland, 2005). How- <br /> polyads (n &#194;&#188; 200 polyads).                                                ever, contrast wild-type plants, strong SYN1 signals were <br />                                                                           observed anaphase I cells various stages of <br />                                                                           meiosis II. Strong SYN1 labeling chromosomes was <br /> expected. Polyads containing microspores arise               observed stretched fragmented chromosomes during <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> cells exhibiting complete failure segregate               anaphase I (Figures 7J 7K) unsegregated <br /> homologous chromosomes.                                                   bivalents telophase I telophase II (Figures 7L 7O). <br />    In summary, reduction AESP RNA levels microsporocytes            Therefore, expression AESP RNAi ATDMC1 promoter <br /> is associated defects chromosome segregation             blocks removal SYN1 chromosomes pre- <br /> consistent persistent sister chromatid cohesion           vents normal segregation chromosomes meiosis. <br /> meiosis. Although ATDMC1-AESP-RNAi 100% effi-                  These results confirm AESP plant separase that <br /> cient eliminating separase activity, meiocytes exhibit            AESP function required release cohesion during <br /> defects chromosome segregation. We expect reduc-              meiosis. <br />  1220      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 6. ATDMC1-AESP-RNAi Plants Are Defective Meiosis I Meiosis II. <br /> Meiotic spreads wild-type ([A] [G]) ATDMC1-AESP-RNAi ([I] [O], [Q], [R]) plants prepared Carnoy&#226;&#8364;&#8482;s fixative stained with <br /> 49,6-diamidino-2-phenylindole (DAPI). Tetrads wild-type plants (H) polyads ATDMC1-AESP-RNAi plants ([P], [S], [T]) stained with <br /> toluidine blue. No differences detected wild-type mutant meiocytes prophase I, including late pachytene ([A] [I]) diakinesis <br /> ([B] [J]), metaphase I ([C] [K]). Homologous chromosomes segregated evenly anaphase I (D) telophase I (E) wild-type <br /> meiocytes, mutant meiocytes, homologous chromosomes tangled stretched metaphase I-to-anaphase I transition (L), <br /> which resulted chromosome bridges ([M], arrows) stretched fragmented chromosomes ([N], arrow) anaphase I telophase I. Wild-type <br /> meiocytes meiotic metaphase II (F), telophase II (G), tetrad stage (H) shown. Most meiocytes RNAi lines displayed four <br /> groups chromosomes telophase II (O) produced polyads microspores, contained different amounts genetic <br /> material (P). Bivalents did segregate meiocytes meiosis I (Q) meiosis II (R) RNAi plants, resulting polyads fewer four <br /> microspores ([S] [T]). Stages abnormal meiocytes approximate based cell morphology stage surrounding tissue. <br /> Bars &#194;&#188; 5 mm. <br />  <br />  <br />  <br /> DISCUSSION                                                                     worms, vertebrate cells (reviewed Losada Hirano, <br />                                                                                2005; Nasmyth Haering, 2005; Queralt Uhlmann, 2005). <br /> Separase Is Required Proper Disjunction                             To investigate situation plants, we <br /> Chromosomes Meiosis Arabidopsis                                      generated analyzed number transgenic plants express- <br />                                                                                ing AESP-RNAi construct expressed meiotic <br /> Separase characterized shown required                   ATDMC1 promoter. Reduced fertility observed 75% of <br /> proper chromosome disjunction mitosis meiosis                  transgenic plants examined, &gt;80% meiocytes <br /> number organisms, including budding fission yeast, flies,               examined reduced-fertility plants displayed meiotic defects. <br />                                                                                                                        Arabidopsis Separase         1221 <br />  <br />  <br />  <br />  <br /> Figure 7. SYN1 Removal Is Defective ATDMC1-AESP-RNAi Plants. <br /> Meiotic spreads wild-type ([A] [E]) ATDMC1-AESP-RNAi ([F] [O]) plants prepared stained anti-SYN1 antibody (green) and <br /> propidium iodide (red). Meiocytes wild-type plants RNAi mutant exhibited similar SYN1 staining pachytene ([A] [F]), diakinesis ([B] and <br /> [G]), prometaphase I ([C] [H]), metaphase I ([D] [I]). SYN1 labeling detected anaphase I (E) wild-type meiocytes. SYN1 was <br /> still detected chromosome arms centromeric regions anaphase I ([J] [K]) intact bivalents various stages meiosis II the <br /> meiocytes RNAi plants ([L] [O]). Bar &#194;&#188; 5 mm. <br />  <br />  <br />  <br /> Therefore, similar situation yeast, worms, Drosophila,               resulting stretched broken chromosomes (Figure 6L). In <br /> and animal cells, separase required meiosis Arabidopsis.             ;35% cells examined, intact bivalents observed. <br />    Although requirement separase meiosis appears                Typically, always, bivalents observed different <br /> to universal, differences meiotic chromosome                   quadrants cell. After telophase II, nuclear envelopes <br /> behavior separase activity inhibited inactivated                 formed bivalents, groups chromosomes, chro- <br /> some organisms. In budding fission yeast, expression                  mosome fragments, resulting variable numbers polyads. <br /> noncleavable form REC8 temperature-sensitive mutations                   Therefore, contrast observations sys- <br /> in separase result chromosome nondisjunction meiosis I                   tems, meiocytes RNAi lines arrest metaphase I <br /> (Buonomo et al., 2000; Kitajima et al., 2003). In budding yeast,               progress meiosis I II, resulting the <br /> separase required onset anaphase               production polyads. Although unique system, this <br /> for spindle disassembly FEAR (Cdc14 early ana-                     finding completely unexpected, plants appear to <br /> phase release) pathway meiosis I (Buonomo et al., 2003). By                 lack meiotic checkpoint control mechanisms many <br /> contrast, accumulation anaphase I spindles                       organisms (Roeder, 1997; Roeder Bailis, 2000; <br /> observed Schizosaccharomyces pombe separase mutant                        Armstrong Jones, 2003). <br /> (Kitajima et al., 2003), suggesting dispensable spindle            The presence intact bivalents meiocytes ATDMC1- <br /> disassembly meiosis I fission yeast. Treatment C. elegans             AESP-RNAi plants raises possibility AESP re- <br /> embryos sep-1 RNAi blocked normal disjunction                      quired proper attachment kinetochores spindle in <br /> homologous chromosomes meiosis I (Siomos et al., 2001).                     plant cells. In cells failed separate bivalents, the <br /> Chromosomes appeared align properly normal meiosis I                   bivalents observed opposite poles, quadrants the <br /> spindle, segregation delayed significantly             cell, anaphase I II, respectively, suggesting one <br /> blocked completely. It concluded chromosomes                      pair sister kinetochores attached spindle. The presence <br /> of sep-1 RNAi embryos come normal tension fail                    intact bivalents explained bivalent was <br /> disjoin (Siomos et al., 2001). In mice, ;70% oocytes overex-                subjected bipolar traction failed separate completely <br /> pressing securin, separase inhibitor, arrest metaphase I                cohesin complexes removed. Depolymeriza- <br /> (Terret et al., 2003).                                                         tion microtubules allow bivalents to <br />    Our analysis meiosis ATDMC1-AESP-RNAi lines                       disperse cell. Separase implicated in <br /> identified defects release chromosome cohesion                  anaphase spindle stabilization coupling anaphase to <br /> that resulted chromosome fragmentation intact bivalents                 mitotic exit budding yeast (Ciosk et al., 1998; Cohen-Fix and <br /> at anaphase I II. In approximately half meiocytes              Koshland, 1999; Uhlmann et al., 2000; Sullivan Uhlmann, <br /> examined, sister kinetochores homologous chromosomes                        2003) coordination chromosome segregation <br /> appeared attach properly spindles, cohesion                spindle formation meiosis (Buonomo et al., 2003). <br /> the chromosome arms prevented proper homolog resolution,                       However, requirement separase attachment of <br />  1222     The Plant Cell <br />  <br />  <br />  <br /> kinetochores spindle demonstrated           arrested embryos presence abnormal endosperm <br /> any organism. Further experiments required determine            giant polyploid nuclei. Mutations SMC proteins, which <br /> whether AESP actually plays role kinetochore attachment.          core components cohesin condensin complexes, <br />                                                                        result ttn-like phenotype (Liu et al., 2002). Other TTN gene <br /> Separase Is Required Release Cohesion                products include ARL2 class GTP binding proteins (TTN1), <br /> Anaphase Not Meiotic Prophase Arabidopsis                protein related tubulin-folding cofactor D (TTN5), protein <br />                                                                        related isopeptidase T class deubiquitinating enzymes <br /> A number studies support notion proteolytic cleavage       (TTN6), plant-specific proteins unknown function <br /> of REC8 separase triggers homolog segregation meiosis I          (Tzafrir et al., 2002). Embryo development typically arrests early <br /> (Buonomo et al., 2000; Siomos et al., 2001; Kitajima et al., 2003).    ttn mutants (Liu et al., 2002; Tzafrir et al., 2002). However, <br /> However, direct demonstration inhibition separase          similar Aesp mutants, ttn6 embryos develop approxi- <br /> activity blocks removal cohesion chromosome arms           mately globular stage. Unlike ttn6 embryo cells, though, <br /> at anaphase I lacking. In report, clearly             appeared rounded disorganized, Aesp embryo <br /> demonstrate presence SYN1 unresolved bivalents            cells appeared relatively normal. The ttn-like phenotype also <br /> meiocytes ATDMC1-AESP-RNAi lines. In contrast wild-            dramatic Aesp mutants ttn6. Specifically, the <br /> type plants, little SYN1 signal observed     increased size nuclei nucleoli pronounced in <br /> chromosomes onset anaphase I (Figure 7E), strong          Aesp endosperm compared ttn6 plants. Further- <br /> SYN1 signals observed anaphase I cells           more, &lt;50% endosperm nuclei examined Aesp plants <br /> various stages meiosis II meiocytes ATDMC1-AESP-RNAi          exhibited enlarged nucleoli. Variations seed phenotype <br /> plants (Figures 7J 7O). Therefore, expression AESP-RNAi          observed ttn mutants, including ttn6. <br /> during meiosis clearly blocked removal cohesion                Although separase, SMC1, SMC3 participate the <br /> protein SYN1 meiotic chromosomes meiosis I II.         control sister chromatid cohesion mitosis meiosis, <br /> This direct example inhibition separase activity      phenotype Aesp <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>null mutants</span> differs somewhat those <br /> blocked removal cohesin chromosomes.                   ttn8 (SMC1) ttn7 (SMC3). The SMC1 SMC3 knockouts <br />    The release sister chromatid cohesion meiosis ap-         exhibit strong ttn-like phenotypes, significant defects in <br /> pears occur three-step process Arabidopsis. A large         early embryo development enlarged endosperm nuclei and <br /> portion arm cohesin released chromosomes            nucleoli (Liu et al., 2002). Our finding Aesp mutant embryos <br /> during prophase I (Cai et al., 2003). Residual arm cohesin          develop endosperm phenotype weaker <br /> released anaphase I allow separation homologous           relative ttn7 ttn8 surprising mutations in <br /> chromosomes, centromeric cohesin released                separase completely block chromosome separation dur- <br /> anaphase II allow separation sister chromatids. Our          ing mitosis. Our finding suggests cells contain pool of <br /> results indicate release cohesin diplotene      AEPS RNA sequestered protein. Results analysis of <br /> diakinesis occurs separase-independent process,           ATDMC1-AESP-RNAi plants suggest cells fact contain <br /> the release cohesin anaphase I anaphase II separase-      pool AESP RNA. It suggested presence of <br /> dependent. It shown recently approximately half        functional gametes, small embryos, polyploid endosperm <br /> cohesin complexes removed arms meiotic                 nuclei seeds containing null mutations genes critical for <br /> chromosomes metaphase I budding yeast (Yu                cellular division reflect diffusion trace amounts of <br /> Koshland, 2005). This process dependent presence            functional protein surrounding heterozygous tissues (Liu <br /> of Cdc5 condensin. Cohesin released separase-           et al., 2002). This true AESP; however, believe <br /> dependent -independent processes mitosis verte-          likely given activity separase be <br /> brate cells. During mitotic prophase, cohesin released         tightly regulated cell cycle proper nuclear <br /> chromosome arms separase-independent process                 division (Papi et al., 2005; Queralt Uhlmann, 2005). <br /> involves phosphorylation cohesin Polo-like kinase             It proposed early cellular abortion the <br /> and histones Aurora kinase (Waizenegger et al., 2000;            embryo relative continued DNA replication nuclear en- <br /> Sumara et al., 2002). Centromeric cohesin released          largement ttn7 ttn8 embryos result different cell <br /> anaphase separase-dependent process. Our results raise            cycle checkpoints operating Arabidopsis embryos endo- <br /> the possibility prophase removal cohesin            sperm (Liu et al., 2002). It possible different cell cycle <br /> meiosis plant cells use mechanism involving condensin         control mechanisms operate embryo endosperm and <br /> and Polo kinase. Studies way investigate           different cell types contain different levels SMC and <br /> possibility better understand formation             separase proteins mRNA proteins. It possible <br /> release sister chromatid cohesion regulated plants.           observed differences attributable proteins <br />                                                                        exhibiting different cellular functions. Separase impli- <br /> AESP Is Essential Embryo Endosperm Development                 cated relatively large number cellular events addition to <br />                                                                        essential role release sister chromatid cohesion. It has <br /> Analysis different T-DNA&#226;&#8364;&#8220;tagged lines demonstrated         implicated positioning spindle body fission <br /> AESP essential embryo endosperm development. The            yeast (Funabiki et al., 1996), proper positioning the <br /> overall phenotype Aesp knockout plants reminiscent        centrosome eggshell development embryogenesis <br /> ttn mutants (Liu et al., 2002). The defining features ttn mutants   C. elegans (Siomos et al., 2001; Rappleye et al., 2002), and <br />                                                                                                                       Arabidopsis Separase          1223 <br />  <br />  <br />  <br /> epithelial cell organization Drosophila (Pandey et al., 2005).            T-DNA inserts regrown soil (1) rescreen plants homozygous <br /> Separase shown play role controlling               T-DNA insertions, (2) study embryo seed development, (3) <br /> mitotic cell cycle yeast animal cells. In budding yeast,              examine seed germination sowing seeds moist filter paper. An <br />                                                                              allelism test Aesp-1 Aesp-2 conducted performing <br /> separase acts Cdc14 link anaphase mitotic exit <br />                                                                              reciprocal crosses heterozygous plants containing Aesp <br /> (Stegmeier et al., 2002; Sullivan Uhlmann, 2003; Stemmann <br />                                                                              mutations scoring number aborted seeds. Approximately 1% <br /> et al., 2005). In human cells, separase appears control multiple <br />                                                                              (4 632) seeds obtained crosses wild-type plants were <br /> aspects G2/M program (Papi et al., 2005). These cell cycle            dry shrunken. By contrast, 125 506 seeds (24.7%) crosses <br /> control activities involve nonproteolytic functions separase,             Aesp-1 Aesp-2 heterozygous lines be <br /> including Cdk1 binding (Gorr et al., 2005; Papi et al., 2005).               defective. Backcrossing Aesp-2 mutations wild-type plants <br /> Therefore, possible embryo/endosperm                  resulted normal inheritance (48 105 progeny heterozygous) of <br /> alterations differences Aesp ttn7                T-DNA male female parental lines. <br /> ttn8 mutants attributable separase possibly SMC                  Transgenic plants expressing AESP-RNAi constructs 35S and <br /> proteins (Lam et al., 2005) having additional roles plant cells.          meiosis-specific ATDMC1 promoters produced analyzed. A <br />                                                                              435-bp fragment AESP cDNA (nucleotides 35 470) amplified <br /> Studies currently way investigate additional poten- <br />                                                                              using primers AESP-686 (59-ACGTTAATTAAGGCGCGCCTCTCATC- <br /> tial roles important protein plant cells. <br />                                                                              GACGTCGGCG-39) AESP-687 (59-CGCAGGACCATTTAAATCCTC- <br />                                                                              CGGCAACAACCGGG-39) cloned sense antisense <br /> METHODS                                                                      directions pFGC5941 (kindly provided Rich Jorgenson). A mod- <br />                                                                              ified version pFGC5941 AESP RNAi plasmid generated <br /> Plant Material                                                               35S promoter replaced meiosis-specific <br />                                                                              ATDMC1promoter (Klimyuk Jones, 1997). Constructs intro- <br /> Arabidopsis thaliana ecotype Wassilewskija used study. The       duced Agrobacterium tumefaciens strain GV3010 used to <br /> SALK T-DNA insertion lines 071659 037016 obtained          transform wild-type Arabidopsis plants (Clough Bent, 1998). Trans- <br /> ABRC Stock Center. Plants grown commercial potting mix        genic plants initially identified BASTA (Farnam Companies) <br /> growth chamber 208C 16-h-light/8-h-dark cycle. Ten days      selection, presence transgene confirmed PCR. <br /> germination, leaves stems collected rosette-stage plants. <br /> After 16 18 d, buds lengths 0.5 0.7 mm were <br />                                                                              Microscopy <br /> collected prebolting plants analysis meiosis. Roots were <br /> collected seeds grown agar plates.                                   Embryo development studied essentially described (Liu and <br />                                                                              Meinke, 1998). Specifically, siliques fixed 3.5% (w/v) glutaralde- <br />                                                                              hyde 1.5% (w/v) paraformaldehyde 0.1 M cacodylate buffer, pH <br /> Molecular Analysis AESP <br />                                                                              7.0, 4 h room temperature rinsed 0.1 M cacodylate buffer <br /> A full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> AESP cDNA generated using RT-PCR. The large size        twice 15 30 min each. Tissues treated 1% (w/v) OsO4 at <br /> AESP impossible clone full-length cDNA. Therefore,     08C 24 h followed washing twice distilled, deionized water 15 <br /> the AESP cDNA initially cloned series overlapping PCR            30 min dehydrated graded ethanol series (20, 30, 50, <br /> fragments subsequently confirmed generation large          70, 95% times 100% ethanol) 30 min each. The <br /> overlapping cDNA fragments RT-PCR. Total root RNA isolated            tissue infiltrated 25, 33.3, 50, 66.6, 75% LR White resin in <br /> (Verwoerd et al., 1989), cDNA produced reverse transcription      ethanol times 100% resin rotation 16 h each. <br /> with oligo(dT) primer (Thermoscript RT-PCR system; Invitrogen). PCR       Tissues embedded 100% resin 588C 12 h, 0.6-mm tissue <br /> was conducted cDNA using series gene-specific primers. The       sections using microtome (Reichert Ultracut) glass <br /> products cloned pGEM-T Easy (Promega) sequenced using          knives followed staining 0.1% toluidine blue/0.1% borax. <br /> a combination vector- gene-specific primers.                             Meiosis transgenic lines examined compared in <br />    The distribution AESP RNA different tissues investigated        wild-type plants using DAPI-stained meiotic chromosome spreads (Ross <br /> using RT-PCR. Total RNA isolated leaves, stems, roots,          et al., 1996). Anthers fixed Carnoy&#226;&#8364;&#8482;s reagent (3:1 ethanol:acetic <br /> buds. An equal (4 mg) total RNA different tissues     acid) 15 min soaked water 15 min twice, followed by <br /> used synthesize cDNA using Thermoscript RT-PCR                 digestion 1.4% b-glucuronidase, 0.3% cytohelicase, 0.3% pecto- <br /> (Invitrogen). ACT8 used standardize cDNA           lyase, 0.3% cellulase 10 mM sodium citrate 40 min. The <br /> the different tissues (An et al., 1996). AESP RNA levels determined     samples treated cold 10 mM sodium citrate 5 min twice <br /> by PCR primers AESP-764 (59-GGGCATATGCCTGGTGGT-                     cleared 60% acetic acid 2 min, squashed, frozen dry ice, and <br /> GATCTCACCG-39) AESP-765 (59-CCCGAATTCCGAGATGAAGAA-                       dried overnight. The samples washed 13 PBS 15 min and <br /> GGAAG-39). PCR products separated 0.7% agarose gels,                 stained DAPI 1,4-diazabicyclo-[2,2,2]-octane (DABCO). Sam- <br /> transferred nylon membranes, hybridized corresponding        ples observed Olympus IX-81 fluorescence deconvolution <br /> ACT8 AESP PCR products labeled [a-32P]dCTP. Radioactivity           microscope system. Data analyzed Image Pro Plus (Media <br /> was detected PhosphorImager (Molecular Dynamics).                     Cybernetics) organized Photoshop. Meiotic stages as- <br />    The effects knockout mutations AESP investigated        signed based chromosome cellular morphology the <br /> analysis independent T-DNA insertion lines. Genomic DNA           stage surrounding tissue. <br /> isolated plants grown seeds obtained SALK T-DNA               The distribution SYN1 antibody chromosomes used to <br /> insertion lines 071659 037016 screened combination         monitor presence cohesin complex. Meiotic chromosome <br /> plant-specific left border primers detect presence T-DNA   spreads prepared described above. The slides stained <br /> insertions. After presence insertions confirmed DNA        anti-SYN1 antibody (Cai et al., 2003) (1:500 dilution) 48C overnight <br /> sequence analysis PCR fragments, SALK T-DNA insertion lines       moist container. The slides washed 2 h changes of <br /> 071659 037016 renamed Aesp-1 Aesp-2, respectively.              wash buffer (13 PBS, 1 mM EDTA, 0.1% Tween 20). The anti-SYN1 <br /> Seeds collected plants confirmed heterozygous        antibody detected Alexa-488 goat anti-rabbit antibody (1:300 <br />  1224      The Plant Cell <br />  <br />  <br />  <br /> dilution) 48C overnight. The slides washed 2 h     cohesion homologous chromosome pairing. J. Cell Sci. 116, <br /> changes wash buffer treated 2 mg/mL propidium iodide           2999&#226;&#8364;&#8220;3007. <br /> at 378C 30 min. The slides rinsed 13 PBS mounted       Chelysheva, L., et al. (2005). AtREC8 AtSCC3 essential the <br /> DABCO antifade mounting medium. Samples observed ana-                  monopolar orientation kinetochores meiosis. J. Cell Sci. <br /> lyzed described above.                                                       118, 4621&#226;&#8364;&#8220;4632. <br />                                                                               Ciosk, R., Zachariae, W., Michaelis, C., Shevchenko, A., Mann, M., <br /> Accession Numbers                                                               Nasmyth, K. (1998). An ESP1/PDS1 complex regulates loss of <br />                                                                                 sister chromatid cohesion metaphase anaphase transition in <br /> The Arabidopsis Genome Initiative numbers AESP ATDMC1 are <br />                                                                                 yeast. Cell 93, 1067&#226;&#8364;&#8220;1076. <br /> At4g22970 At3g22880, respectively. <br />                                                                               Clough, S.J., Bent, A.F. (1998). Floral dip: A simplified method <br />                                                                                 Agrobacterium-mediated transformation Arabidopsis thaliana. <br />                                                                                 Plant J. 16, 735&#226;&#8364;&#8220;743. <br /> ACKNOWLEDGMENTS <br />                                                                               Cohen-Fix, O., Koshland, D. (1999). Pds1p budding yeast has <br /> We thank Richard Edelman Matthew L. Duley technical support             dual roles: Inhibition anaphase initiation regulation mitotic <br /> with microscope microtome, Rich Jorgenson providing             exit. Genes Dev. 13, 1950&#226;&#8364;&#8220;1959. <br /> pFGC5941 plasmid, Marie-Pascale Doutriaux providing plas-           Cohen-Fix, O., Peters, J.M., Kirschner, M.W., Koshland, D. <br /> mid ATDMC1 promoter. We thank Megan Brophy help               (1996). Anaphase initiation Saccharomyces cerevisiae controlled <br /> with T-DNA screening, Xiaohui Yang, Kingsley Boateng, Meghan                APC-dependent degradation anaphase inhibitor Pds1p. <br /> Holdorf, Sriram Devanathan helpful discussions, anon-           Genes Dev. 10, 3081&#226;&#8364;&#8220;3093. <br /> ymous reviewers helpful comments manuscript. SALK T-DNA            Dong, F., Cai, X., Makaroff, C. (2001). Cloning characteriza- <br /> insertion lines obtained Ohio State Stock Center. This            tion Arabidopsis genes belong RAD21/REC8 family <br /> research supported Grant MCB0322171 C.A.M.                   chromosome cohesin proteins. Gene 271, 99&#226;&#8364;&#8220;108. <br /> National Science Foundation.                                                  Eijpe, M., Heyting, C., Gross, B., Jessberger, R. (2000). Associ- <br />                                                                                 ation mammalian SMC1 SMC3 proteins meiotic chromo- <br />                                                                                 somes synaptonemal complexes. J. Cell Sci. 113, 673&#226;&#8364;&#8220;682. <br /> Received August 6, 2005; revised February 7, 2006; accepted March 4,          Eijpe, M., Offenberg, H., Jessberger, R., Revenkova, E., Heyting, <br /> 2006; published March 31, 2006.                                                 C. (2003). Meiotic cohesin REC8 marks axial elements rat <br />                                                                                 synaptonemal complexes cohesins SMC1 beta SMC3. <br />                                                                                 J. Cell Biol. 160, 657&#226;&#8364;&#8220;670. <br /> REFERENCES                                                                    Funabiki, H., Kumada, K., Yanagida, M. (1996). Fission yeast cut1 <br />                                                                                 cut2 essential sister chromatid separation, concentrate <br /> An, Y., McDowell, J., Huang, S., McKinney, E., Chambliss, S.,               metaphase spindle form large complexes. EMBO J. 15, <br />   Meagher, R. (1996). Strong, constitutive expression Arabi-             6617&#226;&#8364;&#8220;6628. <br />   dopsis ACT2/ACT8 actin subclass vegetative tissues. Plant J. 10,         Gorr, I.H., Boos, D., Stemmann, O. (2005). Mutual inhibition of <br />   107&#226;&#8364;&#8220;121.                                                                      separase Cdk1 two-step complex formation. Mol. Cell 19, <br /> Armstrong, S., Jones, G. (2003). Meiotic cytology chromosome            135&#226;&#8364;&#8220;141. <br />   behavior wild-type Arabidopsis thaliana. J. Exp. Bot. 54, 1&#226;&#8364;&#8220;10.          Goyal, L. (2001). Cell death inhibition: Keeping caspases check. Cell <br /> Bai, X., Peirsion, B., Dong, F., Cai, X., Makaroff, C. (1999).              104, 805&#226;&#8364;&#8220;808. <br />   Isolation characterization SYN1, RAD21-like gene essential         Hauf, S., Waizenegger, I.C., Peters, J.M. (2001). Cohesin cleav- <br />   meiosis Arabidopsis. Plant Cell 11, 417&#226;&#8364;&#8220;430.                           age separase required anaphase cytokinesis human <br /> Barrett, A.J., Rawlings, N.D. (2001). Evolutionary lines cysteine        cells. Science 293, 1320&#226;&#8364;&#8220;1323. <br />   peptidases. Biol. Chem. 382, 727&#226;&#8364;&#8220;733.                                       Jensen, S., Segal, M., Clarke, D.J., Reed, S.I. (2001). A novel role <br /> Bhatt, A.M., Lister, C., Page, T., Fransz, P., Findlay, K., Jones, G.H.,        budding yeast separin Esp1 anaphase spindle elongation: <br />   Dickinson, H.G., Dean, C. (1999). The DIF1 gene Arabidopsis            Evidence proper spindle association Esp1 regulated by <br />   required meiotic chromosome segregation belongs             Pds1. J. Cell Biol. 152, 27&#226;&#8364;&#8220;40. <br />   REC8/RAD21 cohesin gene family. Plant J. 19, 463&#226;&#8364;&#8220;472.                       Katis, V.L., Galova, M., Rabitsch, K.P., Gregan, J., Nasmyth, K. <br /> Blat, Y., Protacio, R.U., Hunter, N., Kleckner, N. (2002). Physical         (2004). Maintenance cohesin centromeres meiosis I in <br />   functional interactions basic chromosome organizational             budding yeast requires kinetochore-associated protein related to <br />   features govern early steps meiotic chiasma formation. Cell 111,           MEI-S332. Curr. Biol. 14, 560&#226;&#8364;&#8220;572. <br />   791&#226;&#8364;&#8220;802.                                                                    Kitajima, T.S., Miyazaki, Y., Yamamoto, M., Watanabe, Y. (2003). <br /> Bowman, J. (1994). Arabidopsis: An Atlas Morphology Devel-               Rec8 cleavage separase required meiotic nuclear divisions in <br />   opment. (New York: Springer-Verlag), pp. 133&#226;&#8364;&#8220;332.                             fission yeast. EMBO J. 22, 5643&#226;&#8364;&#8220;5653. <br /> Buonomo, S.B.C., Clyne, R.K., Fuchs, J., Loidl, J., Uhlmann, F.,          Klimyuk, V.I., Jones, J.D.G. (1997). ATDMC1, Arabidopsis <br />   Nasmyth, K. (2000). Disjunction homologous chromosomes                  homologue yeast DMC1 gene: Characterization, transposon- <br />   meiosis I depends proteolytic cleavage meiotic cohesin              induced allelic variation meiosis-associated expression. Plant J. <br />   Rec8 separin. Cell 103, 387&#226;&#8364;&#8220;398.                                           11, 1&#226;&#8364;&#8220;14. <br /> Buonomo, S.B.C., Rabitsch, K.P., Fuchs, J., Gruber, S., Sullivan, M.,         Lam, W.S., Yang, X.H., Makaroff, C.A. (2005). Characterization <br />   Uhlmann, F., Petronczki, M., Toth, A., Nasmyth, K. (2003).                Arabidopsis thaliana SMC1 SMC3: Evidence AtSMC3 <br />   Division nucleolus release CDC14 anaphase            function chromosome cohesion. J. Cell Sci. 118, <br />   meiosis I depends separase, SPO12, SLK19. Dev. Cell 4,              3037&#226;&#8364;&#8220;3048. <br />   727&#226;&#8364;&#8220;739.                                                                    Lee, J., Iwai, T., Yokota, T., Yamashita, M. (2003). Temporally and <br /> Cai, X., Dong, F.G., Edelmann, R.E., Makaroff, C.A. (2003). The             spatially selective loss Rec8 protein meiotic chromosomes <br />   Arabidopsis SYN1 cohesin protein required sister chromatid arm         mammalian meiosis. J. Cell Sci. 116, 2781&#226;&#8364;&#8220;2790. <br />                                                                                                                       Arabidopsis Separase         1225 <br />  <br />  <br />  <br /> Liu, C., McElver, J., Tzafrir, I., Joosen, R., Wittich, P., Patton, D., Van     chromosome segregation meiosis I Caenorhabditis elegans. <br />   Lammeren, A., Meinke, D.W. (2002). Condensin cohesin                  Curr. Biol. 23, 1825&#226;&#8364;&#8220;1835. <br />   knockouts Arabidopsis exhibit titan seed phenotype. Plant J. 29,       Stegmeier, F., Visintin, R., Amon, A. (2002). Separase, polo <br />   405&#226;&#8364;&#8220;415.                                                                      kinase, kinetochore protein Slk19 Spo12 function network <br /> Liu, C.M., Meinke, D.W. (1998). The titan mutants Arabidopsis            controls Cdc14 localization early anaphase. Cell 108, <br />   disrupted mitosis cell cycle control seed develop-          207&#226;&#8364;&#8220;220. <br />   ment. Plant J. 16, 21&#226;&#8364;&#8220;31.                                                   Stemmann, O., Boos, D., Gorr, I. (2005). Rephrasing anaphase: <br /> Losada, A., Hirano, M., Hirano, T. (2002). Cohesin release re-           Separase FEARs shugoshin. Chromosoma 113, 409&#226;&#8364;&#8220;417. <br />   quired sister chromatid resolution, condensin-mediated      Stevens, R., Grelon, M., Vezon, D., Oh, J., Meyer, P., Perennes, C., <br />   compaction, onset mitosis. Genes Dev. 23, 3004&#226;&#8364;&#8220;3016.                Domenichini, S., Bergounioux, C. (2004). A CDC45 homolog in <br /> Losada, A., Hirano, T. (2005). Dynamic molecular linkers             Arabidopsis essential meiosis, shown RNA interference- <br />   genome: The decade SMC proteins. Genes Dev. 19, 1269&#226;&#8364;&#8220;1287.           induced gene silencing. Plant Cell 16, 99&#226;&#8364;&#8220;113. <br /> McGuinness, B.E., Hirota, T., Kudo, N.R., Peters, J.M., Nasmyth,          Sullivan, M., Uhlmann, F. (2003). A non-proteolytic function of <br />   K. (2005). Shugoshin prevents dissociation cohesin centro-            separase links onset anaphase mitotic exit. Nat. Cell Biol. 5, <br />   meres mitosis vertebrate cells. PLoS Biol. 3, 433&#226;&#8364;&#8220;449.              249&#226;&#8364;&#8220;254. <br /> Megee, P.C., Koshland, D. (1999). A functional assay centro-          Sumara, I., Vorlaufer, E., Gieffers, C., Peters, B.H., Peters, J.M. <br />   mere-associated sister chromatid cohesion. Science 285, 254&#226;&#8364;&#8220;257.              (2000). Characterization vertebrate cohesin complexes their <br /> Michaelis, C., Ciosk, R., Nasmyth, K. (1997). Cohesins: Chromo-             regulation prophase. J. Cell Biol. 151, 749&#226;&#8364;&#8220;761. <br />   somal proteins prevent premature separation sister chroma-          Sumara, I., Vorlaufer, E., Stukenberg, P., Kelm, O., Redemann, N., <br />   tids. Cell 91, 35&#226;&#8364;&#8220;45.                                                         Nigg, E., Peters, J. (2002). The dissociation cohesin from <br /> Nasmyth, K., Haering, C.H. (2005). The structure function            chromosomes prophase regulated polo-like kinase. Mol. Cell <br />   SMC Kleisin complexes. Annu. Rev. Biochem. 74, 595&#226;&#8364;&#8220;648.                   9, 515&#226;&#8364;&#8220;525. <br /> Pandey, R., Heidmann, S., Lehner, C.F. (2005). Epithelial re-             Taieb, F., Gross, S., Lewellyn, A., Maller, J. (2001). Activation of <br />   organization dynamics progression mitosis Dro-               <br /> </body></html>