<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />      Sgt1 is required for human kinetochore assembly <br />  <br />      Budding yeast Sgt1 is required for kinetochore assembly, and its                checkpoint are only present on mitotic kinetochores (Cleveland <br />      homologues have a role in cAMP signalling in fungi and pathogen                 et al, 2003). The mechanisms of pre-kinetochore and mitotic <br />      resistance in plants. The function of mammalian Sgt1 is unknown.                kinetochore assembly in mammals are largely uncharacterized. <br />      We report that <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>RNA interference</span>-mediated depletion of Sgt1                      We report that assembly of human kinetochores critically requires <br />      <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> HeLa cells causes dramatic alterations of the mitotic spindle              Sgt1 (also known as Sugt1; Kitagawa et al, 1999). Budding yeast <br />      and problems in chromosome alignment. Cells lacking Sgt1                        Sgt1p binds Skp1p <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> assemble the Cbf3p complex, the core <br />      undergo a mitotic delay due to activation of the spindle                        component of the kinetochore in this organism (Kitagawa et al, <br />      checkpoint. The checkpoint response, however, is significantly                  1999). Sgt1 is also required for pathogen resistance in plants <br />      weakened in Sgt1-depleted cells, and this correlates with a                     (Gray, 2002). These functions probably derive from Sgt1 binding <br />      dramatic reduction in kinetochore levels of Mad1, Mad2 and                      to Hsp90, of which Sgt1 may be a co-chaperone (Garcia-Ranea <br />      BubR1. These effects are explained by a problem in kinetochore                  et al, 2002; Hubert et al, 2003; Liu et al, 2003b; Lu et al, 2003; <br />      assembly that prevents the localization of Hec1, CENP-E, CENP-F,                Takahashi et al, 2003; Lee et al, 2004). Sgt1 has not been <br />      CENP-I, but not CENP-C, to mitotic kinetochores. Our studies                    characterized in mammals. We report here that the <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br />      implicate Sgt1 as an essential protein and a critical assembly                  RNA interference (RNAi)-mediated Sgt1 depletion in HeLa cells <br />      factor for the mammalian kinetochore, and lend credit to the                    defines an essential role in kinetochore assembly. <br />      hypothesis of a kinetochore assembly pathway that is conserved <br />      from yeast to man.                                                              RESULTS AND DISCUSSION <br />      Keywords: Sgt1; Hec1; Mad2; CENP; kinetochore; spindle                          HsSgt1 exists in two isoforms due to alternative splicing (Niikura &amp; <br />      checkpoint                                                                      Kitagawa, 2003). We designed short interfering RNA (siRNA) <br />      EMBO reports (2004) 5, 626&#226;&#8364;&#8220;631. doi:10.1038/sj.embor.7400154                    duplexes (Elbashir et al, 2001) that silenced both isoforms in HeLa <br />                                                                                      cells. Sgt1 levels were greatly reduced between 48 and 72 h after <br />      INTRODUCTION                                                                    siRNAs transfection (Fig 1A and supplementary Fig 1 online). <br />      Kinetochores assemble on centromeric DNA to mediate the                         Concomitantly, there was an accumulation of 4N cells becoming <br />      interaction of chromosomes with the mitotic spindle (Cleveland                  predominant at 72 h (Fig 1B). At 96 h, there was a large sub-G1 <br />      et al, 2003). Mammalian kinetochores exist during interphase as                 population, indicative of cell death. Time-lapse microscopy <br />      pre-kinetochores, poorly characterized structures containing a                  showed that HeLa cells lacking Sgt1 round up and are <br />      subset of the mitotic kinetochore markers (Pluta et al, 1995).                  subsequently delayed in mitosis for several hours, finally exiting <br />      CENP-A, -B, -C, -G, -H and -I reside at kinetochores throughout                 mitosis aberrantly without dividing and becoming multinucleated <br />      the cell cycle, whereas CENP-E, CENP-F, Hec1 (homologue of                      (Fig 1C). We applied a trivariate flow cytometry protocol to cor- <br />      budding yeast Ndc80p) and the components of the spindle                         relate DNA content with cyclin B1 (high during the entire G2&#226;&#8364;&#8220;M) <br />                                                                                      and H3-P (M-phase marker). Accumulation of H3-P-positive 4N <br />      1 <br />                                                                                      cells between 48 and 72 h was observed, indicative of mitotic <br />       Department of Experimental Oncology, European Institute of Oncology, <br />      Via Ripamonti 435, 20141 Milan, Italy                                           arrest (Fig 1D). In addition, a large fraction of 4N cells was <br />      2 <br />       Max-Planck Institute for Biochemistry, Department of Cell Biology,             negative for G2 or M markers, indicative of tetraploid G1 cells. <br />      Am Klopferspitz 18a, 82152 Martinsried, Germany                                 Time-lapse microscopy suggests that these cells result from <br />      3Department of Molecular Pharmacology, St Jude Children&#226;&#8364;&#8482;s Research Hospital, <br />  <br />      332 N Lauderdale Street, Memphis, Tennessee 38105-2794, USA <br />                                                                                      aberrant mitotic exit caused by loss of Sgt1. Thus, cells lacking <br />      4Wellcome Trust Center for Cell Biology, Mayfield Road, Edinburgh EH9 3JR, UK   Sgt1 are temporarily delayed in mitosis and eventually become <br />      + <br />        Corresponding author. Tel: &#195;&#190; 39 02 574 89829; Fax: &#195;&#190; 39 02 574 89851;         tetraploid G1 cells after failing to divide. <br />      E-mail: andrea.musacchio@ieo-research.it                                            To investigate the cause of mitotic delay, we visualized mitotic <br />      Received 30 January 2004; revised 11 March 2004; accepted 31 March 2004;        spindle and kinetochores. This revealed severe chromosome <br />      published online 7 May 2004                                                     congression and alignment problems (Fig 2, panels A4&#226;&#8364;&#8220;A9). Cells <br />  <br />  <br /> 6 2 6 EMBO reports   VOL 5 | NO 6 | 2004                                                            &amp;2004 EUROPEAN MOLECULAR BIOLOGY ORGANIZATION <br />  Function of human Sgt1 in kinetochore assembly <br /> P. Steensgaard et al                                                                                                           scientific report <br />  <br />  <br /> Fig1 | Silencing of Sgt1 affects progression through mitosis. (A) Western blot of HeLa cell lysates harvested at the indicated time points after transfection <br /> with siRNA duplexes against Sgt1. Control injections of anti-luciferase (Luc) siRNAs are also shown. Both splicing isoforms of Sgt1 are silenced. (B) DNA <br /> content was analysed by FC on PI-stained cells. Cells lacking Sgt1 accumulate with 4N DNA, with an increase in a sub-G1 population 96-h after siRNA <br /> transfection. (C) A cell (white arrowhead) lacking Sgt1 enters mitosis and remains arrested for several hours, finally exiting without dividing. This cell <br /> eventually died around 75 h. (D) Trivariate FC analysis of cells lacking Sgt1. White bars: Cells with 4N DNA content; grey bars: cells with 4N DNA content <br /> that were H3-P positive (mitotic cells, M); black bars: cells with 4N DNA content that were positive for Cyclin B1 (total of G2- and M-phase cells). Note the <br /> increase of H3-P-positive cells following Sgt1 depletion, with a concomitant modest increase in G2&#226;&#8364;&#8220;M cells at 72 h. More than 20% of the cell population <br /> had 4N DNA content but stained negative for mitotic markers, indicative of a tetraploid G1 population. <br />  <br />  <br />  <br />  <br /> presented a disordered mass of often hyper-condensed chromo-                                         Thus, depletion of Sgt1 causes a severe reduction of the <br /> somes, most of which had failed to align. Rare metaphase                                         kinetochore levels of three MSC components. It has been shown, <br /> configurations were observed, but one or more unaligned                                          however, that the MSC is maintained under conditions that cause <br /> chromosomes were always detected. Multipolar spindles and                                        a substantial decrease of kinetochore Mad1 and Mad2 (Martin- <br /> fragmented spindle poles were also frequent (see also Fig 2, panels                              Lluesma et al, 2002; DeLuca et al, 2003). To test if Mad2 was <br /> B2 and B3), and many cells contained micronuclei, indicative of                                  required to maintain the mitotic arrest caused by loss of Sgt1, we <br /> G1 cells that had aborted mitosis.                                                               silenced Sgt1 and Mad2 by RNAi (Fig 3D,E). The accumulation of <br />    Problems in chromosome alignment activate the mitotic                                         mitotic cells caused by Sgt1 loss was relieved by Mad2 depletion, <br /> spindle checkpoint (MSC) (Musacchio &amp; Hardwick, 2002). We                                        indicating that the MSC is activated after loss of Sgt1 despite <br /> asked whether the MSC was responsible for mitotic delay caused                                   substantial kinetochore depletion of Mad1, Mad2 and BubR1 <br /> by Sgt1 depletion. MSC proteins Mad1 and Mad2 are recruited to                                   (Fig 3E). Sgt1-depleted cells, however, remain arrested in mitosis <br /> kinetochores before microtubule&#226;&#8364;&#8220;chromosome attachment or if                                      for 11 h on average, significantly shorter than the average 24-h <br /> microtubule&#226;&#8364;&#8220;kinetochore interactions are disrupted with spindle                                  arrest on addition of nocodazole (Fig 3F). In principle, this may <br /> poisons (Musacchio &amp; Hardwick, 2002). Within the detection                                       indicate that sister chromatids slowly achieve bipolar attachment, <br /> limits of our experiment, the kinetochores of Sgt1-depleted cells                                a condition that satisfies the MSC allowing transit through mitosis. <br /> appeared to be devoid of Mad1 even after addition of nocodazole                                  In this case, cells would be expected to arrest longer (24 h or more <br /> (Fig 3A,B). The same was true for Mad2 (not shown). At times, a                                  as in control cells) if the process of kinetochore&#226;&#8364;&#8220;microtubule <br /> faint signal was observed near a small number of kinetochores                                    attachment was reverted. Addition of nocodazole to cells lacking <br /> (1&#226;&#8364;&#8220;3) in nocodazole-treated cells, but at markedly reduced levels                                Sgt1, however, did not restore a normal duration of the MSC <br /> relative to control cells (not shown). Of note, BubR1 was also                                   (Fig 3F), which remained limited to 11 h. The proportion of mitotic <br /> unable to localize to the kinetochores of Sgt1-depleted cells                                    cells was also the same with or without nocodazole (compare <br /> (Fig 3C).                                                                                        Fig 3E with Fig 3G) or taxol (not shown). This shows that cells do <br />  <br />  <br /> &amp;2004 EUROPEAN MOLECULAR BIOLOGY ORGANIZATION                                                                                            EMBO reports VOL 5 | NO 6 | 2004 6 2 7 <br />       scientific report                                                                                      Function of human Sgt1 in kinetochore assembly <br />                                                                                                                                         P. Steensgaard et al <br />  <br />  <br />  <br />  <br />        A                                                                             localized normally (Fig 4E). Because CENP-C requires CENP-A for <br />                  Luc siRNA                          Sgt1 siRNA <br />                                                                                      kinetochore localization in HeLa and other cell types (Howman <br />            A1                      A4                     A7 <br />                                                                                      et al, 2000; Van Hooser et al, 2001; Goshima et al, 2003), we <br />                                                                                      assume that CENP-A localization is not affected. Consistently, <br />                                                                                      CENP-I is not required for CENP-A or CENP-C localization in <br />                                                                                      HeLa cells (Goshima et al, 2003; Liu et al, 2003a). <br />                                                                                          Depletion of Sgt1 results in a marked alteration of kineto- <br />                                                                                      chores. All mitotic kinetochore markers and one pre-kinetochore <br />                 Prometaphase <br />                                                                                      marker tested (Mad1, Mad2, BubR1, Hec1, CENP-E, and CENP-F, <br />            A2                      A5                     A8                         CENP-I) were mislocalized. Loss of kinetochore integrity activates <br />                                                                                      an MSC response of reduced potency, with cells exiting mitosis <br />                                                                                      aberrantly (presumably because they cannot complete attach- <br />                                                                                      ment), becoming tetraploid G1 cells and eventually dying <br />                                                                                      (supplementary Fig 2 online). Reduced potency of the MSC <br />                                                                                      correlates with extensive kinetochore damage caused by Sgt1 <br />                  Metaphase                                                           depletion, and the residual MSC response may be due to traces of <br />            A3                      A6                     A9                         Mad1 and Mad2 at kinetochores of Sgt1-depleted cells. A clear <br />                                                                                      understanding of the mechanisms of MSC signal amplification is <br />                                                                                      required to address this issue rigorously. Our phenotype is similar <br />                                                                                      to the one caused by loss of CENP-I: chromosome misalignments, <br />                                                                                      scrambled spindles, mislocalization of CENP-F, a failure to <br />                                                                                      complete cytokinesis, a dramatic reduction of kinetochore <br />                  Anaphase                                                            Mad1 or Mad2, and a mitotic delay of reduced potency relative <br />                                                                                      to control cells (Nishihashi et al, 2002; Goshima et al, 2003; <br />                 CREST                   &#206;&#177;-Tubulin              DNA <br />                                                                                      Liu et al, 2003a). Cells lacking CENP-I, however, retain <br />       B                                                                              kinetochore BubR1, which is lost from kinetochores of cells <br />            B1                      B2                     B3 <br />                                                                                      lacking Sgt1 (Liu et al, 2003a). <br />                                                                                          What are the parallels between Sgt1 function in yeast and <br />                                                                                      human cells? Budding yeast SGT1 is a dosage suppressor of SKP1. <br />                                                                                      In this organism, Skp1 is a component of the Cbf3p kinetochore <br />                                                                                      complex and of the SCF ubiquitin&#226;&#8364;&#8220;ligase (Kitagawa &amp; Hieter, <br />                 Prometaphase                                                         2001), and its binding to Bub1 is required for mitotic delay <br />                                                                                      induced by kinetochore tension defects (Kitagawa et al, 2003). <br />                 &#206;&#179;-Tubulin               &#206;&#177;-Tubulin              DNA                   Sgt1p does not incorporate into Cbf3p and may be required to <br />      Fig 2 | Spindle defects in HeLa cells lacking Sgt1. (A) CREST (red),            activate the Ctf13p subunit, a step also involving Skp1p and <br />      a-tubulin (green) and DNA (blue) were visualized to investigate spindle         Hsp90 (Kaplan et al, 1997; Kitagawa et al, 1999; Stemmann et al, <br />      structure. Panels A1&#226;&#8364;&#8220;A3: Normal control cells at different stages of mitosis;   2002). Our discovery that the function of Sgt1 in kinetochore <br />      panels A4&#226;&#8364;&#8220;A9: spindle defects in interfered cells. (B) Centrosome problems      assembly is conserved in humans suggests the intriguing <br />      revealed with an anti-g-tubulin antibody (red). Panel B1: A normal control      possibility that the budding yeast and mammalian kinetochores <br />      cell; panels B2 and B3: two interfered cells with fragmented and faint          share their assembly pathway. With the exception of SKP1, the <br />      g-tubulin staining.                                                             vertebrate homologue of which has not been linked to kineto- <br />                                                                                      chore assembly and does not seem to localize to kinetochores of <br />                                                                                      animal cells (Freed et al, 1999; Gstaiger et al, 1999), the <br />      not exit mitosis because the MSC has been satisfied, and we                     homologues of genes encoding the Cbf3p complex have not been <br />      conclude that depletion of Sgt1 reduces the potency of MSC                      identified in other species. Human Sgt1 is a highly soluble protein <br />      response.                                                                       with nuclear and cytoplasmic localization that is not visible at <br />         Alterations of the mitotic spindle and decreased levels of                   kinetochores (SF 1). Sgt1 is devoid of catalytic domains, but <br />      kinetochore Mad1 and Mad2 have been observed after depletion                    contains protein interaction motifs such as TPR and p23 repeats. <br />      of kinetochore proteins such as Hec1, Nuf2, CENP-C, CENP-E or                   The latter identify Sgt1 as Hsp90 co-chaperone (Garcia-Ranea <br />      CENP-I (Wood et al, 1997; Fukagawa et al, 1999; McEwen et al,                   et al, 2002; Hubert et al, 2003; Liu et al, 2003b; Lu et al, 2003; <br />      2001; Martin-Lluesma et al, 2002; Nishihashi et al, 2002; DeLuca                Takahashi et al, 2003; Lee et al, 2004). Co-chaperones are <br />      et al, 2003; Liu et al, 2003a), suggesting the possibility that Sgt1            specificity factors that direct chaperones to specific classes of <br />      depletion causes damage to kinetochores. We probed a subset of                  substrates in localized functional contexts (Young et al, 2003). <br />      kinetochore markers by confocal immunofluorescence analysis                     In view of these findings, we speculate that Sgt1 may act as a <br />      of Sgt1-depleted HeLa cells. Hec1 did not localize to mitotic                   kinetochore assembly factor controlling kinetochore incorporation <br />      kinetochores (Fig 4A). The same was true for CENP-E and CENP-F                  of CENP-I and/or other unknown kinetochore components. <br />      (Fig 4B,C). Next, we addressed constitutive kinetochore residents.              Alternatively, cells lacking Sgt1 may be unable to retain CENP-I <br />      CENP-I, an inner kinetochore protein, was missing from kineto-                  and other proteins to the centromere region. The fundamental <br />      chores of Sgt1-depleted cells (Fig 4D). CENP-C, conversely,                     role of Sgt1 in kinetochore assembly suggests that Sgt1 may be the <br />  <br />  <br /> 6 2 8 EMBO reports     VOL 5 | NO 6 | 2004                                                          &amp;2004 EUROPEAN MOLECULAR BIOLOGY ORGANIZATION <br />  <br /> Fig 3 | Mitotic delay of cells lacking Sgt1 is caused by the spindle checkpoint. (A) Mad1 is absent from the kinetochores of Sgt1-depleted cells despite evident <br /> lack of microtubule attachment. (B) Addition of nocodazole did not cause a return of Mad1 to kinetochores. (C) Also BubR1 was missing from the <br /> kinetochores of Sgt1-depleted cells. (D) Parallel silencing of Sgt1 and Mad2 by RNAi as revealed by western blot on total cell lysates 72 and 96 h after <br /> siRNAs transfection. Mad1, CENP-I and vinculin are shown as controls. (E) Bivariate FC analysis on the indicated samples. White bars: 4N cells; grey bars: <br /> 4N cells that were H3-P positive. Dual ablation of Mad2 and Sgt1 relieves the mitotic block caused by Sgt1 ablation. (F) Cells lacking Sgt1 are unable to <br /> maintain a mitotic arrest beyond an average of 11 h, even if nocodazole is added. Black bars: Duration of mitosis in cells filmed by time-lapse microscopy <br /> from 48 to 96 h after siRNA transfection. (G) Sgt1-depleted cells cannot mount a full-fledge MSC response in nocodazole. White bars: 4N cells; grey bars: 4N <br /> cells that were positive for H3-P. The lower percentage of 4N cells in the total cell population after Sgt1 depletion is explained by a G1 arrest, as documented <br /> in SF 2. Note that the scale of this figure is different from that used in (E). <br />  <br />  <br /> vehicle leading us to the identification of the core constituents of                                 using oligofectamine reagent (Invitrogen). 5-Bromodeoxyuridine <br /> the mammalian kinetochore.                                                                           (BrdU) and nocodazole (Sigma) were used at 33 mM and 200 ng/ml, <br />                                                                                                      respectively. <br /> METHODS                                                                                              Antibodies. Human Sgt1 (NM_006704) was subcloned into <br /> Cell culture and RNA interference. HeLa cells were cultured in                                       pRSET-B (Invitrogen) and expressed in a soluble form in <br /> high-glucose Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s medium with 10% fetal                                       Escherichia coli BL21(DE3) (Invitrogen) after induction with <br /> bovine serum (Life Technologies) in a humidified 37 1C incubator                                     0.2 mM isopropyl-b-D-thiogalactoside at 37 1C for 3 h. The <br /> with 5% CO2. Sgt1 siRNA (50 -AAGGCUUUGGAACAGAAACCA-                                                  purified protein was concentrated to 10 mg/ml and used for <br /> 30 , top strand) was from Dharmacon, Inc. (USA). siRNAs targeting                                    immunization. Mad1 and Mad2 were expressed and purified as <br /> Mad2 (50 -AAGAGUCGGGACCACAGUUUA-30 , top strand) and                                                 described (Sironi et al, 2001) and used for immunization. Mouse <br /> GL2 luciferase have been described (Martin-Lluesma et al, 2002).                                     monoclonal antibodies (mAbs) against Sgt1 and Mad1 were <br /> We transfected annealed double-stranded siRNAs into HeLa cells                                       generated by fusing the splenocytes to NS-2 mouse myeloma cells <br />  <br />  <br /> &amp;2004 EUROPEAN MOLECULAR BIOLOGY ORGANIZATION                                                                                                                EMBO reports VOL 5 | NO 6 | 2004 6 2 9 <br />       scientific report                                                                                Function of human Sgt1 in kinetochore assembly <br />                                                                                                                                   P. Steensgaard et al <br />  <br />  <br />  <br />  <br />      3 days after the final boost. Single-cell cloning was used to         by immunization with pure 6His-Mad2. We used mouse mAbs <br />      generate cell lines producing mouse mAbs against Sgt1 (An32           against g-tubulin (Sigma) and Sgt1 (BD Biosciences), and rabbit <br />      and Ci47) and Mad1 (A. De Antoni and A. Musacchio,                    polyclonal antibodies against a-tubulin (Sigma) and CENP-F <br />      unpublished). A rabbit anti-Mad2 polyclonal antiserum was raised      (Novus-Biologicals). Mouse mAb against BubR1 was purchased <br />                                                                            from Chemicon International. Anti-centromere autoimmune ser- <br />                                                                            um (CREST) was from Antibodies Inc. Rabbit polyclonal anti- <br />                           CREST         DNA        Hec1    Merge           bodies against CENP-I, Hec1 and CENP-C have been described <br />                       A                                                    (Saitoh et al, 1992; Martin-Lluesma et al, 2002; Liu et al, 2003a). <br />          Luc siRNA <br />  <br />  <br />  <br />  <br />                                                                            Immunodetection. Cells grown on coverslips pretreated with <br />                                                                            15 mg/ml poly(D)lysine (Sigma) were fixed in 4% paraformaldehyde <br />                                                                            in PIPES buffer (80 mM PIPES 5 mM EGTA, 2 mM MgCl2) for 5 min, <br />                                                                            treated with phosphate-buffered saline (PBS) plus 0.1% Triton X- <br />                                                                            100 for 10 min, and washed three times in PBS for 5 min. In the <br />          Sgt1 siRNA <br />  <br />  <br />  <br />  <br />                                                                            case of Hec1, the cells were incubated for 5 min in extraction <br />                                                                            buffer (0.1% Triton X-100, 100 mM PIPES, 300 mM sucrose, 1 mM <br />                                                                            EGTA, 1 mM MgCl2) before fixation in 4% paraformaldehyde. <br />                                                                            Primary antibodies diluted in PBS were applied to the coverslips <br />                           CREST         DNA       CENP-E   Merge <br />                                                                            and incubated for 60 min before PBS washing. Secondary <br />                       B <br />                                                                            antibodies against mouse, rabbit and human conjugated to Cy5, <br />          Luc siRNA <br />  <br />  <br />  <br />  <br />                                                                            or AlexaFluor 488 were typically diluted 1:150. DNA was stained <br />                                                                            with 50 mg/ml propidium iodide (PI) for 30 s. <br />                                                                            Microscopy. Wide-field fluorescence microscopy images were <br />                                                                            acquired using a BX61 (Olympus) motorized fluorescence <br />          Sgt1 siRNA <br />  <br />  <br />  <br />  <br />                                                                            microscope equipped with a B/W cooled CCD camera (C5985 <br />                                                                            Hamamatsu). A Bio-Rad MRC 1024 confocal microscope <br />                                                                            equipped with a 20 mW Kr&#226;&#8364;&#8220;Ar laser was used for confocal <br />                                                                            analysis. Time-lapse microscopy was performed with an IX70 <br />                           CREST         DNA       CENP-F   Merge           inverted microscope (Olympus) equipped with an incubation <br />                       C                                                    chamber (Solent Scientific). The system includes a digital camera <br />          Luc siRNA <br />  <br />  <br />  <br />  <br />                                                                            (Sensys, Roper Scientific) to acquire high-resolution images. <br />                                                                            Microscope stage motorization, filter wheels and image acquisi- <br />                                                                            tion and processing were controlled by Metamorph (Universal <br />                                                                            Imaging). <br />                                                                            Flow cytometry. To detect cyclin B1, phosphoH3 and DNA, cells <br />          Sgt1 siRNA <br />  <br />  <br />  <br />  <br />                                                                            were washed with PBS, detached from tissue culture plates and <br />                                                                            fixed in suspension at 2   106 cells/ml in 1% formaldehyde in PBS <br />                                                                            for 15 min at different time points after siRNA transfection. After <br />                                                                            PBS washing, the pellet was resuspended in ice-cold 70% ethanol <br />                           CREST         DNA       CENP-I   Merge <br />                                                                            and stored at 4 1C. For immunodetection, cells were washed twice <br />                       D <br />                                                                            in PBS and permeabilized in 0.1% Triton X-100 in PBS for 10 min. <br />          Luc siRNA <br />  <br />  <br />  <br />  <br />                                                                            After blocking in 5% normal goat serum (NGS) in PBS for 20 min, <br />                                                                            cells were incubated with 2.5 mg/ml anti-cyclin B1 mouse mAb <br />                                                                            (Pharmingen) and anti-P-H3 rabbit polyclonal antibody (Upstate) <br />                                                                            1:100 in PBS plus 1% NGS for 3 h at 37 1C. Cells were rinsed in <br />                                                                            PBS and incubated for 1 h with FITC-conjugated goat anti-mouse <br />          Sgt1 siRNA <br />  <br />  <br />  <br />  <br />                                                                            (1:50, Sigma) and Cy5-conjugated goat anti-rabbit (1:50, Jackson <br />                                                                            Immunoresearch) antibodies. Cells were washed again and <br />  <br />                           CREST         DNA       CENP-C   Merge <br />                       E                                                 b Fig 4 | A kinetochore defect in Sgt1-depleted HeLa cells. (A) Localization of <br />          Luc siRNA <br />  <br />  <br />  <br />  <br />                                                                            Hec1 to kinetochores was compared in control and Sgt1-depleted cells in <br />                                                                            the absence of spindle poisons. The CREST antiserum stains the inner <br />                                                                            kinetochore. In control cells, Hec1 is localized at the periphery of CREST <br />                                                                            staining. There was no kinetochore localization of Hec1 after Sgt1 <br />                                                                            depletion. (B) Localization of CENP-E to the outer kinetochore is affected <br />          Sgt1 siRNA <br />  <br />  <br />  <br />  <br />                                                                            by loss of Sgt1. (C) Localization of CENP-F is also affected, whereas there is <br />                                                                            substantial residual cytoplasmic staining. (D) CENP-I is absent from the <br />                                                                            inner kinetochore of cells lacking Sgt1. (E) Localization of CENP-C is <br />                                                                            unaffected. <br />  <br />  <br />  <br />  Function of human Sgt1 in kinetochore assembly <br /> P. Steensgaard et al                                                                                         <br /> PBS and 12.5 ml RNase 1 mg/ml in PBS and stained overnight at <br />                                                                               <br /> 4 1C before acquisition. To detect BrdU and DNA, exponentially                <br /> growing cells were incubated with 30 mM BrdU (Calbiochem                           <br /> Corp.) for 16 h at 37 1C, fixed in 70% ethanol and kept at 4 1C.            <br /> After DNA denaturation, the cells were washed in 0.1 M sodium                  <br /> tetraborate (pH 8.5). Cells were permeabilized with 0.1% Triton               <br /> X-100 in PBS (Sigma), and incubated with 200 ml of anti-BrdU                      <br /> mAb (Becton Dickinson) diluted 1:5 in PBS plus 1% NGS for 1 h                <br /> at 20 1C in the dark. After three PBS washings, the pellet was                      <br /> incubated with 200 ml of FITC-conjugated goat anti-mouse anti- <br />                                                                               <br /> body (Sigma) diluted 1:50 in PBS plus 1% NGS. For DNA                       <br /> quantification, cells were finally resuspended in 2 ml of PI solution              <br /> containing 2 mg/ml PI in PBS and 25 ml of RNase 1 mg/ml in water,                  <br /> and stained overnight at 4 1C in the dark. Samples were acquired              <br /> on a FACSCalibur (Becton Dickinson) flow cytometer. At least                          <br /> 10,000 events were acquired. Analysis was performed using                      <br /> CellQuest 3.3 (Becton Dickinson).                                                     <br /> Supplementary information is available at EMBO reports online                       <br /> </body></html>