<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2001 Genetics Society America <br />  <br />  <br />  <br /> The G-Protein &#226;?&#164;-Subunit GPB-2 Caenorhabditis elegans Regulates Go&#226;?&#163;&#226;&#8364;&#8220;Gq&#226;?&#163; <br />   Signaling Network Through Interactions With Regulator G-Protein <br />                    Signaling Proteins EGL-10 EAT-16 <br />  <br />     Alexander M. van der Linden, Femke Simmer, Edwin Cuppen Ronald H. A. Plasterk <br />                    Hubrecht Laboratory, Centre Biomedical Genetics, Uppsalalaan 8, 3584 CT, Utrecht, The Netherlands <br />                                                        Manuscript received November 15, 2000 <br />                                                       Accepted publication February 9, 2001 <br />  <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 The genome Caenorhabditis elegans harbors genes G-protein &#226;?&#164;-subunits. Here, the <br />              characterization second G-protein &#226;?&#164;-subunit gene gpb-2. In contrast gpb-1, gpb-2 essential <br />              gene though, like gpb-1, gpb-2 expressed development, nervous system, muscle <br />              cells. A loss-of-function <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span> gpb-2 produces variety behavioral defects, including delayed egg <br />              laying reduced pharyngeal pumping. Genetic analysis shows GPB-2 interacts GOA-1 <br />              (homologue mammalian Go&#226;?&#163;) EGL-30 (homologue mammalian Gq&#226;?&#163;) signaling pathways. GPB-2 <br />              similar divergent mammalian G&#226;?&#164;5 subunit, shown mediate specific <br />              interaction G&#226;?&#165;-subunit-like (GGL) domain RGS proteins. We GPB-2 physically <br />              genetically interacts GGL-containing RGS proteins EGL-10 EAT-16. Taken together, our <br />              results suggest GPB-2 works concert RGS proteins EGL-10 EAT-16 regulate GOA-1 <br />              (Go&#226;?&#163;) EGL-30 (Gq&#226;?&#163;) signaling. <br />  <br />  <br />  <br /> H      ETEROTRIMERIC G proteins, consisting a <br />         guanine nucleotide-binding &#226;?&#163;-subunit &#226;?&#164;&#226;?&#165; <br /> dimer, act signal-transducing molecules couple <br />                                                                              16, regulator G-protein signaling proteins (RGS) <br />                                                                              and/or DGK-1 (DAG-kinase; Hajdu-Cronin et al. 1999; <br />                                                                              Miller et al. 1999), identified suppressors of <br /> serpentine transmembrane receptors wide variety                         activated Go&#226;?&#163; (Hajdu-Cronin et al. 1999; Miller et al. <br /> of intracellular effectors (Kaziro et al. 1991; Simon et                     1999; Nurrish et al. 1999). RGS proteins function as <br /> al. 1991). The G&#226;?&#163;-subunit G&#226;?&#164;&#226;?&#165; dimer inde-                        GTPase-activating proteins (GAPs) accelerate the <br /> pendently interact regulate downstream ef-                          intrinsic GTPase activity G&#226;?&#163;-subunit (N. Watson <br /> fector molecules (Clapham Neer 1993). Despite                            et al. 1996; Koelle 1997; Neer 1997). Genetic analysis <br /> the extensive data biochemical properties                          shows EAT-16 (RGS) functions downstream of, or <br /> these components, major questions re-                        parallel <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>, GOA-1 (Go&#226;?&#163;). However, acting <br /> garding mechanisms G&#226;?&#164;&#226;?&#165; functions                           GAP GOA-1 (Go&#226;?&#163;), EAT-16 acts GAP for <br /> in vivo, relationship G&#226;?&#164;&#226;?&#165; dimers                         EGL-30 (Gq&#226;?&#163;; Hajdu-Cronin et al. 1999). Thus, Go&#226;?&#163; <br /> their potential G&#226;?&#163; partners. For example, unclear                      negatively regulate Gq&#226;?&#163; signaling EAT- <br /> how given G&#226;?&#164;&#226;?&#165; dimer interacts distinct G&#226;?&#163; signal-                     16 functioning direct effector GOA-1 indi- <br /> ing cascades given cell tissue vivo. Genetic                      rectly G&#226;?&#165;-subunit-like (GGL) domain EAT- <br /> analysis model organism Caenorhabditis elegans                        16 functionally mimic G&#226;?&#165;-subunit the <br /> may provide insight vivo G&#226;?&#164;&#226;?&#165; function                        heterotrimeric G-protein complex (Guan Han <br /> its interaction previously studied G&#226;?&#163; signaling path-                   1999). Another GGL-containing RGS protein C. ele- <br /> ways.                                                                        gans, EGL-10, previously identified upstream <br />    Recent genetic experiments C. elegans demonstrate                      negative regulator GOA-1 (Go&#226;?&#163;) signaling (Koelle <br /> that GOA-1 (Go&#226;?&#163;) antagonizes EGL-30 (Gq&#226;?&#163;) signaling,                         Horvitz 1996). Together, studies suggest <br /> with Go&#226;?&#163; pathway acting upstream of, parallel to,                     RGS proteins EGL-10 EAT-16 inhibit Go&#226;?&#163; <br /> the Gq&#226;?&#163; pathway (Hajdu-Cronin et al. 1999; Miller et                         Gq&#226;?&#163; signaling C. elegans, respectively, that <br /> al. 1999). Both Go&#226;?&#163; Gq&#226;?&#163; signaling pathway regu-                      especially EAT-16 important role mediating <br /> late neurotransmitter secretion C. elegans (Rand                      cross-talk Go&#226;?&#163; Gq&#226;?&#163;. At point, however, <br /> Nonet 1997; Lackner et al. 1999; Nurrish et al. 1999).                       known G&#226;?&#164;-subunits function <br /> This antagonism Go&#226;?&#163; Gq&#226;?&#163; involves EAT-                            Go&#226;?&#163;&#226;&#8364;&#8220;Gq&#226;?&#163; signaling network. <br />                                                                                 Two genes encoding G-protein &#226;?&#164;-subunits been <br />                                                                              identified complete C. elegans genome: gpb-1 (van <br />                                                                              der Voorn et al. 1990) gpb-2. The G-protein <br />   Corresponding author: Ronald H. A. Plasterk, Hubrecht Laboratory, <br /> Centre Biomedical Genetics, Uppsalalaan 8, 3584 CT, Utrecht,             &#226;?&#164;-subunit gene, gpb-1, expressed develop- <br /> The Netherlands. E-mail: plasterk@niob.knaw.nl                               ment tissues germline, loss of <br /> Genetics 158: 221&#226;&#8364;&#8220;235 (May 2001) <br />  222                                                 A. M. van der Linden et al. <br />  <br /> gpb-1 results embryonic lethality (Zwaal et al. 1996).           inserting &#226;&#171;&#190;3-kb fragment amplified PCR wild-type <br /> Here, second G-protein &#226;?&#164;-subunit                   gpb-2 genomic sequence using primers GFP-BTWO3 <br />                                                                     (5&#226;&#172;&#732;-ATAGCATGCGGCTCTAGATAGGTATGTAGA) GFP- <br /> gene, gpb-2, expressed neurons muscle             BWTO4 (5&#226;&#172;&#732;-TAGGATCCCATGCATAATACTTTTCCCAC), <br /> cells loss gpb-2 lethal, affects behav-      containing unique SphI BamHI site (shown bold), <br /> iors egg laying pharyngeal pumping. Ani-                respectively, SphI BamHI sites pPD95.77 (A. <br /> mals lacking gpb-2 goa-1 function syn-              Fire, personal communication). The construct pRP2017 was <br /> thetic larval lethal <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span>, rescued               injected 50 &#226;?&#174;g/ml 150 &#226;?&#174;g/ml pMH86 (Han and <br />                                                                     Sternberg 1991) CB1282 [dpy-20(e1282)IV]. This trans- <br /> EGL-30 activity reduced, suggesting Gq&#226;?&#163; activity            genic array integrated irradiating animals 40 Gy <br /> causes gpb-2; goa-1 double mutant synthetic lethality.          gamma radiation 137Cs source (Way et al. 1991) <br /> GPB-2 homologous divergent mammalian                 produce NL2784 (dpy-20(e1282)IV; pkIs1303[gpb-2::gfp,dpy- <br /> G&#226;?&#164;5 subunit that, unlike mammalian G&#226;?&#164;-sub-                20(&#226;&#171;&#185;)]). NL2784 backcrossed N2 wild-type back- <br /> units (G&#226;?&#164;1-4), ability interact GGL             ground produce NL2785. Cells identified reference <br />                                                                     Sulston Horvitz (1977) White et al. (1986). <br /> domain subset RGS proteins vitro (Snow et al.               Generation transgenic rescue gpb-2 loss-of-function <br /> 1998, 1999; Levay et al. 1999). Our results               mutation: A deletion mutant gpb-2, pk751, isolated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> GPB-2 interacts GGL-containing RGS pro-                chemical deletion library using primers BTWO5 (5&#226;&#172;&#732;-ACAAT <br /> teins C. elegans, EGL-10 EAT-16, GPB-2              TGGCAAATGAAGCCG) BTWO6 (5&#226;&#172;&#732;-TCAACGGAAATTG <br /> regulates functions proteins Go&#226;?&#163;&#226;&#8364;&#8220;Gq&#226;?&#163;            AGAGATG) nested primers BTWO7 (5&#226;&#172;&#732;-CACAAGCT <br />                                                                     TAATGACATTCC) BTWO8 (5&#226;&#172;&#732;-AGAAGCCGTGACG <br /> signaling network.                                                  GATGACC) described Jansen et al. (1997). The PCR <br />                                                                     product detecting deletion pk751 sequenced and <br />                                                                     confirmed Southern analysis (data shown). The follow- <br />               MATERIALS AND METHODS                                 ing sequence capitals deleted pk751: (5&#226;&#172;&#732;&#226;&#8364;&#8221;tagtcttc <br />    Nematode strains, culturing, manipulation: General           AAAAATTGG&#226;&#8364;&#8221;TGTAAAAAaagagtg&#226;&#8364;&#8221;). The pk751 allele was <br /> methods used culturing, manipulation, genetics C.        backcrossed times N2 wild-type background, re- <br /> elegans described Lewis Fleming (1995). DNA          sulting strain NL2001 [gpb-2(pk751)I]. <br /> transformation assays C. elegans microinjection          The behavioral defects pk751 complemented with <br /> described Mello Fire (1995). Unless indicated,           wild-type gpb-2 genomic constructs, pRP2015 (see below) <br /> experiments performed 20&#226;&#172;&#353;. The following strains       pRP2051. The rescuing construct pRP2051, containing a <br /> used study: Bristol N2, PS4034 [eat-16(sy438)I; Hajdu-      4.7-kb SpeI-XbaI fragment, including gpb-2 genomic se- <br /> Cronin et al. 1999], DA0823 [egl-30(ad805)I], MT1434 [egl-          quence &#226;&#181;&#8216;1.3 kb upstream sequence, injected at <br /> 30(n686)I; Brundage et al. 1996], NL561 [goa-1(pk62)I; Men-         20 &#226;?&#174;g/ml 150 &#226;?&#174;g/ml pMH86 (Han Sternberg 1991) <br /> del et al. 1995], MT363 [goa-1(n363)I; Se&#194;&#180;galat et al. 1995],       NL2002 [dpy-20(e1282)IV; gpb-2(pk751)I], resulting in <br /> DR102 [dpy-5(e61)unc-29(e403)I], CB73 [unc-15(e73)I], CB1282        strains NL2771, NL2772, NL2773 [dpy-20(e1282)IV; <br /> [dpy-20(e1282)IV], MT8504 [egl-10(md176)V; Koelle               pkEx1321, pkEx1322, pkEx1323, respectively]. Homozygos- <br /> Horvitz 1996], MT1083 [egl-8(n488)V; Trent et al. 1983;             ity pk751 allele confirmed single worm PCR. <br /> Miller et al. 1999], KP1097 [dgk-1(nu62)X; Nurrish et              Overexpression GPB-2: We generated multiple trans- <br /> al. 1999].                                                          genic lines overexpress GPB-2 transgene carrying <br />    Sequence analysis GPB-2: The GENEFINDER prediction            multicopy array, extrachromosomal integrated, of <br /> of gpb-2 open reading frame, annotated C. elegans     gpb-2 gene control endogenous promoter. <br /> database ACeDB (Eeckman Durbin 1995), confirmed             The rescuing construct pRP2051 injected different con- <br /> by cloning sequencing cDNA (see Yeast two-hybrid         centrations plasmid DNA 150 &#226;?&#174;g/ml pMH86 (Han <br /> analysis). The coding sequence GPB-2 submitted          Sternberg 1991) CB1282 [dpy-20(e1282)IV] 20 <br /> to DDBJ/EMBL/GenBank databases accession no.              &#226;?&#174;g/ml produce NL2757 (dpy-20(e1282)IV; pkEx1318 [gpb- <br /> AF291847. The percentages identity similarity given          2(&#226;&#171;&#185;),dpy-20(&#226;&#171;&#185;)]), 30 &#226;?&#174;g/ml produce NL2758 [dpy- <br /> above derived using Genetics Computer Group                20(e1282)IV; pkEx1319], 50 &#226;?&#174;g/ml produce NL2759 <br /> (GCG, version 9.1) GAP program using default parameters.            [dpy-20(e1282)IV; pkEx1320]. All strains obtained showed no <br /> Multiple alignments GPB-2, mammalian, Dictyostelium,             defects egg laying, locomotion, pharyngeal pumping <br /> and Drosophila G -protein &#226;?&#164;-subunit sequences (Swiss-Prot ac-       compared wild type (data shown). Another rescuing <br /> cession nos. P04901, P54311, P54312, P11016, P29387,                construct pRP2015, containing 6.9-kb SpeI fragment includ- <br /> AF291846, AF291847, P16520, P36408, O14775, P29829)             ing gpb-2 genomic sequence &#226;&#181;&#8216;3.5 kb upstream <br /> were using GCG PILEUP program using default                sequence, injected 50 &#226;?&#174;g/ml 150 &#226;?&#174;g/ml pMH86 <br /> parameters.                                                         (Han Sternberg 1991) CB1282 [dpy-20(e1282)IV] <br />    GFP reporter constructs: A translational fusion gpb-2      produce NL1557 (dpy-20(e1282)IV; pEx557[gpb-2(&#226;&#171;&#185;),dpy- <br /> gfp (pRP2016) constructed inserting &#226;&#171;&#190;3-kb fragment         20(&#226;&#171;&#185;)]). This transgenic array integrated irradiating <br /> amplified PCR wild-type gpb-2 genomic sequence using          animals 40 Gy gamma radiation 137Cs source <br /> the primers green fluorescent protein (GFP)-BTWO1 (5&#226;&#172;&#732;-ATA           (Way et al. 1991) produce NL1559 (dpy-20(e1282)IV; <br /> GCATGCTTCCTGGTGATCAGGTCATGT) GFP-BWTO2                          pkIs559[gpb-2(&#226;&#171;&#185;),dpy-20(&#226;&#171;&#185;)]). NL1559 backcrossed two <br /> ( 5 &#226;&#172;&#732; - TAGGATCCAATAGCACATGTTGAATCTCC), contain-                    times N2 wild-type background produce NL2004. The <br /> ing unique SphI BamHI site (shown bold), respectively,     strain NL2004 used behavioral assays described below. <br /> into SphI BamHI sites pPD95.77 (A. Fire, personal        To confirm integrated transgene pkIs559[gpb-2(&#226;&#171;&#185;)] <br /> communication). The construct pRP2016 injected 50            functional, crossed pkIs559[gpb-2(&#226;&#171;&#185;)] NL2002 [dpy- <br /> &#226;?&#174;g/ml 100 &#226;?&#174;g/ml pMH86 (Han Sternberg 1991)                 20(e1282)IV; gpb-2(pk751)I], resulting strain NL2006 (dpy- <br /> into CB1282 [dpy-20(e1282)IV] produce NL1562 (dpy-               20(e1282)IV; gpb-2(pk751)I; pkIs559[gpb-2(&#226;&#171;&#185;),dpy-20(&#226;&#171;&#185;)]). <br /> 20(e1362)IV; pkEx563[gpb-2::gfp,dpy-20(&#226;&#171;&#185;)]). A second transla-         A construct gpb-2 expression control a <br /> tional fusion gpb-2 gfp (pRP2017) constructed          heat-shock promoter (hsp) generated cloning &#226;&#171;&#190;800- <br />                                               GPB-2 Interacts With EAT-16 EGL-10                                             223 <br />  <br /> bp SpeI-AvrII hsp16.2-gpb-2 containing PCR fragment       Horvitz 1996). Pharyngeal pumping food assayed <br /> of AvrII-SpeI fragment gpb-2 pRP2051. In        measuring number pumps minute. L4 staged <br /> PCR, fragments amplified. The 518-bp fragment       hermaphrodites placed separate NGM agar plates <br /> was amplified hsp16.2 promoter pPD49.78                seeded 10 &#226;?&#174;l OP50 allowed develop 29 <br /> (Stringham et al. 1992) using primers HSP1 (5&#226;&#172;&#732;-GGTCGA          hr 20&#226;&#172;&#353;. Pharyngeal pumps counted 2-min intervals <br /> CACTAGTGGATCAAGAGC) HSP3 (5&#226;&#172;&#732;-GAGTTTTCTG                        count represents pumps. The number of <br /> GCATGGTACCGTAGACGC). In way, PCR fragment                 counts multiplied 1.5 yield pumps minute <br /> becomes flanked unique SpeI site (shown        (Hajdu-Cronin et al. 1999). Assays egg laying 5 mg/ <br /> boldface type) the 14 bp         ml serotonin (5-HT) 0.75 mg/ml imipramine per- <br /> gpb-2. The second 296-bp fragment amplified gpb-2           formed interpreted described Trent et al. (1983), <br /> using primers HSP2 (5&#226;&#172;&#732;-CGACGGTACCATGCCAGAA                     exception newly laid eggs counted after <br /> AACTCTCAG) HSP4 (5&#226;&#172;&#732;-CTAACCATGGAATTGTAAAG).                     90 min. Imipramine 5-HT dissolved M9 buffer. <br /> This PCR fragment flanked 10 bp         Aldicarb sensitivity measured assaying time <br /> of hsp16.2 unique AvrII site (shown     course onset paralysis following acute exposure of <br /> in bold). In second PCR, fragments allowed          population animals aldicarb quantifying the <br /> anneal overlapping flanks (underlined) used           population growth rates various concentrations aldicarb <br /> template amplify &#226;&#171;&#190;800-bp fragment using outer prim-       described Lackner et al. (1999) Miller et al. (1999). <br /> ers (HSP1 HSP4). The SpeI-AvrII PCR fragment cut           For statistical analysis multiple groups, ANOVA <br /> and cloned XbaI-AvrII sites construct pRP2051,     followed directed Student&#226;&#8364;&#8482;s t-test used. All results are <br /> resulting pRP2201. The region PCR fragment used          given mean &#226;&#171;&#190; standard errors. <br /> in cloning sequenced completely free              Synthetic lethality: We constructed gpb-2(pk751)goa-1(pk62)/ <br /> of amplification errors. Injection pRP2201        unc-15(e73) mutant mutations closely <br /> varying concentrations plasmid DNA (10&#226;&#8364;&#8220;50 &#226;?&#174;g/ml)           linked right arm chromosome I. The percentage of <br /> 150 &#226;?&#174;g/ml pMH86 (Han Sternberg 1991) CB1282               <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>survival</span> gpb-2 goa-1 double mutant scored placing <br /> [dpy-20(e1282)IV] produce NL2775, NL2776, NL2780            gpb-2 goa-1 / unc-15 L4 heterozygotes separate OP50- <br /> [dpy-20(e1282)IV; pkEx1324, pkEx1325, pkEx1326, respec-        seeded NGM agar plates. The heterozygous animals given <br /> tively]. Transgenic animals different larval stages (L1      opportunity lay eggs 1&#226;&#8364;&#8220;2 days. All progeny were <br /> adult) placed Escherichia coli OP50-seeded NGM agar        transferred separate plates followed generation <br /> plates heat shocked 2 hr 33&#226;&#172;&#353; induce promoter         determine genotype. This yielded 131 Unc animals, <br /> activity.                                                          635 heterozygotes, 195 gpb-2 goa-1 double mutant animals <br />    Construction double mutant strains: Double mutants           95% arrested different larval stages. Similarly, <br /> used study generated standard genetic methods.     picked progeny gpb-2(pk751)/&#226;&#171;&#185;; dgk-1(nu62) het- <br /> Homozygosity alleles confirmed sequenc-       erozygotes. This yielded 69 hyperactive dgk-1 animals, 121 het- <br /> ing amplified genomic DNA strains containing eat-16(sy438),     erozygotes segregated arrested larvae, 53 gpb-2; dgk-1 <br /> egl-30(ad805), egl-30(n686), dgk-1(nu62) single worm     double mutant animals. <br /> PCR strains containing gpb-2(pk751), egl-8(n488), goa-1       We constructed transgenic gpb-2 goa-1 double mutant ani- <br /> (pk62). To detect egl-10(md176), used single worm PCR           mals containing gpb-2 wild-type sequence, rescues <br /> using primers fail amplify rearranged egl-10        synthetic <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>. The rescuing construct <br /> region (Koelle Horvitz 1996). In addition, double          pRP2051 injected 50 &#226;?&#174;g/ml 100 &#226;?&#174;g/ml pRF4 and <br /> mutant strains confirmed crossing N2 wild-type        50 &#226;?&#174;g/ml pRP2017 markers gpb-2(pk751)goa-1(pk62)/ <br /> males re-isolate mutations.                                unc-15(e73) heterozygous animals produce NL2728 gpb-2 <br />    Phenotypic analyses gpb-2 mutant, double mutant,         (pk751)goa-1(pk62);pk1344Ex[gpb-2(&#226;&#171;&#185;),gpb-2::gfp, rol-6 (su1006)]. <br /> and transgenic lines: The rate egg laying assayed        Animals lost transgenic array germline resulted <br /> two methods described (Brundage et al. 1996; Koelle             larval arrest (&#226;&#172;&#381;96%). <br /> and Horvitz 1996). In short, egg-laying assay        To generate egl-30(n686)gpb-2(pk751)goa-1(pk62) triple <br /> number unlaid eggs uterus determined. L4 staged      mutant, egl-30 /&#226;&#171;&#185; males mated gpb-2 goa-1/&#226;&#171;&#185;&#226;&#171;&#185; <br /> hermaphrodites placed E. coli OP50-seeded NGM              hermaphrodites, F1 cross-progeny transferred sep- <br /> agar plates allowed develop 36 hr 20&#226;&#172;&#353;. Adults   arate plates. From plates segregated Egl arrested <br /> were treated sodium hypochlorite eggs counted        larvae, 52 Egl F2 progeny transferred separate plates. <br /> using dissection microscope (Leica MZ75). In second          Two 52 Egls egl-30 gpb-2 goa-1 / egl-30 &#226;&#171;&#185;&#226;&#171;&#185; egl- <br /> egg-laying assay, developmental stage newly laid eggs   30 gpb-2 goa-1 triple mutants isolated. As control, we <br /> determined placing L4 staged hermaphrodites OP50-            crossed egl-30 gpb-2 goa-1 hermaphrodites dpy-5 unc-29/ <br /> seeded NGM agar plates. After 29 53 hr 20&#226;&#172;&#353;, develop-     &#226;&#171;&#185;&#226;&#171;&#185; males selected 19 Dpy non-Unc 16 Unc non-Dpy <br /> mental stage newly laid eggs determined 2.5-hr         F2 recombinants. Fourteen 19 Dpy non-Unc dpy-5 <br /> interval counting cells high power dissection        gpb-2 goa-1/dpy-5&#226;&#171;&#185;&#226;&#171;&#185; unc-29 recombinants one- <br /> microscope (Wild M3C). Newly laid eggs divided           fourth progeny arrested larvae. Five 16 Unc <br /> three categories: eight-cell, nine-cell comma stage,     non-Dpy egl-30 gpb-2 goa-1 unc-29/&#226;&#171;&#185; dpy-5 &#226;&#171;&#185;&#226;&#171;&#185; unc-29 <br /> and post-comma stage. The rate locomotion assayed       recombinants did produce lethal progeny. <br /> by measuring frequency body bends hermaphro-              An egl-8(n488); dpy-5(e61)gpb-2(pk751)goa-1(pk62)/&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; mu- <br /> dite. L4 staged hermaphrodites placed OP50-seeded          tant generated. We placed 10 egl-8; dpy-5 gpb-2 goa-1/&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br /> NGM agar plates allowed develop 24 hr           L4 heterozygotes separate plates. The heterozygous ani- <br /> 20&#226;&#172;&#353;. At stage, locomotion adult hermaphrodite       mals given opportunity lay eggs 2 days. A total <br /> is affected presence excessive unlaid eggs. Each    119 Dpy F1 larvae transferred separate plates and <br /> animal transferred lawn freshly OP50-seeded     followed generation determine genotype. Of the <br /> NGM agar plate 15 min measuring locomotion               119 Dpy larvae, 101 arrested did produce progeny, <br /> counting body bends 3-min intervals. A body bend            18 viable Dpy animals dpy-5 gpb-2 goa-1/dpy-5&#226;&#171;&#185;&#226;&#171;&#185; <br /> defined movement quarter body <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> forward     recombinants. Arrested Dpy larvae seen prog- <br /> or backward direction (Brundage et al. 1996; Koelle            eny recombinants. In experiments, homozygosity for <br />  224                                                  A. M. van der Linden et al. <br />  <br /> gpb-2(pk751) egl-8(n488) presence goa-1(pk62) <br /> was confirmed single worm PCR. Homozygosity egl-30 <br /> and dgk-1 allele confirmed sequencing genomic DNA <br /> of animals containing n686 nu62 mutations, re- <br /> spectively. <br />   Yeast two-hybrid analysis: The coding sequence GPB-1 <br /> (GenBank accession no. AF291846), GPB-2 (AF291847), <br /> GPC-1 (AF291848), GPC-2 (AF291849) regions <br /> encoding GGL EGL-10 (AF291850) EAT-16 <br /> (AF291851) amplified total RNA C. elegans <br /> N2 strain RT-PCR. First strand synthesis initiated using a <br /> mixture random hexamers oligo(dT) oligonucleotides <br /> [100 ng/&#226;?&#174;l total RNA, 200 nm oligo(20)dT, 180 units/&#226;?&#174;l <br /> random hexamers, 0.5 units/&#226;?&#174;l avian myeloblastosis virus re- <br /> verse transcriptase (Roche), 1 mm dNTP, 50 mm Tris-HCl pH <br /> 8.5, 8 mm MgCl2, 70 mm KCl, 1 mm dithiothreitol, 10 ng/&#226;?&#174;l <br /> BSA], cDNAs amplified PCR [10 mm Tris-HCl pH <br /> 8.3, 50 mm KCl, 1.5 mm MgCl2, 0.025 units/&#226;?&#174;l Taq polymerase <br /> (GIBCO, Gaithersburg, MD), 0.2 mm dNTP, 200 nm each <br /> primer] using specific oligonucleotides (GPB-1: 5&#226;&#172;&#732;-GGGAAT <br /> TCATGAGCGAACTTGACCAAC 5&#226;&#172;&#732;-CCCTCGAGTTAAT <br /> TCCAGATCTTGAG; GPB-2: 5&#226;&#172;&#732;-GGGAATTCATGCCAGAAA <br /> ACTCTCAGC 5&#226;&#172;&#732;-CCCTCGAGTCAAGCCCAAATGCGA <br /> ATTG; GPC-1: 5&#226;&#172;&#732;-GGGAATTCATGGAAAACATCAAGGCATC <br /> and 5&#226;&#172;&#732;-CCCTCGAGTTAGAGTACTGAACAGCTT; GPC-2: 5&#226;&#172;&#732;- <br /> GGGAATTCATGGATAAATCTGACATGC 5&#226;&#172;&#732;-CCCTCG <br /> AGTTAGAGCATGCTGCACTTG; EGL-10-GGL: 5&#226;&#172;&#732;-CCCCAT <br /> ATGCCTGGATTACGCCGGTGTAC 5&#226;&#172;&#732;-CCCCTCGAGA <br /> CTATCCTCCCAAAGCTTGAG; EAT-16-GGL: 5&#226;&#172;&#732;-GGCCATA <br /> TGCGGCAGAATGCACAAGGTTA 5&#226;&#172;&#732;-GGCTCGAGTGT <br /> GTTTCGAGCACTTGCCGTC) containing recognition sites <br /> for restriction enzymes in-frame cloning appropriate <br /> two-hybrid bait prey vector (pGBK-T7 pGAD-T7, CLON- <br /> TECH, Palo Alto, CA). Final cDNA constructs checked               Figure 1.&#226;&#8364;&#8221;An alignment dendogram various G-protein <br /> by sequencing potential differences spliced gene        &#226;?&#164;-subunits different species gene structure of <br /> products predicted ACeDB (no differences found)              gpb-2. (A) Alignment dendogram generated PILEUP <br /> and absence mutations. Combinations bait           algorithm using default program parameters. GPB-2 most <br /> prey plasmids introduced yeast strain AH109 poly-         related human G&#226;?&#164;5. (B) Genomic organization gpb-2 <br /> ethylene glycol/lithium acetate transformation assayed           gene. The solid boxes coding sequence. The region <br /> for interactions growth medium lacking histidine           deleted gpb-2(pk751), rescuing constructs, pRP2015 and <br /> adenine, according manufacturer&#226;&#8364;&#8482;s protocols (CLON-            pRP2051, gfp reporter constructs, pRP2016 and <br /> TECH).                                                               pRP2017, indicated. S, SpeI; X, XbaI. <br />  <br />  <br />                            RESULTS                                   subunit&#226;&#8364;&#8221;64% identity 73% similarity amino <br />                                                                      acid sequence level. <br />    GPB-2 homologue divergent mammalian                      gpb-2 widely expressed neurons muscle cells: <br /> G&#226;?&#164;5 subunit: The C. elegans genome project identified                To analyze GPB-2 expression, used translational <br /> two G-protein &#226;?&#164;-subunit genes. The G-protein                   fusions gpb-2 upstream control sequence the <br /> &#226;?&#164;-subunit, gpb-1 (located chromosome II, F13D12.7),               gene encoding GFP (Chalfie et al. 1994). One transla- <br /> is 86% identical conserved mammalian G&#226;?&#164;-sub-                  tional fusion contained gfp fused exon and <br /> units (van der Voorn et al. 1990; Zwaal et al. 1996).                fifth exon gpb-2 (Figure 1B). Both <br /> However, predicted sequence gpb-2 (located                 gfp reporter constructs showed identical expression pat- <br /> chromosome I, F52A8.2) indicates second                     terns development nervous system <br /> G-protein &#226;?&#164;-subunit conserved. We                  muscle cells. Animals integrated gpb-2::gfp <br /> isolated sequenced full-length GPB-2 cDNA (Gen-                  array showed expression embryos comma <br /> Bank accession no. AF291847) differences                stage broad expression seen, excluding the <br /> with sequence predicted GENEFINDER. The de-                   dorsal posterior site comma stage embryo (Figure <br /> duced GPB-2 protein consists 356 amino acid resi-                 2A). From comma stage onward, tissues differ- <br /> dues, encoding seven WD40 repeats characteristic                 entiate organized, hatching the <br /> for mammalian &#226;?&#164;-subunits. However, amino acid                embryos, gpb-2 expression stronger broadly ex- <br /> sequence shares &#226;&#181;&#8216;50% identity GPB-1                pressed head tail ganglia. In larvae adult <br /> mammalian G&#226;?&#164;1-4 subunits (Figure 1A). Interestingly,                 animals, gpb-2 expression seen neu- <br /> GPB-2 similar divergent mammalian G&#226;?&#164;5                 rons, including neurons located head ganglia <br />                                              GPB-2 Interacts With EAT-16 EGL-10                                                225 <br />  <br />                                                                                                            Figure 2.&#226;&#8364;&#8221;Expression pat- <br />                                                                                                         tern gpb-2 detected us- <br />                                                                                                         ing translational GFP gene <br />                                                                                                         fusions. In embryo, ex- <br />                                                                                                         pression gbp-2 de- <br />                                                                                                         tected comma stage <br />                                                                                                         (A, lower embryo) the <br />                                                                                                         1.5-fold stage (A, upper em- <br />                                                                                                         bryo). In adult animals, ex- <br />                                                                                                         pression observed neu- <br />                                                                                                         rons head (B; arrow <br />                                                                                                         indicates head ganglia; ph, <br />                                                                                                         pharyngeal muscle) tail <br />                                                                                                         (C; tg, tail ganglia) regions. <br />                                                                                                         (D) Expression gpb-2 in <br />                                                                                                         vulva muscle cells (vm) <br />                                                                                                         shown, region <br />                                                                                                         surrounding vulva ex- <br /> pression gpb-2 seen ventral nerve cord neurons (arrows indicate single neurons groups neurons). Furthermore, <br /> high expression levels gpb-2 body-wall muscle cells (E) CAN HSN cells (F). <br />  <br />  <br />  <br /> (Figure 2B), ventral nerve cord (Figure 2D),                  stage, maternally rescued animals arrest the <br /> the tail ganglia (Figure 2C). Additionally, gpb-2 expres-             stage larval development (Zwaal et al. 1996). <br /> sion seen hermaphrodite-specific neurons                   However, fact gpb-2(pk751) animals viable, <br /> (HSNs), control egg laying (Figure 2F)                  unlike gpb-1 mutants, allows study function of <br /> canal cell-associated neurons (CANs),                       G&#226;?&#164;-subunit adult behaviors using genetic tools. <br /> thought function osmoregulation (Figure 2F). Be-                   GPB-2 modulates behaviors egg laying, loco- <br /> sides neuronal expression, high expression levels                  motion, pharyngeal pumping: To investigate the <br /> gpb-2 seen muscle cells pharynx (Fig-             function GPB-2 adult animals, quantified the <br /> ure 2B), vulva muscle cells (Figure 2D),                  behavioral defects induced loss-of-function gpb-2 <br /> body-wall muscle cells (Figure 2E). In males, similar                 (pk751) mutant overexpression GPB-2 (Table <br /> neuronal muscle expression observed. Thus,                    1 Figure 3). gpb-2 mutant animals active <br /> gpb-2 widely expressed neurons muscle               egg laying pharyngeal pumping subtle <br /> cells control distinct behaviors.                        defect backward movement; gpb-2 mutants tended to <br />    gpb-2 essential gene: To examine GPB-2                   backward exaggerated flexes the <br /> function vivo, identified loss-of-function gpb-2 mu-          tip tail touches head body bend <br /> tant using reverse-genetic approach. An EMS-derived                 (data shown). No defect defecation dauer <br /> deletion library (Jansen et al. 1997) screened                formation observed (data shown). The rate of <br /> deletions gpb-2 genomic sequence. We isolated                  egg laying determined analyzing stage of <br /> an allele, gpb-2(pk751), gpb-2 coding            newly laid eggs number unlaid eggs the <br /> sequence removed. As shown Figure 1B,                  uterus. We gpb-2 adult hermaphrodites laid <br /> 3380-bp deletion genomic sequence removes                 fewer eggs nine-cell comma stage develop- <br /> five exons gpb-2 1.5 kb downstream                ment (74%) compared wild type (96%) laid <br /> sequence. Consequently, pk751 leaves                  eggs post-comma stage (26%) com- <br /> seven WD repeats intact. Because seven WD repeats                 pared wild type (0%, Table 1). More strikingly, <br /> needed form functional &#226;?&#164;-subunit, likely              24 hr later, gpb-2 adult hermaphrodites laid <br /> pk751 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>null allele</span>. Animals heterozygous pk751                97% eggs post-comma stage compared <br /> were wild type development behavior. Animals                   6% wild-type animals. The number unlaid eggs <br /> homozygous pk751 viable, showed variety                accumulated gpb-2 mutant animals (32.1 &#226;&#171;&#190; 1.6) <br /> of behavioral defects (discussed below). The behavioral               higher wild-type animals (19 &#226;&#171;&#190; 1.2, <br /> defects pk751 complemented introducing                   Figure 3). One possibility serotonergic HSNs <br /> transgene carrying multicopy array, extrachro-               vulva muscle cells control egg laying are <br /> mosomal integrated, wild-type gpb-2 genomic                 defective gpb-2 mutant animals. Egg laying wild- <br /> sequence, demonstrating defects caused                type animals stimulated adding serotonin (5-HT) <br /> loss GPB-2 function (Table 1 Figure 3).                        imipramine, reuptake inhibitor 5-HT. Animals <br />    Mutant gpb-2 animals viable, GPB-2              nonfunctional HSNs respond exogenous 5-HT, <br /> does essential function GPB-1                  resistant imipramine (Trent et al. 1983). <br /> is redundant. To address question genetically               Animals resistant agents defects <br /> is difficult, gpb-1 embryos arrest four-cell             response egg-laying muscles. Adult gpb-2 <br />  226                                              A. M. van der Linden et al. <br />  <br />                                                           TABLE 1 <br />             Rate egg-laying behavior gain- loss-of-function gpb-2 mutant animals rescued transgenics <br />  <br />                                                                                    Developmental stage newly laid eggs <br />                                                                                          (% total no. eggs laid) <br />                                                     Time assayed <br />                                                        post-L4         One- eight-         Nine-cell     Post-       No. of <br /> Genotype                           Transgene      larval stage (hr)        cell               comma        comma       eggs laid <br /> N2 (wild type)                      None                 29                    6                 94            0            189 <br /> gpb-2(pk751)                        None                 29                    0                 74           26            114 <br /> gpb-2(pk751);pkIs559[gpb-2(&#226;&#171;&#185;)]      pRP2015              29                    4                 93            3            197 <br /> pkIs559[gpb-2(&#226;&#171;&#185;)]                   pRP2015              29                    8                 92            0            204 <br /> N2 (wild type)                      None                 53                    0                 94            6            136 <br /> gpb-2(pk751)                        None                 53                    0                  3           97            104 <br /> gpb-2(pk751);pkIs559[gpb-2(&#226;&#171;&#185;)]      pRP2015              53                    0                 95            5            132 <br /> pkIs559[gpb-2(&#226;&#171;&#185;)]                   PRP2015              53                    0                 97            3            140 <br />    At 20 adult animals assayed genotype (see materials methods). Newly laid eggs divided into <br /> three categories: one- eight-cell stage, nine-cell comma stage, post-comma stage. pkIs559[gpb-2(&#226;&#171;&#185;)] integrated <br /> transgene carrying multiple copies gpb-2 wild-type sequence overexpress GPB-2. <br />  <br />  <br /> mutant animals respond 5-HT imipramine,               duce lethargy delayed egg laying, putative <br /> indicating HSNs response egg-             null allele results larval lethality (Trent et al. 1983; <br /> laying muscles normal (Table 2). Despite un-              Brundage et al. 1996). Loss-of-function mutations in <br /> usual backward motion described, frequency                 goa-1 result starvation phenotype, premature egg <br /> body bends locomotion significantly dif-           laying, hyperactive locomotion (Mendel et al. 1995; <br /> ferent gpb-2 mutant animals compared wild               Se&#194;&#180;galat et al. 1995). To determine GPB-2 plays <br /> type (Figure 3). In addition behaviors, no-           role GOA-1 EGL-30 signaling, generated <br /> ticed well-fed adult gpb-2 mutant animals              double mutants gpb-2 goa-1 gpb-2 and <br /> starved appearance &#226;&#181;&#8216;3 days. Although animal              egl-30. When crossed gpb-2(pk751) goa-1(pk62) <br /> can appear starved number reasons, starved             background, saw synthetic phenotype&#226;&#8364;&#8221;lethality at <br /> appearance associated feeding defect              larval stage (&#226;&#172;&#381;95%). Of 195 gpb-2 goa-1 double <br /> (Avery 1993). Because gpb-2 highly expressed               mutant progeny goa-1 gpb-2/unc-15 heterozygous ani- <br /> the pharyngeal muscle cells, determined                mals, 10 escaped adulthood, turn <br /> the starvation phenotype gpb-2 mutant animals              produced inviable progeny (&#226;&#172;&#381;95%). There phe- <br /> caused reduced pharyngeal pumping. Indeed,               notypic difference progeny gpb-2 goa-1 <br /> pharynx pumped slowly gpb-2 mutant animals (129 &#226;&#171;&#190;              mutant animals escaped adulthood gpb-2 <br /> 8 pumps/min) compared wild type (230 &#226;&#171;&#190; 10                  goa-1 progeny heterozygous mothers. It argued <br /> pumps/min, Figure 3).                                             synthetic lethality result additive <br />    To analyze effect gpb-2 gene dosage behav-           feeding defect gpb-2 goa-1 mutants. There- <br /> ior, transgenic animals overexpress GPB-2            fore, investigated pumping rate gpb-2 goa-1 <br /> from chromosomal integrated transgene carrying                double mutant animals (155 &#226;&#171;&#190; 11 pumps/min, n &#226;&#171;&#189; 10) <br /> multicopy array gpb-2 gene, pkIs559[gpb-2(&#226;&#171;&#185;)].             gpb-2 goa-1 heterozygous mothers that <br /> Functionality transgene demonstrated            significant difference (P &#226;&#171;&#189; 0.18) the <br /> ability complement gpb-2(pk751) behavioral de-             rate pharyngeal pumping compared gpb-2 single <br /> fects (Table 1 Figure 3). However,              mutant adult animals (140 &#226;&#171;&#190; 8 pumps/min, Table 3). <br /> overexpression gpb-2 control endogenous           Thus, synthetic lethality gpb-2 goa-1 animals not <br /> (pkIs559[gpb-2(&#226;&#171;&#185;)]) heat-shock promoter (pkEx1324,             feeding defect result decrease <br /> pkEx1325, pkEx1326; data shown) did              pumping rate. To analyze synthetic <br /> any obvious phenotypes respect egg laying, loco-          phenotype gpb-2 goa-1 larvae, generated gpb-2 goa-1 <br /> motion, pharyngeal pumping (Table 1 Figure                double mutant animals rescued transgene con- <br /> 3). Thus, GPB-2 necessary normal egg laying,         taining gpb-2 wild-type sequence. Animals lose <br /> locomotion, pharyngeal pumping, overexpression                transgene germline broods lack <br /> of GPB-2 does affect behaviors.                         functional gpb-2 expression (Zwaal et al. 1996). We <br />    GPB-2 genetically interacts GOA-1 (Go&#226;?&#163;)               progeny gpb-2 goa-1 double mutant <br /> EGL-30 (Gq&#226;?&#163;): The behavioral phenotype gpb-2 mu-               animals lost gpb-2 rescuing transgene pumped <br /> tants affects similar behaviors goa-1 (Go&#226;?&#163;) egl-30         slowly (163 &#226;&#171;&#190; 9 pumps/min, n &#226;&#171;&#189; 10) showed pha- <br /> (Gq&#226;?&#163;) mutants. Reduction-of-function egl-30 alleles in-           ryngeal defects (data shown); pharynx com- <br />                                           GPB-2 Interacts With EAT-16 EGL-10                                          227 <br />  <br /> pressed shrunken 80% animals. This defect        peractive locomotion egg laying have <br /> may severely impair feeding, causing lethality       starvation phenotype, albeit severe goa-1 mu- <br /> we observed. However, exclude expla-          tants. We constructed double mutants gpb-2 <br /> nations lethality. Interestingly, evidence   (pk751) dgk-1(nu62) animals defec- <br /> that behaviors feeding affected         tive gpb-2 dgk-1 arrest (&#226;&#172;&#381;98%). One of <br /> gpb-2 goa-1 double mutants single mu-         53 gpb-2; dgk-1 double mutant progeny gpb-2/ <br /> tant. Before gpb-2 goa-1 double mutant larvae arrested,       &#226;&#171;&#185;; dgk-1 animals survived adulthood produced <br /> these animals moved uncoordinated fashion           progeny arrested larvae (&#226;&#172;&#381;98%). As gpb-2 <br /> occasionally observed gpb-2 goa-1 larvae rotated      goa-1 double mutants, phenotype progeny of <br /> half turns longitudinal axes        homozygous gpb-2; dgk-1 animals does differ from <br /> elongated.                                                    phenotype generation gpb-2; dgk-1 <br />    If combining mutations gpb-2 goa-1 causes         mutants, excluding maternal effects. In addition, <br /> synthetic lethal phenotype, double mutants            phenotype gpb-2; dgk-1 double mutants resembles <br /> gpb-2 dgk-1, encode putative downstream           phenotype gpb-2 goa-1 mutants, e.g., pharyngeal <br /> effector GOA-1 signaling, result similar   defects (95% animals) locomotory behavior. <br /> synthetic phenotype. Like goa-1, dgk-1 mutants hy-        These genetic data suggest gpb-2 acts parallel <br />                                                               pathway goa-1 dgk-1. <br />                                                                  Surprisingly, EGL-30 activity reduced gpb-2 <br />                                                               (pk751)goa-1(pk62) double mutant animals con- <br />                                                               structing egl-30(n686)gpb-2(pk751)goa-1(pk62) triple <br />                                                               mutant, lethality gpb-2 goa-1 double mutants was <br />                                                               suppressed. Only 1 169 egl-30 gpb-2 goa-1 triple <br />                                                               mutants egl-30 gpb-2 goa-1/egl-30 &#226;&#171;&#185;&#226;&#171;&#185; heterozygous <br />                                                               animals arrested larval stage did produce <br />                                                               offspring. We measured egg-laying activity, pha- <br />                                                               ryngeal pumping rate, frequency body bends <br />                                                               locomotion egl-30(n686)gpb-2(pk751)goa-1 <br />                                                               (pk62) triple mutants compared egl-30(n686) single <br />                                                               mutants. Like egl-30(n686) animals, triple mutant <br />  <br />  <br />  <br />                                                                  Figure 3.&#226;&#8364;&#8221;A loss-of-function mutation gpb-2 affects egg <br />                                                               laying pharyngeal pumping; overexpression GPB-2 does <br />                                                               significant effect frequency body bends <br />                                                               locomotion, egg laying, pharyngeal pumping. Egg <br />                                                               laying assayed number eggs contained the <br />                                                               uterus (A), frequency body bends locomotion <br />                                                               (B), pumping rate (C) N2 (wild-type) animals, <br />                                                               gpb-2 loss-of-function animals (pk751), animals transgenic <br />                                                               multicopy array containing gpb-2 gene (pkIs559[gpb-2 <br />                                                               (&#226;&#171;&#185;)]) gpb-2 loss-of-function (pk751) background a <br />                                                               wild-type background. ANOVA comparing groups differ- <br />                                                               ent variables conducted. A significant effect group was <br />                                                               noted unlaid eggs adult (F &#226;&#171;&#189; 43.7, <br />                                                               P &#226;&#172;? 0.001) pharyngeal pumping (F &#226;&#171;&#189; 26.7, P &#226;&#172;? 0.001). <br />                                                               No significant effect group noted frequency body <br />                                                               bends locomotion (F &#226;&#171;&#189; 0.85, P &#226;&#171;&#189; 0.47). Differences <br />                                                               genotypes unlaid eggs adult pharyn- <br />                                                               geal pumping analyzed using Student&#226;&#8364;&#8482;s t-test. gpb-2 <br />                                                               (pk751) mutant animals significantly different respect <br />                                                               unlaid eggs adult rate pharyn- <br />                                                               geal pumping N2 (wild-type) animals (P &#226;&#172;? 0.001), gpb- <br />                                                               2; (pkIs559[gpb-2(&#226;&#171;&#185;)]) animals (P &#226;&#172;? 0.001), (pkIs559[gpb- <br />                                                               2(&#226;&#171;&#185;)]) animals (P &#226;&#172;? 0.001). There significant differ- <br />                                                               ence (pkIs559[gpb-2(&#226;&#171;&#185;)]) N2 (wild type) the <br />                                                               unlaid eggs adult (P &#226;&#171;&#189; 0.051) pharyngeal pumping <br />                                                               rate (P &#226;&#171;&#189; 0.22) gpb-2(pk751); (pkIs559[gpb-2(&#226;&#171;&#185;)]) <br />                                                               N2 (wild type) unlaid eggs adult (P &#226;&#171;&#189; 0.059) <br />                                                               pharyngeal pumping rate (P &#226;&#171;&#189; 0.15). About 20 adult <br />                                                               animals assayed genotype (see materials and <br />                                                               methods). Values reported means &#226;&#171;&#190; standard errors. <br />  228                                              A. M. van der Linden et al. <br />  <br />                                                          TABLE 2 <br />                  Egg-laying assay loss-of-function gpb-2 animals presence 5-HT imipramine <br />  <br />                                    No. animals laying indicated number eggs treatment with: <br />                                     M9 buffer                           5-HT                     Imipramine <br />         Genotype            0      1&#226;&#8364;&#8220;3     4&#226;&#8364;&#8220;7     &#226;&#172;&#381;7         0   1&#226;&#8364;&#8220;3      4&#226;&#8364;&#8220;7     &#226;&#172;&#381;7     0     1&#226;&#8364;&#8220;3     4&#226;&#8364;&#8220;7     &#226;&#172;&#381;7 <br />         N2 (wild type)     19       7       0       0         0    0           7   18     1      7       8      10 <br />         gpb-2(pk751)       21       5       0       0         0    0           2   19     1      6      10       7 <br />           The response egg laying adult hermaphrodites tested adding exogenous serotonin (5-HT) and <br />         imipramine (see materials methods). M9 buffer used control. The number eggs laid by <br />         hermaphrodites indicated follows: 0 eggs, 1&#226;&#8364;&#8220;3 eggs, 4&#226;&#8364;&#8220;7 eggs, &#226;&#172;&#381;7 eggs. Both N2 (wild type) <br />         gpb-2(pk751) sensitive 5-HT imipramine. <br />  <br />  <br /> laid eggs post-comma stage              10 EAT-16 function: The mammalian G&#226;?&#164;5 subunit <br /> bloated retained eggs egl-30(n686) single             interacts G&#226;?&#165;-subunits (A. J. Watson et al. 1996; <br /> mutant (Table 3). Moreover, frequency body                 Zhang et al. 1996), G&#226;?&#164;5 unique ability <br /> bends locomotion triple mutant               interact GGL domain subset RGS <br /> significantly different egl-30(n686) single mutants          proteins (Snow et al. 1998, 1999; Levay et al. 1999). <br /> (Table 3), did observe egl-30 gpb-2 goa-1        The complete genome C. elegans contains GGL- <br /> triple mutants brief periods rapid forward            containing RGS proteins, EGL-10 EAT-16, two <br /> backward movement. Because egl-8, encodes                 canonical G-protein &#226;?&#165;-subunits, GPC-1 GPC-2. We <br /> phospholipase &#226;?&#164; (PLC&#226;?&#164;), putative downstream ef-              examined possibility GPB-2, closest C. eleg- <br /> fector EGL-30, generated egl-8(n488); gpb-2         ans homologue G&#226;?&#164;5, physically interact with <br /> (pk751)goa-1(pk62) triple mutant. Unlike egl-30, egl-8 did        GGL domain EGL-10 EAT-16 G&#226;?&#165;- <br /> not suppress synthetic lethality gpb-2(pk751)              subunits, GPC-1 GPC-2. Yeast two-hybrid analysis <br /> goa-1(pk62) double mutants, suggesting EGL-30 ac-            showed GPB-2 bound GGL domain EGL- <br /> tivity causes lethality gpb-2(pk751)goa-2(pk62) dou-       10 EAT-16 G-protein &#226;?&#165;-subunits, GPC-1 and <br /> ble mutants unknown downstream effector.            GPC-2 (Figure 4). We similar results GPB-1 <br /> The results epistasis analysis consistent        (Figure 4). These findings suggest speci- <br /> gpb-2 goa-1 acting parallel pathways upstream          ficity GPB-2 GPB-1 ability bind to <br /> of, parallel to, egl-30.                                       EGL-10, EAT-16, GPC-1, GPC-2 vitro. However, <br />    If egl-30 acts downstream gpb-2, expect            results reflect vivo situation. <br /> egl-30 gpb-2 double mutants exhibit phenotypes             Mutations eat-16 (RGS) result similar behavioral <br /> egl-30 single mutants. Therefore, constructed double           phenotypes described goa-1 (Go&#226;?&#163;)&#226;&#8364;&#8221;starvation, <br /> mutants gpb-2(pk751) hypomorphs                    premature egg laying, hyperactive locomotion <br /> egl-30. We animals having mutations            (Hajdu-Cronin et al. 1999)&#226;&#8364;&#8221;but mutations egl-10 <br /> egl-30(ad805), strongest reduction-of-function allele         (RGS) result opposite phenotypes (Koelle Hor- <br /> of egl-30, gpb-2(pk751) highly similar egl-30         vitz 1996). Genetic analysis showed EGL-10 an <br /> (ad805) single mutants. egl-30(ad805)gpb-2(pk751) dou-            upstream inhibitor GOA-1 (Go&#226;?&#163;) (Koelle Hor- <br /> ble mutant animals severely lethargic laid               vitz 1996), EAT-16 acts downstream of, par- <br /> 100% eggs post-comma stage (Table 3).             allel to, GOA-1 (Go&#226;?&#163;), turn inhibits EGL-30 <br /> However, egg laying infrequent, suggesting          (Gq&#226;?&#163;) activity (Hajdu-Cronin et al. 1999). Given the <br /> gpb-2 enhance egl-30 phenotype (Table 3). We              physical interaction GPB-2 EGL-10 EAT-16 <br /> also double mutant loss-of-function              vitro behavioral phenotypes gpb-2 mutant <br /> gpb-2 mutation reduction-of-function egl-30 muta-           animals, GPB-2 required function of <br /> tion, n686, gpb-2(pk751) did             EGL-10 EAT-16 both. To test hypothesis, <br /> suppress behavioral defects egl-30(n686) single            analyzed phenotype double mutants between <br /> mutants; anything, slight increase          gpb-2 eat-16 egl-10. We phenotype <br /> egl-30 egg-laying defect (Table 3). Again results           double mutants gpb-2(pk751)eat-16(sy438) gpb- <br /> suggest gpb-2 acts upstream of, parallel to, egl-30       2(pk751); egl-10(md176) indistinguishable the <br /> to control egg-laying activity, locomotion rate,          phenotype gpb-2 mutant animals respect the <br /> the pumping rate. Taken together, genetic data                rate egg laying, pharyngeal pumping, fre- <br /> indicate gpb-2 regulate goa-1 (Go&#226;?&#163;)             quency body bends locomotion (Table 4). <br /> egl-30 (Gq&#226;?&#163;) signaling.                                           Also phenotype gpb-2(pk751)eat-16(sy438); egl-10 <br />    GPB-2 interacts with, necessary for, EGL-          (md176) triple mutant animals different from <br />                                                                                    TABLE 3 <br />                                                             Genetic interactions gpb-2, egl-30, goa-1 <br />  <br />                                                     Developmental stage newly laid eggs <br />                                                           (% total no. eggs laid) <br />                                          One- eight-       Nine-cell        Post-          No.          Locomation rate             Unlaid eggs         Pumping rate <br /> Genotype                                     cell             comma           comma          eggs laid      (body bends min)           adult        (pumps min) <br /> N2 (wild type)                                  5                 95              0            197                  18.4 &#226;&#171;&#190; 2.0            17.6   &#226;&#171;&#190;   0.6         258   &#226;&#171;&#190;   12 <br /> gpb-2(pk751)                                    0                 80             20            156                  17.5 &#226;&#171;&#190; 2.1            30.3   &#226;&#171;&#190;   1.6         140   &#226;&#171;&#190;   8 <br /> egl-30(n686)                                    0                 25             75            131                  10.8 &#226;&#171;&#190; 1.5            39.3   &#226;&#171;&#190;   1.3         279   &#226;&#171;&#190;   12 <br /> egl-30(n686)gpb-2(pk751)                        0                 13             87            128                  10.0 &#226;&#171;&#190; 1.2            37.1   &#226;&#171;&#190;   2.1         256   &#226;&#171;&#190;   15 <br /> egl-30(ad805)                                   0                  0            100             67                     ND                 46.1   &#226;&#171;&#190;   1.9         210   &#226;&#171;&#190;   14 <br /> egl-30(ad805)gpb-2(pk751)                       0                  0            100             14                     ND                 39.1   &#226;&#171;&#190;   1.7         190   &#226;&#171;&#190;   9 <br /> egl-30(n686)gpb-2(pk751)goa-1(pk62)             0                 46             54            170                  11.4 &#226;&#171;&#190; 1.0            40.5   &#226;&#171;&#190;   0.8         267   &#226;&#171;&#190;   19 <br />    Twenty adult animals assayed genotype determine developmental stage newly laid eggs. To determine frequency body bends, unlaid <br /> eggs adult, pharyngeal pumping rate, 10 adult animals assayed genotype (see materials methods). Values reported means &#226;&#171;&#190; <br /> standard errors. The goa-1(pk62) allele putative loss-of-function mutation (Mendel et al. 1995). egl-30(ad805) stronger reduction-of-function mutation compared to <br /> egl-30(n686) (Trent et al. 1983; Brundage et al. 1996). Both egl-30(ad805) mutants egl-30(ad805)gpb-2(pk751) double mutants tracks paralyzed <br />                                                                                                                                                                                   GPB-2 Interacts With EAT-16 EGL-10 <br />  <br />  <br />  <br />  <br /> when undisturbed (ND, determined). ANOVA comparing groups different variables conducted. We significant effect group locomotion (F &#226;&#171;&#189; <br /> 6.12, P &#226;&#172;? 0.001), unlaid eggs adult (F &#226;&#171;&#189; 37.0, P &#226;&#172;? 0.001), rate pharyngeal pumping (F &#226;&#171;&#189; 16.94, P &#226;&#172;? 0.001). We used Student&#226;&#8364;&#8482;s t-test analyze the <br /> differences genotypes. No significant differences obtained frequency body bends locomotion egl-30(n686) egl-30(n686)gpb-2(pk751) <br /> (P &#226;&#171;&#189; 0.35) egl-30(n686)gpb-2(pk751)goa-1(pk62) (P &#226;&#171;&#189; 0.34). The following differences obtained unlaid eggs adult: egl-30(ad805) egl-30(ad805)gpb-2 <br /> (pk751) (P &#226;&#171;&#189; 0.018). No significant differences obtained unlaid eggs adult egl-30(n686) egl-30(n686)gpb-2(pk751) (P &#226;&#171;&#189; 0.18), egl-30(n686)gpb-2 <br /> (pk751)goa-1(pk62) (P &#226;&#171;&#189; 0.27). No significant differences pharyngeal pumping rate egl-30(n686) egl-30(n686)gpb-2(pk751) (P &#226;&#171;&#189; 0.11) egl- <br /> 30(n686)gpb-2(pk751)goa-1(pk62) (P &#226;&#171;&#189; 0.33), egl-30(ad805) egl-30(ad805)gpb-2(pk751) (P &#226;&#171;&#189; 0.056). <br />                                                                                                                                                                                   229 <br />  230                                                 A. M. van der Linden et al. <br />  <br />                                                                     release C. elegans neuromuscular junction (Lack- <br />                                                                     ner et al. 1999; Miller et al. 1999; Nurrish et al. 1999). <br />                                                                     These studies facilitated assaying re- <br />                                                                     sponse animals drug aldicarb, used <br />                                                                     measure acetylcholine levels synapses. Aldicarb <br />                                                                     treatment wild-type animals causes hypercontraction <br />                                                                     body wall muscles paralysis, inhibition <br />                                                                     growth reproduction. Mutations decrease <br />                                                                     acetylcholine release reduced sensitivity resis- <br />                                                                     tance aldicarb, mutations cause elevated <br />                                                                     levels acetylcholine synapses hypersensitive to <br />                                                                     aldicarb. For instance, goa-1 (Go&#226;?&#163;) null mutant animals <br />                                                                     exhibit hyperactive locomotion hypersensitivity to <br />                                                                     aldicarb, egl-30 (Gq&#226;?&#163;) reduction-of-function mu- <br />                                                                     tant animals lethargic resistant aldicarb <br />    Figure 4.&#226;&#8364;&#8221;G-protein &#226;?&#164;-subunits interact directly            (Miller et al. 1999). <br /> G-protein &#226;?&#165;-subunits. The cDNA sequences encoding GPB-1                Since gpb-2 interacts goa-1 (Go&#226;?&#163;) egl-30 <br /> and GPB-2 GPC-1 GPC-2, GGL domains <br /> of EGL-10 EAT-16, cloned two-hybrid prey <br />                                                                     (Gq&#226;?&#163;) signaling pathways, measured sensitivity <br /> and bait vector, respectively. Specific two-hybrid interactions     gpb-2 animals acetylcholinesterase inhibitor <br /> were observed &#226;?&#164;-subunits &#226;?&#165;-subunits &#226;?&#165;-        aldicarb. We observed gpb-2 mutant animals exhibit <br /> like domains, interaction detected control      altered response aldicarb-induced paralysis com- <br /> bait (P53) prey (SV40 large T) proteins. &#226;&#171;&#185;, growth            pared wild-type animals; gpb-2(pk751) animals are <br /> medium lacking histidine adenine; &#226;&#171;&#186;, growth detect- <br /> able 10 days medium lacking histidine adenine.        sensitive aldicarb (Figure 5A). Similar aldicarb <br />                                                                     sensitivities gpb-2(pk751) mutant animals ob- <br />                                                                     tained measuring population growth rates (Fig- <br /> the phenotype gpb-2 mutants (data shown). These              ure 5B). These results suggest presynaptic locus of <br /> results suggest gpb-2 acts downstream of, parallel          action gpb-2. Although gpb-2 mutant animals did not <br /> to, egl-10 eat-16 consistent GPB-2            exhibit hyperactive locomotion, absence aldi- <br /> being needed EGL-10 EAT-16 function.                   carb did tend backward exaggerated <br />    If case GPB-2 required EGL-           body flexion. To investigate GPB-2 is <br /> 10 EAT-16 function, animals defective              required EGL-10 EAT-16 function, ana- <br /> egl-10 eat-16 result phenotype resem-               lyzed response gpb-2 eat-16 gpb-2; egl-10 double <br /> bling gpb-2 mutant phenotype. Indeed,                  mutant animals aldicarb. A putative loss-of-function <br /> that phenotype double mutant animals con-               mutation eat-16 [eat-16(sy438)] loss-of-function <br /> tain putative loss-of-function mutation eat-16 [eat-           mutation egl-10 [egl-10(md176)] showed oppo- <br /> 16(sy438)] loss-of-function mutation egl-10 [egl-          site effects aldicarb&#226;&#8364;&#8221;hypersensitivity resistance, <br /> 10(md176)] resembled gpb-2 mutant phenotypes                    respectively (Figure 5B). When crossed gpb-2 an <br /> with respect rate egg laying, pharyngeal pump-            eat-16 egl-10 background, aldicarb <br /> ing, frequency body bends locomo-                 sensitivity gpb-2(pk751)eat-16(sy438) gpb-2(pk751); <br /> tion, eat-16(sy438) egl-10(md176) phe-            egl-10(md176) double mutants indistinguishable <br /> notype. As shown Table 4, eat-16(sy438); egl-10(md176)           gpb-2(pk751) single mutant (data shown), <br /> double mutants wild type respect fre-              confirming GPB-2 needed EGL- <br /> quency body bends locomotion; however,                10 EAT-16 function. <br /> egg-laying rate intermediate gpb-2(pk751)              Despite fact eat-16; egl-10 double mutant ani- <br /> and egl-10(md176) single mutants. Moreover, well-fed                mals resemble phenotype gpb-2 mutant animals <br /> adult eat-16(sy438); egl-10(md176) double mutant animals            respect rate egg laying, pharyngeal pump- <br /> had similar starved phenotype gpb-2(pk751) mutants;            ing, frequency body bends locomo- <br /> pharyngeal pumping eat-16(sy438); egl-10(md176) dou-             tion, eat-16; egl-10 double mutant animals <br /> ble mutant animals reduced (Table 4). Taken to-                 aldicarb sensitivities significantly dif- <br /> gether, genetic physical interaction GPB-2               ferent wild type, gpb-2 mutant <br /> with EGL-10 EAT-16 consistent GPB-2 func-               animals sensitive (Figure 5B). Interestingly, eat-16; <br /> tion required EGL-10 EAT-16 func-                egl-10 double mutants did exhibit exaggerated <br /> tion vice versa.                                                backward flexion seen gpb-2 mutant animals, sug- <br />    gpb-2 mutant animals sensitive acetylcholin-          gesting hyperflexion aldicarb sensitivity <br /> esterase inhibitor aldicarb: Recent studies showed             phenotypes related. While completely clear <br /> the Go&#226;?&#163;&#226;&#8364;&#8220;Gq&#226;?&#163; signaling network regulates acetylcholine               eat-16 egl-10 complete loss-of-function <br />                                                                                   TABLE 4 <br />                                                            Genetic interaction gbp-2, eat-16, egl-10 <br />  <br />                                              Developmental stage newly laid eggs <br />                                                    (% total no. eggs laid) <br />                                   One-to eight-        Nine-cell         Post-          No.            Locomotion rate            Unlaid eggs           Pumping rate <br /> Genotype                              cell             comma            comma          eggs laid        (body bends min)          adult          (pumps min) <br /> N2 (wild type)                          6                 94               0             189                  21.5   &#226;&#171;&#190;   1.2          19.0   &#226;&#171;&#190;   1.2         230   &#226;&#171;&#190;   10 <br /> gpb-2(pk751)                            0                 74              26             114                  18.8   &#226;&#171;&#190;   1.5          32.1   &#226;&#171;&#190;   1.6         129   &#226;&#171;&#190;   8 <br /> egl-10(md176)                           0                  0             100             134                   2.9   &#226;&#171;&#190;   0.3          43.1   &#226;&#171;&#190;   1.1         238   &#226;&#171;&#190;   8 <br /> gpb-2(pk751);egl-10(md176)              0                 78              22             120                  18.5   &#226;&#171;&#190;   1.7          31.3   &#226;&#171;&#190;   1.3         120   &#226;&#171;&#190;   7 <br /> eat-16(sy438)                          95                  5               0             176                  39.2   &#226;&#171;&#190;   1.1           3.8   &#226;&#171;&#190;   0.3         146   &#226;&#171;&#190;   6 <br /> gpb-2(pk751)eat-16(sy438)               0                 68              32             109                  17.5   &#226;&#171;&#190;   1.6          31.0   &#226;&#171;&#190;   1.5         135   &#226;&#171;&#190;   6 <br /> eat-16(sy438);egl-10(md176)             0                 38              62             170                  20.5   &#226;&#171;&#190;   1.0          30.0   &#226;&#171;&#190;   1.1         141   &#226;&#171;&#190;   7 <br />   At 20 adult animals assayed genotype (see materials methods). Values reported means &#226;&#171;&#190; standard errors. The eat-16(sy438) egl- <br /> 10(md176) alleles putative loss-of-function mutations (Koelle Horvitz 1996; HaiduCronin et al. 1999). ANOVA comparing groups different variables was <br /> conducted. A significant effect group noted locomotion (F &#226;&#171;&#189; 75.9, P &#226;&#172;? 0.001), unlaid eggs adult (F &#226;&#171;&#189; 142.5, P &#226;&#172;? 0.001), pharyngeal pumping <br /> (F &#226;&#171;&#189; 44.5, P &#226;&#172;? 0.001). Differences genotypes frequency body bends locomotion, unlaid eggs adult, pharyngeal pumping analyzed <br />                                                                                                                                                                               GPB-2 Interacts With EAT-16 EGL-10 <br />  <br />  <br />  <br />  <br /> using Student&#226;&#8364;&#8482;s t-test. Differences gpb-2(pk751) frequency body bends locomotion include: eat-16(sy438) (P &#226;&#172;? 0.001) egl-10(md176) (P &#226;&#172;? 0.001). <br /> No significant difference obtained frequency body bends locomotion gpb-2(pk751) N2 (wild type) (P &#226;&#172;? 0.064), gpb-2(pk751);egl-10(md176) <br /> (P &#226;&#171;&#189; 0.44), gpb-2(pk751)eat-16(sy438) (P &#226;&#171;&#189; 0.23), eat-16(sy438);egl-10(md176) (P &#226;&#171;&#189; 0.17). Differences gpb-2(pk751) unlaid eggs adult include: N2 (wild <br /> type) (P &#226;&#172;? 0.001), eat-16(sy438) (P &#226;&#172;? 0.001), egl-10(md176) (P &#226;&#172;? 0.001). No significant difference obtained unlaid eggs adult gpb-2 <br /> (pk751) gpb-2(pk751);egl-10(md176) (P &#226;&#171;&#189; 0.31), gpb-2(pk751)eat-16(sy438) (P &#226;&#171;&#189; 0.30) eat-16(sy438);egl-10(md176) (P &#226;&#171;&#189; 0.11). The following differences obtained <br /> for pharyngeal pumping: gpb-2(pk751) N2 (wild type) (P &#226;&#172;? 0.001) egl-10(md176) (P &#226;&#172;? 0.001). No significant differences pharyngeal pumping <br /> between gpb-2(pk751) gpb-2(pk751);egl-10(md176) (P &#226;&#171;&#189; 0.18), gpb-2(pk751)eat-16(sy438) (P &#226;&#171;&#189; 0.26), eat-16(sy438);egl-10(md176) (P &#226;&#171;&#189; 0.15). <br />                                                                                                                                                                               231 <br />  232                                                   A. M. van der Linden et al. <br />  <br />                                                                       mutations, data suggest GPB-2 functions <br />                                                                       independent EAT-16 EGL-10 function. <br />  <br />  <br />                                                                                               DISCUSSION <br />                                                                          Two G-protein &#226;?&#164;-subunit genes, gpb-1 gpb-2, are <br />                                                                       present complete C. elegans genome. We show <br />                                                                       that, like GPB-1, GPB-2 widely expressed, even <br />                                                                       different specific functions. <br />                                                                       GPB-2 related mammalian G&#226;?&#164;5, spe- <br />                                                                       cifically interacts GGL domain subset of <br />                                                                       RGS proteins (Snow et al. 1998; Levay et al. 1999). Our <br />                                                                       data illustrate genetic physical interaction GPB- <br />                                                                       2 C. elegans GGL-containing RGS proteins, EGL- <br />                                                                       10 EAT-16, functionally regulate Go&#226;?&#163;&#226;&#8364;&#8220;Gq&#226;?&#163; <br />                                                                       signaling network. <br />                                                                          What vivo function G-protein &#226;?&#164;-subunits? <br />                                                                       The G-protein &#226;?&#164;-subunit C. elegans, GPB-1, is <br />                                                                       highly similar mammalian G&#226;?&#164;1-4 subunits (van <br />                                                                       der Voorn et al. 1990; Zwaal et al. 1996), whereas <br />                                                                       GPB-2 related mammalian G&#226;?&#164;5 subunit. <br />                                                                       Both GPB-2 G&#226;?&#164;5 differ amino acid sequence from <br />                                                                       mammalian G&#226;?&#164;-subunits conserved <br />                                                                       regions, particularly amino-terminal re- <br />                                                                       gions. The G&#226;?&#164;1-4 subunits contain glutamine residues <br />                                                                       conserved amino-terminal region absent <br />    Figure 5.&#226;&#8364;&#8221;Loss-of-function gpb-2 mutant animals cause al- <br />                                                                       GPB-2 G&#226;?&#164;5 (Watson et al. 1994). The amino- <br /> dicarb sensitivity. (A) Aldicarb sensitivity gpb-2(pk751),         terminal region G&#226;?&#164;-subunit involved the <br /> goa-1(n363), egl-30(ad805) single mutants quantified        coiled-coil interaction amino-terminal region <br /> assaying time course onset paralysis following          G&#226;?&#165;-subunit (Lupas et al. 1991; Sondek et al. 1996). <br /> acute exposure aldicarb. In experiment, &#226;&#181;&#8216;25 animals              Like G&#226;?&#164;5, GPB-2 expressed neuronal tissues. In <br /> were placed aldicarb plates prodded 10 min over <br /> a 2-hr period determine able move. The <br />                                                                       addition, GPB-2 widely expressed muscle cells. Both <br /> goa-1(n363) allele deletion allele removes region         GPB-1 GPB-2 similar expression patterns and <br /> containing goa-1 gene (Se&#194;&#180;galat et al. 1995). egl-30(ad805)       redundant function certain cells. <br /> is strongest reduction-of-function allele (Brundage et al.        However, basis phenotypes gpb-1 and <br /> 1996). Reducing function gpb-2 leads significantly        gpb-2 mutants, distinct functions. gpb-1 <br /> increased rate paralysis compared N2 (wild type), <br /> whereas reduction goa-1 egl-30 function results ani- <br />                                                                       <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>null mutants</span> embryonic lethal progeny ma- <br /> mals increased decreased rate paralysis           ternally provided gpb-1 animals arrest stage <br /> compared N2 (wild type), respectively. A two-factor re-            larval development (Zwaal et al. 1996); gpb-2 null <br /> peated-measure ANOVA comparing groups time con-              mutant animals develop normally adulthood even <br /> ducted. A significant effect gpb-2(pk751) N2 (wild        GPB-2 expressed embryo the <br /> type) noted (F &#226;&#171;&#189; 19, P &#226;&#172;? 0.001), time course of <br /> 10 min 90 min (F &#226;&#171;&#189; 71.6, P &#226;&#172;? 0.001) significant <br />                                                                       adult. These results suggest GPB-2 compen- <br /> interaction. (B) Aldicarb sensitivity quantified measuring      sate GPB-1, does exclude possibility that <br /> the population growth rates N2 (wild-type) animals, gpb-2          GPB-1 compensate GPB-2. Mosaic gpb-1 animals <br /> (pk751), eat-16(sy438), egl-10(md176) single mutants, eat-     exhibit hyperactive locomotion premature egg lay- <br /> 16(sy438); egl-10(md176) double mutants various concentra-         ing (Zwaal et al. 1996), loss gpb-2 results <br /> tions aldicarb. One percent represents number <br /> of progeny produced starting population L1 larvae <br />                                                                       animals delayed egg laying reduced <br /> over 96-hr period absence aldicarb. Animals          pharyngeal pumping. Thus, loss gpb-1 results differ- <br /> lost eat-16 egl-10 function result aldicarb sensitivi-    ent adult behavioral phenotypes compared loss of <br /> ties significantly different wild type (N2),         gpb-2, suggesting GPB-1 GPB-2 distinct and <br /> gpb-2 animals significantly sensitive aldicarb. Significant <br /> effects groups increasing aldicarb concentration <br /> were analyzed using two-factor repeated-measure ANOVA. <br /> The following significant effects obtained: N2 (wild type) <br /> from gpb-2(pk751) (F &#226;&#171;&#189; 12.6, P &#226;&#171;&#189; 0.005), aldicarb             (F &#226;&#171;&#189; 0.006, P &#226;&#171;&#189; 0.94), significant effect aldicarb <br /> concentration range 10 200 &#226;?&#174;m (F &#226;&#171;&#189; 50.8, P &#226;&#172;? 0.001)             concentration range 10 200 &#226;?&#174;m (F &#226;&#171;&#189; 66.8, P &#226;&#172;? 0.001). <br /> with significant interaction. No significant effect        Error bars represent standard error mean tripli- <br /> N2 (wild type) eat-16(sy438); egl-10(md176) observed          cate experiments. <br />                                          GPB-2 Interacts With EAT-16 EGL-10                                         233 <br />  <br /> specific functions. In agreement hypothesis, <br /> overexpression GPB-1 results animals le- <br /> thargic bloated eggs (Zwaal et al. 1996), <br /> whereas overexpression GPB-2 effect egg <br /> laying, pharyngeal pumping, frequency body <br /> bends locomotion. <br />    A possible explanation distinct phenotypic <br /> patterns overexpression GPB-1, GPB-2, <br /> may sequester specific G&#226;?&#163;-subunits. For <br /> example, reduction-of-function mutation egl-30 (Gq&#226;?&#163;) <br /> has phenotype similar GPB-1 overexpression&#226;&#8364;&#8221; <br /> lethargic delayed egg laying (Brundage et al. 1996). <br /> The GPB-1 overexpression phenotype be <br /> due sequestration EGL-30. Consistent this <br /> hypothesis, identified putative gain-of-function al- <br /> lele egl-30 suppressor lethargy egg- <br /> laying defective phenotype results overexpres- <br /> sion GPB-1 (A. M. van der Linden, H. C. Korswagen <br /> and R. H. A. Plasterk, unpublished results). Further- <br /> more, gpb-2&#226;&#8364;&#8221;or components interact GPB-2 <br /> &#226;&#8364;&#8221;may expressed high levels wild-type <br /> background, way extra GPB-2             Figure 6.&#226;&#8364;&#8221;A model GPB-2 regulate behavior (e.g., <br />                                                              locomotion rate, egg-laying activity) interacting both <br /> have effect. Finally, possible differential    EGL-10 EAT-16, turn inhibit GOA-1 (Go&#226;?&#163;) and <br /> interactions G&#226;?&#165;-subunits G&#226;?&#165;-like subunits ac-        EGL-30 (Gq&#226;?&#163;) activity, respectively. This model based on <br /> count activation different downstream signal-     data presented data Koelle Horvitz <br /> ing pathways, resulting different effects behavior.    (1996), Hajdu-Cronin et al. (1999), Nurrish et al. (1999), <br />    Interaction GPB-2 EGL-10 EAT-16: The          Lackner et al. (1999), Miller et al. (1999). GPB-2&#226;&#8364;&#8221;most <br />                                                              closely related mammalian G&#226;?&#164;5&#226;&#8364;&#8221;is required func- <br /> genome C. elegans harbors canonical G-protein         tion GGL-containing RGS proteins EGL-10 EAT- <br /> &#226;?&#165;-subunits, GPC-1 GPC-2 (Jansen et al. 1999),        16. EGL-30 activates EGL-8&#226;&#8364;&#8221;encoding phospholipase C&#226;?&#164; <br /> two GGL-containing RGS proteins, EGL-10 EAT-16           (PLC&#226;?&#164;)&#226;&#8364;&#8221;which cleaves phosphatidylinositol 4,5-biphosphate <br /> (Koelle Horvitz 1996; Hajdu-Cronin et al. 1999).         (PIP2) diacylglycerol (DAG) inositol-1,4,5-triphos- <br /> Like specific interaction G&#226;?&#164;5 &#226;?&#165;-like         phate (Berridge 1984). DGK-1&#226;&#8364;&#8221;encoding diacylglycerol <br />                                                              kinase (DAG-kinase)&#226;&#8364;&#8221;may activated GOA-1 reduce <br /> (GGL) domains RGS6, 7, 9, 11 (Snow et al. 1998,       DAG levels. The locomotion rate egg-laying activity is <br /> 1999; Levay et al. 1999), GPB-2, closest    indirectly regulated levels DAG. GOA-1 inhibit EGL- <br /> C. elegans homologue G&#226;?&#164;5, interacts EGL-10       30 (Gq&#226;?&#163;) activity directly downstream effectors such <br /> EAT-16. Using yeast two-hybrid interaction assay      EAT-16 and/or DGK-1, indirectly GGL domain of <br /> vitro, demonstrate specific interactions G&#226;?&#164;-      EAT-16 proposed Guan Hall (1999), act in <br />                                                              parallel EGL-30. <br /> subunits G&#226;?&#165;-like subunits, any <br /> differences GPB-1 GPB-2 regard. <br /> In vivo, genetic interactions GPB-2         G&#226;?&#163; GDP. Genetic biochemical data argue that <br /> EGL-10 EAT-16, consistent GPB-2 function            EAT-16 (RGS) functions GAP EGL-30 (Gq&#226;?&#163;; <br /> being required EGL-10 EAT-16 function.          Hajdu-Cronin et al. 1999), genetic data suggest <br /> In gpb-2 eat-16 gpb-2; egl-10 double mutants, func-      EGL-10 (RGS) inhibits GOA-1 (Go&#226;?&#163;) activity (Koelle <br /> tional GPB-1 present cells control        Horvitz 1996). The results GPB-2 inter- <br /> observed phenotypes, e.g., egg laying muscle activ-      acts EGL-10 EAT-16 regulate GOA-1 <br /> ity, suggesting GPB-1 functionally redundant     EGL-30 activity, respectively. This conclusion em- <br /> with GPB-2 phenotypes described. After comple-       phasized finding loss gpb-2 does result <br /> tion study aware recent results     phenotypes similar eat-16 egl-10. Only <br /> that support conclusions: first, interactions be-      eat-16; egl-10 double mutant animals resemble gpb-2 <br /> tween GPB-2 EGL-10 EAT-16 inde-            mutant phenotype, e.g., rate egg laying, pharyn- <br /> pendently observed (Chase et al. 2001), second,          geal pumping, frequency body bends during <br /> consistent observed feeding defects, eat-11      locomotion. In line data studies G&#226;?&#164;5, a <br /> allelic gpb-2 appears interact eat-16     possible role GPB-2 modulate the <br /> egl-10 (Robatzek et al. 2001).                               specificity and/or activity EGL-10 EAT-16 for <br />    Role GPB-2 GOA-1 (Go&#226;?&#163;)&#226;&#8364;&#8220;EGL-30 (Gq&#226;?&#163;)             GOA-1 EGL-30, respectively (Witherow et al. <br /> signaling network: In mammals, clear G&#226;?&#164;5-RGS      2000). It is, however, likely GPB-2 function is <br /> complexes act GAPs G&#226;?&#163;-subunits. GAPs inhibit      completely dependent EGL-10 EAT-16. For <br /> G&#226;?&#163; activity catalyzing exchange GTP bound          example, animals defective gpb-2 exhibit exaggerated <br />  234                                            A. M. van der Linden et al. <br />  <br /> backward motion aldicarb sensitivity phenotypes,           30 GPB-2 inhibits GOA-1 activates EGL-30. How- <br /> whereas eat-16; egl-10 double mutant animals not.           ever, GPB-2 interacts both <br />    Previous genetic data indicated GOA-1 (Go&#226;?&#163;)            EGL-10 EAT-16, favor hypothesis GPB-2 <br /> antagonize EGL-30 (Gq&#226;?&#163;) signaling directly indirectly       inhibits GOA-1 EGL-30. It plausible the <br /> through EAT-16 and/or DGK-1 (DAG-kinase; Hajdu-                apparent activation events downstream EGL-30 by <br /> Cronin et al. 1999; Miller et al. 1999). In model         GPB-2 suggested slight enhancement the <br /> both EAT-16 DGK-1 function downstream of,               egl-30 phenotype egl-30 gpb-2 double mutants can <br /> parallel to, GOA-1 (Go&#226;?&#163;) act synergistically nega-      easily caused inhibition GOA-1, GOA- <br /> tively regulate EGL-30 (Gq&#226;?&#163;) signaling (Figure 6). This        1 antagonizes EGL-30 signaling. With regard sec- <br /> synergy proposed animals defective         ond question, data explicitly distinguish <br /> EAT-16 DGK-1 function synthetic lethal,            models described. In case, we <br /> this lethality suppressed reducing EGL-30          propose GPB-2 plays important role regu- <br /> activity (Hajdu-Cronin et al. 1999). Thus, downstream          lation GOA-1 (Go&#226;?&#163;) EGL-30 (Gq&#226;?&#163;) signaling <br /> signaling components EGL-30 cause lethality         EGL-10 EAT-16. <br /> of eat-16; dgk-1 double mutant. In support            We thank David Weinkove, Hendrik C. Korswagen, Stephen <br /> model, similar genetic interactions              Wicks critically reading manuscript, Dan Chase, Michael <br /> GPB-2, GOA-1, DGK-1. gpb-2; dgk-1 gpb-2 goa-1          Koelle, Merrilee Robatzek, James Thomas communicating <br /> double mutants resulted synthetic lethal phenotype.       results prior publication, Paul Sternberg Yvonne Hajdu-Cronin <br /> Moreover, synthetic lethal phenotype gpb-2 goa-1        kindly providing eat-16(sy438) strain, Caenorhabditis <br />                                                                Genetics Stock Center providing strains used this <br /> double mutants suppressed EGL-30 activity <br />                                                                study. This work supported The Netherlands Organization for <br /> was reduced. No suppression observed              Scientific Research (NWO) grant 014-80-008. <br /> activity EGL-8&#226;&#8364;&#8221;a potential downstream effector of <br /> EGL-30&#226;&#8364;&#8221;was reduced. This result agreement with <br /> the model EGL-8 does mediate effects <br /> of EGL-30 (Miller et al. 1999),                               LITERATURE CITED <br /> be other, far unknown, downstream effectors. An             Avery, L., 1993 The genetics feeding Caenorhabditis elegans. <br /> interesting finding that, unlike gpb-2 goa-1 double             Genetics 133: 897&#226;&#8364;&#8220;917. <br />                                                                Berridge, M. J., 1984 Inositol trisphosphate diacylglycerol as <br /> mutants, goa-1 eat-16 double mutants lethal                second messengers. Biochem. J. 220: 345&#226;&#8364;&#8220;360. <br /> (Hajdu-Cronin et al. 1999), suggesting gpb-2 does         Brundage, L., L. Avery, A. Katz, U. J. Kim, J. E. Mendel et al., 1996 <br /> not interact eat-16 un-            Mutations C. elegans Gq&#226;?&#163; gene disrupt movement, egg laying, <br />                                                                    viability. Neuron 16: 999&#226;&#8364;&#8220;1009. <br /> known pathway. A potential pathway dgk-1 path-          Chalfie, M., Y. Tu, G. Euskirchen, W. W. Ward D. C. Prasher, <br /> way, dgk-1; gpb-2 double mutants result            1994 Green fluorescent protein marker gene expres- <br /> synthetic lethal phenotype.                                        sion. Science 263: 802&#226;&#8364;&#8220;805. <br />                                                                Chase, D. L., G. A. Patikoglou M. R. Koelle, 2001 Two RGS <br />    Recent literature designed number interac-             proteins inhibit G&#226;?&#163;o G&#226;?&#163;q signaling C. elegans neurons <br /> tions GOA-1 (Go&#226;?&#163;)&#226;&#8364;&#8220;EGL-30 (Gq&#226;?&#163;) network                 require G&#226;?&#164;5-like subunit function. Curr. Biol. 11: 222&#226;&#8364;&#8220;231. <br /> functions locomotion egg laying C. elegans           Clapham, D. E., E. J. Neer, 1993 New roles G-protein &#226;?&#164;&#226;?&#165;- <br />                                                                    dimers transmembrane signalling. Nature 365: 403&#226;&#8364;&#8220;406. <br /> (Hajdu-Cronin et al. 1999; Lackner et al. 1999; Miller         Eeckman, F. H., R. Durbin, 1995 ACeDB macace. Methods <br /> et al. 1999; Nurrish et al. 1999). In signaling net-          Cell Biol. 48: 583&#226;&#8364;&#8220;605. <br /> work, GOA-1 (Go&#226;?&#163;) EGL-30 (Gq&#226;?&#163;) assumed              Guan, K. L., M. Han, 1999 A G-protein signaling network medi- <br />                                                                    ated RGS protein. Genes Dev. 13: 1763&#226;&#8364;&#8220;1767. <br /> couple G-protein-coupled receptors shown            Hajdu-Cronin, Y. M., W. J. Chen, G. Patikoglou, M. R. Koelle and <br /> to negatively regulated GGL-containing RGS               P. W. Sternberg, 1999 Antagonism Go&#226;?&#163; Gq&#226;?&#163; in <br /> proteins EGL-10 EAT-16, respectively (Figure 6).               Caenorhabditis elegans: RGS protein EAT-16 necessary Go&#226;?&#163; <br />                                                                    signaling regulates Gq&#226;?&#163; activity. Genes Dev. 13: 1780&#226;&#8364;&#8220;1793. <br /> The rate locomotion activity egg laying          Han, M., P. W. Sternberg, 1991 Analysis dominant-negative <br /> are stimulated activation EGL-30 EGL-8 (phos-            mutations Caenorhabditis elegans let-60 ras gene. Genes Dev. <br /> pholipase C&#226;?&#164;), activation GOA-1 inhibits                5: 2188&#226;&#8364;&#8220;2198. <br />                                                                Jansen, G., E. Hazendonk, K. L. Thijssen R. H. Plasterk, <br /> both behaviors, probably DGK-1 (DAG-kinase). The               1997 Reverse genetics chemical mutagenesis Caenorhabditis <br /> data presented support model GPB-2                 elegans. Nat. Genet. 17: 119&#226;&#8364;&#8220;121. <br /> plays important role GOA-1&#226;&#8364;&#8220;EGL-30 signaling          Jansen, G., K. L. Thijssen, P. Werner, M. van der Horst, E. Hazen- <br />                                                                    donk et al., 1999 The complete family genes encoding G <br /> network interacting GGL-containing RGS                proteins Caenorhabditis elegans. Nat. Genet. 21: 414&#226;&#8364;&#8220;419. <br /> proteins EAT-16 EGL-10 (Figure 6). In view         Kaziro, Y., H. Itoh, T. Kozasa, M. Nakafuku T. Satoh, 1991 <br /> model, questions asked. First, does GPB-2               Structure function signal-transducing GTP-binding pro- <br /> inhibit EGL-30 GOA-1 and, second,                  teins. Annu. Rev. Biochem. 60: 349&#226;&#8364;&#8220;400. <br />                                                                Koelle, M. R., 1997 A new family G-protein regulators&#226;&#8364;&#8221;the RGS <br /> data distinguish GOA-1 acting parallel               proteins. Curr. Opin. Cell Biol. 9: 143&#226;&#8364;&#8220;147. <br /> EGL-30 GOA-1 acting upstream EGL-30? With                Koelle, M. R., H. R. Horvitz, 1996 EGL-10 regulates G protein <br /> respect question, epistatic analysis           signaling C. elegans nervous shares conserved <br />                                                                    domain mammal <br /> </body></html>