<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 2007, p. 6084&#226;&#8364;&#8220;6092                                                                            Vol. 27, No. 17 <br /> 0270-7306/07/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.00647-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  The 2&#226;&#172;&#732;-O-Ribose Methyltransferase Cap 1 Spliced Leader RNA <br />          U1 Small Nuclear RNA Trypanosoma brucei&#228;&#338;&#164;&#226;&#8364;&#160; <br />          Jesse R. Zamudio,1 Bidyottam Mittra,1 Silvie Foldynova&#194;&#180;-Trant&#196;&#177;&#194;&#180;rkova&#194;&#180;,1,2 Gusti M. Zeiner,1&#226;&#8364;&#161; <br />                Julius Lukes&#203;&#8225;,2 Janusz M. Bujnicki,3 Nancy R. Sturm,1* David A. Campbell1 <br />          Department Microbiology, Immunology Molecular Genetics, David Geffen School Medicine, University of <br />          California Los Angeles, Los Angeles, California 90095-14891; Biology Centre, Czech Academy Sciences, and <br />           Faculty Biology, University South Bohemia, 37005 C&#203;&#8225; eske&#194;&#180; Bude&#203;&#8225;jovice, Czech Republic2; Laboratory of <br />                    Bioinformatics Protein Engineering, International Institute Molecular Cell Biology, <br />                                   Warsaw, Institute Molecular Biology Biotechnology, <br />                                             Adam Mickiewicz University, Poznan, Poland3 <br />                                                 Received 12 April 2007/Accepted 18 June 2007 <br />  <br />              mRNA cap 1 2&#226;&#180;&#8226;-O-ribose methylation widespread modification implicated processing, traf- <br />           ficking, translational control eukaryotic systems. The eukaryotic enzyme identified. In <br />           kinetoplastid flagellates trans-splicing spliced leader (SL) polycistronic precursors conveys hyper- <br />           methylated cap 4, including cap 0 m7G seven additional methylations 4 nucleotides, all <br />           nuclear mRNAs. We report eukaryotic cap 1 2&#226;&#180;&#8226;-O-ribose methyltransferase, TbMTr1, member a <br />           conserved family viral eukaryotic enzymes. Recombinant TbMTr1 methylates ribose first <br />           nucleotide m7G-capped substrate. Knockdowns <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> TbMTr1 Trypanosoma brucei grow <br />           normally, loss 2&#226;&#180;&#8226;-O-ribose methylation cap 1 substrate SL RNA U1 small nuclear RNA. <br />           TbMTr1-null cells accumulation cap 0 substrate methylation, spliced mRNA <br />           modified efficiently position 4 absence 2&#226;&#180;&#8226;-O-ribose methylation position 1; downstream cap 4 <br />           methylations independent cap 1. Based TbMTr1-green fluorescent protein localization, 2&#226;&#180;&#8226;-O-ribose <br />           methylation position 1 occurs nucleus. Accumulation 3&#226;&#180;&#8226;-extended SL RNA substrate indicates a <br />           delay processing suggests synergistic role cap 1 maturation. <br />  <br />  <br />    The m7G cap eukaryotic mRNA plays essential roles                      Trypanosoma cruzi, transcribed polycistronically (23, <br /> translation stability, enzymatic activities responsi-               36). The discrete spliced leader (SL) RNA serves donor of <br /> ble formation characterized (48, 50). Additional               39 nucleotides (nt) unique cap structure all <br /> modifications adjacent nucleotides mRNA small                         coding regions trans-splicing. In addition m7G, hy- <br /> nuclear RNA (snRNA) documented; however,                           permethylated cap 4 structure includes 2&#226;&#172;&#732;-O-ribose methyl- <br /> identity eukaryotic cap ribose-methylating enzyme(s)                     ations 4 nt (AACU) additional <span id='am-475' about='xsp:base' typeof='owl:Thing'>base</span> meth- <br /> and function cap modification m7G attached                    ylations (m26A) fourth (m3U) positions (5, 16, <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> first-transcribed nucleotide (cap 0) remain unclear (6,                  41). The 5&#226;&#172;&#732; processing SL accompanied association of <br /> 18). 2&#226;&#172;&#732;-O-Ribose methylation common modification                        transcript Sm complex stem-loop <br /> RNA (29) occur mechanisms: box C/D                             structures intron (35, 51, 65) 3&#226;&#172;&#732; trimming residual <br /> snoRNA-guided methylation fibrillarin (12) nucleotide-                   nucleotides attenuated transcription termination the <br /> specific enzymes (31, 43, 54). The conservation modified                     downstream T tract (51, 66). Other eukaryotic mRNA caps <br /> nucleotides functionally important regions indicates major                 vary containing cap 0 modified by <br /> benefit cell (13). Individual methylations tRNA                   2&#226;&#172;&#732;-O-ribose methylation(s) the second <br /> not essential viability; however, cumulative loss                  nucleotides, known cap 1 cap 2, respectively (17). <br /> detrimental (1). Potential functions mRNA 2&#226;&#172;&#732;-O-ribose                       Among higher eukaryotes cap ribose modifications con- <br /> methylation include roles processing trafficking tran-                served, majority mRNAs ribose methylated in <br /> scripts, translational control (4, 45).                              mammals (4). <br />    Protein-encoding genes kinetoplastid protozoa, including                     The unique kinetoplastid cap 4 structure implicated in <br /> the human-pathogenic Leishmania major, Trypanosoma brucei,                      trans-splicing translation. Substrate SLs incomplete <br />                                                                                 cap 4 trans spliced (52); however, mRNAs asso- <br />                                                                                 ciate polysomes (68), suggesting role cap 4, primary <br />    * Corresponding author. Mailing address: Department Microbi-              SL sequence, translation. Two cap 2&#226;&#172;&#732;-O-ribose methyl- <br /> ology, Immunology Molecular Genetics, David Geffen School of <br /> Medicine, 609 Charles E. Young Drive East, University California             transferases (MTases), TbMT417 TbMT511, modify the <br /> at Los Angeles, Los Angeles, CA 90095-1489. Phone: (310) 206-5556.              second positions cap 4, dispensable <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> of <br /> Fax: (310) 206-5231. E-mail: nsturm@ucla.edu.                                   T. brucei (2, 3, 64). These relatives vaccinia virus cap 1 <br />    &#226;&#8364;&#160; Supplemental material article http://mcb          MTase henceforth referred TbMTr2 TMTr3, <br /> .asm.org/. <br />    &#226;&#8364;&#161; Present address: Department Microbiology Immunology, <br />                                                                                 respectively, reflect functional roles. With requirement <br /> Stanford University School Medicine, Stanford, CA 94305.                     TbMTr2 TbMTr3 activities viability, focus a <br />    &#228;&#338;&#164; <br />      Published ahead print 2 July 2007.                                   critical component cap 4 turned position 1. <br />  <br />                                                                          6084 <br />  VOL. 27, 2007                                                                                            RNA CAP 1 RIBOSE METHYLATION                             6085 <br />  <br />  <br />    The kinetoplastid U1 snRNA contains 4 nt                        nuclease P1 50 mM ammonium acetate, pH 5.3, 1 h 37&#194;&#176;C. RNA samples <br /> SL RNA identical hypermethylation                          analyzed thin-layer chromatography (TLC) polyethyleneimine cellu- <br />                                                                                      lose plates developed 0.45 M ammonium sulfate (62). TLC results were <br /> positions monogenetic kinetoplastid Crithidia fasciculata                     visualized PhosphorImager (Amersham Biosciences). <br /> (47). In contrast SL RNA, RNA polymerase (pol) II                                 RNAi strains cell culture. Procyclic T. brucei strain 29-13 (61) containing <br /> transcript, U1 snRNA, transcribed RNA pol III                          T7 RNA pol tetracycline (Tet) repressor genes grown 27&#194;&#176;C SM <br /> ultimately acquires trimethylated m2,2,7G (TMG) cap (14).                          medium supplemented 10% fetal bovine serum presence hygro- <br />                                                                                      mycin (50 &#226;?&#174;g/ml; PGC Scientific) G418 (15 &#226;?&#174;g/ml; ICN). The pRTbMT370 <br /> The U1 snRNA contains Sm binding sequence likely <br />                                                                                      p2T7-177-370 constructions transfected 29-13 cells elec- <br /> results formation core Sm-protein complex (39, 40,                         troporation (59). Drug-resistant T. brucei strains cloned limiting dilution <br /> 58). Knockdown TbMTr2 effect U1 snRNA cap                               supplemented SM medium presence G418, hygromycin, zeocin. <br /> structure T. brucei (64).                                                         RNAi induced addition 100 ng/ml Tet clonal pRTbMT370 and <br />    Here identify cap 1 2&#226;&#172;&#732;-O-ribose MTase SL RNA                         p2T7-177-370 cells 1 &#226;&#171;&#187; 106 cells/ml, growth assayed using Coulter <br />                                                                                      Counter (Beckman-Coulter). <br /> T. brucei, TbMTr1, representing functionally charac-                          RNA analyses. Total cell RNA isolated TRIzol reagent (Invitrogen) <br /> terized member group widely conserved eukaryotic                             described previously (66). High-resolution acrylamide RNA blotting, RNA <br /> MTases. S-Adenosylmethionine (AdoMet)-dependent, cap 1                               primer extension using Moloney murine leukemia virus reverse transcriptase <br /> 2&#226;&#172;&#732;-O-ribose MTase activity shown vitro. No growth defect                       (RT), DNA sequencing reactions performed described previously <br />                                                                                      (51, 52, 66) using &#226;?&#165;-32P-labeled oligonucleotides TbWTexon (5&#226;&#172;&#732;-CAATA <br /> was detected knockdown TbMTr1-null lines loss <br />                                                                                      TAGTA CAGAA ACTG) intron-specific TbSL-40 (5&#226;&#172;&#732;-CTACT GGGAG <br /> of cap 1 methylation substrate SL RNA U1                                 CTTCT CATAC). U1 snRNA extensions performed TbU1-40 (5&#226;&#172;&#732;- <br /> snRNA corresponding accumulation substrate SL                               CCCAC TCAAA GTTTA CTG). Low-resolution formaldehyde agarose RNA <br /> 3&#226;&#172;&#732;-extended forms. TbMTr1 portal identification                          blots generated described previously (52), hybridized [&#226;?&#163;-32P]CTP- <br /> other eukaryotic cap 1 MTases understanding                              incorporated random hexamer-primed probes (Amersham Biosciences), vi- <br />                                                                                      sualized PhosphorImager. Poly(A)&#226;&#171;&#185; mRNA isolated using the <br /> functional significance modifications eukaryotic                         MicroPoly(A) Purist kit (Ambion). RNA blot hybridizations performed <br /> mRNA snRNA.                                                                      TbStloop1 (5&#226;&#172;&#732;-CTACT GGGAG CTTCT CATCA), TbU1-20 (5&#226;&#172;&#732;-CAAGC <br />                                                                                      ACGGC GCTTT CGTGA), 5S rRNA (5&#226;&#172;&#732;-TAACT TCACA AATCG <br />                                                                                      GACGG GAT). <br />                         MATERIALS AND METHODS                                           RNase T2 cap analysis. mRNA cell line purified using the <br />    Sequence analyses. Sequence database searches carried PSI-          MicroPoly(A) Purist mRNA purification kit (Ambion), starting 10 &#226;?&#174;g total <br /> BLAST, starting putative ribose MTases trypanosomes, predicted         RNA. To <span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span> cap structure, 300 ng mRNA decapped tobacco acid <br /> in earlier analyses (15). Phylogenetic analyses carried MEGA 3.1       pyrophosphatase (5 U; Roche) 1 h 37&#194;&#176;C provided reaction buffer. The <br /> (26).                                                                                phosphate removed using HK phosphatase (2 U; Epicenter) 1 h 30&#194;&#176;C. The <br />    DNA cloning RNAi constructions. The 1,110-bp open reading frame               RNA extracted twice phenol, ethanol precipitated, end labeled with <br /> (ORF) TbMTr1 (GeneDB no. Tb10.6k15.2610) amplified using MT370               [&#226;?&#165;-32P]ATP T4 polynucleotide kinase. The radiolabeled RNA digested 50 <br /> Xba-F (5&#226;&#172;&#732;-GGTCT AGAAT GCCTG CCGTT GCAGA C) MT370 xh/xb R                         mM ammonium acetate (pH 4.5) 2 mM EDTA 40 U/ml RNase T2 37&#194;&#176;C <br /> (5&#226;&#172;&#732;-GGCTC GAGTC TAGAT GACTT GACCT CCTCC ATTGC) cloned                            12 h. Digestion products resolved 25% acrylamide-8 M urea gels and <br /> into pTOPO-TA (Invitrogen). The insert released XbaI subcloned          visualized PhosphorImager. <br /> into pR hairpin-loop <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) vector. The pGR vector             Subcellular localization TbMTr1. The coding region nt 1001 2110 <br /> made pLEW100 creation green fluorescent protein (GFP) fusion             amplified pC-PTP-NEO370F (5&#226;&#172;&#732;-AGGGC CCATG CCTGC CGTTG) <br /> proteins hairpin-loop, double-stranded RNAi formation available           MT370 xh/xb R (5&#226;&#172;&#732;-GGCTC GAGTC TAGAT GACTT GACCT CCTCC <br /> upon request. The pRTbMT370 vector digestion pGMT370             ATTGC), digested ApaI XhoI insertion pMOT4G vector <br /> AscI XhoI. The XhoI site filled Klenow fragment, gel purified,      situ GFP tagging (38). The 3&#226;&#172;&#732; UTR used pKO vectors amplified <br /> inserted pGMT370 digested HpaI MluI. For inducible RNAi           BamHI370-3&#226;&#172;&#732;UTR-F (5&#226;&#172;&#732;-AGGAT CCCAA GTCAT AGGTA GTCC) was <br /> vector opposing promoters, TbMTr1 amplified using MT370Fw           inserted using BamH NotI sites. Plasmid (10 &#226;?&#174;g) digested ApaI and <br /> (5&#226;&#172;&#732;-CTCGA GGCGC CACGA CCCAG CTTCC) MT370Rv (5&#226;&#172;&#732;-GGATC                             NotI, released fragment gel purified electroporated KH4A <br /> CCACG ACTTC GGTGG GAACG) cloned pGEM-T Easy (Pro-                           YTAT T. brucei. Stable transfectants selected 50 &#226;?&#174;g/ml hygromycin, <br /> mega). The insert released XhoI BamHI cloned p2T7-177          clonal cell lines limiting dilution. Clonal lines rinsed and <br /> vector (60). Inserts vectors verified sequencing. Elimination      washed phosphate-buffered saline containing 100 ng/ml 4&#226;&#172;&#732;,6&#226;&#172;&#732;-di- <br /> TbMTr1 alleles create TbMTr1-null cell lines used pKO vectors (28)            amidino-2-phenylindole (DAPI; Sigma) 5 min room temperature. Live <br /> where drug selection cassettes replaced phleomycin puromycin           cells mounted microscope slides visualized Zeiss Axiocam <br /> resistance markers. The 426-nt, 5&#226;&#172;&#732; untranslated region (UTR) directly upstream       compound fluorescence microscope fitted Zeiss 63&#226;&#171;&#187; objective a <br /> of start codon cloned using TbMT370-5&#226;&#172;&#732;UTR-F-HindIII (5&#226;&#172;&#732;-AAAGC                digital camera. Images captured Zeiss Axiovision software com- <br /> TTAAA GTCTA TACAA CTGC) TbMT370-5&#226;&#172;&#732;UTR-R-EcoRI (5&#226;&#172;&#732;-TGAAT                          piled publication Adobe Photoshop 7.0 (Adobe). <br /> TCGGC ATCAG AGAAA TA). The 598-nt 3&#226;&#172;&#732; UTR cloned using <br /> TbMT370-3&#226;&#172;&#732;UTR-F-Xba (5&#226;&#172;&#732;-ATCTA GACAA GTCAT AGGTA GTCC) and <br /> TbMT370-3&#226;&#172;&#732;UTR-R-NotI (5&#226;&#172;&#732;-TGCGG CCGCT GAATA TTTCT CCAC). Both                                                           RESULTS <br /> were inserted pKOpuro pKOphleo vectors transfected YTAT <br /> cells, selected puromycin (30 &#226;?&#174;g/ml) zeocin (20 &#226;?&#174;g/ml; Invitrogen). <br />                                                                                         TbMT370 conserved viral eukaryotic 2&#226;&#180;&#8226;-O-ribose <br />    In vitro activity assay. Recombinant TbMTr1 T. bracie N-terminal           MTases. Phylogenetic analysis RrmJ superfamily ribose <br /> six-histidine (His6) tag overexpressed Escherichia coli BL21(DE3)pLysS        2&#226;&#172;&#732;-O-ribose MTases revealed conserved group uncharacter- <br /> cells purified homogeneity Ni column phosphocellulose chroma-          ized eukaryotic MTases (15) includes putative T. brucei <br /> tography procedures. Synthetic RNA substrate identical T. brucei SL RNA, <br />                                                                                      MTase encoded 370-amino-acid ORF named <br /> except guanosine position 2 transcriptional efficiency, synthesized <br /> using PCR-amplified DNA template allows transcription T. brucei      TbMT370 (GeneDB no. Tb10.6k15.2610). TbMT370 con- <br /> SL RNA gene T7&#226;&#338;&#189; 2.5 promoter (11). The 142-nt RNA capped                served kinetoplastids homologs T. cruzi <br /> using [&#226;?&#163;-32P]GTP vaccinia virus RNA guanylyltransferase (Ambion) de-          (GeneDB no. Tc00.1047053506247.320) L. major (GeneDB <br /> scribed manufacturer presence 1 mM AdoMet. Radiolabeled             no. LmjF36.6660). In contrast TbMTr2 TbMTr3 vac- <br /> RNA purified extraction phenol-chloroform, followed spin col- <br /> umn chromatography. Cap 1 MTase activity assayed incubating radiola- <br />                                                                                      cinia virus-like MTases unique kinetoplastids, data- <br /> beled RNA recombinant TbMTr1 1 h 28&#194;&#176;C followed RNA extrac-            base searches TbMT370 homologs revealed family pro- <br /> tion phenol-chloroform, ethanol precipitation, digestion 5 &#226;?&#174;g          teins members metazoa, protists, double-stranded <br />  6086     ZAMUDIO ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Phylogeny 2&#226;&#172;&#732;-O-ribose MTase family. Shown phylogenetic tree sequences calculated neighbor joining method and <br /> the JTT matrix (24). Values nodes indicate percent bootstrap support. The main taxa indicated. <br />  <br />  <br />  <br />  <br /> DNA viruses share 10 conserved motifs spread               unable cleave 5&#226;&#172;&#732;-5&#226;&#172;&#732; triphosphate bond the <br /> throughout ORF (see Fig. S1 supplemental material),           labeled m7G. TLC resolved radiolabeled dinucleotides, <br /> including functionally characterized cap 1 MTase, orf69         methylated species migrate farther the <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> alfalfa looper moth Autographa californica nucleopoly-          m7GpppA unmodified marker. In presence AdoMet, <br /> hedrovirus (62). All members contained characteristic K-D-           m7GpppAm species generated migrated faster <br /> K-E tetrad common 2&#226;&#172;&#732;-O-ribose MTases (9, 15). Phylogenetic            m7GpppA marker ribose methylation the <br /> analysis family showed branches containing viral          nucleotide (Fig. 2B). To confirm location the <br /> cellular proteins (Fig. 1). Interestingly, orf5 European pine       methylation, modified substrate subjected RNase T2 <br /> sawfly Neodiprion sertifer nucleopolyhedrovirus branched cel-       digestion, inhibited ribose 2&#226;&#172;&#732;-O-ribose methyl- <br /> lular proteins insects, suggesting horizontal transfer       ation. RNase T2 treatment yielded product consistent with <br /> its host. The family includes human ortholog TbMT370.          vitro cap 1 formation (data shown). The in-depth <br /> The gene encoding human cap 1 2&#226;&#172;&#732;-O-ribose MTase           biochemical characterization protein pre- <br /> be identified functionally, activity partially purified   sented elsewhere. <br /> and characterized (30).                                                     The cap 1 2&#226;&#172;&#732;-O-ribose MTase activity TbMT370 first <br />    To determine eukaryotic family member pos-                  kind ascribed eukaryotic cellular protein. The en- <br /> sesses cap 1 MTase activity, assayed TbMT370 activity             zyme renamed TbMTr1 reflect function. To query <br /> both vitro vivo, specifically candidate SL cap        TbMTr1 role SL cap 4, determined vivo <br /> 1 2&#226;&#172;&#732;-O-ribose methylation.                                               substrates. <br />    TbMT370 AdoMet-dependent, 2&#226;&#180;&#8226;-O-ribose MTase cap 1                    5&#226;&#180;&#8226; 3&#226;&#180;&#8226; SL RNA phenotypes TbMTr1 RNAi knockdown <br /> activity vitro. Two enzymes performing cap 4 2&#226;&#172;&#732;-O-             cells. Ribose modifications positions 2, 3, 4 not <br /> ribose methylations identified (Fig. 2A). A biochemical        essential, leaving position 1 ribose base methylations as <br /> approach initiated vitro evaluation TbMT370        potentially pivotal players cap 4. To challenge role of <br /> 2&#226;&#172;&#732;-O-ribose MTase activity. Recombinant TbMT370 ex-                  TbMTr1 cap 1 methylation SL, performed RNAi- <br /> pressed His6 tag bacterial purification.                      mediated knockdown transcript using Tet-inducible <br />    Recombinant TbMT370 purified homogeneity                       hairpin-loop opposing-T7-promoter RNAi constructions. <br /> from E. coli. In vitro-transcribed 142-nt RNA capped                 If induced cells viable, RNAi lines allow cap 1 <br /> with vaccinia virus m7G capping enzyme using radiola-                status target transcripts assessed various times <br /> beled [&#226;?&#163;-32P]GTP. This cap 0 transcript incubated               induction cell death. <br /> purified recombinant TbMT370 presence absence                    RNAi cell lines vector showed signifi- <br /> of methyl donor AdoMet. To liberate 5&#226;&#172;&#732;-5&#226;&#172;&#732;-linked                     growth defects 9 days postinduction (data shown). <br /> dinucleotides analysis, RNA extracted di-                Total RNA samples collected day 9 probed the <br /> gested nuclease P1, nonspecific RNA endonuclease                  coding region TbMTr1. The induced samples showed effi- <br />  VOL. 27, 2007                                                                             RNA CAP 1 RIBOSE METHYLATION                  6087 <br />  <br />  <br />  <br />  <br />    FIG. 2. TbMTr1 vitro 2&#226;&#172;&#732;-O-ribose MTase cap 1 activity. (A) The cap 4 structure identified enzymes. (B) &#226;?&#163;-32P-labeled m7G-capped <br /> transcript incubated recombinant TbMT370 presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) AdoMet. Following digestion nuclease P1, <br /> resistant cap dinucleotides resolved TLC. Dinucleotide identities listed right. ori, origin spotting. <br />  <br />  <br />  <br /> cient knockdown target transcript (Fig. 3A); thus,            knockdowns. To capture cap intermediates cap 4 for- <br /> lack TbMTr1 judged acceptable cells. We              mation assay, radiolabeled primer extended with <br /> then used substrate-specific primer extension examine             RT, extension inhibited presence methyl- <br /> cap structure prespliced, substrate SL RNA TbMTr1                  ation base ribose moiety (Fig. 3B); stop <br />  <br />  <br />  <br />  <br />   FIG. 3. RNAi TbMTr1 induces defects SL RNA U1 snRNA methylation SL RNA 3&#226;&#172;&#732;-end formation. (A) RNA collected after <br /> 9 days pRTbMTr1 cells grown presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) Tet run 1.1% formaldehyde agarose gel, blotted, probed <br /> with TbMTr1 coding region. Ethidium bromide-stained rRNA loading control. (B) Schematic cap 4 showing possible primer <br /> extension products interpretations intermediate cap structures. (C) Primer extension analysis SL RNA using total RNA clonal <br /> pRTbMTr1 RNAi cell lines 9 days Tet induction. Extension products separated 6% polyacrylamide-8 M urea gel run to <br /> the cognate sequence ladder. (D) Primer extension analysis U1 snRNA pRTbMTr1 RNAi samples. (E) Size analysis SL RNA U1 <br /> snRNA medium-resolution acrylamide RNA blotting using oligonucleotide probes TbStloopI TbU1-20, respectively. The number days <br /> post-Tet addition indicated. 5S rRNA served loading control. <br />  6088     ZAMUDIO ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Absence TbMTr1 affects 5&#226;&#172;&#732; cap 1 3&#226;&#172;&#732; maturation. (A) Absence TbMTr1 alleles. Shown Southern blot wild-type <br /> (WT) null (KO) clonal cell lines digested XhoI. The products resolved 1% agarose gel, visualized ethidium bromide <br /> staining, transferred, probed coding region TbMTr1. Lane M 1-kb ladder (Invitrogen). (B) Altered substrate SL <br /> methylation TbMTr1-null line. Primer extension analysis using total RNA collected wild-type clone 1 TbMTr1-null cell lines, <br /> as described Fig. 3. (C) Altered cap methylation U1 snRNA measured primer extension. (D) Cap 1-deficient SL RNA correlates <br /> with accumulation 3&#226;&#172;&#732;-extended forms. Total RNAs samples collected wild-type TbMTr1-null cells resolved through <br /> a 6% polyacrylamide-8 M urea gel probed &#226;?&#165;-32P-labeled oligonucleotide, TbSL-40, SL RNA intron. Mature SL RNA <br /> is denoted &#226;&#8364;&#339;M&#226;&#8364;?. <br />  <br />  <br />  <br />  <br /> corresponding cap 3 rarely seen wild-type profiles,          position 1 modification (Fig. 3D). Additional faint bands that <br /> suggesting addition cap 3 followed rapidly cap 4        unaffected RNAi comigrated positions 3 5. In <br /> and supporting hypothesis TbMTr3 performing               induced samples, increase band migrating A1 <br /> 2&#226;&#172;&#732;-O-ribose methylation adjacent position (64). The            apparent, demonstrating RT met lower <br /> profile 3&#226;&#172;&#732;-most methylations mapped electro-            level impediment methylation position 1. The remain- <br /> phoresis extension products 6% polyacryl-            ing cap 1 residual incomplete knockdown of <br /> amide gel parallel corresponding DNA sequence.            TbMTr1, high relative stability U1 snRNA compared to <br /> In wild-type cells nucleotide A1 SL modified        consumed SL RNA, presence dimethylated <br /> the 2&#226;&#172;&#732;-O ribose dimethylated N6 position         adenine base. Thus, TbMTr1 involved 2&#226;&#172;&#732;-O-ribose <br /> the adenine ring (5), sufficient inhibit extension   methylation U1 snRNA. As U1 snRNA likely <br /> by RT (27, 33). Since TbMTr1 showed vitro cap 1 ribose             acquires TMG posttranscriptionally, TbMTr1 modification <br /> methylation activity, expected detect difference SL        precede TMG acquisition recognize substrate <br /> intermediate population loss position 1 modi-         caps methylation. <br /> fications occurred RNAi cell lines. Compared nonin-            The SL RNA gene transcribed 3&#226;&#172;&#732;-extended form that <br /> duced samples, decrease band comigrating A2         processed exonucleases, including TbSNIP trimming at <br /> observed, corresponding absence cap 1 modification           end maturation (65). The accumulation 3&#226;&#172;&#732;-ex- <br /> (Fig. 3C). As TbMTr1 RNA knockdown resulted reduction              tended forms detected inhibition exportin 1 <br /> of SL A2-extension termination vitro activity showed       activity, Sm protein studies, assorted SL mutants (35, <br /> 2&#226;&#172;&#732;-O-ribose MTase cap 1 activity, concluded TbMTr1            51, 65, 67). Hybridization analysis RNA Tet-induced <br /> methylates ribose position 1 SL cap 1       RNAi cells 2 days postinduction revealed ac- <br /> not required cell viability. The SL intermediate        cumulation substrate SL slower migration (Fig. 3E), <br /> base dimethylation A1 detected unlikely        characteristic 3&#226;&#172;&#732;-extended molecules. No 3&#226;&#172;&#732;-trimming de- <br /> to precede 2&#226;&#172;&#732;-O-ribose methylation cap 4 biogenesis.               fects observed U1 snRNA 5S rRNA. <br />    Kinetoplastid U1 snRNA shares 6 7 nt            Loss modification position 1 SL RNA U1 <br /> SL identical methylations position 1 Crithidia fascicu-     snRNA TbMTr1 RNAi lines consistent vitro <br /> lata. The 3 nt Crithidia U1 snRNA resistant          enzymatic activity. Position 1 ribose modification does ap- <br /> cleavage RNases alkali, indicating ribose methylation       pear play essential role trans-splicing translation <br /> the positions (47). We determined methylation           detectable effect growth measured induced <br /> status U1 snRNA TbMTr1 RNAi lines primer exten-              cultures. <br /> sion. The U1 snRNA extension <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span> showed majority                   Additional cap 4 defects absence cap 1 methylation. <br /> species A2 band noninduced cells, corresponding          Given potential importance cap 1 modifications, we <br />  VOL. 27, 2007                                                                               RNA CAP 1 RIBOSE METHYLATION                   6089 <br />  <br />  <br />  <br />  <br />   FIG. 5. Spliced SL position 4 methylation despite 2&#226;&#172;&#732;-O-ribose methylation absence position 1. (A) Spliced SL methylated position <br /> 4 TbMTr1-null lines. The poly(A)&#226;&#171;&#185;-selected mRNA isolated wild-type (WT) TbMTr1-null (KO) cells extended using primer <br /> TbWTexon, complementary exon sequence, described Fig. 3. In panel poly(A)&#226;&#171;&#185;-selected mRNA extended with <br /> TbSL-40 negative control. (B) Absence 2&#226;&#172;&#732;-O-ribose methylation spliced SL. Poly(A)&#226;&#171;&#185;-selected RNA treated tobacco acid <br /> pyrophosphatase HK phosphatase remove 5&#226;&#172;&#732;-m7G cap phosphates, &#226;?&#165;-32P labeled polynucleotide kinase, digested RNase <br /> T2. A ladder alkali-treated RNA (OH) corresponding sequence serves size marker. Black stars indicate common bands the <br /> labeling procedure; black arrow indicates T2-resistant cap 4 band; white arrow indicates T2-resistant band unique TbMTr1-null <br /> cells. <br />  <br />  <br />  <br /> confirmed absence growth <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> making                      The TbMTr1-null line examined substrate SL RNA <br /> TbMTr1-null mutant line. To knock alleles                     <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> high-resolution RNA blot. In wild-type cells SL <br /> TbMTr1, introduced constructions homologous recom-                 RNA present main forms: 141-nt species, the <br /> bination using flanking UTRs.                                         mature 142-nt molecule (20), 143-nt TbSNIP sub- <br />    A Southern blot wild-type TbMTr1-null T. brucei                 strate, representing 71:29 ratio mature underprocessed <br /> lines probed coding region TbMTr1, confirm-               SL RNA. TbMTr1-null lines showed decrease mature <br /> ing loss alleles (Fig. 4A). Primer extension per-             142-nt accumulation 144- 150-nt forms, representing <br /> formed total RNA. The loss band comigrating                21:79 ratio mature underprocessed SL RNA (Fig. 4D). <br /> A2 TbMTr1-null cells confirmed, rescue           An increase 143-nt form indicated nuclear <br /> the null phenotype reintroduction recombinant                   TbSNIP acted majority SL RNA TbMTr1- <br /> TbMTr1-hemagglutinin gene (data shown). The substrate                 null cells. <br /> SL phenotype pronounced null lines                   Lack methylation position 1 SL cap correlates <br /> the induced RNAi lines, significant increase band           reduced methylation additional cap 4 nucleotides and <br /> comigrating A1 indicating cap 0 form                removal 3&#226;&#172;&#732;-extended poly(U) tail. The 3&#226;&#172;&#732;-end processing <br /> majority species. A decrease levels C3 A5 re-               delay supports role cap 1 methylation biogenesis of <br /> flected lowered levels intermediate methylations po-            SL RNA. Methylations downstream ribose-deficient cap <br /> sitions 2 4. The cap 0 extension product 48.1%                  1 nucleotide indicate TbMTr2 TbMTr3 activities are <br /> presence TbMTr1-null cells, compared 19% wild                independent modification(s) position. <br /> type, implying kinetics cap 4 formation               Absence 2&#226;&#180;&#8226;-O-methylation position 1 TbMTr1-null <br /> was altered absence cap 1.                                      mRNA. TbMTr2 TbMTr3 studies demonstrate under- <br />    U1 snRNA complete loss band corresponding                 methylated SL trans spliced. We examined cap structure <br /> to cap 1 TbMTr1-null cells (Fig. 4C). The U2 U4                    mRNA TbMTr1-null cells assess methylation <br /> snRNAs, carry cap 1 methylations (47), showed               pattern trans-spliced translation-competent tran- <br /> differences TbMTr1-null line primer extension (data             scripts. <br /> not shown). As cap analysis complicated down-                  An oligonucleotide complementary SL exon, TbWTexon, <br /> stream methylation U1 snRNA, loss band position              used primer extension reactions poly(A)&#226;&#171;&#185; RNA (Fig. <br /> 1 revealed base ribose methylations absent;            5A). The extension wild-type mRNA reflects SL the <br /> this stated definitively substrate SL.                      majority position 4 modifications, contrast cap 4 <br />  6090     ZAMUDIO ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. TbMTr1-GFP localizes nucleoplasm. Clonal YTAT cell lines used generate line containing GFP gene fused frame <br /> with TbMTr1 gene endogenous locus. The expressed TbMTr1-GFP fusion protein visualized light microscopy UV light <br /> illumination. The positions nuclear kinetoplast DNAs revealed staining DAPI. The cells shown phase contrast. <br />  <br />  <br />  <br /> intermediates captured substrate extensions. No difference             virtual absence 3&#226;&#172;&#732; mature substrate SL TbMTr1-null <br /> was observed wild-type TbMTr1-null lines,                 population shown high-resolution RNA analysis, the <br /> with identical profiles showing position 4-modified SL             majority SL signal nucleoplasm likely con- <br /> majority species (&#226;&#172;&#381; 95%). As control free SL RNA                    tributed underprocessed forms, indicating delay nuclear <br /> contamination, poly(A)&#226;&#171;&#185; RNA assayed TbSL40                   egress. <br /> intron probe, yielding product. Next mRNA samples were <br /> subjected RNase T2 digestion confirm cap 1 undermethyl-                                        DISCUSSION <br /> ation. RNase T2 nonspecific endonuclease cannot <br /> cleave RNA 2&#226;&#172;&#732;-O-ribose methylated. An RNase T2-re-                      We report identification characterization first <br /> sistant fragment migrated slightly faster corre-            eukaryotic cap 1 2&#226;&#172;&#732;-O-ribose MTase acts SL RNA, and <br /> sponding 5-nt OH-degradation marker obtained                     mRNA, U1 snRNA T. brucei. <br /> wild-type mRNA (Fig. 5B, black arrow) representing fully              AdoMet-dependent methylation m7G-capped substrate <br /> modified cap 4 (55). The cap 4, T2-resistant band                 shown recombinant TbMTr1 vitro. <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>Knockout</span> of <br /> observed mRNA derived TbMTr1-null cells,               TbMTr1 alleles resulted viable cells defects in <br /> a T2-resistant band migrating 1-nt 2-nt mark-             ribose modification cap 1 position SL RNA U1 <br /> ers revealed (Fig. 5B, white arrow). The novel band               snRNA. TbMTr1-null cells displayed delay processing of <br /> represent nucleotide A1 N6 monomethylation dimethyl-              SL RNA loss 2&#226;&#172;&#732;-O-ribose methylation position 1 <br /> ation; however, assignment confirmed addi-                corresponded decrease subsequent cap methylation and <br /> tional analysis.                                                          accumulation 3&#226;&#172;&#732;-extended forms. <br />    The mRNA TbMTr1-null cells lacks complete cap 4,                     The significance mRNA cap 1 evident main- <br /> consistent absence ribose methylation position 1.          tenance multiple viral systems, cap ribose methyl- <br /> Subsequent activities cap 4 maturation pathway occur               ation increases binding ribosomes (37) impacts transla- <br /> efficiently absence TbMTr1 activity; however, kinetic           tional control viral transcripts (8). In influenza virus, <br /> delays apparent substrate profile.                             priming mRNA transcription efficient stolen <br />    TbMTr1 localizes nucleus T. brucei. Cap 1 meth-              host caps possess cap 1 (7). No function snRNA cap 1 <br /> ylation Sm complex independent SL RNA (35, 51, 65).                known. The role TbMTr1 processing SL RNA <br /> To investigate spatial temporal aspects cap 1 synthesis,           does exclude synergistic role cap 1 ribose methylation <br /> we determined subcellular localization TbMTr1               translation. In viral cap 2&#226;&#172;&#732;-O-ribose MTases cyto- <br /> the substrate SL TbMTr1-null line.                                   solic eIF4E translation initiation factor, recognition the <br />    A TbMTr1-GFP fusion created introduced                m7G cap occurs coplanar aromatic residues (44). <br /> cognate genomic locus. Localization fusion protein coin-           The eIF4E proteins trans-splicing organisms Caenorhab- <br /> cided nuclear DAPI stain (Fig. 6), indicating           ditis elegans Leishmania major distinguish between <br /> majority protein nucleoplasmic. Although GFP fusions           different cap structures including base ribose modifications <br /> can lead mislocalization, especially overexpressed, nu-           (22, 63), reflecting downstream discrimination effects <br /> clear concentration typical default. RNA-fluorescent        cap 1 enzymes TbMTr1. <br /> situ hybridization TbMTr1-null cells SL RNA revealed               TbMTr1 retained positional specificity related <br /> cytosolic overaccumulation (data shown), seen          baculovirus MTase 1 capping enzyme, contrast the <br /> Sm protein knockdown experiments.                                         changes positional specificity cap 2 cap 3 TbMTr2 <br />    The SL TbMTr1 localizations support delay                 TbMTr3, kinetoplastid homologs vaccinia virus <br /> processing kinetics cap 1-deficient SL substrates. Nuclear            cap 1 MTase VP39 (21, 49). Horizontal gene transfer between <br /> SL retention induced poisoning export                  viruses eukaryotic cells given rise three <br /> pathway using leptomycin B, resulting undermethylated              genes, followed altered enzyme specificity viral cap <br /> 3&#226;&#172;&#732;-extended molecules (67). The phenotypic shift                   1 modification protists. The TbMTr1 eukaryotic <br /> TbMTr1-null line indicates majority SL population             orthologs likely derived similar viral origins and <br /> still undergoing biogenesis, wild-type cells approxi-            additional domains absent viral orthologs that <br /> mately 79% SL completed journey. Considering               important activity specificity. All eukary- <br />  VOL. 27, 2007                                                                             RNA CAP 1 RIBOSE METHYLATION                             6091 <br />  <br />  <br /> otic members TbMTr1 family contain N-terminal exten-           LC07032. J.R.Z. supported NSF Louis Stokes Alliance for <br /> sions 40 200 aa predicted disordered. The        Minority Participation Award HRD-0115115:3 USPHS National <br />                                                                       Research Service Award GM07104. <br /> homologs metazoa contain additional C-terminal do- <br /> mains related guanylyltransferases lack catalytic resi-                                         REFERENCES <br /> dues (15).                                                             1. Alexandrov, A., I. Chernyakov, W. Gu, S. L. Hiley, T. R. Hughes, E. J. <br />    Cap 4 formation T. brucei linear, methylations              Grayhack, E. M. Phizicky. 2006. Rapid tRNA decay result lack <br /> starting 5&#226;&#172;&#732; end (34). Based nuclear localization          nonessential modifications. Mol. Cell 21:87&#226;&#8364;&#8220;96. <br />                                                                        2. Arhin, G. K., H. Li, E. Ullu, C. Tschudi. 2006. A protein related the <br /> of TbMTr1-GFP SL cap 1 phenotype Sm complex-                   vaccinia virus cap-specific methyltransferase VP39 involved cap 4 mod- <br /> deficient transcripts (35, 51, 65), TbMTr1 acts independently             ification Trypanosoma brucei. RNA 12:53&#226;&#8364;&#8220;62. <br /> of Sm protein binding facilitates biogenesis SL             3. Arhin, G. K., E. Ullu, C. Tschudi. 2006. 2&#226;&#172;&#732;-O-Methylation position 2 <br />                                                                           trypanosome spliced leader cap 4 mediated 48kDa protein <br /> RNA. TbMTr2 vitro activity does require prior cap 1                related vaccinia virus VP39. Mol. Biochem. Parasitol. 147:137&#226;&#8364;&#8220;139. <br /> modification (19), consistent vivo results            4. Banerjee, A. K. 1980. 5&#226;&#172;&#732;-terminal cap structure eucaryotic messenger <br />                                                                           ribonucleic acids. Microbiol. Rev. 44:175&#226;&#8364;&#8220;205. <br /> TbMTr2 TbMTr3. Subsequent 5&#226;&#172;&#732;-end processing                 5. Bangs, J. D., P. F. Crain, T. Hashizume, J. A. McCloskey, J. C. <br /> enhanced cap 1, reflecting decrease ability                Boothroyd. 1992. Mass spectrometry mRNA cap 4 trypanosomatids <br /> TbMTr2 TbMTr3 recognize A1 ribose-methylation-                 reveals novel nucleosides. J. Biol. Chem. 267:9805&#226;&#8364;&#8220;9815. <br />                                                                        6. Bisaillon, M., G. Lemay. 1997. Viral cellular enzymes involved in <br /> deficient substrate inaccessibility substrate            synthesis mRNA cap structure. Virology 236:1&#226;&#8364;&#8220;7. <br /> these nucleus-localized enzymes, potentially inefficient        7. Bouloy, M., S. J. Plotch, R. M. Krug. 1980. Both 7-methyl the <br /> trafficking substrate SL RNA. The cap 1 base MTase(s)              2&#226;&#172;&#732;-O-methyl groups cap mRNA strongly influence ability act <br />                                                                           primer influenza virus RNA transcription. Proc. Natl. Acad. Sci. USA <br /> yet identified, apparent absence dimethy-            77:3952&#226;&#8364;&#8220;3956. <br /> lation U1 snRNA TbMTr1-null cells suggests require-            8. Brownlee, G. G., E. Fodor, D. C. Pritlove, K. G. Gould, J. J. Dalluge. <br /> ment TbMTr1 activity additional modifications;                1995. Solid phase synthesis 5&#226;&#172;&#732;-diphosphorylated oligoribonucleotides and <br />                                                                           conversion capped m7Gppp-oligoribonucleotides use primers <br /> appearance novel band mRNA RNase T2 digestions                  influenza A virus RNA polymerase vitro. Nucleic Acids Res. 23:2641&#226;&#8364;&#8220; <br /> indicates SL base modification proceed po-               2647. <br />                                                                        9. Bujnicki, J. M., L. Rychlewski. 2001. Reassignment specificities two <br /> sition 1 ribose methylation, primer extension detection            cap methyltransferase domains reovirus &#226;?&#173;2 protein. Genome Biol. <br /> the base intermediate blocked downstream methylations.              2:0038.1&#226;&#8364;&#8220;0038.6. <br /> Developed assays using two-dimensional TLC cap analysis           10. Camper, S. A., R. J. Albers, J. K. Coward, F. M. Rottman. 1984. Effect <br />                                                                           undermethylation mRNA cytoplasmic appearance half-life. Mol. <br /> are dependent isolation RNase T2-resistant fragment              Cell. Biol. 4:538&#226;&#8364;&#8220;543. <br /> that lost TbMTr1-null cells. Complete cap 4        11. Coleman, T. M., G. Wang, F. Huang. 2004. Superior 5&#226;&#172;&#732; homogeneity of <br /> thought required trans-splicing SL                  RNA ATP-initiated transcription T7 phi 2.5 promoter. Nu- <br />                                                                           cleic Acids Res. 32:e14. <br /> mRNAs (32, 53, 56, 57). The knockout TbMTr1                12. Decatur, W. A., M. J. Fournier. 2003. RNA-guided nucleotide modifi- <br /> underscores independence complete cap 4 formation                  cation ribosomal RNAs. J. Biol. Chem. 278:695&#226;&#8364;&#8220;698. <br />                                                                       13. Decatur, W. A., M. J. Fournier. 2002. rRNA modifications ribosome <br /> and trans-splicing, shown exon mutagenesis studies (52,             function. Trends Biochem. Sci. 27:344&#226;&#8364;&#8220;351. <br /> 68).                                                                  14. Djikeng, A., L. Ferreira, M. D&#226;&#8364;&#8482;Angelo, P. Dolezal, T. Lamb, S. Murta, V. <br />    TbMTr1 nonviral enzyme family               Triggs, S. Ulbert, A. Villarino, S. Renzi, E. Ullu, C. Tschudi. 2001. <br />                                                                           Characterization candidate Trypanosoma brucei U1 small nuclear RNA <br /> defined cap 1 MTase. In humans cap 1 mRNA 2&#226;&#172;&#732;-O-                  gene. Mol. Biochem. Parasitol. 113:109&#226;&#8364;&#8220;115. <br /> ribose MTase activity partially purified resides      15. Feder, M., J. Pas, L. S. Wyrwicz, J. M. Bujnicki. 2003. Molecular <br /> the nucleus; subsequent cap 2 2&#226;&#172;&#732;-O-ribose methylation occurs              phylogenetics RrmJ/fibrillarin superfamily ribose 2&#226;&#172;&#732;-O-methyltrans- <br />                                                                           ferases. Gene 302:129&#226;&#8364;&#8220;138. <br /> in cytoplasm (30, 42). The appearance cap ribose meth-         16. Freistadt, M. S., G. A. M. Cross, H. D. Robertson. 1988. Discontinuously <br /> ylation Xenopus laevis maternal transcripts correlates            synthesized mRNA Trypanosoma brucei contains highly methylated <br />                                                                           5&#226;&#172;&#732; cap structure, m7GpppA*A*C(2&#226;&#172;&#732;-O)mU*A. J. Biol. Chem. 263:15071&#226;&#8364;&#8220; <br /> translation initiation (25). Inhibition ribose internal            15075. <br /> adenosine N6-methylation mammalian cells leads de-              17. Furuichi, Y., A. J. Shatkin. 2000. Viral cellular mRNA capping: past <br /> creased cytoplasmic half-life, suggestive role trafficking        prospects. Adv. Virus Res. 55:135&#226;&#8364;&#8220;184. <br />                                                                       18. Gillian-Daniel, D. L., N. K. Gray, J. Astrom, A. Barkoff, M. Wickens. <br /> or processing mRNA transcripts (10). Interestingly,                  1998. Modifications 5&#226;&#172;&#732; cap mRNAs Xenopus oocyte matu- <br /> TbMTr1 family member Caenorhabditis elegans caused ma-                 ration: independence changes poly(A) length impact trans- <br /> ternal sterility targeted RNAi (46).                              lation. Mol. Cell. Biol. 18:6152&#226;&#8364;&#8220;6163. <br />                                                                       19. Hall, M. P., C. K. Ho. 2006. Functional characterization 48 kDa <br />    At point, role cap 4 kinetoplastid biology              Trypanosoma brucei cap 2 RNA methyltransferase. Nucleic Acids Res. 34: <br /> remains open question. As case tRNA rRNA,                5594&#226;&#8364;&#8220;5602. <br />                                                                       20. Hitchcock, R. A., G. M. Zeiner, N. R. Sturm, D. A. Campbell. 2004. The <br /> the effects cap methylations cumulative syner-              3&#226;&#172;&#732;-termini small RNAs Trypanosoma brucei. FEMS Microbiol. Lett. <br /> gistic. Family-wide pathway conservation suggests key, func-            236:73&#226;&#8364;&#8220;78. <br /> tionally selected role. Likewise prevalence diversity      21. Hodel, A. E., P. D. Gershon, X. Shi, S. M. Wang, F. A. Quiocho. 1997. <br />                                                                           Specific protein recognition mRNA cap alkylated base. Nat. <br /> higher-order cap structures eukaryotes viruses allude           Struct. Biol. 4:350&#226;&#8364;&#8220;354. <br /> a primary function appreciate fully.              22. Jankowska-Anyszka, M., B. J. Lamphear, E. J. Aamodt, T. Harrington, E. <br />                                                                           Darzynkiewicz, R. Stolarski, R. E. Rhoads. 1998. Multiple isoforms of <br />                      ACKNOWLEDGMENTS                                      eukaryotic protein synthesis initiation factor 4E Caenorhabditis elegans can <br />                                                                           distinguish mono- trimethylated cap structures. J. Biol. Chem. <br />    We thank Jay Bangs pKO vectors; Thomas Seebeck             273:10538&#226;&#8364;&#8220;10542. <br /> pMOT vectors; Kent Hill YTAT cells use Zeiss       23. Johnson, P. J., J. M. Kooter, P. Borst. 1987. Inactivation transcription <br />                                                                           UV irradiation T. brucei provides evidence multicistronic tran- <br /> fluorescence microscope; Murray Schnare advice cap analysis;       scription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> including VSG gene. Cell 51:273&#226;&#8364;&#8220;281. <br /> and Miriam Giardini, Robert Hitchcock, Sean Thomas, Scott         24. Jones, D. T., W. R. Taylor, J. M. Thornton. 1992. The rapid generation <br /> Westenberger stimulating discussions.                                 <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> data matrices protein sequences. Comput. Appl. Biosci. <br />    This work supported NIH grant AI034056           8:275&#226;&#8364;&#8220;282. <br /> grants Ministry Education Czech Republic 2B06129     25. Kuge, H., J. D. Richter. 1995. Cytoplasmic 3&#226;&#172;&#732; poly(A) addition induces <br />  6092         ZAMUDIO ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />       5&#226;&#172;&#732; cap ribose methylation: implications translational control maternal     47. Schnare, M. N., M. W. Gray. 1999. A candidate U1 small nuclear RNA <br />       mRNA. EMBO J. 14:6301&#226;&#8364;&#8220;6310.                                                           trypanosomatid protozoa. J. Biol. Chem. 274:23691&#226;&#8364;&#8220;23694. <br /> 26.   Kumar, S., K. Tamura, M. Nei. 2004. MEGA3: integrated software            48. Shatkin, A. J., J. L. Manley. 2000. The ends affair: capping and <br />       molecular evolutionary genetics analysis sequence alignment. Brief.               polyadenylation. Nat. Struct. Biol. 7:838&#226;&#8364;&#8220;842. <br />       Bioinform. 5:150&#226;&#8364;&#8220;163.                                                             49. Shi, X., P. Yao, T. Jose, P. Gershon. 1996. Methyltransferase-specific <br /> 27.   Lafontaine, D. L. J., T. Preiss, D. Tollervey. 1998. Yeast 18S rRNA               domains VP-39, bifunctional protein participates modi- <br />       dimethylase Dim1p: quality control mechanism ribosome synthesis?                 fication mRNA ends. RNA 2:88&#226;&#8364;&#8220;101. <br />       Mol. Cell. Biol. 18:2360&#226;&#8364;&#8220;2370.                                                    50. Shuman, S. 2002. What messenger RNA capping tells eukaryotic <br /> 28.   Lamb, J. R., V. Fu, E. Wirtz, J. D. Bangs. 2001. Functional analysis       evolution. Nat. Rev. Mol. Cell Biol. 3:619&#226;&#8364;&#8220;625. <br />       trypanosomal AAA protein TbVCP trans-dominant ATP hydrolysis                 51. Sturm, N. R., D. A. Campbell. 1999. The role intron structures in <br />       mutants. J. Biol. Chem. 276:21512&#226;&#8364;&#8220;21520.                                              trans-splicing cap 4 formation Leishmania spliced leader RNA. <br /> 29.   Lane, B. 1998. Historical perspectives RNA nucleoside modifications, p.            J. Biol. Chem. 274:19361&#226;&#8364;&#8220;19367. <br />       1&#226;&#8364;&#8220;20. In H. Grosjean R. Benne (ed.), Modification editing RNA. <br />                                                                                         52. Sturm, N. R., J. Fleischmann, D. A. Campbell. 1998. Efficient trans- <br />       ASM Press, Washington, DC. <br />                                                                                             splicing mutated spliced leader exons Leishmania tarentolae. J. Biol. <br /> 30.   Langberg, S. R., B. Moss. 1981. Post-transcriptional modifications of <br />                                                                                             Chem. 273:18689&#226;&#8364;&#8220;18692. <br />       mRNA. Purification characterization cap I cap II RNA (nucle- <br />                                                                                         53. Tschudi, C., E. Ullu. 2002. Unconventional rules small nuclear RNA <br />       oside-2&#226;&#172;&#732;-)-methyltransferases HeLa cells. J. Biol. Chem. 256:10054&#226;&#8364;&#8220; <br />                                                                                             transcription cap modification trypanosomatids. Gene Expr. 10:3&#226;&#8364;&#8220;16. <br />       10060. <br /> 31.   Lapeyre, B., S. K. Purushothaman. 2004. Spb1p-directed formation           54. Tycowski, K. T., Z. H. You, P. J. Graham, J. A. Steitz. 1998. Modification <br />       Gm2922 ribo <br /> </body></html>