<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> The Saccharomyces CDC13 protein single-strand TG1&#226;&#8364;&#8220;3 <br /> telomeric DNA-binding protein vitro affects <br /> telomere behavior vivo <br /> JING-JER LIN      AND   VIRGINIA A. ZAKIAN* <br />  <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>          Saccharomyces telomeres consist &apos;300 bp                         telomere. Thus, telomere-limited binding proteins could <br /> of C1&#226;&#8364;&#8220;3AyTG1&#226;&#8364;&#8220;3 DNA. Cells lacking activity essential                      theoretically help cells distinguish intact broken DNA <br /> gene CDC13 display cell cycle arrest mediated DNA                           molecules. In vitro, ciliate proteins protect telomeres <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> damage sensing, RAD9 cell cycle checkpoint, presumably                               exonucleolytic degradation (16, 18) mediate telomere&#226;&#8364;&#8220; <br /> because exhibit strand-specific loss telomeric                           telomere interactions (19). Genetic (20) biochemical (21) <br /> telomere-adjacent DNA [Garvik, B., Carson, M. &amp; Hartwell,                            data suggest organisms, including yeast, have <br /> L. (1995) Mol. Cell. Biol. 15, 6128&#226;&#8364;&#8220;6138]. Cdc13p expressed                       telomere-limited binding proteins. <br /> Escherichia coli overexpressed yeast bound specifically                        Replication chromosomal DNA molecules initiated by <br /> single-strand TG1&#226;&#8364;&#8220;3 DNA. The specificity binding displayed                        8- 12-<span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> RNA primers. The following model telomere <br /> by Cdc13p vitro indicates vivo bind                      replication yeast based studies linear plasmids. <br /> the short, constitutive single-strand TG1&#226;&#8364;&#8220;3 tails thought                      Telomeric DNA replicates late S phase (22). After <br /> present telomeres times cell cycle                      semiconservative replication telomeric DNA occurs, both <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> long single-strand TG1&#226;&#8364;&#8220;3 tails intermediates                      telomeres given DNA molecule acquire &apos;50- 100-base <br /> telomere replication. Genes located near yeast telomeres                         ss TG1&#226;&#8364;&#8220;3 tail telomerase independent mechanism (23). <br /> transcriptionally repressed, phenomenon known telomere                          These ss TG1&#226;&#8364;&#8220;3 tails probably generated cell cycle- <br /> position effect. Cells overexpressing mutant form Cdc13p                        regulated degradation C1&#226;&#8364;&#8220;3A strand. The long ss TG1&#226;&#8364;&#8220;3 <br /> had reduced telomere position effect high temperatures.                           tails converted duplex DNA mitosis (22). Al- <br /> These data suggest Cdc13p functions binding directly                         conventional replication activities synthesize the <br /> to telomeric DNA, limiting accessibility degra-                       complementary C1&#226;&#8364;&#8220;3A strand, removal terminal RNA <br /> dation transcription masking factors                          primers generate 8- 12-base ss TG1&#226;&#8364;&#8220;3 tail each <br /> that detect damaged DNA.                                                             telomere. Thus, short ss TG1&#226;&#8364;&#8220;3 tails, potential substrates a <br />                                                                                      ss G tail binding protein, occur telomeres an <br />                                                                                      individual chromosome. <br /> Telomeric DNA virtually eukaryotes consists stretch <br />                                                                                         Cells limited essential protein Cdc13p arrest the <br /> of repeated DNA strand running 59 39 the <br />                                                                                      G2 phase cell cycle RAD9-dependent process (24). <br /> center chromosome end clusters G residues <br />                                                                                      When cdc13-1 cells grown semipermissive temperature, <br /> (reviewed ref. 1). For example, Saccharomyces chromo- <br />                                                                                      increase mitotic recombination telomere- <br /> somes end &apos;300 bp heterogeneous sequence, C1&#226;&#8364;&#8220;3Ay <br />                                                                                      adjacent regions chromosome (25). Moreover, cdc13-1 <br /> TG1&#226;&#8364;&#8220;3 (2). In ciliated protozoa, G strand telomere <br />                                                                                      cells arrested growth restrictive temperatures contain ss <br /> is extended form 12- 16-base single-strand (ss) G tail                       telomeric telomere-adjacent DNA, strand run- <br /> that present cell cycle (3&#226;&#8364;&#8220;5). In yeast,                ning 59 39 end center chromo- <br /> transient ss TG1&#226;&#8364;&#8220;3 tails .30 bases <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> detected                         some, C1&#226;&#8364;&#8220;3A strand, preferentially degraded. In rad9 <br /> plasmid chromosomal telomeres late S phase (6).                           cdc13-1 cells, strand-specific degradation extends as <br />    Yeast chromosomes telomeres unstable (7).                             30 kb end chromosome (25). These <br /> Telomeric DNA essential replication                              data suggest Cdc13p limits C-strand degradation that <br /> protection chromosome ends helps RAD9 DNA                                 occurs end S phase (23). <br /> damage sensing, cell cycle checkpoint distinguish intact                           Here Escherichia coli- Saccharomyces- <br /> broken chromosomes. In addition, yeast, transcription                      produced Cdc13p binds ss TG1&#226;&#8364;&#8220;3 DNA vitro. The binding <br /> telomere-linked genes repressed (8), phenomenon known                           properties Cdc13p consistent able bind <br /> as telomere position effect (TPE).                                                   short ss TG1&#226;&#8364;&#8220;3 tail, probably present the <br />    Yeast telomeric DNA assembled nonnucleosomal                            cell cycle, long ss TG1&#226;&#8364;&#8220;3 tails generated end of <br /> chromatin structure, telosome (9). The major telosomal                           S phase. Cells overexpressing cdc13-1 mutant form the <br /> protein yeast Rap1p (10, 11). Rap1p sequence-specific,                    protein defects TPE high temperature. Together with <br /> duplex DNA-binding protein (12, 13). Although Rap1p                             previous information (25), data suggest Cdc13p a <br /> binds ss TG1&#226;&#8364;&#8220;3 DNA, does orders                         telosomal protein vivo. <br /> magnitude affinity duplex telomeric DNA (14). <br /> Rap1p binding limited telomeres are <br /> many internal binding sites Rap1p vivo (e.g., ref. 15).                                MATERIALS AND METHODS <br />    In contrast Rap1p, major telomere binding proteins                           Oligonucleotides, Competitor Nucleic Acids, Electro- <br /> in ciliated protozoa Oxytricha (16) Euplotes (17)                        phoretic Mobility-Shift Assays. Oligonucleotides (DNAgency, <br /> are telomere-limited. These proteins restricted telo-                         Malvern, PA) listed Table 1. The sequence TG22 was <br /> meres binding requires ss G tail                     based sequence cloned yeast telomere (2). Oligo- <br />  <br /> The publication costs article defrayed page charge           Abbreviations: ss, single-strand; TPE, telomere position effect; HA, <br /> payment. This article marked &#226;&#8364;&#732;&#226;&#8364;&#732;advertisement&#226;&#8364;&#8482;&#226;&#8364;&#8482;           hemagglutinin; YC, yeast complete; FOA, 5-fluoroorotic acid, FOAR, <br /> accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                        FOA resistant. <br />  <br />                                                                              13760 <br />            Biochemistry: Lin Zakian                                                 Proc. Natl. Acad. Sci. USA 93 (1996)     13761 <br />  <br />           Table 1.   Oligonucleotides used study <br />              Name                                                           Sequence <br />           TG22                       59-GTGGTGGGTGGGTGTGTGTGGG-39 <br />           TG43                       59-GTGGTGGGTGGGTGTGTGTGGGTGTGGTGGGTGTGTGGGTGTG-39 <br />           TG19                       59-GCGGATCCCGGGTGTGGTG-39 <br />           CAS9                       59-GGGATCCGC-39 <br />           CAS14                      59-CACCCGGGATCCGC-39 <br />           CAS19                      59-CACCACACCCGGGATCCGC-39 <br />           39 Tel                     59-GCCATGGCCACCGGGATCCCGTGGTGGGTGGGTGTGTGTGGG-39 <br />           Internal Tel               59-GCCATGGCCAGTGGTGGGTGGGTGTGTGTGGGCCGGGATCCC-39 <br />           59 Tel                     59-GTGGTGGGTGGGTGTGTGTGGGGCCATGGCCACCGGGATCCC-39 <br />           CDC135D                    59-TGTTTCTCTTTGGATACGAATGACCGTGGAAACTATCGCCTCGTTCAGAATGACACG-39 <br />           CDC133D                    59-GCAATTTGGCACCGCCGCGTTGGGCTGCGCGGATCATGTCCTCTTGGCCTCCTCTAG-39 <br />           CDC135                     59-GCTCTAGAGCCCGGGCCATGGATACCTTAGAAG-39 <br />           CDC133                     59-AACTGCATACTAGTCGACTCTTGCTTCTTACC-39 <br />           CDC13ts5                   59-AGCTCTAGATGCATGGATACCTTAGAAGAGCCTGAG-39 <br />           CDC13ts3                   59-AGGCTCGCAGTCGACTTGGGCTGCGCGGATCATGTC-39 <br />  <br /> nucleotides 32P-labeled T4 polynucleotide kinase               tag, poly(A) tail inserted SpeI, SmaI, and <br /> (New England Biolabs) purified 12% sequencing               ApaI sites, respectively, pRS425 (a generous gift C. <br /> gel. For ease comparison, competition experiments,           Lin, Fred Hutchinson Cancer Research Center). The resulting <br /> concentration competitors containing ss TG1&#226;&#8364;&#8220;3 DNA              plasmid digested PstI, blunt-ended T4 <br /> expressed terms number bases TG1&#226;&#8364;&#8220;3 DNA. For            DNA polymerase, ligated NcoI linker. In pTHA- <br /> example, concentration oligonucleotide TG22          CDC13, complete CDC13 ORF fused frame with <br /> 1 molecule ml, solution 39-Tel, 42-base oligonucle-         HA tags amino terminus expressed the <br /> otide 22 bases TG1&#226;&#8364;&#8220;3 DNA, equal concen-            PGK1 promoter. Other HA tags, Cdc13p pro- <br /> tration 1 molecule ml, TG43 ss                duced pTHA-CDC13 identical wild-type protein. <br /> pTG270 be, respectively, 0.51 0.08 molecule ml.             To express Cdc13p E. coli, PCR products obtained <br /> To make tailed duplex fully duplex substrates, labeled TG19         using oligonucleotides CDC135 CDC133 digested <br /> was annealed excess CAS9, CAS14, CAS19               NcoI PstI ligated NcoI&#226;&#8364;&#8220;PstI-digested <br /> oligonucleotide, products purified 12%             pKK233-2 (Pharmacia) generate pKKCDC13. In plas- <br /> polyacrylamide gel. Total yeast RNA BDH. To                   mid, CDC13 control isopropyl b-D- <br /> generate pTG270, 276 bp TG1&#226;&#8364;&#8220;3 DNA cloned                     thiogalactoside-inducible E. coli trc promoter first <br /> telomere (26) inserted pYES2 (Invitrogen). ss                 AUG &apos;100 bp past CDC13 termination codon. <br /> pTG270 prepared superinfection XL1-blue cells                   To clone cdc13-1, genomic DNA strain 2758&#226;&#8364;&#8220;8-4b <br /> carrying pTG270 M13-CO8 followed precipitation              (MATa cdc13-1 his7 leu2-3, 112 ura3-52 trp1-289; provided by <br /> phagemids PEG. Phagemids extracted phenoly                L. Hartwell, University Washington) PCR-amplified <br /> chloroform ethanol-precipitated. ss pTG270 sheared             oligonucleotides CDC13ts5 CDC13ts3 using Ex- <br /> by sonication sized agarose gel electrophoresis. Duplex         pand High Fidelity PCR (Boehringer Mannheim). The <br /> pTG270 prepared standard plasmid preparation             PCR product cloned pCRII provided TA <br /> the strain M13-C08 infection.                         Cloning Kit (Invitrogen) generate pTA-cdc13-1. To express <br />    Total yeast E. coli extracts gel mobility-shift assays   cdc13-1 gene yeast, pTA-cdc13-1 digested SalI <br /> were prepared 5-ml cultures described (27). The                XhoI, 2.8-kbp SalI&#226;&#8364;&#8220;XhoI fragment containing the <br /> concentration protein yeast E. coli extracts         gene isolated used replace comparable SalI&#226;&#8364;&#8220; <br /> &apos;1-2 mgyml &apos;2-4 mgyml, respectively. Cell extracts             XhoI fragment CDC13 pTHA-CDC13 generate a <br /> mixed 1.0 ng 32P-labeled TG22 15-ml reaction              plasmid 59 55 bp gene were <br /> mixture containing 50 mM TriszHCl (pH 7.5), 1 mM EDTA,                 derived cdc13-1. The following criteria established that <br /> 50 mM NaCl, 1 mM DTT, 1 mg ss poly(dIzdC)                   pTHA-cdc13-1 contained correct gene: (i) For 11 12 <br /> incubated room temperature 10 min. Three microliters            restriction enzymes, cdc13-1 restriction map as <br /> of 80% glycerol added reaction mixtures               CDC13. (ii) Cloned cdc13-1 EcoRI site present in <br /> loaded 8% nondenaturing polyacrylamide gel. Before               CDC13 present genomic DNA cdc13-1 <br /> loading, gels run 120 V 10 min. Electrophoresis        strain 2758&#226;&#8364;&#8220;8-4b. Although polymorphism se- <br /> in TBE (89 mM Triszboratey2 mM EDTA) 120 V 2.5 hr               quenced, infer position <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>mutation</span> and <br /> after loading samples. The gel dried autoradio-            examination CDC13 sequence generated a <br /> graphed. For competition analysis, 1.0 ng 32P-labeled TG22          proline-to-serine change amino acid 371. (iii) pTHA- <br /> was mixed varying amounts cold competitors              cdc13-1 rescued lethality cdc13D strains 258C and <br /> addition cell extracts. Binding activity quantified         308C, cells growth defects higher temperatures <br /> with Molecular Dynamics PhosphorImager.                              did grow temperatures .378C (data shown). <br />    Expression Cdc13p Bacteria Yeast. To construct               Yeast Methods. All yeast strains constructed the <br /> plasmids overexpressed Cdc13p S. cerevisiae, oligonu-          YPH499 background (MATa ura3-52 lys3-5 ade2-10 trp1-D63 <br /> cleotides CDC135 CDC133 mixed YEP24-                     his3-D200 leu2-D1) (28) URA3 ADE2 inserted <br /> CDC13-161-4, plasmid containing CDC13 gene (25),             near telomeres chromosomes VII-L V-R, respec- <br /> PCR-amplified using Expand High Fidelity PCR                tively, described (8). A null mutant CDC13 con- <br /> (Boehringer Mannheim). The PCR product digested               structed homozygous diploid YPH499 that <br /> NcoI SalI &apos;3-kb fragment gel-purified              heterozygous telomere insertions described (29) <br /> inserted NcoI&#226;&#8364;&#8220;SalI-digested pTHA generate pTHA-                using oligonucleotides CDC135D CDC133D. This proce- <br /> CDC13. pTHA modified form 2-mm LEU2 plasmid                dure precisely deleted CDC13 ORF replaced with <br /> pRS425 (28). To generate pTHA, PGK1 promoter, tri-HA               HIS3 create YJL401-UTAT. Spore products YJL401- <br />  13762     Biochemistry: Lin Zakian                                                            Proc. Natl. Acad. Sci. USA 93 (1996) <br />  <br /> UTAT showed 2:2 segregation viability, viable                      expression Cdc13p caused major growth defects (data <br /> spores His2. Plasmids introduced transforma-                      shown). <br /> tion YJL401-UTAT, tetrads dissected, strains                         Extracts prepared wild-type yeast cells contain multi- <br /> with appropriate genotype identified.                                  ple, abundant ss TG1&#226;&#8364;&#8220;3 binding activities detected gel <br />    For TPE assays, strains streaked yeast complete                      mobility shift assay, activities are <br /> (YC)-Leu plates grown 5 days 258C. Colonies                     probably RNA binding proteins (27). Extracts prepared <br /> resuspended water, aliquots different dilutions                  wild-type cells carrying vector analyzed gel <br /> plated YC-Leu plates 5-fluoroorotic acid (FOA)-Leu                        mobility-shift assay using TG22 detectable protein&#226;&#8364;&#8220; <br /> plates. For cells grown 258C, colonies counted 6                  DNA complex mobility E. coli-generated <br /> days; cells grown 308C, colonies counted 3                    Cdc13p&#226;&#8364;&#8220;TG22 complex (Fig. 1B, lane 4). However, when <br /> days YC-Leu 6 days FOA-Leu plates;                        extract cells overexpressing Cdc13p mixed with <br /> YC-Leu cells grown 378C, colonies counted 6                       labeled TG22, complex mobility expected for <br /> days cdc13DypTHA-cdc13-1 cells 3 days                   Cdc13p-TG22 detected (Fig. 1B, lane 5). Together with <br /> strains. Cells grown 378C FOA-Leu plates counted                     E. coli results, data indicate yeast-generated <br /> after 14 days. The frequency TPE median frequency                  Cdc13p ss TG1&#226;&#8364;&#8220;3 binding protein vitro wild-type <br /> FOA resistant (FOAR) cells 5 10 colonies strain.               cells express little allow detection gel <br />                                                                                 mobility shift assay. <br />                                                                                    Characterization Cdc13p ss TG1&#226;&#8364;&#8220;3 Binding Activity. <br />                              RESULTS                                            To determine specificity Cdc13p binding activity, <br />    E. coli- Saccharomyces-Synthesized Cdc13p Is ss TG1&#226;&#8364;&#8220;3                   extracts prepared E. coli expressing Cdc13p mixed <br />                                                                                 32P-labeled TG22 varying amounts unlabeled <br /> Binding Protein Vitro. The sequence CDC13 gene <br />                                                                                 competitor nucleic acids assayed gel mobility-shift <br /> predicts 924-amino acid protein molecular mass of <br />                                                                                 assay. As expected, unlabeled TG22 competed efficiently <br /> 104,903 Da (25). Extracts prepared E. coli express- <br />                                                                                 TG22, binding reduced 50% when <br /> ing full-length Cdc13p (Fig. 1A, lanes 4 5) cells <br />                                                                                 labeled competitor TG22 present equal concen- <br /> carrying vector (Fig. 1 A, lanes 2 3), mixed a <br /> 32P-labeled 22-base TG                                                          trations (Fig. 2 A, lanes 3&#226;&#8364;&#8220;5). An oligonucleotide the <br />                           1&#226;&#8364;&#8220;3 oligonucleotide (Table 1, TG22), <br />                                                                                 sequence G strand vertebrate telomeric DNA, <br /> and assayed DNA binding using gel mobility-shift assay.                   (T2AG3)3, competed Cdc13p binding (Fig. 2 A, lanes <br /> E. coli cells carrying vector proteins bound ss                  6&#226;&#8364;&#8220;8), &apos;10 times (T2AG3)3 needed to <br /> TG1&#226;&#8364;&#8220;3 DNA, activities appeared RNA-binding                      obtain level competition obtained with <br /> proteins (Fig. 2A, lanes 15&#226;&#8364;&#8220;17). In addition, cells expressing                  TG22 (Fig. 2B). In contrast, G-strand telomeric <br /> Cdc13p new protein&#226;&#8364;&#8220;TG22 complex (Fig. 1, indicated                     DNAs, (T4G4)3, Oxytricha telomeric sequence, and <br /> arrows) absent extract cells carrying                  (T2G4)3, Tetrahymena telomeric sequence, poor com- <br /> vector alone.                                                                   petitors Cdc13p binding (Fig. 2 A, lanes 9&#226;&#8364;&#8220;14). In addition, <br />    To study Cdc13p binding activity yeast, CDC13 ORF                     total yeast RNA (Fig. 2 A, lanes 15&#226;&#8364;&#8220;17) duplex C1&#226;&#8364;&#8220;3Ay <br /> was inserted pTHA, multicopy yeast vector                   TG1&#226;&#8364;&#8220;3 DNA (pTG270; Fig. 2 A, lanes 18&#226;&#8364;&#8220;20; Fig. 3, <br /> transcription inserted genes control                     lane 12) did compete Cdc13p binding. <br /> constitutive, efficient PGK1 promoter. Thus, cells carrying                        To determine ss TG1&#226;&#8364;&#8220;3 needs end the <br /> pTHA-CDC13 expected produce Cdc13p                             oligonucleotide Cdc13p binding, 42-base oligonucleotides <br /> wild-type cells. At 258C, 308C, 378C, wild-type cdc13D                   (Table 1) having TG22 sequences 39 end (39 Tel), 59 <br /> cells carrying pTHA-CDC13 grew rate comparable                     end (59 Tel), middle (internal Tel) oligonu- <br /> of wild-type cells pTHA alone, indicating cloned                  cleotide constructed. Each oligonucleotide tested as <br /> gene complemented cdc13D strain moderate over-                       competitor gel mobility-shift assay using TG22 E. <br />  <br />  <br />  <br />  <br />   FIG. 1. Cdc13p ss TG1&#226;&#8364;&#8220;3 binding protein. 32P-labeled TG22, 22-base oligonucleotide ss TG1&#226;&#8364;&#8220;3 DNA (Table 1), used gel <br /> mobility-shift assays using total extracts bacteria (Ec) yeast (Ysc). All gel mobility-shift assays reported presence <br /> of ss poly(dIzdC). E. coli extracts prepared cells expressing Cdc13p carrying vector alone. Yeast extracts prepared a <br /> CDC13 strain carrying vector pTHA-CDC13 overexpression Cdc13p. The arrows point TG22-protein complex seen <br /> only E. coli cells expressing Cdc13p yeast cells overexpressing Cdc13p. For panels, lane extract. (A) The extract <br /> used indicated. (B) Two micrograms extract used. <br />            Biochemistry: Lin Zakian                                                    Proc. Natl. Acad. Sci. USA 93 (1996)          13763 <br />  <br />  <br />  <br />  <br />                                                                             FIG. 3. Cdc13p binds tailed duplex molecules. Gel mobility-shift <br />                                                                           assays carried using 10 mg extract E. coli cells <br />                                                                           expressing Cdc13p carrying vector alone. 32P-labeled TG19 was <br />                                                                           annealed various oligonucleotides generate substrates 9 bp <br />                                                                           duplex DNA 10-base ss TG1&#226;&#8364;&#8220;3 tail (TG19yCAS9), 14 bp <br />                                                                           duplex DNA 5-base ss TG1&#226;&#8364;&#8220;3 tail (TG19yCAS14), duplex <br />                                                                           TG19 (TG19yCAS19). The arrow indicates position Cdc13p- <br />                                                                           DNA complex. The complex migrating slowly Cdc13p- <br />                                                                           DNA complex vector-alone lanes present reduced <br />                                                                           amounts lane 8, compared lanes 2 5, absent lane 11 <br />                                                                           E. coli-encoded ss TG1&#226;&#8364;&#8220;3 DNA binding <br />                                                                           activity, (lane 8) (lane 11) ss TG1&#226;&#8364;&#8220;3 DNA in <br />    FIG. 2. Cdc13p binding specific. (A) Gel mobility-shift assays      DNA substrates used lanes. <br /> were carried using 1 ng 32P-labeled TG22 10 mg E. coli <br /> extract cells expressing Cdc13p. Prior loading gel,       tains 270-base stretch ss TG1&#226;&#8364;&#8220;3 DNA. The phagemid DNA <br /> TG22 mixed unlabeled ss oligonucleotides sequence         used intact shearing fragments &apos;1000 bases. <br /> and relative concentration (Rel. Con.) indicated lane   Although DNAs effective competitors, sheared ss <br /> or total yeast RNA pTG270, 6300-bp plasmid containing 270 <br /> bp C1&#226;&#8364;&#8220;3AyTG1&#226;&#8364;&#8220;3 DNA. The Cdc13p&#226;&#8364;&#8220;TG22 complex indicated            pTG270 better intact ss pTG270. However, the <br /> the arrow. Lane 1 extract. (B) The gel shift activity    difference relatively small, times intact <br /> in Cdc13p-TG22 complex quantified oligonucleotide        ss pTG270 needed obtain level competition as <br /> at concentration. Relative concentration competitor nucleic       sheared ss pTG270. Thus, Cdc13p modest prefer- <br /> acids gels expressed ratios ss TG1&#226;&#8364;&#8220;3   ence for, does require, end binding. <br /> DNA competitor ss TG1&#226;&#8364;&#8220;3 DNA TG22 (see             By analogy ciliated protozoa, telomeres thought to <br /> Materials Methods); competitors did ss TG1&#226;&#8364;&#8220;3        consist duplex telomeric DNA short 39 ss G tail, <br /> DNA (e.g., Ysc RNA), expressed ratio nanograms of <br />                                                                           called &#226;&#8364;&#732;&#226;&#8364;&#732;tailed duplex.&#226;&#8364;&#8482;&#226;&#8364;&#8482; To determine Cdc13p <br /> competitor nanograms TG22. In subsequent gels, the <br /> amount gel shifted complex TG22 competitor defined      binds tailed duplex molecules vitro, 19-base oligonucleo- <br /> as 100%. The values presented average independent          tide, TG19, containing 10 bases TG1&#226;&#8364;&#8220;3 DNA 9 bases of <br /> experiments; point, values 14%       nontelomeric DNA mixed complementary oligonu- <br /> other.                                                                    cleotides various lengths (Table 1). Tailed duplex substrates <br />                                                                           generated 9 bp duplex DNA a <br /> coli-expressed Cdc13p (Fig. 4A). Because internal Tel 39              10-base 39 ss TG1&#226;&#8364;&#8220;3 tail 14 bp duplex DNA 5-base <br /> Tel competed TG22 59 Tel competed                     39 ss TG1&#226;&#8364;&#8220;3 tail. These substrates mixed E. coli <br /> almost well, ss TG1&#226;&#8364;&#8220;3 DNA need end               extract containing Cdc13p examined gel mobility- <br /> the molecule support Cdc13p binding.                                   shift assay (Fig. 3). As controls, binding Cdc13p the <br />    Cdc13p Has Appropriate Specificity Bind Long             TG19 ss oligonucleotide duplex TG19 examined. <br /> TG1&#226;&#8364;&#8220;3 Tails Present End S Phase                         Cdc13p bound TG19, demonstrating 10 bases ss <br /> Nonreplicating Telomeres. At end S phase, yeast                TG1&#226;&#8364;&#8220;3 sufficient Cdc13p binding (Fig. 3, lane 3), not <br /> telomeres transiently acquire ss TG1&#226;&#8364;&#8220;3 tails .30 bases           duplex TG19 (Fig. 3, lane 12). Cdc13p bound efficiently to <br /> in length (6). To determine longer ss TG1&#226;&#8364;&#8220;3 tails                  tailed duplex containing 10 bases ss TG1&#226;&#8364;&#8220;3 DNA (Fig. 3, <br /> substrates Cdc13p binding vitro, TG22 binding E.                lane 6) weakly tailed duplex 5-base ss <br /> coli-generated Cdc13p examined using 43-base TG1&#226;&#8364;&#8220;3                  TG1&#226;&#8364;&#8220;3 tail (Fig. 3, lane 9). Because binding Cdc13p to <br /> oligonucleotide (TG43; Table 1) competitor (Fig. 4B). TG43             tailed duplex was, anything, better binding ss <br /> competed efficiently assay molecule              TG19 (Fig. 3, compare lanes 3 6), Cdc13p the <br /> TG43 effective molecules TG22. Thus,               appropriate binding specificity associated telomeres <br /> Cdc13p binds efficiently vitro oligonucleotides             vivo. <br /> about length transient ss TG1&#226;&#8364;&#8220;3 tails seen end             Cells Expressing cdc13-1p Have Reduced TPE High <br /> of S phase.                                                           Temperature. Genetic evidence suggests yeast a <br />    To determine Cdc13p requires end binding, Cdc13p             telomere-limited binding protein important TPE <br /> binding TG22 examined using phagemid pTG270                     (20). TPE typically measured determining fraction of <br /> competitor (Fig. 4B) This &apos;6300-base circular ss DNA con-                 cells able grow plates containing FOA (8), compound <br />  13764     Biochemistry: Lin Zakian                                                        Proc. Natl. Acad. Sci. USA 93 (1996) <br />  <br />  <br />  <br />  <br />   FIG. 4. Cdc13p binding does require end. Gel mobility-shift assays carried using 1 ng 32P-labeled TG22 10 mg extract <br /> from E. coli expressing Cdc13p. TG22 mixed unlabeled competitor DNA, extract added, samples examined by <br /> electrophoresis. The % binding activity determined described Fig. 2. In space, portion gel shown. <br /> (A) The competitor DNAs unlabeled 42-base ss oligonucleotides having TG22 39 end (39 Tel), middle oligonucleotide <br /> (Internal Tel), 59 end (59 Tel). (B) The competitor DNA unlabeled TG22, ss pTG270, circular 6300-base ss phagemid containing <br /> 276 bases ss TG1&#226;&#8364;&#8220;3 DNA, ss pTG270 sheared average size 1000 bases, TG43, oligonucleotide having 43 bases ss TG1&#226;&#8364;&#8220;3 DNA. <br />  <br /> that kills cells expressing Ura3p (30), strain URA3           controls. The C1&#226;&#8364;&#8220;3A strand telomeric DNA, the <br /> next telomere. Because CDC13 essential, TPE                  strand URA3 transcribed, degraded in <br /> be measured cdc13D strain. The available conditional              cdc13-1 cells arrested growth restrictive temperatures <br /> allele CDC13 cdc13-1. We did detect defects TPE <br /> in cdc13-1 strain. However, maximum permissive tem- <br /> perature cdc13-1 cells grow 258C (25), these <br /> cells accumulate suppressor mutations permissive temper- <br /> atures. Moreover, individual colonies wild-type and <br /> mutant strains yielded widely varying frequencies TPE when <br /> assayed 258C (Fig. 5). Finally, isogenic wild-type strain <br /> had low TPE, &apos;0.1% cells FOAR <br /> at 258C, wild-type strains typically &apos;30- <br /> 50% FOAR cells analyzed way (8). As an <br /> alternative studying TPE cdc13-1 strain, studied TPE <br /> in cdc13D strain carrying pTHA-cdc13-1. <br />    The cdc13-1 gene cloned high-fidelity PCR (see <br /> Materials Methods) DNA cdc13-1 strain, <br /> inserted pTHA, introduced cdc13D strain with <br /> URA3 telomere chromosome VII-L. The cdc13D cells <br /> carrying pTHA-cdc13-1 grew strain <br /> carrying pTHA-CDC13 258C 308C, slower <br /> growth higher temperatures (data shown). At 378C, <br /> cdc13DypTHA-cdc13-1 took 6&#226;&#8364;&#8220;7 days form robust colony <br /> on complete medium lacking leucine, strain <br /> carrying pTHA-CDC13 took 3 days. The ability cdc13Dy <br /> pTHA-cdc13-1 cells grow higher temperatures than <br /> cdc13-1 cells presumably overexpression the <br /> mutant protein. <br />    The fraction FOAR colonies cdc13DypTHA-cdc13-1 <br /> and control strains, cdc13DypTHA-CDC13, CDC13y <br /> pTHA, CDC13ypTHA-CDC13, determined 258C,                            FIG. 5. TPE reduced high temperature cells overexpressing <br /> 308C, 378C (Fig. 5). At 258C 308C, cdc13Dy                     Cdc13-1p. CDC13 cdc13D cells URA3 telomere of <br />                                                                             chromosome VII-L carrying plasmids indicated bars <br /> pTHA-cdc13-1 produced similar fraction FOAR colonies <br />                                                                             spread plates containing lacking FOA incubated the <br /> as controls, although, noted previously, values            indicated temperatures. The bar median value the <br /> 258C cells varied considerably individual colonies.                   fraction FOAR cells, dots indicating values independent <br /> However, 378C, fraction FOAR cells cdc13Dy                    colonies. Independent colonies identical values represented <br /> pTHA-cdc13-1 1024, &apos;1000-fold                   single dot. <br />           Biochemistry: Lin Zakian                                                 Proc. Natl. Acad. Sci. USA 93 (1996)           13765 <br />  <br /> (25). However, loss TPE seen cells grown semi-         help cell distinguish intact damaged DNA (7). <br /> permissive temperatures attributed degradation         Single-stranded DNA proposed signal to <br /> of telomere-linked URA3 gene cells             RAD9 checkpoint responds (25). Despite likely <br /> URA3 degraded die, experience cell cycle arrest, yield   presence constitutive ss tail, telomeres detected as <br /> FOAR colonies, loss TPE generates FOA-sensitive          damaged DNA (7). We speculate Cdc13p masks ss <br /> colonies. However, C-strand degradation explain        TG1&#226;&#8364;&#8220;3 tails RAD9 pathway. This interpretation <br /> 36% cdc13DypTHA-cdc13-1 produced colonies 378C             suggests loss Cdc13p trigger RAD9-mediated <br /> complete medium lacking leucine.                                    arrest ways, unmasking short, constitutive ss <br />                                                                     TG1&#226;&#8364;&#8220;3 tails allowing extensive degradation C-strand <br />                        DISCUSSION                                   DNA telomere replication. <br /> These experiments demonstrate essential Saccharo-            We thank B. Garvik L. Hartwell&#226;&#8364;&#8482;s laboratory strains, <br /> myces protein Cdc13p binds ss TG1&#226;&#8364;&#8220;3 DNA vitro (Fig. 1).          plasmids, advice, C. Lin R. Reeder&#226;&#8364;&#8482;s laboratory for <br /> There potential telomeric substrates Cdc13p.            pTHA. We thank members laboratory sequence <br /> Throughout cell cycle, chromosomal telomeres        analysis CDC13 (A. Smith), help figures (J. Mangahas <br /> postulated short ss TG1&#226;&#8364;&#8220;3 tails. Cdc13p bound effi-         C. Lewis), careful reading manuscript. This <br /> ciently molecules 9 bp duplex DNA 10-base          work supported grants National Institutes Health. <br /> ss TG1&#226;&#8364;&#8220;3 tail (Fig. 3), substrate mimics expected <br /> structure telomeres cell cycle. In             1.   Zakian, V. A. (1995) Science 270, 1601&#226;&#8364;&#8220;1607. <br /> addition, conventional, semiconservative replication        2.   Shampay, J., Szostak, J. W. &amp; Blackburn, E. H. (1984) Nature <br /> telomeric DNA, yeast telomeres transiently acquire long ss                (London) 310, 154&#226;&#8364;&#8220;157. <br />                                                                      3.   Oka, Y., Shiota, S., Nakai, S., Nishida, Y. &amp; Okubo, S. (1980) <br /> TG1&#226;&#8364;&#8220;3 tails (6). Cdc13p binding 22-base ss TG1&#226;&#8364;&#8220;3 oligo- <br />                                                                           Gene 10, 301&#226;&#8364;&#8220;306. <br /> nucleotide competed efficiently 43-base ss TG1&#226;&#8364;&#8220;3            4.   Klobutcher, L. A., Swanton, M. T., Donini, P. &amp; Prescott, D. M. <br /> oligonucleotide circular phagemid containing 270                 (1981) Proc. Natl. Acad. Sci. USA 78, 3015&#226;&#8364;&#8220;3019. <br /> bases ss TG1&#226;&#8364;&#8220;3 DNA (Fig. 4B), suggesting Cdc13p          5.   Henderson, E. R. &amp; Blackburn, E. H. (1989) Mol. Cell. Biol. 9, <br /> also bind transient ss TG1&#226;&#8364;&#8220;3 tails generated telomere          345&#226;&#8364;&#8220;348. <br /> replication.                                                         6.   Wellinger, R. J., Wolf, A. J. &amp; Zakian, V. A. (1993) Cell 72, <br />    Although direct evidence Cdc1 <br /> </body></html>