<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Reduced U snRNP assembly causes <br /> motor axon degeneration animal <br /> model spinal muscular atrophy <br /> Christoph Winkler,2,5 Christian Eggert,1,5 Dietmar Gradl,3,5 Gunter Meister,4 Marieke Giegerich,2 <br /> Doris Wedlich,3 Bernhard Laggerbauer,1 Utz Fischer1,6 <br /> 1 <br />  Institute Biochemistry 2Institute Physiological Chemistry, Biocenter University W&#195;&#188;rzburg, D-97074 <br /> W&#195;&#188;rzburg, Germany; 3Institute Zoology II, University Karlsruhe, D-76131 Karlsruhe, Germany; 4Max-Planck-Institute <br /> of Biochemistry, D-82152 Martinsried, Germany <br />  <br />  <br /> Spinal muscular atrophy (SMA) motoneuron disease caused reduced levels <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span> motoneuron <br /> (SMN) protein. Previous studies assigned SMN uridine-rich small nuclear ribonucleoprotein particle (U <br /> snRNP) assembly, splicing, transcription, RNA localization. Here, used gene silencing assess <br /> the effect SMN protein deficiency U snRNP metabolism living cells organisms. In HeLa cells, we <br /> show reduction SMN levels SMA patients impairs U snRNP assembly. In line this, <br /> induced silencing SMN expression Xenopus laevis zebrafish arrested embryonic development. Under <br /> less severe knock-down conditions, zebrafish embryos proceeded development exhibited dramatic <br /> SMA-like motor axon degeneration. The observed silencing essential factors the <br /> U snRNP assembly pathway, Gemin2 pICln. Importantly, injection purified U snRNPs either <br /> SMN- Gemin2-deficient embryos Xenopus zebrafish prevented developmental arrest motoneuron <br /> degeneration, respectively. These findings suggest motoneuron degeneration SMA patients direct <br /> consequence impaired production U snRNPs. <br /> [Keywords: Survival motor neurons (SMN); U snRNP assembly; motoneuron; spinal muscular atrophy; <br /> zebrafish] <br /> Supplemental material available http://www.genesdev.org. <br /> Received March 1, 2005; revised version accepted July 27, 2005. <br />  <br />  <br />  <br />  <br /> Spinal muscular atrophy (SMA) hereditary neuro-                      nario, showing induced reduction SMN protein <br /> muscular disease caused reduced levels survival                 levels leads motoneuron defects (Hsieh-Li et al. 2000; <br /> motoneuron (SMN) protein (Lefebvre et al. 1995). In hu-                   Monani et al. 2000; Jablonka et al. 2002; McWhorter et <br /> mans, SMN protein encoded iden-                      al. 2003). Opposed SMN protein insufficiency, com- <br /> tical gene copies, SMN1 SMN2. In healthy                     plete lack SMN protein patients been <br /> affected individuals, small amounts functional                    reported and, homozygous inactivation SMN in <br /> SMN protein expressed SMN2, al-                       mice causes embryonic death, presumed be <br /> teration splice regulatory sequence exon 7 (Lorson                lethal humans. <br /> et al. 1999; Lorson Androphy 2000; Cartegni                          The tissue-specific degeneration cells caused by <br /> Krainer 2002; Kashima Manley 2003). The expres-                       SMN protein deficiency puzzling, given SMN is <br /> sion functional SMN protein largely depends                  expressed ubiquitously restricted moto- <br /> on SMN1. Most pathogenic mutations identified far                      neurons. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> solve paradox, experimental ap- <br /> map copy gene result reduced ex-                    proaches exploited assign SMN specific <br /> pression functional SMN protein. It widely ac-                     function. These studies revealed links SMN to <br /> cepted causes degeneration &#226;?&#163;-mo-                   assembly spliceosomal histone mRNA-pro- <br /> toneurons spinal cord muscular atrophy                     cessing uridine-rich small nuclear ribonucleoprotein par- <br /> symptomatic SMA. Indeed, recent studies carried                   ticles (U snRNPs) (Meister Fischer 2002; Pillai et al. <br /> in mice zebrafish elegantly recapitulated sce-                   2003; Gubitz et al. 2004), pre-mRNA splicing catalysis <br />                                                                           (Pellizzoni et al. 1998; Meister et al. 2000), axonal <br /> 5 <br />  These authors contributed equally work. <br />                                                                           transport (Jablonka et al. 2001; Rossoll et al. 2003; Zhang <br /> 6 <br />  Corresponding author.                                                    et al. 2003), transcription (Strasswimmer et al. <br /> E-MAIL utz.fischer@biozentrum.uni-wuerzburg.de; FAX 49-931-888-           1999). <br /> 4028. <br /> Article publication http://www.genesdev.org/cgi/doi/10.1101/      One function understood is <br /> gad.342005.                                                               SMN&#226;&#8364;&#8482;s contribution assembly spliceosomal <br />  <br />  <br /> 2320       GENES &amp; DEVELOPMENT 19:2320&#226;&#8364;&#8220;2330 &#194;&#169; 2005 Cold Spring Harbor Laboratory Press ISSN 0890-9369/05; www.genesdev.org <br />                                                                                          RNP assembly spinal muscular atrophy <br />  <br />  <br />  <br /> U snRNPs U1, U2, U4, U5. A key step forma-        2001a; Meister Fischer 2002; Pellizzoni et al. 2002), <br /> tion particles takes place cytosol,    focused site-specific inactivation SMN (Pell- <br /> seven so-called Sm-proteins common         izzoni et al. 1998; Grimmler et al. 2005). <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> these <br /> snRNPs associate U snRNA (U1, U2, U4, U5,          experiments, learned SMN complex essen- <br /> respectively) stable, ring-shaped structure (Kambach    tial U snRNP assembly. Nonetheless, unclear <br /> et al. 1999). The functional state U snRNP           U snRNP assembly likewise impaired in <br /> achieved additional, particle-specific, proteins   patients this, indeed, conferred moto- <br /> bound Sm core particle. Ultimately, cap struc-   neuron defects associated SMA <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. As <br /> tures U snRNAs undergo hypermethylation,          mentioned above, patients, suffering from <br /> the resulting U snRNPs imported nucleus,        severe form SMA (type I SMA), express SMN <br /> where function splicing (Will Luhrmann           extent, levels functional SMN protein <br /> 2001). Although formation Sm core occur       amounting 30%&#226;&#8364;&#8220;40% normal level (Lefebvre <br /> spontaneously vitro, shown     et al. 1997). Thus, asked extent of <br /> requires activity SMN test reca-     SMN protein reduction affected U snRNP assembly <br /> pitulates vivo conditions (Meister et al. 2001a; Meis-    all. One approach analyze U <br /> ter Fischer 2002; Pellizzoni et al. 2002). SMN oper-     snRNP assembly primary cells patients, using <br /> ates functional entity termed SMN complex,          assay described previously (Meister et al. <br /> whose major protein components grouped          2001a). As opposed HeLa cells Xenopus egg extract, <br /> Gemins 2&#226;&#8364;&#8220;7, factor termed unrip (Meister           however, did succeed preparing active extracts, <br /> Fischer 2002; Gubitz et al. 2004). Immunodepletion        cells patient origin unaffected <br /> cell extracts showed entity essential U     individuals. We decided elaborate experi- <br /> snRNP assembly. Furthermore, purified SMN complex            mental conditions HeLa cells reflect the <br /> alone shown sufficient transfer Sm      reduced SMN expression observed patients. <br /> proteins U snRNAs (Meister Fischer 2002; Pell-         As approach, tested tissue culture whether <br /> izzoni et al. 2002). Nonetheless, activity SMN    <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) suitable adjusting <br /> complex greatly enhanced interac-        SMN expression levels observed patients. Pri- <br /> tion second entity termed PRMT5 complex (Meis-        mary fibroblasts type I SMA patients used as <br /> ter Fischer 2002). In addition eponym, type   standard pathogenic conditions. Compared fi- <br /> II arginine methyltransferase PRMT5, complex con-       broblasts unaffected individuals, expression of <br /> tains Sm protein-binding factor pICln WD         endogenous SMN protein cells reduced to <br /> repeat protein WD45/MEP50 (Branscombe et al. 2001;           23% normal level (Fig. 1A [cf. lanes 1,2 3,4], B). <br /> Friesen et al. 2001; Meister et al. 2001b). The PRMT5        We mimicked situation HeLa cells using a <br /> complex symmetrically dimethylates arginines          22-nt-long duplex small interfering RNAs (siRNAs) <br /> Sm proteins B/B&#226;&#172;&#732;, D1, D3, enhancing their <br /> affinity SMN complex (Brahms et al. 2001; Frie- <br /> sen et al. 2001). In view this, presume that <br /> the interplay SMN PRMT5 complexes oc- <br /> curs coordinated, likely coregulated, steps, <br /> with SMN itself key player assembly <br /> line. Indeed, evidence regulation of <br /> SMN&#226;&#8364;&#8482;s activity kinase recognizes SMN <br /> and pICln (Grimmler et al. 2005). This recent study <br /> is indicating SMN indispensable for <br /> the formation U snRNPs vivo. <br />    Despite progress unraveling cellular func- <br /> tions SMN, role U snRNP assembly nor <br /> the additional functions assigned SMN (splicing, tran- <br /> scription, axonal transport) convincingly <br /> linked pathology SMA. We dedi- <br /> cated efforts aspect asked im- <br /> paired U snRNP assembly, reduced levels SMN,       Figure 1. Reduction SMN levels HeLa cells. (A) Immu- <br /> can elicit SMA-like symptoms animal test systems.         noblot analysis SMN fibroblasts healthy individual <br />                                                              (lanes 1,2) SMA type I patient (lanes 3,4), HeLa cells <br />                                                              subjected RNAi (lanes 5&#226;&#8364;&#8220;8). Twenty micrograms total pro- <br /> Results                                                      tein (lanes 1,3,5&#226;&#8364;&#8220;8) 5 &#194;&#181;g (lanes 2,4) analyzed SMN <br />                                                              protein content. For RNAi, extracts cells treated with <br /> Induced reduction SMN protein HeLa cells               siRNAs complementary SMN (lanes 6,8) nonspecific con- <br /> impairs U snRNP assembly                                     trol siRNAs (lanes 5,7) analyzed 48 h (lanes 5,6) or <br />                                                              72 h (lanes 7,8) transfection. Lower panels immu- <br /> Most studies carried far aimed depletion    nodetection actin blot. (B) Densitometric quanti- <br /> the SMN complex cell extracts (Meister et al.           fication relative SMN levels versus actin levels shown A. <br />  <br />  <br />                                                                                            GENES &amp; DEVELOPMENT             2321 <br />  Winkler et al. <br />  <br />  <br />  <br /> that corresponds positions 183&#226;&#8364;&#8220;205 human SMN            h RNAi treatment intensely inhibited per- <br /> mRNAs. As shown Figure 1A B, transfection           forming reaction (Fig. 2A, lanes 4,7). Both conditions <br /> HeLa cells duplex reduces SMN protein           did affect formation group bands (Fig. 2A, <br /> level &#226;&#710;&#188;60% 48 h, compared cells trans-      bands denoted R) contain U1-specific A protein <br /> fected control siRNA (Fig. 1A, lanes 5,6). After 72    lack Sm core domain (Meister Fischer 2002). <br /> h, expression SMN suppressed &#226;&#710;&#188;22% SMN            Importantly, inhibition assembly observed <br /> protein remaining (Fig. 1A, lanes 7,8). The transfection      extracts derived cells RNAi-induced reduc- <br /> efficiency experiments range 95%,      tion essential splicing factor hPrp4 (Fig. 2B,C). <br /> ruling possibility significant   This suggests observed assembly defect the <br /> extract derived untransfected cells (see Supple-     extract derived SMN-deficient cells specific. <br /> mentary Fig. 1). These findings showed amounts          The findings support idea of <br /> of SMN protein test adjusted          functional SMN protein cells bottleneck effi- <br /> pathological levels. Cells treated way proceed        cient U snRNP assembly. To validate in <br /> through indicatory defects RNAi time        vivo, designed pulse-chase experiments monitor U <br /> course, disappearance SMN nuclear         snRNP synthesis novo. HeLa cells treated 72 <br /> structures termed gems, slowed proliferation, even-       h SMN control siRNAs, followed 1-h pulse <br /> tually apoptosis (Supplementary Fig. 1A; data             [35S]methionine allow labeling nascent <br /> shown).                                                       proteins. Lysates cells subjected to <br />    Having RNAi tool hand, applied U          immunoprecipitation using antibody directed <br /> snRNP assembly assay extracts RNAi-treated            cap structure U snRNAs (antibody H20) <br /> HeLa cells. This assay based ability HeLa cell   (Bringmann Luhrmann 1986). An immunoprecipi- <br /> extract retard radiolabeled U1 snRNA native poly-     tate containing radiolabeled U snRNP proteins would <br /> acrylamide gel electrophoresis (Meister et al. 2001a).        indicatory novo production U <br /> Consistent previous data, extracts prepared         snRNPs. Indeed, comparison autoradiographs <br /> control cells efficiently promoted formation         Coomassie-stained U snRNP proteins (Fig. 2D, <br /> slowly migrating M complex (Fig. 2A, lane 2). We         lanes 1,3), identified specific bands immuno- <br /> previously shown complex contains fully         precipitate correspond seven Sm proteins and <br /> assembled Sm core domain snRNPs (Meister               U1 snRNP-specific proteins 70K, U1A, U1C (see <br /> Fischer 2002). Consistent this, anti-Sm protein      asterisks Fig. 2D; note additional U snRNP- <br /> antibody Y12 retards complex migration (Fig.        specific proteins visible extended exposure <br /> 2A, lane 5). Importantly, formation M complex      film). In contrast, weaker signals ob- <br /> was significantly reduced extracts cells     tained immunoprecipitation SMN-deficient <br /> been treated SMN siRNA duplex 48 h (Fig. 2A,     cell lysates (Fig. 2D, cf. lanes 4 3; upper <br /> lanes 3,6). Extracts prepared cells harvested 72   panel SMN protein deficiency). The intense <br />  <br /> Figure 2. Induced reduction SMN protein HeLa <br /> cells impairs assembly U snRNPs. (A) Assay for <br /> the assembly U1 snRNP vitro. [32P]-labeled U1 <br /> snRNA (lane 1) incubated RNAi-treated <br /> HeLa cell extracts described Figure 1 (lanes 2&#226;&#8364;&#8220;7) for <br /> 30 min, assembly monitored native gel elec- <br /> trophoresis described Meister et al. (2001a). (Lanes <br /> 5&#226;&#8364;&#8220;7) Supershifts antibody Y12 used iden- <br /> tify complexes contain Sm proteins. (B) The same <br /> assay described A performed extracts <br /> from cells treated control siRNAs (lanes 2,4) or <br /> siRNAs hPrp4 mRNA (lanes 3,5). (C) Immu- <br /> noblot analysis hPrp4 &#226;?&#164;-actin HeLa cell <br /> extracts used B. (D) Impaired synthesis U snRNPs <br /> de novo SMN protein reduction. HeLa cells trans- <br /> fected SMN siRNA (lane 4) control siRNA <br /> (lanes 2,3) pulsed [35S]cysteine [35S]me- <br /> thionine and, exchanging growth medium, ex- <br /> tracts prepared. Immunoprecipitations car- <br /> ried anti-RNA cap antibody H20 (lanes 3,4) or <br /> an unrelated antibody (lane 2), precipitated proteins <br /> were analyzed SDS-PAGE autoradiography. See <br /> lane 1 Coomassie-stained U snRNP proteins. Upper <br /> panels display immunoblot analyses SMN actin <br /> in extracts SMN siRNA-treated cells. The level of <br /> SMN protein quantified densitometry shown a <br /> percentage value relation actin. <br />  <br />  <br /> 2322       GENES &amp; DEVELOPMENT <br />                                                                                         RNP assembly spinal muscular atrophy <br />  <br />  <br />  <br /> band remaining nonspecific, appears the <br /> negative control (Fig. 2D, cf. lanes 4 2). Similar but <br /> less severe effects observed cells containing in- <br /> termediate levels SMN (Supplementary Fig. 2). Thus, <br /> data RNAi approaches, is, analysis of <br /> extracts vitro pulse-chase study vivo, pro- <br /> vide strong support insufficiency SMN protein <br /> impairs U snRNP assembly human cells. <br />  <br />  <br /> Purified U snRNPs rescue developmental arrest <br /> of SMN-deficient Xenopus laevis embryos <br /> The observed effects SMN deficiency U snRNP <br /> assembly encouraged ask disturbances in <br /> this pathway directly linked degeneration of <br /> cells. In case, able protect SMN- <br /> deficient cells degeneration exogenous <br /> supply U snRNPs. Most somatic cell lines available <br /> from SMA patients exhibit dramatic phenotypes <br /> and suitable approach. Therefore, we <br /> devised RNA-interference strategy developing X. <br /> laevis embryos. SiRNAs complementary X. laevis <br /> SMN mRNA injected blastomeres two- <br /> cell-stage embryos. The analysis proteins indi- <br /> vidual embryos Western blots (Fig. 3B) revealed sig- <br /> nificant reduction SMN protein (Fig. 3B, cf. lanes 1&#226;&#8364;&#8220;4 <br /> and 5&#226;&#8364;&#8220;8). This apparent gastrula stages <br /> (probably large maternal pool SMN), indi- <br /> cating injection siRNAs does impair the <br /> initial phase embryonic development. However,           Figure 3. SMN Gemin2 essential development <br /> large fraction embryos exhibited developmental arrest     Xenopus embryos. (A) SiRNAs complementary SMN or <br /> during gastrulation (typically 52% 80%, de-      Gemin2, nonspecific siRNAs injected X. laevis <br /> pending batch eggs), control embryos       embryos, survival scored gastrula (left column) or <br /> developed normally (Fig. 3A,C). At stage 11, control     neurula (middle right columns). Dead developmentally <br />                                                              arrested embryos display abnormal outgrowth white cell <br /> RNAi-injected embryos begun close blastoporus, <br />                                                              mass. (B) Western blot analysis SMN level embryos <br /> whereas SMN-deficient embryos failed finish gas- <br />                                                              treated control (lanes 1&#226;&#8364;&#8220;4) SMN-siRNAs (lanes 5&#226;&#8364;&#8220;8) in <br /> trulation. Instead, detached white cells surrounding     comparison actin levels. Analyses embryos received a <br /> blastoporus indicated massive developmental defects          coinjection U snRNPs (see legend Fig. 4 details) <br /> (Fig. 3A, column). In neurula stages, control-     shown lanes 9&#226;&#8364;&#8220;12. Each lane represents individual <br /> injected embryos formed neural tube, SMN-          embryo. Relative SMN expression levels estimated den- <br /> deficient embryos survived stage did     sitometry comparison actin levels. (C) Quantification of <br /> close blastoporus (Fig. 3A, second column). The     injection studies shown A. Results shown two <br /> phenotype observed SMN-interacting pro-         siRNA concentrations (2 0.2 &#194;&#181;M) different siRNAs <br /> tein Gemin2 targeted. In case, 95%         target. The numbers examined embryos (n) indicated <br />                                                              row. <br /> injected embryos did proceed develop- <br /> ment gastrulation (Fig. 3A,C). This effect was <br /> dose-dependent, 10-fold lower concentration       end products SMN-mediated assembly path- <br /> siRNAs (0.2 &#194;&#181;M) marginal effects develop-        way. In addition U1 snRNP-specific proteins 70K, <br /> ment, higher concentration (2 &#194;&#181;M) toxic        U1A, U1C, isolate contains subset spe- <br /> (Fig. 3C; data shown).                                   cific proteins U2, U4, U5 snRNPs (some of <br />    We asked supplementation purified       indicated Fig. 3A). As approach remove <br /> U snRNPs compensate loss SMN         contaminating SMN complex associated U sn- <br /> direct binding partner, Gemin2. For approach,        RNPs, particles passed anti-SMN im- <br /> used U snRNPs isolated HeLa        munoaffinity matrix injected the <br /> nuclear extract X. laevis cells (XTC) H20 anti-m3G     embryos. Indeed, SMN detected U <br /> immunoaffinity chromatography elevated ionic              snRNP preparation (Fig. 4A, lanes 6&#226;&#8364;&#8220;9). <br /> strength (420 mM KCl) (Bringmann Luhrmann                   Along siRNAs SMN Gemin2, 1 ng of <br /> 1986). This preparation contains Sm core particles    purified human U snRNPs embryo injected. <br /> U1, U2, U4, U5 snRNPs (see Fig. 4A Sm proteins       Given approximate volume embryo 0.5&#226;&#8364;&#8220;1 &#194;&#181;L, <br /> B/B&#226;&#172;&#732;, D1, D2, D3, E, F, G, U snRNAs) reflects    injected roughly corresponds concentra- <br />  <br />  <br />                                                                                           GENES &amp; DEVELOPMENT            2323 <br />  Winkler et al. <br />  <br />  <br />  <br /> tion 1&#226;&#8364;&#8220;2 nM. Strikingly, coinjection efficiently             second control experiment, RNA moiety U <br /> prevented developmental arrest observed embryos               snRNPs digested RNase T1 prior injection <br /> injected siRNAs (Fig. 4B; Fig. 3A                SMN-deficient embryos. As seen Figure 4B, in <br /> [third column], B, lanes 9&#226;&#8364;&#8220;12 knock-down efficiency).            case developmental arrest prevented <br /> Importantly, rescue dose-dependent injec-             (see Supplementary Fig. 3 efficiency RNase <br /> tion lower amounts (i.e., 0.1 ng 0.01 ng U snRNP)           treatment). Taken together, data indicate iso- <br /> had severe effects (Fig. 4C). In contrast, injection        lated intact U snRNPs supplied trans effi- <br /> of isolated U snRNPs se did alter develop-               ciently specifically bypass requirement SMN <br /> ment embryos treated control siRNA (Figs. 3A                 Gemin2 developing embryo. <br /> [first row], 4B). These data verified purified U <br /> snRNPs isolated Xenopus tissue culture (data not <br />                                                                      Gene silencing snRNP assembly pathway <br /> shown). <br />                                                                      causes motor axon degeneration zebrafish <br />    Two controls performed ensure specificity <br /> of rescue. First, siRNAs directed gene in-            The rescue SMN-deficient frog embryos U snRNPs <br /> volved unrelated cellular pathway used                 prompted ask SMA-like motoneuron de- <br /> conjunction U snRNP injections. The siRNAs ap-                  generation likewise linked U snRNP <br /> plied correspond mRNA sequences                   pathway. In case, impaired expression any <br /> endogenous inhibitors DNA replication termed Gemi-                essential factor sole function assembly U <br /> nin H Geminin L (McGarry 2002). Geminins                     snRNPs rise motoneuron phenotype. To <br /> unstable inhibitors DNA replication degraded             address issue, used recently described approach <br /> at metaphase/anaphase transition. In agreement                   study motor axon degeneration zebrafish <br /> with data McGarry (2002), injection siRNAs               (McWhorter et al. 2003). Consistent observa- <br /> against mRNA caused strong developmental ar-                tions X. laevis, authors study showed that <br /> rest gastrulation (Fig. 4B). The developmental defects            injection antisense Morpholinos SMN ze- <br /> elicited RNAi Geminins H L                   brafish embryos terminates development dose-de- <br /> qualitatively temporally equivalent silencing             pendent manner (McWhorter et al. 2003). The transpar- <br /> of SMN Gemin2 expression. In contrast SMN-                  ency embryos allowed close inspection <br /> Gemin2-deprived embryos, however, coinjection U                 neurons, revealing specific defects outgrowth and <br /> snRNPs did circumvent developmental arrest                pathfinding motoneurons. This particularly evi- <br /> embryos treated Geminin siRNAs (Fig. 4B). In                  dent lower Morpholino concentrations, sur- <br />  <br />  <br />  <br />  <br /> Figure 4. Isolated U snRNPs protect SMN- Gemin2-deficient Xenopus embryos developmental arrest. (A) Characterization <br /> of U snRNPs used injection experiments. U snRNP proteins resolved SDS-PAGE analyzed Coomassie <br /> staining (lane 1) silver staining (lanes 3,4). (Lanes 2,5) U snRNAs separated denaturing RNA-gel electrophoresis and <br /> visualized ethidium bromide staining. The U snRNPs passed anti-SMN column prior injection remove residual <br /> SMN protein. Immunoblot analysis SMN-depletion shown lanes 6 8; lanes 7 9 mock depletions. (B) X. <br /> laevis embryos treated injection control siRNA siRNAs directed SMN Gemin2, respectively, alone <br /> or purified U snRNPs. As control specificity rescue, U snRNPs coinjected embryos with <br /> siRNAs proteins function does relate U snRNP assembly (Geminins H L). As second control, embryos were <br /> coinjected digested U snRNPs. Quantification embryonic survival stage 13 injection siRNAs alone, siRNA together <br /> with U snRNPs, siRNA RNase T1-digested U snRNPs. (C) X. laevis embryos coinjected siRNAs against <br /> either SMN Gemin2 increasing amounts U snRNPs. Quantification survival rates shown. <br />  <br />  <br /> 2324       GENES &amp; DEVELOPMENT <br />                                                                                            RNP assembly spinal muscular atrophy <br />  <br />  <br />  <br /> vival rates embryos improved (McWhorter et al. <br /> 2003). We adopted approach consistently <br /> observe abnormal morphologies motoneurons after <br /> the expression SMN knocked 16% <br /> of control level (Fig. 5C, cf. lanes 2 1). Immuno- <br /> histochemistry using znp1 antibody (Melancon et al. <br /> 1997) revealed abnormal outgrowth axons cau- <br /> dal primary (CaP) variable primary (VaP) motoneu- <br /> rons, leaving axons truncated errone- <br /> ously branched (Fig. 5 [cf. B D], A specifica- <br /> tion examined region). More 75% the <br /> SMN-deficient embryos developed phenotype 31 <br /> h post-fertilization (Fig. 5I; Supplementary Table 2), <br /> similar situation described earlier (McWhorter et <br /> al. 2003). Injection control Morpholino no <br /> significant effect axon <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> morphology (Fig. <br /> 5B,I; Supplementary Table 2). <br />    We asked similar phenotypes like- <br /> wise observed knocking U snRNP <br /> assembly factors. For studies chose (1) Gemin2, <br /> a major component SMN complex, (2) pICln, <br /> the Sm protein-binding factor PRMT5 complex <br /> (Meister et al. 2000, 2001b; Friesen et al. 2001). The ze- <br /> brafish orthologs human Gemin2 human pICln <br /> were identified NCBI database exhibit 68% <br /> identity (Gemin2) 67% identity (pICln), respectively. <br /> When Gemin2- pICln-Morpholino in- <br /> jected embryos adjusted sublethal doses, a <br /> phenotype similar SMN morphants ob- <br /> tained (see Fig. 5E&#226;&#8364;&#8220;I; Supplementary Tables 1, 2). <br /> Compared noninjected controls, embryos injected <br /> with sublethal dose SMN-, Gemin2-, pICln-Mor- <br /> pholinos showed slightly delayed development with <br /> shortened body axes kinked tails, de- <br /> veloped normally (see Supplementary Table 2). The myo- <br /> tome regions, representing targets motoneuronal         Figure 5. Motoneuron defects zebrafish embryos re- <br /> innervation, formed normally embryos (regular         duced expression SMN, Gemin2, pICln. (A) Lateral view of <br /> myoD expression embryos injected Mo Ge-            zebrafish embryo 31 h post-fertilization (hpf). Motor axons <br /> min2 [n = 36] Mo pICln [n = 21]) (data shown).          stained znp1 antibody. A rectangle marks area <br /> Also, number motoneuronal cell bodies          midtrunk level contains 16 motor axons evalu- <br /> slightly decreased subset injected embryos (re-        ated analyzed embryos. (B,D,F,H) Lateral views trunk <br /> duced number isl2-positive cells 20 em-       regions showing znp1 positive motor axons noninjected (B) <br /> bryos injected Gemin2 Mo 21          SMN (D), Gemin2 (F), pICln (H) Morpholino-injected <br />                                                                embryos. Arrowheads indicate truncated branched axons. In- <br /> pICln). The remaining expression morphants Ge- <br />                                                                set F shows truncated axon embryo different that <br /> min2 pICln 9% 11%, respectively, control          shown F. (C,E,G) Western blot analysis extracts ze- <br /> levels (Fig. 5E,G). These observations indicate re-   brafish treated Morpholino oligonucleotides SMN <br /> duced expression factors proven role    (C, lane 2), Gemin2 (E, lane 2), pICln (G, lane 2). Extracts from <br /> biogenesis U snRNPs causes motor axon degeneration.         noninjected control embryos shown lane 1. Actin <br />    The results shown encouraged analyze            detected control equal loading relative quantifi- <br /> whether motor axon degeneration prevented             cation respective target protein level. (I) Quantification of <br /> upon coinjection isolated U snRNPs. To test pos-       motor axon defects zebrafish embryos knock-down of <br /> sibility, SMN-Morpholino injected ze-             SMN, Gemin2, pICln. Embryos bifurcated more <br /> brafish embryos presence puri-       truncated motor axon scored defective. <br /> fied U snRNPs. As observed previously, embryos treated <br /> with SMN-Morpholino displayed reduced expres- <br /> sion SMN (Fig. 6A, lane 2) erroneously truncated        lates HeLa cells Xenopus equally suitable <br /> and branched motoneurons (Fig. 6D). Strikingly,          rescue (Fig. 6, cf. E F; Fig. 4A U <br /> motor axon defects efficiently prevented coin-         snRNP composition). The motoneurons animals that <br /> jected U snRNPs (Fig. 6E,F). Due high degree         received U snRNP injection embryonic stage <br /> evolutionary conservation Sm proteins, U snRNP iso-         developed normally resembled control group in- <br />  <br />  <br />                                                                                              GENES &amp; DEVELOPMENT              2325 <br />  Winkler et al. <br />  <br />  <br />  <br />                                                                    fects reduced 80% (n = 25) Gemin2 Morpho- <br />                                                                    lino-injected embryos 50% (n = 24) embryos simul- <br />                                                                    taneously coinjected Gemin2 Morpholino hu- <br />                                                                    man U snRNPs. Thus, SMN- Gemin2-deficient <br />                                                                    zebrafish develop normal motoneurons, provided U <br />                                                                    snRNPs supplied trans. The animal model for <br />                                                                    SMA applied provides evidence vivo that <br />                                                                    interference RNP assembly pathway causes <br />                                                                    motor axon degeneration. <br />  <br />  <br />                                                                    Discussion <br />                                                                    Although genetics spinal muscular atrophy are <br />                                                                    known, search causal links cellular <br />                                                                    processes disease beginning. One <br />                                                                    obstacle endeavor confusing variety func- <br />                                                                    tions assigned SMN protein. Here, focused on <br />                                                                    well-understood activity SMN analyzing, live <br />                                                                    organisms, reduction SMN protein im- <br />                                                                    pairs U snRNP assembly elicits motor axon <br />                                                                    degeneration. The indications SMN levels af- <br />                                                                    fect formation U snRNP vivo were: (1) low <br />                                                                    efficiency assembly extracts cells SMN <br />                                                                    content reduced RNAi pathological lev- <br />  <br /> Figure 6. Coinjection U snRNPs SMN Morpholinos pre- <br /> vents motor axon defects zebrafish embryos. (A) Western blot <br /> analysis SMN levels noninjected control embryos (lanes <br /> 1,4) embryos injected SMN Morpholino (lane 2) or <br /> SMN Morpholino combination isolated U snRNPs from <br /> X. laevis (lane 3) HeLa cells (lane 5; Fig. 4A for <br /> Coomassie-stained proteins). Actin levels served loading con- <br /> trol relative densitometric quantification levels SMN <br /> levels. (B) Quantification motoneuron defects SMN- <br /> deficient zebrafish embryos treated with- <br /> out exogenous U snRNPs. (C&#226;&#8364;&#8220;F) Lateral views zebrafish trunk <br /> regions showing znp1-positive motor axons immunostain- <br /> ing. Motor axon phenotypes control Morpholino-injected em- <br /> bryos (C), SMN-deficient embryos (D), embryos coinjected <br /> with SMN Morpholino U snRNPs X. laevis (E) and <br /> HeLa cells (F). Arrowheads indicate truncated branched <br /> axons SMN Morpholino-injected embryos D. <br />  <br />  <br /> jected standard control Morpholino (Fig. 6, cf. E,F <br /> and C). We evaluated defects caudal primary mo- <br /> toneurons (CaP) statistically relevant number ani- <br /> mals (Fig. 6B) 76.9% motoneurons in <br /> SMN morphants defective, treatment                Figure 7. Coinjection U snRNPs Gemin2 Morpholinos <br /> U snRNPs reduces number control levels (13%&#226;&#8364;&#8220;               prevents motor axon defects zebrafish embryos. (A) Western <br /> 31%). Since treatment U snRNPs does signifi-              blot analysis Gemin2 levels embryos injected no <br /> cantly elevate level SMN protein morphants               Morpholino (lane 1), Gemin2 Morpholino (lane 2), Gemin2 <br /> (Fig. 6A, cf. lanes 2 3,5), consider unlikely       Morpholino combination isolated human U snRNPs. <br /> this effect mediated potential trace amounts             Actin levels served loading control relative densito- <br />                                                                    metric quantification levels Gemin2 levels. (B) Quantifica- <br /> SMN U snRNP preparation. Finally, analyzed <br />                                                                    tion motor axon defects injected zebrafish embryos. <br /> whether similar strategy rescue Gemin2- <br />                                                                    (C&#226;&#8364;&#8220;F) Lateral views zebrafish trunk regions showing znp1- <br /> deficient zebrafish embryos. Indeed, complementation               positive motor axons immunostaining. Motor axon phe- <br /> with human U snRNPs rescued motor axonal defects                   notypes control embryos (C), Gemin2-deficient embryos (D), <br /> (Fig. 7, cf. D,E C). This rescue, however, consid-         embryos coinjected Gemin2 Morpholino U <br /> erably efficient observed SMN (cf. Figs. 6B          snRNPs. Arrowheads indicate truncated branched axons in <br /> and 7B). The number embryos motor axonal de-               Gemin2 Morpholino-injected embryos. <br />  <br />  <br /> 2326       GENES &amp; DEVELOPMENT <br />                                                                                        RNP assembly spinal muscular atrophy <br />  <br />  <br />  <br /> els, (2) impaired synthesis U snRNPs novo        more, U snRNPs digested RNase T1 <br /> these cells. Encouraged observation, pro-          failed rescue SMN-deficiency X. laevis embryos. <br /> ceeded animal models showed induced            Therefore, consider possibility unlikely this <br /> SMN protein deficiency lethal embryos X. lae-       rescue conferred potential impurities U <br /> vis zebrafish. When reduction SMN zebrafish      snRNP preparation. <br /> was stringent, survival rates improved SMA-          The ability U snRNPs prevent degeneration <br /> like degeneration motor axons came light                SMN- Gemin2-deficient motoneurons brings an <br /> (McWhorter et al. 2003; study).                          unanswered question mind, is: Why does <br />    Although findings suggest reduced           reduced expression ubiquitous proteins (SMN and <br /> snRNP assembly cause SMA, correlation            Gemin2) cause tissue-specific phenotype? The second <br /> would speculative direct correlation         question add is: Why U snRNPs, essen- <br /> motoneuron defects U snRNP insufficiency         tial life cell, compensate phenotype? <br /> established. We reasoned data need      In pursuit answer, worth discussing another <br /> be accordance postulates: First, reduced      disease related RNA metabolism, termed Retinitis <br /> expression U snRNP assembly factor        pigmentosa (RP). Like SMA, RP characterized the <br /> likewise lead degeneration motor axons. Sec-        tissue-specific degeneration cells, case retinal <br /> ond, developmental defects observed pre-        photoreceptors. Among loci linked RP, recent <br /> ventable supplementation end product         studies identified mutations integral U <br /> this pathway; is, isolated U snRNPs.                     snRNP components (for review, Faustino Cooper <br />    To test insufficient supply U         2003). Presuming mutated proteins inactive, <br /> snRNP assembly factor elicits SMA-like symptoms,           opposed gain function, resulting haploin- <br /> knocked zebrafish expression pICln          sufficiency strikingly similar SMA. What <br /> Gemin2. Indeed, animals showed errone-         makes comparison SMA RP remark- <br /> ous axonal outgrowth seen SMN morphants.            able fact case affected tissue ex- <br /> Neither Gemin2 pICln far directly linked      hibits tremendous turnover specific mRNAs. While <br /> to functions SMN axonal transport,           development homeostasis photoreceptors require <br /> splicing catalysis, transcription. In case axonal   large quantities opsin (Korenbrot Fernald 1989), <br /> transport, participation Gemin2 ruled       axonal pathfinding motoneurons requires plethora of <br /> (Jablonka et al. 2001). However, worth note     receptors neurotrophins (for reviews, Chisholm <br /> Gemin2, like SMN, cytosolic          Tessier-Lavigne 1999; Tear 1999; Beattie 2000). It is <br /> complex mediates U snRNP assembly,         easy envisage cells meet demand for <br /> nuclear complex. We recently provided evidence           proteins splicing encoding pre-mRNAs is <br /> that complex intermediate assembly          inefficient. Faustino Cooper (2003) recently raised <br /> pathway <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> different function              hypothesis majority ubiquitously ex- <br /> (Grimmler et al. 2005). The functional interplay      pressed pre-mRNAs rapidly spliced tissue- <br /> SMN Gemin2 restricted for-        specific pre-mRNAs. Thinking line, tissue- <br /> mation U snRNPs. As pICln, initial identi-       specific pre-mRNA containing suboptimal splice ele- <br /> fication modulator chloride conductance (Krapi-       ments sensitive low levels of <br /> vinsky et al. 1994) awaits validation,       splicing factors others. The reduction proteins <br /> cellular pathway appears common            involved biogenesis U snRNPs (as SMA) of <br /> SMN U snRNP assembly (Pu et al. 1999; Friesen et al.       U snRNP components (as RP) would, cases, <br /> 2001; Meister et al. 2001b; Meister Fischer 2002).        evoke specific phenotype. <br /> Thus, silencing expression genes encoding Ge-             Alternatively, inefficient U snRNP assembly only <br /> min2 pICln presumed specifically affect U        weaken motoneuron disturbance an- <br /> snRNP assembly pathway. In view consider-            SMN-dependent reaction impair cell cu- <br /> ations, believe reduction SMN leads         mulatively. SMN&#226;&#8364;&#8482;s proposed activity axonal transport <br /> impaired U snRNP assembly causes motor            (Jablonka et al. 2001; Rossoll et al. 2003; Zhang et al. <br /> axon defects zebrafish embryos. This conclusion         2003) function, SMN-deficient <br /> further supported genetic studies Drosophila me-        motoneurons exhibit severe defects pathfinding <br /> lanogaster, showing deficiency U snRNA           (McWhorter et al. 2003; study), process inti- <br /> export factor PHAX induces neuromuscular disorder           mately depends efficient transport (Steward 2002). A <br /> (G. Matera, pers. comm.).                                     detailed evaluation SMN&#226;&#8364;&#8482;s activities con- <br />    Our second probably striking support          text SMN-deficient cells required address <br /> the correlation motor axon degeneration U         properly. <br /> snRNP assembly finding purified U snRNPs            An intriguing question comes motoneuron <br /> injected Xenopus zebrafish embryos compen-        rescue U snRNPs is, time develop- <br /> sate loss SMN Gemin2. U snRNPs rescued         ment SMN-deficiency compensated? Zebrafish <br /> only SMN- Gemin2-deprived embryos                 motoneurons known pass checkpoints their <br /> knock-downs directed proteins function       axon projects tissue (for review, Beattie <br /> (the regulators replication, Geminin H L). Further-     2000). These checkpoints occur spatial temporal <br />  <br />  <br />                                                                                         GENES &amp; DEVELOPMENT             2327 <br />  Winkler et al. <br />  <br />  <br />  <br /> order involve series cytokines receptor                2000), MANSMA 7 MANSMA 21 zebrafish SMN (a kind <br /> proteins presented growth cone axon             gift G. Morris, Robert Jones Agnes Hunt Orthopaedic <br /> (for reviews, Tessier-Lavigne Goodman 1996;                 Hospital, Oswestry, UK), affinity-purified rabbit polyclonal <br /> Chisholm Tessier-Lavigne 1999). The types                antibodies Xenopus SMN, human pICln human <br />                                                                     Gemin2, respectively. <br /> &#226;?&#163;-motoneurons sensitive reduction <br /> of SMN, caudal primary (CaP) variable primary neu-              Transcription U snRNAs vitro reconstitution <br /> rons (VaP), develop zebrafish         U snRNPs <br /> embryo. Their fate determined end gastrula- <br /> tion, which, transformed experimental setup, be-             For U snRNP assembly reactions vitro, U1 snRNA co- <br />                                                                     transcriptionally labeled using [&#226;?&#163;-32P]UTP (3000 Ci/mmol; Am- <br /> gins hours coinjection anti- <br />                                                                     ersham) gel-purified described previously (Meister et al. <br /> SMN-Morpholino U snRNPs. Therefore, albeit <br />                                                                     2000). In standard reconstitution assay, 25 fmol [32P]-labeled <br /> speculative, favor idea short time           U snRNA incubated 10 &#194;&#181;L HeLa cell extract (ad- <br /> frame U snRNP levels critical de-               justed PBS 2.5 mg total protein/mL) presence of <br /> velopment. The work presented raises                  2.5 pmol unlabeled tRNA 30 min 37&#194;&#176;C. For supershift <br /> question robust compensation SMN-de-                analyses complex M, Y12 antibody (2 &#194;&#181;g ascite fluid) was <br /> ficient phenotype U snRNPs stays time. To              added incubation continued 10 min. Samples <br /> end, know rescued SMN morphants survive                mixed 1 volume sample buffer (16% glycerol, 10 <br /> beyond 31 h post-injection, extends event              mg/mL Heparin, 0.025% bromphenol blue, 0.025% xylene cya- <br /> of hatching. Future studies way assess             nole) analyzed native RNA gel-electrophoresis de- <br />                                                                     scribed previously (Meister et al. 2001a). <br /> persistence rescue zebrafish ontogen- <br /> esis. <br />                                                                     Immunoprecipitations <br />                                                                     Antibodies covalently bound protein G-Sepharose beads <br /> Materials methods                                               (Amersham Biosciences) using dimethylpimelimidate (PIERCE) <br />                                                                     according manufacturer&#226;&#8364;&#8482;s protocol. Immunoprecipita- <br />                                                                     tions carried incubating extracts immobi- <br /> Cell culture, RNAi, pulse-chase analysis <br />                                                                     lized antibodies (15 &#194;&#181;g antibody/20 &#194;&#181;L bead volume) 1 h at <br /> HeLa cells human fibroblasts (Jablonka et al. 2001)        4&#194;&#176;C. Beads washed times PBS (pH 7.5)/0.01% <br /> cultured complete DMEM (Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s me-          NP-40 proteins extracted phenol, precipitated <br /> dium containing glucose L-glutamin [PAA laboratories],          trichloroacetate, analyzed 12% SDS-PAGE. H20 an- <br /> 10% fetal calf serum [FCS; PAA laboratories], penicillin [100       tibody directed RNA cap structure (a kind gift from <br /> U/mL; Invitrogen], streptomycin (0.1 mg/mL; Invitrogen]).       R. L&#195;&#188;hrmann, Max-Planck-Institute Biophysical Chemistry, <br /> For RNAi experiments, 22-nt-long RNA duplex corresponding         Gottingen, Germany) used positive control, M5 <br /> to nucleotides 183&#226;&#8364;&#8220;205 SMN cDNA generated. Cells         &#226;?&#163;-Flag antibody (Sigma) served negative control. Isolations of <br /> were transfected siRNAs using Oligofectamine reagent           U snRNPs HeLa XTC cells performed as <br /> (QIAGEN) according manufacturer&#226;&#8364;&#8482;s protocol, incu-        described (Bringmann Luhrmann 1986). For di- <br /> bated 37&#194;&#176;C time indicated. For pulse-chase studies,      gestion human U snRNPs, preparations incubated <br /> cells washed twice PBS 72 h transfection        overnight 5 &#195;&#8212; 10&#226;&#710;&#8217;4 U/&#194;&#181;L RNase T1. <br /> incubated 1 h starvation medium (GIBCO-BRL) contain- <br /> ing 10% FCS, 2 mM L-alanyl-L-glutamine, 25 mM HEPES-            Injection X. laevis embryos <br /> KOH (pH 7.4). Then, [35S]methionine/[35S]cysteine (ProMix; <br />                                                                     Eggs X. laevis fertilized vitro cultivated as <br /> Amersham-Biosciences) added 45 min. Cells were <br />                                                                     described (Kuhl et al. 1996). At two-cell stage, siRNAs (final <br /> washed PBS incubated 1 h complete <br />                                                                     concentration 0.2 &#194;&#181;M 2 &#194;&#181;M, respectively) injected into <br /> DMEM. <br />                                                                     cells, combination affinity-purified <br />                                                                     U snRNPs (final concentration 0.5&#226;&#8364;&#8220;1 &#194;&#181;M; generously provided <br /> Preparation cellular extracts immunoblotting                 R. L&#195;&#188;hrmann). Embryos grown stage 13 1/10 <br />                                                                     MBSH [8.8 mM NaCl, 0.1 mM KCl, 0.24 mM NaHCO3, 0.08 <br /> For preparation extracts HeLa cells, cells were <br />                                                                     mM MgSO4, 0.03 mM Ca(NO3)2, 0.04 mM CaCl2, 1 mM <br /> trypsinized, washed PBS, resuspended lysis buffer (150 <br />                                                                     HEPES pH 7.4] 16&#194;&#176;C scored survival. Sequences of <br /> mM NaCl, 1 mM EDTA, 10 mM HEPES &#194;&#183; KOH pH 7.6, 1.5 <br />                                                                     siRNAs follows (sense strands listed only): Xl-SMN #1 <br /> mM DTT, 0.5% NP-40, 0.5 U/&#194;&#181;L RNAsin [Promega], Complete <br />                                                                     (5&#226;&#172;&#732;-GCAUAUGAUAAAGCAGUGUdTdT-3&#226;&#172;&#732;); Xl-SMN #2 (5&#226;&#172;&#732;- <br /> protease inhibitor [Roche]), homogenized shearing with <br />                                                                     GAGAGCUCUGAAGAAUGAGdTdT-3&#226;&#172;&#732;); Xl-Gemin2 #1 (5&#226;&#172;&#732;- <br /> a 26G needle. The lysate centrifuged 30 min 16,000g, <br />                                                                     GCAGCACGUUGUCCUGAUGdTdT-3&#226;&#172;&#732;); Xl-Gemin2 #2 (5&#226;&#172;&#732;-G <br /> and supernatant used follow-up experiments. Ex- <br />                                                                     UUGCGAAAGAAACAGACCdTdT-3&#226;&#172;&#732;); Xl-GemininH #1 (5&#226;&#172;&#732;- <br /> tracts Xenopus embryos boiling the <br />                                                                     UACCAACAAGAAGCAGAGAdTdT-3&#226;&#172;&#732;); Xl-GemininH #2 (5&#226;&#172;&#732;- <br /> embryos SDS sample buffer. Zebrafish embryos frozen <br />                                                                     UCUCUGCUUCUUGUUGGUAdTdT-3&#226;&#172;&#732;); Xl-GemininL #1 <br /> in liquid nitrogen, resuspended lysis buffer, sheared with <br />                                                                     (5&#226;&#172;&#732;-TCTGCATCTGGGTGCCTTGdTdT-3&#226;&#172;&#732;); Xl-GemininL <br /> a 26G needle. The extract separated insoluble matter <br />                                                                     #2 (5&#226;&#172;&#732;-CAAGGCACCCAGAUGCAGAdTdT-3&#226;&#172;&#732;). <br /> by centrifugations 16,000g. The protein in <br /> each extract quantified Bradford colorimetry equal <br />                                                                     Silencing gene expression U snRNP injections <br /> loading SDS&#226;&#8364;&#8220;polyacrylamide gels verified immu- <br />                                                                     zebrafish <br /> noblotting using monoclonal &#226;?&#163;-actin antibody (Sigma). The an- <br /> tibodies used detection assembly factors 7B10       Zebrafish EST sequences encoding putative orthologs of <br /> mouse monoclonal antibody human SMN (Meister et al.             Gemin2 (accession nos. CN510554, AI793714) pICln <br />  <br />  <br /> 2328       GENES &amp; DEVELOPMENT <br />                                                                                                  RNP assembly spinal muscular atrophy <br />  <br />  <br /> (NM_131424, AAH66713) identified NCBI databases                 RNA 7: 1531&#226;&#8364;&#8220;1542. <br /> using human sequences queries. NCBI BLAST shows 68%               Branscombe, T.L., Frankel, A., Lee, J.H., Cook, J.R., Yang, Z., <br /> identity 83% similarity Gemin2 67% identity              Pestka, S., Clarke, S. 2001. PRMT5 (Janus kinase-bind- <br /> 78% similarity pICln, respectively. The ESTs contained full-        ing protein 1) catalyzes formation symmetric dimeth- <br /> length open reading frames 5&#226;&#172;&#732;-UTR sequences               ylarginine residues proteins. J. Biol. Chem. 276: 32971&#226;&#8364;&#8220; <br /> used designing Morpholino antisense oligonucleotides                32976. <br /> (Gene Tools). Zebrafish embryos injected one-         Bringmann, P. Luhrmann, R. 1986. Purification indi- <br /> two-cell stage &#226;&#710;&#188;0.5 nL following Morpholino anti-           vidual snRNPs U1, U2, U5 U4/U6 HeLa cells and <br /> sense oligonucleotide solutions: SMN (5&#226;&#172;&#732;-CGACATCTTCTG                   characterization protein constituents. EMBO J. 5: <br /> CACCATTGGC-3&#226;&#172;&#732;, 3.1 mg/mL); Gemin2 (5&#226;&#172;&#732;-TCAACTCCTC                        3509&#226;&#8364;&#8220;3516. <br /> GGCGTCTGACTTCAT-3&#226;&#172;&#732;, 6.3 mg/mL); pICln (5&#226;&#172;&#732;-GGTAA                      Cartegni, L. Krainer, A.R. 2002. Disruption SF2/ASF- <br /> ACTCTTCAACAGCACCATCC-3&#226;&#172;&#732;, 6.3 mg/mL), standard                       dependent exonic splicing enhancer SMN2 causes spinal <br /> control (5&#226;&#172;&#732;-CCTCTTACCTCAGTTACAATTTATA-3&#226;&#172;&#732;, 6.3 mg/                       muscular atrophy absence SMN1. Nat. Genet. 30: <br /> mL). For rescue experiments, SMN Gemin2 Morpholinos                 377&#226;&#8364;&#8220;384. <br /> (final concentration 3.1 6.3 mg/mL, respectively) coin-     Chisholm, A. Tessier-Lavigne, M. 1999. Conservation and <br /> jected purified Xenopus human U snRNPs (0.1 mg/mL;              divergence axon guidance mechanisms. Curr. Opin. Neu- <br /> provided R. L&#195;&#188;hrmann) described. Following injection,         robiol. 9: 603&#226;&#8364;&#8220;615. <br /> embryos reared 28&#194;&#176;C prim-15 stage (30&#226;&#8364;&#8220;31 h         Faustino, N.A. Cooper, T.A. 2003. Pre-mRNA splicing and <br /> post-fertilization) (see Kimmel et al. 1995) subjected      human disease. Genes &amp; Dev. 17: 419&#226;&#8364;&#8220;437. <br /> in situ hybridizations immunostainings.                          Friesen, W.J., Paushkin, S., Wyce, A., Massenet, S., Pesiridis, <br />                                                                         G.S., Van Duyne, G., Rappsilber, J., Mann, M., Dreyfuss, <br />                                                                         G. 2001. The methylosome, 20S complex containing JBP1 <br /> RNA situ hybridizations immunostainings <br />                                                                         pICln, produces dimethylarginine-modified Sm pro- <br /> Whole-mount RNA situ hybridizations islet2 expression            teins. Mol. Cell. Biol. 21: 8289&#226;&#8364;&#8220;8300. <br /> in motoneurons (Appel et al. 1995) myoD somites (Wein-        Grimmler, M., Bauer, L., Nousiainen, M., Korner, R., Meister, <br /> berg et al. 1996) performed described earlier (Winkler et       G., Fischer, U. 2005. Phosphorylation regulates ac- <br /> al. 2003). For whole-mount immunostainings, embryos                tivity SMN complex assembly spliceosomal <br /> fixed overnight 4% paraformaldehyde/PBS/0.1% Tween, de-              U snRNPs. EMBO Rep. 6: 70&#226;&#8364;&#8220;76. <br /> chorionated PBS/0.1% Tween, stored methanol. After         Gubitz, A.K., Feng, W., Dreyfuss, G. 2004. The SMN com- <br /> rehydration, embryos soaked water 1 h subse-            plex. Exp. Cell Res. 296: 51&#226;&#8364;&#8220;56. <br /> quently incubated PBDT blocking solution (1% DMSO, 1%             Hsieh-Li, H.M., Chang, J.G., Jong, Y.J., Wu, M.H., Wang, N.M., <br /> BSA, 0.5% Triton X-100, 2.5% goat serum PBS) 1 h. The            Tsai, C.H., Li, H. 2000. A mouse model spinal mus- <br /> samples incubated znp1 antibody (diluted             cular atrophy. Nat. Genet. 24: 66&#226;&#8364;&#8220;70. <br /> 1:100 PBDT) described Melancon et al. (1997) de-        Jablonka, S., Bandilla, M., Wiese, S., Buhler, D., Wirth, B., Sendt- <br /> tected using Vectastain Elite kit (Vector Laboratories)        ner, M., Fischer, U. 2001. Co-regulation survival of <br /> diaminobenzidine chromogenic substrate. The znp1 antibody            motor neuron (SMN) protein interactor SIP1 during <br /> was kindly provided Zebrafish International Resource             development spinal muscular atrophy. Hum. Mol. <br /> Center ZIRC (NIH #RR12546). Trunk regions stained em-              Genet. 10: 497&#226;&#8364;&#8220;505. <br /> bryos manually dissected remaining embryo us-          Jablonka, S., Holtmann, B., Meister, G., Bandilla, M., Rossoll, <br /> ing razor blades mounted flat 100% glycerol photog-          W., Fischer, U., Sendtner, M. 2002. Gene targeting of <br /> raphy.                                                                  Gemin2 mice reveals correlation defects the <br />                                                                         biogenesis U snRNPs motoneuron cell death. Proc. <br />                                                                         Natl. Acad. Sci. 99: 10126&#226;&#8364;&#8220;10131. <br /> Acknowledgments                                                      Kambach, C., Walke, S., Young, R., Avis, J.M., la Fortelle, E., <br />                                                                         Raker, V.A., Luhrmann, R., Li, J., Nagai, K. 1999. Crystal <br /> We indebted Reinhard L&#195;&#188;hrmann providing purified U <br />                                                                         structures Sm protein complexes implica- <br /> snRNPs H20 antibody Glenn Morris antibod- <br />                                                                         tions assembly spliceosomal snRNPs. Cell <br /> ies zebrafish SMN. We thank Greg Matera shar- <br />                                                                         96: 375&#226;&#8364;&#8220;387. <br /> ing unpublished data, Andreas Brandst&#195;&#164;tter Cordula <br />                                                                      Kashima, T. Manley, J.L. 2003. A negative element in <br /> Neuner help immunostainings, Ashwin Chari for <br />                                                                         SMN2 exon 7 inhibits splicing spinal muscular atrophy. <br /> critical readings manuscript. This work funded by <br />                                                                         Nat. Genet. 34: 460&#226;&#8364;&#8220;463. <br /> grants Deutsche Forschungsgemeinschaft (SFB581, <br />                                                                      Kimmel, C.B., Ballard, W.W., Kimmel, S.R., Ullmann, B., and <br /> GRK1048, FOR426) Families SMA. <br />                                                                         Schilling, T.F. 1995. Stages embryonic development the <br />                                                                         zebrafish. Dev. Dyn. 203: 253&#226;&#8364;&#8220;310. <br />                                                                      Korenbrot, J.I. Fernald, R.D. 1989. Circadian rhythm and <br /> References <br />                                                                         light regulate opsin mRNA rod photoreceptors. Nature <br /> Appel, B., Korzh, V., Glasgow, E., Thor, S., Edlund, T., Dawid,         337: 454&#226;&#8364;&#8220;457. <br />    I.B., Eisen, J.S. 1995. Motoneuron fate specification re-     Krapivinsky, G.B., Ackerman, M.J., Gordon, E.A., Krapivinsky, <br />    vealed patterned LIM homeobox gene expression em-              L.D., Clapham, D.E. 1994. Molecular characterization of <br />    bryonic zebrafish. Development 121: 4117&#226;&#8364;&#8220;4125.                       swelling-induced chloride conductance regulatory protein, <br /> Beattie, C.E. 2000. Control motor axon guidance ze-           pICln. Cell 76: 439&#226;&#8364;&#8220;448. <br />    brafish embryo. Brain Res. Bull. 53: 489&#226;&#8364;&#8220;500.                     Kuhl, M., Finnemann, S., Binder, O., Wedlich, D. 1996. <br /> Brahms, H., Meheus, L., Brabandere, V., Fischer, U.,             Dominant negative expression cytoplasmically deleted <br />    Luhrmann, R. 2001. Symmetrical dimethylation arginine             mutant XB/U-cadherin disturbs mesoderm migration dur- <br />    residues spliceosomal Sm protein B/B&#226;&#172;&#732; Sm-like             ing gastrulation Xenopus laevis. Mech. Dev. 54: 71&#226;&#8364;&#8220;82. <br />    protein LSm4, interaction SMN protein.         Lefebvre, S., Burglen, L., Reboullet, S., Clermont, O., Burlet, P., <br />  <br />  <br />                                                                                                    GENES &amp; DEVELOPMENT              2329 <br />  Winkler et al. <br />  <br />  <br />     Viollet, L., Benichou, B., Cruaud, C., Millasseau, P., Zeviani,       110: 537&#226;&#8364;&#8220;540. <br />     M., et al. 1995. Identification characterization spinal   Strasswimmer, J., Lorson, C.L., Breiding, D.E., Chen, J.J., Le, T., <br />     muscular atrophy-determining gene. Cell 80: 155&#226;&#8364;&#8220;165.                  Burghes, A.H., Androphy, E.J. 1999. Identification sur- <br /> Lefebvre, S., Burlet, P., Liu, Q., Bertrandy, S., Clermont, O.,           vival motor neuron transcriptional activator-binding <br />     Munnich, A., Dreyfuss, G., Melki, J. 1997. Correlation            protein. Hum. Mol. Genet. 8: 1219&#226;&#8364;&#8220;1226. <br />     severity SMN protein level spinal muscular          Tear, G. 1999. Neuronal guidance. A genetic perspective. <br />     atrophy. Nat. Genet. 16: 265&#226;&#8364;&#8220;269.                                     Trends Genet. 15: 113&#226;&#8364;&#8220;118. <br /> Lorson, C.L. Androphy, E.J. 2000. An exonic enhancer            Tessier-Lavigne, M. Goodman, C.S. 1996. The molecular <br />     required inclusion essential exon SMA-de-            biology axon guidance. Science 274: 1123&#226;&#8364;&#8220;1133. <br />     termining gene SMN. Hum. Mol. Genet. 9: 259&#226;&#8364;&#8220;265.                   Weinberg, E.S., Allende, M.L., Kelly, C.S., Abdelhamid, A., Mu- <br /> Lorson, C.L., Hahnen, E., Androphy, E.J., Wirth, B. 1999. A           rakami, T., Andermann, P., Doerre, O.G., Grunwald, D.J., <br />     single nucleotide SMN gene regulates splicing           Riggleman, B. 1996. Developmental regulation ze- <br />     responsible spinal muscular atrophy. Proc. Natl. Acad.            brafish MyoD wild-type, tail spadetail embryos. <br />     Sci. 96: 6307&#226;&#8364;&#8220;6311.                                                   Development 122: 271&#226;&#8364;&#8220;280. <br /> McGarry, T.J. 2002. Geminin deficiency causes Chk1-depen-            Will, C.L. Luhrmann, R. 2001. Spliceosomal UsnRNP bio- <br />     dent G2 arrest Xenopus. Mol. Biol. Cell 13: 3662&#226;&#8364;&#8220;3671.             genesis, structure function. Curr. Opin. Cell Biol. <br /> McWhorter, M.L., Monani, U.R., Burghes, A.H., Beattie,                13: 290&#226;&#8364;&#8220;301. <br />     C.E. 2003. Knockdown survival motor neuron (Smn)            Winkler, C., Schafer, M., Duschl, J., Schartl, M., Volff, J.N. <br />     protein zebrafish causes defects motor axon outgrowth           2003. Functional divergence zebrafish midkine <br />     pathfinding. J. Cell Biol. 162: 919&#226;&#8364;&#8220;931.                          growth factors following fish-specific gene duplication. Ge- <br /> Meister, G. Fischer, U. 2002. Assisted RNP assembly              nome Res. 13: 1067&#226;&#8364;&#8220;1081. <br />     purified components: SMN- pICln-complexes cooperate            Zhang, H.L., Pan, F., Hong, D., Shenoy, S.M., Singer, R.H., and <br />     formation spliceosomal U snRNPs. EMBO J.                    Bassell, G.J. 2003. Active transport survival motor <br />     21: 5853&#226;&#8364;&#8220;5863.                                                        neuron <br /> </body></html>