<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Functions of Fission Yeast Orp2 in DNA Replication and Checkpoint Control <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 orp2 is an essential gene of the fission yeast Schizosaccharomyces pombe with 22% identity <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> budding yeast <br />              ORC2. We isolated temperature-sensitive alleles of orp2 using a novel plasmid shuffle based on selection <br />              against thymidine kinase. Cells bearing the temperature-sensitive allele orp2-2 fail to complete DNA replica- <br />              tion at a restrictive temperature and undergo cell cycle arrest. Cell cycle arrest depends on the checkpoint <br />              genes rad1 and rad3. Even when checkpoint functions are wild type, the orp2-2 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>mutation</span> causes high rates <br />              of chromosome and plasmid loss. These phenotypes support the idea that Orp2 is a replication initiation <br />              factor. Selective spore germination allowed analysis of orp2 deletion mutants. These experiments showed <br />              that in the absence of orp2 function, cells proceed into mitosis despite a lack of DNA replication. This <br />              suggests either that the Orp2 protein is a part of the checkpoint machinery or more likely that DNA <br />              replication initiation is required to induce the replication checkpoint signal. <br />  <br />  <br />  <br /> E    UKARYOTIC DNA replication is initiated at many <br />       replication origins on each chromosome. In the <br /> budding yeast Saccharomyces cerevisiae, origin sequences <br />                                                                                Orp2 also binds to the replication activator Cdc18 <br />                                                                              (Leatherwood et al. 1996). Cdc18 is similar to the <br />                                                                              Cdc6 proteins of budding yeast, Xenopus, and humans <br /> are bound by the origin recognition complex (ORC),                           (Kelly et al. 1993). Cdc18/Cdc6 activity is cell cycle <br /> which is composed of six subunits, Orc1 through Orc6                         regulated, and this regulation is important for accurate <br /> (Bell and Stillman 1992). In other eukaryotes, the                           regulation of DNA replication. In fission yeast, overex- <br /> cis-acting sequences of replication origins are more com-                    pression of Cdc18 leads to DNA rereplication (Nishi- <br /> plex and less well understood (Huberman 1998). Never-                        tani and Nurse 1995). It appears that Cdc18 activity <br /> theless, ORC appears to be widely conserved. In the                          is regulated by Cdc2 kinase (Jallepalli et al. 1997; <br /> fission yeast Schizosaccharomyces pombe, genes similar in                    Lopez-Girona et al. 1998). One attractive model is that <br /> sequence to ORC1, ORC2, and ORC5 have been identi-                           Cdc18/Cdc6 interacts with ORC and then catalyzes asso- <br /> fied (Gavin et al. 1995; Muzi-Falconi and Kelly 1995;                        ciation of MCM replication factors with the chromatin <br /> Grallert and Nurse 1996; Leatherwood et al. 1996;                            (J. Leatherwood 1998). <br /> Ishiai et al. 1997).                                                           The cis-acting sequences used for replication origins <br />    The ORC2-like factor, Orp2 (Orc2 related in pombe), is                    in budding yeast include a binding site for ORC and <br /> particularly interesting because it interacts with proteins                  a binding site for a transcription activator; in higher <br /> required for coupling DNA replication to cell cycle pro-                     eukaryotes, the sequences that direct replication initia- <br /> gression. Orp2 was first identified because it binds to                      tion have remained elusive (reviewed in Rowley et al. <br /> the cell cycle kinase Cdc2 (Leatherwood et al. 1996).                        1994; Huberman 1998), and in fission yeast, the deter- <br /> In fission yeast, Cdc2 is the cyclin-dependent kinase                        minants of origin activity appear to be more complex <br /> governing both the G1 to S and the G2 to M transitions                       and possibly more redundant than those of budding <br /> (Nurse and Bisset 1981; Nurse 1990). Initiation of                           yeast (Wohlgemuth et al. 1994; Clyne and Kelly 1995; <br /> DNA replication requires an active complex of Cdc2                           Dubey et al. 1996). To gain a broader understanding <br /> and one of the B-type cyclins Cdc13, Cig1, or Cig2.                          of how replication initiates, it will be informative to <br /> (Fisher and Nurse 1996; Mondesert et al. 1996). The                          learn about the functions of the putative fission yeast <br /> same Cdc2/Cdc13 complex that can activate replication                        ORC complex in vivo. Toward this end, we isolated <br /> is also essential to prevent re-initiation of DNA replica-                   temperature-sensitive alleles of orp2. <br /> tion during G2 (Broek et al. 1991; Hayles et al. 1994).                        To isolate orp2 mutants, we developed a plasmid-shuf- <br /> The interaction of Cdc2 and Orp2 suggests that Cdc2                          fle system for fission yeast. Fission yeast, like other fungi, <br /> regulates DNA replication directly at replication origins                    lacks thymidine kinase. We used the thymidine kinase <br /> (Leatherwood et al. 1996).                                                   gene <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> herpes simplex virus as a new counterselectable <br />                                                                              marker by taking advantage of the fact that the pres- <br />                                                                              ence of thymidine kinase causes sensitivity to the thymi- <br />                                                                              dine analogue 5-fluoro-29-deoxyuridine (FUdR). With <br />   Corresponding author: Janet Leatherwood, Department of Molecular <br /> Genetics and Microbiology, Life Science, Rm. 130, SUNY, Stony                the flood of information from genome projects and the <br /> Brook, NY 11794-5222. E-mail: janet.leatherwood@sunysb.edu                   increasing number of genes identified by two-hybrid <br />  <br /> Genetics 154: 599&#226;&#8364;&#8220;607 (February 2000) <br />  600                                                          J. Kiely et al. <br />  <br />                                                                TABLE 1 <br />                                                            S. pombe strains <br />  <br />          Name                                       Genotype                                                   Source <br />          JLP46          h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 his7-366/his7-366             Leatherwood et al. (1996) <br />                           ura4-D18/ura4-D18 orp21/orp2D::ura41 <br />          JLP125         h2 ade6-M216 leu1-32 his71::padh-TK                                     This study <br />          JLP164         h1 ade6-M210 ura4-D18 leu1-32 his7-366 orp2D::ura41                     This study <br />                           porp2TKade <br />          JLP166         h2 ade6-M216 ura4-D18 leu1-32 his7-366 orp2D::ura41 porp2TKade          This   study <br />          JLP208         h2 orp2-2 ura4-D18 leu1-32                                              This   study <br />          JLP216         h2 orp2-7 ura4-D18 leu1-32                                              This   study <br />          JLP242         h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 his7-366/his7-366             This   study <br />                           ura4-D18/ura4-D18 orp2-2/orp2D::ura4 <br />          JLP247         h1 rad1-1 orp21 ade62 ura4-D18 leu1-32 his7-366                         This study <br />          JLP248         h1 rad1-1 orp2-2 ade62 ura4-D18 leu1-32                                 This study <br />          JLP249         h2 rad11 orp2-2 ade62 ura4-D18 leu1-32                                  This study <br />          JLP250         h2 rad11 orp21 ade62 ura4-D18 leu1-32 his7-366                          This study <br />          CHP429         h2 ade6-M216 leu1-32 his7-366                                           C. Hoffman <br />          PR109          h2 leu1-32 ura4-D18                                                     P. Russell <br />  <br />  <br />  <br /> and other interaction screens, there is an increasing                   (McLeod et al. 1987). The plasmid pJL217 was transformed <br /> need to engineer conditional mutations. The methods                     into S. pombe (strain PR109 h2 leu1-32 ura4-D18, P. Russell) by <br />                                                                         selection for leucine prototrophy; transformants were sensitive <br /> used here should be widely useful in fission yeast stud-                to 25 mm FUdR (Sigma, St. Louis, cat. F-0503) indicating <br /> ies. The approach is also applicable to budding yeast                   that functional thymidine kinase is expressed from the adh1 <br /> (S. B. Haase, unpublished data; Sclafani and Fang-                      promotor (data not shown). The his71, padh -TK plasmid <br /> man 1986), to Neurospora crassa, and other fungi (Gri-                  pJL218 was constructed by cloning the 3.0-kb EcoRI fragment <br /> vell and Jackson 1968). Expression of thymidine ki-                     containing padh -TK from pJL217 into EcoRI-digested his71 <br />                                                                         plasmid pEA13 (Apolinario et al. 1993). To integrate padh - <br /> nase in fission yeast also makes it possible to measure                 TK at the his7 locus, pJL218 was linearized within his71 by <br /> DNA replication by BrdU incorporation or by the use                     Cla1 and used to transform the his7-366 mutant strain CHP429 <br /> of tritiated thymidine (J. Leatherwood, unpublished                     to histidine prototrophy. The resulting allele is designated <br /> data). Such labeling of newly synthesized DNA is not                    his71::padh-TK (strain JLP125 h2 leu1-32 ade6-M216 his71:: <br /> possible in wild-type yeast that lack the thymidine salvage             padh -TK) and also results in FUdR sensitivity (Figure 2). <br />                                                                            The plasmid containing orp21, ade61, and padh -TK was con- <br /> pathway because they cannot use thymidine or BrdU as                    structed in three steps: (1) padh -TK on the 1.8-kb HindIII/ <br /> precursors for DNA synthesis.                                           SmaI fragment from pJL217 was cloned into HindIII/SmaI- <br />    Our previous studies showed that orp2 is essential and               digested pREP3 (Maundrell 1993) to create pJL240; (2) <br /> that the cells lacking orp2 died with phenotypes consis-                orp21 on the 2.8-kb HindIII fragment from pER27 (Leath- <br /> tent with a defect in DNA replication (Leatherwood                      erwood et al. 1996) was cloned into HindIII-digested pJL240 <br />                                                                         to create pJL243; and (3) ade61 on the 3.0-kb PvuII, SpeI <br /> et al. 1996). In this study we investigate S-phase progres-             fragment from pAS1 (Szankasi et al. 1988) was cloned into <br /> sion, chromosome stability, and replication-checkpoint                  pJL243 that had been digested with Nsi I, treated with T4 DNA <br /> control in orp2 mutants.                                                polymerase to remove the Nsi I overhang, purified, and then <br />                                                                         digested with SpeI. The resulting plasmid, pJL261, is referred <br />                                                                         to as porp2 TKade throughout the text and is shown in Fig- <br />                MATERIALS AND METHODS                                    ure 1. orp21 expression plasmid pJL207 has been described <br />                                                                         (Leatherwood et al. 1996). <br />    S. pombe strains and media: The genetic methods and media               orp2 mutagenesis and identification of conditional alleles: <br /> used for S. pombe were essentially as described by Moreno et            orp2 was amplified for 20 cycles (annealing temperature 678) <br /> al. (1991). YSO is rich media low in adenine (0.5% yeast                from plasmid pJL207 with primers oJL18 (nmt promotor se- <br /> extract, 0.2% casamino acids, 3% glucose; add 2% agar when              quence 59 TCT CAC TTT CTG ACT TAT AGT CGC T 39) <br /> making plates). Strains used in this study are listed in Table 1.       and oJL19 (nmt terminator sequence 59 CTA GCA GTA CTG <br />    Thymidine kinase constructs: The thymidine kinase gene               GCA AGG GAG AC 39) using mutagenic PCR conditions de- <br /> from herpes simplex virus (HSV1-TK; McKnight 1980) was                  scribed by Cadwell and Joyce (1992). Reactions contained <br /> cloned so as to be expressed from the adh1 promoter in S.               dNTPs and divalent cations specified in Table 2 and also <br /> pombe ; this is refered to as padh -TK. BamHI recognition sites         contained 1 mm dCTP, 1 mm dTTP, 20 mm Tris pH 8.4, 50 <br /> were added to the coding region of HSV1-TK by PCR amplifi-              mm KCl, 50 ng plasmid template, 1 mg each primer, and 5 <br /> cation using primers BTK59 CGGGATCCCGTATGGCTTCG                         units TAQ polymerase (GIBCO-BRL, Gaithersburg, MD) in <br /> TACC 39 and BTK39 CGGGATCCCGTGTTTCAGTTAGCC 39.                          final volume of 100 ml. PCR products digested with Bsi WI <br /> padh -TK expression plasmid pJL217 was constructed by clon-             and BamHI were cloned into Bsi WI/BamHI-digested pJL207. <br /> ing the 1.13-kb TK PCR product into the BamHI site 39 of                Approximately 1000 Escherichia coli transformants were pooled <br /> the adh1 promotor in fission yeast expression plasmid pART1             for each PCR reaction condition. Plasmid DNA from the <br />                                                      Mutants of Fission Yeast orp2                                               601 <br />  <br />                                                                      low adenine plates), and finally to rich media at 258 and 368 <br />                                                                      to screen for a temperature-sensitive <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span>. Ura2, Ade2, <br />                                                                      temperature-sensitive colonies were analyzed by Southern <br />                                                                      blotting to confirm that orp2D::ura4 was replaced. By this <br />                                                                      method the temperature-sensitive orp2-2 and orp2-7 alleles <br />                                                                      were isolated as stable integrants (allele numbers indicate the <br />                                                                      PCR pools from which the mutants were obtained). <br />                                                                         Selective spore germination and FACS analysis: Diploid <br />                                                                      strain JLP46 (h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 <br />                                                                      ura4-D18/ura4-D18 orp21/orp2D::ura4) has been described <br />                                                                      (Leatherwood et al. 1996). A haploid orp2D::ura4 derived from <br />                                                                      JLP46 was kept alive by orp2 expressed from plasmid pJL207 <br />                                                                      (Leatherwood et al. 1996). This haploid was mated with an <br />                                                                      orp2-2 temperature-sensitive strain to create a diploid from <br />                                                                      which an isolate lacking the pJL207 plasmid was obtained; <br />                                                                      this is JLP242 (h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 <br />                                                                      ura4-D18/ura4-D18 his7-366/his7-366 orp2-2/orp2D::ura4). Se- <br />                                                                      lective spore germination and FACS analysis was performed <br />   Figure 1.&#226;&#8364;&#8221;Map of shuttle vector porp2 TKade (plasmid               as previously described for JLP46; note that ungerminated <br /> pJL261).                                                             spores are removed from the final data presented by gating <br />                                                                      out small objects based on forward scatter and that assignment <br />                                                                      of the gate parameters was determined by analysis of a pure <br />                                                                      population of ungerminated spores (Leatherwood et al. <br /> pooled colonies was transformed into yeast strain JLP166 at <br />                                                                      1996). For the hydroxyurea (HU)-treated samples, 15 mm HU <br /> 258. A total of 200&#226;&#8364;&#8220;1200 yeast transformants were screened <br />                                                                      was added to the media at the beginning of the germination. <br /> for each pool by replica printing to YES and to YES 1 50 <br /> mm FUdR and growing at 258 and 368. The percentage of <br /> nonfunctional and temperature-sensitive orp2 alleles for each <br /> PCR pool is shown in Table 2. A total of 97 temperature-                                       RESULTS <br /> sensitive candidates were picked and retested by replica print- <br /> ing onto FUdR plates at 258, 328, 35.58, and 368. Plasmids were         Isolation of orp2 mutants: Fission yeast orp2 is essen- <br /> rescued from 16 candidates that gave the greatest temperature        tial; therefore recessive orp2 mutants were identified <br /> dependence for growth. Five of these plasmids were retrans-          using a plasmid shuffle. A haploid deleted for orp2 was <br /> formed into JLP166 and all had temperature-sensitive pheno-          kept alive by wild-type orp2 on a plasmid, which also <br /> types when retested.                                                 carried the gene for HSV thymidine kinase. Mutagen- <br />   Integration of orp2 mutants: To experiment further with <br /> the thymidine kinase gene, a thymidine kinase-based approach         ized orp2 on a second plasmid was introduced. Cells that <br /> was devised to integrate selected orp2 * mutants. We reasoned        had lost the wild-type orp2 plasmid were isolated by <br /> that integration of orp2 * into the plasmid shuffle strain JLP164    selecting against the thymidine kinase activity. Recessive <br /> (orp2D::ura4, porp2TKade) would allow the cells to grow at           phenotypes of the mutant orp2 alleles were then scored, <br /> permissive temperature without the porp2TKade plasmid and            plasmids were recovered and retested, and selected mu- <br /> therefore be viable on media containing FUdR. Integrants <br /> were selected on the basis of this idea. orp2 * linear DNA was       tants were integrated into the genome for further study. <br /> introduced into strain JLP164 by LiOAc transformation and               Selection against the thymidine kinase gene: Fungi <br /> the cells were grown for 2 days at 258 in rich media to allow        including S. pombe, S. cerevisiae, and N. crassa lack thymi- <br /> loss of the porp2TKade plasmid. Transformations were then            dine kinase and thus cannot use thymidine as a precur- <br /> plated directly on rich media (YES) containing 100 mm FUdR           sor for DNA synthesis (reviewed in Kornberg and <br /> and incubated at 258 for 3 days. Colonies were screened by <br /> replica printing to plates lacking uracil (if orp2D::ura4 has        Baker 1992). Thymidine kinase phosphorylates thymi- <br /> been replaced, they will fail to grow without uracil), to plates     dine, but also phosphorylates halogenated thymidine <br /> low in adenine (if porp2TKade is lost, colonies will be red on       derivates such as FUdR. Phosphorylated FUdR is a po- <br />  <br />  <br />                                                                TABLE 2 <br />                                                      Mutagenic PCR conditions <br />  <br />                          dATP <br />                          dGTP              Mg21                              % Nonfunctional             % Temperature <br />          PCR             (mm)              (mm)              Mn21                at 258                     sensitive <br />          1                 0.2               5              None                    28                          2.8 <br />          2                 0.2               5              0.5 mm                  70                          2.0 <br />          3                 0.2               8              None                    15                          1.2 <br />          5                 1.0               5              None                    25                          0.9 <br />          6                 1.0               5              0.5 mm                  45                          6.2 <br />          7                 1.0               8              None                    25                          3.5 <br />          8                 1.0               8              0.5 mm                  67                          3.8 <br />  602                                                        J. Kiely et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 3.&#226;&#8364;&#8221;Characterization of thymidine kinase-based <br />                                                                       plasmid shuffle. Cells contain LEU2 selectable vector (middle, <br />   Figure 2.&#226;&#8364;&#8221;Expression of thymidine kinase causes sensitivity         control) or LEU2, orp21 expression plasmid pJL207 (left and <br /> to FUdR. Strain CHP429 (wild type) and thymidine kinase               right). In the middle and right, cells also contain the shuttle <br /> expression strain JLP125 (padh-TK) were grown on YES plates           vector porp2TKade. All cells lack chromosomal orp2 (orp2 <br /> containing 0 mm, 1 mm, and 100 mm FUdR. (A) Cell morphol-             D::ura41) and are ade6 mutants (ade6-m210). (Top) Cells are <br /> ogy after 1 day at 328; (B) overall growth after 2 days at 328.       grown on YSO media so that cells lacking porp2TKade are red. <br />                                                                       (Bottom) Cells are grown on YSO plus 50 mm FUdR so that <br />                                                                       cells containing porp2TKade are killed, and only cells with an <br /> tent inhibitor of thymidylate synthetase and so prevents              additional source of functional orp2 (pJL207) can grow. <br /> de novo synthesis of dTTP (reviewed in Kornberg and <br /> Baker 1992). Without dTTP, cells cannot replicate DNA <br /> and eventually die. It has previously been shown that                 convenient way to identify FUdR resistance due to plas- <br /> budding yeast expressing thymidine kinase can be selec-               mid loss. <br /> tively killed by FUdR (Sclafani and Fangman 1986;                        The orp2 shuttle plasmid is the sole source of orp2 in <br /> S. B. Haase and S. Reed, unpublished data).                           the strains JLP164 and JLP166; the chromosomal gene <br />    We reasoned that fission yeast might also be sensitive             has been deleted and replaced by ura41 (orp2D::ura41). <br /> to killing by FUdR. Wild-type fission yeast and the thymi-            To test the feasibility of the plasmid shuffle, the JLP166 <br /> dine kinase expression strain JLP125 (materials and                   haploid was transformed with either the LEU2 orp21 <br /> methods) were grown on media containing FUdR. This                    expression plasmid pJL207 or the LEU2 vector pRep1. <br /> treatment caused cell cycle arrest of the thymidine ki-               As expected, cells that received the orp21 plasmid can <br /> nase strain JLP125: cells failed to divide but continued              lose the shuttle plasmid porp2TKade and grow on FUdR <br /> to grow in <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> (Figure 2). This cell cycle arrest is               medium whereas those transformed with the vector con- <br /> similar to that seen when DNA replication is blocked                  trol require the porpTKade plasmid and fail to grow on <br /> by hydroxyurea or by replication mutants. JLP125 cells                FUdR medium (Figure 3). <br /> were arrested by 1 mm FUdR, the lowest concentration                     Generation of orp2* mutants: The orp2 open reading <br /> tested, whereas the wild-type strain grows normally on                frame was mutagenized by PCR and cloned into the <br /> media containing 100 mm FUdR and shows only a slight                  LEU21 orp2 expression vector pJL207 to produce orp2* <br /> decrease in growth rate with no cell cycle delay on 600               mutant libraries. Amplified library pools were trans- <br /> mm FUdR.                                                              formed into fission yeast strain JLP166 selecting for <br />    A plasmid shuffle based on thymidine kinase: The                   Leu1. To determine which transformants received func- <br /> goal is to have a plasmid with the wild-type allele of your           tional orp2, colonies were replica printed to FUdR plates <br /> favorite gene (i.e., orp21) and to be able to select cells            to select against the porp2TKade plasmid. Transformants <br /> that lose this plasmid to uncover a mutant allele of this             grew on FUdR only when the orp2/LEU2 plasmid pro- <br /> gene so the function of the mutant can be determined.                 vided the essential functions of orp2. Temperature-sensi- <br /> For our case, this system requires a special orp2 shuttle             tive mutants were identified as transformants that could <br /> plasmid. This plasmid is named porp2TKade and has                     grow on FUdR plates at 258 but not at 368 (see materi- <br /> three important features: (1) wild-type orp21, (2) count-             als and methods for details). Sixteen temperature- <br /> erselectable marker thymidine kinase (padh-TK) so that                sensitive alleles of orp2 were obtained. Two of these, <br /> loss of the plasmid can be selected for by growth on                  orp2-2 and orp2-7, were integrated into the genome (re- <br /> FUdR, and (3) selectable marker ade61 for yeast trans-                placing orp2D::ura41 allele with orp2*) for further study <br /> formation; ade61 can also be detected by colony color.                (materials and methods). <br /> On low adenine medium, ade62 mutants form red colo-                      Analysis of orp2-2 and orp2-7 temperature sensitive <br /> nies, but ade62 mutants containing porp2TKade are Ade1                mutants: As expected for mutations that interfere with <br /> and form white colonies. In this case, colony color is a              DNA replication, the orp2 mutants show cell cycle de- <br />                                                  Mutants of Fission Yeast orp2                                                603 <br />  <br /> fects at restrictive temperature. At 36.58, strains JLP208       heterozygous orp2-2/orp21 or orp2-7/orp21 diploids grow <br /> (orp2-2) and JLP216 (orp2-7) stop dividing, elongate,            and divide at the restrictive temperature of 36.58 indi- <br /> and lose viability. The division arrest of orp2-7 is hetero-     cating that both alleles are recessive. <br /> geneous and not all cells remain arrested, so most exper-           Because we are studying a replication protein, the <br /> iments use orp2-2. The temperature-sensitive pheno-              fact that we used plasmid-borne alleles in the original <br /> types of both JLP208 (orp2-2) and JLP216 (orp2-7) can            screen may have allowed selection of only slightly defec- <br /> be complemented by orp21 on plasmid pJL207. Finally,             tive alleles. The idea is that a cell dependent on a par- <br />                                                                  tially defective replication protein would have lower ef- <br />                                                                  ficiency of plasmid replication leading to less of the <br />                                                                  protein as well as significant plasmid loss. In other cases <br />                                                                  where the mutagenized gene&#226;&#8364;&#8482;s normal function will not <br />                                                                  affect plasmid copy number or stability, screening for <br />                                                                  mutant alleles on plasmids might yield severely defective <br />                                                                  mutants because the plasmid-borne gene will be present <br />                                                                  in many copies; as a result, defective proteins would <br />                                                                  be overexpressed, which could mask subtle defects in <br />                                                                  function. <br />                                                                     FACS analysis shows that orp2-2 arrests in S-phase at <br />                                                                  restrictive temperature (Figure 4A). Unlike cdc10-129, <br />                                                                  which arrests with 1C DNA content, orp2-2 appears to <br />                                                                  arrest in S-phase with between 1C and 2C DNA content. <br />                                                                  By 3.5 hr after shift to restrictive temperature, cell divi- <br />                                                                  sion has ceased in the orp2-2 population; the septation <br />                                                                  index is ,1%, and cells are elongated. This indicates a <br />                                                                  first cycle arrest in S-phase for the orp2-2 allele. <br />                                                                     If the cell division arrest is a result of incomplete DNA <br />                                                                  replication, cell cycle arrest should require replication <br />                                                                  checkpoint functions. To test this, the orp2-2 mutant <br />                                                                  was crossed to the replication checkpoint mutants rad1-1 <br />                                                                  and rad3D. At 368, the single mutant orp2-2 becomes <br />                                                                  highly elongated, indicative of cell cycle arrest. This <br />                                                                  arrest is checkpoint dependent because at 368, the orp2-2 <br />                                                                  rad1-1 or orp2-2 rad3D double mutants continue to di- <br />                                                                  vide. These divisions are aberrant and result in &#226;&#8364;&#339;cut&#226;&#8364;&#65533; <br />                                                                  cells and loss of viability as expected for cells that enter <br />                                                                  mitosis with incompletely replicated chromosomes (Fig- <br />                                                                  ure 4B). <br />                                                                     Even when checkpoint functions are intact, the orp2-2 <br />                                                                  mutation results in genetic instability. To measure chro- <br />                                                                  mosome stability, we used Ch16, a small derivative of <br />                                                                  chromosome III that is stably maintained as a fourth <br />  <br />  <br />  <br />                                                                     Figure 4.&#226;&#8364;&#8221;Phenotypic analysis of orp2-2. (A) FACS analysis <br />                                                                  of DNA replication in orp2-2 mutants: cells were grown to mid- <br />                                                                  log phase and shifted to 36.58 for 0, 2, or 3.5 hr. Wild-type <br />                                                                  and cdc10-129 mutant strains were included as controls. (B) <br />                                                                  Checkpoint dependence of orp2-2 cell cycle arrest. (a) Wild- <br />                                                                  type, (b) rad1-1, (c) orp2-2, and (d) rad1-1 orp2-2 strains were <br />                                                                  grown in liquid media to mid-log phase (258), then shifted to <br />                                                                  restrictive temperature for orp2-2 (36.58) for 3.5 hr, fixed, <br />                                                                  and stained with 49,6-diamidino-2-phenylindole (DAPI). (C) <br />                                                                  Chromosome loss assay: wild-type or orp2-2 mutants containing <br />                                                                  a nonessential fourth chromosome (Ch16) were grown for <br />                                                                  5 hr at restrictive temperature (368), then plated on YSO <br />                                                                  media at permissive temperature (258); 10-fold more mutant <br />                                                                  cells were plated than wild type to compensate for the loss of <br />                                                                  viability of the mutant. Red (dark) colonies are formed when <br />                                                                  the fourth chromosome Ch16 is lost. <br />  604                                                     J. Kiely et al. <br />  <br /> chromosome (Niwa et al. 1986). Ch16 is monitored                   depends on inherited protein, then spores that inherit <br /> by intragenic complementation between ade6-M216 on                 mutant protein may show more severe DNA replication <br /> Ch16 and ade6-M210 on chromosome III. Cells with                   defects than cells inheriting wild-type protein. FACS <br /> Ch16 are Ade1 and form white colonies on media low                 analysis of the germinating spore population shows that <br /> in adenine (YSO) whereas loss of Ch16 results in a                 initial DNA replication is slow in spores that inherit <br /> red sector or colony. Ch16 was crossed into the orp2-2             mutant protein compared to the spores that inherited <br /> mutant, and the cells were incubated at 368 for 5 hr,              wild-type protein. Indeed, there is a clear difference <br /> and then plated on YSO at 258. Most cells died; z10%               between these two populations even at 298, the permis- <br /> survived and formed white or red colonies depending                sive temperature for replication and growth of orp2-2 <br /> on the maintenance of Ch16 (Figure 4C). In this experi-            strains (Figure 5). It appears that inherited mutant <br /> ment, z50% of surviving orp2-2 mutants lost Ch16 com-              Orp2-2 protein does not support replication as well as <br /> pared with only 1 in 1000 loss events in wild type. Chro-          similarly inherited wild-type protein. We had shown that <br /> mosome loss readily accounts for the loss of viability of          spores inheriting wild-type orp2 product were able to <br /> orp2-2 at restrictive temperature. Finally, plasmid main-          replicate their genome approximately twice (Leath- <br /> tenance and transformation efficiency are poor in the              erwood et al. 1996). This strongly suggests that the <br /> orp2-2 mutant even at the permissive temperature.                  protein is stable and functional through successive cell <br />    Analysis of cells lacking orp2 that inherit mutant orp2-2       cycles. Such stable inheritance of Orp2 is consistent with <br /> product: Previously, the orp2D::ura4 deletion allele of            a role for ORC in epigenetic inheritence. <br /> orp2 was analyzed using selective spore germination.                  In the spore germination experiment using spores <br /> Unexpectedly, we found that nearly two complete cycles             from strain orp2-2/orp2D::ura4 (JLP242), nearly 50% of <br /> of DNA replication were achieved by the orp2D::ura4                cells entered mitosis but failed nuclear division (Figure <br /> cells. We hypothesized that this was due to wild-type              7). This resulted in anucleate daughter cells and un- <br /> protein inherited by the spores. We have now used the              equally divided nuclei often cut by septum formation as <br /> orp2-2 mutant to ask directly whether cycles of DNA                well as cells with multiple aberrant septa. These mitotic <br /> replication are carried out using orp2 protein inherited           phenotypes suggest that orp2 function is important for <br /> from the parent.                                                   proper replication checkpoint control of mitosis. We <br />    To create a population of cells with the orp2 gene              had observed similar events at a low frequency in <br /> deleted, we selectively germinated spores containing               orp2D::ura4 cells that inherited wild-type protein, but <br /> orp2D::ura4. The diploid strain orp2-2/orp2D::ura4                 the effect is much more pronounced when the spores <br /> (JLP242) was constructed for comparison with the pre-              inherit mutant protein. At the time when nearly 50% <br /> viously studied diploid orp21/orp2D::ura4 (JLP46). Both            of cells that inherit mutant protein have abnormal divi- <br /> diploids were grown and sporulated at permissive tem-              sions, z5% of the cells that inherit wild-type protein <br /> perature (278), and spores were germinated in media                show this phenotype. Thus, when Orp2 is most defective, <br /> lacking uracil at either 298 or 348. The restrictive temper-       there is no checkpoint arrest in many cells and aberrant <br /> ature of 348 was chosen because preliminary experi-                mitoses occur. <br /> ments had shown that germination is poor at 36.58 in                  We tested whether the checkpoint response to HU <br /> the strains used.                                                  was affected by loss of orp2 function (Figures 6 and <br />    If DNA replication in orp2D::ura4 germinating spores            7). We repeated the selective spore germination in the <br />  <br />  <br />  <br />  <br />                                                                                Figure 5.&#226;&#8364;&#8221;Selective spore germination of orp2 <br />                                                                              deletion spores. (A) orp21/orp2D::ura41 parent <br />                                                                              JLP46 and (B) orp2-2/orp2D::ura41 parent JLP242. <br />                                                                              Spores were germinated in media lacking uracil so <br />                                                                              that only the orp2D::ura4 spores germinate. (Left) <br />                                                                              FACS analysis; (right) the cells fixed and stained <br />                                                                              with DAPI from the 14-hr, 348 sample. <br />                                                  Mutants of Fission Yeast orp2                                              605 <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;Selective spore germination in the presence of <br /> hydroxyurea at 348. Spores from wild-type/orp2D::ura41 par- <br /> ent JLP46 and orp2-2/orp2D::ura41 parent JLP242 were germi- <br /> nated in 15 mm HU media lacking uracil so that only the <br /> orp2D::ura41 spores germinate. (Left) FACS analysis; (right) <br /> the cells fixed and stained with DAPI. <br />  <br />  <br /> presence of HU at 348. FACS data confirm that the <br /> HU treatment largely blocked DNA replication in all <br /> populations in this experment (Figure 6). orp2 deletion <br /> spores from the orp21 diploid synthesized small amounts <br /> of DNA in HU and did not enter mitosis. By contrast, <br /> when orp2 deletion spores from the orp2-2 diploid were              Figure 7.&#226;&#8364;&#8221;Selective spore germination of orp2 deletion <br /> germinated in HU, they often went on to an aberrant              spores; quantitation of normal and aberrant divisions by mi- <br /> mitosis. These results show that a minimal level of Orp2         croscopic analysis of DAPI stained cells (100&#226;&#8364;&#8220;300 cells were <br /> function is required for the restraint of mitosis by the         counted for each sample). Cells from the 348 experiments <br />                                                                  shown in Figures 5 and 6 were analyzed. (A) Septation index <br /> replication checkpoint. <br />                                                                  counting morphologically normal divisions, &#226;&#8364;&#339;normal&#226;&#8364;&#65533; being <br />                                                                  cells with a single septum in the middle of a wild-type-length <br />                                                                  cell (z8&#226;&#8364;&#8220;12 mm long) with clearly divided nuclei. Cells that <br />                        DISCUSSION                                inherit wild-type orp2 product proceed through one appar- <br />    Temperature-sensitive alleles of orp2 were isolated by        ently normal division, which is blocked when these cells are <br />                                                                  treated with HU. (B) Septation index counting aberrant divi- <br /> a new method that uses thymidine kinase as a counterse-          sions, &#226;&#8364;&#339;aberrant&#226;&#8364;&#65533; being cells with multiple septa, unequally <br /> lectable marker. This method will be applicable to many          divided nuclear masses, and cut nuclei in which the septum <br /> other situations where a counterselection is required.           bisects the nuclear mass. <br /> The two orp2 alleles studied in detail were recessive and <br /> resulted in cell division cycle arrest. The orp2-2 mutant <br /> is unable to progress through S-phase as judged by FACS             At restrictive temperature, orp2-2 mutants arrest be- <br /> analysis, it requires replication checkpoint function for        fore mitosis. However, a further reduction in orp2 func- <br /> cell cycle arrest, and the orp2-2 defect results in genomic      tion compromises this checkpoint control. A selective <br /> instability in an otherwise wild-type strain. These pheno-       spore germination experiment was used to obtain cells <br /> types provide strong evidence that orp2 has an essential         deleted for the orp2 gene. When spores lacking orp2 were <br /> function in DNA replication.                                     germinated and analyzed, we found that the replication <br />  606                                                     J. Kiely et al. <br />  <br /> and division fates of the cells depended greatly on the            been found that alleles of these genes that allow little <br /> orp2 product inherited from the diploid parent. When               or no replication also fail to give a checkpoint arrest. <br /> only mutant product was inherited, DNA replication                 That mutations in so many different factors have such <br /> was more severely impaired, and many cells were unable             similar defects leads us to favor the model that all of <br /> to arrest at the replication checkpoint and instead pro-           these mutants are checkpoint defective because they fail <br /> ceeded into an aberrant mitosis. Such cells were also              in a common event, namely the formation of replication <br /> unable to arrest in response to HU treatment. From                 forks. <br /> this we conclude that a minimal level of orp2 function                The cut5 mutant may be in a special class, since cut5 <br /> is necessary to establish DNA replication checkpoint               cells are checkpoint defective even after replication <br /> control.                                                           forks have been initiated (Saka et al. 1994). Thus Cut5 <br />    One unexplained oddity in the data is that there were           may directly link the replication machinery with check- <br /> two populations of cells in the spore germination experi-          point machinery or may be part of the checkpoint ma- <br /> ments: cells of z1N DNA (i.e., cells that synthesized              chinery. Dbp11, the budding yeast homolog of Cut5, is <br /> little or no DNA) and cells of z2N DNA (Figure 5, A                similarly required for replication and for checkpoint <br /> and B). The 1N cells were not ungerminated spores                  responses. The function of Dbp11 appears related to <br /> (materials and methods). We do not know why there                  that of polymerase &#206;&#181; (Araki et al. 1995), consistent with <br /> were two populations of cells. However, we were inter-             the idea that replication forks generate checkpoint sig- <br /> ested in whether the z1N cells had synthesized any DNA             nals (Navas et al. 1995). <br /> at all. As a preliminary test, we treated germinating                 An alternative model is that prereplication complexes <br /> spores with hydroxyurea (Figure 6), with the idea that             themselves mark the DNA as unreplicated and thereby <br /> if cells were replicating even a small amount of DNA,              signal that replication is not completed. In the context <br /> then the presence of hydroxyurea would induce a dam-               of this model, one might imagine that, e.g., the Orp2- <br /> age response and cause a checkpoint arrest. However,               2 protein is defective in forming the prereplication com- <br /> we found that about half of the orp2D cells that inherit           plex and therefore triggers neither replication nor the <br /> orp2-2 product went on to an aberrant mitosis whether              checkpoint. The existence in budding yeast of alleles <br /> hydroxyurea was present or not. This result encourages             of orc5 and orc2 that can inhibit mitosis after replication <br /> us to speculate that perhaps the cells appearing to be             has been completed (Dillin and Rine 1998) is consis- <br /> 1N truly are 1N and have not replicated any DNA at all.            tent with this idea; these might be alleles that form the <br />    Furthermore, there is a good correlation between the            aberrant prereplicative complex-like structures that can <br /> number of cells that appear to be 1N and the number                continue to inhibit mitosis even after replication is com- <br /> that go on to an aberrant mitosis. In Figure 5A, z5%               pleted and normal prereplication complexes have been <br /> of the orp2D cells that inherit orp21 product remain at            dissassembled. The fission yeast orp2-2 allele that we have <br /> 1N, and z5% of the cells undergo aberrant mitosis                  characterized does not inhibit mitosis after replication is <br /> (Figure 7). In Figure 5B, about half of the opr2D cells            complete, but it should be possible to screen for this <br /> that inherit orp2-2 product remain at 1N, and about half           type of mutant in fission yeast. <br /> of the cells undergo aberrant mitosis (Figure 7).                     <br />    There are several possible interpretations of these           <br /> results, but we favor the idea that the role of Orp2 in            <br /> checkpoint signaling is to establish replication forks,       <br /> which in turn initiate checkpoint signals that prevent <br /> mitosis. That is, we propose that there is no checkpoint <br /> for a lack of DNA replication; rather, the checkpoint is                        <br /> activated by ongoing (or aberrant) DNA replication.                <br /> Thus, in a cell completely lacking replication, no check-          <br /> point is triggered and mitosis occurs.                         <br />    There is now a long list of genes that affect both DNA          <br /> replication and the replication checkpoint. In fission                <br /> yeast, these genes include orp2 (this article), orp1, cdc18,      <br /> cdt1, pol1 (DNA polymerase a), rad11 (RPA), rfc2 (RFC),         <br /> and cut5 (Kelly et al. 1993; Saka and Yanagida 1993;              <br /> Saka et al. 1994; Hofmann and Beach 1994; D&#226;&#8364;&#8482;Urso                 <br /> et al. 1995; Grallert and Nurse 1996; Carlson et al.           <br /> 1997; Parker et al. 1997; Reynolds et al. 1999). In           <br /> budding yeast, these genes include CDC6 (a homolog              <br /> of cdc18), CDC7, DBF4, and RFC5 (RFC) (Piatti et al.            <br /> 1995; Toyn et al. 1995; Sugimoto et al. 1996; Tavor-             <br /> mina et al. 1997; Noskov et al. 1998). In general, it has          <br /> </body></html>