<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Functional Annotation of Class I Lysyl-tRNA Synthetase Phylogeny <br />               Indicates a Limited Role for Gene Transfer <br />  <br />             Functional and comparative genomic studies have previously shown that the essential protein lysyl-tRNA <br />          synthetase (LysRS) exists in two unrelated forms. Most prokaryotes and all eukaryotes contain a class II <br />          LysRS, whereas most archaea and a few bacteria contain a less common class I LysRS. In bacteria the class <br />          I LysRS is only found in the &#226;&#65533;&#163;-proteobacteria and a scattering of other groups, including the spirochetes, while <br />          the class I protein is by far the most common form of LysRS in archaea. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> investigate this unusual <br />          distribution we functionally annotated a representative phylogenetic sampling of LysRS proteins. Class I <br />          LysRS proteins <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> a variety of bacteria and archaea were characterized in vitro by their ability to recognize <br />          Escherichia coli tRNALys anticodon mutants. Class I LysRS proteins were found to fall into two distinct groups, <br />          those that preferentially recognize the third anticodon nucleotide of tRNALys (U36) and those that recognize <br />          both the second and third positions (U35 and U36). Strong recognition of U35 and U36 was confined to the <br />          pyrococcus-spirochete grouping within the archaeal branch of the class I LysRS phylogenetic tree, while U36 <br />          recognition was seen in other archaea and an example from the &#226;&#65533;&#163;-proteobacteria. Together with the corre- <br />          sponding phylogenetic relationships, these results suggest that despite its comparative rarity the distribution <br />          of class I LysRS conforms to the canonical archaeal-bacterial division. The only exception, suggested from both <br />          functional and phylogenetic data, appears to be the horizontal transfer of class I LysRS from a pyrococcal <br />          progenitor to a limited number of bacteria. <br />  <br />  <br />    Aminoacyl-tRNA (aa-tRNA) synthesis is the process                        comparable indirect pathways exist for the production of Asn- <br /> whereby an amino acid is esterified to the 3&#226;&#172;&#732; end of a tRNA                 tRNAAsn and selenocysteinyl-tRNASec (reviewed in reference <br /> containing the corresponding anticodon sequence (15). After                 11). While the use of these pathways was initially assumed to <br /> their synthesis, aa-tRNAs are screened and selected by elon-                be idiosyncratic to a small minority of organisms, recent anal- <br /> gation factor Tu (16) and then taken to the ribosomal decoding              yses of completed microbial genome sequences has shown <br /> site, where they are paired with the complementary mRNA                     them to be widespread in prokaryotes with the corresponding <br /> codons (19). Consequently, the availability of a cellular pool of           direct pathways absent. GlnRS is absent from all characterized <br /> correctly aminoacylated tRNAs is essential for the faithful                 archaea and the majority of bacteria, while asparaginyl-tRNA <br /> translation of the genetic information encoded in mRNA into                 synthetase (AsnRS) is absent from most archaea and a signif- <br /> the corresponding polypeptide sequence (14). Aminoacyl-                     icant number of bacteria (31). Initial analyses of the complete <br /> tRNAs are primarily made by the aminoacyl-tRNA synthetase                   genome sequences of the archaea Methanococcus jannaschii <br /> (aaRS) protein family (9). The accuracy of aa-tRNA synthesis                (3) and Methanobacterium thermoautotrophicum (24) sug- <br /> by the aaRSs is ensured by their extremely high substrate                   gested that in addition to AsnRS and GlnRS, genes encoding <br /> specificity, which is further enhanced in some cases by the                 the cysteinyl-tRNA synthetase (CysRS) and lysyl-tRNA syn- <br /> existence of proofreading mechanisms. The 20 aaRS proteins,                 thetase (LysRS) might also be absent from some organisms. <br /> as for example found in Escherichia coli (6), are divided into              Experimental studies showed that while both CysRS (17, 26) <br /> two mutually exclusive structural groups of 10 members each,                and LysRS (13) are present in archaea, they are encoded by <br /> termed class I and class II (reviewed in references 1 and 5).               novel or reassigned genes, making the original annotation of <br /> The assignment of an aaRS specific for a particular amino acid              the corresponding open reading frames difficult. In the case of <br /> to one of the two structural classes is almost completely con-              LysRS experimental approaches identified a gene encoding a <br /> served, reflecting the ancient evolutionary origin of this family           class I aaRS (lysK) in the archaeon Methanococcus maripaludis, <br /> of proteins (21).                                                           in contrast to all previously characterized LysRS proteins that <br />    The 20 canonical aaRS proteins do not represent the sole                 belonged to class II. Comparative genomic analyses showed <br /> means of cellular aa-tRNA synthesis. It was first observed in               class II LysRS-encoding genes (lysS) to be absent from the <br /> Bacillus megaterium that Gln-tRNAGln could be synthesized                   majority of archaea and some bacteria, with lysK being found <br /> via Glu-tRNAGln without the use of glutaminyl-tRNA syn-                     instead in all cases (e.g., see reference 10). Functional (12) and <br /> thetase (GlnRS) (32), and subsequent studies showed that                    structural (29) characterizations have shown that the class I <br />                                                                             and II LysRS proteins are functionally equivalent but structur- <br />                                                                             ally unrelated. <br />   * Corresponding author. Mailing address: Department of Microbi- <br /> ology, The Ohio State University, 484 West 12th Ave., Columbus, OH <br />                                                                                Primarily as a result of the sequencing of numerous micro- <br /> 43210-1292. Phone: (614) 292-2120. Fax: (614) 292-8120. E-mail:             bial genomes, over 30 lysK gene sequences are now known. The <br /> ibba.1@osu.edu.                                                             distribution of organisms containing lysK, which in all but the <br />  <br />                                                                      4594 <br />  VOL. 184, 2002                                                                             EVOLUTION OF LYSYL-tRNA SYNTHETASES                                 4595 <br />  <br />                                                                                    nas aromaticivorans cells were a gift from Margaret Romine (Pacific Northwest <br />                                                                                    National Lab, Richland, Wash.) and Ferroplasma acidarmanus genomic DNA <br />                                                                                    was a gift from Paul F. Predki (DOE Joint Genome Institute, Walnut Creek, <br />                                                                                    Calif.). S. aromaticivorans cells were grown in 50% Luria broth for 36 h at 30&#194;&#176;C <br />                                                                                    and genomic DNA was then prepared according to standard procedures (23). <br />                                                                                       Cloning of lysK and lysS genes. LysRS encoding genes were amplified from <br />                                                                                    genomic DNAs as indicated under standard PCR conditions using the fol- <br />                                                                                    lowing primers (engineered restriction enzyme recognition sites are indicated <br />                                                                                    and underlined for each primer): for S. aromaticivorans lysK, Forward: (NdeI) <br />                                                                                    5&#226;&#172;&#732;-GCATATGACAGACGAGATCCGCACC, and Reverse: (BamHI) 5&#226;&#172;&#732;-GG <br />                                                                                    ATCCTCAGTCGAGCGCCTCGGC; for M. barkeri lysK, Forward: (XhoI) <br />                                                                                    5&#226;&#172;&#732;-GCTCGAGATGGCTGACACAATTCACTGG, and Reverse: (BlpI) 5&#226;&#172;&#732;-G <br />                                                                                    CTGAGCTTATGCCTGTTTTTCGGGGC; for M. barkeri lysS, Forward: (Nd <br />                                                                                    eI) 5&#226;&#172;&#732;-GCATATGAGCATGGAAATTAACAATGAAAAAATGTC, and Re- <br />                                                                                    verse: (BamHI) 5&#226;&#172;&#732;-GGATCCTCAGTCTTCCCTTTTCATCTGTGG; for F. <br />                                                                                    acidarmanus lysK, Forward: (XhoI): 5&#226;&#172;&#732;-GCTCGAGATGTACTGGGCAGAT <br />                                                                                    GCACTG, and Reverse: (BamHI): 5&#226;&#172;&#732;-GGATCCTTACTGTGCAGAAT- <br />                                                                                    TATTCTGGATAAG; for M. magnetotacticum lysK, Forward: (NdeI) 5&#226;&#172;&#732;- <br />                                                                                    GCATATGTCCTCCGATTCCCGC, and Reverse: (BamHI) 5&#226;&#172;&#732;-GGATCCC <br />                                                                                    TACATATCCTCGCCCGC). <br />                                                                                       The amplified fragments were cloned into Topo TA (Invitrogen) vector and <br />                                                                                    subsequently subcloned into expression vector pET15b (Invitrogen) using the <br />                                                                                    restriction sites introduced by PCR. Clones were sequenced prior to transfor- <br />                                                                                    mation into E. coli BL21 C&#226;&#171;&#185; strain. This procedure allowed the overproduction <br />                                                                                    of N-terminally His6-tagged derivatives, which have previously been found not to <br />                                                                                    perturb substrate recognition by LysRS (14). <br />                                                                                       Overexpression and purification of LysRS proteins. Transformants were <br />                                                                                    grown in Luria broth supplemented with ampicillin (100 &#226;&#65533;&#174;g/ml) and chloram- <br />                                                                                    phenicol (34 &#226;&#65533;&#174;g/ml) at 37&#194;&#176;C to an optical density at 600 nm of 0.6. The expression <br />                                                                                    of the recombinant protein was then induced by IPTG (isopropyl-&#226;&#65533;&#164;-D-thiogalac- <br />                                                                                    topyranoside) (final concentration of 1 mM) for 3 h at 37&#194;&#176;C. The cells were <br />                                                                                    harvested by centrifugation and washed with phosphate-buffered saline; this and <br />                                                                                    subsequent steps were performed at 4&#194;&#176;C. The cell paste was resuspended into <br />                                                                                    lysis buffer (50 mM Na2H2PO4, 300 mM NaCl, 5 mM 2-mercaptoethanol, pro- <br />                                                                                    tease inhibitor [Roche]) and sonicated for 10 cycles of 30 s. The cell extract was <br />    FIG. 1. Secondary structure of undermodified E. coli tRNALys. Ab-               obtained by ultracentrifugation at 100,000 &#226;&#171;&#187; g for 1 h and subsequently applied <br /> breviations: D, dihydrouridine; &#226;&#65533;&#186;, pseudouridine; m7G, 7-methyl-                   to an Ni-nitrilotriacetic acid agarose column (Qiagen), according to the standard <br /> guanosine; X, putative 3-(3-amino-3-carboxypropyl) uridine modifica-               procedure. The protein was eluted with an elution buffer (containing 750 mM <br /> tion. The undermodified variants used in this study contain uridine in             imidazole), dialyzed overnight against storage buffer (100 mM Tris-HCl [pH 8.0], <br /> place of 5-methylaminomethyl-2-thiouridine at position 34 and aden-                50 mM KCl, 25 mM MgCl2, 5 mM 2-mercaptoethanol, 50% glycerol), and then <br /> osine in place of N6-threonylcarbamoyladenosine at position 37.                    stored at &#226;&#171;&#186;20&#194;&#176;C. The purity of all the recombinant enzyme preparations was <br /> Changes in the anticodon nucleotides (nucleotides 34 to 36) are indi-              judged to be 95% or greater by sodium dodecyl sulfate-polyacrylamide gel <br /> cated.                                                                             electrophoresis followed by staining with Coomassie brilliant blue. <br />                                                                                       Cloning and in vitro transcription of tRNALys genes. tRNA genes for in vitro <br />                                                                                    T7 RNA polymerase transcription were cloned by annealing of two cDNA <br /> archaeal Methanosarcina group is to the exclusion of lysS, en-                     oligonucleotides and direct ligation into pUC18 plasmid using BamHI and Hin- <br />                                                                                    dIII restriction sites. The clones were transformed into E. coli DH5&#226;&#65533;&#163;, sequenced, <br /> compasses most of the archaea and a scattering of bacteria but <br />                                                                                    and amplified. The 3&#226;&#172;&#732; end of the transcription template was digested overnight <br /> no eukaryotes. Previous studies have attributed this unusual                       either with BstNI at 60&#194;&#176;C or NsiI at 37&#194;&#176;C. In vitro T7 RNA polymerase runoff <br /> evolutionary <span id='am-81' about='xsp:pattern' typeof='owl:Thing'>pattern</span> to either horizontal gene transfer (12, 33,                   transcription was conducted according to standard procedure (18). tRNALys <br /> 34) or selective retention from the last common ancestor in                        transcripts were purified on a denaturing 12% polyacrylamide gel and recovered <br /> particular lineages (22). To further investigate the evolution of                  by electroelution (Elutrap; Schleicher and Schuell), followed by desalting on a <br />                                                                                    Nap-5 Column (Pharmacia Biotech). Plasmid T7-911 for preparation of the <br /> the lysK gene family, we have now extended our previous                            His6-tagged recombinant T7 RNA polymerase was a gift from T. Shrader (De- <br /> preliminary studies and attempted to differentiate a range of                      partment of Biochemistry, Albert Einstein College of Medicine, New York, <br /> class I LysRS proteins based upon their mode of recognition of                     N.Y.). Nucleoside triphosphates were from Sigma. <br /> tRNALys. Comparison of these results with a revised phyloge-                          Genomic and phylogenetic analyses. All predicted amino acid sequences were <br />                                                                                    obtained from publicly available databases. Sequences were initially aligned <br /> netic tree of class I LysRS sequences suggests that selective <br />                                                                                    using the program CLUSTALX (30). The alignment was then used to generate <br /> retention was the major factor in the evolution of this protein                    a neighbor-joining phylogeny using the PHYLIP 3.57c package (distributed by <br /> family with horizontal gene transfer playing a relatively minor                    the author [J. Felsenstein]; Department of Genetics, University of Washington, <br /> role.                                                                              Seattle) with parameters previously used for aaRS phylogenies (20). Maximum- <br />                                                                                    likelihood phylogeny was generated using the program Tree-Puzzle 5.0 (www <br />                                                                                    .tree-puzzle.de) (28) (10,000 puzzle steps, JTT substitution matrix). <br />                        MATERIALS AND METHODS <br />    General. In vivo overproduced E. coli tRNALys variants, in which the antico- <br /> don loop nucleotides 34 and 37 have previously been shown to be unmodified <br />                                                                                                                      RESULTS <br /> (Fig. 1) (4), were prepared as described before (4), as was Borrelia burgdorferi <br /> class I LysRS protein (10). Aminoacylation assays were performed as previously       Aminoacylation activity and tRNA specificity of recombi- <br /> described (12). Methanosarcina barkeri fusaro genomic DNA was a gift from <br /> Kevin Sowers (University of Maryland Biotechnology Institute, Baltimore), Mag- <br />                                                                                    nant LysRS proteins. The recombinant LysRS proteins were <br /> netospirillum magnetotacticum genomic DNA was a gift from Elizabeth Bertani        tested for in vitro activity as determined by their ability to <br /> (Division of Biology, California Institute of Technology, Pasadena), Sphingomo-    aminoacylate transcripts of their own (cognate) tRNALys genes <br />  4596        AMBROGELLY ET AL.                                                                                                        J. BACTERIOL. <br />  <br />  <br />       TABLE 1. In vitro lysylation of tRNALys substrates by LysRS                 tRNA (Fig. 2C), while <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> of U35 to C did not result in <br />                                 proteins                                          any change in activity (Fig. 2D). The only significant reduction <br />                                        Formation of Lys-tRNALyse                  in activity with F. acidarmanus LysRS was observed with <br /> LysRSa protein source                                                             tRNALys where U36 was replaced with C, a change that virtu- <br />                          Cognate                                   E. coli <br />                                        Noncognate transcriptc                     ally eliminated recognition. <br />                         transcriptb                             tRNALysUUUd <br />                                                                                      Anticodon recognition by M. barkeri class II lysyl-tRNA syn- <br /> F. acidarmanus             &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185; <br />                                                                                   thetase. Recognition of the tRNALys anticodon by the class II <br /> M. magnetotacticum         &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185; <br /> M. barkeri                 &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#186;           LysRS of M. barkeri was investigated as described above for <br /> M. barkeri class II        &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185;           class I LysRS proteins. Mutation of the nucleotide U34 to G <br /> S. aromaticivorans         &#226;&#171;&#185;          &#226;&#171;&#185; (M. magnetotacticum)          &#226;&#171;&#185;           (Fig. 3A) led to a significant reduction in recognition by M. <br /> B. burgdorferi             &#226;&#171;&#185;f         &#226;&#171;&#185; (F. acidarmanus)              &#226;&#171;&#185;f          barkeri class II LysRS, whereas the replacement of U36 with C <br />   a <br />      All LysRS proteins are class I except as indicated for M. barkeri.           (Fig. 3B) caused only a slight change, indicating the relative <br />   b <br />      In vitro-transcribed tRNALys genes (UUU anticodon sequence) derived          importance of these positions. The strongest effects on amino- <br /> from the same organism as the LysRS. <br />    c <br />      In vitro-transcribed tRNALys genes (UUU anticodon sequence) derived from     acylation were observed for changes in U35 of tRNALys, the <br /> an organism different from that from which the LysRS was derived, as indicated.   G35 and C35 variants both showing significant reductions in <br />    d <br />      Synthesized in vivo; see text for details.                                   activity compared to wild-type (Fig. 3B). <br />    e <br />      Lysylation of tRNA was performed at 37&#194;&#176;C for 15 min with 1 &#226;&#65533;&#174;M enzyme, <br /> 100 &#226;&#65533;&#174;M lysine, and 4 &#226;&#65533;&#174;M tRNA. Symbols: &#226;&#171;&#186;, no detectable formation of                 Molecular phylogenies of class I LysRS. Phylogenetic anal- <br /> Lys-tRNALys compared to an enzyme-free control; &#226;&#171;&#185;, detectable formation of        yses of the class I LysRS protein family were performed using <br /> Lys-tRNALys. <br />    f <br />      Shown for comparison, taken from reference 13. <br />                                                                                   28 of the 31 sequences currently available in public databases. <br />                                                                                   The three excluded sequences are all from Borrelia genospe- <br />                                                                                   cies and show more than 75% amino acid identity to B. burg- <br />                                                                                   dorferi (N. Mejlhede, A. Montha&#194;&#180;n, M. Thesien, and M. Ibba, <br /> or other noncognate tRNALys species (Table 1). Of the pro-                        submitted for publication). As extensive tRNA anticodon rec- <br /> teins tested, only the class I LysRS of S. aromaticivorans was                    ognition data are available for the class I LysRS of B. burgdor- <br /> able to aminoacylate its own in vitro-transcribed tRNALys. To                     feri (25), this sequence was chosen for inclusion in this study. <br /> investigate whether lack of aminoacylation in the other cases                     Structural studies have revealed a high degree of similarity <br /> resulted from either inactive transcripts and/or inactive en-                     between the two subclass Ib aaRSs LysRS and glutamyl-tRNA <br /> zymes, cross-charging experiments were performed. B. burg-                        synthetase (GluRS) (29), suggesting that GluRS sequences <br /> dorferi class I LysRS, which has previously been shown to                         may be appropriate for the rooting of LysRS phylogenetic <br /> aminoacylate a variety of tRNALys species, was found to ami-                      trees. We therefore generated separate sequence alignments <br /> noacylate in vitro-transcribed F. acidarmanus tRNALys, indi-                      using CLUSTALX of 28 class I LysRS and 11 GluRS proteins <br /> cating that this tRNA was active. In addition, F. acidarmanus                     (chosen as a representative sampling based upon existing phy- <br /> LysRS was able to charge E. coli tRNALys, indicating that the                     logenies [33]) and then manually merged these into a single <br /> enzyme was active. Similar tests with the two M. barkeri LysRS                    alignment using a structure-based sequence alignment of Py- <br /> proteins and in vitro-transcribed M. barkeri tRNALys showed                       rococcus horikoshii LysRS and Thermus thermophilus GluRS <br /> the class I LysRS and tRNA to be inactive but the class II                        (29). Positions containing gaps in any of the sequences were <br /> enzyme to be active. Investigation of M. magnetotacticum                          then removed, leaving a total of 287 aligned residues present in <br /> showed the class I LysRS to be active, albeit at a significantly                  all sequences. This sequence alignment was then used to gen- <br /> lower level than the other active proteins tested, and revealed                   erate neighbor-joining and maximum-likelihood phylogenies. <br /> that in vitro-transcribed M. magnetotacticum tRNALys could be                     In both cases the GluRS sequences formed a single distinct <br /> charged by S. aromaticivorans LysRS.                                              outgroup with high confidence for the corresponding node <br />    Anticodon recognition by class I LysRSs. Anticodon recog-                      (84.3% by neighbor joining, 93% by maximum likelihood). A <br /> nition was investigated using undermodified E. coli tRNALys                       maximum-likelihood tree rooted by GluRS was then generated <br /> variants generated in vivo, which have previously been shown                      for the class I LysRS sequences (Fig. 4). The overall class I <br /> to be efficient substrates for a variety of both class I and class                LysRS phylogeny resembles a somewhat-reduced version of <br /> II LysRSs (4, 25). Initial tests using M. magnetotacticum LysRS                   the universal phylogenetic tree with distinct bacterial and ar- <br /> showed that while it was able to charge the wild-type tRNA                        chaeal groupings. The only exceptions are the placement of the <br /> and nucleotide 34 mutant (UUU and GUU anticodons, re-                             crenarchaeon Cenarchaeum symbiosum within the major bac- <br /> spectively), the level of activity for the other mutants was                      terial grouping and a subgroup of several bacteria close to the <br /> insufficient to detect charging above background. Since the M.                    pyrococcal proteins in the archaeal group. <br /> barkeri class I LysRS had been shown to be inactive, the pat- <br /> tern of anticodon recognition was only investigated in more <br />                                                                                                           DISCUSSION <br /> detail for the F. acidarmanus and S. aromaticivorans proteins <br /> (Fig. 2). Mutation of U34 to G in tRNALys was found to have                          Comparative genomics of tRNALys recognition. While all of <br /> no effect on recognition by S. aromaticivorans LysRS (Fig. 2A),                   the LysRSs, with the exception of the class I protein from M. <br /> changes at U35 resulted in reduced charging and the most-                         barkeri, were found to be active when heterologously pro- <br /> pronounced effect was observed for replacement of U36, which                      duced, only one was found to recognize its cognate in vitro- <br /> almost completely eliminated activity (Fig. 2B). With F. aci-                     transcribed tRNA substrate. Previous studies showed that the <br /> darmanus LysRS replacements of either U34 or U35 by G both                        B. burgdorferi (12) and P. horikoshii (29) class I LysRSs both <br /> led to improvements in recognition compared to wild-type                          recognized cognate in vitro-transcribed tRNALys, while the M. <br />  VOL. 184, 2002                                                                  EVOLUTION OF LYSYL-tRNA SYNTHETASES                     4597 <br />  <br />  <br />  <br />  <br />    FIG. 2. Aminoacylation of E. coli tRNALys variants by class I His6-LysRS proteins. Aminoacylation reactions were performed as described in <br /> the text (20-&#226;&#65533;&#174;l samples) in the presence of 4 &#226;&#65533;&#174;M tRNA and 100 &#226;&#65533;&#174;M [14C]lysine. Comparison of charging of wild-type and mutant tRNALys using <br /> class I LysRS enzymes from S. aromaticivorans (200 nm of enzyme) (A and B) and F. acidarmanus (2 &#226;&#65533;&#174;M enzyme) (C and D). <br />  <br />  <br />  <br /> maripaludis enzyme did not (12). These differences in tRNA               vitro and those that did not with respect to potential nucleotide <br /> gene transcript recognition may result from the absence of               modification sites. This was done by comparing known posi- <br /> essential RNA modifications necessary for recognition or fold-           tions of nucleotide modifications described for tRNALys from <br /> ing, as reported for human mitochondrial tRNALys (8). We                 E. coli (Fig. 1) and Haloferax volcanii (7) with positions con- <br /> attempted to correlate sequence differences between tRNALys              served in either the &#226;&#8364;&#339;active&#226;&#8364;&#65533; or &#226;&#8364;&#339;inactive&#226;&#8364;&#65533; tRNALys gene sets, <br /> genes (UUU anticodon) that gave rise to active transcripts in            but not in both (Fig. 5). These comparisons identified a notable <br />  <br />  <br />  <br />  <br />    FIG. 3. Aminoacylation of E. coli tRNALys variants by M. barkeri class II His6-LysRS. Aminoacylation reactions were performed as described <br /> in the text (20-&#226;&#65533;&#174;l samples) in the presence of 4 &#226;&#65533;&#174;M tRNA, 100 &#226;&#65533;&#174;M [14C]lysine, and 2 &#226;&#65533;&#174;M enzyme. (A) Position 34 variants; (B) position 35 and <br /> 36 variants. <br />  4598     AMBROGELLY ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                                                          during recognition by the class I LysRS from S. aromati- <br />                                                                          civorans, while only U36 was found to be important for the F. <br />                                                                          acidarmanus enzyme. This predominant role for U36 in <br />                                                                          tRNALys recognition was also observed for the class I LysRS of <br />                                                                          M. maripaludis but differs from the B. burgdorferi (25) and P. <br />                                                                          horikoshii (29) enzymes, for which both U35 and U36 were <br />                                                                          found to be critical recognition elements. Taken together with <br />                                                                          previous studies, the data presented here suggest that the class <br />                                                                          I LysRSs can be divided into two broad groups defined by <br />                                                                          whether either U36, or U35 and U36 are the preferred anti- <br />                                                                          codon nucleotides for recognition. This functional divergence <br />                                                                          among the class I LysRSs can also be correlated with the <br />                                                                          overall phylogeny of this family (Fig. 4, discussed below), in <br />                                                                          contrast to the class II LysRS family where the pattern of <br />                                                                          anticodon recognition is more highly conserved. <br />                                                                             Phylogenetic conservation of anticodon recognition by class <br />                                                                          II LysRSs. Previous studies on the activity of tRNALys antico- <br />                                                                          don variants with bacterial (E. coli [4] and Helicobacter pylori <br />                                                                          [25]) class II LysRSs have shown that all three positions of the <br />                                                                          anticodon are recognized, with U35 being the predominant <br />                                                                          recognition site. A similar pattern was also observed during <br />                                                                          studies of the human class II LysRS (27), with U35 and U36 <br />                                                                          both recognized and the former being the main recognition <br />                                                                          element. The data presented here (Fig. 3) indicate that an <br />                                                                          archaeal class II LysRS recognizes the anticodon of tRNALys <br />                                                                          in the same way as its bacterial and eukaryotic counterparts do, <br />                                                                          with U35 being the most important position. This suggests that, <br />                                                                          in contrast to their class I counterparts, class II LysRSs share <br />                                                                          a pattern of tRNA anticodon recognition which is conserved in <br />                                                                          the three primary domains of the universal phylogenetic tree. <br />                                                                          This difference in the evolutionary divergence of a function <br />                                                                          duplicated in the two LysRS classes supports earlier evidence <br />   FIG. 4. Maximum-likelihood phylogenetic tree of class I LysRS          (12) suggesting that these two families are functionally but not <br /> sequences. The tree has been rooted using GluRS sequences. Numbers       mechanistically equivalent. <br /> indicate the percentage occurrence of nodes after 10,000 puzzle steps.      Evolution of the class I LysRS family. The phylogenetic tree <br />                                                                          for class I LysRS sequences contains two major branches <br />                                                                          broadly corresponding to the bacteria and the archaea (Fig. 4). <br /> difference at position 26, which is found as N2,N2-dimethyl-             The inclusion of GluRS sequences places the root of the tree <br /> guanosine in H. volcanii and as adenosine in E. coli. All                between the bacterial and archaeal branches, indicating that <br /> tRNALys genes whose transcripts are recognized by their cog-             the evolutionary pattern of class I LysRS sequences is in agree- <br /> nate class I LysRSs contain A26, with the exception of M.                ment with that of the universal phylogenetic tree derived from <br /> magnetotacticum, while those that are not recognized contain             rRNA sequences (see, for example, reference 2). While the <br /> G26. While the possible role of a modified G26 in tRNALys                class I LysRS tree appears canonical it is somewhat restricted, <br /> folding and/or recognition now requires a more extensive cat-            lacking any known eukaryotic examples and being mainly lim- <br /> aloguing of in vivo RNA modification patterns, the strong                ited to &#226;&#65533;&#163;-proteobacteria in the bacterial branch and euryarcha- <br /> correlation between experimental and comparative genomic                 eota in the archaeal branch. This observation of a limited <br /> approaches provides a testable hypothesis for the observed               canonical phylogeny among the class I LysRSs suggests that <br /> differences in the activity of in vitro transcribed tRNALys              selective retention of the lysK gene in certain lineages has <br /> genes. This is further supported by the observation that the             contributed substantially to their present distribution, as pre- <br /> distribution of A versus G at position 26 broadly correlates             viously suggested (22). <br /> with the molecular phylogeny of class I LysRSs (Fig. 4 and 5).              Closer examination of the class I LysRS phylogeny, however, <br />    Evolutionary divergence of anticodon recognition among                suggests that limited gene transfer has also contributed to the <br /> class I LysRSs. The data presented above (Fig. 2) support                phylogenetic distribution of class I LysRSs. One possible ex- <br /> earlier proposals (e.g., those in reference 25) that the antico-         ception to the canonical universal phylogeny among the class I <br /> don is a crucial site in tRNALys for recognition by class I              LysRSs is seen for the C. symbiosum protein, which is posi- <br /> LysRSs. In agreement with studies using class I LysRSs from              tioned within the main bacterial branch. Earlier analyses based <br /> the bacterium B. burgdorferi and the archaeon M. maripaludis,            upon smaller data sets placed C. symbiosum as a branch pre- <br /> the first nucleotide of the anticodon (position 34) was not              ceding the &#226;&#65533;&#163;-proteobacteria (33, 34), possibly indicating hori- <br /> found to play a significant role during recognition. Nucleotides         zontal transfer from an ancestral crenarchaeote. While our <br /> U35 and especially U36 were both shown to be important                   present analysis suggests horizontal transfer from rather than <br />  VOL. 184, 2002                                                                      EVOLUTION OF LYSYL-tRNA SYNTHETASES                                4599 <br />  <br />  <br />  <br />  <br />    FIG. 5. Alignment of tRNALys sequence (UUU anticodon) from organisms encoding class I LysRSs. Nucleotide 26 is indicated and shown in <br /> boldface type, and the anticodon sequence is underlined. Abbreviations: Rpa, Rhodopseudomonas palustris; CC, Caulobacter crescentus; MMa, M. <br /> magnetotacticum; RP, R. prowazekii; Wol, Wolbachia sp.; SA, S. aromaticivorans; ML, Mesorhizobium loti; RS, Rhodobacter sphaeroides; PF, <br /> Pyrococcus furiosus; PA, Pyrococcus abysii; PH, Pyrococcus horikoshii; BB, B. burgdorferi; SC, Streptomyces coelicolor; TD, Treponema denticola; TP, <br /> Treponema pallidum; HB, Halobacterium sp.; AP, Aeropyrum pernix; MT, M. thermoautotrophicum; MJ, M. jannaschii; MM, M. maripaludis; AF, <br /> Archaeoglobus fulgidus; MB, M. barkeri; FA, F. acidarmanus; TV, Thermoplasma volcanium; TA, Thermoplasma acidophilum. Sequences of genes <br /> encoding tRNALys (UUU) were obtained from The Institute for Genomic Research (http://www.tigr.org), the Sanger Center (http://www.sanger <br /> .ac.uk/Projects/), and public databases (http://www.ncbi.nlm.nih.gov/cgi-bin/Entrez/genom_table_cgi). <br />  <br />  <br />  <br />  <br /> to the &#226;&#65533;&#163;-proteobacteria, a more reliable interpretation of the                Parts of this work were supported by funds from The Ohio State <br /> exact positioning of the C. symbiosum protein within the class              University (M.I.) and the National Institutes of Health (GM-22854 to <br />                                                                             Dieter So <br />                                                                                     &#194;&#168;ll). <br /> I LysRS tree, and of possible gene transfer events, is now <br /> dependent on the availability of other closely related archaeal                                              <br /> sequences. Within the archaeal branch of the class I LysRS <br />                                                                              <br /> tree, the group containing Pyrococcus species also contains                    <br /> several bacterial examples of LysRS. Functional analyses have              <br />                                                                             <br /> shown that bacterial (B. burgdorferi) and archaeal (P. horiko-                <br /> shii) enzymes from within this group share common tRNALys                     <br /> anticodon recognition properties (see above) not seen for class               <br />                                                                           <br /> I LysRSs from elsewhere in the phylogenetic tree, supporting a            <br /> possible common origin. The positioning of this group within              <br /> the canonical archaeal branch suggests that the corresponding              <br />                                                                            <br /> bacterial LysRSs (Borrelia, Treponema, and Streptomyces) orig-               <br /> inated from a gene transfer event from the pyrococcal progen-         <br />                                                                          <br /> itor into an ancestral bacterium. Thus, while selective retention   <br /> appears to have been the main determinant of the distribution               <br /> of class I LysRSs, horizontal gene transfer has also played a <br />                                                                          <br /> significant but less widespread role.                                       <br />   <br /> </body></html>