<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Towards identification essential genes using targeted genome <br /> sequencing comparative analysis <br /> Adam M Gustafson*&#226;&#8364;&#160;1, Evan S Snitkin*&#226;&#8364;&#160;1, Stephen CJ Parker1, <br /> Charles DeLisi1,2 Simon Kasif1,2,3 <br />  <br /> Address: 1Bioinformatics Graduate Program, Boston University, Boston, MA 02215 USA, 2Department Biomedical Engineering, Boston <br /> University, MA 02215 USA 3Children&apos;s Hospital Informatics Program Harvard MIT Division Health Sciences Technology, Boston, <br /> MA, USA <br /> Email: Adam M Gustafson* - gustafad@bu.edu; Evan S Snitkin* - esnitkin@bu.edu; Stephen CJ Parker - parker@bu.edu; <br /> Charles DeLisi - delisi@bu.edu; Simon Kasif - kasif@bu.edu <br /> * Corresponding authors &#226;&#8364;&#160;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 19 October 2006                                                     Received: 18 August 2006 <br />                                                                                Accepted: 19 October 2006 <br /> BMC Genomics 2006, 7:265     doi:10.1186/1471-2164-7-265 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/7/265 <br /> &#194;&#169; 2006 Gustafson et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />             Background: The identification genes essential <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> theoretical importance the <br />             understanding minimal requirements cellular life, practical importance identification <br />             potential drug targets novel pathogens. With great time expense required experimental <br />             studies aimed constructing catalog essential genes given organism, computational approach <br />             identify essential genes high accuracy great value. <br />             Results: We gathered numerous features generated automatically genome sequence <br />             data assessed relationship essentiality, subsequently utilized machine learning construct <br />             integrated classifier essential genes S. cerevisiae E. coli. When looking single features, <br />             phyletic retention, measure number organisms ortholog present in, most <br />             predictive essentiality. Furthermore, construction phyletic retention feature the <br />             time explored evolutionary relationship set organisms presence a <br />             gene predictive essentiality. We E. coli S. cerevisiae optimal sets always <br />             contain host-associated organisms small genomes closely related reference. Using <br />             optimally selected organisms, able improve predictive accuracy compared using all <br />             available sequenced organisms. We hypothesize predictive power genomes consequence <br />             process reductive evolution, parasites symbionts evolved gene content. <br />             In addition, essentiality measured rich media, condition resembles environments these <br />             organisms hosts nutrients provided. Finally, demonstrate integration of <br />             highly predictive features using probabilistic classifier resulted accuracies surpassing any <br />             individual feature. <br />             Conclusion: Using features obtainable directly sequence data, able construct classifier <br />             predict essential genes high accuracy. Furthermore, analysis set genomes in <br />             presence gene predictive essentiality suggest ways targeted <br />             sequencing used identification essential genes. In summary, methods presented here <br />             aid reduction time money invested essential gene identification targeting genes <br />             experimentation predicted essential high probability. <br />  <br />  <br />  <br />                                                                                                                                        Page 1 16 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                      http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br /> Background                                                        essential genes [8-10]. As essential genes a <br /> A fundamental step understanding cells function            unique evolutionary pressure, likely share <br /> is comprehension minimal gene set required          characteristics gleaned from <br /> sustain life. Before core requirements cellular life      genome sequence data. <br /> can understood, necessary identify compo- <br /> nents set diverse organisms. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> date, prediction       With practical theoretical importance the <br /> and discovery essential genes supported          identification essential genes mind, set to <br /> significant experimental work. Procedures               construct effective classifier essential genes which <br /> such single gene knockouts [1], <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> [2],          exploited various genomic descriptors gen- <br /> and conditional knockouts [3] used discov-           erated directly form sequence data. Previous works aimed <br /> ery mechanisms, techniques require            understanding properties human disease genes <br /> large investment time skill perform. With           taken similar approach [11,12]. Interestingly, <br /> increase availability gene <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>knockout</span> data, stud-        predictive descriptors human disease genes <br /> ies undertaken attempt decipher            identified Kondrashov et al. [12] identified our <br /> characteristics essential genes. Through analysis       study predictive microbial essential genes. <br /> essential genes numerous organisms, fundamental evo- <br /> lutionary mechanisms genomic fingerprints              As initial step construction classifier, we <br /> uncovered aid essential gene identification         explored various experimental genomic metrics to <br /> in organisms lacking experimental validation.                     assess relate essentiality E. coli S. <br />                                                                   cerevisiae. A metric shown highly <br /> Several studies taken advantage abundance          predictive essentiality previous studies reten- <br /> experimental data available model organisms order          tion genes different phyla. In order extract the <br /> to understand properties essential genes. For exam-        metric identified subsets organisms <br /> ple, groups suggested relation-      achieving highest accuracy prediction essential <br /> ship degree protein-protein interaction                genes. The predictive sets contained host-associated <br /> networks essentiality [4,5]. The implication      organisms small genomes closely related <br /> hubs networks increased importance          query organism. This result directly suggests how <br /> of abundance interaction partners. Other studies         targeted sequencing genomes used predic- <br /> have revealed relationships essentiality          tion essential genes. With rapidly decreasing invest- <br /> number transcription factor binding sites upstream        ments money time required the <br /> gene [6]. It demonstrated genes          sequencing microbial genome, approach iden- <br /> complex regulation enriched dispensable genes.             tifying essential genes targeted sequencing may <br /> High accuracy predictions essential genes            itself viable alternative identification of <br /> been using flux balance analysis [7]. This method            essential genes. <br /> has advantage generating hypotheses regarding <br /> which genes likely essential wide variety       In addition phyletic retention, assessed rel- <br /> of hypothetical conditions. There little doubt       ative performance previously reported indicators <br /> the plethora experimental data generated, addi-          essentiality protein interaction degree, protein <br /> tional properties essential genes documented        size codon bias. In S. cerevisiae, protein interaction <br /> the coming years.                                                 degree highly predictive essentiality, <br />                                                                   protein size codon bias predictive both <br /> While genome-wide experimental data abundant                organisms. Additionally, S. cerevisiae observed that <br /> model organisms S. cerevisiae E. coli, informa-       high counts certain individual charged amino acids <br /> tion limited newly sequenced organisms,              predictive size alone, implying high <br /> which precludes use data identification        counts amino acids distinguish proteins way <br /> genes essential survival. The ability identify essential   completely captured size. <br /> genes absence experimental data added <br /> importance, allows rationally          Interestingly, aforementioned metrics were <br /> select possible drug targets newly sequenced patho-           predictive essentiality organisms, relative <br /> gens. Fortunately, addition relationships               importance sets genes identified varied. The <br /> between essentiality various experimental measures,           striking difference relationship pro- <br /> there good deal research aimed under-            tein size essentiality. In E. coli, small proteins are <br /> standing genomic features essential genes. Metrics         enriched essential genes, S. cerevisiae essential <br /> such codon bias, number paralogs, phyletic              genes underrepresented smallest proteins. <br /> retention shown distinguishing             We hypothesize indicative pressure <br />  <br />  <br />                                                                                                                       Page 2 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                     http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> in E. coli maintain small proteins absence          Selection organisms phyletic retention measure <br /> other functional constraints, previously sug-        The plummeting cost genome sequencing making <br /> gested [13].                                                     comparative genomics attractive technique, crea- <br />                                                                  tive bioinoformatic methods advantage tar- <br /> After identifying genomic features predictive        geted sequencing prevalent [14-16]. <br /> essentiality E. coli S. cerevisiae, quantified     In vein, set understand evolutionary <br /> predictive limits assembled genomic characteristics       properties sets organisms presence <br /> by integrating using probabilistic classifier con-     ortholog indicative essentiality <br /> junction feature selection criterion novel     gene, hopes sequencing appropri- <br /> bioinformatic applications. Using easily obtainable         ate genomes, high accuracy predictions essential genes <br /> genomic features S. cerevisiae E. coli,         made. Retention gene long evolutionary <br /> that ability predict essential genes competitive       periods form allows recognition using sequence <br /> with classifiers included experimental features. The       similarity based techniques suggests performing <br /> fact able construct classifier using    critical function [17]. In previous studies, sets organ- <br /> descriptors generated sequence data allow broad        isms varying distantly related organisms sev- <br /> application technique organisms,           eral closely related organisms utilized the <br /> only gene annotation required. This ability        examination relationship retention of <br /> potential impact understanding essential          genes essentiality [8,9]. To knowledge, sys- <br /> genes different organisms, search drug     tematic analysis aim understanding nature <br /> targets poorly understood pathogens.                          set organisms presence gene most <br />                                                                  predictive essentiality performed. <br /> Results Discussion <br /> Experimental <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> essentiality                          In order identify set organisms the <br /> During analysis relationship various          retention gene predictive essentiality, we <br /> genomic features essentiality, important        selected small group organisms likely pre- <br /> our results context <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> essentiality.    dictive E. coli essential list, computed the <br /> The <span id='am-397' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> used experiments based          accuracy subsets reduced set. This strategy <br /> growth, lack thereof, mutants rich media con-        employed large number sequenced <br /> ditions. Clearly, conditions representative     organisms prevented examining possible sub- <br /> the wild type environments organ-            sets. In order assemble reduced set ranked <br /> isms inhabited evolved gene content. Study-        organisms based conditional mutual information <br /> ing metabolic pathways S. cerevisiae using flux balance       maximization criteria (CMIM), described Fleuret <br /> analysis indicates two-thirds metabolic       [18] detailed Methods section. Briefly, CMIM <br /> enzymes essential condition,             selection iterative feature selection procedure in <br /> experiments presence rich media result              features selected highest <br /> roughly 20% genes labeled essential [7]. Fur-        mutual information class variable, conditioned <br /> thermore, analysis set genes           features selected. This <br /> labeled dispensable knockout experiments,            procedure implemented organism a <br /> are ubiquitously present different phyla,        feature, feature vector stores occurrence of <br /> revealed overrepresentation biosynthetic pathways          orthologs E. coli genes. When defining class variable <br /> [8]. It likely pathways essential wild   essentiality corresponding genes, CMIM <br /> type conditions, suggested retention through-        selection returns organisms highly predictive of <br /> out evolution, surplus nutrients provided          essentiality set. Using method selected 20 <br /> they identified dispensable. Undoubtedly,        organisms set 180 sequenced prokaryotes and <br /> room debate meaningful <span id='am-398' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> essenti-        looked predictive accuracy possible subsets of <br /> ality, that, general, genes       various sizes. The utility approach validated by <br /> required rich media conditions vital         comparing distribution Receiver Operating Charac- <br /> most conditions. Therefore, feel despite           teristic (ROC) scores intelligently selected subsets <br /> these inconsistencies, valuable understand    random sets organisms (Figure 1), sets <br /> properties artificial set essential genes,    organisms selected using maximal mutual information <br /> although comprehensiveness set ques-           (data shown). We compared performance of <br /> tioned, accuracy limited experimen-        optimal subset organisms larger sets, and <br /> tal bias.                                                        seen Figure 2 Table 1, little lost despite <br />                                                                  small number organisms used. Thus, using only <br />                                                                  optimally selected organisms able achieve <br />  <br />  <br />                                                                                                                      Page 3 16 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                     http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br /> 45 <br />  <br />  <br /> 40               Subsets Of 5 From CMIM Top 20 <br />  <br />                  Random Subsets Of 5 From Set Of 180 <br /> 35 <br />  <br />  <br /> 30 <br />  <br />  <br /> 25 <br />  <br />  <br /> 20 <br />  <br />  <br /> 15 <br />  <br />  <br /> 10 <br />  <br />  <br />  5 <br />  <br />  <br />  0 <br />  0.5                    0.55                     0.6           0.65                             0.7                         0.75 <br />                                                   Area Under Curve <br /> Figure <br /> Distribution <br />          1 ROC areas curves E. coli essential gene prediction using CMIM selected subsets <br /> Distribution ROC areas curves E. coli essential gene prediction using CMIM selected subsets. Prob- <br /> ability density functions areas curves ROC analysis 100,000 randomly selected subsets organisms <br /> selected total 180 sequenced prokaryotes (blue) subsets 5 organisms selected set 20 were <br /> in turn selected based conditional mutual information selection procedure described text (red). Sensitivity and <br /> specificity computed based numbers true positives (essential genes) false positives (non-essential genes) <br /> identified number organisms ortholog given E. coli gene present decreases. <br />  <br />  <br /> better performance using 180 sequenced            1%, compared subsets five <br /> prokaryotes [see Additional file 5 organisms used       selected reduced set 20. As seen Fig- <br /> in analysis].                                              ure 3A, organisms included most <br />                                                                 frequently best performing sets Buchnera <br /> With evidence organism sets best        aphidicola Wigglesworthia glossinidia. Both these <br /> possible, examined distribution organisms        organisms host-associated organisms belonging the <br /> among predictive sets. To identified        Gamma proteobacteria, E. coli belongs. The <br /> those sets organisms achieved ROC score         explanation presence orthologs organ- <br />  <br />  <br />                                                                                                                     Page 4 16 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                 1 <br />  <br />               0.9 <br />  <br />               0.8 <br />  <br />               0.7 <br />  <br />               0.6 <br /> Sensitivity <br />  <br />  <br />  <br />  <br />               0.5 <br />  <br />               0.4 <br />  <br />               0.3                                                                   CMIM5 (AUC = 0.7004 ) <br />  <br />               0.2                                                                   CMIM7 (AUC = 0.6984) <br />  <br />                                                                                     Set Of 180 (AUC = 0.6481) <br />               0.1 <br />  <br />                 0 <br />                  0   0.1        0.2         0.3         0.4     0.5      0.6               0.7         0.8           0.9             1 <br />                                                           1 - Specificity <br /> Figure <br /> ROC curves <br />          2 different sets organisms E. coli essential gene prediction <br /> ROC curves different sets organisms E. coli essential gene prediction. ROC curves based on <br /> the methodology used Figure 1. In red optimal set 5 organisms selected CMIM set 20, in <br /> green optimal set 7 selected CMIM set 20, blue set 180 prokaryotes used study. The <br /> area curve (AUC) indicated graph. <br />  <br />  <br /> isms predictive essentiality E. coli       genome partly dependant sequence loss. In <br /> clear considering evolutionary history life-        words, presence alternative pathways allows <br /> style. First, organisms share relatively recent com-       viable minimal sets arise process <br /> mon ancestor E. coli, adjustment          reductive evolution, making intersection gene <br /> a host associated lifestyle underwent massive gene            sets organisms informative [21]. <br /> loss, minimal genetic novelty [19,20]. In pres-           The second property organisms accounts <br /> ence great selective pressure minimize genome           superiority prediction essentiality relates to <br /> size, majority genes retained absolutely         working definition essentiality survival under <br /> essential. One reason organisms perform bet-           rich media conditions. Although rich media conditions <br /> ter gene set given reduced          argued unrepresentative wild type condi- <br />  <br />  <br />                                                                                                                        Page 5 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Table 1: Influence organism composition phyletic retention performance <br />  <br />       Organisms Present                 Cumulative PPV             Number Of Genes Group        Number Essential Genes <br />  <br />                                                       CMIM Selected Set Of 5 <br />                5                              74.8%                             139                               104 <br />                4                              66.8%                             271                               181 <br />                3                              55.6%                             435                               242 <br />                2                              42.0%                             690                               290 <br />                1                              28.9%                            1302                               376 <br />                0                              16.8%                            3636                               612 <br />                                                      CMIM Selected Set 7 <br />                7                              77.0%                          122                                  94 <br />                6                              73.3%                          206                                  151 <br />                5                              65.3%                          311                                  203 <br />                4                              54.1%                          455                                  246 <br />                3                              43.0%                          633                                  272 <br />                2                              34.8%                          952                                  331 <br />                1                              25.1%                         1613                                  405 <br />                0                              16.8%                         3636                                  612 <br />                                          Set 27 Non-parasitic Gamma Proteobacteria <br />                27                             59.0%                          305                                  180 <br />                24                             46.6%                          545                                  254 <br />                20                             34.5%                          861                                  297 <br />                16                             27.3%                         1247                                  341 <br />                12                             23.9%                         1697                                  405 <br />                 8                             20.3%                         2235                                  453 <br />                 4                             18.3%                         2845                                  521 <br />                                                      Set 179 prokaryotes <br />               179                             75.0%                          12                                    9 <br />               161                             75.3%                          77                                   58 <br />               143                             64.8%                          196                                  127 <br />               126                             54.4%                          294                                  160 <br />               108                             48.4%                          419                                  203 <br />               90                              39.7%                          572                                  227 <br />               72                              33.0%                          817                                  270 <br />               54                              26.8%                         1172                                  314 <br />               37                              22.4%                         1659                                  371 <br />               19                              19.2%                         2379                                  457 <br />                1                              17.0%                         3516                                  596 <br />  <br />  <br />  <br />  <br /> tions organisms, conditions fairly            Performance individual features yeast <br /> accurate representation wild type conditions organ-         We extended analysis essential genes gathering <br /> isms resides inside host, utilizing available          various data sets representative different aspects <br /> nutrients. Again, having organisms preferable,       yeast biology quantify abilities to <br /> the exact set nutrients provided host vary,         distinguish essential non-essential genes. Because <br /> and intersection gene sets best captures generic           value ability predict essentiality studied <br /> host-associated lifestyle [20].                                    organisms, data sets focused features are <br />                                                                    easily obtainable; is, features generated <br /> Performing similar analysis S. cerevisiae using set 26     need extensive laboratory work. These fea- <br /> sequenced eukaryotes returned Schizosaccharomyces pombe,           tures fall general categories: (1) genomic fea- <br /> Encephalitozoon cuniculi, Eremothecium gossypii       tures, based solely sequence data, contain <br /> most abundant organisms highly predictive              features open reading frame (ORF) size, upstream <br /> sets (Figure 3B). Both E. cuniculi E. gossypii lead    size, phyletic retention; (2) protein features, which <br /> host-associated lifestyles, corroborating interpretation       based solely protein characteristics contain <br /> of optimal organism sets E. coli. Furthermore, E. gos-      data sets like amino acid composition, codon bias, and <br /> sypii smallest known eukaryotic genomes,              hydrophobicity. As reference comparing per- <br /> with 9.2 Mb nuclear genome [22].                                 formance genomic protein features, we <br />  <br />  <br />                                                                                                                         Page 6 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />              A <br />                                                                                     Wigglesworthia brevipalpis <br />                                                                                     Haemophilus ducreyi <br />                                                                                     Neisseria gonorrhoeae <br />                                                                                     Blochmannia floridanus <br />                                                                                     Wolinella succinogenes <br />                                                                                     Chlamydophila caviae <br />                                                                                     Phytoplasma OY <br />                                                                                     Bartonella henselae <br />                                                                                     Mycoplasma penetrans <br />                                                                                     Campylobacter jejuni <br />                                                                                     Helicobacter pylori <br />                                                                                     Francisella tularensis <br />                                                                                     Buchnera aphidicola <br />                                                                                     Tropheryma whipplei <br />                                                                                     Treponema pallidum <br />                                                                                     Mycoplasma mobile <br />                                                                                     Methylococcus capsulatus <br />                                                                                     Coxiella burnetii <br />                                                                                     Zymomonas mobilis <br />                                                                                     Fusobacterium nucleatum <br />             160     140     120     100      80      60        40     20        0 <br />  <br />              B <br />                                                                                     Homo sapiens <br />                                                                                     Ashbya gossypii <br />                                                                                     Encephalitozoon cuniculi <br />                                                                                     Schizosaccharomyces pombe <br />                                                                                     Trichomonas vaginalis <br />                                                                                     Cryptosporidium parvum <br />                                                                                     Entamoeba histolytica <br />                                                                                     Dictyostelium discoideum <br />                                                                                     Candida albicans <br />                                                                                     Plasmodium falciparum <br />                                                                                     Cryptosporidium hominis <br />                                                                                     Arabidopsis thaliana <br />                                                                                     Drosophila melanogaster <br />                                                                                     Tetrahymena thermophila <br />                                                                                     Giardia lamblia <br />                                                                                     Trypanosoma brucei <br />                                                                                     Cyanidioschyzon merolae <br />                                                                                     Cryptococcus neoformans <br />                                                                                     Neospora crassa <br />                                                                                     Caenorhabditis elegans <br />  <br />              150                   100                    50                    0 <br /> Figure 3 organism occurrence performing organism sets <br /> Distribution <br /> Distribution organism occurrence performing organism sets. Histograms display the <br /> frequency of CMIM selected 20 organisms 1% organism sets 5 (a) E. coli (b) S. cerevi- <br /> siae. Note organisms displayed histograms left right based order selection CMIM <br /> selection procedure. It seen graphs trend organisms selected earlier often <br /> present best performing sets, validates use selection method. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 7 16 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> included data sets derived lab-intensive experiments         present largest 10% proteins. <br /> such protein-protein interaction cellular localiza-        Although individual function attained significant p- <br /> tion data. The positive predictive value (PPV) selected        value, functions present involved cat- <br /> single features varying coverage displayed Figure        alytic activity interaction nucleic acids. Charged <br /> 4A. A complete list features used,          amino acids present active sites of <br /> rationale including them, detailed Methods [see         enzymes, participate catalytic mechanisms. <br /> Additional file 3 actual feature matrix used].                Additionally, charged amino acids make associations with <br />                                                                   charged substrates favorable electrostatic <br /> Genomic features                                                  interactions. Based observations, hypothesize <br /> As expected, best performing genomic feature          enrichment essential genes pro- <br /> phyletic retention measure, construction                teins high counts charged amino acids part <br /> described above. For clarity, term &apos;phyletic retention&apos;       functional capabilities amino <br /> was used presence ortholog         acids. <br /> organisms, place term &apos;conservation,&apos; order to <br /> prevent confusion measures substitution rate. A           Experimental features <br /> second feature genomic set predictive          In addition examining predictive power features <br /> of essentiality total upstream size gene. Genes      genomic protein set, measured the <br /> with largest upstream sizes markedly enriched          prediction accuracy experimentally derived fea- <br /> dispensable genes. This result explained              tures previously reported indicative essentiality. <br /> considering recent results Yu et al. showing          We observed genes high degree protein <br /> genes complex regulation enriched dispensa-           interaction network likely essential, which <br /> ble genes, conjunction possibility genes         agreement previous work [5,24,25]. Among <br /> with complex regulation larger upstream             proteins 5% degree, 42% essential, <br /> regions order accommodate increased number            considerable enrichment essential genes com- <br /> cis elements [8]. This connection previously             pared ~17% expected chance alone. It be <br /> shown valid Caenorhabditis elegans Drosophila        noted stated literature rela- <br /> melanogaster [23]. Our results suggest relationship        tionship degree essentiality partly <br /> exists regulatory complexity intergenic dis-          biases data [26]. Specifically, Coulumb et al. <br /> tance S. cerevisiae relationship accounts    state protein interaction dataset Data- <br /> the association intergenic distance essential-        <span id='am-396' about='xsp:base' typeof='owl:Thing'>base</span> Interacting Proteins (DIP), used this <br /> ity. Given number transcription factor binding        analysis, biased essential genes accu- <br /> sites present promoter gene determined             mulation interactions small scale experiments <br /> through arduous experimental procedures, beneficial         partial essential genes. The authors go <br /> to able use upstream size proxy regulatory         state partiality accounts significant <br /> complexity.                                                       component relationship degree essen- <br />                                                                   tiality. This contention substantiated showing <br /> Protein features                                                  disappearance relationship essentiality <br /> Examining single features protein subset                 connectivity using unbiased whole-genome <br /> revealed highly predictive essentiality.      yeast two-hybrid experiments. Others recently stated <br /> In addition previously discussed descriptors           lack reliability completeness yeast two- <br /> codon bias protein size, identified enrich-           hybrid data responsible absence relation- <br /> ment essential genes proteins abun-              ship connectivity essentiality [27]. <br /> dance certain amino acids. Specifically, proteins with <br /> the highest counts aspartate, glutamate, lysine        Performance individual features E. coli <br /> enriched essential genes PPVs 29.6%, 31.5%             Despite large evolutionary distance fundamental <br /> and 30.0% respectively 10% predictions.             biological differences, characteristics essential genes <br /> This trend partially explained observing large         S. cerevisiae E. coli largely similar. As seen Fig- <br /> proteins general enriched essential genes,       ure 4B, strongest predictor essentiality E. coli <br /> PPV 25.8% largest 10% proteins.                   phyletic retention, CAI. Number paralogs <br /> Although relationship explains high amino            protein size predictive essentiality in <br /> acid counts predictive essentiality, fails fully   organisms. Features E. coli genes used this <br /> clarify specific amino acids predictive         study available Additional files [see Additional file <br /> others. In order gain insight phenomenon,         4]. <br /> looked enrichment GO molecular function catego- <br /> ries proteins high counts charged amino acids <br />  <br />  <br />                                                                                                                        Page 8 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                          http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />           A <br />                                              S. cerevisiae <br />           60 <br />                                                                                                               Degree <br />           50                                                                                                  Glutamic Acid <br />                                                                                                               Lysine <br />           40                                                                                                  Glycine <br /> PPV (%) <br />  <br />  <br />  <br />  <br />           30                                                                                                  ORF Size <br />                                                                                                               Paralogs <br />           20                                                                                                  CAI <br />           10                                                                                                  TFBS Count <br />                                                                                                               Upstream Size <br />           0                                                                                                   Random <br />                              1           5              10             15             20 <br />                                     Top Percentage Predictions <br />  <br />  <br />           B <br />                                                    E. coli <br />           60 <br />  <br />           50 <br />  <br />           40 <br />                                                                                                               CAI <br /> PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                               Paralogs <br />           30                                                                                                  ORF Size <br />                                                                                                               Serine <br />           20 <br />                                                                                                               Random <br />           10 <br />  <br />            0 <br />                             1           5              10             15              20 <br />                                      Top Percentage Predictions <br /> Comparison <br /> Figure 4 single feature classification accuracies essential genes <br /> Comparison single feature classification accuracies essential genes. Displays accuracy single features in <br /> yeast (A) E. coli (B) classifying essential genes. Predictions shown cutoffs 1, 5, 10, 15 20% (x-axis), <br /> with accuracy, measured PPV, shown y-axis. For predicting essentiality, single features ordered in <br /> descending order, exception ORF size Serine E. coli figure (B). <br />  <br />  <br />  <br />  <br /> Differences feature performance S. cerevisiae             For example, features protein size and <br /> and E. coli                                                          codon bias predictive essentiality, accuracy <br /> Although features performed comparably                sets proteins identified varied <br /> both organisms, noticeable differences.              organisms. <br />  <br />  <br />                                                                                                                           Page 9 16 <br />                                                                                                     (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Protein size predictive essentiality organ-        conserved essential genes. Our set, designated as <br /> isms. Upon analysis determined small pro-         SC_All, composed features require extensive <br /> teins E. coli enriched essential genes, small     experimentation, addition easily obtainable fea- <br /> proteins S. cerevisiae slightly enriched dispensa-      tures, assess impact neglecting <br /> ble genes. In order gain insight discrepancy         experimental data prediction accuracy. A benefit to <br /> between organisms examined distribution            using na&#195;&#175;ve Bayes feature integration clas- <br /> of essentiality small large proteins organ-        sification assigned probability, making natural to <br /> isms context phyletic retention. As seen       rank predictions, allows direct comparison <br /> Figure 5, marked difference distribution     results using individual features. <br /> large small essential genes dissected bins <br /> based phyletic retention. In E. coli, virtually   Feature selection accomplished ranking features <br /> the largest proteins essential ubiquitously         using conditional mutual information maximization <br /> present 27 sequenced non-parasitic                 (CMIM), described Methods [see Additional file 1 for <br /> Gamma proteobacteria assembled (Figure 5B). We                 actual ranking]. The phyletic retention feature <br /> hypothesize enrichment essential genes           achieved highest mutual information essential- <br /> among conserved large proteins indica-            ity, consistent results single feature <br /> tive pressure reduce size individual proteins.     performance. By using 21 informative features in <br /> Larger proteins come increased cost cell          SC_All, 11 SC_GenProt 13 SC_GenProt_No, we <br /> terms raw materials energy expenditures dur-          able improve prediction accuracy inclu- <br /> ing protein synthesis. This idea consistent findings      sion features set [see Additional file 2]. <br /> from Lipman et al., observed increased conser- <br /> vation largest proteins genomes          Overall performance seen Figure 6A, the <br /> attribute pressure maintain small proteins        positive predictive value (PPV) 1, 5, 10, 15 and <br /> the absence functional constraints [13]. The over-       20% predictions shown reference selected single <br /> all enrichment essential genes small proteins         features. Unexpectedly, SC_All outperforms <br /> all levels conservation conserved large           SC_GenProt, small (~5% difference <br /> proteins supportive hypothesis.                  15 20% predictions, Figure 5A), which <br />                                                                   indicates losing little ignoring features <br /> Figure 5C yields insight lack essential genes         derived experimental data. As expected, <br /> among smallest S. cerevisiae ORFs. The enrichment          SC_GenProt_No greatly outperformed other <br /> dispensable genes amid small ORFs S. cerevisiae          feature sets. This consistent results shown <br /> to consequence abundance small species spe-         single feature analysis, phyletic retention <br /> cific genes. In organisms species specific genes         measure far predictive feature. <br /> enriched dispensable genes, expected <br /> based predictive power phyletic retention            As SC_GenProt_No performing significantly worse than <br /> identifying essential genes. Therefore, abundance           feature sets, use identifying espe- <br /> small species specific genes leads apparent trend       cially interesting genes. To assess ability of <br /> dispensability smallest S. cerevisiae genes. It         SC_GenProt_No identify essential genes less <br /> should noted organisms used phyletic          conserved, looked broader conservation <span id='am-399' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> conservation analysis set 16 sequenced fungi            yeast genes set 16 fungi protists. Based on <br /> and protists, diverse set genomes         set 16 organisms, 285 essential yeast <br /> than used optimal predictive set.                          genes orthologs 5 organisms less. In top <br />                                                                   15% predictions SC_GenProt_No, 24.6% of <br /> Performance integrated features yeast                       285 conserved essential genes identified. In <br /> To determine limits predictive abilities        contrast, 3.5% 285 conserved essential <br /> classification essential genes integrating multiple       genes identified SC_GenProt cutoff. <br /> features, utilized na&#195;&#175;ve Bayes classifiers. We assigned        Thus, SC_GenProt_No lowest accuracy of <br /> all features different overlapping sets,     integrated feature sets, useful its <br /> order assess relative contributions different sub-      increased ability predict conserved essential genes. <br /> sets features. The set, designate as <br /> SC_GenProt, composed features              Performance integrated features E. coli <br /> obtained directly sequence data. Our second set,             As yeast, feature sets integrated na&#195;&#175;ve Bayes clas- <br /> which designated SC_GenProt_No, identical             sifier used predict essentiality E. coli. Two sets <br /> SC_GenProt, lacks phyletic retention measure. We          containing easily obtainable features analyzed, <br /> included set order assess ability identify      EC_GenProt EC_GenProt_No. No experimental fea- <br />  <br />  <br />                                                                                                                       Page 10 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                          http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />            A                                                                B <br />            40                                                               45 <br />  <br />            35                                                               40 <br />  <br />                                                                             35 <br />            30 <br />                                                                             30 <br /> # Genes <br />  <br />  <br />  <br />  <br />            25 <br />                                                                             25 <br />            20 <br />                                                                             20 <br />            15 <br />                                                                             15 <br />            10 <br />                                                                             10 <br />             5                                                                 5 <br />  <br />             0                                                                 0 <br />                 0 2 4 6 8 10 12 14 16 18 20 22 24 26                              0 2 4 6 8 10 12 14 16 18 20 22 24 26 <br />  <br />  <br />  <br />            C                                                                D <br />           120                                                               45 <br />  <br />                                                                             40 <br />           100 <br />                                                                             35 <br />  <br />            80                                                               30 <br /> # Genes <br />  <br />  <br />  <br />  <br />                                                                             25 <br />            60 <br />                                                                             20 <br />  <br />            40                                                               15 <br />  <br />                                                                             10 <br />            20 <br />                                                                               5 <br />  <br />             0                                                                 0 <br />                 0   2   4   6    8    10 12 14 16                                 0    2     4    6      8    10 12 14 16 <br />  <br />                         Phyletic Retention                                                 Phyletic Retention <br /> Figure <br /> Phyletic 5distribution smallest largest essential genes S. cerevisiae E. coli <br /> Phyletic distribution smallest largest essential genes S. cerevisiae E. coli. Displays number of <br /> essential genes (blue) superimposed total number genes (gray), binned number organisms gene is <br /> present (phyletic retention) (A) 300 smallest proteins E. coli, (B) 300 largest proteins E. coli, (C) 300 small- <br /> est proteins S. cerevisiae (D) 300 largest proteins S. cerevisiae. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 11 16 <br />                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                      A                     S. cerevisiae <br />                  100 <br />  <br />                      90 <br />  <br />                      80 <br />  <br />                      70 <br />  <br />                      60                                                                        SC_All <br />            PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                SC_GenProt <br />                      50                                                                        SC_GenProt_No <br />                      40                                                                        Random <br />  <br />                      30 <br />  <br />                      20 <br />  <br />                      10 <br />  <br />                       0 <br />                                 1        5         10           15         20 <br />                                        Top Percentage Predictions <br />  <br />  <br />  <br />  <br />                      B                         E. coli <br />                  100 <br />  <br />                      90 <br />  <br />                      80 <br />  <br />                      70 <br />                                                                                                EC_GenProt <br />                      60 <br />            PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                EC_GenProt_No <br />                      50                                                                        Random <br />                      40 <br />  <br />                      30 <br />  <br />                      20 <br />  <br />                      10 <br />  <br />                      0 <br />                                1        5          10           15         20 <br />                                       Top Percentage Predictions <br />  <br /> Figure <br /> Comparison <br />         6 accuracies na&#195;&#175;ve Bayes classifiers <br /> Comparison accuracies na&#195;&#175;ve Bayes classifiers. The predictive accuracies integrating optimal feature sets using a <br /> na&#195;&#175;ve Bayes classifier shown yeast (A) E. coli (B). Accuracy, measured PPV (y-axis) shown 1, 5, <br /> 10, 15 20% predictions different feature sets. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 12 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                      http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> ture set used lack available genome wide         In process constructing integrated classifier, the <br /> analyses.                                                        relationship various genomic characteristics and <br />                                                                  essentiality explored. In E. coli S. cerevisiae, <br /> The phyletic retention measure was, yeast,        phyletic retention, protein size, codon bias iden- <br /> be informative feature ranking condi-           tified single features predictive <br /> tional mutual information [Additional file 1]. However,          essentiality. Furthermore, showed pre- <br /> where yeast obtained best PPV using 13            dictive groups organisms used phyletic retention <br /> features, E. coli required four: phyletic reten-    measure contain host-associated organisms are <br /> tion, serine, tryptophan paralog count (9 features           closely related reference organism. Despite the <br /> were optimally classify EC_GenProt_No). Figure          influence artificial definition essentiality the <br /> 6B shows performance integrated features E.        selection optimal genome sets, result still <br /> coli, PPV shown 1, 5, 10, 15 20%        useful suggesting targeted sequencing used <br /> of predictions.                                                  identification essential genes organisms. <br />                                                                  In addition phyletic retention codon bias, which <br /> Conclusion                                                       related essentiality previous studies, we <br /> The identification essential genes largely        identified relationship protein size essenti- <br /> experimental effort, achieved whole-genome               ality, knowledge explored <br /> knockout techniques. While organisms C.          before. Specifically, observed nature this <br /> elegans, possible devise highly effective screens   relationship differed E. coli S. cerevisiae, small <br /> essential genes using siRNAs [28], cost and/or ineffec-      protein size indicative essentiality E. coli and <br /> tiveness technique organisms makes          true large proteins S. cerevisiae. More- <br /> broad application currently infeasible. In paper         over, largest E. coli proteins, which <br /> assessed potential effectiveness methodology         conserved essential. We hypothesize that <br /> which genes prioritized based likelihood      observations indicative pressure to <br /> of testing positively lethality screen      maintain small proteome E. coli. <br /> subsequent small scale knockout screens per- <br /> formed predictions obtain experimentally           In summary strides prediction <br /> validated genes.                                                 essential genes based solely sequence data two <br />                                                                  fronts. First, gained insight properties of <br /> We investigated efficacy strategy using avail-    sets organisms presence ortholog is <br /> able knockout experiments assess predictive power         predictive essentiality. Second, assessed <br /> of features easily obtainable sequence data        predictive power sequence based features, <br /> and integrating using machine learning meth-           achieved superior prediction accuracy inte- <br /> odologies. By integrating genomic protein character-         gration probabilistic framework intelligent fea- <br /> istics varying predictive power using probabilistic         ture selection. <br /> classifier feature selection, able achieve an <br /> overall predictive accuracy S. cerevisiae E. coli    Methods <br /> that superior performance individual           Sets essential genes <br /> feature. The use descriptors make clas-      Essential gene definitions taken Giaever et al[1] <br /> sifier robust using individual features          Gerdes et al[30] S. cerevisiae E. coli respec- <br /> predictive power likely vary great deal differ-    tively. Additionally, E. coli ORFs removed <br /> ent organisms. For example, codon bias strong predic-       80 amino acids. For S. cerevisiae all <br /> tor essential genes organisms studied here,       ORFs removed FASTA headers contained the <br /> a study 80 bacterial genomes revealed 30%           key words &quot;transposable&quot; &quot;mitochondrial&quot;. In total <br /> no codon bias [29]. Furthermore, able classify        4,728 yeast genes used, 966 essential. In <br /> essential genes reasonable accuracy          E. coli, 3569 genes used, 611 essential. <br /> the use gene conservation measure phyletic <br /> retention, providing added benefit identifying            Selection Features <br /> essential genes organism specific. The abil-        S. cerevisiae E. coli model organisms which <br /> ity identify essential genes sequence data         studied years. We capitalized <br /> has potential great practical importance         fact assessing relationship variety <br /> guiding investigations researchers searching          gene properties essentiality. Following list of <br /> potential drug targets newly sequenced pathogens.             predictors rational including them. Note that <br />                                                                  features used just S. cerevisiae marked a <br />                                                                  star, used E. coli marked stars. <br />  <br />  <br />                                                                                                                      Page 13 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Experimental Characteristics                                      comparison (downloaded UCSC Genome <br /> The following parameters require large exper-         Browser; conserved track). <br /> imental work obtain. These features used in <br /> our classifiers integrating features, excluded       Phyletic retention measure <br /> from using &apos;easily obtainable&apos; features.               Genes ubiquitously present different taxa <br />                                                                   likely essential [1]. As detailed the <br /> *Protein interaction network degree                               Results section, yeast E. coli separately, set five <br /> Generated curated interactions accumulated            organisms selected optimally predicted essenti- <br /> the Database Interacting Proteins (DIP) [31]. The              ality. A count gene reference <br /> degree protein computed summing number             organism (yeast E. coli) represents number of <br /> of unique interactions participates in. Degree,          orthologs present organisms. Bi-directional <br /> along related metrics network position,         best BLAST hits used define orthologous rela- <br /> documented literature indicative essen-        tionship (using E-value cutoff 0.1). <br /> tiality [5,24,25]. Protein interaction data included <br /> for E. coli low coverage.                                  Number paralagous genes <br />                                                                   Genes duplicates likely be <br /> *Protein Localization                                             essential [1]. The rational duplicates func- <br /> It known proteins GO transcriptional regula-         tion backup capacity presence knockout <br /> tion annotations enriched essential genes [1]. Based       <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> original gene. Paralogs defined as <br /> on result tested nuclear localiza-         genes present genome <br /> tion, protein localization categories,       BLASTP E-value 10-20. In addition <br /> useful predicting essentiality. Localization information       ratio larger gene smaller exceed <br /> was obtained previous large-scale study [32].              1.33. <br />  <br /> *Recombination Rate                                               **Strand bias <br /> Clusters essential genes known regions         Essential genes likely encoded lead- <br /> the genome characterized lower recombina-           ing strand circular chromosome [35]. <br /> tion rate [33]. All per-gene recombination rates were <br /> acquired Gerton et al. [34], analyzed according          Protein Characteristics <br /> to procedures used Pal Hurst [33].                         These parameters fall &apos;easily obtainable&apos; category as <br />                                                                   require coding sequence only; labo- <br /> Genomic Characteristics                                           ratory experiments necessary. We represent protein <br /> We consider following parameters &apos;easily obtainable&apos;          characteristics terms following metrics: amino <br /> in automatically generated sequence         acid composition, codon bias, codon adaptation index <br /> data.                                                             (CAI), frequency optimal codons (FOP), isoelectric <br />                                                                   point (PI), hydropathicity score, hydrophobicity <br /> Gene size                                                         score. For yeast data downloaded Saccha- <br /> There trend proteins larger         romyces Genome Database [36]. For E. coli metrics <br /> evolution. We expected gene size            generated using CodonW software package [37]. <br /> indicative essentiality, especially E. coli, ancestral <br /> genes likely essential.                                       Integration features <br />                                                                   Classification essential genes using the <br /> *Regulatory complexity                                            Orange machine learning package&apos;s implementation of <br /> Different genes S. cerevisiae exhibit wide variation        na&#195;&#175;ve Bayes. [38]. All features, exception the <br /> their regulatory complexity. It recently docu-           optimized phyletic retention measure binary <br /> mented relationship regulatory            localization features, discretized using Fayyad and <br /> complexity essentiality [6]. We measured regulatory           Irani&apos;s entropy discretization method [39], imple- <br /> complexity using following parameters: upstream size,         mented Orange. <br /> downstream size, upstream conservation down- <br /> stream conservation. All sizes measured dis-          After discretization, conditional mutual information max- <br /> tance nearest gene. Conservation measured           imization criteria (CMIM), described Fleuret [18], <br /> the number bases (up to) 1000 bp upstream            used rank features. Briefly, CMIM iterative <br /> of ORF start site (up to) 300 bp downstream            method feature vectors selected the <br /> the designated open reading frame, overlap ele-         highest mutual information class vector after <br /> ments identified conserved seven species            conditioning previously selected features vectors. <br />  <br />  <br />                                                                                                                       Page 14 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Let Y class vector, Xn feature, I(Y;Xn) the <br /> mutual information Y Xn, I(Y; Xn | Xm)                            Additional File 3 <br /> be conditional mutual information Y Xn                            Feature matrices S. cerevisiae. A raw data feature matrix, well <br /> conditioned Xm. CMIM implemented follows,                               entropy discretized feature matrix included. <br />                                                                                   Click file <br /> with alternate implementations described Fleuret [18]. <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> A score table s initialized values I(Y ; Xn). The                     2164-7-265-S3.xls] <br /> algorithm picks iteration feature Xm the <br /> highest score, refreshes score s [n] taking                     Additional File 4 <br /> the minimum s [n] I (Y ; Xn | Xm). The algorithm                           Feature matrices E. coli. A raw data feature matrix, an <br /> is run features selected.                                     entropy discretized feature matrix included. <br />                                                                                   Click file <br /> For given feature set, optimal number features                        [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                   2164-7-265-S4.xls] <br /> include classification determined empirically. Fea- <br /> tures ranked CMIM, iteratively removed                            Additional File 1 <br /> at time. At interval classification accuracy                       CMIM feature ranking. This excel file includes tables showing the <br /> measured. The cutoff optimal feature set iden-                        CMIM feature ranking. <br /> tified highest PPV 5% pre-                        Click file <br /> dictions, requirement PPV 1%                            [http://www.biomedcentral.com/content/supplementary/1471- <br /> must higher [see Additional file 2].                                           2164-7-265-S1.xls] <br />  <br />  <br /> To assign probability essentiality genes, the <br />                                                                                   Additional File 2 <br />                                                                                   Performance na&#195;&#175;ve Bayes classifiers using subsets features. For <br /> following procedure used. Half essential                           set features analyzed paper (e.g. SC_GenProt, <br /> half non-essential genes randomly chosen                           EC_GenProt, etc...), CMIM calculated features were <br /> be included training set. The classifier                          ranked order informative least. PPV 1, 5, 10 and <br /> tested remaining genes, assigns probability                        15% predictions shown na&#195;&#175;ve Bayes classifier constructed <br /> of essentiality. Training/testing bootstrapped 100                            using N features. <br />                                                                                   Click file <br /> times, gene probability essentiality was <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> taken average probabilities                            2164-7-265-S2.xls] <br /> assigned [see Additional file 6]. <br />                                                                                   Additional File 6 <br /> GO KEGG enrichment analysis                                                   Na&#195;&#175;ve Bayes classification results. For feature sets used E. <br /> The GeneMerge software used calculate enrichment                           coli S. cerevisiae, probability gene essential, as <br /> of KEGG annotations, using background genes                              reported na&#195;&#175;ve Bayes, provided. <br />                                                                                   Click file <br /> used given organism [40]. P-values given Bon- <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> ferroni corrected.                                                                2164-7-265-S6.xls] <br />  <br /> Authors&apos; contributions <br /> AMG, SCJP, ESS SK conceived project and <br /> designed study. AMG ESS drafted manuscript.                      Acknowledgements <br /> SCJP ESS performed analysis single feature per-                       The authors like thank Rich Roberts comments the <br /> formance. AMG performed integrated classifier analysis.                      manuscript, Joseph Mellor Dustin Holloway valuable <br /> ESS performed analysis identifying optimal organism                       insights project. This work supported NIH grants <br /> sets. CD SK directed research, provided feed-                    1P20GM066401, 1T32GM070409, R01 HG003367-01A1 NSF grant <br /> back project. All authors read                            ITR-048715 <br /> approved final manuscript. <br />                                                                              References <br />                                                                              1.      Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br /> Additional material                                                                  Lucau-Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, El- <br />                                                                                      Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br />                                                                                      M, Davis K, Deutschbauer A, Entian KD, Flaherty P, Foury F, Garfinkel <br />  Additional File 5                                                                   DJ, Gerstein M, Gotte D, Guldener U, Hegemann JH, Hempel S, Her- <br />                                                                                      man Z, Jaramillo DF, Kelly DE, Kelly SL, Kotter P, LaBonte D, Lamb <br />  List organisms used calculate phyletic retention. A list organ-            DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard <br />  isms used calculation phyletic retention shown. KEGG             P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Sche- <br />  letter codes used represent organisms, unless noted.           rens B, Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, <br />  Click file                                                                 Storms RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang CY, <br />  [http://www.biomedcentral.com/content/supplementary/1471-                           Ward TR, Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu <br />                                                                                      K, Bussey H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnston <br />  2164-7-265-S5.xls]                                                                  M: Functional profiling Saccharomyces cerevisiae <br />                                                                                      genome. Nature 2002, 418:387-391. <br />  <br />  <br />                                                                                                                                             Page 15 16 <br />                                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> 2.    Cullen LM, Arndt GM: Genome-wide screening gene func-                25.   Wuchty S: Evolution topology yeast protein inter- <br />       tion using RNAi mammalian cells. Immunol Cell Biol 2005,                    action network. Genome Res 2004, 14:1310-1314. <br />       83:217-223.                                                              26.   Coulomb S, Bauer M, Bernard D, Marsolier-Kergoat MC: Gene <br /> 3.    Roemer T, Jiang B, Davison J, Ketela T, Veillette K, Breton A, Tandia          essentiality topology protein interaction net- <br />       F, Linteau A, Sillaots S, Marta C, Martel N, Veronneau S, Lemieux S,           works. Proc Biol Sci 2005, 272:1721-1725. <br />       Kauffman S, Becker J, Storms R, Boone C, Bussey H: Large-scale           27.   Batada NN, Hurst LD, Tyers M: Evolutionary physiological <br />       essential gene identification Candida albicans applica-                 importance hub proteins. PLoS Comput Biol 2006, 2:e88. <br />       tions antifungal drug discovery.              Mol Microbiol 2003,     28.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana- <br />       50:167-181.                                                                    pin A, Le Bot N, Moreno S, Sohrmann M, Welchman DP, Zipperlen <br /> 4.    Maslov S, Sneppen K: Protein interaction networks arti-                 P, Ahringer J: Systematic functional analysis Caenorhab- <br />       facts. FEBS Lett 2002, 530:255-256.                                            ditis elegans genome using RNAi. Nature 2003, 421:231-237. <br /> 5.    Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central-        29.   Sharp PM, Bailes E, Grocock RJ, Peden JF, Sockett RE: Variation in <br />       ity protein networks. Nature 2001, 411:41-42.                               strength selected codon usage bias bacteria. <br /> 6.    Yu H, Greenbaum D, Xin Lu H, Zhu X, Gerstein M: Genomic anal-                  Nucleic Acids Res 2005, 33:1141-1153. <br />       ysis essentiality protein networks. Trends Genet 2004,         30.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />       20:227-231.                                                                    MD, Somera AL, Kyrpides NC, Anderson I, Gelfand MS, Bhattacharya <br /> 7.    Papp B, Pal C, Hurst LD: Metabolic network analysis                     A, Kapatral V, D&apos;Souza M, Baev MV, Grechkin Y, Mseeh F, Fonstein <br />       causes evolution enzyme dispensability yeast.                        MY, Overbeek R, Barabasi AL, Oltvai ZN, Osterman AL: Experi- <br />       Nature 2004, 429:661-664.                                                      mental determination level analysis essential <br /> 8.    Fang G, Rocha E, Danchin A: How essential nonessential                     genes Escherichia coli MG1655.                  J Bacteriol 2003, <br />       genes? Mol Biol Evol 2005, 22:2147-2156.                                       185:5673-5684. <br /> 9.    Chen Y, Xu D: Understanding protein dispensability               31.   Xenarios I, Rice DW, Salwinski L, Baron MK, Marcotte EM, Eisenberg <br />       machine-learning analysis high-throughput data. Bioinfor-                   D: DIP: database interacting proteins. Nucleic Acids Res <br />       matics 2005, 21:575-581.                                                       2000, 28:289-291. <br /> 10.   Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH: Role            32.   Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, Weissman <br />       duplicate genes genetic robustness null mutations.                  JS, O&apos;Shea EK: Global analysis protein localization bud- <br />       Nature 2003, 421:63-66.                                                        ding yeast. Nature 2003, 425:686-691. <br /> 11.   Smith NG, Eyre-Walker A: Human disease genes: patterns               33.   Pal C, Hurst LD: Evidence co-evolution gene order and <br />       predictions. Gene 2003, 318:169-175.                     <br /> </body></html>