<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genome-wide RNAi screening identifies protein <br /> damage regulator osmoprotective <br /> gene expression <br /> Todd Lamitina*&#226;&#8364;&#160;, Chunyi George Huang&#226;&#8364;&#161;, Kevin Strange*&#194;&#167; <br /> *Departments Anesthesiology Pharmacology, Vanderbilt University, T4208 Medical Center North, 1161 21st Avenue South, Nashville, TN 37232; and <br /> &#226;&#8364;&#161;Department Biological Chemistry, Johns Hopkins University School Medicine, 725 North Wolfe Street, Baltimore, MD 21224 <br />  <br />  <br /> Edited Martin Chalfie, Columbia University, New York, NY, approved June 23, 2006 (received review April 12, 2006) <br />  <br /> The detection, stabilization, repair stress-induced damage              ity-induced expression biosynthetic enzymes (7). In yeast, <br /> are essential requirements cellular life. All cells respond             hypertonicity induces expression glycerol biosynthesis <br /> osmotic stress-induced water loss increased expression                 enzyme glycerol 3-phosphate dehydrogenase (GPD), me- <br /> genes mediate accumulation organic osmolytes, solutes                  diates accumulation organic osmolyte glycerol (8). Cells <br /> that function chemical chaperones restore osmotic ho-                   renal inner medulla accumulate sorbitol myo-inositol <br /> meostasis. The signals signaling mechanisms regulate                  increased expression sorbitol biosynthetic enzyme <br /> osmoprotective gene expression animal cells poorly under-               aldose reductase (9) sodium-coupled myo-inositol trans- <br /> stood. Here, gpdh-1 gpdh-2, genes mediate                porter SMIT (10). <br /> the accumulation organic osmolyte glycerol, essential              The signals signaling mechanisms activate organic <br /> <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> nematode Caenorhabditis elegans osmotic                 osmolyte accumulation bacteria, plants, yeast been <br /> stress. Expression GFP driven gpdh-1 promoter                        studied extensively (6, 7, 11). For example, genetic molecular <br /> (Pgpdh-1::GFP) detected hypertonic stress            characterization yeast identified MAP kinase signaling <br /> induced stressors. Using Pgpdh-1::GFP expression                 cascade regulates hypertonicity-induced GPD expression <br /> <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span>, screened &#226;&#172;&#8225;16,000 genes RNAi feeding                       (11). Activation MAP kinase signaling mediated inter- <br /> identified 122 cause constitutive activation gpdh-1 expres-            action components cascade membrane protein <br /> sion glycerol accumulation. Many genes function                Sho1. In animal cells, hypertonic stress signals the <br /> regulate protein translation cotranslational protein folding               signaling cascades mediate osmosensitive gene expression <br /> and target degrade denatured proteins, suggesting              unknown. Sho1 homologs absent animal genomes, <br /> accumulation misfolded proteins functions signal activate           and, animal homologs yeast hypertonicity-activated <br /> osmoprotective gene expression organic osmolyte accumula-                  kinases exist, consistently shown play <br /> tion animal cells. Consistent hypothesis, 73%            significant role regulating osmosensitive gene expression <br /> protein-homeostasis genes shown slow age-depen-                   (12, 13). <br /> dent protein aggregation C. elegans. Because diverse environ-                  We undertaken studies hypertonic stress response <br /> mental stressors numerous disease states result protein                 Caenorhabditis elegans goal exploiting animal&#226;&#8364;&#8482;s <br /> misfolding, mechanisms exist discriminate                    genetic molecular tractability define animal cell osmo- <br /> osmotically induced forms stress-induced protein                  sensing mechanisms. Recently, demonstrated C. elegans <br /> damage. Our findings provide foundation understanding                adapts hypertonic stress accumulating organic osmolyte <br /> these damage-selectivity mechanisms function.                                  glycerol hypertonicity induces expression glycerol <br />                                                                                biosynthetic enzymes (14). Here, glycerol biosynthesis <br /> Caenorhabditis elegans &#229;&#8230;&#169; functional genomics &#229;&#8230;&#169; organic osmolytes &#229;&#8230;&#169;             essential survival hypertonic environments. Based these <br /> osmotic stress                                                                 findings, developed vivo GFP reporter reflects the <br />                                                                                activation state signaling pathways controlling osmosensitive <br />  <br /> T   ability detect, repair, stabilize cellular molec- <br />     ular damage induced environmental stress essential for <br /> <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> function. All organisms respond environ- <br />                                                                                gene expression. Using genome-wide RNAi screening, identi- <br />                                                                                fied 122 gene inactivations cause constitutive activation of <br />                                                                                osmosensitive GFP reporter absence hypertonic stress. <br /> mental stress increased expression stress-protective                   The majority genes function normally prevent the <br /> genes. For example, heat shock causes protein denaturation                 accumulation damaged denatured proteins cell <br /> induces expression molecular chaperones function                cytoplasm. The results studies suggest hypothesis that <br /> refold denatured proteins (1). Oxidative stress results lipid           signaling pathways controlling osmosensitive gene expression in <br /> protein damage activates expression antioxidant                     animal cells activated, part, increased levels of <br /> enzymes detoxify free radicals (2).                                       hypertonicity-induced protein damage, specifically damage pro- <br />                                                                                                                                                                        PHYSIOLOGY <br />  <br />  <br />  <br />    Hypertonic stress causes cellular water loss, cell shrinkage,               teins undergoing novo synthesis. <br /> elevated cytoplasmic ionic strength, increased expression of <br /> genes mediate organic osmolyte accumulation. Organic                      Results Discussion <br /> osmolytes function chemical chaperones accumu-                   When exposed hypertonic stress, C. elegans accumulates the <br /> lated cells concentrations hundreds millimolar                     organic osmolyte glycerol novo synthesis (14). The worm <br /> without adverse effects (3). Replacement inorganic ions with <br /> organic osmolytes allows cells maintain normal cytoplasmic <br /> ionic strength survive hypertonic environments (4, 5).                  Conflict statement: No conflicts declared. <br />    The effector mechanisms mediate organic osmolyte ac-                   This paper submitted directly (Track II) PNAS office. <br /> cumulation generally defined. For example, Esche-                  &#226;&#8364;&#160;Present address: Department Physiology, University Pennsylvania, 3700 Hamilton <br />  <br /> richia coli, ProP activated hypertonicity mediates                   Walk, A306 Richards Building, Philadelphia, PA 19104. <br /> uptake compatible solutes proline, glycine betaine,                 &#194;&#167;<span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span>   correspondence addressed. E-mail: kevin.strange@vanderbilt.edu. <br /> and ectoine (6). Plants accumulate proline hypertonic-                 &#194;&#169; 2006 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602987103                                                    PNAS &#229;&#8230;&#169; August 8, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 32 &#229;&#8230;&#169; 12173&#226;&#8364;&#8220;12178 <br />                                                                                    (data shown). We crossed ok1558 kb33 worms <br />                                                                                   generate gpdh-1;gpdh-2 double mutants. Glycerol levels (Fig. <br />                                                                                   1a), fertility (Fig. 1b), growth rates (data shown) of <br />                                                                                   gpdh-1(ok1558);gpdh-2(kb33) worms indistinguishable <br />                                                                                   wild-type animals normal culture conditions. How- <br />                                                                                   ever, exposed hypertonic stress, gpdh-1(ok1558);gpdh- <br />                                                                                   2(kb33) worms exhibited greatly reduced glycerol accumulation <br />                                                                                   (Fig. 1a) fertility (Fig. 1b) grew considerably more <br />                                                                                   slowly wild-type animals (Fig. 1c). Therefore, GPDH-1- and <br />                                                                                   GPDH-2-mediated glycerol accumulation essential sur- <br />                                                                                   vival hypertonic environments. <br />                                                                                      GFP transcriptional reporters demonstrated constitutive ex- <br />                                                                                   pression gpdh-2 intestine, hypodermis, excretory <br />                                                                                   cell (Fig. 2a). In contrast, Pgpdh-1::GFP expression virtually <br />                                                                                   undetectable control conditions showed dramatic <br />                                                                                   up-regulation intestine hypodermis worms exposed <br />                                                                                   hypertonicity (Fig. 2a). Pgpdh-1-GFP expression propor- <br />                                                                                   tional level hypertonic stress (Fig. 2b), time <br />                                                                                   course expression (Fig. 2c) correlated of <br />                                                                                   glycerol accumulation (14). Heat shock (30&#194;&#176;C; Fig. 2c), endo- <br />                                                                                   plasmic reticulum stress (10 &#226;?&#174;g&#229;&#8230;&#190;ml tunicamycin), cold (4&#194;&#176;C), and <br />                                                                                   oxidative stress (230 &#226;?&#174;M juglone 100% O2; data shown) <br />                                                                                   failed activate Pgpdh-1::GFP. These data demonstrate that <br />                                                                                   Pgpdh-1::GFP specifically reports activation signaling pathways <br />                                                                                   required glycerol synthesis adaptation C. elegans to <br />                                                                                   hypertonic stress. <br />                                                                                      To identify signals signaling mechanisms regulate <br />                                                                                   osmoprotective gene expression animal cells, performed a <br />                                                                                   genome-wide RNAi feeding screen based Pgpdh-1::GFP ex- <br />                                                                                   pression. The Pgpdh-1::GFP reporter strain used screen, <br />                                                                                   gpdh-1 undergoes striking sustained up-regulation <br />                                                                                   response hypertonicity (ref. 14 data shown), and <br />                                                                                   Pgpdh-1::GFP expression undetectable absence <br />                                                                                   hypertonic stress (Fig. 2a). L1-stage larvae 20 mM NaCl <br />                                                                                   growth plates fed dsRNA-producing bacteria, animals <br />                                                                                   visually scored GFP expression 3 days. We <br />                                                                                   identified 106 gene inactivations consistently activated <br />                                                                                   Pgpdh-1::GFP expression absence hypertonic stress (e.g., <br />                                                                                   Fig. 3a). These genes termed regulators glycerol-3- <br />                                                                                   phosphate-dehydrogenase expression (rgpd; Table 1; see <br />                                                                                   Table 2, published supporting information the <br />                                                                                   PNAS web site). <br /> Fig. 1. gpdh-1 gpdh-2 required adaptation hypertonic stress.          Because screen based activation GFP reporter, <br /> (a) Whole-animal glycerol levels wild type gpdh-1;gpdh-2 double <br />                                                                                   genes identified encode regulators transgene <br /> mutants. L1 larvae grown control 200 mM NaCl agar plates 3&#226;&#8364;&#8220; 4 <br /> days glycerol measurements. For measurements 400 mM NaCl, L1 <br />                                                                                   expression components signaling pathways con- <br /> larvae grown 200 mM NaCl L4 stage development            trolling glycerol accumulation. To determine role rgpd <br /> then shifted 400 mM NaCl plates 24 h. Values means &#226;&#171;&#190; SE (n &#226;&#171;&#189; 3). *,    genes hypertonic stress resistance, measured glycerol levels <br /> P &#226;&#172;? 0.01 compared wild type; &#226;&#8364;&#160;, P &#226;&#172;? 0.001 compared wild type. (b)       subset rgpd genes viable fertile loss-of- <br /> Brood size wild-type gpdh-1;gpdh-2 double-mutant worms. Values         function mutants available. Glycerol levels signifi- <br /> means &#226;&#171;&#190; SE (n &#226;&#171;&#189; 9 &#226;&#8364;&#8220;21). *, P &#226;&#172;? 0.01 compared wild type. (c) Rate growth   cantly elevated 4- 144-fold control animals worm <br /> in wild-type gpdh-1;gpdh-2 double-mutant worms exposed 400 mM              strains harboring loss-of-function mutations genes tested <br /> NaCl. Worm size quantified time-of-flight using COPAS Biosort. <br />                                                                                   (Fig. 3b). <br /> Values means &#226;&#171;&#190; SE (n &#226;&#172;&#381; 300). *, P &#226;&#172;? 0.01 compared wild type. <br />                                                                                      Genome-wide RNAi screening results significant numbers <br />                                                                                   false negatives (15). To identify additional rgpd genes, we <br />                                                                                   queried C. elegans Interactome, genome-wide protein&#226;&#8364;&#8220; <br /> genome contains genes encoding glycerol-3-phosphate-                          protein interaction map comprising 3,228 genes 5,685 yeast <br /> dehydrogenase (gpdh), catalyzes rate-limiting step                   two-hybrid interactions (16). Forty-eight rgpd genes present <br /> glycerol biosynthesis. Microarray (data shown) Northern                   Interactome showed direct interactions. However, <br /> blot analyses (14) demonstrated gpdh-1 exhibits strong                 identified 148 genes interact rgpd genes (see Fig. <br /> sustained transcriptional up-regulation hypertonic stress,                 5, published supporting information PNAS <br /> whereas gpdh-2 weakly transiently up-regulated. To test                    web site). RNAi constructs 124 genes present <br /> the physiological role glycerol biosynthesis hypertonic                 library. We rescreened interacting genes (see <br /> stress response, used deletion alleles gpdh-1 gpdh-2,               Methods) identified additional 16 rgpd genes (see Fig. 5). <br /> ok1558, kb33, respectively. Although rate hyperto-                     The Interactome screen increased rgpd genes 15% 122. <br /> nicity-induced glycerol accumulation slowed gpdh-1-                    This increase consistent false-negative rates 10&#226;&#8364;&#8220;30% <br /> deletion mutant, steady-state glycerol levels control                   estimated C. elegans genome-wide RNAi <br /> hypertonic conditions similar wild-type animals                  screens (15). <br />  <br /> 12174 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602987103                                                                                     Lamitina et al. <br />                                                                                       Fig. 3.    Loss gene function RNAi <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>mutation</span> causes constitutive <br />                                                                                      expression Pgpdh-1::GFP glycerol accumulation absence hyper- <br />                                                                                      tonic stress. (a) Examples genes identified genome-wide RNAi screening <br />                                                                                      activate Pgpdh-1::GFP expression silenced. (b) Glycerol content in <br />                                                                                      worms harboring loss-of-function mutations genes identified RNAi <br />                                                                                      screening. Values means &#226;&#171;&#190; SE (n &#226;&#171;&#189; 3). All glycerol levels mutant worms <br />                                                                                      significantly (P &#226;&#172;? 0.02) different wild type. <br />  <br />  <br />  <br />                                                                                      30 coexpressed gpdh-1 hypodermis and&#229;&#8230;&#190;or <br />                                                                                      intestine (Table 2) (17). rgpd gene functions fell defined <br />                                                                                      cellular processes group genes unassigned functions <br />                                                                                      (Tables 1 2). The defined processes include protein ho- <br />                                                                                      meostasis, extracellular matrix, signaling, metabolism, protein <br />                                                                                      trafficking, transcriptional regulation. <br />                                                                                         Four extracellular matrix genes encode collagens <br />                                                                                      DPY-7, DPY-8, DPY-9, DPY-10. Loss-of-function muta- <br />                                                                                      tions genes increased Pgpdh-1::GFP expression (data not <br />                                                                                      shown) glycerol accumulation (Fig. 3b). The DPY collagens <br />                                                                                      secreted proteins likely function extracellularly. Inter- <br />                                                                                      estingly, identified 10 genes unassigned functions <br />                                                                                      predicted encode secreted proteins (Tables 1 2). <br />                                                                                      Secreted proteins shown play important roles in <br />                                                                                      mechanotransduction. For example, MEC-1 MEC-9 are <br />                                                                                      secreted C. elegans touch neurons genetically interact with <br />                                                                                      collagen MEC-5 (18, 19). These proteins essential <br /> Fig. 2. gpdh transcriptional GFP reporters activated hypertonic stress. (a    components C. elegans touch neuron mechanosensory <br /> Upper) Pgpdh-2::GFP constitutively expressed hypodermis, intestine,    complex (20). Similarly, DPY collagens secreted <br /> excretory cell control conditions. (a Lower) Pgpdh-1::GFP expression un-    proteins function detect hypertonic stress-induced <br /> detectable isotonic conditions dramatically elevated exposure     mechanical signals. In vertebrates, collagens integrins func- <br /> to 200 mM NaCl. Pgpdh-1::GFP expressed intestine (Lower Right, arrow- <br />                                                                                                                                                                      PHYSIOLOGY <br />  <br />  <br />  <br />                                                                                      tion cellular mechanotransduction (21, 22) osmotic <br /> head) hypodermis (Lower Right, arrows). (b) Effect NaCl concentration on <br /> Pgpdh-1::GFP expression young adult worms. Animals placed agar plates <br />                                                                                      stress-induced signaling processes (23&#226;&#8364;&#8220;26). <br /> containing indicated amounts NaCl 24 h. Values means &#226;&#171;&#190; SE (n &#226;&#172;&#381;           dpy-7 dpy-10 shown suppress temperature- <br /> 1,000). Pgpdh-1::GFP expression induced hypertonic KCl, sucrose,      sensitive mutations unrelated genes (27, 28). Phenotypes <br /> sorbitol (data shown), indicating gpdh-1 expression regulated specif-    associated temperature-sensitive mutations thought be <br /> ically hypertonicity-induced water loss elevated Na&#226;&#171;&#185; Cl&#226;&#171;&#186; levels.   misfolding mutated protein reduced low <br /> (c) Changes Pgpdh-1::GFP expression young adult worms hypertonic        temperatures (29, 30). Because organic osmolytes glycerol <br /> stress heat shock (30&#194;&#176;C). Values means &#226;&#171;&#190; SE (n &#226;&#172;&#381; 150). GFP expression b    aid refolding denatured proteins (31), ability dpy <br /> and c quantified using COPAS BioSort normalized time-of-flight <br />                                                                                      mutants suppress temperature-sensitive gene mutations be <br /> (i.e., GFP fluorescence divided time-of-flight). <br />                                                                                      general property rgpd genes activate gpdh-1 expression <br />   Many rgpd genes encode conserved proteins, 72                        cause glycerol accumulation (Fig. 3b). <br /> them human homologs (Table 2). Expression patterns                            Surprisingly, majority (44%, 54 122) rgpd genes fell <br /> been determined using GFP reporters 35 genes,                      category defined protein homeostasis (Table 1). These <br />  <br /> Lamitina et al.                                                                                           PNAS &#229;&#8230;&#169; August 8, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 32 &#229;&#8230;&#169; 12175 <br />  Table 1. Summary gene knockdowns cause constitutive gpdh-1 expression <br />                                     Molecular function                                                                                        Human <br /> Process (no. genes)               (no. genes)             Gene example                          Description                            homolog <br />  <br /> Protein homeostasis (54) <br />   Protein synthesis (38)         Translation initiation (10)   B0511.10&#229;&#8230;&#190;eif-3.E   Translation initiation factor 3, subunit e              EIF3S6 <br />                                  tRNA synthetase (10)          F22D6.3/nrs-1      Asparaginyl-tRNA synthetase                             NARS <br />                                  RNA processing (18)           W08E3.1&#229;&#8230;&#190;snr-2      U1 snRNP component                                      SNRPN <br />   Protein folding (7)            Chaperonins (4)               K01C8.10&#229;&#8230;&#190;cct-4     Chaperonin complex, &#226;?&#166;-subunit                           CCT4 <br />                                  Hsp70 (1)                     F26D10.3&#229;&#8230;&#190;hsp-1     Molecular chaperones HSP70 superfamily                  HSPA8 <br />   Protein degradation (9)        26S proteasome (4)            C23G10.4&#229;&#8230;&#190;rpn-2     26S proteasome-regulatory complex                       PSMD1 <br /> Extracellular matrix (5)         Collagen (4)                  T14B4.7&#229;&#8230;&#190;dpy-10     Collagens (types IV XIII)                           Collagen &#226;?&#163; IV <br /> Signaling (15)                   Scaffolding (2)               ZK849.2            PDZ domain                                              GOPC <br />                                  Transduction (6)              F38H4.9&#229;&#8230;&#190;let-92     Ser&#229;&#8230;&#190;thr protein phosphatase 2A, catalytic               PPP2CB <br />                                                                                     subunit <br /> Metabolism (9)                   Trehalose synthesis (1)       H13N06.3&#229;&#8230;&#190;gob-1     Trehalose 6-phosphate synthase                          None <br /> Protein trafficking (6)          Nuclear import (1)            R06A4.4&#229;&#8230;&#190;imb-2      Nuclear transport receptor                              TNPO2 <br /> Transcriptional regulation (9)   CCR4&#229;&#8230;&#190;NOT complex (2)          F57B9.2&#229;&#8230;&#190;ntl-1      Negative regulator transcription                     CNOT1 <br /> Unassigned function (24)         Secreted protein (10)         C32E12.3&#229;&#8230;&#190;osr-1     Osmotic stress-resistance protein                       None <br />                                  Unknown function (14)         B0035.11           Uncharacterized conserved protein                       LEO1 <br />  <br />  <br />  <br /> genes encode proteins required RNA processing, protein                 protein translation activates Pgpdh-1::GFP expression (Tables 1 <br /> synthesis, protein folding, protein degradation. Protein               2). In cases, inhibition genes predicted to <br /> homeostasis genes function maintain levels properly folded           disrupt protein synthesis. <br /> and functioning cellular proteins. Inhibition protein ho-                  Previous studies shown hypertonic stress, not <br /> meostasis genes expected increase levels damaged              heat oxidative stress, inhibits protein synthesis yeast. <br /> cellular proteins. Recent studies Nollen et al. (32) support       Inhibition transient, recovery occurs Hog1p- <br /> idea. Wild-type GFP expressed C. elegans muscle cells                dependent process likely requires glycerol accumulation <br /> distributed uniformly cytoplasm. However, modified                  (36). Importantly, initiation elongation steps protein <br /> GFPs containing repeats glutamine undergo age-dependent                 synthesis inhibited vitro increases salt concentration <br /> aggregation (33). Genome-wide RNAi screening identified 187                little 10 mM, salt-induced inhibition fully reversed <br /> genes function slow aging-induced protein aggregation              organic osmolytes (37). Inhibition elongation causes dis- <br /> (32). We 34 122 rgpd genes overlapped          engagement actively translating ribosomes mRNAs and <br /> 187-gene data set (Table 2), 24-fold greater overlap                premature termination protein translation, resulting accu- <br /> expected chance (P &#226;&#172;? 0.001). Strikingly, 25 34             mulation incomplete aberrantly folded polypeptides. <br /> overlapping genes predicted function RNA processing,             Because cell shrinkage increases cytoplasmic salt concentration, <br /> protein synthesis, protein folding, protein degradation.               observations consistent model gpdh-1 <br /> Thus, genes function prevent protein aggregation              expression specifically activated osmotically induced dis- <br /> function inhibit gpdh-1 expression. When function          ruption new protein synthesis cotranslational folding <br /> genes disrupted, damaged denatured proteins accumulate <br /> in cells, gpdh-1 expression increased, leading glycerol <br /> accumulation. Our results consistent model which <br /> increased levels damaged denatured proteins act signal <br /> that triggers osmoprotective gene expression organic os- <br /> molyte accumulation (Fig. 4). Accumulation organic os- <br /> molytes expected stabilize protein structure decrease <br /> protein misfolding (31), which, turn, serve autoregu- <br /> late pathway activity. <br />    Interestingly, protein damage induced numerous stressors <br /> including heat shock (Fig. 2c) does activate Pgpdh-1::GFP. <br /> Thus, osmotic stress cause type protein damage that <br /> selectively activates gpdh-1 expression. Recent studies eukary- <br /> otic cells suggest mechanism osmotic stress-induced <br /> damage discriminated forms protein <br /> damage. Albanese et al. (34) demonstrated eukaryotes have <br /> at distinct systems detecting repairing protein <br /> misfolding. Canonical heat-shock proteins (HSPs) function to <br /> refold stress-denatured proteins (35). In contrast, chaperones <br /> that linked protein synthesis (CLIPS) function regulate <br /> de novo protein folding (34).                                              Fig. 4. Model regulation osmosensitive gene expression disruption <br />    The results screen suggest novo protein folding          protein homeostasis. Hypertonic stress-induced water loss causes elevated <br />                                                                            cytoplasmic ionic strength, which, turn, causes protein damage. Damaged <br /> associated protein synthesis plays critical role the <br />                                                                            proteins function signal activates gpdh-1 expression glycerol <br /> hypertonic stress response. Canonical HSPs detected            synthesis. Glycerol replaces inorganic ions cytoplasm functions a <br /> our screen. Instead, T complex chaperones hsp-1&#229;&#8230;&#190;              chemical chaperone aids refolding misfolded proteins. Loss of <br /> F26D10.3 predicted CLIPS (Tables 1 2). Furthermore,                function protein homeostasis genes causes accumulation damaged <br /> RNAi silencing 38 genes involved RNA processing                  proteins activation gpdh-1 expression. <br />  <br />  <br /> 12176 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0602987103                                                                                        Lamitina et al. <br />  rather denaturation existing proteins. Such mech-         Genome-Wide RNAi Screening. Genome-wide RNAi screening was <br /> anism allow cells discriminate osmotically            carried using commercially available RNAi feeding <br /> induced protein damage forms stress-induced               library (MRC Geneservice, Cambridge, U.K.). Single colonies <br /> damage. Our proposed model analogous unfolded                inoculated 100 &#226;?&#174;l LB media containing 25 &#226;?&#174;g&#229;&#8230;&#190;ml <br /> protein response, intracellular signaling tran-        carbenicillin grown 6&#226;&#8364;&#8220;8 h 37&#194;&#176;C. Twenty microliters of <br /> scriptional&#229;&#8230;&#190;translational program activated accumulation        culture spotted individual wells 24-well nem- <br /> of unfolded proteins endoplasmic reitculum (ER) lumen           atode growth medium (NGM) agar plates containing 20 mM <br /> that functions restore ER protein homeostasis (38).                 NaCl, 1 mM isopropyl &#226;?&#164;-D-thiogalactoside (IPTG), 25 <br />    The response multicellular organism C. elegans      &#226;?&#174;g&#229;&#8230;&#190;ml carbenicillin. After overnight induction dsRNA, 30&#226;&#8364;&#8220;40 <br /> hypertonic stress likely involves integration number       L1-stage animals dispensed well. GFP expression <br /> hypertonicity-induced signals signal-transduction pathways.        monitored 72 h 16&#194;&#176;C using M2Bio fluorescence <br /> However, RNAi studies indicate disruption protein          microscope (Zeiss, Thornwood, NY). Clones scored as <br /> homeostasis sufficient activate cellular osmoprotective    positive animals exhibited visible GFP fluores- <br /> pathways. Given wide range environmental stressors           cence. These clones rescreened quadruplicate, those <br /> induces protein damage, mechanisms exist discriminate        induced GFP expression trials were <br /> between osmotically induced protein damage forms          considered bona fide regulators gpdh-1 expression, rgpd <br /> stress-induced protein damage. Our findings provide new foun-        genes. <br /> dation understanding damage-selectivity mechanisms          rgpd genes present C. elegans Interactome (16), their <br /> function defining signaling pathways animal       interacting proteins rescreened rgpd phenotype by <br /> cells respond osmotic stress. Because accumulation misfolded     RNAi. Bacterial strains grown described above. Forty <br /> proteins hallmark diseases Alzheimer&#226;&#8364;&#8482;s Parkin-     microliters bacterial culture spotted individual <br /> son&#226;&#8364;&#8482;s (39), understanding molecular events underlying          wells 24-well NGM RNAi plates left room temperature <br /> detection osmotically induced protein damage provide        overnight. Five 10 L1-stage animals dispensed each <br /> important paradigm defining cellular response disease-      allowed develop gravid adults 16&#194;&#176;C. After <br /> induced protein damage.                                                laying &#226;&#172;&#381;20 eggs, adults removed plate. F1 <br />                                                                        RNAi-treated animals examined GFP expression 3&#226;&#8364;&#8220;4 <br /> Methods                                                                days hatching. The screen repeated twice, positive <br /> C. elegans Strains. All strains used study derived   clones assigned score based level GFP induction <br /> the N2 Bristol wild-type strain. Unless noted, worms         1 (weak) 3 (strong). Scores screen were <br /> were cultured 20&#194;&#176;C. The following strains used: LGI-           summed, clones total score &#226;&#172;&#381;2 considered <br /> RB1032 [osr-1(ok959)], PS3551 [hsf-1(sy441)], LGII-VC616               confer rgpd phenotype. Both genome-wide Inter- <br /> [dab-1(gk216)], CB128 [dpy-10(e128)], LGIV-CB12 [dpy-                  actome screens performed blind identity genes <br /> 9(e12)], LGX-CB88 [dpy-7(e88)], CB130 [dpy-8(e130)].               screened. <br />     gpdh-1(ok1558) obtained Caenorhabditis Genet- <br /> ics Center (University Minnesota, Minneapolis). We gener-           Measurement Brood Size Rate Growth. L1-stage worms <br /> ated gpdh-2(kb33) PCR-based screening. Both mutants            placed NGM plates containing 51, 200, 400 mM NaCl. <br /> outcrossed wild-type animals times exhibited           After 2&#226;&#8364;&#8220;4 days, single L4-stage hermaphrodites placed into <br /> gross phenotypic defects homozygotes. ok1558 encodes              individual wells 24-well plates. Worms transferred new <br /> 1,227-bp deletion F47G4.3 (flanking sequences, tgcaactgat&#229;&#8230;&#190;          wells day egg-laying ceased (3&#226;&#8364;&#8220;4 days). Brood size was <br /> ctagaaacca), kb33 encodes 758-bp deletion K11H3.1             determined summing number viable progeny well. <br /> (f lanking sequences, tttattcctc&#229;&#8230;&#190;aagaactgtt). gpdh-1(ok1558)             Growth rate estimated transferring L1-stage worms <br /> I;gpdh-2(kb33)III animals generated crossing PCR           NGM plates containing 400 mM NaCl. Worms were <br /> genotyping.                                                            washed plates M9 solution containing 400 mM NaCl at <br />                                                                        24 h intervals, animal size quantified time-of-flight, <br /> Plasmid Construction. To create Pgpdh-1::GFP, 3.3-kb PCR frag-       measured using COPAS Biosort (Union Biometrica, <br /> ment, containing 3 kb sequence 5&#226;&#172;&#732; start ATG          Somerville, MA). Time-of-flight direct measure worm <br /> first amino acids exon 2, cloned BamHI           axial <span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> (41). <br /> SphI sites pPD95.75. To create Pgpdh-2::GFP, 4.4-kb PCR <br /> fragment, containing 2.3 kb sequence 5&#226;&#172;&#732; start ATG        Glycerol Measurements. Hypochlorite-synchronized L1-stage lar- <br /> the amino acids exon 2 K11H3.1a gene,        vae placed enriched peptone plates streaked with <br /> cloned SphI sites pPD95.75. Both constructs               NA22 bacteria. After 3&#226;&#8364;&#8220;4 days 20&#194;&#176;C, gravid worms were <br /> confirmed sequencing. Primer sequences available           washed plates processed glycerol measurements as <br /> request.                                                               described (14). <br />  <br />                                                                        Statistical Analyses. Data presented means &#226;&#171;&#190; SE. Statistical <br /> Transgenics. Wild-type animals injected 75 ng&#229;&#8230;&#190;&#226;?&#174;l rol-6 <br />                                                                                                                                                PHYSIOLOGY <br />  <br />  <br />  <br />  <br />                                                                        significance determined using Student&#226;&#8364;&#8482;s two-tailed t test <br /> marker pRF4 25 ng&#229;&#8230;&#190;&#226;?&#174;l GFP construct using standard <br />                                                                        unpaired means. Significance overlap sets <br /> methods (40). Three Pgpdh-1::GFP the <br />                                                                        genes determined calculating representation factor, <br /> Pgpdh-2::GFP lines generated showed identical patterns GFP <br />                                                                        total number overlapping genes divided the <br /> expression control hypertonic growth conditions. <br />                                                                        expected number overlapping genes. Calculation rep- <br /> Array kbEx144 used generate integrated strain <br />                                                                        resentation factor P value carried described (42). <br /> kbIs5. Integration carried exposing &#226;&#172;&#8225;50 kbEx144 L4 <br />                                                                        P values &#226;&#172;?0.05 considered indicate significance. <br /> animals 30,000 &#226;?&#174;J&#229;&#8230;&#190;cm2 generated UV cross-linker <br /> (Hoeffer Instruments, San Francisco, CA). A single integrated          This work supported National Institutes Health (NIH) Grants <br /> line segregating 100% rol-6 animals isolated out-              DK61168 DK64743 (to K.S.) T.L. supported fellowship <br /> crossed times wild-type animals generate kbIs5             National Kidney Foundation. Some nematode strains used in <br /> [Pgpdh-1::GFP;rol-6(su1006)], used subsequent         work provided Caenorhabditis Genetics Center, which <br /> studies.                                                               funded NIH National Center Research Resources. <br />  <br />  <br /> Lamitina et al.                                                                          PNAS &#229;&#8230;&#169; August 8, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169; no. 32 &#229;&#8230;&#169; 12177 <br />   1. Young, J. C., Agashe, V. R., Siegers, K. &amp; Hartl, F. U. (2004) Nat. Rev. Mol.        21. DeMali, K. A., Wennerberg, K. &amp; Burridge, K. (2003) Curr. Opin. Cell Biol. 15, <br />     Cell Biol. 5, 781&#226;&#8364;&#8220;791.                                                                   572&#226;&#8364;&#8220;582. <br />  2. Liu, H., Colavitti, R., Rovira, I. I. &amp; Finkel, T. (2005) Circ. Res. 97, 967&#226;&#8364;&#8220;974.    22. ffrench-Constant, C. &amp; Colognato, H. (2004) Trends Cell Biol. 14, 678&#226;&#8364;&#8220;686. <br />  3. Yancey, P. H. (2005) J. Exp. Biol. 208, 2819&#226;&#8364;&#8220;2830.                                   23. Vom, D. S., Schliess, F., Reissmann, R., Gorg, B., Weiergraber, O., Kocalkova, <br />  4. Burg, M. B. (1995) Am. J. Physiol. 268, F983&#226;&#8364;&#8220;F996.                                       M., Dombrowski, F. &amp; Haussinger, D. (2003) J. Biol. Chem. 278, 27088&#226;&#8364;&#8220;27095. <br />  5. Yancey, P. H., Clark, M. E., Hand, S. C., Bowlus, R. D. &amp; Somero, G. N. (1982)       24. Sheikh-Hamad, D., Youker, K., Truong, L. D., Nielsen, S. &amp; Entman, M. L. <br />     Science 217, 1214&#226;&#8364;&#8220;1222.                                                                  (2000) Am. J. Physiol. 279, C136&#226;&#8364;&#8220;C146. <br />  6. Wood, J. M., Bremer, E., Csonka, L. N., Kraemer, R., Poolman, B., van der            25. Browe, D. M. &amp; Baumgarten, C. M. (2003) J. Gen. Physiol. 122, 689&#226;&#8364;&#8220;702. <br />     Heide, T. &amp; Smith, L. T. (2001) Comp. Biochem. Physiol. A 130, 437&#226;&#8364;&#8220;460.              26. Moeckel, G. W., Zhang, L., Chen, X., Rossini, M., Zent, R. &amp; Pozzi, A. (2006) <br />  7. Hasegawa, P. M., Bressan, R. A., Zhu, J. K. &amp; Bohnert, H. J. (2000) Annu. Rev.           Am. J. Physiol. 290, F223&#226;&#8364;&#8220;F231. <br />     Plant Physiol. Plant Mol. Biol. 51, 463&#226;&#8364;&#8220;499.                                         27. Nishiwaki, K. &amp; Miwa, J. (1998) Mol. Gen. Genet. 259, 2&#226;&#8364;&#8220;12. <br />  8. Hohmann, S. (2002) Microbiol. Mol. Biol. Rev. 66, 300&#226;&#8364;&#8220;372.                           28. Levy, A. D., Yang, J. &amp; Kramer, J. M. (1993) Mol. Biol. Cell 4, 803&#226;&#8364;&#8220;817. <br />  9. Smardo, F. L., Jr., Burg, M. B. &amp; Garcia-Perez, A. (1992) Am. J. Physiol. 262,       29. Gidalevitz, T., Ben Zvi, A., Ho, K. H., Brignull, H. R. &amp; Morimoto, R. I. (2006) <br />     C776&#226;&#8364;&#8220;C782.                                                                               Science 311, 1385&#226;&#8364;&#8220;1386. <br /> 10. Yamauchi, A., Nakanishi, T., Takamitsu, Y., Sugita, M., Imai, E., Noguchi, T.,       30. Van Dyk, T. K., Gatenby, A. A. &amp; LaRossa, R. A. (1989) Nature 342, 451&#226;&#8364;&#8220;453. <br />     Fujiwara, Y., Kamada, T. &amp; Ueda, N. (1994) J. Am. Soc. Nephrol. 5, 62&#226;&#8364;&#8220;67.            31. Auton, M. &amp; Bolen, D. W. (2005) Proc. Natl. Acad. Sci. USA 102, 15065&#226;&#8364;&#8220;15068. <br /> 11. O&#226;&#8364;&#8482;Rourke, S. M., Herskowitz, I. &amp; O&#226;&#8364;&#8482;Shea, E. K. (2002) Trends Genet. 18,             32. Nollen, E. A., Garcia, S. M., van Haaften, G., Kim, S., Chavez, A., Morimoto, <br />     405&#226;&#8364;&#8220;412.                                                                                 R. I. &amp; Plasterk, R. H. (2004) Proc. Natl. Acad. Sci. USA 101, 6403&#226;&#8364;&#8220;6408. <br /> 12. Kwon, H. M., Itoh, T., Rim, J. S. &amp; Handler, J. S. (1995) Biochem. Biophys. Res.     33. Morley, J. F., Brignull, H. R., Weyers, J. J. &amp; Morimoto, R. I. (2002) Proc. Natl. <br />     Commun. 213, 975&#226;&#8364;&#8220;9 <br /> </body></html>