<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Sept. 2003, p. 2875&#226;&#8364;&#8220;2881                                                                   Vol. 47, No. 9 <br /> 0066-4804/03/$08.00&#226;&#171;&#185;0 DOI: 10.1128/AAC.47.9.2875&#226;&#8364;&#8220;2881.2003 <br /> Copyright &#194;&#169; 2003, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       Intracellular Expression Peptide Fusions Demonstration of <br />                        Protein Essentiality Bacteria <br />             R. Edward Benson, Elizabeth B. Gottlin, Dale J. Christensen, Paul T. Hamilton* <br />                                                Karo Bio USA, Inc., Durham, NC 27703 <br />                        Received 16 December 2002/Returned modification 31 March 2003/Accepted 6 June 2003 <br />  <br />             We &#226;&#8364;&#339;protein <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>knockout</span>&#226;&#8364;? technique used identify essential proteins bacteria. This <br />           technique uses phage display select peptides bind specifically purified target proteins. The peptides <br />           expressed intracellularly cause inhibition growth protein essential. In study, <br />           peptides specifically bind seven essential proteins identified phage display then <br />           expressed fusions glutathione S-transferase Escherichia coli. Expression peptide fusions directed <br />           E. coli DnaN, LpxA, RpoD, ProRS, SecA, GyrA, Era dramatically inhibited cell growth. Under <br />           conditions, fusion randomized peptide sequence did inhibit cell growth. In growth- <br />           inhibited cells, inhibition relieved concurrent overexpression relevant target protein not <br />           coexpression irrelevant protein, indicating growth inhibition specific interaction of <br />           expressed peptide target. The protein knockout technique used assess essentiality of <br />           genes unknown function emerging sequencing microbial genomes. This technique be <br />           used validate proteins drug targets, corresponding peptides screening tools, discovery of <br />           new antimicrobial agents. <br />  <br />  <br />    Despite success antimicrobial agents, infectious dis-              prior knowledge protein function. Proteins expressed, <br /> ease second leading cause death worldwide (14).                purified, used phage display isolate peptides bind <br /> This large increased resistance pathogens            specifically them. The peptides expressed intracel- <br /> a variety antimicrobials accelerating trend. It              lularly glutathione S-transferase (GST) fusions, their <br /> evident dynamic resilient nature microbial life,             effect growth monitored. If protein essential, cell <br /> coupled profligate use antibiotics, selected                 growth inhibited. <br /> drug-resistant microbes wide spectrum pathogenic                    The protein knockout strategy builds observations <br /> organisms. There clear urgent need novel antibi-                reported previously. We shown many <br /> otics help combat problem (8, 10, 30).                               peptides selected phage display directed functional <br />    Although antibiotics use today directed             sites target proteins (reviewed reference 26). In ex- <br /> small number bacterial targets, shortage po-               tension previous work isolating peptide inhibitor of <br /> tential targets. Analyses bacterial genomes allowed                  prolyl-tRNA synthetase (ProRS) (24), Tao et al. showed that <br /> estimates &#226;&#8364;&#339;minimal genome set&#226;&#8364;? genes essential                 intracellular expression peptide fused GST could <br /> viability. A comparison genomes Haemophilus influ-                 slow bacterial growth rescue mice lethal <br /> enzae Mycoplasma genitalium identified 256 genes consid-                 dose virulent Escherichia coli. (46). Specific inhibition of <br /> ered necessary sufficient cell viability, study           ProRS cell demonstrated decrease the <br /> the frequency indispensable genes Bacillus subtilis                intracellular pool charged tRNAPro. Walker et al. (47) ex- <br /> genome arrived 250 450 essential genes (25, 32). This                  pressed 20,000 peptides E. coli 21 them <br /> range genes constitute number potential                growth inhibitory; however, targets peptides <br /> targets antimicrobial drug discovery, number far larger                unknown. This study showed importance using <br /> than dozen sites action currently existing              peptides fused proteins amino acid motifs improve <br /> antibiotics. Genome-sequencing efforts increased po-                stability. Blum et al. (1) induced expression library of <br /> tential development new classes antibiotics directed <br />                                                                              peptides fused thioredoxin selected cells possessing <br /> against novel targets (15, 39, 42). Among potential targets <br />                                                                              ThyA&#226;&#171;&#186; <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span> (trimethoprim resistance thymine <br /> revealed sequencing unknown function, well <br />                                                                              requirement growth). They discovered peptides inhib- <br /> as known function used drug <br />                                                                              ited cell growth phenotype sup- <br /> targets. There pressing need technologies can <br />                                                                              pressed overexpression ThyA protein. Peptide expression <br /> validate invalidate targets drug discovery showing <br />                                                                              used knock protein function dissect <br /> whether essential microbial <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>survival</span>. <br />                                                                              signal transduction pathways eukaryotic cells (5, 7, 9, 13, 17, <br />    Here &#226;&#8364;&#339;protein knockout&#226;&#8364;? method demon- <br />                                                                              18, 41). <br /> strating essentiality bacterial proteins requires no <br />                                                                                 In study, showed protein-specific peptides can <br />                                                                              selected vitro used assess essentiality <br />   * Corresponding author. Mailing address: Karo Bio USA, Inc., 4222 <br />                                                                              wide variety proteins bacteria. We demonstrated this <br /> Emperor Blvd., Suite 560, Durham, NC 27703. Phone: (919) 474-8888            method proteins known essential E. coli, includ- <br /> ext. 13. Fax: (919) 474-0103. E-mail: paul.hamilton@karobio.com.             ing involved nearly basic cellular process: DNA <br />  <br />                                                                       2875 <br />  2876       BENSON ET AL.                                                                                                     ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> replication (DnaN), lipid A biosynthesis (LpxA), transcription                          Phage display. Immulon 4 96-well plates (Thermo Labsystems, Franklin, <br /> initiation (RpoD &#226;?&#180;70), translation (ProRS), protein secre-                        Mass.) coated streptavidin (1 &#226;?&#174;g/well) 0.1 M sodium bicarbonate <br />                                                                                      blocked 1% bovine serum albumin. Biotinylated protein (4 pmol/well) <br /> tion (SecA), DNA supercoiling (GyrA),                             diluted TBST (10 mM Tris-HCl [pH 8.0], 150 mM NaCl, 0.05% Tween 20) <br /> essential protein unknown function (Era). We assessed                        allowed bind plate 1 h. The plates washed TBST, and <br /> the specificity intracellular protein-peptide interaction                     phage 20 recombinant M13 libraries (total complexity 2 &#226;&#171;&#187; 1010) <br /> by performing protein overexpression rescue experiments.                             allowed bind 1 h. The plates washed again, bound phage <br />                                                                                      eluted, amplified, used round phage selection de- <br />                                                                                      scribed previously (44). After rounds selection, phage pools were <br />                         MATERIALS AND METHODS                                        plated single plaques purified DNA sequence individual <br />    Bacterial strains media. For work, <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> medium 2&#226;&#171;&#187;YT               isolates determined. Peptides synthesized include 12- 15- <br /> supplemented appropriate antibiotics chloramphenicol (<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> 30            amino-acid sequence displayed phage followed short linker a <br /> &#226;?&#174;g/ml), spectinomycin (to 100 &#226;?&#174;g/ml), and/or kanamycin (to 30 &#226;?&#174;g/ml). E. coli        biotin C terminus allow labeling analysis peptide binding. Isola- <br /> DH5&#226;?&#163;Pro purchased BD Biosciences/Clontech (Palo Alto, Calif.). E.           tion use ProRS-specific phage described previously (24, 46); <br /> coli BL21(DE3) cells electrocompetent JS5 cells purchased Bio-         peptide Pro-3 publications referred peptide 72 work. <br /> Rad (Hercules, Calif.). Other electrocompetent cells, needed, pre-            Cloning vectors intracellular expression peptide fusion protein. <br /> pared follows. The cells grown overnight 37&#194;&#176;C 20 ml             Two commercially available plasmids altered cloning purposes. <br /> appropriate medium, diluted 1 liter, grown optical density 0.5        (i) Vector peptide expression. Plasmid pPROTet.E332-lacZ modified <br /> at wavelength 600 nm. The cells pelleted washed ice-cold sterile    cloning peptide sequences GST fusions introducing cassette <br /> 10% glycerol volumes 1 liter, 500 ml, 250 ml, 125 ml, finally 5 ml. They   SalI NheI sites peptide sequence cloning replacing lacZ <br /> were frozen &#226;&#171;&#186;80&#194;&#176;C.                                                                gene gst gene. Oligonucleotides ProtetSal/Nhe5 ProtetSal/Nhe3 <br />    DNA manipulations synthesis oligonucleotide peptides. PCR          annealed 95&#194;&#176;C 2 min 1&#226;&#171;&#187; SURE/CUT buffer B (Roche) then <br /> carried using Expand High-Fidelity PCR (Roche, Indianapolis,          allowed cool room temperature. The DNA cut HindIII and <br /> Ind.) conjunction Techne Genius thermal cycler (Techne, Princeton,       KpnI ligated pPROTet.E332-lacZ vector previously cut HindIII <br /> N.J.). Restriction enzymes T4 DNA ligase New England Biolabs           KpnI. Transformants screened presence NheI site, and <br /> (NEB; Beverly, Mass.). DNA purified using kits purchased Qiagen             DNA sequences plasmids confirmed. The gst gene amplified by <br /> (Valencia, Calif.). Electroporation carried 0.1-cm cuvettes Bio-     PCR pGEX2T (Amersham BioSciences, Piscataway, N.J.) primers <br /> Rad E. coli Pulser device 1.8 kV. DNA sequence analysis carried        tGST5 tGST3. This set primers introduced linker region between <br /> Sequetech Corp. (Mountain View, Calif.). Oligonucleotides synthesized        multiple-cloning site gst. The PCR product cut HindIII PstI <br /> The Midland Certified Reagent Co., Inc. (Midland, Tex.). Peptides synthe-       ligated modified pPROTet.E332-lacZ vector cut same <br /> sized AnaSpec, Inc. (San Jose, Calif.).                                           enzymes. Transformants screened presence introduced BamHI <br />    Oligonucleotides. Two oligonucleotides annealed used make           site, plasmids confirmed sequence analysis. The resulting plasmid is <br /> cassette introduce SalI NheI sites pPROTet.E332-lacZ (BD Bio-            pKBU-Tet/GST (Fig. 1A). <br /> sciences/Clontech; below). They ProtetSal/Nhe5 ProtetSal/Nhe3,              (ii) Vector target protein expression. Plasmid pProLar.A222 (Clontech) <br /> respectively (5&#226;&#172;&#732;-GAGAAAGGTACCCATGTCGACAGGTTAGGCTAGCAC                                modified introducing cassette NdeI, BspHI, SmaI cloning sites <br /> AAGCTTGGATACT 5&#226;&#172;&#732;-AGTATCCAAGCTTGTGCTAGCCTAACCTG                                   KpnI PacI sites create pKBU-Pro-Lar (Fig. 1B). <br /> TCGACATGGGTACCTTTCTC). Two oligonuclcotides annealed                           Construction plasmids peptide expression. Each peptide expression <br /> used make cassette introduce new multicloning site pPROLar.A222.      plasmid (with exception ProRS peptide-encoding plasmid) made <br /> They pKBU-Pro-Lar5 pKBU-Pro-Lar3, respectively (5&#226;&#172;&#732;-CATATGCTC                 specific pair 5&#226;&#172;&#732;-phosphorylated oligonucleotides insertion the <br /> ATGAGGTAATAACCCGGGCGTTAAT 5&#226;&#172;&#732;-TAACGCCCGGGTTATTA                                   SalI NheI cloning sites pKBU-Tet/GST (Fig. 1A). Each oligonucleotide <br /> CCTCATGAGCATATGGTAC). The PCR primers used clone GST (and                       resuspended 1 &#226;?&#174;g/&#226;?&#174;l 10 mM Tris-HCl (pH 8.0)&#226;&#8364;&#8220;1 mM EDTA (TE), and <br /> hinge) pPROTet.E332 (Hind3-PstI) primers tGST5 tGST3, respec-           9.5 &#226;?&#174;l sense oligonucleotide 9.5 &#226;?&#174;l antisense oligonucleotide <br /> tively (5&#226;&#172;&#732;-GATCGATCAAGCTTGGGATCAGGATCCATGTCCCCTATACT                                 mixed 1 &#226;?&#174;l 5M NaCl TE placed heat block 80&#194;&#176;C. After <br /> AGGTTATTGG 5&#226;&#172;&#732;-GATCGATCCTGCAGCCTCACAGATCCGATTTT                                   30 min, block removed heater allowed cool 30&#194;&#176;C. Each <br /> GGAGGATGGTCGCC).                                                                     annealed cassette diluted TE 200 &#226;?&#174;l ligated pKBU- <br />    To clone target genes pTYB2AT pTYB4AT (see below), following          Tet/GST; 2 &#226;?&#174;l ligation mixture transformed 20 &#226;?&#174;l electro- <br /> pairs PCR primers used, set named protein     competent DH5&#226;?&#163;Pro cells. Cells plated medium containing chloram- <br /> the end gene hybridize (5 &#226;&#171;&#189; 5&#226;&#172;&#732;; 3 &#226;&#171;&#189; 3&#226;&#172;&#732;): LpxA5 (5&#226;&#172;&#732;-ATG          phenicol grown overnight 37&#194;&#176;C. Several isolates ligation were <br /> ATCGACATATGATTGATAAATCCGCCTTTGTGC) LpxA3 (5&#226;&#172;&#732;-GATC                                picked cultured, insert-positive isolates identified PCR ampli- <br /> GACCCGGGACGAATCAGACCGCGCGTTGAGCG), DnaN5 (5&#226;&#172;&#732;-TCGTCA                                  fication primers outside insert region. Positive isolates se- <br /> TCCATATGAAATTTACCGTAGAACGTGAGCA) DnaN3 (5&#226;&#172;&#732;-TCGTC                                 quenced oligonucleotide 5&#226;&#172;&#732;ProTet confirm correct sequences the <br /> ATCCCCGGGCAGTCTCATTGGCATGACAACATAA), RpoD5 (5&#226;&#172;&#732;-ATGA                                  peptide-coding inserts. <br /> TCGACATATGGAGCAAAACCCGCAGTCACAGC) RpoD3 (5&#226;&#172;&#732;-GATC                                    Construction plasmids protein expression. For intracellular expression <br /> GACCCGGGATCGTCCAGGAAGCTACGCAGC), Era5 (5&#226;&#172;&#732;-GATCGTACC                                  experiments, target genes removed pTYB2AT pTYB4AT <br /> ATGGGAAGCATCGATAAAAGTTACTGCGG) Era3 (5&#226;&#172;&#732;-GATCGACC                                 constructs used protein production restriction sites used clone <br /> CGGGAAGATCGTCAACGTAACCGAGACTGCG), GyrA5 (5&#226;&#172;&#732;-TCGTCAT                                  (or flanking sites) recloned compatible sites vector <br /> CCATATGAGCGACCTTGCGAGAGAAATTAC) GyrA3 (5&#226;&#172;&#732;-TCGTCA                                 pKBU-Pro-Lar (Fig. 1B). The resulting constructs transformed JS5 <br /> TCGATATCTTCTTCTTCTGGCTCGTCGTCAACG), SecA5 (5&#226;&#172;&#732;-TCGTC                              cells, sequences confirmed, plasmids repurified trans- <br /> ATCCATATGCTAATCAAATTGTTAACTAAAG) SecA3 (5&#226;&#172;&#732;-TCGTCA                                formed DH5&#226;?&#163;Pro cells. <br /> TCCCCGGGTTGCAGGCGGCCATGGCACTGCTTG).                                                     Induction peptide fusions. Overnight cultures sequence-confirmed con- <br />    The sequencing primer used confirm cloning peptides 5&#226;&#172;&#732;ProTet            structs diluted 1:100 (10 &#226;?&#174;l 1 ml) 96-well block fresh <br /> (5&#226;&#172;&#732;-CATCAGCAGGACGCACTGAC). The sequencing primer used confirm                     selective medium addition inducer anhydrotetracycline (BD Bio- <br /> cloning target genes pKBU-Pro-Lar 5&#226;&#172;&#732;ProLar (5&#226;&#172;&#732;-GTGAGCGCTC                sciences/Clontech) 100 ng/ml. These cultures grown 22&#194;&#176;C shaking, <br /> ACAATTATGATAG).                                                                      optical densities (absorbance 600 nm) read SpectraMax 190 <br />    Cloning target genes protein purification. Genes encoding target pro-      spectrophotometric plate reader (Molecular Devices, Sunnyvale, Calif.) 12, 24, <br /> teins amplified PCR specific oligonucleotides. The amplified prod-      36, 48 h. <br /> ucts cloned pTYB2AT pTYB4AT, vectors based TYB2                     Coinduction peptide fusions target proteins. Cells bearing pKBU-Tet- <br /> TYB4 (NEB), respectively, altered include 15-amino-acid vivo biotinyla-      peptide-GST constructs electrocompetent transformed with <br /> tion signal C terminus purified protein. Each construct trans-     pKBU-Pro-Lar-target constructs, plated 2&#226;&#171;&#187; YT plus kanamycin, and <br /> formed BL21(DE3) cells (Stratagene, La Jolla, Calif.) carrying pBirAcm      grown overnight 37&#194;&#176;C. Isolates picked grown medium chlor- <br /> plasmid (Avidity, Denver, Colo.) encodes biotin ligase. Plasmid DNA         amphenicol, spectinomycin, kanamycin, presence constructs <br /> isolated, inserts confirmed DNA sequencing. Protein pro-         cells confirmed sequencing DNA individual sets of <br /> duced purified NEB IMPACT method.                                         plasmid-specific primers. Overnight cultures cells bearing plasmids were <br />  VOL. 47, 2003                                                                          PROTEIN KNOCKOUT BY PEPTIDE EXPRESSION                                  2877 <br />  <br />                                                                                               TABLE 1. Amino acid sequences peptides described <br />                                                                                                                report <br />                                                                                      Peptide                 E. coli <br />                                                                                                                                          Amino acid sequence <br />                                                                                      ID no.                  target <br />  <br />                                                                                       72a                   ProRS                 SREWHFWRDYNPTSR <br />                                                                                       729                   RpoD                  SRSFSEWREEWLRSR <br />                                                                                       920                   LpxA                  SSGWMLDPIAGKWSR <br />                                                                                       1330                  SecA                  SSFSFDGHVWYPVLPAQSR <br />                                                                                       1337                  DnaN                  SSATRGWHQLDLFSR <br />                                                                                       1396                  GyrA                  SSGWPQWFPGVGWSR <br />                                                                                       1706                  Era                   SSEAWRKWLSADFSR <br />                                                                                       RND                   Control               SREWDRLWHPWLQSR <br />                                                                                        a <br />                                                                                            Peptide 72 peptide Pro-3 references 24 46. <br />  <br />  <br />  <br />  <br />                                                                                      proteins electroblotted polyvinylidene difluoride membranes probed <br />                                                                                      anti-GST antibody covalently linked alkaline phosphatase (Santa Cruz <br />                                                                                      Biotechnology, Santa Cruz, Calif.). The blots developed WesternBlue <br />                                                                                      substrate (Promega Corp., Madison, Wis.). <br />  <br />  <br />                                                                                                                        RESULTS <br />                                                                                         Isolation peptides bind specifically essential E. coli <br />                                                                                      proteins. The genes encoding DnaN, LpxA, RpoD, ProRS, <br />                                                                                      SecA, GyrA, Era essential E. coli (2, 16, 29, 33, <br />                                                                                      35, 36, 40). These genes amplified PCR E. coli <br />                                                                                      strain K-12 genomic DNA cloned expression vectors, <br />                                                                                      DNA sequences confirmed, proteins were <br />                                                                                      expressed purified. Each biotinylated target protein was <br />                                                                                      immobilized individual wells streptavidin-coated plastic <br />                                                                                      96-well plate. Recombinant M13 phage libraries a <br />                                                                                      total expected diversity 2 &#226;&#171;&#187; 1010 peptides added the <br />                                                                                      plate isolate binding phage. The selections carried all <br />                                                                                      bacterial proteins yielded phage particles bound <br />                                                                                      specifically protein irrele- <br />                                                                                      vant control proteins. Replicative intermediate DNA <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                      phage sequenced determine amino acid se- <br />                                                                                      quences peptides displayed phage. Correlations <br />    FIG 1. Expression vectors. (A) Peptide intracellular expression vec-              drawn amino acid sequences strong <br /> tor pKBU-Tet/GST based Clontech pPROTet.E332-lacZ                          binding target (as assessed relative phage affinity exper- <br /> vector. SalI NheI sites introduced cassette,              iments [data shown]), representative peptides were <br /> gene GST 5&#226;&#172;&#732; hinge region inserted HindIII                chosen synthesis. These synthetic peptides character- <br /> PstI sites. Subsequently, peptides cloned SalI NheI <br /> sites. (B) Protein intracellular expression vector pKBU-Pro-Lar                   ized relative binding cross-competition (data not <br /> based Clontech pPROLar.A222 vector. NdeI, BspHI, SmaI                     shown) confirm binding specificity. Peptides identified <br /> sites introduced cassette cloned KpnI PacI sites.             strong, specific binders expressed intracellularly <br /> Subsequently, target genes cloned NdeI                  GST fusions. Sequences peptides described this <br /> BspHI SmaI sites. RBS, ribosome binding site. <br />                                                                                      report shown Table 1. <br />                                                                                         Intracellular expression peptide-GST fusions inhibits <br />                                                                                      bacterial growth. Oligonucleotide cassettes designed en- <br /> diluted 1:100 (10 &#226;?&#174;l 1 ml) 96-well block, fresh selective medium <br /> containing inducer peptide expression, anhydrotetracyline (100 ng/ml), as <br />                                                                                      coding amino acid sequences protein-specific pep- <br /> well inducers protein expression, L-(&#226;&#171;&#185;)-arabinose (0.2 %) isopro-      tides isolated phage display. A random control peptide <br /> pyl-1-thio-&#226;?&#164;-D-galactopyranoside (1 mM). Induced cultures grown 22&#194;&#176;C,        designed general amino acid composi- <br /> and optical densities read 48 h.                                          tion growth-inhibitory peptides different primary <br />    Cultures diluted 1:1,000 3 h induction; plated 2&#226;&#171;&#187; YT <br />                                                                                      sequence. Oligonucleotide cassettes cloned ex- <br /> solid medium containing chloramphenicol, spectinomycin, kanamycin, in- <br /> ducers; grown overnight 37&#194;&#176;C.                                                 pression vector pKBU-Tet/GST. Cells containing con- <br />    Western blotting. Equal volumes overnight cultures cells carrying pKBU-     struct grown overnight 37&#194;&#176;C diluted 1:100 in <br /> Tet-peptide-GST constructs diluted fresh selective medium containing       fresh liquid media inducer anhydrotetra- <br /> inducer grown 37&#194;&#176;C. At 0, 1, 2, 3, 4 h, optical densities measured   cycline. These cultures grown 22&#194;&#176;C, optical <br /> and samples taken. Cell samples resuspended concentration 1 <br /> absorbance <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> 600 nm ml SDS sample buffer (3% sodium dodecyl <br />                                                                                      densities tracked 48 hours. Expression target- <br /> sulfate, 1 mM dithiothreitol, 10 % glycerol, 63 mM Tris-HCl [pH 6.8]), boiled    specific peptide-GST fusion inhibit cell growth (Fig. 2). <br /> 2 min, applied 4 20% gradient polyacrylamide gels. The resolved            The expression protein-specific peptide-GST fusions re- <br />  2878      BENSON ET AL.                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                                                            LpxA DnaN subcloned expression vector <br />                                                                            pKBU/ProLar. Each new construct transformed cells <br />                                                                            containing peptide-GST fusion constructs 920- <br />                                                                            GST 1337-GST. Plasmid DNAs isolated transfor- <br />                                                                            mants sequenced, confirmed presence both <br />                                                                            constructs (a peptide-GST fusion plasmid target plas- <br />                                                                            mid) cell strain generated. <br />                                                                               The peptide-GST fusion target protein in- <br />                                                                            duced, individually simultaneously, growth in <br />                                                                            liquid culture 22&#194;&#176;C tracked 48 h. Induction LpxA <br />                                                                            little effect cell growth, induction DnaN <br />                                                                            slight inhibitory effect cell growth. Overexpres- <br />   FIG. 2. Intracellular peptide expression. In presence the <br /> inducer anhydrotetracycline, peptide specific essential cellular    sion peptide fusions 920-GST 1337-GST resulted <br /> protein produced inside cell N-terminal fusion GST.        significant growth inhibition. However, cells expressing <br /> Specific peptide binding target protein inhibits essential      target protein peptide fusion, peptide-dependent <br /> function causes growth inhibition. <br />                                                                            growth inhibition fully partially relieved, only <br />                                                                            target-specific manner (Fig. 4). These experiments were <br /> sulted various degrees inhibition; representative data           repeated times similar results. The effects <br /> peptides inhibitory effects shown Fig. 3.             seen plated cultures (Fig. 5). Plate B shows growth <br /> These included peptides 729 (RpoD), 1337 (DnaN), 72                        inhibition cells induction peptides. Plate D shows <br /> (ProRS), 1706 (Era), 920 (LpxA), 1396 (GyrA), 1330                     induction peptides targets reveals specificity of <br /> (SecA). The random peptide did inhibit growth,                 rescue: overexpression LpxA rescues growth inhibition <br /> shown Fig. 3. Western blotting cells soon             caused cognate peptide 920 caused the <br /> induction peptide expression using anti-GST antibodies                  DnaN cognate peptide 1337. Similarly, overexpression of <br /> showed fusions expressed               DnaN specifically rescues growth inhibition caused cog- <br /> correlation growth inhibition level peptide                 nate peptide 1337 caused peptide 920. (Plate A <br /> expression (data shown).                                               shows uninduced cells, plate C shows induction target <br />   Expression specific target protein rescues peptide-depen-             alone). Overexpression SecA rescued growth inhibition <br /> dent growth inhibition. DNAs encoding target proteins                  caused SecA cognate peptide 1330; however, overexpres- <br />  <br />  <br />  <br />  <br />   FIG. 3. Intracellular peptide expression inhibits cell growth. Cells plasmid carrying inducible target-specific peptide fused GST <br /> were grown absence (striped bars) presence (solid bars) inducer; absorbance 600 nm read 44 48 h induction. <br /> Data expressed relative uninduced control given peptide, absorbance uninduced control 600 nm representing <br /> 100%. Peptide identification numbers given data set; peptide target given data set. The peptide RND a <br /> noninhibitory peptide, using amino acids generally like inhibitory peptides randomized primary sequence. Data shown are <br /> representative experiments. <br />  VOL. 47, 2003                                                               PROTEIN KNOCKOUT BY PEPTIDE EXPRESSION                        2879 <br />  <br />  <br />  <br />  <br />   FIG. 4. Specific rescue peptide-dependent growth inhibition. Dotted bars, protein (LpxA DnaN) induction; striped bars, peptide (920 or <br /> 1337) induction; cross-hatched bars: coinduction. Peptide 920 originally selected vitro LpxA; peptide 1337 selected DnaN. The <br /> absorbance cultures 600 nm read 48 h induction protein and/or peptide liquid medium. Data expressed relative those <br /> obtained uninduced control cultures grown parallel; error bars standard deviations. <br />  <br />  <br />                                                                           sion RpoD did rescue growth inhibition caused RpoD <br />                                                                           cognate peptide 729 (data shown). <br />  <br />                                                                                                     DISCUSSION <br />                                                                              We demonstrated protein knockout method iso- <br />                                                                           lating peptides phage display libraries specifically <br />                                                                           directed essential bacterial proteins showing that <br />                                                                           intracellular expression peptides inhibits bacterial <br />                                                                           growth. In contrast, intracellular expression peptides with <br />                                                                           amino acid composition scrambled sequence <br />                                                                           did cause inhibition growth. There correlation <br />                                                                           growth inhibition level expression tested <br />                                                                           peptide-GST fusions. Therefore, growth-inhibitory effects <br />                                                                           seen mere protein overexpression the <br />                                                                           amino acid composition fusions but, instead, require the <br />                                                                           specific amino acid sequence peptides identified via <br />                                                                           phage display. Furthermore, overexpression target protein <br />                                                                           relieved inhibition caused corresponding peptide but <br />                                                                           different peptide proteins tested (DnaN, <br />                                                                           LpxA, SecA). These findings support hypothesis that <br />                                                                           intracellular expression peptides directed essential bac- <br />                                                                           terial targets interfere cell growth. That approach <br />   FIG. 5. Specific rescue peptide-dependent growth inhibition.         successful proteins known essential predict <br /> Cultures plated selective medium containing appropriate in-       successfully identify essential proteins unknown <br /> ducers 3 h induction protein and/or peptide liquid medium     function encoded microbial genomes. <br /> and grown 16 h 37&#194;&#176;C. (A) No induction; (B) peptide               It surprising peptides selected bind specific <br /> induction alone; (C) protein induction alone; (D) peptide protein <br /> induction. Each quadrant labeled protein peptide, <br />                                                                           protein inhibit function protein intracellu- <br /> respectively, expression induced. Peptide 920 selected    larly. The phage display process selects peptides bind <br /> LpxA; peptide 1337 selected DnaN.                                  active functional sites random sites protein <br />  2880     BENSON ET AL.                                                                                          ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> surfaces (21, 26). Although phenomenon completely          genomics targets. Knowledge function lacking for <br /> understood, known active sites possess physicochem-         open reading frames (ORFs) discovered genome sequenc- <br /> ical properties deep, flexible clefts evolved     ing, 20% ORFs genome well-studied <br /> bind ligands (28, 48). In addition, peptide binding           bacterium E. coli known function (43). Another ad- <br /> clefts energetically favorable (11). Accordingly, activity      vantage method direct demonstration essenti- <br /> of enzymes inhibited peptides vitro,        ality protein level&#226;&#8364;&#8221;the level antimicrobial <br /> inhibition quite specific. Enzymes inhibited way in-        drug act. Once essentiality protein demonstrated, <br /> clude prolyl- tyrosyl-tRNA synthetases (24), &#226;?&#164;-glucosidase         method used genetic tool acquire <br /> (24), hexokinase (24), glycogen phosphorylase (24), alcohol          functional information. For example, extragenic suppressors of <br /> dehydrogenase (24), enzyme I phosphotransferase sys-            peptide-induced death isolated. Function also <br /> tem (31), rRNA methyltransferase (19), human immunodefi-               addressed use transcription profiling proteom- <br /> ciency virus HIV integrase (37), membrane dipeptidase (38),            ics approaches known protein question is <br /> NADPH-dependent oxidase (12), protein phosphatase I (49),              essential. <br /> Src kinase (34), urease (23), diphosphocytidine methyl-               Peptides shown knock function specific protein <br /> erythritol synthase (A. Jacobi, A., H. Baum, T. Nakano, R. R.          intracellularly used tools drug screening (6). <br /> Annand, S. Anderson, J. Breunig, S. Bowes, P. Goldenblatt,             For example, peptide competed target <br /> S. S. Ashraf, H. Gr&#195;&#184;n, P. Hamilton, D. J. Christensen, H.          protein small molecules suitable drug development. In <br /> Loferer, Program Abstr. 41st Intersci. Conf. Antimicrob.               instance, peptide specifically binds E. coli Dxr was <br /> Agents Chemother., abstr. 2124, 2001). Biologically relevant           used high-throughput &#226;&#8364;&#339;surrogate ligand&#226;&#8364;? displacement <br /> associations proteins peptides              assay: 32,000 compounds diverse historical library were <br /> seen crystal structures. Peptides selected phage display      screened way, 30 compounds identified that <br /> the DM2 oncoprotein resembled portion p53 sub-              inhibited Dxr activity 50% inhibitory concentrations un- <br /> strate seen crystal structure bind DM2 binding        der 20 &#226;?&#174;M (20). This kind peptide competition assay has <br /> pocket (3, 4, 27). A peptide selected phage display         used format high-throughput screens diffi- <br /> another target inhibited target&#226;&#8364;&#8482;s biochemical ac-        cult-to-assay targets, DnaN, sliding-clamp subunit <br /> tivity target&#226;&#8364;&#8482;s active site cocrystal   DNA polymerase III (D. J. Christensen, E. B. Gottlin, R. E. <br /> (P. Bernasconi, Abstr. HTS Appl. Agribusiness, SBS North               Benson, P. T. Hamilton, Program Abstr. 41st Intersci. <br /> Carolina Regional Meeting, 2002). These examples make clear            Conf. Antimicrob. Agents Chemother., abstr. 2131, 2001.) The <br /> that peptides bind active sites inhibit function         format used discover drugs directed to <br /> vitro inhibit function intracellularly.            targets unknown function. For purpose, peptide and <br />    However, difficult draw conclusions intracellular       target validated drug discovery the <br /> peptide expression does lead growth inhibition. We              results peptide overexpression cell growth. <br /> obtained result peptides isolated proteins <br /> dihydrofolate reductase (DHFR) deoxyxylulose reductoi-                                       ACKNOWLEDGMENTS <br /> somerase (Dxr). Mutants defective folA (encoding DHFR)                We thank Erin Anderson, Kellie Duke, E. Sturgis Payne for <br /> are slow growing rich media (22), mutants defective          excellent technical assistance. <br /> yaeM (encoding Dxr) viable rich media unless spe-             This work supported National Institutes Health <br /> cifically supplemented methylerythritol (45). In             grant 5 R43 AI 46036-02. <br /> cases, peptides isolated phage display competed                                          REFERENCES <br /> from protein antimicrobial drugs specific pro-          1. Blum, J. H., S. L. Dove, A. Hochschild, J. J. Mekalanos. 2000. Isolation <br /> tein; additionally, Dxr-specific peptides inhibited             peptide aptamers inhibit intracellular processes. Proc. Natl. Acad. Sci. <br /> the biochemical activity Dxr (reference 20 data                 USA 97:2241&#226;&#8364;&#8220;2246. <br />                                                                         2. Bohman, K., L. A. Isaksson. 1980. A temperature-sensitive mutant in <br /> shown). When growth inhibition intracellular expression              prolinyl-tRNA ligase Escherichia coli K-12. Mol. Gen. Genet. 177:603&#226;&#8364;&#8220; <br /> such peptides seen, number possible                  605. <br /> explanations. These include nonessentiality protein,             3. Bottger, A., V. Bottger, C. Garcia-Echeverria, P. Chene, H. K. Hochkeppel, <br />                                                                            W. Sampson, K. Ang, S. F. Howard, S. M. Picksley, D. P. Lane. 1997. <br /> binding peptide site relevant essential             Molecular characterization hdm2-p53 interaction. J. Mol. Biol. 269: <br /> function protein, protection protein peptide            744&#226;&#8364;&#8220;756. <br />                                                                         4. Bottger, V., A. Bottger, S. F. Howard, S. M. Picksley, P. Chene, C. Garcia- <br /> inhibition substrate assembly multienzyme com-               Echeverria, H. K. Hochkeppel, D. P. Lane. 1996. Identification novel <br /> plexes, insufficient expression peptide. Furthermore,           mdm2 binding peptides phage display. Oncogene 13:2141&#226;&#8364;&#8220;2147. <br /> even intracellular peptide-protein complexes efficiently         5. Caponigro, G., M. R. Abedi, A. P. Hurlburt, A. Maxfield, W. Judd, A. <br />                                                                            Kamb. 1998. Transdominant genetic analysis growth control pathway. <br /> formed, low level residual activity unbound protein              Proc. Natl. Acad. Sci. USA 95:7508&#226;&#8364;&#8220;7513. <br /> may suffice growth. Any factors cause           6. Christensen, D. J., E. B. Gottlin, R. E. Benson, P. T. Hamilton. 2001. <br /> incomplete growth inhibition, seen present study,            Phage display target-based antibacterial drug discovery. Drug Discovery <br />                                                                            Today 6:721&#226;&#8364;&#8220;727. <br /> example, SecA peptide. If sufficient numbers pep-           7. Cohen, B. A., P. Colas, R. Brent. 1998. An artificial cell-cycle inhibitor <br /> tides tested, ambiguity resolved given            isolated combinatorial library. Proc. Natl. Acad. Sci. USA 95:14272&#226;&#8364;&#8220; <br />                                                                            14277. <br /> protein. This technique useful clear growth         8. Cohen, M. L. 2000. Changing patterns infectious disease. Nature 406:762&#226;&#8364;&#8220; <br /> inhibition phenotype observed minimal number                  767. <br /> peptides protein-specific rescue demonstrated.              9. Colas, P., B. Cohen, T. Jessen, I. Grishina, J. McCoy, R. Brent. 1996. <br />                                                                            Genetic selection peptide aptamers recognize inhibit cyclin- <br />    The protein knockout method does depend prior                    dependent kinase 2. Nature 380:548&#226;&#8364;&#8220;550. <br /> knowledge function suited use             10. Coleman, K., M. Athalye, A. Clancey, M. Davison, D. J. Payne, C. R. Perry, <br />  VOL. 47, 2003                                                                               PROTEIN KNOCKOUT BY PEPTIDE EXPRESSION                                      2881 <br />  <br />       I. Chopra. 1994. Bacterial resistance mechanisms therapeutic targets.        31. Mukhija, S., B. Erni. 1997. Phage display selection peptides against <br />       J. Antimicrob. Chemother. 33:1091&#226;&#8364;&#8220;1116.                                                 enzyme I phosphoenolpyruvate-sugar phosphotransferase system <br /> 11.   Conte, L. L., C. Chothia, J. Janin. 1999. The atomic structure protein-          (PTS). Mol. Microbiol. 25:1159&#226;&#8364;&#8220;1166. <br />       protein recognition sites. J. Mol. Biol. 285:2177&#226;&#8364;&#8220;2198.                             32. Mushegian, A. R., E. V. Koonin. 1996. A minimal gene set cellular <br /> 12.   DeLeo, F. R., L. Yu, J. B. Burritt, L. R. Loetterle, C. W. Bond, A. J. Jesaitis,        life derived comparison complete bacterial genomes. Proc. Natl. Acad. <br />       M. T. Quinn. 1995. Mapping sites interaction p47-phox fla-                Sci. USA 93:10268&#226;&#8364;&#8220;10273. <br />       vocytochrome b random-sequence peptide phage display libraries. Proc.          33. Nakamura, Y. 1978. RNA polymerase mutant altered sigma factor in <br />       Natl. Acad. Sci. USA 192:7110&#226;&#8364;&#8220;7114.                                                     Escherichia coli. Mol. Gen. Genet. 165:1&#226;&#8364;&#8220;6. <br /> 13.   Fabbrizio, E., L. Le Cam, J. Polanowska, M. Kaczorek, N. Lamb, R. Brent,            34. Nishi, T., R. J. Budde, J. S. McMurray, N. U. Obeyesekere, N. Safdar, V. A. <br />       C. Sardet. 1999. Inhibition mammalian cell proliferation geneti-              Levin., H. Saya. 1996. Tight-binding inhibitory sequences against <br />       cally selected peptide aptamers functionally antagonize E2F activity.              pp60(c-src) identified using random 15-amino-acid peptide library. FEBS <br />       Oncogene 18:4357&#226;&#8364;&#8220;4363.                                                                  Lett. 399:237&#226;&#8364;&#8220;240. <br /> 14.   Fauci, A. 2001. Infectious diseases: considerations 21st century. Clin.     35. Oliver, D. B., J. Beckwith. 1982. Regulation membrane component <br />       Infect. Dis. 32:675&#226;&#8364;&#8220;685.                                                                required protein secretion Escherichia coli. Cell 30:311&#226;&#8364;&#8220;319. <br /> 15.   Fraser, C. M., J. A. Eisen, S. L. Salzberg. 2000. Microbial genome              36. Oram, M., R. Kuroda, L. M. Fisher. 1992. Escherichia coli DNA gyrase: <br />       sequencing. Nature 406:799&#226;&#8364;&#8220;803.                                                         genetic analysis gyrA gyrB mutations responsible thermosensitive <br /> 16.   Galloway, S. M., C. R. Raetz. 1990. A mutant Escherichia coli defec-             enzyme activity. FEBS Lett. 312:61&#226;&#8364;&#8220;65. <br />       tive step endotoxin biosynthesis. J. Biol. Chem. 265:6394&#226;&#8364;&#8220;6402.     37. Puras Lutzke, R. A., N. A. Eppens, P. A. Weber, R. A. Houghten, R. H. <br /> 17.   Geyer, C. R., R. Brent. 2000. Selection genetic agents random               Plasterk. 1995. Identification hexapeptide inhibitor human im- <br />       peptide aptamer expression libraries. Methods Enzymol. 328:171&#226;&#8364;&#8220;208.                     munodeficiency virus integrase protein using combinatorial chemical <br /> 18.   Geyer, C. R., A. Colman-Lerner, R. Brent. 1999. &#226;&#8364;&#339;Mutagenesis&#226;&#8364;?                    library. Proc. Natl. Acad. Sci. USA 92:11456&#226;&#8364;&#8220;11460. <br />       peptide aptamers identifies genetic network members pathway connec-             38. Rajotte, D., E. Ruoslahti. 1999. Membrane dipeptidase receptor <br />       tions. Proc. Natl. Acad. Sci. USA 96:8567&#226;&#8364;&#8220;8572.                                         lung-targeting peptide identified vivo phage display. J. Biol. Chem. <br /> 19.   Giannattasio, R. B., B. Weisblum. 2000. Modulation erm methyl-                   274:11593&#226;&#8364;&#8220;11598. <br />       transferase activity peptides derived phage display. Antimicrob.            39. Rosamond, J., A. Allsop. 2000. Harnessing power genome in <br />       Agents Chemother. 44:1961&#226;&#8364;&#8220;1963.                                                         search new antibiotics. Science 287:1973&#226;&#8364;&#8220;1976. <br /> 20.   Gottlin, E. B., R. E. Benson, S. Conary, B. Antonio, K. Duke, E. S. Payne,          40. Sakakibara, Y., T. Mizukami. 1980. A temperature-sensitive Escherichia <br />       S. A. Ashraf, D. J. Christensen. 2003. High throughput screen                   coli mutant defective DNA replication: dnaN, new gene adjacent the <br />       inhibitors 1-deoxy-d-xylulose 5-phosphate reductoisomerase surrogate              dnaA gene. Mol. Gen. Genet. 178:541&#226;&#8364;&#8220;553. <br />       ligand competition. J. Biomol. Screen. 8:332&#226;&#8364;&#8220;339.                                   41. Sandrock, T., M. Poritz, M. Kim, M. J. Feldhaus, B. Roth, G. Caponigro, <br /> 21.   Gr&#195;&#184;n, H., R. Hyde-DeRuyscher. 2000. Peptides tools drug discov-               A. Kamb. 2002. Expression levels transdominant peptides pro- <br />       ery. Curr. Opin. Drug Discovery Dev. 3:636&#226;&#8364;&#8220;645.                                         teins Saccharomyces cerevisiae. Yeast 19:1&#226;&#8364;&#8220;7. <br /> 22.   Herrington M. B., N. T. Chirwa. 1999. Growth properties folA null          42. Schmid, M. B. 1998. Novel approaches discovery antimicrobial <br />       mutant Escherichia coli K12. Can. J. Microbiol. 45:191&#226;&#8364;&#8220;200.                          agents. Curr. Opin. Chem. Biol. 2:529&#226;&#8364;&#8220;534. <br /> 23.   Houimel, M., J. P. Mach, I. Corthesy-Theulaz, B. Corthesy, I. Fisch.            43. Serres, M. H., S. Gopal, L. A. Nahum, P. Liang, T. Gaasterland, M. <br />       1999. New inhibitors Helicobacter pylori urease holoenzyme selected             Riley. 2001. A functional update Escherichia coli K-12 genome. Ge- <br />       phage-displayed peptide libraries. Eur. J. Biochem. 262:774&#226;&#8364;&#8220;780.                        nome Biol. 2 [Online.] http://genomebiology.com/2001/2/9/research/0035. <br /> 24.   Hyde-DeRuyscher, R., L. Paige, D. Christensen, N. Hyde-DeRuyscher, A.               44. Sparks, A. B., N. B. Adey, S. Cwirla, B. K. Kay. 1996. Screening phage- <br />       Lim, Z. Fredericks, J. Kranz, P. Gallant, J. S. Zhang, S. Rocklage, D.                  displayed random peptide libraries, p. 227&#226;&#8364;&#8220;253. In B. K. Kay, J. Winter, and <br />       Fowlkes, P. Wendler, P. Hamilton. 2000. Detection small-molecule                 J. McCafferty (ed.), Phage display peptides proteins, laboratory <br />       enzyme inhibitors peptides isolated phage-displayed combinato-                manual. Academic Press, Inc., San Diego, Calif. <br />       rial peptide libraries. Chem. Biol. 7:17&#226;&#8364;&#8220;25.                                        45. Takahashi, S., T. Kuzuyama, H. Watanabe, H. Seto. 1998. A 1-deoxy- <br /> 25.   Itaya, M. 1995. An estimation minimal genome size required life.                 D-xylulose 5-phosphate reductoisomerase catalyzing formation 2-C- <br />       FEBS Lett. 362:257&#226;&#8364;&#8220;260.                                                                 methyl-D-erythritol 4-phosphate alternative nonmevalonate pathway <br /> 26.   Kay, B. K., P. T. Hamilton. 2001. Identification enzyme inhibitors               terpenoid biosynthesis. Proc. Natl. Acad. Sci. USA 95:9879&#226;&#8364;&#8220;9884. <br />       phage-displayed combinatorial peptide libraries. Comb. Chem. High              46. Tao, J., P. Wendler, G. Connelly, A. Lim, J. Zhang, M. King, T. Li, J. A. <br />       Throughput Screen. 4:535&#226;&#8364;&#8220;543.                                                           Silverman, P. R. Schimmel, F. P. Tally. 2000. Drug target validation: <br /> 27.   Kussie, P. H., S. Gorina, V. Marechal, B. Elenbaas, J. Moreau, A. J. Levine,            lethal infection blocked inducible peptide. Proc. Natl. Acad. Sci. USA <br />       N. P. Pavletich. 1996. Structure MDM2 oncoprotein bound               97:783&#226;&#8364;&#8220;786. <br />       p53 tumor suppressor transactivation domain. Science 274:948&#226;&#8364;&#8220;953.                   47. Walker, J. R., J. R. Roth, E. Altman. 2001. An vivo study novel <br /> 28.   Laskowski, R. A., N. M. Luscombe, M. B. Swindells, J. M. Thornton.                  bioactive peptides inhibit growth Escherichia coli. J. Pept. Res. <br />       1996. Protein clefts molecular recognition function. Protein Sci.                58:380&#226;&#8364;&#8220;388. <br />       5:2438&#226;&#8364;&#8220;2452.                                         <br /> </body></html>