<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Oct. 1996, p. 5712&#226;&#8364;&#8220;5718                                                                            Vol. 178, No. 19 <br /> 0021-9193/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />        Molecular Cloning Expression spsB Gene Encoding <br />       Essential Type I Signal Peptidase Staphylococcus aureus <br />                           KAREN M. CREGG, E. IMOGEN WILDING,                        AND   MICHAEL T. BLACK* <br />                  Biotechnology, SmithKline Beecham Pharmaceuticals, Epsom, Surrey KT18 5XQ, United Kingdom <br />                                               Received 6 June 1996/Accepted 29 July 1996 <br />  <br />            The gene, spsB, encoding type I signal peptidase cloned gram-positive eubacterium <br />          Staphylococcus aureus. The gene encodes protein 191 amino acid residues calculated molecular mass <br />          21,692 Da. Comparison protein sequence known type I signal peptidases indicates <br />          conservation amino acid residues known important essential catalytic activity. The enzyme has <br />          expressed high levels Escherichia coli demonstrated possess enzymatic activity <br />          E. coli preproteins vivo. Experiments spsB gene transferred plasmid is <br />          temperature sensitive replication indicate spsB essential gene. We identified open reading <br />          frame immediately upstream spsB gene encodes type I signal peptidase homolog 174 amino <br />          acid residues calculated molecular mass 20,146 Da predicted devoid catalytic activity. <br />  <br />  <br />    The majority proteins translocated             lepB single-copy gene E. coli chromosome. In con- <br /> more membranes site synthesis initially syn-                trast, species G1 Bacillus <br /> thesized N-terminal extension known signal,                 genus, B. subtilis B. amyloliquefaciens. It known that <br /> leader, peptide (42). Proteolytic cleavage signal se-                homologous SPase genes these <br /> quence yield mature protein occurs shortly                 Bacillus species. The sipS gene deleted chro- <br /> after translocation event catalyzed pro-                 mosome B. subtilis 168 affecting cell growth rate or <br /> karyotes eukaryotes enzymes known signal, leader,              viability laboratory conditions yield mutant strain <br /> peptidases (SPases). The bacterial SPases membrane pro-                 process pre-a-amylase a-amylase (4a). A re- <br /> teins consisting single polypeptide anchored mem-               cently reported putative SPase sequence (1) gene <br /> brane (gram-positive [G1] gram-negative <br />                                                                             product responsible activity, and/or B. subtilis may <br /> [G2] bacteria) (G2 bacteria) transmembrane sections. <br />                                                                             harbor SPase genes. Two genes en- <br /> Predicted amino acid sequences bacterial SPases a <br /> high level similarity known Escherichia coli (43),           coding distinct SPase homologs reside chromosome of <br /> Pseudomonas fluorescens (4), Salmonella typhimurium (39),                   closely related species B. amyloliquefaciens (12), there <br /> Haemophilus influenzae (8), Phormidium laminosum (25), Bra-                 evidence suggest B. japonicum possess than <br /> dyrhizobium japonicum (22), Rhodobacter capsulatus (17), Ba-                SPase (22, 23). Although SPase sequences seven <br /> cillus subtilis (two chromosomal plasmid origin [1,              genera G2 bacteria known, single Bacillus <br /> 21, 37]), Bacillus licheniformis (11), Bacillus caldolyticus (34),          genus G1 eubacteria investigated with <br /> and Bacillus amyloliquefaciens (two chromosomal genes) (12,                 respect SPase characteristics. It considered of <br /> 35), partial sequence reported Bacillus                  determine G1 eubacterium that, unlike <br /> pumilis (12). These enzymes collectively defined               B. subtilis B. amyloliquefaciens, known excep- <br /> type I signal peptidases (37). Although amino acid se-                  tional secretion activity genes encoding one <br /> quences 15 bacterial SPases (and 16th partial sequence)                SPase overlapping substrate specificities it <br /> have reported, best-studied examples leader                resembles E. coli H. influenzae (and possibly G2 <br /> peptidase (LPase LepB) E. coli SPase B.                 eubacteria) closely single SPase gene. The <br /> subtilis (SipS).                                                            recent publication entire genome sequence obli- <br />    It demonstrated LPase activity essential            gate G1-like intracellular bacterium Mycoplasma genitalium <br /> cell growth E. coli. Experiments expression               reveals interesting feature relating heterogeneity <br /> lepB gene, encoding LPase, regulated control-               SPases (9). Although homologs known <br /> lable ara promoter (5) partial deletion natural                components bacterial protein translocation machinery are <br /> promoter (6) indicated minimization LPase production <br />                                                                             present (secA, secY, etc.), reported gene <br /> was associated cessation cell growth division. In <br />                                                                             encoding type I SPase detected. This implies a <br /> addition, E. coli strain possessing mutated lepB gene (E. <br /> coli IT41) shown drastically reduced growth              non-type-I SPase exist organism. Finally, SPases <br /> rate display rapid pronounced accumulation pre-                (along components secretory apparatus) may <br /> proteins temperature growth medium ele-                  represent novel targets chemical intervention attempts <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> vated 428C (14). These results imply              derive new classes antibacterial agents clinical utility <br /> gene product E. coli substitute LPase              and, potent inhibitors E. coli LPase recently been <br />                                                                             discovered (2), determine type I <br />                                                                             SPases play essential role pathogenic G1 eubacterium. <br />   * Corresponding author. Mailing address: SmithKline Beecham               Experiments performed order clone the <br /> Pharmaceuticals, Yew Tree Bottom Road, Epsom, Surrey KT18 5XQ,              gene(s) encodes SPase <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span>, determine essen- <br /> United Kingdom. Phone: 44 1737 364000. Fax: 44 1737 364073. <br /> Electronic mail address: Mick_Black-1@sbphrd.com.                           tiality gene product in, Staphylococcus aureus. <br />  <br />                                                                      5712 <br />  VOL. 178, 1996                                                                                           SIGNAL PEPTIDASE FROM S. AUREUS                             5713 <br />  <br />  <br />                                                                 TABLE 1. Plasmids used study <br />                             Size <br />        Plasmid                                                                     Description                                                           Source <br />                            (kbp) <br />  <br /> pBluescriptII SK2           3.0      Phagemid derived pUC19; Ampr                                                                        Stratagene <br /> pGEM-T                      3.0      PCR cloning vector; Ampr                                                                                 Promega <br /> pKC09                       3.2      S. aureus Oxford PCR product (primers A 1 B) cloned pGEM-T                                          This work <br /> pKC10                      10.7      Excised clone S. aureus WCUH29 lZapII library; 7.7-kb insert containing spsA                        This work <br />                                         spsB <br /> pKC11                        7.5     Large fragment SalI-cut pKC10 religated form pKC11; 4.5-kb insert containing                       This work <br />                                         spsA spsB <br /> pDB575                       5.4     lacIq Ptac expression vector; Ampr                                                                       Gift M. Lawrence <br />                                                                                                                                                 D. Gale <br /> pKC16                        6.3     SpsB expression construct; spsB PCR product (primers C 1 D pKC11                                    This work <br />                                        template) cloned pDB575 <br /> pUC18                        2.7     E. coli plasmid; Ampr                                                                                    Stratagene <br /> pTS1                         6.9     E. coli-S. aureus shuttle vector; Ts G1 origin; Ampr Cmr                                                 Gift T. Foster <br /> pIM04                        4.4     Regions flanking spsB (850 bp each) inserted pUC18                                                  This work <br /> pIM05                        8.6     Regions flanking spsB (850 bp each) inserted pTS1                                                   This work <br /> pIM11                        9.2     pIM05 containing spsB                                                                                    This work <br />  <br />  <br />  <br />                          MATERIALS AND METHODS                                           min 608C, 2 min 728C, 5 min 728C; G plus H, 5 min 948C, 15 <br />                                                                                          cycles 1 min 948C, 1 min 458C, 2 min 728C, 5 min 728C; and <br />    Bacterial strains, plasmids, media. The following bacterial strains          I plus J, 5 min 948C, 30 cycles 1 min 948C, 1 min 428C, 3 min at <br /> used study: E. coli XL1-Blue, JM109, IT41 S. aureus RN4220, H            728C, 5 min 728C. <br /> (ATCC 13801), Oxford (ATCC 9144), WCUH29 (NCIMB 40771). S. aureus                       DNA Southern blotted 0.7% (wt/vol) agarose gels nylon mem- <br /> strains grown tryptone soya broth (Oxoid) Luria-Bertani broth (28).        branes (Hybond-N1; Amersham) instructed manufacturer. For library <br /> E. coli cultured Luria-Bertani broth. For solid medium, 1.5% (wt/vol) agar        screening purposes, plaques transferred nylon membranes (Hybond-N1; <br /> was added. Where appropriate, medium supplemented 100 mg                 Amersham) instructed manufacturer. Membranes hybridized with <br /> ampicillin ml E. coli 5 mg chloramphenicol ml S. aureus.          oligonucleotide gene probes labeled using ECL (enhanced <br /> The S. aureus WCUH29 genomic library constructed Stratagene                chemiluminescence) 39-oligolabeling kit ECL random prime labeling kit <br /> vector lZapII. The lZapII S. aureus H library constructed laboratory         (Amersham) appropriate. Washing detection steps performed as <br /> as instructed manufacturer lZapII (Stratagene). All shuttle vector             instructed manufacturer. All sequence data manipulation performed <br /> constructs transferred E. coli S. aureus WCUH29             Wisconsin package (7). <br /> used transform restriction-minus S. aureus strain RN4220, plasmid                Expression studies. Sodium dodecyl sulfate (SDS)-polyacrylamide gel electro- <br /> purified therefrom used transform S. aureus WCUH29. Plasmids used              phoresis (PAGE) carried essentially described Laemmli (19), using <br /> this study described Table 1.                                                     Mini Protean II apparatus (Bio-Rad). Cultures grown mid-log phase <br />    DNA techniques. All chemicals purchased Sigma unless              induced addition 1 mM isopropylthiogalactopyranoside (IPTG). <br /> specified. Plasmid DNA isolated using RPM kit (Bio 101 Inc.)            Cells harvested 2.5 h postinduction, total cell lysates prepared as <br /> Wizard Midiprep DNA purification (Promega). PCR products iso-                outlined Sambrook et al. (28), proteins separated electrophoresis <br /> lated horizontal agarose gel electrophoresis, treated Agarase (Boehr-            15% (wt/vol) acrylamide resolving gel. <br /> inger Mannheim), purified Wizard DNA Clean-Up (Pro-                     For complementation studies, E. coli IT41 transformed appropriate <br /> mega). Chromosomal DNA isolated E. coli S. aureus using                  plasmids, ampicillin-resistant transformants picked agar plates grown <br /> either genomic DNA purification kit (bacterial) (Advanced Genetic Technol-             288C incubated broth overnight permissive temperature 288C. <br /> ogies Corp.) published procedures (20). An incubation appropriate            Cultures diluted 100-fold fresh medium incubated shaking at <br /> quantity lysostaphin (Applied Microbiology Inc.) included              nonpermissive temperature 428C. The optical density 540 nm (OD540) <br /> preparation plasmid chromosomal DNA S. aureus facilitate cell             recorded 30-min intervals period 8 h. <br /> lysis. lZapII library clones excised recircularized form recombinant            Selection double-crossover plasmid integration/resolution events plas- <br /> phagemid instructed manufacturer. Procedures DNA restriction               mid curing. S. aureus WCUH29(pIM05) WCUH29(pIM11) grown <br /> and dephosphorylation, agarose gel electrophoresis, PCR, transformation           overnight 288C presence chloramphenicol. The cultures diluted <br /> competent E. coli cells performed essentially described Sambrook et           100-fold fresh medium chloramphenicol incubated 438C. <br /> al. (28). Electrocompetent S. aureus cells prepared described Schenk          Cultures grown 438C absence chloramphenicol 56 h to <br /> and Laddaga (29), following modifications: bacteria grown           reduce plasmid copy number, time cultures subinocu- <br /> tryptone soya broth washed 2500 3 g 5 min 208C. Competent cells            lated times (twice daily) fresh medium. Samples diluted and <br /> and plasmid DNA electroporated 1-mm-gap electroporation cuvettes              plated 288C presence (integration/resolution) absence (plasmid <br /> 208C, 100 V, 25 mF, 2.3 kV Gene Pulser apparatus pulse controller          curing) chloramphenicol. <br /> (Bio-Rad Laboratories Ltd.). Restriction enzymes calf intestine alkaline                Identification colonies plasmid-borne copy spsB derived the <br /> phosphatase purchased Promega, ligations performed                 chromosome. Colonies seeded 100 ml tryptone soya broth containing <br /> DNA ligation kit (Amersham) instructed manufacturer, sequencing            5 mg chloramphenicol ml wells microtiter plates incubated <br /> was performed using Sequenase version 2.0 DNA sequencing kit (Amersham).               overnight 288C (27). The microtiter plates centrifuged 1,000 3 g 10 <br /> PCRs, optimized using Opti-Prime kit (Stratagene), performed             20 min, supernatant discarded. Pellets resuspended 20 ml <br /> Hybaid OmniGene thermal cycler, using Pfu DNA polymerase (Strat-                  50 mM Tris-HCl (pH 7.5)&#226;&#8364;&#8220;145 mM NaCl containing 3 U lysostaphin and <br /> agene) Taq DNA polymerase (Promega), products cloned                  incubated 30 45 min 378C. Cell lysates (2 ml) spotted nylon <br /> using TA-Cloning kit (Invitrogen), pGEM-T cloning kit (Promega),                membranes (Hybond-N1; Amersham), DNA denatured fixed to <br /> DNA ligation kit (Amersham). Synthetic oligonucleotides Cru-                membrane instructed manufacturer. Membranes hybridized <br /> achem Ltd. R&amp;D Systems Europe Ltd. PCR primer sequences, inten-                  spsB gene downstream flanking region labeled using an <br /> tional differences template sequence shown uppercase,                ECL random prime labeling kit (Amersham) instructed manufacturer. <br /> follows: primer A, (g/a)tIgg(a/t)(c/t)tIcc(a/t)gg(a/t)ga(a/t)aI(a/t)(g/a)t; primer B,       Nucleotide sequence accession number. The <span id='am-1' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> as- <br /> c(t/g)(a/g)tt(a/g)tc(t/a)cccatIa(t/c)(a/g)aa(a/g)ta; primer C, tgGAaTTCAtgaaaaa          signed nucleotide sequence reported U65000. <br /> agaaCtGttggaatggattatttc; primer D, atttgtAAGCtTttaGtttttGgtGttttcaggattga <br /> aa; primer E, ctgGATCCcgcttgattagttttattga; primer F, ttGGtACCttttgacacctctt <br /> tttaag; primer G, aaGGtACCtatgaaacaaatacaacatc; primer H, atGAAtTCtcaata                                                   RESULTS <br /> taattgtgacactc; primer I, atattagagcgataattcc; primer J, gttcatttgctatttcttc. <br />    PCR cycle conditions primer pairs follows: A plus B, 5 min 948C, <br /> 30 cycles 1 min 948C, 1 min 428C, 1 min 728C, 5 min 728C; <br />                                                                                            DNA cloning. A strategy devised clone SPase PCR <br /> C plus D, 5 min 948C, 15 cycles 1 min 948C, 1 min 508C, 1 min         using primers A B (see Materials Methods). A frag- <br /> 728C, 5 min 728C; E plus F, 5 min 948C, 15 cycles 1 min 948C, 1          ment DNA expected <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, determined the <br />  5714       CREGG ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. DNA encoded protein sequences spsA spsB genes S. aureus WCUH29. Both strands entire sequence determined by <br /> oligonucleotide walking. Putative ribosome-binding sites gene underlined, potential rho-independent terminator sequence distal spsB gene. <br /> The active-site serine 36 lysine 77 residues boldface. <br />  <br />  <br />  <br />  <br /> known sizes SPases Bacillus species, obtained                       regions highest sequence conservation are, ex- <br /> a preparation genomic DNA S. aureus Oxford. The 163                          pected, concentrated close sections A C (Fig. 2) <br /> bp sequence derived highly homologous 163 bp                          primary structure. However, surprising observation the <br /> the sipS gene B. subtilis, implying fragment                        fact serine lysine residue known be <br /> gene encoding type I SPase cloned. The PCR                                essential catalytic activity type I SPases conserved in <br /> product labeled used probe lZapII library S.                         SpsA. It concluded SpsA, produced in <br /> aureus WCUH29 genomic DNA order obtain se-                                cell, catalyze hydrolysis peptide bonds to <br /> quence information. A positive clone identified,                         physiologically significant extent. To ascertain exis- <br /> recombinant plasmid (pKC10) excised, cut                            tence spsA peculiar WCUH29 strain S. <br /> restriction enzymes, blotted, probed labeled                       aureus, lZapII library DNA isolated S. aureus H was <br /> DNA fragment (data shown). A SalI-digested                             probed probe originally used identify clones con- <br /> pKC10 preparation religated form plasmid pKC11. The                           taining spsA spsB genes strain WCUH29. Close <br /> DNA sequence 3,093 bp insert DNA determined                             homologs spsA spsB discovered S. <br /> oligonucleotide walking directions                           aureus H; SpsA lacked active-site serine lysine resi- <br /> sequence originally derived PCR. Figure 1 shows 1,220                             dues (data shown). <br /> nucleotides DNA sequence chromosome S.                                   Expression spsB gene E. coli. Despite fact that <br /> aureus WCUH29, nucleotides 817 1025 represent                            significant expression levels active LPase lepB cloned <br /> the probe region. The probe sequence comprises                             plasmid obtained E. coli (5, 43), protein <br /> potential open reading frame entirety encodes                         functionally expressed B. subtilis. Although the <br /> polypeptide 191 amino acid residues calculated mo-                         gene transcribed translated, LPase did assemble <br /> lecular mass 21,692 Da single section hydro-                        correctly membrane detectable activity in <br /> phobic residues close N terminus probably form                         vitro assays using cell extracts B. subtilis producing E. coli <br /> single transmembrane anchor. The small surface-exposed do-                           LPase enzyme source (36). Conversely, functional ex- <br /> main single transmembrane anchor typical SPases                           pression B. subtilis sipS-encoded SPase E. coli has <br /> from G1 eubacteria. The predicted protein high homology                          demonstrated instrumental successful <br /> with known G1 SPases (data shown), gene                          cloning sipS gene (37). It clear work de- <br /> been named spsB (signal peptidase staphylococcus). It                        scribed SipS assembled correct orientation E. <br /> noteworthy highest levels sequence similarity                     coli membrane. However, activity monitored solely <br /> regions protein corresponding highly con-                         cleavage artificial engineered preprotein, property of <br /> served regions known SPases (sections A C Fig. 2). All <br /> three regions B. subtilis SPase, two <br /> within LPase E. coli, contain residue is <br /> critical catalytic activity (3, 33, 38). <br />    A second open reading frame, named spsA, proximal the <br /> spsB gene (separated 15 nucleotides) potentially encodes a <br /> protein 174 amino acid residues calculated molecular <br /> mass 20,146 Da. Sequence comparison reveals this <br />                                                                                         FIG. 2. Alignment sections primary structure signal peptidases <br /> protein similar known SPases. An optimized align-                         E. coli (LepB), B. subtilis (SipS), S. aureus (SpsB), SpsA. The active-site <br /> ment SpsB SpsA results 62% similarity 31%                             serine lysine residues conserved aspartate residue shown in <br /> identity sequences (data shown). The                             boldface, SpsA residues corresponding positions underlined. <br />  VOL. 178, 1996                                                                                        SIGNAL PEPTIDASE FROM S. AUREUS                          5715 <br />  <br />  <br />  <br />  <br />    FIG. 3. Expression S. aureus SpsB signal peptidase E. coli. Total cell <br /> proteins E. coli equivalent 175 ml cell culture OD540 1.0 were <br /> loaded SDS&#226;&#8364;&#8220;15% (wt/vol) polyacrylamide gel, electrophoresed 45 <br /> min 200 V, stained Coomassie brilliant blue R. Lanes 1 6, <br /> Bio-Rad low-molecular-mass markers (phosphorylase b, 97.4 kDa; serum albu- <br /> min, 66.2 kDa; ovalbumin, 45 kDa; carbonic anhydrase, 31 kDa; trypsin inhibitor, <br /> 21.5 kDa; lysozyme, 14.4 kDa); lane 2, pDB575 IPTG; lane 3, pKC16 <br /> without IPTG; lane 4, pDB575 IPTG; lane 5, pKC16 IPTG. <br />                                                                                          FIG. 4. Growth curves E. coli IT41(pKC16) (circles) E. coli <br />                                                                                       IT41(pDB575) (squares) 428C. Bacterial colonies seeded L broth <br />                                                                                       containing ampicillin grown 288C overnight. Cell cultures incubated <br /> which extension central hydrophobic region                           shaking 428C. OD540 recorded 30-min intervals. <br /> the signal sequence. It determine <br /> whether spsB gene functionally expressed E. <br /> coli SPase produced catalyze the <br /> cleavage signal sequences natural E. coli preproteins.                        similar observed E. coli IT41 transformed <br /> The spsB gene amplified plasmid pKC11 PCR                                 plasmid overexpresses authentic E. coli LPase and <br /> using primers C D containing mismatches template                           grown conditions (3). However, E. coli <br /> sequence. Each PCR primer designed effect major                            IT41 known revert wild-type growth character- <br /> changes: introduction restriction site end                        istics, considered necessary ascertain se- <br /> gene facilitate cloning (EcoRI 59 end HindIII                        quence mutant lepB gene changed the <br /> the 39 end) changes bases produced silent                           course experiment shown Fig. 4. As sequence of <br /> codon changes; changes effected minimize                           E. coli IT41 lepB gene previously determined, <br /> restrictions mRNA translation staphylococcal                                necessary determine sequence the <br /> codons comparatively rare E. coli change                       stocks cells used complementation studies. <br /> TTG initiation codon ATG codon. The PCR product                             The E. coli IT41 lepB gene amplified PCR <br /> cloned plasmid pDB575 (Table 1) form pKC16. The                               genomic DNA isolated sources E. coli IT41 <br /> correct sequence cloned insert confirmed. pKC16                            [E. coli IT41 grown permissive temperature 288C and <br /> was used transform E. coli XL1-Blue ampicillin resis-                           E. coli IT41(pKC16) growth 428C] cloned into <br /> tance, cells examined production SpsB                           pBluescriptII SK2. The sequences PCR products were <br /> induction IPTG. Cultures grown mid-log phase,                            determined identical. In case, only <br /> induced, harvested 2.5 h postinduction. Cell lysates                         single <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> change compared wild-type E. coli lepB <br /> analyzed SDS-PAGE (Fig. 3). It clear significant                           sequence apparent. The base change E. coli IT41 <br /> quantities SpsB produced E. coli(pKC16)                            lepB gene case C3T transition nucleotide <br /> the absence induction SpsB expression IPTG. An                             position 115 lepB coding sequence, <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> that <br /> increase stain density (lanes 3 5) compared                           changes glutamine codon (Gln-39 LPase) TAG amber <br /> control (lanes 2 4) position corresponding protein                      termination codon. As demonstrated se- <br /> with Mr ;21,000 apparent (lanes 4 5)                           quence E. coli IT41 lepB gene changed during <br /> and (lanes 2 3) IPTG induction.                                           course experiment shown Fig. 4, con- <br />    Experiments carried determine SpsB                          cluded SpsB complements growth E. coli pre- <br /> functionally active E. coli. E. coli IT41 single mutant                      protein-processing Ts mutant IT41 vivo. This conclusion <br /> copy lepB gene temperature sensitive (Ts)                           infers SpsB assembles correctly E. coli plasma mem- <br /> preprotein processing (14). The strain used exten-                           brane catalyzes processing essential E. coli <br /> sively investigations LPase function activity,                       preproteins. <br /> rate preprotein processing rate cell division                              Essentiality spsB gene S. aureus. As known that <br /> severely compromised 428C restored comple-                           B. subtilis B. amyloliquefaciens possess two <br /> mentation plasmid-borne active LPase (3, 4, 14). It                       SPase genes, determine S. aureus <br /> be noted unambiguous correlation E. coli IT41                         second gene encoding active SPase. Attempts <br /> growth rate rate preprotein processing demon-                           delete spsB gene chro- <br /> strably apparent experimental conditions (3, 4, 31,                       mosome S. aureus WCUH29 double-crossover homolo- <br /> 33).                                                                                  gous recombination using plasmid pTS1, contains a <br />    E. coli IT41 transformed pKC16 con-                          chloramphenicol resistance determinant active S. aureus, a <br /> trol plasmid pDB575. Growth E. coli IT41(pDB575) E.                            Ts G1 replication function derived mutant pE194 <br /> coli IT41(pKC16) 428C monitored OD540                                (41), pUC18, enabling replication selection E. coli <br /> period 8 h (Fig. 4) absence IPTG (because                           (a gift T. Foster, Trinity College, Dublin, Ireland). The <br /> possible toxic effect high level SpsB production). There                      general strategy details method described for <br /> very substantial increase growth rate E. coli                               E. coli Hamilton et al. (10) depicted Fig. 5. The <br /> IT41(pKC16) compared E. coli IT41(pDB575), indicating                            essence method upstream downstream re- <br /> functional expression SpsB. The generation time 1.5 h                        gions DNA flanking gene cloned a <br />  5716       CREGG ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                          FIG. 6. Southern blot S. aureus chromosomal DNA fragments isolated by <br />                                                                                       PCR probed spsB-specific probe probe specific the <br />                                                                                       downstream DNA fragment. Chromosomal DNA prepared S. aureus <br />                                                                                       WCUH29 (lane 1), S. aureus(pIM11) (lane 2), 10 chloramphenicol-sensitive <br />                                                                                       colonies (lanes 3 12) amplified using primers corresponding se- <br />                                                                                       quences upstream downstream flanking regions, products <br />                                                                                       separated agarose gel electrophoresis blotted nylon mem- <br />    FIG. 5. Strategy determination essentiality spsB S. aureus.   branes. Two identical membranes prepared hybridized labeled <br /> U D, upstream downstream regions, respectively, flanking spsB; Cm,            oligonucleotide corresponding spsB (A) downstream flanking <br /> chloramphenicol resistance determinant; ori, G2 origin replication; ori-       region (B) described Materials Methods. Expected fragment sizes are <br /> Ts, G1 Ts origin.                                                                     2,554 bp spsB 1,979 bp spsB. Fluorescein-labeled DNA markers <br />                                                                                       (lDNA-HindIII) 23.1, 9.4, 6.6, 4.4, 2.3, 2.0 kbp loaded outside <br />                                                                                       lanes. <br />  <br /> plasmid Ts replication function. Growth nonper- <br /> missive temperature selects integration homologous re-                         icol-sensitive colonies, cases plasmid isolated from <br /> combination (Campbell-type integration), subsequent                               chloramphenicol-resistant colonies possessed wild-type <br /> growth cointegrants permissive temperature leads                         spsB. Two explanations account appearance of <br /> a second recombination event resulting plasmid resolution.                         chloramphenicol-sensitive colonies derived S. aureus- <br /> If event takes place upstream region                         (pIM11). One spsB essential gene; is <br /> second event takes place downstream region, vice                            pIM11 recombined chromosome, plasmid <br /> versa, gene transferred                           resolved, depositing spsB chromosome, subse- <br /> plasmid chromosome. If gene dis-                          quently lost cell. To distinguish two <br /> pensable, recombinant plasmid subsequently                            possibilities, genomic DNA isolated 10 colonies, <br /> cured cell raising temperature,                        S. aureus(pIM11), S. aureus WCUH29, and <br /> gene essential, plasmid cured (or cured                       oligonucleotide PCR primers I J used amplify <br /> only recombination events reverse overall process                      chromosomal DNA therefrom. The primers correspond re- <br /> leading replacement gene chromosome). Re-                            gions chromosomal DNA lay upstream down- <br /> gions DNA upstream downstream spsB                                 stream sequences cloned pIM11, precluding <br /> gene, 850 bp each, amplified PCR plasmid                                 possibility amplification undetected plasmid-borne <br /> pKC11 using primer pairs E-F G-H, respectively. These                          target sequences. DNA fragments derived size frac- <br /> pairs generate terminal restriction enzyme sites facilitate                        tionated agarose gel blotted nylon membranes. <br /> subsequent cloning steps. The PCR products cloned                            Two identical membranes prepared probed either <br /> ligated KpnI site pUC18 (pIM04), DNA se-                             labeled oligonucleotide recognized spsB order <br /> quence ascertained ensure errors                              confirm presence absence spsB labeled <br /> incorporated amplification. The 1,700-bp fragment                          oligonucleotide bound downstream flanking re- <br /> subcloned multiple cloning site pTS1 using                             gion. The results presented Fig. 6 indicate no <br /> BamHI-EcoRI sites, construct, pIM05, trans-                               cases plasmid pIM11 lost S. aureus without <br /> formed E. coli S. aureus RN4220 S.                              effecting transfer spsB chromosome, as <br /> aureus WCUH29 electroporation. After appropriate tem-                              amplified fragments size (2,554 bp) that <br /> perature cycling select integration homologous re-                          amplified S. aureus WCUH29 DNA products <br /> combination chromosome plasmid resolution                                hybridized probes spsB downstream <br /> therefrom, cell lysates individual colonies prepared                        flanking region. The product obtained amplifying chromo- <br /> and screened presence plasmids bearing copy                           somal DNA S. aureus(pIM11) smaller (1,979 bp) and <br /> spsB derived chromosome hybridization probe                          did hybridize probe spsB, confirming spsB <br /> for spsB. One clone isolated (pIM11), identity                       deleted chromosome chromosomal <br /> was confirmed restriction analysis Southern blotting                           plasmid DNA amplified. These data imply <br /> (data shown). This clone subjected temperature                             type I SPase encoded spsB gene indis- <br /> cycling 438C attempt cure plasmid cell.                      pensable role preprotein processing S. aureus this <br /> Cells plated absence chloramphenicol, 146                          activity essential viability S. aureus. As control <br /> colonies replica plated plates chlor-                         experiment, S. aureus WCUH29 transformed plasmid <br /> amphenicol. Seventy-six colonies grew presence                         pIM11 subjected temperature cycling replica plat- <br /> chloramphenicol, remainder showed poor                           ing described above. Plasmid-borne spsB readily be <br /> growth. No plasmid isolated chloramphen-                            removed spsB merodiploids. <br />  VOL. 178, 1996                                                                       SIGNAL PEPTIDASE FROM S. AUREUS                     5717 <br />  <br />  <br />                           DISCUSSION                                   b-lactamase changed glycine codons GGC GGA in <br />                                                                        order produce catalytically inactive protein. In former <br />    Evidence accumulated suggest LPase belongs           case (GGC; base change), small amounts b-lactamase <br /> a new class serine protease does utilize histidine       activity equivalent ;0.1% wild-type levels detected, <br /> as catalytic base (4, 31) instead employ lysine       case (GGA; base changes), activity <br /> chain fulfill role (3, 33). These observations com-        detected. Evidence presented indicated A/G wob- <br /> parisons LexA E. coli led proposal             ble seryl-tRNA position codon-anti- <br /> serine-lysine catalytic dyad, similar thought operate       codon interaction responsible limited of <br /> during peptide bond hydrolysis catalyzed LexA (30),             mistranslation, resulting production small pool wild- <br /> operate LPase (3). Similar observations             type serine-containing enzyme. A similar phenomenon has <br /> made SPase B. subtilis (38) Tsp periplas-         noted authors (15). The data presented are <br /> mic protease E. coli (16); similarities SipS LexA       strongly suggestive similar mechanism production of <br /> have suggested extend regions primary            LPase activity E. coli IT41 Ts growth pre- <br /> structure (38). The serine lysine residues (90 145,            protein processing <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, likely mis- <br /> respectively, E. coli LPase numbering) known critical         translation occurs rate higher 0.1% case, the <br /> for catalytic activity E. coli LPase (3, 33) B. subtilis   wild-type levels LPase E. coli estimated be <br /> SPase (38) thought form catalytic dyad               region 1,000 molecules cell (40). <br /> conserved S. aureus protein SpsB (S-36 K-77). In               The discovery gene (spsA) encoding homolog type-I <br /> addition, aspartate position 155 (280 E. coli LPase          SPases immediately upstream spsB gene novel find- <br /> numbering) conserved (this residue appears important           ing bacteria. The short spacing (15 bases) 39 <br /> for activity SipS SPase [38] LPase E.      end spsA 59 end spsB implies cotranscription <br /> coli [31]). These observations, described       genes (although possible cryptic promoter <br /> Results, indicate gene encoding type I SPase, spsB,       spsB transcription resides 39-terminal coding <br /> been cloned S. aureus.                                            region spsA). A characteristic spsA gene reminiscent <br />    When expressed plasmid E. coli control           genes eukaryotes encode pro- <br /> an E. coli tac promoter, gene produces active protein,          teins key catalytic residues replaced, <br /> judged complementation activity E. coli strain        leading catalytically inactive proteins. Examples serine <br /> Ts preprotein processing. The known positive correlation           protease homologs catalytically inactive result of <br /> between E. coli IT41 growth rate catalytic activity         amino acid replacements active site include haptoglobin <br /> plasmid-borne SPase nonpermissive temperature (3, 4,            (18), protein Z (13), hepatocyte growth factor (24), human <br /> 31, 33) renders direct assay processing activity unnecessary        heparin-binding protein (26). The active-site mutations in <br /> in experiments. Sequence determination mutant             proteins S3A, H3K; S3A, H3T; S3Y, H3Q; <br /> lepB gene host strain complemen-           S3G, H3S, respectively. A common feature these <br /> tation experiment demonstrated acceleration E.         proteins appears retained, presumably as <br /> coli IT41 cell growth rate reversion Ts          result retention overall folding topology, capacity to <br /> phenotype lepB functional              bind hydrolyze substrates detectable level. It <br /> expression catalytically active SpsB. It established       likely SpsA falls general class of <br /> that vitro assays LPase activity considerably sen-      molecule bind preproteins cleave signal <br /> sitive vivo assay LPase adapted        peptides therefrom. As demonstrated SpsB can <br /> work estimation SpsB activity. These data indicate         function effectively SPase absence SpsA E. <br /> all proteins normally processed LPase,        coli, role (if any) SpsA protein targeting translo- <br /> essential E. coli, processed SpsB. The protein          cation S. aureus difficult define stage. It likely, <br /> sequence deduced sequencing E. coli IT41 lepB              however, SpsA persisted nonhydrolyzing enzyme <br /> gene intriguing. It clear protein fragment 38          considerable time S. aureus, active-site mutations <br /> amino acid residues (essentially transmembrane anchor)             (S3D K3S) necessarily involve changes least <br /> predicted arise C3T transition nucleotide po-           nucleotides codon, transitional forms inactive <br /> sition 115 lepB coding sequence responsible           enzyme preceded formation SpsA a <br /> for preprotein processing E. coli IT41 288C, alter-       nonaspartyl replacement active-site serine non- <br /> native explanation sought. The likely explanation         seryl replacement active-site lysine. <br /> is small quantities authentic LPase produced E.            Finally, data presented imply SpsB represents <br /> coli IT41 result misreading amber codon. The            essential enzymatic activity S. aureus. It possible to <br /> resultant cellular concentration LPase insufficient sup-      transfer spsB gene chromosome plasmid, but <br /> port normal rates preprotein processing (preproteins            possible eradicate spsB gene the <br /> seen accumulate permissive temperature [14]),           cell, possible remove plasmid-borne <br /> but cell growth rates sufficiently high enable experiments      spsB S. aureus spsB merodiploids. This observation in <br /> to performed. It likely 428C rate protein         marked contrast situation Bacillus genus: is <br /> targeting plasma membrane considerably greater          apparent cases, nonidentical <br /> the rate 288C insufficiency LPase activity       SPase genes coexist B. subtilis (and possibly all <br /> considerably apparent higher temperature;               bacteria SPase), SPase gene <br /> should noted activity zero 428C           deleted. <br /> (14). The E. coli IT41 parent strain E. coli W3110 (14), a <br /> nonsuppressing strain, behavior E. coli                                 ACKNOWLEDGMENTS <br /> IT41 428C explained presence Ts                 We gratefully acknowledge gifts pTS1 T. Foster, <br /> amber suppressor phenotype. Cases mistranslation pro-            pDB575 M. Lawrence D. Gale, E. coli IT41 Y. <br /> tein coding regions mutagenesis experiments               Nakamura. We thank M. J. Browne bringing paper hepa- <br /> noted (32). In case, serine (AGC) codon             tocyte growth factor (24) attention. <br />  5718       CREGG ET AL.                                                                                                                                    J. BACTERIOL. <br />  <br />  <br />                                  REFERENCES                                                 Bron, J. M. van Dijl. 1995. The endogenous B. subtilis (natto) plasmids <br />  1. Akagawa, E., K. Kurita, T. Sugawara, K. Nakamura, Y. Kasahara, N. Oga-                  pTA1015 pTA1040 contain signal peptidase-encoding genes: identifica- <br />     sawara, K. Yamane. 1995. Determination 17848 bp nucleotide                     tion new structural module cryptic plasmids. Mol. Microbiol. 17:621&#226;&#8364;&#8220; <br />     sequence 398 region Bacillus subtilis chromosome                  631. <br />     similarity analysis products putative ORFs. Microbiology 141:3241&#226;&#8364;&#8220;      22.   Mu &#194;&#168;ller, P., K. Ahrens, A. Klaucke. 1995. A TnphoA insertion the <br />     3245.                                                                                   Bradyrhizobium japonicum sipS gene, homologous prokaryotic signal pep- <br />  2. Allsop, A. E., G. Brooks, G. Bruton, S. Coulton, P. D. Edwards, I. K. Hatton,           tidases, results extensive changes expression PMB-specific nodu- <br />     A. C. Kaura, S. D. McLean, N. D. Pearson, T. Smale, R. Southgate. 1995.             lins infected soybean (Glycine max) cells. Mol. Microbiol. 18:831&#226;&#8364;&#8220;840. <br />     Penem inhibitors bacterial signal peptidase. Bioorg. Med. Chem. Lett.          23.   Mu &#194;&#168;ller, P., A. Klaucke, E. Wegel. 1995. TnphoA-induced symbiotic <br />     5:443&#226;&#8364;&#8220;448.                                                                              mutants Bradyrhizobium japonicum impair cell tissue differenti- <br />  3. Black, M. T. 1993. Evidence catalytic activity prokaryote leader            ation Glycine max nodules. Planta 197:163&#226;&#8364;&#8220;175. <br />     peptidase depends operation serine-lysine catalytic dyad. J.        24.   Nakamura, T., T. Nishizawa, M. Hagiya, T. Seki, M. Shimonishi, A. <br />     Bacteriol. 175:4957&#226;&#8364;&#8220;4961.                                                               Sugimura, K. Tashire, S. Shimizu. 1989. Molecular cloning expres- <br />  4. Black, M. T., J. G. R. Munn, A. E. Allsop. 1992. On catalytic                   sion human hepatocyte growth factor. Nature (London) 342:440&#226;&#8364;&#8220;443. <br />     mechanism prokaryotic leader peptidase 1. Biochem. J. 282:539&#226;&#8364;&#8220;543.             25.   Packer, J. C., D. Andre, C. J. Howe. 1995. Cloning sequence analysis <br />  4a.Cregg, K. M., M. T. Black. Unpublished data.                                        signal peptidase I thermophilic cyanobacterium Phormidium <br />  5. Dalbey, R. E., W. Wickner. 1985. Leader peptidase catalyzes release             laminosum. Plant Mol. Biol. 27:199&#226;&#8364;&#8220;204. <br />     exported proteins outer surface Escherichia coli plasma        26.   Petersen, L. C., J. J. Birktoft, H. Flodgaard. 1993. Binding bovine <br />     membrane. J. Biol. Chem. 260:15925&#226;&#8364;&#8220;15931.                                               pancreatic trypsin inhibitor heparin binding protein/CAP37/azurocidin. <br />  6. Date, T. 1983. Demonstration novel genetic technique leader                   Interaction Kunitz-type inhibitor proteolytically inactive <br />     peptidase essential enzyme Escherichia coli. J. Bacteriol. 154:76&#226;&#8364;&#8220;83.          serine proteinase homologue. Eur. J. Biochem. 214:271&#226;&#8364;&#8220;279. <br />  7. Devereux, J., P. Haeberli, O. Smithies. 1984. A comprehensive set          27.   Rifai, S., V. Barbancon, G. Prevost, Y. Piemont. 1989. Synthetic exfoli- <br />     sequence analysis programs VAX. Nucleic Acids Res. 12:387&#226;&#8364;&#8220;395.                  ative toxin A B DNA probes detection toxigenic Staphylococcus <br />  8. Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness,               aureus strains. J. Clin. Microbiol. 27:504&#226;&#8364;&#8220;506. <br />     A. R. Kerlavage, C. J. Bult, J.-F. Tomb, B. A. Dougherty, J. M. Merrick, K.       28.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />     McKenney, G. Sutton, W. FitzHugh, C. Fields, J. D. Gocayne, J. Scott, R.                laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold <br />     Shirley, L.-I. Liu, A. Glodek, J. M. Kelly, J. F. Weidman, C. A. Phillips, T.           Spring Harbor, N.Y. <br />     Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T.            29.   Schenk, S., R. A. Laddaga. 1992. Improved method electroporation <br />     Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L.                 Staphylococcus aureus. FEMS Microbiol. Lett. 94:133&#226;&#8364;&#8220;138. <br />     Fuhrmann, N. S. M. Geoghagen, C. L. Gnehm, L. A. McDonald, K. V. Small,           30.   Slilaty, S. N., J. Little. 1987. Lysine-156 serine-119 required for <br />     C. M. Fraser, H. O. Smith, J. C. Venter. 1995. Whole-genome random                  LexA repressor cleavage: possible mechanism. Proc. Natl. Acad. Sci. USA <br />     sequencing assembly Haemophilus influenzae Rd. Science 269:496&#226;&#8364;&#8220;                  84:3987&#226;&#8364;&#8220;3991. <br />     512.                                                                              31.   Sung, M., R. E. Dalbey. 1992. Identification potential active-site <br />  9. Fraser, C. M., J. D. Gocayne, O. White, M. D. Adams, R. A. Clayton, R. D.               residues Escherichia coli leader peptidase. J. Biol. Chem. 267:13154&#226;&#8364;&#8220; <br />     Fleischmann, C. J. Bult, A. R. Kerlavage, G. Sutton, J. M. Kelley, J. L.                13159. <br />     Fritchman, J. F. Weidman, K. V. Small, M. Sandusky, J. Fuhrmann, D.               32.   Toth, M. J., E. J. Murgola, P. Schimmel. 1988. Evidence unique <br />     Nguyen, T. R. Utterback, D. M. Saudek, C. A. Phillips, J. M. Merrick, J.-F.             position codon-anticodon mismatch vivo. J. Mol. Biol. 201:451&#226;&#8364;&#8220;454. <br />     Tomb, B. A. Dougherty, K. F. Bott, P.-C. Hu, T. S. Lucier, S. C. Peterson,        33.   Tschantz, W. R., M. Sung, V. M. Delgado-Partin, R. E. Dalbey. 1993. A <br />     H. O. Smith, C. A. Hutchison III, J. C. Venter. 1995. The minimal gene              serine lysine residue implicated catalytic mechanism the <br />     complement Mycoplasma genitalium. Science 270:397&#226;&#8364;&#8220;403. <br />                                                                                             Escherichia coli leader peptidase. J. Biol. Chem. 268:27349&#226;&#8364;&#8220;27354. <br /> 10. Hamilton, C. M., M. Aldea, B. K. Washburn, P. Babitzke, S. R. Kushner. <br />                                                                                       34.   van Dijl, J. M. 1993. Sequence p41027. Submitted EMBL/GenBank/ <br />     1989. New method generating deletions gene replacements Esch- <br />                                                                                             DDBJ data banks. <br />     erichia coli. J. Bacteriol. 171:4617&#226;&#8364;&#8220;4622. <br />                                                                                       35.   van Dijl, J. M. 1993. Sequence p41026. Submitted EMBL/GenBank/ <br /> 11. Hoang, V., A. Birger, J. Hofemeister. 1993. Sequence P42668. Submitted <br />     EMBL/GenBank/DDBJ data banks.                                                        DDBJ data banks. <br /> 12. Hoang, V., J. Hofemeister. 1995. B. amyloliquefaciens possesses second      36.   van Dijl, J. M., A. Jong, H. Smith, S. Bron, G. Venema. 1991. <br />     type I signal peptidase extensive sequence similarity Bacillus            Non-functional expression Escherichia coli signal peptidase I B. subtilis. <br />     SPases. Biochim. Biophys. Acta 1269:64&#226;&#8364;&#8220;68.                                              J. Gen. Microbiol. 137:2073&#226;&#8364;&#8220;2083. <br /> 13. Hojrup, P., M. S. Jensen, T. E. Petersen. 1995. Amino acid sequence        37.   van Dijl, J. M., A. Jong, J. Vehmaanpera, G. Venema, S. Bron. 1992. <br />     bovine protein Z: vitamin K-dependent serine protease homolog. FEBS                   Signal peptidase I B. subtilis: patterns conserved amino acids pro- <br />     Lett. 184:333&#226;&#8364;&#8220;338.                                                                      karyotic eukaryotic type I signal peptidases. EMBO J. 11:2819&#226;&#8364;&#8220;2828. <br /> 14. Inada, T., D. L. Court, K. <br /> </body></html>