<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Differences evolutionary history disease genes affected by <br /> dominant recessive mutations <br /> Simon J Furney1,3, M Mar Alb&#195;&#160;2 N&#195;&#186;ria L&#195;&#179;pez-Bigas*1 <br />  <br /> Address: 1Genome Bioinformatics Laboratory. Centre Genomic Regulation, Universitat Pompeu Fabra, Pg. Maritim la Barceloneta 37-49, E- <br /> 08003, Barcelona, Spain, 2ICREA &#226;&#8364;&#8220; Institut Municipal d&apos;Investigaci&#195;&#179; M&#195;&#168;dica. Universitat Pompeu Fabra, Dr. Aiguader 80, 08003, Barcelona, Spain <br /> and 3Conway Institute Biomolecular Biomedical Research, University College Dublin, Belfield, Dublin 4, Ireland <br /> Email: Simon J Furney - simon.furney@ucd.ie; M Mar Alb&#195;&#160; - malba@imim.es; N&#195;&#186;ria L&#195;&#179;pez-Bigas* - nuria.lopez@upf.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 03 July 2006                                                        Received: 03 March 2006 <br />                                                                                Accepted: 03 July 2006 <br /> BMC Genomics 2006, 7:165    doi:10.1186/1471-2164-7-165 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/7/165 <br /> &#194;&#169; 2006 Furney et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Global analyses human disease genes computational methods yielded <br />                  important advances understanding human diseases. Generally studies treated <br />                  group disease genes uniformly, ignoring type disease-causing mutations (dominant <br />                  recessive). In report present comprehensive study evolutionary history of <br />                  autosomal disease genes separated mode inheritance. <br />                  Results: We examine differences protein coding sequence conservation dominant <br />                  recessive human disease genes. Our analysis shows disease genes affected dominant <br />                  mutations conserved affected recessive mutations. This a <br />                  consequence fact recessive mutations remain hidden selection heterozygous. <br />                  Furthermore, employ functional annotation analysis investigations disease severity <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                  support hypothesis. <br />                  Conclusion: This study elucidates important differences dominantly- recessively- <br />                  acting disease genes terms protein DNA sequence conservation, paralogy essentiality. <br />                  We propose division disease genes mode inheritance enhance both <br />                  understanding disease process prediction candidate disease genes future. <br />  <br />  <br />  <br />  <br /> Background                                                                     Several reports analysed conservation <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of <br /> Understanding genetic basis human inherited dis-                        disease genes compared rest genes [2-7]. Human <br /> orders primary goals medical genetics. By                     disease genes conserved at <br /> applying knowledge gleaned research, dis-                        protein level, general, remainder the <br /> ease prevention treatment improved. Cur-                            human proteome [2], pos- <br /> rently, 1600 human genes known                          sess homologues mouse rat genomes [3]. Fur- <br /> be associated particular Mendelian disease pheno-                         thermore, study human genes involved disease <br /> types. The analysis group genes global                       lower non-synonymous substitution rate (KA) in <br /> perspective revealed interesting insights                    disease genes compared generic genes [5]. Another <br /> the nature human disease [1-5].                                             study sequence conservation nucleotide level <br />                                                                                human rat small difference <br />  <br />  <br />                                                                                                                                        Page 1 11 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                   http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> between KA/KS ratio human disease genes non-        Sanchez et al. showed diseases caused genes cod- <br /> disease genes, significantly elevated syn-     ing enzymes predominantly recessive, whereas <br /> onymous substitution rate (KS) disease genes        disorders causative gene coded tran- <br /> discovered [3]. Analysis chimpanzee genome         scription factor were, main, dominantly-acting [1]. <br /> found higher synonymous substitution rate disease         Lopez-Bigas et al. reported similar results further <br /> genes attributed higher relative frequency    demonstrate dominant human disease genes have <br /> CpG dinucleotides [6]. Recently work reported          closer paralogues recessive disease genes [14]. Kon- <br /> the analysis human-rodent KA/KS ratios disease genes     drashov Koonin reported haploinsufficient <br /> compared housekeeping rest genes, finding        (dominant) disease genes paralogues the <br /> that housekeeping genes conserved group           human genome haplosufficient (recessive) genes <br /> with disease genes showing intermediate values         [15]. <br /> the housekeeping genes [7]. <br />                                                                However, studies investigated differ- <br /> These studies treated disease genes homogene-      ences sequence conservation disease genes <br /> ous datasets doing neglected inherent        depending mode inheritance. Smith Eyre- <br /> difference sets genes, molecular      Walker [4] analyse gene evolution Jimenez-Sanchez <br /> genotype underlying disease state. Many autosomal          et al. dataset report higher conservation dominant <br /> Mendelian disease phenotypes understood              disease genes compared recessive disease genes, this <br /> terms dominant recessive mutations affecting          study observes selective constraints disease <br /> particular gene. In cases <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> said     genes compared non-disease genes, result con- <br /> be dominantly-acting, mutation single allele     trary evidence reports, to <br /> gene sufficient cause disease state. Genes suscep-   small number genes analysed [5]. <br /> tible disease-causing mutations way often <br /> haploinsufficient. The mutation cause loss function     We undertaken comprehensive study molec- <br /> of protein dominant negative effects, reduced     ular evolution autosomal human disease genes <br /> compromised functioning. Diseases caused recessive          depending mode inheritance; genes <br /> mutations require alleles affected, pro-     affected dominant mutations recessive muta- <br /> duction functional protein allele       tions. We investigated differences coding <br /> enough satisfy physiological requirements (haplosuffi-      sequence divergence, protein conservation, human paral- <br /> ciency).                                                       ogy, C. elegans lethality, protein function, gene structure <br />                                                                severity disease. The comparison evolution- <br /> The phenomenon dominance recessitivity dip-          ary patterns dominant recessive disease genes <br /> loid organisms recorded quantitatively Gregor     reveals important differences sets of <br /> Mendel 1866 [8], evolution dominance         genes understood terms different <br /> been topic debate geneticists cen-   hereditary nature, giving insights under- <br /> tury [9]. Fisher, 1928, reported mutations        standing hereditary human diseases. <br /> observed Drosophila recessive wild-type <br /> alleles [10]. He attributed balance recur-   Results <br /> rent mutations elimination natural selection.     Differences level protein conservation between <br /> However, Wright (1929) challenged hypothesis stat-        disease proteins affected dominant recessive <br /> ing unrealistically high levels selective pressure     mutations <br /> were necessary evolution dominance          The division disease genes reveals depending on <br /> manner [11]. He proposed dominance inher-          mode inheritance level conservation the <br /> ent consequence physiology. Kascer Burns (1981)         protein different. Human proteins encoded genes <br /> developed metabolic model dominance congruent            affected dominant mutations conserved in <br /> with Wright&apos;s physiological model [12]. However,           mouse encoded genes affected recessive <br /> metabolic model criticised solely       mutations (p-value Mann-Whitney (M-W) test = 7.59 <br /> based kinetic properties metabolic enzymes           &#195;&#8212; 10-6; Tables 1 &amp;2). When plot frequency distribu- <br /> [13].                                                          tions protein conservation scores (cs, Methods) <br />                                                                different sets genes, observe set of <br /> Previous studies analysed human disease genes          dominant disease genes (DD) significantly different <br /> terms nature mutation underlying disease     distribution recessive disease genes (DR) (p-value for <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> [1,2,5]. However focus studies      Kolmogorov-Smirnov (K-S) test = 1.95 &#195;&#8212; 10-6, Table 2 and <br /> been mainly differences functional classification    Figure 1) sets disease genes significantly <br /> of dominant recessive disease genes. Jimenez-              different non-disease genes (p-value K-S <br />  <br />  <br />                                                                                                                   Page 2 11 <br />                                                                                             (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                               http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 1: Molecular evolution paralogy disease genes involved dominant recessive diseases protein level. Average values <br /> for mouse paralogues conservation score different sets genes. <br />  <br />                                    CS mouse                        n                      CS paralogues                          n <br />  <br />       Non disease                     0.730                     16789                         0.385                           17962 <br />         Disease                       0.795                     1105                          0.349                            1108 <br />    Disease Dominant                   0.810                      474                          0.420                            478 <br />    Disease Recessive                  0.783                      631                          0.295                            630 <br />  <br />  <br />  <br /> tests &lt; 2.2 &#195;&#8212; 10-16). Recessive disease genes display sharp          dominant disease genes 0.420, higher <br /> decrease frequency high levels protein conserva-              non-disease genes. However, exam- <br /> tion scores (cs &gt; 0.9), trend apparent dominant               ine sequence conservation pattern paralogues more <br /> disease non-disease genes. Consistent results                   closely, observe non-disease genes higher <br /> observed analysis cs vertebrates,                 proportion highly similar paralogues (cs &gt; 0.8) than <br /> namely Rattus norvegicus, Gallus gallus, Takifugu rubripes,            disease gene groups (Figure 2). This differ- <br /> and Danio rerio [see Additional file 1].                               ence pronounced recessive dominant <br />                                                                        disease genes. To assess differences statistically we <br /> Level protein conservation paralogues dominant                calculated Z-scores p-values dominant reces- <br /> and recessive disease genes                                            sive disease genes 10,000 randomly generated sets <br /> Previously, reported genes involved dis-           proteins. Recessive disease genes significantly <br /> ease conserved paralogues human genes                lower expected number highly conserved para- <br /> general [2], presumably highly similar paralogues              logues (Z-score = -2.76, p-value = 2 &#195;&#8212; 10-3 cs &gt; 0.8). The <br /> can potentially compensate mutated protein [16],              number dominant disease gene paralogues cs &gt; <br /> which case disease observed. This con-               0.8 is, however, significantly different expected <br /> firmed analysis, average conservation              random sampling proteins. In conclusion, the <br /> score (cs) paralogues 0.349 human disease genes              lower number conserved paralogues disease ver- <br /> and 0.385 non-disease genes (p-value M-W test =                sus non-disease genes entirely attributed the <br /> 5.24 &#195;&#8212; 10-4, Tables 1 &amp;2). However, classifying dis-            recessive disease genes. <br /> ease genes mode inheritance phenotype, <br /> we significant differences observed                   Selective pressures acting dominant recessive <br /> between genes involved dominant recessive diseases               disease genes DNA level <br /> (p-value M-W test = 4.44 &#195;&#8212; 10-16; Tables 1 2),              We analysed level conservation DNA level of <br /> congruence previous results [14]. The average cs                  sets genes order obtain better understand- <br /> value paralogues recessive disease genes 0.295,               ing selective pressures acting them. We exam- <br />  <br /> Table 2: p-values Mann-Whitney Kolmogorov-Smirnoff test asses statistical significance differences the <br /> averages distributions groups. <br />  <br />                                                          Mann-Whitney Test <br />  <br />                             CS mouse              CS paralogs                 KA                         KS                          KI <br />  <br />       nD vs Dis              &lt;2.2 e-16              3.38 e-04                0.170                    1.36 e-10                  0.260 <br />       nD vs DD               &lt;2.2 e-16              5.24 e-04              6.37 e-04                  8.34 e-07                  0.567 <br />       nD vs DR               2.08 e-11              5.34 e-15              2.99 e-06                  1.05 e-05                  0.309 <br />       DD vs DR               7.59 e-06              4.44 e-16              7.17 e-11                    0.212                    0.798 <br />  <br />                                                      Kolmogorov-Smirnoff Test <br />  <br />                             CS mouse              CS paralogs                 KA                         KS                      KA/KS <br />  <br />       nD vs Dis              &lt;2.2 e-16              1.56 e-05              4.70 e-05                  1.12 e-09                &lt;2.2 e-16 <br />       nD vs DD               &lt;2.2 e-16              8.70 e-08              7.98 e-05                  3.06 e-04                &lt;2.2 e-16 <br />       nD vs DR               &lt;2.2 e-16              1.86 e-10              2.66 e-10                  2.20 e-07                &lt;2.2 e-16 <br />       DD vs DR               1.95 e-06              1.84 e-13              1.26 e-11                    0.122                  2.65 e-10 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Page 3 11 <br />                                                                                                            (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                                        http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br />             0.45 <br />                                                            Average cs                             vation scores (Figure 1) under-representation <br />                                Disease Dominant <br />              0.4 <br />                                                            0.810                                  recessive disease genes observed high conser- <br />                                Disease Recessive           0.783 <br />             0.35 <br />                                Non Disease                 0.730                                  vation scores. <br />              0.3 <br />                                                                                                   Disease genes higher KS values, dominant <br /> frequency <br />  <br />  <br />  <br />  <br />             0.25 <br />                                                                                                   recessive genes, rest human genes (p-value <br />              0.2                                                                                  M-W test disease versus non-disease = 8.34 &#195;&#8212; 10-7 Fig- <br />             0.15 <br />                                                                                                   ure 3B, Table 3). These differences ascribed to <br />                                                                                                   varying mutation rates genome, simi- <br />              0.1 <br />                                                                                                   larity mean KI values (Table 3) non-significant dif- <br />             0.05 <br />                                                                                                   ferences Mann-Whitney test (Table 2) all <br />               0                                                                                   groups negate this. Huang et al. reported a <br />                    0.1   0.2     0.3      0.4        0.5       0.6      0.7   0.8     0.9     1 <br />                                                                                                   significantly elevated KS level human disease genes <br />                                  conservation score mouse <br />                                                                                                   compared non-disease genes analysis between <br /> Conservation <br /> Figure level <br /> protein  1     dominant recessive disease genes                                         human rat genes [3]. This phenomenon dis- <br /> Conservation dominant recessive disease                                                    covered analysis chimpanzee genome [6]. <br /> genes protein level. Distribution conservation score <br /> in mouse dominant recessive disease genes versus                                       Due differences observed KS disease and <br /> rest genes.                                                                                    non-disease genes necessary analyse differences <br />                                                                                                   ratio KA/KS groups genes. This <br />                                                                                                   analysis confirms that, observed conservation <br /> ined non-synonymous (KA) synonymous (KS) coding                                               score KA analyses, dominant disease genes exhibit the <br /> sequence substitution rates, intron sequence                                           lowest evolutionary rates (Figure 3C, Table 3), dis- <br /> substitution rates (KI), human-chimpanzee ortho-                                             ease recessive genes similar average KA/KS value to <br /> logues [6]. KA indicative selective pressure acting                                     rest genes (nD), distribution KA/KS <br /> on sites involve change amino acid, KS                                         values groups genes dif- <br /> more reflective background mutation level. Disease                                         ferent (p-value K-S test &lt; 2.2 &#195;&#8212; 10-16). <br /> genes taken lower KA values non-dis- <br /> ease genes (average KA 0.00295 vs. 0.00318, Table 3),                                          Consistent results analysis KA, KS, and <br /> agreement previous results [5]. However, strong-                                         KA/KS human-mouse-dog orthologues [see Additional <br /> est differences types disease                                        file 2]. <br /> genes (M-W test p-value = 7.17 &#195;&#8212; 10-11), average <br /> KA 0.0026 dominant disease genes 0.0032                                         Essentiality dominant recessive disease genes <br /> recessive disease genes (Table 3, Figure 3A). Dominant                                            Our analysis shows recessive disease genes under- <br /> disease genes evolving slowly, general,                                          represented highly conserved genes. One possibil- <br /> than recessive disease genes non-disease genes,                                          ity highly conserved proteins occurrence of <br /> indicates subjected stronger selective con-                                      double mutations, observed recessive disease genes, <br /> straints. Interestingly, average value KA                                         associated lethality, case disease <br /> between disease recessive non-disease genes                                           condition observed. To assess mapped <br /> similar (0.00320 0.00318), distribution KA val-                                        genes study C. elegans gene orthologues <br /> ues sets clearly different (p-value K-S                                      previously classified Wild Type (WT), <br /> test = 2.66 &#195;&#8212; 10-10 Figure 3A). In fact, observe                                         disease (D) Lethal (L) according RNAi data [17]. <br /> much lower proportion disease recessive genes                                             Genes lethal disrupted considered <br /> very low KA value (&lt;0.001) intermediate KA values                                           &quot;essential                                          genes&quot;. <br /> are over-represented group genes (Figure 3). This <br /> is agreement observed distribution conser- <br />  <br /> Table 3: Molecular evolution disease non-disease genes. Average values KA, KS, KI KA/KS different sets genes. <br />  <br />                                                      KA                               KS                 KI                 KA/KS                       n <br />  <br />        Non disease                                 0.00318                          0.01400           0.01260                0.220                    10914 <br />          Disease                                   0.00295                          0.01644           0.01270                0.194                     886 <br />     Disease Dominant                               0.00260                          0.01790           0.01274                0.157                     367 <br />     Disease Recessive                              0.00320                          0.01540           0.01266                0.222                     519 <br />  <br />  <br />  <br />  <br />                                                                                                                                                       Page 4 11 <br />                                                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                                                  http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br />                                                      Average cs <br />             0.35               Disease Dominant      0.420 <br />                                                                                           A <br />                                Disease Recessive     0.295 <br />                                                                                                          0.35                                                                                           Average Ka <br />              0.3               Non Disease           0.385                                                0.3 <br />                                                                                                                                                                    Disease Dominant                     0.00260 <br />             0.25                                                                                         0.25                                                      Disease Recessive                    0.00320 <br />  <br />  <br />  <br />  <br />                                                                                         frequency <br />                                                                                                                                                                    Non Disease                          0.00318 <br /> frequency <br />  <br />  <br />  <br />  <br />              0.2 <br />                                                                                                           0.2 <br />  <br />  <br />  <br />                                                                                                          0.15 <br />             0.15 <br />  <br />  <br />                                                                                                           0.1 <br />              0.1 <br />  <br />  <br />                                                                                                          0.05 <br />  <br />             0.05 <br />  <br />                                                                                                               0 <br />                                                                                                                       0     0.001         0.002          0.003     0.004     0.005     0.006    0.007   0.008     0.009    &gt;0.009 <br />               0 <br />                    0.1   0.2     0.3      0.4      0.5    0.6     0.7   0.8   0.9   1 <br />  <br />  <br />                                  conservation score paralogues                                                                                                      Ka <br /> ease <br /> Figure  2 protein <br /> Conservation <br />      genes            level dominant recessive dis- <br />               paralogues                                                                                                                                                          Average Ks <br />                                                                                         B <br /> Conservation paralogues dominant reces-                                                         0.3 <br />  <br /> sive disease genes protein level. Distribution con- <br /> servation score paralogues dominant recessive                                                 0.25                          Disease Dominant                                0.01790 <br /> disease genes versus rest genes.                                                                                               Disease Recessive                               0.01540 <br />                                                                                                           0.2 <br />                                                                                                                                                                                       0.01400 <br />                                                                                            frequency <br />                                                                                                                                       Non Disease <br />                                                                                                         0.15 <br />  <br /> Firstly determined essential genes significantly <br />                                                                                                           0.1 <br /> more conserved WT genes (cs human-mouse: WT = <br /> 0.80, L = 0.84, p-value M-W test = 6.79 &#195;&#8212; 10-6; KA human-                                               0.05 <br />  <br /> chimpanzee orthologues: WT = 0.0019, L = 0.0013, p- <br /> value M-W test = 2.19 &#195;&#8212; 10-8). The question                                                          0 <br />                                                                                                                       0    0.0025     0.005             0.0075     0.01     0.0125     0.015   0.0175   0.02     0.0225   &gt;0.0225 <br />  <br /> whether recessive disease genes under-represented <br /> among essential genes. The data set reces-                                                                                                           Ks <br /> sive disease genes lower proportion C. elegans <br /> essential genes (31%) set dominant disease                                  C <br /> genes (36%). In addition, 47% dominant disease                                                  0.3 <br />                                                                                                                                                                                                                Average Ka/Ks <br /> orthologues phenotypic effect (WT)                                                                                                                                 Disease Dominant <br />                                                                                                        0.25                                                                                                        0.157 <br /> mutated, comparison 61% recessive disease ortho-                                                                                                                       Disease Recessive                  0.222 <br /> logues 58% non-disease gene orthologues.                                                         0.2 <br />                                                                                                                                                                                                                    0.220 <br />                                                                                           frequency <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 Non Disease <br />                                                                                                        0.15 <br /> Evolutionary conservation rates functional annotations <br /> Previously proportions dif-                                               0.1 <br />  <br />  <br /> ferent functional annotations dominant recessive <br />                                                                                                        0.05 <br /> disease genes [1,14,15]. Diseases caused <br /> by mutations genes coding enzymes transport-                                                  0 <br />                                                                                                                   0       0.1       0.2           0.3        0.4      0.5       0.6      0.7     0.8    0.9       1        &gt;1 <br /> ers recessive, mutations transcription <br /> regulators, structural molecules, nucleic acid binding <br />                                                                                                                                                                       Ka/Ks <br /> genes signal transducers primarily dominant. <br /> Therefore, differences evolutionary                             Evolutionary <br />                                                                                         dominant <br />                                                                                         Figure 3 andrates   between <br />                                                                                                        recessive disease <br />                                                                                                                       human <br />                                                                                                                           genes <br />                                                                                                                              andatchimpanzee <br />                                                                                                                                    DNA level genes of <br /> rates dominant recessive disease genes                                  Evolutionary rates human chimpanzee <br /> due dissimilarities different types                               genes dominant recessive disease genes at <br /> functional genes distinct evolutionary his-                              DNA level. (A) Distribution KA disease genes versus <br /> tories sets genes. In order rule                            rest genes. (B) Distribution KS disease genes <br /> possibility human disease gene classified accord-                              versus rest genes. (C) Distribution KA/KS ratio of <br /> ing molecular function protein product                                 disease genes versus rest genes. <br /> determined Gene Ontology (GO) &apos;slim&apos; terms [18], <br /> and conservation score mouse KA/KS values <br /> between human chimpanzee dominant reces-                                     tion values observed different functional clas- <br /> sive disease genes functional annotations                                sifications, observe consistently higher conservation <br /> were assessed (Table 4). Despite variation conserva-                             scores lower KA/KS values (Table 4) set dom- <br />  <br />  <br />                                                                                                                                                                                                                Page 5 11 <br />                                                                                                                                                                           (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                       http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 4: Mean CS mouse, paralogues KA/KS values dominant recessive disease genes GO slim category <br />  <br />                                                                          CS mouse <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.778                     0.790                     0.818                        0.755 <br />   Transcription regulation          0.800                     0.860                     0.871                        0.824 <br />     Nucleic acid binding            0.795                     0.835                     0.851                        0.803 <br />          Catalysis                  0.809                     0.819                     0.832                        0.815 <br />     Signal transduction             0.751                     0.748                     0.776                        0.708 <br />     Structural molecule             0.828                     0.831                     0.838                        0.815 <br />         Transporter                 0.824                     0.848                     0.862                        0.841 <br />           Carrier                   0.838                     0.881                     0.884                        0.880 <br />  <br />                                                                        CS paralogues <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.374                     0.272                     0.300                        0.236 <br />   Transcription regulation          0.353                     0.422                     0.440                        0.359 <br />     Nucleic acid binding            0.358                     0.370                     0.431                        0.250 <br />          Catalysis                  0.381                     0.355                     0.430                        0.331 <br />     Signal transduction             0.383                     0.347                     0.406                        0.261 <br />     Structural molecule             0.449                     0.482                     0.595                        0.207 <br />         Transporter                 0.424                     0.400                     0.539                        0.331 <br />           Carrier                   0.425                     0.432                     0.571                        0.402 <br />  <br />                                                                       KA/KS chimpazee <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.163                     0.233                     0.223                        0.251 <br />   Transcription regulation          0.170                     0.144                     0.109                        0.253 <br />     Nucleic acid binding            0.188                     0.164                     0.149                        0.190 <br />          Catalysis                  0.185                     0.179                     0.150                        0.189 <br />     Signal transduction             0.213                     0.173                     0.138                        0.238 <br />     Structural molecule             0.201                     0.183                     0.155                        0.250 <br />         Transporter                 0.176                     0.166                     0.082                        0.213 <br />           Carrier                   0.170                     0.197                     0.123                        0.213 <br />  <br />  <br />  <br /> inant disease genes compared recessive disease             reduction selective pressure decreasing severity of <br /> genes. These results confirm evolutionary differ-        disease (Table 5). While pattern clearly evident in <br /> ences observed dominant recessive disease             recessive disease genes, appears complicated <br /> genes different selective pressures acting          dominant disease genes. However, disease genes of <br /> these sets genes history simply       modes inheritance result severe pheno- <br /> due varying proportions functionally dissimilar             types (i.e. death reproductive age) con- <br /> genes group.                                              served genes consequent disease does <br />                                                                   severely affect reproductive fitness individ- <br /> In addition, paralogues dominant recessive            ual. <br /> disease genes similarly analysed, higher conserva- <br /> tion dominant disease gene paralogues evident            Gene structure autosomal dominant recessive <br /> each functional group (Table 4).                                  disease genes <br />                                                                   Previously, genes involved in <br /> Evolutionary conservation rates severity disease            hereditary diseases different gene structure proper- <br /> Analysis evolutionary constraints disease genes,        ties compared rest genes human genome <br /> as indicated human-chimpanzee KA/KS values,               [2,19]. In particular, disease genes longer coding <br /> categorised severity disease displays trend        sequences, exons alternative splicing [19]. <br />  <br />  <br />                                                                                                                        Page 6 11 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                         http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 5: KA/KS mean values mode inheritance according severity disease: None, Mild (death &gt;60 years), Moderate (death <br /> between puberty 60), Severe (death puberty). <br />  <br />                                       n                       Disease               Disease dominant             Disease recessive <br />  <br />          None                         99                       0.186                     0.133                         0.224 <br />           Mild                        36                       0.167                     0.114                         0.215 <br />         Moderate                      90                       0.182                     0.152                         0.209 <br />          Severe                       78                       0.168                     0.110                         0.186 <br />  <br />  <br />  <br /> These specific sequence properties thought            inversely correlated evolutionary rate, is, that <br /> to higher probability suffering disease-causing              older proteins evolve slowly proteins more <br /> mutations genes longer com-               recent origin [20]. However, that, spite <br /> plex splicing patterns [19]. We analysed structure           fact recessive disease genes higher evolu- <br /> genes involved autosomal dominant recessive dis-             tionary rates, better represented old genes <br /> eases elucidate differences groups.          dominant disease genes [see Additional file 3]. <br />  <br /> We confirmed previous results, finding                Overall, results presented mode of <br /> group disease genes has, average, higher number             inheritance gene important determinant its <br /> of exons (D = 13.53, nD = 9.77), higher number alter-          rate evolution. According nearly neutral theory of <br /> native transcripts (D = 5.03, nD = 4.47), longer gene         evolution slightly deleterious mutations become <br /> sequence (D = 62818 nD = 52066) coding                      fixed populations [21]. Recessive genes accumu- <br /> sequence (D = 657.2 nD = 509.6) compared                 late larger number mutations dominant <br /> complete set human genes. All differences             genes, the mutant &quot;hidden&quot; from <br /> been tested using Mann-Whitney Kolmogorov-                  selection heterozygous [22]. A higher fixation rate <br /> Smirnov tests significant. A similar analysis com-          slightly deleterious mutations recessive genes would <br /> paring set genes involved autosomal recessive          result higher KA/KS ratios type gene [4]. <br /> dominant diseases shows significant differences the <br /> protein gene <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> number alternative              Analyses conservation levels genes involved in <br /> transcripts sets genes. However,               recessive diseases shown significant decrease fre- <br /> there marginally statistically significant difference       quency high levels conservation (high conservation <br /> the number exons (DD = 13.65, DR = 14.63; p-value            score (Figure 1) low KA/KS ratio (Figure 3C)). The most <br /> M-W test = 8.5 &#195;&#8212; 10-4).                                             plausible explanation effect genes high <br />                                                                     degree conservation enriched human essen- <br /> Discussion                                                          tial genes, involved recessive disease <br /> Our analysis reveals selective pressures human          double mutation lethal. The <br /> autosomal disease genes involved dominant reces-             analysis RNAi phenotypes C. elegans human <br /> sive disorders differ significantly. Dominantly-acting dis-         orthologues groups genes analysed sup- <br /> ease genes appear conserved recessively-            ports this. We slightly higher proportion of <br /> acting genes. In addition, confirmed previous               lethal genes set dominant disease genes a <br /> results showing disease genes generally evo-          higher proportion genes effect (WT) the <br /> lutionarily conserved human genes [2,5,6].               recessive disease set. However, considerable <br />                                                                     proportion genes essential C. elegans but <br /> Previously proportions dif-           classified recessive disease genes human. This be <br /> ferent functional annotations dominant recessive             fact that, set essential genes in <br /> disease genes differ substantially [1,14,15]. Therefore,         C. elegans likely enriched human essential <br /> could nature conservation pattern             genes, cases gene essen- <br /> determined solely function protein. However           tial C. elegans humans. <br /> analysis conservation levels groups of <br /> disease genes GO slim functional category               It suggested essential genes evolve at <br /> (Table 4) appear refute this, categories,        slower rates non-essential genes [23]. Certainly, pre- <br /> dominant disease genes consistently conserved              vious works shown essential genes bacteria <br /> than recessive disease genes. Other factors               [24] yeast [25] highly conserved, there <br /> shown correlate protein evolutionary rate. In         controversial results [26,27]. We have <br /> recent report shown age protein        level conservation human genes tends <br />  <br />  <br />                                                                                                                          Page 7 11 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                    http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> to higher corresponding orthologue C. ele-       disease gene gene duplication. Alternatively, gene <br /> gans essential lethal. This supports     duplications dominant disease genes more <br /> the hypothesis sharp decrease recessive disease     recent similar events recessive disease genes. As <br /> genes high conservation levels related       seen Table 4, difference conservation <br /> paucity human essential genes recessive disease       level paralogues determined protein function. <br /> set. <br />                                                                 In study focused autosomal genes are <br /> Therefore, possible greater degree conser-    affected recessively- dominantly-acting mutations <br /> vation exhibited dominant disease genes vs. recessive        lead disease phenotype. We readily acknowledge <br /> disease genes effect higher likelihood fixa-    highlight relevant groups genes <br /> tion slightly deleterious non-synonymous mutations        X-linked disease genes imprinted genes <br /> the recessive disease genes coupled under-repre-        [28,29]. However, cursory evidence X-linked disease <br /> sentation highly conserved genes dataset.       gene evolution included [see Additional file 4]. <br />  <br /> In addition, analysed conservation disease genes      Conclusion <br /> according severity disease (Table 5). Our         We highlight significant differences disease <br /> analysis reveals conservation dominant         genes, terms protein DNA sequence conserva- <br /> and recessive disease genes highest category       tion, paralogy essentiality, categorised their <br /> which disease results death onset puberty,     mode inheritance. <br /> i.e. reproductive fitness equal zero population <br /> genetics perspective. We acknowledge partition      Our analysis reveals genes affected dominant dis- <br /> disease genes rudimentary groups simplifi-      ease mutations conserved recessive disease <br /> cation, however, serves insight relation-   genes. We attribute fact recessive mutations <br /> ship disease severity selective constraints.        remain hidden selection heterozygous, which <br />                                                                 allow recessive disease genes accumulate larger <br /> We confirmed disease genes higher KS        number slightly deleterious mutations eventually <br /> values, dominant recessive genes,         fixed populations. <br /> rest human genes (Figure 3B, Table 3). Huang et al. <br /> reported similar results analysis human           A number studies attempted predict disease <br /> and rat genes [3], phenomenon discov-         genes computationally using features sequence <br /> ered analysis chimpanzee genome [6]. Fur-         length, paralogy, sequence conservation, range tissue <br /> thermore, analysis authors ascribe higher KS    expression, amino-acid composition splicing signals <br /> level disease genes abundant presence         [2,4,5,30]. These studies treated dis- <br /> the mutation-susceptible CpG dinucleotide disease            ease genes homogeneous dataset. We suggest the <br /> genes. They report CpG dinucleotides              differences dominant recessive disease genes <br /> excluded analysis, difference KS rates          accounted future disease gene-prediction <br /> between sets genes non-significant.          studies. <br />  <br /> The tendency non-disease genes closer para-          Methods <br /> logues disease genes previously attributed        Data <br /> to fact close paralogue able ameliorate    Genes involved hereditary disease catalogued The <br /> the effect loss-of-function mutation gene vir-     Online database Mendelian Inheritance Man <br /> tue similar functionality [16]. We recessive   (OMIM) [31]. We retrieved list genes &apos;mor- <br /> disease genes, contain mainly loss-of-function            bid map&apos; table OMIM database. Using NCBI <br /> mutations, deficit close paralogues        LocusLink [32] database Ensembl database [33], <br /> compensate loss function. On contrary,           located corresponding gene sequence records. The <br /> dominant disease genes, possessing closer          result list 1647 genes associated human dis- <br /> hence functionally similar paralogues,          ease. All Ensembl protein-coding genes classi- <br /> reprieved disease state existence close par-   fied non-disease (nD). Disease genes classified <br /> alogues dominant-negative effects. In addi-        according mode inheritance disease they <br /> tion, haploinsufficient genes (i.e. dominant disease)       cause using text mining automatic extraction the <br /> be dosage-dependent, case retention            clinical synopsis section OMIM database manual <br /> paralogues advantageous [15]. However, dif-          curation reported [14]. Genes classified auto- <br /> ference observed groups disease genes        somal Disease Dominant (DD; n = 498) mutation in <br /> could selective pressure recessive     single allele reported causative genetic <br />  <br />  <br />                                                                                                                     Page 8 11 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                    http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> disorder, autosomal Disease Recessive (DR; n = 662)          rates KA, KS KI data genes taken the <br /> mutations alleles required disease phe-     Chimpanzee Sequencing Analysis Consortium sup- <br /> notype, Others (e.g. X-linked diseases) described        plementary data [6]. KA/KS values gene datasets were <br /> previously. Genes type mutations (dom-         calculated summing non-synonymous and <br /> inant recessive) used       synonymous sites dataset reported elsewhere <br /> study.                                                          [6]. Mann-Whitney Kolmogorov-Smirnov tests were <br />                                                                 conducted using KA, KS KI data individual <br /> Calcultation conservation score                              gene, KA/KS excluding genes KS = 0. <br /> Conservation score (cs) estimation divergence <br /> that occurred pair proteins evolu-      Essentiality data C. elegans <br /> tion, independent length proteins [2].     C. elegans genes classified depending pheno- <br /> This score provides estimation non-syn-      type RNAi experiments groups, Wild Type <br /> onymous nucleotide substitution rate pair          (WT), Lethal (L) Disease (D). RNAi phenotypes were <br /> proteins, takes account conservativeness      extracted Wormbase Biomart [17]. We mapped the <br /> of amino acid substitutions. Conservation scores cal-      human orthologue C. elegans gene using ortho- <br /> culated using WUBLASTP (version 2.0) [34],             logues pairwise data Ensembl Compara [39]. The <br /> based public domain NCBI BLAST version 1.4 [35].         number human genes corresponding WT, L D <br /> Hits E-values &gt; 1 &#195;&#8212; 10-10 discarded. Smith-           phenotypes RNAi experiments C. elegans was <br /> Waterman [36] alignment performed pairs              counted groups analysed (DD, DR and <br /> that gave significant BLAST hit. The value cs cal-     nD). <br /> culated datasets DD, DR nD using the <br /> relevant human protein-coding sequence Ensembl ver-          Evolutionary rates functional classification <br /> sion 34 WUBLASTP score closest homologue          The dominant recessive disease genes classified <br /> in Mus musculus divided WUBLASTP score            according molecular function protein as <br /> the protein itself, reported [37]. Sim-    determined Gene Ontology &quot;slim&quot; terms [18]. Pro- <br /> ilarly, conservation score paralogues calculated     tein conservation scores mouse paralogues were <br /> as WUBLASTP score closest paralogue divided          calculated described previously. KA/KS values cal- <br /> by WUBLASTP score protein itself.            culated gene dataset summing non- <br />                                                                 synonymous synonymous sites dataset. <br /> To assess statistical significance results com- <br /> puted non-parametric statistical tests, Kol-            Evolutionary rates severity disease <br /> mogorov-Smirnov (K-S) test Mann-Whitney (M-             Data severity disease genes obtained from <br /> W) test, using R statistics package [38]. To test   Jimenez-Sanchez et al. [1]. In total 303 genes set <br /> there significant deviation random expectation       classified following groups according to <br /> of number paralogues dominant recessive           disease severity: None, Mild (death &gt;60 years), Moderate <br /> disease genes (X) high levels conservation (cs &gt; 0.8)     (death puberty 60), Severe (death before <br /> we randomly generated 10,000 datasets proteins            puberty). KA/KS values group genes calcu- <br /> identical sample size dominant recessive      lated explained before. <br /> disease gene datasets counted set number <br /> of genes paralogues cs higher threshold      Gene structure analysis <br /> (0.8). Z-scores {Zx = (X - &#206;&#188;x)/&#207;&#402;x} p-values {px = &#206;&#163;(nx &#226;&#8240;&#165;    Gene length, coding sequence length number of <br /> X)/N} calculated (where mean = &#206;&#188;x, standard devia-         introns obtained Ensembl database [33]. The <br /> tion = &#207;&#402;x, nx number sets score        number alternative transcripts obtained Alter- <br /> threshold X N total number datasets)                 native Splicing Database [40]. Differences these <br />                                                                 sequences properties different groups (D, nD, <br /> Calculation nucleotide substitution rates DNA level       DD DR) assessed using Kolmogorov-Smir- <br /> The set 13,454 human-chimpanzee orthologues used             nov test. <br /> for gene evolution analysis Chimpanzee Sequenc- <br /> ing Analysis Consortium [6] translated Ent-        Authors&apos; contributions <br /> rez gene entries Ensembl gene IDs [33]. This dataset         SJF carried analysis statistical tests, <br /> was merged set dominant recessive OMIM          participated design study drafted the <br /> disease genes compiled earlier. This resulted non-         manuscript. MMA performed gene age analysis and <br /> redundant dataset 886 disease genes, consisting 367       revised manuscript critically. NLB conceived study, <br /> dominant disease (DD) 519 recessive disease (DR)            participated design coordination, helped to <br /> genes, 10,914 &quot;non-disease&quot; (nD) genes. Substitution <br />  <br />  <br />                                                                                                                    Page 9 11 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                         http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> draft manuscript. All authors read approved                          8.    Mendel: Experiments Plant Hybridization. Cambridge, MA., <br />                                                                                        Harvard University Press, 1965; 1866. <br /> final manuscript.                                                                9.    Porteous JW: Dominance--one years after <br />                                                                                        Mendel&apos;s paper. J Theor Biol 1996, 182:223-232. <br /> Additional material                                                              10.   Fisher RA: The possible modification response the <br />                                                                                        wild type recurrent mutations. Amer Nat 1928, 62:115-126. <br />                                                                                  11.   Wright S: Fisher&apos;s theory dominance. Amer Nat 1929, <br />                                                                                        63:274-279. <br />      Additional file 1                                                           12.   Kacser H, Burns JA: The molecular basis dominance. Genetics <br />      contains supplementary figure 1.                                               1981, 97:639-666. <br />      Click file                                                         13.   Phadnis N, Fry JD: Widespread correlations domi- <br />                                                                                        nance homozygous effects mutations: implications for <br />      [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                        theories dominance. Genetics 2005, 171:385-392. <br />      2164-7-165-S1.pdf]                                                          14.   Lopez-Bigas N, Blencowe BJ, Ouzounis CA: Highly consistent pat- <br />                                                                                        terns inherited human diseases molecular level. <br />      Additional file 2                                                                 Bioinformatics 2006, 22:269-277. <br />                                                                                  15.   Kondrashov FA, Koonin EV: A common framework under- <br />      contains supplementary tables 1 &amp; 2.                                           standing origin genetic dominance evolutionary <br />      Click file                                                               fates gene duplications. Trends Genet 2004, 20:287-290. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                   16.   Strachan T, Read AP: Molecular Pathology. In In Human Molecular <br />      2164-7-165-S2.pdf]                                                                Genetics 3 ed Oxford, BIOS Scientific; 2004. <br />                                                                                  17.   Wormbase [http://www.wormbase.org/biomart/martview] <br />                                                                                  18.   Harris MA, Clark J, Ireland A, Lomax J, Ashburner M, Foulger R, Eil- <br />      Additional file 3                                                                 beck K, Lewis S, Marshall B, Mungall C, Richter J, Rubin GM, Blake JA, <br />      contains supplementary figure 2.                                               Bult C, Dolan M, Drabkin H, Eppig JT, Hill DP, Ni L, Ringwald M, <br />      Click file                                                               Balakrishnan R, Cherry JM, Christie KR, Costanzo MC, Dwight SS, <br />      [http://www.biomedcentral.com/content/supplementary/1471-                         Engel S, Fisk DG, Hirschman JE, Hong EL, Nash RS, Sethuraman A, <br />      2164-7-165-S3.pdf]                                                                Theesfeld CL, Botstein D, Dolinski K, Feierbach B, Berardini T, Mun- <br />                                                                                        dodi S, Rhee SY, Apweiler R, Barrell D, Camon E, Dimmer E, Lee V, <br />                                                                                        Chisholm R, Gaudet P, Kibbe W, Kishore R, Schwarz EM, Sternberg <br />      Additional file 4                                                                 P, Gwinn M, Hannick L, Wortman J, Berriman M, Wood V, la Cruz <br />      contains supplementary table 3.                                                N, Tonellato P, Jaiswal P, Seigfried T, White R: The Gene Ontology <br />      Click file                                                               (GO) database informatics resource. Nucleic Acids Res <br />                                                                                        2004, 32:D258-61. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                   19.   Lopez-Bigas N, Audit B, Ouzounis C, Parra G, Guigo R: Are splicing <br />      2164-7-165-S4.pdf]                                                                mutations frequent cause hereditary disease? <br />                                                                                        FEBS Lett 2005, 579:1900-1903. <br />                                                                                  20.   Alba MM, Castresana J: Inverse relationship evolution- <br />                                                                                        ary rate age mammalian genes. Mol Biol Evol 2005, <br />                                                                                        22:598-606. <br /> Acknowledgements                                                                 21.   Ohta T: Slightly deleterious mutant substitutions evolu- <br />                                                                                        tion. Nature 1973, 246:96-98. <br /> We thank Roderic Guig&#195;&#179;, Desmond G. Higgins Arcadi Navarro val- <br />                                                                                  22.   Charlesworth B, Coyne JA, Barton NH: The relative rates evo- <br /> uable help, comments discussion. We thank Ensembl-Compara                 lution sex chromosomes autosomes. Amer Nat 1987, <br /> group pairwise similarity data. N. L.-B. supported long-term         130:113-146. <br /> post-doctoral fellowship Human Frontiers Science Program. S.J.F.        23.   Wilson AC, Carlson SS, White TJ: Biochemical evolution. Annu <br /> acknowledges support EMBO Short Term Fellowship ASTF 83-2005                      Rev Biochem 1977, 46:573-639. <br />                                                                                  24.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes are <br /> and Instituto Nacional Bioinform&#195;&#161;tica (INB). M.M.A. acknowledges                evolutionarily conserved nonessential genes <br /> support Fundaci&#195;&#179; ICREA Spanish Ministerio Edu-                    bacteria. Genome Res 2002, 12:962-968. <br /> caci&#195;&#179;n y Ciencia grant number BIO2002-04426-C02-01.                              25.   Hirsh AE, Fraser HB: Protein dispensability rate evolu- <br />                                                                                        tion. Nature 2001, 411:1046-1049. <br />                                                                                  26.   Hurst LD, Smith NG: Do essential genes evolve slowly? Curr Biol <br /> References                                                                             1999, 9:747-750. <br /> 1.      Jimenez-Sanchez G, Childs B, Valle D: Human disease genes.               27.   Yang J, Gu Z, Li WH: Rate protein evolution versus fitness <br />         Nature 2001, 409:853-855.                                                      effect gene deletion. Mol Biol Evol 2003, 20:772-774. <br /> 2.      Lopez-Bigas N, Ouzounis CA: Genome-wide identification                28.   Charlesworth B: The effect life-history mode inherit- <br />         genes likely involved human genetic disease. Nucleic                  ance neutral genetic variability. Genet Res 2001, 77:153-166. <br />         Acids Res 2004, 32:3108-3114.                                            29.   Spencer HG: Population genetics evolution genomic <br /> 3.      Huang H, Winter EE, Wang H, Weinstock KG, Xing H, Goodstadt L,                 imprinting. Annu Rev Genet 2000, 34:457-477. <br />         Stenson PD, Cooper DN, Smith D, Alba MM, Ponting CP, Fechtel K:          30.   Adie EA, Adams RR, Evans KL, Porteous DJ, Pickard BS: Speeding <br />         Evolutionary conservation selection human disease                       disease gene discovery sequence based candidate prioriti- <br />         gene orthologs rat mouse genomes. Genome Biol                       zation. BMC Bioinformatics 2005, 6:55. <br />         2004, 5:R47.                                                             31.   Hamosh A, Scott AF, Amberger J, Bocchini C, Valle D, McKusick VA: <br /> 4.      Smith NG, Eyre-Walker A: Human disease genes: patterns                     Online Mendelian Inheritance Man (OMIM), knowledge- <br />         predictions. Gene 2003, 318:169-175.                                           <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> human genes genetic disorders. Nucleic Acids Res <br /> 5.      Kondrashov FA, Ogurtsov AY, Kondrashov AS: Bioinformatical                     2002, 30:52-55. <br />         assay human gene morbidity. Nucleic Acids Res 2004,                   32.   Pruitt KD, Maglott DR: RefSeq LocusLink: NCBI gene-cen- <br />         32:1731-1737.                                                                  tered resources. Nucleic Acids Res 2001, 29:137-140. <br /> 6.      Consortium CS: Initial sequence chimpanzee genome                 33.   Hubbard T, Barker D, Birney E, Cameron G, Chen Y, Clark L, Cox T, <br />         comparison human genome. Nature 2005,                             Cuff J, Curwen V, Down T, Durbin R, Eyras E, Gilbert J, Hammond M, <br />         437:69-87.                                                                     Huminiecki L, Kasprzyk A, Lehvaslaiho H, Lijnzaad P, Melsopp C, <br /> 7.      Tu Z, Wang L, Xu M, Zhou X, Chen T, Sun F: Further understand-                 Mongin E, Pettett R, Pocock M, Potter S, Rust A, Schmidt E, Searle S, <br />         ing human disease genes comparing housekeeping                         Slater G, Smith J, Spooner W, Stabenau A, Stalker J, Stupka E, Ureta- <br />         genes genes. BMC Genomics 2006, 7:31.                                Vidal A, Vastrik I, Clamp M: The Ensembl genome database <br />                                                                                        project. Nucleic Acids Res 2002, 30:38-41. <br />  <br />  <br />  <br />                                                                                                                                            Page 10 11 <br />                                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                   http://www.biomedcentral.com/1471-2 <br /> </body></html>