<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Oct. 1995, p. 5554&#226;&#8364;&#8220;5560                                                                                 Vol. 177, No. 19 <br /> 0021-9193/95/$04.0010 <br /> Copyright q 1995, American Society Microbiology <br />  <br />  <br />  <br />           Functional Analysis ffh-trmD Region Escherichia <br />                  coli Chromosome Using Reverse Genetics <br />                                &#194;&#168; RAN O. BYLUND,1 DOUGLAS E. BERG,2 <br />           BRITT C. PERSSON,1&#226;&#8364;&#160; GO                                                                    AND                   &#194;&#168; M1* <br />                                                                                                           P. MIKAEL WIKSTRO <br />               Department Microbiology, University Ume&#195;&#165;, S-901 87 Ume&#195;&#165;, Sweden,1 Department Molecular <br />                          Microbiology, Washington University Medical School, St. Louis, Missouri 631102 <br />                                                   Received 2 May 1995/Accepted 27 July 1995 <br />  <br />              We analyzed essentiality contribution growth of genes Escherichia coli trmD <br />           operon (rpsP, 21K, trmD, rplS) flanking genes ffh 16K reverse genetic method. Mutant <br />           alleles constructed vitro plasmids transferred recombination corresponding l phage <br />           clone (l439) phage clone E. coli chromosome. An ability obtain recombinants in <br />           cells carrying complementing plasmid indicated mutated gene essential, ability <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />           obtain recombinants plasmid-free cells indicated nonessentiality. In way, Ffh, E. coli homolog the <br />           54-kDa protein signal recognition particle mammalian cells, ribosomal proteins S16 L19 were <br />           shown essential viability. A deletion second gene, 21K, trmD operon reduced growth <br />           rate cells fivefold, indicating wild-type 21-kDa protein important viability. A deletion- <br />           insertion gene resulted accumulation assembly intermediate 50S ribosomal <br />           subunit, result polar effects expression downstream gene, rplS, encodes ribosomal <br />           protein L19. This finding suggests L19, previously considered assembly protein, contributes <br />           assembly 50S ribosomal subunits. Strains deleted trmD gene, gene operon, <br />           encoding tRNA(m1G37)methyltransferase (or TrmD) showed severalfold reduced growth rate. Since such <br />           strain grew slower strain lacking tRNA(m1G37)methyltransferase activity point <br />           <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, TrmD protein second function cell. Finally, 16-kDa protein encoded the <br />           gene located downstream of, convergently transcribed to, trmD operon nonessential <br />           contribute growth. <br />  <br />  <br />    More 1,700 gene products identified Esch-                   trmD operon flanking genes (10). The ffh gene, situ- <br /> erichia coli K-12 (2, 40), half 4,700-kbp                   ated upstream trmD operon, encodes methionine- <br /> genome sequenced (39, 42). In past, genes                      rich 48-kDa protein (Ffh) (9, 10) suggested E. coli <br /> were identified selection mutations conferred spe-                   homolog 54-kDa protein mammalian signal rec- <br /> cific phenotypes, followed mapping various mutations.                      ognition particle (3, 38, 41), plays central role in <br /> However, increasing number open reading frames                       directing export nascent proteins cytoplasm of <br /> predicted DNA sequence analysis, effi-                     mammalian cells (13, 25, 28, 52). The Ffh protein recent- <br /> cient inactivate cloned open reading frames                     ly essential E. coli (37) shown <br /> then transfer mutant alleles chromosome study                      form complex 4.5S RNA, suggested <br /> gene function. Previously, developed reverse genetic                        participate cotranslational secretory pathway cer- <br /> method distinguishing essential nonessential                     tain E. coli proteins (38). The trmD operon contains genes <br /> genes (27). The method entails (i) transfer plasmid-carried                 ribosomal protein S16, 21-kDa protein (21K) un- <br /> inactivated chromosomal gene marked resistance trait                      known function, tRNA(m1G37)methyltransferase TrmD, <br /> to appropriate l phage clone collection Kohara et                  ribosomal protein L19, order (10). The tRNA- <br /> al. (24) homologous recombination chromosomal                         (m1G37)methyltransferase modifies guanosine position <br /> DNA flanking inactivated gene plasmid                         37 anticodon subset tRNAs E. coli <br /> corresponding chromosomal segment l phage clone                       Salmonella typhimurium (5, 6). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> studies trmD <br /> (ii) transduction pure stocks resultant recombinant                  mutant S. typhimurium, concluded modified <br /> phage, presence absence complementing plasmid <br />                                                                                  guanosine important maintenance correct <br /> in recipient strain. The ability obtain transductants only <br />                                                                                  reading frame translation (7, 18). The 21K TrmD <br /> in presence complementing plasmid indicates the <br />                                                                                  proteins 12- 40-fold-lower amounts, respec- <br /> essentiality targeted gene, ability obtain <br />                                                                                  tively, ribosomal proteins (54), different <br /> transductants absence complementing plasmid dem- <br />                                                                                  parts polycistronic mRNA equally abundant (11). <br /> onstrates nonessentiality gene question. Using this <br />                                                                                  The expression ribosomal protein genes trmD <br /> method, examined essentiality genes in <br />                                                                                  operon regulated differently ribo- <br /> the ffh-trmD region (56.7 min) E. coli chromosome (32). <br />                                                                                  somal protein genes, trmD operon expression not <br /> A plasmid covering region contains genes the <br />                                                                                  autoregulated translational feedback mechanism <br />                                                                                  controls overall synthesis ribosomes E. coli (35, <br />    * Corresponding author. Phone: 46-90-176754. Fax: 46-90-772630. <br />                                                                                  56). The gene downstream trmD operon tran- <br /> Electronic mail address: Mikael.Wikstrom@micro.umu.se.                           scribed orientation opposite trmD operon <br />    &#226;&#8364;&#160; Present address: Institut fu <br />                                 &#194;&#168;r Mikrobiologie und Genetik, Lehrstuhl          (10) encodes 16-kDa protein (16K), judged expres- <br /> fu <br />  &#194;&#168;r Mikrobiologie der Universita&#194;&#168;t Mu  &#194;&#168;nchen, D-80638 Mu&#194;&#168;nchen, Ger-            sion minicells (9). Here demonstrate Ffh protein <br /> many.                                                                            ribosomal proteins essential viability; the <br />  <br />                                                                           5554 <br />  VOL. 177, 1995                                                                              DELETION ANALYSIS OF GENES IN E. COLI                              5555 <br />  <br />  <br />                                               TABLE 1. Bacterial strains, bacteriophages, plasmids used <br />      Strain, phage, or <br />                                                                    Relevant markers                                                     Origin referencea <br />           plasmid <br />  <br /> Strains <br />   DB1434                                     lac-3350 galT2 galK22 rpsL179 (lplac5 cI857 Sam7)                                 Strain R594 reference 1; 46 <br />   MW27                                       Hfr P4X l2 DrpsP-1/pMW291 <br />   MW32                                       Hfr P4X l2 DtrmD-2 16K::nptI-1/pDBMW23 1 placIq <br />   MW33                                       Hfr P4X l2 trmD1 16K::nptI-1/pDBMW23 1 placIq <br />   MW35                                       Hfr P4X l2 D21K-1/pMW279 1 placIq <br />   MW37                                       Hfr P4X l2 D21K-2 16K::nptI-1 <br />   MW38                                       Hfr P4X l2 21K1 16K::nptI-1 <br />   MW40                                       Hfr P4X l2 D21K-2 16K::nptI-1/pMW279 1 placIq <br />   MW41                                       Hfr P4X l2 DrplS-1/pMW271 1 pHMG64 <br />   MW42                                       Hfr P4X l2 D21K-1 <br />   MW42/pBY03                                 Hfr P4X l2 D21K-1/pBY03 <br />   MW42/pDBMW9                                Hfr P4X l2 D21K-1/pDBMW9 <br />   MW44                                       Hfr P4X l2 21K1 16K::nptI-1/pMW279 1 placIq <br />   MW46                                       Hfr P4X l2/pMW279 1 placIq <br />   MW48                                       Hfr P4X l2/pMW271 1 pHMG64 <br />   MW49                                       Hfr P4X l2 Dffh-1/pMW292 <br />   MW100                                      Hfr P4X l2                                                                        56 <br />   MW100/pBY03                                Hfr P4X l2/pBY03                                                                  56 <br />   MW100/pMW291                               Hfr P4X l2/pMW291 <br />   MW100/pMW292                               Hfr P4X l2/pMW292 <br /> l phage clonesb <br />   l439/l22D7                                 ffh1-rpsP1-21K1-trmD1-rplS1 16K1                                                  24 <br />   l439Dffh-1                                 Dffh::nptI plasmid pDBMW24 <br />   l439DrpsP-1                                DrpsP::nptI-Plac plasmid pDBMW48 <br />   l439D21K-1                                 D21K::nptI-Ptac plasmid pDBMW5 <br />   l439D21K-2                                 D21K 16K::nptI plasmid pMW287 <br />   l439DtrmD-1                                DtrmD::nptI plasmid pDBMW53 <br />   l439DtrmD-2                                9trmD 16K::nptI plasmid pMW281 <br />   l439DrplS-1                                DrplS::nptI plasmid pDBMW31 <br />   l43916K::nptI-1                            16K::nptI plasmid pDBMW20 <br /> Plasmidsc <br />   pBY03                                      ffh1 rpsP1-21K1-trmD1-rplS1 16K1 bla                                              8 <br />   pBY12                                      ffh1 rpsP1-21K1-trmD1 bla                                                         8 <br />   pBY45                                      ffh1 rpsP1-21K1-trmD1-rplS1 16K1 bla                                              55 <br />   pCL1921                                    lacPOZ9 str/spc                                                                   31 <br />   pDBMW5                                     ffh1 rpsP1-D21K::nptI-Ptac-trmD1-rplS1 16K1 bla <br />   pDBMW9                                     ffh1 rpsP1-21K1-9trmD-rplS1 16K1 bla <br />   pDBMW20                                    ffh1 rpsP1-21K1-trmD1-rplS1 16K::nptI bla <br />   pDBMW23                                    Plac-21K1-trmD1-rplS9-cat-rrnBT1 bla <br />   pDBMW24                                    Dffh::nptI rpsP1-21K1-trmD1-rplS1 16K1 bla <br />   pDBMW31                                    ffh1 rpsP1-21K1-trmD1-DrplS::nptI 16K1 bla <br />   pDBMW48                                    ffh1 DrpsP::nptI-Plac-21K1-trmD1-rplS9-cat-rrnBT1 bla <br />   pDBMW53                                    ffh1 rpsP1-21K1-DtrmD::nptI-rplS1 16K1 bla <br />   pHMG64                                     lacIq pAYCY184                                                                 16 <br />   placIq                                     lacIq pACYC184                                                                 12, 58 <br />   pMW27                                      ffh1 rpsP1-21K9-lac9Z lacY1 lacA1 bla                                             55 <br />   pMW38                                      ffh1 rpsP1-D21K-trmD1-rplS1 16K1 bla                                              53 <br />   pMW146                                     ffh1 PrpsP-lac9Z lacY1 lacA1 bla                                                  57 <br />   pMW271                                     Plac-rplS1 16K1 cat-rrnBT1 bla <br />   pMW279                                     Plac-21K1-cat-rrnBT1 bla <br />   pMW281                                     ffh1 rpsP1-21K1-9trmD-rplS1 16K::nptI bla <br />   pMW287                                     ffh1 rpsP1-D21K-trmD1-rplS1 16K::nptI bla <br />   pMW291                                     ffh1 rpsP1-21K9 spc/str <br />   pMW292                                     ffh1 bla <br />   pUC4K                                      nptI bla                                                                          50 <br />   pUHE21-2                                   Plac-cat-rrnBT1 bla                                                               30 <br />  a <br />      Unless noted, origin study. <br />  b <br />      l439 corresponds systematic numbering miniset Kohara (23), l22D7 refers designations Kohara et al. (24). <br />  c <br />      Plasmid constructions described Materials Methods. <br />  <br />  <br />  <br />  <br /> 21K TrmD proteins important functions                                              MATERIALS AND METHODS <br /> cell, strains lacking corresponding genes <br />                                                                                          Strains, phages, plasmids. The strains, phages, plasmids used are <br /> (21K trmD) fivefold-reduced growth rate;                            listed Table 1. <br /> 16K protein does contribute significantly growth rate,                         Media growth conditions. Rich medium LB (4) supplemented with <br /> least conditions examined.                                              0.4% glucose E1B1 (51). Cultures grown 378C, growth was <br />  5556       PERSSON ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                        pDBMW5). The tac promoter included relieve possible polar effects of <br />                                                                                        nptI transcription downstream trmD rplS genes. Since tnpA <br />                                                                                        gene (second codon) fused frame 39 end 21K, intercistronic <br />                                                                                        region tnpA9-921K trmD identical 21K trmD <br />                                                                                        wild-type cells, ensure proper translation trmD. The entire <br />                                                                                        trmD gene replaced PCR-amplified fragment containing nptI <br />                                                                                        structural gene translational initiation region (Fig. 1, plasmid pD- <br />                                                                                        BMW53). To inactivate rplS gene, translation initiation region the <br />                                                                                        39 codons replaced nptI gene pUC4K (Fig. 1, plasmid <br />                                                                                        pDBMW31). The 16K gene located just downstream transcribed the <br />                                                                                        orientation opposite trmD operon inactivated inserting nptI <br />                                                                                        gene 75th 160 codons (Fig. 1, plasmid pDBMW20). To obtain a <br />                                                                                        selectable marker tightly linked nonpolar deletions 21K trmD, nptI <br />                                                                                        fragment plasmid pDBMW20 inserted NcoI site 16K of <br />                                                                                        plasmids pMW38 (D21K) pDBMW9 (DtrmD). The resulting plasmids with <br />                                                                                        nptI gene orientation trmD operon named pMW287 <br />                                                                                        (D21K) pMW281 (DtrmD). <br />                                                                                           Construction complementing plasmids. The 1,190-bp EcoRI-EcoRV frag- <br />                                                                                        ment carrying rplS 16K plasmid pBY45 cloned lac pro- <br />                                                                                        moter vector pUHE21-2 (Fig. 1, plasmid pMW271). The 21K gene new <br />                                                                                        translation initiation region cloned pUHE21-2 (Fig. 1, plasmid <br />                                                                                        pMW279). The ffh rpsP genes cloned EcoRI-BamHI fragment <br />                                                                                        plasmid pMW27 low-copy-number vector pCL1921 (Fig. 1, plasmid <br />                                                                                        pMW291). Plasmid pMW292 (ffh1; Fig. 1) constructed replacing lac <br />                                                                                        operon BamHI-SalI fragment plasmid pMW146 (ffh1 PS16-lacZYA) by <br />                                                                                        adaptor oligonucleotide 59-GATCGTCGAC-39. <br />                                                                                           Transfer inactivated genes l439 homologous recombination. The <br />                                                                                        mutations shown Fig. 1 transferred phage l439 collection of <br />                                                                                        Kohara et al. (24) recombination mutated DNAs cloned in <br />                                                                                        plasmids homologous region l439 phage clone. The recombinant <br />                                                                                        clones recovered transducing strain DB1434 selecting Kmr at <br />                                                                                        308C described previously (27). <br />                                                                                           PCR amplification DNA sequencing. Bacterial colonies resuspended <br />                                                                                        H2O PCR amplified described (33, 45), using Ampli- <br />                                                                                        taq DNA polymerase Perkin-Elmer Cetus, Norwalk, Conn., Taq DNA <br />                                                                                        polymerase Boehringer Mannheim. To verify chromosomal copies <br />                                                                                        studied genes inactivated plasmid-free plasmid- <br />                                                                                        containing cells, PCR analyses designed primers <br />                                                                                        test bind complementing plasmid, ensuring that <br />                                                                                        PCR product size reflect status chromosomal allele. The <br />                                                                                        postulated mutant strains gave PCR products expected sizes, were <br />                                                                                        different corresponding wild-type strains (data shown). <br />                                                                                           Linear amplification cycle sequencing double-stranded DNA Amplitaq <br />                                                                                        DNA polymerase, using primers end labeled 32P, performed previ- <br />                                                                                        ously described (26). <br />                                                                                           Sucrose gradients. Preparation S30 extracts separation ribosomal <br />                                                                                        subunits sucrose gradient centrifugation described Ryde&#194;&#180;n- <br />                                                                                        Aulin et al. (44). <br />  <br />                                                                                                                          RESULTS <br />  <br />                                                                                           Replacement chromosomal genes inactivated alleles in <br />                                                                                        l phage clones. The mutations shown Fig. 1 trans- <br />                                                                                        ferred phage l439 collection Kohara et al. (24) <br />                                                                                        homologous recombination (see Materials Methods). <br />    FIG. 1. The important plasmids used study. Open boxes indicate         To test genes inactivated essential for <br /> genes chromosomal origin, hatched box indicates nptI gene. <br /> Relevant restriction sites: A, AvaI; B, BamHI; Bs, BspHI; E, EcoRI; Ea, EagI; H,       viability, strain MW100 plasmid pBY03 (Fig. 1) <br /> HpaI; N, NcoI; P, PstI; S, SalI; X, XbaI; Xh, XhoI. Those parentheses      transduced appropriate l phage stocks. Kmr trans- <br /> destroyed result cloning procedure. The arrows 16K, bla,         ductants obtained frequencies 1026 but <br /> cat, ffh, nptI genes 59-to-39 orientation, arrow    cases: transduction strain MW100 phage clones <br /> trmD operon symbolizes trmD operon mRNA. P T indicate promoter <br /> and terminator, respectively, trmD operon. Ptac tac promoter (15,        carrying mutations ffh rpsP (frequencies #3 3 1028; <br /> 43), Plac indicates strong inducible lac promoter PA1/04/03 (30).              Table 2). In contrast, frequencies Kmr transductants, <br />                                                                                        using phage clones strain MW100/pBY03, were <br />                                                                                        .1026 case. The rare Kmr transductants strain <br />                                                                                        MW100, using l439Dffh-1 l439DrpsP-1, segregated Kmr <br /> monitored Zeiss PMQ3 spectrophotometer 420 nm. Tryptone broth, <br /> tryptone agar, tryptone agar plates used preparation l <br />                                                                                        marker grown selection kanamycin (data not <br /> lysates.                                                                               shown), indicating inactivated genes inte- <br />    Construction mutant alleles. The ffh gene, encoding 48-kDa protein,        grated chromosome cells containing preexisting <br /> inactivated replacing region codons 172 414 nptI           duplications targeted region. That transductants <br /> (kanamycin resistance [Kmr]) gene pUC4K (Fig. 1, plasmid pDBMW24). <br /> The gene trmD operon, rpsP, encoding ribosomal protein S16, was <br />                                                                                        contained wild-type mutant copy targeted <br /> replaced entirety nptI gene (Fig. 1, plasmid pDBMW48). To over-          gene confirmed PCR analysis (data shown). Thus, <br /> come polar effects expression downstream genes operon,            results suggest ffh rpsP essential genes. <br /> strong inducible lac promoter PA1/04/03 (30) placed upstream 21K               The transduction frequency obtained using recombi- <br /> gene, new translation initiation region created ensure efficient <br /> translation 21K. All codons 21K replaced <br />                                                                                        nant phages carrying inactivated genes in- <br /> by fragment carrying nptI upstream tac promoter joined transla-        dependent recipient strain MW100 or <br /> tion initiation region tnpA gene (49) transposon gd (Fig. 1, plasmid       MW100/pBY03 (Table 2). Transductants MW100 obtained <br />  VOL. 177, 1995                                                                                DELETION ANALYSIS OF GENES IN E. COLI                                5557 <br />  <br />  <br />  TABLE 2. Transduction strains haploid polyploid                      trmD rplS Ptac chromosome suf- <br />       56.7-min region chromosome pure                                  ficient support normal growth 21K gene pro- <br />            l phages carrying inactivated genes                                     vided trans plasmid. We emphasize, however, in <br />                                           Transduction frequency, growth               cells synthesizing 21K protein, expression trmD <br />      l phage                                     characteristicsa                      rplS does limit growth rate, growth not <br />                                                                                        enhanced presence plasmid pMW38 (data not <br />                                       MW100b                       MW100/pBY03 <br />                                                                                        shown), deleted 21K expresses ffh, rpsP, trmD, <br />                                           28 <br /> l439Dffh-1                        2 3 10 , 11                     5 3 1026, 11         rplS, 16K (56). These findings suggest fivefold- <br /> l439DrpsP-1                       8 3 1029, 1                     5 3 1026, 11         reduced growth rate D21K-1 mutant lack of <br /> l439D21K-1                        2 3 1025, (1)                   7 3 1026, 11         21K protein, limiting amounts TrmD L19 <br /> l439DtrmD-1                       3 3 1026, (1)                   2 3 1026, 11         proteins. <br /> l439DrplS-1                       8 3 1026, (1)                   7 3 1026, 11            The slow growth clones carried DtrmD-1 (DtrmD:: <br /> l43916K::nptI-1                   4 3 1026, 11                    1 3 1026, 1 <br />                                                                                        nptI) mutation complemented plasmid pBY03 carry- <br />   a <br />     Transduction frequency expressed number transductants added           ing intact trmD operon plasmid pDBMW23 <br /> phage. The growth characteristics observed plates            (PA1/04/03-21K1-trmD1; data shown), suggesting the <br /> original transductants: 11, normal growth 16 h incubation; 1, reduced         nptI gene polar expression rplS. <br /> growth rate colonies visible 16 h; (1), small colonies visible 48 h. <br />   b <br />     Recipient strain.                                                                     The DrplS-1 mutation chromosome comple- <br />                                                                                        mented plasmid pMW271 (PA1/04/03-rplS1 16K1), although <br />                                                                                        completely (Table 3). The growth rate wild-type <br />                                                                                        strain containing plasmid pMW271 decreased addition <br /> by using l43916K::nptI showed significant growth impair-                            IPTG (Table 3). On hand, growth rate the <br /> ment rich minimal medium aerobic anaerobic                              DrplS-1 mutant containing pMW271 increased increasing <br /> conditions compared strain MW100, indicating 16K                          concentrations IPTG 0.25 mM, higher con- <br /> a nonessential gene. However, phage carried in-                            centrations decreased (data shown), showing both <br /> activated 21K, trmD, rplS used, transductants                            low high levels expression rplS, reduce the <br /> strain MW100 formed small colonies 48 h,                           growth rate. Probably, plasmid pMW271 complemented the <br /> contrast transductants strain MW100/pBY03,                                 DrplS-1 mutation partially result nonoptimal <br /> formed normal-size colonies 16 h (Table 2). Upon re-                             IPTG-induced expression rplS. <br /> streaking, small colonies MW100 derivatives with <br /> inactivated 21K trmD showed stable <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span>, whereas <br /> MW100 inactivated rplS barely grew formed minute                                    TABLE 3. Complementation inactivated genes the <br /> colonies 72 h. When restreaked, cells                                     chromosome different plasmids expressing <br /> from minute colonies able form                                            corresponding wild-type genes <br /> colonies, faster growing, prob- <br />                                                                                        Chromo-                                                                   Growth <br /> ably result secondary mutations (data shown).                                somal               Plasmid(s) (relevant genotype)              IPTGb    rate <br /> These results indicated mutants lacking 21K trmD                          genotypea                                                                  (k)c <br /> severely affected growth rplS gene essential <br />                                                                                        ffh1    pMW292 (ffh1)                                               2      1.58 <br /> for viability absence compensatory mutations. <br />                                                                                        Dffh-1  pMW292 (ffh1)                                               2      1.63 <br />    Complementation inactivated genes plasmids contain-                           rpsP1   pMW291 (ffh1 rpsP1)                                         1      1.98 <br /> ing different parts ffh-trmD chromosomal region. To test                        DrpsP-1 pMW291 (ffh1 rpsP1)                                         1      2.02 <br /> if observed complementation plasmid pBY03 (Table 2)                             21K1    None                                                        2      1.73 <br /> was wild-type copy inactivated gene, strain                          21K1    None                                                        1      1.81 <br /> MW100 harboring plasmids different segments                                D21K-1  None                                                        2      0.34 <br /> region cloned plasmid pBY03 transduced dif-                            D21K-1  None                                                        1      0.37 <br /> ferent l phage clones. Transductants obtained fre-                             21K1    pMW279 (Plac-21K1) 1 placIq (lacIq)                         1      1.73 <br /> quencies similar Table 2 (data shown). The                             D21K-1  pMW279 (Plac-21K1) 1 placIq (lacIq)                         1      0.77 <br />                                                                                        21K1    None                                                        2      1.89 <br /> Dffh-1 DrpsP-1 mutations fully complemented the <br />                                                                                        D21K-2  None                                                        2      0.40 <br /> plasmids pMW292 (ffh1) pMW291 (ffh1 rpsP1), respec-                                21K1    pMW279 (Plac-21K1) 1 placIq (lacIq)                         1      1.70 <br /> tively (Table 3).                                                                      D21K-2  pMW279 (Plac-21K1) 1 placIq (lacIq)                         1      1.63 <br />    The strain carrying D21K-1 mutation (D21K::nptI-Ptac)                           trmD1   pDBMW23 (Plac-21K1-trmD1-rplS9) 1 placIq                    1      1.54 <br /> showed fivefold-reduced growth rate, induction                                      (lacIq) <br /> expression trmD rplS tac promoter                               DtrmD-2 pDBMW23 (Plac-21K1-trmD1-rplS9) 1 placIq                    1      1.51 <br /> chromosome did significantly increase growth rate (Ta-                                   (lacIq) <br /> ble 3), suggesting demand TrmD L19                                rplS1   pMW271 (Plac-rplS1 16K1) 1 pHMG64 (lacIq)                   2      1.51 <br /> proteins satisfied transcription tac promoter                       rplS1   pMW271 (Plac-rplS1 16K1) 1 pHMG64 (lacIq)                   1      0.56 <br />                                                                                        DrplS-1 pMW271 (Plac-rplS1 16K1) 1 pHMG64 (lacIq)                   2      NP <br /> even absence isopropylthio-b-D-galactopyranoside <br />                                                                                        DrplS-1 pMW271 (Plac-rplS1 16K1) 1 pHMG64 (lacIq)                   1      1.01 <br /> (IPTG) (probably result leakiness Ptac [29]). <br /> However, 21K trmD genes supplied trans                              a <br />                                                                                             The 21K1 strains isogenic D21K-1 strains derivatives strain <br /> plasmid pBY12 (ffh1 rpsP1-21K1-trmD1), faster growth                              MW100, 21K1 strains isogenic D21K-2 strains derivatives of <br />                                                                                        strain MW38 (MW100 16K::nptI). <br /> became IPTG dependent, i.e., requiring expression rplS                              b <br />                                                                                             The concentration IPTG used supporting optimal growth: 0.25 <br /> the chromosome (data shown). This finding agree-                             mM DrplS-1, 0.1 mM DrpsP-1 D21K-1, 1 mM strains <br /> ment need (and synthesis of) higher amounts                            contained plasmid pDBMW23 pMW279. <br />                                                                                           c <br /> ribosomal proteins higher growth rates wild-type E. coli                              The strains grown LB supplemented 0.4% glucose E1B1. <br />                                                                                        For pMW291- placIq-containing strains, 50 mg spectinomycin ml 20 <br /> cells (35). Plasmid pMW279 (PA1/04/03-21K1; Fig. 1) did                            mg tetracycline ml, respectively, added medium. For other <br /> fully complement slow growth D21K-1 mutant                             plasmid-containing strains, 50 mg carbenicillin ml used. k 5 ln2/g, <br /> in presence IPTG (Table 3). Thus, expression                             g mass doubling time hours. NP, possible. <br />  5558     PERSSON ET AL.                                                                                                            J. BACTERIOL. <br />  <br />  <br />    Strains containing nonpolar mutations 21K trmD <br /> have fivefold-reduced growth rate. As shown above, the <br /> D21K-1 DtrmD-1 mutations chromosome not <br /> fully complemented plasmids pMW279 (PA1/04/03-21K1) <br /> and pDBMW23 (PA1/04/03-21K1-trmD1), respectively. The dele- <br /> tion 21K polar expression trmD and <br /> rplS carried plasmid pMW38 (56). Therefore, <br /> the deletion 21K (D21K-2) plasmid pMW38 trans- <br /> ferred chromosome using 16K::nptI linked se- <br /> lectable marker. Similarly, deletion 59 one-third of <br /> trmD (DtrmD-2) combined 16K::nptI transferred <br /> to chromosome. Both D21K-2- DtrmD-2-containing <br /> transductants slowly growing D21K-1 and <br /> DtrmD-1 mutants described above. The DtrmD-2-containing <br /> clones were, however, unstable mutated high fre- <br /> quency slightly faster-growing derivatives, it <br /> impossible determine accurately growth rate orig- <br /> inal mutant. However, mutant formed colonies of <br /> approximately size did strains containing more <br /> stable D21K-2 mutation, growth rate reduction caused by <br /> the DtrmD-2 mutation inferred approximately fivefold. <br /> To examine D21K-2 DtrmD-2 mutations polar <br /> on expression respective downstream genes, the <br /> deletions introduced transduction strain MW100 <br /> containing complementing plasmids. That deletions had <br /> replaced respective wild-type gene chromosome was <br /> confirmed PCR analysis (data shown). The slow growth <br /> of D21K-2 DtrmD-2 mutants fully complemented <br /> by plasmids pMW279 (PA1/04/03-21K1) pDBMW23 (PA1/ <br />           1       1 <br /> 04/03-21K -trmD ), respectively (Table 3). This finding indi- <br /> cated mutations polar expression of <br /> downstream genes. In summary, results comple- <br /> mentation experiments suggest Ffh protein ribo- <br /> somal proteins S16 L19 essential viability that <br /> the 21K TrmD proteins important roles at <br /> high growth rates. <br />    The D21K-1 mutant defective assembly 50S <br /> ribosomal subunit result polar effects rplS expres-         FIG. 2. Separation ribosomal subunits sucrose gradient centrifugation. <br /> sion. The D21K-1 mutant strain MW42 grew extremely poorly           Shown extracts log-phase cells parental strain MW100 (A), log- <br />                                                                     phase cells D21K-1 mutant MW42 (B), stationary-phase cells strain <br /> at 308C, indicating cold sensitive growth. Since    MW42 (C), log-phase cells strain MW42 grown presence 0.5 mM <br /> cold-sensitive strains defective ribosome assembly     IPTG (D), log-phase cells strain MW42 containing plasmid pDBMW9 (ffh1 <br /> (17, 48), examined ribosomal subunits mutant        rpsP1-21K1-9trmD-rplS1 16K1) (E) (a similar sedimentation profile ob- <br /> the parental strain sucrose gradient centrifugation. Loga-       tained strain MW42 containing plasmid pBY03 [ffh1 rpsP1-21K1-trmD1-rplS <br />                                                                     16K]), log-phase cells strain MW37 (D21K-2) (F). 50S* indicates 50S <br /> rithmically growing cells mutant strain MW42           assembly intermediate. <br /> reduced ratio 50S 30S subunits (Fig. 2B). In addition, <br /> there extra peak 50S 30S subunit peaks <br /> from mutant wild-type strain. Two-dimen- <br /> sional ribosomal protein gel analysis indicated fractions      deed caused limiting amounts TrmD L19. To establish <br /> corresponding extra peak contained mainly 50S ribo-         protein limiting ribosome assembly, intro- <br /> somal proteins (data shown). Interestingly, sedimenta-      duced TrmD2 L191 plasmid pDBMW9 (ffh1 rpsP1 21K1- <br /> tion profiles ribosomes mutant cells stationary         9trmD-rplS1 16K1) TrmD1 L191 plasmid pBY03 (ffh1 <br /> phase indicated normal 50S-to-30S subunit ratio re-           rpsP1-21K1-trmD1-rplS1 16K1) strain MW42 exam- <br /> vealed extra peak 50S 30S subunit peaks          ined ribosome assembly. No 50S assembly intermediate was <br /> (Fig. 2C). These findings suggest mutant lower       observed strain (Fig. 2E), suggesting ri- <br /> rate 50S subunit assembly wild-type strain,       bosome assembly defect plasmid-free MW42 nonin- <br /> results accumulation assembly intermediate. Since      ducing conditions limiting amounts ribosomal <br /> slow growth D21K-1 mutant MW42 fully com-            protein L19. <br /> plemented plasmid pMW279 (PA1/04/03-21K1; Fig. 1), we <br /> examined ribosome assembly defect polar                                         DISCUSSION <br /> effects nptI::Ptac expression trmD rplS Ptac <br /> was induced, growth defect mutant          This communication describes functional analysis six <br /> not relieved addition IPTG. No assembly intermediate          genes ffh-trmD region (56.7 min) E. coli chromo- <br /> was detected strain MW42 (D21K-1) grown               some. Our results indicate Ffh protein two <br /> IPTG (Fig. 2D). Moreover, strain MW37 containing non-           ribosomal proteins S16 L19 encoded trmD operon <br /> polar mutation D21K-2 did assembly defect (Fig.        essential E. coli 21K TrmD proteins <br /> 2F). These findings indicate assembly defect in-       important roles high growth rates. In contrast, the <br />  VOL. 177, 1995                                                              DELETION ANALYSIS OF GENES IN E. COLI                                5559 <br />  <br />  <br /> 16-kDa protein encoded gene downstream from,              21K-deficient strain possibility TrmD <br /> convergently transcribed to, trmD operon, nones-           protein different functions. <br /> sential, conditions examined here. <br />    The DtrmD-1 DtrmD-2 mutants described report                                    ACKNOWLEDGMENTS <br /> grew severalfold slowly S. typhimurium trmD <br /> mutant (data shown) lacks tRNA(m1G37)methyl-                  Hermann Bujard Bernt Eric Uhlin gratefully acknowledged <br /> transferase activity (7). This finding indicates TrmD       providing plasmids pUHE21-2 pHMG64, respectively. We <br />                                                                      gratefully acknowledge Kristina Nilsson outstanding technical as- <br /> protein functions. Interestingly, tRNA-       sistance, &#195;&#8230;sa Magnusson running sucrose gradients, Lisbet <br /> modifying enzyme, tRNA(m5U54)methyltransferase,            Lind help graphics. Glenn Bjo      &#194;&#168;rk Tord <br /> to functions: insertion 39 gene       Hagervall acknowledged helpful comments earlier ver- <br /> destroys tRNA-modifying activity results 4%        sion report. <br /> growth rate reduction, insertion upstream            During work, P.M.W. recipient postdoctoral <br /> is lethal (36).                                                      fellowship Swedish Natural Science Research Council (B-PD <br />    A large number mutants ribosomes contain         9911). This project supported grant Swedish Natural <br /> altered, lack, ribosomal proteins isolated         Science Research Council (B-BU 9911) P.M.W. grants to <br /> after heavy chemical mutagenesis (21, 22) antibiotic-          D.E.B. National Institutes Health (HG-00563 DK <br />                                                                      48029). B.C.P. supported grants Glenn R. Bjo    &#194;&#168;rk the <br /> independent revertants mutant strains depended            Swedish Cancer Society (project 680) Swedish Natural Science <br /> antibiotics growth (14, 19). Among mutants tem-         Research Council (B-BU 2930). <br /> perature sensitive growth, altered forms S16 <br /> and L19 (22). It assumed temperature sensitivity                                         REFERENCES <br /> was alterations ribosomal proteins, indicating          1. Bachmann, B. J. 1987. Derivations genotypes mutant deriva- <br /> their essentiality E. coli, possibility         tives Escherichia coli K-12, p. 1190&#226;&#8364;&#8220;1219. In F. C. Neidhardt, J. L. Ingra- <br /> temperature sensitivity mutations genes              ham, K. B. Low, B. Magasanik, M. Schaechter, H. E. Umbarger (ed.), <br />                                                                          Escherichia coli Salmonella typhimurium: cellular molecular biology, <br /> was excluded. Using reverse genetics, dem-              vol. 2. American Society Microbiology, Washington, D.C. <br /> onstrated protein S16 essential growth. Previously     2. Bachmann, B. J. 1990. Linkage map Escherichia coli K-12, edition 8. <br /> was assembly rate vitro 30S subunits             Microbiol. Rev. 54:130&#226;&#8364;&#8220;197. <br /> dramatically reduced protein S16 omitted, ribo-        3. Bernstein, H. D., M. A. Poritz, K. Strub, P. J. Hoben, S. Brenner, P. <br />                                                                          Walter. 1989. Model signal sequence recognition amino-acid se- <br /> somes lacking S16 functional vitro (20). These find-        quence 54K subunit signal recognition particle. Nature (London) 340: <br /> ings suggest inability cells lacking S16 survive       482&#226;&#8364;&#8220;486. <br /> due reduced assembly rate 30S subunits. We          4. Bertani, G. 1951. Studies lysogenesis. I. The mode phage liberation by <br /> that protein L19 essential, transductants lacking L19           lysogenic Escherichia coli. J. Bacteriol. 62:293&#226;&#8364;&#8220;300. <br />                                                                       5. Bjo&#194;&#168;rk, G. R. 1987. Modification stable RNA, p. 719&#226;&#8364;&#8220;731. In F. C. Nei- <br /> did survive restreaking. Further, reduced amounts L19             dhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, H. E. <br /> due polar effects substitution nptI::Ptac 21K             Umbarger (ed.), Escherichia coli Salmonella typhimurium: cellular and <br /> (D21K-1) resulted deficiency assembly 50S             molecular biology, vol. 2. American Society Microbiology, Washington, <br /> subunit ribosome. Three precursor particles             D.C. <br />                                                                       6. Bjo&#194;&#168;rk, G. R., J. U. Ericson, C. E. D. Gustafsson, T. G. Hagervall, Y. H. <br /> identified 50S subunit assembly, p150S, p250S p350S,          Jo&#194;&#168;nsson, P. M. Wikstro   &#194;&#168;m. 1987. Transfer RNA modification. Annu. <br /> sedimenting 34S, 43S, &#226;&#8364;&#732;&#226;&#8364;&#732;near 50S,&#226;&#8364;&#8482;&#226;&#8364;&#8482; respectively (34). We         Rev. Biochem. 56:263&#226;&#8364;&#8220;287. <br /> propose assembly intermediate, 50S*, observed         7. Bjo&#194;&#168;rk, G. R., P. M. Wikstro&#194;&#168;m, A. S. Bystro&#194;&#168;m. 1989. Prevention of <br /> experiments corresponds 43S precursor particle,              translational frameshifting modified nucleoside 1-methylguanosine. <br />                                                                          Science 244:986&#226;&#8364;&#8220;989. <br /> the difference 50S* lacks protein L19. Evidently, L19         8. Bystro                       &#194;&#168;rk. 1982. Chromosomal location cloning of <br />                                                                                &#194;&#168;m, A. S., G. R. Bjo <br /> limiting conversion 43S precursor particles            gene (trmD) responsible synthesis tRNA(m1G)methyltrans- <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;near 50S&#226;&#8364;&#8482;&#226;&#8364;&#8482; particles D21K-1 mutant L19              ferase Escherichia coli K-12. Mol. Gen. Genet. 188:440&#226;&#8364;&#8220;446. <br /> has usually regarded assembly protein (34). We         9. Bystro&#194;&#168;m, A. S., G. R. Bjo&#194;&#168;rk. 1982. The structural gene (trmD) the <br />                                                                          tRNA(m1G)methyltransferase polypeptide operon Esch- <br /> also note addition IPTG relieved assembly          erichia coli K-12. Mol. Gen. Genet. 188:447&#226;&#8364;&#8220;454. <br /> defect did increase growth rate D21K-1            10. Bystro&#194;&#168;m, A. S., K. J. Hjalmarsson, P. M. Wikstro                  &#194;&#168;rk. 1983. <br />                                                                                                                             &#194;&#168;m, G. R. Bjo <br /> mutant, growth rate probably limited num-             The nucleotide sequence Escherichia coli operon containing genes for <br /> ber mature 50S subunits. That is, surplus           tRNA(m1G)methyltransferase, ribosomal proteins S16 L19 and <br />                                                                          21-K polypeptide. EMBO J. 2:899&#226;&#8364;&#8220;905. <br /> ribosomes mutant low growth rate imposed        11. Bystro&#194;&#168;m, A. S., A. von Gabain, G. R. Bjo&#194;&#168;rk. 1989. Differentially ex- <br /> lack 21K protein. On hand, growth rate           pressed trmD ribosomal protein operon Escherichia coli transcribed as <br /> the DrplS-1 mutant expressing L19 plasmid pMW271               polycistronic mRNA species. J. Mol. Biol. 208:575&#226;&#8364;&#8220;586. <br /> (Plac-rplS1 16K1) strictly dependent IPTG        12. Chow, W.-Y., D. E. Berg. 1988. Tn5tac1, derivative transposon Tn5 <br />                                                                          generates conditional mutations. Proc. Natl. Acad. Sci. USA 85:6468&#226;&#8364;&#8220; <br /> added (data shown). Previously, slowly growing mutants               6472. <br /> completely lacking protein L19 isolated (47). How-         13. Connolly, T., R. Gilmore. 1986. Formation functional ribosome- <br /> ever, L19-lacking strains isolated anti-           membrane junction translocation requires participation GTP- <br /> biotic-independent revertants erythromycin-dependent               binding protein. J. Cell Biol. 103:2253&#226;&#8364;&#8220;2261. <br />                                                                      14. Dabbs, E. R., H. G. Wittmann. 1976. A strain Escherichia coli which <br /> strain, excluded compensatory muta-              gives rise mutations large number ribosomal proteins. Mol. Gen. <br /> tions present L19-deficient mutants (47). Interest-          Genet. 149:303&#226;&#8364;&#8220;309. <br /> ingly, L19-lacking transductants reverted       15. Boer, H. A., L. J. Comstock, M. Vasser. 1983. The tac promoter: a <br /> a low frequency clones survived restreaking. We spec-            functional hybrid derived trp lac promoters. Proc. Natl. Acad. <br />                                                                          Sci. USA 80:21&#226;&#8364;&#8220;25. <br /> ulate compensatory mutations allowing viability          16. Go &#194;&#168;ransson, M., B. E. Uhlin. Unpublished results. <br /> L19-lacking cells act correcting observed assembly      17. Guthrie, C., H. Nashimoto, M. Nomura. 1969. Structure function of <br /> defect.                                                                  Escherichia coli ribosomes. VIII. Cold-sensitive mutants defective ribo- <br />    We shown possible systematically inactivate,       assembly. Proc. Natl. Acad. Sci. USA 63:384&#226;&#8364;&#8220;391. <br />                                                                      18. Hagervall, T. G., T. M. F. Tuohy, J. F. Atkins, G. R. Bjo&#194;&#168;rk. 1993. <br /> and distinguish essential nonessential, genes             Deficiency 1-methyl-guanosine tRNA Salmonella typhimurium <br /> the E. coli chromosome using reverse genetics,             induces frameshifting quadruplet translocation. J. Mol. Biol. 232:756&#226;&#8364;&#8220;765. <br /> currently investigating molecular basis slow growth      19. Hasenbank, R., C. Guthrie, G. Sto   &#194;&#168;ffler, H. G. Wittmann, L. Rosen, D. <br />  5560         PERSSON ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />       Apirion. 1973. Electrophoretic immunological studies ribosomal pro-            malian signal recognition particle. Science 250:1 <br /> </body></html>