<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1989 Genetics Societyof America <br />  <br />  <br />  <br />   Identification Characterizationof New Geneof Escherichia coli K-12 <br />                   Involved Outer Membrane Permeability <br />  <br />                              Barbara A. Sampson,&apos; Rajeev Misra Spencer A. Benson&apos; <br />                               Department Biology, Princeton University, Princeton, New Jersey 08544-101 4 <br />                                                Manuscript received November 10, 1988 <br />                                                Accepted publication March 17, 1989 <br />  <br />                                                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  Using geneticselection mutations allow large maltodextrins cross outer membrane <br />               Escherichia coli absence LamB maltoporin, obtained characterized two <br />               mutations definea new locus E. coli. We designated locus imp increased membrane <br />               permeability. Mapping studies imp gene resides approximately 1.2 min E. coli <br />               chromosome. T h e mutations alter permeability outer membrane resulting increased <br />               sensitivity detergents, antibiotics dyes. T h e mutations nonreverting codominant. <br />               Genetic analysis mutationssuggest imp gene essential gene. We general <br />               cloning strategy used clone dominant recessive alleles. Using technique, <br />               cloned wild-type mutant imp alleles low copy number plasmid. <br />  <br />  <br />  <br /> T     HE outer membrane Escherichia coli forms the <br />        permeability barrier cell. All sub- <br /> stances enter cell cross barrier. As <br />                                                           largerthanmaltotriose,a <br />                                                           porin required (WANDERSMAN, <br />                                                                                             functional LamB malto- <br />                                                                                                  SCHWARTZ <br />                                                           ENCI 1979). By starting LamB- strainand <br />                                                                                                             FER- <br />  <br /> such, provides protection variety agents  selecting growth mixture maltodextrins, <br /> including degradative enzymes, detergents, chemical       identified different classes mutants <br /> agents antibiotics. T h e structure mem-      (MISRA et al. 1987). We report ge- <br /> brane consists lipopolysaccharide (LPS) (outer leaf-   netic biochemical characterization these <br /> let), phospholipids (predominantly inner leaf-     classes mutants which, addition conferring a <br /> let), integral peripheral proteins [see    Dex+ <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span>, confers dramatic increase sen- <br /> NIKAIDO    VAARA (1985, 1987) afor     review  sitivity variety antibiotics, detergents anddyes. <br /> structure]. The permeability outer membrane        These mutations define new locus E. coli (imp) <br /> can altered mutations treatment    appears essential gene. <br /> cellswith various chemical agents EDTA <br /> (LEIVE 1965), local anesthetics (LABEDAN 1988)and                       MATERIALSANDMETHODS <br /> inhibitors LPS biosynthesis (GOLDMAN     et al. 1987). <br /> Mutations number genetic loci alter outer        Bacterial strains: Bacterial strains plasmids used in <br />                                                           study listed Table 1. <br /> membrane&apos;s permeability. These includemutations              Media chemicals: Minimal (M63) Luria(L) agar <br /> affecting LPS structure (COLEMAN       DESPHANDE broth media asdescribed SILHAVY, BERMAN           and <br />  1985), lipid biosynthesis (RAETZ andFOULDS1977),         ENQUIST(1 984). Maltodextrins purchased Pfan- <br /> the structure porins (BENSON,   OCCIand SAMP-      stiehl Laboratories,Waukegan, Illinois, andpurified by <br /> SON 1988; MISRA BENSON1988a,b SAMSON              dialysis described previously (BENSON, OCCIand SAMPSON <br />                                                           1988). Antibiotic sensitivity disks purchasedfrom <br /> BENSON1985), mutations acrA (HENSON           Difco Laboratories, Detroit,Michigan, BBL Microbiolog- <br /> and WALKER 1982),    abs (CLARK 1984)   tolC (WHIT-   ical Systems, Cockeysville, Maryland. [&apos;4C]Maltose pur- <br /> NEY 1971), colicin tolerant mutations    chased Amersham Corp., Arlington Heights, Illinois. <br /> (NAGELDE ZWAIC andLURIA1967;                DAVIESand     Novobiocin (Nov), erythromycin (Em), kasugamycin <br /> REEVES1975) result increased permeability        (Ksg) purchased fromSigma Chemical Corp., St. Louis, <br />                                                           Missouri, used 50 pg/ml (Nov) 25 pg/ml (Em <br /> by unknown mechanisms.                                    Ksg). Arginine uracil added minimal media <br />    We described genetic selection mutants      concentration 33 pg/ml. <br /> with increased outer membranepermeability (BENSON            Geneticmanipulations: Procedures involving F&apos; mat- <br /> and DECLOUX     1985). This selection relies fact  ings, Hfr mapping, P1 transductions TnlO pools are <br /> that order E. coli grow maltodextrins        described SILHAVY,BERMANand ENQUIST(1984) and <br />                                                                             MILLER(1972). T h e ksgA560::TnlO insertion isolated <br />                                                                             TnlO pool prepared MC4100. We capitalized <br />    &apos; Present address: Rockefeller University, New York, New York 10021-     fact insertions ksgA gene confer resist- <br /> 6329. <br />      Present address: Department Microbiology, University Maryland,   ance kasugamycin (Ksg&apos;) (ANDRESSON     DAVIES1980). <br /> College Park, Maryland 20742-445 1.                                         A P1 lysate prepared MC4100 TnlO pool used <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Genetics 122: 491-501 (July, 1989) <br />      492                                          B. A. Sampson, R. Misra S. A. Benson <br />                                                                     TABLE 1 <br />                                                           Bacterial strains plasmids <br />  <br />                      Strain <br />                    plasmid                                  Genotype                                                                   Source <br />  <br />                 Strains <br />                   Jef8                   HFR thrA carB8 relA1 metBl spoTl                          GICOTet al. (1984) <br />                   SE14                   F&#226;&#8364;&#8482; 104/ thr-1 leuB6 A(gpt-proA62) recAargE3 <br />                                                                                  13                M. SCHMIDT <br />                                             thi-1 ara-14 lacy1 galK2 xyl-7 mtl-1 tsx-33 <br />                                             supE44 hisG4 rpsL3 1 <br />                   MC4 100                F- araD 139 A(argF-1ac)U1698 rpsL 150 relA                CASADABAN (1 976) <br />                                            JbB5301 ptsF25 deoCl <br />                   MC 1000                F- araDl 39 A(araABC-leu)7679galU galK                    M. CASADABAN <br />                                             A(lac)X74 rpsL thi <br />                     MCR106 <br />                      100 <br />                       MC4                           AlamB 106                                      EMRand SILHAVY <br />                                                                                                                 (1 980) <br /> 6   MCR             DME553                           AompF80                                       This study <br />                     ECB4623              araDl 39 galK A(lac)X74 rpsL thi imp-42 This <br />                                                                                     13                  study <br />           ThisF&#226;&#8364;&#8482;104/MCR106 <br />                imp-4213 <br />                 thr-43::Tn/U <br />                     BAS803                                                                              study <br />                                             sr15 17::Tn5 recA56 <br />                   BAS849                  MCR <br />                                            This106 imp-42 13                                            study <br />                   BAS  1329 <br />                      MCRlO6                This <br />                            A(araABC-leu)7679 <br />                         zab::TnlU       imp-4213                                                        study <br />                                             sr15 17::Tn5 recA56 <br />                   BAS1335 <br />                      MCRlO6 <br />                        1eu::TnlO                              imp-208 sr1517::Tn5 recA56           This study <br />                   BAS This <br />                        1340         1329 F&#226;&#8364;&#8482; 104/BAS                                                     study <br />                   BAS This <br />                        1347         1335 F&#226;&#8364;&#732; 104/BAS                                                     study <br />                   BAS1 <br />                     F&#226;&#8364;&#8482;104-1/MCR106 <br />                         365 zab::Tn/O <br />                                A(araABC-leu)7679 <br />                                            This                                                         study <br />                                             imp-421 3 recA::Km <br />                   BAS 1362 <br />                        MCRlO6 <br />                             imp-4213 <br />                                 srl300::Tn/O                               recA56                  This study <br />                   BASMCRlO6 <br />                        1363 <br />                          sr1300::TnIO                            recA56                            This study <br />                   RAMMCRlO6 <br />                         105                         AompF80 zei06::TnlO                            MISRAand BENSON(1988b) <br />                   RAMMCRlO6 <br />                         127                         AopmF80 imp-208                                MISRAand BENSON( 1 988a) <br />                 Plasmids <br />                   PRAM 1005               Km&#226;&#8364;&#8482;, pSC 10 1 replicon                                   This study <br />                   pSAB5402 <br />                         KmR,                    vector pRAM1005 based, <br />                                                                     cloned      imp gene           This study <br />                   pSAB5420                Km&#226;&#8364;&#8482;, vector PRAM 1005 basedThisimp-42 <br />                                                                             3 1                         study <br />  <br />  <br />  <br />     transduce MCRIO6 tetracycline resistance (Tc&#226;&#8364;&#732;). Of ap-                electrophoresis described (MISRA BENSON <br />     proximately 2000 transductants, Ksg&#226;&#8364;&#732;. To deter-                1988a). <br />     ksgA::TnlO insertions, checked link-                  DNA methods: DNA methods described MISRA <br />     age thecarB (1 &#226;&#8364;&#8482;) leu (2&#226;&#8364;&#8482;) genetic markers. In case           BENSON(1 988b). <br />     (ksgA560::TnlO) demonstrate TnlO de-                      Construction pRAM1005 chromosomal clone <br />     terminant linked c d leu genes showed               bank: A partial restriction map pRAM1005 shown in <br />     &gt;90% linkage imp gene. T h e presence carB               Figure 1. PRAM 1005 unique BamHI EcoRI cloning <br />     <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> assayed plating cells glucose minimal                 sites pSClOl replicon system. It constructed in <br />     plates plus minus supplementation arginine                  steps pRAM1001 (MISRAand BENSON1988b) <br />     uracil (GIGOTet al. 1980).                                               basically inserting HindIII-BamHI fragment carrying <br />        F&#226;&#8364;&#8482; 104-1 constructed recombining thr                   kanamycin resistance determinantfrompNEO          (P-L <br />     carB8 mutations Jef8 ontoF&#226;&#8364;&#8482; 104 series genetic              Biochemicals, Milwaukee, Wisconsin) HinDIII <br />     crosses (datanot      shown). F&#226;&#8364;&#8482;104-1 complement                  EcoRI sites PRAM 1001. Chromosomal DNA was <br />     1eu::TnlO insertion does complement carB8                    prepared RAM105, digested BamHI then <br />     mutation. F&#226;&#8364;&#8482;104-1 crossed BAS1314 yield                 ligated intopRAM1005 hadbeendigested            with <br />     strain BAS1 361. We introduced recA11l::Km in-                  BamHI treated alkaline phosphatase. T h e ligated <br />     sertion strain generate strain BAS1365. T h e               mixture transformedinto        MC1000. Transformants <br />     recA 1 11 ::Km insertion kindly provided GREGPHIL-                selected L plates containing kanamycin. Approxi- <br />     LIPS insertion 1.4-kb fragment carrying the <br />                                                                              mately 10,000 transformants pooled plasmid DNA <br />     kanamycin determinantfrom pUC-4-K (PharmaciaInc.,                        extracted pool. <br />     Piscataway, New Jersey) PstI site recA gene <br />     (G. PHILLIPS, personal communication).                                                          RESULTS <br />        Phenotypic tests:Detergent andantibiotic sensitivity tests <br />     described previously (BENSONand DECLOUX                        Isolation imp-4213 imp-208 mutants: <br />     1985). Growth tests described (MISRAand                     We previously described  selection obtaining <br />     BENSON1988a) dialyzed maltodextrin used <br />     instead defined size maltodextrins.                                   mutations affect outer membrane permeability. <br />        Biochemical tests: [&#226;&#8364;&#732;4C]Maltose uptake assays, prepara-               T h e selection growth maltodextrins the <br />     tions of&#226;&#8364;&#732; whole-cell envelopes SDS-polyacrylamide gel                absence LamB maltoporin, yields      pre- <br />                                                     Permeability Mutants E. coli                                                    493 <br />                                                                                                   TABLE 2 <br />                                                                                       Sensitivity various compounds <br />  <br />                                                                              zone <br />                                                                                 lnhibition <br />                                                                                    Compound                                    (mm) <br />                                                                                  (concentration <br />                                                                             amount)or                     MCRlOGimp+ BAS849(imp-4213) <br />  <br />                                                                       Chemicals, Dyes Detergents <br />                                                                                          14 <br />                                                                         C u S 0 4 (1.19 mg)                    11 <br />                                                                         Deoxycholate ( 1 500 pg)                0               13 <br />                        Digemt    BomHI                                                                     0                8 <br />                                                                         Triton X-100 (750 pg) <br />                        T r o t w t h phonphat m 0                 SDS (750 pg)                            0               12 <br />                        Ligatowith  B o m H I digamtad                   Eosin Y (5%)                            0                7 <br />                        olwomomomal D N A ( &lt; - + ) <br />  <br />  <br />                    I <br />                                                                       Antibiotics <br />                        l s c t ~ m rt ranmi ormant 6               Tetracyclines <br />                        Saroon f o r Nov <br />                                              r                                            20 (30 pg)           14 <br />                                                                         Chlorotetracycline <br />                                                                         Oxytetracycline16  (30 pg)             15 <br />                                                                         Tetracycline (5 pg)                     9               13 <br />                                                                       Betalactams <br />                                                                         Ampicillin (50 pg)                     14              27 <br />                                                                                           29 pg) <br />                                                                         Carbenicillin ( 1 00                   22 <br />                                                                         Cephalothin (3015  pg)                 13 <br />                                                                         Penicillin G (10 units)                 0                0 <br />                                                                       Others <br />                                                                         Bacitracin ( 1 0 units)                 0              22 <br />                                                                         Chloramphenicol (5 fig)                 7              13 <br />                                                                         Erythromycin (15 pg)                    0              19 <br />                                                                         Naladixic Acid (5 pg)                  11              17 <br />                                                                         Novobiocin (30 pg)                      0              22 <br />                                                                         Rifampin (1 5 pg)                       8              26 <br />   FIGURE 1.-Partial    restriction <br />                                  maps pRAM1005 <br />                                                                  Trimethdprim (5   26 pg)               17 <br /> pSAB5402. T h e details isolationof pSAB5402 given the <br /> text. The plasmids drawn scale. T h e multicloning site      Sensitivity expressed size (mm) zone growth <br />                                                                      inhibition center disc.T h e size disc 6 mm. <br /> (MCS) present pRAM1005 following restriction sites; <br /> EcoRI-SalI-KpnI-SamI-B~mHI.    T h e chromosomal insert containing <br /> the imp gene cloned BamHI site. T h e sizes      1.2&#226;&#8364;&#8482; region E . coli chromosome linked <br /> plasmids approximate.                                            known TnlO markers region. In 50 <br />                                                                      independent transductions involving mutant imp <br /> dominantly mutants alterations major po-                 alleles pleiotropic effects conferred altera- <br /> rins, OmpF OmpC(MISRAet d . 1987). In addition                    tions, i.e., antibiotic sensitivity, detergent sensitivity <br /> to types mutations, obtained isolates,               Dex+ phenotype moved single <br /> 421 3 208, Dex+ mutation did                    <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. This suggests various phenotypes the <br /> map porin gene conferred sensitivity                imp-42 13 andimp-208 mutations caused single <br /> MacConkey agar (Mac&#226;&#8364;&#8482;)), Nov (Nov&#226;&#8364;&#8482;), Em (Em&#226;&#8364;&#8482;).                    genetic alterations. <br /> Isolate 421 3 200 spontaneous in-                       The sensitivity profile BAS849 (imp-4213) is <br /> dependent Dex+ mutantsobtainedfromMCRlO6,                            identical tothat original isolate 421 3. The <br /> and mutant did contain                     increased permeability extendstoa          diverse set of <br /> ompF(Dex) mutation. Isolate 208                      compoundsincludingdetergents,          metals, dyes, and <br /> 70 spontaneous independent Dex+ mutants obtained                     numerous antibiotics (Table 2). However, did not <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> RAM105. The exact frequency                      detect increase sensitivity aminoglyco- <br /> isolates occurred difficult obtain ap-              sides; amikacin, gentamycin, kanamycin neomy- <br /> peared colonies 4 6 days plating. The                    cin polymyxin, clindamycin, EDTA. For the <br /> estimated frequencyof ompF(Dex) ompC(Dex)                    detergents; deoxycholate SDS, large antibi- <br /> mutations 5 X lo-&#226;&#8364;&#8482; (BENSON,     OCCIand SAMPSON                  otics as,erythromycin, novobiocin, bacitracin, <br /> 1988)and     1 X         (MISRA BENSON 1988b),                   rifampin increase sensitivity dramatic. <br /> respectively. On basis, frequency 4213               T o quantify degree change determined by <br /> and 208 isolates 1 X 10&#226;&#8364;?&#226;&#8364;&#8482;.                        twofold dilution method minimum inhibitory <br />   Phenotypic characterization imp mutants:                    concentrations (MICs) rifampin (Rif), novobiocin, <br /> To ensure phenotypes represented     effects            erythromycin, ampicillin (Amp) chloroampheni- <br /> of single genetic alteration, transferred imp               col (Cm). The MIC values BAS849 are: Rif, &lt;0.5 <br /> mutations P1 transductionintostrainMCRlO6                         pg/ml; Nov, 2 pg/ml; Em, &lt;0.5 pg/ml; Amp, &lt;0.5 <br /> using linked TnlO marker.Genetic          studies (see             pg/ml; andCm,2pg/ml.           The MICvalues the <br /> below) showed imp mutations mapped                   parentstrain     MCRlO6 are: Rif, &gt;64pglrnl;        Nov, <br />  494                                  B. A. Sampson, R. Misra S. A. Benson <br />  <br /> &gt;128 pg/ml; Em, &gt;128 pg/ml; Amp, 2 pg/ml;                                           TABLE 3 <br /> Cm, 6 pg/ml. Thus, relative change sensitivity                     Three- four-factor mappingdata <br /> to large hydrophobic antibiotics (Nov, Em, Rif) <br /> can large orders magnitude. In con-                                   Genetic Markers <br />                                                                                                                      No. of <br /> trast, theincrease sensitivityto Amp Cm,             Cross      cad     krgA::TnlO    imp     leu      lsolates <br /> predominately cross outer membrane po- <br />                                                               A. Donor         -        Tc&apos;        4312    + <br /> rins, 4- 6-fold.                                      Recipient     +        Tc&apos;         Wt     - <br />    T o determine general structure outer                        -        Tc&apos;         wt      -                   23 <br /> membrane disrupted, assayed sensitivityto                          +        Tc&apos;         wt      -                   48 <br /> following bacteriophages: X, K20,Hy2, SS4, @80,                               +        Tc&apos;         wt      +                    4 <br /> Tu2, Tu4, T6, P1,       U3. In case did detect                      -        Tc&apos;         wt      +                    6 <br /> a difference sensitivity compared control                        +        Tc&apos;        4213     +                   35 <br /> strain MCRlO6. The imp-208 allele results nearly <br />                                                                               +        Tc&apos;        4213     -                   20 <br />                                                                               -        Tc&apos;        4213     +                    8 <br /> identical phenotypes introduced MCRlO6.                             -        Tc&apos;        4213     -                   14 <br />    Geneticcharacterization imp-4213 muta-                                                                  Total   158 <br /> tion: Our initial genetic mapping indicated          B. Donor        -         Tc&apos;        4313 <br /> mutation present isolate 421 3 mapped              Recipient     +        Tcs         Wt <br />  <br /> ompF omPC. T o determine map position                              +        Tc&apos;        4213                     60 <br /> the mutation conferred Dex&apos; increased                            +        Tc&apos;         Wt                      25 <br />                                                                               -        Tc&apos;        4213                     33 <br /> permeability phenotypes, did Hfr mapping using                             -        Tc&apos;         Wt                       4 <br /> Hfrdonor       strains <br />                      BW6164, BW6169,  BW6175                                  -        Tc&apos;        4213                      5 <br /> (WANNER1986)andHfrG6(HOFNUNG, HATIFIELD                                                                          Total    3 <br /> and SCHWARTZ 1974).       The results indicated      C. Donor                  Tc&apos;        4313    + <br /> imp mutation nearthe thr geneat 0&apos;                    Recipient              Tcs         Wt     - <br /> chromosome (data shown). T o define                                    Tc&apos;         wt      -               27 <br /> map location, did two-, three-,andfour-factor                                       Tc&apos;        4213     -               13 <br /> crosses using generalized transducing phage P 1&quot;Ir.                                Tc&apos;        4213     +               17 <br /> A sample mapping results shown Table                                     Tc&apos;         wt      +                2 <br /> 3 cotransduction linkage imp gene to <br />                                                                                                                   Total    59 <br /> adjacent loci given Figure 2. We noted          Transductions described MATERIALS AND METH- <br />                                                            ODS.  In cases, selection Tc&apos; transductants. The imp <br /> linkage imp-42 13 andwild-type alleles flank-    alleles scored sensitivity Nov Em Dex+ <br /> ing markerswas thesame. The linkage imp&apos;        phenotype, wt means wild type, 4213 means imp-4312 confered <br /> allele higher    linkage imp-    sensitivities. The carB leu alleles scored appropriate <br /> 42 13 allele markers, ksgA leu (Figure <br />                                                                                         + <br />                                                            auxotropy, - = auxotropic, = prototropic. <br />  <br /> 2). Since ksgA leu genes lie opposite sides <br />                                                            208/F&apos; 104 imp&apos;). The mutant Dex+ MacS phe- <br /> of imp gene, interpret mean a <br />                                                            notypes recessive NovSand Ems pheno- <br /> selective bias mutant imp alleles for <br />                                                            types dominant (Table 4). We noted the <br /> acquisition imp&apos; allele. <br />                                                            strain wasrecA&apos;     wild-type (imp&apos;)Nov&apos;    MacrDex- <br />    The gene order, based mapping studies, is <br />                                                            colonies quickly accumulated culture. These <br /> carB-ksgA-imp-ara-leu estimate map posi- <br /> tion imp approximately 1.2&apos; E. coli     colonies presumably arise recombinational loss (via <br /> chromosome. The position rab4292::Tn5           homogenization) mutant allele wild type. As <br /> zab::TnlO shown informational purposes.            mapping studies, suggests a <br />    The imp-208 allele maps position     selective bias mutant allele. <br /> imp-4213 mutation. T o determine ifwe               The detergent drug sensitivities conferred by <br /> recombination mutant alleles,           imp mutations provided convenient       means to <br /> crossed mutation selecting       select revertants. We did ways; selection <br /> the adjacent KsgA560::TnlO o r leu+ marker. We did         Mac&apos;, Novr Em&apos; colonies. Spontaneous Macr <br /> notdetect wild-type recombinantsamong              colonies occur frequency approximately 2 X <br /> &gt;900 transductants screened. This suggests        10&quot;j. Of 20 spontaneousand 2-amino-purine induced <br /> two mutations close other,         Mac&apos; revertants tested, pseudorevertants. In <br /> identical alterations, overlapping deletions.           able rescue imp-4213 allele from <br />    Diploid reversion analysisof imp alleles:       thesestrains (D. WOODWORTH,          unpublisheddata). <br /> T o determine imp-42 13 andimp-208 mutations        Selection spontaneous Nov&apos; Em&apos; revertants <br /> aredominantor        recessive, constructed strains     37&quot; true revertants pseudorever- <br /> BASl 340 (imp-4213/F&apos; 104imp&apos;) BASl 347 (imp-          tants. However, mutagenized cultures pos- <br />                                                             Permeability Mutantsof E. coli                                                       495 <br />  <br />  <br />  <br />  <br />                                                                                                               FIGURE2.-Linkage         map E. <br />                                                                                                            coli chromosomalregionsurround- <br />                                                                                                            ing imp gene. Linkages de- <br />                                                                                                            termined twofactor PI cotrans- <br />                                                                                                            ductions.Numbers        brackets    ) <br />                                                                                                            represent frequency thewild- <br />                                                                                                            type imp allele. All numbers <br />                                                                                                            determined using theimp-42 1 3 <br />                                                                                                            allele. In cases, numbers rep- <br />                                                                                                            resenttheaverageofseveralinde- <br />                                                                                                            pendent transduction experiments. <br />  <br />  <br />  <br />  <br />         I                                  I                     I        I <br />         I                                  I                     I        I <br />  <br />  <br />  <br />                                  TABLE 4                                                                 TABLE 5 <br />                      Diploid analysis imp alleles                                    Reversion analysis imp alleles <br />  <br />                                                      Phenotypes                                                                      Reversion <br />                                                                                                                      Reversion      L-Tc-Nov <br />                       Strain                   Dex   Mac    Nov      Em                      Strain               L-Nov plates      plates <br />  <br />       MCRlO6 (wild type)                       -     R     R         R         BASl 329 <br />                                                                                (imp-4213 <br />                                                                                  zab::TnlO) <br />                                                                                    &gt;4                                    X   10-9         ND <br />       BAS1329 <br />                                                                                BASl 340 <br />      zab::TnlO) <br />       (imp-42 13                               +      s      s       s         (imp-4213 <br />                                                                                        zab::TnlO/F&apos;104imp+)5.4           X   lo-&apos;   4.6   X <br />       BASl 340                                                                 BAS1335 <br />           zab::TnIO/F&apos;104imp+) <br />       (imp-4213                                -     R     S         S         (imp-208 1eu::TnlO) <br />                                                                                           &gt;6                             X   lo-&apos;         ND <br />  <br />        BAS1335                                                                 BAS 1347 <br />        (imp-208 <br /> 1eu::TnlO)                                     +      ss       s                                         4.6 <br />                                                                                (imp-208 leu::TnlO/F&apos;104imp+)             X          5.4   X   lo-* <br />       BAS1347                                                                    All strains recA56. L-Nov = Luria medium 50 m/ml <br />       (imp-208 leu::TnlO/F&apos;104imp+)            -     R     S         S        novobiocin, L-Tc-Nov = Luria medium 25 p/ml tetracycline <br />                                                                               plus 50 p/ml novobiocin. ND = done. <br />          All strains MCRlO6 recA56. Phenotypes tested <br />       described MATERIALS AND METHODS. S = sensitive, R = resist-       When maintained selection Tc&apos; frequency <br />                            + <br />       ant, - = growth, = good growth 36 hr. <br />                                                                               Nov&apos; colonies reduced orders mag- <br />                                                                               nitude (Table 5 ) . One explanation findings <br />       sible obtain rare Nov&apos; Em&apos; pseudorevertants <br />                                                                               Nov&apos; colonies arise deletion events and <br />       (R. FORTIER, personal communication). <br />                                                                               eventsare stimulated adjacent TnlO. <br />          The imp gene appears essential: We noted <br />                                                                               T o test this, constructed strain BASl 365, de- <br />       diploid strains BAS1340 BAS1347, un- <br />                                                                               scribed MATERIALS AND METHODS. This strain car- <br />       like haploid strain derived, <br />                                                                               ries F&apos;104-1 carB8 allele allows <br />       segregated Novr colonies highfrequency. Both                         detect loss chromosomal curB+ allele as <br />       strains yield Nov&apos; colonies frequencies high                   shown Figure 3. Strain BAS1365 yields Novr Tcs <br />       lo-&apos; (Table 5) cases Ems phenotype                    CarB- colonies atfrequency <br />                                                                                                                           thus <br />       simultaneously lost. One explanation unusual                   showing Nov&apos; colonies arise deletions <br />       finding colonies arise inactivation                 remove TnlO thecarB gene. <br />       mutant chromosomalimp allele. Many Nov&apos;                         Two observations suggest imp mutation, <br />       colonies Tc&apos;. In fact, able detect                 unusual property TnlO, that <br />       Tcs colonies imposing Nov&apos; selection                     results apparent deletion events. Strains which <br />       simple replica plate techniques(datanot        shown).                  diploid wild type imp allele yield <br />  496                                      B. A. Sampson, R. Misra S. A. Benson <br />  <br />  <br />  <br />  <br />                                                                   - <br />   t hr <br />           +                  carfl   -                 mx) <br />                                                               +                  IOU <br />                                                                                            + <br />  <br />   t hr <br />           +                  carB <br />                                      t <br />                                          306:   rnlO <br />                                                                   p   ara-l ou <br />       I               I        I           1    1 <br />       I               I                    I             I        I                            I <br />    0&#226;&#8364;&#8482;                                     &#226;&#8364;? mp-42 13                                         2&#226;&#8364;&#8482; <br />  <br />  <br />                                                                                                        FIGURE3.&#226;&#8364;?Spontaneous          inacti- <br />                                                                           rrcA:: Ktn                vation mutant imp allele a <br />                                                                                  I <br />                                                                                                     deletion event. T h e rationale and <br />                                                                                                     results thedepictedexperiment <br />                                                                                                     described    text.      The <br />                                                                                                     straight lines relevant genes <br />                                                                                                     present F&#226;&#8364;&#8482; 104-1, it, the <br />                                                                                                     oval depicts relevant genes pre- <br />                                                                                                     senton chromosome. T h e phe- <br />                                                                                                     notype starting diploid strain <br />                                                                                                     represented upper the <br />                                                                                                     figure Km&#226;&#8364;&#8482; Novs Erns Tc&#226;&#8364;&#8482; and <br />                                                                                                     CarB+. T h e lower figure <br />                                                                                                     shows thepredictedstructureafter <br />                                                                                                     deletion event. T h e phenotype of <br />                                                                                                     resulting diploid deletion strains <br />  <br />   t hr+                      carfl   -                 mp <br />                                                               +              1       B U <br />                                                                                            +        shown lower dia- <br />                                                                                                     gram Km&#226;&#8364;&#8482; <br />                                                                                                                   Nov&#226;&#8364;&#8482; Em&#226;&#8364;&#732; Tc&#226;&#8364;&#8482; and <br />                                                                                                     CarB-. <br />  <br />   thrt <br />  <br />  <br />  <br />  <br />                                                                           recA:: B r <br />                                                                                  I <br />  <br />  <br />  <br />  <br /> Tc&#226;&#8364;?colonies, different TnZOs (zab::TnlO                   minimal media (maltodextrins glucose) porin <br /> leu::TnlO) stimulate frequency Nov&#226;&#8364;&#8482; col-                  minus strains regardless imp allele carried <br /> onies (Table 5). However, close byTnZOs                    reduced growth rate. This slower growth phe- <br /> so. Strain BAS803 (thr::TnlO imp-4213/F&#226;&#8364;&#8482;104 imp&#226;&#8364;&#8482;)                     notype indicative porin-deficient phenotype. <br /> does yield Tcs colonies, frequency                    Biochemicalanalysis imp mutants: As a <br /> Novr &#226;&#8364;&#339;revertants&#226;&#8364;? orders magnitude                 crude assay outer membrane structure,we analyzed <br /> thanthat     seen similar diploids carrying                  protein profile cell envelopes soluble <br /> zab::TnlO 1eu::TnlO.                                           fractions prepared fromstrains carrying theimp-42 13, <br />    These observations, coupled fact hap-                imp-208, imp+ alleles. We detect differ- <br /> loid strains revert yield Tcs colonies re-                 ence protein profiles SDS-PAGE gels any <br /> gardless imp TnlOalleles carry, suggest                fractions (data notshown). <br /> that imp gene essential explain                     T o quantitatethe    changes outermembrane <br /> we inactivate mutant imp allele                permeability imp mutations, did growth <br /> diploids.                                                             tests [&#226;&#8364;&#732;4C]maltose uptake assays. In complex me- <br />    Porin independence imp phenotypes: To                       dia strain BAS849(imp-4213) andthe imp&#226;&#8364;&#8482; parent <br /> determine phenotypesconferred imp                       MCR 106 nearly identical growth rates (Fig. 4A). <br /> mutations operated altering properties                  Growth tests using 10 mM maltodextrin carbon <br /> OmpF OmpCporins, constructed strains lack-                     source presence imp mutation <br /> ing bothporins. Strains lack                 increases growth rate (Figure 4B). At higher con- <br /> porins retain imp associated phenotypes i.e.,              centrations maltodextrins (16 mM) strains <br /> Nov&#226;&#8364;&#8482;, Em&#226;&#8364;&#8482;, Macs Dex+ (datanot shown). Thus,                       identical growthrates(datanot        shown). Both <br /> sensitivities conferred imp mutation                    imp mutations increase therateat        labelled <br /> require presence porins. It noted                 maltose taken cell (Figure 5). This best <br />                                                     Permeability Mutants E. coli                                                 497 <br />  <br />  <br />  <br />                                                                             16 <br />  <br /> 0 <br /> 0                                                                           12 <br /> (0 <br />  <br />  <br /> Q0                                                                           8 <br />  <br />  <br />                                                                               4 <br />  <br />  <br />      0.104                                                        1           0- <br />          0      30     60      90     120    150     180    210                   0       20          40          60          SO <br />                                                                       7 <br />                                                                                                 Time (sec) <br />      1 .oo <br />                                                                         FIGURE5.-Rate       [&apos;4C]maltoseuptake imp mutants. <br />                                                                       Uptake experiments described       MATERIALS AND <br />                                                                       METHODS. RAM105      (OmpF-        OmPC&apos; imp&apos;), open triangles; <br /> 0                                                                     MCRlO6(OmpF&apos;OmpC&apos;            imp&apos;), opencircles;BAS849(OmpF&apos; <br /> 0                                                                     OmpC&apos;imp-4213), filled circles; RAM127 (OmpF- O m p C <br /> (0 <br />                                                                       imp-208), filled triangles. <br /> Q0                                                                    frequency. The plasmids colonies iden- <br />      0.1 0 <br />                                                                       tical BamHIrestriction patterns (data notshown). One <br />                                                                       plasmid, pSAB5402, chosen analysis. <br />                                                                       Apartialrestrictionmap        ofplasmid pSAB5402 is <br />      0.05                                                             given Figure 1. This plasmid carries insert of <br />                                                                       approximately16       kb. To provethat      plasmid <br />                              Time (min)                               carried imp gene, recombined imp-4213 <br />    FIGURE4.--Growth test imp&apos; imp-4312 strains. (A) Resultsof      mutation plasmid follows. We introduced <br /> growth tests LB. (B) Results growth test 10 mM maltodex-     pSAB5402 BAS849 (imp-4213), extracted plas- <br /> trin. MCRlO6 (imp&apos;) = open circles BASS49 (imp-4213) = filled         midDNA       andtransformed       DNA into <br /> circles. <br />                                                                       BAS849 selecting plasmid encoded Km&apos; determi- <br />                                                                       nant screened transformantswhich were <br /> seen strain DME553 (LamB- OmpF- OmpC+ <br />                                                                       Dex+ Mac&apos; (diploid theimp-42 13 allele). Recall <br /> imp&apos;) compared strain RAM 127 (LamB- OmpF- <br /> OmpC+ imp-208). StrainMCRlO6 (LamB- OmpF+                             Dex+ MacSphenotypes recessive and <br /> OmpC+ imp&apos;) higher basal rate uptake                      imp-4213 allele present the <br /> comparedto DME553 presence                          plasmid expected phenotypes the <br /> OmpF porin, functionally larger porin                 transformants.Among350          Km&apos; transformants, we <br /> OmpC (NIKAIDOand VAARA 1987; MISRA BEN-                           obtained colonies Dex+ Mac&apos;. <br /> SON 1988a). In cases, rate uptake conferred                Further analysis plasmid (pSAB5420) fromone <br /> by imp mutations 10 15% ob-                    clones showed restriction <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> served LamB+ strains (data shown).                           identical pSAB5402, introduced <br />    Cloning imp gene low copy number                     BAS1363(imp+) conferred Nov&apos; Dex+ phe- <br /> plasmid: A clone bank constructed using low                   notype (Table 6). This proved physically geneti- <br /> copy number vector pRAM1005 described MA-                       cally plasmid carried imp-42 13 allele. The <br /> TERIALS AND METHODSTO        identify clone                 fact restriction map identical pSAB5402 <br /> carried plasmid encoding imp gene,                      suggests imp-4213 mutation does result from <br /> use fact imp-42 13 mutation rapidly                gross rearrangement DNA. We estimated we <br /> lost recombinational correction (see above). Plas-                 detect deletionsor insertions 200 bpor more. <br /> midDNA fromthe BamHI clone bank trans-                               The codominance imp mutations suggest that <br /> formed strain ECB4623. This strain derived                    result production altered gene product, <br /> from MClOOO whichhas increased transformation                         phenotypes seen F&apos; diploids reflect <br /> efficiency plasmid DNA. It necessary use                  contributions mutant wild-type gene <br /> this background imp mutationsreduce                       products. Thus, reasoned gene dosage <br /> transformation efficiency 30- 50-fold (data                 alleles increased result in <br /> shown). Out approximately 2500 Km&apos; transform-                      stronger representationof phenotype conferred <br /> ants screened, resistant (grew on) Mac-                   gene product. This case (Table <br /> Conkey agar segregated      Nov&apos; colonies high                 6). When imp&apos; allele present insix         copies <br />  498                                              B. A. Sampson, R. Misra S. A. Benson <br />                             TABLE 6                                    accurately assign frequency mutations. It <br />   Effects mutant wild-type imp gene low copy           likely, mutations occurred    imposition of <br />                            plasmid                                     selection. As such, possible selection <br />                                                                        itselfmay influenced type mutants we <br />                  imp genotype      Disc sensitivity <br />                                                 zones <br />                                                                        obtained. Recent work CAIRNS, OVERBAUCH           and <br />                                                        Growth on <br />    StrainChromosome      Plasmid SDS D E 0 Eos Nov Em maltodextrin     MILLER(1988) suggest nonlethal selection the <br /> BAS1362      imp-4213 None <br />                       1410                  10 19 15        +          selection influence type mutation obtained. <br />                                                                           We believe mutations simple missence <br /> BAS 1362 imp-42 13 imp&apos;            0    0    0 10 (15)     -/+ <br /> pSAB5402                                                               null mutations based rarity      nonre- <br />                                                                        verting codominate nature, suggest re- <br /> BAS1362 imp-4213 imp-4213 13 <br />                           13                10 21 21       ++          sult similar identical genetic events resulting <br /> pSAB5402 <br />                                                                        gene product partial acquired func- <br /> BAS1363 imp&apos;            None       0    0    0    0 (8)     - <br /> BAS1363 imp&apos;imp&apos;                   0    0    0    0 (14)    - <br />                                                                        tion(s). This explanation account low <br /> pSAB5402                                                               frequency obtained mutations, and <br /> BAS1363 imp&apos;            imp-4213 10     0    0 15 (18)      +          fact unable detect recombination <br /> pSAB5420                                                               mutations. <br />                                                                           The imp mutations specific <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span> pair sub- <br />    The numbers indicate size (mm) killing zone. Number      stitutions, deletions insertions. The possibility <br /> given ( ) indicate zone growth inhibition, killing. + = <br /> growth, - = growth, -/+ = poor growth 36hr. SDS =                unlikely unable obtain true rever- <br /> sodium dodecyl sulfate, D E 0 = deoxycholate, Eos = eosin Y .          tants eitherspontaneously following treatment with <br />                                                                        2-AP, NTG, EMS (data shown). We favor the <br /> (BAS1362/pSAB5402) detergent, Nov&apos;, theDex+                        possibility mutationsresult specific dele- <br /> phenotypes reduced relative haploid strain                  tion events able detect <br /> BAS1362. It appears presence        imp&apos;               exact excisionof insertion reversion test. <br /> allele plasmid itself alter outer membrane                  Analysis imp-42 13 mutation pSAB5420 in- <br /> permeability slightly. We observe weak Dex+ phe-                     dicates mutation deletion, small <br /> notype BAS1362/pSAB5402 (six copies imp&apos;)                      (e200 bp) reside acoding <br /> whereas wild type gene provided single                  region fuse adjacent ones. <br /> copy F&apos; 104the diploid Dex-. Inaddition,                            Mapping mutations places imp gene at <br /> BAS1362/pSAB5402 shows slight increase Ems                        approximately 1.2 min E. coli chromosome <br /> compared BAS1362 (Table 6). When imp-4213                        ksgA m . The gene order obtained <br /> present plasmid (BAS1363/pSAB5402)                        four-factor mappingand consistent two- <br /> is increase sensitivity SDS, Nov Dex+                 factor cotransduction frequencies (Table3, Figure 2). <br /> phenotype (Table 6). The presence single imp&apos; <br />                                                                        However, high linkage ksgA, <br /> gene sufficient eliminate sensitivity deoxycho- <br />                                                                        zab4292::Tn5, zab::TnlO, imp gene the <br /> late reduce NOV&apos; (Table 6). When imp-4213 <br />                                                                        selective bias wild type imp allele, relative <br /> allele present plasmid chro- <br />                                                                        position suggested gene order viewed <br /> mosome (BAS1362/pSAB5420) phenotypes con- <br />                                                                        tentative confirmed physical analysisof the <br /> ferred mutation accentuated (Table 6). <br />                                                                        cloned gene. There nogenes region the <br />                                                                        chromosome known affect outer mem- <br />                          DISCUSSION                                    brane functions. Thus, themap position does give <br />    We utilized genetic selection gain insights               insight nature imp gene product its <br /> into barrier functions outer membrane                    physiological role cell. It does the <br /> E. coli. The selection based uptake                   mutations define new locus. <br /> large sugar, maltodextrin,  yields classes                 The mapping studies, fact strains dip- <br /> mutants unequal frequencies (MISRA et al. 1987;                     loid region segregate Novr colonies high <br /> BENSON,OCCIand SAMPSON        1988). One rare class                 frequency, suggest bias mutant <br /> mutations, imp, results sensitivity variety                 allele. The basis selection mutant <br /> unrelated compounds including detergents, antibiot-                    imp alleles known. The fact haploid <br /> ics dyes. The imp mutations characterized                   strains, carry single copy mutant imp <br /> this study nonreverting codominant muta-                      gene, significant growth disadvantage <br /> tions define new genetic locus appears                 complex media growth advantage in <br /> be essential genethat affects outermembrane                         maltodextrin clouds issue. One possibility <br /> permeability.                                                          disadvantage imposed imp mutation <br />    The imp mutants appeared Dex+ colonies                   exponential growth phase me- <br /> days afterplating and,therefore, difficult                    diates effects phase ofthe <br />                                              Permeability Mutants E. coli                                      499 <br />  <br /> growth cycle, viability stationary phase.        structure (MISRA BENSON198813; BENSON,OCCI <br />    The appearance Novr colonies dip-          SAMPSON     1988). Whether theimp mutations cause <br /> loid strains easily explained inactivation      similar alteration, impart analogous phenotype <br /> the mutant imp allele. This suggestion sup-        different mechanism known. The porin <br /> ported fact colonies lose imp          independence imp phenotypes indicates the <br /> associated phenotypes (Dex+ NovS Ems Mac&apos;). This           porins required increased phenotype. <br /> in contrast pseudorevertants obtained fromhaploid          Since imp mutants exhibit reduced growth the <br /> strains whichlose imp associated             absence bothporins, suggests thatthe imp <br /> sensitivities. Several factors contributetothe            mutations result production new porin <br /> apparent high frequency diploid strains seg-         type protein consistent fact altera- <br /> regate Novr colonies. These include strongselection         tion peptide profile envelope. In addition, <br /> for loss mutant allele, stimulation deletions       suggests diffusion certain substances across <br /> from adjacent TnlOs transposons      (KLECKNER,REI-       outer membraneis limiting growth imp <br /> CHARDT BOTSTEIN          1979) technical diffi-   porin minus mutants. <br /> culty starting population cells does           One explanation observed sensitivities and <br /> not contain preexisting Novr colonies. At         Dex+ phenotype general property the <br /> suggested frequency    (1 O-*         small inno-     outer membrane altered. However the <br /> culums cells contain preexisting Novr cells.          mutantsretain sensitivity bacteriophages <br />     Three lines evidence suggest imp gene      tested significant reduction in <br /> essential. Both imp mutations codominant         growth rate, defect appears result any <br /> are simple null mutations. Diploid strains inacti-        major disruption structure. We unable to <br /> vate themutant allele high frequency              aspecific outer membrane biochemical alteration <br /> eliminates increased permeability pheno-           imp mutants. One explanation the <br /> types; whereas, haploid strains unable this,        imp mutations alter minor component outer <br /> and direct selection reversal increased            membrane present assays sensitive <br /> permeability phenotypes yields pseudorevertants.         enough. In case tolC mutants, have <br /> Lastly, attempts obtain amber mutations, gene fu-          increased permeability phenotype,cloningand <br /> sions to, transposons imp gene            overexpression tolC geneproduct (a minor <br /> unsuccessful.                                                 outer membrane protein) required order to <br />     Diploid reversion analysis studies          biochemically detect alteration (MORONA, MAN- <br /> certain phenotypes conferred imp alterations           NING REEVES      1983). <br /> can separated, i.e., Dex+ MacSfrom NovSand                The fact detect changein sensitivity <br /> Em&apos;. One explanation forthis theimp alteration        LPS requiring bacteriophages, EDTA poly- <br /> effects different barrier functions outer          myxin suggests LPS structure dramatically <br /> membrane, recessive fashion             altered. This supported silver strain- <br /> dominant fashion. We favor alternative simpler            ing LPS imp-208 mutant gives <br /> explanation thevarious phenotypes reflect differ-        apattern identical parentalstrain(datanot <br /> ent levelsof increased permeability single           shown). However, preclude possibility <br /> barrier function. At present possible know       theimp mutations alterthe LPS subtlefashion <br /> which, either, alternatives correct.          detected assays. Recent work Roque <br />     Phenotypic analysisof imp mutantsdemon-               et al. (1988) hyperpermeableabs mutation has <br /> stratesthe outermembranebarrierfunctions                 shown alterations polycationic binding the <br /> been altered large number     compounds (Table       LPS result hyperpermeable phenotype. At <br /> 2 ) . Disc sensitivity tests imp mutations      present, know imp mutations alter <br /> increases sensitivity antibiotics.        property LPS. <br /> MIC determinations greater             T h e suggestion imp gene encodes essen- <br /> 100-fold increase sensitivity certain hydrophobic       tial cellular function implies gene involved <br /> antibiotics normally cross outer membrane           fundamentalcellular process. At present, theonly <br /> poorly small increase sensitivity      known component outer membrane essential <br /> to antibiotics predominately cross outer            cell viability lipid A, fatty acid component <br /> membrane porins. Interestingly, general           LPS (RAETZ 1987).T h e possibility lipid me- <br /> pattern sensitivities similar conferred      tabolism altered imp mutants attractive <br /> small in-frame deletionswithin porin genes    (MISRA      one, atthis time know case. <br /> and BENSON1988b; BENSON,OCCI SAMPSON                         We cloned imp&apos; geneontoa            low copy <br /> 1988). We suggested porin mutations            number plasmid used plasmid-chromosomal re- <br /> conferthesephenotype          alteringthemembrane          combination obtain imp-4213 allele. T h e clon- <br />  500                                             A. Sampson, <br />                                                 Misra <br />                                               and <br />                                              B.      R.                    S. A. Benson <br />  <br /> ing strategy general allows cloning                  CLARK, D., 1984 Novel antibiotic hypersensitive mutants Esch- <br /> both dominant recessive alleles. In case,                    erichia coli genetic mapping chemical characterization. <br />                                                                         FEMS Microbiol. Lett. 21: 189-195. <br /> wild type allele (Nov&apos; Mac&apos;) recessive mutant              COLEMAN,   W. G., JR., K. DESHPANDE,      1985 New cysE-pyrE- <br /> NovSphenotype, dominant Mac&apos; pheno-                         linked rfa mutation Escherichia coli K-12 result a <br /> type. We able identify correct clone                     heptoseless lipopolysaccharide.J. Bacteriol. 161: 1209- 1214. <br /> screeningfor thedominant,         Mac&apos;, plasmid deter-               DAVIES,J. K., P. REEVES, 1975 Genetics resistance to <br /> mined phenotype recombinational loss                      colcins Escherichia coli K-12: cross-resistance colicins <br />                                                                         group A. J. Bacteriol. 123: 102-1 17. <br /> chromosomal NovSphenotype. Most importantly,                     EMR,S., T . J. SILHAVY, 1980Mutations affecting localization <br /> system allowed demonstrate directly                   Escherichia coli outer membrane protein, bacteriophage <br /> cloned correct gene showing mutant                      lambda receptor. J. Mol. Biol. 141: 63-90. <br /> imp alleles berecombined ontothe plasmid                       GIGOT,D., M. CRABEEL,     A. FELLER,   D. CHARLIER,   W. LISSENS,   N. <br /> genome. This eliminated possibility ob-                    GLANSDORFF     A. PIERARD,     1980Patterns Polarity in <br />                                                                         Escherichia coli carAB gene cluster. J. Bacteriol. 143: 914- <br /> served phenotype resulted form overexpression                      920. <br /> unrelated cloned gene. The plasmids used                   GOLDMAN,    R., W. KOHLBRENNER,       P. LARTEYand A. PERNET, <br /> to demonstrate codominant natureof                       1957 Antibacterial agents specifically inhibiting lipopolysac- <br /> mutations gene dosage mutant                       charide synthesis. Nature 3 2 9 162-164. <br />                                                                      HENSON, J. M., J. R. WALKER,     1982 Genetic analysis arcA <br /> and wild-type alleles correlated degreeof the <br />                                                                         lir mutations Escherichia coli. J. Bacteriol. 152: 1301- <br /> imp conferred phenotypes.                                                1302. <br />    At present, understand molecular                    HOFNUNG,  M., D. HATFIELDand M. SCHWARTZ, 1974malB region <br /> basis phenotypes observed imp mutants,                   Escherichia coli K-12: characterization ofnew mutants. J. <br /> nor mechanisms mutations cause                     Bacteriol. 117: 40-47. <br />                                                                      KLECKNER, N.,   K. REICHARDT   D. BOTSTEIN,    1979 Inversions <br /> increased permeability. Thus, time, cannot <br />                                                                         deletions Salmonella chromosome generated the <br /> rule possibility effect outer                   translocatable tetracycline element TnIO. J. Mol.Biol. 127: <br /> membrane permeability indirect                      89-1 15. <br /> imp mutations alter process directly related               LABEDAN,  B., 1988 Increase permeability Escherichia coli <br /> some aspect envelope structure                     outer membrane local anesthetics penetration anti- <br />                                                                         biotics. Antimicrob. Agents Chemother. 32: 153-155. <br /> the cell wall inner membrane.Nonetheless, we <br />                                                                      LEIVE,L., 1965 Releaseof lipopolysaccharide EDTA treat- <br /> believe mutants define anew essential gene                   ment E. coli. Biochem. Biophys.Res. Commun. 21: 290- <br /> of E. coli provide new tools genetic analysis               296. <br /> of complexities outer membrane. Specifi-                  MILLER,J. H., 1972 Experiments Molecular Genetics.Cold Spring <br /> cally, able localized mutagenesis                   Harbor Laboratory, Cold Spring Harbor, NY. <br />                                                                      MISRA,M., S. A. BENSON,      1988a Isolation characteriza- <br /> obtain additional imp mutations, use imp con- <br />                                                                         tion OmpC porin mutants altered Pore properties. J. <br /> ferred phenotypes develop selections pseudo-                     Bacteriol. 1 7 0 528-533. <br /> revertants suppress moreof sensitiv-                MISRA,M., S. A. BENSON,    1988b Genetic identification the <br /> ities conferred mutations. Such pseudorever-                     pore domain OmpC porin Escherichia coli K-12. J. <br /> tants right provide novel insights                     Bacteriol. 1 7 0 361 1-3617. <br /> into outer membrane specifically excludes                    MISRA,R.,B. <br />                                                                                A.         SAMPSON,    J. L. OCCI S. A. BENSON, <br />                                                                         1987 Identification mutants altered outer membrane <br /> hydrophobic compounds cell,                      permeability, pp. 436-442, Antibiotic Inhibition Bacterial <br /> how imp mutations alter outer membrane perme-                       Cell Surface Assembly andFunction, edited P. ACTOR, L. <br /> ability.                                                                DANA-MOORE,      M. HIGGINS,  M. SALTON   G. D. SCHOCKMAN. <br />                                                                         Am. SOC.Microbiol., Washington, DC. <br />  This work supported U.S. Public Health Service grant         MORONA,  R.,P.A. MANNINGand P. REEVES,1983 Identification <br /> GM34810 National Institutes Health.                         characterization TolC protein, outer membrane <br />                                                                         protein Escherichia coli. J. Bacteriol. 153: 693-699. <br />                    L I T E R A T U R E CITED                         NAGELDE ZWAIG, R., S. E. LURIA,1967 Genetics physi- <br /> ANDRESSON,    0. S., andJ. E. DAVIES,1980 Organization              ology colicin-tolerant mutants Escherichia coli. J. Bacteriol. <br />    Restriction Enzyme Clevage Map ksgA-pdxA region            94: 1112-1123. <br />    Escherichia coli chromosome. Mol. Gen. Genet. 179: 21 1-      NIKAIDO,H., M. VAARA,1985 Molecular basis bacterial <br />    216.                                                                 outer membrane permeability. Microbiol. Rev. 49: 1-32. <br /> BENSON, S. A,, DECLOUX, 1985Isolation characterization       NIKAIW, H., M. VAARA,1987 Outer membrane, pp. 7-22, <br />    outer membrane permeability mutants Escherichia coli K-        Escherichia coli andSalmonellatyphimuriumCellularand <br />    12. J. Bacteriol. 161: 361-367.                                      Molecular Biology, edited F. NEIDHARDT,      J. INGRAHAM,     K. <br /> BENSON,S. A,, J. L. Occrand B. A. SAMPSON,      1988 Mutations          B. BROOKS,   B. MAGASANIK,    M. SCHAECHTER    H. E. UMBAR- <br />    alter pore function OmpF porin Escherichia        GER. American Society Microbiology, Washington, D.C. <br />    coli K12. J. Mol. Biol. 2 0 4 961-970.                            RAETZ,C.R. H., 1987 Structure         biosynthesisoflipid A in <br /> CAIRNS, J., J. OVERBAUGH     S. MILLER, 1988 The origins         Escherichia coli, pp. 498-503 Escherich <br /> </body></html>