<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1992 Genetics Society America <br />  <br />  <br />  <br />  The Drosophila couch potato Gene: An Essential Gene Required Normal <br />                               Adult Behavior <br />  <br />           Hugo J. Bellen,**&apos;Harald Vaessin?&apos; Ethan Bier?3 Alex Kolodkin: Diana D&apos;Evelyn,* <br />                                  Sandra Kooyer*and Yuh Nung Jant <br />        Howard Hughes Medical Institute, *Institutefor Molecular Genetics, Baylor College Medicine, Houston, Texas 77030, <br />              tDepartment Physiology, University Calijornia Sun Francisco, Sun Francisco, Calijornia 94143, and <br />              $Department Molecular Cell Biology, University Calijornia Berkeley, Berkeley, Calijornia 94720 <br />                                              Manuscript received January 9, 1992 <br />                                          Accepted publication February 26, 1992 <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  Through enhancer detection screens isolated 14 insertions essential gene is <br />                expressed embryonic sensory mother cells (SMC), cells    mature embryonic peripheral <br />                nervous (PNS), glial cells PNS central nervous (CNS). Embryos <br />                homozygote amorphic alleles die, obvious defects cuticle, PNS CNS. The <br />                gene named couch potato (cpo) insertional alleles alteradult behavior. <br />                Homozygous hypomorphic cpo flies recover slowly ether anaesthesia, aberrant flight <br />                behavior, fail light exhibit normal negative geotactic behavior. However, <br />                flies able groom walk, able fly prodded, indicating all <br />                processes required forbehavior severely affected. A molecular analysis showsthat 14 insertions <br />                confined nucleotides probably contain key regulatory sequences the <br />                gene. The orientation insertions position withinthisDNA             fragment play an <br />                important role couch potato <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>. In situ hybridization mount embryos suggest <br />                insertions affect levels transcription cpo cells whichit expressed. <br />  <br />  <br />      ELLS nervoussystem require theaction                           peripheral nervous (PNS). T h e goal of <br /> C      genes acquire morphological func- <br /> tional identity.A number genes involved nervous <br />                                                                               studies identify genes play impor- <br />                                                                               tant role differentiation and/orfunction the <br /> system development function       identified                    PNS furtherunravel genetic cascade which <br /> and partially characterized Drosophila (forreviews                         leads fully mature PNS. Here, the <br /> see CAMPOS-ORTEGA       KNUST 1990; JAN JAN                           isolation allelic series newly isolated gene <br /> 1990; GHYSEN     DAMBLY-CHAUDIERE         1989; RUBIN                     expressed SMC, embryonic and <br /> 1988; GANETZKY     WU 1986; HALL1985). Genetic                            adult mutant phenotypes, molecular defects that <br /> screens mutants affect nervous                            underlie mutations, cloning of <br /> have typically focussed isolating mutants                         gene. <br /> either defective development, exhibiting mor- <br /> phological aberrantstructures          (e.g., Notch, <br />                                                   scute,                                   MATERIALS AND METHODS <br /> achaete,slit,sim, etc.) mutants display quite <br />                                                                                   Fly stocks: Generation flies carry enhancer detec- <br /> specific behavioral defects adults (e.g., Shaker, ru-                      tors described BELLENet al. (1989), BIER et al. <br /> tabaga, per), exhibiting subtle physiological                        (1989). Some insertion strains described these <br /> biochemical alterations.                                                      papers renamed. The cpl insertion strain corresponds <br />    T h e adventofenhancerdetection           (O&apos;KANEand                       P[IArB] B52.1M3 strain described BELLEN        et al. <br /> GEHRINC1987; BELLENet al. 1989; BIERet al. 1989)                              (1989) WILSONet al. (1989). The 13 insertion corre- <br />                                                                               sponds 1(3)5D4 BIER et al. (1989). Most strains with <br /> provides new means study genes areexpressed                           90D/E deficiencies obtained KEVIN MOSES <br /> and/or required nervous (MLODZIK            et                  (MOSES,ELLISand RUBIN1989), Dfl3R)DG4 which <br /> al. 1990; DOE et al. 1991; review BELLEN,                             obtained DONGAILEY(GAILEY            HALL 1989) <br /> WILSONand GEHRING1990). We screened                                  Dfl3R)P14, obtained Bowling Green <br /> enhancer detector insertion strains express @-                       Drosophila stock center (Ohio) Umea Drosophila <br />                                                                               stock center (Sweden). <br /> galactosidase gene sensory mother cells (SMC)                             The lAcpll lAcp12 alleles recovered remo- <br />                                                                               biking P[IArB], isogenization, described BELLEN <br />    &apos; Corresponding author.                                                    et al. (1989). <br />    &apos; Present address: Ohio State Biotechnology Center Department          I n situ hybridization immunocytochemistry:In situ <br /> Molecular Genetics, Ohio State University, Columbus, Ohio 43210. <br />     Present address: Department Biology, University California San   hybridizations whole-mount embryos performed <br /> Diego, La Jolla, California 92093-0322.                                       according modified protocol TAUTZ         PFEIFLE <br /> Genetics 131: 365-375 (June, 1922) <br />  366                                                    H. J. Bellen et al. <br /> (1989). Probes werelabeled digoxigenin-dUTP               P-galactosidase activity (BIERet al. 1989; BELLEN <br /> hybridization detected monoclonal antibody <br /> against digoxigenin coupled alkaline phosphatase (Boeh- <br />                                                                    et al. 1989; WILSONet al. 1989). Fourteen insertion <br /> ringer Mannheim). In situ hybridization polytene chro-          strains stain   precursor cells the <br /> mosomes performed according modification           developing embryonic PNS selected. Very few <br /> protocol described LANGER-SAFER,LEVINEand           WARD        strains identified screens express the <br /> (1982). This protocol adapted digoxigenin-labeled          lacZ gene SMC, suggesting specific <br /> probes. <br />                                                                    genes active cells. This prompted us <br />     @-Galactosidase activityin embryos detected de- <br /> scribed BELLEN  et al. (1989). /3-Galactosidaseprotein      analyze insertion strains. Surprisingly, <br /> other proteins localized immunohistochemically using           14enhancerdetectorstrainscarryaninsertionat <br /> a polyclonal rabbit anti-&amp;galactosidase antibody (Cappel)       cytological subdivision 90D. These observations indi- <br /> monoclonal antibodies (see text) described BIER et al.       cated enhancer detectors controlled by <br /> (1989). <br />    Phototaxis assay: Phototactic tests performed         regulatory elements gene, the <br /> seven-trial countercurrent apparatus described BEN-          90D division possibly hottest spot forP element <br /> ZER (1967). After 1-min equilibration period, groups          insertions every300 insertions maps at90D <br /> 20-25 flies given 1 min white light          (BIERet al., 1989). Five insertion strains homozy- <br /> source dark room. At end assay,         gous viable, strainsare homozygous lethal. <br /> flies tube counted, number flies in <br /> each tube multiplied numberof times            In separate screen homozygous viable muta- <br /> moved light. The sum ofthese values divided         tions cause reduced flight abilities (A. KOLODKIN <br /> by total number flies. The theoretical range        n d COREY GOODMAN,      personal communication), P <br /> 0-7 (0 = flies light, 7 = flies 7          element-enhancerdetectorstrain          isolated that <br /> times light). Wild-type flies, e.g. Canton-S, typically     maps 90D. Embryos strain express P-galac- <br /> score higher 4. Each assay performed three <br /> times. (+) = index 2 4, (+/-) = 3 5 index &lt; 4,(-) = index          tosidase essentially cells 90D <br /> &lt; 3, nt = tested.                                              insertion strains. <br />    Flight response assay: Groups 20 flies given 30           &amp;Galactosidase expression enhancer detector <br /> sec fly away Petri dish 95 X 28 mm            strainswithinsertions    90D: T h e lac2 staining <br /> vial kept 1 hr anaes-        <span id='am-164' about='xsp:pattern' typeof='owl:Thing'>pattern</span> embryos 14 90D strains is, the <br /> thesia. The assays repeated times. (+) = <br /> 61-loo%, (+/-) = 31-60%, (-) = 0-30% fly away.                     exception number midgut cells, confined <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />    Geotaxis assay: About 20flieswere transferred             nervous system. lacZ expression firstseen in <br /> empty 95 X 28 mm vial subsequently capped           stage 8 embryos cellsat anterior end <br /> another vial. Flies shaken         embryo (Figure 1A) (for staging CAMPOS- <br /> of vialsin dark room number flies that <br /> movebeyond vialin 15 sec                ORTEGAand HARTENSTEIN        1985). Next, observe <br /> counted. The assays repeated times. (+)        staining SMC differentiating cells the <br /> = 61-loo%, (+/-) = 31-60%, (-) = 0-31% up, nt =               PNS stage 9-1 1 embryos (see Figure 1, B, C and <br /> not tested.                                                        D). The pattern consists gradual increase <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />    Ether recovery assay: About 20 flies anaesthetized         cell presumptive segment set four <br /> for 1 min etherizer containing 2 ml ether. The flies <br /> were placed 100-mm Petri dish recovery           two-cell clusters. During extended germ band (stage <br /> time (RT) recorded 50% flies walk away          11) pattern rapidly complex. In <br /> start grooming (RT,,), fliesrecovered (RTloo).        subsequent stages known PNS neurons as <br /> Fifty percent wild-type flies (Canton-S) typically re-     sibling non-neuronal supportcells express theP- <br /> cover 15 min, 100% recover 25 <br />                                                                    galactosidase (Figure 1 , E F) (see Figure 6, B <br /> min. (+) = RT5o &lt;15 min, RTloo&lt; 25 min, (+/-) = 15 min <br /> &lt; RT5(,C 20 min, 25 min &lt; RTloo &lt; 30 min, (-) = 20 rnin            C, BIER et al. 1989). lacZ expression the <br /> &lt; RT5(,,30 min C RTloo.                                            PNS thoracic abdominal segments resem- <br />    Recombinant DNA: To recover DNA sequences flanking              bles expression pattern observed A37 en- <br /> the enhancer detectors used plasmid rescue tech-            hancerdetectorstrain(see          GHYSENand O&apos;KANE <br /> niques described WILSONet al. (1989) BIER et al. <br /> (1 989). Genomic fragments recovered dif-          1989). This enhancer detector maps 80A. <br /> ferent strains. Some fragments used            T h e cells organs thePNS head region <br /> screen lambda DASH genomic library (kindly provided           embryo describedin than <br /> RON DAVIS).Maps constructed EcoR1,                   thoracic abdominal segments (CAM- <br /> Hind111 XbaI restriction enzymes.                              POS-ORTEGA     HARTENSTEIN      1985; JURGENS   et al. <br />                                                                    1986), making assignments particular clusters        of <br />                          RESULTS <br />                                                                    cells labeledin region difficult.T h e <br />   Screening enhancer detectors express 8-                 number cell clusters expressing lacZ gnathal <br /> galactosidase PNS: In attempt identify                segmentsandtheprocephalongraduallyincrease <br /> new genes expressed onset nervous               fromoneduringlategastrulation           (stage 7) t o 1 3 <br /> system development, embryos 4,600 strains carry-                early head involution (stage 14; Figure 2). <br /> ing single P element-enhancer detectors stained               Based o n t h e topological position clusters in <br />                                                              Genetics couch potato                                                            367 <br />  <br />  <br />  <br />  <br />         A <br />  <br />  <br />  <br />  <br />         D                                            E <br />    FtGURE I.-fl-Galactosidase expression pattern homozygous cporP&#226;&#8364;&#8482; embryos various stages development. All embryos shown <br /> with thrir anterior end left. don71 up. A, Stage 8 embryo. lac2 weakly expressed anterior end the <br /> rmhwo ( s e e arrow). R. Stage 10 embryo, IacZ expression confined cells segment. C . Stage 12 embryo. IacZ expression <br /> Imttern a1rr;tdy quite conlplex relatively large clusters cells segment lalx-led. D, Stage I3 embryo. note staining in <br /> tlre head region wrll regular pattern PNS aMominal thoracic segments. E. Stage 16 ernhwo. cells of <br /> tlw srrppnrt cells external sensory organs PNS express @-galactosidase.F, Detailed view stage 16 embryo. Focus the <br /> &lt;:NS cells PNS focus. The CNS hottom embryo. Note glial cells l m t e d along <br /> tlle don71 CNS (see rectangle). <br />  <br />  <br />  <br />  <br />     FIGUREP.-Det;~il fl-galactosidaseexpression pattern head region head involution (stage 14). A. lnteral view the <br /> ceph;dic region rpofP&#226;&#8364;&#8482; embryo stained i~l~munocytocllemicallv          anti-@-galactosidaseantibody j u s t head involution. R. <br /> &lt;:onpsitr schrnwtic drawing lateral view ceplralic region embryo. prior head involution. This drawing based on <br /> olwrvations tnatlr wve1~1            em1,ryos does correspnnd exactly picture shown left. The clusters labeled with <br />                          ; m t m l y numbered theCNS background hatched. For simplicity chosen nomenclature used <br /> ;~llti-fl-~:,ll;lctosid;lsc <br /> b y JURGENS rt ai. ( I 986). I , Central group nlaxillary sense organ. 2. Lower lip organ. 3, Lahial sense organ. 4, Ventral organ. 5. D o r s l <br /> l:ltcral papilla nlaxilliary sense organ. 6 . Hvpopharyngcal organ. 7. Antennal sense organ. 8. Inhral sense o r g n . 9. Dorsn-medial papilla <br /> of nraxi1l;wy sense orpan. 1 0 - 1 3, Clusters cells probably helong PNS head Ixen described previously to <br /> our knowledge. <br />  <br /> stage 14 embryos, believe 13                            believe cell clustersalso belong tothe PNS. <br /> clusters described previously (for compre-                           In addition neurons supportcells the <br /> hensive summary seeTable 7 JURGENS et al. 1986).                           PNS. l d expression observed somecells <br /> The clusters described pre-                          dorsal ventralarea ventral nerve <br /> viously quite small con-                         cord CNS, cells alongthe <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> ofthe bilateral <br /> sist 2-5 cells compared 10-40 cells                          peripheral nerve roots,and cells laterally the <br /> of clusters. Becausethe cells clusters                     anterior fascicle peripheral nerves. 8-Galacto- <br /> are epidermalandsubepidermal, mostof                                  sidaseexpressioncanclearlybeseen          thesecells <br /> staining cells confined cellsof PNS,                            completion germ band retraction (stage <br />  Jfi8                                                                                          1-1. J . I%t4lrnr l      01. <br />  <br />  <br />  <br />  <br />        FKX&#226;&#8364;&#8482;RK :~.-A-(;;ll;lc-tositf;lsc csprcssion ~ ; I I I ~ ~inY l;~rv;~l    I       h l l . ~ Y Tinwgin;ll disc- ; I d ; t c l u l t h c ; ~ t lo l - r l ~ c r. p I inserrioll \ t ~ . t n . A. &#226;&#8364;&#732;lllir(l inst:tr <br /> l:~t.valtwain. UCNC                cspwssion I h r m;litl c - l ~ ~ s ~of    c rc-dls. <br />                                                                                         s         .l&#226;&#8364;&#732;t~c  t w o w111ra1rlu\tcrs c-orrcsl~ot~~l             n I)osition ~ I M Isin. IO 111ccell I&gt;oclic\of <br /> t l l c . I I ~ ~ O I I01-  \ 1111- m d 1 r o o m Iwtlics. . I h t \ w I;II~T:II nwonsll;qwd c111strrsc o r r q m n d t o r l w ;IW;I w h e v ~rhc I)tlotorc.c-c.l)t&#226;&#8364;&#732;,r. c-c*llsof 111c~ v c <br /> in1agin:aIdisc- s v n p ~ .w t h neurons 111cl)r;iin. 51;111vc-c*Ilssc;trtrrctl t l ~ r o u g l ~ o ~r tr~tc . I I C ~ V O I I Ss v s ~ c m;llso cuprcss /nr%. 1%. I~.~.I*-;IIII(.IIII;I~ <br /> imtgitxtl disc-. Uocc IIII. ~IainiugI ) c ~ h l ~ trt~c      l   rllorl&#226;&#8364;&#8482;llogcllc.ric. f ~ ~ r r ~ (1.    n v Sew <br />                                                                                                                .                              :Ill st;lining n ~ ~ c - h111. <br />                                                                                                                       ion ; t d u I t I~c;~tl.                                      r : I I I I I ~ ~ O O I I Ih l w I . <br />                                                                I ~ ; ~ g ~ m~ pI , l ~ cwl l l s r c m n 1 n o u t h ptrrs. <br /> l ; n l n ; t : 11. l&gt;r:ainrind: s. s u l ~ o c s c ~ ~ ~ganglion: <br />  <br />  <br />  <br /> 1 3 ) . Ikwd morpl~ologyand position,                                                                    I n t h c ;ldrtlt flv w~only ex;lmincd @-g;ll;lctosid;~sc <br /> cells prolxlbly corrcspontl t o glial cc*lls.so111cof                                                       expression n sc-ctionso f h c w l . A s sllown n Figure <br /> I1;1vc hccn descri1)ed previously ( ~ACO B Sand (;MI&gt;-                                                            X ; . staining observed n lamina, l n n rind, <br /> MAS 1989:.],\COnS PI / . 1989: ~ l , A \ 4 l 3 &#226;&#8364;&#8482; l ~ ; l I l (d; 0 O D S I A N                                  sul,ocsoph;lgc;~lganglion ant! clusters cc*lls n t h c . <br />  1 9 9 I ; FREI)IEC ;mtl \ ~ A H O W A I , D 1989).                                                               11lo11t hp;lrts. The lac% e s p r c s d m ; l n y nuclei <br />     ~j-(;;ll;lctosid;lsc expression \\.;IS esanlined n                                                     retina, mctlull;l. lol,rlla ;1nd l o l ~ l ; plate,    ~ r ocelli <br /> third instar- I;lrv;rcbrains ant! imaginal discs. A s sho~vn                                                      ocellar ganglion. Hcncc, cells of&#226;&#8364;&#732; t hc visual <br /> i n Iigurc 3:I. lac% expressed n I;wval brain n                                                        systcrn cxl)rcss marker gene. \I;lny cells n the <br /> f ~r nr; t n clusters cells. One set h s ;l n1oonsh;lpcd                                                 ; l r l l ~ ~ l l l 1 ascgrllc&#226;&#8364;&#8482;nts <br />                                                                                                                                           l          ;1lso strongly C&#226;&#8364;&#8482;spr&#226;&#8364;&#8482;ss nlilrkcr <br /> fi)rrn n hcnlisphcrc of&#226;&#8364;&#732; 1)rain. l&#226;&#8364;&#8482;hesc cells                                                          gcne. Furthcrmore. lacZ cxpr-cssccd n cells o f <br /> prolxtl)ly correspond cclls w t h pho-                                                         muslwoom I,oclics othcr parts bin. <br /> torcccptor cc4ls f - o m eye im;lginal disc synapse n                                                     Finally, t w o cells stain ;it t h c h s e m111v of <br /> the h n . T l l c scconcl bilatcrallv symmctr-ic cluster                                                   n~;lcrocl~;~et;w              n hcad. The nl;~rkcrgcv1c is <br /> c . e ~ ~corrcsponcIs <br />          s                size n c t I;,c;Ition wit11 t11c cell                                              csprtwc&#226;&#8364;&#732;d m ; l n y cells ;ldult PSS ; I d <br /> 1)otlic.s nlushroo111 bodies. This S I ~ I I C I I I of <br />                                                              ~ C the <br />                                                                                                                   C S S (PY~NG-IAV                 I-IAN,  personal conlmrrnic;ltiorl). <br /> brain thought t o play I key role n olfhctory learning                                                             Enhancerdetectorinsertions                   90D affect an <br /> and nlc*mory. I n ;ldtlition thcsc cell clusters                                                          essential gene: &#226;&#8364;&#732;1&#226;&#8364;&#8482;0 map Icth;ll mutations (a1)Ix-c- <br /> 111;lny scattcretl cells n brain csprcss 1acZ. <br />                                                                                                                   viatcd Il-lO, 1 = Icth;ll)0 1 1 t h c chromosomcs t l u t cxrry <br /> !:inally, clusters o f cells arc associated                                                           tllc 9 0 1 ) inscv-tions, wc pcrfort1lcd complcmcnt;lriorl <br /> w t I1 t h c h n ;In(! loc;ltcd bet wen 1,0111 ccrclml <br />                                                                                                                   IC-sts w t h ;I series dcficicmcic.s t h t lack cytological <br /> Iwrnisphcrcs c ~ p r c s slarZ ( n o t shown). One clus- <br />                                                                                                                   1);tntls w t h n ;1rountl t h c 9OD/E subdivision. Somc <br /> ter snlall, loc;~tctlccntr;Illy ;Ind b l l sh;qml. The <br />                                                                                                                   o f rcstrlts tlwsc conlplcnlent;ltion tc&#226;&#8364;&#8482;sts arc <br /> other t w o arc largc~,      form ;t pair, ;1nd arc pear slwpcd. <br />                                                                                                                   slwwn n I:iglrrc 4 ;lnd l&#226;&#8364;&#8482;aldc 1 . 1,ethaI rnrrt;ltion(s) on <br /> pl&#226;&#8364;&#8482;l~esc   clusters prolx1l)ly I)clong t o ringgl;\ntl <br />                                                                                                                   ;ill tc1stc.d cIII.oIllosoIllcs t h ; ~c;lrry 901)/1~. inscrtiorls <br /> ;tnd/or stomatogastric nervous s;&#226;&#8364;&#8482;stcrn (&lt;:A\IPOS- <br />                                                                                                                   arc uncovcrcd h y tllc tlcficicmcics. I?ji&#226;&#8364;&#8482;31&lt;)1&#226;&#8364;&#8482;-14. <br /> O R T E G A ;IIlt! I1ARTESS&#226;&#8364;&#732;I&#226;&#8364;&#732;ElS 198.7). <br />     8-(;;llactosirI;tsc csprcssion n i11laginal discs is, w t h <br />                                                                                                                   I)f(3R)g111SIO, I?J&#226;&#8364;&#8482;31&lt;)/(3)11S6;111tl I?j?31&lt;)1)G4( S C C \ f A - <br />                                                                                                                   TERIAI. ANI) \IF.TllOl~S),l\&#226;&#8364;&#8482;hct-&lt;&#226;&#8364;&#8482;;lS I?ji&#226;&#8364;&#8482;3/&lt;)gl11.Y-1 C o I 1 1 l ) k - <br /> t h e exception o f eye disc, confirlcd t o singlc cells <br />                                                                                                                   merits $101) insrrtion ~ ~ l l r o ~ l l o s oJiorcovcr,~~~cs. <br /> o r snl;lll medium size clusters ofcclls.\Ye compared <br /> t hc lac% expression p t t c r n n thc wing imaginal disc <br />                                                                                                                   thc I c t h l insertion chron1oso11ws (11-9 ) fail t o com- <br /> of cpI insertion strain w t h p t t c ~ ntlccril,cd                                                   plcmc1lt c;lch othcr. I Ic11cc*,the inscrtions ilffcct ;I <br /> b y GIWSES O&#226;&#8364;&#8482;KANE( 1 OXO) f&#226;&#8364;&#8482;or X J enhancer                                                             co11111lon             cssenti;ll gtvw t h t 1n;1ps t o t h c 901)2-90!~6 <br /> tlctcctor cspress&#226;&#8364;?d n I&#226;&#8364;&#8482;SS. Rot11 patterns                                                         cytological Ixlnds (see 1:igurc I ) . <br /> arc similar n 11otum triple row bristles                                                                  A 1 1 deficiencies klil complcmcnt t h c <br /> precursor cells n wing region ;\t&#226;&#8364;&#8482;c n o t I;lbclctl n                                                     cIlromoson1c~scontaining 901) insertion ;11sof i l t o <br /> the c p l wing disc. I n t h c c*ye imaginal disc nlany                                                           complcnlcnt sr&#226;&#8364;&#732; (GAIIXY IIAI.1. I W?): s c e 1:igul-e <br /> photoreceptor cells express @-g;ll;lctositl;1scI)chind                                                        4). I-Iolnozygo1rs .w&#226;&#8364;&#8482; flies h;~ve;1 gray str-ipc the <br /> nlorphogcnctic f \ l ~ ~ o(SCC   \ \ . 1;igurc 313). Fi11illly. lac%                                              thorax resr~lt lack t h c dorso <br /> expression restricted PSS CNS                                                               longitudinal indirect flight musclc. (:onlplcnlcnt;ltion <br /> i n instar l r ~ since~ e salivary glands csl)rcss                                                    tests w t h sr&#226;&#8364;&#8482; ; I n d 901) irlsc*rtioncIlro11wsonl(*s <br /> ~~-g;ll;lctosi&lt;l;lsc..                                                                                            indicate t h ; l t csscnti;ll gene ;\I 901) complements <br />                                                                                            Genetics couch potato                                              369 <br />                                         90D/E <br />                                                                                                         allow order different alleles pro- <br />        A      B                  C                                       A         E <br />                                          D          E          F                              C <br />                                                                                                         gressive allelic series partially complement <br />                                                                              1                          homozygous lethal insertions: 11-8 &gt; 19 &gt; c p l = <br />       90 <br />                                         on <br />                                          CDO   lr                        &quot;I <br />                                                                                                         cp2. T h e complementation data indicate of <br />                                                3-620                     J-bl I                         insertional alleles null alleles example <br />                                                93C2-01.91BI-2                          P-f4 <br />                                                                                                         c p l l l l cp1/12 flies reduced viability c p l l <br />                                                9OC7-I). 9181                       g18XfO               deficiency flies sub-lethal Illdeficiency 121 <br />                                               90DI. 9OP10-12                 1(3)~y,5                   deficiency flies lethal (Tabel 1). <br />                                                                                                            T o define embryonic mutant null phenotype it <br />                                              9302-4.9OF3-6                064 <br />                                                                                                         necessary recoveramorph alleles. Hence, we <br />                   9001-2.90~1-2                         srl6                                            performed A2-3-transposase excision mutagenesis <br />      sr-(                    I <br />                   9 x 7 - 8 . 90EI-2   D T ] s r ~ 4 6 ~           I;&apos;                                  (BELLEN      et al. 1989). About 500 ry- flies lost part <br />                                                                                                         ofthe 90Dc p l enhancer detectorwere screened, <br />                                                                                                         newly isolated mutants failed complement <br />      sr+    &lt;                                    90F8-II.91B1                     gJBYI <br />                                                                                                         11-2 Dj(&apos;3R)Pl4. These new alleles were <br />                                                                                                         named 1Acpl I l A c p 1 2 . As shown Table 1, the <br />    FIGURE4,&quot;Extent cytological visible deficiencies                                      l A c p l l <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>mutation</span> fails complement lethality of <br /> I)OI)/E region. The cytological extent deficiencies                                      alleles tested amorph <br /> shown. Some provided KEVIN MOSE~(personal communi-                                              allele. The otherallele l A c p 1 2 behaves relatively <br /> cation) DON GAIIXY(GAILEYand HAIL 1989), were <br /> determined n work (Dj73R)wf6,and Dj73R)sr&quot;&quot;). Since is <br />                                                                                                         weak hypomorph various complementation <br /> 110 known breakpoint SY cpo demarcation                                         tests. Based complementation data presented <br /> the genes unknown.                                                                               Table 1 reordered alleles decreasing <br />                                                                                                         strength following allelic series: Dj(&apos;3R)~r&apos;~~&apos;      = <br /> the sr&apos; allele gray stripe observed transhet- <br />                                                                                                         Dj(&apos;3R)d6= l A c p 1 1 &gt; 11-8 &gt; 19 &gt; l A c p 1 2 &gt; c p l = cp2. <br /> erozygous flies. However, sri6 sr1461,diepoxybu- <br />                                                                                                            cpo required embryonic viability, cpo <br /> tane (DEB)- ethyl methanesulfonate (EMS)-in- <br /> duced alleles (DE LA POMPA,GARCIAandFERRUS                                                              mutant adults couch potatos: We anticipated that <br /> 1989), f l complement lethality chro-                                                 flies homozygous hypomorphic alleles cpo may <br /> mosomes thatcarry90Dinsertions            (see Table 1).                                                adult behavioral defects amutant was <br /> Cytological examination stripe alleles reveals                                                 isolated screen reduced flight ability. We also <br /> that small cytologically visible deficien-                                                 noticed homozygous c p l adultsrecovermore <br /> cies: srI6 = Dj(&apos;3R)9ODI; 9081-2 sr&apos;461 =                                                           slowly ether anaesthesia heterozygous <br /> Dj(&apos;3R)90C7-8;9081-2 (see Figure 4). Based                                                       balanced flies, indicating behavioral de- <br /> data SORSA(1988) estimate deficien-                                                    fects associated insertion chromo- <br /> cies larger 70 kb probably 300                                                   somes. Hence, decided performa series of <br /> kb. Hence, essential gene described                                                    simple behavioral experiments flies carrying in- <br /> stripe genemap 90D1; E1-2 cytological                                                        dependently isolated alleles different ge- <br /> interval.                                                                                               netic screens. We selected simple assays meas- <br />     Insertions 90D hypomorphic alleles: Five                                                     ure phototaxis (BENZER 1967), geotaxis, flight re- <br /> enhancer detector strainswith insertion 90D                                                   sponse recovery time ether anaesthesia (see <br /> homozygous viable. Four ofthese strains isolated                                                   MATERIAL AND METHODS). <br /> o n basis expression pattern,                                                      Table 2 summarizes result behavioral <br /> strain isolated screen mutations thatcause                                                 tests. For assay results easily be <br /> reduced flight abilities. Complementation tests                                                    subdivided groups: high (+), interme- <br /> these strains, various deficiencies shown                                                  diate (+I-),   low scoring (-) class (see MATERIALS <br /> in Figure 4, homozygous lethal alleles                                                  AND METHODS). The data homozygous 4 1 , <br /> indicate viable alleles subdivided                                                 c p 2 v3 flies exhibit behavioral defects assays. <br /> two groups (Table 1). Two insertions, named c p l                                                   In addition,the 11 12 insertional allelesfail to <br /> c p 2 ( c prefers couch potato phenotype, below),                                              complement c p l c p 2 alleles four <br /> fail complement poorly complement defi-                                                      assays. However, homozygous v3 flies are <br /> ciencies partially complement 11 12                                                    behaviorally defective assays, v3 inser- <br /> insertions cause lethality. Therefore, c p l c p 2 ,                                           tion fully complements 11, 12 c p 2 insertions. <br /> are hypomorphic viable alleles essential comple-                                                 This suggests v3 insertion complements mu- <br /> mentation group defined 11-9 insertional al-                                                     tations gene, v3 affects another <br /> leles. The homozygous viable 90D inser-                                                     nearbygene absence reduction causes a <br /> tions, V I , v2 v3, reduced viabilityin                                                     similar phenotype. T o determine mutations <br /> various complementation tests. The c p l c p 2 alleles                                              associated insertions contribute <br />  370                                                                    H. J. Bellen et al. <br />                                                                            TABLE 1 <br />                          Complementation datafor insertional alleles deficiencies cytological subdivision90D <br />  <br />                                    19           11          12            C P1          CP2       V l       v2       v3           IAcp I I       1Acp I2 <br />         19                         - <br />         11                         -            - <br />         12                         -            -           - <br />         CPl                        +          +/-          +/-             + <br />         CP2                        +          +/-          +/-             +             + <br />         V I                        +            +           +              nt            +        + <br />         v2                         +            +           +              +             +        +         + <br />        v3                         nt            +           +              +             +        +         nt       + <br />        IAcpll                      -            -           -              -             -        +         +        +              - <br />        lAcpl2                    +/-          +/-          +/-             +             +        +         +        +              -              - <br />        DfljR)P14                   -            -nt         - <br />                                                             nt           -/+ <br />                                                                           nt            nt                                          -              - <br />        Dfl3R)glEXI 0              nt            -           -              -             -        +         +        +              -              - <br />        Df3R)glBX6                 nt            -           -            -/+             -        +         +        +              -              - <br />        Dfl3R)DG4                   -           -            nt           -/+            -/+       +         +        nt             -              - <br />        DJ3R)srf6                   -           -            -nt          -I+ <br />                                                                           nt            nt                           nt             -              - <br />        Df3R)~r&apos;~~&apos;                -            -            -            -/+ <br />                                                                            nt           nt                 nt        nt             -              - <br />        Dfl3R)glEXI                +            +            +     nt       +       nt    +        nt                                +              + <br />        sr &apos;                       +            +            +              +             +        +         +        +    ~~~~~ <br />                                                                                                                                     +        ~ <br />                                                                                                                                                    + <br />    - = transheterozygous progeny recovered; + = 50% expected            transheterozygous progeny recovered; +/- = 50% but <br /> more 10% expected transheterozygous progeny recovered;        -/+    = 10% expected transheterozygous progeny recovered; nt = <br /> not tested. <br />  <br /> to thebehavioral defects, c p l , v3 11 insertions                          MAb6D5 (data shown). T o screen defects in <br /> were excised using A2-3-transposase mutagenesis                                 CNS andthe PNS cpo mutantstrains, we <br /> scheme. At 10 independent homozygous viable                                   immunocytochemically stained nervous of <br /> strains insertion strains                              embryos homozygous different cpo alleles <br /> recovered andfurther            analyzed. These viable fly                          anti-HRP (JAN JAN 1982), anti-cut (BLOCH- <br /> strains tested behavioral assays. For                               LINGER et al. 1990) monoclonal antibody 22C 10 <br /> each original insertion independent rev-                               (CANALand FERRUS1986). No obvious defects were <br /> ertant strains recovered scored positive                                 observed nervous systems embryos strains <br /> (+) assays. These observations indicatethat                            carryinginsertional alleles C ~ O &apos; ~ PThus,  ~ ~ . cpo is <br /> three insertions responsible forthe behavioral                                  essential observe obvious morphological <br /> defects lethality case I1 insertion.                           defects cuticle, muscles nervous system. <br /> In summary,thesedataindicatethat                     re-                       Enhancer detector insertions cpo map 250- <br /> covered mutants essential gene                                  bpfragment: Genomic DNA flanking inde- <br /> some insertional alleles affect function gene                              pendently isolated insertions near cpo iso- <br /> that required normal behavior.                                               lated using plasmid rescue technique (see WILSON <br />     Viable mutant flies (e.g., c p l l c p l , c p 2 / c p 2 , c p l l l l ,        et al. 1989). These rescued fragments, map at <br /> c p 2 / 1 1 ) abnormal adult behavior                             cytological subdivision 90D1-6, used screen a <br /> their development significantly delayed                               lambda DASH library, 50kb contiguous <br /> these flies eclose 1 usually 2 days                          sequences genomic DNA 90D region were <br /> balanced flies. In addition, flight assays                            isolated (see Figure 5 ) . <br /> give impression cpo mutants unable fly,                                To identify functionally important domains of <br /> many able fly limited distance prodded.                               cpo, mapped positions 13 enhancer <br /> Finally, c p l c p 2 flies groom time                         detectorinsertions Southern analyses. Ali the <br /> being tested behavioral assays.Based                                  insertions mapped far located 250-bp <br /> abnormal hypoactive behavior mutant flies                                genomic fragment. One insertion causes homo- <br /> we named gene couch potato (cpo).                                          zygous lethality (19) maps left half ge- <br />     Most cpo alleles cause lethality late embryonic                              nomic fragment (Figure 5). T h e 19 insertion behaves <br /> development.Forexample,              homozygous cpo&quot;                            relatively weak hypomorph complementation <br /> cpoiACPI1first instar larvae fail hatch aresluggish                          tests (cpo&quot; Table 1). All insertions map 150- <br /> when extracted manually. However, did ob-                                    250 bp right      19 insertion (11-6,8, V I , v2, <br /> serve obvious defects reduction muscle size                               v3, c p l c p 2 ) . The insertions homo- <br /> in cpoiAcP1I,cpo&quot; Df(3R)DG4 embryos using                                    zygous viable (vl-v3) map different orientation <br /> immunocytochemistry muscle-specificantibody                                  insertions. T h e v 3 insertion causes a <br />                                                       Genetics couch potato                                                                                                      37 1 <br />                                                              TABLE 2 <br />                                                         Behavioral assays <br />  <br />                   CP2            CPl   11      12                                                     U3                    cs             TM3                      ry <br />  <br /> Phototactic response <br />   CP1 <br />                                        -       -             -          -                             nt                    +              +/-                      + <br />   cP2 <br />                                        -       -             -          -                             +                     +              +/-                     nt <br />   V3                                   +        +            nt         +                             -                     +              +/-                     nt <br />   (3                                   +        +            +          +                             + nt                  +              nt <br /> Flight response <br />   C P1 <br />                                        -        -            -         nt                             nt                    +              +/-                      + <br />   cP2 <br />                                        -        -            nt        -                              +                     +              +/-                      + <br />   v3        nt                         nt                               +                             -                     +              +/-                      + <br />   cs                                   +        +            +          +                             +                     +               +                      nt <br /> Geotactic response <br />   CP1 <br />                                        -        -       nt   -          nt                                                  +              +/-                     +/- <br />   cP2 <br />                                        -        -            nt         -                             +                     +              +/-                      + <br />   v3                                   +       +/-           nt         +                             -                     +              +/-                     +/- <br />   cs                                   +        +            +          +                             +                     +              nt                      nt <br /> Ether recovery <br />   CP <br />   CP2 <br />        &#226;&#8364;&#8482;                               - <br />                                        - <br />                                                 - <br />                                                 - <br />                                                              - <br />                                                              - <br />                                                                         - <br />                                                                         - <br />                                                                                                       nt <br />                                                                                                       + <br />                                                                                                                          +/- <br />                                                                                                                             + <br />                                                                                                                                            +/- <br />                                                                                                                                            +/- <br />                                                                                                                                                                    -I+ <br />                                                                                                                                                                     + <br />   V3                                   +        +            nt         +                             -                     +              +/-                     -I+ <br />   CS                                   +        +            +          +                             +                     +               -                      nt <br />   See MATERIAL    AND METHODS   symbols. <br />  <br /> cpo phenotype complements cpo alleles,                                                                     SI&#226;&#8364;&#8482; <br />  <br />  <br /> including deficiencies. Six insertions map <br /> approximately 40-60 bp left cluster of <br /> the viables. They homozygous lethal. Fi-                                                                   \      I <br /> nally, cpo&#226;&#8364;&#732;p&#226;&#8364;&#732; cpocp2 insertions map close               EH    xn H xn <br />                                                                               I / <br />                                                                                     I <br />                                                                                     I <br />                                                                                         I <br />                                                                                         I <br />                                                                                             I <br />                                                                                             I <br />                                                                                                 H <br />                                                                                                 I <br />                                                                                                 , <br />                                                                                                       nx <br />                                                                                                       I , <br />                                                                                                       I ! <br />                                                                                                                  E <br />                                                                                                                  I <br />                                                                                                                  I <br />                                                                                                                      f /P        EX <br />                                                                                                                                    I <br />                                                                                                                                  I I <br />                                                                                                                                        n <br />                                                                                                                                        , <br />                                                                                                                                        I <br />                                                                                                                                              HXxn <br />                                                                                                                                                 I / / <br />                                                                                                                                                          n <br />                                                                                                                                                          I <br />                                                                                                                                                              nn <br />                                                                                                                                                              I I <br />                                                                                                                                                                     EEH <br />                                                                                                                                                                     I l l <br />                                                                                                                                                                             n <br />                                                                                                                                                                             I <br />                                                                                                                                                                                        E E <br />                                                                                                                                                                                        I I <br /> vl-3 11-6 insertions. It noted that <br /> these mapping data tentative based                                       -                            -                                               - <br />                                                                                                                                                                          - <br />                                                                                                                                                                          -      61.1 <br />                                                                                                                                                                                 61.2 <br />                                                                                                                                                     Y <br />  <br /> on genomic Southern, andplasmid rescue data. Since                           lnurtlon rlta IQ                                                      1kb <br />  <br />                                                                              Insertion an. 11. 12.13, IS, 6,II <br /> none insertions null alleles cpo,               v     insmlon sit-       VI.   VZ,   v3 <br /> they probably located regulatory region                                  site cpt, CPZ <br />                                                                              ~n.srt~on <br />  <br /> the gene.                                                              FIGURE5.-Molecular map cpo mapping position of <br />    Genomic Southern analysis revealed                     P element insertions hybridization sites three <br /> cpoiAcf&#226;&#8364;?&#226;&#8364;&#8482;is deletion possibly combined              longest cDNAs. ~ p o &#226;&#8364;? ~ &#226;&#8364;? ,the severe allele, deficiency. The <br /> a rearrangement P element sequences (see Figure                  excision event lead loss BamHI site located <br />                                                                     200 bp left c p l insertion. The precise extent of <br /> 5),            an <br />                            internal deletion c p l           deficiency unknown restricted boxed region the <br /> enhancer detector (data shown). Quantitative ge-                figure. The c ~ o &#226;&#8364;? are <br />                                                                                          - ~ hypomorphic lethal alleles, map <br /> nomic Southern analysis showed Dx3R)sri6and                    50 bp. The cpo&#226;&#8364;? allele weaker hypomorph cpo&#226;&#8364;?. <br /> Dx3R)~r&#226;&#8364;&#8482;~~&#226;&#8364;&#8482;  lack DNA corresponding entire               x alleles insertion maps different position. Note that <br />                                                                     insertions homozygous viable ( v l , v2 v 3 ) inserted <br /> genomic walk shown Figure 5. This con- <br />                                                                     opposite orientation insertions. These insertions <br /> firmed situ hybridization polytene chromo-                 behave differently complementation tests c p l cp2 <br /> somes carrying deficiencies using various frag-               insertions inserted opposite orientation. The data <br /> mentsfromthegenomic           walk probes(datanot                indicate position insertion, possibly orienta- <br /> shown).                                                             tion size insertion mayplay role themutant <br />                                                                     phenotype. E = EcoRI, H = Hind111 X = XbaI. <br />    The 90D insertions intron cpo: A 5- <br /> kb NotI-Hind111 plasmid rescued genomic fragment                    screen embryonic lambda gt 10 cDNA library <br /> at 3&#226;&#8364;&#732; end cpo&#226;&#8364;&#732;p&#226;&#8364;&#8482; insertion previously                (provided L. KAUVAR).Two cDNAs 1.6 <br /> shown contain repetitive DNA hybridize                 and1.8kb     isolated, thesewere used to <br /> to major 2.7-2.8-kb transcript (and   possibly           screen size selected 9-12-hr-old embryonic cDNA <br /> minor transcripts 1.6- 4.4-kb range; cpo@&#226;&#8364;&#8482;               library (ZINN, MCALLISTER  GOODMAN 1988).  The <br /> corresponds strainB52.1 M3 Figure 4 WILSON                 longest cDNAs, 17, 61.1 61.2,   62 <br /> et al. 1989; fragment used genomic Southern                 positives selected forfurther     work. These <br /> and Northern blots figure 5-kb NotI-                 cDNAs 3.0, 3.6and 2.5 kblong, respectively. <br /> Hind111 fragment). This genomic fragment used                   Southern analysis shows cDNAs is <br />  372                                                 H.1. Rellen et al. <br />       v                                                                         - <br />                /           . .. <br />  <br />  <br />  <br />  <br />                                                               .. .. <br />                                                                             I <br />  <br />  <br />  <br />                                                           *           0- <br />  <br />  <br />       A                                     B                                   C <br />        4 <br />                                   &quot; <br />  <br />  <br />  <br />  <br />                                        t <br />                                        &apos;I <br />                                                                                 4 <br />                                                                                 4 <br />  <br />       D <br />  <br />  <br />            . <br />                             -               E                                   F <br />  <br />  <br />  <br />  <br />       G <br />  <br />  <br />  <br /> dcrivetl fronl ;I message w t h different splicing pat-     positioned peripheral nerves express <br /> tern. Since 61.2cDNAsstronglvhybridizes                  cpo. The glial cells form sheath periph- <br /> genomic fr;Igments sides insertions           eral nerves, sheath easilv seen the <br /> conclude enhancer detectors likely            vicinitv CNS. The cytoplasm glial cells <br /> inserted n intron transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cpo.        ;IStheir projections clearly labeled, masking <br /> Genomic Southern analyses, Northern analvsis                possible staining axons. In embrvos have <br /> reverse Northern analysisindicate          thethree        able identify strong staining cells located <br /> &lt;:DNAs contain highly repetitive D N A con-            dorsal pair cells ventral <br /> fined ;I 2.2-kb Hind111 fragment shared               CNS. These cells correspond in <br /> three cDNAs (see Figure.5). Reverse Northern exper-             location cells express @-galactosidase <br /> imentscovering entire walk using 9- 12-hr-old               enzyme insertion strains mostlikely <br /> poIV(A+)R N A revealed arc             glial cells. <br /> transcripts detected w t h n introns shown Fig-         A comparisonof       embryonic@-galactosidase <br /> ure 5 (di1t;t sho\\r\.n).                                   staining cpo enhancer detector strains w t h the <br />     As shown n Figure 6 , situ hybridization experi-       expression pattern cpo mRNAs shows the <br /> ments mount embryos w t h digox-           enhancer detectors quite hithfullv reproduce the <br /> igcnin labeled cDNAsshow transcripts               expression pattern transcripts. The onlvob- <br /> cxpressed n SJ4C PNS ;IS n           vious difference patterns shown Figure <br /> differentiated cells PNS (Figure 6 , B C).           1 Figure 6 canbe         attributed nuclear <br /> Some cell clusters, example cells lateral            localization @-galactosidasestaining (BEILEN PI <br /> cluster, <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> intensely (Figure 6E).          al. 1989: M&apos;IISON ~t al. 1989) predominantly <br />  I hecytoplasmof neurons isweakly labeled,                 cvtoplasmic staining cpo transcripts. During the <br /> and m n y axon bundles nerves labeled, espc-           earlydevelopmentalstagesboth           patterns ;Irevery <br /> cially ventral embryo              similar. For example, stage 7-8 embryos express the <br /> pcriphcr;II nerves connect w t h CUS (Figure 61:).        l d transcript fewcells most <br /> I t clear transcripts n neurons contrib-       anterior end embryo. In stage 9-10 embryos a <br /> ute labeling axons glial cellsthat        cclls in presumptive segment express the <br />                                                   Genetics couch potato                                            373 <br />  <br /> transcriptatabout         4:20-4:40 hr development,         detailed analysis lacZ expression pattern the <br /> about 0.5 hr earlier @-galactosidaseexpres-           enhancerdetectorstrains        showed lacZ ex- <br /> sion cells, In addition, types           pressed glial cells CNS PNS and <br /> staining compared, subtle differences beseen           low levelsin cellsof midgut. I n situ <br /> in intensity cells cell clusters        hybridizations mountembryos             cpo <br /> of PNS labeled. T h e ZacZ expression staining         cDNAs indicate cpo expression confined PNS <br /> intensity similar cellsof        lateembryos        glia, possibly CNS. However, have <br /> whereas transcript abundantin             unable establish expression CNS is <br /> specific cells, especially chordotonal organs. The         significantly higher background. We found <br /> major difference betweenthe expression patterns            convincing evidence cpo expressed the <br /> is faint staining observed CNS          midgut. Since defects observed muscles, <br /> in situ hybridizations. However, label         PNS CNS amorphic mutant embryos        possible <br /> be nonspecific homozygous DJ3R)srI6 embryos              cpo required embryonic nervous system <br /> also stain CNS.                                         anessential behavioral, biochemical, physiolog- <br />    Expression cpo altered somemutants:                ical functionwithoutaffecting       morphology of <br /> Mutant embryos thatlack ormost cpo gene            CNS PNS. If cpo required PNS for <br /> should alterations cpo expression. We com-             viability, conclude the <br /> pared expression cpo mutant (cpox/cpox) wild         chordotonalorgansare        essential viable <br /> type (cpox/+, +/+) embryos using mount n            mutations scute-achaete complex cause loss all <br /> situ hybridizations. Expression cpo clearly altered      PNS organs chordotonals. Future molec- <br /> in embryos homozygous deficiencies cpo. Em-             ular genetic experiments address ques- <br /> bryos homozygous DJ3R)srI6 embryonic lethal            tions. <br /> and lack sequences cpo               All insertional alleles 90D mapped far are <br /> cDNAs hybridize. About quarter embryos              confined 250-bp interval.These sequences appear <br /> of D J ~ R ) s ~ &apos; ~ / Tstrain <br />                           M ~ labeling       intron cpo transcript, 5&apos; the <br /> the CNS. Homozygous lethal cpo&quot; embryos larvae              start cDNA (data shown). Several ob- <br /> show obvious defects PNS CNS              servations indicate neighboring se- <br /> about &apos;/4 embryos balanced strains exhibit       quences play important role regulation cpo <br /> a severe reductionof amountof label cells          transcription. First, insertional mutations 90D <br /> of PNS older embryos (stage 14- 17) label           tested cause lethality behavioral defects (the <br /> is seen thechordotonalneurons          (compare        v l v2 insertional strainshave testedyet). <br /> Figure 6G Figure 6E). Similar observations           Second, overall decrease level cpo transcript <br /> made embryos        cp0&quot;~~&quot;/TM3flies, indicating    PNS observed insertion strains, and <br /> that 90D insertional mutationsand deficiency          homozygous 11 embryos transcript confined <br /> insertion site affect level cpo mRNAs           chordotonal neurons stage 14- 17 embryos <br /> the spatial temporal expression cpo. <br />                                                                only. Third, small deletion theinsertion site leads <br />                                                                expression pattern observed in <br />                      DISCUSSION                                homozygous 11 embryos. We conclude that <br />     In work genetic phenotypic        sequences essential proper expression of <br /> characterization alongwith preliminary molecular           cpo PNS theglial cellsand thatthey contain <br /> analysis newly identified gene namedcouch potato          key regulatory sequences. <br /> ( 9 0 ) . Insertions cpo isolated         A number cpo alleles cause severe behavioral <br /> independent   enhancerdetector <br />                             screens                    defects adults. Some behavioral abnormal- <br /> expression lacZ reportergene essentially             ities enhanced uncharacterized mu- <br /> confined theSMC differentiated      cells      tations results various behavioral assays with <br /> of PNS embryonic development. These                 revertants clearly indicate cpo requiredfor <br /> insertionstrains interesting             normal adult behavior. However, behavioral de- <br /> enhancer detector strainsat positions 90D           fects indicative general sickness than <br /> label sensory mother cells progeny. In           specific behavioral deficits. An analysis lacZ <br /> addition, sensory organs dispensi-           pattern various 90D enhancer detector strains <br /> ble viability, number cpo insertional mutations       suggests cpo expressed CNS, PNS and <br /> were foundto beembryonic lethal suggesting            salivary glands instar larvae adults but <br /> gene plays important role development             tissues. We conclude that <br /> function PNS.                                           cpo gene probably required mature nerv- <br />    The observation cpo essential raises im-         ous normal adult behavior. <br /> portant question: In tissue cpo required? A              Complementation behavioral tests showed that <br />  374                                                  J. Bellen et al. <br />  <br /> the v 3 insertion causes behavioral defects butthat           iological role. The cpo gene ispossibly the <br /> these defects observed trans-heterozygous          hottest spot P element-enhancer detector inser- <br /> flies carry av 3 insertion anothercpo insertion      tions genetic evidence indicates genomic <br /> that affects behavior causes lethality. Since excision     region P elements inserted key <br /> of v 3 allowed recover strains exhibit     regulatory region gene. Further molecular and <br /> wild type behavior, conclude v 3 insertion        genetic analyses progress determine the <br /> causes cpo phenotype, uncharacter-           role cpo nervous system. <br /> ized mutations unlikely play arole the <br />                                                                  We grateful CLIVEWILSONfor helping initial <br /> phenotype. It possible v 3 affect         steps project. We thank ROLF BODMERfor teaching H. <br /> gene near cpo plays similar role cpo,          BELLEN   intricacies embryonic PNS, RON DAVIS   and <br /> this unlikely v 3 inserted         KAI ZINN providing libraries. We thank COREYGOODMAN <br /> insertions cause lethality affect         RICHARD    DORSKY    communicating unpublished results, Su- <br />                                                               SAN SHEPERD     ED GRELLfor advice, KEVIN MOSES, GERRY <br /> behavior. We conclude v 3 insertion <br />                                                               RUBIN,DON GAILEY,       JEFF HALL,ALBERTO    FERRUSand Dro- <br /> complements insertions cause cpo phe-            sophila stock centers Bloomington (Indiana), Bowling Green <br /> notype. This intragenic complementation            (Ohio), Umea (Sweden) sending fly strains us. We thank <br /> to transvection-like phenomena insertions           BETHELDON,PATRICK LAWINGER,         YUN-TAIKKIM andJuAN BOTAS <br /> disrupt somatic pairing chromosomes trans-          comments manuscripts, anonymous reviewers <br />                                                               fortheir constructive helpful criticism. This research was <br /> heterozygous flies.                                           supported grant Muscular Dystrophy Association to <br />    Mutations cpo stripe fail complement         H.B.H.B. Assistant Investigator Howard Hughes <br /> tested deficiencies cytological subdivisions 90D/E.        Medical Institute. <br /> Since mutants gene      fail fly, since <br /> preliminary evidence suggests stripe required                             LITERATURE CITED <br /> in nervous (COSTELLO      WYMAN 1986;          BELLEN,   H. J., C. WILSON W. GEHRING, 1990 Dissecting the <br /> DE LA POMPA, GARCIA FERRUS         1989), investi-          complexity nervous enhancer detection. <br /> gatedthe possible relationshipbetweenthese                     BioEssays 12: 199-204. <br /> genes greater detail. Based following obser-        BELLEN,   H., J., C. J. O&apos;KANE,C. WILSON,U. GROSSNIKLAUS,        R. K. <br />                                                                    PEARSON    W. GEHRING,1989 P-element mediated en- <br /> vations believe cpo stripe different               hancer detection: versatile method study development in <br /> genes map close proximity other: first,            Drosophila. Genes Dev. 3: 1288-1300. <br /> mutations genes complement             BENZER,S . , 1967 Behavioral mutants Drosophila isolated by <br /> the typical stripe phenotype, i.e., sr&apos;lcpo flies           countercurrent distribution. Proc. Natl. Acd.Sci. USA 58: <br />                                                                    1112-1119. <br /> show adarkthoracicstripe;second,none                   BIER,E., H. VAESSIN,    S. SHEPHERD,   K. LEE,K. MCCALL,   S . BARBEL, <br /> behavioral defects observedin cpo mutants observed              L. ACKERMAN,     R. CARRETTO,   T. UEMURA,   E. GRELL,L. Y. JAN <br /> in trans-heterozygous srlcpo flies; third, genomic frag-           Y. N. JAN, 1989 Searching pattern mutation in <br /> ments adjacent enhancer detector insertionsin               Drosophila genome P-lacZ vector. Genes Dev. 3: <br /> stripe gene contained 50 kb genomic                  1273-1287. <br />                                                               BLOCHLINGER,     K., R. BODMER,   R., J. W. JACK, L. Y. JAN Y. N. <br /> walk shown Figure 4; fourth, situ hybridization             JAN, 1988 Primary structureand expression aproduct <br /> experiments polytene chromosomes se-                 cut, locus involved specifying sensory organ identity <br /> quences 90D walk revealed                  Drosophila. Nature 333: 629-635. <br /> sequences walks hybridize respectively             CANAL,I . , A. FERRUS,1986 The pattern early neuronal <br />                                                                    differentiation Drosophila melanogaster. J. Comp. Neurol. <br /> 90E1-2 (stripe) 90D2-6 (cpo), agreement                177: 707-720. <br /> the mapping positions respective enhancer de-          CAMPOS-ORTEGA,      J. A., V. HARTENSTEIN,   1985 The Embryonic <br /> tector insertions; fifth, P-galactosidase expression           Development Drosophila melanogaster.Springer Verlag, Berlin. <br /> pattern enhancer detector strains inserted         CAMPOS-ORTEGA,       J. A., E.KNUST, 1990 Genetics early <br /> o r stripe different expression pattern             neurogenesis Drosophila melanogaster. Annu. Rev. Genet. 24: <br />                                                                    387-407. <br /> (data shown). These observations provide com-             COSTELLO,    W. J., R. WYMAN,     1986 Development indi- <br /> pelling evidence genes different. How-               rect flight muscle muscle-specific mutant Drosophila <br /> ever, muscle defects observed               melanogaster. Dev. Biol. 11.8: 247-258. <br /> trans-heterozygous stripelcpo flies, suggesting          DE LA POMPA,    J. L., J. R. GARCIA   A. FERRUS, 1989 Genetic <br />                                                                    analysis muscledevelopment Drosophila melanogaster.Dev. <br /> both genes interact (data shown).                  Biol. 131: 439-454. <br />    In conclusion, shown cpo required          DOE, C., Q. CHU-LAGRAFF,           D. M. WRIGHTand M. SCOTT, <br /> for embryonic viability fornormal adult behavior.              1991 The prosper0 gene specifies cell fates Drosophila <br /> Its expression pattern indicates mayplay                central nervous system. Cell 65: 451-464. <br /> importantrole PNS and/or glia. However,                FREDIEU,   J. R., A. P. MAHOWALD,1989 Glial interactions <br />                                                                    neurons Drosophila embryogenesis. Development <br /> absence gene product does cause ob-                 1 0 6 749-757. <br /> vious morphological defects cells indicating         GAILEY,D., J. C. HALL, 1989 Behavior cytogenetics of <br /> that cpo play unknown biochemical phys-                 fruitless Drosophila melanogaster: different courtship defects <br />                                                         Genetics couch potato                                                    375 <br />      caused separate, closely linked lesions. Genetics 121: 773-       glia axon pathway formation Drosophila em- <br />      785.                                                                 bryo. Glia (in press). <br /> GANETZKY,     B., C. F. Wu, 1986 Neurogenetics membrane         LANGER-SAFER,P. R., M. LEVINE C.D.                     WARD, <br />      excitability Drosophila. Annu. Rev. Genetics 20: 13-44.           1982 Immunological method mapping genes Drosoph- <br />  GHYSEN,A., C. J. O&#226;&#8364;&#8482;KANE,1989 Detection neural en-                 ila polytene chromosomes. Proc. Natl. Acad. Sci. USA 79: <br />                                                                           4381-4385. <br />      hancer-like elements genome Drosophila. Develop- <br />                                                                        MLODZIK,M., Y . HIROMI,U. WEBER,C . S . GOODMAN        G. M. <br />      ment 105: 35-52. <br />                                                                           RUBIN,1990 The Drosophila seven-up gene, member the <br />  GHYSEN,A., C. DAMBLY-CHAUDIERE,          1989 Genesisof the <br />                                                                           steroid receptor gene superfamily, controls photoreceptor cell <br />      Drosophila peripheral nervous system. Trends Genet. 5: 251-          fates. Cell 6 0 21 1-224. <br />      255.                                                              MOSES,K., M. ELLISand G. RUBIN1989 The glass gene encodes <br />  HALL,J., 1985 Genetic analysis behavior ininsects. Compr.             zinc-finger proteinrequired Drosophila photoreceptor <br />      Insect Physiol. Biochem. Pharmacol. 9: 287-373.                      cells. Nature 3 4 0 531-536. <br /> JACOBS, J. R., C. S. GOODMAN,    1989 Embryonic development        O&#226;&#8364;&#8482;KANE,C. J., W. GEHRING,1987 Detection insitu ge- <br />      axon pathways Drosophila CNS. I. A glial scaffold          nomic regulatory elements Drosophila. Proc. Natl. Acad. Sci. <br />      appears growth cones. J. Neurosci. 9 2402-          USA 8 4 9 123-9127. <br />      2411.                                                             RUBIN,G. M., 1988 Drosophilamelanogaster experimental <br /> JACOBS,J. R., Y . HIROMI,N. H. PATELand C. S. GOODMAN,                    organism. Science 240: 1453-1459. <br />      1989 Lineage, migration, morphogenesis longitudinal        SORSA,V., 1988 Chromosome Maps Drosophila, Vol. 2, p. 146. <br />      glia Drosophila CNS revealed molecular lineage        CRC Press, Boca Raton, Fla. <br />                                                                        TAUTZ, D., C. PFEIFLE1989 A non-radioactive situ hybrid- <br />      marker. Neuron 2: 1625-1 631. <br />                                                                           ization method localization specific RNA&#226;&#8364;&#8482;s Drosophila <br /> JAN, L. Y., Y . N. JAN, 1982 Antibodies horseradish per- <br />                                                                           embryos reveals translational control segmentation <br />      oxidase specific neuronal markers Drosophila gras-      gene hunchback. Chromosoma 9 8 81-85. <br />      shopper embryos. Proc. Natl. Acad. Sci. USA 7 9 2700-2704.        WILSON,C., R. K. PEARSON,H. J. BELLEN,C. J. O&#226;&#8364;&#8482;KANE,U. <br /> JAN, Y . N., L. Y .JAN, 1990 Genes required specifying cell       GROSSNIKLAUS     W. GEHRING1989 P-element mediated <br />      fates Drosophila embryonic sensory nervous system. Trends         enhancer detection: anefficient method isolating char- <br />      Neurosci. 21: 493-498.                                               acterizing developmentally regulated genes Drosophila. <br /> JURGENS,G., R. LEHMANN,         M. SCHARDIN  C.NUSSLEIN-VOL-          Genes Dev. 3: 1301-1313. <br />      HARD, 1986 Segmental organization head em-          ZINN,K., L. MCALLISTERand C. S. GOODMAN,         1988 Sequence <br />      bryo Drosophila melanogaster. Roux&#226;&#8364;&#8482;s Arch. Dev. Biol. 195:        analysis neuronal expression fasciclin I grasshopper <br />      359-377.                                                             Drosophila. Cell 53: 577-587. <br />  KLAMBT, C.,   C. S. GOODMAN,   1991 The diversity pattern                                Communica <br /> </body></html>