<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Gene connectivity, function, and sequence conservation: <br /> predictions <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> modular yeast co-expression networks <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Genes and proteins are organized into functional modular networks in which the <br />                  network context of a gene or protein has implications for cellular function. Highly connected hub <br />                  proteins, largely responsible for maintaining network connectivity, have been found <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> be much <br />                  more likely to be essential for yeast <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>survival</span>. <br />                  Results: Here we investigate the properties of weighted gene co-expression networks formed <br />                  from multiple microarray datasets. The constructed networks approximate scale-free topology, <br />                  but this is not universal across all datasets. We show strong positive correlations between gene <br />                  connectivity within the whole network and gene essentiality as well as gene sequence conservation. <br />                  We demonstrate the preservation of a modular structure of the networks formed, and <br />                  demonstrate that, within some of these modules, it is possible to observe a strong correlation <br />                  between connectivity and essentiality or between connectivity and conservation within the <br />                  modules particularly within modules containing larger numbers of essential genes. <br />                  Conclusion: Application of these techniques can allow a finer scale prediction of relative gene <br />                  importance for a particular process within a group of similarly expressed genes. <br />  <br />  <br />  <br />  <br /> Background                                                                    important as understanding its function in isolation [2]. <br /> Genes and their protein products carry out cellular proc-                     Because of this, numerous studies have focused on the <br /> esses in the context of functional modules and are related                    large scale modeling of genomic and proteomic data. Uti- <br /> to each other through a complex network of interactions                       lizing network theory, these studies have yielded insights <br /> [1]. Understanding an individual gene or protein&apos;s net-                       into biological systems. For example, both protein inter- <br /> work properties within such networks may prove to be as                       action networks and gene co-expression networks exhibit <br />  <br />                                                                                                                                        Page 1 of 15 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                                                     http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br />  <br />           $  $UUD\ &apos;DWD                                                                       a strong modularity reflecting functional partitioning. <br />                                                                                               Both of these network types have been frequently <br />                                                                                               observed as having a scale-free topology, with the exist- <br />             ([SUHVVLRQ /HYHO <br />  <br />  <br />                                                                                               ence of highly connected hub nodes [3-7]. <br />  <br />                                                                                               Scale-free networks are resistant to random perturbations <br />                                                                                               but sensitive to targeted removal of highly connected <br />                                                                                               nodes [8]. Comprehensive efforts to determine the func- <br />                                                    6DPSOH 7\SH                                tional consequences of individual gene deletions in yeast <br />                                 &amp;RUUHODWLRQ FRHIILFLHQWV IRU DOO JHQHV                        provide the opportunity to study the relationship between <br />                                                                                               individual gene network properties and gene deletion <br />           %  6LPLODULW\ 0DWUL[  FRUUHODWLRQ                                                   lethality [9]. For example, physical interaction studies in <br />                                *      *     *     *     *      *      *     *     *     *   <br />                                                                                               yeast have allowed comparison of connectivity to gene <br />               *                                                                               essentiality based on gene deletion. [10]. Typical of scale- <br />               *                                                                             <br />               *                                                                               free networks, there were few highly connected proteins <br />               *                                                                               within the network, and the deletion of a protein with a <br />               *                                                                             <br />               *                                                                               large number of binding partners is more likely to be <br />               *                                                                             <br />               *                                                                             <br />                                                                                               lethal in yeast. Thus, the relative position of nodes within <br />               *                                                                               a protein interaction network is strongly affiliated with <br />               *                                                                            <br />                                                                                               distinct biological properties of individual proteins. Simi- <br />                                      7KUHVKROG FRUUHODWLRQV LQWR HGJHV                        larly, analysis of unweighted gene co-expression networks <br />                                                                                               have revealed a relationship between connectivity and <br />           &amp;  $GMDFHQF\ 0DWUL[                                                                 essentiality across all genes [11]. <br />                                *  *  *  *  *                  *      *      *  *  *   <br />                     *          1$ 2 2 (     (                  2      (      2 2 2 <br />                     *           2 1$ (   2 2                   2      (      2 2 2            Correlation of gene expression across a wide variety of <br />                     *           2 ( 1$ 2 2                     2      (      2 2 (            experimental perturbations has been shown to cluster <br />                     *           (  2 2 1$ (                    (      (      2 2 2 <br />                     *           (  2 2 ( 1$                    (      (      2 2 2            genes of similar function [12]. Since this guilt-by-associa- <br />                     *           2 2 2 (     (                 1$      2      2 2 2 <br />                     *           (  (  (  (  (                  2     1$      (  (  2 <br />                                                                                               tion approach may lead to false positive groupings, <br />                     *           2 2 2 2 2                      2      (     1$ (   2          approaches have been refined by comparing orthologs <br />                     *           2 2 2 2 2                      2      (      ( 1$ ( <br />                     *           2 2 (    2 2                   2      2      2 ( 1$           across divergent species indicating that highly conserved <br />                                                                                               co-expression is a strong predictor that two genes will <br />                                                   &apos;UDZ QHWZRUN <br />                                                                                               function in similar pathways [13-15]. This indicates that <br />           &apos;  1HWZRUN                                                                          functionally related genes are under similar expression <br />                                                                                               constraints. The gene co-expression networks that are <br />                                                                                               based on these relationships have been shown in multiple <br />                                                                                               species to be frequently scale-free and exhibit a small <br />                                                                                               world architecture similar to protein interaction networks <br />                                                                                               even though they are generally more strongly connected <br />                                                                                               [3,14]. It is still unclear however, to what degree the net- <br />                                                                                               work properties of individual genes within a co-expres- <br />                                                                                               sion network can predict relative gene importance for a <br />                                                                                               particular process. To assess this, we have constructed <br />                                                                                               three networks based on a weighted measure of connectiv- <br /> Figure 1 a gene co-expression network <br /> Generating                                                                                    ity [16] of correlated gene expression in yeast using three <br /> Generating a gene co-expression network. (A) Illustra-                                        separate microarray data sets. We assessed relationships <br /> tion to show how genes that are highly correlated (blue) will                                 between essentiality and connectivity of each gene within <br /> look across a dataset. (B-C) Flow chart for defining a gene                                   the whole network. Further, we define &apos;modules&apos; (groups <br /> co-expression network based on a simple Pearson correla-                                      of highly correlated genes) and determine that in some <br /> tion matrix. (D) An example visualization of a network pro-                                   instances, the relative importance of genes within these <br /> duced by using the Fruchterman Rheingold algorithm on the                                     modules can be inferred from network connectivity. We <br /> data in Figure 1B and 1C. The node highlighted in yellow, is                                  demonstrate that genes which have high connectivity (i.e. <br /> an example of the kind of highly connected node that this                                     &apos;hub&apos; genes) within a weighted co-expression network are <br /> study shows is more likely to be an essential gene for sur-                                   significantly more likely to be essential for yeast viability. <br /> vival. <br />                                                                                               Furthermore, we demonstrate a relationship between con- <br />                                                                                               nectivity and a measure of sequence conservation. Finally, <br />  <br />  <br />  <br /> Figuregene <br /> Global  2 co-expression networks <br /> Global gene co-expression networks. (A-C) Log-log plots of connectivity distributions in each of the three networks <br /> drawn from DNA Damage, Environmental Response, and Cell Cycle, respectively. The linear relationship indicates a scale-free <br /> structure. (D-F) Correlation plots showing the relationship between gene group connectivity and essentiality in the same order <br /> as A-C. On the y axis of these plots is the percentage of genes determined to be essential in yeast. (G-I) Relationship between <br /> the average blastp score of a node and k for all datasets in the same order as A-C. On the y axis of these plots is the average <br /> log(e score) of genes within that bin. For plots D-I, the connectivity within each network for each gene was determined and <br /> each gene rank ordered by connectivity. 20 equal sized bins for each of the expression datasets were formed and the average <br /> connectivity of the genes in each bin plotted on the X axis <br />  <br /> Figure <br /> A  co-expression <br />           3       network of the DNA Damage dataset <br /> A co-expression network of the DNA Damage dataset. For all panels in the figure: blue color represents members of <br /> the rRNA processing module, yellow color represents members of the protein synthesis module and red color represents <br /> members of the ubiquitin pathway. (A) A hierarchical clustering of the topological overlap matrix for the DD dataset. (B) A <br /> drawn network of gene co-expression from the DD dataset. Edges were computed from the Pearson correlation coefficients. <br /> Network structure was drawn in Pajek [20]. Each gene is represented as a dot. Edges are drawn as grey lines. Colored dots <br /> belong to the module that their color indicates and grey dots indicate all other genes in the network. (C-E) Scatter plots show- <br /> ing the relationship between gene group connectivity and essentiality for rRNA processing, protein synthesis and ubiquitin <br /> from the DD network respectively. On the y axis of these plots is the percentage of genes determined to be essential in yeast. <br /> The number of essential genes in each module/total number of genes in each module was: 196/390 = 50.3 (C), 122/441 = 27.7 <br /> (D), 50/222 = 22.5 (E). (F-H) Relationship between the average blastp score of a node and k for the same module members <br /> listed in C-E. On the y axis of these plots is the average log(e score) of genes within that bin. For plots C-H, the connectivity <br /> within each module for each gene was determined and each gene rank ordered by connectivity. 20 equal sized bins for each of <br /> the expression datasets were formed and the average connectivity of the genes in each bin plotted on the X axis. <br />  <br /> Figure <br /> A  co-expression <br />          4         network of the Environmental Response dataset <br /> A co-expression network of the Environmental Response dataset. For all panels in the figure: blue color represents <br /> members of the rRNA processing module, yellow color represents members of the protein synthesis module and red color <br /> represents members of the ubiquitin pathway. (A) A hierarchical clustering of the topological overlap matrix for the ER data- <br /> set. (B) A drawn network of gene co-expression from the ER dataset. A drawn network of gene co-expression from the DD <br /> dataset. Edges were computed from the Pearson correlation coefficients. Network structure was drawn in Pajek [20]. Each <br /> gene is represented as a dot. Edges are drawn as grey lines. Colored dots belong to the module that their color indicates and <br /> grey dots indicate all other genes in the network. (C-E) Correlation plots showing the relationship between gene group con- <br /> nectivity and essentiality for rRNA processing, protein synthesis and ubiquitin from the ER network respectively. On the y axis <br /> of these plots is the percentage of genes determined to be essential in yeast. The number of essential genes in each module/ <br /> total number of genes in each module was: 317/929 = 34.1 (C), 98/323 = 30.3 (D), 18/37 = 48.6 (E). (F-H) Relationship <br /> between the average blastp score of a node and k for the same module members listed in C-E. On the y axis of these plots is <br /> the average log(e score) of genes within that bin. For plots C-H, the connectivity within each module for each gene was deter- <br /> mined and each gene rank ordered by connectivity. <br />  <br /> Figure <br /> A co-expression <br />          5         network of the Cell Cycle dataset <br /> A co-expression network of the Cell Cycle dataset. For all panels in the figure: blue color represents members of the <br /> rRNA processing module, yellow color represents members of the protein synthesis module and red color represents mem- <br /> bers of the ubiquitin pathway. (A) A hierarchical clustering of the topological overlap matrix for the CC dataset. (B) A drawn <br /> network of gene co-expression from the CC dataset. A drawn network of gene co-expression from the DD dataset. Edges <br /> were computed from the Pearson correlation coefficients. Network structure was drawn in Pajek [20]. Each gene is repre- <br /> sented as a dot. Edges are drawn as grey lines. Colored dots belong to the module that their color indicates and grey dots indi- <br /> cate all other genes in the network. (C-E) Correlation plots showing the relationship between gene group connectivity and <br /> essentiality for rRNA processing, protein synthesis and ubiquitin from the CC network respectively. On the y axis of these <br /> plots is the percentage of genes determined to be essential in yeast. The number of essential genes in each module/total <br /> number of genes in each module was: 154/300 = 51.3 (C), 105/398 = 26.4 (D), 38/312 = 12.2 (E). (F-H) Relationship between <br /> the average blastp score of a node and k for the same module members listed in C-E. On the y axis of these plots is the aver- <br /> age log(e score) of genes within that bin. For plots C-H, the connectivity within each module for each gene was determined <br /> and each gene rank ordered by connectivity. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                  Page 6 of 15 <br />                                                                                                                                                                                                                                                                                                                                       (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                       http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> we show that certain critical modules are conserved from        (Figures 2A, 2B, and 2C). These results are similar to the <br /> one network to another, and that in many cases it is pos-       general trend with genetic perturbation networks and <br /> sible to extend the relationship between connectivity and       other gene co-expression networks [3,22]. However, gene <br /> essentiality or between connectivity and conservation           co-expression networks may better fit an exponential <br /> within given modules. Thus, analysis of gene co-expres-         power law [23]. For two of the three constructed yeast net- <br /> sion networks provides insight into the functional impor-       works (DD and ER), the distribution of connectivity (k) of <br /> tance of individual genes within modules of co-expressed        genes best fits a power law p(k) ~ k-&#206;&#179; and the other (CC) <br /> genes.                                                          best fits an exponentially truncated power law p(k) ~ k-&#206;&#179; <br />                                                                 e-&#206;&#177;k [24]. We point out that in general that regardless of <br /> Results                                                         the overall structure, the major trend is that the high con- <br /> Network validation and properties                               nectivity genes are few in number, and that most genes <br /> In order to test the relationship between gene connectivity     have low connectivity (Figure 2A&#226;&#8364;&#8220;2C). <br /> within the co-expression network and gene essentiality, <br /> we first constructed three networks corresponding to three      Global network connectivity relationships <br /> microarray datasets. The yeast microarray data were             Network modeling in biological systems [2,8,11] stresses <br /> derived from experiments designed to analyze DNA Dam-           the relative importance of highly connected nodes. Thus, <br /> age (DD) (n = 51), Cell Cycle (CC) (n = 44), and Environ-       the small number of high connectivity genes within the <br /> mental Response (ER) (n = 151) [17-19]. Each network            gene co-expression networks defined here are expected to <br /> was constructed as conceptually shown in Figure 1. Nodes        be more important. To test the relative importance of each <br /> were defined as individual genes. The co-expression net-        gene across the entire network, we determined the rela- <br /> work is based upon the correlation relationships between        tionship between connectivity and gene essentiality. All <br /> individual genes across a dataset (Figure 1A). From the         the genes for a network were first ranked according to <br /> microarray data, the absolute value of the Pearson corre-       their weighted connectivity and then placed into 20 equal <br /> lation is the initial measure of gene co-expression similar-    sized bins according to their position in this ranking. The <br /> ity (Figure 1B). This co-expression similarity was then         weighted connectivity of a given gene is defined as the <br /> transformed into an adjacency matrix (Figure 1C) by rais-       sum of its connection strengths with all other genes in the <br /> ing the similarity to a certain power based on a scale free     network (see the methods section). The percentage of <br /> topology criterion as described previously [16]. The pow-       essential genes in each bin was then plotted against the <br /> ers selected were 12 (DD), 10 (CC), and 18 (ER). These          mean connectivity in each bin (Figure 2D&#226;&#8364;&#8220;2F). We also <br /> data can also be visualized as a drawn network (Figure          calculated the Spearman correlation coefficient based on <br /> 1D).                                                            the ranked unbinned data and found that there is a rela- <br />                                                                 tionship between the connectivity and the proportion of <br /> Each of the datasets resulted in the construction of distinct   essential genes in each of the three datasets (R &gt; 0.2, and <br /> networks but with global similarities. The differences in       p value &lt; 10-39) for each network. While the binned plots <br /> the modularity of the three networks is demonstrated by         are well suited for visualizing the results, we calculated the <br /> comparing the heat maps of the topological overlap              correlation coefficients on the raw, unbinned data to <br /> matrices (Figures 3A, 4A, and 5A). A topological overlap        arrive at a valid p-value. <br /> matrix is an indicator of the similarity of all the nodes in <br /> question and is defined in detail in the methods section of     To study whether our conclusions were being unduly <br /> this manuscript. Approximations of all three of the net-        influenced by data outliers, we removed genes whose con- <br /> works are also drawn in Figures 3B, 4B and 5B using the         nectivity was more than two standard deviations away <br /> Fruchterman-Reingold algorithm as implemented in                from mean connectivity but found no substantial varia- <br /> Pajek [20,21] and demonstrate obvious modularity with           tion in the reported relationships. Thus, there is a clear <br /> aggregation of genes into strongly co-expressed groups.         relationship between connectivity of genes, in these <br />                                                                 weighted gene co-expression networks, and gene essenti- <br /> Individual gene connections can be difficult to visualize       ality across the whole network. These results are robust <br /> in such complex graphs as presented. Thus, in order to          with respect to experimental conditions since the three <br /> probe the structure of the network, we first sought to          separate datasets all have the same functional relationship <br /> determine characteristics of the general network topology.      between connectivity and gene essentiality. These results <br /> We considered two models. To quantify the fit to the scale      are consistent with a single whole network generated from <br /> free topology model or to the truncated exponential             gene co-expression data and using an unweighted net- <br /> model, we used the model fitting index R-squared of the         work [11], as well as networks based on protein-protein <br /> corresponding linear model involving the log trans-             interactions [10]. <br /> formed variables [16]. The <span id='am-243' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of network connectiv- <br /> ity in these datasets approximated a scale-free topology <br />  <br />  <br />                                                                                                                     Page 7 of 15 <br />                                                                                               (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                                      http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> Table 1: Identifying modules that repeated from one dataset to another <br />  <br />  Ontology term                                        rRNA                                  Prot                                   Ubiq <br />  <br />                                           DNA          Env         Cell         DNA          Env        Cell          DNA          Env          Cell <br />                                           Dmg         Stress       Cycle        Dmg         Stress      Cycle         Dmg         Stress        Cycle <br />  <br />  35S primary transcript processing         1E-17      8.8E-16     7.5E-16        0.17         0.51       0.52          0.26           1          0.16 <br />  processing of 20S pre-rRNA                2E-32       5E-19      3.8E-32       0.018         0.37        1            0.17           1          0.11 <br />  ribosomal large subunit biogenesis       9.6E-13   0.0000071     3.5E-15        0.38          1         0.38            1            1          0.62 <br />  rRNA modification                        2.1E-08     0.0043      0.00014        0.66        0.019     0.00066           1            1          0.62 <br />  rRNA processing                          1.2E-25     1.6E-16     2.5E-23        0.48          1         0.62          0.11           1         0.046 <br />  protein biosynthesis                     0.0001      1.1E-11      0.0023      5.9E-83     5.6E-108    1.4E-80         0.53         0.42       0.00033 <br />  translational elongation                   0.62        0.14        0.63       2.1E-08       3E-08     1.7E-09           1            1          0.64 <br />  translational initiation                   0.19          1        0.025      0.000017      1.8E-14    3.4E-07         0.41           1          0.25 <br />  ubiquitin-dependent protein               0.039        0.02       0.071        0.027        0.048      0.018        2.5E-25      7.3E-25      6.3E-17 <br />  catabolism <br />  <br />  ATP-dependent RNA helicase               7.2E-11     9.2E-08     4.1E-11        0.39         0.62         1             1           1          0.62 <br />  activity <br />  DNA-directed RNA polymerase              6.1E-13     4E-08       1.4E-09        0.77        0.012       0.13          0.42          1           0.4 <br />  activity <br />  snoRNA binding                           1.3E-27     7.6E-17     2.4E-25       0.042         0.51        0.76         0.42           1          0.16 <br />  structural constituent of ribosome       0.00012     2.2E-11    0.000078      3.1E-92     5.3E-117     1.8E-95       0.097         0.41       0.0021 <br />  translation initiation factor activity     0.16        0.48        0.14       4.7E-08      5.5E-17     2.3E-10         1             1          0.39 <br />  endopeptidase activity                     0.25        0.21        0.62         0.16         0.62        0.39       9.2E-21      2.6E-18      2.7E-11 <br />  <br />  nucleolus                                1.6E-48     3.4E-29     1.3E-45       0.18          0.65       0.84          0.021        0.48       0.0056 <br />  nucleus                                  1.2E-18     8.3E-13     7.8E-10      0.0053       2.1E-07      0.12         0.0048          1        0.033 <br />  small nucleolar ribonucleoprotein        7.9E-20     2.8E-09     1.3E-14       0.16          0.67       0.42          0.63           1         0.39 <br />  complex <br />  cytosolic large ribosomal subunit         0.03     0.0000016     0.0087       2.1E-65      3.4E-75     6.1E-67        0.021         1         0.0056 <br />  cytosolic small ribosomal subunit         0.059      0.0013       0.26        1.5E-41      1.2E-47     4.6E-42        0.073         1          0.03 <br />  ribosome                                    1         0.39         0.3        2.7E-09      6.2E-14     3.7E-09          1           1          0.62 <br />  <br />  *For each module in each dataset analyzed, an EASE analysis [25] was performed by using Fisher&apos;s exact test and the resulting p-values were placed <br />  into the table based on the prevalence of the GO (Gene Ontology) term being searched for. The most significant enrichments for a given term are <br />  indicated in italics <br />  <br />  <br /> Yeast co-expression networks fit an evolutionarily plausi-                  and found that there is a relatively weak but highly signif- <br /> ble model in which network connection distribution is a                     icant correlation indicating that high connectivity genes <br /> natural consequence of gene duplication, <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> and                       are more evolutionarily conserved (R = -0.15, p value = <br /> deletions [13]. The emergence of approximate power-law                      1.3 &#195;&#8212; 10-19, for the DD network; R = -0.22, p value &lt; 1.4 &#195;&#8212; <br /> distribution (scale free topology) is intimately linked to                  10-39 for ER network; and R = -0.18, p value = 4.0 &#195;&#8212; 10-29 <br /> the growth of the network in which new nodes are prefer-                    for the CC network). Our results were robust with respect <br /> entially attached to highly connected already established                   to the removal of outlier genes (i.e. genes whose connec- <br /> nodes [2,8]. A direct consequence of this network growth                    tivity was more than two standard deviations away from <br /> model is that high connectivity &apos;hub&apos; genes are more likely                 mean connectivity). This relationship is similar to that <br /> to be highly conserved across different species. Stated                     reported for gene perturbation networks [22] and in evo- <br /> another way, within a gene co-expression network the                        lutionarily conserved gene co-expression relationships <br /> oldest nodes would be more likely than the newer nodes                      [14]. <br /> to be in hub positions. With this in mind, we examined <br /> the relationship between connectivity and the relative                      Identification of modules <br /> evolutionary sequence conservation of a node (as evi-                       In order to identify modules, we performed hierarchical <br /> denced by the average best blastp hit log(e score) against                  clustering on the topological overlap matrix derived from <br /> N. crassa C. elegans, D. melanogaster, H. sapiens, and M.                   these networks as described in the methods section. In <br /> musculus). High connectivity genes are plotted in Figure                    doing so, we identified modules that involve conserved <br /> 2G&#226;&#8364;&#8220;2I relative to the average of the blastp hit in four other               processes for rRNA processing, protein synthesis, and <br /> eukaryotic genomes. As before, we calculated the Spear-                     ubiquitination (Table 1) using an EASE analysis [25]. One <br /> man correlation for the ranked, but still unbinned, data                    prediction about network constructions is that certain crit- <br />  <br />  <br />                                                                                                                                            Page 8 of 15 <br />                                                                                                                   (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                              http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> ically important modules ought to be repeatedly detected             ture. For the Protein Biosynthesis module, 135 of 172 <br /> in independent datasets. Within the network modules                  overlapping members were represented in the KEGG data- <br /> (groups of high connectivity genes within the network)               <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span>, and of these, 72 out of a possible 86 were compo- <br /> produced by the different datasets for gene ontology                 nents of the KEGG annotated yeast ribosome. Notably, if <br /> enrichment, 3 distinct modules that were common to the               the 172 overlapping members were first rank ordered by <br /> 3 different data sets were detected (Table 1 and Figure 6).          their average network connectivity, and the bottom quar- <br /> Comparing the modules that had an enrichment in rRNA                 tile removed, then 100% of the remaining members are <br /> processing, protein synthesis and the ubiquitin pathway              part of this same yeast ribosome structure, which shows <br /> terms across the DD, CC and ER datasets resulted in a                that in this overlap set the high connectivity genes were in <br /> highly significant overlap of 215 genes (hypergeometric              fact the members of the yeast ribosome. Finally, the over- <br /> distribution p = 3.1e-173). Similarly, the protein synthesis         lap of genes detected in the rRNA modules overlap identi- <br /> module also had significantly more overlap than expected             fied purine and pyrimidine synthesis and RNA <br /> by chance in which 172 genes were present in all three               polymerase 3 pathways. These pathways dominate this <br /> modules (p = 5.6e-99). The ubiquitin module had the                  list with 30 out of the 49 identified matches to the KEGG <br /> least overlap of common genes (12 genes, p = 3.4e-07)                database. Unlike the previous two results however, multi- <br /> probably because of its smaller size in general.                     ple pathways are found that represent this module. The <br />                                                                      three network modules discussed all contain large protein <br /> These three processes are well known to be fundamental               complexes. This is in agreement with the recent discovery <br /> processes and have been described before in great detail,            that there is often a correlation between genes that are <br /> and so we decided to investigate the nature of these over-           expressed together and proteins that bind to each other <br /> lapping regions further. Gene members were selected that             [27]. However, there is no reason to believe that modules <br /> were shared within similar functional modules from the               in co-expression networks will always correspond to pro- <br /> three datasets and searched for in the KEGG [26] database.           tein complexes since our module detection method <br /> In the ubiquitin pathway, 9 of the overlapping 12 genes              should find any set of genes with high pair-wise correla- <br /> were found in KEGG, and all of these were found to be in             tions. Any experimental perturbation that gives rise to a <br /> the same proteasome structure. Since the overlap for this            highly correlated gene set is expected to result in a corre- <br /> module was unusually small due in part to the very small             sponding network module. <br /> size of the ER dataset&apos;s ubiquitin module, we also looked <br /> at the overlap just of the CC and DD ubiquitin modules.              Intramodular network connectivity relationships <br /> Of the 57 genes present in both the CC and DD modules,               Having identified 3 functionally similar modules in the <br /> only 14 were present in the KEGG database. Of those 14               three different datasets, we assessed whether these same <br /> members, all of them were in the same proteasome struc-              modules contain the same information as the entire net- <br />  <br /> Figure <br /> Module 6 members retrieved from very different datasets are conserved <br /> Module members retrieved from very different datasets are conserved. For all panels in the figure: black color rep- <br /> resents modules retrieved from the DD dataset, blue color represents modules retrieved from the ER dataset, and green color <br /> represents modules retrieved from the CC dataset. In each panel A-C a different module is examined for overlap of its mem- <br /> bers in a Venn diagram. In each diagram the relative overlap of the various modules has been diagramed as different amounts of <br /> area with the count represented for each zone by a number that quantifies the precise number of matches. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 9 of 15 <br />                                                                                                     (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                       http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> work. Is the connectivity of a gene within the identified        between k.in and essentiality (DD, ubiquitin module). <br /> modules correlated with the likelihood that a gene was           Thus, the detection of a relationship between intramodu- <br /> essential by deletion analysis and also whether it was con-      lar connectivity and sequence conservation may predict <br /> served (Figures 3, 4, 5)? To test connectivity correlations      those modules with a relationship between k.in and <br /> within these modules, we define the intramodular con-            essentiality. <br /> nectivity (k.in) as the sum of all connectivity measures <br /> only with genes within the same module. Similar to that          Discussion <br /> performed for the total gene connectivity within the             Our results extend the inferences made from protein and <br /> whole network, binned values were plotted to observe the         metabolic networks to the more prolific datasets available <br /> possible relationships (Figures 3C&#226;&#8364;&#8220;H, 4C&#226;&#8364;&#8220;H, and 5C&#226;&#8364;&#8220;H).           from gene expression data. These results are made possi- <br /> To evaluate significance, Spearman correlation coeffi-           ble due to broad surveys of expression and the compre- <br /> cients for the ranked but unbinned data were calculated          hensive characterization of the yeast genome with <br /> for relationship of k.in to essentiality and sequence con-       deletion mutants. In contrast to gene expression data, <br /> servation. Interestingly, there was a general trend to simi-     which are based on the assessment of cells in many differ- <br /> lar correlations as that seen within the whole network.          ent states of perturbation, protein interaction measure- <br /> The correlation with sequence conservation was generally         ments are determined in only a limited set of conditions <br /> weak: The ubiquitin module formed from the DD dataset            which by nature only capture a subset of all possible pro- <br /> and the rRNA processing modules in the ER and CC were            tein-protein interactions. Others [22,28,29] have con- <br /> the most correlated with k.in. There was a correlation of        structed gene perturbation networks by looking at the <br /> k.in and essentiality in all three datasets within the rRNA      expression changes when individual genes are eliminated <br /> Processing module (Figures 3C, 4C and 5C), and for two           or in relation to transcription factor expression. Microar- <br /> of the datasets (DD and ER but not CC) within the ubiq-          ray based surveys of gene expression allow detailed sur- <br /> uitin module, and a trend towards a weak anticorrelation         veys of gene expression across a wide variety of <br /> within the protein synthesis module (Figures 3D, 4D, and         experimental perturbations. From these data a strong cor- <br /> 5D). Thus, some of the relationships between connectiv-          relation of gene expression for two genes implies that <br /> ity and essentiality or sequence conservation and not            both genes act within a common functional group and are <br /> detected when the analysis is restricted within the mod-         under similar transcriptional control. Since these net- <br /> ules.                                                            works are not based on direct measurements of protein <br />                                                                  binding/interaction, the presence of two genes that are <br /> Characterizing modules that contain essential hub genes          highly correlated has a different interpretation than that <br /> The relationship between intramodular connectivity and           derived from a protein interaction network. In a co- <br /> essentiality is not necessarily universal for any module in      expression network, two closely tied nodes would be <br /> any dataset. Whether or not a module contains meaning-           highly correlated but not necessarily direct binding part- <br /> ful information for predicting essentiality depends on the       ners. In spite of this key difference, numerous other stud- <br /> expression data (biological perturbations) that were ana-        ies have indicated that genes that are correlated with each <br /> lyzed. One can view the correlation of k.in and essentiality     other across a modest range of experiments tend to func- <br /> as a test of the modules importance. The key issue is that       tion in similar ways and that this functional categoriza- <br /> in order for a module to demonstrate this relationship,          tion is strengthened when gene co-expression exists across <br /> there must be a sufficient number of essential genes             large evolutionary distances [12,14]. <br /> within the module. As some modules have substantially <br /> fewer essential genes than others, there is little power to      Here, we use a more general weighted gene co-expression <br /> detect any relationship with k.in within those modules.          network construction method for network construction <br /> We calculated the Spearman correlation between the pro-          that can be applied generically to microarray data using <br /> portion of essential genes within a module and the corre-        simple correlational analyses of individual genes [16]. <br /> lation of k.in with essentiality and show that there is a        Advantages of weighted networks over unweighted net- <br /> strong positive correlation (r = 0.48). Since in actual appli-   works based on dichotomizing the correlation matrix <br /> cations the goal will be to find unknown &apos;essential&apos; genes,      include a) that the continuous nature of the gene co- <br /> it would be preferable to have another metric of &apos;essential&apos;     expression information is preserved and b) that the results <br /> gene enrichment. In this regard, it is interesting to note       of weighted network analyses are highly robust with <br /> that of the three modules studied in the 3 datasets (9           respect to the choice of the parameter &#206;&#178;, [16]. We show <br /> total), there were 4 that had a correlation of k.in with         that there is a strong correlation with the weighted gene <br /> sequence conservation (p &lt; 0.05). Of these 4 modules, all        co-expression connectivity measure and gene essentiality <br /> 4 were shown to have a strong relationship of k.in with          and sequence conservation. Our analysis demonstrates <br /> essentiality in the same modules. Of the 5 without a cor-        that individual conserved modules are identifiable within <br /> relation (p &gt; 0.05), only one showed a correlation               a network. Moreover, the presence of these modules, <br />  <br />  <br />                                                                                                                    Page 10 of 15 <br />                                                                                               (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                      http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> which were detected in three independent networks, illus-       likelihood of essentiality and the likelihood of conserva- <br /> trates that these networks have a structure to them which       tion for their gene members within the entire network. <br /> matches known cellular biology. We have also shown that         Furthermore, we show that these networks can be subdi- <br /> when sufficient information is available, these correlation     vided into identifiable modules, which themselves can <br /> modules can provide further information about which of          often contain enough individual information to indicate <br /> their members are more likely to be important for a given       which genes are more likely to be crucial for a particular <br /> process. The ability to assess whether or not a module is       process. We have indicated a means for testing whether <br /> complete enough to be informative using a test like either      this is true or not. Thus, we have illustrated how this proc- <br /> of the ones employed here will be important for validat-        ess can be applied to common microarray data for the <br /> ing the merit of less understood modules in the future.         purpose of predicting both functional assignment as well <br />                                                                 as the ranking of gene significance in a particular process <br /> Our study extends the strong links between the network          with which it is found to be associated. We believe that <br /> properties of a gene within co-expression networks and          these kinds of predictions will be helpful for identifying <br /> the global importance of each gene to the yeast cell, and       gene targets for therapeutics and for directing multi-drug <br /> may have implications for targeted drug therapy in more         therapies. <br /> complex biological systems like cancer. For instance, one <br /> can infer from an analysis of the single gene deletion data     Methods <br /> that single targets for inhibition can be partially predicted   Data sources <br /> from expression data. However, in biological systems an         Yeast microarray data were obtained from the Saccharomy- <br /> optimal effect is frequently achieved by inhibiting multi-      ces Genome Database in ratio format [34]. In each case, <br /> ple specific targets. Can we extrapolate network positions      microarray based gene expression analysis was previously <br /> and relationships to infer effective combinations of tar-       reported to have been performed for the analysis of DNA <br /> geted therapeutics? Yeast is an excellent model for estab-      Damage [18], Cell Cycle [17], and Environmental <br /> lishing the rules for applying network theories to practical    Response [19]. Not all of the data obtained from each <br /> predictions. For instance, will double deletions of high        datasets listed was used in these experiments. Because <br /> connectivity genes within the network which reside in the       missing data is especially confounding for our analysis, <br /> same module have substantially greater negative conse-          arrays where greater than 5% of the gene expression infor- <br /> quences to the yeast than double deletions of low connec-       mation was missing were removed from our analysis. The <br /> tivity genes? Alternatively, will genes which are               exact number of arrays chosen to represent each dataset <br /> connecting distinct modules, and serving as key connec-         was: DNA Damage (n = 51), Cell Cycle (n = 44), and Envi- <br /> tions between integrators prove to be better disruption         ronmental Response (n = 151). Essential genes were deter- <br /> targets? The modeling of such proposed interactions in          mined from the Saccharomyces Genome Deletion Project <br /> yeast is attractive because of the relative tractability of     [35]. Genomic data for conducting the blastp searches <br /> yeast studies and the availability of large scale datasets.     were obtained from the following locations: Neurospora <br /> Technologies like synthetic genetic arrays which automate       genome data (version 7) was obtained from the Broad <br /> double deletion screens or chemical genomic profiling           Institute [36,37]. Drosophila genome data was obtained <br /> provide an expanding capability to survey the effects of        from the Berkeley Drosophila genome project [38], C. ele- <br /> multiple target inhibition or deletion [30-32]. However,        gans genome data was obtained from the wormbase data- <br /> even with these powerful methods, only a small fraction         base [39], while both the human and mouse genomes <br /> of the millions of possible combinations will be assayed.       were obtained from the UCSC genome browser [40]. The <br /> For instance in a substantial recent survey of double dele-     blastp algorithm was run locally. <br /> tions only 0.02% of all possible double deletions were <br /> tested for synthetic lethality [33]. Thus, network based        Weighted Network construction <br /> modeling may be a useful adjunct for the design of double       In gene co-expression networks, each gene corresponds to <br /> <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> experiments or combination inhibition experi-          a node. The neighbors of a node i are the nodes that are <br /> ments to permit a more efficient search for synthetic           connected to the node i. Two genes are connected by an <br /> lethals. An interplay between predictive models and tar-        edge with a weight indicating the connection strength. A <br /> geted large scale inhibition studies may be quite useful for    gene co-expression network can be represented by an <br /> surveying pathway analysis in yeast. In the long term, such     adjacency matrix A = [aij], where aij is the weight of a con- <br /> studies may also be critical in the development of thera-       nection between two nodes i and j. For all of the networks <br /> pies for diseases like cancer.                                  considered in this paper, the connectivity equals the sum <br />                                                                 of connection weights being considered. <br /> Conclusion <br /> In conclusion, our study demonstrates that weighted gene        Our analysis follows a general framework for construction <br /> co-expression networks can be used to predict both the          of gene co-expression networks [16]. To transform the co- <br />  <br />  <br />                                                                                                                    Page 11 of 15 <br />                                                                                               (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                        http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> expression measure (here the absolute value of the Pear-         free property as is highlighted by the Cell Cycle network <br /> son correlation matrix) into measures of pair wise connec-       in this manuscript. <br /> tion strengths, one can make use of an adjacency function. <br /> The choice of the adjacency function determines whether          For each dataset, we first selected the top 4000 most vary- <br /> the resulting network will be weighted (soft thresholding)       ing transcripts based on the standard deviation. The net- <br /> or unweighted (hard thresholding). A widely used adja- <br />                                                                  work adjacency between 2 genes i and j was defined as a <br /> cency function is the signum function which implements <br />                                                                  power of the correlation coefficient between the corre- <br /> &apos;hard&apos; thresholding involving the threshold parameter tau. <br /> Specifically, aij = Ind(|cor(xi, xj)| &gt; &#207;&#8222; where the indicator    sponding gene expression profiles xi and xj, i.e. aij = |cor(xi, <br /> function takes on the value 1 if the condition is satisfied      xj)|&#206;&#178; where the power beta is chosen using the scale free <br /> and 0 otherwise. Hard thresholding using the signum              topology criterion. From this function, the 3000 most- <br /> function leads to intuitive network concepts (e.g., the          connected genes were selected for further detailed net- <br /> node connectivity equals the number of direct neighbors), <br />                                                                  work analysis. Using the 4000 most varying transcripts <br /> but it may lead to a loss of information. For instance, if the <br /> threshold has been set to 0.8, there will be no connection       was not very restrictive since the original data are derived <br /> between two genes if their correlation equals 0.79. To           from yeast arrays and therefore only contained only about <br /> avoid the disadvantages of hard thresholding, Zhang and          6000 genes in total. Many of these genes needed to be <br /> Horvath [16] proposed a &apos;soft&apos; power adjacency function:         removed due to high rates (&gt;40%) of missing data for a <br /> aij = |cor(xi, xj)|&#206;&#178; with the single parameter beta. The power   gene across a data set. By <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, module genes are <br /> adjacency function preserves the continuous nature of the        highly connected with the genes of their module (i.e. <br /> gene co-expression information and, leads to results that        module genes tend have relatively high connectivity). <br /> are highly robust with respect to the choice of beta. <br />                                                                  Thus, for the purpose of module detection, restricting the <br />                                                                  analysis to the most connected genes does not lead to <br /> Zhang and Horvath [16] also proposed a criterion for <br /> choosing the power beta of the adjacency function which          major information loss for the key points of this presenta- <br /> is based on the fact that despite significant variation in       tion. However, there may be applications where genes <br /> their individual constituents and pathways, metabolic            with relatively low connectivity are biologically interest- <br /> networks have been found to display approximate scale            ing so that gene filtering based on connectivity would lead <br /> free topology. The linear regression model fitting index         to information loss. The adjacency matrix was also visual- <br /> R^2 can be used to quantify how well a network satisfies         ized as a graph and the number of topological properties <br /> a scale-free topology. While the scale free model has only       of this graph examined. Important network derived con- <br /> 1 parameter (&#206;&#179;), the truncated exponential model allows <br />                                                                  cepts include: hub (a node with many strong connections <br /> for 2 parameters &#206;&#179; and &#206;&#177;. Empirically, we find that the two <br /> parameters of the truncated exponential model provide            to other nodes), topological overlap (the interconnectiv- <br /> too much flexibility in curve fitting so that the R-squared      ity between two nodes), modules (tightly connected sub- <br /> values are always high [16]. Thus, we focus on the scale         sets of the network), connectivity (the sum of the <br /> free topology model fitting index in this application and <br /> emphasize the major trend that the high connectivity <br />                                                                  adjacencies, ie. ki =   &#226;&#710;&#8216; aiu ) and intramodular connectiv- <br />                                                                                          u&#226;&#8240;&#160;i <br /> genes are few in number, and that most genes have low            ity (the sum of the adjacencies within a module). One <br /> connectivity (Figure 2A&#226;&#8364;&#8220;2C). There is a natural trade-off        potential drawback of soft thresholding is that the net- <br /> between maximizing scale-free topology model fit (R^2) <br />                                                                  work cannot be visualized directly because the number of <br /> and maintaining a high mean number of connections: <br /> parameter values that lead to an R^2 value close to 1 may        relationships to display becomes intractable. A soft adja- <br /> lead to networks with very few connections. These consid-        cency matrix specifies pair wise connection strengths and <br /> erations motivated the scale-free topology criterion. To         all nodes are connected. To visualize the network with <br /> choose the parameters of an adjacency function: Only             Pajek, one needs to threshold the connection strengths. <br /> those parameter values that lead to a network satisfying         The resulting dichotomized adjacency matrix can be con- <br /> scale-free topology at least approximately were considered       sidered as an unweighted network approximation of the <br /> (e.g. signed R^2 &gt; 0.80). When considering the signum            weighted network. Therefore, to make a visual representa- <br /> and power adjacency functions, we find the relationship <br />                                                                  tion, only the strongest correlations (0.9 or greater) were <br /> between R^2 and the adjacency function parameter (tau <br />                                                                  drawn in these renderings. Subsequent to this, networks <br /> or beta) is characterized by a saturation curve. In our <br /> applications, we use the first parameter value where satu-       were then visualized with Pajek [20,41]. <br /> ration is reached as long as it is above 0.8. An R^2 &gt; .80 <br /> results in networks that may only approximate the scale <br />  <br />  <br />                                                                                                                      Page 12 of 15 <br />                                                                                                 (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                                                http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> of Topological Overlap and Modularity                                         by using the standard approach of subtracting it from one <br /> Topological overlap of two nodes (genes) reflects their rel-                             (i.e, the topological overlap based dissimilarity measure is <br /> ative interconnectivity. For a network represented by an <br /> adjacency matrix A = [aij], aij &#226;&#710;&#710; [0,1], a well-known for-                               defined by d&#207;&#8211; <br />                                                                                                      ij = 1 - &#207;&#8211;ij). <br /> mula for defining topological overlap was given by [4], <br /> and it can be extended to weighted networks[16]:                                         In networks involving few nodes, modules can easily be <br />                                                                                          identified by inspecting the network, but for large net- <br />                   lij + aij <br /> &#207;&#8240; ij =                                                (1)                                works involving hundred of nodes, it is useful to generate <br />           min{ki , k j } + 1 &#226;&#710;&#8217; aij                                                       a topological overlap matrix (TOM) plot, which we review <br />                                                                                          in the following. A TOM plot provides a &quot;reduced&quot; view of <br /> where, lij =           &#226;&#710;&#8216; aiu auj         denotes the number of nodes to                  the network that allows one to visualize and identify net- <br />                    u&#226;&#8240;&#160;i, j <br />                                                                                          work modules. The TOM plot is a color-coded depiction <br /> which both i and j are connected, and k is the number of <br />                                                                                          of the values of [ d&#207;&#8211; <br />                                                                                                              ij ], described above. <br /> connections of a node, with ki =                     &#226;&#710;&#8216; aiu and k j =       &#226;&#710;&#8216; a ju . <br />                                                      u&#226;&#8240;&#160;i                   u&#226;&#8240;&#160; j <br />                                                                                          Based on the TOM based dissimilarity matrix we can use <br /> Since            aij        &#226;&#710;&#710;            [0,1],        we         find            that   hierarchical clustering to discriminate one module from <br /> lij =    &#226;&#710;&#8216; aiu auj &#226;&#8240;&#164; min( &#226;&#710;&#8216; aiu , &#226;&#710;&#8216; auj ) = min(&#226;&#710;&#8216; aiu , &#226;&#710;&#8216; a ju ) &#226;&#710;&#8217; aij                   another. We used a dynamic cut-tree algorithm for auto- <br />         u&#226;&#8240;&#160;i, j                  u&#226;&#8240;&#160;i, j     u&#226;&#8240;&#160;i, j           u&#226;&#8240;&#160;i     u&#226;&#8240;&#160; j                 matically and precisely identifying modules in hierarchi- <br /> . Thus, lij + aij &#226;&#8240;&#164; min( &#226;&#710;&#8216; aiu , &#226;&#710;&#8216; a ju ) . Since 1 - aij &#226;&#8240;&#165; 0, we                        cal clustering dendrogram [43]. The algorithm makes use <br />                                    u&#226;&#8240;&#160;i        u&#226;&#8240;&#160; j <br />                                                                                          of the internal structure in a dendrogram and clips <br />                                                                                          branches of the dendrogram. The algorithm is based on an <br /> find that &#207;&#8240;ij is a number between 0 and 1. There are 2 rea-                              adaptive process of cluster decomposition and combina- <br /> sons for adding 1 - aij to the denominator in the topolog-                               tion and the process is iterated until the number of clus- <br /> ical overlap matrix: 1) in this form, the denominator can                                ters becomes stable. No claim is made that our module <br /> never be 0 and 2) for an unweighted network, one can                                     construction method is optimal and future research <br />                                                                                          should compare different method for cluster identifica- <br /> show that &#207;&#8240;ij = 1 only if the node with fewer links satisfies                            tion. Each module represents a group of genes with simi- <br /> two conditions: (a) all of its neighbors are also neighbors                              lar expression profiles across the samples and the <br /> of the other node, i.e. it is connected to all of the neigh-                             expression profile pattern is distinct from those of the <br /> bors of the other node and (b) it is linked to the other                                 other modules. In each case, the modules were then <br /> node.                                                                                    assessed for enrichment in ontology terms based on <br />                                                                                          Fisher&apos;s exact test as described in [25]. These modules and <br /> In contrast, &#207;&#8240;ij = 0 if i and j are unlinked and the two                                 their relative connectivities were then separated out for <br /> nodes don&apos;t have common neighbors. Further, the topo-                                    further analysis. <br /> logical overlap matrix is symmetric, i.e, &#207;&#8240;ij = &#207;&#8240;ji. and its <br /> diagonal elements are set to 0 (i.e. &#207;&#8240;ii = 0). The rationale                             Based on the network modules, we can define the <br /> for considering this similarity measure is that nodes that                               intramodular connectivity (k.in) for each node i (i.e., the <br /> are part of highly integrated modules are expected to have                               number of neighbors in the same module as node i). The <br /> high topological overlap with their neighbors [4].                                       connectivity of node i across the whole network is <br />                                                                                          denoted as k.all. <br /> It is important to note that the <span id='am-241' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of topological <br /> overlap matrix (equation 1) was taken from the methods                                   Blastp <br /> supplement of reference [4] since there is a typo in the pri-                            The translated sequences of the entire yeast genome were <br /> mary text. Different definitions of the topological overlap                              each blasted against the entire translated sequences of the <br /> matrix are possible, but we have found empirically that                                  Human, Mouse, Fly, Neurospora and C. elegans genomes. <br /> many of these possible generalizations lead to very similar                              In each case, the best hit was determined using blastp, and <br /> modules [42].                                                                            the log(e score) scores were averaged across the 5 different <br />                                                                                          comparisons in a manner similar to what was published <br /> Modules are groups of nodes that have high topological                                   by [22]. These average blastp scores were then averaged for <br />                                                                                          each point in (Figure 2H&#226;&#8364;&#8220;2K, 3F&#226;&#8364;&#8220;3H, 4F&#226;&#8364;&#8220;4H, 5F&#226;&#8364;&#8220;5H). <br /> overlap. Module identification is based on the topological <br />                                                                                          This method is not necessarily an optimal measure of <br /> overlap matrix &#226;&#8222;&#166; = [&#207;&#8240;ij defined as (1). To use it in hierar-                             gene conservation, and it is used here simply as an <br /> chical clustering, it is turned into a dissimilarity measure                             approximate measure of gene conservation, that is inde- <br />  <br />  <br />                                                                                                                                             Page 13 of 15 <br />                                                                                                                        (page number not for citation purposes) <br />  BMC Genomics 2006, 7:40                                                                    http://www.biomedcentral.com/1471-2164/7/40 <br />  <br />  <br />  <br /> Table 2: Summary of tests for network predictive power <br />  <br />  Module name                      DNA Dmg                                     Env Stress                                 Cell Cycle <br />  <br />                          Ess Test            Cons Test            Ess Test             Cons Test              Ess Test              Cons Test <br />  <br />  Entire Network         0.23 1.0E-39        -0.15 1.3E-19        0.21 1.0E-39         -0.22 1.4E-39           0.2 1.0E-39          -0.18 4.0E-29 <br />  rRNA proc               0.2 6.3E-5           -0.1 0.049         0.28 1.0E-39         -0.091 0.0066          0.22 0.00011           -0.23 8.3E-5 <br />  Prot Synth             -0.12 0.014           -0.07 0.15         -0.16 0.0035          -0.042 0.46             0.019 0.7           -0.099 0.055 <br />  Ubiq                   0.41 1.8E-10         -0.28 2.9E-5         0.49 0.0022           -0.16 0.34            -0.044 0.44           -0.032 0.59 <br />  <br />  *Spearman correlation and associated p-values for each network and module tested <br />  <br />  <br /> pendent of annotation. Future research should study the                  ing these weighted networks, developing network meas- <br /> advantages and disadvantages of using different measures                 ures, and providing crucial statistical guidance for this <br /> for gene conservation in network analysis.                               project. ZF helped with the statistical analysis. PM and <br />                                                                          SFN both contributed key ideas and suggestions as the <br /> Analysis of module overlap                                               project progressed. All authors participated in the devel- <br /> We used a hyper-geometric p-value to evaluate the signif-                opment of the ideas presented in this manuscript. All <br /> icance of the overlap of the module genes across the 3 data              authors read and approved the final manuscript. <br /> sets. Suppose N1, N2 N3 out of N genes are in rRNA <br /> processing module of the DNA Damage data, the Environ-                   Acknowledgements <br /> mental Response data, and the Cell Cycle data, respec-                   The authors extend their thanks to the SGD for providing data for this anal- <br /> tively. We then denote by &apos;overlap&apos; the number of genes in               ysis. Also, special mention should be made of Allen Day, Ally Perlina and <br /> the rRNA module that overlap in the 3 data sets.                         Andy Yip for their input during the analysis of this project. The work was <br />                                                                          supported by a grant from the NCI (U01 CA881273) as well as the Henry <br />                                                                          Singleton Brain Tumor Program, the Elliott Family Foundation in memory <br /> To arrive at an upper limit of the p-value, we first com-                of Neal Elliott, the Ziering Family Foundation in memory of Sigi Ziering, <br /> puted pair wise p-values as follows. Denote as p12 the p-                Accelerate Brain Cancer Cure, and Art of the Brain. MRJC was supported <br /> value that by randomly selecting N2 genes from a pool of                 by the UCLA Tumor Cell Biology Training Program, funded by NIH/ <br /> N genes with N1 module 1 genes, &#226;&#8240;&#165; overlap selected genes                 National Cancer Institute 5T32CA09056. This work was also supported by <br /> are in the rRNA-processing module in both data sets. This                the UCLA DNA Microarray Facility and the NINDS/NIMH Microarray <br /> p-value can easily be computed using the hypergeometric                  Consortium. <br /> distribution (dhyper function in R). N1 and N2 were then <br /> swapped to obtain a second p-value, which we denoted as          <br /> p21. The maximum of those two values can be used to                   <br /> assess the significance of overlap between data sets 1 and             <br /> 2. Similarly, we computed a p-value of overlap between             <br /> data sets 1 and 3, and data sets 2 and 3. The minimum of           <br /> the 3 resulting p-values provided an upper limit for assess-         <br /> ing the significance of overlap between all 3 data sets, i.e.                 <br /> the final p-value was computed as p = min(max(p12,p21),                       <br /> max(p13,p31), max(p23,p32))                                          <br />  <br />  <br /> </body></html>