<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Relationship insertion/deletion (indel) frequency of <br /> proteins essentiality <br /> Simon K Chan1,2, Michael Hsing2, Fereydoun Hormozdiari3 and <br /> Artem Cherkasov*4 <br />  <br /> Address: 1CIHR/MSFHR Strategic Training Program Bioinformatics, Canada&apos;s Michael Smith Genome Sciences Centre, 570 West 7th Ave &#226;&#8364;&#8220; Suite <br /> 100, Vancouver, BC, V5Z 4S6, Canada, 2Bioinformatics Graduate Program, University British Columbia, 570 West 7th Ave &#226;&#8364;&#8220; Suite 100, <br /> Vancouver, BC, V5Z 4S6, Canada, 3School Computing Science, Simon Fraser University, 8888 University Drive, Burnaby, BC, V5A 1S6, Canada <br /> and 4Division Infectious Diseases, Faculty Medicine, University British Columbia, 2733 Heather Street, Vancouver, BC, V5Z 3J5, Canada <br /> Email: Simon K Chan - sichan@bcgsc.ca; Michael Hsing - mhsing@interchange.ubc.ca; Fereydoun Hormozdiari - fhormozd@cs.sfu.ca; <br /> Artem Cherkasov* - artc@interchange.ubc.ca <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 28 June 2007                                                        Received: 6 November 2006 <br />                                                                                Accepted: 28 June 2007 <br /> BMC Bioinformatics 2007, 8:227   doi:10.1186/1471-2105-8-227 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2105/8/227 <br /> &#194;&#169; 2007 Chan et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: In previous study, demonstrated essential proteins pathogenic <br />                  organisms contained sizable insertions/deletions (indels) aligned human proteins high <br />                  sequence similarity. Such indels provide sufficient spatial differences pathogenic <br />                  protein human proteins allow selective targeting. In example, indel difference was <br />                  targeted large scale in-silico screening. This resulted selective antibodies small <br />                  compounds capable binding deletion-bearing essential pathogen protein <br />                  cross-reactivity highly similar human protein. The objective current study <br />                  investigate indels frequently essential non-essential <br />                  proteins. <br />                  Results: We investigated species, Bacillus subtilis, Escherichia coli, Saccharomyces <br />                  cerevisiae, high-quality protein essentiality data available. Using data, we <br />                  demonstrated t-test calculations mean indel frequencies essential proteins were <br />                  greater non-essential proteins proteomes. The abundance indels both <br />                  types proteins shown accurately modeled Weibull distribution. However, <br />                  Receiver Operator Characteristic (ROC) curves showed indel frequencies be <br />                  used marker accurately discriminate essential non-essential proteins the <br />                  proteomes. Finally, analyzed protein interaction data available S. cerevisiae and <br />                  observed indel-bearing proteins involved interactions greater <br />                  betweenness values Protein Interaction Networks (PINs). <br />                  Conclusion: Overall, findings demonstrated indels randomly distributed across <br />                  studied proteomes likely occur essential proteins that <br />                  highly connected, indicating possible role sequence insertions deletions the <br />                  regulation modification protein-protein interactions. Such observations provide new <br />                  insights indel-based drug design using bioinformatics cheminformatics tools. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 13 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                              http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br />  <br /> Background                                                      hypothesis essential proteins likely contain <br /> Essential genes encode products required       indels following observations: firstly, <br /> viability organism. There major reasons       protein domain profiles characterized databases such <br /> there considerable determining set        Pfam [11] showed protein sequences same <br /> essential genes organism. Firstly, provide      protein interaction domain contained large of <br /> insights basic requirements needed sustain        residue variations multiple species, implied <br /> living cell. For example, sequencing parasitic       single <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>point mutation</span> protein did a <br /> bacterium Mycoplasma genitalium [1] subsequent          large impact function protein interaction <br /> studies determine essential genes [2,3] pro-        domains. Secondly, essential proteins undergo stronger <br /> vided in-depth understanding constitutes         selective pressure accumulate point mutations at <br /> a &apos;minimum genome.&apos; Secondly, essential proteins             slower rate non-essential proteins [12,13]. There- <br /> pathogens potentially excellent drug targets [4,5],      fore, taking considerations together, propose <br /> as interfering proper functioning          formation indels method which <br /> likely interfere important pathway patho-        proteins, especially essential, use acquire <br /> gen, reducing threat host. However, target-     new interaction sites and/or modify existing ones, and <br /> ing essential proteins pathogen major         interaction partners. For example, well <br /> drawback: essential proteins conserved         known PINs tend scale-free [14,15], which <br /> species, drug targets essential protein     majority proteins interaction network <br /> pathogen cross-react similar host protein       fewer interactions highly con- <br /> [6]. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> combat problem, laboratory recently        nected &apos;hub&apos; proteins. Due greater number inter- <br /> developed strategy target insertions/deletions (indels)    actions participate in, hubs tend essential <br /> that occur proteins pathogen             proteins. These hubs gain interactions network <br /> human host. For example, Leishmania donovani proto-        gene encoding interacting partners dupli- <br /> zoan parasite infects inactivates macrophages      cates. This process known preferential attachment <br /> of human host [7]. The main structural difference           [14,15]. If indel occur interaction site of <br /> between essential elongation factor (EF-1 &#206;&#177;) protein        duplicate copy gene, resulting protein <br /> of L. donovani human host 12 amino         reflect change change number of <br /> acid deletion occurs L. donovani sequence [7].      interaction partners. <br /> The 12 amino acid sequence corresponds hair pin <br /> loop present human protein, absent        To knowledge, body work presented the <br /> the L. donovani protein. Using computational chemistry          investigate possible relationship indel <br /> and molecular docking, able develop inhibi-          frequency essentiality. We chose species that <br /> tors directly recognized exposed region L.      complete global <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>knockout</span> data: Bacillus subtilis, <br /> donovani protein cross-reactivity highly     Escherichia coli, Saccharomyces cerevisiae. Specifically, <br /> similar human host protein [6,8,9]. Interestingly,         purpose study determine 1) the <br /> deletion potentially allow EF-1 &#206;&#177; L. donovani       mean indel frequency essential proteins differed from <br /> gain interaction, relative human EF-1 &#206;&#177;, interact     non-essential proteins 2) Weibull dis- <br /> with human tyrosine phosphatase, leads inacti-         tribution accurately model indel abundances in <br /> vation host macrophage [7]. With past stud-        types proteins 3) indel frequency a <br /> ies, showed indels offer structural          protein used marker predict or <br /> differences target specific pathogen essential proteins   given protein essential 4) proteins <br /> well allow acquire and/or modify protein-        indels participated interactions those <br /> protein interactions involved in.                 not. We defined indels insertions deletions <br />                                                                 proteins high sequence similarity (at least <br /> Recently, performed large scale survey potentially     50%), regardless evolutionary relationship with <br /> targetable indels aligning complete proteomes         (i.e. just orthologs species). This <br /> bacterial protozoan pathogens complete               work potentially locate similar situations L. <br /> human proteome [10]. Our results showed sizable            donovani case described explore meth- <br /> indels approximately 5&#226;&#8364;&#8220;10% bacterial           odology targeting indels specific pathogen proteins <br /> proteins 25% protozoan proteins          cross-reactivity human host proteins. <br /> respect human proteins. A large number pro- <br /> teins indels identified essential   Results discussion <br /> respective pathogens. Therefore, current study,      Query subject species analyzed <br /> set determine frequency indels essential    To test indel frequency protein related <br /> proteins differed non-essential proteins. Our      essentiality, obtained protein sequences FASTA <br />  <br />  <br />                                                                                                                     Page 2 13 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> format NCBI RefSeq [16]for B. subtilis, E. coli, S.      Is significant difference indel <br /> cerevisiae. These organisms chosen             frequencies essential non-essential proteins? <br /> genomes sequenced global knockout phe-              To evaluate differences mean <br /> notype data available [17-19]. We referred           indel frequencies essential non-essential proteins <br /> three species &apos;query species,&apos; respective pro-     statistically significant, calculated fre- <br /> teins queries sequence alignments (Table          quencies insertions deletions given minimum <br /> 1). We referred proteins query species         <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> (one amino acids) query species <br /> &apos;query proteins.&apos; Essentiality data available       (see Methods). Next, calculated mean insertion <br /> organisms B. subtilis, E. coli, S. cerevisiae, how-   deletion frequencies essential non- <br /> ever, data produced complete gene dele-         essential proteins query species. Figure 2 contains <br /> tion, E. coli S. cerevisiae, insertion       plots mean insertion deletion frequencies <br /> marker, B. subtilis, transposon mutagenesis          minimum insertion deletion lengths for <br /> (Mycoplasma genitalium [2,3], Haemophilus influenzae [20],        query species. As figure illustrates, mean <br /> Escherichia coli (strain MG1655) [21]) anti-sense RNA          frequencies proteins query species <br /> (Staphylococcus aureus (strains RN450 RN4220) [22]).          decrease minimum indel lengths increase, suggest- <br /> Transposon mutagenesis miss essential genes tol-         ing short indels likely occur long <br /> erate transposon insertions produce false nega-        indels. Next, performed t-tests examine null <br /> tives non-polar insertions. Inhibition anti-sense       hypothesis mean indel frequencies essential <br /> RNA &apos;knock down&apos; knockout gene            non-essential proteins equal. We observed that <br /> and result complete removal tran-           absolute differences mean indel fre- <br /> script target gene. Also, technique limited     quencies small, differences statistically sig- <br /> genes adequate expression anti-sense             nificant assessed t-test calculation (P &lt; 0.05). As <br /> RNA obtained [17,18]. With considerations            seen figure, essential proteins B. subtilis, E. <br /> in mind, performed analyses B. subtilis, E. coli,     coli, S. cerevisiae significantly different insertion <br /> and S. cerevisiae essentiality data        deletion frequencies non-essential coun- <br /> organisms potentially reliable.                         terparts. All significant t-test values positive the <br />                                                                   query species, suggested organ- <br /> We downloaded protein sequences, FASTA format,            isms, essential proteins greater frequency indels <br /> for 22 bacterial 15 eukaryote species fully              non-essential proteins. While insertions and <br /> sequenced genomes. We referred species &apos;sub-          deletions occurred significantly essential <br /> ject species,&apos; respective proteins sub-      proteins non-essential proteins E. coli S. <br /> jects sequence alignments (Additional file 1). We          cerevisiae, deletions minimum length to <br /> referred proteins subject species &apos;subject     amino acids produced significant results B. sub- <br /> proteins.&apos; All together, 14,214 query proteins (8342 bacte-       tilis. <br /> rial 5872 eukaryote) 336,086 subject proteins <br /> (53,454 bacterial 282,632 eukaryote) analyzed.           It interesting note long indels S. cerevi- <br />                                                                   siae significant, greatest t-test value occurred <br /> The comparison indel frequencies essential          indel length defined amino <br /> non-essential proteins performed determine          acids. A large t-test value suggested differences <br /> frequencies differed statistically significant manner.       mean indel frequencies essential non- <br /> We aligned NCBI RefSeq proteins B. subtilis          essential proteins likely chance alone. Fur- <br /> E. coli proteins 22 bacteria subject species,      thermore, indels exactly amino acid long were <br /> and S. cerevisiae proteins 15 eukaryote          randomly distributed essential non-essential <br /> subject species BLASTP. A gap opened query            proteins, expect t-test value a <br /> protein reported deletion query pro-         longer minimum indel length produce greatest <br /> tein insertion subject protein. Similarly,      t-test value. However, case expla- <br /> gap opened subject protein reported          nation trend essential proteins con- <br /> deletion subject protein insertion         tained higher frequency indels exactly amino <br /> query protein. To maintain consistent naming scheme,            acid length. To test possible explanation, re-ran <br /> we reported gaps respect query protein (Figure        BLASTP processing scripts S. cerevisiae, this <br /> 1a). Figure 1b summarizes steps performed               time checking indels length exactly twenty <br /> Additional file 2 shows summary number                amino acids. The results new set (data not <br /> indels proteins high sequence similarity          shown) showed significance one <br /> species-species comparison.                                       amino acid indel length, confirmed suspi- <br />                                                                   cions. <br />  <br />  <br />                                                                                                                       Page 3 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                    http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> Table 1: Selected query species. The query species completed genome projects complete global knockout data <br /> available <br />  <br />       Query Species               Domain      Taxonomy ID   Number Proteins NCBI         Essential Genes be <br />                                                                        RefSeq                    mapped NCBI RefSeq ID: <br />  <br />  Bacillus subtilis (strain 168)    Bacteria     224308                  4105                                  271/271 <br />  Escherichia coli (strain K12)     Bacteria      83333                  4237                                  299/303 <br />   Saccharomyces cerevisiae        Eukaryote       4932                  5872                                 1050/1105 <br />  <br />  <br />  <br />  <br /> While initial t-test results supported predictions          and/or annotation errors protein sequences may <br /> that essential proteins query species              resulted &quot;pseudo-indels&quot; BLASTP align- <br /> have indels respective non-essential pro-             ments. To observe proteins complete set <br /> teins, reasoned frequency indels produced              subject proteins 22 bacteria 15 eukaryotes <br /> is partially dependent specific subject spe-          impacted initial t-test results, repeated <br /> cies chosen. To observe choice subject species             analyses performed BLASTP S. cerevisiae against <br /> may impacted results, repeated t-test anal-           smaller set randomly chosen eukaryote subject <br /> ysis smaller set 14 randomly chosen subject spe-            species, time fully curated reviewed <br /> cies. After performing BLASTP B. subtilis E. coli              NCBI RefSeq proteins included. We focused on <br /> against proteins sequenced bacterial species              S. cerevisiae respective proteins NCBI <br /> and S. cerevisiae proteins sequenced              RefSeq fully curated reviewed, not <br /> eukaryote species (Additional file 3), observed similar            case proteins query spe- <br /> trends essential proteins significantly greater           cies. If resulting trends smaller set subject <br /> indel frequencies non-essential proteins (P &lt; 0.05)              species varied greatly observed with <br /> (Additional file 4). For example, complete set              complete set subject species (15 eukaryotes), it <br /> bacterial subject proteins (22 bacteria species), E. coli             likely results produced com- <br /> insertions minimum length 10 20 amino acids                     plete set subject proteins caused pseudo- <br /> occurred frequently essential proteins non-              indels created alignments predicted non- <br /> essential proteins, smaller set bacterial           curated NCBI RefSeq proteins. However, the <br /> subject proteins, insertions seven               case trends seen highly curated proteins <br /> amino acids occurred frequently essential pro-                similar observed complete <br /> teins. Similarly, complete set bacterial subject            subject species set (Additional file 5). Therefore, con- <br /> proteins, B. subtilis deletions minimum length            cluded unlikely observed trend, in <br /> twenty amino acids occurred frequently essential              indel frequency essential proteins greater <br /> proteins, smaller set, trend extended           non-essential proteins, merely caused by <br /> to seven amino acids. These results showed             sequencing and/or annotation errors. While per- <br /> while choice subject species did alter specific            formed check test results, wish to <br /> indel lengths produced significant results, general,          remind reader sequences NCBI RefSeq repre- <br /> the trends consistent predictions. The             sent nearly non-redundant collection sequences and <br /> exception shorter insertions B. subtilis. We               described &apos;summary&apos; currently available <br /> observed insertions minimum length three, four,               information sequence [16]. <br /> and frequently non-essential pro- <br /> teins, indicated negative t-test values. However,           Cumulative insertion deletion frequencies and <br /> the longer deletions B. subtilis, discussed, followed           approximation Weibull distribution <br /> the predicted trend. With specific result mind,            To investigate abundance indels essential and <br /> now speculate longer indels, say                 non-essential proteins modeled consistent <br /> length greater equaled seven amino acids,              statistical distributions, calculated cumulative dis- <br /> more likely essential proteins.                        tribution functions (CDF) minimum lengths of <br />                                                                       insertions deletions essential non-essential <br /> Another issue impacted initial t-test               proteins query species (Figure 3). As seen in <br /> results quality protein sequences used.             Figure 3, dependences abundance of <br /> A portion proteins obtained NCBI RefSeq                indels essential non-essential proteins and <br /> resulted computational predictions and/or               minimum indel lengths formed typical exponent-like dis- <br /> undergone manual curation. Therefore, sequencing                 tributions. In previous work [10], demonstrated <br />  <br />  <br />                                                                                                                           Page 4 13 <br />                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                            http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br />  <br />                   A) <br />  <br />  <br />  <br />  <br />                   B) <br />                        Obtain RefSeq proteins three <br />                                                                            Obtain RefSeq proteins 22 bacterial <br />                        query species global knock out <br />                                                                            15 eukaryote species <br />                        data: B. subtilis, E. coli, S. cerevisiae <br />  <br />  <br />  <br />  <br />                                                Align B. subtilis &amp; E. coli proteins proteins <br />                                                22 bacterial species BLASTP. <br />                                                Repeat S. cerevisiae eukaryote species <br />  <br />  <br />  <br />  <br />                                                Calculate insertion deletion frequencies <br />                                                B. subtilis, E. coli, S. cerevisiae <br />  <br />  <br />  <br />  <br />                                                      Calculate mean insertion and <br />                                                      deletion frequencies essential <br />                                                      non-essential proteins each <br />                                                      species <br />  <br />  <br />  <br />  <br /> Figure alignment <br /> Sample    1          pipeline <br /> Sample alignment pipeline. A) Sample Alignment: Gaps reported insertions/deletions respect the <br /> query sequence. There seven insertions (red) deletions (blue) sample alignment. B) Pipeline: A summary <br /> of steps taken calculate mean insertion deletion frequencies essential non-essential proteins B. subtilis, <br /> E. coli, S. cerevisiae. <br />  <br />  <br />  <br />  <br />                                                                                                                                   Page 5 13 <br />                                                                                                             (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                                                                                       http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br />  <br />                                                                        S. cerevisiae                                                                            S. cerevisiae <br />                                                      6.735 <br />  <br />  <br />  <br />  <br />                                                                                                                                             6.164 <br />                                                4.5                                                                                     3 <br />                     Mean Insertion Frequency <br />  <br />  <br />  <br />  <br />                                                                                                             Mean Deletion Frequency <br />                                                 4 <br />                                                                                                                                       2.5 <br />                                                3.5 <br />                                                              6.317 <br />  <br />  <br />  <br />  <br />                                                                                                                                                     4.952 <br />                                                 3                                                                                      2 <br />                                                                                      6.008 <br />  <br />  <br />  <br />                                                2.5 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       4.530 <br />                                                                                                                                       1.5 <br />                                                                                  5.539 <br />  <br />  <br />  <br />  <br />                                                 2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   3.773 <br />                                                                               5.294 <br />                                                                             4.805 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                2.679 <br />                                                1.5                                                                                     1 <br />                                                                            4.724 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              2.417 <br />                                                                           4.410 <br />                                                                          4.111 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             1.929 <br />                                                                         3.577 <br />                                                                        3.605 <br />                                                 1 <br />                                                                        3.605 <br />                                                                       3.502 <br />                                                                       3.123 <br />                                                                       3.035 <br />                                                                      3.003 <br />                                                                      2.760 <br />                                                                      2.542 <br />                                                                      2.572 <br />                                                                      2.583 <br />                                                                                                                                       0.5 <br />                                                0.5 <br />  <br />                                                 0                                                                                      0 <br />                                                        1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                     1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                                      Minimum Insertion Length (aa)                                                           Minimum Deletion Length (aa) <br />  <br />  <br />                                                                            E. coli                                                                                      E. coli <br />                                                 2                                                                                      2 <br />                     Mean Insertion Frequency <br />  <br />  <br />  <br />  <br />                                                                                                             Mean Deletion Frequency <br />  <br />                                                1.8                                                                                    1.8 <br />                                                1.6                                                                                    1.6 <br />                                                1.4                                                                                    1.4 <br />                                                1.2                                                                                    1.2 <br />                                                 1                                                                                      1 <br />                                                0.8                                                                                    0.8 <br />                                                0.6                                                                                    0.6 <br />                                                0.4 <br />                                                                                   2.076 <br />                                                                                   2.265 <br />                                                                                   2.339 <br />                                                                                  2.146 <br />  <br />  <br />  <br />  <br />                                                                                                                                       0.4 <br />                                                                                  2.141 <br />                                                                                  2.129 <br />                                                                                  2.127 <br />                                                                                  2.180 <br />                                                                                  2.399 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                1.818 <br />                                                                                  2.116 <br />                                                                                  2.161 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                1.702 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           1.936 <br />                                                                                                                                                                                           1.934 <br />                                                                                                                                                                                           1.816 <br />                                                0.2                                                                                    0.2 <br />                                                 0                                                                                      0 <br />                                                        1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                     1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                                      Minimum Insertion Length (aa)                                                           Minimum Deletion Length (aa) <br />  <br />  <br />                                                                          B. subtilis                                                                               B. subtilis <br />                                                 2                                                                                      2 <br />                     Mean Insertion Frequency <br />  <br />  <br />  <br />  <br />                                                                                                             Mean Deletion Frequency <br />  <br />  <br />  <br />  <br />                                                1.8                                                                                    1.8 <br />                                                1.6                                                                                    1.6 <br />                                                1.4                                                                                    1.4 <br />                                                1.2                                                                                    1.2 <br />                                                 1                                                                                      1 <br />                                                0.8                                                                                    0.8 <br />                                                0.6                                                                                    0.6 <br />                                                                                                                                                                                 1.695 <br />                                                                                                                                                                                 1.856 <br />  <br />  <br />  <br />  <br />                                                0.4 <br />                                                                                                                                                                                1.943 <br />  <br />  <br />  <br />  <br />                                                                                                                                       0.4 <br />                                                                                                                                                                                2.101 <br />                                                                                                                                                                                2.204 <br />                                                                                                                                                                               2.417 <br />                                                                                                                                                                               2.406 <br />                                                                                                                                                                               2.551 <br />                                                                                                                                                                               2.641 <br />                                                                                                                                                                               2.347 <br />                                                                                                                                                                               2.641 <br />                                                                                                                                                                               2.774 <br />                                                                                                                                                                               2.851 <br />  <br />  <br />  <br />  <br />                                                0.2                                                                                    0.2 <br />                                                 0                                                                                      0 <br />                                                        1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                     1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                                      Minimum Insertion Length (aa)                                                           Minimum Deletion Length (aa) <br />  <br />  <br />  <br />  <br /> Figure <br /> Mean    insertion <br />           2       deletion frequencies essential non-essential proteins plotted minimum indel length <br /> Mean insertion deletion frequencies essential non-essential proteins plotted minimum indel <br /> length. Mean insertion deletion frequencies calculated essential non-essential query proteins aligned pro- <br /> teins 22 bacteria 15 eukaryote species. The t-test statistic shown minimum indel lengths found <br /> significantly essential (blue bars) non-essential (purple bars) proteins. Significance set P &lt; 0.05. Note <br /> that difference observed insertions B. subtilis proteins. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                              Page 6 13 <br />                                                                                                                                                                                        (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> that distribution indels varying length      values insertions deletions, suggesting that <br /> proteins studied accurately described             indels occur expand readily non-essential <br /> Weibull distribution:                                             proteins. This observation appeared odds our <br />                                                                   earlier observations mean indel frequency essen- <br />             SDF(x) = exp{-(x/&#206;&#177;)&#206;&#178;}, x &#226;&#8240;&#165; 0, &#206;&#178; &gt; 0                   tial non-essential proteins. We wondered it <br />                                                                   possible non-essential proteins acquire <br /> where SDF(x) <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>survival</span> distribution function, &#206;&#177;        expand indels slightly faster rate yet, in <br /> scaling factor, &#206;&#178; shape parameter reflect       general, observe indels essential proteins. This <br /> the evolutionary rates occurrence/expansion            observation explained differences the <br /> indels proteomes examined. The Weibull distribu-           evolutionary age essential non-essential genes. A <br /> tion statistical function defined extreme value       recent study fungal species [26], Schizosaccharo- <br /> theory used reliability engineering model         myces pombe Saccharomyces cerevisiae, showed the <br /> material strength durability electronic                ancient gene was, likely be <br /> mechanical components [23]. The Weibull distribution              essential. Thus, essential genes time <br /> utilizes time-to-failure measure assess reliability      accumulate expand indels. <br /> of predict stability. A typical time-to- <br /> failure experiment involves applying disruptive stress       Can indel frequencies used discriminate an <br /> a sample objects representative population. The         essential non-essential protein? <br /> time taken object break (i.e. fail) recorded.   While t-test statistic assesses differ- <br /> The resulting values used determine            ence means quantifiable trait popu- <br /> objects population follow Weibull distribution.          lations significant, does consideration <br /> For example, recent study [24] characterized strength       actual magnitude difference. Even mean <br /> of ceramic materials applying mechanical stresses        indel frequency essential proteins statistically dif- <br /> of 70 &#226;&#8364;&#8220; 400 MPa/s determine characteristics break-          ferent non-essential proteins, a <br /> ing. Similarly, formation expansion indels          large overlap distributions, it <br /> the proteome organism place &apos;disruptive          difficult predict protein was <br /> stress&apos; (evolutionary pressure). An indel &apos;breaks&apos; &apos;fails&apos;     essential based merely indel frequency. To <br /> when lost. Because previous Weibull analyses            determine indel frequencies used marker <br /> only considered indels proteins regardless          differentiate essential non-essential pro- <br /> their essentiality [10], examined statistical      teins, Receiver Operating Characteristic (ROC) curves <br /> function accurately abundance               utilized. The Area Under ROC curve (AUROC) <br /> indels essential non-essential proteins separately.        used assessment accuracy predic- <br /> For query species double logarithmic transforma-         tions. An AUROC 1.0 implies predictions were <br /> tion SDF(x), represented CDF, calculated         correct, suggesting essential proteins com- <br /> and plotted:                                                      pletely separated non-essential proteins based on <br />                                                                   indel frequency threshold. An AUROC 0.50 sug- <br />            log(-log(SDF(x)) = &#206;&#178;log(x) - &#206;&#178;log(&#206;&#177;)                   gests using indel frequency predict essentiality has <br />                                                                   50% sensitivity specificity, useful test. <br /> If abundance indels query species       Finally, AUROC 0.50 implies the <br /> be accurately described distribution,            opposite trend, non-essential proteins a <br /> resulting plots linear. We observed            higher frequency indels essential proteins, is <br /> Weibull distribution accurately model depend-           observed. <br /> ence length indels abundance in <br /> the essential non-essential proteins query spe-        We calculated AUROCs query species. Sim- <br /> cies, indicated high r2 values (Figure 4). The &#206;&#178;        ilar t-tests, query species compared <br /> parameter represented slopes             species domain. The AUROC <br /> graphs Figure 4 values suggested observa-          results query species moderate S. cer- <br /> tions. Firstly, described previously [25], &#206;&#178; value   evisiae query species produce AUROCs <br /> than indicates reliable growth sys-      0.57 0.59, B. subtilis E. coli AUROC <br /> tem rate failure decreasing. In case,       values ranged 0.46 0.56 (data shown). These <br /> results indicated indels retained evolu-       weak trends unexpected, reasoning <br /> tionary time, suggesting functional importance. Sec-         allowed non-essential proteins use indels way <br /> ondly, differences &#206;&#178; values              acquire and/or modify protein-protein interactions. <br /> essential non-essential proteins small, non- <br /> essential proteins query species greater &#206;&#178; <br />  <br />  <br />                                                                                                                       Page 7 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                                                           http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br />  <br />                                                 S. cerevisiae                                                                         S. cerevisiae <br />                                      0.35                                                                                  0.35 <br />  <br />  <br />  <br />  <br />                                                                                                  Indel Abundance Non- <br />                                       0.3 <br />                Indel Abundance in <br />  <br />  <br />  <br />  <br />                                                                                                                             0.3 <br />                 Essential Proteins <br />  <br />  <br />  <br />  <br />                                                                                                     essential Proteins <br />                                      0.25                                                                                  0.25 <br />  <br />                                       0.2                                                                                   0.2 <br />  <br />                                      0.15                                                                                  0.15 <br />  <br />                                       0.1                                                                                   0.1 <br />  <br />                                      0.05                                                                                  0.05 <br />  <br />                                        0                                                                                     0 <br />                                             1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                    1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                  Minimum Indel Length (aa)                                                             Minimum Indel Length (aa) <br />  <br />  <br />                                                       E. coli                                                                               E. coli <br />                                      0.35                                                                                   0.4 <br />                                                                                                  Indel Abundance Non- <br />  <br />  <br />  <br />                                       0.3                                                                                  0.35 <br />                Indel Abundance in <br />                 Essential Proteins <br />  <br />  <br />  <br />  <br />                                                                                                     essential Proteins <br />  <br />  <br />  <br />  <br />                                      0.25                                                                                   0.3 <br />  <br />                                                                                                                            0.25 <br />                                       0.2 <br />                                                                                                                             0.2 <br />                                      0.15 <br />                                                                                                                            0.15 <br />                                       0.1 <br />                                                                                                                             0.1 <br />                                      0.05 <br />                                                                                                                            0.05 <br />  <br />                                        0                                                                                     0 <br />                                             1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                    1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                Minimum Insertion Length (aa)                                                         Minimum Deletion Length (aa) <br />  <br />  <br />                                                   B. subtilis                                                                           B. subtilis <br />                                       0.4                                                                                  0.45 <br />                                                                                                  Indel Abundance Non- <br />  <br />  <br />  <br />  <br />                                      0.35                                                                                   0.4 <br />                Indel Abundance in <br />                 Essential Proteins <br />  <br />  <br />  <br />  <br />                                                                                                     essential Proteins <br />  <br />  <br />  <br />  <br />                                       0.3                                                                                  0.35 <br />  <br />                                                                                                                             0.3 <br />                                      0.25 <br />                                                                                                                            0.25 <br />                                       0.2 <br />                                                                                                                             0.2 <br />                                      0.15 <br />                                                                                                                            0.15 <br />                                       0.1 <br />                                                                                                                             0.1 <br />                                      0.05                                                                                  0.05 <br />                                        0                                                                                     0 <br />                                             1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20                                    1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br />  <br />                                                Minimum Insertion Length (aa)                                                         Minimum Deletion Length (aa) <br />  <br />  <br />  <br />  <br /> Figure 3 essential non-essential proteins indels plotted minimum indel length <br /> Proportion <br /> Proportion essential non-essential proteins indels plotted minimum indel length. Insertions <br /> are represented blue bars deletions represented purple bars. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                         Page 8 13 <br />                                                                                                                                                                   (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                                                                http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br />  <br />                                         S. cerevisiae Essential Proteins                                                         S. cerevisiae Non-essential <br />                                                                                                                                            Proteins <br />                                       0.5 <br />  <br />                                       0.4                                                                                 0.5 <br />                                                         y = 0.4654x - 0.2522 <br />                                                               2                                                           0.4 <br />                                                                                                                                                   y = 0.514x - 0.2825 <br />                                       0.3                    R = 0.9997 <br />                   log(-log(CDF(x))) <br />  <br />  <br />  <br />  <br />                                                                                                                                                         2 <br />                                                                                                                                                       R = 0.9902 <br />  <br />  <br />  <br />  <br />                                                                                                       log(-log(CDF(x))) <br />                                       0.2                                                                                 0.3 <br />  <br />                                                                                                                           0.2 <br />                                       0.1 <br />                                                                                                                           0.1 <br />                                         0 <br />                                              0               0.5                 1              1.5                         0 <br />                                       -0.1                                                                                       0                0.5                 1               1.5 <br />                                                                          y = 0.5211x - 0.287                              -0.1 <br />                                       -0.2                                     2 <br />                                                                              R = 0.9993                                   -0.2                                    y = 0.5315x - 0.2937 <br />                                                                                                                                                                         2 <br />                                       -0.3                                                                                                                             R = 0.9992 <br />                                                                                                                           -0.3 <br />                                       -0.4                                                                                -0.4 <br />  <br />                                                  log(x), x = minimum indel length (aa)                                               log(x), x = minimum indel length (aa) <br />  <br />  <br />                                                  E. coli Essential Proteins                                                  E. coli Non-essential Proteins <br />                                       0.5                                                                                 0.6 <br />  <br />                                       0.4                                                                                 0.5 <br />                                                        y = 0.4896x - 0.2661                                                             y = 0.6367x - 0.3534 <br />                                       0.3                   R2 = 0.9833                                                   0.4                 2 <br />                                                                                                                                              R = 0.9979 <br />                   log(-log(CDF(x))) <br />  <br />  <br />  <br />  <br />                                                                                                       log(-log(CDF(x))) <br />  <br />  <br />  <br />  <br />                                                                                                                           0.3 <br />                                       0.2 <br />                                                                                                                           0.2 <br />                                       0.1 <br />                                                                                                                           0.1 <br />                                         0                                                                                   0 <br />                                              0               0.5                 1              1.5                              0                0.5                 1               1.5 <br />                                       -0.1                                                                                -0.1 <br />  <br />                                                                        y = 0.5334x - 0.2944                               -0.2                                 y = 0.6041x - 0.3347 <br />                                       -0.2                                    2 <br />                                                                             R = 0.997                                     -0.3                                       2 <br />                                                                                                                                                                     R = 0.9958 <br />                                       -0.3 <br />                                                                                                                           -0.4 <br />                                       -0.4                                                                                -0.5 <br />  <br />                                                  log(x), x = minimum indel length (aa)                                               log(x), x = minimum indel length (aa) <br />  <br />  <br />                                              B. subtilis Essential Proteins                                               B. subtilis Non-essential Proteins <br />                                       0.5                                                                                 0.6 <br />                                                           y = 0.5874x - 0.3259 <br />                                       0.4                       2 <br />                                                                                                                           0.5                   y = 0.651x - 0.3614 <br />                                                                R = 0.9932                                                                             2 <br />                                       0.3                                                                                 0.4                       R = 0.9981 <br />                   log(-log(CDF(x))) <br />  <br />  <br />  <br />  <br />                                                                                                       log(-log(CDF(x))) <br />  <br />  <br />  <br />  <br />                                       0.2                                                                                 0.3 <br />                                                                                                                           0.2 <br />                                       0.1 <br />                                                                                                                           0.1 <br />                                         0 <br />                                              0               0.5                 1              1.5                         0 <br />                                       -0.1 <br />                                                                                                                           -0.1 0                  0.5                 1               1.5 <br />                                       -0.2 <br />                                                                               y = 0.4944x - 0.2698                        -0.2                                    y = 0.5955x - 0.3308 <br />                                       -0.3                                          2                                                                                   2 <br />                                                                                    R = 0.9934                             -0.3                                         R = 0.9981 <br />                                       -0.4                                                                                -0.4 <br />                                       -0.5                                                                                -0.5 <br />  <br />                                                  log(x), x = minimum indel length (aa)                                               log(x), x = minimum indel length (aa) <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Approximation    abundance indels Weibull distribution <br /> Approximation abundance indels Weibull distribution. r2 values close 1.0 indicated abundance of <br /> insertions (blue points blue line) deletions (purple points purple line) essential non-essential proteins the <br /> three query species accurately modeled Weibull distribution. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                 Page 9 13 <br />                                                                                                                                                                           (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                               http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> While t-tests showed essential proteins sig-      tein B, direct indirect inter- <br /> nificantly indels non-essential proteins,       action protein X. <br /> AUROC results showed indels frequently <br /> enough non-essential proteins make difficult        The na&#195;&#175;ve method used calculate betweenness <br /> accurately predict protein essential        measure require O(n3) time O(n2) in <br /> based solely indel frequency. A recent publication       space, making calculation inefficient. Therefore, we <br /> [27] identified 14 characteristic sequence features,    used faster method developed Brandes [33], which <br /> codon adaptation, hydrophobicity, localization sig-         implemented C executed Linux plat- <br /> nals, potentially associated essential genes     form. Briefly, method calculates betweenness for <br /> in fungal genomes. Thus, different features            particular vertex, v, computing number of <br /> likely indicative essential proteins pre-    times v occurs vertices, x y, in <br /> dictions based indel frequency accurate        network. Next, value known pair-dependency <br /> if features considered.                        calculated. This value proportion shortest paths <br />                                                                 vertices x y v lines on. This step is <br /> Do proteins indels different network properties       repeated vertices v, x, y values are <br /> than indels?                                      summed. Table 2 shows indel containing proteins <br /> It documented essential proteins         greater betweenness, suggesting importance in <br /> often involved greater number interactions (i.e.      S. cerevisiae protein-protein interaction network. <br /> greater connectivity) non-essential proteins [28,29].      Taken together, observations suggested the <br /> Because indels tend occur external surface         presence indels related network properties <br /> proteins, usually reverse turns coils loops        (connectivity betweenness) proteins PINs. One <br /> [30,31], structures play important roles pro-      application results bait-prey pull <br /> tein-protein interactions, reasoned formation        experiments. These results suggest increase <br /> indels means proteins acquired and/         coverage PIN pull experiment, <br /> or modified interactions involved in.         bait contains indel, indel <br /> Using protein-protein interaction counts 4148       containing proteins involved greater number of <br /> S. cerevisiae proteins available Munich Informa-       interactions greater betweenness. <br /> tion Center Protein Sequences (MIPS) database [32], <br /> we determined indel containing proteins S.           Conclusion <br /> cerevisiae greater mean connectivity      We previously conducted large scale analysis poten- <br /> do not. We calculated mean connectivity proteins         tially targetable indels bacterial protozoan patho- <br /> with indels minimum length           gen proteins [10]. In study, located many <br /> ten amino acids (Table 2). While absolute differences       examples essential pathogen proteins contained <br /> between mean connectivity types proteins         sizable indels. Therefore, objective study to <br /> were small, differences statistically significant (P   determine indels related essential non- <br /> &lt; 0.05) determined t-test. Therefore, general,     essential proteins. To knowledge, relationship <br /> proteins indels connections proteins        previously explored. We analyzed <br /> that not. This explained indels creating and/      indels ability discriminate essential <br /> or exposing new interaction sites, result new          non-essential proteins compared network <br /> interactions, illustrated L. donovani example     properties, connectivity betweenness, indel and <br /> [7].                                                            non-indel containing proteins. We determined for <br />                                                                 species, Bacillus subtilis, Escherichia coli, Saccharo- <br /> We considered indel containing proteins            myces cerevisiae, essential proteins greater mean <br /> had greater betweenness proteins indels.         indel frequency non-essential proteins. The abun- <br /> The betweenness measure graph theory deter-      dance indels types proteins accu- <br /> mined counting number times particular vertex       rately modeled Weibull distribution. Furthermore, <br /> is located shortest path vertices       demonstrated ROC curves accurate discrimi- <br /> network [33]. From biological perspective, between-       nation essential non-essential proteins based solely <br /> ness accounts direct indirect influences pro-    indel frequency achieved. Finally, we <br /> teins distant location network. For example,     showed indel containing proteins different net- <br /> two clusters interacting proteins, A B, joined       work properties, greater connectivity <br /> together mutual interaction pro-        betweenness, suggesting possible role indels in <br /> tein X, X high betweenness measure,           regulation interaction partners. <br /> because protein A interact pro- <br />  <br />  <br />  <br />                                                                                                                     Page 10 13 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                                http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> Table 2: Summary mean connectivity betweenness S. cerevisiae proteins indels: The mean connectivity and <br /> betweenness indel containing proteins significantly greater non-indel containing proteins. Significance was <br /> set P &lt; 0.05 <br />  <br />  Min Indel Length (aa)   Number proteins        Mean connectivity      Number proteins       Mean connectivity    Betweenness            Betweenness of <br />                          indel   proteins        proteins     proteins    proteins at <br />                          4 10 aa    indel 4   indel 4   indel   indel 4   indel at <br />                          long                      10 aa long             10 aa long               4 10 aa long   10 aa long             4 10 aa long <br />  <br />  <br />  4                       907                       4.194                     562                      3.986                   15354                     15133 <br />  10                      381                       4.394                     1088                     4.017                   15712                     15115 <br />  <br />  <br />  <br /> In analyses, did consider actual location                                 (35,429 bacterial 75,881 eukaryote). We also <br /> the indels folded dimensional protein struc-                                   obtained fully curated reviewed proteins each <br /> tures, critical effective drug design. Therefore,                              eukaryote species (54,927 reviewed eukaryote <br /> some future directions focus include                                  proteins). <br /> dimensional modeling indel containing proteins as <br /> well identifying functional protein domains                                    BLASTP parameters used determine alignments <br /> commonly disrupted indels. Given indels                                      We used formatdb [36] format subject protein <br /> used selectively target essential pathogen proteins                                 sequences BLAST databases. To align B. subtilis, E. <br /> have high sequence similarity human proteins, charac-                                    coli, S. cerevisiae query proteins subject proteins, <br /> terization indels potentially lead new drug                                conducted BLASTP-based alignment B. subtilis and <br /> targets infectious diseases.                                                            E. coli query proteins 53,454 bacterial subject <br />                                                                                             proteins S. cerevisiae query proteins the <br /> Methods                                                                                     282,632 eukaryote subject proteins. We set maximum E- <br /> Systematic knockout data NCBI RefSeq proteins                                           value 10-5 considered sequence alignments <br /> We conducted broad literature search identify fully                                    minimum 50% similarity. The parameters <br /> sequenced genomes genome-wide knockout                                             used analyses smaller set subject <br /> data available (i.e. protein essentiality defined). We                               species. The BLASTP alignments performed nine <br /> located complete knockout data B. subtilis (strain 168)                                 IBM machines running CentOS Linux operating sys- <br /> [17], E. coli (strain K12) [18], S. cerevisiae [19]. For                                tem. <br /> each species, downloaded complete non- <br /> redundant set proteins (&apos;query proteins&apos;) FASTA for-                                  Processing alignments contain indels <br /> mat NCBI RefSeq [16]. In total, 14,214 query pro-                                      We developed in-house Perl scripts process the <br /> teins analyzed. Next, obtained list essential                                results BLASTP alignments search indels. <br /> genes cross referenced NCBI RefSeq protein                                    For alignments matched BLASTP parameters, <br /> ID using in-house Perl script utilized BioPerl mod-                                 searched gaps opened query pro- <br /> ules (Version 1.5.1) [34] search gene                                    tein (deletions) subject protein (insertions) of <br /> the complete set RefSeq Genbank protein files                                    minimum X amino acids long, values X <br /> particular query organism.                                                                  ranged amino acids. Note gaps <br />                                                                                             reported insertions deletions based the <br /> NCBI RefSeq proteins BLAST databases                                                    query protein (Figure 1a). For insertion minimum <br /> We searched Entrez Genome Project section NCBI                                       X amino acids long, calculated Insertion Frequency <br /> [35] bacterial eukaryote genome projects anno-                                  (IF) follows: <br /> tated completed. From list, wide range bacte- <br /> rial eukaryote species chosen. We chose species                                                                I1 + I2 + I3 + ... + I22 <br /> from wide range different classes avoid biasing                                                    IF = <br />                                                                                                                        H1 + H2 + H3 + ... + H22 <br /> results particular organisms specific class. This <br /> resulted 22 bacterial species 15 eukaryote species                                   Ii number insertions query species <br /> (Additional file 1). Next, obtained complete set                                  shares species Hi number proteins that <br /> protein sequences selected organisms (&apos;subject                                   satisfied alignment parameters query spe- <br /> proteins&apos;) NCBI RefSeq. In total, set consisted                                cies species i. Similarly, calculated Deletion <br /> 53,454 bacterial 282,632 eukaryote subject proteins.                                    Frequency (DF) follows: <br />  <br /> To validate results, randomly chose                                                                D1 + D2 + D3 + ... + D22 <br /> bacterial eukaryote species (Additional file 3)                                                   DF = <br /> obtained respective proteins NCBI RefSeq                                                                    H1 + H2 + H3 + ... + H22 <br />  <br />  <br />  <br />                                                                                                                                                             Page 11 13 <br />                                                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                 http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> where Di number deletions query species <br /> shared species Hi number proteins                      Additional File 3 <br /> satisfied alignment parameters query spe-                        The bacteria eukaryote subject species utilized. <br /> cies species i. Note S. cerevisiae query                     Click file <br />                                                                                  [http://www.biomedcentral.com/content/supplementary/1471- <br /> species, I22, H22, D22 I15, H15, D15,                           2105-8-227-S3.ppt] <br /> respectively, 15 eukaryote subject spe- <br /> cies.                                                                            Additional File 4 <br />                                                                                  Mean insertion deletion frequencies essential non-essential <br /> Calculations statistical analyses                                            proteins plotted minimum indel length. Mean insertion and <br /> Receiver Operator Characteristic (ROC) curves                            deletion frequencies calculated essential non-essential query <br /> corresponding Area Under ROC curve (AUROC)                              proteins aligned proteins 14 randomly chosen subject species. <br />                                                                                  The t-test statistic shown minimum indel lengths found <br /> determined using R statistical package, version 2.3.1 <br />                                                                                  significantly essential (blue bars) non-essential (purple <br /> [37] Linux-like operating systems ROCR pack-                         bars) proteins. Significance set P &lt; 0.05. Note differ- <br /> age [38]. An ROC curve plots Sensitivity (True Posi-                         ence observed deletions E. coli proteins. Also note inser- <br /> tives/(True Positives + False Negatives)) vs False Positive                      tions minimum length three, four, amino acids more <br /> Rate (1 - (True Negatives)/(True Negatives + False Posi-                         frequently non-essential essential proteins B. subtilis. See text <br /> tives)). Perl scripts performing t-test calculations                   discussion. <br />                                                                                  Click file <br /> implemented significance set P &lt; 0.05.                                [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                  2105-8-227-S4.ppt] <br /> Protein-protein interaction counts <br /> The S. cerevisiae protein-protein interaction counts                        Additional File 5 <br /> obtained Munich Information Center Protein                          Mean indel frequency calculated curated eukaryote proteins. <br /> Sequences (MIPS) database [32]. In total, obtained                            Mean indel frequencies calculated curated S. cerevisiae <br /> interaction counts 4148 proteins. Of 4148 S. cere-                       essential non-essential proteins aligned curated proteins the <br />                                                                                  randomly chosen subject species. Note observed trend in <br /> visiae proteins interaction counts, 837 (20.2%) were <br />                                                                                  mean indel frequency essential proteins greater that <br /> essential. We determined best match Homo sapiens                          non-essential proteins seen smaller set curated pro- <br /> using BLASTP algorithm. Again, specified maxi-                          teins, suggesting observed trend seen proteins the <br /> mum E-value 10-5 query subject pro-                          complete set subject species merely sequencing/annota- <br /> teins shared 50% sequence similarity. Using in-                         tion errors. The t-test statistic shown minimum indel lengths that <br /> house Perl scripts, determined proteins                            significantly essential (blue bars) non- <br />                                                                                  essential (purple bars) proteins. Significance set P &lt; 0.05. See text <br /> contained indel amino                      discussion. <br /> acids long.                                                                      Click file <br />                                                                                  [http://www.biomedcentral.com/content/supplementary/1471- <br /> Authors&apos; contributions                                                           2105-8-227-S5.ppt] <br /> SKC acquired data various online resources, <br /> developed <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>, performed analyses, <br /> and wrote manuscript. FH implemented between- <br /> ness algorithm performed network analyses. AC                       Acknowledgements <br /> conceived study, SKC, MH, AC partici-                      SKC funded award CIHR/MSFHR Strategic Training Pro- <br />                                                                             gram Bioinformatics Health Research http://bioinformatics.bcgsc.ca. <br /> pated design interpretation results. All <br />                                                                             MH supported Michael Smith Foundation Health Research <br /> authors read approved final manuscript.                             (MSFHR) Natural Sciences Engineering Research Council <br />                                                                             (NSERC). AC funded Genome Canada Genome BC the <br /> Additional material                                                         PRoteomics Emerging PAthogen REsponse (PREPARE) Project. The <br />                                                                             authors acknowledge helpful comments suggestions provided by <br />                                                                             anonymous reviewers. <br />  Additional File 1 <br />  The 22 bacterial 15 eukaryote subject species utilized.                References <br />  Click file                                                        1.       Fraser CM, Gocayne JD, White O, Adams MD, Clayton RA, Fleis- <br />  [http://www.biomedcentral.com/content/supplementary/1471-                           chmann RD, Bult CJ, Kerlavage AR, Sutton G, Kelley JM, Fritchman <br />  2105-8-227-S1.ppt]                                                                  RD, Weidman JF, Small KV, Sandusky M, Fuhrmann J, Nguyen D, <br />                                                                                      Utterback TR, Saudek DM, Phillips CA, Merrick JM, Tomb JF, Dough- <br />                                                                                      erty BA, Bott KF, Hu PC, Lucier TS, Peterson SN, Smith HO, <br />  Additional File 2                                                                   Hutchison CA 3, Venter JC: The minimum gene complement of <br />  Indel similar protein counts query species compared            Mycoplasma genitalium. Science 1995, 270:397-403. <br />  subject species.                                                      2.       Hutchison CA, Peterson SN, Gill SR, Cline RT, White O, Fraser CM, <br />  Click file                                                                 Smith HO, Venter JC: Global transposon mutagenesis a <br />  [http://www.biomedcentral.com/content/supplementary/1471-                           minimal Mycoplasma genome. Science 1999, 286:2165-2169. <br />  2105-8-227-S2.ppt] <br />  <br />  <br />  <br />                                                                                                                                             Page 12 13 <br />                                                                                                                       (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:227                                                                     http://www.biomedcentral.com/1471-2105/8/227 <br />  <br />  <br />  <br /> 3.    Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, Maruf                 K, Bussey H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnston <br />       M, Hutchison CA 3, Smith HO, Venter JC: Essential genes                       M: Functional profiling Saccharomyces cerevisiae <br />       minimal bacterium. Proc Natl Acad Sci 2006, 103:425-430.                           genome. Nature 2002, 418:387-391. <br /> 4.    Cole ST: Comparative mycobacterial genomics tool                    20.   Akerley BJ, Rubin EJ, Novick VL, Amaya K, Judson N, Mekalanos JJ: A <br />       drug target antigen discovery. Eur Respir J Suppl 2002,                        genome-scale analysis identification genes required for <br />       36:78s-86s.                                                                        growth survival Haemophilus influenzae. Proc Natl Acad <br /> 5.    Chalker AF, Lunsford RD: Rational identification new anti-                      Sci 2002, 99:966-971. <br />       bacterial drug targets essential viability using              21.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />       genomics-based approach. Pharmacol Ther 2002, 95:1-20.                             MD, Somera AL, Kyrpides NC, Anderson I, Gelfand MS, Bhattacharya <br /> 6.    Nandan D, Lopez M, Ban F, Huang M, Li Y, Reiner NE, Cherkasov A:                   A, Kapatral V, D&apos;Souza M, Baev MV, Grechkin Y, Mseeh F, Fonstein <br />       Indel-based targeting essential proteins human patho-                        MY, Overbeek R, Barabasi AL, Oltvai ZN, Osterman AL: Experi- <br />       gens close host orthologue(s): discovery selec-                       mental determination level analysis essential <br />       tive inhibitors Leishmania donovani elongation factor-1&#206;&#177;.                      genes Escherichia coli MG1655.                    J Bacteriol 2003, <br />       Proteins 2007, 67:53-64.                                                           185:5673-5684. <br /> 7.    Nandan D, Reiner NE: Leishmania donovani engages regula-                  22.   Forsyth RA, Haselbeck RJ, Ohlsen KL, Yamamoto RT, Xu H, Trawick <br />       tory interference targeting macrophage protein tyrosine                         JD, Wall D, Wang L, Brown-Driver V, Froelich JM, C KG, King P, <br />       phosphatase SHP-1. Clin Immunnol 2005, 114:266-277.                                McCarthy M, Malone C, Misiner B, Robbins D, Tan Z, Zhu Zy ZY, <br /> 8.    Cherkasov A, Nandan D, Reiner NE: Selective targeting indel-                    Carr G, Mosca DA, Zamudio C, Foulkes JG, Zyskind JW: A genome- <br />       inferred differences spatial structures highly homolo-                       wide strategy identification essential genes Sta- <br />       gous proteins. Proteins 2005, 58:959-954.                                          phylococcus aureus. Mol Microbiol 2002, 43:1387-1400. <br /> 9.    Li YY, Jones SJ, Cherkasov A: Selective targeting indel-                  23.   Coles S: An introduction statistical modeling extreme values Lon- <br />       inferred differences spatial structures homologous pro-                      don:Springer-verlag; 2001. <br />       teins. J Bioinform Comput Biol 2006, 2:403-414.                              24.   Teixeira EC, Piascik JR, Stoner BR, Thompson JY: Dynamic fatigue <br /> 10.   Cherkasov A, Lee SJ, Nandan D, Reiner NE: Large-Scale Survey                       strength characterization ceramic materials. J <br />       Potentially Targetable Indels Bacterial Protozoan                       Mater Sci Mater Med 2007, 18:1219-1224. <br />       Proteins. Proteins 2006, 62:371-380.                                         25.   Cherkasov A, Ho Sui SJ, Brunham RC, Jones SJ: Structural charac- <br /> 11.   Bateman A, Coin L, Durbin R, Finn RD, Hollich V, Griffiths-Jones S,                terization genomes large scale sequence-sequence <br />       Khanna A, Marshall M, Moxon S, Sonnhammer EL, Studholme DJ,                        threading: application reliability analysis structural <br />       Yeast C, Eddy SR: The Pfam protein families database. Nucleic                      genomics. BMC Bioinformatics 2004, 5:101. <br />       Acids Res 2004, 32:D138-141.                                                 26.   Decottignies A, Sanchez-Perez I, Nurse P: Schizosaccharomyces <br /> 12.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes                     pombe essential genes: pilot study. Genome Res 2003, <br />       evolutionarily conserved nonessential genes                          13:399-406. <br />       bacteria. Genome Res 2002, 12:962-968.                                    27.   Seringhaus M, Paccanaro A, Borneman A, Snyder M, Gerstein M: Pre- <br /> 13.   Zhang L, Li WH: Mammalian housekeeping genes evolve                           dicting essential genes fungal genomes. Genome Res 2006, <br />       slowly tissue-specific genes. Mol Biol Evol 2004, 21:236-239.                 16:1126-1135. <br /> 14.   Barabasi AL, Oltvai ZN: Network biology: understanding                   28.   Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central- <br />       cell&apos;s functional organization. Nat Rev Genet 2004, 5:101-113.                     ity protein networks. Nature 2001, 411:41. <br /> 15.   Barabasi AL, Albert R: Emergence scaling random net-                   29.   He X, Zhan Z: Why hubs tend essential protein net- <br />       works. Science 1999, 286:509-512.                                                  works? PLoS Genet 2006, 2:e88. <br /> 16.   Pruitt KD, Tatusova T, Maglott DR: NCBI Reference Sequence                   30.   Pascarella S, Argos P: Analysis insertions/deletions protein <br />       (RefSeq): curated non-redundant sequence database                             structures. J Mol Biol 1992, 224:461-471. <br />       genomes, transcripts proteins. Nucleic Acids Res 2005,                   31.   Benner SA, Cohen MA, Gonnet GH: Empirical structural <br />       33:D501-D504.                                                                      models insertions deletions divergent evolu- <br /> 17.   Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, Arnaud                 tion proteins. Mol Biol Evol 1993, 11:316-324. <br />       M, Asai K, Ashikaga S, Aymerich S, Bessieres P, Boland F, Brignell SC,       32.   Mewes HW, Frishman D, Guldener U, Mannhaupt G, Mayer K, <br />       Bron S, Bunai K, Chapuis J, Christiansen LC, Danchin A, Debarbouille               Mokrejs M, Morgenstern B, Munsterkotter M, Rudd S, Weil B: MIPS: <br />       M, Dervyn E, Deuerling E, Devine K, Devine SK, Dreesen O, Err-                     database genomes protein sequences. Nucleic Acids <br />       ington J, Fillinger S, Foster SJ, Fujita Y, Galizzi A, Gardan R, Eschevins         Res 2002, 30:31-34. <br />       C, Fukushima T, Haga K, Harwood CR, Hecker M, Hosoya D, Hullo                33.   Brandes U: A faster algorithm betweenness centrality. J <br />       MF, Kakeshita H, Karamata D, Kasahara Y, Kawamura F, Koga K,                       Math Sociol 2001, 25:163-177. <br />       Koski P, Kuwana R, Imamura D, Ishimaru M, Ishikawa S, Ishio I, Le            34.   Stajich JE, Block D, Boulez K, Brenner SE, Chervitz SA, Dagdigian C, <br />       Coq D, Masson A, Mauel C, Meima R, Mellado RP, Moir A, Moriya S,                   Fuellen G, Gilbert JG, Korf I, Lapp H, Lehvaslaiho H, Matsalla C, Mun- <br />       Nagakawa E, Nanamiya H, Nakai S, Nygaard P, Ogura M, Ohanan T,                     gall CJ, Osborne BI, Pocock MR, Schattner P, Senger M, Stein LD, <br />       O&apos;Reilly M, O&apos;Rourke M, Pragai Z, Pooley HM, Rapoport G, Rawlins                   Stupka E, Wilkinson MD, Birney E: The Bioperl toolkit: Perl mod- <br />       JP, Rivas LA, Rivolta C, Sadaie A, Sadaie Y, Sarvas M, Sato T, Saxild              ules life sciences. Genome Res 2002, 12:1611-16118. <br />       HH, Scanlan E, Schumann W, Seegers JF, Sekiguchi J, Sekowska A,              35.   Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, <br />       Seror SJ, Simon M, Stragier P, Studer R, Takamatsu H, Tanaka T,                    Chetvernin V, Church DM, DiCuccio M, Edgar R, Federhen S, Geer <br />       Takeuchi M, Thomaides HB, Vagner V, van Dijl JM, Watabe K, Wipat                   LY, Helmberg W, Kapustin Y, Kenton DL, Khovayko O, Lipman DJ, <br />       A, Yamamoto H, Yamamoto M, Yamamoto Y, Yamane K, Yata K,                           Madden TL, Maglott DR, Ostell J, Pruitt KD, Schuler GD, Schriml LM, <br />       Yoshida K, Yoshikawa H, Zuber U, Ogasawara N: Essential Bacillus                   Sequeira E, Sherry ST, Sirotkin K, Souvorov A, Starchenko G, Suzek <br />       subtilis genes. Proc Natl Acad Sci 2003, 100:4678-4683.                            TO, Tatusov R, Tatusova TA, Wagner L, Yaschenko E: Database <br /> 18.   Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko                    resources National Center Biotechnology Infor- <br />       KA, Tomita M, Wanner BL, Mori H: Construction Escherichia                       mation. Nucleic Acids Res 2006, 34:D173-D180. <br />       coli K-12 in-frame, single-<span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: Keio                   36.   BLAST Binaries [ftp://ftp.ncbi.nih.gov/blast/] <br />       collection. Mol Syst Biol 2006, 2:2006.0008.                                 37.   CRAN Project [http://www.r-project.org] <br /> 19.   Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S,            38.   Sing T, Sander O, Beerenwinkel N, Lengauer T: ROCR: visualizing <br />       Lucau-Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, El-                    classifier performance R. Bioinformatics 2005, 20:3940-3941. <br />       Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br />       M, Davis K, Deutschbauer A, Entian KD, Flaherty P, Foury F, Garfinkel <br />       DJ, Gerstein M, Gotte D, Guldener U, Hegemann JH, Hempel S, Her- <br />       man Z, Jaramillo DF, Kelly DE, Kelly SL, Kotter P, LaBonte D, Lamb <br />       DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard <br />       P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Sche- <br />       rens B, Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, <br />       Storms RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang CY, <br />       Ward TR, Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu <br />  <br />  <br />  <br />                                                                                                                         <br /> </body></html>