<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2007 Genetics Society America <br /> DOI: 10.1534/genetics.106.064329 <br />  <br />  <br />  <br />         Functional Analysis Gene Duplications Saccharomyces cerevisiae <br />  <br />                     Yuanfang Guan,*,&#226;&#8364;&#160; Maitreya J. Dunham* Olga G. Troyanskaya*,&#226;&#8364;&#161;,1 <br />                 *Lewis-Sigler Institute Integrative Genomics, Carl Icahn Laboratory, &#226;&#8364;&#160;Department Molecular Biology and <br />                              &#226;&#8364;&#161; <br />                                Department Computer Science, Princeton University, Princeton, New Jersey 08544 <br />                                                      Manuscript received August 2, 2006 <br />                                                   Accepted publication November 7, 2006 <br />  <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Gene duplication occur scales: whole-genome duplications (WGD) smaller-scale dupli- <br />              cations (SSD) involving individual genes genomic segments. Duplication result functionally re- <br />              dundant genes diverge function neofunctionalization subfunctionalization. The effect of <br />              duplication scale functional evolution explored, probably lack global <br />              knowledge protein function different times duplication events. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> address question, used <br />              integrated Bayesian analysis diverse functional genomic data accurately evaluate extent of <br />              functional similarity divergence paralogs global scale. We paralogs resulting <br />              whole-genome duplication likely share interaction partners biological functions <br />              smaller-scale duplicates, independent sequence similarity. In addition, WGD paralogs lower <br />              frequency essential genes higher synthetic lethality rate, instead diverge expression <br />              <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span> upstream regulatory region. Thus, analysis demonstrates WGD paralogs generally have <br />              similar compensatory functions diverging expression patterns, suggesting potential distinct <br />              evolutionary scenarios paralogs arose different duplication mechanisms. Furthermore, by <br />              identifying functional disparities types duplicates, reconcile previous disputes <br />              relationship sequence divergence expression divergence essentiality. <br />  <br />  <br />  <br />  <br /> G     ENE duplication major source new genes <br />        central factor influencing genome <br /> evolution (Ohno 1970; Wolfe Li 2003). Such dup- <br />                                                                          insights respect different attributes duplicate <br />                                                                          genes. For example, basis 41 WGD pairs, <br />                                                                          Baudot et al. (2004) examined function dupli- <br /> lication occur scales: duplication                 cate pairs interaction-network analysis, but <br /> whole genome (WGD) smaller-scale duplications                        simple relationship sequence <br /> (SSD), occur continuously involve individual                   identity functional similarity. An earlier study <br /> genes genomic segments (see review Sankoff                         (Brun et al. 2003) reached similar result basis <br /> 2001). Duplicated genes retained dif-                      limited combination WGD SSD pairs. Also, <br /> ferent selection mechanisms undergo                         basis WGD paralogs, Seoighe Wolfe <br /> different evolutionary fates. Paralogs selected                   (1999) increased levels gene expression <br /> for increased dosage repository gene con-                    significant factor determining genes <br /> version deleterious changes copy                   retained duplicates. However, selection <br /> result functional redundancy (Nadeau Sankoff                      stages duplicates pass be- <br /> 1997; Nowak et al. 1997; Gu 2003; Gu et al. 2003). Alter-                come persistent genome somewhat <br /> natively, paralogs diverge generation                 different WGD SSD sets (Davis Petrov <br /> of new gene functions (neofunctionalization) (Taylor                     2005), modes duplication generate genes <br /> and Raes 2004) subdividing multiple functions                     different molecular attributes. Thus results based <br /> (subfunctionalization) complementary degen-                      combined analysis types duplicates ignore <br /> eration (Force et al. 1999; Stoltzfus 1999; Lynch                    properties unique particular set. Furthermore, in- <br /> Force 2000). However, relative importance                   dividual analysis group does necessarily <br /> mechanisms preserving WGD vs. SSD duplicates,                         generalize other. In fact, differences <br /> indicated resultant functional conservation/                      data sets, previous studies (Wagner 2000a; Gu <br /> divergence paralogs, investigated.                  et al. 2003) drew inconsistent conclusions re- <br />    Functional studies focusing WGD SSD                   lationship sequence divergence dupli- <br /> the combination sets established                    cate genes fitness effect null <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. In <br />                                                                          addition, ways relationship gene <br />   1 <br />                                                                          duplication evolution transcriptional regulation <br />    Corresponding author: Department Computer Science, Princeton <br /> University, 35 Olden St., Princeton, NJ 08544.                           controversial (see review Li et al. 2005): <br /> E-mail: ogt@cs.princeton.edu                                             Wagner (2000b) suggested regulatory sequences <br />  <br /> Genetics 175: 933&#226;&#8364;&#8220;943 (February 2007) <br />  934                                     Y. Guan, M. J. Dunham O. G. Troyanskaya <br />  <br /> and mRNA expression patterns duplicate gene pairs            enabled demonstrate WGD paralogs, in- <br /> evolve independently coding sequence,            dependent sequence divergence level, general <br /> others highly significant relationship        likely share physical protein&#226;&#8364;&#8220;protein interac- <br /> sequence divergence age (Gu et al. 2002, 2005;               tion partners functional relationships. In addition, <br /> Zhang et al. 2004).                                             WGD paralogs lower essentiality higher <br />    The studies based WGD              synthetic lethality frequency. However, functional <br /> duplicates combination duplication groups.         compensation paralogs followed by <br /> An open question WGD SSD duplicates              complete redundancy, diverse expression <br /> undergo different evolutionary scenarios, explain-         patterns upstream regulatory regions between <br /> ing disparities studies. Despite            WGD paralogs suggest role modulating expres- <br /> unambiguous identification WGD blocks Saccharo-           sion level. Moreover, propensity similar, <br /> myces cerevisiae (Dietrich et al. 2004; Kellis et al. 2004a),   compensatory functions diverge expression <br /> few previous studies focused global differ-         patterns unique WGD paralogs, comparison to <br /> ences WGD SSD duplicates. To                SSD paralogs, suggests potential distinct <br /> knowledge, study consider global dif-           evolutionary scenarios paralogs arose through <br /> ferences sets Davis Petrov              different duplication mechanisms <br /> (2005), showed sets differ signifi- <br /> cantly overall molecular functional enrichment, but <br /> are similar respect codon bias evolutionary                                     METHODS <br /> rate. Yet study focused differences between <br /> WGD SSD gene sets respect evolutionary fates           Identifying WGD SSD paralogs: To define set of <br /> and consequent functional conservation/divergence be-           paralogous pairs yeast genome, con- <br /> tween paralogs.                                                 structed set alignments S. cerevisiae proteins. <br />    To address problem, important             Protein sequences ORFs S. cerevisiae (except <br /> unresolved question discriminate functions         dubious ORFs pseudogenes) downloaded <br /> the paralogs evaluate subtle differences. The         Saccharomyces Genome Database (SGD) <br /> divergence paralogs intensively stud-          (Cherry et al. 1998). For ORFs, then <br /> ied sequence level S. cerevisiae (e.g., Lynch     used protein BLAST (Altschul et al. 1990) E &#194;&#188; <br /> Conery 2000; Kondrashov et al. 2002),            0.01 protein hits S. cerevisiae <br /> examining number synonymous (dS) vs. non-                genome. We used alignments identify <br /> synonymous (dN) substitutions. Unfortunately, attempt        suboptimal matches (the best match self-alignment) <br /> to calculate nonsynonymous vs. synonymous sub-              basis Kellis et al. (2004b) method. This <br /> stitution WGD paralogs shows dS&#226;&#8364;&#8482;s          approach takes account fact similarity <br /> are saturated (supplemental Figure S1 http://www.            query protein x target protein y split <br /> genetics.org/supplemental/). This hinders overall            multiple BLAST hits. Intuitively, BLAST hits <br /> perspective functional divergence            x y weighted amino acid per- <br /> WGD paralogs vs. SSD paralogs. The evolutionary rate            centage identity <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> aligned thereby <br /> relative orthologs shown similarly        grouped single match. Compared global align- <br /> biased (Davis Petrov 2005). However, rates              ment, method includes duplicate pairs have <br /> calculated referring orthologs represent           internal inversion members. The detailed <br /> the divergence paralogs, affected          procedure follows. <br /> by gene conversion. One alternative compare              The weight hit assigned as <br /> function paralogs gene ontology (GO)                                 wk &#194;&#188; lk 3 Ik ; k 2 &#195;&#176;1; n&#195;&#382;; <br /> (Ashburner et al. 2000) annotations. Unfortunately, <br /> subtle differences paralogs prevented               lk length Ik overall amino acid <br /> successful function discrimination based GO anno-            identity hit k, n total number hits for <br /> tations (Baudot et al. 2004),              protein x target protein y. <br /> sometimes &#226;&#8364;&#732;&#226;&#8364;&#732;inferred sequence structural simi-             To group BLAST hits single match, the <br /> larity&#226;&#8364;&#8482;&#226;&#8364;&#8482; (ISS).                                                 nonoverlapping portions hits added to <br />    Here took advantage diverse functional geno-           obtain maximized identity number x y. <br /> mic high-throughput data carried genome-            For paralogous pair (x, y), wk ranked its <br /> wide analyses divergence biological function          correspondent start ending sites&#195;&#176;axk ; bxk &#195;&#382;; &#195;&#176;ayk ; byk &#195;&#382;  <br /> between whole-genome paralogs (506 pairs) smaller-          &#195;&#176;axk , bxk ayk , byk &#195;&#382;were recorded. The ranked wk <br /> scale duplicates (1193 pairs including 1862 genes).             added total weight W(x,y) wj <br /> Using Bayesian methodology integrate diverse func-         corresponding start ending sites satisfying <br /> tional genomic data .6500 publications, ac-             &#194;&#189;&#195;&#176;axj . bxk &#195;&#382; &#195;&#176;axk . bxj &#195;&#382;  &#194;&#189;&#195;&#176;ayj . byk &#195;&#382; &#195;&#176;ayk . byj &#195;&#382; were re- <br /> curately predicted specific function gene. This        tained. The process repeated hits <br />                                            Functional Analysis Gene Duplications                                          935 <br />  <br /> were added W(x,y). Hits overlapped an- <br /> other hit higher weight discarded it- <br /> eration. The summed W(x,y) gives maximized identity <br /> number protein x y. Percentage iden- <br /> tity calculated as <br />                                                   ! <br />                       W&#195;&#176;x;y&#195;&#382;         W&#195;&#176;x;y&#195;&#382; <br />        pi&#195;&#176;x;y&#195;&#382; &#194;&#188; Max         3 100%;        3 100% : <br />                       W&#195;&#176;x;x&#195;&#382;         W&#195;&#176;y;y&#195;&#382; <br />  <br />    Suboptimal matches [pi(x,y), x 6&#194;&#188; y], used con- <br /> struct paralogous pairs. WGD duplicates (528 pairs) <br /> were classified depending inclusion WGD <br /> duplicate blocks characterized genomewide com-                  Figure 1.&#226;&#8364;&#8221;The number duplicate pairs se- <br /> parisons S. cerevisiae Kluyveromyces waltii (Kellis       quence divergence level. The duplicates grouped into <br /> et al. 2004a; Byrne Wolfe 2005), diverged             23 bins sliding window 400 pairs size 100 pairs <br /> just prior polyploidization event. SSD duplicate         window slide. Such grouping used following anal- <br /> pairs defined paralogous pairs included           ysis included percentage identity. Thus adjacent bins <br />                                                                 include pairs smooth pattern <br /> the WGD list. Due different methods used             identify general trends different attributes the <br /> identifying paralogous pairs WGD (both synteny          WGD SSD sets. <br /> sequence similarity) SSD (sequence similarity only), <br /> measurements SSD pairs contain fluctua- <br /> tions, necessitating statistical analyses       related proteins using Bayesian data integration <br /> that performed study.                                method described Myers et al. (2005) in- <br />    In way identified 2604 pairs (including 528          corporate diverse genomic data sources. This method <br /> WGDs 2076 potential SSDs) according                integrated different types data (for example, gene <br /> percentage identity. To reduce data fluctuation,          expression, interaction data, high-throughput data, or <br /> constructed 23 groups sliding window 400 pairs        single experiments), using Bayesian network trained <br /> in size (as sum WGD SSD duplicates) 100            using expectation-maximization learning algorithm <br /> pairs window slide. Using nonoverlapping bins               (Dempster et al. 1977) known functionally relevant <br /> demonstrates trends attributes          GO biological process annotations (Myers et al. 2006) <br /> we studied (supplemental Figure S10 http://www.              gold standard. Intuitively, gene i&#226;&#8364;&#8220;gene j <br /> genetics.org/supplemental/). However, overlap-            pair, network asked following question: What is <br /> ping bins unified grouping sets             probability, basis experimental evi- <br /> paralogs ensured WGD SSD pairs             dence presented, products gene gene j have <br /> included bin statistical analysis enabled       functional relationship (i.e., involved same <br /> us compare WGD SSD paralogs similar se-               biological process)? The trained network integrated <br /> quence identity (Figure 1). The percentage identity          data sets weighing relative accuracy coverage of <br /> assigned group figures median value       experimental method; data sets were <br /> of group. Because WGD pa-              accurate predicting known GO annotations <br /> ralogous pairs falling bins (Figure       given higher weight predicting interaction part- <br /> 1), pairs low-alignment bins likely     ners functional relationships. The weighted data <br /> to true paralogs, bins (,20%, ,25         sets used predict confidence rela- <br /> WGD pairs) excluded analysis. Thus            tionship proteins. This Bayesian data inte- <br /> 506 WGD pairs 1193 unique SSD pairs used.              gration step reduced heterogeneous input <br />    We examined effect large number           data protein pairs score indicating like- <br /> ribosomal WGD paralogs results. We identified            lihood functionally (or physically) interact, <br /> ribosomal genes annotated protein             allowing different types data combined with <br /> biosynthesis term gene ontology (see supplemen-          other. <br /> tal files list genes http://www.genetics.org/       We considered types relationships: general <br /> supplemental/). For analysis, gene pair       functional relationships, indicate proteins in- <br /> one paralogs ribosomal genes excluded          volved biological process, physical inter- <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> analysis. Furthermore, control results          actions. The evidence protein&#226;&#8364;&#8220;protein interaction <br /> being biased duplicates large gene families,         predictions included yeast two-hybrid, copurification, <br /> repeated analyses using reciprocal best hits.           affinity-precipitation data, etc. (for list evi- <br />    Prediction shared protein&#226;&#8364;&#8220;protein interaction             dence, supplemental information http:// <br /> partners functional relationships: We predicted             www.genetics.org/supplemental/). Experimental evi- <br /> protein&#226;&#8364;&#8220;protein interaction partners functionally           dence general functional relationship includes <br />  936                                      Y. Guan, M. J. Dunham O. G. Troyanskaya <br />  <br /> all data supportive involvement bio-          1998). The enrichment GO term (in percent- <br /> logical process (Myers et al. 2005), including physical          age) WGD SSD sets using a <br /> and genetic interactions, synthetic data, shared se-             hypergeometric distribution identify en- <br /> quence motifs, curated literature. We considered             riched GO terms lowest Bonferonnni-corrected <br /> predictions Bayesian confidence cutoffs ranging             P-value (see supplemental information http://www. <br /> from 0.2 0.95 experiments (our predictions         genetics.org/supplemental/ results this <br /> physical interaction functional relationship             analysis). To summarize information Figure 2 <br /> available supplemental information). The percent-             supplemental Figure S2 (http://www.genetics.org/ <br /> age shared interaction partners (or shared functional         supplemental/), performed similar analysis with <br /> relationships) paralogs total number            GO slim terms assessed enrichment terms <br /> of interaction partners (or functional relationships)         cumulative distribution function. <br /> the pair calculated as <br />                                                                                         RESULTS <br />                               2 3 n&#195;&#176;x;y&#195;&#382; <br />              pshared&#195;&#176;x;y&#195;&#382; &#194;&#188;              3 100%; <br />                                nx 1 ny                              WGD paralogs exhibit higher propensity SSD <br />                                                                  paralogs share protein&#226;&#8364;&#8220;protein interaction partners <br /> where nx ny represent number interactions/            functional relationships: We addressed the <br /> functional relationships x y proteins, respec-           question WGD SSD duplicates <br /> tively, n(x,y) represents number common               participate different biological processes examin- <br /> interactions/functional relationships x y.           ing biases GO annotations (Ashburner et al. <br /> The cutoffs difference WGD              2000). We significant differences enrichment <br /> SSD groups sensitive used func-             WGD SSD sets respect bio- <br /> tional analyses parsed percentage identity,            logical process (Figure 2) cellular component an- <br /> results robust different cutoffs         notations (supplemental Figure S2A http://www. <br /> cutoff range.                                                    genetics.org/supplemental/). We confirmed the <br />    Essentiality, synthetic lethality, upstream regulatory        enrichment differences terms molecular function <br /> region, expression data retrieval analysis: To           annotation (Davis Petrov 2005, supplemental <br /> assess essentiality, results systematic deletion experi-      Figure S2B http://www.genetics.org/supplemental/). <br /> ments S. cerevisiae retrieved Saccharo-         With respect biological process, example, <br /> myces genome deletion project 11/25/05 (http://               sets enriched genes involved cell <br /> www-sequence.stanford.edu/group/yeast_deletion_                  homeostatis, morphogenesis, protein modification, re- <br /> project/). We used synthetic lethality data retrieved            sponse stress, transport, vesicle-mediated trans- <br /> from GRID (Breitkreutz et al. 2003) MIPS (Tong               port (P , 0.05), WGD genes uniquely enriched (P , <br /> et al. 2001; Mewes et al. 2004).                                 0.05 cumulative distribution function SSD <br />    For analysis upstream regulatory regions,               set ,0.5) conjugation protein biosynthesis <br /> aligned upstream 1000 bp paralogous pair            (Figure 2). On hand, SSD genes uniquely <br /> using method identifying duplicates,         enriched (P , 0.05 cumulative distribution func- <br /> E &#194;&#188; 1, expected faster divergence noncod-           tion WGD set ,0.5) DNA metabolism and <br /> ing sequences. The average percentage identity                protein catabolism (Figure 2). These differences in <br /> between upstream sequences paralogous pairs               enrichment sets duplicates them- <br /> calculated.                                                      selves statistically significant (see supplemental infor- <br />    For analysis transcription factor-binding sites,        mation http://www.genetics.org/supplemental/ for <br /> used Lee et al.&#226;&#8364;&#8482;s (2002) data report confidence values      complete enrichment statistics). <br /> (P-values) binding site&#226;&#8364;&#8220;gene combination. We               In addition general gene ontology enrichment <br /> calculated frequency shared transcription factor-         sets, focused addressing functional <br /> binding sites paralogs different confi-           divergence paralogs, informative of <br /> dence values cutoffs.                                         evolutionary fate duplicates. To assess subtle func- <br />    For coexpression analysis, used microarray             tional differences paralogs whole-genome <br /> data Brem Kruglyak (2005). The data                scale, used heterogeneous high-throughput func- <br /> filtered (for genes missing .50% measurements)             tional genomic data integrated using Bayesian net- <br /> and imputed using KNNimpute (Troyanskaya et al.                  work (see methods). First, predicted physical <br /> 2001) missing values, gene replicates         interaction partners paralog. As proteins that <br /> were averaged together.                                          similar functions share interaction partners ( Jacq <br />    Functional clustering: To perform global compari-           2001; Brun et al. 2003), analysis allowed assess <br /> son WGD SSD sets respect           degree functional similarity paralogous pairs. <br /> gene ontology annotation, obtained GO annotations             Second, predicted functionally related proteins for <br /> (Ashburner et al. 2000) SGD (Cherry et al.              paralog basis Bayesian network. Such <br />                                           Functional Analysis Gene Duplications                                      937 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 2.&#226;&#8364;&#8221;Distribution GO slim <br />                                                                                     biological process annotations. Differ- <br />                                                                                     ential enrichment GO slim biological <br />                                                                                     process categories WGD SSD <br />                                                                                     genes shown. The graph represents <br />                                                                                     enrichment (in cumulative distribu- <br />                                                                                     tion function) set genes in <br />                                                                                     GO term, comparison ge- <br />                                                                                     nome average, darker shading rep- <br />                                                                                     resenting higher enrichment. <br />  <br />  <br />  <br />  <br /> functional relationship represents likelihood         tion prospect duplicate pairs undergo <br /> two proteins involved biological pro-          decelerated evolution initial acceleration ( Jordan <br /> cesses. The frequency shared interaction partners           et al. 2004). <br /> and shared functional relationships significantly             Our results WGD paralogs more <br /> higher WGD paralogs compared SSD                similar protein&#226;&#8364;&#8220;protein interactions <br /> paralogs result robust different con-      functional relationships different levels of <br /> fidence levels used cutoff Bayesian predic-      sequence similarity (Figure 4A, P , 0.05 percent- <br /> tions (Figure 3A, P , 0.01 cutoffs Figure 3B,     age identity .25% Figure 4B, P , 0.05 when <br /> P , 0.002 cutoffs).                                   percentage identity .17%). The robustness the <br />    Propensity share protein-interaction partners        result sequence divergence level indicates <br /> functional relationships intrinsic WGD paralogs          tendencies possibly differences <br /> and independent sequence divergence level: In-              selection pressure duplication (Davis and <br /> terpretation result complicated         Petrov 2005). In addition, relationship between <br /> different times duplication events: WGD         functional relationships sequence similarity ap- <br /> duplicates arose specific time point SSD      pears linear SSD set, WGD set, <br /> paralogs heterogeneous time divergence.        45% identity, obvious decrease the <br /> The saturation dS prevented grouping           paralogous sequences diverge (Figure 4B). Although <br /> the paralogs according number synonymous sub-            trends necessarily indicate biological rela- <br /> stitutions. Thus, grouped paralogous pairs according        tionship, result does suggest using a <br /> to divergence level ORF amino acid sequence         mixture sets (for example, Brun et al. <br /> (see methods). Although sequence divergence does               2003 study did identify relationship) may <br /> not directly represent time duplication,          establish biased result miss characteristics specific <br /> shown varying divergences WGD set,          WGD SSD paralogs (see discussion). <br /> stratification allowed identify trends            A previous study suggested compared to <br /> robust scenarios covering wide range diver-        orthologous nonduplicated genes, evolutionary rate <br /> gence levels. This analysis avoids major complica-      difference WGD SSD duplicates results <br />  938                                        Y. Guan, M. J. Dunham O. G. Troyanskaya <br />  <br />  <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;Frequency shared interaction partners and <br /> functional relationships predicted Bayesian network             Figure 4.&#226;&#8364;&#8221;Propensity sharing interaction partners and <br /> various confidence levels. We predicted interaction partners         functional relationships paralogs sequence di- <br /> and functionally related proteins paralog            vergence levels. (A) Sharing interaction partners between <br /> basis Bayesian analysis diverse genomic data. Then           paralogs predicted Bayesian network basis of <br /> we calculated percentage shared interaction partners/         evidence protein&#226;&#8364;&#8220;protein interactions. (B) Sharing func- <br /> functional relationships paralogs total num-        tional relationships paralogs predicted the <br /> ber interaction partners/functional relationships          Bayesian network predicting functional relationships. For <br /> pair. (A) A Bayesian network integrating evidence physical       level sequence divergence, WGD paralogs <br /> interactions used predict interaction partners. (B) A         likely share protein&#226;&#8364;&#8220;protein interaction partners <br /> Bayesian network integrating diverse genomic data used           functional relationships. The linear relationship between <br /> to predict broad functional relationships. The WGD group             sequence divergence shared functional relationships is <br /> shows substantially higher percentage shared interaction        evident SSD duplicates (R 2 &#194;&#188; 0.8688). In WGD set, <br /> partners functional relationships Bayesian con-   45% sequence identity linear relationship not <br /> fidence levels. Fluctuations WGD graph likely        observable. <br /> due variations availability different experimental data <br /> sets served input Bayesian analysis. <br />  <br />                                                                      reciprocal best hits (supplemental Figure S3, C D, at <br /> mainly overenrichment ribosomal genes                 http://www.genetics.org/supplemental/). The results <br /> the WGD set (Davis Petrov 2005). To assess                  remain robust, suggesting differences <br /> hypothesis, removed pairs ribosomal genes                 WGD SSD sets caused potential <br /> both sets repeated analysis functional                    biases larger gene families SSD set. <br /> divergence. We observed similar pattern, sug-                   Another concern analysis difference <br /> gests trend functionally similar              observed artifact caused differ- <br /> WGD paralogs general effect en-                 ences numbers interaction partners WGD <br /> richment ribosomal genes (see supplemental Figure                 SSD genes. However, likely, the <br /> S3, A B, http://www.genetics.org/supplemental/).              frequency shared interaction partners does ap- <br /> To control differences functional                 pear depend total number interaction part- <br /> enrichment sets paralogs, ran-                 ners (supplemental Figure S5 http://www.genetics. <br /> domly selected subsets SSD pairs GO                 org/supplemental/). In addition, SSD set, lower <br /> slim molecular function annotation distribution               sequence similarity paralogs correlates a <br /> WGD set repeated analysis. The dif-                    higher number interactions, indicating func- <br /> ferences remain normalization func-               tional divergence caused generation new interac- <br /> tional coverage groups paralogs                   tion partners. <br /> (supplemental Figure S4 http://www.genetics.org/                     Compensation WGD paralogs: Because the <br /> supplemental/). We repeated analysis                  WGD paralogs tend similar function suggested <br />                                           Functional Analysis Gene Duplications                                          939 <br />  <br /> by shared interactions functional relationships, we <br /> reasoned high frequency of <br /> compensation. This predict that, first, WGD pa- <br /> ralogs dispensable partner <br /> can functions. Second, high frequency <br /> of compensation result higher rate syn- <br /> thetic lethality WGD paralogs. <br />    The overall proportion essential genes less <br /> (p , 10 300) WGD set (4.15%) SSD set <br /> (18.2%) (the genome average 18.9%), SSD <br /> paralogs lower sequence similarity likely <br /> to essential genes (Figure 5A). Interestingly, the <br /> relationship sequence similarity frequency <br /> of essential genes evident WGD set. We <br /> found similar results removal ribosomal <br /> gene pairs (supplemental Figure S6A http://www. <br /> genetics.org/supplemental/). The relationship between <br /> sequence similarity fitness effect different be- <br /> tween duplicate sets, indicates con- <br /> tradictory conclusions drawn examining <br /> either mixture sets (see discussion). <br />    Second, predict frequency synthetic <br /> lethality paralogs higher WGD <br /> set SSD set, deletion of the <br /> paralogs expected abolish common function. <br /> Indeed, average synthetic lethality frequency the <br /> WGD set 14% compared 3.7% SSD set (p , <br /> 10 300). Synthetic lethality low varying sequence       Figure 5.&#226;&#8364;&#8221;Patterns essentiality synthetic lethality of <br /> divergence levels SSD set, generally       duplicates sequence divergence levels. (A) The per- <br />                                                                centage essential genes WGD SSD sets. Fre- <br /> high moderate high sequence similarity the <br />                                                                quency essentiality SSD duplicate genes increases <br /> WGD duplicates (Figure 5B). Interestingly, decreas-        paralogous sequences diverge. In contrast, essenti- <br /> ing frequency compensation duplicate                ality rate stays low relatively constant level WGD <br /> genes sequences diverge observable WGD            genes. The SSD duplicates generally higher propor- <br /> set SSD set. To control potential           tion essential genes high sequence similarity <br />                                                                (.60%) bins, include 11% SSD set only. (B) <br /> functional bias synthetic lethality results WGD <br />                                                                The percentage synthetic lethal pairs WGD SSD sets. <br /> and SSD pairs, randomly selected SSD pairs         The synthetic lethality rate generally higher WGD paral- <br /> same GO slim biological process distribution            ogs, suggests compensation functional conserva- <br /> WGD genes repeated analysis. As shown               tion paralogs. The synthetic lethality proportion <br /> supplemental Figure S6C (http://www.genetics.org/              decreases paralogous sequences diverge WGD <br />                                                                set, trend observable SSD set. <br /> supplemental/), results robust functional <br /> composition differences sets. In addition, to <br /> control potential complicating factor the <br /> above differences caused larger number            sequence perspective, up- <br /> multigene families SSD set, repeated             stream regulatory regions diverged between <br /> analysis reciprocal best hits. The result (supple-        WGD paralogs level ORF sequence <br /> mental Figure S6, D E) robust              divergence (Figure 6A, P &#194;&#188; 0.01). Such result con- <br /> modification.                                                  sistent removal ribosomal genes (Figure 6B, <br />    Expression pattern diverged             P , 0.005). Second, used Lee et al. (2002) data <br /> WGD paralogs: Population genetics predicts              measuring transcription factor binding ChIP&#226;&#8364;&#8220;chip <br /> duplicate copy entirely redundant            different ORF sequence divergence <br /> maintained genome long time,            levels, transcription factor-binding sites <br /> cases concerted evolution larger         common WGD paralogs (Figure 6C), a <br /> of gene product beneficial (Zhang 2003). As WGD             similar result removal ribosomal genes <br /> genes similar function compared SSD         (Figure 6D). This result robust different cutoff <br /> set, suspect partition function            levels (supplemental Figure S7 http://www.genetics. <br /> achieved expression level. We approached           org/supplemental/, P , 1.0 30 cutoffs) used for <br /> possibility examining lines evidence. First,       identifying transcription factor-binding sites the <br />  940                                  Y. Guan, M. J. Dunham O. G. Troyanskaya <br />  <br /> ChIP&#226;&#8364;&#8220;chip data. These suggest      2005). This experiment uniquely designed find <br /> level sequence similarity, regulatory sequences    genetically segregating determinants gene expres- <br /> WGD paralogs diverge SSD              sion level physiological ones. High correla- <br /> paralogs.                                                    tion gene expression indicate shared <br />    Third, analyzed similarity gene expression      regulators gene expression lower correlation <br /> using microarray data set transcripts        imply different regulators utilized. The <br /> treated quantitative traits (Brem Kruglyak            expression correlation result congruence the <br />                                                              sharing transcription factors (Figure 7A, P , 0.02 for <br />                                                              bins .25% sequence similarity). The lower correla- <br />                                                              tion WGD paralogs pronounced <br />                                                              removal ribosomal genes (Figure 7B, P &#194;&#188; <br />                                                              0.024). Our result suggests coding sequence diver- <br />                                                              gence expression divergence duplicate <br />                                                              genes correlated SSD set WGD <br />                                                              set, reconciles previous disputes (Wagner 2000b; <br />                                                              Gu et al. 2002, 2005; Zhang et al. 2004) relation- <br />                                                              ship (see discussion). <br />  <br />  <br />  <br />                                                                                     DISCUSSION <br />                                                                 Our genome-scale analysis provided systematic discri- <br />                                                              mination WGD SSD sets respect the <br />                                                              tendency functional conservation/divergence <br />                                                              paralogs. Independent sequence diver- <br />                                                              gence, WGD paralogs general likely to <br />                                                              share physical protein-interaction partners func- <br />                                                              tional relationships basis integrated functional <br />                                                              genomic evidence. This major tendency similar <br />                                                              function supported higher rate synthetic <br />                                                              lethality WGD paralogous pairs. However, simi- <br />                                                              larity compensation function does lead to <br />                                                              complete redundancy, compared SSD paral- <br />                                                              ogs, WGD paralogs diverged expression pat- <br />                                                              terns upstream regulatory regions, implying their <br />                                                              role fine tuning expression level. As discussed later, <br />                                                              results global study reconcile pre- <br />                                                              viously inconsistent observations selec- <br />                                                              tion varying sets duplicates suggest different <br />                                                              tendencies evolutionary fate consequences be- <br />                                                              tween whole-genome smaller-scale duplicates. <br />  <br />  <br />  <br />  <br />                                                                 Figure 6.&#226;&#8364;&#8221;Divergence upstream regulatory region <br />                                                              transcription factor-binding sites paralogs across <br />                                                              sequence divergence. (A) Alignment upstream 1000 kb <br />                                                              pairs. The nonoverlapping percentage identity of <br />                                                              upstream 1000 kb (E &#194;&#188; 1) calculated average <br />                                                              taken. The upstream regions background <br />                                                              duplicate pair generally align better, especially high per- <br />                                                              centage identity groups. Such result accordance with <br />                                                              expression pattern WGD pairs diverge more. (B) <br />                                                              Alignment upstream 1000 kb pairs re- <br />                                                              moval ribosomal genes. (C) Frequency shared transcrip- <br />                                                              tion factor-binding sites. WGD paralogs significantly <br />                                                              weaker sharing transcription factor-binding sites. (D) Fre- <br />                                                              quency shared transcription factor-binding sites between <br />                                                              paralogs removal ribosomal genes. <br />                                             Functional Analysis Gene Duplications                                      941 <br />  <br />                                                                  supplemental/). It emphasized the <br />                                                                  observed patterns subject alternative expla- <br />                                                                  nations. It possible retaining functionally <br />                                                                  similar genes caused dosage effect (Zhang <br />                                                                  2003; Kondrashov Kondrashov 2006). <br />                                                                     Another reason keeping similar genes the <br />                                                                  genome suggested lower level coexpression <br />                                                                  sharing upstream regulatory regions between <br />                                                                  WGD paralogs. This suggests WGD pairs, while <br />                                                                  showing lower frequency essentiality, be <br />                                                                  beneficial environmental experimen- <br />                                                                  tal conditions. For example, Ser/Thr kinases DBF2 and <br />                                                                  DBF20 77% identical sequence, highly function- <br />                                                                  ally related (0.99 confidence functional relationship <br />                                                                  predicted Bayesian network), synthetically <br />                                                                  lethal. However, expression correlation coefficient <br />                                                                  negative basis data set Brem and <br />                                                                  Kruglyak (2005). Functionally related WGD paralo- <br />                                                                  gous pairs negative expression correlations not <br />                                                                  uncommon: 74% paralogs negative correlation <br />                                                                  predicted functionally related Bayesian <br />                                                                  method (at confidence level higher baseline <br />                                                                  prior), compared genome average 1.6%. This <br />    Figure 7.&#226;&#8364;&#8221;Divergence expression pattern pa-        suggests possible role WGD duplicates ex- <br /> ralogs sequence divergence. (A) The correlation ex-    pression modulation. Another example phenom- <br /> pression patterns paralogs. Expression patterns          enon WGD sulfate permease paralogs SUL1 and <br /> between WGD paralogs highly diverged especially af-      SUL2, share 65% identity, functionally related <br /> ter removal ribosomal genes, indicating role     basis integrated genomic data, show <br /> finely modulating expression levels. The expression correla- <br /> tion coefficient random genes data set        phenotypic enhancement (Cherest et al. 1997). How- <br /> is average 0.003. (B) The correlation expression pat-      ever, Pearson correlation coefficient 0.163, <br /> terns paralogs removal ribosomal genes.         compared 0.288 SSD group se- <br />                                                                  quence divergence level. Additionally, expression of <br />                                                                  SUL2, SUL1, control SUL3 <br />                                                                  (Cherest et al. 1997). Our result suggests diver- <br />    It explore biological reasons       gence expression level significant the <br /> the functional differences whole-genome              WGD duplicates SSD set. <br /> and smaller-scale duplicates. The higher propen-               We role sequence divergence in <br /> sity WGD paralogs synthetically lethal suggests         functional attributes duplicate genes differ- <br /> that retention copies whole-genome              ent WGD SSD sets. This difference cause <br /> duplication offers robustness genome. Such              studies based different subsets WGD SSD data <br /> result readily agreement global functional        arrive different conclusions. In fact, global ana- <br /> analysis. Besides synthetic lethality, genetic robustness        lyses reconciled previous disputes essentiality <br /> can supported paralogs compen-             expression divergence paralogs relation to <br /> sate expression. The WGD paralogs              sequence divergence. First, investigated relation- <br /> NHP6A NHP6B, share 89% sequence iden-                  ship sequence similarity duplicate genes and <br /> tity, example phenomenon. Deletion             fitness effect null mutation. The conclusion that <br /> NHP6A leads threefold increase NHP6B synthesis           relationship established (Wagner 2000a) <br /> while extra copy NHP6A reduces NHP6B expres-               based 45 duplicate genes having paralog a <br /> sion twofold (Kolodrubetz et al. 2001). Changes           syntenic block yeast chromosome, namely, <br /> NHP6B gene copy number cause similar mod-               duplicates WGD. The opposite conclusion a <br /> erate changes NHP6A expression (Kolodrubetz                   high correlation established (Gu et al. 2003) was <br /> et al. 2001). However, enhanced expression               based genomewide group 1147 duplicated pairs, <br /> one gene response deletion paralog          corresponding combination WGD SSD <br /> be general pattern WGD SSD                sets used study. Our result (Figure 5A) confirms, <br /> sets basis analysis genomewide ex-              qualifies, conclusions showing cor- <br /> pression data deletion strains (Hughes et al. 2000)           relation sequence similarity essentiality, <br /> (supplemental Figure S8 http://www.genetics.org/              absent WGD set suggested Wagner <br />  942                                    Y. Guan, M. J. Dunham O. G. Troyanskaya <br />  <br /> (2000a), observed SSD set Gu et al.        ilarity (Figure 1) follow general trends pro- <br /> (2003). Second, reconcile previous dispute             posed analysis. However, high sequence <br /> whether coding sequence divergence expression              divergence, WGD paralogs diverge function <br /> divergence duplicate genes coupled. A              quickly, indicated shared interaction partners <br /> previous study based 376 WGD paralogous pairs               functional relationships (Figure 4, A B). The <br /> (after removal ribosomal genes) signifi-           frequency synthetic lethal pairs drops substan- <br /> cant correlation protein-sequence divergence           tially (Figure 5B), indicating paralogous pairs lose <br /> and regulatory region/expression divergence (Wagner            functional compensation sequences diverge. <br /> 2000b), agreement result based            Another interesting subset duplicates identified <br /> on WGD set (Figure 6, A B). On hand,         Kellis et al. (2004a), suggested group &#226;&#8364;&#732;&#226;&#8364;&#732;fast&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> a positive correlation expression divergence           WGD pairs showing accelerated protein evolution. The <br /> and nonsynonymous divergence (Gu et al. 2002, 2005)            relatively small number fast WGD pairs prevented us <br /> or age duplicates (Zhang et al. 2004) estab-        thorough analysis partitioning according to <br /> lished basis paralogs identified genome-         sequence divergence. Nevertheless, the <br /> wide, corresponding combination WGD                general enrichment biological processes different <br /> SSD duplicates study. Our global comparative            fast set rest WGD paralogs <br /> result WGD SSD sets reconciles different          (supplemental Figure S9 http://www.genetics.org/ <br /> results (Figure 6, A B, Figure 7) suggests        supplemental/). Thus, study provides global <br /> the correlation expression upstream reg-            comparison functional divergence duplicates that <br /> ulatory region divergence coding sequence diver-           arose whole-genome duplication small-scale <br /> gence evident SSD set, relatively weak    duplications, specific subgroups duplicates ex- <br /> WGD set.                                                       hibit different behaviors. <br />    Similarly, caution taken drawing                We envision directions ad- <br /> conclusions relationship physical            dress differences whole-genome small- <br /> interaction, functional relationship, synthetic lethal-     scale duplications. First, present study focuses on <br /> ity sequence divergence. For example,            behavior duplicated genes sequences of <br /> conclusion simple relationship estab-         paralogs diverge. Several recent studies identified <br /> lished sequence identity functional simi-          asymmetric evolution based WGD duplicates com- <br /> larity (Baudot et al. 2004) reasonable set       pared nonduplicated orthologs (Kellis et al. 2004a,b; <br /> WGD paralogs, linear relationship           Fares et al. 2006; Kim Yi 2006, etc.). The relative <br /> in SSD set (Figure 4B). Using mixture         strength asymmetric evolution WGD vs. SSD <br /> sets (for example, Brun et al. 2003 used 10 WGD 10         duplicates interesting investigate. Second, <br /> SSD duplicates) fails establish relation-      results suggest functional divergence low align- <br /> ship, probably interference WGD                ment. However, discriminate neo- <br /> duplicates. The decoupling sequence divergence              functionalization subfunctionalization. Studies on <br /> and functional divergence WGD set especially         individual pairs using interaction data and/or moving- <br /> worth considering. A recent study (Fares et al. 2006)          window analysis informative. In addition, a <br /> suggests positive selection detected        recent study reports network partition WGD <br /> one WGD paralogs compared               genes (Conant Wolfe 2006). Investigation the <br /> nonduplicated orthologous sequences. However,               presence partition network SSD <br /> found positive selection sequence level       set provide insight whole-genome <br /> cannot connected divergence paralogs             duplication likely result duplication an <br /> on functional level. Most positively selected       entire pathway expression network. Finally, pre- <br /> paralogs Fares et al. identified, despite large     sent study compares WGD set SSD set by <br /> sequence divergence, highly functionally related           stratifying according sequence similarity. Estimating <br /> and, cases, synthetically lethal. For example,      age duplicates clarify di- <br /> both SUMO ligases SIZ1 SIZ2I positively         vergence variation caused age specific duplicate <br /> selected (Fares et al. 2006) in-paralog diver-         set. The growing amounts large-scale genomic data <br /> gence large (32% identity). Nevertheless,          enable comprehensive comparison whole- <br /> predicted share function relationship (0.99 confi-        genome duplication small-scale duplication a <br /> dence predicted Bayesian network)               variety organisms. <br /> synthetically lethal (see supplemental information             We thank Chad Myers Matthew Hibbs insightful discussions, <br /> http://www.genetics.org/supplemental/).                        helping data, processing expression data, and <br />    It emphasized results              correction manuscript. We appreciate constructive sugges- <br />                                                                tions anonymous reviewers. This research partially <br /> represent behavior majority WGD dupli-           supported National Institutes Health grant R01 GM071966 and <br /> cates comparison SSD duplicates. Most         National Science Foundation grant IIS-0513552 O.G.T. O.G.T. an <br /> WGD duplicates moderate high sequence sim-             Alfred P. Sloan Research Fellow. <br />                                                    Functional Analysis Gene Duplications                                                    943 <br />  <br />                       LITERATURE CITED                                     Kellis, M., N. Patterson, B. Birren, B. Berger E. S. Lander, <br />                                                                                 2004b Methods comparative genomics: genome correspon- <br /> Altschul, S. F., W. Gish, W. Miller, E. W. Myers D. J. Lipman,              dence, gene identification regulatory motif discovery. <br />     1990 Basic local alignment search tool. J. Mol. Biol. 215: 403&#226;&#8364;&#8220;             J. Comput. Biol. 11: 319&#226;&#8364;&#8220;355. <br />     410.                                                                   Kim, S. H., S. V. Yi, 2006 Correlated asymmetry sequence and <br /> Ashburner, M., X. A. Ball, J. A. Blake, D. Botstein, H. Butler                  functional divergence duplicate proteins Saccharomy- <br />     et al., 2000 Gene ontology: tool unification biology.            ces cerevisiae. Mol. Biol. Evol. 23(5): 1068&#226;&#8364;&#8220;1075. <br />     The Gene Ontology Consortium. Nat. Genet. 25: 25&#226;&#8364;&#8220;29.                   Kolodrubetz, D., M. Kruppa A. Burgum, 2001 Gene dosage <br /> Baudot, A., B. Jacq C. Brun, 2004 A scale functional diver-              affects expression duplicated NHP6 genes Saccharo- <br />     gence yeast duplicated genes revealed analysis              myces cerevisiae. Gene 272: 93&#226;&#8364;&#8220;101. <br />     protein-protein interaction network. Genome Biol. 5: R76.              Kondrashov, F. A., A. S. Kondrashov, 2006 Role selec- <br /> Breitkreutz, B. J., C. Stark M. Tyers, 2003 The GRID: Gen-              tion fixation gene duplications. J. Theor. Biol. 239: 141&#226;&#8364;&#8220; <br />     eral Repository Interaction Datasets. Genome Biol. 4(3): R23.           151. <br /> Brem, R. B., L. Kruglyak, 2005 The landscape genetic com-           Kondrashov, F. A., I. B. Rogozin, Y. I. Wolf E. V. Koonin, <br />     plexity 5,700 gene expression traits yeast. Proc. Natl.           2002 Selection evolution gene duplications. Genome <br />     Acad. Sci. USA 102(5): 1572&#226;&#8364;&#8220;1577.                                           Biol. 3(2): RESEARCH0008. <br /> Brun, C., A. Gue&#194;&#180;noche B. Jacq, 2003 Approach func-             Lee, T. I., N. J. Rinaldi, F. Robert, D. T. Odom, Z. Bar-Joseph et al., <br />     tional evolution duplicated genes Saccharomyces cerevisiae            2002 Transcriptional regulatory networks Saccharomyces cerevi- <br />     using new classification method based protein-protein inter-           siae. Science 298: 799&#226;&#8364;&#8220;804. <br />     action data. J. Struct. Funct. Genomics 3: 213&#226;&#8364;&#8220;224.                    Li, W. H., J. Yang X. Gu, 2005 Expression divergence between <br /> Byrne, K. P., K. H. Wolfe, 2005 The Yeast Gene Order Browser:               duplicate genes. Trends Genet. 21(11): 602&#226;&#8364;&#8220;607. <br />     combining curated homology syntenic context reveals gene           Lynch, M., J. S. Conery, 2000 The evolutionary fate con- <br />     fate polyploidy species. Genome Res. 15(10): 1456&#226;&#8364;&#8220;1461.                  sequences duplicate genes. Science 290: 1151&#226;&#8364;&#8220;1155. <br /> Cherest, H., J. C. Davidian, D. Thomas, V. Benes, W. Ansorge et al.,       Lynch, M., A. Force, 2000 The probability duplicate gene <br />     1997 Molecular characterization high affinity sulfate                preservation subfunctionalization. Genetics 154: 459&#226;&#8364;&#8220;473. <br />     transporters Saccharomyces cerevisiae. Genetics 145: 627&#226;&#8364;&#8220;635.       Mewes, H. W., C. Amid, R. Arnold, D. Frishman, U. Guldener et al., <br /> Cherry, J. M., C. Adler, C. Ball, S. A. Chervitz, S. S. Dwight et al.,          2004 MIPS: analysis annotation proteins whole <br />     1998 SGD: Saccharomyces genome database. Nucleic Acids Res.                 genomes. Nucleic Acids Res. 32: D41&#226;&#8364;&#8220;D44. <br />     26(1): 73&#226;&#8364;&#8220;79.                                                          Myers, C. L., D. Robson, A. Wible, M. A. Hibbs, C. Chiriac et al., <br /> Conant, G. C., K. H. Wolfe, 2006 Functional partitioning                    2005 Discovery biological networks diverse functional <br />     yeast co-expression networks genome duplication. PLoS              genomic data. Genome Biol. 6(13): R114. <br />     Biol. 4(4): e109.                                                      Myers, C. L., D. R. Barrett, M. A. Hibbs, C. Huttenhower and <br /> Davis, J. C., D. A. Petrov, 2005 Do disparate mechanisms                 O. G. Troyanskaya, 2006 Finding function: evaluation meth- <br />     duplication add similar genes genome? Trends Genet.                  ods functional genomic data. BMC Genomics 7(1): 187. <br />     21: 548&#226;&#8364;&#8220;551.                                                           Nadeau, J. H., D. Sankoff, 1997 Comparable rates gene <br /> Dempster, A. P., N. M. Laird D. B. Rubin, 1977 Maximum like-                loss functional divergence genome duplications early <br />     lihood incomplete data EM algorithm. J. R. Stat. Soc.              vertebrate evolution. Genetics 147: 1259&#226;&#8364;&#8220;1266. <br />     Methodol. 39: 1&#226;&#8364;&#8220;38.                                                    Nowak, M. A., M. C. Boerlijst, J. Cooke J. M. Smith, <br /> Dietrich, F. S., S. Voegeli, S. Brachat, A. Lerch, K. Gates et al.,             1997 Evolution genetic redundancy. Nature 388: 167&#226;&#8364;&#8220;171. <br />     2004 The Ashbya gossypii genome tool mapping              Ohno, S., 1970 Evolution Gene Duplication. Springer-Verlag, New York. <br />     ancient Saccharomyces cerevisiae genome. Science 304(5668): 304&#226;&#8364;&#8220;       Sankoff, D., 2001 Gene genome duplication. Curr. Opin. Genet. <br />     307.                                                                        Dev. 11: 681&#226;&#8364;&#8220;684. <br /> Fares, M. A., K. P. Byrne K. H. Wolfe, 2006 Rate asymmetry             Seoighe, C., K. H. Wolfe, 1999 Yeast genome evolution the <br />     genome duplication causes substantial long-branch attrac-             post-genome era. Curr. Opin. Microbiol. 2: 548&#226;&#8364;&#8220;554. <br />     tion artifacts phylogeny Saccharomyces species. Mol. Biol.   Stoltzfus, A., 1999 On possibility constructive neutral evolu- <br />     Evol. 23(2): 245&#226;&#8364;&#8220;253.                                                       tion. J. Mol. Evol. 49: 169&#226;&#8364;&#8220;181. <br /> Force, A., M. Lynch, F. B. Pickett, A. Amores, Y. L. Yan                   Taylor, J. S., J. Raes, 2004 Duplication divergence: evo- <br />     et al., 1999 Preservation duplicate genes complementary,              lution new genes old ideas. Annu. Rev. Genet. 38: 615&#226;&#8364;&#8220; <br />     degenerative mutations. Genetics 151: 1531&#226;&#8364;&#8220;1545.                            643. <br /> Gu, X., 2003 Evolution duplicate genes versus genetic robustness        Tong, A. H. Y., M. Evangelista, A. B. Parsons, H. Xu, G. D. Bader <br />     null mutations. Trends Genet. 19: 354&#226;&#8364;&#8220;356.                          et al., 2001 Systematic genetic analysis ordered arrays of <br /> Gu, X., Z. Zhang W. Huang, 2005 Rapid evolution ex-    <br /> </body></html>