<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 21 No. 14 pp. 3841&#194;&#177;3851, 2002 <br />  <br />  <br /> tadA, essential tRNA-speci&#194;&#174;c adenosine <br /> deaminase Escherichia coli <br />  <br />  <br />  <br /> Jeannette Wolf, Andre&#195;&#8218; P.Gerber1                                2000). RNA editing represents important <br /> Walter Keller2                                                      mechanism increasing genetic diversity in <br />                                                                     eukaryotes. In cases, pre-mRNA editing requires <br /> Department Cell Biology, Biozentrum, University Basel, <br /> CH-4056 Basel, Switzerland                                          double-stranded RNA (dsRNA) structures, are <br /> 1 <br />                                                                     formed exonic sequences encompassing the <br />  Present address: Department Biochemistry, Stanford University   editing site downstream intronic sequences. <br /> School Medicine, Stanford, CA 94305-5307, USA <br />                                                                        RNA editing pre-mRNAs catalyzed adenosine <br /> 2 <br />  Corresponding author                                               deaminases acting RNA (ADARs; Bass et al., 1997). <br /> e-mail: walter.keller@unibas.ch                                     ADARs common modular organization consisting of <br />                                                                     dsRNA-binding domains (dsRBD) a <br /> We report characterization tadA, &#194;&#174;rst pro-               catalytic deaminase domain containing Zn2+-chelating <br /> karyotic RNA editing enzyme identi&#194;&#174;ed.                        residues proton-shuf&#194;&#175;ing glutamate (reviewed in <br /> Escherichia coli tadA displays sequence similarity               Bass, 1997; Maas Rich, 2000; Gerber Keller, 2001). <br /> the yeast tRNA deaminase subunit Tad2p. Recom-                         Inosine present mRNAs, in <br /> binant tadA protein forms homodimers suf&#194;&#174;-                   tRNAs. It &#194;&#174;rst tRNAAla yeast (Holley <br /> cient site-speci&#194;&#174;c inosine formation wobble              et al., 1965). Eukaryotic tRNAAla contains I two <br /> position (position 34) tRNAArg2, tRNA                   positions: wobble position anticodon (position <br /> having modi&#194;&#174;cation prokaryotes. With                    34) derivative N1-methylinosine (m1I) at <br /> exception yeast tRNAArg, eukaryotic tRNA                position 37, 3&#194;&#162; adjacent anticodon (Holley et al., <br /> substrates modi&#194;&#174;ed tadA. How-                   1965; reviewed Grosjean et al., 1996). In eukaryotes, <br /> ever, arti&#194;&#174;cial yeast tRNAAsp, carries                 seven tRNAs contain I position 34, in <br /> anticodon loop yeast tRNAArg, bound modi-                 prokaryotes plant chloroplasts tRNAArg2 contains <br /> &#194;&#174;ed tadA. Moreover, tRNAArg2 minisubstrate                     modi&#194;&#174;cation. As pre-mRNAs, I tRNAs the <br /> containing anticodon stem loop suf&#194;&#174;cient                 product hydrolytic deamination genomically encoded <br /> for speci&#194;&#174;c deamination tadA. We                       A (Auxilien et al., 1996). m1I position 37 formed a <br /> nucleotides positions 33&#194;&#177;36 suf&#194;&#174;cient inosine            two-step reaction. First, A deaminated I, is <br /> formation mutant Arg2 minisubstrates. The anti-                  methylated methyltransferase (Grosjean et al., <br /> codon major determinant tadA substrate                1996; Bjo&#195;&#710;rk et al., 2001). The genomes Saccharomyces <br /> speci&#194;&#174;city. Finally, tadA essential              cerevisiae prokaryotes encode classical ADAR <br /> gene E.coli, underscoring critical function               proteins, but, based sequence homology ADARs, a <br /> inosine wobble position prokaryotes.                      yeast protein identi&#194;&#174;ed contains deaminase <br /> Keywords: adenosine deaminase/Escherichia coli/                     domain lacks known RNA-binding motif. This <br /> inosine/RNA editing/tRNAArg2                                        protein catalyzes deamination A position 37 in <br />                                                                     yeast tRNAAla named adenosine <br />                                                                     deaminase acting tRNA 1 gene tRNA-speci&#194;&#174;c <br /> Introduction                                                        adenosine deaminase 1 (scADAT1/TAD1; Gerber et al., <br /> The nucleotide inosine (I) observed viral               1998). Tad1p speci&#194;&#174;cally targets A37 tRNAAla. <br /> transcripts eukaryotic mRNAs. In known cases, I             Mutations affecting three-dimensional structure of <br /> results deamination adenosine (A), process            tRNA <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> anticodon loop abolish <br /> termed RNA editing. Because I read guanosine (G)           conversion A I vitro (Gerber et al., 1998). Whereas <br /> the translational machinery (Basilio et al., 1962), RNA             A37 unmodi&#194;&#174;ed Dtad1 strain, modi&#194;&#174;cation A34 <br /> editing change codon speci&#194;&#174;city               unaffected tRNAs tested, suggesting I <br /> amino acid sequence encoded protein, resulting            formation positions catalyzed different <br /> multiple protein products different biological func-           enzymes. Whereas I wobble position (I34) crucial <br /> tion single mRNA precursor. In mammals,                  allow decoding codons single tRNA <br /> example, mRNA precursors (pre-mRNAs) coding                 (Crick, 1966), function m1I37 clear. It was <br /> subunits glutamate-gated ion channel receptors (GluRs)           postulated modi&#194;&#174;cation prevent translational <br /> and serotonin receptor subunit 5-HT2C edited                frameshifts improve translational &#194;&#174;delity (Bjo&#195;&#710;rk et al., <br /> (reviewed Seeburg et al., 1998; Gott Emeson, 2000;           1989). ADAT1 proteins cloned <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Maas Rich, 2000; Gerber Keller, 2001; Keegan                human (Maas et al., 1999), mouse (Maas et al., 2000) and <br /> et al., 2001). Editing detected Drosophila              D.melanogaster (Keegan et al., 2000). <br /> melanogaster, Caenorhabditis elegans hepatitis                  The tRNA adenosine deaminase speci&#194;&#174;cally <br /> delta virus (Polson et al., 1996; Smith et al., 1996; Morse         deaminates A34 partially puri&#194;&#174;ed yeast <br /> and Bass, 1999; Semenov Pak, 1999; Hanrahan et al.,             extracts (Auxilien et al., 1996) identi&#194;&#174;ed by <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                                  3841 <br />  J.Wolf, A.P.Gerber W.Keller <br />  <br />  <br />  <br />  <br /> Fig. 1. Bacterial genomes encode protein (tadA) related Tad2p S.cerevisiae. Multiple sequence alignment E.coli tadA, yeast Tad2p puta- <br /> tive tadA sequences different bacteria. Residues conserved &gt;83% proteins shown black, similar amino acids gray. The three <br /> putative Zn2+-chelating residues (diamond) glutamate thought mediate proton transfer (bullet) marked. The position <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> (D64E) tadA mutant NWL37 indicated asterisk. The alignment generated Clustal_W software European <br /> Bioinformatics Institute (Thompson et al., 1994). <br />  <br />  <br />  <br /> homology Tad1p (Gerber Keller, 1999). Activity                          identi&#194;&#174;cation &#194;&#174;rst prokaryotic tRNA-speci&#194;&#174;c <br /> depended correct tRNA structure                       adenosine deaminase provides insight into <br /> length nucleotide sequence anticodon loop.                      evolution deaminase family enzymes. <br /> The tRNA:A34 deaminase heterodimer sequence- <br /> related subunits named scADAT2/Tad2p scADAT3/ <br /> Tad3p. Both polypeptides contain deaminase domain                            Results <br /> that resembles cytidine deaminase (CDA)                            tadA acts speci&#194;&#174;cally tRNAArg2 <br /> superfamily (Gerber Keller, 1999), Tad2p/                         Database searches S.cerevisiae Tad2p sequence <br /> Tad3p deaminates A. Most cytidine/deoxycytidylate                              FASTA program (Pearson Lipman, 1988) <br /> deaminases catalyze deaminations mononucleotides,                           revealed E.coli open reading frame (ORF) yfhC is <br /> but known cytidine deaminases act                    34% identical Tad2p (Figure 1). <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> determine whether <br /> on RNA (CDARs). One best studied examples                            yfhC encodes adenosine deaminase, protein <br /> APOBEC1, catalyzes deamination cytidine                         overexpressed E.coli puri&#194;&#174;ed apparent <br /> to uridine apoB mRNA, resulting change                        homogeneity (Materials methods; Figure 2A B). <br /> glutamine codon stop codon (reviewed Chester                              Peak fractions &#194;&#174;nal gel &#194;&#174;ltration column were <br /> et al., 2000). Interestingly, deaminase domain Tad3p                    tested adenosine deaminase activity vitro tran- <br /> lacks glutamate that, analogous CDAs CDARs,                         scribed uniformally [a-33P]ATP-labeled tRNAArg2. <br /> is thought active center. The lack                   The recombinant protein converted A I <br /> glutamate suggests Tad3p catalytically inactive                    (Figure 2C). This activity stemmed recombinant <br /> that Tad2p catalytic subunit complex.                            protein endogenous enzyme, since <br />    Deamination tRNA limited eukaryotes,                       glutathione S-transferase (GST) puri&#194;&#174;ed E.coli or <br /> has detected Escherichia coli extracts                            eluates Ni2+&#194;&#177;agarose incubated extract from <br /> (Auxilien et al., 1996). In E.coli, known A I                      empty-vector-transformed cells activity (results not <br /> conversion RNA deamination tRNAArg2                            shown). A 2 ng aliquot recombinant yfhC protein <br /> position 34. Based sequence homology, identi&#194;&#174;ed                          formed 0.5 mol I/mol tRNA (Figure 2C, fraction 12) <br /> an E.coli protein homologous Tad2p. Here, report                         protein needed activity (Figure 2C, <br /> the identi&#194;&#174;cation characterization enzyme,                         fraction 12). This contrast yeast enzyme, which <br /> which necessary suf&#194;&#174;cient catalyze editing                      acts heterodimer composed Tad2p Tad3p <br /> reaction. tadA/ecADAT2 shows sequence homology                              (Gerber Keller, 1999). <br /> yeast Tad2p Tad3p encoded essential                              To determine site tRNAArg2 modi&#194;&#174;ed by <br /> gene, underlining vital function I34. The                          recombinant yfhC, vitro modi&#194;&#174;ed tRNA ampli&#194;&#174;ed <br />  <br /> 3842 <br />                                                                                         An essential tRNA-speci&#194;&#174;c adenosine deaminase <br />  <br />  <br />                                                                            revealed ~90% A34 deaminated I, be <br />                                                                            seen ratio G A position 34 (Figure 2D). <br />                                                                            tRNAArg2 incubated absence protein carried A at <br />                                                                            position 34 modi&#194;&#174;ed (Figure 2D). <br />                                                                            No A tRNAArg2 modi&#194;&#174;ed recombinant <br />                                                                            protein. These results showed yfhC gene encodes a <br />                                                                            protein suf&#194;&#174;cient reconstitute A34:tRNA <br />                                                                            deaminase activity vitro. Therefore, renamed yfhC <br />                                                                            tadA (for tRNA adenosine deaminase A). <br />                                                                               The speci&#194;&#174;city recombinant tadA tested by <br />                                                                            comparing activity various tRNA substrates <br />                                                                            (Figure 3A). In E.coli, tRNA substrate is <br />                                                                            known deaminated position 34 (tRNAArg2), in <br />                                                                            contrast S.cerevisiae seven different tRNAs are <br />                                                                            substrates Tad2p/Tad3p (Auxilien et al., 1996; Gerber <br />                                                                            Keller, 1999). Auxilien et al. (1996) showed not <br />                                                                            tRNAArg2, variant tRNAAsp from <br />                                                                            S.cerevisiae carrying anticodon loop yeast <br />                                                                            tRNAArg, deaminated vitro incubation with <br />                                                                            extracts E.coli. The observation be <br />                                                                            reproduced recombinant tadA, forms to <br />                                                                            0.8 mol I arti&#194;&#174;cial tRNA, suggesting the <br />                                                                            anticodon loop carries important recognition signals for <br />                                                                            enzyme (Figure 3A, lane 4). This hypothesis is <br />                                                                            supported result tadA deaminated yeast <br />                                                                            tRNAArg ef&#194;&#174;ciency E.coli tRNAArg2 <br />                                                                            (Figure 3A, lanes 10 11). These tRNAs a <br />                                                                            conserved anticodon loop sequence (Figure 3B). However, <br />                                                                            parts anticodon stem acceptor stem also <br />                                                                            conserved yeast E.coli tRNAArg, these <br />                                                                            nucleotides contribute recognition the <br />                                                                            tRNA tadA (Figure 3B). <br />                                                                               tRNAAla S.cerevisiae, Bombyx mori Homo <br />                                                                            sapiens deaminated tadA, are <br />                                                                            known contain I34 vivo (Figure 3A, lanes 2, 6 8). <br />                                                                            Notably, tRNAs substrates yeast Tad2p/ <br />                                                                            Tad3p (Figure 3A, lanes 1, 5 7; Auxilien et al., 1996; <br />                                                                            Gerber Keller, 1999). In yeast B.mori, tRNAAla <br />                                                                            positions 34 37 deaminated 2 mol I/mol tRNA <br /> Fig. 2. tadA speci&#194;&#174;cally deaminates A34 tRNAArg2. SDS&#194;&#177;                  generated incubation yeast extract <br /> polyacrylamide gel stained silver (A), western blot (B) tRNA      (Figure 3A, lanes 1 5). The substrate speci&#194;&#174;city of <br /> editing assay (C) &#194;&#174;nal gel &#194;&#174;ltration column puri&#194;&#174;cation   tadA shown UV-cross-linking experi- <br /> recombinant tadA. A 20 ml aliquot fraction separated      ments indicate binding protein tRNA <br /> 12% SDS&#194;&#177;polyacrylamide gel (A) transferred nitrocellulose <br /> membrane detection antibodies (B). The western blot           substrate. Recombinant GST&#194;&#177;tadA UV cross- <br /> probed mouse a-GST monoclonal antibody. Fraction numbers            linked natural substrate tRNAArg2 (Figure 3C, lane 2), <br /> are indicated top, molecular masses size standards    tRNAAla H.sapiens, B.mori S.cerevisiae <br /> kDa left. (C) Two microliters fractions indicated     (Figure 3C, lanes 4&#194;&#177;6). GST cross- <br /> top incubated [33P]ATP-labeled tRNAArg2 30 min at <br /> 37&#194;&#176;C. Reactions treated P1 nuclease products <br />                                                                            linked substrates (results shown). In <br /> separated one-dimensional TLC. The positions AMP, IMP            summary, showed recombinant tadA speci&#194;&#174;cally <br /> the origin indicated right. The position IMP veri&#194;&#174;ed     deaminates tRNAArg2 position 34. <br /> with unlabeled 5&#194;&#162; IMP. In lane `&#194;&#177;&apos;, tRNAArg2 incubated buffer        To determine anticodon stem loop is <br /> only. (D) Sequence analysis vitro modi&#194;&#174;ed tRNAArg2. The nucleo-      suf&#194;&#174;cient deamination tadA, tRNAArg2 mini- <br /> tide sequence surrounding anticodon shown. tRNA reactions <br /> shown (C) ampli&#194;&#174;ed RT&#194;&#177;PCR sequenced. The nucleotide          substrate containing anticodon stem loop was <br /> at position 34 shaded gray, anticodon underlined. Control:       tested vitro (Figure 4A). This minisubstrate was <br /> tRNAArg2 incubated buffer only.                                       deaminated ef&#194;&#174;ciently full-length tRNAArg2 <br />                                                                            (Figure 4B, lane 1; Table I). In majority the <br />                                                                            experiments, deamination wild-type (WT) Arg2 mini- <br /> by RT&#194;&#177;PCR sequenced. Since I <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span>-pairs C                         substrate yielded 1.2 1.4 mol I/mol tRNA, <br /> during reverse transcription, A I deamination changes                   slightly higher expected. Interestingly, when <br /> the sequence A G. tRNAArg2 incubated                          A wobble position mutated G, the <br /> recombinant yfhC contained G position 34, demon-                      minisubstrate deaminated, indicating this <br /> strating template-encoded A position                  nucleotide essential tadA recognition (Figure 4B, <br /> deaminated I. Furthermore, sequencing tRNAArg2                       lane 2). It possible second A deaminated a <br />  <br />                                                                                                                                   3843 <br />  J.Wolf, A.P.Gerber W.Keller <br />  <br />  <br />  <br />  <br /> Fig. 3. Substrate speci&#194;&#174;city tadA. (A) tRNA-editing assay tadA different tRNAs. tRNA substrates incubated 2 ng of <br /> recombinant His6-tadA (lanes 2, 4, 6, 8, 10 11), 20 ng recombinant scTad2p/scTad3p (lane 7) 40 mg S.cerevisiae total protein (lanes 1, 3 <br /> and 5). One femtomoles tRNAAla S.cerevisiae (lanes 1 2), tRNAAsp mutant S.cerevisiae (lanes 3 4), tRNAAla from <br /> B.mori (lanes 5 6), human tRNAAla (lanes 7 8), S.cerevisiae tRNAArg (lanes 9 10) tRNAArg2 E.coli (lane 11) used. Mock <br /> incubation WT yeast tRNAAsp did result I formation (result shown). All reactions incubated 1 h 30&#194;&#176;C processed as <br /> described legend Figure 2 (see Materials methods). (B) Cloverleaf structure E.coli tRNAArg2, S.cerevisiae tRNAArg and <br /> S.cerevisiae tRNAAsp anticodon loop tRNAArg. Completely conserved nucleotides nucleotides conserved purines pyrimidines <br /> between tRNAs different species shown red (Klingler Brutlag, 1993). Nucleotides conserved addition three <br /> tRNAs depicted blue. (C) UV-cross-linking experiments recombinant tadA different tRNAs. A 400 ng aliquot GST&#194;&#177;tadA (lanes 2&#194;&#177;5) <br /> and 100 fmol [33P]ATP-labeled tRNA irradiated samples treated RNaseA. Proteins separated 12% SDS&#194;&#177;polyacrylamide <br /> gel gels exposed phosphoimager screen. E.c., E.coli; S.c., S.cerevisiae; B.m., B.mori; Arg2, tRNAArg2; hAla, human tRNAAla; B.m. Ala, <br /> B.mori tRNAAla; S.c. Ala, S.cerevisiae tRNAAla. <br />  <br />  <br />  <br /> fraction WT minisubstrate. However, consider                        This substrate deaminated, lower <br /> this possibility unlikely I formed A34                    ef&#194;&#174;ciency compared WT RNA (Figure 4B, lane 5; <br /> was mutated deamination second A                        Table I), 3 nt loop suf&#194;&#174;cient tadA <br /> depend presence A34. Although anticodon                         activity (Figure 4B, lane 6). A minisubstrate a <br /> arm Arg2 serves substrate tadA,                  random loop sequence correct anticodon not <br /> differences site speci&#194;&#174;city full-length                        deaminated, showing anticodon only <br /> tRNAArg2 mini Arg2. Sequences structures outside                      determinant tadA activity (Figure 4B, lane 7). When <br /> the anticodon arm important deamination                       position 37 mutant 6 changed G, I formation was <br /> the correct position. In order determine nucleotides                  restored (Figure 4B, lane 8). G position 37 WT <br /> are essential specify short RNAs substrates                   Arg2 minisubstrate did affect tadA activity (results not <br /> tadA, mutant minisubstrates generated                        shown). However, reverting position 33 mutant 6 the <br /> tested vitro. No I formed minisubstrates                     WT U residue restored I formation, was, however, <br /> have mutations position 35 36 anticodon                          ef&#194;&#174;cient WT minisubstrate (Figure 4B, <br /> (Figure 4B, lanes 3 4). This showed nucleotides                   lane 9; Table I). This suggests U33 important <br /> the anticodon key determinants tadA activity. The                    determinant tadA activity, additional nucleotides <br /> importance anticodon loop size investigated                        probably contribute ef&#194;&#174;cient deamination. To determine <br /> with minisubstrate 8 nucleotide (nt) loop.                      stem&#194;&#177;loop structure addition sequence <br />  <br /> 3844 <br />                                                                                               An essential tRNA-speci&#194;&#174;c adenosine deaminase <br />  <br />  <br />  <br />  <br /> Fig. 4. RNA minisubstrates derived E.coli tRNAArg2 tRNAArg3 suf&#194;&#174;cient deamination tadA vitro. (A) Schematic drawing Arg <br /> minisubstrates tested vitro. Nucleotides shown gray mutated compared WT Arg2. A wobble position anticodon is <br /> shown bold. Mutations minisubstrate indicated arrows. (B) Editing assay minisubstrates tadA. A 25&#194;&#177;100 fmol aliquot of <br /> minisubstrate, 100 ng recombinant Flag-tadA-His6 500 ng BSA incubated total volume 25 ml. WT Arg2 (lane 1), mutant 1 <br /> (lane 2), mutant 2 (lane 3), mutant 3 (lane 4), mutant 4 (lane 5), mutant 5 (lane 6), mutant 6 (lane 7), mutant 7 (lane 8), mutant 8 (lane 9), mutant 9 <br /> (lane 10), WT Arg3 (lane 11) mutant 10 (lane 12) used. All reactions incubated 45 min room temperature processed as <br /> described legend Figure 2 (see Materials methods). <br />  <br />  <br />  <br /> is required tadA activity, linear minisubstrate was <br />                                                                                Table I. Quanti&#194;&#174;cation tadA activity obtained tRNA-derived <br /> analyzed (mutant 9, Figure 4A). Although mutant 9                              minisubstrate RNAs <br /> contains essential nucleotides activity as <br /> mutant 7, linear RNA deaminated (Figure 4B,                        Substrate     Mutations compared WT                      mol I/mol <br />                                                                                                                                              RNA <br /> lane 10), indicating stem&#194;&#177;loop structure essential <br /> for tadA activity. As expected, anticodon arm E.coli                    Arg2 WT                                                       1.2&#194;&#177;1.4 <br /> tRNAArg3 substrate tadA (Figure 4B, lane 11)                     mutant 1      A34G                                            0 <br /> because RNA does contain A wobble                              mutant 2      C35G                                            0 <br />                                                                                mutant 3      G36C                                            0 <br /> position (Figure 4A). However, I formation detected                        mutant 4      8 nt loop, U28G, A42C                           0.5 <br /> in Arg3 C34 mutated A (Figure 4B, lane 12).                        mutant 5      3 nt loop, 7 bp G-C stem                        0 <br /> This result provides information key                             mutant 6      U28G, C32U, U33C, A37C, A38U, A42C              0 <br /> nucleotides tadA activity. The anticodon stem                           mutant 7      U28G, C32U, U33C, A37G, A38U, A42C              0 <br /> Arg3 differs Arg2, indicating recognition                    mutant 8      U28G, C32U, A37C, A38U, A42C                    0.26 <br />                                                                                mutant 9      Linear                                          0 <br /> of stem tadA sequence speci&#194;&#174;c (Figure 4A). <br /> In summary, minisubstrate stem&#194;&#177;loop structure                      Arg3 WT                                                       0 <br />                                                                                mutant 10     C34A                                            1.39 <br /> deaminated wobble position anticodon the <br /> RNA sequence UACG loop. However, it <br /> cannot excluded regions tRNA are <br /> needed ef&#194;&#174;cient deamination. The results                          mutant (NWL37) corresponding parental <br /> mutant minisubstrates suggest that, addition                         strain (NWL32). After ampli&#194;&#174;cation tRNAArg2 by <br /> correct sequence loop, structure loop                       RT&#194;&#177;PCR speci&#194;&#174;c primers, cDNAs sub- <br /> size, stem sequence, important                             cloned multiple clones strain sequenced. <br /> deamination tadA.                                                           As expected, clones derived WT strain <br />                                                                                contained G position 34 (results shown). <br /> A point <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>mutation</span> tadA leads inactive                                  Interestingly, 16 17 clones derived NWL37 <br /> enzyme vitro                                                                G position 34, clone contained A <br /> Next asked tadA responsible I formation                      position (results shown). Therefore, point <br /> at position 34 tRNAArg2 vivo took advantage                     mutant tadA fully active vivo, not <br /> non-lethal point mutant described previously (Poulsen                          suprising given mutant strain showed growth <br /> et al., 1992). The mutant form tadA carries glutamic                      defect 42&#194;&#176;C (results shown), tadA <br /> acid instead aspartic acid position 64 (D64E;                         essential gene. <br /> Figure 1). This amino acid highly conserved bacterial,                      Although mutant protein functional cells, its <br /> but eukaryotic, Tad2 proteins.                                          activity reduced in vitro assay system. We <br />    To determine deamination A34 tRNAArg2                         prepared extracts mutant parental <br /> is affected mutation vivo, total RNA isolated                   strains, tested equal amounts deamination assay. <br />  <br />                                                                                                                                                 3845 <br />  J.Wolf, A.P.Gerber W.Keller <br />  <br />  <br />  <br />  <br /> Fig. 5. The point mutation D64E abolishes tadA activity vitro. <br /> Lane 1, protein; lanes 2&#194;&#177;4, 40 mg total protein E.coli extract. <br /> All reactions incubated 2 h 37&#194;&#176;C processed described <br /> in legend Figure 2 (see Materials methods). <br />  <br />  <br />  <br /> Whereas extracts WT cells showed activity <br /> (NWL32; Figure 5, lane 2), I formation be <br /> detected extracts mutant strain (NWL37; <br /> Figure 5, lane 3). However, activity restored by <br /> transforming mutant cells tadA expression <br /> construct (NWL37R; Figure 5, lane 4). Thus, mutation <br /> D64E abolishes enzymatic activity tadA vitro, <br /> although does detectably affect activity cells. <br />  <br />                                                                            Fig. 6. tadA replace yeast Tad2p vitro forms homo- <br /> tadA substitute yeast Tad2p vivo                         dimers. (A) Recombinant yeast Tad2p, Tad3p E.coli tadA were <br /> in vitro                                                                   pre-incubated indicated tested activity vitro with <br /> Because tadA 34% identical yeast Tad2p                        tRNAAla B.mori. As control, yeast Tad2p/Tad3p complex <br /> functionally conserved, attempted replace Tad2p                      used (lane 2). (B) Flag pull-down assay vitro translated <br /> with tadA DTAD2 yeast strain (Gerber Keller,                      tadA. 35S-labeled tadA incubated Flag-tagged tadA (lane 2) or <br />                                                                            buffer (lane 1) subsequently bound Flag&#194;&#177;agarose eluted. Ten <br /> 1999). tadA cloned pGALDTrp-FLIS6 vector                      percent input shown lane 3. Proteins separated a <br /> (Gerber et al., 1998) transformed Dtad2 yeast                   12% SDS&#194;&#177;polyacrylamide gel. Molecular masses protein <br /> strain carrying copy TAD2 plasmid                       standards indicated kDa left. (C) Chemical cross-linking <br /> URA3 marker (pFL38; Gerber Keller, 1999). Cells                        recombinant tadA. A 300 ng aliquot tadA (lanes 2&#194;&#177;4) was <br />                                                                            incubated reaction mixture containing tris(2,2&#194;&#162;-bipyridyl)- <br /> were transferred plates containing 5-&#194;&#175;uoro-orotic                  dichlororuthenium(II)hexahydrate, APS BSA (90 ng, 0.9 mg and <br /> acid (5-FOA) galactose force loss URA3-                  9 mg, respectively). tadA incubated buffer shown lane 1. <br /> marked TAD2 plasmid induce expression tadA,                     Reaction products separated 12% SDS&#194;&#177;polyacrylamide gel <br /> respectively. No cells grew, showing tadA                      blotted nitrocellulose membrane. tadA detected a <br /> substitute Tad2p vivo.                                              mouse anti-Flag antibody. Molecular masses protein standards are <br />                                                                            indicated kDa left. <br />    Next tested tadA functionally replace <br /> Tad2p in vitro deaminase assay. No activity was <br /> detected tadA pre-incubated yeast Tad3p was <br /> incubated B.mori tRNAAla, substrate                    agarose, washed eluted boiling matrix in <br /> the Tad2p/Tad3p heterodimer (Figure 6A, lanes 4 2). <br />                                                                            SDS&#194;&#177;gel loading buffer. The result shown Figure 6B <br /> Thus, tadA replace yeast Tad2p vivo or <br />                                                                            (lane 2). Approximately 10% tadA input pulled <br /> in vitro. <br />                                                                            Flag-tadA. In second experiment, recombinant <br />                                                                            Flag-tadA chromatographed gel &#194;&#174;ltration <br /> tadA forms homodimers                                                      column, separates proteins according size, and <br /> Because yeast Tad2p functions heterodimer                        fractions tested tadA activity. Fifty percent of <br /> Tad3p, E.coli does encode Tad3-like protein,                  tadA eluted position corresponding dimer and <br /> wondered tadA form homodimers                                ~50% tadA eluted monomer (results not <br /> instead. This hypothesis tested different                   shown). This result suggests tadA act a <br /> approaches. We investigated interaction Flag-tadA                   homodimer. However, presence tadA monomers <br /> and vitro translated [35S]methionine-labeled tadA                  indicates tadA does interact strongly itself <br /> Flag pull-down assay. The tagged tadA labeled                          does yeast Tad2p Tad3p (Gerber Keller, 1999). <br /> translated tadA pre-incubated, bound anti-Flag&#194;&#177;                    Formation tadA monomers occur separation of <br />  <br /> 3846 <br />                                                                                                An essential tRNA-speci&#194;&#174;c adenosine deaminase <br />  <br />  <br />  <br />  <br /> Fig. 7. tadA essential gene. (A) The sacB counterselection procedure. The wild-type chromosomal copy tadA exchanged two-step <br /> homologous recombination reaction gene copy disrupted insertion kanamycin-resistance cassette. After second recombination <br /> step, colonies analyzed antibiotic-resistance pro&#194;&#174;le presence absence Kan cassette tadA locus mutational <br /> inactivation sacB gene. The mutation sacB indicated asterisk. (B) Summary clones obtained strategy described (A). <br /> The total number sucrose-resistant colonies analyzed indicated &#194;&#174;rst column (Total clones). The number isolated colonies disrupted <br /> tadA (DtadA) gene chromosome shown second column number colonies restored WT tadA locus (WT) the <br /> second recombination step shown column. Colonies acquired sucrose resistance sacB inactivation listed fourth <br /> column (sacB&#194;&#177;). Plasmids pJW30 pJW117 carry disrupted WT copies tadA ORF, respectively. (C) PCR analysis WT (lane 1) and <br /> DtadA (lane 2) genomic E.coli DNA. Primers indicated arrows anneal 500 bp upstream tadA ORF 3&#194;&#162; end ORF, <br /> respectively. Sizes standards (lane M) indicated bp left. <br />  <br />  <br />  <br />  <br /> tadA dimers gel &#194;&#174;ltration column                             inserting chloramphenicol acetyltransferase (CAT) gene <br /> experimental conditions. To con&#194;&#174;rm dimerization,                            SphI restriction site ORF, removed <br /> carried chemical cross-linking experiments and,                            17 amino acids protein. We re-investigated <br /> indeed, observed complex corresponding size                            tadA essential viability recom- <br /> tadA homodimers (Figure 6C, lanes 2&#194;&#177;4). Complex                                binant tadADC, corresponding <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>knock-out</span> gene <br /> formation strongly stimulated addition                           described Poulsen et al. (1992), active tRNA <br /> BSA, weak cross-link detectable                         editing assay (results shown). The tadA gene was <br /> some experiments absence BSA (results                            inactivated two-step sacB counterselection tech- <br /> shown). The reason stimulation unclear;                            nique (Reece Phillips, 1995). For purpose, a <br /> however, BSA stabilizing effect                            plasmid-borne tadA gene produced, was <br /> interaction highly puri&#194;&#174;ed tadA subunits. tadA                          disrupted interposon carrying kanamycin-resist- <br /> complex formation speci&#194;&#174;c involved tadA                           ance cassette (pJW30; Figure 7A). The disrupted tadA <br /> subunits BSA. Complexes BSA                          cassette introduced chromosomal tadA locus by <br /> higher molecular mass easily                            homologous recombination. Thus, &#194;&#174;rst recombination <br /> distinguished tadA homodimers size. Dimeriza-                          event resulted introduction kanamycin- <br /> tion complete observed ~30%                        disrupted tadA gene plasmid sequences the <br /> tadA. This experimental procedure                       genome WT tadA (Figure 7A). Subsequent growth <br /> that tadA cross-linked suboptimal                        strains 5% sucrose selected excision the <br /> dimerization conditions. The addition E.coli total tRNA                     sacB gene, confers sucrose sensitivity, second <br /> did in&#194;&#175;uence dimer formation assays,                              recombination event. Sucrose-resistant clones then <br /> indicating observed dimerization did require                      analyzed kanamycin plasmid-encoded chloram- <br /> substrate tRNA (results shown). These experiments                          phenicol resistance. If WT copy tadA lost <br /> indicate tadA form homodimer vitro;                              gene essential, expect &#194;&#174;nd <br /> however, remains shown acts                      similar numbers WT tadA DtadA clones <br /> homodimer vivo.                                                             (Figure 7A). However, 400 clones analyzed <br />                                                                                KanS/CamS and, therefore, wild type, indicating that <br /> tadA essential cell viability                                           disrupted form tadA lost selectively that <br /> tadA previously reported non-essential                        tadA essential gene (Figure 7B). To corroborate this, <br /> gene E.coli (Poulsen et al., 1992). tadA deleted                     repeated tadA deletion experiment presence <br />  <br />                                                                                                                                                  3847 <br />  J.Wolf, A.P.Gerber W.Keller <br />  <br />  <br /> of functional copy tadA control            shown tRNA modi&#194;&#174;cation enzymes that <br /> arabinose promoter (PBAD). In case, approximately           were, therefore, suggested different functions <br /> equal numbers KanR/CamS (e.g. DtadA, 47%) KanS/           (Persson et al., 1992; Gutgsell et al., 2000). There two <br /> CamS (e.g. WT, 49%) clones obtained (Figure 7B).            major differences tRNA deamination E.coli <br /> The deletion tadA con&#194;&#174;rmed PCR analysis             S.cerevisiae. First, tRNAArg2 E.coli tRNA <br /> seven KanR/CamS clones (Figure 7C). PCR analysis          A position 34, tRNA <br /> of genomic DNA WT cells led accumulation          target tadA, yeast seven tRNAs containing <br /> a fragment 1000 bp (Figure 7C, lane 1), corresponding         A34. Secondly, tadA suf&#194;&#174;cient catalyze reaction in <br /> to tadA + 500 bp upstream sequence. This product         E.coli, yeast Tad2p/Tad3p heterodimer is <br /> formed genomic DNA DtadA strain. Instead,          needed modi&#194;&#174;y A34 I34 tRNAs (Gerber Keller, <br /> faint band observed ~2500 bp, corresponds        1999). <br /> tadA disrupted kanamycin cassette 500 bp                 The sequences tadA yeast tRNA deaminase <br /> upstream sequence (Figure 7C, lane 2). Note             subunit Tad2p compared predicted Tad2 <br /> primers used speci&#194;&#174;c ampli&#194;&#174;cation             proteins prokaryotic organisms (Figure 1). <br /> genomic tadA locus, rescue copy tadA           Comparing sequences cytidine <br /> not ampli&#194;&#174;ed. In addition, deletion con&#194;&#174;rmed          deaminases revealed share conserved motifs <br /> Southern blot analysis individual clones (results       typical cytidine nucleotide deaminases rather <br /> not shown). These results indicate disruption        adenosine deaminases (Gerber Keller, 1999). On <br /> WT chromosomal copy tadA resulted viable E.coli            basis observations, propose tadA the <br /> only additional plasmid-borne tadA gene         ancestor eukaryotic Tad2/Tad3 proteins. Gerber and <br /> present. The results obtained gene          Keller (1999) suggested Tad2 Tad3 paralogs <br /> disruption experiments performed Caulobacter           appeared gene duplication divergence of <br /> crescentus (results shown). Thus, conclude           prokaryotes eukaryotes. This hypothesis further <br /> tadA essential viability eubacterial organisms.        supported identi&#194;&#174;cation tadA, acts a <br />                                                                  single polypeptide. Prokaryotic organisms encode only <br />                                                                  Tad2-like proteins, appearance Tad3 later <br /> Discussion                                                       event, probably occurred divergence <br /> We cloning functional characterization       prokaryotes eukaryotes. <br /> the E.coli enzyme tadA/ecADAT2. Based following              I-containing tRNAs predicted recognize codons <br /> observations, conclude tadA adenosine              ending C, U A. I wobble position allows a <br /> deaminase speci&#194;&#174;cally converts A34 I34 bacterial      single tRNA read different codons (NNU, NNC, <br /> tRNAArg2. (i) Recombinant tadA, puri&#194;&#174;ed homogeneity,          NNA), which, therefore, encode amino acid <br /> speci&#194;&#174;cally deaminates tRNAArg2 wobble position        (Crick, 1966; Jukes, 1973). tRNAs I wobble <br /> the anticodon does need cofactor. Highly puri&#194;&#174;ed       position present bacteria eukarya which <br /> tadA formed 0.5 mol I/mol tRNA. Auxilien et al. (1996)           representative sets tRNAs characterized <br /> showed 1 mol I/mol tRNAArg2 generated              (Sprinzl et al., 1998). However, Percudani (2001) sug- <br /> incubation E.coli extract. tRNAArg2 incubated       gested I-containing tRNAs contribute <br /> 3 h 130 mg total protein experiments. It is,    signi&#194;&#174;cantly decoding NNA codons higher <br /> therefore, possible pure tadA stable   eukaryotes synonymous U- <br /> presence proteins (e.g. tRNA modi&#194;&#174;cation                starting anticodon read codons ending A <br /> enzymes) stabilized enzyme. Furthermore, struc-          wobbling. Because prokaryotes fewer tRNA <br /> ture vitro transcribed tRNA stabilized     species, I used decode A. tRNAArg2 only <br /> by additional proteins S100 extract          tRNA decodes CGA arginine codon. CGA a <br /> have contributed ef&#194;&#174;ciency I formation. (ii) The       rare codon frequency 0.36% E.coli <br /> recombinant protein highly speci&#194;&#174;c wobble             (http://www.kazusa.or.jp/codon/) speci&#194;&#174;c- <br /> position tRNAArg2. However, eukaryotic tRNAs              ally avoided poor translational properties <br /> deaminated tadA contain anticodon loop         (Curran, 1995). The possibilities codon&#194;&#177;anticodon base <br /> tRNAArg yeast E.coli. (iii) Recombinant tadA             pairing greatly improved I wobble position <br /> forms homodimers. Dimerization tadA com-              tRNAs. This advantage important for <br /> plete tadA monomers observed       <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span> cells selective conditions, might <br /> experiments. However, experi-           promoted spreading A34 editing mechan- <br /> mental procedure does necessarily re&#194;&#175;ect             ism total number tRNA genes an <br /> situation vivo. A I deamination wobble position     organism reduced tRNAs containing I34. <br /> of anticodon conserved bacteria mammals,             An E.coli strain described point <br /> and enzymes catalyzing reaction conserved       mutation tadA protein (formerly designated yfhC; <br /> not sequence, probably           Poulsen et al., 1992). This mutation resulted transient <br /> property acting dimers. Thus, tadA homodimers          resistance overexpression membrane protein <br /> may represent ancestor eukaryotic Tad2p/Tad3p         gef, cell-killing function (reviewed Gerdes <br /> heterodimer. (iv) tadA encoded essential gene.          et al., 1990). It known cells killed the <br /> The fact E.coli cells viable tadA           proteins gef gene family mutation tadA <br /> underscores important role I34 tRNA. However,       leads resistance gef proteins. We that <br /> cannot excluded absence tadA protein          extracts strain NWL32 showed I formation, whereas <br /> rather absence I34 cause lethality. This   extracts tadA mutant strain NWL37 no <br />  <br /> 3848 <br />                                                                                  An essential tRNA-speci&#194;&#174;c adenosine deaminase <br />  <br />  <br /> deamination activity. The absence activity NWL37              It known three-dimensional archi- <br /> extracts probably instability mutated tadA      tecture tRNA important ef&#194;&#174;cient I formation <br /> protein, inactivated extract        tadA. We exclude possibility other <br /> preparation. tadA stabilized interactions       nucleotides, addition anticodon loop tRNA, <br /> with tRNA-modifying enzymes inter-               important tadA binding. Further experiments will <br /> actions impaired mutant protein. This            required tadA binds tRNA and <br /> hypothesis supported observation              nucleotides contacted enzyme. It also <br /> mutated amino acid NWL37 highly conserved               remains shown sequences Tad proteins are <br /> bacterial tadA sequences          needed tRNA binding, enzymes <br /> important function (Figure 1). NWL37 cells showed             described far contains known RNA-binding motif. <br /> growth defect 42&#194;&#176;C, indicating cells        The high substrate speci&#194;&#174;city tadA offers potential <br /> mutation did major disadvantage. In agreement         explanation fact polypeptide is <br /> with proposal, detect de&#194;&#174;ciency           required. A single protein suf&#194;&#174;ce modify <br /> tRNA deamination strain NWL37 vivo. This result            tRNAArg2 bacteria, recognition seven <br /> explains wild-type growth <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> mutant            different tRNAs yeast require elaborate <br /> strain, tadA mutant signi&#194;&#174;cantly reduced            mechanism. <br /> activity vivo expected lethal. It not <br /> known mutation tadA gene makes cells <br /> resistant gef, Poulsen et al. (1992) suggest   Materials methods <br /> is second mutation genome NWL37 because <br />                                                                  Plasmids strains <br /> some E.coli strains transiently resistant gef       See Supplementary data, available The EMBO Journal Online. <br /> when mutated tadA introduced P1 transduction. <br /> This mutation identi&#194;&#174;ed,         Cloning tadA gene <br /> responsible gef resistance.                              The ORF yfhC ampli&#194;&#174;ed PCR 25 ng E.coli genomic DNA <br />                                                                  primer pairs yfhC.1/yfhC.2, yfhC.3/yfhC.4 yfhC.5/yfhC.6. The <br />    In yeast, seven different tRNAs contain I34. The Tad2p/       primers anneal 5&#194;&#162; 3&#194;&#162; end tadA ORF, respectively. The <br /> Tad3p complex probably deaminates tRNAs;               ORF cloned pCR2.1 (Invitrogen) independent clones <br /> however, tested vitro          sequenced. tadA subcloned appropriate restriction <br /> recombinant proteins. tadA unable replace yeast            sites pQE9, pTrc-FLIS6 pGDV1. <br /> Tad2p vitro vivo. However, yeast Tad2p/Tad3p <br />                                                                  Expression tadA E.coli puri&#194;&#174;cation the <br /> proteins capable deaminating tRNAArg2 E.coli         recombinant protein <br /> in vitro (Auxilien et al., 1996). The importance          The procedure described tadA-pGDV1 construct. The <br /> anticodon loop recognition tadA shown            tadA-pGDV1 construct transformed BL21 cells (Invitrogen). <br /> tRNAArg minisubstrates. Mutations anticodon            Two liters 23 YT medium, supplemented 100 mg/l ampicillin and <br />                                                                  10 mM ZnCl2, inoculated 28 ml overnight preculture. <br /> these minisubstrates completely abolished tadA activity,         Cultures grown 37&#194;&#176;C OD600 0.94 induced by <br /> whereas mutations stem position 37 (3&#194;&#162;              adding isopropyl-b-D-galactopyranoside (IPTG) &#194;&#174;nal concentration <br /> adjacent anticodon) effect I formation. In      0.4 mM. Incubation continued 24&#194;&#176;C 5 h. Cells were <br /> summary, results suggest stem&#194;&#177;loop structure        harvested centrifugation pellet resuspended 40 ml of <br /> of anticodon arm nucleotides anticodon        lysis buffer (50 mM Tris&#194;&#177;HCl pH 8.0, 200 mM KCl, 10% glycerol, 0.02% <br />                                                                  NP-40, 2 mM b-mercaptoethanol, 0.5 mM phenylmethylsulfonyl <br /> at position 33 key determinants tadA activity. It    &#194;&#175;uoride, 0.5 mg/ml leupeptin, 0.7 mg/l pepstatin). Cells lysed by <br /> been shown previously tRNA modi&#194;&#174;cation             sonication (3 3 3 min) ice. After centrifugation, cleared lysate was <br /> processing enzymes need determinants           mixed Ni2+-NTA&#194;&#177;agarose previously equilibrated lysis buffer. <br /> tRNA sequence structure speci&#194;&#174;cally catalyze             The mixture stirred 4&#194;&#176;C 45 min loaded column. <br />                                                                  The Ni2+&#194;&#177;agarose washed 100 ml wash buffer (lysis buffer + <br /> reaction. Work tRNA synthetases shown            10 mM imidazole) bound proteins eluted 250 mM <br /> variety minisubstrates used aminoacyl-             imidazole. Fractions 3&#194;&#177;5 applied glutathione&#194;&#177;Sepharose <br /> ation (reviewed Musier-Forsyth Schimmel, 1993;            (Amersham Pharmacia) equilibrated buffer A (Tris&#194;&#177;HCl pH 8.0, <br /> Schimmel Alexander, 1998). Synthetic substrates          130 mM KCl, 10% glycerol, 0.02% NP-40, 1 mM DTT). Proteins were <br /> speci&#194;&#174;c aminoacylation short tetraloop       eluted buffer B (buffer A + 10 mM reduced glutathione). A 50 ml <br />                                                                  aliquot fraction 1 loaded gel &#194;&#174;ltration column (Superdex <br /> a 4 bp stem CCA end. For alanyl-tRNA synthetase,           200 PC3.2/30; Smart System, Pharmacia), equilibrated <br /> the critical determinant aminoacylation tRNAAla           buffer A. <br /> was shown G3:U70 base pair was <br /> also true speci&#194;&#174;c aminoacylation minihelix            Construction E.coli tadA null mutant <br />                                                                  See Supplementary data. <br /> (Francklyn Schimmel, 1989). Furthermore, been <br /> shown RNase P, enzyme forms mature 5&#194;&#162;          In vitro transcription tRNAs enzyme assays <br /> ends tRNAs cleaving leader sequence, act       In vitro transcription tRNAs performed essentially described <br /> on minisubstrates consist 3 bp stem     previously Gerber et al. (1998). For transcription minisubstrates, T7 <br /> short extensions 5&#194;&#162; 3&#194;&#162; ends (reviewed Altman       primer substrate primer mixed 1.2:1 ratio 10 mM Tris <br />                                                                  pH 8 concentration 20 pmol/ml substrate primer 24 pmol/ml <br /> and Kirsebom, 1999). Another example provided          T7 primer. Primers incubated 3 min 95&#194;&#176;C quickly <br /> spliceosomal 15.5 kDa protein. This protein speci&#194;&#174;cally          chilled ice. A 1 ml aliquot primer mix used reaction, <br /> interacts 5&#194;&#162; stem&#194;&#177;loop U4 snRNA plays         carried described previously (Gerber et al., 1998). <br /> important role late stage spliceosome assembly         Minisubstrates puri&#194;&#174;ed 20% urea gel. tadA activity assayed <br />                                                                  reaction mixture containing 50 mM Tris&#194;&#177;HCl pH 8.0, 25 mM KCl, <br /> (Nottrott et al., 1999). The crystal structure 15.5 kDa   2.5 mM MgSO4, 0.1 mM EDTA, 10% glycerol, 2 mM DTT 100&#194;&#177; <br /> protein revealed forms hydrogen bonds          200 fmol 33P-labeled tRNA substrate 50 ml reaction described <br /> &#194;&#174;ve bases U4 snRNA loop (Vidovic et al., 2000).           previously (Gerber et al., 1998). When puri&#194;&#174;ed tadA used, the <br />  <br />  <br />                                                                                                                                     3849 <br />  J.Wolf, A.P.Gerber W.Keller <br />  <br /> reaction supplemented 1 mg BSA. Reactions incubated             Synthetic polynucleotides amino acid <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span>. Proc. Natl Acad. <br /> at 37&#194;&#176;C 15&#194;&#177;90 min. Unlabeled 5&#194;&#162; IMP (Sigma) added                 Sci. USA, 48, 613&#194;&#177;616. <br /> reaction mixture internal standard.                                     Bass,B.L. (1997) RNA editing hypermutation adenosine <br />                                                                                  deamination. Trends Biochem. Sci., 22, 157&#194;&#177;162. <br /> Sequence analysis tRNAs                                                    Bass,B.L., Nishikura,K., Keller,W., Seeburg,P.H., Emeson,R.B., <br /> See Supplementary data.                                                          O&apos;Connell,M.A., Samuel,C.E. Herbert,A. (1997) A standardized <br />                                                                                  nomenclature adenosine deaminases act RNA. RNA, 3, <br /> UV cross-linking                                                                 947&#194;&#177;949. <br /> Ten microliter reactions carried 400 ng recombinant          Bjo&#195;&#710;rk,G.R., Wikstro&#195;&#710;m,P.M. Bystro&#195;&#710;m,A.S. (1989) Prevention of <br /> tadA 100 fmol labeled tRNA 25 mM Tris&#194;&#177;HCl pH 8.0, 2.5 mM               translational frameshifting modi&#194;&#174;ed nucleoside 1-methyl- <br /> MgSO4, 0.05 mM EDTA. The reactions incubated room                        guanosine. Science, 244, 986&#194;&#177;990. <br /> temperature 6 min irradiated ice UV Stratalinker      Bjo&#195;&#710;rk,G.R., Jacobsson,K., Nilsson,K., Johansson,M.J.O., Bystro&#195;&#710;m,A.S. <br /> 500 mJ. tRNAs digested 500 ng RNaseA 25 min 37&#194;&#176;C.            Persson,O.P. (2001) A primordial tRNA modi&#194;&#174;action required for <br /> Proteins separated denaturing 12% SDS&#194;&#177;polyacrylamide gel               evolution life? EMBO J., 20, 231&#194;&#177;239. <br /> and exposed phosphoimager screen.                                        Chester,A., Scott,J., Anant,S. Navaratnam,N. (2000) RNA editing: <br />                                                                                  cytidine uridine conversion apolipoprotein B mRNA. Biochim. <br /> Chemical cross-linking                                                           Biophys. Acta, 1494, 1&#194;&#177;13. <br /> Cross-linking carried according modi&#194;&#174;ed version           Crick,F.H.C. (1966) Codon&#194;&#177;anticodon pairing: wobble hypothesis. <br /> procedure described Fancy Kodadek (1999). Reactions                  J. Mol. Biol., 19, 548&#194;&#177;555. <br /> performed microtiter plates total volume 20 ml. Reactions          Curran,J.F. (1995) Decoding A:I wobble pair inef&#194;&#174;cient. <br /> contained 13 cross-linking buffer (0.13 PBS, 2.5 mM MgSO4, 0.05 mM               Nucleic Acids Res., 23, 683&#194;&#177;688. <br /> EDTA), 300 ng puri&#194;&#174;ed recombinant tadA BSA 90 ng 9 mg.         Fancy,D.A. Kodadek,T. (1999) Chemistry analysis of <br /> Proteins dialyzed 13 PBS Tris buffer inhibits             protein&#194;&#177;protein interactions: rapid ef&#194;&#174;cient cross-linking triggered <br /> cross-linking reaction. Reaction buffer [15 mM sodium phosphate pH 7.5,          long wavelength light. Proc. Natl Acad. Sci. USA, 96, 6020&#194;&#177;6024. <br /> 150 mM NaCl, 0.125 mM tris(2,2&#194;&#162;-bipyridyl)dichlororuthenium(II)-              Francklyn,C. Schimmel,P. (1989) Aminoacylation RNA <br /> hexahydrate] (13) APS &#194;&#174;nal concentration 2.5 mM                 minihelices alanine. Nature, 337, 478&#194;&#177;481. <br /> added dark just irradiation. Reactions irradiated   Gerber,A.P. Keller,W. (1999) An adenosine deaminase that <br /> &#194;&#175;ashlight 30 s immediately quenched 5 ml              generates inosine wobble position tRNAs. Science, 286, <br /> 43 Laemmli loading buffer. Proteins denatured 95&#194;&#176;C 3 min,            1146&#194;&#177;1149. <br /> separated 15% SDS&#194;&#177;polyacrylamide gel blotted                  Gerber,A. Keller,W. (2001) RNA editing base deamination: more <br /> nitrocellulose membrane.                                                         enzymes, targets, new mysteries. Trends Biochem. Sci., 26, <br />                                                                                  376&#194;&#177;384. <br /> Flag pull-down assay                                                          Gerber,A., Grosjean,H., Melcher,T. Keller,W. (1998) Tad1p, yeast <br /> tadA vitro transcribed/translated construct pJW106          tRNA-speci&#194;&#174;c adenosine deaminase, related mammalian <br /> TNT&#195;&#162; Coupled Reticulocyte Lysate System (Promega) according               pre-mRNA editing enzymes ADAR1 ADAR2. EMBO J., 17, <br /> manufacturer&apos;s instructions. A 500 ml aliquot Flag-matrix slurry          4780&#194;&#177;4789. <br /> washed twice buffer TKG25 (50 mM Tris&#194;&#177;HCl pH 8, 25 mM KCl,               Gerdes,K., Poulsen,L.K., Thisted,T., Nielsen,A.K., Martinussen,J. and <br /> 10% glycerol, 0.1 mM EDTA, 0.01% NP-40). The Flag-matrix                Andreasen,P.H. (1990) The hok killer gene family gram-negative <br /> blocked 10 mg BSA 30 min 4&#194;&#176;C washed           bacteria. New Biol., 2, 946&#194;&#177;956. <br /> times buffer TKG25. A 1 mg aliquot Flag-tagged recombinant            Gott,J.M. Emeson,R.B. (2000) Functions mechanisms RNA <br /> tadA puri&#194;&#174;ed Ni2+-NTA&#194;&#177;agarose, 3 ml vitro                 editing. Annu. Rev. Genet., 34, 499&#194;&#177;531. <br /> translated tadA 100 ng BSA buffer TKG25 total volume        Grosjean,H. et al. (1996) Enzymatic formation adenosine inosine <br /> 15 ml incubated room temperature 1 h. Buffer TKG25                   N1-methylinosine transfer RNAs: review. Biochimie, 78, <br /> (23 250 ml) added protein mixture transferred           488&#194;&#177;501. <br /> tube containing Flag-matrix incubated 4&#194;&#176;C 45 min. The          Gutgsell,N., Englund,N., Niu,L., Kaya,Y., Lane,B.G. Ofengand,J. <br /> Flag-matrix washed 33 1 ml buffer TKG25. A 20 ml aliquot          (2000) Deletion Escherichia coli pseudouridine synthase gene <br /> Laemmli loading buffer added Flag-matrix boiled 95&#194;&#176;C          truB blocks formation pseudouridine 55 tRNA vivo, does not <br /> 3 min. The supernatant loaded 12% SDS&#194;&#177;polyacrylamide gel.             affect exponential growth, confers strong selective disadvantage <br />                                                                                  competition wild-type cells. RNA, 6, 1870&#194;&#177;1881 <br /> Supplementary data                                                            Hanrahan,C.J., Palladino,M.J., Ganetzky,B. Reenan,R.A. (2000) <br /> Supplementary data available The EMBO Journal Online.                     RNA editing Drosophila para Na+ channel transcript. <br />                                                                                  Evolutionary conservation developmental regulation. Genetics, <br />                                                                                  155, 1149&#194;&#177;1160. <br /> Acknowledgements                                                              Holley,R.W., Everett,G.A., Madison,J.T. Zamir,A. (1965) <br />                                                                                  Nucleotide sequences yeast alanine transfer RNA. J. Biol. Chem., <br /> We thank Dr G.Church (Harvard Medical School, Boston, MA)                    240, 2122&#194;&#177;2127. <br /> providing plasmid pKO3, Dr H.Loferer (Genome Pharmaceuticals                  Jukes,T.H. (1973) Possibilities evolution genetic <span id='am-396' about='Thesaurus:code' typeof='owl:Thing'>code</span> from <br /> Corporation, Munich, Germany) plasmid pRDC15 Dr S.Molin                  preceding form. Nature, 246, 22&#194;&#177;26. <br /> (Technical University Denmark, Lyngby, Denmark) E.coli strains         Keegan,L.P., Gerber,A.P., Brindle,J., Leemans,R., Gallo,A., Keller,W. <br /> NWL32, NWL37 NWL43. We grateful U.Jenal members                   O&apos;Connell,M.A. (2000) The properties tRNA-speci&#194;&#174;c <br /> of group Caulobacter strains helpful discussions. We thank           adenosine deaminase Drosophila melanogaster support an <br /> B.Dichtl, U.Ru&#195;&#710;egsegger M.Schaub critically reading                  evolutionary link pre-mRNA editing tRNA <br /> manuscript. This work supported University Basel,              modi&#194;&#174;cation. Mol. Cell. Biol., 20, 825&#194;&#177;833. <br /> Swiss National Science Foundation Louis-Jeantet-Foundation        Keegan,L.P., Gallo,A. O&apos;Connell,M.A. (2001) The roles an <br /> Medicine.                                                                        RNA editor. Nat. Rev. Genet., 2, 869&#194;&#177;878. <br />                                                                               Klingler,T.M. Brutlag,D.L. (1993) Detection correlations tRNA <br />                                                                                  sequences structural implications. Proc. Int. Conf. Intell. Syst. <br /> References                                                                       Mol. Biol., 1, 225&#194;&#177;233. <br />                                                                               Maas,S. Rich,A. (2000) Changing genetic information RNA <br /> Altman,S. Kirsebom,L. (1999) Ribonuclease P. In Gesteland,R.F.,              editing. BioEssays, 22, 790&#194;&#177;802. <br />   Cech,T.R. Atkins,J.F. (eds), The RNA World. Cold Spring Harbor          Maas,S., Gerber,A.P. Rich,A. (1999) Identi&#194;&#174;cation and <br />   Laboratory Press, Cold Spring Harbor, NY.                             <br /> </body></html>