<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An essential protein-binding domain <br /> of nuclear RNase P RNA <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Eukaryotic RNase P and RNase MRP are endoribonucleases composed of RNA and protein subunits. The RNA <br /> subunits of each enzyme share substantial secondary structural features, and most of the protein subunits are shared <br /> between the two. One of the conserved RNA subdomains, designated P3, has previously been shown <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> be required <br /> for nucleolar localization. Phylogenetic sequence analysis suggests that the P3 domain interacts with one of the <br /> proteins common to RNase P and RNase MRP, a conclusion strengthened by an earlier observation that the essential <br /> domain can be interchanged between the two enzymes. To examine possible functions of the P3 domain, four <br /> conserved nucleotides in the P3 domain of Saccharomyces cerevisiae RNase P RNA (RPR1 ) were randomized to <br /> create a library of all possible sequence combinations at those positions. Selection of functional genes in vivo <br /> identified permissible variations, and viable clones that caused yeast to exhibit conditional growth phenotypes were <br /> tested for defects in RNase P RNA and tRNA biosynthesis. Under nonpermissive conditions, the mutants had reduced <br /> maturation of the RPR1 RNA precursor, an expected <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> in cases where RNase P holoenzyme assembly is <br /> defective. This loss of RPR1 RNA maturation coincided, as expected, with a loss of pre-tRNA maturation characteristic <br /> of RNase P defects. To test whether mutations at the conserved positions inhibited interactions with a particular <br /> protein, specific binding of the individual protein subunits to the RNA subunit was tested in yeast using the three- <br /> hybrid system. Pop1p, the largest subunit shared by RNases P and MRP, bound specifically to RPR1 RNA and the <br /> isolated P3 domain, and this binding was eliminated by mutations at the conserved P3 residues. These results <br /> indicate that Pop1p interacts with the P3 domain common to RNases P and MRP, and that this interaction is critical <br /> in the maturation of RNase P holoenzyme. <br /> Keywords: P3; protein; RNase MRP; RNase P <br />  <br />  <br /> INTRODUCTION                                                               2000), the huge increase in protein content suggests <br /> RNase P is the endoribonuclease responsible for re-                        that the proteins are serving entirely new functions in <br /> moving the 59 leader of tRNA primary transcripts (Alt-                     the eukaryotic nucleus+ The question of subunit func- <br /> man et al+, 1989; Pace &amp; Smith, 1990; Chamberlain                          tion is further complicated by the fact that another <br /> et al+, 1996b)+ In yeast, the nuclear holoenzyme is com-                   nuclear RNA processing enzyme, RNase MRP, has <br /> posed of an RNA subunit, which is indispensable for                        evolved <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> RNase P+ RNase MRP has an essential <br /> enzyme activity, and at least nine protein subunits+ This                  RNA subunit that is evolutionarily related to RPR1 RNA, <br /> protein content is surprisingly complex, given that the                    and the two enzymes also appear to have eight of the <br /> analogous bacterial RNase P contains only one small                        nine protein subunits in common (Gold et al+, 1989; <br /> protein in addition to its RNA subunit+ In bacteria, the                   Schmitt &amp; Clayton, 1992; Lygerou et al+, 1994, 1996; <br /> protein subunit functions in helping the RNA subunit to                    Morrissey &amp; Tollervey, 1995; Chu et al+, 1997; Dichtl &amp; <br /> fold and in interactions with the pre-tRNA substrate+                      Tollervey, 1997; Stolc &amp; Altman, 1997; Chamberlain <br /> Eighteen-fold more protein is present in the nuclear                       et al+, 1998; Schmitt, 1999)+ RNases P and MRP con- <br /> enzyme, and at the moment there is little evidence as                      tain a homologous subdomain in their RNA subunits, <br /> to what the increased protein content might be used                        designated P3, that is conserved in both secondary <br /> for+ Although the nuclear holoenzyme has additional                        structure and sequence (Fig+ 1A)+ The P3 domain sec- <br /> interactions with the pre-tRNA substrate (Ziehler et al+,                  ondary structure is a helix-internal loop-helix and con- <br />                                                                            tains highly conserved residues primarily in the initial <br />                                                                            helix and internal loop region+ The terminal helix varies <br />   Reprint requests to: David R+ Engelke, Department of Biological <br /> Chemistry, University of Michigan, 1301 East Catherine Street, Ann         in <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> and nucleotide identity between species (For- <br /> Arbor, Michigan 48109-0606, USA; e-mail: engelke@umich+edu+                ster &amp; Altman, 1990; Schmitt et al+, 1993; Tranguch &amp; <br />                                                                      565 <br />  566                                                 W.A. Ziehler et al. <br />  <br />  <br />  <br />  <br />       FIGURE 1. P3 domain comparisons from RNase P and RNase MRP <br />       RNAs+ A: Secondary structure of S. cerevisiae RNase P RNA based <br />       on phylogenetic and biochemical analysis (Tranguch et al+, 1994)+ <br />       The location of the P3 RNA domain is shown in bold+ The location <br />       and sequence of the inserted hybridization tag (Bertrand et al+, 1998) <br />       between positions 133 and 139 is shown in italics+ B: P3 RNA do- <br />       mains from RNase P and MRP are shown from five eukaryotes, with <br />       intraspecies nucleotide conservations in bold+ S. cerevisiae and <br />       S. pombe RNase P sequence is from Tranguch &amp; Engelke (1993) <br />       and Krupp et al+ (1986); human, mouse, and X. laevis RNase P is <br />       from Pitulle et al+ (1998); S. pombe RNase MRP is from Paluh and <br />       Clayton (1995); S. cerevisiae, human, mouse, and X. laevis RNase <br />       MRP is from Schmitt et al+ (1993)+ <br />  An essential protein-binding domain of nuclear RNase P RNA                                                         567 <br />  <br /> Engelke, 1993; Paluh &amp; Clayton, 1995; Pitulle et al+,         RNA (Tranguch &amp; Engelke, 1993) and somewhat less <br /> 1998)+ The P3 domains from Saccharomyces cerevi-              extensive models for RNase MRP RNA (Schmitt et al+, <br /> siae RNases P and MRP have been shown to be func-             1993; Pitulle et al+, 1998)+ A consensus structure for S. <br /> tionally equivalent, because swapping domains between         cerevisiae RNase P RNA is shown in Figure 1A+ One <br /> enzymes was permitted (Lindahl et al+, 2000)+                 structural aspect common to the RNase P and MRP <br />     Minimization experiments using domain deletions from      models is the existence of the P3 stem with an internal <br /> yeast RNase P RNA have shown the P3 domain to be              loop early in the RNA (nt 32&#226;&#8364;&#8220;85 in S. cerevisiae RNA)+ <br /> essential for generating a functional enzyme (Pagan-          Sequence comparison of RNase P and RNase MRP <br /> Ramos et al+, 1994)+ Recent studies have further iden-        RNAs from a variety of species has identified residues <br /> tified the P3 domain, in both human RNase P and RNase         that are widely conserved within species+ In contrast, <br /> MRP, as required for binding the To antigen, a 40-kDa         there is little or no sequence conservation between the <br /> protein immunoprecipitated by human autoimmune sera           P3 internal loops of even the same enzyme in distant <br /> (Reddy et al+, 1983; Liu et al+, 1994; Eder et al+, 1997)+    species+ Several nucleotide positions on the 59 side <br /> Microinjection experiments using synthetic RNase MRP          of the P3 internal loop are conserved between the <br /> RNA result in nucleolar localization of the RNA, con-         RNase P and RNase MRP RNAs within a species+ Bold <br /> sistent with the role of RNase MRP in processing pre-         letters in Figure 1B indicate the positions that appear to <br /> rRNA (Jacobson et al+, 1995)+ Microinjected RNase P           be conserved between the RNase P and RNase MRP <br /> RNA alone localized to the nucleolus only transiently         P3 domains within each species+ It had previously been <br /> (Jacobson et al+, 1997)+ Transient nucleolar localization     noted (Tranguch &amp; Engelke, 1993) in yeast that the <br /> of RNase P RNA was dependent on the P3 domain,                lower P3 helix positions were more tightly conserved <br /> suggesting a potential role for P3 in nucleolar localiza-     among RNase P RNAs than they appear to be be- <br /> tion+ It is currently not clear whether RNase P is also a     tween RNase P and RNase MRP+ Thus, it is possible <br /> nucleolar enzyme, and to what extent this is consistent       that the loop and stem regions of the P3 domain serve <br /> among species+ Yeast nuclear RNase P appears largely          different functions in RNase P+ The conservation of the <br /> nucleolar (Bertrand et al+, 1998), and in situ studies of     P3 internal loop sequences between RNases P and <br /> mammalian RNase P have indicated either transient             MRP is entirely consistent with a specific interaction <br /> nucleolar association (Jacobson et al+, 1997) or limited      with one or more proteins common to RNase P and <br /> steady-state localization to a &#226;&#8364;&#339;perinuclear compart-          MRP+ If the same protein(s) bound to the P3 loop in <br /> ment&#226;&#8364;&#65533; (PNC) located on the nucleolar periphery (Ma-           both enzymes, the protein making direct contact would <br /> tera et al+, 1995; Lee et al+, 1996)+ It has not been clear   probably need to coevolve with both RNA subunits+ <br /> from these data whether the role of the P3 domain in <br /> localization is to direct holoenzyme assembly, which in <br />                                                               Viable P3 variants display limitations <br /> turn causes localization, or whether the RNA interacts <br />                                                               in mutational tolerance <br /> directly with some nonholoenzyme cellular components+ <br />     To better understand the function of the P3 domain        Four of the P3 internal loop positions (U38, U39, C41, <br /> an extensive set of point mutations in the domain has         and A44) are the most tightly conserved among differ- <br /> been created and analyzed for function in vivo+ The           ent yeast RNase P RNAs+ A library of genes was cre- <br /> results suggest that significant sequence variation at        ated in which these four positions were randomized to <br /> highly conserved positions is tolerated for viability, but    give all of the 256 possible sequence combinations+ <br /> that most non-wild-type sequences lead to apparent            Library screening for viable clones showed that only a <br /> defects in RNase P maturation and function+ Further           fifth of the possible variants are able to produce func- <br /> investigation showed that Pop1p, the largest protein          tional enzyme+ Identity, frequency, and growth pheno- <br /> subunit, is likely to be responsible for binding directly     type for 89 sequenced clones are listed in Figure 2+ <br /> and specifically to the P3 domain of the RNA subunit,         Although the screen was not saturating and probably <br /> and that mutations at the conserved nucleotides dis-          did not provide an exhaustive list of permissible se- <br /> rupt this interaction+                                        quences, several trends are apparent in the viable mu- <br />                                                               tants+ First, nearly one-third of the clones contained <br />                                                               wild-type sequence+ This preference for the wild-type <br /> RESULTS <br />                                                               sequence suggests that sequence deviation is unfavor- <br />                                                               able, which is predicted by the highly conserved nature <br /> The P3 domain of RNases P and MRP <br />                                                               of these positions+ Second, no triple or quadruple mu- <br /> is composed of a conserved secondary <br />                                                               tants were isolated, possibly due to an additive effect in <br /> structure with minimal conserved <br />                                                               disrupting favorable contacts between the P3 RNA do- <br /> primary sequence <br />                                                               main and the bound protein+ Third, double mutants at <br /> Phylogenetic analyses of eukaryotic RNase P and               positions 38 and 39 are all temperature sensitive+ The <br /> RNase MRP RNA subunits have established refined               adjacent double mutants might disrupt local structure <br /> secondary structure models for the entire RNase P             to such a degree that an elevated temperature might <br />  568                                                                                                            W.A. Ziehler et al. <br />  <br />                                                                         the cell+ Previous defects in RNase P enzymatic activ- <br />                                                                         ity have shown an accumulation of tRNA species with <br />                                                                         unprocessed 59 ends (Lee et al+, 1991b)+ Similar re- <br />                                                                         sults were observed for the conditional P3 mutants at <br />                                                                         both the permissive (30 8C) and nonpermissive (37 8C) <br />                                                                         temperatures, with precursor accumulation being only <br />                                                                         slightly more elevated at nonpermissive temperature+ <br />                                                                         Examples of the RNA hybridization blots probed with <br />                                                                         an anti-tRNALeu3 oligomer are shown in Figure 3A+ The <br />                                                                         effects include an accumulation of primary tRNA <br />                                                                         transcript and an unusual intermediate in which splic- <br />                                                                         ing precedes terminal maturation (159, 139, intron <br />                                                                         spliced tRNA)+ Accumulation of both of these precur- <br />                                                                         sors is consistent with a reduction in RNase P activity <br />                                                                         (Lee et al+, 1991b)+ There is an absence of 59 cleaved, <br />                                                                         39 exonuclease-trimmed tRNA at the nonpermissive <br />                                                                         temperature, as well+ Clones without conditional growth <br />                                                                         phenotypes displayed RNase P activity ranging from <br />                                                                         near wild-type to defects similar to the conditional clones <br /> FIGURE 2. P3 RNA domain library location and identity, frequency, <br /> and phenotype of sequenced clones+ The randomized positions in          (Fig+ 3B)+ <br /> the mutated RPR1 gene library are indicated by &#226;&#8364;&#339;N&#226;&#8364;&#65533; relative to the P3 <br /> subdomain sequence+ Clones exhibiting temperature-sensitive growth <br /> are denoted by &#226;&#8364;&#339;ts&#226;&#8364;&#339;; otherwise growth is similar to wild type+ The      Conditional mutations in the P3 domain of <br /> sequence denoted by an asterisk contains a single nucleotide inser-     yeast RNase P result in an accumulation <br /> tion <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span>+ <br />                                                                         of precursor RNase P RNA (RPR1 RNA) <br />                                                                         Many of the mutants were found to accumulate precur- <br />                                                                         sor RNase P RNA subunit (RPR1 RNA), in addition to <br /> abolish the RNA&#226;&#8364;&#8220;protein interaction+ Nucleotides 38,                    the accumulation of pre-tRNA substrates+ S. cerevisiae <br /> 39, and 44 exhibited the full range of possible muta-                   RPR1 RNA is an RNA polymerase III transcript con- <br /> tions+ Position U38 was the least particular, varying                   taining an 84-nt 59 leader (Lee et al+, 1991a, 1991b)+ <br /> relatively equally between changes to A, G, and C+                      This precursor is processed by an uncharacterized nu- <br /> Conversely, changes at position U39 were to C over                      cleolytic pathway to produce mature RNase P RNA+ <br /> 50% of the time, and position 44 favored a mutation to                  Previous observations during purification of RNase P <br /> U 67% of the time+ Nucleotide C41 was changed to A                      (Chamberlain et al+, 1996a) showed that this precursor <br /> only three times and to U once+ There were no C41 to                    is assembled into an RNP complex that is similar in <br /> G mutants present in the sequenced clones+ Finally,                     size and chromatographic properties to the mature <br /> nucleotide C41 was the least mutable; only 4 out of 89                  RNase P, suggesting assembly precedes processing of <br /> clones had mutations at this position, and all were sin-                the RPR1 RNA subunit+ Defects in RNase P holoen- <br /> gle nucleotide changes+ A C41-to-U change was the                       zyme assembly, including depletion of individual pro- <br /> only temperature-sensitive single mutation observed+                    tein subunits, are believed to prohibit this processing of <br /> Although detailed interpretations of the nucleotide iden-               RPR1 RNA (Pagan-Ramos et al+, 1994; Chamberlain <br /> tity requirements are not possible without a more ex-                   et al+, 1998)+ In contrast, mutations in catalytically im- <br /> haustive analysis of viable versus nonviable sequences,                 portant residues in RPR1 RNA do not cause pre-RPR1 <br /> these results are consistent with an important role for                 accumulation (Pagan-Ramos et al+, 1996a, 1996b)+ <br /> the P3 internal loop sequences+ This result is also con-                Therefore, observed reductions in mature RNase P lev- <br /> sistent with mutagenesis of the corresponding region in                 els relative to precursor RNase P RNA suggest a role <br /> RNase MRP RNA (Shadel et al+, 2000), which sug-                         for a functional P3 domain in RNase P assembly and <br /> gested that the P3 loop sequences might be essential                    processing, rather than a primary defect in catalytic <br /> even if the distal stem was not+                                        competence+ Figure 4A shows a representative blot of <br />                                                                         RNA from temperature-sensitive clones at permissive <br />                                                                         and nonpermissive temperatures probed using an anti- <br /> Conditional mutations in the P3 domain of <br />                                                                         RPR1 RNA DNA oligomer+ <br /> yeast RNase P result in an accumulation <br />                                                                           P3 library clones that did not exhibit conditional growth <br /> of 59-unprocessed tRNA precursors <br />                                                                         phenotypes (Fig+ 4B) had pre-RPR1 RNA/mature RPR1 <br /> P3 mutations were investigated for their effect on                      RNA ratios ranging from near wild-type levels to accu- <br /> RNase P activity in vivo+ Relative RNase P activity can                 mulation of pre-RPR1 RNA similar to the temperature- <br /> be determined from the profile of pre-tRNA species in                   sensitive clones (Fig+ 4A)+ Clones with wild-type levels <br />  An essential protein-binding domain of nuclear RNase P RNA                                                                          569 <br />  <br />  <br />  <br />  <br />        FIGURE 3. Anti-tRNALeu3 blot hybridization+ A: RNA from temperature-sensitive P3 mutant clones harvested at permissive <br />        (30 8C) and nonpermissive (37 8C) conditions were probed with radiolabeled anti-tRNALeu3 DNA oligomer+ A subset of mutant <br />        RNAs are shown, with sequence listed above each pair of lanes and mutations underlined+ Identities of the various pre-tRNA <br />        intermediates are indicated by arrows and have been determined previously (Lee et al+, 1991b)+ B: RNA from P3 mutants <br />        with wild-type growth were analyzed by the same blotting methods after growth at 30 8C+ The sequence denoted by an <br />        asterisk contains an <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>insertion mutation</span>+ <br />  <br />  <br />  <br />  <br /> of mature RPR1 RNA also had more normal pre-tRNA                      ered for determining specific protein&#226;&#8364;&#8220;RNA interactions+ <br /> levels (compare Figs+ 3B and 4B)+ This is consistent                  Unfortunately, the RNase P protein subunits were, for the <br /> with the notion that reduced levels of mature RPR1                    most part, highly insoluble or unstable when expressed <br /> RNA result in reduced RNase P enzyme activity+                        individually in a variety of ways+ This made direct bio- <br />                                                                       chemical assessment of binary subunit interactions dif- <br />                                                                       ficult to interpret+ Instead, the yeast three-hybrid system <br /> Pop1p, a subunit common to RNases P <br />                                                                       (SenGupta et al+, 1996) was employed to determine <br /> and MRP, interacts with the P3 domain <br />                                                                       which of the S. cerevisiae RNase P protein subunits bind <br /> in a sequence-specific manner <br />                                                                       specifically to the P3 domain+ The three-hybrid system <br /> If the P3 domain nucleates assembly of the holoen-                    is outlined in Figure 5A+ &#226;&#8364;&#339;Bait&#226;&#8364;&#65533; RNAs were tethered to the <br /> zyme ribonucleoprotein, it is expected that one or more               artificial promoter regions of two reporter genes in yeast, <br /> of the protein subunits should interact specifically with             HIS3 and the b-galactosidase coding region+ Protein <br /> the conserved sequences+ This would be consistent                     coding regions for the RNase P proteins or controls were <br /> with the proposed interaction of the bacterial protein                then fused to activation domains and tested for whether <br /> subunit with the analogous P3 region of Escherichia                   interaction between the RNase P protein subunit and the <br /> coli RNase P RNA (M1 RNA) (Morse &amp; Schmidt, 1993)+                    bait RNA could bring the activator domain into position <br /> Both genetic and biochemical analyses were consid-                    to induce expression of both reporter genes+ Complete <br />  570                                                                                                            W.A. Ziehler et al. <br />  <br />  <br />  <br />  <br />        FIGURE 4. Anti-RPR1 RNA blot hybridization+ A: RNA from temperature-sensitive P3 mutant clones harvested at permis- <br />        sive (30 8C) and nonpermissive (37 8C) conditions probed with radiolabeled anti-RPR1 DNA oligomer+ P3 loop sequence is <br />        listed above each pair of lanes, with mutations underlined+ Locations of pre- and mature RPR1 RNAs are designated with <br />        arrows+ B: RNA from P3 mutants with wild-type growth harvested at 30 8C+ The sequence denoted by an asterisk is an <br />        insertion mutation+ <br />  <br />  <br />  <br /> analysis of protein&#226;&#8364;&#8220;RNA subunit analysis through three-             P3 domain alone, and the RPR1 subunit with point <br /> hybrid interactions will be published elsewhere, along              mutation in all four conserved P3 positions (Fig+ 5B)+ <br /> with the analysis of protein&#226;&#8364;&#8220;protein interactions through           The four simultaneous point mutations (A38, A39, G41, <br /> the two-hybrid system (F+H+ Scott, S+ Xiao, C+ Millikin,            and U44) were expected to strongly disrupt P3 func- <br /> J+M+ Zengel, L+ Lindahl, and D+R+ Engelke, in prep+)+ For           tion, because no triple or quadruple mutants were re- <br /> the purposes of this work, it was observed in the larger            covered from the in vivo library screen and the least <br /> set of experiments that deletion of the entire P3 domain            permissible nucleotide was used at each position+ We <br /> caused loss of Pop1p binding+ None of the other pro-                have subsequently determined that the quadruple point <br /> tein subunits displayed P3-dependent, specific binding              mutation does not produce functional RNase P (data <br /> to the RPR1 RNA subunit+ We therefore tested the                    not shown)+ Bait RNA plasmid was also used that had <br /> ability of Pop1p to bind to the P3 RNA subdomain alone,             no inserted sequences in addition to the MS2 binding <br /> and the effects of conserved position mutations on P3-              sequences that tether the RNA to the promoter+ None <br /> dependent binding of Pop1p to the full RPR1 RNA sub-                of these RNAs caused expression of the reporter genes <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>+ Growth in the absence of histidine is one indication          in the absence of Pop1p fused to the activator domain <br /> of stable interaction between the test protein and the              (not shown), nor did Pop1p fused to the activator cause <br /> bait RNA+ This data is shown for Pop1p in Figure 5C,                reporter expression without part of the RPR1 sense <br /> and was completely compatible with expression of the                sequence fused to the bait RNA+ Pop1p gave substan- <br /> other, b-galactosidase gene reporter (not shown)+ The               tial expression of both reporters when the bait con- <br /> expression of the reporter genes for different RNA baits            tained the whole wild-type RPR1 or the P3 domain <br /> in the presence or absence of Pop1p are summarized                  alone+ Antisense RPR1 sequences or quadruple point <br /> in panel D of Figure 5+                                             mutations in RPR1 RNA eliminated reporter expres- <br />   Bait RNAs consisted of whole wild-type RPR1 RNA,                  sion+ This was not due to instability of these RNAs, <br /> the RPR1 antisense sequence (control), the wild-type                because they were expressed at levels comparable to <br />  An essential protein-binding domain of nuclear RNase P RNA                                                                          571 <br />  <br />  <br />  <br />  <br />        FIGURE 5. Testing whether the Pop1p RNase P protein subunit interacts with the P3+ A: The three-hybrid test for protein&#226;&#8364;&#8220; <br />        RNA interactions is shown schematically+ The test RNA (RPR1 RNA, P3 RNA, or controls) are tethered near the promoters <br />        of two reporter genes, HIS3 and the lacZ ORF, by fusing the test RNA to MS2 RNA+ The MS2 RNA is bound by the MS2 <br />        binding protein, which is fused to the LexA DNA-binding domain (LexA DBD), which binds to LexA DNA sites in the <br />        promoters+ When a test protein fused to the Gal4 activation domain (Gal4 AD) binds to the test RNA, the cells are His1 and <br />        produce b-galactosidase+ B: The wild-type sequence of the P3 subdomain is shown, with the identity and location of <br />        quadruple point mutations (&#226;&#8364;&#339;RPR1 mutant P3&#226;&#8364;&#65533; in C and D) indicated by arrows+ &#226;&#8364;&#339;P3 RNA alone&#226;&#8364;&#65533; test RNA contains only the <br />        wild-type P3 sequence as shown+ C: Three-hybrid test strains containing the indicated bait RNAs and the Pop1p-activation <br />        domain fusion were plated as serial dilutions on growth media containing or lacking histidine+ All bait RNAs gave His 2 <br />        phenotypes if only the activation domain was tested in the absence of the Pop1p fusion (not shown)+ His1 phenotypes <br />        occurred only when sense, nonmutated P3 domain was contained in the bait RNA+ D: Three-hybrid test results are <br />        summarized for the HIS3 expression and b-galactosidase expression assays, which gave mutually consistent results+ <br />        Results are given for the indicated RNA baits using both Pop1p fused to the activation domain and the activation domain <br />        alone+ <br />  <br />  <br />  <br />  <br /> the wild-type RPR1 RNA (F+H+ Scott, S+ Xiao, C+ Millikin,             DISCUSSION <br /> J+M+ Zengel, L+ Lindahl, and D+R+ Engelke, in prep+)+ We <br /> therefore conclude that Pop1p binds selectively to P3                 The library screen for viable P3 mutants identified a set <br /> RNA in a manner requiring the positions in the P3 sub-                of rules governing permissible nucleotides at posi- <br /> domain that are conserved between RNases P and MRP+                   tions 38, 39, 41, and 44 of this RNA domain+ The four <br />  572                                                                                                      W.A. Ziehler et al. <br />  <br /> positions each have different degrees of mutational tol-       positions), likely a PCR artifact generated during li- <br /> erance+ The more discriminating positions might pro-           brary construction, was also isolated+ The U44 inser- <br /> vide essential structural elements for P3 function, or         tion behaves similarly to the wild-type construct with <br /> they might directly interact with protein partners+ Be-        respect to both RPR1 RNA levels and pre-tRNA pro- <br /> cause neither triple nor quadruple mutations were re-          cessing, further demonstrating the absence of nega- <br /> covered, it is likely that detrimental effects are additive+   tive effects of an A44-to-U mutation or U insertion+ <br />    Nucleotide U38 is found to mutate to each of the               Temperature-sensitive phenotypes permitted investi- <br /> other three bases rather randomly and without obvious          gation into the function of the P3 domain in context of <br /> preference+ Despite the variable nature of position 38,        RNase P biosynthesis and catalysis+ The RNA blot hy- <br /> mutations at this position have an additive effect when        bridizations determined the effects of P3 mutations on <br /> coupled with a change at its neighbor, nt 39+ When             RNase P activity and RNase P RNA maturation+ Pro- <br /> paired with a proximal mutation in position 39, the dou-       cessing of pre-tRNA substrates is decreased, possibly <br /> ble mutant is temperature sensitive for all such clones        the consequence of increased precursor RNase P RNA <br /> sequenced+ Mutations found at U39 also encompass               and decreased mature RNase P populations+ The in- <br /> all three nucleotide possibilities, although over half of      crease in pre-RNase P RNA suggests possible holo- <br /> all recovered clones contained a cytosine at position          enzyme assembly and maturation defects, as previously <br /> 39, indicating a preference for pyrimidines+                   described (Pagan-Ramos et al+, 1996a)+ These effects <br />    The fewest variations were observed at nucleotide           were consistent for all temperature-sensitive clones ex- <br /> C41; three clones were changed to adenosine and one            amined+ Even several P3 mutants with wild-type growth <br /> to uridine+ These four mutants did not have any second         characteristics exhibited some defects in pre-tRNA and <br /> changes at positions 38, 39, and 44+ In the rare instance      pre-RPR1 RNA processing+ Pre-RPR1 RNA is known <br /> C41 is changed, no other changes appear to be toler-           to assemble into a ribonucleoprotein (RNP) complex <br /> ated+ The C41-to-U change was the only temperature-            very similar, if not identical, to the mature holoenzyme <br /> sensitive single mutation+ All other temperature-sensitive     RNP prior to processing of the RPR1 RNA (Chamber- <br /> clones were double mutations+ The wild-type cytosine           lain et al+, 1996a)+ The temperature-sensitive P3 mu- <br /> is likely contributing an important element for P3 func-       tants appear defective in forming mature RNase P RNA, <br /> tion, such that its loss is equivalent to two mutations in     although it is not clear whether this is due to problems <br /> the other three randomized positions+ Three C41-to-A           with protein subunit association or the actual endonu- <br /> clones were recovered, and although not temperature            cleolytic processing of the pre-RPR1 RNA+ It is appar- <br /> sensitive, they showed an accumulation of pre-RPR1             ent, however, that a critical event in assembly of the <br /> RNA and concomitant defects in pre-tRNA processing             functional yeast RNase P involves a functional P3 <br /> (Figs+ 3B and 4B)+                                             domain+ <br />    Changes in the identity of nucleotide A44 are par-             The yeast three-hybrid results indicate that the larg- <br /> tially restricted+ All possibilities are represented in the    est protein subunit, Pop1p, interacts with RNase P RNA <br /> sequenced viable clones, and this position is frequently       in a sequence-specific manner+ Because the three-hybrid <br /> changed as part of double mutant clones, but 67% of            system is performed in yeast, and the proteins and RNAs <br /> all changes at position 44 are to uridine+ There is no         being screened are from yeast, various explanations can <br /> nucleoside conservation, as in the preferred pyrimidine        account for the three-hybrid results+ There could theo- <br /> change in position 39, or similarity in orientation of hy-     retically be adapter proteins, bridging between Pop1p <br /> drogen bonding potential+ However, this could be rec-          and the RNA domain that result in the observed inter- <br /> onciled by the following scenario+ The adenosine at            action+ However, this adapter would probably be titrated <br /> position 44 might not normally participate in the func-        out of endogenous complexes, because the protein sub- <br /> tion of the P3 domain, such as binding protein, but            units are not normally abundant, but Pop1p and the RNA <br /> when changed to uridine, the position may now provide          bait are expressed from high copy number plasmids+ In <br /> a favorable interaction+ This is supported by the fact         addition, three-hybrid analyses show that only Pop1p, <br /> that only 1 out of 16 U44 mutants with a secondary             out of all the known RNase P protein subunits, binds se- <br /> mutation are temperature sensitive, and G44/C44                lectively to the P3 domain or is sensitive to mutations at <br /> mutants with a second mutation are almost always tem-          conserved positions (data not shown)+ The most direct <br /> perature sensitive+ U44 is therefore a relatively innoc-       interpretation of these results is therefore that Pop1p is <br /> uous mutation, and may even facilitate the function of         binding directly to P3+ This interaction is likely to be a crit- <br /> the P3 domain in some contexts+ The function of the P3         ical step in assembling a functional holoenzyme, and <br /> domain is either independent of A44-to-U mutations,            probably occurs prior to processing of the pre-RPR1 <br /> effectively making double mutants with an A44-to-U             RNA to the mature size+ It should be noted that this data <br /> change behave as single mutants, or the A44 to U               does not exclude the possibility that Pop1p might con- <br /> contributes to the function of the P3 domain enough to         tact other positions in the RNA subunit (and other pro- <br /> prevent temperature-induced defects+ A U44 insertion           teins), even though those contacts are not absolutely <br /> mutation (U inserted between wild-type G43 and A44             essential to specific binding+ <br />  An essential protein-binding domain of nuclear RNase P RNA                                                                    573 <br />  <br />    It is not clear what relation this data has to previous          ase chain reaction and DNA oligomers degenerate at the four <br /> reports that the P3 domain of human RNase MRP in-                   nucleotide positions+ The starting wild-type RPR1 clone con- <br /> teracts either with a 40-kDa component of the To auto-              tains a phenotypically silent insert of 20 extra nucleotides at <br /> antigen (Yuan et al+, 1991; Liu et al+, 1994) or with at            the end of the stem loop between positions 133 and 139 that <br /> least two smaller proteins (Pluk et al+, 1999)+ Both stud-          was used as a hybridization tag for mutated RNAs+ This silent <br />                                                                     insertion has been tested previously (Bertrand et al+, 1998) <br /> ies relied on relatively crude assembly of RNPs, in which <br />                                                                     and is shown in Figure 1+ The library was constructed in a <br /> human Pop1 protein might have played a role in as-                  single copy (CEN) vector with a LEU selectable marker as <br /> sembly, but not contacted the RNA in such a way as to               previously described (Pagan-Ramos et al+, 1996b) and trans- <br /> crosslink+ Indeed, in the latter study it was concluded             formed into the haploid yeast strain JLY1 (MATa ade2-1 his3-11 <br /> that Pop1 protein bound in a way that required the P3               leu2-3,112 trp1-1 ura3-1 can1-100 RPR1::HIS3 )+ JLY1 has <br /> domain, but that there was no evidence that the inter-              the essential RPR1 chromosomal gene deleted and replaced <br /> action was direct+ The present data suggests that Pop1              by HIS3 (Lee et al+, 1991b) and a wild-type RPR1 on a coun- <br /> can interact directly with the RNA subunit in a fashion             terselectable URA3 -marked plasmid (Pagan-Ramos et al+, <br /> that requires conserved P3 positions+ One way to rec-               1996b)+ Transformation of this strain with the mutated RPR1 <br /> oncile these data with the previous MRP experiments                 library was followed by screening for clones that were viable <br /> is to propose that both Pop1p and smaller subunits                  when the wild-type RPR1 gene was removed by growth me- <br />                                                                     dia containing 5-fluororotic acid (Pagan-Ramos et al+, 1996b)+ <br /> interact with P3, but that Pop1 does not crosslink well <br />                                                                     The four randomized positions allow for 256 possible vari- <br /> to the RNA+ Alternatively, it is possible that protein sub-         ants; 1,000 clones were screened yielding 193 viable P3 <br /> units do not interact in a directly analogous manner                library clones+ Each viable clone was tested for temperature <br /> between the yeast RNase P RNA and the human RNase                   (37 8C) and cold (18 8C) sensitivity+ Temperature sensitivity <br /> MRP RNA+ Whichever is the case, the ability of Pop1p                was scored as zero growth under nonpermissive conditions+ <br /> to specifically bind the RNA subunit suggests that this             There were 37 temperature-sensitive clones, two of which <br /> recognition might be a key early element in assembling              were also cold sensitive+ Sequence, frequency, and temper- <br /> the full RNP complex+ Continued efforts are in progress             ature sensitivity of 89 viable clones are listed in Figure 2+ A <br /> to use biochemical reconstitution of the both enzymes               wild-type RPR1 gene carried on the identical plasmid used to <br /> from purified components to address these issues+                   produce the library provided a control for growth and RNA <br />                                                                     profile characterizations+ <br />  <br /> MATERIALS AND METHODS <br />                                                                     RNA blot hybridization of pre-tRNALeu3 <br /> Phylogenetic analysis of RNase P and                                Effects of P3 mutations on RNase P enzymatic activity were <br /> RNase MRP P3 RNA domains                                            determined by probing whole cellular RNA with a radio- <br />                                                                     labeled anti-tRNALeu3 DNA oligonucleotide+ Total cellular RNA <br /> The P3 RNA domains from RNase P and RNase MRP were                  was isolated from temperature-sensitive clones grown at per- <br /> compared to provide a consensus structure+ P3 domains from          missive (30 8C) or nonpermissive temperatures (37 8C) for 4 h <br /> the following species were used: S. cerevisiae and Schizo-          (Kohrer &amp; Domdey, 1991)+ Concentration of isolated RNA <br /> saccharomyces pombe RNase P (Tranguch &amp; Engelke, 1993);             was determined by measuring UV absorbance at 260 nm and <br /> human, mouse, and Xenopus laevis RNase P (Pitulle et al+,           6-mg sample quantities were run on denaturing polyacryl- <br /> 1998); S. pombe RNase MRP (Paluh &amp; Clayton, 1995); S.               amide gels, transferred to Nytran membrane and probed using <br /> cerevisiae, human, mouse, and X. laevis RNase MRP (Schmitt          a radiolabeled DNA oligonucleotide complementary to Leu3 <br /> et al+, 1993)+ Secondary structure is conserved throughout all      tRNA (59- 32 P-CTTAGACCGCTCGGCCAAAC-39)+ A radio- <br /> species P3 domains in RNase P and RNase MRP; however                labeled DNA oligonucleotide complementary to signal recog- <br /> there is very little primary sequence conservation among            nition particle RNA (SCR1 RNA, 59- 32 P-GGCGTGCAAT <br /> distant species (Fig+ 1B)+ Most intraspecies conservation be-       CCGTGTCT-39) was also used to probe the blots to normalize <br /> tween P3 domains from RNases P and MRP is located in the            hybridization signal+ RNA blot hybridization profiles were ex- <br /> 59 side of the internal loop+ In the case of S. cerevisiae, there   amined for changes characteristic of RNase P enzymatic ac- <br /> are seven internal loop P3 nucleotides that are identical in        tivity defects (Lee et al+, 1991b)+ These changes include an <br /> RNase P and RNase MRP (Fig+ 1B)+ However, four of the               accumulation of 59 unprocessed pre-tRNA, 59 unprocessed <br /> seven (U38, U39, C41, and A44) are also conserved be-               with mature 39 end, and 59 unprocessed with spliced intron <br /> tween six different yeast species RNase P P3 domains pre-           tRNA species+ <br /> viously sequenced (Tranguch &amp; Engelke, 1993), suggesting <br /> the identity of these four nucleotides is important for the func- <br /> tion of the P3 domain in yeast+                                     RNA blot hybridization of RNase P RNA <br />                                                                     Whole cellular RNA from temperature-sensitive mutants grown <br /> Generation of P3 RPR1 library by directed                           at permissive and nonpermissive conditions was probed with <br /> degenerate DNA oligomers                                            a radiolabeled anti-RPR1 DNA oligonucleotide+ The probe is <br />                                                                     complementary to nt 310&#226;&#8364;&#8220;327 of S. cerevisiae RNase P RNA <br /> The S. cerevisiae RNase P RNA gene (RPR1 ) was random-              (59-GACGTCCTACGATTGCAC-39)+ Electrophoresis and hy- <br /> ized at positions U38, U39, C41, and A44 using the polymer-         bridization were as previously described (Pagan-Ramos et al+, <br />  <br />  <br /> 1996a)+ Holoenzyme assembly defects result in an accumu-                  <br /> lation of precursor RNase P RNA as previously described                <br /> (Pagan-Ramos et al+, 1996a)+ The same radiolabeled anti-              <br /> SCR1 RNA DNA oligonucleotide used in the pre-tRNALeu3                   <br /> blots was used as a hybridization control+                        <br /> Three-hybrid test for RNA&#226;&#8364;&#8220;protein                                     <br /> interactions                                                               <br />                                                                    <br /> The three-hybrid test was performed as previously de-                  <br /> scribed (SenGupta et al+, 1996)+ The wild-type and mutant               <br /> forms of RPR1 RNA were cloned into the Sma I restriction             <br /> site of pIIIA-MS2-2 in which an RNA polymerase III leader/            <br /> promoter (from the RPR1 gene) directs transcription of the           <br /> hybrid RNA containing two tandem MS2 sites and the RPR1                <br /> terminator+ The pIIIA-MS2-2 plasmid is a modified version             <br /> of pIIIMS2-2 containing the ADE2 gene+ The open reading              <br /> frame of Pop1p was cloned between the Xma I and Xho I                   <br /> restriction sites of the pACTII plasmid, which produces a           <br /> hybrid protein linked to the GAL4 activation domain+ The                <br /> L40-coat yeast strain (Mat a, ura3-52, leu2-3, his3D200,                 <br /> trp1D1, ade2, LYS2 ::(lexAop)-HIS3, ura3 ::(lexA-op)-lacZ,          <br /> LexA-MS2 coat(TRP1)) was transformed with the pIIIA-                   <br /> MS2-2 plasmid containing no RNA insert or wild-type or                  <br /> mutant forms of the RPR1 RNA and pACTII-Pop1p ORF+                   <br /> Double transformants were selected for their ability to grow       <br /> on SD-ura-lys-trp-leu media+ Transformants were then tested          <br />                                                       <br /> for their ability to grow on SD-ura-lys-trp-leu-his media in           <br /> the absence or presence of 1-mM, 5-mM, 10-mM, and                     <br /> 20-mM 3-aminotriazole (Sigma)+ Double transformants were              <br /> also tested for their ability to produce b-galactosidase in           <br /> the presence of 5-bromo-4-chloro-3-indolyl b-galactoside by            <br /> colony color assay on nitrocellulose filters+ The pIIIA-MS2-2       <br /> and pACTII plasmids as well as the L40-coat yeast strain             <br /> were the kind gifts of the M+ Wickens laboratory+                         <br /> </body></html>