<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Nov. 2002, p. 6182&#226;&#8364;&#8220;6189                                                                             Vol. 184, No. 22 <br /> 0021-9193/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.184.22.6182&#226;&#8364;&#8220;6189.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       Involvement Two Putative Alternative Sigma Factors Stress <br />                 Response Radioresistant Bacterium <br />                         Deinococcus radiodurans <br />                                        Amy K. Schmid1 Mary E. Lidstrom1,2,3* <br />             Program Molecular Cellular Biology1 Departments Chemical Engineering2 Microbiology,3 <br />                                    University Washington, Seattle, Washington 98195-1750 <br />                                              Received 30 May 2002/Accepted 22 August 2002 <br />  <br />             Two genes bearing similarity alternative sigma factors identified Deinococcus radiodurans <br />          genome sequence designated sig1 sig2. These genes cloned inactivated, found <br />          important <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>survival</span> heat ethanol stress, sig1 mutants displayed severe <br />          <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> sig2 mutants. Reporter gene fusions groESL dnaKJ operons transformed into <br />          mutant backgrounds indicated sig1 required heat shock induction groESL dnaKJ, <br />          sig2 mutants moderate defect dnaKJ induction impaired groESL <br />          induction. Essentiality tests suggested sig1 sig2 essential conditions. Sequence <br />          comparisons demonstrated sig1 gene product classed distinctly extracytoplasmic function <br />          (ECF) sigma factors, Sig2 appears divergent sigma factor ortholog. These results suggest <br />          sig1 encodes major ECF-derived heat shock sigma factor D. radiodurans plays central <br />          role positive regulation heat shock genes. sig2, contrast, appears play minor role heat <br />          shock protection serve modulate expression heat protective genes. <br />  <br />  <br />    Deinococcus radiodurans extremely radioresistant, non-             Bacillus subtilis, transcribe stationary-phase osmotic <br /> pathogenic bacterium forms major known phy-                 stress (14), heat shock (11, 42), general stress-protective <br /> logenetic branches bacterial domain, Thermus                 (12) genes, respectively. A subfamily group 3 sigma factors, <br /> the closely related genus (3). The genome D. radio-                 extracytoplasmic function (ECF) sigma factors (22), controls <br /> durans sequenced annotated shown                  transcription genes involved mainly extracytoplasmic <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> contain high-G&#226;&#171;&#185;C genetic elements, including                    cell protective functions, including production aerial mycelia <br /> chromosomes, megaplasmid, plasmid (40). D. radio-                   Streptomyces coelicolor (4), virulence Salmonella enterica <br /> durans maintains 4 10 copies genome, depending                 serovar Typhimurium (18) osmotolerance heat shock <br /> the growth phase (2).                                                       survival B. subtilis (15, 16). Most bacterial genomes contain <br />    The radioresistance mechanisms D. radiodurans have <br />                                                                             vegetative sigma alternative sigma factor and <br /> been studied intensively organism&#226;&#8364;&#8482;s ability to <br />                                                                             ECF orthologs. <br /> survive acute dose 5 megarads &#226;?&#165;-irradiation and <br />                                                                                Surprisingly, searches complete annotated ge- <br /> up 1,000 J UV irradiation/m2 <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span> (2, 35). <br />                                                                             nome sequence D. radiodurans (40) revealed three <br /> Despite interest, little known organism <br /> responds environmental stressors. The studies               predicted sigma factors, including classed a <br /> D. radiodurans stress response systems indicate organ-             vegetative &#226;?&#180;70 (rpoD/sigA, DR0916), classed an <br /> ism mounts regulated protective response DNA dam-                 ECF subfamily member (sig1, DR0180), putative <br /> age induced &#226;?&#165;-irradiation (2, 35, 36), UV light (8),                 sigma factor (sig2, DR0804). Conspicuously missing or- <br /> oxidative stress (23, 39); however, mechanisms reg-             thologs nitrogen regulation (rpoN), general starvation <br /> ulation investigated.                                     osmotic stress (rpoS), general stress (sigB), heat shock <br />    In bacteria, transcriptional regulation genes in-               (rpoH) sigma factor genes. Other known stress-regulatory gene <br /> volved stress response mediated alternative sigma fac-             orthologs recognized genome sequence, <br /> tors, direct RNA polymerase specific stress promoter               including genes <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> heat shock negative regulators, <br /> sequences. Sigma factors divided major                   hrcA, ctsR, hspR (30), oxidative-stress regula- <br /> groups based sequence function, &#226;?&#180;70 &#226;?&#180;54                      tors oxyS (1) soxRS (34), osmotic-stress regulators <br /> classes. Of &#226;?&#180;70 class, groups 1 2 responsible               ompR. Such regulators absent the <br /> transcription housekeeping promoters, group 3                    genome divergent recognizable. This <br /> sigma factors (21), &#226;?&#180;S &#226;?&#180;H Escherichia coli &#226;?&#180;B            suggests stress response regulation D. radiodurans may <br />                                                                             differ significantly established paradigms. We <br />                                                                             sought identify function alternative <br />    * Corresponding author. Mailing address: Program Molecular            sigma factors D. radiodurans stress response regulation. <br /> and Cellular Biology, Department Chemical Engineering, Univer- <br /> sity Washington, Seattle, WA 98195-1750. Phone: (206) 616-5282.          Here report cloning, mutation, phenotypic charac- <br /> Fax: (206) 616-5721. E-mail: lidstrom@u.washington.edu.                     terization putative sigma factors. <br />  <br />                                                                      6182 <br />  VOL. 184, 2002                                                                              STRESS SIGMA FACTORS IN D. RADIODURANS                                  6183 <br />  <br />  <br />                                                   TABLE 1. Bacterial strains plasmids used study <br />                                                                                                                                                               Source or <br />   Strain plasmid                                                            Relevant genotype <br />                                                                                                                                                               reference <br />  <br /> E. coli <br />   JM109                                        (mcrA) recA1 endA1 gyrA96 thi-1 hsdR17(rK&#226;&#171;&#186; mK&#226;&#171;&#185;) supE44 relA1                                              Promega <br />                                                  &#226;&#338;&#172;(lac-proAB) [F&#226;&#172;&#732; traD36 proAB] lacIqZ&#226;&#338;&#172;M15] <br />   TOP10                                        F&#226;&#171;&#186; mcrA&#226;&#338;&#172; (mrr-hsdRMS-mcrBC) &#226;?&#190;80lacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;lacX74 recA1 deoR                                                Invitrogen <br />                                                  araD139&#226;&#338;&#172;(ara-leu)7679 galU galK rpsL(Strr) endA1 nupG <br /> D. radiodurans R1 <br />   R1                                           Wild-type strain                                                                                          J. R. Battista <br />   AKS10                                        R1 sig1::aph (Kmr) [Sig1&#226;&#171;&#186;]; result recombination pAKS10                                        This study <br />                                                 chromosome R1 <br />   AKS11                                        R1 sig2::aph (Kmr) [Sig2&#226;&#171;&#186;]; result recombination pAKS11                                        This study <br />                                                 chromosome R1 <br /> plasmids <br />   pCR2.1-TOPO                                  Cloning vector PCR-generated products, Apr Kmr, 3.9 kb; uses                                          Invitrogen <br />                                                  covalently linked topoisomerase instead DNA ligase <br />   pMTL23                                       General cloning vector E. coli; Apr, 2.5 kb                                                           5 <br />   pUC19                                        General cloning vector E. coli; Apr, 2.7 kb                                                           41 <br />   pAKS10                                       pMTL23 carrying D. radiodurans sig1 gene disrupted                                             This study <br />                                                  pCR2.1 KmR gene driven D. radiodurans groESL minimal <br />                                                  promoter; KmR ApR, 4.7kb <br />   pAKS11                                       Like pAKS10; carrying disrupted D. radiodurans sig2 gene                                              This study <br />   pAY/K2                                       pUC-based insertion vector carrying D. radiodurans amyE                                           24 <br />                                                  gene disrupted pUC4K Kmr gene; Apr Kmr <br />   pGROES4Z                                     D. radiodurans suicide vector disruption amyE containing                                       25 <br />                                                  groES promoter insert; Apr Kmr, 9 kb <br />   pRADZ1                                       Promoter probe vector uses lacZ reporter, derived                                          24 <br />                                                  pRAD1, 6.8 kb <br />   pRADZ3                                       pRADZ1 carrying R1 groESL promoter                                                                    25 <br />   pRADZ8                                       pRADZ1 carrying R1 dnaK promoter                                                                      This study <br />  <br />  <br />  <br />  <br />                         MATERIALS AND METHODS                                             Disruption sig1 sig2 genes. pAKS10 pAKS11 insertion vectors for <br />                                                                                        purpose disrupting sig1 sig2 genes constructed follows. First, <br />    Bacterial strains plasmids. The bacterial strains plasmids used this <br />                                                                                        sig1 (DR0180) sig2 (DR0804) open reading frames (ORF) flanking <br /> study listed Table 1. <br />                                                                                        regions PCR amplified D. radiodurans genomic DNA fol- <br />    Media growth conditions. E. coli cultures grown Luria-Bertani <br />                                                                                        lowing primers: 0180 fwd (TGGCTACGACCCCCTGT) 0180 rev (GGT- <br /> broth Luria-Bertani agar 37&#194;&#176;C; D. radiodurans cultures grown in <br /> TGY broth TGY agar 30&#194;&#176;C (29). Ampicillin (100 &#226;?&#174;g/ml), kanamycin (50           CAGCCAGAAGCGGT) amplify sig1 0804 fwd (CTGCTGGTCT- <br /> &#226;?&#174;g/ml E. coli, 20 &#226;?&#174;g/ml screening sig1 sig2 mutants plates    TCGCTGCT) 0804 rev (GGGGCACTCTTCCCATTT) amplify sig2. sig1 <br /> D. radiodurans, 10 &#226;?&#174;g/ml routine growth plates, 8 &#226;?&#174;g/ml growth       sig2 PCR products cloned pCR2.1-TOPO (Invitrogen) then <br /> liquid), chloramphenicol (3 &#226;?&#174;g/ml D. radiodurans) added culture          transferred KpnI-XhoI fragment pMTL23 (5), yielding pSIG123 and <br /> media appropriate. Transformations E. coli performed commer-           pSIG223, respectively. To construct kanamycin resistance (Kmr) cassette, the <br /> cially available cells (TOP10 JM109 Invitrogen, Carlsbad, Calif.,         BamHI-BglII groESL promoter fragment pGROES4Z (25) cloned into <br /> Promega, Madison, Wis., respectively). Transformations D. radiodurans          unique BglII site pCR2.1 upstream promoterless Kmr marker. The <br /> performed electroporation (H. Rothfuss, unpublished data). Electrocompe-            groESL-Kmr fusion PCR amplified blunt-ended, restriction-tagged <br /> tent cells prepared washing 1 mM HEPES previously described (9).         primers cloned pCR2.1-TOPO generate pPgroK. The groESL-Kmr <br /> TGY broth (900 &#226;?&#174;l) added immediately electroporation, cells         fusion removed pPgroK EcoRV-Eco47III fragment and <br /> allowed recover 16 h plating selective medium.                        inserted T4 DNA polymerase-filled, unique EcoNI site pSIG123 or <br />    DNA manipulations. DNA isolated E. coli alkaline lysis              unique Eco47III site pSIG223, yielding pAKS10 pAKS11, respec- <br /> method according Sambrook Russel (32). D. radiodurans chromosomal               tively. pAKS10 pAKS11 transformed D. radiodurans R1 to <br /> DNA Southern analysis PCR amplification prepared using                disrupt chromosomal copies sig1 sig2, respectively. Transformants <br /> detergent lysis method follows. Cells collected overnight cultures     screened double-crossover mutation diagnostic colony PCR analysis <br /> centrifugation (16,000 &#226;&#171;&#187; g 2 min) resuspended 500 &#226;?&#174;l T50E50 buffer       using primers 0180 304 bp ups (TGCTGGTCCTCGCCTAT) 0180 171 <br /> (50 mM Tris-HCl, 50 mM EDTA [pH 8.0]) supplemented lysozyme (2                    bp dns (AACGCGCTGATGTCAGG) screen disruption sig1, 0804 <br /> mg/ml) NaCl (0.15 M). The suspension incubated 37&#194;&#176;C 1 h.                fwd 0804 rev (see above) used screen disruption sig2. Positive <br /> Proteinase K (0.6 mg/ml; Roche Diagnostics Corp., Mannheim, Germany)               clones retested Southern blot analysis DrdI-digested chromosomal <br /> Streptomyces griseus protease (0.6 mg/ml; Sigma Chemical Corp., St. Louis, Mo.)        DNA using enhanced chemiluminescence (ECL) kit Amersham Phar- <br /> were added, suspension incubated 37&#194;&#176;C hour.            macia Biotech (Piscataway, N.J.) according manufacturer&#226;&#8364;&#8482;s protocol. <br /> After lysis 100 &#226;?&#174;l 10% sodium dodecyl sulfate, aqueous phase              Stress survival assays. To characterize stress conditions wild type, <br /> extracted twice equal volume phenol-chloroform-isoamyl alcohol              cultures D. radiodurans R1 grown mid-exponential phase (optical <br /> (24:24:1) (Roche Corp.), chromosomal DNA precipitated 0.5                 density 600 nm, &#226;&#172;&#402;0.3 0.4) shifted range stress conditions. After <br /> volume isopropanol. Finally, pelleted DNA resuspended 100 &#226;?&#174;l          4 h exposure stress, survival assessed plating serial dilutions 50 <br /> T10E1 (10 mM Tris-HCl, 1 mM EDTA [pH 8.0]) stored 4&#194;&#176;C.                          mM phosphate buffer duplicate TGY agar. After 3 days growth 30&#194;&#176;C, <br />    Restriction enzymes obtained New England Biolabs (Beverly,                surviving colonies counted. Conditions resulted 10- 100-fold loss <br /> Mass.), Roche Corp. (Indianapolis, Ind.), Promega used              survival 4 h chosen stress conditions D. radiodurans R1 <br /> supplier described. T4 polymerase T4 DNA ligase purchased New            used comparison sigma mutants subsequent experiments (see <br /> England Biolabs. Taq polymerase custom primers obtained Gibco            below). <br /> BRL (Grand Island, N.Y.).                                                                 To characterize sigma mutant phenotypes, D. radiodurans R1, AKS10 (sig1 <br />  6184       SCHMID AND LIDSTROM                                                                                                              J. BACTERIOL. <br />  <br /> mutant), AKS11 (sig2 mutant) grown mid-exponential phase 30&#194;&#176;C           EcoNI site double-crossover allelic exchange insertion; 217 <br /> in TGY selection shifted stress dividing culture      bp DNA sig1 gene left upstream insertion <br /> two aliquots. One aliquot induced stress follows: heat shock was <br /> induced transfer 48&#194;&#176;C, salt stress induced addition NaCl final <br />                                                                                        563 bp left downstream. sig2 disrupted insertion <br /> concentration 1.2 M, acid shock induced addition 0.3 N HCl shift       kanamycin resistance marker unique <br /> the pH culture 6.8 3.7, oxidative stress induced          Eco47III site; 546 bp DNA sig2 gene left upstream <br /> addition 120 mM H2O2. The second aliquot continued 30&#194;&#176;C                insertion 299 left downstream. <br /> nonstress control. After induction stress, cultures plated 2 h to <br />                                                                                           Cultures transformed kanamycin resistance pAKS10 <br /> 8 h assess survival. Survival expressed CFU milliliter. <br />    Cloning dnaK promoter. The putative dnaKJ promoter PCR am-               pAKS11 screened 20 &#226;?&#174;g kanamycin ml for <br /> plified D. radiodurans genome 564-bp fragment using               double-crossover insertion sig1 sig2 diagnostic <br /> following primers: dnaK_fwd2 (CGCACCAATCCGCACCT), anneals                        colony PCR strategy using primers flanking gene inter- <br /> 468 bp upstream predicted dnaK (DR0129) start codon, dnaK_rev2              est. Because D. radiodurans contains multiple copies its <br /> (CGGCGTTGACGATCACT), anneals 89 bp downstream pre- <br /> dicted ATG. The PCR fragment cloned pCR2.1-TOPO (Invitrogen) <br />                                                                                        genome (2), potential retain wild-type and <br /> and subcloned BglII fragment pRADZ1, yielding pRADZ8 (Table 1).              mutated copies. This PCR strategy detects allelic differences at <br />    &#226;?&#164;-Galactosidase assays. D. radiodurans cultures expressing lacZ perme-         locus interest, allowing identify clones are <br /> abilized toluene previously described (25), quantitative analyses       homozygous insertion. Approximately half clones <br /> lacZ expression conducted according Miller (26). Results expressed <br />                                                                                        tested sig1 sig2 insertions heterozy- <br /> in Miller units, averages &#226;&#171;&#190; standard deviations replicate <br /> experiments reported.                                                              gous, maintaining insertion high copy numbers also <br />    Sequence comparison predictions. Computational analyses DNA              wild-type gene low copy numbers. Sequencing the <br /> predicted amino acid sequences performed using internet-based pro-             PCR products confirmed identity. One-sixth clones <br /> grams follows. D. radiodurans sequence data obtained com-             did contain insertion presumably spontaneous <br /> pleted annotated genome Institute Genomic Research (http:// <br /> www.tigr.org) (40). Sequences bacteria obtained GenBank <br />                                                                                        kanamycin-resistant clones. However, one-third the <br /> (http://www.ncbi.nlm.nih.gov) Pasteur Institute (http://genolist.pasteur.fr).   clones tested homozygously null, genotype was <br /> Multiple alignments comparing sequences D. radiodurans          verified Southern analysis. One representative homozygous <br /> performed using ClustalW 1.7 (http://searchlauncher.bcm.tmc.edu) analyzed          clone mutation chosen subsequent ex- <br /> with BoxShade software (http://www.ch.embnet.org/software/BOX_form.html). <br />                                                                                        periments. Single crossover recombinants obtained at <br />                                                                                        sig1or sig2 mutant loci, contrast our <br />                                                                                        previous experience insertion DR0405 locus (H. <br />                                   RESULTS <br />                                                                                        Rothfuss, unpublished). <br />    Identification, cloning, disruption D. radiodurans al-                          Assessment sig1 sig2 gene essentiality. Because E. coli <br /> ternative sigma factors. Two ORFs predicted encode alter-                           rpoE known essential ECF sigma factor gene (7) and <br /> native sigma factors identified complete an-                           D. radiodurans sig1 sig2 genes class ECF <br /> notated genome sequence D. radiodurans R1 (40). The first,                          sigma factors, possible sig1 sig2 <br /> DR0180, designated sig1, 687-bp predicted ORF                            genes essential growth conditions. Since <br /> whose product bears identity ECF family sigma fac-                           E. coli rpoE mutants require acquisition suppressor mu- <br /> tors (22), highest identity (38%) Mycobacterium                            tation second site order grow, form colonies <br /> tuberculosis sigE. The second, DR0804, designated                             times slower following transformation mutants car- <br /> sig2, 846-bp predicted ORF deduced amino acid                               rying complementing gene trans (7). Therefore, D. <br /> sequence bears similarity sigma factors, 31%                             radiodurans sig1 sig2 essential genes, expected <br /> identity putative Magnetococcus sp. ECF-like sigma factor.                        heterozygous clones form larger colonies faster trans- <br /> Genome searches using conserved sigma factor regions 2                             formation homozygous null clones. However, differ- <br /> and 4 (10, 13) did identify potential sigma factor                       ences colony size time colony formation observed <br /> genes.                                                                                 homozygous null heterozygous mutants (data not <br />    Both sig1 sig2 genes cloned PCR products am-                            shown). <br /> plified D. radiodurans genomic DNA disrupted                                  To test possibility gene essentiality, com- <br /> allelic exchange insertion kanamycin resistance marker                         pared efficiency disruption sig1 sig2 a <br /> described Materials Methods. One difficulties                        nonessential amylase gene (amyE) (25). D. radiodurans cells <br /> gene disruption strain high natural resis-                        transformed pAKS10, pAKS11, pAY/K2, which <br /> tance D. radiodurans R1 kanamycin, 4 &#226;?&#174;g kana-                           disrupt sig1, sig2, amyE, respectively; efficiency of <br /> mycin ml (R. Meima, unpublished data) low-level                            transformation assessed. Since acquisition compensa- <br /> kanamycin resistance obtained existing cassettes, 8 &#226;?&#174;g                         tory mutation depends background mutation rate <br /> kanamycin ml (25). The result high percentage                              (&#226;&#172;&#402;1/106), expected observe orders magnitude fewer <br /> colonies insert spontaneous resistance. There-                          transformants pAKS10 pAKS11 pAY/K2 if <br /> fore, new kanamycin cassette constructed D. radio-                           sig1 sig2 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>null mutants</span> acquired second site suppressor <br /> durans, kanamycin resistance marker driven pre-                          mutation survive. However, insertion amyE only <br /> viously characterized D. radiodurans groESL minimal                                    10-fold efficient insertion sig1 about <br /> promoter region, contains &#226;?&#180;70-controlled major                          5-fold efficient sig2 (Table 2). This trend re- <br /> start site (25). This new cassette allowed screening concen-                        flects <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> DNA flanking regions available for <br /> trations kanamycin 25 &#226;?&#174;g/ml, greatly reduced                         recombination allelic exchange, factor ob- <br /> the number colonies arising spontaneous resistance.                            served affect recombination frequencies D. radiodurans. <br /> To disrupt sig1, new cassette inserted unique                          Since pAKS10 carries 300-bp flanking sequences recombi- <br />  VOL. 184, 2002                                                            STRESS SIGMA FACTORS IN D. RADIODURANS                       6185 <br />  <br />  <br />  TABLE 2. Comparison insertion efficiencies sig1 sig2 <br />          versus nonessential amylase gene (amyE) <br /> Gene                 Plasmid               Transformants/&#226;?&#174;g DNA <br />  <br /> amyE                pAY/K2                      5,501 &#226;&#171;&#190; 370.0 <br /> sig1                pAKS10                        476 &#226;&#171;&#190; 230.0 <br /> sig2                pAKS11                      1,703 &#226;&#171;&#190; 967 <br />  <br />  <br />  <br />  <br /> nation sig1, pAKS11 500-bp flanking sequences for <br /> insertion sig2, pAY/K2 carries 700-bp flanking se- <br /> quences disruption amyE, variability transforma- <br /> tion efficiency observed insertion plasmids is <br /> consistent differences flanking-region length. Taken <br /> together, data suggest sig1 sig2 be <br /> essential genes, mutants appear ac- <br /> quired suppressor mutations second site order sur- <br /> vive. <br />    Determination stress conditions. Since previously pub- <br /> lished studies tested survival UV (8), gamma irradia- <br /> tion (35), oxidative stress (23, 39) D. radiodurans, stress <br /> survival assessed osmotic, ethanol, acid, and <br /> heat stress conditions described Materials Methods. <br /> The condition caused 10- 100-fold loss survival and <br /> significantly impaired growth wild-type D. radiodurans dur- <br /> ing 4 h cultivation following shift stress chosen as <br /> the stress condition subsequent experiments mutant <br /> strains. These conditions 1.2 M NaCl, 20% ethanol, 120 <br /> mM H2O2 (39), pH 3.7, 48&#194;&#176;C. <br />    The role sig1 sig2 stress protection. To determine <br /> the involvement sig1 sig2 protective stress response, <br /> the mutants tested growth survival compared to <br /> the wild type stress conditions noted above. No <br /> growth differences D. radiodurans R1 wild type and <br /> the sig1 sig2 mutants observed 30&#194;&#176;C (data               FIG. 1. Survival wild-type D. radiodurans R1 sig1 (AKS10) <br /> shown). In addition, differences survival          sig2 (AKS11) mutant strains stress conditions. (A) Survival <br /> mutants wild type observed salt, oxidative,        R1 mutants heat shock. (B) Survival R1 mutants <br /> acid stress conditions (data shown). However, cells          ethanol stress. R1, AKS10, AKS11 grown TGY <br />                                                                      broth 30&#194;&#176;C mid-exponential phase (optical density 600 nm &#226;&#171;&#189; <br /> defective sig1 exhibited impairment survival 3        0.3 0.4) shifted heat shock 48&#194;&#176;C (time zero) ethanol <br /> orders magnitude compared wild type following heat             stress addition 20% ethanol. Samples removed <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> shock (Fig. 1A) 2 orders magnitude shift     culture 2 h, serial dilutions plated assess <br /> ethanol stress (Fig. 1B). Cells mutant sig2       survival. Error bars represent standard deviations averages of <br /> susceptible heat ethanol stress wild-type cells      experiments. <br /> less sensitive heat ethanol stress sig1 mutants (Fig. <br /> 1). Taken together, results suggest sig1 and <br /> sig2 gene products involved survival following heat       promoterless lacZ transcriptional fusions groESL and <br /> ethanol stress. However, sig1 gene product           dnaKJ promoters, respectively. The pRADZ1 vector, <br /> more important heat shock ethanol stress survival        encodes low &#226;?&#164;-galactosidase activity induced <br /> the sig2 gene product.                                               heat shock, transformed strains a <br />    Expression heat shock genes sig1 sig2 back-         control. The pRADZ1 construct present same <br /> grounds. In E. coli bacteria, heat shock-           copy number chromosomes D. radiodurans (24). <br /> specific sigma factor responsible transcription num-     Transformants grown mid-exponential phase 30&#194;&#176;C <br /> ber heat shock genes, including chaperone operons         subsequently shifted heat shock 40&#194;&#176;C mid- <br /> groESL dnaKJ heat stress conditions (42). To de-           exponential phase induce promoter-reporter activity. In this <br /> termine sig1 sig2 functions similarly heat       experiment, 40&#194;&#176;C used heat shock temperature <br /> stress response D. radiodurans, expression groESL          instead 48&#194;&#176;C avoid heat instability &#226;?&#164;-galactosidase. <br /> (25) dnaKJ (see Materials Methods) promoters               At 30&#194;&#176;C, basal level &#226;?&#164;-galactosidase activity the <br /> assessed sig1 sig2 backgrounds. Mutant wild-          groESL promoter remained relatively constant (500 700 <br /> type D. radiodurans strains transformed pRADZ3             Miller units) D. radiodurans R1, sig1 sig2 strains <br /> (24) pRADZ8 (Table 1), promoter-probe vectors carrying            course experiment (Fig. 2A). Within 60 <br />  6186      SCHMID AND LIDSTROM                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                                                           bility tested using control, vector pRADZ1, which <br />                                                                           exhibits low-level constitutive expression &#226;?&#164;-galactosidase. <br />                                                                           Activity pRADZ1 sig1, sig2, R1 wild-type strains <br />                                                                           remained constant approximately 60 Miller units both <br />                                                                           nonstress heat shock conditions 60 min of <br />                                                                           experiment (data shown). However, 2 h shift to <br />                                                                           heat shock, &#226;?&#164;-galactosidase activity mutants decreased <br />                                                                           1.6-fold, wild type maintained activity (not <br />                                                                           shown). Similarly, &#226;?&#164;-galactosidase activity the <br />                                                                           groESL dnaKJ promoters heat shock remained at <br />                                                                           30&#194;&#176;C background level hour the <br />                                                                           experiment sig1 sig2 mutants, prolonged exposure <br />                                                                           (&#226;&#177;&#8211;2 h) mutant strains elevated temperature re- <br />                                                                           sulted significant reduction enzyme activity (Fig. 2). This <br />                                                                           suggests results hour exposure reflect <br />                                                                           promoter activity, later time points reflect combi- <br />                                                                           nation promoter activity &#226;?&#164;-galactosidase instability. <br />                                                                              Taken together, results promoter-probe fusion <br />                                                                           experiments suggest (i) Sig1 required heat shock <br />                                                                           induction groESL dnaKJ promoters, (ii) Sig2 is <br />                                                                           involved dnaKJ groESL induction, (iii) stability <br />                                                                           &#226;?&#164;-galactosidase, presumably proteins, compro- <br />                                                                           mised mutant strains prolonged exposure ele- <br />                                                                           vated temperatures. <br />                                                                              ClustalW analysis sig1 sig2. Although ECF <br />                                                                           sigma factors known transcribe cytoplasmic genes <br />                                                                           (27), example ECF family member known to <br />                                                                           transcribe cytoplasmic heat shock chaperones SigH M. <br />                                                                           tuberculosis, controls clpB dnaK transcription (31). <br />                                                                           It surprising Sig1 function cyto- <br />                                                                           plasmic heat shock sigma factor despite original annotation <br />                                                                           ECF-like sigma factor. We compared Sig1 <br />                                                                           Sig2 amino acid sequences group 3 &#226;?&#180;70 family <br />    FIG. 2. Induction &#226;?&#164;-galactosidase activity sig1 (AKS10)      members (21) ECF subfamily members using Clust- <br /> sig2 (AKS11) mutant wild-type (R1) strains carrying plasmid-        alW 1.7 program (http://searchlauncher.bcm.tmc.edu) (Fig. 3). <br /> borne lacZ transcriptional fusion groESL (A) dnaKJ (B). Heat        Since D. radiodurans divergent organism based 16S <br /> shock protocol Fig. 2, temperature shifted to <br />                                                                           rRNA sequencing, representative sigma factor sequences were <br /> 40&#194;&#176;C. Duplicate samples removed 0, 30, 60, 120 min <br /> postinduction assess &#226;?&#164;-galactosidase activity. Error bars represent    selected wide range bacteria multiple se- <br /> standard deviations averages experiments. Error bars    quence alignment. As shown Fig. 3, Sig1 amino acid <br /> for 30&#194;&#176;C background data points overlap (not shown).                      sequence appears cluster closely ECF sigma fac- <br />                                                                           tors sigma factors group 3 &#226;?&#180;70 class. In addition, <br />                                                                           conservation Sig1 ECF sigma factors seems <br /> min shift heat shock 40&#194;&#176;C, however, expression lacZ           highest region 4.2, thought responsible for <br /> from groESL promoter induced approximately twofold                binding &#226;&#171;&#186;35 region promoters (19, 20, 33). These <br /> in R1 (wild type), increase activity slowed         results correlate initial BLAST analysis, indi- <br /> reach maximum 2 h postinduction (Fig. 2A) (25). In                 cated Sig1 38% identical amino acid level M. <br /> contrast, sig1 mutants grown 40&#194;&#176;C failed induce groESL              tuberculosis SigE, hits group 3 &#226;?&#180;70 factors <br /> expression 30&#194;&#176;C background level course              (data shown). <br /> of experiment (Fig. 2A). Similarly, dnaKJ expres-               Sig1 exhibits features typical ECF sigma factor <br /> sion wild type increased twofold 40&#194;&#176;C, sig1 mutants             subfamily member (22). First, Sig1 sequence contains a <br /> were impaired dnaKJ induction heat shock (Fig. 2B).            large gap alignment region 3, indicating this <br />    In contrast, sig2 mutants transformed pRADZ3                 region shorter group 3 &#226;?&#180;70 family members (Fig. <br /> not impaired induction groESL compared wild                 3). Second, like ECF factors, Sig1 lacks hydrophobic <br /> type, exhibiting twofold induction groESL 60 min           residues region 2.3. Taken BLAST results, <br /> shift heat stress (Fig. 2A). However, sig2 mutants showed            sequence comparison suggests Sig1 true sigma <br /> slight impairment induction dnaKJ expression            factor evolutionarily derived ECF family <br /> heat shock compared wild type (Fig. 2B).                           ancestor. <br />    One difficulty interpreting results lack          The Sig2 amino acid sequence divergent the <br /> induction chaperones mutants affect stability          sigma factors (Fig. 3) (28). The Sig2 sequence showed <br /> &#226;?&#164;-galactosidase, invalidating use reporter. This possi-          highest number conserved residues region 2, is <br />  VOL. 184, 2002                                                                   STRESS SIGMA FACTORS IN D. RADIODURANS                        6187 <br />  <br />  <br />  <br />  <br />    FIG. 3. ClustalW alignment D. radiodurans Sig1 Sig2 amino acid sequences characterized ECF sigma factors and <br /> group 3 &#226;?&#180;70 family alternative sigma factors. Identical residues shaded black, similar residues shaded gray, gaps indicated by <br /> dashes. Lines numbers alignment indicate conserved regions 2 4. Region 1 included ECF sequences <br /> lack region. Numbers left sequence represent amino acid residue number relative sequence start. Sequences E. <br /> coli (Eco), B. subtilis (Bsu), M. tuberculosis (Mtu) obtained Pasteur Institute Internet genome sites, located http: <br /> //genolist.pasteur.fr. D. radiodurans (Dra) Sig1 (DR0180) Sig2 (DR0804) sequences obtained Institute Genomic Research <br /> (www.tigr.org) (40). Myxococcus xanthus (Mxa) SigB obtained GenBank, accession number X55500. <br />  <br />  <br />  <br /> responsible sigma subunit interaction RNA poly-                response regulatory systems bacterium absent <br /> merase core (38). Region 2 highly conserved                highly divergent. Therefore, study role sigma <br /> among sigma factor sequences included alignment.                 factors stress response regulation undertaken the <br /> This evidence suggests Sig2 encode sigma factor,                 organism. Our results suggest recognizable <br /> but characterization required confirm              alternative sigma factor orthologs genome (sig1 sig2) <br /> hypothesis.                                                                 involved survival following shift ethanol heat <br />                                                                             stress. In bacteria stresses involve accu- <br />                                                                             mulation misfolded proteins, likely the <br />                            DISCUSSION <br />                                                                             regulon senses and/or responds signal. The difference in <br />    Searches D. radiodurans genome indicated                survival mutants wild type demonstrates <br /> although stress response genes appear present, stress                 D. radiodurans mounts protective response heat <br />  6188     SCHMID AND LIDSTROM                                                                                                            J. BACTERIOL. <br />  <br />  <br /> and ethanol stress, novel finding, previous            We performed essentiality tests sig1 sig2 compar- <br /> reports focused protective response         ing transformation efficiencies nonessential gene. <br /> DNA damage (2, 8, 23, 36, 39).                                       Our results suggestive sig1 sig2 essential <br />    Phenotypic characterization mutants suggests        conditions, definitive proof require trans- <br /> Sig1 required positive regulation groESL, dnaKJ,       formation mutating vector presence absence <br /> and presumably genes products protect protein            wild-type gene trans. Attempts perform experi- <br /> stability elevated temperature. This function similar       ment successful interference highly <br /> that E. coli &#226;?&#180;H, 32-kDa heat shock factor            efficient recombination D. radiodurans (data not <br /> responsible transcription heat shock chaperone        shown). In addition, tightly regulated inducible promoter is <br /> protease genes groESL, dnaKJ, clpXP, (11,         available titrate gene activity D. radiodurans. <br /> 42). However, ClustalW results indicate D. radio-       Therefore, definitive proof gene essentiality system <br /> durans Sig1 sequence classed distinctly ECF sigma       awaits amenable genetic tools. <br /> factors, E. coli &#226;?&#180;H known classed group 3         During course phenotypic characterization the <br /> &#226;?&#180;70 factors (21) (Fig. 3). These results suggest sig1       sigma mutants, observed D. radiodurans R1 survived <br /> encode major heat shock sigma factor D. radiodurans           better organisms stress. For example, <br /> responsible positive regulation heat shock genes     D. radiodurans R1 loses 10-fold survival 20% ethanol <br /> as groESL dnaKJ Sig1 alternative sigma factor       1.2 M NaCl 8 h cultivation. In contrast, B. subtilis <br /> appears derived ECF family ancestor.            168 survival reported decrease 100-fold in <br /> The example M. tuberculosis,       9% ethanol 4 h 150-fold 200 mM salt 3 h (37). <br /> a high-G&#226;&#171;&#185;C gram-positive organism alternative            In addition, B. subtilis shows 100-fold loss survival within <br /> sigma factor transcribes classical chaperones dnaK       2 h shift pH 7 pH 4.3. D. radiodurans, other <br /> clpB belongs ECF family (31). However,       hand, withstand pH 3.7 4 h 10-fold loss of <br /> we present example ECF-like sigma factor in-         survival (data shown). Out stress conditions have <br /> volved transcription groESL. Together, ECF-          tested, heat shock stress D. radio- <br /> derived alternative sigma factors represent novel class     durans exhibits susceptibility typical soil bacteria (37). <br /> heat shock regulators high-G&#226;&#171;&#185;C organisms. Although             From observations, conclude D. radiodurans is <br /> no obvious ECF consensus groESL puta-            extremely resistant environmental stress conditions <br /> tive dnaKJ promoter sequences, ECF promoters tend              compared soil organisms, B. subtilis. The lack <br /> more conserved organism organisms             recognizable stress response regulators D. radiodurans <br /> (6, 17, 27, 31). In addition, D. radiodurans promoters general    genome sequence suggests D. radiodurans employ <br /> appear divergent, consensus promoter           novel mechanisms stress survival represents new <br /> type remains unclear (25).                                           model organism understanding diversity stress re- <br />    In contrast sig1 mutants, sig2 mutants exhibit heat        sponse regulation bacteria. <br /> ethanol survival moderately impaired relative to <br /> those wild type level dnaKJ heat shock                                       ACKNOWLEDGMENTS <br /> induction intermediate wild type sig1       We thank Heather Rothfuss Rob Meima useful discussions <br /> mutants. Sig2 shown involved groESL            Franc&#194;&#184;ois Baneyx helpful comments manuscript. <br /> regulation. Our ClustalW results indicate Sig2 se-            This work funded University Washington Molecular <br />                                                                      Cellular Biology Training Grant (GM0720) grant the <br /> quence divergent closely related        DOE EMSP program (ER20294). <br /> ECF factors group 3 &#226;?&#180;70 factors (28). Our results far <br /> suggest sig2 gene product divergent putative                                        REFERENCES <br /> sigma factor plays minor role protection       1. Altuvia, S., D. Weinstein-Fischer, A. Zhang, L. Postow, G. Storz. 1997. <br />                                                                          A small, stable RNA induced oxidative stress: role pleiotropic reg- <br /> heat ethanol stress D. radiodurans.                               ulator antimutator. Cell 90:43&#226;&#8364;&#8220;53. <br />    It surprising alternative sigma factor or-        2. Battista, J. R. 1997. Against odds: survival strategies Deinococcus <br /> thologs genome, bacteria              radiodurans. Annu. Rev. Microbiol. 51:203&#226;&#8364;&#8220;224. <br />                                                                       3. Battista, J. R., A. M. Earl, O. White. 2000. The stress responses of <br /> with similar genome size contain considerably sigma               Deinococcus radiodurans, p. 383&#226;&#8364;&#8220;391. In G. Storz R. Hengge-Aronis <br /> factor genes. It surprising alternative sigma       (ed.), Bacterial stress responses. ASM Press, Washington, D.C. <br />                                                                       4. Bibb, M. J., V. Molle, M. J. Buttner. 2000. &#226;?&#180;BldN, extracytoplasmic <br /> factor orthologs D. radiodurans appear protect         function RNA polymerase sigma factor required aerial mycelium forma- <br /> same stress. It possible other, divergent sigma             tion Streptomyces coelicolor A3(2). J. Bacteriol. 182:4606&#226;&#8364;&#8220;4616. <br /> factor orthologs involved protection stresses     5. Chambers, S. P., S. E. Prior, D. A. Barstow, N. P. Minton. 1988. The <br />                                                                          pMTL nic-cloning vectors. I. Improved pUC polylinker regions facilitate <br /> other heat ethanol stress present genome,            use sonicated DNA nucleotide sequencing. Gene 68:139&#226;&#8364;&#8220;149. <br /> but unlikely far sigma factors con-       6. Dartigalongue, C., D. Missiakas, S. Raina. 2001. Characterization the <br /> served bacterial domain (21). The D. radio-               Escherichia coli &#226;?&#180;E regulons. J. Biol. Chem. 276:20866&#226;&#8364;&#8220;20875. <br />                                                                       7. De las Pen&#203;&#339; as, A., N. Connolly, C. A. Gross. 1997. &#226;?&#180;E essential sigma <br /> durans genome searched conserved regions 2 4             factor Escherichia coli. J. Bacteriol. 197:6862&#226;&#8364;&#8220;6864. <br /> sig1 sig2, genes significant identity        8. Earl, A. M., S. K. Rankin, K.-P. Kim, O. N. Lamendola, J. R. Battista. <br />                                                                          2002. Genetic evidence uvsE gene product Deinococcus radio- <br /> these regions found. Because genes encoding                   durans R1 UV damage endonuclease. J. Bacteriol. 194:1003&#226;&#8364;&#8220;1009. <br /> known regulators stress response, soxRS, rpoS,         9. Gerhardt, P., R. G. E. Murray, W. A. Wood, N. R. Krieg. 1994. Methods <br /> ompR, recognizable D. radiodurans genome,              general molecular bacteriology. ASM Press, Washington, D.C. <br />                                                                      10. Gribskov, M., R. R. Burgess. 1986. Sigma factors E. coli, B. subtilis, <br /> remains determined stresses regulated             phage SP01, phage T4 homologous proteins. Nucleic Acids Res. <br /> the organism.                                                            14:6745&#226;&#8364;&#8220;6763. <br />  VOL. 184, 2002                                                                                STRESS SIGMA FACTORS IN D. RADIODURANS                                 6189 <br />  <br /> 11. Grossman, A. D., J. W. Erickson, C. A. Gross. 1984. The htpR gene                    sigma factors 31 completely sequenced genomes. J. Mol. Microbiol. Bio- <br />     product E. coli sigma factor heat-shock promoters. Cell 38:383&#226;&#8364;&#8220;              technol. 4:77&#226;&#8364;&#8220;91. <br />     390.                                                                               29.   Murray, R. G. E. 1992. The family Deinococcaceae, p. 3732&#226;&#8364;&#8220;3744. In A. <br /> 12. Hecker, M., U. Vo  &#194;&#168;lker. 2001. General stress response Bacillus subtilis         Balows, H. G. Tru    &#194;&#168;per, H. Dworkin, W. Harder, K.-H. Schleifer (ed.), <br />     bacteria. Adv. Microb. Physiol. 44:35&#226;&#8364;&#8220;91.                                      The prokaryotes, 2nd ed. Springer Verlag, New York, N.Y. <br /> 13. Helmann, J. D., M. J. Chamberlin. 1998. Structure function              30.   Narberhaus, F. 1999. Negative regulation bacterial heat shock genes. Mol. <br />     bacterial sigma factors. Annu. Rev. Biochem. 57:839&#226;&#8364;&#8220;872.                                 Microbiol. 31:1&#226;&#8364;&#8220;8. <br /> 14. Hengge-Aronis, R. 1996. Back log phase: sigma S global regulator        31.   Raman, S., T. Song, X. Puyang, S. Bardaraov, W. R. Jacobs, Jr., R. N. <br />     osmotic control gene expression E. coli. Mol. Microbiol. 21:887&#226;&#8364;&#8220;893.           Husson. 2001. The alternative sigma factor SigH regulates major compo- <br /> 15. Horsburgh, M. J., A. Moir. 1999. Sigma M, ECF RNA polymerase                      nents oxidative heat stress responses Mycobacterium tuberculosis. J. <br />     sigma factor Bacillus subtilis 168, essential growth survival           Bacteriol. 183:6119&#226;&#8364;&#8220;6125. <br />     high concentrations salt. Mol. Microbiol. 32:41&#226;&#8364;&#8220;50.                             32.   Sambrook, J., D. W. Russel. 2001. Molecular cloning: laboratory <br /> 16. Huang, X., A. Decatur, A. Sorokin, J. D. Helmann. 1997. The Bacillus                 manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, <br />     subtilis &#226;?&#180;X protein extracytoplasmic function &#226;?&#180; factor contributing             N.Y. <br />     survival high temperature. J. Bacteriol. 179:2915&#226;&#8364;&#8220;2921.                         33.   Siegele, D. A., J. C. Hu, W. A. Walter, C. A. Gross. 1989. Altered <br /> 17. Huang, X., J. D. Helmann. 1998. Identification target promoters               promoter recognition mutant forms sigma 70 subunit Escherichia <br />     Bacillus subtilis &#226;?&#180;X factor using consensus-directed search. J. Mol. Biol.         coli RNA polymerase. J. Mol. Biol. 206:591&#226;&#8364;&#8220;603. <br />     279:165&#226;&#8364;&#8220;173.                                                                       34.   Storz, G., M. Zheng. 2000. Oxidative stress, p. 47&#226;&#8364;&#8220;59. In G. Storz R. <br /> 18. Humphreys, S., A. Stevenson, A. Bacon, A. B. Weinhardt, M. Roberts.                  Hengge-Aronis (ed.), Bacterial stress responses. ASM Press, Washington, <br />     1999. The alternative sigma factor, &#226;?&#180;E, critically important viru-            D.C. <br />     lence Salmonella typhimurium. Infect. Immun. 67:1560&#226;&#8364;&#8220;1568. <br />                                                                                        35.   Sweet, D. M., B. E. B. Moseley. 1976. The resistance Micrococcus <br /> 19. Kenney, T. J., K. York, P. Youngman, C. P. Morgan, Jr. 1989. Genetic <br />                                                                                              radiodurans killing mutation agents damage DNA. Mutat. <br />     evidence RNA polymerase associated sigma A factor uses sporu- <br />                                                                                              Res. 34:175&#226;&#8364;&#8220;186. <br />     lation-specific promoter Bacillus subtilis. Proc. Natl. Acad. Sci. USA 86: <br />     9109&#226;&#8364;&#8220;9113.                                                                         36.   Tanaka, A., H. Hirano, M. Kikiuchi, S. Kitayama, H. Watanabe. 1996. <br /> 20. Kenney, T. J., C. P. Morgan, Jr. 1991. Genetic evidence interaction              Changes cellular proteins Deinococcus radiodurans following &#226;?&#165;-irradi- <br />     sigma A promoters Bacillus subtilis. J. Bacteriol. 173:3282&#226;&#8364;&#8220;              ation. Radiat. Environ. Biophys. 35:95&#226;&#8364;&#8220;99. <br />     3290.                                                                              37.   Vo&#194;&#168;lker, U., B. Maul, M. Hecker. 1999. Expression sigma B- <br /> 21.  <br /> </body></html>