<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Autophagic <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> Recycling Arabidopsis Directed <br /> by ATG8 ATG12 Conjugation Pathways1 <br /> Allison R. Thompson, Jed H. Doelling2, Anongpat Suttangkakul, Richard D. Vierstra* <br /> Department Genetics, University Wisconsin, Madison, Wisconsin 53706 <br />  <br />  <br /> Autophagy important mechanism nonselective intracellular breakdown cytosol organelles are <br /> encapsulated vesicles, engulfed digested lytic vacuoles/lysosomes. In yeast, encapsulation <br /> employs set autophagy (ATG) proteins direct conjugation ubiquitin-like protein tags, ATG8 ATG12, <span id='am-10' about='protege:TO' typeof='owl:Thing'>to</span> <br /> phosphatidylethanolamine ATG5 protein, respectively. Using Arabidopsis (Arabidopsis thaliana) atg7 mutant unable <br /> to ligate tag, previously showed ATG8/12 conjugation important <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span> nitrogen- <br /> limiting growth conditions. By reverse-genetic analyses single Arabidopsis gene encoding ATG5, the <br /> subpathway forms ATG12-ATG5 conjugate essential role plant <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> recycling. Similar plants <br /> missing ATG7, missing ATG5 display early senescence hypersensitive nitrogen carbon starvation, <br /> which accompanied rapid loss organellar cytoplasmic proteins. Multiple ATG8 isoforms detected <br /> immunologically seedling extracts. Their abundance substantially elevated atg5 atg7 mutants, caused in <br /> part increase abundance ATG8 mRNAs. Using green fluorescent protein-ATG8a fusion combination <br /> with concanamycin A, detected accumulation autophagic bodies inside vacuole. This accumulation was <br /> substantially enhanced starvation blocked atg7 background. The use fusion conjunction atg mutants <br /> now provides important marker track autophagic vesicles planta. <br />  <br />  <br />  <br />    Protein turnover plays numerous essential roles                    contents degraded wide range of <br /> plants, including precise removal short-lived                     vacuolar proteases, peptidases, lipases, other <br /> regulatory proteins, elimination abnormal pro-                    hydrolytic enzymes. <br /> teins, maintenance amino acid pools needed                       In yeasts, autophagic engulfment delivery the <br /> continual protein synthesis, recycling car-                   vacuole occurs morphologically distinct but <br /> bon (C) nitrogen (N) senescence apop-                     mechanistically overlapping routes, microautophagy <br /> tosis (Vierstra, 1996). Whereas regulatory                       macroautophagy (Klionsky, 2004). Microautoph- <br /> protein turnover accomplished highly selec-                    agy involves sequestration cytosol tubular <br /> tive ubiquitin (Ub)/26S proteasome pathway (Smalle                       invaginations tonoplast. These tubes pinch <br /> and Vierstra, 2004), plants, like eukaryotes,                 release intravacuolar vesicles called autophagic <br /> use less-selective high-throughput mechanisms de-                    bodies, cargo degraded vacuolar <br /> grading intracellular constituents bulk. One                   hydrolases following breakdown limiting mem- <br /> most conspicuous cytoplasm autophagy. Here,                    brane. Macroautophagy, contrast, involves en- <br /> individual proteins, protein complexes, entire or-                   trapment cytosol double membrane-bound <br /> ganelles engulfed membrane-bound vesi-                         vesicles called autophagosomes. These vesicles appear <br /> cles, delivered vacuole (yeast/plants)                  arise tubular network called preauto- <br /> or lysosome (animals; Ohsumi, 2001; Klionsky, 2004;                      phagosomal structure. It encircles portions the <br /> Thompson Vierstra, 2005). These vesicles                   cytoplasm cage-like structure, coa- <br />                                                                          lesces seals form autophagosome. The outer <br />                                                                          membrane autophagosome fuses the <br />   1 <br />       This work supported National Research Initiative     vacuole/lysosome release internal vesicle as <br /> the U.S. Department Agriculture Cooperative State Research, Edu-      autophagic body. In yeast, smaller version the <br /> cation Extension Service (grant no. 2002&#226;&#8364;&#8220;01452 R.D.V.),    autophagosome used deliver functional protein <br /> National Institutes Health (Postdoctoral Fellowship J.H.D.),    complexes vacuole pathway called cyto- <br /> a Thailand Predoctoral Fellowship (to A.S.), Wisconsin        plasm-to-vacuole targeting (Huang Klionsky, 2002). <br /> Alumni Research Foundation Louis Elsa Thomsen Wisconsin             While microautophagy macroautophagy have <br /> Distinguished Predoctoral Fellowships (to A.R.T.). <br />    2                                                                     extensively described cytological level, <br />       Present address: Department Genetics Developmental <br /> Biology, 1090 Agricultural Science Building, West Virginia Univer- <br />                                                                          recently underpinning mechanisms <br /> sity, Morgantown, WV 26506&#226;&#8364;&#8220;6108.                                         revealed. Especially informative ge- <br />    * Corresponding author; e-mail vierstra@wisc.edu; fax 608&#226;&#8364;&#8220;262&#226;&#8364;&#8220;        netic screens yeasts Saccharomyces cerevisiae and <br /> 2976.                                                                    Pichia pastoris identified diverse set autophagy <br />    Article, publication date, citation information   (ATG) proteins (also known APG, AUT, CVT, and <br /> www.plantphysiol.org/cgi/doi/10.1104/pp.105.060673.                      GSA; Klionsky et al., 2003). In particular, Ub-like <br /> Plant Physiology, August 2005, Vol. 138, pp. 2097&#226;&#8364;&#8220;2110, www.plantphysiol.org &#195;&#8220; 2005 American Society Plant Biologists       2097 <br />  Thompson et al. <br />  <br />  <br /> conjugation systems emerged essential                Here, report continued description of <br /> autophagosome formation macroautophagy                 autophagy plants analysis the <br /> and invagination tonoplast micro-               ATG8/12 conjugation system. <span id='am-5' about='protege:FROM' typeof='owl:Thing'>From</span> reverse-genetic <br /> autophagy (Ohsumi, 2001; Klionsky, 2004). These con-          analyses single gene encodes ATG5, we <br /> jugation pathways employ Ub-like peptide tags:            disruption ATG12 conjugation sub- <br /> ATG8 ATG12. Through ATP-dependent conjuga-                pathway just debilitating inability con- <br /> tion cascades analogous ubiquitination, tags         jugate ATG8 ATG12. Furthermore, found <br /> become activated common E1, ATG7,             ATG12-ATG5 conjugation important not <br /> are transferred E2s, ATG3 ATG10,         survival N-limiting conditions also <br /> specific ATG8 ATG12, respectively. Finally,           C starvation. Using green fluorescent protein <br /> ATG8 ATG12 covalently attached re-           (GFP)-ATG8a fusion combination concanamy- <br /> spective targets, lipid phosphatidylethanolamine          cin A, detected accumulation intravacuolar <br /> (PE) ATG5 protein. Linkage amide            vesicles represent autophagic bodies await- <br /> bond C-terminal Gly carboxyl group             ing catabolism. Both expanding collection atg <br /> the tags free amino group PE               mutants GFP-ATG8a reporter provide new tools <br /> e-amino group specific Lys ATG5. Production        follow previously enigmatic recycling process. <br /> the ATG8-PE ATG12-ATG5 conjugates appears to <br /> help promote formation closure autophagic <br /> vesicles subsequent delivery vacuole <br /> (Ohsumi, 2001; Klionsky, 2004). The ATG8-PE conju-            RESULTS <br /> gate bound autophagic membrane                 Expression Patterns ATG Genes <br /> presumably lipid moiety. The ATG12-ATG5 <br /> conjugate assembles ATG16 form proposed                In previous studies, discovered a <br /> hetero-octameric structure loosely associated        collection Arabidopsis genes related yeast <br /> vesicle. The dynamics conjugates vivo sug-        ATG group, including small families encode <br /> gest formation ATG12-ATG5 adduct pre-             components ATG8/12 conjugation pathway <br /> cedes formation ATG8-PE adduct enhances            (Doelling et al., 2002; Hanaoka et al., 2002). From an <br /> this lipidation reaction crosstalk        analysis Expressed Sequence Tag (EST; http:// <br /> conjugation reactions (Suzuki et al., 2001).                  www.arabidopsis.org) Massively Parallel Signa- <br />    Following discovery yeast ATG system,           ture Sequences (http://mpss.udel.edu/at) databases <br /> sequence relatives detected eukary-             DNA microarray results Ma et al. (2005), it <br /> otes, suggesting autophagic process               appears known Arabidopsis loci required for <br /> strongly conserved (Mizushima et al., 2001; Thumm             ATG8/12 conjugation expressed (Fig. 1; data not <br /> and Kadowaki, 2001; Tanida et al., 2002; Marino et al.,       shown). In fact, ATG3, 4, 5, 7, 8, 12 <br /> 2003; Melendez et al., 2003; Otto et al., 2003). For plants   transcripts detected tissues examined, <br /> in particular, (Doelling et al., 2002; Thompson        indicating pathway restricted spe- <br /> Vierstra, 2005) Ohsumi colleagues (Hanaoka            cific tissue type developmental window (Fig. 1, A <br /> et al., 2002) retrieved collection ATG orthologs         B). High-level expression observed floral <br /> from Arabidopsis (Arabidopsis thaliana) genomic           organs, including sepals, petals, stamens (Fig. 1), <br /> database. In contrast S. cerevisiae, single ATG8     implying ephemeral tissues use autophagy <br /> and ATG12 genes present, Arabidopsis encodes              rapid senescence following fertilization. <br /> nine ATG8 (AtATG8a&#226;&#8364;&#8220;i) ATG12 (AtATG12a                 Comparably low-level expression evident ger- <br /> and AtATG12b) isoforms, suggesting auto-             minating seeds, suggesting autophagic events <br /> phagic process plants complex. Arabidopsis         involving ATG proteins play major role in <br /> mutants disrupted single loci encoding ATG7            seed metabolism. <br /> and ATG9 (a transmembrane protein needed                We observed significant differences expres- <br /> autophagosome assembly) loci encoding             sion members ATG4 ATG8 gene <br /> the ATG4 protease (needed maturation ATG8)             families, implying tissue/cell-specific roles among <br /> grow normally display accelerated senescence          isoforms exist. Within nine-member ATG8 <br /> are hypersensitive N-deficient conditions (Doelling        family, ATG8a displayed highest expression level <br /> et al., 2002; Hanaoka et al., 2002; Yoshimoto et al.,         various flower parts, ATG8e highest in <br /> 2004). A similar <span id='am-167' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> apparent Arabi-          roots (Fig. 1A). In cases, microarray data for <br /> dopsis mutant missing vesicular soluble n-ethyl-          family members supported analysis the <br /> maleimide-sensitive factor attachment protein receptor        EST database. For example, higher expression of <br /> (v-SNARE) VTI12 (Vps interacting),           ATG4b relative ATG4a paralleled 3-fold <br /> required docking fusion autophagosomes          higher number cDNAs EST collection. Within <br /> the tonoplast vacuole (Surpin et al., 2003).           ATG8 family, highly expressed isoforms <br /> Therefore, autophagy essential            ATG8a, e, f 21, 25, 52 ESTs, respectively, <br /> plants N-rich growth conditions,             expressed isoforms, ATG8c g, <br /> important survival N limiting.                    represented ESTs, respectively. <br /> 2098                                                                                        Plant Physiol. Vol. 138, 2005 <br />                                                                                                    Autophagic Recycling Arabidopsis <br />  <br />  <br />                                                                       stream exon bearing codon Lys-128, <br />                                                                       altering surrounding genomic sequence. <br />                                                                       For atg5-2, T-DNA sequence interrupted 5# <br />                                                                       untranslated region (UTR), 40 nucleotides upstream of <br />                                                                       translation start codon. Basta herbicide resistance <br />                                                                       conferred BAR gene T-DNA segre- <br />                                                                       gated 3:1 <span id='am-324' about='xsp:pattern' typeof='owl:Thing'>pattern</span> mutant alleles, suggest- <br />                                                                       ing single insertion site. In populations <br />                                                                       homozygous atg5-1 atg5-2 plants, genomic PCR <br />                                                                       5# 3# gene-specific primers (primers 1 1 2) <br />                                                                       confirmed absence uninterrupted wild-type <br />                                                                       ATG5 gene, PCR 5# gene-specific <br />                                                                       primer T-DNA left-border (LB) primer (primers <br />                                                                       LB 1 2) showed presence introduced T-DNA <br />  <br />  <br />  <br />  <br /> Figure 1. Expression levels Arabidopsis ATG genes. Microarray <br /> expression levels nine-member ATG8 gene family (A) other <br /> ATG genes (B) various Arabidopsis tissues. Expression levels are <br /> graphed normalized expression units 3 102. <br />  <br />  <br /> Isolation Mutants Affecting ATG5 <br />    Previous studies Arabidopsis T-DNA dis- <br /> ruption mutant affecting ATG7 showed conjuga- <br /> tion ATG8 ATG12, essential, is <br /> important survival N-deficient growth con- <br /> ditions (Doelling et al., 2002). To help define func- <br /> tions ATG12 conjugation pathway individually, <br /> we searched mutants affecting target protein <br /> ATG5. BLASTP searches near-complete genomes <br /> of Arabidopsis rice (Oryza sativa) identified single <br /> loci encoding ATG5 species (At5g17290 and <br /> AP004084, respectively). By analysis genomic and <br /> full-<span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> cDNA sequences, Arabidopsis ATG5 <br /> gene predicted encode 337-amino acid protein <br /> with 66%, 31%, 42%, 44% similarity rice, S. <br /> cerevisiae, Drosophila melanogaster, human counter- <br /> parts, respectively (Fig. 2). Among numerous con- <br /> served residues present ATG5 collection                 Figure 2. Sequence comparison ATG5 protein eukar- <br /> a Lys (position 128 AtATG5) identified yeast                yotes. Amino acid sequence comparison Arabidopsis (At) ATG5 <br /> version attachment site ATG12.                             (At5g17290) rice (Os; AP004084), yeast (S. cerevisiae, Sc; <br />                                                                       Kametaka et al., 1996), Drosophila (Dm; AAL39741), human (Hs) <br />    In screen Syngenta Arabidopsis Insertion <br />                                                                       orthologs (Hammond et al., 1998). Identical similar amino acids <br /> Library (SAIL) T-DNA insertion population                      shown reverse type gray boxes, respectively. Dots denote <br /> wild-type Columbia-0 (Col-0) background (Sessions                     gaps. Numbers end indicate amino acid length each <br /> et al., 2002), identified mutant alleles ATG5,              protein. The asterisk marks Lys (K128 At) provides the <br /> designated atg5-1 atg5-2 (Fig. 3A). The T-DNA                  attachment site ATG12. The arrowhead locates T-DNA insertion <br /> atg5-1 inserted fourth intron, just down-                  site atg5-1 mutant. <br />  <br /> Plant Physiol. Vol. 138, 2005                                                                                                        2099 <br />  Thompson et al. <br />  <br />  <br />  <br />  <br />         Figure 3. Description Arabidopsis atg5-1 atg5-2 mutants. A, Diagram Arabidopsis ATG5 gene. Lines indicate <br />         introns. Boxes indicate exons, coding regions black 5# 3# UTRs gray. The arrowhead locates Lys (K-128) <br />         provides attachment site ATG12. The locations T-DNA atg5-1 atg5-2 mutants shown. Arrows <br />         locate priming sites used B C. B, Genomic PCR analysis atg5-1 atg5-2 mutants. Genomic DNA isolated from <br />         wild-type Col-0, atg5-1, atg5-2, atg5-1 complemented 35S:ATG5 subjected PCR using number 1 and <br />         number 2 primer pairs (1 1 2), number 2 primer LB T-DNA (LB 1 2), primers specific 35S:ATG5 <br />         transgene (Trans). C, RT-PCR analysis atg5-1 atg5-2 mutants. Total RNA isolated wild-type Col-0, atg7-1, atg5-1, <br />         atg5-2, atg5-1 complemented 35S:ATG5 subjected RT-PCR using number 3 number 4 primer pair <br />         (3 1 4) number 3 number 5 primer pair (3 1 5). A primer pair specific PAE2 used internal control. D, <br />         Immunoblot analysis atg5-1 atg5-2 mutants. Crude protein extracts wild-type Col-0, atg7-1, atg5-1, atg5-2, and <br />         atg5-1 complemented 35S:ATG5 prepared 10-d-old seedlings subjected SDS-PAGE immunoblot analysis <br />         anti-ATG7, anti-ATG5, anti-ATG8 antibodies. Equal protein loads confirmed immunoblot analysis anti- <br />         PBA1 antibodies. Black white arrowheads identify presumed ATG12-ATG5 conjugate (50 kD) free ATG5 (40 kD), <br />         respectively. <br />  <br />  <br />  <br /> sequences (Fig. 3B). Reverse transcription (RT)-PCR                         Wassilewskija (WS) background fail accumulate <br /> analysis demonstrated T-DNA prevented ac-                          E1 needed conjugate ATG12 ATG5 (Doelling <br /> cumulation normal ATG5 mRNA atg5-1 seed-                          et al., 2002). Here, 50-kD species completely <br /> lings. Although PCR product containing region                         absent 40-kD species substantially <br /> upstream T-DNA generated                                abundant (Fig. 3D), strongly supporting the <br /> mRNA (primers 3 1 5), product encompassing                            proposal 50-kD protein ATG12-ATG5 <br /> full-length transcript (primers 3 1 4) apparent                     conjugate 40-kD form free ATG5. <br /> (Fig. 3C). For atg5-2, however, RT-PCR showed                             Consistent RT-PCR analyses showing a <br /> expression gene appreciably attenuated                       full-length ATG5 transcript generated from <br /> by T-DNA. Both sets primers able                             atg5-2 allele, detected corresponding ATG5 <br /> amplify ATG5 mRNA, indicating expression                           protein homozygous atg5-2 seedlings. In contrast to <br /> of full-length protein possible (Fig. 3C).                          wild-type seedlings, free ATG5 ATG12- <br />    To test atg5-1 atg5-2 mutants affected ac-                    ATG5 conjugate present near equal amounts, <br /> cumulation ATG5 protein, performed immu-                          indicating T-DNA misregulate ATG5 <br /> noblot analysis crude extracts homozygous                           expression (Fig. 3D). Since atg5-2 <span id='am-245' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-246' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span> did <br /> seedling using antibodies generated recombi-                        affect level ATG12-ATG5 conjugate and <br /> nant ATG5. As shown Figure 3D, abundant                               phenotypically silent, characterized <br /> 50-kD protein minor 40-kD protein detected                       further. <br /> in wild-type extracts. Both species (and smaller forms)                        Given possibility loss ATG5 affects ATG8 <br /> were absent extracts atg5-1 seedlings,                          conjugation Arabidopsis does yeast (Suzuki <br /> demonstrating atg5-1 mutation likely                      et al., 2001), prepared antibodies recombi- <br /> represents <span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null allele</span> species                    nant ATG8a assay protein family. This antibody <br /> derived ATG5 locus. The 40-kD species                          readily identified recombinant ATG8i, most <br /> close apparent mass predicted ATG5                           dissimilar isoform highly conserved family <br /> (38,479 D), 50-kD species close                          (69%&#226;&#8364;&#8220;96% amino acid sequence identity the <br /> the prediction ATG12-ATG5 conjugates (49,033                        family; Doelling et al., 2002), demonstrating it <br /> and 48,819 D ATG12a b isoforms, respec-                         recognize members ATG8 family <br /> tively). To help confirm assignments, ana-                         (data shown). In wild-type Col-0 WS <br /> lyzed extracts atg7-1 seedlings wild-type                       seedling extracts, detected ATG8 proteins <br /> 2100                                                                                                              Plant Physiol. Vol. 138, 2005 <br />                                                                                                   Autophagic Recycling Arabidopsis <br />  <br />  <br /> of 11 15 kD, indicating multiple ATG isoforms                     ATG12, hypersensitive N-deficient growth con- <br /> accumulate (Fig. 3D). Their steady-state levels                      ditions (Doelling et al., 2002). A similar response was <br /> substantially higher homozygous atg5-1                         observed homozygous atg5-1 seedlings grown <br /> atg7-1 backgrounds atg5-2 background,                      long-day photoperiod Suc-containing <br /> suggesting defects entire ATG8/12 conju-                      medium N. Following 17 d N-deficient <br /> gation ATG12 specifically                      medium, atg5-1 seedlings, like atg7-1 seedlings, <br /> enhance synthesis ATG8 proteins interfere                       displayed enhanced chlorosis cotyledons and <br /> with breakdown (see below).                                         reduced true leaf formation compared wild-type <br />                                                                           seedlings (Fig. 4B). Enhanced chlorosis evi- <br /> Molecular Characterization N- C-Limiting                       dent detached leaves intact plants grown in <br /> Phenotypes atg5-1                                                      N-deficient liquid medium (data shown). When <br />                                                                           plants exposed various lengths N starva- <br />    Similar Arabidopsis T-DNA mutants disrupting                        tion returned N-rich medium, the <br /> ATG7 (Doelling et al., 2002) ATG9 (Hanaoka et al.,                    atg5-1 atg7-1 mutants experienced dramatic drop <br /> 2002), homozygous atg5-1 seedlings germinated                         percentage seedlings recovered. While <br /> developed normally, flowered set seed                      nearly 100% wild type resumed growth 28 d on <br /> same rate wild-type plants long-day pho-                       N-deficient medium, approximately 20% the <br /> toperiod (16 h light/8 h dark; Fig. 4B; data shown).                  atg5-1 atg7-1 plants recovered (Fig. 4C). <br /> However, short-day photoperiod (8 h light/                           To test atg5-1 atg7-1 plants dis- <br /> 16 h dark), atg5-1 plants, like atg7-1 plants (Doelling               played slower growth accelerated senescence un- <br /> et al., 2002), grew slower resulting smaller plants,                   der C-limiting conditions, developed assay to <br /> flowered later wild type reduced fecundity,                     test effects C starvation specifically. Seedlings <br /> and showed enhanced senescence rosette leaves                          grown 3 weeks N/C-rich agar me- <br /> (data shown; Fig. 9A).                                                dium transferred soil 3 weeks short <br />    Prior growth studies showed plants missing                        days maintain adequate supply photosynthate <br /> ATG7, unable conjugate ATG8                          (Brouquisse et al., 1998; Matt et al., 1998). The seed- <br />                                                                           lings placed darkness extended pe- <br />                                                                           riods 1-week recovery short-day <br />                                                                           photoperiod. Previous studies showed pro- <br />                                                                           longed darkness facile way activate vacuole- <br />                                                                           specific proteolysis corn, possibly autophagic <br />                                                                           mechanism (Brouquisse et al., 1998), accentuate <br />                                                                           autophagic defects Arabidopsis (Hanaoka et al., <br />                                                                           2002). As seen Figure 5, wild-type <br />                                                                           WS Col-0 parents tolerate dark in- <br />                                                                           cubation 8 d resume growth the <br />                                                                           light. At 10 d, Col-0 plants the <br />                                                                           WS plants died, defining upper limit for <br />                                                                           tolerance extended darkness. When atg5-1 and <br />                                                                           atg7-1 seedlings tested similarly, striking hy- <br />                                                                           persensitivity darkness apparent. Both mutants <br />                                                                           barely recovered 4 d dark, none <br />                                                                           survived 6 d (Fig. 5, C D). <br />                                                                              To examine effects C starvation induced by <br />                                                                           darkness molecular level, collected wild-type <br />                                                                           atg5-1 plants immediately dark treatment <br />                                                                           examined protein mRNA levels. Plants <br />                                                                           left progressively longer times dark became <br />                                                                           severely chlorotic flaccid, condition was <br />                                                                           pronounced atg5-1 mutant (Fig. 5B). Con- <br />                                                                           sistent problems, atg5-1 mutant lost <br /> Figure 4. Enhanced sensitivity atg5-1 plants N-deficient condi-     protein faster respective Col-0 wild type dur- <br /> tions. Plant lines include atg7-1 atg5-1 respective wild-   ing dark treatment, seen silver staining <br /> types WS Col-0. A, Diagram treatment. One-week-old seedlings       total protein immunoblot analyses for <br /> were sown N-rich agar medium transferred N-deficient agar <br /> medium various lengths time transfer N-rich <br />                                                                           specific polypeptides (Fig. 6, A B). Spectrophoto- <br /> medium, long-day photoperiod. B, Representative seedlings <br />                                                                           metric analysis showed chlorophyll also <br /> after 0 d 17 d N-deficient medium. C, The percentage plants     degraded accelerated rate mutants. Despite <br /> that survived growth various days N-deficient medium            faster rate, complement proteins degraded in <br /> determined resumption growth transferred N-rich             atg5-1 background (as detected silver staining) <br /> medium. Each point represents average 10 seedlings.                indistinguishable wild type, little <br /> Plant Physiol. Vol. 138, 2005                                                                                                  2101 <br />  Thompson et al. <br />  <br />  <br />                                                                            evidence specific proteins selectively re- <br />                                                                            tained lost mutant (Fig. 6A). This comparable <br />                                                                            breakdown supported immunoblot analyses <br />                                                                            battery antibodies cytosolic and <br />                                                                            organellar proteins. As seen Figure 6B, Rubisco <br />                                                                            magnesium-protoporphyrin chelatase, are <br />                                                                            markers chloroplasts, mitochondrial marker (the <br />                                                                            ATPase F1 subunit), cytosolic/nuclear marker <br />                                                                            (the 26S proteasome b1 subunit PBA1), disap- <br />                                                                            peared similarly dark treatment, with <br />                                                                            loss accelerated atg5-1 seedlings. <br />                                                                               With respect ATG8/12 conjugation pathways, <br />                                                                            levels ATG7 decreased faster <br />                                                                            atg5-1 mutant background. While free 40-kD <br />                                                                            form ATG5 barely detectable wild-type <br />                                                                            plants dark treatment, species became <br />                                                                            abundant following prolonged incubation the <br />                                                                            dark (Fig. 6B). Not surprisingly, increase free <br />                                                                            ATG5 Col-0 plants coincided drop in <br />                                                                            ATG7, necessary conjugation. Unlike <br />                                                                            ATG5 ATG7 seedling proteins, the <br />                                                                            levels ATG8 protein family wild-type seed- <br />                                                                            lings did drop dark-induced C starvation <br />                                                                            increased slightly despite substantial loss of <br />                                                                            polypeptides (Fig. 6B). Similar the <br />                                                                            observations Figure 3D, levels ATG8 <br />                                                                            proteins substantially higher homozygous <br />                                                                            atg5-1 background, new species becoming <br />                                                                            predominant (Fig. 6B). Most species peaked in <br />                                                                            abundance 2 6 d darkness declined <br />                                                                            thereafter. After extended dark treatments, two <br />                                                                            lower molecular mass species approximately 8 kD <br />                                                                            appeared (Fig. 6B). Although remains possible that <br />                                                                            species ATG8-PE conjugates, appear- <br />                                                                            ance time ATG7 nearly undetectable <br />                                                                            argued otherwise. <br />                                                                               To confirm response plants ex- <br />                                                                            tended darkness senescence-related process, we <br />                                                                            examined abundance transcripts (Fig. <br />                                                                            6C), including SEN1, gene specifically <br />                                                                            expressed senescence (Weaver et al., 1998). <br />                                                                            Similar previous studies detached leaves <br />                                                                            (Doelling et al., 2002), CAB mRNA levels dropped <br />                                                                            rapidly following dark incubation short 2 d, <br />                                                                            consistent instability CAB mRNA and <br />                                                                            light requirement continued CAB transcription <br />                                                                            (Giuliano et al., 1988). Conversely, transcripts the <br />                                                                            cell-division regulator CDC2a senescence- <br /> Figure 5. Enhanced sensitivity atg7-1 atg5-1 plants C-limiting   associated SEN1 genes increased C-limiting <br /> conditions induced extended darkness. Plant lines include atg7-1        treatment. Whereas CDC2a mRNA levels fell after <br /> and atg5-1 respective wild types WS Col-0. A, Diagram     prolonged dark exposure, levels SEN1 transcript <br /> treatment. Six-week-old plants grown short-day photo-         remain high especially atg5-1 background, <br /> period, transferred darkness various lengths time,      supporting view dark-induced chlorosis <br /> transferred short-day photoperiod. B, Plants immediately       represents form senescence. <br /> before 6-d dark treatment. C, Plants 1-week recovery         Previous studies Arabidopsis ATG8 <br /> from 2-, 4-, 6-d dark treatments. D, Percentage plants that <br />                                                                            mRNAs mildly increased Suc starvation <br /> survived regime determined resumption growth plotted <br /> versus days dark. Each point represents average 10 <br />                                                                            culture cells (Contento et al., 2004) the <br /> seedlings.                                                                 senescence detached leaves (Doelling et al., 2002). <br />                                                                            To test similar increase explain increase <br />                                                                            ATG8 protein dark-induced C starvation in <br /> 2102                                                                                                     Plant Physiol. Vol. 138, 2005 <br />                                                                                                     Autophagic Recycling Arabidopsis <br />  <br />  <br />                                                                             wild-type plants large increase ATG8 pro- <br />                                                                             tein atg5-1 background, compared abun- <br />                                                                             dance ATG8 transcripts proteins. Similar to <br />                                                                             detached leaves (Doelling et al., 2002), abundance <br />                                                                             ATG8 mRNA increased dark-induced C <br />                                                                             starvation intact wild-type plants (Fig. 6C). More <br />                                                                             ATG8 mRNA evident atg5-1 background, <br />                                                                             suggesting expression genes up- <br />                                                                             regulated autophagic defect. <br />                                                                                To help identify ATG8 transcripts <br />                                                                             responsible increase C starvation <br />                                                                             increase atg5-1 mutant, an- <br />                                                                             alyzed abundance dark treat- <br />                                                                             ment real-time RT-PCR. We quantified ATG8a to <br />                                                                             ATG8i mRNA levels using comparative threshold <br />                                                                             cycle (CT) method (Giulietti et al., 2001). Values were <br />                                                                             normalized internal control mRNA (histone <br />                                                                             H2A) expressed relative dark <br />                                                                             treatment. As seen Figure 7A, relative <br />                                                                             abundance most, all, ATG8 mRNAs was <br />                                                                             slightly dramatically increased 6-d C starva- <br />                                                                             tion. Several increased atg5-1 back- <br />                                                                             ground, especially ATG8e ATG8h (Fig. 7B). A time <br />                                                                             course induction showed ATG8e and <br />                                                                             ATG8h transcripts rose dramatically relative abun- <br />                                                                             dance beginning day 6 increased day <br />                                                                             8 despite fact tissue noticeably se- <br />                                                                             nescent reduced total protein (Figs. 5B 6, A <br />                                                                             B). Collectively, data ATG8 mRNA <br />                                                                             levels up-regulated C starvation several <br />                                                                             responsive mutation ATG5. These changes <br />                                                                             turn responsible increase ATG8 <br />                                                                             proteins atg5-1 seedlings C starvation <br />                                                                             (Fig. 6B). <br />  <br />                                                                             Disruptions ATG5 ATG7 Affect Profile <br />                                                                             ATG8 Proteins <br />  <br />                                                                                Recent studies Yoshimoto et al. (2004) detected <br />                                                                             new ATG8 species N starvation could <br />                                                                             represent ATG8-PE adducts based on <br />                                                                             faster migration rate. To examine pos- <br />                                                                             sibility, compared profiles ATG8 proteins <br />                                                                             nonstarved C-starved wild-type, atg5-1, and <br />                                                                             atg7-1 seedlings, using SDS-PAGE without <br /> Figure 6. Protein RNA analyses atg5-1 plants exposed              6 M urea. The inclusion urea based pre- <br /> C-limiting conditions induced extended darkness. Tissue col-         vious studies yeast ATG8 showing the <br /> lected wild-type Col-0 atg5-1 seedlings just removing        PE-conjugate migrates faster free ATG8 under <br /> the plants indicated lengths extended darkness (see Fig. 5B).   electrophoretic condition (Kirisako et al., 1999). <br /> A B, SDS-PAGE analysis total protein equal         Unfortunately, number ATG8 species were <br /> tissue fresh weight. A, Gels silver stained total protein. B, Gels   observed, matched expectations the <br /> subjected immunoblot analysis antibodies large <br />                                                                             ATG8-PE adducts. As shown Figure 8, several <br /> subunit Rubisco, magnesium-protoporphyrin chelatase (Chel), F1 <br /> subunit mitochondrial ATPase (ATPase), b-subunit A 26S pro- <br />                                                                             ATG8 isoforms detected wild-type Col-0 <br /> teasome (PBA1), ATG7, ATG5, ATG8. Black white arrowheads            WS seedlings grown light, their <br /> identify presumed ATG12-ATG5 conjugate (50 kD) free ATG5            abundance complexity enhanced the <br /> (40 kD), respectively. C, RNA gel-blot analysis equal amounts total   atg5-1 atg7-1 mutations. In wild-type plants, no <br /> RNA using probes CAB, SEN1, CDC2a, ATG8a. Equal loading          new species (especially faster migrating forms) be- <br /> total RNA confirmed staining rRNA methylene blue            came evident (with urea) following pro- <br /> (data shown).                                                           longed dark treatment 8 d sufficient impair <br />                                                                             survival. A faster migrating form detected in <br /> Plant Physiol. Vol. 138, 2005                                                                                                    2103 <br />  Thompson et al. <br />  <br />  <br />                                                                          complement mutation ectopic expression the <br />                                                                          wild-type ATG5 cDNA. atg5-1 plants transformed <br />                                                                          ATG5 transgene control cau- <br />                                                                          liflower mosaic virus (CaMV) 35S promoter were <br />                                                                          selected hygromycin resistance conferred the <br />                                                                          T-DNA. T1 plants self-fertilized, progenies <br />                                                                          homozygous atg5-1 allele 35S:ATG5 trans- <br />                                                                          gene identified PCR hygromycin resis- <br />                                                                          tance, respectively (Fig. 3B). Both ATG5 transcript <br />                                                                          ATG5 proteins detected complemen- <br />                                                                          tation line, ratio 40- 50-kD proteins <br />                                                                          similar wild-type seedlings (Fig. 3, C <br />                                                                          D). Phenotypically, plants harboring 35S:ATG5 <br />                                                                          transgene behaved like wild-type Col-0. Under the <br />                                                                          short-day photoperiod, senesced similar the <br />                                                                          wild type later atg5-1 mutant <br />                                                                          parent (Fig. 9A). The homozygous atg5-1/35S:ATG5 <br />                                                                          seedlings like wild type sensitive <br />                                                                          C-limiting conditions induced darkness than <br />                                                                          atg5-1 seedlings (Fig. 9, B C). We observed new <br />                                                                          phenotypes complementation lines, suggesting <br />                                                                          possible misexpression ATG5 caused use <br />                                                                          CaMV 35S promoter does interfere with <br />                                                                          autophagy particular overall growth general. <br />  <br /> Figure 7. Relative abundance individual ATG8 mRNAs Col-0       Visualization ATG8 Labeled Vesicles Is Dependent <br /> atg5-1 plants exposed C-limiting conditions induced extended <br />                                                                          ATG7 <br /> darkness. Relative abundance obtained real-time RT-PCR using <br /> the CT method expressed average triplicate technical         To help detect various structures associated with <br /> repeats SE. All values graphed fold increase mRNA       autophagy, examined possibility using GFP <br /> abundance relative day 0 normalized the <br />                                                                          fusions ATG8 proteins cytological markers. Pre- <br /> abundance histone H2A mRNA, used internal control. A, <br /> Relative increases mRNA abundance ATG8 transcript after <br />                                                                          vious studies yeast mammalian cells <br /> 6 d dark treatment compared abundance treatment. Inset   demonstrated fusions good markers <br /> shows expanded scale ATG8a ATG8g ATG8i                 visualizing autophagic bodies vivo (Suzuki et al., <br /> transcripts. B, mRNA abundance ATG8e ATG8h              2001; Mizushima et al., 2004). More recently, similar <br /> course 8-d dark treatment relative treatment.       GFP-ATG8 fusion <span id='am-6' about='rdfs:label' typeof='owl:Thing'>label</span> autophagic <br />                                                                          body-like vesicles Arabidopsis vacuoles (Yoshimoto <br /> C-starved atg5-1 plants absence presence <br /> of urea (Fig. 8). However, similarly sized species were <br /> also observed (albeit weaker) atg7-1 plants, which <br /> should lack capacity form lipidated adducts. <br />    In attempt enrich possible ATG8-PE ad- <br /> ducts, separated total cell lysates C-starved <br /> and nonstarved seedlings ultracentrifugation and <br /> assayed immunoblotting membrane-associated <br /> fraction species. Although the <br /> presumed ATG12-ATG5 conjugate 50 kD the <br /> supernatant (soluble) membrane fraction (pellet) <br /> consistent peripheral membrane association, <br /> none ATG8 species pellet (data not <br /> shown). Collectively, results suggest the <br />                                                                          Figure 8. Differential accumulation ATG8 proteins C-limiting <br /> ATG8-PE conjugates, produced, become <br />                                                                          growth conditions induced extended darkness. Tissue collected <br /> a significant portion ATG8 pool con-                    wild types (Col-0 WS), atg5-1, atg7-1 immediately before <br /> ditions studied here.                                                    8 d extended dark. ATG8 proteins detected by <br />                                                                          immunoblot analysis anti-ATG8 antibodies following SDS-PAGE <br /> Complementation atg5-1                                                (bottom) (top) 6 M urea separating gel. Total <br />                                                                          protein equivalent amounts tissue fresh weight used each <br />   To confirm atg5-1 phenotypes directly                    case. Intensities PBA1 detected anti-PBA1 antibodies reflect <br /> caused loss ATG5 protein, attempted                        levels total protein. <br />  <br /> 2104                                                                                                           Plant Physiol. Vol. 138, 2005 <br />                                                                                                          Autophagic Recycling Arabidopsis <br />  <br />  <br />                                                                            expected size chimeric protein (data shown). <br />                                                                            Using fluorescence confocal microscopy, moni- <br />                                                                            tored hypocotyl cells young seedlings 2 d in <br />                                                                            N-rich N-deficient media. As shown top <br />                                                                            sections Figure 10, GFP-ATG8a predominately <br />                                                                            cytoplasmic wild-type cells N <br />                                                                            starvation. Upon concanamycin A treatment, punctate <br />                                                                            structures accumulated central vacuole, with <br />                                                                            substantially present N starvation (Fig. 10, <br />                                                                            middle sections). Upon closer examination, re- <br />                                                                            sembled autophagic bodies, appearing hollow <br />                                                                            spherical vesicles diameter 0.5 1.5 mm. <br />                                                                            Unlike transvacuolar strands/tubes highly <br />                                                                            dynamic appear vesicles cross section <br />                                                                            (Hoffmann Nebenfuhr, 2004; Takatsuka et al., <br />                                                                            2004), vesicles labeled GFP-ATG8a displayed <br />                                                                            slow random motion, indicating free- <br />                                                                            floating vacuolar lumen. The accumulation <br />                                                                            vesicles required ATG8/12 conjugation sys- <br />                                                                            tem, supporting possibility rep- <br />                                                                            resents autophagic bodies. Even N starvation and <br />                                                                            treatment concanamycin A, failed detect these <br />                                                                            vesicles atg7-1 seedlings (Fig. 10, sections). <br />                                                                            This pattern distinct observed with <br />                                                                            wild-type plants expressing GFP CaMV <br />                                                                            35S promoter. Here, observed diffuse staining of <br />  <br />  <br />  <br />  <br /> Figure 9. Complementation atg5-1 mutation ATG5. Plant <br /> lines include wild type (Col-0), atg5-1, atg5-1 complemented with <br /> a 35S:ATG5 transgene. A, Plants grown soil 3 months under <br /> a short-day photoperiod. B, Six-week-old plants grown short- <br /> day photoperiod, transferred darkness 2 6 d, then <br /> transferred short-day photoperiod. Images taken after <br /> a 1-week recovery dark treatment. C, Percentage plants that <br /> survived extended darkness determined resumption growth <br /> is plotted versus days dark. Each point represents average of <br /> 10 seedlings. <br />  <br />  <br /> et al., 2004). Their visualization Arabidopsis re- <br /> quired pretreatment vacuolar H1-ATPase <br /> inhibitor concanamycin A, appears block <br /> vacuolar hydrolysis raising internal pH above <br /> the optimum vacuolar enzymes (Tamura                              Figure 10. Use GFP-ATG8 fusion detect possible autophagic <br /> et al., 2003).                                                             vesicles inside vacuole. Eight-day-old wild-type Col-0 atg7-1 <br />    Here, introduced transgene expressing GFP                        seedlings expressing GFP-ATG8a fusion grown N-rich liquid <br />                                                                            medium 6 d, transferred N-rich (1N) N-deficient (2N) liquid <br /> fusion N terminus ATG8a wild-type <br />                                                                            media approximately 1.5 d, incubated additional 12 <br /> Col-0 atg7-1 plants. (The atg5-1 mutant                        16 h 0.5 mM concanamycin A (1CA) equal volume of <br /> used does priori affect conjugation                     dimethyl sulfoxide (2CA). Hypocotyls visualized fluorescence <br /> of ATG8.) Immunoblot analysis anti-ATG8a                          confocal microscopy GFP. Size bars images equal ap- <br /> anti-GFP antibodies detected 40-kD protein                       proximately 50 mm. Five-times magnifications possible autophagic <br /> plants expressing GFP-ATG8a, consistent                           bodies vacuole wild-type plants included. <br />  <br /> Plant Physiol. Vol. 138, 2005                                                                                                               2105 <br />  Thompson et al. <br />  <br />  <br /> just cytoplasm, did change N            induce developmental growth de- <br /> starvation concanamycin A (data        fects. Given central importance encoded <br /> shown).                                                    proteins, corresponding mutant phenotypes may <br />                                                            represent severe abnormalities caused by <br />                                                            inactivation ATG system. <br /> DISCUSSION                                                    Contrary expectations, disruption ATG <br />                                                            autophagy led acceleration of <br />    Although autophagy proposed play            senescence faster breakdown cytoplasmic <br /> important roles <span id='am-8' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> recycling plants,       organellar proteins starvation. As judged by <br /> underpinning mechanism(s) remained elusive         loss Rubisco magnesium-protoporphyrin <br /> the absence molecular, cytological, genetic mark-   chelatase, chloroplasts lost faster atg5-1 <br /> ers analyze process (Moriyasu Klionsky,         atg7-1 mutants (Doelling et al., 2002; article), <br /> 2004; Thompson Vierstra, 2005). Clearly, signif-       despite proposal autophagic events are <br /> icant advances possible discovery         responsible removal (e.g. Moriyasu and <br /> of orthologs components yeast ATG           Klionsky, 2004; Niwa et al., 2004). Clearly, faster <br /> system Arabidopsis, including Ub-like conjuga-      breakdown implies alternative route(s) became <br /> tion attaches peptide tags ATG8        prominent loss ATG to <br /> ATG12 PE ATG5 protein, respectively             maintain supply C N needed survival. <br /> (Doelling et al., 2002; Hanaoka et al., 2002; Menand       Remarkably, alternatives degrade cytosolic, chlo- <br /> et al., 2002; Surpin et al., 2003). Through reverse-       roplastic, mitochondrial proteins faster nor- <br /> genetic analyses ATG7 (the E1 required ATG8         mal appear compromise viability mutant <br /> and ATG12 attachment) ATG5 (the protein target         plants. The nature alternative route(s) un- <br /> of ATG12), ATG particular       known. The Ub/26S proteasome does appear to <br /> and possibly autophagy general, essential,    represent option given 26S proteasome (as <br /> is important mobilizing nutrients senes-        judged PBA1 subunit) selectively retained <br /> cence survival N- C-limiting conditions      C starvation atg5-1 mutant. <br /> (Doelling et al., 2002; article). Disruption      Analysis ATG5 supports expectations <br /> single genes encoding ATG5 ATG7 generated        ATG8/12 conjugation pathway plants is <br /> plants senesce earlier hypersensitive      mechanistically similar yeast. In wild-type <br /> N-deficient media C starvation induced extended     Arabidopsis seedlings, ATG5 pool exists as <br /> darkness. However, aspects plant develop-         higher molecular mass species. Both apparent <br /> ment employed autophagic events            mass species absence atg7-1 plants <br /> occur normally mutants, including seed germi-       support identity ATG12-ATG5 adduct. Sur- <br /> nation, pollen function, various forms apoptosis    prisingly, little free ATG5 available wild-type <br /> (e.g. vessel element fiber maturation). Whether        plants nonstarvation conditions, implying <br /> this insensitivity indicates processes   conjugation ATG12 constitutively active. <br /> require autophagy use ATG-independent mecha-            However, free ATG5 pool did rise wild-type <br /> nism(s) known.                                  plants starvation. Its coincidence drop <br />    The phenotypes atg5-1 mutant grown         ATG7 implies conjugation machinery may <br /> C- N-rich limiting conditions indistin-       limiting. A significant portion the <br /> guishable atg7-1, demonstrating         ATG5 pool free form homozygous <br /> disruption just ATG12 conjugation pathway     atg5-2 plants. In case, possible scenario that <br /> effective simultaneously abrogating ATG8       level ATG5 lines exceeded of <br /> and ATG12 conjugation pathways Arabidopsis.             ATG12. Another free form reflects misex- <br /> Both severity starvation phenotypes         pression ATG5 proteins cells/tissues active <br /> the ability plants recover equally ex-      ATG12 conjugation. For atg5-2, upregulation <br /> acerbated atg5-1 atg7-1 mutants. Similar        and/or misregulation induced in- <br /> phenotypes evident plants defective ATG9,      sertion T-DNA 5# UTR. Regardless of <br /> a scaffolding protein functions upstream           reason, rise free ATG5 does appear be <br /> ATG8/12 conjugation autophagy (Hanaoka              deleterious phenotype corresponding <br /> et al., 2002), plants missing v-SNARE VTI12,       atg5-2 plants appeared indistinguishable wild <br /> which likely functions downstream vesicle docking       type. <br /> with tonoplast (Surpin et al., 2003). A recently          With respect ATG8, number distinct species <br /> published work Yoshimoto et al. (2004) reported         evident C-starved nonstarved wild- <br /> a similar set phenotypes double mutant miss-      type seedlings complexity enhanced <br /> ing ATG4a ATG4b proteases required mat-        atg5-1 atg7-1 mutations. Which, any, of <br /> uration seven ATG8s possible      species ATG8-PE adducts unclear. SDS- <br /> recycling PE adduct. Taken together,        PAGE analyses urea total seedling <br /> mutant phenotypes suggest disruption ATG       extracts failed detect likely candidates frac- <br /> system sequence cascade likely         tions enriched membrane proteins. The reason for <br /> 2106                                                                                      Plant Physiol. Vol. 138, 2005 <br />                                                                                    Autophagic Recycling Arabidopsis <br />  <br />  <br /> this failure detect lipidated form unknown.      members ATG4 ATG8 families, <br /> Certainly, complex banding patterns ATG8        suggesting tissue specificity exists protein <br /> protein family interfere detection       isoforms. Like previous studies yeast mam- <br /> ATG8-PE conjugates. Another possibility ATG8-       malian cells (Suzuki et al., 2001; Mizushima et al., <br /> PE adducts rapidly turned vivo and/or          2004), ATG8 proteins provide <br /> rapidly disassembled cytosolic lipases vitro be-     valuable markers detecting autophagic events in <br /> fore analysis. With regard vitro artifacts,     plants. Using Arabidopsis expressing GFP fusion to <br /> extraction tissue directly SDS-PAGE sample       ATG8a, detected fluorescence confocal micros- <br /> buffer followed rapid heat denaturation failed       copy hypocotyl cells ATG8a diffusively throughout <br /> detect new ATG8 species. Yoshimoto et al. (2004) re-       cytoplasm concentrated small spherical <br /> cently identified ATG8 species       vesicles vacuolar lumen. These vesicles mor- <br /> ATG8-PE adducts, using cell fractionation coupled          phologically resembled autophagic bodies described <br /> with atg4a atg4b double mutant missing ATG4         eukaryotes, and, like autophagic bodies, their <br /> protease. However, given ATG8        accumulation required ATG7 enhanced upon <br /> isoforms need protease maturation, remains     N starvation. In recent work Yoshimoto et al. (2004), <br /> possible new species represent partial cleav-   similar punctate structures observed fluores- <br /> age products ATG8 lipidated forms        cence microscopy root cells expressing GFP fusions <br /> proposed. Clearly, biochemical analyses spe-      ATG8a, ATG8e, ATG8i. Electron microscopy of <br /> cies needed resolve issue.                     vesicles identified cargo organelles, <br />    In accord recent data, levels    strongly suggesting autophagic bodies <br /> of most, all, Arabidopsis ATG8 mRNAs            (Yoshimoto et al., 2004). Under condi- <br /> increased starvation. Whereas data analyzed         tions, accumulation vesicles dramatically <br /> the effects dark-induced C starvation, similar global   aided prior incubation seedlings con- <br /> up-regulation evident N C starvation        canamycin A. By blocking tonoplast H1-ATPase, <br /> of seedlings (Yoshimoto et al., 2004; Buchanan-      inhibitor raises vacuolar pH, turn <br /> Wollaston et al., 2005) Suc starvation suspen-      inactivates various vacuolar hydrolyases acidic <br /> sion culture cells (Contento et al., 2004; Rose et al.,    pH optima (Drose et al., 1993; Matsuoka et al., 1997). <br /> 2005). A increase discovered atg5-1     Presumably, inactivation stabilizes auto- <br /> seedlings C starvation (this article), suggesting   phagic bodies breakdown. The protease inhibitor <br /> that impairment autophagic process          E64c appears similar effect blocking <br /> activates expression gene family. In particular,   autophagy-mediated events inhibiting vacuolar <br /> the relative abundance ATG8e ATG8h             Cys proteases (Moriyasu Ohsumi, 1996; Takatsuka <br /> mRNAs dramatically increased, implying            et al., 2004). By mechanisms, concanamycin <br /> a subset ATG8 genes responsive          A E64c shown effectively <br /> defect. Coincidently, isoforms     stabilize GFP targeted vacuole trans- <br /> the ones identified Contento et al. (2004)      port pathways (Tamura et al., 2003). <br /> dramatically up-regulated Suc starvation cul-           One surprising feature possible autophagic <br /> tured cells. The increased levels ATG8 mRNAs         bodies abundance. As seen Figure 10, <br /> turn likely responsible increased ATG8         vacuoles N-starved hypocotyls filled <br /> protein levels seen atg5-1 atg7-1 back-         hundreds vesicles 16-h in- <br /> grounds. However, rule possibility       cubation concanamycin A. Such activity implies <br /> that defects autophagy caused muta-        autophagy encapsulate substantial portions of <br /> tions block turnover ATG8 proteins.        cytoplasm <span id='am-9' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-limiting conditions. Al- <br /> Current models microautophagy macro-          fewer vesicles evident non- <br /> autophagy propose portion ATG8 pool          starved plants, presence suggests autophagy <br /> (possibly ATG8-PE conjugate) decorates pre-        active nonlimiting growth conditions. <br /> autophagosomal structure membranes form               While remains possible autophagy be <br /> autophagosomes macroautophagy               accomplished routes plants, complete <br /> tonoplast invaginations generate autophagic bod-      absence vesicles atg7-1 cells argues that <br /> ies microautophagy (Ohsumi, 2001; Suzuki            autophagic recycling requires ATG <br /> et al., 2001). During process, portion       system. <br /> ATG8 pool trapped result-             Certainly, exploitation ATG by <br /> ing vesicles degraded vesicle     reverse genetics cell biological approaches <br /> the vacuole. Our ability tag possible autophagic        offers excellent opportunity dissect autophagy <br /> bodies GFP-ATG8a fusions consistent      plants. In particular, discovery GFP-ATG8 <br /> scenario.                                                  fusions appear label autophagic bodies, can <br />    DNA microarray analyses indicate       visualized live cells fluorescence confocal <br /> all components Arabidopsis ATG8/12 conjuga-         microscopy, provides marker follow for- <br /> tion pathway expressed organ types exam-        mation cell- environmental-specific man- <br /> ined. Paralog-specific patterns evident          ners. Labeled ATG8s enable purification <br /> Plant Physiol. Vol. 138, 2005                                                                                   2107 <br />  Thompson et al. <br />  <br />  <br /> of autophagic bodies, contents                                Plant Growth Conditions <br /> analyzed define types cargo degraded                                   Arabidopsis seeds vapor-phase sterilized, incubated water 4&#194;&#176;C <br /> autophagic processes.                                                           2 d, grown Gamborg&#226;&#8364;&#8482;s B5 (Sigma, St. Louis) agar liquid medium <br />                                                                                 1% Suc. The plates liquid cultures incubated 21&#194;&#176;C 16-h- <br />                                                                                 light/8-h-dark photoperiod long days, 8-h-light/16-h-dark photoperiod <br />                                                                                 short days, continuous light. For plants exposed long-day <br /> MATERIALS AND METHODS                                                           photoperiod, plants transferred soil 2 weeks. <br />                                                                                     For exposure N-limiting conditions, 1-week-old seedlings grown in <br /> Expression Analysis ATG Genes                                                long-day photoperiod transferred N-deficient agar medium: <br />     DNA microarray design experimental data extracted Ma          Murashige Skoog <span id='am-1' about='Thesaurus:Micronutrient' typeof='owl:Thing'>micronutrient</span> salts (Sigma), 3 mM CaCl2, 1.5 mM MgSO4, <br /> et al. (2005). The 70-mer oligonucleotide set Arabidopsis (Arabidopsis   1.25 mM KH2PO4, 5 mM KCl, 2 mM MES, pH 5.7. After various amounts time <br /> thaliana) Col-0 genome designed synthesized Qiagen/Operon            N-deficient medium, seedlings transferred Gamborg&#226;&#8364;&#8482;s <br /> (http://oligos.qiagen.com/arrays/omad.php) based Arabidopsis ge-         B5 agar. For exposure C-limiting conditions, seedlings grown short- <br /> nome information available February 20, 2002. Roots 6-d-old seed-       day photoperiod 3 weeks transferred soil grown 3 more <br /> lings, stems 4-week-old plants, rosette leaves 3-week-old plants,     weeks. The plants transferred continuous darkness various <br /> floral organs flowering, siliques 3 d pollination, seeds      lengths time, collected immediately returned short-day <br /> collected wild-type Col-0 Arabidopsis. Suspension culture cells       photoperiod 1-week recovery. For confocal microscopy, seeds were <br /> prepared seeds collected logarithmic growth phase. A            germinated Gamborg&#226;&#8364;&#8482;s B5 liquid. After 1 week, seedlings trans- <br /> negative control cutoff value determining genes &#226;&#8364;&#732;&#226;&#8364;&#732;expressed&#226;&#8364;&#8482;&#226;&#8364;&#8482;     ferred N-deficient medium 2-d incubation. Twelve 16 h prior ex- <br /> versus &#226;&#8364;&#732;&#226;&#8364;&#732;not expressed&#226;&#8364;&#8482;&#226;&#8364;&#8482; hybridization obtained ranking     amination, 0.5 mM concanamycin A (Sigma) added growth medium. <br /> values negative control mismatch oligonucleotides high low       The seedlings incubated dark enhance visualiza- <br /> and using 90th percentile spot cutoff value (Ma et al., 2005).       tion GFP vacuole (Tamura et al., 2003). <br /> Expression data included signal majority of <br /> the corresponding spots multiple experiments exceeded cutoff. <br /> The signal intensities different organs normalized equalizing    Protein Isolation Immunoblot Analysis <br /> median value individual gene intensities organ. <br />                                                                                     Total protein isolated liquid-grown soil-grown plants by <br />                                                                                 homogenization 2:1 (volume fresh weight) SDS-PAGE sample buffer <br /> Sequence Analysis ATG5 Proteins                                              (125 mM Tris, pH 6.8, 5% SDS, 20% glycerol, 10% 2-mercaptoethanol), and <br />                                                                                 extracts clarified low-speed centrifugation. Proteins subjected to <br />     ATG5 loci identified Arabidopsis ecotype Col-0 (http://         SDS-PAGE 6 M urea separating gel, stained <br /> www.arabidopsis.org) rice (Oryza sativa; http://www.tigr.org) genomic       silver transferred polyvinylidene difluoride membrane (Millipore, <br /> databases using yeast ATG5 query. Intron/exon positions AtATG5        Bedford, MA) immunoblot analysis using alkaline phosphatase-labeled or <br /> were determined alignment full-length cDNA sequence The        peroxidase-labeled goat anti-mouse goat anti-rabbit immunoglobulins <br /> Arabidopsis Information Resource (http://www.arabidopsis.org). Amino            (Kirkegaard &amp; Perry Laboratories, Gaithersburg, MD) detection. Sample <br /> acid sequence comparisons performed using MACBOXSHADE (In-                 sizes adjusted reflect equal protein equal fresh weight as <br /> stitute Animal Health, Pirbright, UK). The GenBank accession numbers     indicated. <br /> the sequences described article BAB10516 (AtATG5), AP004084             Antibodies ATG5 ATG8a produced rabbits (Polyclonal <br /> (OsATG5), NP_015176 (ScATG5), AAL39741 (DmATG5), AAH02699                   Antibody Service, Madison, WI) using recombinant proteins expressed with <br /> (HsATG5).                                                                       N-terminal His-6 tag. The full-length coding regions inserted NdeI/ <br />                                                                                 SacI fragments NdeI/SacI site pET28a (Stratagene, La Jolla, CA) and <br />                                                                                 introduced Escherichia coli BL21 Codon Plus cells (Novagen, Madison, <br /> Isolation atg5-1 atg5-2 Complementation                              WI). Following 3-h induction log-phase cultures addition 1 mM <br /> of atg5-1                                                                       isopropyl-b-D-thiogalactoside, soluble ATG5 ATG8a proteins puri- <br />                                                                                 fied NiNTA column (QIAGEN Sciences, Germantown, MD). For ATG5, <br />     The atg5-1 atg5-2 T-DNA insertion mutants generated Col-0      protein purified SDS-PAGE gel fragments were <br /> ecotype obtained SAIL (Sessions et al., 2002). The mutations      injected directly. For ATG8a, His-6 tag removed thrombin cleavage, <br /> were tracked PCR using 3# gene-specific primers CAATTCACAGATG-           protein purified second pass NiNTA <br /> GATTGTAACTGCAGAG (atg5-1) CACAAAGGAGATCGAAAAGAA-                            column. Antibodies large subunit spinach Rubisco the <br /> CACCTGTTG (atg5-2) combination LB T-DNA-specific primer             CH-42 subunit pea magnesium-protoporphyrin chelatase (Guo et al., 1998) <br /> GCCTTTTCAGAAATGGATAAATAGCCTTGCTT (Sessions et al., 2002),                       provided Drs. Archie Portis Tanya Falbel, respectively. Anti- <br /> and Basta resistance conferred T-DNAs. The mutants backcrossed   body F1 subunit mitochondrial ATPase described <br /> three times wild-type Col-0 help remove extraneous mutations. Ho-     (Luethy et al., 1993). The anti-PBA1 ATG7 antibodies Doelling <br /> mozygous atg5-1 atg5-2 mutants generated self-pollination hetero-     et al. (2002) Smalle et al. (2002). <br /> zygous plants verified presence T-DNA-specific product and <br /> by absence full-length ATG5 gene product using PCR 5# gene- <br /> specific primer ACGTTAGCCACCAACAGATTAAGCAGTGT combination                    DNA/RNA Isolation, RT-PCR, Analysis <br /> with 3# gene-specific primers.                                        mRNA Abundance <br />     For complementation, full-length coding region ATG5 cDNA was <br /> amplified PCR using primers CTGCATATGGCGAAGGAAGCGGT-                             Genomic DNA isolated 10-d -old seedlings, PCR amplified <br /> CAAGTATGTA TGTCTAGACAATTCACAGATGGATTGTAACTGC-                               35 cycles Ex-Taq polymerase (TaKaRa, Shiga, Japan) ATG5 5# <br /> AGA designed introduce NdeI site 5# end XbaI site    gene-specific primer ACGTTAGCCACCAACAGATTAAGCAGTGT (primer <br /> 3# end, respectively. A single Myc tag cassette appended 5# end   1; Fig. 3), ATG5 3# gene-specific primer CAATTCACAGATGGATTG- <br /> the ATG5 coding sequence, preceded XbaI site. The resulting     TAACTGCAGAG (primer 2; Fig. 3), LB T-DNA primer, primers specific <br /> product digested XbaI inserted downstream CaMV 35S          35S:ATG5 transgene (GGAGAGGTACGTATTTTTACAA TCACC- <br /> promoter binary vector pGSVE9, digested similarly (E.        TTTGAGGAGCTTT). <br /> Babiychuk S. Kushnir, unpublished data). The vector introduced         RNA isolated liquid-grown soil-grown plants using Trizol <br /> Agrobacterium tumefaciens strain GV3101 transformed homozy-       reagent (Invitrogen, Carlsbad, CA). RNA RT-PCR treated DNase <br /> gous atg5-1 plants floral dip method (Clough Bent, 1998). The        RQI (Promega, Madison, WI) prior synthesis first-strand cDNA by <br /> 35S:ATG5 transgene atg5-1 mutation followed hygromycin          Moloney murine leukemia virus-reverse transcriptase (Promega). The first- <br /> and Basta resistance, respectively. T2 plants homozygous atg5-1         strand synthesis primers ATG5 gene-specific primers TCACCTTT- <br /> mutation confirmed contain transgene PCR. T3 seeds          GAGGAGCTTT CACAAAGGAGATCGAAAAGAACACCTGTTG (prim- <br /> resulting self-fertilization used phenotypic analyses.            ers 4 5, respectively; Fig. 3), PAE2 3# gene-specific primer <br />  <br /> 2108                                                                                                                      Plant Physiol. Vol. 138, 2005 <br />                                                                                                                       Autophagic Recycling Arabidopsis <br />  <br />  <br /> CTCAACTCGATAAAATCCATTATCTG (control; Fig. 3). RT-PCR included 35                       Sequence data article deposited EMBL/ <br /> cycles Ex-Taq polymerase (TaKaRa), first-strand synthesis primer,       GenBank data libraries accession numbers BAB10516 (AtATG5), <br /> either ATG5 5# gene-specific primer ATGGCGAAGGAAGCGGTCA                          AP004084 (OsATG5), NP_015176 (ScATG5), AAL39741 (DmATG5), and <br /> (primer 3; Fig. 3) PAE2 5# gene-specific primer CTGACATTGAGGTT-               AAH02699 (HsATG5). <br /> TATCTCAGATCG (control; Fig. 3). <br />     For RNA gel-blot analysis, total RNA isolated according Smalle et al. <br /> (2002). 32P-labeled riboprobes synthesized T7, SP6, T3 RNA poly- <br /> merase using linearized pGEMT (Promega) pBluescript (Stratagene)              ACKNOWLEDGMENTS <br /> cDNA constructions Riboprobe (Promega). The CAB, CDC2a, <br />                                                                                         We thank Drs. Archie Portis, Tanya Falbel, Thomas Elthon the <br /> SEN1, ATG8a probes Doelling et al. (2002). Membranes were <br />                                                                                      supply various antibodies; Dr. Dave Ehrhardt pEGAD vector; Drs. <br /> hybridized overnight 68&#194;&#176;C washed according Smalle et al. (2002) <br />                                                                                      Sean Carrol Steve Paddock help confocal microscopy; Dr. <br /> prior autoradiography. <br />                                                                                      Patrick Krysan Maria Cristina Suarez-Rodriguez help real-time <br />     Abundance individual ATG8 mRNAs quantified using fluorescence- <br />                                                                                      RT-PCR; Dr. Xing-Wang Deng Ligeng Ma access microarray <br /> based real-time RT-PCR. DNase-treated mRNA (1 mg) soil-grown tis- <br />                                                                                      expression data advance publication; Dr. Jan Smalle technical <br /> sue used first-strand cDNA synthesis oligo(dT)18 Moloney <br />                                                                                      assistance; Dr. Brian Downes Joseph Walker helpful discussions. <br /> murine leukemia virus-reverse transcriptase. PCR reactions performed <br /> according manufacturer&#226;&#8364;&#8482;s recommendations, using iCycler iQ Multi-              Received February 2, 2005; revised May 12, 2005; accepted May 24, 2005; <br /> Color Real-Time PCR detection (Bio-Rad, Hercules, CA) iQ SYBR             published July 22, 2005. <br /> Green Supermix (Bio-Rad). Primer pairs engineered generate approx- <br /> imately size PCR product (approximately 300 bp) spanned an <br /> intron ensure amplification cDNA versus genomic DNA. Amplifica- <br /> tion conditions 3 min 95&#194;&#176;C followed 47 cycles PCR (denaturing, <br /> 95&#194;&#176;C 10 s; annealing, 63&#194;&#176;C 30 s; extension, 72&#194;&#176;C 30 s). Fold increases <br />                                                                                      LITERATURE CITED <br /> in RNA levels obtained using CT method (Giulietti et al., 2001). DCT        Brouquisse R, Gaudillere JP, Raymond P (1998) Induction carbon- <br /> values samples determined subtracting CT values              starvation-related proteolysis maize plants submitted <br /> target gene dark-treated tissue CT values target gene            light/dark cycles extended darkness. Plant Physiol 117: <br /> tissue dark treatment background (Col-0 atg5-1). To          1281&#226;&#8364;&#8220;1291 <br /> correct sample variations, relative mRNA abundances nor-                Buchanan-Wollaston V, Page T, Harrison E, Breeze E, Lim PO, Nam HG, <br /> malized Histone H2A transcript. Triplicate values              Lin JF, Wu SH, Swidzinski J, Ishizaki K, et al (2005) Comparative <br /> fold increases reported SE. Gene-specific primers used                 transcriptome analysis reveals significant differences gene expression <br /> follows: ATG8a (At4g21980), TTCAAGATCTCTAACCCTCTCGAGGCA                             signaling pathways developmental dark/starvation- <br /> AGACATCAATGCAGCAGTTGGAGGCAA; ATG8b (At4g04620), TCCTCTG-                                induced senescence Arabidopsis. Plant J 42: 567&#226;&#8364;&#8220;585 <br /> GAGATGAGAATGGCTGAGTCTA TAGAAAGATCCACCAAATGTG-                                    Clough SJ, Bent AF (1998) Floral dip: simplified method Agrobacterium- <br /> TTCTCTCC; ATG8c (At1g62040), TTCCAGTGATCGTAGAGAGAGCTG-                                  mediated transformation Arabidopsis thaliana. Plant J 16: 735&#226;&#8364;&#8220;743 <br /> AAAGA             GTTCTCTCCACTGTAAGTCATGTAGAGAA;                       ATG8d     Contento AL, Kim SJ, Bassham DC (2004) Transcriptome profiling the <br /> (At2g05630), CTCTTGAAAAGAGACAAGCTGAAGCAG ACCACTG-                                   response Arabidopsis suspension culture cells Suc starvation. Plant <br /> TAACTCATGTATAGAAACCCGT; ATG8e (At2g45170), GCATCTTTAA-                                  Physiol 135: 2330&#226;&#8364;&#8220;2347 <br /> GATGGACGACGATTTCGAA ATGTGTTCTCGCCACTGTAAGTGATG-                                  Cutler SR, Ehrhardt DW, Griffitts JS, Somerville CR (2000) Random <br /> TAA; ATG8f (At4g16520), GAATGGCAAAAAGCTCGTTCAAGCAAGAG                                   GFP::cDNA fusions enable visualization subcellular structures in <br /> and CATCATCCTTTTTCTCTTCGTACACAGAA; ATG8g (At3g60640),                                   cells Arabidopsis high frequency. Proc Natl Acad Sci USA 97: <br /> ATTTCGAGAAGAGGAAAGCTGAGGCTTTA GTCTTCTTCCTTATT-                                      3718&#226;&#8364;&#8220;3723 <br /> CTCATCGTAAATGG; ATG8h (At3g06420), AGTCTTTCAAGGATCAA-                                Davis SJ, Vierstra RD (1998) Soluble, highly fluorescent variants green <br /> TTCTCCTCTGAT AAAGTATTGTAGAGAGAGTCCATGCGACT; ATG8i                                   fluorescent protein (GFP) use higher plants. Plant Mol Biol 36: <br /> (At3g15580), TGAAATCGTTCAAGGAACAATACACGTTG TAAGA-                                   521&#226;&#8364;&#8220;528 <br /> TTCGTAGACCGAGTCCATCAGAGC; H2A (At4g27230), CGATTTTT-                             Doelling JH, Walker JM, Friedman EM, Thompson AR, Vierstra RD <br /> GAAAGCCGGTAAGTACGCCGA GCAACTTGCTTAGCTCCTCATCA-                                      (2002) The APG8/12-activating enzyme APG7 required proper <br /> TTCCTC. Sequence analysis PCR products confirmed transcript                  nutrient recycling senescence Arabidopsis thaliana. J Biol Chem <br /> specificity primer pair.                                                        277: 33105&#226;&#8364;&#8220;33114 <br />                                                                                      Drose S, Bindseil KU, Bowmama EJ, Siebers A, Zeeck A, Altendorf K <br />                                                                                         (1993) Inhibitory effect modified bafilomycins concanamycins <br /> GFP-ATG8 Lines Fluorescence Confocal Microscopy                                     P- V-type adenosinetriphosphatases. Biochemistry 32: <br />                                                                                         3902&#226;&#8364;&#8220;3906 <br />     The full-length coding region ATG8a cDNA amplified PCR             Giuliano G, Hoffman NE, Ko K, Scolnik PA, Cashmore AR (1988) A light- <br /> using primers CTCAATTGATGGCTAAGAGTTCCTTC TCGGATCCT-                                 entrained circadian clock controls transcription plant genes. <br /> CATCCAAAAGTGTTCTC designed introduce MunI site 5# end                       EMBO J 7: 3635&#226;&#8364;&#8220;3642 <br /> and BamHI site 3# end, respectively. The resulting product              Giulietti A, Overbergh L, Valckx D, Decallonne B, Bouillon R, Mathieu C <br /> digested enzymes inserted downstream CaMV 35S                 (2001) An overview real-time quantitative PCR: applications to <br /> promoter GFP coding region binary vector pEGAD (Cutler et al.,             quantify cytokine gene expression. Methods 25: 386&#226;&#8364;&#8220;401 <br /> 2000), digested EcoRI BamHI. The vector introduced            Guo R, Luo M, Weinstein JD (1998) Magnesium-chelatase develop- <br /> into A. tumefaciens strain GV3101 transformed wild-type                   ing pea leaves: characterization soluble extract chloroplasts <br /> Col-0 homozygous atg7-1 plants floral dip method (Clough                 resolution required protein fractions. Plant Physiol 116: <br /> Bent, 1998). The 35S:GFP-ATG8a transgene followed Basta resistance.              605&#226;&#8364;&#8220;615 <br /> T2 plants homozygous atg7-1 mutation confirmed PCR. T3               Hammond EM, Brunet CL, Johnson GD, Parkhill J, Milner AE, Brady G, <br /> seeds resulting self-fertilization tested GFP expression using            Gregory CD, Grand RJ (1998) Homology human apoptosis <br /> a fluorescence dissecting microscope immunoblot analysis anti-              specific protein product APG5, gene involved autophagy <br /> GFP antibodies. Antibodies GFP produced rabbits (Polyclonal             yeast. FEBS Lett 425: 391&#226;&#8364;&#8220;395 <br /> Antibody Service) using recombinant protein described Davis                Hanaoka H, Noda T, Shirano Y, Kato T, Hayashi H, Shibata D, Tabata S, <br /> Vierstra (1998).                                                                        Ohsumi Y (2002) Leaf senescence starvation-induced chlorosis are <br />     Confocal fluorescence microscopy conducted using Bio-Rad MCR1024              accelerated disruption Arabidopsis autophagy gene. Plant <br /> laser scanning confocal microscope 600 3 magnification (Richmond,                    Physiol 129: 1181&#226;&#8364;&#8220;1193 <br /> CA). Excitation 480 nm optical sections collected               Hoffman <br /> </body></html>