<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Got1p and Sft2p: membrane proteins involved in <br /> traffic <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the Golgi complex <br />  <br />  <br />  <br />                               implicated in every fusion event that has been studied <br />                         so far. Yeast has eight identifiable syntaxins distributed <br />                                                               amongst the various organelles of the cell together with a <br />               dozen or so other SNAREs, and extensive localization <br />  <br />  <br />                                                               and functional studies have resulted in a route-map of the <br />  <br />                                                               trafficking pathways along and between the exocytic and <br />                                                               endocytic pathways (Pelham, 1999). <br />  <br />                                                                  Despite the importance of the SNAREs, it has become <br />                                                               apparent that they are not sufficient for efficient membrane <br /> Traffic through the yeast Golgi complex depends on a <br />                                                               fusion, and that mutual recognition between them cannot <br /> member of the syntaxin family of SNARE proteins, <br />                                                               entirely account for the specificity of membrane fusion <br /> Sed5p, present in early Golgi cisternae. Sft2p is a <br />                                                               events. Additional peripheral membrane proteins, some <br /> non-essential tetra-spanning membrane protein, found <br />                                                               recruited by small GTPases of the rab/ypt family, can <br /> mostly in the late Golgi, that can suppress some sed5 <br /> alleles. We screened for mutations that show synthetic        mediate specific tethering of membranes to each other prior <br /> lethality with sft2 and found one that affects a pre-         to SNARE engagement, and may contribute significantly to <br /> viously uncharacterized membrane protein, Got1p, as           the selectivity of the process (Simonsen et al., 1998; <br /> well as new alleles of sed5 and vps3. Got1p is an             Sonnichsen et al., 1998; Ungermann et al., 1998b). In <br /> evolutionarily conserved non-essential protein with a         yeast Uso1p, a protein containing extensive coiled-coil <br /> membrane topology similar to that of Sft2p. Immuno-           motifs is recruited to membranes by Ypt1p and can mediate <br /> fluorescence and subcellular fractionation indicate that      the docking of ER-derived vesicles with Golgi membranes <br /> it is present in early Golgi cisternae. got1 mutants, but     in vitro, even when SNAREs are inactivated (Cao et al., <br /> not sft2 mutants, show a defect in an in vitro assay for      1998). Other proteins that are required for transport to <br /> ER&#226;&#8364;&#8220;Golgi transport at a step after vesicle tethering to       and through the Golgi complex include Sec35p <br /> Golgi membranes. In vivo, inactivation of both Got1p          (VanRheenan et al., 1998) and the TRAPP complex <br /> and Sft2p results in phenotypes ascribable to a defect in     (Sacher et al., 1998). How these proteins are bound to <br /> endosome&#226;&#8364;&#8220;Golgi traffic, while their complete removal          membranes is not known. Other than the SNAREs, few <br /> results in an ER&#226;&#8364;&#8220;Golgi transport defect. Thus the             integral membrane proteins with a transport function are <br /> presence of either Got1p or Sft2p is required for vesicle     known, although one has recently been described: the <br /> fusion with the Golgi complex in vivo. We suggest             Ypt1p-interacting integral membrane protein Yip1p (Yang <br /> that Got1p normally facilitates Sed5p-dependent fusion        et al., 1998). <br /> events, while Sft2p performs a related function in the           In yeast, Golgi function shows an absolute requirement <br /> late Golgi.                                                   for only one syntaxin, Sed5p, which cycles through the <br /> Keywords: Golgi/SNARE/syntaxin/yeast                          ER but is abundant in early Golgi cisternae (Hardwick <br />                                                               and Pelham, 1992; Holthuis et al., 1998b; Wooding and <br />                                                               Pelham, 1998). Sed5p interacts with at least seven other <br />                                                               SNAREs, forming at least three distinct complexes <br /> Introduction                                                  (Nichols and Pelham, 1998). One of these, characterized <br /> Traffic of proteins through the secretory and endocytic       by the presence of the SNARE Bos1p, is involved in the <br /> pathways of eukaryotic cells involves the budding and         fusion of ER-derived vesicles with early Golgi membranes, <br /> fusion of transport vesicles. Membrane fusion is highly       and/or the homotypic fusion of such vesicles to form new <br /> specific, and typically involves the docking of a vesicle     cisternae. The other two, which contain Gos1p and Vti1p, <br /> with a larger structure such as a Golgi cisterna, followed    respectively, are probably formed when retrograde vesicles <br /> by the bilayer fusion event itself. These events depend on    originating <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> later Golgi cisternae and from the endo- <br /> integral membrane proteins termed SNAREs which form           somal system fuse with early cisternae. Two more syn- <br /> tight helical bundles with each other, and in so doing help   taxins, Tlg1p and Tlg2p, seem to be at least partly <br /> to force membranes together (Rothman, 1994; Hay and           associated with the Golgi complex, though they are <br /> Scheller, 1997; Nichols and Pelham, 1998; Sutton et al.,      probably also in early endosomes (Abeliovich et al., 1998; <br /> 1998). Disruption of these bundles prior to (or after)        Holthuis et al., 1998b; Seron et al., 1998). They co- <br /> fusion is mediated by proteins termed SNAP (Sec17p in         localize with late Golgi markers such as Kex2p and are <br /> yeast) and NSF (Sec18p), and is necessary to allow            required for the retrieval of proteins from the endocytic <br /> formation of complexes which span the fusing membranes        pathway to the yeast equivalent of the trans-Golgi network <br /> (So&#194;&#168;llner et al., 1993; Ungermann et al., 1998a). Of          (Holthuis et al., 1998a,b). <br /> particular importance are the members of the syntaxin            A screen for genes which, when overexpressed, could <br /> family of SNAREs, a member of this family having been         suppress the growth <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>phenotype</span> of a sed5 temperature- <br />  <br /> 3934                                                                          &#194;&#169; European Molecular Biology Organization <br />                                                                                    Got1p and Sft2p Golgi transport proteins <br />  <br />  <br /> sensitive <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> yielded two genes termed sft1 and sft2      sectoring. The mutated allele of sed5 was isolated by PCR <br /> (Banfield et al., 1995). Sft1p is a SNARE that binds to        and sequenced, which revealed a single <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> Sed5p but is mostly present in late Golgi membranes            Y311&#226;&#8224;&#8217;D. This amino acid is nine residues from the <br /> which lack Sed5p; it is required for transport of resident     transmembrane domain, and the mutation introduces a <br /> Golgi proteins from late to early Golgi cisternae (Banfield    charge on the predominantly hydrophobic face of the <br /> et al., 1995; Wooding and Pelham, 1998). Sft2p is a            coiled-coil domain of the protein. Hence, it would be <br /> conserved protein with four putative transmembrane             expected to hinder the formation or function of Sed5p- <br /> domains which is also located in a late Golgi compartment,     containing SNARE complexes. The synthetic lethality of <br /> but enters presumptive retrograde intra-Golgi vesicles         this mutation with sft2 mirrors the suppression of sed5-1 <br /> whose fusion depends on Sft1p and Sed5p (Wooding and           by SFT2 overexpression, and further emphasizes the link <br /> Pelham, 1998). Its genetic interaction with Sed5p suggests     between these two proteins. <br /> that Sft2p may be an additional membrane component                The sectoring phenotype of the second mutant was <br /> involved in the docking or fusion process, but surprisingly    restored by a region of chromosome IV containing the <br /> strains lacking SFT2 grow normally and show no obvious         VPS3 gene. VPS3 encodes a large, hydrophilic, membrane- <br /> secretory defect, and hence the normal role of this gene       associated protein which is required for efficient sorting <br /> has remained unknown.                                          of carboxypeptidase Y (CPY) to the vacuole; its precise <br />    We sought clues to the function of SFT2 by isolating        function is unknown. Our isolation of this gene suggests <br /> mutant yeast strains in which it is essential. Mutations       a possible involvement of SFT2 in traffic between the <br /> that show synthetic lethality with sft2 include a new allele   endocytic and exocytic pathways, but we did not analyse <br /> of sed5, and a disruption of a previously uncharacterized      the mutant further. <br /> gene we term GOT1. Got1p is a functionally conserved              The final mutant was complemented by an 8 kb genomic <br /> protein with four putative transmembrane domains, which        fragment from chromosome XIII. After restriction and <br /> partially co-localizes with Sed5p. It is not essential for     subcloning, the complementing activity was localized <br /> growth, but its removal significantly reduces the efficiency   to the previously uncharacterized open reading frame <br /> of ER&#226;&#8364;&#8220;Golgi transport in vitro, hindering not the initial      YMR292w. Sequencing of this region from the mutant <br /> tethering of vesicles, but their subsequent SNARE-             revealed a single point mutation, G73&#226;&#8224;&#8217;S. To confirm that <br /> dependent fusion. In vivo, simultaneous removal of both        mutation of YMR292w was responsible for the synthetic <br /> Sft2p and Got1p affects both ER&#226;&#8364;&#8220;Golgi transport and the        lethality with sft2, we disrupted this gene in our wild-type <br /> retrieval of proteins from the endocytic pathway. Thus         strain (SEY6210). The disruptant was viable and had no <br /> these two integral membrane proteins seem to have par-         obvious growth or secretory defect. However, when the <br /> tially redundant roles in the fusion of vesicles with Golgi    cells were mated with sft2 null cells and the diploids <br /> membranes. We suggest that Got1p normally facilitates          sporulated, no viable haploids carrying both disruptions <br /> Sed5p-dependent fusion events, while Sft2p provides an         could be recovered. This establishes YMR292w as a non- <br /> equivalent function in the late Golgi.                         essential gene that is required for growth in the absence <br />                                                                of SFT2. In the light of subsequent experiments we named <br />                                                                this gene GOT1, for Golgi transport, and the G73&#226;&#8224;&#8217;S <br /> Results <br />                                                                allele got1-1. <br /> Synthetic lethal screen for isolation of mutants <br /> that require Sft2p function                                    Got1p is a member of an evolutionarily conserved <br /> To gain insight into the function of Sft2p, we undertook       family of proteins <br /> a genetic approach. A synthetic lethal screen was used to      The predicted protein sequence encoded by GOT1 is <br /> identify other genes whose mutation renders Sft2p essential    shown in Figure 1A. Got1p is a 138-residue protein with <br /> for <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span>. The screen was based on a colony sectoring       a calculated mass of 15.4 kDa. It is likely to be an integral <br /> assay (Koshland et al., 1985; Kranz and Holm, 1990).           membrane protein since its sequence contains at least <br /> Strains of each mating type with mutations in ade2, ade3       three and probably four hydrophobic stretches of sufficient <br /> and sft2 were transformed with a centromeric plasmid           <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> to be transmembrane domains (Figure 1B). <br /> carrying URA3, ADE3 and SFT2. The presence of the                 Database searches revealed that Got1p is a member of <br /> ADE3 gene on the plasmid caused these strains to form          a phylogenetically conserved family of proteins. Homo- <br /> red colonies, but under non-selective conditions spon-         logues were identified in Caenorhabditis elegans (39.8% <br /> taneous loss of the plasmid generated white sectors. After     identity), Plasmodium falciparum (33% identity) and in <br /> UV mutagenesis, colonies that no longer sectored were          mammals. None of these homologues had previously been <br /> selected for further analysis. From a total of ~100 000        characterized. As shown in Figure 1A, two related gene <br /> colonies screened, three mutants with a clear synthetic        sequences were also identified in the human expressed <br /> <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> were identified. The corresponding            sequence tag (EST) database. We designated these hGot1a <br /> mutant genes were identified by complementation of the         and hGot1b, with 38.4% and 41.3% identity to Got1p, <br /> sectoring phenotype after transformation with a yeast          respectively. The sequence of hGot1a was constructed <br /> genomic DNA library. For each mutant, one or two               entirely from overlapping ESTs. The sequence of hGot1b <br /> complementing plasmids were obtained and sequences             was not completely represented in ESTs, but was obtained <br /> determined from each end of the genomic DNA inserts.           from the corresponding cDNA clone. <br />    The DNA fragment able to complement the first mutant           To confirm the functional relevance of these homologies <br /> consisted of a region of chromosome XII which contains         we expressed the hGot1b sequence from a centromere- <br /> the complete SED5 gene, and transformation of a plasmid        based yeast vector, and found that it could support growth <br /> containing only the SED5 gene was sufficient to restore        of a got1 sft2 double null strain at a rate comparable with <br />  <br />                                                                                                                       3935 <br />  S.Conchon et al. <br />  <br />  <br />  <br />  <br /> Fig. 1. Structure and membrane association of Got1p and Sft2p. (A) Predicted amino acid sequences of Got1p and Sft2p aligned with those of <br /> homologues identified for Got1p in P.falciparum (accession number AL010285), C.elegans (accession number U23521) and humans (hGot1a and <br /> hGot1b), and for Sft2p in C.elegans (accession number CAA93859) and mouse (accession number AA790425). Residues identical in all species are <br /> shown large and bold, similar residues in bold. Lines indicate the hydrophobic potential transmembrane domains. The position of the single amino <br /> acid change in got1-1, G73&#226;&#8224;&#8217;S, is indicated. Note that the exon containing the N-terminus of the C.elegans Got1p sequence could not be <br /> unequivocally identified. (B) Hydropathy profiles of Got1p and Sft2p. (C) Membrane association of myc-tagged proteins expressed in yeast. Myc- <br /> tagged proteins were expressed in yeast, 100 000 g membrane pellets prepared and extracted with buffer, sodium chloride, sodium carbonate or <br /> Triton X-100 as indicated before recentrifugation and analysis of the pellets (P) and supernatants (S) by immunoblotting. (D) Protease protection <br /> assays. Membrane preparations containing the indicated proteins tagged at either the N- or C-terminus, as indicated, were digested with proteinase K <br /> in the presence or absence of detergent, and then analysed by immunoblotting. (E) Binding of anti-myc mAb 9E10 to unpermeabilized microsomes. <br /> Membranes from cells expressing the indicated myc-tagged proteins, or none (no myc), were incubated with antibody, pelleted and bound antibody <br /> light chains detected by immunoblotting. <br />  <br />  <br /> 3936 <br />                                                                                      Got1p and Sft2p Golgi transport proteins <br />  <br />  <br /> that of the wild-type strain (see Materials and methods <br /> for details). Thus Got1p evidently provides a function that <br /> is common to evolutionarily distant eukaryotes. <br />    For comparison, Figure 1A also shows the sequence of <br /> Sft2p, and related sequences from C.elegans and mouse <br /> (the latter a partial sequence from an EST). Though not <br /> closely related to Got1p, Sft2p also has a compact region <br /> of four putative transmembrane domains, and it is these <br /> that show most evolutionary conservation. <br />  <br /> Got1p and Sft2p are membrane proteins <br /> To investigate the membrane location of Got1p, the <br /> molecular clone was modified to encode three myc epitopes <br /> at the C-terminus of the protein. Yeast cells expressing <br /> this tagged protein produced a single product detectable <br /> on immunoblots with anti-myc antibodies, with an apparent <br /> size of ~25 kDa. Expression of the tagged protein from a <br /> centromere vector was sufficient to allow sectoring (i.e. <br /> loss of untagged Got1p) of the original got1-1 sft2&#226;&#710;&#8224; <br /> mutant strain, and also supported growth of the got1&#226;&#710;&#8224; <br /> sft2&#226;&#710;&#8224; double disruptant, indicating that the tagged Got1p <br /> is functional. A version of Sft2p tagged with three myc <br /> epitopes at the N-terminus has previously been described <br /> (Wooding and Pelham, 1998). <br />    Both Sft2p and Got1p were efficiently pelleted from        Fig. 2. Immunofluorescent localization of Got1p. Myc-tagged Got1p <br />                                                               was compared with endogenous Sed5p by double-<span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> immuno- <br /> cell homogenates by centrifugation at 100 000 g, and          fluorescence (top panels) or with a co-expressed GFP&#226;&#8364;&#8220;Sft2p fusion <br /> Figure 1C shows that they could not be extracted from         protein. Arrows indicate identical positions in each pair of images. <br /> these pellets with sodium chloride or sodium carbonate, but <br /> could be solubilized by detergent, confirming a membrane <br /> location for both proteins.                                   contained the early Golgi t-SNARE Sed5p. Such partial <br />    Previous work indicated that the N-terminus of Sft2p       co-localization has often been seen for proteins that reside <br /> was likely to be cytoplasmic (Wooding and Pelham, 1998),      in the same region of the Golgi, presumably reflecting <br /> and this was confirmed by protease digestion of the           differences in the kinetics of their delivery to and removal <br /> membranes (Figure 1D). Whether or not detergent was           from individual cisternae (Rayner and Munro, 1998; <br /> added, the N-terminal myc tag was completely digested         Wooding and Pelham, 1998). In contrast, there was much <br /> by proteinase K, no protected fragments being detectable.     less obvious overlap with GFP&#226;&#8364;&#8220;Sft2p, which has pre- <br /> As a control, the Golgi enzyme Mnn1p, myc-tagged at its       viously been shown to be functional and localized to <br /> (lumenal) C-terminus, was subjected to the same procedure     a later Golgi compartment than Sed5p (Wooding and <br /> and found to be substantially protease resistant in the       Pelham, 1998). <br /> absence of detergent (Figure 1D).                                To investigate further the localization of Got1p a strain <br />    The Myc-tagged C-terminus of Got1p was found to be         expressing the tagged protein was analysed by subcellular <br /> fully accessible to protease. Furthermore, a version of the   fractionation. A 100 000 g pellet was fractionated on an <br /> protein bearing three myc tags at the N-terminus was          equilibrium sucrose-density gradient (Figure 3), and the <br /> similarly protease sensitive, and again no smaller frag-      distribution of Got1&#226;&#8364;&#8220;myc compared with that of other <br /> ments could be detected (Figure 1D). This suggests that       markers: Vam3p (vacuoles), Pep12p (endosomes), Kex2p, <br /> the protein has an even number of transmembrane domains,      Tlg1p and Tlg2p (late Golgi/early endosomes) and Sed5p. <br /> with both termini cytoplasmically exposed. This was           All the data shown are from a single set of gradient <br /> confirmed by an independent assay in which anti-myc           fractions, but for clarity the results are shown in two panels, <br /> antibodies were bound to and co-sedimented with un-           each containing the profile of Got1p. The distributions of <br /> permeabilized microsomes bearing Got1p with myc tags          the different markers were similar to those previously <br /> at either the N- or C-terminus (Figure 1E). The cytoplasmic   described (Holthuis et al., 1998b). Under these conditions, <br /> location of the C-terminus was also confirmed for hGot1a      Got1&#226;&#8364;&#8220;myc was separated from the TGN, endosomal and <br /> (see below).                                                  vacuolar markers, but most of it was found in fractions <br />    Based on these results and the hydropathy profiles, we     containing Sed5p. As expected from the immunofluores- <br /> conclude that both Got1p and Sft2p are likely to be tetra-    cence observations, the Sed5p and Got1p profiles were <br /> spanning membrane proteins with their termini on the          similar but not completely identical. We conclude that <br /> cytoplasmic side of the membrane.                             Got1p resides mostly in early Golgi cisternae. <br />                                                                  Further evidence for the Golgi location of Got1p was <br /> Got1p is an early Golgi protein                               provided by expression and immunolocalization of a <br /> Visualization of myc-tagged Got1p by immunofluores-           C-terminally myc-tagged version of hGot1a in COS cells. <br /> cence revealed a punctate <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, characteristic of the      Figure 4 shows co-staining of cells for hGot1a and &#206;&#178;9-COP <br /> yeast Golgi apparatus. As shown in Figure 2, some             (mainly found in the cis Golgi), &#206;&#179;-adaptin and TGN46 <br /> but not all of the structures containing Got1&#226;&#8364;&#8220;myc also        (TGN markers). Though all proteins gave Golgi-like <br />  <br />                                                                                                                               3937 <br />  S.Conchon et al. <br />  <br />  <br />  <br />  <br />                                                                           Fig. 4. Location of hGot1a expressed in COS cells. Cells expressing <br />                                                                           myc-tagged protein were analysed by double-label immuno- <br />                                                                           fluorescence, using antibodies to &#206;&#178;9-COP, &#206;&#179;-adaptin or KDEL- <br />                                                                           containing ER proteins. The hGot1a is concentrated primarily in the <br />                                                                           Golgi region, though some faint ER-like staining is also visible. The <br />                                                                           lower panels show cells that were freeze&#226;&#8364;&#8220;thawed to rupture the plasma <br /> Fig. 3. Subcellular fractionation of cells expressing myc-tagged Got1p.   membrane alone, or subsequently treated with detergent to <br /> Membranes from a 100 000 g pellet were separated on a sucrose-            permeabilize internal membranes. Note that the C-terminal tag on <br /> density gradient, and fractions probed by immunoblotting (or, in the      hGot1a is accessible to antibodies with or without detergent treatment, <br /> case of Kex2p, by enzyme assay). All data are from a single gradient,     whereas the lumenal domain of TGN46 can be detected in the Golgi <br /> plotted in two parts for clarity. Note that the Got1p profile most        region only after detergent treatment. <br /> closely resembles that of Sed5p. <br />  <br /> patterns, we consistently observed better co-localization                    We first transformed the non-sectoring got1-1 sft2&#226;&#710;&#8224; <br /> of hGot1a with &#206;&#178;9-COP than with the TGN markers. This                     strain with a plasmid containing the SFT2 cDNA down- <br /> suggests that hGot1a is not restricted to the trans/TGN                   stream of a GAL1 inducible promoter. These cells grew <br /> region, and may be preferentially localized in cis cisternae.             normally in galactose-containing medium and were able <br /> We also observed some staining of the nuclear envelope                    to lose the SFT2-containing plasmid which had been <br /> and ER (Figure 4). When cells were lysed by freeze&#226;&#8364;&#8220;                       supporting their growth. Subsequent transfer of the cells <br /> thawing, without detergent, the C-terminal myc tag on                     to glucose-containing medium resulted in growth arrest <br /> hGot1a was clearly accessible to antibodies. In contrast,                 after 12&#226;&#8364;&#8220;15 h. Electron microscopy (EM) samples were <br /> an antibody recognizing the lumenal domain of TGN46                       prepared of these cells after 14 h in either galactose or <br /> stained the Golgi region only after detergent treatment.                  glucose. As shown in Figure 5B, C and G, the repression <br /> Thus, hGot1a has a membrane orientation and distribution                  of Sft2p synthesis resulted in a significant accumulation <br /> in mammalian cells comparable with that observed for                      of ER membranes. This suggests that the presence of <br /> Got1p in yeast.                                                           either Sft2p or Got1p is required, directly or indirectly, <br />                                                                           for the maintenance of efficient ER&#226;&#8364;&#8220;Golgi transport. In <br /> Isolation and properties of conditional alleles of                        addition, we observed a moderate number of darkly <br /> got1 and sft2                                                             staining circular structures similar to those previously seen <br /> Deletion of either GOT1 or SFT2 alone did not affect cell                 in cells lacking the TGN/early endosomal SNARE Tlg1p <br /> growth, and preliminary studies showed no obvious defect                  (Holthuis et al., 1998b). <br /> in the processing or sorting of CPY in these mutants. We                     We then used PCR mutagenesis to prepare thermosensi- <br /> therefore sought clues to their function by analysing the                 tive (ts) alleles of both GOT1 and SFT2, selecting them <br /> phenotypes of cells lacking both proteins.                                for their ability to maintain growth of the got1&#226;&#710;&#8224; sft2&#226;&#710;&#8224; <br />  <br /> 3938 <br />                                                                                                      Got1p and Sft2p Golgi transport proteins <br />  <br />  <br />  <br />  <br /> Fig. 5. Electron micrographs of got1 and sft2 mutants. Cells were permanganate-fixed to accentuate membranes. (A) sft2&#226;&#710;&#8224;, with SFT2 on a plasmid <br /> (SCY11); indistinguishable from wild type. (B) sft2&#226;&#710;&#8224; got1-1 with pGAL1-SFT2, grown on galactose. (C) As (B), but after transfer to glucose for <br /> 14 h. Note accumulation of ER (arrowheads) and some small darkly staining structures (arrow). (D) got1&#226;&#710;&#8224; sft2-1ts, grown at 25&#194;&#176;C. (E) and (F) As <br /> (D), but after incubation at 37&#194;&#176;C for 3 h. Note some accumulation of ER in (E) (arrowhead), and a cluster of vesicles or tubules, enlarged in (F). <br /> N, nucleus; V, vacuole. (G) Quantitation of the ER accumulation in various strains. Random fields of cells were analysed, and each cell section <br /> scored for the proportion of the circumference that was occupied by peripheral ER. <br />  <br />  <br /> strain at low but not high temperature. One ts allele of                     to the non-permissive temperature there was some accumu- <br /> each gene was selected for further study. The got1&#226;&#710;&#8224; sft2-1                   lation of ER membranes, and clusters of small vesicles <br /> strain had the strongest growth phenotype: it grew slightly                  were frequently seen (Figure 5D&#226;&#8364;&#8220;G). It appears, therefore, <br /> more slowly than an SFT2 control at 25&#194;&#176;C and upon                            that the ts alleles have a less severe phenotype than the <br /> shifting to 37&#194;&#176;C ceased growth altogether after a lag of                     complete depletion of both Got1p and Sft2p; even under <br /> 2 h. The sft2&#226;&#710;&#8224; got1-2 strain grew normally at 25&#194;&#176;C but                       conditions where their growth was inhibited, their morpho- <br /> took longer to stop growing at 37&#194;&#176;C, inhibition being                        logy did not suggest a tight block to secretion. <br /> apparent by 2 h but complete only after 6 h.                                    As a further test of secretory pathway function, the <br />    EM samples of these strains were prepared and their                       maturation of CPY was examined. In pulse&#226;&#8364;&#8220;chase experi- <br /> morphologies at 25 and 37&#194;&#176;C were compared. The sft2&#226;&#710;&#8224;                         ments (Figure 6A) CPY appears as a p1 precursor in the <br /> got1-2 ts strain showed no obvious abnormality at 25&#194;&#176;C,                      ER, is then modified to a larger form, p2, in the Golgi <br /> and at 37&#194;&#176;C accumulated only a small amount of ER                            and passes through endosomes to vacuoles where it is <br /> membrane (Figure 5G). The got1&#226;&#710;&#8224; sft2-1 strain also                           processed to its mature form (m) by proteolysis. The <br /> appeared normal at low temperature; when it was shifted                      got1-1 GAL&#226;&#8364;&#8220;SFT2 strain, when grown on galactose to <br />  <br />                                                                                                                                              3939 <br />  S.Conchon et al. <br />  <br />  <br />                                                                           maintain levels of Sft2p, was able to process CPY but <br />                                                                           showed some delay, with a noticeable persistence of the <br />                                                                           p1 form. After 14 h on glucose, to repress Sft2p synthesis, <br />                                                                           CPY processing was completely blocked at the p1 stage, <br />                                                                           in agreement with the EM data showing accumulation of <br />                                                                           ER under these conditions. <br />                                                                              As with the EM analysis, the CPY processing phenotype <br />                                                                           of the ts alleles was much milder than in cells depleted <br />                                                                           of both proteins. In the got1&#226;&#710;&#8224; sft2-1 ts strain CPY <br />                                                                           maturation was slower than normal, with a persistence of <br />                                                                           the p1 precursor form, but maturation was not completely <br />                                                                           blocked even at 37&#194;&#176;C. In the sft2&#226;&#710;&#8224; got1-2 strain processing <br />                                                                           was only slightly slowed, even at 37&#194;&#176;C (Figure 6B). In <br />                                                                           this experiment a portion of the p2 form of CPY was <br />                                                                           secreted into the medium by the mutants, but this amounted <br />                                                                           to no more than 10&#226;&#8364;&#8220;15% of the total (not shown). This <br />                                                                           was also apparent when secreted CPY was adsorbed <br />                                                                           on to nitrocellulose and detected by immunoblotting <br />                                                                           (Figure 6C). Both ts mutants showed slightly enhanced <br />                                                                           secretion of CPY at the non-permissive temperature, but <br />                                                                           this effect was mild compared with the level of secretion <br />                                                                           from pep12&#226;&#710;&#8224; cells, which have a strong mis-sorting <br />                                                                           phenotype. <br />                                                                              We also examined the secretion of periplasmic invertase <br />                                                                           from the ts mutants (Figure 6D). Since our standard <br />                                                                           strains lack the chromosomal invertase gene, they were <br />                                                                           transformed with a plasmid which expresses a myc-tagged <br />                                                                           version of the secreted form of invertase. Examination of <br />                                                                           the intracellular invertase by immunoblotting showed that <br />                                                                           there was a slight increase in the hyperglycosylated forms <br />                                                                           of the enzyme in the mutant strains, suggesting a reduced <br />                                                                           rate of transport through the Golgi to the cell surface. In <br />                                                                           addition, the invertase present in the got1&#226;&#710;&#8224; sft2-1 strain <br />                                                                           had slightly shorter polysaccharide chains than normal, <br />                                                                           consistent with a reduction in mannosyltransferase activity <br />                                                                           in the Golgi apparatus of this strain. <br />                                                                              Total glycoprotein secretion was also examined in these <br />                                                                           strains by incubating spheroplasts with 35S-labelled amino <br />                                                                           acids and monitoring the appearance of labelled concanav- <br />                                                                           alin A-binding glycoproteins in the medium (as described <br />                                                                           in Holthuis et al., 1998b). No significant quantitative or <br />                                                                           qualitative abnormalities were observed (data not shown). <br />                                                                              Together, these results show that Got1p and Sft2p affect <br />                                                                           ER&#226;&#8364;&#8220;Golgi transport, as shown by the accumulation of ER <br />                                                                           when both are absent or depleted, but that the growth <br />                                                                           defect of the ts alleles cannot be accounted for entirely <br />                                                                           by a block to this process, or indeed to any severe block <br />                                                                           to the exocytic pathway. Thus, Got1p and Sft2p must <br /> Fig. 6. Processing of CPY and invertase in got1 sft2 mutants. <br /> (A) Pulse&#226;&#8364;&#8220;chase analysis of CPY in strains containing and lacking <br />                                                                           also contribute to some other function that is required <br /> Got1p and Sft2p. Wild-type cells, or cells of the got1-1 GAL&#226;&#8364;&#8220;SFT2         for growth. <br /> strain grown in galactose or transferred to glucose medium for 14 h <br /> were labelled for 10 min, and chased for the times indicated at 30&#194;&#176;C <br /> before lysis and immunoprecipitation of CPY. The positions of the ER      Evidence for defects in endosome&#226;&#8364;&#8220;Golgi traffic <br /> precursor (p1), the Golgi-modified precursor (p2) and the mature          The Golgi complex receives traffic not only from the ER, <br /> vacuolar enzyme (m) are indicated. (B) Pulse&#226;&#8364;&#8220;chase analysis of CPY        but also from the endocytic pathway. This is necessary <br /> in the temperature-sensitive mutants. Cells were labelled and chased at <br /> 25&#194;&#176;C, or preincubated at 37&#194;&#176;C for 2.5 h and then labelled and chased <br />                                                                           both for the retrieval of TGN proteins from endosomes <br /> at this temperature. (C) Analysis of secreted CPY. Patches containing     and for the recycling of at least some plasma membrane <br /> similar numbers of cells were grown in contact with a nitrocellulose      proteins. Defects in this process are revealed by a loss of <br /> filter at either 25&#194;&#176;C or 37&#194;&#176;C for 3 h, and the secreted CPY detected on   the Kex2p activity. Kex2p, a processing protease, usually <br /> the filter with antibodies. (D) Immunoblotting of intracellular myc- <br /> tagged invertase precursors. The indicated strains were incubated at <br />                                                                           recycles between the TGN and an endosomal compartment, <br /> 25&#194;&#176;C, or for 2.5 h at 37&#194;&#176;C prior to analysis. The positions of            but if retrieval is blocked it passes instead to the vacuole, <br /> molecular weight markers, and of the heterodisperse mature forms of       where it is degraded (e.g. Holthuis et al., 1998b). Figure 7 <br /> invertase (line) are indicated.                                           shows that in both ts strains, a shift to the non-permissive <br />  <br /> 3940 <br />                                                                                                Got1p and Sft2p Golgi transport proteins <br />  <br />  <br />  <br />  <br /> Fig. 7. Loss of Kex2p activity from ts mutants. Cells of the strains <br /> indicated were incubated at 37&#194;&#176;C for 4 h, then 100 000 g membrane <br /> pellets were prepared and assayed for Kex2p activity. The percentage <br /> activity remaining was estimated relative to that in the same strains <br /> maintained at 25&#194;&#176;C (set to 100%). <br />  <br />  <br /> temperature resulted in loss of Kex2p activity, whereas <br /> no such loss was apparent in wild-type cells. <br />    Another way to study endosome&#226;&#8364;&#8220;Golgi traffic is to <br /> examine the distribution of a plasma membrane protein <br /> that normally recycles by this route (Holthuis et al., <br /> 1998a). One example is provided by a GFP-tagged version <br /> of Snc1p, a v-SNARE that mediates vesicle fusion with <br /> the plasma membrane. This is found mostly on the cell                   Fig. 8. Distribution of GFP&#226;&#8364;&#8220;Snc1p in got1 and sft2 mutants. Confocal <br /> surface, but is continually undergoing endocytosis; when                images of unfixed cells grown at 25&#194;&#176;C or incubated for 4 h at 37&#194;&#176;C <br /> retrieval from endosomes is blocked, for example in a                   are shown. (A) Wild-type, 25&#194;&#176;C; (B) got1&#226;&#710;&#8224;, 25&#194;&#176;C; (C) sft2&#226;&#710;&#8224;, 25&#194;&#176;C; <br />                                                                         (D) wild-type, 37&#194;&#176;C; (E) got1&#226;&#710;&#8224;, 37&#194;&#176;C; (F) sft2&#226;&#710;&#8224;, 37&#194;&#176;C; (G) sft2&#226;&#710;&#8224; <br /> tlg1 mutant, GFP&#226;&#8364;&#8220;Snc1p accumulates in late endosomes                    got1-2, 37&#194;&#176;C; (H) got1&#226;&#710;&#8224; sft2-1, 37&#194;&#176;C. Note the absence of plasma <br /> (op. cit. in Holthuis et al., 1998a; M.Lewis and H.Pelham,              membrane staining in (G) and (H). <br /> unpublished observations). We therefore examined the <br /> distribution of GFP&#226;&#8364;&#8220;Snc1p in sft2 and got1 mutants, to <br /> see whether it had a similar fate in these strains.                     membranes, allowing the alpha factor precursor to acquire <br />    GFP&#226;&#8364;&#8220;Snc1p was expressed in five different strains:                   Golgi-specific carbohydrate modifications. This assay has <br /> wild-type, got1&#226;&#710;&#8224;, sft2&#226;&#710;&#8224;, got1&#226;&#710;&#8224; sft2-1 and sft2&#226;&#710;&#8224; got1-2. In              been characterized in detail, and steps corresponding to <br /> the parental strain and the single got1&#226;&#710;&#8224; and sft2&#226;&#710;&#8224; mutants              the budding of vesicles (dependent on added COPII coat <br /> GFP&#226;&#8364;&#8220;Snc1p was found primarily at the plasma membrane                    components), their initial tethering to Golgi acceptor <br /> at 25&#194;&#176;C, though some could be observed in internal                      membranes (which requires addition of Uso1p) and sub- <br /> structures as well (Figure 8A&#226;&#8364;&#8220;C). Incubating these three                sequent SNARE-dependent fusion can each be followed <br /> strains at 37&#194;&#176;C increased the amount of internal GFP&#226;&#8364;&#8220;                   (Barlowe, 1997; Cao et al., 1998). As shown in Figure 9, <br /> Snc1p, but it remained clearly visible at the cell surface.             sft2&#226;&#710;&#8224; cells showed no defect in this assay&#226;&#8364;&#8221;in fact, they <br /> Strikingly, however, both the sft2&#226;&#710;&#8224; got1-2 and got1&#226;&#710;&#8224; sft2-1             consistently showed a higher efficiency of in vitro transport <br /> strains showed no GFP&#226;&#8364;&#8220;Snc1p at the cell surface at 37&#194;&#176;C;                than wild-type cells. However, got1&#226;&#710;&#8224; cells showed a very <br /> instead it was present in punctate structures and in an                 substantial reduction in transport efficiency (Figure 9A). <br /> intracellular haze which may correspond to small vesicles.              Further investigation showed that vesicle budding, <br /> As shown above, these strains are not defective in secre-               measured by the appearance of labelled pro-alpha factor <br /> tion, and we confirmed that a mutant form of GFP&#226;&#8364;&#8220;Snc1p                  in slowly sedimenting structures, was not affected, nor <br /> that is defective for endocytosis could reach the surface               was the initial Uso1p-dependent tethering of these vesicles <br /> at 37&#194;&#176;C (M.Lewis and S.Conchon, unpublished observ-                     to membranes (Figure 9B). This provides direct evidence <br /> ations). We conclude that the sft2 got1 mutants can                     that Got1p is involved in vesicular transport, and suggests <br /> endocytose the normal form of GFP&#226;&#8364;&#8220;Snc1p, but fail to                    that it acts at or close to the Sed5p-dependent fusion step <br /> retrieve it from the endocytic pathway to the Golgi.                    which follows tethering (Cao et al., 1998). <br />                                                                            In a further experiment, the membrane in which Got1p <br /> Role of Got1p in ER&#226;&#8364;&#8220;Golgi transport                                     acts was defined more precisely. ER-derived vesicles were <br /> The most dramatic phenotype that we observed in vivo                    prepared in vitro from both wild-type and got1&#226;&#710;&#8224; cells, <br /> was the block to ER&#226;&#8364;&#8220;Golgi transport when both Sft2p and                 and mixed with Golgi acceptor membranes which were <br /> Got1p were absent. To examine this defect directly, we                  similarly either wild-type or mutant. The results <br /> made use of an in vitro assay in which ER-derived vesicles              (Figure 10) show that vesicles lacking Got1p could fuse <br /> containing radiolabelled pro-alpha factor fused with Golgi              well with a wild-type Golgi acceptor, but that Golgi <br />  <br />                                                                                                                                       3941 <br />  S.Conchon et al. <br />  <br />  <br />  <br />  <br />                                                                            Fig. 10. Got1p function is required on acceptor membranes and not on <br />                                                                            ER-derived vesicles. COPII vesicles containing [35S]gp-&#206;&#177;-factor were <br />                                                                            isolated from wild-type and got1&#226;&#710;&#8224; semi-intact cell membranes. A <br />                                                                            second stage incubation was then performed mixing vesicles with <br />                                                                            acceptor membranes prepared from wild-type or got1&#226;&#710;&#8224; cells in the <br />                                                                            absence (open bars) or presence (solid bars) of purified fusion factors <br />                                                                            (Uso1p and LMA1). The percentage transport reflects the amount of <br />                                                                            outer-chain modified [35S]gp-&#206;&#177;-factor divided by the total amount of <br />                                                                            [35S]gp-&#206;&#177;-factor. In this experiment, maximal transport represents <br />                                                                            850 c.p.m. <br />  <br />  <br />  <br />                                                                            their in vitro transport activity at this temperature. Though <br />                                                                            modest, this difference in temperature sensitivity between <br />                                                                            mutant and wild-type was observed reproducibly. It strongly <br />                                                                            suggests that Got1p is directly involved in the ER&#226;&#8364;&#8220;Golgi <br />                                                                            transport step itself. <br /> Fig. 9. Influence of got1&#226;&#710;&#8224; and sft2&#226;&#710;&#8224; mutations on ER&#226;&#8364;&#8220;Golgi transport. <br /> (A) Overall transport of [35S]gp-&#206;&#177;-factor to the Golgi complex in <br /> semi-intact cell membranes (SIC) prepared from wild-type, got1&#226;&#710;&#8224; or         Discussion <br /> sft2&#226;&#710;&#8224; strains. Membranes were incubated alone (open bars) or with the <br /> reconstitution proteins (solid bars) consisting of COPII, Uso1p and        Sft2p was originally identified because its overexpression <br /> LMA1. The percentage transport was quantified after precipitation of       can compensate for the temperature-sensitive growth <br /> outer-chain modified forms of [35S]gp-&#206;&#177;-factor. In this experiment, <br /> maximal transport represents ~4000 c.p.m. Note that got1&#226;&#710;&#8224; membranes <br />                                                                            phenotype of several sed5 alleles (Banfield et al., 1995). <br /> display a defect in overall transport. (B) Vesicle budding and tethering   This property suggested an involvement of Sft2p in <br /> in wild-type and mutant strains. The level of freely diffusible vesicles   SNARE-dependent membrane traffic, but because deletion <br /> in cell-free reactions containing wild-type, got1&#226;&#710;&#8224; or sft2&#226;&#710;&#8224; semi-intact    of SFT2 has no obvious phenotype its role has been <br /> cell membranes alone (open bars) or in the presence of COPII proteins      unclear. In the present study we have identified mutations <br /> (solid bars) or COPII proteins plus the tethering factor Uso1p (hatched <br /> bars).                                                                     which make Sft2p essential for growth. One of these is <br />                                                                            in sed5, underlining the connection between Sft2p and <br />                                                                            Sed5p, but we also found that deletion of a novel gene <br /> membranes from got1&#226;&#710;&#8224; cells were severely impaired in                       GOT1, whilst not itself affecting growth, made cells <br /> their ability to fuse with vesicles from wild-type cells.                  dependent on Sft2p. This suggests that Got1p and Sft2p <br /> Hence, Got1p appears to function, in this assay at least,                  might have related functions, and we have sought to <br /> in Golgi membranes.                                                        determine what the role of Got1p might be. The results <br />    Though got1&#226;&#710;&#8224; cells grew well, we were concerned that                    are likely to be of general significance since both proteins <br /> the continuous absence of Got1p might have indirect effects,               have homologues in other species and human Got1p will <br /> such as a reduction in the levels of Golgi glycosyltransfer-               even function in yeast. <br /> ases, that could influence the in vitro assay in a trivial way.               If Got1p and Sft2p share similar functions, one might <br /> To address this, we repeated the assay using the temperature-              expect to see some similarity between the two proteins. <br /> sensitive sft2&#226;&#710;&#8224; got1-2 strain. This strain showed little defect            Indeed, although they show little direct sequence similarity, <br /> in ER&#226;&#8364;&#8220;Golgi traffic in vivo, even at 37&#194;&#176;C, but we reasoned                 both appear to be tetra-spanning membrane proteins with <br /> that the mutant Got1p might show at least some temperature                 the same orientation, and the transmembrane domains are <br /> sensitivity in vitro. Indeed, while the transport efficiency of            contained in regions of similar length (~120 residues). In <br /> this strain was high at 23&#194;&#176;C, it was halved when the assay                 both proteins it is the membrane-associated region that <br /> was performed at 29&#194;&#176;C (Golgi modification falling from                     seems to be important: it is the most evolutionarily <br /> 32.9 to 16.9%). In contrast, wild-type cells retained 80% of               conserved, and a single Gly&#226;&#8224;&#8217;Ser mutation within one of <br />  <br /> 3942 <br />                                                                                      Got1p and Sft2p Golgi transport proteins <br />  <br />  <br /> the predicted transmembrane domains of Got1p is suffi-          Role of Sft2p <br /> cient to abolish function.                                      In contrast to Got1p, Sft2p does not co-localize extensively <br />                                                                 with Sed5p, and sft2&#226;&#710;&#8224; cells do not show a defect in ER&#226;&#8364;&#8220;Golgi <br />                                                                 transport in vitro. Nevertheless, Sft2p has been conserved <br /> A Golgi membrane protein required for transport <br />                                                                 during evolution. It follows that Sft2p is likely to have an <br /> from the ER                                                     important function that is distinct from that of Got1p. A <br /> Both Got1p and Sft2p are located predominantly in the           clue comes from a second kind of defect exhibited in vivo <br /> Golgi apparatus, but whereas Got1p shows substantial co-        by cells deficient for both Sft2p and Got1p activity. With <br /> localization with Sed5p in early cisternae, Sft2p is found      the temperature-sensitive alleles in particular, the pheno- <br /> mainly in later cisternae (Wooding and Pelham, 1998).           types are strikingly similar to those previously characterized <br /> In vivo, removal of both proteins has a profound inhibitory     in mutants lacking the late Golgi/endosomal SNAREs Tlg1p <br /> effect on ER&#226;&#8364;&#8220;Golgi transport, as demonstrated by an             and Tlg2p (which like Sed5p are members of the syntaxin <br /> accumulation of ER membranes and a block to the                 family). Like the got1 sft2 double mutant, tlg1&#226;&#710;&#8224; cells show <br /> maturation of p1 CPY. Careful examination of the data           reduced size of invertase precursors, a mild CPY sorting <br /> reveals that a modest effect on CPY transport can be            defect, loss of Kex2p and a failure to recycle GFP&#226;&#8364;&#8220;Snc1p <br /> observed even when Sft2p is still present, for example in       from endosomes to the cell surface (Holthuis et al., 1998a,b). <br /> a got1&#226;&#710;&#8224; GAL&#226;&#8364;&#8220;SFT2 strain grown on galactose, or in a             All these effects can be explained by a defect in the retrieval <br /> got1&#226;&#710;&#8224; sft2ts strain at the permissive temperature (Figure 6).   of proteins from early endosomes to the Golgi. <br /> This effect is not seen when residual Got1p activity               A role for Sft2p (and Got1p) in this step could explain <br /> remains, for example in a sft2&#226;&#710;&#8224; got1ts strain.                  another apparent mystery, namely that a defect in ER&#226;&#8364;&#8220; <br />    A more dramatic and specific effect can be observed          Golgi transport does not seem to account for growth <br /> in vitro: even in the presence of Sft2p there is a severe       inhibition in the temperature-sensitive got1 and sft2 <br /> impairment in the ability of ER-derived vesicles to fuse with   mutants. It seems likely that the ts mutations do not <br /> Golgi membranes lacking Got1p, though budding from the          completely inactivate the proteins; it is notoriously difficult <br /> ER is unaffected. Furthermore, a ts allele of got1 shows        to obtain such alleles for hydrophobic proteins buried in <br /> temperature-sensitive transport in vitro. The stronger          membranes. Low residual activity could be sufficient to <br /> phenotype of got1&#226;&#710;&#8224; mutations in vitro than in vivo is not       sustain ER&#226;&#8364;&#8220;Golgi transport, but inefficient endosome&#226;&#8364;&#8220; <br /> without precedent: for example, the LMA1 proteins are           Golgi transport may eventually block growth. This is <br /> needed for ER&#226;&#8364;&#8220;Golgi traffic in vitro (Barlowe, 1997), but       plausible since other mutations that affect such traffic, <br /> are not essential in vivo. Taken as a whole, the evidence is    such as tlg1, also inhibit growth at high temperature <br /> consistent with an important role for Got1p in ER&#226;&#8364;&#8220;Golgi         without affecting secretion (Holthuis et al., 1998b). <br /> transport, albeit one that can be partially bypassed in vivo.      Though their functions are clearly not identical, the <br />    What is this role? Two-stage transport assays indicate       genetic interactions between GOT1 and SFT2 are striking. <br /> that Golgi membranes from a got1&#226;&#710;&#8224; strain are defective,         It is not simply that defects in late Golgi traffic are <br /> whereas ER-derived vesicles isolated from this same strain      intrinsically deleterious to a got1 mutant. Since direct <br /> are fully functional, implying that Got1p acts on the           testing of one such mutant, tlg2, showed that it was not <br /> acceptor membranes. Similarly, in vitro experiments with        lethal in combination with got1. In the light of all the <br /> a temperature-sensitive sed5 strain indicate a requirement      data, we suggest the following hypothesis: that Sft2p <br /> in this assay for Sed5p function on acceptor membranes          performs a function in vesicular transport analogous to <br /> but not on transport vesicles (X.Cao and C.Barlowe,             that of Got1p, but acts primarily in later Golgi compart- <br /> unpublished observations). Like Sed5p, Got1p is required        ments. Its role would be to facilitate fusion of endosome- <br /> for a step that follows vesicle tethering. Together with the    derived vesicles with the Golgi, perhaps in combination <br />                                                                 with one or both of the late Golgi syntaxins, Tlg1p and <br /> genetic interactions, these results suggest that the function <br />                                                                 Tlg2p. The redundancy of Sft2p and Got1p functions <br /> of Got1p is closely connected to that of Sed5p, which in <br />                                                                 in vivo could then be simply explained. Their distributions <br /> turn is intimately concerned with membrane fusion. <br />                                                                 are likely to overlap&#226;&#8364;&#8221;Sft2p at least recycles through early <br />    How exactly Got1p contributes to vesicle fusion remains      Golgi compartments (Wooding and Pelham 1998)&#226;&#8364;&#8221;and <br /> a question for the future. Several possibilities can be         each protein may thus be able to act throughout the Golgi <br /> imagined, but the overall structure of the protein is           and partially substitute for the other. When the activity of <br /> reminiscent of that of an ion channel subunit and we have       Sed5p is limiting they evidently co-operate to sustain ER&#226;&#8364;&#8220; <br /> preliminary evidence that it is oligomeric, which would         Golgi traffic, explaining why sft2 shows synthetic lethality <br /> be consistent with this. Interestingly, ER&#226;&#8364;&#8220;Golgi transport      with sed5. Functional redundancy in endosome&#226;&#8364;&#8220;Golgi <br /> is known to involve a Ca21-dependent step that follows          transport could arise either by direct substitution of Got1p <br /> vesicle docking (Rexach and Schekman, 1991) and it has          for Sft2p, or because the traffic routes are flexible, vesicles <br /> recently been reported that vacuolar fusion, which is also      being able to fuse with either late or early Golgi cisternae <br /> Ca21 dependent, is accompanied by the release of calcium        (Fischer von Mollard et al., 1997; Holthuis et al., 1998b; <br /> ions from the organelles at the time of fusion (Peters and      Nichols and Pelham, 1998; Nichols et al., 1998). <br /> Mayer, 1998). It has also been suggested, based on studies         Just how similar the roles of Sft2p and Got1p are will <br /> with neuroendocrine cells, that calcium ions facilitate         only become apparent when their biochemical properties <br /> SNARE complex formation (Chen et al., 1999). By                 are better understood. For Got1p, at least, the in vitro assay <br /> analogy, it is conceivable that Ca21 is released from Golgi     will allow detailed analysis of its key features, its inter- <br /> membranes, that this aids Sed5p function and that Got1p         actions with other proteins and its effects on vesicular <br /> is somehow involved in this process.                            transport. <br />  <br />                                                                                                                         3943 <br />  S.Conchon et al. <br />  <br />  <br /> Table I. Strains used in this study <br />  <br /> Strain No. Genotype <br />  <br /> IAY11       MAT&#206;&#177; ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 ade2-101 ade3-&#226;&#710;&#8224;853 <br /> IAY25       MATa ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 ade2-101 ade3-&#226;&#710;&#8224;853 <br /> SCY11       MAT&#206;&#177; sft2&#226;&#710;&#8224;::HIS3 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 ade2-101 ade3-&#226;&#710;&#8224;853 containing pADE3SFT2 (CEN6, URA3, ADE3, SFT2) <br /> SCY25       MATa sft2&#226;&#710;&#8224;::HIS3 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 ade2-101 ade3-&#226;&#710;&#8224;853 containing pADE3SFT2 (CEN6, URA3, ADE3, SFT2) <br /> SEY6210     MAT&#206;&#177; ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 lys2-801 <br /> SEY6211     MATa ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY01       MAT&#206;&#177; sft2&#226;&#710;&#8224;::HIS3 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 lys2-801 <br /> SCY02       MATa got1&#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY03       MAT&#206;&#177; sft2&#226;&#710;&#8224;::HIS3 got1&#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 containing p316GOT1 (CEN6, URA3, GOT1). <br /> SCY04       MAT&#206;&#177; LEU2::sft2-1 sft2&#226;&#710;&#8224;::HIS3 got1&#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY05       MAT&#206;&#177; LEU2::got1-2 sft2&#226;&#710;&#8224;::HIS3 got1&#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY07       MAT&#206;&#177; LEU2::GFP&#226;&#8364;&#8220;SFT2 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 lys2-801 containing p316GOT3m (CEN6, URA3, GOT1-3myc) <br /> SCY08       MAT&#206;&#177; URA3::GFP&#226;&#8364;&#8220;SNC1 sft2&#226;&#710;&#8224;::HIS3 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 lys2-801 <br /> SCY09       MATa URA3::GFP&#226;&#8364;&#8220;SNC1 got1&#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY12       MAT&#206;&#177; URA3::GFP&#226;&#8364;&#8220;SNC1 LEU2::sft2-1 sft2&#226;&#710;&#8224;::HIS3 got1 &#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br /> SCY13       MAT&#206;&#177; URA3::GFP&#226;&#8364;&#8220;SNC1 LEU2::got1-2 sft2&#226;&#710;&#8224;::HIS3 got1 &#226;&#710;&#8224;::TRP1 ura3-52 his3-&#226;&#710;&#8224;200 leu2-3, &#226;&#8364;&#8220;112 trp1-&#226;&#710;&#8224;901 suc2-&#226;&#710;&#8224;9 ade2-101 <br />  <br />  <br /> Materials and methods                                                         with a TRP1/CEN-based yeast genomic library and three strains gave <br />                                                                               transformants that sectored and carried plasmids that did not contain <br /> Plasmids                                                                      SFT2. The complementing genes were identified by sequencing the ends <br /> Constructs containing the SFT2 and GOT1 open reading frames were              of the inserts, testing smaller restriction fragments and finally by <br /> generated by PCR amplification of yeast genomic DNA using Pfu                 expression of individual ORFs prepared by PCR. <br /> polymerase (Stratagene) and subsequent ligation into the vectors pRS315, <br /> pRS316 or pRS414 (all CEN-based, LEU2, URA3 or TRP1, respectively,            Generation of ts alleles of SFT2 and GOT1 <br /> Sikorski and Hieter, 1989), behind the TPI or the GAL1 inducible              The plasmids p315SFT2 and p315GOT1 were used as templates for <br /> promoter. They were designated by vector and gene name (e.g.                  PCR-mediated mutagenesis and gap repair as described by Muhlard <br /> p315SFT2), the promoter region being TPI unless otherwise stated. The         et al. (1992). The PCR products and gapped plasmid (pRS315) were <br /> plasmid used for the synthetic lethal screen (pADE3SFT2) was created          cotransformed into SCY03. Following selection on 5-FOA (to remove <br /> by inserting the SFT2 sequence behind the TPI promoter into a                 the wild-type GOT1 plasmid), colonies were selected that were viable <br /> pRS316 derivative carrying ADE3 (kindly provided by I.Adams, MRC,             at 25&#194;&#176;C but not at 37&#194;&#176;C. The corresponding plasmids were rescued <br /> Cambridge). C-terminally myc-tagged GOT1 was expressed from the               (Ward, 1990), the ts alleles subcloned into a pRS405 vector behind a <br /> TPI promoter in a pRS414 or pRS316 vector, the terminal sequence              TPI promoter, and integrated at LEU2 in SCY03. The rescuing plasmid <br /> being changed from RVLPV* to RVLPVFEMEQKLISEEDLNSR-                           p316GOT1 was then eliminated on 5-FOA plates. One ts allele of each <br /> EQKLISEEDLMEQKLISEEDLN*, where the myc epitope is underlined.                 gene (strains SCY04 and SCY05) was further characterized. <br /> The GFP&#226;&#8364;&#8220;SFT2 and myc-tagged invertase constructs have been described <br /> previously (Banfield et al., 1995; Wooding and Pelham, 1998). The             Antibodies and immunoblotting <br /> pRS406 vector which contains the GFP&#226;&#8364;&#8220;SNC1 construct was a generous            Rabbit polyclonal antibodies to Sed5p, CPY, Tlg1p, Tlg2p, Pep12p and <br /> gift from M.Lewis and will be described elsewhere.                            Vam3p were described previously (Hardwick and Pelham, 1992; Banfield <br />    The cDNA clones 0428529 and 0267460 which contain the GOT1                 et al., 1995; Nichols et al., 1997; Holthuis et al., 1998b). The human <br /> human homologues were obtained from the I.M.A.G.E. Consortium. The            c-myc epitope was detected with mouse mAb 9E10 or with rabbit <br /> inserts were cloned by PCR and sequenced. The myc-tagged version of           polyclonal antibody (Santa Cruz). Monoclonal antibodies used in COS <br /> hGot1a (tagged as described above for GOT1) was inserted into a SMH5-         cells were: 23C, anti-&#206;&#178;9-COP (Harrison-Lavoie et al., 1993); 100/3, anti- <br /> derived vector (Levine and Munro, 1998) for transfection into COS cells.      adaptin (Ahle et al., 1988) and anti-TGN46 (Prescott et al., 1997). For <br />                                                                               immunoblotting, all antibody incubations were carried out in phosphate- <br /> Yeast strains and growth conditions                                           buffered saline (PBS) containing 5% dried milk and 0.5% Tween-20. <br /> Yeast strains are described in Table I. These were constructed by standard    After incubation with peroxidase-conjugated secondary antibodies <br /> methods and grown in yeast extract&#226;&#8364;&#8220;peptone&#226;&#8364;&#8220;dextrose or synthetic              (Bio-Rad), detection was performed using enhanced chemiluminescence <br /> dextrose (SD) medium as appropriate. Standard yeast techniques for            (ECL kit, Amersham). <br /> sporulation, tetrad analysis and gene disruption were employed as <br /> described in Guthrie and Fink (1991). Yeast transformations were carried      Confocal microscopy <br /> out as described by Elble (1992).                                             Indirect immunofluorescence was performed as described previously <br />    IAY11 and IAY25 were kindly provided by I.Adams and J.Kilmartin.           (Hardwick and Pelham, 1992). Both primary and secondary antibody <br /> SCY11, SCY25 and SCY01 were prepared from IAY11, IAY25 and                    incubation were carried out in PBS supplemented with 2% dried milk <br /> SEY6210, respectively, by replacing the entire coding region of SFT2          for 2 h at room temperature. Primary antibody to Sed5p was used at a <br /> with HIS3. For GOT1 disruption the entire coding sequence was replaced        dilution of 1:2000. Fluorescein or Cy3-conjugated secondary antibodies <br /> with TRP1 in SEY6210/11 diploid cells which were then sporulated to           (Amersham) were used for visualization. Dual images were obtained <br /> yield SCY02. This was transformed with p316GOT1, crossed with                 with an MRC-600 confocal laser scanning microscope (Bio-Rad) using <br /> SCY01 cells and sporulated to give SCY03. This strain was unable to           separate excitation at the appropriate wavelengths to avoid bleed-through. <br /> grow on 5-fluoro-orotic acid (5-FOA) plates (to counter-select the               Cells expressing GFP&#226;&#8364;&#8220;Snc1p fusion protein were grown in SD medium <br /> plasmid). It was used to test the functionality of proteins such as Got1p-3   at 25&#194;&#176;C to early log phase (0.3 OD600 per ml) and then split into two, <br /> myc, by transformation of the corresponding plasmid and analysis of           and further incubated for 3&#226;&#8364;&#8220;4 h at 25 or 37&#194;&#176;C. They were placed on to <br /> the ability to survive on 5-FOA.                                              a slide, a coverslip was added and they were immediately examined <br />                                                                               under the MRC-600 confocal microscope. <br /> Synthetic lethality screen <br /> Mutants synthetically lethal with sft2 were isolated using the ade2/ade3,     COS cells transfection and protein localization <br /> red/white sectoring system (Kranz and Holm, 1990). The SCY11 and              COS cells were transfected using DEAE&#226;&#8364;&#8220;dextran as described previously <br /> SCY25 strains were red but give white sectors under non-selective             (Munro and Pelham, 1987). After 24 h, cells were split on to eight-well <br /> conditions due to plasmid loss. They were mutagenized with ultraviolet        slides [C.A.Hendley (Essex) Ltd] and on the following day, were fixed <br /> light and 20 non-sectoring colonies identified among 100 000 screened.        in 2% formaldehyde, 0.1% glutaraldehyde in PBS, permeabilized in <br /> Five of them regained their sectoring ability after transformation with a     0.5% Triton X-100 in PBS and immunofluorescence performed as <br /> LEU2-based vector carrying SFT2 (p315SFT2). They were transformed             described by Munro (1995). For freeze&#226;&#8364;&#8220;thaw permeabilization, cells on <br />  <br />  <br /> 3944 <br />                                                                                                      Got1p and Sft2p Golgi transport proteins <br />  <br />  <br /> polylysine-coated slides were washed once in 25 mM HEPES, 125 mM            <br /> KOAc, 5.5 mM glucose, 2.5 mM MgOAc, pH 7, then frozen on dry <br /> ice. The cells were quickly thawed by adding the same buffer, then fixed     <br /> as above.                                                              <br /> Electron microscopy <br />                                                                       <br /> Cells were grown in SD medium and harvested at a density of ~1 OD600 <br />                                                                              <br /> per ml. Permanganate fixation, dehydration and embedding in Spurr&#226;&#8364;&#8482;s <br />                                                                            <br /> resin (Agar Scientific, Stansted, UK) were carried out as described by <br />                                                                        <br /> Kaiser and Schekman (1990). Sections were stained with 5% uranyl <br /> acetate for 10 min at 60&#194;&#176;C followed by 5 min in Reynold&#226;&#8364;&#8482;s lead citrate <br /> at room temperature. Sections were observed using a Philips CM10 <br /> transmission electron microscope.                                         <br /> Membrane association and protease protection assays                          <br /> A p100 fraction was prepared and resuspended as described by Holthuis        <br /> et al. (1998b). Aliquots were incubated in buffer alone or with 0.5 M          <br /> NaCl or 0.1 M Na2CO3 (pH 11) or 1% Triton X-100 for 20 min at 4&#194;&#176;C.        <br /> Samples were centrifuged at 100 000 g for 45 min and equivalent              <br /> amounts of supernatant and pellet fractions analysed by immunoblotting.    <br /> For protease treatment, a 13 000 g supernatant (Holthuis et al., 1998b)      <br /> was split into aliquots that were incubated with or without proteinase K     <br /> (5 &#194;&#181;g/ml) and with or without Triton X-100 (2%) for 20 min at room        <br /> temperature. Proteinase K was inhibited by adding PMSF to a final            <br /> concentration of 10 mM before gel analysis. Binding of anti-myc mAb            <br /> (9E10) to the membrane fraction was determined by centrifugation at          <br /> 100 000 g through a sucrose cushion, bound antibody being detected by     <br /> immunoblotting.                                             <br /> Subcellular fractionation and Kex2p activity measurement                    <br /> Preparation of the samples to be loaded on the sucrose gradients was          <br /> carried out as described by Holthuis et al. (1998b) except that the           <br /> 100 000 g membrane pellet was resuspended in a final volume of 1 ml           <br /> of buffer (50 mM Tris&#226;&#8364;&#8220;HCl pH 7.5, 200 mM sorbitol, 1 mM EDTA)                <br /> that was loaded on to a 22&#226;&#8364;&#8220;60% sucrose step gradient which was                <br /> generated using the following steps [all sucrose solutions were made         <br /> (w/w, %) with ultra-pure sucrose (Gibco-BRL)]: 1.5 ml 60%, 1.0 ml          <br /> 40%, 1.0 ml 37%, 1.5 ml 34%, 2 ml 32%, 2 ml 29%, 1.5 ml 27% and          <br /> 1.0 ml 22%. After 17&#226;&#8364;&#8220;18 h centrifugation in a Beckman SW40Ti rotor            <br /> at 170 000 g, 16 fractions of 0.78 ml were collected. Kex2p endoprotease   <br /> activity was determined in each of these fractions (Cunningham and            <br /> Wickner, 1989) and in membrane pellets prepared from 100 000 g          <br /> centrifugation following a comparable procedure.                   <br /> Analysis of CPY and invertase processing                                     <br /> Pulse&#226;&#8364;&#8220;chase analysis of carboxypeptidase Y was performed as described          <br /> by Stepp et al. (1997), although the incubation temperatures were either       <br /> 25 or 37&#194;&#176;C. Intracellular invertase was analysed as described by Holthuis     <br /> et al. (1998b).                                                  <br /> In vitro transport assays                                                   <br /> Yeast semi-intact cells from wild-type, got1&#226;&#710;&#8224; or sft2&#226;&#710;&#8224; strains were           <br /> prepared from log phase cultures grown at 25&#194;&#176;C and stored frozen at           <br /> &#226;&#8364;&#8220;70&#194;&#176;C. Prior to assays, cells were quickly thawed, washed with buffer     <br /> to remove cytosol and [35S]prepro-&#206;&#177;-factor was post-translationally         <br /> translocated into ER membranes at 10&#194;&#176;C (Baker et al., 1988). Vesicle           <br /> budding, tethering and fusion assays were performed as described (Cao      <br /> et al., 1998). Briefly, the addition of purified COPII proteins buds           <br /> ER-derived vesicles from semi-intact cells that are freely diffusible and     <br /> remain in the supernatant fraction after centrifugation at 12 000 g.        <br /> Protease protected [35S]glyco-pro-(gp)-&#206;&#177;-factor contained in budded             <br /> vesicles was quantified after solubilization and precipitation with con-      <br /> canavalin A&#226;&#8364;&#8220;Sepharose. For transport assays, COPII proteins in addition       <br /> to purified Uso1p and LMA1 were added as indicated in figure legends           <br /> and the amount of Golgi-modified [35S]gp-&#206;&#177;-factor was measured by            <br /> immunoprecipitation with &#206;&#177;1,6-mannose specific antibodies. For two-            <br /> stage vesicle fusion experiments, COPII vesicles that contained [35S]gp-       <br /> &#206;&#177;-factor were generated from wild-type and got1&#226;&#710;&#8224; membranes under              <br /> standard budding conditions (Cao et al., 1998). Freely diffusible vesicles      <br /> (10 &#194;&#181;l) were mixed with washed semi-intact cells from indicated strains        <br /> in a total volume of 25 &#194;&#181;l. The same membrane, protein and nucleotide        <br /> concentrations as in one-stage transport reactions were maintained in           <br /> these reactions. The extent of fusion with Golgi membranes was measured       <br /> after incubation at 25&#194;&#176;C by precipitation of Golgi modified [35S]gp-           <br /> &#206;&#177;-factor. The data plotted in these experiments are the average of          <br /> duplicate determinations and the error bars represent the range.               <br /> </body></html>