<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2002 Genetics Society America <br />  <br />  <br />  <br />         Regulation Capsule Synthesis Cell Motility Salmonella enterica <br />                              Essential Gene igaA <br />  <br />                          David A. Cano,*,1 Gustavo Dom&#196;&#177;&#194;&#180;nguez-Bernal,&#226;&#8364;&#160; Alberto Tierrez,&#226;&#8364;&#160; <br />                               Francisco Garcia-del Portillo&#226;&#8364;&#160; Josep Casadesu&#194;&#180;s*,2 <br />          *Departamento Gene&#194;&#180;tica, Universidad Sevilla, Seville 41012, Spain &#226;&#8364;&#160;Departamento Biotecnolog&#196;&#177;&#194;&#180;a Microbiana, <br />                         Centro Nacional Biotecnolog&#196;&#177;&#194;&#180;a, CSIC, Campus Cantoblanco, Madrid 29049, Spain <br />                                                       Manuscript received April 29, 2002 <br />                                                   Accepted publication September 3, 2002 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Mutants Salmonella enterica carrying igaA1 allele, selected able overgrow fibroblast <br />              cells culture, mucoid reduced motility. Mucoidy caused derepression wca genes <br />              (necessary capsule synthesis); genes regulated RcsC/YojN/RcsB phosphorelay system <br />              RcsA coregulator. The induction wca expression igaA1 mutant suppressed by <br />              mutations rcsA rcsC. Reduced motility caused lowered expression flagellar master operon, <br />              flhDC, suppressed mutations rcsB rcsC, suggesting mutations igaA gene reduce <br />              motility activating RcsB/C system. A null igaA allele maintained igaA&#226;&#171;&#185;/igaA <br />              merodiploid, indicating igaA essential gene. Lethality suppressed mutations rcsB, rcsC, <br />              yojN, rcsA, suggesting viability defect igaA null mutant mediated the <br />              RcsB/RcsC system, independently RcsA (and wca genes). Because defects <br />              associated igaA mutations suppressed mutations block RcsB/RcsC system, propose <br />              functional interaction igaA gene product Rcs regulatory network of <br />              regulated products. <br />  <br />  <br />  <br />  <br /> A    screen mutants able overgrow fibro- <br />        blast cell line identified functions Salmonella <br /> enterica involved growth restraint nonphago- <br />                                                                            swarming motility (Dufour et al. 1998). Current <br />                                                                            evidence suggests umoB encodes membrane pro- <br />                                                                            tein regulates genes involved flagellar synthesis <br /> cytic eukaryotic cells (Cano et al. 2001). One mu-                 (Dufour et al. 1998). <br /> tants carried <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> hitherto unknown                         Below igaA gene S. enterica is <br /> locus, located near mcrA centisome 75. This novel                       essential laboratory conditions. We provide <br /> locus called igaA (intracellular growth attenuator;                    evidence igaA gene product S. enterica a <br /> EMBL accession no. AJ301649). The study showed                        pleiotropic regulator exerts positive control the <br /> that IgaA&#226;&#171;&#186; mutants S. enterica mucoid aviru-                    flagellar master operon flhDC negative control on <br /> lent murine typhoid model (Cano et al. 2001).                       colanic acid cluster wca. Genetic analysis unveils <br />    Analysis silico igaA DNA sequence indicated                   functional relationship igaA gene product <br /> the existence open reading frame (ORF) homolo-                       two-component regulatory RcsB-RcsC. <br /> gous yrfF ORF Escherichia coli. The yrfF                     Originally described regulator capsule synthesis <br /> igaA ORFs located equivalent position                     E. coli (Gottesman et al. 1985), RcsB-RcsC system <br /> chromosomal gene maps E. coli and S. enterica                       shown participate variety cellular <br /> serovar Typhimurium (Blattner et al. 1997; McClel-                         processes, include cell division control E. coli <br /> land et al. 2001). A BLAST databank search (Altschul                       (Carballe`s et al. 1999), regulation Vi antigen synthe- <br /> et al. 1997) indicated igaA yrfF loci                  sis Salmonella typhi (Virlogeux et al. 1996), synthesis <br /> turn homologous umoB gene Proteus mirabilis                      flagellin invasion proteins S. typhi (Arricau <br /> (Dufour et al. 1998). The umoB gene identified                    et al. 1998), expression E. coli tolQRA operon <br /> multicopy suppressor swarming defect associated                     (Clavel et al. 1996), synthesis E. coli outer mem- <br /> with flgN mutations proved required                     brane protein OsmC (Davalos-Garcia et al. 2001), re- <br />                                                                            sistance chlorpromazine-induced stress E. coli <br />                                                                            (Conter et al. 2002), production exopolysaccha- <br />    1 <br />     Present address: Diabetes Center, Department Medicine, Univer-      ride Erwinia amylovora (Bereswill Geider 1997). <br /> sity California, San Francisco, 513 Parnassus Ave., San Francisco,      The RcsB-RcsC main components, <br /> CA 94143-0540. <br />    2 <br />                                                                            sensor protein RcsC transcriptional activator <br />     Corresponding author: Departamento Gene&#194;&#180;tica, Facultad Bio- <br /> log&#196;&#177;&#194;&#180;a, Universidad Sevilla, Avenida Reina Mercedes 6, Sevilla 41012,   RcsB (Brill et al. 1988), includes second <br /> Spain. E-mail: casadesus@us.es                                             transcriptional activator, RcsA (Gottesman et al. 1985); <br /> Genetics 162: 1513&#226;&#8364;&#8220;1523 (December 2002) <br />  1514                                                     D. A. Cano et al. <br />  <br />                                                              TABLE 1 <br />                                             Strains S. enterica serovar Typhimurium <br />  <br /> Strain                                                                       Genotype <br /> MST1753a                       leuA414 hsdL (r&#226;&#171;&#186;m&#226;&#171;&#186;)Fels2&#226;&#171;&#186;/F&#226;&#172;&#732;114(ts) lac&#226;&#171;&#185; zzf-20::Tn10 zzf-3551::Mud P <br /> SV4215                         igaA1 <br /> SV4254                         igaA1 zhf-6311::Tn10d Tc <br /> SV4321                         DUP[argA9000*Mud P*cysG1573] <br /> SV4322                         DUP[argA9000*Mud P*cysG1573] igaA2::KIXX <br /> SV4324                         DUP4102[f liC5050*Mud A*f lhDC5213] zhf-6311::Tn10dTc <br /> SV4325                         DUP4102[f liC5050*Mud A*f lhDC5213] igaA1 zhf-6311::Tn10dTc <br /> SV4343                         igaA1 zhf-6311::Tn10d Tc rcsA51::MudQ <br /> SV4345                         igaA2::KIXX rcsC52::MudQ <br /> SV4357                         igaA1 zhf-6311::Tn10d Tc gmm-21::Mud J <br /> SV4379                         rcsA51::Mud P <br /> SV4380                         rcsC52::Mud P <br /> SV4402                         igaA1 zhf-6311::Tn10d Tc rcsB70::Tn10dCm <br /> SV4404                         igaA1 zhf-6311::Tn10d Tc rcsC52::MudQ <br /> SV4406                         rcsB70::Tn10dCm <br /> SV4407                         gmm-21::Mud J <br /> SV4411                         gmm-21::Mud J rcsB70::Tn10dCm <br /> SV4412                         igaA1 zhf-6311::Tn10d Tc gmm-21::Mud J rcsB70::Tn10dCm <br /> SV4443                         igaA1 zhf-6311::Tn10d Tc rcsA51::MudQ <br /> SV4450                         igaA1 <br /> SV4577                         SL1344/pNG1166 <br /> SV4578                         igaA2::KIXX/pNG1166 <br /> SV4614                         zhf-6311::Tn10dTc f lhDC5213::Mud A <br /> SV4615                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A <br /> SV4616                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A rcsA51::Mud P <br /> SV4617                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A wcaK21::Mud P <br /> SV4618                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A manB571::Mud P <br /> SV4619                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A wcaF22::Mud P <br /> SV4620                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A gmm::Mud P <br /> SV4622                         zhf-6311::Tn10d Tc f lhDC5213::Mud A rcsB70::Tn10dCm <br /> SV4623                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A rcsB70::Tn10dCm <br /> SV4624                         zhf-6311::Tn10d Tc f lhC5213::Mud A rcsC52::Mud P <br /> SV4625                         igaA1 zhf-6311::Tn10d Tc f lhDC5213::Mud A rcsC52::Mud P <br /> SV4629                         igaA2::KIXX rcsC52::Mud P/pNG1166 <br /> SV4630                         f liA6002::Mud A zhf-6311::Tn10dTc <br /> SV4631                         f liA6002::Mud A igaA1 zhf-6311::Tn10dTc <br /> TR5878b                        r(LT2)&#226;&#171;&#186; m(LT2)&#226;&#171;&#186; r(S)&#226;&#171;&#185; ilv-542 metA22 trpB2 Fels2&#226;&#171;&#186; f liA66 thrA120 xyl-404 metE551 hspL56 hspS29 <br /> TT10423b                       proAB47/F&#226;&#172;&#732;128 pro&#226;&#171;&#185; lac&#226;&#171;&#185; zzf-1836::Tn10dTc <br />   a <br />       LT2 derivative, obtained Stanley R. Maloy. <br />   b <br />       LT2 derivatives, obtained John R. Roth (University Utah, Salt Lake City). <br />  <br />  <br /> a recently described ancillary protein, YojN (Takeda et             Typhimurium. Unless indicated otherwise, strains derive <br /> al. 2001); additional sensors transmit external            mouse-virulent strain SL1344 (Hoiseth Stocker <br />                                                                     1981). Transductional crosses using phage P22 HT 105/1 <br /> signals membrane-bound RcsC sensor (Clarke                   int201 (Schmieger 1972; G. Roberts, unpublished results), <br /> et al. 1997; Kelley Georgopoulos 1997; Chen et                  henceforth called P22 HT, used strain construction <br /> al. 2001). We mutations affecting              operations involving chromosomal markers transfer <br /> rcsB, rcsC, yojN genes suppress defects associated          plasmids Salmonella strains. The transduction pro- <br /> with igaA mutations (lethality, mucoidy, reduced                tocol described Maloy (1990). <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> obtain phage- <br />                                                                     free isolates, transductants purified streaking green <br /> motility). Mucoidy suppressed rcsA mutations.            plates (Chan et al. 1972). Phage sensitivity tested cross- <br /> These observations provide genetic evidence IgaA               streaking clear-plaque mutant P22 H5. The alleles <br /> might novel signaling pathway involving                flhDC5213::Mud A fliA6002::Mud A originally har- <br /> the RcsB-RcsC system.                                               bored strains KK1107 KK1108, respectively (Kutsu- <br />                                                                     kake Iino 1994). Both strains derive LT2 were <br />                                                                     obtained Kazuhiro Kutsukake (Okayama University, <br />                 MATERIALS AND METHODS                               Okayama, Japan). Mud A Mud 1 derivative conditional <br />                                                                     transposition (Hughes Roth 1984). The duplication <br />   Bacterial strains, bacteriophages, strain construction:       DUP4102[fliC5050*Mud A*flhDC5213], originally carried on <br /> All S. enterica strains listed Table 1 belong serovar     LT2-derived strain (TH4313), gift Kelly T. Hughes <br />                                                  Regulation wca flhDC igaA                                               1515 <br />  <br /> (University Washington, Seattle). Strain MST1753, used           plasmid pNG1166, carries igaA gene the <br /> for transposon replacement, carries F-prime containing         control arabinose-dependent PBAD promoter (Guzman <br /> Mud P element (Youderian et al. 1988); strain ob-           et al. 1995). <br /> tained Stanley R. Maloy (University Illinois, Urbana,          Mutagenesis Mud J: We employed cis-complementa- <br /> IL). The allele rcsB70::Tn10dCm obtained Eduardo           tion procedure Hughes Roth (1988), a <br /> Groisman (Washington University School Medicine, St.             defective Mud J element cotransduced Mud 1 element <br /> Louis). This allele originally carried 14028s derivative   transiently provides transposition functions. Mud 1 the <br /> (strain EG12711). Allele numbers S. enterica mutants de-        specialized transducing phage Mud 1(Ap Lac cts62) constructed <br /> scribed study obtained Salmonella Ge-         Casadaban Cohen (1979). MudI1734[KmLac] (Cas- <br /> netic Stock Center (University Calgary, Calgary, Alberta,        tilho et al. 1984) transposition-deficient Mu derivative <br /> Canada). The E. coli recipient plasmids strain DH5&#226;?&#163;          generates operon fusions insertion; element <br /> (Yanisch-Perron et al. 1985). The E. coli host &#226;?&#178;-dependent      renamed Mud J Hughes Roth (1988). <br /> suicide plasmids S17 &#226;?&#173;pir (Simon et al. 1983).                     Mutagenesis Tn10d Tc: A lysate grown strain <br />    Media chemicals: The E medium Vogel Bonner            TT10423 used transduce strain mutagenized. <br /> (1956) used standard minimal medium. The rich            The carried plasmid pNK2280 (Kleckner et al. 1991) <br /> medium Luria broth (LB), prepared according Maloy            permit complementation defective Tn10dTc element <br /> (1990). Carbon sources 0.2% glucose 0.2%             ATS transposase. Transducing mixtures directly <br /> arabinose. Solid media contained agar 1.5% final concentra-      LB plates containing tetracycline incubated 24 <br /> tion. Auxotrophic requirements antibiotics used         hr 37&#226;&#172;&#353;. For preparation insertion pools, plates con- <br /> the final concentrations described Maloy (1990). As           taining Tcr transductants replica printed LB-EGTA. <br /> exception, viability assays involving plasmid pNG1166          Several thousands colonies harvested used <br /> carried plates containing lower concentration           inoculate single, mixed culture LB-EGTA. When satu- <br /> ampicillin (30 &#226;?&#174;g/ml), recommended Guzman et al.              rated, culture harvested centrifugation washed <br /> (1995). d-Mecillinam, gift Juan A. Ayala (Centro          times LB. An aliquot used for <br /> Biolog&#196;&#177;&#194;&#180;a Molecular, CSIC, Cantoblanco, Spain), used         standard lysis P22 HT. <br /> described below. Green plates prepared according               Replacement Mud J element Mud P: A lysate grown <br /> Chan et al. (1972), methyl blue (Sigma, St. Louis)      strain MST1753 used transposon replacement by <br /> substituted aniline blue. Motility assays carried      homologous recombination. In substitution, Mud J ele- <br /> in LB prepared yeast extract (Gillen Hughes             ment replaced MudP element recombination <br /> 1991). Solid motility medium contained agar 0.25% final          ends elements. P22 HT transductions se- <br /> concentration.                                                      lecting incoming marker (Cmr) carried out. Cmr <br />    Surveys d-mecillinam resistance: Minimal inhibitory con-      transductants scored loss resident marker <br /> centrations (MIC) d-mecillinam determined solid          (Kmr). <br /> media (LB E) using late exponential cultures strains        &#226;?&#164;-Galactosidase assays: Levels &#226;?&#164;-galactosidase activity <br /> to tested, prepared Luria broth. The final concentrations     assayed described Miller (1972), using CHCl3- <br /> of d-mecillinam ranged 0.05 &#226;?&#174;g/ml 1 &#226;?&#174;g/ml. Levels           sodium dodecyl sulfate permeabilization procedure. <br /> of mecillinam resistance assessed filter paper          Bacterial transformation: Transformation E. coli DH5&#226;?&#163; <br /> discs soaked solution mecillinam (40 &#226;?&#174;g/ml). The discs      plasmids followed procedure Inoue et al. (1990). <br /> were placed LB E plates, previously spread 105           Transformation S. enterica according Lederberg and <br /> colony-forming units strain tested. After 18&#226;&#8364;&#8220;24 hr     Cohen (1974). The Salmonella host plasmids strain <br /> incubation 37&#226;&#172;&#353;, diameters inhibition halos       TR5878. Plasmids transformed TR5878 transferred <br /> measured.                                                           suitable recipients transduction P22 HT. <br />    Plasmids: Plasmid pBAD18 (Apr) member pBAD              Matings: Transfer pIZ1551 E. coli S17 &#226;?&#173;pir S. enterica <br /> series vectors, designed study essential genes        assayed using exponential cultures donor and <br /> (Guzman et al. 1995). Plasmid pUC4-KIXX (Kmr Apr), prod-          recipient. To obtain higher cell concentrations, cultures <br /> uct Pharmacia Biotech Europe (Sant Cugat del Valle`s,            harvested centrifugation concentrated 10- 100- <br /> Spain), carries Tn5 kanamycin-resistance gene flanked        fold. Donor recipient aliquots 100 &#226;?&#174;l mixed on <br /> restriction sites facilitate cloning. Plasmid pGEM-3Z (Apr)    LB plate incubated 8 hr 37&#226;&#172;&#353; replica printing <br /> is cloning vector Promega (Madison, WI). Plasmid             selective plates. <br /> pIZ994, constructed laboratory, pGEM-3Z derivative         DNA amplification polymerase chain reaction: Am- <br /> that carries igaA locus S. enterica. Plasmid pIZ998 (Apr     plification reactions carried Perkin Elmer Gene- <br /> Kmr) pIZ994 derivative igaA locus        Amp PCR System 2400 (Perkin Elmer Cetus, Foster City, CA). <br /> interrupted KIXX cassette pUC4-KIXX, gen-          The final volume reactions 50 &#226;?&#174;l, final <br /> erate igaA2::KIXX allele. Plasmid pGP704 (Apr) pBR322      concentration MgCl2 1 mm. Reagents used the <br /> derivative carries origin replication R6K    following concentrations: dNPTs, 200 &#226;?&#174;m; primers, 1 &#226;?&#174;m; and <br /> mob region RP4 (Miller Mekalanos 1988). Plasmid              Taq polymerase, 1 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> reaction. The thermal program <br /> pIZ1551 (Apr) pGP704 derivative constructed labo-       included following steps: (i) initial denaturation, 10 min <br /> ratory carries igaA2::KIXX allele pIZ998 cloned        94&#226;&#172;&#353;; (ii) 35 cycles denaturation (94&#226;&#172;&#353;, 30 sec), annealing <br /> at EcoRI site pGP704.                                        (55&#226;&#172;&#353;, 30 sec), extension (72&#226;&#172;&#353;, 1 min); (iii) final incuba- <br />    Construction plasmid pNG1166: Genomic DNA                tion 72&#226;&#172;&#353; 10 min, complete extension. <br /> strain SL1344 PCR amplified using primers                DNA sequencing: Chromosomal DNA prepared <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> sides igaA gene: 5&#226;&#172;&#732;-TCT GTG GTA CCA CGC CTG ACA              1.5-ml overnight cultures LB. Cells harvested by <br /> GAC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CAA TAT CTA GAT GCA TGG GGA ACT GC-                   centrifugation resuspended 1.5 ml 10 mm Tris-HCl, <br /> 3&#226;&#172;&#732;, introduce KpnI XbaI sites (underlined), respec-       pH 8.0, 25 mm EDTA, pH 8.0. A volume 0.55 ml lysozyme <br /> tively. The amplified DNA fragment digested KpnI           solution (10 mg/ml 10 mm Tris-HCl, pH 8.0, 1 mm EDTA, <br /> and XbaI permit oriented cloning igaA gene             pH 8.0) added. The mixture incubated 20 min <br /> pBAD18 (Guzman et al. 1995). Ligation mixtures used            37&#226;&#172;&#353;. Proteinase K (100 &#226;?&#174;g/ml) added, the <br /> to transform E. coli DH5&#226;?&#163;, selecting Apr transformants LB-       preparation incubated 1 hr 55&#226;&#172;&#353;. After four <br /> ampicillin plates. One transformants source       extractions phenol chloroform-isoamyl alcohol (24:1), <br />  1516                                                      D. A. Cano et al. <br />  <br /> DNA precipitated ammonium acetate absolute              <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> associated igaA1 <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, igaA1 <br /> ethanol, finally suspended 10 mm Tris-HCl, pH 8.0,            strain mutagenized MudJ; nonmucoid colonies <br /> 1 mm EDTA, pH 8.0. DNA sheared 23G <br /> needle cleaned Sephadex G50 column. The primer              sought the Kmr transductants. <br /> used sequencing boundaries Tn10 insertions         The strain used, SV4254, carried Tn10 insertion (zhf- <br /> 5&#226;&#172;&#732;-CTA ATG ACA AGA TGT GT-3&#226;&#172;&#732; (Way Kleckner 1984).                6311::Tn10dTc) cotransducible igaA1 mutation <br /> For Mud J insertions, primer 5&#226;&#172;&#732;-CGA ATA ATC CAA              (Cano et al. 2001). Nonmucoid Kmr transductants were <br /> TGT CCT CC-3&#226;&#172;&#732; (Torreblanca et al. 1999), henceforth called           subjected following tests: <br /> MuL. Sequencing performed described elsewhere <br /> (Cano et al. 2001).                                                    i. For reconstruction analysis candidates lysed <br />    Antibodies: Rabbit anti-FtsZ polyclonal antibody gift            P22 HT, lysates used transduce <br /> from Miguel Vicente (Centro Nacional Biotecnolog&#196;&#177;&#194;&#180;a, <br /> CSIC, Madrid).                                                            SV4254, selecting kanamycin resistance. A 100% <br />    Rabbit anti-IgaA polyclonal antibody raised              linkage Kmr marker nonmu- <br /> recombinant N-IgaA-6xHis protein, produced                  coid phenotype indicated existence suppres- <br /> purified follows: 707-bp PCR fragment (nucleotide posi-              sor mutation. Only independent isolates passed <br /> tions 64&#226;&#8364;&#8220;771 putative translation start igaA; EMBL            test. <br /> accession no. AJ272210) PCR amplified primers <br /> 5&#226;&#172;&#732;-TCC GGG CCA TGG CCA GAC GAG GG-3&#226;&#172;&#732; 5&#226;&#172;&#732;-AAT                       ii. The lysates used transduce wild- <br /> TTC CTC GAG CGC GCT TTT CGA-3&#226;&#172;&#732;, introduce NcoI                      type strain, selecting tetracycline resistance. If true <br /> and XhoI sites (underlined), respectively. The resulting frag-            suppression occurred, igaA1 mutation would <br /> ment digested NcoI XhoI inserted in-frame                persist chromosome, phenotype <br /> upstream His tag sequence expression vector               reappear absence suppressors. The <br /> pET21d&#226;&#171;&#185; (Novagen, Madison, WI). The resulting plasmid, <br /> pETN1-2, verified sequencing insert                  candidates passed test: cases, &#226;&#181;&#8216;35% of <br /> junctions. This plasmid used transform E. coli                Tcr transductants mucoid, indicating that <br /> BL21 (DE3; Novagen). The resulting strain grown 37&#226;&#172;&#353;                igaA1 mutation cotransduced the <br /> with shaking early exponential phase, expression                insertion zhf-6311::Tn10dTc. <br /> of IgaA6xHis induced addition 0.1 mm isopropyl             iii. Cotransduction zhf-6311::Tn10dTc in- <br /> thiogalactoside (IPTG). After 3 hr induction, bacteria were <br /> harvested centrifugation, suspended column-binding                  sertion MudJ element detected in <br /> buffer (5 mm imidazole, 500 mm NaCl, 5 mm HEPES,                      suppressor-carrying isolates, indicating <br /> pH 7.9), disrupted French press. Bacterial debris              suppressor mutations linked to <br /> was removed centrifugation protein purified               igaA locus. <br /> the supernatant cobalt-affinity chromatography according <br /> to manufacturer&#226;&#8364;&#8482;s instructions. Following elution 150          To identify loci MudJ element had <br /> mm imidazole, purified IgaA6xHis concentrated             inserted, DNA sequencing performed directly on <br /> desired volume using centriplus YM-10 (Millipore, Bedford,           chromosome, using MuL primer (Torreblanca <br /> MA). Polyclonal antibodies obtained standard immu- <br /> nization rabbits N-IgaA-6xHis protein.                   et al. 1999). The MudJ insertions suppressed mu- <br />    SDS-PAGE Western blot analysis: Proteins bacte-          coidy igaA1 background gmm, <br /> rial extracts prepared described Kaniga et al. (1995)     manB, wcaK, wcaF genes, wca <br /> and analyzed SDS-PAGE Tris/Tricine buffer using             cluster colanic acid synthesis (Stevenson et al. <br /> 10% acrylamide gels. Blots probed rabbit affinity-         1996), rcsA rcsC, Rcs <br /> purified anti-FtsZ (1:200) rabbit anti-IgaA (1:5000) poly- <br /> clonal antibodies. Goat anti-rabbit HRP-conjugated secondary         network controls capsule synthesis (Gottesman <br /> antibody (Bio-Rad Life Science, El Prat Llobregat, Spain)         Stout 1991; Gottesman 1995). Interestingly, all <br /> was used secondary antibody detect specific proteins        mutations suppressed mucoid phenotype <br /> the enhanced chemiluminescence assay.                                reduced intracellular proliferation rate NRK <br />    Shift IgaA deprivation conditions: An overnight culture        fibroblast cell cultures (data shown), suggesting that <br /> of strain SV4578 [igaA2::KIXX/pNG1166 (igaA&#226;&#171;&#185;)] grown in <br /> LB medium supplemented 0.2% arabinose ampicillin            capsule overproduction necessary igaA1 mu- <br /> (100 &#226;?&#174;g/ml) diluted 1:100 medium. After              tant overgrow inside eukaryotic cells. <br /> 2 hr incubation 37&#226;&#172;&#353;, bacteria washed twice PBS,           A recent study describes S. enterica mutants dis- <br /> suspended LB medium adjusted OD600 0.05,             play mucoid phenotype suppressed rcsC <br /> transferred separate flasks, glucose (2%)         mutations (Costa Anto&#194;&#180;n 2001). The map position <br /> arabinose (0.2%) added. To maintain bacteria continu- <br /> ous exponential growing conditions, cultures refreshed          locus affected, mucM, similar igaA <br /> every hour. Samples taken hour SDS-PAGE           (Cano et al. 2001; Costa Anto&#194;&#180;n 2001), raising <br /> Western blot analysis.                                               possibility igaA1 allele mucM alleles <br />                                                                      ascribed single class. Because mucM <br />                                                                      mutants isolated resistance d-mecillinam (Costa <br />                           RESULTS <br />                                                                      Anto&#194;&#180;n 2001), carried tests d-mecillinam <br />    Suppression mucoid phenotype igaA1                   resistance strains SL1344 (igaA&#226;&#171;&#185;) SV4450 <br /> mutant MudJ insertions: The igaA1 mutation causes                 (igaA1). Both strains exhibited MIC values 0.8 <br /> mucoidy, best observed colonies grown                 &#226;?&#174;g/ml LB agar 0.6 &#226;?&#174;g/ml E plates. The diame- <br /> rich medium. To identify genes involved mucoid                ters growth inhibition halos similar both <br />                                                 Regulation wca flhDC igaA                                           1517 <br />  <br />                            TABLE 2 <br />       Effect igaA rcsB mutations expression <br />                       gmm::lac fusion <br />  <br />                       Relevant                 &#226;?&#164;-Galactosidase <br /> Strain                genotypea                   activityb <br /> SV4407         Wild type                           6.3   &#226;&#171;&#190;   0.7 <br /> SV4357         igaA1                              35.9   &#226;&#171;&#190;   1.2 <br /> SV4412         igaA1 rcsB70::Tn10dCm               7.7   &#226;&#171;&#190;   1.2 <br /> SV4411         rcsB70::Tn10dCm                     8.6   &#226;&#171;&#190;   0.8 <br />   a <br />     All strains carry, addition, gmm1::lac(Mud J) fusion. <br />   b <br />     The data shown means standard deviations          Figure 1.&#226;&#8364;&#8221;Effect igaA1 mutation expression <br /> at independent experiments.                            flhDC::lac fliA::lac fusions. The following strains were <br />                                                                    used, bottom: SV4614, SV4615, SV4324, SV4325, <br />                                                                    SV4630, SV4631. The data shown means and <br />                                                                    standard deviations independent experiments. <br /> strains. This absence differences suggests the <br /> igaA1 allele unrelated mucM alleles described by <br /> Costa Anto&#194;&#180;n (2001). Further differences between <br />                                                                    wild type (data shown). Therefore, delayed halo <br /> igaA1 mucM alleles described below. <br />                                                                    formation associated igaA1 mutation inter- <br />    The igaA1 mutation increases wca expression: In the <br />                                                                    preted motility defect. <br /> gmm::MudJ insertion described suppressor of <br />                                                                       The igaA gene regulates expression flagellar <br /> the mucoid phenotype igaA1 mutant (henceforth <br />                                                                    master operon &#239;&#172;&#8218;hDC: The motility defect UmoB&#226;&#171;&#186; <br /> called gmm-21::MudJ), MudJ element inserted <br />                                                                    mutants P. mirabilis known caused their <br /> in appropriate orientation generate lac fusion: <br />                                                                    inability activate flhDC operon (Dufour et al. <br /> The insertion Lac&#226;&#171;&#186; IgaA&#226;&#171;&#185; background and <br />                                                                    1998). In enteric bacteria, P. mirabilis, flhDC <br /> Lac&#226;&#171;&#185; presence igaA1 mutation. Analysis            operon encodes transcription factors necessary trig- <br /> &#226;?&#164;-galactosidase activity confirmed igaA1 muta-            ger genetic cascade controls flagellar synthesis <br /> tion caused increase expression gmm-              (Gygi et al. 1995; Macnab 1996; Furness et al. 1997; <br /> 21::lac fusion (Table 2). For experiments, cultures          Claret Hughes 2000). Hence, defect flhDC <br /> were prepared LB, mucoidy caused               expression causes reduced motility (Macnab 1996). <br /> igaA mutation higher. In wild type, expression           The homology igaA umoB led investi- <br /> gmm low, previously described E. coli wca genes         gate igaA1 mutation affected expression <br /> (Gottesman 1995). An additional observation               flagellar genes, using transcriptional lac fusions in <br /> the presence rcsB mutation suppressed derepres-              flhDC fliA. Evidence flhDC undergoes autoge- <br /> sion gmm igaA1 background (Table 2). This                nous positive regulation (Kutsukake 1997) led also <br /> result suggests derepression colanic acid synthe-          compare flhDC expression FlhDC&#226;&#171;&#185; FlhDC&#226;&#171;&#186; <br /> sis igaA1 mutation exerted Rcs regulatory        backgrounds. The experiments performed <br /> system. This view supported observation, de-             merodiploid strain (SV4325), permits analysis <br /> scribed above, rcsA rcsC mutations suppress           flhDC5213::lac fusion presence wild- <br /> mucoid phenotype igaA1 mutant.                               type flhDC operon. The results shown Figure 1 can <br />    The igaA1 mutation causes defect motility: The             summarized follows: <br /> DNA sequence relatedness igaA ORF and <br /> the umoB gene P. mirabilis (Dufour et al. 1998) raised           i. Expression flhDC decreased sixfold igaA1 <br />                                                                        background. The difference threefold when <br /> the possibility igaA1 mutants impaired in <br />                                                                        flhDC expression measured FlhDC&#226;&#171;&#186;/FlhDC&#226;&#171;&#185; <br /> motility. Hence, compared motility isogenic <br />                                                                        merodiploid. <br /> IgaA&#226;&#171;&#185; IgaA&#226;&#171;&#186; strains. Cultures prepared in <br />                                                                    ii. As expected, igaA1 mutation reduced expres- <br /> motility medium incubated 37&#226;&#172;&#353; shaking. At <br />                                                                        sion fliA::lac fusion, used representative of <br /> the stage midexponential growth, sterile toothpick <br />                                                                        class II genes, require FlhDC activation (Mac- <br /> was soaked culture used inoculate motil- <br />                                                                        nab 1996). <br /> ity agar plate. The diameters bacterial growth <br /> halos calculated hour. Halo formation                  The correlation reduced motility low- <br /> the igaA1 mutant proceeded speed half              ered flhDC expression igaA1 background in <br /> of wild type: 3.2 &#226;&#171;&#190; 0.2 mm/hr vs. 7.5 &#226;&#171;&#190; 0.9 mm/                agreement report mutations that <br /> hr, respectively. The doubling time igaA1 mutant            lower flhDC expression (e.g., crp, cya, hns) de- <br /> in liquid motility medium identical             crease motility (Kutsukake 1997). <br />  1518                                                   D. A. Cano et al. <br />  <br />                                                                   coid transductants. A screen &#226;&#172;&#381;2000 independent <br />                                                                   transductants did reveal mucoid colony, sug- <br />                                                                   gesting igaA1 mutation recessive. <br />                                                                      Segregation analysis (Anderson Roth 1977) <br />                                                                   confirmed recessivity igaA1 mutation. Twenty- <br />                                                                   Tcr Cmr transductants grown LB without <br />                                                                   antibiotics. Cultures diluted spread LB <br />                                                                   agar. Single colonies replica printed LB <br />                                                                   agar tetracycline. Eight 24 transductants <br />                                                                   segregated mucoid colonies. All mucoid colonies <br />                                                                   Cms, confirming nonmucoid <br />                                                                   merodiploids carried igaA1 mutation. The igaA1 <br />                                                                   mutation recessive and, principle, may <br />                                                                   cause loss function. <br />                                                                      The igaA gene S. enterica essential: The igaA1 <br />                                                                   mutation <span id='am-552' about='xsp:base' typeof='owl:Thing'>base</span> substitution putatively changes <br />                                                                   histidine moiety arginine (Cano et al. 2001); thus, <br />                                                                   priori, igaA1 allele null leaky. To com- <br />                                                                   pare properties igaA1 definite <br />    Figure 2.&#226;&#8364;&#8221;Effects rcs wca mutations flhDC operon     <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>null allele</span>, igaA gene knocked vitro. The <br /> expression igaA1 background. The strains used were, <br /> from bottom, SV4614, SV4615, SV4616, SV4623,               wild-type igaA allele carried plasmid pIZ998 inter- <br /> SV4625, SV4617, SV4618, SV4620, SV4619, SV4622,               rupted single BamHI site KIXX (Kmr) <br /> SV4624. The data shown means standard devia-          cassette pUC4-KIXX. Attempts replace chro- <br /> tions independent experiments.                  mosomal igaA gene construct failed (data not <br />                                                                   shown), raising possibility igaA an <br />                                                                   essential gene. To test hypothesis, suicide plasmid <br />    MudJ insertions rcsB rcsC, rcsA, sup-         containing igaA2::KIXX mutation (pIZ1551) was <br /> press defect &#239;&#172;&#8218;hDC expression caused muta-           constructed. Plasmid pIZ1551 pGP704 derivative <br /> tion igaA1: To investigate reduced expression             (Miller Mekalanos 1988) stably main- <br /> of flhDC operon igaA1 background sup-               tained E. coli S17 &#226;?&#173;pir (Simon et al. 1983). Plasmid <br /> pressed suppressors mucoidy, constructed set           pIZ1551 transferred conjugation strain <br /> of strains carried flhDC::lac fusion igaA1        SV4321, carries MudP-held duplication the <br /> mutation combination wca rcs mutations.                cysG-argA chromosomal region (Camacho Casade- <br /> Aside rcsA rcsC alleles described above,         su&#194;&#180;s 2001). Transconjugants selected LB plates <br /> a collection rcsB allele (rcsB70::Tn10dCm) used.         supplemented kanamycin (to select plasmid trans- <br /> As shown Figure 2, rcsB rcsC mutations suppressed          fer) chloramphenicol (to counterselect donor). <br /> the flhDC expression defect igaA1 background.               Because plasmid replicate strain devoid <br /> None wca insertions exerted effect flhDC            &#226;?&#178;-protein, Kmr transconjugants formed only <br /> expression. An rcsA mutation failed restore flhDC         plasmid integration chromosome or <br /> expression igaA1 mutant, suggesting IgaA               homologous recombination resident igaA <br /> might regulate flhDC RcsB/RcsC RcsA               gene incoming igaA2::KIXX allele. The latter <br /> not required. It noted examples             event yields Kmr Aps transconjugants, plasmid inte- <br /> loci regulated RcsB/RcsC, RcsA,           gration generates Kmr Apr transconjugants. An ampicil- <br /> in literature (Gottesman 1995; Clavel et al. 1996;            lin-sensitive igaA&#226;&#171;&#185;/igaA2::KIXX merodiploid (SV4322) <br /> Virlogeux et al. 1996; Davalos-Garcia et al. 2001).               obtained procedure. To confirm pres- <br />    The igaA1 mutation recessive: Strain 4215 carries           ence alleles, DNA SV4322 PCR ampli- <br /> a G &#226;&#8224;&#8217; A transition putative igaA ORF (Cano et              fied using igaA-flanking primers. Two amplification frag- <br /> al. 2001). To examine mutation domi-             ments 4.2 5.8 kb obtained (data shown). <br /> nant recessive, igaA1 mutation introduced              The size igaA locus 4.2 kb KIXX <br /> into merodiploid strain carrying chromosomal dupli-           cassette 1.6 kb: merodiploid contained both <br /> cation argA-cysG region, encompasses centi-          intact igaA locus <span id='am-474' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-475' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-476' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-478' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-479' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-480' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-491' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-508' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-529' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-532' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-533' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-534' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-542' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-543' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-546' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-551' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>knockout</span> allele. <br /> somes 64&#226;&#8364;&#8220;75 (Camacho Casadesu&#194;&#180;s 2001). For                  When duplication carried strain SV4322 was <br /> purpose, strain SV4321 transduced P22 HT               allowed segregate chloramphenicol-free medium, <br /> lysate grown SV4254. Tcr Cmr transductants                Cms colonies kanamycin sensitive. The absence <br /> selected. Because zhf-6311::Tn10dTc insertion              haploid segregants carrying igaA2::KIXX allele <br /> 35% linked igaA1 mutation (Cano et al. 2001),              supported view null igaA allele lethal. <br /> dominance expected result &#226;&#181;&#8216;35% mu-                This view received support experiments in <br />                                                Regulation wca flhDC igaA                                          1519 <br />  <br /> which P22 HT lysate grown strain SV4322 (igaA&#226;&#171;&#185;/ <br /> igaA2::KIXX) used transduce SL1344 (igaA&#226;&#171;&#185;, hap- <br /> loid) SV4321 (igaA&#226;&#171;&#185;/igaA&#226;&#171;&#185;, merodiploid). Kana- <br /> mycin-resistant transductants appeared frequencies <br /> 1000-fold higher SV4321 SL1344: 10&#226;&#171;&#186;6 <br /> and 10&#226;&#171;&#186;9 plaque-forming <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (PFU), respectively. <br /> Altogether, observations suggest igaA lo- <br /> cus S. enterica essential, conditions <br /> assayed. A corollary original igaA1 mutation <br /> may leaky. Interestingly, essential condition of <br /> igaA S. enterica shared related gene umoB <br />                                                                      Figure 3.&#226;&#8364;&#8221;Growth IgaA&#226;&#171;&#185;, IgaA&#226;&#171;&#186;, IgaA&#226;&#171;&#186; RcsC&#226;&#171;&#186; <br /> of P. mirabilis, insertion mutations de-          strains LB-glucose-ampicillin (left) LB-arabinose-ampi- <br /> scribed (Dufour et al. 1998).                                     cillin (right). The strains used SV4577 (igaA&#226;&#171;&#185;/ <br />    Because igaA promoter-proximal gene              pNG1166), SV4629 (igaA2::KIXX rcsC52::Mud P/pNG1166), <br /> a putative operon contain genes (yrfF,              SV4578 (igaA2::KIXX/pNG1166). In latter, expres- <br />                                                                   sion plasmid-borne igaA gene conditional, driven <br /> yrfG, yrfH, yrfI ), investigated lethality <br />                                                                   arabinose-dependent PBAD promoter. The assays were <br /> of igaA2::KIXX mutation caused polar effect          carried described Guzman et al. (1995), the <br /> on downstream genes. For purpose, KIXX cassette            concentration ampicillin 30 mg/liter. <br /> was introduced second ORF putative op- <br /> eron, yrfG (data shown). The yrfG::KIXX mutant <br /> was viable, indicated observation strains          plates, glucose. In contrast, IgaA&#226;&#171;&#185; IgaA&#226;&#171;&#186; <br /> SL1344 (igaA&#226;&#171;&#185;, haploid) SV4321 (igaA&#226;&#171;&#185;/igaA&#226;&#171;&#185;, mero-            RcsC&#226;&#171;&#186; strains (SV4577 SV4629) grew both <br /> diploid) transduced similar frequencies               media. These experiments confirm lack igaA <br /> (&#226;&#181;&#8216;10&#226;&#171;&#186;6/PFU) P22 lysate carrying yrfG::KIXX             expression prevents growth S. enterica stan- <br /> construct. We concluded lethality            dard laboratory conditions. <br /> igaA2::KIXX mutation solely igaA inactiva-                MudJ insertions rcsB rcsC, rcsA, sup- <br /> tion. Furthermore, yrfG::KIXX mutant nonmu-               press lethality caused null igaA allele: We have <br /> coid unable derepress flhDC5213::lac fusion            described rcsA rcsC mutations suppress <br /> (data shown), indicating mucoidy flhDC               mucoidy reduced motility caused viable allele <br /> derepression igaA-associated traits.                    igaA1. To investigate mutations affecting the <br />    Finally, tested lethality caused           Rcs regulatory suppressed lethality <br /> null igaA mutation complemented wild-               caused null allele igaA2::KIXX, transductional <br /> type igaA allele carried plasmid. For purpose,          crosses performed. The recipients SV4379 <br /> plasmid pNG1166 transduced SL1344. The re-                 (RcsA&#226;&#171;&#186;), SV4380 (RcsB&#226;&#171;&#186;), SV4406 (RcsC&#226;&#171;&#186;). The <br /> sulting strain transduced P22 HT lysate           donor igaA&#226;&#171;&#185;/igaA2::KIXX merodiploid (SV4322). <br /> grown strain SV4322 (igaA&#226;&#171;&#185;/igaA2::KIXX), selecting             Kanamycin-resistant transductants selected upon <br /> kanamycin resistance LB plates containing 0.2% ara-            P22 HT transduction. Their frequencies (per PFU) were <br /> binose. Kmr transductants appeared frequencies                 &#226;&#181;&#8216;10&#226;&#171;&#186;6 RcsB&#226;&#171;&#186; RcsC&#226;&#171;&#186; recipients &#226;&#172;?10&#226;&#171;&#186;9 <br /> &#226;&#181;&#8216;10&#226;&#171;&#186;6/PFU, suggesting igaA2::KIXX mutation               RcsA&#226;&#171;&#186; recipient. Hence, lack RcsB RcsC <br /> did impair viability presence igaA&#226;&#171;&#185; allele      functions does suppress lethality igaA null <br /> carried pNG1166. To confirm igaA2::KIXX               mutation, lack RcsA does not. This suppressor <br /> construct recombined recipient chromo-               <span id='am-553' about='xsp:pattern' typeof='owl:Thing'>pattern</span> identical flhDC expression <br /> some (and plasmid pNG1166), putatively                 igaA1 background (see above). <br /> complemented isolate lysed P22 HT. The lysate               Search Tn10 insertions suppress lethality <br /> was used transduce SL1344, selecting Apr. Transduc-            caused null igaA allele: Mutagenesis Tn10dTc <br /> tants obtained frequency 10&#226;&#171;&#186;5/PFU,               carried strain SL1344; pools Tn10dTc <br /> all (100/100) Kms. If lysate used            insertions, containing 10,000 independent <br /> select kanamycin resistance, transductants extremely         isolates, prepared. The pools grown in <br /> rare (&#226;&#181;&#8216;10&#226;&#171;&#186;9/PFU) Aps. Transduction kana-              LB &#226;&#171;&#185; EGTA midexponential phase used as <br /> mycin resistance merodiploid strain SV4321 was,            recipients P22 HT-mediated transductional crosses. <br /> however, successful. These experiments confirmed             The donor merodiploid strain SV4322 (igaA&#226;&#171;&#185;/ <br /> the donor carried intact plasmid pNG1166              igaA2::KIXX). Kmr transductants selected on <br /> the chromosomal igaA2::KIXX construct. This strain            LB-kanamycin plates. Each cross yielded 5&#226;&#8364;&#8220;30 Kmr trans- <br /> propagated SV4578.                                             ductants. Segregation analysis nonselective growth <br />    Occurrence complementation confirmed                 (in LB kanamycin) allowed discard igaA&#226;&#171;&#185;/ <br /> plate tests, shown Figure 3. Strain SV4578 (igaA2::KIXX/       igaA2::KIXX merodiploids segregated Kms colo- <br /> pNG1166) able grow arabinose-containing                 nies. Stable Kmr isolates carried putative Tn10dTc inser- <br />  1520                                                    D. A. Cano et al. <br />  <br />  <br />  <br />  <br />                                                                       Figure 5.&#226;&#8364;&#8221;Levels IgaA FtsZ protein extracts from <br />    Figure 4.&#226;&#8364;&#8221;Levels IgaA FtsZ protein extracts      SL1344 (wildtype, lane 1), SV4215 (igaA1, lane 2), SV4379 <br /> strain SV4578 (igaA2::KIXX/pNG1166) transfer glu-          (rcsA, lane 3), SV4406 (rcsB, lane 4), SV4380 (rcsC, lane 5), <br /> cose medium (left) fresh arabinose medium (right).           SV4343 (igaA1 rcsA, lane 6), SV4402 (igaA1 rcsB, lane 7), <br /> Samples taken hour SDS-PAGE Western             SV4404 (igaA1 rcsC, lane 8), SV4345 (igaA2::KIXX rcsC, lane <br /> blot analysis. The time sampling (in hours) indicated     9), SV4443 (igaA2::KIXX rcsB, lane 10). The extracts were <br /> the sample; zero designates time transfer       prepared bacteria growing exponentially LB. <br /> to fresh media. <br />  <br />                                                                    RcsC regulatory shown <br /> tions suppressed lethality associated            regulate positively cell division genes ftsA ftsZ <br /> igaA2::KIXX mutation. These isolates sub-                (Carballe`s et al. 1999), conceivable explanation for <br /> jected reconstruction analysis:                                 lethality associated null igaA alleles be <br />                                                                    cell division arrest, consequence overproduction <br />  i. Fifty independent candidates lysed P22 <br />                                                                    cell division proteins. In E. coli, key protein <br />     HT, resulting lysates used transduce <br />                                                                    regulates frequency timing cell divi- <br />     SL1344, selecting tetracycline resistance. <br />                                                                    sion FtsZ (Ward Lutkenhaus 1985). On these <br /> ii. Tcr transductants obtained crosses were <br />                                                                    grounds, investigated decrease IgaA <br />     transduced SV4322 lysate, selecting kanamy- <br />                                                                    synthesis altered level FtsZ. For purpose, <br />     cin resistance. As control, merodiploid strain <br />                                                                    carried shift-down experiments strain <br />     SV4321 (igaA&#226;&#171;&#185;/igaA&#226;&#171;&#185;) transduced the <br />                                                                    SV4578 (igaA2::KIXX/pNG1166) transferred from <br />     lysate. Whenever Kmr transductants ob- <br />                                                                    arabinose glucose medium. The amounts IgaA and <br />     tained similar frequencies SV4231 the <br />                                                                    FtsZ cell extracts obtained different times after <br />     putative suppressor-carrying isolates, evidence ex- <br />                                                                    arabinose depletion analyzed Western blotting <br />     isted igaA2::KIXX mutation viable the <br />                                                                    using anti-IgaA anti-FtsZ polyclonal antibodies. The <br />     genetic background latter. In cases, the <br />                                                                    results clear-cut: While swift decrease IgaA was <br />     frequency transductants &#226;&#181;&#8216;10&#226;&#171;&#186;6/PFU. Other- <br />                                                                    observed, level FtsZ remained unchanged (Figure <br />     wise, transduction occurred frequencies &#226;&#181;&#8216;10&#226;&#171;&#186;9/ <br />                                                                    4). These data suggest viability defect igaA <br />     PFU; rare transductants presumed to <br />                                                                    mutants caused FtsZ overproduction reveal <br />     carry suppressor mutations spontaneous origin <br />                                                                    difference igaA mucM alleles, <br />     discarded. <br />                                                                    increase transcription ftsA1 <br />    These experiments provided independent              promoter ftsQAZ operon (Costa Anto&#194;&#180;n <br /> suppressor-carrying isolates suppression le-           2001). Further evidence proposed functional <br /> thality appeared associated Tn10dTc inser-            interaction IgaA RcsB-RcsC system <br /> tion. One boundary Tn10dTc insertion              does involve changes level FtsZ pro- <br /> sequenced using Tn10-derived primer (Way                     vided following observations: (i) The levels of <br /> Kleckner 1984). Two insertions mapped rscC,              FtsZ remained largely unchanged strains carrying <br /> thereby confirming lack RcsC suppresses                igaA mutations, combined rcsA, rcsB, and <br /> lethality igaA null mutation. The remaining             rcsC alleles (Figure 5) (ii) strains carrying igaA1 <br /> Tn10dTc insertions mapped yojN, locus located                 mutation form minicells (not shown). <br /> between ompC rcsB Salmonella chromosome <br /> (McClelland et al. 2001). Mutations yojN sup- <br />                                                                                           DISCUSSION <br /> pressed mucoidy associated igaA mutations and <br /> the low expression flhDC (data shown). Suppres-                Mutants S. enterica carrying igaA1 allele a <br /> sion yojN mutations strengthens evidence               mucoid phenotype caused derepression wca genes. <br /> IgaA interacts RcsB-RcsC regulatory system,               The induction wca gene expression igaA1 mu- <br /> since YojN shown participate phos-              tant suppressed mutations rcsA rcsC. The <br /> phorelay signaling pathway RcsC &#226;&#8224;&#8217; RcsB (Takeda et al.              igaA1 mutation causes lowered expression the <br /> 2001).                                                             flagellar master operon, flhDC, results lowered <br />    The lethality caused null igaA allele does             motility. The defect suppressed mutations in <br /> involve changes level cell division protein          rcsB rcsC. Altogether, observations support a <br /> FtsZ: Because IgaA appears interact RcsB-              model loss IgaA causes activation the <br />                                              Regulation wca flhDC igaA                                                     1521 <br />  <br /> RcsB/C system, leading derepression wca              merodiploid carrying wild-type igaA allele, <br /> genes repression flhDC operon. However,         chromosomal plasmid borne. However, null <br /> noteworthy observation that, igaA1 background,         IgaA&#226;&#171;&#186; mutants viable presence rcsB, rcsC, <br /> activation colanic acid synthesis requires RcsA        yojN mutations, indicating existence a <br /> repression flagellar formation RcsA independent.          functional relationship IgaA RcsB-RcsC <br />    The opposite effects igaA gene product             system. A tentative explanation that, absence <br /> colanic acid synthesis flagellar formation remi-        IgaA, activation RcsB-RcsC lead <br /> niscent inverse relationship capsule syn-        derepression gene operon overexpres- <br /> thesis flagellar biogenesis described bacte-     sion lethal. An obvious candidate ftsQZA op- <br /> rial species: (i) Biofilm-forming E. coli repress flagellar     eron, known positively regulated the <br /> genes induce capsular gene expression (Prigent-             RcsB/C (Carballe`s et al. 1999; Costa and <br /> Combaret et al. 1999); (ii) production alginate          Anto&#194;&#180;n 2001). However, provide evidence the <br /> synthesis flagella Pseudomonas aeruginosa in-        viability defect null igaA mutants does involve <br /> verse mutually exclusive regulation (Garrett et al.         FtsZ-mediated cell division arrest. <br /> 1999); (iii) absence flagella signal induce         The viability igaA1 allele admits simple (albeit <br /> capsular polysaccharide synthesis Vibrio cholerae (Wat-      stage unproved) explanation. The mutation <br /> nick et al. 2001). One hypothesize forma-         causes single amino acid substitution recessive. <br /> tion flagella capsule synthesis mutu-      Hence, igaA1 allele leaky, residual func- <br /> ally exclusive Salmonella igaA gene             tion permit cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>survival</span>. However, view that <br /> product coordinate them, response              point igaA mutations tolerated countered <br /> to environmental signals. This coordination            observations E. coli, partial deletion <br /> exerted RcsB/C regulatory system.                       igaA homolog, yrfF, appears viable and <br />    The idea IgaA sensor protein specu-       causes mucoidy (Meberg et al. 2001). <br /> lative stage. However, analysis silico          We grateful John Roth, Kazuhiro Kutsukake, Kelly Hughes, <br /> predicted IgaA amino acid sequence unveils pres-            Stan Maloy, Eduardo Groisman providing strains; Nuria <br /> ence putative phosphorelay motifs similar           Go&#194;&#180;mez-Lo&#194;&#180;pez DNA sequencing reactions construction plas- <br /> found two-component regulatory systems (data             mid pNG1166; Francisco Ramos critical reading the <br />                                                                 manuscript. This work supported grants Spanish <br /> shown). Several potential transmembrane domains                 Ministry Science (BIO2001-0232-CO2) European Union <br /> (Hofmann Stoffel 1993) detected. Immu-             (QLK2-1999-00310). A.T. recipient Ph.D. fellowship the <br /> nodetection subcellular fractionation con-             Comunidad Madrid. <br /> firmed IgaA protein membrane bound (data <br /> not shown). With features mind, tempting <br /> to speculate IgaA novel pathway <br /> of signal transduction RcsB-RcsC system. A                                LITERATURE CITED <br /> membrane protein, DjlA, interacts RcsB/C          Altschul, S. F., T. L. Madden, A. A. Schaffer, J. Zhang, Z. Zhang <br /> participates wca activation described                   et al., 1997 Gapped BLAST PSI-BLAST: new generation <br />                                                                     database search programs. Nucleic Acids Res. 25: 3389&#226;&#8364;&#8220;3402. <br /> (Clarke et al. 1997; Kelley Georgopoulos 1997),             Anderson, R. P., J. R. Roth, 1977 Tandem genetic duplications <br /> and additional transmitters appear exist (Chen et al.            phage bacteria. Annu. Rev. Microbiol. 31: 473&#226;&#8364;&#8220;505. <br /> 2001). IgaA tentative addition list        Arricau, N., D. Hermant, H. Waxin, C. Ecobichon, P. S. Duffey <br />                                                                     et al., 1998 The RcsB-RcsC regulatory Salmonella typhi <br /> of sensors converging Rcs system. An alternative             differentially modulates expression invasion proteins, fla- <br /> possibility functional interaction              gellin Vi antigen response osmolarity. Mol. Microbiol. <br /> IgaA RcsB/C indirect. In E.                 29: 835&#226;&#8364;&#8220;850. <br />                                                                 Bereswill, S., K. Geider, 1997 Characterization rcsB <br /> coli, mutants affected mdoH gene (Ebel et al.                gene Erwinia amylovora influence exopolysaccha- <br /> 1997) defects similar igaA1                ride synthesis virulence blight pathogen. J. Bacte- <br /> mutants S. enterica (e.g., mucoidy suppressed muta-           riol. 179: 1354&#226;&#8364;&#8220;1361. <br />                                                                 Blattner, F., G. Plunkert, III, C. A. Bloch, N. T. Perna, V. Burland <br /> tions rcsA, rcsB, rcsC, impaired motility). The           et al. 1997 The complete genome sequence Escherichia coli <br /> mdoH gene product involved synthesis mem-              K-12. Science 277: 1432&#226;&#8364;&#8220;1434. <br /> brane-derived oligosaccharides (MDOs; Kennedy 1996).            Brill, J. A., C. Quinlan-Walshe S. Gottesman, 1988 Fine- <br />                                                                     structure mapping identification regulators capsule <br /> A model proposes periplasmic levels MDOs act                synthesis Escherichia coli K-12. J. Bacteriol. 170: 2599&#226;&#8364;&#8220;2611. <br /> to signal RcsC activate capsule synthesis (Ebel et al.       Camacho, E. M., J. Casadesu&#194;&#180;s, 2001 Genetic mapping dupli- <br /> 1997). Mutations genes lipopolysaccharide synthe-            cation segregation Salmonella enterica. Genetics 157: 491&#226;&#8364;&#8220;502. <br />                                                                 Cano, D. A., M. Martinez-Moya, M. G. Pucciarelli, E. A. Groisman, <br /> sis activate RcsB/C (Parker et al. 1992). In                J. Casadesu&#194;&#180;s et al., 2001 Salmonella enterica serovar Enterica <br /> analogous fashion, igaA gene product affect               response involved attenuation pathogen intracellular prolif- <br /> the RcsB-RcsC specific components                 eration. Infect. Immun. 69: 6463&#226;&#8364;&#8220;6474. <br />                                                                 Carballe`s, F., C. Bertrand, J. P. Bouche&#194;&#180; K. Cam, 1999 Regula- <br /> the cell envelope.                                                  tion Escherichia coli cell division genes ftsA ftsZ two- <br />    Null igaA mutations lethal maintained             component rcsC-rcsB. Mol. Microbiol. 34: 442&#226;&#8364;&#8220;450. <br />  1522                                                              D. A. Cano et al. <br /> Casadaban, M. J., S. N. Cohen, 1979 Lactose genes fused                   brane spanning protein fragments. Biol. Chem. Hoppe-Seyler <br />     exogenous promoters step using Mu-lac bacteriophage:                  347: 166. <br />     vivo probe transcriptional control sequences. Proc. Natl.         Hoiseth, S. K., B. A. Stocker, 1981 Aromatic-dependent Salmo- <br />     Acad. Sci. USA 76: 4530&#226;&#8364;&#8220;4533.                                                nella enterica non-virulent effective live vaccines. Nature <br /> Castilho, B. A., P. Olfson M. J. Casadaban, 1984 Plasmid                     291: 238&#226;&#8364;&#8220;239. <br />     insertion mutagenesis lac gene fusion Mini-Mu bacterio-         Hughes, K. T., J. R. Roth, 1984 Conditionally transposition- <br />     phage transposons. J. Bacteriol. 158: 488&#226;&#8364;&#8220;495.                               defective derivative Mu d1(Amp Lac). J. Bacteriol. 159: 130&#226;&#8364;&#8220; <br /> Chan, R. K., D. Botstein, T. Watanabe Y. Ogata, 1972 Special-                137. <br />     ized transduction tetracycline phage P22 Salmonella typhi-      Hughes, K. T., J. R. Roth, 1988 Transitory cis complementation: <br />     murium. II. Properties high frequency transducing lysate.               method provide transposition functions defective transpo- <br />     Virology 50: 883&#226;&#8364;&#8220;898.                                                        sons. Genetics 109: 263&#226;&#8364;&#8220;282. <br /> Chen, M. H., S. Takeda, H. Yamada, Y. Ishii, T. Yamashino et al.,            Inoue, H., H. Nojima H. Okayama, 1990 High efficiency trans- <br />     2001 Characterization RcsC &#226;&#8224;&#8217; YojN &#226;&#8224;&#8217; RcsC phos-                        formation Escherichia coli plasmids. Gene 96: 23&#226;&#8364;&#8220;28. <br />     phorelay signalling pathway involved capsular synthesis Esche-     Kaniga, K., S. Tucker, D. Trollinger J. E. Galan, 1995 Homo- <br />     richia coli. Biosci. Biotechnol. Biochem. 65: 2364&#226;&#8364;&#8220;2367.                     logs Shigella IpaB IpaC invasins required Salmo- <br /> Claret, L., C. Hughes, 2000 Functions subunits                 nella typhimurium entry cultured epithelial cells. J. Bacteriol. <br />     FlhD(2)C(2) transcriptional master regulator bacterial flagel-            177: 3965&#226;&#8364;&#8220;3971. <br />     lum biogenesis swarming. J. Mol. Biol. 303: 467&#226;&#8364;&#8220;478.                 Kelley, W. L., C. Georgopoulos, 1997 Positive control the <br /> Clarke, D. J., I. B. Holland A. Jacq, 1997 Point mutations                component RcsB/C signal transduction network DjlA: a <br />     transmembrane domain DjlA, membrane-linked DnaJ-                    member DnaJ family molecular chaperones Escherichia <br />     like protein, abolish function promoting colanic acid pro-            coli. Mol. Microbiol. 25: 913&#226;&#8364;&#8220;931. <br />     duction Rcs signal transduction pathway. Mol. Microbiol.         Kennedy, E. P., 1996 Membrane-derived oligosaccharides (periplas- <br />     25: 933&#226;&#8364;&#8220;944.                                                                 mic &#226;?&#164;-D-glucans) Escherichia coli, pp. 1064&#226;&#8364;&#8220;1071 Escherichia <br /> Clavel, T., J. C. Lazzaroni, A. Vianney R. Portalier, 1996                   coli Salmonella: Cellular Molecular Biology, edited F. C. <br />     Expression tolQRA genes Escherichia coli K-12 controlled        Neidhardt, R. Curtiss, III, J. L. Ingraham, E. C. C. Lin, K. B. <br />     RcsC sensor protein involved capsule synthesis. Mol.               Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter <br />     Microbiol. 19: 19&#226;&#8364;&#8220;25.                                                        H. E. Umbarger. American Society Microbiology, Wash- <br /> Conter, A., R. Sturny, C. Gutierrez K. Cam, 2002 The RcsCB                   ington, DC. <br />     His-Asp phosphorelay essential overcome chlorprom-          Kleckner, N., J. Bender S. Gottesman, 1991 Uses transpo- <br />     azine-induced stress Escherichia coli. J. Bacteriol. 184: 2850&#226;&#8364;&#8220;           sons emphasis Tn10. Methods Enzymol. 204: 139&#226;&#8364;&#8220;180. <br />     2853.                                                                    Kutsukake, K., 1997 Autogenous global control flagellar <br /> Costa, C. S., D. N. Anto&#194;&#180;n, 2001 Role ftsA1p promoter                 master operon, flhD, Salmonella typhimurium. Mol. Gen. Genet. <br />     resistance mucoid mutants Salmonella enterica mecil-         254: 440&#226;&#8364;&#8220;448. <br />     linam: characterization new type mucoid mutant. FEMS             Kutsukake, K., T. Iino, 1994 Role FliA-FlgM regulatory <br />     Microbiol. Lett. 200: 201&#226;&#8364;&#8220;205.                                               transcriptional control flagellar regulon and <br /> Davalos-Garcia, M., A. Conter, I. Toesca, C. Gutierrez K.                    flagellar formation Salmonella typhimurium. J. Bacteriol. 176: <br />     Cam, 2001 Regulation osmC gene expression two-                     3598&#226;&#8364;&#8220;3605. <br />     component rcsB-rcsC Escherichia coli. J. Bacteriol. 183:       Lederberg, E. M., S. N. Cohen, 1974 Transformation Salmo- <br />     5870&#226;&#8364;&#8220;5876.                                                                   nella typhimurium plasmid deoxyribonucleic acid. J. Bacteriol. <br /> Dufour, A., R. B. Furness C. Hughes, 1998 Novel genes                   119: 1072&#226;&#8364;&#8220;1074. <br />     upregulate Proteus mirabilis flhDC master operon controlling         Macnab, R. M., 1996 Flagella motility, pp. 123&#226;&#8364;&#8220;145 Escherichia <br />     flagellar biogenesis swarming. Mol. Microbiol. 29: 741&#226;&#8364;&#8220;751.              coli Salmonella: Cellular Molecular Biology, edited F. C. <br /> Ebel, W., G. J. Vaughn, H. K. Peters, III J. E. Trempy, 1997                 Neidhardt, R. Curtiss, III, J. L. Ingraham, E. C. C. Lin, K. B. <br />     Inactivation mdoH leads increased expression colanic                Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter <br />     acid capsular polysaccharide Escherichia coli. J. Bacteriol. 179:         H. E. Umbarger. American Society Microbiology, Wash- <br />     6858&#226;&#8364;&#8220;6861.                                                                   ington, DC. <br /> Furness, R. B., G. M. Fraser, N. A. Hay C. Hughes, 1997 Nega-            Maloy, S. R., 1990 Experimental Techniques Bacterial Genetics. <br />     tive feedback Proteus class II flagellum export defect             Jones &amp; Bartlett, Boston. <br />     flhDC master operon controlling cell division flagellum          McClelland, M., K. E. Sanderson, J. Spieth, S. W. Clifton, P. <br />     assembly. J. Bacteriol. 179: 5585&#226;&#8364;&#8220;5588.                                      Latreille et al., 2001 Complete genome sequence Salmonella <br /> Garrett, E. S., D. Perlegas D. J. Wozniak, 1999 Negative                     enterica serovar Enterica. Nature 413: 852&#226;&#8364;&#8220;856. <br />     control flagellum synthesis Pseudomonas aeruginosa modu-        Meberg, B. M., F. C. Sailer, D. E. Nelson K. D. Young, 2001 <br />     lated alternative sigma factor AlgT (AlgU). J. Bacteriol.             Reconstruction Escherichia coli mcrA (PBP 1a) mutants lacking <br />     181: 7401&#226;&#8364;&#8220;7404.                                                              multiple combinations penicillin binding proteins. J. Bacteriol. <br /> Gillen, K. L., K. T. Hughes, 1991 Negative regulatory loci                   183: 6148&#226;&#8364;&#8220;6149. <br />     coupling flagellar synthesis flagellar assembly Salmonella         Miller, J. H., 1972 Experiments Molecular Genetics. Cold Spring <br />     typhimurium. J. Bacteriol. 173: 2301&#226;&#8364;&#8220;2310.                                   Harbor Laboratory Press, Cold Spring Harbor, NY. <br /> Gottesman, S., 1995 Regulation capsule synthesis: modification            Miller, V. I., J. J. Mekalanos, 1988 A novel suicide vector and <br />     two-component paradigm accessory unstable regula-               use construction insertion mutations: osmoregulation of <br />     tor, pp 253&#226;&#8364;&#8220;262 Two-Component Signal Transduction, edited              outer membrane proteins virulence determinants Vibrio <br />     J. A. Hoch T. A. Silhavy. American Society Microbiology,             cholerae requires toxR. J. Bacteriol. 170: 2575&#226;&#8364;&#8220;2583. <br />     Washington, DC.                                                          Parker, C. T., A. W. Kloser, C. A. Schnaitman, M. A. Stein, S. <br /> Gottesman, S., V. Stout, 1991 Regulation capsular polysac-                Gottesman et al., 1992 Role rfaG rfaP genes in <br />     charide synthesis Escherichia coli K12. Mol. Microbiol. 5: 1599&#226;&#8364;&#8220;          determining lipopolysaccharide core structure cell sur- <br />     1606.                                                                        face properties Escherichia coli K-12. J. Bacteriol. 144: 2525&#226;&#8364;&#8220;2538. <br /> Gottesman, S., P. Trisler A. Torres-Cabassa, 1985 Regula-                Prigent-Combaret, C., O. Vidal, C. Dorel P. Lejeune, 1999 <br />     tion capsular polysaccharide synthesis Escherichia coli K-12:          Abiotic surface sensing biofilm-dependent regulation gene <br />     identification regulatory genes. J. Bacteriol. 162: 1111&#226;&#8364;&#8220;           expression Escherichia coli. J. Bacteriol. 181: 5993&#226;&#8364;&#8220;6002. <br />     1119.                                                                    Schmieger, H., 1972 Phage P22 mutants increased de- <br /> Guzman, L. M., D. Belin, M. J. Carson J. Beckwith, 1995 Tight                creased transducing abilities. Mol. Gen. Genet. 119: 75&#226;&#8364;&#8220;88. <br />     regulation, modulation, high level expression vectors con-        Simon, R., U. Priefer A. Pu&#194;&#168;hler, 1983 A broad host range <br />     taining arabinose PBAD promoter. J. Bacteriol. 177: 4121&#226;&#8364;&#8220;4130.           mobilization vivo genetic engineering: transposon <br /> Gygi, D., M. J. Bailey, C. Allison C. Hughes, 1995 Require-                  mutagenesis gram-negative bacteria. Methods Enzymol. 118: <br />     ment FlhA flagella assembly swarm-cell differentiation            640&#226;&#8364;&#8220;659. <br />     Proteus mirabilis. Mol. Microbiol. 15: 761&#226;&#8364;&#8220;769.                       Stevenson, G., K. Adrianopoulos, M. Hobbs P. R. Reeves, 1996 <br /> Hofmann, K., W. Stoffel, 1993 TMbase&#226;&#8364;&#8221;a database mem-                      Organization Escherichia coli K-12 gene cluster responsible <br />                                                     Regulation wca flhDC igaA                                                1523 <br />     production extracellular polysaccharide colanic acid.    Ward, J. E., J. Lutkenhaus, 1985 Overproduction FtsZ in- <br />     J. Bacteriol. 178: 4885&#226;&#8364;&#8220;4893.                                           duces minicell formation E. coli. Cell 42: 941&#226;&#8364;&#8220;949. <br /> Takeda, S., Y. Fujisawa, M. Matsubara, H. Aiba T. Mizuno,           Watnick, P. I., C. M. Lauriano, K. E. Klose, L. Croal R. Kolter, <br />     2001 A novel feature multistep phosphorelay Esche-            2001 The absence flagellum leads altered colony mor- <br />     richia coli: revised model RcsC &#226;&#8224;&#8217; YojN &#226;&#8224;&#8217; RcsC signalling       phology, biofilm development virulence Vibrio cholerae <br />     pathway implicated capsular synthesis swarming behav-            O139. Mol. Microbiol. 39: 223&#226;&#8364;&#8220;235. <br />     iour. Mol. Microbiol. 40: 440&#226;&#8364;&#8220;450.                                  Way, J. C., N. Kleckner, 1984 Essential sites transposon <br /> </body></html>