<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Nov. 2000, p. 8343&#226;&#8364;&#8220;8351                                                                         Vol. 20, No. 22 <br /> 0270-7306/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />        Functional Interaction Ssu72 Rpb2 Subunit of <br />               RNA Polymerase II Saccharomyces cerevisiae <br />                                      DONALD L. PAPPAS, JR.,            AND     MICHAEL HAMPSEY* <br />             Department Biochemistry, Division Nucleic Acids Enzymology, Robert Wood Johnson Medical School, <br />                                                   Piscataway, New Jersey 08854 <br />                          Received 18 July 2000/Returned modification 9 August 2000/Accepted 15 August 2000 <br />  <br />              SSU72 essential gene encoding phylogenetically conserved protein unknown function interacts <br />           general transcription factor TFIIB. A recessive ssu72-1 allele identified synthetic enhancer <br />           TFIIB (sua7-1) defect, resulting heat-sensitive (Ts&#226;&#180;&#353;) <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> dramatic downstream shift in <br />           transcription start site selection. Here new allele, ssu72-2, confers Ts&#226;&#180;&#353; phenotype a <br />           SUA7 wild-type background. In effort define Ssu72, isolated suppressors ssu72-2 <br />           <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span>. One suppressor allelic RPB2, gene encoding second-largest subunit RNA polymerase <br />           II (RNAP II). Sequence analysis rpb2-100 suppressor defined cysteine replacement phyloge- <br />           netically invariant arginine residue position 512 (R512C), located homology block D Rpb2. The <br />           ssu72-2 rpb2-100 mutations adversely affected noninduced gene expression, apparent effects on <br />           activated transcription vivo. Although isolated suppressor ssu72-2 Ts&#226;&#180;&#353; defect, rpb2-100 enhanced <br />           transcriptional defects associated ssu72-2. The Ssu72 protein interacts directly purified RNAP II <br />           coimmunoprecipitation assay, suggesting genetic interactions ssu72-2 rpb2-100 are <br />           consequence physical interactions. These results define Ssu72 highly conserved factor physically <br />           functionally interacts RNAP II core machinery transcription initiation. <br />  <br />  <br />    Eukaryotic RNA polymerase II (RNAP II) multisubunit                  Rpb1 subunit RNAP II, promoter melting promoter <br /> enzyme responsible transcription protein-                 clearance (25). <br /> encoding genes. Saccharomyces cerevisiae RNAP II 12-                       The GTFs discovered based requirement for <br /> subunit complex encoded genes RPB1 RPB12 (re-                      accurate initiation vitro (29, 52) highly conserved <br /> viewed references 2, 53, 55). The largest subunits,               eukaryotic organisms (reviewed references 17, 30, 33, <br /> Rpb1 Rpb2, homologous &#226;?&#164;&#226;&#172;&#732; &#226;?&#164; subunits                    38). Furthermore, TBP TFIIB requirements for <br /> bacterial RNA polymerase (RNAP), respectively (23, 49). The                  transcription predate divergence Eukarya Archaea <br /> yeast counterpart bacterial &#226;?&#163; subunit appears                   (reviewed reference 5). Nonetheless, RNAP II require- <br /> shared Rpb3 Rpb11 subunits (26, 57). Ac-                     ment GTFs universal instead dictated by <br /> cordingly, Rpb1, Rpb2, Rpb3, Rpb11 comprise func-                    promoter sequence architecture (16, 35, 51). Indeed, ge- <br /> tional equivalent bacterial &#226;?&#163;2&#226;?&#164;&#226;?&#164;&#226;&#172;&#732; core RNAP. The func-                nome-wide expression analysis revealed components <br /> tions remaining RNAP II subunits clear,                      RNAP II transcriptional machinery dispensable for <br /> although subunits, Rpb5, Rpb6, Rpb8, Rpb10, Rpb12,                  expression subsets genes (21). Conversely, a <br /> are shared forms RNAP                       limited number promoters analyzed vitro, is <br /> function transcription genes. With exception                conceivable additional GTFs identified based on <br /> Rpb4 Rpb9, subunits essential cell viability.                requirement accurate initiation specific promot- <br /> Rpb4 forms subcomplex Rpb7 required pro-                  ers. Other factors identified based ability <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> moter-specific initiation dispensable elongation (13).            stimulate activator-independent transcription. For example, <br />    RNAP II unable initiate promoter-specific transcription             yeast Tsp1 (Sub1) protein identified positive <br /> on own. Promoter recognition requires general tran-                  effector vitro transcription absence sequence- <br /> scription factors (GTFs), include TATA binding                     specific activator (20). Tsp1 interacts physically genetically <br /> protein (TBP), TFIIB, TFIIE, TFIIF, TFIIH (reviewed                   TFIIB homologous human transcriptional <br /> references 17 33). TBP nucleates assembly transcrip-                cofactor PC4 (20, 27). Additional basal factors likely be <br /> tion preinitiation complex binding TATA element                   identified, requirement promoter-specific <br /> inducing sharp bend DNA template. TFIIB binds                   initiation, ability stimulate activator-independent <br /> TATA-TBP complex responsible defining polar-                  transcription, genetic interactions GTFs RNAP II <br /> ity transcription binding asymmetrically BRE ele-               subunits. <br /> ment, located immediately upstream TATA box (28, 50).                    The yeast gene encoding TFIIB (SUA7) identified an <br /> RNAP II, association TFIIF, binds TATA-TBP-                      effector transcriptional accuracy (36). Replacements ei- <br /> TFIIB ternary complex, followed TFIIE TFIIH. TFIIH                    ther phylogenetically invariant residues, glutamate-62 <br /> is multisubunit complex catalytic activities responsible              (E62) arginine-78 (R78), caused marked start site shift <br /> for phosphorylation carboxy-terminal repeat domain                 downstream normal site CYC1 ADH1 pro- <br />                                                                              moters (37). Nearly identical effects transcriptional accu- <br />    * Corresponding author. Mailing address: Department Biochem-           racy conferred sua8 alleles RPB1, suggesting a <br /> istry, Robert Wood Johnson Medical School, 675 Hoes Lane, Piscat-            functional interaction TFIIB Rpb1 start <br /> away, NJ 08854. Phone: (732) 235-5888. Fax: (732) 235-5889. E-mail:          site selection (6). In effort identify factors that <br /> hampsemi@umdnj.edu.                                                          affect initiation, isolated suppressors sua7-1 (E62K) <br />  <br />                                                                       8343 <br />  8344       PAPPAS AND HAMPSEY                                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> mutation. The ssu71 ssu73 suppressors encode altered                             glucose mid-log phase (optical density 600 nm [OD600] &#226;&#171;&#189; 0.6). Cells were <br /> forms largest subunit TFIIF (Tfg1) Rpb9                            collected, washed, used inoculate 50-ml cultures synthetic complete <br />                                                                                      medium containing 2% glucose 3% glycerol plus 2% ethanol sole <br /> subunit RNAPII, respectively (46, 47). In contrast, ssu72                     carbon sources. Cultures incubated 30&#194;&#176;C 8 h, cells har- <br /> identified enhancer TFIIB E62K defect:                              vested. Total RNA isolated S1 nuclease protection assays per- <br /> ssu72-1 allele confers heat-sensitive (Ts&#226;&#171;&#186;) growth defect                      formed described previously (22), using synthetic oligonucleotides 5&#226;&#172;&#732;-GTGT <br /> a dramatic downstream start site shift, effects                      AGCACCTTTCTTAGCAGAACCGGCCTTGAATTCAGTCGGACGG and <br />                                                                                      5&#226;&#172;&#732;-GGAATTTCCAAGATTTAATTGGAGTCGAAAGCTCGCCTTA CYC1 <br /> dependent sua7-1 allele (48).                                               tRNAW, respectively. <br />    The SSU72 gene essential cell viability encodes                         INO1 transcript levels analyzed Northern blotting. Cells grown at <br /> novel protein undefined function (48). To charac-                         30&#194;&#176;C synthetic complete medium mid-log phase, collected centrifugation, <br /> terize Ssu72, generated new ssu72 allele confers                         washed, used inoculate 50-ml cultures synthetic complete medium <br />                                                                                      lacking containing inositol (55 &#226;?&#174;M). Cultures incubated 30&#194;&#176;C <br /> tight Ts&#226;&#171;&#186; phenotype, independent TFIIB defect. We                          8 h, cells harvested. Total RNA extracted analyzed as <br /> taken advantage ssu72 Ts&#226;&#171;&#186; phenotype isolate new                          described previously (36). The INO1 probe generated nick translation of <br /> rpb2 mutation encoding altered Rpb2 subunit                        1.0-kb INO1 EcoRI-HindIII fragment derived plasmid pJM310 the <br /> RNAP II. The results presented demonstrate Ssu72                           presence [&#226;?&#163;-32P]dCTP. The U6 control probe nick-translated 450 bp <br />                                                                                      fragment encompassing SNR6 gene. CYC1 INO1 transcript levels were <br /> is transcription factor interacts core RNAP II                       quantified using National Institutes Health Image program (http://rsb.in- <br /> machinery vivo vitro.                                                 fo.nih.gov/nih-image/) normalized tRNAW U6 RNA loading <br />                                                                                      control signals, respectively. <br />                                                                                         &#226;?&#164;-Galactosidase assays. Strains harboring CYC1-lacZ (pLG265-UP1 [15]), <br />                         MATERIALS AND METHODS <br />                                                                                      INO1-lacZ (pJS325 [42]), PGK1-lacZ (pN1086 [7]) reporter plasmids were <br />    Yeast strains. The yeast strains used study derived LRB535      grown 30&#194;&#176;C &#226;&#171;&#186;Ura medium mid-log phase (OD600 &#226;&#171;&#189; 0.6). Cells were <br /> (MATa his3&#226;&#338;&#172;200 leu2-3,112 ura3-52). YZS84 isogenic LRB535,          harvested resuspended &#226;&#171;&#186;Ura medium containing (i) 2% glucose or <br /> the ssu72-2 allele, generated site-directed mutagenesis subse-      3% glycerol plus 2% ethanol CYC1-lacZ strains, (ii) synthetic complete <br /> quent allele replacement, described below. YDP19 (ssu72-2 rpb2-100 shs2-1)        medium containing lacking 55 &#226;?&#174;M inositol INO1-lacZ strains, (iii) <br /> is spontaneous Ts&#226;&#171;&#185; revertant YZS84; YDP135 (ssu72-2 rpb2-100)              addition PGK1-lacZ strains. Cells incubated 8 h, harvested by <br /> segregant derived cross YDP19 YDP87 (ssu72-2 RPB2 SHS2).              centrifugation, resuspended 500 &#226;?&#174;l breaking buffer (100 mM Tris-HCl <br /> The plasmid-shuffle strain YZS89A (ssu72::LEU2 [SSU72-URA3]) described           [pH 8], 1 mM dithiothreitol, 20% glycerol; 2 mM phenylmethylsulfonyl fluoride). <br /> previously (48).                                                                     Cell extracts prepared vortexing 0.5-mm-diameter glass beads six <br />    Growth media, genetic methods, phenotypes. All growth media              times 15-s bursts. &#226;?&#164;-Galactosidase assays described previously <br /> prepared according standard recipes (43). &#226;&#171;&#186;Ino medium synthetic complete       (24). Activities determined duplicate assays independent trans- <br /> medium lacking inositol; &#226;&#171;&#185;Ino control medium contains 55 &#226;?&#174;M inositol. Stan-          formants expressed according following formula: (1.7 ml &#226;&#171;&#187; OD420 <br /> dard yeast genetic methods used making crosses, selecting diploids,         units)/(0.0045 &#226;&#171;&#187; cell extract volume [milliliters] &#226;&#171;&#187; reaction time [minutes] &#226;&#171;&#187; <br /> inducing sporulation, dissecting tetrads (44). The following designations    protein concentration [milligrams milliliter]). <br /> used phenotypes: Ts&#226;&#171;&#186;, temperature sensitivity, defined impaired growth           In vitro transcription translation reactions. Radiolabeled TBP, TFIIB, <br /> on YPD medium 37&#194;&#176;C; Cs&#226;&#171;&#186;, cold sensitivity, defined impaired growth          Ssu72 generated using TNT kit (Promega) presence of <br /> YPD medium 16&#194;&#176;C; Ino&#226;&#171;&#186;, impaired growth &#226;&#171;&#186;Ino medium 30&#194;&#176;C                 [35S]methionine according vendor specifications. Template DNAs puri- <br /> relative growth &#226;&#171;&#185;Ino control medium. Plasmids carrying URA3 marker         fied using Mini-Prep kit (Qiagen). Plasmids pT7-IID (9) pET-SUA7 were <br /> were counterselected synthetic medium containing 5-fluoro-orotic acid (8).        used templates synthesis TBP TFIIB, respectively. SSU72 ssu72 <br />    Construction ssu72-2 mutant. The ssu72-2 allele constructed         allelic DNAs amplified PCR using oligonucleotides 5&#226;&#172;&#732;-GGAATTCCA <br /> site-directed mutagenesis described previously (48) using synthetic oligonu-      TATGCCTAGTCATCGC 5&#226;&#172;&#732;-CGCGGATCCTTAGTGATGGTGATGGT <br /> cleotide oZS-200 (5&#226;&#172;&#732;-GAAGATTTGATGAATGCAGGTGGGAAATTAAAC-                              GATGTTTGTAATATGAAGGAGCG. This amplifies SSU72 ORF with <br /> 3&#226;&#172;&#732;). The underlined GC bases GCA triplet encode alanine amino acid         sequences coding hexahistidine tag C terminus flanking NdeI and <br /> position 129. The EcoRI-KpnI DNA fragment encompassing ssu72-2 open              BamHI sites (underlined). The PCR products digested ligated the <br /> reading frame (ORF) cloned corresponding sites plasmid pRS314        NdeI BamHI sites pCITE-4a(&#226;&#171;&#185;) (Novagen), generating ssu72 tem- <br /> (45), creating ssu72-2 TRP1 CEN plasmid pM721. The phenotype ssu72-2          plates pM1540 (SSU72), pM1541 (ssu72-1), pM1542 (ssu72-2), pM1543 <br /> was initially determined plasmid shuffle using host strain YZS89A.                (ssu72-4). <br />    Allele replacement normal SSU72 chromosomal locus                 RNAP II immunoprecipitation. Immunoprecipitation assays performed <br /> integration-excision (40). The EcoRI-KpnI fragment encompassing ssu72-2          described previously (34), minor modifications. Highly purified yeast <br /> cloned corresponding sites plasmid pRS306 (45), yielding ssu72-      RNAP II (425 ng) vitro-transcribed translated input proteins (10 &#226;?&#174;l) <br /> 2-integrating plasmid pM728. pM728 linearized unique ClaI site            incubated 250 &#226;?&#174;l P100 buffer (50 mm Tris-acetate [pH 7.9], 10% <br /> located SSU72 coding region introduced strain LRB535. Ura&#226;&#171;&#185;         glycerol, 100 mM potassium acetate, 14 mM magnesium acetate, 4 mM dithio- <br /> transformants isolated subsequently counterselected 5-fluoro-orotic      threitol, 1 mM EDTA, 50 &#226;?&#174;g bovine serum albumin ml, protease <br /> acid medium. Strains retaining ssu72-2 allele identified based       inhibitors [0.5 &#226;?&#174;g pepstatin A ml, 0.5 &#226;?&#174;g leupeptin ml, 2 &#226;?&#174;g of <br /> Ts&#226;&#171;&#186; phenotype associated allele (see above) confirmed           chymostatin ml, 2.5 &#226;?&#174;g antipain ml, 1 mM phenylmethylsulfonyl <br /> ability Ts&#226;&#171;&#186; phenotype complemented plasmid-borne SSU72.              fluoride, 2 mM benzamidine]) 4&#194;&#176;C 18 h. Pansorbin (Calbiochem) was <br /> Southern blot analysis confirmed integration excision SSU72 locus         harvested centrifugation resuspended blocking buffer (50 mM Tris- <br /> (D. L. Pappas, Jr., unpublished results).                                            acetate [pH 7.9], 100 mM potassium acetate, 25 mg PVP-40 [Sigma] ml <br />    Isolation sequence analysis rpb2-100 allele RPB2. The rpb2-100      30 mg bovine serum albumin ml [pH 7.9]). Following incubation for <br /> allele RPB2 recovered gap repair (40). Plasmid RY2119 (RPB2 URA3           20 min room temperature, Pansorbin centrifuged resuspended its <br /> CEN) digested completion SnaBI, removes RPB2           original volume P100 buffer. Twenty-five microliters preblocked Pansorbin <br /> ORF. Vector DNA flanked RPB2 sequences isolated agarose gel                added reaction mixture. Samples incubated 4&#194;&#176;C 30 min <br /> electrophoresis, purified, introduced strain YDP135 (rpb2-100). Ura&#226;&#171;&#185;        centrifuged 15,000 rpm 2 min. Supernatants transferred fresh <br /> colonies selected subsequently screened retention Cs&#226;&#171;&#186;        tubes containing 5 &#226;?&#174;g anti-RNAP II monoclonal antibody (8WG16; Covance, <br /> Ino&#226;&#171;&#186; phenotypes conferred rpb2-100 allele RPB2. Plasmid DNA            Inc.) incubated 90 min 4&#194;&#176;C. Preblocked Pansorbin (25 &#226;?&#174;l) added, <br /> recovered, amplified Escherichia coli, analyzed restriction digestion   reaction mixtures incubated 60 min 4&#194;&#176;C. The samples were <br /> confirm gap repair rpb2-100 locus. This resulting plasmid, pDP73, failed      centrifuged 16,000 &#226;&#171;&#187; g 2 min. The pellets washed times 250 <br /> to rescue Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes introduced strains YDP19         &#226;?&#174;l P100 buffer containing 0.1% NP-40 250 &#226;?&#174;l P100 buffer. <br /> YDP135, confirming recovery rpb2-100. To facilitate sequence analysis,        Pellets resuspended 40 &#226;?&#174;l sample buffer, 20 &#226;?&#174;l was <br /> 1.88-kb KpnI-EcoRI, 1.05-kb EcoRI-BglII, 2.5-kb PstI-SalI rpb2-100 frag-         resolved sodium dodecyl sulfate&#226;&#8364;&#8220;12% polyacrylamide gel. Following elec- <br /> ments pDP73 cloned KpnI-EcoRI sites pRS316, EcoRI-             trophoresis, gel treated En3Hance (Dupont NEN), dried, and <br /> BamHI sites pRS316, PstI-SalI sites pRS424, respectively. Single-          exposed phosphor screen. Products visualized using PhosphorImager <br /> stranded DNA generated using VCS-M13 helper phage presence             ImageQuant software (Molecular Dynamics). <br /> kanamycin described previously (41). The DNA sequence entire RPB2 <br /> ORF determined using ABI Prism automated DNA sequencer. DNA <br /> sequence comparisons performed using BLAST algorithm accessed <br />                                                                                                                        RESULTS <br /> via Saccharomyces Genome Database (http://genome-www.stanford.edu <br /> /Saccharomyces/). <br />                                                                                        The Ssu72 protein phylogenetically conserved. Ssu72 was <br />    RNA analyses. CYC1 transcript levels analyzed S1 nuclease protec-         reported novel protein apparent homologs in <br /> tion. Strains grown 30&#194;&#176;C synthetic complete medium containing 2%          databases (48). Genome sequencing projects since <br />  VOL. 20, 2000                                                                                                     Ssu72-RNAP II INTERACTION                  8345 <br />  <br />  <br />  <br />  <br />   FIG. 1. Sequence similarities eukaryotic Ssu72 proteins. The Ssu72 proteins S. cerevisiae (SC), S. pombe (SP), human (HS1 HS2), fruit fly (DM), <br /> and A. thaliana (AT) identified using BLAST algorithm (1) aligned using University Wisconsin Genetics Computer Group programs PILEUP and <br /> BOXSHADE. Identical similar amino acid residues highlighted black gray, respectively. The overbar encompassing residues 122 150 S. cerevisiae <br /> Ssu72 indicates region sequence similarity ATP-dependent RNA helicases noted previously (48). The asterisks indicate residues region are <br /> phylogenetically invariant helicases. The ssu72-2 allele encodes alanine replacement invariant arginine position 129 (R129A). <br />  <br />  <br />  <br />  <br /> identified homologs Ssu72 Schizosaccharomyces pombe,                          Ssu72 (Fig. 1). Since SSU72 essential cell viability, we <br /> human, Drosophila melanogaster, Arabidopsis thaliana,                           reasoned replacement conserved amino acid within <br /> Caenorhabditis elegans cells. Indeed, distinct homologs                      region confer growth phenotype. Accordingly, we <br /> Ssu72 human cells. The yeast, human, fly,                        constructed allele, ssu72-2, encodes alanine replace- <br /> and plant proteins approximately 200 residues <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>                     ment invariant arginine position 129 (R129A). Using <br /> and exhibit 46 54% sequence identity (BLAST values                            plasmid-shuffle assay, ssu72-2 mutant was <br /> &#226;&#172;?e&#226;&#171;&#186;39) S. cerevisiae Ssu72 (Fig. 1). The C. elegans protein                   viable failed grow 37&#194;&#176;C. The plasmid-borne ssu72-2 <br /> much larger, comprising 1,357 residues, region                      allele used replace chromosomal copy of <br /> sequence similarity confined N terminus, showing 43%                      SSU72, yielding strain YZS84. Consistent phenotypes <br /> identity S. cerevisiae Ssu72 (data shown). No proteins                       associated plasmid-borne ssu72-2 mutation, YZS84 <br /> with structural similarity Ssu72 bac-                       viable distinctly Ts&#226;&#171;&#186; (Fig. 2). <br /> terial archaeal protein sequence databases, suggesting                         Genetic suppression ssu72-2. To identify factors in- <br /> Ssu72 function common eukaryotes conserved                         teract Ssu72, isolated suppressors ssu72-2 Ts&#226;&#171;&#186; <br /> in eubacteria archaea.                                                    defect. Forty spontaneous Ts&#226;&#171;&#185; revertants strain YZS84 were <br />    Construction ssu72-2 mutant. The original ssu72 allele                    isolated 37&#194;&#176;C. When scored pleiotropic phenotypes, four <br /> (ssu72-1) enhances sua7-1 defect conferring Ts&#226;&#171;&#186;                            revertants failed grow absence inositol grew <br /> growth defect dramatically shifting transcription initia-                    normally synthetic complete medium. This phenotype is <br /> tion downstream normal site ADH1 gene (48).                           denoted Ino&#226;&#171;&#186; associated defects compo- <br /> These pronounced effects dependent                             nents general transcriptional machinery (18). Two the <br /> sua7-1 mutation. Indeed, ssu72-1 mutation confers ap-                        Ino&#226;&#171;&#186; revertants cold sensitive (Cs&#226;&#171;&#186;), growing poorly <br /> parent phenotype SUA7 wild-type background.                                    YPD medium 16&#194;&#176;C. One strains, YDP19, exhib- <br />    As efforts characterize SSU72, sought                       ited distinct Ts&#226;&#171;&#185; Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes (Fig. 2) chosen <br /> ssu72 mutant exhibits phenotypes independent sua7-1.                        characterization. <br /> The Ssu72 amino acid sequence includes region, encompass-                            Strain YDP19 (Ts&#226;&#171;&#185; Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;) backcrossed strain <br /> ing residues 122 150, highly conserved ATP-                        YDP87 (Ts&#226;&#171;&#186; Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;), resulting diploid strain was <br /> dependent RNA helicases (Fig. 1). Although sequences                          phenotypically Ts&#226;&#171;&#186; Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, indicating YDP19 <br /> common class helicases, including DEAD box                           phenotypes result recessive mutation(s). When this <br /> and HRIGRXXR motif, Ssu72,                              strain sporulated dissected (11 four-spore tetrads), the <br /> invariant residues conserved region present                       Ts&#226;&#171;&#185;:Ts&#226;&#171;&#186; phenotypes segregated 0:4, 1:3, 2:2, the <br />  8346       PAPPAS AND HAMPSEY                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Growth defects associated ssu72, rpb2-100, shs2 mutations. Relevant genotypes indicated wild-type strain LRB535 (SSU72 SHS1 SHS2), <br /> primary mutant YZS84 (ssu72-2 RPB2 SHS2), revertant YDP19 (ssu72-2 rpb2-100 shs2-1), YDP19 derivative carrying plasmid-borne copy RPB2 (ssu72-2 RPB2 <br /> shs2-1), YDP135 segregant (ssu72-2 rpb2-100 SHS2). The complete genotypes strains growth media defined Materials Methods. <br />  <br />  <br />  <br /> Cs&#226;&#171;&#185;:Cs&#226;&#171;&#186; Ino&#226;&#171;&#185;:Ino&#226;&#171;&#186; phenotypes segregated 2:2. Further-                          YDP135. The resulting transformants phenotypically <br /> more, Ts&#226;&#171;&#185; segregants Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;. These data in-                         Ts&#226;&#171;&#186;, Cs&#226;&#171;&#185;, Ino&#226;&#171;&#185;, confirming RPB2, a <br /> dicate suppression ssu72-2 Ts&#226;&#171;&#186; phenotype                        flanking gene, complements suppressor mutation. To con- <br /> result mutations unlinked genes mutation                     firm suppressor allelic RPB2, URA3 gene was <br /> one genes confers Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes. We                         integrated homologous recombination adjacent RPB2 <br /> tentatively designated suppressor genes shs1-1 (rpb2-                    locus strain YDP87. The resulting strain (ssu72-2 ura3 <br /> 100 [see below]) shs2-1. The shs1-1 allele responsible                  RPB2-URA3) crossed YDP19 (ssu72-2 ura3 shs1-1 <br /> the Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes, shs2-1 confers pro-                        shs2-1), diploid strain sporulated dissected. A <br /> nounced phenotype absence shs1-1.                                        total 44 segregants obtained scored. All Ura&#226;&#171;&#185; <br />    Identification shs1-1 suppressor. We exploited                       progeny phenotypically Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, Ura&#226;&#171;&#186; prog- <br /> Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes shs1-1 clone wild-type gene                      eny Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;. Thus, suppressor segregates op- <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> YCp50 genomic library (39). From approximately                              posite RPB2, confirming shs1-1 allelic to <br /> 90,000 Ura&#226;&#171;&#185; transformants YDP135 (ssu72-2 shs1-1), 25 <br />                                                                                    RPB2. Accordingly, renamed shs1-1 suppressor <br /> Cs&#226;&#171;&#185; strains isolated, 4 restored Ino&#226;&#171;&#185;. <br />                                                                                    rbp2-100. <br /> Plasmid DNAs isolated transfor- <br />                                                                                       The rpb2-100 mutation. The rpb2-100 allele cloned by <br /> mants include overlapping DNA inserts <br /> within YCp50 vector. One plasmid, pDP99, reintro-                          gap repair, entire ORF sequenced using collec- <br /> duced YDP135, scored transformants                        tion primers span RPB2 ORF. A single-<span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span>-pair <br /> Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, indicating complementation Cs&#226;&#171;&#186;                       substitution position 1534 (C1534T) identified, encod- <br /> Ino&#226;&#171;&#186; phenotypes consequence plasmid DNA                            ing replacement arginine-512 cysteine (R512C). The <br /> than strain reversion.                                                             Rpb2 subunit RNAP II structurally conserved among <br />    Restriction analysis plasmid pDP99 defined insert                      eukaryotic, archaeal, bacterial RNAPs, including eukary- <br /> approximately 8.0 kb YCp50 vector. Partial DNA se-                          otic RNAPs I, II, III. Sequence comparisons identified <br /> quence analysis identified segment yeast genome                      blocks, designated A I, highly conserved <br /> chromosome XV encompassing RPB2, gene encoding                             RNAPs (49). R512 amino acid within <br /> second-largest subunit RNAP II. To define re-                       homology block D phylogenetically invariant (Fig. 3). <br /> lationship shs1-1 RPB2, plasmid RY2119,                          The results presented indicate R512, despite phy- <br /> includes entire RPB2 coding region                          logenetic invariance, essential cell viability. Indeed an <br /> flanking ORFs original clone, introduced                           rpb2-100 allele normal (SSU72&#226;&#171;&#185;) genetic back- <br />  VOL. 20, 2000                                                                                                        Ssu72-RNAP II INTERACTION                    8347 <br />  <br />  <br />                                                                                       wild-type strain (compare lanes 1 2). Induced levels <br />                                                                                       diminished approximately 70% normal the <br />                                                                                       ssu72-2 mutant (lane 4) reduced 43% of <br />                                                                                       normal ssu72-2 rpb2-100 shs2-1 suppressor strain (lane <br />                                                                                       6). Interestingly, rpb2-100 dramatic effect the <br />                                                                                       absence shs2-1, nearly eliminating INO1 expression (lane 8). <br />                                                                                       These results consistent effects ssu72-2 and <br />                                                                                       rpb2-100 INO1-lacZ reporter gene expression re- <br />                                                                                       pressing conditions (Fig. 5B). Although Northern analysis <br />                                                                                       does distinguish effects noninduced versus <br />  <br />  <br />  <br />  <br />    FIG. 3. The rpb2-100 mutation. The rpb2-100 allele RPB2 encodes an <br /> R512C replacement phylogenetically invariant position homology <br /> block D Rpb2. The alignment homology block D (30 residues) shown for <br /> S. cerevisiae (Sc), S. pombe (Sp), A. thaliana (At), Plasmodium falciparum (Pf), C. <br /> elegans (Ce), Homo sapiens (Hs), Drosophila melanogaster (Dm), Pyrococcus <br /> horikoshii (Ph), second-largest subunits (Rpa135 Rpc128) RNAP I <br /> and RNAP III S. cerevisiae [Sc (I) Sc (III), respectively], &#226;?&#164; <br /> subunits E. coli (Ec) T. aquaticus (Ta) RNAPs. Amino acids indicated <br /> by single-residue <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span>; numbers right indicate position the <br /> C-terminal residue block D. <br />  <br />  <br />  <br />  <br /> ground displays weak growth impairment 30&#194;&#176;C is <br /> severely Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; (data shown). <br />    The ssu72-2 rpb2-100 alleles affect noninduced tran- <br /> scription vivo. The effects ssu72-2 rpb2-100 mutations <br /> on transcription specific promoters determined in <br /> vivo using wild-type strain (LRB535), ssu72-2 primary <br /> mutant (YZS84), ssu72-2 rpb2-100 shs2-1 suppressor strain <br /> (YDP19), ssu72-2 rpb2-100 segregant (YDP135). Ef- <br /> fects CYC1 expression presented Fig. 4A. Under <br /> repressing conditions (2% glucose), CYC1 transcript level <br /> was reduced ssu72-2 mutant 42% normal (compare <br /> lanes 1 3). Expression diminished the <br /> ssu72-2 rpb2-100 shs2-1 revertant 17% normal (lane 5), an <br /> effect attributed rpb2-100 shs2-1, <br /> since level expression observed ssu72-2 <br /> rpb2-100 mutant (lane 7). These effects occur level of <br /> noninduced transcription, strains retained ability <br /> to respond derepressing conditions. <br />    The effects CYC1 transcription confirmed using a <br /> CYC1-lacZ reporter. In wild-type strain 1,300 U activity <br /> was measured repressing conditions, activity <br /> was enhanced 3.5-fold derepressing conditions (Fig. 4B). <br /> Noninduced expression diminished 9-fold ssu72-2 <br />                                                                                          FIG. 4. Effects ssu72 shs mutations CYC1 expression. (A) The top <br /> primary mutant (140 U) reduced additional 30-fold                          panel indicates CYC1 transcript levels, determined S1 nuclease protection, for <br /> in ssu72-2 rpb2-100 shs2-1 suppressor strain (4.8 U). The                         strains grown repressing (2% glucose) (lanes R) derepressing (3% <br /> shs2-1 allele did contribute effect, activity                   glycerol 2% ethanol) (lanes D) conditions. Strains LRB535 (wild type <br /> equally low ssu72-2 rpb2-100 strain (3.3 U). Each mutant                        [WT]), YZS84 (ssu72-2), YPD19 (ssu72-2 rpb2-100 shs2-1), YDP135 <br />                                                                                       (ssu72-2 rpb2-100). Transcript levels repressing conditions presented <br /> responded derepressing conditions, absolute                           percentages, normalized level wild-type strain. The panel <br /> activities diminished parallel effects basal ac-                      loading control indicates tRNAW levels. (B) &#226;?&#164;-Galactosidase activities <br /> tivity. These results consistent RNA analyses (Fig.                      expressed CYC1-lacZ plasmid pLG265-UP1. Activity units re- <br /> 4A), confirming ssu72-2 rpb2-100 diminish nonin-                             pressing derepressing conditions, respectively, 1,300 &#226;&#171;&#190; 66 4,900 &#226;&#171;&#190; <br />                                                                                       1030 (WT), 140 &#226;&#171;&#190; 13 310 &#226;&#171;&#190; 40 (YZS84), 4.8 &#226;&#171;&#190; 1.2 19 &#226;&#171;&#190; 3.5 (YDP19), <br /> duced transcription CYC1 promoter.                                           3.3 &#226;&#171;&#190; 0.54 97 &#226;&#171;&#190; 14 (YDP135). The indicated units &#226;?&#164;-galactosidase <br />    INO1 expression determined Northern blot analysis                           activity represent means independent transformants assayed in <br /> (Fig. 5A). Strong induction INO1 mRNA observed                              duplicate; standard deviations indicated error bars. <br />  8348       PAPPAS AND HAMPSEY                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     activated expression, measurable levels &#226;?&#164;-galactosidase <br />                                                                                     activity INO1-lacZ reporter repressing condi- <br />                                                                                     tions demonstrate effects ssu72-2 rpb2-100 on <br />                                                                                     INO1 expression accounted effects nonin- <br />                                                                                     duced transcription. <br />                                                                                        The INO1-lacZ reporter assays repeated using strain <br />                                                                                     YDP135 complemented plasmid-borne <br />                                                                                     copies SSU72 and/or RPB2, allowing effects ssu72-2 <br />                                                                                     rpb2-100 assessed isogenic background inde- <br />                                                                                     pendent shs2-1. Consistent previous results, ssu72-2 <br />                                                                                     minor effect INO1-lacZ expression, <br />                                                                                     effects rpb2-100 single ssu72-2 rpb2-100 <br />                                                                                     double mutations dramatic, yielding 10% <br />                                                                                     normal activity (Fig. 5C). Thus, ssu72-2 exerts adverse ef- <br />                                                                                     fects gene expression markedly enhanced the <br />                                                                                     rpb2-100 allele, effects manifest level of <br />                                                                                     noninduced transcription. <br />                                                                                        The effects ssu72-2 rpb2-100 assessed using <br />                                                                                     PGK1 promoter, expressed high constitutive <br />                                                                                     level. A PGK1-lacZ reporter plasmid introduced each <br />                                                                                     strain, &#226;?&#164;-galactosidase activities measured (Fig. 6). <br />                                                                                     Expression PGK1 promoter diminished 60% <br />                                                                                     normal ssu72-2 mutant 40% normal the <br />                                                                                     ssu72-2 rpb2-100 shs2-1 triple mutant. Expression mark- <br />                                                                                     edly diminished ssu72-2 rpb2-100 double mutant, 3% <br />                                                                                     normal. Although experiments distinguish non- <br />                                                                                     induced activated expression, dramatic effect the <br />                                                                                     double mutant indicates ssu72-2 rpb2-100 exert <br />                                                                                     synergistic effects gene expression. The adverse effect <br />                                                                                     rpb2-100 SHS2&#226;&#171;&#185; background consistent its <br />                                                                                     effect INO1 expression (Fig. 5). These results demonstrate <br />                                                                                     shs2-1, required addition rpb2-100 sup- <br />                                                                                     pression ssu72-2 Ts&#226;&#171;&#186; phenotype (Fig. 2), suppresses <br />                                                                                     adverse effects rpb2-100 noninduced transcription. <br />                                                                                        Physical interaction Ssu72 RNAP II. The ge- <br />                                                                                     netic interactions ssu72, sua7, rpb2 suggest that <br />                                                                                     Ssu72 interacts directly core transcriptional machin- <br />                                                                                     ery. Indeed, Ssu72 directly binds TFIIB, consistent allele- <br />                                                                                     specific interactions ssu72 sua7 (54). Physical <br />                                                                                     interaction Ssu72 RNAP II investigated a <br />                                                                                     coimmunoprecipitation assay using purified yeast RNAP II <br />                                                                                     monoclonal antibody directed Rpb1 subunit <br />                                                                                     RNAP II. Input proteins 35S-labeled Ssu72, la- <br />                                                                                     beled TBP TFIIB controls. RNAP II known bind <br />                                                                                     TFIIB, TBP, interaction RNAP II requires <br />                                                                                     TFIIB bridging factor. The Ssu72 input included two <br />                                                                                     forms protein, corresponding full-length Ssu72 and <br />                                                                                     N-terminally truncated Ssu72 resulting initiation AUG <br />                                                                                     codon 23. The results demonstrate RNAP II binds stably <br />                                                                                     TFIIB Ssu72 TBP control (Fig. 7A). Thus, <br />                                                                                     genetic interaction ssu72-2 rpb2-100 likely <br />                                                                                     reflect physical interaction Ssu72 RNAP II. <br />                                                                                        The N terminus Ssu72 essential Ssu72 function and <br />                                                                                     cell viability, RNAP II binds smaller form Ssu72, <br />                                                                                     lacking N-terminal 22 residues. This investi- <br />  <br />  <br />  <br />                                                                                     (&#226;&#171;&#186;Ino medium) (lanes D) conditions. Strains identical Fig. 4. The <br />                                                                                     panel loading control showing U6 RNA levels. (B) &#226;?&#164;-Galactosidase <br />                                                                                     activities expressed INO1-lacZ plasmid pJS325 repressing con- <br />                                                                                     ditions. Activity units 310 &#226;&#171;&#190; 39 (wild type [WT]), 210 &#226;&#171;&#190; 36 (YZS84), 86 &#226;&#171;&#190; <br />                                                                                     12 (YDP19), 17 &#226;&#171;&#190; 4.1 (YDP135). Error bars indicate standard deviations. <br />                                                                                     (C) Same panel B, host strain case YDP135 (ssu72-2 <br />                                                                                     rpb2-100) carrying INO1-lacZ reporter plasmid-borne copies wild-type <br />    FIG. 5. Effects ssu72 shs mutations INO1 expression. (A) The       SSU72, RPB2, vectors alone, indicated genotypes. Activity units are <br /> panel indicates INO1 transcript levels, determined Northern blot analysis,   56 &#226;&#171;&#190; 2.8 (WT), 47 &#226;&#171;&#190; 4.7 (ssu72-2), 1.9 &#226;&#171;&#190; 0.07 (rpb2-100), 5.3 &#226;&#171;&#190; 0.63 (ssu72-2 <br /> strains grown repressing (&#226;&#171;&#185;Ino medium) (lanes R) derepressing             rpb2-100). <br />  VOL. 20, 2000                                                                                                         Ssu72-RNAP II INTERACTION                      8349 <br />  <br />  <br />  <br />  <br />    FIG. 6. &#226;?&#164;-Galactosidase activities expressed PGK1-lacZ plasmid. <br /> Strains identical Fig. 4 Fig. 5A B, was <br /> transformed PGK1-lacZ reporter plasmid pN1086. Activity units are <br /> 2,500 &#226;&#171;&#190; 88 (wild type [WT]), 1,500 &#226;&#171;&#190; 160 (YZS84), 1,000 &#226;&#171;&#190; 52 (YDP19), and <br /> 86 &#226;&#171;&#190; 3.6 (YDP135). <br />  <br />  <br />  <br /> gated asking specific mutations Ssu72 N <br /> terminus affect RNAP II binding. The ssu72-1 allele,                               FIG. 7. Physical interaction Ssu72 RNAP II. (A) Equal <br /> enhances sua7-1 mutation, encodes 10-amino-acid dupli-                          amounts highly purified yeast RNAP II incubated 35S-labeled mock, <br /> cation N terminus Ssu72 (residues 9 18),                             TBP, TFIIB, Ssu72 input proteins. Samples immunoprecipitated (IP) <br /> ssu72-4 encodes serine replacement cysteine 15 abol-                        using monoclonal antibody directed C-terminal repeat domain of <br />                                                                                       Rpb1. Following centrifugation, samples washed times buffer <br /> ishes cell viability (48). The interaction Ssu72                          containing 100 mM potassium acetate plus 0.1% NP-40 buffer <br /> RNAP II appeared unaffected ssu72-2 (R129A)                              containing 100 mM potassium acetate resuspended sample buffer. <br /> and ssu72-4 (C15S) mutations (Fig. 7B, lanes 5 8). Interest-                       Input proteins (5% total; lanes 1, 3, 5, 7, respectively) the <br /> ingly, however, 10-amino-acid duplication resulted pref-                       precipitates (50% total; lanes 2, 4, 6, 8, respectively) fractionated <br />                                                                                       sodium dodecyl sulfate-polyacrylamide gel electrophoresis analyzed by <br /> erential binding RNAP II longer form Ssu72 (Fig.                         autoradiography. WT, wild type. (B) Same panel A, different forms <br /> 7B, lanes 1 4). These results confirm Ssu72 binds di-                         Ssu72 used input proteins. The ssu72-1 allele encodes duplication of <br /> rectly RNAP II imply Ssu72 N terminus affects                         amino acid residues 9 18, ssu72-2 encodes R129A replacement, and <br /> this interaction.                                                                     ssu72-4 encodes inviable C15S replacement. The upper lower bands of <br />                                                                                       Ssu72 input proteins correspond, respectively, full-length protein a <br />                                                                                       truncated protein N terminus corresponds methionine position 23. In <br />                                DISCUSSION                                             case Ssu72-1, RNAP II interacts full-length truncated <br />                                                                                       forms protein. Control reactions lacking RNAP II failed coimmuno- <br />    The results paper define genetic physical inter-                       precipitate Ssu72 (data shown). <br /> actions Ssu72 RNAP II support role for <br /> Ssu72 basal (noninduced) transcription RNAP II. Ssu72 <br /> was initially identified based genetic interaction                          tags added N terminus appear expose cryptic pro- <br /> TFIIB. The ssu72-1 allele confers synthetic Ts&#226;&#171;&#186; growth de-                          teolytic site just Ssu72 N terminus, removing <br /> fect dependent sua7-1 mutation (48). The                             tag, sequences added C terminus render <br /> interaction ssu72-1 sua7 specific, resulting syn-                       Ssu72 nonfunctional vivo, conserved, <br /> thetic phenotypes combination sua7 alleles                          tandem aromatic residues C terminus (Fig. 1). Interest- <br /> affect transcription start site selection (54). Ssu72 ge-                     ingly, apparent Ssu72 homologs bacterial or <br /> netically linked Sub1 (Tsp1), homolog human gen-                          archaeal genomes, implying Ssu72 function specific to <br /> eral transcriptional cofactor PC4, sub1 deletion                       eukaryotes. In yeast, depletion Ssu72 causes growth arrest <br /> enhances sua7 mutations does allele-specific man-                        diminishes total poly(A) RNA levels 20 40% <br /> ner identical specificity ssu72-1 sua7 inter-                   effect RNAP I RNAP III transcription <br /> actions (54). Consistent genetic data, TFIIB directly                        (Pappas, unpublished results). These results suggest Ssu72 <br /> interacts Ssu72 Sub1 proteins (27, 54). Here                          critical transcription many, all, RNAP II genes <br /> demonstrate Ssu72 binds stably RNAP II                           plays role RNAP I RNAP III transcription. <br /> ssu72-2 rpb2-100 alleles affect basal transcription. Taken                           All evidence points role Ssu72 core promoter <br /> together, data demonstrate functional interactions                           function. The original ssu72-1 allele, encodes 10-ami- <br /> Ssu72 TFIIB, Sub1, Rpb2 provide strong support                           no-acid duplication near N terminus, affects transcription <br /> for Ssu72 RNAP II transcription factor.                                         start site selection (48). This effect dramatic, depen- <br />    We know specific function Ssu72. Our efforts                         dent sua7 allele (TFIIB) affects initiation. <br /> to characterize Ssu72 hampered sequence                             Here demonstrate ssu72-2 allele adversely affects <br />  8350     PAPPAS AND HAMPSEY                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> noninduced transcription. The rpb2-100 suppressor ssu72-2          R512C replacement context RNAP II <br /> also diminishes noninduced transcription, effect         three-dimensional structure. However, crystallographic <br /> be synergistic ssu72-2. The magnitude ssu72-2 effect      structure RNAP E. coli determined 12-&#195;&#8230; <br /> on transcription appears promoter specific:             resolution (11, 12), and, recently, RNAP T. <br /> CYC1 transcription diminished nearly 10-fold,            aquaticus determined 3.3-&#195;&#8230; resolution (56). R512 <br /> 2-fold effects observed INO1 PGK1 expression. The         highly conserved, corresponding R548 R428 &#226;?&#164; <br /> promoter elements responsible effects unknown           subunits RNAPs E. coli T. aquaticus, respectively <br /> but readily addressed whole-genome microarray               (Fig. 3). The sequence conservation forms RNAP <br /> analysis using wild-type ssu72-2 strains.                         nearly superimposable structures E. coli T. <br />    It noteworthy rpb2-100 isolated suppressor     aquaticus RNAPs (32) allow R512C replacement be <br /> the ssu72-2 Ts&#226;&#171;&#186; growth defect, rpb2-100 ssu72-2 exert         interpreted context T. aquaticus crystal structure. <br /> similar effects transcription. This anomaly explained       Moreover, comprehensive description site-specific inter- <br /> by results genetic analysis YDP19 revertant,          actions catalytically competent RNAP promoter <br /> revealing mutations genes, rpb2-100 shs2-1,       open DNA recently defined protein-DNA photo-cross- <br /> required suppression ssu72-2 Ts&#226;&#171;&#186; phenotype. In-            linking (31). Accordingly, R428 forms binding site <br /> deed, effect allele absence   double-stranded DNA downstream position &#226;&#171;&#185;1 is <br /> enhance ssu72-2 phenotype, essentially eliminating growth         positioned make direct contact phosphate 5&#226;&#172;&#732; to <br /> at 37&#194;&#176;C (Fig. 2). This result accounts similar       position &#226;&#171;&#185;6 nontemplate strand. The position this <br /> effects ssu72-2 rpb2-100 transcription          residue suggests possible roles R428 stabilizing RNAP- <br /> consistent synergistic effects transcription       DNA interaction, facilitating clamping RNAP duplex <br /> the PGK1 promoter (Fig. 6). Although isolated suppres-           DNA, facilitating RNAP translocation, and/or defining <br /> sor, rpb2-100 allele enhancer ssu72-2 absence     downstream boundary transcription bubble. Further- <br /> of shs2-1.                                                            more, catabolite gene activator protein effect on <br />    The rpb2-100 allele encodes single amino acid replace-           cross-linking RNAP core promoter region <br /> ment, R512C, located position homology            (31). These results readily account effects rpb2-100 <br /> block D Rpb2 (Fig. 3). The viability R512C mutant           noninduced transcription underscore premise that <br /> indicates R512, despite phylogenetic invariance,      Ssu72 core RNAP II transcription factor. <br /> functionally invariant. Not does rpb2-100 allele (in <br /> combination shs2-1) suppress ssu72-2 Ts&#226;&#171;&#186; phenotype,                                    ACKNOWLEDGMENTS <br /> but ssu72-2 rpb2-100 shs2-1 strain (YDP19) exhibits only <br />                                                                         We especially grateful Zu-Wen Sun constructing the <br /> marginally impaired growth rich (YPD) medium 30&#194;&#176;C.              ssu72-2 mutant Sung-Joon Kim Danny Reinberg their <br /> The viability rpb2-100 mutants attributed           generous gift purified yeast RNAP II. We grateful to <br /> suppressive effect ssu72-2 rpb2-100, rpb2-100        Richard Ebright, Danny Reinberg David Gross valuable dis- <br /> mutant viable SSU72&#226;&#171;&#185; background, growth          cussions comments manuscript Steve Buratowski, <br /> rate comparable YDP19 (Pappas, unpublished re-             Bryan Cullen, Leonard Guarente, Lucy Robinson, Hans-Joachim <br /> sults). Furthermore, rpb2-100 SSU72&#226;&#171;&#185; strain exhibits          Schu&#194;&#168;ller, Nancy Woychik strains plasmids. <br /> same severe Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes YDP19,                   This work supported NIH grant GM39484. <br /> confirming phenotypes R512C re- <br />                                                                                                        REFERENCES <br /> placement. <br />                                                                        1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990. <br />    Other rpb2 alleles isolated genetic selections            Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410. <br /> for mutations affect core promoter function (3, 4, 14, 19).       2. Archambault, J., J. D. Friesen. 1993. Genetics eukaryotic RNA <br /> The spt alleles RPB2 identified suppressors &#226;?&#166;-el-           polymerases I, II, III. Microbiol. Rev. 57:703&#226;&#8364;&#8220;724. <br /> ement insertions HIS4 LYS2 promoters (19). These            3. Archambault, J., D. B. Jansma, J. H. Kawasoe, K. T. Arndt, J. Greenblatt, <br />                                                                           J. D. Friesen. 1998. Stimulation transcription mutations affecting <br /> alleles encode amino acid replacements region              conserved regions RNA polymerase II. J. Bacteriol. 180:2590&#226;&#8364;&#8220;2598. <br /> homology blocks B C Rpb2 underscore role              4. Arndt, K. T., C. A. Styles, G. R. Fink. 1989. A suppressor HIS4 <br /> the Rpb2 subunit core promoter recognition. The sit1               transcriptional defect encodes protein homology catalytic sub- <br /> sit2 alleles RPB1 RPB2, respectively, isolated             <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> protein phosphatases. Cell 56:527&#226;&#8364;&#8220;537. <br />                                                                        5. Bell, S. D., S. P. Jackson. 1998. Transcription Archaea. Cold Spring <br /> enhancers HIS4 transcription absence trans               Harb. Symp. Quant. Biol. 63:41&#226;&#8364;&#8220;51. <br /> activators Gcn4, Bas1, Bas2 (4). These mutations com-          6. Berroteran, R. W., D. E. Ware, M. Hampsey. 1994. The sua8 suppressors <br /> parable rpb2-100 affect noninduced tran-             Saccharomyces cerevisiae encode replacements conserved residues <br /> scription, albeit opposite effects. Furthermore, sit1            largest subunit RNA polymerase II affect transcription start <br />                                                                           site selection similarly sua7 (TFIIB) mutations. Mol. Cell. Biol. 14:226&#226;&#8364;&#8220;37. <br /> and sit2 mutants, like rpb2-100 mutant, inositol auxo-         7. Blair, W. S., B. R. Cullen. 1997. A yeast TATA-binding protein mutant <br /> trophs. The sit1 alleles encode Rpb1 amino acid replacements              selectively enhances gene expression weak RNA polymerase II <br /> in region D region F, near          promoters. Mol. Cell. Biol. 17:2888&#226;&#8364;&#8220;2896. <br /> the active center RNAP (3). In Thermus aquaticus                8. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. A positive selection for <br />                                                                           mutants lacking orotidine-5&#226;&#172;&#732;-phosphate decarboxylase activity yeast: <br /> RNAP structure, sit1 region F replacements adjacent            5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197:345&#226;&#8364;&#8220;346. <br /> R428, counterpart R512 Rpb2. The opposite effects         9. Buratowski, S., M. Sopta, J. Greenblatt, P. A. Sharp. 1991. RNA poly- <br /> the sit1 rpb2-100 mutations transcription ac-             merase II-associated proteins required DNA conformation change <br /> counted proximity secondary channel,            transcription initiation complex. Proc. Natl. Acad. Sci. USA 88:7509&#226;&#8364;&#8220; <br />                                                                           7513. <br /> the distinction stimulatory inhibitory effects         10. Cramer, P., D. A. Bushnell, J. Fu, A. L. Gnatt, B. Maier-Davis, N. E. <br /> might accounted structural changes affect sub-             Thompson, R. R. Burgess, A. M. Edwards, P. R. David, R. D. Kornberg. <br /> strate access active site and, consequently, efficiency        2000. Architecture RNA polymerase II implications transcrip- <br /> of promoter clearance.                                                    tion mechanism. Science 288:640&#226;&#8364;&#8220;649. <br />                                                                       11. Darst, S. A., A. Polyakov, C. Richter, G. Zhang. 1998. Insights into <br />    A crystal structure yeast RNAP II recently solved            Escherichia coli RNA polymerase structure combination x-ray and <br /> a resolution 3 &#195;&#8230; (10). Unfortunately, R512 located           electron crystallography. J. Struct. Biol. 124:115&#226;&#8364;&#8220;122. <br /> an approximately 13-residue gap precludes interpretation         12. Darst, S. A., A. Polyakov, C. Richter, G. Zhang. 1998. Structural studies <br />  VOL. 20, 2000                                                                                                          Ssu72-RNAP II INTERACTION                       8351 <br />  <br />  <br />       Escherichia coli RNA polymerase. Cold Spring Harb. Symp. Quant. Biol.         35. Parvin, J. D., B. M. Shykind, R. E. Meyers, J. Kim, P. A. Sharp. 1994. <br />       63:269&#226;&#8364;&#8220;276.                                                                          Multiple sets basal factors initiate transcription RNA polymerase II. J. <br /> 13.   Edwards, A. M., C. M. Kane, R. A. Young, R. D. Kornberg. 1991. Two               Biol. Chem. 269:18414&#226;&#8364;&#8220;18421. <br />       dissociable subunits yeast RNA polymerase II stimulate initiation      36. Pinto, I., D. E. Ware, M. Hampsey. 1992. The yeast SUA7 gene encodes <br />       transcription promoter vitro. J. Biol. Chem. 266:71&#226;&#8364;&#8220;75.                      homolog human transcription factor TFIIB required normal <br /> 14.   Fan, H. Y., K. K. Cheng, H. L. Klein. 1996. Mutations RNA                 start site selection vivo. Cell 68:977&#226;&#8364;&#8220;988. <br />       polymerase II transcription machinery suppress hyperrecombination mu-        37. Pinto, I., W.-H. Wu, J. G. Na, M. Hampsey. 1994. Characterization of <br />       tant hpr1 Delta Saccharomyces cerevisiae. Genetics 142:749&#226;&#8364;&#8220;759.                   sua7 mutations defines domain TFIIB involved transcription start site <br /> 15.   Forsburg, S. L., L. Guarente. 1989. Identification characterization          selection yeast. J. Biol. Chem. 269:30569&#226;&#8364;&#8220;30573. <br />       HAP4: component CCAAT-bound HAP2/HAP3 heteromer.               38. Roeder, R. G. 1996. The role general initiation factors transcription by <br />       Genes Dev. 3:1166&#226;&#8364;&#8220;1178.                                                              RNA polymerase II. Trends Biochem. Sci. 21:327&#226;&#8364;&#8220;335. <br /> 16.   Goodrich, J. A., R. Tjian. 1994. Transcription factors IIE IIH       39. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. Fink. 1987. A <br />       ATP hydrolysis direct promoter clearance RNA polymerase II. Cell 77:              Saccharomyces cerevisiae genomic plasmid bank based centromere- <br />       145&#226;&#8364;&#8220;156.                                                                             containing shuttle vector. Gene 60:237&#226;&#8364;&#8220;243. <br /> 17.   Hampsey, M. 1998. Molecular genetics RNA polymerase II general            40. Rothstein, R. 1991. Targeting, disruption, replacement, allele rescue: <br />       transcriptional machinery. Microbiol. Mol. Biol. Rev. 62:465&#226;&#8364;&#8220;503.                    integrative DNA transformation yeast. Methods Enzymol. 194:281&#226;&#8364;&#8220;301. <br /> 18.   Hampsey, M. 1997. A review phenotypes Saccharomyces cerevisiae.            4 <br /> </body></html>