<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> DOMINANT SUPPRESSORS OF A MUSCLE MUTANT DEFINE AN <br />       ESSENTIAL GENE OF CAENORHABDZTIS ELEGANS <br />  <br />                        IVA S. GREENWALD           AND   H. ROBERT HORVITZ <br />       Department Biology, Massachusetts Institute Technology, Cambridge M A 02139 <br />  <br />                                    Manuscript received October 14,1981 <br />                                    Revised copy accepted March 5, 1982 <br />  <br />                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />  <br />               The sup-I1 I locus C. elegans defined rare dominant suppressors <br />           unc-93(e1500) I I I , <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> affects muscle structure. All of <br />           dominant suppressors recessive &#226;&#8364;&#339;scrawny&#226;&#8364;? <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>phenotype</span>. Two addi- <br />           tional classes sup-I1 alleles identified. One class, null alleles, was <br />           obtained reversion dominant suppressor activity. These null alleles <br />           recessive embryonic lethals, indicating sup-I1 essential gene. <br />           Members second class, rare semidominant revertants &#226;&#8364;&#339;scrawny&#226;&#8364;? <br />           phenotype, partial suppressors unc-93(e1500). The genetic properties <br />           dominant suppressor mutations suggest rare missense <br />           mutations confer novel activity sup-I1 protein. We consider <br />           ways sup-11 alleles suppress unc-93(e1500), including <br />           possibilities altered sup-I1 proteins restore function protein <br />           complex modified products gene member unc-93 <br />           gene family. <br />  <br />   N paper, genetic analysis s u p - l I locus Caeno- <br /> I rhabditis elegans. This study comprises continued characterization <br /> of muscle mutant unc-93(ei500) 111: sup-li locus defined alleles <br /> that: dominant suppressors unc-93 (e25OO). We undertook isolation and <br /> characterization dominant suppressors extension initial study <br /> of unc-93(e1500), defined extragenic suppressor loci reces- <br /> sive, apparently null, mutations (GREENWALD     HORVITZ    1980). We thought <br /> that dominant suppressor mutations define new muscle genes. We also <br /> hoped study dominant suppressors lead better understand- <br /> ing unc-93 locus. <br />    We previously identified visible unc-93 alleles (ej500,n200) confer a <br /> characteristic uncoordinated, egg-laying defective phenotype. We determined <br /> that e1500 72200 mutations rare alleles probably result the <br /> synthesis toxic unc-93 proteins. Null alleles unc-93, eliminate the <br /> toxic products, generated intragenic revertants visible al- <br /> leles. These null alleles confer visible phenotype, i.e., null phenotype of <br /> the unc-93 locus wild-type. We suggested reasonable explanation the <br /> wild-type null phenotype exists protein substitute for <br /> the missing unc-93 protein; specifically, proposed U&#226;&#8364;&#339;-93 mem- <br /> Genetics 101: 211-225 June, 1982 <br />  212                            I. S. GREENWALD A N D H. R. HORVITZ <br />  <br /> ber gene family substituent function provided another <br /> member putative unc-93 gene family. This proposal, discuss <br /> further below, implications interpretation observations con- <br /> cerning sup-11 locus. <br />  <br />                                      MATERIALS A N D METHODS <br />  <br />     General methods: General methods handling, culturing ethyl methanesulfonate <br /> (EMS) mutagenesis C. elegans described BRENNER(1974). Experiments were <br /> performed 20&quot; unless indicated. <br />     Strains genetic nomenclature: Caenorhabditis elegans var. Bristol strain N2 wild- <br /> type parent strains used n work. <br />     The sup-11 I , sup-9 I I , unc-93 111, sup-IO X alleles studied listed Table 1. Other <br /> mutations used listed below; isolated BRENNER(1974). The informational <br /> suppressors sup-5 111 sup7 X described WATERSTON         BRENNER    (1978) and <br /> WATERSTON     (1980). Two mutants isolated characterized laboratory: reces- <br /> sive bivulva mutation Zin-I7(n677) I maps approximately 12% left dpy-5, the <br /> semidominant multivulva mutation lin-l2(n137) I11 tightly linked unc-32 (C. FERGU- <br /> SON,personal communication). <br />    The mutations are: <br /> LGI: lin-I7(n677), unc-11(e47), dpy-5(e61) <br /> LGII: dpy-lO(e128) <br /> EGIII: dpy-I7(e164), lon-l(e185), sup-5(e1464), lin-12(n137), dpy-lS(e364) <br />  <br />                                                  TABLE 1 <br />                                 Some genes alleles used study <br />  <br />   Gene                         Allele(s)                     Reference allele    Symbol <br />  <br /> sup-I1 I            n187, n401, n402, n403,                       n403          sup-IZ(d) <br />                     n404, n405, n616, n628, <br />                     n710, n711 <br />                     n403 n406,      n403 n425,                 n403 n406        sup-I1( r ) <br />                     n403 n426,      n403 n525, <br />                     12403 n526,     n403 n712, <br />                     n403 n713,      n403 n714 <br />                     n403 n681, n403 n682,                      n403 n682        sup-I1 ( 0 ) <br />                     n403 n68?, n403 n684 <br /> sup-9 I I           n180 <br /> unc-93 111          eI500*, n200* <br />                     e1500n2341                                                  unc-93 ( 0 ) <br /> sup-10 x             n183 <br />  <br />    The sup-9, unc-93 sup-10 mutations, sup-I1 (n187) described previously (GREEN- <br /> WALD   HORVITZ  1980). All sup-11 mutations generated study. The <br /> dominant suppressor alleles generated mutagenesis unc-93 (el500) described in <br /> MATERIALS AND METHODS. The sup-I1 alleles named double mutants they <br /> were generated sup-ll(n403) and, cases, original n403 mutation probably <br /> retained. <br />    * Visible unc-93 alleles. <br />   1 unc-93 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null allele</span>   (no visible phenotype). <br />                               THE SUP-I   I LOCUS OF C . elegans                             213 <br /> LGIV: dpy-l3(e184) <br /> LGV: dpy-11(e224) <br /> LGX: dpy-7(e1324), sup-7(st5) <br />     This paper conforms standard nomenclature C. elegans genetics (HORVITZ         et al. <br /> 1979). <br />     Isolation dominant suppressors unc-93 (e1500) : Ten dominant suppressor alleles of <br /> sup-I1 [&#226;&#8364;&#339;sup-l1(d)&#226;&#8364;?] independently isolated. Six alleles obtained single <br /> experiment distributing EMS-mutagenized unc-93(e1500) hermaphrodites mixed ages <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> 33 100 mm Petri plates screening 106-107 total progeny animals improved mobility <br /> after three, days (i.e., generation). Three sup-Il(d) alleles were <br /> identified additional reversion experiments. One sup11 ( d ) mutation (n187) obtained <br /> in initial set revertants (GREENWALD     HORVITZ   1980). <br />     All sup-ll(d) alleles recessive scrawny (Scr) phenotype. <br />     Mapping dominmt suppressors unc-93 (e1500) : The reference allele sup-ll(n403) was <br /> shown autosomal, linked t o dpy-5 I (approximate recombination frequency p = 8 % ) and <br /> unlinked markers dpy-IO I I , unc-93(e1500) I I I , dpy-13 IV dpy-11 V; other <br /> dominant suppressor mutations linked dpy-5 I (data shown). <br />     Standard three-factor crosses (BRENNER1974) performed utilizing recessive scrawny <br />  (Scr) phenotype sup-ll(n403). <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> heterozyygote     +   unc-I1 dpy-5/sup-ll  + +,    15/15 <br /> Dpy non-Unc recombinants segregated sup-11. From heterozygote f unc-11 +/sup-ll 4- <br /> dpy-5, 7/18 Dpy non-Scr recombinants segregated unc-11. From heterozygote En-17 4- <br /> dpy-5/+ sup-11 f, 7/13 Lin non-Dpy recombinants segregated sup-11. Thus, map order <br /> in region LGI lin-I7 sup-I1 unc-11 dpy-5. The lin-17 unc-I1 closest flanking <br /> genes identified date. <br />     Complementation tests: (a) W e previously demonstrated classes unc-93(el500) <br /> revertants relatively common afer EMS mutagenesis (GREENWALD          HORVITZ  1980) j one <br /> class comprised null alleles unc-93, classes comprised alleles of <br /> the extragenic suppressors sup-9 I1 sup-IO X. These classes recessive, allowing <br /> rapid identification sup-9, sup-10 unc-93 null alleles complementation tests with <br /> three canonical revertants: (1) unc-93(e1500 n234); ( 2 ) sup-9; e1500; ( 3 ) e1500; sup-10. <br /> Males of genotypes crossed revertant hermaphrodites unknown <br /> genotype. Most recessive suppressors fail complement tester strains (yielding only <br /> wild-type cross progeny) complement (yielding Unc(e1500) cross progeny). <br /> Dominant suppressors, sup-ll(d), yield wild-type progeny crosses. <br /> Recessive suppressors sup-9, sup-10 o r unc-93 null alleles, sup-ll(r) (see <br /> below), complement tester strains yield Unc(e1500) cross progeny. <br />      (b) All sup-ll(d) mutations described study result recessive Scr phenotype. <br /> To demonstrate dominant suppressor mutations belong single complementation <br /> group, complementation tests Scr phenotype performed reference sup-ll(d) <br /> mutation sup-I1 (n403). Males heterozygous dominant suppressor mutation crossed <br /> with sup-Il(n403) dpy-5; unc-93(e1500) hermaphrodites. The appearance Scr non-Dpy <br /> cross progeny indicated dominant suppressor allele sup-11. <br />     (c) The recessive Scr phenotype sup-llld) mutations enabled complementation <br /> test recessive lethal allele sup-ll(n403 n682), appears null allele; this <br /> complementation test demonstrated sup-ll(d) mutations belong single com- <br /> plementation group. Males heterozygous sup-ll(d) mutation crossed bal- <br /> anced heterozygote supdl(n403 n682) dpy-5/uncc-ll,and mating animals transferred <br /> to fresh Petri plates daily. The appearance Scr non-Dpy cross progeny indicated the <br /> dominant suppressor allele sup-11. <br />     (d) To demonstrate putative sup-I1 null alleles (&#226;&#8364;&#339;sup-21(0)&#226;&#8364;?) belong a <br /> single complementation group, complementation tests recessive lethality performed. <br /> Balanced hermaphrodite heterozygotes genotype sup-ll(0) dpy-5/unc-11 crossed with <br /> wild-type males, cross progeny males individually mated (1) unc-11 dpy-5 <br /> hermaphrodites 24 hr ( 2 ) sup-li(n403 n682) dpy-5/unc-l1 hermaphrodites <br />  2144                            I. S. GREENWALD A N D H. R . HORVITZ <br />  <br /> f o r 24 hr. The appearance cross progeny Dpy non-Unc males mating (1) lack <br /> of appearance Dpy bales males produced mating (2) individual male <br /> indicated s u p - l l ( 0 ) allele failed complement sup-ll(n403 n682) recessive <br /> lethality. <br />        (e) The &#226;&#8364;&#339;sup-ll(r)&#226;&#8364;? revertants Scr phenotype sup-ll(n403) recessive partial <br /> suppressors e2500: larvae similar Unc(e1500) larvae, adult hermaphrodites lay <br /> eggs improved mobility. To demonstrate sup-ll(r) revertant n403 n406 fails <br /> to complement sup-ll(0) alleles, s u p - l l ( 0 ) dpy-5/++; lin-l2/+ males crossed with <br /> s u p - l l ( r ) dpy-5; unc-93(el500) hermaphrodites Dpy Lin hermaphrodites picked <br />  [genotype sup-ll(0) dpy-5/sup-ll(r) dpy-5; unc-93(el500)                  +/+             lin-121. Several non-Lin <br /> progeny hermaphrodites picked; partially suppressed pheno- <br /> type. Approximately 2/3 partially suppressed hermaphrodites segregated 1/4 arrested <br /> embryos presumed sup-ll(0) dpy-5/sup-ll(r) dpy-5; unc-93jel500): some <br /> cases genotype verified crossing sup-ll(n403)/+ males observing Scr <br /> cross progeny. Approximately 1/3 partially suppressed hermaphrodites failed segregate <br /> arrested embryos presumed s u p - l l ( r ) dpy-5/sup-ll(r) dpy-5; unc-Y3(el500); <br /> in cases genotype verified crossing hermaphrodites sup-ll(n403)/+ <br /> male$ observing Scr cross progeny. The s u p - l l j 0 ) alleles used were <br /> n403 n681 n403 n682. <br />        (f) T o demonstrate sup-ll(r) alleles fail complement partial suppressor <br /> activity, reference allele sup-ll(n403 n406) chosen. Males genotype sup-ll(r)/+; <br /> unc-93(el500)/+ crossed sup-ll(n403 n406) dpy-5; unc-Y3(el500) hermaphrodites <br /> and 12 cross progeny Unc(el500) non-Dpy larvae picked. That approximately 1/2 of <br /> these larvae developed partially suppressed adults indicated sup-11( r ) allele failed <br /> t o complement sup-ll(n403 n406) partial suppressor activity. <br />        ( g ) The s u p - l l ( r ) alleles, derived s u p - l l ( d ) allele n403, are <br /> semidominant revertants n403, i.e., n40?/sup-ll(r) hermaphrodites non-Scr although <br />  scmewhat smaller wild-type. To determine phenotype various s u p - l l { d ) / s u p - l l ( r ) <br /> hermaphrodites, sup-ll(d)/+ ; e2500/+ males crossed sup-ll(r) ; e1500; dpy-ll <br /> hermaphrodites. The lack appearance Scr non-Dpy hermaphrodites indicated sup- <br /> l l { d ) / s u p - l l ( r ) non-Scr, non-Scr hermaphrodites smaller wild type <br />  [cf. n403/sup-ll(r)] generally seen. Heterozygotes s u p - l l ( r ) alleles (n403 <br /> n406, n403 n425, n403 n426) s u p - l l ( d ) alleles constructed. <br />       Suppression studies sup-5 111 sup-7 X: The sup-5(el464) 111 suppressor appar- <br /> ently specific subclass null alleles genes (WATEXSTON                      BRENNER    1978). We <br /> tested recessive lethal alleles s u p - l l , referred sup-ll(O), suppression sup-5. <br /> Males genotype lon-2 sup-5/++ mated hermaphrodites genotype sup-ll(0) <br /> dpy-5/sup-ll(n403) dpy-5, cross progeny hermaphrodites genotype s u p l l ( 0 ) dpy- <br /> 5/f+; lon-l sup-5/++ recognized segregation Lon animals, arrested embryos, <br /> and ozcasional Dpy recombinants. At 16 Lon progeny picked individual Petri <br /> dishes: approximately 2/3 sup-ll(0) dpyJ/++; lon-l sup-5 and <br />  1/3     ++/++;               Zon-2 sup-5. Two s u p - l l ( 0 ) alleles, n403 n682 n403 n683 <br /> were suppressed sup-5: approximately 2/3 Lon hermaphrodites segregated 1/4 <br /> arrested embryos occasional Dpy recombinants (dpy-5 epistatic lon-I). One sup-ll(0) <br /> allele, 72403 n682, appeared partially suppressed sup-5: virtually Lon hermaph- <br /> rodites segregated arrested embryos, approximately 2/3 Lon hermaphro- <br /> dites segregated 1/4 arrested larvae. Based cell lineage criteria (SULSTON                         HORVITZ <br /> 1977), larvae arrested mid-L1 stage. The proportion arrested embryos arrested <br /> larvae indicated copies sup5 necessary suppress embryonic lethality of <br /> sup-ll(n40.3 n681): 532 total progeny n403 n681 dpy-5/++;                                  lon-l sup-5/++ <br /> hermaphrodites, 96 arrested embryos 21 arrested larvae, similar the <br /> values expected (3/16 arrested embryos, 1/16 arrested Ll&#226;&#8364;&#8482;s) copies sup-5 neces- <br /> sary development proceed L1 stage. <br />                               THE SUP-III LOCUS OF       C . elegans                      215 <br />    Like sup-5(ef464) Ill, suppressor sup-7(st5) X apparently specific null alleles <br /> (WATERSTON   1980) ; alleles suppressed suppressors (WATERSTON      1980; <br /> HORVITZ  SULSTON1980). W e tested suppressor suppression sup-ff(n403 n681). <br /> At 25&quot;, males genotype sup-ff(n403 n 6 8 f ) dpy-5/++ crossed dpy-7 sup7 her- <br /> maphrodites cross progeny hermaphrodites genotype sup-f f (n403 n68f) d p y J / + + ; <br /> dpy-7 sup-7/++ recognized segregation occasional Dpy-5 progeny. From these <br /> hermaphrodites, Dpy-7 self-progeny picked, sup-ff (n403 n 6 8 f ) dpy-5/++; dpy-7 <br /> sup-7 animals recognized segregation occasional Dpy-5 progeny. These her- <br /> maphrodites allowed lay eggs 25&quot; 15&quot; (sup-7 suppression stronger at <br /> 15&quot; 25&quot;; WATERSTON    1980). At temperatures, approximately 1/4 arrested larvae <br /> were seen; larvae arrested mid-L1 stage. <br />  <br />                                             RESULTS <br />  <br />   Dominant suppressors unc-93 (e1500) : Ten dominant suppressors unc- <br /> 93 (e1500) isolated F, progeny EMS-mutagenized hermaphrodites as <br /> described MATERIALS AND METHODS. These revertants, picked as <br /> motile hermaphrodites relatively normal body size, heterozygous for <br /> dominant suppressor mutations result recessive &quot;scrawny&quot; (Scr) pheno- <br /> type (Figure l ) : non-Scr non-Unc hermaphrodite revertants segregated ap- <br /> proximately 2 non-Scr non-Unc: 1 Scr non-Unc: 1 Unc ( e l500). The Scr non-Unc <br /> animals homozygous suppressor, smaller thinner wild- <br /> type, reduced brood size, vigorous lay eggs. The presence <br /> of dominant suppressor mutations confirmed appropriate crosses [com- <br /> plementation test (a), MATERIALS AND METHODS]. One dominant suppressor <br /> mutation, 4 0 3 , mapped define new gene, sup-12 I (see MA- <br /> TERIALS AND METHODS data). The dominant suppressor mutations <br /> appear alleles sup-11: linked dpy-5 I fail complement <br /> n403 Scr phenotype [complementation test (b), MATERIALS AND METH- <br /> ODS]; fail complement sup-ll(n403 n682), null allele be <br /> discussed below, Scr phenotype [complementation test (c), MATERIALS <br /> AND METHODS]. The Scr phenotype sup-ll(n403) does depend the <br />  <br />  <br />  <br />  <br />    FIGURE <br />         1.-Bright-field      photomicrographs mutants described paper. <br /> (a) wild type, ( b ) unc-93(ef500), (c) sup-ff(n403), (d) sup-ff(n403n406). Bar = 0.1 mm. <br />  216                   I. S .   GREENWALD A N D H.   R. HORVITZ <br /> unc-93 genotype, i.e., sup-ll(n403); unc-93 animals Scr unc-93 <br />       +, <br /> alleles e1500, e1500 n234 (an unc-93 null allele). <br />    The sup-11 alleles differ efficiency suppression e1500. Eight <br /> alleles. including reference allele n403, strong suppressors; alleles <br /> (n628, n710) weaker suppressors. The phenotype sup-11 (n403)/+; U ~ C - <br /> 93 (el500) animals resembles e1500/+: hermaphrodites motile and <br /> lay eggs, completely wild-type, males vigor- <br /> ous e1500 homozygotes wild-type movement gen- <br /> erally unable mate. Hermaphrodites genotype sup-11 (n628)/+; uric-93 <br /> (e1500) sup-11(n710)/+; u1nc-93(e15O0)are longer uncoordinated <br /> than sup-11 (n403)/+; unc-93 (e1500) hermaphrodites. <br />    The dominant suppressors [&#226;&#8364;&#339;sup-11(d)&#226;&#8364;?] tested suppression the <br /> other unc-93 visible allele, n200, phenotype similar severe <br /> than e1500. Hermaphrodites genotype sup-21 ( d ) ; unc-93(n200) dpy- <br /> 17 crossed unc-93 (n200)/dpy-17 males cross progeny non-Dpy <br /> hermaphrodites [genotype sup-11(d)/+; unc-93 (n200) dpy-17/unc-93 (n200) <br /> +]  examined suppression. All sup-11 (d) mutations suppressed unc- <br /> 93 (n200). <br />    Dominant suppressors e1500 relatively rare: The frequency which <br /> dominant suppressors generated EMS mutagenesis indica- <br /> tion nature mutational event: low frequency suggest rela- <br /> tively specific event; high frequency suggest relatively nonspecific <br /> event, generation null alleles. Dominant suppressor mutations ap- <br /> peared relatively rare EMS mutagenesis (GRBENWALD        HORVITZ <br /> 1980; MATERIALS AND METHODS above). <br />    We compared directly frequency dominant suppressors are <br /> generated class mutations known frequency. W e previously de- <br /> termined EMS mutagenesis unc-93 null alleles occur frequency <br /> of 5 x 10-i/mutagenized gamete, mutations an <br /> average C. elegans gene (BRENNER        1974; MENEELY   HERMAN      1979). We <br /> have compared directly frequency dominant suppressors unc- <br /> 93 null alleles generated. We dominant suppressors 21 unc-93 <br /> null alleles using protocol shown Figure 2, indicating dominant sup- <br /> pressors rare EMS mutagenesis occur 1/20 aver- <br /> age frequency null mutations. This relatively low frequency suggested that <br /> the dominant suppressors unc-93(e1500) result null mutations. <br />    The null phenotype sup-11 recessiue lethal: Null alleles genes have <br /> been obtained reversion dominant &#226;&#8364;&#339;neomorphic&#226;&#8364;? mutations (which result <br /> in acquisition novel activities) &#226;&#8364;&#339;hypermorphic&#226;&#8364;? mutations (which re- <br /> sult overproduction gene activity) genes (e.g., LIFSHYTZ    and <br /> FALK1969; LIFSHYTZ       GREEN1979). Since dominant suppressor alleles of <br /> sup-11 appeared likely null alleles, possible reversion of <br /> the dominant suppressor activity sup-11 (n403) generate sup-11 null <br /> alleles (&#226;&#8364;&#339;sup-ll(0)&#226;&#8364;?).We generated revertants dominant suppressor <br /> activity sup-11 (n403) detailed Figure 3. All recessive lethal muta- <br />                              THE   sup-i II LOCUS     OF   C. elegans                       217 <br />  <br />  <br />  <br />  <br />                                        U   + 6&apos;8 x        u(O)mff <br />  <br />                       SELF:                 U( 0)m/u( <br />                                                      1      0)m        M <br />                    CROSS:                        U   +/u(O) m          Unc <br />                                            - <br />                                            0)+ /u(O) m <br />                                             U(                         WT <br />                                   s(d)/+ ; U +/u(O) m                  WT <br />  <br />     FIGURE    &amp;.-Schematic representation experiment determine relative frequencies <br /> after EMS mutagenesis dominant suppressors unc-93(e1500) unc-93 null alleles. <br /> Lower case letters indicate genotypes: U = unc-93(el500), u(0) = unc-93 null allele, m = a <br /> marker linked unc-93, s(d) = dominant suppressor unc-93(e1500). Underlining indicates <br /> mutations expected result mutagenesis. Upper case letters indicate phenotypes, i.e., <br /> M I=phenotype m, Unc = uncoordinated, WT I=wild type. <br />     According scheme, males homozygous unc-93(e1500) [U] muta- <br /> genized EMS crossed hermaphrodites homozygous unc-93(e1500 n234) [u(O)] <br /> and linked marker, unc-32 [m]. Self progeny Unc-32 phenotype, <br /> and cross progeny Unc(ei500) phenotype, null alleles uric-93 <br /> are recessive e1500. Rare non-Unc(e1500) cross progeny expected; be <br /> candidates carrying new unc-93 null alleles [s(O)] dominant suppressor mutations <br /> [ ~ ( d ) ] Hermaphrodite <br />             .             candidates carrying new unc-93 null alleles segregate Unc <br /> (e1500) progeny. Hermaphrodites carrying dominant suppressor mutations segregate <br /> Unc(e1500) progeny, sup-11 heterozygotes expceted segregate Scr well. <br />     This experiment performed essentially represented above. However, unc-93 <br /> (d500) males mate, recessive suppressor e1500 (either sup-9 sup-10) included <br /> to enable mating; suppressors recessive, phenotypes cross progeny <br /> as predicted scheme affected. Twenty-three independent non-Unc(e1500) <br /> hermaphrodite candidates obtained estimated 25,000 cross progeny hermaphro- <br /> dites; 21 fertile candidates, segregated Unc(e1500), appropriate comple- <br /> mentation tests [complementation test (a), MATERIALS AND METHODS] confirmed all <br /> contained unc-93 null alleles. <br />  <br /> tions tightly linked sup-ll(n403). For lethal mutation n682, Scr re- <br /> combinants appeared 926 total progeny n403 n682 dpy-51 <br /> unc-ll hermaphrodites (recombination frequency p &lt; 0.2%) ; addition, Scr <br /> animals seen maintenance balanced heterozy- <br /> gous stocks. <br />    The recessive lethality sup-ll(n403 n682) complemented the <br /> other recessive lethal mutations generated reverting sup-ll (n403) <br />  [complementation test (d) , MATERIALS AND METHODS], indicates that <br /> these mutations allelic. All s u p - l l dominant suppressor alleles fail to <br /> complement lethal mutation n403 n682 Scr phenotype [complementa- <br /> tion test (b), MATERIALS AND METHODS] ,suggesting lethal mutations are <br /> alleles sup-ll. The phenotype arrested embryos similar alleles. <br />  218                        I. S. GREENWALD A N D H.      R. H O R V I T Z <br />  <br />  <br />  <br />  <br />                               ++;ut&amp;          x <br />                                                    EMS   L <br />                                                    s(d)m;uOrO&quot; <br />                                                1                            Scr M <br />                    SELF:             s(d) m/s(d)m;u <br />                 CROSS:                  + +/s(d)m;u                         WT <br />                                                 - <br />                                         + +/s(O)m;u&apos;                        UnC <br />  <br />  <br />     FIGURE     3.-Schematic representation experiment revert dominant suppressor <br /> activity sup-ll(n403). Lower case letters indicate genotypes: U = unc-93(e1500), s(d) = <br /> sup-ll(n403), m = marker linked sup-11, s(0) = putative null allele sup-11 would <br /> be generated experiment. Upper case letters indicate phenotypes Figure 2, with <br /> the addition Scr = scrawny. <br />     According scheme, males homozygous unc-93(e1500) mated to <br /> EMS-mutagenized hermaphrodites homozygous sup-ll(n403), linked marker as <br /> dpy-5, unc-93(e1500). Self progeny Scr Dpy cross progeny be <br /> wild type dominant suppressor activity sup-ll(n403). Rare cross progeny of <br /> the characteristic Unc (e1500) phenotype candidates harboring heterozygous <br /> sup-I1 null alleles. The rare Unc(e1500) cross progeny hermaphrodites segregate homo- <br /> zygotes putative sup-11 null alleles recognized linked dpy-5 marker. <br />     This experiment performed essentially represented above; however, unc-93 <br /> je1500) males mate. sup-10, recessive suppressor e1500, included (see Figure 2 <br /> legend). Four independent Unc (e1500) hermaphrodite candidates obtained ap- <br /> proximately 9,000 cross progeny. Each failed segregate Scr Scr Dpy segre- <br /> gated dpy-5 homozygotes, suggesting lethal mutation linked sup-11 <br /> and dpy-5 generated (the dpy-5 homozygotes presumably recombinants). <br /> Progeny candidates mated lin-17 dpy-5/++          unc-11 dpy-5/++ males <br /> to recover chromosome containing putative recessive lethal mutation, Dpy <br /> hermaphrodites crosses mated unc-ll/+ males establish balanced <br /> strain s u p - l l ( 0 ) dpy-5/unc-l1 (the markers eliminated segregation). These <br /> strains segregated 1/4 arrested embryos occasional Dpy-5 recombinants. <br />  <br /> The arrest occurs onset morphogenesis; embryos abnormal, <br /> but nonetheless pharyngeal, intestinal tail structures evident (Figure 4). <br />    We tested recessive lethal mutations suppression sup-5 111 <br />  (see MATERIALS AND METHODS) ,which informational suppressor specific for <br /> null alleles (WATERSTON    BRENNER1978) encodes amber suppressor <br /> tRNA (R. WATERSTON,      N. WILLS,and R. GESTELAND,      personal communication; <br /> J. KARN,personal communication). OEe allele, sup-ll(n403 n682), par- <br /> tially suppressed sup-5 homozygotes: 20&#194;&#176;, animals genotype sup-11 <br /> (72403 72681) dpy-5; sup-5 arrest L1 larvae (Figure 4 ) . We tested sup-ll <br /> (n403 72681) suppression sup-7 X (see MATERIALS AND METHODS), another <br /> informational suppressor (WATERSTON       1980) suppresses alleles as <br /> s u p 5 (WATERSTON   1980; HORV~TZ    SULSTON   1980; MENEELY    HERMAN <br /> 1981) encodes amber suppressor tRNA (R. WATERSTON,           N. WILLS <br /> and R. GESTELAND,   personal communication; J. KARN,personal communication). <br />                             THE   sup-i I LOCUS OF C . elegans                          219 <br />  <br />  <br />  <br />  <br />    FIGURE4.-Photomicrographs arrested individuals homozygous sup-11 null alleles; <br /> NOMARSKI   differential interference contrast optics. (a) Arrested embryo. sup-ll(n403 d 8 l ) <br /> dpy-5, (h) Arrested L1 larva, sup-fl(n403 n681) dpy-5; sup5 grown 20&apos;. Bar = 0.01 mm. <br /> Note: phenotype sup-fl(n403 n681) d p y - 5 ( + ) the phenotype of <br /> sup-fl(n403 n481) dpy-5 (not shown). <br />  <br /> The sup-ZZ(n403 n682) dpy-5; sup-7 animals arrest Ll larvae 25&quot; and <br /> 15&quot;. We sup-ZZ(d) suppressor activity weakly re- <br /> stored sup-ZZ(n403 n682) dpy-5/++; unc-93(e2500) sup-5 hermaphrodites, <br /> which slightly better unc-93(eZ500) hermaphrodites remain <br /> very egg-laying defective; addition. sup-22 (n403 n682) dpy-5; unc-93 <br /> (e2500) sup-5-arrested larvae motile sup-ZZ(n403 n682) dpy-5; <br /> sup5-arrested larvae. <br />    These recessive lethal mutations appear null alleles sup-22: cis, <br /> they abolish dominant suppressor activity sup-ZZ(n403) trans <br /> they fail complement sup-22 (n403) recessive scrawny phenotype. In <br /> addition, arise EMS mutagenesis t frequency 4 x lo4, is <br /> similar frequency 5 x lo-&apos; expected mutations eliminate gene <br /> activity (BRENNER      1974; MENEELYand HERMAN       1979; GREENWALD     HOR- <br /> VITZ 1980). Finally, alleles, sup-ll(n403 n682), suppressible by <br /> the null allele-specific informational suppressors sup-5 sup7. <br />    Rare sup-11 alleles recessive partial suppressors unc-93 (el 500) : The <br /> F, F, progeny EMS-mutagenized sup-22 (n403); unc-93(e2500) her- <br /> maphrodites examined non-Scr revertants. Five revertants ob- <br /> tained; revertants, t detected F, non-Scr <br /> motile individuals, rare (estimated frequency lo-&quot;). Approxi- <br /> mately 3/4 progeny revertants new phenotypic class <br /> and proved homozygous partial suppressors [&quot;sup-IZ(r)**]of unc-93 <br />  (e2500): young larvae phenotype similar unc-93(e2500) <br /> larvae, adults lay eggs motile. These partially suppressed hermaphro- <br /> dites crossed wild-type males, reversion site proved be <br /> tightly linked n403: Scr hermaphrodites detected 300 <br /> Fi progeny revertant. For canonical revertant n403 n406, motile <br /> recombinant larvae detected approximately 68,000 progeny n403 <br /> n406 +/ 4- dpy-5; e2500 (recombination frequency p &lt; 0.003%). <br />    Three revertants Scr phenotype sup-22 (n403); unc-93(+) also <br /> generated EMS. As revertants described above, revertants <br /> are semidominant suppressors n403 partial suppressors e2500, the <br />  reversion site linked n403. <br />  220                        I. S. GREENWALD A N D H .   R. HORVITZ <br />     The tight linkage demonstrated 12403 n406 suggested these <br /> revertants intragenic. A complementation test [ (e), MATERIALS AND METH- <br /> ODS] n403 n406 s u p - l l ( 0 ) supports interpretation: hermaphro- <br /> dites genotypes n403 n406/sup-12(0); unc-93(el500) partially sup- <br /> pressed phenotype similar n403 n406; unc-93(el500) distinct the <br /> Unc(l500) phenotype +/sup-ll ( 0 ) ; unc-93 (el500) n403 n 4 0 6 / f ; unc- <br /> 93(el500), i.e., null allele sup-ll fails complement n403 n406 f o r re- <br /> cessive suppressor activity. <br />     The linked revertants generated reversion Scr phenotype of <br /> s u p - l l (n403); unc-93(eI5OO) sup-ll(n403);         +       fail complement sup-ll <br /> (n403 n406) partial suppressor activity probably mu- <br /> tations sup-ll [complementation test (f) ,MATERIALS AND METHODS]. In addi- <br /> tion, sup-ll ( d ) / s u p - l l ( r ) heterozygotes constructed ten <br /> s u p - l l ( d ) alleles, s u p - l l ( r ) alleles non-Scr [comple- <br /> mentation tests (g) ,MATERIALS AND METHODS]. <br />     The phenotype sup-ll (n403 n406); unc-93 (+) essentially wild type, in- <br /> dicating uncoordinated phenotype sup-11(n403 n406); unc-93 <br /> (el500) results el500 mutation. In addition, reversion partially <br /> uncoordinated phenotype s u p - l l (n403 n406); unc-93( e l 500) yields sup-9, <br /> sup-10 unc-93 null alleles average frequency 5 X 104/mutagenized <br /> gamete, establishing suppressors el500 suppress uncoordinated pheno- <br /> type sup-12 (n403 n406); unc-93(e1500). <br />  <br />                                         DISCUSSION <br />  <br />    W e identified classes sup-ll I alleles. The sup-ll locus de- <br /> fined relatively rare dominant suppressors [&#226;&#8364;&#339;sup-ll(d)&#226;&#8364;?] unc-93 (el500), <br /> an uncoordinated egg-laying defective muscle mutant described pre- <br /> viously ( GREENWALD           HORV~TZ   1980). These dominant suppressors re- <br /> sult recessive &#226;&#8364;&#339;scrawny&#226;&#8364;? (Scr) phenotype. Reversion Scr phenotype of <br /> sup-ll (n403) generated class relatively rare sup-11 alleles [&#226;&#8364;&#339;sup-ll(r)&#226;&#8364;?] <br /> that recessive partial suppressors e1500. Reversion dominant sup- <br /> pressor activity sup-11(n403) generated class relatively common sup-ll <br /> alleles [&#226;&#8364;&#339;sup-ll(U)&#226;&#8364;?] recessive lethal appear null alleles. <br /> Table 2 summarizes phenotypes hermaphrodites various sup-ll geno- <br /> types. I n discussion, consider classes sup-ll <br /> mutations. W e consider nature interaction sup-ll and <br /> unc-93. <br />    The sup-ll ( d ) mutations dominant suppressors unc-93(el500) and <br /> have recessive Scr phenotype. A s u p - l l ( d ) mutation alter the <br /> level structure sup-ll gene product. Genetic considerations suggest that <br /> s u p - l l ( d ) does simply alter level sup-11 gene product. It unlikely <br /> that s u p - l l ( d ) results cis-dominant overproduction gene product, be- <br /> cause sup-ll ( d ) / s u p - l l ( d ) contain product s u p - l l <br /> (d)/+, turn contain product sup-11 ( d ) / s u p - l l ( 0 ) ; <br /> thus, s u p - l l (d)/+ mutant s u p - l l ( d ) / s u p - l l ( 0 ) . <br />                                         THE SUP-I   I LOCUS OF C . elegans                           221 <br />                                                      TABLE 2 <br />                 Some phenotypic characteristics resulting uarious sup-I Igenotypes <br />  <br />              Genotype                                     Morphology            Suppressor activity <br />  <br />     ~ ~ p -(d)/sup-ll(d) <br />             11                                               Scr                     strong <br />     sup-11 (0)/sup-11(0)                              arrested embryo                 ...... <br />     sup-11 (r)sup-ll(r)                                     WT                       partial <br />     su~-1l(d)/+                                             WT                       strong <br />     sup-ll(0)      +                                        WT                       none <br />     sup-li(r)+                                              WT                       none <br />     sup-I2 (d)/sup-lI(0)                                    Scr                      strong <br />     sup-I1 ( r ) / s u p - l l ( 0 )                        WT                       partial <br />     sup-ll(d)/sup-12 ( r )                            non-Scr, small                 strong <br />     sup-ll(0); sup-5                                  arrested L1 larva*             weak <br />     sup-I1(0)/+; sup-5                                      ND                       weak <br />  <br />    The genotype abbreviations text Table 1. &#226;&#8364;&#732;iM~rphology&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> refers phenotype animals unc-93(+) genotype; exception, indicated an <br /> asterisk (*), unc-93(e2500) genotype. &#226;&#8364;&#339;Suppressor activity&#226;&#8364;? refers suppression of <br /> unc-93(el500). The phenotypes &#226;&#8364;&#339;partial&#226;&#8364;? &#226;&#8364;&#339;weak&#226;&#8364;? suppressors described text; <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;partial&#226;&#8364;? suppression greater &#226;&#8364;&#339;weak&#226;&#8364;? suppression. The phenotypes presence of <br /> sup-5 f o r sup-ll(0) allele sup-li(n403 n681). WT =wild type, Scr = Scrawny, ND = <br /> not done, - = unable determine. <br />  <br /> However, s u p - l l (a)/+ non-Scr sup-ll ( d ) / s u p - l l ( d ) sup- <br /> l l ( d ) / s u p - l l ( 0 ) Scr. Similarly, unlikely s u p - l l ( d ) merely lowers <br /> the level s u p - l l product, sup-ll(O)/+ contain prod- <br /> uct sup-ll (a)/+; thus, sup-ll(O)/+ better suppressor than <br /> is sup-ll(d)/+. However, sup-ll(O)/+; el500 Unc(e1500) pheno- <br /> type, s u p - l l (d)/+; el500 suppressed. These considerations suggest <br /> that s u p - l l (d) mutation results altered s u p - l l product. The dominance <br /> of suppressor activity relative rarity dominant suppressor muta- <br /> tions EMS mutagenesis consistent interpretation novel <br /> function acquired s u p - l l ( d ) product. The Scr phenotype, how- <br /> ever, result loss particular activity wild-type <br /> s u p - l l product toxic effect altered product; Scr <br /> phenotype recessive, wild-type allele provides missing activity <br /> or overcomes toxic effect. <br />      Rare intragenic revertants Scr phenotype obtained EMS <br /> mutagenesis. These sup-ifl(I) mutations recessive partial suppressors of <br /> unc-93(e1500) : hermaphrodites genotype s u p - l l ( r ) ; unc-93(el500) es- <br /> sentially mutant larvae lay eggs improved motility adults. <br /> Genetic considerations suggest s u p - l l ( r ) mutations, like s u p - l l <br /> (d) mutations derived, result altered sup-ll products <br /> rather altered levels s u p - l l [specifically, s u p - l l ( d ) ] product. It is <br /> unlikely s u p - l l ( r ) lowers level s u p - l l ( d ) product, this <br /> case s u p - l l ( d ) / s u p - l l ( I ) , non-Scr, contain product than <br /> sup-ll ( d ) / s u p - l l ( O ) , Scr [as s u p - l l ( d ) / s u p - l l ( d ) ] . Similarly, <br /> it unlikely s u p - l l ( r ) elevates level sup-ll ( d ) s u p - l l (I)/ <br />  222                           I. S.   G R E E N W A L D A N D H.   R. HORVITZ <br /> s u p - l l ( r ) contain product sup-21 ( r ) / s u p - l l ( d ) , would <br /> contain product s u p - l l ( r ) / s u p - l l ( 0 ) ; s u p - l l ( r ) / s u p - l l ( r ) <br /> and s u p - l l ( r ) / s u p - l l ( 0 ) partially suppress unc-93 (el500) of <br /> wild-type size s u p - l l ( r ) / s u p ( d ) strongly suppresses unc-93(el500) <br /> and somewhat smaller wild type. Thus, s u p - l l ( r ) alleles appear result <br /> in novel s u p - l l products different s u p - l l ( d ) product; the <br /> s u p - l l ( r ) alleles probably double mutants retained s u p - l l ( d ) mu- <br /> tation acquiring second-site mutation structural gene. It <br /> seems llkely partial suppression el500 s u p - l l ( r ) results from <br /> residual s u p - l l ( d ) activity, conceivable new suppressor <br /> activity present double mutant. That s u p - l l ( d ) / s u p - l l ( r ) non-Scr <br /> implies s u p - l l ( r ) mutations restore activity lost s u p - l l ( d ) <br /> mutants o r result product overcome toxic effects s u p - l l <br /> ( d ) product; respect, s u p - l l ( r ) mutations similar s u p - l l <br /> (+) allele. <br />     Reversion dominant suppressor activity s u p - l l ( d ) generated null <br /> alleles [ s u p - l l ( O ) ] s u p - l l . These null alleles recessive lethal. em- <br /> bryos homozygous sup-II null alleles arrest onset morphogenesis. <br /> The recessive embryonic lethal null phenotype demonstrates func- <br /> tion sup-11 essential &#226;&#8218;&#172;or development. One s u p - l l ( 0 ) allele sup- <br /> pressible null allele-specific suppressors sup-5 sup-7 (WATERSTON                         and <br /> BRENNER           1978; WATERSTON           1980). Recent results indicated sup-5 <br /> and sup-7 suppressors amber suppressor tRNA&#226;&#8364;&#8482;s (R. WATERSTON,                           N. WILLS <br /> and R. GESTELAND,             personal communication; J. KARN,personal communication) ; <br /> therefore, sup-ll polypeptide product. Although embryonic lethality <br /> of s u p - l Z ( 0 ) suppressed informational suppressors, animals ar- <br /> rest L1 stage. These arrested larvae exhibit abnormal pharyngeal pump- <br /> ing gut characteristics &#226;&#8364;&#339;starved&#226;&#8364;? animals, suggesting arrest <br /> because obtain sufficient nutrition; interpretation consistent <br /> with observation wild-type animals arrest L1 stage eggs <br /> hatch absence bacteria. In addition, s u p - l l ( d ) suppressor activity is <br /> only weakly restored sup-ll (O)/+;                   unc-93 (e1500) sup-5 hermaphrodites. <br /> These results suggest sup-12 product required stoichiometric <br />  (as opposed catalytic) quantities (see HORVITZ               SULSTON    1980). <br />     We consider ways s u p - l l ( d ) mutations might <br /> suppress unc-93(el500). Although mechanisms conceivable, will <br /> discuss o d y reasonable. <br />     One possible way sup-11 ( d ) mutations suppress unc-93(el500) <br /> is acting informational suppressors (HARTMAN                    ROTH 1973). We do <br /> not think possibility likely. Informational suppressors allele- <br /> specific gene-nonspecific. The sup-11 ( d ) suppressors appear allele- <br /> nonspecific (see below) gene-specific: course work, strains <br /> containing sup-21 ( d ) variety apparently unrelated d p y unc muta- <br /> tions constructed, suppression mutations observed; in <br />                          THE   sup-i I LOCUS OF C. elegans                    223 <br /> addition, reversion studies mutations carried laboratories <br /> have generated sup-11 alleles. <br />    We consider ways sup-11( d ) mutations suppress <br /> unc-93(e1500, n200) based assumption sup-11 unc-93 in- <br /> volved muscle function. Because disrupt muscle structure, mutations in <br /> unc-93(e1500, n200) genes considered define a <br /> set genes responsible muscle structure function (WATERSTON,       THOM- <br /> SON BRENNER1980; ZENGEL         EPSTEIN1980). That sup-11( d ) muta- <br /> tions correct muscle defect unc-93(e2500) implies sup-12 affects <br /> muscle function well. However, sup21 product likely to <br /> function muscle cells, nonmuscle cell activities: because <br /> proteins closely related muscle structural proteins nonmuscle <br /> cells (e.g., CLARKE  SPUDICH1977) plausible recessive lethality <br /> of sup-11 null alleles results loss nonmuscle function sup-21 (+) <br /> and loss muscle function se. <br />    The sup-ll ( d ) mutations suppress unc-93(e1500) restoring proper <br /> function unc-93 product, e.g., restoring function complex pro- <br /> teins consisting products sup-21 unc-93. Ample precedent <br /> for suppression <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>missense mutation</span> alteration interacting <br /> polypeptide exists (HARTMAN      ROTH1973). However, suppressors might <br /> be expected allele-specific. Because visible unc-93 alleles, e1500 and <br /> n200, confer easily distinguishable phenotypes, likely different <br /> mutations; similarly, sup-21( d ) alleles differ efficiencies of <br /> suppression e1500, likely different mutations. All <br /> sup-11( d ) mutations, isolated suppressors e1500, suppress <br /> n200 appear unc-93 allele-nonspecific, subject caveat <br /> that visible alleles unc-93 test products these <br /> two alleles shared structural similarity recognized the <br /> sup-11( d ) products. <br />    Alternatively, sup11 ( d ) mutations suppress unc-93 (e1500) by <br /> bypasing defect caused unc-93(el500) restoring function the <br /> e1500 product. A bypass mechanism display allele-nonspecificity, which <br /> is consistent observations. W e like consider specific bypass <br /> mechanism context earlier studies unc-93: suggested (GREEN- <br /> WALD HORVITZ     1980) unc-93 <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>null mutants</span> wild type there <br /> exists alternative unc-93 product, provided another <br /> member unc-93 gene family; rare semidominant mutation e1500 may <br /> result toxic product interferes muscle structure function even <br /> in presence putative related gene products. More generally, proposed <br /> that members gene families wild-type null phenotypes <br /> and identified rare semidominant mutations. Currently, C . elegans <br /> genes genetic properties identified number Qf labora- <br /> tories, including own, preliminary results genes sug- <br /> gest number likely increase. Recently, proposal genes <br />  224                        I. S. GREENWALD A N D H. R. HORVITZ <br />  <br /> are members gene families supported molecular studies of <br /> these genes: unc-92 V,which appears member actin gene family <br />  (C. LANDEL        D. HIRSH, personal communication), sup7 X , tRNA gene <br />  (R. WATERSTON,         N. WILLS,and R. GESTELAND,    personal communication). <br />     If unc-93 member gene family, modification fam- <br /> ily member product effective presence e1500 <br /> product way bypass accomplished. That e1500/+ much <br /> less mutant e1500/e1500 implies unc-93 wild-type product can <br /> compete overcome toxic effects e1500 product; the <br /> s u p - l l ( d ) mutations, appear missense mutations (see above), alter <br /> the sup-11 product confer unc-93 (+)-like activity. In words, sup-11 <br /> and unc-93 nonidentical members gene family, s u p - l l ( d ) <br /> mutations conferring unc-93 (i-)         activity sup-11 product. Studies Qf <br /> genes 23 24 bacteriophage T4 suggest mutations possible: <br /> the wild-type products structural components phage head, null <br /> mutations gene lethal; however, missense mutafit gene 23 <br />  (which encodes major capsid protein) isolated suppresses 24- <br /> amber mutants, apparently conferring 24f-like activity gene 23 prod- <br /> uct (MCNICOL,         SIMONand BLACK1974). <br />     We thank P. STERNBERG determining stages arrested larvae assistance with <br /> photomicrography. We thank  <br /> </body></html>