<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2001 Genetics Society America <br />  <br />  <br />  <br /> Mutations YRB1 Gene Encoding Yeast Ran-Binding-Protein-1 That Impair <br />        Nucleocytoplasmic Transport Suppress Yeast Mating Defects <br />  <br />   Markus Ku&#194;&#168;nzler,*,&#226;&#8364;&#160; Joshua Trueheart,*,1 Claudio Sette,* Eduard Hurt&#226;&#8364;&#160; Jeremy Thorner* <br />  *Department Molecular Cell Biology, Division Biochemistry Molecular Biology, University California, Berkeley, California <br />          94720-3202 &#226;&#8364;&#160;Ruprecht-Karls-Universita&#194;&#168;t Heidelberg, Biochemie-Zentrum Heidelberg, D-69120 Heidelberg, Germany <br />                                                        Manuscript received July 13, 2000 <br />                                                   Accepted publication November 21, 2000 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 We identified temperature-sensitive (ts) mutations essential gene, YRB1, encodes the <br />              yeast homolog Ran-binding-protein-1 (RanBP1), known coregulator Ran GTPase cycle. Both <br />              mutations result single amino acid substitutions evolutionarily conserved residues (A91D R127K, <br />              respectively) Ran-binding domain Yrb1. The altered proteins reduced affinity Ran <br />              (Gsp1) vivo. After shift restrictive temperature, mutants display impaired nuclear protein import <br />              reduces poly(A)&#226;&#171;&#185; RNA export, suggesting primary defect nucleocytoplasmic trafficking. <br />              Consistent conclusion, yrb1ts mutations display deleterious genetic interactions mutations <br />              genes involved nucleocytoplasmic transport, including SRP1 (&#226;?&#163;-importin) several <br />              &#226;?&#164;-importin family members. These yrb1ts alleles isolated ability suppress different types <br />              mating-defective mutants (respectively, fus1&#226;&#338;&#172; ste5ts), indicating reduction nucleocytoplasmic <br />              transport enhances mating proficiency. Indeed, yrb1ts mutants, Ste5 (scaffold protein the <br />              pheromone response MAPK cascade) mislocalized cytosol, absence pheromone. <br />              Also, yrb1ts mutations suppress mating defect null <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> MSN5, encodes the <br />              receptor pheromone-stimulated nuclear export Ste5. Our results suggest reimport Ste5 into <br />              nucleus important downregulating mating response. <br />  <br />  <br />  <br /> M       ATING yeast Saccharomyces cerevisiae the <br />          culmination complex series events re- <br /> quired cellular nuclear fusion haploid <br />                                                                          nucleus response pheromone, observed the <br />                                                                          regulated import export nuclear proteins <br />                                                                          (Kaffman O&#226;&#8364;&#8482;Shea 1999). <br /> cells opposite mating type (Sprague Thorner                         Proteins protein-RNA complexes cross nu- <br /> 1992). Mating pheromones (secreted peptides) bind                        clear envelope nuclear pores comprised &#226;&#181;&#8216;50 <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> G-protein-coupled receptors, stimulating mitogen-                   different proteins, termed nucleoporins (Ryan and <br /> activated protein kinase (MAPK) cascade (Bardwell et                     Wente 2000). Nucleocytoplasmic transport re- <br /> al. 1994) evokes dramatic changes gene transcrip-                quires soluble factors. Transport receptors im- <br /> tion, cell cycle arrest, pronounced alterations                   port export (&#226;?&#164;-importin relatives) bind their <br /> cell morphology nuclear reorganization (Leberer                      cargo shuttle cytosol nucleo- <br /> et al. 1997; Stone et al. 2000). Pheromone-activated                 plasm (Go&#194;&#168;rlich Kutay 1999). The small Ras-like <br /> pheromone-induced gene products required cell                        GTPase Ran associated factors confer directional- <br /> fusion deposited localized site plasma                   ity transport (Macara et al. 2000). In S. cerevisiae, <br /> membrane leading edge mating projection                      GSP1 GSP2 encode Ran isoforms (Belhumeur et <br /> (&#226;&#8364;&#339;shmoo tip&#226;&#8364;?; Madden Snyder 1998). Proteins re-                      al. 1993; Kadowaki et al. 1993). Ran exists predomi- <br /> quired nuclear fusion recruited nucleus                   nantly GTP-bound form nucleus; the <br /> (Rose 1996). How signal initiated plasma                      cytosol, Ran mainly GDP-bound. This asymmetry is <br /> membrane transmitted nucleus activate                     imposed subcellular distribution Ran regula- <br /> gene expression unclear. Two components                  tors: Ran-specific guanine-nucleotide exchange fac- <br /> pathway, Ste5 (Pryciak Huntress 1998; Mahanty                        tor (RanGEF1), PRP20/SRM1/MTR1 gene product <br /> et al. 1999) Far1 (Blondel et al. 1999), shuttle                     S. cerevisiae, confined nucleus, the <br /> between nucleus cytosol, predominantly                   Ran-specific GTPase-activating protein (RanGAP1), the <br /> nuclear na&#196;&#177;&#194;&#168;ve cells, rapidly ejected                RNA1 gene product S. cerevisiae, located the <br />                                                                          cytoplasm. GSP1, PRP20, RNA1 essential <br />                                                                          genes, recessive mutations block nuclear <br />   Corresponding author: Markus Ku&#194;&#168;nzler, Ruprecht-Karls-Universita&#194;&#168;t     protein import poly(A)&#226;&#171;&#185; RNA export (Corbett <br /> Heidelberg, Biochemie-Zentrum Heidelberg (BZH), Im Neuen- <br /> heimer Feld 328, 4. OG, D-69120 Heidelberg, Germany.                     Silver 1997; Oki et al. 1998). <br /> E-mail: markus.kuenzler@urz.uni-heidelberg.de                              Transport receptors bind specifically GTP- <br />   1 <br />     Present address: Microbia, Inc., Cambridge, MA 02139.                bound form Ran conserved domain N <br /> Genetics 157: 1089&#226;&#8364;&#8220;1105 (March 2001) <br />  1090                                                M. Ku&#194;&#168;nzler et al. <br />  <br /> termini (Go&#194;&#168;rlich Kutay 1999). RanGTP-binding               isolated (Katz et al. 1987) extragenic suppressor <br /> to export receptor enhances affinity export       missense mutations STE5, encodes scaffold <br /> substrate; conversely, binding RanGTP import           protein pheromone-activated MAPK cascade <br /> receptor prevents binding import substrate.           (Elion 1998). As described here, yrb1 mutations <br /> Hence, high RanGTP nuclear compartment po-               cause clear defects nucleocytoplasmic trafficking of <br /> tentiates association export cargo export recep-        various proteins, including Ste5, able sup- <br /> tors triggers release import cargo import           press mating defect msn5&#226;&#338;&#172; mutant. These data <br /> receptors. Ran-binding-protein-1 (RanBP1)            suggest preventing efficient reimport Ste5 after <br /> protein binds specifically RanGTP (Ku&#194;&#168;nzler et          pheromone-induced release nucleus sus- <br /> al. 2000; Plafker Macara 2000). RanBP1 contains             tains mating-competent state. <br /> a conserved Ran-binding domain (RBD) &#226;&#181;&#8216;140 resi- <br /> dues (Beddow et al. 1995; Vetter et al. 1999), which <br /> is necessary sufficient high-affinity binding                      MATERIALS AND METHODS <br /> RanGTP nuclear export RanBP1,               Strains growth conditions: Yeast strains used this <br /> yeast (Ku&#194;&#168;nzler et al. 2000). Homologous RBDs               study listed Table 1. Strains JTY2483 JTY2484 were <br /> found nuclear proteins, like vertebrate                obtained backcrossing strain 381G-42E-P1 times <br /> RanBP2/NUP358 (Yokoyama et al. 1995) RanBP3                 YPH499 YPH500. msn5&#226;&#338;&#172;::TRP1 strain HMK30 <br /> (Mueller et al. 1998), S. cerevisiae nuclear proteins,      derived strain LH90 (Blondel et al. 1999) three <br />                                                                 consecutive backcrosses W303-1A derivatives, CRY1 <br /> S. cerevisiae Nup2 (Booth et al. 1999) Yrb2 (Taura          CRY2. DNA-mediated transformation yeast cells per- <br /> et al. 1998). Transport receptors block stimulation          formed using modified version lithium acetate method <br /> Ran-mediated GTP hydrolysis RanGAP1; contrast,            (Gietz et al. 1992). The fus1&#226;&#338;&#172; mutation deletes 90% the <br /> RanBP1 acts coactivator RanGAP1-stimulated              coding sequence (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> FUS1 promoter codon 460) and <br /> GTP hydrolysis Ran and, moreover, required            constructed two-step gene disruption method (Boeke <br />                                                                 et al. 1987). Heterozygous diploid strain JTY2501 derived <br /> nucleotide hydrolysis RanGTP bound trans-          CRY3 transplacing YRB1 locus homolog <br /> port receptor (Go&#194;&#168;rlich Kutay 1999). These bio-             chromosome IV transformation EcoRI-XbaI frag- <br /> chemical activities, fact RanBP1 abun-          ment containing yrb1&#226;&#338;&#172;::HIS3 construct excised plas- <br /> dant, shuttles nucleus cytoplasm,           mid pMK112n (Table 2). To construct strain HMK21, JTY2501 <br /> but exclusively cytosol steady        transformed plasmid pMK103, sporulated, a <br />                                                                 MATa His&#226;&#171;&#185; Ura&#226;&#171;&#185; 5-fluoro-orotic acid (5-FOA)-sensitive spore <br /> state (Ku&#194;&#168;nzler et al. 2000; Plafker Macara 2000),          chosen. Strain JTY2486 obtained transformation of <br /> suggest RanBP1 major role cytoplasm           CRY1 EcoRI-SpeI fragment containing nup2::HIS3 <br /> both recycling transport receptors release         construct excised plasmid pJON115 (Loeb et al. 1993). <br /> of export cargo (Peterson et al. 2000). Consistent         Strain HMK29 constructed analogously using BamHI- <br /> this view, S. cerevisiae YRB1, encoding yeast RanBP1,        HindIII fragment containing gsp2&#226;&#338;&#172;::LEU2 construct (Kado- <br />                                                                 waki et al. 1993). Correct transplacements verified by <br /> essential cell viability required nu-       Southern hybridization analysis. <br /> clear protein import poly(A)&#226;&#171;&#185; RNA export                       Unless indicated otherwise, yeast cells propagated at <br /> (Schlenstedt et al. 1995).                                      30&#226;&#172;&#353;. Rich medium (YP), synthetic complete medium (SC), and <br />    A link yeast mating Ran GTPase               synthetic minimal medium (SM) prepared described <br /> cycle identification srm1-1 mutation,        (Kaiser et al. 1994). Glucose (Glc) raffinose (Raf) were <br />                                                                 added carbon source final concentration 20 g/liter <br /> known reside RanGEF1, suppressed                autoclaving; induction galactose (Gal) per- <br /> mating defect cells lacking pheromone receptors          formed adding Gal (final concentration 2%) Raf-grown <br /> increased basal expression pheromone-respon-           cells. Drop-out media (SC lacking appropriate nutrients) <br /> sive reporter gene (Clark Sprague 1989). Another            used maintain selection plasmids. Agar plates con- <br /> connection mating nucleocytoplasmic                 taining 5-FOA prepared described Boeke et al. <br />                                                                 (1987). Escherichia coli strain DH5&#226;?&#163; (Hanahan 1983) used <br /> transport finding ste21 mutation, identi-      propagation plasmid DNAs. Bacteria cultivated <br /> fied screen enhancers mating defect          using standard methods (Sambrook et al. 1989). <br /> a temperature-sensitive (ts) mutation STE4 (encoding            Quantitative mating assays: Quantitative mating assays were <br /> the &#226;?&#164;-subunit pheromone receptor-coupled het-            performed previously described (Sprague 1991). Briefly, <br /> erotrimeric G-protein; Akada et al. 1996), resides           MATa strains tested MAT&#226;?&#163; tester strains pre- <br />                                                                 grown 26&#226;&#172;&#353; midlogarithmic phase selective rich <br /> MSN5, encoding nuclear receptor pheromone- <br />                                                                 medium, respectively. Cells washed water 106 <br /> stimulated export Ste5 (Mahanty et al. 1999)             cells MATa strains tested mixed 107 <br /> Far1 (Blondel et al. 1999). As described here, iso-          cells MAT&#226;?&#163; tester strain. In case experiment <br /> lated yrb1ts mutation suppressor mutant             shown Table 3, mixture spread directly pre- <br /> (fus1&#226;&#338;&#172;) defective cell fusion step mating. Fus1       cooled (14&#226;&#172;&#353;) SMGlc plates, plates incubated 3 <br /> is pheromone-induced, O-glycosylated, integral mem-           days 14&#226;&#172;&#353;, 3 days room temperature. The <br />                                                                 resultant diploid colonies counted normalized to <br /> brane protein acts late stage mating               titer input MATa cells (determined plating same <br /> (Trueheart Fink 1989). While mapping yrb1              dilutions plates selective MATa strain tested <br /> mutation, allelic mutation            incubating 3 days room temperature). In case <br />                                                 Ran-Binding-Protein-1 Yeast Mating                                                1091 <br />  <br />                                                                  TABLE 1 <br />                                                                Yeast strains <br />  <br /> Strain                                                 Characteristics                                                    Source <br /> YPH499                MATa ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 lys2-801am ade2-101oc                         Sikorski Hieter     (1989) <br /> YPH500                MAT&#226;?&#163; ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 lys2-801am ade2-101oc                         Sikorski Hieter     (1989) <br /> YPH501                MATa/MAT&#226;?&#163; (YPH499 &#226;&#171;&#187; YPH500)                                                           Sikorski Hieter     (1989) <br /> CRY1 (W303-1A)        MATa ura3-1 trp1-1 his3-11,15 leu2-3,112 ade2-1 can1-100 GAL                          R. S. Fuller <br /> CRY2 (W303-1B)        MAT&#226;?&#163; ura3-1 trp1-1 his3-11,15 leu2-3,112 ade2-1 can1-100 GAL                          R. S. Fuller <br /> CRY3 (W303D)          MATa/MAT&#226;?&#163; (CRY1 &#226;&#171;&#187; CRY2)                                                               R. S. Fuller <br /> 9399-7B               MATa ura3-52 his4-&#226;&#338;&#172;29 GAL                                                             Trueheart Fink      (1989) <br /> JY416                 MAT&#226;?&#163; ura3-52 leu2-3,112 fus1&#226;&#338;&#172;                                                         This study <br /> JTY2023               MATa ura3-52 trp1-&#226;&#338;&#172;63 his4&#226;&#338;&#172;29 ade2-101oc GAL                                          This study <br /> JTY2024               JTY2023 fus1&#226;&#338;&#172;                                                                         This study <br /> JTY2025               JTY2024 yrb1-51 (sfo1-1)                                                              This study <br /> JTY2488               MATa ura3-52 trp1-&#226;&#338;&#172;63 his4-&#226;&#338;&#172;29 fus1&#226;&#338;&#172; yrb1-51                                          This study <br /> JTY2501               CRY3 yrb1&#226;&#338;&#172;::HIS3/YRB1                                                                 This study <br /> HMK21                 CRY1 yrb1&#226;&#338;&#172;::HIS3 (pMK103)                                                             This study <br /> JTY2026               MATa ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ade2-101oc yrb1-51                            This study <br /> JTY2027               MAT&#226;?&#163; ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 lys2-801am yrb1-51                            This study <br /> JTY2482               MATa/MAT&#226;?&#163; ( JTY2026 &#226;&#171;&#187; JTY2027)                                                        This study <br /> 381G-42E-P1           MATa ade2-1 lys2 oc tyr1oc his4-580 trp ste5-3 yrb1-52 (stp52) CRY1 SUP4-3 ts   Katz et al. (1987) <br /> JTY2483               MATa ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 lys2-801am ade2-101oc yrb1-52                 This study <br /> JTY2484               MAT&#226;?&#163; ura3-52 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 lys2-801am ade2-101oc yrb1-52                 This study <br /> JTY2485               MATa/MAT&#226;?&#163; ( JTY2483 &#226;&#171;&#187; JTY2484)                                                        This study <br /> JTY2500               MATa ura3-52 his3-&#226;&#338;&#172;200 leu2-3,112 trp1-901 canR gal4-542 gal80-538                    Inouye et al. (1997a) <br />                         ADE2::PGAL-URA3 LYS2::lexop-lacZ <br /> JTY2486               MATa ura3-1 trp1-1 his3-11,15 leu2-3,112 ade2-1 can1-100 GAL nup2-5::HIS3             This study <br /> HMK29                 CRY1 gsp2&#226;&#338;&#172;::LEU2                                                                      This study <br /> HMK30                 CRY1 msn5&#226;&#338;&#172;::TRP1                                                                      This study <br /> DC17                  MAT&#226;?&#163; his1                                                                             J. B. Hicks <br />  <br />  <br />  <br /> of experiment shown Table 5, mating mixture               DNA complementation ts <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> sfo1-1 cells <br /> collected 0.45-&#226;?&#174;m pore filter incubated 6 hr             (see below) revealed identical YRB1 gene, and <br /> 30&#226;&#172;&#353; YPGlc. After incubation, cells resuspended               sequence analysis (see below) demonstrated both <br /> in SMGlc medium plated appropriate dilutions                 sfo1-1 stp52 mutations reside YRB1 locus, these <br /> SMGlc plates appropriate nutrients select diploids.          alleles renamed yrb1-51 yrb1-52, respectively. <br /> As control number viable MATa cells used                Recovery analysis yrb1ts alleles: The <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> sequence <br /> mating mixture, 106 cells MATa cells collected            alterations corresponding yrb1-51 yrb1-52 mutations <br /> a separate filter, incubated above, resuspended YPGlc,             determined cloning sequencing DNA isolated <br /> and plated YPGlc plates appropriate dilutions. Mating              mutants. The polymerase chain reaction (PCR) was <br /> efficiency expressed percentage input MATa                 used amplify 636-bp products comprising entire YRB1 <br /> haploids formed diploid colonies.                                   open reading frame (ORF) using genomic DNA JTY2026 <br />    Isolation yrb1-51: JTY2024 (MATa fus1&#226;&#338;&#172;) mutagen-               (yrb1-51) 381G-42E-P1 (yrb1-52) template oligo- <br /> ized ethyl methanesulfonate (108 cells/ml; 3% EMS; 1 hr)            nucleotide primers, 5&#226;&#172;&#732;-GGG GAT CCG AAT GTC TAG CGA <br /> to 25% <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span>, spread YPGlc plates. After 3 days             AGA TAA G-3&#226;&#172;&#732; (OSFO1) 5&#226;&#172;&#732;-GGT CTA GAC GCA AGT <br /> 28&#226;&#172;&#353;, &#226;&#181;&#8216;72,000 colonies replica plated precooled                 AAC AAG C-3&#226;&#172;&#732; (OSFO5), corresponded, respectively, <br /> (14&#226;&#172;&#353;) SMGlc plates containing uracil (20 mg/liter),             positions &#226;&#171;&#186;2 &#226;&#171;&#185;18 &#226;&#171;&#185;635 &#226;&#171;&#185;616 201-codon <br /> 2 A600 nm units JY416 (MAT&#226;?&#163; fus1&#226;&#338;&#172;) cells spread.             YRB1 sequence (where &#226;&#171;&#185;1 base initiator <br /> These plates incubated 4 days 14&#226;&#172;&#353;. Candidate clones          codon ORF) included restriction sites their <br /> that gave positive mating response (35 colonies total)            5&#226;&#172;&#732;-ends facilitate cloning PCR products. Reaction <br /> restreaked master plate, retested suppression,              products isolated, digested BamHI XbaI, and <br /> and examined ability grow various temperatures.          inserted E. coli vector pUC19 sequencing. Nucleotide <br /> A single isolate ( JTY2025) displayed ts phenotype coseg-         sequence multiple inserts determined strands <br /> regated ability suppress mating defect            using M13/pUC universal M13/pUC reverse sequenc- <br /> fus1&#226;&#338;&#172; cells 14&#226;&#172;&#353; (data shown). The mutation conferring             ing primers (New England Biolabs, Beverly, MA) and, when <br /> these phenotypes initially named sfo1-1 (suppressor fus           necessary, sequence-specific primers. The single-base-pair mu- <br /> one). In course crosses, shown, first,          tations recovered tested ability confer ts <br /> sfo1 tightly linked trp1 (no recombinants 31 tetrads;          phenotype substituting mutant YRB1 ORFs (excised <br /> distance &#226;&#177;&#8226;1.6 cM) and, second, complementation tests,                 SalI fragments pUC19 derivatives) corre- <br /> that sfo1-1 mutation allelic stp52, suppressor        sponding segment pMK103 introducing entire <br /> of mating defects mapped region (Katz               yrb1-51 yrb1-52 genes EcoRI-XbaI fragments (excised <br /> et al. 1987). Because cloning corresponding wild-type             pMK103 derivatives) pRS314, yielding TRP1- <br />  1092                                                      M. Ku&#194;&#168;nzler et al. <br />  <br />                                                               TABLE 2 <br />                                                               Plasmids <br />  <br />          Plasmid                                         Characteristics                               Source <br />          YEp24                           2-&#226;?&#174;m URA3                                          Botstein et al. (1979) <br />          pRS314                          CEN ARS TRP1                                       Sikorski Hieter (1989) <br />          pRS424                          2-&#226;?&#174;m TRP1                                          Sikorski Hieter (1989) <br />          pRS425                          2-&#226;?&#174;m LEU2                                          Sikorski Hieter (1989) <br />          pUN100                          CEN ARS LEU2                                       Elledge Davis (1988) <br />          YEp352                          2-&#226;?&#174;m URA3                                          Hill et al. (1986) <br />          pGAD424                         2-&#226;?&#174;m LEU2 ADH1p-GAL4TAD-MCS-ADH1t                  Bartel Fields (1995) <br />          pBTM116                         2-&#226;?&#174;m TRP1 ADH1p-LexADBD-MCS-ADH1t                  Bartel Fields (1995) <br />          pRSETA                          AmpR T7-His6-MCS                                   Invitrogen (Carlsbad, CA) <br />          pNOPGFP2L                       pRS425-NOP1p-GFP                                   K. Hellmuth E. Hurt <br />                                                                                               (unpublished results) <br />          pJON115                         nup2-5::HIS3                                       Loeb et al. (1993) <br />          pPS815                          2-&#226;?&#174;m URA3 ADH1p-SV40NLS-GFP-lacZ                   Lee et al. (1996) <br />          pPS817                          2-&#226;?&#174;m URA3 GAL1p-SV40NLS-GFP-lacZ                   Lee et al. (1996) <br />          pGADGFP                         2-&#226;?&#174;m LEU2 ADH1p-SV40NLS-GAL4TAD-GFP                Shulga et al. (1996) <br />          pNOPGFPAU-NPL3                  2-&#226;?&#174;m ADE2 URA3 NOP1p-GFP-NPL3                      Senger et al. (1998) <br />          YEplac195AU-L25NLS-GFP          2-&#226;?&#174;m ADE2 URA3 RPL25NLS-GFP-MEX67t                 O. Gadal E. Hurt <br />                                                                                               (unpublished results) <br />          pKW430                          2-&#226;?&#174;m URA3 ADH1p-SV40NLS-PKINES-(GFP)2              Stade et al. (1997) <br />          pLDB419                         2-&#226;?&#174;m LEU2 YAP1-GFP                                 Yan et al. (1998) <br />                                          pNOPGFP2L-STE5                                     This study <br />          pSB415                          YEp24-NTH1-YRB1                                    This study <br />          pMK102                          pUC19-YRB1                                         This study <br />          pMK103                          YEp352-YRB1                                        This study <br />          PMK104                          pRSETA-YRB1                                        This study <br />          pMK112n                         pRS316-yrb1&#226;&#338;&#172;::HIS3                                 This study <br />          pNOPPATA-GSP1G21V               pUN100-NOP1p-ProtA-TEV-GSP1(G21V)-ADH1t            Hellmuth et al. (1998) <br />          pMK275                          pRS314-YRB1                                        This study <br />          pMK277                          pRS314-yrb1-51                                     This study <br />          pMK278                          pRS314-yrb1-52                                     This study <br />          pMK284n                         pRS314-YRB1-GFP(S65T)                              Hellmuth et al. (1998) <br />          pMK294-51                       pRS314-yrb1-51-GFP(S65T)                           This study <br />          pMK294-52                       pRS314-yrb1-52-GFP(S65T)                           This study <br />          pMK291-wt                       pRS424-YRB1-GFP(S65T)                              This study <br />          pMK291-51                       pRS424-yrb1-51-GFP(S65T)                           This study <br />          pMK291-52                       pRS424-yrb1-52-GFP(S65T)                           This study <br />          pMK199-wt                       pGAD424-YRB1                                       This study <br />          pMK199-51                       pGAD424-yrb1-51                                    This study <br />          pMK199-52                       pGAD424-yrb1-52                                    This study <br />          pMK195-GV                       pBTM116-GSP1(G21V)                                 This study <br />  <br />  <br />  <br /> marked plasmids pMK277 pMK278, respectively. Finally,             complement ts phenotype yrb1-51 mutant. The <br /> HMK21 (yrb1&#226;&#338;&#172; [pMK103]) transformed                    smallest original isolate (pSB415) contained YRB1 gene <br /> pMK277, pMK278, control plasmid (pMK275) carrying                neighboring gene NTH1 encoding neutral treha- <br /> the normal YRB1 gene, plated 5-FOA plates 23&#226;&#172;&#353; select         lase. Subsequent subcloning localized complementing ac- <br /> against resident URA3-marked YRB1-containing plasmid              tivity 1.3-kb chromosomal EcoRI-XbaI fragment containing <br /> (pMK103), resulting isolates analyzed          YRB1, used construct pMK102, pMK103, <br /> ability grow elevated temperature.                              pMK275. <br />    Construction plasmids: Standard techniques used               To construct plasmid (pMK112n) carrying yrb1&#226;&#338;&#172;::HIS3 <br /> for manipulation recombinant DNA (Sambrook et al.              deletion construct, internal BglII fragment excised <br /> 1989). Plasmid DNA E. coli isolated according             YRB1-containing insert pMK101 replaced by <br /> Del Sal et al. (1988). Unless specified otherwise, PCR amplifi-       BamHI fragment containing HIS3 gene, in- <br /> cations performed using Vent DNA polymerase (New                 serted transcriptional orientation YRB1. Con- <br /> England Biolabs). Correct sequence PCR-generated con-              struction plasmid pNOPPATA-GSP1G21V, expresses <br /> structs verified nucleotide sequence analysis. Plasmids        GTPase-defective mutant form Gsp1, Gsp1(G21V), fused <br /> used study listed Table 2.                             N terminus cleavage site (ENLYEQG) tobacco <br />    The YRB1 gene isolated yeast genomic library            etch virus (TEV) protease immunoglobulin G (IgG) <br /> (Carlson Botstein 1982) carried high-copy-number             binding domains Protein A (ProtA), control the <br /> yeast/E. coli shuttle vector (YEp24) virtue ability      NOP1 promoter ADH1 terminator, described <br />                                                 Ran-Binding-Protein-1 Yeast Mating                                           1093 <br />  <br />                            TABLE 3                                   erated STE5 NcoI-BamHI fragment comprising entire <br />                                                                      ORF (using primers OSTE5-1, 5&#226;&#172;&#732;-GGGGGGCCATGGGTAT <br />  The yrb1-51 (sfo1-1) mutation suppresses cold-sensitive         GGAAACTCCTACAGAC-3&#226;&#172;&#732;, OSTE5-2, 5&#226;&#172;&#732;-GGGGGGAT <br />               mating defect fus1&#226;&#338;&#172; mutant                        CCCTATATATAATCCATATGG-3&#226;&#172;&#732;) pNOPPATA (Hell- <br />                                                                      muth et al. 1998) subsequent recloning insert as <br />                            Plasmid: mating efficiency (&#226;&#171;&#187;10&#226;&#171;&#186;5)a       PstI fragment pNOPGFP2L. This vector based on <br />                                                                      pRS425 contains 1.4-kb BamHI-PstI NOP1p-GFP cassette <br /> MATa strainb                YEp24        YEp24-YRB1 (pSB415)         (K. Hellmuth E. Hurt, unpublished results). <br />                                                                         Preparation rabbit polyclonal anti-Yrb1 antiserum: To <br /> FUS1 YRB1                   250                   165                generate (His)6-Yrb1 fusion protein containing the <br /> fus1&#226;&#338;&#172; YRB1                   [1.0]                  0.6              10 residues Yrb1, corresponding YRB1 coding se- <br /> fus1&#226;&#338;&#172; yrb1-51 (sfo1-1)       39                     1.8              quence excised SalI fragment pMK102 and <br />    a <br />      Mating efficiency defined number diploids          ligated XhoI site pRSETA (Invitrogen), yielding <br /> formed number input haploids strain tested.            pMK104. For expression E. coli, strain BL21(DE3)/pLysS <br /> The values given represent average independent          (Studier 1991) transformed pMK104 fusion <br /> trials, performed triplicate, normalized       protein induced addition isopropyl-&#226;?&#164;-d-thio-galacto- <br /> to mating efficiency fus1&#226;&#338;&#172; mutant.                        pyranoside (IPTG) final concentration 0.4 mm followed <br />    b <br />      The indicated strains [JTY2023, MATa FUS1 YRB1;                 incubation 37&#226;&#172;&#353; 2 hr. Selection pMK104 to <br /> JTY2024, MATa fus1&#226;&#338;&#172; YRB1; JTY2025, MATa fus1&#226;&#338;&#172; yrb1-51                maintained adding 50 mg/liter carbenicillin (Sigma, St. <br /> (sfo1-1)] transformed YEp24 (a URA3-marked          Louis), stable derivative ampicillin, medium <br /> 2-&#226;?&#174;m DNA vector) pSB415 (YEp24-YRB1) mated               fusion protein relatively toxic cells. <br /> JY416 (MAT&#226;?&#163; fus1&#226;&#338;&#172;), described materials methods.           The fusion protein purified E. coli using Ni2&#226;&#171;&#185;-chelate <br />                                                                      affinity chromatography (Ni-NTA resin; QIAGEN, Chats- <br />                                                                      worth, CA), according manufacturer&#226;&#8364;&#8482;s recommenda- <br /> previously (Hellmuth et al. 1998). Plasmids pMK294-51            tions. The purified protein used raise polyclonal anti- <br /> pMK294-52, pMK291-wt, pMK291-51, pMK291-52, ex-              sera adult female New Zealand White rabbits (nos. 1390 <br /> pressing Yrb1-green fluorescent protein (GFP) fusions          1391), following standard protocols (Harlow Lane <br /> control authentic YRB1 promoter, constructed             1988), using 600 &#226;?&#174;g protein 50% Titermax (CytRx, Nor- <br /> by replacing internal BglII fragment YRB1 coding           cross, GA) immunization 400 &#226;?&#174;g protein <br /> sequence pMK284n, expresses functional Yrb1-              50% incomplete Freund&#226;&#8364;&#8482;s adjuvant (Sigma) two <br /> GFP chimera (Hellmuth et al. 1998), corresponding           subsequent immunizations administered 3 5 weeks, <br /> fragments yrb1-51 yrb1-52, followed subsequent           respectively. Bleeds taken 4 weeks (2 ml), 6 weeks <br /> recloning respective YRB1-GFP gene fusions EcoRI-          (50 ml), 7 weeks (2 ml), 8 weeks (terminal) stored <br /> NotI fragments pRS424.                                          0.02% sodium azide &#226;&#171;&#186;70&#226;&#172;&#353;. For detection Yrb1 by <br />    Fusions full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Yrb1 Gal4 transcriptional activa-   immunoblotting resulting antisera (nos. 1390 1391) <br /> tion domain (TAD) full-length Gsp1(G21V) E. coli          used primary antibodies dilution 1:5000. <br /> LexA DNA-binding domain (DBD) generated PCR,                   Two-hybrid assay: To assess interactions LexA <br /> using two-hybrid vectors pGAD424 pBTM116, respec-            (DBD)-Gsp1(G21V) Gal4(TAD)-Yrb1 fusion proteins, <br /> tively. Fragments comprising entire YRB1 ORF syn-           strain JTY2500 harboring E. coli lacZ gene control <br /> thesized using 5&#226;&#172;&#732;-CCG AAT TCG GTC CAG GTG GTA GCG                    LexA-binding sites cotransformed appro- <br /> AAG ATA AGA AAC CTG TCG-3&#226;&#172;&#732; (OSFO15) M13/                     priate pBTM116- pGAD424-based plasmids. Transfor- <br /> pUC reverse sequencing primer (New England Biolabs)               mants grown SCGlc medium lacking leucine tryp- <br /> the primers pUC19 carrying chromosomal YRB1-con-             tophan midexponential phase (A546 nm &#226;&#171;&#189; &#226;&#181;&#8216;1) assayed <br /> taining EcoRI-XbaI fragment (pMK102), pUC19 carrying              &#226;?&#164;-galactosidase acitivity described previously (Ku&#194;&#168;nzler <br /> the corresponding fragments yrb1-51 yrb1-52 ORFs,        Hurt 1998). <br /> as templates. The PCR products digested EcoRI             Preparation yeast cell extracts: Yeast cells washed <br /> PstI inserted corresponding sites pGAD424,           culture volume cold phosphate-buffered <br /> yielding pMK199-wt, pMK199-51, pMK199-52, respec-                saline (PBS), aliquoted 1.5-ml microcentrifuge tubes <br /> tively. Similarly, fragment comprising entire ORF coding       (&#226;&#181;&#8216;20 A546 nm units tube), stored pellets &#226;&#171;&#186;70&#226;&#172;&#353;. <br /> for Gsp1(G21V) produced using 5&#226;&#172;&#732;-GCG AGG CCT TGC                 Frozen cell pellets thawed adding 0.2 ml cold lysis <br /> CCC AGC TGC TAA CGG TGA AG-3&#226;&#172;&#732; (OGSP7) RSET                       buffer (50 mm Tris-HCl pH 7.5, 150 mm NaCl, 20 mm MgCl2, <br /> (5&#226;&#172;&#732;-AAC TGC AGC CAA CTC AGC TTC C-3&#226;&#172;&#732;) primers,                10% glycerol, 2 mm DTT, 1 mm PMSF) lysed by <br /> and E. coli expression vector pRSETB (Invitrogen, Carlsbad,          vigorous vortexing 0.2 g acid-washed glass beads (0.45&#226;&#8364;&#8220; <br /> CA) carrying PCR-mutated genomic PvuII-HindIII fragment            0.6 mm diameter) 30-sec periods (separated 1-min <br /> coding Gsp1(G21V) template. The resulting PCR             periods cooling ice). The lysate clarified centrifu- <br /> product cleaved StuI PstI inserted         gation 5 min 13,000 &#226;&#171;&#187; g 4&#226;&#172;&#353; protein concentra- <br /> SmaI PstI sites pBTM116, yielding plasmid pMK195-GV.          tion determined dye-binding method (Bradford <br />    Plasmid YEplac195-AU-L25NLS-GFP derived YE-              1976) using commercially available reagents (Bio-Rad, Her- <br /> plac195-ADE2-URA3-L25-GFP (Hurt et al. 1998) removing             cules, CA) bovine serum albumin (BSA) standard. <br /> most RPL25 coding sequence, 5&#226;&#172;&#732;-end                Purification ProtA-TEV-Gsp1(G21V) yeast: Trans- <br /> that encodes 52 residues L25 (and contains           formants wild-type strain CRY1, coexpressing ProtA-TEV- <br /> intron), using two-step PCR procedure (Giebel Spritz           Gsp1(G21V) pNOPPATA-GSP1G21V Yrb1- <br /> 1990; mutagenic primer, 5&#226;&#172;&#732;-GGG ACA ACT CCA GTG AAA                   GFP, Yrb1(A91D)-GFP, Yrb1(R127K)-GFP plasmids <br /> AGT CTT CTC TTT GCT CTC GAG TGG AAC AGC CTT                          pMK284n, pMK294-51, pMK294-52, respectively, were <br /> GGA AGC-3&#226;&#172;&#732;; O. Gadal E. Hurt, unpublished results).              grown selective medium 26&#226;&#172;&#353; A546 nm &#226;&#171;&#189; &#226;&#181;&#8216;1.5. Purifica- <br /> Plasmid pNOPGFP2L-STE5 expressing GFP-Ste5 fusion pro-             tion Gsp1(G21V) cells performed essentially <br /> tein multicopy-plasmid control consti-           described (Hellmuth et al. 1998), minor modifica- <br /> tutive NOP1-promoter constructed inserting PCR-gen-         tion universal buffer (Ku&#194;&#168;nzler Hurt 1998) used <br />  1094                                                        M. Ku&#194;&#168;nzler et al. <br />  <br /> throughout purification, including cell lysis, washing steps,       recessive allele YRB1, encodes homolog of <br /> and elution. Elution cleavage TEV-protease (GIBCO-              mammalian RanBP1 (Ouspenski et al. 1995; Schlen- <br /> BRL, Gaithersburg, MD) performed incubation for <br /> 1 hr room temperature.                                               stedt et al. 1995). Hence, sfo1-1 redesignated yrb1-51. <br />    Nuclear protein import RNA export assays: Tempera-                  A recessive ts mutation, stp52 (sterile pseudorever- <br /> ture-sensitive mutants, isogenic wild-type          sion), closely linked TRP1, isolated ex- <br /> strains, containing plasmids express constitutively nuclear        tragenic suppressor mating defect MATa <br /> transport substrates fused GFP(S65T), SV40NLS-Gal4-           ste5-3 &#226;&#171;&#187; MAT&#226;?&#163; ste5-3 cross restrictive temperature <br /> (TAD)-GFP (pGADGFP), GFP-Npl3 (pNOPGFPAU-NPL3), <br /> and L25NLS-GFP (YEp195-AU-L25NLS-GFP), cultivated               (Katz et al. 1987). The stp52 mutation suppressed <br /> early exponential phase (A546 nm &#226;&#171;&#189; &#226;&#181;&#8216;0.5) selective SCGlc             ste5, ste4, ste7 missense (ts) alleles (Katz et <br /> medium 23&#226;&#172;&#353;, split equal portions, incubated             al. 1987). Although linkage analysis reported by <br /> at 23&#226;&#172;&#353; 37&#226;&#172;&#353; various periods time. Strains carrying      Katz et al. (1987) assigned stp52 mutation the <br /> plasmids expressing SV40NLS-GFP-&#226;?&#164;-galactosidase (pPS817)                opposite TRP1 locus yrb1-51, found <br /> under control GAL1 promoter pregrown early <br /> exponential phase (A546 nm &#226;&#171;&#189; &#226;&#181;&#8216;0.5) selective SCRaf medium            yrb1-51/stp52 diploid strain ts, the <br /> at 23&#226;&#172;&#353; Gal (2%) added cultures                ts growth defect stp52 mutant completely <br /> cells incubated 23&#226;&#172;&#353; hour (to allow mRNA             rescued cloned YRB1 gene plasmid (data <br /> synthesis export). The induced cultures split             shown). These results demonstrated stp52 <br /> two equal portions, portion shifted 37&#226;&#172;&#353;              mutation recessive allele YRB1, was <br /> 3 hr, portion maintained 23&#226;&#172;&#353; the <br /> same period. Fluorescence microscopy living yeast cells              confirmed sequencing mutant DNA (see be- <br /> expressing GFP fusion proteins according Hell-              low). Hence, stp52 redesignated yrb1-52. <br /> muth et al. (1998). Cells concentrated brief centrifuga-           Phenotypic characterization yrb1-51 yrb1- <br /> tion resuspended residual growth medium              52 mutations: To understand alterations YRB1 <br /> any washing steps. To assay mRNA export, cells cultivated          suppress mating-defective mutants, examined, <br /> in YPGlc medium described strains harboring <br /> constitutively expressed GFP fusion proteins. Poly(A)&#226;&#171;&#185; RNA              first, physiology yrb1 mutants. At 23&#226;&#172;&#353;, yrb1-51 <br /> was localized situ hybridization described previously          mutant cells grew nearly wild-type cells, <br /> (Segref et al. 1997).                                                   yrb1-52 mutant cells displayed impaired <br />    Miscellaneous: SDS-PAGE immunoblotting con-                 growth conditions; yrb1-51 and <br /> ducted described previously (Ku&#194;&#168;nzler Hurt 1998).                yrb1-52 cells ceased growth lost viability 3&#226;&#8364;&#8220;6 <br /> Multiple sequence alignment using CLUSTALW <br /> 1.7 (Thompson et al. 1994) BOXSHADE 3.21 [Bioinfor-                 hr shift 37&#226;&#172;&#353; (data shown). Similar results <br /> matics group Swiss Institute Experimental Cancer             observed corresponding homozygous dip- <br /> Research (ISREC)] programs. Identities Yrb1 sequence             loids (data shown). As judged immunoblotting <br /> were calculated basis pairwise alignments using           cell lysates (Figure 1A), shift 37&#226;&#172;&#353; 3 hr, <br /> ALIGN algorithm FASTA package (Pearson                     product yrb1-51 allele hardly detectable, <br /> Lipman 1988). <br />                                                                         yrb1-52 product remained relatively stable <br />                                                                         6 hr temperature shift. Thus, yrb1-51 muta- <br />                            RESULTS <br />                                                                         tion appears destabilize gene product higher <br />                                                                         temperature, yrb1-52 product stable un- <br />                       ts <br />    Isolation yrb1 mutations suppressors mating                 der conditions. Upon prolonged incubation <br /> defects: The mating deficiency fus1&#226;&#338;&#172; mutant                     37&#226;&#172;&#353;, apparent degradation product Yrb1 accu- <br /> much pronounced 14&#226;&#172;&#353; 30&#226;&#172;&#353;. At 14&#226;&#172;&#353;, dip-                   mulated yrb1-52 cells, observed the <br /> loid formation MATa fus1&#226;&#338;&#172; &#226;&#171;&#187; MAT&#226;?&#163; fus1&#226;&#338;&#172; cross                       wild-type control cells. Yrb1 expressed similar <br /> is &#226;&#172;?0.5% MATa FUS1 &#226;&#171;&#187; MAT&#226;?&#163; fus1&#226;&#338;&#172; cross                         levels MATa, MAT&#226;?&#163;, MATa/MAT&#226;?&#163; cells (data <br /> (Table 3). A screen extragenic suppressors                  shown) level MATa cells elevated in <br /> mating defect (see materials methods) yielded                       response treatment &#226;?&#163;-factor mating pheromone <br /> a single mutation, sfo1-1. This suppressor mutation re-                 (data shown). <br /> producibly enhanced mating competence fus1&#226;&#338;&#172;                           Examination cell morphology revealed that <br /> mutant 30&#226;&#8364;&#8220;50-fold (Table 3), did fully restore                  haploid yrb1-51 cells arrested enlarged cells <br /> mating proficiency level FUS1 cell. The sup-                large bud large unbudded cells shift <br /> pressor segregated 2:2 backcrosses                  37&#226;&#172;&#353; (data shown; Ba&#194;&#168;umer et al. 2000), is <br /> a fus1&#226;&#338;&#172; strain cosegregated recessive ts growth              reminiscent cell cycle progression mutants. Similar <br /> defect (in &#226;&#172;&#381;15 tetrads analyzed cross). Genetic map-                results previously observed stp52/yrb1-52 cells <br /> ping mutation right arm chromosome                     (Katz et al. 1987; Ouspenski 1998). Another striking <br /> IV CEN4 TRP1 gene (data shown),                     phenotype yrb1ts alleles appearance of <br /> complementation ts growth defect wild-                    chains elongated nonseparated cells, evident in <br /> type YRB1 gene plasmid (data shown), elimina-                  homozygous diploids grown plates semipermis- <br /> tion suppression phenotype plasmid-borne                      sive temperature (30&#226;&#172;&#353;; Figure 1B). Such cell elongation <br /> YRB1 (Table 3), nucleotide sequencing mu-                    diagnostic mutations delay G2-M progression <br /> tant DNA (see below) established sfo1-1                  (Lew 2000). We observed similar morphological de- <br />                                               Ran-Binding-Protein-1 Yeast Mating                                      1095 <br />  <br />                                                                    nonical cell division cycle (cdc) mutations, might <br />                                                                    indicate role Yrb1 multiple stages cell <br />                                                                    cycle. Consistent notion, yrb1-51 muta- <br />                                                                    tion interferes G1/S transition the <br />                                                                    passage mitosis (Ba&#194;&#168;umer et al. 2000), dis- <br />                                                                    played synthetic growth defects combined with <br />                                                                    different cdc28 alleles (cdc28-4 cdc28-1N) that <br />                                                                    diagnostic different cell cycle stages (G1 and <br />                                                                    G2/M, respectively; Table 4). <br />                                                                       Yrb1-51 Yrb1-52 altered conserved residues <br />                                                                    Ran-binding domain defective Ran-bind- <br />                                                                    ing vivo: To determine nature alterations <br />                                                                    mutant proteins, PCR used recover the <br />                                                                    YRB1 coding sequences mutant strains (see <br />                                                                    materials methods). The DNA sequence each <br />                                                                    mutant ORF contained single <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>point mutation</span>, both <br />                                                                    alter highly conserved residue RBD <br />                                                                    (Figure 2A). The yrb1-51 mutation C-to-A transver- <br />                                                                    sion coding strand position 272 (where &#226;&#171;&#185;1 is <br />                                                                    base initiator ATG), substitutes <br />                                                                    Asp Ala codon 91 (A91D). The yrb1-52 allele a <br />                                                                    G-to-A transition coding strand position 380, <br />                                                                    substitutes Lys Arg codon 127 (R127K). <br />                                                                    On basis homology modeling Yrb1 the <br />                                                                    crystal structure RBD (RanBD1) mamma- <br />                                                                    lian Nup358 (RanBP2) complexed Ran bound to <br />    Figure 1.&#226;&#8364;&#8221;Effects yrb1-51 yrb1-52 mutations        nonhydrolyzable GTP analog (Vetter et al. 1999), <br /> vivo stability Yrb1 cell morphology. (A) Stability       A91D replaces nonpolar residue hydrophobic <br /> normal mutant Yrb1 restrictive temperature. Haploid <br /> strain YPH499 (YRB1) congenic derivatives, JTY2026         core Yrb1 bulkier, charged residue (Figure <br /> (yrb1-51) JTY2483 (yrb1-52), grown 23&#226;&#172;&#353; SCGlc        2B). This change destabilize global fold of <br /> medium midexponential phase, shifted 37&#226;&#172;&#353;, samples        Yrb1, consistent rapid degradation mu- <br /> were withdrawn indicated times. Total protein ex-       tant protein observed restrictive temperature (Figure <br /> tracted sample analyzed SDS-PAGE              1A). Two existent yrb1 alleles, yrb1-1 yrb1-2 <br /> immunoblotting using rabbit polyclonal anti-Yrb1 antiserum <br /> (no. 1390). A band cross-reacts nonspecifically      (Schlenstedt et al. 1995), alter residues (F187 L93, <br /> anti-Yrb1 antiserum served loading control. The asterisk      respectively) project hydrophobic <br /> indicates major degradation product Yrb1. (B) Morphol-        pocket A91. In contrast, R127K makes seemingly <br /> ogy homozygous diploid yrb1ts cells. Strains YPH501 (YRB1/      modest change surface-exposed residue forms <br /> YRB1), JTY2482 (yrb1-51/yrb1-51), JTY2485 (yrb1-52/yrb1-       bridge residues long C-terminal &#226;&#8364;&#339;arm&#226;&#8364;? of <br /> 52) cultivated YPGlc plates 16&#226;&#172;&#353; 30&#226;&#172;&#353;, indicated, <br /> and viewed Nomarski optics.                                     Ran embraces RBD (Figure 2B), alteration <br />                                                                    unlikely disrupt overall structure, consistent with <br />                                                                    observed stability mutant protein restrictive <br /> fect srp1-31ts/srp1-31ts diploids (data shown).             temperature (Figure 1A). We confirmed muta- <br /> Srp1/&#226;?&#163;-importin adaptor necessary recogni-              tion necessary sufficient confer ts <br /> tion nuclear import proteins contain clas-           phenotype corresponding allele inserting each <br /> sical nuclear localization signal (NLS) Kap95/              mutant DNA plasmid introducing a <br /> &#226;?&#164;-importin receptor (Enenkel et al. 1995). At 37&#226;&#172;&#353;, srp1-           yrb1&#226;&#338;&#172; background (see materials methods). <br /> 31 cells impaired import NLS-containing re-                 Two independent approaches demonstrated the <br /> porter proteins arrest uniformly large-budded               yrb1-51 yrb1-52 mutations interfere Yrb1-Ran <br /> cells indicative defect mitosis (Loeb et al. 1995).        (Gsp1) interaction vivo. The GTPase-deficient form <br /> Correspondingly, degradation Clb2, destruc-               Gsp1, Gsp1(G21V), binds strongly Yrb1 than <br /> tion required exit mitosis, impaired             normal Gsp1 (Schlenstedt et al. 1995); hence, used <br /> srp1-31 cells (Loeb et al. 1995); likewise, degradation         Gsp1(G21V) analyses. First, applied two- <br /> Clb2 anaphase inhibitors, Pds1 Sic1,             hybrid method using full-length wild-type Yrb1, Yrb1 <br /> also impaired yrb1-51 cells (Ba&#194;&#168;umer et al. 2000). De-          (A91D), Yrb1(R127K) fused Gal4 transcrip- <br /> spite similarities, absence uniform cell            tional activation domain [Gal4(TAD)] full-length <br /> cycle arrest phenotype distinguishes yrb1-51 yrb1-         Gsp1(G21V) fused LexA DNA-binding domain <br /> 52 mutations srp1-31 mutation ca-                [LexA(DBD)] reporter strain carrying chromo- <br />  1096                                                     M. Ku&#194;&#168;nzler et al. <br />  <br />                                                              TABLE 4 <br />           Summary genetic interactions yrb1-51 (and yrb1-52) nucleocytoplasmic transport factors <br />  <br />         Mutation                                                 Source                                Genetic interactiona <br />         Ran GTPase cycle <br />           rna1-1                        Atkinson et al. (1985)                                               &#226;&#171;&#185;&#226;&#171;&#185; (sl) <br />           prp20-1                       Aebi et al. (1990)                                                   &#226;&#171;&#185; (&#226;&#171;&#185;) <br />           srm1-1                        Clark Sprague (1989)                                             &#226;&#171;&#185; (nd) <br />           prp20-10                      Fleischmann et al. (1996)                                            &#226;&#171;&#186; (nd) <br />           gsp1-1, -2                    Wong et al. (1997)                                                   &#226;&#171;&#185;&#226;&#171;&#185; (nd) <br />           gsp2::HIS3                    Kadowaki et al. (1993)                                               &#226;&#171;&#185; (nd) <br />           yrb2::HIS3                    I. Macara (personal communication)                                   &#226;&#171;&#186; (sl) <br />           nup2::HIS3                    Loeb et al. (1993)                                                   &#226;&#171;&#185; (sl) <br />         Nucleoporins <br />           nsp1ts                        Nehrbass et al. (1993)                                               &#226;&#171;&#185; (nd) <br />           nup133::HIS3                  Doye et al. (1994)                                                   &#226;&#171;&#185; (&#226;&#171;&#185;) <br />           nup116::URA3                  Bailer et al. (1998)                                                 &#226;&#171;&#185; (&#226;&#171;&#185;) <br />         Nuclear import receptors <br />           srp1-31                       Loeb et al. (1995)                                                   sl (sl) <br />           srp1-49                       Schroeder et al. (1999)                                              sl (sl) <br />           rsl1-4                        Koepp et al. (1996)                                                  sl (sl) <br />           mtr10::HIS3                   Senger et al. (1998)                                                 sl (sl) <br />           kap104::HIS3                  Aitchison et al. (1996)                                              sl (nd) <br />           pse1-1                        Seedorf Silver (1997)                                            &#226;&#171;&#185; (nd) <br />           yrb4::HIS3                    Schlenstedt et al. (1997)                                            &#226;&#171;&#186; (nd) <br />           pse1-1 yrb4::HIS3             Seedorf Silver (1997)                                            sl (nd) <br />         Nuclear export receptors <br />           los1::HIS3                    Hellmuth et al. (1998)                                               &#226;&#171;&#186; (nd) <br />           msn5::TRP1                    Blondel et al. (1999)                                                sup (sup) <br />           cse1-1                        Xiao et al. (1993)                                                   sl (nd) <br />           xpo1-1                        Stade et al. (1997)                                                  &#226;&#171;&#185; (sl) <br />           crm1-1, -2, -3                Yan et al. (1998); F. Stutz (personal communication)                 &#226;&#171;&#186; (nd) <br />         Others <br />           cdc28-4                       Reed (1980)                                                          &#226;&#171;&#185;   (nd) <br />           cdc28-1N                      Piggott et al. (1982)                                                &#226;&#171;&#185;   (nd) <br />           rat1-1                        Amberg et al. (1992)                                                 &#226;&#171;&#186;   (nd) <br />           nsr1::URA3                    Kondo Inouye (1992)                                              &#226;&#171;&#186;   (nd) <br />           plc1::HIS3                    Flick Thorner (1993)                                             &#226;&#171;&#185;   (nd) <br />           Abbreviations parentheses indicate phenotype observed yrb1-52 allele. <br />           a <br />             &#226;&#171;&#186;, synthetic growth phenotype; &#226;&#171;&#185;, synthetic growth defect (see text details); sl, synthetic lethality; nd, <br />         determined; sup, synthetic growth defect extragenic suppression mating defect (see Table 5). <br />  <br />  <br /> somally inserted copy E. coli lacZ control               muth et al. (1998), eluted cleavage TEV-protease <br /> eight LexA-operator sites. In system, Yrb1(A91D)                (see materials methods), analyzed SDS- <br /> showed reproducible reduction (&#226;&#172;&#381;3-fold) Yrb1                   PAGE, Coomassie blue staining, immunoblotting <br /> (R127K) showed dramatic reduction (&#226;&#172;&#381;50-fold)                    using polyclonal anti-Yrb1 antisera (no. 1391). Exami- <br /> interaction Gsp1(G21V) compared wild-type                    nation input material (Figure 3B, load), flow- <br /> Yrb1 (Figure 3A). Immunoblotting anti-Yrb1 antise-              fractions (data shown), demonstrated that <br /> rum (see materials methods) anti-Gsp1 anti-                  expression normal mutant Yrb1-GFP fusions was <br /> bodies (gift P. Belhumeur) showed constructs             comparable, stabilities purification. <br /> were expressed equivalent levels (data shown).                No detectable Yrb1(R127K)-GFP copurified Gsp1 <br />   These results confirmed biochemical pro-                 (G21V), significant amounts wild-type <br /> cedure (Figure 3B). Yrb1-GFP, Yrb1(A91D)-GFP,                     Yrb1-GFP endogenous Yrb1 retained the <br /> Yrb1(R127K)-GFP, produced authentic YRB1                    beads (Figure 3B). A detectable Yrb1 <br /> promoter CEN plasmids, expressed wild-type                (A91D)-GFP copurified ProtA-Gsp1(G21V), its <br /> cells (strain CRY1) producing ProtA-(TEV site)-               level markedly wild-type <br /> Gsp1(G21V) constitutive NOP1 promoter                    Yrb1-GFP retained conditions (Figure 3B). <br /> a CEN plasmid. After growth 26&#226;&#172;&#353;, protein complexes                   yrb1-51 yrb1-52 mutants defective nuclear <br /> bound bead-immobilized ProtA-(TEV site)-Gsp1(G21V)                protein RNA transport: To determine yrb1-51 and <br /> were recovered cell extracts described Hell-              yrb1-52 cause defects nuclear protein import RNA <br />                                          Ran-Binding-Protein-1 Yeast Mating                                         1097 <br />  <br />  <br />  <br />  <br />                                                                 Figure 2.&#226;&#8364;&#8221;Positions altered residues Yrb1 re- <br />                                                              sulting yrb1-51 yrb1-52 mutations. (A) Alignment <br />                                                              various Ran-binding domains (RBDs). Sequences shown <br />                                                              grouped subfamilies (RanBP1, RanBP2, and <br />                                                              RanBP3), basis certain shared sequence characteris- <br />                                                              tics, include following (with GenBank accession num- <br />                                                              bers): S. cerevisiae Yrb1 (L38489), Schizosaccharomyces pombe <br />                                                              Sbp1 (D86381), mouse RanBP1 (X56045), human RanBP1 <br />                                                              (X83617), Xenopus laevis RanBP1 (U09128), Arabidopsis thali- <br />                                                              ana RanBP1 (U62742), mouse RanBP2 nucleoporin (X87337), <br />                                                              human NUP358 (D38076), Bos taurus RanBP2 nucleoporin <br />                                                              (L41691), Caenorhabditis elegans Ranup96 (Z34801), S. cerevisiae <br />                                                              Nup2 (X69964), S. pombe Hba1 (U38783), human RanBP3 <br />                                                              (Y08697). Sequence S. cerevisiae Yrb2/Nup36 the <br />                                                              Swiss Protein Database (accession no. P40517). The mouse <br />                                                              bovine RanBP2 incomplete derived <br />                                                              partial cDNA clones. An insert 24 residues (possibly <br />                                                              intron) omitted actual C. elegans Ranup96 <br />                                                              sequence optimize alignment RBDs. Identi- <br />                                                              ties shared 11 (or more) RBDs shown indicated <br />                                                              white-on-black letters; chemically similar residues shown <br />                                                              black-on-grey letters. The positions mutated yrb1-51, <br />                                                              A91A, yrb1-52, R127K, indicated top. (B) <br />                                                              Positions residues (A91 R127) altered yrb1- <br />                                                              51 yrb1-52 mutants, respectively, modeled on <br />                                                              RBD (RanBD1) human NUP358 complexed with <br />                                                              Ran bound nonhydrolyzable GTP analog (Vetter et al. <br />                                                              1999). Blue, Yrb1; purple, Ran; red, GTP analog. <br />  <br />  <br />  <br />  <br /> export, observed yrb1-1 yrb1-2 al-     cultures split equal portions, one <br /> leles (Schlenstedt et al. 1995), examined       maintained 23&#226;&#172;&#353; shifted <br /> distribution poly(A)&#226;&#171;&#185;-RNA situ hybridization.       37&#226;&#172;&#353;. Samples withdrawn various times for <br /> Mutant wild-type control cells grown midex-       analysis. Results readily visualized diploid <br /> ponential phase permissive temperature (23&#226;&#172;&#353;),         cells larger size; however, similar find- <br />  1098                                                     M. Ku&#194;&#168;nzler et al. <br />  <br />                                                                      ings haploid cells (data shown). In <br />                                                                      homozygous yrb1-51/yrb1-51 diploids shifted 37&#226;&#172;&#353; for <br />                                                                      2 hr, rapid clear-cut nuclear accumula- <br />                                                                      tion poly(A)&#226;&#171;&#185;-RNA cell (Figure 4A); this <br />                                                                      effect readily apparent &#226;&#181;&#8216;50% cells even <br />                                                                      1 hr shift (data shown). Poly(A)&#226;&#171;&#185; RNA <br />                                                                      accumulated nucleus distinctly punctate pat- <br />                                                                      tern, feature seen mutants strong RNA <br />                                                                      export defect, mex67-5 (Segref et al. 1997). How- <br />                                                                      ever, onset RNA export defect yrb1-51/ <br />                                                                      yrb1-51 cells slower mex67-5 mutants <br />                                                                      nuclear RNA accumulation complete <br />                                                                      (some cytosolic poly(A)&#226;&#171;&#185; RNA signal remains 2 hr <br />                                                                      shift 37&#226;&#172;&#353;). In striking contrast, yrb1-52/yrb1-52 <br />                                                                      diploids did accumulation poly(A)&#226;&#171;&#185; <br />                                                                      RNA (even 5 hr shift 37&#226;&#172;&#353;), just like wild-type <br />                                                                      control (Figure 4A). Although yrb1-51 cells manifested a <br />                                                                      clear defect RNA export, yrb1-51 mutants nor <br />                                                                      yrb1-52 mutants detectable effect nuclear <br />                                                                      export proteins containing leucine-rich NES, such <br />                                                                      SV40NLS-PKINES-GFP (Stade et al. 1997) yAP1-GFP <br />                                                                      (Yan et al. 1998; data shown). <br />                                                                         To monitor effect yrb1-51 yrb1-52 muta- <br />                                                                      tions nuclear protein import, different GFP <br />                                                                      fusions nuclear proteins examined. To assess <br />                                                                      &#226;?&#163;-importin/Srp1 &#226;?&#164;-importin/Kap95/Rsl1-depen- <br />                                                                      dent pathway, chimeras containing SV40 NLS <br />                                                                      used: galactose-inducible SV40NLS-GFP-&#226;?&#164;-galactos- <br />                                                                      idase, large diffuse the <br />                                                                      nucleus delivered (Lee et al. <br />                                                                      1996), constitutively expressed SV40NLS-Gal4TAD- <br />                                                                      GFP, which, small size, diffuse of <br />    Figure 3.&#226;&#8364;&#8221;Yrb1(A91D) (yrb1-51) Yrb1(R127K) (yrb1-             nucleus unless ongoing import occurs continuously <br /> 52) bind Gsp1(G21V) reduced affinity vivo. (A) Interac-      (Ku&#194;&#168;nzler Hurt 1998). The reporter a <br /> tion wild-type Yrb1, Yrb1(A91D), Yrb1(R127K)          fusion GFP Npl3, mRNA-binding protein, whose <br /> Gsp1(G21V) determined using two-hybrid method             nuclear entry depends importin, Kap111/Mtr10 <br /> described materials methods. In brief, Yrb1 proteins          (Senger et al. 1998). GFP-Npl3 accumulates rapidly in <br /> were fused Gal4(TAD), Gsp1(G21V) fused to <br /> the LexA(DBD). In recipient strain ( JTY2500), E. coli       cytoplasm import impaired Npl3 contin- <br /> lacZ gene control LexA-operator elements.          uously shuttles nucleus cytosol. <br /> &#226;?&#164;-Galactosidase activity expressed arbitrary units. Each       The fourth transport substrate constitutively ex- <br /> value represents average single measurements          pressed composed GFP fused NLS of <br /> three independent transformants; error bars indicate standard        ribosomal protein L25 (O. Gadal E. Hurt, per- <br /> deviation mean. (B) Binding Yrb1 Gsp(G12V) <br /> was assessed copurification. Cultures strain CRY1 (YRB1        sonal communication). Transport L25NLS-GFP into <br /> GSP1) carrying CEN plasmids expressing ProtA-TEV-                  nucleus utilizes different import receptors, <br /> Gsp1(G21V) fusion NOP1 promoter wild-            Kap121/Pse1 Kap123/Yrb4 (Schlenstedt et al. <br /> type Yrb1-GFP, Yrb1(A91D)-GFP, Yrb1(R127K)-GFP ex-                1997). Like SV40NLS-Gal4TAD-GFP, L25NLS-GFP small <br /> pressed authentic YRB1 promoter grown               diffuse nucleus unless import <br /> selective SCGlc medium 26&#226;&#172;&#353;. Extracts prepared and <br /> the ProtA-TEV-Gsp1(G21V) purified IgG-Sepharose               occurs continuously. <br /> (Pharmacia, Uppsala, Sweden) eluted digestion               Control strains yrb1-51 yrb1-52 mutants carrying <br /> recombinant TEV protease (GIBCO-BRL, Gaithersburg, MD).              reporter plasmids described cultivated <br /> Equal fractions load eluate column            selective medium, shifted restrictive temperature, <br /> were resolved SDS-PAGE analyzed, indicated,             examined fluorescence microscopy. For in- <br /> Coomassie blue staining immunoblotting using rabbit <br /> polyclonal anti-Yrb1 antiserum (no. 1391). Endogenous Yrb1           ducible reporter, transformants grown midexpo- <br /> served control confirm equivalent loading function-      nential phase Raf-containing medium permissive <br /> ality immobilized Gsp1(G21V).                                 temperature induced 1 hr addition 2% <br />                                                                      Gal (to allow mRNA synthesis export) before <br />                                                                      shift 37&#226;&#172;&#353;. For reporter proteins, a <br />                                            Ran-Binding-Protein-1 Yeast Mating                                        1099 <br />  <br />  <br />  <br />  <br />                                                                   Figure 4.&#226;&#8364;&#8221;Nuclear transport defects yrb1ts mutants. (A) <br />                                                                To examine poly(A)&#226;&#171;&#185;RNA export, cultures homozygous <br />                                                                diploid strains YPH501 (YRB1/YRB1), JTY2482 (yrb1-51/yrb1- <br />                                                                51), JTY2485 (yrb1-52/yrb1-52) grown YPGlc at <br />                                                                23&#226;&#172;&#353; early exponential phase, split equal portions, <br />                                                                incubated 2 hr 23&#226;&#172;&#353; 37&#226;&#172;&#353;. After <br />                                                                fixation formaldehyde, cells stained DNA <br />                                                                dye 4&#226;&#172;&#732;,6-diamidino-2-phenylindole, analyzed situ hybrid- <br />                                                                ization using CY3-labeled oligo(dT) probe visualize the <br />                                                                subcellular distribution poly(A)&#226;&#171;&#185;RNA, viewed fluo- <br />                                                                rescence microscopy using appropriate band-pass filters. (B <br />                                                                C) To examine nuclear protein import, strains <br />                                                                A transformed multicopy plasmids expressing <br />                                                                SV40NLS-Gal4TAD-GFP (B) L25NLS-GFP (C), respec- <br />                                                                tively, cultivated shifted A, viewed directly by <br />                                                                fluorescence microscopy Nomarski optics. <br />  <br />  <br />  <br />  <br /> significant cytoplasmic accumulation yrb1-51 yrb1-      haploids reporters (data not <br /> 52 mutants shift 37&#226;&#172;&#353;, compared wild-type cells,    shown). In yrb1-52 cells, defect noticeable even <br /> indicating general defect nuclear protein import.         permissive temperature. Our results showing defect <br /> Results SV40NLS-Gal4TAD-GFP (Figure 4B) L25NLS-        nuclear protein import yrb1-52 cells odds <br /> GFP (Figure 4C) reporters homozygous diploid                previously published report mutant <br /> strains shown; but, similar results obtained          (Ouspenski 1998). <br />  1100                                                M. Ku&#194;&#168;nzler et al. <br />  <br />    To exclude possibility cytoplasmic localiza- <br /> tion reporter proteins yrb-51 yrb1-52 cells <br /> was &#226;&#8364;&#339;leakiness&#226;&#8364;? mutant nuclei, accumula- <br /> tion constitutively expressed SV40NLS-GFP-&#226;?&#164;-galactosi- <br /> dase (encoded pPS815; Lee et al. 1996) examined <br /> in strains temperature shift. <br /> In contrast short-term assay inducible <br /> version reporter protein (see above), no <br /> increased cytoplasmic GFP signal observed (data <br /> not shown), demonstrating nuclei yrb1- <br /> 51 yrb1-52 mutants &#226;&#8364;&#339;leaky&#226;&#8364;? wild- <br /> type nuclei. <br />    Genetic interactions yrb1 mutations nucleocy- <br /> toplasmic transport factors: As independent means <br /> to confirm yrb1-51 yrb1-52 compromise nucleo- <br /> cytoplasmic trafficking permissive temperature, <br />                                                                    Figure 5.&#226;&#8364;&#8221;Genetic interactions yrb1ts alleles and <br /> genetic interactions alleles mutations in <br />                                                                 components nucleocytoplasmic transport machinery. <br /> genes encoding variety factors involved           yrb1-51 (or yrb1-52) mutant strains crossed strains <br /> nucleocytoplasmic transport examined. Strains              carrying mutation gene interest. The resulting <br /> carrying mutations crossed strains        diploids subjected sporulation, growth individ- <br /> carrying yrb1-51 yrb1-52 mutation, re-           ual spores tetratype asci examined various temper- <br />                                                                 atures. Left, genetic interaction yrb1-51 nup2::HIS3 is <br /> sulting diploids sporulated. Double mutant segre- <br />                                                                 manifested extremely poor growth (&#226;&#8364;&#339;&#226;&#171;&#185;&#226;&#8364;? Table 4) of <br /> gants tetratype asci compared single          yrb1-51 nup2::HIS3 double mutant 28&#226;&#172;&#353;, temperature <br /> mutant segregant wild-type segregant             clearly permissive congenic yrb1-51 nup2::HIS3 <br /> their ability grow various temperatures (see Figure       single mutants (yrb1-51 restrictive temperature of <br /> 5). Mutations tested included alterations genes en-          31&#226;&#172;&#353; conditions nup2::HIS3 obvi- <br />                                                                 ous growth defect higher temperatures). Right, yrb1- <br /> coding components Ran GTPase cycle, nuclear <br />                                                                 51 mutant carrying URA3-marked multicopy plasmid ex- <br /> import export receptors, nucleoporins (see Table        pressing wild-type YRB1 (pMK103) crossed strain <br /> 4). The plc1&#226;&#338;&#172;::HIS3 mutation (Flick Thorner 1993)           carrying ts mutation, srp1-31, &#226;?&#163;-importin. The resulting <br /> was tested evidence role PLC1-          diploid sporulated individual spores tetratype <br /> encoded phosphatidylinositol-specific phospholipase C           ascus streaked medium containing 5-FOA count- <br />                                                                 erselect plasmid. Genetic interaction yrb1-51 <br /> in mRNA export (York et al. 1999; J. Flick, personal <br />                                                                 srp1-31 manifested inviability &#226;&#8364;&#339;synthetic lethal- <br /> communication). The yrb1-51 mutation displayed dele-            ity&#226;&#8364;? (&#226;&#8364;&#339;sl&#226;&#8364;? Table 4) yrb1-51 srp1-31 double mutant at <br /> terious genetic interactions           temperature, congenic yrb1-51 srp1-31 <br /> classes mutants affect nucleocytoplasmic trans-         single mutants able grow permissive temperatures <br /> port (Table 4). Combination yrb1-52 mutation             (here shown 26&#226;&#172;&#353;). Other double mutant combinations <br />                                                                 tested phenotype listed Table 4. <br /> with 12 mutations revealed essentially <br /> the growth defects. For xpo1-1 yrb1-52 and <br /> yrb2&#226;&#338;&#172;::HIS3 yrb1-52, growth defect            ing, presumably, ability mutations sup- <br /> severe corresponding double mutant            press mating defects. Far1 Ste5 currently the <br /> yrb1-51 (Table 4). Equally pronounced growth defects            components mating pheromone response <br /> were observed yrb1-51 combined muta-              pathway known shuttle nucleus cyto- <br /> tions genes encoding certain nuclear transport recep-        plasm (Blondel et al. 1999; Mahanty et al. 1999) and <br /> tors (Table 4). In cases, double mutant       yrb1-52 isolated extragenic suppressor a <br /> not viable temperature (synthetic lethality);        ste5 <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>missense mutation</span> (Katz et al. 1987). Hence, we <br /> the genetic interactions observed, double mutants         constructed functional GFP-Ste5 chimera exam- <br /> were viable permissive temperature (23&#226;&#172;&#353;) re-         ined subcellular localization yrb1-51 yrb1-52 <br /> vealed restrictive temperature considerably          mutants (and control cells) absence phero- <br /> (&#226;&#172;&#381;3&#226;&#172;&#353;; &#226;&#171;&#185;&#226;&#171;&#185;) slightly (&#226;&#177;&#8226;3&#226;&#172;&#353;; &#226;&#171;&#185;) lower            mone avoid complications signal-induced <br /> any single mutants (synthetic growth defect).        changes. In wild-type cells steady state, GFP-Ste5 accu- <br /> Thus, functions Yrb1(A91D) Yrb1(R127K)               mulated nucleus, cytoplasmic stain- <br /> are partially defective permissive          ing evident (Figure 6), observed before <br /> conditions.                                                     (Mahanty et al. 1999; Pryciak Huntress 1998). <br />    Nucleocytoplasmic trafficking Ste5 altered          In contrast, nuclear accumulation observed in <br /> yrb1-51 yrb1-52 cells: Our results indicate im-        yrb1ts mutants, permissive tem- <br /> paired nucleocytoplasmic transport primary cause         perature (Figure 6). Since Ste5 shuttles continuously <br /> of phenotypes yrb1-51 yrb1-52 mutants, includ-       nucleus cytoplasm (Mahanty et al. 1999), <br />                                             Ran-Binding-Protein-1 Yeast Mating                                              1101 <br />  <br />                                                                                           TABLE 5 <br />                                                                 Suppression partial mating defect msn5&#226;&#338;&#172; null <br />                                                                        mutant yrb1-51 yrb1-52 mutations <br />  <br />                                                                 Genotypea                              Mating efficiency (%)b <br />                                                                 MSN5 YRB1                                    92.3   &#226;&#171;&#190;   10 <br />                                                                 msn5&#226;&#338;&#172; YRB1                                    20    &#226;&#171;&#190;   12 <br />                                                                 MSN5 yrb1-51                                 96.6   &#226;&#171;&#190;   4.7 <br />                                                                 msn5&#226;&#338;&#172; yrb1-51                                 89    &#226;&#171;&#190;   12.2 <br />                                                                 MSN5 YRB1                                    67.3   &#226;&#171;&#190;   18.7 <br />                                                                 msn5&#226;&#338;&#172; YRB1                                   18.3   &#226;&#171;&#190;   2.8 <br />                                                                 MSN5 yrb1-52                                 89.3   &#226;&#171;&#190;   8.9 <br />                                                                 msn5&#226;&#338;&#172; yrb1-52                                70.6   &#226;&#171;&#190;   24.5 <br />                                                                    a <br />                                                                     MATa spores indicated genotype cross be- <br />                                                                 tween strains HMK30 (msn5&#226;&#338;&#172;), JTY2027 (yrb1-51) and <br />                                                                 JTY2484 (yrb1-52), respectively, assayed quantitative <br />                                                                 mating MAT&#226;?&#163; tester strain DC17 described in <br />                                                                 materials methods. <br />                                                                   b <br />                                                                     Mating efficiencies ratio number diploids <br />                                                                 formed number viable input MATa haploids, the <br />                                                                 average independent trials, given percent- <br />                                                                 age standard deviations mean. <br />  <br />  <br />                                                                                         DISCUSSION <br />                                                                    Ran GTPase highly conserved pro- <br />    Figure 6.&#226;&#8364;&#8221;Localization GFP-Ste5 yrb1ts mutants. Hap-   teins nucleated cells (Macara et al. 2000; Sazer and <br /> loid strains YPH499 (YRB1), JTY2026 (yrb1-51), JTY2483      Dasso 2000). Ran action implicated nucleo- <br /> (yrb1-52) expressing GFP-Ste5 fusion protein a <br />                                                                 cytoplasmic transport, microtubule assembly, nuclear <br /> multicopy plasmid cultivated, shifted, viewed de- <br /> scribed legend Figure 4, B C.                     envelope formation, maintenance chromatin struc- <br />                                                                 ture nuclear (and nucleolar) organization, chromo- <br />                                                                 segregation, DNA replication, RNA metabolism, <br /> the observed mislocalization cells defective Yrb1       cell cycle progression. It matter some <br /> could due, principle, inhibition Ste5        debate pleiotropic phenotypes alter- <br /> import nucleus enhancement Ste5 export           ations Ran GTPase cycle solely the <br /> from nucleus. Since demonstrated yrb1-         established role Ran nucleocytoplasmic transport <br /> 51 yrb1-52 clearly impede nuclear import various         (Go&#194;&#168;rlich Kutay 1999) Ran addi- <br /> reporter proteins, possibility            tional roles cell. Recent studies using vitro <br /> likely.                                                         systems implicated Ran function directly micro- <br />    To obtain evidence suppression mating        tubule organization (Kahana Cleveland 1999) <br /> defects yrb1-51 yrb1-52 arises impairment           nuclear envelope formation (Hetzer et al. 2000; <br /> of nuclear import Ste5, tested yrb1ts       Zhang Clarke 2000). Our evidence indicates that <br /> mutations suppress mating defect msn5&#226;&#338;&#172;           Ran GTPase cycle linked signaling events <br /> mutant. Msn5 thought nuclear receptor              required mating consequence role of <br /> required pheromone-stimulated export Ste5           Ran nucleocytoplasmic transport. <br /> the nucleus (Mahanty et al. 1999). Each yrb1ts mutant              A decade ago, function essential Ran <br /> was crossed msn5&#226;&#338;&#172; strain wild-type            regulator, RanGEF1, fully appreciated, ts mutation <br /> single mutant, double mutant spores re-            (srm1-1) yeast homolog (SRM1/PRP20/MTR1) was <br /> sulting tetratype asci tested relative mating    isolated suppressor mating defect haploid <br /> proficiency using quantitative mating assay performed         cells lacking pheromone receptors (Clark Sprague <br /> at semipermissive temperature (30&#226;&#172;&#353;) yrb1ts alleles.     1989). Unlike ts mutations yeast RanGEF1 iden- <br /> Both yrb1-51 yrb1-52 mutations restored mat-        tified subsequently (Amberg et al. 1993), srm1-1 does <br /> ing efficiency msn5&#226;&#338;&#172; mutant essentially           cause dramatic nuclear accumulation poly(A)&#226;&#171;&#185; <br /> wild-type level (Table 5). This suppression consistent       RNA restrictive temperature (Kadowaki et al. 1993; <br /> with idea higher cytoplasmic pool Ste5 (due       effect nuclear protein import exam- <br /> to inefficient nuclear import yrb1ts mutants)        ined), left open possibility Ran <br /> compensates inefficient pheromone-stimulated            GTPase cycle role mating distinct from <br /> export nucleus (due msn5&#226;&#338;&#172; mutation).            function nucleocytoplasmic transport. We have <br />  1102                                                   M. Ku&#194;&#168;nzler et al. <br />  <br /> shown independently isolated suppressors             required cytoskeletal reorganization, is <br /> of early late defects mating pathway                necessary polarized cell growth mating <br /> alterations essential Ran regulator, RanBP1,            partner (Blondel et al. 1999; Shimada et al. 2000). The <br /> that clearly impair nuclear protein import, per-           role recruitment Ste5 shmoo tip be <br /> missive temperature. We Ste5                similar, given interaction Ste5 Bem1, pro- <br /> scaffold protein, shuttles nucleus               tein involved polarized cell growth (Leberer et al. <br /> cytoplasm required pheromone response,               1997). Since relocalization Far1 (Blondel et al. 1999; <br /> mislocalized cytosol mutants. Consistent           Shimada et al. 2000) Ste5 (Pryciak Huntress <br /> with finding, yrb1ts alleles suppressed partial      1998; Mahanty et al. 1999) transient, nuclear import <br /> mating defect null mutation MSN5, en-                defect kept proteins nucleus longer <br /> codes nuclear receptor required pheromone-                 increase mating efficiency cells the <br /> stimulated export Ste5.                                         cell fusion process nonoptimal. <br />    In retrospect, obvious perturbation               In agreement model, yrb1-51 <br /> of nucleocytoplasmic transport affect yeast mating           yrb1-52 mutations increased cytoplasmic <br /> since response pheromone requires: (a)                concentration Ste5 able suppress the <br /> signal carrier enter nucleus induce transcrip-          mating defect msn5 null mutation. The MSN5 gene <br /> tion genes, (b) RNPs containing newly synthe-              encodes nuclear export receptor &#226;?&#164;-importin <br /> sized mRNAs mating-specific components exit                family (Kaffman et al. 1998; DeVit Johnston 1999) <br /> nucleus, (c) translation products             identified STE21 null mutation <br /> of transcripts recruited nucleus            locus confers partial mating defect (Akada et al. <br /> in support late mating events, like karyogamy (Rose             1996; Alepuz et al. 1999; Blondel et al. 1999). Impaired <br /> 1996). Accordingly, known compo-               nuclear protein import yrb1ts mutants keeps Ste5 <br /> nents pheromone response machinery, pre-               (and proteins required mating) in <br /> existing pheromone induced, localized               cytosol longer, need their <br /> nucleus. On hand, shown                 continuous Msn5-dependent reexport nu- <br /> two components, Far1 (Blondel et al. 1999) Ste5                cleus. Also consistent model, MATa yrb1- <br /> (Pryciak Huntress 1998; Mahanty et al. 1999),                  51 MATa yrb1-52 mutants able respond nor- <br /> rapidly relocalize nucleus site cell            mally &#226;?&#163;-factor conditions (by inducing the <br /> fusion (shmoo tip) pheromone treatment. Our                   pheromone-responsive reporter FUS1-lacZ) un- <br /> results suggest following mechanism explain             dergoing G1 arrest (as judged standard halo <br /> suppression mating defects mutations nucleocy-            bioassay; data shown); hence, mutants not <br /> toplasmic transport factors.                                       defective early signaling events. If, however, proteins <br />    Because yrb1ts alleles described defective        required late events mating maintained longer <br /> in nuclear protein import, impaired             cytosol yrb1 mutants defective nuclear <br /> in poly(A)&#226;&#171;&#185; RNA export, act suppressors             protein import, efficiency mating en- <br /> mating defects, presumably import defect            hanced, observed. Finally, model also <br /> leads suppression. Also, recalled             explain spectrum mating defects sup- <br /> yrb1-51 isolated basis rescue mating          pressed yrb1-51, yrb1-52, srm1-1. All muta- <br /> debility fus1&#226;&#338;&#172; mutant, intact phero-             tions suppressed gene products targeted, <br /> mone response pathway partially mating                 directly indirectly, shmoo tip pheromone <br /> defective. Likewise, yrb1-52 isolated basis          treatment involved pheromone- <br /> its ability rescue ste5 missense mutations (and          induced remodeling cortical cytoskeleton (Leb- <br /> so ste4 ste7 missense mutations), unable            erer et al. 1997). Pheromone receptors (Ste2 Ste3) <br /> to rescue null alleles genes (Katz et al.            localized shmoo tip participate cell <br /> 1987). These considerations indicate residual sig-            polarity determination mating partner discrimina- <br /> naling pheromone response pathway                   tion (Jackson et al. 1991). Ste4 (G&#226;?&#164;), released <br /> present suppression occur. How impair-                receptors G&#226;?&#164;&#226;?&#165; complex response to <br /> ment nuclear protein import improve mating                  pheromone binding, responsible direct recruit- <br /> efficiency partially mating-defective mutants?             ment major regulators cytoskeletal structure and <br /> One reasonable scenario nuclear import cer-             cell polarity, including Ste20 (Leeuw et al. 1998), Far1 <br /> tain signaling components results downregulation             (Shimada et al. 2000), Ste5 (Inouye et al. 1997b; <br /> the mating response. If so, mutations reduce              Pryciak Huntress 1998). These components in- <br /> rate nuclear entry factor(s) enhance           teract molecules, like Bem1, contribute <br /> the efficiency mutants partially defective        reorganization cytoskeleton (Leberer et al. <br /> in mating. Both Far1 Ste5 reasonable candidates            1997). Even Ste7, encodes MAPK kinase <br /> for factors. For Far1, pheromone-induced relo-            (MAPKK) pheromone-responsive MAPK cascade, <br /> calization nucleus shmoo tip              suppressed yrb1-52, albeit weakly <br />                                                     Ran-Binding-Protein-1 Yeast Mating                                                   1103 <br />  <br /> (Katz et al. 1987), bound Ste5,                        European Molecular Biology Organization Long-Term Fellowship <br />                                                                            funds provided Swiss National Science Foundation (to M. <br /> tethered shmoo tip interaction G&#226;?&#164;&#226;?&#165;. <br />                                                                            Ku&#194;&#168;nzler), National Science Foundation Postdoctoral Fellowship <br /> Finally, Fus1 integral membrane protein localized                    DMB-8807575 National Cancer Institute Postdoctoral Traineeship <br /> to tip mating projection involved                     CA09041 (to J. Trueheart), Postdoctoral Fellowship the <br /> the cell fusion step mating (Trueheart Fink                         Italian-American Cancer Foundation (to C. Sette), National Insti- <br /> 1989). Indeed, mutational analysis Fus1 indicates                  tutes Health Research Grant GM21841 (to J. Thorner), by <br />                                                                            facilities provided Berkeley campus Cancer Research La- <br /> its large O-glycosylated exocellular domain dispens- <br />                                                                            boratory. <br /> able function SH3 domain two <br /> potential actin-binding motifs relatively short, cyto- <br /> solic, C-terminal tail essential function, sug- <br /> gesting primary role Fus1 mating                                             LITERATURE CITED <br /> process contribution modifying structure                     Aebi, M., M. W. Clark, U. Vijayraghavan J. Abelson, 1990 <br />                                                                                A yeast mutant, PRP20, altered mRNA metabolism mainte- <br /> of cytoskeleton (J. Trueheart J. Thorner,                              nance nuclear structure, defective gene homologous <br /> unpublished results).                                                          human gene RCC1, involved control of <br />    A primary defect analyzed yrb1ts mutants                          chromosome condensation. Mol. Gen. Genet. 224: 72&#226;&#8364;&#8220;80. <br />                                                                            Aitchison, J. D., G. Blobel M. P. Rout, 1996 Kap104p: a <br /> nucleocytoplasmic transport explain ob-                         karyopherin involved nuclear transport messenger RNA <br /> served mitotic phenotypes, similar mitotic distur-                       binding proteins. Science 274: 624&#226;&#8364;&#8220;627. <br /> bances reported mutants deficient                         Akada, R., L. Kallal, D. I. Johnson J. Kurjan, 1996 Genetic <br />                                                                                relationships G protein &#226;?&#164;&#226;?&#165; complex, Ste5p, Ste20p <br /> other factors involved nucleocytoplasmic transport,                         Cdc42p: investigation effector roles yeast phero- <br /> for example, Srp1 (&#226;?&#163;-importin; Loeb et al. 1995),                          mone response pathway. Genetics 143: 103&#226;&#8364;&#8220;117. <br /> Cse1 (Xiao et al. 1993), required reexport                    Alepuz, P. M., D. Matheos, K. W. Cunningham F. Estruch, <br />                                                                                1999 The Saccharomyces cerevisiae RanGTP-binding protein <br /> of Srp1 nucleus (Ku&#194;&#168;nzler Hurt 1998).                             Msn5p involved different signal transduction pathways. Ge- <br /> Both yrb1-51 yrb1-52 caused cytoplasmic accumula-                          netics 153: 1219&#226;&#8364;&#8220;1231. <br /> tion different reporter proteins SV40                      Amberg, D. C., A. L. Goldstein C. N. Cole, 1992 Isolation <br />                                                                                characterization RAT1: essential gene Saccharomyces <br /> NLS displayed synthetic lethality different                       cerevisiae required efficient nucleocytoplasmic trafficking <br /> srp1ts mutations, rsl1ts mutation (&#226;?&#164;-importin),                     mRNA. Genes Dev. 6: 1173&#226;&#8364;&#8220;1189. <br /> with cold-sensitive cse1 allele, suggesting impair-                 Amberg, D. C., M. Fleischmann, I. Stagljar, C. N. Cole M. <br />                                                                                Aebi, 1993 Nuclear PRP20 protein required mRNA ex- <br /> ment import nuclear proteins classical                        port. EMBO J. 12: 233&#226;&#8364;&#8220;241. <br /> NLS explain mitotic defects observed.                              Atkinson, N. S., R. W. Dunst A. K. Hopper, 1985 Characteriza- <br />    Although study rigorously rule                           tion essential Saccharomyces cerevisiae gene related RNA <br />                                                                                processing: cloning RNA1 generation new allele with <br /> possibility Ran RanBP1 play role                           novel phenotype. Mol. Cell. Biol. 5: 907&#226;&#8364;&#8220;915. <br /> the mating pathway independent functions                       Bailer, S. M., S. Siniossoglou, A. V. Podtelejnikov, A. Hellwig, M. <br /> nucleocytoplasmic transport, based findings pre-                        Mann et al., 1998 Nup116p Nup100p interchangeable <br />                                                                                conserved motif constitutes docking site for <br /> sented here, observed suppression mating defects                        mRNA transport factor Gle2p. EMBO J. 17: 1107&#226;&#8364;&#8220;1119. <br /> and defects mitosis caused yrb1-51                       Bardwell, L., J. G. Cook, C. Inouye J. Thorner, 1994 Signal <br />                                                                                propagation regulation mating pheromone pathway <br /> yrb1-52 mutations likely direct consequences                          yeast Saccharomyces cerevisiae. Dev. Biol. 166: 363&#226;&#8364;&#8220;379. <br /> of impaired import nuclear proteins. Because                     Bartel, P. L., S. Fields, 1995 Analyzing protein-protein interac- <br /> relatively slow onset, apparent mRNA export defect                         tions using two-hybrid system. Methods Enzymol. 254: 241&#226;&#8364;&#8220; <br />                                                                                263. <br /> manifested yrb1-51 allele reflect indirect                   Ba&#194;&#168;umer, M., M. Ku&#194;&#168;nzler, P. Steigemann, G. H. Braus S. <br /> consequence primary defect import.                                     Irniger, 2000 Yeast Ran-binding protein Yrb1p required for <br />                                                                                efficient proteolysis cell cycle regulatory proteins Pds1p <br />    We thank Markus Aebi (Federal Institute Technology, Zu&#194;&#168;rich,             Sic1p. J. Biol. Chem. 275: 38929&#226;&#8364;&#8220;38937. <br /> Switzerland), Pierre Belhumeur (McGill University, Montreal), Laura        Beddow, A. L., S. A. Richards, N. R. Orem I. G. Macara, <br /> Davis (Brandeis University, Waltham, MA), Gerald Fink (Massachus-              1995 The Ran/TC4 GTPase-binding domain: identification by <br /> setts Insitute Technology, Boston), Molly Fitzgerald-Hayes (Univer-         expression cloning characterization conserved sequence <br /> sity Massachussetts, Amherst, MA), Anita Hopper (Pennsylvania               motif. Proc. Natl. Acad. Sci. USA 92: 3328&#226;&#8364;&#8220;3332. <br /> State University, Hershey, PA), Masayasu Nomura (University Cali-       Belhumeur, P., A. Lee, R. Tam, T. DiPaolo, N. Fortin et al., 1993 <br /> fornia, Irvine, CA), Pamela Silver (Dana Farber Cancer Center, Bos-            GSP1 GSP2, genetic suppressors prp20-1 mutant in <br />                                                                                Saccharomyces cerevisiae : GTP-binding proteins involved the <br /> ton), Franc&#194;&#184;oise Stutz (Centre Hospitalier Universitaire Vaudois, Lau- <br />                                                                                maintenance nuclear organization. Mol. Cell. Biol. 13: 2152&#226;&#8364;&#8220; <br /> sanne, Switzerland), Alan Tartakoff (Case Western Reserve University,          2161. <br /> Cleveland), Linda S. Huang (University California, San Francisco),      Blondel, M., P. M. Alepuz, L. S. Huang, S. Shaham, G. Ammerer <br /> and Karsten Weis (University California, Berkeley, CA)              et al., 1999 Nuclear export Far1p response pheromones <br /> generous gifts reagents; Stefan Irniger (Georg-August-Universita&#194;&#168;t,         requires export receptor Msn5/Ste21p. Genes Dev. 13: 2284&#226;&#8364;&#8220; <br /> Go&#194;&#168;ttingen, Germany) Jeff Flick (Vanderbilt University, Nashville,         2300. <br /> TN) sharing unpublished results; Stephanie Richards Ian        Boeke, J. D., J. Trueheart, G. Natsoulis G. R. Fink, 1987 5-Flu- <br />                                                                                oroorotic acid selective agent yeast molecular genetics. <br /> Macara (University Virginia, Charlottesville, VA) advice and <br />                                                                                Methods Enzymol. 154: 164&#226;&#8364;&#8220;175. <br /> material assistance early stages work. We grateful   Booth, J. W., K. D. Belanger, M. I. Sannella L. I. Davis, 1999 <br /> members laboratory, especially Jeanette Gowen Cook, Elana               The yeast nucleoporin Nup2p involved nuclear export of <br /> Swartzman, Namrita Dhillon, Lee Bardwell, Carla Inouye,                importin alpha/Srp1p. J. Biol. Chem. 274: 32360&#226;&#8364;&#8220;32367. <br /> technical assistance valuable discussions. This work supported     Botstein, D., S. C. Falco, S. E. Stewart, M. Brennan, S. Scherer <br />  1104                                                              M. Ku&#194;&#168;nzler et al. <br />     et al., 1979 Sterile host yeasts (SHY): eukaryotic                nuclear guanine nucleotide release protein members the <br />     biological containment recombinant DNA experiments. Gene                  Ras superfamily. EMBO J. 12: 2929&#226;&#8364;&#8220;2937. <br />     8: 17&#226;&#8364;&#8220;24.                                                                 Kaffman, A., E. K. O&#226;&#8364;&#8482;Shea, 1999 Regulation nuclear localiza- <br /> Bradford, M. M., 1976 A rapid sensitive method quantita-                  tion: key door. Annu. Rev. Cell Dev. Biol. 15: 291&#226;&#8364;&#8220;339. <br />     tion microgram quantities protein utilizing principle           Kaffman, A., N. Rank Miller, E. M. O&#226;&#8364;&#8482;Neill, L. S. Huang E. K. <br />     protein-dye binding. Anal. Biochem. 72: 248&#226;&#8364;&#8220;254.                           O&#226;&#8364;&#8482;Shea, 1998 The receptor Msn5 exports phosphorylated <br /> Carlson, M., D. Botstein, 1982 Two differentially regulated                   transcription factor Pho4 nucleus. Nature 396: 482&#226;&#8364;&#8220; <br />     mRNAs different 5&#226;&#172;&#732; ends encode secreted intracellular                486. <br />     forms yeast invertase. Cell 28: 145&#226;&#8364;&#8220;154.                               Kahana, J. A., D. W. Cleveland, 1999 Beyond nuclear trans- <br /> Clark, K. L., G. F. Sprague, 1989 Yeast pheromone response                    port. Ran-GTP determinant spindle assembly. J. Cell Biol. <br />     pathway: characterization suppressor restores mating             146: 1205&#226;&#8364;&#8220;1210. <br />     receptorless mutants. Mol. Cell. Biol. 9: 2682&#226;&#8364;&#8220;2694.                      Kaiser, C., S. Michaelis A. Mitchell, 1994 Methods Yeast <br /> Corbett, A. H., P. A. Silver, 1997 Nucleocytoplasmic transport                Genetics: A Laboratory Course Manual. Cold Spring Harbor Labora- <br />     macromolecules. Microbiol. Rev. 61: 193&#226;&#8364;&#8220;211.                               tory Press, Cold Spring Harbor, NY. <br /> Del Sal, G., G. Manfioletti C. Schneider, 1988 A one-tube                 Katz, M. E., J. Ferguson S. I. Reed, 1987 Temperature-sensitive <br />     plasmid DNA mini-preparation suitable sequencing. Nucleic                 lethal pseudorevertants ste mutations Saccharomyces cerevisiae. <br />     Acids Res. 16: 9878.                                                          Genetics 115: 627&#226;&#8364;&#8220;636. <br /> DeVit, M. J., M. Johnston, 1999 The nuclear exportin Msn5                 Koepp, D. M., D. H. Wong, A. H. Corbett P. A. Silver, 1996 <br />     required nuclear export Mig1 glucose repressor               Dynamic localization nuclear import receptor inter- <br />     Saccharomyces cerevisiae. Curr. Biol. 9: 1231&#226;&#8364;&#8220;1241.                           actions transport factors. J. Cell Biol. 133: 1163&#226;&#8364;&#8220;1176. <br /> Doye, V., R. Wepf E. C. Hurt, 1994 A novel nuclear pore                   Kondo, K., M. Inouye, 1992 Yeast NSR1 protein struc- <br />     protein Nup133p distinct roles poly (A)&#226;&#171;&#185; RNA transport                tural similarity mammalian nucleolin involved pre-rRNA <br />     nuclear pore distribution. EMBO J. 13: 6062&#226;&#8364;&#8220;6075.                         processing. J. Biol. Chem. 267: 16252&#226;&#8364;&#8220;16258. <br /> Elion, E. A., 1998 Routing MAP kinase cascades. Science 281: 1625&#226;&#8364;&#8220;            Ku&#194;&#168;nzler, M., E. C. Hurt, 1998 Cse1p functions nuclear <br />     1626.                                                                         export receptor importin yeast. FEBS Lett. 433: 185&#226;&#8364;&#8220;190. <br /> Elledge, S. J., R. W. Davis, 1988 A family versatile centro-           Ku&#194;&#168;nzler, M., T. Gerstberger, F. Stutz, F. R. Bischoff E. Hurt, <br />     meric vectors designed use sectoring-shuffle mutagene-             2000 Yeast Ran-binding protein 1 (Yrb1) shuttles nu- <br />     sis assay Saccharomyces cerevisiae. Gene 70: 303&#226;&#8364;&#8220;312.                      cleus cytoplasm exported nucleus a <br /> Enenkel, C., G. Blobel M. Rexach, 1995 Identification                    CRM1(XPO1)-dependent pathway. Mol. Cell. Biol. 20: 4295&#226;&#8364;&#8220; <br />     yeast karyopherin heterodimer targets import substrate                4308. <br />     mammalian nuclear pore complexes. J. Biol. Chem. 270: 16499&#226;&#8364;&#8220;              Leberer, E., D. Y. Thomas M. Whiteway, 1997 Pheromone <br />     16502.                                                                        signalling polarized morphogenesis yeast. Curr. Opin. <br /> Fleischmann, M., I. Stagljar M. Aebi, 1996 Allele-specific                    Genet. Dev. 7: 59&#226;&#8364;&#8220;66. <br />     suppression Saccharomyces cerevisiae prp20 mutation overex-       Lee, M. S., M. Henry P. A. Silver, 1996 A protein shuttles <br />     pression nuclear serine threonine protein kinase. Mol. Gen.              nucleus cytoplasm important mediator <br />     Genet. 250: 614&#226;&#8364;&#8220;625.                                                          RNA export. Genes Dev. 10: 1233&#226;&#8364;&#8220;1246. <br /> Flick, J. S., J. Thorner, 1993 Genetic biochemical charac-            Leeuw, T., C. Wu, J. D. Schrag, M. Whiteway, D. Y. Thomas et al., <br />     terization phosphatidylinositol-specific phospholipase C              1998 Interaction G-protein beta-subunit conserved <br />     Saccharomyces cerevisiae. Mol. Cell. Biol. 13: 5861&#226;&#8364;&#8220;5876.                     sequence Ste20/PAK family protein kinases. Nature 391: 191&#226;&#8364;&#8220; <br /> Giebel, L. B., R. A. Spritz, 1990 Site-directed mutagenesis using             195. <br />     double-stranded DNA fragment PCR primer. Nucleic Acids             Lew, D. J., 2000 Cell-cycle checkpoints ensure coordination <br />     Res. 18: 4947.                                                                nuclear cytoplasmic events Saccharomyces cerevisiae. <br /> Gietz, D., A. St. Jean, R. A. Woods R. H. Schiestl, 1992 Im-                  Curr. Opin. Genet. Dev. 10: 47&#226;&#8364;&#8220;53. <br />     proved method high efficiency transformation intact yeast          Loeb, J. D. J., L. I. Davis G. R. Fink, 1993 NUP2, novel yeast <br />     cells. Nucleic Acids Res. 20: 1425.                                           nucleoporin, functional overlap proteins the <br /> Go&#194;&#168;rlich, D., U. Kutay, 1999 Transport cell nu-                   nuclear pore complex. Mol. Biol. Cell 4: 209&#226;&#8364;&#8220;222. <br />     cleus cytoplasm. Annu. Rev. Cell Dev. Biol. 15: 607&#226;&#8364;&#8220;660.          Loeb, J. D. J., G. Schlenstedt, D. Pellman, D. Kornitzer, P. A. <br /> Hanahan, D., 1983 Studies transformation Escherichia coli              Silver et al., 1995 The yeast nuclear import receptor required <br />     plasmids. J. Mol. Biol. 166: 557&#226;&#8364;&#8220;580.                                         mitosis. Proc. Natl. Acad. Sci. USA 92: 7647&#226;&#8364;&#8220;7651. <br /> Harlow, E., D. Lane, 1988 Antibodies: A Laboratory Manual.                Macara, I., A. Brownawell K. Welch, 2000 Ran, pp. 198&#226;&#8364;&#8220;221 <br />     Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.                  GTPases, edited A. Hall. Oxford University Press, New York. <br /> Hellmuth, K., D. Lau, R. Bischoff, M. Ku&#194;&#168;nzler, G. Simos et al.,              Madden, K., M. Snyder, 1998 Cell polarity morphogenesis <br />     1998 Yeast Los1p properties exportin-like nucleocy-                 budding yeast. Annu. Rev. Microbiol. 52: 687&#226;&#8364;&#8220;744. <br />     toplasmic transport factor tRNA. Mol. Cell. Biol. 18: 6374&#226;&#8364;&#8220;           Mahanty, S. K., Y. Wang, F. W. Farley E. A. Elion, 1999 Nu- <br />     6386.                                                                         clear shuttling yeast scaffold Ste5 required recruitment <br /> Hetzer, M., D. Bilbao-Cortes, T. C. Walther, O. J. Gruss                      plasma membrane activation mating MAPK <br />     I. W. Mattaj, 2000 GTP hydrolysis Ran required nu-                  cascade. Cell 98: 501&#226;&#8364;&#8220;512. <br />     clear envelope assembly. Mol. Cell 5: 1013&#226;&#8364;&#8220;1024.                          Mueller, L., V. C. Cordes, R. Bischoff H. Ponstingl, 1998 <br /> Hill, J. E., A. M. Myers, T. J. Koerner A. Tzagoloff, 1986                    Human RanBP3, group nuclear RanGTP binding proteins. <br />     Yeast/E. coli shuttle vectors multiple unique restriction sites.         FEBS Lett. 427: 330&#226;&#8364;&#8220;336. <br />     Yeast 2: 163&#226;&#8364;&#8220;167.                                                         Nehrbass, U., E. Fabre, S. Dihlmann, W. Herth E. C. Hurt, <br /> Hurt, E., S. Hannus, B. Schmelzl, D. Lau, D. Tollervey et al., 1998               1993 Analysis nucleo-cytoplasmic transport thermosensi- <br />     A novel vivo assay reveals inhibition ribosomal nuclear export          tive mutant nuclear pore protein NSP1. Eur. J. Cell Biol. <br />     Ran-cycle nucleoporin mutants. J. Cell Biol. 144: 389&#226;&#8364;&#8220;401.             62: 1&#226;&#8364;&#8220;12. <br /> Inouye, C., N. Dhillon, T. Durfee, P. C. Zambryski J. Thorner,            Oki, M., E. Noguchi, N. Hayashi T. Nishimoto, 1998 Nuclear <br />     1997a Mutational analysis STE5 yeast Saccharomyces                  protein import, mRNA export, defective Saccharo- <br />     cerevisiae : application differential interaction trap assay         myces cerevisiae mutants produce temperature-sensitive <br />     examining protein-protein interactions. Genetics 147: 479&#226;&#8364;&#8220;492.                forms Ran GTPase homologue Gsp1p. Mol. Gen. Genet. <br /> Inouye, C., N. Dhillon J. Thorner, 1997b Ste5 RING-H2 do-                     257: 624&#226;&#8364;&#8220;634. <br />     main: role Ste4-promoted oligomerization yeast phero-              Ouspenski, I. I., 1998 A RanBP1 mutation does visibly <br />     mone signaling. Science 278: 103&#226;&#8364;&#8220;106.                                         affect nuclear import reveal additional functions Ran <br /> Jackson, C. L., J. B. Konopka L. H. Hartwell, 1991 S. cerevisiae              GTPase system. Exp. Cell Res. 244: 171&#226;&#8364;&#8220;183. <br />     alpha pheromone receptors activate novel signal transduction            Ouspenski, I. I., U. W. Mueller, A. Matynia, S. Sazer, S. J. Elledge <br />     pathway mating partner discrimination. Cell 67: 389&#226;&#8364;&#8220;402.                  et al., 1995 Ran-binding protein-1 essential component of <br /> Kadowaki, T., D. Goldfarb, L. M. Spitz, A. M. Tartakoff M.                    Ran/RCC1 molecular switch budding yeast. J. Biol. <br />     Ohno, 1993 Regulation RNA processing transport                    Chem. 270: 1975&#226;&#8364;&#8220;1978. <br />                                                       Ran-Binding-Protein-1 Yeast Mating                                                     1105 <br /> Pearson, W. R., D. J. Lipman, 1988 Improved tools biological          Sikorski, R. S., R. Hieter, 1989 A shuttle vectors and <br />     sequence analysis. Proc. Natl. Acad. Sci. USA 85: 2444&#226;&#8364;&#8220;2448.                  yeast host strains designed efficient manipulation DNA in <br /> Peterson, C., N. Orem, J. Trueheart, J. W. Thorner I. G.                      Saccharomyces cerevisiae. Genetics 122: 19&#226;&#8364;&#8220;27. <br />     Macara, 2000 Random mutagenesis functional analysis                Sprague, G. F. J., 1991 Assay yeast mating reaction. Methods <br />     Ran-binding protein, RanBP1. J. Biol. Chem. 275: 4081&#226;&#8364;&#8220;4091.               Enzymol. 194: 77&#226;&#8364;&#8220;93. <br /> Piggott, J. R., R. Rai B. L. A. Carter, 1982 A bifunctional               Sprague, G. F. J., J. Thorner, 1992 Pheromone response and <br />     gene product involved phases yeast cell cycle. Nature           signal transduction mating process Saccharomyces <br />     298: 391&#226;&#8364;&#8220;393.                                                                 cerevisiae, pp. 657&#226;&#8364;&#8220;744 The Molecular Cellular Biology the <br /> Plafker, K., I. G. Macara, 2000 Facilitated nucleocytoplasmic                 Yeast Saccharomyces: Gene expression, edited E. W. Jones, J. R. <br />     shuttling Ran binding protein RanBP1. Mol. Cell. Biol.                 Pringle J. R. Broach. Cold Spring Harbor Laboratory Press, <br />     20: 3510&#226;&#8364;&#8220;3521.                                                                Cold Spring Harbor, NY. <br /> Pryciak, P. M., F. A. Huntress, 1998 Membrane recruitment                 Stade, K., C. S. Ford, C. Guthrie K. Weis, 1997 Exportin 1 <br />     kinase cascade scaffold protein Ste5 G&#226;?&#164;&#226;?&#165; complex                (Crm1p) essential nuclear export factor. Cell 90: 1041&#226;&#8364;&#8220;1050. <br />     underlies activation yeast pheromone response pathway.             Stone, E. M., P. Heun, T. Laroche, L. Pillus S. M. Gasser, <br />     Genes Dev. 12: 2684&#226;&#8364;&#8220;2697.                                                     2000 MAP kinase signaling induces nuclear reorganization in <br /> Reed, S. I., 1980 The selection S. cerevisiae mutants defective             budding yeast. Curr. Biol. 10: 373&#226;&#8364;&#8220;382. <br />     start event cell division. Genetics 95: 561&#226;&#8364;&#8220;577.                   Studier, F. W., 1991 Use bacteriophage T7 lysozyme improve <br /> Rose, M. D., 1996 Nuclear fusion yeast Saccharomyces cerevisiae.           inducible T7 expression system. J. Mol. Biol. 219: 37&#226;&#8364;&#8220;44. <br />     Annu. Rev. Cell Biol. 12: 663&#226;&#8364;&#8220;695.                                        Taura, T., H. Krebber P. A. Silver, 1998 A member the <br /> Ryan, K. J., S. R. Wente, 2000 The nuclear pore complex:                    Ran-binding protein family, Yrb2p, involved nuclear protein <br />     protein machine bridging nucleus cytoplasm. Curr. Opin.               export. Proc. Natl. Acad. Sci. USA 95: 7427&#226;&#8364;&#8220;7432. <br />     Cell Biol. 12: 361&#226;&#8364;&#8220;371.                                                   Thompson, J. D., D. G. Higgins T. J. Gibson, 1994 CLUSTAL <br /> Sambrook, J., E. Fritsch T. Maniatis, 1989 Molecular Cloning:                 W: improving sensitivity progressive multiple sequence <br />     A Laboratory Manual, Ed. 2. Cold Spring Harbor Laboratory Press,              alignment sequence weighting, position-specific gap <br />     Cold Spring Harbor, NY.                                                       penalties weight matrix choice. Nucleic Acids Res. 22: 4673&#226;&#8364;&#8220; <br /> Sazer, S., M. Dasso, 2000 The Ran decathlon: multiple roles                   4680. <br />     Ran. J. Cell Sci. 113: 1111&#226;&#8364;&#8220;1118.                                      Trueheart, J., G. R. Fink, 1989 The yeast cell fusion protein <br /> Schlenstedt, G., D. H. Wong, D. M. Koepp P. A. Silver, 1995                   FUS1 O-glycosylated spans plasma membrane. Proc. <br />     Mutants yeast Ran binding protein defective nuclear               Natl. Acad. Sci. USA 86: 9916&#226;&#8364;&#8220;9920. <br />     transport. EMBO J. 14: 5367&#226;&#8364;&#8220;5378.                                         Vetter, I., C. Nowak, T. Nishimoto, J. Kuhlmann A. Witting- <br /> Schlenstedt, G., E. Smirnova, R. Deane, J. Solsbacher, U. Kutay                   hofer, 1999 Structure Ran-binding domain complexed <br />     et al., 1997 Yrb4p, yeast Ran-GTP-binding protein involved               Ran bound GTP analogue: implications nuclear <br />     import ribosomal protein L25 nucleus. EMBO J. 16:                 transport. Nature 398: 39&#226;&#8364;&#8220;46. <br />     6237&#226;&#8364;&#8220;6249.                                                                Wong, D. H., A. H. Corbett, H. M. Kent, M. Stewart P. A. <br /> Schroeder, A. J., X. H. Chen, Z. Xiao M. Fitzgerald-Hayes,                    Silver, 1997 Interaction small GTPase Ran/Gsp1p <br />     1999 Genet <br /> </body></html>