<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Basal Body Duplication and Maintenance Require One <br /> Member of the Tetrahymena thermophila Centrin Gene <br /> Family <br />  <br /> Centrins, small calcium binding EF-hand proteins, function in the duplication of a variety of microtubule organizing <br /> centers. These include centrioles in humans, basal bodies in green algae, and spindle pole bodies in yeast. The ciliate <br /> Tetrahymena thermophila contains at least four centrin genes as determined by sequence homology, and these have <br /> distinct localization and expression patterns. CEN1&#226;&#8364;&#8482;s role at the basal body was examined more closely. The Cen1 protein <br /> localizes primarily <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> two locations: one is the site at the <span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span> of the basal body where duplication is initiated. The other <br /> is the transition zone between the basal body and axoneme. CEN1 is an essential gene, the deletion of which results in <br /> the loss of basal bodies, which is likely due to defects in both basal body duplication and basal body maintenance. <br /> Analysis of the three other centrins indicates that two of them function at microtubule-rich structures unique to ciliates, <br /> whereas the fourth is not expressed under conditions examined in this study, although when artificially expressed it <br /> localizes to basal bodies. This study provides evidence that in addition to its previously known function in the <br /> duplication of basal bodies, centrin is also important for the integrity of these organelles. <br />  <br />  <br /> INTRODUCTION                                                             rioles are distinguished by 9-fold symmetry of microtubules <br />                                                                          in a &#226;&#8364;&#339;flower&#226;&#8364;&#65533; arrangement, with the majority of metazoans <br /> Centrins, also called caltractins, are widely conserved mem-             displaying triplet microtubules at these positions. Basal bod- <br /> bers of the EF-hand superfamily of calcium-binding proteins              ies are structurally similar to centrioles and differ primarily <br /> (Salisbury, 1995). Although sharing &#226;&#172;&#402;50% identity with the               in the manner in which they organize microtubules. Cilia <br /> ubiquitous calcium sensor calmodulin, centrins perform dis-              and flagella, which consist of nine doublet microtubules <br /> tinct functions within the eukaryotic cell. In particular, cen-          surrounding a central microtubule pair, are nucleated <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> trins have a variety of roles at microtubule organizing cen-             the distal end of the basal body (reviewed in Beisson and <br /> ters (MTOCs) and microtubule-based structures (Salisbury,                Wright, 2003). In some situations, it is the identical organelle <br /> 1995; Vaughn and Harper, 1998; Chapman et al., 2000). Ad-                that acts as a centriole and then is converted to a basal body, <br /> ditionally, centrins are constituents of calcium-responsive              or vice versa, at particular points in the life cycle of the cell. <br /> contractile fibers found in many protists (Salisbury et al.,             This is the case in many vertebrate cells where the oldest <br /> 1984; Wright et al., 1985).                                              centriole is capable of forming a single &#226;&#8364;&#339;9 &#226;&#171;&#185; 0&#226;&#8364;&#65533; cilium <br />    One of the widely recognized roles for centrin is in the              (known as the primary cilium) that is assembled during the <br /> duplication of microtubule organizing centers, including                 G1 phase of the cell cycle and disassembled during mitosis <br /> centrosomes, basal bodies, and the yeast spindle pole body               (Jensen et al., 1987). The budding yeast spindle pole body is <br /> (Baum et al., 1986; Salisbury et al., 2002; Koblenz et al., 2003).       structurally distinct from the centriole and basal body. This <br /> Centrosomes consist of two orthogonally oriented centrioles              trilaminar structure remains embedded within the nuclear <br /> surrounded by amorphous pericentriolar material from                     envelope throughout the cell cycle. Spindle and astral mi- <br /> which spindle and astral microtubules are organized. Cent-               crotubules emanate from the inner and outer plaques, re- <br />                                                                          spectively. Although dissimilar in appearance to the centro- <br />                                                                          some, the spindle pole body performs the same task of <br /> This article was published online ahead of print in MBC in Press         organizing mitotic and meiotic spindles, and some compo- <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E04 &#226;&#8364;&#8220;10 &#226;&#8364;&#8220; 0919)          nents of these organelles, including centrin, are conserved <br /> on June 8, 2005. <br />                                                                          between these divergent organelles (Adams and Kilmartin, <br /> &#226;&#8211;&#161;  The online version of this article contains supplemental material <br /> D <br />                                                                          2000). <br /> at MBC Online (http://www.molbiolcell.org).                                 Temperature-sensitive mutations in the yeast centrin gene <br /> Present addresses: &#226;&#8364;&#160; Institute for Molecular Bioscience, University of   CDC31 result in failure in an early step of spindle pole body <br /> Queensland, Brisbane QLD 4072, Australia; &#194;&#167; SUNY Upstate Medi-           duplication and a cell cycle arrest in G2/M phase, such that <br /> cal University, 750 East Adams St., Syracuse, NY 13210-2375; &#229;&#8218;&#168; The       cells arrest with a single, enlarged spindle pole body (SPB) <br /> American Association for the Advancement of Science, 1200 New            (Byers, 1981; Baum et al., 1986). Like SPB duplication, basal <br /> York Ave. NW, Washington, DC 20005.                                      body and centriole duplication is dependent on centrin func- <br /> Address correspondence to: Mark Winey (mark.winey@colorado.edu).         tion, as demonstrated by <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) experi- <br />  <br /> 3606                                                                                      &#194;&#169; 2005 by The American Society for Cell Biology <br />                                                                                                                             Tetrahymena Centrins <br />  <br />  <br /> ments in both human cells and Chlamydomonas. Thus far,              MATERIALS AND METHODS <br /> three centrin genes have been cloned from human cells (Lee <br /> and Huang, 1993; Errabolu et al., 1994; Middendorp et al.,          Strains, Culture Conditions, and Matings <br /> 1997). When human centrin2 levels are reduced by RNAi,              T. thermophila strains B2086, CU427, and CU428 (all generous gifts from Dr. <br />                                                                     Peter Bruns, Cornell University, Ithaca, NY) were the starting point for new <br /> cells exhibit defects in centriole duplication (Salisbury et al.,   strain development. Strains with constructs encoding GFP-Centrin fusion <br /> 2002). The same is true when levels of the Chlamydomonas            proteins were created as described below. UCB8 and UCB9 are CEN1 knock- <br /> centrin (Vfl2) are decreased to 5% of the normal level. In          out heterokaryons (described below). B*VI (gift from Dr. Aaron Turkewitz, <br />                                                                     University of Chicago, Chicago, IL) was used for star crosses in strain con- <br /> these cells, the number of basal bodies is reduced to &#226;&#172;&#402;0.3 per      struction (Hai et al., 2000). All were grown in SSP (2% proteose peptone, 0.1% <br /> cell from the normal condition of two per cell, indicating a        yeast extract, 0.2% glucose, 0.003% FeEDTA) at 30&#194;&#176;C. Starvation media con- <br /> failure in basal body duplication (Koblenz et al., 2003). De-       sisted of 10 mM Tris, pH 7.5 (Orias et al., 2000). For matings, an equal number <br /> fects also were observed in the structure of remaining basal        of cells of different mating types, previously starved for 18 h, were mixed <br />                                                                     (Gaertig and Kapler, 2000). <br /> bodies, indicating a possible role for centrin in basal body <br /> maturation or stability.                                            Gene Identification and Sequence Analysis <br />    In addition to its essential role in MTOC duplication,           To search for centrin genes in Tetrahymena, PCR using degenerate primers <br /> centrin is involved in other cellular processes. In yeast,          EIK-F and VAK-R, designed to represent regions of high conservation be- <br /> Cdc31 is required for Kic1 kinase activity that is important        tween centrins, was conducted (Supplemental Table 1 lists all primers used in <br /> for proper cell wall development (Sullivan et al., 1998).           this study). Standard molecular biology techniques were used to isolate a <br />                                                                     centrin gene, which was subsequently reported by Guerra et al. (2003). To <br /> Cdc31 also is involved in mRNA export at the nuclear pore           search for additional centrin genes, preliminary sequence data were obtained <br /> complex (Fischer et al., 2004). The contractile properties of       from The Institute for Genomic Research Web site at http://www.tigr.org. <br /> centrin are important in a number of structures, including          Tetrahymena CEN1 translated sequence was used to query the Tetrahymena <br />                                                                     database using the tblastn program. CEN2 was found on sequence #8254664 <br /> the nucleus-basal body connector and stellate fibers associ-        between positions 498633 and 499370, with possible introns spanning 498790 &#226;&#8364;&#8220; <br /> ated with Chlamydomonas basal bodies (Sanders and Salis-            498909, 499005&#226;&#8364;&#8220; 499063, and 499256 &#226;&#8364;&#8220; 499320. CEN3 was found on sequence <br /> bury, 1989; Taillon et al., 1992; Sanders and Salisbury, 1994),     #8253915 between positions 239917 and 240426. CEN4 was found on sequence <br /> and the infraciliary lattice, a contractile cortical network in     #8254437 between positions 298144 and 298659. Sequences were compared <br />                                                                     and analyzed at the Pasteur Bioweb site (bioweb.pasteur.fr). ClustalW <br /> Paramecium (Madeddu et al., 1996; Klotz et al., 1997). Centrin      (Thompson et al., 1994) was used to align the sequences, and BOXSHADE <br /> also has been associated with the inner dynein arm impor-           (written by Hofmann and Baron) was used to create the image of the align- <br /> tant for axonemal beating (LeDizet and Piperno, 1995;               ment. Using PHYLIP (Phylogeny Inference Package), protein distances were <br /> Guerra et al., 2003).                                               calculated with PROTDIST (Jones&#226;&#8364;&#8220;Taylor&#226;&#8364;&#8220;Thornton matrix), and a tree was <br />                                                                     constructed using NEIGHBORS followed by DRAWGRAM for image gener- <br />    In some organisms, such as yeast and Chlamydomonas, the          ation. For statistical analysis, 1000 replicates were performed and bootstrap <br /> product of a single centrin gene carries out multiple func-         values were calculated. (Felsenstein, J. 2004. PHYLIP version 3.6. Distributed <br /> tions. However, other species have multiple centrin genes           by the author, Department of Genome Sciences, University of Washington, <br /> that may encode products that are specialized to carry out          Seattle, WA) <br /> different tasks or to carry out similar tasks at different stages   Plasmid Construction <br /> of the life cycle. In humans, centrin1 expression is confined <br />                                                                     As a starting point for the creation of a CEN1 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> plasmid, pHM74 was <br /> to the testes and retinal cells, whereas centrins 2 and 3 are       constructed. A 2.1-kb fragment containing CEN1 was amplified from genomic <br /> more ubiquitously expressed (Hart et al., 1999; Laoukili et al.,    Tetrahymena DNA by PCR (Saiki et al., 1988) using primers RM13 and RM14, <br /> 2000). Strikingly, when a Leishmania centrin gene was de-           which were engineered to include unique BamHI and SacII sites at their <br />                                                                     respective 5&#226;&#172;&#732; ends. This fragment, which begins 1 kb upstream of the CEN1 <br /> leted, one of the two lifestyle forms of this parasite was          start codon, was cloned into the BamHI and SacII sites of p4T2-1 (Gaertig et al., <br /> unable to duplicate its basal bodies and was unable to              1994a) downstream of the NEO2 cassette. Primers RM11 and RM12, which <br /> progress through mitosis, whereas the other was unaffected,         were engineered to contain unique KpnI and XhoI sites at their respective <br /> suggesting that a different centrin was active during this          ends, were then used to amplify a 1.6-kb genomic fragment upstream of <br />                                                                     CEN1. This fragment was cloned into KpnI/XhoI-digested p4T2-1, upstream <br /> stage of its life cycle (Selvapandiyan et al., 2004).               of the NEO2 cassette, to create pHM74. pHM74 was digested with KpnI and <br />    Because the ciliate Tetrahymena thermophila has &#226;&#172;&#402;750 basal       BamHI, eliminating the NEO2 cassette and the 1.6-kb CEN1 upstream region. <br /> bodies per cell (Frankel, 2000; Frankel and Nelsen, 2001) and       A KpnI-BamHI adapter from EZCloneSystems (New Orleans, LA) was used to <br /> sophisticated genetic and molecular biological techniques           recircularize the plasmid. This construct was digested with EcoRI to remove <br />                                                                     most of the CENI coding sequence, including the start codon, and blunted. A <br /> are available for its study (Bruns and Cassidy-Hanley,              SmaI, EcoRV fragment from pHM74 containing the NEO2 cassette was in- <br /> 2000a,b; Gaertig and Kapler, 2000; Hai et al., 2000), we            serted into this plasmid, creating pCentrin:NEO 1. To obtain more sequence <br /> turned to Tetrahymena to gain insights into basal body/             downstream of the CEN1 gene, a library of 3- to 4-kb PstI, EcoRI-digested <br /> centriole duplication. Centrin has been localized to basal          genomic DNA was created in pBluescript KSII&#226;&#171;&#186; (Stratagene, La Jolla, CA) and <br />                                                                     transformed into Escherichia coli for colony lifts. A PCR fragment generated by <br /> bodies and cilia in Tetrahymena, and a centrin gene has been        amplification of pHM74 DNA with primers RM10 and T7 was used as a <br /> cloned (Guerra et al., 2003). We independently cloned this          probe. Lifts and labeling of the probe were conducted as per instructions for <br /> centrin gene, called CEN1, and have begun characterizing it         Hybond-N&#226;&#171;&#185; nylon membrane and the AlkPhos direct labeling system (Am- <br />                                                                     ersham Biosciences, Piscataway, NJ). A clone was recovered with &#226;&#172;&#402;3 kb of <br /> and three other centrins we identified through sequence             sequence downstream of CEN1, and a BglII, SacI fragment was cloned from <br /> similarity in the recently sequenced Tetrahymena genome. By         this into pCentrin:NEO 1 to create pCentrin:NEO1&#226;&#171;&#185;48. To use all of the <br /> using fusions with the green fluorescent protein (GFP)              upstream sequence available, a PCR fragment from genomic DNA spanning <br /> (Chalfie et al., 1994), we localized Cen1 and the second of the     the site where the NEO2 cassette had been inserted in the sequence upstream <br />                                                                     of CEN1 was amplified with primers NEORemove1 and NEORemove2, di- <br /> centrins (CEN2) to basal bodies. We created a null <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span>         gested with BclI and SpeI, and inserted into similarly digested pHM74, cre- <br /> in CEN1 and have shown it is required for basal body                ating pHM74-NEORemove. Finally, a BclI, KpnI fragment from this plasmid <br /> duplication and stability. Furthermore, we found that the           was ligated into similarly digested pCentrin:NEO1&#226;&#171;&#185;48 to create pMicCen- <br /> third centrin (CEN3) localizes to specialized structures in the     trinKO1. <br />                                                                        For the plasmids encoding the GFP-centrin fusions, genomic DNA was <br /> oral apparatus, a complex network of cilia, and accessory           amplified with XhoI and ApaI sites engineered into the primers to allow <br /> structures that sweeps food into the cell. The fourth centrin       cloning into similar sites in pVGF-1 (Wiley et al., 2000). Primers were used as <br /> (CEN4) also localizes to these structures in the oral appara-       follows: for CEN1, primers XhoI-5&#226;&#172;&#732;Centrin and ApaI-3&#226;&#172;&#732;Centrin; for CEN2, <br />                                                                     primers 5&#226;&#172;&#732;XhoI TtCentrin2 and 3&#226;&#172;&#732;ApaI TtCentrin2; for CEN3, primers 5&#226;&#172;&#732;XhoI <br /> tus as well as to contractile vacuole pores (the openings for       TtCentrin3 and 3&#226;&#172;&#732;ApaI TtCentrin3; and for CEN4, primers 5&#226;&#172;&#732;XhoI TtCentrin4 <br /> the contractile vacuoles that maintain the proper osmolarity        and 3&#226;&#172;&#732;ApaI TtCentrin4. Clones were sequenced at Macrogen (Seoul, Korea) to <br /> of the cells) and to basal bodies (Frankel, 2000).                  confirm that errors had not been introduced by PCR. <br />  <br />  <br /> Vol. 16, August 2005                                                                                                                           3607 <br />  A. J. Stemm-Wolf et al. <br />  <br />    To create a version of CEN1 that could be expressed in E. coli, a PCR          digested grl3/4:BLAST-prCEN1&#226;&#8364;&#8220;3&#226;&#172;&#732;UTR, followed by selection with Blastici- <br /> fragment containing the coding region of CEN1 was amplified from a Tetra-         din S (InvivoGen, San Diego, CA). <br /> hymena cDNA library (generous gift from Dr. Aaron Turkewitz) using prim- <br /> ers 5&#226;&#172;&#732;BamHI-Centrin and 3&#226;&#172;&#732;PstI-Centrin, which were engineered to contain          Micronuclear Knockout and Heterokaryon Generation <br /> the indicated restriction sites. This fragment was cloned into similarly di- <br /> gested pUC18. Because Tetrahymena uses an alternative genetic <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span>, in which      pMicCentrinKO1 was digested with SacI and introduced into the micronu- <br /> UAA and UAG represent glutamine codons, it was necessary to modify the            cleus of B2086- and CU428-conjugating cells by biolistic bombardment as <br /> CEN1 sequence to allow expression in E. coli. Four glutamine codons were          described previously (Bruns and Cassidy-Hanley, 2000a). To check for proper <br /> changed to the canonical CAA or CAG codons using the U.S.E. site-directed         integration of the CEN1 knockout construct, total Tetrahymena DNA was <br /> mutagenesis system (Amersham Biosciences, Piscataway, NJ) with the fol-           isolated (Gaertig et al., 1994b) and digested with NsiI and PstI. Standard <br /> lowing primers: CentSw1, CentSw2, CentSw3, and TMSelect. The resulting            Southern blot analysis was performed using a probe representing sequence <br /> plasmid (pUC18-TtCentrinMFE) was digested with BamHI and PstI, and the            upstream of the transforming fragment. The probe was generated by PCR <br /> CEN1 fragment was ligated into the pQE10 6-His fusion expression vector           amplification with primers CentrinUS1 and CentrinUS2 and labeled by the <br /> (QIAGEN, Valencia, CA), creating pQE10-CentrinMFE.                                AlkPhos Direct (Amersham Biosciences) labeling system. Crosses to confirm <br />    To create the CEN1 rescuing construct, a 1.0-kb BamHI, MfeI CEN1 pro-          that the transformation occurred in the micronucleus were performed, and <br /> moter region was excised from pHM74 and cloned into similarly digested            the knockout heterokaryon was constructed, as described previously (Hai et <br /> pUC18-TtCentrinMFE. CEN1, with its promoter, was then excised from this           al., 2000) with the modification that an outcross was performed on the original <br /> construct with SmaI and SphI, blunted, and ligated into the blunted BsmBI site    transformant strains to rid them of an apparent micronuclear aneuploidy <br /> of grl3/4:BLAST (a construct that replaces the GRL3 and GRL4 promoter             before mating with B*VI. <br /> region with the Bsr gene, which provides resistance to Blasticidin S, a gener- <br /> ous gift from Aaron Turkewitz). Finally, the CEN1 3&#226;&#172;&#732; untranslated region          Analysis of cen1&#226;&#338;&#172; Cells <br /> (UTR) was amplified by PCR of genomic DNA using primers Centrin_BstXI             Cultures containing equal numbers of UCB8 and UCB9 cells were starved and <br /> and BsmBI-dsCentrin and ligated into BstXI (partial), BsmBI-digested grl3/        mixed. Ten hours after mixing, an equal volume of 2&#226;&#171;&#187; SPP (1% proteose <br /> 4:BLAST-PrCEN1 to create grl3/4:BLAST-PrCEN1&#226;&#8364;&#8220;3&#226;&#172;&#732;UTR. The resulting                 peptone, 0.1% yeast extract, 0.2% glucose, 0.003% FeEDTA final) was added. <br /> plasmid was sequenced to ensure no errors had been introduced.                    Seventeen hours after mixing, paromomycin was added to 120 &#226;&#65533;&#174;g/ml to <br />                                                                                   select against cells that had not gone through conjugation. Samples were <br /> Protein Expression and Antibody Generation                                        removed for Western blotting at 6-, 25-, 50-, 75-, and 96-h time points. Samples <br /> pQE10-CentrinMFE was transformed into the E. coli strain M15, and a culture       were removed from separate mating cultures and prepared for immunoflu- <br /> was grown in 500 ml of Luria Broth &#226;&#171;&#185; 0.2% glucose with 50 &#226;&#65533;&#174;g/ml kanamycin         orescence microscopy at 17-, 24-, 48-, and 72-h time points. Wild-type (WT) <br /> and 100 &#226;&#65533;&#174;g/ml ampicillin. At an OD600 of 0.4, cells were cooled to room           cells were treated with paromomycin as a control. To determine whether cells <br /> temperature for 1 h. Isopropyl &#226;&#65533;&#164;-d-thiogalactoside was added to 0.3 mM and        could continue to grow without Cen1, individual conjugating pairs were <br /> induction was allowed to proceed for 2.5 h. Cells were pelleted, washed once      isolated into 30-&#226;&#65533;&#174;l drops of SPPA media, and cells were counted at 50 and <br /> with phosphate-buffered saline (PBS), and stored frozen. Pellets were thawed,     75 h. <br /> and resuspended in 10 ml of PBS containing 1 mM phenylmethylsulfonyl <br /> fluoride, 0.5 &#226;&#65533;&#174;g/ml leupeptin, 1 &#226;&#65533;&#174;g/ml aprotinin, and 1 &#226;&#65533;&#174;g/ml pepstatin A.        Fluorescence Microscopy <br /> Lysozyme was added to 200 &#226;&#65533;&#174;g/ml before a 10-min incubation on ice. Cells <br />                                                                                   Cells containing the GFP-Centrin fusions were placed on slides and allowed <br /> were sonicated five times for 15 s each with 1-min intervals of rest on ice and <br />                                                                                   to dry just until cells no longer swam. GFP-Cen1 cells were from logarithmic <br /> centrifuged 15 min at 10,000 &#226;&#171;&#187; g at 4&#194;&#176;C. The supernatant was loaded onto a <br />                                                                                   cultures, GFP-Cen2 cells were from dense cultures, and GFP-Cen3 and GFP- <br /> column containing Talon resin (BD Biosciences Clontech, Palo Alto, CA) <br />                                                                                   Cen4 were from starved cultures, as dense and starved cultures had less <br /> equilibrated in PBS, and fractions were eluted in PBS &#226;&#171;&#185; 200 mM imidozole. <br />                                                                                   nonspecific background signal. Immunofluorescence microscopy was per- <br /> Fractions with Cen1 were pooled, and 1 mg was sent to Animal Pharm <br />                                                                                   formed as described previously (Stuart and Cole, 2000). Cells were fixed with <br /> (Healdsburg, CA), for antibody generation in rabbits. <br />                                                                                   both formaldehyde and EtOH as described previously. Fixed cells were <br />                                                                                   visualized on Antibody slides (Bellco Glass, Vineland, NJ). Antibodies were <br /> Northern Blot Analysis                                                            diluted in PBS &#226;&#171;&#185; 1% bovine serum albumin. Tetrahymena Cen1 antibody was <br /> RNA was isolated using the TRI Reagent (Molecular Research Center, Cin-           diluted 1:1000, the 20H5 monoclonal centrin antibody raised against Chlamy- <br /> cinnati, OH) as per the manufacturer&#226;&#8364;&#8482;s instructions. Samples (each containing     domonas centrin (a generous gift from Dr. J. Salisbury, Mayo Clinic, Rochester, <br /> 10 &#226;&#65533;&#174;g of RNA) were run on formaldehyde gels and transferred to Hybond N&#226;&#171;&#185;          MN) was diluted 1:2500, the antibody against polyglutamylation (a generous <br /> (Amersham Biosciences) in 20&#226;&#171;&#187; SSC (Ausubel et al., 1997). Northern blots          gift from Dr. M. Gorovsky) (Shang et al., 2002) was diluted 1:500, and the <br /> were probed with randomly primed [&#226;&#65533;&#163;-32P]dATP-labeled fragments (Fein-             12G10 mAb against &#226;&#65533;&#163;-tubulin (a generous gift of Dr. J. Frankel, University of <br /> berg and Vogelstein, 1983), which were obtained by amplification by PCR of        Rochester, Rochester, NY) was diluted 1:50. Secondary antibodies were used <br /> genomic DNA with the primers used to create the GFP fusions described             at a 1:1000 dilution and included goat anti-rabbit Alexa594, goat anti-rabbit <br /> above. Probes were specific for the target sequences as determined by South-      Alexa488, and donkey anti-mouse Alexa488 (Molecular Probes). Antibody <br /> ern blot analysis of total DNA (our unpublished data).                            incubations were carried out at room temperature for 1 h, or alternatively, <br />                                                                                   overnight at 4&#194;&#176;C. To visualize DNA, 4,6-diamidino-2-phenylindole (DAPI) <br />                                                                                   was applied to the cells at 1 &#226;&#65533;&#174;g/ml for 3 min. After each antibody incubation, <br /> Western Blot Analysis                                                             cells were washed six times with PBS &#226;&#171;&#185; 0.1% bovine serum albumin. Stan- <br /> Cells were lysed in buffer as described previously (Williams, 2000), and whole    dard fluorescence microscopy was carried out using a Leica DMRXA/RF4/V <br /> cell extracts were run on 12.5% or 11.5% standard Laemmli acrylamide gels         automated microscope with a Cooke Sensi-Cam digital camera. Images were <br /> (Ausubel et al., 1997) along with BenchMark prestained molecular weight           collected and subjected to no neighbors or nearest neighbors deconvolution <br /> standards (Invitrogen, Carlsbad, CA). Proteins were electrophoretically trans-    algorithms using the Slidebook software package (Intelligent Imaging Inno- <br /> ferred to Immobilon-P membrane and blotted according to the manufactur-           vations, Denver, CO). <br /> er&#226;&#8364;&#8482;s instructions (Millipore, Billerica, MA). Membranes were incubated with <br /> antibodies diluted in Tris-buffered saline containing 0.05% Tween and 1% <br /> bovine serum albumin for 1 h at room temperature. Primary antibodies <br />                                                                                   Immunoelectron Microscopy <br /> (either serum from bleed 4 of a Cen1-injected rabbit, an anti-GFP monoclonal      Tetrahymena cells were pelleted, frozen under high pressure in a BAL-TEC <br /> antibody [mAb] [B34 from Abcam, Cambridge, MA]), or an anti-&#226;&#65533;&#163;-tubulin             HPM-010 high-pressure freezer, freeze substituted in 0.25% gluteraldehyde/ <br /> mAb (12G10; a kind gift of Joseph Frankel, University of Iowa, Iowa City, IA)     0.1% uranyl acetate, and embedded in Lowicryl HM20. Serial sections (55 nm) <br /> were diluted 1:2000, 1:5000, and 1:1000, respectively. Secondary antibodies       were taken and labeled with 1:200 dilutions of primary antibody in 1% nonfat <br /> (anti-rabbit IR800 [Li-Cor Biosciences, Lincoln, NE], anti-mouse Alexa680         milk phosphate-buffered saline/Tween 20, which included an affinity-puri- <br /> (Molecular Probes, Eugene, OR), anti Mouse horseradish peroxidase [HRP],          fied rabbit anti-GFP primary antibody (a gift from Jason Kahana, Ludwig <br /> and anti-rabbit HRP [Pierce Chemical, Rockford, IL]) were diluted 1:10,000.       Institute for Cancer Research, New York, NY, and Pamela Silver, Dana-Farber <br /> Blots were visualized either on a Li-Cor Odyssey scanner or by SuperSignal        Cancer Institute, Boston, MA) and the Cen1 antibody described above. A 1:20 <br /> West Pico chemiluminescent substrate (Pierce Chemical) and exposure to            dilution of 15-nm gold-conjugated goat anti-rabbit secondary antibody (Ted <br /> film.                                                                             Pella, Redding, CA) was used. Grids were stained with uranyl acetate and <br />                                                                                   lead citrate and were visualized in a Philips CM10 electron microscope. <br /> Macronuclear Transformations <br /> Plasmids encoding GFP-centrin fusions were transformed into the developing        RESULTS <br /> macronucleus during late stages of conjugation of strains B2086 and CU428 by <br /> electroporation (Gaertig and Kapler, 2000). Paromomycin-resistant cells were      Identification of a Tetrahymena Centrin Gene Family <br /> selected and viewed on the fluorescence microscope. The cen1&#226;&#338;&#172; rescuing <br /> construct was transformed into the developing macronucleus of the strains         We wished to isolate the T. thermophila centrin gene for <br /> UCB8 and UCB9 during conjugation by electroporation, using KpnI, XhoI-            analysis with respect to its potential role at basal bodies. A <br />  <br /> 3608                                                                                                                          Molecular Biology of the Cell <br />                                                                                                                       Tetrahymena Centrins <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison of centrins from different organisms. Tt, T. thermophila; Hs, Homo sapiens; Mm, Mus musculus; Cr, Chlamydomonas <br /> reinhardtii; Pt, P. tetraurelia; Cdc31 from S. cerevisiae; Cmd, calmodulin. (A) Twelve representative centrins were aligned along with two <br /> calmodulin proteins. Residues where 50% or more are identical between the proteins are shaded black, and residues where 50% or more are <br /> similar are shaded gray. (B) The centrin and calmodulin sequences were subjected to phylogenetic analysis using the Neighbor-Joining <br /> method; 1000 replicates were performed, and a representative unrooted tree is displayed as a dendogram. Bootstrap values are displayed as <br /> percentages at each node and indicate how often the separation delineated by the branching occurred. Bar &#226;&#171;&#189; 0.1 substitutions per one amino <br /> acid (see Materials and Methods). <br />  <br />  <br /> fragment was amplified by PCR from Tetrahymena genomic                 Koblenz et al., 2003). Cen2 is most closely related to budding <br /> DNA using degenerate primers designed based on the align-              yeast Cdc31, which is essential for spindle pole body dupli- <br /> ment of several published centrins (see Materials and Meth-            cation, and to human Centrin3. Cen3 and Cen4 are similar to <br /> ods). Flanking sequences were cloned by inverse PCR, and               centrins found in the infraciliary lattice of the ciliate P. <br /> DNA sequencing revealed the fragment to be a centrin gene.             tetraurelia (Klotz et al., 1997), and this more distantly related <br /> During the course of this work, this centrin gene was inde-            branch of the centrin family may be unique to the ciliates. <br /> pendently reported by Guerra et al. (2003). We refer to this              We checked for expression of these sequences to confirm <br /> gene as CEN1.                                                          that they encoded genes. Initially, PCR was carried out on a <br />    The corresponding protein sequence of CEN1 was used as              cDNA library generated from vegetatively growing cells, <br /> a query to search the recently sequenced Tetrahymena ge-               and an uncloned cDNA preparation also from vegetatively <br /> nome at The Institute for Genomic Research (TIGR, www.                 growing cells. In both cases, amplification of sequence re- <br /> tigr.org), and three additional potential centrin genes were           sulting from primers specific to CEN1, CEN3, and CEN4 was <br /> identified. Two of these genes, called CEN3 and CEN4                   observed, but not for primers specific to CEN2, which are <br /> herein, contain 510 and 515 base pairs, respectively, of un-           capable of amplification of CEN2 from the genome (Figure <br /> interrupted protein coding sequence. The remaining gene,               2A; our unpublished data). Five clones from each of these <br /> CEN2, contains 492 base pairs of coding sequence with three            PCR products were sequenced, confirming that the primers <br /> introns predicted by the protein alignment and standard                used were specific for the targeted sequences. To determine <br /> splicing rules (Alberts et al., 2002). All three centrins had          whether CEN2 was expressed during other stages of the <br /> predicted molecular masses of 19.5 kDa, which is similar to            Tetrahymena life cycle and to examine expression patterns for <br /> Cen1&#226;&#8364;&#8482;s molecular mass of 19.4 kDa. The translated se-                  the three other centrins, RNA was isolated from vegetatively <br /> quences were aligned with a variety of characterized cen-              growing cells, cells in starvation media, and cells at various <br /> trins as well as the Tetrahymena and human calmodulins and             time points through the mating cycle (conjugation) for <br /> placed in a phylogenetic tree (see Materials and Methods)              Northern analysis with probes derived from each of the four <br /> (Figure 1). The phylogenetic analysis revealed three distinct          centrin genes (Figure 2B). CEN1, CEN3, and CEN4 have <br /> branches: one for the calmodulins, one for two centrins                similar expression profiles: bands of the expected size are <br /> identified in the ciliate Paramecium tetraurelia and CEN3 and          observed during vegetative growth, and the intensity of <br /> CEN4 from Tetrahymena, and the third for all the remaining             these bands decreases in cells in starvation media, which <br /> centrins included in the analysis. Of the Tetrahymena cen-             were no longer dividing. During conjugation, which oc- <br /> trins, Cen1 is most similar to the Chlamydomonas centrin and           curred in a starvation medium, expression increases to levels <br /> human Centrin2, which are involved in duplication of basal             approximately twice those seen during logarithmic growth, <br /> bodies and centrioles, respectively (Salisbury et al., 2002;           peaking 9 h after initiation of mating. During conjugation, <br />  <br /> Vol. 16, August 2005                                                                                                                 3609 <br />  A. J. Stemm-Wolf et al. <br />  <br />  <br />  <br />  <br />                                                                            Figure 3. Schematic drawings of centrin containing structures. (A) <br />                                                                            A basal body from a cortical row is represented along with some of <br />                                                                            the structures associated with it. bb, basal body; kf, kinetodesmal <br />                                                                            fiber; tm, transverse microtubules; pc, postciliary microtubules. (B) <br />                                                                            Some of the structures within the oral apparatus are represented. <br />                                                                            Um, undulating membrane; m, membranelles, of which there are <br />                                                                            three; or, oral-rib (gray in diagram); df, deep-fiber; ffr, fine-filamen- <br />                                                                            tous reticulum (striped in diagram). The undulating membrane and <br />                                                                            membranelles contain basal bodies and cilia. The oral-rib is a sheet <br />                                                                            of microtubules along the inside of the oral apparatus cavity be- <br />                                                                            neath the undulating membrane, edged with ribbed walls (not <br />                                                                            labeled). The deep-fiber consists of a subset of microtubules that <br />                                                                            continue inward from the oral-rib into the cell. The fine-filamentous <br />                                                                            reticulum lies beneath the oral ribs. <br />  <br />  <br />  <br />                                                                            centrioles in cell culture, and a C-terminal centrin-GFP fu- <br /> Figure 2. Expression of genes in the Tetrahymena centrin family.           sion protein localized to Chlamydamonas basal bodies (White <br /> (A) PCR products using primers specific for CEN1, CEN2, CEN3, <br />                                                                            et al., 2000; Ruiz-Binder et al., 2002), although these fusions <br /> and CEN4 on cDNA from cells during vegetative growth displayed <br /> on an agarose gel. Sizes of select marker (M) bands are displayed in       are probably not fully functional (Ruiz-Binder et al., 2002; <br /> kilobase pairs. (B) Northern blot analysis of RNA from vegetatively        Kilmartin, 2003). Although Cen2 mRNA is not normally <br /> growing cells (V), starved cells (S), and conjugating cells at various     detectable during vegetative growth, we tagged this protein <br /> time points after initiation of mating (2, 4, 6, 9, 18 h) (see Materials   along with the others to gain some understanding of where <br /> and Methods). Similar amounts of total RNA were loaded as assessed         it might function if expressed. Transformants were recov- <br /> by absorbance spectroscopy and ethidium bromide staining our               ered for each construct, and live cells were imaged by fluo- <br /> unpublished data).                                                         rescence microscopy. GFP-Cen1 and GFP-Cen2 both local- <br />                                                                            ized to the oral apparatus, a complex cytoskeletal structure <br />                                                                            rich in basal bodies, microtubule structures, and other non- <br /> cells lose the oral apparatus, and its reformation is an early             microtubule-based filaments (Sattler and Staehelin, 1976; <br /> event after conjugants separate (Kiersnowska et al., 1993);                Frankel, 2000) and to cortical rows in a <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> similar to that <br /> the increase in the centrins&#226;&#8364;&#8482;s expression during later stages              observed for basal bodies. Interestingly, transformation of <br /> of conjugation could play a role in this process. Among the                both constructs resulted in an apparent disorganization of <br /> centrin genes, CEN1 seems to have the highest expression                   the cortical rows, such that the rows sometimes ended pre- <br /> levels, followed by CEN4 and then CEN3, although it is                     maturely, or curled around in unusual patterns, suggestive <br /> possible that differences in band intensity may be the result              of a dominant-negative <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> (Figure 4A, a and b). <br /> of the ability of each probe to recognize the appropriate                  Consistent with these constructs being deleterious alleles, <br /> message. Because mRNA was detected for CEN1, CEN3, and                     the transformants grew slowly, and neither construct was <br /> CEN4, these sequences represent three bona fide members of                 stably maintained over the course of extended vegetative <br /> a Centrin family in Tetrahymena. CEN2 expression was not                   growth. Nonetheless, Cen1 and Cen2 seem to localize to <br /> detected for this limited set of conditions, although this                 basal bodies. <br /> result does not preclude expression during other untested                     In contrast, GFP-Cen3 and GFP-Cen4 transformants did <br /> growth conditions.                                                         not have any obvious growth defects. The fluorescence sig- <br />                                                                            nal for Cen3 and Cen4 fusion proteins also was strikingly <br /> GFP Tagging and Localization of Tetrahymena Centrin                        different from that of the other centrin fusions. GFP-Cen3 <br /> Family Members                                                             localized primarily to the oral apparatus. However, this <br /> To understand more about the functions of the centrin                      fluorescence signal was distinct from the GFP-Cen1 and <br /> genes, we attempted to localize their products within the                  GFP-Cen2 signal at the oral apparatus, which was punctate <br /> cell. Each gene was fused at its N terminus to the GFP                     and in all likelihood originated from GFP signal at the oral <br /> (Chalfie et al., 1994) in the vector pVGF1 (Wiley et al., 2000).           apparatus basal bodies. Rather, the GFP-Cen3 signal was <br /> This vector is based on a ribosomal DNA processing vector                  found in a crescent-shaped region to the cell&#226;&#8364;&#8482;s left of the <br /> in which rDNA sequences on the plasmid, into which the                     undulating membrane and extended along the deep rootlet. <br /> GFP fusion is inserted, are processed in the developing                    Signal also was present at the three membranelles of the oral <br /> macronucleus, resulting in amplification and strong expres-                apparatus, and in a crescent-shaped band at the apical tip of <br /> sion (Karrer, 2000). An N-terminal GFP-centrin fusion pro-                 the cell (Figures 3B and 4A, c). GFP-Cen4 also localized to <br /> tein has been reported previously to localize correctly to                 the structures where GFP-Cen3 was observed as well as to <br />  <br /> 3610                                                                                                               Molecular Biology of the Cell <br />                                                                                                                            Tetrahymena Centrins <br />  <br />  <br />  <br />  <br /> Figure 4. Localization of GFP-centrin fusion proteins. (A) Fluorescence signal of live cells: a, GFP-Cen1; b, GFP-Cen2; c, GFP-Cen3; and d, <br /> GFP-Cen4. The oral apparatus is on the right side in each image. In a and to a greater extent b, cortical rows are affected by the overexpression <br /> of the GFP fusion protein. The strong signal in c is at the oral apparatus primarily in the oral crescent, internal to the undulating membrane, <br /> and in the apical band. These same regions are visible in d. The contractile vacuole pores are the two small circles near the posterior of the <br /> cell. Bar, 10 &#226;&#65533;&#174;m. (B) Immunoelectron microscopy with a GFP antibody and a 15-nm gold-conjugated secondary antibody. Arrowheads <br /> indicate representative concentrations of gold particles. a and b, GFP-Cen1 localization; c and d, GFP-Cen2 localization. kf, kinetodesmal <br /> fiber; pc, postciliary microtubules; bb, basal body; c, cilia. Representative cross-sectional (a and c) and longitudinal views (b and d) of basal <br /> bodies are shown. e and f, localization to the fine-filamentous reticulum of GFP-Cen3, with f also showing some localization along the oral <br /> ribs. Ffr, fine-filamentous reticulum; rw, ribbed wall; or, oral-ribs. GFP-Cen4 localization is shown in g, h, and i. g, localization in the fine <br /> filamentous reticulum of the oral apparatus. h, sections of both contractile vacuole pores as well as some associated microtubules. cvp, <br /> contractile vacuole pore; mt, microtubules; cv, contractile vacuole. h, two basal bodies in cross section with signal apparent along the <br /> kinetodesmal fibers. <br />  <br />  <br />  <br /> Vol. 16, August 2005                                                                                                                        3611 <br />  A. J. Stemm-Wolf et al. <br />  <br />  <br /> Figure 5. Reactivity of the Cen1 antibody. <br /> (A) Western immunoblot analysis using the <br /> Cen1 antibody as a probe on recombinant <br /> 6-histidine&#226;&#8364;&#8220;tagged Centrin (lane 1) and Tet- <br /> rahymena whole cell extract (wce) from cells <br /> in logarithmic growth (lane 2). The reactiv- <br /> ity of the preimmune serum on wce also is <br /> shown (lane 3). (B) Reactivity of the Cen1 <br /> antibody with the overexpressed GFP-cen- <br /> trin fusion proteins. Whole cell extracts <br /> from strains expressing different fusion pro- <br /> teins were probed with either the Cen1 an- <br /> tibody (left) or a GFP antibody (right). Be- <br /> cause the fusion proteins are overexpressed, <br /> the signal overwhelms the endogenous Cen- <br /> trin signal, which is visible with longer ex- <br /> posures. Select marker bands are noted and <br /> sizes are in kilodaltons. <br />  <br />  <br /> the contractile vacuole pores (CVPs) through which the            strongly during logarithmic growth, and a GFP fusion pro- <br /> contractile vacuole expels fluid at the posterior of the cell     tein localizes to basal bodies at the precise location where <br /> (Frankel, 2000). Additionally, faint signal was observed          initiation of basal body duplication is thought to occur <br /> along the cortical rows, reminiscent of basal body localiza-      (Allen, 1969). <br /> tion (Figure 4A, d). <br />    We examined cells by immunoelectron microscopy to bet-         Basal Body Localization of Endogenous Cen1 <br /> ter resolve the localization of the four centrin fusions. Cells   A polyclonal antibody was raised against full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> 6-his- <br /> were prepared for electromicroscopy by high-pressure              tidine&#226;&#8364;&#8220;tagged Cen1 (see Materials and Methods) and initially <br /> freezing and freeze substitution, and sections were stained       characterized by Western blot analysis. The antibody de- <br /> using an antibody that recognizes GFP followed by a sec-          tected recombinant 6-histidine&#226;&#8364;&#8220;tagged Cen1 (Figure 5A, lane <br /> ondary antibody conjugated to 15-nm gold particles (see           1), and a band the size expected for centrin (19.5 kDa) in a <br /> Materials and Methods). GFP-Cen1 is abundant at the base of       whole cell Tetrahymena extract (lane 2). We then took advan- <br /> the basal body adjacent to the kinetodesmal fiber, a nonmi-       tage of the GFP-centrin fusion strains, to examine the anti- <br /> crotubule-based striated rootlet that extends from the basal      body&#226;&#8364;&#8482;s reactivity with each individual Tetrahymena centrin <br /> body toward the anterior of the cell, at the location where       family member. GFP adds approximately 27 kDa to the <br /> new basal body formation occurs (Figures 3A and 4B, a and         molecular mass of the protein, thus allowing for easy sepa- <br /> b) (Allen, 1969). Additionally, signal was observed along the     ration of the tagged allele from the endogenous proteins, <br /> length of the basal body, with a second concentration at the      which all have a molecular mass of 19.5 kDa. To control for <br /> transition zone between the basal body and the ciliary ax-        the individual expression level of each protein, antibody <br /> oneme. Transition zone localization for centrin has been          recognizing the GFP tag was used as a control. As expected, <br /> reported in Chlamydomonas (Sanders and Salisbury, 1994).          the Cen1 antibody strongly recognized GFP-Cen1 (Figure <br /> The overall signal from GFP-Cen2 was significantly weaker         5B, left). The antibody had minimal cross-reactivity with <br /> than that for GFP-Cen1 in this experiment, but when mul-          GFP-Cen3 and slightly more cross reactivity with GFP-Cen2 <br /> tiple images were analyzed signal was detected in a pattern       and GFP-Cen4 compared with the GFP probed immunoblot. <br /> similar to that of GFP-Cen1 (Figure 4B, c and d). GFP-Cen3        Endogenous centrin was clearly visible with the Cen1 anti- <br /> and GFP-Cen4 both localized primarily to the region directly      body on longer exposures (our unpublished data). We there- <br /> underneath the ribbed wall and the oral-ribs within the oral      fore concluded that our antibody is largely specific to Cen1, <br /> apparatus that corresponds to the fine filamentous reticulum      but it may interact with the other centrins at a low-to- <br /> (Williams and Luft, 1968; Williams and Bakowska, 1982)            moderate level. <br /> (Figures 3B and 4B, e&#226;&#8364;&#8220; g), and GFP-Cen4 signal was clearly           To examine localization of endogenous Cen1, we used the <br /> detected at the contractile vacuole pores (Figure 4B, h).         Cen1 antibody for immunofluorescence and immunoelec- <br /> Weak GFP-Cen4 signal also was observed at the basal bod-          tron microscopy. Immunofluorescence was performed on <br /> ies, primarily on the kinetodesmal fiber (Figure 4B, i). When     cells from a logarithmic culture (Figure 6A) using our rabbit <br /> a strain not expressing GFP was submitted to identical an-        polyclonal antibody (anti-Cen1), and a mAb (20H5) that <br /> tibody incubations, there was almost no background signal         recognizes centrins from a variety of species and has previ- <br /> apparent, supporting the authenticity of the localization         ously been used to localize centrin in Tetrahymena (Jerka- <br /> data (our unpublished data).                                      Dziadosz et al., 1995; Guerra et al., 2003). Staining patterns <br />    Together, the localization of the GFP fusion proteins and      from each antibody were very similar but not identical. Both <br /> the expression patterns of the different centrins suggest that    recognized the basal bodies along the cortical rows and in <br /> each member of the Tetrahymena centrin family has a unique        the oral apparatus, but the 20H5 antibody recognized addi- <br /> role within the cell. The possibility remains that there is       tional structures that the Cen1 antibody did not. The signal <br /> some redundancy among the centrin genes, particularly be-         from 20H5 was observed at the oral crescent (corresponding <br /> tween CEN3 and CEN4, because their protein products have          to the fine filamentous reticulum) and also at the apical <br /> similar distributions within the oral apparatus. Because our      filamentous band (Jerka-Dziadosz, 1981; Jerka-Dziadosz et <br /> primary interest is the role of centrin in basal body duplica-    al., 1995). Both these structures were revealed by GFP-Cen3 <br /> tion during vegetative growth, we chose to focus on Cen1.         and GFP-Cen4. However, the 20H5 antibody did not recog- <br /> Of the four Tetrahymena centrins, Cen1 is most similar to         nize the contractile vacuole pores, raising the possibility that <br /> human centrin2, which is required for centriole duplication.      this antibody can recognize Cen1 and Cen3, but not Cen4, or <br /> Furthermore, we have shown that Cen1 is expressed                 that Cen4 is not accessible to the antibody at the CVPs. <br />  <br /> 3612                                                                                                 Molecular Biology of the Cell <br />                                                                                                                           Tetrahymena Centrins <br />  <br />  <br />  <br />  <br /> Figure 6. Cen1 localization at basal bodies. (A) Immunofluorescence microscopy using the Cen1 antibody and the monoclonal centrin <br /> antibody 20H5 raised against Chlamydamonas centrin. DNA is visualized with DAPI. Bar, 10 &#226;&#65533;&#174;m. (B) Serial section immunoelectron <br /> microscopy of a basal body in cross section with the Cen1 antibody. Panels proceed from the axoneme distal part of the basal body (a) and <br /> track upwards through the structure. Figure 3A serves as a reference for some of the basal body associated structures that are present. Certain <br /> basal body associated structures are identified in sections where they are most prominent, although this does not preclude these same <br /> structures from being visible in other sections. cw, cartwheel; kf, kinetodesmal fiber; pc, postciliary microtubules; tz, transition zone; tm, <br /> transverse microtubules; c, cilia; bb, basal body. Centrin concentrates on the transverse microtubules that are seen obliquely, on the <br /> kinetodesmal fiber, and at the transition zone (k). (C) Serial section immunoelectron microscopy of three basal bodies in longitudinal section. <br />  <br />  <br />  <br /> Consistent with this interpretation, the 20H5 antibody                    cells from a logarithmic culture using the Cen1 polyclonal <br /> strongly recognized Cen1 and Cen3 by Western blot analy-                  antibody. This analysis on endogenous levels of Cen1 re- <br /> sis, whereas showing moderate affinity to Cen4 and virtually              vealed a similar localization pattern to that observed with <br /> no affinity for Cen2 (our unpublished data). Like the GFP-                the GFP-Cen1 strain. Serial sections through six basal bodies <br /> Cen1 data, data from the Cen1 antibody indicates that Cen1                where examined. At the basal body, there are two major <br /> localizes specifically to basal bodies.                                   centers of Cen1 concentration. The first is at the base of the <br />    To map Cen1 localization within basal bodies, immuno-                  basal body, along and adjacent to the kinetodesmal fiber <br /> electron microscopy was performed on serial sections of                   (Figures 3A, 6B, a&#226;&#8364;&#8220; e, and 6C, a&#226;&#8364;&#8220; c). At the very base of the <br />  <br /> Vol. 16, August 2005                                                                                                                      3613 <br />  A. J. Stemm-Wolf et al. <br />  <br />  <br /> basal body (most distal to the axoneme), this signal is im- <br /> mediately adjacent to the basal body (Figure 6B, a), the site <br /> of new basal body formation (Allen, 1969). As the cross- <br /> sectional serial sections are tracked through the basal body, <br /> this signal moves away from the basal body, following a <br /> path along the transverse microtubules that extend to the <br /> adjacent cortical row (for reference, see Figures 3A and 6B, <br /> c&#226;&#8364;&#8220;j). There is also a moderate amount of signal from the <br /> Cen1 antibody within the basal body cylinder that extends <br /> the length of the basal body immediately adjacent to basal <br /> body microtubules (Figure 6B, g&#226;&#8364;&#8220;j). The second concentra- <br /> tion of Cen1 occurs at the transition zone between the basal <br /> body and the axoneme (Figure 6B, k). These localization <br /> patterns also were observed in longitudinal sections (Figure <br /> 5C, a&#226;&#8364;&#8220; c). <br />  <br /> Cen1 Is Required for Basal Body Duplication and <br /> Maintenance in Tetrahymena <br /> Both Cen1&#226;&#8364;&#8482;s localization to the base of the basal body where <br /> duplication is initiated, and the disruption of the cortical <br /> rows caused by overexpression of the GFP-Cen1 fusion pro- <br /> tein, are suggestive that Cen1 has an important role at the <br /> basal body. To assess this more directly, a null strain was <br /> created. We replaced CEN1 with the NEO2 drug resistance <br /> cassette, which provides resistance to paromomycin (Gaer- <br /> tig et al., 1994a), in the diploid micronucleus by homologous <br /> recombination (Bruns and Cassidy-Hanley, 2000a), as con- <br /> firmed by Southern analysis (Figure 7A). This strain was <br /> then used to create a knockout heterokaryon (Hai et al., <br /> 2000), a strain homozygous for the knockout in the micro- <br /> nucleus, but containing only wild-type CEN1 in the ex- <br /> pressed macronucleus. When two such strains with different <br /> mating types are mated to each other, a new macronucleus <br /> is formed based on the DNA of the germline micronuclei of <br /> the parent strains. The resulting progeny lack the CEN1 <br /> gene, having the NEO2 gene cassette in its place (Figure 7A). <br /> CEN1 knockout heterokaryon strains UCB8 and UCB9 were <br /> mated to each other, and 120 &#226;&#65533;&#174;g/ml paromomycin was               Figure 7. Cen1 protein levels decrease when CEN1 is deleted. (A) <br /> added to select for cells that had proceeded successfully        Southern blot analysis of the CEN1 locus. The fragment used for <br /> through mating and development of progeny. The cell den-         transformation extends from the SacI site to the PstI site with the <br /> sity of the culture increased only slightly by 72 h after        NEO2 cassette replacing CEN1. Total Tetrahymena DNA from wild- <br />                                                                  type cells (lane 1), the micronuclear transformant (lane 2), and <br /> initiation of mating (starting at 2.5 &#226;&#171;&#187; 105 cells/ml, and <br />                                                                  progeny from the knockout heterokaryon strains rescued by CEN1 <br /> increasing to 2.8 &#226;&#171;&#187; 105 cells/ml), and eventually all cells      transformed at a second genomic location (lane 3) was digested with <br /> died, either due to sensitivity to paromomycin (nonmating,       NsiI and PstI. The sizes of select marker bands (M) are noted in <br /> parental cells) or due to the lack of Cen1 (progeny). No         kilobase pairs. (B) Western blot analysis of progeny from the cen1&#226;&#338;&#172; <br /> &#226;&#8364;&#339;survivors&#226;&#8364;&#65533; were observed in these cultures, indicating that     knockout heterokaryon strains UCB8 and UCB9 (cen1&#226;&#338;&#172;) and of the <br /> none of the other centrins can substitute for Cen1 in its        progeny from wild-type strains B2086 and CU428 (CEN1). Cen1 is <br /> essential function. Western blot analysis confirmed that         visualized with the Cen1 polyclonal antibody, and a mAb to &#226;&#65533;&#163;-tu- <br /> Cen1 protein levels were reduced significantly by 50 h after     bulin is used as a loading control. V, vegetative growth; S, starva- <br /> mating and that the reduction in Cen1 protein levels contin-     tion; numbers correspond to hours after parental cells were mixed. <br />                                                                  Marker bands are noted, and sizes are in kilodaltons. <br /> ued over time (Figure 7B). These Western blot results further <br /> indicate that either the antibody is very specific for Cen1 <br /> and/or Cen1 is the predominantly expressed centrin in cells, <br /> because the decrease in Cen1 was not masked by the other         a stable vegetative culture. However, of the 45% that did <br /> centrins.                                                        divide after conjugation, more than half divided approxi- <br />    The ability of cen1&#226;&#338;&#172; cells to divide was tested by mating     mately three times. Therefore, at least some of the cen1&#226;&#338;&#172; <br /> UCB8 and UCB9 cen1&#226;&#338;&#172; knockout heterokaryon strains and            strains were able to undergo several cell divisions before <br /> isolating individual conjugating pairs into small drops of       arresting and eventually dying. <br /> media. As a control, B2086 and CU428 cells (both wild-type          The effect of the lack of Cen1 on basal body integrity was <br /> for CEN1) were subjected to the same procedure. Cells were       observed by fixing and staining cells at various time points <br /> counted after 50 h of growth and checked again at 75 h, and      after mating with both the 20H5 monoclonal centrin anti- <br /> an estimate as to how many divisions each cell had com-          body and an antibody raised against a polyglutamic acid <br /> pleted was made. Both wild-type and cen1&#226;&#338;&#172; cells had a            peptide that recognizes a tubulin modification, polyglu- <br /> significant population of cells that were completely unable      tamylation, that is abundant at basal bodies (Shang et al., <br /> to divide after conjugation (29 and 55%, respectively). Most     2002) (Figure 8). The number of cells with an obviously <br /> wild-type cells went on to form stable vegetative cultures       reduced number of basal bodies compared with wild-type <br /> (62%). In contrast, none of the cen1&#226;&#338;&#172; cells progressed to form   was counted at each time point. Cells proceeded through <br />  <br /> 3614                                                                                                  Molecular Biology of the Cell <br />                                                                                                                       Tetrahymena Centrins <br />  <br />  <br />  <br />  <br /> Figure 8. Deletion of CEN1 results in a failure to duplicate basal bodies. cen1&#226;&#338;&#172; knockout heterokaryon strains UCB8 and UCB9 were mated, <br /> and their progeny were analyzed at 17-, 24-, 48-, and 72-h time points by immunofluorescence microscopy. The monoclonal centrin antibody <br /> 20H5 was used along with an antibody that recognizes polyglutamic acid, which is prominent on modified tubulin at basal bodies. DNA is <br /> visualized with DAPI. WT is strain CU428 in logarithmic growth. Percentages refer to the proportion of cells with reduced numbers of basal <br /> bodies, with the images shown being representative (n &#226;&#171;&#189; 200). Rescue refers to UCB8 &#226;&#171;&#187; UCB9 progeny that were rescued by CEN1 <br /> transformed into a second site in the genome. In the WT and 17-h samples, 100% of cells had an apparently normal number of basal bodies. <br /> The oral apparatus is oriented on the right side of each panel. Bar, 10 &#226;&#65533;&#174;m. <br />  <br />  <br />  <br /> Vol. 16, August 2005                                                                                                                 3615 <br />  A. J. Stemm-Wolf et al. <br />  <br />  <br /> conjugation without apparent problems, and 17 h after ini-         antibody signal (our unpublished data). The possibility that <br /> tiation of mating the cells had separated and looked normal.       Cen1 is important for maintenance of basal bodies was <br /> However, 24 h after mating, some cells displayed reduced           addressed further by immunofluorescence microscopy us- <br /> centrin signal and fewer basal bodies along the cortical rows.     ing an antibody to &#226;&#65533;&#163;-tubulin, which detects cilia, a marker <br /> By 48 h, cells had fewer basal bodies, and by 72 h most of the     for a functioning basal body, as well as basal bodies. By 48 h <br /> cells showed a severe phenotype, including loss of most            after the initiation of mating of cen1&#226;&#338;&#172; cells, many punctate <br /> basal bodies, disorganization of the remainder along the           spots reminiscent of basal bodies remained but a large per- <br /> cortical rows, and loss of the oral apparatus. Cell division       centage of these did not have attached cilia, indicating the <br /> intermediates were observed in the samples, and although           absence of functional basal bodies at these sites (Figure 9B). <br /> these cells looked capable of completing the division cycle,       Together, the results from the micronuclear knockout strains <br /> because they did not accumulate at later time points, they         indicate that centrin is important for both the duplication <br /> were unable to form a new oral apparatus and had fewer             and stability of basal bodies. <br /> basal bodies, especially in the posterior half of the cell (Sup- <br /> plemental Figure 1A). Additionally, there was signal from          DISCUSSION <br /> the polyglutamic acid antibody at locations where there was <br /> no centrin signal, indicating that there was no longer enough      We report the initial characterization of a family of four <br /> Cen1 to localize to all of the remaining basal bodies.             centrin genes in the ciliate T. thermophila. Two of these genes <br />    To ensure that the loss of basal bodies was due to the lack     are closely related to the majority of the centrin genes found <br /> of Cen1, and not the result of the introduction of paromo-         in other organisms, whereas the other two are related to the <br /> mycin to the cultures, three controls were performed. First,       centrin genes identified in another ciliate, P. tetraurelia. Cil- <br /> UCB8 and UCB9 cen1&#226;&#338;&#172; knockout heterokaryon strains were             iates contain many distinct and highly specialized cytoskel- <br /> mated to each other, and the resulting culture was allowed         etal structures. In Tetrahymena, these include basal bodies, <br /> to progress in the absence of drug, allowing parental, un-         cilia, micronuclear spindles, macronuclear microtubule ar- <br /> mated cells to largely overtake the population. Nevertheless,      rays, a cortical network organized around the basal bodies <br /> the loss of centrin signal and basal bodies was still observed     of the ciliary rows, the contractile vacuole pores, and the <br /> in some cells, whereas in a wild-type control, cross-centrin       complex microtubule and filamentous structures that com- <br /> signal remained high throughout the population (Supple-            prise the oral apparatus (Gaertig et al., 1993; Frankel, 2000). <br /> mental Figure 1B; our unpublished data). In the second             Because centrins play multiple roles at structures associated <br /> control, a wild-type strain was subjected to 120 &#226;&#65533;&#174;g/ml paro-       with microtubules (Taillon et al., 1992), it would not be <br /> momycin and observed at time points after addition of drug         surprising if specialized centrins evolved along with some of <br /> similar to those used in the experiment with the knockout          these structures unique to ciliates. For example, GFP-Cen3 <br /> strains. Although dying cells often had disorganized basal         localizes predominantly to the fine filamentous reticulum of <br /> bodies, these cells were distinct from cen1 knockout cells in      the oral apparatus and a band at the apex of the cell, and <br /> that centrin signal remained strong, and these cells were          GFP-Cen4 localizes primarily to these structures and to the <br /> small and rounded (Supplemental Figure 1C). In the third           contractile vacuole pores. These structures are involved in <br /> control, a construct that reintroduces CEN1 at the GRL3/4          dynamic processes: the undulating membrane of the oral <br /> locus (Verbsky and Turkewitz, 1998) rescued cen1&#226;&#338;&#172; cells.           apparatus sweeps media (food) into the oral cavity, whereas <br /> UCB8 and UCB9 cells were mated, and this construct was             when the contractile vacuole expands, contracts, and rup- <br /> transformed into the developing macronucleus by electro-           tures, fluid is expelled through the CVPs maintaining the <br /> poration. Transformants were recovered that were resistant         cell&#226;&#8364;&#8482;s proper osmolarity. The function of the apical band is <br /> both to blasticidin S, which selects for the bsr gene used in      not known; it is distinct from the contractile division furrow <br /> this construct, and to paromomycin, which selects for the          ring, although it, too, may be contractile and involved in <br /> NEO2 cassette used to disrupt CEN1 (Figure 7A; our unpub-          regulating cell shape (Jerka-Dziadosz, 1981). In Paramecium <br /> lished data). Rescued cultures grew normally and were sub-         multimicrnucleatum, the number of contractile vacuoles in- <br /> jected to immunofluorescence microscopy (Figure 8, bot-            creases in response to an increased amount of calcium in the <br /> tom). The cortical rows and oral apparatus in these cells          environment (Iwamoto et al., 2003). The assumption is that <br /> were indistinguishable from wild type, indicating that when        this is a response with which the cell rids itself of excess <br /> Cen1 is restored to these cells, they no longer have basal         calcium, but it is possible that calcium, through centrin, <br /> body defects.                                                      more directly stimulates the assembly of this structure. Al- <br />    The immunofluorescence data on the cen1&#226;&#338;&#172; strain strongly        though the roles for Cen3 and Cen4 at the fine filamentous <br /> suggests that basal bodies are absent from these cells at later    reticulum and the contractile vacuole pore are not known, it <br /> time points. However, fluorescence signal at late time points      is possible the contractile properties of centrin affect the <br /> was still observed, more so with the antibody to polyglu-          dynamic properties of these structures (Salisbury et al., <br /> tamic acid, and the question remained whether this signal          1984). Additionally, Tetrahymena Cen1 along with inner arm <br /> was at basal bodies, or some identifiable remnant of basal         dynein is involved in ciliary beat changes in response to <br /> bodies. To address this issue, electron microscopy was per-        calcium (Hennessey et al., 2002; Guerra et al., 2003), and <br /> formed on cells processed 68 h after the initial mixing of the     perhaps centrins at the fine filamentous reticulum perform a <br /> two knockout heterokaryon strains. Fewer basal bodies were         similar task for this structure. <br /> observed in sections from the cen1 knockout strains com-              Tetrahymena CEN1 and CEN2 are on branches of the cen- <br /> pared with a wild-type control cross. When basal bodies            trin tree that are most widely conserved throughout the <br /> were detected, they often seemed incomplete without cilia          eukaryotic world, and here we examined the function of <br /> attached (Figure 9A). Occasionally the electron dense struc-       Cen1 in MTOC duplication. Centrin is required for this <br /> tures associated with the base of the basal body were present      process in species as diverse as yeast, green algae, and <br /> without all of the microtubules that normally comprise a           humans (Baum et al., 1986; Salisbury et al., 2002; Koblenz et <br /> basal body (our unpublished data). Immunoelectron micros-          al., 2003). Although GFP-Cen1 and GFP-Cen2 both localize <br /> copy with the Cen1 antibody revealed that basal bodies with        to basal bodies, CEN2 is not normally expressed during <br /> more Cen1 were better preserved than basal bodies with less        vegetative growth, or conjugation. This situation may be <br />  <br /> 3616                                                                                                   Molecular Biology of the Cell <br />                                                                                                                             Tetrahymena Centrins <br />  <br />  <br />  <br />  <br /> Figure 9. Cen1 is required for basal body maintenance. (A) Serial sections of a cen1&#226;&#338;&#172; cell (top) and a cell from a wild-type control cross <br /> (bottom). The asterisks (*) (top left) indicate the location of one incomplete basal body and a second site of interest. The basal body (top left) <br /> does not nucleate a cilium and seems disorganized at its base. The material at the bottom right is unrecognizable as a basal body, although <br /> it is present where a basal body would be expected given the location of the mitochondrion on the right (m) and the indentation in the cell <br /> surface. In the wild-type panels, three basal bodies (*), two with cilia clearly attached, are visible. Basal bodies are between mitochondria (m), <br /> and the cilia emerge from indentations in the cell surface. Bar, 0.5 &#226;&#65533;&#174;m. (B) &#226;&#65533;&#163;-Tubulin localization in cen1&#226;&#338;&#172; cells. Times are after mating <br /> initiation of the cen1&#226;&#338;&#172; knockout heterokaryon strains, and cells were stained with a mAb raised against &#226;&#65533;&#163;-tubulin (12G10). By 48 h, many <br /> apparent basal bodies do not have cilia attached. Bar, 10 &#226;&#65533;&#174;m. <br />  <br />  <br /> similar to that in humans, in which centrin1 expression is                 other centrins, in particular yeast Cdc31 and human cen- <br /> limited to the testes and retinal cells (Hart et al., 1999). We            trin3, leads us to suspect that it is a functional gene. CEN1, <br /> have not determined whether CEN2 is expressed under                        however, is highly expressed in vegetative and mating cells. <br /> different conditions, or not at all, although the similarity to            Localization of the endogenous protein places the majority <br />  <br /> Vol. 16, August 2005                                                                                                                         3617 <br />  A. J. Stemm-Wolf et al. <br />  <br />  <br /> of Cen1 at two positions on the basal body. One is at the          tenance. The genetic malleability of these cells will likely <br /> transition zone between the basal body and the axoneme of          lead to further insights into the mechanisms by which Cen1 <br /> the cilium, as has been reported in Chlamydomonas (Sanders         functions at the basal body, and Cen3 and Cen4 function at <br /> and Salisbury, 1994). The second is at the base of the basal       their respective locations in the cell. <br /> body adjacent to the kinetodesmal fiber where probasal <br /> body formation is initiated (Allen, 1969). Disruption of            <br /> CEN1 led to the progressive loss of basal bodies. Of the basal <br /> bodies that remained, many were defective and unable to act        <br /> as MTOCs for the cilia. Cen1 is a stable molecule, and small       <br /> amounts of Cen1 remained even 72 h after mating initiation        <br /> of the knockout heterokaryon strains. Therefore, our exper-  <br /> iments examine the results of a slow depletion of Cen1. As         <br /> Cen1 decreased, cells were unable to fill their cortical rows     <br /> with the proper number of basal bodies, or to form a new          <br /> oral apparatus, suggesting the requirement for Cen1 in the     <br /> duplication of basal bodies.                                       <br />    The deterioration of the remaining basal bodies indicates    <br /> that Cen1 also is required for proper basal body mainte-            <br /> nance. The depletion of &#226;&#65533;&#165;-tubulin yielded a related pheno-         <br /> type, demonstrating the necessity for this protein in basal       <br /> body maintenance as well as basal body duplication. When            <br />                                                                    <br /> &#226;&#65533;&#165;-tubulin is depleted, oral apparatus basal bodies are lost         <br /> before cortical basal bodies, suggesting that turnover of <br /> &#226;&#65533;&#165;-tubulin is more rapid in the oral apparatus than at the <br /> basal bodies of the cortical rows, and required for these          <br /> organelles&#226;&#8364;&#8482; stability (Shang et al., 2002). The CEN1 knockout       <br /> strain had no such hierarchy of loss, indicating that turnover      <br /> of Cen1 is not significantly different at basal bodies along the    <br /> cortical rows and basal bodies at the oral apparatus. Instead,     <br /> Cen1 seems to be very stably associated with basal bodies          <br /> and plays some role in preserving these organelles. Hence, a        <br /> cen1&#226;&#338;&#172; cell preparing for division can have one existing oral     <br /> apparatus that seems normal and a highly aberrant devel-           <br /> oping oral apparatus (Supplemental Figure 1A). We do not           <br /> know whether Cen1&#226;&#8364;&#8482;s roles in basal body duplication and <br /> maintenance can be separated, but it is tempting to speculate     <br /> that Cen1 localization to the base of the basal body is im-       <br /> portant for duplication, whereas localization to the transi- <br />                                                                     <br /> tion zone is important for basal body maintenance and             <br /> function. Future work on Cen1 binding partners will shed <br />                                                                   <br /> light on this issue.                                               <br />    Our experiments did not reveal any association between <br />                                                                     <br /> the Tetrahymena centrin proteins and the micronucleus. This        <br /> is somewhat surprising because centrins are found at mitotic <br />                                                            <br /> spindle poles in a wide range of organisms. This supports           <br /> the previous finding that centrin was not detected with            <br /> &#226;&#65533;&#165;-tubulin at the micronucleus (Shang et al., 2002). However, <br /> centrin antibodies tested to date have not revealed centrin        <br /> localization at the contractile vacuole pores, and our analysis <br /> of Cen4 indicates that it is likely present at this structure.     <br /> Thus, the possibility remains that the experiments have not         <br /> been sensitive enough to reveal centrin at spindle poles.          <br /> Also, the GFP tag could interfere with centrin localization to      <br /> the spindle pole in Tetrahymena. Another possibility is that       <br /> there are yet more Tetrahymena centrin genes encoding iso-         <br /> forms that are at the spindle pole. Finally, it is possible that  <br /> Tetrahymena does not require centrin for its mitotic appara- <br /> tus. Few components of the Tetrahymena mitotic apparatus         <br /> have been identified, and this issue may take some time to          <br /> resolve. <br />                                                            <br />    We have shown there exists a family of Tetrahymena cen-          <br /> trin genes with at least three bona fide members based on <br /> expression. Two of the centrins likely function at specialized      <br /> fibrillar structures in the oral apparatus and the apical tip,      <br /> and one of these also may act at the contractile vacuole pore.      <br /> The predominant centrin, encoded by CEN1, localizes to              <br /> basal bodies, and is required for their duplication and main-       <br />  <br /> </body></html>