<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The RNA silencing endonuclease <br /> Argonaute 2 mediates specific antiviral <br /> immunity Drosophila melanogaster <br /> Ronald P. van Rij,1 Maria-Carla Saleh,1 Bassam Berry,2 Catherine Foo,1 Andrew Houk,1 <br /> Christophe Antoniewski,2 Raul Andino1,3 <br /> 1 <br />  Department Microbiology Immunology, University California San Francisco, San Francisco, California 94143, <br /> USA; 2Department Developmental Biology, Centre national la recherche scientifique (CNRS) URA 2578 Pasteur <br /> Institute, Paris 75015, France <br />  <br />  <br />  <br />  <br /> Most organisms evolved defense mechanisms protect viruses pathogens. <br /> Arthropods lack protein-based adaptive immune response vertebrates. Here the <br /> central catalytic component RNA-induced silencing complex (RISC), nuclease Argonaute 2 (Ago-2), <br /> is essential antiviral defense adult Drosophila melanogaster. Ago-2-defective flies hypersensitive <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> infection major fruit fly pathogen, Drosophila C virus (DCV), Cricket Paralysis virus (CrPV). <br /> Increased mortality ago-2 mutant flies associated dramatic increase viral RNA accumulation <br /> and virus titers. The physiological significance antiviral mechanism underscored finding that <br /> DCV encodes potent suppressor <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi). This suppressor binds long double-stranded <br /> RNA (dsRNA) inhibits Dicer-2-mediated processing dsRNA short interfering RNA (siRNA), but <br /> does bind short siRNAs disrupt microRNA (miRNA) pathway. Based results propose <br /> that RNAi major antiviral immune defense mechanism Drosophila. <br /> [Keywords: Antiviral defense; RNAi suppressor; dsRNA-binding domain; Drosophila C virus; Cricket Paralysis <br /> virus] <br /> Received August 14, 2006; revised version accepted September 15, 2006. <br />  <br />  <br />  <br />  <br /> Higher vertebrates rely highly sophisticated                      plementary sites cytoplasmic RNA (&#226;&#8364;&#339;Slicer&#226;&#8364;? activity) <br /> &#226;&#8364;&#339;adaptive&#226;&#8364;? immune targets pathogens                        (Liu et al. 2004). The related distinct microRNA <br /> exquisite specificity broad-action, nonspe-                    (miRNA) pathway uses short, endogenously encoded <br /> cific, &#226;&#8364;&#339;innate&#226;&#8364;? immune response includes in-                       hairpin RNAs sequentially processed ma- <br /> duction nonspecific RNaseL (Silverstein 1989). In-                   ture miRNAs RNases: nuclear Drosha and <br /> triguingly, invertebrates lack protein-based adaptive                   cytoplasmic Dicer. These miRNAs incorporated <br /> immune response, capable effectively fighting                    miRISC, promotes RNA cleavage transla- <br /> viral infections. This raised question                   tional inhibition recognition cognate target sites <br /> there additional adaptive and/or innate antiviral                       3&#226;&#172;&#732; untranslated region (UTR) mRNAs (Bartel <br /> mechanisms distinct classical mammalian im-                        2004). <br /> mune (Brennan Anderson 2004).                                       While miRNAs known regulate gene expression <br />    RNA interference (RNAi) evolutionarily con-                        essential proper development cell physiology <br /> served mechanism sequence-specific gene silencing                       (John et al. 2004; Bentwich et al. 2005; Berezikov et al. <br /> guided double-stranded RNA (Meister Tuschl                           2005), function RNAi cleavage pathway in <br /> 2004). The nonspecific endonuclease Dicer cleaves long,                     animals remains unclear. Given RNA silencing <br /> exogenous double-stranded RNA (dsRNA) short in-                        pathway plays critical role antiviral defense plants <br /> terfering RNA (siRNA), &#226;&#710;&#188;21- 25-<span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span>-pair (bp) dsRNA                      (Baulcombe 2004; Voinnet 2005), hypoth- <br /> with 2-nucleotide (nt) 3&#226;&#172;&#732; overhang. These siRNAs                      esized RNAi cleavage pathway animals also <br /> subsequently incorporated RNA-induced silenc-                        provides defense mechanism parasitic nucleic <br /> ing complex (RISC), guide recognition                        acids, viruses transposons. Several observa- <br /> Argonaute (Ago)-2-mediated cleavage perfectly com-                       tions support hypothesis. Introduction artificial <br />                                                                             dsRNA siRNAs insects mammalian cells elic- <br /> 3 <br />                                                                             effective antiviral response protects cells <br />  Corresponding author. <br /> E-MAIL raul.andino@ucsf.edu; FAX (415) 514-4112.                            highly lytic viruses (Gitlin et al. 2002; Li et al. <br /> Article online http://www.genesdev.org/cgi/doi/10.1101/gad.1482006.   2002; van Rij Andino 2006). The high specificity of <br />  <br />  <br />          GENES &amp; DEVELOPMENT 20:2985&#226;&#8364;&#8220;2995 &#194;&#169; 2006 Cold Spring Harbor Laboratory Press ISSN 0890-9369/06; www.genesdev.org         2985 <br />  van Rij et al. <br />  <br />  <br />  <br /> antiviral RNAi highlighted observation          expressing FHV B2 defective RNA silencing elic- <br /> resistant viruses readily emerge single mismatches        ited long dsRNA siRNA, miRNA <br /> within siRNA-target sequence (Gitlin Andino            pathway (B. Berry C. Antoniewski, unpubl.). B2 <br /> 2003; Gitlin et al. 2005). Thus, RNA-based antiviral        transgenic flies experienced rapid course dis- <br /> defense mechanism capacity provide highly         ease died faster inoculation DCV. Median <br /> sequence-specific response. However, observed sus-         <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> 2&#226;&#8364;&#8220;3 d shorter presence B2 in <br /> ceptibility viruses synthetic siRNAs does prove      control flies (Fig. 1A). Since FHV B2 inhibits dsRNA <br /> that antiviral RNAi response initiated          processing Dicer (Lu et al. 2005; Sullivan Ganem <br /> natural infection.                                             2005), results suggest Drosophila Dicer-2 can <br />    Recently, evidence surfaced supporting con-         digest viral dsRNA, consistent recent reports dem- <br /> cept RNAi involved defense viruses      onstrating increase viral replication Dicer-2-null <br /> in insects (Keene et al. 2004; Sanchez-Vargas et al. 2004;     mutant flies (Galiana-Arnoux et al. 2006; Wang et al. <br /> van Rij Andino 2006). Components RNAi ma-           2006). Digestion viral dsRNA could, principle, ac- <br /> chinery, including Dicer-2 cofactor R2D2,         count antiviral activity Dicer. Alternatively, <br /> shown required defense fruit flies        Dicer initiate sequence-specific antiviral <br /> viral infection (Galiana-Arnoux et al. 2006; Wang et al.       response, viral siRNAs generated Dicer-2 are <br /> 2006). Because Dicer sequence-nonspecific endo-           incorporated RISC, used target single- <br /> nuclease, principle possible antiviral       stranded viral RNA destruction. If case, <br /> function RNAi pathway elements relies cleav-         hypothesized antiviral activity flies would <br /> age essential double-stranded intermediates RNA          impaired loss RISC function. <br /> replication Dicer, similar innate response car- <br /> ried RNaseL mammals. Alternatively, an- <br /> tiviral activity rely sequence-specific activi- <br /> ties RNAi response. In scenario, viral siRNAs <br /> produced Dicer-2 incorporated RISC, leading <br /> to specific cleavage viral mRNAs RISC and, <br /> consequently, suppression virus infection. <br />    To distinguish possibilities, test <br /> whether RNAi plays central auxiliary role in <br /> antiviral defense, investigate role RISC in <br /> the response viral infection Drosophila melanogas- <br /> ter. We flies defective catalytic com- <br /> ponent RISC, nuclease Ago-2, hypersensitive <br /> to infection Drosophila C virus (DCV) Cricket <br /> Paralysis virus (CrPV), resulting 1000-fold increase <br /> in virus production. Thus, Ago-2/Slicer key an- <br /> tiviral response adult fruit flies. Furthermore, iden- <br /> tified suppressor RNAi encoded DCV genome, <br /> supporting central role RNAi natural antiviral <br /> defense flies. This suppressor specifically binds long <br /> dsRNA, inhibits Dicer processing, inhibits <br /> dsRNA-initiated RNAi. Together, results provide <br /> direct support notion RNAi central the <br /> natural antiviral immune response Drosophila. <br />  <br />  <br /> Results <br />                                                                Figure 1. D. melanogaster RNAi mutant flies hypersensi- <br /> The RISC catalytic component Ago2 required                  tive virus infection. (A) Transgenic expression flock <br />                                                                house virus B2 protein, inhibitor Dicer function, increases <br /> for protection virus infection <br />                                                                sensitivity DCV infection. Flies indicated genotypes <br /> To examine mechanism antiviral immunity          injected abdomen 350 TCID50 DCV, sur- <br /> fruit fly employed Drosophila C virus (DCV), posi-        vival monitored daily. B2 expression (under control the <br /> tive-strand RNA virus bicistrovirus family        UASp sequence) induced crossing UAS-B2 flies flies <br /> a major pathogen Drosophila. Injection DCV        expressing Gal4 transcription factor Daughterless <br />                                                                promoter. The F1 progeny cross (B2/+; Gal4/+, open <br /> abdomen fly results dose-dependent mortality <br />                                                                squares) expressed B2, B2 expression detectable <br /> (Cherry Perrimon 2004). To evaluate importance         control crosses (data shown). (B,C) Survival Ago- <br /> of insect RNAi machinery controlling virus repli-       2-null male female mutant flies virus infection. Ho- <br /> cation, initially studied effect potent viral      mozygous ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217;, dcr-2&#226;&#710;&#8217;/&#226;&#710;&#8217;, wild-type (wt) flies in- <br /> suppressor RNAi, flock house virus B2 protein (Lu       jected 350 TCID50 DCV (B) CrPV (C) monitored <br /> et al. 2005; Sullivan Ganem 2005). Transgenic flies        daily survival. <br />  <br />  <br /> 2986        GENES &amp; DEVELOPMENT <br />                                                                                                                    Antiviral function Ago2 <br />  <br />  <br />  <br />    To directly establish antiviral response in <br /> flies mediated RISC examined function of <br /> the catalytic component RISC, Ago2, protection <br /> against virus infection. The ago2414 mutant flies (ago2&#226;&#710;&#8217;/&#226;&#710;&#8217;) <br /> carry partial deletion ago-2 gene, conse- <br /> quently produce Ago2 defective RNAi <br /> (Okamura et al. 2004). Strikingly, ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; Dicer-2&#226;&#710;&#8217;/&#226;&#710;&#8217; <br /> mutant flies hypersensitive virus infection with <br /> either related bicistroviruses, DCV CrPV. In <br /> Ago2-defective flies, observed earlier onset of <br /> disease wild-type flies (Fig. 1B,C; Table 1). After <br /> injection 350 TCID50 virus, median survival of <br /> ago2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies &#226;&#710;&#188;5 d viruses, compared with <br /> &gt;14 d wild-type flies. Furthermore, disease occurred at <br /> lower viral doses. The 50% lethal dose (LD50) ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; <br /> or B2 transgenic flies 40- 280-fold lower <br /> than control flies (Table 1). These experiments indi- <br /> cate RISC activity plays central role effective <br /> antiviral defense adult flies. <br />  <br />  <br /> Increased susceptibility virus infection ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; <br /> mutant flies associated increased levels <br /> of viral RNA accumulation virus titers <br /> There important caveat studying viral sensi- <br /> tivity flies defective components major <br /> cellular pathway, RNAi. Enhanced death follow- <br />                                                                          Figure 2. DCV replicates higher levels ago-2-null mutant <br /> ing viral infection caused decrease fitness <br />                                                                          flies. (A) Homozygous ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; wild-type flies injected <br /> or general health mutant animal, a <br />                                                                          350 TCID50 DCV, virus production flies was <br /> direct antiviral activity deleted component. Ac-                  monitored time. At time point, pools flies <br /> cordingly, tested increased mortality                  homogenized, viral titer homogenate was <br /> ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; mutant flies associated direct in-                   determined end-point dilution. Titers represent averages and <br /> crease virus replication (Fig. 2). Even early time                 standard deviations independent pools flies each. <br /> points post-infection, viral titers 1000-fold higher                (B,C) DCV RNA accumulation efficient ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217;-null <br /> in ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies compared wild-type controls (Fig.                 mutant flies. ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; wild-type flies injected 350 <br /> 2A), indicating Ago2 plays direct role sup-                TCID50 DCV, 25 flies harvested daily detection of <br /> pressing viral replication. The viral titers wild-type                viral RNA. (B) Ethidium bromide-stained agarose gel total <br />                                                                          RNA preparation. (C) Northern blot analysis viral RNA. As a <br /> controls follow-up increased beyond <br />                                                                          loading control, blots stripped rehybridized a <br /> &#226;&#710;&#188;5 log10 TCID50 fly (at days 5, 8, 11 post-infec- <br />                                                                          probe specific actin 42A. Numbers panels <br /> tion), reached virus production levels                indicate days post-infection. <br /> observed ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies days 2 3 post-infection <br />  <br />                                                                          (&#226;&#710;&#188;8.0 log10 TCID50). The increase viral titers ago- <br /> Table 1. Lethal viral doses RNAi-defective flies                      2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies mirrored dramatic increase viral <br /> Drosophila strain                 Virus                LD50(TCID)a       RNA levels (Fig. 2B,C). Indeed, viral RNA Ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; <br />                                                                          mutant flies infected DCV 2 3 d be <br /> Wild-type                         DCV                       171          directly detected following ethidium bromide staining of <br /> ago&#226;&#710;&#8217;/&#226;&#710;&#8217;                                                        4 <br />                                                                          agarose gel analyzing total RNA. In contrast, wild- <br /> +/+; Gal4/+                       DCV                       118 <br /> B2/+; +/+                                                    70 <br />                                                                          type control flies viral RNA undetectable 9 <br /> B2/+; Gal4/+                                                &lt;21b         d post-infection (Fig. 2B). We confirmed iden- <br /> Wild-type                         CrPV                     1359          tity RNA bands observed ethidium bro- <br /> ago&#226;&#710;&#8217;/&#226;&#710;&#8217;                                                        5          mide-stained gel Northern blot analysis (Fig. 2C). <br />                                                                          While wild-type flies viral RNA barely detected <br /> Calculation LD50 values mutant strain using the <br />                                                                          day 6, ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; mutant flies virus RNA was <br /> Reed Muench method. <br /> a <br />   LD50 values expressed TCID (tissue culture infectious <br />                                                                          clearly observed 24 h post-infection, 48 h virus <br /> doses).                                                                  RNA accumulated higher levels wild- <br /> b <br />   In B2/+; Gal4/+ cross, &gt;50% flies died highest di-   type control flies (Fig. 2C). We conclude en- <br /> lution tested, LD50 value established.     hanced viral susceptibility ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies their <br /> This number represents upper limit actual LD50 value.          inability control virus replication. The dramatic in- <br />  <br />  <br />                                                                                                       GENES &amp; DEVELOPMENT              2987 <br />  van Rij et al. <br />  <br />  <br />  <br /> crease viral titers observed RISC inactivated <br /> establishes ago-2 host susceptibility locus virus <br /> infection provides direct rationale acceler- <br /> ated progression disease ago-2&#226;&#710;&#8217;/&#226;&#710;&#8217; flies. <br />  <br />  <br /> DCV encodes potent suppressor <br /> of RNAi pathway <br /> Our results indicate RNAi pathway central <br /> and highly potent antiviral defense mechanism flies. <br /> Given plant viruses partially overcome the <br /> antiviral RNAi response encoding suppressors of <br /> RNA silencing, hypothesized DCV, un- <br /> der stringent RNAi control flies, evolved <br /> similar RNAi suppressive activities. To determine <br /> whether DCV encodes suppressor RNAi, Drosophila <br /> S2 cells infected DCV and, 24 h later, were <br /> cotransfected reporter RNAi silencing <br /> activity, consisting Renilla luciferase-expressing <br /> plasmid specific 541-nt-long dsRNA targeting Re- <br /> nilla mRNA. A transfection control consisting the <br /> unrelated firefly luciferase included experi- <br />                                                              Figure 3. Drosophila C virus encodes suppressor long <br /> ment. In uninfected control cells, Renilla expression    dsRNA-initiated RNAi. (A) Uninfected DCV-infected Dro- <br /> efficiently silenced (798-fold suppression compared     sophila S2 cells transfected expression plasmids en- <br /> a control dsRNA). In contrast, silencing ineffi-    coding firefly Renilla luciferase 541-bp dsRNA <br /> cient DCV-infected cells (sixfold suppression) (Fig.      targeting Renilla luciferase unrelated control dsRNA. Renilla <br /> 3A). Thus, DCV infection inhibits RNAi in-           luciferase counts normalized using firefly luciferase <br /> duced 541-nt-long dsRNA. We examined         counts, data expressed fold silencing compared with <br /> mechanism viral inhibition RNAi silencing. Be-         control dsRNA. (B) Experiment performed described in <br /> cause long dsRNA needs processed siRNA         A, 21-bp synthetic siRNA targeting firefly luciferase un- <br />                                                              related control siRNA cotransfected initiate RNAi. (C) <br /> Dicer-2 order induce gene silencing, deter- <br />                                                              Experiment performed described A. RNAi initi- <br /> mined DCV inhibits RNAi disrupting <br />                                                              ated dsRNAs 21&#226;&#8364;&#8220;592 nt <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, indicated. Silencing <br /> dsRNA processing. To end, repeated experi-       expressed percent silencing DCV-infected unin- <br /> ment using synthetic 21-bp siRNA silence firefly lu-      fected S2 cells. Data A&#226;&#8364;&#8220;C indicate averages standard de- <br /> ciferase expression short RNAs directly in-     viations independent experiments. (D) Processing of <br /> corporated RISC, overcoming need           dsRNA Drosophila S2 cell extract. S2 cells mock-infected, <br /> Dicer processing. In case, siRNA-induced silencing      infected DCV CrPV 20 h preparation cell <br /> was affected DCV infection (Fig. 3B). To      extracts, MOI 1. Processing 116-bp dsRNA 21-bp <br /> define substrate DCV RNAi suppressor,          siRNA analyzed 12% denaturing polyacrylamide gel. An <br /> used panel dsRNA preparations different sizes     end-labeled 21-bp synthetic siRNA used size marker. <br /> induce RNAi firefly luciferase reporter (Fig. 3C). <br /> DCV suppressed RNAi induced dsRNA ranging            involved processing long dsRNA RNAi <br /> 31 bp 592 bp length, unable inhibit      pathway (Lee et al. 2004), Dicer-1, involved in <br /> 21-bp siRNA-initiated RNAi; furthermore, suppression         miRNA biogenesis. To determine suppres- <br /> of RNAi efficient longer dsRNA prepara-        sor activity encoded DCV general inhibitor of <br /> tions (Fig. 3C). These data suggest DCV        Dicer-like activities, examined DCV <br /> suppressor RNAi acts upstream RISC,         disrupts miRNA pathway. To end, moni- <br /> the level dsRNA processing Dicer-2, pro-         tored miRNA levels infected S2 cells time by <br /> vides siRNAs RNAi silencing pathway. To          Northern blotting (Fig. 4A). The levels mature <br /> directly test possibility, studied virus     miRNA ratio pre-miRNA mature miRNA <br /> infection inhibits Dicer-2 activity cell extracts. We     (a processing step mediated Dicer-1) not <br /> prepared S2 cell extracts uninfected cells      changed infected cells. We examined whether <br /> cells infected DCV CrPV 20 h prior      endogenous miRNAs assemble functional <br /> preparation extracts. In uninfected extracts dsRNA        RISC DCV-infected cells. We used firefly luciferase <br /> was efficiently processed 21-bp siRNAs (Fig. 3D, lane     reporter coding mRNA containing copies a <br /> 2). In contrast, siRNA production inhibited DCV-      sequence targeted mature miR2b 3&#226;&#172;&#732;UTR <br /> infected extracts (Fig. 3D, lane 3). Inhibition Dicer-2   (Fig. 4B, antisense; Saleh et al. 2006). As control, we <br /> function specific DCV, closely related     used firefly luciferase mRNA containing se- <br /> CrPV did affect dsRNA processing (Fig. 3D, lane 4).      quences reverse orientation (Fig. 4B, sense). S2 <br />    Drosophila encodes Dicer-related genes: Dicer-2,      cells cotransfected firefly luciferase <br />  <br />  <br /> 2988        GENES &amp; DEVELOPMENT <br />                                                                                                              Antiviral function Ago2 <br />  <br />  <br />  <br />                                                                        identified canonical dsRNA-binding domain (dsRBD) <br />                                                                        N terminus ORF-1 DCV. We hypothesized <br />                                                                        domain responsible suppression of <br />                                                                        RNAi, given plant virus suppressors RNAi <br />                                                                        bind dsRNA (Lakatos et al. 2006; Merai et al. 2006) and <br />                                                                        observation dsRNA processing inhibited by <br />                                                                        DCV infection. We subcloned dsRBD varying <br />                                                                        lengths flanking sequence tested ability sup- <br />                                                                        press RNAi Drosophila S2 cells (Fig. 5A). As the <br />                                                                        dsRBD likely N-terminal functional viral <br />                                                                        gene product, tentatively refer protein as <br />                                                                        DCV-1A. <br />                                                                           Expression small fragment 99 residues, <br />                                                                        include N-terminal 22 amino acids ORF-1, <br />                                                                        66-amino-acid minimal dsRBD, 11 amino acids <br />                                                                        C-terminal sequence, able efficiently suppress <br />                                                                        RNAi (Fig. 5B). Increasing length DCV frag- <br />                                                                        ment did augment efficiency suppres- <br />                                                                        sion RNAi. dsRBDs highly conserved well <br />                                                                        studied protein domains (Tian et al. 2004; Chang and <br />                                                                        Ramos 2005). To determine DCV-1A suppres- <br />                                                                        sion requires binding dsRNA, mutated highly con- <br />                                                                        served residues dsRBD (L28Y AA80KK) or <br />                                                                        residues expected interact dsRNA based avail- <br />                                                                        able structures (E51A KK73AA) (Fig. 5C). Three out <br />                                                                        mutations abolished RNAi suppression, includ- <br />                                                                        ing KK73AA substitution expected directly inter- <br />                                                                        act dsRNA (Fig. 5D). <br /> Figure 4. DCV infection does affect miRNA biogenesis              To test DCV-1A inhibit RNA silencing in <br /> function. (A) Processed miRNA levels DCV-infected un-           intact animal, generated transgenic flies express- <br /> infected S2 cells. S2 cells infected DCV MOI        ing WT-DCV-1A, inactive KK73AA mutant, GFP <br /> 0.01 100, presence miR2b time analyzed         control developing eye. Efficiency RNA <br /> by Northern blot. As loading control, blot stripped      silencing monitored flies crossing them <br /> rehybridized probe specific U6 snRNA. (B) miR2b          flies expression mini-white genes was <br /> assembled functional RISC complexes. Uninfected or <br />                                                                        suppressed expression hairpin dsRNA targeting <br /> DCV-infected (MOI 1) S2 cells transfected plas- <br /> mid expressing luciferase mRNA containing copies a <br />                                                                        white gene. Under conditions white gene <br /> sequence perfectly complementary mature miR2b            efficiently silenced, evidenced loss red eye <br /> either sense antisense orientation 3&#226;&#172;&#732;UTR. Lucifer-       pigmentation resulting orange eye color GFP <br /> ase activity normalized expression cotransfected       control flies (Fig. 5E, GFP). In flies expressing wild- <br /> Renilla luciferase plasmid does contains miR2b comple-        type DCV-1A, RNA silencing efficiently suppressed, <br /> mentary sites. Luciferase activity expressed ratio nor-   resulting expression white gene and, accord- <br /> malized luciferase counts luciferase-miR-2 antisense lucif-    ingly, red eye color. As expected, flies expressing the <br /> erase-miR-2 sense transfections. Bars indicate averages stan-      KK73AA DCV-1A mutant, unable suppress <br /> dard deviations independent experiments.                       RNAi cell culture (Fig. 5D), white gene si- <br />                                                                        lenced, albeit efficiently GFP-transgenic <br /> miR2b reporters Renilla luciferase                   control flies (Fig. 5E). Taken together, data indicate <br /> transfection control. Twenty-four hours transfec-                DCV-1A protein acts potent RNAi suppres- <br /> tion, luciferase expression induced luciferase                 sor flies confirm dsRBD crucial the <br /> activity monitored. The ratio firefly lucif-               suppressor activity. <br /> erase activity mRNA containing sense antisense <br /> miR2b sequence virtually identical infected and <br />                                                                        DCV-1A binds long dsRNA high affinity and <br /> uninfected S2 cells (Fig. 4B). Thus, DCV encodes ac- <br />                                                                        inhibits Dicer processing <br /> tivity specifically inhibits RNAi silencing initiated by <br /> long dsRNAs does affect miRNA pathway.                     To examine DCV-1A suppressor activ- <br />                                                                        ity mediated direct binding dsRNA Dicer-2 <br />                                                                        substrate, expressed DCV-1A fusion glutathi- <br /> The RNAi suppressor dsRNA-binding protein <br />                                                                        transferase (WT-DCV-1A) Escherichia coli and <br /> encoded N terminus ORF-1 <br />                                                                        analyzed dsRNA binding electrophoretic mobility <br /> To define viral factor responsible RNAi suppres-               shift assays (EMSA). As control, analyzed inac- <br /> sion, examined amino acid sequence DCV                   tive mutant suppressor, L28Y-DCV-1A (Fig. 3C). While <br />  <br />  <br />                                                                                                   GENES &amp; DEVELOPMENT            2989 <br />  van Rij et al. <br />  <br />  <br />  <br />  <br /> Figure 5. Drosophila C virus dsRBD pro- <br /> tein suppressor RNAi. (A) Sche- <br /> matic representation DCV genome. <br /> The graphic indicates dsRBD-con- <br /> taining domains cloned ex- <br /> amined suppression RNAi. (B) Dro- <br /> sophila S2 cells transfected with <br /> plasmids expressing firefly Renilla lu- <br /> ciferase indicated dsRBD-contain- <br /> ing DCV ORF-1 domains. RNAi initi- <br /> ated adding dsRNA specific firefly <br /> luciferase control dsRNA culture <br /> supernatant. Data expressed fold si- <br /> lencing compared control unrelated <br /> dsRNA. Note soaking dsRNA less <br /> efficient method induce silencing than <br /> cotransfecting dsRNA, Fig- <br /> ure 3. (C) Alignment dsRBD sequences <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> DCV different model organisms. <br /> Boxed residues indicate residues di- <br /> rectly interact dsRNA Xenopus <br /> laevis RNA-binding protein A X-ray <br /> structure (Ryter Schultz 1998). (D) <br /> Suppression RNAi wild-type and <br /> mutant DCV dsRBD S2 cells. Experi- <br /> ment performed described B. <br /> Data B D indicate averages and <br /> standard deviations independent <br /> experiments. (E) DCV-1A suppresses <br /> RNAi vivo. RNA silencing white <br /> gene inverted repeat (IR [W]) flies <br /> that transgenic wild-type (wt) or <br /> KK73AA DCV-1A, GFP control. <br /> Maximum white gene expression the <br /> absence RNAi examined the <br /> GMR-Gal4/+; UASp-GFP/+ control flies <br /> that express inverted repeat (left <br /> panel, control). <br />  <br />  <br /> WT-DCV-1A associated 211-bp-long dsRNA              affinity diverse set dsRNAs, indicating the <br /> probe high affinity (Fig. 6A,B), L28Y-DCV-1A      interaction sequence nonspecific (data shown). In <br /> mutant unable interact dsRNA (Fig. 6C,     tissue culture, DCV infection inhibited RNAi function <br /> lanes 4&#226;&#8364;&#8220;10). With increasing concentrations WT-DCV-     elicited exogenous dsRNA 31 bp longer, <br /> 1A, discrete bands appeared decreasing mobility,      DCV unable prevent siRNA-initiated si- <br /> indicating multiple binding events DCV-1A single   lencing (Fig. 3B). Furthermore, suppression RNAi was <br /> dsRNA. We observed binding ssRNA DCV-1A           efficient longer, 116- 592-bp, dsRNA prepa- <br /> (Fig. 6C, lanes 11&#226;&#8364;&#8220;18), DCV-1A bound similar      rations (Fig. 3C). We examined binding <br />  <br />  <br /> 2990        GENES &amp; DEVELOPMENT <br />                                                                                                                       Antiviral function Ago2 <br />  <br />  <br />  <br />  <br /> Figure 6. DCV-1A dsRNA-binding activity inhibits Dicer processing dsRNA. (A) RNA mobility shift analyses showing <br /> binding DCV-1A dsRNA 211 bp (left panel) 31 bp (middle panel), 21-bp siRNA (right panel). Each probe was <br /> incubated buffer (&#226;&#710;&#8217;) increasing concentrations WT-DCV-1A. Concentrations protein 0 (lanes 1,9,15), 0.0001 (lanes <br /> 2,16), 0.001 (lanes 3,10,17), 0.005 (lanes 4,11,18), 0.01 (lanes 5,12,19), 0.05 (lanes 6,13,20), 0.1 (lanes 7,14,21), 0.5 &#194;&#181;M (lane 8). <br /> Position free probe indicated arrows. (B) Binding affinity WT-DCV-1A dsRNAs indicated lengths. The percentage <br /> of radiolabeled probe bound DCV-1A total probe mobility shift assays plotted WT-DCV-1A concentrations. (C) <br /> RNA mobility shift assay mutant L28Y-DCV-1A 211-bp dsRNA (left panel) WT-DCV-1A 211-nt ssRNA (right panel). <br /> Each probe incubated buffer (&#226;&#710;&#8217;) increasing concentrations wild-type mutant DCV-1A. A mobility shift assay of <br /> WT-DCV-1A 211-bp dsRNA probe run parallel positive control (lanes 1,2). Concentrations protein 0 (lanes <br /> 1,3,11), 0.0001 (lanes 4,12), 0.001 (lanes 5,13), 0.005 (lanes 6,14), 0.01 (lanes 7,15), 0.05 (lanes 8,16), 0.1 (lanes 9,17), 0.5 &#194;&#181;M (lanes <br /> 2,10,18). (D) DCV-1A inhibits Dicer processing S2 cell extracts. Radiolabeled 116-bp dsRNA processed siRNA S2 cell <br /> extract presence increasing concentrations wild-type L28Y-DCV-1A buffer (&#226;&#710;&#8217;). Reaction products analyzed on <br /> a 12% denaturing polyacrylamide gel. An end-labeled 21-bp synthetic siRNA used size marker (Mkr). Concentrations of <br /> DCV-1A protein 0 (lane 2), 0.5 (lane 3), 0.125 (lanes 4,8), 0.05 (lanes 5,9), 0.005 (lanes 6,10), 0.00125 &#194;&#181;M (lanes 7,11). <br />  <br />  <br />  <br /> of DCV-1A depends size dsRNA. WT-DCV-                       Dicer-2 (Fig. 6D, lane 2). Addition wild-type DCV-1A <br /> 1A does bind efficiently 21-bp siRNA,                   inhibited Dicer activity dose-dependent manner (Fig. <br /> formed weak complexes highest protein concen-                      6D, lanes 3&#226;&#8364;&#8220;7). In contrast, L28Y DCV-1A mutant <br /> tration used (Fig. 6A [lanes 15&#226;&#8364;&#8220;21], B). In contrast,                  effect Dicer activity (Fig. 6D, lanes 8&#226;&#8364;&#8220;11). <br /> observed increasing binding affinity DCV-1A                        These results indicate DCV-encoded RNAi sup- <br /> dsRNA probes increasing size (Fig. 6A,B). Thus,                    pressor requires dsRNA-binding activity inhibition of <br /> affinity DCV-1A different lengths dsRNA                     Dicer-2 function. Furthermore, biochemical analy- <br /> closely correlates efficiency DCV mediate sup-                 ses, indicated binding short dsRNAs the <br /> pression RNAi (Fig. 3C).                                               suppressor inefficient, explain observations <br />    To establish DCV-1A directly respon-                       DCV unable inhibit siRNA- miRNA-initi- <br /> sible observed inhibition Dicer activity (Fig.                 ated RNAi vivo. <br /> 3D), analyzed processing dsRNA siRNA in <br /> cell extracts presence recombinant DCV-1A. S2 <br />                                                                           Discussion <br /> cell extracts incubated dsRNA presence <br /> of increasing concentrations wild-type mutant                       Our results support notion RNA silencing is <br /> DCV-1A. In absence exogenous viral protein,                        crucial antiviral defense adult D. melanogaster. <br /> dsRNA efficiently processed 21-bp siRNA                        The severity infection observed flies lacking func- <br />  <br />  <br />                                                                                                          GENES &amp; DEVELOPMENT              2991 <br />  van Rij et al. <br />  <br />  <br />  <br /> tional RISC, infectious viral particles suf-       trigger dsRNA imply natural viral sub- <br /> fice rapidly kill adult animal (Fig. 1; Table 1), argue   strate Dicer-2 consists dsRNA replication <br /> for central role RNAi antiviral response. The         intermediate stem loops longer 30 nt. Additional <br /> antiviral mechanism depends combined action            insight comes study small RNA profile <br /> Dicer-2 Ago-2. Dicer-2 senses cleaves dsRNA,             developing flies (Aravin et al. 2003). The Dro- <br /> thus generating siRNA recognition signal guides          sophila stocks used study persistently in- <br /> RISC cleavage viral RNA Ago-2. In turn, DCV            fected DCV, 17 small RNAs derived from <br /> encodes counterdefense mechanism consists            DCV cloned. Based size profile pre- <br /> canonical dsRBD suppresses RNA silencing initi-             dominant detection small RNAs genomic <br /> ated long dsRNA, inhibition Dicer activity. Be-         viral RNA, authors argued small RNAs <br /> cause siRNAs sequence-specific guides cleavage,          true Dicer cleavage products (Aravin et al. <br /> this antiviral mechanism represent inverte-            2003). A detailed systemic analysis siRNAs <br /> brate homolog sequence-specific &#226;&#8364;&#339;adaptive&#226;&#8364;? im-              DCV infection needed clearly define the <br /> mune response, respond ongoing            Dicer-2 substrate virus-infected flies. <br /> infection pathogen-specific manner. It remains           Another important question determine how <br /> tested specific signals persist organism          viruses escape inhibitory effect RNAi. One mecha- <br /> for prolonged periods time, providing akin          nism, identified study, relies suppressor with <br /> to &#226;&#8364;&#339;immunological memory,&#226;&#8364;?                 dsRNA-binding activity inhibits Dicer processing of <br /> mechanisms spread &#226;&#8364;&#339;immune&#226;&#8364;? antiviral activ-           dsRNA. This mechanism observed RNA <br /> ity organism.                                     silencing suppressors plant viruses (Vargason et al. <br />    RNA silencing instrumental local sys-          2003; Ye et al. 2003; Lakatos et al. 2006; Merai et al. <br /> temic antiviral response plants. The RNAi pathway             2006), B2 protein insect virus Flock <br /> has diversified extensively evolution plants.          house virus (Chao et al. 2005; Lingel et al. 2005; Lu et al. <br /> For example, Arabidopsis encodes Dicer-like genes           2005; Sullivan Ganem 2005). Of note, DCV does not <br /> and 10 Argonaute genes. This diversification allow           provide general inhibition Dicer-2 Ar- <br /> plants efficiently process different dsRNA substrates,        gonaute-2 activity, displays substrate-specific <br /> and silence gene expression different effector mecha-        inhibition RNAi. While RNAi initiated long <br /> nisms (Baulcombe 2004; Xie et al. 2004). Indeed,           dsRNA inhibited, short siRNA-mediated RNAi not <br /> seems differential requirement Dicer-like        affected. From evolutionary perspective, viruses adapt <br /> genes, DCL2 DCL4, local antiviral silencing          level pathogenicity hosts ensure that <br /> spread silencing signal, respectively (Deleris et al.     transmit viral progeny. The presence an <br /> 2006). However, effector mechanism antiviral             RNAi suppressor allow virus fine-tune the <br /> RNA silencing plants remains undefined. Of 10             severity infection, may, example, cru- <br /> Argonaute family members Arabidopsis, Argo-              cial establish persistent infection natural infec- <br /> naute 1 far demonstrated contain Slicer         tions. <br /> activity, does appear associate viral            We conclude inhibition RNAi dsRNA bind- <br /> siRNA (Baumberger Baulcombe 2005). Thus              ing extends realm plant viruses, also <br /> stage unclear antiviral defense           occurs insect viruses. Of note, CrPV, closely <br /> plants dependent Slicer-dependent RNAi cleavage          related DCV, encodes RNAi suppressor maps <br /> activity.                                                        genomic region DCV (Wang et al. 2006). <br />    Drosophila encodes Dicer genes Argo-              However, DCV depends canonical dsRBD <br /> naute genes, dedicated miRNA pathway               RNAi suppression, domain pre- <br /> (Dicer-1 Ago-1) RNAi cleavage pathway                 dicted CrPV (and members bicistrovirus <br /> (Dicer-2 Ago-2) (Lee et al. 2004; Okamura et al.             family) (R. van Rij R. Andino, unpubl.). In agreement <br /> 2004). Our study, recent reports (Galiana-         this, CrPV did inhibit dsRNA processing S2 <br /> Arnoux et al. 2006; Wang et al. 2006), establish        cells, suggesting different mechanism action the <br /> antiviral function RNAi conserved evolu-           CrPV RNAi suppressor. This suggests independent, con- <br /> tion, antiviral immunity Drosophila de-           vergent evolution RNAi suppressor function in <br /> pendent Slicer/Ago-2-mediated effector mechanism.           closely related viruses. <br /> While reduced genetic complexity makes Drosophila <br /> an attractive model study antiviral func- <br /> tion RNAi, questions remain. For instance,         Materials methods <br /> unclear types RNA structures recognized by <br /> Dicer-2 initial cleavage          Cells, plasmids, viruses <br /> mechanism feature allowing RNAi machinery to <br />                                                                  Drosophila S2 cells (Invitrogen) cultured 25&#194;&#176;C in <br /> selectively target viral species endogenous              Schneider&#226;&#8364;&#8482;s Drosophila medium (GIBCO), supplemented with <br /> dsRNA. A number observations provide insights            10% heat-inactivated fetal calf serum, 2 mM L-glutamine, 100 <br /> what physiological substrates Dicer-2 be.            U/mL penicillin, 100 mg/mL streptomycin. Firefly (Photi- <br /> Our observation suppression RNAi mediated             nus pyralis) Renilla reniformis luciferase sequences from <br /> DCV increased efficiency increasing length            plasmids pGL3 pRL-CMV (Promega) cloned into <br />  <br />  <br /> 2992        GENES &amp; DEVELOPMENT <br />                                                                                                                 Antiviral function Ago2 <br />  <br />  <br /> pMT/V5-HisB (Invitrogen), generating pMT-Luc pMT-Ren,            Fly infections vivo RNAi <br /> allowing copper-inducible expression metallothein pro- <br />                                                                      Flies reared standard medium 25&#194;&#176;C. Ago-2414 and <br /> moter. cDNA fragments spanning DCV dsRBD cloned <br />                                                                      Dcr-2L811fsX flies described previously (Lee et al. 2004; <br /> under control Actin promoter pAc5/V5-HisB. Mu- <br />                                                                      Okamura et al. 2004); w1118 flies used wild-type con- <br /> tations constructs introduced using Quick- <br />                                                                      trols. UAS-B2 flies described (B. <br /> Change site-directed mutagenesis kit (Stratagene). As sensor <br />                                                                      Berry C. Antoniewski, prep.). Briefly, FHV B2 ORF and <br /> for miRNA biogenesis function, copies sequence <br />                                                                      wild-type KK73AA mutant DCV-1A PCR-amplified <br /> that perfectly complementary mature miR2b the <br />                                                                      cloned pENTR-D-TOPO vector using pENTR <br /> sense antisense orientation cloned 3&#226;&#172;&#732;UTR the <br />                                                                      directional TOPO Cloning Kits (Invitrogen), subsequently <br /> luciferase gene pMT-Luc, described (Saleh et al. 2006). <br />                                                                      transferred pUASP vector (Rorth 1998) adapted the <br />    Transfections performed using Effectene transfection re- <br />                                                                      Gateway (details provided request). Transgenic <br /> agent (Qiagen) according manufacturer&#226;&#8364;&#8482;s recommenda- <br />                                                                      stocks established using standard injection procedure. B2 <br /> tions. Luciferase expression assayed using Dual-Lucif- <br />                                                                      expression induced crossing UAS-B2 flies Daugh- <br /> erase Reporter Assay System (Promega) analyzed Tecan <br />                                                                      terless Gal4 flies, F1 progeny challenged with <br /> Ultra-evolution plate reader. DsRNA generated vitro <br />                                                                      DCV. Two- three-day-old female flies injected 50 <br /> transcription T7-promoter-flanked PCR products. Syn- <br />                                                                      nL appropriate virus dilution 10 mM Tris-Cl (pH 7.5) as <br /> thetic siRNAs purchased Dharmacon. <br />                                                                      described previously (Cherry Perrimon 2004), using a <br />    The DCV dsRBD (DCV position 1&#226;&#8364;&#8220;99) cloned into <br />                                                                      Drummond nanoject injector. Fly mortality day 1 attrib- <br /> pGex4T.1 (GE Healthcare) expression glutathione S- <br />                                                                      uted damage invoked injection procedure, these <br /> transferase fusion protein E. coli BL21(DE3) strain. The <br />                                                                      flies excluded analyses. Mortality moni- <br /> fusion protein purified using glutathione agarose resin <br />                                                                      tored daily 14 d, 3&#226;&#8364;&#8220;4 d flies transferred to <br /> (GST-bind, Novagen) according recommendations the <br />                                                                      fresh food. In experiments, 40&#226;&#8364;&#8220;60 flies genotypic group <br /> manufacturer, dialyzed overnight 20 mM Tris-HCl, <br />                                                                      injected. Unless noted otherwise, female flies used. <br /> 0.5 mM EDTA, 5 mM MgCl2, 1 mM DTT, 140 mM NaCl, 2.7 <br />                                                                      No significant difference survival observed between <br /> mM KCl, 30% glycerol (pH 7.9). <br />                                                                      Ago-2 wild-type flies injection buffer (data not <br />    DCV CrPV stocks prepared low-passage S2 cells <br />                                                                      shown). For Northern blots, RNA isolated 25 flies <br /> and titered end-point dilution. Briefly, 25,000 S2 cells per <br />                                                                      using Trizol reagent. Viral titers flies determined by <br /> well 96-well plate inoculated 10-fold dilutions of <br />                                                                      end-point dilution fly homogenate pools flies. <br /> viral stocks. Cells transferred fresh medium day 7, <br />                                                                      At indicated time points, flies harvested stored at <br /> and CPE monitored visually 14 d. Viral titers were <br />                                                                      &#226;&#710;&#8217;70&#194;&#176;C processing. We confirmed absence of <br /> calculated according method Reed Muench (1938). <br />                                                                      endogenous virus fly stocks titration uninfected fly <br />                                                                      homogenate S2 cells (data shown). For vivo RNAi <br />                                                                      experiments, established UAS-WT DCV-1A UAS- <br /> RNAi S2 cells <br />                                                                      KK73AA DCV-1A transgenic lines DCV suppres- <br /> The ability DCV suppress RNAi analyzed silencing        sor variant form expressed control the <br /> luciferase expression presence absence virus infec-     GAL4-inducible UAS promoter (Rorth 1998). A P{GMR-wIR} <br /> tion. Twenty-four hours prior transfection, S2 cells in-     transgenic stock (Lee et al. 2004) recombined genetic <br /> fected DCV multiplicity infection (MOI) 1. These     crosses w1118, P{w+, GMR-GAL4} line obtain the <br /> cells cotransfected pMT-GL3 pMT-Ren,          w1118, P{GMR-wIR}; P{w+,GMR-GAL4} homozygous RNAi sen- <br /> with dsRNA targeting Renilla luciferase synthetic            sor line. Homozygous P{w+, pUAS-Wt-DCV}, P{w+, pUAS- <br /> siRNA targeting firefly luciferase. Twenty-four hours          KK73AA-DCV}, P{w+, pUAS-GFP} crossed sensor <br /> transfection, expression luciferase induced adding         line, adult progeny scored 2 d hatching. <br /> CuSO4 culture supernatant, cell lysates gener- <br /> ated additional 18 h incubation. MiRNA function         Gel mobility shift assays <br /> tested cotransfection S2 cells pMT-Ren        Uniformly labeled 116- 211-bp dsRNA probes gener- <br /> the miRNA sensor pMT-Gl3-miR2b sense antisense. The               ated vitro transcription sense antisense sequence in <br /> experiment performed described above. The abil-       presence &#226;?&#163;-32P-UTP T7 RNA polymerase. The probes <br /> ity DCV dsRBD suppress RNAi analyzed co-            purified 6% denaturing acrylamide gel, dis- <br /> transfection pAc-dsRBD pMT-luc pMT-Ren. At day           solved annealed HMK buffer (200 mM Hepes pH 7.9, <br /> 2, RNAi induced adding dsRNA targeting firefly lucif-         600 mM KCl, 100 mM MgCl2). Synthetic 21-bp siRNA 31-, <br /> erase culture supernatant. Luciferase expression in-      40-, 51-, 116-bp dsRNA, end-labeled &#226;?&#165;-32P using <br /> duced 24 h feeding dsRNA, cell lysates pre-        polynucleotide kinase purified G25 microspin column <br /> pared additional 18 h incubation.                           (GE Healthcare). Probe (1.5 nM) incubated varying <br />                                                                      protein concentrations presence 31.25 mM DTT, 0.625 <br />                                                                      mg/mL yeast tRNA binding buffer (5 mM Hepes pH 8.0, 25 <br /> Northern blots                                                       mM KCl, 2 mM MgCl2, 3.8% glycerol) 30 min room <br /> MiRNAs detected using Northern blot previously de-           temperature. Gel shifts 21-bp siRNA 31- 116-bp <br /> scribed (Aravin et al. 2003). The blot hybridized 32P-      dsRNA assayed 8% native polyacrylamide gels; gel <br /> labeled oligonucleotide antisense mature miR-2b. To control       shifts 200-bp dsRNA assayed 6% native gels. Gels <br /> for equal loading, blot stripped hybridized probe   run 4&#194;&#176;C. Radioactivity mobility shifts quantified <br /> complementary U6 snRNA (5&#226;&#172;&#732;-TGGAACGCTTCAC                          Typhoon 9400 Imager (Amersham Biosciences). <br /> GATTTTG-3&#226;&#172;&#732;). Viral RNA detected Northern blot using <br />                                                                      Analysis Dicer activity <br /> standard procedures random primed DNA probe corre- <br /> sponding nucleotides 1947&#226;&#8364;&#8220;2528 DCV (accession no. NP           Dicer activity determined cell extracts uninfected <br /> 044945).                                                             DCV- CrPV-infected S2 cells. Cells infected 20 h <br />  <br />  <br />                                                                                                    GENES &amp; DEVELOPMENT              2993 <br />  van Rij et al. <br />  <br />  <br /> before preparation cell extracts, MOI 1. Cells           pathogenesis. Nat. Immunol. 5: 81&#226;&#8364;&#8220;87. <br /> washed PBS, resuspended lysis buffer (15 mM Hepes/            Deleris, A., Gallego-Bartolome, J., Bao, J., Kasschau, K.D., Car- <br /> KOH pH 7.4, 50 mM KOAc, 1 mM Mg[OAc]2, protease in-                    rington, J.C., Voinnet, O. 2006. Hierarchical action and <br /> hibitors), frozen &#226;&#710;&#8217;80&#194;&#176;C. Prior use, extracts               inhibition plant Dicer-like proteins antiviral defense. <br /> thawed ice centrifuged 14,000g 20 min. The re-              Science 313: 68&#226;&#8364;&#8220;71. <br /> sulting supernatant tested Dicer activity 3 h 25&#194;&#176;C,    Galiana-Arnoux, D., Dostert, C., Schneemann, A., Hoffmann, <br /> as previously described (Haley et al. 2003), using gel-purified           J.A., Imler, J.L. 2006. Essential function vivo for <br /> uniformly labeled 116-bp dsRNA. The reaction products                Dicer-2 host defense RNA viruses Drosophila. <br /> deproteinized proteinase K phenol extraction ana-              Nat. Immunol. 7: 590&#226;&#8364;&#8220;597. <br /> lyzed 12% denaturing polyacrylamide gel.                         Gitlin, L. Andino, R. 2003. Nucleic acid-based immune <br />                                                                           system: The antiviral potential mammalian RNA silenc- <br />                                                                           ing. J. Virol. 77: 7159&#226;&#8364;&#8220;7165. <br />                                                                       Gitlin, L., Karelsky, S., Andino, R. 2002. Short interfering <br /> Acknowledgments <br />                                                                           RNA confers intracellular antiviral immunity human <br /> We thank members Andino Pat O&#226;&#8364;&#8482;Farrell laboratory               cells. Nature 418: 430&#226;&#8364;&#8220;434. <br /> for discussions, technical support, advice fly work. We        Gitlin, L., Stone, J.K., Andino, R. 2005. Poliovirus escape <br /> thank O. Voinnet J.L. Imler helpful discussions. We               RNAi: siRNA-target recognition implications for <br /> thank Judith Frydman Annemieke Jansens critical read-             therapeutic approaches. J. Virol. 79: 1027&#226;&#8364;&#8220;1035. <br /> ing manuscript. Peter Sarnow Sarah Cherry kindly pro-         Haley, B., Tang, G., Zamore, P.D. 2003. In vitro analysis of <br /> vided DCV CrPV virus stocks. We grateful C. Vaury              RNA interference Drosophila melanogaster. Methods 30: <br /> for gift B2 DNA, D. Fagegaltier, L. R&#195;&#182;der, E.           330&#226;&#8364;&#8220;336. <br /> Georgenthum, BestGene, Inc., generating transgenic        John, B., Enright, A.J., Aravin, A., Tuschl, T., Sander, C., and <br /> flies. Armin Hekele prepared dsRNA used               Marks, D.S. 2004. Human MicroRNA targets. PLoS Biol. 2: <br /> study. We thank Richard Carthew providing GMR-wIR                 e363. <br /> fly stock, Phil Zamore providing fly RNAi mutants         Keene, K.M., Foy, B.D., Sanchez-Vargas, I., Beaty, B.J., Blair, <br /> experimental advice Dicer assays S2 cell extracts. B.B.         C.D., Olson, K.E. 2004. RNA interference acts natu- <br /> is Lebanese CNRSL fellow. This work financially sup-                ral antiviral response O&#226;&#8364;&#8482;nyong-nyong virus (Alphavirus; <br /> ported Pasteur Institute CNRS, grant              Togaviridae) infection Anopheles gambiae. Proc. Natl. <br /> from ARC (#3203) C.A. NIH grants AI40085                   Acad. Sci. 101: 17240&#226;&#8364;&#8220;17245. <br /> AI064738 R.A.                                                      Lakatos, L., Csorba, T., Pantaleo, V., Chapman, E.J., Carrington, <br />                                                                           J.C., Liu, Y.P., Dolja, V.V., Calvino, L.F., Lopez-Moya, J.J., <br />                                                                           Burgyan, J. 2006. Small RNA binding common strat- <br />                                                                           egy suppress RNA silencing viral suppressors. <br /> References <br />                                                                           EMBO J. 25: 2768&#226;&#8364;&#8220;2780. <br /> Aravin, A.A., Lagos-Quintana, M., Yalcin, A., Zavolan, M.,            Lee, Y.S., Nakahara, K., Pham, J.W., Kim, K., He, Z., Sonthei- <br />    Marks, D., Snyder, B., Gaasterland, T., Meyer, J., Tuschl,         mer, E.J., Carthew, R.W. 2004. Distinct roles Dro- <br />    T. 2003. The small RNA profile Drosophila melano-               sophila Dicer-1 Dicer-2 siRNA/miRNA silencing <br />    gaster development. Dev. Cell 5: 337&#226;&#8364;&#8220;350.                              pathways. Cell 117: 69&#226;&#8364;&#8220;81. <br /> Bartel, D.P. 2004. MicroRNAs: Genomics, biogenesis, mecha-            Li, H.W., Li, W.X., Ding, S.W. 2002. Induction suppres- <br />    nism, function. Cell 116: 281&#226;&#8364;&#8220;297.                                 sion RNA silencing animal virus. Science 296: <br /> Baulcombe, D. 2004. RNA silencing plants. Nature 431: 356&#226;&#8364;&#8220;             1319&#226;&#8364;&#8220;1321. <br />    363.                                                               Lingel, A., Simon, B., Izaurralde, E., Sattler, M. 2005. The <br /> Baumberger, N. Baulcombe, D.C. 2005. Arabidopsis                      structure flock house virus B2 protein, viral suppres- <br />    ARGONAUTE1 RNA Slicer selectively recruits                  sor RNA interference, shows novel mode double- <br />    microRNAs short interfering RNAs. Proc. Natl. Acad.                stranded RNA recognition. EMBO Rep. 6: 1149&#226;&#8364;&#8220;1155. <br />    Sci. 102: 11928&#226;&#8364;&#8220;11933.                                             Liu, J., Carmell, M.A., Rivas, F.V., Marsden, C.G., Thomson, <br /> Bentwich, I., Avniel, A., Karov, Y., Aharonov, R., Gilad, S.,             J.M., Song, J.J., Hammond, S.M., Joshua-Tor, L., Hannon, <br />    Barad, O., Barzilai, A., Einat, P., Einav, U., Meiri, E., et al.       G.J. 2004. Argonaute2 catalytic engine mammalian <br />    2005. Identification hundreds conserved noncon-              RNAi. Science 305: 1437&#226;&#8364;&#8220;1441. <br />    served human microRNAs. Nat. Genet. 37: 766&#226;&#8364;&#8220;770.                   Lu, R., Maduro, M., Li, F., Li, H.W., Broitman-Maduro, G., Li, <br /> Berezikov, E., Guryev, V., van Belt, J., Wienholds, E., Plas-          W.X., Ding, S.W. 2005. Animal virus replication and <br />    terk, R.H., Cuppen, E. 2005. Phylogenetic shadowing                RNAi-mediated antiviral silencing Caenorhabditis ele- <br />    computational identification human microRNA                     gans. Nature 436: 1040&#226;&#8364;&#8220;1043. <br />    genes. Cell 120: 21&#226;&#8364;&#8220;24.                                            Meister, G. Tuschl, T. 2004. Mechanisms gene silencing <br /> Brennan, C.A. Anderson, K.V. 2004. Drosophila: The genet-             double-stranded RNA. Nature 431: 343&#226;&#8364;&#8220;349. <br />    ics innate immune recognition response. Annu. Rev.          Merai, Z., Kerenyi, Z., Kertesz, S., Magna, M., Lakatos, L., and <br />    Immunol. 22: 457&#226;&#8364;&#8220;483.                                                  Silhavy, D. 2006. Double-stranded RNA binding a <br /> Chang, K.Y. Ramos, A. 2005. The double-stranded RNA-                  general plant RNA viral strategy suppress RNA silencing. <br />    binding motif, versatile macromolecular docking platform.            J. Virol. 80: 5747&#226;&#8364;&#8220;5756. <br />    FEBS J. 272: 2109&#226;&#8364;&#8220;2117.                                            Okamura, K., Ishizuka, A., Siomi, H., Siomi, M.C. 2004. <br /> Chao, J.A., Lee, J.H., Chapados, B.R., Debler, E.W.,                      Distinct roles Argonaute proteins small RNA-directed <br />    Schneemann, A., Williamson, J.R. 2005. Dual modes               RNA cleavage pathways. Genes &amp; Dev. 18: 1655&#226;&#8364;&#8220;1666. <br />    RNA-silencing suppression Flock House virus protein B2.         Reed, L.J. Muench, H. 1938. A simple method estimating <br />    Nat. Struct. Mol. Biol. 12: 952&#226;&#8364;&#8220;957.                                   fifty percent endpoints. Am J. Hyg. 27: 493&#226;&#8364;&#8220;497. <br /> Cherry, S. Perrimon, N. 2004. Entry rate-limiting step       Rorth, P. 1998. Gal4 Drosophila female germline. Mech. <br />    viral infection Drosophila melanogaster model              Dev. 78: 113&#226;&#8364;&#8220;118. <br />  <br />  <br /> 2994        GENES &amp; DEVELOPMENT <br />                                                                               Antiviral function Ago2 <br />  <br />  <br /> Ryter, J.M. Schultz, S.C. 1998. Molecular basis double- <br />    stranded RNA&#226;&#8364;&#8220;protein interactions: Structure dsRNA- <br />    binding domain complexed dsRNA. EMBO J. 17: 7505&#226;&#8364;&#8220; <br />    7513. <br /> Saleh, M.C., van Rij, R.P., Hekele, A., Gillis, A., Foley, E., <br />    O&#226;&#8364;&#8482;Farrell, P.H., Andino, R. 2006. The endocytic pathway <br />    mediates cell entry dsRNA induce RNAi silencing. <br />    Nat. Cell Biol. 8: 793&#226;&#8364;&#8220;802. <br /> Sanchez-Vargas, I., Travanty, E.A., Keene, K.M., Franz, A.W., <br />    Beaty, B.J., Blair, C.D., Olson, K.E. 2004. RNA interfer- <br />    ence, arthropod-borne viruses, mosquitoes. Virus Res. <br />    102: 65&#226;&#8364;&#8220;74. <br /> Silverstein, A.M. 1989. The history immunology. In Funda- <br />    mental immunology (ed. W.E. Paul), pp. 21&#226;&#8364;&#8220;38. Raven Press, <br />     New York. <br /> Sullivan, C. Ganem, D. 2005. A virus-encoded inhibitor <br />     blocks RNA interference mammalian cells. J. Virol. <br />     79: 7371&#226;&#8364;&#8220;7379. <br /> Tian, B., Bevilacqua, P.C., Diegelman-Parente, A., and <br />     Mathews, M.B. 2004. The double-stranded-RNA-binding <br />     motif: Interference more. Nat. Rev. Mol. Cell Biol. <br />     5: 1013&#226;&#8364;&#8220;1023. <br /> van Rij, R.P. Andino, R. 2006. The silent treatment: RNAi <br />     defense virus infection mammals. Trends Bio- <br />     technol. 24: 186&#226;&#8364;&#8220;193. <br /> Vargason, J.M., Szittya, G., Burgyan, J., Tanaka Hall, T.M. <br />     2003. Size selective recognition siRNA RNA silenc- <br />     ing suppressor. Cell 115: 799&#226;&#8364;&#8220;811. <br /> Voinnet, O. 2005. Induction suppression RNA silencing: <br />     Insights viral infections. Nat. Rev. Genet. 6: 206&#226;&#8364;&#8220;220. <br /> Wang, X.H., Aliyari, R., Li, W.X., Li, H.W., Kim, K., Carthew, R., <br />     Atkinson, P., Ding, S.W. 2006. RNA interference directs <br />     innate immunity viruses adult Drosophila. Sci- <br />     ence 312: 452&#226;&#8364;&#8220;454. <br /> Xie,  <br /> </body></html>