<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> Function <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> Shape: A Novel Role of a Formin in <br /> Morphogenesis of the Fungus Ashbya gossypii&#226;&#8211;&#161;V <br />  <br />  <br />  <br />  <br />  <br /> Morphogenesis of filamentous ascomycetes includes continuously elongating hyphae, frequently emerging lateral <br /> branches, and, under certain circumstances, symmetrically dividing hyphal tips. We identified the formin AgBni1p of the <br /> model fungus Ashbya gossypii as an essential factor in these processes. AgBni1p is an essential protein apparently lacking <br /> functional overlaps with the two additional A. gossypii formins that are nonessential. Agbni1 <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>null mutants</span> fail to develop <br /> hyphae and instead expand to potato-shaped giant cells, which lack actin cables and thus tip-directed transport of <br /> secretory vesicles. Consistent with the essential role in hyphal development, AgBni1p locates to tips, but not to septa. The <br /> presence of a diaphanous autoregulatory domain (DAD) indicates that the activation of AgBni1p depends on Rho-type <br /> GTPases. Deletion of this domain, which should render AgBni1p constitutively active, completely changes the branching <br /> <span id='am-320' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of young hyphae. New axes of polarity are no longer established subapically (lateral branching) but by symmetric <br /> divisions of hyphal tips (tip splitting). In wild-type hyphae, tip splitting is induced much later and only at much higher <br /> elongation speed. When GTP-locked Rho-type GTPases were tested, only the young hyphae with mutated AgCdc42p split <br /> at their tips, similar to the DAD deletion mutant. Two-hybrid experiments confirmed that AgBni1p interacts with <br /> GTP-bound AgCdc42p. These data suggest a pathway for transforming one axis into two new axes of polar growth, in <br /> which an increased activation of AgBni1p by a pulse of activated AgCdc42p stimulates additional actin cable formation <br /> and tip-directed vesicle transport, thus enlarging and ultimately splitting the polarity site. <br />  <br />  <br /> INTRODUCTION                                                           genome analysis using gene targeting methods or autono- <br />                                                                        mously replicating plasmids (Wright and Philippsen, 1991; <br /> Elongated cells, such as neurites, pollen tubes, and root hair         Steiner and Philippsen, 1994; Steiner et al., 1995; Wendland et <br /> cells, are generated when polar growth is maintained for               al., 2000), which have promoted functional analyses of po- <br /> extended time periods. An extreme case of polar growth has             larity genes in this fungus. Polar growth in A. gossypii starts <br /> evolved in filamentous fungi, which are able to extend the             from an isotropically growing germ bubble. The first steps in <br /> tips of their tubelike cells, called hyphae, for unlimited time,       polarity establishment involve the A. gossypii proteins <br /> provided nutrients are available (Gow, 1995; Momany, 2002;             AgCdc24p and AgCdc42p (Wendland and Philippsen, 2001). <br /> Harris et al., 2005). Hyphae not only very efficiently elongate        Once the first germ tube has emerged hyphal growth of A. <br /> but regularly establish new axes of polarity along their               gossypii proceeds with frequent lateral branching and a <br /> cortex, thus forming lateral branches, which themselves                steadily increasing elongation speed from initially 5 &#226;&#65533;&#174;m/h <br /> again generate lateral branches. This results in a fast spread-        up to a maximum of 170 &#226;&#65533;&#174;m/h (Knechtle et al., 2003). For <br /> ing network of hyphae and the typical appearance of a                  this process of hyphal maturation AgBem2p, AgRho3p, <br /> fungal mycelium. In most filamentous fungi the initial hy-             AgCla4p, AgSpa2p, and AgRsr1p are important (Ayad-Dur- <br /> phal tip elongation speed can increase by an order of mag-             ieux et al., 2000; Wendland and Philippsen, 2000, 2001; <br /> nitude or even more. Some filamentous fungi display hyphal             Knechtle et al., 2003; Bauer et al., 2004). Although AgBem2p, <br /> tip splitting, the unique ability to simultaneously generate at        AgRho3p and AgRsr1p are responsible for maintenance of <br /> tips of fast growing hyphae two sister hyphae.                         polarity, both AgCla4p and AgSpa2p are necessary to reach <br />    The ascomycete Ashbya gossypii shows all the hallmarks of           maximal growth speed. <br /> fungal filamentous growth, including tip splitting, although              An important late step in the development to a fast <br /> its recently completed genome sequence reveals an evolu-               spreading A. gossypii mycelium is the splitting of hyphal tips <br /> tionary relation with the Saccharomyces cerevisiae genome              which, under optimal growth conditions, begins 12&#226;&#8364;&#8220;14 h <br /> (Dietrich et al., 2004). A. gossypii is amenable to functional         after emergence of the first hypha (Ayad-Durieux et al., <br />                                                                        2000). A study with a GFP-labeled AgSpa2p showed that <br />                                                                        during tip splitting the existing polarity control center di- <br /> This article was published online ahead of print in MBC in Press       vides into two new centers of polarity, yielding two hyphae <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E05&#226;&#8364;&#8220; 06 &#226;&#8364;&#8220; 0479)        that elongate, after a short lag phase, with a speed similar to <br /> on October 19, 2005. <br />                                                                        that before tip splitting (Knechtle et al., 2003). So far, the <br /> &#226;&#8211;&#161;  The online version of this article contains supplemental material <br /> V <br />                                                                        molecular basis for the initiation of hyphal tip splitting is <br /> at MBC Online (http://www.molbiolcell.org).                            unknown. We assumed that the apparent duplication of <br /> Address correspondence to: Peter Philippsen (peter.philippsen@         polar growth capacity depends on an approximately two- <br /> unibas.ch).                                                            fold increase in secretory vesicle transport at or shortly after <br />  <br /> 130                                                                                    &#194;&#169; 2005 by The American Society for Cell Biology <br />                                                                                                      Formin for Tip Growth and Tip Splitting <br />  <br />  <br /> tip splitting and that, before this increase, additional tip-         MATERIALS AND METHODS <br /> located actin cables had to form. Given the conserved role of <br />                                                                       A. gossypii Strains and Growth Conditions <br /> formins in nucleating actin cables (Pruyne et al., 2002; Sagot <br />                                                                       All strains were constructed by PCR-based gene targeting according to the <br /> et al., 2002b), we therefore hypothesized that a formin ho-           method described by Wendland et al. (2000). Either pGEN3 (Wendland et al., <br /> molog could play an important role for the regulation of tip          2000), pGUG (Knechtle et al., 2003), or pUC19NATPS (D. Hoepfner, personal <br /> splitting.                                                            communication) were used as templates to generate gene-targeting cassettes <br />                                                                       encoding geneticin-resistance, GFP plus geneticin-resistance, and ClonNAT- <br />    Formins are common to all eukaryotic species and partic-           resistance, respectively. Strains and names of the oligonucleotides and tem- <br /> ipate in many different processes, from cell polarization to          plates used are given in Table 1. The strains were verified by PCR using a PTC <br /> embryonic development (see Wallar and Alberts (2003) and              100 thermocycler (MJ Research, Waltham, MA). All oligonucleotides are listed <br /> Evangelista et al. (2003) for reviews). Except for some cases         in Table 2. Strains were grown at 30&#194;&#176;C in AFM (Ashbya Full Medium) with or <br />                                                                       without geneticin or ClonNAT or at room temperature on synthetic medium <br /> in higher cells where a formin is involved in signaling (Ha-          for live cell imaging (Knechtle et al., 2003). <br /> bas et al., 2001), most formins participate in the organization <br /> of the actin cytoskeleton. Recently, the ability of formins to        DNA Manipulations and Sequencing <br /> nucleate actin at the barbed end of actin filaments was               All DNA manipulations were carried out according to Sambrook et al. (2001) <br /> described for different organisms (Pruyne et al., 2002; Sagot         with DH5&#226;&#65533;&#163;F&#226;&#172;&#732; as host (Hanahan, 1983). Sequencing was done using an ABI <br />                                                                       prism 377 DNA sequencer (PE Applied Biosystems, Foster City, CA) accord- <br /> et al., 2002b; Kobielak et al., 2003; Kovar et al., 2003; Li and      ing to the manufacturer&#226;&#8364;&#8482;s instructions. Plasmids were isolated from yeast <br /> Higgs, 2003). Actin nucleation is mediated by the conserved           using the High Pure Plasmid Purification Kit (Roche Diagnostics, Mannheim, <br /> formin homology domain FH2. As found by analyzing the                 Germany) according to the instruction manual for plasmid preparation from <br />                                                                       Escherichia coli but with the following modifications: 5 ml of a yeast culture <br /> crystal structures of the mouse formin, mDIA, and the yeast           was collected by centrifugation and the supernatant was discarded. The cells <br /> formin, ScBni1p, the core of the FH2 domain seems to have             were resuspended in solution 1 and 0.2 g of 0.5-mm glass beads were added. <br /> actin binding capacity, whereas adjacent amino acids are              Cells were lysed by vortexing for 8 min at 4&#194;&#176;C. From here on the instructions <br />                                                                       of the manufacturer were followed, only the elution volume was decreased to <br /> necessary for oligomerization and gain of polymerization              20 &#226;&#65533;&#174;l. Ten microliters of the elute were transformed into E. coli for plasmid <br /> capability (Shimada et al., 2004; Xu et al., 2004). A subclass of     amplification. <br /> formins, the so-called diaphanous related formins (DRFs), <br /> are defined by two properties: First, they are activated when         Plasmids and Constructs <br /> a GTP-bound Rho-type protein interacts with their amino               All plasmids are listed in Table 3. All constructs carrying genes for Rho-type <br />                                                                       GTPases were constructed using the same scheme. Each gene was amplified <br /> terminus (Kohno et al., 1996; Evangelista et al., 1997;               from A. gossypii genomic DNA using the corresponding primers from Table 2 <br /> Imamura et al., 1997; Watanabe et al., 1997; Habas et al.,            for PCR. All primers were designed to amplify the open reading frame (ORF) <br /> 2001). Second, they posses a carboxy-terminal diaphanous              from the start codon to the end, excluding the nucleotides coding for the <br />                                                                       CAAX motif at the carboxy terminus, thus avoiding lipid modification (prim- <br /> autoregulatory domain (DAD), which binds to the amino-                ers AgRHOx-ATG, AgRHOx-TAG and AgCDC42-ATG, and AgCDC42- <br /> terminus in the inactive state of these formins (Watanabe et          TAG). To facilitate cloning, EcoRI and BamHI restriction sites were added to <br /> al., 1999; Alberts 2001).                                             the oligonucleotides. The fragments were purified, cut, and ligated into <br />    In fungi, six formins have been studied to date. In S.             pUC19 (Vieira and Messing, 1991). Activated alleles of all Rho-GTPases were <br />                                                                       constructed using the method by Boles and Miosga (1995) with primers <br /> cerevisiae, the two formins Bni1p and Bnr1p are required for          named AgRHOx and AgCDC42 plus the corresponding nucleotide exchange. <br /> cell polarity and cytokinesis with some overlapping and               DNA of wild-type and mutant alleles was cloned into pGBT9 (Bartel et al., <br /> some different functions (Zahner et al., 1996; Evangelista et         1993) using again EcoRI and BamHI. The resulting plasmids were verified by <br />                                                                       sequencing. AgBNI1 was amplified by PCR from genomic A. gossypii DNA <br /> al., 1997; Imamura et al., 1997; Kamei et al., 1998; Pruyne et al.,   using primers AgBNI1-ATG2Hy and AgBNI1-TAG2Hy, which add on both <br /> 2004). In Schizosaccharomyces pombe, three formins exist              sides of the amplified AgBNI1 45 <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span> pairs homologous to pGBT9 for in vivo <br /> which are also involved in cell polarity and cytokinesis. The         cloning. This PCR product was cotransformed with EcoRI- and BamHI-di- <br />                                                                       gested pGBT9 into the yeast strain DHD5. Transformants were restreaked on <br /> protein SpCdc12p is involved in cytokinesis (Chang et al.,            selective medium, and recombinant plasmids were isolated from the yeast <br /> 1997; Kovar et al., 2003), SpFus1p in cell fusion (Petersen et        cells and amplified in E. coli. This resulted in pGBT9AgBNI1. From this, a <br /> al., 1998b) and SpFor3p in cell polarity control via regulation       2.5-kb EcoRI-BglII fragment was cut out and ligated into the EcoRI-BamHI sites <br />                                                                       of pGAD424 (Bartel et al., 1993). Again the plasmid was first verified by <br /> of the actin and microtubule network (Feierbach and Chang,            digestion and sequencing. <br /> 2001; Nakano et al., 2002). The only formin family member                The plasmid carrying AgBNI1 (pAMK1) was constructed by ligation of an <br /> described so far in a filamentous fungus is the SepA protein          HindIII/SmaI fragment generated by PCR using primers AgBNI1-P_HindIII <br /> of Aspergillus nidulans. SepA is an essential protein that            and AgBNI1-T_SmaI from genomic A. gossypii DNA. GFP was fused to the <br />                                                                       gene by in vivo recombination in S. cerevisiae strain DHD5 with a PCR <br /> locates to growing hyphal tips and to sites of cytokinesis            fragment derived from pGUG with primers AgBNI1-GS1 and AgBNI1-GS2. <br /> (septation). Some SepA mutants still grow in a filamentous            The fusion was verified by sequencing. <br /> manner although they can no longer form septa (Harris et al.,            An amino-terminal fusion of AgSEC4 to GFP was constructed by a sequen- <br />                                                                       tial series of PCR amplification and cloning steps. First, the AgSEC4 promoter <br /> 1997; Sharpless and Harris, 2002).                                    was amplified by PCR out of the genome using primers Sec4P-Hind and <br />    In this article we first document by videomicroscopy the           Sec4P-Bam and cloned into YCPlac111 using HindIII and BamHI. The result- <br /> distinct differences in polar growth of budding yeast and A.          ing plasmid was cut with BamHI and EcoRI to allow fusion of the SEC4 <br />                                                                       promoter to GFP that was amplified by PCR from pGUG with primers <br /> gossypii and compare the domain compositions of the three             GFP-Eco and GFP-Bam. In a final step the AgSEC4 ORF and terminator were <br /> A. gossypii formins with the two S. cerevisiae formins. Then          fused to the GFP-ORF using the enzymes EcoRI and SpeI and a fragment <br /> we show that mutants lacking the AgBnr1p or AgBnr2p                   generated from genomic DNA using primers Sec4-Eco and Sec4-Spe. <br />                                                                          The S1642G allele of AgBNI1 was generated using the method described by <br /> formin develop like wild type and that mutants lacking both           Storici et al. (2001) using primers Agbni1&#226;&#8364;&#8220;12C1, Agbni1&#226;&#8364;&#8220;12C2, Agbni1&#226;&#8364;&#8220; <br /> formins are unable to grow. We next provide evidence that             12IRO1, and Agbni1&#226;&#8364;&#8220;12IRO2 and S. cerevisiae strain CEN/PK2 (http:// <br /> the AgBni1p formin is essential for hyphal emergence and              www.rz.uni-frankfurt.de/FB/fb16/mikro/euroscarf/) transformed with <br />                                                                       pAMK1. The resulting plasmid pAMK1ts12 was isolated from yeast digested <br /> elongation, that it localizes at hyphal tips, and that it is          with AscI and dephosphorylated. The linearized vector was cotransformed <br /> essential for organization of actin cables and thus tip-di-           together with a PCR product generated with primers AgBNRCFP-N and <br /> rected transport of secretory vesicles. In addition, we dem-          AgBNI1-GS2 from pGUG for in vivo recombination to integrate the URA3 <br /> onstrate that constitutively active AgBni1p leads to prema-           terminator from S. cerevisiae behind the mutagenized Agbni1 gene. The result- <br />                                                                       ing vector was isolated and verified by sequencing of the altered regions. An <br /> ture tip splitting and that this is most likely triggered by          XbaI fragment containing the altered region of AgBNI1, the URA3 terminator <br /> AgCdc42p-GTP.                                                         and the G418 resistance cassette was cloned into pUC19. The resulting plas- <br />  <br />  <br /> Vol. 17, January 2006                                                                                                                           131 <br />  H.-P. Schmitz et al. <br />  <br />  <br />  <br /> Table 1. Ashbya gossypii strains and details of construction <br />  <br /> Strain                                                 Genotype                            Oligo-nucleotides            Template                Reference <br />  <br /> ATCC10895                        Wild type                                                         &#226;&#8364;&#8221;                        &#226;&#8364;&#8221;              Ashby and <br />                                                                                                                                              Nowell (1926) <br /> &#226;&#338;&#172;l&#226;&#338;&#172;t                             Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                                                   &#226;&#8364;&#8221;                        &#226;&#8364;&#8221;              Altmann-Johl and <br />                                                                                                                                              Philippsen <br />                                                                                                                                              (1996) <br /> Agbni1&#226;&#338;&#172;2                         Agbni1&#226;&#338;&#172;::GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                           AgBNI1-S1                  pGEN3                 This study <br />                                                                                            AgBNI1-S2 <br /> AgBNI1&#226;&#338;&#172;D                         AgBNI1&#226;&#338;&#172;5308&#226;&#8364;&#8220;5757::GEN3                                   AgBNI1dD-S1                pGEN3                 This study <br />                                                                                            AgBNI1dD-S2 <br /> AgBNI1-GFP                       AgBNI1-GFP-GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                         AgBNI1-GS1                 pGUG                  This study <br />                                                                                            AgBNI1-GS2 <br /> AgSPA2-GFP                       AgSPA2-GFP-GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                               &#226;&#8364;&#8221;                           &#226;&#8364;&#8221;              Knechtle et al. <br />                                                                                                                                              (2003) <br /> Agbni1&#226;&#338;&#172; AgSPA2-GFP               Agbni1&#226;&#338;&#172;::NAT1, AgSPA2-GFP-GEN3, Agleu2&#226;&#338;&#172;                  AgBNI1-NS1                 pUC19NATPS            This study <br />                                    Agthr4&#226;&#338;&#172;                                                  AgBNI1-NS2 <br /> Agbni1S1642G                     Agbni1T4924G, C4925G-GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;               See text for details                             This study <br /> Agbni1&#226;&#338;&#172;1530G, K1620R             Agbni1A4589G, A4859G-GEN3, Agleu2&#226;&#338;&#172;                       See text for details                             This study <br />                                    Agthr4&#226;&#338;&#172; <br /> Agbnr1&#226;&#338;&#172;-GEN3                     Agbnr1&#226;&#338;&#172;::GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                           AgBNR1-S1                  pGEN3                 This study <br />                                                                                             AgBNR1-S2 <br /> Agbnr1&#226;&#338;&#172;-NAT1                     Agbnr1&#226;&#338;&#172;::NAT1, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                           AgBNR1-NS1                 pUC19NATPS            This study <br />                                                                                             AgBNR1-NS2 <br /> Agbnr2&#226;&#338;&#172;                          Agbnr2&#226;&#338;&#172;::GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                           AgBNR2-S1                  pGEN3                 This study <br />                                                                                             AgBNR2-S2 <br /> Agbnr1,2&#226;&#338;&#172;GL                      Agbnr1&#226;&#338;&#172;::GEN3, Agbnr2&#226;&#338;&#172;::LEU2, Agleu2&#226;&#338;&#172;                    AgBNR2-S1                  pScLEU2               This study <br />                                    Agthr4&#226;&#338;&#172;                                                  AgBNR2-S2 <br /> Agbnr1,2&#226;&#338;&#172;GN                      Agbnr1&#226;&#338;&#172;::GEN3, Agbnr2&#226;&#338;&#172;::NAT1                             AgBNR1-NS1                 pUC19NATPS            This study <br />                                                                                             AgBNR1-NS2 <br /> Agcdc42*                         Agcdc42G183C-GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                       See text for details                             This study <br /> AgBNI1&#226;&#338;&#172;D-GFP                     AgBNI1&#226;&#338;&#172;D-GFP-GEN3, Agleu2&#226;&#338;&#172; Agthr4&#226;&#338;&#172;                       AgBNI1dD-F1                pAGT141               This study <br />                                                                                             AgBNI1dD-F2 <br />  <br />  <br />  <br /> mid pHPS232 was cut again with XbaI and transformed into the A. gossypii         cence and phase-contrast images were colored and overlaid using Meta- <br /> &#226;&#338;&#172;l&#226;&#338;&#172;t strain. Construction of the D1530R,K1620R allele of AgBNI1 was identical    Morph&#226;&#8364;&#8482;s &#226;&#8364;&#339;overlay images&#226;&#8364;&#65533; tool. For time-lapse acquisition spores were cul- <br /> to the S1642G allele except for a sequential round of mutagenesis first using    tured on a slide with a cavity (time-lapse slide) that was filled with solid <br /> primers Agbni1&#226;&#8364;&#8220;11.1C1, Agbni1&#226;&#8364;&#8220;11.1C2, Agbni1&#226;&#8364;&#8220;11.1IRO1 and Agbni1&#226;&#8364;&#8220;                medium (Ashbya Full Medium). Spores were incubated in a humid chamber <br /> 11.1IRO2 and Agbni1&#226;&#8364;&#8220;11.2C1, Agbni1&#226;&#8364;&#8220;11.2C2, Agbni1&#226;&#8364;&#8220;11.2IRO1 and Agbni1&#226;&#8364;&#8220;           without coverslip until they reached the required developmental stage. Then <br /> 11.2IRO2 respectively.                                                           a coverslip was applied. For Supplementary Movies S1 to S7 cells or spores <br />    For construction of the Q61H allele of AgCDC42 the gene was first PCR-        were cultured on Petri plates with AFM agar, cultivated in a humid chamber, <br /> amplified from genomic A. gossypii DNA using primers cdc42genom_sense            and placed for image acquisitions under a light microscope. The acquisition <br /> and cdc42genom_antisense. The resulting product was cut with EcoRV, and          frequency varied between 1 and 0.2 min&#226;&#171;&#186;1. The time-lapse picture series were <br /> PstI was ligated into pRS415 cut with SmaI and PstI. In the resulting plasmid    exported from MetaMorph as 8-bit TIFF files converted to a QuickTime <br /> pCDC42 the codon CAG starting at position 181 was deleted by integration of      (Apple Computer, Redmond, WA) or AVI movie with Adobe Premiere 4.2 <br /> the pCORE-cassette (Storici et al., 2001) using primers AgCdc42_Q-H-sense        (Adobe Systems, San Jose, CA). <br /> and AgCdc42_Q-H-antisense. This resulted in plasmid pCDC42cas. By ho- <br /> mologous recombination in yeast with annealed primers and counterselec-          Two-hybrid Experiments <br /> tion, the pCORE-cassette was replaced by the mutagenized codon, resulting <br /> in pcdc42cons. The Gen3 resistance cassette was amplified from pGEN3 using       For two-hybrid experiments prey and bait plasmids were cotransformed into <br /> primers INT_CDC42*_S1 and INT_CDC42*_S2. The resulting PCR product               S. cerevisiae strain PJ69-4a (James et al., 1996) and selected on minimal medium <br /> was integrated behind the cdc42 gene in the vector pcdc42cons yielding           lacking leucine and tryptophan but containing a fourfold concentration (80 <br /> pcdc42kanr. The 3.1-kb BamHI/PstI fragment from pcdc42kanr carrying the          mg/l) of adenine. Transformants were grown in the same liquid medium to <br /> mutagenized cdc42 and the resistance cassette was cloned into pUC19, result-     an OD600 of &#226;&#172;&#402;1. Of these, 5 &#226;&#65533;&#174;l were spotted on plates lacking in addition to <br /> ing in pUC19cdc42cons. DNA of this vector was again cut with BamHI/PstI          leucine and tryptophan either histidine or adenine to monitor activity of the <br /> and transformed into A. gossypii. Transformants resistant against G418 were      reporter genes. <br /> sporulated and spores were separated to get homokaryotic Geneticin (G418) <br /> resistant mycelium. From this mycelium the CDC42 gene was amplified by <br /> PCR using primers AgCDC42-ATG and AgCDC42-TAG. The resulting PCR                 RESULTS <br /> product was sequenced to verify the presence of the desired <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>mutation</span>. <br />                                                                                  Filamentous Growth of A. gossypii Compared with <br /> Image Acquisition and Processing                                                 Unicellular Growth of S. cerevisiae <br /> The microscopy setup used was the same as described in Knechtle et al. (2003).   A. gossypii grows like a filamentous ascomycete (Momany, <br /> Actin staining was done according to Knechtle et al. (2003). The illumination    2002) and the main differences to yeastlike growth are <br /> time and light intensity for standard phase contrast, DIC, or fluorescence <br /> acquisitions was chosen to reach minimally 25% of the maximal intensity. For <br />                                                                                  shown in Figure 1A. Sustained polar growth results in con- <br /> multiple exposures of the same sample bleaching of the sample had to be          tinuous tip extensions of hyphae and lateral branches. Thus, <br /> taken into account. The Z-distance between two planes in stack acquisitions      growth of the hyphal surface is restricted to the tips, and <br /> was set between 0.1 and 0.5 &#226;&#65533;&#174;m. Phase contrast, DIC, and single-plane            isotropic growth phases are absent. In a yeast cell, a short <br /> fluorescence images were processed using the &#226;&#8364;&#339;scale image&#226;&#8364;&#65533; and &#226;&#8364;&#339;unsharp <br /> mask&#226;&#8364;&#65533; feature in MetaMorph (Universal Imaging, West Chester, PA). Stacks <br />                                                                                  phase of polar growth of the emerging bud is followed by <br /> were deblurred with MetaMorph&#226;&#8364;&#8482;s &#226;&#8364;&#339;remove haze,&#226;&#8364;&#65533; flattened by maximum              isotropic growth of the bud. Then a septum grows at the <br /> projection with &#226;&#8364;&#339;stack arithmetics&#226;&#8364;&#65533; and scaled as mentioned above. Fluores-      junction between the mother cell and the bud and both cells <br />  <br /> 132                                                                                                                          Molecular Biology of the Cell <br />                                                                                              Formin for Tip Growth and Tip Splitting <br />  <br />  <br />  <br /> Table 2. Oligonucleotides used in this study <br /> Name                                                              Sequencea                                             Purpose <br />  <br /> AgBNI1-S1                  CTCGACCGTAGAGTCGATAGCGAGTTCTGCTCTAGCAAAATCCCCGCTAGGGATAACAGGGTAAT                     Deletion <br /> AgBNI1-S2                  GAGTTGTTCGATGGAGATCTTGACCACCTGAAGTACAGGTTCAAGAGGCATGCAAGCTTAGATCT                     Deletion <br /> AgBNI1-G1                  CCCGACCACCGTCTTGGACC                                                                  Verification <br /> AgBNI1-G2                  GGGCATTTGCGCCGGGAGAG                                                                  Verification <br /> AgBNI1-GS1                 CGGTGAACACCCGGAGTCTCGCAAGTCAATGCTCGATGAGCACAAGGGTGCAGGCGCTGGAGCTG                     GFP-tagging <br /> AgBNI1-GS2                 GCGGCGCGGCGGCGACATCGCGCTGGTCTATCAGTTTCTTGGTGCGGCAGGGACCTGGCACGGAGC                    GFP-tagging <br /> AgBNI1-I1                  CAGATCGGGCCTGTGTTACC                                                                  Verification <br /> AgBNR2-S1                  GATAAAGAACATCGCTAGTTAATTTTGCTAATAAAGAAGAGACCTTTAAgctagggataacagggtaat                 Deletion <br /> AgBNR2-S2                  GGCCGATAAGTGTGGAATCGGATAAGTGTTGAATAGGGTAACGGAAGAGGCATGCAAGCTTAGATCT                   Deletion <br /> AgBNR2-I1                  CGCACTCCGCACCGTGGACC                                                                  Verification <br /> BNR2G1neu                  CTGTCGAGGAATTAACTTTAAG                                                                Verification <br /> BNR2G2neu                  GCCAGCGCATTTTGCACCGG                                                                  Verification <br /> AgBNR1-S2                  GCCCGAGCGCCGAAGGCGAAGGGATGCGAAAAGTATATAGATGCAAGTTGaggcatgcaagcttagatct                Deletion <br /> AgBNR1-S1                  GCGGGAGCAGTAGATTTGTGCCTGGGCCCCTCCCCCGCGAGTGCGCAATGgctagggataacagggtaat                Deletion <br /> AgBNR1-I1                  CAGGCTGTGCGGCGCGCGCG                                                                  Verification <br /> BNR1G1neu                  GAGCAGCTGGTCGCTGGGCATCTCGTGG                                                          Verification <br /> BNR1G4neu                  GCCTCGAATCACCAATCCCC                                                                  Verification <br /> AgBNR1-NS1                 GGCGGGAGCAGTAGATTTGTGCCTGGGCCCCTCCCCCGCGAGTGCGCAccagtgaattcgagctcgg                   Deletion <br /> AgBNR1-NS2                 GCCCGAGCGCCGAAGGCGAAGGGATGCGAAAAGTATATAGATGCAAGTTG tacgccaagcttgcatgcct               Deletion <br /> V2PDC1P                    GAACAAACCCAAATCTGATTGCAAGGAGAGTGAAAGAGCCTT                                            Verification <br /> V3PDC1T                    GACCAGACAAGAAGTTGCCGACAGTCTGTTGAATTGGCCTG                                             Verification <br /> AgBNI1dD-S1                TCTGGAGTACAAGCGCGCGCAGGAGTTTAACCGCAAGATCTAAGCTAGGGATAACAGGGTAAT                       Deletion <br /> AgBNI1dD-S2                GCTGGTCTATCAGTTTCTTGGTGCGGCGCTGGCGAACCTTAGGCATGCAAGCTTAGATCT                          Deletion <br /> Agbni1-12 C1               CTTGAACTACGTGGAACGCATCGTCAGCCAGAATTATCCAgagctcgttttcgacactgg                          pCORE integration <br /> Agbni1-12 C2               CCTGAAGTACAGGTTCAAGTTCCTGGAGAAAGCTGTTGAAtccttaccattaagttgatc                          pCORE integration <br /> Agbni1-12 IRO1             ATGACTTTCTTGAACTACGTGGAACGCATCGTCAGCCAGAATTATCCAGGGTTCAACAGC                          Mutagenesis <br /> Agbni1-12 IRO2             TTGACCACCTGAAGTACAGGTTCAAGTTCCTGGAGAAAGCTGTTGAACCCTGGATAATTC                          Mutagenesis <br /> Agbni1-11.1 C1             TAAGTCTCCGGAAAAGGATCCAAGCGAGCTTCAGCGATCAgagctcgttttcgacactgg                          pCORE integration <br /> Agbni1-11.1 C2             AGTAGCTTTGAAGATTAACAATGAGGTTCAAGAATAACTGtccttaccattaagttgatc                          pCORE integration <br /> Agbni1-11.1 IRO1           GAGGATGCTAAGTCTCCGGAAAAGGATCCAAGCGAGCTTCAGCGATCAGGCCAGTTATTC                          Mutagenesis <br /> Agbni1-11.1 IRO2           TGAGCTCCAGTAGCTTTGAAGATTAACAATGAGGTTCAAGAATAACTGGCCTGATCGCTG                          Mutagenesis <br /> Agbni1-11.2 C1             GGGGTTCAAGTTGAGTACATTGCAGAGGCTCACCTTCATTgagctcgttttcgacactgg                          pCORE integration <br /> Agbni1-11.2 C2             GTTCCACGTAGTTCAAGAAAGTCATGCTGTTTTTCTCGTCtccttaccattaagttgatc                          pCORE integration <br /> Agbni1-11.2 IRO1           GCACAGGGGTTCAAGTTGAGTACATTGCAGAGGCTCACCTTCATTAGAGACGAGAAAAAC                          Mutagenesis <br /> Agbni1-11.2 IRO2           CTGACGATGCGTTCCACGTAGTTCAAGAAAGTCATGCTGTTTTTCTCGTCTCTAATGAAG                          Mutagenesis <br /> AgBNRCFP-N                 CCAGCATGTCAACCACTATATTGATCACCGATATATGGACTTCCACACCAACTAGacgcggccgccagctgaagc           In vivo recombination <br /> AgBNI1-GS2                 GCGGCGCGGCGGCGACATCGCGCTGGTCTATCAGTTTCTTGGTGCGGCagggacctggcacggagc                    In vivo recombination <br /> AgRHO1a-ATG                GAGATCGAATTCATGGCGTACCAGACAGGCGGCA                                                    Cloning <br /> AgRHO1aG204C               GCCGGTCGTAGTCCTCcTGGCCCGCTGTATCCC                                                     Mutagenesis <br /> AgRHO1a-TAG                CGACGGATCCCTTTCTTCTTCTTCTTGTCACCGT                                                    Cloning <br /> AgRHO1b-ATG                GATCGAATTCATGTCTCAGCAAATGCATAAC                                                       Cloning <br /> AgRHO1bG207C               GCCTGTCGTAATCCTCGTGCCCAGCCGTATCC                                                      Mutagenesis <br /> AgRHO1b-TAG                CGACGGATCCCCTTTTTCTTCTTCTTCTTGTCAGAC                                                  Cloning <br /> AgRHO2-ATG                 GAGATCGAATTCATGACGGTCAACGTTGTGAGAC                                                    Cloning <br /> AgRHO2G195C                CAGACGCTCGTATTCTTCGTGACCAGCAGTATCC                                                    Mutagenesis <br /> AgRHO2-TAG                 CGACGGATCCCGCCTTGACCCGGCTCCTTGTTCAC                                                   Cloning <br /> AgRHO3-ATG                 GATCGAATTCATGCCTCTGTGTGGGTCGAG                                                        Cloning <br /> AgRHO3G219C                GCAACCGGTCAAACTCCTCGTGCCCAGCAGTGTCCCACAGG                                             Mutagenesis <br /> AgRHO3-TAG                 GACGGATCCCACTGCTGCTTTCGGCCTCGG                                                        Cloning <br /> AgRHO4-ATG                 GAGATCGAATTCATGAGCGCAGGGCCGTTGCAAG                                                    Cloning <br /> AgRHO4G333C                CAGCCGGCTGTACTCCTCGTGCCCAGCAGTGTCCCACAGC                                              Mutagenesis <br /> AgRHO4-TAG                 GACGGATCCCGCGGTGCTTGCGCACCCGCTTG                                                      Cloning <br /> AgRHO5-ATG                 GATCGAATTCATGTGTTTTTCGCAGAGCGGCAG                                                     Cloning <br /> AgRHO5G255Cneu             CCGCAACCGGTCGTACTCCTCGTGCCCCGCCGTGTCCCACAG                                            Mutagenesis <br /> AgRHO5-TAG                 CGACGGATCCCTCTAGATCTCTTCCTCCTCTTTTTC                                                  Cloning <br /> AgCDC42-ATG                GATCGAATTCATGCAGACATTGAAGTGCGTGGTC                                                    Cloning <br /> AgCDC42-TAG                CGACGGATCCCCTTCTTGCTCTTCTTGATG                                                        Cloning <br /> AgCDC42G183C               GCCGCAACCTGTCGTAGTCCTCGTGGCCGGCAGTGTCGAACAAGC                                         Mutagenesis <br /> u-40                       GTTTTCCCAGTCACGAC                                                                     Mutagenesis <br /> Reverse                    CAGGAAACAGCTATGACCATG                                                                 Mutagenesis <br /> Sec4-ECO                   cgcgaattcTCGGGGCTAAGAACGGTGTC                                                         Cloning <br /> Sec4P-BAM                  ccggatccCATTATAGCTACTGTACTGC                                                          Cloning <br /> Sec4P-HIND                 ggaagcttCTACTACGCCTGAGCGCCGC                                                          Cloning <br /> Sec4-SPE                   caccaactagtTGGGCGGCCTGTGCTGCAAAG                                                      Cloning <br /> GFP-BAM                    cgggatccAGTAAAGGAGAAGAACTTTTCAC                                                       Cloning <br /> GFP-ECO                    cgcgaattcTTTGTATAGTTCATCCATGCC                                                        Cloning <br /> AgBNI1-ATG2Hy              GTAGTAACAAAGGTCAAAGACAGTTGACTGTATCGCCGGAATTCatgaagaagtccacgcactcgaac                  In vivo recombination <br /> AgBNI1-TAG2Hy              CATAAGAAATTCGCCCGGAATTAGCTTGGCTGCAGGTCGACGGATCCttacttgtgctcatcgagcattg                In vivo recombination <br /> AgBNI1-P_HindIII           gcgcgcaagcttCTGGGACGACGACAAGCCTAC                                                     Cloning <br /> AgBNI1-T_SmaI              gcgccccgggCCTGCACCGAAATCCGCCGA                                                        Cloning <br /> AgBNI1-GS1                 CGGTGAACACCCGGAGTCTCGCAAGTCAATGCTCGATGAGCACAAGggtgcaggcgctggagctg                     In vivo recombination <br /> AgBNI1-GS2                 GCGGCGCGGCGGCGACATCGCGCTGGTCTATCAGTTTCTTGGTGCGGCagggacctggcacggagc                    In vivo recombination <br /> AgBNI1-NS1                 GGTATAGTGGCAGCGTCGGCGGCTGGGCACATATGCAAGGccagtgaattcgagctcgg                           Deletion <br /> AgBNI1-NS2                 GCTGGTCTATCAGTTTCTTGGTGCGGCGCTGGCGAACCTTtacgccaagcttgcatgcct                          Deletion <br /> Gal4-ad                    GTTTGGAATCACTACAGGG                                                                   sequencing <br /> Gal4-bd                    GATTGGCTTCAGTGGAG                                                                     sequencing <br /> Gad-term                   GAGATGGTGCACGATGCACAGTTG                                                              sequencing <br /> Gbt-term                   CGTTTTAAAACCTAAGAGTCAC                                                                sequencing <br /> AgCdc42_c1                 GGACGAGCCGTACACGTTGGGCTTGTTCGACACTGCCGGGgagctcgttttcgacactgg                          pCORE integration <br /> AgCdc42_c2                 GTCGACGGGTACGACAACGGCCGCAACCTGTCGTAGTCCTCtccttaccattaagttgatc                         pCORE integration <br /> AgCdc42_Q-H-antisense      cacgtccgtcgacgggtacgacaacggccgcaacctgtcgtagtcctcgtgcccggcagtg                         Mutagenesis <br /> AgCdc42_Q-H-sense          gtgatgatcggggacgagccgtacacgttgggcttgttcgacactgccgggcacgaggac                          Mutagenesis <br /> INT_CDC42*_S1              ACCAAGTGCCAGCTCTAGCGCCAGGACCCGCGCGAGACCTGCTAGGGATAACAGGGTAAT                          Mutagenesis <br /> INT_CDC42*_S2              TTGCCGATTTTTGGGCAGGGCGCAAAGACAGATATAGCAGAGGCATGCAAGCTTAGATCT                          Mutagenesis <br /> AgCDC42-ATG                GATCGAATTCATGCAGACATTGAAGTGCGTGGTC                                                    Verification <br /> AgCDC42-TAG                CGACGGATCCCCTTCTTGCTCTTCTTGATG                                                        Verification <br /> AgBNI1dD-F1                GACTTTATTCTGGAGTACAAGCGCGCGCAGGAGTTTAACCGCAAGATCaaaacgacggccagtgaattcg                In vivo recombination <br /> AgBNI1dD-F2                GACATCGCGCTGGTCTATCAGTTTCTTGGTGCGGCGCTGGCGAACCTTaccatgattacgccaagcttgc                In vivo recombination <br />  <br /> a <br />     Underscore indicates nucleotides exchanged for mutagenesis. <br />  <br />  <br /> Vol. 17, January 2006                                                                                                              133 <br />  H.-P. Schmitz et al. <br />  <br />  <br />  <br /> Table 3. Plasmids <br />  <br /> Name                                                     Insert                                               Reference <br />  <br /> pUC19                                                      &#226;&#8364;&#8221;                                          Vieira and Messing (1991) <br /> pGBT9                                                      &#226;&#8364;&#8221;                                          Bartel et al. (1993) <br /> pGAD424                                                    &#226;&#8364;&#8221;                                          Bartel et al. (1993) <br /> YCPlac111                                                  &#226;&#8364;&#8221;                                          Gietz and Sugino (1988) <br /> pRS415                                                     &#226;&#8364;&#8221;                                          Sikorski and Hieter (1989) <br /> pUC19NATPS                                                 &#226;&#8364;&#8221;                                          D. Hoepfner, personal <br />                                                                                                         communication <br /> pAGT141                 pUC19 carrying the PstI fragment from pGEN and the SacI fragment from         A. Kaufmann, personal <br />                           pGUG                                                                          communication <br /> pCORE                                                       &#226;&#8364;&#8221;                                         Storici et al. (2001) <br /> pUC19AgRHO1a            A. gossypii RHO1a from nucleotide 1&#226;&#8364;&#8220;615                                       This study <br /> pUC19AgRHO1b            A. gossypii RHO1b from nucleotide 1&#226;&#8364;&#8220;609                                       This study <br /> pUC19AgRHO2             A. gossypii RHO2 from nucleotide 1&#226;&#8364;&#8220;546                                        This study <br /> pUC19AgRHO3             A. gossypii RHO3 from nucleotide 1&#226;&#8364;&#8220;660                                        This study <br /> pUC19AgRHO4             A. gossypii RHO4 from nucleotide 1&#226;&#8364;&#8220;765                                        This study <br /> pUC19AgRHO5             A. gossypii RHO5 from nucleotide 1&#226;&#8364;&#8220;870                                        This study <br /> pUC19AgCDC42            A. gossypii CDC42 from nucleotide 1&#226;&#8364;&#8220;561                                       This study <br /> pGBTAgRHO1a             A. gossypii RHO1a from nucleotide 1&#226;&#8364;&#8220;615                                       This study <br /> pGBTAgRHO1b             A. gossypii RHO1b from nucleotide 1&#226;&#8364;&#8220;609                                       This study <br /> pGBTAgRHO2              A. gossypii RHO2 from nucleotide 1&#226;&#8364;&#8220;546                                        This study <br /> pGBTAgRHO3              A. gossypii RHO3 from nucleotide 1&#226;&#8364;&#8220;660                                        This study <br /> pGBTAgRHO4              A. gossypii RHO4 from nucleotide 1&#226;&#8364;&#8220;765                                        This study <br /> pGBTAgRHO5              A. gossypii RHO5 from nucleotide 1&#226;&#8364;&#8220;870                                        This study <br /> pGBTAgCDC42             A. gossypii CDC42 from nucleotide 1&#226;&#8364;&#8220;561                                       This study <br /> pGBTAgRHO1a*            A. gossypii RHO1a G204C from nucleotide 1&#226;&#8364;&#8220;615                                 This study <br /> pGBTAgRHO1b*            A. gossypii RHO1b G207C from nucleotide 1&#226;&#8364;&#8220;609                                 This study <br /> pGBTAgRHO2*             A. gossypii RHO2 G195C from nucleotide 1&#226;&#8364;&#8220;546                                  This study <br /> pGBTAgRHO3*             A. gossypii RHO3 G219C from nucleotide 1&#226;&#8364;&#8220;660                                  This study <br /> pGBTAgRHO4*             A. gossypii RHO4 G333C from nucleotide 1&#226;&#8364;&#8220;765                                  This study <br /> pGBTAgRHO5*             A. gossypii RHO5 G255C from nucleotide 1&#226;&#8364;&#8220;870                                  This study <br /> pGBTAgCDC42*            A. gossypii CDC42 from nucleotide 1&#226;&#8364;&#8220;561                                       This study <br /> pGBTAgBNI1-N            A. gossypii BNI1 from nucleotide 1&#226;&#8364;&#8220;3152                                       This study <br /> pGADAgBNI1-N            A. gossypii BNI1 from nucleotide 1&#226;&#8364;&#8220;3152                                       This study <br /> YCp111GFPSEC4           N-terminal Fusion of GFP to A. gossypii SEC4                                  This study <br /> pAMK1                   A. gossypii BNI1 from nucleotide &#226;&#171;&#186;682 to &#226;&#171;&#185;523                                 This study <br /> pCDC42                  AgCDC42 including promoter and terminator                                     This study <br /> pCDC42cas               Agcdc42 with codon CAG starting at position 181 replaced by pCORE             This study <br /> pcdc42cons              pCDC42 with codon CAG starting at position 181 altered to CAC                 This study <br /> pcdc42kanr              pcdc42cons with integrated GEN3 cassette behind ORF                           This study <br /> pUC19cdc42cons          3.1 kb BamHI/PstI fragment from pcdc42kanr with pUC19 backbone                This study <br /> pSEC4NAT                PstI fragment from pUC19NATPS carrying ClonNAT cassette ligated into          This study <br />                           PstI site of YCP111GFPSEC4. Orientation reverse <br />  <br />  <br />  <br /> separate. Septation in A. gossypii like in other filamentous      min, the S. cerevisiae cells have undergone only one or two <br /> ascomycetes occurs in the growing hyphae and at the base of       additional divisions whereas the A. gossypii young myce- <br /> branches but separation of hyphal segments does not occur.        lium has already developed nine growing hyphae. At 450 <br /> Lateral branching leads to the successive establishment of        min, the S. cerevisiae cells have divided one or two more <br /> several novel axes of polarity at the cortex of growing hy-       times, and the A. gossypii young mycelium is now spreading <br /> phae, in A. gossypii usually in an about right angle from the     from 23 hyphal tips. After nearly 700 min, the fastest hyphae <br /> growth axis of the mother hyphae. In budding yeast, only          (marked with open arrow heads) have started to symmetri- <br /> one novel axis of polarity (site of bud emergence) is estab-      cally divide at their tips, the common mode of branching <br /> lished in each cell cycle. Finally, the process of dividing the   when hyphal tip speeds have reached 80 &#226;&#65533;&#174;m/h or more. <br /> polarity-axis at the tip of mature hyphae (tip splitting) is      This switch in branching pattern allows optimal radial <br /> restricted to a subset of filamentous ascomycetes including       spreading of an A. gossypii mycelium. The apparently sym- <br /> A. gossypii. Budding yeasts do not simultaneously generate        metric division of the polar growth machinery leads only to <br /> two buds (two axes of polarity).                                  a transient decrease of tip elongation speeds by &#226;&#172;&#402;20% as <br />    To visualize the differences in growth dynamics of S.          determined for seven tip splitting events in Supplementary <br /> cerevisiae and A. gossypii, we simultaneously monitored, by       Movie S1 (unpublished data). Using similar conditions, <br /> time-lapse video microscopy, growth of both fungi on AFM          Knechtle et al. (2003) presented the dynamics of successive <br /> agar (Movie S1). Four representative frames were selected to      tip splitting events every 90 &#226;&#8364;&#8220;120 min at the edge of a radi- <br /> show the development of S. cerevisiae mini colonies and of an     ally expanding mature A. gossypii mycelium. <br /> A. gossypii mycelium (Figure 1B). Arrows in the zero minute          How can these differences in morphogenesis be explained <br /> frame point to eight S. cerevisiae cells, which have just fin-    when both organisms carry a very similar set of proteins? <br /> ished the first division, and to one germinated A. gossypii       One possible answer is that homologous polar growth com- <br /> spore from which the first hypha has emerged. After 225           ponents may exert not only common functions but also <br />  <br /> 134                                                                                                 Molecular Biology of the Cell <br />                                                                                                           Formin for Tip Growth and Tip Splitting <br />  <br />  <br />                                                                                distinct functions in the cellular networks of both organisms. <br />                                                                                One example for shared and specific functions is AgSpa2p, <br />                                                                                the first component identified in A. gossypii to permanently <br />                                                                                localize at hyphal tips, where it plays an essential role to <br />                                                                                reach maximal tip speeds (Knechtle et al., 2003). These are <br />                                                                                properties specific for A. gossypii, because ScSpa2p, the ho- <br />                                                                                molog in S. cerevisiae, does not localize permanently but only <br />                                                                                transiently to the bud tip and because a cellular process, con- <br />                                                                                trolling maximal bud tip speed, is unknown in morphogenesis <br />                                                                                of S. cerevisiae (Sheu et al., 1998). Because ScSpa2p is part of the <br />                                                                                polarisome, we speculated that A. gossypii homologues of other <br />                                                                                components of the polarisome complex, in particular the <br />                                                                                formin homology protein ScBni1p (Evangelista et al., 2002; <br />                                                                                Sagot et al., 2002a), could fulfill, in addition to common func- <br />                                                                                tions, also specific functions in sustained hyphal tip growth <br />                                                                                and tip splitting. Therefore, we searched for homologues in A. <br />                                                                                gossypii to ScBni1p and the related formin ScBnr1p and inves- <br />                                                                                tigated to which degree the homologues would contribute to <br />                                                                                sustained hyphal tip growth and tip splitting. <br />  <br />                                                                                The A. gossypii Formin Family Members <br />                                                                                Using FASTA database searches (Pearson, 1990), we identi- <br />                                                                                fied three formin homology members in the genome of A. <br />                                                                                gossypii (Dietrich et al., 2004). By combining information <br />                                                                                from homology searches with synteny, these proteins can be <br />                                                                                categorized as follows: one syntenic homolog to the S. cer- <br />                                                                                evisiae BNI1 gene (AFR669W, in the following referred to as <br />                                                                                AgBNI1) and one syntenic and one nonsyntenic, telomere- <br />                                                                                located homolog to the S. cerevisiae BNR1 gene (AFR301C <br />                                                                                and AGL364C, referred to as AgBNR1 and AgBNR2, respec- <br />                                                                                tively). The formins ScBni1p and AgBni1p have almost the <br />                                                                                same size and domain composition (Figure 2A) using the <br />                                                                                domain <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> from Evangelista et al. (2002). The two <br /> Figure 1. Differences in polar growth between A. gossypii and S. cerevi- <br />                                                                                homologues to ScBnr1p differ much more in size and, inter- <br /> siae. (A) Schematic presentation of landmarks of hyphal growth of A. <br /> gossypii: Sustained polar growth (accelerating growth along a selected         estingly, these size differences are located exclusively within <br /> polarity-axis by continuous surface expansion at the tip), septation           the FH1 domains leading to differences in the number of <br /> (growth of chitin-containing open rings at 30&#226;&#8364;&#8220;50-&#226;&#65533;&#174;m distances at the           short polyproline stretches (Figure 2B). <br /> cortex of hyphae and at the base of branches), lateral branching (successive      It was shown that the FH1 domain of formins can mediate <br /> selection of new axes of polarity at the cortex of elongating hyphae), and     interactions with profilin, an abundant actin-binding pro- <br /> tip splitting (simultaneous generation of two new axes of polarity at fast     tein, thereby sequestering actin monomers (Pruyne et al., <br /> growing tips). (B) Simultaneous monitoring of A. gossypii and S. cerevisiae    2002; Sagot et al., 2002b; Moseley et al., 2004). The FH2 <br /> growth. A. gossypii spores and S. cerevisiae cells were mixed and plated on    domain provides the activity for nucleation of actin cables <br /> AFM agar. After 4 h the agar surface was screened under the microscope. <br />                                                                                (Pruyne et al., 2002; Sagot et al., 2002a; Kovar et al., 2003; <br /> An area was selected for the start of the video (top left panel) showing <br /> eight S. cerevisiae cells after their first division (small arrows) and one    Harris et al., 2004; Moseley et al., 2004). This domain is the <br /> germinated A. gossypii spore with the first emerged hypha (F). Growth in       most conserved and best characterized domain within form- <br /> this area was monitored for 15 h at 25&#194;&#176;C with pictures taken every 3 min       ins and was used as a basis for the phylogenetic classifica- <br /> (Supplementary Movie S1). After 225 min, the S. cerevisiae mini-colonies       tion of over a hundred formins (Higgs and Peterson, 2005). <br /> consist of four to eight cells, whereas the A. gossypii young mycelium has     The FH3 or Formin Homology 3 domain is only loosely <br /> already developed nine extending hyphae. Two emerged, in addition to           conserved among formins and is defined by its role, in S. <br /> the first hypha, from the germinated spore and six originated from the         cerevisiae, to help locating ScBni1p to the bud tip (Ozaki- <br /> initiation of lateral branches at the cortex of growing hyphae (two            Kuroda et al., 2001). <br /> branches are marked by black arrow heads). At 450 min, the S. cerevisiae <br />                                                                                   Other domains shown in Figure 2 are known to be in- <br /> minicolonies have 12&#226;&#8364;&#8220;32 cells and the A. gossypii young mycelium now <br /> consists of 23 hyphae due to 14 newly emerged lateral branches (black          volved in diverse protein interactions in S. cerevisiae. The <br /> arrow heads). At 690 min the fastest hyphae, marked with open arrow-           Spa2-binding domain (SBD) and to a lesser degree the Bud6- <br /> heads, have started to symmetrically divide at their tips. Ten additional      binding domain (BBD) are important for localization of <br /> tip-splitting events were observed until the end of the movie at 897 min.      ScBni1p to the growing bud tip (Ozaki-Kuroda et al., 2001; <br /> The elongation speeds of the main hyphae during early mycelial devel-          Sagot et al., 2002a). The Rho-binding domain (RBD) interacts <br /> opment are &#226;&#172;&#402;10 &#226;&#65533;&#174;m/h (0 min), 15 &#226;&#65533;&#174;m/h (225 min), and 25 &#226;&#65533;&#174;m/h (450               with small GTPases of the Rho-type and also undergoes an <br /> min), as determined from the elongation of the four hyphae marked in           intramolecular association with a short carboxy-terminal do- <br /> frame 225 min, which were measured every four frames until 501 min. For        main called diaphanous autoregulatory domain (DAD). This <br /> technical reasons 21 frames were not recorded during the early period          association, indicated by the arrow in Figure 2A, prevents <br /> between the frames representing 132 and 135 min. To determine hyphal <br /> speeds at tip splitting hyphal elongations were measured during 18 <br />                                                                                actin cable assembly unless an activated GTPase binds to <br /> frames before and after tip splitting. This mode of branching starts at        this domain (Alberts, 2001). A short sequence similar to the <br /> elongation speeds of 80 &#226;&#65533;&#174;m/h. The highest speed observed at the end of         known functional DAD residues of ScBni1p is also found in <br /> Supplementary Movie S1 before a tip splitting was 140 &#226;&#65533;&#174;m/h. Four cases         all members of the Bnr1 group (small black bars in Figure <br /> of consecutive tip splitting events were observed with an average distance     2B) but its function remains unclear. Importantly, in contrast <br /> of 190 &#226;&#65533;&#174;m and an average time interval of 140 min. Scale bar, 20 &#226;&#65533;&#174;m.           to ScBnr1p, am RBD could not be identified in the A. gossypii <br />  <br /> Vol. 17, January 2006                                                                                                                          135 <br />  H.-P. Schmitz et al. <br />  <br />  <br />  <br />  <br /> Figure 2. Domain comparisons between formin homologues of S. cerevisiae and A. gossypii. (A) Comparison of AgBni1p and ScBni1p. The <br /> domain structure of ScBni1p is based on the one published by Evangelista et al. (2002). Domain borders were first determined using the <br /> gap program of the GCG Wisconsin Package (Accelrys), which uses an Needleman-Wunsch algorithm (Needleman and Wunsch, 1970). <br /> Each domain was realigned and the identity values were taken from the output. All distances are drawn to a relative scale. RBD, Rho-binding <br /> domain; FH1&#226;&#8364;&#8220;3, Formin Homology domain 1&#226;&#8364;&#8220;3; SPD, Spa2-binding domain; BBD, Bud6-binding domain; DAD, diaphanous autoregulatory <br /> domain, which is drawn as black bar within BBD. The point mutations and the DAD deletion used in this work are indicated as well as the <br /> boundaries of the amino-terminal fragment used in two-hybrid studies. (B) Comparison of AgBnr1p and AgBnr2p with ScBnr1p. The domain <br /> structure of ScBnr1p is based on that of ScBni1p. Domain borders were determined and marked as described in A. No clear indication of an <br /> RBD was found in the two A. gossypii homologues. Small black diamonds indicate positions of short stretches of two or more prolines that <br /> are not separated by more than one other residue. <br />  <br />  <br />  <br /> homologues searching for this domain (DRF_GBD) in PFAM                   Because the two S. cerevisiae formins have partially over- <br /> version 17.0. This result is consistent with our finding that          lapping functions allowing, for example, single deletions to <br /> neither full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> nor amino-terminal parts of AgBnr1p and            grow, we expected, based on the domain comparisons, also <br /> AgBnr2p bind to wild-type or activated forms of all A. gossypii        to find overlapping functions in the three formins of A. <br /> Rho-GTPases in a two-hybrid test (unpublished data).                   gossypii. <br />  <br /> 136                                                                                                          Molecular Biology of the Cell <br />                                                                                                    Formin for Tip Growth and Tip Splitting <br />  <br />  <br />                                                                           attempts to locate an AgBnr1-GFP fusion remained unsuc- <br />                                                                           cessful. <br />                                                                              The viability of both deletions indicates that the two AgBnrp <br />                                                                           homologues probably encode overlapping functions. The <br />                                                                           genomic arrangement of the A. gossypii BNR genes indeed <br />                                                                           suggests that AgBNR2 most likely originates from a dupli- <br />                                                                           cation of an ancient AgBNR1 gene involving transposition of <br />                                                                           one copy to the telomere region of chromosome VI. We <br />                                                                           therefore constructed and tested double deletions of both <br />                                                                           homologues. Because of the lack of a functional mating <br />                                                                           system in A. gossypii, we had to delete the AgBNR2 gene in <br />                                                                           a strain already deleted for the AgBNR1 gene. This resulted <br />                                                                           in transformants that are homokaryotic for the Agbnr1 dele- <br />                                                                           tion (ClonNAT resistance marker) and heterokaryotic for the <br />                                                                           Agbnr2 deletion (geneticin resistance marker). PCR-verified <br />                                                                           heterokaryotic transformants were sporulated and spores <br />                                                                           were dissected on different selection media. Spores were <br />                                                                           viable on ClonNAT, selecting for the Agbnr1 deletion, but <br />                                                                           nonviable on geneticin, selecting for the Agbnr1/Agbnr2 <br />                                                                           double deletion (n &#226;&#172;&#381; 100). To avoid the possibility that the <br />                                                                           Agbnr1 deletion strain used to construct the double deletion <br />                                                                           carried an unknown mutation, which is colethal with the <br />                                                                           introduced Agbnr2 deletion, we repeated the experiment <br />                                                                           with a newly constructed Agbnr1 transformant as the recip- <br />                                                                           ient strain, now marked by geneticin resistance. Then we <br />                                                                           used leucine auxotrophy as selection for the Agbnr2 deletion. <br />                                                                           Again, we were unable to isolate any mature mycelium <br />                                                                           carrying a homokaryotic Agbnr1/Agbnr2 double deletion. To <br />                                                                           see if the nonviable spores showed residual growth, we <br />                                                                           looked for germination structures under the microscope. <br />                                                                           None of the spores formed a germ bubble (n &#226;&#172;&#381; 100), indi- <br />                                                                           cating a common function of AgBnr1p and AgBnr2p essen- <br /> Figure 3. Wild type-like growth of Agbnr1 and Agbnr2 deletions.           tial very early in germination. <br /> (A) Mycelial colonies from A. gossypii wild-type and single deletion <br /> strains. Small aliquots of mycelia were transferred to the middle of      Deletion of AgBNI1 Is Lethal <br /> full medium plates and grown for 6 d at 30&#194;&#176;C. Bar, 1.0 cm. (B and C)      To investigate the functions of AgBNI1, we first deleted the <br /> Visualization of actin patches, actin cables, and actin rings by stain-   entire ORF. More than 100 spores obtained from the primary <br /> ing young mycelium of the respective deletion strains with Alexa- <br />                                                                           heterokaryotic mycelium were dissected and placed on se- <br /> phalloidin (Knechtle et al., 2003). In both panels a growing tip is at <br /> the top and an actin ring is seen at the bottom below an emerged          lective medium. None of these spores gave rise to a mature <br /> lateral branch. Bars, 5 &#226;&#65533;&#174;m.                                               mycelium. Thus, deletion of AgBNI1 is lethal despite the pres- <br />                                                                           ence of the two other formin genes, AgBNR1 and AgBNR2, <br />                                                                           indicating an essential role of the formin AgBni1p. In S. <br />                                                                           cerevisiae, mutants with deletions of either one of the two <br /> Single Deletions of Agbnr1 and Agbnr2 Are Viable But                      formin genes, ScBNI1 and ScBNR1, are viable and only <br /> Not the Double Deletion                                                   deletion of both genes is lethal (Imamura et al., 1997). <br /> To investigate the cellular function of the two homologues of                Microscopic observation of Agbni1&#226;&#338;&#172; (Figure 4) revealed <br /> ScBnr1p, we first deleted the entire ORFs of AgBNR1 and                   that spores started swelling like wild type, but were not able <br /> AgBNR2 by PCR-based gene targeting using a cassette cod-                  to initiate hyphal growth. Instead, irregular isotropic growth <br /> ing for resistance against geneticin (G418). Young mycelium               proceeded for at least 20 h, generating giant, potatolike cells <br /> with 20 &#226;&#8364;&#8220; 40 nuclei was transformed to geneticin-resistance.              containing over 100 nuclei. Because formins were reported <br /> These primary transformants are heterokaryotic because                    to catalyze actin cable polymerization (Pruyne et al., 2002; <br /> their haploid nuclei carry either the wild-type or the deletion           Sagot et al., 2002b), we investigated the actin cytoskeleton of <br /> allele. Homokaryotic deletion transformants were obtained                 Agbni1&#226;&#338;&#172; stained with Alexa-phalloidin. Long, polarized ac- <br /> from spores that contain single nuclei and that were dis-                 tin cables, reaching the tips, have been reported for A. gos- <br /> sected on selective agar medium. Growing homokaryotic                     sypii wild type (Knechtle et al. (2003) and Figure 4D). The <br /> transformants were rechecked for the loss of the wild-type                images in Figure 4, E and F, show that Agbni1 deletion <br /> and presence of the deletion allele and were phenotypically               mutants do not contain polarized actin cables and that actin <br /> characterized.                                                            patches are distributed all over the cell cortex. This is in <br />    As can be seen in Figure 3A, the mycelial development of               contrast to wild-type hyphae, where patches are concen- <br /> both single deletions is not perturbed. Consistent with this              trated at the tips The only concentration of actin patches is <br /> wild type-like growth, all actin structures can be observed               observed along structures which, according to the DIC im- <br /> by Alexa-phalloidin staining as shown in Figure 3, B and C.               ages in Figure 4, B and C, most likely mark sites of septum <br /> Staining with calcofluor white revealed an increased num-                 formation as described before (Knechtle et al., 2003). Thus, <br /> ber of incomplete chitin rings in the Agbnr2 deletion, and a              AgBNI1 might not be essential for septum formation but is <br /> fusion with GFP revealed that at least a fraction of the                  required for actin cable formation and hyphal morphology. <br /> AgBnr2p-GFP proteins locates at septa, whereas no fluores-                   The loss of actin cable formation in the Agbni1 deletion <br /> cence was detected at the tips (unpublished data). Several                raises an important question. Why is this loss not compen- <br />  <br /> Vol. 17, January 2006                                                                                                                 137 <br />  H.-P. Schmitz et al. <br />  <br />  <br />  <br />  <br />                                                                                       Figure 4. Lack of mycelial development af- <br />                                                                                       ter deletion of the single formin gene AgBNI1. <br />                                                                                       Spores from wild type and from heterokary- <br />                                                                                       otic Agbni1&#226;&#338;&#172; transformants were grown over <br />                                                                                       night at 30&#194;&#176;C in AFM and AFM-Geneticin <br />                                                                                       medium, respectively. DIC images are shown <br />                                                                                       for wild-type (A) and Agbni1&#226;&#338;&#172; (B and C), and <br />                                                                                       fluorescence images for rhodamine-stained <br />                                                                                       wild-type (D) and Alexa-phalloidin-stained <br />                                                                                       Agbni1&#226;&#338;&#172; (E and F). Arrows mark giant sep- <br />                                                                                       tumlike structures in Agbni1&#226;&#338;&#172; that are visible <br />                                                                                       by DIC and by actin staining. The needlelike <br />                                                                                       structures attached to the middle of the <br />                                                                                       young mycelium (A) and to one pole of the <br />                                                                                       giant cells (B and C) originate from the spores, <br />                                                                                       which have the shape of &#226;&#172;&#402;20-&#226;&#65533;&#174;m-long nee- <br />                                                                                       dles. Bars, (A and D) 10 &#226;&#65533;&#174;m and 5 &#226;&#65533;&#174;m for all <br />                                                                                       other images. <br />  <br />  <br />  <br /> sated, as in S. cerevisiae, by one of the two other formin        Localization of the Polarisome Component AgSpa2p in <br /> homology members? The domain comparisons of Figure 2              the Agbni1&#226;&#338;&#172; Mutant <br /> do not reveal an answer because there is no significant differ-   One possible essential process AgBni1p that could be in- <br /> ence between formin homologues of both organisms with re-         volved in at hyphal tips is the assembly or stabilization of a <br /> spect to the domains responsible for actin cable formation.       polarisome-like structure, which acts as a polarity control <br /> Therefore we hypothesized that AgBni1p is essential because       complex. For example, Bauer et al. (2004) could recently <br /> its essential function may be connected to its location.          demonstrate that the GTPase AgBud1p is important to stably <br />                                                                   maintain the polarisome marker AgSpa2p-GFP at hyphal <br />                                                                   tips and that transient loss of this tip marker in an Agbud1 <br /> AgBni1p Locates to Hyphal Tips                                    deletion causes an immediate arrest of polar growth. To test <br /> To determine the location of AgBni1p, we constructed a GFP        a possible essential role for AgBni1p in the assembly or <br /> fusion at the genomic locus of AgBNI1 and first tested            stabilization of the &#226;&#8364;&#339;polarisome complex,&#226;&#8364;&#65533; we deleted AgBNI1 <br /> whether the fusion protein is functional. Spores expressing       in a strain expressing GFP-labeled AgSpa2p. AgSpa2p <br /> AgBni1p-GFP developed to a young mycelium with actin              permanently localizes to the tips of elongating hyphae <br /> cables, actin patches, and actin rings indistinguishable from     (Knechtle et al., 2003). In the giant Agbni1&#226;&#338;&#172; cells AgSpa2p- <br /> wild type. Mature hyphae displayed regular tip splitting,         GFP is always found at the cortex of the slowly expanding <br /> only their maximal elongation speed was slightly slower           zone opposite from the spore needle (Figure 5C), indicating <br /> than wild type (unpublished data). Microscopic images re-         that AgBni1p does not play an essential role in polarizing <br /> vealed a very weak GFP signal, always located at the tip of       AgSpa2p. The observed polarization may be the reason why <br /> hyphae, which was detectable in both, young mycelium              giant Agbni1&#226;&#338;&#172; cells form potato-shaped structures instead of <br /> (Figure 5A) and mature hyphae (Figure 5B). Interestingly, no      large round cells, like mutants of Agcdc24 and Agcdc42, <br /> signal could be detected at sites of septation in over hundred    which are blocked in early steps of polarity establishment <br /> apical regions inspected. This is in contrast to S. cerevisiae    (Wendland and Philippsen, 2001). <br /> where Bni1p-GFP localizes, in addition to polarized bud <br /> tips, also as ringlike structures at sites of septation (Ozaki-   Mutation of a Single Amino Acid in the FH2 Domain of <br /> Kuroda et al., 2001), where it is most likely involved in cell    AgBni1p Is Lethal <br /> separation, a process that does not occur in growing A.           The experiments presented so far and the domain compar- <br /> gossypii hyphae. Therefore the observed localization is con-      ison of Bni1p from A. gossypii and S. cerevisiae discussed in <br /> sistent with an exclusive role for AgBni1p in tip growth.         Figure 2 suggest similar molecular properties of both pro- <br />  <br /> 138                                                                                                   Molecular Biology of the Cell <br />                                                                                                  Formin for Tip Growth and Tip Splitting <br />  <br />  <br />                                                                         the AgBNI1 deletion is only due to a tip-located loss of actin <br />                                                                         polymerization capability and subsequent loss of actin ca- <br />                                                                         bles rather than loss of other functions. To test this, we <br />                                                                         constructed point mutations in the FH2 domain of AgBni1p, <br />                                                                         leading to the amino acid substitutions S1642G in one and <br />                                                                         D1530G, K1620R in a second mutant. Especially the residues <br />                                                                         mutated in the second mutant are highly conserved <br />                                                                         throughout all formin proteins (Higgs and Peterson, 2005). <br />                                                                         In S. cerevisiae, both homologous substitutions S1623G and <br />                                                                         D1511G, K1601R lead to temperature-sensitive strains in a <br />                                                                         background deleted for ScBNR1, the second formin gene <br />                                                                         (Evangelista et al., 2002). We expected to find temperature- <br />                                                                         sensitive phenotypes for both A. gossypii mutants. Surpris- <br />                                                                         ingly, the analogous substitutions in AgBni1p were lethal, <br />                                                                         even at 16&#194;&#176;C. Like the complete deletion, these point mu- <br />                                                                         tants were able to form giant cells with many actin patches <br />                                                                         but without actin cables (Figure 5D). These experiments <br />                                                                         indicate that the tip-located assembly of actin cables repre- <br />                                                                         sents the essential function of the AgBni1p formin, which <br />                                                                         emphasizes the importance of actin cables for hyphal <br />                                                                         growth. <br />  <br />                                                                         Actin Cable-based Vesicle Transport Is Defect in Agbni1D <br />                                                                         The loss of actin cables should severely impede delivery of <br />                                                                         secretory vesicles to hyphal tips. To confirm that AgBni1p <br />                                                                         is needed for secretory vesicle transport, we isolated the <br />                                                                         AgSEC4 gene by homology to the S. cerevisiae SEC4 gene and <br />                                                                         constructed a GFP-fusion. In budding yeast, a GFP-Sec4p <br />                                                                         fusion protein localizes to secretory vesicles and moves in a <br />                                                                         directed manner along actin cables toward the bud tip <br />                                                                         (Schott et al., 2002). We transformed A. gossypii with a plas- <br />                                                                         mid expressing an amino-terminal GFP-AgSec4p fusion un- <br />                                                                         der control of the native AgSEC4 promoter. As shown in <br />                                                                         Figure 6A, the fusion product localizes mainly to the tip. <br />                                                                         Addition of latrunculin A, which disrupts actin structures, <br />                                                                         abolishes vesicular movement and apical localization, giving <br />                                                                         rise to a uniform fluorescence within hyphae (Figure 6B). To <br />                                                                         observe the localization of vesicles in the absence of <br />                                                                         AgBni1p, we transformed a heterokaryotic Agbni1&#226;&#338;&#172; myce- <br />                                                                         lium with the plasmid expressing the GFP-AgSec4p fusion <br />                                                                         protein. Spores of these transformants were grown under <br />                                                                         conditions selective for both, the Agbni1 deletion and the <br />                                                                         plasmid. In all growing giant cells the GFP fluorescence was <br />                                                                         evenly distributed (Figure 6C). This verifies that continuous <br />                                                                         tip-directed transport of secretory vesicles via actin cables is <br />                                                                         essential for hyphal growth. <br />  <br />                                                                         Hyphal Morphogenesis in the Presence of Constitutively <br /> Figure 5. Localization of AgBni1p-GFP at hyphal tips. (A) Bright-       Active AgBni1p (AgBNI1&#226;&#338;&#172;D) <br /> field and fluorescence image of an A. gossypii germling carrying one <br /> copy of AgBNI1-GFP expressed from the endogenous BNI1 promoter. <br />                                                                         We wanted to know whether an overactivation of the <br /> Because of high autofluorescence of spores, the fluorescence image is   AgBni1 protein would increase hyphal elongation rates. We <br /> only shown for the area corresponding to the white rectangle. Scale     speculated that such a mutant should assemble more actin <br /> bar, 5 &#226;&#65533;&#174;m. (B) Brightfield and fluorescence image of a mature hypha     cables emanating from the tips, thus enhancing the transport <br /> expressing AgBni1p-GFP from a genomic fusion gene. Scale bar, 5 &#226;&#65533;&#174;m.     rate of secretory vesicles toward the tip. To test this hypoth- <br /> (C) Localization of AgSpa2p-GFP in Agbni1&#226;&#338;&#172;. DIC image and fluores-      esis, we deleted in AgBNI1 the coding sequence of the car- <br /> cence images of Agbni1&#226;&#338;&#172; cells carrying a SPA2-GFP fusion at the         boxy-terminal DAD (see Figure 2), thereby eliminating in <br /> AgSPA2 locus. Spores from heterokaryotic transformants were grown       the expressed protein the inactivating interaction of the <br /> in AFM-Geneticin medium over night at 30&#194;&#176;C. Scale bar, 5 &#226;&#65533;&#174;m. (D)        DAD domain with the amino terminus. <br /> <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>Phenotype</span> of the single S1642G substitution in the FH2 domain of <br /> AgBni1p. Spores from heterokaryotic transformants were grown over <br />                                                                            Several independent mutants were isolated. All show a <br /> night in liquid AFM-Geneticin medium at 30&#194;&#176;C and stained for actin      complete change in branching pattern. Although morpho- <br /> with Alexa phalloidin. The fluorescence image (top) and the DIC image   genesis of young wild-type mycelium exclusively displays <br /> (bottom) were taken from the same cell. Scale bar, 10 &#226;&#65533;&#174;m.               lateral branching (Figure 7, A&#226;&#8364;&#8220;C), the overactivation of the <br />                                                                         AgBni1p formin suppresses this branching mode and in- <br />                                                                         duces successive and symmetric hyphal splitting at tips <br /> teins. One important property of AgBni1p and ScBni1p is the             (Figure 7, D&#226;&#8364;&#8220;F). Lateral branching events occur only late and <br /> nucleation of actin cables, which resides within the FH2                very rarely in this mutant, as concluded from videomicros- <br /> domain. We asked whether the characteristic phenotype of                copy of 10 developing young mycelia. Examples are docu- <br />  <br /> Vol. 17, January 2006                                                                                                               139 <br />  H.-P. Schmitz et al. <br />  <br />  <br />                                                                             The nonregulatable, overactive AgBni1&#226;&#338;&#172;D formin leads to <br />                                                                          a growth advantage during the early mycelial development <br />                                                                          but has a negative effect on the further development to a fast <br />                                                                          spreading colony (Figure 8H). Already after 6 to 7 h the <br />                                                                          elongation rate of mutant hyphae and with that the surface <br />                                                                          expansion rate reaches a plateau, whereas the elongation of <br />                                                                          wild-type hyphae continues to accelerate (Figure 8, A and <br />                                                                          B). This difference in growth rates may be due to the high <br />                                                                          frequency of premature tip splitting events in the mutant, <br />                                                                          which impedes hyphal speed acceleration and thus leads to <br />                                                                          a slower growing mycelium compared with wild type. <br />  <br />                                                                          AgBni1p Is an Effector of AgCdc42p <br />                                                                          The domain composition of AgBni1p predicts that it is acti- <br />                                                                          vated when a GTP-bound Rho-type GTPase interacts with <br />                                                                          the amino-terminal RBD (Figure 2). We searched for candi- <br />                                                                          dates of such activator among A. gossypii homologues of <br />                                                                          Rho-type GTPases using two criteria: Its GTP-bound form <br />                                                                          should interact with the amino-terminus of AgBni1p, and a <br />                                                                          GTP-locked form of this protein should induce tip splitting <br />                                                                          in young hyphae due to a permanent activation of AgBni1p, <br />                                                                          similarly as observed for young hyphae of the AgBNI1&#226;&#338;&#172;D <br />                                                                          mutant. By systematic two-hybrid analysis of the AgBni1p <br />                                                                          amino-terminus against all A. gossypii Rho-type GTPases <br />                                                                          and by videomicroscopy of GTP-locked mutants of the three <br />                                                                          Rho-type GTPases, which scored positive in the first test, we <br />                                                                          identified AgCdc42p as the only candidate. The two-hybrid <br />                                                                          result is summarized in Figure 9A and two microscopic <br />                                                                          images showing tip splitting in young hyphae expressing <br />                                                                          GTP-locked AgCdc42p are presented in Figure 9B. These <br />                                                                          hyphae carry only the mutated AgCDC42 allele and show <br />                                                                          growth defects preventing the young mycelia from further <br />                                                                          development. This is most likely due the multitude of effec- <br />                                                                          tor proteins that are regulated by AgCdc42p. <br /> Figure 6. Distribution of GFP-Sec4p in wild type and Agbni1&#226;&#338;&#172;. (A) <br /> Brightfield (left) and GFP-fluorescence (right) micrographs of a         DISCUSSION <br /> single wild-type hypha carrying a GFP-SEC4 fusion. The main <br /> fluorescence localizes to the tip. (B) Localization of GFP-Sec4p prior   The filamentous fungus A. gossypii and the budding yeast S. <br /> (left) and after (right) treatment with 200 &#226;&#65533;&#174;M latrunculin A for 1.5     cerevisiae have different life styles (see Supplementary Movie <br /> min. (C) Brightfield (left) and GFP-fluorescence (right) images of       S1) despite very similar gene contents and conserved do- <br /> Agbni1&#226;&#338;&#172; carrying a GFP-SEC4 fusion gene. All strains were grown <br />                                                                          main compositions of gene products (Dietrich et al., 2004). <br /> in liquid selective medium at 30&#194;&#176;C over night. Scale bars, 5 &#226;&#65533;&#174;m. <br />                                                                          Both organisms can establish polar growth but in S. cerevisiae <br />                                                                          periods of growth alternate with cell divisions, whereas A. <br /> mented in Supplementary Movies S2-S4 for wild type and                   gossypii cells (hyphae) grow for unlimited periods without <br /> Supplementary Movies S5-S7 for AgBNI1&#226;&#338;&#172;D (see Supple-                     undergoing divisions. In addition, A. gossypii hyphae can <br /> mentary Material).                                                       establish multiple new axes of polarity, e.g., during lateral <br />    Quantitative evaluation of the movies reveals similar ini-            branching or tip splitting. In contrast, wild-type yeast cells <br /> tial tip extension speeds for wild type and AgBNI1&#226;&#338;&#172;D (Fig-               never form more than one new bud at the same time (Cav- <br /> ure 8A). However, the average diameter of young hyphae of                iston et al., 2002). We describe here a role for the AgBni1p <br /> the mutant has almost doubled compared with wild type                    formin in two of these processes: the establishment of po- <br /> based on 300 measurements of three hyphae each (4.63 &#226;&#171;&#190;                   larity (formation of hyphae) and the controlled division of <br /> 0.46 &#226;&#65533;&#174;m for wild type and 7.52 &#226;&#171;&#190; 1.1 &#226;&#65533;&#174;m for mutant hyphae).              one axis of polarity into two new polarity axes (tip splitting), <br /> These parameters allowed us to calculate for each time point             a novel function for this protein class. <br /> the surface expansion rate, which, during the first 4 h, is up <br /> to three times higher in the mutant compared with wild type              Redundance of Formins in A. gossypii <br /> (Figure 8B). This increase in surface growth strongly indi-              The lethality of the Agbni1 deletion was surprising because <br /> cates that an overactivation of AgBni1p can enhance the                  A. gossypii contains two additional formin genes, AgBNR1 <br /> traffic of tip-directed secretory vesicles. This idea is sup-            and AgBNR2, and because all three formin genes encode <br /> ported by three experimental results. First, constitutively              proteins with similar domains, notably the actin nucleation <br /> activated AgBni1&#226;&#338;&#172;D proteins still locate to the tips (20/20) of          domain FH2. The deletion of either one of the AgBNR genes <br /> hyphae (Figure 8C). Second, a two to three times higher                  is fully viable indicating overlapping functions in the AgBnr <br /> number of actin cables can be observed in the apical region              formins. These results made us at first conclude that the <br /> of mutant hyphae (Figure 8D) compared with wild-type                     lethality of Agbni1&#226;&#338;&#172; is due to distinct functions of members <br /> hyphae (Figure 8E). And third, more vesicles are transported             of the AgBni1p and the AgBnr proteins. This idea was sup- <br /> to the tip region as evident from the larger area of accumu-             ported by comparison to the more distantly related fission <br /> lated secretory vesicles in mutant hyphae (Figure 8, F and G)            yeast S. pombe, which expresses three members of the formin <br /> compared with wild-type hyphae (Figure 6A).                              family with clear separation of functions. For3 is involved in <br />  <br /> 140                                                                                                         Molecular Biology of the Cell <br />                                                                                                 Formin for Tip Growth and Tip Splitting <br />  <br />  <br />  <br />  <br /> Figure 7. Altered morphogenesis of AgBNI1&#226;&#338;&#172;D mycelium. Wild-type and AgBNI1&#226;&#338;&#172;D spores were incubated in liquid AFM medium at <br /> 30&#194;&#176;C. After 18 h aliquots were stained with Alexa-phalloidin and mounted for microscopy. (A) Idealized branching scheme of the young <br /> wild-type mycelium shown as DIC image in B and as fluorescence image in C. The white arrows mark the needle of the spore from which <br /> two main hyphae developed in opposite directions. New branches always emerged distant from the tip and in an about right angle from <br /> the polarity-axis of the main hyphae, leading to a typical &#226;&#8364;&#339;Christmas tree&#226;&#8364;&#65533; appearance. (D) Idealized tip splitting pattern of the young <br /> AgBNI1&#226;&#338;&#172;D mycelium shown as DIC image in E and as fluorescence image in F. The white arrows mark the original position of the spore. <br /> Scale bar, 10 &#226;&#65533;&#174;m. <br />  <br />  <br />  <br /> cell polarity (Feierbach and Chang, 2001), Cdc12 is involved          and already during early phases of bud formation to the <br /> in cytokinesis (Chang et al., 1997) and Fus1 is necessary for         forming septum. However, budding yeast cells can still sur- <br /> cell fusion during mating (Petersen et al., 1998a). Although S.       vive in the absence of ScBni1p because the septum-associ- <br /> pombe for3&#226;&#338;&#172; had no visible actin cables (Feierbach and                ated ScBnr1p is also able to nucleate actin cables, thereby <br /> Chang, 2001) or the cables were shorter in size and reduced           providing the close-by growing bud with secretory vesicles. <br /> in number (Nakano et al., 2002), cells were still able to grow        This short distance between the septum and the growing <br /> in a polarized manner unlike a deletion of Agbni1.                    bud tip, e.g., during axial budding, is probably of key im- <br />    Different functions were also described for the baker&#226;&#8364;&#8482;s            portance for the viability of cells lacking the ScBni1p formin. <br /> yeast formins ScBni1p and ScBnr1p (Pruyne et al., 2004).              At increasing distances between the septum and the grow- <br /> ScBni1p localizes mainly to the bud tip where it forms actin          ing tip, formin deletion experiments may give different re- <br /> cables. It is also found at the septum in a very late phase of        sults. For example, assuming similar actin nucleating func- <br /> bud formation, when the direction of growth polarity has              tions for the formins in A. gossypii but distinct protein <br /> already switched. ScBnr1p in contrast localizes exclusively           localization requirements, the lethality of the Agbni1 dele- <br />  <br /> Vol. 17, January 2006                                                                                                               141 <br />  H.-P. Schmitz et al. <br />  <br />  <br />  <br />  <br /> Figure 8. Increase in surface expansion and number of actin cables in AgBNI1&#226;&#338;&#172;D. (A) Elongation speeds in &#226;&#65533;&#174;m/h of wild-type hyphae (F) <br /> and AgBNI1&#226;&#338;&#172;D hyphae (&#226;&#189;&#167;) during the first 700 min of hyphal development. The data points represent the average from two hyphae followed <br /> in three young mycelia each, starting shortly after the first hypha has emerged from a germinated spore. (B) Surface expansion rates at hyphal <br /> tips of wild type (F) and AgBNI1&#226;&#338;&#172;D (&#226;&#189;&#167;) were calculated from the elongation speeds determined in A, and the hyphal diameters were <br /> measured every 10 &#226;&#65533;&#174;m. (C) Localization of AgBni1&#226;&#338;&#172;D-GFP fusion proteins at hyphal tips. (D) Visualization by epifluorescence microscopy <br /> of rhodamine-phalloidin-stained F-actin in AgBNI1&#226;&#338;&#172;D. For several hyphae a stack of 16 planes taken at 0.4-&#226;&#65533;&#174;m distances was analyzed. <br /> Because of the high density of actin cables along the cortex only a maximum projection of the top third of the hypha (three planes) is shown. <br /> Four to five actin cables can be seen per cross section, which represents about one-third of the density of cortical actin cables in the hypha. <br /> (E) Rhodamine-phalloidin stained F-actin in wild-type A. gossypii. The stack of 16 planes was processed as described for D, and only the top <br /> third of the hypha is shown. Only one to two cortical actin cables can be seen per cross section. (F and G) Visualization of GFP-AgSEC4- <br /> labeled vesicles in the tip region of an AgBNI1&#226;&#338;&#172;D hypha and the corresponding DIC image. (H) Mycelial colonies of wild type and AgBNI1&#226;&#338;&#172;D <br /> after growth on AFM agar at 30&#194;&#176;C for 5 d. Bars, (C&#226;&#8364;&#8220;F) 5 &#226;&#65533;&#174;m, (H) 2 cm. <br />  <br />  <br />  <br />  <br /> tion could be explained by the large distance between the                 mutant formins can probably still induce actin cables at <br /> growing tip and the nearest septum, preventing septum-                    hyphal tips, or another, so far unidentified formin gene, is <br /> associated formin functions to complement the loss of such                present in the genome of A. nidulans. <br /> functions at hyphal tips. This alternative interpretation is <br /> supported by the fact that we were able to visualize                      AgBni1p and the Actin Cytoskeleton <br /> AgBni1p-GFP at hyphal tips, but not at septa, and AgBnr2p-                The phenotype observed for the AgBNI1 deletion, lack of <br /> GFP at septa but not at hyphal tips.                                      actin cables and hyphal growth, indicated that an important <br />    The fact that AgBni1p functions in hyphal emergence and                function of AgBni1p might be the regulation of vesicle trans- <br /> elongation, but probably not in septum formation, indicates               port via actin cables. A. gossypii hyphae are capable to grow <br /> that it is also functionally different from SepA, the only other          with very high growth speeds of up to 170 &#226;&#65533;&#174;m/h (Knechtle <br /> formin described in a filamentous fungus so far. SepA from                et al., 2003). This speed obviously requires a constant and <br /> Aspergillus nidulans localizes to the tip of hyphae and to                very efficient transport of growth material toward the elon- <br /> septa, and it is essential for morphogenesis. Interestingly,              gating tips. Our results show that single amino acid changes <br /> some SepA mutants, defective in septum formation, are still               in the FH2 domain of AgBni1p, which is responsible for actin <br /> able to form hyphae (Sharpless and Harris, 2002). Therefore,              cable assembly, can only expand to potato-shaped cells, <br />  <br /> 142                                                                                                             Molecular Biology of the Cell <br />                                                                                                    Formin for Tip Growth and Tip Splitting <br />  <br />  <br />                                                                           Regulation of Tip Splitting by AgBni1p <br />                                                                           Tip splitting, the symmetric division of the polar growth <br />                                                                           zone of a single growing cell, is to our knowledge limited to <br />                                                                           filamentous fungi and to dendrites of neuronal cells. Hy- <br />                                                                           phae of A. gossypii usually start tip splitting under optimal <br />                                                                           conditions 12&#226;&#8364;&#8220;14 h after the first hyphae emerged from <br />                                                                           germinated spores (Ayad-Durieux et al., 2000). Premature <br />                                                                           tip splitting, already after 6 &#226;&#8364;&#8220; 8 h of elongation and before <br />                                                                           any lateral branching, as found for the constitutively active <br />                                                                           AgBni1p protein, was not described before. In an Agcla4 <br />                                                                           deletion strain, premature tip splitting was previously ob- <br />                                                                           served but after normal development of a young mycelium, <br />                                                                           including lateral branching (Ayad-Durieux et al., 2000). <br />                                                                           AgCla4p is a homologue of the PAK21 kinase Cla4p from S. <br />                                                                           cerevisiae, which is involved in polarization of the cytoskel- <br />                                                                           eton during the cell cycle (Holly and Blumer, 1999). Evi- <br />                                                                           dence for an involvement of actin in tip splitting was re- <br />                                                                           ported in studies of other filamentous fungi in which tip <br />                                                                           splitting was not observed in wild type but was seen in <br />                                                                           different mutants as a neomorphic phenotype. For example, <br />                                                                           the ramosa-1 mutant of Aspergillus niger can divide at hyphal <br />                                                                           tips when shifted to the restrictive temperature (Reynaga- <br />                                                                           Pena and Bartnicki-Garcia, 1997), and the authors suggested <br />                                                                           that tip splitting might be triggered by a transient alteration <br />                                                                           in the cytoskeletal organization. Similarly, viable mutations <br />                                                                           in the formin SepA in A. nidulans show tip splitting <br />                                                                           (Sharpless and Harris, 2002). In Neurospora crassa, several <br />                                                                           mutants are known that lead to hyphal tip splitting (Sone <br />                                                                           and Griffiths, 1999; Bok et al., 2001; Seiler and Plamann, 2003; <br /> Figure 9. AgCdc42p regulates tip splitting. (A) Two-hybrid assay          Virag and Griffiths, 2004). It was suggested that a calcium <br /> of AgCdc42p-GTP and the amino-terminal half of AgBni1p. Each <br /> spot corresponds to 5 &#226;&#65533;&#174;l of a yeast culture of strain PJ69 &#226;&#8364;&#8220; 4a with an <br />                                                                           gradient controls the transport of vesicles. This control <br /> OD600 of 1, transformed with plasmids encoding fusions of the             mechanism also involves actin and determines whether a tip <br /> indicated proteins to either the Gal4-activation domain (AD) or to        divides or not. The increased surface expansion rate in A. <br /> the Gal4-DNA-binding domain (BD) or, as control, transformed              gossypii hyphae with an activated allele of AgBNI1, reported <br /> with the vector plasmid. The cells were spotted on a medium               here, is most likely caused by the increase in actin cables and <br /> allowing to select for the plasmids and were grown overnight at           thus vesicle transport. This overstimulation in vesicle trans- <br /> 30&#194;&#176;C to screen for protein-protein interactions by growth in the <br />                                                                           port also leads to premature tip splitting. The fact that <br /> absence of histidine. (B) Altered morphogenesis of A. gossypii ex- <br /> pressing GTP-locked AgCdc42p. A heterokaryotic strain of A. gos-          GTP-bound AgCdc42p interacts with AgBni1p in a two- <br /> sypii was constructed carrying the <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>point mutation</span> AgCdc42-Q61H            hybrid assay and that mutants expressing the activated <br /> that encodes a protein mimicking the GTP-bound and therefore              allele of AgCDC42 also show premature tip splitting suggest <br /> activated state of the GTPase. Spores were dissected by microma-          that this event is regulated by an AgBni1p branch of the <br /> nipulation and allowed to germinate under conditions selecting only       AgCdc42p signaling network. The observed growth problems <br /> for spores carrying the GTP-locked AgCDC42 allele. The two micro- <br />                                                                           of the activated Agcdc42 mutant compared with the activated <br /> scopic images show examples of tip splitting soon after the first hypha <br /> emerged from the germling. These hyphae carry only the mutated            Agbni1 mutant can be explained by the fact that changes in the <br /> AgCDC42 allele and show growth defects preventing the young my-           activity state of AgCdc42p will influence the interaction with <br /> celia from further development. This is most likely due the multitude     multiple effector proteins (Nelson et al., 2003). <br /> of effector proteins that are regulated by AgCdc42p. Scale bar, 10 &#226;&#65533;&#174;m.       The involvement of the Cdc42 GTPase in tip splitting is <br />                                                                           also supported by mutants described for the CDC24 ho- <br />                                                                           molog of N. crassa (Seiler and Plamann, 2003) because <br /> which lack polarized actin cables like the deletion mutant.               Cdc24p is a GDP-GTP exchange factor for the small GTPase <br /> Therefore, it might be the defect of actin cable-based trans-             Cdc42p. <br /> port, which leads to the phenotype observed when AgBNI1                      On the basis of the data discussed here, we suggest a <br /> is deleted. It is known from several studies, that actin cables           molecular model for tip splitting regulation. AgCdc42p ac- <br /> can serve as tracks for different cargos such as secretory                tivates AgBni1p which in turn increases the number of actin <br /> vesicles (Johnston et al., 1991) and several organelles (Simon            cables emanating from the tip. This leads than to a stimula- <br /> et al., 1995; Hill et al., 1996; Hoepfner et al., 2001; Rossanese         tion of actin cable-based vesicle transport, which first en- <br /> et al., 2001) as well as mRNA (Bobola et al., 1996; Sil and               larges and finally divides the polar growth site into two new <br /> Herskowitz, 1996; Takizawa et al., 2000). Using the AgSec4p               sites. These data, when combined, suggest that altering the <br /> fused to GFP, we were able to show that secretory vesicles                activities of formin molecules by Cdc42p can lead to dra- <br /> use actin cables for accumulation at the hyphal tips. They no             matically different cell shapes. Incorporation of future ex- <br /> longer do so in AgBNI1-deleted hyphae even though the                     periments into this model should help to increase our un- <br /> polarisome component AgSpa2p localizes correctly to the                   derstanding of this unique process like e.g., its growth speed <br /> cortex at the slowly expanding growth zone, which explains                dependence or it might help us to understand in more detail <br /> the potatolike shape of the giant mutant cells.                           how tip splitting is triggered. <br /> ter submission of this manuscript,                     <br /> two other fungal formins, CaBni1p and CaBnr1p, were                         <br /> shown to play an important but nonessential role in yeast <br /> and hyphal growth of Candida albicans (Li et al., 2005).                          <br /> </body></html>