<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2000 Genetics Society America <br />  <br />  <br />  <br /> A Yeast taf17 Mutant Requires Swi6 Transcriptional Activator Viability <br />            Shows Defects Cell Cycle-Regulated Transcription <br />  <br />                Neil Macpherson,* Vivien Measday,*,1 Lynda Moore* Brenda Andrews* <br />                  *Department Molecular Medical Genetics, University Toronto, Toronto, Ontario M55 1A8, Canada <br />                                                       Manuscript received September 29, 1999 <br />                                                       Accepted publication January 5, 2000 <br />  <br />  <br />                                                                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 In Saccharomyces cerevisiae, Swi6 protein component transcription factors, SBF MBF, <br />              promote expression large group genes late G1 phase cell cycle. Although SBF <br />              required cell viability, SWI6 essential gene. We performed synthetic lethal screen <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />              identify genes required viability absence SWI6 identified 10 complementation groups of <br />              swi6-dependent lethal mutants, designated SLM1 SLM10. We interested mutants <br />              showing cell cycle arrest <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>; slm7-1 swi6&#226;&#338;&#172; slm8-1 swi6&#226;&#338;&#172; double mutants accumulated <br />              large, unbudded cells increased 1N DNA content showed temperature-sensitive growth arrest <br />              presence Swi6. Analysis transcript levels cell cycle-regulated genes slm7-1 SWI6 mutant <br />              strains permissive temperature revealed defects regulation subset cyclin-encoding genes. <br />              Complementation allelism tests showed SLM7 allelic TAF17 gene, encodes a <br />              histone-like component general transcription factor TFIID SAGA histone acetyltransferase <br />              complex. Sequencing showed slm7-1 allele TAF17 predicted encode version Taf17 <br />              truncated highly conserved region. The cell cycle transcriptional defects caused by <br />              taf17 slm7-1 consistent role TAFIIs modulators transcriptional activation reflect <br />              role TAF17 regulating activation SBF MBF. <br />  <br />  <br />  <br /> I  N budding yeast Saccharomyces cerevisiae, irrevers- <br />      ible commitment mitosis occurs late G1 <br /> phase cell cycle. Passage commitment <br />                                                                             MBF. SBF contains Swi4 Swi6 proteins usu- <br />                                                                             ally activates transcription cis-acting element <br />                                                                             called SCB (SWI4/6 cell cycle box; consensus CAC <br /> phase, START, depends activation cyclin                        GAAA) (Nasmyth 1985; Breeden Nasmyth 1987a; <br /> dependent kinase (Cdk) Cdc28 association                        Andrews Herskowitz 1989a,b; Nasmyth and <br /> G1 cyclins Cln1, Cln2, Cln3 (Nasmyth Dirick                         Dirick 1991; Ogas et al. 1991; Taba et al. 1991; Koch <br /> 1991; Ogas et al. 1991; Dirick et al. 1992; Lowndes et                      Nasmyth 1994). The Swi4 protein specifically binds <br /> al. 1992; Breeden Mikesell 1994; Cross et al. 1994;                     SCB elements, Swi6 does bind DNA <br /> Stuart Wittenberg 1994). In absence CLN1                         specifically (Sidorova Breeden 1993) pres- <br /> and CLN2, CDK Pho85 associated G1 cyclins                       ent SBF virtue interaction C terminus <br /> Pcl1 Pcl2 essential START (Espinoza                          Swi4 (Andrews Moore 1992; Primig et al. 1992; <br /> et al. 1994; Measday et al. 1994). Consistent                    Sidorova Breeden 1993). In absence Swi4 <br /> role START, CLN1, CLN2, PCL1, PCL2 period-                       Swi6, G1 cyclin transcription greatly reduced but <br /> ically transcribed, peak transcription late G1                      detectably periodic (Breeden Nasmyth 1987a; <br /> (Wittenberg et al. 1990; Fernandez-Sarabia et al.                           Nasmyth Dirick 1991; Ogas et al. 1991; Dirick et <br /> 1992; Tyers et al. 1993; Measday et al. 1994). Recent                       al. 1992; Lowndes et al. 1992; Breeden Mikesell <br /> global analysis transcription patterns DNA mi-                        1994; Cross et al. 1994; Stuart Wittenberg 1994). <br /> croarray hybridization reveals CLN1, CLN2, PCL1,                         The Swi6 protein interacts second DNA- <br /> and PCL2 coordinately expressed time                         binding protein, Mbp1, form distinct transcription <br /> of START large group &#226;&#181;&#8216;120 genes (Cho et                           factor, MBF (MCB-binding factor, known DSC1; <br /> al. 1998; Spellman et al. 1998). Cell cycle-dependent                       Dirick et al. 1992; Koch et al. 1993; Lowndes et al. <br /> transcription group genes governed                     1991, 1992; Verma et al. 1992). MBF/DSC1 activates <br /> heterodimeric transcription factors, called SBF                         START-specific expression SWI4 gene, the <br />                                                                             S-phase cyclin genes CLB5 CLB6, many <br />                                                                             genes needed DNA synthesis RNR1 and <br />   Corresponding author: Brenda J. Andrews, Department Molecular          TMP1 (Dirick et al. 1992; Lowndes et al. 1992; Foster <br /> and Medical Genetics, University Toronto, Rm. 4285, Medical Sci-         et al. 1993; Koch et al. 1993; Schwob Nasmyth <br /> ences Bldg., 1 King&#226;&#8364;&#8482;s College Circle, Toronto, Ontario M5S 1A8, <br /> Canada. E-mail: brenda.andrews@utoronto.ca                                  1993). SBF- MBF-dependent gene expression is <br />   1 <br />    Present address: Centre Molecular Medicine Therapeutics,         strongly activated Cln3-Cdc28 kinase cells <br /> University British Colombia, Vancouver V5Z 4H4, Canada.                  reach critical size START (Tyers et al. 1993; Dirick <br /> Genetics 154: 1561&#226;&#8364;&#8220;1576 (April 2000) <br />  1562                                              N. Macpherson et al. <br />  <br /> et al. 1995; Stuart Wittenberg 1995). However,            tionally redundant Swi6 indicated fact that <br /> the precise mechanism Cln3-Cdc28 action SBF             swi6 deletion mutants alive despite absence <br /> and MBF remains mystery. After START, repression            SBF MBF activity. As discussed above, this <br /> of G1/S program caused Cdc28 kinase          explained residual, relatively nonperiodic <br /> association B-type cyclins Clbs. Cells       expression essential targets SBF MBF swi6&#226;&#338;&#172; <br /> lack Clb1-4 unable repress SBF-dependent genes         strains. Thus, alternative pathway expression of <br /> (Amon et al. 1993) &#226;&#181;&#8216;100 transcripts repressed         SBF- MBF-driven genes important cell viability <br /> by overexpression Clb2 (Spellman et al. 1998). Re-         absence Swi6. We reasoned the <br /> pression SBF-regulated genes involve physical          redundant pathway nonfunctional SWI4 were <br /> association Swi4 Clb2 G2, Mbp1        turned off, cells lacking SWI6 die. To identify <br /> does appear interact directly Clb2 (Amon          factors maintain cell viabilility absence of <br /> et al. 1993; Siegmund Nasmyth 1996).                      SWI6, performed genetic screen mutations <br />    Although SWI4 SWI6 essential gene,       create requirement Swi6. <br /> their importance demonstrated fact a <br /> cell deleted SWI4 SWI6 arrests START <br />                                                                              MATERIALS AND METHODS <br /> (Breeden Nasmyth 1987a). This &#226;&#8364;&#339;synthetic lethal- <br /> ity&#226;&#8364;? reduced expression G1 cyclins        Yeast strains, media, genetic methods: Yeast strains used <br /> ectopic expression CLN1, CLN2, PCL1 rescues           experiments described Table 1. Synthetic mini- <br /> swi4&#226;&#338;&#172; swi6&#226;&#338;&#172; strain (Nasmyth Dirick 1991; Ogas             mal media containing glucose galactose (SD SG) and <br />                                                               rich media containing glucose galactose (YEPD YEPG) <br /> et al. 1991). Unlike SBF, MBF nonessential complex       prepared described (Sherman Hicks 1991). <br /> since swi6&#226;&#338;&#172; mbp1&#226;&#338;&#172; cell alive (Koch et al. 1993). The     5-Fluoroorotic acid (5-FOA) plates SD SG &#226;&#171;&#186;Ura plates <br /> viability strains lacking MBF likely reflects     supplemented 50 mg/l uracil 1 mg/ml 5-FOA (To- <br /> fact MCB-regulated genes downregulated           ronto Research Chemicals). Crosses, sporulation, tetrad <br /> in cells lacking MBF instead expressed constitu-      dissection performed essentially described (Sherman <br />                                                               Hicks 1991). Yeast transformations performed using <br /> tively cell cycle (Koch et al. 1993). Al-      slight modifications standard techniques (Gietz et al. 1992). <br /> though SWI6 essential gene,            NY7 (MATa slm6-1), NY11 (MATa slm7-1), NY15 (MATa <br /> the DNA-binding proteins associated Swi6 re-          slm8-1), NY34 (MAT&#226;?&#163; slm7-1) recovered meiotic <br /> quired START, swi4&#226;&#338;&#172; mbp1&#226;&#338;&#172; cell dies prior         segregants diploids formed mating NYS6 (swi6 <br /> to DNA synthesis (Koch et al. 1993).                          slm6-1 [pNM1]), NYS7 (swi6 slm7-1 [pNM1]), NYS8 (swi6 <br />                                                               slm8-1 [pNM1]) wild-type strain (BY109). Segregants were <br />    One implication observations        tested growth medium lacking histidine death at <br /> must substantial cross talk SBF MBF        37&#226;&#172;&#353; YEPD. For construction strain NY36 (TAF17::LEU2), <br /> pathways vivo. Indeed, MBF complex formation           plasmid pNM11, contains genomic fragment carrying <br /> be specifically competed SCB oligomers vitro, indi-     TAF17 gene integrating vector (see description be- <br /> cating MBF bind SCBs (Lowndes et al.         low), cleaved insert direct homologous inte- <br />                                                               gration transformed wild-type strain ( JO34). As a <br /> 1991; Dirick et al. 1992). In addition, SBF binds          result, site integration TAF17 gene marked <br /> MCB elements CLN1 promoter vitro          plasmid-borne LEU2 gene. <br /> elements depend Swi4 Swi6 vivo               Plasmids: To construct plasmid pNM1, XhoI-BglII frag- <br /> function (Partridge et al. 1997). It important note,    ment containing SWI6 gene (Breeden Nasmyth <br /> however, vivo footprinting experiments reveal         1987b) cloned XhoI-BamHI sites plasmid pJO14. <br />                                                               The pJO14 plasmid CEN/ARS-based shuttle vector derived <br /> that SCB sequences detectably protected          pRS316 (Sikorski Hieter 1989) URA3 se- <br /> chemical modification swi4&#226;&#338;&#172; swi6&#226;&#338;&#172; strains (Har-         lectable marker GAL1-10 promoter inserted the <br /> rington Andrews 1996; Koch et al. 1996). In           polylinker sequences. To construct LEU2-SWI4 plasmid, a <br /> absence Swi6, Swi4 Mbp1 bind weakly             XhoI-SalI fragment containing SWI4 gene flanking <br /> SCB MCB elements function activate tran-           sequences isolated clone B3.2 (Andrews Her- <br />                                                               skowitz 1989b) cloned SalI site pUC18Bgl2 <br /> scription levels sufficient cell viability.            (a derivative pUC18 BglII linkers inserted EcoRI <br />    A great deal evidence suggests existence         HindIII sites) create plasmid pBA313. The SWI4 gene <br /> proteins functionally redundant interact     isolated BglII fragment pBA313 cloned <br /> Swi6. For example, transcription SWI4 gene itself      BamHI-digested pRS315 create plasmid pBA417. The <br /> is cell cycle-regulated peak expression occurring        pNM10 plasmid constructed inserting genomic <br />                                                               BamHI fragment containing TAF17 BamHI site of <br /> slightly HO (Breeden Mikesell              pRS315. The pNM11 plasmid constructed inserting <br /> 1991). Cell cycle-regulated expression SWI4 par-        genomic BamHI fragment containing TAF17 into <br /> tially controlled MCB sequences, substantial         pRS305 integrating vector (Sikorski Hieter 1989). <br /> contribution regulation MCB independent               Mutagenesis: Strain NY3 mutagenized exposure to <br /> yet dependent SWI6 (Foster et al. 1993). This        ethyl methanesulfonate (EMS; Sigma Chemical Co., St. Louis). <br />                                                               Twelve cultures NY3 individual colonies grown <br /> observation suggests existence factor     overnight 2 ml YEPG. The cultures washed, resus- <br /> may interact Swi6 regulate cell cycle-dependent       pended 1.5 ml 0.1 m sodium phosphate buffer pH 7.0, <br /> transcription. The existence regulators func-     sonicated 10 sec separate clumpy cells. An aliquot <br />                                                           TAF17 SWI6                                                        1563 <br />  <br />                                                               TABLE 1 <br />                                                   Yeast strains used study <br />  <br /> Strain                                            Genotype                                                        Source <br /> BY108        MAT&#226;?&#163; swi4&#226;&#338;&#172;::HIS3 TRP1 ura3 leu2 lys2 ade2                                                  Andrews lab collection <br /> BY109        MAT&#226;?&#163; TRP1 his3 ura3 leu2 lys2 ade2                                                         Andrews lab collection <br /> BY147        MAT&#226;?&#163; swi4&#226;&#338;&#172;::HIS3 swi6&#226;&#338;&#172;::LEU2 his3&#226;&#338;&#172;::scb-lacZ trp1 ura3 lys2 ade2 [pGAL::SWI4 TRP1]         Andrews lab collection <br /> BY167        MATa swi4&#226;&#338;&#172;::HIS3 bar1&#226;&#338;&#172;::LEU2 HO::lacZ trp1 ura3 lys2 ade2                                  Andrews lab collection <br /> BY168        MATa swi6&#226;&#338;&#172;HIS3 bar1&#226;&#338;&#172;LEU2 HO::lacZ trp1 ura3 lys2 ade2                                      Andrews lab collection <br /> JO34         MATa TRP1 ura3 leu2 his3 lys2 ade2                                                         Ogas et al. (1991) <br /> NY3          MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ TRP1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                             This study <br /> NY7          MATa slm6-1 SWI6 TRP1 ura3 leu2 lys2 ade2                                                  This study <br /> NY11         MATa slm7-1 SWI6 TRP1 ura3 leu2 lys2 ade2                                                  This study <br /> NY15         MATa slm8-1 SWI6 TRP1 ura3 leu2 lys2 ade2                                                  This study <br /> NY34         MAT&#226;?&#163; slm7-1 SWI6 trp1 ura3 leu2 lys2 ade2                                                  This study <br /> NY36         MATa TAF17::LEU2 TRP1 ura3 leu2 lys2 ade2                                                  This study <br /> NYS1         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm1-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS2         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm2-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS3         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm3-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS4         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm4-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS5         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm5-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS6         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm6-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS7         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm7-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS8         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm8-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS9         MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm9-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                           This study <br /> NYS10        MAT&#226;?&#163; swi6&#226;&#338;&#172;::HIS3 HO::lacZ slm10-1 ura3 leu2 lys2 ade2 GAL2 [pNM1]                          This study <br /> YSW93        MATa TRP1 ura3-52 taf145&#226;&#338;&#172;::LEU2 pRS313-taf145ts-2                                          Walker et al. (1996) <br /> Y260         MATa trp1::hisG ura3-52 rpb1-1                                                             Gift Jim Friesen <br />                                                                                                           (University Toronto) <br />  <br />  <br />  <br /> (200 &#226;?&#174;l) removed serve untreated control            null <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>mutation</span> SWI4, synthetically lethal with <br /> 20 &#226;?&#174;l EMS added remaining culture. The cells           swi6&#226;&#338;&#172; 25&#226;&#172;&#353;, causes mutant cells die 37&#226;&#172;&#353;, even <br /> were incubated aeration 30&#226;&#172;&#353; 70 min washed            presence wild-type SWI6 (Breeden Nasmyth <br /> in 1.5 ml 5% sodium thiosulfate inactivate EMS. The        1987a; Ogas et al. 1991). Growth YEPG 37&#226;&#172;&#353; also <br /> cells washed twice 1.5 ml water, resuspended         eliminate temperature-sensitive mutations sup- <br /> in 1 ml YEPG, plated YEPG plates (100 &#226;?&#174;l/plate).         pressed high levels Swi6. <br /> This treatment resulted &#226;&#181;&#8216;50% cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span>.                          Cells did contain pNM1 plasmid, contained <br />    Assays mutant phenotypes: The colonies derived            mutations URA3, dying reasons Swi6 <br /> survivors EMS treatment replica plated              levels. Mutant isolates failed grow medium lacking <br /> 5-FOA-containing plates YEPD plates. One set plates       uracil studied further. Mutants tested for <br /> was incubated 25&#226;&#172;&#353; 37&#226;&#172;&#353; 3 days. Colonies       rescue FOA sensitivity transformation SWI6 plas- <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> 5-FOA-containing YEPD plate replica           mid carrying LEU2 nutritional marker; isolates <br /> plated YEPD incubated 2 days tempera-         rescued transformation SWI6-LEU2 plasmid were <br /> ture previous plates. Colonies did grow        studied. <br /> treatment 5-FOA dead cells              Complementation allelism tests: The original MAT&#226;?&#163; <br /> survive Swi6. Cells failed grow YEPD died         swi6&#226;&#338;&#172; slm isolates mated BY168, isogenic MATa <br /> because levels Swi6 produced          swi6&#226;&#338;&#172; strain (Table 1). The mutation judged domi- <br /> the swi6&#226;&#338;&#172; slm mutant alive cells            nant diploid dead nonpermissive conditions. <br /> metabolize glucose. Colonies did grow sec-        MATa swi6&#226;&#338;&#172; slm mutant segregants isolated these <br /> ond incubation streak purified original YEPG           diploids analyzing meiotic progeny. The MATa swi6&#226;&#338;&#172; slm <br /> plates patched YEPG 25&#226;&#172;&#353;. The resulting patches           isolates mated MAT&#226;?&#163; swi6&#226;&#338;&#172; strains containing differ- <br /> were tested growth 5-FOA plates containing galactose          ent slm mutation. The resulting diploids tested under <br /> as carbon source. Mutants grew plates         restrictive minimal conditions death of <br /> judged defective glucose metabolism            mutants (25&#226;&#172;&#353; YEPD restrictive 37&#226;&#172;&#353; <br /> studied further.                                                     5-FOA restrictive) assay complementation <br />    Mutants died YEPD 5-FOA-containing medium              slm mutant phenotype. If diploid strain unable <br /> at 37&#226;&#172;&#353; assayed growth YEPG 37&#226;&#172;&#353;. Mutants          survive restrictive conditions, slm mutants <br /> died conditions judged temperature            judged complementation group. <br /> sensitive presence high levels Swi6           Representatives complementation group back- <br /> studied further. Mutants died 25&#226;&#172;&#353; assayed        crossed times BY165 (wild type) obtain swi6&#226;&#338;&#172; slm <br /> this manner, mutation synthetically lethal      strains likely carry confounding secondary mutations. <br /> swi6&#226;&#338;&#172; 25&#226;&#172;&#353; result temperature sensitivity,        The resulting swi6&#226;&#338;&#172; slm strains transformed pNM1 are <br /> the presence Swi6. An example type mutation         designated NYS1 NYS10 contain mutations in <br />  1564                                                     N. Macpherson et al. <br />  <br /> slm1 slm10, respectively (see Table 1). The number           colonies survived 37&#226;&#172;&#353; rescue temperature- <br /> of mutant strains complementation group              sensitive phenotype NY11 confirmed retransforma- <br /> following: slm1, 6 isolates; slm2, 6 isolates; slm3, 17 isolates;    tion NY11 (slm7-1) appropriate plasmid. <br /> slm4, 2 isolates; slm5, 7 isolates; slm6, 18 isolates; slm7, 27         The genomic insert NY11-complementing <br /> isolates; slm8, 15 isolates; slm9, 26 isolates; slm10, 4 isolates.   clones sequenced fragment containing TAF17 was <br />    To test allelism slm mutations SWI4, NYS4 NYS6        subcloned. Plasmid pNM10 (pRS315-TAF17) trans- <br /> were mated strain BY167 (swi4&#226;&#338;&#172;). The resulting diploid        formed NY11 (slm7-1) NYS7 (slm7-1 swi6&#226;&#338;&#172;). These <br /> sporulated segregation lethality meiotic progeny       transformants tested growth YPD 37&#226;&#172;&#353; on <br /> was assessed. The allelism TAF17 slm7 tested          5-FOA 25&#226;&#172;&#353;, respectively. Other rescuing plasmids tested <br /> mating NY34 (MAT&#226;?&#163; slm7-1) NY36 (MATa TAF17::LEU2)                 PCR determine contained TAF17. Primers <br /> and monitoring segregation leucine auxotrophy             used amplify TAF17 5&#226;&#172;&#732; primer (T17-1), GAGGATCCT <br /> temperature sensitivity (see results).                               TATGAACGGCGGAGG 3&#226;&#172;&#732; primer (T17-2), GCGGATCC <br />    Cell cycle analysis: For time course studies shown Figure      TCACATAGACTTTGGG. To confirm rescue slm7 mu- <br /> 3A, logarithmically growing cells NYS6, NYS7, NYS8,            tant phenotype, NY11 (slm7-1) transformed pZM255 <br /> BY147 growing SG medium 25&#226;&#172;&#353; collected, resus-             (Moqtaderi et al. 1996a) pNM10 (pRS315-TAF17). <br /> pended OD600 0.2 SD medium, grown 37&#226;&#172;&#353;.            pZM255 plasmid containing TAF17 gene control <br /> In Figure 3B, log phase cultures NY7, NY11, NY15, BY108,          GAL promoter. The resulting transformants spot- <br /> and JO34 SD medium 30&#226;&#172;&#353; diluted OD600             ted glucose- galactose-containing media grown at <br /> 0.2 grown SD medium 37&#226;&#172;&#353;. Samples removed              30&#226;&#172;&#353; 37&#226;&#172;&#353; 2 days (glucose) 6 days (galactose). <br /> from cultures 2 hr, fixed, prepared FACS              Sequencing slm7-1: The TAF17 locus NY11 ampli- <br /> analysis (Measday et al. 1994). Where indicated, fixed cells         fied PCR using primers T17-1 T17-2 (see above). Both <br /> were treated 10 &#226;?&#174;g/ml DAPI (4&#226;&#172;&#732;,6-diamidino-2-phenylin-          strands different PCR fragments resulting this <br /> dole; Sigma Chemical Co.) washed times          amplification sequenced using primers T17-1 T17-2. <br /> water. The fixed cells photographed 400&#226;&#171;&#187; magnifica- <br /> tion Nikon Microphot-FXA Kodak TX-400 film or <br /> with CCD camera mounted Leica DM-LB microscope. <br /> Images CCD camera analyzed using Northern                                      RESULTS <br /> Exposure Imaging System (Empix Imaging, Inc., Mississauga, <br />                                                                         Screen swi6-dependent lethal mutants: The Swi6 <br /> Ontario). DAPI images obtained using 355&#226;&#8364;&#8220;375-nm <br /> excitation filter 400-nm long pass barrier filter.             protein plays important regulatory role tran- <br />    Northern analysis: Logarithmically growing cells JO34          scription factors, SBF MBF, required the <br /> (wild type) NY11 (slm7-1) treated 165 min          induction gene expression START S. cerevisiae. <br /> 3 &#226;?&#174;m &#226;?&#163;-factor, washed ice-cold water, resuspended          Despite fact genes controlled by <br /> YEPD 25&#226;&#172;&#353;. Samples collected 10-min intervals over <br />                                                                      SBF MBF essential cell cycle progression, a <br /> 200 min JO34 15-min intervals 300 min for <br /> NY11, doubling time NY11 &#226;&#181;&#8216;1.5&#226;&#171;&#187; wild          swi6 deletion strain viable. As such, likely <br /> type. Total RNA extracted samples previously       gene(s) pathway(s) exists yeast cells that <br /> described (Measday et al. 1997). An aliquot 20 &#226;?&#174;g RNA          partially compensates lack SWI6 function. <br /> from sample run MOPS/formaldehyde gel                 To identify genes involved parallel pathway, <br /> transferred nylon membrane. The RNA immobilized <br />                                                                      identify genes functioning the <br /> on membrane UV crosslinking 120 mJ/cm2 a <br /> Fisher Scientific UV crosslinker. Immobilized RNA probed         pathway SBF, devised synthetic lethal <br /> as described (Measday et al. 1997) PCL1, PCL2, SWI5,             screen mutants require SWI6 viability. This <br /> CLN2, ACT1, PCL9 transcripts. NY11 wild-type RNA             strategy useful identification genes in- <br /> immobilized fresh membrane probed CLB5, PCL1,           volved regulating common process (Bender and <br /> and ACT1. All membranes treated identically         Pringle 1991; Costigan et al. 1992; Guarente 1993; <br /> entire procedure. The resulting membranes exposed to <br /> a phosphorimager screen 16&#226;&#8364;&#8220;24 hr screen              Holtzman et al. 1993; Igual et al. 1996). Our strategy <br /> scanned using Molecular Dynamics (Sunnyvale, CA) phos-             identifying genes support growth strain <br /> phorimager. The transcripts quantitated Image-             lacking Swi6 mutagenize swi6&#226;&#338;&#172; strain trans- <br /> QuaNT (v4.0) software. Individual bands quantitated          formed pNM1, centromere-based plasmid that <br /> drawing box band using identical box             contains URA3 nutritional marker SWI6 <br /> in lane background. All bands normalized <br /> to ACT1 transcript resulting data plotted           gene expressed inducible GAL1-10 promoter. <br /> using Microsoft Excel.                                               When swi6&#226;&#338;&#172; [pNM1] strain (NY3, Table 1) grown <br />    Measuring total mRNA mutant cells: Wild type (BY109),          glucose-containing medium, expression SWI6 <br /> slm7-1 (NY11), taf145ts (YSW93), rbp1-1 (Y260) cells        GAL promoter repressed extent <br /> grown OD600 &#226;&#171;&#189; 0.2 30&#226;&#172;&#353; shifted 37&#226;&#172;&#353;. Aliquots        Swi6 undetectable Western blotting using <br /> were taken 0, 1, 2, 4 hr transfer 37&#226;&#172;&#353;. RNA was <br /> isolated pellets (see above), 10 &#226;?&#174;g RNA       anti-Swi6 antiserum (data shown). However, more <br /> each sample spotted nylon membrane (Hybond N&#226;&#171;&#185;,            sensitive biological assay revealed residual SWI6 <br /> Amersham, Buckinghamshire, UK) using hybridot manifold             expression glucose-repressed GAL1-10 pro- <br /> (BRL). The blot washed probed 32P-labeled          moter pNM1 plasmid. Whereas swi4&#226;&#338;&#172; swi6&#226;&#338;&#172; <br /> oligo(dT)20 described Walker et al. (1997).                    [pGAL::SWI4] strain (BY147) dies glucose-contain- <br />    Cloning SLM7: A yeast genomic library (Engebrecht et <br /> al. 1990) transformed strain NY11 (slm7-1)          ing media inadequate expression G1 cyclins <br /> resulting transformants replica plated SD medium             (Nasmyth Dirick 1991), same <br /> and grown 37&#226;&#172;&#353; 5 days. Plasmids isolated             strain transformed pNM1 alive glucose-con- <br />                                                          TAF17 SWI6                                                      1565 <br />  <br />  <br />  <br />  <br />                                                                     Figure 2.&#226;&#8364;&#8221;Rescue NYS4 NYS6 SWI4 SWI6. <br />                                                                  NYS4 NYS6 strains transformed pBA417 (SWI4) <br />                                                                  pBA354 (SWI6). The resultant transformants grown <br />                                                                  OD600 &#226;&#171;&#189; 0.1 serial dilutions spotted YEPD <br />                                                                  25&#226;&#172;&#353; 37&#226;&#172;&#353; grown 6 days (25&#226;&#172;&#353;) 2 days (37&#226;&#172;&#353;). <br />  <br />  <br />                                                                  minimal conditions required lethality (Figure 1, data <br />                                                                  shown). We mated 162 strains parental <br />                                                                  swi6&#226;&#338;&#172; strain mutations <br />                                                                  tested recessive. <br />    Figure 1.&#226;&#8364;&#8221;slm mutants require SWI6 viability. The indi-      To determine number loci repre- <br /> cated cultures incubated 25&#226;&#172;&#353;, serial dilutions of <br /> log phase cultures spotted YEPG-, YEPD-, and <br />                                                                  sented 161 recessive SWI6-requiring mutant <br /> 5-FOA-containing media. The plates containing strains       strains, complementation test performed. Diploids <br /> show synthetic lethality 25&#226;&#172;&#353; incubated 25&#226;&#172;&#353;,   created pairwise crossing haploid mutants (see <br /> those containing strains synthetic lethality 37&#226;&#172;&#353;    materials methods) tested ability to <br /> were incubated 37&#226;&#172;&#353;. NYS&#226;&#8364;&#339;X&#226;&#8364;? indicates slmX swi6&#226;&#338;&#172; [pNM1]      grow absence Swi6. Of 161 slm strains, 128 <br /> strain, BY147 swi4&#226;&#338;&#172; swi6&#226;&#338;&#172; [pGAL::SWI4], BY109 wild type, <br /> and NY3 parental swi6&#226;&#338;&#172; strain. <br />                                                                  fell 10 complementation groups designated SLM1 <br />                                                                  SLM10. The 34 mutants did fit into <br />                                                                  10 groups multiple comple- <br /> taining media, indicating residual low level         mentation groups members of the <br /> Swi6 produced pNM1 sufficient comple-                10 complementation groups. Representatives each <br /> ment deletion swi6 swi4&#226;&#338;&#172; mutant background.              complementation group backcrossed iso- <br /> Given observation, mutagenized swi6&#226;&#338;&#172; [pNM1]              genic swi6&#226;&#338;&#172; [pNM1] strain. The resulting strains con- <br /> cells screened Swi6-dependent viability           taining mutations SLM1 SLM10 desig- <br /> ways: (1) glucose-containing medium decrease               nated NYS1 NYS10 (Figure 1). <br /> expression SWI6 GAL promoter pNM1                    Rescue slm mutants SWI4: We expected isolate <br /> to low levels (2) 5-FOA-containing medium            mutants defective SWI4 screen, swi4&#226;&#338;&#172; <br /> to select cells lost URA3-containing            swi6&#226;&#338;&#172; strains inviable (Breeden Nasmyth <br /> plasmid (Boeke et al. 1984) completely eliminate          1987a). To assay slm isolates potential SWI4 muta- <br /> all Swi6 coding sequences cell.                         tions, slm swi6&#226;&#338;&#172; strains tested suppression of <br />    Initial characterization slm mutants: We screened          swi6-dependent synthetic lethality transformation <br /> &#226;&#181;&#8216;40,000 colonies 12 independent pools strain             plasmid carrying SWI4 gene (Figure 2 and <br /> NY3 treated EMS ability grow YEPD           data shown). Viability restored NYS4 and <br /> (glucose-containing) 5-FOA-containing medium. Col- <br /> onies tested ability grow both <br /> high (37&#226;&#172;&#353;) low (25&#226;&#172;&#353;) temperatures allow iden-                                    TABLE 2 <br /> tification temperature-sensitive slm mutants. In            Segregation slm alleles potentially allelic genes <br /> way, total 244 colonies identified failed <br /> to grow YEPD 5-FOA 25&#226;&#172;&#353; 37&#226;&#172;&#353;. Of mutant                                            Segregation synthetic <br /> isolates, 58 defective glucose metabolism, 20                                             lethality (viable : inviable) <br /> carried mutations conferring temperature-sensitive le-           Diploid                        2:2           3:1            4:0 <br /> thality independent Swi6, 4 unable <br />                                                                  swi6&#226;&#338;&#172; slm4-1 &#226;&#171;&#187; swi4&#226;&#338;&#172;            10            0                0 <br /> to grow medium lacking uracil (see materials              swi6&#226;&#338;&#172; slm6-1 &#226;&#171;&#187; swi4&#226;&#338;&#172;            4             6                2 <br /> methods). The remaining 162 isolates showed different <br />  1566                                               N. Macpherson et al. <br />  <br />  <br />  <br />  <br />                                                                                                Figure 3.&#226;&#8364;&#8221;Analysis arrest <br />                                                                                             phenotypes slm mutants. (A) <br />                                                                                             Strains BY147 (swi4&#226;&#338;&#172; swi6&#226;&#338;&#172; <br />                                                                                             pGAL::SWI6), NYS6 (swi6&#226;&#338;&#172; <br />                                                                                             slm6-1 pGAL::SWI6), NYS7 <br />                                                                                             (swi6&#226;&#338;&#172; slm7-1 pGAL::SWI6), <br />                                                                                             NYS8 (swi6&#226;&#338;&#172; slm8-1 <br />                                                                                             pGAL::SWI6) grown to <br />                                                                                             log phase galactose-con- <br />                                                                                             taining medium then <br />                                                                                             shifted glucose-containing <br />                                                                                             medium 37&#226;&#172;&#353; (nonpermissive <br />                                                                                             conditions) 8 hr. Samples <br />                                                                                             harvested 2 hr, fixed <br />                                                                                             stained propidium io- <br />                                                                                             dide FACS analysis (top of <br />                                                                                             A) DAPI fluores- <br />                                                                                             cence microscopy. Photomi- <br />                                                                                             crographs DAPI-stained <br />                                                                                             cells Nomarski images of <br />                                                                                             cells shown [DAPI and <br />                                                                                             DIC (differential interference <br />                                                                                             contrast) indicated right of <br />                                                                                             panel]. Time incubation at <br />                                                                                             37&#226;&#172;&#353; glucose shown the <br />                                                                                             right FACS profiles (0, <br />                                                                                             2, 4, 6, 8 hr) left <br />                                                                                             photomicrographs (0, 4, <br />                                                                                             8 hr). (B) FACS profile of <br />                                                                                             NY7 (slm6-1 SWI6), NY11 <br />                                                                                             (slm7-1 SWI6), NY15 (slm8-1 <br />                                                                                             SWI6), BY108 (swi4&#226;&#338;&#172; SWI6), <br />                                                                                             JO34 (wild type) grown to <br />                                                                                             log phase 30&#226;&#172;&#353; glucose and <br />                                                                                             shifted 37&#226;&#172;&#353;. Cells were <br />                                                                                             treated described for <br />                                                                                             FACS analysis photo- <br />                                                                                             graphed Nomarski optics. <br />  <br />  <br />  <br />  <br /> NYS6 transformed centromere-             meiotic segregants tested viability on <br /> derived plasmid containing SWI4 gene,         5-FOA-containing medium 25&#226;&#172;&#353; (the restrictive condi- <br /> transformed vector (Figure 2). To test              tions slm6 allele used cross). Some the <br /> whether slm4 mutant harbored mutation SWI4,           tetrads yielded progeny 3:1 4:0 segregation of <br /> we crossed slm4 swi6&#226;&#338;&#172; [pNM1] strain swi4&#226;&#338;&#172; SWI6          viability:lethality, indicating recovery SLM6 and <br /> strain (NYS4 &#226;&#171;&#187; BY167) subjected diploids            SWI4 recombinants (Table 2). We conclude that, al- <br /> tetrad analysis. Of 10 tetrads analyzed, viable    slm6 swi6&#226;&#338;&#172; lethality rescued ectopic <br /> and inviable meiotic segregants tested             expression SWI4, slm6 strain mutated locus <br /> 5-FOA-containing medium select plasmid-         unlinked SWI4. <br /> borne SWI6 gene (Table 2). This 2:2 segregation pat-              Cell cycle phenotypes slm mutants: If alterna- <br /> tern, reflects lack recombination swi4        tive pathway expression SBF- MBF-driven genes <br /> and slm4, showed SLM4 SWI4                impaired slm mutant, swi6&#226;&#338;&#172; slm double <br /> allelic.                                                       mutant expected specific cell cycle- <br />    We performed similar segregation test slm6         arrest phenotype. Since particularly interested <br /> mutant strain crossing slm6 swi6&#226;&#338;&#172; [pNM1] strain           mutants identify pathways func- <br /> to swi4&#226;&#338;&#172; SWI6 strain (NYS6 &#226;&#171;&#187; BY167). In case, 12      tion Swi6 regulate gene expression at <br /> tetrads recovered galactose-containing medium          START, used FACS analysis examine arrest <br />                                                        TAF17 SWI6                                                 1567 <br />  <br />  <br />  <br />  <br />                                                                                               Figure 3.&#226;&#8364;&#8221;Continued. <br />  <br />  <br />  <br />  <br /> phenotype swi6&#226;&#338;&#172; slm mutants [slm4 (swi4)            slm7-1 swi6&#226;&#338;&#172; slm8-1 swi6&#226;&#338;&#172; mutants non- <br /> mutants examined; slm2 slm5 mutant                permissive conditions showed cells G2. The <br /> strains extremely clumpy reproducible FACS            nonuniformity arrest reflect residual SWI6 <br /> profiles difficult obtain]. Four slm mutants        SLM gene function allowing DNA replication some <br /> not cell cycle-specific arrest: slm1, slm3, slm9,   cells culture reflect uncoupling of <br /> slm10 (data shown). Neither FACS profiles          bud morphogenesis DNA replication pathways <br /> the budding index mutants altered            cells lacking Swi6 Slm7 Slm8. In case, the <br /> cells arrested nonpermissive conditions             arrest phenotype consistent redundant role for <br /> (data shown).                                              SLM7 SLM8 SWI6 regulating G1 progression. <br />    Three slm swi6&#226;&#338;&#172; mutants did cell cycle-        DAPI staining did reveal obvious nuclear defects <br /> arrest phenotype incubated restrictive con-         arresting swi6 slm7 swi6 slm8 cells. <br /> ditions. Both slm7 slm8 strains showed uniform              In contrast slm7 slm8, FACS profile the <br /> morphological arrest 85&#226;&#8364;&#8220;92% large, unbudded               slm6 swi6&#226;&#338;&#172; double mutants showed shift larger <br /> cells cultures 8-hr incubation          G2 peak nonpermissive conditions (Figure 3A, <br /> nonpermissive conditions (467/508 unbudded cells           slm6-1 swi6). After 4 hr arrest conditions, the <br /> slm7 435/514 unbudded cells slm8; Figure 3A,           NYS6 cells multiply budded two <br /> slm7-1 swi6, slm8-1 swi6). Arrest large, unbudded cells     nuclei visible DAPI staining. After 8 hr, majority <br /> is typical cells unable execute START,           swi6&#226;&#338;&#172; slm6 cells arrested uninucleate cells <br /> those deleted SWI4 SWI6 (swi4 swi6, Figure        elongated buds (Figure 3A, slm6-1 swi6, 8 hr). Some <br /> 3A).                                                           cells did appear nuclei, at <br />    Although morphological arrest phenotype          permissive temperatures, indicating possible nuclear <br /> slm7 slm8 cells uniform, FACS profile           migration defect swi6&#226;&#338;&#172; slm6 cells. <br />  1568                                             N. Macpherson et al. <br />  <br />  <br />  <br />  <br />                                                                                           Figure 4.&#226;&#8364;&#8221;Northern analysis <br />                                                                                        transcripts NY11 (slm7-1) and <br />                                                                                        JO34 (wild-type) cells following re- <br />                                                                                        lease cell cycle &#226;?&#163;-fac- <br />                                                                                        tor block. (A) Aliquots wild-type <br />                                                                                        cells taken 10 min and <br />                                                                                        aliquots slm7-1 SWI6 cells were <br />                                                                                        taken 15 min following re- <br />                                                                                        lease &#226;?&#163;-factor block. The <br />                                                                                        blots probed PCL2 and <br />                                                                                        SWI5 transcripts, stripped and <br />                                                                                        probed CLN2 ACT1 tran- <br />                                                                                        scripts, stripped and <br />                                                                                        probed PCL9 transcripts. A <br />                                                                                        separate set blots probed <br />                                                                                        CLB5 transcripts, then <br />                                                                                        stripped probed ACT1 <br />                                                                                        transcripts. The membranes con- <br />                                                                                        taining NY11 JO34 RNA were <br />                                                                                        probed identical condi- <br />                                                                                        tions identical probes. The <br />                                                                                        blots shown representative of <br />                                                                                        independent experiments. <br />                                                                                        (B) Quantification tran- <br />                                                                                        scripts A. All bands nor- <br />                                                                                        malized ACT1 transcript. <br />                                                                                        The intervals samples <br />                                                                                        taken following release from <br />                                                                                        cell cycle block indicated on <br />                                                                                        abscissa. The upper values of <br />                                                                                        abscissa indicate times <br />                                                                                        samples taken from <br />                                                                                        cells containing slm7-1 muta- <br />                                                                                        tion (slm7), lower val- <br />                                                                                        ues indicate times sam- <br />                                                                                        ples taken wild-type <br />                                                                                        cells (wt). <br />  <br />  <br />  <br />  <br />    For cell cycle analysis slm swi6 [pGAL::SWI6]      fect wild-type (SWI6) background. For <br /> mutants, changes growth condi-         slm7-1 strain, result unexpected since <br /> tions: carbon source cells changed      slm7-1 swi6 isolates showed synthetic lethality low <br /> galactose glucose temperature raised          temperature (25&#226;&#172;&#353;) originally screened for <br /> from 25&#226;&#172;&#353; 37&#226;&#172;&#353;. To eliminate effects changing        temperature sensitivity galactose-containing medium <br /> both carbon source level Swi6 protein,        (see materials methods Figure 1). However, <br /> we examined slm mutants wild-type        slm8 swi6&#226;&#338;&#172; mutant showed synthetic lethality only <br /> background (slm SWI6). We recovered meiotic segreg-          high temperature double mutant strain grew <br /> ants diploids formed mating NYS6, 7, 8        galactose-containing medium 37&#226;&#172;&#353; (see Figure <br /> wild-type (JO34) cells. The slm6-1 SWI6 strain (NY7)         1). We conclude temperature-sensitive pheno- <br /> was temperature sensitive showed cell cycle-      type slm8-1 mutant strain suppressed over- <br /> specific defects (Figure 3B). In contrast, spores       expression SWI6 galactose-containing medium at <br /> carried wild-type SWI6 gene (His&#226;&#171;&#186;) tem-     37&#226;&#172;&#353; (Figure 1, NYS8). We saw suppression when <br /> perature sensitive viability (on YEPD 37&#226;&#172;&#353;)       high-copy (2&#226;?&#174;) plasmid containing SWI6 used to <br /> recovered frequency consistent slm7-1        transform slm8 strain (data shown). <br /> slm8-1 strains having temperature-sensitive growth de-        When incubated restrictive temperature, the <br />                                                      TAF17 SWI6                                              1569 <br />  <br />  <br />  <br />  <br />   Figure 4.&#226;&#8364;&#8221;Continued. <br />  <br />  <br />  <br /> slm7-1 SWI6 strain (NY11) showed similar FACS profile      accumulation cells 1N DNA content, the <br /> to wild-type control strain (JO34, Figure 3B),    majority cells arresting 2 hr (Figure 3B). <br /> uniform morphological arrest observed. Therefore,        This 1N accumulation accompanied mor- <br /> the G1 arrest phenotype observe slm7-1        phological arrest, cells continued bud <br /> swi6 double mutant low temperature requires muta-         did appreciably larger contin- <br /> tion slm7 swi6. By contrast, FACS analysis       ued incubation restrictive temperature. We con- <br /> of culture slm8-1 SWI6 cells (NY15) showed           clude mutation SLM8 causes defect S-phase <br />  1570                                                 N. Macpherson et al. <br />  <br /> progression results G1 arrest phenotype the <br /> absence Swi6. <br />    Analysis cell cycle-regulated gene expression a <br /> slm7 mutant: In synthetic lethal screen SWI6, <br /> we expected isolate genes that, like SWI6, in- <br /> volved controlling cell cycle-regulated transcription. <br /> We examined transcript levels cell cycle- <br /> regulated genes slm6-1, slm7-1, slm8-1 strains. We <br /> analyzed gene expression permissive temperature <br /> (25&#226;&#172;&#353;) defects gene expression seen per- <br />                                                                     Figure 5.&#226;&#8364;&#8221;RNA polymerase II-dependent transcription in <br /> missive growth conditions likely reflect true <br />                                                                  slm7-1. Wild type, slm7-1, taf145ts, rbp1-1 strains grown <br /> gene expression defect opposed secondary phe-            log phase 30&#226;&#172;&#353; shifted 37&#226;&#172;&#353;. RNA samples taken <br /> notype growth arrest.                                     0, 1, 2, 4 hr temperature shift. A total 10 &#226;?&#174;g <br />    Yeast cells arrested START mating phero-         total RNA dot blotted nylon membrane and <br /> mone (&#226;?&#163;-factor), released fresh medium, RNA             probed 32P-labeled oligo(dT)20 Walker et al. (1997). <br /> was prepared samples cells culture <br /> progressed synchronously cell cycle. To <br /> accommodate longer cell cycle slm7 mutant,            factor Taf145 (taf145ts) general transcrip- <br /> we harvested cells longer period time sam-          tional defect specific defect cyclin <br /> pled 15 min, instead 10 min wild-         transcription (Walker et al. 1997). Total RNA ex- <br /> type culture. Northern blot analysis RNA samples              tracted cells taken arresting cultures at <br /> showed slm7-1 SWI6 strain displayed defects        0, 1, 2, 4 hr shift nonpermissive tempera- <br /> in cell cycle-regulated transcription (Figure 4 data         ture. Levels total polyA&#226;&#171;&#185; mRNA assessed by <br /> not shown slm8-1, slm6-1). The maximal level              hybridization 32P-labeled oligo(dT)20 10 &#226;?&#174;g total <br /> expression cell cycle-regulated gene PCL2,          RNA spotted nylon membrane (Figure 5). While <br /> expression depends SBF, CLB5, expres-            rbp1-1 strain showed rapid decrease general <br /> sion depends MBF, reproducibly reduced                 transcription, slm7-1 taf145ts strains did an <br /> START cells containing slm7-1 mutation (peak              equivalent drop transcript levels. This result suggests <br /> expression reduced 2.5-fold CLB5 2.3-fold          temperature sensitivity slm7-1 strain is <br /> for PCL2). However, maximal level expression              general defect gene expression but <br /> of CLN2, PCL1 (SBF-dependent), PCL9 (Swi5-               specific deregulation subset impor- <br /> dependent) genes normal (Figure 4, data shown            tant genes includes CLB5 PCL2. <br /> for PCL1). We examined cell cycle-regulated ex-                Rescue slm7-1: We interested identifying <br /> pression G2-regulated gene SWI5 control            mutated genes slm strains showed a <br /> changes synchrony wild-type slm7              defect cell cycle-regulated transcription fo- <br /> SWI6 mutant cultures. The levels SWI5 transcript indi-        cused attention SLM7. To clone SLM7, trans- <br /> cated wild-type slm7 SWI6 cultures             formed slm7-1 (NY11) strain genomic library <br /> progressed synchronously cell cycles, al-            (Engebrecht et al. 1990) identified 17 plasmids <br /> though loss synchrony observed sec-           allowed slm7-1 cells grow 37&#226;&#172;&#353;. Although not <br /> ond cell cycle slm7-1 SWI6 culture.                       NY11-rescuing plasmids identical, PCR <br />    A slm7-1 swi6&#226;&#338;&#172; [pNM1] strain dies presence          analysis revealed contained gene <br /> 5-FOA 25&#226;&#172;&#353; (Figure 1) arrests G1 phase high          encoding TBP-associated factor Taf17 (data not <br /> temperature (Figure 3A). However, slm7-1 SWI6 cell             shown). We subcloned fragment encompassing the <br /> does cell cycle-specific defect 37&#226;&#172;&#353; (Figure        entire open reading frame TAF17 adjacent <br /> 3B). It possible lack cell-cycle specific       genomic DNA yeast vector (pNM10). The plasmid <br /> arrest slm7-1 SWI6 restrictive temperature         carrying subcloned fragment rescued tempera- <br /> indicates slm7-1 mutants general transcrip-          ture-sensitive phenotype slm7-1 (Figure 6) the <br /> tional defect, specific defect cyclin transcription.    synthetic lethality slm7-1 swi6&#226;&#338;&#172; background (data <br /> To test slm7-1 strain generally defective       shown). Confirmation rescue attribut- <br /> transcription, wild type (BY109), slm7-1 (NY11), taf145ts        able TAF17 gene genomic se- <br /> (YSW93), rbp1-1 (Y260) strains grown log             quences provided expressing TAF17 the <br /> phase 30&#226;&#172;&#353; shifted 37&#226;&#172;&#353;. RBP1 encodes largest        galactose-inducible GAL1-10 promoter (Moqtaderi et <br /> subunit RNA polymerase II cells containing            al. 1996b). This pGAL::TAF17 plasmid allowed growth <br /> rbp1-1 mutation rapid decrease overall tran-           slm7-1 strain galactose 37&#226;&#172;&#353; (Figure 6) not <br /> script levels restrictive temperature. By contrast,       glucose 37&#226;&#172;&#353;, indicating expression TAF17 <br /> cells carrying temperature-sensitive allele gene        rescue temperature-sensitive growth defect in <br /> encoding TATA-binding protein (TBP)-associated               slm7 strain. <br />                                                         TAF17 SWI6                                                1571 <br />  <br />                                                                 slm strains contained mutant allele SWI4; muta- <br />                                                                 tion SWI4 known synthetically lethal with <br />                                                                 mutation SWI6 (Breeden Nasmyth 1987a) due <br />                                                                 inadequate expression G1 cyclins. The slm7-1 mu- <br />                                                                 tant strain harbored temperature-sensitive allele of <br />                                                                 TAF17 arrested G1 phase cell cycle the <br />                                                                 nonpermissive temperature Swi6 levels low. <br />                                                                 At permissive temperature, taf17 slm7-1 SWI6 cells had <br />                                                                 specific defects cell cycle-regulated transcription <br />                                                                 subset Swi6-dependent target genes. Our data <br />    Figure 6.&#226;&#8364;&#8221;Rescue slm7-1 TAF17. NY11 (slm7-1 SWI6) <br />                                                                 implicate transcriptional regulatory complexes con- <br /> transformed vector, pNM10 (pTAF17) pZM255              taining Taf17 cell cycle regulation. <br /> (pGAL::TAF17), grown 30&#226;&#172;&#353; 37&#226;&#172;&#353; glucose galactose.        Cell cycle-arrest phenotypes slm6-1 swi6 slm8-1 <br /> Log phase cultures diluted OD600 0.1 serial   swi6 mutants: In screen hoped identify genes <br /> dilutions spotted plate. Spots grown 2    involved activating SBF- MBF-driven gene expres- <br /> days glucose 6 days galactose 30&#226;&#172;&#353; 37&#226;&#172;&#353;. <br />                                                                 sion absence SWI6. We identified mutations <br />                                                                 complementation groups, slm6, slm7, slm8, <br />                                                                 died absence Swi6 uniform <br />    To ask slm7-1 corresponds mutant allele of <br />                                                                 morphological arrest; arrest phenotype these <br /> TAF17, used integrating plasmid target LEU2 <br />                                                                 strains rescued overexpression cyclins <br /> to TAF17 locus disrupting gene. The <br />                                                                 PCL2, CLB5, CLN2 (data shown). To date, we <br /> TAF17::LEU2 strain (NY36) crossed slm7-1 strain <br />                                                                 identified mutated gene slm7 strains <br /> (NY34). The TAF17::LEU2/slm7-1 diploid sporu- <br />                                                                 discuss result below. However, arrest <br /> lated progeny resulting tetrads tested for <br />                                                                 phenotypes slm6 slm8 strains suggest that <br /> growth media lacking leucine (caused LEU2 <br />                                                                 analysis SLM6 SLM8 genes of <br /> gene) death SD 37&#226;&#172;&#353; (caused slm7- <br />                                                                 interest. A slm6-1 mutation caused swi6 cells to <br /> 1 mutation). Of 26 tetrads tested, progeny showed <br />                                                                 accumulate G2 phase elongated buds this <br /> cosegregation LEU2 slm7, indicating SLM7 <br />                                                                 arrest overcome increasing copy number <br /> is allelic TAF17. <br />                                                                 SWI4. The arrest phenotype slm6-1 swi6 mutants <br />    To identify TAF17 mutation slm7-1 strain, <br />                                                                 reflect problem budding cell wall biosyn- <br /> we used cell PCR amplify TAF17 gene from <br />                                                                 thesis. Both SBF Pkc1/Slt2 mitogen-acti- <br /> strain NY11 (slm7-1). Both strands different <br />                                                                 vated protein (MAP) kinase pathway required for <br /> PCR fragments sequenced resulting se- <br />                                                                 maximal induction cell wall biosynthetic genes encod- <br /> quences compared Saccharomyces Genome Data- <br />                                                                 ing proteins used budding (Igual et al. 1996). Swi6 <br /> <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> sequence. All sequences agreement and <br />                                                                 phosphorylated yeast cells Slt2-dependent man- <br /> showed slm7-1 mutation changed codon 133 of <br />                                                                 ner and, like slt2 mutants, G2 phase arrest slm6-1 <br /> TAF17 tryptophan codon (TGG) amber <br />                                                                 swi6 mutant cells overcome overexpression <br /> stop codon (TAG). The mutated gene predicted to <br />                                                                 SWI4 CLN2 (Igual et al. 1996; Madden et al. <br /> encode Taf17 derivative lacking C-terminal 25 <br />                                                                 1997). Perhaps overexpression SWI4 slm6-1 <br /> amino acids, include conserved region 11 <br />                                                                 swi6 mutant allows activation SBF target genes re- <br /> amino acids (Figure 7). We conclude mutation of <br />                                                                 quired cell wall biosynthesis budding. Allelism <br /> the Taf17 protein lead defects G1-regulated <br />                                                                 SLM6 genes encoding components Pkc1 <br /> gene expression causes cell cycle arrest G1 phase <br />                                                                 pathway tested mutation SWI4 or <br /> when Swi6 levels low cells grown under <br />                                                                 SWI6 known cause synthetic lethality mutation <br /> nonpermissive conditions. <br />                                                                 genes PKC-MAP kinase pathway (Gray et al. <br />                                                                 1997). However, unlike known alleles PKC1 SLT2, <br />                                                                 slm6-1 allele does cause cells temperature <br />                        DISCUSSION <br />                                                                 sensitive SWI6 background. <br />   Swi6 regulatory component transcription              In contrast slm6, slm8 mutation caused tem- <br /> factors, SBF MBF, promote G1-specific tran-            perature-dependent accumulation cells G1 and <br /> scription number genes essential           cell death absence SWI6. In addition, mutation <br /> START S-phase budding yeast (reviewed                 SLM8 presence wild-type SWI6 caused a <br /> Breeden 1995; Koch Nasmyth 1994). We                   marked accumulation cells 1N DNA content <br /> identified slm mutants (swi6-dependent lethal           cells shifted 25&#226;&#172;&#353; 37&#226;&#172;&#353;. The arrest <br /> mutants) cause inviability absence Swi6.         NY15 cells G1 DNA content accompa- <br /> A subset slm mutants showed cell cycle-specific      nied morphological G1 arrest. Rather, cells <br /> arrest phenotype restrictive conditions. One           continued produce buds, indicating slm8 swi6 <br />  1572                                               N. Macpherson et al. <br />  <br />  <br />  <br />  <br />                                                                                                Figure 7.&#226;&#8364;&#8221;Alignment of <br />                                                                                             yeast Taf17slm7-1 wild-type <br />                                                                                             yeast Taf17 human and <br />                                                                                             Drosophila homologues. This <br />                                                                                             alignment performed with <br />                                                                                             MSA version 2.1 (Gupta et al. <br />                                                                                             1995). Identical residues are <br />                                                                                             darkly shaded; similar residues <br />                                                                                             lightly shaded. The histone <br />                                                                                             fold predicted Xie et al. <br />                                                                                             (1996) underlined. The <br />                                                                                             boxed region region of <br />                                                                                             greatest similarity these <br />                                                                                             proteins predicted be <br />                                                                                             missing protein en- <br />                                                                                             coded slm7-1 allele of <br />                                                                                             TAF17. The translational ter- <br />                                                                                             mination signal Taf17slm7-1 is <br />                                                                                             indicated asterisk. <br />  <br />  <br />  <br />  <br /> cells budding DNA synthesis unlinked. At            temperature-sensitive alleles TAF90 TSM1, gene <br /> the restrictive temperature, swi4&#226;&#338;&#172; cells arrest       encodes yeast homologue higher eukary- <br /> opposite phenotype (BY108, Figure 3B), large unbud-         otic TAFII150 (Apone et al. 1996; Walker et al. 1996). <br /> ded cells mixed DNA content (Ogas et al. 1991;            TAFII homologues different organisms also <br /> Gray et al. 1997). These phenotypes suggest Swi4          functionally conserved; temperature-sensitive allele <br /> and Slm8 acting different processes Start,        (CCG1) hTAF250, mammalian homologue of <br /> with Swi4 having role budding Slm8 having             yTAF145, causes mutant cells arrest G1 (Sekiguchi <br /> a role DNA replication. Elucidation precise          et al. 1991; Hisatake et al. 1993; Ruppert et al. 1993). <br /> relationship SWI6, SLM6, SLM8 awaits           Also, human homologue yeast TSM1, hTAFII150, <br /> cloning characterization SLM6 SLM8              appears required G2/M transition mam- <br /> genes.                                                         malian cell lines activation B-type cyclin gene <br />    slm7-1 allelic gene encoding histone        expression (Martin et al. 1999). <br /> H3-like protein Taf17: Complementation allelism               yTaf145 histone acetylase activity activity <br /> tests revealed slm7-1 allelic TAF17. TAF17      resides region conserved mammals <br /> an essential gene encoding TBP-associated factor           yeast (Mizzen et al. 1996). A group yTafIIs&#226;&#8364;&#8221; <br /> Taf17/Taf20. Taf17 TBP-associated            Taf17, 25, 60, 61, 90&#226;&#8364;&#8221;copurifies his- <br /> factors (TAFIIs) subunits general tran-        tone acetylase complex called SAGA (Grant et al. 1997, <br /> scription factor IID (TFIID). TFIID general          1998). A subset TAFIIs (yTaf17, Taf60, and <br /> transcription factors components RNA poly-          Taf61) homology histone proteins (H3, H4, <br /> merase II holoenzyme (reviewed Berk 1999). TAFIIs           H2B, respectively; reviewed Burley Roeder <br /> are highly conserved proteins 12 yeast TAFIIs          1996) form histone-like octamer (Hoffmann <br /> (yTAFIIs) homologues eukaryotes,        et al. 1996; Nakatani et al. 1996; Xie et al. 1996). Consis- <br /> metazoans additional cell type-specific TAFIIs (re-       tent discovery common TAFIIs TFIID and <br /> viewed Tansey Herr 1997). Roles TAFIIs           SAGA, TAFIIs promote transcriptional activation by <br /> cell cycle regulation suggested number          transcription factors TFIID SAGA <br /> of studies yeast mammalian cells. In yeast,        complex (Klemm et al. 1995; Drysdale et al. 1998). The <br /> transcription G1 cyclins dependent TAF145         isolation mutations TAF17 screen be <br /> gene; G1 cyclin transcript levels reduced re-       reflection role TFIID SAGA <br /> strictive temperature taf145ts strain, leading cell    histone acetyl transferase complex. <br /> cycle arrest G1 phase (Walker et al. 1996, 1997). A            slm7-1 (taf17) mutants transcriptional defects in <br /> role TAFIIs regulating cell cycle-dependent       cell cycle-regulated genes: Cells containing tempera- <br /> gene expression suggested G2 phase arrest            ture-sensitive mutations TafIIs broad de- <br /> seen restrictive temperature yeast cells carrying    fects transcriptional activation nonpermissive <br />                                                         TAF17 SWI6                                                    1573 <br />  <br /> conditions, defects global (Apone et          proteins promote stability Taf17-containing <br /> al. 1996, 1998; Shen Green 1997; Walker et al.              complexes. <br /> 1997; Holstege et al. 1998; Moqtaderi et al. 1998;                 The specific defects cell cycle-regulated transcrip- <br /> Natarajan et al. 1998). Rather, mutant TafIIs cause             tion taf17 slm7-1 cells suggest Taf17slm7-1 protein <br /> gene-specific Taf-specific defects transcription         regulatory importance context of <br /> under nonpermissive conditions (Walker et al. 1997;             certain promoters. Our data suggest CLB5 and <br /> Wang et al. 1997; Holstege et al. 1998). For example,           PCL2 promoters partially Taf17 dependent that <br /> as noted above, taf145ts mutants defective tran-         CLN2, PCL1, PCL9 expression relatively indepen- <br /> scription specific subset genes, G1 cyclin-         dent Taf17. Our results agree DNA <br /> encoding genes (Walker et al. 1997), genome-wide            microarray analysis taf17 ts mutant strains de- <br /> analysis transcript levels taf145ts mutants reveals       scribed (http://gaiberg.wi.mit.edu/cgi-bin/young <br /> that 16% genes examined dependent                  _public/factor.cgi?gene&#226;&#171;&#189;TAF17) (Holstege et al. <br /> Taf145 largest subunit RNA poly-          1998). In experiments, CLB5 transcription de- <br /> merase II (Holstege et al. 1998). Our discovery            creased 8.7-fold taf17 ts cells 45 min the <br /> taf17slm7-1 mutants specific gene expression defects       restrictive temperature, CLN2 PCL9 tran- <br /> at permissive temperature, cell cycle-           scription decreased 1.1- 1.5-fold, respectively. <br /> arrest phenotype nonpermissive temperature               Reproducible results PCL2 obtained. We <br /> when Swi6 levels low, provides example           did observe induction PCL1 taf17 slm7-1 cells at <br /> a Taf play role cell cycle-dependent gene         25&#226;&#172;&#353; gene-chip analysis, PCL1 induced 3.2- <br /> expression.                                                     fold nonpermissive conditions. PCL1 induced <br />    Genome-wide assessment transcript levels using            high temperature (Madden et al. 1997), induc- <br /> DNA microarrays strains carrying temperature-sensi-          tion observed Holstege et al. occur 37&#226;&#172;&#353;. <br /> tive mutations Taf17 shows Taf17 required        The DNA microarray studies clarify the <br /> transcription &#226;&#181;&#8216;67% genes tested restrictive         <span id='am-317' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene expression taf17 slm7-1 mutant <br /> conditions (Apone et al. 1998; Holstege et al. 1998).           role TFIID role SAGA com- <br />                                                                 plex. <br /> A study using Taf17-depleted cells concluded that <br />                                                                    Reasons synthetic lethality mutations TAF17 <br /> most promoters Taf17-dependent (Moqtaderi et <br />                                                                 mutations SWI6: We consider possible expla- <br /> al. 1998). Whereas studies examined transcript <br />                                                                 nations synthetic lethality mutations SWI6 <br /> levels heat-treated, dying cells, study examined <br />                                                                 mutations TAF17. Either SCB and/or MCB- <br /> cell cycle-regulated genes taf17 ts mutant per- <br />                                                                 driven transcription Taf17 dependent, SCBs and <br /> missive temperature; conditions, cells <br />                                                                 MCBs Taf17 independent Taf17 controls cell <br /> remained viable heat stressed. Also, the <br />                                                                 cycle-regulated transcription elements. <br /> TAF17 alleles used studies contained mutations <br />                                                                 If SCBs MCBs Taf17 dependent, synthetic <br /> in histone fold (Apone et al. 1998; Michel et al. <br />                                                                 lethality mutations TAF17 mutations SWI6 <br /> 1998), necessary interaction his- <br />                                                                 reduced induction SBF- MBF-depen- <br /> tone-like TafIIs (Hoffmann et al. 1996; Nakatani et al. <br />                                                                 dent genes. In swi6 cells, induction SCB-containing <br /> 1996; Xie et al. 1996; Apone et al. 1998; Michel et al.         promoters Start reduced relative wild type (Nas- <br /> 1998). Many TAFIIs examined nonpermis-             myth Dirick 1991; Ogas et al. 1991; Dirick et al. <br /> sive conditions taf145ts taf17 ts cells unstable     1992; Lowndes et al. 1992; Breeden Mikesell <br /> (Walker et al. 1997; Apone et al. 1998; Holstege et al.         1994; Cross et al. 1994; Stuart Wittenberg 1994). <br /> 1998). This temperature-sensitive            The residual levels induction essential SBF-depen- <br /> mutations Taf145 Taf17 prevented formation             dent genes swi6 cells require Swi4, elimina- <br /> a Taf-containing complex nonpermissive temper-           tion Swi4 cell causes cell death (Breeden <br /> ature, causing constituent proteins complex         Nasmyth 1987a). In swi6 cells, induction MCB- <br /> to accessible degradation machinery.                  containing genes decreased, basal transcrip- <br />    The stability Taf17slm7-1 protein        tion level genes intermediate acti- <br /> established, defects transcription observed        vated inactivated levels (Koch et al. 1993). Mutation <br /> taf17 slm7-1 cells degradation Taf17slm7-1     Taf17 result reduction basal level, <br /> and associated proteins specific muta-       compromise cell viability. <br /> tion Taf17slm7-1. However, taf17 slm7-1 mutation         If SCBs MCBs Taf17 independent, are <br /> predicted encode Taf17 derivative intact           possibilities explain synthetic lethality <br /> histone fold, deletion encompassing highly         mutations SWI6 mutations TAF17. First, SBF- <br /> conserved region outside histone fold. Since            MBF-dependent genes contain Taf17-dependent <br /> histone fold Taf17slm7-1 intact, Taf17 variant   elements distinct SCBs MCBs. Alterna- <br /> interact normally components              tively, Taf17 activate transcription SBF- MBF- <br /> histone-like Taf octamer, association        independent genes necessary progression <br />  1574                                                   N. Macpherson et al. <br />  <br /> through Start absence Swi6. In                complexes containing Taf17 cell cycle-regulated tran- <br /> cases, synthetic lethality mutations TAF17           scription SBF MBF. <br /> mutations SWI6 defect cell cycle                We thank Helena Friesen, Lea Harrington, Andrew Spence, and <br /> regulation transcriptional activators. An exam-           Mike Tyers comments manuscript Jacqueline Segall <br /> ple SBF- MBF-independent element              helpful discussions. This work supported grant the <br /> be Taf17 dependent ECB promoter element. ECB                Medical Research Council Canada B.J.A., holds Scientist <br />                                                                    award agency. N.M. supported Univer- <br /> elements upstream cell cycle-regulated <br />                                                                    sity Toronto Open Fellowship. <br /> genes, including SWI4, control the <br /> Mcm1 protein proteins (McInerny et <br /> al. 1997). Effects SWI4 expression unlikely to <br /> account entirely phenotype taf17 slm7-1 strain,                             LITERATURE CITED <br /> since expression SWI4 heterologous promoter              Amon, A., M. Tyers, B. Futcher K. Nasmyth, 1993 Mecha- <br /> failed rescue synthetic lethality taf17 slm7-1 swi6        nisms help yeast cell cycle clock tick: G2 cyclins transcrip- <br /> strain (data shown results).                                   tionally activate G2 cyclins repress G1 cyclins. Cell 74: 993&#226;&#8364;&#8220; <br />                                                                        1007. <br />    Role chromatin-remodeling complexes tran-                 Andrews, B. J., I. Herskowitz, 1989a Identification DNA <br /> scriptional activation SBF MBF: Recent chromatin                 binding factor involved cell cycle-control yeast HO gene. <br /> immunoprecipitation experiments shown                    Cell 57: 21&#226;&#8364;&#8220;29. <br />                                                                    Andrews, B. J., I. Herskowitz, 1989b The yeast Swi4 protein <br /> chromatin remodelling complexes, SAGA Swi/Snf,                     contains motif present developmental regulators part <br /> facilitate binding SBF HO promoter G1                 complex involved cell cycle-dependent transcription. Na- <br /> phase (Cosma et al. 1999). Our discovery genetic                  ture 342: 830&#226;&#8364;&#8220;833. <br />                                                                    Andrews, B. J., L. Moore, 1992 Interaction yeast Swi4 <br /> interaction Taf17 Swi6 reflect re-                   Swi6 cell cycle regulatory proteins vitro. Proc. Natl. Acad. <br /> quirement chromatin remodelling factors                     Sci. USA 89: 11852&#226;&#8364;&#8220;11856. <br /> proper cell cycle regulation SBF- MBF-                Apone, L. M., C.-m. A. Virbasius, J. C. Reese M. R. Green, 1996 <br />                                                                        Yeast TAFII90 required cell-cycle progression G2/ <br /> dependent genes. There ways Taf17-                M general transcription activation. Genes Dev. 10: <br /> containing SAGA TFIID complexes assist                   2368&#226;&#8364;&#8220;2380. <br /> activation RNA polymerase SBF MBF. The pres-              Apone, L., C. Virbasius, F. Holstege, J. Wang, R. Young et al., <br />                                                                        1998 Broad, universal, transcriptional requirement for <br /> ence coactivator disruption histone-                  yTAFII17, histone H3-like TAFII present TFIID SAGA. <br /> DNA contacts encourage binding SBF MBF                       Mol. Cell 2: 653&#226;&#8364;&#8220;661. <br /> to target sites. Alternatively, coactivators assist          Bender, A., J. R. Pringle, 1991 Use screen synthetic <br />                                                                        lethal multicopy suppressor mutants identify new <br /> transcriptional activation binding SBF or <br />                                                                        genes involved morphogenesis Saccharomyces cerevisiae. Mol. <br /> MBF TFIID SAGA; binding occur                   Cell. Biol. 11: 1295&#226;&#8364;&#8220;1305. <br /> a direct interaction Taf17/Taf60/Taf68 his-               Berk, A. J., 1999 Activation RNA polymerase II transcription. <br /> tone-like octamer. Since combining defects taf17                Curr. Opin. Cell Biol. 11: 330&#226;&#8364;&#8220;335. <br />                                                                    Boeke, J. D., F. LaCroute G. R. Fink, 1984 A positive selection <br /> mutants swi6 mutants results death,                mutants lacking orotidine-5&#226;&#172;&#732;-phosphate decarboxylase activity <br /> epistatic other. Binding SBF MBF                     yeast: 5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197: <br /> Taf17 wild-type cell occur                  345&#226;&#8364;&#8220;346. <br />                                                                    Breeden, L., 1995 Start-specific transcription yeast. Curr. Top. <br /> Swi4 Mbp1. Support direct binding model                       Microbiol. Immunol. 208: 95&#226;&#8364;&#8220;127. <br /> comes studies transcription factors. The             Breeden, L., G. E. Mikesell, 1991 Cell cycle-specific expression <br /> Gcn4 transcriptional activator binds components                     SWI4 transcription factor required cell cycle regula- <br />                                                                        tion HO transcription. Genes Dev. 5: 1183&#226;&#8364;&#8220;1190. <br /> both TFIID SAGA, including Taf17, human                Breeden, L., G. Mikesell, 1994 Three independent forms of <br /> and Drosophila homologues Taf17 bind                     regulation affect expression HO, CLN1 CLN2 the <br /> VP16 p53 transcriptional activators (Goodrich et                   cell cycle Saccharomyces cerevisiae. Genetics 138: 1015&#226;&#8364;&#8220;1024. <br />                                                                    Breeden, L., K. Nasmyth, 1987a Cell cycle control yeast <br /> al. 1993; Lu Levine 1995; Thut et al. 1995; Drys-                  HO gene: cis trans acting regulators. Cell 48: 389&#226;&#8364;&#8220;397. <br /> dale et al. 1998). Also, histone H2B-like protein              Breeden, L., K. Nasmyth, 1987b Similarity cell <br /> Taf61 required Gcn4 recruit SAGA complex                  cycle genes fission yeast budding yeast Notch <br />                                                                        gene Drosophila. Nature 329: 651&#226;&#8364;&#8220;654. <br /> (Natarajan et al. 1998).                                           Burley, S., R. Roeder, 1996 Biochemistry structural biol- <br />    In summary, mutations 10 genes,                    ogy transcription factor IID (TFIID). Annu. Rev. Biochem. 65: <br /> SLM1&#226;&#8364;&#8220;10, synthetically lethal deletion                769&#226;&#8364;&#8220;799. <br />                                                                    Cho, R., M. Campbell, E. Winzeler, L. Steinmetz, A. Conway et <br /> SWI6. Mutations SLM genes, SLM6, caused                  al., 1998 A genome-wide transcriptional analysis mitotic <br /> swi6&#226;&#338;&#172; cells accumulate G1 phase lead                     cell cycle. Mol. Cell 2: 65&#226;&#8364;&#8220;73. <br /> to defect budding cell wall biosynthesis. Two              Cosma, M. P., T. Tanaka K. Nasmyth, 1999 Ordered recruit- <br />                                                                        ment transcription chromatin remodeling factors cell <br /> mutations, slm7 slm8, caused accumulation G1                 cycle-regulated developmentally regulated promoter. Cell <br /> phase swi6 mutant background. The slm7-1 mutant                   97: 299&#226;&#8364;&#8220;311. <br /> had altered cell cycle-regulated transcription             Costigan, C., S. Gehrung M. Snyder, 1992 A synthetic lethal <br /> genes controlled SBF MBF. We cloned                    screen identifies SLK1, novel protein kinase homolog impli- <br />                                                                        cated yeast cell morphogenesis cell growth. Mol. Cell. <br /> the SLM7 gene showed allelic TAF17.                Biol. 12: 1162&#226;&#8364;&#8220;1178. <br /> Our data strongly implicate transcriptional regulatory             Cross, F., M. Hoek, J. D. McKinney A. H. Tinkelenberg, 1994 <br />                                                                    TAF17 SWI6                                                              1575 <br />     Role Swi4 cell cycle regulation CLN2 expression. Mol.          Koch, C., T. Moll, M. Neuberg, H. Ahorn K. Nasmyth, 1993 <br />     Cell. Biol. 14: 4779&#226;&#8364;&#8220;4787.                                                  A role transcription factors Mbp1 Swi4 progression <br /> Dirick, L., T. Moll, H. Auer K. Nasmyth, 1992 A central role                G1 S phase. Science 261: 1551&#226;&#8364;&#8220;1557. <br />     SWI6 modulating cell cycle Start-specific transcription       Koch, C., A. Schleiffer, G. Ammerer K. Nasmyth, 1996 <br />     yeast. Nature 357: 508&#226;&#8364;&#8220;513.                                                 Switching transcription yeast cell cycle: <br /> Dirick, L., T. Bohm K. Nasmyth, 1995 Roles regulation                Cln/Cdc28 kinases activate bound transcription factor SBF <br />     Cln-Cdc28 kinases start cell cycle Saccharomyces       (Swi4/Swi6) Start, Clb/Cdc28 kinases displace from <br />     cerevisiae. EMBO J. 14: 4803&#226;&#8364;&#8220;4813.                                          promoter G2. Genes Dev. 10: 129&#226;&#8364;&#8220;141. <br /> Drysdale, C., B. Jackson, R. McVeigh, E. Klebanow, Y. Bai et al.,           Lowndes, N. F., A. L. Johnson L. H. Johnston, 1991 Coordina- <br />     1998 The Gcn4p activation domain interacts specifically vitro            tion expression DNA synthesis genes budding yeast by <br />     RNA polymerase II holoenzyme, TFIID, Adap-Gcn5p                cell-cycle regulated trans factor. Nature 350: 247&#226;&#8364;&#8220;250. <br />     coactivator complex. Mol. Cell. Biol. 18: 1711&#226;&#8364;&#8220;1724.                    Lowndes, N. F., A. L. Johnson, L. Breeden L. H. Johnston, <br /> Engebrecht, J., J. Hirsch G. S. Roeder, 1990 Meiotic gene                   1992 SWI6 protein required transcription periodi- <br />     conversion crossing over: relationship              cally expressed DNA synthesis genes budding yeast. Nature <br />     chromosome synapsis segregation. Cell 62: 927&#226;&#8364;&#8220;937.               357: 505&#226;&#8364;&#8220;508. <br /> Espinoza, F. H., J. Ogas, I. Herskowitz D. O. Morgan, 1994              Lu, H., A. Levine, 1995 Human TAFII31 protein transcrip- <br />     Cell cycle control complex cyclin HCS26(PCL1)               tional coactivator p53 protein. Proc. Natl. Acad. Sci. USA <br />     kinase PHO85. Science 266: 1388&#226;&#8364;&#8220;1391.                                   92: 5154&#226;&#8364;&#8220;5158. <br /> Fernandez-Sarabia, M. J., A. Sutton, T. Zhong K. T. Arndt,              Madden, K., Y.-J. Sheu, K. Baetz, B. Andrews M. Snyder, 1997 <br />     1992 SIT4 protein phosphatase required normal accu-              SBF cell cycle regulator target yeast PKC-MAP kinase <br />     mulation SWI4, CLN1, CLN2 HCS26 RNAs late G1.                 pathway. Science 275: 1781&#226;&#8364;&#8220;1784. <br />     Genes Dev. 6: 2417&#226;&#8364;&#8220;2428.                                                Martin, J., R. Halenbeck J. Kaufmann, 1999 Human transcrip- <br /> Foster, R., G. E. Mikesell L. Breeden, 1993 Multiple Swi6-                  tion factor hTAFII150(CIF150) involved transcriptional regu- <br />     dependent cis-acting elements control SWI4 transcription                    lation cell cycle progression. Mol. Cell. Biol. 19: 5548&#226;&#8364;&#8220;5556. <br />     cell cycle. Mol. Cell. Biol. 13: 3972&#226;&#8364;&#8220;3801.                 McInerny, C. J., J. F. Partridge, G. E. Mikesell, D. P. Creemer <br /> Gietz, D., A. St. Jean, R. A. Woods R. H. Schiestl, 1992 Im-                L. L. Breeden, 1997 A novel Mcm1-dependent promoter <br />     proved method high efficiency transformation intact yeast            element SWI4, CLN3, CDC6, CDC47 promoters activates <br />     cells. Nucleic Acids Res. 20: 1425.                                         M/G1-specific transcription. Genes Dev. 11: 1277&#226;&#8364;&#8220;1288. <br /> Goodrich, J., T. Hoey, C. Thut, A. Admon R. Tjian, 1993 Dro-            Measday, V., L. Moore, J. Ogas, M. Tyers B. Andrews, 1994 <br />     sophila TAFII40 interacts VP16 activation domain                The PCL2 (ORFD)-PHO85 cyclin-dependent kinase complex: a <br />     basal transcription factor TFIIB. Cell 75: 519&#226;&#8364;&#8220;530.                 cell cycle regulator yeast. Science 266: 1391&#226;&#8364;&#8220;1395. <br /> Grant, P., L. Duggan, J. Cote, S. Roberts, J. Brownell et al., 1997         Measday, V., L. Moore, R. Retnakaran, J. Lee, M. Donoviel et al., <br />     Yeast Gcn5 functions multisubunit complexes acetylate             1997 A family cyclin-like proteins interact cyclin- <br />     nucleosomal histones: characterization Ada complex                dependent kinase, Pho85. Mol. Cell. Biol. 17: 1212&#226;&#8364;&#8220;1223. <br />     SAGA (Spt/Ada) complex. Genes Dev. 11: 1640&#226;&#8364;&#8220;1650.                   Michel, B., P. Komarnitsky S. Buratowski, 1998 Histone- <br /> Grant, P., D. Schieltz, M. Pray-Grant, D. Steger, J. Reese et al.,              like TAFs essential transcription vivo. Mol. Cell 2: <br />     1998 A subset TAF(II)s integral components SAGA               663&#226;&#8364;&#8220;673. <br />     complex required nucleosome acetylation transcriptional         Mizzen, C., X. Yang, T. Kokubo, J. Brownell, A. Bannister et al., <br />     stimulation. Cell 94: 45&#226;&#8364;&#8220;53.                                                1996 The TAF(II)250 subunit TFIID histone acetyltrans- <br /> Gray, J., J. Ogas, Y. Kamada, M. Stone, D. Levin et al., 1997 A                 ferase activity. Cell 87: 1261&#226;&#8364;&#8220;1270. <br />     role Pkc1 MAP kinase pathway Saccharomyces cerevisiae        Moqtaderi, Z., Y. Bai, D. Poon, P. Weil K. Struhl, 1996a TBP- <br />     bud emergence identification putative upstream regu-            associated factors generally required transcriptional <br />     lator. EMBO J. 16: 4924&#226;&#8364;&#8220;4937.                                               activation yeast. Nature 383: 188&#226;&#8364;&#8220;191. <br /> Guarente, L., 1993 Synthetic enhancement gene interaction:             Moqtaderi, Z., J. DePaulo Yale, K. Struhl S. Buratowski, <br />     genetic tool come age. Trends Genet. 9: 362&#226;&#8364;&#8220;366.                         1996b Yeast homologues higher eukaryotic TFIID subunits. <br /> Gupta, S., J. Kececioglu A. Schaffer, 1995 Improving                    Proc. Natl. Acad. Sci. USA 93: 14654&#226;&#8364;&#8220;14658. <br />     practical space time efficiency shortest-paths approach      Moqtaderi, Z., M. Keaveney K. Struhl, 1998 The histone <br />     sum-of-pairs multiple sequence alignment. J. Computational               H3-like TAF broadly required transcription yeast. Mol. <br />     Biol. 2: 459&#226;&#8364;&#8220;472.                                                           Cell 2: 675&#226;&#8364;&#8220;682. <br /> Harrington, L. A., B. J. Andrews, 1996 Binding yeast             Nakatani, Y., S. Bagby M. Ikura, 1996 The histone folds in <br />     Swi4,6-dependent cell cycle box, CACGAAA, cell cycle regu-               transcription factor TFIID. J. Biol. Chem. 271: 6575&#226;&#8364;&#8220;6578. <br />     lated vivo. Nucleic Acids Res. 24: 558&#226;&#8364;&#8220;565.                          Nasmyth, K., 1985 A repetitive DNA sequence confers cell- <br /> Hisatake, K., S. Hasegawa, R. Takeda, Y. Nakatani, M. Horikoshi                 cycle START (CDC28) dependent transcription HO gene <br />     et al., 1993 The p250 subunit native TATA-binding factor                 yeast. Cell 42: 225&#226;&#8364;&#8220;235. <br />     TFIID cell cycle regulatory protein CCG1. Nature 362:            Nasmyth, K., L. Dirick, 1991 The role SWI4 SWI6 in <br />     179&#226;&#8364;&#8220;181.                                                                    activity G1 cyclins yeast. Cell 66: 995&#226;&#8364;&#8220;1013. <br /> Hoffmann, A., C. Chiang, T. Oelgeschlager, X. Xie, S. Burley et             Natarajan, K., B. Jackson, E. Rhee A. Hinnebusch, 1998 <br />     al., 1996 A histone octamer-like structure TFIID. Nature             yTAFII61 general role RNA polymerase II transcription <br />     380: 356&#226;&#8364;&#8220;359.                                                               required Gcn4p recruit SAGA coactivator com- <br /> Holstege, F., E. Jennings, J. Wyrick, T. Lee, C. Hengartner et al.,             plex. Mol. Cell 2: 683&#226;&#8364;&#8220;692. <br />     1998 Dissecting regulatory circuitry eukaryotic genome.        Ogas, J., B. J. Andrews I. Herskowitz, 1991 Transcriptional <br />     Cell 95: 717&#226;&#8364;&#8220;728.                                                           activation CLN1, CLN2 putative new G1 cyclin (HCS26) <br /> Holtzman, D. A., S. Yang D. Drubin, 1993 Synthetic-lethal                   SWI4, positive regulator G1 specific transcription. Cell <br />     interactions identify novel genes, SLA1 SLA2, control          66: 1015&#226;&#8364;&#8220;1026. <br />     membrane cytoskeleton assembly Saccharomyces cerevisiae. J. Cell     Partridge, J. F., G. E. Mikesell L. L. Breeden, 1997 Cell cycle- <br />     Biol. 122: 635&#226;&#8364;&#8220;644.                                                         dependent transcription CLN1 involves Swi4 binding MCB- <br /> Igual, J. C., A. L. Johnson L. H. Johnston, 1996 Coordinated                like elements. J. Biol. Chem. 272: 9071&#226;&#8364;&#8220;9077. <br />     regulation gene expression cell cycle transcription factor    Primig, M., S. Sockanathan, H. Auer K. Nasmyth, 1992 Anat- <br />     Swi4 protein kinase C MAP kinase pathway yeast cell             omy transcription factor important start cell <br />     integrity. EMBO J. 15: 5001&#226;&#8364;&#8220;5013.                                           cycle S. cerevisiae. Nature 358: 593&#226;&#8364;&#8220;597. <br /> Klemm, R., J. Goodrich, S. Zhou R. Tjian, 1995 Molecular                Ruppert, S., E. H. Wang R. Tjian, 1993 Cloning expression <br />     cloning expression 32-kDa subunit human TFIID                 human TAFII250: TBP-associated factor implicated cell- <br />     reveals interactions VP16 TFIIB mediate transcrip-            cycle regulation. Nature 362: 175&#226;&#8364;&#8220;179. <br />     tional activation. Proc. Natl. Acad. Sci. USA 92: 5788&#226;&#8364;&#8220;5792.            Schwob, E., K. Nasmyth, 1993 CLB5 CLB6, new pair of <br /> Koch, C., K. Nasmyth, 1994 Cell cycle regulated transcription               B cyclins involved DNA replication Saccharomyces cerevisiae. <br />     yeast. Curr. Opin. Cell Biol. 6: 451&#226;&#8364;&#8220;459.                                Genes Dev. 7: 1160&#226;&#8364;&#8220;1175. <br />  1576                                                           N. Macpherson et al. <br /> Sekiguchi, T., Y. Nohiro, Y. Nakamura, N. Hisamoto T. Nishi-           Taba, M. R., I. Muroff, G. Lydall, G. Tebb K. Nasmyth, 1991 <br />     moto, 1991 The human CCGI gene, essential progression                  Changes Swi4,6-DNA-binding complex occur time of <br />     G1 phase, encodes 210-kilodalton nuclear DNA-binding              HO activation yeast. Genes Dev. 5: 2000&#226;&#8364;&#8220;2013. <br />     protein. Mol. Cell. Biol. 11: 3317&#226;&#8364;&#8220;3325.                               Tansey, W. P., W. Herr, 1997 TAFs: Guilt Association? Cell <br /> Shen, W., M. Green, 1997 Yeast TAF(II)145 functions core              88: 729&#226;&#8364;&#8220;732. <br />     promoter selectivity factor, general coactivator. Cell 90:       Thut, C., J. Chen, R. Klemm R. Tjian, 1995 p53 transcriptional <br />     615&#226;&#8364;&#8220;624.                                                                   activation mediated coactivators TAFII40 TAFII60. Science <br /> Sherman, F., J. Hicks, 1991 Micromanipulation dissection               267: 100&#226;&#8364;&#8220;104. <br />     asci, pp. 21&#226;&#8364;&#8220;37 Guide Yeast Genetics Molecular Biology,   Tyers, M., G. Tokiwa B. Futcher, 1993 Comparison the <br />     edited C. Guthrie G. Fink. Academic Press, San Diego.               Saccharomyces cerevisiae G1 cyclins: Cln3 upstream activa- <br /> Sidorova, J., L. Breeden, 1993 Analysis SWI4/SWI6                   tor Cln1, Cln2 cyclins. EMBO J. 12: 1955&#226;&#8364;&#8220;1968. <br />     protein complex, directs G1/S-phase specific transcription       Verma, R., J. Smiley, B. Andrews J. Campbell, 1992 Regulation <br />     Saccharomyces cerevisiae. Mol. Cell. Biol. 13: 1069&#226;&#8364;&#8220;1077.               yeast DNA replication genes MluI cell cycle box <br /> Siegmund, R. F., K. Nasmyth, 1996 The Saccharomyces cerevisiae             dependent SWI6. Proc. Natl. Acad. Sci. USA 89: 9479&#226;&#8364;&#8220;9483. <br />     Start-specific transcription factor Swi4 interacts an-     Walker, S., J. Reese, L. Apone M. Green, 1996 Transcription <br />     kyrin repeats mitotic Clb2/Cdc28 kinase               activation cells lacking TAFIIS. Nature 383: 185&#226;&#8364;&#8220;188. <br />     conserved carboxy terminus Swi6. Mol. Cell. Biol. 16:         Walker, S. S., W.-C. Shen, J. C. Reese, L. M. Apone M. R. Green, <br />     2647&#226;&#8364;&#8220;2655.                                                                 1997 Yeast TAFII145 required transcription G1/S cyclin <br /> Sikorski, R. S., P. Hieter, 1989 A shuttle vectors           genes regulated cellular growth state. Cell 90: 607&#226;&#8364;&#8220;614. <br />     yeast host strains designed efficient manipulation DNA       Wang, E. H., S. Zou R. Tjian, 1997 TAFII250-dependent tran- <br />     Saccharomyces cerevisiae. Genetics 12: 19&#226;&#8364;&#8220;27.                              scription cyclin A directed ATF activator proteins. Genes <br /> Spellman, P. T., G. Sherlock, M. Q. Zhang, R. I. Vishwanath, K.                Dev. 11: 2658&#226;&#8364;&#8220;2669. <br />     Anders et al., 1998 Comprehensive identification cell cycle-        Wittenberg, C., K. Sugimoto S. I. Reed, 1990 G1-specific <br />     regulated genes yeast Saccharomyces cerevisiae microarray        cyclins S. cerevisiae: Cell cycle periodicity,  <br /> </body></html>