<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />           Distinct Mutations in Yeast TAFII25 Differentially Affect the <br />                 Composition of TFIID and SAGA Complexes as <br />                     Well as Global Gene Expression Patterns <br />  <br />  <br />              The RNA polymerase II transcription factor TFIID, composed of the TATA-binding protein (TBP) and <br />           TBP-associated factors (TAFIIs), nucleates preinitiation complex formation at protein-coding gene promoters. <br />           SAGA, a second TAFII-containing multiprotein complex, is involved in transcription regulation in Saccharo- <br />           myces cerevisiae. One of the essential protein components common <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> SAGA and TFIID is yTAFII25. We define <br />           a minimal evolutionarily conserved 91-amino-acid region of TAFII25 containing a histone fold domain that is <br />           necessary and sufficient for growth in vivo. Different temperature-sensitive mutations of yTAFII25 or chimeras <br />           with the human homologue TAFII30 arrested cell growth at either the G1 or G2/M cell cycle phase and displayed <br />           distinct phenotypic changes and gene expression patterns. Immunoprecipitation studies revealed that TAFII25 <br />           <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span>-dependent gene expression and phenotypic changes correlated at least partially with the integrity of <br />           SAGA and TFIID. Genome-wide expression analysis revealed that the five TAFII25 temperature-sensitive <br />           mutant alleles individually affect the expression of between 18 and 33% of genes, whereas taken together they <br />           affect 64% of all class II genes. Thus, different yTAFII25 mutations induce distinct phenotypes and affect the <br />           regulation of different subsets of genes, demonstrating that no individual TAFII mutant allele reflects the full <br />           range of its normal functions. <br />  <br />  <br />    Initiation of transcription of protein-encoding genes by                  cerevisiae SAGA (ySAGA) complex (26), comprises at least <br /> RNA polymerase II (Pol II) requires the assembly of general                  four distinct classes of gene products: (i) the Ada proteins <br /> transcription factors (TFIIA, TFIIB, TFIID, TFIIE, TFIIF,                    (yAda1, yAda2, yAda3, yGcn5 [yAda4], and yAda5 [ySpt20]), <br /> and TFIIH) on promoters to form a preinitiation complex (31).                which were isolated in a genetic screen for mutations that <br /> Preinitiation complex assembly on both TATA-containing and                   relieve the toxicity of the strong VP16 activation domain when <br /> TATA-less promoters is often nucleated by transcription fac-                 overexpressed as a fusion with the Gal4 DNA-binding domain <br /> tor TFIID, which comprises the TATA-binding protein (TBP)                    in yeast cells (5); (ii) Spt proteins (ySpt3, ySpt7, ySpt8, and <br /> and 14 TBP-associated factors (TAFIIs) (4, 23, 68). To date,                 ySpt20), initially identified as suppressors of transcription ini- <br /> most of the TFIID components <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Saccharomyces cerevisiae,                  tiation defects caused by promoter insertions of the transpos- <br /> Drosophila, and humans have been identified, partially charac-               able Ty element (16, 17, 22, 65); (iii) a subset of TBP-associ- <br /> terized, and shown to be well conserved throughout evolution                 ated factors (TAFIIs), including TAFII17, TAFII25, TAFII60, <br /> (18, 19, 68).                                                                TAFII68/61, and TAFII90 (27); and (iv) the product of the <br />    A large number of recent studies have provided a direct                   essential gene Tra1 (28, 66), which is also a component of the <br /> molecular link between histone acetylation and transcriptional               Esa1-containing HAT complex NuA4 (1). <br /> activation (reviewed in references 11 and 40). It has been                      In addition, several similar human multiprotein complexes <br /> shown that several previously identified coactivators/adapters               have been characterized, such as the TBP-free TAFII-contain- <br /> of transcription possess intrinsic histone acetyltransferase                 ing complex (10, 81), the PCAF/GCN5 complex (58), and the <br /> (HAT) activity or are associated in large multiprotein com-                  SPT3-TAFII31-GCN5 acetyltransferase complex (48), all of <br /> plexes with HATs (71). One of these, the 1.8- to 2-MDa S.                    which contain homologues of the S. cerevisiae GCN5 HAT, <br />                                                                              ADA proteins, SPTs, TAFIIs, and the human homologue of <br />                                                                              yTRA1, TRRAP. <br />   * Corresponding author. Mailing address: Institut de G&#195;&#169;n&#195;&#169;tique et             Initial sequence comparisons indicated that human TAFII80 <br /> de Biologie Mol&#195;&#169;culaire et Cellulaire, CNRS/INSERM/ULP, BP 163,              (hTAFII80; homologous to Drosophila TAFII60 [dTAFII60] <br /> F-67404 Illkirch Cedex, CU de Strasbourg, France. Phone: 33 3 88 65 <br /> 34 44. Fax: 33 388 65 32 01. E-mail: laszlo@igbmc.u-strasbg.fr.              and yTAFII60), hTAFII31 (dTAFII40, yTAFII17), and hTAFII20 <br />   &#226;&#8364;&#160; Present address: Institut Curie, UMR 218 du CNRS, 75248 Paris            (dTAFII30&#226;&#65533;&#163;, yTAFII61/68) have obvious homology to histones <br /> Cedex 05, France.                                                            H4, H3, and H2B, respectively (33, 38), and X-ray crystallog- <br />   &#226;&#8364;&#161; Present address: Wellcome/CRC Institute and Department of Pa-            raphy showed that dTAFII60 and dTAFII40 or hTAFII28 and <br /> thology, University of Cambridge, Cambridge CB2 1QR, United King- <br /> dom. <br />                                                                              hTAFII18 interact via their histone fold domains (HFDs) (6, <br />   &#194;&#167; Present address: Friedrich Miescher Institute, CH-4002 Basel,            82). The histone fold is a protein-protein interaction motif <br /> Switzerland.                                                                 originally described in the heterodimerization of the core his- <br />  <br />                                                                       3178 <br />  VOL. 22, 2002                                          TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                              3179 <br />  <br />  <br />                                                             TABLE 1. Strains <br />  Strain                                                Relevant genotype                                                       Plasmid <br />  <br /> PL3(2n)                       ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3 <br /> DKY12                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       PRS316-TAF25&#226;&#171;&#185; <br /> DKY18                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-&#226;&#338;&#172;C-TAF25 <br /> DKY19                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-&#226;&#338;&#172;N-TAF25 <br /> DKY21                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim. 21 <br /> DKY23                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-&#226;&#338;&#172;N-TAF30 <br /> DKY24                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-TAF30&#226;&#171;&#185; <br /> DKY30                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-TAF25&#226;&#171;&#185; <br /> DKY32                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 32 <br /> DKY37                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim. 37 <br /> DKY38                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim.38 <br /> DKY41                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 41 <br /> DKY42                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 42 <br /> DKY43                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 43 <br /> DKY71                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 71 <br /> DKY72                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 72 <br /> DKY92                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 92 <br /> DKY96                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim. 96 <br /> DKY97                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim. 97 <br /> DKY98                         MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Chim. 98 <br /> DKY105                        MATa ura3-&#226;&#338;&#172;1 his3&#226;&#338;&#172;-200 leu2-&#226;&#338;&#172;1 trp1::3ERE-URA3        taf25::KanMX2                       YEP10FL-Del. 105 <br />  <br />  <br />  <br />  <br /> tones H4 and H3 and histones H2A and H2B and their assem-                     Furthermore, genome-wide expression analysis, comparing <br /> bly into a nucleosome (3, 46). The HFD comprises three &#226;&#65533;&#163;-he-               single conditionally temperature-sensitive mutations in genes <br /> lices linked by two loops. The HFDs of histones H3 and H2B                 encoding components of either TFIID or SAGA complexes <br /> contain an additional &#226;&#65533;&#163;-helix extension at the N- (&#226;&#65533;&#163;N) or                  (29, 34), indicated that expression of most genes requires one <br /> C-terminal (&#226;&#65533;&#163;C) end, respectively (46).                                    or more of the common TAFII subunits, suggesting that the <br />    In vitro transcription assays using different cell-free systems         functions of either TFIID or SAGA or both are widely re- <br /> as well as cell transfection experiments in mammalian cells                quired for gene expression (42), although the contribution of <br /> suggested that TAFIIs are essential for activation of transcrip-           either complex remains to be investigated. Among the subunits <br /> tion in response to transcriptional activators (13, 25, 35, 50, 63,        shared by TFIID and SAGA are five histone fold-containing <br /> 73) and important for core promoter recognition (12, 49, 64,               TAFIIs, yTAFII90, yTAFII68/61, yTAFII60, yTAFII25, and <br /> 75). Moreover, the largest of the TAFIIs, TAFII250, has been               yTAFII17, mutants of which have been shown to affect distinct <br /> shown to possess enzymatic activities: a kinase activity (14, 57),         gene expression patterns in S. cerevisiae. <br /> a HAT activity (53), and a ubiquitin-activating/conjugating ac-               To date, many questions remain concerning the role of <br /> tivity (59). Thus, TAFIIs seem to be involved in several steps in          TAFIIs in transcription, including whether TAFIIs are required <br /> the regulation of transcription, but their exact role and indi-            at most or only a small subset of promoters. The interpretation <br /> vidual contributions are unknown.                                          of the published data is complicated by the fact that different <br />    Surprisingly, data from experiments carried out by TAFII                temperature-sensitive alleles of a single TAFII can either result <br /> depletion or with temperature-sensitive TAFII mutants sug-                 in broad effects on Pol II transcription (39, 67) or only affect <br /> gested that some TAFIIs are not generally required for tran-               the transcription of a subset of genes (29, 34, 42). Furthermore, <br /> scription activation and that different yTAFIIs selectively affect         it has not yet been possible to separate the function of a TAFII <br /> the transcription of different subsets of genes (2, 54, 69, 80).           in TFIID from that in SAGA. <br /> Moreover, TAFII-dependent and TAFII-independent promot-                       Here we describe a structure-function study of yTAFII25 in <br /> ers have been described in S. cerevisiae, and it has been pro-             S. cerevisiae. We define a minimal 91-amino-acid region of <br /> posed that following heat shock, TAFII occupancy of TAFII-                 yTAFII25 that is necessary and sufficient for vegetative growth. <br /> independent promoters increases (41, 44). Nevertheless,                    This region of yTAFII25 contains a putative HFD similar to <br /> TAFIIs are essential in S. cerevisiae, since full gene deletions           that in eight other yTAFIIs (18). The HFD of yTAFII25 was <br /> are not viable (55, 62).                                                   shown to mediate selective heterodimerization with TFIID- <br />    In contrast to the TAFII class of SAGA subunits, all other              specific components yTAFII47 and yTAFII65 as well as SAGA- <br /> subunits, with the exception of Tra1, are not essential for                specific component ySPT7 in S. cerevisiae two-hybrid and bac- <br /> growth (72). The observation that cells containing a tempera-              terial coexpression assays (20). <br /> ture-sensitive mutation in certain TAFII alleles arrest at par-               Here we show that yTAFII25 is required for normal cell <br /> ticular points in the cell cycle may further support the idea that         cycle progression and that different mutations in the minimal <br /> individual TAFIIs have selective effects on different subsets of           region of TAFII25 arrest yeast cell growth at different cell cycle <br /> genes required for cell cycle regulation. Arrest of individual             phases with distinct phenotypes, giving evidence for the mul- <br /> yTAFII90 and yTAFII150 mutant strains occurs at the G2/M                   tiple functions of TAFII25. To investigate the possibility that <br /> boundary, whereas yTAFII130 mutant strains arrest in the G1                the different phenotypes of the TAFII25 mutants studied here <br /> phase of the cell cycle (2, 79, 80).                                       can be attributed to TFIID- or SAGA-specific functions of <br />  3180       KIRSCHNER ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> TAFII25, we examined the subunit composition of both TFIID                             cells/ml, and aliquots of different dilutions were spotted on YPD plates and <br /> and SAGA by immunoprecipitation and tested the genome-                                 exposed to either 28 or 37&#194;&#176;C. <br />                                                                                           Microscopy. Cells were grown as described above, and aliquots were taken at <br /> wide expression <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span> in TAFII25 mutant strains. Our results                         the given time points, directly cooled on ice, washed with phosphate-buffered <br /> show that different temperature-sensitive mutations differen-                          saline (PBS), and mounted on slides. For phase contrast and fluorescence mi- <br /> tially affect the integrity of both complexes and that each                            croscopy, a Leica DMLB microscope was used. DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phe- <br /> TAFII25 mutant allele influences the expression pattern of                             nylindole) staining was performed at a concentration of 1 &#226;&#65533;&#174;g/ml. <br /> different and unique sets of genes. However, taken together,                              FACS. Prior to fluorescence-activated cell sorting (FACS) analysis, exponen- <br />                                                                                        tially growing cells in full medium were synchronized in G2/M phase by the <br /> our TAFII25 mutations affect the transcription level of about                          addition of nocodazole to a final concentration of 20 &#226;&#65533;&#174;g/ml. G2/M arrest of the <br /> 64% of all class II genes, similar to the number of genes                              cells was also monitored microscopically. When 90 to 100% of cells were arrested <br /> affected by a combination of SAGA and TFIID mutants, as                                in the G2/M phase, cells were collected by centrifugation, washed twice with <br /> published by Lee et al. (42).                                                          ice-cold YPD medium, resuspended in warm (37&#194;&#176;C) YPD for temperature shift, <br />                                                                                        and further incubated at 37&#194;&#176;C. Aliquots were taken from exponentially growing <br />                                                                                        cultures (28&#194;&#176;C) after synchronization by nocodazole 10 min after the shift at 37&#194;&#176;C <br />                          MATERIALS AND METHODS                                         (t &#226;&#171;&#189; 0) and then each hour for 3 h (Fig. 4). <br />    Strains, medium, and yeast cell transformation. All strains used in this study         For FACS analysis, samples were prepared as described previously (43) except <br /> (Table 1) are isogenic DKY12 generated from strain PL3(2n) (60). S. cerevisiae         that the incubation time for the RNase treatment and pepsin digestion was <br /> strains were propagated according to standard procedures in either rich medium         doubled. The samples were analyzed on a FACScan (Becton Dickinson) <br /> (YPD) or appropriate selective medium (SD) without tryptophan and/or uracil.           equipped with a single argon ion laser. A minimum of 20,000 cells per sample <br /> For temperature shift experiments, cells were grown at 28&#194;&#176;C until mid-log phase        were analyzed using CellQuest software (Becton Dickinson). <br /> (optical density at 600 nm [OD600], 0.3 to 0.5). Prior to shifting to 37&#194;&#176;C, an equal      Whole-cell extract preparation, Western blot analysis, and antibody produc- <br /> volume of warm (46&#194;&#176;C) medium was added to the cultures, and the cultures were          tion. Cultures were grown to mid-log phase at 28&#194;&#176;C or shifted for 4 h to 37&#194;&#176;C as <br /> transferred to 37&#194;&#176;C for the indicated time. Standard S. cerevisiae genetic manip-      described above, then cells were harvested by centrifugation for 5 min at 3,400 <br /> ulations were performed as described previously (30). Yeast cell transformation        rpm, washed in cold H2O, and recentrifuged, and the pellet was frozen at &#226;&#171;&#186;80&#194;&#176;C <br /> was carried out as described previously (24).                                          and thawed. One volume of buffer A (40 mM Tris-HCl [pH 7.8], 350 mM NaCl, <br />    Disruption of genomic TAFII25 locus. PCR-based homologous recombination             0.1% Tween, 10% glycerol, 1&#226;&#171;&#187; protease inhibitor cocktail [1&#226;&#171;&#187; PIC; 2.5 mg each <br /> was used to replace the entire coding region of one allele of the endogenous           of leupeptin, pepstatin, chymostatin, antipain, and aprotinin per ml]) was added <br /> TAFII25 gene in strain PL3(2n) by the kanamycin resistance gene (KanMX2)               to the pellet, and cells were broken by addition of acid-washed glass beads (0.5 <br /> (78). Cells were transformed with the centromeric URA3 plasmid (PRS316) (70)           mm diameter; Sigma). Tubes were vortexed vigorously 6 times for 20 s each with <br /> harboring the wild-type yTAFII25 cDNA. The DKY12 strain was obtained from              1 min of cooling on ice. Whole-cell extracts were clarified by centrifugation for <br /> isolated spores by selection for kanamycin resistance and uracil auxotrophy. The       30 min at 4&#194;&#176;C, and the protein concentration was determined. Protein extracts <br /> proper gene replacement was verified by Southern blot analysis and PCR on the          were stored at &#226;&#171;&#186;80&#194;&#176;C. <br /> genomic DNA.                                                                              Whole-cell extracts (10 &#226;&#65533;&#174;g) were loaded on a sodium dodecyl sulfate (SDS)&#226;&#8364;&#8220; <br />    Construction of truncations and deletions of yTAFII25 and of chimeras be-           10% polyacrylamide gel, separated by electrophoresis, and electroblotted. The <br /> tween yTAFII25 and hTAFII30. The TAFII25 constructs listed in Fig. 1 were              blots were then treated with the indicated antibodies, followed by incubation with <br /> generated by PCR, digested with XhoI and BamHI, and cloned in-frame into the           peroxidase-conjugated goat anti-rabbit immunoglobulin G antibody (Jackson <br /> corresponding sites of the 2&#226;&#65533;&#174;m expression shuttle vector YEP10-FL (TRP1)               Immuno Research). Chemiluminescence detection was performed according to <br /> (60), containing a Flag epitope tag sequence in-frame and N-terminal to the            the manufacturer&#226;&#8364;&#8482;s instructions (Amersham). Specific polyclonal rabbit antibod- <br /> XhoI site. The inserts in the YEP10-FL plasmid are expressed under the control         ies recognizing TFIID- and SAGA-specific subunits were generated and purified <br /> of the phosphoglycerate kinase (PGK) promoter (60). All the constructs were            as described previously (22, 67, 68). <br /> verified by sequencing. PCR primers used to generate the different fragments are          Two-hybrid assays. The L40 S. cerevisiae strain with a lexA-driven &#226;&#65533;&#164;-galacto- <br /> available upon request.                                                                sidase reporter gene was transformed by the polyethylene glycol-lithium acetate <br />    Complementation and temperature sensitivity tests. To determine the abilities       method with the plasmid constructs as described in the text and earlier (20, 21). <br /> of the yTAFII25 mutants to complement the TAFII25 gene, a plasmid shuffle              The two-hybrid assay was done as described previously (76). <br /> technique was used (8). Strain DKY12 was transformed with plasmid YEP10FL                 Immunoprecipitation analysis. Immunoprecipitations were carried out using <br /> containing either wild-type or mutant alleles of TAFII25, and the transformants        anti-Flag M2&#226;&#8364;&#8220;agarose beads (Sigma) equilibrated in buffer A. The different <br /> were selected for Ura&#226;&#171;&#185; and Trp&#226;&#171;&#185; prototrophy. Positive colonies were transferred        TAFII25 mutant-containing whole-cell extracts (5 mg) were mixed with 300 &#226;&#65533;&#174;l of <br /> to 5-fluoro-orotic acid plates to screen for cells that had lost the PRS316-           M2 beads each and incubated for 12 h at 4&#194;&#176;C by rotation. Beads were washed <br /> TAFII25&#226;&#171;&#185; plasmid. Complementing Trp&#226;&#171;&#185; mutants were further maintained on                three times with 10 volumes of buffer A and then three times with 10 volumes of <br /> medium with URA and without TRP. For the temperature sensitivity tests, the            buffer B (40 mM Tris-HCl [pH 7.9], 150 mM NaCl, 0.1% Tween, 10% glycerol, <br /> mutants cells were grown in YPD medium to a density of 0.4 &#226;&#171;&#187; 107 to 1 &#226;&#171;&#187; 107            1&#226;&#171;&#187; PIC). Bound protein complexes were eluted with 200 &#226;&#65533;&#174;l of buffer B contain- <br />  <br />  <br />  <br />  <br />    FIG. 1. Mapping the functional domains of yTAFII25. (A) Schematic representation of the predicted secondary structure of the yTAFII25 and <br /> the hTAFII30 proteins. Amino acids important for the generation of the different mutants are indicated for the S. cerevisiae (top) and human <br /> (bottom) proteins. The &#226;&#65533;&#163; helices (&#226;&#65533;&#163;) and the loops (L) of the histone fold motif are also shown. (B) Schematic representation of the different <br /> deletion and chimeric mutants of yTAFII25 and hTAFII30. yTAFII25 and its derivatives are shown in grey, with the nonconserved insertion <br /> indicated by the hatched box (L2). The hTAFII30 sequences are indicated in black. Chimeric constructs between the S. cerevisiae and the human <br /> protein are characterized by their color and labeled with their amino acid positions. In the complementation assay, carried out at 28&#194;&#176;C, a &#226;&#171;&#185; <br /> indicates the ability of a construct to complement the lethal &#226;&#338;&#172;taf25 <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> and a &#226;&#171;&#186; indicates noncomplementation. Asterisks indicate the <br /> severity of the slow-growth phenotype of the mutants tested at 28&#194;&#176;C. Temperature sensitivity (ts) at 37&#194;&#176;C is indicated by a &#226;&#171;&#185;. When the mutant <br /> is viable at the elevated temperature, it is indicated by &#226;&#171;&#186;, while &#226;&#171;&#185;/&#226;&#171;&#186; indicates slow growth at 37&#194;&#176;C. Cell cycle arrest phenotypes are indicated by <br /> whether they arrest in the G1 or the G2/M (G2) phase of the cell cycle. (C) The noncomplementing yTAFII25 and hTAFII30 mutant proteins <br /> expressed before the plasmid shuffle. Whole-cell extracts were made from the indicated strains and analyzed by Western blotting using either the <br /> anti-Flag monoclonal antibody (lanes 1 to 4) or the 4G2 anti-hTAFII30 monoclonal antibody (lanes 5 to 8), which recognizes the C-terminal end <br /> of hTAFII30 (35). (D) The complementing yTAFII25 mutants and hTAFII30 chimeras are not overexpressed after the plasmid shuffle compared <br /> to yTAFII25 levels in wild-type PLa cells (wt). Whole-cell extracts were made from the indicated strains and analyzed by Western blot using <br /> polyclonal anti-yTAFII25 antibodies. The same blot was exposed for either 2 (lanes 1 to 3) or 40 s. To facilitate the comparison, Del. 32 is shown <br /> on both exposures. The wild-type yTAFII25, the Flag-tagged full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> yTAFII25, and the different TAFII25 mutant proteins are indicated with <br /> arrowheads. NS, nonspecific. <br />  VOL. 22, 2002   TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION   3181 <br />  3182       KIRSCHNER ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br /> ing 2.5 mg of the Flag peptide per ml for 3 h at 4&#194;&#176;C. From these eluates, equal       strain was normal at 28&#194;&#176;C but slightly impaired at 37&#194;&#176;C (Fig. <br /> volumes (15 &#226;&#65533;&#174;l) were analyzed by Western blot.                                        1B, Del. 100). However, a further 3-amino-acid deletion (ami- <br />    RNA isolation and DNA microarray analysis. For RNA isolation, two distinct <br /> cultures of each mutant were grown in YPD medium at 28&#194;&#176;C to an OD600 of 0.25. <br />                                                                                       no acids 87 to 89; strain Del. 72) resulted in a temperature- <br /> One volume of YPD medium at 46&#194;&#176;C was added to bring each culture to 37&#194;&#176;C.             sensitive phenotype characterized by slow growth at 28&#194;&#176;C and <br /> The cultures were then incubated for 45 min at 37&#194;&#176;C. Following this, the cultures     nonviability at 37&#194;&#176;C (Fig. 1B and 2). Further N- or C-terminal <br /> were immediately cooled, pelleted in cooled tubes for 5 min at 3,400 rpm, washed      deletions were not able to complement the TAF25 null phe- <br /> in cold H2O, repelleted, and frozen in liquid nitrogen. Total RNA was isolated <br />                                                                                       notype (Del. 41 to 43 and Del. 105 in Fig. 1B). <br /> by the hot phenol-chloroform method using RNA-Solv reagent (Omega-Blotem) <br /> following the manufacturer&#226;&#8364;&#8482;s instructions and subsequently quantified. Synthesis         Thus, the shortest deletion mutant that is sufficient to sup- <br /> of cDNA and biotin-labeled cRNA was performed according to the recommen-              port viability with no obvious growth defects at 28&#194;&#176;C on full <br /> dations of Affymetrix Inc. and previously published protocols (45). The frag-         medium and no temperature-sensitive phenotype at 37&#194;&#176;C con- <br /> mented cRNA was hybridized to Affymetrix oligonucleotide arrays (GeneChip             tains less than 50% of the wild-type yTAFII25 protein (Del. 71 <br /> YG-S98). Absolute and comparison analyses were conducted using Microarray <br /> Suite 4.0 software. The total fluorescence intensity for each array was scaled to <br />                                                                                       in Fig. 1B), i.e., a 91-amino-acid region (from amino acids 84 <br /> a uniform value by normalizing the average intensity of all genes (total intensity/   to 149 and from 180 to 206) including the previously described <br /> number of genes) to a fixed value of 1,000.                                           HFD (20). These results show that the evolutionarily con- <br />    The quality of total RNA and cRNA (before and after fragmentation) was             served HFD-containing region (including loop N) of yTAFII25 <br /> verified by capillary electrophoresis and on gels. Before application to experi- <br />                                                                                       is necessary and sufficient for viability. <br /> mental arrays, cRNA quality was confirmed on test chips. Hybridization quality <br /> was controlled with bacterial cRNAs.                                                     In an attempt to dissect the HFD further and determine <br />    Data analysis was carried out with Microarray Suite 4.0 and Data Mining Tool       important structural features, yTAFII25 was analyzed by Bal- <br /> 2.0 from Affymetrix, Inc. A change in mRNA transcription levels as detected by        last (61), a program to predict local maximum segments (i.e., <br /> the gene chips was considered significant if the change was at least twofold up or    sequence segments conserved relative to their flanking re- <br /> down when comparing a mutant strain to the wild-type strain at the nonpermis- <br /> sive temperature. The twofold cutoff was chosen after analyzing a duplicate <br />                                                                                       gions). Two short amino acid sequences, the PIIPD and <br /> sample preparation of the control TAFII25 wild-type strain PL3a at 37&#194;&#176;C. Com-         EYGLN motifs (amino acids 86 to 90 and 194 to 198, respec- <br /> paring changes in expression between two independent samples using the same           tively, in yTAFII25), were identified by Ballast as presenting <br /> criteria as for comparing mutant samples to wild-type samples gave 40 genes           the highest degree of conservation. As described above, dele- <br /> whose expression was more than twofold up or down (0.63%) and only 4 genes <br />                                                                                       tion of the PIIPD motif located in the loop N region (N-loop) <br /> that were more than threefold up or down (0.063%) (see Fig. 7 for a represen- <br /> tation of the average difference change comparison).                                  between the &#226;&#65533;&#163;N and &#226;&#65533;&#163;1 helices of the HFD is important for <br />                                                                                       yTAFII25 function (Del. 72 in Fig. 1B and 2). Deletion of the <br />                                                                                       EYGLN motif C-terminal to the &#226;&#65533;&#163;3 helix did not affect growth <br />                                   RESULTS <br />                                                                                       under the conditions tested at 28&#194;&#176;C but caused temperature <br />    A 91-amino-acid region of yTAFII25 containing the putative                         sensitivity at 37&#194;&#176;C (Del. 92 in Fig. 1B and Fig. 2), indicating a <br /> HFD is necessary and sufficient for vegetative growth in S.                           potential role of this region in the stability of the complexes <br /> cerevisiae. Yeast TAFII25 protein is encoded by a single-copy                         (see below). <br /> essential gene (37) containing a putative HFD (20) (see also                             Neither full-length human TAFII30 nor its conserved HFD <br /> Fig. 1A). To study the structure-function relationship of                             can substitute for yTAFII25 function, but partial substitutions <br /> TAFII25, an S. cerevisiae &#226;&#338;&#172;taf25 strain complemented with                             in the HFD support viability and give rise to temperature- <br /> yTAFII25 expressed from a URA/CEN plasmid was generated.                              sensitive phenotypes. To study the functional conservation be- <br /> Subsequently the plasmid shuffle technique (7) was used to                            tween S. cerevisiae TAFII25 and its human homologue hTAFII30, <br /> introduce plasmids expressing different Flag epitope-tagged                           full-length hTAFII30 or its conserved HFD was tested for <br /> yTAFII25 wild-type and mutant proteins (Fig. 1B). Protein                             complementation in the TAF25 null strain. Neither full-length <br /> expression was verified either before (noncomplementing mu-                           hTAFII30 nor its N-terminally truncated version, which retains <br /> tants; Fig. 1C) or after (complementing mutants; Fig. 1D) the                         the entire HFD (&#226;&#338;&#172;N-hTAFII30), was able to complement <br /> plasmid shuffle.                                                                      yTAFII25 (Fig. 1B), while the presence of these proteins is <br />    Although complementing yTAFII25 derivatives were ex-                               easily detectable (Fig. 1C). A chimera comprising the N-ter- <br /> pressed from a 2&#226;&#65533;&#174;m plasmid under the control of the strong                            minal 73 amino acids of S. cerevisiae TAFII25 fused to the <br /> PGK promoter, none of the mutant TAFII25 proteins resulting                           conserved HFD of hTAFII30 was also unable to rescue the <br /> in a temperature-sensitive phenotype were overexpressed com-                          TAF25 null phenotype (Chim. 21, Fig. 1B and C). <br /> pared to yTAFII25 expression in a wild-type strain (Fig. 1D).                            Thus, despite the relatively high amino acid conservation in <br /> The proteins that were overexpressed from the 2&#226;&#65533;&#174;m plasmid                             the HFD of the S. cerevisiae and human proteins (67% simi- <br /> were the full-length TAFII25 and Del. 32, which did not result                        larity and 43% identity), the human protein cannot function- <br /> in detectable phenotypes (Fig. 1B and D). In complementation                          ally replace its S. cerevisiae counterpart. This can be explained <br /> tests, the N-terminal nonconserved 73 amino acids of TAFII25                          by the fact that in two-hybrid interaction assays, the HFD of <br /> (&#226;&#338;&#172;N-yTAFII25 in Fig. 1B) and the nonconserved extended                                hTAFII30 interacts with yTAFII47 but not with the two other <br /> loop 2 (L2) of the potential HFD (between amino acids 149                             yTAFII25 interaction partners, the TFIID subunit yTAFII65 <br /> and 180; Del 32. in Fig. 1B) (23) could be deleted without any                        and the SAGA component SPT7 (20). <br /> consequence on vegetative growth at 28&#194;&#176;C (Fig. 2).                                       We then analyzed chimeric proteins in which predicted <br />    Interestingly, a well-conserved 10-amino-acid region be-                           structural motifs within the conserved minimal yTAFII25-com- <br /> tween amino acids 74 and 83, encoding the putative &#226;&#65533;&#163;N-helix                           plementing region were replaced by the corresponding regions <br /> of the HFD (&#226;&#65533;&#163;N in Fig. 2), could also be deleted without                              of hTAFII30. To this end, either the first two (&#226;&#65533;&#163;N and &#226;&#65533;&#163;1) or <br /> affecting growth (Del. 71 in Fig. 1B). When a further 3 amino                         the second two (&#226;&#65533;&#163;2 and &#226;&#65533;&#163;3) &#226;&#65533;&#163;-helices of yTAFII25 were re- <br /> acids were deleted (amino acids 84 to 86), the growth of this                         placed by the corresponding human HFDs (Chim. 37 and 38 in <br />  VOL. 22, 2002                                          TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                           3183 <br />  <br />  <br />  <br />  <br />  FIG. 2. Several yTAFII25 mutants show temperature-sensitive phenotypes. The indicated strains were spotted in 10&#226;&#171;&#186;1 serial dilutions on YPD <br /> medium and tested for growth at 28&#194;&#176;C (left panel) and at 37&#194;&#176;C (right panel) in comparison to the strain expressing full-length yTAFII25. <br />  <br />  <br /> Fig. 1B). The fusion protein containing the &#226;&#65533;&#163;N and &#226;&#65533;&#163;1 helices          cell cycle defects. Next, the morphological changes of the <br /> from the human protein and &#226;&#65533;&#163;2 and &#226;&#65533;&#163;3 from the S. cerevisiae            above-described TAFII25 temperature-sensitive mutant strains <br /> protein did complement the TAF25 null phenotype (Chim. 38              were analyzed at 28 and 37&#194;&#176;C by light microscopy, and the <br /> in Fig. 1B and 2), whereas the reverse chimera did not (Chim.          nuclei were visualized by using fluorescence microscopy fol- <br /> 37, Fig. 1B and C, lane 8). This suggests that the &#226;&#65533;&#163;2 and &#226;&#65533;&#163;3           lowing DAPI staining. At the permissive temperature, only <br /> helices of the HFD have species-specific function, probably            slight differences were found between strains (Fig. 3). For <br /> involved in interaction specificity, whereas the &#226;&#65533;&#163;N and &#226;&#65533;&#163;1 re-         example, cells containing Chim. 38 often had elongated cell <br /> gions do not interfere with the essential functions of the S.          and bud morphologies (Fig. 3). At the nonpermissive temper- <br /> cerevisiae protein under normal growth conditions (YPD,                ature (37&#194;&#176;C), most of the temperature-sensitive strains showed <br /> 28&#194;&#176;C). The fusion protein Chim. 38 showed, however, a tem-             abnormal morphologies that could be divided in two catego- <br /> perature-sensitive phenotype at 37&#194;&#176;C (Fig. 1B and 2), indicat-         ries: (i) a high proportion of cells with no or &#226;&#8364;&#339;mini&#226;&#8364;&#65533; buds <br /> ing that the &#226;&#65533;&#163;N and &#226;&#65533;&#163;1 regions of the S. cerevisiae protein are        together with an aberrant nuclear morphology (Fig. 3, Del. 72 <br /> not completely interchangeable with the homologous human               and Del. 92) and (ii) large cells with elongated bud morphol- <br /> sequences and that they may be involved in protein stability           ogies (10 to 30% of cells with multiple buds) and with several <br /> and/or protein interaction determination (see two-hybrid data          &#226;&#8364;&#339;stained bodies&#226;&#8364;&#65533; in the same cell (Fig. 3, Chim. 38, Chim. 97, <br /> below).                                                                and Del. 98). <br />    To dissect which region of the &#226;&#65533;&#163;N and &#226;&#65533;&#163;1 helices of the <br />                                                                           Since most of these morphological changes are typical of cell <br /> human protein was responsible for the temperature-sensitive <br />                                                                        cycle defects and as some other TAFII temperature-sensitive <br /> phenotype, we made three further chimeras in which only the <br />                                                                        mutants were shown to arrest at different cell cycle stages (see <br /> &#226;&#65533;&#163;N helix, only the N-loop, or only the &#226;&#65533;&#163;1 helix from the human <br />                                                                        the introduction), we analyzed the DNA content of these mu- <br /> protein were exchanged with the corresponding regions of the <br />                                                                        tants by FACS. Cells were grown at 28&#194;&#176;C to mid-log phase and <br /> S. cerevisiae protein (Chim. 96 to 98 in Fig. 1B). Chim. 96, <br />                                                                        synchronized with nocodazole, a drug that blocks cells in the <br /> containing the human &#226;&#65533;&#163;N helix, fully complemented the <br />                                                                        G2/M phase (2 n). Following synchronization, nocodazole was <br /> TAF25 null phenotype, in good agreement with the above <br /> deletion studies (Fig. 1B, compare Chim. 96 and Del. 71),              eliminated and cells were shifted to 37&#194;&#176;C for 3 h. The DNA <br /> further indicating that the &#226;&#65533;&#163;N helix is not contributing to any        content was measured every 60 min (Fig. 4). <br /> essential TAFII25 function under the conditions tested. How-              Control strains (PL&#226;&#65533;&#163; and Del. 32) grew asynchronously at <br /> ever, the other two chimeras, containing either the human              28&#194;&#176;C (with about 55% of control strain cells having n and 45% <br /> N-loop region (Chim. 97) or the human &#226;&#65533;&#163;1 helix (Chim. 98),             having 2 n DNA content), after nocodazole treatment almost <br /> grew normally at 28&#194;&#176;C, albeit somewhat slower than the wild            all cells were arrested in G2/M phase with 2 n DNA content, <br /> type, but were temperature sensitive at 37&#194;&#176;C, Chim. 97 to a            and after release the cells rapidly entered G1 phase with 1 n <br /> lesser degree (Fig. 1B and 2).                                         DNA content; after 3 h at 37&#194;&#176;C they again became asynchro- <br />    It should be noted that the exchange of the S. cerevisiae for       nous (Fig. 4). In contrast, Del. 72 and Del. 92 strains could not <br /> the human sequences in Chim. 97 and 98 reflects only three-            be completely arrested with nocodazole in G2/M phase, with 10 <br /> and six-amino-acid differences, respectively, compared to wild-        to 30% of the cells remaining with 1 n DNA content in G1 <br /> type yTAFII25. These results, in addition to the two-hybrid            phase (Fig. 4). After release from the arrest and shift to the <br /> data below, strongly suggest that the N-loop and &#226;&#65533;&#163;1 regions of         nonpermissive temperature, both strains accumulated with 1 n <br /> TAFII25 are important function-determining regions and are             DNA content in G1 phase. After 3 h at 37&#194;&#176;C, about 95% of <br /> involved in interaction specificity.                                   Del. 72 and 70% of Del. 92 cells had accumulated in G1 (Fig. <br />    Distinct TAFII25 mutants have different morphology and              4). <br />  3184     KIRSCHNER ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Changes in cell morphology and nuclear organization of distinct yTAFII25 mutants upon shift to the nonpermissive temperature. <br /> Morphological changes in the yTAFII25 temperature-sensitive mutants were visualized by phase contrast microscopy (CT). Nuclei were visualized <br /> by fluorescence microscopy with DAPI staining. In the two upper lines, S. cerevisiae strains are shown at 28&#194;&#176;C, whereas in the two bottom lines, <br /> strains were shifted for 4 h to 37&#194;&#176;C. <br />  <br />  <br />  <br />    In contrast, Chim. 38 cells, which already showed a higher             The fact that human TAFII30 does not complement the TAF25 <br /> proportion of cells with 2 n DNA content relative to the con-             deletion and that small sequence exchanges between the S. <br /> trol strains at 28&#194;&#176;C (about 30% of cells have 1 n and 70% have            cerevisiae and human homologues in the HFD protein-protein <br /> 2 n DNA content), were synchronized efficiently with nocoda-              interaction domain caused severe growth defects or a temper- <br /> zole in G2/M phase. After release from the arrest and shift to            ature-sensitive phenotype (Del. 72, Del. 92, Chim. 38, Chim. <br /> the nonpermissive temperature Chim. 38 showed a slow pro-                 97, and Chim. 98 in Fig. 1B and 2) prompted us to test in a <br /> gression to G1 phase, and about 70% of the cells arrested in              two-hybrid assay whether these mutants would interact with <br /> G2/M after 3 h at 37&#194;&#176;C (Fig. 4). Chim. 98 behaved like Chim.              the previously described interaction partners, yTAFII47, <br /> 38, although the G2/M arrest was less pronounced (Fig. 4).                yTAFII65, and ySPT7 (20). <br />    All these results were confirmed by using another synchro-                Full-length yTAFII47 and yTAFII65 and the HFD of ySPT7 <br /> nization method (&#226;&#65533;&#163;-factor) and by shifting the cells to the               (amino acids 964 to 1051) were fused to the VP16 activation <br /> nonpermissive temperature without prior synchronization                   domain, and the full-length TAFII25 and Del. 32, Del. 72, Del. <br /> (data not shown). Finally, mutant Chim. 97 did not show a                 92, Chim. 38, Chim. 97, and Chim. 98 mutants were fused to <br /> pronounced arrest at any particular cell cycle phase in any of            the LexA DNA-binding domain. LexA DNA-binding domain <br /> the cell cycle assays performed (Fig. 4 and data not shown).              fusion proteins were then tested for interaction with the dif- <br /> Thus, our TAFII25 mutants can be classified in three groups:              ferent VP16 fusions in a series of S. cerevisiae two-hybrid ex- <br /> (i) Del. 72 and Del. 92, arresting in G1; (ii) Chim. 38 and Chim.         periments at either 28 or 37&#194;&#176;C (Fig. 5). Interactions were <br /> 98, arresting G2/M; and (iii) Chim. 97, for which no specific cell        assessed by measuring specific &#226;&#65533;&#164;-galactosidase activity in the S. <br /> cycle arrest stage could be determined.                                   cerevisiae reporter strain L40, which harbors a LexA-respon- <br />    These data together demonstrate that S. cerevisiae TAFII25             sive lacZ gene (77). <br /> is involved in proper cell cycle regulation and that distinct                LexA-TAFII25 in the presence of VP16 alone does not ac- <br /> TAFII25 mutants display different morphology and cell cycle               tivate the reporter gene (20). Also, with the exception of <br /> defects. Moreover, they suggest that mutations Del. 72 and                TAFII47, neither of the unfused TAFII25 interaction partners, <br /> Del. 92 affect TFIID (or TFIID and SAGA) function, which is               TAFII65 and SPT7, activated transcription in the presence of <br /> known to arrest cells in G1 phase, whereas the Chim. 38 and               LexA-TAFII25 (data not shown). Activation of the reporter by <br /> Chim. 98 mutants, which arrest in G2/M phase, might rather                TAFII47 (coexpressed with LexA-TAFII25) alone was low but <br /> compromise SAGA function (see below and Discussion).                      detectable (870 U of specific &#226;&#65533;&#164;-galactosidase activity, com- <br />    Distinct TAFII25 temperature-sensitive mutants display dif-            pared to 3,500 U with the VP16 fusion; see Fig. 5, left panel). <br /> ferential interaction profiles with their interaction partners.           Since this activity of TAFII47 is also the result of a specific <br />  VOL. 22, 2002                                             TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                                3185 <br />  <br />  <br />  <br />  <br />    FIG. 4. Distinct yTAFII25 temperature-sensitive mutants accumulate at different cell cycle stages. Cells were grown in YPD medium at 28&#194;&#176;C <br /> and synchronized with nocodazole. The nocodazole was eliminated, and the cells were shifted to 37&#194;&#176;C. Aliquots were taken from exponentially <br /> growing cultures (28&#194;&#176;C) after synchronization by nocodazole (Noco.), 10 min after the shift at 37&#194;&#176;C (t &#226;&#171;&#189; 0), and then each hour for 3 h. Cells were <br /> fixed, stained with propidium iodide, and analyzed by FACS. Wild-type and Del. 32 were used as the reference strains. Strains carrying the Del. <br /> 72 and Del. 92 mutations accumulate in G1 phase during the 5 h at 37&#194;&#176;C, whereas Del. 38 and Del. 98 strains accumulate in G2/M phase after the <br /> shift to 37&#194;&#176;C. The x axis indicates the DNA content (1 n or 2 n), the y axis indicates the cell number, and the z axis indicates the time. For each <br /> histogram, approximately 20,000 particles were gated. <br />  <br />  <br /> recruitment by LexA-TAFII25 to the promoter, it does not                    wild-type TAFII25. In this assay, Chim. 92 showed interactions <br /> interfere with our analysis of TAFII25 mutants affecting an                 with both SPT7 and TAFII65 (similar to the wild type) but a <br /> interaction with the TAFII25 interaction partners.                          reduced level of interaction with TAFII47. <br />    Full-length TAFII25 and Del. 32 (Fig. 1B) showed qualita-                   The two-hybrid interaction results suggest that Chim. 98 <br /> tively similar interactions with yTAFII47, yTAFII65, and ySPT7              and, more specifically, Chim. 38 have a defect in interaction <br /> (Fig. 5). The effect of the temperature shift to 37&#194;&#176;C on the                with SPT7 and that the observed phenotypes for these two <br /> protein interaction is in general (with a few exceptions) not               proteins are at least partially due to an impaired interaction <br /> very pronounced. This could be due to a stabilization of the                with this protein in the SAGA complex. In contrast, Del. 72, <br /> mutant proteins by their fusion partners or a shift in the ther-            which displayed a slow-growth phenotype already at 28&#194;&#176;C, had <br /> modynamic equilibrium under two-hybrid conditions, where                    no obvious defect in SPT7 interaction but was completely im- <br /> both proteins are highly overexpressed. Thus, a small decrease              paired for interaction with TAFII65 and, of the mutants tested, <br /> in interaction strength between two proteins in this assay may              had the most severe reduction in interaction with TAFII47. <br /> reflect a severe functional defect in vivo at endogenous expres-            These results indicate that the deletions in Del. 72 most se- <br /> sion levels. Nevertheless, it is possible to qualitatively distin-          verely affect interactions in TFIID. Surprisingly, Del. 92 did <br /> guish among the different mutants for their capacity to interact            not show any apparent reason in the two-hybrid assay for its <br /> with the TAFII25 interaction partners.                                      functional impairment at the nonpermissive temperature, but <br />    Chim. 97 showed interactions comparable to the wild type                 showed in the SAGA/TFIID immunoprecipitation experi- <br /> with TAFII47 and SPT7 but a two- to fourfold decrease in                    ments (see below) one of the most severe phenotypes at the <br /> interaction with TAFII65. However, mutants Del. 72, Chim. 38,               nonpermissive temperature. <br /> and Chim. 98 all showed severely impaired interaction with                     Taken together, the two-hybrid data support the idea that <br /> both TAFII47 and TAFII65 (Fig. 5). Chim. 38, particularly at                within the HFD of TAFII25, the N-loop and &#226;&#65533;&#163;1 regions are <br /> the nonpermissive temperature, and Chim. 98 also showed a                   important for determining interaction specificity and that the <br /> diminished interaction with ySPT7, with which all the other                 different observed phenotypes of the mutants tested may be <br /> mutants seemed to interact in a manner comparable to that of                linked to changes in this interaction domain, resulting in dif- <br />  3186      KIRSCHNER ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Two-hybrid assay for interaction of TAFII25 deletion mutants and chimeras with the TFIID-specific (TAFII47 and TAFII65) and <br /> SAGA-specific (SPT7) interaction partners. The indicated TAFII25 mutants were expressed as LexA DNA-binding domain fusions; TAFII25 <br /> interaction partners (TAFII47, TAFII65, and SPT7) were expressed as VP16 activation domain (VP16-AD) fusions (amino acid positions are <br /> indicated). The reporter gene is &#226;&#65533;&#164;-galactosidase with four LexA-binding sites in the promoter. The results are shown as specific &#226;&#65533;&#164;-galactosidase <br /> activity. Interactions were tested at 28 and 37&#194;&#176;C (after 4 h of incubation). Shown are the results of at least two qualitatively identical experiments; <br /> the error was less than 20%. None of the LexA-TAFII25 fusion constructs activated the reporter on their own or in the presence of the VP16 <br /> activation domain only (20) (data not shown). <br />  <br />  <br /> ferential interaction with their putative partners, affecting dif-            ponents (i.e., yTAFII90, yTAFII61, yTAFII60, yTAFII47, <br /> ferent complexes.                                                             yTAFII40, and yGCN5) were present at comparable levels, <br />    Distinct TAFII25 mutations differentially affect the integrity             indicating that, independent of the expression level of the <br /> of the TFIID and SAGA complexes. To test whether the loss of                  different TAFII25 mutants, they incorporated similarly into <br /> pairwise interactions observed between the different TAFII25                  endogenous TFIID and SAGA complexes (Fig. 6B). Note that <br /> mutants and their respective interaction partners in the two-                 in these experiments, the Del. 32 strain is a better control <br /> hybrid assay influences the integrity of TFIID and/or SAGA                    because the temperature-sensitive mutants were made in the <br /> complexes, we analyzed the polypeptide composition of these                   Del. 32 mutant context (Fig. 1B). <br /> complexes in the different TAFII25 mutant strains. First, S.                     Surprisingly, in complexes prepared from the Del. 72 strain, <br /> cerevisiae whole-cell extracts were prepared from the wild-type               already at 28&#194;&#176;C two of the TFIID-specific TAFIIs, yTAFII65 <br /> and the mutant strains grown either at 28&#194;&#176;C or for 4 h at 37&#194;&#176;C                and yTAFII130, were missing, and after the shift to 37&#194;&#176;C two <br /> and analyzed by Western blot for the presence of different                    additional SAGA-specific components (SPT7 and ADA2) <br /> TFIID and SAGA components (Fig. 6A).                                          were also lost (Fig. 6B, lanes 3 and 10). The lack of yTAFII65 <br />    In extracts prepared from strains grown at 28&#194;&#176;C, most of the               from the Del. 72-containing complexes is in good agreement <br /> analyzed TAFIIs and SAGA components were comparably ex-                       with the two-hybrid results. Moreover, the Del. 72-containing <br /> pressed (Fig. 6A, lanes 1 to 7). However, in the Del. 72 strain,              complexes contained only trace amounts of TBP at 37&#194;&#176;C (Fig. <br /> the yTAFII65 level was lower than in the other strains. Inter-                6B, lane 10). In the TAFII25-containing complexes prepared <br /> estingly, in extracts prepared from most of the temperature-                  from the Del. 92 strain at 28&#194;&#176;C, most of the tested subunits <br /> sensitive strains grown at the nonpermissive temperature,                     were present (except for yTAFII17), whereas at 37&#194;&#176;C almost all <br /> many of the TFIID and SAGA components were expressed at                       of the tested TFIID and SAGA components were missing (Fig. <br /> similar levels (Fig. 6A, lanes 8 to 14). Only two strains had                 6B, lanes 4 and 11), although most of these components were <br /> altered TAFII expression levels in the whole-cell extracts at                 easily detectable in Del. 92 whole-cell extract at 37&#194;&#176;C (Fig. 6A, <br /> 37&#194;&#176;C: strain Del. 72, in which the yTAFII65 level was lower                   lane 11). This indicates a loss of Del. 92 from both complexes <br /> than in the other strains, and strain Del. 92, in which there was             rather than the disintegration of SAGA and TFIID. <br /> no detectable yTAFII90 or yTAFII65 and very low levels of                        In immunopurified complexes prepared from the Chim. 38, <br /> yTAFII61 and yTAFII17.                                                        Chim. 97, and Chim. 98 strains at 37&#194;&#176;C, all the tested compo- <br />    As all of the different TAFII25 proteins contained a Flag                  nents of TFIID were comparable to those present in the Del. <br /> epitope tag on their N-terminal ends, the TAFII25 wild-type                   32 control complexes (Fig. 6B, lanes 12 to 14). However, the <br /> and mutant proteins were immunoprecipitated with an anti-                     Chim. 38-containing complexes clearly lacked the SAGA-spe- <br /> Flag antibody from the whole-cell extracts. The presence of the               cific components SPT7 and ADA2 (lane 12), and the Chim. <br /> TFIID and SAGA subunits in the immunoprecipitations was                       98-containing complexes lacked SPT7 (lane 14), in agreement <br /> analyzed by Western blot (Fig. 6B). The different TAFII25                     with the above-observed impaired interaction of these two <br /> mutant proteins were completely depleted from the whole-cell                  mutants with SPT7 (see Fig. 5). <br /> extracts by the anti-Flag immunoprecipitation, as a large excess                 These results demonstrate that the temperature-sensitive <br /> of antibody was used in the immunoprecipitations (data not                    phenotype of the tested strains (Del. 72 and 92 and Chim. 38, <br /> shown). Importantly, in the immunopurified TAFII25-con-                       97, and 98) at 37&#194;&#176;C (see Fig. 2) is not the consequence of a <br /> taining complexes, most of the tested TFIID and SAGA com-                     general disintegration of the respective TAFII25-containing <br />  VOL. 22, 2002                                           TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                              3187 <br />  <br />  <br />  <br />  <br />   FIG. 6. Analysis of TFIID and SAGA complex integrity in yTAFII25 wild-type and mutant strains. (A) Immunoblot of whole-cell extracts <br /> (WCE) prepared from yeast cells grown at 28&#194;&#176;C or after a 4-h shift to 37&#194;&#176;C. Cultures were harvested, whole-cell extracts were prepared, and 10 <br /> &#226;&#65533;&#174;g of the respective whole-cell extracts was separated by SDS-PAGE, blotted, and analyzed by Western blot with the indicated antibodies. <br /> (B) Whole-cell extracts prepared from the different strains (as indicated) were immunoprecipitated (IP) using a resin-bound anti-Flag antibody <br /> which recognizes the Flag epitope localized at the N terminus of each construct. Peptide-eluted complexes were separated by SDS-PAGE, blotted, <br /> and analyzed by Western blot with the indicated antibodies. In panels A and B, the bottom panels show the full-length TAFII25 and the different <br /> TAFII25 mutants (indicated with arrowheads) revealed with an anti-Flag antibody. <br />  <br />  <br /> complexes. Instead, the studied TAFII25 mutations affect the              other SAGA subunit expression in whole-cell extracts pre- <br /> TAFII25-containing complexes in several different ways: (i) in            pared from mutant strains can be misleading and that the <br /> the Del. 92 strain at 37&#194;&#176;C, the corresponding TAFII25 muta-               corresponding multiprotein complexes have to be immunopu- <br /> tion is not able to incorporate in either TFIID or SAGA; (ii) in          rified to analyze the weakened subunit interactions. <br /> the slow-growing Del. 72 strain, two TFIID-specific TAFIIs                   Distinct mutations in TAFII25 differentially affect the ex- <br /> (yTAFII130 and yTAFII65) are absent from TFIID already at                 pression of class II genes. To investigate how the different <br /> 28&#194;&#176;C, while at 37&#194;&#176;C additional subunits dissociated from the              TAFII25 mutants influence Pol II transcription, Affymetrix <br /> corresponding complexes without triggering a total disintegra-            DNA microarray analysis was performed (see Materials and <br /> tion of TFIID; (iii) complexes from the Chim. 38 and 98 strains           Methods). Total RNA was prepared from the above-described <br /> lost only SAGA-specific subunits at 37&#194;&#176;C, while TFIID was <br />                                                                           temperature-sensitive strains (Del. 72, Del. 92, Chim. 38, <br /> unaffected; and (iv) complexes from the Chim. 97 strain have <br />                                                                           Chim. 97, and Chim 98) grown at 28&#194;&#176;C or transferred for 45 <br /> a TFIID and SAGA composition similar to that of those pre- <br />                                                                           min to 37&#194;&#176;C. As a reference, we used the wild-type strain PL3a. <br /> pared from the Del. 32 reference strain, which shows no <br />                                                                           To test the reproducibility of the results and to determine a <br /> growth defect. Thus, the distinct TAFII25 mutations differen- <br /> tially affect the composition of the corresponding TFIID and              cutoff for changes in gene expression, we compared the results <br /> SAGA complexes: Del. 72 seems to affect TFIID (at 28&#194;&#176;C) or                of two independent RNA preparations, each carried out on the <br /> TFIID and SAGA composition (at 37&#194;&#176;C), whereas Chim. 38                    wild-type strain at 37&#194;&#176;C and tested on Affymetrix DNA mi- <br /> and 98 seem to affect SAGA composition specifically (at 37&#194;&#176;C).            croarrays. The results obtained indicate a very high reproduc- <br /> These results appear to be in good agreement with the two-                ibility (Fig. 7). Only about 0.6% of mostly weakly expressed <br /> hybrid interaction assay.                                                 genes changed twofold up or down when the two independent <br />    Moreover, our results, together with the study of Natarajan            preparations were compared. We therefore chose a twofold <br /> et al. (56), demonstrate that conclusions based on yTAFII and             cutoff as being significant for changes in gene expression in our <br />  3188      KIRSCHNER ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Evaluation of microarray data: average difference scatter graph plot of two independent sample preparations analyzed with the <br /> Affymetrix GeneChip. The x and y axes show relative hybridization signal intensities for all genes tested. Higher values correspond to stronger <br /> expression. Two- and threefold change lines are shown. Only genes that were qualified as present by the Affymetrix software in either of the sample <br /> preparations were included (5,734 out of 6,332; see also Materials and Methods and Results). The data show very good reproducibility. Only some <br /> of the weakly expressed genes tend to fall out of the two- or threefold change lines. <br />  <br />  <br />  <br />  <br /> mutant analysis, with the possibility of a 0.6% error (see also             ential requirements for TFIID and SAGA as published by Lee <br /> Materials and Methods).                                                     et al. (42). This list contains four categories of 10 genes: TFIID <br />    A total of 6,332 Pol II genes with a systematic locus name               dependent, SAGA dependent, TFIID and SAGA dependent, <br /> were included in the analysis (Table 2). Of these genes, 64%                and genes whose transcription is affected only in a SAGA <br /> (4,073 genes) were either up- or downregulated more than                    TFIID double mutant. Surprisingly, out of 10 genes in each <br /> twofold at the nonpermissive temperature compared to the                    category, we found between one and four genes unaffected <br /> wild-type strain. These genes include the nonredundant af-                  by all five of our TAFII25 mutants. However, in mutant Del. <br /> fected genes (up- and downregulated) in the five mutant                     72, four genes of the first category (CAM1, WTM2, GNA1, <br /> strains together. In any individual mutant, the number of af-               and MSH6), five genes of the second (PHO84, PHO5, SPL2, <br /> fected genes was between 1,121 and 2,110 (or 18 and 33% of all              YHB1, and YJL012C), four genes in the third (AAH1, FAR1, <br /> genes analyzed) (see Table 2). Interestingly, in the different <br /> mutants, more genes showed increased expression than a de- <br /> crease in transcription, 40 and 30%, respectively.                                         TABLE 2. Total gene expression analysisa <br />    Surprisingly, only 132 genes were similarly affected in all five                                   Change, no. of genes (% of 6,332 analyzed) <br /> mutant strains analyzed. This is either 3.2% of all genes af-                TAFII25 <br />                                                                              mutant          Downregulated         Upregulated        Up- or downregulated <br /> fected by any one of the mutations (4,073 genes) or 2.1% of all                                &#226;&#172;&#381;2-fold              &#226;&#172;&#381;2-fold                  &#226;&#172;&#381;2-fold <br /> genes analyzed (6,332 genes). In addition, when mutant strains <br /> were compared pairwise for genes that are commonly de-                      Del. 72            1,124 (18)            447 (7)               1,571 (25) <br />                                                                             Del. 92              554 (9)             700 (11)              1,254 (20) <br /> creased twofold or more, the overlap was limited to 11% to                  Chim. 38             516 (8)           1,106 (17)              1,622 (26) <br /> 28% or to 5% to 13% when comparing any group of three                       Chim. 97             524 (8)           1,586 (25)              2,110 (33) <br /> mutants. For upregulated genes, comparable results were ob-                 Chim. 98             483 (8)             638 (10)              1,121 (18) <br /> tained. These data show that, based on their affected global                  Total            1,925 (30)          2,512 (40)              4,073 (64) <br /> gene expression pattern, our TAFII25 temperature-sensitive                     a <br />                                                                                  Affymetrix DNA microarray analysis was performed. Total RNA was pre- <br /> mutants cannot be put in well-defined categories. Thus, any                 pared from the Del. 72, Del. 92, Chim. 38, Chim. 97, and Chim 98 strains grown <br />                                                                             at 28&#194;&#176;C and transferred for 45 min to 37&#194;&#176;C. As a reference, we used the wild-type <br /> given individual TAFII25 mutant affects (directly or indirectly)            strain PL3a. Numbers and percentages of genes affected by different TAFII25 <br /> a very specific and only partially overlapping subset of Pol II             mutation at the nonpermissive temperature compared to the wild-type strain <br /> genes.                                                                      PL3a are indicated. Total reflects the nonredundant number of genes up- and <br />                                                                             downregulated by the mutants. Genes were considered affected when their ex- <br />    To further test our mutants for SAGA and/or TFIID spec-                  pression level was at least twofold above or below that of the wild-type control. <br /> ificity, we compared our data to a list of genes showing differ-            The error was estimated to be about 0.6%. <br />  VOL. 22, 2002                                        TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                        3189 <br />  <br />  <br /> YOR095C, and PHO3), and two genes of the fourth category            cointegration of other TFIID- and SAGA-specific components <br /> (HEM4 and TOM7) are downregulated by more than twofold,             (Del. 72 at 37&#194;&#176;C) or is unable to integrate into TFIID and <br /> suggesting that this TAFII25 mutant may affect both SAGA            SAGA complexes (Del. 92 at 37&#194;&#176;C), cells arrest in the G1 <br /> and TFIID activity.                                                 phase, suggesting that TFIID function is crucial for entry into <br />    In contrast, Chim. 38 and Chim. 98 do not downregulate the       the S-phase of the cell cycle. Interestingly, the immunopurified <br /> expression of any gene in the first and fourth categories (with     TFIID and SAGA complexes from the Del. 72 strain were very <br /> the exception of one gene downregulated by Chim. 38 in the          similar when purified from cells grown at 28 or 37&#194;&#176;C; however, <br /> first category), but the expression of five genes is decreased      the Del. 72 cells were able to grow (though extremely slowly) <br /> more then twofold in the second (PHO84, PHO5, SPL2, YHB1,           at 28&#194;&#176;C but not at 37&#194;&#176;C (Fig. 2). One reason for this could be <br /> and YDL124W), SAGA-specific category, and four genes in the         that in the Del. 72-containing complexes, TBP incorporation is <br /> third category (AAH1, FAR1, YOR095C, and YOR306C), sug-             seriously impaired at 37&#194;&#176;C. Moreover, it is possible that weak- <br /> gesting that these two mutants influence gene transcription by      ened interactions between the Del. 72 mutant and yTAFII65 <br /> affecting SAGA function rather than that of TFIID. The re-          and/or the Del. 72 mutant and yTAFII130 are still functional in <br /> sults for the two other mutants, Del. 92 and Chim. 97, are          the cells at 28&#194;&#176;C, allowing them to survive, but these interac- <br /> much less clear when compared to these published lists of           tions may be disrupted during biochemical purification and <br /> genes. The data suggest, however, that different mutations of       TAFII130 as well as TAFII65 may be lost from the immuno- <br /> the same TAFII shared by SAGA and TFIID can specifically            precipitation. <br /> affect the function of one or the other complex.                       The fact that strains containing the Del. 72 and Del. 92 <br />                                                                     mutant proteins arrest in the G1 stage of the cell cycle, together <br />                         DISCUSSION                                  with the observation that in mouse cells, in which the two <br />                                                                     alleles of the vertebrate homologue of the yeast taf25 gene, <br />    The results described in this study indicate that a minimal      TAFII30, have been conditionally disrupted, also arrest in G1 <br /> 91-amino-acid region containing the HFD of the yTAFII25             phase (51), suggests that TAFII25 and its metazoan homo- <br /> protein is necessary and sufficient for vegetative growth. This     logues are required (directly or indirectly) for progression <br /> minimal region may even be shorter, as the last seven amino         through the G1 phase. <br /> acids of yTAFII25 (C-terminal to the HFD) can be deleted               On the other hand, yeast cells containing the Chim. 38 and <br /> without influencing important HFD-HFD interactions be-              Chim. 98 mutant TAFII25 proteins, in which solely the <br /> tween yTAFII25 and yTAFII47 or yTAFII25 and yTAFII47 (20;           polypeptide composition of the SAGA complex is compro- <br /> unpublished data). We also show that the distinct temperature-      mised, accumulate at the G2/M stage. These TAFII25 mutants <br /> sensitive mutations (truncations, deletions, and multiple point     seem to affect the interaction mainly with the SAGA-specific <br /> mutations) in the minimal HFD-containing region of yTAFII25         HFD-containing interaction partner SPT7 in both immunopre- <br /> do not result in a single well-defined phenotype but in different   cipitation and two-hybrid experiments. Moreover, an ada1 null <br /> morphological and cell cycle phenotypes as well as in surpris-      mutation (another SAGA component) results in morphologi- <br /> ingly distinct gene expression patterns.                            cal features (large elongated cells with several elongated buds; <br />    These findings are in agreement with other recent studies        E. vom Baur, unpublished observation) very similar to those of <br /> (15, 36, 52, 74), arguing that most individual temperature-         the Chim. 38 and Chim. 98 mutants at 37&#194;&#176;C (Fig. 3). Thus, <br /> sensitive mutants of a given TAFII cannot be used to evaluate       there seems to be a good correlation between the TAFII25 <br /> all functions of that TAFII in gene regulation. Depending on        mutations that compromise SAGA integrity and mutations of <br /> the nature of the temperature-sensitive mutation, it can result     other SAGA-specific components. These results together sug- <br /> in very different phenotypes and thus reflect many different        gest that when cells are impaired in TFIID (and SAGA) func- <br /> functions of the same protein. Moreover, our observations also      tion, they arrest at the G1 phase of the cell cycle, and that when <br /> show that the effects of a TAFII mutation on its function           mostly SAGA-specific functions are compromizsed, cells accu- <br /> should be studied in the biochemical context of the corre-          mulate in the G2/M phase of the cell cycle. <br /> sponding multiprotein complexes. Our data indicate that ef-            Interestingly, deleting the &#226;&#65533;&#163;N and loop N regions of the <br /> fects on complex integrity need to be determined and that           HFD in strain Del. 72 resulted in cells accumulating in G1 <br /> characterization of SAGA- and TFIID-specific functions of           phase at 37&#194;&#176;C; in contrast, replacing the &#226;&#65533;&#163;N, loop N, and &#226;&#65533;&#163;1 <br /> shared TAFIIs should be possible by using appropriate muta-         regions with the corresponding human sequences (strain Chim. <br /> tions in the histone fold domain.                                   38) resulted in cells accumulating in G2/M phase at 37&#194;&#176;C. The <br />    Different observed phenotypes of tested mutants may be           compositions of TFIID and SAGA complexes at 37&#194;&#176;C are also <br /> linked to changes in TFIID and/or SAGA complexes. Compar-           different in these strains: TAFII25-containing TFIID and <br /> ison of the different results obtained with our yTAFII25 mu-        SAGA complexes from strain Del. 72 seem to be partially <br /> tants suggests that the effect of some mutations can be corre-      dissociated, whereas in the Chim. 38 strain, the SAGA complex <br /> lated with the integrity of the TAFII25-containing complexes.       is affected, but not TFIID (Fig. 6). In agreement with the <br /> Strains in which the TAFII25 gene is deleted are not viable         biochemical data, the gene expression pattern in these two <br /> (37), suggesting that TFIID and SAGA complexes are not              mutants differs significantly (see below). Together, these ob- <br /> functional without TAFII25. It is thus conceivable that the         servations show that temperature-sensitive mutations that are <br /> TAFII25 mutant-containing complexes represent TFIID and             generated in the same region of yTAFII25 can result in a <br /> SAGA complexes which are sufficient for viability at 28&#194;&#176;C.          different composition of the corresponding multiprotein com- <br />    In strains where the mutated yTAFII25 protein is able to         plexes, different phenotypic changes, and different gene ex- <br /> enter the TAFII25-containing complexes but prevents the             pression patterns. <br />  3190     KIRSCHNER ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />    Moreover, the fact that in the Chim. 97 temperature-sensi-      yTAFII47 interaction or that yTAFII47 makes additional con- <br /> tive strain the protein-protein interactions within the TAFII25-   tacts with other proteins in TFIID. <br /> containing complexes are not seriously affected (Fig. 6) sug-         The second possibility is consistent with immunoelectron <br /> gests that other key interactions between the TAFII-containing     microscopy data, which indicated the presence of at least two <br /> complexes and transcription factors and/or the promoter DNA        molecules of yTAFII47 in the same TFIID particle (C. Leu- <br /> are impaired, and this may lead to the observed phenotypic         rent, S. L. Sanders, V. Mallouh, P. A. Weil, D. B. Kirschner, L. <br /> and gene expression changes.                                       Tora, and P. Schultz, submitted for publication). Moreover, <br />    How do the different TAFII25 mutations influence TAFII-         based on two-hybrid data, the Del. 92 mutation does not com- <br /> TAFII or TAFII-SAGA component interactions? Our observa-           pletely destroy the interaction between TAFII47 and TAFII25 <br /> tions suggest that the loop N region of the HFD is crucial for     even at 37&#194;&#176;C, and thus a lack of interaction cannot be the <br /> the interaction between yTAFII25 and yTAFII65, because a           reason for the failure of this TAFII25 mutant to incorporate <br /> deletion of this region (mutant Del. 72) disrupts this interac-    into the TFIID complex at 37&#194;&#176;C. Since the Del. 92 mutation <br /> tion (Fig. 5) and, in good agreement, yTAFII65 (together with      does not show any obvious reason for its functional impair- <br /> yTAFII130) is lost from the Del. 72-containing TFIID complex       ment, we speculate that this mutation may affect other inter- <br /> (Fig. 6B). Interestingly, when the loop N region is present but    actions than those of its known binding partners, and the lack <br /> the S. cerevisiae &#226;&#65533;&#163;1 helix is replaced with the corresponding      of these would then result in a rapid degradation of many other <br /> human sequences (mutant Chim. 98), yTAFII25 fails to inter-        TAFIIs (see TAFII90, TAFII65, TAFII61, and TAFII17 in the <br /> act with yTAFII65 in the two-hybrid assay (Fig. 5), but            whole-cell extracts at 37&#194;&#176;C; Fig. 6A). <br /> yTAFII65 stays associated with the Chim. 98-containing TFIID          It has been shown that the HFD of the SAGA component <br /> complexes (together with yTAFII130; Fig. 6B). These data           SPT7 can interact with yTAFII25 but not with human TAFII30, <br /> suggest that in Chim. 98-containing complexes, other addi-         both in two-hybrid assays and in vitro (20). Comparison of the <br /> tional TAFII-TAFII interactions are able to stabilize the weak-    functional complementation experiment (Fig. 1B) with our <br /> ened Chim. 98-yTAFII65 interaction in TFIID and that the           two-hybrid interaction studies (Fig. 5) and with the SAGA/ <br /> loop N region is important in this stabilization effect. The       TFIID immunoprecipitation experiment (Fig. 6B) suggests <br /> intact HFD of human TAFII30 does not interact with                 that the human loop N region does not influence the TAFII25- <br /> yTAFII65 (20), further underlining the importance of the ad-       SPT7 interaction (Chim. 97), whereas the human &#226;&#65533;&#163;1 region <br /> ditional TAFII-TAFII interactions in our mutants.                  does (Chim. 98). However, when the corresponding mutant <br />    Mutants Chim. 38, which has the entire human &#226;&#65533;&#163;N-N-              contains both of these human regions (Chim. 38), this results in <br /> loop-&#226;&#65533;&#163;1 region, and Chim. 97, which has only the human loop        a severe defect in the two-hybrid TAFII25-SPT7 interaction <br /> N region (with three amino acid differences compared to the S.     and in the parallel disintegration of the SAGA complex. In <br /> cerevisiae sequences) and the remaining S. cerevisiae sequences    contrast, the amino acid changes in the Chim. 38 and Chim. 98 <br /> (Fig. 1B), retain about 10 to 40% of the two-hybrid interac-       mutants do prevent yTAFII47 and yTAFII65 from associating <br /> tions with yTAFII65 and have an unchanged TFIID composi-           with the corresponding TFIID complexes (Fig. 6B). <br /> tion, suggesting that the loop N region is important for the          Thus, introducing either the human &#226;&#65533;&#163;N-N-loop-&#226;&#65533;&#163;1 region of <br /> interaction between yTAFII25 and yTAFII65. Moreover, our           the HFD or the human &#226;&#65533;&#163;1 region of the HFD into the S. <br /> study predicts important interactions among TAFII25,               cerevisiae yTAFII25 backbone creates mutants that seem to <br /> yTAFII65, and TAFII130, as the results obtained with the Del.      mainly impair SAGA composition and function. Importantly, <br /> 72-containing TFIID complex indicate that the loop N regions       both the Chim. 38 and Chim. 98 mutants arrest in the G2/M <br /> of TAFII25 and yTAFII65 are both necessary for the stable          cell cycle phase, suggesting that in both mutants similar, pos- <br /> association of yTAFII130 with TFIID. These interactions are        sibly SAGA-specific functions are affected at the nonpermis- <br /> consistent with immunoelectron microscopy data which               sive temperature. The human &#226;&#65533;&#163;1 region contains six amino <br /> showed that TAFII25, yTAFII65, and TAFII130 are all present        acid substitutions, and thus in the future it will be important to <br /> in close proximity in the same lobe of the yTFIID (9; C.           test which of these six amino acids is crucial for the yTAFII25- <br /> Leurent, S. L. Sanders, V. Mallouh, P. A. Weil, D. B. Kirsch-      SPT7 interaction. <br /> ner, L. Tora, and P. Schultz, submitted for publication). In          Individual TAFII25 mutants influence different allele-spe- <br /> agreement, an interaction between hTAFII250, the human ho-         cific gene expression patterns. The main suggestion that fol- <br /> mologue of yTAFII130, and hTAFII30 has also been reported          lowed from the above observations is that distinct conditionally <br /> (35).                                                              temperature-sensitive mutations in the same TAFII may affect <br />    The HFD of yTAFII25 and that of the human TAFII30 have          transcription in different ways, partially due to specific effects <br /> also been shown to interact both physically and genetically with   on TFIID and/or SAGA function. To test this hypothesis, a <br /> that of the yTFIID component TAFII47 (20). Interestingly,          genome-wide transcription analysis was carried out with DNA <br /> none of the TAFII25 mutants tested completely abolished the        microarrays. We found that only 2.1% of the genome is af- <br /> interaction between yTAFII47 and yTAFII25 in the two-hybrid        fected in the same way in all five mutants, whereas about 64% <br /> assay (however, some of them weakened the interactions by          of global gene expression is affected when all of the nonredun- <br /> five- to sevenfold at 37&#194;&#176;C [Fig. 5]), and the presence of          dant transcription effects on any gene caused by the mutations <br /> yTAFII47 in the immunopurified complexes was not affected          are added. Our gene array data, together with other mutant <br /> (Fig. 6B). Even the Del. 72 mutant, which showed the weakest       TAFII allele analyses (15, 36, 52, 74), argue that results ob- <br /> interaction with TAFII47, did not impair the cointegration of      tained with any single temperature-sensitive mutation in a <br /> TAFII47 into the TFIID complex, suggesting that additional         TAFII does not necessarily reveal all of the functions of this <br /> interactions are able to stabilize the weakened Del. 72-           protein. <br />  VOL. 22, 2002                                        TAF25p MUTATIONS DIFFERENTIALLY AFFECT TRANSCRIPTION                                          3191 <br />  <br />  <br />    It is clear that our yTAFII25 mutants individually did not       the nucleosome bound to DNA showed how numerous the <br /> affect the expression of all genes (between 18 and 33%); how-       histone-DNA contacts are and how little a single or even a few <br /> ever, the sum of the individual contributions revealed a global     contacts influence the overall nucleosome structure (47). <br /> role for yTAFII25 (64% of total genes analyzed). Moreover, it          It is clear that the HFD is for most of the HFD-TAFIIs the <br /> is conceivable that by testing more or other yTAFII25 mutants,      only essential&#226;&#8364;&#8221;usually evolutionary conserved&#226;&#8364;&#8221;part of the <br /> it would be possible to show a combined effect on a larger          protein, whereas N- and C-terminal sequences can be deleted <br /> portion of the genome or maybe on the whole genome. In this         without apparent phenotype. Nevertheless, these sequences <br /> respect, it is interesting that in a recent genome-wide analysis,   may play very important and specific roles at particular pro- <br /> the combined effect of single mutations in several shared           moters, but they may not contribute to the general structural <br /> TFIID and SAGA subunits can account for changes in more             organization of the TAFII-containing complexes. Thus, to un- <br /> than 70% of the genome (42). Unexpectedly, here we found            derstand the role of HFD-TAFIIs, it will be of paramount <br /> that expression of most genes was affected by multiple individ-     importance to understand the organization and three-dimen- <br /> ual mutations in an HFD-containing TAFII shared by TFIID            sional structure of TAFIIs in both TFIID and SAGA. If HFD- <br /> and SAGA.                                                           TAFIIs really form a nucleosome-like structure(s), it will be <br />    As the distinct mutations in the HFD of yTAFII25 differently     important to identify mutations (or combinations of muta- <br /> impair TAFII-TAFII, TAFII-SAGA, and TAFII-other tran-               tions) that effectively destroy the whole structure in one or the <br /> scription interactions (see above) we speculate that there are      other complex to understand their real importance in tran- <br /> different requirements for the specific regions of the TAFII25      scription. <br /> HFD in global gene expression. As our TAFII25 mutants do <br /> not result in a total disintegration of TFIID and/or SAGA                                      ACKNOWLEDGMENTS <br /> complexes (with the possible exception of Del. 92), we think <br />                                                                        We are grateful to F. Winston for antibodies; C. Waltzinger for help <br /> that the differential gene expression patterns observed reflect     in the flow cytometry studies; R. Martin, C. Gaudon, and R. Losson for <br /> distinct functional defects caused by the given mutation in         advice and reagents; U. Kothe for help at certain stages of this study; <br /> TFIID and/or SAGA complexes and that partial complexes              A. Collet for help in the figure preparations; and B. Bell and W. <br /> can still play a role in gene regulation at certain promoters.      Mohan for critically reading the manuscript. We also thank F. <br /> Moreover, the fact that there were only 132 genes whose tran-       Ruffenach for oligonucleotide synthesis. <br />                                                                        D.B.K. was supported by fellowships from the European Commu- <br /> scription was affected in all five mutants in the same way          nity (Marie Curie) and the Fondation pour la Recherche M&#195;&#169;dicale <br /> suggests that these genes are the ones which are the most           (FRM), and E.v.B. was supported by a European Community RTN <br /> sensitive to changes in TAFII25.                                    grant (HPRN-CT-2000-00087). This work was supported by funds <br />    Interestingly, when looking for the expression of the genes      from the Institut National de la Sant&#195;&#169; et de la Recherche M&#195;&#169;dicale, the <br />                                                                     CNRS, the H&#195;&#180;pital Universitaire de Strasbourg, the Association pour <br /> which were proposed to be TFIID, SAGA, and TFIID and                la Recherche sur le Cancer, the FRM, the Ligue Nationale contre le <br /> SAGA dependent (42), we found that Del. 72 affected most of         Cancer, European Community grants HPRN-CT-2000-00087 and <br /> the genes in these categories, suggesting that Del. 72 may affect   HPRN-CT-2000-00088, the Human Frontier Science Program (RG <br /> both TFIID- and SAGA-specific functions. This observation is        196/98) to L.T., and NIH grant GM 52461 to P.A.W. <br /> also in good agreement with our biochemical purification data, <br />                                                                                                        <br /> showing that in Del. 72-containing complexes both TFIID- and <br /> SAGA-specific subunits are lost at 37&#194;&#176;C. Moreover, the Chim.        <br />                                                                       <br /> 38 and Chim. 98 mutants did influence the expression of genes         <br /> in the SAGA-dependent but not in the TFIID- or TFIID- plus              <br />                                                                   <br /> SAGA-dependent categories, further supporting a mostly SA-           <br /> GA-specific defect for these mutants. Thus, it seems possible          <br /> to create mutations in shared TAFIIs that affect mainly TFIID     <br />                                                                       <br /> (and SAGA) or SAGA functions. To identify such SAGA- and                <br /> TFIID-specific mutants, it seems to be necessary to combine             <br /> expression profiles and biochemical analysis of the complexes         <br /> involved.                                                             <br />    The fact, however, that almost every TAFII25 mutant (36;           <br /> this study) shows an individual phenotype with rather limited          <br /> congruity with any other mutant argues against a very specific          <br /> role of TAFII25 (and probably the other HFD-TAFIIs) in me-         <br /> diating specific interactions with distinct activators or pro-         <br /> moter elements. Instead, these observations favor a view in             <br /> which the histone fold domain TAFIIs would play instead a           <br /> structural role in which numerous interactions, both between           <br /> the HFD-TAFIIs and with promoter sequences, activators,              <br /> and/or (general) transcription factors, would contribute to             <br /> transcriptional regulation by the proposed HFD-TAFII nucleo-        <br /> some-like structure(s). Likewise, a great number of histone            <br /> mutations were generated, giving rise to a plethora of pheno-      <br /> types and transcription defects (32). The crystal structure of           <br /> </body></html>