<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 2000, p. 2376&#226;&#8364;&#8220;2386                                                                                                       Vol. 182, No. 9 <br /> 0021-9193/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />            A Rac Homolog Is Required Induction Hyphal Growth <br />                    Dimorphic Yeast Yarrowia lipolytica <br />                         CLEOFE A. R. HURTADO,1 JEAN-MARIE BECKERICH,2 CLAUDE GAILLARDIN,2 <br />                                                                       1 <br />                                           AND RICHARD A. RACHUBINSKI * <br />  <br />              Department Cell Biology, University Alberta, Edmonton, Alberta T6G 2H7, Canada,1 Laboratoire de <br />                           Genetique des Microorganismes, INRA-CNRS, 78850 Thiverval-Grignon, France2 <br />                                                       Received 12 January 2000/Accepted 31 January 2000 <br />  <br />                Dimorphism fungi believed constitute mechanism response adverse conditions represents <br />             important attribute development virulence number pathogenic fungal species. We have <br />             isolated YlRAC1, gene encoding 192-amino-acid protein essential hyphal growth dimorphic <br />             yeast Yarrowia lipolytica represents Rac homolog described fungi. YlRAC1 an <br />             essential gene, deletion does affect ability mate impair actin polarization Y. lipolytica. <br />             However, strains lacking functional YlRAC1 alterations cell morphology, suggesting function <br />             YlRAC1 related aspect polarization cell growth. Northern blot analysis showed that <br />             transcription YlRAC1 increases steadily yeast-<span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>-hypha transition, Southern blot analysis <br />             genomic DNA suggested presence RAC family members Y. lipolytica. Interestingly, strains <br />             lacking functional YlRAC1 able grow pseudohyphal form invade agar, pointing <br />             function YlRAC1 downstream MHY1, previously isolated gene encoding C2H2-type zinc finger <br />             protein ability bind putative stress response elements activity essential hyphal <br />             pseudohyphal growth Y. lipolytica. <br />  <br />  <br />    The members Rho family Ras-related small GTPases                             mechanism shows biochemical morpholog- <br /> (Cdc42, Rac, Rho proteins) signaling molecules that,                              ical features similar apoptosis mammalian cells <br /> like Ras proteins, act molecular switches transduc-                           (30). <br /> ing signals GTP-bound conformation, inac-                                 In yeast, Rho family important function the <br /> tive GTP-bound state (12, 13). Although Rho family                                 budding process known involved control of <br /> GTPases originally thought involved solely                              actin cytoskeleton dynamics response extracellular signals <br /> organization actin cytoskeleton (22, 23, 56, 57, 66, 67,                           (5, 14, 36). This family represented Saccharomyces cerevi- <br /> 70), recent evidence implicated increasing num-                            siae proteins (Rho1p, Rho2p, Rho3p, Rho4p, Cdc42p, <br /> ber vital cellular processes, activation kinase                         Ynl180cp), thought regulate partially over- <br /> cascades, regulation gene expression membrane traf-                                lapping pathways (19). No Rac homolog de- <br /> ficking, cell cycle control, induction apoptosis,                           scribed fungi. <br /> critical regulators oncogenic transformation mammalian                                 The yeast Yarrowia lipolytica received increasing atten- <br /> cells (10, 11, 16, 24, 31, 45, 55, 58, 59, 70, 74). Rac proteins                     tion model study dimorphic transition its <br /> also shown regulate NADPH oxidase                                   ability alternate unicellular yeast form dis- <br /> phagocytes (7) plants (30) induce cell death rice                           tinct filamentous forms (hyphae pseudohyphae). This fact, <br />  <br />  <br />  <br />                                                         TABLE 1. Y. lipolytica strains used study <br />               Straina                                                                                         Genotype <br />  <br /> E122............................................................................MATA ura3-302 leu2-270 lys8-11 <br /> 22301-3 .......................................................................MATB ura3-302 leu2-270 his1 <br /> CHY1220 ...................................................................MATA ura3-302 leu2-270 lys8-11 fi11 <br /> CHY1220-A30 ...........................................................MATA ura3-302 leu2-270 lys8-11 &#226;&#338;&#172;rac1::URA3 <br /> CHY1220-B36 ...........................................................MATB ura3-302 leu2-270 his1 &#226;&#338;&#172;rac1::URA3 <br /> mhy1KO9 ...................................................................MATA ura3-302 leu2-270 lys8-11 &#226;&#338;&#172;mhyl::URA3 <br /> E122//22301-3 ............................................................MATA/MATB ura3-302/ura3-302 leu2-270/leu2-270   lys8-11/&#226;&#171;&#185;   his1/&#226;&#171;&#185; <br /> E122//CHY1220-B36 ................................................MATA/MATB ura3-302/ura3-302 leu2-270/leu2-270           lys8-11/&#226;&#171;&#185;   his1/&#226;&#171;&#185; &#226;&#338;&#172;rac1::URA3/&#226;&#171;&#185; <br /> 22301-3//CHY1220-A30............................................MATA/MATB ura3-302/ura3-302 leu2-270/leu2-270             lys8-11/&#226;&#171;&#185;   his1/&#226;&#171;&#185; &#226;&#338;&#172;rac1::URA3/&#226;&#171;&#185; <br /> CHY1220-A30//CHY1220-B36................................MATA/MATB ura3-302/ura3-302 leu2-270/leu2-270                     lys8-11/&#226;&#171;&#185;   his1/&#226;&#171;&#185; &#226;&#338;&#172;rac1::URA3/&#226;&#338;&#172;rac1::URA3 <br />    a <br />      Strains E122 22301-3 C. Gaillardin, Thiverval-Grignon, France. Strain mhy1KO9 work described previously (26). All strains were <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> study. <br />  <br />  <br />   * Corresponding author. Mailing address: Department Cell Biol- <br /> ogy, University Alberta, Medical Sciences Building 5-14, Edmonton, <br /> Alberta T6G 2H7, Canada. Phone: (780) 492-9868. Fax: (780) 492- <br /> 9278. E-mail: rick.rachubinski@ualberta.ca. <br />  <br />                                                                                    2376 <br />  VOL. 182, 2000                                                                                               DIMORPHISM IN Y. LIPOLYTICA                      2377 <br />  <br />  <br />                                                  TABLE 2. Oligonucleotide primers used study <br />   Primer                                                                Sequence                                                                    Restriction site <br />  <br /> KO1                         5&#226;&#172;&#732;-CTC TCC TGA TCT GCA TCT GAT CTG-3&#226;&#172;&#732; <br /> KO2                         5&#226;&#172;&#732;-TAG CTG AAG ACT CAA TCT GGA GGG-3&#226;&#172;&#732; <br /> T3                          5&#226;&#172;&#732;-CCA AGC TCG AAA TTA ACC CTC ACT AA-3&#226;&#172;&#732; <br /> T7                          5&#226;&#172;&#732;-GTA ATA CGA CTC ACT ATA GGG CGA AT-3&#226;&#172;&#732; <br /> CDC42U                      5&#226;&#172;&#732;-CCT AGC CCG TGC ACA GAC CCT CAA-3&#226;&#172;&#732; <br /> CDC42M                      5&#226;&#172;&#732;-TCC GAG TGT GTA GGG CTC GTC TCC-3&#226;&#172;&#732; <br /> PR1                         5&#226;&#172;&#732;-TTA GGG CCC AAT CTA AGA TAG ACA CAC GCT CAC CAC CCA-3&#226;&#172;&#732;                                                                  ApaI <br /> PR2                         5&#226;&#172;&#732;-CTG GTC GAC CAT TTT GGA ACC GGT AGC GAG AGT GGA TGT AGG-3&#226;&#172;&#732;                                                              SalI <br /> NT1                         5&#226;&#172;&#732;-ATG GTC GAC CAG AGT ATA AAA TGT GTC GTC ACT GGC GAC GGG-3&#226;&#172;&#732;                                                              SalI <br /> NT2                         5&#226;&#172;&#732;-GGC CCG CGG TAT CCC AAA GTC CGA GGT TTA TCG GTT TGT-3&#226;&#172;&#732;                                                                  SacII <br /> SE1                         5&#226;&#172;&#732;-ACT AGT GAG TCG ATG GGC AAC AAA CCA CAG-3&#226;&#172;&#732;                                                                              SpeI <br /> SE2                         5&#226;&#172;&#732;-AGA ATT CAG AGA GCT TAG TGC ACG GCT GGC TTG-3&#226;&#172;&#732;                                                                          EcoRI <br /> EH1                         5&#226;&#172;&#732;-TGA ATT CTT GTT GTG CTG AGT TTG TCT TTT TTT CAT CAA-3&#226;&#172;&#732;                                                                  EcoRI <br /> EH2                         5&#226;&#172;&#732;-AAA GCT TGT GGT TTG GGT GGT GAG CGT GTG TCT ATC-3&#226;&#172;&#732;                                                                      HindIII <br /> HB1                         5&#226;&#172;&#732;-CAA GCT TTC TTT TGC ACA CCA CCC CAC GAC CGA AAC-3&#226;&#172;&#732;                                                                      HindIII <br /> HB2                         5&#226;&#172;&#732;-CAG ATC TTG TAG TGA GTG ACG CAA AAA CTG AGA CCG-3&#226;&#172;&#732;                                                                      BglII <br /> BS1                         5&#226;&#172;&#732;-AAG ATC TGC ACA AGT CTC AAT CAA GAC ACT CGC AAG-3&#226;&#172;&#732;                                                                      BglII <br /> BS2                         5&#226;&#172;&#732;-GGT CGA CCA TTT TGG AAC CGG TAG CGA GAG TGG ATG TA-3&#226;&#172;&#732;                                                                   SalI <br />  <br />  <br />  <br />                                                                                    combined availability specific molecular genetic <br />                                                                                    tools, provided Y. lipolytica number advantages <br />                                                                                    S. cerevisiae Candida albicans investigation the <br />                                                                                    molecular mechanisms underlying dimorphic transition (26). <br />                                                                                    The dimorphic switch induced environmental signals and <br />                                                                                    thought important virulence number patho- <br />                                                                                    genic fungi (8, 38, 41, 52, 62). <br />                                                                                       In paper, report isolation initial character- <br />                                                                                    ization YlRAC1, gene encoding reported fungal <br />                                                                                    Rac homolog, involved yeast-to-hypha transi- <br />                                                                                    tion Y. lipolytica. <br />  <br />                                                                                                            MATERIALS AND METHODS <br />                                                                                       Yeast strains microbial techniques. The Y. lipolytica strains used this <br />                                                                                    study listed Table 1. The mutant strain CHY1220 isolated after <br />                                                                                    chemical mutagenesis Y. lipolytica E122 cells 1-methyl-3-nitro-1-ni- <br />                                                                                    trosoguanidine, previously described (51). Media components follows: <br />                                                                                    YEPD, 1% yeast extract, 2% peptone, 2% glucose; YNA, 0.67% yeast <br />                                                                                    nitrogen <span id='am-82' about='xsp:base' typeof='owl:Thing'>base</span> amino acids 2% sodium acetate; YNBD, 0.67% yeast <br />                                                                                    nitrogen base amino acids 2% glucose; YNBGlc, 1.34% yeast ni- <br />                                                                                    trogen base amino acids 1% glucose; YNBGlcNAc, 1.34% yeast <br />                                                                                    nitrogen base amino acids, 1% N-acetylglucosamine, 50 mM citric <br />                                                                                    acid (pH 6.0). YNA YNBD supplemented uracil, leucine, lysine, <br />                                                                                    histidine, 50 &#226;?&#174;g/ml, required. YNBGlc YNBGlcNAc were <br />                                                                                    supplemented 2&#226;&#171;&#187; Complete Supplement Mixture (Bio101, Vista, Calif.) or <br />                                                                                    2&#226;&#171;&#187; Complete Supplement Mixture minus leucine, required. Media, growth <br />                                                                                    conditions, procedures mating, sporulation, transformation Y. <br />                                                                                    lipolytica described previously (9, 51). <br />                                                                                       Standard techniques DNA manipulation growth Escherichia coli <br />                                                                                    used described previously (6). <br />                                                                                       Mycelial induction. Mycelial growth induced described previously (20). <br />                                                                                    Cells grown 12 h YNBGlc, harvested centrifugation room <br />                                                                                    temperature, washed sterile distilled water, kept ice 15 min YNB <br />                                                                                    carbon source, inoculated final density 107 cells/ml in <br />                                                                                    prewarmed YNBGlcNAc (for induction yeast-to-hypha transition) or <br />                                                                                    YNBGlc (for growth yeast form). <br />                                                                                       Cloning characterization Y. lipolytica RAC1 gene. The Y. lipolytica <br />                                                                                    RAC1 (YlRAC1) gene isolated functional complementation strain <br />                                                                                    CHY1220 using Y. lipolytica genomic DNA library contained replicative <br />    FIG. 1. Colony morphology various Y. lipolytica strains. (A) Wild-type       E. coli shuttle vector pINA445 (51). Plasmid DNA introduced yeast cells <br /> strain E122; (B) original mutant strain CHY1220; (C) RAC1 disruptant strain        electroporation, Leu&#226;&#171;&#185; transformants screened YNA agar plates <br /> CHY1220-A30; (D) MHY1 disruptant strain mhy1KO9; (E) strain CHY1220                ability rise filamentous colonies. <br /> transformed plasmid pRAC1; (F) strain CHY1220-A30 transformed               Complementing plasmids recovered transformation E. coli, the <br /> plasmid pRAC1; (G) strain mhy1KO9 transformed plasmid pRAC1; (H)              smallest fragment capable restoring hyphal growth determined. Restric- <br /> strain CHY1220 transformed plasmid pMHY1; (I) strain CHY1220-A30              tion fragments prepared genomic insert constructs <br /> transformed plasmid pMHY1; (J) wild-type strain 22301-3; (K) RAC1 dis-        (pRAC1) subcloned vector pGEM-5Zf(&#226;&#171;&#185;) pGEM-7Zf(&#226;&#171;&#185;) (Pro- <br /> ruptant strain CHY1220-B36; (L) RAC1/RAC1 diploid strain E122//22301-3; (M         mega, Madison, Wis.) pBluescript II SK(&#226;&#171;&#185;) (Stratagene, La Jolla, Calif.) for <br /> and N) RAC1/rac1 diploid strains 22301-3//CHY1220-A30 E122//CHY1220-           dideoxynucleotide sequencing strands. The deduced polypeptide se- <br /> B36, respectively; (O) rac1/rac1 diploid strain CHY1220-A30//CHY1220-B36.          quence, Y. lipolytica Rac1p (YlRac1p), compared known protein <br /> Colonies photographed 3 days incubation 28&#194;&#176;C YNA agar          sequences using BLAST Network Service National Center Bio- <br /> plates. Magnification, &#226;&#171;&#187;100.                                                       technology Information (Bethesda, Md.). <br />  2378       HURTADO ET AL.                                                                                                                         J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. Nucleotide sequence YlRAC1 gene deduced amino acid sequence YlRac1p. The transcriptional start site YlRAC1 gene indicated by <br /> an arrow. Putative STREs indicated. Consensus sequences intron splicing underlined. Putative transcription termination signals doubly underlined. <br />  <br />  <br />  <br />  <br />   Nucleic acid manipulation. Genomic DNA, plasmid DNA, total RNA                Fig. 2) PCR using oligonucleotide pairs SE1 SE2, EH1 EH2, HB1 <br /> were prepared Y. lipolytica, described (6). Southern          HB2, BS1 BS2 (Table 2). The PCR products (369-bp SpeI- <br /> Northern blot analyses performed DNA probes prepared             EcoRI, 100-bp EcoRI-HindIII, 197-bp HindIII-BglII, 122-bp BglII-SalI frag- <br /> ECL direct nucleic acid labeling detection (Amersham Life Sciences,      ments) ligated 396-bp SalI-SacII fragment obtained PCR <br /> Oakville, Ontario, Canada). Electrophoresis transfer nitrocellulose mem-     using oligonucleotides NT1 NT2 (Table 2). The resulting SpeI-SacII <br /> branes carried described previously (6). Hybridization, stringency   fragment (&#226;&#172;&#402;1.2 kbp) used replace counterpart plasmid pRAC1. <br /> washes, signal generation detection recommended man-         The integrity final construct confirmed sequencing. <br /> ufacturer, analysis Y. lipolytica genomic DNA, temper-      Nucleotide sequence accession number. The sequence data reported for <br /> ature hybridization washes reduced 42 39&#194;&#176;C.                      YlRAC1 Y. lipolytica CDC42 (YlCDC42) available EMBL/Gen- <br />    Immunofluorescence microscopy. F-actin detected incubating cells     Bank/DDBJ accession numbers AF176831 AF209750, respectively. <br /> 1.3 &#226;?&#174;M Oregon Green 488 phalloidin (Molecular Probes, Eugene, Oreg.), as <br /> previously described (2). Images scanned using SPOT software 1.2.1 (Di- <br /> agnostic Instruments, Sterling Heights, Mich.), processed Photoshop 4.0.1                                        RESULTS <br /> (Adobe Systems, San Jose, Calif.), printed DS8650 PS color printer <br /> (Eastman-Kodak, Rochester, N.Y.).                                                      Isolation Y. lipolytica mutant strain CHY1220. The Y. <br />    Mutagenesis CCCCT elements YlRAC1 promoter region. Three-base <br /> substitutions introduced putative stress response elements (STREs) <br />                                                                                     lipolytica mutant strain CHY1220 (Fig. 1B) initially iso- <br /> found promoter region YlRAC1 gene (CCCCT ATTCT                  lated inability form wild-type rough-surfaced colonies <br /> STRE1, CCCCT AGCTT STRE2, CCCCT GATCT STRE3) (see                   YEPD agar plates 3 days incubation 28&#194;&#176;C (Fig. <br />  VOL. 182, 2000                                                                                 DIMORPHISM IN Y. LIPOLYTICA                  2379 <br />  <br />  <br /> 1A J). Further analysis revealed that, like &#226;&#338;&#172;rac1 strains, <br /> strain CHY1220 able form colonies having small <br /> number peripheral extensions prolonged periods of <br /> incubation rich minimal media these <br /> extensions consisted chains elongated cells following a <br /> pseudohyphal <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Fig. 1B, C, K, O; Fig. 7). <br />    Isolation characterization YlRAC1 gene. The <br /> YlRAC1 gene isolated Y. lipolytica genomic DNA <br /> library contained replicative E. coli shuttle vector <br /> pINA445 (51) ability restore hyphal growth CHY1220 <br /> cells. Of approximately 70,000 transformants screened, 5 showed <br /> a restored filamentous <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. 1E). Restriction en- <br /> zyme analysis demonstrated complementing plasmids <br /> shared 2.2-kbp SpeI-ClaI fragment capable inducing hy- <br /> phal growth CHY1220 cells. Sequencing fragment <br /> revealed open reading frame (ORF) 576 bp interrupted <br /> by introns, codons 12 36 (nucleo- <br /> tides &#226;&#171;&#185;36 &#226;&#171;&#185;205 &#226;&#171;&#185;278 &#226;&#171;&#185;328 relative A residue <br /> of potential initiating codon, respectively) (Fig. 2). The <br /> putative 5&#226;&#172;&#732;-splice donor sequences introns (GTA <br /> AGTPu) diverge fourth positions the <br /> GTGAGTPu GTATGT consensus motifs Y. <br /> lipolytica (39, 65) S. cerevisiae (69), respectively. As the <br /> Y. lipolytica genes SEC14 (39) PYK1 (65), 3&#226;&#172;&#732;-splice ac- <br /> ceptor CAG sequence nucleotide downstream of <br /> the consensus TACTAAC box (69) abbreviated form <br /> CTAAC (Fig. 2). <br />    No obvious TATA box CT/CA-rich region, be- <br /> lieved play role transcriptional regulation Y. lipolytica <br /> (50, 71), seen promoter region YlRAC1. However, <br /> analysis cDNA showed transcription YlRAC1 <br /> gene starts preferentially position &#226;&#171;&#186;286 relative A <br /> nucleotide ATG codon polyadenylation <br /> occurs following G nucleotide position &#226;&#171;&#185;1075. Other <br /> features YlRAC1 include presence conserved A nu- <br /> cleotides positions &#226;&#171;&#186;1 &#226;&#171;&#186;3 relative A nucleotide <br /> of initiating ATG putative STRE (pentanucle- <br /> otide CCCCT) (32) upstream region (Fig. 2). <br />    The deduced protein product YlRAC1, YlRac1p, 192 <br /> amino acids <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> predicted molecular mass              FIG. 3. Amino acid sequence alignment Rac1p Y. lipolytica (YlRac1p) <br />                                                                        Rac proteins Homo sapiens (HsRac1, HsRac2, HsRac3), Mus <br /> 21,173 Da (Fig. 2). Comparison predicted amino acid             musculus (MmRac1 MmRac2), Drosophila melanogaster (DmRac1 and <br /> sequence YlRac1p sequences Rac proteins            DmRac2), Caenorhabditis elegans (CeRac1 CeRac2), Canis familiaris <br /> different sources suggests closest homolog human           (CfRac1), Xenopus laevis (XlRac). GenBank accession numbers M29870 <br /> Rac1 (Fig. 3). In addition, YlRac1p relatively high pI           (HsRac1), CAB45265 (HsRac2), AAC51667 (HsRac3), CAA40545 (MmRac1), <br />                                                                        Q05144 (MmRac2), AAA62870 (DmRac1), P48554 (DmRac2), AAA28141 <br /> (8.47), attribute distinguishes Rac proteins          (CeRac1), AAB40386 (CeRac2), P15154 (CfRac1), AAD50299 (XlRac). <br /> from Rho Ras proteins (whose pIs range 5.0 <br /> to 6.5) (17). <br />    Consensus elements GXXXXGK (GDGAVGK, residues 10 <br /> to 16) DXXG (DTAG, residues 57 60) (Fig. 3),              C. albicans (46, 47), decided provide evi- <br /> are involved interactions phosphate portion         dence identified fungal RAC gene isolating the <br /> GTP molecule, YlRac1p positions conserved              CDC42 gene Y. lipolytica (YlCDC42). <br /> among GTP-binding proteins (15, 18, 28, 46). Conserved mo-                We combined sequence partial YlCDC42 clone <br /> tifs present regions implicated interaction        previously obtained probing Y. lipolytica genomic DNA <br /> the GTPase-activating protein (TVFDNY, residues 35 40)              library oligonucleotide derived highly conserved <br /> (61) membrane association prior biological activity          sequence Rab family proteins (53) sequence <br /> (CVIL, residues 189 192) (23, 73) (Fig. 3). Notably, YlRac1p        YlCDC42 cDNA obtained PCR Y. lipolytica cDNA <br /> also contains conserved motif TKXD (TKLD, residues 115             library constructed ZAP Express vector (Stratagene) <br /> to 118), responsible nucleotide specificity Rac        oligonucleotides T3, T7, CDC42U, CDC42M (Table <br /> proteins involved determination unusually         2) obtain sequence YlCDC42 gene. The YlCDC42 <br /> high intrinsic rate GTP hydrolysis distinguishes Rac           gene contains ORF 573 bp, interrupted two <br /> proteins Rho family members (17).                           introns, YlRAC1 gene (Fig. 4). The introns found <br />    Isolation characterization YlCDC42 gene. Since           codons 16 17 codon 45 (nucleotides &#226;&#171;&#185;49 to <br /> no RAC gene previously reported fungi           &#226;&#171;&#185;157 &#226;&#171;&#185;244 &#226;&#171;&#185;327 relative A nucleotide the <br /> the CDC42 gene, encodes protein belongs            potential initiating codon, respectively). The putative 5&#226;&#172;&#732;-splice <br /> Rac subfamily Rho GTPases (19), shown                donor sequences introns identical consensus <br /> involved regulation filamentous growth S. cerevisiae      motif GTGAGTPu Y. lipolytica genes (39, 65), <br />  2380     HURTADO ET AL.                                                                                                                  J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Nucleotide sequence YlCDC42 gene deduced amino acid sequence YlCdc42p. Consensus sequences intron splicing underlined. <br />  <br />  <br />  <br />  <br /> and 3&#226;&#172;&#732;-splice acceptor CAG sequence                     dimorphism and, like dimorphism, phenomenon that <br /> nucleotides downstream consensus TACTAAC box (69)                      involves dramatic changes cell morphology response to <br /> (Fig. 4).                                                                     environmental conditions (40), investigated dis- <br />    The deduced protein YlCDC42, YlCdc42p, 191 amino                     ruption YlRAC1 gene effect mating <br /> acids long predicted molecular mass 21,336 Da                    ability Y. lipolytica. Crossing A mating type strain <br /> (Fig. 4) estimated pI 6.09, characteristic              CHY1220-A30 (&#226;&#338;&#172;rac1) B mating type wild-type strain <br /> Cdc42 proteins (17). In addition, YlCdc42p contains motifs                22301-3 readily attained (Fig. 1M), indicating mat- <br /> required biological function small GTPases                  ing defect associated disruption YlRAC1 these <br /> Rho family (GDGAVGK, residues 10 16; TVFDNY, resi-                         strains. To determine lack effect mating by <br /> dues 35 40; DTAG, residues 57 60; CIVL, residues                    disruption YlRAC1 gene confined A mating type <br /> 188 191) (Fig. 4 5). Comparison YlCdc42p Cdc42                 cells, B mating type strain, CHY1220-B36 (Fig. 1K Table <br /> proteins number different organisms suggests               1), YlRAC1 gene deleted, obtained sporulation <br /> closest relative Schizosaccharomyces pombe Cdc42 (Fig. 5).                 diploid strain 22301-3//CHY1220-A30 (Fig. 1M and <br /> Importantly, YlCdc42p contains signature sequence                      Table 1) selection haploids inability form <br /> Cdc42 proteins (27), motif TQXD (TQVD, residues 115                    hyphal cells. The &#226;&#338;&#172;rac1::URA3 genotype CHY1220-B36 <br /> 118) region responsible nucleotide specificity.                    strain confirmed cosegregation Ura&#226;&#171;&#185; Fil&#226;&#171;&#186; <br />    Strains YlRAC1 gene disrupted viable un-                  phenotypes random spore analysis (data shown), and <br /> affected mating ability. A 1.0-kbp ApaI-NdeI fragment                   strain able mate wild-type E122 <br /> YlRAC1 replaced 1.6-kbp ApaI-NdeI fragment con-                      CHY1220-A30 strains (Fig. 1N O), demonstrating <br /> taining Y. lipolytica URA3 gene (Fig. 6). This construct              YlRAC1 essential mating. One copy the <br /> digested DraI XbaI liberate 2.4-kbp fragment                    YlRAC1 gene sufficient support dimorphic transition in <br /> containing entire URA3 gene flanked 0.5 0.3 kbp                 diploid strains Y. lipolytica, slight reduction the <br /> the 5&#226;&#172;&#732; 3&#226;&#172;&#732; regions YlRAC1 gene, respectively. This                  proportion hyphal cells observed strains <br /> linear fragment used transform wild-type Y. lipoly-                (Fig. 1M N). <br /> tica strain E122 uracil prototrophy.                                          Disruption YlRAC1 gene affects cell morphology but <br />    Of 303 Ura&#226;&#171;&#185; transformants obtained, 3 showed smooth                      does impair actin polarization cell invasiveness. Since <br /> phenotype 3 days YEPD agar plates. Two                      organization actin cytoskeleton directly involved in <br /> transformants confirmed Southern blot analysis                    determination cell shape Rac proteins play <br /> PCR YlRAC1 gene correctly replaced                     fundamental role process (21, 22, 57, 67), disruption of <br /> URA3 gene (Fig. 6), them, CHY1220-A30 (Fig. 1C                     YlRAC1 anticipated result morphological defects in <br /> and Table 1), selected analysis.                              Y. lipolytica cells. Indeed, exponentially growing &#226;&#338;&#172;rac1 mutant <br />    Because mating intimately connected                   cells round shape, clearly contrasting with <br />  VOL. 182, 2000                                                                                             DIMORPHISM IN Y. LIPOLYTICA                    2381 <br />  <br />  <br />  <br />  <br />    FIG. 5. Amino acid sequence alignment Cdc42p Y. lipolytica (YlCdc42p) <br /> and homologs S. cerevisiae (ScCdc42p) (28), C. albicans (CaCdc42p) (46), <br /> S. pombe (SpCdc42p) (44), Caenorhabditis elegans (CeCdc42p) (15), Mus mus- <br /> culus (MmCdc42p) (43), Homo sapiens (HsCdc42p) (48). <br />  <br />  <br />  <br /> the typically ovoid cells observed wild-type strains (Fig. 7, <br />                                                                                      FIG. 6. Integrative disruption YlRAC1 gene. (A) Diagram illustrating <br /> top panels). Continued incubation rich medium yielded                          replacement 1.0-kbp ApaI-NdeI fragment YlRAC1 1.6-kbp <br /> wild-type cultures composed yeast cells, pseudohyphae,                     ApaI-NdeI fragment containing Y. lipolytica URA3 gene. (B) Southern blot <br /> a germ tubes, &#226;&#338;&#172;rac1 cultures contained small                     analysis SpeI-HpaI-digested genomic DNA, PCR analysis total genomic <br /> proportion pseudohyphal cells germ tubes (Fig. 7,                       DNA, wild-type strain E122 strain CHY1220-A30, confirming the <br />                                                                                   correct replacement YlRAC1 gene URA3-containing linear mol- <br /> middle panels). In general, pseudohyphal &#226;&#338;&#172;rac1 cells                         ecule strain CHY1220-A30. Primers KO1 KO2 (Table 2) indicated by <br /> found shorter wild-type counterparts (Fig. 7,                    black arrows panel A. <br /> middle panels). As stationary phase reached, hyphal <br /> growth predominant wild-type cultures, a <br /> limited number chains composed pseudohyphal cells were <br /> seen &#226;&#338;&#172;rac1 cultures (Fig. 7, panels). Germ tubes                    starvation period 4&#194;&#176;C, followed transfer prewarmed <br /> arising pseudohyphal cells seen                        (28&#194;&#176;C) YNBGlcNAc. Under conditions, 80% <br /> wild-type strain (Fig. 7, right panel, inset). Interest-                   cell population gave rise germ tubes 10 h of <br /> ingly, invasive pseudohyphal growth substan-                      incubation, cells transferred fresh YNBGlc grew al- <br /> tially induced &#226;&#338;&#172;rac1 strain incubation minimal                       exclusively budding form, described previously <br /> medium containing glucose sole carbon source (YNBD                         (20, 26). Northern blot analysis performed total RNA <br /> agar), effect observed &#226;&#338;&#172;rac1 cells grown                  extracted cells harvested 0, 1, 3, 10 h incubation <br /> on minimal medium containing acetate sole carbon                           YNBGlcNAc showed YlRAC1 mRNA levels increased <br /> source (YNA agar) mhy1KO9 (&#226;&#338;&#172;mhy1) cells incubated                           steadily yeast-to-hypha transition, re- <br /> under condition (Fig. 8). No difference invasiveness                    mained virtually constant incubation YNBGlc (Fig. <br /> was observed original mutant strain CHY1220                           10). <br /> and &#226;&#338;&#172;rac1 strain CHY1220-A30 subjected                            Putative STREs YlRAC1 gene necessary the <br /> to growth conditions (data shown).                             induction hyphal growth. Genetic interaction YlRAC1 <br />    As described number fungi (1, 3, 4, 25, 34, 42, 60,                   MHY1 initially suggested observation a <br /> 72), actin-rich zones sites growth (apices germ                      pINA445-based multicopy plasmid bearing MHY1 gene <br /> tubes, hyphae, pseudohyphae, yeast cells), combined                      (pMHY1) (26) able restore hyphal growth intro- <br /> a background diffuse actin staining punctate actin                        duction CHY1220, CHY1220-A30 (&#226;&#338;&#172;rac1), cells <br /> patches, observed wild-type cells Y. lipolytica (Fig.                 (Fig. 1H I). The inability pRAC1 induce dimorphic <br /> 9A G). Interestingly, despite alterations cell morphology,                  transition &#226;&#338;&#172;mhy1 cells (Fig. 1D G); presence of <br /> &#226;&#338;&#172;rac1 mutant cells appeared retain ability concentrate                  copies pentanucleotide STRE sequence CCCCT <br /> actin granules apices pseudohyphal cells emerg-                     promoter region YlRAC1 (Fig. 2), has <br /> ing buds (Fig. 9H I).                                                         shown specifically bind vitro-synthesized Mhy1p <br />    Transcription YlRAC1 gene increased                       (26); finding &#226;&#338;&#172;rac1 mutant cells form <br /> yeast-to-hypha transition. The dimorphic switch induced                       pseudohyphae, &#226;&#338;&#172;mhy1 mutants unable grow as <br /> in exponentially growing E122 cells 15-min carbon source                     hyphal pseudohyphal form (Fig. 7 8) sug- <br />  2382      HURTADO ET AL.                                                                                                                         J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. Disruption YlRAC1 affects cell morphology impairs hyphal growth, pseudohyphal growth, Y. lipolytica. Strains grown YEPD. Top <br /> panels, exponential growth phase (optical density 600 nm [OD600] &#226;&#171;&#189; 1). Middle panels, late exponential growth phase (OD600 &#226;&#171;&#189; 4). Bottom panels inset, <br /> stationary phase (OD600 &#226;&#171;&#189; 10). WT, wild-type strain E122. &#226;&#338;&#172;rac1, strain CHY1220-A30. Bars, 5 &#226;?&#174;m. <br />  VOL. 182, 2000                                                                                             DIMORPHISM IN Y. LIPOLYTICA               2383 <br />  <br />  <br />                                                                                      characteristics filamentous growth fungi. Pollen <br />                                                                                      tube elongation based process known tip growth, <br />                                                                                      polarized secretion restricted apex, cell <br />                                                                                      membrane cell wall material delivered exclusively to <br />                                                                                      location (33, 37, 64, 68). Significantly, plant Rac homologs <br />                                                                                      shown localized plasma membrane of <br />                                                                                      pollen tube region tip (37). The localization of <br />                                                                                      YlRac1p Y. lipolytica far remained elusive. Although <br />                                                                                      epitope-tagged YlRac1p detected growing <br />                                                                                      tip region filamentous Y. lipolytica cells (data shown), <br />                                                                                      fusion protein unable induce hyphal growth in <br />                                                                                      &#226;&#338;&#172;rac1 cells, making impossible ascertain dis- <br />                                                                                      tribution tagged protein truly representative that <br />                                                                                      natural protein. The availability antibodies specific for <br />                                                                                      YlRac1p extremely useful addressing question. <br />                                                                                         Although Rac homolog previously described <br />                                                                                      fungi, newly identified S. cerevisiae ORF, Ynl180c, codes <br />                                                                                      protein number features typical Rac proteins, <br />                                                                                      including presence conserved motifs required for <br />                                                                                      biological function estimated pI 8.76. However, this <br />                                                                                      putative protein, 52.1% identical YlRac1p, at <br />                                                                                      331 amino acids considerably larger known Rac pro- <br />                                                                                      teins, known time functional. <br />                                                                                         Disruption YlRAC1 gene lethal does not <br />                                                                                      abolish ability cells polarize actin site growth. <br />                                                                                      These findings explained presence other <br />                                                                                      genes Y. lipolytica closely related YlRAC1, as <br />    FIG. 8. Invasive filamentous growth different Y. lipolytica strains. Follow-   suggested complex banding pattern revealed South- <br /> ing 5 days incubation 28&#194;&#176;C minimal agar medium containing glucose        ern analysis Y. lipolytica genomic DNA conditions of <br /> acetate sole carbon source, plates washed running water          low stringency. Nevertheless, alterations cell morphol- <br /> remove cells agar surface. Pictures taken washes.     ogy &#226;&#338;&#172;rac1 mutants inability strains grow <br /> WT, wild-type strain E122. &#226;&#338;&#172;rac1, strain CHY1220-A30. &#226;&#338;&#172;mhy1, strain mhy1KO9. <br />                                                                                      hyphal form strongly suggest YlRAC1 functions in <br />                                                                                      aspect polarization cell growth. <br />                                                                                         We previously reported isolation characteriza- <br /> gested Mhy1p act promote hyphal growth                                 tion gene MHY1, codes putative transcrip- <br /> through regulatory elements YlRAC1. In order                            tion factor binds vitro oligonucleotide containing <br /> investigate role putative STRE sequences                         STRE1 YlRAC1 (Fig. 2) essential filamentous <br /> induction hyphal growth, mutagenesis elements                         growth Y. lipolytica (26). Here report deletion <br /> pRAC1 performed. No defect observed ability                           MHY1 completely abolishes ability Y. lipolytica to <br /> to induce dimorphic transition introduction                          grow hyphal pseudohyphal forms solid <br /> plasmid pRAC1-Mut (which contains mutations                             minimal medium containing glucose acetate the <br /> STREs) &#226;&#338;&#172;rac1 strain (CHY1220-A30), suggesting                            sole carbon source, strains lacking functional YlRAC1 gene <br /> these elements necessary induction hyphal                         able form pseudohyphae invade agar glu- <br /> growth YlRAC1 (data shown).                                                  cose-based minimal medium, suggesting, suggested <br />   Genomic DNA analysis RAC genes Y. lipolytica. As                        C. albicans (38), morphologies Y. lipolytica <br /> organisms multiple Rac Rho homologs, looked                              controlled, part, parallel signaling path- <br /> for evidence RAC genes Y. lipolytica. Genomic                            ways, different additive input, they <br /> DNA E122 strain digested various combi-                            represent sequence events single pathway filamen- <br /> nations restriction endonucleases analyzed Southern                        tous growth requiring quantitatively stronger regulatory in- <br /> blotting low-stringency conditions labeled 240-bp                       produce hyphae pseudohyphae. Likewise, <br /> SacII-NdeI fragment YlRAC1 gene. A complex pattern                            analysis Ras homolog Aspergillus nidulans sug- <br /> of bands observed, suggestive presence                         gested scenario thresholds Ras concen- <br /> RAC-related superfamily members Y. lipolytica (Fig. 11).                          tration exist, allows development proceed to <br />                                                                                      certain point, producing proper cell type inhibiting <br />                                DISCUSSION                                            development (63). Our findings overexpression of <br />                                                                                      MHY1 restore hyphal growth CHY1220 in <br />    Rac proteins implicated control diverse                     CHY1220-A30 (&#226;&#338;&#172;rac1) cells, disruption YlRAC1 affects <br /> and extensive set cellular processes (74), including notably                      hyphal growth disruption MHY1 blocks both <br /> the regulation assembly organization actin                         hyphal pseudohyphal growth, pseudohyphal cells <br /> cytoskeleton (21, 22, 49, 56, 57, 66, 67, 70). As consequence,                     rise hyphae (Fig. 7, right panel, inset) <br /> Rac-mediated changes actin organization gene expres-                        support scenario suggest MHY1 acts upstream <br /> sion believed play pivotal role control cell                      YlRAC1 filamentous pathway(s). <br /> shape, cell attachment, cell motility invasion, cell-cell in-                       It important point regardless fact that <br /> teraction, cell proliferation differentiation (74). In                       mutagenesis putative STREs promoter of <br /> plants, Rac homologs thought involved regu-                         YlRAC1 did affect ability induce hyphal growth Y. <br /> lation growth pollen tube, process shares                           lipolytica (data shown), role elements the <br />  2384      HURTADO ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 9. Actin localization different stages development wild-type &#226;&#338;&#172;rac1 cells. Actin detected staining cells Oregon Green 488 <br /> phalloidin followed fluorescence microscopy. (A G) Wild-type strain E122. (H I) &#226;&#338;&#172;rac1 strain CHY1220-A30. (A H) Yeast-like cells; (B, G, I) <br /> pseudohyphal growth; (C) early germ tube formation; (D) late germ tube formation; (E F) hyphal growth. Bars, 5 &#226;?&#174;m. <br />  <br />  <br />  <br /> induction dimorphism ruled out. The activity                      phal growth Y. lipolytica. Greater knowledge func- <br /> other unidentified regulatory elements YlRAC1 gene                      tion(s) YlRAC1 interactions Y. lipolytica <br /> compensation loss transcriptional induction                         genes important better understanding the <br /> YlRAC1 activation related GTPases ex-                         mechanisms environmental conditions induce <br /> plain negative result. We currently investigating these <br /> possibilities. <br />    Here report isolation initial characterization <br /> of YlCDC42 gene. In C. albicans, transient increase the <br /> CDC42 mRNA levels observed switch hyphal <br /> growth (46), remains determined whether <br /> Cdc42p plays role pseudohyphal formation this <br /> organism. Although variation abundance Cdc42p <br /> has observed cell cycle S. cerevisiae (73), <br /> CDC42 shown potent regulator filamentous <br /> growth yeast, acting downstream RAS2 activating <br /> pseudohyphal growth diploid cells invasive growth of <br /> haploid cells response nitrogen starvation STE20 (35,                      FIG. 10. YlRAC1 mRNA levels increased dimorphic transi- <br /> 47, 54). We currently investigating YlCDC42                      tion. Total RNA isolated E122 cells incubated 28&#194;&#176;C YNBGlcNAc <br /> plays role induction filamentous growth Y. lipo-                   (induction filamentous growth) YNBGlc (control culture, yeast-like cells) <br /> lytica.                                                                           times indicated subjected Northern blot analysis. Ten micrograms <br />                                                                                   RNA time point separated formaldehyde agarose gel and <br />    In conclusion, report isolation initial character-                  transferred nitrocellulose. Blots hybridized probe specific the <br /> ization fungal Rac homolog provide evidence                      YlRAC1 gene (1.0-kbp ApaI-NdeI fragment [see Fig. 6]). Equal loading RNA <br /> that YlRac1p plays important role regulation hy-                     ensured ethidium bromide staining (data shown). <br />  VOL. 182, 2000                                                                                                   DIMORPHISM IN Y. LIPOLYTICA                        2385 <br />  <br />  <br />                                                                                       10. Bazenet, C. E., M. A. Mota, L. L. Rubin. 1998. The small GTP-binding <br />                                                                                           protein Cdc42 required nerve growth factor withdrawal-induced neu- <br />                                                                                           ronal death. Proc. Natl. Acad. Sci. USA 95:3984&#226;&#8364;&#8220;3989. <br />                                                                                       11. Block, C., A. Wittinghofer. 1995. Switching Rac Rho. Structure <br />                                                                                           3:1281&#226;&#8364;&#8220;1284. <br />                                                                                       12. Boguski, M. S., F. McCormick. 1993. Proteins regulating Ras its <br />                                                                                           relatives. Nature 366:643&#226;&#8364;&#8220;654. <br />                                                                                       13. Bourne, H. R., D. A. Sanders, F. McCormick. 1991. The GTPase super- <br />                                                                                           family: conserved structure molecular mechanism. Nature 349:117&#226;&#8364;&#8220;127. <br />                                                                                       14. Bussey, H. 1996. Cell shape determination: pivotal role Rho. Science <br />                                                                                           272:224&#226;&#8364;&#8220;225. <br />                                                                                       15. Chen, W., H. H. Lim, L. Lim. 1993. The CDC42 homologue from <br />                                                                                           Caenorhabditis elegans. J. Biol. Chem. 268:13280&#226;&#8364;&#8220;13285. <br />                                                                                       16. Chuang, T.-H., K. M. Hahn, J.-D. Lee, D. E. Danley, G. M. Bokoch. <br />                                                                                           1997. The small GTPase Cdc42 initiates apoptotic signaling pathway in <br />                                                                                           Jurkat T lymphocytes. Mol. Biol. Cell 8:1687&#226;&#8364;&#8220;1698. <br />                                                                                       17. Delmer, D. P., J. P. Pear, A. Andrawis, D. M. Stalker. 1995. Genes <br />                                                                                           encoding small GTP-binding proteins analogous mammalian rac pref- <br />                                                                                           erentially expressed developing cotton fibers. Mol. Gen. Genet. 248:43&#226;&#8364;&#8220;51. <br />                                                                                       18. Dever, T. E., M. J. Glynias, W. C. Merrick. 1987. GTP-binding domain: <br />                                                                                           consensus sequence elements distinct spacing. Proc. Natl. Acad. <br />                                                                                           Sci. USA 84:1814&#226;&#8364;&#8220;1818. <br />                                                                                       19. Garcia-Ranea, J. A., A. Valencia. 1998. Distribution functional <br />                                                                                           diversification ras superfamily Saccharomyces cerevisiae. FEBS Lett. <br />                                                                                           434:219&#226;&#8364;&#8220;225. <br />                                                                                       20. Guevara-Olvera, L., C. Calvo-Mendez, J. Ruiz-Herrera. 1993. The role <br />                                                                                           polyamine metabolism dimorphism Yarrowia lipolytica. J. Gen. Mi- <br />                                                                                           crobiol. 193:485&#226;&#8364;&#8220;493. <br />                                                                                       21. Hall, A. 1994. Small GTP-binding proteins regulation actin <br />    FIG. 11. Southern blot analysis E122 genomic DNA. Ten micrograms                 cytoskeleton. Annu. Rev. Cell. Biol. 10:31&#226;&#8364;&#8220;54. <br /> DNA lane digested indicated restriction enzymes, separated        22. Hall, A. 1998. Rho GTPases actin cytoskeleton. Science 279:509&#226;&#8364;&#8220; <br /> electrophoresis, transferred nitrocellulose, probed 240-bp SacII-           514. <br /> NdeI-labeled fragment YlRAC1 (boxed), described Materials              23. Hancock, J. F., A. I. Magee, J. E. Childs, C. J. Marshall. 1989. All ras <br /> Methods.                                                                                  proteins polyisoprenylated palmitoylated. Cell 57: <br />                                                                                           1167&#226;&#8364;&#8220;1177. <br />                                                                                       24. Harii, Y., J. E. Beeler, K. Sakaguchi, M. Tachibana, T. Miki. 1994. A <br />                                                                                           novel oncogene, ost, encodes guanine nucleotide exchange factor that <br /> changes pattern cell growth, phenomenon                                  potentially links Rho Rac signalling pathways. EMBO J. 13:4776&#226;&#8364;&#8220;4786. <br /> strong implications development virulence fungal                        25. Heath, I. B. 1987. Preservation labile cortical array actin microfila- <br /> pathogens (8, 38, 41, 52, 62) elucidation                              ments growing hyphal tips fungus Saprolegnia ferax. Eur. J. Cell Biol. <br />                                                                                           44:10&#226;&#8364;&#8220;16. <br /> molecular mechanisms controlling differentiation higher eu-                        26. Hurtado, C. A. R., R. A. Rachubinski. 1999. MHY1 encodes C2H2-type <br /> karyotes. Furthermore, mechanisms Ras trans-                             zinc finger protein promotes dimorphic transition yeast Yarrowia <br /> formation relies signaling cascades controlled Rac                              lipolytica. J. Bacteriol. 181:3051&#226;&#8364;&#8220;3057. <br /> Rho GTPases, dimorphic yeast Y. lipolytica represent                          27. Johnson, D. I. 1999. Cdc42: essential Rho-type GTPase controlling eu- <br />                                                                                           karyotic cell polarity. Microbiol. Mol. Biol. Rev. 63:54&#226;&#8364;&#8220;105. <br /> a suitable model understanding complex                           28. Johnson, D. I., J. R. Pringle. 1990. Molecular characterization of <br /> events mammalian cells, important implications                            CDC42, Saccharomyces cerevisiae gene involved development cell <br /> search potential drug targets Ras-mediated malignan-                              polarity. J. Cell Biol. 111:143&#226;&#8364;&#8220;152. <br /> cies (29, 66, 74).                                                                    29. Joneson, T., D. Bar-Sagi. 1999. Suppression Ras-induced apoptosis by <br />                                                                                           Rac GTPase. Mol. Cell. Biol. 19:5892&#226;&#8364;&#8220;5901. <br />                                                                                       30. Kawasaki, T., K. Henmi, E. Ono, S. Hatakeyama, M. Iwano, H. Satoh, and <br />                           ACKNOWLEDGMENTS                                                 K. Shimamoto. 1999. The small GTP-binding protein Rac regulator of <br />    This work supported International Research Scholarship                       cell death plants. Proc. Natl. Acad. Sci. USA 96:10922&#226;&#8364;&#8220;10926. <br />                                                                                       31. Khosravi-Far, R., M. Chrzanowska-Wodnicka, P. A. Solski, E. Alessandra, <br /> from Howard Hughes Medical Institute R.A.R. R.A.R.                            K. Burridge, C. J. Der. 1994. Dbl Vav mediate transformation via <br /> Medical Research Council Canada Senior Scientist.                                      mitogen-activated protein kinase pathways distinct acti- <br />                                                                                           vated oncogenic Ras. Mol. Cell. Biol. 14:6848&#226;&#8364;&#8220;6857. <br />                                    REFERENCES                                         32. Kobayashi, N., K. McEntee. 1993. Identification cis trans com- <br />  1. Adams, A. E. M., J. R. Pringle. 1984. Relationship actin tubulin           ponents novel heat shock stress regulatory pathway Saccharomyces <br />     distribution bud growth wild-type morphogenetic-mutant Saccha-              cerevisiae. Mol. Cell. Biol. 13:248&#226;&#8364;&#8220;256. <br />     romyces cerevisiae. J. Cell Biol. 98:934&#226;&#8364;&#8220;945.                                     33. Kost, B., E. Lemichez, P. Spielhofer, Y. Hong, K. Tolias, C. Carpenter, and <br />  2. Adams, A. E. M., J. R. Pringle. 1991. Staining actin fluoro-              N.-H. Chua. 1999. Rac homologues compartmentalized phosphatidyl- <br />     chrome-conjugated phalloidin. Methods Enzymol. 194:729&#226;&#8364;&#8220;731.                           inositol 4,5-biphosphate act common pathway regulate polar pollen <br />  3. Akashi, T., T. Kanbe, K. Tanaka. 1994. The role cytoskeleton            tube growth. J. Cell Biol. 145:317&#226;&#8364;&#8220;330. <br />     polarized growth germ tube in Candida albicans. Microbiology        34. Kwon, Y. H., H. C. Hoch, R. C. Staples. 1991. Cytoskeletal organization <br />     140:271&#226;&#8364;&#8220;280.                                                                          Uromyces urediospore germling apices appressorium formation. <br />  4. Alfa, C. E., J. S. Hyams. 1990. Distribution tubulin actin         Protoplasma 165:37&#226;&#8364;&#8220;50. <br />     cell division cycle fission yeast Schizosaccharomyces japonicus var.   35. Leberer, E., C. Wu, T. Leeuw, A. Fourest-Lieuvin, J. E. Segall, D. Y. <br />     versitalis: comparison Schizosaccharomyces pombe. J. Cell Sci. 96:71&#226;&#8364;&#8220;77.         Thomas. 1997. Functional characterization Cdc42p binding domain of <br />  5. Arellano, M., P. M. Coll, P. Perez. 1999. RHO GTPases control              yeast Ste20p protein kinase. EMBO J. 16:83&#226;&#8364;&#8220;97. <br />     cell morphology, cell polarity, actin localization fission yeast.       36. Li, R., Y. Zheng, D. G. Drubin. 1995. Regulation cortical actin <br />     Microsc. Res. Tech. 47:51&#226;&#8364;&#8220;60.                                                         cytoskeleton assembly polarized cell growth budding yeast. J. Cell <br />  6. Ausubel, F. J., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A.           Biol. 128:599&#226;&#8364;&#8220;615. <br />     Smith, K. Struhl (ed.). 1989. Current protocols molecular biology.         37. Lin, Y., Y. Wang, J.-K. Zhu, Z. Yang. 1996. Localization Rho <br />     Greene Publishing Associates, New York, N.Y.                                          GTPase implies role tip growth movement generative cell in <br />  7. Baggiolini, B., M. P. Wymann. 1990. Turning respiratory burst.             pollen tubes. Plant Cell 8:293&#226;&#8364;&#8220;303. <br />     Trends Biochem. Sci. 15:69&#226;&#8364;&#8220;72.                                                    38. Lo, H.-S., J. R. Ko&#194;&#168;hler, B. DiDomenico, D. Loebenberg, A. Cacciapuoti, and <br />  8. Banuett, F. 1995. Genetics Ustilago maydis, fungal pathogen induces         G. R. Fink. 1997. Nonfilamentous C. albicans mutants avirulent. Cell <br />     tumors maize. Annu. Rev. Genet. 29:179&#226;&#8364;&#8220;208.                                        90:939&#226;&#8364;&#8220;949. <br />  9. Barth, G., H. Weber. 1986. Improvement sporulation yeast            39. Lopez, M. C., J.-M. Nicaud, H. B. Skinner, C. Vergnolle, J. C. Kader, V. A. <br />     Yarrowia lipolytica. Antonie Leewenhoek. J. Microbiol. Serol. 51:167&#226;&#8364;&#8220;177.             Bankaitis, C. Gaillardin. 1994. A phosphatidylinositol/phosphatidylcho- <br />  2386         HURTADO ET AL.                                                                                                                             J. BACTERIOL. <br />  <br />  <br />       line transfer protein required differentiation dimorphic yeast      56. Ridley, A. J. 1994. Membrane ruffling signal transduction. Bioessays <br />       Yarrowia lipolytica yeast mycelial form. J. Cell Biol. 124:113&#226;&#8364;&#8220;       16:321&#226;&#8364;&#8220;327. <br />       117.                                                                              57. Ridley, A. J. 1995. Rho-related proteins: actin cytoskeleton cell cycle. <br /> 40.   Madhani, H. D., G. R. Fink. 1998. The control filamentous differen-            Curr. Opin. Genet. Dev. 5:24&#226;&#8364;&#8220;30. <br />       tiation virulence fungi. Trends Cell Biol. 8:348&#226;&#8364;&#8220;353.                      58. Ridley, A. J., P. M. Comoglio, A. Hall. 1995. Regulation scatter <br /> 41.   Maresca, B., G. S. Kobayashi. 1989. Dimorphism Histoplasma cap-                factor/hepatocyte growth factor responses Ras, Rac, Rho MDCK <br />       sulatum: model study cell differentiation pathogenic fungi.           cells. Mol. Cell. Biol. 15:1110&#226;&#8364;&#8220;1122. <br />       Microbiol. Rev. 53:186&#226;&#8364;&#8220;209.                                                       59. Ridley, A. J., H. F. Paterson, C. Johnston, D. Diekmann, A. Hall. 1992. <br /> 42.   Marks, J., J. S. Hyams. 1985. Localization F-actin cell            The small GTP-binding protein rac regulates growth factor-induced mem- <br />       division cycle Schizosaccharomyces pombe. Eur. J. Cell Biol. 39:27&#226;&#8364;&#8220;32.             brane ruffling. Cell 70:401&#226;&#8364;&#8220;410. <br /> 43.   Marks, P. W., D. J. Kwiatkowski. 1996. Genomic organization               60. Roberson, R. W. 1992. The actin cytoskeleton hyphal cells Sclerotium <br />       chromosomal loc <br /> </body></html>