<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />         Role of GlnK in NifL-Mediated Regulation of NifA Activity in <br />                            Azotobacter vinelandii <br />  <br />              In several diazotrophic species of Proteobacteria, PII signal transduction proteins have been implicated in the <br />           regulation of nitrogen fixation in response <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> NH4&#226;&#180;&#8482; by several mechanisms. In Azotobacter vinelandii, expres- <br />           sion of nifA, encoding the nif-specific activator, is constitutive, and thus, regulation of NifA activity by the <br />           flavoprotein NifL appears to be the primary level of nitrogen control. In vitro and genetic evidence suggests <br />           that the nitrogen response involves the PII-like GlnK protein and GlnD (uridylyltransferase/uridylyl-removing <br />           enzyme), which reversibly uridylylates GlnK in response to nitrogen limitation. Here, the roles of GlnK and <br />           GlnK-UMP in A. vinelandii were studied to determine whether the Nif &#226;&#180;&#353; <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> of glnD strains was due to <br />           an inability to modify GlnK, an effort previously hampered because glnK is an essential gene in this organism. <br />           A glnKY51F <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span>, encoding an unuridylylatable form of the protein, was stable only in a strain in which <br />           glutamine synthetase activity is not inhibited by NH4&#226;&#180;&#8482;, suggesting that GlnK-UMP is required to signal <br />           adenylyltransferase/adenylyl-removing enzyme-mediated deadenylylation. glnKY51F strains were significantly <br />           impaired for diazotrophic growth and expression of a nifH-lacZ fusion. NifL interacted with GlnK and <br />           GlnKY51F in a yeast two-hybrid system. Together, these data are consistent with those obtained <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> in vitro <br />           experiments (Little et al., EMBO J., 19:6041&#226;&#8364;&#8220;6050, 2000) and support a model for regulation of NifA activity <br />           in which unmodified GlnK stimulates NifL inhibition and uridylylation of GlnK in response to nitrogen <br />           limitation prevents this function. This model is distinct from one proposed for the related bacterium Klebsiella <br />           pneumoniae, in which unmodified GlnK relieves NifL inhibition instead of stimulating it. <br />  <br />  <br />    Transcriptional control of nitrogenase synthesis is a common               eric signal transduction components generally referred to as <br /> mechanism for the regulation of nitrogen fixation in many                     PII proteins (2, 36). In E. coli, nitrogen limitation, sensed as an <br /> diazotrophic bacteria. In the Proteobacteria, regulation of ni-               internal glutamine deficiency (24), results in the uridylylation <br /> trogen fixation genes (nif genes) occurs by controlling the ex-               of GlnB (PII) by GlnD and also in expression and uridylylation <br /> pression and/or activity of the &#226;&#65533;&#180;54-dependent activator, NifA.                of a second PII-like protein, GlnK (44, 45). Uridylylation of <br /> Within the &#226;&#65533;&#165; subgroup of the Proteobacteria, which includes the               GlnB and GlnK prevents activation of the phosphatase activity <br /> aerobic diazotroph Azotobacter vinelandii and the facultative                 of NtrB by unmodified GlnB, leading to the phosphorylation of <br /> diazotroph Klebsiella pneumoniae, a second protein, NifL, con-                NtrC and an Ntr response wherein phosphorylated NtrC <br /> trols NifA activity. The nifL gene is cotranscribed with nifA,                (NtrC-P) activates a subset of genes required for growth under <br /> and together, the two encoded proteins comprise an unusual                    these conditions (33). Importantly, the expression of glnK is <br /> two-component system in which NifL inhibits NifA activity                     controlled by NtrC-P, which has implications for using E. coli <br /> stoichiometrically by the formation of an inactive complex (17,               as a heterologous system to study regulation of NifA activity. <br /> 21); inactivation of nifL can lead to overexpression of nif genes             Another key target of the Ntr response is the ammonia-assimi- <br /> (6, 32). Inhibition of NifA by NifL occurs in response to in- <br />                                                                               latory enzyme glutamine synthetase (GS), encoded by glnA. <br /> creased oxygen tension or fixed-nitrogen excess (13); ADP also <br />                                                                               When PII and GlnK are uridylylated, both the expression, <br /> increases the inhibitory properties of NifL in vitro, suggesting <br />                                                                               activated by NtrC-P, and activity of GS are high (33, 44). The <br /> that NifL may respond to the energy state of the cell (16). <br />                                                                               activity of GS is controlled by GlnE, which catalyzes its revers- <br /> Oxygen control of NifL activity involves the oxidation and <br />                                                                               ible adenylylation (inactivation) in response to nitrogen excess <br /> reduction of a flavin moiety bound to the N-terminal PAS <br />                                                                               (1). <br /> domain of NifL (22, 39). The nitrogen response is thought to <br /> involve components of a nitrogen-sensing system homologous                       In K. pneumoniae, the expression of nifLA is tightly con- <br /> to those best characterized in the nondiazotrophic organism                   trolled by NtrC-P, which limits expression to conditions of low <br /> Escherichia coli and is less well understood in A. vinelandii.                fixed-N supply (14). K. pneumoniae, like E. coli, has both GlnB <br />    Nitrogen sensing in E. coli and other bacteria involves the                and GlnK, and expression of glnK requires NtrC-P (23). Re- <br /> concerted activities of the glutamine sensor, GlnD (uridylyl-                 cently, it was also discovered that GlnK, but not GlnB (PII), <br /> transferase/uridylyl-removing enzyme), and one or more trim-                  modulates NifA activity (20, 23). Moreover, the uridylylation <br />                                                                               state of GlnK is apparently irrelevant with respect to NifA <br />                                                                               activity in experiments using a glnKY51N allele, which encodes <br />   * Corresponding author. Mailing address: Department of Plant Pa- <br />                                                                               a protein that cannot be uridylylated, or in a glnD background <br /> thology, Forbes 204, The University of Arizona, Tucson, AZ 85721. <br /> Phone: (520) 621-9835 or (520) 621-3263. Fax: (520) 621-9290. E-mail:         of E. coli (20) or K. pneumoniae (15). These results indicate <br /> ckennedy@u.arizona.edu.                                                       that in K. pneumoniae, effective transcriptional control of glnK <br />  <br />                                                                         812 <br />  VOL. 184, 2002                                                                                   AZOTOBACTER VINELANDII GlnK                         813 <br />  <br />  <br /> and nifLA and modulation of NifL inhibition by GlnK are               presence of NifL, demonstrating that NifL, and not NifA, <br /> important factors for the regulation of nitrogenase biosynthe-        responds to these PII-like proteins. Interestingly, 2-oxogluta- <br /> sis.                                                                  rate, at physiological concentrations, was shown to favor dis- <br />    There are several differences between regulation of nitroge-       sociation of the NifL-NifA complex, possibly indicating an <br /> nase expression in A. vinelandii and in K. pneumoniae. For            integrated role for carbon sensing (28). <br /> example, in A. vinelandii, nifLA expression is not activated by          In this report, the following two questions are addressed: is <br /> NtrC-P, nor does glnK expression appear to be N regulated (8,         GlnD-mediated uridylylation of GlnK required for A. vinelan- <br /> 31); hence, control of NifA activity appears to be the main           dii NifA activity in vivo, and does GlnK potentiate NifL inhi- <br /> mechanism regulating nitrogenase expression in this organism.         bition directly? These questions were addressed in order to <br /> Early mutagenesis experiments in A. vinelandii identified             compare nitrogen regulation of NifA activity in A. vinelandii to <br /> GlnD, previously named NfrX, as a key regulator of NifA               that in K. pneumoniae and also to study the consequence of <br /> activity because mutations in the 3&#226;&#172;&#732; end of glnD resulted in a        GlnK uridylylation under natural gene dosage. To this end, the <br /> Nif&#226;&#171;&#186; phenotype that could be suppressed to Nif&#226;&#171;&#185; by deletion           glnD suppressor strain MV72 (gln-71) was used to construct a <br /> of nifL (12, 38). This result suggested that NifA activity de-        glnKY51F mutation, encoding a protein that cannot be uridy- <br /> pends on the uridylylation of a PII-like protein. Complicating        lylated. This mutation was not stable in a wild-type strain, <br /> these results, recent experiments have shown that glnD null           indicating a requirement for GlnK-UMP for deadenylylation <br /> mutations are conditionally lethal in the wild-type A. vinelandii     of GS. Strains carrying glnKY51F failed to derepress nitroge- <br /> background because homogenous replacement of the wild-type            nase expression in response to nitrogen limitation, and GlnK <br /> glnD allele with a <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>null allele</span> does not occur in the absence of       protein-protein interactions with NifL were examined in a <br /> extragenic suppressors which elevate GS activity. These stabi-        yeast two-hybrid system. Together, results from these experi- <br /> lizing suppressor mutations include (i) a site-directed mutation      ments support a model for regulation of NifA activity involving <br /> of GS (Y407F) preventing adenylylation and (ii) unlinked sup-         GlnK where uridylylation prevents unmodified GlnK from <br /> pressor mutations (11). The inability to homogenously replace         stimulating the inhibitory properties of NifL, a model that is <br /> the wild-type glnD allele supports the hypothesis that in cells       clearly distinct from that proposed for the related K. pneu- <br /> lacking glnD, GS is always inactivated by adenylylation. Since        moniae. <br /> in A. vinelandii GS is thought to be the sole ammonia-assimi- <br /> latory pathway and glnA (encoding GS) <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>null mutants</span> cannot be                                   MATERIALS AND METHODS <br /> isolated (43), an inability to deadenylylate GS represents a             Bacterial strains and growth conditions. The bacterial strains and plasmids <br /> lethal event. One spontaneous glnD null suppressor, gln-71, is        used in this study are listed in Table 1. A. vinelandii strains were grown in Burk&#226;&#8364;&#8482;s <br /> probably in the glnE gene, encoding adenylyltransferase/adeny-        medium supplemented with 1% sucrose (BS) at 30&#194;&#176;C (35). Liquid cultures were <br />                                                                       grown with shaking in air (200 rpm). Strains carrying gln-71 were grown with 5 or <br /> lyl-removing enzyme (ATase/AR), because introduction of a             10 mM ammonium acetate (BSN5 and BSN10); higher concentrations decrease <br /> wide-host-range plasmid carrying the E. coli glnE gene into           the growth rate, probably due to depletion of glutamate pools, as observed for a <br /> gln-71 glnD&#226;&#171;&#185; strains results in reestablishment of normal GS          glnE strain of Salmonella enterica serovar Typhimurium (26). Growth of A. <br /> regulation (GS adenylylation) by NH4&#226;&#171;&#185; (11). Nevertheless,             vinelandii strains for determination of &#226;&#65533;&#164;-galactosidase activity is described below. <br />                                                                       A. vinelandii was made competent by plating on BS lacking Fe and Mo (compe- <br /> glnD null mutants are Nif&#226;&#171;&#186;, as were the original nfrX isolates, <br />                                                                       tence medium [CM]) (6). Cells were resuspended in liquid CM containing 16 <br /> and fail to uridylylate a PII-like protein, indicating that, unlike   mM MgSO4 prior to transformation. E. coli strains were grown in Luria-Bertani <br /> in K. pneumoniae, uridylylation of a PII-like protein may be          medium (LB) at 37&#194;&#176;C with shaking (250 rpm). P1vir transduction was performed <br /> required for NifA activity as well as GS deadenylylation.             as described in reference 30. E. coli was transformed by standard techniques (29). <br />    Only one PII-like protein has been identified in A. vinelandii,    Saccharomyces cerevisiae strains were maintained or grown in or on rich yeast <br />                                                                       extract-peptone-dextrose medium or minimal SD medium (yeast nitrogen <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span>, <br /> and it is named GlnK because the gene encoding this protein           dextrose, ammonium sulfate) with shaking for selection of strains carrying plas- <br /> is linked to the methylammonium transporter gene amtB, as             mids encoding prototrophy markers according to the manufacturer&#226;&#8364;&#8482;s protocol <br /> occurs in many other bacteria (42). Unfortunately, efforts to         book for the Matchmaker two-hybrid system (Clontech, Palo Alto, Calif.). Liquid <br /> demonstrate a role for this protein in A. vinelandii have been        cultures of yeast were grown at 30&#194;&#176;C with shaking (200 rpm). Yeast transforma- <br />                                                                       tions were conducted by the lithium acetate method (Clontech). Media were <br /> hampered because glnK, but not amtB, is an essential gene <br />                                                                       supplemented with antibiotics where appropriate: for A. vinelandii, ampicillin (25 <br /> (31). Therefore GlnK has been studied in vitro and in a het-          &#226;&#65533;&#174;g/ml), chloramphenicol (50 &#226;&#65533;&#174;g/ml), kanamycin (1 &#226;&#65533;&#174;g/ml), streptomycin (1 &#226;&#65533;&#174;g/ <br /> erologous E. coli system (28, 37, 40). In E. coli cells carrying a    ml), and tetracycline (5 &#226;&#65533;&#174;g/ml); for E. coli, ampicillin (100 &#226;&#65533;&#174;g/ml), chloramphen- <br /> K. pneumoniae nifH-lacZ reporter and expressing A. vinelandii         icol (30 &#226;&#65533;&#174;g/ml), kanamycin (50 &#226;&#65533;&#174;g/ml), spectinomycin (30 &#226;&#65533;&#174;g/ml), streptomycin <br /> nifLA in trans, the NifL-NifA system responds to both oxygen          (50 &#226;&#65533;&#174;g/ml), and tetracycline (10 &#226;&#65533;&#174;g/ml). <br />                                                                          Construction of a plasmid carrying glnKY51F. Forward primer glnB F H3 <br /> and fixed nitrogen (40). In this system, E. coli PII (GlnB) and       (5&#226;&#172;&#732;-TCCATAACCACAAGCTTGATAGGGG-3&#226;&#172;&#732;), which introduces a HindIII <br /> not GlnK is required for NifL-mediated inhibition in response         site (underlined) for cloning, and reverse mutagenic primer glnB Y-F (5&#226;&#172;&#732;-GCA <br /> to fixed nitrogen (37). This result is in contrast to what occurs     GAAAGTCGACTACGAATTCCG-3&#226;&#172;&#732;), which introduces an EcoRI site at ty- <br /> in K. pneumoniae, where NifL inhibition is relieved and not           rosine 51 (TAC 3 TTC) and contains a recognition site for HincII (both un- <br />                                                                       derlined), were used to amplify the 5&#226;&#172;&#732; half of glnK from pPR101 with cloned Pfu <br /> stimulated by GlnK and not GlnB (20, 23). For the A. vinelan- <br />                                                                       polymerase (Stratagene, La Jolla, Calif.). pPR101 was then digested with <br /> dii heterologous system, NtrC also appears to have some role          HindIII, which cuts in the vector, and HincII and ligated to a similarly digested <br /> in limiting NifA activity in response to excess fixed N; how this     glnKY51F PCR product to give pPR102. The presence of the glnKY51F mutation <br /> occurs was not examined (37). In vitro, formation of an inactive      and the absence of other mutations was verified by sequencing of the pPR102 <br /> A. vinelandii NifL-NifA complex, as measured by a decrease in         (LMSE sequencing facility, University of Arizona). Standard recombinant tech- <br />                                                                       niques were performed essentially as described by Maniatis et al. (29). To <br /> open complex formation at the nifH promoter, is stimulated by         construct pPR118 (Fig. 1), a 1.4-kb EcoRI/AvaI fragment containing the Tetr <br /> E. coli PII and A. vinelandii GlnK in their unuridylylated forms.     gene from pBR322 was blunt-ended with the addition of deoxynucleoside <br /> The corresponding decrease in NifA activity depends on the            triphosphates and the Klenow fragment of DNA polymerase I and cloned into <br />  814        RUDNICK ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />                                                  TABLE 1. Bacterial strains and plasmids used in this study <br /> Strain or plasmid                                             Relevant genotype and/or characteristics                                           Reference or source <br />  <br /> Strains <br />   A. vinelandii <br />      UW136                              Rifr derived from strain UW (ATCC 13705)                                                                   7 <br />      UW136Ha                            nifH1-lacZ/KSS                                                                                             This   work; 46 <br />      MV71                               gln-71 glnD::&#226;&#65533;&#8364;                                                                                             11 <br />      MV72                               gln-71 (spontaneous glnD1::&#226;&#65533;&#8364; suppressor)                                                                   11 <br />      MV378                              nifL1::KIXX nifH1-lacZ/KSS                                                                                 6 <br />      MV577                              gln-71 glnKY51F amtB3::Tetr                                                                                This   work <br />      MV577H                             gln-71 glnKY51F amtB3::Tetr nifH1-lacZ/KSS                                                                 This   work <br />      MV578                              gln-71 amtB3::Tetr                                                                                         This   work <br />      MV578H                             gln-71 amtB3::Tetr nifH1-lacZ/KSS                                                                          This   work <br />      MV579                              gln-71 glnKY51F amtB3::Tetr nifL1::KIXX                                                                    This   work <br />      MV579H                             gln-71 glnKY51F amtB3::Tetr nifL1::KIXX nifH1-lacZ/KSS                                                     This   work <br />      MV580                              gln-71 amtB3::Tetr nifL1::KIXX                                                                             This   work <br />      MV580H                             gln-71 amtB3::Tetr nifL1::KIXX nifH1-lacZ/KSS <br />      MV581                              gln-71 nifL1::KIXX                                                                                         This work <br />      MV581H                             gln-71 nifL1::KIXX nifH1-lacZ/KSS <br />  <br />   E. coli <br />     DH5&#226;&#65533;&#163;                                supE44 hsdR17 recA1 thi-1 &#226;&#338;&#172;lacU169 (&#226;&#65533;&#190;80lacZ&#226;&#338;&#172;M15) endA1 gyrA96 relA1                                        18 <br />     MC4100                              [&#226;&#338;&#172;(lac)U169 araD139 rpsL thi]                                                                              10 <br />     CK1005                              MC4100 with (&#226;&#338;&#172;glnB)::Camr <br />     CK1007                              MC4100 with (&#226;&#338;&#172;glnB)::Camr glnK::Spcr                                                                       This work <br />     CK1008                              MC4100 with (&#226;&#338;&#172;glnB)::Camr glnK::Spcr glnD99::Tn10                                                          This work <br />     NCM1971                             glnK::Spcr                                                                                                 20 <br />     NCM1686                             glnD99::Tn10                                                                                               19 <br />     NCM1736                             (&#226;&#338;&#172;glnB)::Camr                                                                                              19 <br />  <br />   S. cerevisiae <br />      SFY526                             UAS-GAL1-lacZ reporter (His&#226;&#171;&#186; Trp&#226;&#171;&#186; Leu&#226;&#171;&#186;)                                                                    Clontech <br />      HF7c                               UASG 17-mer(X3) CYC1-lacZ UAS-GAL1-HIS3                                                                    Clontech <br />  <br />  <br /> Plasmids <br />   pAB29                                 pTZ19 derivative carrying carrying nifL1::KIXX                                                             6 <br />   pBR322                                Tetr vectors source of Tetr gene                                                                           9 <br />   pGAD424b                              GAL4 activation domain fusion two-hybrid vector                                                            Clontech <br />   pGBT9b                                GAL4 binding domain fusion two-hybrid vector                                                               Clontech <br />   pJAW2                                 Suicide plasmid carrying nifH1-lacZ/KSS                                                                    46 <br />   pPR101                                pTZ19 carrying A. vinelandii glnK amtB                                                                     31 <br />   pPR102                                pTZ19 carrying A. vinelandii glnKY51F amtB                                                                 This work <br />   pPR113                                Plasmid encoding GlnKHis6 in pTrcHisB                                                                      This work <br />   pPR115                                Plasmid encoding GlnKY51FHis6 in pTrcHisB                                                                  This work <br />   pPR118                                pPR102 with a 1.4-kb EcoRI/AvaI Tetr gene cloned into amtB                                                 This work <br />   pTrcHis B                             His tag cloning vector                                                                                     Invitrogen <br />   pTD1                                  Activation domain control fusion for two-hybrid system                                                     Clontech <br />   pVA3                                  Binding domain control fusion for two-hybrid system                                                        Clontech <br />   a <br />       Strain UW136H is a remake of strain MV101 (46). <br />   b <br />       For descriptions of two-hybrid fusion constructs, see Materials and Methods. <br />  <br />  <br />  <br />  <br /> the single NspV site of the amtB gene in pPR102. glnKY51F was verified in this           Construction of yeast two-hybrid plasmids. Plasmid vectors used in the yeast <br /> clone by sequencing prior to transformation of A. vinelandii. A clone carrying the    two-hybrid assays were pGAD424 (activation domain fusion) and pGBT9 (bind- <br /> tetracycline resistance gene in the forward orientation, pPR118, was isolated to      ing domain fusion) from the Matchmaker two-hybrid system. For cloning of nifL, <br /> be used for mutagenesis of the A. vinelandii chromosome (Fig. 1).                     glnK, or glnKY51F, PCR products were amplified from plasmid or chromosomal <br />    Construction of plasmids expressing GlnKHis6 and GlnKY51FHis6. pPR113              template DNA and ligated to pGEM-T EASY, again, as an intermediate step. <br /> was constructed by PCR amplification of glnK from pPR101 with the primers             For each of these genes, the entire reading frame was cloned by the introduction <br /> GLNK FB (5&#226;&#172;&#732;-GAGATGGATCCGATGAAGCTAGT-3&#226;&#172;&#732;) and GLNK RK (5&#226;&#172;&#732;-                              of restriction site compatible with sites in the two-hybrid vectors to generate <br /> AATCCACGGTACCCTCCTG-3&#226;&#172;&#732;), which introduce a BamHI and a KpnI site                      in-frame fusions. nifL was amplified with the primers NifL.For.ApoI (5&#226;&#172;&#732;-TCGC <br /> (underlined), respectively, for directional cloning into the His tag vector <br />                                                                                       CGAATTTCTTGGATCGACGAGG-3&#226;&#172;&#732;) and NifL.Rev.BamHI (5&#226;&#172;&#732;-GGTTGG <br /> pTrcHisB (Invitrogen, Carlsbad, Calif.). The PCR product was first cloned into <br />                                                                                       ATCCATGGGCATTCAT-3&#226;&#172;&#732;) from chromosomal DNA as the template; glnK <br /> the T-A cloning vector pGEM-T EASY as an intermediate (Promega, Madison, <br />                                                                                       and glnKY51F were amplified from pPR101 and pPR102, respectively. Amplifi- <br /> Wis.). The BamHI/KpnI fragment was then cloned into similarly digested <br /> pTrcHisB to give pPR113. pPR115 was constructed in essentially the same               cation of glnK or glnKY51F was performed with the primers GlnK.For.EcoRI <br /> manner, except that glnKY51F was amplified from plasmid pPR102 and cloned             (5&#226;&#172;&#732;-TTACACGGAATTCTGTTTCATGAA-3&#226;&#172;&#732;) and GlnK.R.BamHI (5&#226;&#172;&#732;-CGGG <br /> directly into pTrcHisB following digestion. pPR113 and pPR115 have N-terminal         GGGGATCCTGGGCT-3&#226;&#172;&#732;). The cloned products were fully or partially digested <br /> translational fusions that contains six consecutive histidine residues, encoded by    with EcoRI and cloned into either pGBT9 or pGAD424. All plasmids were <br /> pTrcHisB.                                                                             verified by sequencing (LMSE sequencing facility, The University of Arizona). <br />  VOL. 184, 2002                                                                                                 AZOTOBACTER VINELANDII GlnK                         815 <br />  <br />  <br />  <br />  <br />    FIG. 1. Map of the glnK amtB region and construction of <br /> glnKY51F. To construct the glnKY51F mutation, the 5&#226;&#172;&#732; half of glnK was <br /> amplified with a mutagenic primer that contained the TAC 3 TTC <br /> base change encoding residue 51. The mutation of Y51 prevents uri- <br /> dylylation of the encoded protein and introduces a new EcoRI site. For <br /> selection of A. vinelandii transformants carrying the <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>point mutation</span>, a                FIG. 2. PCR analysis of glnKY51F amtB3::Tet transformants. (A) <br /> tetracycline resistance gene was introduced into the 5&#226;&#172;&#732; end of the                  Large- and small-colony Tetr Amps transformants of wild-type strain <br /> downstream amtB gene carried on the plasmid. A. vinelandii mutants                  UW136. (Top) amplified PCR products; (bottom) products after <br /> were selected on the basis of Tetr and screened for sensitivity to                  EcoRI digestion. EcoRI cleaves glnKY51F in half, while glnK is EcoRI <br /> vector-borne Ampr. Of these, glnK was amplified and digested with                   resistant. Lanes: 1 to 5, small-colony transformants; 6 to 9, large-colony <br /> EcoRI to identify mutants carrying the glnKY51F allele. The genetic                 isolates; 10, glnKY51F PCR product amplified from plasmid pPR118; <br /> structure of the stable gln-71 transformants MV577 and MV578 is                     11, glnK amplified from plasmid pPR101. 12, Ampr Tetr isolate result- <br /> shown. A, AvaI; E, EcoRI; H, HincII; Hd, HindIII; N, NspV; P PstI.                  ing from integration of plasmid pPR118 carrying both alleles in tan- <br />                                                                                     dem. (B) Transformation of glnD1::&#226;&#65533;&#8364; suppressor strain MV72. Lanes: <br />                                                                                     1 to 5, Tetr Amps isolates; 6, glnKY51F amplified from plasmid <br />                                                                                     pPR118; 7, wild-type glnK from pPR101. <br />    Recombination of glnKY51F onto the A. vinelandii chromosome. The suicide <br /> plasmid pPR118 was used to transform A. vinelandii strains UW136 and MV72 <br /> followed by selection on tetracycline. To screen for allelic replacement mutants, <br /> Tetr transformants were patched onto BSN plates supplemented with ampicillin.       throughout a 12-h period, at which time the OD600 was recorded for each <br /> Tetr Amps clones were purified and analyzed for the presence of the glnKY51F        sample. Samples were immediately frozen and stored at &#226;&#171;&#186;20&#194;&#176;C. &#226;&#65533;&#164;-Galactosidase <br /> mutation by PCR amplification of the &#226;&#172;&#402;350-bp glnK gene using primers K101U          activities were determined for each sample in units per milliliter according to the <br /> (5&#226;&#172;&#732;-ACTTGAATCGGGATCGTTT-3&#226;&#172;&#732;) and K101D (5&#226;&#172;&#732;-GCCTTTGCGCAGCG                            following calculation, which has been modified for an enzyme-linked immu- <br /> TCAT-3&#226;&#172;&#732;) from single colonies as templates followed by EcoRI digestion.             nosorbent assay microtiter plate reader: 1,000[OD414 &#226;&#171;&#186; (1.75 &#226;&#171;&#187; OD540)]/(&#226;&#338;&#172;t &#226;&#171;&#187; <br /> nifL1::KIXX strains were constructed by transformation with pAB29 (6) fol-          v), where &#226;&#338;&#172;T is the time of the reaction and v is the volume of extract. Each time <br /> lowed by selection on kanamycin and screening for sensitivity to ampicillin.        point was assayed in duplicate, and the experiment was repeated for each strain <br />    Construction of nifH1-lacZ/KSS reporter strains. A. vinelandii strains UW136,    at least three times (34). The activities for each time point were then plotted <br /> MV72, MV577, MV578, and MV579 were transformed with pJAW2 (47) to                   against OD600. This analysis indicated that the differential rate of &#226;&#65533;&#164;-galactosidase <br /> generate nifH1-lacZ/KSS transcriptional fusions on the chromosome. pJAW2 is         synthesis was constant throughout the growth curve and therefore the slope of <br /> a suicide vector carrying the nifH1-lacZ/KSS fusion, which carries a nifH-lacZ      each line was approximated to give rates, which are reported in Table 2. &#226;&#65533;&#164;-Ga- <br /> fusion followed by the kanamycin and streptomycin marker genes from Tn5.            lactosidase assays of yeast strains were performed by the protocols supplied by <br /> Transformants were selected on streptomycin and screened for sensitivity to         the manufacturer (Clontech). Briefly, single colonies of strains carrying the <br /> chloramphenicol to identify allelic exchange mutations. The fusions in Smr Cams     appropriate plasmids were inoculated into SD selective media and grown to <br /> isolates were verified by PCR amplification of a nifH-lacZ product from the         saturation at 30&#194;&#176;C. Cultures were then diluted into 8 ml of yeast extract-peptone- <br /> chromosome (not shown).                                                             dextrose and grown to an OD600 of 0.5 to 0.8, and 1.5 ml of culture was pelleted, <br />    Construction of E. coli strains. To construct an E. coli strain devoid of both   washed, concentrated, and assayed for determination of &#226;&#65533;&#164;-galactosidase activity. <br /> glnB and glnK in strain MC4100, &#226;&#338;&#172;glnB::Camr was moved from strain NCM1736              Preparation of cell extracts, in vitro uridylylation assays, and Western blot- <br /> (19) by P1 transduction to generate CK1005. CK1005 was then transduced with         ting. For preparation of cell extracts, CK1007 or CK1008 carrying plasmid <br /> glnK::Spcr with a lysate grown on strain NCM1971 (20) to generate strain            pPR113 or pPR115 was grown in 25 ml of LB in a 125-ml flask to an OD600 of <br /> CK1007. A triple glnB glnK glnD mutant strain was constructed by transducing        &#226;&#172;&#402;0.5, at which time the cultures were induced with 500 &#226;&#65533;&#174;M IPTG (isopropyl-&#226;&#65533;&#164;- <br /> glnD99::Tn10 from NCM1686 (19) followed by selection on tetracycline and            D-galactopyranoside) for 2 to 3 h. Strain CK1008 grew markedly slower than <br /> screening for glutamine bradytrophy.                                                CK1007; plasmid pPR115 inhibited growth of CK1007, CK1008, and MC4100, <br />    Growth curve and &#226;&#65533;&#164;-galactosidase assays. Cells used for growth curves were       with greater inhibition observed after induction. The cells were harvested, <br /> first grown overnight in 10-ml starter cultures in BSN5. The next morning, the      washed once in an equal volume of uridylylation buffer (50 mM Tris-Cl, 100 mM <br /> cultures were diluted to 10 Klett units (optical density at 600 nm [OD600] &#226;&#172;&#8225; 0.1)   KCl, 10 mM MgCl2 [pH 7.5]), and resuspended in 0.5 or 1.0 ml of the same buffer <br /> in either BS or BSN10 and grown for 27 h. Samples were read periodically on a       (4). The cell suspensions were lysed with three 5-s bursts on a Branson sonicator <br /> Klett-Summerson colorimeter through a no. 54 green filter measuring the OD526       on medium-high power followed by three cycles of rapid freezing in liquid N2 and <br /> in sidearm flasks, and values were plotted on a semilog graph. For analysis of      rapid thawing at 37&#194;&#176;C. Cell debris was then pelleted by centrifugation at 12,000 &#226;&#171;&#187; <br /> nifH1-lacZ fusions in A. vinelandii, 10-ml overnight cultures were grown in BS      g for 15 min at 4&#194;&#176;C, and the soluble proteins were recovered in the supernatant. <br /> supplemented with 5 mM urea (BSU5) and the appropriate antibiotics. The             All extracts were quantitated for protein content by a modified Bradford method <br /> following day, the cultures were diluted to approximately 20 Klett units in fresh   (Bio-Rad, Hercules, Calif.). Uridylylation of GlnKHis6 was carried out in vitro <br /> BSN10 for repression or BSU2 for derepression. Five samples were taken              basically as described in reference 25. Briefly, 30 &#226;&#65533;&#174;g of soluble protein containing <br />  816      RUDNICK ET AL.                                                                                                                         J. BACTERIOL. <br />  <br />  <br /> GlnKHis6 or GlnKY51FHis6 was incubated in uridylylation buffer plus 0.2 mM              TABLE 2. Expression of nifH1-lacZ in glnKY51F strains of A. <br /> ATP. [&#226;&#65533;&#163;32-P]UTP (1 &#226;&#65533;&#174;l; 800Ci/mmol) and 1 mM 2-oxoglutarate (pH 7.5) were                                       vinelandii <br /> added to initiate labeling of GlnK. The reaction mixture was incubated for 30 <br /> min. at 30&#194;&#176;C. Duplicate samples were incubated with unlabeled UTP for Western                                                                 &#226;&#65533;&#164;-Galactosidase <br /> analysis and were processed in parallel. All samples were mixed in 1&#226;&#171;&#187; sodium                                                                       activity <br />                                                                                              b <br />                                                                                     Strain                 Relevant genotype                 (U/ml/OD600 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>)a <br /> dodecyl sulfate sample buffer and boiled for 5 min, and half of the reaction <br /> mixture was separated by sodium dodecyl sulfate&#226;&#8364;&#8220;12% polyacrylamide gel elec-                                                                 &#226;&#171;&#185;Nc          &#226;&#171;&#186;Nd <br /> trophoresis (5). The gel containing labeled proteins was dried overnight between <br /> cellulose sheets and exposed to Kodak Biomax X-ray film for approximately 30       UW136H         Wild type                                    200       16,100 <br /> min. The nonradioactive gel was blotted to a nitrocellulose membrane with a        MV72H          gln-71                                       240       15,130 <br /> semidry transblotter (Bio-Rad) following equilibration in Bjerrum and Schafer-     MV71H          gln-71 glnD1::&#226;&#65533;&#8364;                              160          160 <br /> Nielson transfer buffer (48 mM Tris, 39 mM glycine, 20% methanol [pH 9.2])         MV577H         gln-71 amtB3::Tetr glnKY51F                  260          860 <br /> (Bio-Rad electroblotter manual). Polyclonal E. coli PII rabbit antiserum was       MV578H         gln-71 amtB3::Tetr                           191       12,010 <br /> applied as the primary antibody. Alkaline phosphatase conjugate was then used      MV579H         gln-71 amtB3::Tetr glnKY51F                4,880       14,610 <br /> to detect the proteins, by using an AP substrate pack for colorimetric develop-                      nifL1::KIXX <br /> ment (Bio-Rad).                                                                    MV580H         gln-71 amtB3::Tetr nifL1::KIXX              NTe         5,870 <br />                                                                                    MV581H         gln-71 nifL1::KIXX                         5,260        9,570 <br />                                                                                     a <br />                                  RESULTS                                              See Materials and Methods for determination of &#226;&#65533;&#164;-galactosidase activities. <br />                                                                                     b <br />                                                                                       All strains carry a nifH1-lacZ/KSS transcriptional fusion (46). <br />                                                                                     c <br />    Uridylylation of GlnK is essential for growth of wild-type                         BS medium supplemented with 10 mM ammonium acetate. <br />                                                                                     d <br />                                                                                       BS containing 2 mM urea as a nonrepressive nitrogen source. <br /> strain UW136. One hypothesis for the lethality of glnD null                         e <br />                                                                                       NT, not tested. <br /> mutations in A. vinelandii is that uridylylation of GlnK is re- <br /> quired for deadenylylation of GS. Inactivation of GS in A. <br /> vinelandii by adenylylation would be lethal, as are glnA null                      dominant negative effect on colony growth, since amtB muta- <br /> mutations in this organism, attributed to the lack of an alter-                    tions do not impair growth on the selective medium (31). To <br /> nate ammonia-assimilatory mechanism and an inability to                            determine if the glnKY51F mutation had recombined into the <br /> transport glutamine (43). This is apparently the case, since                       chromosome along with amtB3::Tetr, glnK was PCR amplified <br /> suppressors of glnD mutations (i.e., gln-71) resemble glnE                         from several transformants. Following amplification, the PCR <br /> strains of other organisms, and E. coli glnE complements this                      products were digested with EcoRI, which cuts the glnKY51F <br /> suppressor strain for restoration of normal regulation of GS                       allele in half, while glnK&#226;&#171;&#185; is EcoRI resistant. This analysis <br /> activity by NH4&#226;&#171;&#185; (11). Therefore, attempts were made to con-                       revealed that cells in the larger of the two colony types, as <br /> struct glnK null mutations in a gln-71 strain (data not shown).                    hypothesized, were glnK&#226;&#171;&#185; (Fig. 2A, lanes 6 to 9). The smaller <br /> These attempts failed, which indicates that GlnK is essential                      colonies contained both alleles, demonstrating an inability of <br /> for at least two functions in A. vinelandii: deadenylylation of                    glnKY51F to homogeneously replace glnK in these cells (lanes <br /> GS and another, unknown function. Nevertheless, if the sec-                        1 to 5) and consistent with other presumed lethal mutations in <br /> ond essential function of GlnK does not require the uridyly-                       A. vinelandii reported by this and other laboratories (11, 31, 43, <br /> lated form but only the unmodified form, then it might be                          49). The controls indicate that the enzyme digestion was spe- <br /> possible to construct glnK mutations altered only at the site of                   cific for glnKY51F and was nearly complete (lanes 10 and 11). <br /> uridylylation. This protein would perform the functions of the                     Lane 12 contains glnK amplification products from a Tetr <br /> unmodified form but lack the activities of GlnK-UMP, includ-                       Ampr transformant carrying and integrated copy of pPR118. <br /> ing those that may include regulation of NifA activity. To test                    When these products are digested, the concentration of the <br /> this hypothesis, a glnKY51F mutation was constructed by elon-                      smaller glnKY51F band appears to be higher than in the un- <br /> gation of a mutagenic primer (see Materials and Methods).                          stable small-colony transformants. This result indicates that <br /> The point mutation at Y51 of glnK created a new EcoRI site,                        the number of copies of the glnKY51F-containing chromosome <br /> which was used to screen for the presence of the mutation. To                      in the unstable colonies may be lower than that obtained by <br /> select for A. vinelandii strains carrying this mutation, a Tetr                    integrating a second copy of glnK on the chromosome by single <br /> cassette was cloned into the 5&#226;&#172;&#732; end of the adjacent amtB gene                      recombination. <br /> to give pPR118 (Fig. 1). Previous studies in this organism, K.                        In cases of unstable alleles, the drug resistance carried on <br /> pneumoniae, and heterologous E. coli systems have determined                       the drug cassette is lost when the selective pressure for the <br /> that amtB mutations have little or no effect on the regulation of                  mutated copy is removed. Therefore, UW136 glnKY51F trans- <br /> nitrogenase expression (23, 31, 37). Therefore, we expected                        formants were tested for stability of the tetracycline resistance <br /> these two mutations together to reflect the effects of the                         marker by plating on BSN10 in the absence of tetracycline <br /> glnKY51F allele. However, since the complete function of                           followed by reculturing on tetracycline-containing medium. <br /> AmtB has not been defined, it cannot be ruled out that loss of                     This experiment revealed that the antibiotic resistance in <br /> amtB may cause differences in nif expression in response to                        strains carrying glnKY51F amtB3::Tetr was unstable in the wild- <br /> ammonium in the growth medium (41). Therefore, glnK&#226;&#171;&#185;                               type UW136 background; cells from the small colonies did not <br /> strains carrying amtB3::Tetr were examined in parallel.                            grow when recultured on tetracycline, in contrast to the large <br />    To place the glnKY51F mutation on the chromosome, sui-                          glnK&#226;&#171;&#185; transformants. These analyses indicate that the <br /> cide plasmid pPR118 was used to transform the wild-type A.                         glnKY51F mutation is lethal in the wild-type background. <br /> vinelandii strain UW136. Transformation gave rise to small and                        Strains carrying the gln-71 suppressor mutation stabilize <br /> large Tetr Amps (allelic exchange mutants) colony types, sug-                      glnKY51F. Recent work on the glnD gene of A. vinelandii re- <br /> gesting that the glnKY51F mutation, perhaps present in the                         vealed that null mutations, where the 5&#226;&#172;&#732; or central region is <br /> smaller of the two colony types, might be unstable or have a                       removed or replaced by gene cassettes (&#226;&#65533;&#8364; or KIXX), are lethal <br />  VOL. 184, 2002                                                                            AZOTOBACTER VINELANDII GlnK               817 <br />  <br />  <br /> and cannot be stabilized in the wild-type background. One of <br /> these alleles (glnD1::&#226;&#65533;&#8364;) was stabilized by a suppressor muta- <br /> tion, gln-71, identified after spontaneous appearance of a large, <br /> stably antibiotic-resistant (Spcr) colony, in which GS activity is <br /> not regulated by ammonium. The gln-71 mutation is apparently <br /> in the glnE gene, encoding ATase/AR, because GS activity <br /> becomes regulated in MV72 (a glnD&#226;&#171;&#185; derivative of MV71 <br /> carrying only gln-71) (i.e., is inactivated by ammonium) after <br /> introduction of a plasmid carrying the E. coli glnE gene (11). <br />                                                                         FIG. 3. Western analysis and uridylylation of A. vinelandii GlnK <br /> To determine whether the glnKY51F allele could be stabilized         and GlnKY51F in E. coli. (Top) Western blot of E. coli extracts <br /> by the gln-71 suppressor, MV72 was transformed with pPR118.          incubated with 2-oxoglutarate and UTP for 30 min. (Bottom) Auto- <br /> Transformants were selected on BSN5 with tetracycline, and all       radiograph of the same extracts labeled with [&#226;&#65533;&#163;32-P]UTP. Lanes: 1, <br /> were of a uniform large colony size. About 60% of the Amps           purified GlnKHis6 as a size standard; 2, CK1007 (glnB glnK); 3, <br />                                                                      CK1007(pPR113); 4, CK1007(pPR115); 5, CK1008 (glnB glnK glnD); <br /> Tetr transformants carried only the glnKY51F allele; the others      6, CK1008(pPR113); 7, CK1008(pPR115); 8, no protein. <br /> were glnK&#226;&#171;&#185;, as determined by PCR amplification followed by <br /> EcoRI digestion of the PCR products (Fig. 2B, lanes 1 to 5). In <br /> contrast to the wild-type A. vinelandii pPR118 transformants, <br />                                                                      In these experiments, the doubling time of MV577 was greater <br /> none of the MV72 transformants carried both the mutated and <br />                                                                      than 11 h in N-free media. MV578, the corresponding glnK&#226;&#171;&#185; <br /> wild-type glnK alleles. In addition, Tetr in the glnKY51F <br />                                                                      strain, doubled in about 2.5 h (Fig. 4A). An increase in the <br /> amtB3::Tetr transformants was stable and not lost after sub- <br />                                                                      growth rate of MV577 was observed with the addition of 10 <br /> culturing on antibiotic-free medium (data not shown). One <br />                                                                      mM ammonium acetate to the cultures (Fig. 4B), although <br /> gln-71 glnKY51F amtB3::Tetr transformant was named MV577, <br />                                                                      there was no increase in the rate of MV578, which may indicate <br /> and one corresponding glnK&#226;&#171;&#185; transformant was named MV578 <br />                                                                      effects of GlnKY51F on other cellular functions in N-rich me- <br /> (Fig. 1). These results support the hypothesis that GlnK-UMP <br /> is required for control of GS activity, almost certainly by sig- <br /> naling deadenylylation of GS by ATase/AR. Importantly, they <br /> also indicate that MV72 could probably be used as a host strain <br /> to study the role of GlnK in the regulation of NifA activity in <br /> A. vinelandii. <br />    glnKY51F encodes a protein which cannot be uridylylated. <br /> To test whether the GlnKY51F protein could be modified by <br /> uridylylation, glnKY51F and wild-type glnK were expressed in <br /> E. coli from plasmids pPR113 and pPR115 (see Materials and <br /> Methods). These two plasmids express the complete GlnK or <br /> GlnKY51F proteins fused to a six-residue N-terminal His tag. <br /> Since E. coli expresses GlnB and GlnK which can be uridyly- <br /> lated, a strain was constructed that lacked both glnB and glnK <br /> (CK1007) or glnB glnK and glnD (CK1008) (see Materials and <br /> Methods) (Table 1). For the uridylylation experiments, strains <br /> CK1007 and CK1008 carrying either pPR113 or pPR115 were <br /> grown to mid-log phase in LB or LB plus glutamine (100 <br /> &#226;&#65533;&#174;g/ml), and expression of GlnK or GlnKHis6 was induced from <br /> the plasmid with 0.5 mM IPTG. In extracts, GlnKHis6 could be <br /> uridylylated in a GlnD-dependent fashion while GlnKY51FHis6 <br /> could not (Fig. 3, bottom). Western blotting of the extracts <br /> used in the uridylylation experiments indicated that both plas- <br /> mids expressed stable proteins which cross-reacted with E. coli <br /> PII antisera (Fig. 3, top) (gift from W. vanHeeswijk). In the <br /> upper panel of Fig. 3 it is also evident that GlnKY51F is not <br /> detected at the same level as GlnK. This may be because the <br /> level of expression from pPR115 is lower or that the Y51F <br /> mutation disrupts a region of important antigenicity. <br />    Uridylylated GlnK is required for growth in N-free media <br /> and for NifA activity. MV577 (glnKY51F amtB3::Tetr gln-71) <br /> formed only very small colonies on N-free medium, indicating <br /> that the glnKY51F mutation affected nitrogen fixation and/or <br /> assimilation (data not shown). Growth rates were determined             FIG. 4. Growth of glnKY51F strains in BS. (A) N-free growth; (B) <br />                                                                      growth with the addition of 10 mM ammonium acetate. &#228;&#161;&#178;, UW136 <br /> for strains MV577, MV578, and MV579 and control strains in           (wild-type strain); &#228;&#8240;&#172;, MV72 (gln-71); &#197;&#8217;, MV577 (gln-71 glnKY51F <br /> order to quantitate the effects of the glnKY51F mutation on          amtB3::Tetr); &#226;&#171;&#187;, MV578 (gln-71 amtB3::Tetr); (&#226;&#171;&#187; &#226;&#177;&#65533; ), MV579 (gln-71 <br /> growth in the presence and absence of fixed nitrogen (Fig. 4).       glnKY51F amtB3::Tetr nifL1:: KIXX). <br />  818     RUDNICK ET AL.                                                                                                              J. BACTERIOL. <br />  <br />  <br /> dium. Nonetheless, the glnKY51F mutation severely impaired                   TABLE 3. GlnK interactions in a yeast two-hybrid assay <br /> growth under diazotrophic conditions. Also in these experi-                                        Fusionb                        &#226;&#65533;&#164;-Galactosidase <br /> ments, a nifL mutation was epistatic to the glnKY51F mutation        Straina                                                          activity <br /> (MV579). This result is consistent with the hypothesis that the                       BD                      AD                 (U/ml/OD600 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>)c <br />  <br /> unmodified GlnK positively influences NifL inhibition and            S122  d,e <br />                                                                                  p53               SV40 large T-antigen                  110 <br /> GlnK does not effect nitrogenase activity per se, as measured        S112a       NifL              GlnK                                   18 <br /> by diazotrophic growth.                                              S113e       GlnK              NifL                                   49 <br />                                                                      S114e       NifL              GlnKY51F                               16 <br />    To examine the effect of the glnKY51F mutation on NifA            S124        GlnKY51F          NifL                                  &#226;&#172;&#65533;2 <br /> activity, nifH1-lacZ reporter strains were constructed (Table 1)     S103        Vector only       Vector only                           &#226;&#172;&#65533;2 <br /> and the differential rates of &#226;&#65533;&#164;-galactosidase synthesis were         S123        GlnK              Vector only                           &#226;&#172;&#65533;2 <br /> determined (see Materials and Methods). These data indicate          S125        GlnKY51F          Vector only                           &#226;&#172;&#65533;2 <br />                                                                      S132        Vector only       NifL                                  &#226;&#172;&#65533;2 <br /> that the gln-71 suppressor mutation, used to stabilize the           S134        NifL              Vector only                           &#226;&#172;&#65533;2 <br /> glnKY51F mutation, had little effect on nifH expression under        S135        Vector only       GlnK                                  &#226;&#172;&#65533;2 <br /> derepressed conditions (MV72H), as did the combined gln-71           S136        Vector only       GlnKY51F                              &#226;&#172;&#65533;2 <br /> and amtB3::Tetr mutations in strain MV578H. Therefore, any             a <br />                                                                           All strains are derivatives of yeast strain SFY526 carrying two plasmids. <br /> effects in the isogenic glnKY51F strains are a consequence of          b <br />                                                                           BD, fusion to GAL4 DNA-binding domain; AD, fusion to GAL4 transcrip- <br /> the glnK mutation and not other mutations in these strains. For      tional activation domain. <br />                                                                         c <br />                                                                           See Materials and Methods for determination of &#226;&#65533;&#164;-galactosidase activities. <br /> comparison, a nifH1-lacZ fusion in MV71 (glnD1::&#226;&#65533;&#8364; gln-71)               d <br />                                                                           Strain S122 carries plasmid pVA3 and pTD1 from the Clontech kit as <br /> was constructed and assayed. Compared to MV578H (glnK&#226;&#171;&#185;),             positive controls. <br />                                                                         e <br />                                                                           This strain also gives sufficient expression of a GAL4-his reporter in host <br /> strain MV577H (glnKY51F) expressed about 5% of the amount            strain HF7c to allow growth on His&#226;&#171;&#186; DO (Drop Out) medium (Clontech, Palo <br /> of &#226;&#65533;&#164;-galactosidase under derepressing conditions but still more      Alto, Calif.). <br /> than the glnD strain (compare MV577H and MV71H), indi- <br /> cating that uridylylated GlnK is required for full activation of <br /> the nifH promoter in vivo (Table 2).                                 demonstrating a mechanism for activating the inhibitory prop- <br />    To determine if the effect of GlnK on NifA activity was           erties of NifL in response to nitrogen sufficiency. <br /> mediated through NifL, nifL in MV577 was disrupted with a <br /> KIXX cassette encoding kanamycin resistance to give strain                                         DISCUSSION <br /> MV579 and the corresponding nifH1-lacZ reporter strain, <br />                                                                         GlnK-UMP is essential for activation of GS in A. vinelandii. <br /> MV579H. Both growth on N-free medium (Fig. 4A) and <br />                                                                      In this work, we proposed that gln-71, a likely allele of glnE, <br /> nifH1-lacZ expression (Table 2) were restored to wild-type <br />                                                                      would stabilize glnKY51F, as it did glnD1::&#226;&#65533;&#8364;, because either <br /> levels under derepressing conditions by mutation of nifL, and <br />                                                                      glnD or a glnKY51F mutation would create a state in which GS <br /> as observed with other nifL strains, expression of nifH1-lacZ <br />                                                                      would be irreversibly inactivated by adenylylation, if GlnK <br /> was not fully repressed in the presence of NH4&#226;&#171;&#185; (Table 2) (6). <br />                                                                      positively influences the adenylylation of GS (11). The allelic <br /> As controls, nifL1::KIXX was introduced into strains MV72H <br />                                                                      instability of these mutations mutation, due to the effects on <br /> and MV578H for comparison (MV580H and MV581H).                       GS, might be expected because A. vinelandii does not transport <br /> Taken together, these data indicate that unmodified GlnK             glutamine and lacks a glutamate dehydrogenase activity as an <br /> negatively regulates NifA activity by a mechanism involving          alternate ammonia-assimilatory pathway (43). The gln-71 mu- <br /> NifL.                                                                tation did stabilize the glnKY51F allele, suggesting a depen- <br />    GlnK and GlnKY51F interact with NifL in a yeast two-              dence on the uridylylated form of GlnK for deadenylylation of <br /> hybrid assay. To assess if GlnK interacts directly with NifL,        GS. However, the effects of the glnKY51F are probably not <br /> translational fusions to the GAL4 DNA binding and activation         restricted to those of GS, because a significant decrease in <br /> domains were made using nifL, glnK, and glnKY51F. The re-            growth rate of the glnKY51F strain (MV577) over the glnK&#226;&#171;&#185; <br /> sulting plasmids and pairs of plasmids were used to transform        strain (MV578) was noted when fixed N was present in the <br /> yeast strain SFY526, carrying a lacZ reporter downstream of          culture medium (Fig. 4B). This result illustrates the complexity <br /> the GAL1 upstream activation sequence (Matchmaker sys-               of the mutation and suggests the possible occurrence of other <br /> tem). In this system, activation of the reporter is dependent on     GlnK targets in A. vinelandii and in other organisms. <br /> the fusion proteins coming into proximity long enough to ac-            To determine whether the only reason for the lethality of <br /> tivate transcription, a function that is dependent on interaction    glnK mutations was this requirement, the null allele <br /> of the fused target proteins. The results indicate that none of      glnK::KIXX (31) was used to transform MV72 (gln-71). Kanr <br /> the plasmids carrying fusions activated expression when paired       Amps transformants were unstable (high-frequency loss of <br /> with either vector alone; in these strains, &#226;&#65533;&#164;-galactosidase ac-      Kanr after growth without kanamycin), indicating that GlnK in <br /> tivity was less than 2 U (Table 3). As a positive control, p53 and   its unmodified form is required for an as-yet-unidentified func- <br /> simian virus 40 large T antigen (supplied with the kit) together     tion in A. vinelandii. The reason why glnK null mutants cannot <br /> activated transcription sufficiently to give &#226;&#172;&#402;100 &#226;&#65533;&#164;-galactosidase    be isolated remains unknown, although A. vinelandii is well <br /> units. In test assays, GlnK and NifL interacted in both vector       suited for a study that may reveal potentially new regulatory <br /> combinations (16 to 49 &#226;&#65533;&#164;-galactosidase units). GlnKY51F in-          targets for this class of proteins, possibly involved in carbon or <br /> teracted with NifL, but only when NifL was fused to the GAL4         energy metabolism or DNA processing, as may occur in <br /> DNA-binding domain (Table 3). These results indicate that            Rhodobacter capsulatus, where new PII targets, including dini- <br /> NifL and unuridylylated GlnK may physically interact in vivo,        trogenase reductase ADP-ribosyl transferase (DRAT), have <br />  VOL. 184, 2002                                                                                 AZOTOBACTER VINELANDII GlnK                         819 <br />  <br />  <br /> been identified by using a yeast two-hybrid system (Werner           hybrid system, both GlnK and GlnKY51F interacted signifi- <br /> Klipp, personal communication). In these experiments, as in          cantly with A. vinelandii NifL; that NifL and NifA interact in <br /> other two-hybrid screens, caution should be used before any          yeast was shown previously (27). The possibility that GlnK <br /> conclusions can be made concerning the validity of these new         interacts with NifA as well could not be ruled out. This, how- <br /> targets, and each will have to be verified in vivo or in vitro. On   ever, seems unlikely based on in vitro data (28). Nevertheless, <br /> a positive note, however, an interaction with DRAT in this           the demonstration of interaction between GlnK and NifL is <br /> organism might be predicted from genetic evidence in the             further evidence that these two proteins interact in such a way <br /> related organism Rhodospirillum rubrum (48).                         that GlnK stabilizes the NifL-NifA nonactivating complex or <br />    GlnKY51F cannot relieve the inhibitory properties of NifL.        promotes an inhibitory conformation of NifL and uridylylation <br /> An advantage of these in vivo experiments over other ap-             prevents either function. <br /> proaches is that in assays for NifA activity, gene expression was       Since no glnK null mutants of this organism are available, it <br /> limited by both the chromosomal location and the preservation        is difficult to predict what may be the result of loss of GlnK <br /> of native promoters for the regulatory proteins. These results       with respect to NifA activity and to other cellular functions in <br /> should therefore reflect natural changes in nitrogenase expres-      A. vinelandii. However, experiments with E. coli indicate that <br /> sion with respect to growth phase and fixed nitrogen content         NifL is less inhibitory to NifA activity in the absence of both <br /> without the ambiguity that might arise from the use of heter-        GlnB and GlnK than in their presence (37). This suggests that <br /> ologous or in vitro systems or highly expressed promoters. The       in A. vinelandii, GlnK may act solely as a negative regulator of <br /> most significant finding of this work is that, in contrast to what   NifA activity by stimulating the inhibitory properties of NifL <br /> occurs in K. pneumoniae, uridylylation of A. vinelandii GlnK is      and that in its absence, nif gene expression would occur con- <br /> apparently required for relief of NifL inhibition of NifA activ-     stitutively, regardless of N status. It will be important to learn <br /> ity. However, this requirement is not absolute because signif-       if interaction of GlnK with NifL is dependent on the uridyly- <br /> icant nifH-lacZ expression was detected with strain MV577H           lation state of GlnK and which residues are important for such <br /> (glnKY51F) under N-limited conditions. The residual activity         an interaction. This question can be addressed either in vitro <br /> might indicate that GlnKY51F may not form as tight a complex         or by using a two-hybrid approach. It is also of interest to <br /> with NifL as does unmodified wild-type GlnK, such that NifL          discover new regulatory targets for A. vinelandii GlnK. Finding <br /> inhibition can be partially relieved by a reduction in fixed         these targets may explain the apparent essentiality of this pro- <br /> nitrogen, perhaps by the binding of 2-oxoglutarate (28) or           tein and have implications for all other organisms whose ge- <br /> other mechanisms affecting GlnK activity.                            nomes contain one or more homologs of this interesting group <br />    As has been proposed for A. vinelandii and other organisms        of signal transduction proteins. <br /> which probably have only one PII-like protein, the composition <br /> of A. vinelandii GlnK may be an amalgam of residues such that                                <br /> a single protein can function with all regulatory targets (2). In      <br /> light of this, it is also not surprising that while GlnK of A.     <br /> vinelandii and GlnK of E. coli are more similar (83% identity)      <br />                                                                     <br /> than are A. vinelandii GlnK and E. coli GlnB (75% identity), <br />                                                                     <br /> the T-loop region of A. vinelandii GlnK containing Y51 is more        <br /> characteristic of GlnB proteins (3). This may be why E. coli         <br /> GlnB and A. vinelandii GlnK apparently interact with NifL <br />                                                                                                     <br /> while E. coli GlnK does not (28). If GlnK is an essential protein <br />                                                                      <br /> in A. vinelandii, its rapid uridylylation in response to nitrogen        <br /> flux mediates protein activities and would rely less on changes         <br /> in the level of GlnK by transcriptional control. This is consis-        <br />                                                                     <br /> tent with the observation that neither glnA (encoding GS) nor         <br /> nifL gene expression is regulated by NH4&#226;&#171;&#185; in A. vinelandii (8,         <br /> 43). In contrast, K. pneumoniae harbors two PII-like proteins,     <br />                                                                       <br /> probably because they provide a selective advantage over a              <br /> single protein. In this organism, PII (GlnB), whose expression <br /> is not regulated by nitrogen status, may be specific for targets        <br /> involved in more subtle environmental tuning. GlnK, on the              <br /> other hand, may be specific to nitrogen fixation and/or severe         <br />                                                                      <br /> N starvation responses. In this scenario, it would seem suffi-         <br /> cient to express glnK in the absence of combined nitrogen to          <br /> activate NifA, which has been suggested for K. pneumoniae          <br />                                                                     <br /> (20, 23) and for E. coli (44).                                   <br />    Unmodified GlnK interacts with NifL in a yeast two-hybrid          <br />                                                                       <br /> assay. In this work it was proposed that A. vinelandii GlnK          <br /> might exert its effects directly on the NifL. While a GlnK         <br /> NifL-NifA interaction has been observed in vitro (28), it was        <br />                                                                   <br /> important to study the interaction of the proteins in vivo and to     <br /> determine which protein GlnK might target. In the yeast two- <br /> </body></html>