<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Gene loss rate: a probabilistic measure <br /> for the conservation of eukaryotic genes <br />                                                                          common measures of gene functional importance, including <br />                                                                                   its essentiality (1&#226;&#8364;&#8220;5) and the degree of its encoded protein <br /> The rate of conservation of a gene in evolution is                                in a protein&#226;&#8364;&#8220;protein interaction (PPI) network (3,6,7). These <br /> believed <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> be correlated with its biological impor-                              observations support the long-standing &#226;&#8364;&#732;<span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> rate&#226;&#8364;&#8482; <br /> tance. Recent studies have devised various conser-                                hypothesis of Wilson et al. (8), which claims that the greater <br /> vation measures for genes and have shown that they                                the effect of a <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>gene knockout</span> on fitness the slower is its <br /> are correlated with several biological characteristics                            evolutionary rate. The observed correlation between conser- <br /> of functional importance. Specifically, the state-of-                             vation and importance varies however between various con- <br /> the-art propensity for gene loss (PGL) measure was                                servation measures, underscoring the need for accurate and <br /> shown to be strongly correlated with gene essential-                              general measures for the rate of gene conservation. <br /> ity and its number of protein&#226;&#8364;&#8220;protein interactions                                   Nucleotide substitution rate is a traditional measure of the <br /> (PPIs). The observed correlation between conser-                                  conservation of a gene in evolution. Hirsh and Fraser (1) have <br />                                                                                   shown that the growth rate of a gene deletion mutant corre- <br /> vation and functional importance varies however <br />                                                                                   lates with the gene&#226;&#8364;&#8482;s evolutionary rate. Further studies, com- <br /> between conservation measures, underscoring the                                   puting more accurate evolutionary rate estimations, extended <br /> need for accurate and general measures for the rate                               these findings and showed marked differences between the <br /> of gene conservation. Here we develop a novel                                     evolutionary rates of essential and nonessential genes (2,5). <br /> maximum-likelihood approach to computing the                                         An alternative measure for evolutionary conservation, <br /> rate in which a gene is lost in evolution, motivated                              which measures the propensity for gene loss (PGL), was <br /> by the same principles as those underlying PGL.                                   introduced by Krylov et al. (3). PGL is computed based on <br /> However, in difference to PGL which considers only                                the <span id='am-323' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of presence and absence of genes across multiple <br /> the most parsimonious ancestral states of the                                     genomes, considering their phylogeny, and was shown to <br /> internal nodes of the phylogenetic tree relating the                              have higher correlations with gene dispensability than <br /> species, our approach weighs in a probabilistic                                   sequence evolution rate. Indeed, these two measures of evolu- <br />                                                                                   tionary conservation capture different characteristics of the <br /> manner all possible ancestral states, and includes <br />                                                                                   genes: Sequence evolution rate represents the selective con- <br /> the branch <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> information as part of the proba-                               straints on protein structure and sequence, whereas PGL cap- <br /> bilistic model. In application to data of 16 eukaryotic                           tures the essentiality of the gene&#226;&#8364;&#8482;s function. As pointed by <br /> genomes, our approach shows higher correlations                                   Krylov et al., a protein linked to an essential function could <br /> with experimental data than PGL, including data on                                potentially have a low propensity to be lost, but still evolve <br /> gene lethality, level of connectivity in a PPI network                            relatively fast due to relaxed functional constraints. However, <br /> and coherence within functionally related genes.                                  a moderate correlation between the two measures was found, <br />                                                                                   corroborating the intuitive notion that weakly constrained <br />                                                                                   proteins are lost during evolution significantly more often <br />                                                                                   than strongly constrained ones. <br /> INTRODUCTION                                                                         The propensity for gene loss is computed based on three <br /> Large scale sequencing projects are producing genome data at                      types of biological data sources, characterizing the gene <br /> an ever increasing pace. Interpreting the data to study gene                      and a group of species: (i) the phyletic pattern of a gene, <br /> importance and function is a major goal of functional                             which is the pattern of presence&#226;&#8364;&#8220;absence of the gene in the <br /> genomics. Recently, several studies have related different                        set of species genomes; (ii) a phylogenetic tree topology <br /> measures of evolutionary conservation of a gene to various                        relating the different species; and (iii) branch length estimates <br />  <br /> *To whom correspondence should be addressed. Tel: +972 3 640 5378; Fax: +972 3 640 9357; Email: borens@post.tau.ac.il <br /> *Correspondence may also be addressed to Tomer Shlomi. Tel: +972 3 640 5378; Fax: +972 3 640 9357; Email: shlomito@post.tau.ac.il <br />  <br /> The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors <br />  <br />   2006 The Author(s). <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  e7   Nucleic Acids Research, 2007, Vol. 35, No. 1                                                                              PAGE 2    OF   8 <br />  <br />  <br /> for the tree. (i) and (ii) are used for constructing the ancestral   that when considering only a single parsimonious ancestral <br /> phyletic pattern of the presence&#226;&#8364;&#8220;absence of the gene in inter-       phyletic pattern and restricting gene losses to branches of <br /> nal nodes of the tree. This reconstruction is based on the           equal length, GLR collapses to a simple measure that can <br /> Dollo parsimony principle (9), in which a gene loss is deemed        be analytically related to PGL. We compare the biological <br /> irreversible. Given the ancestral phyletic pattern, each branch      plausibility of GLR and PGL using an extensive data set of <br /> is treated as an independent trial where the gene was either         16 Eukaryotic species, showing that GLR better correlates <br /> preserved or lost. The PGL value of the gene is then defined         with existing measures of gene functional importance. <br /> as the ratio between the total length of branches in which the <br /> gene is lost and the total length of branches in which the gene <br /> could have been lost (i.e. preserved or lost). This <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>       METHODS <br /> captures the idea that the longer the time a gene could have         The GLR measure <br /> been lost but was not, the lower the propensity of this gene to <br /> be lost.                                                             We developed a novel measure, GLR, for the loss rate of a <br />    While intuitively reasonable, a careful examination of the        gene based on maximum likelihood principles. For a phyloge- <br /> <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of PGL reveals two potential weaknesses:                  netic tree T (with estimated branch lengths for the species <br />                                                                      under study) and a gene&#226;&#8364;&#8482;s phyletic pattern PP, we define <br />  (i) Assuming that a gene loss measure should reflect some           GLR as the maximum-likelihood estimate for the rate m of <br />      evolutionary loss rate, a gene which is lost only once          gene loss: <br />      during a long branch, may have a lower rate than that of a                                   X <br />      gene lost several times during short branches. However,         argmaxm L&#195;&#176;m j PP&#195;&#382; &#194;&#188; argmaxm      P&#195;&#176;PP&#226;&#8364;&#353;APP j m&#195;&#382;            1 <br />      PGL considers only the total lengths of branches in which                                        APP <br />      a gene is lost, ignoring the possible variation in lengths,     where APP runs over all possible ancestral phyletic patterns <br />      and may lead to counterintuitive predictions, as shown          for the gene. In the next sections we provide the details of <br />      below.                                                          the probabilistic model underlying GLR and the derivation <br /> (ii) PGL is based on a single ancestral phyletic pattern             of the GLR measure. We first consider the simple case of a <br />      obtained by a parsimony principle. However, the simple          fixed ancestral phyletic pattern and then present the general <br />      parsimony model may not describe the correct ancestral          case, considering all possible ancestral phyletic patterns. <br />      phyletic pattern and ignores alternative possible patterns. <br />      Moreover, the parsimonious construction of the ancestral <br />      phyletic pattern does not take into account the available       GLR under a fixed ancestral phyletic pattern <br />      branch lengths.                                                 Let T be a phylogenetic tree with branch lengths. Given both <br />                                                                      a phyletic pattern PP and an ancestral phyletic pattern APP of <br />    PGL additionally lacks an underlying systematic probabilis-       a gene, GLR is defined as argmaxm P&#195;&#176;PP&#226;&#8364;&#353; APP j m&#195;&#382;. Let <br /> tic model that properly describes the process of gene loss. In       c1 &#226;&#8364;&#353;. . . &#226;&#8364;&#353;cn1 denote the lengths of branches in which a gene <br /> contrast, a conservation measure based on such a model will          is conserved, and let l1 &#226;&#8364;&#353;. . . &#226;&#8364;&#353;ln2 denote the lengths of branches <br /> provide a natural way to incorporate additional sources of bio-      in which a gene is lost. Then <br /> logical data (e.g. confidence level of gene presence or <br /> absence) and could be easily extended to allow for prediction        P&#195;&#176;PP&#226;&#8364;&#353;APP j m&#195;&#382; &#194;&#188; Pni&#194;&#188;1 <br />                                                                                         1 <br />                                                                                            e mci Pni&#194;&#188;1 <br />                                                                                                     2 <br />                                                                                                        &#195;&#176;1   e mli &#195;&#382; <br /> of various pertaining parameters such as branch lengths. <br />    Here we present a novel approach for computing a con-             where e mt represents the probability that a gene is conserved <br /> servation measure for genes, the gene loss rate (GLR),               along a branch of length t, in accordance with the standard <br /> addressing the above shortcomings. The measure is based              model of nucleotide substitutions (15). <br /> on a simple probabilistic model of gene evolution, whose                To find the rate, m, that maximizes the above probability, <br /> underlying assumption is that each gene has a certain rate           we take the log of both sides of the equation: <br /> in which it is lost during evolution. Given a loss rate estima-                                     X <br />                                                                                                     n1           X <br />                                                                                                                  n2 <br /> tion, the probability of a gene phyletic pattern can be calcu-       log P&#195;&#176;PP&#226;&#8364;&#353;APP j m&#195;&#382; &#194;&#188;   m              ci &#195;&#190;         log&#195;&#176;1   e mli &#195;&#382; <br /> lated, considering all possible ancestral phyletic patterns.                                        i&#194;&#188;1          i&#194;&#188;1 <br /> GLR is defined as the maximum-likelihood estimate of this <br />                                                                      and get that the probability is maximized when: <br /> loss rate, i.e. the loss rate that maximizes the probability of <br /> the phyletic pattern associated with the gene. Applying              X <br />                                                                      n1            Xn2 <br />                                                                                          li e mli <br /> such a maximum-likelihood estimate, our model resembles                     ci &#194;&#188; <br />                                                                                        1   e mli <br /> that of (10&#226;&#8364;&#8220;13), who used a stochastic birth and death process        i&#194;&#188;1          i&#194;&#188;1 <br /> model to examine gene family evolution, but assumed a uni-           It is easy to show that the log likelihood function above is <br /> form birth and death rate for all genes. Recently, a related         concave; thus, the maximizing rate can be obtained via gradi- <br /> measure of gene loss was mentioned as part of an analysis            ent ascent. A naive analytical approximation to the rate m can <br /> covering numerous characteristics of gene function and               be obtained by assuming a uniform length l of all branches in <br /> evolution (14). However, as the above work did not focus             which the gene is lost. Under this assumption, we get that: <br /> on gene loss measures, no systematic comparison with previ- <br /> ous methods was conducted.                                           X <br />                                                                      n1 <br />                                                                                     n2 le ml <br />    In the following, we provide a rigorous <span id='am-321' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of GLR                ci &#194;&#188; <br />                                                                                    1   e ml <br /> and derive an algorithm to efficiently compute it. We show            i&#194;&#188;1 <br />  PAGE 3    OF   8                                                                    Nucleic Acids Research, 2007, Vol. 35, No. 1        e7 <br />  <br />  <br /> And thus, GLR, defined as the rate m which maximizes the                 while considering all possible ancestral phyletic patterns. <br /> probability function, is given in this simple case by:                   Given the phyletic pattern and loss rate, the probability that <br />                                                                          the gene was lost in the branch leading <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> node i to <br />           1                n2 l                                          node j is: <br /> m&#194;&#188;          log &#195;&#176;1          P1      &#195;&#382;                               2 <br />           l          n2 l &#195;&#190; ni&#194;&#188;1 ci <br />                                                                          P&#195;&#176;bi &#194;&#188;P1&#226;&#8364;&#353; bj &#194;&#188; 0 j PP&#226;&#8364;&#353; m&#195;&#382; <br /> where &#194;&#189;n2 l/&#195;&#176;n2 l &#195;&#190; Sni&#194;&#188;1 <br />                        1 <br />                           ci &#195;&#382;  is the PGL value.                                     P&#195;&#176;bi &#194;&#188; 1&#226;&#8364;&#353; bj &#194;&#188; 0&#226;&#8364;&#353; PP; APP j m&#195;&#382; <br />                                                                             &#194;&#188; APP <br />                                                                                              P&#195;&#176;PP j m&#195;&#382; <br /> General ancestral phyletic pattern <br /> In the general case of GLR computation, we again apply gra-              where both the numerator and denominator can be calculated <br /> dient ascent to find the rate m that maximizes the probability           as shown in the previous section, using the variant of <br /> of the phyletic pattern, P(PPjm), but now, weigh all possible            Felsenstein&#226;&#8364;&#8482;s algorithm. Specifically, the numerator is calcu- <br /> ancestral phyletic patterns (Equation 1). To this end, we apply          lated by modifying this algorithm to consider only ancestral <br /> a variant of Felsenstein&#226;&#8364;&#8482;s tiny maximum-likelihood algorithm             phyletic patterns that are consistent with a gene loss between <br /> (16). For simplicity, we describe it for a binary tree. Let i            nodes i and j, setting P&#195;&#176;T i j bi &#194;&#188; 0&#195;&#382; &#194;&#188; 0 and P&#195;&#176;T j j ji &#194;&#188; 1&#195;&#382; &#194;&#188; 0 <br /> denote a node in the phylogenetic tree, and let i1 and i2 denote         in the recursion. The expected number of gene losses for <br /> its two children. Let Ti denote the subtree rooted at node i.            a branch is the sum of probabilities over all gene phyletic <br /> Let p&#195;&#176;i&#226;&#8364;&#353; a&#195;&#382;&#226;&#8364;&#353;a 2 f0&#226;&#8364;&#353; 1g denote the probability that the gene is           patterns. <br /> lost (0) or conserved (1) along the li-long branch from the <br /> parent of i to i. As in the previous section, we assume that             Data acquisition and processing <br /> p&#195;&#176;i&#226;&#8364;&#353; 0&#195;&#382; &#194;&#188; 1   e mli and p&#195;&#176;i&#226;&#8364;&#353;1&#195;&#382; &#194;&#188; e mli . Let bi denote whether <br /> the gene is present (1) or absent (0) at i. For any given m,             A phylogenetic tree of the following 16 Eukaryotes was <br /> the probability of the subtree Ti is:                                    obtained from NCBI (17): Homo sapiens, Canis familiaris, <br />                                                                          Mus musculus, Rattus norvegicus, Drosophila melanogaster, <br /> P&#195;&#176;T i j bi &#194;&#188; 0&#195;&#382; &#194;&#188; P&#195;&#176;T i1 j bi1 &#194;&#188; 0&#195;&#382;P&#195;&#176;T i2 j bi2 &#194;&#188; 0&#195;&#382;                     Anopheles gambiae, Caenorhabditis elegans, Pan troglo- <br /> P&#195;&#176;TXi j bi &#194;&#188; 1&#195;&#382; &#194;&#188;                                                        dytes, Gallus gallus, Schizosaccharomyces pombe, Saccharo- <br />           P&#195;&#176;T i1 j bi1 &#194;&#188; x&#195;&#382;p&#195;&#176;i1 &#226;&#8364;&#353;x&#195;&#382;P&#195;&#176;T i2 j bi2 &#194;&#188; y&#195;&#382;p&#195;&#176;i2 &#226;&#8364;&#353;y&#195;&#382;             myces cerevisiae, Kluyveromyces lactis, Neurospora crassa, <br /> x&#226;&#8364;&#353; y2f0&#226;&#8364;&#353; 1g <br />                                                                          Arabidopsis thaliana, Oryza sativa and Magnaporthe grisea. <br />                                                                          The divergence time estimates were collected from multiple <br /> It should be noted that in this recursion, as in Felsenstein&#226;&#8364;&#8482;s           sources (3,18&#226;&#8364;&#8220;21). <br /> algorithm (16), the probabilities of the two subtrees, T i1 and             Clusters of homologous genes for these species were <br /> T i2 , are calculated given a certain presence-absence assign-           obtained from NCBI&#226;&#8364;&#8482;s HomoloGene database (17). Homolo- <br /> ment and are hence independent. Since the gene is always                 Gene is a system that automatically arranges genes into putative <br /> present in the root node of the tree, the probability of the phy-        homology clusters based on DNA and protein-based align- <br /> letic pattern, P&#195;&#176;PP j m&#195;&#382;, is given by P&#195;&#176;T root j broot &#194;&#188; 1&#195;&#382;. The         ment measures, identifying clear-cut paralogs and orthologs. <br /> computation of the derivative in this case is more involved                 The data on the lethality of gene knockouts in yeast was <br /> and is done using dynamic programming as follows:0 Denote                obtained from MIPS database (22). The PPI data along with <br /> by p0 (i,a) the0 derivative of p(i,a) according to m, i.e. p &#195;&#176;i&#226;&#8364;&#353;0&#195;&#382; &#194;&#188;     interactions reliability scores were obtained from (23,24). <br /> li e li m and p &#195;&#176;i&#226;&#8364;&#353; 1&#195;&#382; &#194;&#188;   li e li m . The derivative of the probabil-   These reliability scores are essential for controlling errors <br /> ity of the subtree Ti is:                                                that stem from noisy large-scale PPI experiments, and were <br />     0 <br />                          X       0 <br />                                                                          previously calculated based on the type of experiments in <br />  P &#195;&#176;T i j bi &#194;&#188; 1&#195;&#382; &#194;&#188;            &#194;&#189;P &#195;&#176;T i1 j bi1 &#194;&#188; x&#195;&#382;p&#195;&#176;i1 &#226;&#8364;&#353;x&#195;&#382;               which the interaction was observed, and the number of obser- <br />                     x&#226;&#8364;&#353; y2f0&#226;&#8364;&#353; 1g                                          vations in each experimental type. In total we assembled <br />                                          0 <br />                    &#195;&#190; P&#195;&#176;T i1 j bi1 &#194;&#188; x&#195;&#382;p &#195;&#176;i1 &#226;&#8364;&#353;x&#195;&#382;                          14 319 and 3926 interactions in yeast and worm, respectively. <br />                                                                             Data on protein complexes was obtained from the MIPS <br />                    P&#195;&#176;T i2 j bi2 &#194;&#188; y&#195;&#382;p&#195;&#176;i2 &#226;&#8364;&#353;y&#195;&#382;                             catalog (25), considering only manually curated complexes <br />                          0 <br />                    &#195;&#190; &#194;&#189;P &#195;&#176;T i2 j bi2 &#194;&#188; y&#195;&#382;p&#195;&#176;i2 &#226;&#8364;&#353;y&#195;&#382;                         (i.e. removing category 550). We used all leaves of the <br />                                          0                               MIPS complex hierarchy, collapsing nodes of level &gt; 3 to <br />                    &#195;&#190; P&#195;&#176;T i2 j bi2 &#194;&#188; y&#195;&#382;p &#195;&#176;i2 &#226;&#8364;&#353;y&#195;&#382;                          level 3. Overall, 35 complexes were compiled. Large scale <br />                     P&#195;&#176;T i1 j bi1 &#194;&#188; x&#195;&#382;p&#195;&#176;i1 &#226;&#8364;&#353;x&#195;&#382;&#226;&#8364;&#353;                           data on 21 phenotypic effects of single gene knockouts in <br />           0 <br />                                                                          yeast was obtained from Dudley et al. (26). <br /> where P &#195;&#176;T i j bi &#194;&#188; 0&#195;&#382; &#194;&#188; 0 for any node i, as the probability of <br /> a subtree that does not have the gene present in its root <br /> is0 independent of m. We initialize the recursion by setting <br /> P &#195;&#176;i j bi &#194;&#188; x&#195;&#382; &#194;&#188; 0 for all leaf nodes i and x 2 f0&#226;&#8364;&#353; 1g.                  RESULTS <br />                                                                          In the following we describe the results of applying the GLR <br />                                                                          measure to analyze a tree of 16 eukaryotic organisms, and a <br /> Estimating the expected number of gene losses per <br />                                                                          comprehensive comparison to the PGL measure. The perfor- <br /> branch <br />                                                                          mance evaluation was based on comparing the two gene loss <br /> Given a GLR value for a phyletic pattern, we compute the                 measures to an array of experimental data on yeast and worm, <br /> probability that the gene is lost in each branch of the tree,            including data on gene lethality and level of connectivity in a <br />  e7    Nucleic Acids Research, 2007, Vol. 35, No. 1                                                                                               PAGE 4    OF   8 <br />  <br />  <br />                              (a)                                                   (b) <br />  <br />  <br />  <br />  <br /> Figure 1. A simple phylogenetic tree, demonstrating the difference between the GLR and PGL measures. Presence or absence of a gene is indicated by 1 or 0, <br /> respectively, next to the corresponding node (based on a parsimonious reconstruction of the ancestral phyletic pattern). Dotted lines represent branches in which <br /> the gene was lost. (a) The gene is lost in one branch of length l. (b) The gene is lost in two branches, each of length l/2. <br />  <br />  <br /> protein interaction network. To exemplify the difference <br /> between the two measures, we start with a simple comparison <br /> of GLR and PGL on synthetic data. <br />  <br /> An example tree <br /> To demonstrate the difference between PGL and GLR, we <br /> first analyze a synthetic tree shown in Figure 1. The tree con- <br /> sists of five species, marked A to E, and two divergence <br /> points. The first divergence point, the tree root, occurred <br /> l years ago, and the second occurred l/2 years ago. We con- <br /> sider two phyletic patterns of genes: (i) a gene is present in all <br /> species other than D (Figure 1a); and (ii) a gene is present in <br /> all species other than B and C (Figure 1b). We reconstruct the <br /> corresponding most parsimonious ancestral phyletic patterns <br /> for both cases, finding that in phyletic pattern 1, the gene is <br /> lost in a branch of length l, and in phyletic pattern 2, the <br /> gene is lost in two branches of length l/2. Given these recon- <br /> structions, it would be more reasonable to assign a higher loss <br /> rate score to the second phyletic pattern, as it reflects two <br /> gene losses during short time periods, compared to a single <br /> loss in a longer time period for the first phyletic pattern. <br />    Computing the GLR and PGL scores for both cases reveals <br /> that the PGL scores for both phyletic patterns are the same, <br /> while GLR correctly captures the above intuitive trend. Spe- <br /> cifically, since PGL is defined as the ratio between the total <br /> length of the branches in which a gene is lost and the total <br /> length of branches in which the gene could have been lost, <br /> we get the same PGL value of 1/4 in both cases. In contrast, <br /> the GLR score for the second phyletic pattern, calculated <br /> based on Equation 2 (see Methods; a fixed ancestral phyletic <br /> pattern is assumed), is m &#194;&#188;   2=l&#195;&#176;log&#195;&#176;1   1=4&#195;&#382;&#195;&#382; which is <br /> twice the size of the GLR for the first phyletic pattern. <br />    Having shown that GLR correctly captures variability in <br /> branch lengths in which a gene is lost, while PGL may fail <br /> to do so, we turn to compare the two measures using real phy- <br /> logenetic data and various biological measures of functional                       Figure 2. The phylogenetic tree used in our analysis, relating 16 eukaryotes. <br /> importance.                                                                        Estimated divergence times (in millions of years ago) are shown for all <br />                                                                                    internal nodes. The number in parentheses next to each branch indicates the <br />                                                                                    expected number of gene losses (see Methods). <br /> A phylogenetic tree of 16 eukaryotes <br /> We obtained clusters of homologous genes in 16 Eukaryotic <br /> species from NCBI&#226;&#8364;&#8482;s HomoloGene database (17). The                                  were present in that cluster. The corresponding phylogenetic <br /> Eukaryotic species include nine animals, five fungi, and                           tree was taken from NCBI and the divergence time esti- <br /> two plants (Methods). The phyletic pattern associated with                         mates were collected from multiple sources (Figure 2 and <br /> each cluster was determined by the species whose genes                             Methods). We focused on a subset of the gene clusters in <br />  PAGE 5   OF   8                                                              Nucleic Acids Research, 2007, Vol. 35, No. 1       e7 <br />  <br />  <br /> which a <span id='am-318' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> was present in either S.cerevisiae        GLR, PGL and sequence evolution rate <br /> or C.elegans, for which we had additional biological data to <br />                                                                   Next, we examined the correlation between GLR and PGL <br /> validate the conservation measures. Furthermore, as similarly <br />                                                                   as well as their correlations with a common sequence evo- <br /> done by Krylov et al. (3) on their data, we considered only <br />                                                                   lution rate (SER) measure obtained from (4). We found a <br /> the clusters for which the gene was present in the common <br />                                                                   statistically significant correlation of 0.833 (P &lt; 10 6 , Spear- <br /> ancestor of the Fungi/Metazoa group and plants under the <br />                                                                   man correlation test) between GLR and PGL. Considering <br /> optimal parsimony reconstruction. These genes are assumed <br />                                                                   that both measures are based on the same data and attempt <br /> to be present in the root of the phylogenetic tree we study <br />                                                                   to capture a similar notion of loss rate, this strong correlation <br /> and, hence, form a natural set for gene loss analysis. Further- <br />                                                                   is not surprising. Interestingly, the significant difference <br /> more, focusing on simple gene loss dynamics without consid- <br />                                                                   between the accuracy of these two measures, demonstrated <br /> ering genes duplication, we restrict the analysis to genes <br />                                                                   below, stems from this seemingly low disagreement. <br /> whose family size is one, i.e. genes that have only one <br />                                                                      PGL was previously shown to be significantly correlated <br /> copy in the genome (see Discussion). <br />                                                                   with SER. The PGL values computed here also show a sig- <br />    Given a phylogenetic tree, both the GLR and PGL values <br />                                                                   nificant correlation with SER (r &#194;&#188; 0.54, P &#194;&#188; 7:9 &#194;&#183; 10 4 , <br /> of a gene are uniquely determined according to the gene phy- <br />                                                                   Spearman correlation test), higher than the corresponding <br /> letic pattern and, hence, cannot distinguish between gene <br />                                                                   correlation of GLR with SER (r &#194;&#188; 0.475, P &#194;&#188; 2:1 &#194;&#183; 10 2 ). <br /> clusters with identical phyletic patterns. Therefore, we <br />                                                                   The correlation between GLR measures and SER is interest- <br /> group genes clusters according to their phyletic patterns; for <br />                                                                   ing because evolutionary rate reflects selective constraints on <br /> each such group and a biological property of interest, we <br />                                                                   protein structure and function whereas GLR captures gene <br /> associate with the group the average value of that property <br />                                                                   dispensability. However, homology detection by sequence <br /> over the group&#226;&#8364;&#8482;s genes. To obtain robust biological data of <br />                                                                   comparison is affected by evolutionary rate, and homology <br /> these phyletic patterns, we included in our analysis only pat- <br />                                                                   relations for fast evolving genes may remain uncovered. As <br /> terns that were associated with at least five gene clusters. <br />                                                                   a consequence of the latter, one would expect that the mea- <br />    Clearly, the number of different possible values (for either <br />                                                                   sured GLR would be larger for fast evolving genes. In <br /> PGL or GLR) is bounded by the number of possible phyletic <br />                                                                   other words, the observed correlation is at least in part due <br /> patterns. PGL was previously calculated based on phyletic <br />                                                                   to the intrinsic properties of the measurement procedures. <br /> patterns consisting of six species (in fact, the phylogenetic <br /> tree consisted of seven species, but all phyletic patterns <br /> included the species Arabidopsis thaliana), markedly limiting     Gene loss correlation with lethality of yeast knockouts <br /> the number of possible PGL values. The extended phyloge- <br />                                                                   Genes that have low loss rate are assumed to be associated <br /> netic tree used here allows for a significantly higher resolu- <br />                                                                   with an essential function for <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span> and, thus, their knock- <br /> tion of gene loss rate values. <br />                                                                   out is expected to have lethal effects. Such a trend was previ- <br />                                                                   ously shown for PGL, where the fraction of essential genes <br /> The accuracy and robustness of GLR estimations                    was found to be significantly higher for genes with a PGL <br /> To estimate the accuracy of the GLR estimations we per-           value of zero, compared to genes with higher PGL values <br /> formed a series of simulations of gene loss dynamics. In          (3). Other studies have also shown similar correlations <br /> each simulation, we chose a &#226;&#8364;&#732;true&#226;&#8364;&#8482; loss rate value, and used      between SER and lethality (2,5). <br /> it to randomly generate a phyletic pattern according to the          The correlation between the GLR measure and the phyletic <br /> gene evolution dynamics assumed in our model (Supplemen-          pattern lethality was  0.655 (P &lt; 10 7 , Spearman correla- <br /> tary Data 1). We computed the GLR value based on the              tion test). The correlation for PGL and lethality was smaller <br /> derived phyletic pattern and compared it with the true loss       (r &#194;&#188;  0.605, P &lt; 10 6 ). The correlation for SER and lethality <br /> rate used in the simulation. We found a highly significant cor-   in these settings was only  0.457 (P &lt; 10 2 ). As the advan- <br /> relation (r &#194;&#188; 0:802&#226;&#8364;&#353;P &lt; 10 300 , Spearman correlation test;       tages of GLR are expected to be revealed when the gene is <br /> Supplementary Figure 1a) between the true and estimated           lost in several branches, we next restricted the analysis to <br /> loss rates. The correlation between PGL and the true loss         phyletic patterns for which the most parsimonious reconstruc- <br /> rates was markedly lower (r &#194;&#188; 0:659&#226;&#8364;&#353; P &lt; 10 300 ).                tion of the ancestral phyletic pattern included at least two <br />    We further examined the robustness of the GLR estimates        branches in which the gene was lost. Using this restricted <br /> to modifications of the phylogenetic tree and to phyletic pat-    set of patterns, the correlation between GLR and lethality <br /> tern perturbations (Supplementary Data 2). We found that          was indeed markedly higher and more significant (r &#194;&#188; <br /> systematically deleting each species in turn from the tree         0.880, P &lt; 10 13 ) than that of PGL (r &#194;&#188;  0.709, <br /> has a relatively minor effect on the estimated GLR values         P &lt; 10 6 ). The correlation of SER with lethality did not <br /> (Supplementary Figure 2a). However, using a substantially         markedly change in this setting (r &#194;&#188;  0.449, P &#194;&#188; 0.03). In <br /> smaller tree from (3), which contains only seven species,         the rest of the analysis presented in this paper we continue <br /> we got significantly less accurate estimations (Supplementary     to focus on this subset of phyletic patterns. <br /> Data 2). The effect of noise in phyletic patterns was examined       To further evaluate the performance of the two measures, <br /> by changing the presence&#226;&#8364;&#8220;absence values for each species at       we tested their prediction power in classifying the set of <br /> a time. Following such data modifications, the GLR values         yeast genes to lethal and nonlethal. Figure 3 shows the <br /> obtained were highly correlated with the original estima-         receiver operating characteristic (ROC) curves obtained for <br /> tions, and significantly more robust than the PGL estimates       each of the measures. Evidently, GLR outperforms PGL in <br /> (Supplementary Figure 2b).                                        this comparison. <br />  e7   Nucleic Acids Research, 2007, Vol. 35, No. 1                                                                                       PAGE 6   OF   8 <br />  <br />  <br />                                                                                Table 1. Correlation between the GLR or PGL measure and the connectivity <br />                                                                                level in a PPI network <br />  <br />                                                                                                   Yeast                             Worm <br />  <br />                                                                                GLR                 0.429 (P &#194;&#188; 0.004)                 0.524 (P &#194;&#188; 10 4) <br />                                                                                PGL                 0.316 (P &#194;&#188; 0.04)                 not significant <br />  <br />  <br />  <br />                                                                                tend to be coherently conserved (28). Motivated by this <br />                                                                                observation, we examined whether genes that are part of <br />                                                                                the same complex have coherent GLR (and PGL) values. <br />                                                                                   To this end, we compiled a list of 135 proteins from <br />                                                                                59 manually curated complexes (Methods). For each com- <br />                                                                                plex, we calculated the standard deviations of the GLR and <br />                                                                                PGL values of the constituent complex proteins as a measure <br />                                                                                of coherency. To avoid bias stemming from different distribu- <br />                                                                                tions of GLR and PGL values, we first normalized both mea- <br />                                                                                sures to have mean 0 and SD 1. Comparing the coherence of <br /> Figure 3. ROC curves for GLR and PGL illustrating the increased specificity <br /> and sensitivity of GLR in predicting the lethality of genes. The areas under   PGL and GLR for all these pairs, we found that genes from <br /> the ROC curves are 0.78 and 0.72 for GLR and PGL, respectively.                the same complex tend to have a smaller GLR standard <br />                                                                                deviation than PGL (P &lt; 0.00027, Wilcoxon signed-rank <br />                                                                                test for paired data). <br />    Finally, we examined whether genes that are hardly distin-                     Similarly, proteins whose knockout has a certain pheno- <br /> guishable by one of the measures can be correctly classified                   typic effect are expected to be coherently conserved through <br /> by the other measure. To this end, we sampled pairs of phy-                    evolution. To test whether similarity in phenotypic effect is <br /> letic patterns for which the PGL scores of the two phyletic                    manifested in coherency of GLR values, we compiled a list <br /> patterns were similar (i.e. the difference between the two                     of 21 phenotypic effects for 312 single gene knockouts in <br /> values falls within the low 10% of differences between all                     yeast (26). Comparing the coherency of PGL and GLR values <br /> possible pairs), and computed the differences in GLR for                       of proteins associated with the same <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>phenotype</span>, as described <br /> those patterns. We found a significant correlation (r &#194;&#188;                        above, we found higher coherency for GLR than PGL <br />  0.615, P &lt; 10 9 , Spearman correlation test) between these                    (P &lt; 0.0034, Wilcoxon signed-rank test for paired data). <br /> GLR differences and the corresponding differences of lethal-                   This provides further support for the superiority of GLR <br /> ity values. Conversely, when testing pairs with similar GLR                    over PGL. <br /> values, no significant correlation could be detected between <br /> the differences in PGL values and the differences in lethality                 Expected number of gene losses per branch <br /> values.                                                                        We computed the expected number of gene losses in each <br />                                                                                branch of the phylogenetic tree based on the optimal GLR <br /> Gene loss correlation with protein                                             values (see Methods and Figure 2). Expectedly, we identified <br /> interaction information                                                        a significant correlation of 0.43 (P &lt; 0.01, Spearman correla- <br /> The degree of a protein in a PPI network was shown to be                       tion test) between the expected numbers of gene losses and <br /> correlated with its dispensability (6,7). Indeed, Krylov et al.                branch lengths. A few branches in the tree exhibited rela- <br /> (3) report a significant correlation between the PGL value                     tively high gene loss values. Notably, an expected massive <br /> of a gene and the degree of its corresponding protein in the                   loss of 603 genes was identified in the relatively short branch <br /> yeast PPI network.                                                             leading from the common ancestor of fungi and metazoa to <br />    To overcome the noise in protein interaction data (27) we                   fungi. As this loss is identified along a branch leading to a <br /> assigned reliability estimates to reported interactions and                    relatively large subtree, it is based on the robust evidence <br /> defined the level of connectivity of a protein as the expected                 concerning the absence of genes from multiple genomes <br /> number of interactions involving this protein (Methods).                       (five, in this case). Other putative massive losses were <br /> Table 1 summarizes the correlations obtained between the                       identified between O.sativa and its common ancestor with <br /> GLR or PGL measure of a gene and the connectivity level                        A.thaliana (946 genes), and between P.troglodytes and its <br /> of its corresponding protein in the PPI networks of yeast                      common ancestor with H.sapiens (599 genes). The latter, <br /> and worm. As evident from the table, the correlation of                        however, were supported by only a single genome and require <br /> GLR with the experimental network data is significantly                        further investigation (see Discussion). <br /> higher. Notably, the correlation between SER and connectiv- <br /> ity level was not statistically significant. <br />                                                                                DISCUSSION <br /> Gene loss coherence within functionally related genes <br />                                                                                We have provided a novel maximum-likelihood measure, <br /> Proteins that are part of the same complex (and, hence, share                  GLR, for the rate of gene loss. GLR is based on a probabilis- <br /> a functional role) are likely to have similar functional impor-                tic model that takes into account the phylogenetic tree of the <br /> tance. Specifically, it has been shown that protein complexes                  species under study, its branch lengths, and a phyletic pattern <br />  PAGE 7   OF   8                                                               Nucleic Acids Research, 2007, Vol. 35, No. 1       e7 <br />  <br />  <br /> representing the presence and absence of the gene in those          weaken the signal of importance, although this is handled <br /> species. The measure is shown to be highly correlated with          to some extent by taking into account the reliabilities of the <br /> experimental data on gene lethality and connectivity levels,        interactions. <br /> and more aligned with biological data compared to the                  Although the underlying motivation of GLR is similar <br /> state-of-the-art PGL.                                               to that of PGL, and the two measures are largely in <br />    Previously, more complex probabilistic models of gene            agreement (as exemplified by the high correlation between <br /> content evolution that account for events of gene loss,             them), GLR was shown to provide higher correlations with <br /> duplication and transfer were proposed (10&#226;&#8364;&#8220;13). The estima-         biological measures of gene functional importance and an <br /> tion of multiple parameters in such complex models requires         improved lethal/nonlethal classification (see Figure 3). We <br /> extensive phylogenetic data and cannot be performed indi-           believe that the demonstrated superiority of GLR stems <br /> vidually per gene based solely on its content data. To obtain       from its enhanced probabilistic model, better capturing the <br /> reliable rate estimations, Gu et al. (11) assumed uniform           gene loss process. <br /> duplication and loss parameters for all genes under all tree           Furthermore, as GLR is based on a probabilistic model it is <br /> branches, and hence used the entire ensemble of gene content        extensible to utilizing additional data sources in the rate com- <br /> data to estimate these two parameters. The work of Hahn             putation. Specifically, additional data on the confidence level <br /> et al. (12) assumed uniformity of duplication and loss              of gene presence or absence in homology clusters can be eas- <br /> parameters across all genes but allowed variability within          ily incorporated into the model, addressing noise problems in <br /> branches. In the recent work of Csuros and Miklos (10),             the sequence data. Such data can be obtained for example <br /> duplication, loss and transfer rates were estimated for large       from the significance scores of BLAST sequence alignment, <br /> groups of genes that have similar evolutionary dynamics.            which is commonly used for gene homology detection (33). <br /> However, while the above works focus on global parameters           Additional data on the confidence level of branch length <br /> characterizing gene evolution, the focus of this work is the        estimations can also be considered. We believe that such con- <br /> variability in evolutionary dynamics between different              fidence measures can further improve the accuracy of GLR, <br /> genes. Estimating evolutionary parameters associated with a         as branch length estimations are obtained through various <br /> single gene based solely on its observed phyletic pattern           computational and experimental methods and are known to <br /> is a challenging optimization task. Considering a single            be noisy (21). <br /> phyletic pattern at a time prevents the usage of a complex,            The GLR model presented here assumes (as does PGL) <br /> multi-parameter model. Consequently, we employ a single-            that the gene was present in the root of the tree and may <br /> parameter model that characterizes simple gene loss                 have been lost multiple times in different branches. To this <br /> dynamics [see also (29,30)] and apply it to those cases that        end, we have focused on genes that are likely to be present <br /> can be plausibly described by such dynamics. Specifically,          in the root of the phylogenetic tree (i.e. present in both the <br /> we focus on Eukaryotic species in which horizontal gene             Fungi/Metazoa group and plants, similarly to PGL). In gen- <br /> transfer is unlikely, and on genes whose genomic copy num-          eral, for genes whose lowest common ancestor does not lie <br /> ber is one. Our analysis suggests that the model accurately         at the root of the tree, the uncertainty about its location can <br /> predicts gene-specific loss rates for these genes. Moreover,        be incorporated into the probabilistic model underlying GLR. <br /> resorting to a simple model of gene loss evolution, our                In our application, GLR was computed based on all possi- <br /> maximum-likelihood algorithm allows for an optimal loss             ble ancestral phyletic patterns, representing the presence and <br /> rate estimation. Notably, the works mentioned above do not          absence of genes in internal nodes of the tree. Alternatively, <br /> guarantee the identification of optimal parameters.                 GLR can be computed along with a single, most likely ances- <br />    Evaluating the correctness of a gene loss measure by cor-        tral phyletic pattern by employing the ancestral maximum- <br /> relating it with various estimates of gene importance is a          likelihood approach (34). This can be accomplished by <br /> challenging task, owing to the inherent difficulty of defining      modifying the gradient ascent search to find the loss rate <br /> and measuring gene importance. Gene essentiality is a com-          that maximizes the probability of optimal ancestral phyletic <br /> mon and natural measure of importance, and as we have               pattern using the ancestral maximum-likelihood algorithm. <br /> found, it has indeed a markedly high correlation with the cal-         GLR and its underlying probabilistic model can also be <br /> culated loss rates in yeast. However, one cannot expect             generalized to support different applications. For example, <br /> GLR to be an ideal predictor of gene lethality as gene dis-         GLR may be utilized to improve current estimations of <br /> pensability is only one out of many factors determining             branch lengths or to estimate branch specific loss rates. <br /> gene evolution dynamics (31). Other measures, previously            This could be done by simultaneously searching for branch <br /> used to estimate gene importance, are based on various char-        specific parameters and GLR values that maximize the proba- <br /> acteristics of the gene functionality. Specifically, the involve-   bility of a given set of phyletic patterns (for several genes). <br /> ment of a protein in various cellular processes is assumed to       Another application, providing a maximum-likelihood esti- <br /> reflect its functional importance, motivating a heuristic mea-      mation of the expected number of gene losses per branch <br /> sure based on its degree of connectivity in a PPI network.          (including internal branches) while considering all possible <br /> Membership in the same protein complex also indicates a             ancestral phyletic patterns was described above. <br /> shared functional role, and, hence, a similar functional impor-        Determining the importance of genes and, specifically, its <br /> tance measure. Such characteristics, however, are only              effect on genome evolution is a fundamental problem in biol- <br /> indirect measures of importance. This is also evident by the        ogy. As genomic data continues to accumulate we expect <br /> weaker correlation we have found between gene loss and              GLR to capture the biological significance of genes more <br /> protein interaction information (Table 1). In the case of PPI       and more accurately, facilitating an improved analysis of <br /> networks, the noisy nature of the data (24,32) may further          gene functionality and evolution. <br /> </body></html>