<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> A high-accuracy consensus map yeast protein complexes reveals <br /> modular nature gene essentiality <br /> G Traver Hart, Insuk Lee Edward R Marcotte* <br />  <br /> Address: Center Systems Synthetic Biology Institute Cellular Molecular Biology University Texas Austin 2500 Speedway, MBB <br /> 3.210 Austin, Texas, 78712, USA <br /> Email: G Traver Hart - traver_hart@yahoo.com; Insuk Lee - insuklee2001@yahoo.com; Edward R Marcotte* - marcotte@icmb.utexas.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 2 July 2007                                                         Received: 22 December 2006 <br />                                                                                Accepted: 2 July 2007 <br /> BMC Bioinformatics 2007, 8:236   doi:10.1186/1471-2105-8-236 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2105/8/236 <br /> &#194;&#169; 2007 Hart et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Identifying protein complexes organism major goal systems biology. <br />                  In past 18 months, results genome-scale tandem affinity purification-mass <br />                  spectrometry (TAP-MS) assays yeast published, corresponding complex <br />                  maps. For complexes, published data sets surprisingly uncorrelated. It therefore <br />                  useful consider raw data study generate accurate complex map a <br />                  high-confidence data set integrates results earlier assays. <br />                  Results: Using unsupervised probabilistic scoring scheme, assigned confidence score <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                  interaction matrix-model interpretation large-scale yeast mass-spectrometry <br />                  data sets. The scoring metric proved accurate filtering schemes used original <br />                  data sets. We took high-confidence subset interactions derived set of <br />                  complexes using MCL. The complexes high correlation existing annotations. <br />                  Hierarchical organization protein complexes evident inter-complex interactions. <br />                  Conclusion: We demonstrate scoring method generate integrated high- <br />                  confidence subset observed matrix-model interactions, subsequently used derive <br />                  accurate map yeast complexes. Our results indicate essentiality product the <br />                  protein complex individual protein, achieved near saturation of <br />                  yeast high-abundance, rich-media-expressed &quot;complex-ome.&quot; <br />  <br />  <br />  <br />  <br /> Background                                                                     putational methods provided 50,000 putative <br /> The molecular machines carry basic cellular proc-                     yeast protein-protein interactions date, sub- <br /> esses typically individual proteins protein                        stantial fraction spurious[1,2]. An array of <br /> complexes. Even relatively simple model organism                        analytical methods aimed generating high-quality com- <br /> Saccharomyces cerevisiae, machines process                       plexes data applied, including both <br /> store biological information fact large protein com-                    unsupervised [3-5] trained [6,7] techniques. Other <br /> plexes comprised subunits.                                             genomic proteomic data sets, gene expression, <br />                                                                                <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>knockout</span> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>phenotype</span>, subcellular localization, genetic <br /> The path measuring protein interactions defining                       interaction profiles, phylogenetic profiles [5,6,8-10], <br /> complexes studied. Experimental com-                         integrated raw interaction data in <br />  <br />  <br />                                                                                                                                        Page 1 12 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                               http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br /> an effort broaden deepen ability accurately        Results <br /> define protein complexes.                                        In large-scale interaction assay, consider pro- <br />                                                                  tein&apos;s interactions random sample popu- <br /> Two recent genome-scale tandem affinity purification/            lation observed interactions. A simple general <br /> mass spectrometry (TAP-MS) experiments perfomed               theoretical error model, based hypergeometric dis- <br /> Gavin et al. [11] Krogan et al. [12], produced       tribution, used calculate probability of <br /> enormous new data, allowing complete            observing interaction random background. <br /> analysis universe yeast protein complexes. How-        This model builds related models previously <br /> ever, complex maps published independently            applied linkage interaction types [18- <br /> two groups surprising lack correlation,          21]. Within given dataset, probability (P-value) an <br /> can partially explained different analytical      interaction proteins A B observed at <br /> methods applied generating raw data [1,13].            random is: <br />  <br /> TAP-MS data typically consist tagged &quot;bait&quot; protein                                                min(n,m) <br /> and associated &quot;prey&quot; proteins co-purify          p(# interactions &#226;&#8240;&#165; k | n, m, N) =     &#226;&#710;&#8216;        p(i | n, m, N) <br /> bait. Interaction data sets generated raw data                                              =k <br /> using spoke method, considers bait-prey         where <br /> interactions, matrix method, includes all <br /> prey-prey interactions given bait pull-down [14]. <br />                                                                                                  &#226;&#381;&#8250; n &#226;&#381;&#382;&#226;&#381;&#8250; N &#226;&#710;&#8217; n &#226;&#381;&#382; <br /> As affinity purification process generally isolates stable                                   &#226;&#381;&#339; &#226;&#381;&#376;&#226;&#381;&#339;         &#226;&#381;&#376; <br /> complexes, clear-cut way differentiate                                            &#226;&#381;? &#226;&#381;&#160;&#226;&#381;? m &#226;&#710;&#8217; &#226;&#381;&#160; <br />                                                                                 p(i | n, m, N) = <br /> between direct physical interactions indirect interac-                                            &#226;&#381;&#8250;N&#226;&#381;&#382; <br /> tions mediated members complex &#226;&#8364;&#8220; or,                                              &#226;&#381;&#339; &#226;&#381;&#376; <br />                                                                                                       &#226;&#381;?m&#226;&#381;&#160; <br /> that matter, proteins appear simply result of <br /> experimental noise. Thus, spoke model contains          k = number times interaction A <br /> direct physical interactions sampling indirect      B observed, n m total number inter- <br /> interactions complex, plus               actions proteins A B, N total number of <br /> noise, matrix model captures larger             interactions observed entire data set. When applied <br /> number true indirect interactions price             matrix model interpretation protein interactions, <br /> decreased accuracy linking spurious protein        scoring scheme identify highly accurate subsets of <br /> every &quot;real&quot; one, linking proteins hetero-       interactions. The process illustrated Figure 1. <br /> geneous complexes contain bait. While <br /> some efforts use filtered subset          We generated matrix-model interpretations Ho, <br /> matrix-model interactions improve accuracy [9,15,16],         Gavin, Krogan datasets. The TAP-MS data <br /> analysis mass spectrometry interaction data typi-         set significant scale [22] subset [11] was <br /> cally carried using spoke model [3,5].              omitted. We applied scoring method each, <br />                                                                  applying interaction dataset P-value calcu- <br /> Here offer simple robust statistical scoring            lated observations set. We eval- <br /> scheme assigning confidence observed interactions.        uated quality scoring calculating recall and <br /> The scheme based comparing observed versus                 precision versus set protein complexes manually <br /> expected numbers interactions matrix model          defined literature sources Munich Informa- <br /> protein-protein interactions, provides greatly               tion center Protein Sequences (MIPS) [23]. Recall was <br /> increased recall and/or precision standard spoke        scored TP/(TP + FN), TP, true positives, exper- <br /> model interpretation. A advantage       imental interactions MIPS set FN, false <br /> that used integrate data sets different        negatives, MIPS interactions present the <br /> sources. We use scoring scheme combine Gavin          experimental data. Precision defined TP/(TP + FP), <br /> et al. [11], Krogan et al. [12], Ho et al. [17] co-complex   TP FP, false positives, interac- <br /> data sets define high-quality subset comprised          tions observed experimentally corresponding <br /> 1689 proteins 390 complexes. We             proteins MIPS set, interaction not. For <br /> essential proteins strongly cluster together, supporting       data sets, method displays improved recall <br /> complex-centric gene-centric basis essenti-      and/or precision relative spoke model <br /> ality large fraction essential genes.                   interpretation dataset, group&apos;s <br />                                                                  published complexes (Figure 2). As co-complex data <br />                                                                  set represents independent experimental observation, <br />                                                                  probabilities combined provide higher con- <br />  <br />  <br />                                                                                                                       Page 2 12 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                  http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />  <br />   A                                              B                                            C <br />  Krogan (+10) <br />  <br />                                                                                                                            P=20.82 <br />       Pri1                Pri2                     Pri1                  Pri2                   Pri1                                    Pri2 <br />                                                                                                             (2 P= <br />                                                                                                               0. 2 <br />                                                                                                                 8 2 7. <br />                                                                                                                    + 24 <br />  <br />  <br />  <br />  <br />                                                                                                   P=20.82 <br />  <br />  <br />  <br />  <br />                                                                                                                                          P=20.82 <br />                                                                                                                     6. <br />                                                                                                                        42 <br />                                                                                                                           ) <br />                                                                                                                       .   82 <br />                                                                                                                    20 <br />                                                                                                               P= <br />                                                                                                                        P=43.49 <br />    Pol12                  Pol1                    Pol12                  Pol1                  Pol12                (20.82 + 22.57)     Pol1 <br />   Krogan (+8)           Krogan (+20) <br />   Gavin (+22)            Gavin (+0) <br />  <br /> Applying <br /> Figure 1the matrix-model scoring algorithm <br /> Applying matrix-model scoring algorithm. The subunits DNA primase core complex detected using <br /> the scoring algorithm. (A) In Gavin et al. TAP-MS data set, Pol1 Pol12 purified bait corresponding <br /> bait-prey, spoke model interactions shown blue (plus number additional prey identified shown parentheses). In the <br /> Krogan et al. assay (shown orange), baits plus Pri1 purified. (B) In matrix model, bait-prey prey- <br /> prey interactions considered. Within given dataset, total number links observed pair proteins is <br /> recorded P-value calculated described text. The PICO network generated multiplying P-values the <br /> same interaction derived different data sets, e.g. Pol1&#226;&#8364;&#8220;Pol12 discovered Gavin Krogan scored accord- <br /> ingly. (C) The PICO network integrates probability scores data sources, represented -ln(P-value). Values in <br /> black final PICO scores; separate scores Gavin et al. (blue) Krogan et al. (orange) shown applicable. No <br /> data Ho et al. relevant example. <br />  <br />  <br /> fidence repeated observations. We combined           accuracy set complexes, measured the <br /> the scored data sets multiplying P-values        Hubert statistic, H, derived complexes versus ref- <br /> a given interaction datasets, applying P-      erence set complexes [26]. Briefly, calculating H <br /> value 1 interaction missing dataset.         involves generating matrix M protein pairs (i, j) where <br /> The combined interaction dataset, Prob-         M(i, j) = 1 proteins complex 0 <br /> abilistic Integrated Co-complex (PICO) network,           otherwise. The correlation experimental and <br /> accurate provides greater coverage            reference matrices measured, resulting score <br /> individual datasets comprises.                                 -1 1, 1 implying identical complex assign- <br />                                                                   ments values near zero indicating random assign- <br /> The PICO network contains large number (~160,000)            ment. We measured Hubert statistic complexes <br /> protein-protein interactions, relative confi-         measured threshold set curated <br />                                                                   MIPS complexes [23] ribosomal subunits removed <br /> dence measure described P-value. The list is <br />                                                                   filtered set Gene Ontology (GO) Cellular <br /> available download [see Additional File 1]. We filtered <br />                                                                   Component (CC) annotations (see Methods). The corre- <br /> out low-confidence interactions deriving com-              lations generally improve increasing stringency (Fig- <br /> plexes data, beginning rank-ordering              ure 3B), rate increase correlation with <br /> interactions P-value, lowest highest. We               GO component drops sharply 10-2 cutoff. This <br /> applied series increasingly stringent expected (E)           improvement accuracy comes price decreasing <br />                                 n                                 coverage, reflected decreasing number interac- <br /> value thresholds, E = &#226;&#710;&#8216; Pi , starting E = 1        tions threshold shown Figure 3A. In an <br />                                =1                               attempt balance accuracy coverage, selected the <br /> tightening order magnitude increments E = 10-6.          complexes derived E = 10-2 threshold, hereafter <br /> The number interactions PICO network            called E-2 complexes, study. <br /> threshold shown Figure 3A. <br />                                                                   Features E-2 complexes <br /> We derived set complexes threshold using          The E-2 complexes contain 1689 proteins grouped into <br /> MCL [24], implementation Markov clustering                390 clusters sizes ranging 35 subunits. A <br /> algorithm. MCL evaluated [25] used              network view complexes, generated using Cyto- <br /> derive complexes raw data [12]. To evaluate           scape [27], shown Figure 4; Cytoscape file avail- <br />  <br />  <br />                                                                                                                                      Page 3 12 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                 http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Performance    curves probabilistic scoring method <br /> Performance curves probabilistic scoring method. We measured performance various datasets <br /> against reference set consisting matrix-model interaction set generated MIPS curated complexes, excluding the <br /> large small ribosomal subunits (which account half interactions set). Single points <br /> represent entire dataset. Curves represent dataset scored using hypergeometric scoring algorithm, rank <br /> ordered, plotted symbol representing cumulative addition 500 highest scoring interactions (i.e. <br /> tail curve represents entire dataset). The scoring scheme outperforms raw data filtered, published <br /> sets cases; integrated PICO net outperforms individual scored data sets, derived complexes slightly <br /> more accurate PICO (for thresholds; data shown). <br />  <br />  <br /> able download [see Additional File 2]. To measure         accurate. Of 132 complexes subu- <br /> accuracy individual complexes, tested sig-         nits, 69% (91) highly enriched spe- <br /> nificant enrichment GO component annotation. GO                cific GO component annotations; 44 complexes of <br /> component annotations enriched P &lt;0.01 (with Bonfer-           size larger, 84% (37) annotated. Further- <br /> roni correction multiple hypothesis testing) noted        more, virtually uncharacterized genes these <br /> for complex [see Additional File 3]. The Simpson             large complexes, appear relatively <br /> coefficient enriched annotation listed      weak connections members respective <br /> easily understood metric measuring completeness           clusters. This suggests yeast community has <br /> with GO term describes complex (or vice               achieved fairly complete description large fraction <br /> versa).                                                           &quot;complex-ome,&quot; complexes containing <br />                                                                   proteins. In fact, complex size or <br /> The large fraction E-2 complexes correspond            greater consists entirely unnamed subunits thus <br /> existing annotations suggest data set highly          considered truly novel (complex C132, com- <br />  <br />  <br />                                                                                                                        Page 4 12 <br />                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                               http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />                                                                  includes RNA polymerase I, II, III, largely all <br />                                                                  enzymes contain Rpb5, Rpb8, Rpb10, and <br />                                                                  Rpo26 subunits. Likewise, complex C7 contains the <br />                                                                  TAFIID complex SAGA transcription factor/chro- <br />                                                                  matin remodeling complex; complexes share the <br />                                                                  Taf5, 6, 9, 10, 12 proteins. It clear the <br />                                                                  RNA polymerase case E-2 clusters occasionally <br />                                                                  contain discrete complexes presumably phys- <br />                                                                  ically interact. <br />  <br />                                                                  Even clusters lack significant GO terms tend to <br />                                                                  subunits share similar free-text descriptions in <br />                                                                  Saccharomyces Genome Database (SGD) [28]. For <br />                                                                  example, complex C44 contains proteins, of <br />                                                                  essential. Of these, seven explicitly <br />                                                                  described SGD involved 60 S ribosome bio- <br />                                                                  genesis components 66 S pre-ribosomal particles, <br />                                                                  eighth involved export pre-ribosomal large <br />                                                                  subunits nucleus. No GO term enrichment is <br />                                                                  CC annotation typically &quot;nucleolus,&quot; <br />                                                                  weak term excluded analysis (see Methods). <br />                                                                  Likewise, unannotated complexes C20, C30, C78 <br />                                                                  contain 13, 10, 5 proteins, respectively (10, 9, 5 <br />                                                                  essential), known suspected involved <br />                                                                  ribosome biogenesis. Other unannotated complexes <br />                                                                  include C43, largely nonessential proteins the <br />                                                                  well-described cyclin/cyclin-dependent kinase group; <br />                                                                  C51, seven nonessential proteins involved catabolite <br />                                                                  inactivation FBPase; C72, proteins (five essen- <br /> Effect <br /> accuracy <br /> Figure of3thresholds network size derived complex         tial), involved retrograde Golgi-to-ER <br /> Effect thresholds network size derived                 trafficking sixth, Sec39, unknown function <br /> complex accuracy. (A) Interactions PICO network           &quot;proposed involved protein secretion.&quot; <br /> were rank ordered, E-value calculated the <br /> sum P-values. The number interactions E-value <br />                                                                  Hierarchical structure co-complex network <br /> threshold counted; total decreases increasingly <br /> stringent threshold applied. (B) At E-value threshold,   The high-confidence subset PICO network from <br /> the subset interactions clustered MCL           E-2 complexes derived contains 5,352 <br /> parameters optimized correlation filtered set   interactions; these, 4,411 present E-2 com- <br /> GO component annotations [see Methods]. The correlation          plex map 390 complexes. The remaining 941 interac- <br /> with GO component (filled circles) MIPS complexes            tions occur subunits different complexes. <br /> (hollow circles) generally improves stringency   We examined structure interactions col- <br /> E-value cutoff. We judged 10-2 cutoff provides rea-   lapsing complex single node looking at <br /> sonable tradeoff increasing accuracy decreasing      interactions complexes. The resulting inter- <br /> coverage, chose subset study.               complex network, depicted Figure 5, suggests hierar- <br />                                                                  chical organization protein complexes cell. Over <br />                                                                  one-third interactions (341, 36%) appear just <br />                                                                  clusters: U4/U6 &#195;&#8212; U5 tri-snRNP complex its <br />                                                                  neighbors (191 interactions), C20/C30/C44/C78 <br /> posed proteins YAL049C, YDL025C, YGR016W,                 ribosome biogenesis nexus (86 interactions), C17 <br /> YHR009C).                                                        histone-associated complex (64 interactions). In three <br />                                                                  cases, intercomplex interactions link complexes that <br /> Several E-2 clusters represent amalgamations known            involved closely related physiological processes. <br /> complexes. The MCL algorithm assigns protein             Taken together, observations suggest yeast pro- <br /> exactly complex, protein complexes shared            teins complexes exhibit hierarchical organization, with <br /> subunits combined single              complexes interacting well-ordered <br /> cluster E-2 complexes. The C1 cluster, example,       fashion. <br />  <br />  <br />                                                                                                                     Page 5 12 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                 http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />  <br /> Figure <br /> A  subset4of E-2 complex map <br /> A subset E-2 complex map. After applying E = 10-2 threshold PICO interaction set, subset 5,352 <br /> interactions clustered MCL, using parameters maximized correlation filtered set GO component anno- <br /> tations. Interactions clusters (4,411) plotted Cytoscape using included &quot;organic&quot; layout algorithm. Interac- <br /> tions clusters (941) omitted clarity. Yellow nodes indicate essential proteins; red, nonessential. For full <br /> image Additional File 4. <br />  <br />  <br /> Essentiality protein complexes                                 tical set complexes, repeating process 10,000 times, <br /> The E-2 network shows enrichment essential genes            calculating mean bin. We took the <br /> in general: 1689 proteins network comprise             log ratio observed random frequencies <br /> 29% yeast proteins, contain 58% essential       bin. The results, plotted Figure 6, clear <br /> proteins (602 essentials 1033 total). The descrip-         enrichment complexes essential (&gt;70%) <br /> tions above, glance complex map Fig-       completely nonessential (&lt;10%), under- <br /> ure 4, suggests concentration essential proteins          representation intermediate values. <br /> some complexes, exclusion (see Addi- <br /> tional file 4). To measure concen-        Discussion <br /> tration, considered distribution complexes         Modular nature essentiality <br /> respect fraction essential proteins         The concentration essential proteins complexes <br /> sorted distribution uniformly spaced bins.          suggests essentiality is, cases, product of <br /> We bootstrapped background distribution randomly             complex function individual protein function. <br /> assigning number essential genes iden-          This phenomenon observed Barabasi <br />  <br />  <br />                                                                                                                        Page 6 12 <br />                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                                                        http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />  <br />                            2 <br />  <br />  <br />                                                                                                                                                                      4 <br />                                                     9 <br />                                                                                 9 <br />                                        6 <br />                   5                                                                                                                            6 <br />                            2                                                        8 <br />                                                                   10                                              8 <br />                                    2                                            4 <br />                                                                                                           6           4 <br />  <br />                                                                                                                           9 <br />  <br />  <br />  <br />  <br />                                                                                                                                        4 <br />  <br />  <br />                                                     10 <br />                                6 <br />  <br />  <br />  <br />                                                                        8 <br />  <br />                                            10 <br />  <br />  <br />  <br />                                                          10 <br />  <br />  <br />                                                                            10 <br />                                                                                                      14                        4 <br />  <br />  <br />                                                                                                                           6        4 <br />  <br />  <br />  <br />                                                               7 <br />                                8 <br />                                                 8 <br />  <br />                                                                                                                   4 <br />                                                                                                                           30 <br />  <br />                                            9                                                                                               6 <br />  <br />  <br />  <br />  <br />                       42 <br />                                        4 <br />                                                                                             6 <br />  <br />                                                                                                               4 <br />  <br />  <br />  <br />  <br />                                                                                                 8 <br />  <br />  <br />                                                                                         4 <br />                                                                                                                                                    4 <br />  <br />  <br />                                                                                         8 <br />  <br />  <br />  <br />  <br />                                                                                                     29 <br />                                                                                             9 <br />  <br />                                                                                                                                                                          4 <br />  <br />  <br />  <br />  <br /> Figure 5 <br /> Inter-complex   interactions <br /> Inter-complex interactions. Interactions E-2 complex map represent 4,411 5,352 interactions PICO <br /> network E = 10-2 threshold. The 941 remaining protein-protein interactions (PPI) collapse 248 complex-complex <br /> interactions. Here map 128 inter-complex interactions, comprising protein-protein interactions (821 PPI <br /> total); singletons omitted clarity. Nodes represent E-2 complexes: yellow indicates &gt;70% essential subunits; labels indi- <br /> cate highest-scoring GO component, applicable. Edge thickness reflects number interactions complex subu- <br /> nits, ranging (thinnest) 24 (thickest) PPI; number interactions shown edge. Density PPI <br /> between complexes similar function (e.g. 190 PPI U4/U6/U5 tri-snRNP complex neighbors; 86 PPI C20/ <br /> C30/C44/C78 ribosome biogenesis modules; 64 PPI linking C17 histone-associated complex neighbors; shaded blue) illus- <br /> trates hierarchical nature yeast complex network. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      Page 7 12 <br />                                                                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />                                                                   essentiality include C2, corresponds 26 S <br />                                                    *              proteasome complex. The complex 58% essential but <br />                                                                   diagram cluster reveals number of <br />                                                                   loosely connected proteins annotated pro- <br />                                                                   teasomal. The 24 core subunits diagram 71% <br />                                                                   essential. Also, previously described C7 complex is <br />                                                                   comprised nonessential SAGA complex the <br />              * <br />                                                         *         essential TAFIID complex (Figure 4). <br />  <br />                                                                   Comparison Collins et al <br />                                                                   After submission article, study Collins et al. <br />                                                                   published Gavin Krogan TAP-MS <br />                            *    * <br />                       *                                           data sets re-analyzed [29]. Using supervised algo- <br />                                                                   rithm derived Bayesian methods optimized with <br />                                                                   empirically-derived parameters, study posited over <br />                                                                   9,000 high-confidence interactions labeling many <br />                                                                   previously published interactions lower confi- <br /> Essential <br /> Figure 6proteins concentrated subset complexes        dence. Comparing PICO network E = 10-2 <br /> Essential proteins concentrated subset                threshold (E-2; 5,352 interactions) Collins results <br /> complexes. The distribution essential proteins com-         shows overlap 4,356 interactions (Figure 7). The <br /> plexes compared randomized background. The frac-         interactions unique Collins data set are <br /> tion essential proteins complex calculated,        highly enriched ribosomal proteins: 4,714 inter- <br /> sorted equal-sized bins, compared expected         actions Collins PICO, 2,964 involve <br /> background generated randomly assigning essential pro-         ribosomal proteins. As proteins commonly co- <br /> teins set complexes. The log ratio observed <br />                                                                   purified tagged baits TAP purifications (and sub- <br /> to expected frequency bin plotted here: positive <br /> values indicate observed frequency random; negatives        sequently identified mass spectrometry), inter- <br /> indicate random. The distribution illustrates con-      preted promiscuous interactors matrix model of <br /> centration essential proteins complexes,         protein connectivity, considers bait-prey as <br /> corresponding absence essentials others. Bars marked        prey-prey interactions given purification. Such <br /> with asterisk represent statistically significant deviations   high-degree interactors penalized hypergeo- <br /> from random expectation (P &lt;10-3).                                metric scoring model; therefore, interac- <br />                                                                   tions scored model, virtually exceed the <br />                                                                   stringent score threshold applied. <br />  <br />                                                                   Further comparison shows hypergeometric scor- <br /> group [5] analysis Ho Gavin 2002[22]. In             ing method Collins method yielded data sets of <br /> using raw data assays, prior study             nearly equal accuracy. We rank-ordered sets of <br /> assigns bait pull-down discrete complex             interactions respective scores, divided into <br /> does correct sampling complex mul-          bins 500 interactions, plotted cumulative <br /> tiple baits. Thus, example, purifications derived        recall precision versus MIPS co-complex inter- <br /> TAP-tagged Nsp1, Nup60, Nup82, Nup116                 actions [see Additional File 5]. The Collins data set <br /> considered discrete complexes high fraction          achieves greater coverage PICO network, at <br /> of essential proteins, reality factors     somewhat lower overall accuracy, performance is <br /> constituents nuclear pore complex.                    calculated entire MIPS reference. The differ- <br />                                                                   ence entirely inclusion ribosomal <br /> The current analysis provides accurate defini-          protein interactions Collins: ribosome is <br /> tion complexes and, owing breadth raw            removed MIPS reference set, networks pro- <br /> data, greater coverage yeast proteins. The correspond-         vide nearly identical recall (~34%) precision (~81%). <br /> ing signal essentiality complexes             That networks generated methods overlap <br /> strong. In E2 complex set, 64 complexes        strongly, despite inclusion Ho dataset and <br /> &gt;70% essential subunits, containing 330 essential          use higher confidence threshold Krogan <br /> 379 total proteins &#226;&#8364;&#8220; accounting 32% essential          raw data, suggests networks capture highly accurate <br /> genes yeast. Of complexes, 35 largest contain        subset yeast co-complex interactions, sim- <br /> 271 essential proteins (of 320 total), 26% essen-       ple probabilistic method offered study effec- <br /> tial genes (Table 1). Other complexes strong <br />  <br />  <br />                                                                                                                      Page 8 12 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                                        http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br /> Table 1: Essential Complexes. Selected essential complexes E-2 complex set. Complexes listed composed 4 <br /> subunits, &gt;70% essential. For complex, table lists E-2 complex identifier, size complex, fraction <br /> of essential proteins, significant GO cellular component annotation complex, list proteins complex. <br /> Twenty-six percent essential genes yeast represented complexes <br />  <br />  Complex ID   Size   % Essential   Most significantly enriched GO CC term                Complex members <br />  <br />  <br />  C1           35     74%           DNA-directed RNA polymerase III complex               DST1, IWR1, RET1, RPA12, RPA135, RPA14, RPA190, RPA34, RPA43, RPA49, <br />                                                                                          RPB10, RPB11, RPB2, RPB3, RPB4, RPB5, RPB7, RPB8, RPB9, RPC11, RPC17, <br />                                                                                          RPC19, RPC25, RPC31, RPC34, RPC37, RPC40, RPC53, RPC82, RPO21, <br />                                                                                          RPO26, RPO31, SPT4, TFG1, TFG2 <br />  C4           27     93%           small nucleolar ribonucleoprotein complex             BMS1, DIP2, ECM16, EMG1, IMP3, MPP10, NAN1, NOC4, NOP14, POL5, <br />                                                                                          PWP2, SOF1, UTP10, UTP13, UTP14, UTP15, UTP18, UTP20, UTP21, UTP30, <br />                                                                                          UTP4, UTP5, UTP6, UTP7, UTP8, UTP9, YGR210C <br />  C11          20     75%           mRNA cleavage polyadenylation specificity         BUD14, CFT1, CFT2, FIP1, GIP3, GLC7, GLC8, MPE1, PAP1, PFS2, PTA1, PTI1, <br />                                    factor complex                                        REF2, SDS22, SSU72, SWD2, SYC1, YPI1, YSH1, YTH1 <br />  C12          20     85%           U4/U6 &#195;&#8212; U5 tri-snRNP complex                          AAR2, BRR2, DIB1, LEA1, LSM8, PRP11, PRP21, PRP3, PRP31, PRP38, PRP4, <br />                                                                                          PRP6, PRP8, PRP9, RSE1, SMX2, SNU114, SNU23, SNU66, SPP381 <br />  C13          18     72%           proteasome core complex, alpha-subunit                FLC2, GRH1, OSM1, PRE1, PRE10, PRE2, PRE3, PRE4, PRE5, PRE6, PRE7, PRE8, <br />                                    complex (sensu Eukaryota)                             PRE9, PUP1, PUP2, PUP3, RED1, SCL1 <br />  C14          18     72%           snRNP U1                                              BRR1, LUC7, MUD1, NAM8, PRP39, PRP40, PRP42, SMB1, SMD1, SMD2, <br />                                                                                          SMD3, SME1, SMX3, SNP1, SNU56, SNU71, STO1, YHC1 <br />  C20          13     77%           (no significant annotation)                           BRX1, CIC1, DRS1, ERB1, FPR4, HAS1, MAK5, NOC2, NOC3, PUF6, PWP1, <br />                                                                                          RRP5, YTM1 <br />  C26          11     73%           eukaryotic translation initiation factor 2B           CDC123, GCD1, GCD11, GCD2, GCD6, GCD7, GCN3, PET111, SUI2, SUI3, <br />                                    complex                                               YVH1 <br />  C30          10     90%           (no significant annotation)                           EBP2, MRT4, NOG1, NOP15, NOP2, NOP7, NUG1, RLP7, RPF2, TIF6 <br />  C38          8      88%           nuclear pore                                          GLE2, NIC96, NSP1, NUP116, NUP159, NUP49, NUP57, NUP82 <br />  C41          8      88%           DASH complex                                          ASK1, DAD1, DAD2, DAD3, DAM1, DUO1, SPC19, SPC34 <br />  C42          8      100%          exocyst                                               EXO70, EXO84, SEC10, SEC15, SEC3, SEC5, SEC6, SEC8 <br />  C44          8      100%          (no significant annotation)                           DBP10, NIP7, NSA1, RIX7, RPF1, RRP1, SPB1, SPB4 <br />  C46          7      86%           Arp2/3 protein complex                                ARC15, ARC18, ARC19, ARC35, ARC40, ARP2, ARP3 <br />  C48          7      71%           DNA replication factor C complex                      CTF18, ELG1, RFC1, RFC2, RFC3, RFC4, RFC5 <br />  C53          7      100%          transcription factor TFIIH complex                    CCL1, KIN28, RAD3, SSL1, TFB1, TFB3, TFB4 <br />  C54          7      86%           signal recognition particle (sensu Eukaryota)         LHP1, SEC65, SRP14, SRP21, SRP54, SRP68, SRP72 <br />  C55          7      100%          nucleolar ribonuclease P complex                      POP1, POP3, POP4, POP5, POP7, POP8, RPP1 <br />  C65          6      100%          nuclear origin replication recognition complex     ORC1, ORC2, ORC3, ORC4, ORC5, ORC6 <br />  C67          6      100%          transcription factor TFIIIC complex                   TFC1, TFC3, TFC4, TFC6, TFC7, TFC8 <br />  C72          6      83%           (no significant annotation)                           DSL1, SEC22, SEC39, TIP20, UFE1, USE1 <br />  C74          6      100%          chaperonin-containing T-complex                       CCT2, CCT3, CCT4, CCT5, CCT6, TCP1 <br />  C78          5      100%          (no significant annotation)                           IPI1, IPI3, RIX1, RSA4, SDA1 <br />  C79          5      100%          nuclear cohesin complex                               CDC5, IRR1, MCD1, SMC1, SMC3 <br />  C85          5      80%           GINS complex                                          CTF4, PSF1, PSF2, PSF3, SLD5 <br />  C86          5      100%          nuclear condensin complex                             BRN1, SMC2, SMC4, YCG1, YCS4 <br />  C89          5      80%           nucleolar preribosome, small subunit precursor        ENP1, HRR25, LTV1, RIO2, TSR1 <br />  C101         4      100%          MIND complex                                          DSN1, MTW1, NNF1, NSL1 <br />  C106         4      100%          alpha DNA polymerase:primase complex                  POL1, POL12, PRI1, PRI2 <br />  C110         4      75%           (no significant annotation)                           CIA1, MET18, NAR1, YHR122W <br />  C111         4      75%           (no significant annotation)                           NAB3, NAB6, NRD1, SEN1 <br />  C115         4      100%          mRNA cleavage factor complex                          CLP1, PCF11, RNA14, RNA15 <br />  C124         4      75%           transcription factor TFIIE complex                    DBP2, PPN1, TFA1, TFA2 <br />  C92          4      75%           outer plaque spindle pole body                     SPC72, SPC97, SPC98, TUB4 <br />  C93          4      100%          Ndc80 complex                                         NUF2, SPC24, SPC25, TID3 <br />  <br />  <br />  <br />  <br /> tive tool assigning relative confidence rankings                                scored false positives (provided proteins present <br /> observations large-scale data sets.                                                 reference set). We observe cases this. For <br />                                                                                        example, Tub4 gamma tubulin complex composed <br /> It worth noting highest-scoring interac-                              Spc97, Spc98, Tub4, defined GO Cellular <br /> tions analyses reach 100% precision ver-                             Component MIPS annotation. The E2 derived com- <br /> sus MIPS reference. This                                     plex includes Spc72, spindle pole body compo- <br /> incompleteness reference set. An interaction                                 nent interacts Tub4p complex [30]. The <br /> defined false positive corre-                             MIPS reference does include Spc72 gamma <br /> sponding proteins present reference set                             tubulin complex does include protein the <br /> interaction not. Thus, true interactions detected                          &quot;Spindle Pole Body Components&quot; collection proteins. <br /> experimentally absent reference set                               Thus interactions Spc72 members of <br />  <br />  <br />                                                                                                                                                   Page 9 12 <br />                                                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                               http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br />                                                                  plexes essential nonessential. The are <br />       PICO-E2:                                     Collins:      mixed &#226;&#8364;&#8220; example, SAGA/TAFIID complex &#226;&#8364;&#8220; lead to <br />       5,352                                          9,070       interesting questions essentiality specific <br />                                                                  interactions [31]. We anticipate complex descrip- <br />                                                                  tions offered here, general scoring method, <br />                                            4,714                 used functional genomics systems <br />                                                                  biology studies. <br />                                           Incl. 2,964 PPI <br />        996             4,356             involving ribo-         Methods <br />                                          somal proteins <br />                                                                  Data sources <br />                                                                  Data Ho et al. taken Table S1 [17]. <br />                                                                  Interactions Gavin et al. taken Supple- <br />                                                                  mentary Table 1 [11]. In cases, bait-prey pairs were <br />                                                                  generated list purifications, bait <br />                                                                  removed prey list applicable. Interactions from <br />                                                                  Krogan et al. [12] taken raw LCMS and <br /> Figure 7 <br />                                                                  MALDI purification data. Bait-prey pairs LCMS puri- <br />                                                                  fications confidence &gt; = 99.6 MALDI purifica- <br />                                                                  tions score &gt; = 3.4 included. Matrix-model data <br />                                                                  sets generated considering prey-prey pairs if <br /> the gamma tubulin complex, certainly &quot;true&quot;         prey purified bait. <br /> co-complex interactions, scored false positives <br /> when calculating precision versus MIPS. All experi-         Reference data sets <br /> mentally detected inter-complex interactions absent       MIPS filtered data: The MIPS curated complex data were <br /> the MIPS reference set. Thus incompleteness ref-      downloaded mpact[23]. All high-throughput data, <br /> erence set prevents high-accuracy experimental data set        large small ribosomal subunits, were <br /> from achieving perfect precision.                                excluded. An all-by-all set interactions generated <br />                                                                  complex used reference calculate <br /> Conclusion                                                       recall/precision curves experimental data. The co-com- <br /> We described simple robust unsupervised               plex data used calculate Hubert statistic. <br /> method assigning confidence levels interactions <br /> observed large-scale assay, combining data       GO filtered reference set: The complete yeast GO Cellular <br /> from independent assays integrated            Component ontology downloaded Saccha- <br /> can used study. We used method inte-      romyces Genome Database [28] 5 December 2006. <br /> grate data large-scale affinity purification-mass     Annotations sorted number genes which <br /> spectrometry assays yeast generate high-confidence       applied; annotations equal larger the <br /> subset interactions, derived accurate        size &quot;small cytoplasmic ribosomal subunit&quot; were <br /> set protein complexes. The recall MIPS co-complex          discarded. The resulting set annotations com- <br /> interactions indicates 46% total        plexes, small number discrete cellular localiza- <br /> co-complex interactome yeast assayed TAP-         tions included. This annotation set used calculate <br /> MS methods (with 34% high confidence E2              GO term enrichment Hubert statistic. <br /> set). Nonetheless, high proportion complexes that <br /> correspond existing annotations small number          Analaytic techniques <br /> of uncharacterized genes present high-confidence          The MCL program downloaded [24]. For E- <br /> data strongly suggest community largely satu-       value threshold PICO network, MCL run with <br /> rated fraction complex-ome accessible      following parameter space: -I, 1.8 3.0 0.2 incre- <br /> the methods (TAP-MS) conditions (aerobic growth           ments; -C, 0.5 1.5 0.25 increments; -S, 0 7. The <br /> rich media) explored far. Therefore,        Hubert statistic (H) calculated MCL result <br /> would likely fruitful explore conditions         GO filtered reference set MCL result <br /> growth states extend interactome.                         highest H score considered optimal result <br />                                                                  E-value. The -S parameter no <br /> Our complex data support notion that,           effect results. <br /> cases, essentiality tied protein gene itself, <br /> but molecular machine protein               Calculation Hubert statistic, H, performed as <br /> belongs. We clearly separate majority com-            described [26]. As matrices equal size, the <br />  <br />  <br />                                                                                                                    Page 10 12 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                                  http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br /> calculation performed potential interaction <br /> space defined set proteins present                          Additional File 4 <br /> experimental reference protein sets.                                          A subset E-2 complex map <br />                                                                                   A subset E-2 complex map. After applying E = 10-2 threshold <br />                                                                                   PICO interaction set, subset 5,352 interactions clustered <br /> The Simpson coefficient, Cs similarity sets                         MCL, using parameters maximized correlation filtered <br /> proteins A B, is:                                                             set GO component annotations. Interactions clusters (4,411) <br />                                                                                   plotted Cytoscape using included &quot;organic&quot; layout algo- <br /> Cs = (# proteins A B)/min(# proteins A, # pro-                          rithm. Interactions clusters (941) omitted clarity. Yellow <br />                        teins B)                                                nodes indicate essential proteins; red, nonessential. <br />                                                                                   Click file <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> The list essential ORFS downloaded Sac-                           2105-8-236-S4.pdf] <br /> charomyces Genome Database. We considered veri- <br /> fied uncharacterized ORFs.                                                     Additional File 5 <br />                                                                                   cumulative recall precision versus MIPS co-complex <br /> Authors&apos; contributions                                                            interactions <br /> IL developed probabilistic scoring method. TH con- <br /> ceived study, performed data collection                            Click file <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> analysis, drafted manuscript guidance                           2105-8-236-S5.pdf] <br /> and supervision EMM. All authors read approved the <br /> final manuscript. <br />  <br /> Additional material                                                          Acknowledgements <br />                                                                              This work supported grants N.S.F., N.I.H., Welch Founda- <br />                                                                              tion (F1515), Packard Fellowship (E.M.M). <br />  Additional File 1 <br />  This table gives co-complex interactions combined PICO network <br />                                                                              References <br />                                                                              1.      Hart GT, Ramani AK, Marcotte EM: How complete current <br />  associated scores, given -ln(p-value).                                 yeast human protein-interaction networks? Genome Biol <br />  Click file                                                                 2006, 7(11):120. <br />  [http://www.biomedcentral.com/content/supplementary/1471-                   2.      Reguly T, Breitkreutz A, Boucher L, Breitkreutz BJ, Hon GC, Myers <br />  2105-8-236-S1.htm]                                                                  CL, Parsons A, Friesen H, Oughtred R, Tong A, Stark C, Ho Y, Bot- <br />                                                                                      stein D, Andrews B, Boone C, Troyanskya OG, Ideker T, Dolinski K, <br />                                                                                      Batada NN, Tyers M: Comprehensive curation analysis of <br />  Additional File 2                                                                   global interaction networks Saccharomyces cerevisiae. J Biol <br />  The Cytoscape file                                                                  2006, 5(4):11. <br />                                                                              3.      Maciag K, Altschuler SJ, Slack MD, Krogan NJ, Emili A, Greenblatt JF, <br />  Click file                                                                 Maniatis T, Wu LF: Systems-level analyses identify extensive <br />                                                                                      coupling gene expression machines. Mol Syst Biol 2006, <br />  [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                      2:2006 0003. <br />  2105-8-236-S2.zip]                                                          4.      Krause R, von Mering C, Bork P: A comprehensive set protein <br />                                                                                      complexes yeast: mining large scale protein-protein inter- <br />  Additional File 3                                                                   action screens. Bioinformatics 2003, 19(15):1901-1908. <br />                                                                              5.      Dezso Z, Oltvai ZN, Barabasi AL: Bioinformatics analysis of <br />  E-3 Complexes                                                                       experimentally determined protein complexes yeast <br />  Each complex shown single line containing following informa-            Saccharomyces cerevisiae. Genome Res 2003, 13(11):2450-2454. <br />  tion: Complex ID, Size, %Essential, List Proteins(| - deliminated). If   6.      Zhang LV, Wong SL, King OD, Roth FP: Predicting co-complexed <br />  constituent proteins enriched GO Cellular Component anno-             protein pairs using genomic proteomic data integration. <br />  tation (see Methods) subsequent lines contain following informa-           BMC Bioinformatics 2004, 5:38. <br />  tion: Enrichment score(-ln(p)), Simpson coefficient, Annotation name/       7.      Asthana S, King OD, Gibbons FD, Roth FP: Predicting protein <br />                                                                                      complex membership using probabilistic network reliability. <br />  GO_ID/(Number proteins <span id='am-160' about='Thesaurus:GO_Annotation' typeof='owl:Thing'>GO annotation</span>)                               Genome Res 2004, 14(6):1170-1175. <br />  Click file                                                         8.      Jansen R, Greenbaum D, Gerstein M: Relating whole-genome <br />  [http://www.biomedcentral.com/content/supplementary/1471-                           expression data protein-protein interactions. Genome <br />  2105-8-236-S3.htm]                                                                  Res 2002, 12(1):37-46. <br />                                                                              9.      Lichtenberg U, Jensen LJ, Brunak S, Bork P: Dynamic complex <br />                                                                                      formation yeast cell cycle.                Science 2005, <br />                                                                                      307(5710):724-727. <br />                                                                              10.     von Mering C, Krause R, Snel B, Cornell M, Oliver SG, Fields S, Bork <br />                                                                                      P: Comparative assessment large-scale data sets pro- <br />                                                                                      tein-protein interactions. Nature 2002, 417(6887):399-403. <br />                                                                              11.     Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau <br />                                                                                      C, Jensen LJ, Bastuck S, Dumpelfeld B, Edelmann A, Heurtier MA, <br />                                                                                      Hoffman V, Hoefert C, Klein K, Hudak M, Michon AM, Schelder M, <br />                                                                                      Schirle M, Remor M, Rudi T, Hooper S, Bauer A, Bouwmeester T, <br />                                                                                      Casari G, Drewes G, Neubauer G, Rick JM, Kuster B, Bork P, Russell <br />                                                                                      RB, Superti-Furga G: Proteome survey reveals modularity of <br />                                                                                      yeast cell machinery. Nature 2006, 440(7084):631-636. <br />  <br />  <br />                                                                                                                                             Page 11 12 <br />                                                                                                                       (page number citation purposes) <br />  BMC Bioinformatics 2007, 8:236                                                                 http://www.biomedcentral.com/1471-2105/8/236 <br />  <br />  <br />  <br /> 12.   Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu            Botstein D, Cherry JM: &quot;Saccharomyces Genome Data- <br />       S, Datta N, Tikuisis AP, Punna T, Peregrin-Alvarez JM, Shales M,              <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span>&quot;. . <br />       Zhang X, Davey M, Robinson MD, Paccanaro A, Bray JE, Sheung A,          29.   Collins SR, Kemmeren P, Zhao XC, Greenblatt JF, Spencer F, Hol- <br />       Beattie B, Richards DP, Canadien V, Lalev A, Mena F, Wong P, Star-            stege FC, Weissman JS, Krogan NJ: Toward Comprehensive <br />       ostine A, Canete MM, Vlasblom J, Wu S, Orsi C, Collins SR, Chandran           Atlas Physical Interactome Saccharomyces cerevi- <br />       S, Haw R, Rilstone JJ, Gandi K, Thompson NJ, Musso G, St Onge P,              siae. Mol Cell Proteomics 2007, 6(3):439-450. <br />       Ghanny S, Lam MH, Butland G, Altaf-Ul AM, Kanaya S, Shilatifard A,      30.   Knop M, Schiebel E: Receptors determine cellular localiza- <br />       O&apos;Shea E, Weissman JS, Ingles CJ, Hughes TR, Parkinson J, Gerstein            tion gamma-tubulin complex site of <br />       M, Wodak SJ, Emili A, Greenblatt JF: Global landscape protein              microtubule formation. Embo J 1998, 17(14):3952-3967. <br />       complexes yeast Saccharomyces cerevisiae. Nature 2006,           31.   He X, Zhang J: Why hubs tend essential protein net- <br />       440(7084):637-643.                                                            works? PLoS Genet 2006, 2(6):e88. <br /> 13.   Goll J, Uetz P: The elusive yeast interactome. Genome Biol 2006, <br />       7(6):223. <br /> 14.   Bader GD, Hogue CW: Analyzing yeast protein-protein inter- <br />       action data obtained different sources. Nat Biotechnol <br />       2002, 20(10):991-997. <br /> 15.   D&apos;Haeseleer P, Church GM: Estimating improving protein <br />       interaction error rates. Proc IEEE Comput Syst Bioinform Conf <br />       2004:216-223. <br /> 16.   Gagneur J, Krause R, Bouwmeester T, Casari G: Modular decom- <br />       position protein-protein interaction networks. Genome Biol <br />       2004, 5(8):R57. <br /> 17.   Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, Adams SL, Millar A, <br />       Taylor P, Bennett K, Boutilier K, Yang L, Wolting C, Donaldson I, <br />       Schandorff S, Shewnarane J, Vo M, Taggart J, Goudreault M, Muskat B, <br />       Alfarano C, Dewar D, Lin Z, Michalickova K, Willems AR, Sassi H, <br />       Nielsen PA, Rasmussen KJ, Andersen JR, Johansen LE, Hansen LH, Jes- <br />       persen H, Podtelejnikov A, Nielsen E, Crawford J, Poulsen V, <br />       Sorensen BD, Matthiesen J, Hendrickson RC, Gleeson F, Pawson T, <br />       Moran MF, Durocher D, Mann M, Hogue CW, Figeys D, Tyers M: <br />       Systematic identification protein complexes Saccharo- <br />       myces cerevisiae mass spectrometry.                 Nature 2002, <br />       415(6868):180-183. <br /> 18.   Marcotte CJ, Marcotte EM: Predicting functional linkages from <br />       gene fusions confidence.              Appl Bioinformatics 2002, <br />       1(2):93-100. <br /> 19.   Lee I, Narayanaswamy R, Marcotte EM: Bioinformatic prediction <br />       yeast gene function . In Yeast Gene Analysis Edited by: Stansfield <br />       I. Elsevier Press; 2007. <br /> 20.   Samanta MP, Liang S: Predicting protein functions redun- <br />       dancies large-scale protein interaction networks. Proc Natl <br />       Acad Sci U S A 2003, 100(22):12579-12583. <br /> 21.   Schlitt T, Palin K, Rung J, Dietmann S, Lappe M, Ukkonen E, Brazma <br />       A: From gene networks gene function. Genome Res 2003, <br />       13(12):2568-2576. <br /> 22.   Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, <br />       Schultz J, Rick JM, Michon AM, Cruciat CM, Remor M, Hofert C, <br />       Schelder M, Brajenovic M, Ruffner H, Merino A, Klein K, Hudak M, <br />       Dickson D, Rudi T, Gnau V, Bauch A, Bastuck S, Huhse B, Leutwein <br />       C, Heurtier MA, Copley RR, Edelmann A, Querfurth E, Rybin V, <br />       Drewes G, Raida M, Bouwmeester T, Bork P, Seraphin B, Kuster B, <br />       Neubauer G, Superti-Furga G: Functional organization the <br />       yeast proteome systematic analysis protein complexes. <br />       Nature                    2002,                 415(6868):141-147. <br /> 23.   Guldener U, Munsterkotter M, Oesterheld M, Pagel P, Ruepp A, <br />       Mewes HW, Stumpflen V: MPact: MIPS protein interaction <br />       resource yeast. Nucleic Acids Res 2006, 34(Database <br />       issue):D436-41. <br /> 24.   Enright AJ, Van Dongen S, Ouzounis CA: An efficient algorithm <br />       large-scale detection protein families. Nucleic Acids Res <br />       2002, 30(7):1575-1584.                                                            Publish Bio Med Central every <br /> 25.   Brohee S, van Helden J: Evaluation <br /> </body></html>