<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>&#239;&#8250;&#8482; 1996 Oxford University Press                                            Nucleic Acids Research, 1996, Vol. 24, No. 20 4009&#226;&#8364;&#8220;4014 <br />  <br />  <br /> Allelism PSO4 PRP19 links pre-mRNA <br /> processing recombination error-prone DNA <br /> repair Saccharomyces cerevisiae <br /> Martin Grey, Andreas D&#195;&#188;sterh&#195;&#182;ft1, Joao A.P. Henriques2 Martin Brendel* <br />  <br /> Institut f&#195;&#188;r Mikrobiologie der J.W.Goethe Universit&#195;&#164;t, Theodor-Stern-Kai 7, Haus 75, 60590 Frankfurt/Main, <br /> Germany, 1Qiagen GmbH, Max-Volmer-Stra&#195;&#376;e 4, 40724 Hilden, Germany 2Department of <br /> Biophysics/Biotechnology Center&#226;&#8364;&#8220;UFRGS, 91507-970 Porto Alegre, RS, Brazil <br />  <br /> Received June 19, 1996; Revised Accepted August 27, 1996                                DDBJ/EMBL/GenBank accession no. X99770 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                              yeast mutants sensitive photoactivated psoralens (pso mutants; <br />                                                                       9) allocated epistasis group, <br /> The radiation-sensitive mutant pso4-1 Saccharo-                    depending mutagen applied biological end point <br /> myces cerevisiae shows pleiotropic <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span>,                       scored respective double mutants (8). A mutant allele of <br /> including sensitivity DNA cross-linking agents,                    gene PSO4, involved error-prone repair some <br /> nearly blocked sporulation reduced mutability. We                 types recombination, i.e. gene conversion, crossing-over and <br /> have cloned putative yeast DNA repair gene PSO4                   intrachromosomal recombination (10,11,12), originally <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> genomic library complementation                      isolated X-ray-sensitive mutant xs9 (13); be <br /> blocked UV-induced mutagenesis sporulation                     particularly sensitive photoactivated bifunctional 8-methoxy- <br /> in diploids homozygous pso4-1. Sequence analysis                  psoralen renamed pso4-1 (10). This yeast mutant is <br /> revealed gene PSO4 consists 1512 bp located                   phenotypically similar recA mutant Escherichia coli in <br /> upstream UBI4 chromosome XII encodes                      combines mutagen radiation sensitivity block in <br /> putative protein 56.7 kDa. PSO4 allelic PRP19,               recombination induced mutagenesis. The pleiotropic repair <br /> a gene encoding spliceosome-associated protein,                     phenotype caused singly existing pso4-1 mutant led <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> but shares significant homology yeast                   allocation PSO4 epistasis group (8,14). <br /> genes. Gene disruption destroyed reading frame                    More recent studies showed heterologous expression recA <br /> of PSO4 clone resulted death haploid cells,                 increased resistance UV ionizing radiation S.cerevisiae <br /> confirming finding PSO4/PRP19 essential                wild-type recombination-deficient rad52-1 mutant <br /> gene. Thus, PSO4 essential DNA repair                    cells (15,16). However, recA-like yeast mutant pso4-1 showed <br /> gene yeast S.cerevisiae.                                 restored induced mutability transformation multicopy <br />                                                                       vector containing E.coli recA gene (17). Biochemical analysis <br /> INTRODUCTION                                                          proved normal incision 8-MOP + UVA-induced interstrand <br /> At 50 genes encoding proteins involved DNA repair        cross-links (ICLs), failure DNA strand rejoining pso4-1 <br /> known yeast Saccharomyces cerevisiae (1,2) and,          (16), depend recombinational step. The <br /> molecular studies revealed close               apparent role gene PSO4 DNA repair processes attributed <br /> relationship DNA repair enzymes humans                genes belonging epistasis group II (recombination) III <br /> mammals, simple eukaryote advanced model             (error-prone) makes interesting candidate future <br /> which complex mechanisms underlying DNA repair           understanding interconnections repair processes <br /> unraveled near future. The DNA damage-sensitive           yeast. Progress elucidating function Pso4 protein <br /> yeast mutants initially grouped phenotypic groups     necessitated molecular cloning PSO4, wish to <br /> (3,4) which, double mutant analysis measured synergistic      report communication. <br /> or epistatic interactions, allocated epistasis groups <br /> (5&#226;&#8364;&#8220;7). This classification holds today,       MATERIALS AND METHODS <br /> group I comprising &#226;&#710;&#188;20 genes RAD3-like encode            Yeast strains plasmids <br /> proteins nucleotide excision repair, group II comprising &#226;&#710;&#188;12 <br /> genes RAD52-like encode proteins           The yeast strains used listed Table 1. Single copy vector <br /> involved repair DNA double-strand breaks recombina-         pRS316 (18) multicopy vector pRS426 (19), containing <br /> tional mechanisms largest, far                URA3 selectable marker, used subcloning, comple- <br /> heterogeneous, group III, defined RAD6, comprising &#226;&#710;&#188;20             mentation tests sequencing. The original complementing <br /> genes, responsible error-prone DNA repair       isolate pMG470 based multicopy plasmid YEp24 (20). <br /> (1,2,8). Some DNA damage-sensitive mutants, e.g.       Hence, subcloning performed using multicopy vector pRS426 <br />  <br />  <br /> * To   correspondence addressed <br />  4010 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Subcloning pso4-complementing gene passenger DNA pMG470 multicopy plasmids. The restriction maps passenger DNAs <br /> of pMG470-derived plasmids shown. pMG470: original clone isolated gene bank (20) harboring 16 kb passenger. Only 6.5 kb area that <br /> overlaps second positive clone, pMG490 (not shown), depicted detail. pMG471, pMG473 pMG475: non-complementing subfragments pMG470. <br /> pMG480: fragment containing just gene, PRP19, able complement phenotypes pso4-1 mutant. Abbreviations cleavage sites: B/S, ligation <br /> site BamHI Sau3A; E, EcoRI; L, SalI; X, XbaI; Y, BstYI. +/&#226;&#8364;&#8220;: Plasmid complements/does complement pso4-1/pso4-1 pso4-1/prp19-0 mutant. <br /> (B) Non-complementation pso4-1 prp19-0 alleles. Diploids containing different allele combinations constructed tested <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span> 30 36 C <br /> by streaking cell suspensions twoYEPD plates subsequently incubated 36 30 C respectively. (C) Survival mutagenesis irradiation with <br /> UV254 light heteroallelic pso4-1/prp19-0 diploid MG5101 transformed plasmid pRS426 (triangles) pMG480 containing justPRP19 (squares) <br /> and heterozygous pso4-1/PRP19 diploid MG5103 transformed plasmid (circles) positive control. <br />  <br />  <br />  <br /> to allow screening possible multicopy suppressor                        CAN1/can1 marker possible select forward mutants <br /> PSO4 (Fig. 1A).                                                                          homozygous can1 replica-plating synthetic medium <br />                                                                                          lacking arginine uracil supplemented 20 mg/l <br /> Cloning PSO4 <br />                                                                                          canavanine (SynCo&#226;&#8364;&#8220;Arg&#226;&#8364;&#8220;Ura+Can). After irradiation with <br /> Gene PSO4 cloned transforming diploid strain MG5128                               UV254 light (20 J/m2) plates incubated 30 C the <br /> homozygous pso4-1 yeast genomic library (20)                                  dark 3&#226;&#8364;&#8220;5 days. Clones papillae SynCo <br /> according lithium acetate-based protocol described                             &#226;&#8364;&#8220;Arg&#226;&#8364;&#8220;Ura+Can isolated corresponding master <br /> Gietz et al. (21). Due previously introduced heterzygous                          plate tested ability sporulate. <br />  <br /> Table 1. Saccharomyces cerevisiae strains used study <br />  <br />  Strain         Genotype                                                                                                                           Source <br />  xs9            MATa lys2 pso4-1                                                                                                                   A.C.Schenberg <br />  W303           MATa/MAT&#206;&#177; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/PSO          R.Rothstein <br />  MG5100         MATa/MAT&#206;&#177; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/pso4::HIS3   This study <br />  MG5101         MATa/MAT&#206;&#177; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15/HIS can1-100/CAN pso4::HIS3/pso4-1                           This study <br />  MG5103         MATa/MAT&#206;&#177; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15/HIS can1-100/CAN PSO4/pso4-1                                 This study <br />  MG5128         MATa/MAT&#206;&#177; ura3-52/ura3 ade2/ADE can1/CAN pso4-1/pso4-1                                                                             This study <br />                                                                                                               4011 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24,No. <br />                                                                                                               Vol.22,  No.120         4011 <br />  <br /> Mutagenicity assay survival                                         The putative Pso4 protein <br /> Exponentially growing cells cultivated liquid synthetic medium       We determined size PSO4/PRP19 gene 1512 bp, <br /> supplemented appropriate amino acids bases            corresponds 56.7 kDa protein consisting 503 amino <br /> washed resuspended phospate buffer (67 mM, pH 7)            acids, instead 502, originally described PRP19 Cheng <br /> final density 107 cells/ml irradiated different UV254       et al. (22), owing additional proline position 239. <br /> doses. For determination survival cells plated       Sequence analysis putative protein revealed myb-like <br /> complete medium (SynCo) determination forward            DNA binding domain (W[ST]X2E[DE]X2[LIV]) C-terminus <br /> mutations (can1) SynCo arginine plus canavanine          Pso4/Prp19 protein (positions 457&#226;&#8364;&#8220;465, WTKDEESAL). <br /> (20 mg/l) respectively.                                                 The retroviral oncogene v-myb cellular counterpart c-myb <br />                                                                         encode nuclear DNA binding proteins. In S.cerevisiae, the <br /> DNA sequencing                                                          myb-related genes include DNA-binding proteins REB1 (23) <br /> DNA sequencing performed framework            BAS1 (24). However, Pso4/Prp19 protein does share <br /> European Union BIOTECH program sequence entire                   considerable homology putative proteins encoded by <br /> chromosome XII. A set random subclones plasmid cloning            yeast genes. The nucleotide sequence PSO4 <br /> vector pUC18 generated insert region cosmid             appear EMBL, GenBank DDBJ Nucleotide <br /> 1F17. These subclones served starting points primer        Sequence Databases accession no. X99770. <br /> walking strategy. Double-stranded template DNA cosmid <br /> and DNA subclones purified using QIAwell                 Construction phenotype pso4 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>null allele</span> <br /> Purification System (Qiagen). Cycle sequencing reactions were <br /> carried using Taq DNA polymerase fluorescent terminators.       After vitro disruption Pso4/Prp19-encoding ORF the <br /> Gel electrophoresis automated <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> calling          SalI site inserting HIS3 (cf. Fig. 1), performed one-step <br /> ABI 373A Stretch DNA Sequencer (Applied Biosystems).                    disruption experiments haploid diploid wild-type cells. <br /> Contig assembly DNA sequence editing using            While haploid transformants correct insertion be <br /> DNASTAR software package (Lasergene).                                   found, diploid disruptants (MG5100, Table 2) showed two <br />                                                                         surviving His auxotrophic ascospores sporulation tetrad <br /> RESULTS                                                                 analysis. The absence His prototrophic spores confirmed <br />                                                                         previously reported essentiality PSO4/PRP19 gene <br /> Cloning DNA repair gene PSO4                                         (22). However, demonstrated this <br />                                                                         essentiality restricted spore germination. Therefore, the <br /> Diploids homozygous mutant allele pso4-1 higher than <br />                                                                         heterozygous diploid MG5100 transformed pMG480 <br /> wild-type UV sensitivity, lower wild-type UV-induced <br />                                                                         (URA3; PSO4), resulting viable spores sporulation. <br /> <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span> mitotic recombination sporulate poorly <br />                                                                         Plasmid loss experiments spores containing HIS3 <br /> (&lt;1% asci) appropriate medium (8). PSO4 molecularly <br />                                                                         disruption marker did yield viable Ura auxotrophs, while <br /> cloned screening transformants diploid MG5128 <br />                                                                         haploid His auxotrophic non-disruptants showed plasmid loss of <br /> homozygous pso4-1 heterozygous CAN1 (Table 1), in <br />                                                                         &#226;&#710;&#188;20% 36 h. This indicates essentiality PSO4 is <br /> which inducibility mutagenesis UV254 restored <br />                                                                         restricted spore germination, loss plasmid <br /> (CAN1&#226;&#8224;&#8217;can1r). Amongst 6000 transformants to <br />                                                                         results lethality haploid disruptants. As essentiality be <br /> show wild-type-like UV-induced mutagenesis furthermore <br />                                                                         verified construct using URA3 reporter gene (not <br /> restored sporulation (40% transformants respective <br />                                                                         shown), disrupted alleles generally named pso4-0 for <br /> wild-type versus maximal number &lt;1% asci mutant) <br />                                                                         purpose simplification discussion. <br /> and restored resistance DNA cross-linking mutagens. Restriction <br /> analysis complementing plasmids, named pMG470                PSO4 allellic PRP19 <br /> and pMG490, showed passengers contained an <br /> identical 6.5 kb overlapping fragment (depicted Fig. 1A).            Besides fact gene PRP19 sufficient to <br /> Hybridization 32P-labeled 2 kb EcoRI fragment            complement pso4-1 mutation, final evidence allelism <br /> area (Fig. 1A, pMG475) revealed passengers        cloned ORF PSO4 obtained showing null <br /> chromosome XII (not shown). Two subfragments obtained                allele prp19::HIS3 complement phenotype pso4-1 <br /> restriction plasmid pMG470 singular SalI site yielded     mutant heteroallelic diploid constructed crossing <br /> two plasmids pMG471 pMG473 (Fig. 1A), unable                original pso4-1 mutant xs9 haploid prp19::HIS3 <br /> restore PSO phenotype (Table 2A), indicating SalI           disruptant MG5100-1C opposite mating-type, latter <br /> located pso4-1 complementing open reading frame              containing PSO4-harboring plasmid pMG480 ensure viability of <br /> (ORF) promoter. We determined DNA sequence           haploid disruptant. After plasmid loss, resulting Ura <br /> area match gene PRP19 (22), containing SalI        auxotrophic diploid MG5101, heteroallelic pso4-1/prp19::HIS3, <br /> site. Further sequencing revealed PSO4/PRP19 located            viable showed pleiotropic phenotype typical a <br /> &#226;&#710;&#188;1.5 kb upstream UBI4 shares promoter region        homozygous pso4-1 diploid, including strong reduction in <br /> &#226;&#710;&#188;320 bp ORF (L0916, Fig. 1A) upstream PRP19             sporulation induced mutagenesis sensitivity <br /> encoding hitherto uncharacterized protein containing putative       DNA cross-linking mutagens (Table 2 Fig. 1C). <br /> ATP/GTP binding site. Subcloning 2.4 kb fragment                   Since prp19-0 allele unable complement pso4-1 mutant <br /> containing gene PRP19 (pMG480, Fig. 1A) revealed         phenotype, final proof genes allelic. Thus <br /> gene sufficient restore wild-type-like phenotype   terms pso4::HIS3 (pso4-0) prp19::HIS3 (prp19-0) be <br /> transformed pso4 mutant (Fig. 1C Table 2A).                  considered synonymous. <br />  4012 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br /> Table 2. (A) Complementation analysis different subclones pMG470 (see Fig. 1A); (B) complementation analysis set plasmids transformed <br /> into heteroallelic diploid MG5101; (C) non-complementation prp19 pso4 mutant alleles (MG5101) evidence allelism (see Fig. 1B C) <br />  <br />  Strain            Relevant genotype      Plasmid              Sporulation          HN2                   30 C                 36 C              Mutability <br />  (A) <br />  <br />  MG5128            pso4-1/pso4-1          pRS426               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG470               45%                  R                     +                    +                 + <br />                                           pMG471               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG473               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (B) <br />  <br />  MG5101            prp19-0/pso4-1         pRS426               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG470               40%                  R                     +                    +                 + <br />                                           pMG471               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG473               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (C) <br />  <br />  W303              PSO/PSO                None                 35%                  R                     +                    +                 + <br />  MG5100            prp19-0/PSO            None                 33%                  R                     +                    +                 + <br />  MG5128            pso4-1/pso4-1          None                 &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />  MG5103            pso4-1/PSO             None                 32%                  R                     +                    +                 + <br />  MG5101            prp19-0/pso4-1         None                 0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />  <br /> R, wild-type-like resistance; S, pso4 mutant-like sensitivity HN2. <br /> +/&#226;&#8364;&#8220; corresponds viability/non-viability 30 36 C mutability/reduced mutability respectively. <br /> Only plasmids pMG470 pMG480 complement pso4-1 mutation. <br /> Sporulation efficiency, sensitivity DNA cross-linking mutagen nitrogen mustard (HN2), mutability determined 30 C. <br />  <br /> pso4-1 non-lethal allele essential gene                                  mutagenesis, lowered spontaneous induced recombination <br /> PSO4/PRP19                                                                           nearly totally blocked pre-meiotic DNA synthesis and <br />                                                                                      sporulation (8,12,27). We isolated gene PSO4, which <br /> Although disruption PSO4/PRP19 lethal, mutants containing                    member RAD52 epistasis group complementation <br /> the pso4-1 allele viable above-mentioned                            pso4-1 mutant&#226;&#8364;&#8482;s drastically reduced induced mutability and <br /> phenotype. Often, viable mutants essential genes isolated                     blocked sporulation. Characterization DNA repair gene PSO4 <br /> as sensitve temperatures &gt;30 C. We tested                           showed allelism yeast gene PRP19, encodes 502 <br /> diploids harboring different alleles PSO4 possible temperature                amino acid spliceosome-associated protein (22,28). In contrast, <br /> sensitivity. In fact, diploids homoallelic pso4-1                 PSO4 sequence contains additional proline-coding CCC <br /> heteroallelic (pso4-1/prp19-0) sensitive 36 C. Again,                    triplet encodes 503 amino acid protein. However, this <br /> prp19-0 allele failed complement sensitivity,                      discrepancy strain-specific difference is <br /> more confirming allelism genes (Fig. 1B). When                               presently likely irrelevant discussion putative <br /> comparing growth yield diploid strains 30 C                         protein. <br /> found heteroallelic pso4-1/pso4-0 diploid MG5101 grow                          The allelism PSO4 PRP19 implicates yeast gene <br /> at 30 C (cf. above) severely retarded growth rate                     encodes protein functions RNA splicing in <br /> a generation time &gt;3 h YEPD medium, isogenic                            error-prone DNA repair, recombination sporulation this <br /> PSO4/pso4-0 diploid MG5103 showed generation time &#226;&#710;&#188;1.5 h,                       organism. It DNA repair gene function <br /> comparable wild-types (not shown).                                        linked processing RNA essentiality underscores <br />                                                                                      vital function process. Next RAD3 and <br /> DISCUSSION                                                                           SSL2/RAD25, PSO4/PRP19 DNA repair gene is <br />                                                                                      essential growth viability S.cerevisiae. <br /> Repair DNA double-strand breaks S.cerevisiae involves                             The Prp19 protein demonstrated essential for <br /> genes RAD52 epistasis group. The corresponding proteins                       splicing pre-mRNA (28). However, biochemical characterization <br /> are supposed act recombinational mechanism                         revealed tightly associated snRNAs, is <br /> of shown constitute protein complex                        associated spliceosome splicing reaction (22). <br /> has termed recombinosome (25,26). A pso4-1 mutant                         Interestingly, Cheng et al. discussed fact Prp19 distinct <br /> previously shown exhibit pleiotropic phenotype, i.e.                       Prp proteins spliceosomal components <br /> sensitivity DNA damaging agents, nearly blocked induced                      regarding protein sequence does contain of <br />                                                                                                                 4013 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24,No. <br />                                                                                                               Vol.22,  No.120              4013 <br />  <br /> the motifs Prp sequences. It      (35) require splicing proper function. Since <br /> demonstrated Prp19 protein associated protein       pso4-1 mutants able incise DNA (14), known <br /> complex different spliceosome, consisting         splicing-dependent repair gene functional, pointing <br /> seven proteins addition Prp19, itself likely present     normal splicing activity pso4-1 mutants 30 C. Also, if <br /> an oligomeric form (29). However, Prp19-associated complex,        non-functional splicing resulted repair deficiency pheno- <br /> itself big spliceosome, unlikely bind     types, &#226;&#710;&#188;50 DNA repair genes cloned ought to <br /> latter complex (29). The authors suggest Prp protein      encode proteins needed splicing, i.e. turned <br /> be released associated splicing complex. This       allelic PRP genes. <br /> event concomitant just dissociation         The severe handicap growth complete medium 30 C of <br /> of U4 snRNA ATP-dependent conformational change             heteroallelic pso4-1/pso4-0 heterozygous pso4-1/PSO4 <br /> before formation functional spliceosome, suggesting          diploids indicates quantity protein produced one <br /> Prp19 function step spliceosome assembly (30).          pso4-1 mutant allele sufficient normal spliceosome <br />    What be, hand, function               activity, protein encoded PSO4 wild-type <br /> PRP19/PSO4-encoded protein recombinational error-prone          allele normal growing PSO4/pso4-0 diploid apparently <br /> repair? All evidence gathered genetic biochemical             satisfies cellular demand spliceosome-associated protein <br /> experiments using allele pso4-1 point late function repair     function. In context partial complementation yeast <br /> of DNA lesions missing mutant strain. The excision           pso4-1 mutant&#226;&#8364;&#8482;s sensitivity photoactivated 8-MOP + UVA by <br /> 8-MOP + UVA-induced ICLs DNA thought proceed                overexpression RecA protein (17) new significance. RecA <br /> normally, production DNA double-strand breaks repair        protein bind single-and double-stranded DNA, thereby <br /> intermediates which, however, rejoined pso4-1 (14). It      facilitating initiation recombination (36). Interstrand cross-link <br /> has suggested repair DNA lesions ICL type          repair E.coli yeast shown proceed via <br /> would require modes repair (31,32),             double-strand breaks (cf. 2,31); secondary lesions be <br /> removal damage repairosome postulated                substrate RecA RecA-like proteins which, helical <br /> nucleotide excision repair (33) second enzyme          filament formation, initiate recombinational repair. The RecA <br /> complex repair strand breaks, PSO4/PRP19           Rad51 proteins share considerable sequence homology and <br /> encoded protein function. The existence S.cerevisiae   strong evidence Rad51 binds Rad52 protein <br /> of DNA double-strand repair-specific recombinosome,           (26). The sensitivity pso4-1 mutant 8-MOP + UVA, its <br /> complex containing proteins encoded RAD51, RAD52,          partial complementation overexpressed RecA protein the <br /> RAD55 RAD57, recently suggested (26). We suggest          epistatic interactions pso4-1 mutant rad51 rad52 <br /> that Prp19/Pso4-associated protein complex Tarn           mutant alleles (17) lead hypothesis functions in <br /> and co-workers (29) above-mentioned recombino-            repair primary secondary induced DNA strand breaks via <br /> some.                                                                  recombinational mechanism. The association functional <br />    The fact existing viable alleles essential gene    protein complex, forming recombinosome (26), <br /> PSO4/PRP19, represented previously described prp19           function Pso4/Prp19 protein; other, more <br /> mutant (22,34) pso4-1 mutant respectively, exhibit        essential yeast cell, role spliceosome- <br /> temperature sensitivity implies 503 amino acid protein        associated protein pre-mRNA processing. Recently, similar <br /> encoded PSO4/PRP19 functional                two-way association described yeast transcription <br /> domains, active protein     factor TFIIH: participates formation holocomplex active <br /> expressed haploid diploid pso4-1 mutants:        RNA polymerase II transcription initiation be <br /> no obvious growth retardation 30 C pso4-1 haploid            repairosome, specific nucleotide excision <br /> homoallelic diploid mutants assume RNA splicing           repair (37,38). <br /> proceeds efficiency close identical the <br /> wild-type. This hypothesis underscored fact         ACKNOWLEDGEMENTS <br /> viable prp19 mutant accumulates unprocessed pre-mRNA the <br /> non-permissive permissive temperature (34),             We thank Dr A.C.Schenberg pso4-1 mutant Drs <br /> suggesting viability pso4/prp19 mutant dependent       J.Brozmanova W.Siede helpful discussion. Sequencing of <br /> functional splicing. Survival pso4-1 mutant pso4-0     left arm chromosome XII supported grant AD <br /> therefore suggests mutant allele expresses     European Union BIOTECH program coor- <br /> partially functional protein 30 C; role spliceosomal   dination Joerg Hoheisel, DKFZ, Heidelberg. Research was <br /> assembly function normal function            supported Qualit&#195;&#164;tsweine D.Brendel. The Brazilian&#226;&#8364;&#8220;German <br /> repair DNA single- double-strand breaks recombinational     cooperation facilitated CNPq-DAAD travel grants. <br /> processes significantly perturbed. We suggest these <br /> phenotypes disturbed binding proteins <br /> to Pso4 multienzyme complex Tarn et al. (29),          REFERENCES <br /> indicating hitherto unidentified proteins    1 Haynes,R.H. Kunz,B. (1981) In: Strathern,J.N., Jones,E.W. and <br /> may belong RAD52 epistasis group.                                  Broach,J.R (eds), The Molecular Biology Yeast Saccharomyces <br />    An alternative, opinion highly unlikely, explanation        cerevisiae. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, <br /> for repair phenotype pso4-1 mutant                pp. 341&#226;&#8364;&#8220;414. <br />                                                                         2 Friedberg,E.C., Walker,G.C. Siede,W. (1995) DNA Repair and <br /> incorrect processing transcripts DNA repair genes. Amongst          Mutagenesis. ASM Press, Washington, DC. <br /> the repair genes yeast one, RAD14,            3 Brendel,M., Khan,N.A. Haynes,R.H. (1970) Mol. Gen. Genet., 106, <br /> is involved incision damaged DNA, intron-containing              289&#226;&#8364;&#8220;295. <br />  4014 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br />  4   Game,J.C. Cox,B.S. (1971) Mutat. Res., 12, 328&#226;&#8364;&#8220;331.                       23 Ju,Q., Morrow,B.E. Warner,J.R. (1990) Mol. Cell. Biol., 10, <br />  5   Game,J.C. Cox,B.S. (1972) Mutat. Res., 16, 353&#226;&#8364;&#8220;362.                          5226&#226;&#8364;&#8220;5234. <br />  6   Brendel,M. Haynes,R.H. (1973) Mol. Gen. Genet., 125, 197&#226;&#8364;&#8220;216.             24 Tice-Baldwin,K., Fink,G.R. Arndt,K.T. (1989) Science, 246, 931&#226;&#8364;&#8220;935. <br />  7   Game,J.C. Cox,B.S. (1973) Mutat. Res., 20, 35&#226;&#8364;&#8220;44.                         25 Firmenich,A.A., Elias-Arnanz,M. Berg,P. (1995) Mol. Cell. Biol., 15, <br />  8   Henriques,J.A.P. Brendel,M. (1990) Curr. Genet., 18, 387&#226;&#8364;&#8220;393.                1620&#226;&#8364;&#8220;1631. <br />  9   Henriques,J.A.P. Moustacchi,E. (1980) Genetics, 95, 273&#226;&#8364;&#8220;288.              26 Hays,S.L., Firmenich,A.A. Berg,P. (1995) Proc. Natl. Acad. Sci. USA, <br /> 10   Henriques,J.A.P., Vicente,E.J., daSilva,K.V.C.L. Schenberg,A.C.G.            92, 6925&#226;&#8364;&#8220;6929. <br />      (1989) Mutat. Res., 218, 111&#226;&#8364;&#8220;124.                                             27 daSilva,K.V.C.L, Morais,M.A.,Jr, Henriques,J.A.P. (1995) Curr. <br /> 11   Andrade,H.H.R., Marques,E.K., Schenberg,A.C.G. Henriques,J.A.P.              Genet., 27, 207&#226;&#8364;&#8220;212. <br />      (1989) Mol. Gen. Genet., 217, 419&#226;&#8364;&#8220;426.                                        28 Tarn,W.-Y., Lee,K.R. Cheng,S.-C. (1993) Mol. Cell Biol., 13, <br /> 12   Meira,L.B., Fonseca,M.B., Averbeck,D., Schenberg,A.C.G.                      1883&#226;&#8364;&#8220;1891. <br />      Henriques,J.A.P. (1992) Mol. Gen. Genet., 235, 311&#226;&#8364;&#8220;316.                       29 Tarn,W.Y., Hsu, C.H., Huang,K.T., Chen,H.R., Kao,H.Y., Lee,K.R. and <br /> 13   Benathen,I.A. Beam,C.A. (1977) Radiat. Res., 69, 99&#226;&#8364;&#8220;116.                     Cheng,S.C. (1994) EMBO J., 13, 2421&#226;&#8364;&#8220;2431. <br />                                                                                    30 Tarn,W.Y., Lee,K.R. Cheng,S.C. (1993) Proc. Natl. Acad. Sci. USA, <br /> 14   Morais,M.A.,Jr, Vicente,E.J., Brozmanov&#195;&#161;,J., Schenberg,A.C.G. and <br />                                                                                       90, 10821&#226;&#8364;&#8220;10825. <br />      Henriques,J.A.P. (1996) Curr. Genet., 29, 211&#226;&#8364;&#8220;218. <br />                                                                                    31 Jachymczyk,W.J., von Borstel,R.C., Mowat,M.R.C. Hastings,P.J. <br /> 15   Brozmanov&#195;&#161;,J., Cern&#195;&#161;kov&#195;&#161;,L., Vlckov&#195;&#161;,V., DurajJ. Fridrichov&#195;&#161;,I. <br />                                                                                       (1981) Mol. Gen. Genet., 182, 196&#226;&#8364;&#8220;205. <br />      (1991) Mol. Gen. Genet., 227, 473&#226;&#8364;&#8220;480.                                        32 Maga&#195;&#177;a-Schwencke,N., Henriques,J.A.P.,Chanet,P. Mousacchi,E. <br /> 16   Cernakov&#195;&#161;,L., Fridrichov&#195;&#161;,I., Pirsel,M., Kleibl,K., Duraj,J.                 (1982) Proc. Natl. Acad. Sci. USA, 79,1722&#226;&#8364;&#8220;1726. <br />      Brozmanov&#195;&#161;,J. (1991) Biochimie, 73, 285&#226;&#8364;&#8220;288.                                  33 Bardwell,A.J., Bardwell,L., Iyer,N., Svefstrup,J.Q., Feaver,R.D., <br /> 17   Morais,M.A.,Jr, Brozmanov&#195;&#161;,J., Benfato,M., Duraj,J., Vlckov&#195;&#161;,V.              Kornberg,R.D. Friedberg,E.C. (1994) Mol. Cell. Biol., 14, 3569&#226;&#8364;&#8220;3576. <br />      Henriques,J.A.P. (1994) Mutat. Res., 314, 209&#226;&#8364;&#8220;220.                            34 Vijayraghavan,U., Company,M. Abelson,J. ( <br /> </body></html>