<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br /> Naf1 p is a box H/ACA snoRNP assembly factor <br /> A. Fatica, M. Dlakic and D. Tollervey <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Box H/ACA small nucleolar ribonucleoprotein particles (snoRNPs) contain four essential proteins, Cbf5p, Gar1p, <br /> Nhp2p, and Nop10p, each of which, with the exception of Gar1p, is required for box H/ACA snoRNA accumulation. <br /> Database searches identified a novel essential protein, which we termed Naf1p, with a region of homology <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the <br /> RNA-binding domain of Gar1p and other features in common with hnRNP-like proteins. Naf1p is localized to the <br /> nucleus and is not a stable component of the H/ACA snoRNPs, but it is required for the accumulation of all box H/ACA <br /> snoRNAs. This requirement is not at the level of snoRNA transcription initiation or termination. Naf1p shows in vitro <br /> RNA-binding activity and also binds directly to Cbf5p and Nhp2p. Naf1p was shown to bind to the CTD in vivo in a <br /> two-hybrid assay, and the phosphorylated CTD, but not the nonphosphorylated CTD, was shown to precipitate tagged <br /> Naf1p <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> a cell lysate. We propose that Naf1p is recruited to the CTD of RNA polymerase II and binds to nascent box <br /> H/ACA snoRNAs promoting snoRNP assembly. <br /> Keywords: ACA snoRNPs; box H; snoRNP assembly <br />  <br />  <br />  <br /> INTRODUCTION                                                          proteins, which generally have some role in ribosome <br />                                                                       synthesis or function (Kiss, 2002)+ The yeast H/ACA <br /> Ribosomal RNA modification and processing requires <br />                                                                       snoRNAs are rather unusual in being predominately <br /> two large families of small nucleolar RNAs (snoRNAs), <br />                                                                       synthesized from independent transcription units, and <br /> which are associated with protein components in small <br />                                                                       only snR44 is known to be intron encoded+ Despite the <br /> nucleolar ribonucleoproteins (snoRNPs)+ The box C/D <br />                                                                       diversity of genomic organization, in all cases, snoRNA <br /> snoRNPs select sites of rRNA 29-O -methylation, <br />                                                                       synthesis relies on the generation of entry sites for <br /> whereas the box H/ACA snoRNPs select sites of pseudo- <br />                                                                       exonucleases in the precursor molecule followed by <br /> uridine formation (reviewed in Tollervey &amp; Kiss, 1997; <br />                                                                       exonuclease trimming to produce a mature snoRNA+ <br /> Weinstein &amp; Steitz, 1999; Kiss, 2002)+ In addition, a few <br />                                                                       Concomitant with the processing of precursor mol- <br /> members of each class are required for correct pro- <br />                                                                       ecule, snoRNP-specific complexes assemble on <br /> cessing of the pre-rRNA to the mature rRNA+ In yeast, <br />                                                                       snoRNA sequence and protect it from further exonucle- <br /> these include the box H/ACA snoRNAs snR10 and <br />                                                                       olytic digestion (Filipowicz &amp; Pogacic, 2002)+ <br /> snR30, both of which are required for normal process- <br />                                                                         During the synthesis of yeast snRNAs and snoRNAs, <br /> ing on the pathway of 18S rRNA synthesis (Tollervey, <br />                                                                       a hnRNP-like protein, Nrd1p, is proposed to bind to <br /> 1987; Morrissey &amp; Tollervey, 1993)+ The box H/ACA <br />                                                                       nascent transcripts, recruiting another RNA-binding pro- <br /> snoRNPs contain four common proteins, Nhp2p, <br />                                                                       tein, Nab3p, and the putative RNA helicase Sen1p, and <br /> Nop10p, Gar1p, and the pseudouridine synthase Cbf5p+ <br />                                                                       promoting transcription termination (Steinmetz et al+, <br /> Each of these proteins is required for H/ACA snoRNA <br />                                                                       2001), probably via interactions with the C-terminal do- <br /> accumulation, with the notable exception of Gar1p (Gi- <br />                                                                       main of the largest subunit of RNA polymerase II (RNA <br /> rard et al+, 1992; Henras et al+, 1998; Lafontaine et al+, <br />                                                                       pol II; Yuryev et al+, 1996; Conrad et al+, 2000)+ <br /> 1998; Watkins et al+, 1998)+ <br />                                                                         In vertebrate cells, the assembly of snRNPs in the <br />   Many snoRNAs have an unusual biosynthetic path- <br />                                                                       cytoplasm and snoRNPs in the nucleus are likely to be <br /> way, being excised from the introns of pre-mRNAs+ In <br />                                                                       mediated by a large oligomeric complex, termed the <br /> most cases, the exons of these pre-mRNAs encode <br />                                                                       SMN complex, consisting of the SMN (<span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>survival</span> of mo- <br />                                                                       tor neurons) protein together with the Gemin2 to 6 pro- <br />                                                                       teins, and probably other factors (Fischer et al+, 1997; <br />   Reprint requests to: David Tollervey, Wellcome Trust Centre for <br /> Cell Biology, Swann Building, King&#226;&#8364;&#8482;s Buildings, University of Edin-   Charroux et al+, 1999, 2000; Pellizzoni et al+, 2001a, <br /> burgh, Edinburgh EH9 3JR, UK; e-mail: d+tollervey@ed+ac+uk+           2002; Gubitz et al+, 2002)+ In the nucleus, the SMN <br />                                                                   1502 <br />                                      Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> H/ACA snoRNP synthesis                                                                                                                  1503 <br />  <br /> complex also associates with RNA pol II (Pellizzoni                      ships to known snoRNP proteins and/or protein in- <br /> et al+, 2001b)+ SMN binds directly to the glycine and                    volved in snoRNP synthesis+ FASTA searches (Pear- <br /> arginine-rich repeat domains (RGG or GAR domains),                       son &amp; Lipman, 1988) of the yeast proteome identified a <br /> which are present in the Sm protein component of the                     largely uncharacterized, essential ORF (YNL124w) en- <br /> snRNPs, in fibrillarin, the box C/D snoRNA-associated                    coding a protein with a region of significant homology <br /> 29-O -methylase, and in hGar1p (Brahms et al+, 2001;                     (32% identity, 49% similarity) to the RNA-binding do- <br /> Jones et al+, 2001; Meister et al+, 2001; Pellizzoni et al+,             main of the box H/ACA snoRNP protein Gar1p1p (Bagni <br /> 2001a)+ However, although the snoRNP and Sm pro-                         &amp; Lapeyre, 1998) (Fig+ 1A,B)+ We termed this protein <br /> teins are highly conserved in the evolution, no clear                    nuclear assembly factor 1 (Naf1p)+ Sequence compar- <br /> homologs of the SMN complex have been identified in                      isons against the yeast proteome often produce in- <br /> Saccharomyces cerevisiae or in plants+                                   flated and unrealistic estimates of statistical significance <br />   Database searches identified Naf1p, a novel protein                    due to the small database size, and we sought confir- <br /> with homology to both Gar1p and hnRNP-like proteins+                     mation of the Naf1p&#226;&#8364;&#8220;Gar1p relationship by iteratively <br /> Naf1p is required for accumulation of H/ACA snoRNAs                      searching the nonredundant protein database+ Be- <br /> and may bind to the same site as Gar1p in the snoRNA                     cause of its high compositional bias, Naf1p sequence <br /> and participates in snoRNP assembly+                                     was processed with SEG (Wootton &amp; Federhen, 1996) <br />                                                                          to remove low-complexity regions and to delineate three <br />                                                                          globular regions that were used individually for further <br /> RESULTS <br />                                                                          sequence analysis+ PSI-BLAST searches (Altschul et al+, <br />                                                                          1997) with the Naf1p domain encompassing residues <br /> Naf1p is a novel yeast hnRNP-like protein <br />                                                                          110&#226;&#8364;&#8220;220 readily identified orthologs in Schizosaccha- <br /> We reasoned that novel proteins involved in yeast                        romyces pombe and several metazoans (Fig+ 1C)+ Re- <br /> snoRNP assembly could be detected by their relation-                     ciprocal searches with Naf1p orthologs confirmed the <br />  <br />  <br />  <br />  <br />         FIGURE 1. A: Schematic representation of Naf1p+ Colored boxes indicate regions of homologies with other proteins+ <br />         Putative domains are indicated+ B: Alignment of S. cerevisiae Naf1p with the RNA-binding domain of yeast Gar1p+ <br />         C: Alignments of Naf1p homologs+ SwissProt protein names are shown together with species abbreviations (Sc: Saccha- <br />         romyces cerevisiae; Sp: Schizosaccharomyces pombe; Hs: Homo sapiens; Dm: Drosophila melanogaster; Ce: Caenorhab- <br />         ditis elegans; At: Arabidopsis thaliana )+ Numbers flanking the alignments indicate the residue numbers of aligned sequences+ <br />         Identical residues are shaded yellow and indicated with uppercase letters in the consensus sequences+ The lowercase <br />         letters on the consensus line have the following meaning: a, aromatic (FHWY); b, big (EFHIKLMQRWY); c, charged <br />         (DEHKR); h, hydrophobic (ACFGHILMTVWY); l, aliphatic (ILV); p, polar (CDEHKNQRST); s, small (ACDGNPSTV); <br />         1, positively charged (HKR); 2, negatively charged (DE)+ <br />                                Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> 1504                                                                                                   A. Fatica et al. <br />  <br /> homology with Gar1p; for example, the Drosophila             and the homology suggested the possibility of inter- <br /> ortholog of Naf1p (CG10341) identified Gar1p in the          actions between Naf1p and the transcription machinery+ <br /> first iteration (E 5 0+002) and retrieved other homologs <br /> of Gar1p in subsequent iterations (data not shown)+ <br />                                                              Naf1p is localized to the nucleus <br /> The region of sequence similarity is longer than that <br /> shown in Figure 1B, although it is discontinuous be-         To determine the subcellular localization of Naf1p, <br /> cause of insertions and deletions+ Moreover, the two         a C-terminal fusion between Naf1p and the tandem <br /> proteins show similarity at the level of secondary struc-    affinity purification (TAP) tag was constructed and ex- <br /> ture even when clear primary sequence similarity is          pressed under the control of its own promoter (Naf1- <br /> absent, and their relatedness is unquestionably solid        TAP; see Materials and Methods)+ The fusion protein is <br /> based on PSI-BLAST statistics+ Finally, subsequent           the only source of Naf1p in this strain, which had a <br /> TBLASTN searches of the EST database revealed that           growth rate identical to that of the otherwise isogenic <br /> Naf1p orthologs are widespread in animal and plant           wild-type strain (data not shown), showing the tagged <br /> species+                                                     protein to be fully functional+ The protein A moiety of <br />    Despite being studied experimentally for more than a      the TAP tag was used to localize the fusion protein <br /> decade, it is not clear whether Gar1p and its homologs       by indirect immunofluorescence microscopy with anti- <br /> are related to structurally characterized proteins+ To       protein A antibodies (Fig+ 2C)+ To allow visualization of <br /> address this issue, we performed threading of Naf1p          the nucleolus, the Naf1-TAP strain was additionally <br /> and Gar1p versus a library of known protein structures+      transformed with a plasmid expressing DsRed-tagged <br /> Threading is the process where a protein of unknown          Nop1p (Fig+ 2B; Milkereit et al+, 2001)+ This decorated <br /> three-dimensional structure is compared with known           a crescent-shaped region of the nucleus, which is char- <br /> folds in an attempt to recognize the structural similarity   acteristic of the yeast nucleolus+ The cells were also <br /> between them (Bowie et al+, 1991; Jones et al+, 1992)+       treated with DAPI to visualize the nucleoplasm <br /> The compatibility between the sequence and the pro-          (Fig+ 2A,F)+ Superimposition of the different signals <br /> posed structure is evaluated by means of a set of            (Fig+ 2B,D,E) showed that the Naf1-TAP signal over- <br /> empirical potentials derived from proteins of known          lapped the regions decorated by both DAPI and DsRed- <br /> structure+ To improve the chance of finding a correct        Nop1p+ The otherwise isogenic wild-type strain, which <br /> fold, the original threading protocol (Jones, 1999)          was utilized as control, gave no clear signal in the <br /> was supplemented by secondary structure predictions          channels used to detect DsRed-Nop1p or Naf1-TAP <br /> (Jones, 1999)+ Both Naf1p and Gar1p showed struc-            (Fig+ 2F&#226;&#8364;&#8220;L)+ These data indicate that Naf1p is localized <br /> tural homology to the common fold of the Sm proteins         to both the nucleolus and nucleoplasm+ <br /> (Kambach et al+, 1999) and to the structurally related <br /> tudor domain of the SMN protein (Selenko et al+, 2001; <br />                                                              Construction of a conditional NAF1 allele <br /> see Materials and Methods for details)+ This fold as- <br /> signment was confirmed by building a robust three-           To study the function of Naf1p, we constructed a con- <br /> dimensional model of Gar1p using human Sm proteins           ditional allele by placing the expression of a N-terminal <br /> as a template (M+ Dlakic&#194;&#180;, A+ Fatica, &amp; D+ Tollervey, in     ProtA-tagged Naf1p under the control of a repressible <br /> prep+)+                                                      GAL10 promoter (strain YAF9)+ In galactose liquid me- <br />    Sequence analysis and structural prediction identi-       dia, growth of the GAL::ProtA-naf1 strain was identical <br /> fied second and third globular domains of Naf1p (res-        to that of the isogenic wild-type strain (data not shown)+ <br /> idues 277&#226;&#8364;&#8220;365 and 391&#226;&#8364;&#8220; 448) that contain a short RE/         Following transfer to glucose liquid media, the growth <br /> RS-rich region and a C-terminal P 1 Q-rich domain,           rate of both strains was initially the same, but growth of <br /> respectively+ These are features present in several yeast    the GAL::ProtA-naf1 strain decreased progressively, <br /> hnRNP-like proteins (Wilson et al+, 1994), including         commencing 15 h after transfer (Fig+ 3A)+ By 24 h after <br /> Nrd1p and Nab3p, although the putative RNA-binding           transfer, growth was severely reduced+ Western blot <br /> domains of these proteins are not homologous to Naf1p        analysis confirmed the depletion of ProtA-Naf1p on glu- <br /> (data not shown)+                                            cose medium (Fig+ 3B)+ <br />    Together these features suggested a role for Naf1p <br /> in the metabolism of box H/ACA snoRNAs+ <br />                                                              Naf1p is required for the synthesis <br />    In addition, the N terminus of Naf1p was reported <br />                                                              of H/ACA snoRNAs <br /> to show 26% identity (44% similarity) to the human <br /> protein TFIIA alpha/beta-like factor (ALF; see www+          The homology between Naf1p and the RNA-binding <br /> incyte+com/proteome for alignment), a testis-specific        domain of Gar1p suggested the possible involvement <br /> homolog of the large subunit of the general transcrip-       of Naf1p in the metabolism of box H/ACA snoRNAs, <br /> tion factor TFIIA (Upadhyaya et al+, 1999; reviewed in       the levels of which were therefore determined during <br /> Veenstra &amp; Wolffe, 2001)+ ALF aids the association of        Naf1p depletion (Fig+ 4A)+ Northern hybridization with <br /> the TATA-binding protein (TBP) with promoter regions,        probes specific for snoRNAs showed that all tested <br />                                       Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> H/ACA snoRNP synthesis                                                                                                            1505 <br />  <br />  <br />  <br />  <br />          FIGURE 2. Naf1p is localized to the nucleus+ Indirect immunofluorescence was performed with cells expressing a TAP- <br />          tagged Naf1p and transformed with a plasmid expressing a DsRed-tagged a Nop1p (YAF15 strain)+ Isogenic wild-type cells <br />          were utilized as control+ A,F: DNA was stained with DAPI+ B,G: Localization of the DsRed-Nop1p+ C,H: Indirect immuno- <br />          fluorescence with a rabbit anti-Protein A antibody+ D,I: Superimposition of signals obtained from Nop1p and Naf1-TAP+ <br />          E,L: Superimposition of the signals obtained from Nop1p, Naf1p, and the DAPI staining+ <br />  <br />  <br />  <br /> H/ACA snoRNAs, both independently transcribed and                      C/D snoRNAs, the U4 snRNA, or the RNA component <br /> intron encoded, were strongly depleted following trans-                of the signal recognition particle, scR1+ The signals for <br /> fer of the GAL::ProtA-naf1 strain to glucose medium+ In                these RNAs actually increased during depletion of <br /> contrast, no depletion was observed for any tested box                 Naf1p+ It should, however, be noted that the gels were <br />                                                                        loaded using constant amounts of total RNA, and de- <br />                                                                        pletion of the rRNAs (see below) leads to some over- <br />                                                                        loading of RNAs that are not depleted (Henras et al+, <br />                                                                        1998; Lafontaine et al+, 1998)+ <br />                                                                           To investigate the association of Naf1p with the box <br />                                                                        H/ACA snoRNAs, immunoprecipitation was performed <br />                                                                        on lysates of the Naf1-TAP strain and an isogenic strain <br />                                                                        expressing only nontagged Naf1p (Fig+ 4B)+ No signif- <br />                                                                        icant coprecipitation was observed for most of the tested <br />                                                                        independently transcribed H/ACA snoRNAs and the <br />                                                                        box C/D snoRNA U3+ Stronger coprecipitation was seen <br />                                                                        for snR30 and snR44, the only intron-encoded box <br />                                                                        H/ACA snoRNA in yeast+ This may be specific, as sim- <br />                                                                        ilar levels of recovery were observed when precipita- <br />                                                                        tion was performed in 150 or 500 mM NaCl+ Recovery <br />                                                                        of weakly associated RNAs is expected to be reduced <br />                                                                        under the more stringent conditions of 500 mM NaCl+ <br />                                                                        We conclude that Naf1p is not a major component of <br />                                                                        the box H/ACA snoRNPs but may transiently associate <br />                                                                        with the snoRNAs+ The observation that Naf1-TAP did <br />                                                                        not show nucleolar enrichment also suggested that it is <br />                                                                        not associated with most of the H/ACA snoRNA pop- <br />                                                                        ulation+ We therefore predicted that Naf1p was specif- <br />                                                                        ically required for the synthesis of H/ACA snoRNAs or <br />                                                                        RNP assembly+ <br />  <br />                                                                        Naf1p is required for pre-rRNA processing <br /> FIGURE 3. A: Growth rate of GAL::ProtA-naf1 (squares) and wild- <br /> type (circles) strains following a shift from galactose- to glucose-   To test the requirement for Naf1p in rRNA synthe- <br /> containing media+ B: Western blot analysis of Naf1p depletion+         sis, the levels of mature rRNA and pre-rRNAs were <br /> Whole-cell extracts were prepared from samples harvested at the        assessed by northern hybridization during Naf1p de- <br /> indicated times+ Equal amount of proteins were separated by 12% <br /> SDS-PAGE and ProtA-Naf1p was detected by western blot using            pletion (Fig+ 5)+ RNA was extracted from the GAL::ProtA- <br /> peroxidase-conjugated rabbit IgG (Sigma)+                              naf1 and wild-type strains grown in galactose medium <br />                                Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> 1506                                                                                                             A. Fatica et al. <br />  <br />  <br />  <br />  <br />                                                               FIGURE 4. Naf1p is required for accumulation of H/ACA snoRNAs+ <br />                                                               A: Northern analysis of snoRNA steady-state levels in cells progres- <br />                                                               sively depleted of Naf1p+ Total RNAs were extracted from wild-type <br />                                                               (lanes 1&#226;&#8364;&#8220;2) or GAL::ProtA-Naf1 (lanes 3&#226;&#8364;&#8220;8) strains following a shift <br />                                                               from galactose- to glucose-containing media+ Extracted RNAs were <br />                                                               separated by 6% acrylamide/urea gels and transferred onto a nylon <br />                                                               membrane+ Various snRNAs were detected with antisense oligo- <br />                                                               nucleotide probes+ B: Coprecipitation experiments with IgG agarose <br />                                                               beads were performed with lysates from cells expressing Prot A <br />                                                               epitope-tagged Naf1p (Naf1-TAP) and wild-type isogenic control strain <br />                                                               (WT)+ Immunoprecipitations were performed in 150 mM (lanes 2&#226;&#8364;&#8220;3) <br />                                                               or 500 mM (lanes 4&#226;&#8364;&#8220;5) NaCl+ RNA was extracted from equivalent <br />                                                               amounts of the total (lanes T), supernatant (lanes S) and pellet (lanes <br />                                                               P) fractions and analyzed by northern blotting+ <br />  <br />  <br />  <br />  <br /> or shifted to glucose medium and resolved on agarose/            Northern analysis of low molecular weight RNAs <br /> formaldehyde gels (Fig+ 5D&#226;&#8364;&#8220;I) or acrylamide/urea gels         showed that later precursors on the 5+8S synthesis path- <br /> (Fig+ 5J&#226;&#8364;&#8220;L)+ Following transfer of the GAL::ProtA-naf1        way, the 7S and 6S pre-rRNAs, were not depleted <br /> strain to glucose medium, the mature 18S rRNA was             (Fig+ 5J), and the mature 5+8S was only mildly depleted <br /> depleted (Fig+ 5I) as were the 27SA2 (Fig+ 5D&#226;&#8364;&#8220;F) and          in the GAL::ProtA-naf1 strain on glucose medium <br /> 20S (Fig+ 5H) pre-rRNAs+ In contrast, the 35S pre-            (Fig+ 5H&#226;&#8364;&#8220;L)+ The increased signal for the 7S and 6S <br /> rRNA was strongly accumulated (Fig+ 5D) and the ab-           pre-rRNAs and the 5S RNA (Fig+ 5L) may be a conse- <br /> errant 23S RNA appeared (Fig+ 5D,E,H)+ The 23S RNA            quence of the depletion of the 18S rRNA, leading to <br /> originates from direct cleavage of the 35S pre-rRNA at        increased loading of other RNA species, or may reflect <br /> site A3 in the absence of prior cleavage at sites A0 , A1 ,   a mild delay in processing+ The increased signal seen <br /> and A2 (see Fig+ 5C)+ Much less depletion was seen for        in the wild-type strain between the 0 h (galactose) and <br /> the 27SB pre-rRNA (Fig+ 5F) or 25S rRNA (Fig+ 5G) in          the 24 h (glucose) samples is a consequence of nutri- <br /> the GAL::ProtA-naf1 strain on glucose medium+ A fur-          tional upshift increasing ribosome synthesis+ <br /> ther aberrant RNA was accumulated following Naf1p                The pre-rRNA defects seen in the GAL::naf1 <br /> deletion, which is predicted to extend from site D, the 39    strain on glucose medium reveal an inhibition of the <br /> end of the 18S rRNA, to site B2 , the 39 end of the 25S       early pre-rRNA cleavages at sites A0 , A1 , and A2 (see <br /> rRNA (labeled D-B2 in Fig+ 5D&#226;&#8364;&#8220;F)+ This RNA was de-            Fig+ 5B,C)+ Similar defects were observed following de- <br /> tected with probes 001, 003, 006 (Fig+ 5), and 002 (data      pletion of the box H/ACA snoRNAs snR10 and snR30 <br /> not shown)+                                                   (Tollervey, 1987; Morrissey &amp; Tollervey, 1993)+ The loss <br />                                     Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> H/ACA snoRNP synthesis                                                                                                                 1507 <br />  <br />  <br />  <br />  <br />        FIGURE 5. Naf1p is required for pre-rRNA processing+ A: Structure and processing sites of the 35S pre-rRNA+ This <br />        precursor contains the sequences for the mature 18S, 5+8S, and 25S, which are separated by the two internal transcribed <br />        spacers, ITS1 and ITS2, and flanked by the two external transcribed spacers, 59ETS and 39ETS+ The positions of the <br />        oligonucleotides probes utilized in northern hybridizations are indicated+ B: Pre-rRNA processing pathways+ In wild-type <br />        cells, the 35S pre-rRNA is first cleaved at site A0 , producing the 33S pre-rRNA; this molecule is rapidly cleaved at site A1 <br />        to produce 32S pre-rRNA+ 32S is cleaved at site A2 , releasing 20S and 27SA2 + 27SA2 is then processed via two alternative <br />        pathways+ It may be cut at site A3 to produce 27SA3 , which is then trimmed to site B1S , producing 27SBS + Alternatively, it <br />        can be processed to 27SBL by an as yet unknown mechanism+ 27SBS and 27SBL are matured to the 5+8S and 25S rRNAs <br />        following identical pathways+ Cleavage at site C2 generates the 7S and 26S pre-rRNAs+ The 7S pre-rRNA is digested 39 to <br />        59 to 6S pre-rRNA and then to the mature 5+8S rRNA+ The 26S pre-rRNA is digested 59 to 39 to the 25S9 pre-rRNA and <br />        then to the mature 25S rRNA+ For a review on pre-rRNA processing and the known processing enzymes see Venema <br />        and Tollervey (1999) and Kressler et al+ (1999)+ C: Structure of aberrant pre-rRNA processing intermediate 23S, which <br />        is generated from premature cleavage in the 35S pre-rRNA at site A3 in the absence of cleavage at sites A0 , A1 , and A2 + <br />        D&#226;&#8364;&#8220;L: Northern analysis of pre-rRNA processing+ Strains GAL-ProtA-naf1 and wild-type were grown at 30 8C in YPGal then <br />        shifted to YPD+ Cells were harvested at the indicated times and total RNAs were extracted+ Equal amounts of RNA (5 mg) <br />        were resolved on 1+2% agarose/formaldehyde gels (D&#226;&#8364;&#8220;I) and 6% acrylamide/urea gels (J&#226;&#8364;&#8220;L) and transferred to a nylon <br />        membrane+ The membranes were consecutively hybridized with the probes indicated in A+ The position of the mature rRNAs <br />        and pre-rRNAs are indicated+ <br />  <br />  <br /> of snR10 and snR30 from the GAL::ProtA-naf1 strain                        To test this, a one-step PCR technique was used to <br /> on glucose medium may underlie these processing de-                     replace the promoter of the chromosomal SNR10 gene <br /> fects+ However, the appearance of the D-B2 fragment                     with the regulated MET3 promoter (Colley et al+, 2000) <br /> was not seen on depletion of snR10 or snR30, and has                    in the GAL::ProtA-naf1 strain+ Correct integration was <br /> not been reported for strains lacking other box H/ACA                   confirmed by PCR and expression of snR10 was shown <br /> snoRNAs or snoRNP proteins, but may be a conse-                         to be subject to methionine repression (Fig+ 6A, com- <br /> quence of the global loss of the H/ACA snoRNAs+                         pare lanes 3 and 4)+ The abundance of snR10 in ga- <br />                                                                         lactose medium was slightly lower in the MET::snr10 ; <br />                                                                         GAL::ProtA-naf1 strain than in the wild type, very likely <br /> Naf1p is not required for specific <br />                                                                         due to lower efficiency of the MET3 promoter com- <br /> transcription of H/ACA snoRNAs <br />                                                                         pared to snoRNA promoters+ Even a strong GAL- <br /> The homology between Naf1p and the human tran-                          promoter underexpressed snR30 (Morrissey &amp; Tollervey, <br /> scription factor ALF suggested that it might function as                1993)+ The MET::snr10 ; GAL::ProtA-naf1 and wild-type <br /> a transcription factor specific for the promoters of in-                strains were grown without methionione, to induce <br /> dependently transcribed H/ACA snoRNA genes and                          snR10 expression, in galactose medium, and trans- <br /> the host gene for snR44+                                                ferred to glucose medium to repress Naf1p synthesis+ <br />                                     Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> 1508                                                                                                              A. Fatica et al. <br />  <br />                                                                         lanes 4 to 7)+ Accumulation of snR10 was therefore <br />                                                                         dependent on Naf1p even when a heterologous regu- <br />                                                                         lated promoter drove its transcription+ <br />                                                                            The host gene for snR44 is RPS22B and the level of <br />                                                                         the mRNA transcribed from this gene was also as- <br />                                                                         sessed by northern hybridization during Naf1p deple- <br />                                                                         tion (Fig+ 6B)+ The level of the RPS22B mRNA strain <br />                                                                         showed the effects of nutritional upshift following trans- <br />                                                                         fer to glucose medium, as expected for a ribosomal <br />                                                                         protein gene, and then decreased only slightly even <br />                                                                         after 24 h in glucose, by which time growth was se- <br />                                                                         verely reduced (Fig+ 6A and Fig+ 3A)+ In contrast, the <br />                                                                         levels of snR44 were decreased 6 h after transfer and <br />                                                                         were strongly reduced after 12 h in glucose (see Fig+ 4)+ <br />                                                                            From these results, we conclude that Naf1p acts af- <br />                                                                         ter transcription initiation+ <br />  <br />  <br />                                                                         Naf1p is not required for termination <br />                                                                         of snoRNA transcripts <br />                                                                         Naf1p shares structural features with Nrd1p, a pro- <br />                                                                         tein implicated in the synthesis of both snRNAs and <br />                                                                         snoRNAs (Steinmetz et al+, 2001)+ Strains carrying the <br />                                                                         nrd1-5, sen1-1, or rna15-2 mutations are defective in <br />                                                                         39-end formation and transcription termination on snRNA <br />                                                                         and snoRNA genes, leading to the appearance of read- <br />                                                                         through transcripts (Steinmetz et al+, 2001; Morlando <br />                                                                         et al+, 2002)+ We therefore tested for the presence of <br />                                                                         H/ACA snoRNA read-through transcription products in <br />                                                                         the GAL::ProtA-naf1 strain (Fig+ 6C)+ Primer extension <br />                                                                         analysis was performed with primers complementary <br />                                                                         to the mRNAs YCR015C and CWH36, which are tran- <br />                                                                         scribed from genes that lie immediately downstream of <br />                                                                         SNR33 and SNR43, respectively (see schematic rep- <br />                                                                         resentation in Fig+ 6C)+ As positive control, primer ex- <br /> FIGURE 6. Naf1p is not specifically required for transcription or 39- <br /> end formation on H/ACA snoRNA genes+ A: Northern analysis of <br />                                                                         tension analyses were performed on RNA extracted <br /> snoRNA steady-state levels in wild-type (lanes 1&#226;&#8364;&#8220;2) and GAL::ProtA-     from a sen1-1 strain 1 h after transfer to 37 8C (Fig+ 6C, <br /> naf1; MET::snR10 (lanes 3&#226;&#8364;&#8220;7) strains+ Cells were grown in galactose     lane 2), in which read-through transcripts were readily <br /> medium and shifted to glucose medium for the time indicated+ Total <br /> RNA was extracted from cells grown in selective minimal medium <br />                                                                         detected (Steinmetz et al+, 2001)+ No read-through tran- <br /> lacking methionine with the exception of lane 3, in which methionine    scripts were detected in the wild-type strain (Fig+ 6C, <br /> was added to a final concentration of 20 mM+ RNAs were resolved on      lane 1) or in the GAL::ProtA-naf1 strain either grown in <br /> 6% acrylamide/urea gel, transferred to a nylon membrane, and hy- <br /> bridized with oligonucleotides specific for the H/ACA snoRNAs snR10 <br />                                                                         galactose medium or 24 h after transfer to glucose <br /> and snR8, and the box C/D snoRNA snR13+ B: Accumulation of              (Fig+ 6C, lanes 3&#226;&#8364;&#8220; 4)+ The pol III-transcribed U6 snRNA <br /> RPS22B mRNA in Naf1p-depleted cells+ The membrane utilized for          was used as a normalization control+ We conclude that <br /> the hybridization is the same as in Figure 5+ C: Oligonucleotides <br /> complementary to the coding strand of the YCR015C and CWH36 <br />                                                                         Naf1p is not required for transcription termination on <br /> genes were utilized for a primer extension analysis on total RNA        H/ACA snoRNA genes+ <br /> extracted from wild-type (lane 1), sen1-1 (lane 2), and GAL::ProtA- <br /> naf1 (lanes 3&#226;&#8364;&#8220; 4) strains+ The sen1-1 strain were grown at 25 8C and <br /> then shifted to 37 8C for 1 h+ GAL::ProtA - naf1 strain was grown in    Naf1p binds the H/ACA snoRNP proteins <br /> galactose medium and shifted to glucose medium for 24 h+ The U6 <br /> snRNA was utilized as a normalization control+ As schematically rep- <br />                                                                         Nhp2p and Cbf5p <br /> resented above the gels, the extended products correspond to an <br /> elongated RNA starting at the 59 end of the snoRNA transcripts+ <br />                                                                         During the course of this work, interactions between <br />                                                                         Naf1p and two H/ACA snoRNP proteins were reported <br />                                                                         from high-throughput screens; Naf1p was reported to <br /> Northern hybridization showed that the H/ACA snoRNAs                    interact with Cbf5p and Nhp2p in a two-hybrid screen <br /> snR10 and snR8 were strongly, and apparently equally,                   (Ito et al+, 2001) and was copurified with overexpressed <br /> reduced on depletion of Naf1p, whereas no depletion                     FLAG-tagged Cbf5p in a proteomic analysis (Ho et al+, <br /> was seen for the box C/D snoRNA snR13 (Fig+ 6A,                         2002)+ To confirm these putative interactions and de- <br />                                   Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> H/ACA snoRNP synthesis                                                                                                           1509 <br />  <br /> termine whether they are mediated by direct contacts,               snR36 (Fig+ 7B)+ Several retarded complexes were ob- <br /> coprecipitation assays were performed in vitro (Fig+ 7A)+           tained with GST-Naf1p (Fig+ 7B, lanes 2&#226;&#8364;&#8220; 4) but not with <br /> A bacterially expressed GST&#226;&#8364;&#8220;Naf1p fusion protein was                GST alone (Fig+ 7B, lane 5), indicating that the protein <br /> immobilized on glutathione beads+ These were then                   can bind directly to RNA+ The multiple complexes formed <br /> incubated with in vitro translated, [ 35 S]methionine la-           between Naf1p and snR36 suggests that in our exper- <br /> beled Nhp2p and Cbf5p (Fig+ 7A, lane 1)+ GST-Naf1p                  imental condition, the protein can interact with several <br /> specifically retains Nhp2p and Cbf5p (Fig+ 7A, lane 3)              binding sites in the snoRNA+ To investigate the speci- <br /> whereas no retention was observed using GST alone                   ficity of the RNA binding, we carried out competition <br /> (Fig+ 7A, lane 2)+ This result confirms the reported in             experiments using a 200-fold molar excess of un- <br /> vivo interactions, and shows that they are not depen-               labeled RNAs (Fig+ 7C)+ Both snR36 and its antisense <br /> dent on intermediary proteins+                                      transcript were able to efficiently compete for the bind- <br />                                                                     ing (Fig+ 7C, lanes 3&#226;&#8364;&#8220;5)+ In contrast, no competition <br />                                                                     was observed with the snRNA U6 and tRNAs (Fig+ 7C, <br /> Naf1p is an RNA-binding protein <br />                                                                     lanes 4&#226;&#8364;&#8220;6)+ These data indicate that GST-Naf1p prob- <br /> The potential RNA-binding activity of Naf1p was as-                 ably recognizes structural features more than specific <br /> sessed by bandshift analyses using recombinant GST-                 sequences during RNA binding in vitro+ Similar observa- <br /> tagged Naf1p and [ 32 P]-labeled in vitro-transcribed               tions have been made for the box H/ACA protein Nhp2p, <br />  <br />  <br />  <br />  <br />        FIGURE 7. Naf1p physically interacts with Cbf5p, Nhp2p, and RNA+ A: In vitro-translated 35 S-labeled Cbf5p and Nhp2p <br />        (lane 1) were individually utilized in GST pull-down experiments with recombinant GST (lane 2) or GST-Naf1p protein <br />        (lane 3)+ Proteins recovered from each experiment were analyzed by SDS-PAGE+ B: Gel mobility shift assay performed with <br />        the in vitro-transcribed H/ACA snoRNA snR36+ GST-Naf1p or GST purified from E. coli were used at the concentration <br />        indicated+ C: Competition experiment+ Gel shift assay was performed in the presence of 200-fold molar excess of the <br />        following competitor RNAs: snR36 (lane 3), U6 snRNA (lane 4), antisense RNA of snR36 (lane 5), tRNA (lane 6)+ Reactions <br />        in lanes 2&#226;&#8364;&#8220;6 contained GST-Naf1p at a concentration of 10 mM+ Complexes were resolved by electrophoresis on native 6% <br />        acrylamide/bisacrylamide (80:1) gels+ <br />                                 Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> 1510                                                                                                            A. Fatica et al. <br />  <br /> which binds to imperfect stem-loop structures in vitro, <br /> rather than to specific sequences (Henras et al+, 2001)+ <br />  <br /> Naf1p binds to the CTD of RNA <br /> polymerase II <br /> Many hnRNP-like proteins, including Nrd1p, bind to the <br /> C-terminal domain of the largest subunit of RNA poly- <br /> merase II+ Two-hybrid screens have previously been <br /> used to assess the interactions between the CTD and <br /> RNA processing factors including Rna15p, Pcf11p, and <br /> Nrd1p (Yuryev et al+, 1996; Barilla et al+, 2001)+ The <br /> potential in vivo association of Naf1p and the CTD was <br /> assessed using a yeast two-hybrid system previously <br /> used to show the interaction of the cleavage and poly- <br /> adenylation factors Pcf11p and Yhh1p with the CTD <br /> (Fig+ 8A; Barilla et al+, 2001; B+ Dichtl &amp; W+ Keller, pers+ <br /> comm+)+ The CTD of the yeast RNA polymerase II was <br /> expressed as a fusion with the GAL4 DNA-binding do- <br /> main, whereas the full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Naf1p sequence was ex- <br /> pressed as a fusion with the transcription activation <br /> domain (see Materials and Methods)+ Interaction be- <br /> tween these fusion proteins activates transcription of <br /> both the lacZ and HIS3 genes+ Coexpression of the <br /> CTD and Naf1p-fusion constructs robustly stimulated <br /> expression of b-galactosidase as judged by a filter color <br /> development assay (data not shown) or in enzymatic <br /> activity in cell extracts (Fig+ 8A) compared to control <br /> strains expressing the constructs individually+ To test <br /> for expression of HIS3, we assessed the ability of the <br /> strains to grow in the absence of exogenous histidine+ <br />                                                                FIGURE 8. A: Quantitative liquid b-galactosidase assays for the two- <br /> Substantially better growth was seen for the strain ex-        hybrid interaction between the CTD and Naf1p+ Mean values are <br /> pressing both the binding domain-CTD fusion and the            shown in the graph+ B: Naf1p associates with phosphorylated CTD+ <br /> activation domain Naf1p fusion than for the control            Whole-cell extract from the Naf1-TAP strain was utilized in GST <br />                                                                pull-down experiments with GST (lanes 2, 5), GST-CTD (lanes 3, 6), <br /> strains lacking either fusion construct (data not shown)+      and phosphorylated GST-CTD (lanes 4, 7)+ The yeast extract (2% of <br />     To confirm this interaction, we performed a GST-           total input; lane 1), bound fractions (100%, lanes 2&#226;&#8364;&#8220; 4), and super- <br /> CTD pull-down experiment on cell extracts from the             natant fractions (10%; lanes 5&#226;&#8364;&#8220;7) were loaded onto a SDS-PAGE <br />                                                                and analyzed by western blot with rabbit anti-Protein A antibody and <br /> strain expressing TAP-tagged Naf1p (Fig+ 8B)+ Gluta-           a mouse monoclonal anti-Nop1p+ <br /> thione affinity beads were prepared containing immo- <br /> bilized GST (Fig+ 8B, lanes 2 and 5), nonphosphorylated <br /> GST-CTD (Fig+ 8, lanes 3 and 6), or phosphorylated <br /> GST-CTD (Fig+ 8, lanes 4 and 7)+ Yeast whole extract           therefore tested for its association with the GST-CTD+ <br /> was incubated with the beads and, after extensive wash-        Under these conditions, Naf1p did not show clear as- <br /> ing, the bound material was recovered and analyzed             sociation with the phosphorylated or nonphosphory- <br /> by western blotting+ Anti-ProtA antibodies were utilized       lated CTD (data not shown)+ Similar behavior has been <br /> to detect Naf1-TAP and anti-Nop1p antibodies to de-            reported for Rna15p (Barilla et al+, 2001), and this sug- <br /> tect Nop1p, as negative control+ Naf1-TAP was re-              gests that the interaction between CTD and Naf1p may <br /> tained on the beads carrying the phosphorylated CTD,           be indirect+ <br /> but was not retained by the nonphosphorylated CTD or <br /> the control GST+ The efficiency of Naf1p precipitation <br />                                                                DISCUSSION <br /> was low, but comparable to that reported for Pcf11p, <br /> which other assays show to interact with the CTD (Ba-          Based on sequence homology with the H/ACA snoRNP <br /> rilla et al+, 2001)+                                           protein Gar1p, we identified a novel essential protein <br />     The yeast two-hybrid analysis and the GST pull-            that we termed Naf1p+ The results presented in this <br /> down from cell extracts do not discriminate between a          study show that Naf1p is required for a posttranscrip- <br /> direct association and indirect interactions, involving        tional step in box H/ACA snoRNP synthesis+ Naf1p is <br /> one or more partners+ In vitro-translated Naf1p was            not a component of the major box H/ACA snoRNP <br />                                 Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> H/ACA snoRNP synthesis                                                                                              1511 <br />  <br /> population but is required for the accumulation of all      which are homologous to ribosomal proteins Rpl7p and <br /> tested box H/ACA snoRNAs+ Four proteins, Gar1p,             Rpl24p, respectively, are present in pre-60S ribosomal <br /> Cbf5p, Nhp2p, and Nop10p, are stable components of          particles but absent from mature ribosomes (Dunbar <br /> the box H/ACA snoRNPs (Kiss, 2002)+ However, al-            et al+, 2000; Saveanu et al+, 2001)+ <br /> though Cbf5p, Nhp2p, and Nop10p are each required              Human H/ACA snoRNPs can be assembled in vitro <br /> for box H/ACA snoRNA accumulation, Gar1p is dis-            (Dragon et al+, 2000), but the in vivo assembly pathway <br /> pensable for snoRNA accumulation (Girard et al+, 1992;      is likely to involve additional factors (Terns &amp; Terns, <br /> Henras et al+, 1998; Lafontaine et al+, 1998)+ The H/ACA    2001)+ In human cells, and probably most other eukary- <br /> snoRNP proteins are reported to associate with the          otes, assembly of small nuclear and small nucleolar <br /> snoRNA precursors in strains in which processing is         RNP complexes is likely to be mediated by the SMN <br /> blocked (Henras et al+, 1998) suggesting they, and          complex (reviewed in Terns &amp; Terns, 2001)+ S. cerevi- <br /> Naf1p, bind to the snoRNAs at an early stage of H/ACA       siae lacks clear homologs of the SMN complex com- <br /> snoRNP assembly, possibly during transcription+             ponents, but we propose that Naf1p fulfils a function in <br />    Naf1p-related proteins are widespread in eukary-         box H/ACA snoRNP assembly that is in some way anal- <br /> otes, indicating that they perform a conserved function+    ogous+ The region conserved between Naf1p and Gar1p <br /> The family contains a region that is clearly related to     has a predicted fold that is similar to that of the tudor <br /> the RNA-binding domain of yeast Gar1p and its homo-         domain of SMN (Selenko et al+, 2001) and the con- <br /> logs+ The functional significance of this homology is       served core of the Sm proteins (Kambach et al+, 1999)+ <br /> supported by the in vitro RNA-binding activity of Naf1p+    The tudor domain of human SMN binds a repeat do- <br /> During the course of these analyses, Naf1p was re-          main rich in glycine and dimethylated arginine (GAR or <br /> ported to interact with the box H/ACA snoRNP proteins       RGG domain) that is present in Gar1p as well as the <br /> Cbf5p and Nhp2p in a high-throughput two-hybrid screen      box C/D snoRNP protein fibrillarin and the Sm protein <br /> (Ito et al+, 2001), and was copurified with overexpressed   components of the spliceosomal snRNPs (Fischer et al+, <br /> FLAG-tagged Cbf5p in a proteomic analysis (Ho et al+,       1997; Buhler et al+, 1999; Friesen &amp; Dreyfuss, 2000; <br /> 2002)+ Here we show that Naf1p binds directly Cbf5p         Brahms et al+, 2001; Jones et al+, 2001; Pellizzoni et al+, <br /> and Nhp2p in vitro, and also interacts with the CTD of      2001a; Selenko et al+, 2001)+ In the case of the Sm <br /> RNA pol II+ We therefore propose that Naf1p associ-         proteins, this interaction predominately involves the rec- <br /> ates with the CTD of RNA pol II and binds to newly          ognition of symmetrical arginine dimethylation (sDMA; <br /> synthesized or nascent box H/ACA snoRNAs+ Naf1p             Brahms et al+, 2001)+ Whether yeast proteins contain <br /> then either recruits Nhp2p and Cbf5p to the snoRNP or       sDMA has not been determined, but S. cerevisiae Hsl7p <br /> binds together with them as a preformed complex+ The        (Ybr133p) is a functional homolog of human JPB1, the <br /> low specificity observed for in vitro RNA binding with      methyltransferase component of the 20S methylosome <br /> Naf1p alone and the fact that coprecipitation was not       complex that is responsible for this modification (Lee <br /> detected between the CTD and recombinant Naf1p              et al+, 2000; Friesen et al+, 2001; Meister et al+, 2001)+ <br /> alone suggests that the latter may be the case+             Additionally, in the nucleus, the SMN complex is phys- <br />    In view of the sequence and predicted structural ho-     ically and functionally associated with RNA pol II (Pel- <br /> mology between Naf1p and the RNA-binding domain             lizzoni et al+, 2001b), and SMN can bind directly to the <br /> of Gar1p, we propose that they bind to the same re-         U1 snRNA, an activity that is required for snRNP as- <br /> gions of the snoRNAs+ Once the snoRNP structure is          sembly (Yong et al+, 2002)+ <br /> established, Gar1p may replace Naf1p, as the former is         There are, therefore, some apparent similarities be- <br /> a major component of the mature snoRNPs, unlike             tween Naf1p and SMN; both bind, directly or indirectly, <br /> Naf1p itself+ It may well be that displacement of Naf1p     to RNA polymerase II, have a region with an Sm fold, <br /> requires the completion of later snoRNA processing          and are implicated in box H/ACA snoRNP assembly+ <br /> events+ These may involve the putative ATPase p50/          However, humans and other eukaryotes have a protein <br /> Rvb2p that has been implicated in the formation of both     that is more closely related to Naf1p and more likely to <br /> C/D and H/ACA snoRNPs, and depletion of which re-           be a direct homolog+ Whether and how hNaf1p and <br /> sults in delocalization of Gar1p (King et al+, 2001)+ In    SMN interact during snoRNP assembly remains to be <br /> the absence of correct initial assembly with Naf1p and      determined+ <br /> the snoRNP proteins, the snoRNAs are presumably <br /> rapidly degraded+ The activity responsible for this deg- <br /> radation has not been determined, but the exosome           MATERIALS AND METHODS <br /> complex, which degrades other defective nuclear RNA <br /> precursors (Bousquet-Antonelli et al+, 2000; Torchet        Strains and microbiological techniques <br /> et al+, 2002), appears a likely candidate+                  Standard techniques were employed for growth and handling <br />    It is notable that there are other known precursor-      of yeast+ Yeast strains used in this work are: CYM1333 (a, <br /> specific factors with homology to RNA-binding proteins      trp1-D, his3-D, ura3-52, lys2-801, ade2-101), YAF9 (a, trp1-D, <br /> that are present in mature RNPs+ Rlp7p and Rlp24p,          his3-D, ura3-52, lys2-801, ade2-101, HIS3::GAL::ProtA-Naf1), <br />                                  Downloaded from www.rnajournal.org on February 14, 2006 <br />  <br />  <br /> 1512                                                                                                           A. Fatica et al. <br />  <br /> YAF10 (a, ade2, his3, leu2, trp1, ura3, NAF1::TAP::TRP1),         coupled to FITC (Sigma) at a 1:100 and a 1:200 dilution, re- <br /> YAF15 (a, ade2, his3, leu2, trp1,ura3, NAF1::TAP::TRP1            spectively+ To stain nuclear DNA, DAPI was included in the <br /> [pUN100 Dsred-Nop1, LEU2]), YAF25 (a, trp1-D his3-D,              mounting medium (Vectashield, Vector Laboratories)+ Cells <br /> ura3-52,lys2-801,ade2-101, HIS3::GAL::ProtA-Naf1,TRP1             were viewed on a Zeiss Axioscop fluorescence microscope and <br /> ::pMET::snR10), and Y190 (MATa ura3-52, trp1-901, ade2-           pictures were obtained with a Xillix Microimager CCD camera+ <br /> 101, leu2-3, 112 his3-200r gal4D gal80D, URA3::GAL1-lacZ, <br /> LYS2::GAL1-HIS3 cyhr)+ The YAF9 and YAF25 strains were <br /> created by use of a one-step-PCR strategy as previously           In vitro protein-binding assay <br /> described (Lafontaine &amp; Tollervey, 1996; Colley et al+, 2000)+ <br />                                                                   PCR fragment corresponding to the YNL124W (NAF1) open <br /> TAP tagging of Naf1p was performed as described in Rigaut <br />                                                                   reading frame (ORF) was amplified by genomic DNA and <br /> et al+ (1999)+ <br />                                                                   cloned into the pGEX-2T vector (Pharmacia) to obtain the <br />                                                                   pGEX-NAF1 vector+ NHP2 and CBF5 ORFs were ampli- <br /> Oligonucleotides                                                  fied and cloned into the Bluescript KS vector (Stratagene) <br />                                                                   to generate the BS-NHP2 and BS-CBF5 vectors+ The <br /> Oligonucleotides anti-snR10, anti-snR30, anti-snR41, anti-        [ 35 S]methionine-labeled proteins were produced by an in vitro <br /> snR42, anti-snR36, anti-snR11, anti-snR43, anti-snR3, anti-       coupled transcription&#226;&#8364;&#8220;translation reaction (Promega) in the <br /> snR44, anti-snR13, anti-snR190, anti-U3, anti-u14, anti-U4,       presence of [35S]methionine (Amersham)+ All the gluta- <br /> and anti-scR1 were previously described (Chanfreau et al+,        thione S-transferase (GST) fusion proteins were expressed <br /> 1998; Lafontaine &amp; Tollervey, 1999)+ Oligonucleotides used        from the GST expression vector pGEX-2T (Pharmacia) in the <br /> for pre-rRNAs analysis were described in Fatica et al+ (2002)+    Escherichia coli strain BL21 and purified by using glutathione- <br /> Oligonucleotides utilized for primer extension analysis were:     Sepharose according to the manufacturer&#226;&#8364;&#8482;s protocol (Phar- <br /> anti-YCR015, CTGAAATAATAATGGTTTTC, anti-CWH36,                    macia)+ GST or GST&#226;&#8364;&#8220;Naf1p was incubated with 25 mL bed <br /> TACGTTCCTTTGCCAGCTCCAT, and antiU6, AAAACGAA                      volume of glutathione&#226;&#8364;&#8220;Sepharose beads in 300 mL of NET <br /> ATAAATTCTTTGTAAAAC+                                               buffer (20 mM Tris, pH 8, 100 mM NaCl, 0+2 mM PMSF) for <br />                                                                   30 min at 4 8C+ After extensive washing in 1 M NaCl and then <br />                                                                   in 100 mM NaCl, [ 35 S]methionine-labeled Nhp2p or Cbf5p <br /> RNA extraction, northern hybridization,                           was incubated with GST and GST&#226;&#8364;&#8220;Naf1p prebound beads+ <br /> and primer extension                                              Incubation was allowed to proceed for 30 min at 4 8C+ Beads <br />                                                                   were treated with RNase A and washed four times with NET <br /> For depletion of the Naf1p protein, cells were harvested at <br />                                                                   buffer+ Bound proteins were then recovered by heating for <br /> intervals following a shift from RSG medium (2% galactose, <br />                                                                   4 min at 95 8C in loading buffer and analyzed by SDS-PAGE+ <br /> 2% sucrose, 2% raffinose), or YPGal medium containing 2% <br />                                                                   [ 35 S]methionine-labeled proteins were detected by fluorog- <br /> galactose, to YPD medium containing 2% glucose+ Other- <br />                                                                   raphy+ GST-CTD was phosphorylated for 2 h at 30 8C with <br /> wise strains were grown in YPD medium+ RNA was extracted <br />                                                                   casein kinase I (New England Biolabs) according to the manu- <br /> as described previously (Fatica et al+, 2002)+ Northern hy- <br />                                                                   facturer&#226;&#8364;&#8482;s protocol+ GST pull-down experiments were per- <br /> bridization and primer extension were as described (Stein- <br />                                                                   formed as described above, but SDS/PAGE gels were <br /> metz et al+, 2001; Fatica et al+, 2002)+ Standard 1+2% agarose/ <br />                                                                   transferred to nitrocellulose membranes and analyzed by im- <br /> formaldehyde and 6% acrylamide/urea gels were used to <br />                                                                   munoblotting+ TAP-Naf1p and Nop1p were detected with a <br /> analyze the high and low molecular weight RNA species, <br />                                                                   1:100 dilution of a rabbit anti-Protein A antibody (Sigma) and <br /> respectively+ <br />                                                                   a mouse monoclonal anti-Nop1p, respectively+ A 1:200 dilu- <br />                                                                   tion of secondary peroxidase-conjugated antibodies (Sigma) <br /> Immunoprecipitation of ProtA                                      was utilized for the detections+ <br /> epitope-tagged Naf1p <br /> YAF10 and wild-type strains were utilized for the immunopre-      Mobility shift assays <br /> cipitation experiments+ Immunoprecipitations were performed <br />                                                                   In vitro transcription of RNAs was performed following stan- <br /> on IgG-agarose beads as previously described (Colley et al+, <br />                                                                   dard procedures with templates obtained by genomic PCR <br /> 2000)+ RNA from the total, supernatant, and pellet fractions <br />                                                                   amplification and carrying the T7 promoter+ The binding re- <br /> were resolved on a 6% acrylamide/urea gel and analyzed by <br />                                                                   action was carried out in binding buffer (30 mM Tris-HCl, <br /> Northern blot analysis+ <br />                                                                   pH 7+4, 150 mM KCl, 2 mM MgCl2 , 0+1% Triton X-100, 20% <br />                                                                   glycerol) in the presence of 1 mM DTT, tRNA (1 mg/mL), 10 <br /> Fluorescence microscopy                                           fmol of 32 P-labeled RNA, and 0&#226;&#8364;&#8220;20 mM of recombinant pro- <br />                                                                   tein in a reaction volume of 10mL+ Labeled RNA was heat <br /> Indirect fluorescence was performed in the YAF10 strain as        denatured at 65 8C for 10 min, followed by slow cooling to <br /> previously described (Grandi et al+, 1993)+ Plasmid pUN100        room temperature, and then added to the binding reaction+ <br /> DsRed-Nop1 was introduced into yeast cells by transforma-         For competition experiments, a 200-fold molar excess of cold <br /> tion, and selected on SD-URA medium+ Individual transfor-         competitor RNA was added+ The reactions were incubated at <br /> mants were grown in selective medium, fixed by incubation in      room temperature for 30 min and then loaded on a 6% native <br /> 4% (v/v) formaldehyde for 30 min at 25 8C, and sphero-            acrylamide/bisacrylamide (80:1), 4% glycerol gels in 0+53 <br /> plasted+ TAP fusion was detected with a rabbit anti-Protein A     TBE buffer+ Prior to loading, the gels were prerun for 1 h and <br /> antibody (Sigma) and a secondary goat anti-rabbit antibody        then run for 3 h at 250 V in the cold room+ <br />  <br />  <br /> Sequence analysis and threading                                 <br /> Initial sequence searches were done on a yeast proteome                    <br /> using FASTA (Pearson &amp; Lipman, 1988) with default param-          <br /> eters+ Subsequent PSI-BLAST searches (Altschul et al+, 1997)            <br /> of the nonredundant protein database were done with E 5             <br /> 0+01 as the inclusion threshold for further iterations+                <br />    Secondary structure predictions were carried out using                <br /> position-specific scoring matrices obtained from PSI-BLAST           <br /> searches (Jones, 1999)+ Predictions for each residue were         <br /> assigned the confidence on a scale of 0&#226;&#8364;&#8220;9, and only predic-                 <br /> tions with confidence 7 or higher were used for threading+ In            <br /> retrospect, the secondary structure predictions of Gar1p and          <br /> Naf1p alone were indicative of their potential relatedness to         <br /> Sm proteins because they share the same <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span>: an                     <br /> N-terminal a-helix followed by five b-strands (Kambach et al+,         <br /> 1999)+                                                                <br />    THREADER (Jones et al+, 1992) was used for threading             <br /> calculations with an updated version of the fold library+ The <br /> initial calculations were always done using the sequence                      <br /> information alone+ Additional runs were done by taking into            <br /> account the sequence and confidently predicted parts of                <br /> the secondary structure (Jones, 1999)+ Interesting hits were            <br /> sorted either based on their cumulative Z-scores or individ-                 <br /> ual pairwise energies+ In both cases, three to five Sm pro-             <br /> teins (Kambach et al+, 1999), including structurally related                 <br /> SMN protein (Selenko et al+, 2001), were among the top                       <br /> 10 matches+                                                              <br />   <br /> </body></html>