<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Genetic Interactions of Yeast Eukaryotic Translation Initiation Factor 5A (eIF5A) <br /> Reveal Connections <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> Poly(A)-Binding Protein and Protein Kinase C Signaling <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 The highly conserved eukaryotic translation initiation factor eIF5A has been proposed to have various <br />              roles in the cell, <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> translation to mRNA decay to nuclear protein export. To further our understanding <br />              of this essential protein, three temperature-sensitive alleles of the yeast TIF51A gene have been character- <br />              ized. Two mutant eIF5A proteins contain mutations in a proline residue at the junction between the two <br />              eIF5A domains and the third, strongest allele encodes a protein with a single <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> in each domain, <br />              both of which are required for the growth defect. The stronger tif51A alleles cause defects in degradation <br />              of short-lived mRNAs, supporting a role for this protein in mRNA decay. A multicopy suppressor screen <br />              revealed six genes, the overexpression of which allows growth of a tif51A-1 strain at high temperature; <br />              these genes include PAB1, PKC1, and PKC1 regulators WSC1, WSC2, and WSC3. Further results suggest <br />              that eIF5A may also be involved in ribosomal synthesis and the WSC/PKC1 signaling pathway for cell wall <br />              integrity or related processes. <br />  <br />  <br />  <br />  <br /> R   EGULATION of the fate of cytoplasmic mRNA in- <br />       volves a delicate balance between its recruitment <br /> to ribosomes for translation and its degradation by <br />                                                                              eIF5A being an initiation factor for general protein syn- <br />                                                                              thesis. <br />                                                                                 Alternatively, it was suggested that eIF5A could be <br /> nucleolytic enzymes. As researchers have studied sepa-                       involved in the translation of a specific subset of mRNAs, <br /> rately the factors that are important for translation and                    for example, those involved in the cell cycle progression <br /> those that are required for mRNA turnover, it has be-                        (G1/S transition; Park et al. 1993, 1997). Expression of <br /> come clear that these two processes are intimately                           eIF5A has also been correlated with cell proliferation: <br /> linked. eIF5A is an enigmatic protein that has been                          an increase in G1-arrested cells is observed after deple- <br /> implicated in several steps of RNA metabolism including                      tion of this factor in yeast (Kang and Hershey 1994). <br /> both translation and mRNA degradation.                                       Conversely, eIF5A expression is induced in activated <br />   eIF5A is a highly conserved protein encoded in the                         human T lymphocytes (Bevec et al. 1994). <br /> genomes of eukaryotes and archaebacteria (Chen and                              The yeast Saccharomyces cerevisiae contains two 90% <br /> Liu 1997). Yeast and mammalian eIF5A proteins are                            identical genes encoding eIF5A, TIF51A (HYP2), and <br /> 63% identical, suggesting the importance of this protein                     TIF51B (HYP1) (Schnier et al. 1991), and each of these <br /> in basic cellular processes (Schnier et al. 1991). Origi-                    proteins undergoes two post-translational modifications. <br /> nally purified from ribosomes of rabbit reticulocyte ly-                     The N-terminal acetylated serine residue of eIF5A is <br /> sates (Kemper et al. 1976), eIF5A was described as a                         phosphorylated (Kang et al. 1993; Klier et al. 1993), <br /> translation initiation factor due to its ability to stimulate                and mutational analysis revealed that unphosphorylated <br /> the synthesis of methionyl-puromycin in vitro (Benne                         eIF5A is sufficient to support growth of a tif51A&#226;&#338;&#172; haploid <br /> and Hershey 1978; Park et al. 1993). However, deple-                         strain (Klier et al. 1993), demonstrating that phosphor- <br /> tion of this factor in yeast caused only a small (30%)                       ylation is not essential for eIF5A function in vivo. The <br /> reduction in the protein synthesis rate (Kang and Her-                       second modification is hypusination of a specific lysine <br /> shey 1994). These results have been used to argue against                    (Chen and Liu 1997; Park et al. 1997). Hypusination <br />                                                                              of eIF5A is essential in yeast: strains in which hypusina- <br />                                                                              tion is blocked, by mutation of the target lysine (K51R) <br />   1 <br />                                                                              or deletion of the deoxyhypusine synthase gene, are <br />    Corresponding author: Dana-Farber Cancer Institute, 44 Binney St., <br /> SM922, Boston, MA 02115. E-mail: pamela_silver@dfci.harvard.edu <br />                                                                              inviable (Schnier et al. 1991; Sasaki et al. 1996; Park <br />   2 <br />    Present address: Department of Biology, Bowdoin College, Bruns-           et al. 1998). Similarly, inhibitors of hypusination block <br /> wick, ME 04011.                                                              the proliferation of different mammalian cell lines (Han- <br />  <br /> Genetics 160: 393&#226;&#8364;&#8220;405 (February 2002) <br />  394                                                  S. R. Valentini et al. <br />  <br />                                                           TABLE 1 <br />                                                Yeast strains used in this study <br />  <br />         Strain                                         Genotype                                               Source <br />         W303a         MATa ade2-1 trp1-1 ura3-1 leu2-3,112 his3-11,15 can1-100                            A. Tzagoloff <br />         W303&#226;&#65533;&#163;         MAT&#226;&#65533;&#163; ade2-1 trp1&#226;&#338;&#172;63 ura3-1 leu2-3,112 his3-11,15 can1-100                           A. Tzagoloff <br />         FY23          MATa ura3-52 trp1&#226;&#338;&#172;63 leu2&#226;&#338;&#172;1                                                         F. Winston <br />         DL376         MATa trp1-1 ura3-52 leu2-3,112 his4 can1 r pkc1&#226;&#338;&#172;::LEU2                              D. Levin <br />         xpo1-1        MATa ade2-1 trp1-1 ura3-1 leu2-3,112 his3-11,15 can1-100 xpo1::LEU2 &#226;&#171;&#185;               K. Weis <br />                        pKW457 (xpo1-1 HIS3) <br />         PSY8          MATa ade2-1 trp1-1 ura3-1 leu2-3,112 his3-11,15 can1-100 cyt1::HIS3 tif51A-1        A. Chiang <br />         PSY338        MAT&#226;&#65533;&#163; ade2-1 trp1-1 ura3-1 leu2-3,112 his3-11,15 can1-100 cyt1::HIS3 tif51A-2        A. Chiang <br />         PSY341        MATa ade2-1 trp1-1 ura3-1 leu2-3,112 his3-11,15 can1-100 cyt1::HIS3 tif51A-3        A. Chiang <br />         PSY1241       MATa ade2-1 trp1-1 leu2-3,112 his3-11 can1-100 TIF51A::URA3                         This study <br />         PSY1242       MAT&#226;&#65533;&#163; ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 cyt1::HIS3 tif51A-1        This study <br />         PSY1243       MAT&#226;&#65533;&#163; ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 cyt1::HIS3 tif51A-2        This study <br />         PSY1244       MAT&#226;&#65533;&#163; ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 cyt1::HIS3 tif51A-3        This study <br />         PSY1245       MAT&#226;&#65533;&#163; ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 tif51A-1                   This study <br />         PSY1248       MATa ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 tif51A-2                   This study <br />         PSY1249       MATa ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 tif51A-3                   This study <br />         PSY1250       MATa/&#226;&#65533;&#163; ura3-52/- leu2&#226;&#338;&#172;1/- his3&#226;&#338;&#172;200/- trp1&#226;&#338;&#172;63/&#226;&#171;&#185;                                      This study <br />         PSY1251       MAT&#226;&#65533;&#163; ura3-52 leu2&#226;&#338;&#172;1 his3&#226;&#338;&#172;200 tif51a::HIS3 [pPS1592]                                 This study <br />         PSY1252       MATa ade2-1 trp1-1 ura3-1 his3-11,15 leu2-3,112 can1-100 cyt1::HIS3 tif51A-1        This study <br />  <br />  <br />  <br /> auske-Abel et al. 1994; Park et al. 1994; Chen et al. 1996;        proteins and their roles in RNA metabolism, including <br /> Shi et al. 1996a). eIF5A is the only eukaryotic protein            translation, mRNA decay, and ribosome biogenesis. <br /> known to be hypusinated. <br />   Although eIF5A has been suggested to participate in <br /> the nucleocytoplasmic trafficking of the HIV-1 Rev pro-                           MATERIALS AND METHODS <br /> tein/RRE complex (Ruhl et al. 1993; Bevec et al. 1996;                Yeast strains, growth conditions, and plasmids: S. cerevisiae <br /> Bevec and Hauber 1997; Liu et al. 1997; Hofmann et                 strains and their genotypes used in this work are listed in <br /> al. 2001), other studies do not agree on such a role for           Table 1. Procedures for cell growth and genetic manipulations <br /> eIF5A (Henderson and Percipalle 1997; Shi et al.                   were according to standard protocols (Rose et al. 1990). Plas- <br /> 1997; Lipowsky et al. 2000). Localization studies also             mids used in this work were obtained as described below and <br />                                                                    are listed in Table 2. Cloning by PCR was performed with Pfu <br /> do not support the involvement of this protein in the              DNA polymerase (Stratagene, La Jolla, CA) or Vent DNA <br /> nuclear transport of macromolecules. In mammalian                  polymerase (New England Biolabs, Beverly, MA) and oligonu- <br /> cells, eIF5A is mainly cytoplasmic with a fraction associ-         cleotides listed in Table 3. <br /> ated with the endoplasmic reticulum (ER) membrane                     Construction of a marked TIF51A allele and linkage assays: <br /> through ribosomes (Shi et al. 1996b). Its cytoplasmic              To mark TIF51A with the URA3 gene, a 1.4-kb PstI fragment <br />                                                                    containing TIF51A was removed from the plasmid pPS1483 <br /> localization, combined with its interaction with the ribo-         and ligated into the NsiI site of the vector YIp5, generating <br /> somal protein L5 (Schatz et al. 1998), strengthens the             the plasmid pPS1591. This plasmid was linearized using HpaI, <br /> connection between eIF5A and translation.                          which cleaves within the TIF51A gene, and used to transform <br />   Recently it was shown that a yeast mutant harboring              a W303a/&#226;&#65533;&#163; diploid strain. After selection on uracil dropout <br /> a temperature-sensitive allele of TIF51A exhibits a defect         plates, transformed diploids were sporulated. Genomic DNA <br />                                                                    from the selected diploids and their haploid progeny was <br /> in mRNA decay, accumulating uncapped mRNAs at the                  digested with PstI and subjected to Southern blot analysis using <br /> restrictive temperature. In addition, this strain shows an         the 1.4-kb PstI fragment as a probe. One of the haploid cells <br /> &#226;&#181;&#8216;30% decrease in protein synthesis at high temperature             that contained the URA3 gene integrated next to TIF51A was <br /> (Zuk and Jacobson 1998). These results suggest that                chosen for linkage assay (PSY1241). To examine linkage of <br /> eIF5A may be involved in mRNA degradation as well as               the tif51A temperature-sensitive (ts) alleles and TIF51A, the <br />                                                                    tif51A mutants (PSY1242&#226;&#8364;&#8220;PSY1244) were initially crossed to <br /> translation.                                                       this strain containing a TIF51A-marked allele (PSY1241). The <br />   Here we have characterized three novel alleles of                diploids resulting from these crosses were sporulated and sub- <br /> TIF51A and have used them to address the proposed                  jected to tetrad analysis. <br /> functions of eIF5A. We also present data linking eIF5A                Amplification and sequencing of the tif51A temperature- <br /> with both poly(A)-binding protein and protein kinase               sensitive alleles: Genomic DNA was prepared from the tif51A <br />                                                                    temperature-sensitive strains (PSY1245, PSY1248, and PSY- <br /> C. The isolation of PAB1 and PKC1 as multicopy suppres-            1249). The TIF51A gene was amplified by PCR using Taq DNA <br /> sors of a temperature-sensitive allele of TIF51A, tif51A-1,        polymerase (Perkin-Elmer, Norwalk, CT) and the primers <br /> suggests important connections between these three                 NPL2-A and NPL2-B. PCR products were purified and se- <br />                                                       Genetic Analysis of eIF5A                                                    395 <br />  <br />                                                                 TABLE 2 <br />                                                     Plasmids used in this study <br />  <br />          Plasmid                                     Features                                           Source <br />          pGEX-4T-1        Bacterial expression vector for GST fusion proteins                Pharmacia <br />          pRS315           Yeast LEU2 CEN expression vector                                   Sikorski and Hieter (1989) <br />          pRS316           Yeast URA3 CEN expression vector                                   Sikorski and Hieter (1989) <br />          pRS426           Yeast URA3 2&#226;&#65533;&#174; expression vector                                    Christianson et al. (1992) <br />          pPS1483          1.4-kb PstI fragment with TIF51A in YCpLacIII LEU2 CEN             R. Zitomer <br />          pPS1494          Yeast Gal GFP-Rev URA3 2&#226;&#65533;&#174;                                          Taura et al. (1998) <br />          YIp5             URA3 targeted integration vector                                   Struhl et al. (1979) <br />          pCGF-1a          Yeast Gal GFP URA3 2&#226;&#65533;&#174; expression vector                            Kahana and Silver (1996) <br />          pPS1591          TIF51A PstI fragment in YIp5                                       This study <br />          pPS1592          TIF51A in pRS316                                                   This study <br />          pPS1593          tif51A-1 in pRS315                                                 This study <br />          pPS1594          tif51A-2 in pRS315                                                 This study <br />          pPS1595          tif51A-3 in pRS315                                                 This study <br />          pPS1596          TIF51A in pGEX-4T-1                                                This study <br />          pPS1597          TIF51A in pCGF-1a                                                  This study <br />          pPS1598          TIF51A in pRS426                                                   This study <br />          pPS1599          YOR137C in pRS426                                                  This study <br />          pPS1600          FspI-NotI fragment containing PKC1 in pRS426                       This study <br />          pPS1601          PAB1 in pRS426                                                     This study <br />          pPS1602          WSC1 in pRS426                                                     This study <br />          pPS1603          WSC2 in pRS426                                                     This study <br />          pPS1604          WSC3 in pRS426                                                     This study <br />          pPS2440          PKC1::HA in pPS1600                                                This study <br />          pPS2441          pkc1-K853R::HA in pPS1600                                          This study <br />  <br />  <br />  <br /> quenced using the same primers at the Dana-Farber Cancer             digested with BamHI and SalI. GST-eIF5A was bacterially ex- <br /> Institute Molecular Biology Core Facility.                           pressed from this plasmid, the fusion protein purified, and <br />    Cloning TIF51A and the tif51A temperature-sensitive alleles:      eIF5A released from GST-eIF5A by thrombin (Sigma, St. <br /> TIF51A was cloned from genomic DNA purified from a wild-             Louis) cleavage essentially as described (Henry and Silver <br /> type strain (FY23) and the ts alleles from the tif51A ts strains     1996). A polyclonal anti-eIF5A rabbit antiserum was raised at <br /> (PSY1245, PSY1248, and PSY1249). PCR reactions were per-             Covance Research Products using this recombinant eIF5A as <br /> formed using primers NPL2-C and NPL2-D, which contain a              immunogen. <br /> BamHI site. PCR products were digested with BamHI and gel               To determine eIF5A levels in tif51A mutant strains, cells <br /> purified. The fragment containing wild-type TIF51A was cloned        were grown to midlog phase at 25&#226;&#172;&#353;, shifted to 37&#226;&#172;&#353; or left at <br /> into the BamHI site of pRS316, generating the plasmid pPS1592.       25&#226;&#172;&#353; for 3 hr, and then lysed in radio-immune precipitation <br /> For the mutated alleles, PCR fragments were transferred into         buffer (150 mm NaCl, 1% Nonidet P-40, 0.5% deoxycholate, <br /> pRS315, resulting in plasmids pPS1593 (tif51A-1), pPS1594            0.1% SDS, 50 mm Tris-HCl, pH 8.0) with 1 mm phenylmethyl- <br /> (tif51A-2), and pPS1595 (tif51A-3). All constructs were con-         sulfonyl fluoride as previously described (McBride et al. 2000). <br /> firmed by sequencing.                                                Total protein (5 &#226;&#65533;&#174;g) was resolved by SDS-PAGE, transferred <br />    Disruption of TIF51A: A PCR strategy (Baudin et al. 1993)         to nitrocellulose, and eIF5A levels were detected by immu- <br /> was used to disrupt TIF51A in a wild-type diploid (PSY1250).         noblotting with a 1:10,000 dilution of anti-eIF5A antiserum <br /> Briefly, a DNA fragment containing the HIS3 gene flanked             and enhanced chemiluminescence detection (Amersham, <br /> by the 5&#226;&#172;&#732; and 3&#226;&#172;&#732; sequences of TIF51A was obtained by PCR             Buckinghamshire, UK). <br /> using primers NPL2-G and NPL2-H. The PCR product was                    Localization of eIF5A: Indirect immunofluorescence was <br /> purified and transformed into PSY1250. Transformants were            used to localize endogenous eIF5A using the polyclonal anti- <br /> selected on histidine dropout plates and screened by PCR. A          eIF5A at a dilution of 1:1000. This experiment was performed <br /> diploid strain harboring a disrupted copy of TIF51A was se-          essentially as described (Lee et al. 1996). Localization of eIF5A <br /> lected, transformed with pPS1592, sporulated, and subjected          in wild-type (FY23) and xpo1-1 (Stade et al. 1997) cells was <br /> to tetrad analysis. The tif51A::HIS3 disruption was confirmed        determined following a 1-hr shift to 37&#226;&#172;&#353;. <br /> by PCR and Southern blot. One strain carrying the tif51A::HIS3          To localize eIF5A in living cells, a plasmid that expressed <br /> disruption (PSY1251) was chosen to confirm the function of           a fusion between green fluorescent protein (GFP) and eIF5A <br /> the plasmid-borne mutated alleles.                                   was transformed into a wild-type strain (FY23). This plasmid, <br />    Anti-eIF5A Western blotting: To produce anti-eIF5A anti-          pPS1597, was generated by PCR amplification of TIF51A from <br /> sera, a plasmid to express GST-eIF5A in Escherichia coli             FY23 genomic DNA using primers NPL2-L and NPL2-M. The <br /> (pPS1596) was constructed as follows. The TIF51A gene was            PCR product was digested with BamHI and HindIII and ligated <br /> amplified by PCR from genomic DNA of a wild-type strain              into the pCGF-1a vector (Kahana and Silver 1996), placing <br /> (FY23) using primers TIF51A-1 and TIF51A-2. The PCR prod-            GFP-TIF51A under the control of a galactose-inducible pro- <br /> uct was digested with BamHI and AvaI, gel purified, and ligated      moter. GFP fusion proteins were visualized after 1&#226;&#8364;&#8220;3 hr of <br /> into pGEX-4T-1 (Pharmacia, Piscataway, NJ) that had been             induction as described (Lee et al. 1996). <br />  396                                                     S. R. Valentini et al. <br />  <br />                                                              TABLE 3 <br />                                                 Oligonucleotides used in this study <br />  <br />          Oligo                                       Sequence (5&#226;&#172;&#732;-3&#226;&#172;&#732;)a                                        Site <br />          NPL2-A              GCCAATTACTCATAGACTCC <br />          NPL2-B              GCGAAGAGTACATGATGTGA <br />          NPL2-C              CGCGGATCCTGTATTCCGTAGCGTTATATCG                                            BamHI <br />          NPL2-D              CGCGGATCCGAATTACACTGCCTGCATAAGG                                            BamHI <br />          NPL2-G              ACTCATAGACTCCCAAACACACACAAATACCAACTCATATATACA <br />                                GGCCTCCTCTAGTACACTC <br />          NPL2-H              TTTTTTCTTTTTTCATTTATATCCCATGCCATGATGTTAACCGGT <br />                                GCGCGCCTCGTTCAGAATG <br />          NPL2-L              CGCGGATCCATGTCTGACGAAGAACATACC                                             BamHI <br />          NPL2-M              CCCAAGCTTAGCCGGTAGATATGCGC                                                 HindIII <br />          TIF51A-1            CGCGGATCCATGTCTGACGAAGAACATACCCCGCTCGAG                                    BamHI TIF51A-2 <br />                              CCGGTAGATATGCGC                                                            XhoI/AvaI <br />          137-E               CGCGGATCCCGGACATAAATTAGACTCTAAGG                                           BamHI <br />          137-F               CGCGGATCCGTATCTACTCCAAACTTCTATGG                                           BamHI <br />          PAB1-C              CGCGGATCCAAGAGGTCATACTGTATGAAGCC                                           BamHI <br />          PAB1-D              CGCGGATCCAGTATTTTCCTGGAACCTGTTGG                                           BamHI <br />          WSC1-A              CGCGGATCCACTACTTTCGTAGAATTCCTTCC                                           BamHI <br />          WSC1-B              CGCGGATCCCTATGTAACTAAGTTAAGGTTGG                                           BamHI <br />          WSC2-A              CGCGGATCCATCTAGCACTTCTCCCAGAAGTGC                                          BamHI <br />          WSC2-B              CGCGGATCCTGATGGTGATTTGAGACAGACC                                            BamHI <br />          WSC3-A              CGCGGATCCAGCCGATTCGTTAGTAGGAATGC                                           BamHI <br />          WSC3-B              CGCGGATCCGTACCACAAACATCATTAGAAGCTGC                                        BamHI <br />            a <br />                Introduced restriction sites are underlined and italics indicate TIF51A sequences. <br />  <br />  <br />    mRNA stability assay: Exponentially growing yeast cultures         clone WSC2 (pPS1603); and WSC3-A and WSC3-B to clone <br /> were shifted from 25&#226;&#172;&#353; to 37&#226;&#172;&#353;. At various times, samples were          WSC3 (pPS1604). In addition, hemagglutinin (HA)-tagged forms <br /> collected and quickly frozen in a dry ice-ethanol bath. Total         of wild-type and K853R mutant PKC1 were subcloned into <br /> RNA was isolated from yeast cells by a modified hot phenol            pPS1600 by digestion of pGAL[PKC1::HA] and pGAL[pkc1- <br /> method (Oliveira and McCarthy 1995). Yeast strains were               K853R::HA] (Watanabe et al. 1994) with MscI and SphI and <br /> incubated for 4 hr at 37&#226;&#172;&#353;, when 20 &#226;&#65533;&#174;g/ml thiolutin was added          insertion of the 2.4-kb fragment into MscI/SphI-digested <br /> and samples were taken at 0, 5, 10, 15, 20, and 30 min after          pPS1600, resulting in pPS2440 and pPS2441, respectively. <br /> addition of thiolutin. RNAs were separated by electrophoresis <br /> on 1.3% agarose gels, after denaturing with glyoxal. RNA <br /> gels were subjected to Northern blotting using Hybond nylon                                    RESULTS <br /> membranes (Amersham). Membranes were probed with a <br /> 32 <br />   P-labeled oligonucleotide complementary to the 25S rRNA                Characterization of three new temperature-sensitive <br /> and a random primer-labeled DNA fragment corresponding                alleles of TIF51A: In a screen for conditional mutants <br /> to the HHT2 gene. Blots were analyzed by using a phosphorim-          defective in nuclear protein localization in the yeast S. <br /> ager (Molecular Dynamics, Sunnyvale, CA). <br />    High-copy suppressor screen: A URA3/2&#226;&#65533;&#174; genomic yeast               cerevisiae, several complementation groups were ob- <br /> library (Connelly and Hieter 1996) was transformed into               tained (Sadler et al. 1989). Five temperature-sensitive <br /> a tif51A-1 strain (PSY1252). Approximately 140,000 trans-             mutants were classified in a complementation group <br /> formants were selected by plating on uracil dropout plates,           initially named npl2 (nuclear protein localization 2). <br /> incubating at 25&#226;&#172;&#353; overnight, and shifting plates to 36&#226;&#172;&#353; for 3&#226;&#8364;&#8220;4       However, these mutants did not show a general defect <br /> days. Plasmids were rescued from temperature-resistant clones <br /> and retransformed into PSY1252 to show plasmid linkage. The           in nuclear protein import or export or mRNA export. <br /> genomic segment present in each of the selected clones was            They did show defects in localization of the large ribo- <br /> determined by sequencing the ends with T3 and T7 primers              somal subunit (Stage-Zimmermann et al. 2000). <br /> and using these sequences to search the S. cerevisiae genome             Using one of the original mutants (PSY8) and a yeast <br /> database.                                                             genomic library, TIF51A was cloned by complementa- <br />    High-copy suppressor genes were characterized by subclon- <br /> ing different segments of the original clone into pRS426              tion of the temperature-sensitive <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> (Chiang <br /> (URA3/2&#226;&#65533;&#174;) and testing in PSY1252. The minimal suppressing             1993). Here, three of these mutants (PSY8, PSY338, and <br /> subclone containing the PKC1 open reading frame (ORF) is              PSY341) were further characterized. These strains were <br /> pPS1600. Also, TIF51A and the five other high-copy suppressor         first backcrossed three times to the parental W303 <br /> genes were cloned into pRS426 using the following primers             strains to generate a number of new strains containing <br /> containing a BamHI site: NPL2-C and NPL2-D to clone TIF51A <br /> (pPS1598); 137-E and 137-F to clone YOR137C (pPS1599);                the same mutated alleles of TIF51A. To determine <br /> PAB1-C and PAB1-D to clone PAB1 (pPS1601); WSC1-A and                 whether the temperature-sensitive mutations were linked <br /> WSC1-B to clone WSC1 (pPS1602); WSC2-A and WSC2-B to                  to the TIF51A locus, npl2 strains (PSY1242&#226;&#8364;&#8220;PSY1244) <br />                                                  Genetic Analysis of eIF5A                                                397 <br />  <br /> were crossed to a strain containing the URA3 gene in- <br /> serted next to TIF51A (PSY1241). The diploids that re- <br /> sulted from these crosses were sporulated and subjected <br /> to tetrad analysis. If the mutations present in the npl2 <br /> strains were linked to TIF51A, the temperature-sensitive <br /> phenotype of the mutants would never cosegregate with <br /> the URA3 gene. For npl2-1, 25 complete asci were ana- <br /> lyzed and no temperature-sensitive Ura&#226;&#171;&#185; spore was ob- <br /> served. The same result was obtained with npl2-2 and <br /> npl2-3, after analyzing 15 and 17 complete asci, respec- <br /> tively. These results demonstrate that npl2-1, npl2-2, and <br /> npl2-3 are linked to TIF51A. Therefore, the npl2 mutants <br /> presented here contain temperature-sensitive alleles of <br /> the gene encoding eIF5A and are henceforth referred <br /> to as tif51A-1, tif51A-2, and tif51A-3. <br />    These three TIF51A temperature-sensitive alleles were <br /> cloned by PCR into pRS315 (CEN/LEU2) using genomic <br /> DNA obtained from the tif51A strains. To test whether <br /> the cloned mutant alleles of TIF51A were functional, <br /> we generated a strain in which TIF51A was disrupted. <br /> Sporulation of a heterozygous TIF51A/tif51A&#226;&#338;&#172;::HIS3 <br /> diploid revealed that TIF51A is an essential gene since <br /> only two viable his&#226;&#171;&#186; spores were obtained in 24 tetrads <br /> analyzed. This result agreed with Wohl et al. (1993) <br /> and with the isolation of temperature-sensitive alleles <br /> of TIF51A. In contrast, another report showed that cells <br /> lacking TIF51A grow slowly and become inviable only <br /> when TIF51B is also disrupted (Schnier et al. 1991). <br /> This discrepancy may be due to strain background dif- <br /> ferences. <br />    The three TIF51A alleles demonstrate varying degrees <br /> of temperature sensitivity upon growth at 37&#226;&#172;&#353; on rich <br /> media, YEPD (Figure 1A). While the allele tif51A-2 dis-           Figure 1.&#226;&#8364;&#8221;Temperature sensitivity of tif51A alleles. (A) <br /> plays a moderate reduction in growth at the nonpermis-         Temperature sensitivity of tif51A mutant strains: Wild-type <br /> sive temperature, alleles tif51A-1 and tif51A-3 display        (W303), tif51A-1 (PSY1245), tif51A-2 (PSY1248), and tif51A-3 <br /> more pronounced growth defects. To determine the               (PSY1249) strains were streaked to rich plates (YEPD) and <br /> time of onset of the growth defect in the tif51A mutant        grown at either 25&#226;&#172;&#353; or 37&#226;&#172;&#353; for 3&#226;&#8364;&#8220;4 days. (B) Strains W303, <br />                                                                PSY1245 (tif51A-1), PSY1248 (tif51A-2), and PSY1249 (tif51A-3) <br /> strains, midlog phase cultures were monitored at differ-       were grown at 25&#226;&#172;&#353; in rich medium (YEPD) to midlog phase <br /> ent times after shift to the nonpermissive temperature         (2.5 hr). The cultures were then split between 25&#226;&#172;&#353; and 37&#226;&#172;&#353; <br /> (Figure 1B). Although tif51A mutant strains showed             and growth was monitored for 8 hr by spectrophotometry <br /> slightly slower growth than the wild-type strain at 25&#226;&#172;&#353;        (OD600 nm). (&#228;&#352;&#65533;) Wild type, (&#228;&#8240;&#171;) tif51A-1, (&#228;&#352;&#352;) tif51A-2, (&#228;&#8240;&#173;) <br /> (left), they showed a significant decrease in growth rate      tif51A-3. <br /> by 2&#226;&#8364;&#8220;3 hr after shift to 37&#226;&#172;&#353; (right). The discrepancy in <br /> the relative growth defect between tif51A-1 and tif51A-2 <br /> on plates vs. liquid may reflect a growth defect that is       to-tyrosine change at position 39 (C39Y) and a glycine- <br /> overcome after longer incubation times.                        to-aspartic acid change at position 118 (G118D). <br />    To define the mutations in the TIF51A gene present              Two crystal structures of archaebacterial eIF5A pro- <br /> in the tif51A temperature-sensitive mutants, genomic           teins have been solved (Kim et al. 1998; Peat et al. 1998) <br /> DNA was isolated from the tif51A strains and the TIF51A        and the positions of these mutations in the Methanococ- <br /> gene was amplified by PCR. The 0.7-kb fragment con-            cus jannaschii eIF5A structure (Kim et al. 1998) are shown <br /> taining TIF51A was purified and then sequenced. These          (Figure 2). The proline mutated in tif51A-1 and tif51A-2 <br /> mutations were confirmed by sequencing the cloned              proteins is at the junction between the two domains of <br /> alleles in pPS1593, -1594, and -1595. A single mutation        eIF5A. In contrast, the mutations in tif51A-3, both of <br /> of the same residue was detected for the tif51A-1 and          which are required for temperature sensitivity (data not <br /> tif51A-2 alleles. The proline at position 83 was changed       shown), are found on separate domains of eIF5A. C39Y <br /> to serine (P83S) or leucine (P83L), respectively. Two          is found in a &#226;&#65533;&#164;-sheet 12 residues N-terminal to the hypus- <br /> mutations were mapped in the tif51A-3 allele, a cysteine-      inated lysine that is critical for eIF5A function; G118D <br />  398                                                     S. R. Valentini et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 4.&#226;&#8364;&#8221;Localization of wild-type eIF5A. (A) Endoge- <br />                                                                       nous eIF5A localization in fixed cells. Wild-type cells (FY23) <br />   Figure 2.&#226;&#8364;&#8221;Lesions in tif51A temperature-sensitive alleles.          were grown at 25&#226;&#172;&#353; to midlog phase and fixed by incubation <br /> The locations of mutated residues in the eIF5A structure are          with formaldehyde. Cells were placed on polylysine-coated <br /> shown on the archaebacterial eIF5A crystal structure (Kim et al.      slides and, after permeabilization, incubated first with a rabbit <br /> 1998; yeast eIF5A residue numbers are shown in parentheses).          anti-eIF5A polyclonal serum and then with an anti-rabbit fluo- <br /> Tif51A-1 and tif51A-2 have mutations at Pro83 and tif51A-3            rescein-conjugated secondary antibody. Localization of eIF5A <br /> has mutations at Cys39 and Gly118 in the C-terminal domain            was then visualized by fluorescence microscopy. (B) GFP- <br /> loop (arrow). The essential hypusinated lysine is also shown.         eIF5A localization in living cells. A functional GFP-eIF5A fu- <br />                                                                       sion protein was expressed in wild-type yeast cells (FY23) under <br />                                                                       the control of a galactose promoter. Production of the fusion <br /> is located within a loop in the C-terminal domain. Thus               protein was induced for 1&#226;&#8364;&#8220;2 hr at 25&#226;&#172;&#353; prior to visualization <br /> both domains of eIF5A appear to be important for its                  by fluorescence microscopy. (C) Localization of eIF5A and <br />                                                                       Rev in xpo1-1 cells. Xpo1-1 cells (PSY1105; Stade et al. 1997) <br /> function.                                                             were transformed with 2&#226;&#65533;&#174; URA3 plasmids containing either <br />    To determine eIF5A protein levels in the tif51A mu-                GFP-eIF5a (pPS1597) or GFP-Rev (pPS1494) under the con- <br /> tant strains, a rabbit polyclonal antiserum against eIF5A             trol of the inducible GAL promoter. Cells were grown in 2% <br /> was produced using recombinant yeast eIF5A expressed                  raffinose until midlog phase, induced with 2% galactose for <br /> in E. coli. This antiserum was used to detect eIF5A by                3&#226;&#8364;&#8220;5 hr, shifted to glucose, incubated at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; for 1 hr, <br />                                                                       and fixed, and eIF-5A and REV localization was determined <br /> Western blot in cell lysates prepared from wild-type and              by fluorescence microscopy. <br /> temperature-sensitive TIF51A strains grown at the per- <br /> missive and nonpermissive temperature (Figure 3). At <br /> 25&#226;&#172;&#353;, eIF5A of the expected molecular mass (&#226;&#181;&#8216;20 kD)                    suggesting that these are loss-of-function alleles. Indeed, <br /> was detected in the wild-type (WT) and in the tif51A-1                after 12 hr at 37&#226;&#172;&#353;, eIF5A mutant protein is no longer <br /> and tif51A-2 strains. In the case of tif51A-3, eIF5A mi-              detectable by Western blotting (data not shown). <br /> grates slightly more slowly, due to the G118D mutation                   Localization of eIF5A in the yeast S. cerevisiae: eIF5A <br /> in eIF5A in the tif51A-3 strain (data not shown). After               has been proposed to be an export factor for the human <br /> a 3-hr shift to 37&#226;&#172;&#353;, levels of mutant eIF5A proteins are              immunodeficiency virus (HIV) Rev protein, by virtue of <br /> significantly lower than that of the wild-type protein,               acting as an adapter to the transporter exportin/CRM1 <br />                                                                       (Ruhl et al. 1993; Bevec et al. 1996; Bevec and Hauber <br />                                                                       1997; Hofmann et al. 2001). Therefore, we wished to <br />                                                                       test whether the localization of yeast eIF5A was consis- <br />                                                                       tent with this hypothesis, and, if so, whether tif51A mu- <br />                                                                       tant alleles affected nuclear export. eIF5A localization <br />                                                                       was performed initially by indirect immunofluores- <br />                                                                       cence, using formaldehyde-fixed wild-type cells and anti- <br />                                                                       eIF5A antiserum (Figure 4A). eIF5A is localized mainly <br />                                                                       in the cytoplasm with a concentration in the perinuclear <br />                                                                       region. A comparison of the eIF5A localization to 4&#226;&#172;&#732;,6- <br />    Figure 3.&#226;&#8364;&#8221;Expression of mutant eIF5A proteins. Samples             diamidino-2-phenylindole (DAPI) staining of nuclei re- <br /> from cultures in Figure 1B were collected at 3 hr postshift, <br /> cells were lysed, and 5 &#226;&#65533;&#174;g (or 1 &#226;&#65533;&#174;g for the lower dilution of         veals that only minor amounts of eIF5A are found in <br /> the wild-type culture at 37&#226;&#172;&#353;) of total protein was analyzed by        the nucleus. <br /> Western blotting with an anti-eIF5A antiserum.                           To ascertain whether this localization reflected eIF5A <br />                                                  Genetic Analysis of eIF5A                                                399 <br />  <br /> distribution in live cells, TIF5A was inserted downstream <br /> of a galactose-inducible green fluorescent protein gene. <br /> This plasmid produces functional eIF5A in that it re- <br /> stores the ability of tif51A-1 to grow at the restrictive <br /> temperature after galactose induction (data not shown). <br /> This plasmid was transformed into a wild-type strain <br /> and after 1&#226;&#8364;&#8220;2 hr induction with galactose, GFP-eIF5A <br /> localization was determined by fluorescence microscopy <br /> (Figure 4B). As expected, GFP-eIF5A was detected pre- <br /> dominantly in the cytoplasm with a stronger signal in <br /> the perinuclear region (Figure 4B, left). These results <br /> conclusively show that eIF5A is a cytoplasmic protein <br /> and accumulates in the perinuclear region. <br />    If eIF5A were to facilitate HIV Rev export by binding <br /> both Rev and the Xpo1/Crm1 exporter, export of eIF5A <br /> itself would be expected to be blocked by mutations in            Figure 5.&#226;&#8364;&#8221;Localization of GFP-Rev in tif51A mutant strains. <br /> the export machinery. To test if eIF5A is exported by          Wild-type (W303), tif51A-1 (PSY1245), tif51A-2 (PSY1248), and <br />                                                                tif51A-3 (PSY1249) strains were transformed with a 2&#226;&#65533;&#174; URA3 <br /> Xpo1/Crm1, its localization was monitored in xpo1-1            plasmid containing GFP-Rev (pPS1494) under the control of <br /> cells, which are temperature sensitive for export by this      the inducible GAL promoter. Cells were prepared as in Figure <br /> pathway. Xpo1-1 cells carrying GFP-eIF5A or GFP-Rev on         4C with a temperature shift to 37&#226;&#172;&#353; for 3 hr and visualized by <br /> 2&#226;&#65533;&#174; URA3 plasmids under the galactose-inducible pro-            fluorescence microscopy. <br /> moter were grown to midlog phase and then induced <br /> with galactose for 3 hr, shifted to 37&#226;&#172;&#353; for 1 hr, fixed, <br /> and then visualized by fluorescence microscopy (Figure         and Jacobson 1998). To test whether the tif51A alleles <br /> 4C). GFP-eIF5A localization was predominantly cyto-            described above affect mRNA stability at the nonpermis- <br /> plasmic in xpo1-1 cells at 37&#226;&#172;&#353;. Comparison with DAPI           sive temperature, wild-type or mutant strains were grown <br /> staining shows no marked increase in nuclear localiza-         for 4 hr at 37&#226;&#172;&#353;, transcription was inhibited, and samples <br /> tion of eIF5A, which would be expected if eIF5A shuttled       were taken at several times post-transcription (Figure <br /> in an Xpo1-dependent manner, suggesting that eIF5A             6). Northern blot analysis with a probe against the mod- <br /> is not exported by Xpo1 in yeast. In contrast, GFP-Rev         erately stable HHT2 mRNA shows a significant decrease <br /> showed a marked accumulation of fluorescent signal in          in the levels of these RNAs in wild-type cells over the <br /> nuclei of xpo1-1 cells after 1 hr at the nonpermissive         course of the assay as compared to the 25S rRNA loading <br /> temperature (Figure 4C).                                       control (Figure 6, TIF51A). However, in tif51A-1 and <br />    To further test the dependence of Rev export upon <br /> eIF5A, GFP-Rev localization was examined in strains <br /> containing the TIF51A temperature-sensitive alleles. <br /> After a 3-hr shift to the nonpermissive temperature, <br /> GFP-Rev displayed cytoplasmic localization similar to <br /> that seen in the wild-type control (Figure 5). The slight <br /> increase in signal in the mutant alleles after temperature <br /> shift does not coincide with the nucleus as determined <br /> by DAPI staining (Figure 5). These data are consistent <br /> with a recent study in mammalian cells that revealed that <br /> exportin 4, rather than CRM1, exports eIF5A (Lipowsky <br /> et al. 2000). Since no yeast homolog of exportin 4 exists, <br /> eIF5A localization was tested in a battery of strains with <br /> mutations in putative transport proteins. None of these <br /> strains showed a nuclear accumulation of eIF5A (data <br /> not shown), suggesting either that yeast eIF5A does not <br />                                                                   Figure 6.&#226;&#8364;&#8221;mRNA decay defect in tif51A temperature-sensi- <br /> shuttle between the nucleus and the cytoplasm or that its      tive alleles. Northern blot analysis of HHT2 mRNA. Following <br /> exporter has no homology to known transport proteins.          incubation at 37&#226;&#172;&#353; for 4 hr, thiolutin was added to cultures to <br /> These data, in combination with the finding that Rev           the final concentration of 20 &#226;&#65533;&#174;g/ml and cell samples were <br /> export is not blocked in tif51A mutant strains, argue          collected at the times indicated. Total RNA was extracted and <br /> against the hypothesis that Rev export by Xpo1/CRM             analyzed by Northern blotting using probes complementary <br />                                                                to the 25S rRNA and to the HHT2 mRNA. Quantitation of <br /> is mediated by eIF5A.                                          the 25S rRNA served as an internal control to visualize the <br />    mRNA decay defect of tif51A alleles: eIF5A has also         amount of total RNA loaded in each lane. Blots were analyzed <br /> been proposed to play a role in mRNA decay (Zuk                and bands quantitated by using a phosphorimager. <br />  400                                                     S. R. Valentini et al. <br />  <br />                            TABLE 4 <br />       Multicopy suppressors of the temperature-sensitive <br />                        mutant tif51A-1 <br />  <br />              No. of <br /> Clone       isolates    Chromosome             Suppressor <br /> 1              8             XV            YOR137C (SIA1 ) <br /> 2              2             II            PKC1 <br /> 3              2             XIV           WSC2 <br /> 4              1             V             PAB1 <br /> 5              1             XV            WSC1/SLG1/HCS77 <br /> 6              1             XV            WSC3 <br />  <br />  <br />  <br /> tif51A-3 strains the HHT2 RNA is stabilized, with levels <br /> slightly higher than those in wild-type cells at 15&#226;&#8364;&#8220;30 <br /> min post-inhibition (Figure 6). The HHT2 RNA level <br /> in tif51A-2 cells is similar to that in wild-type cells. Similar <br /> results were obtained by probing the PAB1 transcript <br /> (data not shown). Cells submitted to heat shock for a <br />                                                                         Figure 7.&#226;&#8364;&#8221;High-copy suppressors of tif51A-1 temperature <br /> shorter period of time (2 hr at 37&#226;&#172;&#353;) did not display                  sensitivity. tif51A-1 strain PSY1252 was transformed with a 2&#226;&#65533;&#174; <br /> significant stabilization of CYH2, PAB1, and HHT2                     URA3 library of plasmids and transformants were screened <br /> mRNAs (data not shown).                                               for restoration of growth at 36&#226;&#172;&#353;. Candidates that still sup- <br />    Suppression of the temperature-sensitive phenotype                 pressed after plasmid rescue and retransformation into <br /> of tif51A-1: To expand our understanding of the func-                 PSY1252 were restreaked to plates lacking uracil at 25&#226;&#172;&#353; and <br />                                                                       36&#226;&#172;&#353;. Wild-type TIF51A and the 2&#226;&#65533;&#174; URA3 vector, positive and <br /> tion of eIF5A, we wished to identify other genes that                 negative controls, are also shown. <br /> encode proteins that work similarly or together with <br /> eIF5A, and therefore we performed a high-copy suppres- <br /> sor screen. A yeast genomic library was transformed into              (Figure 8). Overexpression of PAB1 and PKC1 did not <br /> the tif51A-1 strain and transformants that could grow                 have a significant effect on steady-state levels of eIF5A at <br /> at 36&#226;&#172;&#353; were selected. Twenty-three plasmids that still                either permissive or nonpermissive temperatures. The <br /> suppressed temperature sensitivity after retransforma-                slight increase in eIF5A-3 levels in the presence of excess <br /> tion into tif51A-1 were characterized by DNA sequenc-                 Pkc1 was not reproducible. <br /> ing. Six clones, representing 15 of these plasmids, were                 Connections between PKC1 signaling and eIF5A: To <br /> selected and used to define the ORF that suppresses                   examine the importance of PKC1 signaling in tif51A <br /> the temperature-sensitive phenotype of tif51A-1. These                mutant strains, we tested whether a kinase-inactive form <br /> clones are presented in Table 4. One clone contained                  of PKC1 could suppress the tif51A-1 temperature sensi- <br /> the entire PKC1 gene, which was cloned twice in this                  tivity. Strain PSY1245 was transformed with high-copy <br /> screen, without flanking ORFs. Subcloning revealed the <br /> other suppressing ORFs to be the following: YOR137C, <br /> WSC2, WSC1, WSC3, and PAB1. To eliminate partial flank- <br /> ing ORFs, these five genes were also cloned by PCR and <br /> the constructs tested for suppression of the tempera- <br /> ture-sensitive phenotype of tif51A-1 strain. Figure 7 <br /> shows that the ability of tif51A-1 to grow at 36&#226;&#172;&#353; was <br /> restored by expression of these genes in high copy. In <br /> addition, all tif51A-1 suppressors were allele specific, as <br /> they were unable to rescue growth of tif51A-3 at high <br /> temperatures (data not shown). The plate phenotype <br /> of tif51A-2 at 36&#226;&#172;&#353; was too weak to test suppression in                   Figure 8.&#226;&#8364;&#8221;eIF5A protein levels in the presence of high- <br /> this manner.                                                          copy PAB1 and PKC1. W303 bearing high-copy vectors pRS426 <br />    To test whether the two most-well-studied suppressor               and PSY1245 (tif51A-1); PSY1248 (tif51A-2); and PSY1249 (tif- <br /> proteins, poly(A)-binding protein (Pab1) and protein                  51A-3) bearing pRS426, pPS1600 (PKC1), or pPS1601(PAB1) <br /> kinase C (Pkc1), acted by increasing the levels of mutant             were grown at 25&#226;&#172;&#353; to midlog phase and then shifted to 37&#226;&#172;&#353; or <br />                                                                       left at 25&#226;&#172;&#353; for 3 hr. Cells were lysed and 5 &#226;&#65533;&#174;g total protein <br /> eIF5A protein, mutant strains bearing the multicopy                   was analyzed by anti-eIF5A Western blotting as in Figure 3. <br /> PAB1 and PKC1 plasmids were grown at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; for 3                 The asterisk denotes a common degradation product of eIF5A <br /> hr and eIF5A levels were monitored by Western blotting                (Kang et al. 1993). <br />                                                    Genetic Analysis of eIF5A                                                 401 <br />  <br />  <br />  <br />  <br />    Figure 9.&#226;&#8364;&#8221;PKC kinase activity is required for suppression <br />                                                                     Figure 10.&#226;&#8364;&#8221;Temperature sensitivity of tif51A mutant <br /> of tif51A-1 temperature sensitivity. Dilutions of PSY1245 <br />                                                                  strains is partially suppressed by sorbitol. Dilutions of wild- <br /> (tif51A-1) and DL376 (pkc1&#226;&#338;&#172;) strains bearing plasmids pRS426 <br />                                                                  type (W303), tif51A-1 (PSY1245), tif51A-2 (PSY1248), tif51A-3 <br /> (&#226;&#171;&#186;), pPS1600 (WT), pPS2440 (WT::HA), and pPS2441 <br />                                                                  (PSY1249), and pkc1&#226;&#338;&#172; (DL376; Levin and Bartlett-Heu- <br /> (K853R::HA) were grown for 2 days on plates lacking uracil <br />                                                                  busch 1992) strains were grown for 2 days at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; on <br /> and sorbitol at 25&#226;&#172;&#353; and 37&#226;&#172;&#353;. <br />                                                                  YEPD plates with or without 1 m sorbitol as an osmotic stabi- <br />                                                                  lizer. <br /> plasmids encoding wild-type Pkc1, HA-tagged wild-type <br /> Pkc1, or HA-tagged Pkc1 with a lysine-to-arginine change <br /> in the active site (K853R). Serial dilutions of each trans-      report, three temperature-sensitive alleles of TIF51A <br /> formed strain were then plated at 25&#226;&#172;&#353; or 37&#226;&#172;&#353; (Figure             were characterized (tif51A-1, tif51A-2, and tif51A-3). While <br /> 9). Whereas the ability of PKC1 to suppress tif51A-1             mRNA decay experiments indicated that the stronger <br /> temperature sensitivity was not impaired by the addition         mutations led to stabilization of mRNAs, localization <br /> of an HA tag (WT vs. WT::HA), the active site mutation           studies of wild-type eIF5A suggested that this protein is <br /> (K853R::HA) eliminated suppression. Growth of the                unlikely to be involved in HIV Rev export. The tif51A-1 <br /> pkc1&#226;&#338;&#172; strain, which requires PKC activity, is shown as a         allele was used to contribute further to the search for <br /> control (Watanabe et al. 1994). Therefore, kinase activ-         a function for this highly conserved protein. Screening <br /> ity of PKC1 is required both for signaling and for growth        of a 2&#226;&#65533;&#174; plasmid library containing genomic DNA uncov- <br /> of the tif51A-1 strain, indicating a functional link be-         ered six genes that allowed growth of tif51A-1 strains at <br /> tween eIF5A and PKC1 signal transduction.                        high temperature. Further experiments supported the <br />    Although PKC1 is involved in multiple signaling path-         importance of the PKC/WSC signaling pathway for cell <br /> ways in yeast, the additional isolation of three WSC genes       wall integrity in tif51A-1 mutant cells. <br /> as high-copy suppressors of tif51A-1 suggested a connec-            In fluorescence microscopic experiments, eIF5A was <br /> tion between eIF5A and a specific pathway involved in            detected primarily in the cytoplasm with a concentra- <br /> stress response and maintenance of cell wall integrity           tion in the perinuclear region (Figure 4). This result <br /> (for review see Heinisch et al. 1999). Deletion of PKC1          agrees with the subcellular localization of mammalian <br /> or WSC1 results in cell wall defects and cell lysis on           eIF5A (Shi et al. 1996b, 1997). Ruhl et al. reported that <br /> media lacking an osmotic stabilizer (Paravicini et al.           a substantial fraction of eIF5A was also present in the <br /> 1992; Verna et al. 1997). To determine whether the               nucleus and suggested that eIF5A was a shuttling protein <br /> temperature sensitivity of tif51A mutant strains might           involved in HIV Rev export (Ruhl et al. 1993; Bevec et <br /> reflect a defect in cell wall integrity, dilutions of each       al. 1996; Bevec and Hauber 1997). However, in yeast, <br /> strain were grown at 37&#226;&#172;&#353; on rich media without or with           eIF5A is not mislocalized to the nucleus in an xpo1-1 <br /> 1 m sorbitol to lend osmotic stability (Figure 10). The          strain, a temperature-sensitive mutant of the export re- <br /> addition of sorbitol significantly increased the ability of      ceptor for Rev and other proteins that contain leucine- <br /> tif51A-1 and tif51A-3 strains to grow at 37&#226;&#172;&#353; and com-            rich nuclear export signals (NESs), nor does eIF5A itself <br /> pletely suppressed the growth defect of tif51A-2. Growth         have a canonical NES (Stade et al. 1997, Figure 4). <br /> of a strain lacking PKC1, which requires an osmotic              Indeed, exportin 4, rather than CRM1, was recently <br /> stabilizer at all temperatures, is shown as a control            found to mediate eIF5A export in mammalian cells <br /> (Levin and Bartlett-Heubusch 1992). These results                (Lipowsky et al. 2000). Thus, our data from S. cerevisiae <br /> suggest that the ability of PKC1 and WSC genes to sup-           support the cytoplasmic localization of eIF5A, perhaps <br /> press the temperature sensitivity of tif51A-1 is due to          associated with the ER/ribosomes as suggested by its <br /> their roles in maintaining cell wall integrity.                  concentration in the perinuclear region. Furthermore, <br />                                                                  our data do not support the hypothesis of eIF5A being <br />                                                                  involved in the export of NES-containing proteins <br />                        DISCUSSION <br />                                                                  through the export receptor Xpo1p/Crm1p. <br />   In spite of two decades of research on eIF5A, the role            The identification of conditional tif51A alleles in a <br /> of this ubiquitous protein remains mysterious. In this           screen for mislocalization of a nuclear localization se- <br />  402                                                S. R. Valentini et al. <br />  <br /> quence (NLS)-reporter protein initially would seem to            translation and mRNA decay are underscored by the <br /> suggest that eIF5A is crucial for nuclear import (Sadler         roles of these proteins in both processes: binding of <br /> et al. 1989; Chiang 1993). The reporter protein in this          Pab1p stabilizes mRNA in a translation-dependent man- <br /> screen contained an SV40 NLS fused to cytochrome c1              ner (Coller et al. 1998) and mutations in eIF-4F com- <br /> (Cyt1). Thus, in a strain with a <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>null allele</span> of CYT1,            plex genes as well as in PAB1 result in destabilization <br /> growth on glycerol media requires the mislocalization            of mRNA (Schwartz and Parker 1999; Brown et al. <br /> of the normally nuclear NLS-cytochrome c protein to              2000). <br /> mitochondria. The npl2/tif51A complementation group                 Like Pab1, eIF5A has been implicated in both transla- <br /> of mutant strains grew on glycerol plates at permissive          tion and mRNA decay. Although eIF5A was first thought <br /> temperature; however, the NLS reporter proteins still            to be a translation initiation factor (Benne and Her- <br /> localized to the nucleus as detected by immunofluores-           shey 1978; Park et al. 1993), a temperature-sensitive <br /> cence microscopy (Chiang 1993). In addition, strains             mutant of TIF51A (ts1159) decreases the decay rate of <br /> bearing the mutant tif51A alleles did not mislocalize            several mRNAs tested and results in a twofold accumula- <br /> the nuclear protein Npl3p, a cytoplasmic NLS-NES-GFP             tion of uncapped URA5 mRNA (Zuk and Jacobson <br /> fusion protein, mRNA, or the mutant eIF5A proteins               1998). This mutant also displays slowed translation, as <br /> themselves (data not shown). These contradictory re-             was shown in the eIF5A depletion studies (Kang and <br /> sults suggest that the mechanism by which mutations              Hershey 1994; Zuk and Jacobson 1998). Here we have <br /> in TIF51A allow growth on a nonfermentable carbon                shown that tif51A-1 and tif51A-3 alleles result in mRNA <br /> source does not involve redirecting the majority of cyto-        decay defects at the nonpermissive temperature (Figure <br /> chrome c1 to mitochondria. These mutations may alter             5), although the rate of mRNA stabilization seen in the <br /> the translation or decay rate of the CYT1 mRNA and               mutants tif51A-1 and tif51A-3 is less drastic than that <br /> thereby increase the production of the fusion protein,           observed in the strain ts1159 (Zuk and Jacobson 1998). <br /> perhaps allowing a minor fraction to reach the mito-             No effect on mRNA decay was observed in the mutant <br /> chondria to allow growth.                                        strain tif51A-2. These results support the idea that the <br />    A high-copy suppressor screen with the tif51A-1 allele        effects on mRNA decay are allele specific. Significant <br /> revealed six genes that could suppress the temperature           mRNA stabilization is not detected until after 4 hr of <br /> sensitivity of this mutant. These suppressors are PAB1,          heat shock, suggesting the possibility that secondary <br /> YOR137C, PKC1, WSC1, WSC2, and WSC3. The most                    effects are being measured. Furthermore, these results <br /> frequent suppressor in the screen, YOR137C [named                suggest that the suppression of tif51A-1 temperature <br /> here suppressor of eIF5A (SIA1)], encodes a protein              sensitivity by overexpression of PAB1 may reflect com- <br /> with a peptide signal for membrane localization and was          plementary roles of eIF-5A and Pab1 in mRNA decay. <br /> recently associated with activation of the yeast plasma             The high-copy suppressor focused on in this work, <br /> membrane H&#226;&#171;&#185;-ATPase by glucose (de la Fuente et al.               PKC1, is the yeast homolog of the metazoan PKC gene <br /> 1997). Although the fluorescence <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of a GFP-               and encodes a regulator of one of the four mitogen- <br /> Sif51A fusion protein, cytoplasmic and concentrated in           activated protein-kinase cascades characterized so far in <br /> the perinuclear region, is similar to that of eIF5A (data        S. cerevisiae (Posas et al. 1998). Pkc1-signaling pathways <br /> not shown), it is not clear how overexpression of Sif51Ap        can be activated by hypotonic and heat shock and they <br /> could suppress tif51A-1 temperature sensitivity.                 are involved with cell cycle control and cell wall biosyn- <br />    Another high-copy suppressor of tif51A-1, PAB1 has            thesis (Levin et al. 1990; Madden et al. 1997; Posas et <br /> been shown to be involved in multiple aspects of RNA             al. 1998). Remarkably, three other suppressors isolated <br /> metabolism, including 3&#226;&#172;&#732;-end processing, translation,            simultaneously are upstream regulators of the cell wall <br /> and mRNA decay. The yeast gene, PAB1, is essential but           integrity signaling pathway: WSC1 (SLG1 or HCS77), <br /> the lethality caused by deletion of PAB1 can be suppressed       WSC2, and WSC3 (Gray et al. 1997; Verna et al. 1997; <br /> by mutations in a number of other genes, including               Jacoby et al. 1998). <br /> genes that encode components of the translational and               Although PKC signaling has not been shown to have a <br /> mRNA degradation machinery (Sachs and Davis 1989,                direct role in mRNA translation or decay, recent studies <br /> 1990; Caponigro and Parker 1995; Boeck et al. 1998;              have revealed the importance of PKC/WSC signaling <br /> Bonnerot et al. 2000). Pab1 participates in the <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>           in the regulation of ribosome synthesis (Nierras and <br /> control of poly(A) tails of messenger RNAs in the yeast          Warner 1999; Li et al. 2000). Ribosomal RNA and ribo- <br /> nucleus (Amrani et al. 1997; Kessler et al. 1997; Min-           somal protein mRNA transcription are inhibited during <br /> vielle-Sebastia et al. 1997), and the poly(A) tail en-           the stress response caused by a secretion block and <br /> hances the translation of mRNAs ( Jacobson and Fav-              this downregulation requires PKC1 and the WSC genes. <br /> reau 1983; Galili et al. 1988; Munroe and Jacobson               Several results also point to the possibility that eIF5A <br /> 1990). This translational enhancement is mediated by             may have a role in regulating ribosome synthesis. At high <br /> the binding of Pab1p to initiation factor eIF-4G, which          temperature two of the mutant strains characterized in <br /> is part of the mRNA cap-binding complex eIF-4F                   this study, tif51A-1 and tif51A-2 (referred to as npl2-1 and <br /> (Tarun et al. 1997). The intimate connections between            npl2-2), were found to mislocalize the large ribosomal <br />  <br />  <br /> subunit protein L11 fused to GFP in 10&#226;&#8364;&#8220;40% of cells,                      <br /> reflecting a defect in either ribosome assembly or export                <br /> (Stage-Zimmermann et al. 2000). In permeabilized cell                <br /> assays, purified eIF5A was shown to accumulate in            <br /> nucleoli, the site of ribosome synthesis (Lipowsky et al.               <br /> 2000). The initial identification of eIF5A as a ribosome-            <br /> associated factor (Kemper et al. 1976; Benne and Her-                  <br /> shey 1978) and the 30% decrease in translation found <br /> in cells depleted of eIF5A (Kang and Hershey 1994) <br /> may reflect the involvement of eIF5A in ribosome syn-                               <br /> thesis, rather than a direct role in translation. <br />                                                                         <br />    The ability of tif51A-1 temperature sensitivity to be                 <br /> partially suppressed either by overexpression of PKC or                     <br /> WSC genes (Figure 6) or by the addition of sorbitol                   <br /> (Figure 9) suggests that eIF5A may act either in the                      <br /> PKC pathway for cell integrity or in a parallel pathway.                <br /> eIF5A is phosphorylated in vivo, but the kinase responsi-                <br /> ble for this nonessential modification has not been iden-                     <br /> tified. PKC1 overexpression may suppress tif51A-1 by                         <br /> increasing phosphorylation of the mutant protein. Al-              <br /> ternatively, PKC1 could phosphorylate a protein that                   <br /> interacts with eIF5A and that is important for its func-                    <br /> tion. One candidate eIF5A-interacting protein is deoxy-                   <br /> hypusine synthase, the essential enzyme that catalyzes                   <br /> the first step in the lysine-to-hypusine modification                      <br /> (Kang et al. 1995; Sasaki et al. 1996). This protein has                <br /> been shown to be phosphorylated specifically by PKC                           <br /> in vitro (Kang and Chung 1999) and contains three                          <br /> potential target S/T residues with basic residues on both                <br /> sides. It is also intriguing to note that Pkc1p is an up- <br />                                                                            <br /> stream regulator of the SBF transcription factor, which                   <br /> is a regulator of the G1-to-S phase (Madden et al. 1997),               <br /> and that eIF5A has been implicated in the translation                      <br /> of mRNAs involved in the same cell cycle transition                       <br /> (Hanauske-Abel et al. 1995). The exact mechanism of                      <br /> PKC suppression will be the focus of future investiga-                   <br /> tion.                                                                 <br />    The isolation of PAB1, PKC1, and the WSC genes as                 <br /> high-copy suppressors of tif51A-1 points to different                     <br /> roles for eIF5A in RNA metabolism from mRNA decay to                         <br /> ribosome synthesis. The allele specificity of suppression                   <br /> suggests that eIF5A may have more than one important                   <br /> role in the cell, leading to functional differences be-                     <br /> tween the alleles. It is interesting to note that PKC1 was             <br /> identified in a Pab1p two-hybrid screen (Mangus et                     <br /> al. 1998). The genetic interactions of these genes with                     <br /> TIF51A suggest that these three proteins are intimately                <br /> associated, but further experiments will be necessary to                   <br /> clarify these interactions. Thus, although the precise                   <br /> function of eIF5A remains an open question, the data                      <br /> presented in this report suggest that eIF5A acts in multi-                 <br /> ple steps of RNA metabolism. Future research will help                     <br /> define the exact roles played by these proteins in post-                     <br /> transcriptional regulation.                                             <br /> </body></html>