<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>    Cns1 Is an Essential Protein Associated with the Hsp90 Chaperone <br />          Complex in Saccharomyces cerevisiae That Can Restore <br />             Cyclophilin 40-Dependent Functions in cpr7D Cells <br />   <br />              Saccharomyces cerevisiae harbors two cyclophilin 40-type enzymes, Cpr6 and Cpr7, which are components of <br />           the Hsp90 molecular chaperone machinery. Cpr7 is required for normal growth and is required for maximal <br />           activity of heterologous Hsp90-dependent substrates, including glucocorticoid receptor (GR) and the oncogenic <br />           tyrosine kinase pp60v-src. In addition, it has recently been shown that Cpr7 plays a major role in negative <br />           regulation of the S. cerevisiae heat shock transcription factor (HSF). <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> better understand functions associated <br />           with Cpr7, a search was undertaken for multicopy suppressors of the cpr7D slow-growth <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span>. The screen <br />           identified a single gene, designated CNS1 (for cyclophilin seven suppressor), capable of suppressing the cpr7D <br />           growth defect. Overexpression of CNS1 in cpr7D cells also largely restored GR activity and negative regulation <br />           of HSF. In vitro protein retention experiments in which Hsp90 heterocomplexes were precipitated resulted in <br />           coprecipitation of Cns1. Interaction between Cns1 and the carboxy terminus of Hsp90 was also shown by <br />           two-hybrid analysis. The functional consequences of CNS1 overexpression and its physical association with the <br />           Hsp90 machinery indicate that Cns1 is a previously unidentified component of molecular chaperone com- <br />           plexes. Thus far, Cns1 is the only tetratricopeptide repeat-containing component of Hsp90 heterocomplexes <br />           found to be essential for cell viability under all conditions tested. <br />  <br />  <br />    Cells rely on molecular chaperones to facilitate the folding               the enzyme (23). We have shown that Hsp90 and the Hsp90- <br /> of nascent polypeptides, prevent protein aggregation, and en-                 associated cyclophilin 40 (Cyp40)-type enzyme Cpr7 are re- <br /> able proper inter- and intramolecular interactions. Indispens-                quired for negative regulation of heat shock transcription <br /> able in both Saccharomyces cerevisiae (5) and Drosophila mela-                factor (HSF) in S. cerevisiae (reference 17 and this report), <br /> nogaster (13), Hsp90 is one of the most abundant and highly                   revealing that some components of the Hsp90 machinery par- <br /> conserved chaperones. In vitro, Hsp90 displays general chap-                  ticipate in regulation of the heat shock response. <br /> erone properties by preventing the aggregation of proteins                       Hsp90 is assisted in its various roles by a cohort of associated <br /> such as citrate synthase (52) and casein kinase II (30) and by                proteins including the chaperones Hsp70 and DnaJ, the im- <br /> maintaining b-galactosidase (21) in a folding-competent state.                munophilins Cyp40, FKBP51, and FKBP52, and other, less <br /> In vivo, however, Hsp90 is not required for the folding of most               well characterized proteins, including Hop (p60), Hip (p48), <br /> proteins (33) but instead plays a key role in the maturation of <br />                                                                               p23, and PP5, a protein phosphatase with immunophilin prop- <br /> a small subset of proteins typically involved in signal transduc- <br />                                                                               erties (4, 25, 38, 42, 46, 48). The Hsp90-associated proteins are <br /> tion (3, 32, 42). It is needed for the maturation of certain <br /> steroid hormone receptors (40, 43, 44), basic helix-loop-helix                highly conserved among eukaryotes, and the composition of <br /> transcription factors (45), serine/threonine kinases (30, 41, 50,             Hsp90 heterocomplexes is also generally conserved (9). In <br /> 51), and oncogenic tyrosine kinases (7). Depending on the                     vitro reconstitution studies of steroid receptor maturation have <br /> particular substrate, the interactions with Hsp90 required for                shown that the components of Hsp90 complexes are assembled <br /> maturation can be transient or continuous (32).                               in an ordered, multistep process (31, 38). This pathway in- <br />    In some cases, the role of the Hsp90 machinery is also reg-                volves a dynamic association of Hsp90, Hsp70, p60, and p48 at <br /> ulatory. For example, while the chaperone is required to fold                 early or intermediate stages, while the final mature complex <br /> glucocorticoid receptor (GR) into its mature, ligand-respon-                  contains Hsp90, lower levels of Hsp70, p23, and an immu- <br /> sive conformation, it also ensures that the receptor remains in               nophilin (47). The mature complex enables the steroid recep- <br /> an inactive state in the absence of ligand (3, 6, 8). Similarly, the          tor to achieve a conformation responsive to ligand stimulation. <br /> association of Hsp90 with newly synthesized oncogenic tyro-                      The immunophilins associated with Hsp90 harbor peptidyl- <br /> sine kinase pp60v-src keeps the kinase inactive until it dissoci-             prolyl isomerase domains and constitute two distinct families: <br /> ates <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Hsp90 (7).                                                          FKBPs and cyclophilins, which are capable of binding the im- <br />    Recent discoveries have revealed additional regulatory roles               munosuppressive drugs FK506 and cyclosporin A, respectively <br /> for Hsp90 in signal transduction. For example, in endothelial                 (22, 29, 38). In addition to an isomerase domain, the Hsp90- <br /> cells, the induction of nitric oxide production elicits the binding           associated immunophilins also contain three tetratricopeptide <br /> of Hsp90 to nitric oxide synthase, increasing the activity of                 repeat (TPR) motifs. TPRs are involved in protein-protein <br />                                                                               interactions (27), and the immunophilins interact with Hsp90 <br />    * Corresponding author. Mailing address: Department of Biochem- <br />                                                                               via these domains (38). Biochemical experiments have shown <br /> istry, Molecular Biology and Cell Biology, 2153 Sheridan Rd., North-          that only a single species of TPR-containing protein is found in <br /> western University, Evanston, IL 60208. Phone: (847) 491-5452. Fax:           association with a particular Hsp90 complex (27, 36). Further- <br /> (847) 467-1422. E-mail: r-gaber@nwu.edu.                                      more, binding to Hsp90 by the immunophilins appears to be <br />  <br />                                                                        7353 <br />  7354       MARSH ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br /> competitive, providing support for the hypothesis that Hsp90                      trol vectors were introduced into wild-type cells as needed to maintain similar <br /> dimers generate a single TPR-accepting pocket (27, 36, 39).                       prototrophic backgrounds. <br />                                                                                      Screen for multicopy suppressors of cpr7D slow growth. hsc82D cpr7D cells <br />    S. cerevisiae harbors two genes, CPR6 and CPR7, that en-                       were transformed with plasmid DNA from a URA3-marked multicopy library of <br /> <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> Cyp40-type cyclophilins (18). Bolstered by biochemical                       S. cerevisiae genomic inserts and plated to selective media. Rapidly growing <br /> evidence showing that conservation of Hsp90 heterocomplexes                       transformant colonies were picked and, after streaking, assessed for growth rate <br /> between mammals and S. cerevisiae includes the presence of                        by comparing colony sizes to those of control cells treated similarly. The rapid <br />                                                                                   growth rate of the candidates was analyzed for plasmid dependency by plating to <br /> Cyp40-type cyclophilins (9), we showed that both Cpr6 and                         media containing 5-fluoro-orotic acid. Candidates that exhibited a return to the <br /> Cpr7 can interact directly with Hsp90 (Hsp82) (16). Although                      slow growth rate of hsc82D cpr7D cells on 5-fluoro-orotic acid were analyzed <br /> both Cpr6 and Cpr7 are components of Hsp90 heterocom-                             further. Plasmid DNA from positive candidates was rescued by transformation in <br /> plexes, Cpr7 appears to be functionally distinct from Cpr6, as                    Escherichia coli and reintroduced into cpr7D hsc82D and cpr7D recipients to <br />                                                                                   confirm the ability to suppress the growth defect of these cells. Plasmids dem- <br /> cells that harbor a <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>null allele</span> of CPR7 exhibit a slow-growth                     onstrating an ability to suppress cpr7D slow-growth phenotypes were sequenced <br /> defect not observed with cpr6D cells (18). In fact, S. cerevisiae                 at the junction of the genomic insert and analyzed by using BLAST and the <br /> cells lacking all eight of the cyclophilin genes exhibit only the                 Saccharomyces genome database. <br /> slow-growth phenotype conferred by the cpr7D <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation</span> (15).                          GR activity assay. Wild-type and cpr7D cells were transformed with the GR <br />                                                                                   expression plasmid p2A/GRGZ, which harbors the lacZ gene under the control <br /> Two lines of evidence indicate that the physical association                      of glucocorticoid response elements (32). A multicopy CNS1 plasmid, pJM107 or <br /> between Cpr7 and the Hsp90 machinery reflects a functional                        pJM25 (p2m-CNS1; isolated from the genomic library described above), was <br /> relationship. First, cells devoid of Cpr7 exhibit decreased                       introduced into these cells. Cells harboring either control plasmids or the mul- <br /> Hsp90-dependent functions including decreased activity of the                     ticopy CNS1 plasmids were assayed for b-galactosidase activity as described <br />                                                                                   previously (16, 19). <br /> heterologous Hsp90 substrates GR and the oncogenic kinase                            HSF activity assay. Wild-type and cpr7D cells harboring the HSF-dependent <br /> pp60v-src. Second, mutations such as hsc82D and sti1D, which                      reporter construct pHSE2-lacZ were transformed with the multicopy CNS1 <br /> decrease Hsp90 activity but do not by themselves impair                           plasmid pJM111. Logarithmically growing cultures were lysed and assayed for <br /> growth, result in a severe growth defect in cells lacking CPR7                    b-galactosidase activity as previously described (17). <br />                                                                                      Protein retention assays. Cell lysis for the protein retention assays was per- <br /> (16). Surprisingly, a derivative of Cpr7 lacking the isomerase                    formed as follows. Logarithmically growing cultures of these cells were harvest- <br /> domain retains the ability to interact with Hsp90 and is suffi-                   ed, washed, and suspended in lysis buffer (LyB; 10 mM Tris [pH 7.3], 50 mM <br /> cient to restore Cpr7-dependent functions to cpr7D cells (16,                     NaCl, 50 mM KCl, 10 mM MgCl2, 20% [wt/vol] glycerol, 1 mM dithiothreitol, <br /> 19).                                                                              aprotinin [0.4 mg/ml; Sigma], leupeptin [0.4 mg/ml; Sigma], antipain [0.5 mg/ml; <br />                                                                                   Sigma], 2 mM phenylmethylsulfonyl fluoride). Cells were lysed by using acid- <br />    To better understand the role(s) of Cpr7 in the function of                    washed beads (B. Braun Biotech International) with six cycles of vortexing for <br /> the Hsp90 chaperone and the requirement for Cpr7 in normal                        30 s followed by placement on ice for 60 s. Lysates were cleared by centrifuga- <br /> growth, we pursued the analysis of suppressors of the cpr7D                       tion, and protein concentrations were measured by using Bio-Rad protein assay <br /> slow-growth phenotype. The isolation and analysis of sponta-                      reagents. <br />                                                                                      Precipitation using glutathione S-transferase (GST) fusion proteins expressed <br /> neous suppressors identified three unlinked genes (50a). Here <br />                                                                                   in E. coli was performed as described previously (16). Briefly, glutathione-Sepha- <br /> we describe a multicopy suppressor encoded by the essential                       rose 4B beads (Pharmacia) were incubated with bacterial lysates containing <br /> open reading frame (ORF) YBR155w, which we have designat-                         either GST alone or GST-Cpr7 for 1 h at 4&#194;&#176;C with mixing, and the beads were <br /> ed CNS1 (for cyclophilin seven suppressor). We show that                          collected by centrifugation. After a brief wash, equal amounts of Sepharose- <br /> overexpression of CNS1 restores multiple Cpr7-dependent ac-                       bound fusion protein were added to reaction mixtures containing 53 BB (13 BB <br />                                                                                   consists of 20 mM Tris-HCl [pH 7.5], 150 mM KCl, 2 mM CaCl2, 2 mM MgCl2, <br /> tivities to cpr7D cells. Biochemical evidence indicating that                     5 mM dithiothreitol, 0.5 mM phenylmethylsulfonyl fluoride, and 0.5% Nonidet <br /> Cns1 is a new component of the Hsp90 chaperone machinery                          P-40), yeast lysate, and H2O for a final volume of 350 ml. After 1 h of incubation <br /> is also presented.                                                                at 4&#194;&#176;C, the Sepharose beads were collected and washed seven times in phos- <br />                                                                                   phate-buffered saline. Bound proteins were eluted in glutathione elution buffer <br />                                                                                   (20 mM reduced glutathione [Sigma] in 50 mM Tris-HCl [pH 8.0]). The eluate <br />                        MATERIALS AND METHODS                                      was separated from the Sepharose beads by brief centrifugation, and aliquots of <br />                                                                                   the supernatant were analyzed by sodium dodecyl sulfate-polyacrylamide gel <br />    Strains and plasmids. All S. cerevisiae strains used are derivatives of W303   electrophoresis (SDS-PAGE). <br /> background (leu2-112 ura3-1 trp1-1 his3-11,15 ade2-1 can1-100 GAL SUC2)              A plasmid expressing histidine-tagged Hsp82 (Hsp82FP) was used to analyze <br /> except for YBR155w/YBR12.05 (MATa/a cns1D::LEU2/CNS1 trp1/trp1 ura3/              components of Hsp90 complexes by affinity chromatography on a nickel resin. <br /> ura3 his3/his3 leu2/leu2; gift of T. Miosga), GPD::HSP82FP (MATa can1 ade2        Histidine-tagged Hsp82 expressed in S. cerevisiae cells as the sole source of <br /> leu2 his3 trp1 ura3 hsc82D::LEU2 hsp82D::LEU2 [pTGPD-H-Hsp82FP]; gift of          Hsp90 was used to collect Hsp90 protein complexes on a nickel affinity matrix <br /> S. Lindquist), and PJ69-4a (MATa trp1-901 leu2-3, 112 ura3-52 his3-200 gal4D      essentially as described previously (9). To detect Cns1 in Hsp90 complexes, wild- <br /> gal80D LYS2::GAL1-HIS3 GAL2-ADE2 met2::GAL7-lacZ; gift of P. James and            type cells or hsc82D hsp82D pTGPD-H-Hsp82FP cells were transformed with <br /> E. Craig) (26). Strains harboring deletions for HSC82, STI1, and/or CPR7 have     pJM111 (multicopy CNS1-3HA). A 50% slurry of Ni21-nitriloacetic acid beads <br /> been described previously (16, 18).                                               (100 ml; Qiagen) was washed three times with double-distilled H2O and equili- <br />    CNS1 was cloned into centromeric and multicopy plasmids. pJM105 is a           brated with LyB. Equal amounts of lysates were added to the beads and mixed <br /> URA3-marked centromeric plasmid that contains an epitope-tagged derivative of     for 30 min at 4&#194;&#176;C. The beads were harvested and washed twice with wash buffer <br /> Cns1 expressed under control of the CNS1 promoter. It was constructed as fol-     1 (LyB, 5 mM imidazole, 1% Triton X-100) and twice with wash buffer 2 (LyB, <br /> lows. CNS1 (lacking the 39 termination codon but containing 0.6 kb of the 59      10 mM imidazole, 0.2% Triton X-100). Proteins were eluted by incubation in 150 <br /> untranslated sequence) was PCR amplified and cloned into XhoI/EcoRI sites of      mM imidazole (Sigma) for 10 min at 4&#194;&#176;C with mixing. The imidazole-released <br /> pRS316. A 99-bp fragment encoding three tandem copies of the influenza virus      proteins were precipitated with 10% (vol/vol) trichloroacetic acid in the presence <br /> A hemagglutinin (HA) epitope was cloned in frame immediately downstream of        of 8 mg of carrier protein (b-lactoglobulin; Sigma) per ml upon centrifugation for <br /> CNS1 at the EcoRI/SpeI sites of pRS316. Expression of the Cns1-3HA fusion         10 min after 15 min on ice. The pellets were washed with 100% ethanol and air <br /> protein was confirmed by immunoblot analysis. The CNS1::3HA fusion was            dried before suspension in 13 sample buffer for SDS-PAGE analysis. <br /> subcloned into pRS426, a multicopy URA3-marked vector (creating pJM107),             Gels were stained with Coomassie blue or processed for immunoblot analysis. <br /> into pRS314, a centromeric TRP1-marked vector (creating pJM110), and into         Antibodies with specificity for Hsp82 (5) (gift of S. Lindquist), Ssa (9) (gift of <br /> pRS424, a multicopy TRP1-marked vector (creating pJM111). Expression of           E. Craig), Sti1 (10) (from D. Toft; gift of S. Lindquist), and HA (monoclonal <br /> Cns1-3HA from pJM107, pJM110, and pJM111 was confirmed by immunoblot              antibody 12CA5; BAbCo, Inc., Richmond, Calif.) were used. Enhanced chemi- <br /> analysis. pJM110 was used as template to subclone the CNS1::3HA (begin-           luminescence immunoblot detection reagents (Amersham) were used to detect <br /> ning with the 59 ATG of CNS1) fusion into the two-hybrid system bait vector       the antibodies. <br /> pGBD-C2, creating pJM123. Plasmids used in assays for GR activity (p2A/              Two-hybrid analysis. A modified two-hybrid system designed by James et al. <br /> GRGZ and pJM98), HSF activity (pHSE2-lacZ), and protein retention experi-         (26) was used to investigate interactions between Cns1 and Hsp90. This system <br /> ments (pTGPD-H-Hsp82FP, pJM5, and pGEX-2T) and plasmids overexpressing            utilizes three reporter genes (LYS2::GAL1-HIS3 GAL2-ADE2, and met2::GAL7- <br /> CPR7 (pAAD97), CPR6 (pJM90), and STI1 (pUSti1) have been described pre-           lacZ) to detect protein-protein interactions. Plasmid pJM123 expressing the bait <br /> viously (9, 10, 16, 19, 32, 49).                                                  GAL4BD (GAL4 DNA binding domain)-Cns1-3HA was transformed into the <br />    Yeast transformations were carried out by the standard lithium acetate         two-hybrid reporter strains PJ69-4a and HKY140 (a cpr7D::TRP1 derivative of <br /> method, and bacterial transformations were performed by electroporation. Con-     PJ69-4a). A prey construct expressing GAL4AD-Hsp82C (a fusion between the <br />  VOL. 18, 1998                                                                                             SUPPRESSION OF cpr7D BY CNS1                     7355 <br />  <br />  <br />                                                                                   ments encompassing the ORF YBR155w, identified through <br />                                                                                   the S. cerevisiae genome sequencing project (2); we designated <br />                                                                                   this ORF CNS1. Although we expected to obtain HSC82 by <br />                                                                                   this screen, it was not found among the suppressors, probably <br />                                                                                   because hsc82D cpr7D pHSC82 cells grow as slowly as cpr7D <br />                                                                                   cells. In contrast, when present on a multicopy plasmid, the <br />                                                                                   genomic fragment harboring CNS1 conferred growth rates in <br />                                                                                   both hsc82D cpr7D and cpr7D cells that were nearly equal to <br />                                                                                   wild-type rates (Fig. 1). <br />                                                                                      To confirm that the CNS1/YBR155w ORF was responsible <br />                                                                                   for the suppression of the cpr7D slow-growth phenotype, we <br />                                                                                   constructed a TRP1-marked multicopy plasmid harboring the <br />                                                                                   CNS1 ORF (pJM111) and tested its abilities to suppress the <br />                                                                                   lethality of a cns1D (ybr155wD::LEU2) mutation (2) and in- <br />                                                                                   crease the growth rate of a cpr7D recipient. Introduction of <br />                                                                                   pJM111 into a diploid heterozygous for the ybr155wD::LEU2 <br />                                                                                   mutation resulted in the occurrence of viable Leu1 spore col- <br />                                                                                   onies among the meiotic progeny, confirming that pJM111 <br />                                                                                   expressed a functional CNS1 gene. In contrast, control diploids <br />                                                                                   transformed with vector yielded only the expected 0:2 Leu1: <br />                                                                                   Leu segregation <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> among the offspring (data not shown). <br />                                                                                   The overexpression of CNS1 from pJM111 was able to sup- <br />                                                                                   press cpr7D-dependent slow-growth phenotypes. When intro- <br />                                                                                   duced into cpr7D hsc82D and cpr7D recipient cells, pJM111 was <br />                                                                                   able to increase the growth rate of these cells to nearly that <br />                                                                                   of wild-type cells (Fig. 1B). Thus, the ORF corresponding to <br />    FIG. 1. Overexpression of CNS1 suppresses the slow-growth phenotype of         CNS1/YBR155w is sufficient for suppression of the slow- <br /> cpr7D hsc82D and cpr7D cells. cpr7D hsc82D and cpr7D cells harboring multicopy    growth phenotype of cpr7D hsc82D and cpr7D cells. <br /> CNS1 plasmids were streaked to selective medium and incubated for 2 days at <br /> 30&#194;&#176;C. (A) A multicopy plasmid containing the genomic region encompassing <br />                                                                                      The growth rate of cpr7D cells can also be exacerbated by <br /> CNS1/YBR155w is able to suppress the slow growth of cpr7D hsc82D cells.           deletion of STI1 (16), which encodes a nonessential compo- <br /> Wild-type (HSC82 CPR7) and hsc82D cpr7D control cells harbor vectors only. (B)    nent of Hsp90 heterocomplexes (34). To test further the ability <br /> Suppression of slow-growth phenotype of cpr7D cells by a multicopy CNS1           of CNS1 overexpression to suppress growth defects conferred <br /> plasmid (pJM111). Wild-type and cpr7D control cells harbor vectors only. <br />                                                                                   by deletion of CPR7, we introduced pJM111 into a sti1D cpr7D <br />                                                                                   recipient strain and assessed the growth rates of the transfor- <br />                                                                                   mants by streaking for single colonies on plasmid-selective me- <br /> Gal4 activation domain and the last 182 amino acids of Hsp82) was isolated from <br /> a yeast genomic two-hybrid library and transformed into PJ69-4a and HKY140, <br />                                                                                   dium. Overexpression of CNS1 from pJM111 suppressed the se- <br /> both harboring pJM123. PJ69-4a and HKY140 cells containing both the Cns1          vere slow-growth phenotype of sti1D cpr7D cells (data not shown). <br /> bait (pJM123) and Hsp82C prey plasmids exhibited the His1 Ade1 LacZ1              Thus, overexpression of CNS1 can functionally replace Cpr7 even <br /> phenotype that resulted from activation of all three reporter genes. Cells har-   in backgrounds in which Cpr7 is nearly essential for growth. <br /> boring either bait or prey plasmids alone were His2 Ade2, although they ex- <br /> pressed low levels of b-galactosidase. Liquid b-galactosidase assays were con- <br />                                                                                      Although overexpression of CNS1 suppressed growth de- <br /> ducted as described above.                                                        fects that resulted from loss of CPR7, the overexpression of <br />                                                                                   two other Hsp90-associated proteins, Cpr6 and Sti1, failed to <br />                                                                                   confer even partial suppression of the cpr7D growth phenotype <br />                                 RESULTS <br />                                                                                   (unpublished results). <br />    Screen for multicopy suppressors of the cpr7D slow-growth                         Overexpression of CNS1 can suppress other cpr7D pheno- <br /> phenotype. Cpr7 is a component of the Hsp90 machinery and                         types. In addition to its role in normal growth, Cpr7 is required <br /> is required for normal growth (16, 18). In contrast, deletion of <br /> HSC82, one of the two S. cerevisiae genes encoding Hsp90, <br /> does not affect the growth rate of cells at normal temperatures <br /> even though the level of Hsp90 is reduced by 90% (5). How- <br /> ever, the cpr7D slow-growth phenotype is greatly exacerbated <br /> upon deletion of HSC82 (16). We took advantage of the severe <br /> growth phenotype of hsc82D cpr7D cells to identify genes whose <br /> overexpression could suppress the growth defect associated <br /> with the loss of Cpr7. hsc82D cpr7D cells were transformed with <br /> plasmid DNA from a multicopy S. cerevisiae library, and trans- <br /> formants that exhibited plasmid-dependent rapid growth (Fig. <br /> 1A) were retained for analysis. The plasmids harbored by the <br /> rapidly growing colonies were rescued, reintroduced into cpr7D <br /> hsc82D and cpr7D cells to confirm their ability to suppress the <br /> cpr7D-dependent growth phenotypes, and sequenced to identify                         FIG. 2. Overexpression of CNS1 restores GR activity to cpr7D cells. cpr7D <br /> the genomic region responsible for the suppression phenotype.                     cells harbored a GR-dependent lacZ reporter gene and either the control vector <br />    An exhaustive screen of the library revealed only two types                    (middle lane) or plasmids expressing CPR7 (pAAD97) or CNS1 (pJM107) as <br />                                                                                   indicated. Cells were treated with hormone and assayed for GR activity as <br /> of clones capable of suppressing the growth defect of cpr7D                       described in Materials and Methods. Each column is the mean 6 standard <br /> cells. One (11 isolates) contained genomic fragments encom-                       deviation of at least four independent experiments expressed as a percentage of <br /> passing the CPR7 gene. The other (nine isolates) contained frag-                  activity of cpr7D cells harboring pAAD97. <br />  7356       MARSH ET AL.                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     pressed in cpr7D cells, resulting in a significant increase in <br />                                                                                     steady-state HSF-dependent gene expression (17). Thus, as <br />                                                                                     another test of the functional relationship between Cns1 and <br />                                                                                     Cpr7, we assessed the ability of Cns1 to restore repression of <br />                                                                                     HSF activity in cpr7D cells. Wild-type and cpr7D cells harbor- <br />                                                                                     ing the plasmid-borne HSF-dependent reporter HSE2::lacZ <br />                                                                                     were transformed with the multicopy CNS1 plasmid pJM111 or <br />                                                                                     with vector and assayed for b-galactosidase activity. In these <br />                                                                                     experiments, HSE2::lacZ expression was eightfold higher in <br />                                                                                     vector-harboring cpr7D cells than in the wild-type CPR7 con- <br />                                                                                     trol cells. Although overexpression of CNS1 in wild-type cells <br />                                                                                     did not affect HSE2::lacZ expression, in cpr7D cells the in- <br />    FIG. 3. Overexpression of CNS1 restores down regulation of HSF activity in       creased CNS1 gene dosage largely restored repression of the <br /> cpr7D cells. Steady-state b-galactosidase activity in wild-type (CPR7) and cpr7D    reporter gene (Fig. 3). Thus, overexpression of CNS1 is able to <br /> cells harboring the HSF-dependent reporter plasmid pHSE2-lacZ and either            substitute for Cpr7 in the down regulation of HSF. <br /> vector alone or the multicopy CNS1 plasmid pJM111 (p2m-CNS1) was measured <br /> as described in Materials and Methods. Indicated activities are relative to those      Overexpression of Hsp90-associated TPR-containing pro- <br /> of wild-type cells. Each bar represents the mean 6 standard deviation of at least   teins is unable to suppress the <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> of cns1D cells. <br /> six independent experiments.                                                        CNS1/YBR155w is an essential gene encoding a protein 385 <br />                                                                                     amino acids in <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> that is weakly related to Sti1 (13% <br />                                                                                     identity) (2), the S. cerevisiae homolog of the Hsp90-associated <br /> for full activity of heterologously expressed Hsp90 substrates                      protein p60 (34). Cns1 harbors three TPR motifs in its amino- <br /> such as GR (16). To test the breadth of Cns1 involvement in                         terminal half which share significant sequence identity with <br /> Cpr7-dependent functions, pJM107, a URA3-marked multi-                              those of Cpr6 and Cpr7 (Fig. 4) and are also related to the <br /> copy plasmid expressing CNS1 was introduced into cpr7D re-                          TPR motifs in other Hsp90-associated proteins, including Sti1 <br /> cipient cells that express the heterologous GR and harbor the                       and Ppt1, the S. cerevisiae homolog of PP5. A search of the <br /> glucocorticoid response element GRE-lacZ reporter construct                         databases failed to identify proteins that share significant se- <br /> p2A/GRGZ. GR-promoted lacZ expression was measured and                              quence identity with the carboxy-terminal half of Cns1. <br /> compared to the activity in cpr7D cells and cpr7D cells express-                       To investigate further the relationship between Cns1 and the <br /> ing CPR7 from a centromeric plasmid. In these experiments                           Hsp90 machinery, other TPR-containing proteins known to <br /> GR activity was nearly fivefold lower in the cpr7D cells than in                    associate with the chaperone were tested for the ability, when <br /> the CPR7 control cells. In contrast, GR activity was increased                      overexpressed, to suppress the lethality of cns1D cells. Plas- <br /> to nearly wild-type levels in cpr7D cells harboring the multicopy                   mids overexpressing Cpr6, Cpr7, or Sti1 were introduced <br /> CNS1 plasmid (Fig. 2). Thus, increased gene dosage of CNS1                          into a CNS1/cns1D heterozygous diploid, transformants were <br /> is sufficient to substantially restore Cpr7-dependent Hsp90                         sporulated, and the meiotic progeny were analyzed. Overex- <br /> chaperoning of this heterologous substrate. Although overex-                        pression of CPR7, CPR6, or STI1 failed to suppress the cns1D <br /> pression of CNS1 restored GR activity to cpr7D cells, the                           lethal phenotype (data not shown) indicating that the essential <br /> overexpression of other Hsp90-associated proteins, namely,                          function performed by Cns1 cannot be replaced by these TPR- <br /> Cpr6 (19) and Sti1 (data not shown), could not do so.                               containing proteins. <br />    We recently discovered a specific endogenous role for Hsp90                         Cns1 is a component of the Hsp90 chaperone machinery. <br /> in regulation of the heat shock response in S. cerevisiae (17).                     Because Cpr7 is a component of Hsp90 heterocomplexes (16), <br /> Hsp90 negatively regulates the transcriptional activity of HSF                      the ability of multicopy CNS1 plasmids to suppress the cpr7D <br /> under both normal and stress conditions. Furthermore, Cpr7                          phenotypes suggested that Cns1 might physically interact with <br /> plays a major role in this regulation. HSF activity is dere-                        the chaperone complex. Physical association between Cns1 and <br />  <br />  <br />  <br />  <br />    FIG. 4. TPR motifs in Cns1. The location of each of the three TPR motifs in Cns1 is depicted by a hatched box. The TPR motifs of CN21 are compared with those <br /> of Cpr6 and Cpr7. Sites containing identical amino acids are shaded. Underlined amino acids correspond to residues that fit the TPR consensus motif *--*G-*Y/ <br /> F-----*--A*--Y/F--A*-*-P----- (24), where p represents any large hydrophobic amino acid and - represents any amino acid. <br />  VOL. 18, 1998                                                                                                     SUPPRESSION OF cpr7D BY CNS1                      7357 <br />  <br />  <br />                                                                                          severe growth defect of cpr7D hsc82D cells, overexpression of <br />                                                                                          CNS1 also suppressed the slow-growth phenotype of cells con- <br />                                                                                          taining only the cpr7D mutation. Increased CNS1 gene dosage <br />                                                                                          also restored other Cpr7-dependent activities in cpr7D cells, <br />                                                                                          including the activity of the heterologously expressed GR and <br />                                                                                          negative regulation of HSF, both of which are Hsp90 depen- <br />                                                                                          dent (6, 17). In addition, two-hybrid and in vitro protein-pro- <br />                                                                                          tein interaction analyses indicated that Cns1 can interact with <br />    FIG. 5. Precipitation of Cpr7 coprecipitates Cns1-3HA. Wild-type cells har- <br />                                                                                          one or more components of Hsp90 heterocomplexes. Thus, our <br /> boring the multicopy plasmid pJM111 expressing HA epitope-tagged Cns1 were               results show that Cns1 is a previously unidentified component <br /> cultured and lysates prepared as described in Materials and Methods. Cell lysates        of the Hsp90 chaperone complex that is essential for viability. <br /> were incubated with glutathione beads charged with either GST or GST-Cpr7                   The ability of increased CNS1 gene dosage to suppress dis- <br /> fusion protein. Proteins precipitated by either GST or GST-Cpr7 or total cellular pro- <br /> teins (Lysate) were identified by immunoblotting with the appropriate antibodies. <br />                                                                                          tinct cpr7D phenotypes suggests that Cns1 broadly substitutes <br />                                                                                          for Cpr7. While the precise function of Cpr7 in molecular <br />                                                                                          chaperoning remains unknown, the cyclophilin clearly plays a <br /> the Hsp90 machinery was tested in wild-type cells harboring a                            major role in the Hsp90 activities, as it is required not only for <br /> multicopy plasmid (pJM111) expressing HA epitope-tagged                                  rapid growth but also for full activity of GR and pp60v-src and <br /> alleles of CNS1 (see Materials and Methods). Lysates from S.                             for negative regulation of HSF (16&#226;&#8364;&#8220;18). Cns1 appears to be the <br /> cerevisiae cells expressing Cns1-3HA fusion protein were mixed                           only TPR-containing component of Hsp90 heterocomplexes <br /> with glutathione beads coated with either GST or GST-Cpr7                                that can substitute functionally for Cpr7. The overexpression <br /> fusion protein purified from bacterial extracts. The beads were                          of CPR6 or STI1 does not increase the growth rate or restore <br /> then washed, and bound proteins were eluted and analyzed.                                the GR activity of cpr7D cells (reference 19 and data not <br /> Immunoblots of the eluates revealed the presence of Hsp90                                shown). Despite an exhaustive search of a multicopy genomic <br /> and Ssa (an S. cerevisiae homolog of Hsp70), indicating that                             library, CNS1 was the only extragenic suppressor obtained. <br /> GST-Cpr7 precipitated Hsp90 heterocomplexes from the ly-                                    Two scenarios could explain the suppression of cpr7D phe- <br /> sates (Fig. 5). Immunoblotting with anti-HA antibody revealed                            notypes by Cns1. Since CNS1 acts as a suppressor of a null <br /> the presence of Cns1 among the precipitated proteins, dem-                               allele of CPR7, formally, Cns1 appears to act at or downstream <br /> onstrating that Cns1 physically interacts with Hsp90 or its as-                          of Cpr7. Thus, it is possible that the phenotypes that arise from <br /> sociated proteins. The fact that much less Cns1-3HA is ob-                               the loss of Cpr7 might actually be due to decreased Cns1 <br /> tained by precipitation with GST-Cpr7 than with Hsp90 (see                               activity and that the overexpression of CNS1 merely restores <br /> below) suggests that only a minority of Cns1-containing com-                             Cns1-dependent functions. Alternatively, the ability of Cns1 <br /> plexes also contain Cpr7. GST-Cpr7 fusion protein was also                               to suppress cpr7D phenotypes might reflect direct functional <br /> able to precipitate Cns1-3HA from lysates expressing the                                 overlap between the two proteins. In this case, because both <br /> epitope-tagged fusion protein from the low-copy-number plas-                             Cpr7 and Cns1 contain three-<span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> TPR domains and because <br /> mid pJM110 (not shown).                                                                  both are components of Hsp90 heterocomplexes, a reasonable <br />    A second approach was used to test Hsp90 complexes for                                scenario is one in which Cns1 substitutes for Cpr7 in binding to <br /> the presence of Cns1. Wild-type cells and cells expressing                               the TPR-accepting pocket(s) of Hsp90 dimers of the appropri- <br /> Hsp82FP, a histidine-tagged Hsp90 fusion protein, were trans-                            ate heterocomplexes, imparting upon them Cpr7-like func- <br /> formed with a low-copy-number (pJM110) or multicopy (pJM107                              tions. If so, it is not surprising that the overexpression of Sti1 <br /> or pJM111) CNS1::3HA plasmid. Hsp82FP but not Hsp82 was                                  cannot suppress the cpr7D phenotypes because data from in <br /> precipitated from lysates by a nickel matrix. Immunoblotting                             vitro reconstitution studies with mammalian Hsp90 hetero- <br /> of the eluates showed that Ssa and Sti1 coprecipitated with                              complexes indicate that p60 (Sti1) and the Cyp40-type cyclo- <br /> Hsp82FP (Fig. 6). Immunoblotting with anti-HA antibody re-                               philins are not found in the same heterocomplex (35). How- <br /> vealed that Cns1 also coprecipitated with the Hsp90 complex.                             ever, it is surprising that the closely related cyclophilin Cpr6 <br /> Thus, two protein retention assays revealed that Cns1 is a                               cannot substitute for Cpr7, even when overexpressed, as the <br /> component of the Hsp90 heterocomplexes.                                                  region containing the TPR motifs of Cpr6 is 34% identical to <br />    Finally, we tested for the ability of Cns1 to interact with                           that of Cpr7 whereas the TPR-containing regions of Cns1 and <br /> Hsp90 by the yeast two-hybrid assay (26). Using a full-length                            Cpr7 are only 9.6% identical. This is all the more surprising <br /> Cns1-3HA fusion protein expressed from pJM123 as bait, we                                given that overexpression of the TPR-containing carboxy ter- <br /> found that the carboxy-terminal 182 amino acids of Hsp82 <br /> were sufficient to strongly induce the HIS3, ADE2, and lacZ <br /> reporter genes (see Materials and Methods). Cells expressing <br /> GAL4BD-Cns1-3HA and GAL4AD-Hsp82C yielded 678 6 91 <br /> U of b-galactosidase activity (n 5 3), whereas cells harboring <br /> the GAL4AD-Hsp82C construct alone yielded only 26 6 3.6 U <br /> of activity (n 5 3). In addition, this interaction was not signif- <br /> icantly diminished by deletion of CPR7 (631 6 61 U; n 5 3). <br /> Thus, physical association between Cns1 and the Hsp90 com- <br /> plex is independent of Cpr7. <br />  <br />                                 DISCUSSION                                                  FIG. 6. Precipitation of Hsp90 coprecipitates Cns1-3HA. Cells expressing <br />                                                                                          wild-type Hsp90 (Hsp82) or histidine-tagged Hsp90 (Hsp82FP) were transformed <br />   CNS1 is an essential gene that we identified by a screen for                           with a low-copy-number (pJM110) or multicopy (pJM111) CNS1::3HA plasmid. <br />                                                                                          Total cellular protein (lanes 1 and 2) and proteins eluted from a nickel affinity <br /> multicopy suppressors of the severe slow-growth phenotype                                matrix (lanes 3 and 4) were analyzed by immunoblotting with the appropriate <br /> caused by deletion of the Hsp90-associated Cyp40-type cyclo-                             antibodies. Coprecipitation of Cns1-3HA was also observed from lysates express- <br /> philin Cpr7. Although isolated by its ability to suppress the                            ing the fusion protein from the low-copy-number plasmid pJM110 (not shown). <br />  7358     MARSH ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br /> minus of Cpr7 restores Cpr7-dependent functions to cpr7D                 with TPR-containing Hsp90-associated proteins FKBP51, <br /> cells (19). The inability of Cpr6 to replace Cpr7 does not               FKBP52, and hTom34p. <br /> appear to be due to an inhibitory function specific to the <br /> isomerase domain of Cpr6 since a Cpr6-Cpr7 chimera contain-                                                <br /> ing the amino-terminal half of Cpr6 and the carboxy-terminal  <br /> half of Cpr7 can functionally substitute for Cpr7 in vivo (un-   <br /> published results). The hypothesis that Cns1 might occupy the  <br /> TPR-accepting pocket(s) of the Hsp90 complex is also sup-   <br /> ported by recent work involving chimeric proteins between the  <br /> Hsp90-associated immunophilins FKBP51 and FKBP52 (1).       <br /> Individual chimeras displayed preferential binding to Hsp90-   <br /> receptor complexes, depending on which TPR-containing car-   <br /> boxyl termini were present.                              <br />    The repertoire of TPR proteins known to interact with     <br /> Hsp90 heterocomplexes was broadened by the recent discovery  <br /> that the mammalian protein phosphatase PP5, which harbors  <br /> four TPR units, interacts with Hsp90 (11, 46). Although it can     <br /> bind FK506 weakly, PP5 is not a member of the highly con-      <br /> served FKBP family of proteins (46). This was the first evi-    <br /> dence that a nonimmunophilin TPR-containing protein can   <br /> associate with Hsp90 in mature receptor complexes. Our dis-   <br /> covery (and that described in reference 14) that Cns1 is an     <br /> essential nonimmunophilin TPR-containing component of the   <br /> Hsp90 machinery suggests that these complexes are even more    <br /> heterogeneous than previously thought. Whether Cns1 occu-     <br /> pies specific niches among different types of heterocomplexes         <br /> or even defines such complexes remains to be determined. The   <br /> mammalian Hsp90-associated immunophilins do not precipi-         <br /> tate with other Hsp90-associated immunophilins, indicating           <br /> that TPR-containing immunophilins are not present in the                <br /> same complex (see references 37 and 47 for reviews). Based on      <br /> these and other studies, the stoichiometry of Hsp90-receptor                <br /> complexes has been postulated to include one TPR-containing                <br /> immunophilin per Hsp90 dimer (12, 39, 47). However, we                    <br /> found Cns1 among the proteins that could be removed from           <br /> solution by GST-Cpr7. Thus, if Cns1 interacts directly with             <br /> Hsp90 via its TPR domains, the one-TPR-accepting-pocket&#226;&#8364;&#8220;                <br /> one-TPR-partner hypothesis may be insufficient to explain the              <br /> precipitation of Cns1 by Cpr7.                                         <br />                                                                        <br />    Although Hsp90-associated proteins are highly conserved     <br /> among eukaryotes, in S. cerevisiae many, including Sti1 (p60),            <br /> Cpr6 and Cpr7 (Cyp40), Ydj1 (DnaJ), Sba1 (p23), and Ppt1                <br /> (PP5), are not essential (16, 20, 28, 34, 46). In contrast, Cns1       <br /> is essential at all temperatures (2). Moreover, the lethality                <br /> caused by the cns1D mutation cannot be suppressed by over-              <br /> expression of other TPR-containing proteins, including Cpr6,              <br /> Cpr7, or Sti1. Taken together, these observations suggest a key           <br />                                                                            <br /> functional role for Cns1 in Hsp90 chaperone complexes.                  <br />   <br /> </body></html>