<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2006 Genetics Society America <br /> DOI: 10.1534/genetics.106.057893 <br />  <br />  <br />  <br />   Glial Neuronal Functions Drosophila Homolog Human <br />    SWI/SNF Gene ATR-X (DATR-X ) jing Zinc-Finger Gene Specify <br />            Lateral Positioning Longitudinal Glia Axons <br />  <br />                                Xuetao Sun, Tatiana Morozova Margaret Sonnenfeld1 <br />                  Department Cellular Molecular Medicine, University Ottawa, Ottawa, Ontario K1H 8M5, Canada <br />                                                         Manuscript received March 3, 2006 <br />                                                       Accepted publication April 27, 2006 <br />  <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Neuronal&#226;&#8364;&#8220;glial communication essential constructing orthogonal axon scaffold the <br />               developing Drosophila central nervous (CNS). Longitudinal glia (LG) guide extending <br />               commissural longitudinal axons pioneer commissural neurons maintain glial <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>survival</span> <br />               positioning. However, transcriptional regulatory mechanisms controlling processes not <br />               known. Previous studies showed midline function jing C2H2-type zinc-finger transcription <br />               factor partially required axon scaffold formation Drosophila CNS. We therefore <br />               screened gain-of-function enhancers jing gain function eye identified Drosophila <br />               homolog disease gene human a-thalassemia/mental retardation X-linked (ATR-X) as <br />               genes potential roles gene expression, translation, synaptic transmission, cell cycle. jing <br />               DATR-X reporter genes expressed CNS neurons glia, including LG. Coexpression <br />               jing DATR-X embryonic neurons synergistically affects longitudinal connective formation. <br />               During embryogenesis, jing DATR-X autonomous nonautonomous roles lateral <br />               positioning LG, neurons, longitudinal axons shown cell-specific knockdown gene <br />               expression. jing DATR-X required autonomously glial survival. jing DATR-X mutations <br />               synergistic effects longitudinal axon formation suggesting functionally related. <br />               These observations support model downstream gene expression controlled potential <br />               DATR-X&#226;&#8364;&#8220;Jing complex facilitates cellular positioning axon guidance, ultimately allowing proper <br />               connectivity developing Drosophila CNS. <br />  <br />  <br />  <br />  <br /> D    URING central nervous (CNS) develop- <br />        ment, axons navigate long distances are <br /> faced attractive repulsive guidance cues <br />                                                                             (Kidd et al. 1998a; Kinrade et al. 2001). Signaling and <br />                                                                             cell&#226;&#8364;&#8220;cell contact maintain ipsilateral positions of <br />                                                                             LG connectives. In fact, Slit&#226;&#8364;&#8220;Robo signaling <br /> that properly interpreted (Tessier-Lavigne                          cancels attraction longitudinal axons the <br /> and Goodman 1996). Many interneurons, cell                            CNS midline Netrin&#226;&#8364;&#8220;Frazzled (Bhat 2005). Com- <br /> bodies lie CNS midline, project                      missureless (Comm) transmembrane protein that <br /> axons midline form commissural                            prevents delivery Robo growth cones, <br /> tracts. The &#226;&#8364;&#732;&#226;&#8364;&#732;decision&#226;&#8364;&#8482;&#226;&#8364;&#8482; axon cross midline                 specifically commissural neurons, allowing their <br /> the Drosophila ventral nerve cord (VNC)                             axons cross midline (Tear et al. 1996; Keleman <br /> vertebrate spinal cord depends differential                          et al. 2002; Keleman et al. 2005). A downregulation of <br /> response axons midline repellent Slit                      Robo genetic means overexpression comm <br /> the attractant Netrins (Seeger et al. 1993; Tessier-                        results excess axons CNS midline (Kidd <br /> Lavigne 1994; Battye et al. 1999; Kidd et al. 1999;                         et al. 1998b). Therefore, differential localization of <br /> Long et al. 2004; Bhat 2005). After commissural axons                       Comm, Robo, Slit determines directions <br /> cross midline, turn fasciculate                        navigating axons scaffold follow. The Slit&#226;&#8364;&#8220; <br /> longitudinal tracts run parallel midline                    Robo important conserved mechanism <br /> are repelled midline Slit.                                      establish cellular positioning boundaries the <br />    The ligand Slit Roundabout (Robo),                        developing vertebrate invertebrate nervous systems <br /> located longitudinal glia (LG) associated                        (Kidd et al. 1999; Rajagopalan et al. 2000a,b; Simpson <br /> pioneer neuron growth cones adjacent midline                         et al. 2000a,b; Rasband et al. 2003; Barresi et al. 2005). <br />                                                                                The relationship neurons glia the <br />   1 <br />                                                                             formation Drosophila CNS axon tracts been <br />    Corresponding author: Department Cellular Molecular Medicine, <br /> University Ottawa, Ottawa, Ontario K1H 8M5, Canada.                      extensively studied genetic cell ablation methods <br /> E-mail: msonnenf@uottawa.ca                                                 (Hidalgo Brand 1997, 2000; Booth et al. 2000). <br />  <br /> Genetics 173: 1397&#226;&#8364;&#8220;1415 ( July 2006) <br />  1398                                      X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> The longitudinal axon tracts constructed ex-        disease gene MR syndromes <br /> tensions pioneer neurons (Bate Grunewald           (Gibbons et al. 1995b; Villard et al. 1996a,b; Abidi <br /> 1981; Jacobs Goodman 1989; Hidalgo Brand               et al. 1999; Xue et al. 2003; Tang et al., 2004). <br /> 1997). <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> form longitudinal fascicle, dMP2              To explore transcriptional mechanisms control- <br /> MP2 pioneer neurons extend axons posteriorly          ling axon patterning, investigated cell-specific <br /> contact anteriorly projecting growth cones          roles jing DATR-X regulating axon formation in <br /> vMP2 pCC neurons ( Jacobs Goodman 1989).               embryonic CNS. The roles DATR-X jing the <br /> In hemisegment, LG act mobile guideposts           CNS studied reducing expression specif- <br /> the migrating axons (Hidalgo Booth 2000). Early            ically neurons glia using <span id='am-394' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-395' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) <br /> ablation LG affects joining descending       analyzing GOF effects gene. We show <br /> ascending pioneer growth cones subsequent              jing DATR-X (1) autonomous roles CNS <br /> fasciculation defasciculation pioneer later         glial survival, (2) autonomous nonautonomous <br /> follower axons (Booth et al. 2000; Hidalgo Booth           roles LG axon positioning, (3) autonomous <br /> 2000). Despite important guidance role LG,          nonautonomous roles longitudinal axon out- <br /> these cells depend pioneer axons survival         growth. The phenotypes jing DATR-X mutations <br /> (Hidalgo et al. 2001; Kinrade et al. 2001). In addi-           derive perturbation extensive neuronal&#226;&#8364;&#8220; <br /> tion, contralateral neuron cell bodies needed          glial communication mechanisms govern CNS <br /> axon pathfinding longitudinal connective              axon scaffold formation. Early ubiquitous expression <br /> (Whitington et al. 2004). The proper migration              DATR-X mRNA enrichment transcripts and <br /> follower glia fly optic lobe requires preexisting     reporter gene expression embryonic neurons and <br /> photoreceptor axon scaffold (Hidalgo et al. 2001;              glia reveal similarity expression <span id='am-473' about='xsp:pattern' typeof='owl:Thing'>pattern</span> jing <br /> Kinrade et al. 2001; Dearborn Kunes 2004).                 (Sedaghat et al. 2002). The combined effects jing <br /> Therefore, neuronal&#226;&#8364;&#8220;glial interactions, addition         DATR-X knockdown GOF synergistic and <br /> guidance molecules, instrumental axon               phenotypes resulting single gene knockdown <br /> patterning (Oland Tolbert 2002).                           similar, providing strong evidence jing and <br />    The jing gene originally identified genetic      DATR-X work together. Therefore, jing DATR-X <br /> screens regulators border cell migration         function instrumental neuronal&#226;&#8364;&#8220;glial commu- <br /> ovary midline cell development embryo-          nication mechanisms govern CNS axon scaffold <br /> genesis (Liu Montell 2001; Sedaghat et al. 2002).          formation. <br /> During embryogenesis, jing transcripts accumulate in <br /> the CNS midline, adjacent neuroectoderm, brain, and <br /> trachea (Sedaghat et al. 2002; Sonnenfeld et al. 2004).                      MATERIALS AND METHODS <br /> In CNS midline trachea, Jing functions down- <br /> stream basic helix-loop-helix PAS (Per&#226;&#8364;&#8220;Arnt&#226;&#8364;&#8220;Sim)-          Drosophila strains: All flies raised standard <br />                                                                Drosophila cornmeal medium 25&#194;&#176; (Ashburner 1989). <br /> containing (bHLH-PAS) transcription factors control         The collection chromosome transgenic EP strains, <br /> tyrosine kinase signaling epidermal growth         generated described R&#195;&#184;rth (1996), obtained <br /> factor receptor (Egfr) fibroblast growth factor re-        Hungarian Szeged stock center. GMR-Gal4 (second <br /> ceptor Breathless (Sedaghat et al. 2002; Sonnenfeld            chromosome) obtained J. Nambu (Hay et al. 1994; <br /> et al. 2004). In CNS midline, jing required com-    Wing et al. 2002) paired (prd)-Gal4 (third chromosome) <br />                                                                obtained S. Crews. The driver, ELAV-Gal4 (on X), <br /> missural longitudinal axon formation midline           used drive expression neurons promoter <br /> expression wild-type jing does completely rescue        fusion embryonic lethal abnormal vision gene <br /> axon defects mutants, suggesting functions       (Robinow White 1988; Yao White 1994); was <br /> of Jing contribute axon tract formation (Sonnenfeld         obtained Bloomington stock center. GCM-Gal4 <br /> et al. 2004).                                                  drives expression CNS glia expressing glial cells <br />                                                                missing (GCM) gene obtained M. Freeman <br />    In search additional factors important jing       (Freeman et al. 2003; Hosoya et al. 1995; Jones et al. 1995;). <br /> function, carried genetic screen identify          btl-Gal4 used drive expression trachea (Shiga <br /> genes function modify jing gain-      et al. 1996). <br /> of-function (GOF) assay developing eye-imaginal            UAS-jingE, jing EMS alleles, jing deficiency [Df <br /> disc. Seven chromosome enhancer/promoter-                (2R)ST1] previously described (Sedaghat et al. 2002; <br />                                                                Sonnenfeld et al. 2004). UAS-jingU second chromosome <br /> tagged (EP) genes identified GOF en-                insertion expresses jing determined situ <br /> hanced jing. This group genes specifically          hybridization. <br /> interacts jing ommatidial              Genetic crosses: Balancers detected using Cyo wingless- <br /> formation. Of these, identified Drosophila              lacZ TM3 ubx-lacZ. DATR-X jing overexpressed in <br /> homolog disease gene human a-thalassemia/            CNS neurons crossing homozygous ELAV-Gal4 homo- <br />                                                                zygous UAS-jingE, EP(3)0635, UAS-DATR-X. For coexpres- <br /> mental retardation (MR) X-linked (DATR-X) (Gibbons             sion experiments, flies carrying ELAV-Gal4 UAS-jingE were <br /> et al. 1995b). The human ATR-X gene encodes zinc-            crossed flies homozygous EP(3)0635. Controls in- <br /> finger ATPase involved chromatin remodeling         cluded ELAV-Gal4/1, UAS-jingE/1, EP(3)0635/1. For <br />                                                  Lateral Positioning LG Axons                                           1399 <br />  <br /> double RNAi flies carrying ELAV-Gal4 UAS-jing RNAi         sequence tags (ESTs) SD07188 LD28477] obtained <br /> crossed flies carrying UAS-DATR-XRNAi.                           Open Biosystems jing cDNA previously obtained <br />    EP screen: Males EP line crossed virgin        Research Genetics (Sedaghat et al. 2002). DATR-X EST <br /> females carrying GMR-Gal4. The eye morphologies F1 male          SD07188 contains 2382 bp DATR-X coding sequence beginning <br /> progeny cross analyzed dissecting            <span id='am-472' about='xsp:base' typeof='owl:Thing'>base</span> pair 1968 respect full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> LD28477 cDNA <br /> microscope male flies straight wings             sequence. DATR-X clones SD07188 LD28477 subjected <br /> crossed virgin females carrying UAS-jingE. One copy           DNA sequence analysis compared Gadfly <br /> UAS-jingE associated rough eye <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>          database (Celera genomics) (Adams et al. 2000). Homology <br /> therefore rough eye phenotype combination             searches using BLAST (http://www.ncbi.nlm.nih.gov). <br /> particular EP line GMR-Gal4 represented interaction.         Full-length DATR-X sequence (accession AAL13821; FlyBase) was <br /> The number flies rough eyes determined          aligned Caenorhabditis elegans (AAD55361), mouse <br /> dissecting microscope 600 progeny               (AAC08741), human (AAB49970) using Clustal X (1.83). <br /> analyzed trials 25&#194;&#176;. Controls included       Protein sequences compared using BL2SEQ (http://www. <br /> each trial included GMR-Gal4/UAS-jingE GMR-Gal4/1               ncbi.nlm.nih.gov/blast/bl2seq/wblast2.cgi) (Tatusova and <br /> and did rough eye 25&#194;&#176;. As additional control,      Madden 1999). Protein domains analyzed using <br /> eyes inspected flies heterozygous UAS-jingE       SMART PROSITE (Hulo et al. 2004; Letunic et al. 2004). <br /> the enhancing EP line. Enhancing lines subjected               Full length DATR-X cDNA (LD28477) subcloned into <br /> three genetic trials GMR-Gal4 UAS-jingE 25&#194;&#176;.       pUAST vector (Brand Perrimon 1993) EcoRI&#226;&#8364;&#8220; <br /> The penetrance determined number rough eyes           KpnI fragment used generate germline transformant <br /> divided 25% carried GMR-Gal4, UAS-jingE,        flies standard procedures (Genetic Services, Cambridge, <br /> particular EP strain. A total 591 individual chromo-       MA) (Rubin Spradling 1983). Expression DATR-X was <br /> some EP lines screened. Enhancer EP lines tested          confirmed multiple lines situ hybridization embryos <br /> for interactions inter se additional EP lines expressing   expressing different DATR-X upstream activating sequence <br /> DNA regulatory genes random EP lines                transgenes control prd gene transactivator (prd- <br /> method described above.                                          GAL4). <br />    Microscopy: For scanning electron microscopy (SEM), 1-           In situ hybridization reporter gene construction: <br /> 3-day-old adult heads dissected, fixed, dehydrated      Whole-mount situ hybridization carried using <br /> 15-min incubations graded ethanol series. Dehydrated           digoxigenin-labeled RNA probes according Janody et al. <br /> heads sent Mount Sinai Bioimaging Center                (2000). The DATR-X riboprobe generated linearizing <br /> (Toronto) sputter coating gold-palladium SEM           SD07188 EST HaeII. Sense probes synthesized <br /> examination. Three eyes sample examined              using T7 promoter antisense using Sp6 according <br /> and processed using Adobe Photoshop software. For light             manufacturers&#226;&#8364;&#8482; specifications (Roche). Probe specificity <br /> microscopy, 1- 3-day-old adult heads dissected,             tested GOF DATR-X embryos carrying prd-Gal4 <br /> analyzed Zeiss Axioskop, images captured           driver EP(3)0635 UAS-DATR-X transgenes. Third instar <br /> a Nikon DXM1200 digital camera processed using Adobe            larval nerve cords dissected PBS fixed 4% <br /> Photoshop software. Confocal microscopy carried        formaldehyde 20 min. Fixed nerve cords rinsed three <br /> BioRad1024 microscope using 340 water immersion [1.15               times fresh PBS washed 1 hr. Nerve cords were <br /> numerical aperture (NA)] 320 (1.1 NA) objectives.                subjected situ hybridization described above. <br />    Antibodies immunostaining Drosophila embryos:             Digoxigenin-stained nerve cords embryos mounted <br /> The following antibodies obtained Develop-            80% glycerol examined Zeiss Axioskop 2. <br /> mental Studies Hybridoma Bank (Iowa City, IA): BP102 (1:10),           A total 1553 bp 59 region jing obtained <br /> anti-Repo (8D12, 1:10) (Campbell et al. 1994), 1D4 (Fasciclin       PCR using Drosophila w1118 genomic DNA template, <br /> II, 1:10)(Van Vactor et al. 1993), anti-Robo (13C9, 1:10)       sequenced, cloned KpnI NotI sites the <br /> (Kidd et al. 1998a). A rabbit antibody b-galactosidase      pCaSper vector vivo expression. A total 593 bp were <br /> (Promega) used 1:100 dilution. HRP-conjugated goat           obtained PCR 59 DATR-X region surrounding <br /> anti-mouse IgG used 1:100. Texas red FITC-               EP element insertion site. This fragment cloned into <br /> conjugated secondary antibodies used confocal              EcoRI site pCaSper. To generate transgenic strains, the <br /> microscopy. Monoclonal antibody 2A12 used visualize          pCaSper plasmids co-injected plasmid <br /> tracheal tubules (1:3).                                             encoding D2-3 transposase w1118 embryos P-element- <br />    Embryos fixed PBS containing 4% formaldehyde             mediated transformation (Genetic Services) (Rubin and <br /> and embryos incubated overnight primary anti-             Spradling 1983). Multiple strains analyzed lacZ <br /> bodies PTN (13 PBS, 0.1% Triton X-100, 5% normal goat            expression. <br /> serum). Horseradish peroxidase-conjugated secondary anti-              RNA interference: For DATR-X RNAi, 697-bp sequence <br /> bodies used H2O2/diaminobenzedine histochemis-            generated PCR base pair positions 3181&#226;&#8364;&#8220;3837 <br /> try. Stained embryos dehydrated ethanol             using LD28477 cDNA template. For RNAi jing, 597-bp <br /> series, mounted methyl salicylate, examined No-           sequence generated PCR base pair position 2289 <br /> marski optics Zeiss Axioskop microscope. Images           ATG LD36562 cDNA. The DNA fragments <br /> captured Nikon DXM1200 digital camera processed            individually subcloned sequentially, opposite direc- <br /> using Adobe Photoshop software.                                     tions, AvrII NheI sites pWIZ plasmid (a gift <br />    Molecular genetic analysis DATR-X generation           R. Carthew) inverted sequence direc- <br /> UAS lines: Genomic cDNA sequences encoding Dro-             tionally cloned using XbaI NheI restriction enzymes (Lee <br /> sophila DATR-X homolog identified searching             Carthew 2003). The final inverted repeat containing <br /> Berkeley Drosophila Genome Project (BDGP) database          vectors UAS-DATR-X RNAi UAS-jing RNAi confirmed <br /> EP(3)0635 associated gene (Adams et al. 2000; Rubin et al. 2000).   sequence analysis. To generate transgenic strains, UAS- <br /> The location EP element EP(3)0635 confirmed        DATR-X RNAi UAS-jing RNAi plasmids, co-injected <br /> PCR using inverted repeat primers flanking genomic DATR-X       plasmid encoding D2-3 transposase w1118 <br /> sequence (CGACGGGACCACCTTATGTTATTTCATCATG) fol-                     embryos P-element-mediated transformation (Genetic <br /> lowed DNA sequence analysis. DATR-X cDNAs [expressed             Services) (Rubin Spradling 1983). <br />  1400                                         X. Sun, T. Morozova M. Sonnenfeld <br />  <br />    Several independent UAS-DATR-X RNAi UAS-jing RNAi                                       TABLE 1 <br /> transformant lines bred homozygosity, mapped, and <br /> tested double-stranded RNAi activity crossing CNS-                 EP lines enhanced jing gain function <br /> (ELAV-Gal4) tracheal-specific Gal4 (btl-Gal4) drivers and <br /> immunostaining monoclonal antibodies 1D4 2A12,            EP line              Gene              Predicted function <br /> respectively. To confirm downregulation mRNA expression, <br /> embryos carrying armadillo-Gal4 (Ahmad Henikoff 2001)          EP(3)0635     ATR-X/XNP             Chromatin remodeling <br /> and UAS-DATR-X RNAi UAS-jing RNAi subjected             EP(3)0473     D1                    Chromatin remodeling <br /> in situ hybridization using DATR-X jing riboprobes, re-         EP(3)3145     Dataxin-2             Translational regulation <br /> spectively. The number stage 13 embryos reduced            EP(3)3060     lap                   Receptor-mediated <br /> expression transgene- Gal4 driver-carrying embryos                                            endocytosis <br /> compared control embryos carrying          EP(3)3705     CG32137               Microtubule binding <br /> driver transgene alone.                                                                           (cell cycle) <br />    Quantitative analysis longitudinal axon glial cell       EP(3)3354     CG17383 ( JIGR-1)     Transcriptional regulation <br /> defects: The number crossovers segment counted          EP(3)3084     CG15507               Unknown <br /> in stage 16/17 control embryos expressing multiple <br /> jing DATR-X RNAi transgenes driven ELAV-           Rough eyed flies scored population carrying GMR- <br /> Gal4 GCM-Gal4 stained 1D4 anti-Robo.            Gal4, UAS-jing, EP line dissecting micro- <br /> Four independent transgenic lines controls ana-           scope. Crosses repeated times 25&#194;&#176; total- <br /> lyzed. Neuronal expression pWiz-DATR-X022, pWiz-DATR-           ing sample size .1200 flies. <br /> X28-MO1, pWiz-DATR-X26-MO2, pWiz-DATR-X28-MO2 <br /> resulted 20.9% (n &#194;&#188; 200), 22% (n &#194;&#188; 178), 27% (n &#194;&#188; 187), <br /> and 30% (n &#194;&#188; 145) embryos axon defects, respectively.         An enhancer screen carried using third <br /> Neuronal expression pWiz-jingNC1A, pWiz-jingORO3,               chromosome collection EP lines (see materials and <br /> pWiz-jingNA2, pWiz-jing006 resulted 25% (n &#194;&#188; 180), 29%          methods) (R&#195;&#184;rth 1996). Of 591 third-chromosome EP <br /> (n &#194;&#188; 210), 28% (n &#194;&#188; 168), 31% (n &#194;&#188; 201) embryos        lines screened, 7 repeatedly interact <br /> axon defects, respectively. <br />                                                                    genetically coexpressed jing eye under <br />    Quantitative analysis glia carried light <br /> microscopy anti-Repo stained whole-mount embryos           control GMR-Gal4 (Table 1; Figure 1). Eye-specific <br /> stage 11 16. Glia counted nerve cord segment       expression copy UAS-jing transgene no <br /> of embryos expressing independent jing DATR-X RNAi         visible effects ommatidial formation (Figure 1B) as <br /> transgenes driven ELAV-Gal4 GCM-Gal4.           compared eyes heterozygous GMR-Gal4 (Figure <br /> Controls included embryos heterozygous RNAi <br />                                                                    1A). However, coexpression jing EP(3)0635, <br /> transgene, Gal4 driver w1118. Standard deviations <br /> in glial numbers sample determined using              controls expression Drosophila homolog <br /> Microsoft Excel sample sizes exceeded 250 nerve cord           human ATR-X (DATR-X), disrupted ommatidial for- <br /> segments. Differences number glia sample         mation (Figure 1D). Eye-specific expression EP(3)0635 <br /> relative controls evaluated significance using         effect (Figure 1C). jing interacted with <br /> Student&#226;&#8364;&#8482;s paired t-test. Any space longitudinal con- <br />                                                                    4 EP lines, including EP(3)3145, EP(3)3705, <br /> nectives considered longitudinal break. <br />                                                                    EP(3)3354, EP(3)0473, resulting rougher eyes <br />                                                                    (Figure 1, F, K&#226;&#8364;&#8220;M). A loss pigmentation occurred in <br />                                                                    case EP(3)3354. These lines effect when <br />                          RESULTS <br />                                                                    expressed eye (Figure 1N; Table 2). <br />    Identification Drosophila homolog human                  Two additional EP lines interacted jing <br /> ATR-X additional genes involved jing function:              caused rough eye phenotype expressed alone <br /> Our approach identify genes interacting jing           eye, including EP(3)3084 (Figure 1G) and <br /> to use EP collection strains including genes tagged         EP(3)3060 (Figure 1I). GMR-Gal4-driven expression of <br /> by insertion transposable element containing UAS              EP(3)3084 associated morphological defects, <br /> sites (Szeged stock center, Szeged, Hungary) (Brand                mild reductions pigmentation, glossy eyes in <br /> and Perrimon 1993; R&#195;&#184;rth 1996). We reasoned                   100% flies (Figure 1, G N; Figure 2A). However, <br /> jing GOF dosage effects larval eye            coexpression EP(3)3084 jing resulted an <br /> could easily scored adult neural              enhancement phenotype eye size was <br /> tissue sensitive studies transcriptional regulatory       reduced, eyes rougher, pigmentation was <br /> mechanisms (Rubin 1988; Zipursky Rubin 1994)                   completely lost (Figure 1H; Figure 2B). A total of <br /> and jing neuronal function (Sedaghat et al. 2002).           35% eyes phenotype enhanced (n &#194;&#188; 345). <br />  <br />                                                                                                                          &lt; <br />    Figure 1.&#226;&#8364;&#8221;Identification enhancer/promoter (EP) lines enhance jing gain &#226;&#8364;&#8220;of function eye. (A&#226;&#8364;&#8220;M) Scanning <br /> electron micrographs adult eyes following genotypes shown: (A) GMR-GAL4/1; (B) GMR-GAL4/1; UAS-jingE/1; (C) <br /> GMR-GAL4 /1; EP(3)0635/1; (D) GMR-GAL4/1; UAS-jingE /1; EP(3)0635/1; (E) GMR-GAL4/1; EP(3)3145/1; (F) GMR- <br /> GAL4/1; UAS-jingE /1; EP(3)3145/1; (G) GMR-GAL4/1; EP(3)3084/1; (H) GMR-GAL4/1; UAS-jingE /1; EP(3)3084/1; <br /> (I) GMR-GAL4/1; EP(3)3060/1; (J) GMR-GAL4/1; UAS-jingE /1; EP(3)3060/1; (K) GMR-GAL4/1; EP(3)3705 /1; UAS- <br /> jingE/1; (L) GMR-GAL4/1; UAS-jingE /1; EP(3)3354/1; (M) GMR-GAL4/1; UAS-jingE /1; EP(3)0473/1. (N) Penetrance <br /> values enhancing phenotypes. Bars, 100 mm. <br />  Lateral Positioning LG Axons   1401 <br />  1402                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br />                                                             TABLE 2 <br />                                          Random screening jing interacting EP lines <br />  <br />                          Eye phenotype            Sample size                             Eye phenotype          Sample size <br />                                                                       P[GMR-Gal4]- <br /> P[GMR-Gal4] 3                                                         EP(3)0473 3 <br />   EP(3)3339            Wild   type                    250               EP(3)3339           Rough                 200   (60) <br />   EP(3)3303            Wild   type                    178               EP(3)3303           Rough                 250   (40) <br />   EP(3)3258            Wild   type                    189               EP(3)3258           Rough                 214   (37) <br />   EP(3)3916            Wild   type                    201               EP(3)3916           Rough                 210   (23) <br />   EP(3)893             Wild   type                    200               EP(3)893            Rough                 276   (16) <br />   EP(3)3377            Wild   type                    215               EP(3)3377           Rough                 234   (42) <br />  <br /> P[GMR-Gal4]-                                                          P[GMR-Gal4]- <br /> EP(3)3084 3            Glossy, Ypigment               400             EP(3)3060 3           Rough                 450 <br />   EP(3)3339            No change                      260               EP(3)3339           Enhanced               76   (20) <br />   EP(3)3303            No change                      201               EP(3)3303           Enhanced               60   (53.3) <br />   EP(3)3258            No change                      192               EP(3)3258           Enhanced               60   (6.7) <br />   EP(3)3916            No change                      189               EP(3)3916           No change             150 <br />   EP(3)893             No change                      234               EP(3)893            No change             200 <br />   EP(3)3377            No change                      250               EP(3)3377           Enhanced              160   (75) <br />  <br /> P[GMR-Gal4];                                                          P[GMR-Gal4]; <br /> EP(3)0635 3            Wild   type                    650             EP(3)3145 3           Wild   type           630 <br />   EP(3)3339            Wild   type                    251               EP(3)3339           Wild   type           151 <br />   EP(3)3303            Wild   type                    234               EP(3)3303           Wild   type           145 <br />   EP(3)3258            Wild   type                    243               EP(3)3258           Wild   type           130 <br />   EP(3)3916            Wild   type                    267               EP(3)3916           Wild   type           125 <br />   EP(3)893             Wild   type                    154               EP(3)893            Mild   rough          218 (6.4) <br />   EP(3)3377            Wild   type                    134               EP(3)3377           Wild   type           245 <br />  <br /> P[GMR-Gal4];                                                          P[GMR-Gal4]; <br /> EP(3)3354 3            Wild   type                    550             EP(3)3705 3           Wild   type           601 <br />   EP(3)3339            Wild   type                    200               EP(3)3339           Wild   type           160 <br />   EP(3)3303            Wild   type                    250               EP(3)3303           Wild   type           134 <br />   EP(3)3258            Wild   type                    214               EP(3)3258           Wild   type           191 <br />   EP(3)3916            Wild   type                    210               EP(3)3916           Wild   type           230 <br />   EP(3)893             Wild   type                    276               EP(3)893            Wild   type           212 <br />   EP(3)3377            Wild   type                    234               EP(3)3377           Wild   type           213 <br />   Sample sizes include total number progeny. Sample sizes EP lines phenotype, EP(3)3060, EP(3)3084, <br /> and EP(3)0473, include number eyes phenotype. Percentages parentheses indicate penetrance number <br /> of enhanced eyes total number eyes phenotype. <br />  <br />  <br /> Eye-specific expression EP(3)3060 caused mild                Courey 2002), translational regulation [EP(3)3145, Datx2] <br /> rough eye (Figure 1I) loss pigmentation                (Satterfield et al. 2002; Ciosk et al. 2004), synaptic <br /> (Figure 2G). Enhancement phenotype                    vesicle transport [EP(3)3060] (Pena-Rangel et al. 2002), <br /> observed EP(3)3060 jing coexpression,             cell cycle control [EP(3)3705] (BDGP). To test the <br /> included rougher eyes (Figure 1J) loss pigmen-             specificity interactions jing, enhancers <br /> tation (Figure 2H). A total 38% eyes                 crossed random EP lines, including EP(3) <br /> phenotype enhanced (n &#194;&#188; 301).                                3339, EP(3)3303, EP(3)3258, EP(3)3916, EP(3)893, <br />    Specificity EP interactions: The genetic synergy            EP(3)3377, driven eye GMR- <br /> between jing enhancers suggests                Gal4. Of enhancers, EP(3)3060 EP(3)0473 <br /> genes function similar processes neural                 interacted multiple EP lines (Table 2; Figure 2, I <br /> cells. However, group jing-interacting genes               J). In addition, EP(3)3145 interacted EP(3)0893, <br /> are involved wide range functions, including              resulting mildly rough eyes (Table 2). <br /> chromatin remodeling (DATR-X D1) (Rodriguez-                     Next, addressed specificity D1 DATR-X <br /> Alfageme et al. 1980; Ashley et al. 1989; Gibbons et al.          enhancement testing interactions reg- <br /> 1995b; Picketts et al. 1996), transcriptional regulation          ulatory genes EP collection. DATR-X [EP(3)635] <br /> [EP(3) 3354] (Ewel et al. 1990; England et al. 1992;              did interact EP elements upstream <br /> Cutler et al. 1998; Brody et al. 2002; Bhaskar                genes encoding DNA-binding proteins, including <br />                                              Lateral Positioning LG Axons                                          1403 <br />  <br />  <br />  <br />  <br />   Figure 2.&#226;&#8364;&#8221;Genetic interactions jing enhancers. Light level images adult eyes following genotypes shown: (A) <br /> GMR-GAL4 /1; EP(3)3084/1; (B) GMR-GAL4/1; EP(3)3084/1; UAS-jingE/1; (C) GMR-GAL4/1; EP(3)3084/1; (D) <br /> EP(3)3377/1; GMR-GAL4/1; EP(3)0635/1; EP(3)3084/1; (E) GMR-GAL4/1; EP(3)3084/1; EP(3)3354/1; (F) GMR- <br /> GAL4/1; EP(3)3084/1; EP(3)3145/1; (G) GMR-GAL4/1; EP(3)3060/1; (H) GMR-GAL4/1; EP(3)3060/1; UAS-jingE/1; <br /> (I) GMR-GAL4/1; EP(3)3060/1; EP(3)3377/1; (J) GMR-GAL4/1; EP(3)3060/1; EP(3)3303/1; (K) GMR-GAL4/1; <br /> EP(3)3060/1; EP(3)3145/1; (L) GMR-GAL4/1; EP(3)3354/1; (M) GMR-GAL4/1; EP(3)3354/1; EP(3)3303/1; (N) GMR- <br /> GAL4/1; EP(3)0635/1; EP(3)3354/1; (O) GMR-GAL4/1; EP(3)3354/1; EP(3)3145/1; (P) GMR-GAL4/1; EP(3)3145/1; <br /> (Q) GMR-GAL4/1; EP(3)3145/1; EP(3)0893/1; (R) GMR-GAL4/1; EP(3)3145/1; EP(3)0635/1; (S) GMR-GAL4/1; <br /> EP(3)0635/1. Bars, 50 mm. <br />  <br />  <br /> EP(3)1005 (n &#194;&#188; 411) EP(3)1096 (n &#194;&#188; 387). The P                EP(3)3205 (35% penetrance, n &#194;&#188; 301), lies <br /> element line EP(3)1005 lies upstream CG15514,             upstream CG7552 encoding protein WW <br /> which encodes protein containing DNA-binding                  domain interacts genetically cyclin E (Tseng <br /> BED finger chromatin-boundary-element-                   Hariharan 2002). In contrast, EP(3)0473 inter- <br /> binding proteins transposases (FlyBase) (Aravind              acted EP(3)1005 (48%, n &#194;&#188; 205), EP(3)3205 (12%, <br /> 2000). The P element line EP(3)1096 lies upstream           n &#194;&#188; 150), EP(3)3058 (83%, n &#194;&#188; 256), indicating <br /> a gene encoding CXXC zinc finger (FlyBase,                      general transcriptional regulator. <br /> CG11033). EP(3)0635 did interact EP(3)3058                 Interenhancer interactions: Last, determined if <br /> (n &#194;&#188; 200), lies upstream poly U binding            jing enhancers interacted eye. <br /> factor 68kD (pUf68) gene (Page-McCaw et al. 1999;                 Eye-specific coexpression EP(3)3084 EP(3)0635, <br /> Lasko 2000). However, EP(3)0635 did interact                 EP(3)3354, EP(3)3145 resulted severe <br />  1404                                      X. Sun, T. Morozova M. Sonnenfeld <br />  <br />                                                           TABLE 3 <br />                                             Screening jing interacting EP lines <br />  <br /> P[GMR-Gal4];                                                     P[GMR-Gal4]; <br /> EP(3)3145           Eye phenotype         % penetrance      N    EP(3)0635             Eye phenotype       % penetrance    N <br />                                                                                                                     a <br /> EP(3)0635       Glossy, YY pigment             17.9        217     EP(3)3084         YYY pigment              46.1        256 <br /> EP(3)3060       Bumpy, Y pigment               49.0a       300     EP(3)3354         Glossy, YYY pigment      17.2         93 <br /> EP(3)3354       Glossy, YY pigment             14.1         85     EP(3)3060         Glossy, YY pigment       88.9a        90 <br /> EP(3)3084       Rougher, YYY pigment           30.8a        65     EP(3)3705         Rough                    20          200 <br /> EP(3)3705       Rough                           8.5         94     EP(3)0473         Rough                    18.8        223 <br /> EP(3)0473       Rough                          15.0        167 <br />  <br /> P[GMR-Gal4];                                                     P[GMR-Gal4]; <br /> EP(3)3084           Eye phenotype         % penetrance      N    EP(3)3060             Eye phenotype       % penetrance    N <br />                                                                                                                 a <br /> EP(3)3060       No change                       0          250     EP(3)3354         Smaller, rougher         40           70 <br /> EP(3)3354       Glossy, YYY pigment            21.4a       234     EP(3)473          Rougher                   7.5a        90 <br /> EP(3)0473       Rougher                        24.9a       201     EP(3)3705         Smaller, rougher         18.7a        75 <br /> EP(3)3705       Rougher                        20.8a       120 <br />  <br /> P[GMR-Gal4]; <br /> EP(3)3354           Eye phenotype         % penetrance      N <br /> EP(3)3705               Small                  18.4        250 <br /> EP(3)0473               Rough                  15.6        225 <br />   Penetrance indicated percentage eyes phenotype progeny carrying GMR-Gal4 EP elements. <br /> N, sample sizes. <br />   a <br />     Penetrance EP lines phenotype EP(3)3060, EP(3)3084, EP(3)0473 percentage enhanced <br /> eyes total eyes phenotype. <br />  <br />  <br /> rough eye phenotypes including losses pigmentation             EP(3)3354 (Figure 2, R O). Collectively, these <br /> (Figure 2, D&#226;&#8364;&#8220;F; Table 3). Eye-specific expression              results identify group genes related func- <br /> EP(3)3060 resulted rougher eyes coexpressed               tions pertaining specifically jing dur- <br /> with EP(3)3145, EP(3)3354, EP(3)0635, EP(3)0473,                  ing eye development. <br /> and EP(3)3705 (Figure 2K; Table 3). However,                     Part ATR-X highly conserved: Previous studies <br /> widespread interactions EP(3)3060 suggest           showed ATR-X involved gene regulation by <br /> results considered nonspecific (Table 2; Figure            chromatin remodeling functions (Cardoso et al. 1998; <br /> 2, I J).                                                      Xue et al. 2003; Tang et al. 2004). Given GOF <br />    Eye-specific coexpression EP(3)3354                    interactions DATR-X jing eye, we <br /> EP(3)3145 EP(3)0635 resulted glossy eyes              proposed regulatory relationship <br /> severely reduced pigmentation representing synergistic            DATR-X Jing embryonic CNS and <br /> interactions (Table 3; Figure 2, N O). We                initiated studies DATR-X. Database searches <br /> designated EP(3)3354 jing interacting gene regula-             genomic sequence flanking EP(3)0635 P ele- <br /> tory 1 ( JIGR1), given potential role regulating           ment confirmed transposon strain we <br /> gene expression (Brody et al. 2002). The EP element            obtained inserted 59-untranslated region <br /> JIGR1 lies upstream transcript CG17383,              predicted gene CG4548 encoding Drosophila <br /> was identified differential head cDNA screen                 homolog human XNP/DATR-X (GadFly). <br /> (Brody et al. 2002). JIGR1 contains MADF domain                   DATR-X located cytological interval 96E1 the <br /> shown Adf-1 transcription activator bind DNA            chromosome (BDGP) encodes predicted <br /> specifically developmentally regulated Dro-            polypeptide 1311 amino acids 49.4% overall <br /> sophila gene promoters (Ewel et al. 1990; England                 similarity human ATR-X alignment using <br /> et al. 1992; Cutler et al. 1998; Bhaskar Courey               Clustal X algorithm (Figure 3A). ATR-X homologs <br /> 2002). JIGR1 interacted EP(3)3705 resulting          human, mouse, C. elegans, planarians, <br /> a reduced eye size (Table 3).                                     Drosophila (Figure 3, A C) (Villard et al. <br />    EP(3)3145 lies upstream Drosophila ataxin2 (Datx2)          1999). Most sequence conservation between <br /> and interacted jing enhancers                 human DATR-X confined helicase and <br /> screen randomly chosen EP lines.                SNF2 domains. The SNF2 domains C. elegans, <br /> We observed glossy eyes reduced pigmentation                  mouse, human ATR-X 54.9, 66.4, and <br /> after coexpression EP(3)3145 EP(3)635                 65.8% amino acid identity DATR-X, <br />                                               Lateral Positioning LG Axons                                           1405 <br />  <br />  <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;ATR-X homologs. (A) Comparison DATR-X protein domain architecture predicted using SMART PROSITE. <br /> Invertebrate ATR-X truncated comparison mouse human shares high identities C-terminal amino acids <br /> with proteins. Percentage identity SNF2 helicase C domains C. elegans, mouse, human in <br /> comparison D. melanogaster. (B) Comparison amino acid identity regions containing mutations the <br /> ATPase domain ATR-X human patients ATR-X syndrome. The mutations include Mu K1600R Ped23 <br /> D2035V highly conserved regions. (C) Phylogenetic analysis ATR-X orthologs. The distance sequences <br /> is sum horizontal branch length separating them. Sequences aligned using CLUSTALW (Tree View). The sequence <br /> residues column colored basis alignment consensus, calculated automatically (for color <br /> information, refer CLUSTAL W online help). Species designation follows protein acronym (Dm, Drosophila melanogaster; <br /> Ag, Anopheles gambiae; Mm, murine; Hs, human; Ce, Caenorhabditis elegans). Bar indicates number substitutions. <br />  <br />  <br /> respectively (Figure 3A). The ATPase domains C.                 teins (Figure 3A) (Cardoso et al. 2000). These zinc <br /> elegans, mouse, human ATR-X 58.9, 63,                 fingers required nuclear localization DNA <br /> 63% amino acid identity DATR-X, respec-               binding ATR-X (Cardoso et al. 2000). Therefore, it <br /> tively (Figure 3A). ATR-X nucleosome-stimulated                possible invertebrate ATR-X proteins require <br /> ATPase activity used modify chromatin struc-            site-specific DNA targeting carry ATPase <br /> ture common site patient mutations                activities. The C terminal ATR-X region rich in <br /> (Gibbons et al. 1995b; Xue et al. 2003; Tang et al.                poly glutamine repeats (Figure 3A) proposed be <br /> 2004). Mutations associated MR invariant               involved protein&#226;&#8364;&#8220;protein interactions not <br /> residues amino acids sites human             present D. melanogaster C. elegans ATR-X <br /> mutations, designated Mu K1600R Ped23 D2035V,                  (Picketts et al. 1996; data shown). Sequence dis- <br /> are conserved DATR-X (Figure 3B).                               tances ATR-X orthologs different species are <br />    There differences amino acid se-                schematized Figure 3C. <br /> quences vertebrate invertebrate ATR-X. Human                   DATR-X jing-lacZ expressed CNS glia and <br /> and mouse ATR-X zinc-finger DNA-binding do-                 neurons: To determine expression pattern DATR- <br /> main consisting multicysteine zinc-finger motifs          X development, hybridized whole-mount <br /> (C2-C2) present invertebrate ATR-X pro-             control embryos digoxigenin-labeled antisense <br />  1406                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> and sense DATR-X riboprobes. Ectopic activation                 sion copies transgene alone. <br /> EP(3)0635 detected prd-expressing stripes              An average 65% segments longitudinal <br /> our antisense DATR-X probe confirming specificity              connectives jing DATR-X coexpression (Figure <br /> of probe EP line (Figure 4A). DATR-X transcripts           5D, arrowhead; Figure 5G) comparison expression <br /> are present low level cellular blastoderm          copies UAS-jing (5% segments, n &#194;&#188; 80) and <br /> embryos (Figure 4B) gastrulation (Figure                EP(3)0635 (18% segments, n &#194;&#188; 55) (Figure 5G). <br /> 4C). In stage 15 embryos, DATR-X transcripts                Therefore, jing DATR-X coexpression syn- <br /> enriched neuroectoderm supraesophageal                  ergistic effects specifically embryonic CNS neurons. <br /> ganglion (brain) (Figure 4, D G). In mature                   Neuronal-specific functions jing DATR-X are <br /> ventral nerve cord, DATR-X mRNA expressed cells              required repulsion longitudinal axons glia <br /> along position longitudinal axon tracts                 CNS midline: We targeted DATR-X jing <br /> (Figure 4G, arrows) laterally located glia             mutations discern neuronal glial contribu- <br /> (Figure 4G, arrowhead). Similar CNS expression pat-                tions gene product axon scaffold formation. <br /> terns observed embryos hybridized                 DATR-X jing expression knocked using <br /> sense DATR-X riboprobes (Figure 4F). DATR-X tran-                  conditional RNAi (Hammond et al. 2001; Lee and <br /> scripts present brains            Carthew 2003). A total 697- 567-bp sequences <br /> instar wild-type larvae predominant optic           nonconserved regions DATR-X jing were <br /> lobe region (Figure 4E).                                           separately subcloned P[UAST] derivative plasmid <br />    To better understand pattern expression,              produce intron-spliced hairpin RNA corresponding <br /> costained embryos carrying jing- DATR-X-lacZ                   DATR-X jing genes, respectively. The UAS/ <br /> reporter genes anti-b-galactosidase glial-            Gal4 (Brand Perrimon 1993) used <br /> specific anti-Repo antibody. Analysis confocal mi-              allow hairpin RNA conditionally downregulate <br /> croscopy revealed jing-lacZ DATR-X-lacZ               DATR-X jing expression specific cell lineages, <br /> expressed glial neuronal lineages. Many             confirmed situ hybridization. Pene- <br /> the DATR-X-lacZ- jing-lacZ-expressing glia pre-            trance values CNS axon phenotypes associated with <br /> sent positions characteristic LG (Figure 4, I&#226;&#8364;&#8220;R).        neuronal glial knockdown jing DATR-X <br /> Other DATR-X-lacZ- jing-lacZ-expressing glia occupy            ranged 24 30% (see materials methods). <br /> more lateral (Figure 4, K L, arrowheads) ventral               Longitudinal connective formation analyzed <br /> regions (Figure 4, N, O, S).                                       homozygous jing mutant embryos with <br />    Pan-neural coexpression DATR-X jing syner-               neuronal-specific knockdown jing DATR-X <br /> gistically disrupts longitudinal axon formation: Since             stained antibody Fasciclin II (1D4). Neuronal <br /> coexpression jing DATR-X strongly affected adult            specificity directed ELAV-Gal4 driver. In these <br /> neuronal development wanted determine               mutant embryos, FasII-positive longitudinal axons ab- <br /> coexpression similar affects embryonic neurons.             errantly cross CNS midline stage 16 embryos <br /> jing DATR-X expressed postmitotic                  (Figure 6, B&#226;&#8364;&#8220;D, arrowheads; Figure 6M). In addition, <br /> neurons ELAV-GAL4. Expression jing               breaks longitudinal connectives sug- <br /> DATR-X resulted subtle defects CNS axon            gesting axon outgrowth defects (Figure 6, C D, <br /> scaffold observed BP102 staining. These defects            arrows). In embryos simultaneous neuronal knock- <br /> included thinner longitudinal connectives (Figure 5, B             jing DATR-X FasII-positive lateral <br /> and C, arrowheads) reduced spacing                 fascicles fuse single tract CNS midline <br /> anterior posterior commissures (Figure 5, B C,             (Figure 6E). These results establish autonomous <br /> arrows). Additive effects observed expression           neuronal requirement jing DATR-X function in <br /> of copies jing (Figure 5E) DATR-X                 outgrowth lateral positioning longitudinal <br /> (Figure 5F) commissural axons properly              axons genetic synergy process. <br /> separated (Figure 5, E F, arrows).                                Consistent results, pan-neural knockdown <br />    Pan-neural coexpression jing DATR-X resulted             jing DATR-X associated high levels Robo <br /> in severe defects axonal patterning expres-           CNS midline stage 12 (Figure 6, G <br />  <br />  <br />                                                                                                                                &lt; <br />   Figure 4.&#226;&#8364;&#8221;Drosophila ATR-X jing expressed CNS glia neurons. In situ hybridization using DATR-X digoxigenin- <br /> labeled riboprobe showing wild-type whole-mount embryos anterior left (A&#226;&#8364;&#8220;G). Confocal micrographs stage 15 <br /> whole-mount embryos (H&#226;&#8364;&#8220;S). Embryos stained polyclonal rabbit anti-b-galactosidase (FITC-conjugated secondary anti- <br /> body, green) anti-Repo (Texas Red-conjugated secondary antibody, red). (A) To validate probe specificity, embryo <br /> ectopically expressing DATR-X ectodermal stripes (arrow) paired gene hybridized DATR-X riboprobe. The em- <br /> bryo carries paired-GAL4 EP(3)0635. (B) DATR-X transcripts present cellular blastoderm embryos. (C) A 5.5- 6-hr-old <br /> embryo showing DATR-X transcripts supraesophageal ganglion (arrowhead) neuroectoderm (arrow) sagittal view. <br /> (D) A sagittal stage 15 embryo showing elevated DATR-X transcripts supraesophageal ganglion (arrowhead) ventral <br /> nerve cord (arrow). (E) Third instar larval CNS expressing DATR-X brain hemispheres (arrow) ventral ganglion <br />                                                 Lateral Positioning LG Axons                                              1407 <br />  <br />  <br />  <br />  <br /> (arrowhead). DATR-X expressed strongly ventral region ganglion (vg, arrowhead) optic lobe (ol, arrow). <br /> (F) In situ hybridization using DATR-X negative control sense probe. (G) DATR-X expression stage 15 ventral nerve cord <br /> (ventral view). Strong expression observed cells lining longitudinal connectives (arrows) lateral cells (arrowhead). <br /> (H&#226;&#8364;&#8220;R) Focal planes DATR-X jing-lacZ expression longitudinal glia (LG) (arrows). (N O) Sagittal views show <br /> that DATR-X-lacZ expressed ventral glia (arrowhead) dorsal longitudinal glia (arrow). DATR-X-lacZ expressed in <br /> neurons (double arrowhead). (P&#226;&#8364;&#8220;R) jing-lacZ strongly expressed rows longitudinal glial cells. (S) jing-lacZ also <br /> expressed bilateral glial lineage (arrow) laterally located lineage (arrowhead) ventral focal plane. Bars: (A&#226;&#8364;&#8220;F) <br /> 200 mm; (G) 30 mm; (H&#226;&#8364;&#8220;S) 50 mm. <br />  1408                                    X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> and H) stage 16 (Figure 6, J K). The phenotypes      expression genes required Robo read <br /> were variable (Figure 6K), severe case    Slit cue. Alternatively, jing DATR-X regulate <br /> Robo-positive axons present midline (Figure      longitudinal positioning Robo-independent man- <br /> 6K) severe cases midline Robo observed       ner (Kinrade et al., 2001). <br /> in fewer hemisegments (Figure 6L, arrowhead). We           Glial functions Jing DATR-X required for <br /> observed breaks Robo-positive connectives (Figure         ipsilateral positioning longitudinal glia, glial sur- <br /> 6L, arrow). The high Robo levels jing DATR-X          vival, patterning longitudinal axons: Genetic <br /> pan-neural mutants suggest genes           cell ablation experiments shown critical <br /> regulate robo expression. However, medial axon           role neuronal&#226;&#8364;&#8220;glial communication plays during <br /> displacement suggests mutations affect        pioneering longitudinal axon tracts (Booth et al. <br />                                                              2000; Hidalgo Booth 2000; Hidalgo et al. 2001; <br />                                                              Kinrade et al. 2001; Whitington et al. 2004). We <br />                                                              thought informative examine <br />                                                              jing DATR-X play role glial guidance of <br />                                                              longitudinal axons. Glial development examined <br />                                                              pan-glial expression jing DATR-X RNAi <br />                                                              transgenes control glial cells missing (gcm)- <br />                                                              Gal4 (Hosoya et al. 1995; Jones et al. 1995) using <br />                                                              anti-Repo marker assess glial fates (Campbell <br />                                                              et al. 1994; Xiong et al. 1994; Halter et al. 1995). <br />                                                                 Robo-positive Repo-positive LG already <br />                                                              misplaced medially stage 12 pan-glial <br />                                                              knockdown jing DATR-X (Figure 7, C&#226;&#8364;&#8220;F) in <br />                                                              jing hemizygotes (Figure 7B). Wild-type numbers of <br />                                                              glia mutants stage reveal jing and <br />                                                              DATR-X involved glial differentiation in <br />                                                              division specification cells (Figure 7O). <br />                                                              Later wild-type embryogenesis Robo present <br />                                                              axons (see Figure 6I) glial move- <br />                                                              ment restricted Robo-independent mechanisms, <br />                                                              including axon contact trophic support (Kinrade <br />                                                              et al., 2001). In mature cord, LG <br />                                                              longitudinal axons medially misplaced the <br />                                                              CNS midline jing hemizygotes glial knock- <br />                                                              jing DATR-X (Figure 7, H&#226;&#8364;&#8220;J, M, N). Despite <br />                                                              maintenance glial&#226;&#8364;&#8220;axonal contact, numbers <br />                                                              total glia steadily decreased embryogenesis in <br />                                                              jing DATR-X glial mutants (Figure 7O). However, <br />                                                              glial numbers unaffected neuronal-specific <br />  <br />  <br />                                                                 Figure 5.&#226;&#8364;&#8221;Pan-neural expression DATR-X jing in <br />                                                              embryonic CNS neurons synergistically affects axon pattern- <br />                                                              ing. Ventral views ventral nerve cord (VNC) stage <br />                                                              15 embryos stained BP102 (A&#226;&#8364;&#8220;F) shown ante- <br />                                                              rior up. (A) Control embryo heterozygous elav-Gal4 and <br />                                                              w1118. Thick longitudinal connectives connect hemiseg- <br />                                                              ment (arrowhead). The arrow denotes space between <br />                                                              anterior posterior commissural axons. (B C) <br />                                                              After overexpression jing (B) DATR-X (C) postmitotic <br />                                                              CNS neurons, longitudinal axons (arrowheads) thin <br />                                                              space commissural axons smaller (arrows). <br />                                                              (D) Coexpression copy EP(3)0635 UAS-jingE in <br />                                                              postmitotic neurons synergistic effects. Note absence <br />                                                              longitudinal connectives (arrowhead). (E F) Additive <br />                                                              effects overexpressing copies UAS-jingE (E) or <br />                                                              EP(3)0635 (F) Note commissural axons not <br />                                                              separated (arrows) longitudinal connectives thin <br />                                                              (arrowheads). (G) Quantitative analysis longitudinal con- <br />                                                              nective formation. Bars, 50 mm. <br />                                               Lateral Positioning LG Axons                                           1409 <br />  <br />  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;DATR-X jing functions required specifically CNS neurons longitudinal connective formation. DATR-X <br /> and jing RNAi transgene expression driven neurons ELAV-Gal4. (A&#226;&#8364;&#8220;E F&#226;&#8364;&#8220;L) Shown close-up views VNC of <br /> whole-mount stage16 embryos stained anti-Fasciclin II monoclonal antibody (1D4) (A&#226;&#8364;&#8220;E) anti-Robo (F&#226;&#8364;&#8220;L). Stage 12 <br /> embryos stained anti-Robo (F&#226;&#8364;&#8220;H). (A) In wild-type embryos, longitudinal fascicles clearly delineated. (B&#226;&#8364;&#8220;D) In <br /> jing (B C) DATR-X (D) mutants, longitudinal fascicles misroute midline (arrowheads) broken (arrows). <br /> (E) After coexpression jing DATR-X RNAi transgenes neurons, longitudinal fascicles fuse tract midline. <br />                                                                                                                      (continued) <br />  1410                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> jing DATR-X knockdown despite medial misplace-                  Vertebrate ATR-X C2C2 zinc-finger motif the <br /> ment mispositioning LG (Figure 7, K, L, O).                 amino terminus similar plant homeodomain <br /> Therefore, jing DATR-X glial-specific mutations                finger previously identified proteins involved in <br /> perturb autonomous survival function                chromatin-mediated transcriptional regulation (Aasland <br /> be compensated axon contact.                                et al. 1995; Gibbons et al. 1997). Interestingly, D. mela- <br />                                                                    nogaster C. elegans ATR-X proteins contain the <br />                                                                    zinc-finger domain, suggesting structures may <br />                       DISCUSSION <br />                                                                    acquired evolution necessity <br />    A conserved role ATR-X embryonic CNS?                vertebrate chromatin remodeling mechanisms. Pa- <br /> Of candidates screen, chose             tients identified mutations ATPase <br /> study DATR-X possible involvement Jing CNS             zinc-finger domains ATR-X, confirming these <br /> function disease relevance. Mutations human             essential functional regions protein (Villard <br /> ATR-X gene associated X-linked MR                 et al. 1996b,c; Gibbons Higgs 2000). <br /> phenotypes lead cognitive delay, facial dysmor-               Given absence zinc-finger domains in <br /> phism, microcephaly, skeletal genital abnormali-               DATR-X, postulate invertebrate DATR-X proteins <br /> ties, neonatal hypotonia (Gibbons et al. 1995a,b;              complexed proteins containing nuclear <br /> Villard et al. 1996a,b; Gibbons Higgs 2000). A                 targeting DNA-binding motif regulate gene <br /> total 87% MR genes fruit fly homolog              expression proper regulatory sites. This be <br /> 76% candidate functional ortholog revealing               role Jing embryonic expression <br /> remarkable conservation humans D. mela-                pattern mutant overexpression pheno- <br /> nogaster (Inlow Restifo 2004). Some orthologs               types similar DATR-X. Therefore, it <br /> human MR genes cellular phenotypes involving                  ATPase domain DATR-X been <br /> neurons, glia, neural precursor cells arise           conserved evolution regions <br /> defects proliferation, migration, process exten-            protein evolved suit specific needs <br /> sion arborization (Inlow Restifo 2004). For                 cell. In summary, different mechanisms ATR-X <br /> example, targeted <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>mutation</span> ATR-X early                   function different binding partners species <br /> forebrain mice leads cortical progenitor cell death          account divergence sequence with <br /> and reduced forebrain size (Be&#194;&#180;rube&#194;&#180; et al. 2005). In              respect amino terminal Q-rich repeats while <br /> addition, mutations genes controlling identity              main chromatin remodeling aspects ATR-X <br /> of forebrain neuronal precursors result holopro-            remain similar. <br /> sencephaly brain hemispheres                      Jing encodes nuclear protein putative DNA- <br /> separate (Wallis Muenke 2000). An increased                    binding transcriptional regulatory domains (Liu <br /> understanding molecular cellular bases              Montell 2001; Sedaghat et al. 2002). The C2H2 <br /> hereditary MR critical generation drug               zinc fingers Jing similar (50% identical) to <br /> treatments.                                                        mouse adipocyte enhancer binding protein <br />    ATR-X belongs SWI/SNF group chromatin                 2 (AEBP2) 25% homology the <br /> remodeling proteins, use energy provided              Kru&#194;&#168;ppel family transcription factors, including those <br /> ATP hydrolysis disrupt histone&#226;&#8364;&#8220;DNA associations             encoding gli ZIC2 (Liu Montell 2001). AEBP2 <br /> move nucleosomes different positions (Kingston                  function implicated chromatin remodeling events <br /> and Narlikar 1999; Whitehouse et al. 1999). This                   (Cardoso et al. 1998; Cao Zhang 2004) has <br /> chromatin modulation allows access activators           strong expression brain (He et al. 1999). <br /> or repressors DNA binding sites target              Genetic screening identifies related group jing- <br /> genes. The helicase C SNF2N domains ATR-X                   interacting genes: We utilized background <br /> have shown DNA translocase nucleo-                sensitive jing function conduct genetic screen <br /> some-remodeling activities (Xue et al. 2003; Tang et al.           eye. For GOF screen, hypothesized that <br /> 2004). Accordingly, mutations ATR-X                   misexpression jing eye combination with <br /> mapped helicase C SNF2N domains,                  genes involved jing transcriptional regula- <br /> show  60% homology DATR-X                   tion lead alterations gene expression <br /> been conserved C. elegans humans. This conser-             consequently disrupt ommatidial formation. The <br /> vation supports conserved role Drosophila ATR-X              genetic relationship DATR-X jing em- <br /> in chromatin remodeling (Tang et al. 2004; work).             bryonic neurons glia shows screen was <br />  <br />  <br /> (F) Wild-type Robo-positive glia present midline. (G H) Medial misplacement Robo-positive glia <br /> (arrowheads) occurs neuronal-specific knockdown jing DATR-X. (I) Lateral positioning Robo-positive axons wild- <br /> type embryos. (J&#226;&#8364;&#8220;L) Robo-positive axons cross midline (arrowheads) neuronal-specific knockdown DATR-X jing. <br /> Axonal breaks evident (arrow L). Arrow J denotes remaining connective. (M) Graph displaying average number of <br /> longitudinal axons cross CNS midline embryo. <br />  Figure 7.&#226;&#8364;&#8221;DATR-X jing required glial patterning survival. (A&#226;&#8364;&#8220;D G&#226;&#8364;&#8220;L) CNS glia identified in <br />                                                                                                         (continued) <br />  1412                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> successful identifying genes function adult               During stage 12, Robo present LG responds <br /> neuronal cells relevant jing function                 repulsive midline Slit molecules maintain lateral <br /> embryonic CNS. This consistent previous find-              positioning (Kinrade et al. 2001). The medial misplace- <br /> ings fly eye valid targeting genes        ment Robo- Repo-positive LG stage 12 after <br /> that function tissues (Raymond et al. 2004).              jing DATR-X glial-specific knockdown suggests that <br />    EP(3)3084 contains transposon proximity               breakdown Robo-dependent <br /> novel gene known FlyBase transcript identifier           repulsive mechanisms. However, fact Robo <br /> CG15507. Despite strong effects EP(3)3084 expres-               protein present jing DATR-X glial and <br /> sion eye specifically strongly enhanced          neuronal knockdown suggests robo expression may <br /> after coexpression jing, DAtx2, JIGR1. Further-           regulated Jing DATR-X. Alternatively, Jing <br /> more, gene specifically interacted            DATR-X regulate expression factor that <br /> but randomly chosen EP lines, suggesting                controls Robo reads Slit signal. In support, <br /> functional relationship genes. The EP             misrouting axons midline presence of <br /> elements lines located 59 untranslated         Robo occurs calmodulin Son sevenless mutants <br /> region downstream genes, suggesting                proteins required process Sli <br /> result overexpression (BDGP). Given regulatory              signal (Fritz VanBerkum 2000). It possible <br /> role MADF domains, possible JIGR1 regu-              jing DATR-X regulate expression factors <br /> lates gene expression Jing DATR-X (England                controlling glial neuronal positioning Robo- <br /> et al. 1992; Bhaskar Courey 2002). Alternatively,              independent fashion (Kinrade et al. 2001). <br /> JIGR1 regulate expression Jing/DATR-X                    jing DATR-X mutations clearly affect than <br /> target gene. Likewise, DAtx2 involved regu-              Robo-mediated LG positioning. First, glial survival is <br /> lating translation protein essential           affected robo mutant embryos, glia die <br /> component Jing/DATR-X/JIGR1 complex down-                despite continuous axonal contact jing DATR-X <br /> stream target genes (Satterfield et al. 2002;             glial-specific mutants. Therefore, loss CNS glia <br /> Ciosk et al. 2004). A role orthologs transla-           reflect breakdown intrinsic survival pathway <br /> tional regulators MR shown Dro-                mediated jing DATR-X. The expression jing <br /> sophila ortholog fragile-X MR 1 (Dfmr1). Dfmr1 regulates        DATR-X reporter genes glia consistent with <br /> the MAP1B homolog Futsch control synaptic                    role. Furthermore, jing DATR-X/ATR-X <br /> structure function embryonic Drosophila                 implicated <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>cell survival</span> processes the <br /> CNS (Zhang et al. 2001). Therefore, genetic screening              CNS midline tracheal cells cortical progen- <br /> and phenotypic analysis Drosophila power            itors, respectively (Sedaghat et al. 2002; Sonnenfeld <br /> decipher pathways cellular bases MR genes.              et al. 2004; Be&#194;&#180;rube&#194;&#180; et al. 2005). <br />    Neuronal glial functions jing DATR-X                    Second, robo mutants central pCC/MP2 <br /> dictate axon tract formation: In wild-type Drosophila              fascicle, outer longitudinal fascicles, is <br /> embryos, LG assume characteristic positions             affected. However, jing DATR-X glial neuronal <br /> cross midline adjacent VNC segments (Ito               mutants outer fascicles fused, broken, and <br /> et al. 1995). This multiple mechanisms dif-           seen crossing midline. These defects simi- <br /> ferent stages development, including response re-            lar ablation neurons glia after <br /> pulsive attractive molecules, cell&#226;&#8364;&#8220;cell contact, trophic       genetic loss glia gcm mutants (Hosoya et al. 1995; <br /> support, axon contact (Kinrade et al., 2001). A dis-           Jones et al. 1995; Vincent et al. 1996; Hidalgo Brand <br /> ruption processes perturb formation           1997). These observations suggest multiple biological <br /> of glial axonal scaffolds. The expression jing          processes require proper function genes and <br /> and DATR-X reporter genes LG correlates LG             consistent important upstream role jing <br /> phenotypes associated mutations genes.               DATR-X glial neuronal differentiation. <br />  <br />  <br />  <br />  <br /> whole-mount embryos stained anti-Repo. (E, F, M, N) Embryos stained anti-Robo. Shown close-up ventral views <br /> of nerve cord stage 12/3 (A&#226;&#8364;&#8220;F) stage 15 (G&#226;&#8364;&#8220;N) embryos. (A) In wild-type stage 12/3 embryos, longitudinal glia (LG) <br /> have migrated medially positions guide pioneer longitudinal axons. (B&#226;&#8364;&#8220;D) In contrast, LG inappropriately <br /> occupy positions CNS midline hemizygous jing embryos (B) glial-specific knockdown jing (C) <br /> and DATR-X (D) driven gcm-Gal4. (E F) Robo-positive glia misplaced medially expression jing (E) and <br /> DATR-X (F) RNAi transgenes CNS glia. (G) In wild-type embryos, LG occupy two-cell wide row lining longitudinal <br /> connectives (arrows). The LG observed dorsal plane view. (H&#226;&#8364;&#8220;J) In jing hemizygotes (H) jing (I) and <br /> DATR-X (J), glial mutants glia misplaced midline (arrows). (K L) In jing (K) DATR-X (L) neuronal mutants, <br /> glia cross midline scattered nerve cord. (M N) Glial knockdown jing (M) DATR-X (N) results in <br /> breaks Robo-positive longitudinal axons (arrows) inappropriate midline crossings (arrowheads). (N) Quantitative analysis <br /> of glia stages 12 15 expressing jing DATR-X RNAi transgenes glia neurons. Bars, 50 mm. <br />                                                          Lateral Positioning LG Axons                                                     1413 <br />  <br />    Evidence accumulating chromatin accessibility                       Bhat, K. M., 2005 Slit-Roundabout signaling neutralizes netrin-- <br />                                                                                    frazzled-mediated attractant cue specify lateral positioning <br /> plays key role transcriptional regulation cell-                        longitudinal axon pathways. Genetics 170: 149&#226;&#8364;&#8220;159. <br /> type-specific gene expression CNS (Hsieh et al.                         Booth, G. E., E. F. Kinrade A. Hidalgo, 2000 Glia maintain <br /> 2004). The conservation ATPase domains                                    follower neuron survival Drosophila CNS development. <br /> with similar phenotype DATR-X jing muta-                                Development 127: 237&#226;&#8364;&#8220;244. <br />                                                                                Brand, A. H., N. Perrimon, 1993 Targeted gene expression a <br /> tions expression patterns raises possi-                           means altering cell fates generating dominant pheno- <br /> bility Jing involved targeting chromatin                       types. Development 118: 401&#226;&#8364;&#8220;415. <br /> remodeling complex containing DATR-X transcrip-                             Brody, T., C. Stivers, J. Nagle W. F. Odenwald, 2002 Iden <br />                                                                                    tification novel Drosophila neural precursor genes using dif- <br /> tional target genes products required                                ferential embryonic head cDNA screen. Mech. Dev. 113: 41&#226;&#8364;&#8220;59. <br /> the response longitudinal growth cones glia                          Campbell, G., H. Goring, T. Lin, E. Spana, S. Anderson et al., <br /> guidance cues.                                                                     1994 RK2, glila-specific homeodomain protein required for <br />                                                                                    embryonic nerve cord condensation viability Drosophila. <br />    In summary, identified group genes                            Development 120: 2957&#226;&#8364;&#8220;2966. <br /> pertain jing function specifically genetically                          Cao, R., Y. Zhang, 2004 SUZ12 required histone <br /> interact adult neuronal cells. Our results                            methyltransferase activity silencing function EED- <br />                                                                                    EZH2 complex. Mol. Cell 15(1): 57&#226;&#8364;&#8220;67. <br /> specific neural glial developmental defects underlie                       Cardoso, C., S. Timsit, L. Villard, M. Khrestchatisky, M. Fontes <br /> the problems axon guidance associated muta-                                et al., 1998 Specific interaction ATR-X gene product <br /> tions DATR-X jing. More studies using targeted                              SET domain human EZH2 protein. Hum. Mol. <br />                                                                                    Genet. 7: 679&#226;&#8364;&#8220;684. <br /> mutations MR genes alleviate view brain                       Cardoso, C., Y. Lutz, C. Mignon, E. Compe, D. Depetris et al., <br /> phenotypes result generic effects height-                            2000 ATR-X mutations cause impaired nuclear location and <br /> ened sensitivity brain.                                                     altered DNA binding properties ATR-X protein. J. Med. <br />                                                                                    Genet. 37: 746&#226;&#8364;&#8220;751. <br />    We thank Marc Freeman gcm-gal4 S. Hayashi btl-gal4. We          Ciosk, R., M. DePalma J. R. Priess, 2004 ATX-2, C. elegans <br /> thank Doug Holmyard Bioimaging Centre Mount Sinai Hos-                   ortholog ataxin 2, functions translational regulation the <br /> pital performing SEM. We thank Helga Agah help immu-                  germline. Development 131: 4831&#226;&#8364;&#8220;4841. <br /> nohistochemistry. We thank Andrew Ridsdale OGHRI use               Cutler, G., K. M. Perry R. Tjian, 1998 Adf-1 nonmodular <br /> of confocal microscope training. This work supported              transcription factor contains TAF-binding Myb-like motif. <br /> grant Canadian Institutes Health Research M.S.                      Mol. Cell Biol. 18: 2252&#226;&#8364;&#8220;2261. <br />                                                                                Dearborn, R., Jr., S. Kunes, 2004 An axon scaffold induced by <br />                                                                                    retinal axons directs glia destinations Drosophila optic <br />                                                                                    lobe. Development 131: 2291&#226;&#8364;&#8220;2303. <br />                        LITERATURE CITED                                        England, B. P., A. Admon R. Tjian, 1992 Cloning Dro- <br /> Aasland, R., T. J. Gibson A. F. Stewart, 1995 The PHD finger:                  sophila transcription factor Adf-1 reveals homology Myb onco- <br />      implications chromatin-mediated transcriptional regulation.               proteins. Proc. Natl. Acad. Sci. USA 89: 683&#226;&#8364;&#8220;687. <br />      TIBS 20: 56&#226;&#8364;&#8220;59.                                                           Ewel, A., J. R. Jackson C. Benyajati, 1990 Alternative DNA- <br /> Abidi, F., N. J. Carpenter, L. Villard, M. Curtis, M. Fontes et al.,               protein interactions variable-length internucleosomal regions <br />      1999 The Carpenter-Waziri syndrome results mutation                    associated Drosophila Adh distal promoter expression. Nucleic <br />      DATR-X. Am. J. Med. Genet. 85: 249&#226;&#8364;&#8220;251.                                    Acids Res. 18: 1771&#226;&#8364;&#8220;1781. <br /> Adams, M. D., S. E. Celniker, R. A. Holt, C. A. Evans, J. D. Gocayne           Freeman, M. R., J. Delrow, J. Kim, E. Johnson C. Q. Doe, <br />      et al., 2000 The genome sequence Drosophila melanogaster.                  2003 Unwrapping glial biology: Gcm target genes regulating glial <br />      Science 5461: 2185&#226;&#8364;&#8220;2195.                                                      development, diversification function. Neuron 38: 567&#226;&#8364;&#8220;580. <br /> Ahmad, K., S. Henikoff, 2001 Modulation transcription                 Fritz, J. L., M. F. VanBerkum, 2000 Calmodulin son sev- <br />      factor counteracts heterochromatic gene silencing Drosophila.              enless dependent signaling pathways regulate midline crossing of <br />      Cell 104: 839&#226;&#8364;&#8220;847.                                                            axons Drosophila CNS. Development 27(9): 1991&#226;&#8364;&#8220;2000. <br /> Aravind, L., 2000 The BED finger, novel DNA-binding domain                   Gibbons, R. J., D. R. Higgs, 2000 Molecular-clinical spectrum <br />      chromatin- boundary-element-binding proteins transpo-                  ATR-X syndrome. Am. J. Med. Genet. 97: 204&#226;&#8364;&#8220;212. <br />      sases. Trends Biochem. Sci. 25: 421&#226;&#8364;&#8220;423.                                  Gibbons, R. J., L. Brueton, V. J. J. Buckle Burn, J. Clayton-Smith, <br /> Ashburner, M., 1989 Drosophila: A Laboratory Manual. Cold Spring                   B. C. C. Davison et al., 1995a The clinical hematological <br />      Harbor Laboratory Press, Cold Spring Harbor, NY.                              features X-linked a-thalassemia/mental retardation syn- <br /> Ashley, C. T., C. G. Pendleton, W. W. Jennings, A. Saxena                      drome (ATR-X). Am. J. Med. Genet. 55: 288&#226;&#8364;&#8220;299. <br />      C. V. Glover, 1989 Isolation sequencing cDNA clones en-            Gibbons, R. J., D. J. Picketts, L. Villard D. R. Higgs, <br />      coding Drosophila chromosomal protein D1. A repeating motif                1995b Mutations putative global transcriptional regulator <br />      proteins recognize DNA. J. Biol. Chem. 264: 8394&#226;&#8364;&#8220;8401.               cause X-linked mental retardation a-thalassemia (ATR-X <br /> Bate, C. M., E. B. Grunewald, 1981 Embryogenesis in-                     syndrome). Cell 80: 837&#226;&#8364;&#8220;845. <br />      sect nervous II: second class neuron precursor cells          Gibbons, R. J., S. Bachoo, D. J. Picketts, S. Aftimos, B. Asenbauer <br />      origin intersegmental connectives. J. Embryol.                 et al., 1997 Mutations transcriptional regulator ATRX estab- <br />      Exp. Morphol. 61: 317&#226;&#8364;&#8220;330.                                                    lish functional significance PHD-like domain. Nat. Genet. <br /> Barresi, M. J. F., L. D. Hutson, C-B. Chien R. O. Karlstrom,                   17: 146&#226;&#8364;&#8220;148. <br />      2005 Hedgehog regulated Slit expression determines commis-                Halter, D. A., J. Urban, C. Rickert, S. S. Ner, K. Ito et al., <br />      sure glial cell position zebrafish forebrain. Develop-             1995 The homeobox gene repo required differentia- <br />      ment 132: 3643&#226;&#8364;&#8220;3656.                                                          tion maintenance glia function embryonic nervous <br /> Battye, R., A. Stevens J. R. Jacobs, 1999 Axon repulsion                  Drosophila. Development 121: 317&#226;&#8364;&#8220;332. <br />      midline Drosophila CNS requires slit function. Develop-        Hammond, S. M., A. A. Caudy G. J. Hannon, 2001 Post-tran- <br />      ment 126: 2475&#226;&#8364;&#8220;2481.                                                          scriptional gene silencing double-stranded RNA. Nat. Rev. <br /> Be&#194;&#180;rube&#194;&#180;, N. G., M. Mangelsdorf, M. Jagla, J. Vanderluit, D. Garrick               Genet. 2(2): 110&#226;&#8364;&#8220;119. <br />      et al., 2005 The chromatin-remodeling protein ATRX critical        Hay, B. A., T. Wolff G.M. Rubin, 1994 Expression baculovirus <br />      neuronal survival corticogenesis. J. Clin. Invest. 115: 258&#226;&#8364;&#8220;267.       P35 prevents cell death Drosophila. Development 120: 2121&#226;&#8364;&#8220;2129. <br /> Bhaskar, V., A. J. Courey, 2002 The MADF-BESS domain fac-                  He, G.-P., S. Kim H.-S. Ro, 1999 Cloning characterization of <br />      tor Dip3 potentiates synergistic activation Dorsal Twist.              novel zinc finger transcriptional repressor. J. Biol. Chem. 274: <br />      Gene 299: 173&#226;&#8364;&#8220;184.                                                            14678&#226;&#8364;&#8220;14684. <br />  1414                                                X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> Hidalgo, A., A. H. Brand, 1997 Targeted neuronal ablation:              Page-McCaw, P. S., K. Amonlirdviman P. A. Sharp, 1999 PUF60: <br />      role pioneer neurons guidance fasciculation               novel U2AF65-related splicing activity. RNA 5: 1548&#226;&#8364;&#8220;1560. <br />      CNS Drosophila. Development 124: 3253&#226;&#8364;&#8220;3262.                     Pena-Rangel, M. T., I. Rodriguez J. R. Riesgo-Escovar, <br /> Hidalgo, A., G. Booth, 2000 Glia dictate trajectories pio-               2002 A misexpression study examining dorsal thorax formation <br />      neer neurons Drosophila embryonic CNS. Development                  Drosophila melanogaster. Genetics 160: 1035&#226;&#8364;&#8220;1050. <br />      127: 393&#226;&#8364;&#8220;402.                                                          Picketts, D. J., D. R. Higgs, S. Bachoo, D. J. Blake, O. W. J. <br /> Hidalgo, A., E. F Kinrade M. Georgiou, 2001 The Drosophila                  Quarrell et al., 1996 ATRX encodes novel member the <br />      neuregulin vein maintains glial survival axon guidance           SNF2 family proteins: mutations point common mecha- <br />      CNS. Dev. Cell 1: 679&#226;&#8364;&#8220;690.                                             nism underlying ATR-X syndrome. Hum. Mol. Genet. 5: <br /> Hosoya, T., K. Takizawa, K. Nitta Y. Hotta, 1995 Glial cells                1899&#226;&#8364;&#8220;1907. <br />      missing: binary switch neuronal glial determina-         Rajagopalan, S., E. Nicolas, V. Vivancos, J. Berger B. J. Dick- <br />      tion Drosophila. Cell 82: 1025&#226;&#8364;&#8220;1036.                                    son, 2000a Crossing midline. Roles regulation Robo <br /> Hsieh, J., K. Nakashima, T. Kuwabara, E. Mejia F. H. Gage,                  receptors. Neuron 28: 767&#226;&#8364;&#8220;777. <br />      2004 Histone deacetylase inhibition-mediated neuronal differ-          Rajagopalan, S., V. Vivancos, E. Nicolas B. J. Dickson, <br />      entiation multipotent adult neural progenitor cells. Proc. Natl.        2000b Selecting longitudinal pathway. Robo receptors specify <br />      Acad. Sci. USA 101: 16659&#226;&#8364;&#8220;16664.                                           lateral position axons Drosophila CNS. Cell 103: <br /> Huang, Y., S. J. Myers R. Dingledine, 1999 Transcriptional re-              1033&#226;&#8364;&#8220;1045. <br />      pression REST: recruitment Sin3A histone deacetylase         Rasband, K., M. Hardy C. B. Chien, 2003 Generating X: for- <br />      neuronal genes. Nat. Neurosci. 2(10): 867&#226;&#8364;&#8220;872.                          mation optic chiasm. Neuron 39: 885&#226;&#8364;&#8220;888. <br /> Hulo, N., C. J. A. Sigrist, V. Le Saux, P. S. Langendijk-Genevaux, L.       Raymond, K., E. Bergeret, A. Avet-Rochex, R. Griffin-Shea and <br />      Bordoli et al., 2004 Recent improvements PROSITE                    M.-O. Fauvarque, 2004 A screen modifiers RacGAP(84C) <br />      database. Nucleic Acids Res. 32: D134&#226;&#8364;&#8220;D137.                                gain-of-function Drosophila eye revealed LIM kinase <br /> Inlow, J. K., L. L. Restifo, 2004 Molecular comparative                 Cdi/TESK1 downstream effector Rac1 spermato- <br />      genetics mental retardation. Genetics 166: 835&#226;&#8364;&#8220;881.                     genesis. J. Cell Sci. 117: 2777&#226;&#8364;&#8220;2789. <br /> Ito, K., J. Urban G. M. Technau, 1995 Distribution, classifica-         Robinow, S., K. White, 1988 The locus ELAV Drosophila mel- <br />      tion development Drosophila glial cells late embry-          anogaster expressed neurons developmental stages. Dev. <br />      onic early larval ventral nerve cord. Roux&#226;&#8364;&#8482;s Arch. Dev. Biol.          Biol. 126: 294&#226;&#8364;&#8220;303. <br />      204: 284&#226;&#8364;&#8220;307.                                                          Rodriguez-Alfageme, G. T., G.T. Rudkin andL. Cohen, 1980 Isolation, <br /> Jacobs, J. R., C. S. Goodman, 1989 Embryonic development                 properties cellular distribution D1, chromosomal protein of <br />      axon pathways Drosophila CNS. II. Behavior pioneer               Drosophila. Chromosoma 78: 1&#226;&#8364;&#8220;31. <br />      growth cones. J. Neurosci. 9: 2402&#226;&#8364;&#8220;2411.                               R&#195;&#184;rth, P., 1996 A modular misexpression screen Drosophila <br /> Janody, F., J. Reischl N. Dostatni, 2000 Persistence                     detecting tissue- specific phenotypes. Proc. Natl. Acad. Sci. <br />      Hunchback terminal region Drosophila blastoderm              USA 93: 12418&#226;&#8364;&#8220;12422. <br />      embryo impairs anterior development. Dev. 127: 1573&#226;&#8364;&#8220;1582.              Rubin, G. M., 1988 Drosophila melanogaster experimental organ- <br /> Jones, B. W., R. D. Fetter, G. Tear C. S. Goodman, 1995 Glial               ism. Science 240: 1453&#226;&#8364;&#8220;1459. <br />      cells missing: genetic switch controls glial versus neuronal    Rubin, G. M., A. C. Spradling, 1983 Vectors P element-mediated <br />      fate. Cell 82: 1013&#226;&#8364;&#8220;1023.                                                  gene transfer Drosophila. Nucleic Acids Res. 11: 6341&#226;&#8364;&#8220;6351. <br /> Keleman, K., S. Rajagopalan, D. Cleppien, D. Teis, K. Palha et al.,         Rubin, G. M., L. Hong, P. Brokstein, M. Evans-Holm, E. Frise et al., <br />      2002 Comm sorts Robo control axon guidance Dro-                  2000 A Drosophila complementary DNA resource. Science 5461: <br />      sophila midline. Cell 110: 415&#226;&#8364;&#8220;427.                                        2222&#226;&#8364;&#8220;2224. <br /> Keleman, K., C. Ribeiro B. J. Dickson, 2005 Comm function  <br /> </body></html>