<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                     The cyclope Gene of Drosophila Encodes a Cytochrome c <br />                                  Oxidase Subunit VIc Homolog <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                Cytochrome c oxidase is the terminal enzyme of the mitochondrial electron transfer chain. In eukaryotes, <br />              the enzyme is composed of 3 mitochondrial DNA-encoded subunits and 7&#226;&#8364;&#8220;10 (in mammals) nuclear DNA- <br />              encoded subunits. This enzyme has been extensively studied in mammals and yeast but, in Drosophila, <br />              very little is known and no mutant has been described so far. Here we report the genetic and molecular <br />              characterization of mutations in cyclope (cype) and the cloning of the gene encoding a cytochrome c oxidase <br />              subunit VIc homolog. cype is an essential gene whose mutations are lethal and show pleiotropic phenotypes. <br />              The 77-amino acid peptide encoded by cype is 46% identical and 59% similar <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> the human subunit (75 <br />              amino acids). The transcripts are expressed maternally and throughout development in localized regions. <br />              They are found predominantly in the central nervous system of the embryo; in the central region of <br />              imaginal discs; in the germarium, follicular, and nurse cells of the ovary; and in testis. A search in the <br />              Genome Annotation Database of Drosophila revealed the absence of subunit VIIb and the presence of 9 <br />              putative nuclear cytochrome c oxidase subunits with high identity scores when compared to the 10 human <br />              subunits. <br />  <br />  <br />  <br /> C    YTOCHROME c oxidase (COX) is the terminal en- <br />       zyme in both the eukaryotic and prokaryotic respi- <br /> ratory chain complex that catalyzes the conversion of <br />                                                                             mitochondrial DNA and form the catalytic core. They <br />                                                                             are synthesized inside the mitochondrion and are ho- <br />                                                                             mologous to the three major subunits found in purple <br /> redox energy to ATP (reviewed in Capaldi 1990; Poy-                         bacteria as Paracoccus denitrificans. The other subunits <br /> ton and McEwen 1996; Lenka et al. 1998). The elec-                          are specific to eukaryotes and are nuclear-DNA en- <br /> tron-transport and oxidative phosphorylation (OXPHOS)                       coded. Their numbers vary according to the organism: <br /> chain (reviewed in Saraste 1999) contains five multi-                       10 subunits are found in mammals and 8 in yeast. They <br /> subunit complexes and is located on the inner mito-                         are synthesized on cytosolic ribosomes, mostly as precur- <br /> chondrial membrane. Different substrates are metabo-                        sors carrying N-terminal basic presequences for mito- <br /> lized, producing reducing equivalent via complex I                          chondrial targeting and import. Although the exact <br /> (NADH dehydrogenase) and II (succinate dehydroge-                           structural and functional role of these small subunits <br /> nase). Ubiquinone (coenzyme Q) shuttles electrons                           remains unclear, they are believed to modulate the over- <br /> <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> complexes I and II to complex III (ubiquinol-                          all activity of the complex. Some of the nuclear-encoded <br /> cytochrome c oxidoreductase), which in turn reduces                         subunits are expressed as tissue- and developmental- <br /> cytochrome c. Cytochrome c transfers electrons to com-                      specific isoforms. In different vertebrates, subunits IV, <br /> plex IV (cytochrome c oxidase) driving ATP generation                       Va, Vb, VIb, VIIb, VIIc, and the mammalian liver iso- <br /> in complex V (ATPase). In human the OXPHOS system                           forms (L), VIa, VIIa, and VIII are detected in all tissues <br /> contains 83 polypeptides of which 70 are nuclear-DNA                        and are classified as ubiquitous, although the mRNA <br /> encoded, whereas the other 13 are encoded by mtDNA                          levels for individual subunits vary in different tissues. <br /> (Leonard and Schapira 2000a; Schon 2000).                                   The mammalian tissue-specific H isoforms, VIa, VIIa, <br />    Cytochrome c oxidase is a complex heme-copper con-                       and VIII, are specific to heart and skeletal muscle. <br /> taining metalloprotein embedded in the mitochondrial <br />                                                                                In yeast (Saccharomyces cerevisiae), <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> for sub- <br /> inner membrane as either a monomer or a dimer. The <br />                                                                             units I, II, III, IV, Va, Vb, VIc, and VIIa (nomenclature <br /> enzyme catalyzes the reaction: 4 ferrocytochrome c &#226;&#171;&#185; <br />                                                                             of Kadenbach and Merle 1981) do not grow on nonfer- <br /> 8 H&#226;&#171;&#185; &#226;&#171;&#185; O2 &#226;&#171;&#189; 4 ferricytochrome c &#226;&#171;&#185; 4 H&#226;&#171;&#185; &#226;&#171;&#185; 2 H2O. In <br />                                                                             mentable carbon sources and either fail to assemble <br /> mammals, the enzyme is composed of 13 subunits. The <br />                                                                             cytochrome c oxidase or yield a cytochrome c oxidase <br /> 3 major subunits (I, II, and III) are encoded by the <br />                                                                             with diminished activity. In contrast, a strain deleted for <br />                                                                             the gene encoding subunit VIIc has 80% wild-type levels <br />                                                                             of cytochrome c oxidase activity and grows on nonfer- <br />   Corresponding author: Re&#194;&#180;gine Terracol, Institut Jacques Monod, 2 <br /> Place Jussieu, Tour 43, 75251 Paris Cedex 05, France.                       mentable carbon sources. Subunit VIIc is required, how- <br /> E-mail: terracol@ijm.jussieu.fr                                             ever, for the production of stable cytochrome c oxidase <br />  <br /> Genetics 158: 1629&#226;&#8364;&#8220;1643 (August 2001) <br />  1630                                            S. Szuplewski and R. Terracol <br />  <br /> dimers and affects the low-affinity electron-transfer site       were provided by the Bloomington Stock Center. cact99 is de- <br /> from cytochrome c.                                               scribed in Roth et al. (1991), ea161.13 in Jin and Anderson <br />                                                                  (1990), and dpphr4 and dpphr27 in Spencer et al. (1982). Balancer <br />    A number of human diseases with various clinical              chromosomes and other stocks are described in Lindsley and <br /> symptoms have been attributed to defects in the cyto-            Zimm (1992) and in FlyBase (1997). To perform a secondary <br /> chrome c oxidase complex (reviewed in Taanman 1997;              P-element mutagenesis, the cype1 PZ[ry] element was mobilized <br /> Poyton 1998; Leonard and Schapira 2000a,b; Schon                 by providing the &#226;&#338;&#172;2-3 external source of transposase (Robert- <br /> 2000). Partial cytochrome c oxidase deficiency in the            son et al. 1988). cype1/CyO; ry506/ry506 females were crossed to <br />                                                                  w/Y; Sp/CyO; ry506 Dr P[ry&#226;&#171;&#185; &#226;&#338;&#172;2-3]/TM6 Ubx males to generate <br /> adolescent or the adult is often associated with specific        cype1/CyO; ry506 Dr P[ry&#226;&#171;&#185; &#226;&#338;&#172;2-3]/ry506 males. They were crossed <br /> point mutations in tRNA as well as rearrangements in             individually to cype1/CyO females to select ry&#226;&#171;&#186;/CyO individuals <br /> subpopulations of mtDNA. For instance, defective mus-            that were crossed to cype1/CyO to test for lethality. Embryonic <br /> cle fiber in Kearns-Sayre syndrome, as well as ocular            cuticles were prepared as described in Wieschaus and Nu&#194;&#168;ss- <br /> myopathy and progressive external ophthalmoplegia,               lein-Volhard (1986). Wings were dissected, collected in 70% <br />                                                                  ethanol, and mounted in Euparal (Labosi). <br /> are attributed to single large or multiple deletions or             Clonal analysis: Mitotic clones were generated using the <br /> duplications in mtDNA. Fatal infantile and benign my-            FLP/FRT technique (Golic and Lindquist 1989; Xu and <br /> opathies as well as cardiomyopathy are associated with           Rubin 1993). cype1 was recombined onto a P[ry&#226;&#171;&#185; hs-neo FRT]40A <br /> deficiency of subunit VIIa and/or VIIb with, moreover,           second chromosome. To generate germline clones cype1 P[ry&#226;&#171;&#185; <br /> in the case of benign myopathy, a lack of subunit II. In         hs-neo FRT]40A/CyO females were crossed to w P[ry&#226;&#171;&#185;hs FLP]1/ <br />                                                                  Y; P[w&#226;&#171;&#185; ovoD1] P[ry&#226;&#171;&#185; hs-neo FRT]40A/CyO males and clones <br /> Leigh syndrome a severe encephalopathy is attributed             were analyzed in w P[ry&#226;&#171;&#185; hs FLP]1/&#226;&#171;&#185;; P[w&#226;&#171;&#185; ovoD1] P[ry&#226;&#171;&#185; hs-neo <br /> to a decrease in COX activity due to incomplete assem-           FRT]40A/cype1 P[hs-neo-FRT]40A female progeny (Chou et al. <br /> bly of the enzyme complex. The sequence of nuclear-              1993). To generate somatic clones in adult y w/y w; cype1 P[ry&#226;&#171;&#185; <br /> encoded subunits reveals no defect, but mutations in the         hs-neo-FRT]40A/CyO females were crossed to y P[ry&#226;&#171;&#185; hs FLP]1/ <br /> nuclear gene Surf 1, encoding a mitochondrial protein            Y; P[y&#226;&#171;&#185;] P[ry&#226;&#171;&#185; hs-neo FRT]40A/Bc males and yellow clones were <br />                                                                  analyzed in y P[ry&#226;&#171;&#185; hs FLP]1/y w; P[y&#226;&#171;&#185;] P[ry&#226;&#171;&#185; hs-neo FRT]40A/ <br /> involved in COX assembly and maintenance, have been              cype1 P[ry&#226;&#171;&#185; hs-neo FRT]40A female progeny. Clones were heat- <br /> found in many patients.                                          shock induced at third larval instar by 1-hr exposure to 38&#226;&#172;&#353;. <br />    In Drosophila, several genes encoding mitochondrial           To generate mitotic clones in the eye we used the EGUF/hid <br /> proteins have been identified. Two of them encode es-            method (Stowers and Schwarz 1999). y w/y w; P[ry&#226;&#171;&#185; hs-neo <br /> sential components of the OXPHOS chain, the ATPase               FRT]40A GMR-hid l(2)CL-L1/CyO; ey-GAL4 UAS-FLP/ey-GAL4 <br />                                                                  UAS-FLP females were crossed to y w/Y; cype1 P[ry&#226;&#171;&#185; hs-neo <br /> &#226;&#65533;&#163;-subunit of complex V (Jacobs et al. 1998) and the              FRT]40A/CyO males at 29&#226;&#172;&#353;. To observe eye phenotypes in the <br /> ubiquinol-cytochrome c oxidoreductase subunit 9 of               progeny, flies were fixed in 3% glutaraldehyde (2 hr at RT <br /> complex III (Frolov et al. 2000). The sequences of the           and then 24 hr at 4&#226;&#172;&#353;), dehydrated in ethanol series and then <br /> mitochondrial DNA including COX subunits (I, II, III)            amyl acetate series. Electron microscopy was performed on a <br /> genes (de Bruijn 1983) and of the nuclear COX subunit            JEOL 6100 scanning electron microscope. <br />                                                                     DNA and RNA analysis: Standard molecular biology experi- <br /> Va (Caggese et al. 1999) gene of complex IV have also <br />                                                                  ments were performed as described in Sambrook et al. (1989). <br /> been identified, but no functional information and no            Genomic DNA was extracted from adult flies. About 50 individ- <br /> COX mutants have been described so far. Identification           uals were ground in 200 &#226;&#65533;&#174;l of 0.1 m Tris pH 9.2, 0.1 m EDTA pH <br /> of genes encoding nuclear COX subunits in Drosophila             8, 1% SDS and incubated 30 min at 65&#226;&#172;&#353;. After centrifugation, <br /> and analysis of corresponding mutations will provide             proteins were precipitated 30 min on ice after addition of K <br /> tools to study the modulation of their role during devel-        acetate to 1 m final and DNA was recovered after isopropanol <br />                                                                  precipitation. Southern blots were hybridized with [32P]dCTP <br /> opment and will contribute to our understanding of               random primed DNA probes in 0.25 m Na2HPO4 pH 7.2, 7% <br /> the human diseases that result from alteration of COX            SDS and washed in 20 mm Na2HPO4 pH 7.2, 1% SDS at 65&#226;&#172;&#353;. <br /> activity. We report here the identification of a previously      PCR amplifications were performed with 0.25&#226;&#8364;&#8220;0.5 &#226;&#65533;&#174;g of geno- <br /> undescribed Drosophila lethal locus, cyclope, whose mu-          mic DNA. 5&#226;&#172;&#732;P primer (reverse) was 5&#226;&#172;&#732;-CCTCTCAACAG <br /> tations lead to pleiotropic phenotypes and the genera-           CAAACGTGTACTG-3&#226;&#172;&#732; and 3&#226;&#172;&#732;P primer (forward) was 5&#226;&#172;&#732;-TCT <br />                                                                  CTTGCCGACGGGACC-3&#226;&#172;&#732;. Genomic primer positions were <br /> tion and molecular characterization of new alleles. We           established from cDNA GH04604 start (position 1): primer 1 <br /> show that germline cype clones are lethal and that so-           (forward) 5&#226;&#172;&#732;-ATCTGCCATACTTGAATC-3&#226;&#172;&#732; (&#226;&#171;&#186;2080/&#226;&#171;&#186;2062), <br /> matic clones lead to inhibition of cell growth. We have          primer 2 (reverse) 5&#226;&#172;&#732;-GATATGGCCAATGGTATG-3&#226;&#172;&#732; (141/ <br /> cloned the gene and show that cype encodes a cyto-               122), primer 3 (reverse) 5&#226;&#172;&#732;-TAGACTACTTGTGTTTGGG-3&#226;&#172;&#732; <br /> chrome c oxidase subunit VIc homolog. A search in the            (597/578), primer 4 (reverse) 5&#226;&#172;&#732;-TCAATACAGAAGCGCA <br />                                                                  ACTC-3&#226;&#172;&#732; (640/621). PCR products were cloned into pGEM <br /> Genome Annotation Database of Drosophila identified              plasmid (Promega, Madison, WI) and sequenced using primer <br /> nine putative nuclear COX subunits.                              2, SP6 or T7 primers, or directly sequenced using primer 2. <br />                                                                  RNAs were extracted with pH 5 hot phenol at 60&#226;&#172;&#353; (Sato et <br />                                                                  al. 1982), enriched for poly(A)&#226;&#171;&#185; RNAs by oligo(dT) cellulose <br />               MATERIALS AND METHODS                              chromatography (Pharmacia, Uppsala, Sweden), and run in <br />                                                                  1% formaldehyde agarose gels. Northern blots were hybrid- <br />    Fly stocks and mutagenesis: Flies were raised on standard     ized with [32P]dCTP random primed DNA probes in 50% <br /> media and all crosses were performed at 25&#226;&#172;&#353; unless otherwise     formamide, 5&#226;&#171;&#187; SSPE, 5&#226;&#171;&#187; Denhardt&#226;&#8364;&#8482;s, 0.1% SDS at 42&#226;&#172;&#353; and <br /> specified. l(2)03771 (Karpen and Spradling 1992; Spradling       washed at high stringency. <br /> et al. 1999), renamed here cype1, FLP/FRT, and &#226;&#338;&#172;2-3 stocks          cDNA library screening and sequencing: &#226;&#65533;&#173;57 phage genomic <br />                                                  cyclope Encodes a COX VIc Homolog                                           1631 <br />  <br /> DNA (Canton-S), isolated from a previous walk (George and           se cross die at the embryo stage (25%). However, cype/ <br /> Terracol 1997), was subcloned in pBluescript II KS&#226;&#171;&#185; vector.         Df(2L)tkvSz2 individuals die at first larval stage. This is <br /> Oregon-R cDNAs were isolated from 1.5- to 5-hr embryonic <br /> &#226;&#65533;&#173;gt10 library (a gift from M. Goldschmidt-Clermont). Canton-        true with the cype1 allele and the six new revertant alleles <br /> S cDNAs were isolated from 0- to 24-hr embryonic and late           (cype1R). This was observed in both directions of the <br /> third larval instar &#226;&#65533;&#173;Zap II libraries (gifts from C. S. Thummel).   cype/&#226;&#171;&#185; &#226;&#171;&#187; Df(2L)tkvSz2/&#226;&#171;&#185; cross. However, from this cross, <br /> &#226;&#65533;&#173;gt10 cDNA were sequenced directly using &#226;&#65533;&#173;gt10 primers.             we found 6% dead embryos when the females were <br /> Forward primer was 5&#226;&#172;&#732;-GCAAGTTCAGCCTGGTTAAG-3&#226;&#172;&#732; and                   cype1/&#226;&#171;&#185; and 2% when the females were Df(2L) <br /> reverse primer was 5&#226;&#172;&#732;-AGGTGGCTTATGAGTATTTC-3&#226;&#172;&#732;. &#226;&#65533;&#173;Zap <br /> II cDNAs were excised (Stratagene rapid excision kit, La Jolla,     tkvSz2/&#226;&#171;&#185;, a difference that is at the limit of significance. <br /> CA) and sequenced using T3 and T7 primers. The sense and            This deficiency deletes the entire cype gene (George <br /> antisense genomic strands were sequenced by the dideoxy-            and Terracol 1997), and this result was also observed <br /> chain termination method (Sanger et al. 1977) using the             with other deficiencies covering cype. No significant em- <br /> United States Biochemicals sequencing kit (Pharmacia).              bryonic lethality was observed either way with the cross <br /> cDNA and PCR product sequencing was performed on an ABI <br /> Prism 377 DNA sequencer (Applied Biosystems, Foster City,           cype1/&#226;&#171;&#185; &#226;&#171;&#187; &#226;&#171;&#185;/&#226;&#171;&#185; (2% when females were cype1/&#226;&#171;&#185; and <br /> CA). DNA sequences were compiled using the Genetics Com-            1% when females were &#226;&#171;&#185;/&#226;&#171;&#185;), showing no dominant <br /> puter Group software (GCG; Devereux et al. 1984) and com-           maternal effect in this context. With the heterozygous <br /> pared to the Drosophila database using the Berkeley Drosophila      combinations, cype1/cype1R2.5 and cype1/cype1R15.1, individu- <br /> Genome Project (BDGP; http://www.fruitfly.org/blast/) Blast         als die either in embryo or at the beginning of first larval <br /> Searches program (Altschul et al. 1990). Protein alignments <br /> were performed on the Blast servers of the National Center for      instar. The slight difference in lethal stage, observed <br /> Biotechnology Information (NCBI; http://ncbi.nlm.nih.gov/           between homozygous, heterozygous, and hemizygous <br /> blast/) and of the BDGP.                                            cype, is not due to another lethal <span id='am-244' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> that had <br />    Whole mount in situ hybridizations and lacZ staining: Whole      occurred on the second chromosome, since the stocks <br /> mount in situ hybridizations in embryos were performed ac-          were cleaned up, and, furthermore, lethality is lost after <br /> cording to Tautz and Pfeifle (1989) and Me&#194;&#180;vel-Ninio et al. <br /> (1991). Purified restriction fragments or pBScDNA19 were            excision of the P element. The lethality of the different <br /> used to synthesize digoxigenin-labeled DNA or antisense RNA         alleles, inter se and in hemizygotes, was rescued with a <br /> probes (Boehringer Mannheim labeling kit, Mannheim, Ger-            transgene including the cype1 insertion site. <br /> many). lacZ staining was as described in Mlodzik and Hiromi            Most cype1/cype1 embryos do not present any strong <br /> (1992).                                                             cuticular alterations except for head defects (Figure <br />    Transformation experiments: The 3.8-kb BglII-XbaI geno- <br /> mic fragment including cype cDNAs was cloned at the BglII-          1B). However 10% of the embryos present Filzko&#194;&#168;rper <br /> XbaI sites of Pw6 mini white P-element vector (Klemenz et al.       defects and lateral extension of the ventral denticle belt <br /> 1987). The construct was co-injected with the pUCh&#226;&#338;&#172;2-3              (Figure 1C). The phenotypes are identical with the six <br /> helper plasmid in pole cell region of w1118 preblastoderm em-       new lethal lines. <br /> bryos (Spradling and Rubin 1982). A transgene on the X                 Germline cype1 clones were performed using the FLP/ <br /> chromosome was used for the rescue experiments. <br />                                                                     ovoD1FRT system. No embryos were recovered, indicating <br />                                                                     that the mutation is either cellular lethal or required <br />                                                                     for ovarian development. Dissection of clone-bearing <br />                            RESULTS <br />                                                                     female ovaries showed that they were no different from <br />    Secondary cyclope mutagenesis: The original lethal               ovoD1 ovaries. Therefore cype1 clones do not allow correct <br /> mutation, l(2)03771, was recovered after a PZ (Mlodzik              ovarian development and, unless the <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>phenotype</span> is simi- <br /> and Hiromi 1992) element insertion mutagenesis (Kar-                lar to ovoD1, lead to cell death. <br /> pen and Spradling 1992) and is renamed here cype1.                     Adult phenotypes and somatic clones: Mutant pheno- <br /> The P element was localized as a single insertion site in           types were also observed, at low penetrance, in the cype1/ <br /> the 25D4-6 interval (Spradling et al. 1999). Prior to               CyO flies. In the most frequent phenotype (18%), flies <br /> analysis, the cype1 chromosome was cleaned up by cross-             have atrophic tergites. Defects are also observed (6%) <br /> ing several times to the ry506/ry506 strain, to eliminate the       in legs (Figure 2A) and in wings (Figure 2B). Less fre- <br /> other potential lethal mutations that occurred on the               quently (1/250) flies with half (Figure 2C) or split tho- <br /> second chromosome. To obtain null or stronger alleles               rax or rough or deformed eyes (Figure 2D) are found. <br /> we performed a secondary P-induced mutagenesis by                   Lethality is observed in 35% of the cype1/CyO pupae, <br /> excision of the PZ[ry] transposon of cype1. Starting with           CyO/CyO individuals dying during the early L1 larval <br /> 20 cype1/CyO; ry506 Dr P[ry&#226;&#171;&#185; &#226;&#338;&#172;2-3]/ry506 individual males           stage. Flies with abnormally close eyes (Figure 2E) or, <br /> (materials and methods), we recovered 20 viable (9                  exceptionally, fused eyes (Figure 2F) are observed at <br /> independent) and 12 lethal (6 independent) ry lines.                low frequency. This phenotype is sublethal and these <br /> The excision of the P element leads to viability indicat-           flies generally die in pupae. This phenotype is not due <br /> ing that the initial lethality was due to the P insertion.          to another dominant or recessive mutation that had <br /> Six independent ry lethal lines were retained for further           occurred in the stock, since, when fused eye flies were <br /> analysis.                                                           crossed inter se, the frequency of this phenotype was not <br />    cype embryonic phenotypes and germline clones: With              increased in either the progeny or the second genera- <br /> all the alleles, cype/cype individuals from a cype/&#226;&#171;&#185; inter          tion. This is also true for the other phenotypes. Dead <br />  1632                                            S. Szuplewski and R. Terracol <br />  <br />  <br />  <br />  <br />                                                                                        Figure 1.&#226;&#8364;&#8221;cyclope embryonic pheno- <br />                                                                                      types. The embryos are oriented with an- <br />                                                                                      terior to the left and dorsal up. (A) Wild- <br />                                                                                      type larvae. (B) cype1/cype1 with head de- <br />                                                                                      fects and ventral denticle compressed. <br />                                                                                      (C) cype1/cype1 embryo with dorsoventral <br />                                                                                      polarity defects (moderately ventralized <br />                                                                                      V3). (D) V3 embryo from cype1/&#226;&#171;&#185;; <br />                                                                                      ea161.13/&#226;&#171;&#185; females crossed to wild-type <br />                                                                                      males. (E) V3 embryo from cype1&#226;&#171;&#185;/&#226;&#171;&#185; <br />                                                                                      cact99 females crossed to wild-type males. <br />                                                                                      (F) Moderately ventralized embryo (V3) <br />                                                                                      from cype1/CyO females crossed to <br />                                                                                      dpphr27/CyO males. Classification of phe- <br />                                                                                      notypes is described in Table 1 ac- <br />                                                                                      cording to the nomenclature of Roth et <br />                                                                                      al. (1991). <br />  <br />  <br />  <br />  <br /> fused eye pupae were also observed in the progeny of the         maternal genes easter (ea) and cactus (cact) of the dorsal <br /> new alleles crossed to dpphr27. Most of these phenotypes         pathway, as well as those of the zygotic gene decapen- <br /> (except fused eyes) were also observed with deficiencies         taplegic (dpp) of the dpp pathway (reviewed in Morisato <br /> of the region. Therefore these phenotypes are probably           and Anderson 1995). <br /> due to the haploinsufficiency of cype. This was confirmed           cype was associated with ea maternally (Table 1). Of <br /> by the fact that both cype1R2.5/CyO and cype1R15.1/CyO new       the embryos from cype1/&#226;&#171;&#185;; ea161.13/&#226;&#171;&#185; females crossed to <br /> lines obtained after the secondary mutagenesis have a            wild-type males, 99% die with ventralized phenotypes <br /> 2-day emergence delay at 25&#226;&#172;&#353; when compared to cype1/             varying from V5 (very weakly ventralized) to V3 (moder- <br /> CyO flies. This result also indicates that cype1 is probably     ately ventralized) with 80% V4 (weakly ventralized; Fig- <br /> not a <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>null allele</span> and that these new lines, unless they          ure 1D). The detail of the nomenclature used (Roth <br /> alter other genes, are stronger alleles than cype1. How-         et al. 1991) is presented in Table 1. With the control <br /> ever, the study of the embryonic phenotypes and the              ea161.13/TM3, 62% of the embryos die and are 100% <br /> stage of the lethal phase of the different alleles did not       V5. The same type of cross was performed with cact. cype <br /> show a stronger phenotype in the lethal revertant lines.         &#226;&#171;&#185;/&#226;&#171;&#185; cact99 females crossed to wild-type males produce <br />    Somatic clones were induced with the FLP/FRT tech-            6% dead embryos of which 80% are V4 and 20% V3 <br /> nique and were recovered and observed in adults. As              (Figure 1E). In the cact99/CyO control cross, 6% of the <br /> shown in Figure 2G, y cype1 clones present atrophic or           embryos also die but are 100% V4. These results show <br /> missing bristles. This phenotype is observable on the            that altering cype maternally enhances the maternal ven- <br /> whole adult cuticle. At the wing margin, bristles are            tralizing effect of ea and cact mutations. <br /> shorter and thinner due probably to the reduction in                The effect of cype was tested on two different dpp <br /> cell size (Figure 2H). cype1 clones were induced in the          alleles. cype1/CyO females crossed to dpphr27/CyO (strong <br /> eye with the EGUF/hid method (Stowers and Schwarz                allele) males yield only 4% of the expected &#226;&#171;&#185; cype1/dpp &#226;&#171;&#185; <br /> 1999). This technique makes it possible to generate eyes         progeny, while 38% are recovered with dpphr4 (weaker <br /> that are composed exclusively of clones, because all             allele). In the cross performed the other way the ex- <br /> other genotypes are eliminated. This is done by using            pected progeny is recovered with dpphr27 and 71% with <br /> the dominant photoreceptor cell lethal GMR-hid and a             dpphr4 (Table 2). The lethality is embryonic and the em- <br /> recessive cell death mutation. cype1/cype1 eyes have a           bryos are ventralized with phenotypes varying from V4 <br /> rough aspect (Figure 3C), disorganized ommatidia with            to V3 (Figure 1F). The haploinsufficient effect of dpphr27 <br /> an abnormal morphology, and duplicated or missing                gives only V5 embryos. These results show a maternal <br /> bristles (Figure 3D).                                            enhancement of dpp mutant ventralizing embryonic <br />    cype enhances mutant phenotypes of dorsoventral pat-          phenotypes by the alteration of cype. The same results <br /> terning genes: The cype1 embryonic defects are usually           were observed with the other cype alleles obtained after <br /> observed in weakly ventralized embryos. To determine             the secondary mutagenesis. <br /> whether these phenotypes were the result of an alter-               cype deficiency mapping: The PZ element (P-lacZ- <br /> ation in dorsoventral polarity, we searched for possible         rosy&#226;&#171;&#185;) from the lethal line l(2)03771: cype1 (Karpen and <br /> interactions with genes implicated in this process (Ter-         Spradling 1992) was localized by in situ hybridization <br /> racol and Lengyel 1994). We tested the effects of the            in the 25D4-6 interval, by the BDGP (http://www.fruit <br />                                          cyclope Encodes a COX VIc Homolog                                         1633 <br />  <br />  <br />  <br />  <br />                                                                            Figure 2.&#226;&#8364;&#8221;cype1 adult phenotypes. (A&#226;&#8364;&#8220;F) <br />                                                                         cype1/CyO adult flies. (A) Crooked legs. (B) <br />                                                                         Atrophic wing. (C) Fly with half a thorax. (D) <br />                                                                         Anterior part of the eye flattened. (E) Close <br />                                                                         eyes. (F) Fused eyes. (G and H) Mitotic y cype1/ <br />                                                                         y cype1 clones performed with the FLP/FRT <br />                                                                         technique (materials and methods). (G) <br />                                                                         Clone on the thorax with thinner and smaller <br />                                                                         y machrochaete. (H) Clones with thinner and <br />                                                                         smaller y bristles at the wing margin. <br />  <br />  <br />  <br />  <br /> fly.org/p_disrupt/; Spradling et al. 1995, 1999). We       lined). The 8-bp duplication is characteristic of P-ele- <br /> previously mapped the insertion site by plasmid rescue     ment insertions and is excised with the P element in <br /> within a genomic walk in the 25D region (George and        precise excision process (O&#226;&#8364;&#8482;Hare and Rubin 1983). <br /> Terracol 1997). The sequence at the junction between          Southern blot analysis was performed with cype1 and <br /> the genomic DNA and the 5&#226;&#172;&#732;P end is 5&#226;&#172;&#732;-GAATTTG              six lethal independent new cype lines. Genomic DNA <br /> CAAATGTGAGTAC/CATGATGAAA-3&#226;&#172;&#732;, which is iden-                was EcoRI digested and the blot was hybridized with <br /> tical to the sequence provided by the BDGP                 the 4.6-kb EcoRI fragment (E4.6, Figure 5D) extending <br /> (AQ025616). The genomic sequence downstream from           upstream from the 5&#226;&#172;&#732;P-insertion site. A 4.6-kb band was <br /> the 3&#226;&#172;&#732;P end was obtained by PCR amplification. The         generated in the wild-type Oregon-R and CyO chromo- <br /> sequence at the junction between the 3&#226;&#172;&#732;P end and the       somes while a 5.1-kb band was generated in cype1 by the <br /> genomic DNA is 5&#226;&#172;&#732;-TTTCATCATG/GTGAGTACATAA                  insertion of the P element (Figure 4). These results <br /> CAAGGTTTTT-3&#226;&#172;&#732;. This sequence shows an 8-bp dupli-          indicate that there are new patterns in four lines and <br /> cation (underlined) of genomic DNA present, in the         that deletions have occurred in the genomic region <br /> wild-type DNA, upstream from the 5&#226;&#172;&#732;P end (under-           flanking the 5&#226;&#172;&#732;P end. The same new bands were ob- <br />  1634                                                S. Szuplewski and R. Terracol <br />  <br />                                                                                                 TABLE 1 <br />                                                                                 Maternal enhancement of easter and cactus <br />                                                                                           phenotypes by cyclope <br />  <br />                                                                                                                     Phenotypes (%) <br />                                                                                                  Dead <br />                                                                      Female genotype          embryos (%)          V5     V4         V3 <br />                                                                      ea161.13/TM3                 62               100     0          0 <br />                                                                      cype1/&#226;&#171;&#185;; ea161.13/&#226;&#171;&#185;          99                10    80         10 <br />                                                                      cact99/CyO                    0               100     0          0 <br />                                                                      cype1 &#226;&#171;&#185;/&#226;&#171;&#185; cact99              6                 0    80         20 <br />                                                                        In all cases, females were crossed to wild-type males. V5: <br />                                                                      very weakly ventralized; head involution does not occur and <br />                                                                      some embryos have a tail up phenotype, probably due to <br />                                                                      defective retraction of the germband; Filzko&#194;&#168;rper are normal. <br />                                                                      V4: weakly ventralized; the head is not involuted and the <br />                                                                      posterior segments are internalized; Filzko&#194;&#168;rper are disorga- <br />                                                                      nized. V3: moderately ventralized; the ventral denticle belt is <br />                                                                      extended laterally; Filzko&#194;&#168;rper are reduced and disorganized; <br />                                                                      the head is not involuted and head and thorax are extended <br />                                                                      and convoluted. The phenotypic categories are according to <br />                                                                      Roth et al. (1991). <br />  <br />  <br />                                                                      indicate that the ry phenotype of these lines is due to <br />                                                                      an internal rearrangement within the rosy gene. cype1R4.5 <br />                                                                      and cype1R5.7 lines produced only amplified DNA with <br />                                                                      primer 3&#226;&#172;&#732;P-primer 3 or 4 pair giving the same band (0.6 <br />                                                                      kb) as the control, cype1. This indicates that a deletion <br />                                                                      of the 5&#226;&#172;&#732;P region had occurred. It was not possible to <br />    Figure 3.&#226;&#8364;&#8221;Scanning electron micrographs of cype1/cype1            recover any PCR product with these lines using various <br /> clonal eye. Clones were generated by the EGUF/hid method at <br /> 29&#226;&#172;&#353;. Photoreceptor cells bearing the dominant photoreceptor <br />                                                                      primers within the ry gene and, therefore, the exact <br /> cell lethal transgene GMR-hid die during metamorphosis. The          extent of the deletions could not be determined. Finally, <br /> l(2)CL-L1 recessive cell lethal mutation leads to lethality at an    cype1R2.5 and cype1R15.1 DNAs were not amplified with either <br /> earlier stage of eye development. (A and B) cype1[ry&#226;&#171;&#185; hs-neo         pair of primers. Both mutant line DNAs were amplified, <br /> FRT]40A/CyO; ey-GAL4 UAS FLP/&#226;&#171;&#185;, wild-type eye. (C and D)             however, with the primer 1-primer 3 or 4 pair giving a <br /> cype1/cype1 clonal eye from a cype1[ry&#226;&#171;&#185; hs-neo FRT]40A/[ry hs- <br /> neo FRT]40A GMR-hid l(2)CL-L1; ey-GAL4 UAS FLP/&#226;&#171;&#185; fly. (C) <br />                                                                      2.3-kb band with cype1R2.5 and a 1.7-kb band with cype1R15.1. <br /> Clonal eye with a rough aspect. (D) The ommatidia are disor-         These results indicate that the P element was excised <br /> ganized and present an abnormal morphology, bristles are             leading to a deletion in the genomic adjacent region <br /> missing or duplicated.                                               (Figure 5B). The DNA sequence resulting from the <br />                                                                      cype1R2.5 excision is 5&#226;&#172;&#732;-TTGCTATATT/ATATATATTATA <br />                                                                      TATATGTTATTTCATCATG/GTGAGTACATAACAA <br /> served with the 2.2-kb BglII-EcoRI internal probe (BE2.2,            GGT-3&#226;&#172;&#732;. The excision has generated a 402-bp deletion in <br /> Figure 5D) indicating that the deletions do not reach                the genomic DNA and the elimination of most of the <br /> the BglII site. In the excised lines, new fragments of 4.2           transgene. An insertion of 14 nucleotides (AT) not present <br /> kb (cype1R2.5), 5 kb (cype1R4.5), 7.5 kb (cype1R5.7), and 3.6 kb <br /> (cype1R15.1) replace the 5.1-kb EcoRI band. Hybridization <br /> of the same blot with the 2.2-kb EcoRI fragment (E2.2,                                          TABLE 2 <br /> Figure 5D) mapping downstream from the P-element                              Maternal enhancement of dpp lethality by cype1 <br /> 3&#226;&#172;&#732; end insertion site gave a single 2.2-kb EcoRI band in <br /> all cases, indicating that this fragment was not altered                                               % of expected &#226;&#171;&#185; cype1/dpp &#226;&#171;&#185; <br /> by the insertion or the excisions.                                                                            adult progeny <br />    PCR amplifications were performed to determine the <br />                                                                      Allele            From              dpphr27               dpphr4 <br /> extent of the deletions. Positions of the primers are <br /> shown in Figure 5B and the results are summarized in                 cype1            Females               4                   38 <br /> Table 3. cype1R7.1 and cype1R9.1 lines produced amplified                             Males               104                   71 <br /> DNA with both primer 1-primer 5&#226;&#172;&#732;P and primer 3&#226;&#172;&#732;P-                       The values given are percentages of &#226;&#171;&#185; cype1/dpp &#226;&#171;&#185; adult <br /> primer 3 or 4 pairs giving the same bands, 2.2 and 0.6               flies compared to the quantity of cype1/CyO control progeny. <br /> kb long, respectively, as the control, cype1. These results          At least 300 flies were counted in each experiment. <br />                                                cyclope Encodes a COX VIc Homolog                                              1635 <br />  <br />                                                                                            TABLE 3 <br />                                                                                    cype deficiency mapping <br />  <br />                                                                                                          Primers <br />                                                                  cype allele           1-5&#226;&#172;&#732;P                3&#226;&#172;&#732;P-4              1-4 <br />                                                                  1                      2.2                  0.6               &#226;&#8364;&#8221; <br />                                                                  1R2.5                  &#226;&#8364;&#8221;                    &#226;&#8364;&#8221;                 2.3 <br />                                                                  1R4.5                  &#226;&#8364;&#8221;                    0.6               &#226;&#8364;&#8221; <br />                                                                  1R5.7                  &#226;&#8364;&#8221;                    0.6               &#226;&#8364;&#8221; <br />                                                                  1R7.1                  2.2                  0.6               &#226;&#8364;&#8221; <br />                                                                  1R9.1                  2.2                  0.6               &#226;&#8364;&#8221; <br />                                                                  1R15.1                 &#226;&#8364;&#8221;                    &#226;&#8364;&#8221;                 1.7 <br />                                                                     Genomic DNA from cype1 and the six revertant lines was <br />                                                                  PCR amplified with three pairs of primers. The <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> of the <br />                                                                  fragments is given in kilobases and primer positions are shown <br />                                                                  in Figure 5B. <br />   Figure 4.&#226;&#8364;&#8221;Southern blot analysis of the cype1 excised lines. <br /> About 10 &#226;&#65533;&#174;g of DNA (10 flies) were EcoRI digested in 30 &#226;&#65533;&#174;l <br /> (20 units overnight) with 10 &#226;&#65533;&#174;g RNAse A. (&#226;&#171;&#185;) Oregon-R wild-      from the cype1R15.1 excision is 5&#226;&#172;&#732;-TACGGATTTT/TATTT <br /> type DNA, compared to the cype1 (1) and the six excised lines.   CATCATG/GTGAGTACATAACAAGGT-3&#226;&#172;&#732;. The excision <br /> The blot was hybridized with the 4.6-kb EcoRI fragment (E4.6,    has generated a 973-bp deletion in the genomic DNA <br /> Figure 5D). <br />                                                                  and the elimination of most of the P element. Twelve <br />                                                                  nucleotides resulting from the imprecisely excised P-ele- <br /> in the genomic DNA and 17 nucleotides (underlined)               ment 3&#226;&#172;&#732; end (underlined) remain between the two geno- <br /> from the imprecisely excised P-element 3&#226;&#172;&#732; end separates          mic sequences. The 8-bp duplication is still present in the <br /> the two genomic sequences. The DNA sequence resulting            excised lines. <br />  <br />  <br />                                                                                                   Figure 5.&#226;&#8364;&#8221;Genomic map of <br />                                                                                                the cyclope region. (A) DM194B8: <br />                                                                                                SP6 end of a cosmid isolated by <br />                                                                                                the European Drosophila Map- <br />                                                                                                ping Consortium. The extent of <br />                                                                                                the deletions generated in the PZ <br />                                                                                                element and the position of the <br />                                                                                                PZ element of cype1 are shown. The <br />                                                                                                scale for the PZ element (15 kb) <br />                                                                                                was reduced by half to fit the fig- <br />                                                                                                ure. Only EcoRI (E) and XbaI (Xb, <br />                                                                                                plasmid rescue cloning site) re- <br />                                                                                                striction sites are shown. The posi- <br />                                                                                                tion of the 5&#226;&#172;&#732; and 3&#226;&#172;&#732; P primers is <br />                                                                                                shown. (B) The restriction map of <br />                                                                                                the region is indicated (E, EcoRI; <br />                                                                                                B, BglII; Xh, XhoI; Xb, XbaI). The <br />                                                                                                position of the primers (1, 2, 3, <br />                                                                                                4) used for PCR amplification and <br />                                                                                                the extent of the deletions gener- <br />                                                                                                ated in the genomic region are <br />                                                                                                shown. (C) The positions and ori- <br />                                                                                                entations of the cyclope cDNAs, pu- <br />                                                                                                tative GadFly genes, CG14023, <br />                                                                                                CG14028 (cyclope), CG14022, <br />                                                                                                CG14027, and CG14021, are shown <br />                                                                                                together with the BDGP cDNAs iso- <br />                                                                                                lated in the region, GH04604 and <br />                                                                                                LP10225 (cyclope) and GH20612, <br />                                                                                                GH14865 (CG14021). (D) The ex- <br />                                                                                                tent of the probes used for South- <br />                                                                                                ern blot and cDNA screening <br />                                                                                                analyses and of the transgene <br />                                                                                                Pw6BX3.8, used for rescue experi- <br />                                                                                                ments, is indicated. <br />  1636                                            S. Szuplewski and R. Terracol <br />  <br />    cyclope encodes a cytochrome c oxidase subunit VIc <br /> homolog: cDNA &#226;&#65533;&#173;-libraries were screened using genomic <br /> DNA fragments mapping on both sides of the cype1 transpo- <br /> son. No cDNAs were found with the 2.2-kb BglII-EcoRI <br /> fragment (BE2.2, Figure 5D) mapping upstream from the <br /> P-element 5&#226;&#172;&#732; end while many cDNAs (16) were isolated <br /> using the 2.2-kb EcoRI fragment (E2.2) mapping down- <br /> stream from the P-element 3&#226;&#172;&#732; end (map, Figure 5D). Two <br /> cDNAs were isolated from a &#226;&#65533;&#173;gt10 embryonic (1.5- to <br /> 5-hr) library, 365 and 431 bp long. Two cDNAs 249 and <br /> 171 bp long were isolated from a 0- to 24-hr embryonic <br /> &#226;&#65533;&#173;Zap II cDNA library. Finally, 12 cDNAs 171 to 449 bp <br /> long were isolated from a &#226;&#65533;&#173;Zap II late L3 stage library. <br /> These cDNAs overlap one another and also overlap with <br /> two cDNAs isolated by the BDGP (http://www.fruitfly.org/ <br /> EST/; Rubin et al. 2000), GH04604 (399 bp), isolated <br /> from adult head, and LP10225 (371 bp) from larval-early <br /> pupal cDNA libraries. They correspond to the CG14028 <br /> GadFly gene identified by the Genome Annotation Data- <br /> <span id='am-322' about='xsp:base' typeof='owl:Thing'>base</span> of Drosophila (http://www.fruitfly.org/annot/; <br /> Adams et al. 2000). Two putative genes are proposed in <br /> GadFly on the other strand. The 5&#226;&#172;&#732; end of the first one <br /> (CG14023) maps 2.5 kb upstream from the P-element <br /> insertion site and the 3&#226;&#172;&#732; end of the other (CG14022) maps <br /> 60 bp downstream from the 3&#226;&#172;&#732; end of the longest cDNA <br /> we recovered. Neither the BDGP screen nor our screen                Figure 6.&#226;&#8364;&#8221;cyclope genomic and predicted amino acid se- <br /> identified corresponding cDNAs. Given its position, at           quences. The sequence corresponding to cDNAs is in capital <br />                                                                  letters. The start (1) corresponds to the first nucleotide of <br /> the 3&#226;&#172;&#732; end of CG14028, and its orientation (opposite),           the longest cDNA (GH04604) at the 5&#226;&#172;&#732; end and the end <br /> CG14022 is unlikely to be altered by the cype1 transposon.       corresponds to the longest cDNA at the 3&#226;&#172;&#732; end (743). Position <br /> Furthermore, no transcripts corresponding to CG14022             19 corresponds to cDNA19 start and is the longest 5&#226;&#172;&#732; end that <br /> were detected in developmental Northern blots with a 0.5-        we recovered. The most representative cDNA starts and ends <br /> kb ClaI genomic probe (C0.5, Figure 5D) including this           are marked by an arrowhead, and the insertion site of the PZ <br />                                                                  element l(2)03771 at position 55 is shown (V). The two introns <br /> putative transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. Therefore no other cDNAs            are shown (lowercase letters) and the donor (gt), acceptor <br /> were identified within the 8 kb surrounding CG14028              (ag), the ATG, TAG, and polyadenylation sites are in boldface. <br /> (Figure 5). GH04604 is the longest cDNA at the 5&#226;&#172;&#732; end,           The <span id='am-4' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> for the sequence reported <br /> which corresponds to position 1 in the genomic sequence          here is AY029256. <br /> (Figure 6). Most of the cDNAs (13/18) differ slightly at <br /> the 5&#226;&#172;&#732; end and start at positions 1, 19, 34, 35 (4), 41 (3), <br /> 43 (2), 46. Fifteen cDNAs have a poly(A) tail. Eight are         downstream from the 5&#226;&#172;&#732; end of the first intron. The splic- <br /> identical at the 3&#226;&#172;&#732; end (666), and four end at position          ing of this first intron is probably defective and it is there- <br /> 670. The others end, respectively, at 672, at 738, and           fore very likely that the 15-kb PZ element insertion of cype1 <br /> the longest at the 3&#226;&#172;&#732; end, at position 743 (1.5- to 5-hr         alters the function of the gene corresponding to these <br /> embryonic).                                                      cDNAs. We found three sequences that fit the consensus <br />    Comparison of cDNA and genomic sequences (Figure              polyadenylation site, AATAAA (Proudfoot and Brown- <br /> 6) revealed the presence of two introns. The first (47&#226;&#8364;&#8220;262)      lee 1976), preceding most of the cDNA ends: AAATA <br /> is 216 bp long and the second (437&#226;&#8364;&#8220;503) is 67 bp long.           between positions 646 and 650 followed by the related <br /> One cDNA (late L3) starts in intron 1 (89) and shows a           sequences AATTTA (685&#226;&#8364;&#8220;690) and AAATTA (730&#226;&#8364;&#8220; <br /> 152-bp first intron (111&#226;&#8364;&#8220;262) due to alternative splicing.       735). Between positions &#226;&#171;&#186;93/&#226;&#171;&#186;85 we found the se- <br /> In these introns are found the predicted splice donor and        quence TAATATATA that fits the TATA box consensus, <br /> acceptor sites (GT at 5&#226;&#172;&#732; and AG at 3&#226;&#172;&#732; splice sites) that fit     and the CAAT sequence that fits the CAT box consensus <br /> the invertebrate splice junction consensus (Mount et al.         is present between positions &#226;&#171;&#186;80/&#226;&#171;&#186;77. The first ATG <br /> 1992). The splice site predictions from the BDGP program         in frame is situated at position 267 after the first intron, <br /> gave, for the three donor sites (GT) observed, the scores        and a stop codon, TAG, is found at position 565. The <br /> 0.91 (47), 0.51 (111), and 0.42 (437). The scores for the        sequence flanking the translation start is CGACATG, <br /> acceptor sites (AG) are 0.93 (262) and 0.97 (503). These         which fits the Drosophila consensus C/AAAA/CATG <br /> results are in good agreement with our findings. The cype1       (Cavener 1987). There is only one difference, a G, which <br /> P element is inserted in base position 55, eight nucleotides     is a secondary preference, instead of an A at position &#226;&#171;&#186;3. <br />                                                cyclope Encodes a COX VIc Homolog                                            1637 <br />  <br />                                                                                              Figure 7.&#226;&#8364;&#8221;Comparison of the Dro- <br />                                                                                            sophila COX VIc protein with the ho- <br />                                                                                            mologs in human, rat, and yeast: Dro- <br /> sophila 77 aa (GenBank accession no. AAK32965), human 75 aa (Otsuka et al. 1988; GenBank accession no. NP 004365), rat <br /> 76 aa (Suske et al. 1988; GenBank accession no. P11951), and yeast (S. cerevisiae) 59 aa (Wright et al. 1986; GenBank accession <br /> no. J02633). <br />  <br />  <br />    This analysis shows that the cype1R2.5 and cype1R15.1 dele-    The subunit VIII has the lowest identity score; neverthe- <br /> tions, 402 and 973 bp, respectively, eliminate the whole 5&#226;&#172;&#732;       less, the size of the putative protein, 68 aa, is in good <br /> transcript region upstream from position 55. This includes        agreement with the size of the human subunit VIII (69 <br /> the first intron 5&#226;&#172;&#732; end, the transcript initiation sites, and     aa). Four subunits are present in two isoforms. This is the <br /> the CAT and TATA putative boxes. However, among the               case for subunits, IV, Vb, VIa, and VIIa. Identities between <br /> 18 cDNAs studied one (late L3) starts in the first intron         the isoforms is in the range of 40&#226;&#8364;&#8220;50%, which is, in most <br /> (89) and shows an alternative splicing. The corresponding         cases, lower than the score obtained for a given <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> com- <br /> sequence is not altered in either these lines or cype1. Since     pared to the corresponding human <span id='am-324' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. This finding <br /> the coding sequence is also preserved, it is possible that        indicates that the gene duplication is ancient. The two <br /> these mutations do not abolish the expression of the gene,        genes encoding the isoforms for subunits Vb and VIIa are <br /> but, rather, severely alter its regulation. Alternatively, be-    both duplicated at the same chromosomal site and the <br /> cause this cDNA is unique among the cDNAs studied (18),           isoforms are also the more divergent (42% identity for <br /> it is likely that it is not very abundant and does not allow      both). Subunit Vb genes are duplicated in tandem and <br /> significant protein production. Therefore these alleles           are separated by 382 bp, while subunit VIIa genes are <br /> could be null alleles.                                            duplicated in opposite directions and are separated by <br />    A 3.8-kb genomic transgene (Figure 5D) including this          2.7 kb. Only 1 subunit, Va, was previously identified as <br /> transcription <span id='am-325' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> was able to completely rescue, in one          corresponding to the CoVa gene but no mutant has been <br /> dose, the cype1 and the four new cype alleles, in hemizygotes     described (Caggese et al. 1999). Therefore, cyclope is so <br /> and inter se. These results demonstrate that these cDNAs          far the first cytochrome c oxidase mutant to be recovered <br /> correspond to the cyclope gene and also show that the 973-        in Drosophila. <br /> bp cype1R15.1 deletion does not alter any other vital genes.         Temporal and spatial expression of cype transcripts: <br />    The predicted 77-amino acid protein (8.295 kD) is              Northern blot analysis (Figure 8) shows the presence of <br /> closely related to the mammalian cytochrome c oxidase             transcripts of &#226;&#181;&#8216;370 bp throughout development. This size <br /> subunit VIc (Figure 7). The Drosophila protein shares             is consistent with the length of the cDNAs recovered and <br /> over 65 amino acids (aa), 46% identity, and 59% similarity        with those of the mammalian cytochrome c oxidase VIc <br /> with the human protein (Otsuka et al. 1988) and 43%               transcripts (450&#226;&#8364;&#8220;510 bp). There is no major quantitative <br /> identity and 59% similarity with the rat subunit (Suske et        variation between the different developmental stages <br /> al. 1988). Both the percentages of identity and similarity        when compared with the internal control, rp49, except in <br /> and the size of the protein, 75 aa in human and 76 in             males where transcripts appear relatively more abundant <br /> rat, correlate with the hypothesis that cyclope encodes the       than in female and in other development stages. <br /> cytochrome c oxidase VIc subunit. The corresponding                  To confirm that the cype RNA expression is altered in <br /> yeast subunit (Wright et al. 1986) is more divergent.             the different cype alleles we performed Northern blots with <br />    In the higher eukaryotes, cytochrome c oxidase contains        total RNA from adult heterozygous females and moni- <br /> 3 mitochondrial DNA-encoded subunits and up to 10                 tored cype RNA (Figure 9). cype RNA level is decreased to <br /> nuclear DNA-encoded subunits in mammals. In Drosoph-              0.61 &#226;&#171;&#190; 0.08 in Df(2L)tkvSz2/&#226;&#171;&#185;, 0.63 &#226;&#171;&#190; 0.07 in cype1/&#226;&#171;&#185;, <br /> ila the 3 mitochondrial DNA-encoded subunits were pre-            0.66 &#226;&#171;&#190; 0.08 in cype1R2.5/&#226;&#171;&#185;, and 0.54 &#226;&#171;&#190; 0.06 in cype1R15.1/&#226;&#171;&#185; <br /> viously identified (de Bruijn 1983). As shown in Table 4,         flies, when compared to &#226;&#171;&#185;/&#226;&#171;&#185; flies. These values are not <br /> these subunits are highly conserved when compared to              significantly different and show that the cype RNA level is <br /> the human peptides (Anderson et al. 1981). Subunit I is           abolished or strongly reduced in the three cype alleles. <br /> the best conserved with 68% identity and 79% similarity.             In situ hybridization performed in embryos (Figure 10, <br /> Subunit II is 58% identical and 69% similar and subunit           A&#226;&#8364;&#8220;D) reveals the presence of a maternal product in pre- <br /> III 61% identical and 75% similar to the human proteins.          blastoderm embryos (A). Then, at stage 10 (germband <br /> A search for homologs of the human subunit precursors in          extension), RNAs are detectable in neuroblasts (B). In <br /> the Genome Annotation Database of Drosophila (http://             stage 13 embryos (C), RNAs are present in the central <br /> www.fruitfly.org/annot/; Adams et al. 2000), using the            nervous system (CNS). At stage 17 the labeling is present <br /> BDGP Blast server, revealed 9 nuclear subunits while 1,           only in the condensed CNS (D). lacZ staining in cype1 <br /> VIIb, was not found (Table 4). These subunit putative             (Figure 10, E&#226;&#8364;&#8220;M) shows, in stage 10 embryos, a labeling <br /> genes are located on all chromosomes (except chromo-              in head and neuroblasts (E) and at stage 16 in the CNS <br /> some 4) and present a high level of identity with the             (F). lacZ staining is detected in larval brain (not shown), <br /> human subunits, ranging from 29% (VIII) to 59% (VIb).             in the central region of imaginal discs (G and H), and in <br />  1638                                            S. Szuplewski and R. Terracol <br />  <br />                                                            TABLE 4 <br />           Putative cytochrome c oxidase subunits encoded by the mitochondrial and nuclear genome of Drosophila <br />  <br />                                                                  AA(N)           I/S(%)          I/S(%) <br />         Subunit            GadFly             Location          Dro/Hum          Dro/Dro        Dro/Hum           Ref. <br />         I                                   mtDNA                511/513                           68/79          1,   2 <br />         II                                  mtDNA                228/227                           58/69          1,   2 <br />         III                                 mtDNA                262/261                           61/75          1,   2 <br />         IV             CG10664              2L; 38A8             182/169          51/66            39/56          3 <br />                        CG10396              2R; 41A3             162/169                           38/59          3 <br />         Va             CG14727/CoVa         3R; 86F10-11         149/150                           52/70          4,   5 <br />         Vb             CG11015              2L; 26E2             120/129          42/67            51/67          6 <br />                        CG11043              2L; 26E2             154/129                           34/55          6 <br />         VIa            CG17280              2R; 59E3             109/109          45/68            55/69          7 <br />                        CG14077              3L; 75E4             118/109                           47/60          7 <br />         VIb            CG14235              X; 18F1               92/86                            59/77          8 <br />         VIc            CG14028/cype         2L; 25D5-6            77/75                            46/59          9 <br />         VIIa           CG18193              3R; 84F13-14         106/79           42/65            57/77         10 <br />                        CG9603               3R; 84F13             89/79                            38/61         10 <br />         VIIb                &#226;&#8364;&#8221;                                                                                    11 <br />         VIIc           CG2249               2R; 46D7              51/63                            46/64         12 <br />         VIII           CG7181               3L; 78E1              68/69                            29/51         13 <br />            Putative cytochrome c oxidase nuclear genes were searched for in the Genome Annotation Database of <br />         Drosophila (GadFly) using the human precursor peptide sequence. The nomenclature used is from Kadenbach <br />         and Merle (1981). AA(N), comparison of the number of amino acids in Drosophila (Dro) and human (Hum). <br />         I/S(%), comparison of percentage of identity (I) and similarity (S) between two Drosophila isoforms or <br />         between Drosophila and human peptides. (1) de Bruijn (1983); (2) Anderson et al. (1981); (3) Zeviani et <br />         al. (1987); (4) Caggese et al. (1999); (5) Rizzuto et al. (1988); (6) Zeviani et al. (1988); (7) Fabrizi et al. <br />         (1989); (8) Taanman et al. (1989); (9) Otsuka et al. (1988); (10) Arnaudo et al. (1992); (11) Sadlock et al. <br />         (1993); (12) Koga et al. (1990); (13) Rizzuto et al. (1989). <br />  <br />  <br />  <br /> ovarian germarium (I), follicular and nurse cells (J). A          been observed that changes in the respiratory chain <br /> stronger cype expression is detected in developing oocytes        activity are followed by alterations in nuclear gene ex- <br /> rather than in follicular cells. High level lacZ staining is      pression. This is the case, for example, for mutants in <br /> detected in larval male gonad (K), in adult testis (L),           the oxen gene of Drosophila encoding the ubiquinol- <br /> and spermatozoids (M). The strong expression in male              cytochrome c oxidoreductase subunit 9 (Frolov et al. <br /> reproductive organs is likely to be responsible for the           2000). The mutation preferentially exerts a negative <br /> enhanced cype expression observed by Northern blot analy-         dosage effect upon the expression of the white, brown, <br /> sis in adult males (Figure 8). The correlation between in         and scarlet genes. This mechanism, called retrograde <br /> situ and lacZ staining in the embryo and in the other             regulation, has been hypothesized to monitor and adjust <br /> tissues (not shown) is consistent with the implication of         cell functions in response to perturbations in mitochon- <br /> the P element in the alteration of the cytochrome c oxidase       dria (Parikh et al. 1987; Poyton and McEwen 1996). <br /> function.                                                         cype is haploinsufficient, and 35% of the cype/&#226;&#171;&#185; pupae <br />                                                                   die. Adult phenotypes are observed in cype/&#226;&#171;&#185; individu- <br />                                                                   als, mostly due to defects in imaginal disc development. <br />                        DISCUSSION                                 Alteration of imaginal disc derivatives has been de- <br />    cyclope is a vital gene with pleiotropic effects: cype is      scribed for genes encoding mitochondrial proteins. Mu- <br /> a previously undescribed homozygous lethal gene. The              tants in the colt gene encoding a mitochondrial carrier <br /> P element of the initial allele cype1 was excised leading         exhibit loss of venation and reduction in wing size <br /> to viability, showing that the lethality was due to the           (Hartenstein et al. 1997). Mutations in the tumor sup- <br /> presence of the P element. New alleles were generated             pressor gene, l(2)tid, encoding a transmembrane mito- <br /> by imprecise excision of cype1 P element. The mutation            chondrial protein lead to imaginal discs that fail to dif- <br /> causes slight defects in the dorsoventral polarity of the         ferentiate and grow into lethal tumors (Kurzik-Dumke <br /> embryo. The defects are enhanced by decreasing simul-             et al. 1992, 1998). Mutations in the tamas gene, encoding <br /> taneously the maternal activity of cype and the activity          a mitochondrial DNA polymerase catalytic subunit, in- <br /> of maternal or zygotic dorsoventral polarity genes. How           duce smaller eyes with a rough aspect and disorganized <br /> do COX alterations cause defects in dorsoventral polar-           ommatidia (Iyengar et al. 1999). A similar phenotype <br /> ity? Although this may appear rather surprising, it has           is observed in cype1 clonal eyes. cype1 germline clones are <br />                                                   cyclope Encodes a COX VIc Homolog                                             1639 <br />  <br />  <br />  <br />  <br />                                                                          Figure 9.&#226;&#8364;&#8221;Northern blot analyses of cype expression. Total <br />                                                                       adult female RNA (20 &#226;&#65533;&#174;g) was loaded in each lane and three <br />                                                                       independent Northern blots were performed using the same <br />                                                                       RNA extract. Each blot was hybridized with the cype cDNA19 <br />                                                                       and the rp49 cDNA as a loading control. The blots were quanti- <br />                                                                       fied with a Storm 860 (Molecular Dynamics) phosphorimager. <br />                                                                       The cype values were normalized to the rp49 values. The final <br />                                                                       results are mean values corresponding to fractions of the &#226;&#171;&#185;/&#226;&#171;&#185; <br />                                                                       values. <br />  <br />  <br />  <br />                                                                       encodes a 77-aa peptide 46% identical and 59% similar <br />                                                                       to the human COX VIc subunit. No other types of cDNA <br />                                                                       were isolated and no other transcripts were detected in <br />                                                                       the region screened, 2.2 kb upstream and 2.2 kb down- <br />    Figure 8.&#226;&#8364;&#8221;Developmental Northern blot analysis. Poly               stream of the P-insertion site. Furthermore, the BDGP <br /> (A&#226;&#171;&#185;) Oregon-R RNA (10 &#226;&#65533;&#174;g) was loaded and run in a 1%                  screen (Rubin et al. 2000) revealed no other cDNAs within <br /> formaldehyde agarose gel. The blot was hybridized with                the 8 kb surrounding the COX VIc gene, although they <br /> cDNA19 from &#226;&#65533;&#173;Zap II embryonic (0- to 24-hr) library. cDNA19 <br />                                                                       were searched for at all developmental stages and in differ- <br /> is 394 bp long and maps between positions 19 and 670 (Figure <br /> 6). The cDNA rp49 encoding a ribosomal protein was used               ent types of tissues. The presence of the P element in the <br /> as a loading control. The developmental stages are as follows:        first intron of the COX VIc cDNA together with the fact <br /> embryonic stages are 0&#226;&#8364;&#8220;4 hr, 4&#226;&#8364;&#8220;8 hr, 8&#226;&#8364;&#8220;24 hr (hours after egg         that no other transcripts were found in the close vicinity <br /> laying); L3, third instar larval stage (3&#226;&#8364;&#8220;5 days after egg laying);   argues in favor of the implication of the P element in the <br /> P, pupal stage (6&#226;&#8364;&#8220;7 days after egg laying); F, adult female; <br /> and M, adult male. <br />                                                                       alteration of the COX VIc transcript. Furthermore, cype <br />                                                                       RNA level is strongly reduced in cype mutants. This was <br />                                                                       unambiguously demonstrated by the complete rescue of <br /> lethal while somatic clones in the adult cuticle lead to              cype1 and the four new mutations with one dose of a 3.8-kb <br /> reduced bristles and cell size. These phenotypes are                  genomic transgene that includes the COX VIc transcript. <br /> compatible with a diminished COX activity that is likely                 Different initiation sites were identified in cype cDNAs <br /> to induce cell death and a decrease in cell growth. The               as well as a TATA and a CAT box upstream from the <br /> cype embryonic and adult phenotypes are probably due                  transcription starts. Multiple initiation sites are generally <br /> to a general decrease in COX activity detected preferen-              observed in nuclear DNA-encoded COX subunits. In <br /> tially in processes requiring a high energy level. In hu-             mammals, for instance, a single transcription initiation <br /> man, for instance, COX diseases affect mainly brain,                  site is observed in rat and bovine subunit VIII genes only <br /> eye, and muscle tissue (Taanman 1997; Poyton 1998;                    (Lenka et al. 1998). Transcription initiation sites of diverse <br /> Leonard and Schapira 2000a,b).                                        tissue-specific COX and ubiquitous genes fall into two <br />    CYPE protein is homologous to human cytochrome c                   categories, those containing a TATA box and the others. <br /> oxidase subunit VIc: The gene was cloned by localization              Generally, genes that lack the TATA box are of the <br /> of the initial P-element allele cype1. The new alleles gener-         housekeeping type and show 5&#226;&#172;&#732; heterogeneity. However, <br /> ated by imprecise excision of cype1 P element were analyzed           TATA and CAT boxes were identified upstream from the <br /> at the molecular level and shown to delete the 5&#226;&#172;&#732; end of              multiple initiation sites of subunits VIa (H) and VIc (rat) <br /> the gene. Around the insertion site, 16 overlapping cDNAs             genes. A TATA box was also found upstream from the <br /> with a size of &#226;&#181;&#8216;370 bp were isolated from embryonic and               single initiation site of rat and bovine subunit VIII (H) <br /> third larval instar libraries. The gene contains two introns          genes. Subunit VIc gene from rat resembles cype gene with <br /> and the P element was localized in the first one. The gene            multiple initiation sites and TATA and CAT boxes. Most <br />  1640                                              S. Szuplewski and R. Terracol <br />  <br />                                                                    firmed by the study of the COX9 gene of yeast encoding <br />                                                                    the COX VIc (VIIa in yeast nomenclature) homolog show- <br />                                                                    ing that the subunit lacks a leader peptide (Wright et al. <br />                                                                    1986). There is, however, a precursor longer by one amino <br />                                                                    acid (M) at the NH3 terminus and four amino acids at <br />                                                                    the COOH terminus. In the mutant strain with deletion <br />                                                                    of the COX9 gene, cells lack a functional cytochrome c <br />                                                                    oxidase, which suggests that the subunit VIc is an essential <br />                                                                    component of the holoenzyme. In yeast the three smallest <br />                                                                    subunits (VIc, VIIa, and VIIc) are transmembranous and <br />                                                                    play a role in assembly and stability, which is consistent <br />                                                                    with the fact that mutants in COX9 also lack many of the <br />                                                                    other COX subunits. <br />                                                                       cype is expressed in localized domains during develop- <br />                                                                    ment: Study of spatial and temporal transcripts showed <br />                                                                    that cype is expressed throughout development (Northern) <br />                                                                    with no significant quantitative modulation, except in <br />                                                                    adult male where transcripts are more abundant. In situ <br />                                                                    hybridization in preblastoderm embryos revealed the pres- <br />                                                                    ence of a maternal product. This finding confirms the <br />                                                                    genetic analysis that demonstrates the maternal effect of <br />                                                                    the gene. Transcripts were preferentially observed in em- <br />                                                                    bryonic neuroblasts and CNS, which was confirmed by <br />                                                                    lacZ staining. This correlation confirms the role of the <br />                                                                    P element insertion in cype expression. Later in develop- <br />                                                                    ment, lacZ staining is observed in larval brain, in imaginal <br />                                                                    discs, in the germarium, nurse, and follicular cells of the <br />                                                                    ovary, and in male reproductive organs. Therefore cype <br />                                                                    appears to be expressed in specific tissues and in localized <br />                                                                    regions, unlike the ubiquitous mammalian COX VIc gene. <br />                                                                    Only a small number of tissues have been studied in mam- <br />                                                                    mals, essentially heart and liver tissues, and expression <br />                                                                    during development has not been studied as yet. These <br />    Figure 10.&#226;&#8364;&#8221;In situ hybridizations and lacZ staining. (A&#226;&#8364;&#8220;D) <br /> In situ hybridizations in wild-type embryos performed with the <br />                                                                    localized transcripts may reflect a greater energy require- <br /> digoxigenin-labeled DNA fragment E2.2 (Figure 5D). The             ment. It is noteworthy that mutations in numerous mito- <br /> results are identical when using the antisense RNA probe           chondrial proteins result in nervous system defects. Behav- <br /> made from pBScDNA19. Stages are according to Campos-               ioral defects (bang sensitivity) are observed in mutants in <br /> Ortega and Hartenstein (1985). (A) Preblastoderm em-               tko with altered ribosomal protein S12 (Royden et al. 1987) <br /> bryo showing a maternal contribution. (B) Extended germ- <br /> band stage with a segmental <span id='am-323' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. (C) Stage 13 embryo <br />                                                                    and in the stress sensitive B gene encoding an adenine <br /> with expression in the CNS. (D) Stage 17 embryo showing            nucleotide translocase (Zhang et al. 1999). The sluggish-A <br /> labeling exclusively in the condensed CNS. (E&#226;&#8364;&#8220;M) lacZ stain-       gene encoding a proline oxidase is predominantly ex- <br /> ing in the cype1 allele. (E) Stage 10 embryo with labeling in      pressed in the embryonic central nervous system and <br /> neuroblasts. (F) Stage 17 embryo showing labeling in CNS.          depletion of its activity induces locomotory defects <br /> (G and H) Staining in the central region of leg (G) and eye- <br />                                                                    (Hayward et al. 1993). Mutants in the colt gene encod- <br /> antennal imaginal discs (H). (I and J) Staining in ovaries with <br /> labeling in germarium (I) and in follicular and nurse cells at     ing a mitochondrial carrier exhibit a sluggish response <br /> stage 10 ( J). cype RNA expression is higher in developing         to physical contact (Hartenstein et al. 1997). Similar <br /> oocyte than in follicular cells. (K&#226;&#8364;&#8220;M) Staining in male repro-     defects are observed in mutants in the oxen gene encod- <br /> ductive organs. (K) Larval (L3) gonads are stained except at       ing the ubiquinol-cytochrome c oxidoreductase subunit <br /> the anterior end (germ cells). Adult testis (L) and spermatozo- <br />                                                                    9 (Frolov et al. 2000). Finally, mutations in the tamas <br /> ids (M) strongly labeled. <br />                                                                    gene, encoding a DNA polymerase catalytic subunit, <br />                                                                    disrupt the development of the visual system and larvae <br /> (eight) of the mammalian nuclear-encoded COX subunits              present a reduced response to light due to locomotory <br /> contain N-terminal mitochondrial targeting sequence that           deficiency (Iyengar et al. 1999). <br /> is cleaved off during importation. Presequence size ranges            Nine putative cytochrome c oxidase nuclear subunits <br /> from 12 aa (VIa) to 41 aa (Va) and is not found in COX             are found in Drosophila: Comparison of the three mito- <br /> VIb and COX VIc. It is therefore likely that CYPE does             chondrial DNA-encoded subunits I, II, and III with the <br /> not contain either presequence. This hypothesis was con-           corresponding human peptide shows a high degree of <br />                                             cyclope Encodes a COX VIc Homolog                                                     1641 <br />  <br /> conservation in size and sequence (about 65% identity).       is observed. Nevertheless, although cells in mitochon- <br /> This is not surprising because these subunits are found       drial diseases are disabled, in many cases they do not <br /> in all species from bacteria to human and constitute          die. For example, muscle necrosis is rarely found in <br /> the catalytic core of the enzyme. A search in the Genome      mitochondrial myopathies and brain regions in KSS and <br /> Annotation Database of Drosophila (Adams et al. 2000)         MELAS syndromes show little neuronal loss. <br /> revealed the presence of nine putative nuclear subunits          It will be therefore interesting to research, in cyclope <br /> homologous to human peptides. Except for subunit              mutants, the occurrence of defects in the nervous system <br /> VIIb, we found a Drosophila homolog in all cases. We          and behavior. Furthermore, isolation and study of other <br /> searched for these subunits, however, using the human         mutations in the OXPHOS chain and, principally, in <br /> sequences and the VIIb peptide is perhaps too divergent       the cytochrome c oxidase (complex IV) of Drosophila <br /> to be found. Cytochrome c oxidase subunits have been          will probably contribute to our understanding of, not <br /> studied essentially in mammals and no sequences from          only the human degenerative OXPHOS diseases, but <br /> species closer to Drosophila are available. Alternatively,    also the aging process. <br /> it is possible that this subunit does not exist in Drosoph-      <br /> ila. In yeast, for example, there are only eight nuclear      <br /> subunits, VIIb and VIII being absent (Capaldi 1990).        <br /> Drosophila subunit VIII is the most divergent (29%          <br /> identity) and was not identified in the database as a        <br /> COX VIII homolog. However, the size of the peptide       <br /> (68 aa) is very close to that of the human (69 aa). These      <br /> subunits are found on all chromosomes (except four) <br />                                                        <br /> and four have isoforms, IV, Vb, VIa, and VIIa. Two             <br /> forms of subunits VIa and VIIa have also been found <br /> in mammals while two forms of subunit IV have been <br /> identified in yeast. Two forms of subunit VIc have been                             <br /> found in Dictyostelium but only one in Drosophila. The <br />                                                                <br /> presence of isoforms reflects the different energy re-             <br /> quirements depending on the tissues, developmental                <br /> stages, or growth conditions.                           <br />     Human OXPHOS diseases affect mainly brain and            <br /> muscles: In the human, OXPHOS diseases (reviewed in     <br /> Poyton 1998; Wallace 1999; Leonard and Schapira            <br /> 2000a,b; Schon 2000) predominantly affect brain and        <br /> muscles, both types of tissue having a high energy re-  <br /> quirement. OXPHOS mtDNA defects induce myopa-      <br /> thies in progressive external ophthalmia and Kearns   <br /> Sayre syndrome (KSS), encephalomyopathies in mito-              <br /> chondrial encephalomyopathy lactic acidosis and stroke-            <br /> like episodes (MELAS) and myoclonus epilepsy with        <br /> ragged-red fibers (MERRF) syndromes, and neuropa-          <br /> thies in Leber&#226;&#8364;&#8482;s heredity optic neuropathy. OXPHOS          <br /> neurodegenerative disorders are also found in nuclear         <br /> gene defects. Mutations have been found in subunits   <br /> of complex I, including NADH binding site and in succi-         <br /> nate dehydrogenase (SDH complex II) in Leigh syn-             <br /> drome. COX deficiences, due to mutations in the Surf1       <br /> gene involved in COX assembly and maintenance, are      <br /> also found in Leigh syndrome. In Freidreich&#226;&#8364;&#8482;s ataxia,     <br /> deficiencies of complexes II and III due to mutations            <br />                                                             <br /> in the nuclear gene frataxin are observed. Dystonia,          <br /> parkinsonism, and psychiatric illness are observed in         <br /> Wilson&#226;&#8364;&#8482;s disease due to mutations in the P type ATPase.          <br /> In Huntington&#226;&#8364;&#8482;s disease there is loss of medium spiny              <br /> neurons. The defect is due to mutations in the hunting-        <br /> tin gene and results in deficiencies in complexes II, III,    <br /> and IV (COX). In Parkinson&#226;&#8364;&#8482;s disease, complex I defect        <br /> </body></html>