<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>YgbQ, cell division protein Escherichia coli <br /> and Vibrio cholerae, localizes codependent <br /> fashion FtsL division site <br /> Nienke Buddelmeijer, Nicholas Judson, Dana Boyd, John J. Mekalanos, Jonathan Beckwith* <br /> Department Microbiology Molecular Genetics, Harvard Medical School, 200 Longwood Avenue, Boston, MA 02115 <br />  <br /> Contributed Jonathan Beckwith, March 1, 2002 <br />  <br /> YgbQ cell division protein Escherichia coli Vibrio             E. coli genome proteins structural features similar <br /> cholerae. In E. coli ygbQ gene discovered result         FtsL yielded candidates, YgbQ <br /> computer search E. coli genome designed potential           protein promising. <br /> interacting partners cell division protein FtsL. In V. cholerae,         At time, Mekalanos laboratory working on <br /> ygbQ identified essential gene using transposon             unrelated project identify essential genes bacterium <br /> that fuses genes arabinose promoter. The role YgbQ cell        Vibrio cholerae. These studies led identification gene, <br /> division supported following. Cells depleted YgbQ          absence product led filamentation the <br /> both organisms form long filaments, DNA segregation             bacterium. A chance discussion hallway led discov- <br /> affected. YgbQ localizes constriction site wild-type E. coli     ery studying homologous ORFs organ- <br /> cells. Localization E. coli YgbQ constriction site depends       isms. Our combined studies, reported here, strongly indicate that <br /> on cell division proteins FtsQ FtsL FtsW FtsI, placing     product ygbQ gene E. coli homologue, <br /> YgbQ sequential dependency order proteins localizing          VC0527, V. cholerae required cell division process. <br /> the division site. Localization green fluorescent protein-FtsL     In addition, results consistent suggestion that <br /> depends YgbQ, indicating FtsL YgbQ colocalize           FtsL YgbQ interact colocalize septum. <br /> division site E. coli. Our results colocalization proteins <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> bacterial midcell E. coli raise possibility   Material Methods <br /> proteins interact coiled-coil structure.                              Bacterial Strains, Plasmids, Media. Bacterial strains plas- <br />                                                                            mids described Table 1. Growth rich medium, NZY <br />                                                                            (E. coli) LB (V. cholerae; ref. 5). When needed, ampicillin <br /> C    ell division bacteria takes place midcell occurs <br />      DNA duplicated segregated two <br /> daughter nucleoids. In Escherichia coli, process requires set <br />                                                                            200 &#226;?&#174;g&#229;&#8230;&#190;ml (plasmid) 25 &#226;?&#174;g&#229;&#8230;&#190;ml (chromosome), chloramphen- <br />                                                                            icol 30 &#226;?&#174;g&#229;&#8230;&#190;ml, kanamycin 40 &#226;?&#174;g&#229;&#8230;&#190;ml, spectinomycin 100 &#226;?&#174;g&#229;&#8230;&#190;ml, <br /> of proteins localize constriction site        streptomycin 500 &#226;?&#174;g&#229;&#8230;&#190;ml added. Medium supple- <br /> septum. These proteins coordinate invagination cell                 mented 0.1% (V. cholerae) 0.2% L-arabinose (E. coli) and <br /> membrane, inward growth peptidoglycan layer, and,                   0.2% D-glucose induce repress expression genes con- <br /> finally, separation daughter cells. The proteins, FtsZ,            trolled PBAD promoter, respectively. Standard techniques, <br /> FtsA, ZipA, FtsK, FtsQ, FtsL, FtsW, FtsI, FtsN,              PCR, electroporation, transformation, P1 transduction, <br /> identified largely genetic approaches. Because sys-             used cloning analysis DNA (6, 7). The enzymes <br /> tematic genetic approach identifying proteins            manipulating DNA New England Biolabs. <br /> performed devised, number thus-far- <br /> undiscovered proteins play role cell division. Informa-          Isolation Arabinose-Dependent V. cholerae. The introduction of <br /> tion available regarding function               TnAraOut V. cholerae screen insertion mutants <br /> proteins (reviewed refs. 1 2).                                      arabinose-dependent growth described <br />    Studies number laboratories led sequential          (8) following modification. NJ348 isolated by <br /> dependency model assembly group proteins             mating Tn10cm (pBSL181; ref. 9) N16961Sm generate a <br /> the cell septum (3). FtsZ arrives cell septum, providing      chloramphenicol-resistant V. cholerae reduce background <br /> a scaffold recruitment subsequent proteins. FtsA            conjugation pNJ17. NJ406 isolated strain <br /> ZipA localize septum independently other,               arabinose-dependent growth <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>phenotype</span> a <br /> each depends presence FtsZ localization. The                 TnAraOut insertion 66 bp upstream VC0527. This strain <br /> remaining proteins assemble midcell strictly sequential            forms tiny colonies LB plates fails grow LB broth <br /> dependency order follows: FtsK&#226;&#8364;&#8220;FtsQ&#226;&#8364;&#8220;FtsL&#226;&#8364;&#8220;FtsW&#226;&#8364;&#8220;FtsI&#226;&#8364;&#8220;                     absence arabinose. <br /> FtsN. The mechanisms order recruitment not <br /> understood.                                                                Construction ygbQ ygbQ-P-B-O Operon Under pBAD <br />    The work reported arose projects different               Promoter. The ygbQ gene amplified PCR chromo- <br /> goals Beckwith Mekalanos laboratories. The Beckwith             somal DNA wild-type E. coli strain using ygbQ5EcoRI <br /> laboratory studied cell division E. coli focusing atten-         (5&#226;&#172;&#732;-CCGGA AT TCTCGGATGCATGGGATGAT-3&#226;&#172;&#732;) and <br /> tion proteins, FtsQ, FtsL, FtsI. These proteins          ygbQ3XbaI (5&#226;&#172;&#732;-CGCTCTAGATGTTAATTCCCGGGCTGA- <br /> similar membrane topologies, i.e., short amino-terminal do-              3&#226;&#172;&#732;) primers. The PCR fragment digested EcoRI and <br /> main facing cytoplasm, single transmembrane segment, and <br /> a carboxyl-terminal domain located periplasm. FtsL a <br />                                                                            Abbreviations: GFP, green fluorescent protein; IPTG, isopropyl &#226;?&#164;-D-thiogalactoside. <br /> leucine zipper-like motif periplasmic domain, indicating <br /> that form coiled-coil homo- heterooligomers. It forms            *To reprint requests addressed at: Harvard Medical School, Department <br />                                                                             Microbiology Molecular Genetics, 200 Longwood Avenue, Room 212B, Boston, MA <br /> homodimers vitro likely weakly dimerizes vivo (ref.          02115. E-mail: jon_beckwith@hms.harvard.edu. <br /> 4 N.B. J.B., unpublished results). We suspected               The publication costs article defrayed page charge payment. This <br /> there partner protein forms heterodimers                article marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 U.S.C. <br /> higher order heterooligomers FtsL. A search               &#194;&#167;1734 solely indicate fact. <br />  <br />  <br />  <br /> 6316 &#226;&#8364;&#8220; 6321 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; April 30, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 9                                                         www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.092128499 <br />  Table 1. Strains plasmids <br /> Strain plasmid                               Relevant genetic marker(s) feature(s)                                  Source reference <br />  <br /> E. coli <br />   JOE309                 MC4100 araD&#226;&#171;&#185;                                                                                     3 <br />   NB974                  JOE309 &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIqpBAD18-ygbQ                                                         This study <br />   NB946                  JOE309 ygbQ&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIqpBAD18-ygbQ                                                This study <br />   NB805                  JOE309 &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQec                                                    This study <br />   NB802                  JOE309 &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQvc                                                    This study <br />   NB841                  JOE309 ftsL&#226;&#172;&#8249;TnPhoA (Kanr) &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQec&#229;&#8230;&#190;pLMG180                         This study, 5 <br />   NB836                  JOE309 ftsQ&#226;&#172;&#8249;TnPhoA50 (Kanr) &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQec&#229;&#8230;&#190;pJC10                         This study, 26 <br />   NB966                  JOE309 ftsI&#226;&#172;&#8249;TnPhoAI137&#226;&#338;&#172;IS50R (Kanr) &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQec&#229;&#8230;&#190;pDSW262               This study, 10 <br />   NB991                  JOE309 ftsW&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq P207-gfp-ygbQec&#229;&#8230;&#190;pDSW406                                   This study, 27 <br />   NB976                  JOE309 ygbQ&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq pBAD18-ygbQ att&#226;?&#190;80&#226;&#172;&#8249;pJC115(P207-gfp-ftsI)                  This study, 3 <br />   NB977                  JOE309 ygbQ&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq pBAD18-ygbQ att&#226;?&#190;80&#226;&#172;&#8249;pJC116(P207-gfp-ftsL)                  This study, 3 <br />   NB980                  JOE309 ygbQ&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq pBAD18-ygbQ att&#226;?&#190;80&#226;&#172;&#8249;pJC118(P207-gfp-ftsQ)                  This study, 3 <br />   NB1000                 JOE309 ygbQ&#226;&#172;&#8249;Kan &#226;&#338;&#172;(&#226;?&#173;attL-lom)&#226;&#172;&#8249;bla lacIq pBAD18-ygbQ P209-gfp-ftsW                                 This study <br /> V. cholerae <br />   N16961Sm               El Tor                                                                                           28 <br />   NJ348                  N16961Sm Tn10cm insertion pBSL181                                                           This study <br />   NJ406                  YgbQ&#226;&#172;&#8249;TnAraOut                                                                                    This study <br />   NJ600                  NJ406&#229;&#8230;&#190;pNJ25                                                                                      This study <br /> Plasmids <br />   pNB10                  pBAD18-ygbQ                                                                                      This study, 29 <br />   pNB11                  pBAD18-ygbQPBO                                                                                   This study, 29 <br />   pNB12                  pBAD33-ygbQPBO                                                                                   This study, 29 <br />    pNJ17                 TnAraOut delivery vehicle                                                                        8 <br />    pNJ25                 pDSW207-ygbQvc                                                                                   This study, 10 <br />    pBSL181               Tn10cm                                                                                           9 <br />  <br />  <br />  <br /> XbaI ligated sites pBAD18, resulting      gfp-ftsQ, gfp-ftsL, gfp-ftsI fusion strains att site the <br /> plasmid named pNB10 (restriction sites underlined       lambdoid phage &#226;?&#190;80 described Chen Beckwith (3). <br /> primer sequences). The complete ygbQ-P-B-O operon am- <br /> plified PCR chromosomal DNA using ygbQ5EcoRI                   Construction ygbQ Null Strain E. coli. To disrupt chro- <br /> (5&#226;&#172;&#732;-CCGGA AT TCTCGGATGCATGGGATGAT-3&#226;&#172;&#732;)                              mosomal ygbQ gene recombined PCR product chro- <br /> ygbO3XbaI (5&#226;&#172;&#732;-CGCTCTAGACAT TCACGCCA AGATA-                             mosomal DNA (12). A DNA fragment amplified PCR <br /> GACG-3&#226;&#172;&#732;) primers. The PCR fragment digested                using pKD4 (12) template DNA H1P1ygbQ (5&#226;&#172;&#732;-GGATG- <br /> EcoRI XbaI ligated sites pBAD18,          TAGAGTCGTCTTCGGATGCATGGGATGATGATGCC- <br /> the resulting plasmid named pNB11. The ygbQ-P-B-O genes            GT T T T TCAGGTGTAGGCTGGAGCTGCT TC-3&#226;&#172;&#732;) and <br /> were recloned pBAD18 chloramphenicol vector con-             P2H2Q (5&#226;&#172;&#732;-GCGCA A ACATCCA A ATGAGTGGT TGC- <br /> struct ygbQ depletion strain (see below). pNB11 digested         CATGT TA AT TCCCGGGCTGAT T TACATATGA ATA- <br /> with NheI SalI. The 2.8-kb fragment containing ygbQ-           TCCTCCTTAG-3&#226;&#172;&#732;) primers. The resulting fragment was <br /> P-B O genes ligated NheI- SalI-digested            composed kanamycin-resistance gene extensions that <br /> pBAD18 chloramphenicol vector, resulting plasmid           homologous regions adjacent end ygbQ <br /> named pNB12.                                                           gene (underlined sequences). This PCR fragment trans- <br />                                                                        formed competent cells strain NB885 carrying plasmids <br /> Construction Strains Containing Green Fluorescent Protein (GFP)     pKD46 (12) pNB12. Cells selected growth on <br /> Fusions. ygbQ E. coli amplified PCR using pNB10           kanamycin presence L-arabinose. Both phage &#226;?&#173; red <br /> template ygbQ5EcoRIgfp (5&#226;&#172;&#732;-CGAGAATTCAACAA-                         recombinase gene pKD46 ygbQ-P-B O genes on <br /> CA ACATGGGTA A ACTA ACGCTGCTGT TGC-3&#226;&#172;&#732;)                             pNB12 control pBAD promoter. The replace- <br /> ygbQ3HdIIIgfp (5&#226;&#172;&#732;-TCGAAGCTTTTATCGATTGTTTTGC-                           ment ygbQ kanamycin confirmed PCR using <br /> CCCGCAGAC-3&#226;&#172;&#732;) primers. The amplified product di-                kanamycin-, ygbE-, ygbP-specific primers. Kanamycin- <br /> gested EcoRI HindIII, ligated sites          resistant colonies tested growth ampicillin, test for <br /> pDSW207 (10), named pNB20. The GFP-YgbQ fusion                     loss pKD46, L-arabinose D-glucose test for <br /> protein linker sequence YKEFNNM, Y K             depletion ygbQ. Strains disrupted ygbQ gene grew on <br /> the residues GFP, M residue YgbQ. ygbQ     D-glucose plates small colonies, likely basal <br /> of V. cholerae amplified PCR chromosomal DNA               expression pNB12. The ygbQ::Km transduced a <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> N16961 using primers ygbQF (5&#226;&#172;&#732;-GCCGAATTCAT-                   strain carried extra chromosomal copy ygbQ under <br /> GCGAGTATTTGCTTTGACC-3&#226;&#172;&#732;) ygbQR (5&#226;&#172;&#732;-GGCTG-                           pBAD &#226;?&#173;att site complementation avoid residual <br /> CAGTTGGCGCGACTCCTCACCGAT-3&#226;&#172;&#732;). The amplified                            expression ygbQ-P-B O pNB12 (NB946). <br /> product digested EcoRI HindIII ligated into <br />                                                                                                                                                  MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> the sites pDSW207 (10). The resulting plasmid              Growth Conditions GFP-Fusion Protein Localization. For localiza- <br /> named pNJ25.                                                           tion GFP fusions wild-type background, cells grown <br />   Stable introduction gfp-ygbQ fusions gfp-ftsW        overnight 30&#194;&#176;C NZY medium plus 25 &#226;?&#174;g&#229;&#8230;&#190;ml ampicillin, <br /> the E. coli chromosome &#226;?&#173; attachment site obtained          diluted 1:200 fresh medium ampicillin, grown to <br /> with InCh integration vector (11). The construction             optical density 600 nm (OD600) 0.5. The cultures were <br />  <br /> Buddelmeijer et al.                                                                           PNAS &#229;&#8230;&#169; April 30, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 9 &#229;&#8230;&#169; 6317 <br />  diluted medium containing 2.5 &#226;?&#174;M (gfp-ftsI), 5 &#226;?&#174;M (gfp-ftsQ), <br /> or 10 &#226;?&#174;M (gfp-ftsL gfp-ygbQ) isopropyl &#226;?&#164;-D-thiogalactoside <br /> (IPTG). Cells kept logarithmic growth phase serial <br /> dilutions medium grown 4 h. Cells fixed <br /> directly growth medium processed microscopy as <br /> described below. For experiments involving depletion strains, <br /> the overnight culture grown 30&#194;&#176;C NZY medium <br /> containing chloramphenicol, kanamycin, 0.2% L-arabinose. <br /> The culture diluted 1:200 medium grown to <br /> an OD600 0.5. After washing cells medium        Fig. 1. Phenotype ygbQ depletion strain V. cholerae. (A) Image cells <br /> sugar, cells resuspended medium containing L-              grown arabinose express ygbQ. (B) Image cells grown glucose <br /> arabinose D-glucose IPTG. Cells kept logarithmic        deplete YgbQ. (Scale bar, 10 &#226;?&#174;m.) <br /> growth phase serial dilutions medium grown <br /> for hours. Cells fixed filaments substantial <br /> <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> observed. To study localization GFP-YgbQ         genes downstream ygbQ, ygbP, ygbB essential (17). To <br /> an FtsQ, FtsL, FtsW, FtsI depletion strain, gfp-ygbQ            determine inability grow absence arabinose was <br /> induced 10 &#226;?&#174;M IPTG strains. Strains study           caused ygbQ downstream genes, plasmid containing <br /> localization GFP-FtsQ, GFP-FtsL, GFP-FtsW, GFP-FtsI             ygbQ, pNJ25, constructed. NJ406 comple- <br /> in ygbQ depletion strain constructed using gfp fusions       mented IPTG-inducible expression ygbQ pNJ25, <br /> integrated phage &#226;?&#190;80 att site (3). The cultures grown      indicating gene complement filamen- <br /> in presence 2.5 &#226;?&#174;M (gfp-ftsI), 5 &#226;?&#174;M (gfp-ftsQ), 10 &#226;?&#174;M           tation phenotype. A discussion cell division phenotypes <br /> (gfp-ftsL), 1 mM (gfp-ftsW) IPTG.                                   N.J. J.B. led realization studying <br />                                                                        gene. <br /> Fluorescence Microscopy. Cells fixed mixture para- <br /> formaldehyde glutaraldehyde microscopic analysis         A Computer Search Potential Interacting Partners FtsL E. coli. <br /> described (13). After fixation, cells stained 0.2 &#226;?&#174;g&#229;&#8230;&#190;ml      The unique features cell division protein FtsL fact that <br /> 4&#226;&#172;&#732;,6-diamidino-2-phenylindole PBS 10 min room tem-           FtsL homodimers weak exist temporarily <br /> perature, washed twice PBS, immobilized agarose              vivo led Beckwith lab look potential partner proteins <br /> slides described (14). To examine cells used Orca 12-bit      interacted FtsL. We sought protein share <br /> cooled charge-coupled device camera (Hamamatsu Photonics,              features structure membrane topology of <br /> Hamamatsu City, Japan) mounted Axioskop 2 microscope             FtsL. A search developed using entire E. <br /> (Zeiss). Images taken using Zeiss AXIOVISION software          coli chromosome look proteins (i) unknown <br /> and analyzed public domain program OBJECTIMAGE 1.62p          function, (ii) contained leucine zipper-like motif predicted to <br /> by Norbert Vischer (University Amsterdam, Amsterdam,                form coiled-coil structure, (iii) predicted be <br /> http:&#229;&#8230;&#190;&#229;&#8230;&#190;simon.bio.uva.nl&#229;&#8230;&#190;object-image.html). Images pro-           integral cytoplasmic membrane proteins. We such <br /> cessed Adobe PHOTOSHOP presentation.                            proteins including putative tyrosine kinase, proteins with <br />                                                                        considerably higher molecular weight FtsL, small <br /> Preparation Membranes Detection FtsL Western Blotting.    proteins. <br /> Membranes prepared sonicating cells 100 mM                     The small proteins YqhH (SwissProt accession no. <br /> Tris&#228;&#161;&#160;HCl, pH 8.0, containing 1 mM PMSF. Whole cells               Q46860), outer membrane lipoprotein precursor, YgbQ <br /> removed centrifugation 5,000 rpm 5 min. Membranes            (Q46894), hypothetical protein conserved bacterial <br /> were pelleted 100,000 &#226;&#171;&#187; g Beckman TL100 ultracentrifuge        species shows weak homology (20 59 residues are <br /> for 60 min resuspended 100 mM Tris&#228;&#161;&#160;HCl, pH 8.0. A total         identical) protein Bacillus halodurans. The B. halo- <br /> of 10 &#226;?&#174;g membranes applied lane SDS&#229;&#8230;&#190;PAGE.                durans protein itself homologue cell division initiator <br /> Electrophoresis immunoblotting performed de-               protein DivIC Bacillus subtilis. (Homology searches did not <br /> scribed Laemmli (15) Towbin et al. (16), respectively. The      YgbQ direct homolog B. subtilis DivIC.) In <br /> membrane probed FtsL-specific polyclonal antibodies           B. subtilis, FtsL DivIC function complex cell <br /> and subsequently horseradish peroxidase-conjugated goat           division (18). <br /> anti-rabbit antibodies developed chemiluminescence               YgbQ composed 103 aa residues (11 kDa) is <br /> reagents (Amersham Pharmacia).                                         predicted bitopic membrane protein. A three-residue <br />                                                                        amino-terminal domain predicted face cytoplasm, <br /> Results                                                                residues L4&#226;&#8364;&#8220;G22 resembles transmembrane segment, the <br /> Identification Characterization ygbQ V. cholerae. The        presumed periplasmic region, containing L46&#226;&#8364;&#8220;L67, leucine <br /> mariner-based transposon, TnAraOut, causes genes tran-           zipper-like motif predicted form single coiled coil <br /> scriptionally fused arabinose-inducible promoter PBAD           (Fig. 2). A hybrid protein alkaline phosphatase was <br /> after transposition genome (8). Essential genes          fused carboxyl terminus YgbQ resulted dark blue <br /> identified, strains TnAraOut insertions fused          colonies 5-bromo-5-chloro-3-indolylphosphate (XP) indica- <br /> essential genes depend presence arabinose growth.        tor plates, suggesting carboxyl terminus protein <br /> We TnAraOut inserted (strain NJ406) V. cholerae 66          facing periplasm (data shown; ref. 19). This set of <br /> bp upstream predicted translational start VC0527        structural features resembles FtsL. The gene gene <br /> depended arabinose growth. The gene VC0527              cluster, including downstream genes ygbP, ygbB, ygbO, <br /> similar E. coli gene ygbQ (and referred       Gram-negative Gram-positive bacteria. Both <br /> to ygbQ).                                                           ygbP ygbB essential genes involved pathway for <br />   Growth strain NJ406 LB broth presence                isoprenoid biosynthesis E. coli (17). <br /> arabinose showed wild-type growth phenotype. With cells                 Because Bacillus information role homol- <br /> grown absence arabinose, long filaments V. cholerae       ogous proteins YgbQ closely analogous <br /> formed (Fig. 1). The genomic organization region V.         domain structure FtsL, considered interesting <br /> cholerae E. coli similar. In E. coli known   candidate FtsL-interacting protein. We initiated studies to <br />  <br /> 6318 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.092128499                                                                                Buddelmeijer et al. <br />                                                                                      ygbQ affect ygbQ expression the <br />                                                                                     downstream genes ygbP, ygbB, ygbO. However, we <br />                                                                                     used (12) supposed allow transcription genes downstream <br />                                                                                     disrupted gene. <br />                                                                                        The strain carrying ygbQ deletion carried addi- <br />                                                                                     tional copy ygbQ PBAD control bacteriophage &#226;?&#173;att <br />                                                                                     site. This depletion strain able grow medium supple- <br />                                                                                     mented arabinose glucose. The cells form <br />                                                                                     extensive filaments YgbQ depleted (see Fig. 4). 4&#226;&#172;&#732;,6- <br />                                                                                     Diamidino-2-phenylindole staining indicated nucleoids <br /> Fig. 2. CLUSTALW 1.81 alignment YgbQ E. coli (Ec) V. cholerae (Vc). The   filaments spaced regularly; thus, DNA segregation not <br /> leucine residues form leucine zipper-like motif periplasmic         affected ygbQ null strain (data shown). A plasmid <br /> domain YgbQ E. coli indicated arrows. The overlined regions          expressing gfp-ygbQ complements ygbQ deletion strain for <br /> represent putative transmembrane (TM) domain (residues 4 &#226;&#8364;&#8220;22)           growth. Depletion ygbQ strain expressing ygbP, B, O <br /> coiled-coil domain (residues 29 &#226;&#8364;&#8220; 67) YgbQ. Conserved amino acid residues        different promoter results filamentous phenotype, <br /> are indicated asterisks. <br />                                                                                     indicates ygbQ null, expression ygbP, B, <br />                                                                                     O genes sufficient growth (data shown). Fur- <br /> inactivate ygbQ gene characterize phenotype                      thermore, ygbP ygbB depletion strains described by <br /> null <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span>. At point discovered labora-                     Campos et al. (17) depletion phenotype different the <br /> tories working homologous genes (50% identity) E.                        ygbQ depletion strain; stop growing immediately the <br />                                                                                     complementing pathway inactivated, cells filament. <br /> coli V. cholerae occupying chromosomal position in <br />                                                                                     These results that, V. cholerae, YgbQ E. coli is <br /> the organisms. The V. cholerae protein, having a <br />                                                                                     essential, depletion results inhibition cell division. <br /> strict leucine-zipper motif, predicted form coiled-coil <br /> structure COILS 2.2 program (20). Given inacti- <br />                                                                                     Localization YgbQ Depends FtsQ FtsL Not FtsW and <br /> vation V. cholerae gene led filamentous phenotype, we <br />                                                                                     FtsI. The results studies V. cholerae E. coli <br /> were encouraged pursue study E. coli gene. <br />                                                                                     YgbQ homologues strongly suggested YgbQ plays im- <br /> While proceeding construct ygbQ null mutation, decided <br />                                                                                     portant role cell division. We wished determine position <br /> also determine immediately YgbQ localized the <br />                                                                                     YgbQ process assembly cell division proteins at <br /> cell septum.                                                                        midcell. To make determination studied localization <br />                                                                                     GFP-YgbQ mutants depleted cell division <br /> E. coli YgbQ Localizes Division Site Wild-Type Background. <br />                                                                                     proteins appeared late pathway, FtsQ, FtsL, FtsW, <br /> All known E. coli cell division proteins localize constric-                  FtsI, discovery YgbQ E. coli based on <br /> tion site; unrelated membrane proteins examined                      hunt partner FtsL. Depletion strains cell division <br /> do (see e.g. refs. 4, 10, 13). Thus, localization protein              proteins contained null mutation protein be <br /> to midcell provides strong suggestion protein involved                    depleted plasmid expressing cell division protein <br /> in cell division. We asked YgbQ localized cell                       PBAD promoter. The cells grow normally presence <br /> septum. A gene fusion expressed YgbQ GFP                           arabinose, shifted glucose-containing medium, <br /> amino terminus constructed inserted chromo-                        cells eventually developed long filaments. As cells are <br /> some single copy &#226;?&#173;InCh (11). The strains merodiploid                     depleted glucose medium, fix prepare them <br /> for ygbQ, i.e., wild-type copy ygbQ present normal                   exhibiting logarithmic growth. <br /> chromosomal location, gfp-ygbQ copy located &#226;?&#173;                          The cells depletion strains FtsQ, FtsL, FtsW, FtsI <br /> attachment site expressed regulatable trc promoter.                        grown arabinose normal length, GFP-YgbQ is <br /> The strains grown different concentrations IPTG                     septum (data shown). When cells are <br /> the media induce different levels expression GFP-                      grown glucose-containing medium, formed filaments. <br /> YgbQ fusion protein. Using fluorescent microscopy, looked                        GFP-YgbQ localized septal region filaments the <br /> for localization GFP-YgbQ midcell IPTG con-                          FtsW FtsI depletion strains FtsQ FtsL <br /> centrations ranging 10 50 &#226;?&#174;M. Most cells exhibited                      depletion strains (Fig. 3). As positive negative controls, we <br /> a bright band fluorescence midpoints, indicating                   showed that, previously (13, 21), GFP-FtsQ localizes to <br /> GFP-YgbQ localizes division site (data shown).                           potential division sites FtsL depletion filaments, GFP-FtsI <br /> GFP-FtsL localization region detected previ-                       does (data shown). These results suggest (i) <br /> ously 10 &#226;?&#174;M inducer used induce GFP-FtsL                           YgbQ localizes cell septum FtsL FtsW or <br /> fusion protein.                                                                     (ii) YgbQ colocalizes septum FtsL. <br />    We looked localization YgbQ V. cholerae <br /> (YgbQvc) E. coli using GFP-fusion protein analogous                      Localization FtsQ, FtsL, FtsW, FtsI YgbQ Depletion Strain. <br /> the GFP-YgbQ E. coli. We observed fluorescent signal                      To clarify position YgbQ septal assembly sequence <br /> expression gfp-ygbQvc induced 50 &#226;?&#174;M IPTG com-                           further, studied localization GFP-FtsL ygbQ null <br /> pared 10 &#226;?&#174;M GFP-YgbQec; 50 &#226;?&#174;M IPTG,                          background using GFP-FtsQ, GFP-FtsW, GFP-FtsI as <br /> fluorescent signal intense GFP-YgbQ E.                        likely positive negative controls. Because complementing <br /> coli (data shown). Perhaps higher induction levels                       copy pBAD-ygbQ integrated &#226;?&#173;att site YgbQ <br /> GFP-YgbQvc necessary observe septal fluorescence                             depletion strain, integrated GFP-fusion constructs into <br /> because nonnative protein competes wild-type                     chromosome att site lambdoid phage &#226;?&#190;80 (3). <br /> YgbQ E. coli localization midcell.                                        Strains grown NZ medium supplemented arabinose <br />                                                                                                                                                              MICROBIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                     glucose 30&#194;&#176;C, expression gfp-ftsQ, gfp-ftsL, gfp-ftsI, <br /> Depletion ygbQ E. coli Is Correlated Filamentation. A YgbQ               gfp-ftsW induced 5 &#226;?&#174;M, 10 &#226;?&#174;M, 2.5 &#226;?&#174;M, 1 mM <br /> depletion strain E. coli constructed chromosomal                          IPTG, respectively. In cells grown arabinose, observed a <br /> disruption ygbQ (12). Because ygbQ cluster                     fluorescent signal GFP-FtsQ, GFP-FtsL, GFP-FtsW, and <br /> essential genes involved isoprenoid synthesis, disruption                     GFP-FtsI cell midpoint (Fig. 4). In filaments formed after <br />  <br /> Buddelmeijer et al.                                                                                        PNAS &#229;&#8230;&#169; April 30, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169; no. 9 &#229;&#8230;&#169; 6319 <br />  Fig. 3. Localization GFP fusions YgbQ FtsQ (A), FtsL (B), FtsW (C), and <br /> FtsI (D) depletion strains. (Scale bars, 10 &#226;?&#174;m.) <br />  <br />  <br />  <br /> depletion YgbQ, obtained results consistent order <br /> predicted above; GFP-FtsQ localizes septum, GFP- <br /> FtsW GFP-FtsI not. However, observed localiza- <br /> tion GFP-FtsL midcell YgbQ-depleted filaments (Fig. 4). <br /> Because YgbQ itself septum FtsL-depleted <br /> cells, results suggest FtsL YgbQ localize the <br /> division site codependent fashion. <br />    Results B. subtilis suggest YgbQ analogue, DivIC, <br /> stabilizes FtsL. We asked depletion YgbQ might <br /> affect FtsL stability E. coli assayed FtsL protein levels in <br /> a YgbQ depletion strain. We detected FtsL YgbQ depletion <br /> strain grown arabinose cells grown glucose (Fig. <br /> 5). The likely instability FtsL conditions suggests                 Fig. 4. Localization GFP fusions FtsQ (Q), FtsL (L), FtsI (I) ygbQec <br /> that YgbQ stabilizes FtsL, indicating FtsL&#226;&#8364;&#8220;YgbQ interaction.                    depletion strain. (A) Localization GFP-fusion proteins cells grown with <br />                                                                                    arabinose express ygbQ. (B) Localization GFP-fusion proteins cells <br /> Discussion                                                                         grown glucose deplete ygbQ. (Scale bars, 10 &#226;?&#174;m.) <br /> In article, report characterization gene, ygbQ, <br /> encoding protein cell division pathway E. coli V. <br />                                                                                    act (18). Localization Bacillus proteins been <br /> cholerae. In organisms, depletion YgbQ leads a <br /> filamentous phenotype typical <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>null mutants</span> cell                        examined. <br /> division genes. Furthermore, YgbQ localizes midcell E.                      Our studies position YgbQ protein assembly <br /> coli, feature known far specific                       pathway E. coli septum consistent an <br /> proteins directly involved cell division process. Finally,                  interaction FtsL YgbQ. YgbQ depends FtsQ not <br /> studies localization YgbQ assign specific place                  FtsW localization septum, does FtsL. But neither <br /> the septal assembly proteins required cell division                   proteins, YgbQ FtsL, seen septum in <br /> and consistent interaction FtsL.                        absence other. These results consistent a <br />    We detected ygbQ V. cholerae screen essential genes.                model FtsL YgbQ colocalize septum. Such <br /> In E. coli, focused ygbQ searching genome                      codependence explained solel <br /> </body></html>