<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1984 Genetics Society America <br />  <br />  <br />  <br />  <br />            MUTATIONS AFFECTING FUNCTIONS OF T H E <br />                   DROSOPHILA GENE GLUED <br />  <br />            ALAN GAREN, BARBARA R. MILLER AND M. LUISA PACO-LARSON <br />   Department Molecular Biophysics Biochemistry, Yale University, New Haven, Connecticut 0651 1 <br />                                 Manuscript received December 5 , 1983 <br />                                 Revised copy accepted April 14, 1984 <br />  <br />                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />  <br />              Glued mutations Drosophila comprise essential complementation group <br />           complex developmental effects. The original Glued <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> (GI) has <br />           dominant nonlethal effects heterozygous flies, principally morphogen- <br />           esis visual system. GI recessive lethal effect early development. <br />           Mutations reverse dominant visual effects GI (GIR mutations) were <br />           induced y-radiation insertions transposable P element. The GIR(&apos;&apos; <br />           mutations induced y-radiation reverse lethal effect GI; these <br />           appear null mutations, (and possibly all) delete segments of <br />           Glued region. The            mutations induced insertion P element <br />           reverse concomitantly recessive lethal effect GI, suggesting both <br />           recessive dominant effects controlled gene. The reversal <br />           lethal effect G1 P element remarkable, indicates an <br />           essential gene function restored insertion unrelated DNA. Another <br />           class lethal Glued mutations induced normal G1+ strain ethyl <br />           methanesulfonate (EMS). The EMS mutations belong essential <br />           complementation group GI, strong dominant effects G1 <br />           visual system.-The         GIR(- mutations provide molecular marker for <br />           Glued gene, used map gene 70C2 band chromosome <br />           3L situ hybridization P element probe polytene chromosomes <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />           GI&quot; strains isolate clones Glued genomic DNA molecular <br />           studies normal gene various Glued mutations. <br />  <br />      HE Drosophila mutation Glued (GL) principal effects devel- <br /> T      opment organism. One dominant effect heterozygous GL/GL+ <br /> flies development visual system, causing severe morphological and <br /> functional abnormalities eye neural connections optic lobe <br /> in brain and, also, bristles (PLOUGH IVES 1935; MEYEROWITZ      and <br /> KANKEL1978; GARENand KANKEL1983). The recessive lethal effect <br /> on development homozygous GL/G1 individuals, die late embry- <br /> ogenesis, apparently general defect cell proliferation differ- <br /> entiation (HARTEand KANKEL1982). GL and, long time. the <br /> only Glued mutation identified, recently additional classes Glued <br /> mutations described (HARTE      KANKEL1982; report). One class <br /> causes reversal dominant visual effects GI the <br /> recessive lethal effects. T h e class comprises recessive lethal mutations, <br /> induced normal GI+ strain EMS, complement lethal <br /> Genetics 107: 645-655 August, 1984. <br />  646                A. GAREN, B. R. MILLER AND M. L. PACO-LARSON <br />  <br /> effects G1; lethal mutations, contrast GI, strong <br /> dominant effects visual bristles. All lethal Glued mutations <br /> have assigned complementation group, defines Glued <br /> gene essential functional <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. T h e dominant effects G1 visual <br /> system involve gene separate gene, extent the <br /> genetic defect GI known. In report properties of <br /> various Glued mutations, including new class induced insertion the <br /> transposable P element (RUBINand SPRADLINC          1982) Glued region of <br /> the genome. T h e insertions reverse recessive lethal effect G1 addition <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> dominant effects visual system, suggesting dominant and <br /> recessive effects involve gene. <br />  <br />                                  MATERIALS AND METHODS <br />  <br />    Nomenclature: The gene called Glued original mutation GI dominant effect on <br /> the eye, causing facets appear glued irregular array (PLOUGH      IVES 1935; <br /> MEYEROWITZand KANKEL1978; GARENand KANKEL 1983); GI recessive lethal effect <br /> during early development. The standard Glued gene designated G1+, different GI+ alleles <br /> could occur various standard strains Drosophila melanogaster. Mutations GI reverse <br /> the dominant eye <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, resulting normal nearly normal external eye morphology, are <br /> designated GIR(&apos;&quot; induced 7-radiation, GIR@)when induced EMS CIR(Owhen induced <br /> by transposable P element; independently induced GIR mutation assigned identifying <br /> numeral, e.g., R I , R2 forth. The notation       used previously (HARTEand KANKEL1982) <br /> but changed report mutations restore normal Glued functions. <br /> Recessive lethal Glued mutations induced standard G1+ strain EMS designated GI&quot;; each <br /> independently <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>induced mutation</span> assigned identifying numeral, e.g., 11, 12 forth. Six of <br /> the GI&quot; mutations isolated study described (HARTEand KANKEL1982) <br /> using different nomenclature, follows: 1-3, 7-1, 7-6, 5-5, 7-E 15-C, correspond 11 <br /> through 16. <br />    Strains: All strains derived D. melanogaster. The mutation G1 induced heat <br /> treatment unspecified strain (PLOUGH      IVES 1935); scarlet (st) marker introduced <br /> by recombination, resulting G1 st chromosome 3 balanced T M 3 Sb Ser (LINDSLEY <br /> and GRELL1968). This G1 st/TM3 Sb Ser strain used induction GIR reversions for <br /> the complementation tests involving G1. For induction lethal Glued mutations EMS, a <br /> multiply marked GI+ strain produced introducing chromosome 3 markers st red ca into <br /> an inbred strain Oregon-R. <br />    GIR(G) mutations: Males balanced G1 st/TM3 Sb Ser strain exposed 5000 r of <br /> 7-radiation Cow source mated T M l I T M 3 Ser females (HARTEand KANKEL <br />  1982). Since T M l / T M 3 Sb Ser progeny die, adults morphologically normal eyes should <br /> be revertants G1. A total 96 male revertants IO6 adult progeny <br /> examined, estimated counting number flies vials multiplying average by <br /> the total number vials screened. Each male revertant crossed individually DTS-4ITM3 Sb <br /> Ser females (DTS-4 dominant temperature-sensitive lethal mutation chromosome 3), the <br /> developing organisms kept 25 restrictive temperature DTS. The progeny <br />                                      O, <br />  <br />  <br /> of cross used prepare GIR(&quot; stlTM3 Sb Ser stock. <br />    GIR@&apos;mutations: The EMS-induced reversions G1 isolated HARTEand KANKEL <br /> (1982) provided stocks study. <br />    GIR&quot;) mutations: Males wild-type strains containing transposable P element (BRE- <br /> GLIANO KIDWELL 1982), namely, Cranston, Harwich Kerbinion (obtained MARGARET <br /> KIDWELL),   crossed GI st/TM3 Sb Ser females, male progeny showing Glued eye <br /> phenotype crossed homozygous Gl+st/Gl+st females. Revertants G1 induced the <br /> P element develop adults morphologically normal scarlet eyes. A total 18 <br /> revertants 20,000 progeny examined. The male revertants individually <br />                               GLUED MUTATIONS IN DROSOPHILA                                        647 <br /> crossed G1 stlTM3 Sb Ser females, male progeny wild-type eyes collected. <br /> Since progeny mixture GI+ stlTM3 Sb Ser GIR&quot; st/TM? Sb Ser genotypes, individual <br /> males crossed GI stlTM3 Sb Ser females, order distinguish lethal GIR(flst/G1 <br /> st nonlethal GI+ st/G1 st combinations; crosses did produce progeny a <br /> Glued scarlet eye phenotype used establish GIR(* st/TM? Sb Ser stock. Note cross <br /> selects GIR(flmutations lethal heterozygous combination GI. <br />    EMS-induced lethal mutations Glued region: Young GI+ males inbred stock containing <br /> the chromosome 3 markers st red ca fed solution containing 5% sucrose 200-300 pg/ml <br /> of stock solution EMS (as supplied Eastman Kodak Company, Rochester, New York) for <br /> about 20 hr. Groups mutagenized males crossed DTS-4/TM3 Ser females. <br /> The progeny grown 25&quot; eliminate carrying DTS-4 chromosome, male <br /> progeny group individually tested mutations lethal trans combination <br /> with deletion Dj(3)G1+R2,which deletes 70C-70F region chromosome ?L (HARTEand <br /> KANKEL1982). The test crosses follows. red st ca/TM? Ser X Dj(3)GI+R9st/TM? Ser. If <br /> no progeny scarlet eyes found, indicating presence lethal mutation the <br /> mutagenized chromosome, st red ca/TM? Ser stock prepared. Only lethal mutation was <br /> isolated group individual males tested; frequency lethal mutations about <br /> one 50 groups tested. Each lethal mutation tested complementation lethal effect <br /> of G1; failed complement assigned complementation group 1 (see Table 2). <br />  <br />                                               RESULTS <br />  <br />    T h e GIR mutations study induced treating heterozygous GI <br /> males ?-radiation (GZR(G))  EMS (GZR(E))  dysgenic cross hetero- <br /> zygous GZ females males carrying P element transposon (GIR(p)),   the <br /> mutations identified normal, nearly normal, eye phenotype of <br /> GZR/GZ+flies (Table 1). Although dominant effects GI on <br /> the eye reversed, GIR mutations restores completely normal <br /> Glued function, indicated lethality GZR/GZheterozygotes compared <br /> to viability GI+/GZ heterozygotes (Table 1). <br />    All GZR mutations tested complementation recessive lethal <br /> mutations 11 essential complementation groups flanking Glued (Tables 2 <br /> and 3). Some GIR mutations tested complementation the <br /> flanking nonlethal mutations frizzled ( f )and Dichaete (Di)(HARTEand KANKEL <br /> 1982; Table 3). Of 96 GIR(&apos;) mutations tested, 22 failed complement <br /> the lethal mutations flanking groups, indicating the <br /> effects mutations extend limits Glued. Some these <br /> mutations shown cytogenetic molecular methods result from <br /> deletions segments 70C-70F region chromosome 3L, Glued <br /> and flanking groups map ( M . L . PACO-LARSON, M . LEVINE, B. R. MILLER <br /> and A. GAREN, unpublished results). T h e remaining 74 GZR(&apos;) mutations, and <br /> also GIR@&apos;and GIR(&apos;&apos; mutations, complemented flanking <br /> groups, suggesting reversions affect Glued function only. <br />    Reversal dominant phenotypic effects GI GIR mutations could <br /> occur restoring normal Glued function inactivating Glued <br /> gene, hemizygous individuals Glued region homolog <br /> is deleted appear develop normally (HARTEand KANKEL1982; Table 1). <br /> T h e mechanism, second, reverse recessive lethal <br /> effect GI. To distinguish mechanisms, GIR mutation was <br /> tested lethal effect hemizygous combination deletion. All the <br />  648                     A. GAREN, B. R. MILLER AND M. L. PACO-LARSON <br />  <br />                                                TABLE 1 <br />                                      Classes Glued mutations <br />  <br />                                                                   Effect organism <br />                     Original      Mutagenic <br />                    Glued gene     treatment                Genotype                    Phenotype <br />                                                          Gl+/Gl+                   Normal adults <br />                                                          Gl+/deletion              Normal adults <br />  <br />                       GI+        Heat                    Gl/Gl+                    Abnormal adults <br />                                                          GlIG1                     Lethal <br />                                                          Gl/deletion               Lethal <br />  <br />                       GI         y-Radiation             G1R(G)/G1+                Normal adults <br />                                                          GIRfG)/G1                 Lethal <br />                                                          GIRfG)/deletion           Lethal <br />  <br />                       GI         EMS                     GIRfE)/GI+                Normal adults <br />                                                          GlR&quot;&apos;/Gl                  Lethal <br />                                                          GIRfE)/deletion           Lethal <br />  <br />                       G1         P element               ClR(&apos;)/G1+                Normal adults <br />                                    insertion             GIRfP)/G1                 Lethal <br />                                                          GlRfP)/deletion           Normal adults <br />  <br />                       GI+        EMS                     G1&apos;(E)/GI+                Normal adults <br />                                                          GlifE)/G1                 Lethal <br />                                                          G1i(E)/deletion           Lethal <br /> ~             ~~ <br />  <br />  <br />  <br />  <br />   The various heterozygous combinations Glued mutations generated crosses between <br /> parental strains contained indicated Glued mutation recessive eye color marker <br /> scarlet (st) chromosome 3 homolog balancer TM3 Ser TM3 Sb Ser other <br /> homolog. The Glued heterozygotes scored phenotypically lethal scarlet eye adults <br /> were 100 adult progeny examined. When scarlet eye progeny did appear, <br /> these examined dissecting microscope mainly external abnormalities eyes <br /> characteristic Gl/G1+ heterozygotes; adults scored phenotypically normal such <br /> eye abnormalities detected. The deletion used produce hemizygous combinations was <br /> Df(3)G1+R2isolated HARTEand KANKEL,       deletes region 70C-70F chromosome 3L <br /> including entire Glued gene (HARTEand KANKEL1982; M. LEVINE,M. L. PACO-LARSON              and <br /> A. GAREN,unpublished results). The origin various Glued mutations described the <br /> MATERIALS AND METHODS. <br />  <br />  <br />  <br />     GIR(&apos;) GIR(E)mutations caused early lethality. The G P P )mutations, contrast <br />     others, supported development viable apparently normal flies <br />     (Table 4), indicating mutations restore essential Glued function; <br />     however, GIR(&apos;) mutations restore completely normal Glued function, <br />     GIR(P)/GIcombinations lethal (Table 4). <br />        The Gl&apos;c&apos;) class 17 recessive lethal Glued mutations induced in <br />     GI+ strain EMS (Table 2). All mutations lethal homozygously <br />     heterozygous combinations GI hemizygous combinations <br />     deletion (Table 1). All pairwise heterozygous combinations 17 <br />     mutations failed complement, indicating intragenic comple- <br />     mentation lethal effect mutations. Complementation tests also <br />     heterozygous combinations GI&quot;&apos;) mutations three <br />                               GLUED MUTATIONS IN DROSOPHILA                                                      649 <br />  <br />                                                    TABLE 2 <br />          Complementation groups EMS-induced lethal mutations fail complement a <br />                     deletion spanning 7OC-70F region chromosome 3 L <br />  <br /> Complementation group              1       2   3        4       5   6         7       8       9       10   11    12 <br /> No. mutations               1       7   8   8    2       1   2   8     2       3   7   3       1   0     1     2 <br />    The EMS-induced lethal mutations isolated described MATERIALS AND METHODS and <br /> were tested complementation pairwise combinations follows: lethal st red calTM3 Sb Ser <br /> X lethal st red calTM3 Sb Ser. The progeny cross kept 29&quot; devel- <br /> opment, resulting adult flies scored eye color, dark orange for <br /> st red calst red ca homozygotes red (wild type) st red calTM3 Sb Ser heterozygotes. Each <br /> cross yielded orange-eye flies 100 examined, indicating two <br /> EMS-induced mutations failed complement and, therefore, belong group, about <br /> 30% orange-eye flies, indicating mutations belong different groups. When more <br /> than mutation classified group, usually used further <br /> tests mutations groups. Group 1 contains G1&quot;&quot; mutations fail comple- <br /> ment lethal effect GI. One mutations failed complement mutations groups <br /> 5 7, possibly deletion spanning groups Occurrence inde- <br /> pendent mutations, group. This mutation tentatively assigned groups. <br />  <br />                                                    TABLE 3 <br />                    Complementation tests Glued mutations lethal mutations <br />                                           flanking groups <br />  <br />                                                                           One more <br />                                                     All flanking        flanking groups <br />                                                     groups              not <br />                               Glued class          complemented          complemented <br />  <br />  <br />  <br />  <br />                    The complementation tests involved genetic crosses between <br />                 strain containing Glued mutation strain con- <br />                 taining lethal mutation group flanking Glued (see Table <br />                 2), marked recessive eye color mutation st and <br />                 balanced chromosome 3 balancer T M 3 Ser T M 3 Sb <br />                 Ser. The cross scored noncomplementing flies ex- <br />                 pressing scarlet eye color 100 <br />                 progeny flies examined, indicating heterozygous combi- <br />                 nation Glued mutation flanking mutation lethal. <br />                 In complementing crosses, one-third progeny <br />                 flies expressed scarlet eye color. <br />  <br />  <br /> classes GIR mutations. GIR(&quot; GIR(&apos;) mutations did complement the <br /> lethal effect of GI1(&apos;) mutations, GIR(&apos;) mutations complemented <br /> most GI&quot;&apos;) mutations, 9 10 (Table 4). <br />    All GZR(G), GZR(&quot; GZ&apos;&quot;) mutations described lethal hemizygous <br /> combinations deletion (Table 1). However, stage lethality, is <br /> the advanced developmental stage reached organism dies, not <br /> the mutations. T h e stage lethality examined GZ, <br /> 17 GI1(&apos;) mutations 21 GIR(&apos;) mutations appear affect the <br />  650                  A.   GAREN,       B. R. MILLER AND M. L. PACO-LARSON <br />  <br />                                                   TABLE 4 <br />                                 Complementation behavior GIwp&apos;mutations <br />  <br />                                                             GI1&quot; mutations <br />  GIRr) <br /> mutation GI GlR4G&apos; 11      12     13    14   15   16   17    18   19   110 fll 112 113 114 115 116   117 <br />  R 2 0   1 - +        +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R202    -       +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R 2 0   3   -   +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R 2 0   4   -   +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R 2 0   5   -   +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R 2 0   6   -   +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R 2 0   7   -   +    +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />  R208    - +          +    +      +     +    +    +    +     +    -    -     +   +   +   +   +   +   + <br />   The complementation tests described Table 3. A ne tive result indicates that <br /> the heterozygous combination Glued mutations lethal. GIE(q deletion the <br /> 70C-70F region including entire Glued gene. <br />  <br />  <br /> Glued gene (Table 3), heterozygous combination deletion (Table <br /> 5 ) . Most mutations, including GIR(&apos;) mutations, caused first- <br /> instar stage lethality; these, possibly all, null mutations that <br /> eliminate zygotic Glued function. An earlier embryonic stage lethality <br /> occurred GZ GZ&apos;&quot; mutations, 9 10, only <br /> mutations fail complement GZR(&apos;) mutations (Table 4).A later <br /> pupal stage lethality occurred GZt0 mutations, might <br /> be leaky mutations partially block Glued function. <br />    T h e GZR(&apos;) strains tested insertion P element region <br /> of Glued gene, previously mapped 70C region chromo- <br /> some 3L genetic crosses various marker strains including cytologically <br /> defined deletions (HARTEand KANKEL1982). The tests situ <br /> hybridization biotin-labeled DNA probe containing P element sequences to <br /> salivary gland chromosomes GZR(p)/GZ+larvae, progeny a <br /> cross balanced GZR(&apos;) strain standard GZ+ strain lacking P <br /> element; therefore, chromosomes derived GZR(&apos;) parent should <br /> contain P element, derived GI+ parent not. T h e <br /> hybridization <span id='am-241' about='xsp:pattern' typeof='owl:Thing'>pattern</span> GIR(&apos;)strains shows 16 hybridization sites for <br /> the P element scattered genome (Figure 1A); region where <br /> the homologs separated, indicated arrows, hybridization sites <br /> occur homolog, derived GZR(&apos;) <br /> parent. All hybridization sites positions 60 80 chromosome <br /> 3L, includes 70C site Glued gene, mapped the <br /> GIR(&apos;) strains (Table 6). A total 13 hybridization sites identified, <br /> of 70C site common strains. The invariant correlation <br /> between P element insertion 70C site induction GIR(&apos;) <br /> mutation provides evidence mutations result insertion <br /> at site. Higher resolution mapping 70C region GIR(&apos;) <br /> strains places insertion site 70C2 band (Figure 1B). <br />                           GLUED MUTATIONS IN DROSOPHILA                                       651 <br />                                           TABLE 5 <br />                       Stages lethality caused lethal Glued mutations <br />  <br />                    Embryo                       1st-instar larva             Early   P U P <br />  <br />  <br />  <br />                             19               R2               11                13 <br />                             110              R4               12                111 <br />                                              R5               14                114 <br />                                              R7               15 <br />                                              R9               16 <br />                                              R12              17 <br />                                              R18              18 <br />                                              R19              112 <br />                                              R23              113 <br />                                              R29              115 <br />                                              R32              116 <br />                                              R36              117 <br />                                              R37 <br />                                              R46 <br />                                              R48 <br />                                              R50 <br />                                              R52 <br />                                              R56 <br />                                              R57 <br />                                              R63 <br />                                              R64 <br />             Strains containing indicated Glued mutations individually <br />           crossed strain containing mutation Df(3)GPR*,which deletes <br />           entire Glued region (HARTEand KANKEL 1982; M. LEVINE,M. L. <br />           PACO-LAWN A. GAREN,unpublished results); strain was <br />           marked recessive mutation red balanced chro- <br />           mosome 3 balancer TM3 Ser TM3 Sb Ser. Several eggs were <br />           collected cross, developing organisms kept at <br />           29&quot;. The first-instar larvae examined expression red <br />           marker, identifies larvae containing Glued mutation and <br />           Df(3)G1+R4deletion. If larvae expressed red marker, <br />           stage lethality scored embryonic. If red marker was <br />           expressed larvae, larvae transferred fresh <br />           food examined period 5 days determine stage <br />           lethality. A first-instar stage indicates larvae did develop <br />           apparently died day hatching. An early <br />           pupal stage indicates larvae pupated expected time <br />           did subsequently develop differentiated pupal tissue. <br />  <br />  <br />                                          DISCUSSION <br />  <br />   The mutations Drosophila gene Glued described report and <br /> elsewhere (HARTEand KANKEL 1982) provide basis initial characteri- <br /> zation gene functional <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. The mutations include deletions induced <br /> by y-radiation, insertions induced transposable P element mutations <br /> induced EMS probably include <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span> substitutions. Of central importance <br /> to genetic analysis original mutation GI, dominant nonlethal <br /> effect principally development visual Gl/Gl+ heterozygotes <br />  652                A. CAREN, R. R. MILLER A N D M. L. PACO-LARSON <br />  <br />  <br />  <br />  <br />                                                  A <br />  <br />  <br />  <br />  <br />                                                  B <br />    FIGUREI .-Chromosomal insertion sites P element DNA Clem reversion strain. Salivary <br /> gland chromosome squashes prepared heterozygous CIRm/Gl+larvae, progeny <br /> of cross balanced strain containing Glm mutation R201 n Oregon-R standard <br /> strain does contain P element. A DNA probe prepared pBR322 recombinant <br /> plasmid p 25. I , contains P element sequences (RUBINand SPRADLING      1982). labeled with <br /> biotin (LANCER.WALDRUPand WARD 1981) n d hybridized chromosome squashes. T h e <br /> hybridiiation sites visualized reacting chromosomes goat antibiotin serum, then <br /> with rabbit antigoat serum labeled peroxidase finally peroxidase stain containing 2.3- <br /> diaminobenzidine H*Op(LANCER-SAFER,          LEVINEand WARD 1982). A. Chromosomes a <br /> nucleus asynapsis chromosome 2R homologs extending branch point <br /> near telomere (indicated arrows); note homologs, presumably <br /> derived GIm&apos;&quot; parent, contains P element hybridization sites. E. A higher magnification of <br /> region 68-70 chromosome 3L nucleus; note hybridization site band near <br /> the 70B/C border delineates position Glued gene (see Table 6). <br />  <br />  <br /> and recessive lethal effect early development GI/GI homozygotes <br /> and Gl/deletion hemizygotes (Table 1). Both dominant nonlethal reces- <br /> sive lethal effects GI probably involve gene, GIR&apos;&quot; mutations of <br /> GI induced insertion P element, selected reversing the <br /> dominant effects GI, complement recessive lethal effect deletion <br /> of Glued region (Table 4). T h e GIR&apos;&quot; mutations induced  <br /> </body></html>