<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1997 Genetics Society America <br />  <br />  <br />  <br />      Distinct Requirements Somatic Germline Expressionof Generally <br />                       Expressed C m h b d t s elegam Gene <br />  <br />                           William G. Kelly, SiQun Xu, Mary K. Montgomery Andrew Fire <br />                          Carnegie Institution Washington, Department Embryology, Baltimore, Maryland 21210 <br />                                                        Manuscript received December 18, 1996 <br />                                                       Accepted publication February 14, 1997 <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                   In screening embryonic-lethal mutations Cuenorhubditiseleguns, defined essential gene <br />                (let-858) encodes nuclear protein rich acidic basic residues. We named product <br />                nucampholin. Closely homologous sequencesin yeast, plants, mammals demonstrate strong evolu- <br />                tionary conservation eukaryotes. Nucampholin resides nuclei C. eleguns essential in <br />                early development differentiating tissue. Antisense-mediated depletion LET-858 activity in <br />                early embryos causes lethal <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> similar characterized treatments blocking embryonic gene <br />                expression.Usingtransgene-rescue, demonstratedtheadditionalrequirementfor               let-858 inthe <br />                larval germline. The broad requirements allowed investigation soma-germline differences gene <br />                expression. When introduced standard transgene arrays,let-858 (like C. eleguns genes) <br />                functionswell soma poorlyin germline. We observed incremental silencing    simple let-858 arrays <br />                thefirstfew generations followingtransformationandhypothesizedthatsilencingmightreflect <br />                recognition arrays repetitive heterochromatin-like.<span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> transgene physiological <br />                context, included excess random genomic fragments injected         DNA. The resulting <br />                transgenes robust expression germline soma. Our results suggest possibility        of <br />                concerted mechanisms silencing unwanted germline expression repetitive sequences. <br />  <br />  <br />  <br /> G     ERMLINE totipotency critical reproductive <br />        function. To maintain totipotency, be <br /> mechanisms ensure germline insu- <br />                                                                              cent. After animal hatches, dramatic in- <br />                                                                              crease number          volume germ <br />                                                                              cells animal, fromtwo small cellsto population <br /> latedfrom somatic differentiation pathways                          thousand nuclei, comprises adult <br /> generate adiversity tissues.A striking example                     germline. This increase cell number mass <br /> early insulation germline somatic processes                       presumably involves newexpression wide varietyof <br /> seen nematodeC. ekgans. In theearliest divisions,                     genes, including responsible housekeeping <br /> a series germline blastomeres (P lineage) eachdivide                      functions, cell proliferation, responses environmen- <br /> to daughter cells; rise                      tal cues, multitudeof factors destined incorpo- <br /> to somatic tissue lineages, otherwill maintain                     ration developing oocytes (AUSTINand KIMBLE <br /> its germline characteristics (SULSTON   et al. 1983). The                    1987; SCHEDI,    1996). With onset extensive new <br /> generation germlineand somatic fates                        gene expression germline, remains need <br /> division germline blastomere parent results                     prevent somatic (and perhapsirreversible) differenti- <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> asymmetric distribution cytoplasmic com-                         ation occurring inthe germ lineage. Thus might <br /> ponents daughter cells (for review, GUOand                        expected germline C. ekgans uses or <br /> KEMPHUES 1996b). Although germline cell                          efficient mechanisms distinguish activity in <br /> newly formed somatic sisters appear marginally                          gene expression somatic neighbors. <br /> different microscope, levelof gene                             One mechanism conceivablyserve this <br /> expression striking difference. At early                    function chromatin elimination. In certain <br /> stages development, appears germ cell                         nematode species, massive rearrangement chromo- <br /> nucleus fails express (and all) mRNA                         somes elimination certain interspersed germline <br /> encoding genes ( S ~ O Uand  X FIRE1994). At                        sequences occurs eachsuccessive germline-soma divi- <br /> time transcription different componentsis be-                        sion, genes expressed soma a <br /> ing activated somatic sisters cousins                      strikingly different genomic &#226;&#8364;&#339;neighborhood&#226;&#8364;? the <br /> germ lineage.                                                                genes germline (MUL.LERet al. 1996). This <br />    The germline remain transcriptionally quies-                       massive elimination DNA soma seen in <br />                                                                              C. ekgans, quite possible change the <br />                                                                              nature chromatin soma germline might <br />   Cvrre.rpondingauthor; William G. Kelly, Carnegie Institution Wash- <br /> ington, Department Embryology, 115 W. University Parkway, Balti-          critical C. elegans parasitic relatives. <br /> more, MD 21210. E-mail: fire@maill.ciwemb.edu                                   Transgenic studies C. ekgans suggest such <br /> Genetics 1 4 6 227-238 (May, 1997) <br />  228                 Kelly             G.                W.           et al. <br />  <br /> soma/germline contextual differences ex-                  Anti-LET-858   antibodies:    A cDNA fragmentencoding <br /> ist. Transgenes heritably transmitted extrachro-           amino acids 137-333 LET-858 inserted in-frame into <br />                                                                   trpE bacterial expression vector, PATH1 (KOEHNERet al. <br /> mosomal arrays, linear products re- <br />                                                                   1991). Following IPTG induction, resulting TRP-E::LET- <br /> combination ligation DNA injected intothe syn-             858 fusion protein purified antibody preparation. An- <br /> cytial gonad oftheadult             (STINCHCOMB      et al. 1985; tigen injections mice, resulting antisera collections, <br /> MEL.L,Oet al. 1991). A wide variety lac-Z n d           performed JosmanLaboratories (Napa, C A ) . The <br /> Aeguorea victoria green fluorescent protein (GFP) re-             antisera tested specificity Western blot analysis of <br />                                                                   bacterial C. elegans lysates. In situ 1ocaliz.ation en- <br /> porter gene constructs introduced               C. dogenous LET-858 protein subsequently carried on <br /> elegans n m n n e r , efficient expression   fixed C. elegans adults embryos described (MILLERand <br /> reporter somatic tissues(FIRE et al. 1990; HOPE           SHAKES1995). <br /> 1991; CHALFIEet al. 1994).An exception success,              In situ hybridization localize RNA transcripts: Whole <br /> in laboratory others, inability              mount    situ hybridizations wild-type (N2) C. elegans em- <br />                                                                   bryos performed described (SEXDOUX        FIRE1994, <br /> to observe reporter transgene expression               1995). Digoxigenin (DIG)-labeled single-stranded DNA <br /> germline.Thisfailure,coupledwithnotablypoor                       probes synthesized multiple cycles primer exten- <br /> transgenic rescue maternal effect mutations, led           sion presence DIGdUTP using full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br /> us conclusion transgenes &#226;&#8364;&#339;viewed&#226;&#8364;?             let-858 cDNA clone (pBK 4.1) partial let-858 genomic <br /> differently germline soma sug-             clone   (pMM215) template essentially described (PATEL <br />                                                                   GOODMAN       1992).Antisense probes generated either <br /> gests mechanism gene expression mayplasmid gave similar results; signal detected sense <br /> be differentially repressed postembryonic germ-            probes (data shown). <br /> line. The tractable feature mechanism                let-858::$p fusions deletion constructs:The codingre- <br /> the selective silencing transgenes germline.            gion theAequorea victm.a green fluorescent protein (GFP) <br /> We useda generally expressed C. elegans            inserted unique AgeI site kt-858 genomic con- <br />                                                                   struct, pBK42, resulting internal, in-frame fusion the <br /> gene, let-858, begin address theproblem germ-            fourth exon let-858 GFP coding sequence. This con- <br /> line transgenic expression try understand           struct parent vector deletions inser- <br /> different rules govern germline somatic ex-              tions used test &amp;acting elements driving germline ex- <br /> pression genes.                                                pression ofLET-858 (see text). In general, all5&#226;&#8364;&#8482; deletions <br />                                                                   generated following strategy: BamHI/AgeI frag- <br />                                                                   ment pBK48 (encompassing genomic 5&#226;&#8364;&#8482; flanking <br />                 MATERIALSAND METHODS                              DNA let-858 AgeI GFPfusion site) replaced <br />    EMS lethalscreen: To isolate lethalmutationsin             PCR-generated fragments encoding increasing deletions <br /> mnDf90/cey-l region LC 11, L4 stage unc-4 (e120) animals       5&#226;&#8364;&#732; flanking region let-858. These fragments were <br /> were mutagenized ethane methyl sulfonate (EMS)            using BamHI site-containing sense primers correspond- <br /> described(BRENNER 1974) andmated               unc-4[e120)/  ing  sequences increasingly 3&#226;&#8364;&#732; let-858 5&#226;&#8364;&#8482; flanking <br /> mnCl(dpy-10 unc-52) males. Phenotypically wild-type F, prog-      region, coupled antisense primers corresponding g@ <br /> eny cloned screened         failure segregate sequences AgeI site. <br /> fertile adult unc-4 homozygotes. Sterile lethal mutations        All internal and3&#226;&#8364;&#8482; deletions tested constructed <br /> linked unc-4 kept furtherstudy com-         excision ligation internal restriction frag- <br /> plemented mnDf-77 butnot mnDf-87 mnDf-90.               ments, exception intron 5-intron 6 deletion: <br /> Thirty-four alleles belonging seven separate complementa-      construct generated replacement the <br /> tion groups, designated let-852 let-858, recov-      genomic fragment corresponding fragment a <br /> ered screen.                                            cDNA clone. <br />    Cloning sequencingof leb858: Mutations oneof           Transgenicanalyses: dp fusion reporter constructs were <br /> genes recovered EMS screen, let-858, rescued        introduced     using methods. The method <br /> by injection mixtures containing thecosmid F33A8 using       involved co-injection circular reporter transforma- <br /> described procedures (MELLOand FIRE 1995). The rescuing           tion marker plasmids previously described (MEI.LOand <br /> portion cosmid delineated combina-        FIRE1995). The resulting transformed lines generated from <br /> tion restriction digests Ex0111 digestions parent     &#226;&#8364;&#339;standard&#226;&#8364;? transformation protocol carried reporter <br /> 11-kb XbaI fragment, yielding 6.8-kb rescuecompetent sub-       marker plasmids heritable extrachromosomal arrays <br /> clone, sequenced. To determine genomic ar-          proved incompetent reporterexpression germ- <br /> rangement kt-858, C. elegans lambda-Zap (Stratagene)         line(see text). To optimize array complexity, another <br /> cDNA library probed high stringency XbaI          approach involving dilution carrier DNAwas              used. <br /> fragment, positive clones isolated sequenced.        Briefly, s;fp reporterand      transformationmarker       plasmid <br /> Additional sequences cDNAs corresponding let-858were         [pRF4; containing dominant <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span> collagen gene, <br /> obtained searches expressed sequence-tagged        rolG(su1006)] DNAs linearized restriction digestion at <br /> databases. The 5&#226;&#8364;&#8482; end let-858 transcript identified    unique sites vector backbone. Where feasible, blunt <br /> by PCR amplification mixed-stage cDNA pool using          end-generating enzymes used: T4 polymerase used <br /> internal gene-specific antisense primer paired primers        create blunt ends ondigested plasmids &#226;&#8364;&#339;sticky-end over- <br /> corresponding theC. elegans SLl orSL2 trans-spliced leader     hangs&#226;&#8364;? generated unique backbone sites <br /> sequences. A single product generated reactions con-       available. The linearized plasmids mixed with <br /> taining SL1 oligonucleotide. The structureof product     PvuII-digested N2 (wild-type) strain genomic DNA had <br /> indicated thatthe let-858 mRNA contains theSLl splice             column-purified (Wizard Prep columns, Promega) and <br /> leader, followed 5 bp 5&#226;&#8364;&#8482;UTR preceding initiator           phenol-chloroform extracted. The mass ratio N2 DNA Lo <br /> codon.                                                             reporter marker plasmid DNA injection mix was <br />                                                     Germline Somatic Expression                                                 229 <br />  routinely -50-100:1, N2 DNA concentration 50-                                      TABLE 1 <br />  100 ,ug/ml.The DNA injection mix formulated either <br /> dH20 TE buffer (10 mM Tris, pH 7.5, 1 mM EDTA).                      Mutations essential genes defined region LGII <br />     In allassays, transformed F1 progeny injected animals <br /> were cloned checked transmission heritable             Complementation                                  Phenotype <br /> arrays, indicated presence marked transformants              group                     Alleles           (arrest point) <br /> in (F2)progeny. Any clones segregating arrays were <br />  deemed independent lines study.    cc502For         kt-852                                    Larval <br />  deletion generational silencing studies, multiple animals                                       cc503            Larval <br />  (more eight) multiple independent lines (more                                             cc504            Larval <br />  five)wereanalyzed.Relative        fluorescence deter-                                      cc520            LdNd <br />  <br />  mined comparing GFP fluorescence intensityof nuclei                 kt-853                   cc505            Embryonic <br />  test lines nuclei similar tissues &#226;&#8364;&#339;bench-                                   cc506            Embryonic <br />  mark&#226;&#8364;? line. The &#226;&#8364;&#339;benchmark&#226;&#8364;? line used kt-858 (cc534)                                          re521            Larval <br />  homozygous line rescued fertility ceinjection the <br /> pBK48.1 kt-858::gfp reporter construct N2 genomic DNA                   kt-854                   cc50 7           Embryonic <br /> fragments.                                                                                           cc508            Larval <br />     Antisense RNA injections: Antisense RNA generated <br />  vitro, using linearized kt-858 cDNA clone template,                kt-855                   cc509 <br />  according instructions supplied RNA polymerase                                            re510 <br />  source (Promega). Thetranscription reaction terminated                                          cc511 <br />  ice, products purified successive phe-                                       cc512            All Larval <br />  nol :: chloroform extractions, followed chloroform extrac-                                       cc513 <br />  tion ethanol precipitation. Each antisense RNAwas in-                                           cc522 <br /> jected concentration 0.5-1.0 mg/ml. The injected                                             cr523 <br />  animals wereallowed torecoverovernight.Animalswere                                                  cc524 <br />  transferred following day newplates allowed lay                                       cc525 <br />  eggs 4-6 hr observation. The fraction arrested <br />  offspring determined plates counting un-                 kt-856                   cc514 <br />  hatched embryos following day. The fraction arrested                                         cc515 <br />  embryos antisense-injected animals typically exceeded                                          cc526            All Larval <br />  90%. Antisense RNA mRNAs, including pie-l                                             cc52 7 <br />  mex-3, injected parallel N2 animals compari-                                       cc528 <br /> son arrest phenotype assess efficiency                                          cc529 <br /> injection protocol. Sense RNA controls attempted,                                           cc530 <br /> given known mimicry antisense loss-of-functionpheno- <br />  types sense RNA injections C. elegans (FIREet al. 1991;              kt-85 7                cc516              Embryonic <br />  Guo KEMPHUES 1995).                                                                         1 cc5 7              Embryonic <br />                                                                                                    cc518              Larval <br />                                                                                                    cc531              Larval <br />                             RESULTS                                                                cc532              Larval <br />                                                                                                    cc533              Larval <br />    Essential genes defined region chromosome <br />                                                                             kt-858                   cc500            All embryonic <br /> Zk We sought identify genetic loci products                                                 cc501 <br /> might required ubiquitously develop-                                                   cc534 <br /> ment. HERMAN     colleagues provideda number <br /> of genetic tools make chromosome     I1 amenable                 Thirty-four mutations sevencomplementation groups re- <br />                                                                       covered EMS screen grouped gene designa- <br /> region isolation genetic characterization le-              tion (kt-852to k t 8 5 8 allele designation (cc no.). Embry- <br /> thal mutations (HERMAN     1978; SICURDSON    et al. 1984).           onic arrest, alleles homozygous embryos het- <br /> From chemical mutagenesis screen defined re-                   erozygous mothers failed hatch; arrested, arresting <br /> gion chromosome,     obtained 34 allelesof                 hatching L4 larval molt. <br /> seven essential genes (Table 1). For initial analysis, we <br /> chose gene screen, let-858. All let-              fines 2.8-kb open reading frame (Figure 1C).RT-PCR <br /> 858 alleles uniform embryonic arrest phenotype                 analysis showed mRNA trans-spliced the <br /> in 100% embryos fail hatch.                           SL1 splice leader. Analysis genomic sequence indi- <br />   Molecular structureof let-858: In initial rescue exper-             cated open reading frame    interrupted by <br /> iments, cosmid F33A8 (Figure 1, A                   intron segments spans      majority the <br /> B) sufficient rescue embryonic lethal defect               original 8.4kb fragment (Figure    1C). Deletions ei- <br /> in let-858 homozygotesderivedfrom        heterozygous               ther end open reading frame prevent transgene <br /> mother. Subsequent experirrents using restriction     frag-           rescue, demonstrating coding region indeed <br /> ments F33A8 narrowed rescuingactivity                  corresponds let-858. <br /> 8.4kb Xbul-ApuI fragment. This fragment used                      The LET-858 protein predicted 897 amino <br /> probe C. ekguns embryonic cDNA library, resulting                acid polypeptide thatis highly hydrophilic, beingrichly <br /> the isolation cDNAs, longest de-                 scattered acidic basic residues(42% <br />  230                                                      W. G. Kelly et ul. <br />  <br />  A                                                                   ekguns homologue led choose &#226;&#8364;&#339;nucam- <br />                                                                      pholins&#226;&#8364;? group homologous proteins. <br />                                                                         Distribution let-858 products: A kt-858 genomic <br />                                                                      clone used probe Northern blot containing <br />                                                                      mRNA different developmental stages. A band of <br />                                                                      -2.8 kb detected; abundant em- <br />                                                                      bryos larvae, decreased abundance adults <br />                                                                      (Figure 3 ) . Muchof adult message appeared to <br />                                                                      contributed germline, evidenced the <br />                                                                      decrease adults homozygous germlinedeficient <br />                                                                      mutation, gZ3-4 (bn2).In situ analysis ofthe distribution <br />                                                                      let-858 mRNA embryos showed RNA to <br />                                                                      present cytoplasm cellsat embryonic <br />                                                                      stages examined (Figure 4, A B). <br />                                                                         To assess cellular tissue distribution the <br />                                                                      protein, antibodies generated bacteria- <br />                                                                      expressed fusion carrying amino acids 139-336 LET- <br />                                                                      858. These antibodies used immunofluores- <br />         4 <br />        ATG <br />                                              4 <br />                                             Poly A                   cence localization mount preparations. As <br />                                                                      shown Figure 4, antibody fluorescence observed <br />                                                                      localized nuclei adult tissues and <br />    FIGURE1.-Geneticandphysicalorganizationofthe             let-     cells embryos (Figure 4, C-E). The nuclear <br /> 858 locus. The position let-858 relative nearby deficien-      staining nonuniform adult tissue nuclei; LET- <br /> cies genetic markers chromosome 11, shown A.            858 protein localized nucleus a <br /> cq-1 unc-53 genes physically mapped           somewhat globular <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> thatoverlaps not <br /> to theregion. (B) The physical restriction map cosmid <br /> F33A8; locations let-858 cq-I indicated. (C)          restricted to, regions filled chromosomes (as visu- <br /> The genomic organization shown let-858 rescuing            alized DNA-binding dye DAPI; shown). The <br /> fragment. The minimal rescuing activity confined         proteinappears excluded nucleolus. <br /> Ex0111 deletion endpointat 5&#226;&#8364;&#8482; end ApuI site        During mitosis, protein released thenucleus <br /> 3&#226;&#8364;&#8482; end; truncations XhoI (5&#226;&#8364;&#8482;) and/or SulI (3&#226;&#8364;&#732;) sites          (Figure 4D). In adult germline, association of <br /> abolished rescuing activity. A larger construct flanking <br /> XbuI sites used probe mixed-stage C. eleguns cDNA           LET-858with chromatin evident germ cells <br /> library, cDNAs obtained sequenced.                 mature oocytes; subsequently LET-858 disperses <br />                                                                      throughoutthe nucleoplasm oocytes become <br />                                                                      fully diakinetic (not shown). The antibody signal in <br /> charged residues). The predicted protein contains                  germline nuclei stronger somatic tissues <br /> cluster cysteine residues (five cysteinesbetween posi-            (Figure 4E, inset); consistent germline <br /> tions 521 560) flanked number histidine                  enrichment forlet-858 mRNAseen Northern analysis <br /> residues (505, 568, 584), metal bind-               (Figure 3 ) . Pre-immunecontrolsexhibiteda         diffuse <br /> ingdomain(although         strictly resembling               background fluorescence nuclear concentra- <br /> known &#226;&#8364;&#339;zinc finger&#226;&#8364;? motif). A    striking serine aspar-          tion signal (not shown). <br /> tate-rich stretch near C terminus (residues                Loss function phenotypes let858: When ho- <br /> 696-729; Figure 2). Searches forgenesinGenbank                       mozygous kt-858 mutant animals derived a <br /> with similarity kt-858 (using BLAST; ALTSCHUL et ul.              heterozygous mother, uniform arrestphenotype is <br /> 1990) yielded expressed sequences                  seen (Figure 5A).These embryos arrest partial <br /> other species [human, mouse, Arabidopsis, Sac-                   elongation, indicating occurrence    earlyhypoder- <br /> charomyces (Figure 2)].The identified coding seg-                    mal differentiation (PIUESSand HIRSCH1986). Func- <br /> ments fromthese species remarkable conservation                 tional muscle made, indicated fact the <br /> at aminoacid level corresponding regions               animals move, gut pharyngeal differentiation <br /> LET-858 protein. A single putative homologue                observed. The animals eventually stop moving, fail <br /> identified yeast genome sequencingproject; like-              hatch, develop large vacuoles arise most <br /> wise, far beenonly kt-858gene iden-               frequently head region. <br /> tified 50-60% worm genome sequenced                       Depletion maternally donatedlet858 antisense <br /> to date. Thecoding regions theother species                RNA results severe arrest phenotype: Since <br /> all derived large-scale cDNA genomic sequenc-                maternal contributionsof kt-858 RNA protein have <br /> ing efforts; information available concerning              observed, likely phenotypesof homo- <br /> function of species. The highly amphoteric              zygous offspring heterozygous mother not <br /> nature intracellular localization (below) C.              reflective complete loss function due <br />                                                                                                                        Germline <br />                                                                                                                        Expression <br />                                                                                                                                Somatic                                                                                                                                                                                                   231 <br />  <br /> C.Elsgam    1       ~    ~    1       9   1   1       ~   E       P       ~   T   P   K   A   A       ~       ~   X   1       l   ~       ~       l   K   X   ~   E   ~       ~    P   ~   T       P   K       T   ~       ~   l   K       S       A   ~   ~   E 70~    ~       P       ~   ~   ~   ~   ~       V       E   Q   8   T    X       T       ~   ~   W   ~   l   K <br /> S.cemul&amp;.   1       . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .                                                                                                                                                                        0 <br /> ~malla~ I           _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - - o- - - - - <br /> M . m ~ / m 1       _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - - 0- - - - - <br /> H.-jens     1       _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - - 0- - - - - <br /> ~.e-m         71    D         O           D       D           ~           P       ~       T           P           ~       ~           ~           ~       ~       ~       ~            T       ~  ~ FI&lt;;URE <br />                                                                                                                                                                                                         T  P ~     2.-Sequence <br />                                                                                                                                                                                                                    ~   A <br />                                                                                                                                                                                                                        ~   ~   P <br />                                                                                                                                                                                                                                ~    P ~  K K           ~       ~ I40K <br /> S.c&quot;*e          1   &quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;                                                                                                                                                                                                          0 <br /> A.ma{hM         1   ----------------------------------------------------------------------                                                                                                        evolutionary conser-                                       o <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - - 0- &quot; - - <br />                     _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - - 0- - - - - vationofnucamphilins. <br /> M,mr&amp;           1 <br /> H.-jens         1 <br />                                                                                                                                                                                                   The predicted protein se- <br /> C.a@am        141   P A P L P ~ K K K E ~ P L D X L l T n T a A r I P P A K L n L n Q Q Q I D K Q ~ r Q R n U N I n N K E ~ I n L v n n v ~ A K n L v <br /> S.m&amp;a           1   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - W S ~ A T X Q D l D X K ? Q R X ~ N l W X R S E V ~ P ~ I ~ ~                                        38 L T n quenceforC.p/qamLET- <br />                                                                                                                                                                                         210       D U ~ Q <br /> A.rnakM         1    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - ~ - - - - - - - - - - - - - - - - - - - - - - - o- - - - 858isdepictedandaligned <br /> M.mwls          1    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - -0 - - - - -(using ClustalW software <br /> H.-kn$          1    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - -0 - - - - - <br /> C.ElsganS <br /> s.CWmi&amp;S <br />               211 <br />                39 <br />                     (I IVRXLLQ <br />                     8 SEIIDL?Q <br />                                                I R S K O L C D I 1 0 A . d ~ra P - - aQa <br />                                                L I O R N I I K ~ V ~ D ? T L ~ K Q ~ O ~ L I P A L B A L I DAI LP D                  LI <br />                                                                                                                                               m q p mvmpQP               na <br />                                                                                                                                               T L A K I I L M I V ~ ~ ~ 108 <br />                                                                                                                                                                                    m 278 <br />                                                                                                                                                                                            N R <br />                                                                                                                                                                                                   BLOSUM weight <br />                                                                                                                                                                                                   matrix; T~~~~~ d a~ <br /> AhalhM          1    - - - - - - - - ........................................................... o                                                                                                1994) predicted p r e <br /> M.mwb           1    ________________________________________------------------------ <br />                      - - - - - - - - 8 n v z n E G t e n r x I o a ~x - - p z~ E                                             ~ x m r m x m Lm            . ii~ i;Q <br /> H. w m s <br />  <br /> c. S w f l S 279 <br /> S.cmVieia109 <br />                 1 <br />  <br />                     9       0 V T V B ~~1~~~                                                    I[%BmT D <br />                     K D Y V ~ C O I L ~ C L ~ ~ L I L IY~ D. I Y ~ ~ ~ ~ ~ L D L L . ~ H - - - B L R L V I I Y * ~ ~ <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ . . . . . _ _ . _ _ _ _ _ _o _ . The <br />                                                                                                                                                               D m : :z : <br />                                                                                                                                                               ;; ;  ;  ; ;         .&quot; <br />                                                                                                                                                                                         :o        tein sequences <br />  <br />                                                                                                                                                                                                   _ . S <br />                                                                                                                                                                                                            I           homo&apos;+~ <br />                                                                                                                                                                                                   gous sequencesrecovered <br />                                                                                                                                                                                                   database searches. <br />                                                                                                                                                                                                           .cereuisiaesequence    is_ . _ _ . <br />                                                                                                                                                                                                                                                        : ;: ::: <br /> /\.maluna <br /> M,murcd(# <br /> H. w e n s <br />                1 <br />                1 <br />                61   mr. <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ . . <br />                             0 L c L T A ~ I K F E I u ~ e I E ; L IT n* L m x m T D D                              ~       E   E <br />                                                                                                                                        _ J <br />                                                                                                                                            -_ - <br />                                                                                                                                            _ L <br />                                                                                                                                               _   _ _ <br />                                                                                                                                                     - <br />                                                                                                                                                     Q <br />                                                                                                                                                       _ -_I <br />                                                                                                                                                             -_ - <br />                                                                                                                                                             _  _  _ -_ _ -_ -_ - <br />                                                                                                                                                                             _    _   _ -0 - ~ - ~ - - - - - ~ - - - - - - <br />                                                                                                                                                                                          117 <br />                                                                                                                                                                                                          _ <br />  <br />                                                                                                                                                                                                   predicted <br />                                                                                                                                                                                                              _ _ ~ . _ _ _ <br />  <br />                                                                                                                                                                                                                  open reading <br /> C.awnS <br />  <br /> Am!@- <br />           349 <br /> ~cemvirila 17s <br />            1 <br /> M,~n&quot;$c&quot;Im 1 <br />                     R L R A X L <br />                     J J r D          T a[Js                                   . <br />                                      X T 8 ~ ~ I ~ A L D R ~ I Q Y ~ X K T A W ~ I ~ K D E ~ A A Y P A V X K D L D L X E 8 8 D Q I X E T L N L l D A4 1V8 D P ~ N O L <br />                                                    T L n s L 1 T L r x x n Q E D r K a Q - - - - - - - - - - - - - - - - - - - - - - L r I L D p 7 8 &apos;IT v 221 <br />  <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - - 0- - - - - the <br />                      _ _ _ _ _ _ _ _ _ _ _ . _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _0                                                                                                       _ _ _ _ <br />                                                                                                                                                                                                   frame obtained theS. <br />                                                                                                                                                                                                   m h G e n o m e database; <br />                                                                                                                                                                                                   - <br />                                                                                                                                                                                                                     _ _ _ are <br />                                                                                                                                                                                                             _ _sequences <br />                                                                                                                                                                                                                        _ - - - - - - - <br />  <br />                                                                                                                                                                                                   predictedfrom cDNAsin <br /> c..klpnr <br /> snnsvrjse 224 <br /> ,tamane <br /> M.muSCUlUr <br />               419 <br />  <br />                 1 <br />                 1 <br />                     ~ V T K L ~ P X T X K ~ X ~ ~ ~ X X I ~ K X I I ~ ~ A D I ~ D X D ~ ~ D ~ L ~ D X I X D~P ~ <br />                     T 8  (I &apos;I I v D I D I A rn X X L 0 u r E c n n                               ,g, <br />                                                                                         L 7 n A ,QT L n Q K L L 1 n n <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ . - - - - . - - - - - - . - - - - - o- - - ties <br />                                                                                                                              a[&quot;     8n a L i <br />                                                                                                                                               s.y <br />                                                                                                                                                 [ <br />                                                                                                                                                 [Q                    TD <br />                                                                                                                                                                     I IDL T X <br />                                                                                                                                                                  T Dv <br />                                                                                                                                                                               A ~I R AR488 <br />                                                                                                                                                                                    X 293 <br />  <br />  <br />                      _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - - - - - - - - - - - - - - - - - - - - - - - 0- - - - - - - of <br />                                                                                                                                                                                                   ExpreswdSequenceTag <br />                                                                                                                                                                                            P K X T T X I I D W <br />                                                                                                                                                                                                   databases. Sequence identi- <br />                                                                                                                                                                                                   -----------------.-. <br />                                                                                                                                                                                                                                                        .I;]. <br />                                                                                                                                                                                                   comparedsequences <br />                                                                                                                                                                                                   outlined conservative <br />                                                                                                                                                               0RaZ[&apos;ET:m <br />                                                                                                                                                         n L n L X a rm <br />               ::: <br />                                                                                                                                                                                        558                                             ~ <br />  <br />  <br />                                                  ~ = A A ~ K L L IK C I P D S W Q ~ ~ ~ D ~ rA r                                 ~rQ R T <br /> =:e                                                                                                                                                                                               amino acid      changes       are <br />                     :      ;     ~     ~     ~     D    ~       ~      ~       A      W    U    L   K    K   S     V     V    D <br />                                                                                                                                        c c c n r n x&apos;.E* &apos;                           iz3 <br />  <br />                                                                                                                                           :r <br />                                                                                                                                                                                                   shaded. Residues identified <br />  <br />  <br />                                                                              #E ::Fmru <br /> Am*&quot;            1   Txr Trx               vDr.8                 LLKxKL.PaaXn[.Lcl,~~z:~sa,:~.,c.~.                                                             x En[: <br />                                                                                                                                                                  xn <br /> M,m-lm          1    .......................................................                                                                                                           0 <br />  <br />  <br />  <br />  <br />                                                                                   l: :FmB:I; <br />                                                                                                                                                                                                   &quot;X&quot; reflect ambieu- <br /> : z z e <br /> ~         m <br /> M,IIIOSCYI~ 1 <br />               ::: :::BizB: : <br />                m <br />                71 <br />                                                                           :F::rgt <br />                     4C t~~ V ~ Q Y B - ~ I R.&quot; R L X T ~ K~ L ER R~ ~ ~ E L L O T O A L P X ~ V L X ~ I ~ ~ T ~ I39 <br />                      - - - - - - . - - - .. - - - - . - .. . .- - - - - - - - - - - L s l s V = . c I r L 9 x K = ~ ~ s I x r v r r r v * L c E y w o L 37 <br />                                                                                                                                                                                                   ously assigned amino acid <br />                                                                                                                                                                                                   residues cDNA <br />                                                                                                                                                                                          E D ~ T ~ ~ ~ ~ I ~ I E ~ L ~ ~ ~ L <br />                                                                                                                                                                                                   sequence.Notethatthe <br />  <br />  <br />                                                                                                                   K <br />                                                                                                                   yL <br />                                                                                                                    xmo <br />                                                                                                                                                                                                   posible metal binding resi-                                                                                                           -I <br />  <br />  <br />  <br />  <br /> C.efagans 628       v L II                                ~       T       ~       W     E         R T T x ~ aL T L X L R X <br />                                                                                               EAX W T                                L A K O L Q X X E O ~697L D;lues                               ~ L XA 6. E   ~~ <br />                                                                                                                                                                                                                    e@mare     con- <br /> S.CBRVills.434       D        ~       L         F : e BZ. - . r . =~a . . . x x .             xn r T * r             T s D I n s I L T I x Q II v x D A x X x I - - - - - - 494                 sewed comparisons to <br /> AUMfbM                                                                                                                                                                                  175 <br />                                                                                                    ,r T [ m D r n E m nKT p m 1v A Q X p n L n K K p 107 putative Arabidopsis <br />           140        I L L N n R L Q D P f M X B L E             r P X D H P K n T X r X I I b I T <br /> M. m m ~ m 18        p K c n A n L r D E T L p p rm&amp;?&amp;IJn                             uTR rA <br />                                                                                                                                                                                                   homolog seen <br /> c. elsganr <br /> S.CW&amp;E495 <br />  <br /> M. # <br />    N <br />    l&amp;Wb108 <br />               698    8 DS l p D S D B <br />                      - - - - - - - - - - - - - - - - - - ~ - - - - - - - - - - - - - - - - - - - - R ~ ~ R ~ l ~ ~ &apos; ~ E L ~ K K A ~ ~ ~5 2 5Q P T The <br />  <br />                      A L T      m     X     D    AE <br />                                                     G <br />  <br />  <br />                                                       G <br />                                                         ]     D D m D s 8 D a s 8 X 8 E p 8 p P X E X x n K n E E 8 S X E K 8 K 8 n I n R D n D E R A E 767 <br />  <br />  <br />                                                               A 8 DQS = <br />                                                                                                                                                           n                            0 <br />                                                                                                                                                                                          131 <br />                                                                                                                                                                                                Saccharomyces gene. <br />  <br />                                                                                                                                                                                                   858 mRNA <br />                                                                                                                                                                                                           K X ~ ~ sequence <br />                                                                                                                                                                                                    Q O P DNA      ~ ~ <br />  <br />                                                                                                                                                                                                               beenhas <br />                                                                                                                                                                                                                              Id- <br />                                                                                                                                                                                                                               sub- <br />                                                                                                                                                                                                   mitted GenBank (acces <br /> c.elsgam     768 <br /> S.CeIBviril# 526    L <br />                      I ~ ~ D ~ ~ V ~ ~ X D X D ~ R ~ ~ Q D <br />                          ~        L       Q   K       D   T           ~       I   U   S   R <br />                                                                                                   K ID DI II U K R D ~ ~ P I~ ~~ Q~ D D <br />                                                                                                   ~       ~       ~   P       ~       T <br />                                                                                                                                              S D D ~ D I X O R 837 <br />                                                                                                                                               V       X   T   R   ~       ~       ~    ~   0 <br />                                                                                                                                                                                          I ~ ~sion <br />                                                                                                                                                                                            ~       ~ <br />                                                                                                                                                                                                         D ~ U19615). <br />                                                                                                                                                                                                      ~ no. <br />                                                                                                                                                                                                            ~ <br />                                                                                                                                                                                                                ~ ~ ~ ~ ~ ~ D ~ ~ ~ ~ ~ <br />                                                                                                                                                                                                                    P       ~   ~   O           ~       ~   n   E 578 <br />                                                                                                                                                                                                                                                                    n        n       ~       ~   n   T       ~       ~       ~   R   R        Q       ~       E   ~   ~ <br />  <br />  <br />  <br />  <br /> C.-m          838   n ~ ~ ~ x ~ n n ~ n ~ n s ~ n n ~ ~ n n ~ n n n ~ n n ~ n ~ ~ n n ~ n ~ ~ n o n n n898 <br />                                                                                                                        ~ v ~ ~ ~ ~ n n n n n - - - - - - - - - <br />  <br />  <br />  <br />  <br /> to eventual depletion maternally donated prod-                                                                                                                             sense-induced arrest cell divisions(not shown). This <br /> ucts. To effectively examine development                                                                                                                              gastrulation defect phenotypeis similar thatobserved <br /> absence let-858 product, used antisense                                                                                                                             embryos arrested antisense RNA large <br /> RNA approach pioneered Guo KEMPHUES                                                                                                                                     subunit RNA polymerase 11, amnl (POWELL-COFF- <br /> (1995). Antisense RNA seen specifically                                                                                                                           MAN et al. 1996), mutations presumably <br /> effecthely eliminate expression fGr widevariety                                                                                                                              interfere early <br />                                                                                                                                                                                                      zygotic        gene expression (see <br /> maternally expressed genes ( e g . , Guo KEMPHUES                                                                                                                             BUCHER    SEYDOUX     1994). <br /> 1996a; MELLO et ul. 1996). Antisense injections let-                                                                                                                             A requirement &quot;858 proliferation the <br /> 858 lead earlier arrest phenotype                                                                                                                             g e d e revealed transgene    rescueexperi- <br /> observed homozygous offspring (Figure 5B). Anti-                                                                                                                               ments: C. eleguns transgenes generally con- <br /> sense-arrested embryos develop ballof-120                                                                                                                                  sist large tandem arrays assembled multiple <br /> cells,with evident differentiation gut, muscle,                                                                                                                             copies injected DNA, arrays frequently <br /> pharynx hypodermis. Although early cell move-                                                                                                                             rise structures aretransmitted stable and <br /> ments observed, did observe gastrulation,                                                                                                                             heritable extrachromosomal arrays (STINCHCOMB        et ul. <br /> which normally occurred anti-                                                                                                                               1985).An observation groups been <br />  232                                                        W. G. Kelly rt al. <br />  <br />  <br /> A                  let-858 <br />  <br />  <br />       e: w  ..i,         *.%&lt;i .rlw)          h&apos;f,*,    2.8 Kb <br />  <br />  <br />  <br />  <br /> B <br />                      - <br />                      bZIP <br />  <br />                                                                            ~~        ~                   ~~~~ <br />  <br />  <br />  <br />  <br />       1     2      3        4 5 6 7 <br />    FIGCRE3.-Nortllern         blot analysis let-858 mRNA. (A) <br /> Polv-A+ RNA cle\~elopmentallystaged aninlals puri- <br /> tiecl, separated gel electrophoresis,blotted probed <br /> with random priming-lahelecl L/-858 cDNA. Lane 1, etn- <br /> hryos; lane2. L1 larvae; lane 3. L3/L4 latvae; lane 4 5 , <br /> 1X 2X loading aclult RNA, respectively; lane 6 , glp <br /> 4(bn2) adults; lane 7, , j h - I adults. (B) As loading control, <br /> the blot reprobed labeled construct specific <br /> for bBP, factor present n developmental stages <br />  (P. OWN\, personal communicarion).                                       FIc:llw. 4.-Localization    lek858 gene products. (A and <br />                                                                        B) In situ analysis let-858 tnRNA mount wild-type <br /> that arrays tend express poorly germline tis-               embtyos. Emblyos hybridized let-858 antisense probe <br /> sues (FIREP/ nl. 1990; HOPE1991). We array                  visualized using alkaline-phosphatase-niediated detection. h - <br />                                                                        858 mRNA (purple color) appearsuniformly localized all <br /> transgenes let-8358 genomic constructs                  stages emblyos; X. DAPI; B, alkaline phosphatase. (C-E) <br /> capable efficiently rescuing embryonic lethality                Antibody analysis lef-858protein mount wild-type <br /> seen homozygotes, animals devel-                    embtyos (C D) adult (E). FITGlabeled secondary <br /> oped adulthood. The resulting adult animals, how-                   antibodies used locate specific binding mouse anti- <br /> ever, sterile (Figure 5D). Examination                   LET-858 antibodies. Arrow heads C D indicate tnitotic <br />                                                                        cells; inset n E enlargement pachytene-stage <br /> animals revealed lackof        gerrnline proliferation:              nuclei adult gonad. <br /> whereas primordial germ cells, Z2 23, were <br /> observed newly hatched larvae (not shown), evi- <br /> dence germ cells seen later lalvae                   array, GFP fluorescence seen somatic cells, but <br /> adults. This phenotype similar seen glp                  markedly absent primordial germ cells 22 and <br /> (germ line proliferation) mutants (SWEDI.1996).                        23.Later development, gonad appearsto consist <br />    Using sequences A. victorin green fluores-                 entirely ofGFP-positive      somatic gonadal cells (not <br /> cent protein (GFP), dptagged version k/-858                   shown). The results consistent hypothesis <br /> used investigate expression function                lackof rescue n thegennline poor <br /> of gene. This construct produced inserting                  transgene expression germline tissue. I n addition, <br /> the GFP coding region, frame, fourth exon                  results indicate absenceof let-858 func- <br /> of kt-838. All let-838 coding sequences present              tion, germ cells incapable proliferation. <br /> in resulting openreadingframe. This construct,                        Generationalsilencing let-858txansgenesin the <br /> when introduced standard transgene arrays                    germline: We examined expression d, <br /> wild-type genetic background. results n GFP fluores-                 tagged let-858 constructs establishment of <br /> cence highly nuclear localized. In established                 transgenic lines. In experiments, wesaw occa- <br /> transgenic lines, activity observed somatic nu-              sional low-level expression GFP adult germline <br /> clei, striking absence GFP activity isseen             generations following injection. After a <br /> germline (Figure6, A B). Like parent construct,                number generations, lines established, no <br /> the ,g@tagged IF/-858rescues embryonic defect                  expression n germline observed. These results <br /> yields sterile adults.                                                 suggest generational silencing let-858 transgenes in <br />    I n newly hatched animals homozygous                 germline.This silencing restricted germline <br /> chromosomal kt-8558 mutation carly transgene                   expression transgene, levels observed n <br />                                                Germline Solmatic Expression                                             233 <br />  <br />  <br />  <br />  <br />                                                                                                 .    &quot;. <br />  <br />  <br />  <br />  <br />                                                    -    .   I <br />  <br />  <br />  <br />  <br />                                                                     . I                                      ..&quot;-      * <br />  <br />  <br />  <br />  <br />    FIGURE5.-kt-858 loss function phenotypes. (A) Arrest <br /> phenotype kt-858 homozygous offspring heterozy- <br /> gous parent. Large vacuole head region indicated. (B) <br /> Arrest phenotype F1 embryo wild-type parent injected <br /> with anti-kt-858 mRNA syncytialgonad. (C) Distal and <br /> proximal region arm wild-type adult hermaphro- <br /> dite. (D) Similar region residual gonad seen kt- <br /> 858 homozygotes rescued embryonic lethality line <br /> transgenic 6.8-kb kt-858 genomic construct shown in <br /> Figure 1C. While somatic structures gonad <br /> appear present (bracket), descendants 22 and <br /> 23 germ cell lineage detected. <br />  <br /> somatic tissue appreciably change (Figure 7). <br /> Pedigree analyses lines low levels germ- <br /> line expression initially observed expres-            FIGURE  6.-Contextdependent germline       expression a <br /> sion increasingly mosaic, expression of-            let-858::gfp reporter construct. (A B) Nomarski (A) and <br /> ten detectable arm gonad. The fre-            fluorescence (B) micrographs showing gonadal arm a <br /> quency silencingincreases generation,               heterozygous kt-858 adult transformed in-frame kt- <br />                                                                  858::gfpfusion construct, pBK48.1, standard injection <br /> until germline expression effectively absent. The si-         protocol [plasmids pBK48.1 pRF4 (rol8d) 1:5 ratio <br /> lencing does appear easily reversible,           mass]. Transgenic animals obtained protocol that <br /> we observed reemergence expression               homozygous kt-858 mutation exhibit identical <br /> in silenced lines (data shown).                              phenotype illustrated Figure 5D. The bracket indi- <br />    Context-mediated desilencingof l&amp;58 transgenesin              cates syncytial germ cell region gonad. (C-F) No- <br />                                                                  marski (C E) fluorescence (D F) micrographs <br /> thegennline: We undertook approaches                  gonadal arm (C D) early embryos (E F) <br /> obtaining let-858 transgenes germline function. In          kt-858 homozygotes transformed bypBK48.1 coinjected <br /> initial experiments,we looked animals whichthe            50- 100-fold mass excess random genomic DNA <br /> germline silencingof array relieved. This screen         fragments described MATWUAIS AND METHODS. <br /> used strain carryinga let-858 transgene,with initial <br /> genetic background heterozygous fora chromo-               germline defect dependent transgene. <br /> somal let-858 mutation. A linked marker (unc-4)              One line obtained, analysis this <br /> used obtain population animalshomozygous                 line indicated transgene array altered <br /> for chromosomal let-858mutation. As noted above,             capable germline expression. Although <br /> most animalsare sterile. We screened pop-          nature change inthis transgene hasnot been <br /> ulation fertile animals, looking   rescue         investigated, properties indicated cer- <br />                                                         W. G . Kelly et nl. <br />  <br />                                                                      obtained. Of these, majority showed robust <br />                                          + Germline <br />                                                  (N2 DNA)            expression germline soma (Figure 6, C - <br />                                             --+..- Germline          F) . Examination expression severalgenerations <br />                                                      (pRF4 plasmid)  showed resulting transgenic lines main- <br />                                             ....&amp;... Soma            tain germline expression (Figure 7), in <br />                                                      (N2 carrier) <br />                                                                      cases seen decrease continued <br />                                             + Soma                   passage (not shown). Several ofthe lines germline <br />                                                      (pRF4 plasmid) <br />                                                                      expression tested formutant rescue were <br />                                                                      rescue somatic germline defects <br />                                                                      let-858 mutations. The fertile animals resulting from <br />                                                                      rescue maintained homozygotes indefinitely <br />           1     2      3      4    5     6 <br />                                                                      (for 5 2 4 generations date). <br />                      Generation <br />                                                                         The improvementingermline           expression lines <br />     FIGURE7.-Contextdependent generational silencing              constructed using C. ekguns DNA carrier con- <br />  extrachromosomal let-858::g;fpreporter construct. Wild-type         ceivably eitherto nature carrier DNA <br />  hermaphrodites wereinjectedwiththepBK48.1            let-858::dp <br />  reporter construct absence ( + A) pres-        effective dilution let-858::dp sequences with <br />  ence ( 0 A ) N2 DNA fragments described (MATERI-          excess carrier. To distinguish possibilities, car- <br /> A I S AND METHODS). Transgenic lines resulting in-        ried set standard transformation experiments <br /> jections followed generations,starting        @,,tagged let-858 plasmid diluted with <br />  FP, assessedforfluorescence        intensity soma       increasing ratios rol-6 plasmid pRF4 molar ratios <br />  (A A) germline ( + 0).       Relative intensity was <br />                                                                      31, 48:1, 192:l (rol-6/let-858&amp;p) used; no <br />  determined comparing nuclear g;fp respective tissues <br /> with observedin &#226;&#8364;&#339;benchmark&#226;&#8364;? line (see MATERIAIS               complex    carrier DNA included. We thatsuch <br /> AND METHODS).                                                        simple  dilutions  let-858 sequences sufficient <br />                                                                      produce transgenic lines germline expression <br />  tain features transgene array critical              (five lines examined concentra- <br /> for observed silencing effects.                                  tion). Instead, dilution kt-858 sequences resulted in <br />     Therefore, second approach obtaining                 lines exhibited overall decreases somatic expres- <br />  transgene activity germline, hoped modify              sion germline expression observed (data <br />  transformation protocol repetitive na-              shown).This indicated nature     carrier <br />  ture transgene arrays minimized. In standard             DNA ratherthanthe           dispersal dilution let- <br />  transformations, test construct injected mix-             858::dp sequences key feature allowing <br />  tures composed test plasmid                 germline expression. <br /> plasmid (usually similar vector backbone) containing                 Proximalsequencerequirements germliie and <br />  dominant transformation marker.This necessarily re-               somatic expression let-858: We looked &amp;acting <br />  sults thein vivo construction low complexity arrays           sequences let-858 rescuing fragment might <br />  highly repetitive nature. In various genetic             specificallyrequired forits germline expression. This <br /> systems, tandem repeats shown lead                   analysis involved sequential deletionsof 5&#226;&#8364;&#8482; flanking <br />  heterochromatic effects resulting gene silencing                 DNA (FigureSA),deletions intronandexon               se- <br />  (DORERand HENIKOFF                     L HENIKOFF 1996). quences (Figure SB), replacement 3&#226;&#8364;&#732; UTR <br />                             1994; S A ~ and <br />  In previous studies somatic enhancer function,                non-germline <br />                                                                                                   gene           (Figure SB, <br />  observed certaintransgenes exhibit decreased              pPD100.29). In cases,we similar requirements <br /> activity inactive context long tan-           germline somatic expression; i.e., removal of <br />  dem arrays (OKKEMA        et al. 1993; KFuUSE et al. 1994).         element affecting germline expression hadcon- <br /> We recently observed effects                 comitant effects somatic expression. At 5&#226;&#8364;&#8482; end <br />  somatic enhancer function mitigated               gene, deletion -184 leads dramatic loss <br />  complex carrier DNAs used transformation                 activity bothsoma germline, -201 dele- <br />  protocol (A. FIRE,S. Xu, J. HSICHand B. HARFE, unpub-               tions fully active tissues. Although data <br />  lished observations). In attempt minimize ef-             strongly suggests similar set transcription factors <br />  fect repetitive arrays gene expression germ            mediating germline soma expression let-858, we <br />  line, performed set injections short               rule possibility separate germline <br />  linear random segments genomic DNA C. ele-                  somatic activating sequences exist interven- <br />  guns used carrier. It hoped tested              ing region. <br />  construct dispersed imbedded                       Context-dependent silencingin germline be <br />  regions high sequence complexity array              general   feature ubiquitously expressed genes:We <br />  approximate euchromatin.                                 used  similar   methods examine requirements for <br />     Numerous transgenic lines resulting co-injec-               germline     somatic expression second broadly <br />  tion of&amp;,tagged       let-858 withcleaved genomic DNA               expressed gene, unc-37. MILLERand colleagues (PFLU- <br />                                                                    Germlinc Somatic Expression                                                          235 <br />  <br />                    A <br />  <br />                                                kQ 1 <br />                                                U <br />                                                =       1 <br />                                                % <br />                                                            0 <br />                                                                -566 -528 -508 -492 -460 -414 -338 -292 -240 -201    . l M -124 <br />  <br />  <br />  <br />  <br />                                                                                       5&apos; END <br />  <br />  <br />  <br />  <br />                   B                                                                                                               I   . . <br />                                                                                                                                             GFP <br />                                                                                                                                  FLUORESCENCE <br />  <br />  <br />  <br />  <br />                           - <br />                                   1   2    3   4       5                                 6 7 8     9     10 <br />  <br />  <br />               pPD103.21   -1                                               YES                                                   YES <br />                                                4       5                                 6 7 8     9     10 <br />  <br />               pPD109.29                                                                 ell- YES                                 YES <br />                                   1   2    3   4       5                                 6 7 8     9     10 <br />  <br />  <br />               pPD99.88                                                                                         Q- <br />                                                                                                                10 <br />                                                                                                                                  YES          YES (weak) <br />                                   1    2   3   4   5 <br />  <br />  <br />               pPD104.17 4      - H &quot; <br />                                   1   2    3   4       5                                 6 7 <br />                                                                                                                b- <br />                                                                                                                10 <br />                                                                                                                                  YES          YES <br />  <br />               pPDlW.29                                                                                                           YES <br />                                   1    2   3   4       5                                 6 7 8     9     10 u n c - 5 8 3 UTR <br />  <br />  <br />    FIGURE8.-Deletion analyses kt-858 genomic sequences identification germline somatic specific control <br /> elements. (A) 5&apos; deletions: deletions 5&apos; flanking DNA pBK48.1 constructed, cwinjectcd N2 genomic fragments, <br /> and resulting lines assessed g f j ~fluorescence soma germline described (MAT\TI.:RIAISAZNDM E I I I O D S ) , <br /> with data presented average lines tested construct. Relative fluorescence refers nuclear gfjj <br /> intensity respective tissue compared somatic nuclear intensity benchmark line,which assigned value <br /> of 3.0. x axes indicate 5&apos; bp kt-858genomic sequence remaining construct,     basepair the <br /> initiator codonset + 1. (B) Exon/intron deletions. S&apos;UTR replacement: in-frame deletions ofcodingsequences     intervening <br /> sequences constructs derived parent pBK48.1 plasmid illustrated gaps sequence diagram each <br /> construct. pPD109.29 generated replacement 01&apos; genomic sequence corresponding cDNA <br /> sequence, creating in-frame deletion 3&apos; end exon 5, introns 5 6. pPD100.29 truncation <br /> of exon 10 COOH terminus, kt-8583&apos; UTR replaced 3&apos; UTR myosin <br /> heavy-chain gene, unc-54. <br />  <br /> GRAD el  dl. 1997) shown unc-37 encodes                                               germline somatic cell types. We tagged unc- <br /> C. ekgalzs homologue Drosophila grouch0 tran-                                                3 7 gf$, using similar approach used for <br /> scription factor, factor present                                            Ipt-858. &quot; h e n introduced high copy tandem arrays <br />  236                                                W. G. Kelly et al. <br />  <br /> by standard transformation techniques, observe ini-        different sets heterochromatin sensing and/or <br /> tial low levels ofexpression germline tissue, followed     establishment mechanisms exist soma germ- <br /> by complete silencing generations ob-         line. Perhaps germline maintains generalized si- <br /> servedwith let-858. Ubiquitous somatic expression          lencing DNA, germline specific housekeep- <br /> observed continuously. Injection &amp;fptagged unc-        ing genes protected silencing specific sequence <br /> ?7 context complex C. elegans carrier DNA           elements. <br /> results strong expression germline. This ex-           This model suggest germline chromatinis <br /> pression persists passage. Thus unc-?7 transgenes        structurally different somatic chromatin. Indeed, <br /> exhibit properties observed let-858. An-      chromatin morphology midstage embryonic germ- <br /> other broadly expressed gene, cap-2, recently        line nuclei appearsmorecompactthanthat               sur- <br /> assayed using protocol fortransformation com-        rounding somatic cells. This difference seen early <br />             u. <br /> plex carrier WADDLE,      personal communication);       P4 blastomere maintained into <br /> gene appears depend presence com-           early stages oflarval germline proliferation (W. <br /> plex carrier DNA germline expression.                     KELLY, unpublished observations; G. SEYDOUX,              T. <br />                                                               SCHEDL,  J. PRIESS,personal communications). From this <br />                      DISCUSSION                               perspective, interesting note DPY-26, struc- <br />                                                               tural protein involved X chromosome-specific tran- <br />    Distinctcontextualrequirementsforsomatic               scriptional repression somatic tissue, bound to <br /> germline gene expression: We germ-            autosomal X chromosomal DNA germ cells <br /> line somatic tissuesof C. elegans display distinct        (LIEBet al. 1996). <br /> criteria efficient transgene expression. Expression          The C. elegans germline employs multiple selective <br /> in soma long known occur large           mechanisms prevent <br />                                                                          expression                       somatic <br /> arrays injected plasmid sequences;        genes: We demonstrated paper C. <br /> cases, expression shown reflect proper       elegans germline capable silencing expression <br /> regulation. By contrast, definitive expression re-         genes presented repetitive array context. We <br /> porter transgenes C. elegans germline        hypothesize means the <br /> been observed work.                               germline prevents somatic gene expression conse- <br />    Our data indicate ubiquitously       quently avoids differentiation somatic pathways. <br /> expressed genes,  the <br />                     context          theintroduced         We found, using let-858 unc-?7 reporter trans- <br /> transgene critical determining whetherexpression        genes repetitive arrays, germline silencing <br /> will occur germline. Embedding trans-         mechanism active larval adult life. It <br /> genes complex sequences derived C.            likelythat different mechanisms <br /> elegans genome sufficient allow stable expression       employed C. elegans germline suppress the <br /> in germline. One possible explanation         expression somatic genes. In early embryo, a <br /> that germline view repetitive arrays        different problem presented germline. Rapid <br /> heterochromatic act silence expression.          divisions germline precursors act stem-cell- <br /> Alternatively, specific features C. elegans sequences      like fashion produce series somatic blastomeres <br /> might required reliefthe silencing thegerm-      rapidly acquire aspects somatic identity. <br /> line. If features involve specific sequence ele-      Previous studies (SEYDOUX       FIRE1994) havesug- <br /> ment,then elements wouldhave               gested early stages, complete block to <br /> abundant chromosomal DNA and/or capable act-            germline mRNA expression place. It conceivable <br /> ing large distances, mgority N2           sensitive stage germline so- <br /> DNA-containing arrays (each different DNA com-           matic cells diverge common cytoplasm, there <br /> position structures) proved capable germline           stringent requirement preventing <br /> expression.                                                   early inappropriate somatic expression. The mater- <br />    The silencing repetitive arrays germline     nal factor pie-1 shown play key role in <br /> occurs course generations, pro-      repression new gene expression germline blasto- <br /> cess understood      know        meres (SEYDOUX      et al. 1996). pie-l products de- <br /> of clear precedent. Silencing effects repeated          tectable following middle stages embryogenesis, <br /> transgenes seen avariety species (h/LAR-      unlikely PIE-1 playsa directrole later <br /> TIN WHITELAW       1996). The sensitivity repetitive   suppression somatic gene expression (MELLO et al. <br /> sequence, increasing mosaicism, reminis-          1996). Intriguingly, genes identified <br /> cent position effect variegation repeat-induced        codingsequences similar pie-1 (MELLO et al. <br /> silencing seen Drosophila (DORERandHENIKOFF                1996; GUEDES     et al. 1997). It tempting speculate <br /> 1994; SABLand HENIKOFF1996). These observations               play later role in <br /> lead hypothesis C. eleguns germline     context-dependent silencing. <br /> perceiving arrays heterochromatic, suggesting           Whatever mechanism context-dependent si- <br />                                              Germline Somatic Expression                                                   237 <br />  <br /> lencing, process applicable variety       transcriptional activity (STARCK1977) nu- <br /> stages differentiation. This includes mitotically      clei: transcriptionally active stages diaki- <br /> quiescent germ cells late embryo early L1          nesis, LET-858appears colocalize chromatin; in <br /> larva (22and Z3), mitotically active germ cells        transcriptionally silent mature oocytes LET-858 uni- <br /> later larvae, mitotically meiotically        formly dispersed nucleus. <br /> active stages ofgerm cells seen adults. Some common           We thank G. SEYDOUX,   L. COLES,A. COULSON,   D. MILLER,  J. MEIR, <br /> feature nuclei responsible         J. PRIESS,T. SCHEDL,J. WADDLE,  P. OKKEMA, B. HARFE,J. GOODIJFFE, <br /> germ-line specific behavior.                                  S. KOSTAS, J. GALL,A. PINDER, members  lab help, <br />    Inaddition    mechanisms described above,           sharing unpublished data, suggestions course of <br />                                                               work. This work supported U S . Public Health Senice <br /> there additional mechanisms protect the <br />                                                               grants (A.F.,R01CM-37706; W.K., F32-HD-07794; M.M., F32-GM- <br /> germline expression somatic foreign genes.          17164) CarnegieInstitution Washington.Somestrains <br /> A suggestion mechanism comes             used work provided Caenorhabditis Genetics Cen- <br /> from (so far unsuccessful) attempts use kt-858         ter (funded NationalInstitutes Health Center Research <br /> and unc-37 generate general vectors expressing         Resources). <br /> arbitrary coding sequences germline. We find <br /> that fusions containing just 5&#226;&#8364;&#8482; sequences kt-858                           LITEFCATURECITED <br /> unc-37 sufficient drive somatic, germline      AUSTIN,  J,, J. KIMBLE,1987 glpl required germ line <br /> accumulation GFP P-galactosidase reporter pro-             regulation decision mitosis meiosis C. <br /> teins (W. KELLY A. FIRE,unpublished observations).            ekgam. Cell 51: 589-599. <br />                                                               ALTSCHUL,    S., W. GIsH, W. MILLER,E.MYERSand D. J . LIPMAN,1990 <br /> Thus sequences genes, features             Basic local alignment search tool. J. Mol. Biol. 2 1 5 403-410. <br /> the mRNA protein sequences, areimportant               BRENNER,   S., 1974 The genetics Caaorhabditzs elegans. Genetics <br /> germline activity.                                                77: 71-94. <br />                                                               BUCHER,   E. A,, G. SEYDOUX,   1994 Gastrulation nematode <br />    Inaddition thepotential roles silencing               Caenurhubditis eleguns. Semin. Dev. Biol. 5 121-130. <br /> developmental distinctions, conceivable silenc-    CHAI.FIE, M., Y. Tu, G. EUSKIRCHEN,   W. W. WARDand D. C. PRASHER, <br /> ing mechanisms help prevent proliferation viruses            1994 Green fluorescent protein marker gene expres- <br />                                                                   sion. Science 263: 802-805. <br /> and transposons. From perspective, intriguing      DORER,D. R., S. HENIKOFF,1994 Expansions transgene re- <br /> that dispersed multicopy transposon Tcl prefer-            peats cause heterochromatin formation gene silencing in <br /> entially active somatic tissue (EMMONS     YESNER          Drosophila. Cell 77:993-1002. <br />                                                               EMMONSS. W., L. YESNER, 1984 High-frequency excision of <br />  1984).                                                           transposable element Tcl nematode C. elegans limited <br />    LET-858 defines evolutionarilyconservedand                   somatic cells. Cell 36: 599-605 <br /> ubiquitousnuclearfactorthat essential somatic           FERCUSON,    K. C., P. J. HEID andJ.H. ROTHMAN,   1996 The SL1 trans <br />                                                                   spliced leader RNA performs essential embryonic function in <br /> andgermlinedevelopment: We given                     Cuenmhabditis elegans supplied SL2 RNA. <br />  &#226;&#8364;&#339;nucampholin&#226;&#8364;? kt-858 product indicate               Genes Dev. 10: 1543-1556. <br /> nuclear localization amphipathicnature ofthis             FIRE, A. 1986 Integrativetransformation Cuenorhabditis elegans. <br />                                                                   EMBO J. 5: 2673-2680. <br /> product. The apparentevolutionary conservation sin-        FIRE,A., S. W. HARRISON     D. DIXON,1990 A modular set lacZ <br /> gle cell multicellular eukaryotes underscores             fusion vectors studying gene expression Caenorhabditis <br /> importance nucampholin cellular events. At             elegans. Gene 93: 189-198. <br />                                                               FIRE,A,, D. ALBERTSON,      S. W. HARRISON D. G. MOERMAN,     1991 <br /> point, defined C. ekgans nucampholin en-              Production antisense RNA leads effective specific inhi- <br /> coded let-858at genetic, molecular cytologi-           bition gene expression C. elegans muscle. Development 113: <br /> callevels. The nuclear localization protein             503-514. <br />                                                               Guo, S., K. J. KEMPHUES,     1995 par-I, gene required estab <br /> suggestive key role gene expression overall         lishing polarity C. elegans embryos, encodes putative ser/thr <br /> nuclear architecture. Consistent hypothesis,            kinase asymmetrically distributed. Cell 81: 61 1-620. <br /> the kt-858 antisense phenotype similar timing      GUo, S., K J. KEMPHUES, 1996a A non-muscle myosin required <br />                                                                   embryonic polarity Caaorhabditis &amp;guns. Nature 3 8 2 455- <br /> and severity reported elimination RNA           458. <br /> polymerase I1 function (POWELL-COFFMAN        et ul. 1996).   Guo, S., K. J. KEMPHUES,    199613 Molecular genetics asymmet- <br /> The disruption gastrulation observed zen-1          ric cleavage early C. &amp;guns embryo. Curr. Opin. Genet. <br />                                                                   Dev. 6: 408-415. <br /> mutants. zen-1 encodes theSL1 transplice leader precur-       GUEDES,S., J. R. PRIES, 1997 The C. elegans MEXl protein is <br /> sor involved normal processing majority             present germline blastomeres P granule component. <br /> of C. eZeguns mRNAs (FERGUS <br /> </body></html>